<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/6/1623" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/6/1623.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/6/1623.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/6/1623.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/6/1623/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/6/1623.atom" />
<meta name="HW.pisa" content="ggg;7/6/1623" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Inbred Strain Variant Database (ISVdb): A Repository for Probabilistically Informed Sequence Differences Among the Collaborative Cross Strains and Their Founders" />
<meta name="DC.Identifier" content="10.1534/g3.117.041491" />
<meta name="DC.Date" content="2017-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Oreper et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Collaborative Cross (CC) is a panel of recently established multiparental recombinant inbred mouse strains. For the CC, as for any multiparental population (MPP), effective experimental design and analysis benefit from detailed knowledge of the genetic differences between strains. Such differences can be directly determined by sequencing, but until now whole-genome sequencing was not publicly available for individual CC strains. An alternative and complementary approach is to infer genetic differences by combining two pieces of information: probabilistic estimates of the CC haplotype mosaic from a custom genotyping array, and probabilistic variant calls from sequencing of the CC founders. The computation for this inference, especially when performed genome-wide, can be intricate and time-consuming, requiring the researcher to generate nontrivial and potentially error-prone scripts. To provide standardized, easy-to-access CC sequence information, we have developed the Inbred Strain Variant Database (ISVdb). The ISVdb provides, for all the exonic variants from the Sanger Institute mouse sequencing dataset, direct sequence information for CC founders and, critically, the imputed sequence information for CC strains. Notably, the ISVdb also: (1) provides predicted variant consequence metadata; (2) allows rapid simulation of F1 populations; and (3) preserves imputation uncertainty, which will allow imputed data to be refined in the future as additional sequencing and genotyping data are collected. The ISVdb information is housed in an SQL database and is easily accessible through a custom online interface (&lt;http://isvdb.unc.edu&gt;), reducing the analytic burden on any researcher using the CC." />
<meta name="DC.Contributor" content="Daniel Oreper" />
<meta name="DC.Contributor" content="Yanwei Cai" />
<meta name="DC.Contributor" content="Lisa M. Tarantino" />
<meta name="DC.Contributor" content="Fernando Pardo-Manuel de Villena" />
<meta name="DC.Contributor" content="William Valdar" />
<meta name="article:published_time" content="2017-06-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Inbred Strain Variant Database (ISVdb): A Repository for Probabilistically Informed Sequence Differences Among the Collaborative Cross Strains and Their Founders" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The Collaborative Cross (CC) is a panel of recently established multiparental recombinant inbred mouse strains. For the CC, as for any multiparental population (MPP), effective experimental design and analysis benefit from detailed knowledge of the genetic differences between strains. Such differences can be directly determined by sequencing, but until now whole-genome sequencing was not publicly available for individual CC strains. An alternative and complementary approach is to infer genetic differences by combining two pieces of information: probabilistic estimates of the CC haplotype mosaic from a custom genotyping array, and probabilistic variant calls from sequencing of the CC founders. The computation for this inference, especially when performed genome-wide, can be intricate and time-consuming, requiring the researcher to generate nontrivial and potentially error-prone scripts. To provide standardized, easy-to-access CC sequence information, we have developed the Inbred Strain Variant Database (ISVdb). The ISVdb provides, for all the exonic variants from the Sanger Institute mouse sequencing dataset, direct sequence information for CC founders and, critically, the imputed sequence information for CC strains. Notably, the ISVdb also: (1) provides predicted variant consequence metadata; (2) allows rapid simulation of F1 populations; and (3) preserves imputation uncertainty, which will allow imputed data to be refined in the future as additional sequencing and genotyping data are collected. The ISVdb information is housed in an SQL database and is easily accessible through a custom online interface (http://isvdb.unc.edu), reducing the analytic burden on any researcher using the CC.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/06/01" />
<meta name="citation_mjid" content="ggg;7/6/1623" />
<meta name="citation_id" content="7/6/1623" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/6/1623" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/6/1623.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/6/1623.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/6/1623.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.041491" />
<meta name="citation_pmid" content="28592645" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="1623" />
<meta name="citation_lastpage" content="1630" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Daniel Oreper" />
<meta name="citation_author_institution" content="Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599-7265" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4569-5160" />
<meta name="citation_author" content="Yanwei Cai" />
<meta name="citation_author_institution" content="Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599-7265" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3950-3794" />
<meta name="citation_author" content="Lisa M. Tarantino" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265" />
<meta name="citation_author_institution" content="Division of Pharmacotherapy and Experimental Therapeutics, Eshelman School of Pharmacy University of North Carolina, Chapel Hill, North Carolina 27599-7265" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9984-2600" />
<meta name="citation_author" content="Fernando Pardo-Manuel de Villena" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5738-5795" />
<meta name="citation_author" content="William Valdar" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265" />
<meta name="citation_author_institution" content="Lineberger Comprehensive Cancer Center," />
<meta name="citation_author_email" content="william.valdar@unc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2419-0430" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. L. Aylor;citation_author=W. Valdar;citation_author=W. Foulds-Mathes;citation_author=R. J. Buus;citation_author=R. A. Verdugo;citation_title=Genetic analysis of complex traits in the emerging collaborative cross.;citation_volume=21;citation_year=2011;citation_pmid=21406540;citation_doi=10.1101/gr.111310.110" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.;citation_pages=210-223;citation_volume=84;citation_year=2009;citation_issue=2;citation_pmid=19200528;citation_doi=10.1016/j.ajhg.2009.01.005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_title=The Genome Architecture of the Collaborative Cross Mouse Genetic Reference Population;citation_pages=389-401;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=M. T. Ferris;citation_author=D. L. Aylor;citation_author=D. Bottomly;citation_author=A. C. Whitmore;citation_author=L. D. Aicher;citation_title=Modeling host genetic regulation of influenza pathogenesis in the collaborative cross.;citation_pages=e1003196;citation_volume=9;citation_year=2013;citation_pmid=23468633;citation_doi=10.1371/journal.ppat.1003196" />
<meta name="citation_reference" content="citation_author=C.-P. Fu;citation_author=C. E. Welsh;citation_author=F. P.-M. de Villena;citation_author=L. McMillan;citation_pages=105-112;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. M. Gatti;citation_author=K. L. Svenson;citation_author=A. Shabalin;citation_author=L.-Y. Wu;citation_author=W. Valdar;citation_title=Quantitative trait locus mapping methods for diversity outbred mice.;citation_pages=1623-1633;citation_volume=4;citation_year=2014;citation_doi=10.1534/g3.114.013748" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=L. E. Gralinski;citation_author=V. D. Menachery;citation_author=A. P. Morgan;citation_author=A. Totura;citation_author=A. Beall;citation_title=Allelic Variation in the Toll-Like Receptor Adaptor Protein Ticam2 Contributes to SARS-Coronavirus Pathogenesis in Mice;citation_pages=1653-1663;citation_volume=7;citation_year=2017;citation_issue=6;citation_pmid=28592648;citation_doi=10.1534/g3.117.041434" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=R. Green;citation_author=C. Wilkins;citation_author=S. Thomas;citation_author=A. Sekine;citation_author=D. M. Hendrick;citation_title=Oas1b-dependent Immune Transcriptional Profiles of West Nile Virus Infection in the Collaborative Cross;citation_pages=1665-1682;citation_volume=7;citation_year=2017;citation_issue=6;citation_pmid=28592649;citation_doi=10.1534/g3.117.041624" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Guan;citation_author=M. Stephens;citation_title=Practical issues in imputation-based association mapping.;citation_pages=e1000279-e1000279;citation_volume=4;citation_year=2008;citation_issue=12;citation_pmid=19057666;citation_doi=10.1371/journal.pgen.1000279" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=M. E. Hawley;citation_author=K. K. K. Kidd;citation_title=HAPLO: A Program Using the EM Algorithm to Estimate the Frequencies of Multi-site Haplotypes;citation_pages=409-411;citation_volume=86;citation_year=1995;citation_issue=5;citation_pmid=7560877" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J. M. Hickey;citation_author=B. P. Kinghorn;citation_author=B. Tier;citation_author=J. H. J. van der Werf;citation_author=M. A. Cleveland;citation_title=A phasing and imputation method for pedigreed populations that results in a single-stage genomic evaluation.;citation_pages=9-9;citation_volume=44;citation_year=2012;citation_pmid=22462519;citation_doi=10.1186/1297-9686-44-9" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. N. Howie;citation_author=P. Donnelly;citation_author=J. Marchini;citation_title=A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.;citation_pages=e1000529-e1000529;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19543373;citation_doi=10.1371/journal.pgen.1000529" />
<meta name="citation_reference" content="citation_author=S. Huang;citation_author=C.-Y. Kao;citation_author=L. McMillan;citation_author=W. Wang;citation_pages=595;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. M. Keane;citation_author=L. Goodstadt;citation_author=P. Danecek;citation_author=M. A. White;citation_author=K. Wong;citation_title=Mouse genomic variation and its effect on phenotypes and gene regulation.;citation_pages=289-294;citation_volume=477;citation_year=2011;citation_issue=7364;citation_pmid=21921910;citation_doi=10.1038/nature10413" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. G. King;citation_author=S. J. Macdonald;citation_author=A. D. Long;citation_title=Properties and power of the Drosophila synthetic population resource for the routine dissection of complex traits.;citation_volume=191;citation_year=2012;citation_pmid=22505626;citation_doi=10.1534/genetics.112.138537" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_author=Z. Kutalik;citation_author=T. Johnson;citation_author=M. Bochud;citation_author=V. Mooser;citation_author=P. Vollenweider;citation_title=Methods for testing association between uncertain genotypes and quantitative traits.;citation_volume=12;citation_year=2011;citation_pmid=20543033;citation_doi=10.1093/biostatistics/kxq039" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=Y. Li;citation_author=C. Willer;citation_author=S. Sanna;citation_author=G. Abecasis;citation_title=Genotype imputation.;citation_pages=387-406;citation_volume=10;citation_year=2009;citation_pmid=19715440;citation_doi=10.1146/annurev.genom.9.081307.164242" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=Y. Li;citation_author=C. J. Willer;citation_author=J. Ding;citation_author=P. Scheet;citation_author=G. R. Abecasis;citation_title=MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes.;citation_pages=816-834;citation_volume=34;citation_year=2010;citation_issue=8;citation_pmid=21058334;citation_doi=10.1002/gepi.20533" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. Y. Liu;citation_author=Q. Zhang;citation_author=L. McMillan;citation_author=F. Pardo-Manuel de Villena;citation_author=W. Wang;citation_title=Efficient genome ancestry inference in complex pedigrees with inbreeding.;citation_volume=26;citation_year=2010;citation_pmid=20529906;citation_doi=10.1093/bioinformatics/btq187" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Marchini;citation_author=B. Howie;citation_title=Genotype imputation for genome-wide association studies.;citation_pages=499-511;citation_volume=11;citation_year=2010;citation_issue=7;citation_pmid=20517342;citation_doi=10.1038/nrg2796" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Marchini;citation_author=B. N. Howie;citation_author=S. Myers;citation_author=G. McVean;citation_author=P. Donnelly;citation_title=A new multipoint method for genome-wide association studies by imputation of genotypes.;citation_pages=906-913;citation_volume=39;citation_year=2007;citation_issue=7;citation_pmid=17572673;citation_doi=10.1038/ng2088" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. McLaren;citation_author=B. Pritchard;citation_author=D. Rios;citation_author=Y. Chen;citation_author=P. Flicek;citation_title=Deriving the consequences of genomic variants with the Ensembl API and SNP effect predictor.;citation_volume=26;citation_year=2010;citation_pmid=20562413;citation_doi=10.1093/bioinformatics/btq330" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=A. P. Morgan;citation_author=C. E. Welsh;citation_title=Informatics resources for the collaborative cross and related mouse populations.;citation_pages=521-539;citation_volume=26;citation_year=2015;citation_pmid=26135136;citation_doi=10.1007/s00335-015-9581-z" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=A. P. Morgan;citation_author=C.-P. Fu;citation_author=C.-Y. Kao;citation_author=C. E. Welsh;citation_author=J. P. Didion;citation_title=The mouse universal genotyping array: from substrains to subspecies.;citation_pages=263-279;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. P. Morgan;citation_author=D. M. Gatti;citation_author=T. M. Keane;citation_author=R. J. Galante;citation_author=A. I. Pack;citation_title=Structural Variation Shapes the Landscape of Recombination in Mouse;citation_pages=603-619;citation_volume=206;citation_year=2017;citation_issue=2;citation_pmid=28592499;citation_doi=10.1534/genetics.116.197988" />
<meta name="citation_reference" content="citation_journal_title=Toxicol. Sci;citation_title=Candidate risk factors and mechanisms for Tolvaptan-induced liver injury are identified using a collaborative cross approach.;citation_author=M. Mosedale;citation_author=Y. Kim;citation_author=W. J. Brock;citation_author=S. E. Roth;citation_author=T. Wiltshire;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Mott;citation_author=C. J. Talbot;citation_author=M. G. Turri;citation_author=A. C. Collins;citation_author=J. Flint;citation_title=A method for fine mapping quantitative trait loci in outbred animal stocks.;citation_volume=97;citation_year=2000;citation_pmid=11050180;citation_doi=10.1073/pnas.230304397" />
<meta name="citation_reference" content="citation_journal_title=Genes Immun.;citation_author=J. Phillippi;citation_author=Y. Xie;citation_author=D. R. Miller;citation_author=T. A. Bell;citation_author=Z. Zhang;citation_title=Using the emerging collaborative cross to probe the immune system.;citation_pages=38-46;citation_volume=15;citation_year=2014;citation_pmid=24195963;citation_doi=10.1038/gene.2013.59" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Ramasamy;citation_author=D. Trabzuni;citation_author=J. R. Gibbs;citation_author=A. Dillman;citation_author=D. G. Hernandez;citation_title=Resolving the polymorphism-in-probe problem is critical for correct interpretation of expression QTL studies.;citation_volume=41;citation_year=2013;citation_pmid=23435227;citation_doi=10.1093/nar/gkt069" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. L. Rasmussen;citation_author=A. Okumura;citation_author=M. T. Ferris;citation_author=R. Green;citation_author=F. Feldmann;citation_title=Host genetic diversity enables Ebola hemorrhagic fever pathogenesis and resistance.;citation_volume=346;citation_year=2014;citation_pmid=25359852;citation_doi=10.1126/science.1259595" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Sargolzaei;citation_author=J. P. Chesnais;citation_author=F. S. Schenkel;citation_title=A new approach for efficient genotype imputation using information from relatives.;citation_pages=478;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. R. Shorter;citation_author=F. Odet;citation_author=D. L. Aylor;citation_author=W. Pan;citation_author=K. Chia-Yu;citation_title=Male Infertility Is Responsible for Nearly Half of the Extinction Observed in the Mouse Collaborative Cross;citation_pages=557-572;citation_volume=206;citation_year=2017;citation_issue=2;citation_pmid=28592496;citation_doi=10.1534/genetics.116.199596" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Srivastava;citation_author=A. P. Morgan;citation_author=M. Najarian;citation_author=V. K. Sarsani;citation_author=J. S. Sigmon;citation_title=Genomes of the Mouse Collaborative Cross;citation_pages=537-556;citation_volume=206;citation_year=2017;citation_issue=2;citation_pmid=28592495;citation_doi=10.1534/genetics.116.198838" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=F. Tian;citation_author=P. J. Bradbury;citation_author=P. J. Brown;citation_author=H. Hung;citation_author=Q. Sun;citation_title=Genome-wide association study of leaf architecture in the maize nested association mapping population.;citation_pages=159-162;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217756;citation_doi=10.1038/ng.746" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=P. M. VanRaden;citation_author=J. R. O’Connell;citation_author=G. R. Wiggans;citation_author=K. A. Weigel;citation_title=Genomic evaluations with many more genotypes.;citation_pages=10-10;citation_volume=43;citation_year=2011;citation_pmid=21366914;citation_doi=10.1186/1297-9686-43-10" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. P. Verbyla;citation_author=A. W. George;citation_author=C. R. Cavanagh;citation_author=K. L. Verbyla;citation_title=Whole-genome QTL analysis for MAGIC.;citation_pages=1753-1770;citation_volume=127;citation_year=2014;citation_pmid=24927820;citation_doi=10.1007/s00122-014-2337-4" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=C. E. Welsh;citation_author=D. R. Miller;citation_author=K. F. Manly;citation_author=J. Wang;citation_author=L. McMillan;citation_title=Status and access to the Collaborative Cross population.;citation_pages=706-712;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22847377;citation_doi=10.1007/s00335-012-9410-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Yalcin;citation_author=J. Flint;citation_author=R. Mott;citation_title=Using progenitor strain information to identify quantitative trait nucleotides in outbred mice.;citation_volume=171;citation_year=2005;citation_pmid=16085706;citation_doi=10.1534/genetics.104.028902" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. Zhang;citation_author=W. Wang;citation_author=W. Valdar;citation_title=Bayesian Modeling of Haplotype Effects in Multiparent Populations;citation_pages=139-156;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236455;citation_doi=10.1534/genetics.114.166249" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Zheng;citation_author=M. P. Boer;citation_author=F. A. van Eeuwijk;citation_title=Reconstruction of genome ancestry blocks in multiparental populations.;citation_volume=200;citation_year=2015;citation_pmid=26048018;citation_doi=10.1534/genetics.115.177873" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=J. Zheng;citation_author=Y. Li;citation_author=G. R. Abecasis;citation_author=P. Scheet;citation_title=A comparison of approaches to account for uncertainty in analysis of imputed genotypes.;citation_pages=102-110;citation_volume=35;citation_year=2011;citation_issue=2;citation_pmid=21254217;citation_doi=10.1002/gepi.20552" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Inbred Strain Variant Database (ISVdb): A Repository for Probabilistically Informed Sequence Differences Among the Collaborative Cross Strains and Their Founders" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="twitter:description" content="The Collaborative Cross (CC) is a panel of recently established multiparental recombinant inbred mouse strains. For the CC, as for any multiparental population (MPP), effective experimental design and analysis benefit from detailed knowledge of the genetic differences between strains. Such differences can be directly determined by sequencing, but until now whole-genome sequencing was not publicly available for individual CC strains. An alternative and complementary approach is to infer genetic differences by combining two pieces of information: probabilistic estimates of the CC haplotype mosaic from a custom genotyping array, and probabilistic variant calls from sequencing of the CC founders. The computation for this inference, especially when performed genome-wide, can be intricate and time-consuming, requiring the researcher to generate nontrivial and potentially error-prone scripts. To provide standardized, easy-to-access CC sequence information, we have developed the Inbred Strain Variant Database (ISVdb). The ISVdb provides, for all the exonic variants from the Sanger Institute mouse sequencing dataset, direct sequence information for CC founders and, critically, the imputed sequence information for CC strains. Notably, the ISVdb also: (1) provides predicted variant consequence metadata; (2) allows rapid simulation of F1 populations; and (3) preserves imputation uncertainty, which will allow imputed data to be refined in the future as additional sequencing and genotyping data are collected. The ISVdb information is housed in an SQL database and is easily accessible through a custom online interface (&lt;http://isvdb.unc.edu&gt;), reducing the analytic burden on any researcher using the CC." />
<meta name="og-title" property="og:title" content="Inbred Strain Variant Database (ISVdb): A Repository for Probabilistically Informed Sequence Differences Among the Collaborative Cross Strains and Their Founders" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/6/1623" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Collaborative Cross (CC) is a panel of recently established multiparental recombinant inbred mouse strains. For the CC, as for any multiparental population (MPP), effective experimental design and analysis benefit from detailed knowledge of the genetic differences between strains. Such differences can be directly determined by sequencing, but until now whole-genome sequencing was not publicly available for individual CC strains. An alternative and complementary approach is to infer genetic differences by combining two pieces of information: probabilistic estimates of the CC haplotype mosaic from a custom genotyping array, and probabilistic variant calls from sequencing of the CC founders. The computation for this inference, especially when performed genome-wide, can be intricate and time-consuming, requiring the researcher to generate nontrivial and potentially error-prone scripts. To provide standardized, easy-to-access CC sequence information, we have developed the Inbred Strain Variant Database (ISVdb). The ISVdb provides, for all the exonic variants from the Sanger Institute mouse sequencing dataset, direct sequence information for CC founders and, critically, the imputed sequence information for CC strains. Notably, the ISVdb also: (1) provides predicted variant consequence metadata; (2) allows rapid simulation of F1 populations; and (3) preserves imputation uncertainty, which will allow imputed data to be refined in the future as additional sequencing and genotyping data are collected. The ISVdb information is housed in an SQL database and is easily accessible through a custom online interface (&lt;http://isvdb.unc.edu&gt;), reducing the analytic burden on any researcher using the CC." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
    <title>Inbred Strain Variant Database (ISVdb): A Repository for Probabilistically Informed Sequence Differences Among the Collaborative Cross Strains and Their Founders | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ttWjJLYsnWx3_Zsr-6CxDvl8j2SnRC9xUn0pWgwLf5A","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484767","apath":"\/ggg\/7\/6\/1623.atom","pisa":"ggg;7\/6\/1623","ac":{"\/ggg\/7\/6\/1623.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/6\/1623.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1623"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1623"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/6\/1623":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+7tApyQpwbbRruXxd2uru3ohPH5XZXQQxOtlOU14kT2\\\\\\\/Y\\\\\\\/8qCNrm11OEi9QWBEPDHPacAfepioGndI2bq5gOlPLrht7QoyOudxnZREp0NinRDK5vipfJAURvKFeq+Ksf57gbx1tYMi\\\\\\\/lgQqfEV9+i7cyIcavKGImOedoxK2+lYe\\\\\\\/DfEyg5QYIUiJSG7bHF55nH6oJAAwBJ5RYeVbZ+9WXgLlvFRMpBId2i\\\\\\\/vgARe\\\\\\\/544guKGb1LN4rq++QUkeamf1t52aZNpT4d4S\\\\\\\/0SzbU6zJylmWi+G72PY\\\\\\\/mhMFQW50G4XCLZpxSQZ\\\\\\\/DBlyoOr6M8+e4RA5RqKL95GavPNTuMQmid9uJGYrFtR+Tesny4iHv3iSlr1xLDNXS6gWYLgC1DXdDyIzWTZqi4FfqnmRxKeM6pPtdYvIaUAACmt1i9teHe1Iurog6aDFdXlCdBl2yKt01pP5NTJFEAYJ6fZnmpy1h7NHEBSIupi4nEjwXb8ra8FwEvgKolvIdukIiXOcWWUUWlW3E3s2KfKDEw5HQkzGT7iMLi5095N4R7wu55d6vSkjhDQI5vlEx+zDklgjSMG+yjn1SX9k\\\\\\\/z\\\\\\\/4kiD0dreHZ9kj8y6kW+2YQMteQAOY33ihdJaAnojZE\\\\\\\/NzEJrlj5pQ\\\\\\\/nXaWArEklIwUz8SO6IjQlqFxwbMZRKUiMT5VsGSMMjILlypcII82nEk65\\\\\\\/Cs3J+1dMAG8sKhAbCXYozO6u+aTkPs468rJDbw\\\\\\\/NghpYWQZbVuYCO2WR91Fo1p34BzrgKKgcx2uC8TWOJqStAiDhxI7cX9JD+hNjxJpK6cy4qpmiKA6MIH70zpfAdljq9+tYw6aD8hwx3mZVh3o7TZHS9mIAJMUojBZKVGDbP1nxjsh+pI90cEt6PB1WXXSL28SwF+xpwfEh37qgLSreP864ZdA6HzZ\\\\\\\/dtWOXLUjFBvp5du9e46KeMzkONqTV8JtzkdYoN\\\\\\\/7j2gBM48IaAla46qRMGjNwcXwsSb6i92GP+kRnJ9JF4rNrJlaNdLeU\\\\\\\/j5Rfwkh229eZu34Sr6Kh7VRbQh\\\\\\\/dM4nIQOFchn1nvd4RhM9DiurnmCmJWIn\\\\\\\/iHG1WaYYMLt\\\\\\\/rJD67QR8xgpIbW6GbJb5mEWH3ySRoYW8Tk4UBSjtbvyWupvreOsduovugNWvUgIffrtsTHOqsF84dpGYa7f9atjDXAqicgambsz9DoI22BC85V22rlkPl9VTmdqDWC3JQiYd539QKHImjBcoBpoje1S\\\\\\\/YGB1QLl4bVEws9wYMgwoPlv94KLP4BSTORb5t\\\u0022,\\\u0022iv\\\u0022:\\\u00225wbUI16pm6hfWkzc\\\\\\\/7KXeA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c6367b1a237cb7ae9a94c2b13e03cd0a47f26996094c5554616a0453763bb401\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/gpmzJB27Knyo2SP9YDgVbtIMK9srfdTMdPQq0I84PcR6hCz+lyH4yEbbIPILliFgo55xSk6brv87tDlr2dobCfvjErkHTvIjsOKF8\\\\\\\/IDrS\\\\\\\/xv7mq3DWOnDDE6HFxxC2M8AJoI6LvjJfI29wzPGREB3iQtpenYeIF6cGANo1CEGAiRIA1budhpa6AHClpUOF+WvT9pFeHFUvpWGUp\\\\\\\/rp2LtRpJ5NWQvqPMDidFYHhZrXB1BfCaIpyagg1EqQMI\\\\\\\/nWeo+tU3WmaAvCIkTS5f5GHTugtZCY93hNa8CLjk\\\\\\\/E8L6e7tCn+FAii0ibgnOp3O1ptfd4Dhc2d7rZx6XsZPxEv6h+JcFpk9GcOhhdaokgFIoFxFxNLDOPshi+8txUByU1hZJewm0YRd+5Gn++eEgLTWLhQ5PmCKbapDDYoWDvpXqMAPcxzTxd8gWnDCFBPbzJdW0SOmBPGnpES8Qua828rBS+uBkW6QnMnQGn8rPjmCNqC\\\\\\\/YKDJZAHqPs404Xp1Yl8cXE+HzoLKcspkMpvD41x2nYAF01JDoFLtaSktRt4RaBAkAwrLdc3ig8KNDvTSjzG3ZFfxzfI0KMuHPpEvtWrhcKVlXYbvuivy+sX8T3Iu0LCdA\\\\\\\/rW42TfU5QRo3SPH5ykU4A0UlsOWtiAtS63sy4fkNHwLhoGZsjo\\\\\\\/af1Aim\\\\\\\/BRZQ41IZFcJM3rdhFSeJTEDQTKTBhQBpVrTG0qaBQU4p2UZDynlggU0RxuKsbC7h3dqKNf9QRau65HQy0eDukebQgdUXmeugmfzwLmyyFLsabyoyQK9lpFbkKTtc7S\\\\\\\/8EBkpKT63i9m4Y+GZcMO0UZuUtTb1tZ82L9H3xHPx9MDO0j86\\\\\\\/RwQWsQkmVmXDPmGBUYq9mJ+dtUHkP1GqBIPS0J+4vbUR+Zndxs7gvkbUgXyYRh\\\\\\\/TH4Iz4Uyb8TiPznFnZm8fo2gaStkfhH\\\\\\\/K7cu51CBnNPegFU9m3pQu\\\\\\\/aiWhpinQYeK5lILCPJOBSgtJtjBoIT2ZrIfWMsbb4O0FNnnLEKL7+lQK2IE9yjVkJ2slPK3TPy3p25rUJaOjnGGfyFiR7vPkaYjH7szKuhgafqS33awaXNQ0uy+KD8OVvD74QYIzuiME6EGPrJ0CKmoitamuTknBeOa+1hMh7yndLSgZZKR5p3pTE4ypoIGU+PsZgNSrDR3kE0ABauGeEixaxnXsTfjoTcjVq0h4Udhu5XVgDakxgwEX72N18WCG4g1pBarRkl5tsZwUZq5NUeOPaWYovlrdUMf\\\\\\\/fvt7sOOQT4zXRBk16RKgJr\\\\\\\/4rZNZ9ml0VpqiMLSLwkQh4+W86h6ksFRG3KNUbozUgGmhHWaoqlVv565MSsIDl6oB\\\\\\\/dkqDm9Ildm9NdzeofZUycvaKbtXnunlOaf\\\\\\\/iCXP3IM7XR6RtxM9Rn6DbbvsIRJS6U7NztLM\\\\\\\/QdIcd8NcqHQx+CE1pHjAWWxc0c83wUQUsueNS3wr6c\\\\\\\/LEQv0Y43GBSeXvHf2b3d0ayW8rD\\\\\\\/8hZNIHVr+itfLMFDI0\\\\\\\/AxTaAbINXWsgIF0WwJpJPQYofG6byeVC5OYVqU1RgkVrZYbNrXF0OkG3Plu4NzacdwOBwfSl6C8kes\\\\\\\/IXBo7ucnMtRHbA\\\\\\\/DvgPk7tqxhP+f7kjbCoxUNyH6Jd0rRnv\\\\\\\/5MojAz7hgxSsjsAr1SGhzyIIeH2mulcwwUKD5Zq6qiurAxJebrYv5Wp7xLUODbkwGNouwKzo+KVIzU8mhkIAakFv8R4XqWLbWBqy2x75+IeR25gJGanmZmZr+vxaHlr4F\\\\\\\/I6LGOupghVzAjVK2UKbIV+mKGQXsB8eEOBd4+QkxpAUvZyydNk5KUQAadbDT6v06tKXv+xUqzq4a8d7ehnOxYR7bUpE6m\\\\\\\/Xk7HB9Z6IyTBSxUa7rnQqrRvRkJ0MfPTpnXzCB1D7Amlj1n2VAJxpuSG72u2wU7m2\\\\\\\/nq4UU9mU4WGgo7SjlXLhtEqu6FkjZmoyJ2ypYdnjJvyAtyXuI1+uZEU9KJok7ZNvyHRfGaL4\\\\\\\/AJNJ7hITrrdQoXI+HUQccDOorr0XGuXaeAdM6+AspeEcKUFLP33eEoWVEfnyD0YuD0kjMqX9s04ZDI\\\\\\\/+SLbdMQt\\\\\\\/5dwCjlsBbJA6cKTqCIoeq+5nJySkuN4D6rHoqdqq0W1Hv4\\\\\\\/ORb7KZCPowVqbSpOxafFzaFcMSWGUTUlf6G3ClMIla6uS0oAxIrGYBIXHEuTIWG0+WkCv1EFcVA\\\\\\\/QFEgc4tDWPm57BWXv+sR\\\\\\\/VqDi31NolGpqzXy\\\\\\\/9TSerru\\\\\\\/3wFYYI+eLLg3M88LkK4RCQSe5dWNvbGipv\\\\\\\/QuqjX\\\\\\\/\\\\\\\/R3\\\\\\\/BfR\\\\\\\/vPm7ykbRABNN+7c1giNRiSanSfTZUJracJttqVyCm5TdLi+HbP1kgzF2v\\\\\\\/\\\\\\\/C6U+cE47xbsyJ8frxzMVgVHth9uxRsc7jY2xvyFi24zG5Fzslw3lXdb6Z8ye3+zaX1OXI7V0PEx3iVhNWSkAj87YnWRF1mb6wamjIpjudPNHzXqB1Hi3w19mEPJAGvy9NOQtgREEjyhtIuzPuWoQvtXlviZHc8ZkgaT1Xo33ywhtZsLpwKb0FQWDxR91G2wmUc3tmawzORa9z8xZDLXsz9mkiKiMlaJsCAUV+lQuh9C5sIU66zjwpTu4ufyYJWSkV+KfmYSegpQQjpky\\\\\\\/uB5F4uhupckyH4g7D+USIRySsK8V3kYaugXOJFZxxa+7HhVi3ETF\\\\\\\/sDQtMRq6JFWiVRkfNk39RTLKCREHGbacmO4LCHfOUAWpqUx6Va6Eecquzulec8+ZzwKUjb+vYTgN\\\\\\\/DQ3kUu9dbGTYkU4Hpc2xYVYZKb2tLbxAqHhDcGWJHk5E9z\\\\\\\/OE1o\\\\\\\/J0FqdTPIgqYbET+\\\\\\\/ejGn+GWgrM\\\\\\\/9C16cFoP4dbmGRm0jVFDsLmezzbjqfpfra\\\\\\\/m7asGsR3ZrfDXFK+w7++YlRQ3olBZmFDXf\\\\\\\/EOcXbruYA2V4K3wSI6cjfmkKzUa0Ta9HigUiwZ\\\\\\\/Sud429+MxEH1peX1NPM83SPQFghePULlq1rSLAXWvucCGJ6zeh\\\\\\\/XBjxVDCdNh6JUXBxy7IWB7M\\\\\\\/1oYoPenTH\\\\\\\/IM1p0g4bR4+tfaNZGTb159JtjvLmsLeL9VJnoxqQGwcwaY3UzdSPC+bKyIaaZwdiRXsr6FePaI0N\\\\\\\/5tlVm\\\\\\\/CuJi36Zi2HisQrBpJc5J1Hz59ARouxxiYjvpdFKDObFPJb2UzHk1sF1LvLMEHpvAdZC4jk1RDsWgDtLcQYSMZ49vI8ob7iBc+axeRzzBpaqeILwFfpynRoOMCQlasn9yrajp9J\\\\\\\/SE+KWReQjTAFh8GdA5Q4Fx7cy+Ls7WuL4SaMSlm2UAbvGLfdaqFj60f94PEpFY8woamTLMoqPvZTp0VTUxao1Y0aiw73NvHWzqCpEHsHbMKXRI\\\\\\\/NpIeMKrLalSj9c+LO+XrcnWHOKh5+tAPjvK636p18bSXnpzIOVLG9PMoUTEsGvGoTEJuQDMLPLgwjkFuhjtedUKRbmrZONIBMLTlPbz457Vsu5cxLnS9tJWrZQIsuiUogax36Aobg\\\\\\\/Ivlk9vEkm+viUzYOdu7qU4PM0Juo8pJU\\\\\\\/JsOdbQvh5xsVdEeMFH4tiTgYPJFdG5Z\\\\\\\/pVB3v\\\\\\\/AwU6TcRQDiuwFF2+XBJBR353ahVvzdsRPLt5s1xh571MY+Vd8D1+DuE77VPk1GoJg5WiSQJqxZXAGFsf8uMcRRgt0qN7sUg8EV18crol6XIAPS+IPNlJdzhSRW8hESh5MFCmUZO67YBEM4\\\\\\\/\\\\\\\/fXxv1wyeQ35jw7iIzxEp1cmy6KTISjL4ZHbULRbSVUIFV3tAtxVdIO6oAMkpZ5USSiDG1935PIxdO\\\\\\\/l0RdsEwqP7SftLk2mB7sLYQ97ikne2+nPFb+pD3krJL3RzaiAOKbYPWmOY7rmU6jFCjDZcdnsEvJL5NDERddgGaA\\\\\\\/8W92Rf+QYJ5xdQGQLDjNrBbXAJjY23pDIqdnOYwS1nN7Kq8yISqU8UgTDM+qbmvAC8pVrtwIm0E6xb3CySaQgJgtBrpi6ro8aLrWTusQ\\\\\\\/k7kPbKdmn51T0+XMozAQk9VaLlzXMAqVndcMcsN5k\\\\\\\/\\\\\\\/3HgWgRL1dJXH1uOlsYlOtdflr0dfd6vFfoEC5cjrA6hHTNPvkiY1lTmcSn8TzW8ZN1OFQZAgWGewf4riwijXUNGRlwqLTFuaI70oCK\\\\\\\/WdHaF1PNVRi\\\\\\\/pmaXrenWiF5o4+XPQB+72SWzXutGBL4bmHz+dOzrY1g4ExQbVVzZFgRy\\\\\\\/qqTbN24IYIyXrx8ibbheDB\\\\\\\/GL4+dCvEH2asrPhNyhdJqvz6fZ++6MRT8PDAn72mvXapwkFD4rhGnmlGNfMANc4amOdAfUq2kxsqAJNfptI4H5SKo6EgLk\\\\\\\/y3DDCueZoc7ckVbGm9p+nXr3msjN67qZNUaqQPsF19yqKnCLMQ63EsFAZES5o33Dy6wA38zzx4LRycL9plitWmwdC\\\\\\\/+OXCdiPyFJTIwsdOGqRuymYHqE1OzdwurMcOmuLGCtrbwnQ7lNAIoQEKi0UTmoV5V\\\\\\\/ohNhJGa9fSaipSY3lS7zcUm\\\\\\\/a++dy3gSY9Rpy0vIuFSQVywtJTSpifZcc5r4OFdcPQhKZYLnzgfX9EjOIO+tZKxHVdDeXbMXVEzAFQKb+C\\\\\\\/gR9AoJ2pu0aoiFGHJPc0ERM91GZWygaHRjvdlXEh60tdB5e3JWw8Mmigh9l+kgkoW7dx3hQlzBP77yC6pw6ADfOBYe0kciBRlAQEu\\\\\\\/605xFbWL7TXuI39gWXVSyFn6hQmw\\\\\\\/MoXFRkUa8M1A6Jv1mytS4VgSm6v2GdunXTYcCdib4Yk6+ZV9MACPwINPkMQwxeak+yAzZJxczv7D6PhshCmaoq39Vcc6auj9oIHwW+z7Ql1TQJXvV9airgtsdFhKZ9y4BDP034fybYSzcUQNTk0\\\\\\\/8ZDjMmc1TB8rodeJxyiiq082hEbf1tzOL29L443KK2lN4hzNOKBO08LHKQZzIvH6bywFtYrT4PPwicwnVa0lRw3uvZNWuKaYuxppKkU5PWtaNMriUvcdOjH9bdq3FDU2gRdKbXZmjG+3uU0OLs9PULYKdx81HkX4hCRh9K4fpJYPS4quWnE2ibxUKCk21Am4QxX\\\\\\\/xSsl3ImtPdci4kO2gYAnQUPirxLzUXb+9HODkm7LrugJ\\\\\\\/OePpqD4cO2dQfSLC8azJuDfLth2UOyvMUTndjt5AEjfJYD4D5aVlPvPjijZct7OcrX4v\\\\\\\/zdVH\\\\\\\/yGFGNmCvdCgDga7BoNT593faRtWVA9uxy6pbY62vSk3S+5oX8uz0yUQ7qemmS3gARCeNJgBTyEK+TFKDjA+KrvzkNFWypwJ1T6B8bmop76R6nnAbW2uA2a9xjd9PlLe5b5bxOhtpT9+OAJTKL24QCqdeD4cSHqNTRU1fIdjjfZd\\\\\\\/Coy\\\\\\\/2cxgCAT9y6LLSynZ6\\\\\\\/vqgiPoeO1+fpiJ\\\\\\\/wiAfeG+tE95uyAzyAIxRNuGJNR1pjvCSsTbfkYPQc1c8Yzu7urMZxNxtJItPY3geAmWA\\\\\\\/V\\\\\\\/HxRnjpKm5ybVJwQV6ZAYJHIefUwauipafsgvQLuGnwsXIx6R63cMUi1sDKqUk8CLP4Ko0K4myeAeVL7IvxupzKncWy\\\\\\\/0SSZKKErE4QEXtzenWNXgTfLjG3pTiTZ89YutOsS4uSeNzxMp911T1ujvb8PEAKmx+lUYmNBmymhA\\\\\\\/ez75+XnaJOePnqZQDUSsle60+PTD5RJ1P6VcyIbJrp8BP55SgFFVtyKXX5F7Z1uId1XNrn3GDkYXMTilTTjUccYu181glgAg17NaH+H\\\\\\\/LVHCA\\\\\\\/vZiy63X6o6M+X65\\\\\\\/nHzQQEY\\\\\\\/HqZO\\\\\\\/j9xR4R9BzrWR2rcFNUFZjBXr3gYiroU\\\\\\\/tBqYzTi4CFDpYo324FJQkmXMOQa5uyjhkboqT\\\\\\\/6CXAWUzyBgxxXQsBS0tULEJE+MWTyfDOwH8Vok07Zc3cfPuebsjiNPPzW4Uj3JHJjMSsSN+OJsEn4wYUcDDGkN6JKmytMbrrLIPI7CCsGMVfWci1mZ1+UzRXkGp3MnO5HVsqLjAiZSrNelO4ER55f2GiLs6s4AJ6OatshLj1\\\\\\\/hV\\\\\\\/Rdx+dJas2Yx2TskJ8cUGOlqdekObVgDBb3+\\\\\\\/ewElNDR+Kc3VRzM6sTg7yTQcRMa7kRlnhLeyItEYBG+r4OG\\\\\\\/SqYYRnd5r4i5bEtb4Zz\\\\\\\/6uXorLmgmgvn5ct6cS59aovM3ye3RsimOjvIlfL6Vv+cMUgSOdEcSyONpcHAIXw6MkVXXPmoaefVrlPuggdK4gfKdsAt3FzfG1p9Wwx0u2YS+ef0piD9Uw5KEwVHaA9FABTxaM6JCJovp\\\\\\\/6j7kK7z6l96snauBCfK\\\\\\\/QNsy56iGrnpAp2TftkIJ0Kf11Ey0rzSuID2k0q5iy79oftxadA5ItIKVurr20CIgnz6blQnsHumvsQtPJptER8gQ8JM14RIgZy1dfKuA5ALPtiaiGin2klSXtXVcfm3kTLBd71vt00FQXEu8Kqx8lzjWA53NBZWvEvGQHlSNB8e4xX6ewIeJFegA\\\\\\\/0M6O2WnHeQLBSAe0xqQr1Xl\\\\\\\/NBctMRq3PUu21Q0uXXBo2M+RJYjgCOqgWwQzUmbOuRNRrOCDpECTfx5lJR+F3r8ur1O5Fch\\\\\\\/32wBY0HnHWmJXU1svCh\\\\\\\/7jAtBTYJXObXhwLgAxFfyPexPgwsHgEZhzw3XmGSNce8surFUEXtGZkXthaZLOL+cVZJMHs3Vs5Oo\\\\\\\/MVr7qqiGFeo4nkacarXzu\\\\\\\/92iqwsFrkVaVQBFZJzXcDNeRAL+zIiD7eqeXcBNMNVjd8wxALOX6hrB2bw\\\\\\\/s8hrVpAp+GxRkRadbaFIAkBm2w7f4lBRzcmaCfa6Aqo42d2jkweExcmgoI+i92HiwOPC8cLBUun3iEUYubX3NWw+sRoLsnneU2GAJjVcVGAsyMAql3Hks6X62SW2Ztud78l5iQTfY42fgrAk5UnNbWKWK9KPIWgaO+0gxbZMqwrS0EDQYjKQoGZIY0WfFpIQMEp5E8VeFs02v9woa5k7TFe\\\\\\\/hrLeh3tflRlm1pq\\\\\\\/BPpXXh083hsD+Rcr9\\\\\\\/H9HScbd0Py1Y2g6E7q3\\\\\\\/u6FGXJQP9UQb5ddLEgH69yzXKxQH\\\\\\\/oFkBlo30pN8vJzD\\\\\\\/zrt5QirNeWGr2Ek1Sf472m0HviSD2RKH21MMeeOI+0wcdx9z59TZPNOpLX+CvbLe66W1RBoHKf8Do5w38r9B5\\\\\\\/VRsxDenMhYi6tSJxRLJlCi3Wz9Ip5OWlRKtpqNni8Zd+TyG66S2e2waJI\\\\\\\/E+gIQNpSYvmgXe8J9OyXhYtdyT9bXWIRj9UgMzRbu1aAC1tL0L4xv9B\\\\\\\/YMmCAgbp3xFotkYiONzRX7u5g5bQPOUQc0ljuHwe7Gc3ufJrn9rHFv0EdKzeddymAM1RS1+9wDk89A0Nq\\\\\\\/XOc9Z02YwQGPjI\\\\\\\/nz+EMX2UXqb13pE9bvvhpGLLdyGNrfMZRQvMAkMvwLt+xlYggdM81ERt0A1+X9ubQ7ygptkvs\\\\\\\/W5w+FEbTxiZvyNXb0moLAQGYwUFG+RHm9PhXD96TAmI1NQhW+T3jNeuSEBnc7T+wdqSmAjqmR7zAZoTm9PYuEIRbyif9r7pnBJHSTDpdscfH5obPe49habqf5awviCnfKY6x\\\\\\\/PsEmbrRD+8gzSjfbdAOMoa64pg2qAGjgboC\\\\\\\/acCpf1tGHZMv0rK5uMLvuyLN0bSrdUzLHnkSiFfHjV2zZHZIqT1zVUliidPDsbEBUhOEThAiTazrDDze\\\\\\\/xhTUC9DKbqEsQhJNgwuClBDktgCo6Q\\\\\\\/RJlqkPOCfhv0bJAx1tzU8XhMoGpDV3IMrhLqQ+BIt5NFhMwvIbXzWfhLcSz22vmhUk7kY70EcKPGkKz+ZfAsrDNVeCnN93KPI8QUNOTE\\\\\\\/fHNtonJbptg4CGMKCyO4xPlv3M42u59q1Vpgqr\\\\\\\/nRrwA9kopfT9jPeiPVtMVIwrlYc7rsx3WaE\\\\\\\/b6cVGeerGJ2KmjT5V3wM6z2L+a+g83SpwMKhZp4zp2HIwyk5Oq\\\\\\\/qFwQcIP1fA3At1Ul\\\\\\\/1ADK9cprczp+7ekRTvDub90hLwjOTjnWHC4hqm5amP6gpkvy3lIXcRYKyVpVr0isuDSjQdVGbW+3VCJ5TB1Jy7eBq4CJFwk5nsvHFJ5OeWLpXkrRxlAWiG4u9NNmWh6mXAwPr1F0Z4jxl0EDjppZvUYcpRP3DjEh3gOtHDmgaxHQ6yieCFalewf5da5J7W55Q8OX4f47qEfaewI+2qSBLm35gYtoYEDMw7o1enBs+wWx5PKsQKMfCWr9rgsVnt54nasY3DWSAsTTQZ\\\\\\\/1JihFBrlLMlmVUkq+bvq\\\\\\\/vDW0hMvVLKryVEp9wRj14BzjDWzwWBDMtowEiQtztSPouiSdyrcVJ3Y29tO4WRVmdK2qMhNo\\\\\\\/tT5BWEeroDzZWFEUxNom2ctY3JUh8zYdBZ+12H041BEXHgJqziaiUc+\\\\\\\/VphNVSeyRNW6WftbvlpNhc5L35EI9dzWwgFSO7WOBzuc8p3cXsALPb2eVzxIRCtEHCp29apGxa50YF1pCT9\\\\\\\/Qr8HMW3k\\\\\\\/oQDMuNc2PoM91iT4Cucxec9pZJcPnaiWByISIfA4Oj9IJzy+djfJeCE9vUvWhn4pEo0ozK7gMjY3NmESiEjgERi56c3TouEItQKd1TY1dyWgOfdnyBtuGT4gP4c1Un8Au8xJjtsVGUeXVuxOuIvTHDUAwFxZ5SlUkoIjbv6nwMfueBwn\\\\\\\/k5EQmXTliCzww+HAzz+lnyeM8HrEg54tcLr\\\\\\\/0dSvKeVtaFsXUe2N5chEawEeDNRSSpOCkHDIBsfTjPWv31+LCWnsaurDXak9aEoEuc1M75LOMfusEjsVrUjgAWnSJ4qk6HsqCfK3g0ZiydAIdE\\\\\\\/V1SXSgZdfF5k8RPC1CE8SaDqRMlQl++xDkjCKWWRGHhKmML8dljutVqFJORSTLD05QNAL0RN+f\\\\\\\/O1Dema6Up1dlnFTTbdcy0AffKNKUkcnfj1KVcupVYeoiNY0JHxbv9XLymByMjLuZzmhCoa8MJwKihM1RscrxD6sYtoUPppi6OQ9+AjTP0riqjfhiLsN4ANygOdyllhXT2kiIz9fDkxd0w1Yaemm9rHhyxuH7zhRaGW7AZScDugG3cQajhuKU0TH6nrRuw8TP0o9yDO0WK0UeTeRrMepXRye7w4nyJF+5chJdkRMNom0Uk6PWizEY0SVE4ZLVXG\\\\\\\/9l\\\\\\\/C2tg\\\\\\\/zUb7m91LVyahINoquC86kZ0Gk7kZvRggob5A3tPszUQ2Z9RH8iNLZ+BXh4fXJAgJzitABigyJAuZBBdNjlX0EcPWGfPV4tT9iinDFM3uEZIDz8XQELi24BywvbWjLut1siZC4yMoXsrRAOngxXvNJsF9vIzSQqdB231maYLAk8IgkkKbD9+KW4aC\\\\\\\/CVqenkZnW51WgrD+Q+lz1zZxHP+BOmGzL\\\\\\\/mCE9HYzsr0jNpav0y6UaPweSUJEXs60RJZF\\\\\\\/ENDmzoPLn\\\\\\\/VN0poWsju\\\\\\\/0hwo77\\\\\\\/x3JltdqKtkwyV5yY8mIPx4GrUbxgoMCgbppIxUUr1SEY5KYGSAmSl9QyJv\\\\\\\/LSdv4wTe9PoCCtcvk6vepl29jUElJadAjJKSlI+boVv5SpcqrwrWTbdp0xxa11KaWc1wJk9U22XvLUVdGKkMhlUlmE2eMMnfyO67jB3j7gobrH3uLvlDLmOb8w59bSez4464nPuyNgKH4c4vuOkrM1R0qAs+DmNrmG\\\\\\\/DBPXm0tb9v7x1Pu\\\\\\\/EX8i5OtPkskId9M1fIhdOU\\\\\\\/fUI5Y691evmmWXZhTQQCmPX3nB+FkcxzziSiELMKe1GIP2fOBzL\\\\\\\/st4PJqR0ZOM6Hnd2r4\\\\\\\/vxfjVd6T02eG45JoZEcGgmyTf2UsVmhXQxTiBm4PHGcrYAORQLu5FobBNTr4v+kc5kc1dyXczyiz8CLJRkvNZfhUHBjJEJ6akUmNfiO5akWqVVT71jltygt9u\\\\\\\/IePlOJd2EGfT8bZLB6I40XYslS0pvZPgjWHQMUi9zTCY0QzwGwDXttx3A+Mn4ytoQX1qDPHqh0mOBZc1S3s5oPq6jEfvoPvNktfFkTwKNJbw9ZKVYBxZAqok3MN\\\\\\\/YeO6HnRce0Mkk48gpx2zjzN\\\\\\\/elshoJfztU3PbSDtbGc3d9e1341uX\\\\\\\/edeRfvZamMRAZxqF1UL2V9iY8\\\\\\\/IawMzTgY+FIoYCSaEi+y64XsOgwi04BIJSyFqQKY++Iq9bnJPJvhZxc4LW3K3ahErByN+hDFszm4n9H6ff6Gfa5HWSifmtlnx4jfXBUlemWHQlln35WG38ZTviiDmt2No0U0PAWP8TTOV5qs\\\\\\\/GYwLhkqQEw1G+cMsATeSI+6I4r6I4CwumDzCXbhsuDfjvcFwPTjJjaR1JpC5FCmq0YKdhwxLQ\\\\\\\/RS18aUr97NCDKx+CSrTqG62YdzMg7ki1AaHIRuHg\\\\\\\/dFZDJap\\\\\\\/UiCIRdsYRl4uaW9vRdPTSonDcdV6X0anmOAMeMpbHuQ8no0bSeYNTDbuRbVNc2fJVivCe24IB8zcp+MCBSIw9079rYT0rwiYxNjNawd9+nVhBsqVTdAbQpEgX30UuJNAh5jxYJgyPwSDH0S7tpYEKaY1IXlpjYGB9Q0BjcJ\\\\\\\/LWfH2nqp\\\\\\\/E1z7Bz6daLWZkl5jNcFhL28a1P8CIWhgF5A9EF3e9ELXlQ31z\\\\\\\/5BOSwo4FnlA910gu7KQfqgR2+3oVD9BK3b5W50tUXCZc83cc6CjC0HCQGKDQro\\\\\\\/yBKzXJ7kdnO1IDFQIKSLMxB73n6jq0VUCExiS1pDt9ifo3KgAzLYcJ0HtYGCqyeVcgfAsb7tTkd\\\\\\\/Ismdqeaf7AZkZ1AVmy4vFknxvj+1S8QyRsRY9hTVbApuV8550GAzOqRGaEgqLNj0vVGGBZxmhU59xZgJz\\\\\\\/fJQsIqI3xWIOTMPurO78+hID\\\\\\\/obchPYJq6pRwdj3ZgzKsWz9OoA2cQpQ5txKHElSxz0LwNh+gmWlfzeJnzeZnUO54UGdbxvvvQP6BHadokqEs\\\\\\\/ovUmlEeXfSHOht3qrFzqfvJ+eJALvBORoRGQdodPr\\\\\\\/GTXSWM2VTStUI7dLQ2ll50LE3U9E+4wL5ZvgLuFui4k+\\\\\\\/4vuB+jQ3GqbTFe0TYwF2hWvWVolkeN1PvLRjgPQJ+yFyTVoA+erqNSZy5W7ZAt\\\\\\\/Q\\\\\\\/oUjJMhm4ft+xL7jLLGhStxswxVYMKY+5vuaiuvhWwRHXB2Y3LzXO\\\\\\\/qtY1o8zXXeOARvV\\\\\\\/No\\\\\\\/r6B2m9NFnspj8qzAZaFq\\\\\\\/DCRZkXsgCIgKc8vwG4i+LoxeLi0A14qfsDd\\\\\\\/6baUr1LzCp6x\\\\\\\/vTYM6FQM8EaZCZ9AkI4aUXR5z5Mh+IgFyVXdU+iFYs0gRMB5kn+0DvD+80NaAymWcvQXPRHoq0O0YWmLp6xNeVWtlQuVYW3EhhAVpEJjl8k7WotdiVmNUrwXSJF+71+W7\\\\\\\/SimJ4TpvdfYMmjorQCxYzgfFEaFC+SQ\\\\\\\/t4yE2a\\\\\\\/uwqlwjRhEQFfrRmXHUerv2xh5DYVvG1kwvoCT6vKgPbbWZBQdjC7yVdJgTxtM2p0uznKPygjke4Zj7zejwk9pGsJ0Tbwestf7YPti+pOEa77BL9NyOeojmiAdYl56efavAUYmBA1i9dpfM0qhwtq2K1DP6VVd3RXru4e\\\\\\\/CNcBThHtoUapTtRuojiLRSkzVwO5QRIryRN+X3iG13spb\\\\\\\/RbnRwoSDKsrmA5VBcPgXm0LAUUyP4Gsule+o83tN65Fmker8uyqQrMkALdDYNh\\\\\\\/Lc6bRCroZp01E3ahcTj1AC+j9TgZ8ym5qRVRqLb+ECTdyERKRAN5EzdVZtZboQXDqDHmCQ6mWT\\\\\\\/IdyHR35JH1RJxd53YMihdOuqnYNBjB5hrRJHpMFxLMwqxZqT4f1Kl2f2E+XA0tcP51ExxY\\\\\\\/BLzvsmpMqsl\\\\\\\/ARYFmOqwl6Y\\\\\\\/VdLBay49W5XCoIOJU5KlFQlSOpKxPgYNx3XNkBUHy9hc0WGje7q4+WTr\\\\\\\/lcU1sXZtOab26KT\\\\\\\/72BwWngC4h7O9W30S5nAgLRnbTaCr7M4CI9i1E2HXUN3mk2d4Z7y100RhKg6Pa\\\\\\\/tRqxhhqqBwIwgs9LyAB9oOxIq9eBoVIWy2IWqzCAA\\\\\\\/zmAK018iCVmoitICYv4CdDIkUXWn3h6rR7yjUUISm+nN+O4g9ei70mvVfjJ8H4vvyiDs2w9J41PMjRpPD4o7l0tuq6PksLQ6TTfdu4\\\\\\\/DTFWLLrmn28jr\\\\\\\/PSwpvLuPyVk20K51SGdZ+GiS9tFqv9wwL\\\\\\\/oVg7uCjk+zKKBJtCnOu1Z8D1tdsD05eJus\\\\\\\/m2BJAVv2Snvb2rIh\\\\\\\/G2P3pWJba5+HTpbkfKf9GHtQh2jE3D+vw5SBZF8gAU2i4\\\\\\\/y9iOsziTVLF8aJK6kemcVqYyyY91CE1IaaOaf\\\\\\\/VPP2kc+aTAg0P\\\\\\\/fK63zXtir\\\\\\\/jsZsXZJeNmNM2zJRFZJ9GscuFtWzSHrugKvhPdPl8t7zknO+jdzP+mQy6hQ5qgBJ64oM\\\\\\\/L4gSrVjOJVB0oByW8jhTTmcX2ogMT06sDnuvkniiLNiJrnbMsRyH5kQvXd\\\\\\\/pyvbEXcDt9wx2Sm7P2cjlGFYazGHGbv26cXQBa79+o3y1XtCNP4XzWHV\\\\\\\/wmEskgym6XB4AkPjieyvwo8\\\\\\\/YFQirZB77UXeQtUx4KDqMvsOELyDTeQpIpPjmKSzFxf4eNjXtHCYh4fraw\\\\\\\/IfcTt7WG1Oxx\\\\\\\/nyskMF3lJ7UZCpbJI4JoBmQYtvx18f1cccxVhulMjwDS0XwHBMPX67N+YWol\\\\\\\/HR1sC535dCjGjdKUE5o\\\\\\\/ha2x6pQJvB+1D0U\\\\\\\/lfXfUhCrvf9U8V19dN4+wJko\\\\\\\/N7E8LnljGe7Hou1pdlhJGYKnx9RM2NQB+6a+LjDvfSS0Y0pz3N3YYw6Kjr2Wb4BWsRb9LHnnyZg4OEkpFaAecI3YhLEBb7lWebf9Rc+TAcs9U0kh9hIpXRrexs3NzUPmOyq3sFwroLb4WUHjOSOEMEM5vA+8VUHaTVf\\\\\\\/GcLqCJD4edYl1faOhttEUiYQ0Y66bH9EZjsoblWnlQefMR2vFDmKXEp0ECWR4fn0\\\\\\\/olgFtSVNGt4Xk4HP2iSwygN0V8gnMrKZYxPzqxQ8aAg0VMZyGndL0xqgtCOCjlDUjHS+ecmU+75jVvfvmNOeQd\\\\\\\/wLpFBxKt++7jgSL8ZR0fhFpQsM+kE8BZO9a4WNy6mzgM7xIVoeZJYq4J45bfmNdcPCj9xP\\\\\\\/I7p3HWhT3bK3cnNxbeaJUEu0WrTYulLCvhDy9fDXX8eiM\\\\\\\/MVxNN3A\\\\\\\/MEInxCWdAF+KA7TQchInwAYF05wFA77A04erqIvUGxln\\\\\\\/ucJ719MXxVFgOiMIe4\\\\\\\/\\\\\\\/NF3sUmH67m6TDOVfFipfDoNBjeUtKrZ8fJ4UrpYG1e9LakKzxiWNUf+ox0a1zQlldNmU4dRKbVfIXdQ4vhwpJvDO\\\\\\\/LB95ncqIjh1hP2I2hJ3yCY2k5acnQI1Nyq0w2hsep15KaErerH0PhJEpO0wUI6TI\\\\\\\/W5FrpH2F2GQwq15huxA8QGkovf\\\\\\\/WJX54fHh\\\\\\\/iOw\\\\\\\/R0u6FXUhwNfUee8NahWfj\\\\\\\/0\\\\\\\/uti3YQ+60\\\\\\\/CtNxyNEeQw361ADpasljzGiYIKT8q3rSC\\\\\\\/yhkg4YwYdq8Tvn\\\\\\\/lj9\\\\\\\/O\\\\\\\/kEYsVt78ljQPJH6nQCzU2Jg3mDuDbfWiNTm+EERV8UN4Yn3xcMmY3WTUNlx9OKuzjT6akLTLGyr8hWqkVQjQLOgAe91z7uhneDo9Rb\\\\\\\/zE\\\\\\\/Ph9p0H8FhiGEUYlGiFdDwom8pU2gdEawRWImF4JtzVffXqyjL05C3vW4VtgvZQAjsLnrpDnXR6NGICYZJAPAySgxp2RckJ6tYPvFhZSyl9Y7DDkLs6FxXsagf9XphJ0kaNDnWdwAbj59GAKjjgeZ6uNjL7D6RRbmzm\\\\\\\/5WfO2zEf+VCrQYuXPbhO9+A7xo3u5mfFbpKwS9Yjsjc4k2M29nsGRDqfEdO7\\\\\\\/hVNCMNtjd7SQklB\\\\\\\/eFw1MeGDqi8TFGiwEIuIQOPF7RtM3xrS79+OAbYwyH6bjpwLAV5P8ALoSxtoJWaTahukTkCXcxXIPMJsYD8aoGfs\\\\\\\/BeuPLkLhymkY0YsltV1A08DprFN7uWeKNSXDX3lrsgcexKD57hYfFAz7yI5x1qZyAAApyuSnvpET4GEFHUobxIQE4IaxnRSPZcxLQOZSSBq+NexJwt7bdmWFjrxiPTQ\\\\\\\/8f0GbL5\\\\\\\/b5hfS5+dycFVrOd7MwLy0HpDKJ2iglD7PuFuWOP6l9sbT43IfiIeX5bZztxroCh7rQYUU53xPrZevjisk13YgFXrURpLb0TL4fgFh8EdMaRMRju0RzbFbCBaMlxxZhCaowrMpvTLmRvI9KVgXQZ7yF+m9IwWpn2SkUEWqEj8sQiS\\\\\\\/KqPtQUnZoA4djIoB8DE2aY1kon4otKL5R3wpj1RB6YAhof\\\\\\\/rWKDSYObTNzEhG54DTcEbMsvUgtGwt15jRgknWrDRz1MeSkBwZ8F+X+lurcvyG0jtLHxcOWK3E6kDNUsOQ3P+O1lqzZO0ur3mLXhtC+ZYx8JgXVfNwuTyzP1EjhMQHECj0EodRdgy9PKoGw0AfJZNIYgoiLS1G69XoFiNEXHFscacPY5CqymapFzesFvuncAVK1MkkTHIxAq2xoDSz8vmYnZXtpvlJY\\\\\\\/rr858A5kytLr\\\\\\\/zc2wsZj9Xf84B\\\\\\\/HMfIk6G+xp0exQ9GI01KQ\\\\\\\/pvL+u3lUts1yCSsGv6cRlhAKyIB7NDb+htCdTuVtveobR8rUlsB7mRhFcVF\\\\\\\/ZScBvYcbgrJFpt9UAzioEWNwhpub7t3Rxig91rphE79xhY6w8z0aR8l75095Hj+UWqj2UF6iobXGA89mXQ5Os+1urYACKYZxZmuyC41k48DMdw2lMYZDDvogcHjI06YXbaVhXV\\\\\\\/4kf4nKRLIUQOn4khEY1cLbyZf3RQP64CyK0x9bhNsDdB6FoonPybsACrEs4y798jg6EsllGhbbC08Te0EyhWium0725V6g0AOKDxZWvCTAbUmEuAUYwwUN8KdiAS9ivVjqAZTbUGedWATwLlyF5fgvsipjVGb2Xw3CIbDneVKnB9VnlaH1KxivhAzy7dnuvMOLTzFq6eumlk7SsHemmWqWpd7PJzIahxtNKIZGt9ND0fmdtyKH\\\\\\\/PJEJAZebXTcq5S3A3nAVbGybNm+GHq\\\\\\\/OS+xYAb5lF0SJ4EDfmys+9aClislxWfHUzbFWqZDyposLD\\\\\\\/b22RVgqPBu55DkEuWyFjaVtgiXmCAJTEe\\\\\\\/RUoMgmuRC+2jgukcdMtuMFN9VdM1pzbP0vKSPqGMtKRvtsfmhlXAjLsdSd8YhoTOge\\\\\\\/FNrLDylSf5fZGMZYCQ73DH19kBJxUHADQIhwyiZgVBit+Rv1zGrVgQ9EZF8s\\\\\\\/Vqte8Yp5PgpH6odWXLImqUwXC7XRxppPyyCek5tWWm2bX4ZGdpEV1ZdGE8OnrLbAOsZdEG\\\\\\\/zScc+AOzgKuQxu4yq2Cewk3UWyoyBTkLq5+saJ6NhwZ50WmIBuhl9qoEy\\\\\\\/AajXrdCkaTXhCT\\\\\\\/AE7\\\\\\\/i9fea6+E2EYkyMzojo89YAckkQMPsUQyD1finavrtaNJG+QsnYyuwn+J1RHIVK3OCiA++oy32ajZbPHQvsxFGmLDlBFzwJ56dpfR\\\\\\\/9RTUirpd94E\\\\\\\/GlKIhLlRfWLjPUDje5aUZHzzHB1yyvWJhIBtXhwLFxsICD8CFdELOAgPWVT5ijSvVe0cYElwjCZtGfqrGrOcyNR76gbW+5rtHf8ueqWRjF9YSGYDYQwQzJoyLP\\\\\\\/PjfpkM2v4chPoxy+y0jEYW97YD3mX6grHmnUnrl6K\\\\\\\/Tr9+r4rPomkq\\\\\\\/SEjDl5Q73Yovi8mLhViXqGXB9XxsOc29o1UMYyL1dr5BFHDukHDvHRoJGywI4lEVLMFSSQPYI\\\\\\\/JpQKWSqOstR2dBRIHdU57ijSXdugSLOaxr2AcuEk2MtIScizQN8GQu+XZ7UUNo4RYpdjkBv+z481V8U8cqfml2SgO5BAf\\\\\\\/or5s1EDPRTtF58dDqirM33W\\\\\\\/BgXGt3QsTcFLOPi6DYMQqhChkkVJONvmfBZ39z4zCyw2LmVOwTMxjeT3LUKy7DwWVMulx+8pk3MWcmx4YVQSaOlpVIuwu9mklGkh2TqjjbbgW4PoKGZo++Tn7tyG+moojzgyJLrt10KMvEZhbAwsK+Bp7WW86gX1g+CUpxSUrAFSHLaV85IsxiihxZ8+U+tfRaD52X2U++UwlV\\\\\\\/GBy+l8uFWwaKiiLTg4rCWZVu2X6Q4ERPkBhklHWt3LUuv3BGvuRP4\\\\\\\/d\\\\\\\/7efJos0KQmKmVwxiM55diZE3FbV\\\\\\\/XGNyMaSAvltKMdPN7VYOtjZVvukeTNI8FoTL\\\\\\\/A9d3L0Tag5Qykzkxiw1OFBj0XAlMzZZlMoZkszQzz40uEVPHfuu3mL+ESaPk3\\\\\\\/P7nleWUlroG7qNvySTdgT4H26wd7l53Pz+Q49Dvvdd1\\\\\\\/MXpxQtgs7u9gx\\\\\\\/5Lt7eD6kId0CCkh5XVi\\\\\\\/04wQOkt+foZFUkQ+AZ+Y9oOkshG7oe72ke\\\\\\\/3R1iplrgKApiMwya83KLfUZSatG04rxaXFZZQiChVr3BcqR4aJnOVTCN\\\\\\\/n+HDeuvy39McTACyNUhuonZzZUr+wNwr95jRNh\\\\\\\/vyFO8LYw+6BswmsJZzzPwjSJnutLIwEHXih5xLOZDEUFhNWryzkmSK+p6AcUF68cq56w3WpP0HyJU2QFlV7xMXFOsvQGXQ43ZykO\\\\\\\/cEKHsN2BgloLKiXhYUgt3r8ASRaeEg\\\\\\\/axjka64p4Id6JiNCOReQp6i3Ll0cKUYe5Jpb3Irp1jIcajeEYXeGx4vR3XPW7FD5IWia0k2nmPm6xXwSwYr69XcSidK6Q4K4mBiHuXP3np9Zz836+Gm090lF4Yuk0trO\\\\\\\/OticKgAX03LvRuzoQfOAG0gR32Ir7fPtuukvzyQ6PJ0BNd2Kva2pln7pPhDn39S4J+V0oP769M20CZ+2nZq0tE1ZFlttj5\\\\\\\/ryzkz8c4IBG4yVsI1NIlP3yhL7EpsAhRVBAm3lXqbg9e0UEQYxu2sjQiCwEGOwMxpaqKe0AF8ntLQimBY2gTTbnvzWD0pgTo4PawQjlJVZnb2eEia2\\\\\\\/shO2lPDMpO\\\\\\\/Uopbmg4P1dpGVgyLUFIyET1CMUntskrBY1t2nuxkFEnKvMMUfTBYnuerqgbURPSDNlzz\\\\\\\/c+YBzafjPN1CFIFw1orCNVFCsUttfVS4B+RNdXhBsGpTXjOG7GseHfiEvetsIDUNCKZD5gNuI1Isa8IFcaoym2D8NhE1MGOjbF1O66kd7Jv6Oas8S+DNvvGbNdaV2gNDD6VgzugmTvsEtCiNAF8xg01kOKTKgkA5wLOBb9k196CQoHK1LzlKiQgbf0ElSyv3DTuNI3ekB2rfXdV5I6Ruq8pXlrM4RnBVaddOAM9KjlnNqgGz4kE03BNlTOq72pOEd43FXNF9UO5gN7oBt5pXXrl\\\\\\\/8PSCEB7Tte0f5wScPIqvuGtJP7VaAcEfq8B7Myc78vWwhysBpnLl96e9FoXXFw7WKgkYOHZM+7\\\\\\\/TAqwWZPlT8CuAVZBlBDbIjVn+T95MfoHlhvTETU5mNjD2OY8SygUborvi6zBJZ165T1cDZiNVmaLcyghibLioSZuayMiEKYxzU5Tudb+alh6ugkO1m1T\\\\\\\/JEFd\\\\\\\/Ho+dPlHANdME3uiYSraHg2TZjgnfeocUrGokbn8tNnznrRfd0YYd4bjbBnkrDhiKwsCSyc6h+1jUTDRJ0xIrzOig6e47ovRuVsbEyCdzMBwqjWifKUdAVtXjt+G\\\\\\\/KHRiOCwW1wo6U\\\\\\\/iyDr7LcyytG1l04vwmmL6Vj5WR1vbA+KmrNy2o5jCtlXrX\\\\\\\/NCd3lE2oUwMdafXkC0NdIihXwOeQVyRjTu+\\\\\\\/8blFk\\\\\\\/p5BNXqEythP6+8Welzo7HyFDcRsCzfa2X8OCSO8Z5UYkVyrp47hoWY58PozYuzJlwaFsabptIyo3rpirV350jzC3DtxvFnlEAv1K4aYJlDFbT3rNOcBaTNFXcyQo0rNnPVNLab0lzxxddyD1qCXR3ZMR2VAzE8d8HXFGjfDSVG8eMyEA1hivkuhJ4VKYlY5lwRLUHPMi+6DWJ9t05oRMd0Pz56\\\\\\\/0wwkoFw\\\\\\\/TRhehp5B2QaphtJZnV6dlVTVGRhHcjfYCyGnmiBfwkLd1aMzg4mA7Xa91+20Tlisb7eBo8qEV4GmY\\\\\\\/mjNZe9+Row23IDGt7mQu7z545cKaeK\\\\\\\/xcSjQSvUYzhGBWQMmLJWpSNaPurTW7AkDmMu0YkXca3T96BQqsGp0eXmr8w8Qy+6oiFhabIiam4SobpjYk9xjgRqpXVLoezuhGRkZuk+E1GWULsBk6wmFmr+OcIWD\\\\\\\/A5KEo8hLtTSRq6fucU32a8ImjLFbq1JnYMCRAiEuVWrtbax5wIh4+SQsy3LL3+DfN0C4WMwbwpQwFsnbs8CiNjxe4\\\\\\\/g1HAiggxEeWMij1Q461LjORF3MhYqOTgA0x2dX3oMGsqVVEyEaO6AFXj9bFJVs1qa0fxzpb15rhKJbE0EHKpeCOfmY99SDpfgA2pEd3xZmn3mpEZ0NJrr\\\\\\\/nPjswYMOQo1nZd2iyOMFjdpCQOy\\\\\\\/H62cdgKvmqjJnos6+FsXsG\\\\\\\/H01+8fMkzz4+7JXYOrValA1ryfOoK+YPzHkAGukGGTEbHYBfkMfgxnsZYDnHmhwcMulICrv\\\\\\\/pop\\\\\\\/HtNr6vTyueACECh9Y0A8clY0a121pasRj41UXeT1hYyOdkJbrZE1g0R7EuwOGNz7OOp1Sa6IwjvwIm1nM5kK5MMdPPUzFOMd5xpitgHum2nerXVo3LjIPcmdN7\\\\\\\/JPoot+XKXJUOm4KA6V8HaZar66X1O7nJiK\\\\\\\/VilFR5WqEBS5xczAr8X\\\\\\\/dfLKjn2jUWK6QBUsD2CE0FyH7nyXNuffE3GBhpkUZfCmbLCpX0zfBmBDqFv0T3XXAogIqy08YClYLGNvDBy9AUdBFKN8gVPvI8wGMsbwKpYhJ0nYkZ7ofjK2K65isQtlSRn\\\\\\\/5HvGl\\\\\\\/QjelD\\\\\\\/h+f+zzJHUNsOp+5j8OrVXaml0wwFl29b3vhJxjRVilUCMG6cmHsggZ1gvIvzJJTz5WWYZ7lMQFaHD1tfzwpq+KIBghZHPcTwYNg\\\\\\\/6RAMAYyrUzFgH+DlXWWHgLQgb0sHm8qUjjMTKzyLnrKv3ADLnm1earRdfkoC85PYRXCUkNKdEWTKSBz+IuEBitYe2PUM\\\\\\\/3SZa6+kHoCWHbAeU1FLQhgBDepcdtY3TwZeU2ERDzsN5\\\\\\\/raVe2X5Ox\\\\\\\/Jb5Gv0nhVtbuTZIepGlE4GQdhQsd\\\\\\\/7s1Rg\\\\\\\/PZ2A+iDu6YnY\\\\\\\/yj9DjgFa3bCtVUZTzi3VIOPW4leur9An05ELs25g19jwUSC8kS0UWS9KgW2YGkElckQjLBsWM2HWHqyD5rbme7rXXaopbD9rF7VdaeMsFa6sH1S4Vtw42VLHi8lQ72q1b1kMjPnPw\\\\\\\/ZN7Sty1mK1nRo3TVd6oiNC7WDc+hoh804iA0iLSMQu7nsUF4HKD4XIZiLncAUoeCdPcDh2YFgaqfPt3FujYjye\\\\\\\/woChtW+6P0CGFJTZlpyy077TmkWJMtxejGuzVdAEBh2Q2fvsY0j1OjovgcdjngN1Qt1\\\\\\\/HF2Xws6eMPH6eOYu+EPjyfXK2Cgjo3GYRYle6ZUoyTwpk+Xzv6EU8WiywSLYHNzdDAtvVkTTXlNVLnVtDcXOxhNwnqmIYeNJ+RtYLi\\\\\\\/RE55TqwSX9CISHYZM3IL\\\\\\\/l2Y3a78WTRoiq3lQhwOweLRKUJ\\\\\\\/XNNdHRxza\\\\\\\/Im7DeSxXz1t0rNiaezlU\\\\\\\/GEeQ5hY1OUPo6HM8\\\\\\\/X7MiFX2Agav0pToFBaP8QKqxiTE2f\\\\\\\/9\\\\\\\/q8yOCPx\\\\\\\/q6GTGb7R1Aq9pJRAipsW4ctlfRaPsphdbauNT9XPMuxkzvc7nYC1Uj1dz3GRnb\\\\\\\/7gFaDCnQtlYBGCBZrcru0WbjBUVApsfkjB\\\\\\\/Xey+g8XpUP2eKt6lLlPstrMPiUgI7YaOWppcNy\\\\\\\/N7eeStTIn6zYOT2sNxExadjyJePoxUOMiOvXAYDFqXOEelTqdLKsMJ8hzrdJVdL7+hhKfoshc+TFRr9SUkg4UdYl8IntiKIQW7jLVD5YUSGSDgFzS+9AP30eaXtGPhFI0A1xwb64xJOM97\\\\\\\/gRhxEugr\\\\\\\/O9XojpxiIVuYOIMCjB8sH4QVqJB3omM4+UBFX0T2htlrQw4V21zMV68f4HG7pmQXwRZbJaIs3sgnyYaRzzYC9Fq8ffOLU3hPHZrXKOjzm24LEjlUXWHDIpyfW3+KghhNAx2EfA+2\\\\\\\/EyeTuuQ2BZ9OgNot5P9Mtzbz9NiU5PnO00iiEXWDPhzaqe87Y15Gi8eUNc\\\\\\\/udJNp7HRVu5eVk+x+sbXmtRQ4M17F2JlmtWHzmu39e5S7Z16rScVdj2kjHlihFonTiD81gxf2k4KNUc5ldTtLtSz\\\\\\\/rEPQ0Ih071EEl3ITNH4DU+7\\\\\\\/BrpPODSJ\\\\\\\/dM5bn6eSC3cBjdgNY4gkcS6ZrA75zt195jCet\\\\\\\/izzF82tdHo6NOXgfV0\\\\\\\/6nQyA2VSu6YPYWvZkdJKhxyRtSsQ3A9mTLhPNAfIE6Xv1mmLRRPzZlFCtfnR9tVIxJxoaTQOLzA9wKtVlGG4LrEkFmA7iafxS4PaGZb93qyv2Mj0YSKfZkkIAotXk+cqnNCXYuMRKyvIXsDllrMqkIIG0DrFj8RV9901xHIa\\\\\\\/XkqVHfUt5b+Jwo\\\\\\\/X9YFb11wlKVbBm8txXWT37aELE2hn93wgRTgHTJ7dyC1QTIwlC3MG+iiAlDyXVZxwAz8Gc5wJYGWWW\\\\\\\/4mRToiJ8masFUxX2Ogjb0yy+2ZGSjsS43scT95c2ZKU4DeLVwbUawhtSqJOTsayxJkDem\\\\\\\/LEviJMRmfQFFpNikZgObPIizfqFcDrWvzHlptF+wYtj+xFVz7xpCiykIakUz41ZZdDw2iX8yzGF1eQuF9R3\\\\\\\/buKZp+rrX+wMwIpkS1kD79XFCwWnANd6jUvF99eQmB2BOoz8mR1TP1doudicM+HYl4MWEyFjT7JhMhKu0xTY95hFOXKgabtal6ZmfG6t23djv0k5EdaSsC\\\\\\\/mib34uJrabD86WIpfekUnUpdrYA4F8YKbeI\\\\\\\/d2ia4Rhy8o+gPwRVBDaXi+GHMQsEG6GP\\\\\\\/qvLegIXb+IBq\\\\\\\/\\\\\\\/v4BSmk9ivtdZ0kx6b3YTaswHu27my4VA+hAGyDtSntrAaudvftwUZnVQLDft5gKRMQ0VtIPN+1JdjMVU5PmIyuZbDLZ\\\\\\\/lYoE5Z2IHRhlHp4c9kF5WdCHxtaGnSjV+Zy6NX30wGZEEDvO4QE0LUGK7Et7brujMLrIXjsfVYape5NZnFG9MDy40LjA6+Sa87v63HKVuGT5E4P4+oU5Ht60jWifSjIKnX4FmmEZTSZOeycxF4CCfqDmrdQ2MEGrxxU6ny7vAWM7PWJ5UjeekhgplEoh\\\\\\\/WyiflmpjYSNv3nLy3tXyhWL07hE9E1NloaAk7F5x+jfflMztwKyXJFRQAOTt071KhVFZVDWIzXhxBa8YQn\\\\\\\/6rCkBmihrlHSvWtFNyosRjYgm5N\\\\\\\/\\\\\\\/ochdlaCPImGTiWLasyRPuH2Zw70ikal+go4bPNp1dLuLhuVMVKtRoIRZE+Q3Yb5nUpIWJBz4kGCVoGu7CqywD8klzAJ5CTLsPE5vH1D7tRDWmBfw9LqNY9YFPgbowPZOnJxoJvgCrz2zzGSc3NujL+eWSU7g3FG6Fo+WN2e5d4pwOEHP1uQwONTWh3SnX+ZR8EdYqCLwNmO0Yevn1CnmNXKASqTYCSqtjX5fhiiUtCsLRZAXh4\\\\\\\/JAQz64DiNtN4w1B3a8kqGV9\\\\\\\/gxBBzP0ng8at1Eusvxw5KD6RhcSV9ZKQ0X6DGoasJdKQSCj0aacjUGmFMQB5xc6FQHHZRcedu1VTTqWLi0KFnlEPsi6iGC5rF8sOhhUBtVdh8hZe6fVSalwiY\\\\\\\/H5NWGWu3esnRYkEKJoNnCOvjkJNhLiBCMIaj9nXrCho1HNMCsrWAgRO9Fao9b7S6eHxCbUlSuFOoztwArJG2IAJ\\\\\\\/4Htqv0jmTdcci4x3KCAePi12mfQwqAvjtsP1GZlCUXoiq50cKNsl7QUPAiuoNgkySJa5VwHTV2xwmY88WgwZOCJd\\\\\\\/TAcGN6lZKE8SqkEEG+o7lp+yfnj43M18fTUHqRma4KVaTHQIlv3Usvwff46xmZOnJQTQaB\\\\\\\/GB8JWrFL3U6bmxQqAC81clGEfN2S5mBU04RT++tl+OYlq9xeAnEv7wYglfMaXwHvtPSOErW0UeWVlm0S70DV3k8xB8ipzBf30CpXJnEDzCnn5lMIFvYV3rxP4LZvxEcjP1BoDyxhPsB3XtEwvFY63gLP4baKKAZRG\\\\\\\/bT5i2MkvmpsrwP0o0gbDY6E5FJBDaE85H1ujB1Yx6Va2d5zShBgmgG4j4db\\\\\\\/7+9We5M8z3KrhMcUybbGWeJlKnXpEiQc0A7L0QngLz7hSPOSKncNSEupzNXQ4ycDkwn8Ubq1j1l8C7zfuGd2AJcfNgCTu\\\\\\\/nETdMK+WOg1HaVLkme5JO4F2JiNcGJnGlJVezXrtS5OPgDTTeVhpN5GGpm2KQA2RCxcSZLrbUj4+P74+Ax68JoMrjOOVB6g0+n6MevRZR0vlsyVAPJI+9\\\\\\\/ymjkzPlapnq+l5BlT6seU1dG\\\\\\\/I+eqsAcZGcECwp1qCHx82ZTAl3vyPqLVhiWSRVDYR9MKfVTWIS3d6J1Cth5MQ0aRmTKJpiC6jxsKBvw+3cRN6wgVFpO4+aAQOi54ZNuBja943mY3r6+Sd\\\\\\\/GCJLJuiXA0gX3oGJjE+Y4\\\\\\\/QcNvGqwc1raUZ1fIaGqJ6MtvwezC\\\\\\\/uGGD68iYjghqIzyCjZZha\\\\\\\/8svRnQWbuKPRzI15e14fl9u81sC6g0YuQUYCpgov4WUmn9th+kwedSaw3CM7X+3CouqDhJUaR3PbuI4ZOl3KOGWJoTw14X0ElxXg0MmZrKAIkE+ZitUxgMBWowsnep8KlG62WXXYA6fJwODqFybYNB85RAxenJQqMrUMzMxkgnfw\\\\\\\/60XOU350KTc6O5aAkBblDD9XWP60pgum3BtpGQPEUJ6Klb1+\\\\\\\/3fKnkWLkvgnAOhUBNRE7dLSWK5wX9TsRSP7pRkiz2\\\\\\\/s0mlH4t1K+LWbVhUDoyYnYxjlISXud+NhzViMvayOAHo8HNXCk5Hl5e7IvjNT7lZNtXX9gb4hlYH3qo3n0XLBSQemuMDZFaBASuUbgZ4p+PcMTRqqUd\\\\\\\/+sQOzIZuFO8Y1rC06103HQV\\\\\\\/Q+VauvvrPWV+cSzEPFTsVrbAyuRceOAOVeR+qSTh53izBJwICOiRxuRMBjHy30svpJ8CWvTi5X0V8gXsgHdQIKbJEXlRDqFINp5itbvedf\\\\\\\/uQ+sYGsn4V1+oYs2dANxDjM68fJqkAVJfnY8c\\\\\\\/N23hb\\\\\\\/MQdW1XcDz7PyNEEPhUnMSyC6ldlSubZ9YY29Bkwq3Kud1Asr308o1bKGWsPzyIOReeE1eDaCEKcsIrWvn4D7gnANBodtVGLvsiM65U4UzCKqZteKWvHo3gi79GRRSFMgpQyum7NxcFgPZjcoygMb0rAW4eP7fjDyu6ufrV9diqSE86ZUgP1iC4KMxho4jHsRUGExfOjHhET2Q+Ey1qgepl0td5JIcIUdR1VMiQ5GYnfzxzh8oQGHq0MOKoHi8S+6OdP4utaLxV00MJTu\\\\\\\/j13ud8EKW2KSoIAXvmTvADQPFbH51371GbVo6oWN4KaS0VvaGswpiwgrNpsG0wI0kq5q\\\\\\\/TgT0aAAD7u+Z1h2pLbiruoBGj+kcdw1bUF3JNvXW\\\\\\\/SzYu7TBQj1oblJvB4q6+IUR4DWEmFLyeqYZ6jup6qVKX2T1Ad8KNdFvwoh\\\\\\\/lxQPIeWJEoEsO9Iu7TTIzHjhB0lN5Korso2dpOhMnk4GOQmI7pNSV8JXwAt4JBxVMi5NUz0ozkWKDX1Kx3nrwQXUI9WEHFXP+hXl9JlUGwHc943EVZhTTkbHw3Rxppw9ZtvLaK8QdT9d2+OuQjVSkA3VAyQvJxxHnoB+OZqUz\\\\\\\/TWIi5uaCXBCe+vXD+5HTnwvBfTTaRks65Oi4V\\\\\\\/g5JpVhvnU0DweZMLkPxUvD0TA0hXvfYY3ynWrIHgBzMRTTMCZKB5zfxd7VLWcqo6CxqUNtHaXM1LsPp3OYVGnY5Jo6q3XJOqz\\\\\\\/8oW4Y3p\\\\\\\/ZYJgNl9AYVB3cD73xK53nYXCsQ0CClbDSh7fSJfchsCs8XkbUvnWaJ0KArh2CrtohfY7J2xOjCW8sht+OzS1SR26bCsdJSUn4SvuB05\\\\\\\/XT\\\\\\\/5CDefLBQh7+I5xPaLU1Ps7y78mymPy369XY8dwAmrO3YbF4KHYTR8ns9g+tiohvHymNQU1v+qHXT6p933RbYFUEIJJ4cbKEx2H+tEgHu3QrT3UGwi\\\\\\\/\\\\\\\/UZFbbcH63Mu2LlVsWE40ob9MZ0\\\\\\\/n3\\\\\\\/2wV3dR30A6WMNrTx73Y85cU\\\\\\\/GpvOFpV5OErNybV8Z7RrwWnXrIOGoSKlXAIC80zPGgtk0aVXvs\\\\\\\/v0PBw36yc1arjzMmsK6StskTPmWZxa+VVOdFBCKVXjIsCW\\\\\\\/ScFLDrWLXI7uTW6VNjlyB\\\\\\\/LVwS3DxnVUgo9gNC1B2o5D9mOcH8aIEevF0EvHn9zzSsZbc2Y3n0Yre6V\\\\\\\/UwzTdLb2mWsBLdzkmTMe4ul980mQfQSbnaEyZSquYPBQ3cynee+ml7AalExBiMBYzqw9gBBFsRGN8+N7rekeAQcfuHhUGztH8Jmp2Mi\\\\\\\/C98TvrOFr2kPjhoZEF2zblpa74KrulAVVixzFshFK4H0escDNv+QwZGVf5\\\\\\\/JJchxK1QvoUu5ZWORnGgQLyMDRJ8iyrbPS7yEWeHY8YsyZHwmMpiG1rw55S94fsocK+\\\\\\\/cQqhH\\\\\\\/Nj3\\\\\\\/qc+2SGvYV92HY6IHhusWcozZVPlzorEYzJh1hj+FzmsZAT8NYuCvdWQsYFsj7apsY+19HNmtsxBrnsLgU8k\\\\\\\/fdbAaGTX+yikDO3Trpxvzm+o906CmM4lDf6Nmu\\\\\\\/EUqNCOLgZEBP4BsvKSM3dpiHhQeH6xLGdS\\\\\\\/7S9bFcAZhkNIGUztFvPR1VmAiQXUfiMxqPhHuHgCIZoiNlH15QlFAny\\\\\\\/V8K8Y1Q6ptJIOMrSTf18tMHFjGJOAYdah6dDqamOd8aEcWsDMcUqBKWW7ac96LI4rKh\\\\\\\/\\\\\\\/RkHcKXI4bw92VZL56OHypVvLf8gIRjI4s\\\\\\\/nJak0j0Dql9q6+1ixl9LvdI6ckomA3kHBF\\\\\\\/P5G6QxYu+vRubsFITxXM\\\\\\\/9bUhx2eBg2n4zk0PlOvVr7UfBgftbUQ+83qToGgQaDRrQ4MvaIiHaHxjGU\\\\\\\/ZyLvo0drQpdUFSLhaFW+XLccyhyClOGHbUPN6mLgdd\\\\\\\/rB98AMlOpehHF5inn0beZtqEUTj8Ck68LhTLrvegCMDX6OYXBkMLsP6oO4Fqoa28TPPKWkVPzsN7gFZJuJEj4DRIBcuvqBMhnhPLvevmjxuw4NxTkPLmw1hVomtdYUC0qgZgR4+UH9qlwo2nlN3DQu2Fq41TM\\\\\\\/bGoCqXqEIt+BKamNdiVmUmLgB9vTmBLhGVqN4rLE+rFIbcC3Rp1n\\\\\\\/7RgtbsBfW0w9U\\\\\\\/wVjZCgsE075KZLgV8d4DQegrhFIyVFGWdUjp6NL7X7rIN9aN5JKUNw5o6SMgPfIJurlcMOi2XLAwDJMYNWZ+63h9BmIVM8OC52cDUp5o692JMveDDGeCvHoufSTEV9u\\\\\\\/Esn93\\\\\\\/CWMq0EqZMYDyZDep\\\\\\\/YElsQar5f0s9pTRJ1zSS2XVThKwtUE6Js07xhkTPQY03cAP\\\\\\\/GmHlVvTRg\\\\\\\/w0pjjBSoWJw1S1Yi6FSs9S+EBsvcqYygebT4pr8LnUP4zi7Sp8NvbDq3wKK\\\\\\\/jJtcx98kjTG+\\\\\\\/5bfWUODj5F7Ff3RgYLr+KYJo\\\\\\\/6lObldknzspAYnUENJ8QjnPuIdNj9+CYev460XXvt5OsfOuIa3+gunzDTWQbiXNGx0yp5COeclrty2V0RpKhuLAZL9m4BjU\\\\\\\/cRmyfgVHtCeVJj0apAICLBW9hDBDTaDDFCjxodQ6wQbOtU\\\\\\\/+N2kvHTzWWBUZUjCOujhuQz9EYDH1iB8Scr2herfsmMr64h\\\\\\\/hDsuBNTtL4qcXY3AHN13R4HScWMHrVYWGj38nF\\\\\\\/gqSS7U4J2HW\\\\\\\/0W4d6KweMk9CwryucdrguSxVEfqJaT9aV919EIuNffSaK6JEElysxcKezi5OPFC2xtC3+McsLwSoHWbRnOm8CC4k8a8spLZXEPM2HPphSQY5+YikwWLFaycK3+Vrvdo4Zmc9+ZV5jfDFu7kpZcw3GcqGAJWWwj12fZVqvmS2ZPysLVC29UFVqfnXsooD59G43G3MdLRxokkS2J8G+20mhbGYlwiGomQIXaA9Oa8R+267KmHtisoYDz3mnYWCql5dyTPA4j\\\\\\\/pdViWUpnHgIK6BPVTfMI9uiI\\\\\\\/2m6SQ3Po\\\\\\\/Py3f6ZL40tslu4qd39VXk6nK4Hi4eHz0cfdkKRK4G2vefsI4bGv333vJLRJlcG06ZP7UD0UF6Y\\\\\\\/M3WFvQb22G4ZXsOJL\\\\\\\/QpnyiilZvrq4C4fcBC2yoCx8nz75KEdzSx\\\\\\\/KC1POJlwsqQ\\\\\\\/DxUa23YMUIq4a0a6n\\\\\\\/fEAki2ItezHL\\\\\\\/7t4K+Q7K0Zr0GF31MhGL+nFzDnYewJoajooInhCrXr0UnMMCsR+XlyQdyPzdIx+vdqPXSIxjyar7V+k9yBJEDrUrBEQok0phN5N9h0eaZ6IhIWtkPhjwuwnYQpMHJghn0XU8+JhOf6\\\\\\\/KosmnpdLLFg+gV40UFxpPAdFE5KFHcFj9+z8Nvlm\\\\\\\/9izqlKibzHkEVpNdy26oQdXRC72xGniAYq2S5\\\\\\\/BDiV29aW6sKtw8s1ixjJUteJfUP+++wBCZNXn+626sZJIJNHnoo2g+dXJKj80Ivs0uSRX8e\\\\\\\/xXx4Z+vPa7yZVdzjgGDde4TKwCJ4MbNMDOP40gSV1v2+tbJIJ9O0Z\\\\\\\/7a07iVq7TLbsKKfYIrqij8wCEZyAKkPRNV1KrRNnv+z5NSLY90FihYrdmbcwBq9XoSO1JJulUKRnB4tv6TB1\\\\\\\/hkaG\\\\\\\/xsmg2S7SoZEwD3nCEacfOa4fvcKiQ8sBXME8FLkx8gwKBeQMHGD64yjl4IOhkd2f3j5F07qJJxzVGCYj4\\\\\\\/przO3WyZ7qNH86y5lwpAUqKyyYie1uyoli2t+JfNsD1+SEpzwb+\\\\\\\/l8O4GthKdlA8m2+ONVEB9mdxLIrXP3dV7VPYDJ+rLV8ZrTzpeloihEXgOdy7Kv3QqIo02YWNcBkOYijVbYQy5jwOWnj2CjdpcoITdOHiQ5kO4DdmG8f6vPlh7DA5kn4u9WIW5CK5DkqitGYBX0QstCrAK2iMrv7\\\\\\\/PbqrZjdNgMxUJKPHQetosmcSjkPuKmKKh2H5+DgEqTyjFyWU2wyFPLG0zpod0OBpHItawEgozMoV6RPuy2\\\\\\\/Fp1dzsgrYQy0h7dOw8uTt5Fv4xGAPUIemEW+IlfsVzry3iSSOVNv7yfRK5aIURzzIioWLZ3uFxdnLjAEKcEyZ5FLcJiYmIDow5TlILkFxu5bZSwl1L94FQntb9krHuzxdfvz7B07AGR47iLe932dZo7+k4KUDlSW5R\\\\\\\/p6sJJW2SAptf5C1b0MztPW5MO8jB\\\\\\\/dLf6Smeb\\\\\\\/K75XhVnHOAo6UZUFGhBz\\\\\\\/x+0cFX\\\\\\\/U7pjmk1avlGpI2p7L\\\\\\\/COtiqoeML3w0RmUor8oVzdZ93DMmzTCHjfi6gcf1Xt0EPJXZ4F28R8wVbGEQFTCjvOrP\\\\\\\/hT8UXsWbG01eCB+dtqzFVNX6x4NfZ3YO0IjqsesGuqLS6AorkebW\\\\\\\/oM5N\\\\\\\/79+d0rATnMMZIBepldq0hOIyC7psZQ4MTwC9nhdlVlfFo8FGlNWdesnxQ63VRFLcw0nZ5MNOYpBR49oVcHZPRy931Y\\\\\\\/H4cp5Qaql8XnK2CFfhh5zAMuX4GPSHvSnWQHUgIOI2wx0amrmDhl8Pbg3zGtMtBGWibpC3IY42vlLgDcIMqndrPnMT970SqcK5NWb8mpknqQI8JuEXW4Xrr9yNaHsWUnlsNVYoLueEKWNhKhurtvRDKvArjR4CrfvH\\\\\\\/nDLrUulpSed9RjrqIsRgGhXbK6CNe5Th0Ly+e\\\\\\\/RBsx69EjSJOokUIRBQKdOiK4oMFVgA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022U4fQlm3EAjRi0Yj1A88iKg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d9809198e99a65a23c07660bbcdf371c0a66c90d04e3b028113436bc3d01ac10\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yslU6BEbvPHyFHIsan2BShXS1XwDrN8skgjwzStkJ4qsNcEWn8qn2h+BP2G\\\\\\\/RN2C1WEWcmbz9o67ANlOTidgY5hwN0npysJ5QPh7Z7xitJ3GBGPUfKZFeAX3xd1Wjb3uxzfivuYSCQEaedvn\\\\\\\/T4jeroN+PIMLGiALVDGKBlapkraZF6JVwy+W15lYnhV+KB8iJm58GtjBi5UXkPL+I+YqWYfMbJGdz6\\\\\\\/hlD3wbgQH9kk1ajGwlH6\\\\\\\/FEBvft1coAMg6pb3iN9IhUUG\\\\\\\/WL8YLGwgiNE\\\\\\\/0r3y2fXqL9SkVA01AByzMUS8s9Ok2W5ty+SKyFFs4pr\\\\\\\/taU9U3mFyvCumsYA73lkbVIthb6yiY4itcif9lHsQziL2lkfUOOK4j6sdiySgQ+4+19Rde+\\\\\\\/gLVMCh6OhC9LyFTp5mx3giyEn+LWGiQjBREMY9vxwpaa0ywcDIhpfYnrk\\\\\\\/1zA7keNhEhHr8T05SaJecGglo+sC+TPDySneuvd0KvudgumkYfMEtTHaM2bzNRmZnXcOA41oGvxuyNOzVZDw00yvu\\\\\\\/jKHwZWumld49gnStDEjHcZj6Esya4lpfae90oqsg2F3YW1hDI22AMTXKWzRo\\\\\\\/xnLU9EMBmyAk2nlfUXMvtYSKwAlTjMBFh167EUAtfCBZbWLZUiBrLVLE4YwDDUkU6ABVHXtGccbJIoReuWGlsVN9TaWjJ\\\\\\\/tjzDQTe1azvnxkL6Ou+n1Ah13xFlO5ag7uLETAwIqnQ5v0zMtCz0Q8vnK6PZWD97q9kboLRtNYUb\\\\\\\/yDjM8+2BcOl5fCsySab\\\\\\\/3HAvWXHhKRANj\\\\\\\/N4F9WVHcXmaeAciBHDzUB1KIVND6eMzB03DX9MRbJPU2WXFtEqKsPf65MSeg5vd9xkNjZg7sjDI6oWlpeD9pBgt5LZIqh8s35rAhj0DguY2IoMp2r8ifElyQWoQ04fv8\\\\\\\/Y9li0qv+aJrP1xwZJ00P5y7eQkm9kGjk49LmS7B6CSFp\\\\\\\/mDhPendp0yR1luV\\\\\\\/QxwChurLQLxNQmA5j1Cjj6HG61T6mOsDPWnJQrPLyiMZaoKh77oPqbPlidN6z7BUoIKG+E86oUnVz6Ze4V785UgOfKMQI0uAjwkBcbX1pgKUKP0lOQ4HPThnzFhRJNXIH\\\\\\\/whV7U5B2Jnp93Mt1F2RFKNQjaci3Xqgzt7wMEZ8wbipnBAVaBohUJePH6nH09p+Pbf+8pD3eDtXoZtbUWYvR7TfkCXzM4YXsMCs14nFekfbKogRdqW6D4tP\\\\\\\/YL4qI1BCe1vuCzAEvvyeGuE3jfJt2m3X7qt7FTO8exQgD+zLnpSz76Mm8\\\\\\\/z6OoKzinWhDivmRhmBi\\\\\\\/sA8DgLZ1fUFBUvLm6ggjYW44jKFmGdwUtGKHhID1jg0BzXtFxnFjKQsSqlA26cP4sgGSgzKePCmIcQr3VRR9GJzqBbGzOS3aJxzGuGsfFp7q9gSq0XZIQiyLzDTMvcA34Kzjx7Ibp7KgCh1Qu0bfvufgCcL8nNhkOTT68MVJZfq8s53uKILcVGJLnw5Z7HIMf5SBgtJVI9F9EKBu\\\\\\\/XHnHgJCXyXyrTM01W1XkSo\\\\\\\/bT5UR4TlUcbsge8lzw4IAUHf+qH+0HuBPRNsdhNYsx6elcRRXNmF7syRayuqSuDD331qnlkykaZzWzlWKAZ+5rIW6Q3JE9XYOdOVhCjgYpk6RbpaJLFZCB45KMFs+NTjfZZ3eyn8R+ggxeWSe\\\\\\\/6Islh6CXnUYvjysi0fLlaEWIgss2cIzBRGgBXFAaaaZsrqhIH3LQOH\\\\\\\/mrijBt\\\\\\\/qj29DZ\\\\\\\/u+8+t9V+kIcfOHAODHAYnSHFpp6letvGZjj0oJWkBfq07upr\\\\\\\/6MrTK8v4\\\\\\\/DLLhn07vQmOytcayKu9fbcPMTPbAob9g3rBI7M+EuL1CZ0I74ax3fsZlJ8oldnGIKJfH571u6pxgGJHnxxjJPLDrNxvoTWegyEC8bWFXawRv004rYrcIUpmtnGyLcPi5I7dWKg2P3NlC+RU7kmZMz1UancVi8VBs\\\\\\\/UBJ+yQ95bg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xxX9kYQQvYHb3QDy+iqd2g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d02260d596a3f6714bc70e7cd0534a8d02f705063b5ac71ca516ea69bf503161\u0022}"},"ajax":{"edit-submit--5":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484767 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1623" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hsG7WXLTca15Ty6DhwJtaciWhf_zq-_BgVMNPPg1P7M" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1623" method="post" id="highwire-search-quicksearch-form-1" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--3">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--3" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--7" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fSRXmisfWhM7CGcVSDDGIVbD7wf01WFcfzX4h_XH65Q" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_1" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-3" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1619" title="Epistasis: Searching for Interacting Genetic Variants Using Crosses" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1631" title="Loci Contributing to Boric Acid Toxicity in Two Reference Populations of Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484767" id="node484767" data-pisa="ggg;7/6/1623" data-pisa-master="ggg;g3.117.041491" data-apath="/ggg/7/6/1623.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inbred Strain Variant Database (ISVdb): A Repository for Probabilistically Informed Sequence Differences Among the Collaborative Cross Strains and Their Founders</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4569-5160" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel Oreper</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3950-3794" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yanwei Cai</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9984-2600" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lisa M. Tarantino</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fernando Pardo-Manuel de Villena</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1623-1630;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041491 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484767" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Oreper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BOreper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oreper%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BOreper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4569-5160" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel Oreper</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yanwei Cai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanwei%2BCai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cai%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYanwei%2BCai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3950-3794" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yanwei Cai</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lisa M. Tarantino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Division of Pharmacotherapy and Experimental Therapeutics, Eshelman School of Pharmacy University of North Carolina, Chapel Hill, North Carolina 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BM.%2BTarantino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tarantino%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALisa%2BM.%2BTarantino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9984-2600" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lisa M. Tarantino</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Villena%20FP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fernando Pardo-Manuel de Villena</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/6/1623" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484767" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484767/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1623.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484767" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484767/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1623.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484767" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484767/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1623.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484767" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484767/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/6/1623.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484767" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484767/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The Collaborative Cross (CC) is a panel of recently established multiparental recombinant inbred mouse strains. For the CC, as for any multiparental population (MPP), effective experimental design and analysis benefit from detailed knowledge of the genetic differences between strains. Such differences can be directly determined by sequencing, but until now whole-genome sequencing was not publicly available for individual CC strains. An alternative and complementary approach is to infer genetic differences by combining two pieces of information: probabilistic estimates of the CC haplotype mosaic from a custom genotyping array, and probabilistic variant calls from sequencing of the CC founders. The computation for this inference, especially when performed genome-wide, can be intricate and time-consuming, requiring the researcher to generate nontrivial and potentially error-prone scripts. To provide standardized, easy-to-access CC sequence information, we have developed the Inbred Strain Variant Database (ISVdb). The ISVdb provides, for all the exonic variants from the Sanger Institute mouse sequencing dataset, direct sequence information for CC founders and, critically, the imputed sequence information for CC strains. Notably, the ISVdb also: (1) provides predicted variant consequence metadata; (2) allows rapid simulation of F1 populations; and (3) preserves imputation uncertainty, which will allow imputed data to be refined in the future as additional sequencing and genotyping data are collected. The ISVdb information is housed in an SQL database and is easily accessible through a custom online interface (<a href="http://isvdb.unc.edu">http://isvdb.unc.edu</a>), reducing the analytic burden on any researcher using the CC.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/collaborative-cross" class="hw-term hw-article-keyword hw-article-keyword-collaborative-cross" rel="nofollow">Collaborative Cross</a></li><li class="kwd"><a href="/keyword/inbred-strain" class="hw-term hw-article-keyword hw-article-keyword-inbred-strain" rel="nofollow">inbred strain</a></li><li class="kwd"><a href="/keyword/online-gui" class="hw-term hw-article-keyword hw-article-keyword-online-gui" rel="nofollow">online GUI</a></li><li class="kwd"><a href="/keyword/variant-imputation" class="hw-term hw-article-keyword hw-article-keyword-variant-imputation" rel="nofollow">variant imputation</a></li><li class="kwd"><a href="/keyword/haplotype" class="hw-term hw-article-keyword hw-article-keyword-haplotype" rel="nofollow">haplotype</a></li><li class="kwd"><a href="/keyword/multiparental-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations" rel="nofollow">multiparental populations</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-4">The Collaborative Cross (CC) is a large panel of recombinant inbred mouse strains derived from a genetically diverse set of eight inbred founder strains: A/J (AJ), C57BL/6J (B6), 129S1Sv/ImJ (129), NOD/ShiLtJ (NOD), NZO/HlLtJ (NZO), CAST/EiJ (CAST), PWK/PhJ (PWK), and WSB/EiJ (WSB). These eight founder strains were first outcrossed for three generations to produce mice with contributions from all eight founder strains. These outcrosses were initiated, with different founder orderings, in over 1000 independent breeding funnels (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Shorter <em>et al.</em> 2017</a>). Mice within each funnel were subsequently inbred for multiple generations until two or more animals were identified by MegaMUGA genotyping as collectively having over 90% of the genome fixed (<em>i.e.</em>, homozygous and consistent for a founder haplotype). These animals, hereafter termed the most recent common ancestors (MRCAs), were then chosen to become the obligate ancestors of all subsequent generations and bred to produce a distinct CC strain. The set of MRCAs from all strains composes the CC’s obligate ancestors; that is, the set of individuals that together circumscribes the initial genetic material that can be passed on to subsequent CC mice. As a result of this breeding scheme, the inbred CC strain genomes are random and independent mosaics of the eight founder haplotypes <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Collaborative Cross Consortium (2012)</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">(Srivastava <em>et al.</em> 2017)</a> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>; more details at <a href="http://csbio.unc.edu/CCstatus">http://csbio.unc.edu/CCstatus</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1623/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Breeding process for two Collaborative Cross (CC) strains. Both funnels begin by outcrossing the same eight founders, but the initial outcrossing order differs, resulting in completely independent populations per funnel. Animals are outcrossed for three generations, then inbred until genotyping reveals at least two animals with &gt;90% consistent homozygosity by haploytpe. These homozygous animals [a.k.a., the most recent common ancestors (MRCAs)] are chosen to become the obligate ancestors for the CC strains; all subsequent generations of a CC strain descend from a subset of the MRCAs. In (A), arrows show CC1 MRCA regions of inconsistent homozygosity (L1) and residual heterozygosity (L2 and L3). After further inbreeding, only L2 continues to segregate. In (B), the CC2 MRCA set includes three animals rather than two. After further inbreeding, only L1 continues to segregate, but a de novo mutation has become fixed at L2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1031389120" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Breeding process for two Collaborative Cross (CC) strains. Both funnels begin by outcrossing the same eight founders, but the initial outcrossing order differs, resulting in completely independent populations per funnel. Animals are outcrossed for three generations, then inbred until genotyping reveals at least two animals with &gt;90% consistent homozygosity by haploytpe. These homozygous animals [a.k.a., the most recent common ancestors (MRCAs)] are chosen to become the obligate ancestors for the CC strains; all subsequent generations of a CC strain descend from a subset of the MRCAs. In (A), arrows show CC1 MRCA regions of inconsistent homozygosity (L1) and residual heterozygosity (L2 and L3). After further inbreeding, only L2 continues to segregate. In (B), the CC2 MRCA set includes three animals rather than two. After further inbreeding, only L1 continues to segregate, but a &lt;em&gt;de novo&lt;/em&gt; mutation has become fixed at L2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1623/F1.medium.gif" width="399" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/6/1623/F1.medium.gif" width="399" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1623/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1623/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484849" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-5" class="first-child">Breeding process for two Collaborative Cross (CC) strains. Both funnels begin by outcrossing the same eight founders, but the initial outcrossing order differs, resulting in completely independent populations per funnel. Animals are outcrossed for three generations, then inbred until genotyping reveals at least two animals with &gt;90% consistent homozygosity by haploytpe. These homozygous animals [a.k.a., the most recent common ancestors (MRCAs)] are chosen to become the obligate ancestors for the CC strains; all subsequent generations of a CC strain descend from a subset of the MRCAs. In (A), arrows show CC1 MRCA regions of inconsistent homozygosity (L1) and residual heterozygosity (L2 and L3). After further inbreeding, only L2 continues to segregate. In (B), the CC2 MRCA set includes three animals rather than two. After further inbreeding, only L1 continues to segregate, but a <em>de novo</em> mutation has become fixed at L2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-6">This combination of independent genomes and high genetic diversity, along with the reproducibility of inbred strains, has made the CC a unique resource in mammalian genetics, and early studies on the CC have begun to exploit these features (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aylor <em>et al.</em> 2011</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ferris <em>et al.</em> 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Phillippi <em>et al.</em> 2014</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rasmussen <em>et al.</em> 2014</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Mosedale <em>et al.</em> 2017</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Green <em>et al.</em> 2017</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gralinski <em>et al.</em> 2017</a>). To make optimal use of the CC strains, it is desirable to have an accurate catalog of the genetic differences between them—specifically, the positions and other characteristics of all known CC strain genetic variants—and to be able to predict which variants will be polymorphic in future hypothetical crosses of CC strains and of CC strains with other laboratory strains.</p><p id="p-7">Such information could be determined directly by sequencing, but CC sequencing, soon to be released for a single male per strain <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">(Srivastava <em>et al.</em> 2017)</a>, is not yet easily accessible. Furthermore, sequencing from a single animal will not resolve uncertainty arising from residual heterozygosity, since two animals from the same strain could easily differ at residually heterozygous loci. More generally, whole-genome sequencing in most organisms is expensive and inconvenient.</p><p id="p-8">A commonly used alternative is haplotype-based variant imputation, whereby comparatively sparse and cheap genotyping data are combined with more complete information about allelic state in (even extremely distant) relatives to infer allelic state at ungenotyped positions in the target sample. This typically involves inferring shared haplotype blocks and, by assuming that individuals sharing a haplotype block also share the corresponding allelic state, using this to impute genotype (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2009</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Marchini and Howie 2010</a>). A broad array of such imputation methods have been developed for use in humans (<em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hawley and Kidd 1995</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Scheet and Stephens 2006</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Marchini <em>et al.</em> 2007</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Browning and Browning 2009</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Howie <em>et al.</em> 2009</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li <em>et al.</em> 2010</a>) and livestock (<em>e.g.</em>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">VanRaden <em>et al.</em> 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hickey <em>et al.</em> 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sargolzaei <em>et al.</em> 2014</a>); these typically either start by inferring haplotypes from the genotype data or by approximating the pool of extant haplotypes via a large reference panel, and then use those haplotypes as a means to impute variant genotypes, which are assumed to be the primary interest.</p><p id="p-9">In MPPs of model organisms, where the founder haplotypes are usually known, it is more common for primary interest to focus on reconstructing the haplotype mosaic itself, <em>e.g.</em>, for the purpose of linkage disequilibrium mapping (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Mott <em>et al.</em> 2000</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Liu <em>et al.</em> 2010</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">King <em>et al.</em> 2012</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Fu <em>et al.</em> 2012</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gatti <em>et al.</em> 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Verbyla <em>et al.</em> 2014</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zheng <em>et al.</em> 2015</a>). In such cases, haplotype-based imputation of specific variants may proceed as a second, refinement step to inform fine-mapping and candidate prioritization (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Yalcin <em>et al.</em> 2005</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Tian <em>et al.</em> 2011</a>). In any analyses using imputed variants, it important to note that the haplotype-based variant imputation is inherently probabilistic. A failure to account for variant imputation uncertainty can negatively affect the robustness of downstream decisions (<em>e.g.</em>, overconfidence in a functional assignment), and can also produce misleading estimations of association significance and/or variant effects (<em>e.g.</em>, <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Marchini <em>et al.</em> 2007</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Guan and Stephens 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kutalik <em>et al.</em> 2011</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zheng <em>et al.</em> 2011</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhang <em>et al.</em> 2014</a>).</p><p id="p-10">Haplotype-based variant imputation lends itself particularly well to MPPs because the haplotype blocks and the variants within are drawn from a known and relatively limited number of founders that can be (more) affordably deeply sequenced and genotyped. This in turn reduces variant imputation uncertainty. For MPP RI strains in particular, once an animal’s variants are imputed, the need for even sparse genotyping is largely obviated in its inbred descendants; they are effectively genotyped as well. Haplotypes can, and have been, similarly imputed for the entire CC population based on the CC MRCAs. In particular, a hidden Markov model (HMM)-based method (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Fu <em>et al.</em> 2012</a>) has previously been applied to MegaMUGA genotyping of the CC MRCA animals coupled with MegaMUGA genotyping from founder animals, (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Welsh <em>et al.</em> 2012</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Morgan <em>et al.</em> 2015</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Srivastava <em>et al.</em> 2017</a>) to impute a probabilistic estimate of each CC strain’s haplotype mosaic. Sequencing of the founders by the Sanger Institute has provided a catalog of the sequence variants within the founder haplotypes (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Keane <em>et al.</em> 2011</a>) as well as their predicted functional consequences (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">McLaren <em>et al.</em> 2010</a>), allowing for CC variant imputation.</p><p id="p-11">However, although probabilistic imputed descriptions of CC haplotypes are already available, the final step of imputing probabilistic CC variant state using these haplotypes is currently left up to the researcher. This imputation step can be time-consuming, especially genome-wide, and it typically requires the researcher to develop their own ad-hoc, nontrivial scripts to parse and process input files. We sought to ease this burden by creating the ISVdb. This database computes and stores imputed probabilistic CC variant information once, and then provides efficient, uniform, and convenient access through a publicly accessible webtool.</p><div class="section" id="sec-1"><div id="sec-2" class="subsection"><h3>ISVdb stored data and functionality</h3><p id="p-12">For all Sanger Institute sequencing variant positions that are exonic (or 100 bp upstream or downstream), polymorphic between CC founders, and correspond to SNP/indels, the ISVdb provides conveniently accessible information on the following:</p><ol class="list-alphalower " id="list-1"><li id="list-item-1"><p id="p-13">unphased genotypes for CC strains/founders, including the functional consequences per genotype, per transcript;</p></li><li id="list-item-2"><p id="p-14">unphased haplotype pairs (hereafter, “diplotypes”) for CC strains, derived probabilistically from MegaMUGA genotyping;</p></li><li id="list-item-3"><p id="p-15">unphased genotypes for hypothetical F1 crosses among and between CC and founder strains;</p></li><li id="list-item-4"><p id="p-16">unphased diplotypes for hypothetical F1 crosses among and between CC and founder strains.</p></li></ol><p id="p-17">All information in the ISVdb is associated with a probability, to reflect the uncertainty of inference (discussed in the next section). The ISVdb interface is designed to be practical and oriented toward concrete tasks. For example, the ISVdb could be used to answer the following questions:</p><ol class="list-alphalower " id="list-2"><li id="list-item-5"><p id="p-18">Given microarray measurements of CC expression, which probes ought to be masked from analysis to minimize the effect of differential hybridization due to variants within the corresponding probed regions?</p></li><li id="list-item-6"><p id="p-19">Where should PCR primers be designed to bind so as to avoid differential hybridization, while still amplifying informative regions?</p></li><li id="list-item-7"><p id="p-20">What are the alleles per variant per CC strain in a given region, in order to perform association mapping?</p></li><li id="list-item-8"><p id="p-21">Given a pair of CC strains, or set of pairs of CC strains, where would the resulting F1 offspring be heterozygous? Which CC strains could be crossed with one another, or against a founder strain, to ensure that a certain region is heterozygous in the offspring?</p></li><li id="list-item-9"><p id="p-22">Which CC strains contain a stop-gain codon in a particular gene?</p></li><li id="list-item-10"><p id="p-23">What is the ratio of missense to synonymous mutations on a particular chromosome?</p></li><li id="list-item-11"><p id="p-24">Which regions are fixed across all CC strains? Which regions are still segregating in a subset of CC strains?</p></li><li id="list-item-12"><p id="p-25">Which regions are most uncertain, either in haplotype or in genotype, across CC strains?</p></li></ol></div><div id="sec-3" class="subsection"><h3>ISVdb preserves uncertainty</h3><p id="p-26">The primary purpose of the ISVdb is to provide genome-wide, inferred CC genotypes. However, this inference depends on several processes and measurements that are themselves imprecise: (1) sequencing of founder strains was imperfect so some founder variant calls are ambiguous or incorrect; (2) uncertainty in the sparse, genotyping-based estimates of CC diplotypes; and (3) the CC strains themselves are still segregating in some regions.</p><p id="p-27">Properly representing and accounting for such sources of uncertainty is essential to avoid inaccuracies in downstream inference. Consider the effect of diplotype uncertainty on predicting the functional consequences of the alleles in an F1 hybrid from two CC mice.</p><p id="p-28">Suppose one of the CC parents had a 40:30:30 probability of AJ/AJ <em>vs.</em> 129/129 <em>vs.</em> NOD/NOD diplotype, where AJ carries a synonymous mutation, whereas 129 and NOD both carry a stop-gain mutation. Assuming the most likely diplotype, AJ/AJ, would imply a synonymous mutation in the F1, even though there is a greater (60%) probability of a stop-gain mutation.</p><p id="p-29">The closest similar resource to the ISVdb, the CC “pseudogenomes” set, (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Morgan and Welsh 2015</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Huang <em>et al.</em> 2013</a>) was designed primarily for sequence alignment: it employs most-likely point estimates of genotype and assumes that all alleles are fixed. Therefore, a key secondary goal of the ISVdb is to provide a resource that retains all of the aforementioned uncertainty in CC and F1 genotype inference. The ISVdb achieves this by storing multiple records per variant, in which each record includes a probability of that variant state.</p></div></div><div class="section methods" id="sec-4"><h2 class="">Methods</h2><div id="sec-5" class="subsection"><h3>Inputs for database construction</h3><p id="p-30">All inputs are based on the GRCm38 mouse reference assembly. Probabilistic estimates of CC unphased diplotypes were computed as of March 24, 2016. These diplotype estimates were derived from a HMM applied to MegaMUGA microarray measurements (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Morgan and Welsh 2015</a>).</p><p id="p-31">Founder variants were determined using the Sanger Institute Mouse Genomes Project mouse variant VCF files, REL-1410 (from October 2014), corresponding to Ensembl release 75 of GRCm38 (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Keane <em>et al.</em> 2011</a>). These VCF files included SNPs and indels (1–100 bp). Exon boundaries were drawn from Ensembl release 75 as well (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ramasamy <em>et al.</em> 2013</a>).</p></div><div id="sec-6" class="subsection"><h3>CC genotype and diplotype inference: derivation</h3><p id="p-32">By tracing the potential transmission path of alleles from founders to CC strains, an expression can be derived for the probabilistic distribution of the unphased CC genotype, entirely in terms of known quantities. That is, we can derive an expression for the unphased CC genotype at each founder variant position, in terms of known unphased founder genotype probabilities from sequencing, and known unphased MegaMUGA haplotyping at markers. Along the way, we can also derive the probability distributions of the CC diplotypes. The equations below are specific to a given variant; they need to be recalculated for every founder variant and CC strain combination.</p><p id="p-33">First, note that the unphased CC genotype probability distribution can be defined in terms of that of the phased CC genotype:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-2.gif"/></noscript></span></span> is the unphased genotype of CC strain <em>c</em>, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-3.gif"/></noscript></span></span> is an unphased genotype with (possibly identical) alleles <em>a</em> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-4.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-5.gif"/></noscript></span></span> is the phased genotype of CC strain <em>c</em>, and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-6.gif"/></noscript></span></span> is a phased genotype such that <em>a</em> is inherited maternally and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-7.gif"/></noscript></span></span> is inherited paternally.</p><p id="p-34">The probability of the (typically unobservable) phased CC genotype can be expressed in terms of phased founder diplotype as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-8.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-9.gif"/></noscript></span></span> is the phased diplotype of <em>c</em> at the variant, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-10.gif"/></noscript></span></span> describes the phased diplotype composed of (possibly identical) haplotypes <em>h</em> and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-11.gif"/></noscript></span></span> which transmit alleles <em>a</em> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-12.gif"/></noscript></span></span> respectively, and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-13.gif"/></noscript></span></span> is the set of all haplotypes.</p><p id="p-35">Assuming maternal and paternal alleles are transmitted independently, the first term in the product of the right hand side of (2) can be expressed as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-15.gif"/></noscript></span></span> is the allele transmitted from parental haplotype <em>h</em>. <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-16.gif"/></noscript></span></span> in turn depends on the <em>founder h</em> genotype, which is uncertain due to potential sequencing error, and on the number of <em>a</em> alleles in that founder genotype, which could be zero (homozygous for the minor allele), two (homozygous for the major allele), or even one (heterozygous) as some loci are not fully inbred in the founders. Assuming transmission of either copy is equally likely,<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-17.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-18.gif"/></noscript></span></span> is the unphased genotype for the founder of haplotype <em>h</em>, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-19.gif"/></noscript></span></span> = the set of all possible genotypes and <em>g</em> is a genotype.</p><p id="p-36">The second term in the product within Equation (2), <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-20.gif"/></noscript></span></span> also needs to be derived in terms of known values: we do not know the probability of a given diplotype at any arbitrary variant position. Rather, we <em>do</em> know diplotype probabilities of the markers to the left and right of each variant position. As such, we can linearly interpolate between the two markers:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-21.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-22.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-23.gif"/></noscript></span></span> are the distances from the variant position to the left-nearest and right-nearest haplotyping markers, respectively. <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-24.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-25.gif"/></noscript></span></span> are the phased diplotypes of the cc strain at the left-nearest and right-nearest haplotyping markers, respectively.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1623/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Example workflow of the Inbred Strain Variant Database (ISVdb) online graphical user interface (GUI). A user has queried the genotype of CC012, on chromosome 19, from 6054740:6054749. The &#x201C;Primary Query&#x201D; panel also allows additional strains, and/or specification of the region by genes instead. The user is interested in all zygosity variants, of all consequences, and all probabilities, and thus has applied no secondary restriction. After the user clicked &#x201C;Submit!,&#x201D; a URL to download the resulting table was generated, as well as an online version of the table. The first three rows of the table are shown here: noticeably they all represent the same variant in the same strain. The difference between the rows is highlighted in the yellow box: the genotype per row and its associated probability. Collectively, the three rows represent that there is a  probability of a  genotype,  of  and  of  at this variant in CC012. (B) The remaining wrapped columns of output from part (A) (A was too wide). Note that each genotype has a different consequence, accentuating that only accounting for the most likely genotype would cause a nonnegligible loss of information. Also, note that this figure was pieced together from a screen capture to fit on a single page." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1031389120" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Example workflow of the Inbred Strain Variant Database (ISVdb) online graphical user interface (GUI). A user has queried the genotype of CC012, on chromosome 19, from 6054740:6054749. The &#x201C;Primary Query&#x201D; panel also allows additional strains, and/or specification of the region by genes instead. The user is interested in all zygosity variants, of all consequences, and all probabilities, and thus has applied no secondary restriction. After the user clicked &#x201C;Submit!,&#x201D; a URL to download the resulting table was generated, as well as an online version of the table. The first three rows of the table are shown here: noticeably they all represent the same variant in the same strain. The difference between the rows is highlighted in the yellow box: the genotype per row and its associated probability. Collectively, the three rows represent that there is a &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-26&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; probability of a &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-27&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; genotype, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-28&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-29&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-30&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-31&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; at this variant in CC012. (B) The remaining wrapped columns of output from part (A) (A was too wide). Note that each genotype has a different consequence, accentuating that only accounting for the most likely genotype would cause a nonnegligible loss of information. Also, note that this figure was pieced together from a screen capture to fit on a single page.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1623/F2.medium.gif" width="440" height="382"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/6/1623/F2.medium.gif" width="440" height="382"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1623/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1623/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484794" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-37" class="first-child">(A) Example workflow of the Inbred Strain Variant Database (ISVdb) online graphical user interface (GUI). A user has queried the genotype of CC012, on chromosome 19, from 6054740:6054749. The “Primary Query” panel also allows additional strains, and/or specification of the region by genes instead. The user is interested in all zygosity variants, of all consequences, and all probabilities, and thus has applied no secondary restriction. After the user clicked “Submit!,” a URL to download the resulting table was generated, as well as an online version of the table. The first three rows of the table are shown here: noticeably they all represent the same variant in the same strain. The difference between the rows is highlighted in the yellow box: the genotype per row and its associated probability. Collectively, the three rows represent that there is a <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/F2/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/F2/embed/mml-math-26.gif"/></noscript></span></span> probability of a <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/F2/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/F2/embed/mml-math-27.gif"/></noscript></span></span> genotype, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/F2/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/F2/embed/mml-math-28.gif"/></noscript></span></span> of <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/F2/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/F2/embed/mml-math-29.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/F2/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/F2/embed/mml-math-30.gif"/></noscript></span></span> of <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/F2/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/F2/embed/mml-math-31.gif"/></noscript></span></span> at this variant in CC012. (B) The remaining wrapped columns of output from part (A) (A was too wide). Note that each genotype has a different consequence, accentuating that only accounting for the most likely genotype would cause a nonnegligible loss of information. Also, note that this figure was pieced together from a screen capture to fit on a single page.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Equation (5) is expressed in terms of phased diplotypes, but our observed MegaMUGA marker diplotype probabilities are unphased. If we assume that both phasings are equally likely, then for the left marker (similarly for the right),<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-32.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-33.gif"/></noscript></span></span> is the unphased diplotype. The probabilities of each unphased diplotype are known from microarray assays of the CC strains.</p><p id="p-39">This concludes the derivation: using (1)–(6), the unphased CC genotype distribution, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-34.gif"/></noscript></span></span> can be fully expressed from known unphased founder genotype distribution <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-35.gif"/></noscript></span></span> and known unphased CC marker diplotype distributions <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-36.gif"/></noscript></span></span></p></div><div id="sec-7" class="subsection"><h3>Genotype and diplotype inference for simulated F1 offspring</h3><p id="p-40">Once CC genotype/diplotype probabilities have been inferred, unphased genotype/diplotype probabilities for F1 offspring between CC strains and/or founder strains can be inferred as well. As in the previous derivation, we can begin by expressing unphased genotype in terms of phased genotype:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-37.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>Where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-38.gif"/></noscript></span></span> is the unphased genotype of the offspring of inbred strain 1 and inbred strain 2. <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-39.gif"/></noscript></span></span> is the phased genotype.</p><p id="p-41">Each phased genotype can then be rewritten in terms of the alleles transmitted from its parent strains. Assuming transmission of an allele from strain 1 is independent of transmission of an allele from strain 2,<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-40.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>Combining (7), (8), and (4), the unphased offspring genotype distribution can be expressed in terms of known quantities. Diplotype probabilities for F1 offspring can be derived nearly identically.</p></div><div id="sec-8" class="subsection"><h3>Functional consequence inference</h3><p id="p-42">Functional consequences per variant, and per transcript, have been predicted in the founder strains by the variant effect predictor (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">McLaren <em>et al.</em> 2010</a>). We assume that any CC strain inheriting a founder’s haplotype inherits the same transcripts and functional consequences as were in the founder in that haplotype region. This assumption, that the genetic background will not affect functional consequence, is not necessarily true; if a recombination event were to occur within a gene (joining the sequence of two different founders into the same gene in a CC strain) and was upstream of some variant, that variant might no longer have the same effect. Nonetheless, our assumption is mostly reasonable: given the relatively small number of recombinations per CC line—on average 135 (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Srivastava <em>et al.</em> 2017</a>)—the number of midgene recombinations is necessarily small, and an even smaller number of these recombinations will actually have an effect on downstream variants. What might in fact be problematic to functional prediction within the CC, more so than within-gene recombination, is the original process by which functional consequences were predicted in the founder strains: predictions were made a single variant at a time, without accounting for other, potentially compensatory variants within the same gene.</p><p id="p-43">Of note, functional consequences in the ISVdb are represented as uncertain; <em>i.e.</em>, the probability of a given CC genotype is also applied to the functional consequence of that genotype.</p></div><div id="sec-9" class="subsection"><h3>Database and GUI implementation</h3><p id="p-44">Scripts to parse VCF and haplotype files, perform genotype and diplotype inference, and store the resulting processed information in a MariaDB database, were implemented using a combination of Python, VCFtools (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Danecek <em>et al.</em> 2011</a>), and R (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">R Core Team 2015</a>). The ISVdb online GUI was implemented using the Python Flask library. The GUI was deployed online on the Carolina Cloud Apps managed platform, provided by UNC Information Technology Services.</p><p id="p-45">Two sets of tables are stored within the MariaDB database: (1) an almost completely normalized set of tables with minimal redundancy and (2) a set of prejoined, nonnormalized tables derived from the normalized table that are designed to allow the GUI (which provides for typical ISVdb use cases) to return results efficiently, using a minimum number of joins. This second set of tables was an intentional trade-off of space for time.</p><p id="p-46">A few additional database optimizations of note were necessary, especially to rapidly generate probabilistic variant states for F1 crosses. In particular, the ISVdb.v1.1 “database” is actually implemented (using R code) as a collection of smaller databases, in which each smaller database represents a single chromosome. Where applicable, most tables were indexed by variant and strain. Tables sizes were reduced by dropping those variant diplotypes (and corresponding genotypes) whose probability was &lt; 0.001. Consequently, probability distributions at some variants may not sum exactly to 1.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-47">The ISVdb version corresponding to this publication is ISVdb.v1.1, uploaded on March 15, 2017. A frozen snapshot of ISVdb.v1.1, including the v1.1 website interface and the variant information it provides, will be maintained at <a href="http://isvdb.unc.edu/archive">http://isvdb.unc.edu/archive</a>. Subsequent ISVdb versions will be housed in the archive as well. The frozen ISVdb.v1.1 contents, and the inputs used to generate those contents, are also permanently stored on Zenodo at <a href="https://doi.org/10.5281/zenodo.399474">https://doi.org/10.5281/zenodo.399474</a> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Oreper <em>et al.</em> 2017b</a>). All ISVdb.v1.1 results can be recreated from Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS10.pdf">File S8</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS10.pdf">File S9</a>, which are briefly described below, and in further detail in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS1.docx">File S1</a>. As subsequent versions of ISVdb are developed, their contents will also be archived (at another location) on Zenodo.</p><p id="p-48">For ISVdb.v1.1: <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS1.docx">File S1</a> contains detailed descriptions of all supplemental files; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS10.pdf">File S2</a> contains marker diplotype data for CC strains; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS10.pdf">File S3</a> contains nonmitochondrial SNPs for founders; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS10.pdf">File S4</a> contains indels for founders; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS10.pdf">File S5</a> contains mitochondrial SNPs for founders; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS10.pdf">File S6</a> contains the custom format specification for these VCF files; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS10.pdf">File S7</a> contains BL37.75 exons (along with other genomic features); <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS10.pdf">File S8</a> contains an ISVdb.v1.1 dump of the imputed CC diplotypes per strain and per chromosome, in CSV format; and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS10.pdf">File S9</a> contains an ISVdb.v1.1 dump of the imputed CC genotypes per strain and per chromosome, in CSV format.</p><p id="p-49">Code used to generate the ISVdb and its GUI is available at <a href="https://github.com/danoreper/ISVdb.git">https://github.com/danoreper/ISVdb.git</a> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Oreper <em>et al.</em> 2017a</a>).</p></div></div><div class="section results-discussion" id="sec-11"><h2 class="">Results and Discussion</h2><p id="p-50">The ISVdb houses and provides GUI access to imputed probabilistic genotype and diplotype data, for all eight founders and all (as of March 24, 2016) 72 CC strains. CC allelic state is imputed at all exonic (± 100 bp) founder SNPs and indels (which can be as long 100 bp), but not at founder large structural variant positions (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Morgan <em>et al.</em> 2017</a>).</p><p id="p-51">According to the ISVdb estimates of allelic state across strains, the genotype in most variants is known with high certainty (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS10.pdf">File S10</a>). Variants with uncertain genotype appear widely and evenly distributed across the genome (Figures S2–S103 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/FileS10.pdf">File S10</a>). Residual heterozygosity, a key driver of uncertainty, is estimated to affect <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-41.gif"/></noscript></span></span> of exonic (± 100 bp) variants overall, but can vary dramatically between strains and chromosomes; <em>e.g.</em>, the proportion of variants affected by residual heterozygosity ranges from 0 in CC003 on chr 2 to 0.38 in CC056 on chr 8 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/TableS1.pdf">Table S1</a>). Note that heterozygous variants are defined as those with at least a 25% chance of continuing to segregate.</p><p id="p-52">Approximately 72.6% of (the polymorphic) CC strain variants are identical to the B6 mouse reference genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/TableS1.pdf">Table S1</a>). Intronic (8.8%), downstream (4.3%), noncoding transcript (3.4%), and upstream variants (2.9%) differing with respect to B6 are the next most common mutations, while alleles expected to have a large effect, such as stop-gain (0.003%) or stop-loss (0.001%) mutations, are extremely rare (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1/TableS2.pdf">Table S2</a>).</p><div id="sec-12" class="subsection"><h3>Database accessibility/usability: ISVdb GUI</h3><p id="p-53">The intended interface to ISVdb data is through the publicly accessible ISVdb GUI, hosted at <a href="http://isvdb.unc.edu">http://isvdb.unc.edu</a>; the GUI allows what we believe to be the most common types of queries.</p><p id="p-54">The ISVdb GUI can be broken up into roughly three panels:</p><ol class="list-alphalower " id="list-3"><li id="list-item-13"><p id="p-55">A primary query panel that allows the user to query the ISVdb for: (i) inbred strain genotypes, diplotypes, and F1 genotypes and diplotypes; (ii) specify strains of interest; and to (iii) specify the genomic region(s) of interest; by basepair window, by genes, or by (internal ISVdb) variant IDs.</p></li><li id="list-item-14"><p id="p-56">A secondary restriction panel that allows the user to limit results: (i) to the maximum probability estimate of variant state; (ii) and/or only to that variant state which is more likely than a user-specified probability threshold; (iii) to variants of a particular zygosity; and (iv) for genotype and genotype cross queries, to variants having particular functional consequences.</p></li><li id="list-item-15"><p id="p-57">An output panel that allows the user to: (i) submit the primary query with secondary restrictions; (ii) save the full results by opening a download URL in a browser; and (iii) examine the (first 1000) results in a sortable and searchable table displayed online.</p></li></ol><p id="p-58">Additionally, the GUI provides: (i) a link to complete archived versions of the ISVdb and (ii) a link to CSV dump files of genotype and diplotype, per strain, per chromosome.</p></div><div id="sec-13" class="subsection"><h3>GUI-based genotype query</h3><p id="p-59">The ISVdb is most typically used to determine the genotype of a set of CC strains in some region. When the ISVdb GUI is queried for genotype, it produces a table with the following columns:</p><ul class="list-simple " id="list-4"><li id="list-item-16"><p id="p-60">variant_id: an internal ISVdb variant ID per variant.</p></li><li id="list-item-17"><p id="p-61">chrom: chromosome of variant.</p></li><li id="list-item-18"><p id="p-62">pos: variant start position, in mm10 coordinates.</p></li><li id="list-item-19"><p id="p-63">strain: the inbred strain– either a CC or a founder strain.</p></li></ul><p id="p-64">The unphased genotype (allele_1 and allele_2 are arbitrary):</p><ul class="list-simple " id="list-5"><li id="list-item-20"><p id="p-65">allele_1: the sequence of one allele at the variant.</p></li><li id="list-item-21"><p id="p-66">allele_2: the sequence of the other allele at the variant.</p></li><li id="list-item-22"><p id="p-67">prob: the probability that at this variant, this is the actual genotype. Note that at any given variant position there is a distribution of possible genotypes; as such there will be multiple rows representing each variant position, each with its own probability.</p></li><li id="list-item-23"><p id="p-68">is_max: whether this is the maximum likelihood genotype at this variant.</p></li><li id="list-item-24"><p id="p-69">gene_id: the Ensembl ID of a gene enclosing the variant. There may be multiple overlapping genes enclosing a variant, resulting in a separate row per gene for the same variant.</p></li><li id="list-item-25"><p id="p-70">transcript_id: the Ensembl ID of a transcript enclosing a variant. A single variant will usually be enclosed by multiple transcripts, each of which is affected differently by the variant; <em>i.e.</em>, there will different consequences per transcript, at the same variant, necessitating a separate row per transcript for the same variant.</p></li><li id="list-item-26"><p id="p-71">consequence_1: the predicted consequence of allele_1 on the transcript, with respect to B6. If allele_1 is the B6 allele, the consequence is “reference”. Note that a consequence is always with respect to some transcript.</p></li><li id="list-item-27"><p id="p-72">consequence_2: the predicted consequence of allele_2 on the transcript.</p></li></ul></div><div id="sec-14" class="subsection"><h3>Example workflow for a genotype query</h3><p id="p-73">We provide an example of a genotype query to illustrate a partial ISVdb workflow, and also to demonstrate how uncertainty is represented in the ISVdb by storing multiple rows for a single variant in a single strain. Details are provided in the <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> caption.</p></div><div id="sec-15" class="subsection"><h3>Genotype queries are similar to other ISVdb queries</h3><p id="p-74">Genotype queries are just one of the four types of queries enabled by the ISVdb GUI. The remaining types of ISVdb queries are almost identical to a genotype query, and all of them represent uncertainty in the same manner as a genotype query. Rather than describing them exhaustively, we will emphasize how each differs from a genotype query.</p><ol class="list-alphalower " id="list-6"><li id="list-item-28"><p id="p-75">Genotype cross query: rather than accepting a list of strains, this query only accepts two strains, simulates their F1 offspring, and returns data nearly identical in structure to genotype query data, except for a second strain per record.</p></li><li id="list-item-29"><p id="p-76">Diplotype query: there is no notion of a functional consequence with regard to a diplotype, thus, diplotype queries return neither functional consequences nor transcript IDs, which are closely tied to functional consequence in the ISVdb. Additionally, instead of records with (allele1, allele2) genotype, a diplotype query returns records with (haplotype1, haplotype2) diplotypes.</p></li><li id="list-item-30"><p id="p-77">Diplotype cross query: just like the genotype cross query, this query accepts two strains as input and simulates their F1. It returns diplotype data that is nearly identical in structure to that from a diplotype query.</p></li></ol></div><div id="sec-16" class="subsection"><h3>Incorporation of CC Sequencing and up-to-date CC genotyping</h3><p id="p-78">The basis for the current version of ISVdb variant calls is integration of MegaMUGA genotyping data from the MRCAs of each CC strain with the sequencing data from a single mouse per founder strain (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Keane <em>et al.</em> 2011</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Fu <em>et al.</em> 2012</a>; <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Srivastava <em>et al.</em> 2017</a>) Consequently, the ISVdb currently has important limitations regarding completeness, heterozygosity, and imputation precision. With regard to completeness, in the generations since the MRCAs, additional mutations have accumulated and, thanks to the extremely small effective population size within each CC strain, rapidly become fixed. Sequencing results suggest that the number of variants now segregating in the CC has increased by 2% since the MRCA generation. (<a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Srivastava <em>et al.</em> 2017</a>). Similarly, many of the regions harboring residual heterozygosity in the MRCA animals have by now become fixed. Regarding the precision of the imputation itself, the ability to construct the underlying haplotype mosaic is limited by, among other things, the resolution of the MegaMUGA genotyping array (or of any array), and particularly how well that array can mitigate the inherent difficulties arising with inferring haplotype state at recombination breakpoints and regions of identity-by-descent between founders. Our imputed variants thus reflect an incomplete and uncertain view of the current generation’s CC genomes.</p><p id="p-79">To address these limitations and gain a deeper understanding of the CC population, finer resolution data from a more recent breeding generation has been collected: the genome of a single male per CC strain has been sequenced (<a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">Srivastava <em>et al.</em> 2017</a>), and this will not only identify the <em>de novo</em> CC mutations but also reduce the uncertainty in CC strain genotypes at known variants. Nonetheless, since sequencing is limited to a single animal per strain, it does not by itself provide a definitive answer genome-wide, largely due to residual heterozygosity. In the near future, a set of three other males per strain from the sequencing generation will be genotyped on the MUGA platform (Fernando Pardo-Manuel de Villena, personal communication).</p><p>The ISVdb will progressively incorporate these new sets of data into its variant representation according to the following release schedule:</p><ul class="list-simple " id="list-7"><li id="list-item-31"><p id="p-81">IDVdb.v1.1 (March 15, 2017): the current ISVdb version, corresponding to this publication.</p></li><li id="list-item-32"><p id="p-82">ISVdb.v1.2 (<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-42.gif"/></noscript></span></span>June 2017): inclusion of the whole genome (rather than exons ± 100 bp as in ISVdb.v1.1), and the incorporation of the latest MGP sequencing of the founder panel (REL-15.04).</p></li><li id="list-item-33"><p id="p-83">ISVdb.v2.0 (<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-43.gif"/></noscript></span></span>July 2017): inclusion of sequenced <em>de novo</em> variants as new records in the ISVdb. This depends on projected availability of the sequencing data, which is expected in April 2017.</p></li><li id="list-item-34"><p id="p-84">ISVdb.v2.1 (<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-44.gif"/></noscript></span></span>Sep 2017): inclusion of a first pass at integration of sequencing and same-generation MUGA genotyping, to better impute state at known (rather than <em>de novo</em>) variants: where sequencing and MUGA variant calls are inconsistent, ISVdb probabilities will be some sort of weighted sum of the data sources. Depends on projected availability of MUGA genotyping data, expected in July 2017.</p></li><li id="list-item-35"><p id="p-85">ISVdb.v3.0 (<span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-45.gif"/></noscript></span></span>December 2017): (a) a more sophisticated method of modeling the inconsistencies between and among genotyping and sequencing data to arrive at better estimates of genotype uncertainty and residual heterozygosity; (b) this may be coupled with reestimation of the founder genotypes themselves using CC sequencing data, which indirectly provide us with ultradeep founder sequencing and could be thought of as ancestral variant imputation; and (c) inclusion of indels/SNPs for the whole genome rather than only for exons.</p></li><li id="list-item-36"><p id="p-86">ISVdb.v3.1 (<span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1623/embed/mml-math-46.gif"/></noscript></span></span>February 2018): inclusion of structural variants (&gt; 100 bp) and submission of a follow-up publication describing the latest developments.</p></li></ul><p id="p-87">As the reference genome and its exon annotations changes, the ISVdb will be updated as well.</p><p id="p-88">In addition to updating the ISVdb with more accurate state, incorporation of the newest high-resolution data will allow a useful comparison between generations. In particular, detection of loci that have become fixed since the MRCA generation will open a new line of inquiry as to the possible selective advantage of the newly fixed alleles.</p><p id="p-89">In summary, we have developed a database that stores imputed probabilistic variant state for CC strains and founder strains, and can rapidly generate probabilistic variants states for F1 populations. Imputed state includes alleles as well as predicted functional consequences of those alleles. This resource is a useful complement to sequencing data, and is easily accessible at <a href="http://isvdb.unc.edu">http://isvdb.unc.edu</a>.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-90">This work was primarily supported by the National Institute of Mental Health of the National Institutes of Health under award R01-MH100241 (to L.M.T. and W.V.). D.O. also received partial support from a Pharmaceutical Research and Manufacturers of America Foundation predoctoral informatics fellowship. High-performance disk space, as well as implementation guidance, was generously provided by the University of North Carolina Information Technology Services.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-91">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041491/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-92"><em>Communicating editor: S. Sen</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-05"><span class="received-label">Received </span>January 5, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-03-20"><span class="accepted-label">Accepted </span>March 20, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Oreper <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.1" data-doi="10.1101/gr.111310.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds-Mathes</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdugo</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic analysis of complex traits in the emerging collaborative cross.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1213</span>–<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111310.110%26rft_id%253Dinfo%253Apmid%252F21406540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.2" data-doi="10.1016/j.ajhg.2009.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DB.%2BL.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D210%26rft.epage%253D223%26rft.atitle%253DA%2Bunified%2Bapproach%2Bto%2Bgenotype%2Bimputation%2Band%2Bhaplotype-phase%2Binference%2Bfor%2Blarge%2Bdata%2Bsets%2Bof%2Btrios%2Band%2Bunrelated%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.01.005%26rft_id%253Dinfo%253Apmid%252F19200528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19200528&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263799700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.3" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the collaborative cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCollaborative%2BCross%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D389%26rft.epage%253D401%26rft.atitle%253DThe%2BGenome%2BArchitecture%2Bof%2Bthe%2BCollaborative%2BCross%2BMouse%2BGenetic%2BReference%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.4" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.5" data-doi="10.1371/journal.ppat.1003196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bottomly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitmore</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aicher</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Modeling host genetic regulation of influenza pathogenesis in the collaborative cross.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D9%26rft.spage%253De1003196%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003196%26rft_id%253Dinfo%253Apmid%252F23468633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23468633&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.6.1623.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Villena</span>  <span class="cit-name-given-names">F. P.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> Inferring ancestry in admixed populations using microarray probe intensities. <em>Proceedings of the ACM Conference on Bioinformatics</em>, <em>Computational Biology and Biomedicine</em>,<span class="cit-publ-name"> Orlando, FL</span>, pp. <span class="cit-fpage">105</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.7" data-doi="10.1534/g3.114.013748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative trait locus mapping methods for diversity outbred mice.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1623%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.013748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.114.013748&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.8" data-doi="10.1534/g3.117.041434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gralinski</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menachery</span>  <span class="cit-name-given-names">V. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Totura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beall</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Allelic variation in mouse Ticam2 contributes to SARS-CoV pathogenesis.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1653</span>–<span class="cit-lpage">1663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DGralinski%26rft.auinit1%253DL.%2BE.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D1653%26rft.epage%253D1663%26rft.atitle%253DAllelic%2BVariation%2Bin%2Bthe%2BToll-Like%2BReceptor%2BAdaptor%2BProtein%2BTicam2%2BContributes%2Bto%2BSARS-Coronavirus%2BPathogenesis%2Bin%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.041434%26rft_id%253Dinfo%253Apmid%252F28592648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNi8xNjUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.9" data-doi="10.1534/g3.117.041624"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekine</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendrick</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Oas1b-dependent immune transcriptional profiles of West Nile virus infection in the collaborative cross.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1665</span>–<span class="cit-lpage">1682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DGreen%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D1665%26rft.epage%253D1682%26rft.atitle%253DOas1b-dependent%2BImmune%2BTranscriptional%2BProfiles%2Bof%2BWest%2BNile%2BVirus%2BInfection%2Bin%2Bthe%2BCollaborative%2BCross%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.041624%26rft_id%253Dinfo%253Apmid%252F28592649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNi8xNjY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.10" data-doi="10.1371/journal.pgen.1000279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Practical issues in imputation-based association mapping.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGuan%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De1000279%26rft.epage%253De1000279%26rft.atitle%253DPractical%2Bissues%2Bin%2Bimputation-based%2Bassociation%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000279%26rft_id%253Dinfo%253Apmid%252F19057666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19057666&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawley</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">K. K. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">HAPLO - a program using the EM algorithm to estimate the frequencies of multisite haplotypes.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DHawley%26rft.auinit1%253DM.%2BE.%26rft.volume%253D86%26rft.issue%253D5%26rft.spage%253D409%26rft.epage%253D411%26rft.atitle%253DHAPLO%253A%2BA%2BProgram%2BUsing%2Bthe%2BEM%2BAlgorithm%2Bto%2BEstimate%2Bthe%2BFrequencies%2Bof%2BMulti-site%2BHaplotypes%26rft_id%253Dinfo%253Apmid%252F7560877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo2OiJqaGVyZWQiO3M6NToicmVzaWQiO3M6ODoiODYvNS80MDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.12" data-doi="10.1186/1297-9686-44-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinghorn</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A phasing and imputation method for pedigreed populations that results in a single-stage genomic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHickey%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D44%26rft.spage%253D9%26rft.epage%253D9%26rft.atitle%253DA%2Bphasing%2Band%2Bimputation%2Bmethod%2Bfor%2Bpedigreed%2Bpopulations%2Bthat%2Bresults%2Bin%2Ba%2Bsingle-stage%2Bgenomic%2Bevaluation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-9%26rft_id%253Dinfo%253Apmid%252F22462519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22462519&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.13" data-doi="10.1371/journal.pgen.1000529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%2BN.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000529%26rft.epage%253De1000529%26rft.atitle%253DA%2Bflexible%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bmethod%2Bfor%2Bthe%2Bnext%2Bgeneration%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000529%26rft_id%253Dinfo%253Apmid%252F19543373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543373&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.6.1623.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Transforming genomes using MOD files with applications. <em>Proceedings of the International Conference on Bioinformatics</em>, <em>Computational Biology and Biomedical Informatics</em>, <span class="cit-publ-name">Washington DC, </span>p. <span class="cit-fpage">595</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.15" data-doi="10.1038/nature10413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mouse genomic variation and its effect on phenotypes and gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKeane%26rft.auinit1%253DT.%2BM.%26rft.volume%253D477%26rft.issue%253D7364%26rft.spage%253D289%26rft.epage%253D294%26rft.atitle%253DMouse%2Bgenomic%2Bvariation%2Band%2Bits%2Beffect%2Bon%2Bphenotypes%2Band%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10413%26rft_id%253Dinfo%253Apmid%252F21921910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921910&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294852400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.16" data-doi="10.1534/genetics.112.138537"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Properties and power of the Drosophila synthetic population resource for the routine dissection of complex traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.138537%26rft_id%253Dinfo%253Apmid%252F22505626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvOTM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.17" data-doi="10.1093/biostatistics/kxq039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kutalik</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bochud</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mooser</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vollenweider</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Methods for testing association between uncertain genotypes and quantitative traits.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxq039%26rft_id%253Dinfo%253Apmid%252F20543033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvc3RzIjtzOjU6InJlc2lkIjtzOjY6IjEyLzEvMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.18" data-doi="10.1146/annurev.genom.9.081307.164242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanna</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genotype imputation.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D10%26rft.spage%253D387%26rft.epage%253D406%26rft.atitle%253DGenotype%2Bimputation.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.9.081307.164242%26rft_id%253Dinfo%253Apmid%252F19715440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.9.081307.164242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19715440&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270313200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.19" data-doi="10.1002/gepi.20533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willer</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D34%26rft.issue%253D8%26rft.spage%253D816%26rft.epage%253D834%26rft.atitle%253DMaCH%253A%2Busing%2Bsequence%2Band%2Bgenotype%2Bdata%2Bto%2Bestimate%2Bhaplotypes%2Band%2Bunobserved%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20533%26rft_id%253Dinfo%253Apmid%252F21058334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21058334&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284719100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.20" data-doi="10.1093/bioinformatics/btq187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel de Villena</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Efficient genome ancestry inference in complex pedigrees with inbreeding.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">i199</span>–<span class="cit-lpage">i207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq187%26rft_id%253Dinfo%253Apmid%252F20529906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTIvaTE5OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.21" data-doi="10.1038/nrg2796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype imputation for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D511%26rft.atitle%253DGenotype%2Bimputation%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2796%26rft_id%253Dinfo%253Apmid%252F20517342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20517342&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.22" data-doi="10.1038/ng2088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A new multipoint method for genome-wide association studies by imputation of genotypes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">906</span>–<span class="cit-lpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMarchini%26rft.auinit1%253DJ.%26rft.volume%253D39%26rft.issue%253D7%26rft.spage%253D906%26rft.epage%253D913%26rft.atitle%253DA%2Bnew%2Bmultipoint%2Bmethod%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bby%2Bimputation%2Bof%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2088%26rft_id%253Dinfo%253Apmid%252F17572673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17572673&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247619800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.23" data-doi="10.1093/bioinformatics/btq330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rios</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flicek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Deriving the consequences of genomic variants with the Ensembl API and SNP effect predictor.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2069</span>–<span class="cit-lpage">2070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq330%26rft_id%253Dinfo%253Apmid%252F20562413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTYvMjA2OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjIzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.24" data-doi="10.1007/s00335-015-9581-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Informatics resources for the collaborative cross and related mouse populations.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D26%26rft.spage%253D521%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-015-9581-z%26rft_id%253Dinfo%253Apmid%252F26135136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-015-9581-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26135136&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">C.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The mouse universal genotyping array: from substrains to subspecies.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.26" data-doi="10.1534/genetics.116.197988"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galante</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pack</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Structural variation shapes the landscape of recombination in mouse.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">603</span>–<span class="cit-lpage">619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMorgan%26rft.auinit1%253DA.%2BP.%26rft.volume%253D206%26rft.issue%253D2%26rft.spage%253D603%26rft.epage%253D619%26rft.atitle%253DStructural%2BVariation%2BShapes%2Bthe%2BLandscape%2Bof%2BRecombination%2Bin%2BMouse%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.197988%26rft_id%253Dinfo%253Apmid%252F28592499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNjAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mosedale</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roth</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiltshire</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Candidate risk factors and mechanisms for Tolvaptan-induced liver injury are identified using a collaborative cross approach.</span> <abbr class="cit-jnl-abbrev">Toxicol. Sci</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.28" data-doi="10.1073/pnas.230304397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turri</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A method for fine mapping quantitative trait loci in outbred animal stocks.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">12649</span>–<span class="cit-lpage">12654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.230304397%26rft_id%253Dinfo%253Apmid%252F11050180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjMvMTI2NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-web" id="cit-7.6.1623.29"><div class="cit-metadata unstructured">Oreper, D., Y. Cai, L. M. Tarantino, F. Pardo-Manuel de Villena, and W. Valdar, 2017a Github repository. Available at: <a href="https://github.com/danoreper/ISVdb.git">https://github.com/danoreper/ISVdb.git</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-web" id="cit-7.6.1623.30"><div class="cit-metadata unstructured">Oreper, D., Y. Cai, L. M. Tarantino, F. Pardo-Manuel de Villena, and W. Valdar, 2017b Zenodo repository. Available at: <a href="https://doi.org/10.5281/zenodo.399474">https://doi.org/10.5281/zenodo.399474</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.31" data-doi="10.1038/gene.2013.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillippi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Using the emerging collaborative cross to probe the immune system.</span> <abbr class="cit-jnl-abbrev">Genes Immun.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BImmun.%26rft.volume%253D15%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fgene.2013.59%26rft_id%253Dinfo%253Apmid%252F24195963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/gene.2013.59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24195963&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1623.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.33" data-doi="10.1093/nar/gkt069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramasamy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trabzuni</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dillman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Resolving the polymorphism-in-probe problem is critical for correct interpretation of expression QTL studies.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">e88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt069%26rft_id%253Dinfo%253Apmid%252F23435227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjQxLzcvZTg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.34" data-doi="10.1126/science.1259595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okumura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldmann</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Host genetic diversity enables Ebola hemorrhagic fever pathogenesis and resistance.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>: <span class="cit-fpage">987</span>–<span class="cit-lpage">991</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259595%26rft_id%253Dinfo%253Apmid%252F25359852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDYvNjIxMi85ODciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sargolzaei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesnais</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schenkel</span>  <span class="cit-name-given-names">F. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A new approach for efficient genotype imputation using information from relatives.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.36" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.37" data-doi="10.1534/genetics.116.199596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shorter</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Odet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia-Yu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Male infertility is responsible for nearly half of the extinction observed in the collaborative cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DShorter%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D206%26rft.issue%253D2%26rft.spage%253D557%26rft.epage%253D572%26rft.atitle%253DMale%2BInfertility%2BIs%2BResponsible%2Bfor%2BNearly%2BHalf%2Bof%2Bthe%2BExtinction%2BObserved%2Bin%2Bthe%2BMouse%2BCollaborative%2BCross%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.199596%26rft_id%253Dinfo%253Apmid%252F28592496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.38" data-doi="10.1534/genetics.116.198838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Najarian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarsani</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigmon</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The genomes of the collaborative cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSrivastava%26rft.auinit1%253DA.%26rft.volume%253D206%26rft.issue%253D2%26rft.spage%253D537%26rft.epage%253D556%26rft.atitle%253DGenomes%2Bof%2Bthe%2BMouse%2BCollaborative%2BCross%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.198838%26rft_id%253Dinfo%253Apmid%252F28592495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.39" data-doi="10.1038/ng.746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study of leaf architecture in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTian%26rft.auinit1%253DF.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D162%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bleaf%2Barchitecture%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.746%26rft_id%253Dinfo%253Apmid%252F21217756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217756&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.40" data-doi="10.1186/1297-9686-43-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connell</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic evaluations with many more genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DVanraden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D43%26rft.spage%253D10%26rft.epage%253D10%26rft.atitle%253DGenomic%2Bevaluations%2Bwith%2Bmany%2Bmore%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-10%26rft_id%253Dinfo%253Apmid%252F21366914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21366914&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.41" data-doi="10.1007/s00122-014-2337-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole-genome QTL analysis for MAGIC.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1753</span>–<span class="cit-lpage">1770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1753%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2337-4%26rft_id%253Dinfo%253Apmid%252F24927820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2337-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24927820&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.42" data-doi="10.1007/s00335-012-9410-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manly</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Status and access to the collaborative cross population.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">706</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DWelsh%26rft.auinit1%253DC.%2BE.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D706%26rft.epage%253D712%26rft.atitle%253DStatus%2Band%2Baccess%2Bto%2Bthe%2BCollaborative%2BCross%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9410-6%26rft_id%253Dinfo%253Apmid%252F22847377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9410-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22847377&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.43" data-doi="10.1534/genetics.104.028902"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Using progenitor strain information to identify quantitative trait nucleotides in outbred mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.028902%26rft_id%253Dinfo%253Apmid%252F16085706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcxLzIvNjczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.44" data-doi="10.1534/genetics.114.166249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Bayesian modeling of haplotype effects in multiparent populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D139%26rft.epage%253D156%26rft.atitle%253DBayesian%2BModeling%2Bof%2BHaplotype%2BEffects%2Bin%2BMultiparent%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.166249%26rft_id%253Dinfo%253Apmid%252F25236455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMTM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2MjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.45" data-doi="10.1534/genetics.115.177873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reconstruction of genome ancestry blocks in multiparental populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177873%26rft_id%253Dinfo%253Apmid%252F26048018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMC80LzEwNzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTYyMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1623.46" data-doi="10.1002/gepi.20552"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A comparison of approaches to account for uncertainty in analysis of imputed genotypes.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">102</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DZheng%26rft.auinit1%253DJ.%26rft.volume%253D35%26rft.issue%253D2%26rft.spage%253D102%26rft.epage%253D110%26rft.atitle%253DA%2Bcomparison%2Bof%2Bapproaches%2Bto%2Baccount%2Bfor%2Buncertainty%2Bin%2Banalysis%2Bof%2Bimputed%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20552%26rft_id%253Dinfo%253Apmid%252F21254217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20552&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21254217&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1623.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/6/1623.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1619" title="Epistasis: Searching for Interacting Genetic Variants Using Crosses" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1631" title="Loci Contributing to Boric Acid Toxicity in Two Reference Populations of Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/6"><span>Volume 7 Issue 6, June 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/6.cover-source.jpg?itok=5_J-OKHc" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1623" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rARcUMQ1kIS9SKzpSzHXgb2N72943JcwDz-xh8UrOCU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/6/1623" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/6/1623" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/6/1623" class="active" data-icon-position="" data-hide-link-title="0">Inbred Strain Variant Database (ISVdb): A Repository for Probabilistically Informed Sequence Differences Among the Collaborative Cross Strains and Their Founders</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484767" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-mI77kU3AtJ8Cu1hZNGkNLWILQkv7aH3DUA16zt82gRc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit--2" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1623.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/6/1623" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--5" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-gZITW-mv0NPRFXr7AYmVdWvTIpefW0Mxl7B8eRflv50" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/6/1623" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1623.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484767" id="node484767--2" data-pisa="ggg;7/6/1623" data-pisa-master="ggg;g3.117.041491" data-apath="/ggg/7/6/1623.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inbred Strain Variant Database (ISVdb): A Repository for Probabilistically Informed Sequence Differences Among the Collaborative Cross Strains and Their Founders</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4569-5160" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel Oreper</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3950-3794" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yanwei Cai</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9984-2600" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lisa M. Tarantino</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fernando Pardo-Manuel de Villena</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1623-1630;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041491 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484767--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Oreper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BOreper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oreper%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BOreper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4569-5160" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel Oreper</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yanwei Cai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanwei%2BCai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cai%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYanwei%2BCai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3950-3794" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yanwei Cai</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lisa M. Tarantino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Division of Pharmacotherapy and Experimental Therapeutics, Eshelman School of Pharmacy University of North Carolina, Chapel Hill, North Carolina 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BM.%2BTarantino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tarantino%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALisa%2BM.%2BTarantino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9984-2600" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lisa M. Tarantino</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Villena%20FP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fernando Pardo-Manuel de Villena</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/6/1623" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/6/1623&amp;title=Inbred%20Strain%20Variant%20Database%20%28ISVdb%29%3A%20A%20Repository%20for%20Probabilistically%20Informed%20Sequence%20Differences%20Among%20the%20Collaborative%20Cross%20Strains%20and%20Their%20Founders" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/6/1623&amp;title=Inbred%20Strain%20Variant%20Database%20%28ISVdb%29%3A%20A%20Repository%20for%20Probabilistically%20Informed%20Sequence%20Differences%20Among%20the%20Collaborative%20Cross%20Strains%20and%20Their%20Founders" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/6/1623&amp;title=Inbred%20Strain%20Variant%20Database%20%28ISVdb%29%3A%20A%20Repository%20for%20Probabilistically%20Informed%20Sequence%20Differences%20Among%20the%20Collaborative%20Cross%20Strains%20and%20Their%20Founders" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/6/1623&amp;text=Inbred%20Strain%20Variant%20Database%20%28ISVdb%29%3A%20A%20Repository%20for%20Probabilistically%20Informed%20Sequence%20Differences%20Among%20the%20Collaborative%20Cross%20Strains%20and%20Their%20Founders" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/6/1623&amp;title=Inbred%20Strain%20Variant%20Database%20%28ISVdb%29%3A%20A%20Repository%20for%20Probabilistically%20Informed%20Sequence%20Differences%20Among%20the%20Collaborative%20Cross%20Strains%20and%20Their%20Founders" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/6/1623&amp;t=Inbred%20Strain%20Variant%20Database%20%28ISVdb%29%3A%20A%20Repository%20for%20Probabilistically%20Informed%20Sequence%20Differences%20Among%20the%20Collaborative%20Cross%20Strains%20and%20Their%20Founders" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/6/1623&amp;title=Inbred%20Strain%20Variant%20Database%20%28ISVdb%29%3A%20A%20Repository%20for%20Probabilistically%20Informed%20Sequence%20Differences%20Among%20the%20Collaborative%20Cross%20Strains%20and%20Their%20Founders" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/6/1623&amp;title=Inbred%20Strain%20Variant%20Database%20%28ISVdb%29%3A%20A%20Repository%20for%20Probabilistically%20Informed%20Sequence%20Differences%20Among%20the%20Collaborative%20Cross%20Strains%20and%20Their%20Founders" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484767/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484767" id="node484767--3" data-pisa="ggg;7/6/1623" data-pisa-master="ggg;g3.117.041491" data-apath="/ggg/7/6/1623.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Inbred Strain Variant Database (ISVdb): A Repository for Probabilistically Informed Sequence Differences Among the Collaborative Cross Strains and Their Founders</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-4569-5160" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel Oreper</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3950-3794" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yanwei Cai</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9984-2600" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lisa M. Tarantino</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fernando Pardo-Manuel de Villena</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>William Valdar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1623-1630;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041491 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484767--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Oreper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BOreper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oreper%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BOreper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4569-5160" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel Oreper</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yanwei Cai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Curriculum in Bioinformatics and Computational Biology, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanwei%2BCai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cai%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYanwei%2BCai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3950-3794" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yanwei Cai</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lisa M. Tarantino </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Division of Pharmacotherapy and Experimental Therapeutics, Eshelman School of Pharmacy University of North Carolina, Chapel Hill, North Carolina 27599-7265</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BM.%2BTarantino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tarantino%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALisa%2BM.%2BTarantino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9984-2600" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lisa M. Tarantino</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Villena%20FP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5738-5795" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fernando Pardo-Manuel de Villena</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">William Valdar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, University of North Carolina, Chapel Hill, North Carolina 27599-7265</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Lineberger Comprehensive Cancer Center,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BValdar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valdar%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWilliam%2BValdar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2419-0430" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for William Valdar</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:william.valdar@unc.edu" class="" data-icon-position="" data-hide-link-title="0">william.valdar@unc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1623&amp;rft.epage=1630&amp;rft.atitle=Inbred%20Strain%20Variant%20Database%20%28ISVdb%29%3A%20A%20Repository%20for%20Probabilistically%20Informed%20Sequence%20Differences%20Among%20the%20Collaborative%20Cross%20Strains%20and%20Their%20Founders&amp;rft.volume=7&amp;rft.issue=6&amp;rft.date=2017-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Oreper%2C+Daniel&amp;rft.au=Cai%2C+Yanwei&amp;rft.au=Tarantino%2C+Lisa+M.&amp;rft.au=de+Villena%2C+Fernando+Pardo-Manuel&amp;rft.au=Valdar%2C+William"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484767/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484767/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484767/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484767/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484767/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484767/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484767/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484767/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484767/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484767/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484767/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484767/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/6/1623" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/6/1623.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/6/1623.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/6/1623.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/6/1623.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
