<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/6/1693" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/6/1693.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/6/1693.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/6/1693.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/6/1693/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/6/1693.atom" />
<meta name="HW.pisa" content="ggg;7/6/1693" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Identification of Nitrogen Consumption Genetic Variants in Yeast Through QTL Mapping and Bulk Segregant RNA-Seq Analyses" />
<meta name="DC.Identifier" content="10.1534/g3.117.042127" />
<meta name="DC.Date" content="2017-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Cubillos et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Saccharomyces cerevisiae is responsible for wine must fermentation. In this process, nitrogen represents a limiting nutrient and its scarcity results in important economic losses for the wine industry. Yeast isolates use different strategies to grow in poor nitrogen environments and their genomic plasticity enables adaptation to multiple habitats through improvements in nitrogen consumption. Here, we used a highly recombinant S. cerevisiae multi-parent population (SGRP-4X) derived from the intercross of four parental strains of different origins to identify new genetic variants responsible for nitrogen consumption differences during wine fermentation. Analysis of 165 fully sequenced F12 segregants allowed us to map 26 QTL in narrow intervals for 14 amino acid sources and ammonium, the majority of which represent genomic regions previously unmapped for these traits. To complement this strategy, we performed Bulk segregant RNA-seq (BSR-seq) analysis in segregants exhibiting extremely high and low ammonium consumption levels. This identified several QTL overlapping differentially expressed genes and refined the gene candidate search. Based on these approaches, we were able to validate [ARO1][1] , [PDC1][2] , [CPS1][3] , [ASI2][4] , [LYP1][5] , and [ALP1][6] allelic variants underlying nitrogen consumption differences between strains, providing evidence of many genes with small phenotypic effects. Altogether, these variants significantly shape yeast nitrogen consumption with important implications for evolution, ecological, and quantitative genomics.

 [1]: http://www.yeastgenome.org/locus/S000002534/overview
 [2]: http://www.yeastgenome.org/locus/S000004034/overview
 [3]: http://www.yeastgenome.org/locus/S000003708/overview
 [4]: http://www.yeastgenome.org/locus/S000005103/overview
 [5]: http://www.yeastgenome.org/locus/S000005212/overview
 [6]: http://www.yeastgenome.org/locus/S000005214/overview" />
<meta name="DC.Contributor" content="Francisco A. Cubillos" />
<meta name="DC.Contributor" content="Claire Brice" />
<meta name="DC.Contributor" content="Jennifer Molinet" />
<meta name="DC.Contributor" content="Sebastién Tisné" />
<meta name="DC.Contributor" content="Valentina Abarca" />
<meta name="DC.Contributor" content="Sebastián M. Tapia" />
<meta name="DC.Contributor" content="Christian Oporto" />
<meta name="DC.Contributor" content="Verónica García" />
<meta name="DC.Contributor" content="Gianni Liti" />
<meta name="DC.Contributor" content="Claudio Martínez" />
<meta name="article:published_time" content="2017-06-01" />
<meta name="article:section" content="Multiparental Population" />
<meta name="citation_title" content="Identification of Nitrogen Consumption Genetic Variants in Yeast Through QTL Mapping and Bulk Segregant RNA-Seq Analyses" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; is responsible for wine must fermentation. In this process, nitrogen represents a limiting nutrient and its scarcity results in important economic losses for the wine industry. Yeast isolates use different strategies to grow in poor nitrogen environments and their genomic plasticity enables adaptation to multiple habitats through improvements in nitrogen consumption. Here, we used a highly recombinant &lt;i&gt;S. cerevisiae&lt;/i&gt; multi-parent population (SGRP-4X) derived from the intercross of four parental strains of different origins to identify new genetic variants responsible for nitrogen consumption differences during wine fermentation. Analysis of 165 fully sequenced F12 segregants allowed us to map 26 QTL in narrow intervals for 14 amino acid sources and ammonium, the majority of which represent genomic regions previously unmapped for these traits. To complement this strategy, we performed Bulk segregant RNA-seq (BSR-seq) analysis in segregants exhibiting extremely high and low ammonium consumption levels. This identified several QTL overlapping differentially expressed genes and refined the gene candidate search. Based on these approaches, we were able to validate &lt;i&gt;ARO1&lt;/i&gt;, &lt;i&gt;PDC1&lt;/i&gt;, &lt;i&gt;CPS1&lt;/i&gt;, &lt;i&gt;ASI2&lt;/i&gt;, &lt;i&gt;LYP1&lt;/i&gt;, and &lt;i&gt;ALP1&lt;/i&gt; allelic variants underlying nitrogen consumption differences between strains, providing evidence of many genes with small phenotypic effects. Altogether, these variants significantly shape yeast nitrogen consumption with important implications for evolution, ecological, and quantitative genomics.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/06/01" />
<meta name="citation_mjid" content="ggg;7/6/1693" />
<meta name="citation_id" content="7/6/1693" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/6/1693" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/6/1693.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/6/1693.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/6/1693.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.042127" />
<meta name="citation_pmid" content="28592651" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Population" />
<meta name="citation_firstpage" content="1693" />
<meta name="citation_lastpage" content="1705" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Francisco A. Cubillos" />
<meta name="citation_author_institution" content="Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile" />
<meta name="citation_author_institution" content="Millennium Nucleus for Fungal Integrative and Synthetic Biology (MN-FISB), 9170201 Santiago, Chile" />
<meta name="citation_author" content="Claire Brice" />
<meta name="citation_author_institution" content="Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile" />
<meta name="citation_author_institution" content="Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile" />
<meta name="citation_author" content="Jennifer Molinet" />
<meta name="citation_author_institution" content="Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile" />
<meta name="citation_author_institution" content="Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile" />
<meta name="citation_author" content="Sebastién Tisné" />
<meta name="citation_author_institution" content="Centre de coopération internationale en recherche agronomique pour le développement (CIRAD), UMR AGAP (Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit), Campus International de Baillarguet, 34398 Montpellier, France" />
<meta name="citation_author" content="Valentina Abarca" />
<meta name="citation_author_institution" content="Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile" />
<meta name="citation_author" content="Sebastián M. Tapia" />
<meta name="citation_author_institution" content="Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile" />
<meta name="citation_author_institution" content="Millennium Nucleus for Fungal Integrative and Synthetic Biology (MN-FISB), 9170201 Santiago, Chile" />
<meta name="citation_author" content="Christian Oporto" />
<meta name="citation_author_institution" content="Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile" />
<meta name="citation_author_institution" content="Millennium Nucleus for Fungal Integrative and Synthetic Biology (MN-FISB), 9170201 Santiago, Chile" />
<meta name="citation_author" content="Verónica García" />
<meta name="citation_author_institution" content="Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile" />
<meta name="citation_author_institution" content="Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile" />
<meta name="citation_author" content="Gianni Liti" />
<meta name="citation_author_institution" content="Institute for Research on Cancer and Ageing of Nice (IRCAN), Centre national de la recherche scientifique (CNRS) unités mixtes de recherche (UMR) 7284, Institut national de la santé et de la recherche médicale (INSERM) U1081, University of Nice Sophia Antipolis, 06107, France" />
<meta name="citation_author" content="Claudio Martínez" />
<meta name="citation_author_institution" content="Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile" />
<meta name="citation_author_institution" content="Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile" />
<meta name="citation_author_email" content="claudio.martinez@usach.cl" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Anders;citation_author=W. Huber;citation_title=Differential expression analysis for sequence count data.;citation_pages=R106-R106;citation_volume=11;citation_year=2010;citation_issue=10;citation_pmid=20979621;citation_doi=10.1186/gb-2010-11-10-r106" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Anders;citation_author=P. T. Pyl;citation_author=W. Huber;citation_title=HTSeq–a Python framework to work with high-throughput sequencing data.;citation_volume=31;citation_year=2015;citation_issue=2;citation_pmid=25260700;citation_doi=10.1093/bioinformatics/btu638" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=F. N. Arroyo-Lopez;citation_author=Z. Salvado;citation_author=J. Tronchoni;citation_author=J. M. Guillamon;citation_author=E. Barrio;citation_title=Susceptibility and resistance to ethanol in Saccharomyces strains isolated from wild and fermentative environments.;citation_pages=1005-1015;citation_volume=27;citation_year=2010;citation_issue=12;citation_pmid=20824889;citation_doi=10.1002/yea.1809" />
<meta name="citation_reference" content="citation_journal_title=Systematic and Applied Microbiology;citation_journal_abbrev=Systematic and Applied Microbiology;citation_author=G. Beltran;citation_author=M. J. Torija;citation_author=M. Novo;citation_author=N. Ferrer;citation_author=M. Poblet;citation_title=Analysis of yeast populations during alcoholic fermentation: a six year follow-up study.;citation_pages=287-293;citation_volume=25;citation_year=2002;citation_issue=2;citation_pmid=12353885;citation_doi=10.1078/0723-2020-00097" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. Ben-Ari;citation_author=D. Zenvirth;citation_author=A. Sherman;citation_author=L. David;citation_author=M. Klutstein;citation_title=Four linked genes participate in controlling sporulation efficiency in budding yeast.;citation_pages=e195-e195;citation_volume=2;citation_year=2006;citation_issue=11;citation_pmid=17112318;citation_doi=10.1371/journal.pgen.0020195" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=85;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Bergstrom;citation_author=J. T. Simpson;citation_author=F. Salinas;citation_author=B. Barre;citation_author=L. Parts;citation_title=A high-definition view of functional genetic variation from natural yeast genomes.;citation_volume=31;citation_year=2014;citation_issue=4;citation_pmid=24425782;citation_doi=10.1093/molbev/msu037" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=R. Bernardo;citation_title=Genomewide markers as cofactors for precision mapping of quantitative trait loci.;citation_pages=999-1009;citation_volume=126;citation_year=2013;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. S. Bloom;citation_author=I. Kotenko;citation_author=M. J. Sadhu;citation_author=S. Treusch;citation_author=F. W. Albert;citation_title=Genetic interactions contribute less than additive effects to quantitative trait variation in yeast.;citation_pages=8712;citation_volume=6;citation_year=2015;citation_pmid=26537231;citation_doi=10.1038/ncomms9712" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=M. A. Bogue;citation_author=G. A. Churchill;citation_author=E. J. Chesler;citation_title=Collaborative cross and diversity outbred data resources in the mouse phenome database.;citation_pages=511-520;citation_volume=26;citation_year=2015;citation_issue=9–10;citation_pmid=26286858;citation_doi=10.1007/s00335-015-9595-6" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=J. Bordallo;citation_author=P. Suarez-Rendueles;citation_title=Control of Saccharomyces cerevisiae carboxypeptidase S (CPS1) gene expression under nutrient limitation.;citation_pages=339-349;citation_volume=9;citation_year=1993;citation_issue=4;citation_pmid=8511964;citation_doi=10.1002/yea.320090404" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. J. Brauer;citation_author=C. M. Christianson;citation_author=D. A. Pai;citation_author=M. J. Dunham;citation_title=Mapping novel traits by array-assisted bulk segregant analysis in Saccharomyces cerevisiae.;citation_volume=173;citation_year=2006;citation_issue=3;citation_pmid=16624899;citation_doi=10.1534/genetics.106.057927" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. S. Breunig;citation_author=S. R. Hackett;citation_author=J. D. Rabinowitz;citation_author=L. Kruglyak;citation_title=Genetic basis of metabolome variation in yeast.;citation_pages=e1004142;citation_volume=10;citation_year=2014;citation_issue=3;citation_pmid=24603560;citation_doi=10.1371/journal.pgen.1004142" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Brice;citation_author=I. Sanchez;citation_author=F. Bigey;citation_author=J. L. Legras;citation_author=B. Blondin;citation_title=A genetic approach of wine yeast fermentation capacity in nitrogen-starvation reveals the key role of nitrogen signaling.;citation_pages=495;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=C. Brice;citation_author=I. Sanchez;citation_author=C. Tesniere;citation_author=B. Blondin;citation_title=Assessing the mechanisms responsible for differences between nitrogen requirements of saccharomyces cerevisiae wine yeasts in alcoholic fermentation.;citation_volume=80;citation_year=2014;citation_issue=4;citation_pmid=24334661;citation_doi=10.1128/AEM.03856-13" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. K. Burke;citation_author=G. Liti;citation_author=A. D. Long;citation_title=Standing genetic variation drives repeatable experimental evolution in outcrossing populations of Saccharomyces cerevisiae.;citation_volume=31;citation_year=2014;citation_issue=12;citation_pmid=25172959;citation_doi=10.1093/molbev/msu256" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=L. Crepin;citation_author=T. Nidelet;citation_author=I. Sanchez;citation_author=S. Dequin;citation_author=C. Camarasa;citation_title=Sequential use of nitrogen compounds by Saccharomyces cerevisiae during wine fermentation: a model based on kinetic and regulation characteristics of nitrogen permeases.;citation_volume=78;citation_year=2012;citation_issue=22;citation_pmid=22983966;citation_doi=10.1128/AEM.02294-12" />
<meta name="citation_reference" content="citation_journal_title=Curr. Genet.;citation_author=F. A. Cubillos;citation_title=Exploiting budding yeast natural variation for industrial processes.;citation_pages=745-751;citation_volume=62;citation_year=2016;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=F. A. Cubillos;citation_author=E. J. Louis;citation_author=G. Liti;citation_title=Generation of a large set of genetically tractable haploid and diploid Saccharomyces strains.;citation_volume=9;citation_year=2009;citation_issue=8;citation_pmid=19840116;citation_doi=10.1111/j.1567-1364.2009.00583.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=F. A. Cubillos;citation_author=E. Billi;citation_author=E. Zorgo;citation_author=L. Parts;citation_author=P. Fargier;citation_title=Assessing the complex architecture of polygenic traits in diverged yeast populations.;citation_pages=1401-1413;citation_volume=20;citation_year=2011;citation_issue=7;citation_pmid=21261765;citation_doi=10.1111/j.1365-294X.2011.05005.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. A. Cubillos;citation_author=L. Parts;citation_author=F. Salinas;citation_author=A. Bergstrom;citation_author=E. Scovacricchi;citation_title=High-resolution mapping of complex traits with a four-parent advanced intercross yeast population.;citation_volume=195;citation_year=2013;citation_issue=3;citation_pmid=24037264;citation_doi=10.1534/genetics.113.155515" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=L. J. de Assis;citation_author=R. B. Zingali;citation_author=C. A. Masuda;citation_author=S. P. Rodrigues;citation_author=M. Montero-Lomeli;citation_title=Pyruvate decarboxylase activity is regulated by the Ser/Thr protein phosphatase Sit4p in the yeast Saccharomyces cerevisiae.;citation_volume=13;citation_year=2013;citation_issue=6;citation_pmid=23692511;citation_doi=10.1111/1567-1364.12052" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=J. R. Dickinson;citation_author=L. E. Salgado;citation_author=M. J. Hewlins;citation_title=The catabolism of amino acids to long chain and complex alcohols in Saccharomyces cerevisiae.;citation_volume=278;citation_year=2003;citation_issue=10;citation_pmid=12499363;citation_doi=10.1074/jbc.M211914200" />
<meta name="citation_reference" content="citation_journal_title=Biochemical Journal;citation_journal_abbrev=Biochem. J.;citation_author=K. Duncan;citation_author=R. M. Edwards;citation_author=J. R. Coggins;citation_title=The pentafunctional arom enzyme of Saccharomyces cerevisiae is a mosaic of monofunctional domains;citation_pages=375-386;citation_volume=246;citation_year=1987;citation_issue=2;citation_pmid=2825635;citation_doi=10.1042/bj2460375" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Durrant;citation_author=H. Tayem;citation_author=B. Yalcin;citation_author=J. Cleak;citation_author=L. Goodstadt;citation_title=Collaborative cross mice and their power to map host susceptibility to Aspergillus fumigatus infection.;citation_volume=21;citation_year=2011;citation_issue=8;citation_pmid=21493779;citation_doi=10.1101/gr.118786.110" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Symp. Quant. Biol.;citation_author=I. M. Ehrenreich;citation_author=J. P. Gerke;citation_author=L. Kruglyak;citation_title=Genetic dissection of complex traits in yeast: insights from studies of gene expression and other phenotypes in the BYxRM cross.;citation_volume=74;citation_year=2009;citation_pmid=19734204;citation_doi=10.1101/sqb.2009.74.013" />
<meta name="citation_reference" content="citation_journal_title=International journal of food microbiology;citation_journal_abbrev=Int J Food Microbiol;citation_author=G. H. Fleet;citation_title=Yeast interactions and wine flavour.;citation_pages=11-22;citation_volume=86;citation_year=2003;citation_issue=1–2;citation_pmid=12892919;citation_doi=10.1016/S0168-1605(03)00245-9" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=G. H. Fleet;citation_title=Wine yeasts for the future.;citation_volume=8;citation_year=2008;citation_issue=7;citation_pmid=18793201;citation_doi=10.1111/j.1567-1364.2008.00427.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Forsberg;citation_author=M. Hammar;citation_author=C. Andreasson;citation_author=A. Moliner;citation_author=P. O. Ljungdahl;citation_title=Suppressors of ssy1 and ptr3 Null Mutations Define Novel Amino Acid Sensor-Independent Genes in Saccharomyces cerevisiae;citation_pages=973-988;citation_volume=158;citation_year=2001;citation_issue=3;citation_pmid=11454748" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=A. P. Gasch;citation_author=P. T. Spellman;citation_author=C. M. Kao;citation_author=O. Carmel-Harel;citation_author=M. B. Eisen;citation_title=Genomic Expression Programs in the Response of Yeast Cells to Environmental Changes;citation_pages=4241-4257;citation_volume=11;citation_year=2000;citation_issue=12;citation_pmid=11102521;citation_doi=10.1091/mbc.11.12.4241" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=P. Godard;citation_author=A. Urrestarazu;citation_author=S. Vissers;citation_author=K. Kontos;citation_author=G. Bontempi;citation_title=Effect of 21 different nitrogen sources on global gene expression in the yeast Saccharomyces cerevisiae.;citation_volume=27;citation_year=2007;citation_issue=8;citation_pmid=17308034;citation_doi=10.1128/MCB.01084-06" />
<meta name="citation_reference" content="citation_journal_title=Journal of Agricultural and Food Chemistry;citation_journal_abbrev=Journal of Agricultural and Food Chemistry;citation_author=S. Gomez-Alonso;citation_author=I. Hermosin-Gutierrez;citation_author=E. Garcia-Romero;citation_title=Simultaneous HPLC analysis of biogenic amines, amino acids, and ammonium ion as aminoenone derivatives in wine and beer samples.;citation_pages=608-613;citation_volume=55;citation_year=2007;citation_issue=3;citation_pmid=17263449;citation_doi=10.1021/jf062820m" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. Gutierrez;citation_author=G. Beltran;citation_author=J. Warringer;citation_author=J. M. Guillamon;citation_title=Genetic basis of variations in nitrogen source utilization in four wine commercial yeast strains.;citation_pages=e67166;citation_volume=8;citation_year=2013;citation_issue=6;citation_pmid=23826223;citation_doi=10.1371/journal.pone.0067166" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=J. Gutin;citation_author=A. Sadeh;citation_author=A. Rahat;citation_author=A. Aharoni;citation_author=N. Friedman;citation_title=Condition-specific genetic interaction maps reveal crosstalk between the cAMP/PKA and the HOG MAPK pathways in the activation of the general stress response;citation_pages=829-829;citation_volume=11;citation_year=2015;citation_issue=10;citation_pmid=26446933;citation_doi=10.15252/msb.20156451" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. G. Hinnebusch;citation_author=G. R. Fink;citation_title=Positive regulation in the general amino acid control of Saccharomyces cerevisiae.;citation_pages=5374-5378;citation_volume=80;citation_year=1983;citation_issue=17;citation_pmid=6351059;citation_doi=10.1073/pnas.80.17.5374" />
<meta name="citation_reference" content="citation_journal_title=Molecular biotechnology;citation_journal_abbrev=Mol Biotechnol;citation_author=J. Hofman-Bang;citation_title=Nitrogen catabolite repression in Saccharomyces cerevisiae.;citation_pages=35-73;citation_volume=12;citation_year=1999;citation_issue=1;citation_pmid=10554772;citation_doi=10.1385/MB:12:1:35" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. H. Hu;citation_author=M. H. Wang;citation_author=T. Tan;citation_author=J. R. Li;citation_author=H. Yang;citation_title=Genetic dissection of ethanol tolerance in the budding yeast Saccharomyces cerevisiae.;citation_volume=175;citation_year=2007;citation_issue=3;citation_pmid=17194785;citation_doi=10.1534/genetics.106.065292" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=X. Huang;citation_author=M. J. Paulo;citation_author=M. Boer;citation_author=S. Effgen;citation_author=P. Keizer;citation_title=Analysis of natural allelic variation in Arabidopsis using a multiparent recombinant inbred line population.;citation_volume=108;citation_year=2011;citation_issue=11;citation_pmid=21368205;citation_doi=10.1073/pnas.1100465108" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=W. Huang da;citation_author=B. T. Sherman;citation_author=R. A. Lempicki;citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.;citation_pages=44-57;citation_volume=4;citation_year=2009;citation_issue=1;citation_pmid=19131956;citation_doi=10.1038/nprot.2008.211" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=C. Huxley;citation_author=E. D. Green;citation_author=I. Dunham;citation_title=Rapid assessment of S. cerevisiae mating type by PCR.;citation_pages=236-236;citation_volume=6;citation_year=1990;citation_issue=8;citation_pmid=2238077;citation_doi=10.1016/0168-9525(90)90190-H" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Ibstedt;citation_author=S. Stenberg;citation_author=S. Bages;citation_author=A. B. Gjuvsland;citation_author=F. Salinas;citation_title=Concerted evolution of life stage performances signals recent selection on yeast nitrogen use.;citation_volume=32;citation_year=2015;citation_issue=1;citation_pmid=25349282;citation_doi=10.1093/molbev/msu285" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Jara;citation_author=F. A. Cubillos;citation_author=V. Garcia;citation_author=F. Salinas;citation_author=O. Aguilera;citation_title=Mapping genetic variants underlying differences in the central nitrogen metabolism in fermenter yeasts.;citation_pages=e86533;citation_volume=9;citation_year=2014;citation_issue=1;citation_pmid=24466135;citation_doi=10.1371/journal.pone.0086533" />
<meta name="citation_reference" content="citation_journal_abbrev=J Biosci Bioeng;citation_author=T. Katou;citation_author=M. Namise;citation_author=H. Kitagaki;citation_author=T. Akao;citation_author=H. Shimoi;citation_title=QTL mapping of sake brewing characteristics of yeast.;citation_pages=383-393;citation_volume=107;citation_year=2009;citation_issue=4;citation_pmid=19332297;citation_doi=10.1016/j.jbiosc.2008.12.014" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=E. Kellermann;citation_author=P. G. Seeboth;citation_author=C. P. Hollenberg;citation_title=AnalysIs of the primary structure and promoter function of a pyruvate decarbozylase gene (PDCI) from Saceharomyces cerevisiae;citation_pages=8963-8977;citation_volume=14;citation_year=1986;citation_issue=22;citation_pmid=3537965;citation_doi=10.1093/nar/14.22.8963" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. S. Kim;citation_author=J. C. Fay;citation_title=Genetic variation in the cysteine biosynthesis pathway causes sensitivity to pharmacological compounds.;citation_volume=104;citation_year=2007;citation_issue=49;citation_pmid=18042712;citation_doi=10.1073/pnas.0708194104" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. X. Kover;citation_author=W. Valdar;citation_author=J. Trakalo;citation_author=N. Scarcelli;citation_author=I. M. Ehrenreich;citation_title=A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in Arabidopsis thaliana.;citation_pages=e1000551-e1000551;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593375;citation_doi=10.1371/journal.pgen.1000551" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. L. Legras;citation_author=D. Merdinoglu;citation_author=J. M. Cornuet;citation_author=F. Karst;citation_title=Bread, beer and wine: Saccharomyces cerevisiae diversity reflects human history.;citation_pages=2091-2102;citation_volume=16;citation_year=2007;citation_issue=10;citation_pmid=17498234;citation_doi=10.1111/j.1365-294X.2007.03266.x" />
<meta name="citation_reference" content="citation_journal_title=Appl. Microbiol. Biotechnol.;citation_author=H. Lei;citation_author=H. Li;citation_author=F. Mo;citation_author=L. Zheng;citation_author=H. Zhao;citation_title=Effects of Lys and His supplementations on the regulation of nitrogen metabolism in lager yeast.;citation_pages=8913-8921;citation_volume=97;citation_year=2013;citation_issue=20" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_issue=16;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. Liti;citation_author=E. J. Louis;citation_title=Advances in quantitative trait analysis in yeast.;citation_pages=e1002912-e1002912;citation_volume=8;citation_year=2012;citation_issue=8;citation_pmid=22916041;citation_doi=10.1371/journal.pgen.1002912" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Biochem. Soc. Trans.;citation_author=P. O. Ljungdahl;citation_title=Amino-acid-induced signalling via the SPS-sensing pathway in yeast.;citation_volume=37;citation_year=2009;citation_issue=Pt. 1;citation_pmid=19143640;citation_doi=10.1042/BST0370242" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. O. Ljungdahl;citation_author=B. Daignan-Fornier;citation_title=Regulation of Amino Acid, Nucleotide, and Phosphate Metabolism in Saccharomyces cerevisiae;citation_pages=885-929;citation_volume=190;citation_year=2012;citation_issue=3;citation_pmid=22419079;citation_doi=10.1534/genetics.111.133306" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=A. D. Long;citation_author=S. J. Macdonald;citation_author=E. G. King;citation_title=Dissecting complex traits using the Drosophila synthetic population resource.;citation_pages=488-495;citation_volume=30;citation_year=2014;citation_issue=11;citation_pmid=25175100;citation_doi=10.1016/j.tig.2014.07.009" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=G. Lucchini;citation_author=A. Biraghi;citation_author=M. L. Carbone;citation_author=A. de Scrilli;citation_author=G. E. Magni;citation_title=Effect of mutation in the aromatic amino acid pathway on sporulation of Saccharomyces cerevisiae.;citation_pages=55-62;citation_volume=136;citation_year=1978;citation_issue=1;citation_pmid=361708" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=B. Magasanik;citation_author=C. A. Kaiser;citation_title=Nitrogen regulation in Saccharomyces cerevisiae.;citation_pages=1-18;citation_volume=290;citation_year=2002;citation_issue=1–2;citation_pmid=12062797;citation_doi=10.1016/S0378-1119(02)00558-9" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=A. M. Marini;citation_author=S. Vissers;citation_author=A. Urrestarazu;citation_author=B. Andre;citation_title=Cloning and expression of the MEP1 gene encoding an ammonium transporter in Saccharomyces cerevisiae.;citation_pages=3456-3463;citation_volume=13;citation_year=1994;citation_issue=15;citation_pmid=8062822" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=A. M. Marini;citation_author=S. Soussi-Boudekou;citation_author=S. Vissers;citation_author=B. Andre;citation_title=A family of ammonium transporters in Saccharomyces cerevisiae;citation_pages=4282-4293;citation_volume=17;citation_year=1997;citation_issue=8;citation_pmid=9234685;citation_doi=10.1128/MCB.17.8.4282" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=S. Marsit;citation_author=S. Dequin;citation_title=Diversity and adaptive evolution of Saccharomyces wine yeast: a review.;citation_volume=15;citation_year=2015;citation_issue=7;citation_pmid=26205244;citation_doi=10.1093/femsyr/fov067" />
<meta name="citation_reference" content="citation_journal_title=Electron. J. Biotechnol.;citation_author=C. Martínez;citation_author=V. García;citation_author=D. González;citation_author=M. Jara;citation_author=M. Aguilera;citation_title=Gene expression of specific enological traits in wine fermentation.;citation_pages=13;citation_volume=16;citation_year=2013;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=P. Marullo;citation_author=M. Aigle;citation_author=M. Bely;citation_author=I. Masneuf-Pomarede;citation_author=P. Durrens;citation_title=Single QTL mapping and nucleotide-level resolution of a physiologic trait in wine Saccharomyces cerevisiae strains.;citation_volume=7;citation_year=2007;citation_issue=6;citation_pmid=17537182;citation_doi=10.1111/j.1567-1364.2007.00252.x" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=T. L. Nissen;citation_author=U. Schulze;citation_author=J. Nielsen;citation_author=J. Villadsen;citation_title=Flux Distributions in Anaerobic, Glucose-Limited Continuous Cultures of Saccharomyces Cerevisiae;citation_pages=203-218;citation_volume=143;citation_year=1997;citation_issue=Pt. 1;citation_pmid=9025295;citation_doi=10.1099/00221287-143-1-203" />
<meta name="citation_reference" content="citation_journal_title=Microb. Cell Fact.;citation_author=J. Noble;citation_author=I. Sanchez;citation_author=B. Blondin;citation_title=Identification of new Saccharomyces cerevisiae variants of the MET2 and SKP2 genes controlling the sulfur assimilation pathway and the production of undesirable sulfur compounds during alcoholic fermentation.;citation_pages=68;citation_volume=14;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Nogami;citation_author=Y. Ohya;citation_author=G. Yvert;citation_title=Genetic complexity and quantitative trait loci mapping of yeast morphological traits.;citation_pages=e31-e31;citation_volume=3;citation_year=2007;citation_issue=2;citation_pmid=17319748;citation_doi=10.1371/journal.pgen.0030031" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=D. J. Omnus;citation_author=P. O. Ljungdahl;citation_title=Latency of transcription factor Stp1 depends on a modular regulatory motif that functions as cytoplasmic retention determinant and nuclear degron.;citation_volume=25;citation_year=2014;citation_issue=23;citation_pmid=25253722;citation_doi=10.1091/mbc.E14-06-1140" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=J. Ono;citation_author=A. C. Gerstein;citation_author=S. P. Otto;citation_title=Widespread genetic incompatibilities between first-step mutations during parallel adaptation of Saccharomyces cerevisiae to a common environment.;citation_pages=e1002591;citation_volume=15;citation_year=2017;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=I. Pedruzzi;citation_author=F. Dubouloz;citation_author=E. Cameroni;citation_author=V. Wanke;citation_author=J. Roosen;citation_title=TOR and PKA signaling pathways converge on the protein kinase Rim15 to control entry into G0.;citation_pages=1607-1613;citation_volume=12;citation_year=2003;citation_issue=6;citation_pmid=14690612;citation_doi=10.1016/S1097-2765(03)00485-4" />
<meta name="citation_reference" content="citation_journal_title=International journal of food microbiology;citation_journal_abbrev=Int J Food Microbiol;citation_author=A. Querol;citation_author=M. T. Fernandez-Espinar;citation_author=M. del Olmo;citation_author=E. Barrio;citation_title=Adaptive evolution of wine yeast.;citation_pages=3-10;citation_volume=86;citation_year=2003;citation_issue=1–2;citation_pmid=12892918;citation_doi=10.1016/S0168-1605(03)00244-7" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_issue=6;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=T. Rossignol;citation_author=L. Dulau;citation_author=A. Julien;citation_author=B. Blondin;citation_title=Genome-wide monitoring of wine yeast gene expression during alcoholic fermentation.;citation_pages=1369-1385;citation_volume=20;citation_year=2003;citation_issue=16;citation_pmid=14663829;citation_doi=10.1002/yea.1046" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=F. Salinas;citation_author=F. A. Cubillos;citation_author=D. Soto;citation_author=V. Garcia;citation_author=A. Bergstrom;citation_title=The Genetic Basis of Natural Variation in Oenological Traits in Saccharomyces cerevisiae.;citation_pages=e49640-e49640;citation_volume=7;citation_year=2012;citation_issue=11;citation_pmid=23185390;citation_doi=10.1371/journal.pone.0049640" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=F. Salinas;citation_author=C. G. de Boer;citation_author=V. Abarca;citation_author=V. Garcia;citation_author=M. Cuevas;citation_title=Natural variation in non-coding regions underlying phenotypic diversity in budding yeast.;citation_pages=21849;citation_volume=6;citation_year=2016;citation_pmid=26898953;citation_doi=10.1038/srep21849" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=J. M. Salmon;citation_title=Effect of Sugar Transport Inactivation in Saccharomyces cerevisiae on Sluggish and Stuck Enological Fermentations;citation_pages=953-958;citation_volume=55;citation_year=1989;citation_issue=4;citation_pmid=16347895" />
<meta name="citation_reference" content="citation_journal_title=Comptes rendus biologies;citation_journal_abbrev=C R Biol;citation_author=D. Sicard;citation_author=J. L. Legras;citation_title=Bread, beer and wine: yeast domestication in the Saccharomyces sensu stricto complex.;citation_pages=229-236;citation_volume=334;citation_year=2011;citation_issue=3;citation_pmid=21377618;citation_doi=10.1016/j.crvi.2010.12.016" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=E. N. Smith;citation_author=L. Kruglyak;citation_title=Gene-environment interaction in yeast gene expression.;citation_pages=e83-e83;citation_volume=6;citation_year=2008;citation_issue=4;citation_pmid=18416601;citation_doi=10.1371/journal.pbio.0060083" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. M. Steinmetz;citation_author=H. Sinha;citation_author=D. R. Richards;citation_author=J. I. Spiegelman;citation_author=P. J. Oefner;citation_title=Dissecting the architecture of a quantitative trait locus in yeast.;citation_pages=326-330;citation_volume=416;citation_year=2002;citation_issue=6878;citation_pmid=11907579;citation_doi=10.1038/416326a" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=D. Steyer;citation_author=C. Ambroset;citation_author=C. Brion;citation_author=P. Claudel;citation_author=P. Delobel;citation_title=QTL mapping of the production of wine aroma compounds by yeast.;citation_pages=573-573;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23110365;citation_doi=10.1186/1471-2164-13-573" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Storey;citation_author=R. Tibshirani;citation_title=Statistical significance for genomewide studies.;citation_volume=100;citation_year=2003;citation_issue=16;citation_pmid=12883005;citation_doi=10.1073/pnas.1530509100" />
<meta name="citation_reference" content="citation_journal_abbrev=Cell Div;citation_author=E. Swinnen;citation_author=V. Wanke;citation_author=J. Roosen;citation_author=B. Smets;citation_author=F. Dubouloz;citation_title=Rim15 and the crossroads of nutrient signalling pathways in Saccharomyces cerevisiae.;citation_pages=3-3;citation_volume=1;citation_year=2006;citation_pmid=16759348;citation_doi=10.1186/1747-1028-1-3" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=H. Sychrova;citation_author=M. R. Chevallier;citation_title=APL1, a yeast gene encoding a putative permease for basic amino acids.;citation_pages=653-657;citation_volume=10;citation_year=1994;citation_issue=5;citation_pmid=7941748;citation_doi=10.1002/yea.320100509" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=E. G. ter Schure;citation_author=N. A. van Riel;citation_author=C. T. Verrips;citation_title=The role of ammonia metabolism in nitrogen catabolite repression in Saccharomyces cerevisiae.;citation_volume=24;citation_year=2000;citation_issue=1;citation_pmid=10640599;citation_doi=10.1111/j.1574-6976.2000.tb00533.x" />
<meta name="citation_reference" content="citation_journal_title=Appl. Microbiol. Biotechnol.;citation_author=C. Tesniere;citation_author=C. Brice;citation_author=B. Blondin;citation_title=Responses of Saccharomyces cerevisiae to nitrogen starvation in wine alcoholic fermentation.;citation_pages=7025-7034;citation_volume=99;citation_year=2015;citation_issue=17" />
<meta name="citation_reference" content="citation_journal_title=Antonie van Leeuwenhoek;citation_journal_abbrev=Antonie van Leeuwenhoek;citation_author=M. J. Torija;citation_author=N. Rozes;citation_author=M. Poblet;citation_author=J. M. Guillamon;citation_author=A. Mas;citation_title=Yeast population dynamics in spontaneous fermentations: comparison between two different wine-producing areas over a period of three years.;citation_pages=345-352;citation_volume=79;citation_year=2001;citation_issue=3–4;citation_pmid=11816978;citation_doi=10.1023/A:1012027718701" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Trapnell;citation_author=L. Pachter;citation_author=S. L. Salzberg;citation_title=TopHat: discovering splice junctions with RNA-Seq.;citation_volume=25;citation_year=2009;citation_issue=9;citation_pmid=19289445;citation_doi=10.1093/bioinformatics/btp120" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=X. Wang;citation_author=L. Kruglyak;citation_title=Genetic basis of haloperidol resistance in Saccharomyces cerevisiae is complex and dose dependent.;citation_pages=e1004894;citation_volume=10;citation_year=2014;citation_issue=12;citation_pmid=25521586;citation_doi=10.1371/journal.pgen.1004894" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Warringer;citation_author=E. Zorgo;citation_author=F. A. Cubillos;citation_author=A. Zia;citation_author=A. Gjuvsland;citation_title=Trait variation in yeast is defined by population history.;citation_pages=e1002111-e1002111;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21698134;citation_doi=10.1371/journal.pgen.1002111" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=V. M. Williamson;citation_author=J. Bennetzen;citation_author=E. T. Young;citation_author=K. Nasmyth;citation_author=B. D. Hall;citation_title=Isolation of the structural gene for alcohol dehydrogenase by genetic complementation in yeast.;citation_pages=214-216;citation_volume=283;citation_year=1980;citation_issue=5743;citation_pmid=6985717;citation_doi=10.1038/283214a0" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Yang;citation_author=J. Fresnedo-Ramirez;citation_author=Q. Sun;citation_author=D. C. Manns;citation_author=G. L. Sacks;citation_title=Next generation mapping of enological traits in an F2 interspecific grapevine hybrid family.;citation_pages=e0149560;citation_volume=11;citation_year=2016;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=A. Zargari;citation_author=M. Boban;citation_author=S. Heessen;citation_author=C. Andreasson;citation_author=J. Thyberg;citation_title=Inner nuclear membrane proteins Asi1, Asi2, and Asi3 function in concert to maintain the latent properties of transcription factors Stp1 and Stp2.;citation_volume=282;citation_year=2007;citation_issue=1;citation_pmid=17085444;citation_doi=10.1074/jbc.M609201200" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Identification of Nitrogen Consumption Genetic Variants in Yeast Through QTL Mapping and Bulk Segregant RNA-Seq Analyses" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="twitter:description" content="Saccharomyces cerevisiae is responsible for wine must fermentation. In this process, nitrogen represents a limiting nutrient and its scarcity results in important economic losses for the wine industry. Yeast isolates use different strategies to grow in poor nitrogen environments and their genomic plasticity enables adaptation to multiple habitats through improvements in nitrogen consumption. Here, we used a highly recombinant S. cerevisiae multi-parent population (SGRP-4X) derived from the intercross of four parental strains of different origins to identify new genetic variants responsible for nitrogen consumption differences during wine fermentation. Analysis of 165 fully sequenced F12 segregants allowed us to map 26 QTL in narrow intervals for 14 amino acid sources and ammonium, the majority of which represent genomic regions previously unmapped for these traits. To complement this strategy, we performed Bulk segregant RNA-seq (BSR-seq) analysis in segregants exhibiting extremely high and low ammonium consumption levels. This identified several QTL overlapping differentially expressed genes and refined the gene candidate search. Based on these approaches, we were able to validate [ARO1][1] , [PDC1][2] , [CPS1][3] , [ASI2][4] , [LYP1][5] , and [ALP1][6] allelic variants underlying nitrogen consumption differences between strains, providing evidence of many genes with small phenotypic effects. Altogether, these variants significantly shape yeast nitrogen consumption with important implications for evolution, ecological, and quantitative genomics.

 [1]: http://www.yeastgenome.org/locus/S000002534/overview
 [2]: http://www.yeastgenome.org/locus/S000004034/overview
 [3]: http://www.yeastgenome.org/locus/S000003708/overview
 [4]: http://www.yeastgenome.org/locus/S000005103/overview
 [5]: http://www.yeastgenome.org/locus/S000005212/overview
 [6]: http://www.yeastgenome.org/locus/S000005214/overview" />
<meta name="og-title" property="og:title" content="Identification of Nitrogen Consumption Genetic Variants in Yeast Through QTL Mapping and Bulk Segregant RNA-Seq Analyses" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/6/1693" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Saccharomyces cerevisiae is responsible for wine must fermentation. In this process, nitrogen represents a limiting nutrient and its scarcity results in important economic losses for the wine industry. Yeast isolates use different strategies to grow in poor nitrogen environments and their genomic plasticity enables adaptation to multiple habitats through improvements in nitrogen consumption. Here, we used a highly recombinant S. cerevisiae multi-parent population (SGRP-4X) derived from the intercross of four parental strains of different origins to identify new genetic variants responsible for nitrogen consumption differences during wine fermentation. Analysis of 165 fully sequenced F12 segregants allowed us to map 26 QTL in narrow intervals for 14 amino acid sources and ammonium, the majority of which represent genomic regions previously unmapped for these traits. To complement this strategy, we performed Bulk segregant RNA-seq (BSR-seq) analysis in segregants exhibiting extremely high and low ammonium consumption levels. This identified several QTL overlapping differentially expressed genes and refined the gene candidate search. Based on these approaches, we were able to validate [ARO1][1] , [PDC1][2] , [CPS1][3] , [ASI2][4] , [LYP1][5] , and [ALP1][6] allelic variants underlying nitrogen consumption differences between strains, providing evidence of many genes with small phenotypic effects. Altogether, these variants significantly shape yeast nitrogen consumption with important implications for evolution, ecological, and quantitative genomics.

 [1]: http://www.yeastgenome.org/locus/S000002534/overview
 [2]: http://www.yeastgenome.org/locus/S000004034/overview
 [3]: http://www.yeastgenome.org/locus/S000003708/overview
 [4]: http://www.yeastgenome.org/locus/S000005103/overview
 [5]: http://www.yeastgenome.org/locus/S000005212/overview
 [6]: http://www.yeastgenome.org/locus/S000005214/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
    <title>Identification of Nitrogen Consumption Genetic Variants in Yeast Through QTL Mapping and Bulk Segregant RNA-Seq Analyses | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__49R9wsCpiKJS6e6ZOiXRsC0989bO7FF1XdNTikSRKGM__k1m2vpL1Nu9n43bJnV24EQ0Z_UWo-Iq0zZ2FkQLL-iU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"IRUCzwnmw9NlAt5Wu7RFKrDUtaieospaEPfxope9-SQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484773","apath":"\/ggg\/7\/6\/1693.atom","pisa":"ggg;7\/6\/1693","ac":{"\/ggg\/7\/6\/1693.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/6\/1693.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1693"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1693"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/6\/1693":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Roa+nyXcrKGbehOleFyqeSqhbslx7pGmF\\\\\\\/geWGhvOviosfl+0YA8cUIhBpNinriBqOHrCMMKtS4QID1lADRBHJD5k3Y1ZQnIlLkYZhuOVsqVnHJQ9CZQP23on2402SYsLPJSNrD1jABaM03HrWkk4jjxMQMlUH4M6l35qb9fSEXevXtW8\\\\\\\/fwKnoI6JQCihDmmm6gz9vLsGBNOKuL6L2t0iB2QH862GcRZ7GOLUaTTjx51bp2DlJ0TBURksN1wkAQPzCkuecG5bgdRqy4io+J6hqcL1QDRl6E+cpF7uSrr5AfGua1nWs45waRhkb7pIWatoM0vgQbfVknbyy6mvv8Kc+WbL6j6EzpTQ6Z9tmJ+fsHYth5XXiBh6IHdOYmFuC9\\\\\\\/29qV8cN2Ze0bDEJz7levg\\\\\\\/9PN0IHo5lDamdHKkKTnDFeH8T2Un6Wo32gNtxyZMyC5hYoS0f6GOFYIrvpCdiGKZVEOaorlVOctTcj2U834vZLhLaNJctsd3AVM\\\\\\\/nWnOzs3\\\\\\\/88Gw1K\\\\\\\/ZfDWouyTW1YTdTd8d+\\\\\\\/lbHhssBQGfpq5LcFnPAkLiiP5QNZSvSUgNJ0\\\\\\\/HqqlvyhaOTCkwBdc8SxH6\\\\\\\/IXP34ac5Ns5sC4UK6pXf+agwJBdW8QZF0Ne\\\\\\\/Yt6vMEv50FSPwzEbdXhkjelI0cE93UHMnx7mlljOQ6P5DGXwT+tjZCDxCVleGy6fTHUQqbP0bFZpb2zlsd7BUiQMPRWk4he3kApyUMgNtA5jC8q1cN+kNC54Ett7aEvdQN09mBGP0AOzYMAuq\\\\\\\/adwk7XSgMRnYmFP7xW4e8s5+y1dvHcqUGGc3156dDYq4EYFNEMGTwCr\\\\\\\/9TFndZuWdocnrkhHCBr+FvpUyp9kEnYcdY66wQLOMr0s8CnchDaQw1Wdz9CBfkLfVJHllXKKeKWh2eTmFGiR1o3NxuXqYTKOQIZnE2u4en7stHlJFyrfxEbbQ8AjYHFP+KTZvrZAZFXQDNt07k02iRNe1fXfa\\\\\\\/5omCymOGvDD6T6dSSJhHC+b7Bhs5+tPjO1v070kvbfuCNPQzzGuMuzvcXsnKkqJdstXB0nyt1pv2SlMKc4rTSI+\\\\\\\/k601iA2alowcGaRPs1lbOCjEd9pRjR0taoomVzJBk13eZKonxBP+YCovHRsr7onXsBD2AIMI0+rHk4Jpc4ycd0dGs6e2SjfNi8Vk6OrBLX0Li3Djx1tlexEe3aQkVrJ4UxZiuJb8fofeYwSxr\\\\\\\/prxw7xVm3hmfIBtRDc8QcIZOumf\\\\\\\/357tLb8Hyz\\\\\\\/UDfrEwuES2+\\\u0022,\\\u0022iv\\\u0022:\\\u0022u+15ZaKGRv0Wav6d2RuktA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022624f0b5cc6fa73437ad1241cbf673ce50e7ba744c3dfcfa62cbb0fcea68abb93\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022P2l7vjPitmriXbK6x+TzgmTkQFrJc2gpL6SgsVhco6nrxPLDnqcSO4ia3e11vO9+EAUPpg0XmUEUntWN4u+c8VkXejv3E6JOlnO+lazGXy3pW7KBcbJ64P207wKnhD\\\\\\\/MRpAMER9dywTH10Li3ImZogEiDNN6lQGhDbI7dvjc9b4kBO+M5X+qakVXtcEyPGvcR9BFJfyUg682xsvi3br52n\\\\\\\/AfUT\\\\\\\/ApZaZugBaCkZbN3X70YkaJX7VWytsZ1044vqF9S6tazc69fNzLw3V0NDvjN3KpIrhs\\\\\\\/g9KQRtSMLYuHxP7EWlAMMbAtmj7d0GF2T0Wbv9nwjVC94Aww0PhFsWRqll5ghwqLBRuK6m4ILg5v8mWoCmZzp79E\\\\\\\/Qld37fLpSJz5V5hsNbYVBfpYvnJI9a19yldX36JQ7fK5+Y+5\\\\\\\/QSV63vWa\\\\\\\/ekF\\\\\\\/Q\\\\\\\/GsFQYLAfCcOror8144PQpprdz3yTVHhax1PnkMaEQu9X4mUOKwgZ6QZRn20ywxHQ4UZtdxYcZrvYEhuM4yGATYmwGA1NDNRd4z+9g0hsnt3fMryTTvqZa7yHVQYd04nUgbllcNobxlj6NaLBxUtM4d9dlYIWFNl+Or39VcwCFAafli9oyqPHNUkqMY5HKigYh4Om20P5D6w\\\\\\\/s1lWNC3GWSYGD\\\\\\\/XYbD4bJEkAq15rjFtrY2OdwDtsMSW8WZxIGv+xdXkfxQ8T+T8OYX3j+NhHl1iTfcDoG+vYVpujFc8XNv2xyjA+qT7jDawdg7Q3uWzwRW9weMRE2pmhP\\\\\\\/Bn8RQ6K5RxWR1xKwRpI6X1NHRtEVpJouUXBRDSFllGvx1rPoNUZG6VA3uPEyRPhMpx4IReyD\\\\\\\/DHYuiwtu28LlsOEjwm7reoaO99IXFLi5nqdYQ8d1nDhRSMVjCfdcFFJrPenhD9Qkhxv2gaz3Q134GPf1EgyA\\\\\\\/KixdvcyWXwVBQzCARFccuMKGjK5e9DdxqNbAOpwj4B6qsW2Pfejmcy0KiZj7yC51u7RqhGzui+d2jOdJ3fgufAwDR++iBYv3U5EJHzHxdk5PYZmYVQexCDQQEcmLimZB9sA98cP1zK0tnxVuxgc0095znD\\\\\\\/x5DRK+hkLd1IjQ6ky3bEKcy9KLIPBSInpZb7C4DxOa+YEsSYYstTgaP38oTuOWxoNJwNfv0MR7Thee0DwMMhUyCHOCs6CGGxZgAOUv2irrJdKT\\\\\\\/u6Q+EZdBC+xyvU6po9a9ETHR3f2J9MEkBL4zVLEEcYbA+hc5AY6NwkAWsPIQXoIRrWWw1gjlAb6BblYT\\\\\\\/F+qCeSOjI6gdwQk781bR5fH\\\\\\\/o7d7aD8vA\\\\\\\/IdaSoc2Q5sML5Wi62UTBsORkhbZDS2LLUsfdXROrwNFkIBYIP1jLbptkqTFjpnuiZXL8yi1FepqFdL8AhVr6gJjbkZTkvna6t24nb0u19ysW746qQ\\\\\\\/EQS8YsAmyGngeWJW8oto0J645VH74JyJV2TKM8ETZJXmmIyINKLA\\\\\\\/OwJUh1YTPzX0y2zMEZBU\\\\\\\/g9ywH7P5V0VyXMqCtmCtXNX7RDDeJqX+uqHR4c4+SHZj53Wg0xAWEXIgqnylb1iVyWGQc1iQ2mtwArsTlIdeHjQJVlQqF8qpDkughrCDHw9Hbn063HxXV2Taa2HObmH1ppTCp9B3QwWWaAWDn+Fg3cNuGam+Gum5AS65jPOcLKIv6xu\\\\\\\/eBpaTnkgr\\\\\\\/kWWK1wKgOAUdRtY8cF8cDVT8spQnqySj0YAqI0zrUa45piz+HadzRYZ+ovRof7OEMV1e9OYntVvqtf8zDQXcHV43F3kPp0ws8M\\\\\\\/7ENV6c1TbDau+ArORfe9ORN421jHYRLM0EV1gNROsudn1TW6nXxbeb7pyqnrDcv\\\\\\\/nEbE8TjF0bSobG3m\\\\\\\/B\\\\\\\/P3eCOrXatkcLrdVv0rhErswfBvl4kcKsct2nofLsFMel1y2f\\\\\\\/l\\\\\\\/kBmgzEbE7mjo66m+90CiNNC0VwE35fDx5lDqkG4lV1Gtntb1IoEQ+gF8eZToqvP3a3FXg2EeWBC9cvRQjZxla04YFmwam\\\\\\\/YVDYKphZ\\\\\\\/iKoEkm7FPzESfSkDmsNWKPitayp7mUWubkuocal2h2x5Ub31gNF+NI9VmFjRp4nI24ZchiUA0J2adlCH3FFlvy63x1UnLQ9cNAgm2TNZsaPtvbYXgp6kpluK4SWDWP6tHKB6VOWnjdy4kgxE53qbBH1ATpzVU0hj2R+VS73On7i7hFPeCzs1ks0qYrV+tX1kqfzlhuCa+u2dokIFoqWYnp9e\\\\\\\/spO2rVe1D17QxTRy7QCLj5tXnEwKrFM+thlCnEmtSFFCuDfTOvYKFQlj3tCKtceAijTUrkp+F+kbnwgWK2GRiRMHqORu\\\\\\\/FQ19Suw8SBS2a+bBmbWG7YtqE8IgeD6it+OypXGLjvA2Kw+dLcnKJtDqlaiY3TYcwaOpISV\\\\\\\/CdHbGKosQdq1G\\\\\\\/5HZhktGmWh1dEqnAvy+4y+310tRdgS5FkVr\\\\\\\/oiCT5dAg0SKnbzpmC8bM5SpBvPwxnNPCUeVou+ToEQRo7RJzqcoaAED1J3tGGmklrAZsgdzfP4x\\\\\\\/GEQtWbqEyh2kJzk6ExMHNJ8MrpzU8T5zbDJBhXeci0wDBeMVPBjdDn9c\\\\\\\/yhzlgt1nsqj4mQhCczr0F6wncSHHc7NOM7vmjSj0bf1WTVcdBs7oYJqsLgatzPLZmKIufyOtHtcTf9yLq7dfQQoUUw+OJUGnKLiXowvVxAnBnvWz9I0S1PMMbMCWKBGzVO9FbL8kCfv\\\\\\\/x\\\\\\\/WztZ3ZK+TIiV1m6sMmduFiJUkKHobKogboz2KFlsz+ndY+z1HsX19Ch0Ua1BtrAVqKG5Tj4gut59cBdG++LGmHpFUsrGN\\\\\\\/cD5uLaR2810fQA5tLsZc4AEu8djtaMLDqjrFd1o+KSjxzcoJNf2N8wc4jOXkaY3kFgvuM11JCqrFKjt8bNKZbrcrJO+7GE6BB5Nj\\\\\\\/mHcGHbxEpjbSXStEv3aBvrPFRREGTMMqJ17SuOVT660nopW028vP9TCAF2l36YhQviYNgDAxLOcDCOM0M74fXLhYPno\\\\\\\/FJvNd0V\\\\\\\/HliITZFmYtfy9\\\\\\\/016gZYsCmkxEig7Pw0j17NYxlDBNdUe7lgI2++sB1s+yPEMO2aa+kTut8knRqrmxNYvb5thZPqWPSc8s4zpyT6SHx2BNVNMVpBEFhVSI3JzhvQ00HXXdlsEExX9pQ9tlsAiVCUmgkysAjv3wHrqIIHJxGgh6YsITp6VU2NVwIr6H5FSW8hfn7iscc+nOUHeTl1\\\\\\\/ZULI3jKuq1QJ0J5cuNU94RYADVLpLB8fReg7upezFmylQiYmoNgGxGyknNVw6rtC7uGxEdJzCYuOsQfPe9a1NT7HjNqeicVWOHMjcP8u2aTBEo0rO0LItNA1zln1xyMNvleC7jB+2nFACyKmHWbLY7Lsvw2G7ngPYhoRyXdvNN07O8UvQlWIqsanyDlF+ceDjDnAXMM47A1lV5c6+Nq\\\\\\\/AcYYMzCzngtVi\\\\\\\/iWkdhvWijZzFDzSM3kpPgDsZvEQPgSWobY95mjWRBDxcZZ7qEqNrLtqr3q7oInaFhJASjHKkzoMlmv4+NcMknSJjZP1Lbnqy6dPnKbcpgN\\\\\\\/BkVB29dZEAA+Fls2KZ\\\\\\\/xx9l9MueaUc0ES7G1hwPm1I\\\\\\\/U4pB4ONJuz3jf4pdsFm6m+fyBg9Y\\\\\\\/\\\\\\\/p2PS4bzPgF7M2Z90sYDgEeyK6C3I\\\\\\\/FGl5M72c3OOIQATl4m7QkeG+E\\\\\\\/0ggVVDZqAglHKsqMGGasOqzdnIMhaLd5nLqiOzt8EVi\\\\\\\/Qf556TxR4z5j1LCmef\\\\\\\/eLn\\\\\\\/qM02I6i\\\\\\\/jYX1G3HARZkWcOBkUu2vhA4fgjmX+y8BVXUtrnkd6tKjfShRhSegkdu7krot1K5r297z4m+MX\\\\\\\/TclqzjaNXAQfrumFfkXT69mh4YyzKfLha7bI7LFrKMlVtCqiYF4mRd5d6LdW0uXF3kJno0+gTr+9BfIYEjV3i8+g144OnpCamF\\\\\\\/XWzUoy16a57+EVvXU0tnP9FUMScKR+r\\\\\\\/d1h1362pkqOSYVUT+shLQlEFtpQe+fOaHM+MbdXjb40t4wNARp5tKBNAcfgvmtMRmXm0C4YOwWEUmt4ZCpRqC62t2Y\\\\\\\/0fyo0sXlaRbn4PGiTKadc6DSoIpN4C7no0FrybmoZliPrP2GOw8EBLdGd5e\\\\\\\/\\\\\\\/rOihX5HyvZA6EmC8TVRCa4CcGhV6Axzf6VScU+Yk9HR4HcxgBIwpocpFShY5QeSl7vmJkOi1hUYI6NKgZ1r59Pj6kSe3Lepln7giHKbddWUuJ7p5TAFMaYyI2EiThr\\\\\\\/NoUk95uityNCuFnQFCwZ9AiWBUhRMSjxu8kv1KNSToNN+16Y4yxGGY1eOuaaSTCWeviYzAA+iI7\\\\\\\/lIUNoCuMFqbTKEZEpnlgZtGxOeaD2yIfVo47mA5jUj+fl4IsjYljVx11ZIsiqkh23FV+MLugEyHxb5YTRyY4fKHngQLplOcDbklrcjCB+jNRHxfzF4tWrNpcDL7nQLb\\\\\\\/fPpD\\\\\\\/vVBpSJIHOjWNnB+z8utDAr9lvFMr7dvqrz7q6hSG3MpgXbxiav6wq9ADeZVyLnUnH\\\\\\\/OMvK5rQC1LQTZCeoZUpYtGrH941GIYQeqS8DRQhmlcSCcCJHSlHZdj53hn09bfOEVpxlYvMezacGkLo8PArOmxs5AQY6BNVZFoiOLsEo82J\\\\\\\/mEbmwjnH6lDNaadazqf04DPDefFPMZPAWYKRilX8U8XunB\\\\\\\/XcBvHXMy9oIKIxysbg5DxOFvZZn8wsCuefXhw+GlZt5zJujUQ0XX2uHO1jDjU6vehrLdoPiR+Bjrlo1iP+I6EVz8+mh0UTlq5cwBBQKkl68kJ1dF2KBKTStd+m3VauYY4+4txfqNos3SJW97wx6x+rdpEE3TKSrp\\\\\\\/luQbf\\\\\\\/Y5UnmLTyYRyRk5zDGGL58BVGfvQ610Vn9rVfFuuDo80IRtK\\\\\\\/5JBnq6bkrRMSAsRjhF87Il\\\\\\\/YHGJn2gUpYv9cI6WWHsl5Wnw1UWq7dvJJQCD8Jy4ZUY01PqYgbZxWAvvcvcNEA1T2Bv7u91rgkFNroLN7q03ehHHIMBp379KrrAS79fUfy+xHU+3mdVX73xlcK1rSRk8WIUo1zyqDLtrHNP8isrDzn9KqCApGSJsdIS6c98XS3zBR1maH3wRg01Uh5LpMSHunk9k1rH\\\\\\\/jj46ntikbVTs1Iah2kyRvy+PNAlsNDo4zR5OlaTDndhzU+99+b2xbFB2CVL0YwztVIEYcwfalBlkl4JRPERujQIfAxd8FoSShcJz8+IGC00nuBvb6EIPsSXB+AIf35bUdj1uyFgnpuHb2f2JGccU+o3RDRBt9SoAIxJP06XlFZhdyEr0HYaUnDJOlakvxgL6\\\\\\\/Kx2XDn7\\\\\\\/4T0+vciizOCZX9T5x5+iQcLn0qln1QKZqbdZlLuvW+ip6MKo7CZnfKOBi\\\\\\\/0ol6rktPhOmAzrm6BWiNe+kp1nGfk88uc7+ZCJreFcGecMGZPZu0F7qITjpYb5Pk\\\\\\\/WPESphxeT3j\\\\\\\/8u8I5cvoZCKFf2bmmq5Bk6f2NkNswH7d4yyTXGACZCIwztoeL+fWLRt+7vwjCAYyvpc18lABREa47x5yd9y1LHJOHEU6LXfUPXb4qrffgLz1Glbl0QxpeO1L4Tq1gr1ABLrrdfLY1t3ClqSA0wchXuIQQD5jyCUcKVsZRD5vLq8s\\\\\\\/VdOCdxOvibKUTMFWkgqKfxD3l0ThggkTYfDVD1MSf6DbxJYtpippBL2UwQst8oFH0LoJ8mdiWBx2e9iRHYpObXi\\\\\\\/pu\\\\\\\/\\\\\\\/2zCVURIw6shyrxfs984Zp600qKrFXXZRpRSUfjTjyiNGtzzn3tlnFQQ1DQapSfcqVTFStCVBJDg35I+n0ajs5iUQfBHjTu6gaO9m0LOT3ydW+fK6mDiRwwHhSP7NYl+BsXHfgAahPo5eK3QA9Q1r+DcXn9sUuJ\\\\\\\/wzXXC+z+AFMZACA1EbDBoeFsBquyW7UaSfwqzpfaFWit7xBIyPWE7nB6494+BbdrbsuP42pitxIEyxLW81D2vusOBaFa7ZqX7F60V95LOUdyeZ+6i00tjx+V5UAFtSMdvHRTDGAkr7LOb8dp2y3k3w+MbQG5z60vDvylhMumQ7NhAptgEdCV5m91cF9byhry5dulD+hnHF046iF7hzXMFA9b1XKK14WC5h3G+ISwidNb3fGJEpXhHm8ES2aJJm8V9jJ2KexfvCowvmpIlML5QOx8YkKE0UXNlZWZZ2Siz9g0I678Rg1Jiqu3gtTsF8U9czQm3u9rc+UwCY1mZl6g5hDN4a8DqTTgHzXS0JE1o5splx0lQxo9fZ8XENj\\\\\\\/zIeg2gXuJ+4tcjRSKgzON9TCU07unMcUm5u60Rib0p1Oiqn69VFSALPXKBDl93OjjNlJUzAFk+b1QsC5n5fQywwn39YaDaETIzLLwC1RNKgelN6tG4UmRy3noRblucmmid5UJWpAOym2+Qriix8JDqEDYBgpSS3JyLhhrp3np3etqWnPk4pzTdZTLLKI7wq6xIxJYOVjh8Z\\\\\\\/EFqASRVsySSXT6QLGOu4HrnG6DyRH+Iu9U2mCWdh9EYorFBzkd5E6UPG3EyZgCNMmEbtyLhUzK6lxVX+KP70pVcDGlAbTfi8vb2C0uR2EE1VA2amIMUPuF7+jThsKki922\\\\\\\/Z0fsZTxvF3pjjFL8qz+Zkb9H9ASHg5RdqQrRuoOv4vUdofhkoKmGP\\\\\\\/GyEFSgxVL\\\\\\\/gs6sTwLmZIbum+Mw5CE0IoRpMjmPKoqKS2LnMENnssgO6a+BHPepcpGkMr3D9JjbcnYRQFrL8d398LXqtazPUuvdbcVmOGE6XuCdhxunYfkxKJpTP3xrd9LGw6zPtFTk5kny7hTMlKZeXl9zzsdWi7\\\\\\\/8ejaFSwTRgN1zaHONTrZGHf5tMTVmq4XgX+wqUfCH552XezVA6pb0D1HCdu1\\\\\\\/z5wNKVGqxex6gq4JyAJhczOBBq\\\\\\\/p55BOs2CWif\\\\\\\/xyE9ph5SZbTXjvXpWxCtnyOgBHH\\\\\\\/lvGoxVd76GTbXRrj8KaR2zZFNIjjECHpx+Lkb9hyJdBK+71LC4sjQx+rbDR39QxciXJEBqnEYrNWEpWvuKyYq4N0PuJXROFv2a6HCEhcKZQnWmlQ5UHazXfcrSuMjYwuMEPXb3iVGRkmzYgxOHV88Qql+EZ1pbgA7HWOGy+f3oYcGCoPvLq6\\\\\\\/AiC4bLISRlNJ8Z2dn1s0GROjHxDvNJn0r1DBwKDieDkB0BQrkRWM8VluODU5f2TjZJaNzUSK8GgWKYtbnUcePNH22Elz4YvUpZvuPWPa\\\\\\\/cedBjYGkDA7HMFp6PaLoOoUPyeTt6TTqyFdBzofVB\\\\\\\/E7cgygqt4eod+aLJc5\\\\\\\/aHtbrrlGVpiijMptqpm3zx2nNjH65jswrJswqtV\\\\\\\/sLOR\\\\\\\/lS9Jhpj6LsndzIPk\\\\\\\/lqza8I8w40Bd6jomWGdb7twI3t8hO74dXIvbALAIWaGJWylk4PKBWgLU+x6r7D3k9T5slYfsqvFC23vQ54TEpIaR7MGrKN49mjmADvbhVc9fHp6AtRO8oGuXNQMNS\\\\\\\/dNwcR0i96CjDYk2by3Dr7vJO5IftrvRfMTAE3CSA0G8lKan7v5e1jubvJdFU4Yo2UvRkXGOw+X\\\\\\\/NPuRW3YTTGVZfpaTOFPVOCCvZIcpM8wcPqp1\\\\\\\/caf\\\\\\\/p2sX7I5YRAR93Mdi\\\\\\\/Bv4VOKsH9460sB6lYF3+\\\\\\\/K+L0VZtDFBULQObQt7Q4UrqO+z7xaNEYyRx5fc8fsbH3rzv1nGww9rRk4gdk4wSx0SEVxLW0u5PBDX12r6iUMrgc62SK5CpfQc+FQHAvbm1ogq1oafRY9czBb46865EFL\\\\\\\/sEKl7oylOdUYt81i9R6EuvAXZUCvY\\\\\\\/raZwle7uwkOZRDUpcqtBS6R5aaQu1g6Hmfp\\\\\\\/W14X5Jf8\\\\\\\/0NVF4wftYZccAc4ay2QsNsBvbUyDNewYstlWz9YJiXn8\\\\\\\/6KVFPuXVzbirv3jWGrx7sNNQ27H1eIKRGeyX3eJsv191vLnGNpcPVFnbeVWGoshmvoulrOvsgf2sVMHP1jsmQFNiCHyOUaFmIZtPObUNnUlsFt1qtrQUOb7n\\\\\\\/O7PO0qIiTAT4WiV0wYasVd90rC9oU5\\\\\\\/zVhAt9XAiGtmuumpGa8\\\\\\\/a6m7N0LuI1SBJ6QjCQnavVOkaN16KQ+prV2tmvmnSULTtY4BZ1qSd94lQODCBnksQg1p\\\\\\\/\\\\\\\/XRVMy9sIk7OVNyhfUwVxU8m2Lxk46tcIjKmqBuHSvlphNE9fuZ6GObXyBNj7hRf8EkSt8Z12hF9JGGkihfk\\\\\\\/szfMCaESf6LN521nXB7K3FxiSPvxFXMzFpIfQ\\\\\\\/ZZ1TGP4o7rxoPRmzHcqZPV11+k8gYA0HXZmElktvAmrh1P59Y5zQJPsGVd\\\\\\\/+Yb99xDqevWRIzt3ITnZqj\\\\\\\/xc8bCyqdLoFFE8ju01PhbWPtZd1iiCWs8TPQKIoZdbLaiVUBUVaX8svbdIG9XDopWhQT8nDNsgCbmFoFnX3AxK0t4fQfHqXsPBqcQU+wkiZeWibwTxp5ErxGtWjcRPAoQrfD9b0p5BJm5ArzNLiPHXrPFc1URs5FU+4g3nZFOTpzpfG0ySz0UVYo26bkmWHr9ATA22x5\\\\\\\/Uv0vkjFay4PS8bFZYa7xoYvZzz96STbHaQuZiSqTrqrJX+uwwRtTSOcfaz5wmlayOWnRbUi53M7bbIpY2h7nCoKNu9V84yUxTNriCWxy95K6p1gcikjaKbvTl\\\\\\\/DVhPtIetcXUZQJgIXJhjSBnNpFw81y8wUqIMb2NP65hfhjMXUnBw\\\\\\\/whaGeg\\\\\\\/olnt9HiJ1ZcWkOKeVshCQ3WrStTH2xmx11voy+EFrPvI75cfZODLSL1OnuBnxIwzlXnHDBXKtnz10Uv5YW2D81c4bfKerVgo3mvRpaankkkRBHWVlhxVGSrvPqoiPyyhjh5rEM+uziJF0yu5GY4BGAPplxXe6NfdeM8xL\\\\\\\/I9500SrNgUbSHfa11vDBoP6CTzzfOT0n1gUuRQAOQTYitpSFqsAm9oe4vnGMsf0Ehsij+MC3tAeBhvTfqd5jdyxRz5BnclxFiKLz38IUUmzg4xbMd5\\\\\\\/1HeNaoN7u9kMJBmoeKCbsAKSamT0G7cRaB4gWiN4A\\\\\\\/JA9T9xUVgPrD\\\\\\\/f375m0BRMX1LhRvH096HmsiiN4ZIiWOC4VWjMLYYfwxk\\\\\\\/VcqbhcaMTQqhhQgo3xaAlmzwenuaSX1Vvr69aMMGxCaW8ltXKWUzq\\\\\\\/pRZhXyulI\\\\\\\/Ujv3cPy0h3HNP+vxb2HqDfKzgceF\\\\\\\/Z1rNlBqG394F3CD27cZMbLrm\\\\\\\/7kY1fZhRwPaPKJnyaGBr33r7S\\\\\\\/kPVUDKCmkyCgm81Y4YehkJ0wVv\\\\\\\/00jgnwnWR8yWIy\\\\\\\/rhC3h0K0pdGXj7oLmf6SZBPwtEjMO3Cs0FmmVca7ojSl4s2j103PdpIx7nYHFBsDZ3cv2ZqB5kM\\\\\\\/nmSxqWK6LtA\\\\\\\/qxV5jLe6wPMB5o8IkzXBAsUoLoRDcuKTvSHqFlUiyd5mJQPMx6cJK5VdDpeImLC9c4ForDwBcCk8E21f8h8DBgAjcfE+oSgax5\\\\\\\/GPxw8s75J6xtD2RqZYP39zv1WPvXqteZZlrDLjuKxqK5SxP+bvf2SGUXzxeuMtOE4s78ptv76mVtWYIMK1JXTUEdEDC0AgMsjtLoB\\\\\\\/vLQfdKFA6YOa+Xg2WAeosAWLB+SLsBgoJDcxPJZSqnOXg37uq9RqMka6HU6nES\\\\\\\/g8GiBh7tCBxrVvopRoQnub+0L\\\\\\\/Ex\\\\\\\/HjXoeR0yzfrtwvVv\\\\\\\/2V1HnZI4MyXFdWmuE71\\\\\\\/6OnhhQC54RPsp63BeG0L2Ngjm5SBUrLBls3AZSpuKs1TG7Cbu871hvziwuQxa89tdn4cYYWioUurUBUcMMmnwenbqKgKz61uBv\\\\\\\/DiRWa9IA7cVGzqhp9lgmpGFJELKA7bgR0WLFBDWKXCCSR5SLM2w4tlZjHEiO+BxSSGU\\\\\\\/oLsa8ol\\\\\\\/btW\\\\\\\/zhysvnZFIr6j2D5qCQPwsWA4wOYK3j24cF1+wVSBIIoA6Cw9Cx7q28Rv5gtjp+NnXo\\\\\\\/APXzMyrK2rGPlKhmEBgRo8i5B+V749dKUErdX9qYHIOGntP8VSE+WkyocSxPalMKVdhQLPCzq7mxxFAXaXdoCcaBim1xyMOexJYI69V+8\\\\\\\/ZMpNcKwVGEbAWDgK0JeFLLuMpGm7SNJOMjvVL1xRvLegmMziYC+tUnLJI7rlvSRZkVuHTDU+Am408rJSJySeMZns9c1q29zCOOj5drlh+M8bgBmlMdUsyKxz2na19W+YRX0Fqo7YrVIjHbcPH\\\\\\\/WP87zu0u6VRQfG4aEX6W9GaLk5Y75rcNc2L693zIjJRqfAOcfEMR70C7SpQMoI\\\\\\\/Y9IJky9vne6vyXmoFp3vK3OnmsofYLCntaaZztnn1ERb8zfUmPH88uAWWFU2hIOP6XkRYKVfLuscQeYLrmsRDkHBviHzj6cW8zMpW9H2jHM0VofJuFtVs0s8lH5qRNtMVuN7NxpbrWaoerU0q4EZNR18tL3vSG1j5Sz8RR3KhayGGB4W9i0f5qX\\\\\\\/hSnmSY3S8Itu5\\\\\\\/Fli9F2oq0gsubvgPNdTfyu3C0I2Pdacv9RRokKXdI9VbWH021oNp\\\\\\\/3I1Su5bxBdUjk8aZKGptrmy+QyGr3Gx5y8fbJa9aIrmgj36rFTFggQ10C3yE3lP5JL\\\\\\\/RhqXLbSvkGeu6i+rUPNRfvQbGK4cN2gZY4vCS+67lquLrSACdba3N3b2Q2NnbPbAoJCanIYVkPsy2CZWn5Bq5XIMQknL\\\\\\\/hQsHUeRGFMbqqXsqmwe4BKsZHguYGC\\\\\\\/fn1Piilc\\\\\\\/3I9ses7wGIAT8iWCUzLZ7ASK9iulCPVnsdm37WHkEIVnlVj\\\\\\\/tcrn4DGJhBe7OXM2hx+HgTSJovcHH3K919g3PLRZSmE\\\\\\\/leEusyQ36o8m6x03Ef0fv3taFtW6Ssc8A8pOxy2XP3Ec3Y2PAvlhGVMxim2vombClITO+oisQWmeAZFNfKYCV8nHOtvir7TuxisCb55XCzSHkqNsAkzN9sTX7vTVkYN1887gLyx5NJE\\\\\\\/IOf0HPncuiE4gYy+Lx7VzSb9OVKuQ50gtjMwA2bR7ouTpY\\\\\\\/9jrgxAyuRXbMyLUexKvCHrtPNVJgi1dgmApvZLgHZtBx2P7pHZ4RU4QVVcBtXEmf90Dy1nA8gX8OPpyZZWK7f7vhE6LMBla\\\\\\\/VUFf9gMa+7\\\\\\\/gVhBNSqvLGIJzDqSRJKFD4M92ZhJWDE7tWESgZQTMyObmqyWvifke7s2piJqa7VeAq+zvZAgEU8pDVWRtAqcFHpgmNiT7AL0oCDdn9IWdA3NgDL7wD8GKJoGOC\\\\\\\/8Uk59J8vxHE82sCYV0WQ+HUUmWYxFuPJMyTP0l4bIjBuPH\\\\\\\/\\\\\\\/EYjTD7MSgqfkA35aF92sZismwg7XhV0WVT1835LFLKDsc81KR+g1TcgWsn3lbhqN\\\\\\\/eEqLnIe45lBikT2iZmlKnSyri1\\\\\\\/CWuant6Xh6jLt\\\\\\\/f0uKb9n7q9sp6rYNW7MnhM+6Q2ULDJ4N\\\\\\\/1sjV4knx7X3Tz34iXz\\\\\\\/8DF8v5hhqHIDKSZe1FryEYqYnzo1aCxB99UWI53UM60H2qPZvfvOX6YONXaWfdxDd8ICQdruUtHaITjnwmQiCX0Uhd1qV3WhfGuRJkLGNX3O\\\\\\\/Zs0xNHYj3qyb4OBBU+l9dDP1X1t1YPfN1JRZWOJX6y1xxLsG4nElydDtqHEt7rQXonQ3SpDewN0m1UQIK+8rPJt6v8bFqwjG18i7+d26xJKYyiqyDLcw1cWK9rnG1igTX67ZtHRbAgtTXrv7ISL5TYKfMcYoHyo+FXyapQrYRHHOwnwsJobSh6wiPA2fJrLJaXvWRvfziZrFlx05JP8pszbBrV6YLV8mgl8nSC9twNh0YGr5qZpELFDezG7rzzsNN9zhzjMoi7eEqNrFA4n\\\\\\\/nUQLY7FdBa0bceScUSeMe9Jw855upqQ8giVluzTSiEENGvIqFoGqYImhIyVfwTI\\\\\\\/3gJ+jCQNqwWJXCAfjIp5UmfiXsNttT0foSqhwcxIasBcNmob5eJTebu9Z2PsPIsgtRYB8Vv9FJS7xuva6nXiN1kAIHBEOiQlIGEclLZNmqpZYZCSHyl0BVxRXz23N6btZtiL1YpP2Ug\\\\\\\/9y+JUfZC+kOol\\\\\\\/Ck8mOKTe2brAjvzRABDO7loRDBRNruLpoMGmWCmwZ98YUpMC\\\\\\\/GFQf4XdfHOGMagXK+ptoJks8XGUa0LkO9wWzocY136cMX+HJbv+tkzinwGwJtjQ+7F70m3El\\\\\\\/sCUFlOHSk7XBss\\\\\\\/KwRyxYNLhA6+OzmGQLEIqFXmgBLfnkHs+st51Hr6mLjBlRljt0H0xyeWT0CI7WK9gUtEofkhDocL1+nlKIg27u2BSDCuCDmXRO5TcKIGpsqIJ2SJAzz1ENWGcOpKgZJq33K78N\\\\\\\/Fiz367F6n4wB02+rO6iaM29ovCmTzFOntkdtUI8ULtoWNyBjWnJCUnaMv2kRgaXiP7i59lfBHZTmHTkvhq7\\\\\\\/cIJmiJHaeCuH9y+Z2UGhWXgf0s2kX\\\\\\\/0Ne8nWgT59p5QvIPSejtpCWeQJGWSv6OxV0KzO8W0EQH43Risd1SJAiqCv\\\\\\\/f3Jth3VuVnHL0h6UziT6ksEZv1j4H9hqiVmDCFxXREL1fEX34pPTgdT39WKHNdFbmZoFhw7AOVg19zgr3XlpY2O3HEERMjCSLs\\\\\\\/ABbUDNJNq4HU\\\\\\\/HYaLPqSmW9pbvvqG8n2oFT78d67\\\\\\\/7t7HZ\\\\\\\/oeJDz2Lzpo87fqMK6paDHKfoSQvP9u3HdcSm4ByPBT9JmTt1CuKCn18sO+8IRy7vjOi1IyUwpYk0DDeDf8lkA4nTmB0mN+UbZLEwrNABLf3dEkGnDLfMELP2M\\\\\\\/f\\\\\\\/JAaWkmX5\\\\\\\/TmaQVcYXhdc5yIvZNyxkEPFLqdCz8uWZh4brbc52WBvkOQDXvw71GIMfP3syh97yH2KHuOgTPWGDoCreHh8oyI4yuEJ2fPaAjVAgx3c+Ugei1lL+VOgQhbiDecrv9ZYRqbaJWBcm81n0N8LAxqog6TxgcwbEt3vE2mQa5VU3ER+7NVkMMD\\\\\\\/GkQY0PpxYUBguAxW+Do9dS6Rrbhg7nReKrzsNdtkhUtvU6s0C9YGrTWiQ2tdXdZGxiwlb94gILDqk9+aW3t9J9v1aPu5JL6KZFdsTWqTDIKazAzC1mhwg7NzLk\\\\\\\/K8mmXRUDZlXarJwg3aGkInGhgrL22J6pbprmVGhjomO47hBRudt6ezmVB0DP1riMEe32ZIAIU5B6BXNfwz0w5UE0zSBVfCf6R70ZDgykjRgEQ44P3ddfI+ABLhjlOfR32Uy6V7ZS+F9iyBfS0C6Hz4Jb9eHwHB4l7NUOySFUoEy8aVaBwFRMNWiyXxIW9+KQBjLBOihTjZXqzv348gYBQ+POIXGlBtvy5u+fA0aWB0OgLWhrZpcIcSclMtID63tb\\\\\\\/yxVsU2miy1miMfnuTmEkhbo0lcL+cuN9VTuobmfat9Yo12Y2jtLlQ+kL11d+nfmjGDN1Xz2QYpLA4WEo80WYdFLns\\\\\\\/dD0ag71f8GDAtoaAEgq3GwvcbJXoc56CKLMjb9qZSYAIfMLFXBGC56jXvrWT5hi0RNdUY\\\\\\\/UOBfomJ0vo7yZtB3YO71QLfNs8GDEM1HAenCIGcdCUNWxCfR\\\\\\\/XdBmnUzz688fH1mmXsNTzMrltSQZIGBckWS3ZxookXCqMOVJTIGXL9yXyRiHrDkR0wDVHOd6PtE5vzC4p2TfZmDk6tVL3TIOonCVVQQ\\\\\\\/S\\\\\\\/xk7Yg5iu8tPFXHvv0PlDksuW1+\\\\\\\/xe8sl2gazkmsgWTOnj8KB+LdKH4sYfGAJaLGtwhY\\\\\\\/rQxuPr3DXm66xWH2kNRLG0K9ygY+mEMErSg8XtAWCGqQAk1mhPnNvvKHxnLwOb2bLAC6qjP5ic+bi2CDGuEl+Ec5RbQ5U9YyMYbrNclKEQHEwL04\\\\\\\/0k22mMB\\\\\\\/dj1rxUfVQ9dGWOVbNqoTEE7Yj26fgzEglQKDAjcck4F5oZlYUKhnlRCmtj7d\\\\\\\/Ruvs8U8Zpeo3DROIeLxgo8UAtfc5cQPxQRmb\\\\\\\/2IamhdTyDhPnhwPnDwQVG33fVP4omsDgU+\\\\\\\/u206voeDS0HJQvjalv2fZIhPgYRs0PkxYx1EpWL0gv+wOOKMz1JoFcawGA\\\\\\\/nwQK+53Eb99hSdIXDwp0gWs1f+dITvvHbhyog\\\\\\\/F7F7xtyE4Kr1nPiEGE6pfRbWoGM5BzC1pXeJ63j7PGXlM0tuV+AFmITGNWbFY6eec42AHN65NmK1mMogjBWRXYSs6FcYiqztYvDrNBe92i3iigGvrpeiLHuOUxu5BWDbcQBBmZ5OYf67c7Lvl7TELjolmjYYAsodm0OK8KpQlB1+AqU27X\\\\\\\/KvKOdPUi\\\\\\\/rweg\\\\\\\/NNsNdwb2mc+u1kVRoK+4a5fEYpjnc4q48cc42+\\\\\\\/3yOFI+QTVAk0DpLEYVn+t5zTPMI3msAf03odLWuEnNT4kqQgQrRDhndiojJR2zW4TawMstC3tCqIOu3OQhTSP6g9B50NymwEsspZcJ8tpm5Mey3xi2rSlbrUEh8Vzm+dj9ZmGJx1cWflQRdVxNw0JBPW\\\\\\\/CbkXuTJIyWbyYU7VUKF259mflPucVFzmgmBqnmsmAveJWR5EZg8TCNPBLh0DC3GO7cDZBhylYpZmjV8Q0HZZHV1Ewwz5TdVtdUnGBlVOBUmDLPuwfEWG7\\\\\\\/a9vUwy14DxOi5HW\\\\\\\/U0NX7D9\\\\\\\/KwMXxf77yHm0Z6+a9PMlKFpRpc9C4Z\\\\\\\/lscG7Eve7\\\\\\\/KbJHZ2g0DO+qHyi6+DRzngK07mEYpGuJSzhj44aKikRquJ9+1M4rc\\\\\\\/EAhsb6PeaQgnA1O8co7HhiB5dfs8Ug3cY12MYe4KcLX3hqmjz5bHX7hMuxlYdGNSbxt3Un1GjAkdyoe4lAqXmHXsxZhnpTsv4AqJ+xbKqIMtgqnpeMPw3VQtNko5D5kavu4rtKWK4HuO1YgYe69Y7RvMmKLvxViF6xGL1ISK6d7VIOfZJAFssgOCPQyCsww0kWQpoonA53mjN\\\\\\\/2975sWvDvQkKXYym2OIBUdbMyOr9+LSVOSxCAQub8N9cxEvEzDP5kOfBTtRGRDrK9NdilfBJEQSajTBhtSdjJX1eyZkPH2jdv2ga+evvp2JhBBA\\\\\\\/OLwCv6l6goml6RjfeJD0ptLsT7bqfUsJN3tYyRZwgyC80o9OFHwDoas1pmGKDUUGq\\\\\\\/6FWlxcJm56bQQBioe+RZBFl7O7s3RU\\\\\\\/u7rGHB1CZceA0tgSKB2Rnu1NnrOYtZwy\\\\\\\/1uNuB5qW0mmbDtQtZKiLQcE8003TzW541wl76kgqJ8NNg59mjSjZeimesZoK1uApnWL71UUXbLRU2D3USOSw4DCx1+hDjZIrF7AuWh4attjvnHi7T3DWSr5CqYigIQJ9dXhPo7Cgfu0eQtk9DOnbDDnoxrV8wd29azauK7nyWaDlGSHnue8IEej5F0lGa8vVKkLEXpgqyKOjN6guiCRE7FA9UmFMcFMmEXyMWl28M6hwofYwd7RZzCuTP28X7C7OUjnCZfwXAyAsPt6jbLHP0+0f4Ij3NrEWI9G2ne66K26rR3CBSHmuDAzzNggcyZjGTLxr3xsLDQj3HIbE+SLgn64ciQTZqgy+DzsX2fv3D3F2wjeFsLcPSOXKeFoPK7SlghNseT+YPzTLX5UhYeTuGDZvsuUEUzPSL5YL5V\\\\\\\/Vm5hqju2y0c46xy0h9Rq3NmfMBHhCBJRTXc8wQDZ\\\\\\\/qbyUgCxFG8PhmV1IGvqOSVWXw5L5okKrdx8QK1hkfG7FMib1tNE61fHFaiBQOZ4REjBJ6kraVgYKXjp\\\\\\\/x4nquRYMrknA7UI+1fiXxHdG6o\\\\\\\/BYGBX9n6wQqiUlctfINKRuv7V3p6ex\\\\\\\/Ap\\\\\\\/VgrD\\\\\\\/G7n+TiADGTgkHNWsTy7dqH0wDO7Ir4M\\\\\\\/a0mt3iVgp5eFl9SbSTyf5PeOg5hI0r+rbqmNmP7W2gskLXzA+xC3tpkDlg0NTYV7osUg9ftdN9bFsKQJdn0zEXvEf5Xt+WBqLBzf0Y6gXPtmUtBFT1mUJSCW+Kjrfg8+VyoNSAb6832fvoypMM\\\\\\\/+uuCMNHRuxIDfEpqePnYU5BytIk2t0sNA1s5vJs+dx4Q8w++IgRhOWEsh0xOkhFaxLgLWJuuTqTb9w\\\\\\\/9lsZZHgWt\\\\\\\/w3XUT\\\\\\\/8StuUs7fniHWVgcPLuxQa15rFPaPje3iunCyh4tSipTPotZ8zrfxXJNRtlEDunJtBnQqYoKkNTyqOEpPFGdb7kp7w6vNFTdpn5OMUUoHPQEU0tREmtcCmFfd94tkueP\\\\\\\/aDonWgXIVd0vz6F6K7H6cXrJl1fjy0Gox\\\\\\\/aCR3g3N9lvkk2jpgaLJS5GTfWV2CO\\\\\\\/KijLu5nn8sfu4J2EM2E\\\\\\\/dwR76umuSDgqfk8P8nRH+QHdQcbx8mJ9QEKkS4mQtOgYqPSy4mcS+Ha00HCfqYu786iGwsF5o6zYQJTWLDvajtM0OcL80e8CDX83EQCjfZZOaKLGOcgXH0fkXvYNVYjDvohoB7haJolm1OTpBLj7easrkE044SUOP75OF3XtX+9iCFGRwJFr7RRlM\\\\\\\/0HgWAjT0CqCLZTqVnyN\\\\\\\/ab8U\\\\\\\/SFIEomWUaKVe3z2EnlqAc1BEUTgqp68yP8i8wBdRYfe\\\\\\\/266gL6gLxtXg3l4MXtwV8UhSZfyGRuzryLMwoW1gcj4hbY39YrP8fK1c6fyJa2S3C8e3nrZDQgMjFAc2ooq4hGtg0yPSwFJ8yHqk9YoYRUUpLq4+r17SwPdhEwu3vlIOYvZLzy3wqrIG9v4RBu4TAvRueWn8CCbp7cdUWriGhbHQM0GWv+4OmY4jbEyDpPIVsIDo1EccLXotbt3hbzVpTUFhDfB5oRN+4q7tQOIzNnz9OhGzNwu\\\\\\\/n+Tcp6c7eugkL3yzR\\\\\\\/5l0y93rtovzbJaXsuWcjIKIaDOpmrTt+VjT8eEWLIQEmITDW4AVc8gLQL65MksfPHa1HDjSY7gYc4cO4DuZSePG9qOT1Tyvd1brq5iYbJokgHqPR5aCVXjxouhrhhTFmZlbr3xOw\\\\\\\/mWZaoU59aRw+fhBQYtPljuwcKsFKMue0\\\\\\\/uJjC7fOyryPbbTnitxtgwsVWcgXYZxq4IUfyyQic5xryKvyg89Ah6IVo0JIH5MnEydBFZVzY\\\\\\\/+WrccG1kLuo5Nnl7Wa2N2c78JLsyeiGQco0f+cZrLLuZdDqkzglH8KkVqIwf+K3KbaI6\\\\\\\/vGn70iLb+qWVZrMGb6V0U4caCG7smLLwu+IDzHHBV2xt6VGl0Tb\\\\\\\/KU6oHYVKB6Xz+CIbhofT1RhvfWbRQ7ByXN8o7G01gZM+zgJTFTQa8rMmos8gcTkfr3e8z3Ak9JHcPjG1JGN\\\\\\\/ISq5PR6FFJ\\\\\\\/HKY7\\\\\\\/r8zQIVwE3HvtkUe1MdgK1tVUjMash9UZo1F++N1ml020nUVpJBlFm21aKDh4LjsMUWWLVaWmPOICJoV112g+WsSJiJ8wTEz6tCNfBMX31TiCRzJ1VE7X0qGjRd2t+t6\\\\\\\/e8pXOgqpYyKFxg9pLnAvQDg7INZ0Rks3uZu9JBnvC2w6UfAWEKgvEI6feDmLlcsM8jFqOlQN6PG86jnaHuJVqojtvlBETVtKmfDOEzQ10W+o3dt\\\\\\\/LaH5igYgoK\\\\\\\/o4eNE6Bs\\\\\\\/5w1Ekw\\\\\\\/KSsQs2TXroJG9oDEvmeWbCIFKMZ6SuY3H8CAaGEj6j9HqtMK2db9ixuPFE3DzTBk8UfdI2+CzCu\\\\\\\/yJ7a0vSRZeVKoigpFKfNB9bSHDZrdZgy1idYQoklzSxvfW6fD6gJ1lqQ8jO0W\\\\\\\/IytVoXkp3whFjZmsvLkmEd1irg3OWrrnjJQmag8x1hFjUjC\\\\\\\/d4+eIPBBTLMIZo35ehavsY3q2bqc\\\\\\\/tsm0YvkhGfIgDnFXYDthBhXC7id9K8inzPQJ4AX1GrFQrYiqGuz+RZrhhEPEHUlHTRGVFVGo8IM0PqydC+S510TM1lWg66oOqveLR4B58iI0NEEtlnv9FPJOxJYKwtvOb9j7MVi5dffUpSvKK9jB9YsgXofH5ZqRX8DQB9GdYTy4g+Nsabwh67yq+ahVSInPLG8rKtJ0X1uML+yNnd42zNTyQglaUdT1QxXSMxMx8hYcEVHcD5Zdthgz6nM2l69TG32H7PanB6lHWJJ9hwgR1QyGR3kbRVCCnIE\\\\\\\/tfTZZwWW2Ws+HRzYoDvaWhBY6NagKmt6rxrkJub2T1S+idGBfwfEDlfozrRxK3\\\\\\\/l2I1SddN5y9LH5x6ifV7\\\\\\\/c+eVXmHAPAxDgih5t7856zrNNdei5V0rSF8BYGJPIEig+PY85o7eA6wXWrPtNalVO6korFVbFN\\\\\\\/UlYryPlkMw+5u\\\\\\\/zDnQ8+Llz6hI6D+h\\\\\\\/ZgVznAy37LJTv\\\\\\\/t\\\\\\\/c5SkOzD3Z0A6Zu4y3ND\\\\\\\/LRuS3K1ygRTbp5LodWNHUdZWvKflvY95LWvvq7utoejtNurO2U+x+q4WvbPjZaMNWZNUKZkUf5tGNmzFpVmfDDctaSCGd3+S+gIkIsN7v\\\\\\\/xiftA1Wtq819cYjOYBtCYTPSxcIJmpiql0D+xCwFFrl5c3S6xC2STJhvHhOwWK\\\\\\\/R\\\\\\\/NlBJH5sM9cHTIxlCY147ZMQWpKID2pnOcH4ZwSEATn1xxzcaxDFaRdpLFBPl5RDdPMkvFe5enMsj1xdCztTW\\\\\\\/hJgnjSbCHkjrye7r2vERJyWPq3qSHEqxQFH2FEsgkDdlNz9eYH1ALWzh3krwqIGQnEJlugubD969za3R9QuvlqV\\\\\\\/e42sIxVqHS0p6L2C\\\\\\\/Ot4ejBzHl77VfE9wG2iDoalp7V+wLDWQ+Xzy8LpJ3PZRMWWmPmGtxrJ+Zw0br9DMRZiIPZKEq3L3px+s+VHJM6v5L4OonqQM0iSVlgzZyRkUPwH9kKF3IDR4ZFs20EbMsS+ikv7zYKMRJbDz8T+ErNi83yC72CzFMXxAm2aAjfnDY88jLLEWY4FemwD5y0\\\\\\\/RTyvUnYgd6JaPpEggxBFNatQFJxSGZMt+TwtyraD78KZ4bSf1U9fj51uAMvzbZi+NCbCCcVYyBvU\\\\\\\/Y1B03ayrF5\\\\\\\/L0XxdxmIlsrpu+TYSQTKujeyhmcKJu4gEbR8ENMuQacKcDWSxXqhU5hSaCP9nAqKUeWzIK1C1ZE88a+dua8eZ3kvI3HkiXy7\\\\\\\/punyi8rVduoOOKnyNR\\\\\\\/lAlbIUvnZ+4hOiP2pIdlxSDK\\\\\\\/Ay9\\\\\\\/XiqqqiGbFTW8KpJbGFNCQfZqTNiaamkkoZm67hcqpACRQ4nJj0VerCTlsuo2K0y+KMWO5J+ut5TjkNq2qWmj6ktDNmSWylojGFPPBvlNMtN7njBXnQX2Qumm7bGQWIpK3T6wvhZPb7zYAQ3M5x3tXTRDYiklRzBPzO9OW\\\\\\\/7f1Xj7XMApGNeYjfualeA+\\\\\\\/4UaBBdXLb8uHDGskM2eaOcozNg5\\\\\\\/wRLF3\\\\\\\/QU2oZGdxfvrydqeCdnCHQOXPebZD2swLZaS8tOsLfOXHsz2bPoGsgxavyn\\\\\\\/eCYetXrwJMBE9leUO2sQMyzB13sb7h5PDep49+xhzARMyfRWPRyw42cUOKAQQMjPDOC4K9ksNqNtqv3pjJEN\\\\\\\/Y0pFgQg1KtkLZsL2WJ3gBElFHsLRno7DErJsqzHq5EuHwpuXwQ8ZFtHuIXXXTQ9X8HLjQaRSOc65CtFIwaEAadNm36Nx9gtDdE\\\\\\\/nG4jpuoQIloYsNMDEBy8v+4xmu9UK+b913adZAbOS8yKQbYm9ZNyFnM8io9RStJcPQ6ZeUUqfZp2ys6ukN3OEOLfopeOO\\\\\\\/y\\\\\\\/bMXCvEmfkxQV31BT2Z+RyfZnGoyMj94RDgZ0Bnmklmiw2zADc222WngLjxZUAqlj\\\\\\\/8h9VcukV20JLXHVw7ukxWG3pPDREqdj54ZvAWgo4eBAHnHkVTUCWaCGWvdIpevysbNAahbypzf8xAHSkDXTLMElINEs\\\\\\\/5p38SRbjoxXcQmheWznpvwe9W3ba6IkJjHikTE5Or+xJ7vQxc7Kb8TW4Dv2x+JMDCz8eSrVJWWev06E1zF8wMl3UnhIBoG2bnZ3B3QSo+BNhBJRq1lXbUtoHc3O0L53JZjshe+btw5tsW00KBp0nPZBV4ElCqNcIbysUb2ePhz6keTy8vwA9KRxbicTHLBAIeJBxQVn2uliTQOwU6uzVEskWHj5NnHx3Fv9Sd7XDcxDhkSnU0JayZCvfPtn4QwqJ3hCm7n8FZbO\\\\\\\/zEkcp1AeD7\\\\\\\/WbrnYUQi\\\\\\\/ZVM4l66h7zPk7aSdDCWBEGWk+trcHKDUaspgkWYovh8bH+TwFj1NJ+6RKhvq3r+tC+Pj88x6P\\\\\\\/fMoJ4XzEBEIu9q2tJTOorm5bRZm6wwR\\\\\\\/pj3laiLJ11UjUMs5c6oKNebQ5Hqcfx30\\\\\\\/DTye0NB4ze3W+Y0WmU+G9pxo0cJ3RqEgNtHDADdrNa8OHZCYO2U7+L30aB5oaeWrJGzKylPijgFU6AqtI6Gms2yJDCVMunQChVfkkSen1VTwjRHu9syq31ttGJTLVr\\\\\\\/yftxR\\\\\\\/VHKGO9cjM0DCLzBSDNyYb+tkfdUhIWtslmotyhc8WOeX7Fv\\\\\\\/rSts+FBzgrZsds8++db18iMLNBXXiWdqGxYtLUO9NTPygoNACEuccZHkeidDsw6mk62oDp5RUoh+QReqalGjN4kPPMJMB8DFg1qK21tEGKkY4iFbW+WTzJdTXGNGZkdskIkK68q0o8EkE289SRRjoZ4Rb3SRiC5A30dXCDgAK7pbrX9NkwfN3qIzK1VMtWHBmPErJL0wU7qJlQufD6zP53fBTp8Az7TUlPa27bfxFb5yVU2fPnDNbIU8+mOHqb6fZ3nOKK6y8qDAATwPzT+dVFCBxiOBF04+RMLTg5EA0DnTR\\\\\\\/hRLGb2I78is0q2u41Pn7d8ClII8MHhU76jdPe0ENJ3+eIyFHW6RwySy6dzpflr5Bki4fAzvsWJohYkA4eRYB6PQQysAiqAFu2NLNTjVxGHEeWapJ8ARQMMmHa80lIBxPoctmV7yYTx2jAaO7SPsNusF\\\\\\\/WyYiJ+5jlZmRtpX9zYuhBvxeiS\\\\\\\/iyPiDUSWxGejNGsplFKlwwlwfMI\\\\\\\/dpp7\\\\\\\/CQ9J86WMoHc62DzRNedoW3WgA9wieTsr5SEf7mZwxV1z5mcmps8xDGZTaU5VKYDQ4G16iGLVsXCektVhI8cHu3ldBt2MEI6aVQqfq\\\\\\\/P068nSpv2GmqT3cpUkJL7rb2QxA1IO7g4nJbJuau\\\\\\\/2Q4UhHGF8ddrgrEdF0wQx96SsijcqrLrfAEFpWTpv8v\\\\\\\/qovX1q8Qws7p\\\\\\\/91MIXmh9+kZiTGkny4k6R4RiaKodbiURajzqffuk7R0\\\\\\\/P3W2OBaaCks70it1IxqvZj49Ix\\\\\\\/A1sZZzWER2GIx4rGfB89rFcdS6ZHLbcgf9IbE92+7ByWzGourgWgrc6eWvZ8u1WOqgH+OivOrUwHG0RfdRnCkMNf1nWxHWwEsYFuSjJv07zfJZUUDRYXh9MPbJoICX5oVnjd9RCDapNhPtCZLCGJ+u\\\\\\\/iiAAdDwDw1KIxiKKGAEqSeBA06dFjtqsISybl2BkLv2CQTpGMqI6QL8MrhjhZRye71ceEEINjEYhdYnQ0GUI27Sxa2ptl5VmLuoh5bgqsbT9YjYaIb7WkWfSsMilJAEemtHLE7XRr5n3hMI73\\\\\\\/ZDeN4ECcgjkMsDosa3H\\\\\\\/W0fqzWpPAyBl1aOXdvnL7pID64Xe8ULmDkv1jmCGAEG8OUaZf6gS0ePNIdi3xxJjvHQ3mV\\\\\\\/Vbhol69oZVeMsiHmrbLgJIohO6EbQCCxK+iPzxMeO8dGFzT6nDa10QNziEOKFfQJs7z461VJPYqhcJG63NPaF6L0\\\\\\\/lS4LuW1vUqdPoi0WB9zuMrmOm7wCUsH7Mh+Y1EE0wq5GZ1GlnaAAT6OSdzCHJzmM1C\\\\\\\/IeqS8IOwcFLMsloPzFuMSU9Mrt3DbM5w5VlKOC9y5HLBn\\\\\\\/9737fCVyWUlhARPeiazubarV89wrVtGbBH7Wxh3LwABnRhImTIVKs3HZHGNBQoB0Jpyn+\\\\\\\/zRbxW7uTK1hj42N8myrDLegcMD3w4r7YtGlBJzBygIveUxmZFRZ59dXdp6TH4hfj3V0t+vSD3GkGouCuv+y4xK7lBf\\\\\\\/7O1ol445rEmfs6A3N8STn9Eid35Cf+7QJrPQ+mvGgu+jf9p7RsxnwHSSFKDPCp5j2am9A3c9swU7lqDyKpsGZgcbVdALPUGEcEc6fmEQ91rS7rNvMQqCAofLJY7ZnQeCs4cbJZA7wzUGYwqSGUZywqH2Ce5R0DUkDIZjtGO\\\\\\\/DfAUxDKcUgRsxm9E2o\\\\\\\/WPWW5aOJUZ\\\\\\\/5zm1zSzyK4Gr1Z8miNWsCyNolREKFDtntBIwF+Tcrp9q6arc7DQhXeHmRZNb9R5JyiqvCCZdXwo08XRobdlD34ASsMlUnLH2RlHb6qQr39AqlIvG6oDlAVnvNT\\\\\\\/6XsaQZb7K9OeU0cDUFUr5exVBvsimlnVTNzVcW3Nbbsq\\\\\\\/hCAVzfpzZvN4JxEEJ4n56VCppBRMgEvbamw4XE86cuIQWvZhwH+31UFvxZIt0Mhk9q9gUunKH6rtpnkI1q6eoBk9NlVf\\\\\\\/7kRXIDwwpcazef5SX6JGgYYq6ZEecgrGNQleBzETvxstcM6qMzX8WMTDxFFobmi3DPCbBgWw21Pi8y0EfpREMM2k31bTWfqFkHlf5FZD3PUwtVfslIGV0U8L1fu6n9pdr6+Pp0nlRPhNnu9UrovprHwTLeIYv+Q+LLhejWJOaHkylI7bjF3aez5a6IFCjaq8Z8Y8lczGD8mftWxsF+9LYIx2+viJBAjJ63DhUZUvQcKMkFBX40xhi5RDDzTmzSvXUoLs54CMQfdINC0dNPCw9QC0JnVvkgW8mN9zNkAO3s4rTOVe\\\\\\\/R7PWQKIOGQILMC9IfGIWnBQVfG0o0WAnt0ma5CM1BG\\\\\\\/+7KOa8o7h9bnnhMbffz9P2KXwungzzcbpreLh0qkG3SRdY7Hl0loiBljUIONq95tU83mXUkUhVbfHmuw63CMDU\\\\\\\/SWlFGNCAYEWQCkhvB78PcwQ9vrVvHK7HrbkR1DQpFIUs1DnDP6Lsy9Kkp0QooS4WUATLPMkl6GpXaxGmUV2g2pAyI5MD+z5n9kAiK5QwwJVRw32z\\\\\\\/a7ioT5IQ18xwzOTIB1e20tnNuxQB8\\\\\\\/0OYmW379hGbXaNF3vqd5qWwyXWNtZQ4IGWSfCUV5MHVdZyxPK3uRviKpauUJRtSyDAqy1Y3+IvBK\\\\\\\/BAtv6R73CIm2zVklgKh6G1FFgg7M\\\\\\\/HwnqmF3zc0w3fNBkPDPBxuRF7eFKJ6p2unJtCBsvX62xLc+sIBFJR0uTw9iXf8FBfmaYArBMcX1la\\\\\\\/bMyLIIySjk91ZL8EA3hsddf7TABg8JO9G7xS1m9S4hoIUOIL3Ayz+FpPUwfnMqZf8NKLdnljOEJTWJQ7w4VnoFdPVtc+pmr4d+rOpsZSMBNBgDmptLoiuuuK9Flc7fNZooMY0DCBcO1i5KKfssWEQWmnMD1wI5KFQrG4NE\\\\\\\/KXMeyHJ5sWKMfKuSpgdzxpC2Lq2iAYoVXaY79gz8Yph63RvZ09rvTIKetrfKuyUdn5\\\\\\\/TE9\\\\\\\/xI43Cz6GR2odW9ixXCOHzU\\\\\\\/0b7WAEZSLcYzrVGrhyYaTeWUnhIeCRJ1xiSHqRQ3qfxy0V5cR5rlkkHrikrtGErbRxWY8S+lENmICLNFhLVwokHz83b7ggk9r+78Rk4zFaGN1lqkrA\\\\\\\/wJ0Rz4jDiLsgjKgnTQK9mgoIDMty+QyANld4few3ivfvHx1SBPUY4vX5DQxg0LguahP9LddNrd4w8xslPfaRB0GECr4\\\\\\\/li9c+Xp6pBf5RLLXmmmVZfxRZ+U\\\\\\\/mirn6qvPCExq4B0zfMhAdJr5OROPRKEodwdmH5VK\\\\\\\/+hMpKQpq5s7huxcR0WupVirh3LERdjpfZpknMoRcfeBr9\\\\\\\/\\\\\\\/B5z3T862MHaev\\\\\\\/Ac\\\\\\\/cXwg90f718DoS9+WcBSJUEKl8xsHUfjPSnohn5fcImkg7de3+TFXAYB2qvbZ72j9DL2eml3eKefXhYoS3xLbLolr+XZPxnxvDp8l29tDM7SFWC+dr2M\\\\\\\/pqcEQnoT+kBpyg\\\\\\\/VDIBCe0yr9Y8uDwFmXHMjWplOmf1wFCIr+EQrtYklgJUfOtzKqWnqxmVluMmiK28dRl24H0aLiog91bsZVlA2+JMgQQRS5wkizba\\\\\\\/+4nh8XvqB\\\\\\\/OKELj5aL2delUF2yBVn9QsIhPTtTUlREDVyEOrrDlva2wJq2mxBKziRFBGylng9HQ51OQhRkHk5PMikvHRGK2FpRcbCpWfMd+u\\\\\\\/FO+FaEDlE6NynbmyZHXOX0YbmoIAFJ2igAra85HIrFjYYfosIzTOooS5dbzEEJ1rEx9qCp\\\\\\\/pNdFFsyqCZa1L26SZEuZ99AdwyhoQbo01CP+7oQ7sJZhyCLXR50aFU+K8Aew8rnMzHhHFLh7G+se9XNwYOSjJfQEgfUeKHGrG8sH4Xw2NYvFAknzeipfcMN8V+LsEsYNBdQy02yME0LAYAdgu71HIi\\\\\\\/Yanp8moR3woXEJghi7OvjRsb\\\\\\\/1bveQfMNgOpygRH4BHj9wcPrnpVjKQNtf4TFTPQmDumo\\\\\\\/PJxG3s+z8gpsePDMY2dtRFkhU67IBHBP8mK3n2iawuXjZYEpB0t6EmjzvaRftSRTBUpFpOEDZUfE\\\\\\\/NmYxxjHR4ysX5f1CcS\\\\\\\/tQ6PJmnl6Q7nhmUNy3lhLUZ8egin8g1xg8JC\\\\\\\/CIgTATyRtbjNQv1ktOU4cQbsANMKeis94M\\\\\\\/l3qS6kwfkQ39XMbSV54JieeSHKoZu9kfiXeb5esZuPA46ybHe05W2cg7F8TJEIBjXlhAwnk2z+LHRZYkXyUlgqvjfuMbfy3Pee4pZsM1fGr7wlzWbvLm8o+BBZ+CfBK90WQ3iyTiboKsdejBFBtdMUP\\\\\\\/xS7WKsBF\\\\\\\/2iQI7IYpgT\\\\\\\/+JmPY\\\\\\\/4eTQT1F\\\\\\\/iRBC5xWM6xSOHNqgLVLRA5Gm4COEJglvOLCpBSnn75czWFtkz7JnOKX7hWIk5YU2IYZs+2P+\\\\\\\/slv4gFDaxERdkwk4jpMG8NHjk8kEqxgX1q5CDNyMZHOoC3XPc+CNR9ct0ibEaIzkHeWW8MhpaPQK08sssE\\\\\\\/Bi6U2NCxIUls+yxs9bHpsteblB707KKfV17TwA\\\\\\\/sIGBH3SSmpsZDfcxDUYpEX6yZbYWryhQhfzHEnCXjy5Fo1coCDW+A2LVfeE9QnI8Hb2vZxfpOqTxSxV5bj4YopmV7yDSq3IyHKyneXjY1iM0BLqX6fHi6UlUkyJgPKeHDsdw3g1H7UgRY+696Q\\\\\\\/E2W4Ymj2sEec+E3TVDzg0wl71\\\\\\\/rcmrLAyov+\\\\\\\/9PfWwzZQpB3UJaclbVtJLwklO6WMPW\\\\\\\/f60dprRy\\\\\\\/VZ5gJ9J8oro+TkCUwFrNZTf5EFxHl4hSGhwNFsjOobIuwpjRYX6ufgC2zzlebSt77z6AZPdLHKRxYmtQwjplGIuBtLmuSwHix6Rv6px2c7KSzWRX0rq4YSjutoW6mFQ84sO4w6814HNMlAsjLDo0jEHm1Y2bpmVeSHy8ZsykcRc8jWNui3G\\\\\\\/j1b5mpQmz6eRuGD3fo71Nbe2zrGsTJCVZ8G5AJbgSyCvboJS8d9pKqDwcB5OVGkTp6pSOC6EvElhq4P9ftx5Zp3MNSGhnlomzKszW6ULCbX\\\\\\\/qZSCq9Z9wltN5OJjH9FI1tYFJ2gFhfDbaaq0lJx+MU+scf9EBtvXuo8GwRIiFlexQsXmic8Y5gPfdqD4EX+Kr0rNYcTraEbUczylICirpUHc2k6V7b2S5hJNyeWdWS47NjT7dbyaLYvMAbjL\\\\\\\/HBBlq2\\\\\\\/WthMDieBc244ReOFQgEctOmIMZNUXtGIdD0fKYzuPJKbYvO\\\\\\\/dEMDAC0GizMajG4+PoTWN4cYocLYgEoLrk50BA1Tv\\\\\\\/mrWizygs4FJl9uY3WNECCiA12tIhXCA0FmMO\\\\\\\/p6x9x6gRnadlT7Wdc4\\\\\\\/waEG3jw16SXGGgd+M1RkimdbDSWgqRJfFeaJqByePxTqa6jNdP8HZdqSqVy0dn3IxD7eh4nNjLGf\\\\\\\/t7TtbYT6MJKG8v\\\\\\\/rc43LIdPAxIpwWERASNPRXBDNXVFS6dY9GU6\\\\\\\/DBubPV9w49glWwyiJk+axdeh5UNGfrec7aBM1DMI8avjl0b1WOiFK16\\\\\\\/I0gAJ1XCG2d0jGFKiAwJUCzyjpDAHJI7BeQK62pC8aKLxgAFkBtVNblOL1x4VIIYBqwfsg3FFwFTewZoyVqpBk07zjM08YiehKkDZ9ajhttdiKYUT\\\\\\\/QO2ntRW5VKTst6\\\\\\\/waTP6eIr9Gsszk+zLQwpkxvwJlvFnJfAw5dzGc1X5ZQCEBSweRv3wnfuiZ1z7\\\\\\\/0ch4y\\\\\\\/voojQJmn3ROiRJrYI7fTZIK\\\\\\\/eQQJ+nXpCV7PHWw58mfOR3D7cp\\\\\\\/1QIaW4Wa\\\\\\\/UBUa0Agak\\\\\\\/kLj6NJYPAmEk9RWIxlWb13nrSwnII5uVPeRFSYxnu37cfFaFfsDhenz8LyTPJlLWAejlEg6N8eM0XUlQg1qYwsY0630A3Ozq4DjxH0m8FrF77ZPntjihvQQSngF38hA4l3HU\\\\\\\/jA31jCEVGlQOCwuspdP9n16a\\\\\\\/sDZaKfls0oFdpC04YS1BPY7xnyLXhVfzjYPL+OZUBK0beE9ZhdDBxcN2lUEn\\\\\\\/GnA\\\\\\\/HVUB3frlLeN41dwq\\\\\\\/BSnAIC5qu\\\\\\\/H0wTdix2EzRiwHmfmXCswFjy2WowAPkDgOuouB780HFL8QfTGkJNjf1rY1T600xHlO1Z3H548zhz+PqVJ25gVXhYosS\\\\\\\/6cIV07zAc30+H0hWKXh\\\\\\\/lKfjAwgo\\\\\\\/QCWgFW3QC7xHLVPUcFjjJk2gABlPpjI2DCESg6K3bd7ZuIdhitdnl0+A2oPLq45AHuf9FbYB\\\\\\\/5PLelk29gjX1I3huZWaPFiIbGdkaNJ0e4jkVu2yiY2LhfsZuy3eoXus7CQc\\\\\\\/+gEXlkphVVjeFP+kGMeipxWS5wzSwYCYrtEHMQQDx2nWWGdfBgBdbQWWUF+vepVjjFLiWlLrcaD4JDunsqN4xML1bcQx9+32c+0wTKCAblIfb3QJ9kPor\\\\\\\/GmXBmA+OcwxBxtStzreCGmpgKQx7KT0EjEaK2LbjtYUPVyPEUhk5JJ5nh6cg35ic0SFjHD00xkik2T2k0zazUaJnJxMRlOU\\\\\\\/RdtozdGDi4icrfkcnbrwavIWTJ1L2pbOGunApXjv+1RGkWSCdxbMAfviZOs1LE2YR1jvFTUv4PgGf8BdZsW8g47tvFBNxiVAnDXngnPgiSrhxtTQ92xPxlXq2kl4MJrRKPiAnok5aIyHY5TcaJXhuyqEWdUAlQjb6bRhFiYpKl898yCgygtEJyT+tE8sqR02k7qxQJQJGR08HDUnHtK4Viod5GhqgRwOar9Jx8nwb97XTf0i76pU14Dnbz7ls5LOMGQzRVNta3\\\\\\\/n\\\\\\\/gnpCPXpm6LmZ3+cZ76E7gVsLhYKjj969Mr+tdWK9f\\\\\\\/vkr0KUZq5q+9eUDAgPheW19YM4wFuMCnifRtNBUF6NHi4mgv\\\\\\\/1x88tNgtYcC2AvMl523gxrwIiZSexPnu9wAzePjCunRNUotpVko07RlOecyu3uzZyLibk2tq6F7tRgh9m7w\\\\\\\/ueMw0w8Nw7ZjKnHClk3FAWu3sQQgdCwXjuQZZItUrifRqDz0XiJ94OeZNP7Kj0jO6wrvrluOvCIVDlvjBPkspArTla8UwLoQWsj+eSGl6yhF\\\\\\\/WJCv0cT9cQR+\\\\\\\/HzvbMRxc7uYVC\\\\\\\/8ImEx46vctgc3ecTBIQy2cnJifptul6nNWaU8gCcEu1nNMKI+QIXB+ojjURbkWOpodVwVgttSP6jYtIxr9CQOg9iyc2xE1vIrpS6y1tDyclhFubfkHMl2FUK3bBIiYUbEcTwtGSBtuDLdO1ELpm3WMqoDt3Q5lWVdRY1QYD6UOND4HuU\\\\\\\/3UMVEALrTCCqq53wKEo84CKYUVf61mNYNg2Pg6LUTxDq\\\\\\\/YSBii0kVGf1mTqnzX6\\\\\\\/kQDA1SfySjNxPIHOmU77NArecLQrAKgjqQsSt3268\\\\\\\/qofFOUa+AuQ4HA\\\\\\\/0SWa0FF8mSBPBMKFm\\\\\\\/Aw9f90DpeSgK9LG7Z8OKPZygNlyKnaxHohfaNfQV+0DXzp7IDI0rnOh99XgXvOrJnr1JOa9w8clx1gzhfsSoaqD0OWD4GWoQj8HJaxPKkwWqhoRb1C7\\\\\\\/+6kpu4ABnv+PsA+lbpdnZQCGbuTJjfw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lQ6bsVvzaxC8z7j5E2Fjyw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e87404e0138bd1fe1b3cf8a47233848ef26ec69f56b005c1a8c2af40931ad957\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ca5h2dmAItgFx9e1SjJUF1SUFngEHM3qqIOFoR3PT2+v6mEJe2LO1UfGPpggUc0lOAyKEAUwSBOA\\\\\\\/wlDe9POSpFJHbZvE5f1NYJWRx5CCjrd4rhLT+q5pG2pXsmWaW\\\\\\\/D\\\\\\\/g+XNGBX4vckIl0smhLRXOBUddymWGsgZia3dmLz2yNARnzzFBwJ0VrfAGVTT7li+qXE6L2Q2bvmmq6auDmflleFyqzmZYwus+OP64aRKTLEWlfJr1IwHZgw\\\\\\\/H4X2bq7WVqXH3T0KMq4LGn1Q0pbE3ADA2EgYv1KTMgJromdTccxNi6wAO6kgjdX2RYa6N0tZDRS5+V5FMPC2LyZl\\\\\\\/xerfol3dGjLkA5mcJQ\\\\\\\/r8XZ8XivKqhHO1K1EwYQQDfmCN0U2Z8WIIWguW+qFugeXcrn3xcvMqCqr4+LB88SE4q3h\\\\\\\/mJJrS9GVGnH5czJcQ9VcLvWJOwBuAN\\\\\\\/URpiDmg\\\\\\\/Ja5Iy6SIHouoHkZbauydSLhOGYiqCsOoznLg5VwVwwhbfb9QqgBu6OMiwAtCtrhdTMwJR+S30ybUP3Cj1FLt5a8pnb+nqHeNxT8HYTQSInFRwisJAKJU3S\\\\\\\/7M5U+Imgr6iY6ypAhSrE2\\\\\\\/ladnhKgdBwLte3NsiFKYF7Lcjker1iJ9yFcd53smejQw5GCxZFOHKqpV2QfuykQ16fGo60xzip+4DnHNTtfuZ5IgtkyKg2PCbfMujmid1247Ll8DOD0MosaZzJRDVlHmbFpKfB\\\\\\\/jb4AmCBUXTVSZoOhaDkxD+i1D5vUR\\\\\\\/wZH8MkDk9ok3ZqsRWtGS0JmIj837lnT7hV8TI7uYVqjN\\\\\\\/\\\\\\\/s\\\\\\\/joHo81qGnEpWykvh4G3PzLFoCJu9\\\\\\\/zSKUH4Tj\\\\\\\/Jz2LSAlk3jfJQ8uI07bA0Nx4C0RLC5TzwmGENVA5WMV\\\\\\\/lK766OOuEbpjPZBE6rtmnUAoPIFIA+LtcS0P2ojAQIJ1O1mC8113vEk0lDydsJXk7Kvoljx50s3N\\\\\\\/lhF8A5duAL6OY4EHUGg0LGUr8BRT9HQ6HiG+XxmaGdwrL8wwqoRUtsMvFD+x+THCzW2cOXVv\\\\\\\/SolIOSm1GL8jR+4oaeaEEM8dBdwE72wugDyBU5Zu8iJn7bWw5hY1Zi3yqVluilni0fU4zfbaSmfctpmCpLR7AzT1eP54q9IvlMQrerrh9Tr64QI8JfWbkkpnd4HrC7x+fBZWTMK0If8sxqjreA6lIbtjjatBWKlmF+rpb9CfRW4A23M+hhQu0xinUaUJyLlZGxQNcNIHo5RRYgRqJ6IB2nCWJ1GuDPmTJSXRayepnPgBfbKlDkIU0WnDv3\\\\\\\/j+pHVUHbImi4lrV0HFKdAWfuzk4OFhBn\\\\\\\/Ccgz3Ju\\\\\\\/AO6oorwEtBDPNrTtlWeF94j5f6EFPZEuUOcErAX9ds6azdcstJS5Wo6EWytnb9UYmoNChBVqadE67zvNrv28RCYafikP2e0ffkgjBNQaEMvJQSqgng1oJ8pdHYoQmGapzajeWsJ6M9NP6Cs+YOnAMDE1GBAf0JMixxQu46YXCV4oVJ1t\\\\\\\/G\\\\\\\/yXmUEMkrS8LN+7wQTSq86wmhGjFpKQzzUPqhZ0BPk2apnwhrcLXpnTc7WR88vFAqlSOf5eEk65BC22BQEAn+Wwn0HBD3+8qtMvKNvnAoLcwIHC9c+weoOesAiMt\\\\\\\/80umvk48BiXrVp66xUWlOafwHbCWYuOa9gYj2IZxP13MsDeGykAq2I9z2gv20Csz4\\\\\\\/YTy4TZBla4H8N21dI+2nl8wmSjVvWwwpTXPYC\\\\\\\/rAaNYPaDJ8KiluW+e5ZNH1AiWX+ZztZQyjUo3F0wSK5Wq0gNSCyjJLQtjFu\\\\\\\/t+FlmnNDS45xfRWHUQ9qfoHmVSfj50gbsysXSp2paYGcverEIPpGd7PK62xnJ9aYdKHpYsavN5TFoIORVlTIxp74863aXJ8Qe9Aql2aScMb6CoGvTywqWfCbqeK3iIzGCwI\\\\\\\/r0GtvaojOWKnwhTdVXotjWb1R9fkh2gYFCgVSximqnQRullq9+pJn+7ijqH\\\\\\\/htg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022tAd\\\\\\\/h6mM3i\\\\\\\/x+g9U0GwpOw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228346697c25f7113291a89571580eee5bdef94de3e7867d97aa849e351d1c8b0b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484773 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-population">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1693" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sUeJ3seeIIKorKMPTPW1578ymvGI_PUFWduTxTEaiy0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1693" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8_T0mxvDcQ3MhQQNJ4fDRIwoUdHs3cvN7ihKUdYGVZc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1683" title="Identification of Ganoderma Disease Resistance Loci Using Natural Field Infection of an Oil Palm Multiparental Population" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1707" title="Pedigree-Based Analysis in a Multiparental Population of Octoploid Strawberry Reveals QTL Alleles Conferring Resistance to Phytophthora cactorum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484773" id="node484773" data-pisa="ggg;7/6/1693" data-pisa-master="ggg;g3.117.042127" data-apath="/ggg/7/6/1693.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Nitrogen Consumption Genetic Variants in Yeast Through QTL Mapping and Bulk Segregant RNA-Seq Analyses</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Francisco A. Cubillos</span>, <span class="highwire-citation-author" data-delta="1">Claire Brice</span>, <span class="highwire-citation-author" data-delta="2">Jennifer Molinet</span>, <span class="highwire-citation-author" data-delta="3">Sebastién Tisné</span>, <span class="highwire-citation-author" data-delta="4">Valentina Abarca</span>, <span class="highwire-citation-author" data-delta="5">Sebastián M. Tapia</span>, <span class="highwire-citation-author" data-delta="6">Christian Oporto</span>, <span class="highwire-citation-author" data-delta="7">Verónica García</span>, <span class="highwire-citation-author" data-delta="8">Gianni Liti</span> and <span class="highwire-citation-author" data-delta="9">Claudio Martínez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1693-1705;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042127 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484773" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Francisco A. Cubillos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Millennium Nucleus for Fungal Integrative and Synthetic Biology (MN-FISB), 9170201 Santiago, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BA.%2BCubillos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cubillos%20FA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BA.%2BCubillos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Claire Brice </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BBrice%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brice%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaire%2BBrice%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer Molinet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BMolinet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Molinet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BMolinet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sebastién Tisné </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre de coopération internationale en recherche agronomique pour le développement (CIRAD), UMR AGAP (Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit), Campus International de Baillarguet, 34398 Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebasti%C3%A9n%2BTisn%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tisn%C3%A9%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebasti%25C3%25A9n%2BTisn%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Valentina Abarca </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Valentina%2BAbarca%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abarca%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AValentina%2BAbarca%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sebastián M. Tapia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Millennium Nucleus for Fungal Integrative and Synthetic Biology (MN-FISB), 9170201 Santiago, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebasti%C3%A1n%2BM.%2BTapia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tapia%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebasti%25C3%25A1n%2BM.%2BTapia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Christian Oporto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Millennium Nucleus for Fungal Integrative and Synthetic Biology (MN-FISB), 9170201 Santiago, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BOporto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oporto%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BOporto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Verónica García </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ver%C3%B3nica%2BGarc%C3%ADa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garc%C3%ADa%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVer%25C3%25B3nica%2BGarc%25C3%25ADa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Gianni Liti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute for Research on Cancer and Ageing of Nice (IRCAN), Centre national de la recherche scientifique (CNRS) unités mixtes de recherche (UMR) 7284, Institut national de la santé et de la recherche médicale (INSERM) U1081, University of Nice Sophia Antipolis, 06107, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gianni%2BLiti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liti%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGianni%2BLiti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Claudio Martínez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudio%2BMart%C3%ADnez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADnez%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClaudio%2BMart%25C3%25ADnez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:claudio.martinez@usach.cl" class="" data-icon-position="" data-hide-link-title="0">claudio.martinez@usach.cl</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/6/1693" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484773" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484773/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1693.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484773" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484773/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1693.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484773" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484773/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1693.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484773" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484773/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/6/1693.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484773" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484773/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2"><em>Saccharomyces cerevisiae</em> is responsible for wine must fermentation. In this process, nitrogen represents a limiting nutrient and its scarcity results in important economic losses for the wine industry. Yeast isolates use different strategies to grow in poor nitrogen environments and their genomic plasticity enables adaptation to multiple habitats through improvements in nitrogen consumption. Here, we used a highly recombinant <em>S. cerevisiae</em> multi-parent population (SGRP-4X) derived from the intercross of four parental strains of different origins to identify new genetic variants responsible for nitrogen consumption differences during wine fermentation. Analysis of 165 fully sequenced F12 segregants allowed us to map 26 QTL in narrow intervals for 14 amino acid sources and ammonium, the majority of which represent genomic regions previously unmapped for these traits. To complement this strategy, we performed Bulk segregant RNA-seq (BSR-seq) analysis in segregants exhibiting extremely high and low ammonium consumption levels. This identified several QTL overlapping differentially expressed genes and refined the gene candidate search. Based on these approaches, we were able to validate <em><a href="http://www.yeastgenome.org/locus/S000002534/overview">ARO1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004034/overview">PDC1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003708/overview">CPS1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005103/overview">ASI2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005212/overview">LYP1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005214/overview">ALP1</a></em> allelic variants underlying nitrogen consumption differences between strains, providing evidence of many genes with small phenotypic effects. Altogether, these variants significantly shape yeast nitrogen consumption with important implications for evolution, ecological, and quantitative genomics.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/nitrogen" class="hw-term hw-article-keyword hw-article-keyword-nitrogen" rel="nofollow">nitrogen</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/rna-seq" class="hw-term hw-article-keyword hw-article-keyword-rna-seq" rel="nofollow">RNA-seq</a></li><li class="kwd"><a href="/keyword/bsr-seq" class="hw-term hw-article-keyword hw-article-keyword-bsr-seq" rel="nofollow">BSR-seq</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li><li class="kwd"><a href="/keyword/multiparental-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations" rel="nofollow">multiparental populations</a></li></ul><p id="p-3">Alcoholic fermentation is exclusively carried out by yeasts and the process transforms sugar within the grape must into ethanol and CO<sub>2</sub>. In this context, a large number of studies have exploited a large panel of <em>Saccharomyces cerevisiae</em> strains and classified them into two groups: domesticated and wild populations (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Querol <em>et al.</em> 2003</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Legras <em>et al.</em> 2007</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Sicard and Legras 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Cubillos <em>et al.</em> 2011</a>). Thorough phylogenetic analysis revealed four evolutionary lineages (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Liti <em>et al.</em> 2009</a>), with many other lineages recently found (G. Liti and J. Schacherer, personal communication). <em>S. cerevisiae</em> strains show the greatest fermentation capacity in wine-making environments (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Torija <em>et al.</em> 2001</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beltran <em>et al.</em> 2002</a>), mostly because of their high tolerance to ethanol (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arroyo-Lopez <em>et al.</em> 2010</a>) and production of secondary-end metabolites that positively contribute to wine character. Yeast needs various nutrients to perform these complex biochemical transformations, predominantly nitrogen and carbon sources, which are found in variable concentrations. Nitrogen represents 6–10% of cellular biomass and is primarily used by yeast for growth and protein synthesis through a wide variety of nitrogen-containing compounds. These compounds are present in natural must and the main examples are ammonium ions (NH<sub>4</sub><sup>+</sup>) and amino acids. The availability of assimilable nitrogen in musts influences the fermentation performance of yeast cells. For example, musts with low nitrogen concentrations are problematic and generally lead to sluggish fermentations (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Salmon 1989</a>) and the production of undesirable volatile compounds. Thus, to avoid problematic fermentations due to nitrogen deficiency, winemakers add di-ammonium phosphate to musts to counterbalance this nitrogen deficit. However, in winemaking this practice is not systematic, since poor control of the procedure may cause microbial instability and undesirable effects on the final sensory profile.</p><p id="p-4">Nitrogen sources can have different impacts on cell development depending on their classification as preferred or not-preferred sources. Ammonium, glutamine, glutamate, and asparagine are preferred nitrogen sources whereas urea, proline, and allantion are not-preferred (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Magasanik and Kaiser 2002</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">ter Schure <em>et al.</em> 2000</a>). Yeast preferentially uses substrates that allow the best growth through a regulation mechanism called nitrogen catabolite repression (NCR). This system promotes the expression of permeases for the preferred nitrogen source and the degradation of permeases of those not-preferred sources (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Magasanik and Kaiser 2002</a>). Thus, during the fermentation process, yeast cells adjust their metabolism depending on the nitrogen sources available (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Rossignol <em>et al.</em> 2003</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gasch <em>et al.</em> 2000</a>), mostly through TOR pathway regulation. Indeed, TOR is able to sense the quantity and quality of available nitrogen sources and, thus, adapt cell metabolism by regulating the activity of certain systems such as NCR (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Pedruzzi <em>et al.</em> 2003</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Swinnen <em>et al.</em> 2006</a>).</p><p id="p-5">Previous studies have shown that nitrogen assimilation profiles vary between strains (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Gutierrez <em>et al.</em> 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Crepin <em>et al.</em> 2012</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Jara <em>et al.</em> 2014</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Brice <em>et al.</em> 2014a</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Ibstedt <em>et al.</em> 2015</a>). These different phenotypes result from the high genetic diversity that exists between strains involving a large number of allelic variants (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Salinas <em>et al.</em> 2016</a>). These variants can be dissected through QTL approaches (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Cubillos 2016</a>). In <em>S. cerevisiae</em>, several QTL analyses have already focused on identifying the genetic bases for specific physiological traits such as cell morphology (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Nogami <em>et al.</em> 2007</a>), sporulation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ben-Ari <em>et al.</em> 2006</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Katou <em>et al.</em> 2009</a>), drug sensitivity (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Kim and Fay 2007</a>), flocculation (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Brauer <em>et al.</em> 2006</a>), wine aroma production (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Steyer <em>et al.</em> 2012</a>), ethanol tolerance, and growth (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Katou <em>et al.</em> 2009</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Nogami <em>et al.</em> 2007</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hu <em>et al.</em> 2007</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Marullo <em>et al.</em> 2007</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Smith and Kruglyak 2008</a>), together with sulfur assimilation (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Noble <em>et al.</em> 2015</a>). Because of the importance of nitrogen in fermentation processes, preliminary research has been carried out on nitrogen consumption (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Jara <em>et al.</em> 2014</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Brice <em>et al.</em> 2014b</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Yang <em>et al.</em> 2016</a>). For oenological strains, variation in nitrogen assimilation profiles are explained by polymorphisms in genes involved in mechanisms related to nitrogen signaling (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Brice <em>et al.</em> 2014b</a>). Our group previously demonstrated that these variations in nitrogen signaling mechanisms can also be observed between strains from different lineages [Wine/European (WE) and Sake (SA)], identifying three genes for which allelic variation resulted in differences in nitrogen assimilation (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Jara <em>et al.</em> 2014</a>). Two of these genes, <em><a href="http://www.yeastgenome.org/locus/S000002330/overview">GLT1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002374/overview">GDH2</a></em> (a glutamate synthetase and a glutamate dehydrogenase, respectively), belong to the central nitrogen metabolism pathway. Nevertheless, this study only involved two different genetic backgrounds and omitted a larger number of strains derived from different origins that may have developed independent adaptive responses to nitrogen assimilation based on the availability of nitrogen compounds in the distinct environments (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Cubillos <em>et al.</em> 2011</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Salinas <em>et al.</em> 2012</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Tesniere <em>et al.</em> 2015</a>).</p><p id="p-6">Although biparental crosses have brought fruitful results, their power is limited to the extent of genetic variation between the two parental strains and identified QTL can only explain a small percentage of the total phenotypic variance within the species. Instead, utilizing an alternative design incorporating multi-parent mapping populations has proven successful for the mapping of larger number of QTL in several popular model organisms, for instance: <em>Drosophila</em> (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Long <em>et al.</em> 2014</a>), mice (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bogue <em>et al.</em> 2015</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Durrant <em>et al.</em> 2011</a>), and plants (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Kover <em>et al.</em> 2009</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Huang <em>et al.</em> 2011</a>). Recently, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Cubillos <em>et al.</em> (2013)</a> used this type of strategy on the yeast model based on a mapping population obtained from outcrossing four founders representative of the main <em>S. cerevisiae</em> lineages, denominated SGRP-4X. Given the high genetic diversity and recombination levels in budding yeast, this type of analysis has shown the advantage of breaking linkage disequilibrium and increasing haplotype diversity. Based on these antecedents, we investigated the genetic basis of nitrogen assimilation variation between strains from various origins by performing linkage mapping in SGRP-4X. We have combined our QTL approach with a BSR-seq analysis in segregants exhibiting extremely high and low ammonium consumption levels. Confirmation by RNA mass sequencing and the identification of multiple allelic variants demonstrates that this is a successful strategy for mapping low effect genes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Parental strains and SGRP-4X segregants</h3><p id="p-7">Haploid parental strains YPS128 (“NA,” <em>Mat a</em>, <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">ho</a></em>::<em>HygMX</em>, <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>::<em>KanMX</em>), DBVPG6044 (“WA,” <em>Mat a</em>, <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">ho</a></em>::<em>HygMX</em>, <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>::<em>KanMX</em>), DBVPG6765 (“WE,” <em>Mat a</em>, <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">ho</a></em>::<em>HygMX</em>, <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>::<em>KanMX</em>), and Y12 (“SA,” <em>Mat</em> α <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">ho</a></em>::<em>HygMX</em>, <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>::<em>KanMX</em>), together with the 165 SGPR-4X segregants used in this study, were previously described (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Cubillos <em>et al.</em> 2009</a>, 2013). Briefly, to generate SGRP-4X segregants, North American (NA) and West African (WA) strains (both <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>::<em>KanMX</em>) were crossed to generate diploid F1 hybrids as well as SA and WE strains (both <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">lys2</a></em>::<em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em>). To confirm successful crosses, individual colonies were isolated and mating tests using tester strains Y55-2369 (<em>MATa</em>, <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">ho</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000003666/overview">ura2</a>-1</em>, <em><a href="http://www.yeastgenome.org/locus/S000000370/overview">tyr1</a>-1</em>) and Y55-2370 (<em>MATa</em>, <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">ho</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000003666/overview">ura2</a>-1</em>, <em><a href="http://www.yeastgenome.org/locus/S000000370/overview">tyr1</a>-1</em>), as well as diagnostic PCR for the <em>MAT</em> locus (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Huxley <em>et al.</em> 1990</a>), were performed. F1 hybrids were replicated onto KAc at 23° for sporulation for 10 d before cells were collected in water, treated with an equal amount of ether, and vortexed for 10 min to kill unsporulated cells. After washing the cells in water, they were treated with Zymolase (10 mg/ml) to remove asci. Cell mixtures were vortexed for 5 min to increase spore dispersion. Spores from both crosses were mixed, grown on YPD, and replica plated on minimal media to select for successful crosses. This procedure was repeated 11 times to create the F12 population. Viable spores with correct 2:2 segregations for the <em>MAT</em> locus, and <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">lys2</a></em> auxotrophies, were selected. We picked a total of 192 segregants (some from the same tetrad), out of which 165 were used in this study. All the strains used in this work were short-term maintained on YPDA solid media (2% glucose, 0.5% peptone, 0.5% yeast extract, and 2% agar). Genotypes are described in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Martinez_Identification__data_policy_documentation.docx">File S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Fermentation and nitrogen assimilation estimation</h3><p id="p-8">Fermentations were carried out as previously described (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Jara <em>et al.</em> 2014</a>). Briefly, each individual was fermented in duplicate in synthetic wine must (SM300) and prepared according to <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Rossignol <em>et al.</em> (2003)</a>. SM300 was supplemented with a final concentration of 300 mgN/L of assimilable nitrogen (YAN) corresponding to 120 mgN/L of ammonium and 180 mgN/L of a mixture of 19 amino acids (612.6 mg/L L-proline, 503.5 mg/L L-glutamine, 503.5 mg/L L-arginine monohydrochloride, 179.3 mg/L L-tryptophan, 145.3 mg/L L-alanine, 120.4 mg/L L-glutamic acid, 78.5 mg/L L-serine, 75.92 mg/L L-threonine, 48.4 mg/L L-leucine, 44.5 mg/L L-aspartic acid, 44.5 mg/L L-valine, 37.9 mg/L L-phenylalanine, 32.7 mg/L L-isoleucine, 50.0 mg/L L-histidine monohydrochloride monohydrate, 31.4 mg/L L-methionine, 18.3 mg/L L-tyrosine, 18.3 mg/L L-glycine, 17.0 mg/L L-lysine monohydrochloride, and 13.1 mg/L L-cysteine). The strains were initially grown under constant agitation in 10 ml of SM300 for 16 hr at 25°. Next, 1 × 10<sup>6</sup> cells/ml were inoculated into 12 ml of SM300 (in 15 ml conical tubes) and incubated at 25°, with no agitation for 6 d, the stage at which most nitrogen consumption differences can be observed (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Martínez <em>et al.</em> 2013</a>; <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Jara <em>et al.</em> 2014</a>). After 6 d, 12 ml of synthetic grape must (SM300) were centrifuged at 9000 × <em>g</em> for 10 min and the supernatant was collected. 20 μl of SM300 were injected in a Shimadzu Prominence HPLC equipment (Shimadzu) using a Bio-Rad HPX –87H column according to <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Nissen <em>et al.</em> (1997)</a>. The concentration of each amino acid was measured using the HPLC analysis as previously described (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Gomez-Alonso <em>et al.</em> 2007</a>). The consumption of each nitrogen source was estimated as the difference between the initial and final amounts of each source before and after fermentation, respectively. Phenotyping results are described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S2</a>.</p></div><div id="sec-4" class="subsection"><h3>QTL mapping</h3><p id="p-9">QTL mapping was performed using the linear mixed effect model, with QTL effect as the fixed effect and genetic background controlled with a polygenic random effect as proposed in <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bernardo (2013)</a>. Independently, on each case for the 15 phenotypes tested, QTL presence was assessed for each of the 99,900 segregating sites identified in <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Cubillos <em>et al.</em> (2013)</a> (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S1</a>) in the linear mixed model as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-1.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-2.gif"/></noscript></span></span> is an <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-3.gif"/></noscript></span></span> observation vector of phenotypic value (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-4.gif"/></noscript></span></span> = 165, the number of segregants), <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-5.gif"/></noscript></span></span> is a <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-6.gif"/></noscript></span></span> design matrix relating observations to the mating-type locus and auxotrophic markers <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">LYS2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> fixed effects <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-7.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-8.gif"/></noscript></span></span> being a <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-9.gif"/></noscript></span></span> vector (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-10.gif"/></noscript></span></span> = 4, the four combinations of <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">LYS2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> markers), <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-11.gif"/></noscript></span></span> is a <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-12.gif"/></noscript></span></span> design matrix relating observations to the QTL parental allele fixed effects <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-13.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-14.gif"/></noscript></span></span> being a <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-15.gif"/></noscript></span></span> vector (<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-16.gif"/></noscript></span></span> = 4, the number of segregating parental alleles), <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-17.gif"/></noscript></span></span> is a <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-18.gif"/></noscript></span></span> design matrix relating observations to the <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-19.gif"/></noscript></span></span> vector of polygenic additive random effects <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-20.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-21.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-22.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-23.gif"/></noscript></span></span> vector of residual effects with <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-24.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-25.gif"/></noscript></span></span> is an identity matrix and <em>A</em> is a genomic-based kinship based on segregation information away from the chromosome to which the position tested belongs. Coefficients of matrix <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-26.gif"/></noscript></span></span> were calculated between pairs of individuals as the proportion of shared parental alleles on the segregating sites from the remaining chromosomes. At each tested position <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1693/embed/mml-math-27.gif"/></noscript></span></span> the linear mixed effect model was estimated by REML using the R-ASReml package for R, and p-values for the QTL fixed effect obtained with the Wald test. Vectors of 99,900 p-values were then adjusted for the false discovery rate (FDR) using R function p.adjust implementing the method from <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Benjamini and Hochberg (1995)</a> and significant associations were retained at adjusted <em>q</em>-values = 0.5. Initially, we used this high threshold as a mean to obtain a greater number of putative QTL to incorporate a larger number of candidate genes into our screen. QTL mapping results are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S3</a>.</p></div><div id="sec-5" class="subsection"><h3>RNA-seq data analysis</h3><p id="p-10">A total of 16 individuals with extreme phenotypes for ammonium consumption (eight segregants with high and eight with low consumption levels, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S2</a>) were fermented. Fermentations were carried out as previously described in duplicate for each individual for 24 hr. Cultures were harvested by centrifugation and cells were treated with 2 U of Zymolyase for 30 min at 37°. RNA was individually extracted for each segregant utilizing the E.Z.N.A. Total RNA Kit I (OMEGA) according to the supplier’s instructions. RNA samples were then treated with DNase I (Promega) to remove genomic DNA traces and total RNA was recovered using the GeneJET RNA Cleanup and Concentration Micro Kit (Thermo Scientific). RNA integrity was confirmed using a Fragment Analyzer. Later, four RNA pools were generated utilizing equal amounts (1 μg) of RNA per individual: two high levels of ammonium consumption (HLA) and two low levels of ammonium consumption (LLA). Each pool was generated from independent replicates. The RNA-seq libraries were constructed using the TruSeq RNA Sample Prep Kit v2 (Illumina). Briefly, mRNA from 1 μg of total RNA was enriched by mRNA purification magnetic beads. Enriched mRNA was eluted and fragmented at 94° for 5 min. The double-stranded cDNA was acquired by RT-PCR using the above fragmented mRNA, followed by end repair, single A base addition, and adapter index ligation. The ligation product was amplified by PCR. The size of the end product was ∼260 bp. The sequencing was conducted on a HiSequation 2500 (Illumina) in a single lane for the four samples. Reads are available in the Biosample Database Project #PRJNA379146 with accession codes SAMN06602320, SAMN06602321, SAMN06602322, and SAMN06602323 for HLA-1, HLA-2, LLA-1, and LLA-2 samples respectively.</p><p id="p-11">Raw reads were first assessed for their quality using the FASTQC tool kit (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>). Low quality reads were discarded using the Trimmomatic tool using default score settings and a phred score cut-off of 30 (<a href="http://www.usadellab.org/cms/?page=trimmomatic">http://www.usadellab.org/cms/?page=trimmomatic</a>). RNA-seq reads were then aligned to the S288c reference [<em>S. cerevisiae</em> genome obtained on 03/03/2016, from the <em>Saccharomyces</em> Genome Database (SGD), FTP SITE: <a href="http://downloads.yeastgenome.org/sequence/S288C_reference/genome_releases/">http://downloads.yeastgenome.org/sequence/S288C_reference/genome_releases/</a> corresponding to a stable release from January 2015] utilizing Tophat with default score settings (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Trapnell <em>et al.</em> 2009</a>). BAM files were sorted and indexed using Samtools with default score settings (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Li <em>et al.</em> 2009</a>). Alignments were then processed and gene counts were attained utilizing HTSeq (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anders <em>et al.</em> 2015</a>) with the no-stranded and –gene counts configuration from the S288c gff file. Differential expression was assessed with DESeq utilizing the nbinomTest function (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anders and Huber 2010</a>). The results obtained from the DESeq analysis provided differentially expressed genes between pools with FDR &lt; 0.05. FDR was estimated utilizing the default Benjamin–Hochberg correction (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Benjamini and Hochberg 1995</a>).</p><p id="p-12">SNP frequencies were estimated by initially extracting SNP data from the four pools (two HLA and two LLA) utilizing the mpileup tool from Samtools with –u and –f options together with bcftools view –vcg options to export and obtain a .vcf file. Only high quality SNPs (at least 10 reads for the reference and alternative nucleotides) present in all four samples were considered. For this, we utilized the intersectbed tool from Bedtools filtering out SNPs absent in at least one of the pools (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Quinlan and Hall 2010</a>). Differential SNP frequency was estimated through a Fisher test using R and corrected using the <em>q</em>-value package with default options (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Storey and Tibshirani 2003</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bass <em>et al.</em> 2015</a>). A FDR &lt; 0.001 was considered in order to obtain &lt; 10 false positive results.</p></div><div id="sec-6" class="subsection"><h3>Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways enrichment</h3><p id="p-13">Differentially expressed genes and SNPs exhibiting significant frequency differences between pools were used in the DAVID Bioinformatics Resource (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Huang da <em>et al.</em> 2009</a>) to test for a significant enrichment of genes in pathways in the KEGG. FDR was estimated utilizing the default option (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Benjamini and Hochberg 1995</a>). We selected categories with a significant overrepresentation utilizing a FDR &lt; 0.05%.</p></div><div id="sec-7" class="subsection"><h3>Candidate gene validation</h3><p id="p-14">Genomic regions comprising 15 kb up- and downstream for selected QTL were examined in the SGD) for candidate genes. The sequences of the candidate genes were compared between strains utilizing SIFT analysis (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bergstrom <em>et al.</em> 2014</a>). To validate the presence of a QTL, we performed a reciprocal hemizygosity assay (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Steinmetz <em>et al.</em> 2002</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Figure">Figure S1</a>). The gene <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> (essential for pyrimidine biosynthesis) previously deleted in the parental strains (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Cubillos <em>et al.</em> 2009</a>) was used as a selectable marker with some modifications. Briefly, we used haploid versions of the corresponding parental strains with extreme phenotypes for the QTL region to delete each target gene and construct all possible combinations of single deletions. Next, mutated parental strains were crossed to generate the reciprocal hemizygote strains and selected in double drugs plates (50 mg/ml Hygromycin B and 100 mg/ml Nourseothricin). The diploid hybrid strains were confirmed by <em>MAT</em> locus PCR (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Huxley <em>et al.</em> 1990</a>) and the deletions of the target genes were confirmed by PCR using the primers pairs A1/S8 or A4/S5 (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Salinas <em>et al.</em> 2012</a>). Nitrogen assimilation profiles between strains were compared using a classical Student’s <em>t</em>-test (p-value &lt; 0.05). Total amino acid differences were compared through a paired <em>t</em>-test.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-15">As stated above, genotype data can be obtained from <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S1</a>. These genetic marker data were already published before in <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Cubillos <em>et al.</em> (2013)</a>. Phenotype data are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S2</a>. Gene transcript data can be obtained from the Biosample Database Project #PRJNA379146 with accession codes SAMN06602320, SAMN06602321, SAMN06602322, and SAMN06602323 for HLA-1, HLA-2, LLA-1, and LLA-2 samples, respectively. </p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Identification of genomic regions underlying nitrogen consumption differences in SGRP-4X</h3><p id="p-16">In order to obtain broader evidence of the genetic basis underlying nitrogen assimilation differences in <em>S. cerevisiae</em>, we selected a multi-parent recombinant population for QTL mapping analysis: SGRP-4X (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Cubillos <em>et al.</em> 2013</a>). The four parental founders of this population are representative of the NA (YPS128), SA (Y12), WE (DBVPG6765), and WA (DBVPG6044) clusters (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Liti <em>et al.</em> 2009</a>). Initially, we inoculated 1 × 10<sup>6</sup> cells/ml in 12 ml of synthetic wine must (SM300) and characterized nitrogen consumption preferences in all founder strains after 6 d of fermentation. The WE isolate was able to consume greater levels of ammonium compared to any of the other strains (p-value &lt; 0.05, <em>t</em>-test), as we have previously described (<a id="xref-ref-43-6" class="xref-bibr" href="#ref-43">Jara <em>et al.</em> 2014</a>). On the other hand, SA and WA isolates were able to efficiently consume amino acids rather than ammonium (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), while the NA isolate consumed the lowest levels of amino acids and ammonium, representing the least adapted isolate to wine fermentation must. Based on this premise, a subset of 165 SGRP-4X fully sequenced recombinants were utilized to identify genetic variants responsible for nitrogen assimilation differences between strains (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Cubillos <em>et al.</em> 2013</a>). Thus, the whole population was phenotyped for consumption of 14 amino acids as well as ammonium (see <em>Materials and Methods</em>). Transgression levels were found to be relatively low, with the exception of serine and histidine, with 23.6 and 27.2%, respectively. Most transgressions were positive transgressive segregants (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S2</a>), suggesting that combinations resulting in an improved nitrogen assimilation profile are rare.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL Mapping in SGRP-4X. (A) Ammonium and amino acid total YAN consumption in parental strains. Student&#x2019;s t-test was performed between the WE isolate and the other strains; * p-value &lt; 0.05, ** p-value &lt; 0.01, and *** p-value &lt; 0.001. (B) Plot produced by QTL scan using Wald test for Arginine in the SGRP-4X population. The y-axis shows the p-value (&#x2013;log10 scale) for the effect of a QTL along the chromosomes (x-axis), which are shown in different colors. Dotted line denotes cut-off FDR &lt; 0.5. FDR, false discovery rate; ns, not-significant difference; QTL, quantitative trait loci; WE, Wine/European; YAN, assimilable nitrogen." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1280902443" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;QTL Mapping in SGRP-4X. (A) Ammonium and amino acid total YAN consumption in parental strains. Student&#x2019;s &lt;em&gt;t&lt;/em&gt;-test was performed between the WE isolate and the other strains; * p-value &lt; 0.05, ** p-value &lt; 0.01, and *** p-value &lt; 0.001. (B) Plot produced by QTL scan using Wald test for Arginine in the SGRP-4X population. The &lt;em&gt;y&lt;/em&gt;-axis shows the p-value (&#x2013;log10 scale) for the effect of a QTL along the chromosomes (&lt;em&gt;x&lt;/em&gt;-axis), which are shown in different colors. Dotted line denotes cut-off FDR &lt; 0.5. FDR, false discovery rate; ns, not-significant difference; QTL, quantitative trait loci; WE, Wine/European; YAN, assimilable nitrogen.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1693/F1.medium.gif" width="326" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/6/1693/F1.medium.gif" width="326" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484920" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">QTL Mapping in SGRP-4X. (A) Ammonium and amino acid total YAN consumption in parental strains. Student’s <em>t</em>-test was performed between the WE isolate and the other strains; * p-value &lt; 0.05, ** p-value &lt; 0.01, and *** p-value &lt; 0.001. (B) Plot produced by QTL scan using Wald test for Arginine in the SGRP-4X population. The <em>y</em>-axis shows the p-value (–log10 scale) for the effect of a QTL along the chromosomes (<em>x</em>-axis), which are shown in different colors. Dotted line denotes cut-off FDR &lt; 0.5. FDR, false discovery rate; ns, not-significant difference; QTL, quantitative trait loci; WE, Wine/European; YAN, assimilable nitrogen.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">Large linkage blocks have historically been the main obstacle when analyzing complex traits at this experimental scale. A key reason for using the highly recombinant SGRP-4X population was to incorporate a greater genetic diversity and to distinguish and precisely map genes underlying a QTL within small linkage blocks compared to biparental F1 or F2 populations. Thus, to accurately identify genomic regions responsible for nitrogen consumption differences, we performed a QTL mapping strategy utilizing the high-resolution genetic map previously obtained for SGRP-4X (<a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Cubillos <em>et al.</em> 2013</a>) on the 15 phenotypes scored during the fermentation process. Each available marker across the genome (&gt; 100,000 SNPs) was assessed for an association between the phenotype and parental allele fixed effects. We used a linear mixed model including correction for polygenic random effects and retained peaks with an empirical FDR &lt; 0.5 (<a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">Cubillos <em>et al.</em> 2013</a>). Overall, we were able to identify 29 QTL evenly distributed along the genome, aside from chromosomes I, V, VI, and VIII, which did not map for any QTL. Based on overlapping intervals, these QTL can be summarized into 26 major QTL regions (QTL1–QTL26), out of which three were present in more than a single nitrogen source (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S4</a>). In all cases, these pleiotropic QTL were found for two related amino acids. For example, QTL4 mapped for lysine (−logPvalue 12.26, FDR = 0.04) and arginine (−logPvalue 8.54, FDR = 0.14) traits, both amino acids with positively-charged side chains. Similarly, QTL10 mapped for phenylalanine (−logPvalue 14.39, FDR = 0.01) and methionine (−logPvalue 10.1, FDR = 0.3), two amino acids with hydrophobic side chains. These findings are in agreement with the phenotypic correlation between these couples exhibiting significant Spearman correlations (p &lt; 0.05, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Figure">Figure S2</a>). On the other hand, arginine and isoleucine showed the greatest number of QTL with seven and six significant regions in each case, respectively (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>).</p></div><div id="sec-11" class="subsection"><h3>Validation of candidates based on the QTL mapping approach</h3><p id="p-19">Based on our QTL mapping strategy, we looked for candidate genes for molecular validation. The majority of the QTL found in this study were mapped for arginine together with isoleucine; therefore, we focused on these amino acids for subsequent analysis (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S4</a>). Thus, we examined all these QTL regions to select genes with a potential function in nitrogen metabolism. Three QTL were identified for arginine where candidate genes were identified. QTL21 mapped for isoleucine and peaks at chromosome XIV – 323 kb, which is 15 kb away from <em><a href="http://www.yeastgenome.org/locus/S000005103/overview">ASI2</a></em>, a gene that encodes for an inner nuclear membrane protein and maintains the repressed state of gene expression in the absence of inducing amino acids (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Forsberg <em>et al.</em> 2001</a>). Similarly, QTL19 was identified in arginine peaks at chromosome XIV – 158 kb near <em><a href="http://www.yeastgenome.org/locus/S000005214/overview">ALP1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000005212/overview">LYP1</a></em>, two genes encoding for an arginine transporter and lysine permease, respectively. QTL8 mapped for arginine near <em><a href="http://www.yeastgenome.org/locus/S000003708/overview">CPS1</a></em>, a vacuolar carboxypeptidase-S, shown to be under the influence of NCR and more particularly by <em><a href="http://www.yeastgenome.org/locus/S000000842/overview">GLN3</a></em>. Expression of the vacuolar carboxypeptidase-S gene in <em>S. cerevisiae</em> is regulated by nutrient availability (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hofman-Bang 1999</a>).</p><p id="p-20">To determine the impact of the different allelic variants upon nitrogen consumption for these selected genes, we performed a functional analysis by hemizygotic strain comparison (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Figure">Figure S1</a>). For each gene, we selected the two parental strains for which the recombinant SGRP-4X segregants showed the most important differences in amino acid consumption for the QTL region implicated, particularly concerning arginine consumption for <em><a href="http://www.yeastgenome.org/locus/S000005212/overview">LYP1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003708/overview">CPS1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005214/overview">ALP1</a></em> genes, and isoleucine consumption for <em><a href="http://www.yeastgenome.org/locus/S000005103/overview">ASI2</a></em>. In this way, the screening allows selection of the most appropriate parental strains for the cross and gene validation. The hybrid WE × SA was used to study <em><a href="http://www.yeastgenome.org/locus/S000003708/overview">CPS1</a></em> for which segregants with the molecular markers originating from the WE background showed greater consumption levels for arginine compared to the SA strain. For the other three genes, <em><a href="http://www.yeastgenome.org/locus/S000005214/overview">ALP1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005212/overview">LYP1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005103/overview">ASI2</a></em>, we chose the WE × WA hybrid, for which the identification of molecular markers show a greater consumption of arginine, isoleucine, and tyrosine in the WE parental strain. Each cross included two allelic versions of the selected gene, one parent bearing an inactivated form and the other parent containing a functional form (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Figure">Figure S1</a>).</p><p id="p-21">For the four genes tested, the comparison of residual amino acids revealed a statistically significant difference in YAN consumption for some amino acids (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S5</a>). Reciprocal hemizygosity analysis for <em><a href="http://www.yeastgenome.org/locus/S000005214/overview">ALP1</a></em> showed that <em><a href="http://www.yeastgenome.org/locus/S000005214/overview">ALP1</a><sup>WE</sup></em> (reciprocal hemizygote carrying the WE allelic variant) conferred a higher consumption capacity for arginine, tyrosine, and phenylalanine, with a difference consumption of 17.1, 5.4, and 2.9%, respectively, between the two hemizygote constructions (p-value &lt; 0.05, <em>t</em>-test, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). These results are in agreement with the phenotype found in the SGRP-4X segregant population. The same findings apply for <em><a href="http://www.yeastgenome.org/locus/S000005103/overview">ASI2</a></em>, with a greater consumption of glutamic acid, arginine, and phenylalanine sources (6.7, 17.3, and 4%, respectively) and marginally significant for isoleucine (p-value = 0.07, <em>t</em>-test), in the WE allelic variant compared to the WA background (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Concerning <em><a href="http://www.yeastgenome.org/locus/S000005212/overview">LYP1</a></em>, the hybrid containing the WE allele provided a better arginine and lysine consumption capacity with a 37.5 and 1.8% difference between the two hemizygotes (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). <em><a href="http://www.yeastgenome.org/locus/S000003708/overview">CPS1</a></em> is the only gene for which the assessment of amino acid consumption between hemizygotes is different compared to the other genes. The allelic variant from the WE background showed significantly higher arginine consumption levels (equivalent to 19.5%), compared to the allele originating from SA; however, the <em><a href="http://www.yeastgenome.org/locus/S000003708/overview">CPS1</a><sup>SA</sup></em> reciprocal hemizygote showed greater tryptophan consumption levels (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>) equivalent to 2.6%, likely due to a compensation effect. Overall, these results demonstrate the role of these four genes in the YAN consumption phenotype and validate the power of the SGRP-4X multi-parent population to identify small-effect allelic variants.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Reciprocal hemizygosity analysis on ALP1, LYP1, ASI2, and CPS1 mapped through a QTL approach. The hybrid hemizygote strains consumption levels (mg/L) are shown. *, **, and *** represent a significant statistical difference between the hemizygote strains for the same gene using a t-test; * p-value &lt; 0.05, ** p-value &lt; 0.01, and *** p-value &lt; 0.001. Reciprocal hemizygotes for (A) SA and WE ALP1, (B) ASI2 WA and WE, (C) LYP1 WA and WE, and (D) CPS1 WA and WE. &#x394;/WE denotes hemizygotes carrying the WE allele, while XX/&#x394; denotes hemizygotes carrying the alternative allele. QTL, quantitative trait loci; SA, Sake; WA, West African; WE, Wine/European." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1280902443" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Reciprocal hemizygosity analysis on &lt;em&gt;ALP1&lt;/em&gt;, &lt;em&gt;LYP1&lt;/em&gt;, &lt;em&gt;ASI2&lt;/em&gt;, and &lt;em&gt;CPS1&lt;/em&gt; mapped through a QTL approach. The hybrid hemizygote strains consumption levels (mg/L) are shown. *, **, and *** represent a significant statistical difference between the hemizygote strains for the same gene using a &lt;em&gt;t&lt;/em&gt;-test; * p-value &lt; 0.05, ** p-value &lt; 0.01, and *** p-value &lt; 0.001. Reciprocal hemizygotes for (A) SA and WE &lt;em&gt;ALP1&lt;/em&gt;, (B) &lt;em&gt;ASI2&lt;/em&gt; WA and WE, (C) &lt;em&gt;LYP1&lt;/em&gt; WA and WE, and (D) &lt;em&gt;CPS1&lt;/em&gt; WA and WE. &#x394;/WE denotes hemizygotes carrying the WE allele, while XX/&#x394; denotes hemizygotes carrying the alternative allele. QTL, quantitative trait loci; SA, Sake; WA, West African; WE, Wine/European.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1693/F2.medium.gif" width="410" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/6/1693/F2.medium.gif" width="410" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484981" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Reciprocal hemizygosity analysis on <em>ALP1</em>, <em>LYP1</em>, <em>ASI2</em>, and <em>CPS1</em> mapped through a QTL approach. The hybrid hemizygote strains consumption levels (mg/L) are shown. *, **, and *** represent a significant statistical difference between the hemizygote strains for the same gene using a <em>t</em>-test; * p-value &lt; 0.05, ** p-value &lt; 0.01, and *** p-value &lt; 0.001. Reciprocal hemizygotes for (A) SA and WE <em>ALP1</em>, (B) <em>ASI2</em> WA and WE, (C) <em>LYP1</em> WA and WE, and (D) <em>CPS1</em> WA and WE. Δ/WE denotes hemizygotes carrying the WE allele, while XX/Δ denotes hemizygotes carrying the alternative allele. QTL, quantitative trait loci; SA, Sake; WA, West African; WE, Wine/European.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>BSR-Seq identifies additional candidate genes</h3><p id="p-23">Ammonium represents the main nitrogen source in wine must; therefore, in order to identify additional genetic variants underlying ammonium consumption differences, we utilized a BSR-Seq approach as a means to complement the QTL mapping strategy. We separately fermented eight individuals (in duplicate) that exhibited HLA consumption in the first round of fermentations and another eight with LLA (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S2</a>). RNA was obtained after 24 hr of fermentation and four pools from each individual replicate were generated (two HLA and two LLA pools), which were sequenced through an Illumina HiSeq2500 platform (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). We estimated differential expression between pools utilizing the DEseq package and found 725 genes differentially expressed at a 0.05 FDR (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S6</a>), with 491 genes expressed significantly higher in HLA compared to LLA and 234 vice versa.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="BSR-Seq experimental approach. (A) Experimental strategy followed to obtained mRNA counts and SNP frequencies from HLA and LLA pools. (B) Pathways enriched (FDR &lt; 5%) in genes differently expressed between pools. (C) CO2 lost (g/L) in HLA (black) and LLA (dotted gray lines) pools. SDs are shown for each measured time-point. (D) Nitrogen consumption (mgN/L) in HLA (black) and LLA (gray) pools after 6 d of fermentation. * p-value &lt; 0.05, ** p-value &lt; 0.01, and *** p-value &lt; 0.001. BSR-seq, Bulk segregant RNA-sequencing; FDR, false discovery rate; HLA, high levels of ammonium consumption; LLA, low levels of ammonium consumption; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1280902443" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;BSR-Seq experimental approach. (A) Experimental strategy followed to obtained mRNA counts and SNP frequencies from HLA and LLA pools. (B) Pathways enriched (FDR &lt; 5%) in genes differently expressed between pools. (C) CO&lt;sub&gt;2&lt;/sub&gt; lost (g/L) in HLA (black) and LLA (dotted gray lines) pools. SDs are shown for each measured time-point. (D) Nitrogen consumption (mgN/L) in HLA (black) and LLA (gray) pools after 6 d of fermentation. * p-value &lt; 0.05, ** p-value &lt; 0.01, and *** p-value &lt; 0.001. BSR-seq, Bulk segregant RNA-sequencing; FDR, false discovery rate; HLA, high levels of ammonium consumption; LLA, low levels of ammonium consumption; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1693/F3.medium.gif" width="306" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/6/1693/F3.medium.gif" width="306" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484821" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">BSR-Seq experimental approach. (A) Experimental strategy followed to obtained mRNA counts and SNP frequencies from HLA and LLA pools. (B) Pathways enriched (FDR &lt; 5%) in genes differently expressed between pools. (C) CO<sub>2</sub> lost (g/L) in HLA (black) and LLA (dotted gray lines) pools. SDs are shown for each measured time-point. (D) Nitrogen consumption (mgN/L) in HLA (black) and LLA (gray) pools after 6 d of fermentation. * p-value &lt; 0.05, ** p-value &lt; 0.01, and *** p-value &lt; 0.001. BSR-seq, Bulk segregant RNA-sequencing; FDR, false discovery rate; HLA, high levels of ammonium consumption; LLA, low levels of ammonium consumption; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Subsequently, we utilized the KEGG database to determine what pathways could be enriched within this subset of genes. We found several implicated fermentation-related pathways associated with the differences found between pools, representing interesting candidates for gene validation (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S7A</a>). Among others, we found carbon metabolism-related traits such as “glycolysis” and “citrate cycle (TCA cycle)” together with nitrogen metabolism traits such a: “phenylalanine, tyrosine, and tryptophan biosynthesis” and “alanine, aspartate, and glutamate metabolism,” with 10 and 12 genes, respectively, the majority of which were upregulated in the HLA pool. Indeed, these HLA segregants showed greater CO<sub>2</sub> release levels (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>), finished the fermentation earlier, and consumed 5.4 times more ammonium than LLA segregants, demonstrating their greater fermentation capacity due to their higher consumption ability (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Read counts in genes differentially expressed between pools. Reads ratio [WE/ (NA + SA + WA)] for HLA (black) and LLA (gray) pools (upper panel) are shown. The number of reads per genotype, WE (orange) and NA + SA + WA (multi-color) bars, are depicted in the bottom panel. (A) SNP XII.232561 (chromosome XII &#x2013; position 232561) located in PDC1 and (B) SNP XIII. 179112 located in GSF2. HLA, high levels of ammonium consumption; LLA, low levels of ammonium consumption; NA, North American; SA, Sake; SNP, single nucleotide polymorphism; WA, West African; WE, Wine/European." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1280902443" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Read counts in genes differentially expressed between pools. Reads ratio [WE/ (NA + SA + WA)] for HLA (black) and LLA (gray) pools (upper panel) are shown. The number of reads per genotype, WE (orange) and NA + SA + WA (multi-color) bars, are depicted in the bottom panel. (A) SNP XII.232561 (chromosome XII &#x2013; position 232561) located in &lt;em&gt;PDC1&lt;/em&gt; and (B) SNP XIII. 179112 located in &lt;em&gt;GSF2&lt;/em&gt;. HLA, high levels of ammonium consumption; LLA, low levels of ammonium consumption; NA, North American; SA, Sake; SNP, single nucleotide polymorphism; WA, West African; WE, Wine/European.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1693/F4.medium.gif" width="440" height="287"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/6/1693/F4.medium.gif" width="440" height="287"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484938" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">Read counts in genes differentially expressed between pools. Reads ratio [WE/ (NA + SA + WA)] for HLA (black) and LLA (gray) pools (upper panel) are shown. The number of reads per genotype, WE (orange) and NA + SA + WA (multi-color) bars, are depicted in the bottom panel. (A) SNP XII.232561 (chromosome XII – position 232561) located in <em>PDC1</em> and (B) SNP XIII. 179112 located in <em>GSF2</em>. HLA, high levels of ammonium consumption; LLA, low levels of ammonium consumption; NA, North American; SA, Sake; SNP, single nucleotide polymorphism; WA, West African; WE, Wine/European.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Within each pathway, several genes emerged as causative candidates for the consumption differences between individuals. To help us elucidate causative genes and increase the power of our study, we initially focused the analysis on differentially expressed genes within nitrogen-related pathways such as: “phenylalanine, tyrosine, and tryptophan biosynthesis” together with the “alanine, aspartate, and glutamate metabolism” pathways and cross-referenced this data with the QTL mapping results. Moreover, to augment the likelihood of finding common regions, a p-value &lt; 10<sup>−6</sup> (−logPvalue &gt; 6) was used for all phenotypes. Eight genes present in both datasets were found (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S7B</a>). Interestingly, <em><a href="http://www.yeastgenome.org/locus/S000002534/overview">ARO1</a></em>, which encodes for a protein involved in chorismate biosynthesis (a precursor of aromatic amino acids), represents a good candidate since it emerged with a p-value &lt; 10<sup>−7</sup> in the QTL mapping approach for ammonium and is upregulated in the HLA pool with respect to the LLA pool (0.82 log<sub>2</sub>-fold change). Similarly, <em><a href="http://www.yeastgenome.org/locus/S000001378/overview">HIS5</a></em>, which encodes for a histidinol-phosphate aminotransferase and is involved in the general control of amino acid biosynthesis, showed a p-value &lt; 10<sup>−9</sup> for isoleucine and is upregulated in HLA. Altogether, these RNA-seq results complement the QTL mapping approach and provide relevant genetic and biological insights toward the understanding of nitrogen source preferences and niche-ecological adaptation.</p></div><div id="sec-13" class="subsection"><h3>Identification of SNPs highly enriched in pools of segregants</h3><p id="p-28">The BSR-seq approach enables the finding of differentially expressed genes between pools, but also allelic variants highly represented at the RNA level. These variants could be present because of allele enrichment in the pool of segregants or due to differences in expression levels, which consequently affect allele frequency distribution. In order to identify SNPs enriched within either pool that could be associated with the ammonium trait and find an additional set of genes that could complement the previous findings, we looked for single variants (utilizing samtools) by comparing the reads to the reference assembly (S288c). Overall, we identified 29,934 SNPs found in all four pools (HLA and LLA in duplicates), out of which only SNPs that were present in at least 10 reads within each pool were considered (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S8</a>). Next, we compared allele frequency ratios (#references/#alternative reads per nucleotide) between HLA and LLA pools through a Fisher exact test (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Figure">Figure S3</a>). A total of 2231 SNPs were found to be differentially represented between pools at a 0.001 FDR (&lt; 2.2 false positives expected by chance). These SNPs belonged to 1460 genes and KEGG pathways analysis indicated that many are part of fermentation relevant traits (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S9</a>). Examples include: “Biosynthesis of secondary metabolites” (FDR = 0.0078) and “Biosynthesis of amino acids” (FDR = 0.01), extending the repertoire of functional variants for molecular validation. In this context, <em><a href="http://www.yeastgenome.org/locus/S000004034/overview">PDC1</a></em>, a pyruvate decarboxylase that represents a key enzyme in alcoholic fermentation, was found within several of these pathways where the WE variant is highly represented in the HLA pool with respect to the LLA pool (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>, FDR = 10<sup>−14</sup>), and could partly explain its adaptation and affinity for ammonium consumption.</p><p id="p-29">Consequently, we generated bulk expression frequency ratios between pools (HLA/LLA) and found 626 genes with ratios &gt; 5 in either direction (335 HLA &gt; LLA and 291 for LLA &gt; HLA, respectively). Among those genes exhibiting extreme HLA/LLA ratios, we found <em><a href="http://www.yeastgenome.org/locus/S000004511/overview">GSF2</a></em>, an integral membrane protein that may promote secretion of certain hexose transporters (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). Moreover, the WE variant is highly represented in the HLA pool, suggesting a greater fermentation capacity of this allelic variant.</p></div><div id="sec-14" class="subsection"><h3>Candidate validation derived from BSR-seq analysis</h3><p id="p-30">Allelic variants that emerged from our BSR-seq analysis were evaluated through reciprocal hemizygosity. <em><a href="http://www.yeastgenome.org/locus/S000002534/overview">ARO1</a></em>, which encodes for a multifunctional enzyme involved in amino acid metabolism was found within a marginally significant QTL for ammonium at position IV.713 (−logPvalue = 6.94) and differentially expressed between pools (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S6</a>), where the SA allele was overrepresented in the LLA pool. Thus, we chose to use the WE × SA cross since the WE genotype was enriched in those SGRP-4X segregants exhibiting high ammonium consumption levels. Molecular validation in reciprocal hemizygotes demonstrated phenotypic differences for four amino acids (glutamic acid, tryptophan, leucine, and isoleucine) between allelic variants. Interestingly, glutamic acid and leucine were highly consumed by <em><a href="http://www.yeastgenome.org/locus/S000002534/overview">ARO1</a><sup>SA</sup></em>, while the aromatic amino acid tryptophan and the nonpolar isoleucine were preferentially assimilated by <em><a href="http://www.yeastgenome.org/locus/S000002534/overview">ARO1</a><sup>WE</sup></em>, in agreement with the “phenylalanine, tyrosine, and tryptophan biosynthesis” KEGG pathway findings (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S7</a>). In contrast, we did not find differences for ammonium ion consumption, suggesting that <em><a href="http://www.yeastgenome.org/locus/S000002534/overview">ARO1</a></em> might not be a direct causal factor for differences for this nitrogen source, but rather an intermediate with consequences upon the consumption of other sources.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Reciprocal hemizygosity analysis on PDC1 and ARO1 genes mapped through BSR-seq approach. The hybrid hemizygote strains consumption levels (mg/L) are shown. *, **, and *** represent a significant statistical difference between the hemizygote strains for the same gene using a t-test; * p-value &lt; 0.05, ** p-value &lt; 0.01, and *** p-value &lt; 0.001. Reciprocal hemizygotes for (A) NA and WE PDC1, and (B) ARO1 WA and WE. &#x394;/WE denotes hemizygotes carrying the WE allele, while XX/&#x394; denotes hemizygotes carrying the alternative allele. BSR-seq, Bulk segregant RNA-sequencing; NA, North American; WA, West African; WE, Wine/European." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1280902443" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Reciprocal hemizygosity analysis on &lt;em&gt;PDC1&lt;/em&gt; and &lt;em&gt;ARO1&lt;/em&gt; genes mapped through BSR-seq approach. The hybrid hemizygote strains consumption levels (mg/L) are shown. *, **, and *** represent a significant statistical difference between the hemizygote strains for the same gene using a &lt;em&gt;t&lt;/em&gt;-test; * p-value &lt; 0.05, ** p-value &lt; 0.01, and *** p-value &lt; 0.001. Reciprocal hemizygotes for (A) NA and WE &lt;em&gt;PDC1&lt;/em&gt;, and (B) &lt;em&gt;ARO1&lt;/em&gt; WA and WE. &#x394;/WE denotes hemizygotes carrying the WE allele, while XX/&#x394; denotes hemizygotes carrying the alternative allele. BSR-seq, Bulk segregant RNA-sequencing; NA, North American; WA, West African; WE, Wine/European.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1693/F5.medium.gif" width="440" height="306"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/6/1693/F5.medium.gif" width="440" height="306"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1693/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484926" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-31" class="first-child">Reciprocal hemizygosity analysis on <em>PDC1</em> and <em>ARO1</em> genes mapped through BSR-seq approach. The hybrid hemizygote strains consumption levels (mg/L) are shown. *, **, and *** represent a significant statistical difference between the hemizygote strains for the same gene using a <em>t</em>-test; * p-value &lt; 0.05, ** p-value &lt; 0.01, and *** p-value &lt; 0.001. Reciprocal hemizygotes for (A) NA and WE <em>PDC1</em>, and (B) <em>ARO1</em> WA and WE. Δ/WE denotes hemizygotes carrying the WE allele, while XX/Δ denotes hemizygotes carrying the alternative allele. BSR-seq, Bulk segregant RNA-sequencing; NA, North American; WA, West African; WE, Wine/European.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Similarly, <em><a href="http://www.yeastgenome.org/locus/S000004034/overview">PDC1</a></em>, a pyruvate decarboxylase, was chosen based on allelic frequency differences between HLA and LLA pools and its role in several overrepresented pathways (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S9</a>). In this case, the WE × NA hybrid was evaluated given the high frequency of the WE allele in the HLA pool. Reciprocal hemizygotes for <em><a href="http://www.yeastgenome.org/locus/S000004034/overview">PDC1</a></em> showed statistically significant differences for tryptophan and tyrosine (p-value &lt; 0.05, <em>t</em>-test), but not for ammonium as might have been expected (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). In this case, <em><a href="http://www.yeastgenome.org/locus/S000004034/overview">PDC1</a><sup>WE</sup></em> showed a 6.2 and 13.1% greater consumption level for tryptophan and tyrosine, respectively.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-33">The utilization of biparental crosses has been the main strategy utilized for QTL mapping and the identification of regions underlying complex traits, mostly because the limited number of sequenced strains has impeded other approaches (for example genome-wide association studies) (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Liti and Louis 2012</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Cubillos 2016</a>). However, in most cases, mapped regions are limited to a few hotspots, such as: <em><a href="http://www.yeastgenome.org/locus/S000004246/overview">HAP1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005029/overview">MKT1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000344/overview">IRA1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005441/overview">IRA2</a></em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ehrenreich <em>et al.</em> 2009</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Wang and Kruglyak 2014</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Breunig <em>et al.</em> 2014</a>). This is likely due to the utilization of a small set of biparental crosses between laboratory and wine domesticated strains, where a few polymorphisms have significant effects upon a wide set of phenotypes. In this context, the need for a greater panel of strains to understand the ecology, biology, genetics, and evolution of <em>S. cerevisiae</em> is vital.</p><p id="p-34">To overcome the power limitation of genetically restricted biparental crosses, we previously reported the generation of SGRP-4X. This <em>S. cerevisiae</em> mapping population was obtained by outcrossing four founders representative of the main lineages for 12 generations (<a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">Cubillos <em>et al.</em> 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Burke <em>et al.</em> 2014</a>), and contains 165 segregants with fine-grained mosaic genomes that have been sequenced for linkage mapping studies. We took advantage of the enormous sequence diversity of this population to unveil a greater number of genetic variants underlying nitrogen consumption differences between wild strains from different origins. Nitrogen consumption traits are important since: (i) nitrogen is an essential nutrient for any kind of fermentation, (ii) consumption profiles are genetically determined at the beginning of the fermentation process, and (iii) preferences are generally strain-dependent (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Fleet 2003</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">2008</a>). Previous studies utilizing the four parental strains used in this study (and others) clearly demonstrate that natural strains vary greatly in their capacity to use nitrogen sources, likely due to differences in nitrogen metabolism, pathways, and more precisely in Ssy1-Ptr3-Ssy5 (SPS) and NCR systems between strains (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Ibstedt <em>et al.</em> 2015</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Crepin <em>et al.</em> 2012</a>). These differences could be explained as an adaptive response to a specific nitrogen environment, particularly for the strains that have been isolated from a wide range of conditions (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Liti <em>et al.</em> 2009</a>).</p><p id="p-35">In this study, we were able to extend the number of reported variants known to influence nitrogen consumption and specifically demonstrate that allelic variants in four candidate genes, mapped within three QTL for arginine, had an impact on YAN consumption performances (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). Our findings demonstrate that all tested genes have a mild, but significant effect on nitrogen assimilation. This can be explained by the extreme complexity of this population, with many allelic variants from different genetic backgrounds interacting within a single individual, buffering large effect alleles, and minimizing their impact upon phenotypes. Classical QTL mapping studies in yeast using two-parent crosses identify a few large effect loci per trait, missing small effect QTL (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bloom <em>et al.</em> 2015</a>), complex gene–gene interactions (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Ono <em>et al.</em> 2017</a>), and a great fraction of pathways’ response programs (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Gutin <em>et al.</em> 2015</a>), which altogether shape an individual’s phenotype. Our results validate the role of several genes in arginine consumption and their likely allele–allele interactions with a functional compensation, difficult to detect in quantitative genetic and genomics studies utilizing two-parent crosses.</p><p id="p-36">Interestingly, ammonium ion consumption was unaffected by the inactivation of the four alleles (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S5</a>), likely explained by the fact that ammonium ions are transported by three specific MEP permeases, namely <a href="http://www.yeastgenome.org/locus/S000003353/overview">Mep1p</a>, <a href="http://www.yeastgenome.org/locus/S000005086/overview">Mep2p</a>, and <a href="http://www.yeastgenome.org/locus/S000006342/overview">Mep3p</a> (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Marini <em>et al.</em> 1994</a>, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">1997</a>), which were not mapped in our study. For three of the four genes, allelic variants with a positive effect on amino acid consumption belong to the WE genetic background, in agreement with the fact that this parental strain presents the highest YAN consumption and demonstrating that it likely possesses a selective advantage for oenological conditions. In this context, the WE background may have evolved a series of mechanisms and pathways to enforce the consumption capacity of overrepresented nitrogen sources (<em>i.e.</em>, ammonium, glutamine, and arginine) within wine must. Indeed, the BSR-Seq strategy demonstrated that carbon source assimilation pathways and nitrogen metabolism were significantly overrepresented in WE allelic variants and differentially expressed compared to other allelic backgrounds, demonstrating the underlying molecular mechanisms behind the greater fermentation profile in HLA segregants (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S7</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S9</a>). Differences in assimilation profiles and adaptation mechanisms could be driven by the availability of nitrogen sources within each strain’s niche. These key elements interact to determine ecological relationships between nutrient sources and strains’ adaptation capacity. In this context, a series of genomic changes have been associated with niche adaptation in wine yeast strains, such as <em><a href="http://www.yeastgenome.org/locus/S000006013/overview">SSU1</a></em> (a sulfite-nitrite pump) translocations related to sulfite resistance and copy number variation of <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP-1</a></em> (encoding a copper-binding metallothionein) implicated in elevated copper tolerance (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Warringer <em>et al.</em> 2011</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Marsit and Dequin 2015</a>).</p><p id="p-37">The comparison of the RNA-seq results and cross information with the QTL mapping strategy provided a set of genes enriched for metabolic pathways that could explain how strains were able to evolve and adapt to their preferred nitrogen sources and partly explain ecological differentiation. In particular, through RNA-seq, we were able to identify <em><a href="http://www.yeastgenome.org/locus/S000004034/overview">PDC1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002534/overview">ARO1</a></em> as candidate genes. <em><a href="http://www.yeastgenome.org/locus/S000004034/overview">PDC1</a></em> is a pyruvate decarboxylase <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kellermann <em>et al.</em> (1986)</a>. This enzyme has a key role in the glycolytic pathway, which is essential for directing the glucose flux to ethanol production (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Williamson <em>et al.</em> 1980</a>). Coupled with two other decarboxylase proteins, <a href="http://www.yeastgenome.org/locus/S000004124/overview">Pdc5p</a> and <a href="http://www.yeastgenome.org/locus/S000003319/overview">Pdc6p</a>, <a href="http://www.yeastgenome.org/locus/S000004034/overview">Pdc1p</a> contributes to the catabolism of branched amino acids (isoleucine and valine) and aromatic amino acids (phenylalanine and tryptophan) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Dickinson <em>et al.</em> 2003</a>). In our study, we found assimilation differences between <em><a href="http://www.yeastgenome.org/locus/S000004034/overview">PDC1</a></em> allelic variants for tyrosine (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1/Table">Table S5</a>). Likewise, <em><a href="http://www.yeastgenome.org/locus/S000002534/overview">ARO1</a></em> was found to underlie differences for several amino acids (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). This gene encodes for an enzyme involved in the biosynthesis of aromatic amino acid precursors playing a central role in the superpathway of phenylalanine, tyrosine, and tryptophan biosynthesis (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Duncan <em>et al.</em> 1987</a>). Most of the genes encoding for the aromatic amino acid biosynthetic enzymes are regulated by the transcriptional activator <a href="http://www.yeastgenome.org/locus/S000000735/overview">Gcn4p</a> which is a key player of the TOR pathway. <a href="http://www.yeastgenome.org/locus/S000000735/overview">Gcn4p</a> participates in the regulation of the metabolic pathway GAAC (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hinnebusch and Fink 1983</a>) and is potentially involved in the regulation of the NCR pathway (<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Tesniere <em>et al.</em> 2015</a>). Besides its association with the TOR pathway, it was demonstrated that mutations within <em><a href="http://www.yeastgenome.org/locus/S000002534/overview">ARO1</a></em> can cause the accumulation of intermediate metabolites (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Lucchini <em>et al.</em> 1978</a>) and disrupt the biosynthesis of certain amino acids. This disruption of the intracellular nitrogen pool could have repercussions on TOR sensing and signaling.</p><p id="p-38">The QTL mapping strategy provided another set of genes that are also related to the TOR, SPS, and NCR pathways. <em><a href="http://www.yeastgenome.org/locus/S000005103/overview">ASI2</a></em> is involved in the SPS signaling pathway through the transcription factor <a href="http://www.yeastgenome.org/locus/S000002871/overview">Stp1p</a> (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Zargari <em>et al.</em> 2007</a>). <a href="http://www.yeastgenome.org/locus/S000005103/overview">Asi2p</a> acts in a multi-protein complex together with <a href="http://www.yeastgenome.org/locus/S000004725/overview">Asi1p</a> and <a href="http://www.yeastgenome.org/locus/S000004953/overview">Asi3p</a> to ensure the control of the transcription factors <a href="http://www.yeastgenome.org/locus/S000002871/overview">Stp1p</a> and <a href="http://www.yeastgenome.org/locus/S000001048/overview">Stp2p</a>, regulating the stability of the SPS sensor system (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Omnus and Ljungdahl 2014</a>). In <em>S. cerevisiae</em>, the SPS signaling pathway enables cells to respond to the presence of extracellular amino acids and induce their uptake rate (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Ljungdahl and Daignan-Fornier 2012</a>). It is well known that the SPS system is upregulated at the beginning of the fermentation process (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ljungdahl 2009</a>), suggesting that variation in <a href="http://www.yeastgenome.org/locus/S000005103/overview">Asi2p</a> could modulate the SPS activity, resulting in gene expression differences associated with this system and impacting amino acid assimilation. On the other hand, <em><a href="http://www.yeastgenome.org/locus/S000005212/overview">LYP1</a></em> encodes for a lysine permease whose activity is regulated by the SPS pathway. Lysine is an important amino acid for yeast nitrogen metabolism and fermentation performance (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lei <em>et al.</em> 2013</a>), and the identification of allelic variants of <em><a href="http://www.yeastgenome.org/locus/S000005212/overview">LYP1</a></em> supports the hypothesis of variation in SPS activity between parental strains. Our findings correlate well with this previous work, which demonstrated that lysine supplementation improves the expression of SPS-regulated genes, suggesting a fine modulation of the SPS system for the detection of extracellular amino acids. We were also able to validate the role of <em><a href="http://www.yeastgenome.org/locus/S000005214/overview">ALP1</a></em>, encoding for an arginine transporter controlled by the NCR system. The nitrogen environment changes during fermentation, and yeast cells need to readapt their nitrogen metabolism (<a id="xref-ref-83-3" class="xref-bibr" href="#ref-83">Tesniere <em>et al.</em> 2015</a>). To achieve this, the NCR system is regulated by TOR activity, and previous studies have demonstrated that strains can have differences in TOR that impact their capacity to ferment (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Brice <em>et al.</em> 2014a</a>). Interestingly, <em><a href="http://www.yeastgenome.org/locus/S000005212/overview">LYP1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000005214/overview">ALP1</a></em> have high sequence homology and are adjacent (separated by 881 bp) (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Sychrova and Chevallier 1994</a>). However, they are controlled by two distinct nitrogen regulation mechanisms (SPS and NCR, respectively), demonstrating the architectural complexity of the phenotype. Similarly, we validated the role of <em><a href="http://www.yeastgenome.org/locus/S000003708/overview">CPS1</a></em>, which encodes a permease involved in the degradation of proteins into amino acids, to provide nitrogen sources for the cell (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Hofman-Bang 1999</a>). Expression of <em><a href="http://www.yeastgenome.org/locus/S000003708/overview">CPS1</a></em> is under the influence of NCR and more precisely the transcription factor <a href="http://www.yeastgenome.org/locus/S000000842/overview">Gln3p</a> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bordallo and Suarez-Rendueles 1993</a>), which is in agreement with NCR being upregulated in the presence of arginine (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Godard <em>et al.</em> 2007</a>). Changes in <em><a href="http://www.yeastgenome.org/locus/S000003708/overview">CPS1</a></em> activity can perturb the equilibrium of the nitrogen pool, impacting the TOR pathway and affecting NCR activity together with YAN consumption (<a id="xref-ref-83-4" class="xref-bibr" href="#ref-83">Tesniere <em>et al.</em> 2015</a>). In connection with these results, the pyruvate decarboxylase activity of <a href="http://www.yeastgenome.org/locus/S000004034/overview">Pdc1</a> is regulated by the protein phosphatase, <a href="http://www.yeastgenome.org/locus/S000002205/overview">Sit4p</a>, which is also an important component of the TOR pathway. <a href="http://www.yeastgenome.org/locus/S000002205/overview">Sit4p</a> is likewise involved in the regulation of the NCR system through <a href="http://www.yeastgenome.org/locus/S000000842/overview">Gln3</a>. <em><a href="http://www.yeastgenome.org/locus/S000004034/overview">PDC1</a></em> is a known candidate for regulation of fermentative activity (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">de Assis <em>et al.</em> 2013</a>) and it is very likely that, in our study, variation in <em><a href="http://www.yeastgenome.org/locus/S000004034/overview">PDC1</a></em> gene expression constitutes a marker determining the fermentative efficiency of a strain. This is coherent with the overexpression of <em><a href="http://www.yeastgenome.org/locus/S000004034/overview">PDC1</a></em> in the WE parental strain (HLA pool), which is consistently the most suitable strain in the oenological fermentation environment. We can, thus, claim that, under oenological conditions, the WE strain shows the best capacity to use some of the available amino acids through the modulation of the TOR pathway sensing system.</p><div id="sec-16" class="subsection"><h3>Conclusions</h3><p id="p-39">In this study, we identified a series of QTL and candidate genes through an integrative QTL-mapping–BSR-seq approach utilizing a complex AIL multi-parent population denominated SGRP-4X. The causal variants for <em><a href="http://www.yeastgenome.org/locus/S000003708/overview">CPS1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005212/overview">LYP1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005214/overview">ALP1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005103/overview">ASI2</a></em> are responsible for mild nitrogen assimilation differences for arginine, demonstrating how complex multi-parent populations can untangle small effect sources of natural variation. Our molecular validation demonstrates the role of several genes, which can confluence through multiple molecular mechanisms with significant effects on nitrogen assimilation, gene expression differences, and wine fermentation. In this context, these results support variations in nitrogen signaling pathways, and more precisely in SPS and NCR as the main systems responsible for differences in YAN consumption between strains from various origins. The SPS and NCR systems are key factors controlling nitrogen consumption and are highly conserved in <em>S. cerevisiae</em> (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Crepin <em>et al.</em> 2012</a>). In contrast, nitrogen metabolism regulation differs between strains (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Gutierrez <em>et al.</em> 2013</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Brice <em>et al.</em> 2014a</a>; <a id="xref-ref-43-7" class="xref-bibr" href="#ref-43">Jara <em>et al.</em> 2014</a>), and variations in these systems could be linked to differences in the specific nitrogen environments causing ecological differentiation of these strains. Our results provide evidence of the molecular changes leading to niche adaptation with important implications for evolution and quantitative genomics in yeast.</p></div></div><div class="section" id="sec-17"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">This work was supported by grants from the Comisión Nacional de Investigación Científica y Tecnológica (CONICYT) Fondo Nacional de Desarollo Científico y Tecnológico (FONDECYT) (grants 11140097 to F.A.C., 1150522 to C.M., 3150159 to C.B., and 11130148 to V.G.; Millennium Nucleus in Fungal Integrative and Systems Biology (MN-FISB) grant 120043 to F.A.C., Programa de Cooperación Científica ECOS/CONICYT (C13B02) to F.A.C., C.M., and G.L.; and Proyecto Basal USA 1555 - Vridei O21771CR_CONT Universidad de Santiago de Chile.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-41">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042127/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-42"><em>Communicating editor: K. Nichols</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-04"><span class="received-label">Received </span>January 4, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-03-31"><span class="accepted-label">Accepted </span>March 31, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Cubillos <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.1" data-doi="10.1186/gb-2010-11-10-r106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Differential expression analysis for sequence count data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">R106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAnders%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253DR106%26rft.epage%253DR106%26rft.atitle%253DDifferential%2Bexpression%2Banalysis%2Bfor%2Bsequence%2Bcount%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-10-r106%26rft_id%253Dinfo%253Apmid%252F20979621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-10-r106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20979621&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.2" data-doi="10.1093/bioinformatics/btu638"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyl</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">HTSeq–a Python framework to work with high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">166</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu638%26rft_id%253Dinfo%253Apmid%252F25260700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIzMS8yLzE2NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.3" data-doi="10.1002/yea.1809"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arroyo-Lopez</span>  <span class="cit-name-given-names">F. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salvado</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tronchoni</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillamon</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrio</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Susceptibility and resistance to ethanol in Saccharomyces strains isolated from wild and fermentative environments.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1005</span>–<span class="cit-lpage">1015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DArroyo-Lopez%26rft.auinit1%253DF.%2BN.%26rft.volume%253D27%26rft.issue%253D12%26rft.spage%253D1005%26rft.epage%253D1015%26rft.atitle%253DSusceptibility%2Band%2Bresistance%2Bto%2Bethanol%2Bin%2BSaccharomyces%2Bstrains%2Bisolated%2Bfrom%2Bwild%2Band%2Bfermentative%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1809%26rft_id%253Dinfo%253Apmid%252F20824889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1809&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20824889&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-web" id="cit-7.6.1693.4"><div class="cit-metadata unstructured">Bass, J., A. Dabney, and D. Robinson, 2015 qvalue: Q-value Estimation for False Discovery Rate Control. R package version 2.6.0. <a href="http://github.com/jdstorey/qvalue">http://github.com/jdstorey/qvalue</a>. Accessed: November 1, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.5" data-doi="10.1078/0723-2020-00097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beltran</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torija</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poblet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Analysis of yeast populations during alcoholic fermentation: a six year follow-up study.</span> <abbr class="cit-jnl-abbrev">Syst. Appl. Microbiol.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">287</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Band%2BApplied%2BMicrobiology%26rft.stitle%253DSystematic%2Band%2BApplied%2BMicrobiology%26rft.aulast%253DBeltran%26rft.auinit1%253DG.%26rft.volume%253D25%26rft.issue%253D2%26rft.spage%253D287%26rft.epage%253D293%26rft.atitle%253DAnalysis%2Bof%2Byeast%2Bpopulations%2Bduring%2Balcoholic%2Bfermentation%253A%2Ba%2Bsix%2Byear%2Bfollow-up%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1078%252F0723-2020-00097%26rft_id%253Dinfo%253Apmid%252F12353885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1078/0723-2020-00097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12353885&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178458000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.6" data-doi="10.1371/journal.pgen.0020195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ben-Ari</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenvirth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klutstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Four linked genes participate in controlling sporulation efficiency in budding yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBen-Ari%26rft.auinit1%253DG.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253De195%26rft.epage%253De195%26rft.atitle%253DFour%2Blinked%2Bgenes%2Bparticipate%2Bin%2Bcontrolling%2Bsporulation%2Befficiency%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020195%26rft_id%253Dinfo%253Apmid%252F17112318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17112318&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D85%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.8" data-doi="10.1093/molbev/msu037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barre</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-definition view of functional genetic variation from natural yeast genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">872</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu037%26rft_id%253Dinfo%253Apmid%252F24425782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMxLzQvODcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomewide markers as cofactors for precision mapping of quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">999</span>–<span class="cit-lpage">1009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.10" data-doi="10.1038/ncomms9712"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kotenko</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sadhu</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Treusch</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic interactions contribute less than additive effects to quantitative trait variation in yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">8712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D8712%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms9712%26rft_id%253Dinfo%253Apmid%252F26537231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms9712&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26537231&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.11" data-doi="10.1007/s00335-015-9595-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bogue</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Collaborative cross and diversity outbred data resources in the mouse phenome database.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">9–10</span>): <span class="cit-fpage">511</span>–<span class="cit-lpage">520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D26%26rft.spage%253D511%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-015-9595-6%26rft_id%253Dinfo%253Apmid%252F26286858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-015-9595-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26286858&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.12" data-doi="10.1002/yea.320090404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bordallo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suarez-Rendueles</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Control of Saccharomyces cerevisiae carboxypeptidase S (CPS1) gene expression under nutrient limitation.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">339</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DBordallo%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D339%26rft.epage%253D349%26rft.atitle%253DControl%2Bof%2BSaccharomyces%2Bcerevisiae%2Bcarboxypeptidase%2BS%2B%2528CPS1%2529%2Bgene%2Bexpression%2Bunder%2Bnutrient%2Blimitation.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320090404%26rft_id%253Dinfo%253Apmid%252F8511964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320090404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8511964&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LA17000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.13" data-doi="10.1534/genetics.106.057927"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brauer</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christianson</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mapping novel traits by array-assisted bulk segregant analysis in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1813</span>–<span class="cit-lpage">1816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.057927%26rft_id%253Dinfo%253Apmid%252F16624899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE4MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.14" data-doi="10.1371/journal.pgen.1004142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breunig</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabinowitz</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic basis of metabolome variation in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e1004142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004142%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004142%26rft_id%253Dinfo%253Apmid%252F24603560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24603560&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brice</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigey</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legras</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">A genetic approach of wine yeast fermentation capacity in nitrogen-starvation reveals the key role of nitrogen signaling.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.16" data-doi="10.1128/AEM.03856-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brice</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesniere</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Assessing the mechanisms responsible for differences between nitrogen requirements of saccharomyces cerevisiae wine yeasts in alcoholic fermentation.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">80</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1330</span>–<span class="cit-lpage">1339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.03856-13%26rft_id%253Dinfo%253Apmid%252F24334661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjgwLzQvMTMzMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.17" data-doi="10.1093/molbev/msu256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Standing genetic variation drives repeatable experimental evolution in outcrossing populations of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">3228</span>–<span class="cit-lpage">3239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu256%26rft_id%253Dinfo%253Apmid%252F25172959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xMi8zMjI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.18" data-doi="10.1128/AEM.02294-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crepin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nidelet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dequin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camarasa</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sequential use of nitrogen compounds by Saccharomyces cerevisiae during wine fermentation: a model based on kinetic and regulation characteristics of nitrogen permeases.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">78</span>(<span class="cit-issue">22</span>): <span class="cit-fpage">8102</span>–<span class="cit-lpage">8111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.02294-12%26rft_id%253Dinfo%253Apmid%252F22983966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3OC8yMi84MTAyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Exploiting budding yeast natural variation for industrial processes.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">62</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">745</span>–<span class="cit-lpage">751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BGenet.%26rft.volume%253D62%26rft.spage%253D745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.20" data-doi="10.1111/j.1567-1364.2009.00583.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Generation of a large set of genetically tractable haploid and diploid Saccharomyces strains.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1217</span>–<span class="cit-lpage">1225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00583.x%26rft_id%253Dinfo%253Apmid%252F19840116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjg6IjkvOC8xMjE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.21" data-doi="10.1111/j.1365-294X.2011.05005.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Billi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorgo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fargier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Assessing the complex architecture of polygenic traits in diverged yeast populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1401</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DCubillos%26rft.auinit1%253DF.%2BA.%26rft.volume%253D20%26rft.issue%253D7%26rft.spage%253D1401%26rft.epage%253D1413%26rft.atitle%253DAssessing%2Bthe%2Bcomplex%2Barchitecture%2Bof%2Bpolygenic%2Btraits%2Bin%2Bdiverged%2Byeast%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05005.x%26rft_id%253Dinfo%253Apmid%252F21261765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05005.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21261765&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288705300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.22" data-doi="10.1534/genetics.113.155515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scovacricchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution mapping of complex traits with a four-parent advanced intercross yeast population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1141</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155515%26rft_id%253Dinfo%253Apmid%252F24037264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExNDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.23" data-doi="10.1111/1567-1364.12052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Assis</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zingali</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masuda</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodrigues</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montero-Lomeli</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pyruvate decarboxylase activity is regulated by the Ser/Thr protein phosphatase Sit4p in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">518</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1567-1364.12052%26rft_id%253Dinfo%253Apmid%252F23692511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjg6IjEzLzYvNTE4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.24" data-doi="10.1074/jbc.M211914200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickinson</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salgado</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hewlins</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The catabolism of amino acids to long chain and complex alcohols in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">278</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">8028</span>–<span class="cit-lpage">8034</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M211914200%26rft_id%253Dinfo%253Apmid%252F12499363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNzgvMTAvODAyOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.25" data-doi="10.1042/bj2460375"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duncan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coggins</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The pentafunctional arom enzyme of Saccharomyces cerevisiae is a mosaic of monofunctional domains.</span> <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">246</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">375</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2BJournal%26rft.stitle%253DBiochem.%2BJ.%26rft.aulast%253DDuncan%26rft.auinit1%253DK.%26rft.volume%253D246%26rft.issue%253D2%26rft.spage%253D375%26rft.epage%253D386%26rft.atitle%253DThe%2Bpentafunctional%2Barom%2Benzyme%2Bof%2BSaccharomyces%2Bcerevisiae%2Bis%2Ba%2Bmosaic%2Bof%2Bmonofunctional%2Bdomains%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbj2460375%26rft_id%253Dinfo%253Apmid%252F2825635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6OToiMjQ2LzIvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.26" data-doi="10.1101/gr.118786.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durrant</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tayem</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Collaborative cross mice and their power to map host susceptibility to Aspergillus fumigatus infection.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1239</span>–<span class="cit-lpage">1248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.118786.110%26rft_id%253Dinfo%253Apmid%252F21493779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIzOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.27" data-doi="10.1101/sqb.2009.74.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic dissection of complex traits in yeast: insights from studies of gene expression and other phenotypes in the BYxRM cross.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BSymp.%2BQuant.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fsqb.2009.74.013%26rft_id%253Dinfo%253Apmid%252F19734204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjg6Ijc0LzAvMTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.28" data-doi="10.1016/S0168-1605(03)00245-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fleet</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Yeast interactions and wine flavour.</span> <abbr class="cit-jnl-abbrev">Int. J. Food Microbiol.</abbr> <span class="cit-vol">86</span>(<span class="cit-issue">1–2</span>): <span class="cit-fpage">11</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bfood%2Bmicrobiology%26rft.stitle%253DInt%2BJ%2BFood%2BMicrobiol%26rft.aulast%253DFleet%26rft.auinit1%253DG.%2BH.%26rft.volume%253D86%26rft.issue%253D1-2%26rft.spage%253D11%26rft.epage%253D22%26rft.atitle%253DYeast%2Binteractions%2Band%2Bwine%2Bflavour.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-1605%252803%252900245-9%26rft_id%253Dinfo%253Apmid%252F12892919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-1605(03)00245-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12892919&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184727700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.29" data-doi="10.1111/j.1567-1364.2008.00427.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fleet</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Wine yeasts for the future.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">979</span>–<span class="cit-lpage">995</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2008.00427.x%26rft_id%253Dinfo%253Apmid%252F18793201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjgvNy85NzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forsberg</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andreasson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moliner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ljungdahl</span>  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Suppressors of ssy1 and ptr3 null mutations define novel amino acid sensor-independent genes in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">973</span>–<span class="cit-lpage">988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DForsberg%26rft.auinit1%253DH.%26rft.volume%253D158%26rft.issue%253D3%26rft.spage%253D973%26rft.epage%253D988%26rft.atitle%253DSuppressors%2Bof%2Bssy1%2Band%2Bptr3%2BNull%2BMutations%2BDefine%2BNovel%2BAmino%2BAcid%2BSensor-Independent%2BGenes%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F11454748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU4LzMvOTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.31" data-doi="10.1091/mbc.11.12.4241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmel-Harel</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genomic expression programs in the response of yeast cells to environmental changes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">4241</span>–<span class="cit-lpage">4257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DGasch%26rft.auinit1%253DA.%2BP.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D4241%26rft.epage%253D4257%26rft.atitle%253DGenomic%2BExpression%2BPrograms%2Bin%2Bthe%2BResponse%2Bof%2BYeast%2BCells%2Bto%2BEnvironmental%2BChanges%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.11.12.4241%26rft_id%253Dinfo%253Apmid%252F11102521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMi80MjQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.32" data-doi="10.1128/MCB.01084-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Godard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urrestarazu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vissers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kontos</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bontempi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Effect of 21 different nitrogen sources on global gene expression in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">3065</span>–<span class="cit-lpage">3086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01084-06%26rft_id%253Dinfo%253Apmid%252F17308034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMzA2NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.33" data-doi="10.1021/jf062820m"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Alonso</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hermosin-Gutierrez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Romero</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Simultaneous HPLC analysis of biogenic amines, amino acids, and ammonium ion as aminoenone derivatives in wine and beer samples.</span> <abbr class="cit-jnl-abbrev">J. Agric. Food Chem.</abbr> <span class="cit-vol">55</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">608</span>–<span class="cit-lpage">613</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAgricultural%2Band%2BFood%2BChemistry%26rft.stitle%253DJournal%2Bof%2BAgricultural%2Band%2BFood%2BChemistry%26rft.aulast%253DGomez-Alonso%26rft.auinit1%253DS.%26rft.volume%253D55%26rft.issue%253D3%26rft.spage%253D608%26rft.epage%253D613%26rft.atitle%253DSimultaneous%2BHPLC%2Banalysis%2Bof%2Bbiogenic%2Bamines%252C%2Bamino%2Bacids%252C%2Band%2Bammonium%2Bion%2Bas%2Baminoenone%2Bderivatives%2Bin%2Bwine%2Band%2Bbeer%2Bsamples.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjf062820m%26rft_id%253Dinfo%253Apmid%252F17263449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/jf062820m&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17263449&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243840200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.34" data-doi="10.1371/journal.pone.0067166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutierrez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beltran</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillamon</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic basis of variations in nitrogen source utilization in four wine commercial yeast strains.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e67166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De67166%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0067166%26rft_id%253Dinfo%253Apmid%252F23826223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0067166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23826223&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.35" data-doi="10.15252/msb.20156451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sadeh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rahat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aharoni</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Condition-specific genetic interaction maps reveal crosstalk between the cAMP/PKA and the HOG MAPK pathways in the activation of the general stress response.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DGutin%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D829%26rft.epage%253D829%26rft.atitle%253DCondition-specific%2Bgenetic%2Binteraction%2Bmaps%2Breveal%2Bcrosstalk%2Bbetween%2Bthe%2BcAMP%252FPKA%2Band%2Bthe%2BHOG%2BMAPK%2Bpathways%2Bin%2Bthe%2Bactivation%2Bof%2Bthe%2Bgeneral%2Bstress%2Bresponse%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.20156451%26rft_id%253Dinfo%253Apmid%252F26446933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjk6IjExLzEwLzgyOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.36" data-doi="10.1073/pnas.80.17.5374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hinnebusch</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Positive regulation in the general amino acid control of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">80</span>(<span class="cit-issue">17</span>): <span class="cit-fpage">5374</span>–<span class="cit-lpage">5378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHinnebusch%26rft.auinit1%253DA.%2BG.%26rft.volume%253D80%26rft.issue%253D17%26rft.spage%253D5374%26rft.epage%253D5378%26rft.atitle%253DPositive%2Bregulation%2Bin%2Bthe%2Bgeneral%2Bamino%2Bacid%2Bcontrol%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.80.17.5374%26rft_id%253Dinfo%253Apmid%252F6351059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODAvMTcvNTM3NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.37" data-doi="10.1385/MB:12:1:35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hofman-Bang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Nitrogen catabolite repression in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Biotechnol.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">35</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiotechnology%26rft.stitle%253DMol%2BBiotechnol%26rft.aulast%253DHofman-Bang%26rft.auinit1%253DJ.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D35%26rft.epage%253D73%26rft.atitle%253DNitrogen%2Bcatabolite%2Brepression%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1385%252FMB%253A12%253A1%253A35%26rft_id%253Dinfo%253Apmid%252F10554772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1385/MB:12:1:35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10554772&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083243200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.38" data-doi="10.1534/genetics.106.065292"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic dissection of ethanol tolerance in the budding yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1479</span>–<span class="cit-lpage">1487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.065292%26rft_id%253Dinfo%253Apmid%252F17194785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS8zLzE0NzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.39" data-doi="10.1073/pnas.1100465108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulo</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Effgen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keizer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Analysis of natural allelic variation in Arabidopsis using a multiparent recombinant inbred line population.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">4488</span>–<span class="cit-lpage">4493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1100465108%26rft_id%253Dinfo%253Apmid%252F21368205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzExLzQ0ODgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.40" data-doi="10.1038/nprot.2008.211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang da</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lempicki</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">44</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHuang%26rft.auinit1%253Dd.%2Ba.%2B%2B.%2BW.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D57%26rft.atitle%253DSystematic%2Band%2Bintegrative%2Banalysis%2Bof%2Blarge%2Bgene%2Blists%2Busing%2BDAVID%2Bbioinformatics%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.211%26rft_id%253Dinfo%253Apmid%252F19131956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2008.211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19131956&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265781800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.41" data-doi="10.1016/0168-9525(90)90190-H"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huxley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Rapid assessment of S. cerevisiae mating type by PCR.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DHuxley%26rft.auinit1%253DC.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253D236%26rft.epage%253D236%26rft.atitle%253DRapid%2Bassessment%2Bof%2BS.%2Bcerevisiae%2Bmating%2Btype%2Bby%2BPCR.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0168-9525%252890%252990190-H%26rft_id%253Dinfo%253Apmid%252F2238077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0168-9525(90)90190-H&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2238077&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DT54000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.42" data-doi="10.1093/molbev/msu285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ibstedt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stenberg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bages</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjuvsland</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Concerted evolution of life stage performances signals recent selection on yeast nitrogen use.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">153</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu285%26rft_id%253Dinfo%253Apmid%252F25349282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMyLzEvMTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.43" data-doi="10.1371/journal.pone.0086533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilera</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mapping genetic variants underlying differences in the central nitrogen metabolism in fermenter yeasts.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e86533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De86533%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0086533%26rft_id%253Dinfo%253Apmid%252F24466135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0086533&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24466135&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.44" data-doi="10.1016/j.jbiosc.2008.12.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katou</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Namise</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitagaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimoi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">QTL mapping of sake brewing characteristics of yeast.</span> <abbr class="cit-jnl-abbrev">J. Biosci. Bioeng.</abbr> <span class="cit-vol">107</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">383</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BBiosci%2BBioeng%26rft.aulast%253DKatou%26rft.auinit1%253DT.%26rft.volume%253D107%26rft.issue%253D4%26rft.spage%253D383%26rft.epage%253D393%26rft.atitle%253DQTL%2Bmapping%2Bof%2Bsake%2Bbrewing%2Bcharacteristics%2Bof%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jbiosc.2008.12.014%26rft_id%253Dinfo%253Apmid%252F19332297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jbiosc.2008.12.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19332297&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.45" data-doi="10.1093/nar/14.22.8963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellermann</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeboth</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollenberg</span>  <span class="cit-name-given-names">C. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Analysis of the primary structure and promoter function of a pyruvate decarboxylase gene (PDC1) from Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">22</span>): <span class="cit-fpage">8963</span>–<span class="cit-lpage">8977</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKellermann%26rft.auinit1%253DE.%26rft.volume%253D14%26rft.issue%253D22%26rft.spage%253D8963%26rft.epage%253D8977%26rft.atitle%253DAnalysIs%2Bof%2Bthe%2Bprimary%2Bstructure%2Band%2Bpromoter%2Bfunction%2Bof%2Ba%2Bpyruvate%2Bdecarbozylase%2Bgene%2B%2528PDCI%2529%2Bfrom%2BSaceharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F14.22.8963%26rft_id%253Dinfo%253Apmid%252F3537965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8yMi84OTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.46" data-doi="10.1073/pnas.0708194104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic variation in the cysteine biosynthesis pathway causes sensitivity to pharmacological compounds.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>(<span class="cit-issue">49</span>): <span class="cit-fpage">19387</span>–<span class="cit-lpage">19391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0708194104%26rft_id%253Dinfo%253Apmid%252F18042712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzQ5LzE5Mzg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.47" data-doi="10.1371/journal.pgen.1000551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kover</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trakalo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scarcelli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A multiparent advanced generation inter-cross to fine-map quantitative traits in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">e1000551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKover%26rft.auinit1%253DP.%2BX.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000551%26rft.epage%253De1000551%26rft.atitle%253DA%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2Bto%2Bfine-map%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000551%26rft_id%253Dinfo%253Apmid%252F19593375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593375&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.48" data-doi="10.1111/j.1365-294X.2007.03266.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legras</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merdinoglu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cornuet</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karst</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Bread, beer and wine: Saccharomyces cerevisiae diversity reflects human history.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">2091</span>–<span class="cit-lpage">2102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLegras%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D16%26rft.issue%253D10%26rft.spage%253D2091%26rft.epage%253D2102%26rft.atitle%253DBread%252C%2Bbeer%2Band%2Bwine%253A%2BSaccharomyces%2Bcerevisiae%2Bdiversity%2Breflects%2Bhuman%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03266.x%26rft_id%253Dinfo%253Apmid%252F17498234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03266.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17498234&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246366800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lei</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Effects of Lys and His supplementations on the regulation of nitrogen metabolism in lager yeast.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">97</span>(<span class="cit-issue">20</span>): <span class="cit-fpage">8913</span>–<span class="cit-lpage">8921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BMicrobiol.%2BBiotechnol.%26rft.volume%253D97%26rft.spage%253D8913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.50" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.51" data-doi="10.1371/journal.pgen.1002912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Advances in quantitative trait analysis in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">e1002912</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253De1002912%26rft.epage%253De1002912%26rft.atitle%253DAdvances%2Bin%2Bquantitative%2Btrait%2Banalysis%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002912%26rft_id%253Dinfo%253Apmid%252F22916041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002912&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22916041&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.52" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>(<span class="cit-issue">7236</span>): <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.53" data-doi="10.1042/BST0370242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ljungdahl</span>  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Amino-acid-induced signalling via the SPS-sensing pathway in yeast.</span> <abbr class="cit-jnl-abbrev">Biochem. Soc. Trans.</abbr> <span class="cit-vol">37</span>(<span class="cit-issue">Pt. 1</span>): <span class="cit-fpage">242</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BSoc.%2BTrans.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBST0370242%26rft_id%253Dinfo%253Apmid%252F19143640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoicHBiaW9zdCI7czo1OiJyZXNpZCI7czo4OiIzNy8xLzI0MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.54" data-doi="10.1534/genetics.111.133306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ljungdahl</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daignan-Fornier</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Regulation of amino acid, nucleotide, and phosphate metabolism in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">885</span>–<span class="cit-lpage">929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLjungdahl%26rft.auinit1%253DP.%2BO.%26rft.volume%253D190%26rft.issue%253D3%26rft.spage%253D885%26rft.epage%253D929%26rft.atitle%253DRegulation%2Bof%2BAmino%2BAcid%252C%2BNucleotide%252C%2Band%2BPhosphate%2BMetabolism%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.133306%26rft_id%253Dinfo%253Apmid%252F22419079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzMvODg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.55" data-doi="10.1016/j.tig.2014.07.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dissecting complex traits using the Drosophila synthetic population resource.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">488</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D30%26rft.spage%253D488%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.07.009%26rft_id%253Dinfo%253Apmid%252F25175100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.07.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25175100&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lucchini</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biraghi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbone</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Scrilli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magni</span>  <span class="cit-name-given-names">G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Effect of mutation in the aromatic amino acid pathway on sporulation of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">136</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">55</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DLucchini%26rft.auinit1%253DG%26rft.volume%253D136%26rft.issue%253D1%26rft.spage%253D55%26rft.epage%253D62%26rft.atitle%253DEffect%2Bof%2Bmutation%2Bin%2Bthe%2Baromatic%2Bamino%2Bacid%2Bpathway%2Bon%2Bsporulation%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F361708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6ODoiMTM2LzEvNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.57" data-doi="10.1016/S0378-1119(02)00558-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magasanik</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaiser</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Nitrogen regulation in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">290</span>(<span class="cit-issue">1–2</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DMagasanik%26rft.auinit1%253DB.%26rft.volume%253D290%26rft.issue%253D1-2%26rft.spage%253D1%26rft.epage%253D18%26rft.atitle%253DNitrogen%2Bregulation%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252802%252900558-9%26rft_id%253Dinfo%253Apmid%252F12062797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(02)00558-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12062797&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176479500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marini</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vissers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urrestarazu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andre</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Cloning and expression of the MEP1 gene encoding an ammonium transporter in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">15</span>): <span class="cit-fpage">3456</span>–<span class="cit-lpage">3463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D13%26rft.spage%253D3456%26rft_id%253Dinfo%253Apmid%252F8062822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8062822&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PB90900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.59" data-doi="10.1128/MCB.17.8.4282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marini</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soussi-Boudekou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vissers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andre</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A family of ammonium transporters in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">4282</span>–<span class="cit-lpage">4293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMarini%26rft.auinit1%253DA.%2BM.%26rft.volume%253D17%26rft.issue%253D8%26rft.spage%253D4282%26rft.epage%253D4293%26rft.atitle%253DA%2Bfamily%2Bof%2Bammonium%2Btransporters%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.17.8.4282%26rft_id%253Dinfo%253Apmid%252F9234685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE3LzgvNDI4MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.60" data-doi="10.1093/femsyr/fov067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marsit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dequin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Diversity and adaptive evolution of Saccharomyces wine yeast: a review.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">fov067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsyr%252Ffov067%26rft_id%253Dinfo%253Apmid%252F26205244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjExOiIxNS83L2ZvdjA2NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martínez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilera</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene expression of specific enological traits in wine fermentation.</span> <abbr class="cit-jnl-abbrev">Electron. J. Biotechnol.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectron.%2BJ.%2BBiotechnol.%26rft.volume%253D16%26rft.spage%253D13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.62" data-doi="10.1111/j.1567-1364.2007.00252.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marullo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aigle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bely</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masneuf-Pomarede</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrens</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Single QTL mapping and nucleotide-level resolution of a physiologic trait in wine Saccharomyces cerevisiae strains.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">941</span>–<span class="cit-lpage">952</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2007.00252.x%26rft_id%253Dinfo%253Apmid%252F17537182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjcvNi85NDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.63" data-doi="10.1099/00221287-143-1-203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nissen</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulze</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villadsen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Flux distributions in anaerobic, glucose-limited continuous cultures of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">143</span>(<span class="cit-issue">Pt. 1</span>): <span class="cit-fpage">203</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DNissen%26rft.auinit1%253DT.%2BL.%26rft.volume%253D143%26rft.issue%253D1%26rft.spage%253D203%26rft.epage%253D218%26rft.atitle%253DFlux%2BDistributions%2Bin%2BAnaerobic%252C%2BGlucose-Limited%2BContinuous%2BCultures%2Bof%2BSaccharomyces%2BCerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00221287-143-1-203%26rft_id%253Dinfo%253Apmid%252F9025295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00221287-143-1-203&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9025295&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WE41100025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of new Saccharomyces cerevisiae variants of the MET2 and SKP2 genes controlling the sulfur assimilation pathway and the production of undesirable sulfur compounds during alcoholic fermentation.</span> <abbr class="cit-jnl-abbrev">Microb. Cell Fact.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrob.%2BCell%2BFact.%26rft.volume%253D14%26rft.spage%253D68%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.65" data-doi="10.1371/journal.pgen.0030031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nogami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohya</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yvert</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic complexity and quantitative trait loci mapping of yeast morphological traits.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DNogami%26rft.auinit1%253DS.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253De31%26rft.epage%253De31%26rft.atitle%253DGenetic%2Bcomplexity%2Band%2Bquantitative%2Btrait%2Bloci%2Bmapping%2Bof%2Byeast%2Bmorphological%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030031%26rft_id%253Dinfo%253Apmid%252F17319748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17319748&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.66" data-doi="10.1091/mbc.E14-06-1140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Omnus</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ljungdahl</span>  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Latency of transcription factor Stp1 depends on a modular regulatory motif that functions as cytoplasmic retention determinant and nuclear degron.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">23</span>): <span class="cit-fpage">3823</span>–<span class="cit-lpage">3833</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E14-06-1140%26rft_id%253Dinfo%253Apmid%252F25253722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8yMy8zODIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ono</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Widespread genetic incompatibilities between first-step mutations during parallel adaptation of Saccharomyces cerevisiae to a common environment.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e1002591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D15%26rft.spage%253De1002591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.68" data-doi="10.1016/S1097-2765(03)00485-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedruzzi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubouloz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameroni</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wanke</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roosen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">TOR and PKA signaling pathways converge on the protein kinase Rim15 to control entry into G0.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1607</span>–<span class="cit-lpage">1613</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DPedruzzi%26rft.auinit1%253DI.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D1607%26rft.epage%253D1613%26rft.atitle%253DTOR%2Band%2BPKA%2Bsignaling%2Bpathways%2Bconverge%2Bon%2Bthe%2Bprotein%2Bkinase%2BRim15%2Bto%2Bcontrol%2Bentry%2Binto%2BG0.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252803%252900485-4%26rft_id%253Dinfo%253Apmid%252F14690612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(03)00485-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14690612&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187511600027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.69" data-doi="10.1016/S0168-1605(03)00244-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Querol</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Espinar</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">del Olmo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrio</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Adaptive evolution of wine yeast.</span> <abbr class="cit-jnl-abbrev">Int. J. Food Microbiol.</abbr> <span class="cit-vol">86</span>(<span class="cit-issue">1–2</span>): <span class="cit-fpage">3</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bfood%2Bmicrobiology%26rft.stitle%253DInt%2BJ%2BFood%2BMicrobiol%26rft.aulast%253DQuerol%26rft.auinit1%253DA.%26rft.volume%253D86%26rft.issue%253D1-2%26rft.spage%253D3%26rft.epage%253D10%26rft.atitle%253DAdaptive%2Bevolution%2Bof%2Bwine%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-1605%252803%252900244-7%26rft_id%253Dinfo%253Apmid%252F12892918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-1605(03)00244-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12892918&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184727700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.70" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.71" data-doi="10.1002/yea.1046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rossignol</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dulau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Julien</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genome-wide monitoring of wine yeast gene expression during alcoholic fermentation.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">1369</span>–<span class="cit-lpage">1385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DRossignol%26rft.auinit1%253DT.%26rft.volume%253D20%26rft.issue%253D16%26rft.spage%253D1369%26rft.epage%253D1385%26rft.atitle%253DGenome-wide%2Bmonitoring%2Bof%2Bwine%2Byeast%2Bgene%2Bexpression%2Bduring%2Balcoholic%2Bfermentation.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1046%26rft_id%253Dinfo%253Apmid%252F14663829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14663829&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187610000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.72" data-doi="10.1371/journal.pone.0049640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soto</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergstrom</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genetic basis of natural variation in oenological traits in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e49640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSalinas%26rft.auinit1%253DF.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De49640%26rft.epage%253De49640%26rft.atitle%253DThe%2BGenetic%2BBasis%2Bof%2BNatural%2BVariation%2Bin%2BOenological%2BTraits%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0049640%26rft_id%253Dinfo%253Apmid%252F23185390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0049640&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23185390&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.73" data-doi="10.1038/srep21849"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Boer</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abarca</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Natural variation in non-coding regions underlying phenotypic diversity in budding yeast.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">21849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D21849%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep21849%26rft_id%253Dinfo%253Apmid%252F26898953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep21849&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26898953&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Effect of sugar transport inactivation in Saccharomyces cerevisiae on sluggish and stuck enological fermentations.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">55</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">953</span>–<span class="cit-lpage">958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DSalmon%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D55%26rft.issue%253D4%26rft.spage%253D953%26rft.epage%253D958%26rft.atitle%253DEffect%2Bof%2BSugar%2BTransport%2BInactivation%2Bin%2BSaccharomyces%2Bcerevisiae%2Bon%2BSluggish%2Band%2BStuck%2BEnological%2BFermentations%26rft_id%253Dinfo%253Apmid%252F16347895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjg6IjU1LzQvOTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2OTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.75" data-doi="10.1016/j.crvi.2010.12.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sicard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legras</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Bread, beer and wine: yeast domestication in the Saccharomyces sensu stricto complex.</span> <abbr class="cit-jnl-abbrev">C. R. Biol.</abbr> <span class="cit-vol">334</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">229</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComptes%2Brendus%2Bbiologies%26rft.stitle%253DC%2BR%2BBiol%26rft.aulast%253DSicard%26rft.auinit1%253DD.%26rft.volume%253D334%26rft.issue%253D3%26rft.spage%253D229%26rft.epage%253D236%26rft.atitle%253DBread%252C%2Bbeer%2Band%2Bwine%253A%2Byeast%2Bdomestication%2Bin%2Bthe%2BSaccharomyces%2Bsensu%2Bstricto%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.crvi.2010.12.016%26rft_id%253Dinfo%253Apmid%252F21377618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.crvi.2010.12.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21377618&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.76" data-doi="10.1371/journal.pbio.0060083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">E. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene-environment interaction in yeast gene expression.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">e83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DSmith%26rft.auinit1%253DE.%2BN.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De83%26rft.epage%253De83%26rft.atitle%253DGene-environment%2Binteraction%2Bin%2Byeast%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060083%26rft_id%253Dinfo%253Apmid%252F18416601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18416601&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.77" data-doi="10.1038/416326a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spiegelman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oefner</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Dissecting the architecture of a quantitative trait locus in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">416</span>(<span class="cit-issue">6878</span>): <span class="cit-fpage">326</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSteinmetz%26rft.auinit1%253DL.%2BM.%26rft.volume%253D416%26rft.issue%253D6878%26rft.spage%253D326%26rft.epage%253D330%26rft.atitle%253DDissecting%2Bthe%2Barchitecture%2Bof%2Ba%2Bquantitative%2Btrait%2Blocus%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F416326a%26rft_id%253Dinfo%253Apmid%252F11907579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/416326a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11907579&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174482200044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.78" data-doi="10.1186/1471-2164-13-573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steyer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambroset</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brion</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claudel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delobel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">QTL mapping of the production of wine aroma compounds by yeast.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSteyer%26rft.auinit1%253DD.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D573%26rft.epage%253D573%26rft.atitle%253DQTL%2Bmapping%2Bof%2Bthe%2Bproduction%2Bof%2Bwine%2Baroma%2Bcompounds%2Bby%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-573%26rft_id%253Dinfo%253Apmid%252F23110365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-573&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23110365&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.79" data-doi="10.1073/pnas.1530509100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Statistical significance for genomewide studies.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">9440</span>–<span class="cit-lpage">9445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1530509100%26rft_id%253Dinfo%253Apmid%252F12883005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE2Lzk0NDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.80" data-doi="10.1186/1747-1028-1-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swinnen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wanke</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roosen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smets</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubouloz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Rim15 and the crossroads of nutrient signalling pathways in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Cell Div.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BDiv%26rft.aulast%253DSwinnen%26rft.auinit1%253DE.%26rft.volume%253D1%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DRim15%2Band%2Bthe%2Bcrossroads%2Bof%2Bnutrient%2Bsignalling%2Bpathways%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1747-1028-1-3%26rft_id%253Dinfo%253Apmid%252F16759348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1747-1028-1-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16759348&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.81" data-doi="10.1002/yea.320100509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sychrova</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chevallier</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">APL1, a yeast gene encoding a putative permease for basic amino acids.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">653</span>–<span class="cit-lpage">657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DSychrova%26rft.auinit1%253DH.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D653%26rft.epage%253D657%26rft.atitle%253DAPL1%252C%2Ba%2Byeast%2Bgene%2Bencoding%2Ba%2Bputative%2Bpermease%2Bfor%2Bbasic%2Bamino%2Bacids.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320100509%26rft_id%253Dinfo%253Apmid%252F7941748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320100509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7941748&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NM58900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.82" data-doi="10.1111/j.1574-6976.2000.tb00533.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">ter Schure</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Riel</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verrips</span>  <span class="cit-name-given-names">C. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The role of ammonia metabolism in nitrogen catabolite repression in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">67</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6976.2000.tb00533.x%26rft_id%253Dinfo%253Apmid%252F10640599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3JlIjtzOjU6InJlc2lkIjtzOjc6IjI0LzEvNjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.83"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tesniere</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brice</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Responses of Saccharomyces cerevisiae to nitrogen starvation in wine alcoholic fermentation.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">99</span>(<span class="cit-issue">17</span>): <span class="cit-fpage">7025</span>–<span class="cit-lpage">7034</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BMicrobiol.%2BBiotechnol.%26rft.volume%253D99%26rft.spage%253D7025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.84" data-doi="10.1023/A:1012027718701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torija</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozes</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poblet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillamon</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Yeast population dynamics in spontaneous fermentations: comparison between two different wine-producing areas over a period of three years.</span> <abbr class="cit-jnl-abbrev">Antonie van Leeuwenhoek</abbr> <span class="cit-vol">79</span>(<span class="cit-issue">3–4</span>): <span class="cit-fpage">345</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntonie%2Bvan%2BLeeuwenhoek%26rft.stitle%253DAntonie%2Bvan%2BLeeuwenhoek%26rft.aulast%253DTorija%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D79%26rft.issue%253D3-4%26rft.spage%253D345%26rft.epage%253D352%26rft.atitle%253DYeast%2Bpopulation%2Bdynamics%2Bin%2Bspontaneous%2Bfermentations%253A%2Bcomparison%2Bbetween%2Btwo%2Bdifferent%2Bwine-producing%2Bareas%2Bover%2Ba%2Bperiod%2Bof%2Bthree%2Byears.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1012027718701%26rft_id%253Dinfo%253Apmid%252F11816978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1012027718701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11816978&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171533400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.85" data-doi="10.1093/bioinformatics/btp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TopHat: discovering splice junctions with RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1105</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp120%26rft_id%253Dinfo%253Apmid%252F19289445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNS85LzExMDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY5My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.86" data-doi="10.1371/journal.pgen.1004894"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic basis of haloperidol resistance in Saccharomyces cerevisiae is complex and dose dependent.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">e1004894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004894%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004894%26rft_id%253Dinfo%253Apmid%252F25521586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004894&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25521586&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.87" data-doi="10.1371/journal.pgen.1002111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorgo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjuvsland</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Trait variation in yeast is defined by population history.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e1002111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWarringer%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002111%26rft.epage%253De1002111%26rft.atitle%253DTrait%2Bvariation%2Bin%2Byeast%2Bis%2Bdefined%2Bby%2Bpopulation%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002111%26rft_id%253Dinfo%253Apmid%252F21698134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21698134&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.88" data-doi="10.1038/283214a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasmyth</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Isolation of the structural gene for alcohol dehydrogenase by genetic complementation in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">283</span>(<span class="cit-issue">5743</span>): <span class="cit-fpage">214</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWilliamson%26rft.auinit1%253DV.%2BM.%26rft.volume%253D283%26rft.issue%253D5743%26rft.spage%253D214%26rft.epage%253D216%26rft.atitle%253DIsolation%2Bof%2Bthe%2Bstructural%2Bgene%2Bfor%2Balcohol%2Bdehydrogenase%2Bby%2Bgenetic%2Bcomplementation%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F283214a0%26rft_id%253Dinfo%253Apmid%252F6985717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/283214a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6985717&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1693.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JB44400049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fresnedo-Ramirez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manns</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sacks</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Next generation mapping of enological traits in an F2 interspecific grapevine hybrid family.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">e0149560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0149560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1693.90" data-doi="10.1074/jbc.M609201200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zargari</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boban</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heessen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andreasson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thyberg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Inner nuclear membrane proteins Asi1, Asi2, and Asi3 function in concert to maintain the latent properties of transcription factors Stp1 and Stp2.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">282</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">594</span>–<span class="cit-lpage">605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M609201200%26rft_id%253Dinfo%253Apmid%252F17085444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjk6IjI4Mi8xLzU5NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjkzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/6/1693.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1683" title="Identification of Ganoderma Disease Resistance Loci Using Natural Field Infection of an Oil Palm Multiparental Population" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1707" title="Pedigree-Based Analysis in a Multiparental Population of Octoploid Strawberry Reveals QTL Alleles Conferring Resistance to Phytophthora cactorum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/6"><span>Volume 7 Issue 6, June 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/6.cover-source.jpg?itok=5_J-OKHc" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1693" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-d76b6MIX7SuZM_7H3qr5t7WD0_swQ-ML3OdjY0SscwA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-population" class="" data-icon-position="" data-hide-link-title="0">Multiparental Population</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/6/1693" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/6/1693" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/6/1693" class="active" data-icon-position="" data-hide-link-title="0">Identification of Nitrogen Consumption Genetic Variants in Yeast Through QTL Mapping and Bulk Segregant RNA-Seq Analyses</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484773" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-KmZjC135YBJvPLIdw3293e7b81LAeZr936AZrNz-8wo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1693.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/6/1693" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-gTvZ7QNI5ixAuqZbv3std2EU4zaH4gLv-IHOZNDXaJs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/6/1693" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1693.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484773" id="node484773--2" data-pisa="ggg;7/6/1693" data-pisa-master="ggg;g3.117.042127" data-apath="/ggg/7/6/1693.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Nitrogen Consumption Genetic Variants in Yeast Through QTL Mapping and Bulk Segregant RNA-Seq Analyses</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Francisco A. Cubillos</span>, <span class="highwire-citation-author" data-delta="1">Claire Brice</span>, <span class="highwire-citation-author" data-delta="2">Jennifer Molinet</span>, <span class="highwire-citation-author" data-delta="3">Sebastién Tisné</span>, <span class="highwire-citation-author" data-delta="4">Valentina Abarca</span>, <span class="highwire-citation-author" data-delta="5">Sebastián M. Tapia</span>, <span class="highwire-citation-author" data-delta="6">Christian Oporto</span>, <span class="highwire-citation-author" data-delta="7">Verónica García</span>, <span class="highwire-citation-author" data-delta="8">Gianni Liti</span> and <span class="highwire-citation-author" data-delta="9">Claudio Martínez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1693-1705;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042127 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484773--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Francisco A. Cubillos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Millennium Nucleus for Fungal Integrative and Synthetic Biology (MN-FISB), 9170201 Santiago, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BA.%2BCubillos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cubillos%20FA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BA.%2BCubillos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Claire Brice </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BBrice%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brice%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaire%2BBrice%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer Molinet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BMolinet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Molinet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BMolinet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sebastién Tisné </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre de coopération internationale en recherche agronomique pour le développement (CIRAD), UMR AGAP (Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit), Campus International de Baillarguet, 34398 Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebasti%C3%A9n%2BTisn%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tisn%C3%A9%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebasti%25C3%25A9n%2BTisn%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Valentina Abarca </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Valentina%2BAbarca%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abarca%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AValentina%2BAbarca%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sebastián M. Tapia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Millennium Nucleus for Fungal Integrative and Synthetic Biology (MN-FISB), 9170201 Santiago, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebasti%C3%A1n%2BM.%2BTapia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tapia%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebasti%25C3%25A1n%2BM.%2BTapia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Christian Oporto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Millennium Nucleus for Fungal Integrative and Synthetic Biology (MN-FISB), 9170201 Santiago, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BOporto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oporto%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BOporto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Verónica García </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ver%C3%B3nica%2BGarc%C3%ADa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garc%C3%ADa%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVer%25C3%25B3nica%2BGarc%25C3%25ADa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Gianni Liti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute for Research on Cancer and Ageing of Nice (IRCAN), Centre national de la recherche scientifique (CNRS) unités mixtes de recherche (UMR) 7284, Institut national de la santé et de la recherche médicale (INSERM) U1081, University of Nice Sophia Antipolis, 06107, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gianni%2BLiti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liti%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGianni%2BLiti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Claudio Martínez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudio%2BMart%C3%ADnez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADnez%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClaudio%2BMart%25C3%25ADnez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:claudio.martinez@usach.cl" class="" data-icon-position="" data-hide-link-title="0">claudio.martinez@usach.cl</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/6/1693" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/6/1693&amp;title=Identification%20of%20Nitrogen%20Consumption%20Genetic%20Variants%20in%20Yeast%20Through%20QTL%20Mapping%20and%20Bulk%20Segregant%20RNA-Seq%20Analyses" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/6/1693&amp;title=Identification%20of%20Nitrogen%20Consumption%20Genetic%20Variants%20in%20Yeast%20Through%20QTL%20Mapping%20and%20Bulk%20Segregant%20RNA-Seq%20Analyses" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/6/1693&amp;title=Identification%20of%20Nitrogen%20Consumption%20Genetic%20Variants%20in%20Yeast%20Through%20QTL%20Mapping%20and%20Bulk%20Segregant%20RNA-Seq%20Analyses" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/6/1693&amp;text=Identification%20of%20Nitrogen%20Consumption%20Genetic%20Variants%20in%20Yeast%20Through%20QTL%20Mapping%20and%20Bulk%20Segregant%20RNA-Seq%20Analyses" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/6/1693&amp;title=Identification%20of%20Nitrogen%20Consumption%20Genetic%20Variants%20in%20Yeast%20Through%20QTL%20Mapping%20and%20Bulk%20Segregant%20RNA-Seq%20Analyses" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/6/1693&amp;t=Identification%20of%20Nitrogen%20Consumption%20Genetic%20Variants%20in%20Yeast%20Through%20QTL%20Mapping%20and%20Bulk%20Segregant%20RNA-Seq%20Analyses" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/6/1693&amp;title=Identification%20of%20Nitrogen%20Consumption%20Genetic%20Variants%20in%20Yeast%20Through%20QTL%20Mapping%20and%20Bulk%20Segregant%20RNA-Seq%20Analyses" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/6/1693&amp;title=Identification%20of%20Nitrogen%20Consumption%20Genetic%20Variants%20in%20Yeast%20Through%20QTL%20Mapping%20and%20Bulk%20Segregant%20RNA-Seq%20Analyses" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484773/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484773" id="node484773--3" data-pisa="ggg;7/6/1693" data-pisa-master="ggg;g3.117.042127" data-apath="/ggg/7/6/1693.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Nitrogen Consumption Genetic Variants in Yeast Through QTL Mapping and Bulk Segregant RNA-Seq Analyses</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Francisco A. Cubillos</span>, <span class="highwire-citation-author" data-delta="1">Claire Brice</span>, <span class="highwire-citation-author" data-delta="2">Jennifer Molinet</span>, <span class="highwire-citation-author" data-delta="3">Sebastién Tisné</span>, <span class="highwire-citation-author" data-delta="4">Valentina Abarca</span>, <span class="highwire-citation-author" data-delta="5">Sebastián M. Tapia</span>, <span class="highwire-citation-author" data-delta="6">Christian Oporto</span>, <span class="highwire-citation-author" data-delta="7">Verónica García</span>, <span class="highwire-citation-author" data-delta="8">Gianni Liti</span> and <span class="highwire-citation-author" data-delta="9">Claudio Martínez</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1693-1705;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042127 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484773--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Francisco A. Cubillos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Millennium Nucleus for Fungal Integrative and Synthetic Biology (MN-FISB), 9170201 Santiago, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BA.%2BCubillos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cubillos%20FA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BA.%2BCubillos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Claire Brice </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BBrice%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brice%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaire%2BBrice%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer Molinet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BMolinet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Molinet%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BMolinet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sebastién Tisné </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre de coopération internationale en recherche agronomique pour le développement (CIRAD), UMR AGAP (Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit), Campus International de Baillarguet, 34398 Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebasti%C3%A9n%2BTisn%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tisn%C3%A9%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebasti%25C3%25A9n%2BTisn%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Valentina Abarca </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Valentina%2BAbarca%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abarca%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AValentina%2BAbarca%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sebastián M. Tapia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Millennium Nucleus for Fungal Integrative and Synthetic Biology (MN-FISB), 9170201 Santiago, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebasti%C3%A1n%2BM.%2BTapia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tapia%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebasti%25C3%25A1n%2BM.%2BTapia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Christian Oporto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Millennium Nucleus for Fungal Integrative and Synthetic Biology (MN-FISB), 9170201 Santiago, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christian%2BOporto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oporto%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristian%2BOporto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Verónica García </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ver%C3%B3nica%2BGarc%C3%ADa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garc%C3%ADa%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVer%25C3%25B3nica%2BGarc%25C3%25ADa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Gianni Liti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute for Research on Cancer and Ageing of Nice (IRCAN), Centre national de la recherche scientifique (CNRS) unités mixtes de recherche (UMR) 7284, Institut national de la santé et de la recherche médicale (INSERM) U1081, University of Nice Sophia Antipolis, 06107, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gianni%2BLiti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liti%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGianni%2BLiti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Claudio Martínez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centro de Estudios en Ciencia y Tecnología de Alimentos (CECTA), Universidad de Santiago de Chile (USACH), Chile</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Ciencia y Tecnología de los Alimentos, Universidad de Santiago de Chile (USACH), 9170201 Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudio%2BMart%C3%ADnez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mart%C3%ADnez%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClaudio%2BMart%25C3%25ADnez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:claudio.martinez@usach.cl" class="" data-icon-position="" data-hide-link-title="0">claudio.martinez@usach.cl</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1693&amp;rft.epage=1705&amp;rft.atitle=Identification%20of%20Nitrogen%20Consumption%20Genetic%20Variants%20in%20Yeast%20Through%20QTL%20Mapping%20and%20Bulk%20Segregant%20RNA-Seq%20Analyses&amp;rft.volume=7&amp;rft.issue=6&amp;rft.date=2017-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cubillos%2C+Francisco+A.&amp;rft.au=Brice%2C+Claire&amp;rft.au=Molinet%2C+Jennifer&amp;rft.au=Tisn%C3%A9%2C+Sebasti%C3%A9n&amp;rft.au=Abarca%2C+Valentina&amp;rft.au=Tapia%2C+Sebasti%C3%A1n+M.&amp;rft.au=Oporto%2C+Christian&amp;rft.au=Garc%C3%ADa%2C+Ver%C3%B3nica&amp;rft.au=Liti%2C+Gianni&amp;rft.au=Mart%C3%ADnez%2C+Claudio"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484773/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484773/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484773/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484773/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484773/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484773/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484773/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484773/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484773/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484773/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484773/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484773/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484782" id="node484782" data-pisa="ggg;7/6/1721" data-pisa-master="ggg;g3.117.042101" data-apath="/ggg/7/6/1721.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/7/6/1721" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Approaches in Characterizing Genetic Structure and Mapping in a Rice Multiparental Population</span></a></div>
</cite>
</div></li><li class="last even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484763" id="node484763" data-pisa="ggg;7/6/1707" data-pisa-master="ggg;g3.117.042119" data-apath="/ggg/7/6/1707.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/7/6/1707" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Pedigree-Based Analysis in a Multiparental Population of Octoploid Strawberry Reveals QTL Alleles Conferring Resistance to <em>Phytophthora cactorum</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Population" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Population</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/6/1693" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/6/1693.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/6/1693.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/6/1693.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/6/1693.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
