<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/6/1899" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/6/1899.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/6/1899.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/6/1899.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/6/1899.atom" />
<meta name="HW.pisa" content="ggg;7/6/1899" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Adaptive Roles of SSY1 and SIR3 During Cycles of Growth and Starvation in Saccharomyces cerevisiae Populations Enriched for Quiescent or Nonquiescent Cells" />
<meta name="DC.Identifier" content="10.1534/g3.117.041749" />
<meta name="DC.Date" content="2017-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Wloch-Salamon et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Over its evolutionary history, Saccharomyces cerevisiae has evolved to be well-adapted to fluctuating nutrient availability. In the presence of sufficient nutrients, yeast cells continue to proliferate, but upon starvation haploid yeast cells enter stationary phase and differentiate into nonquiescent (NQ) and quiescent (Q) cells. Q cells survive stress better than NQ cells and show greater viability when nutrient-rich conditions are restored. To investigate the genes that may be involved in the differentiation of Q and NQ cells, we serially propagated yeast populations that were enriched for either only Q or only NQ cell types over many repeated growth–starvation cycles. After 30 cycles (equivalent to 300 generations), each enriched population produced a higher proportion of the enriched cell type compared to the starting population, suggestive of adaptive change. We also observed differences in each population’s fitness suggesting possible tradeoffs: clones from NQ lines were better adapted to logarithmic growth, while clones from Q lines were better adapted to starvation. Whole-genome sequencing of clones from Q- and NQ-enriched lines revealed mutations in genes involved in the stress response and survival in limiting nutrients ( [ECM21][1] , [RSP5][2] , [MSN1][3] , [SIR4][4] , and [IRA2][5] ) in both Q and NQ lines, but also differences between the two lines: NQ line clones had recurrent independent mutations affecting the [Ssy1p][6]-[Ptr3p][7]-[Ssy5p][8] (SPS) amino acid sensing pathway, while Q line clones had recurrent, independent mutations in [SIR3][9] and [FAS1][10] . Our results suggest that both sets of enriched-cell type lines responded to common, as well as distinct, selective pressures.

 [1]: http://www.yeastgenome.org/locus/S000000197/overview
 [2]: http://www.yeastgenome.org/locus/S000000927/overview
 [3]: http://www.yeastgenome.org/locus/S000005476/overview
 [4]: http://www.yeastgenome.org/locus/S000002635/overview
 [5]: http://www.yeastgenome.org/locus/S000005441/overview
 [6]: http://www.yeastgenome.org/locus/S000002567/overview
 [7]: http://www.yeastgenome.org/locus/S000001925/overview
 [8]: http://www.yeastgenome.org/locus/S000003692/overview
 [9]: http://www.yeastgenome.org/locus/S000004434/overview
 [10]: http://www.yeastgenome.org/locus/S000001665/overview" />
<meta name="DC.Contributor" content="Dominika M. Wloch-Salamon" />
<meta name="DC.Contributor" content="Katarzyna Tomala" />
<meta name="DC.Contributor" content="Dimitra Aggeli" />
<meta name="DC.Contributor" content="Barbara Dunn" />
<meta name="article:published_time" content="2017-06-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Adaptive Roles of SSY1 and SIR3 During Cycles of Growth and Starvation in Saccharomyces cerevisiae Populations Enriched for Quiescent or Nonquiescent Cells" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Over its evolutionary history, &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; has evolved to be well-adapted to fluctuating nutrient availability. In the presence of sufficient nutrients, yeast cells continue to proliferate, but upon starvation haploid yeast cells enter stationary phase and differentiate into nonquiescent (NQ) and quiescent (Q) cells. Q cells survive stress better than NQ cells and show greater viability when nutrient-rich conditions are restored. To investigate the genes that may be involved in the differentiation of Q and NQ cells, we serially propagated yeast populations that were enriched for either only Q or only NQ cell types over many repeated growth–starvation cycles. After 30 cycles (equivalent to 300 generations), each enriched population produced a higher proportion of the enriched cell type compared to the starting population, suggestive of adaptive change. We also observed differences in each population’s fitness suggesting possible tradeoffs: clones from NQ lines were better adapted to logarithmic growth, while clones from Q lines were better adapted to starvation. Whole-genome sequencing of clones from Q- and NQ-enriched lines revealed mutations in genes involved in the stress response and survival in limiting nutrients (&lt;i&gt;ECM21&lt;/i&gt;, &lt;i&gt;RSP5&lt;/i&gt;, &lt;i&gt;MSN1&lt;/i&gt;, &lt;i&gt;SIR4&lt;/i&gt;, and &lt;i&gt;IRA2&lt;/i&gt;) in both Q and NQ lines, but also differences between the two lines: NQ line clones had recurrent independent mutations affecting the Ssy1p-Ptr3p-Ssy5p (SPS) amino acid sensing pathway, while Q line clones had recurrent, independent mutations in &lt;i&gt;SIR3&lt;/i&gt; and &lt;i&gt;FAS1&lt;/i&gt;. Our results suggest that both sets of enriched-cell type lines responded to common, as well as distinct, selective pressures.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/06/01" />
<meta name="citation_mjid" content="ggg;7/6/1899" />
<meta name="citation_id" content="7/6/1899" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/6/1899" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/6/1899.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/6/1899.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/6/1899.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.041749" />
<meta name="citation_pmid" content="28450371" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1899" />
<meta name="citation_lastpage" content="1911" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Dominika M. Wloch-Salamon" />
<meta name="citation_author_institution" content="Institute of Environmental Sciences, Jagiellonian University, Krakow 30-387, Poland" />
<meta name="citation_author_email" content="dominika.wloch-salamon@uj.edu.pl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0040-1838" />
<meta name="citation_author" content="Katarzyna Tomala" />
<meta name="citation_author_institution" content="Institute of Environmental Sciences, Jagiellonian University, Krakow 30-387, Poland" />
<meta name="citation_author" content="Dimitra Aggeli" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University, California 94305-5120" />
<meta name="citation_author" content="Barbara Dunn" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University, California 94305-5120" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7041-0035" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. Aguilaniu;citation_author=L. Gustafsson;citation_author=M. Rigoulet;citation_author=T. Nyström;citation_title=Asymmetric inheritance of oxidatively damaged proteins during cytokinesis.;citation_volume=299;citation_year=2003;citation_pmid=12610228;citation_doi=10.1126/science.1080418" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Cell Biology;citation_journal_abbrev=JCB;citation_author=C. Allen;citation_author=S. Buttner;citation_author=A. D. Aragon;citation_author=J. A. Thomas;citation_author=O. Meirelles;citation_title=Isolation of quiescent and nonquiescent cells from yeast stationary-phase cultures.;citation_pages=89-100;citation_volume=174;citation_year=2006;citation_issue=1;citation_pmid=16818721;citation_doi=10.1083/jcb.200604072" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=C. P. Allen;citation_author=S. S. Anna-A;citation_author=J. E. Jaetao;citation_author=M. C. Werner-Washburne;citation_title=Separation and characterization of quiescent and senescent cells from S-cerevisiae stationary-phase cultures.;citation_pages=367a-368a;citation_volume=15;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Aging cell;citation_journal_abbrev=Aging Cell;citation_author=A. L. Alvers;citation_author=L. K. Fishwick;citation_author=M. S. Wood;citation_author=D. Hu;citation_author=H. S. Chung;citation_title=Autophagy and amino acid homeostasis are required for chronological longevity in Saccharomyces cerevisiae.;citation_pages=353-369;citation_volume=8;citation_year=2009;citation_issue=4;citation_pmid=19302372;citation_doi=10.1111/j.1474-9726.2009.00469.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=A. D. Aragon;citation_author=A. L. Rodriguez;citation_author=O. Meirelles;citation_author=S. Roy;citation_author=G. S. Davidson;citation_title=Characterization of differentiated quiescent and nonquiescent cells in yeast stationary-phase cultures.;citation_volume=19;citation_year=2008;citation_pmid=18199684;citation_doi=10.1091/mbc.E07-07-0666" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=L. M. Bono;citation_author=L. B. Smith;citation_author=D. W. Pfennig;citation_author=C. L. Burch;citation_title=The emergence of performance trade‐offs during local adaptation: insights from experimental evolution.;citation_pages=1720-1733;citation_volume=26;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Broomfield;citation_author=B. L. Chow;citation_author=W. Xiao;citation_title=MMS2, encoding a ubiquitin-conjugating-enzyme-like protein, is a member of the yeast error-free postreplication repair pathway;citation_pages=5678-5683;citation_volume=95;citation_year=1998;citation_issue=10;citation_pmid=9576943;citation_doi=10.1073/pnas.95.10.5678" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=S. Brunke;citation_author=B. Hube;citation_title=Adaptive prediction as a strategy in microbial infections.;citation_pages=e1004356;citation_volume=10;citation_year=2014;citation_pmid=25275642;citation_doi=10.1371/journal.ppat.1004356" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. Čáp;citation_author=L. Váchová;citation_author=Z. Palková;citation_title=Yeast colony survival depends on metabolic adaptation and cell differentiation rather than on stress defense.;citation_volume=284;citation_year=2009;citation_pmid=19801643;citation_doi=10.1074/jbc.M109.022871" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. Čáp;citation_author=L. Štěpánek;citation_author=K. Harant;citation_author=L. Váchová;citation_author=Z. Palková;citation_title=Cell differentiation within a yeast colony: metabolic and regulatory parallels with a tumor-affected organism.;citation_pages=436-448;citation_volume=46;citation_year=2012;citation_issue=4;citation_pmid=22560924;citation_doi=10.1016/j.molcel.2012.04.001" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_author=K. E. Chiotti;citation_author=D. J. Kvitek;citation_author=K. H. Schmidt;citation_author=G. Koniges;citation_author=K. Schwartz;citation_title=The valley-of-death: reciprocal sign epistasis constrains adaptive trajectories in a constant, nutrient limiting environment.;citation_pages=431-437;citation_volume=104;citation_year=2014;citation_pmid=25449178;citation_doi=10.1016/j.ygeno.2014.10.011" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. J. Connelly;citation_author=P. Yuan;citation_author=H.-C. Hsu;citation_author=Z. Li;citation_author=R.-M. Xu;citation_title=Structure and Function of the Saccharomyces cerevisiae Sir3 BAH Domain;citation_pages=3256-3265;citation_volume=26;citation_year=2006;citation_issue=8;citation_pmid=16581798;citation_doi=10.1128/MCB.26.8.3256-3265.2006" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=M. Conrad;citation_author=J. Schothorst;citation_author=H. N. Kankipati;citation_author=G. Van Zeebroeck;citation_author=M. Rubio-Texeira;citation_title=Nutrient sensing and signaling in the yeast Saccharomyces cerevisiae.;citation_volume=38;citation_year=2014;citation_pmid=24483210;citation_doi=10.1111/1574-6976.12065" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=F. A. Cubillos;citation_author=E. J. Louis;citation_author=G. Liti;citation_title=Generation of a large set of genetically tractable haploid and diploid Saccharomyces strains.;citation_volume=9;citation_year=2009;citation_pmid=19840116;citation_doi=10.1111/j.1567-1364.2009.00583.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. J. Cullen;citation_author=G. F. Sprague;citation_title=Glucose depletion causes haploid invasive growth in yeast.;citation_volume=97;citation_year=2000;citation_pmid=11095711;citation_doi=10.1073/pnas.240345197" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=G. S. Davidson;citation_author=R. M. Joe;citation_author=S. Roy;citation_author=O. Meirelles;citation_author=C. P. Allen;citation_title=The proteomics of quiescent and nonquiescent cell differentiation in yeast stationary-phase cultures.;citation_volume=22;citation_year=2011;citation_pmid=21289090;citation_doi=10.1091/mbc.E10-06-0499" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. DeLuna;citation_author=K. Vetsigian;citation_author=N. Shoresh;citation_author=M. Hegreness;citation_author=M. Colón-González;citation_title=Exposing the fitness contribution of duplicated genes.;citation_pages=676-681;citation_volume=40;citation_year=2008;citation_issue=5;citation_pmid=18408719;citation_doi=10.1038/ng.123" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=C. De Virgilio;citation_title=The essence of yeast quiescence.;citation_volume=36;citation_year=2012;citation_pmid=21658086;citation_doi=10.1111/j.1574-6976.2011.00287.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. Dhar;citation_author=R. Sägesser;citation_author=C. Weikert;citation_author=A. Wagner;citation_title=Yeast adapts to a changing stressful environment by evolving cross-protection and anticipatory gene regulation.;citation_volume=30;citation_year=2013;citation_pmid=23125229;citation_doi=10.1093/molbev/mss253" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=T. Didion;citation_author=B. Regenberg;citation_author=M. U. Jorgensen;citation_author=M. C. Kielland-Brandt;citation_author=H. A. Andersen;citation_title=The permease homologue Ssy1p controls the expression of amino acid and peptide transporter genes in Saccharomyces cerevisiae.;citation_pages=643-650;citation_volume=27;citation_year=1998;citation_issue=3;citation_pmid=9489675;citation_doi=10.1046/j.1365-2958.1998.00714.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_title=PHYLIP: Phylogenetic Inference Program, Version 3.6;citation_author=J. Felsenstein;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Forsberg;citation_author=M. Hammar;citation_author=C. Andréasson;citation_author=A. Molinér;citation_author=P. O. Ljungdahl;citation_title=Suppressors of ssy1 and ptr3 Null Mutations Define Novel Amino Acid Sensor-Independent Genes in Saccharomyces cerevisiae;citation_pages=973-988;citation_volume=158;citation_year=2001;citation_issue=3;citation_pmid=11454748" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=R. F. Gaber;citation_author=K. Ottow;citation_author=H. A. Andersen;citation_author=M. C. Kielland-Brandt;citation_title=Constitutive and Hyperresponsive Signaling by Mutant Forms of Saccharomyces cerevisiae Amino Acid Sensor Ssy1;citation_pages=922-929;citation_volume=2;citation_year=2003;citation_issue=5;citation_pmid=14555474;citation_doi=10.1128/EC.2.5.922-929.2003" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=M. Gagiano;citation_author=D. Van Dyk;citation_author=F. F. Bauer;citation_author=M. G. Lambrechts;citation_author=I. S. Pretorius;citation_title=Msn1p/Mss10p, Mss11p and Muc1p/Flo11p are part of a signal transduction pathway downstream of Mep2p regulating invasive growth and pseudohyphal differentiation in Saccharomyces cerevisiae.;citation_pages=103-116;citation_volume=31;citation_year=1999;citation_issue=1;citation_pmid=9987114;citation_doi=10.1046/j.1365-2958.1999.01151.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. R. Gartenberg;citation_author=J. S. Smith;citation_title=The Nuts and Bolts of Transcriptionally Silent Chromatin in Saccharomyces cerevisiae;citation_pages=1563-1599;citation_volume=203;citation_year=2016;citation_issue=4;citation_pmid=27516616;citation_doi=10.1534/genetics.112.145243" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=J. V. Gray;citation_author=G. A. Petsko;citation_author=G. C. Johnston;citation_author=D. Ringe;citation_author=R. A. Singer;citation_title=&quot;Sleeping Beauty&quot;: Quiescence in Saccharomyces cerevisiae;citation_pages=187-206;citation_volume=68;citation_year=2004;citation_issue=2;citation_pmid=15187181;citation_doi=10.1128/MMBR.68.2.187-206.2004" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Guidi;citation_author=M. Ruault;citation_author=M. Marbouty;citation_author=I. Loïodice;citation_author=A. Cournac;citation_title=Spatial reorganization of telomeres in long-lived quiescent cells.;citation_pages=1;citation_volume=16;citation_year=2015;citation_pmid=25583448;citation_doi=10.1186/s13059-014-0572-2" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Hersen;citation_author=M. N. McClean;citation_author=L. Mahadevan;citation_author=S. Ramanathan;citation_title=Signal processing by the HOG MAP kinase pathway.;citation_volume=105;citation_year=2008;citation_pmid=18480263;citation_doi=10.1073/pnas.0710770105" />
<meta name="citation_reference" content="citation_journal_title=Microb. Cell;citation_author=S. M. Honigberg;citation_title=Similar environments but diverse fates: responses of budding yeast to nutrient deprivation;citation_pages=302-328;citation_volume=3;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=R. Kassen;citation_title=The experimental evolution of specialists, generalists, and the maintenance of diversity.;citation_pages=173-190;citation_volume=15;citation_year=2002;citation_doi=10.1046/j.1420-9101.2002.00377.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=H. Klasson;citation_author=G. R. Fink;citation_author=P. O. Ljungdahl;citation_title=Ssy1p and Ptr3p Are Plasma Membrane Components of a Yeast System That Senses Extracellular Amino Acids;citation_pages=5405-5416;citation_volume=19;citation_year=1999;citation_issue=8;citation_pmid=10409731;citation_doi=10.1128/MCB.19.8.5405" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=M. M. Klosinska;citation_author=C. A. Crutchfield;citation_author=P. H. Bradley;citation_author=J. D. Rabinowitz;citation_author=J. R. Broach;citation_title=Yeast cells can access distinct quiescent states.;citation_volume=25;citation_year=2011;citation_pmid=21289062;citation_doi=10.1101/gad.2011311" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=U. Kruegel;citation_author=B. Robison;citation_author=T. Dange;citation_author=G. Kahlert;citation_author=J. R. Delaney;citation_title=Elevated proteasome capacity extends replicative lifespan in Saccharomyces cerevisiae.;citation_pages=e1002253-e1002253;citation_volume=7;citation_year=2011;citation_issue=9;citation_pmid=21931558;citation_doi=10.1371/journal.pgen.1002253" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Kryazhimskiy;citation_author=D. P. Rice;citation_author=E. R. Jerison;citation_author=M. M. Desai;citation_title=Global epistasis makes adaptation predictable despite sequence-level stochasticity;citation_pages=1519-1522;citation_volume=344;citation_year=2014;citation_issue=6191;citation_pmid=24970088;citation_doi=10.1126/science.1250939" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Kueng;citation_author=M. Tsai-Pflugfelder;citation_author=M. Oppikofer;citation_author=H. C. Ferreira;citation_author=E. Roberts;citation_title=Regulating repression: roles for the sir4 N-terminus in linker DNA protection and stabilization of epigenetic states.;citation_pages=e1002727-e1002727;citation_volume=8;citation_year=2012;citation_issue=5;citation_pmid=22654676;citation_doi=10.1371/journal.pgen.1002727" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genet.;citation_author=S. Kueng;citation_author=M. Oppikofer;citation_author=S. M. Gasser;citation_title=SIR proteins and the assembly of silent chromatin in budding yeast.;citation_pages=275-306;citation_volume=47;citation_year=2013;citation_pmid=24016189;citation_doi=10.1146/annurev-genet-021313-173730" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. J. Kvitek;citation_author=G. Sherlock;citation_title=Whole genome, whole population sequencing reveals that loss of signaling networks is the major adaptive strategy in a constant environment.;citation_pages=e1003972;citation_volume=9;citation_year=2013;citation_pmid=24278038;citation_doi=10.1371/journal.pgen.1003972" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=G. I. Lang;citation_author=D. P. Rice;citation_author=M. J. Hickman;citation_author=E. Sodergren;citation_author=G. M. Weinstock;citation_title=Pervasive genetic hitchhiking and clonal interference in forty evolving yeast populations.;citation_pages=571-574;citation_volume=500;citation_year=2013;citation_pmid=23873039;citation_doi=10.1038/nature12344" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=D. Laporte;citation_author=F. Courtout;citation_author=S. Tollis;citation_author=I. Sagot;citation_title=Quiescent Saccharomyces cerevisiae forms telomere hyperclusters at the nuclear membrane vicinity through a multifaceted mechanism involving Esc1, the Sir complex, and chromatin condensation;citation_volume=27;citation_year=2016;citation_pmid=27122604;citation_doi=10.1091/mbc.E16-01-0069" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=R. E. Lenski;citation_author=M. R. Rose;citation_author=S. C. Simpson;citation_author=S. C. Tadler;citation_title=Long-term experimental evolution in Escherichia coli. 1. Adaptation and divergence during 2000 generations.;citation_pages=1315-1341;citation_volume=138;citation_year=1991;citation_doi=10.1086/285289" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. F. Levy;citation_author=N. Ziv;citation_author=M. L. Siegal;citation_title=Bet hedging in yeast by heterogeneous, age-correlated expression of a stress protectant.;citation_pages=e1001325-e1001325;citation_volume=10;citation_year=2012;citation_issue=5;citation_pmid=22589700;citation_doi=10.1371/journal.pbio.1001325" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. H. Lin;citation_author=J. A. MacGurn;citation_author=T. Chu;citation_author=C. J. Stefan;citation_author=S. D. Emr;citation_title=Arrestin-related ubiquitin-ligase adaptors regulate endocytosis and protein turnover at the cell surface.;citation_pages=714-725;citation_volume=135;citation_year=2008;citation_issue=4;citation_pmid=18976803;citation_doi=10.1016/j.cell.2008.09.025" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. O. Ljungdahl;citation_author=B. Daignan-Fornier;citation_title=Regulation of Amino Acid, Nucleotide, and Phosphate Metabolism in Saccharomyces cerevisiae;citation_pages=885-929;citation_volume=190;citation_year=2012;citation_issue=3;citation_pmid=22419079;citation_doi=10.1534/genetics.111.133306" />
<meta name="citation_reference" content="citation_journal_title=Trends in cell biology;citation_journal_abbrev=Trends Cell Biol;citation_author=H. D. Madhani;citation_author=G. R. Fink;citation_title=The control of filamentous differentiation and virulence in fungi.;citation_pages=348-353;citation_volume=8;citation_year=1998;citation_issue=9;citation_pmid=9728395;citation_doi=10.1016/S0962-8924(98)01298-7" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=J. N. McKnight;citation_author=J. W. Boerma;citation_author=L. L. Breeden;citation_author=T. Tsukiyama;citation_title=Global promoter targeting of a conserved lysine deacetylase for transcriptional shutoff during quiescence entry.;citation_pages=732-743;citation_volume=59;citation_year=2015;citation_pmid=26300265;citation_doi=10.1016/j.molcel.2015.07.014" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=P. W. Messer;citation_author=S. P. Ellner;citation_author=N. G. Hairston;citation_title=Can population genetics adapt to rapid evolution?;citation_pages=408-418;citation_volume=32;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Mitchell;citation_author=G. H. Romano;citation_author=B. Groisman;citation_author=A. Yona;citation_author=E. Dekel;citation_title=Adaptive prediction of environmental changes by microorganisms.;citation_pages=220-224;citation_volume=460;citation_year=2009;citation_issue=7252;citation_pmid=19536156;citation_doi=10.1038/nature08112" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. Mitchell;citation_author=P. Wei;citation_author=W. A. Lim;citation_title=Oscillatory stress stimulation uncovers an Achilles’ heel of the yeast MAPK signaling network.;citation_volume=350;citation_year=2015;citation_pmid=26586187;citation_doi=10.1126/science.aab0892" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=T. Motwani;citation_author=M. Poddar;citation_author=S. G. Holmes;citation_title=Sir3 and epigenetic inheritance of silent chromatin in Saccharomyces cerevisiae.;citation_volume=32;citation_year=2012;citation_pmid=22586263;citation_doi=10.1128/MCB.06399-11" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=A. M. New;citation_author=B. Cerulus;citation_author=S. K. Govers;citation_author=G. Perez-Samper;citation_author=B. Zhu;citation_title=Different levels of catabolite repression optimize growth in stable and variable environments.;citation_pages=e1001764;citation_volume=12;citation_year=2014;citation_pmid=24453942;citation_doi=10.1371/journal.pbio.1001764" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. Onishi;citation_author=G.-G. Liou;citation_author=J. R. Buchberger;citation_author=T. Walz;citation_author=D. Moazed;citation_title=Role of the conserved Sir3-BAH domain in nucleosome binding and silent chromatin assembly.;citation_pages=1015-1028;citation_volume=28;citation_year=2007;citation_issue=6;citation_pmid=18158899;citation_doi=10.1016/j.molcel.2007.12.004" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Ozcan;citation_author=J. Dover;citation_author=A. G. Rosenwald;citation_author=S. Wölfl;citation_author=M. Johnston;citation_title=Two glucose transporters in Saccharomyces cerevisiae are glucose sensors that generate a signal for induction of gene expression.;citation_pages=12428-12432;citation_volume=93;citation_year=1996;citation_issue=22;citation_pmid=8901598;citation_doi=10.1073/pnas.93.22.12428" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=P. Poulsen;citation_author=B. Wu;citation_author=R. F. Gaber;citation_author=M. C. Kielland-Brandt;citation_title=Constitutive Signal Transduction by Mutant Ssy5p and Ptr3p Components of the SPS Amino Acid Sensor System in Saccharomyces cerevisiae;citation_pages=1116-1124;citation_volume=4;citation_year=2005;citation_issue=6;citation_pmid=15947203;citation_doi=10.1128/EC.4.6.1116-1124.2005" />
<meta name="citation_reference" content="citation_journal_title=Molecular membrane biology;citation_journal_abbrev=Mol Membr Biol;citation_author=P. Poulsen;citation_author=R. F. Gaber;citation_author=M. C. Kielland-Brandt;citation_title=Hyper- and hyporesponsive mutant forms of the Saccharomyces cerevisiae Ssy1 amino acid sensor.;citation_pages=164-176;citation_volume=25;citation_year=2008;citation_issue=2;citation_pmid=18307103;citation_doi=10.1080/09687680701771917" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. T. Rutledge;citation_author=M. Russo;citation_author=J.-M. Belton;citation_author=J. Dekker;citation_author=J. R. Broach;citation_title=The yeast genome undergoes significant topological reorganization in quiescence.;citation_volume=43;citation_year=2015;citation_pmid=26202961;citation_doi=10.1093/nar/gkv723" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=M. L. Siegal;citation_title=Shifting sugars and shifting paradigms.;citation_pages=e1002068;citation_volume=13;citation_year=2015;citation_doi=10.1371/journal.pbio.1002068" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=B. Smets;citation_author=R. Ghillebert;citation_author=P. De Snijder;citation_author=M. Binda;citation_author=E. Swinnen;citation_title=Life in the midst of scarcity: adaptations to nutrient availability in Saccharomyces cerevisiae.;citation_pages=1-32;citation_volume=56;citation_year=2010;citation_issue=1;citation_pmid=20054690;citation_doi=10.1007/s00294-009-0287-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. S. Smith;citation_author=C. B. Brachmann;citation_author=L. Pillus;citation_author=J. D. Boeke;citation_title=Distribution of a Limited Sir2 Protein Pool Regulates the Strength of Yeast rDNA Silencing and Is Modulated by Sir4p;citation_pages=1205-1219;citation_volume=149;citation_year=1998;citation_issue=3;citation_pmid=9649515" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=H. G. Suresh;citation_author=A. X. S. dos Santos;citation_author=W. Kukulski;citation_author=J. Tyedmers;citation_author=H. Riezman;citation_title=Prolonged starvation drives reversible sequestration of lipid biosynthetic enzymes and organelle reorganization in Saccharomyces cerevisiae.;citation_volume=26;citation_year=2015;citation_pmid=25761633;citation_doi=10.1091/mbc.E14-11-1559" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=I. Tagkopoulos;citation_author=Y.-C. Liu;citation_author=S. Tavazoie;citation_title=Predictive behavior within microbial genetic networks.;citation_volume=320;citation_year=2008;citation_pmid=18467556;citation_doi=10.1126/science.1154456" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Takeda;citation_author=T. Yoshida;citation_author=S. Kikuchi;citation_author=K. Nagao;citation_author=A. Kokubu;citation_title=Synergistic roles of the proteasome and autophagy for mitochondrial maintenance and chronological lifespan in fission yeast.;citation_volume=107;citation_year=2010;citation_pmid=20133687;citation_doi=10.1073/pnas.0911055107" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=K. Tanaka;citation_author=M. Nakafuku;citation_author=F. Tamanoi;citation_author=Y. Kaziro;citation_author=K. Matsumoto;citation_title=IRA2, a second gene of Saccharomyces cerevisiae that encodes a protein with a domain homologous to mammalian ras GTPase-activating protein.;citation_pages=4303-4313;citation_volume=10;citation_year=1990;citation_issue=8;citation_pmid=2164637;citation_doi=10.1128/MCB.10.8.4303" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=F. Tsang;citation_author=C. James;citation_author=M. Kato;citation_author=V. Myers;citation_author=I. Ilyas;citation_title=Reduced Ssy1-Ptr3-Ssy5 (SPS) signaling extends replicative life span by enhancing NAD+ homeostasis in Saccharomyces cerevisiae.;citation_volume=290;citation_year=2015;citation_pmid=25825491;citation_doi=10.1074/jbc.M115.644534" />
<meta name="citation_reference" content="citation_journal_title=Cell cycle (Georgetown, Tex.);citation_journal_abbrev=Cell Cycle;citation_author=J. R. Valcourt;citation_author=J. M. Lemons;citation_author=E. M. Haley;citation_author=M. Kojima;citation_author=O. O. Demuren;citation_title=Staying alive: metabolic adaptations to quiescence.;citation_pages=1680-1696;citation_volume=11;citation_year=2012;citation_issue=9;citation_pmid=22510571;citation_doi=10.4161/cc.19879" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=A. M. Van der Linden;citation_author=M. Beverly;citation_author=S. Kadener;citation_author=J. Rodriguez;citation_author=S. Wasserman;citation_title=Genome-wide analysis of light- and temperature-entrained circadian transcripts in Caenorhabditis elegans.;citation_pages=e1000503-e1000503;citation_volume=8;citation_year=2010;citation_issue=10;citation_pmid=20967231;citation_doi=10.1371/journal.pbio.1000503" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=S. Venkataram;citation_author=B. Dunn;citation_author=Y. Li;citation_author=A. Agarwala;citation_author=J. Chang;citation_title=Development of a comprehensive genotype-to-fitness map of adaptation-driving mutations in yeast.;citation_pages=1585-1596.e22;citation_volume=166;citation_year=2016;citation_doi=10.1016/j.cell.2016.08.002" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=J. Wang;citation_author=E. Atolia;citation_author=B. Hua;citation_author=Y. Savir;citation_author=R. Escalante-Chong;citation_title=Natural variation in preparation for nutrient depletion reveals a cost–benefit tradeoff.;citation_pages=e1002041;citation_volume=13;citation_year=2015;citation_pmid=25626068;citation_doi=10.1371/journal.pbio.1002041" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. W. Wenger;citation_author=J. Piotrowski;citation_author=S. Nagarajan;citation_author=K. Chiotti;citation_author=G. Sherlock;citation_title=Hunger artists: yeast adapted to carbon limitation show trade-offs under carbon sufficiency.;citation_pages=e1002202-e1002202;citation_volume=7;citation_year=2011;citation_issue=8;citation_pmid=21829391;citation_doi=10.1371/journal.pgen.1002202" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. Wenz;citation_author=S. Schwank;citation_author=U. Hoja;citation_author=H.-J. Schüller;citation_title=A downstream regulatory element located within the coding sequence mediates autoregulated expression of the yeast fatty acid synthase gene FAS2 by the FAS1 gene product;citation_pages=4625-4632;citation_volume=29;citation_year=2001;citation_issue=22;citation_pmid=11713312;citation_doi=10.1093/nar/29.22.4625" />
<meta name="citation_reference" content="citation_journal_title=Yeast;citation_author=D. Wloch-Salamon;citation_author=J. Majewska;citation_author=J. Pajak;citation_author=K. Pawlik;citation_author=M. Plech;citation_title=Genetic modifications allowing for obtaining non-clumpy Saccharomyces cerevisiae populations from various strain backgrounds.;citation_pages=196;citation_volume=30;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=J. Biosci.;citation_author=D. M. Wloch-Salamon;citation_title=Sociobiology of the budding yeast.;citation_pages=225-236;citation_volume=39;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res. DNA Repair;citation_author=W. Xiao;citation_author=B. L. Chow;citation_author=T. Fontanie;citation_author=L. Ma;citation_author=S. Bacchetti;citation_title=Genetic interactions between error-prone and error-free postreplication repair pathways in Saccharomyces cerevisiae.;citation_pages=1-11;citation_volume=435;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=A. H. Yona;citation_author=I. Frumkin;citation_author=Y. Pilpel;citation_title=A relay race on the evolutionary adaptation spectrum.;citation_pages=549-559;citation_volume=163;citation_year=2015;citation_pmid=26496602;citation_doi=10.1016/j.cell.2015.10.005" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. P. Young;citation_author=C. Hillyer;citation_author=K. Hokamp;citation_author=D. J. Fitzpatrick;citation_author=N. K. Konstantinov;citation_title=Distinct histone methylation and transcription profiles are established during the development of cellular quiescence in yeast.;citation_pages=107;citation_volume=18;citation_year=2017" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Adaptive Roles of SSY1 and SIR3 During Cycles of Growth and Starvation in Saccharomyces cerevisiae Populations Enriched for Quiescent or Nonquiescent Cells" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="twitter:description" content="Over its evolutionary history, Saccharomyces cerevisiae has evolved to be well-adapted to fluctuating nutrient availability. In the presence of sufficient nutrients, yeast cells continue to proliferate, but upon starvation haploid yeast cells enter stationary phase and differentiate into nonquiescent (NQ) and quiescent (Q) cells. Q cells survive stress better than NQ cells and show greater viability when nutrient-rich conditions are restored. To investigate the genes that may be involved in the differentiation of Q and NQ cells, we serially propagated yeast populations that were enriched for either only Q or only NQ cell types over many repeated growth–starvation cycles. After 30 cycles (equivalent to 300 generations), each enriched population produced a higher proportion of the enriched cell type compared to the starting population, suggestive of adaptive change. We also observed differences in each population’s fitness suggesting possible tradeoffs: clones from NQ lines were better adapted to logarithmic growth, while clones from Q lines were better adapted to starvation. Whole-genome sequencing of clones from Q- and NQ-enriched lines revealed mutations in genes involved in the stress response and survival in limiting nutrients ( [ECM21][1] , [RSP5][2] , [MSN1][3] , [SIR4][4] , and [IRA2][5] ) in both Q and NQ lines, but also differences between the two lines: NQ line clones had recurrent independent mutations affecting the [Ssy1p][6]-[Ptr3p][7]-[Ssy5p][8] (SPS) amino acid sensing pathway, while Q line clones had recurrent, independent mutations in [SIR3][9] and [FAS1][10] . Our results suggest that both sets of enriched-cell type lines responded to common, as well as distinct, selective pressures.

 [1]: http://www.yeastgenome.org/locus/S000000197/overview
 [2]: http://www.yeastgenome.org/locus/S000000927/overview
 [3]: http://www.yeastgenome.org/locus/S000005476/overview
 [4]: http://www.yeastgenome.org/locus/S000002635/overview
 [5]: http://www.yeastgenome.org/locus/S000005441/overview
 [6]: http://www.yeastgenome.org/locus/S000002567/overview
 [7]: http://www.yeastgenome.org/locus/S000001925/overview
 [8]: http://www.yeastgenome.org/locus/S000003692/overview
 [9]: http://www.yeastgenome.org/locus/S000004434/overview
 [10]: http://www.yeastgenome.org/locus/S000001665/overview" />
<meta name="og-title" property="og:title" content="Adaptive Roles of SSY1 and SIR3 During Cycles of Growth and Starvation in Saccharomyces cerevisiae Populations Enriched for Quiescent or Nonquiescent Cells" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/6/1899" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Over its evolutionary history, Saccharomyces cerevisiae has evolved to be well-adapted to fluctuating nutrient availability. In the presence of sufficient nutrients, yeast cells continue to proliferate, but upon starvation haploid yeast cells enter stationary phase and differentiate into nonquiescent (NQ) and quiescent (Q) cells. Q cells survive stress better than NQ cells and show greater viability when nutrient-rich conditions are restored. To investigate the genes that may be involved in the differentiation of Q and NQ cells, we serially propagated yeast populations that were enriched for either only Q or only NQ cell types over many repeated growth–starvation cycles. After 30 cycles (equivalent to 300 generations), each enriched population produced a higher proportion of the enriched cell type compared to the starting population, suggestive of adaptive change. We also observed differences in each population’s fitness suggesting possible tradeoffs: clones from NQ lines were better adapted to logarithmic growth, while clones from Q lines were better adapted to starvation. Whole-genome sequencing of clones from Q- and NQ-enriched lines revealed mutations in genes involved in the stress response and survival in limiting nutrients ( [ECM21][1] , [RSP5][2] , [MSN1][3] , [SIR4][4] , and [IRA2][5] ) in both Q and NQ lines, but also differences between the two lines: NQ line clones had recurrent independent mutations affecting the [Ssy1p][6]-[Ptr3p][7]-[Ssy5p][8] (SPS) amino acid sensing pathway, while Q line clones had recurrent, independent mutations in [SIR3][9] and [FAS1][10] . Our results suggest that both sets of enriched-cell type lines responded to common, as well as distinct, selective pressures.

 [1]: http://www.yeastgenome.org/locus/S000000197/overview
 [2]: http://www.yeastgenome.org/locus/S000000927/overview
 [3]: http://www.yeastgenome.org/locus/S000005476/overview
 [4]: http://www.yeastgenome.org/locus/S000002635/overview
 [5]: http://www.yeastgenome.org/locus/S000005441/overview
 [6]: http://www.yeastgenome.org/locus/S000002567/overview
 [7]: http://www.yeastgenome.org/locus/S000001925/overview
 [8]: http://www.yeastgenome.org/locus/S000003692/overview
 [9]: http://www.yeastgenome.org/locus/S000004434/overview
 [10]: http://www.yeastgenome.org/locus/S000001665/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
    <title>Adaptive Roles of SSY1 and SIR3 During Cycles of Growth and Starvation in Saccharomyces cerevisiae Populations Enriched for Quiescent or Nonquiescent Cells | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TBnfRA2Y3kYR8o3AzJH2UsqK677oygGsUu-NQhNX1io","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484758","apath":"\/ggg\/7\/6\/1899.atom","pisa":"ggg;7\/6\/1899","ac":{"\/ggg\/7\/6\/1899.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/6\/1899.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1899"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1899"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/6\/1899.full":true,"\/content\/7\/6\/1899":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XOw\\\\\\\/BnJLFxm6sBBvQ5GCTVlsTxSz3m+ohJqvnW3sgyiZ\\\\\\\/JSaUM62VyhXvdHFdwKFwb8wRU4sWhjBmmk92GqnDQShLsOcwXLuEBvmlxp33nUXNyeYqzJdsOVKlVpMuhSITzQ3PL6gZ\\\\\\\/WXAWCtmjhlVSktvo3b\\\\\\\/GYWvHH+nYCS5UX69MXjlK6hpptCi1XGzBBO+8XDHFsT1Bn7zRBllEk+zKJXZWhchAwEjOwxBGNEAbMkgaOWSWU62o2su93CWCHFhdFElDk5sOTeFS4nKPvTgTbT+VUoO2ncDhcE6i4Ec980uyAkSE4bWBAPqXAObZUglryNINHuDr084PNtAE5FXHlf+j+HpiReMBBtPKngYo+p2PXPOFfI9SgFJjI68gaZPzSOWP5sUqqTtV85fw7rNO0esn6KE5HOyc0sNopwNUs4daOjq5hWOtnUgMUdJZWKx5bKPvFQA2zQQQgCHFRzHqcfX5BRHCHJPAkeuuksL6CaApSw2tSnhQaG9byHexi9Q1uA9fZra+PiSdyWh9oresgbpZXRvCuEQICn9kl+TI9D6ufr8ZWBCEMeOywkLn9Gi6TFOWWHdRsp5dwyhAQlLlY8UVBUatw8YjMUFUODEltHA8t6ZnwIvrUxa8HemyPHDXBHiFjmDbH7c2tUyicfeHoGDTdwI2p81Mp+zGdht3FmkuYNRjd4unTiV5O1b4rOcHMmX9odXQNrLExtxbAy\\\\\\\/BBqEhr3306UoyJXs+yN4oLgCbpL49H4cldrv9RWJkJeUFFvw6LDo5ei6VOnfbSO4ieNWSa\\\\\\\/ndKF+FRGQ\\\\\\\/0BAdSBU8VFWPyqeEJIW\\\\\\\/+O13JSXEQvcOfuXRDg9nIxzuc4PoSIxVAYzRY9gsYt2D90LQSZHDxVp8xn4ZGHt52mle3C8O5X89dEopk+csB7qQiyFsxbxHHOWYVU\\\\\\\/xTqUlMQ0cA8ErbCdkeTWvfEF9\\\\\\\/yMIj\\\\\\\/jkwcUEknCNyAqWnH2GBZx7glRb5Sj4Z7uxpdtSHDwI6JJA0gvBnVFaiM9ciPf4jsk\\\\\\\/\\\\\\\/UFwZuqoLaS+7Dl9gaVq+1qHJKEl4EVS2in\\\\\\\/m\\\\\\\/afhtkAljUou++q6JM9\\\\\\\/Oif\\\\\\\/Xppy+\\\\\\\/GqLp5k7GbH0vDV2AhaFV6yZRkaB7OHI5kY2cRfbywGpw81EKpOcIk7C+a\\\\\\\/qvH8QpM0JQRFtyj882oczQCmIXjzOjFHsbdULCZT0yE3L4vKTJeaHvOHMbn+jK3sjjtMSx\\\\\\\/lPUjTYCO4Bj5mwWb5YTyeMXEf6g\\\\\\\/hYKhpnhY7624yE4sVcu3igouhOxAmn\\\u0022,\\\u0022iv\\\u0022:\\\u0022s9FzOTI6qmJffuU9emfXzw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022869268430e5cc4ea77a333e9ab25b81617997514f4bcab564cbf81b4ebfc97b9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022irQNMVYC1jLIwdfFcBbJ1Zx9IxtRZE77cMgkrgomA1hVKfiyJoO1VaJRx1GVKUn3Rh70Po9oIvXHFTc7R99oNfVb3zr4XeS\\\\\\\/9n2GaiTfnKg6X96IdniA2KlaXxC1vfARTkWXHff\\\\\\\/n6THnQcPO+IjuVbBu4DfrZXDaR3bXDUUJODJauj8OGROmNXtRnEaYt8FlEydhsn08\\\\\\\/bhj1zguy3JGU+G1KirF1JYVr2DpoFmngkQQBsBx8uOa6t5m9Lc2O99t8rdgqNni6RZrRv8eduj60o1GkqOuACD+ZDS5z4GMHHfEqlS9JuJR0lns\\\\\\\/H6J46EOjGvnKfFRX521SzRDVrjCxtZkLbkZGK\\\\\\\/bT7hxvgSEVwFspmeqZapDvUrgzFGGG18t37za+lGxrfuX+MBshBDFG1n\\\\\\\/MglGcfG9GFeAXCXS\\\\\\\/9VLNnB71jhSQYx0mIzCIDlY5wuuFwlMumN5YY\\\\\\\/qsoWBY3iGzYCqKEPNjd5G0wx6+\\\\\\\/+7bdYQmt58IMy3evTOcTOYXEqD\\\\\\\/NK3SxD2tItcsoyQuzJS0bMFzZeEjswYSlQXMX02Wi10CuLnEWqcmjFl3Mm44tbg6t0oNPm\\\\\\\/lelPj9brZyYxIC719plk8j3WYwuJ2Zyn7j3kAkThhVsTRrpCrVCcweB3cbp7m338ayc55SYwh11bfKQLMxUIh78lFPmL1m9dWYVnq7TQUclYj8H2kL9CAYgpo3xsqkPAi7qtx50qfc5tdsBbSuylLvnMbf6iv\\\\\\\/1PAZ0IG1NIK1fn0hYX1WEYH59LJgqMHSxtAL8VB7c40McV94NjPuLPbmASHvfjWu9vKqlgoAVbhQvr\\\\\\\/GFWtG5DUyIWY2Z57wUka2ZG8Ebh1zlhUDio1gyKS+iGkOvS+6XxUGzCC5JDCkSoA8MWNm8c6SZSQjOzs+P0XnXtWvXrUIcSZzdZMBz7bVjk7tn0FzNRnu1kyDHegyihl+kAWwPkItzqMBWfmSMZ1A5aYdNHEKzxSIoFkMndOfXSsUaokaZGq1C0XLuIiIPEHiE3Pk5WFvUf6X0I8+yCCxYPtfo+RfeOdSwidDVqZBMZ\\\\\\\/\\\\\\\/WCAslj31lv6lVU6f7V0aHfZNiPhY4uiPf6pxuiwkQBqKVt\\\\\\\/SuJRDsyvjqKBbXGLy4aHwbt87lc9RZlb1U3d5fgG2zJacd608w3GOKmJAAXIEqmIp79YZ7piB3M6oTtkjw9JghpqKN+O4OrmS97i+\\\\\\\/lneSKLWeUME7GlgJgRDUFmFgSjPlQ9MrIwqy3SRBsN40dmTM6Rl0V1ynkB2RtBBwslaRoClT9lWc75ztKUD5bXgslhqX63ay4fw4WzAARpR4XToN3L4u1WrcnXwMedxbIbZ1tSVOzV24ZnxGilebgSa14xGj7ovnePBiDuoPxgQaYVvD69SNlRh3Xc3VPNbGh7itnskCW95KG79ecZGyTBuceoII5\\\\\\\/wZP+0I369ycEK4LtyGvC\\\\\\\/bOIDFMQxsCRKomMFCygFuw+wZpKZX+FFDefkb\\\\\\\/GgXJT+Lw\\\\\\\/4WHdg2lGNd2xGSrqkQshw8jR\\\\\\\/MEly6MRuxp23hTYVfpj7eP7TJuoNh8JJqXOYK3Y4l6g2gCiPgtmOxVjrtYefRYTuLCzbQ8DmH\\\\\\\/ko0P6s9ebcwQ8HsPpRI0U65iYU4d6ErdgkmI0WRW4BQUXpNe39cPbzA950TSYt3NcYFfhP+j1Pm0GunmRLiQi1oaxoNWZbZv2T7KGcEOJ5U2OeITc\\\\\\\/Ff0gHgLoCIylyfjJRtnvST3dPjnCfTTtSqChdLmrsgU5cvEoIljhWra7C0oFn8\\\\\\\/3kfw2eVnzuUoCiMxveYX2JIz\\\\\\\/4jyhiW4Tkpbm5frtrLGuTgS7IPQP0fNkobWSzBNNGweDntiqqqH1kqAEM\\\\\\\/wjQKy186ozU3R9IQwERcIddo9RY6OCSVzCL2h4cFdNMVVnYuaq7XHM0giWGomednUSTfOgRy85v\\\\\\\/fFnvQu5Wrfm7DiLHV+UCVGrlCWJaysH+IKFPQPrZ1qcrB6LzIUOUPkiwKWiTweTA3WAwGby499paMA0sI0LWYn9bU0i2R0XSxhKSFSqZCckKtLeYjIr6eq1s5MVrFXluYlTukYA1vs3XLvnyl4NlpJAoOuMJlFFm9He8rnBrKNG9qwPMNrvQzU7xRnWNLn1+CANUFMqpLv71V3\\\\\\\/TsbTXtu2dU9+9uTc+Dj56IyFUeXiJC8FaasEqoQFFjC5gbWNhWGjuWVnhny8zTj51hcl00ETnkTnQJF2HhK94c2QT8uHHF9+ObQCMA5xDI09LsJJmnUeREl2wNrNmwu\\\\\\\/17c8Ft0EaHQVzaHdj3TUyUPxo6wDxilXJ9gZCYmHy90ICvQ3FMaYIia4419PQSwwt2G\\\\\\\/vKQH6DoRQU9\\\\\\\/dYNgjfLTKH9VWMUe36LrMg6uDY6l7mycG0skR4tZegO\\\\\\\/0IUBbK5L3FI8fdxUSYCWz1ynx0Jr+xiALmULk5OB+XZweIIFhF6fjLYa56kPmAeKEmf7qz5NAk1mP6d\\\\\\\/qXKUy3dbeSGpbC3GsIgqO7KUMWdPLCvoyI6YHz2z5AIZjnL7eXsUSCcl1YeMCgnlZGPN5bOaC4yu44AE+IXocBqaMpEhjDVd0BeppRricI6x\\\\\\\/brZByDMOZkaEnekji+bs6DBM5l6IiikwZy2mETvQst4YsHshoI4FZfzJz0s7mDpxJfRPSFPyAq52xwkO6IGu9VFQGhvEze86eJh5\\\\\\\/eaj8G0m65Rdf308SRPNAbDmSo+BBES0ZuR6vqy4\\\\\\\/6KnSKtn3zJozSf2XwndS7UnXvBmcGQxJdlOmZq6zNbhix\\\\\\\/cvGGrax\\\\\\\/cJEnRz2BgK1xrDqz3SxL0DtslZ2SEceXvntRaXw3QkKGKajnk0gvVtmwdJan++j6YaMT03CrUa39jo+bD0skA9\\\\\\\/Pdp7DuptSp2fgfFKfQ8YDisrxEx3q1TDqlisTqStPn8XKUuPfNbXrqyZ0B8a5IU3nKY17Rr0nE0erviVTMuqipVfrqqQSl2BCvEbvD8AY0kbZRgG7WgqjQCpEVQ\\\\\\\/S7CFuvtHuQm16u2ymsDnbmxl9yyLQjlsRV0eeU9ZdDkaayWSc6aCtdNR9qK3oIAJdduDdogcC+f57YQP7BBIC1NLCPRXqYmd+Z15As7J46cOZJuN120GL1JUIyY8nuf25RhfBXXDvjjARC5RrR9aNDdDDxo3Z22KJvTQQ6WyQCjX3MgPbot7DjqTz2xsaJAFebwu+1oWUPhaDNqMRgbv6\\\\\\\/bZOX6PrTUf4bqm1Z0mJ2RDcEeiVZbq4O3rY4dh4Oxl6MaAb+nfa4rk9aNkFlNFuY+Cn7keqMSgKTWJsWayjbbC3ySdujUIsWSWbA1WCH24NFZP6Ls52Y99uSet3rdBI1c607q1+x0jLqLd9MjACXKSNDaeDgWwY3pfOYjJdkfSuizywCBalH4ExtNiUsegRt354gHvaTwb4kxyDB5cPdMuI4QUQm63f7oOXSJ6CKm4gx3S284DcRO4GJQG+FR1aZJgcRELuaYpGvTtMupxYZzohVnXe3TEDxLBhf75YezojQSx5n8xXF\\\\\\\/PwDF12jc7jm1tmJRbarnZInIx0tRaod\\\\\\\/nJOkva+Y1xLLXetB08VkvrVBoX9nvqlt+0\\\\\\\/uqWXrQKbcNpo4Q2cbNhwpeY5oeg8x37cjZbsn4LNbdZPNdxv3K2bW50BHrVW7jKD7m10n8lK5n8djSS9l\\\\\\\/\\\\\\\/sVoMQdkwBZ3OunD55tVyZDGZV23V46Zt1acM975HgmhbH9uC3JAi1\\\\\\\/QoZ1mZtJIctWVJVFo+BscZD0KNBDoMcOnc5Gr7LOUKtE\\\\\\\/J2yU6PgsIher1jNvSjt8Sf0\\\\\\\/+Jq9DqOlOo6iaOGFI+PcWv2aKhfLz6nJ5WQdmcg9F83ces\\\\\\\/MuZi1nsYwWDI6aYpFtwOwfxAv0EhE4H7+Ocua9Q4fAuyjS5Cxrm\\\\\\\/SV5ogIP6lQYI8M2t6N500pBqPGVLUH7bNMj3Crslqx\\\\\\\/3kJm5UOD+czDogDuIFDFRnBA37McUN\\\\\\\/lQPBwQO1x5D\\\\\\\/t3912r\\\\\\\/cH7O4YlGJA3GEdeYYSxOn6vYFDip9w3gXI4Mw4hxMPoPK9T0iLr27jOZhEFV+3suKp4\\\\\\\/2p4zkt3asGTPsQ\\\\\\\/gaEi1uvZf1jqykEYVHWaiYl87b9+LjI4wNmmKUveznNc3IMF1B2oYKLnyMT5T4v4Hfjp00LLpmu1SglAQg8VQf8aTJNFQo0lhXCEcF238gTvAPOq4\\\\\\\/8vWMa64CWTCX1Cyj4tfP1Wu\\\\\\\/Q9TygLBsR3MgKp8UpgHGii7fncg8gqM4Vguvchq4\\\\\\\/AaI6C5XrjlaDrsJtmUS+QK8nck+nopNG3t6bQdP4DPgOsWqo9ip17Xs\\\\\\\/okdnZtIP\\\\\\\/CVX6pHfN7YXTZroGYGToIsI8lvPeGDzfhPcYV\\\\\\\/pveD4j213JoG0sVREx3Izx\\\\\\\/YW75OKW9Zjz06tY752KR4Yl390b4sF+JUq39ffTSyGnP1Qx1YgkXwNwtrnX2k+\\\\\\\/X7bbdw9L9HMMTgGKOacIZjo1MVPDmrzvzE8kiUofj51Mz0taFQZc1YoZlHrVeRGjCudT1RisYHHWCWXSlokmSz4HfehOgYuxrj35GSthCrST4o\\\\\\\/vstgXiQj4kP5cSTGrivBCiBraev5xDu4hSb0AWbu7CkXA9B9ST42uKUVDtdyMEhfHT9tACXG\\\\\\\/I8tmJaqaHAUTxmPYsHIEU74tZT+mmKN1eEbicnQ\\\\\\\/Iwp7dCOUh4WD1xLIaQwxlnDwAvm5IH\\\\\\\/+AlfcsSsFOwzk4DssYkiK4IYdLqNimqTph5r+Fs29eUfHXM4s5Z8CG4Mxk\\\\\\\/KjOONyu7DuofGbDWhE2ir0FnzcPU5Ixb66U\\\\\\\/c5eOvCw5yYpVQhxlKRHVuv5ARFVG0NizhZIhhdNHbim5Ya6jZyYQxOlIl1R9ZgE5M+CKg\\\\\\\/X6iS2CZoCc5+ZyTst3YY2kVwDDMdiyoFKYckMTy316jld2c6msyJ2Dfqx1lfdlNbsoh3MdT7pikt8W4aXSkezNXPALBeDUe7Bls6GzBAcOzRJAFmup3eoo75BICBLtCoZx36LthGZhwqCF+GVqEz8vgU1msqWFyCme3KG54auUIJGDEEZ9ngdHz09lo4uM+QzSyyrHRaFLZknRYYn603uPy6sCH9aOkNLX8FxwfQ7pgAWGEgXkWf9itNYgeOM0Za6Sx7Wr1qWPUgTff1qQDC1JRC4XLwdPPCA4T\\\\\\\/76tEIUJT5gOrYZ1gCZ7nVUTwUEYErBCI1vnvqWQq4z0RdImUL5VNKcbvL7JCmYElzr7G2mY0FTPt3wvOl6uFwxis5\\\\\\\/CHjwEQEI3xEmHokTfjvMAw8b3wAyrZzK8FlmZ9q5ILw0CirmPOxId1UcLkvCsfS1iQqFJVrLzDUo8HBTWDu48EWpDpJmKA+kycFtd\\\\\\\/VfQAxaMMzNU6nHh4FESL6hk2UmGf\\\\\\\/yVxOlqx3CtHpupxAU7oa\\\\\\\/AvtssiPw8guXCuomfcaEKZLQIsiVo6ABzXLwqdmsk0VTiOFGEt34vqra30jb79pFq6HPHiE589aa\\\\\\\/sok80ATPtIKszp+XLkBqtu\\\\\\\/S0AFOiKif85BRaFo4sR60zTxdfGcENtzeKsRJqKzS1meoSp9d2gUkusEOccAhPiI4rQVokUtbwi\\\\\\\/fA3vxSge\\\\\\\/wajsLgYgpQkoAUb3dHdRTMXIIlfCooRlIsPmYyOgLgsf+CMX99tqxT84vER18Kh\\\\\\\/HRcG5OCK7V5ltGksUnYUqzhtijf476nBWYadynKCdAbyaNLZlmb9eMwLMF5G5MEFqYUq1PqarKND2+GBPeRyvQQ1bBkbxGJc6D0YjNdJnFDQoU9QEH4v7PbEIwMTdeJD7LPJTSht4lPDgkxDwca6kXh66ScNMmf9wC3Y27DB0X0\\\\\\\/FO871S\\\\\\\/XpVCwDfg2aQkhNTdi3c00WNU5X2FJz9okbYP4rK1GAB8dgMzI\\\\\\\/ooROsoAuTEtHIgSs7zLcuzWoZlil5M0OiRMhmaQyMrhje6C0G5Cx9LWax\\\\\\\/zaJKk\\\\\\\/5BBHfgihLhAOkKW6LdgenzE+lzQYmQloVIHg2x3u80nnXxuU9nMAYKW6ZBxkg156zfwNovwsufbxZ8fLRFajKGENCxk6+dq\\\\\\\/BPF69OrMZBV7g8\\\\\\\/f7pQ1djxubFvrrhHX2MYlICuY3AXfAwEkd7yaBzvE+lflV2m6MyoWcb6mBdBI86LrmTfkDGT8gwvRurfM\\\\\\\/r4JJl2nYQsUmgb9UNqcsrROLqoT0dob\\\\\\\/raYObgzcezYHak\\\\\\\/zRb7UAiKjbgkl6WOHIceI+1QIpeI+vMttN92cD46k92ly\\\\\\\/133SFwZAoPxe4urRwQulq+\\\\\\\/AUgjN8gZ42GSGzYGGofr7yWBNEQHjm8lrgejxsKfx06uRzb9zRQKYZyIPvjCTqi\\\\\\\/dYY4tl91SXRXkHap+sB1c+ads41AXCBg9+B0L65mKClxU\\\\\\\/Hqp5ImirEu+nE7FdK8ibo\\\\\\\/e9q2BckEOg3iFvAbU2LFRUjV4G7P2yz0QR0T+NYmr60kbVDH+Qz6c6\\\\\\\/y92Is2PREsyEPBQnjEWjsKjvd\\\\\\\/eb2vd60faYNhZhVL2vKvAm8v8difJWdEOKt\\\\\\\/2rHJDYS6gCugspcCqMwtIYcmwZqgXHe1liI9A25LOPT5oBQKFadIMPnVJcN6iDtVe\\\\\\\/LdDlxLf+OuF9taVhVtb\\\\\\\/ANjtdxgVRnQJ4owaWn\\\\\\\/2mHbJE\\\\\\\/pRERUU3Jj4sMSAcuuftV4uwEGykTzJYVfcsKw1vk6kn5GP1YPCJoZ4n6Q5Yr1oNyfQHqLpFrXPxqyY9fd4maI9votmqig7q5PtlKrHCyO9xZf5S\\\\\\\/px8iDy2On\\\\\\\/FUYm4PvpMy6TDG\\\\\\\/3JY4w51nsbaDq+SrEHoyC4cKgE7YuLiJ5xNcjwtNNOfbpYVIfa+wrtT515kI0obz5zP89pHhkt0fqkZIQbKVfzohdYSU8gXcN1+7UEkpDQD8f9F3RoYAeXvtkiZTHvYrySsWNXbAYF54ZJZENU+SRBqCxg2nEwrB7PkNNLRQ2vh1SCQA0y4Bcb2vFJCJUiANW7RN0SCh1FsJDaJgSDwXsHfPQF6\\\\\\\/XQTEyjy0X+u0RBAyyI32kA\\\\\\\/4zKmSxYDWq\\\\\\\/3iIyrhpZW9pFzkDZ4SCqYOftlZ90qDvuFr8K7u6gmW2KwUWp\\\\\\\/N92crrK1YzDgHnTCVmMdxRwvG0WvFgXIks3Ks5RqJsz\\\\\\\/r5fgasq2hB5sF93zn8VKzjyhimHb6eGzxB4IvxC\\\\\\\/2\\\\\\\/3xQh+mAAdj13nYPEDOfEi+c+pqCHJ8HLc6WgXxro3CeDI0vylfIm0MqZLkemGQYjtBQ4owhY2e+yFU+P9vk90+23sywQu0Xi8FZiXaXlA6tJ3yJMMw6x9X4SGGlkB0Epx\\\\\\\/KPGteVeg6APzohKh4HgJsjfjkegw4OCn3X3pFL6i3\\\\\\\/I8GobrxsQYGUeWYqivyGkJsEva6mHV\\\\\\\/olqa5O+UVB8VsQnfsULN1ARllWfV1fDU8820cdt1ERTQ\\\\\\\/8OkSSxJk1Z19ebKc8c6vuaXI0cM+0g2IyHc8xxyJICZjryA\\\\\\\/4qvraR88C7ri4t09mscBj+yUGeWWiCNFkPrAR3ZH+4SQkZIEnzxpiM+L7OfWfxKj6Dk57JVedsMkLWTtLF0OvSTv5m2ASgBcwaS5tQiSYZxfwIjJ8D215dG\\\\\\\/A7yQWfzzX9Hclz67VLTYBm+FYkuksJG39j7mVdD733I9CGZKD1zjuq2\\\\\\\/re4oQUMWiH+mFzih+Co2gc9uo9JbKU+XhyKbQVZhkFwXyiQuY5ekIlcrhHrKgGt7kuWu\\\\\\\/OpbjeSpahGPfegNJISh\\\\\\\/GbE68XicmuQxqiYE01DPF8eWKBFPx06dCRhi07+eQKtB8Ov6oV99jGxEMRkFoTxbggHRaOYWovWM4N7\\\\\\\/fzFhK7RqeCLfvtCepokaPq+OFiIT6QllvCXYDqhDl4HwfcUx9WPCdJHzkt5bpoYlcE428ciAWqCD2ZcGBnnzgpQ0MKXtDtfSRCgsqI5UW0KgC8w8V4rQnJvY5lCPfhybN0mFE5F\\\\\\\/VpdnKilgf6n1AdyOWMSGO2SaK9L\\\\\\\/hu9jU9o9BMVt4Et5FW4eTlPRJtTRoh\\\\\\\/wGyg8PoEDIYHVNOn5Uq5\\\\\\\/o7feXxmnMjQHiCo64d8nry143gFPVHg7IHsfEJWijEnbwKOV7VxYDlfLbeKxaXXvFdcSmis8ecywUexXx14pZdN18\\\\\\\/fJyfYHyPyj880vKZ3wfeenk6vk+YlokdOk6IXEYovIUXf5i5Pmu+c4eZfMxBwgRGii5vv9oxo4fsRzxBO8qlJbRtGNyLTvJAe0rixNCibOG4738RJXYvDnVkK5I0GdC9QvxZ9YGPxCypAovowTO3IDlGnMaRJ2Tru\\\\\\\/AONSCwYK4zPTz+rnNlIo21XQEQc7kQMeTDBdfgnVx1162jmi5VMhM8YNj3qXpcuErvPUcpYoM1CTWPPnUJUzH+EfIO3z0P6YuWgJio2hrFD775rKHw8JIDgcAykPXG0AXAlY6nwaEg+R9Sx5ATv7MMcsA6+jn4PMcsrvam\\\\\\\/1zCOu4SNWl8fsNL+MBQNZGDSlGGjfXNl93Rd4xH1fYhTk01ZT1GWLAKNEmtAxYkcMQ1HUnTFnqN2R6qFFXroV\\\\\\\/s\\\\\\\/YhU0KVjsBIxejPhx9Ly7XnszKfskaesFScEo8ROjQV1f7ZmpxeSXAh27CSdVEwZ16vKPOL6hIsDb34oPt4G\\\\\\\/rebDYFJmEmXAhfkK\\\\\\\/DKoKIuJBe3DWEdXR6F3tsle+gVDH8+7go7gKUd\\\\\\\/BnDptvikS6tS18MPo4HR69m4qvGPu5IvDIYyXNSUZ6dRJI+3LaLWnZJVU8+11tuDjjBGwT2Os3Pp3CkBBprcwzejN1zYcq+HZO\\\\\\\/kmrNJZxNuuOV+FrvFmKWL5ViXpu+in3g7j5my\\\\\\\/TvsgYAvZ4Q1QQDLm+YxNbSnYY0NW5Lefy8a88k4OcclyhwUAdM7yhMXPFBLAB4QJLeefHg+Q9haR2Z61I+3r6MVSPYqrUiIU2pqSpqFuCXin\\\\\\\/jsVH\\\\\\\/U4mlTeiFJMUsTZkZK+zFL1bndIzOBX7II+p1Wiqy4kIslTUsyv5ERxjeZVUST28NafxJ0d\\\\\\\/i0ZJaWZbMsQS\\\\\\\/XN0BtOCrMWIAf7UZzEeCrwWZRt7jrwG8EGID7NyoWYRuwoDSbnloNrSR84ik51+BbveonWTfgS1v21en7mZ98\\\\\\\/+vB0nNo6JGmXxcU7Ewq2eSLEHi1DQAms6NzB9UwyjpbxU2z5ftIbdFT+nuZqsGX6uuH2wH17x89VrhEPC2vfMOYkTKHM69r5RICHNJHI7o8qsfedRZIOSXqiD+qDgLP8IFgQEhJTQIVNnhYhYSAyg+u89qrzQtF4LxIySIW6+zEg3v9h8SblOyAiHXvArWw\\\\\\\/o5vdUKI2dLDfVweLwqyEhT1DZ5o6LtvB3Tw6dVzi52xL2X0E0z6jzb+I2XZgon+4byC4ze3kBcYBofQY1kgqd5crW+TIccEUcAgH8KEI94M4oWa077pbBNqU7OeuQbaxOI96G+4PWhG7zLZzaWk3HmCloBxpH8gdTjKRmS3u9ZdvD3iA3q6\\\\\\\/vRDtFlVJtHOejPK\\\\\\\/uWgvJMnDctN1BheVFUyiNKRj1Rh9Ri5lMw9Z8rti0HAlXp\\\\\\\/\\\\\\\/LAKayzAzbbWoM3VuvUXNPJGobyHReXvU9Y8Fe8FyRfPBC\\\\\\\/tsjmP0Epa2alk3RMsLMeGKgbaRrigwixjfjm\\\\\\\/bqtgwJ\\\\\\\/2CIvIQdXHXsFjfxScQV+qfokq2qpHbfSbCOG3y4lCG1FwppUg1\\\\\\\/w4MCN0YmCo1vO6WTB5Q+BvUt0rB15U48BoexKItvtvTWFfScLkF8noqjIC6r4HkizmRU7CHwj1DKUfGkUv3pDOOArEvJ1DcZtVLMtMI4Lo7tFEHprgA9hVjDD33HqvXIU4e0uh1NO\\\\\\\/xcVi49Db8Nxuc4g5HdxIK0RtoiUEwYn0vHxPW279HxyXEtVhNJdqjYsU9F\\\\\\\/zd1AlpP24XZmKDQjlahUjcz5ToJZhoHVooqzznjFIGpNuBaEl2nwexIPcdRskurE5L\\\\\\\/\\\\\\\/ZCfSAlDYHEIJP5jPTgXBEif\\\\\\\/3Q7VX7ziHUOuX9y4Fx\\\\\\\/ggimjS1ur2TCarPcXSd3LNeP4X4kH+ITIRBn0iG+aFOPAWq4F2\\\\\\\/\\\\\\\/kAiPDKV0+4PcJ8Em8yECDC3oCHrbdj3XAYuFCf4MscWeaPNkxnSZosyaNSzAK9\\\\\\\/1PEm75rHHN9WB4gZSy5izWOQhwC1AuofGxkaofgwJmmGAS9XRBP+V67PECP7NNarUf6yARvjuFLKeG8f51qfKArZx7CT2eP\\\\\\\/NHNlr+E42KheBnnkZrPL48QMCXF6o\\\\\\\/gQng2gSdzG0oMmifvPLJQVMkeiiF3+htwR78p5TL\\\\\\\/7ykzkaZ29fqWUYGskeNY3uaQoMinRokCIPb3fvahO8OzwWN5jWjHgFbeZExow+JG0Vo0O59AO9nnox7QfEdChdUDH96xa+gHcnULohq4FxJESFyjtCyboLxqjc4kpMWxKQoO2PDyG8gTNRzbvYzso1c6MTFPqqFYo9CP4vuZLcbgxtA6C+QLqNuZOeiW0FAvfj8M0mPnFfNVGtAB5ZfBagZnPrVDvmeJqIhiqeBEnSDkWU7luUJ\\\\\\\/52gUOjgAqTROFUHSXmT9H\\\\\\\/545RyAzelHrcCk+Wp3H\\\\\\\/SFS9HM7jHeb7PGYifEhKgHocODn78YbTnf7WaqlzBYdMYH7Ps4aizaKoOXg0NZ6VV2wdG+GbDl+ACVWFv0DBzm6J+h8MCaNNuhtex0uAgK7Mcqhbn\\\\\\\/c2vC2\\\\\\\/n21dX9xvNLk\\\\\\\/Q8mgSEXkjBw+QIAFXW4HqdZYq9ZXcp91fS9VhGxVySOHeMc\\\\\\\/ZPoKJ9\\\\\\\/cvsubrWJoAz9EHv3kaB89EFf2ikX8oY5T9CazHwmHrgqRFmOxRUf3gauo0gVF\\\\\\\/pyuXS8kAE2KaOERmw\\\\\\\/nGFZ7OIYlflfH4xoYCFgUIuFTyRaBrtOhO3ezj5UN8A6Ybqvhchkt0qX1utEqXKHHt273NrbWa+dkLnXsWIE+OwzQxurnW+maMDCxkLt45yPqx\\\\\\\/BJrnKVkTdLTwayiF3k6xgYRHRwHdDy+AVnq511DFo8dZQ6LMgR0h9qDONKrwTs6Y+28NkzgP3KZCzpwIiJsYms46IVQiHvdZgb98Czqx2V2h5owR7PsT9UeIZmdlg\\\\\\\/gSqsHUyTgVRhaclo73okSPZ7rvMcKCkGeLfAr1Qe2bjJwbG24cfiE5YJJlQELpMnifivMR9Z39tIPaa4T01CoWFRzcnMp88XBU16DCNcRXedyvvGrpkDR6qVH0iAibV7U38Tjb1bEJjnk\\\\\\\/tse+WxfpKpIcATa3odwvzIgxb5NqzGQU2uc56Wais2EjdvQDmOcNOchCugxFwQZFiVLjVJLD\\\\\\\/zn6vumc6nJSpt63Rd\\\\\\\/F7NG\\\\\\\/fQMJYJJcHCI55UA0bdG499w1oBVqA2qhYQfcD8CPPKj4D2FNd537s4XXzCo4uyiAr3Fag1ad1xvATpp\\\\\\\/rM82BnLqPLsfRh+b46G2Sd\\\\\\\/eFimSsPxhulYOQizu4LUh1WOt0juvh4w0DBKEhLfhPBDpchLOSxSAImKtkgElPDAXrjdFWrN1qeXaDVetxxw3WCb6CiFFXJ4Ws\\\\\\\/FC1uKuwCNkARen1o+AbPPlWai89l23CyPGUrpUxgRAkU8Gvl964fXUePfx9QfeRu6NKXQnscQXz0MYr5R5L\\\\\\\/E\\\\\\\/vcW9tPCZmplH5\\\\\\\/VIs5sKIxT1jpXunvVnOKvkGw2+ttKxrVAZ2w0S57fSxKs89qAeSqsTJrOECYckvj6CVv5tW5o1exD4mzm68HGolzE\\\\\\\/6cyrO8I5s4VD8pAgRQXcmR4crRHYCpaYOWcaDl1fPdJDbrFIWx+rBKn9xmoFtevdq7hSRmOavHgOxqHjrckUoiO1TJvBNfTYetFjMmvndvDjD4lg8LC1q3habCWSeIYNDh2l9ydKxUpIkIMpEz21ARxRWnbkO5AyOitlOpz4LNcurtYbbfq9S3U5Oh\\\\\\\/96n+jN0Uaj3fwB6rrTPj493b9Xy9pxaM9HH9JmBgIy2TeGTzulH0p3XYl6htxEYjVJ6KpaJh9r35zW8ZsbYVudckOHOpjHYgxdAJdvzK\\\\\\\/NvG4I2PkMmAjufMylNv8HBQE8B+iySNt4RtdfY4t8WJ5sNtxJAXpQsut4IedTwHJTZ2NzIJhp528KZWpGRYCa3VkSmsAlKHOJnB45VSFF1IGvKZjD6U7kzIA\\\\\\\/pg0aFjFgFQQisM2NSwc7X\\\\\\\/Dw19iO62iQ2kshZY3V22SxxzKsYFYwHzLORTERyjf\\\\\\\/t4Z\\\\\\\/goa6dH6eVBfrGzO1\\\\\\\/cRDl4dCQUjOx4BYSW6n8rYf1WRhhjq1e63itvNdy3LHOgCXA3vYcXwtTeMbdreAq5siFxWuiAFDm1K\\\\\\\/ENRPXfzQCUgVVCX+IXTJ1AEh4e\\\\\\\/RKJdcQ2TTJ8LJdSs8UQi9yR9nSx6Gs\\\\\\\/Seds36820lZef2IbvwqoKhoHCS0J0URcePLKWNYyZv8x4H1RbG4PJq\\\\\\\/u5htSF5MRmUg+MN+Twr+tNF92pPTavvgmAGNwQh0Lmsy1gHsmIZs2cNns3JFuT2cBJw7nV8JLB6gzjrQENaQxzFNv1aCavtOBe9Yo1u9AV0fDcGKvxbdfeaytgKDyoY8ECOXU0xj1l0fMCkiv0i12K8FcEITVgpblzma9TvNPScYcNcOhsUSgRBfcZA1g2sFXUTiwSHNBXypGe\\\\\\\/oroijJT1vLGkK500i\\\\\\\/B4il9GcL3uNCOI5oyvv523V+BpxcJMShaTaTke0Kh68wStdzLg7cWnDnkvytE4pn4PFwOkgrLKVXoZq3fLVPCYFX7TWRA1iLkt2iBLsx8W+LoJ3TQfOglyYSB4rz6UoxU6EcXm7IQGemp5oUN5igtvnZx4MZyZ9fwcRF2hslER3Sz+3buLOdRJp6BfwVTR46AcXzgSRywyY+nDrUPylmcithjQhTUQTyz6CkH4\\\\\\\/AqM\\\\\\\/\\\\\\\/RluzUW11fyUwf\\\\\\\/nVFlhZJ2+GS6UpWgje1xdSUNb2rWCHNY3cyJpJe3SNBL4qnx+zJ7bv9AqGDe17mFXPKGdOrx3tF83SIX34r4PZjolEUI1B5mlc\\\\\\\/yl0J7J7A8UCxnwfTwSTB4JSjqGjqKCtxRiSuPgMuXOgtLcxSID\\\\\\\/A2Hyqt8xPTwdxjht+S3aa7sTUvdiWrRmK2ratw0u7YsAfYuFfdMvAmG\\\\\\\/q\\\\\\\/mQ567OmIlNvKxA\\\\\\\/J9\\\\\\\/ZSPz3Tp0OWhq0c24MlJ+tK0FckR3iirLNB8xM7paRqTyLm\\\\\\\/AbZfsTOPII+OD\\\\\\\/KscVyfDVCJz\\\\\\\/qdCtDh218XZSBbjAgmes+nDWs2VVqJ+lTQuy+iKEHxx2yzaiQluvMCjf2vfywTAffhmHw\\\\\\\/w8pvBLSLgCkV6RJpye9c3Gsf5YxbXYHSBtPRRZpFT56CYWOYLrweTIOBiR95R0pgms+FbflXvyh1VVdMwTj\\\\\\\/SF9jK7mljscz0BxAx7pZxjBD5VNSFC8PPVJMPsYSecxeJGsOJccYufCHRbGEaVe2d4HkG4pxTFLdsrNP2BNj6CrOQSx7ffGGKiB37WgN+SW8EcflHa\\\\\\\/YfKZZ3zq0JcqDf1QYpLGnXxNZ+LCmzrymoxIzSWB\\\\\\\/Ss6wTFQum07jaTve8gpx82a5eZX6oaTGhC7pJsrK6YFbAO4N5d8FrwXzXRq9SySHuYwwbAHwdSByN9L2w3RNB56cd8WI4Xa63iW8yrkJ5FdvUEWUATvw1\\\\\\\/mxX\\\\\\\/9iDsHV45SFQs6+oYxGaHlgkXrzjEd1J06+MVO0Sx2MU9DC7C4vzsCszNW6AbERCMSgCmpstLy+Hr2G0af7Ui60HfcrW3Jz0ioLUeGzv6m5MT3pDo4WDIlFkeN4IbYz61ED\\\\\\\/51jYPWpnglywZtWvW7x7YldYqvq0ZflYMM4vW3r6QPfaoNw1h2EV3PXMQAmUBBwmTf4La5Eo\\\\\\\/xBv\\\\\\\/Nl1ko7NWOBnpErJ000bxqgzYuP5LeTarc\\\\\\\/E7I1v4cj9OEl61noqgUG94SVXH8TwtSMgD0LbPumJmwVMY52keL\\\\\\\/3Sol8O06aUhtL6shGimKQAx0sy5ZwB+UZP7Ge40l2v0CjYxrpdsTw0FGkMuJPYsII0MerTq463KrC3Zn3JWAZRvUnDXJoPLxKdhEnwshSOoTNxM+fzG3lcmmG5ZeYhBQa2MSa7aiNsy6JUv9Q9R1r9tX1fiwH8VieKV3c6iOcluGssREa90PuWBzl8fPVWVl+JuRe3ShYzFk+MZ7sIi5Orry\\\\\\\/ZZquL1m4ccPPMXWFCYRLbqybHJJGunBCdzSbbE\\\\\\\/uZF9iQZuCkMmO9q8ctEBLBZ1aAIYVd4qJ2fk\\\\\\\/9bgusQo7+1Q6r8UWlKi3A+hQlFrkkHgRuHJ1pDKpXrXH4oxG22COM31rT4UZEorAK5B+B5OPWXRk7GJ6AIMQfoD4e+YTwjvb2p\\\\\\\/4UFZAbESu2FTVAHeIXvJnq6j1iDCFtlcdOm77RYg+tjQyyZ97ZRC1DT71M9rJ6TYMTb3a4SIatZjTllqcqP2VwANoftoQhoUd0E6CJsEz7v11oa+p95ws1HA4qtmMeVz1q3XmqeU\\\\\\\/6FndA34kC5Y1z7No27YyH2r\\\\\\\/S2H0dsG4lKWFh9EnenOZ6GVcRTH7i4icUqVNL8A4oa1SEaB3U4ewmHpZVwiXoPboUciI4U\\\\\\\/NYqZTP8yNi1OWQKlEpLVD2HrBqFAsaLB\\\\\\\/tjMoFbqvmuN8RXqmSe9Nq5SDxR5o1ABlQBVMCQlnfm+uervfnQeF8oCB+s8ej2i+m01ADk83xJrOn6OGP6ki1uZniWz77jW7wBsQUlN2hxHYV86d9PVHJpMoWqYgfiUtjR2acDh\\\\\\\/LhoAMOUpaoT95kSwHkylT+EG31z68rgv15f5l+OOmOdXTNX8Qkwm9AEQ62jMq\\\\\\\/Z9p1r9GWjASeTl4NBauwnLD1wW6oJkTVbv867fURJVebk5wsVOlO7rDEDgwgcpIV6oW9u6vjUbbHJFhApE6TD\\\\\\\/4w+FlhS3RlKE+Cj03vw\\\\\\\/qRjr4IlnULfinIC2l5weeDELkMqPJSY5kVm9uLswvs68Ei\\\\\\\/LJgCImRv4NJcemDGtVh3l7QKfVG+0lUneKmwvt1zZjKIJxEPIxEf1JTt6x6wYkpO8CtA+t8B2tJcC5UySN5cl3hRXdpLIszlSC6lVV0TEirRUeoHTqsVVTgnIjpeADZInQJH4e6IBvRfR1p\\\\\\\/rGT\\\\\\\/tka5ovzE9PkrGoLcL+sj\\\\\\\/iLkjsenKGGfdcpityZZlOloqAxLuuQU5E+5tdxkXOvVtw8NO95qSZE4d4WGBfU2z2pkmO2qhq2xQQG1mXKQ4IJ3zF6tkkCtbBSeIkgeU5L4mLtd9ubrYeWjqUSRiWOOhNFRRfy6a8Oo7V0b8G7+bRKTyA2uQIjGyU6WL4RB1yNr+BOaPAlU6DgmpsFVMaStvy6CLsETxpAzTacmDg7C3rWnZpkCc+k52adzLbAMN51+R09GhzuZq201PR2yzQulHlSyo86vHuRqNGzMYTOlkua09jCyWUm0BcajXqr\\\\\\\/UhgYu9Z4ukcQDVUCyUHZxfdaa7JOHjS+hCwgoGsZnJp8akc+m4A4r8LTdJNx9zvMdG3mmtIhwUtSjjj9EyzAQVdTheWZUAgb5FKFeky6lPC4zMgETPdqp4ZSapiJdLzFJNSOE8GXkcrnQwkeqv1gIW1xWqboOkCCEZahGhgJ02UqCE3KfDgphh0lzzQn4NdASpG3nkqymoz2M\\\\\\\/KesjLfuLuvrcBFW\\\\\\\/yuLis7bjmkqkVk4g8SSYE3UjQJ3YqiyXrDiWYtfzpqFrNv9CTPhACjvePvRvu4T2hkMKCXavW3uxWp6XGv6vm+nKMlL+LWLW4\\\\\\\/T3bEIO6D8cXdSuG19+kvGP71eZ20F6tBrHP5TRaQZ4mFrZr\\\\\\\/pPH5ZeKOA0PNLYjAP9i2UUM8aHrAvBqUfnp+6JLgGu+EKBNlwLxa7RQXUDs2qXg8PshSR+4bABZBVI8KCH3nGaCBEvHwAMyR\\\\\\\/7X0+ND7alDazJaBCGGxH5DCqVMOzGhyvi9dbxd8vJds3dDBtGcZQTsoOeF7GKc\\\\\\\/JyWHZoe8mi+puDXZgWXD\\\\\\\/tEfuH6J0SnB1+kCIn+tBdkJPGsvWGMUJQdGJHnanX1ICaSwepJbkQfOsO1hiGwZp8fUZSZTDyeB99qmeI6bmdGWLvOnsmUf2l8QQdP1ysl1HWThddF0XmfvgoaNMuiTMdU\\\\\\\/Kd6cIjkwWl5jD6fVxj1hea3G\\\\\\\/Q5GMd9htyA1386NYnGTyjEY+aWNv+L3PhyOK5bnSTaOJ\\\\\\\/W3LDUUzeq0t\\\\\\\/fH2sIYStMgoYw\\\\\\\/jKqATtLfVHUwro5mKeRY8quat5bkaoTBa0uiD1FU23c\\\\\\\/zubQ3XTUp1e63XG8bffBR+xTHgGiQI4ozDf4QIhaOtZ20ccOlpZhf7wagbd4doNYHf+H7NFAqs2ywN3mH5\\\\\\\/NsXxbAjQ6DN78AFLsEJPb+7kbwuKBKU4kcqA9CkYGMgTSCk5MBU7my6+HVUsuAhFfsIcLzEuQOyqy5hXwMe12LBXuGaQfEpeday7rOkmFtqUlUecxu\\\\\\\/4zM\\\\\\\/rSrfgKmuOHnFhADUsOpAQN11V1xnJGEXgpIos0EhIhCDsRvtMFaqhUqvSvdiDdxeoE2gc7VEvSagT33ZufJHZ7C6b8z9IcHiPVN0DC50JOsJ9VmL8BsWvGhOcyvhxbTyFkHbNUnh9eBi1PfGUo+bhQoMcHQDQdP5ZzIP+i6gp1NaodLv9OljLvGKuSFB2ZrSPS+C96LN7E43NWOK3Oq1hq\\\\\\\/NMsjuhqO0HQm6mpvn3ZpsRkG021O6lKtwSKiZiXzoVQowRt1NrRIgE1PGiSP32WI5GKmsTdWSFdKJGqY6LflK3zKCffMJUwTcvryL0D8bdW3YbKFPfPR66WIqhx7EuQW6vog0iyL3qfqDhaMLQ0N2m4XnSI4SuEM6HtrcY2SYGorwQO5QNgU6xZcReoqfzdZ7Cq6RnKlyzNbnoMHM4PxoOXC2UtY0dsdc5W3A0i0\\\\\\\/Xz9\\\\\\\/RlMGWMq0T8x\\\\\\\/clk387dASPH22nixY7gdq4+Fix3IECHbUe7txMYOucPNGayUiLmqapqa8lO8cJQzTHJwCWaNrvY\\\\\\\/0t6ckAoo1l4uHvMJdb5Edd13MnuiYweT1jdQ1+EoEhF08SxghP6Njd6sa6UJl9X8HzWuinqsGOMZxwKU8C8QIK8VN3QYP6bV\\\\\\\/2n9TcXj3aDP7P99o6FtcZ8l5uUJbsqbivV0\\\\\\\/mxzeZ1es1ZAoaCHht3nmnqU5BJ14uhXFbDyxBalkNGjnt9NLZrT11vH1OJlQWY4dFp8ZvemQgFe\\\\\\\/vKwSPvzhCVoGsO1cfoC+z4\\\\\\\/faxaK+czXqHuI3KZPpcoFntzZba1xnQC8qr7ZhJ7qI7RLscF\\\\\\\/KHV48IEYEu8BI8VKo5seCV7hfUhTMfX\\\\\\\/nMbi+RD9NyxvuqAn76HLfDpVU9EMwFUKwrJ\\\\\\\/juityJcfKaMdyevEprCLZeHggXFhiXgfLNfBEepKTtlevj+MUcCxyDpbiwq0dBYQYB0ZVpQMdEUbjY8T\\\\\\\/fX2U0jbrbwKYZHiiz2oFonqoDnKYtO2vDzNdlWHLdMeds1+mw\\\\\\\/drunMpxI5eqnWerbseYNe40zoSvVtDDdDxz7MXhB7eCIXwIjb5AFKLl17VpTguZbuZHSi6nVeJrMhWu277T8VUaFDsxm8hR+GVuinihowXrIcMDHaTHN9Gyws2gHE+Wvpo+pV2VorDHt1LQPFuk9jGpn1VZY2LGhtid7kdcrX1ayXv6wsLbY4fylZdpQpjHsra9e1zb2Vn14vCyRACfhsj6sEAd2L2gUdsEG\\\\\\\/RxyggOsFaq72NeHmfWuJlLHbSqvDl0Gkjn32YYyOuLrr1VarKGDzW+7zHwWpjNovJSI\\\\\\\/517Fr01lghuf3tB8EueCNFLlt2bh3TCPBF1rpBLAcDBddVOwrq4NFYuk4GQaHO8p6zRy3G1WhXqUgOCqyuYg63tBfKk9UyvIHt89k+Ul1npOU6CADlfORdxdT0x4ab6zioXCrlZGKOyF4mqJre3ef3vudgLG9tCHKcc0+rRRgV1Gw0Lkz2RuaiUqmYgN1NguI9ekyBgVAhTM0iv3LH1mCFRXz6dyPleq2xo8MeVHl2Y3wioib1+h0xvhORHgcqcUHuifJUhZK68HtOLeh2DGSNeAlBysayZ\\\\\\\/5puRVKhyCyWJgdaYPX59b5+ixL2\\\\\\\/Cyw0yuIa4bohoqtV1LqZU5q59rrSEW17bWguf02MiKDZdpNpSIYaah8ZR0NKjwNakY04L1bR7AJaIml4d7jkfZAYlZAVEqHyCoaJKb\\\\\\\/5p0GJhiTeaudZ\\\\\\\/la\\\\\\\/kJpfjHMg0U8dz3HGaEz\\\\\\\/NacDLeyZFbo4xwgdSbvpwl3jEXVqpTfvcI1h1tGhxj0NRcpGYB6tHx2o789Me7CA3er9fzywgVep\\\\\\\/hvUxGa7x2i0yNgx7JRiqC9Jqrs+KZlO6FT0G\\\\\\\/t1ZBCDgsMe1z+hrGAHLm14j4sicbeZrbSxA32qD9\\\\\\\/YJrZzeLme6feGlS0r+ZUmTL\\\\\\\/rAIzGRWFPteBry844M4lvxSJitWjDN3+Dg1WCoXmG\\\\\\\/PGNQBhzKZJNXoceiqURby43YZ3qa6dXUtINyNJLSxELQXkk2zPln5GVdDXpXS6bjT7GzkmjGwfempiP7cX8iD8vAnKFQXS1PQvJDiOHVazFXZZnhnr\\\\\\\/+BXZnTRS2Dze2xaf1cHLdd7GDDk7wYn9CIvgf5T3IWjq1wXTMzZHZrLtY+1sbmtGp\\\\\\\/wX1oOJnQcghHjfgrfeOzw5bUSmE8SlvSxa\\\\\\\/kpI12AGnpvPlkfBgPUrtoRhDV1C3iEdtvcT8sqnTvpGi92CZJJs85Snfs+4VF1cTspH2Zu0NmLwudDQlSukb19vJR7gddwwwd5s1HpKoFQN8pUAWcG35Duo9\\\\\\\/gVPPrQQ08F\\\\\\\/VfXmqqCM06wL6fe9\\\\\\\/M3CHJJeECp1v2h4K6Y0RDrnySrySIxa8SU18sv2dm5wYSsY0hiZZlX+o68XXjaWGLeqmybg4\\\\\\\/SsCQXMrmcH0edLWbc6X7TwEaAH5wq2ei5+F\\\\\\\/WiY10AGQ4g5\\\\\\\/F7aL\\\\\\\/ijnJNXYAT076bhTxSFLN6f0wgbSMLD4KDsPi+eq1Ym24GftJ9R9oibkhATVsnHBp2pjjS2G\\\\\\\/Rk+p+41jsQml871p1G6vC7iQcwpqf7NAGHYihEYZmJE45gaSLZpeWjAA+Cq\\\\\\\/E2E2xfoqrZdEiRS0b0vTwZx4FJTk+SWqGALHwgssQvP09yo7Sy0N\\\\\\\/14rvxyBpQ5Q\\\\\\\/fYAvanlZzTQ\\\\\\\/4vBuI+0ovyv+MK1WzYT\\\\\\\/zzXKIhC5n9E+j5u0hdK1NBuT+Llcp6uRBUXtIp7Y+wNTzGrsheMWcFSghdtSN28WZf+8xtTIPWBPIbEKTEl5kbxzu+hpzO5VDyAENynSZfdMRjKfSWY9t7Rr7D7jsEKP0uO1rCviSzGCucqi\\\\\\\/XH1Hkyak6iDVfxRncBRTYSADrDODuqqL4xDTeREwKLZQ8LwFshLFOL9IBrI5yfuTOYgy68Ckzi7nFwE8JMdRWdTaIIIBX3xbM9+kOIrRP\\\\\\\/OP5pff2owYLK4UkASn\\\\\\\/WJDFz6qDPV9mMpGRM8k7dEEj0AYWgBB9hJbAtHjoQNyaHIZ+i7ew4fvpf\\\\\\\/30PEvz\\\\\\\/IH5W36GO89D1t+ZvLuqAtVfRHD+OhryxLm0ID5ePsSQZICQZ\\\\\\\/Y2pBtdRm193QYQCNzvDkYiGGXuR6RR4Z8CzMjEbIBld+AFv6KVoWZ5cLcb\\\\\\\/\\\\\\\/DD0akW\\\\\\\/TMdGz8rW8apJzaqq5DaloDINV9tgov8NJ42ktbKnE\\\\\\\/L+7cA2HgZEc5CZsi6zyNkpsQ1QGZ2JhVIqRz86KQ7dr6A3vF93p+yoyhm4ExtumPQDTjgi1seNVaJLsBkksbH4gdy8fSdCA2fB3ogc9ReIN+3QrbwK5d+zUWrTqukA4MrhIoAjJjA3AVxOVAjVssXr5HScSo4I\\\\\\\/fpfv\\\\\\\/GN9yUaeLGXC4uLmQAaug6veTj9mdvkZK9FlxeikPoLarnZ0KCaTWD1jb1R0oW0AOH21pi\\\\\\\/hAv9VcWUGT508NK+e5N\\\\\\\/H6l\\\\\\\/TW2cvEMkfjPTYgOAO5EzkqTVr1UCVt7\\\\\\\/+jM7\\\\\\\/v7YGRDU\\\\\\\/XB\\\\\\\/d3+d9SEcUJMp9llXWn6PYZwcEKVNM9JexOSGeEdItpSkHKtHTDpooi7xIZAFv2HU+146P\\\\\\\/QUoA1p5d\\\\\\\/XM679Ala+Tztv0Gw1SbCle+0TBOq06FdgTQRhOAQx\\\\\\\/mqWgC28MMSQbZvCtFWQ3NFDtWbywlt50mjguzAqpckG3GfsuKpv8tyYIKnv5r\\\\\\\/hm9yGjfIyDGSvnnhatJfsMGWL7GcFF9drijf9eg026NC97AYnKZ1GRFczeWh5WraxPqY2GwqHdhNDJQQEdBFV5vbvI44Phpu\\\\\\\/EGPLT13fu8LpdUWPJhVRzQov9isfoRLTFIIn9RSzdwz8QTJcxfYofuWh\\\\\\\/8cIwzvimlSO20OX7ZXuFvM1CV80vqYt1y4zUbtRjoq7F1vA5D2gfdKwaqWxgVQKY3tDpFB\\\\\\\/C9ABmu0ELsvLdHi21VEg38PYJ+ks152a\\\\\\\/Z6PX9SEmW4sx9\\\\\\\/sY1mV9Diab82QwZBOXjWt8xdcliQj6hgDYSJzs4zwXAIBQNYawmSx4T\\\\\\\/nRoQynmzeeXZgJUn3+N2zDLSHBIQsYhJSkVmBP3bRo+y2DJt82+6vB7xyqMbQOsWrTS3UbwyeOmppcdXH0352fuasEvEk+29rr9jp6QcBGwOuqSe6da9zrR23j2MjEKkwextMV8Whnh0J0qkSSpNGcm3qhXCyy2UsWJmAk07Np90PXLOlAaiNfKgUh0u0TnTNNoDyCkoLx+nXMthJGH25i9rQaMfp2ME\\\\\\\/n2tVkuJDhWRs4QZft3OpgYcAP+3vfsX9Q4t08Gbac7DL\\\\\\\/mv\\\\\\\/jhRkifP5NDQmvkp\\\\\\\/jEIr4L8\\\\\\\/EIlf08kya0ugK9hPYxvuZyexo4U2nmPsgkS8C92o4Wt4ihIuG1SX1pw++ahk\\\\\\\/zfuQDMn+ONbMyzUvVhbkVk9pooDGo9d+OCdl4TkKCLZZjLVEcYMU10cIBaFM03J5dCTvFRF496Zt9ugqW3VN1dq20+KmN+kvlJqu0pxejdkxTN6fqFOXxv+aVWBi3\\\\\\\/NJlNiLUanG\\\\\\\/ysEN9VyiWwu5YwZZerwp+TVMixEQ0xH3TYwaMgapXjhULRpk41xGBtndvAD\\\\\\\/pavBsMR7djQBWZEDy5WMvjBBp3tTjSvNJLNOnGNKOoYz65SpqR3T9VpAgaUY1fxdg1+CQKofj43h7PIvyHdsIKQJaJSPyBwTphgaBw3Pryx05xwMzAVYBgZY6cq0uD5GecJeEwFOIh2FuOLg5lryVQlGwsvCecs2BTSj+VqJwX\\\\\\\/1YAe8aG+Ehdhoc2qPmkNza0y7\\\\\\\/00mdUxEHSRbjE\\\\\\\/S2aSNm7t6SfMrSYfCurSz7ZnUaHPDTlPCkBoOJm0hpS1d4\\\\\\\/A34ld063x+3WnVKBkwwR\\\\\\\/VIp4U0GQcUQaQ8wrqCVgrHG6L0Nt2IttTuDkqSus+9emwZJe0hpS3w5orpe\\\\\\\/fWBC0UKRLWwYDHDKNvWsPS0s2Qc1xfIELm4bvd+om1K9NoxdcgGwl6wRQafuvz46Badl3Q3sot4AikpACq9eLfEU+hkHtvkZ8VAe7Aaiptx\\\\\\\/ZXB+6ymR9wNPruA4PreMziOUXPFUfxdPnAVTF5+cyI2tu6DqP\\\\\\\/3yjgFt7NTPXbU+vsrXUfE1ODU1p+dr3lU\\\\\\\/d3nAR00VSMeq8Tg7\\\\\\\/sv579chztM\\\\\\\/kvP8Da9dSFzgHL3nRBD+kXbP8SC8W78WdQkqp6oTbRr7it\\\\\\\/vxQzwhIZcuxn4HdPwSfgEluSJVbEHk8pntRzQdMuDpllfqkKs+hlnhZWhxQVUdpukNhe0X4P5AmHmgCAdfSNu4llWEZj3j7saAOI5CjJanh6eWil50Bz9Az9IlUAn8lXvim345DBfWXQoFm+dEr3\\\\\\\/Cw3O9++flC4lCQBy+sYDtD2dUpuofgeELR+ME85EKgzyTH+bc5I2P0EVnDu7Ew2acV6obJnccAWd8cz4YfQr8SFOyzchWIjhqWVTZcMqhCoOw\\\\\\\/7DyEGUqgqJiPAw9TVffT5K6LTPh\\\\\\\/MVrpNVtAgG0TnbEJirTgIVQV6s2vHA\\\\\\\/7CtvFLCxUzDdbWFdlGd\\\\\\\/v\\\\\\\/jBT56xXZLch2ezPglJKbr0TalMmtom3Y+CblDoqCUxGeiB08R49DKLz9EU3iAdcKSmraiNB5LtyT9KMY661ZRlr3FMRJkojUT0QRUzZ4WuElqh5nQ2lh3Re\\\\\\\/6aLDgJYRX\\\\\\\/SO22N1Q8RDiL\\\\\\\/50pACv481tIZjRnFXvvluHTKR02R2AsNmalRD2s9SPb5fem\\\\\\\/uTr0iWm0wD3YcVkmxvR3Koed7BKVCzglS+vgZwl6k\\\\\\\/gKCaBrZjqJ55gFdpiY\\\\\\\/cCfjZzEVtxCAgrexNWZz9jXw3vHDg1nLLcXoCXrjQzqaxN3EhI9GMI9QYgx9E352cml4di0KWb8HBcT9TgtGwEvRjEmRGuCjg22YuwebLWTA\\\\\\\/OZnk3OJSZS+wIfiyqrqfTbuNvpE1\\\\\\\/WILsu5gcGh09l57p4g3vCU\\\\\\\/mKEFOaXMv5FnJXlECUoobTgBZu+HQaOuL8B9eskUxU5uw\\\\\\\/6QI8Lq9bej\\\\\\\/Pftx7R1M1rRZ9MR2vN73LNOiPUT5E5RyNGlfse4XjnC0u6N4SSiLxGTuT\\\\\\\/6ii4lQxaDKZne4SGo+A9gz0ywa9nOdsXRE6owOJqroA3SicLnHVdRx9DHjALcdjOnN5te7\\\\\\\/uz62DTJ8w5y\\\\\\\/dbmuFPGpViuYoFNWKW+bfV90G6jmg7+5NoR5IzB4Tq3dE80zMy6JrzCmWqJdrsJNSgDxX+A+atY0HCeimcHoDCAqIKgC\\\\\\\/FrFNrcVaNWYnEMFPHW6xwRfeq1GPSKG5WKmGm0Ry0a45BrIt\\\\\\\/\\\\\\\/iEpFYMcA\\\\\\\/PnUrPN\\\\\\\/4c5ae9hgYHUqiF0mFm5adAAuQLuhnsnMZogSoiiJ\\\\\\\/l8EXRsBIyJ\\\\\\\/Jw9XrlHIPZZ0uQMaQcGlOJlWhjflM+AT86CtUxGyFUhkQ4UbGQDSjha\\\\\\\/x8G2ed4HihDTJv9Ja6mawzvwa835wZrP+DSZ0MjExDC+SWhog+fqb7ESfgmCSowzP8AV4TeO0rIVX+uDAZ9VvJzrw1d2iTZZa3v8jATpQbbmN1WhFfc+m2r12P2df9wfPE4DHfmCtP3NYKrqPaG9CCOep7wUC+3CEBbiN+gCwSF0uYyo8eN9p56IHdDjKbAQCOorLwLMLcPa1bhwzh8ZFY4zRMUbwyYvLN2n\\\\\\\/LVJ5KuUD8+u+BXSfwRc1f\\\\\\\/cUSl5\\\\\\\/rXCF8GUhQdHiXKvBan5ZH\\\\\\\/WpfhMycWNYH1zuwWLVgRZW7URu5FFnWgV7ic82NGN\\\\\\\/ZX0a83hnuLyBFgIuU5VKcXCezaPfvaWi8v+t8ukz2wb6s\\\\\\\/JwTrW0yP8hpun9nyzrKuM5Wnt+MWMncf2hMuTstbxMORACG0vYV8LWbg6u4iARwMOe3IBjoP1kEz1gVXrlWaXpl+OS0tlxNAojruKp\\\\\\\/tJzHMUdABvI6pytjDHuouJB1A2BQmBWtA5711NDjda1mVLHNI40khKRzJ5qbsiFoDS2V\\\\\\\/5dlI1XIEbSUl6Jilv46gk7kSaTbEi9IjVkBqj6\\\\\\\/rQnKHbrxroS88xTjMmTup11SC2xTIpX\\\\\\\/99UCMYebRuiAJ3UiMCoOj\\\\\\\/6eJyv3VxAZ4ZjFfVGVDpwfNYVwTGgaJg6j6Me+\\\\\\\/gVoBcbEBqTvuZlSi6KrnzEcMyjj0\\\\\\\/UtdcAK4zXiaE\\\\\\\/73RkYHDu1PrOIsx+ycm3Fw9Cp1Yaz1PWJg6Z95qj0dzWWZa1dAZKoSh95RIqexOh70FZ3ywlGGX8eqWIp5OhIcgxt7kuCJOs49uq95cFWawNaf67MiJwwMC3MipuaWQYfFEziZ9AoOBJ46Bct+3QEnCDzuq0H9VemY4RUbsLjunRntF0y5GxAj+JsmALFQb6X5sn7HdeteLGhzYOa27Y+W5y5rhEoCCd0Ek0CfJYJdEGWkJyLjKpD4TglTkY8QCbMLNWdIlBLbh+hiiWIXaIJ9U4Dqrgqk62eJNXJZA+LNzVcLUb1+cfihWWJrOB3f17hsxBKCwoBlhq\\\\\\\/\\\\\\\/sqhLE\\\\\\\/YKJQr69wBJJAJB8bY4NkCDD86fXt29NyAfVaNbC5qJ0SXpG0ZlNnd0ltegYvHD7En1AeYA2EIvLCNDizIzFUz5HXUqL7lwKwwsPm1Mdnwl\\\\\\\/8TBHXidZITvojYksuRgZcZQevUmLJSRfXBEc9xEBhP4n9msYknazHstWRwX4x+eiLBos4gdrt8HX3ti72BiX5aTHeaxyxLTq+PwVGytYMNa1uXxLQ9BzRV+xK\\\\\\\/gYhzkVL5CHb2q1JPGClBOoST3Td0xuvakTCYo59n103lX3I2gTHwxB6raKPRoqwolR3sgLj3QpkilIKLp5z8HVJXcvqMuw9kv2sQRRarR4xwdRKVUatHdJ4hR5FPFIMe\\\\\\\/F2J2fvtnUdnysW1vX4fJLanI7Ses33LdDTwa\\\\\\\/PT+gusP3wUg1T+7qsLIx\\\\\\\/U+AkteMVoNc6uI4nQMiYR6nUDbGfCvMpAZbr\\\\\\\/3YTHy3LyHUjwe5jC\\\\\\\/e\\\\\\\/EhgYqjo4K3g\\\\\\\/ZBjbdOsLneEv9P4WY5gurEoUC4HqbL\\\\\\\/uqHMUnwO3WV8Gmg3AwQcpNZyGviWZ+KkTtU\\\\\\\/ZVTDEdCDPkIO4UCUtGDfxM3IVR9m4UZXemWmfsLc6j8fvnvai9ZSNrVR4jI72yVOuv1\\\\\\\/eUyiK70WlBUL37+N2MnlcSr8vpXi4rFDl+ju6Wo+\\\\\\\/Gt\\\\\\\/ueX9z2Hta7V1mzxu6OqBwf+CMdxoqZV1b3ez\\\\\\\/Erdpqu0JHDlbG7\\\\\\\/HC7zbXSIP6K6nAM1CWIRciG8sGWjqVr3SoHp0xDhlFU2q2h\\\\\\\/8FlQYSdYydX87j3Y5bB2nLOK+E5zK1tBynaAmC9zyX2fZ1gf70HZcabMHBtuuFpioJlvIizqTWHLXB7dUTEcRiNK9s0o0imHp\\\\\\\/Qf4fFHVOPWf6j6QS32BHo+RVX8uFfNJDFKzRzKM504wqxHI2kP6SEbdge+gmfPmc44blcFrzqFZzS5+LtsN\\\\\\\/0qtrUEOR6yohlXHT2cq\\\\\\\/3nUHqEDL4U3km0IxjyWzdfOtGBQYauSWDgjGVNMmsnHxzCpvJDn34g+XDHf2bVHrphHLBx5TiI29fl7pDw1VjjOxlU5OEmwMKZoGK2aqt12bVCqZHSyzfmPWgXrV4zRM6YWfa5N6kX1Ow0jRfegCtXvyNX52Stoe5qQUMkmgWuuObbb1Lfk9m2yxM7mrJ15lK069m7qrGHtyRKyRGZNs0AU1hyqcm7Jd+lb9\\\\\\\/kGit\\\\\\\/Y3bdTkaNWIDwi9HopjwpOMObsOKK3nDqp6GhVp3JmxCfv3Wen4DjNYy1xr72TVPn7WOABBtZYuE0W453UHNzh5As3plfutLDiybDx1R14eXjQyqtgYi1dH2BIFr3MK9zrg2niDO5mGsTKVpZoX\\\\\\\/ihzd8E2uB+XTbRjL6EHLeJhG18usT3ASNSDAa4NDFJW7b+8d5tErBeaCj6nanqffymd4oronAW+BPrlOvJ6a6b7Fl5tp5Jh3Wz+RvsuMcOcW47vd3TJ052S6bhDL1uuERFH\\\\\\\/XS6hYLTNNmkc8q2eNfHNbRH8m4FkCZMhw6a0pW5BIO4X8B6qhhtLLmH+5kw7VQYtvuFkSMFL\\\\\\\/3DrVrI4qV+sRGSx4sCBy11GzNsMVor5IQp3XzT+bbd\\\\\\\/POzFRA\\\\\\\/TvIpHOHxHLw88Sm6fHgXM5jbVsFmosRkVYhdC7\\\\\\\/SkE95VSnZWF9GN5Hy7aQinlxqFj1RkcjAY1r1dnuIGK87Lphbx71ZJCVsoCfaIBjgvjSdos3D+yTiKLpODvYujC5wVhskFEpRsI\\\\\\\/ieps4tNeVCF5Ml1VdfUx0lujTgdKlD8s0KstXw9KHKl29Qui2EaUXGoT8wlt1eaxYuL0ugSlNq4FjBAEC8OVs8w8r\\\\\\\/g8y2yKofmsZhU\\\\\\\/+aiqZE3GKrqgw8fV4Fz5RSnpBLhVCmLw7+7Yxl8Hf2oMIiLdkXvkWA8HAie+F+AUGqpX445T5XBEUD\\\\\\\/by9jkfQC0MPmYvnfwUdBVIs5YBf0eV6GHZOnAHh0Ui0xPpKXRvl89vLWf5\\\\\\\/zwMy4VUb9UAezknRE\\\\\\\/qFpFq+vGVS2eD7dJsTUShdQxEfhPEEtluxIfKA9AwHc3aMaClg04w7hdhO+Q88enQFIXVTRwCKKVUPvxhmbZ8CPpeT7i\\\\\\\/nRq+2uO3pOEJLOn2ku3nd9uQSFbNGUudK05ZOj5o1b2nhtA07anG1w\\\\\\\/IaUwKpx8WHr69Pxvj72BFZ8hHKdP3yTbO\\\\\\\/80SoVYCvwujrz8kYTeY6BtS607fJmQMt\\\\\\\/PI3pyY6XAGVV84MUN3gRcNM9btQsZ0KjLu2nFv1iBsz6etb38QfcrFcd8A4mrfx1Sy3yGEIAHdcEH1LhuGv0M+mCdP8IC4NTSwIFmXA8ptC7SDX4wrZdDzIuLvQz1fLxyBUN9IDDVm16apAxU\\\\\\\/cTjLwjoKwQBqa0do07tY2nx3xiYZXLbUVBgqIkVYtFV5Kf4fcOrSYWdJ5Is7voW+zR\\\\\\\/mUDUMHU5NBrSxYuP9exIK8Qmp0rrXSmLPvzZMLfBYAe245StTuNg2LCRGC44D4asA8D7oYp5ES1LgYYVAbJ38QX+4aEKMAqIQH9ydZyukGVv4WGpRp94RtjDFjvIDolO4cHjcr3aBi5P7xdGitGPVuqVfiy+tB5si6D7ZIv7zKMe5BiTeJAkd\\\\\\\/RzYroLC5ML+JM4LrhoGydTO6rk8id19mgsTfiZPt5dKS6OaIU7NMcEVMGIfV9ufKgRGhwPjh4sqsyoJXXAfMGJS8q8CqWklWpliGufzbI\\\\\\\/QThCpTBHyvCx6MCMPbGf1My8Ru\\\\\\\/Pv6Tz1C5\\\\\\\/b7PSnqpJKiRqpeng4ONCcRMBv7Bw4kq1TxMNOY0KDcvNApwK60Jb\\\\\\\/BfUIJOkLS0GB4lfxG2vd8EDYU1KoTOD9aUTXmxjYVJHzS0Khen1FCAsOQpZJuF8WMNSSDnUcKysCCRLor2S2lIO9nFjQGvJryel\\\\\\\/FbvNteDqXZo+dC5kcBthbYRO5k7piojjycFjUyT0piQG8UXPeOiOWXmkc8T31iZq9gR1tdMc4Fk9\\\\\\\/cON1iW+S1JJBxNWUSO8hZB2VGxOKJ0eURxS1imkeU1df9B4VZ9vVIUNZJ6vc+4y\\\\\\\/J7QlmcaT6zBLTCVaICexW\\\\\\\/b4vv3jhkqbfvStEoz9aMwvTe1A\\\\\\\/xyWXyE7fI91jYQ06MtKwEI24ktriPzat1663GksoHaXk4n8YaWvTTtpLlI4N4QhF+yVVN\\\\\\\/nb7g3YlRS0lxrLjxpoLDfU2grQGRRiBjkcAtQwW542dpkmd3zLITvkivRrLFIhMqsVu1r35iBkydRFrnLHnfZoU539FvbV+5Ba5kAvkixf7nxnI3iGcSxMYoIDvFoo1yEZm8hNvfy+Bg\\\\\\\/lK00DxTd+n4RKJpXYrlsEzc2OUGBI0OyB9dXkad8jcFoUwWiURsxosO666+c5ZGaPRskFMtJ8jbtM+9P07UMGgaZsTWoIXS2F7Y0J02LfWVnXvdhVZmHvp4MiI38HEOSieX9GbHckTIP5REhKcYs7yLEibFA9Pb1+PvKpK3OOmsBnIM9wMD3f85GMpyRE9n4KzvmoDlLabAn6MgtT15nag09r0DrBqsQYQN4EoNMeTgRymjTVZIq2YcxbGML6tW4h4vxFSX7Pddb6Az7QzdzRnIV1sQIotmF7FgZyMTrPKCFRyhg+BQgUtg8MkqU8v7l0jP8p6UItdxZYglPaDkZTG\\\\\\\/vb9CW2RxCOvjsNgMW+dAapuE5KDgr1ZYcIrNUB7XrmaXYdwuZ+\\\\\\\/urwTHxOBZbL1h\\\\\\\/9oh3hSe1r0O8ZbNkufVo9CBIR651XQVIZuIek\\\\\\\/Ya35+dszim8DrjtZpuWmvxh4ozmJB68AQmmbvN8HHnj4ngBcNekvBolay02lpuVZae2bSFq0fEl88hDdf5sX3Z31gFfKUfvKg1OOif5yAj7Yhe2TwgyUS2dkQ0VYJFgvmHA1lf\\\\\\\/97SOoEUEtYu5m31vHiBaB3A0sDqqBRkHgoRSEjWVL+yUIt709\\\\\\\/zaLa+ZTmdBU89a6p976EKS2n9WQXROXyz6U8equIeqFyRuGsOGHQODLrxlMWv3S8wZqRFuu97sXD0nMDGltg9OqH5P+jt19b62sbcs0YC3kCu0mpr\\\\\\\/PzWeMQRiABkSfZUMVdqoyc1UZDciVxoIMP5IElyjn5oVp3J7dSAjqP+WAh\\\\\\\/n\\\\\\\/gLT3tzQjb2VH2nWvrI\\\u0022,\\\u0022iv\\\u0022:\\\u0022WVZI13Glg7dnrYRFPQ2mTQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e6618e10e973df5f0415a9b0703486162f80730afa0dee4fa1f47e665b607fef\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DruYwm4WiXqIL28jyW04XDcA73QxJh+jGCtgFaxxLWKV3+1sr72Lo1XSMeHUuCSNbctGisDqYUkYCLnuzMRuUs2+m7cwgYqJTyqrEjFmXMvH9uDOZwWs0DoDHeDv+lrLmby06bJCdjUz7pyGc83dpkS0IgRpzpcbJiJKUX5ersSCXuR7SxZRKBNfxIeeHE7VlG7JZWQnlUWbJ3I9meYdZ9HFpal78pSoo7lSYae24THMWukqA2Ppd3C6U5m6HGBtvKwByWH\\\\\\\/Ay3a8MGOyPQDaaFRbs\\\\\\\/QBgxTjQi9qD3238wePFFWNjAsLexpoD7exTXHBfZ5WcMavtPn\\\\\\\/aoL9oTN0LLr+Un0Z30hB30yyYXyXJDaIm7ktyi4HDlvnyoJQBe7nGOCpPHczs2XIfBL2EOpagRrgUMW8YteNAHfdLsE+HJbj3xYXaSeI77D9uUAK+phgRQDuhoqQOo+R6U\\\\\\\/9gdp6iAY7VliqU+WCYhEDq4vRmWfKfbBtWYQXM9zWed5RERtzxaVEVDGaKupm2Q07PvhxnIEnv\\\\\\\/3KG6C\\\\\\\/XIrGurzHwjY4V9PHjO7rdtwPVqDuDLZVhi7H7wE2lSwoj2D2oQtsCZA2LCXy9sGnEz0F80T7Vk2rGqJOaK+uNtGr7E5ZMqSOmgjU8j84y6+ENlMBoZ9aUxY\\\\\\\/5Te+ZXiSaiHlLOOEAN9rXT11hRZ5b3nEw2D3mC1aYeZUAUufgnZljRxtJXZSmuyQbQjDCD\\\\\\\/RFEZX3hEGf\\\\\\\/Ex5nI1JJ2awRIn4J13BdKbYN5pSj7olstpglli0d8O8jQ+SN5lJV3RhfniKA56+\\\\\\\/W5pSnhyFmJ+oRC7ySn5pLGK07UTVxvzhhb6A9IY9GszvK\\\\\\\/3RfyvD4tz64ZmEOQ\\\\\\\/\\\\\\\/NDlI1KZwo0hmSfWPKz0PeLoDrVzHXlwxKvKjFmfx5x0+71O9OLbd5jSYjKdyEto3FutcUrCXvPdUORhciN2lz5+4Os4CiXROAGj4yb6uWwo2tewpAsT\\\\\\\/5s03SEtNU6VeWycqbj\\\\\\\/KY0qPRpgydMdtnwQNZvEMJdCZQnbiwGKI6qwgZ1fB4x\\\\\\\/8bZ04BINV49xtj3h1JAtnmpahP3k3tIXWaOvmWVFgyvWXCPTPRmIW+Pr+6rqCgQ9wJIYL5\\\\\\\/zLvRGNJc6zk96BMo3FbqDE5qx\\\\\\\/On291toBbxDEQ01kypQgtkTSI2Be269y6Pd3CWtOWwxWC6z85fNmn3Y\\\\\\\/u6Sy81YPTLQaX+dGivtqQa9N42nW4gNBpWW0oRRHv3P8pepkSkiDLDPPcXtQkOxYMkwlQqK9\\\\\\\/E\\\\\\\/ouIyza2jL3QhV0gQONQHO5Rr3nmyWUvRG2DVv9Wt5p\\\\\\\/eMIxMoD+hByUE\\\\\\\/kZvvYEhIS4V+SHOarCewmolHEFEM3bjBOvYh6IF9T5k++6b+ZztJJA7Bz\\\\\\\/BlVDpTBaSDkP1oaxjgIWOsSN7kGOWnzRUzUXTFE\\\\\\\/gy5Yjja6wSENd3zI1QjxRN6doTdrJpEMZbSYQymmd8RrWMm9wVqB8tm64HCHPGbWtIbCY0VGrg1de0i3JtBxZB+n8N8ozPxIITh0zn+jXXFkglqwxJaxSecouFztZvFbhN3bL5tERRQWq8quIz88b4+rommwBguWWUiXj5AcMTwpF2JszczP8Evd2pMmvrbiSb4PSJweB1qqoUzJV4FyzWyzBgDnee5g\\\\\\\/Rikdz3ygxrgZKRAv7iTQnii7wdVpt9IVL1HhEJ6A2qQZcbXx1bWVqYy9r\\\\\\\/9qFItMqzXD1ExoqJnT\\\\\\\/qkpJYRkdndQzXyLikWrn2eJNfxmLudBdlidRuxqkK59C6MMmTXVWBpdPsoHwjNpcjqHzxFTpo8+ssyRhIoURQ7Gy3BLHejTBmfDPT2blef+5qqHmWWJRzwmHEUgowW+so8zz3N\\\\\\\/WFtjViDotrOODc5TME2nR73zaduV+ZnSSsu29+sH2rOOqORU+4FY2gaP5jt9Dj4\\\\\\\/8UnYuMX9Iaqy0pJXYepBBzmsTZlr4xnn5QffHMoSAa39guPnIxfOQ37qeP4Y6RyxL7iU2m01MMuQrvfKiLKLFWv1o=\\\u0022,\\\u0022iv\\\u0022:\\\u0022gqFryXWj86ZVOIn6Hq+vlw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224716c683b7e518d883d4fff8cdebe14f2872289b3b95fc04d31064bdb151c3b4\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484758 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1899" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-71_Vkxq5Ep2Sruny2kY4ptUrlRP-gXYAMIpGzSdGE2E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1899" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-V6E09cvgrg7qYCor4jOkW79sPwdOlKJedjNnuuWt8XQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1887" title="Predictive Genomic Analyses Inform the Basis for Vitamin Metabolism and Provisioning in Bacteria-Arthropod Endosymbioses" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1913" title="Genetic Architecture of a Rice Nested Association Mapping Population" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484758" id="node484758" data-pisa="ggg;7/6/1899" data-pisa-master="ggg;g3.117.041749" data-apath="/ggg/7/6/1899.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Adaptive Roles of <em>SSY1</em> and <em>SIR3</em> During Cycles of Growth and Starvation in <em>Saccharomyces cerevisiae</em> Populations Enriched for Quiescent or Nonquiescent Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0040-1838" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dominika M. Wloch-Salamon</span>, <span class="highwire-citation-author" data-delta="1">Katarzyna Tomala</span>, <span class="highwire-citation-author" data-delta="2">Dimitra Aggeli</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7041-0035" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Barbara Dunn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1899-1911;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041749 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484758" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dominika M. Wloch-Salamon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Environmental Sciences, Jagiellonian University, Krakow 30-387, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominika%2BM.%2BWloch-Salamon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wloch-Salamon%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADominika%2BM.%2BWloch-Salamon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0040-1838" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dominika M. Wloch-Salamon</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dominika.wloch-salamon@uj.edu.pl" class="" data-icon-position="" data-hide-link-title="0">dominika.wloch-salamon@uj.edu.pl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katarzyna Tomala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Environmental Sciences, Jagiellonian University, Krakow 30-387, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katarzyna%2BTomala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tomala%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatarzyna%2BTomala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dimitra Aggeli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Stanford University, California 94305-5120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dimitra%2BAggeli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aggeli%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADimitra%2BAggeli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Barbara Dunn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Stanford University, California 94305-5120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barbara%2BDunn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunn%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABarbara%2BDunn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7041-0035" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Barbara Dunn</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/6/1899" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484758" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484758/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1899.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484758" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484758/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1899.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484758" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484758/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1899.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484758" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484758/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/6/1899.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484758" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484758/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Over its evolutionary history, <em>Saccharomyces cerevisiae</em> has evolved to be well-adapted to fluctuating nutrient availability. In the presence of sufficient nutrients, yeast cells continue to proliferate, but upon starvation haploid yeast cells enter stationary phase and differentiate into nonquiescent (NQ) and quiescent (Q) cells. Q cells survive stress better than NQ cells and show greater viability when nutrient-rich conditions are restored. To investigate the genes that may be involved in the differentiation of Q and NQ cells, we serially propagated yeast populations that were enriched for either only Q or only NQ cell types over many repeated growth–starvation cycles. After 30 cycles (equivalent to 300 generations), each enriched population produced a higher proportion of the enriched cell type compared to the starting population, suggestive of adaptive change. We also observed differences in each population’s fitness suggesting possible tradeoffs: clones from NQ lines were better adapted to logarithmic growth, while clones from Q lines were better adapted to starvation. Whole-genome sequencing of clones from Q- and NQ-enriched lines revealed mutations in genes involved in the stress response and survival in limiting nutrients (<em><a href="http://www.yeastgenome.org/locus/S000000197/overview">ECM21</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000927/overview">RSP5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005476/overview">MSN1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002635/overview">SIR4</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005441/overview">IRA2</a></em>) in both Q and NQ lines, but also differences between the two lines: NQ line clones had recurrent independent mutations affecting the <a href="http://www.yeastgenome.org/locus/S000002567/overview">Ssy1p</a>-<a href="http://www.yeastgenome.org/locus/S000001925/overview">Ptr3p</a>-<a href="http://www.yeastgenome.org/locus/S000003692/overview">Ssy5p</a> (SPS) amino acid sensing pathway, while Q line clones had recurrent, independent mutations in <em><a href="http://www.yeastgenome.org/locus/S000004434/overview">SIR3</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001665/overview">FAS1</a></em>. Our results suggest that both sets of enriched-cell type lines responded to common, as well as distinct, selective pressures.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/ssy1" class="hw-term hw-article-keyword hw-article-keyword-ssy1" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">SSY1</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/sir3" class="hw-term hw-article-keyword hw-article-keyword-sir3" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">SIR3</em></a></li><li class="kwd"><a href="/keyword/sps-pathway" class="hw-term hw-article-keyword hw-article-keyword-sps-pathway" rel="nofollow">SPS pathway</a></li><li class="kwd"><a href="/keyword/quiescence" class="hw-term hw-article-keyword hw-article-keyword-quiescence" rel="nofollow">quiescence</a></li><li class="kwd"><a href="/keyword/evolution" class="hw-term hw-article-keyword hw-article-keyword-evolution" rel="nofollow">evolution</a></li></ul><p id="p-3">Organisms live in changing environments, and thus evolutionary adaptation to varying conditions is crucial for an organism’s long-term survival. Some environmental stressors may be rare while others may occur more frequently, even on a regular basis, such as circadian cycles. Such changes include fluctuation in temperature or nutrient levels (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Tagkopoulos <em>et al.</em> 2008</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Van der Linden <em>et al.</em> 2010</a>), and seasonality is an ubiquitous driver of fluctuating selection in organisms with generation times of a month or less (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Messer <em>et al.</em> 2016</a>). A few mechanisms have been identified that can help organisms to prepare for recurring stressors (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dhar <em>et al.</em> 2013</a>), for example, bet hedging, which results in expression of different sets of genes (or different levels of gene expression) within different subgroups of cells in the population, thereby generating phenotypic heterogeneity within an otherwise isogenic population. This strategy has been shown to increase the long-term fitness of yeast grown with variable application of either heat shock, “diauxic lag” phase duration, or utilization of different carbon sources (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Levy <em>et al.</em> 2012</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">New <em>et al.</em> 2014</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Wang <em>et al.</em> 2015</a>). Another mechanism is adaptive anticipation, where an organism uses the information of the present environment to preadapt in the anticipation of the forthcoming changes. Physiological adaptive anticipation in single-cell organisms (including yeast) is well-documented and is becoming a new paradigm for microbiology (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Mitchell <em>et al.</em> 2009</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">2015</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brunke and Hube 2014</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Siegal 2015</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Yona <em>et al.</em> 2015</a>).</p><p id="p-4">In response to starvation for one or more nutrients, a fraction of the cells in a stationary yeast population exit the mitotic cycle and become Q, a state physiologically similar to that seen in higher eukaryotic G0 cells (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gray <em>et al.</em> 2004</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Valcourt <em>et al.</em> 2012</a>). The organization of a Q cell’s internal structures and genome is very different from that of a proliferating, NQ cell’s; there is an increase in storage carbohydrates and stress protectants such as glycogen and trehalose, increased width of the cell wall (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Aragon <em>et al.</em> 2008</a>), sequestration of proteins (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Suresh <em>et al.</em> 2015</a>), telomere clustering (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Guidi <em>et al.</em> 2015</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rutledge <em>et al.</em> 2015</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Laporte <em>et al.</em> 2016</a>), and global transcriptional shutoff (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">McKnight <em>et al.</em> 2015</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Young <em>et al.</em> 2017</a>). These changes are programmed, energy dependent, and are considered physiologically adaptive during stress (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Smets <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Klosinska <em>et al.</em> 2011</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">De Virgilio 2012</a>). Most of the transcriptional changes associated with transition to Q state are simply correlated with slower growth during starvation, and as such are not specific for the Q state (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Valcourt <em>et al.</em> 2012</a>). However, a set of Q-specific core genes, showing altered transcription rates during starvation that are independent of any growth rate-associated patterns of expression, has been previously identified. For example, increased transcription was detected for genes involved in membrane lipid biosynthesis, protein modification, response to toxins, and metal ion transport, while decreased transcription was found for genes related to cytokinesis, chromosome organization and biogenesis, and organization of the nuclear pore complex (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Klosinska <em>et al.</em> 2011</a>). Still, the most significant property of yeast Q cells, relative to proliferating NQ cells, is their ability to maintain viability over long periods of time during the growth-arrested phase, and to resume mitotic growth once growth-promoting conditions are restored (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Gray <em>et al.</em> 2004</a>). Transition to Q cells may be triggered by early signals of nutrient depletion, so that the population contains individuals in different physiological states with associated transcriptional profiles. Production of a stable fraction of Q cells is characteristic for yeast strains. A population-level bet hedging strategy could result in subgroups of Q cells and NQ cells, which may increase the chances of that population’s survival in variable environmental conditions (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Wloch-Salamon <em>et al.</em> 2013</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Wloch-Salamon 2014</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Honigberg 2016</a>).</p><p id="p-5">The extracellular concentration of nutrients available for free-living fungi can change by many orders of magnitude during the growth of a population over time, and even during the lifespan of a single cell. Micro-organisms cope with such changes in part by controlling the rates of uptake of these molecules, by tuning their ability to sense them, and by transcriptional induction of relevant permease genes. In <em>Saccharomyces cerevisiae</em>, distinct mechanisms have evolved to sense the extracellular presence of carbon sources (such as glucose) and nitrogen sources (including amino acids) (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Ozcan <em>et al.</em> 1996</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Didion <em>et al.</em> 1998</a>). <em>S. cerevisiae</em> appears to have adapted to a wide range of nutrient availability by maintaining at least 15 different amino acid permease genes and at least 17 sugar transporter genes (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gaber <em>et al.</em> 2003</a>). An extreme adaptation to scarcity of nutrients is manifested in diploids by the production of spores, and in haploids by the production of Q cells.</p><p id="p-6">In this study, we set out to identify genetic mechanisms involved in the regulation of Q cell formation. To this end, we repeatedly subjected populations of haploid <em>S. cerevisiae</em> cells to two sequential environmental conditions: (1) feast, under which adapted cells are expected to divide as quickly and efficiently as possible, and (2) famine, where some cells in the population transition to nondividing Q cells, which are able to survive long periods of starvation and thus are crucial to the long-term survival of the population. After each cycle, we enriched our experimental lines for either Q or NQ cells, exploiting the fact that Q cells are more dense than NQ cells (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Allen <em>et al.</em> 2004</a>), and repeated the process for 30 such cycles, which we determined was equivalent to ∼300 generations (cell doublings). For one set of lines, only Q cells were repeatedly propagated, while for the other set of lines, only NQ cells were propagated. The NQ line populations are expected to be enriched in cells that do better in the nutrient-rich condition, as they ignore environmental signals of nutrient depletion and do not enter the Q program. By contrast, the Q line populations are expected to be enriched in cells that do better under famine, detecting and/or anticipating upcoming nutrient scarcity, since repeated harvesting of only Q cells should reward cells that enter the Q cell program. For the Q-enriched population, we presume that fitness advantages during logarithmic growth become less important overall, because we assume that only Q cells make it to the next cycle. Because the populations were grown and then starved on the same rich medium (YPD plates) the change in environment was not acute, but instead was gradual. This may allow cells, when reaching early stationary phase, to naturally anticipate signals of the consecutive changes, and respond by differentiating into Q or NQ cells. After the 30 successive cycles of cell type enrichment, growth, starvation, and separation, we isolated several single colonies from the final populations for whole-genome sequencing. These 30-cycle clones (C30) were also used to investigate possible trade-offs in their fitness under conditions differing from those in which they were selectively enriched.</p><p id="p-7">During the course of the experiment, we observed that populations produced a significantly increased fraction of the cell type (Q or NQ) for which they were enriched, which was also true for the C30 clones. Analysis of whole-genome sequences of the C30 clones allowed us to identify potentially Q- or NQ-promoting mutations. Interestingly, NQ cell enrichment in our conditions may be enhanced by disabling the SPS signaling pathway, as <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em> was frequently mutated in NQ lines but not in Q lines. In Q lines, we found repeated <em><a href="http://www.yeastgenome.org/locus/S000004434/overview">SIR3</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000001665/overview">FAS1</a></em> mutations. We also found genes independently mutated in both lines, including <em><a href="http://www.yeastgenome.org/locus/S000000197/overview">ECM21</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000927/overview">RSP5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005476/overview">MSN1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002635/overview">SIR4</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005441/overview">IRA2</a></em>. Our results reveal distinct patterns of molecular adaptation for increased production of either Q or NQ cells.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains, media, and cycles of growth and starvation (G-S)</h3><p id="p-8">We used a derivative of the laboratory haploid strain s288C (Mat α, <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>::KanMX4, <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">his3</a></em>Δ1 <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">leu2</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">lys2</a></em>Δ) [modified according to <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cubillos <em>et al.</em> (2009</a>)]. A single colony from a YPD plate was grown to stationary phase in 10 ml liquid YPD at 30° for 2 d (density of ∼2 × 10<sup>8</sup> cell/ml). The culture was diluted 1:100 and three 100 μl aliquots were used to start replicate populations for the experiment by plating them as patches on 10 ml YPD small Petri plates (50 mm diameter), such that the initial density of the entire plate was ∼2 × 10<sup>5</sup>/ml. After 4 d of incubation at 28°, the entire patch was washed from the plate using 10 ml of sterile water, 4 ml of which was then fractionated (see below) in a Percoll gradient, to give two fractions: a more dense, lower fraction (Q) and a less dense, upper fraction (NQ). Each of the six resulting fractions (an NQ and a Q from each of the three replicates) was then used to found experimental lines NQ I–III, and Q I–III, respectively (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). We used calibration curves, based on combined OD reads and Burker chamber cell counting for the NQ and Q lines, to plate 1 × 10<sup>6</sup> cells onto a 10 ml YPD plate (for a final media density of 1 × 10<sup>5</sup>/ml), which was incubated for 3.5 d at 28°, at which point the cells had reached stationary phase (density ∼2 × 10<sup>8</sup> cells/ml). The fractionation, followed by the YPD plating, was then repeated for 30 cycles. For all of the NQ experimental lines, we used only cells from the upper fractions (NQ cells) for each subsequent cycle, and likewise for the Q lines we selected only the lower fractions after each centrifugation (Q cells). Starting at cycle 10, we sampled aliquots of the populations from each fraction every five cycles, and stored them as frozen glycerol stocks at −80°. The total quantity of the Q and NQ cells in the populations was estimated by optical density.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1899/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Experimental design and (B) procedure shown for the separation of NQ and Q line populations. (A) We started the experiment with one clone, which was grown to stationary phase. This population was then diluted and plated onto three plates. The initial density of each population on the plate was &#x223C;2 &#xD7; 105/ml. Plates were incubated for 3.5 d and all the cells were harvested. Each plate population was separately fractionated in Percoll solution. From each of the three plates we obtained two distinct fractions of cells that were founders of the six experimental lines. NQ, lines of NQ cell selection, Q, lines of Q cell selection. (B) Continuation of experiment shown via the example of one NQ and one Q line. We progressed the experiment by using only NQ or only Q cells from fractionated cultures, diluted them, and plated at an initial density of &#x223C;2 &#xD7; 105/ml. Plates were incubated for 3.5 d, then cells were harvested, and fractionated. For the NQ lines we propagated only NQ cells, for Q lines only Q cells. We carried out the experiment for 30 cycles of growth and starvation. From the final fractionation, we isolated 15&#x2013;16 clones from each experimental population for further analysis. NQ, nonquiescent; o/e, overnight; Q, quiescent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2019998984" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Experimental design and (B) procedure shown for the separation of NQ and Q line populations. (A) We started the experiment with one clone, which was grown to stationary phase. This population was then diluted and plated onto three plates. The initial density of each population on the plate was &#x223C;2 &#xD7; 10&lt;sup&gt;5&lt;/sup&gt;/ml. Plates were incubated for 3.5 d and all the cells were harvested. Each plate population was separately fractionated in Percoll solution. From each of the three plates we obtained two distinct fractions of cells that were founders of the six experimental lines. NQ, lines of NQ cell selection, Q, lines of Q cell selection. (B) Continuation of experiment shown via the example of one NQ and one Q line. We progressed the experiment by using only NQ or only Q cells from fractionated cultures, diluted them, and plated at an initial density of &#x223C;2 &#xD7; 10&lt;sup&gt;5&lt;/sup&gt;/ml. Plates were incubated for 3.5 d, then cells were harvested, and fractionated. For the NQ lines we propagated only NQ cells, for Q lines only Q cells. We carried out the experiment for 30 cycles of growth and starvation. From the final fractionation, we isolated 15&#x2013;16 clones from each experimental population for further analysis. NQ, nonquiescent; o/e, overnight; Q, quiescent.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1899/F1.medium.gif" width="319" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/6/1899/F1.medium.gif" width="319" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1899/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1899/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484954" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">(A) Experimental design and (B) procedure shown for the separation of NQ and Q line populations. (A) We started the experiment with one clone, which was grown to stationary phase. This population was then diluted and plated onto three plates. The initial density of each population on the plate was ∼2 × 10<sup>5</sup>/ml. Plates were incubated for 3.5 d and all the cells were harvested. Each plate population was separately fractionated in Percoll solution. From each of the three plates we obtained two distinct fractions of cells that were founders of the six experimental lines. NQ, lines of NQ cell selection, Q, lines of Q cell selection. (B) Continuation of experiment shown via the example of one NQ and one Q line. We progressed the experiment by using only NQ or only Q cells from fractionated cultures, diluted them, and plated at an initial density of ∼2 × 10<sup>5</sup>/ml. Plates were incubated for 3.5 d, then cells were harvested, and fractionated. For the NQ lines we propagated only NQ cells, for Q lines only Q cells. We carried out the experiment for 30 cycles of growth and starvation. From the final fractionation, we isolated 15–16 clones from each experimental population for further analysis. NQ, nonquiescent; o/e, overnight; Q, quiescent.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Fractionation and isolation of Q and NQ cells</h3><p id="p-10">We used a previously described method (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allen <em>et al.</em> 2006</a>) to obtain the gradient for density fractionation. In brief, we mixed Percoll and 1.5 M NaCl, 9:1 v/v, and 7.5 ml of this solution was centrifuged in a swinging bucket rotor at 10,078 × <em>g</em> for 20 min. Then 4 ml of the culture washed from the plates (in 50 mM Tris, pH 7.5) were placed on top of the gradient and were centrifuged for 60 min at 417 × <em>g</em>. We carefully removed the upper (NQ) and lower (Q) cell fractions into separate tubes using pipettes. Based on OD, ∼10<sup>6</sup> cells were transferred to a YPD plate of 10 ml, for an initial density of 10<sup>5</sup> cells/ml (for the entire volume of the small Petri plate). In addition, at the beginning of the experiment, at the 10th and then every 5th cycle (that is 15th, 20th, 25th, and 30th), the quantity of each of the Q and NQ fractions in all experimental lines was assessed. All fractions were collected to separate tubes and then serially diluted. Based on specific density curves (separate for Q and NQ fraction) and spectrophotometric OD measurement, the population density was estimated.</p></div><div id="sec-4" class="subsection"><h3>Refractionation</h3><p id="p-11">During the experiment we observed an increase in the quantity of all selected fractions in each of the experimental populations. That is, the NQ fractions from the NQ lines became thicker as did the Q fractions from the Q lines (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). To determine if this phenotypic effect was heritable and still present in the populations derived from the evolved clones, we checked the quantity of NQ and Q fractions of the ancestral strain and 11 evolved clones. Latin numbers refer to the experiment (QI 1, 13; QII 5, 6; QIII 8, 9; NQI 3, 11; NQII 15; and NQIII 6, 7). Each of these clones and the ancestral clones were subjected to a single cycle of G-S prior to fractionation, similarly to as in the main experiment. It included growth on YPD plate for 3.5 d and single fractionation in a Percoll gradient. The number of cells in each of the isolated fractions was estimated based on OD measurements. This procedure was repeated three independent times for each of the clones.</p></div><div id="sec-5" class="subsection"><h3>Phenotypic changes of evolved clones: maximum growth rate (MGR)</h3><p id="p-12">MGR of each C30 clone was estimated in liquid YPD medium culture. A frozen sample was transferred by a metal pin to fresh liquid YPD medium dispensed into 500 μl aliquots in 96-deep well plates and incubated without shaking for 3 d at 30°. Then, 2 μl of this stationary phase culture were transferred to 100 μl of fresh YPD liquid medium in a 96-well plate, which was then sealed. Plates were then placed into a Tecan instrument, incubated at 30° with agitation at 1250 rpm, and wide-band optical density (OD) readouts were acquired every 10 min with the Infinite m200 Microplate Reader (Tecan). The OD readouts were corrected for background effects, log-transformed, and used to determine the range of exponential growth. This was performed by finding Q and NQ limits of OD, between which the Pearson coefficient <em>r</em> was highest. Regressions were based on no less than 10 time points and had an <em>r</em> of 0.99. MGR (1/hr) was then estimated from the slope of the linear regression of the log OD <em>vs.</em> time. This experiment was repeated in triplicate for each C30 clone. One-way ANOVA and <em>post hoc</em> Tukey test were used for statistical analysis.</p></div><div id="sec-6" class="subsection"><h3>Phenotypic changes of evolved clones: relative growth (RG)</h3><p id="p-13">Initially, reference and query strains were mixed in 1:1 proportions. The conditions used were exactly the same as for the cycles of G-S that were used during the experiment, except that instead of fractionation on a Percoll gradient we kept the populations in Percoll solution for 2 hr with no centrifugation or fractionation (the same amount of time that was required for centrifugation and sample collection during the experimental conditions), and then washed the cells using standard Tris buffer (as during the experimental procedure). The initial mixed populations were diluted to a density of 2 × 10<sup>5</sup>/ml to: (1) calculate their initial proportions and (2) start the actual G-S cycle. We applied two cycles of G-S, and then counted the fraction of cells of each competing strain by counting the number of fluorescent and nonfluorescent colonies on a YPD plate. Fitness as the RG rate on plates in competition with ancestral marked with YFP was measured assuming transitivity of fitness interactions. The YFP marker was amplified from genomic DNA from the BY4741-YFP.natR strain and integrated into the ancestral strain according to previously described protocol (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">DeLuna <em>et al.</em> 2008</a>). The reference ancestral YFP-marked strain, 16 evolved experimental clones from NQ and Q lines (QI 1, 13; QII 5, 6, 12; QIII 8, 9, 11; NQI 11, 14; NQII 3, 15; and NQIII 6, 8, 11) and the unmarked, original ancestral strain, were separately grown overnight to stationary phase in liquid cultures. The reference strain was mixed with each of the query strains isostoichiometrically. Mixed cultures were diluted and ∼10<sup>5</sup> cells were used to inoculate a plate (10 ml YPD) for the first round of G-S. After 3.5 d, the resulting large patches were harvested and incubated for 2 hr in Percoll solution. Then, ∼10<sup>5</sup> cells were used to inoculate another plate (10 ml YPD) for the second round of G-S. After each round of G-S, the frequency of competing strains was estimated by plating dilutions to count CFU (using a transilluminator to distinguish colonies that were florescent). The fitness of each of the clones relative to the reference strain was computed as the ratio of the Malthusian parameters of both strains (Lenski 1991). At least three replicate competition experiments per strain were performed. Ancestral clones grew 6% (average RG = 1.06) faster than the YFP-marked derivative. We corrected for this difference by dividing growth of each evolved query clone by this number. General Linear Models: Nested ANOVA model was used: repetitions nested in clones that were nested in line. <em>Post hoc</em> Tukey test was used for statistical analysis.</p></div><div id="sec-7" class="subsection"><h3>Phenotypic changes of evolved clones: regrowth after starvation</h3><p id="p-14">Clones derived from the final experimental cycle (C30) (Q I 1, 13; Q II 5, 6; Q III 8; NQ I 11; NQ II 3, 15; and NQ III 6, 11) were transferred from the frozen culture to fresh YPD agar medium using a metal pin. The colonies were incubated for 3 d. Then, a small quantity of a colony was transferred to liquid culture for overnight growth on a rotating wheel at 30°. The next day, five drops of 50 μl were put onto 25 ml fresh YPD plates such that the grown patches did not touch each other; this was done in triplicate. After 2, 4, 8, 15, and 22 d, the population density and viability were estimated. Each time, one patch from the plate was washed out using water. Cell density was estimated based on Coulter Counter results. Then we serially diluted the cultures and plated them on a fresh YPD plate. After 3 d of incubation at 30°, we counted CFU and estimated the fraction of the cells able to undertake mitotic divisions and grow to form a colony on the plate. Viability is expressed as the cell fraction of cells that were able to undertake mitotic division and form a colony in the fresh YPD plate.</p></div><div id="sec-8" class="subsection"><h3>Plate-washing assay</h3><p id="p-15">The plate-washing assay was performed as described (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cullen and Sprague 2000</a>). Ten microliters of the stored population derived from each of the experimental and ancestral clones were put on a YPD plate. Invasion was allowed to proceed for 3 d at 30°. Plates were washed with water to remove cells that did not invade the agar. Care was taken not to break the agar surface. Photos were taken right before and immediately after performing the washing assay. Colonies for which we observed visible intrusion into the agar surface were marked X when only part of the area of the original colony was visible, and XX when the whole area of the colony was visible (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/FigureS1.pdf">Figure S1</a>).</p></div><div id="sec-9" class="subsection"><h3>Whole-genome sequencing of evolved clones</h3><p id="p-16">We struck out the experimental populations from the three NQ (NQ I, NQ II, and NQ III) and three Q (Q I, Q II, and Q III) lines from the final cycle (C30) to single cells on YPD plates. We isolated two ancestral clones and a total of 47 clones from the NQ lines (16, 15, and 16 from NQ I, NQ II, and NQ III, respectively) and 47 clones from the Q lines (16, 16, and 15 from Q I, Q II, and Q III, respectively). Clones were grown separately in fresh liquid YPD in a 96-deep well plate. After 2 d of growth at 30°, we used the YeaStar Genomic DNA Kit (Zymoresearch) according to the provided protocol for isolation of total genomic DNA from each clone. We measured DNA concentrations using a Qubit fluorometer and the Quant-iT dsDNA HS kit (Invitrogen) (according to provided protocols), and prepared Nextera-based genomic libraries as previously described (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kryazhimskiy <em>et al.</em> 2014</a>). These 96 (94 evolved and two ancestral clones) DNA libraries were mixed together and sequenced on a single flow-cell lane on an Illumina HiSequation 2000 to generate paired-end, 100 bp reads.</p></div><div id="sec-10" class="subsection"><h3>Sequencing data analysis: counting SNV in whole-genome sequences</h3><p id="p-17">We used two independent methods to analyze whole-genome sequences. For the first method, obtained sequences were mapped to the reference genome (R64-1-1; RefSeq assembly accession: GCF 000146045.2) using bowtie2 software (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Langmead and Salzberg 2012</a>) with option very sensitive. Obtained SAM files were transformed to BAM, sorted, and indexed using samtools (for more information go to <a href="http://www.htslib.org/doc/samtools-0.1.19.html">http://www.htslib.org/doc/samtools-0.1.19.html</a>). Optical duplicates were deleted using samtools rmdup. Samtools mpileup was used to look for the SNV (with the following settings: -u, and –q20). The resulting file was transformed to VCF format (using bcftools view). Variant filtering was done with a custom Python script. Variants with the following setting were qualified for further analysis: not present in the ancestral (t02) sequence, QUAL parameter (in VCF file) was ≥30, coverage was &gt;9, present in ≥85% of all good quality reads. Coverage and percent of variant confirming reads were extracted from the DP4 field of the VCF file. For annotation, we used software annotate-0.1 (<a href="http://depts.washington.edu/sfields/software/annotate/">http://depts.washington.edu/sfields/software/annotate/</a>). For the second method, genomic analysis was done using CLC-Genomic Workbench 7.5 software. Unmapped postfilter, paired-end reads as FASTQ files provided by the Stanford Sequencing Centre were downloaded directly to the CLC GW software. We mapped two of the ancestral clones (t01 and t02) and 94 evolved, C30 clones to the previously downloaded reference sequence: <em>S. cerevisiae</em>. <em>EF4</em>. <em>68</em>. <em>dna</em>. <em>toplevel</em> (<em>Genome</em>). No masking was applied to the reference genome. Mapping was random with the following parameters set: mismatch cost 2, insertion and deletion costs 3, length of fraction 0.5, and similarity of the fraction 0.8. The mapped sequences of clone t01 were filtered against known variation in the mapped sequences of the t02 clone, which was treated as the reference. We only considered variation supported by &gt;85% of the mapping reads. We mapped whole-genome sequences of the 94 evolved clones to the reference genome. Then, we checked basic variant detection using the following parameters: ignore broken pairs, ploidy: 2; minimum coverage: 5; minimum count: 2; minimum frequency: 50%. Detected single nucleotide polymorphisms (SNPs) were filtered against previously found variations in clone t02. Variations found in each of the clones (ignoring broken pairs and nonspecific matches) were checked for amino acid change in the annotated genes. Each of the SNPs was confirmed by visual inspection in ∼50% of the cases of the aligned reads, and then annotated as to whether they were in coding or noncoding regions.</p></div><div id="sec-11" class="subsection"><h3>Construction of the mutation phylogenetic tree</h3><p id="p-18">A fasta file was generated for all clones, and each clone was represented by a sequence, made out of all the variant positions present across all clones. The file also had an entry for the reference sequence. The fasta sequences were aligned using the MUSCLE package (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Edgar 2004</a>). The aligned file was then converted to a format that can be read by programs in the PHYLIP software distributed by the author (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Felsenstein 2005</a>). The tree was generated by the programs dnapars, retree, and drawgram from the PHYLIP software and imported in illustrator CS6.</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-19">Strains and isolated clones are available upon request. Genome sequences have been deposited in SRA (NCBI) as BioProject: PRJNA383491, with accession number SRP104610. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/FileS1.docx">File S1</a> contains descriptions of all Supplemental files. The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Experimental design</h3><p id="p-20">Experiments were performed using a derivative of the <em>S. cerevisiae</em> S288C laboratory strain (Matα, <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>::<em>KanMX4</em>, <em><a href="http://www.yeastgenome.org/locus/S000005728/overview">his3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">leu2</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">lys2</a></em>Δ) (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Cubillos <em>et al.</em> 2009</a>). A single colony of this strain was grown in liquid YPD media to stationary phase and the resulting population was separated into Q and NQ cells by density centrifugation on a Percoll gradient, with the upper (less dense) fraction consisting of NQ cells and the lower (more dense) fraction consisting of Q cells (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>) (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Allen <em>et al.</em> 2006</a>). We then serially propagated cells from only the Q cell layer, or from only the NQ layer, through 30 G-S cycles. Each cycle started by plating 100 µl of the culture, containing ∼10<sup>6</sup> cells, as a “patch” onto a small Petri plate containing 10 ml of YPD agar medium (2% glucose), yielding an initial population density of 10<sup>5</sup>/ml for the entire plate of medium. After 3.5 d of growth at 28°, after the point where the cells had already reached stationary phase and were experiencing nutrient deprivation (density of ∼2 × 10<sup>8</sup>/ml for the whole plate) (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Allen <em>et al.</em> 2006</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Davidson <em>et al.</em> 2011</a>), Q and NQ cells were separated by centrifugation, and the relevant fraction (Q or NQ) was diluted and ∼10<sup>6</sup> cells plated again to give 10<sup>5</sup> cells/ml (for the 10 ml YPD agar plate) to start the next cycle. In three of the experimental lines (named NQ I, NQ II, and NQ III) only NQ cells were serially propagated, while in the other three lines (Q I, Q II, and Q III) only the Q cells were serially propagated (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). Each of the six experimental lines were subjected to 30 of these 3.5-d G-S cycles (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>). After the 30th cycle (∼120 d), 15–16 clones were isolated from each of the experimental lines (a total of 47 from NQ lines and 47 from Q lines) and were whole-genome sequenced to identify mutations that might have been selected by the one selection regime or the other. These clones (all or only chosen ones) were in addition checked for phenotypic traits: MGR, survival during G-S cycles, survival in starvation, and invasive growth.</p></div><div id="sec-15" class="subsection"><h3>Increase of the enriched cell type proportion in the experimental populations</h3><p id="p-21">To determine whether serial enrichment of a particular cell fraction resulted in a stably increased production of such cells within the population, we measured the relative proportions of each cell type (on a population basis for each of the enriched lines) throughout the experiment. We observed significant increases in the relative proportion of each serially enriched cell type (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a>). The ancestral population, prior to the serial cell type enrichment cycles, had a NQ cell proportion typical of the S288C strain of 25% ± 1% and a corresponding Q cell proportion of 75% ± 1%. After 30 G-S cycles with serial enrichment for NQ cells, the proportion of NQ cells (taking the average of all three NQ lines) increased to 82% ± 2%, and the proportion of Q cells correspondingly decreased to 18% (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). The percentage of NQ cells for the individual NQ I, NQ II, and NQ III lines, after 30 cycles, was 87, 79, and 80%, respectively. Likewise, after 30 G-S cycles, but with serial enrichment for Q cells, the percentage of Q cells (average of all Q lines) increased to 87% ± 5%, while NQ cells correspondingly decreased to 13% (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). The 30-cycle Q percentages of the individual Q I, Q II, and Q III lines were 91, 76, and 95%, respectively. The Q II line, which acquired a mutation in <em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> and a mutator phenotype (explained below), showed no significant change in cell type proportions during the 30-cycle experiment.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1899/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average proportions of the quiescent (Q) cells (red bars) and nonquiescent (NQ) cells (blue bars) in the populations at the selected experimental cycles of growth and starvation in: (A) NQ lines (average of I&#x2013;III) where only NQ cells were selected for following cycles of growth and starvation, and (B) Q lines (average I&#x2013;III) where only Q cells were selected for the following cycle. (C) Average proportions of the quiescent (Q) cells (red bars) and nonquiescent (NQ) cells (blue bars) after one cycle of growth and starvation in the populations derived from the clones isolated after 30 cycles, at the end of the experiment, and in the ancestral strain (Figure 1B). " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2019998984" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average proportions of the quiescent (Q) cells (red bars) and nonquiescent (NQ) cells (blue bars) in the populations at the selected experimental cycles of growth and starvation in: (A) NQ lines (average of I&#x2013;III) where only NQ cells were selected for following cycles of growth and starvation, and (B) Q lines (average I&#x2013;III) where only Q cells were selected for the following cycle. (C) Average proportions of the quiescent (Q) cells (red bars) and nonquiescent (NQ) cells (blue bars) after one cycle of growth and starvation in the populations derived from the clones isolated after 30 cycles, at the end of the experiment, and in the ancestral strain (Figure 1B). &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1899/F2.medium.gif" width="440" height="226"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/6/1899/F2.medium.gif" width="440" height="226"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1899/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1899/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484851" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Average proportions of the quiescent (Q) cells (red bars) and nonquiescent (NQ) cells (blue bars) in the populations at the selected experimental cycles of growth and starvation in: (A) NQ lines (average of I–III) where only NQ cells were selected for following cycles of growth and starvation, and (B) Q lines (average I–III) where only Q cells were selected for the following cycle. (C) Average proportions of the quiescent (Q) cells (red bars) and nonquiescent (NQ) cells (blue bars) after one cycle of growth and starvation in the populations derived from the clones isolated after 30 cycles, at the end of the experiment, and in the ancestral strain (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>). </p><div class="sb-div caption-clear"></div></div></div><p id="p-23">To determine whether the changes in cell type proportions displayed by the enriched populations were also reflected by individual cells taken from the populations, we chose 11 single colonies from the six final (30th cycle) populations, and one colony from the ancestral preenrichment parent strain, and measured the fractions of each cell type that they generate under the experimental conditions. We performed one cycle of growth, starvation, and fractionation, identical to the experimental serial G-S enrichment cycles (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1B</a>) in triplicate for each of the chosen clones. The average proportion of NQ cells for all NQ lines was 84% ± 4%; the average for each line was: NQ I 88.9% ± 2.2%; NQ II 82.9% ± 2.2%; and NQ III 89.0% ± 8.9%. By contrast, the average proportion of Q cells in all Q lines was 96% ± 6%, and the average for each line was: Q I 98.59% ±7.9%; Q II 89.54% ± 6.6%; and Q III 99.61% ± 4% (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>).</p></div><div id="sec-16" class="subsection"><h3>Adaptations to a specific environment results in trade-offs in different environments</h3><p id="p-24">We hypothesized that there may be fitness trade-offs due to adaptation to the specific conditions (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kassen 2002</a>; Bono <em>et al.</em> 2016) that were used in our experiment, similar to previous findings (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wenger <em>et al.</em> 2011</a>). To test this hypothesis, we determined the fitness of the ancestral clone and clones from the NQ and Q experimental lines in three different conditions: (1) Growth in YPD medium, with the presence of sufficient glucose (2%), <em>i.e.</em>, a condition to which clones producing increased numbers of constantly dividing NQ cells may be better adapted; (2) a fluctuating environment, consisting of two alternating G-S cycles, without separation and selection of cells (in this changing environment, the ancestral, “generalist” cells might be better adapted than either of the enriched, evolved “specialist” populations from Q or NQ lines); and (3) viability after long-term starvation and regrowth ability, a condition under which clones producing increased numbers of Q cells might be better adapted.</p><p id="p-25">Condition (1): We used MGR, measuring each evolved clones’ individual growth rate during logarithmic growth, as our assay. Division time (1/hr) measured in liquid YPD culture is an approximation for the early stage of each experimental G-S cycle, when cells are actively growing on solid YPD medium (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1B</a>). Under these conditions, we observed some differences between the lines (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>): one-way ANOVA showed differences in MGR between lines (<em>F</em> = 17.61, d.f. = 6, <em>P</em> &lt; 0.0001), yet there was no significant difference between the NQ (I–III) and the Q (I–III) lines. Instead, a significant difference was observed between the ancestral and the evolved clones (<em>post hoc</em> Tukey test). All evolved populations statistically increased their MGR, with NQ lines having the highest MGR (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). The MGR of the average of all (15–16) experimental clones from each experimental line (with ancestral MGR set as (1) was as follows: NQ cells: MGR<sub>NQ I</sub> = 0.89 ± 0.05; MGR<sub>NQ II</sub> = 0.90 ± 0.05; and MGR<sub>NQ III</sub> = 0.89 ± 0.06; and Q cells: MGR<sub>Q I</sub> = 0.79 ± 0.08; MGR<sub>Q II</sub> = 0.80 ± 0.1; and MGR<sub>Q III</sub> = 0.75 ± 0.06.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1899/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Test of trade-offs in fitness of evolved clones in three different environmental conditions. Each clone was measured at least three times. Error bars represent 95% C.I. (A) average maximum growth rate (MGR) of the ancestral clone and all clones from the final populations of each experimental line; NQ I: 16 clones, NQ II: 15 clones; NQ III: 16 clones; Q I: 16 clones; Q II: 13 clones; and Q III: 15 clones. (B) Fitness of the 16 selected clones from ancestral and experimental lines NQ I&#x2013;III (NQI 11 and 14; NQII 3 and 15; NQIII 6, 8, 11, and 12; QI 1 and 13; QII 5, 6, and 12; QIII 8, 9, and 11) relative to yellow fluorescent protein (YFP)-marked ancestral strain. The result of the competition of ancestral and YFP-marked ancestral clone is standardized to one. (C) Colony-forming capacity of starved cultures determined by plating assay. Values are expressed as the percentage of colony forming units (CFU) to the total number of cells in the population. Gray bar represents ancestral population, blue and red stand for NQ and Q, respectively. NQ, nonquiescent; Q, quiescent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2019998984" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Test of trade-offs in fitness of evolved clones in three different environmental conditions. Each clone was measured at least three times. Error bars represent 95% C.I. (A) average maximum growth rate (MGR) of the ancestral clone and all clones from the final populations of each experimental line; NQ I: 16 clones, NQ II: 15 clones; NQ III: 16 clones; Q I: 16 clones; Q II: 13 clones; and Q III: 15 clones. (B) Fitness of the 16 selected clones from ancestral and experimental lines NQ I&#x2013;III (NQI 11 and 14; NQII 3 and 15; NQIII 6, 8, 11, and 12; QI 1 and 13; QII 5, 6, and 12; QIII 8, 9, and 11) relative to yellow fluorescent protein (YFP)-marked ancestral strain. The result of the competition of ancestral and YFP-marked ancestral clone is standardized to one. (C) Colony-forming capacity of starved cultures determined by plating assay. Values are expressed as the percentage of colony forming units (CFU) to the total number of cells in the population. Gray bar represents ancestral population, blue and red stand for NQ and Q, respectively. NQ, nonquiescent; Q, quiescent.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1899/F3.medium.gif" width="440" height="350"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/6/1899/F3.medium.gif" width="440" height="350"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1899/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1899/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484913" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Test of trade-offs in fitness of evolved clones in three different environmental conditions. Each clone was measured at least three times. Error bars represent 95% C.I. (A) average maximum growth rate (MGR) of the ancestral clone and all clones from the final populations of each experimental line; NQ I: 16 clones, NQ II: 15 clones; NQ III: 16 clones; Q I: 16 clones; Q II: 13 clones; and Q III: 15 clones. (B) Fitness of the 16 selected clones from ancestral and experimental lines NQ I–III (NQI 11 and 14; NQII 3 and 15; NQIII 6, 8, 11, and 12; QI 1 and 13; QII 5, 6, and 12; QIII 8, 9, and 11) relative to yellow fluorescent protein (YFP)-marked ancestral strain. The result of the competition of ancestral and YFP-marked ancestral clone is standardized to one. (C) Colony-forming capacity of starved cultures determined by plating assay. Values are expressed as the percentage of colony forming units (CFU) to the total number of cells in the population. Gray bar represents ancestral population, blue and red stand for NQ and Q, respectively. NQ, nonquiescent; Q, quiescent.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Condition (2): To assess the evolved clones’ adaptation to a changing environment, we measured the RG rate of 16 selected evolved experimental clones from NQ and Q lines and ancestral clones (“query” strains) in competition with a YFP-tagged version of the ancestral strain (the “reference” strain) on plates during two G-S cycles (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). Nested ANOVA showed significant difference between lines (ancestral, NQ, and Q): <em>F</em><sub>lines</sub> = 9.57; d.f. = 6; <em>P</em> = 0.003; however, there was no difference between the clones within each line and within their replicates: <em>F</em><sub>clones</sub> = 1.25, d.f. = 12, <em>P</em> = 0.39, <em>F</em><sub>repetition</sub> = 0.89, d.f. = 34, <em>P</em> = 0.63). A <em>post hoc</em> Tukey test showed a significant difference in RG between the ancestral line (RG<sub>ancestral</sub> = 1.00 ± SD = 0.01) and all NQ and Q II–III lines; however, a <em>post hoc</em> Tukey test showed that average RG from clones from the Q I line (RG<sub>Q I</sub> = 1.02 ± 0.02) did not differ from the ancestral clone’s RG rate (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>). Though not significantly different, the average RG rate of Q clones showed higher fitness by 2% compared to the average of all the NQ line clones (RG<sub>allNQlines</sub> = 0.99 ± 0.03 and RG<sub>allQlines</sub> = 1.01 ± 0.02, respectively) (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>).</p><p id="p-28">Condition (3): We examined the clonal population’s ability to regrow after starvation on YPD plates. We used two clones from the ancestral population, t01, and t02 (derived from the same single colony population that was used to start the whole experiment), and 10 clones from the experimental lines. From each clone, we derived populations that were checked for ability to regrow after 2, 8, 15, and 22 d of starvation on YPD plates. Analysis of GLM nested ANOVA showed significant differences between days (not nested) and lines (ancestral, Q, and NQ), but no difference between clones within lines (<em>F</em><sub>days</sub> = 166.210, d.f. = 3, <em>P</em> &lt; 0.001; <em>F</em><sub>lines</sub> = 16.22, d.f. = 6, <em>P</em> &lt; 0.001; and <em>F</em><sub>clones</sub> = 0.479, d.f. = 4, <em>P</em> = 0.75). Differences in viability, measured as the percentage of the entire population that was still able to reenter mitosis, divide, and produce a colony, became statistically significantly different among lines after 8 d of starvation with the Q lines showing a significant advantage. In this condition, clones derived from Q lines were fitter than either the ancestral or NQ line-derived clones (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a>). The difference became more pronounced with increasing time of starvation, as measured on days 15 and 22.</p></div><div id="sec-17" class="subsection"><h3>Strains may have acquired adaptive mutations during the experiment</h3><p id="p-29">To determine whether selection for Q or NQ cells resulted in the enrichment of particular mutations that might predispose cells to form those particular cell types, we whole-genome sequenced t01, t02, and 94 evolved clones (15–16 from each of the three Q and three NQ lines). In total, across all the sequenced genomes of all clones, we found 138 (including 13 frameshifts) nonunique events of nonsynonymous SNV within gene coding regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/TableS1.xlsx">Table S1</a>). The phylogenetic relationships among clones are shown in the (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/FigureS1.pdf">Figure S1</a>). The presence of multiple mutations shared across different experimental lines is an unexpected result. We can hypothesize that: (1) it is possible that some of the shared mutations arose early in the experiment, before splitting to independent lines, and as such were spread within the experimental lines; (2) that some of these mutations were present in the initial culture because although we used what we believed was a single colony, there may have been more than one colony that was picked (if that was the case, then low frequency mutations would not have been detected when we sequenced just two clones from the ancestral population); and (3) there could have been unintended mis-labeling during the experiments [if so, we believe it is most probable (although we cannot be sure) that there was not any switching between Q and NQ lines]. Overall, we feel that because our choice of the presumably adaptive mutations is based on the fact that multiple mutations in a gene are present in several experimental clones and not that the clones belong to any specific line (except what is shared between NQ or Q line), that if mis-labeling occurred, it did not influence the choice of which mutations were adaptive.</p><p id="p-30">One of the identified frameshift mutations was in the <em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> gene (Thr44fs, middle branch in the <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/FigureS1.pdf">Figure S1</a>). The <em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> gene is involved in error-free postreplicative repair (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Broomfield <em>et al.</em> 1998</a>) and, when inactivated, confers a mutator phenotype, significantly increasing the spontaneous mutation rate (note that <em><a href="http://www.yeastgenome.org/locus/S000003055/overview">mms2</a></em> mutants do not display any characteristic mutational spectrum) (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Xiao <em>et al.</em> 1999</a>). As expected, the 22 clones that carry this particular <em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> mutation have acquired substantially more SNV than the rest of the clones, with an average of 15.7 mutations per clone as compared to only 4.9 mutations per clone for the remaining 72 clones. These 22 clones are excluded from the following analysis and discussion; however, their associated datasets are included in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/TableS1.xlsx">Table S1</a>. After excluding clones with the <em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> mutation, we identified 50 SNV in 40 genes (nuclear genome only), including one dubious ORF, and one gene of unknown function (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/TableS1.xlsx">Table S1</a>) from a total of 72 sequenced clones. In 32 of these genes, only a single mutation was observed, with these mutations being present in 1–16 clones. Because of the experimental set-up where we transferred a fraction of the populations to the next G-S cycle, we do expect that clones isolated from each line at the end of the experiment (C30) would share some of the SNV that arose during the experiment. That is indeed the case, and SNV was often shared among clones with common ancestry from the same experimental line (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/TableS1.xlsx">Table S1</a>).</p><p id="p-31">To identify mutations that may have been selected by the experimental conditions, we focused on genes in which we observed multiple different mutations, either specific to only Q or NQ lines, or found among both Q and NQ lines. In NQ lines only, we observed multiple mutations in <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em>, while in Q lines only, we observed multiple mutations in <em><a href="http://www.yeastgenome.org/locus/S000004434/overview">SIR3</a></em>. We also observed genes that were independently mutated multiple times in both Q and NQ lines (<em><a href="http://www.yeastgenome.org/locus/S000005476/overview">MSN1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000197/overview">ECM21</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002635/overview">SIR4</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000927/overview">RSP5</a></em>). We also focused on single mutations that were observed in many Q clones but no NQ clones (<em><a href="http://www.yeastgenome.org/locus/S000001665/overview">FAS1</a></em>) or in all NQ clones and only three Q clones (<em><a href="http://www.yeastgenome.org/locus/S000005441/overview">IRA2</a></em>). We interpret the mutations that arose in this group of eight genes as possibly adaptive to the growth and enrichment environments we performed (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1899/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Presumably adaptive mutations identified in the whole-genome sequenced, nonmutator clones isolated from experimental Q and NQ lines on the 30th experimental cycle. A complete list of all identified mutations is in Table S1. NQ, nonquiescent; Q, quiescent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2019998984" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Presumably adaptive mutations identified in the whole-genome sequenced, nonmutator clones isolated from experimental Q and NQ lines on the 30th experimental cycle. A complete list of all identified mutations is in Table S1. NQ, nonquiescent; Q, quiescent.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1899/F4.medium.gif" width="440" height="333"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/6/1899/F4.medium.gif" width="440" height="333"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1899/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1899/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484989" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-32" class="first-child">Presumably adaptive mutations identified in the whole-genome sequenced, nonmutator clones isolated from experimental Q and NQ lines on the 30th experimental cycle. A complete list of all identified mutations is in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/TableS1.xlsx">Table S1</a>. NQ, nonquiescent; Q, quiescent.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Adaptive mutations shared between NQ and Q lines</h3><p id="p-33">There are four genes in which we identified widespread multiple mutations across both the NQ and Q experimental lines (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/TableS1.xlsx">Table S1</a>). These are <em><a href="http://www.yeastgenome.org/locus/S000005476/overview">MSN1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000197/overview">ECM21</a></em>, <a href="http://www.yeastgenome.org/locus/S000000927/overview"><em>RSP5</em></a> and <em><a href="http://www.yeastgenome.org/locus/S000002635/overview">SIR4</a></em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Two different missense mutations in <em><a href="http://www.yeastgenome.org/locus/S000005476/overview">MSN1</a></em>, both affecting the same codon, are present among all non-<em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> clones: Lys86Thr is found in QI and QIII clones, and Lys86Glu is found in clones from all six lines (Q I–III and NQ I–III). These results suggest that these mutations arose in the beginning of the experiment before splitting of the lines. <em><a href="http://www.yeastgenome.org/locus/S000005476/overview">MSN1</a></em> encodes a transcriptional activator of invertase and glucoamylase expression. <em><a href="http://www.yeastgenome.org/locus/S000005476/overview">MSN1</a></em>-null mutants show either a decrease or complete absence of invasive growth, typical for haploids growing on solid agar in response to nitrogen and carbon limitation (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gagiano <em>et al.</em> 1999</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Cullen and Sprague 2000</a>). Because our experimental protocol involves a step where we collect cells for the next cycle by washing the cells off an agar plate, we performed a plate-washing test (see <em>Materials and Methods</em>) to determine the invasive growth phenotype for all 94 sequenced clones. Despite having a wild-type <em><a href="http://www.yeastgenome.org/locus/S000005476/overview">MSN1</a></em> gene, the ancestor clones did not show an invasive growth phenotype; this was expected as the S288C strain background does not show invasive growth. The invasive growth phenotype was also absent from all 36 Q clones, even though they all had one of the two <em><a href="http://www.yeastgenome.org/locus/S000005476/overview">MSN1</a></em> missense mutations, suggesting that these mutations do not increase invasive growth. However, we found that 33 NQ clones (out of a total of 36, all of which have the <em><a href="http://www.yeastgenome.org/locus/S000005476/overview">MSN1</a></em> Lys86Glu mutation) do show an invasive growth phenotype (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/TableS1.xlsx">Table S1</a>). Thus, apparently, the <em><a href="http://www.yeastgenome.org/locus/S000005476/overview">MSN1</a></em> mutations are not responsible for any changes in the invasive growth phenotype, and instead, the presence of other NQ line-specific mutations such as the SPS pathway genes (see <em>Discussion</em> below) may be conferring the invasive growth phenotype to the NQ clones. Interestingly, the three NQ clones that lack an invasive growth phenotype are the only clones (out of all sequenced clones) that contain a presumed inactivating frameshift mutation in the <em><a href="http://www.yeastgenome.org/locus/S000003013/overview">RIM8</a></em> gene; null mutations of <em><a href="http://www.yeastgenome.org/locus/S000003013/overview">RIM8</a></em> are known to decrease or abolish invasive growth (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Madhani and Fink 1998</a>), and thus this mutation may be responsible for the lack of invasive growth in these NQ clones. Overall, invasive growth among NQ cells suggests an expected phenotypic trait for cells in search of nutrients, as opposed to cells more prone to enter quiescence.</p><p id="p-34">We also observed multiple mutations in <a href="http://www.yeastgenome.org/locus/S000002635/overview"><em>SIR4</em></a> and <a href="http://www.yeastgenome.org/locus/S000000197/overview"><em>ECM21</em></a> across both Q and NQ cell lines. We observed five mutations (Ser57*, Asn89fs, Ser154*, Gln595fs, and Pro1124fs) in the <em><a href="http://www.yeastgenome.org/locus/S000002635/overview">SIR4</a></em> gene, which regulates <em>de novo</em> assembly of heterochromatin by scaffolding, nucleation, and anchoring (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kueng <em>et al.</em> 2012</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">2013</a>). Mutated sites are widespread along the entirety of this long gene (1358 aa). The first three mutations, all presumably inactivating since they are nonsense or frameshift, occur within the first domain (aa 1–270), which is responsible for <a href="http://www.yeastgenome.org/locus/S000004712/overview">yKu80</a> and DNA binding. The Gly595fs frameshift mutation leaves the above <a href="http://www.yeastgenome.org/locus/S000004712/overview">yKu80</a>/DNA binding domain intact as well as a second <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1</a> binding domain (aa 142–591) intact, although it is not known if a partial protein would be stable. The frameshift mutation Pro1124fs resides in the last domain, which is responsible for <a href="http://www.yeastgenome.org/locus/S000002635/overview">Sir4p</a> dimerization and <a href="http://www.yeastgenome.org/locus/S000004434/overview">Sir3p</a> and <a href="http://www.yeastgenome.org/locus/S000004897/overview">YKu70</a> binding (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Kueng <em>et al.</em> 2012</a>, <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gartenberg and Smith 2016</a>). Found among all lines, and shared by 67 clones, were two mutations in the <em><a href="http://www.yeastgenome.org/locus/S000000197/overview">ECM21</a></em> gene. Both mutations introduce premature stop (positions Arg106* and Glu282*) before the PY-motif, which is necessary for interaction with <a href="http://www.yeastgenome.org/locus/S000000927/overview">Rsp5p</a> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lin <em>et al.</em> 2008</a>).</p></div><div id="sec-19" class="subsection"><h3>IRA2 mutation found in all NQ clones</h3><p id="p-35">An <em><a href="http://www.yeastgenome.org/locus/S000005441/overview">IRA2</a></em> stop mutation, Gln1365*, is present in 39 clones that were derived mainly from NQ cell lines (the mutation is present in all 36 NQ clones, but in only three Q clones). It is possible that this mutation also arose at the beginning of the experiment, before the culture was split between the NQ and Q selection lines. <a href="http://www.yeastgenome.org/locus/S000000344/overview">Ira1</a> and <a href="http://www.yeastgenome.org/locus/S000005441/overview">Ira2</a> are Ras GTPase-activating proteins (GAP) that act as negative regulators of the Ras-cAMP signaling pathway by increasing the rate Ras proteins (encoded by <em><a href="http://www.yeastgenome.org/locus/S000005627/overview">RAS1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000005042/overview">RAS2</a></em>) hydrolyze GTP to GDP (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Tanaka <em>et al.</em> 1990</a>). Such mutations in the Ras/cAMP signaling pathway have been previously identified as adaptive under glucose-limited conditions (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lang <em>et al.</em> 2013</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chiotti <em>et al.</em> 2014</a>). These mutations result in elevated glucose signaling, resulting in the cell’s misperception that glucose is present even when it is not (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Smets <em>et al.</em> 2010</a>).</p></div><div id="sec-20" class="subsection"><h3>The amino acid signaling pathway (SPS) and endocytosis may play a role in the enrichment of NQ cell frequency</h3><p id="p-36">There are 22 mutated genes among the 36 non-<em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> NQ clones (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/TableS1.xlsx">Table S1</a>). YeastMine GO analysis of this gene group reveals enrichment for the biological process of <em>response to amino acid</em> (GOID 0043200, Bonferroni corrected <em>P</em>-value = 0.05) based on the presence of three genes in the group, <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001925/overview">PTR3</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003692/overview">SSY5</a></em>, which together constitute the SPS amino acid sensing pathway (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Didion <em>et al.</em> 1998</a>). Another enriched process is <em>ubiquitin-dependent endocytosis</em> (GOID 0070086, Bonferroni corrected <em>P</em>-value = 0.04) based on the genes <em><a href="http://www.yeastgenome.org/locus/S000000197/overview">ECM21</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000927/overview">RSP5</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003013/overview">RIM8</a></em>. The most significantly enriched cellular components are the <em>plasma membrane part</em> (GOID 0044459, Bonferroni corrected <em>P</em>-value = 0.006), based on genes <em><a href="http://www.yeastgenome.org/locus/S000000500/overview">PHO89</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000927/overview">RSP5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001925/overview">PTR3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003013/overview">RIM8</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003692/overview">SSY5</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005441/overview">IRA2</a></em>; the <em>plasma membrane extrinsic component</em> (GOID 0019897, Bonferroni corrected <em>P</em>-value = 0.03) due to genes <em><a href="http://www.yeastgenome.org/locus/S000000927/overview">RSP5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001925/overview">PTR3</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003692/overview">SSY5</a></em>; and the <em>plasma cytoplasmic side</em> (GOID 0009898, Bonferroni corrected <em>P</em>-value = 0.04) due to genes <em><a href="http://www.yeastgenome.org/locus/S000000927/overview">RSP5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003013/overview">RIM8</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005441/overview">IRA2</a></em>.</p><p id="p-37">We identified multiple mutations in the <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em> gene, all of which occurred only in clones derived from NQ lines (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/TableS1.xlsx">Table S1</a>). The <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em> gene encodes a nontransporting nutrient sensor, and acts as the first step in the SPS pathway (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Didion <em>et al.</em> 1998</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Klasson <em>et al.</em> 1999</a>) that senses extracellular amino acids (the following steps involve the <em><a href="http://www.yeastgenome.org/locus/S000001925/overview">PTR3</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003692/overview">SSY5</a></em> gene products, see above). All four of our observed <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em> mutations, Tyr747*, *853Tyr, Glu526fs, and Gly492Arg, presumably leave an intact cytoplasmic domain (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Conrad <em>et al.</em> 2014</a>). The other members of the SPS amino acid signaling pathway are mutated only in the NQ clones (<em><a href="http://www.yeastgenome.org/locus/S000001925/overview">PTR3</a></em>: D263fs and <em><a href="http://www.yeastgenome.org/locus/S000003692/overview">SSY5</a></em>: E155fs). Quite interestingly, we observed mutations related to SPS signaling and membrane receptor catabolism across all Q-enriched lines. In particular, point mutations in <em><a href="http://www.yeastgenome.org/locus/S000004725/overview">ASI1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000006005/overview">BRO1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001696/overview">DOA1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000927/overview">RSP5</a></em> arise exclusively in Q lines (apart from a single instance in NQ lines, involving <em><a href="http://www.yeastgenome.org/locus/S000000927/overview">RSP5</a></em>). It was shown that mutations affecting these genes restore amino acid uptake in SPS sensor-deficient strains (<em><a href="http://www.yeastgenome.org/locus/S000002567/overview">ssy1</a> <a href="http://www.yeastgenome.org/locus/S000000523/overview">leu2</a></em>). This allows for growth of cells that are otherwise unable to respond to extracellular amino acid concentrations, by compromising degradation pathways responsible for membrane receptor turnover (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Forsberg <em>et al.</em> 2001</a>). The expectation then would be that these mutants, which arose in clones from the Q lines, should result in upregulated degradation pathways. This mechanism is also suggested by studies linking degradation pathways to chronological lifespan in yeasts and other systems (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Alvers <em>et al.</em> 2009</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Takeda <em>et al.</em> 2010</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kruegel <em>et al.</em> 2011</a>).</p></div><div id="sec-21" class="subsection"><h3>Chromatin silencing role in populations with an elevated Q cell fraction</h3><p id="p-38">We found 20 genes mutated in the 36 non-<em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> clones from Q lines (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/TableS1.xlsx">Table S1</a>). Yeast Mine GO analysis of this gene group showed enrichment in the cellular components of <em>transcriptional repressor complex</em> (GOID 0090568; 0017053, Bonferroni corrected <em>P</em>-value, respectively, = 0.001 and 0.003), due to the presence of genes <em><a href="http://www.yeastgenome.org/locus/S000004434/overview">SIR3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002635/overview">SIR4</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000961/overview">BUR6</a></em> in the group, and enrichment in <em>cytoplasmic side of membrane</em> (GOID 0009898; Bonferroni corrected <em>P</em>-value = 0.02, genes: <em><a href="http://www.yeastgenome.org/locus/S000000927/overview">RSP5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001575/overview">BUD2</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005441/overview">IRA2</a></em>).</p><p id="p-39">We identified two mutations in the <em><a href="http://www.yeastgenome.org/locus/S000004434/overview">SIR3</a></em> gene present in a total of 16 Q clones (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/TableS1.xlsx">Table S1</a>). Mutation E317* is present in clones from all three Q lines, suggesting that it might have been preexisting before splitting into experimental lines, or that it might be the result of unintentional mislabeling of clones (as explained before). An additional mutation in the <em><a href="http://www.yeastgenome.org/locus/S000004434/overview">SIR3</a></em> gene is present within the QII line clones that carry the <em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> mutation. Both <em><a href="http://www.yeastgenome.org/locus/S000004434/overview">SIR3</a></em> mutations (E317* and I246fs) leave the first <em><a href="http://www.yeastgenome.org/locus/S000004434/overview">SIR3</a></em> domain, BAH (bromo-adjacent homology, aa 1–214), intact, which is crucial for chromatin repression. All 15 of the clones with <em><a href="http://www.yeastgenome.org/locus/S000004434/overview">SIR3</a></em> E317* mutations have a preexisting <em><a href="http://www.yeastgenome.org/locus/S000001665/overview">FAS1</a></em> missense mutation (G588A) that may have a synergistic effect. It has been suggested that the <a href="http://www.yeastgenome.org/locus/S000001665/overview">Fas1</a> “housekeeping” gene product (a subunit of fatty acid synthetase), in addition to its catalytic function, is required for the biosynthetic control of the yeast FAS complex (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Wenz <em>et al.</em> 2001</a>).</p></div><div id="sec-22" class="subsection"><h3>Mutations in SIR2 and other possible adaptive mutations among the clones with MMS2 mutation (mutator lineage)</h3><p id="p-40">We cannot entirely exclude the possibility of additional adaptive mutations in the clones with the <em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> mutation (mutator clones). The presence of additional, including multiple, SNVs in the genes that we classified as possibly adaptive—such as <em><a href="http://www.yeastgenome.org/locus/S000005476/overview">MSN1</a></em> (Ser200Phe and Arg186Trp) found in Q and NQ mutator clones, <em><a href="http://www.yeastgenome.org/locus/S000004434/overview">SIR3</a></em> (Val653Phe) found only in Q mutator clones, and <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em> (*853Tyr) present only in NQ mutator clones—supports this hypothesis. There are two genes that acquired multiple SNVs, at least one of which is present within the mutator clones: <em><a href="http://www.yeastgenome.org/locus/S000002200/overview">SIR2</a></em> (Leu332Pro, Cys469*, and Ser420*) and <em><a href="http://www.yeastgenome.org/locus/S000001201/overview">KEL1</a></em> (Arg237Thr and Gln814*). All of the <a href="http://www.yeastgenome.org/locus/S000002200/overview"><em>SIR2</em></a> mutations are spread within NQ clones. This may reflect the unique role of <a href="http://www.yeastgenome.org/locus/S000002200/overview">Sir2p</a> among sir complex genes in the asymmetric partitioning of oxidation-damaged proteins between mother and daughter cells (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguilaniu <em>et al.</em> 2003</a>). Furthermore, <a href="http://www.yeastgenome.org/locus/S000002200/overview">Sir2p</a> is responsible for rDNA silencing (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Smith <em>et al.</em> 1998</a>), an activity that supports entry to quiescence. Mutations in <em><a href="http://www.yeastgenome.org/locus/S000002200/overview">SIR2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em> cooccur as in the cases of C469* in <a href="http://www.yeastgenome.org/locus/S000002200/overview"><em>SIR2</em></a>/G492R in <a href="http://www.yeastgenome.org/locus/S000002567/overview"><em>SSY1</em></a> and L332P in <a href="http://www.yeastgenome.org/locus/S000002200/overview"><em>SIR2</em></a>/*853Y in <a href="http://www.yeastgenome.org/locus/S000002567/overview"><em>SSY1</em></a> (the latter pair of mutations appears within the mutator line). This might suggest an epistatic relationship between these genes that confers fitness advantage among NQ cells.</p><p id="p-41">Finally, there are five genes with multiple SNVs present only within mutator clones; <em><a href="http://www.yeastgenome.org/locus/S000000337/overview">HSL7</a></em> (Val198Glu and Thr347fs), <em><a href="http://www.yeastgenome.org/locus/S000001326/overview">EFM4</a></em> (Ser96Tyr and Ser21Pro), <em><a href="http://www.yeastgenome.org/locus/S000005314/overview">SSK2</a></em> (Tyr852Ser and Leu853*), <em><a href="http://www.yeastgenome.org/locus/S000005569/overview">WHI2</a></em> (Gly141* and Leu76fs), and <em><a href="http://www.yeastgenome.org/locus/S000006005/overview">BRO1</a></em> (Asp228Tyr and Leu216Ile).</p><p id="p-42">Two mutations, G141* and Leu76fs, that deactivate the <em><a href="http://www.yeastgenome.org/locus/S000005569/overview">WHI2</a></em> gene are present within 26 mutator clones. The mutations leave intact the first out of five protein domains, similar to the mutations found previously in yeast evolving under nutrient-scarce conditions (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Wenger <em>et al.</em> 2011</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lang <em>et al.</em> 2013</a>). <a href="http://www.yeastgenome.org/locus/S000005569/overview">Whi2p</a> is required for full activation of the general stress response and regulates growth during the diauxic shift. Null mutants have increased competitive fitness in YPD media. We also found two mutations in the <em><a href="http://www.yeastgenome.org/locus/S000005314/overview">SSK2</a></em> gene (L853* and Y852S) in the mutator strains. <a href="http://www.yeastgenome.org/locus/S000005314/overview">Ssk2</a> is a kinase (MAPK) of the <a href="http://www.yeastgenome.org/locus/S000004103/overview">Hog1</a> mitogen activated pathway, activated in yeast by high osmolarity stress (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hersen <em>et al.</em> 2008</a>). Although it is difficult to presume the adaptive nature of these mutations in strains that contain many other mutations simultaneously, the presence of these mutations (the mutations in the same genes as seen in non-<em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> clones, as well as the <em><a href="http://www.yeastgenome.org/locus/S000005569/overview">WHI2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000005314/overview">SSK2</a></em> mutations seen only in <em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> clones) is suggestive of an adaptive nature.</p></div></div><div class="section discussion" id="sec-23"><h2 class="">Discussion</h2><p id="p-43">In this study, we sought to elucidate genetic mechanisms involved in the regulation of Q cell formation. Haploid yeast cells were sequentially subjected to two opposite environmental conditions of (1) feast, when cells should divide as quickly and efficiently as possible, and (2) famine, when transition to quiescence is crucial for survival. After successive enrichment for only one type of cells (either NQ or Q) that were used to found subsequent generations, we observed changes that indicated cell survival strategies differing from the ancestor. The enriched cell lines showed persistent and significant differences from the ancestor in proportions of Q and NQ cells, and also showed fitness trade-offs when grown in different environments. We found that the ancestral clone, a laboratory strain that presumably has been long adapted to a changing environment due to cyclic growth (serial transfer) on rich YPD-type medium, has higher fitness than either the NQ- or Q-enriched cell line clones in an assay monitoring growth through two feast–famine cycles; the ancestor thus retained its ability to both grow fast when nutrients are abundant, but also to be able to physiologically adapt to periods of starvation while nutrients are scarce. By contrast, clones from the NQ lines, which we found to be much more highly enriched in NQ cells than the ancestral strain, grow faster than either the ancestor or Q line cells in conditions when logarithmic growth rate is supported and there is no need to produce Q cells. Finally, clones from the Q experimental lines, which produce a slightly (but significantly) higher percentage of Q cells than the ancestor, outcompete the ancestral as well as NQ line clones in viability assays after prolonged starvation.</p><p id="p-44">There are no easy-to-use markers for Q and NQ cells developed so far (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Čáp <em>et al.</em> 2009</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Rutledge <em>et al.</em> 2015</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Laporte <em>et al.</em> 2016</a>); however, separation of Q and NQ cells from the population can be achieved based on differences between the densities of NQ and Q cells (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Allen <em>et al.</em> 2006</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Davidson <em>et al.</em> 2011</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Klosinska <em>et al.</em> 2011</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Čáp <em>et al.</em> 2012</a>), which is the method we used for our serial enrichment strategy. However, it is possible that to some extent we also selected or enriched for cells of various density or size characteristics not necessarily linked to their Q or NQ status. Although none of the genes we identified as possibly adaptive (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>) were listed in the yeast GO annotation as <em>cell size control</em> regulator (GO:0008361), we cannot unequivocally exclude such a possibility, since the effects of gene mutations on cell size and density are not usually assessed, which also takes into account the fact that most clones acquired several mutations during the experiment.</p><div id="sec-24" class="subsection"><h3>Increase of the NQ cell fraction may be a consequence of a dysregulated SPS signaling pathway</h3><p id="p-45">An increase in the production of NQ cells in the NQ lines was already visible after 10 cycles of G-S and did not change much after further cycles (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a>). The most striking result from the whole-genome sequencing of our C30 serially enriched clones was the fact that multiple mutations in the <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em> gene (Tyr747*, Glu526fs, and Gly492Arg) were found in 32 of the 36 NQ clones (note that this does not include the <em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> mutator NQ clones that also have mutations in the <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em> gene, see Supplemental Material). The <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em> gene encodes a nontransporting nutrient sensor (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Didion <em>et al.</em> 1998</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Klasson <em>et al.</em> 1999</a>), acting as the first step in the SPS pathway, during which the extracellular amino acid concentration results in conformational changes of the <a href="http://www.yeastgenome.org/locus/S000002567/overview">Ssy1p</a> cytoplasmic domain, which signals to downstream targets (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Conrad <em>et al.</em> 2014</a>). In the presence of extracellular amino acids, the primary amino acid sensor <a href="http://www.yeastgenome.org/locus/S000002567/overview">Ssy1p</a> is stabilized in a conformation that triggers intracellular signaling (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ljungdahl and Daignan-Fornier 2012</a>). It has been previously observed that mutation of <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em> codon 382 from threonine to lysine (or arginine) results in constitutive signaling, showing that mutant <a href="http://www.yeastgenome.org/locus/S000002567/overview">Ssy1p</a> can initiate signaling in the absence of extracellular signaling (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Gaber <em>et al.</em> 2003</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Poulsen <em>et al.</em> 2005</a>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">2008</a>). All the mutations we observed (Gly492Arg, Glu526fs, and Tyr747*, plus *853Tyr in the mutator clones) presumably leave an intact cytoplasmic domain (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Conrad <em>et al.</em> 2014</a>). This, similarly to previous discoveries (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Gaber <em>et al.</em> 2003</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Poulsen <em>et al.</em> 2005</a>, <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">2008</a>), could possibly result in constitutively active signaling in the absence of extracellular amino acids. In all the clones with <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em> mutations, the other genes from the SPS pathway were not mutated, which is also the condition for constitutive signaling found in previous research (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Gaber <em>et al.</em> 2003</a>). <em><a href="http://www.yeastgenome.org/locus/S000001925/overview">PTR3</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003692/overview">SSY5</a></em> were also mutated but in NQ clones with no mutation in <em><a href="http://www.yeastgenome.org/locus/S000002567/overview">SSY1</a></em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/TableS1.xlsx">Table S1</a>). In our experiments, an altered activity of the SPS pathway could be a mechanism that led to increased NQ cell production. In this rationale, constitutive activity of <a href="http://www.yeastgenome.org/locus/S000002567/overview">Ssy1p</a>, the extracellular amino acid sensor, could lead cells to overestimate the amount of extracellular nutrients, which should hinder entry to quiescence. Alternatively, if the mutations are inactivating, downregulation of the pathway, <em>e.g.</em>, by inactivation of the downstream effector Ssy5p, increases replicative lifespan (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tsang <em>et al.</em> 2015</a>); again, this is a trait that is expected to confer a fitness advantage among NQ cells.</p><p id="p-46">Another gene that, when mutated, may confer advantage to NQ cells is <em><a href="http://www.yeastgenome.org/locus/S000005441/overview">IRA2</a></em>; we observed a single frameshift (and thus possibly inactivating) mutation that was present in all 36 NQ clones, but in only three Q clones (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/TableS1.xlsx">Table S1</a>). We presume that this mutation is more adaptive for the NQ than the Q cells. Inactivating mutations in <em><a href="http://www.yeastgenome.org/locus/S000005441/overview">IRA2</a></em> (and its close homolog <em><a href="http://www.yeastgenome.org/locus/S000000344/overview">IRA1</a></em>) are known to provide a fitness benefit in multiple contexts in glucose-limiting conditions (<a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Wenger <em>et al.</em> 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kvitek and Sherlock 2013</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chiotti <em>et al.</em> 2014</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Venkataram <em>et al.</em> 2016</a>) and can result in cells continuing to divide even in conditions of low glucose. Interestingly, <em><a href="http://www.yeastgenome.org/locus/S000005441/overview">IRA2</a></em> had an elevated transcriptome level during glucose quiescence (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Klosinska <em>et al.</em> 2011</a>). Taken together, it seems quite plausible that the <em><a href="http://www.yeastgenome.org/locus/S000005441/overview">IRA2</a></em> frameshift mutation may provide a fitness benefit for NQ-enriched lines, where uncontrolled growth during early starvation conditions may confer an advantage.</p></div><div id="sec-25" class="subsection"><h3>Q cell advantage may arise from a possible role of SIR3 mutations</h3><p id="p-47">We observed a significant increase in the fraction of Q cells in two Q lines (Q I and Q III lines). Ultimately, Q line cells survive to the next experimental cycle only when they efficiently convert to the Q state. However, these cells also need to be able to reestablish cell division when the conditions improve. Both of these needs have to be taken into account when considering the possible role of the selected mutations. Present in all Q lines, two mutations in the <em><a href="http://www.yeastgenome.org/locus/S000004434/overview">SIR3</a></em> gene (Ile246fs and E317*) introduce truncation after the BAH domain. Remarkably, the BAH domain alone has the binding properties of full-length <a href="http://www.yeastgenome.org/locus/S000004434/overview">Sir3p</a>, suggesting that the domain represents an independent nucleosome-binding module (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Connelly <em>et al.</em> 2006</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Onishi <em>et al.</em> 2007</a>). We hypothesize that the <em><a href="http://www.yeastgenome.org/locus/S000004434/overview">SIR3</a></em> mutant product is still available for selective binding, which attracts the <em><a href="http://www.yeastgenome.org/locus/S000002200/overview">SIR2</a>-<a href="http://www.yeastgenome.org/locus/S000002635/overview">SIR4</a></em> complex. Thus, the chromatin silencing function is sustained. However, if the protein is truncated, it would seem that <a href="http://www.yeastgenome.org/locus/S000004434/overview">Sir3p</a> dimerization through its extreme C-terminal wH (winged helix) domain, necessary for SIR-complex spreading along nucleosomes, might be impaired (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Kueng <em>et al.</em> 2013</a>). <em><a href="http://www.yeastgenome.org/locus/S000004434/overview">SIR3</a></em> also has an important role in telomere hypercluster formation that extends chronological life span during quiescence. However, this is a rather slow process that reaches its plateau at ∼6–7 d of culture at 30° (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Guidi <em>et al.</em> 2015</a>). In addition, recent research shows that telomere hyperclustering is not required for cell survival in early quiescence (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Laporte <em>et al.</em> 2016</a>). Thus, for our experiments, where cells were starved for &gt;3 d, we speculate that telomere hyperclusters may not have yet formed, and that this function of <a href="http://www.yeastgenome.org/locus/S000004434/overview">Sir3p</a> may not be not adaptive during our short-term starvation conditions, although it is possible that it plays an early role that is adaptive in our conditions. <a href="http://www.yeastgenome.org/locus/S000004434/overview">Sir3p</a> also plays a role in the epigenetic inheritance of silent chromatin in yeast (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Motwani <em>et al.</em> 2012</a>). Differentiation into distinct cell types involves establishing unique patterns of gene expression. Once established, epigenetic mechanisms operate to maintain these patterns as cells divide.</p></div><div id="sec-26" class="subsection"><h3>Mutations shared among Q and NQ lines are important for the general stress response and nutrient acquisition</h3><p id="p-48">Invasive growth is believed to allow budding yeast to search for nutrients, as the elongated cells remain associated and in communication with each other, but also to anchor yeast colonies to the stable surface (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Honigberg 2016</a>). Thus, as the population proliferates, cells can invade solid agar media or move through static liquid media in search of carbon or nitrogen. The invasive growth test revealed an obvious distinction between NQ cells, in which the invasive phenotype was present, and Q cells, where this phenotype was completely absent (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/FigureS1.pdf">Figure S1</a>). This was the case even for Q cells that were carrying the <em><a href="http://www.yeastgenome.org/locus/S000003055/overview">MMS2</a></em> mutation, suggesting that other factors influence this phenotype. Interestingly, <em><a href="http://www.yeastgenome.org/locus/S000003013/overview">rim8</a></em> mutation within a NQ lineage appeared to reduce invasiveness (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1/FigureS1.pdf">Figure S1</a>), consistent with a role of <em><a href="http://www.yeastgenome.org/locus/S000003013/overview">RIM8</a></em> in invasive growth (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Madhani and Fink 1998</a>). Mutations, such as in <a href="http://www.yeastgenome.org/locus/S000005441/overview"><em>IRA2</em></a> (despite the predominant partitioning in NQ lines) and <a href="http://www.yeastgenome.org/locus/S000005476/overview"><em>MSN1</em></a> may directly reflect additional modes of adaptation in the growth media, nutrient depletion, and/or yeast harvesting from the agar plate.</p></div><div id="sec-27" class="subsection"><h3>Conclusions</h3><p id="p-49">With this experimental set-up we wanted to begin to explore the regulation of the yeast Q state (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Klosinska <em>et al.</em> 2011</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Čáp <em>et al.</em> 2012</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Honigberg 2016</a>) and the growing interest in nonnatural, oscillating environment experimental design (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Mitchell <em>et al.</em> 2015</a>). Nutrient depletion causes yeast populations to produce Q cells at a certain frequency. By introducing selection and serial enrichment of Q or NQ cells, we uncovered pathways that appear to facilitate or hinder entry to quiescence, respectively. In the case of NQ-enriched cells, the presumed adaptive mutations we observed in the SPS signaling pathway may result in dysregulation of their adaptive anticipation mechanism and thus allow them to bypass transition to quiescence. In the case of Q-enriched cell populations, we speculate that because they were repeatedly subjected to &gt;3.5 d of starvation, the deep reorganization of the internal cell structures such as (1) spatial confinement of enzymes, (2) mitochondrial and ER protein reorganization, and (3) loss of organelle contact sites might be not necessary or even a “waste of energy” (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Suresh <em>et al.</em> 2015</a>). In such a case it could be advantageous for cells to avoid it. All of the adaptations we observed for NQ- and Q-enriched populations seem to be costly and incur trade-offs in the other conditions. Our findings demonstrate the value of nonnatural dynamic perturbations in exposing hidden sensitivities of cellular regulatory networks.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">We thank the reviewers for helpful comments; Ryszard Korona, Gavin Sherlock, Joanna Rutkowska, and Sandeep Venkataram for help, remarks, and comments; Katarzyna Pawlik and Joanna Bobula for technical assistance; and Sever Salamon for assisting with realization of the figures. The work was funded by the National Science Centre via a HARMONIA grant to D.M.W.-S. (DEC-2011/01/M/NZ8/01031), grant number 2014/13/B/NZ8/04668 awarded to K.T., and grant number DS/INoS/762/2017.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-51">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041749/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-52"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-03-04"><span class="received-label">Received </span>March 4, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-04-11"><span class="accepted-label">Accepted </span>April 11, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Wloch-Salamon <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.1" data-doi="10.1126/science.1080418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilaniu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gustafsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rigoulet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nyström</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Asymmetric inheritance of oxidatively damaged proteins during cytokinesis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">299</span>: <span class="cit-fpage">1751</span>–<span class="cit-lpage">1753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1080418%26rft_id%253Dinfo%253Apmid%252F12610228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTkvNTYxMy8xNzUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.2" data-doi="10.1083/jcb.200604072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buttner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aragon</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meirelles</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Isolation of quiescent and nonquiescent cells from yeast stationary-phase cultures.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.volume%253D174%26rft.issue%253D1%26rft.spage%253D89%26rft.epage%253D100%26rft.atitle%253DIsolation%2Bof%2Bquiescent%2Band%2Bnonquiescent%2Bcells%2Bfrom%2Byeast%2Bstationary-phase%2Bcultures.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200604072%26rft_id%253Dinfo%253Apmid%252F16818721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjg6IjE3NC8xLzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anna-A</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaetao</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Werner-Washburne</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Separation and characterization of quiescent and senescent cells from S-cerevisiae stationary-phase cultures.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">367a</span>–<span class="cit-lpage">368a</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft.volume%253D15%26rft.spage%253D367a%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.4" data-doi="10.1111/j.1474-9726.2009.00469.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alvers</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fishwick</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Autophagy and amino acid homeostasis are required for chronological longevity in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Aging Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAging%2Bcell%26rft.stitle%253DAging%2BCell%26rft.aulast%253DAlvers%26rft.auinit1%253DA.%2BL.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253D353%26rft.epage%253D369%26rft.atitle%253DAutophagy%2Band%2Bamino%2Bacid%2Bhomeostasis%2Bare%2Brequired%2Bfor%2Bchronological%2Blongevity%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1474-9726.2009.00469.x%26rft_id%253Dinfo%253Apmid%252F19302372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1474-9726.2009.00469.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19302372&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268213300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.5" data-doi="10.1091/mbc.E07-07-0666"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aragon</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meirelles</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Characterization of differentiated quiescent and nonquiescent cells in yeast stationary-phase cultures.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1271</span>–<span class="cit-lpage">1280</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E07-07-0666%26rft_id%253Dinfo%253Apmid%252F18199684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE5LzMvMTI3MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.6.1899.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bono</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfennig</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burch</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The emergence of performance trade‐offs during local adaptation: insights from experimental evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1720</span>–<span class="cit-lpage">1733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D1720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.7" data-doi="10.1073/pnas.95.10.5678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broomfield</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">MMS2, encoding a ubiquitin-conjugating-enzyme-like protein, is a member of the yeast error-free postreplication repair pathway.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">5678</span>–<span class="cit-lpage">5683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBroomfield%26rft.auinit1%253DS.%26rft.volume%253D95%26rft.issue%253D10%26rft.spage%253D5678%26rft.epage%253D5683%26rft.atitle%253DMMS2%252C%2Bencoding%2Ba%2Bubiquitin-conjugating-enzyme-like%2Bprotein%252C%2Bis%2Ba%2Bmember%2Bof%2Bthe%2Byeast%2Berror-free%2Bpostreplication%2Brepair%2Bpathway%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.10.5678%26rft_id%253Dinfo%253Apmid%252F9576943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTUvMTAvNTY3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.8" data-doi="10.1371/journal.ppat.1004356"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brunke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hube</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Adaptive prediction as a strategy in microbial infections.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D10%26rft.spage%253De1004356%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1004356%26rft_id%253Dinfo%253Apmid%252F25275642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1004356&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25275642&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.9" data-doi="10.1074/jbc.M109.022871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Čáp</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Váchová</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palková</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Yeast colony survival depends on metabolic adaptation and cell differentiation rather than on stress defense.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">32572</span>–<span class="cit-lpage">32581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M109.022871%26rft_id%253Dinfo%253Apmid%252F19801643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODQvNDcvMzI1NzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.10" data-doi="10.1016/j.molcel.2012.04.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Čáp</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Štěpánek</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harant</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Váchová</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palková</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cell differentiation within a yeast colony: metabolic and regulatory parallels with a tumor-affected organism.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DCap%26rft.auinit1%253DM.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D436%26rft.epage%253D448%26rft.atitle%253DCell%2Bdifferentiation%2Bwithin%2Ba%2Byeast%2Bcolony%253A%2Bmetabolic%2Band%2Bregulatory%2Bparallels%2Bwith%2Ba%2Btumor-affected%2Borganism.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.04.001%26rft_id%253Dinfo%253Apmid%252F22560924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.04.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22560924&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304518900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.11" data-doi="10.1016/j.ygeno.2014.10.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chiotti</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kvitek</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koniges</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The valley-of-death: reciprocal sign epistasis constrains adaptive trajectories in a constant, nutrient limiting environment.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.volume%253D104%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2014.10.011%26rft_id%253Dinfo%253Apmid%252F25449178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2014.10.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25449178&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.12" data-doi="10.1128/MCB.26.8.3256-3265.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">H.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">R.-M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Structure and function of the <em>Saccharomyces cerevisiae</em> Sir3 BAH domain.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">3256</span>–<span class="cit-lpage">3265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DConnelly%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D26%26rft.issue%253D8%26rft.spage%253D3256%26rft.epage%253D3265%26rft.atitle%253DStructure%2Band%2BFunction%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2BSir3%2BBAH%2BDomain%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.26.8.3256-3265.2006%26rft_id%253Dinfo%253Apmid%252F16581798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI2LzgvMzI1NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.13" data-doi="10.1111/1574-6976.12065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schothorst</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kankipati</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Zeebroeck</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubio-Texeira</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Nutrient sensing and signaling in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">254</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1574-6976.12065%26rft_id%253Dinfo%253Apmid%252F24483210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3JlIjtzOjU6InJlc2lkIjtzOjg6IjM4LzIvMjU0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.14" data-doi="10.1111/j.1567-1364.2009.00583.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Generation of a large set of genetically tractable haploid and diploid <em>Saccharomyces</em> strains.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1217</span>–<span class="cit-lpage">1225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00583.x%26rft_id%253Dinfo%253Apmid%252F19840116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjg6IjkvOC8xMjE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.15" data-doi="10.1073/pnas.240345197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cullen</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sprague</span>  <span class="cit-name-given-names">G. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Glucose depletion causes haploid invasive growth in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">13619</span>–<span class="cit-lpage">13624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.240345197%26rft_id%253Dinfo%253Apmid%252F11095711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjUvMTM2MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.16" data-doi="10.1091/mbc.E10-06-0499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joe</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meirelles</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The proteomics of quiescent and nonquiescent cell differentiation in yeast stationary-phase cultures.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">988</span>–<span class="cit-lpage">998</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E10-06-0499%26rft_id%253Dinfo%253Apmid%252F21289090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjIyLzcvOTg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.17" data-doi="10.1038/ng.123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeLuna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vetsigian</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoresh</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegreness</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colón-González</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Exposing the fitness contribution of duplicated genes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">676</span>–<span class="cit-lpage">681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDeluna%26rft.auinit1%253DA.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D676%26rft.epage%253D681%26rft.atitle%253DExposing%2Bthe%2Bfitness%2Bcontribution%2Bof%2Bduplicated%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.123%26rft_id%253Dinfo%253Apmid%252F18408719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18408719&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255366700039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.18" data-doi="10.1111/j.1574-6976.2011.00287.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Virgilio</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The essence of yeast quiescence.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6976.2011.00287.x%26rft_id%253Dinfo%253Apmid%252F21658086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3JlIjtzOjU6InJlc2lkIjtzOjg6IjM2LzIvMzA2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.19" data-doi="10.1093/molbev/mss253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dhar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sägesser</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weikert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Yeast adapts to a changing stressful environment by evolving cross-protection and anticipatory gene regulation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss253%26rft_id%253Dinfo%253Apmid%252F23125229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMwLzMvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.20" data-doi="10.1046/j.1365-2958.1998.00714.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Didion</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regenberg</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorgensen</span>  <span class="cit-name-given-names">M. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kielland-Brandt</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The permease homologue Ssy1p controls the expression of amino acid and peptide transporter genes in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DDidion%26rft.auinit1%253DT.%26rft.volume%253D27%26rft.issue%253D3%26rft.spage%253D643%26rft.epage%253D650%26rft.atitle%253DThe%2Bpermease%2Bhomologue%2BSsy1p%2Bcontrols%2Bthe%2Bexpression%2Bof%2Bamino%2Bacid%2Band%2Bpeptide%2Btransporter%2Bgenes%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.1998.00714.x%26rft_id%253Dinfo%253Apmid%252F9489675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.1998.00714.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9489675&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071999000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.21" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMyLzUvMTc5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1899.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">PHYLIP: <em>Phylogenetic Inference Program</em>, <em>Version 3.6</em></span>. <span class="cit-publ-name">University of Washington</span>, <span class="cit-publ-loc">Seattle</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forsberg</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andréasson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molinér</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ljungdahl</span>  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Suppressors of <em>ssy1</em> and <em>ptr3</em> null mutations define novel amino acid sensor-independent genes in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">973</span>–<span class="cit-lpage">988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DForsberg%26rft.auinit1%253DH.%26rft.volume%253D158%26rft.issue%253D3%26rft.spage%253D973%26rft.epage%253D988%26rft.atitle%253DSuppressors%2Bof%2Bssy1%2Band%2Bptr3%2BNull%2BMutations%2BDefine%2BNovel%2BAmino%2BAcid%2BSensor-Independent%2BGenes%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F11454748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU4LzMvOTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.24" data-doi="10.1128/EC.2.5.922-929.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaber</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ottow</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kielland-Brandt</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Constitutive and hyperresponsive signaling by mutant forms of <em>Saccharomyces cerevisiae</em> amino acid sensor Ssy1.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">922</span>–<span class="cit-lpage">929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DGaber%26rft.auinit1%253DR.%2BF.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D922%26rft.epage%253D929%26rft.atitle%253DConstitutive%2Band%2BHyperresponsive%2BSignaling%2Bby%2BMutant%2BForms%2Bof%2BSaccharomyces%2Bcerevisiae%2BAmino%2BAcid%2BSensor%2BSsy1%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.2.5.922-929.2003%26rft_id%253Dinfo%253Apmid%252F14555474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiIyLzUvOTIyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.25" data-doi="10.1046/j.1365-2958.1999.01151.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gagiano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Dyk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambrechts</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pretorius</span>  <span class="cit-name-given-names">I. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Msn1p/Mss10p, Mss11p and Muc1p/Flo11p are part of a signal transduction pathway downstream of Mep2p regulating invasive growth and pseudohyphal differentiation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DGagiano%26rft.auinit1%253DM.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D103%26rft.epage%253D116%26rft.atitle%253DMsn1p%252FMss10p%252C%2BMss11p%2Band%2BMuc1p%252FFlo11p%2Bare%2Bpart%2Bof%2Ba%2Bsignal%2Btransduction%2Bpathway%2Bdownstream%2Bof%2BMep2p%2Bregulating%2Binvasive%2Bgrowth%2Band%2Bpseudohyphal%2Bdifferentiation%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.1999.01151.x%26rft_id%253Dinfo%253Apmid%252F9987114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.1999.01151.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9987114&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077954400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.26" data-doi="10.1534/genetics.112.145243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gartenberg</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The nuts and bolts of transcriptionally silent chromatin in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">1563</span>–<span class="cit-lpage">1599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGartenberg%26rft.auinit1%253DM.%2BR.%26rft.volume%253D203%26rft.issue%253D4%26rft.spage%253D1563%26rft.epage%253D1599%26rft.atitle%253DThe%2BNuts%2Band%2BBolts%2Bof%2BTranscriptionally%2BSilent%2BChromatin%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.145243%26rft_id%253Dinfo%253Apmid%252F27516616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMy80LzE1NjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.27" data-doi="10.1128/MMBR.68.2.187-206.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petsko</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ringe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">“Sleeping beauty”: quiescence in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DGray%26rft.auinit1%253DJ.%2BV.%26rft.volume%253D68%26rft.issue%253D2%26rft.spage%253D187%26rft.epage%253D206%26rft.atitle%253D%2522Sleeping%2BBeauty%2522%253A%2BQuiescence%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.68.2.187-206.2004%26rft_id%253Dinfo%253Apmid%252F15187181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2OC8yLzE4NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.28" data-doi="10.1186/s13059-014-0572-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guidi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruault</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marbouty</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loïodice</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cournac</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Spatial reorganization of telomeres in long-lived quiescent cells.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0572-2%26rft_id%253Dinfo%253Apmid%252F25583448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0572-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25583448&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.29" data-doi="10.1073/pnas.0710770105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hersen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClean</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahadevan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramanathan</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Signal processing by the HOG MAP kinase pathway.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">7165</span>–<span class="cit-lpage">7170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0710770105%26rft_id%253Dinfo%253Apmid%252F18480263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzIwLzcxNjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Honigberg</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Similar environments but diverse fates: responses of budding yeast to nutrient deprivation</span>. <abbr class="cit-jnl-abbrev">Microb. Cell</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">302</span>–<span class="cit-lpage">328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrob.%2BCell%26rft.volume%253D3%26rft.spage%253D302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.31" data-doi="10.1046/j.1420-9101.2002.00377.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kassen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The experimental evolution of specialists, generalists, and the maintenance of diversity.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D15%26rft.spage%253D173%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1420-9101.2002.00377.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1420-9101.2002.00377.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000174709000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.32" data-doi="10.1128/MCB.19.8.5405"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klasson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ljungdahl</span>  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Ssy1p and Ptr3p are plasma membrane components of a yeast system that senses extracellular amino acids.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">5405</span>–<span class="cit-lpage">5416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DKlasson%26rft.auinit1%253DH.%26rft.volume%253D19%26rft.issue%253D8%26rft.spage%253D5405%26rft.epage%253D5416%26rft.atitle%253DSsy1p%2Band%2BPtr3p%2BAre%2BPlasma%2BMembrane%2BComponents%2Bof%2Ba%2BYeast%2BSystem%2BThat%2BSenses%2BExtracellular%2BAmino%2BAcids%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.8.5405%26rft_id%253Dinfo%253Apmid%252F10409731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE5LzgvNTQwNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.33" data-doi="10.1101/gad.2011311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klosinska</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crutchfield</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradley</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabinowitz</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broach</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Yeast cells can access distinct quiescent states.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">336</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.2011311%26rft_id%253Dinfo%253Apmid%252F21289062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjUvNC8zMzYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.34" data-doi="10.1371/journal.pgen.1002253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruegel</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robison</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dange</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kahlert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delaney</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Elevated proteasome capacity extends replicative lifespan in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKruegel%26rft.auinit1%253DU.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De1002253%26rft.epage%253De1002253%26rft.atitle%253DElevated%2Bproteasome%2Bcapacity%2Bextends%2Breplicative%2Blifespan%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002253%26rft_id%253Dinfo%253Apmid%252F21931558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21931558&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.35" data-doi="10.1126/science.1250939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kryazhimskiy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jerison</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Global epistasis makes adaptation predictable despite sequence-level stochasticity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">344</span>: <span class="cit-fpage">1519</span>–<span class="cit-lpage">1522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKryazhimskiy%26rft.auinit1%253DS.%26rft.volume%253D344%26rft.issue%253D6191%26rft.spage%253D1519%26rft.epage%253D1522%26rft.atitle%253DGlobal%2Bepistasis%2Bmakes%2Badaptation%2Bpredictable%2Bdespite%2Bsequence-level%2Bstochasticity%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1250939%26rft_id%253Dinfo%253Apmid%252F24970088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDQvNjE5MS8xNTE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.36" data-doi="10.1371/journal.pgen.1002727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kueng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai-Pflugfelder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oppikofer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Regulating repression: roles for the Sir4 N-terminus in linker DNA protection and stabilization of epigenetic states.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKueng%26rft.auinit1%253DS.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002727%26rft.epage%253De1002727%26rft.atitle%253DRegulating%2Brepression%253A%2Broles%2Bfor%2Bthe%2Bsir4%2BN-terminus%2Bin%2Blinker%2BDNA%2Bprotection%2Band%2Bstabilization%2Bof%2Bepigenetic%2Bstates.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002727%26rft_id%253Dinfo%253Apmid%252F22654676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002727&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22654676&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.37" data-doi="10.1146/annurev-genet-021313-173730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kueng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oppikofer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasser</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SIR proteins and the assembly of silent chromatin in budding yeast.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenet.%26rft.volume%253D47%26rft.spage%253D275%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-021313-173730%26rft_id%253Dinfo%253Apmid%252F24016189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-021313-173730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24016189&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.38" data-doi="10.1371/journal.pgen.1003972"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kvitek</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole genome, whole population sequencing reveals that loss of signaling networks is the major adaptive strategy in a constant environment.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003972</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003972%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003972%26rft_id%253Dinfo%253Apmid%252F24278038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003972&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24278038&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.39" data-doi="10.1038/nature12344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">G. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickman</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sodergren</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinstock</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Pervasive genetic hitchhiking and clonal interference in forty evolving yeast populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">500</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D500%26rft.spage%253D571%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12344%26rft_id%253Dinfo%253Apmid%252F23873039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23873039&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323625900033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.40" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.41" data-doi="10.1091/mbc.E16-01-0069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laporte</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courtout</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sagot</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Quiescent <em>Saccharomyces cerevisiae</em> forms telomere hyperclusters at the nuclear membrane vicinity through a multifaceted mechanism involving Esc1, the Sir complex, and chromatin condensation</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1875</span>–<span class="cit-lpage">1884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E16-01-0069%26rft_id%253Dinfo%253Apmid%252F27122604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xMi8xODc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.6.1899.42" data-doi="10.1086/285289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tadler</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Long-term experimental evolution in Escherichia coli. 1. Adaptation and divergence during 2000 generations.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">1315</span>–<span class="cit-lpage">1341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D138%26rft.spage%253D1315%26rft_id%253Dinfo%253Adoi%252F10.1086%252F285289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/285289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991HB55000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.43" data-doi="10.1371/journal.pbio.1001325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziv</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegal</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Bet hedging in yeast by heterogeneous, age-correlated expression of a stress protectant.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1001325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DLevy%26rft.auinit1%253DS.%2BF.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253De1001325%26rft.epage%253De1001325%26rft.atitle%253DBet%2Bhedging%2Bin%2Byeast%2Bby%2Bheterogeneous%252C%2Bage-correlated%2Bexpression%2Bof%2Ba%2Bstress%2Bprotectant.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001325%26rft_id%253Dinfo%253Apmid%252F22589700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22589700&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.44" data-doi="10.1016/j.cell.2008.09.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacGurn</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stefan</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emr</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Arrestin-related ubiquitin-ligase adaptors regulate endocytosis and protein turnover at the cell surface.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLin%26rft.auinit1%253DC.%2BH.%26rft.volume%253D135%26rft.issue%253D4%26rft.spage%253D714%26rft.epage%253D725%26rft.atitle%253DArrestin-related%2Bubiquitin-ligase%2Badaptors%2Bregulate%2Bendocytosis%2Band%2Bprotein%2Bturnover%2Bat%2Bthe%2Bcell%2Bsurface.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.09.025%26rft_id%253Dinfo%253Apmid%252F18976803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.09.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18976803&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260886900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.45" data-doi="10.1534/genetics.111.133306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ljungdahl</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daignan-Fornier</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Regulation of amino acid, nucleotide, and phosphate metabolism in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">885</span>–<span class="cit-lpage">929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLjungdahl%26rft.auinit1%253DP.%2BO.%26rft.volume%253D190%26rft.issue%253D3%26rft.spage%253D885%26rft.epage%253D929%26rft.atitle%253DRegulation%2Bof%2BAmino%2BAcid%252C%2BNucleotide%252C%2Band%2BPhosphate%2BMetabolism%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.133306%26rft_id%253Dinfo%253Apmid%252F22419079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzMvODg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.46" data-doi="10.1016/S0962-8924(98)01298-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Madhani</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The control of filamentous differentiation and virulence in fungi.</span> <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcell%2Bbiology%26rft.stitle%253DTrends%2BCell%2BBiol%26rft.aulast%253DMadhani%26rft.auinit1%253DH.%2BD.%26rft.volume%253D8%26rft.issue%253D9%26rft.spage%253D348%26rft.epage%253D353%26rft.atitle%253DThe%2Bcontrol%2Bof%2Bfilamentous%2Bdifferentiation%2Band%2Bvirulence%2Bin%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0962-8924%252898%252901298-7%26rft_id%253Dinfo%253Apmid%252F9728395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0962-8924(98)01298-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9728395&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075432700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.47" data-doi="10.1016/j.molcel.2015.07.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKnight</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boerma</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breeden</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsukiyama</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Global promoter targeting of a conserved lysine deacetylase for transcriptional shutoff during quiescence entry.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">732</span>–<span class="cit-lpage">743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D59%26rft.spage%253D732%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2015.07.014%26rft_id%253Dinfo%253Apmid%252F26300265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2015.07.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26300265&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellner</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hairston</span>  <span class="cit-name-given-names">N. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Can population genetics adapt to rapid evolution?</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">408</span>–<span class="cit-lpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D32%26rft.spage%253D408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.49" data-doi="10.1038/nature08112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romano</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groisman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yona</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekel</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Adaptive prediction of environmental changes by microorganisms.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">460</span>: <span class="cit-fpage">220</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMitchell%26rft.auinit1%253DA.%26rft.volume%253D460%26rft.issue%253D7252%26rft.spage%253D220%26rft.epage%253D224%26rft.atitle%253DAdaptive%2Bprediction%2Bof%2Benvironmental%2Bchanges%2Bby%2Bmicroorganisms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08112%26rft_id%253Dinfo%253Apmid%252F19536156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19536156&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267761000033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.50" data-doi="10.1126/science.aab0892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Oscillatory stress stimulation uncovers an Achilles’ heel of the yeast MAPK signaling network.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">350</span>: <span class="cit-fpage">1379</span>–<span class="cit-lpage">1383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aab0892%26rft_id%253Dinfo%253Apmid%252F26586187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTAvNjI2Ni8xMzc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.51" data-doi="10.1128/MCB.06399-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Motwani</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poddar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">S. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sir3 and epigenetic inheritance of silent chromatin in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2784</span>–<span class="cit-lpage">2793</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.06399-11%26rft_id%253Dinfo%253Apmid%252F22586263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8xNC8yNzg0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.52" data-doi="10.1371/journal.pbio.1001764"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">New</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cerulus</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Govers</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Samper</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Different levels of catabolite repression optimize growth in stable and variable environments.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1001764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D12%26rft.spage%253De1001764%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001764%26rft_id%253Dinfo%253Apmid%252F24453942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001764&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24453942&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.53" data-doi="10.1016/j.molcel.2007.12.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Onishi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liou</span>  <span class="cit-name-given-names">G.-G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchberger</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walz</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moazed</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Role of the conserved Sir3-BAH domain in nucleosome binding and silent chromatin assembly.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1015</span>–<span class="cit-lpage">1028</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DOnishi%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D6%26rft.spage%253D1015%26rft.epage%253D1028%26rft.atitle%253DRole%2Bof%2Bthe%2Bconserved%2BSir3-BAH%2Bdomain%2Bin%2Bnucleosome%2Bbinding%2Band%2Bsilent%2Bchromatin%2Bassembly.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2007.12.004%26rft_id%253Dinfo%253Apmid%252F18158899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2007.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18158899&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252170000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.54" data-doi="10.1073/pnas.93.22.12428"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ozcan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dover</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenwald</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wölfl</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Two glucose transporters in Saccharomyces cerevisiae are glucose sensors that generate a signal for induction of gene expression.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">12428</span>–<span class="cit-lpage">12432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DOzcan%26rft.auinit1%253DS.%26rft.volume%253D93%26rft.issue%253D22%26rft.spage%253D12428%26rft.epage%253D12432%26rft.atitle%253DTwo%2Bglucose%2Btransporters%2Bin%2BSaccharomyces%2Bcerevisiae%2Bare%2Bglucose%2Bsensors%2Bthat%2Bgenerate%2Ba%2Bsignal%2Bfor%2Binduction%2Bof%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.22.12428%26rft_id%253Dinfo%253Apmid%252F8901598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTMvMjIvMTI0MjgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.55" data-doi="10.1128/EC.4.6.1116-1124.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poulsen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaber</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kielland-Brandt</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Constitutive signal transduction by mutant Ssy5p and Ptr3p components of the SPS amino acid sensor system in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1116</span>–<span class="cit-lpage">1124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DPoulsen%26rft.auinit1%253DP.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253D1116%26rft.epage%253D1124%26rft.atitle%253DConstitutive%2BSignal%2BTransduction%2Bby%2BMutant%2BSsy5p%2Band%2BPtr3p%2BComponents%2Bof%2Bthe%2BSPS%2BAmino%2BAcid%2BSensor%2BSystem%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.4.6.1116-1124.2005%26rft_id%253Dinfo%253Apmid%252F15947203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI0LzYvMTExNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.56" data-doi="10.1080/09687680701771917"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poulsen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaber</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kielland-Brandt</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Hyper-and hyporesponsive mutant forms of the <em>Saccharomyces cerevisiae</em> Ssy1 amino acid sensor.</span> <abbr class="cit-jnl-abbrev">Mol. Membr. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">164</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmembrane%2Bbiology%26rft.stitle%253DMol%2BMembr%2BBiol%26rft.aulast%253DPoulsen%26rft.auinit1%253DP.%26rft.volume%253D25%26rft.issue%253D2%26rft.spage%253D164%26rft.epage%253D176%26rft.atitle%253DHyper-%2Band%2Bhyporesponsive%2Bmutant%2Bforms%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2BSsy1%2Bamino%2Bacid%2Bsensor.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F09687680701771917%26rft_id%253Dinfo%253Apmid%252F18307103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/09687680701771917&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18307103&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253656800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.57" data-doi="10.1093/nar/gkv723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutledge</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belton</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broach</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The yeast genome undergoes significant topological reorganization in quiescence.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">8299</span>–<span class="cit-lpage">8313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv723%26rft_id%253Dinfo%253Apmid%252F26202961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0My8xNy84Mjk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.58" data-doi="10.1371/journal.pbio.1002068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siegal</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Shifting sugars and shifting paradigms.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1002068</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D13%26rft.spage%253De1002068%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.59" data-doi="10.1007/s00294-009-0287-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smets</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghillebert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Snijder</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swinnen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Life in the midst of scarcity: adaptations to nutrient availability in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DSmets%26rft.auinit1%253DB.%26rft.volume%253D56%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D32%26rft.atitle%253DLife%2Bin%2Bthe%2Bmidst%2Bof%2Bscarcity%253A%2Badaptations%2Bto%2Bnutrient%2Bavailability%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00294-009-0287-1%26rft_id%253Dinfo%253Apmid%252F20054690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00294-009-0287-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20054690&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273786800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brachmann</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillus</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeke</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Distribution of a limited Sir2 protein pool regulates the strength of yeast rDNA silencing and is modulated by Sir4p.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">1205</span>–<span class="cit-lpage">1219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D149%26rft.issue%253D3%26rft.spage%253D1205%26rft.epage%253D1219%26rft.atitle%253DDistribution%2Bof%2Ba%2BLimited%2BSir2%2BProtein%2BPool%2BRegulates%2Bthe%2BStrength%2Bof%2BYeast%2BrDNA%2BSilencing%2Band%2BIs%2BModulated%2Bby%2BSir4p%26rft_id%253Dinfo%253Apmid%252F9649515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0OS8zLzEyMDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.61" data-doi="10.1091/mbc.E14-11-1559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suresh</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">dos Santos</span>  <span class="cit-name-given-names">A. X. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kukulski</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyedmers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riezman</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Prolonged starvation drives reversible sequestration of lipid biosynthetic enzymes and organelle reorganization in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1601</span>–<span class="cit-lpage">1615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E14-11-1559%26rft_id%253Dinfo%253Apmid%252F25761633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI2LzkvMTYwMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.62" data-doi="10.1126/science.1154456"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tagkopoulos</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavazoie</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Predictive behavior within microbial genetic networks.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">1313</span>–<span class="cit-lpage">1317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1154456%26rft_id%253Dinfo%253Apmid%252F18467556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4MS8xMzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.63" data-doi="10.1073/pnas.0911055107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takeda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kikuchi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kokubu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Synergistic roles of the proteasome and autophagy for mitochondrial maintenance and chronological lifespan in fission yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">3540</span>–<span class="cit-lpage">3545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0911055107%26rft_id%253Dinfo%253Apmid%252F20133687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzgvMzU0MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.64" data-doi="10.1128/MCB.10.8.4303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakafuku</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamanoi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaziro</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">IRA2, a second gene of Saccharomyces cerevisiae that encodes a protein with a domain homologous to mammalian ras GTPase-activating protein.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">4303</span>–<span class="cit-lpage">4313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DTanaka%26rft.auinit1%253DK%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D4303%26rft.epage%253D4313%26rft.atitle%253DIRA2%252C%2Ba%2Bsecond%2Bgene%2Bof%2BSaccharomyces%2Bcerevisiae%2Bthat%2Bencodes%2Ba%2Bprotein%2Bwith%2Ba%2Bdomain%2Bhomologous%2Bto%2Bmammalian%2Bras%2BGTPase-activating%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.10.8.4303%26rft_id%253Dinfo%253Apmid%252F2164637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEwLzgvNDMwMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.65" data-doi="10.1074/jbc.M115.644534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ilyas</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reduced Ssy1-Ptr3-Ssy5 (SPS) signaling extends replicative life span by enhancing NAD<sup>+</sup> homeostasis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">12753</span>–<span class="cit-lpage">12764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M115.644534%26rft_id%253Dinfo%253Apmid%252F25825491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyOTAvMjAvMTI3NTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.66" data-doi="10.4161/cc.19879"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valcourt</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemons</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kojima</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demuren</span>  <span class="cit-name-given-names">O. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Staying alive: metabolic adaptations to quiescence.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1680</span>–<span class="cit-lpage">1696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bcycle%2B%2528Georgetown%252C%2BTex.%2529%26rft.stitle%253DCell%2BCycle%26rft.aulast%253DValcourt%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D11%26rft.issue%253D9%26rft.spage%253D1680%26rft.epage%253D1696%26rft.atitle%253DStaying%2Balive%253A%2Bmetabolic%2Badaptations%2Bto%2Bquiescence.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fcc.19879%26rft_id%253Dinfo%253Apmid%252F22510571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/cc.19879&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22510571&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303528500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.67" data-doi="10.1371/journal.pbio.1000503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Linden</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beverly</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadener</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasserman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide analysis of light-and temperature-entrained circadian transcripts in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253Dvan%2Bder%2BLinden%26rft.auinit1%253DA.%2BM.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253De1000503%26rft.epage%253De1000503%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Blight-%2Band%2Btemperature-entrained%2Bcircadian%2Btranscripts%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000503%26rft_id%253Dinfo%253Apmid%252F20967231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20967231&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.68" data-doi="10.1016/j.cell.2016.08.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venkataram</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Development of a comprehensive genotype-to-fitness map of adaptation-driving mutations in yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1585</span>–<span class="cit-lpage">1596.e22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D1585%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.08.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.69" data-doi="10.1371/journal.pbio.1002041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atolia</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hua</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savir</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Escalante-Chong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Natural variation in preparation for nutrient depletion reveals a cost–benefit tradeoff.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1002041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D13%26rft.spage%253De1002041%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1002041%26rft_id%253Dinfo%253Apmid%252F25626068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1002041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25626068&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.70" data-doi="10.1371/journal.pgen.1002202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wenger</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piotrowski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagarajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiotti</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hunger artists: yeast adapted to carbon limitation show trade-offs under carbon sufficiency.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWenger%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De1002202%26rft.epage%253De1002202%26rft.atitle%253DHunger%2Bartists%253A%2Byeast%2Badapted%2Bto%2Bcarbon%2Blimitation%2Bshow%2Btrade-offs%2Bunder%2Bcarbon%2Bsufficiency.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002202%26rft_id%253Dinfo%253Apmid%252F21829391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21829391&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.71" data-doi="10.1093/nar/29.22.4625"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wenz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwank</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoja</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schüller</span>  <span class="cit-name-given-names">H.-J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A downstream regulatory element located within the coding sequence mediates autoregulated expression of the yeast fatty acid synthase gene FAS2 by the FAS1 gene product.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">4625</span>–<span class="cit-lpage">4632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DWenz%26rft.auinit1%253DP.%26rft.volume%253D29%26rft.issue%253D22%26rft.spage%253D4625%26rft.epage%253D4632%26rft.atitle%253DA%2Bdownstream%2Bregulatory%2Belement%2Blocated%2Bwithin%2Bthe%2Bcoding%2Bsequence%2Bmediates%2Bautoregulated%2Bexpression%2Bof%2Bthe%2Byeast%2Bfatty%2Bacid%2Bsynthase%2Bgene%2BFAS2%2Bby%2Bthe%2BFAS1%2Bgene%2Bproduct%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.22.4625%26rft_id%253Dinfo%253Apmid%252F11713312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8yMi80NjI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wloch-Salamon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Majewska</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pajak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pawlik</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plech</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic modifications allowing for obtaining non-clumpy <em>Saccharomyces cerevisiae</em> populations from various strain backgrounds.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">196</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%26rft.volume%253D30%26rft.spage%253D196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wloch-Salamon</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sociobiology of the budding yeast.</span> <abbr class="cit-jnl-abbrev">J. Biosci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiosci.%26rft.volume%253D39%26rft.spage%253D225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fontanie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bacchetti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genetic interactions between error-prone and error-free postreplication repair pathways in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mutat. Res. DNA Repair</abbr> <span class="cit-vol">435</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%2BDNA%2BRepair%26rft.volume%253D435%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.75" data-doi="10.1016/j.cell.2015.10.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yona</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frumkin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilpel</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A relay race on the evolutionary adaptation spectrum.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D163%26rft.spage%253D549%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.10.005%26rft_id%253Dinfo%253Apmid%252F26496602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26496602&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1899.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hillyer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hokamp</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konstantinov</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Distinct histone methylation and transcription profiles are established during the development of cellular quiescence in yeast.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/6/1899.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1887" title="Predictive Genomic Analyses Inform the Basis for Vitamin Metabolism and Provisioning in Bacteria-Arthropod Endosymbioses" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1913" title="Genetic Architecture of a Rice Nested Association Mapping Population" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/6"><span>Volume 7 Issue 6, June 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/6.cover-source.jpg?itok=5_J-OKHc" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1899.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LkSuwl2HYA9Qknh4XMYc_H7yDyIpx3dwlerlVRcSdIE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/6/1899" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/6/1899" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/6/1899" class="active" data-icon-position="" data-hide-link-title="0">Adaptive Roles of SSY1 and SIR3 During Cycles of Growth and Starvation in Saccharomyces cerevisiae Populations Enriched for Quiescent or Nonquiescent Cells</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484758" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-S7CWjW93184DNnE_buDCQBpiavz5m7AeF0rDrnp3Z7U" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1899.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/6/1899" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-cmCPw27M_5FzL8BlJN0mcyhSyNQUddGUF9RomIyK9Hw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/6/1899" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1899.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484758" id="node484758--2" data-pisa="ggg;7/6/1899" data-pisa-master="ggg;g3.117.041749" data-apath="/ggg/7/6/1899.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Adaptive Roles of <em>SSY1</em> and <em>SIR3</em> During Cycles of Growth and Starvation in <em>Saccharomyces cerevisiae</em> Populations Enriched for Quiescent or Nonquiescent Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0040-1838" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dominika M. Wloch-Salamon</span>, <span class="highwire-citation-author" data-delta="1">Katarzyna Tomala</span>, <span class="highwire-citation-author" data-delta="2">Dimitra Aggeli</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7041-0035" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Barbara Dunn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1899-1911;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041749 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484758--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dominika M. Wloch-Salamon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Environmental Sciences, Jagiellonian University, Krakow 30-387, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominika%2BM.%2BWloch-Salamon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wloch-Salamon%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADominika%2BM.%2BWloch-Salamon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0040-1838" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dominika M. Wloch-Salamon</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dominika.wloch-salamon@uj.edu.pl" class="" data-icon-position="" data-hide-link-title="0">dominika.wloch-salamon@uj.edu.pl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katarzyna Tomala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Environmental Sciences, Jagiellonian University, Krakow 30-387, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katarzyna%2BTomala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tomala%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatarzyna%2BTomala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dimitra Aggeli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Stanford University, California 94305-5120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dimitra%2BAggeli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aggeli%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADimitra%2BAggeli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Barbara Dunn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Stanford University, California 94305-5120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barbara%2BDunn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunn%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABarbara%2BDunn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7041-0035" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Barbara Dunn</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/6/1899" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/6/1899&amp;title=Adaptive%20Roles%20of%20SSY1%20and%20SIR3%20During%20Cycles%20of%20Growth%20and%20Starvation%20in%20Saccharomyces%20cerevisiae%20Populations%20Enriched%20for%20Quiescent%20or%20Nonquiescent%20Cells" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/6/1899&amp;title=Adaptive%20Roles%20of%20SSY1%20and%20SIR3%20During%20Cycles%20of%20Growth%20and%20Starvation%20in%20Saccharomyces%20cerevisiae%20Populations%20Enriched%20for%20Quiescent%20or%20Nonquiescent%20Cells" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/6/1899&amp;title=Adaptive%20Roles%20of%20SSY1%20and%20SIR3%20During%20Cycles%20of%20Growth%20and%20Starvation%20in%20Saccharomyces%20cerevisiae%20Populations%20Enriched%20for%20Quiescent%20or%20Nonquiescent%20Cells" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/6/1899&amp;text=Adaptive%20Roles%20of%20SSY1%20and%20SIR3%20During%20Cycles%20of%20Growth%20and%20Starvation%20in%20Saccharomyces%20cerevisiae%20Populations%20Enriched%20for%20Quiescent%20or%20Nonquiescent%20Cells" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/6/1899&amp;title=Adaptive%20Roles%20of%20SSY1%20and%20SIR3%20During%20Cycles%20of%20Growth%20and%20Starvation%20in%20Saccharomyces%20cerevisiae%20Populations%20Enriched%20for%20Quiescent%20or%20Nonquiescent%20Cells" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/6/1899&amp;t=Adaptive%20Roles%20of%20SSY1%20and%20SIR3%20During%20Cycles%20of%20Growth%20and%20Starvation%20in%20Saccharomyces%20cerevisiae%20Populations%20Enriched%20for%20Quiescent%20or%20Nonquiescent%20Cells" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/6/1899&amp;title=Adaptive%20Roles%20of%20SSY1%20and%20SIR3%20During%20Cycles%20of%20Growth%20and%20Starvation%20in%20Saccharomyces%20cerevisiae%20Populations%20Enriched%20for%20Quiescent%20or%20Nonquiescent%20Cells" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/6/1899&amp;title=Adaptive%20Roles%20of%20SSY1%20and%20SIR3%20During%20Cycles%20of%20Growth%20and%20Starvation%20in%20Saccharomyces%20cerevisiae%20Populations%20Enriched%20for%20Quiescent%20or%20Nonquiescent%20Cells" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484758/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484758" id="node484758--3" data-pisa="ggg;7/6/1899" data-pisa-master="ggg;g3.117.041749" data-apath="/ggg/7/6/1899.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Adaptive Roles of <em>SSY1</em> and <em>SIR3</em> During Cycles of Growth and Starvation in <em>Saccharomyces cerevisiae</em> Populations Enriched for Quiescent or Nonquiescent Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0040-1838" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dominika M. Wloch-Salamon</span>, <span class="highwire-citation-author" data-delta="1">Katarzyna Tomala</span>, <span class="highwire-citation-author" data-delta="2">Dimitra Aggeli</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7041-0035" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Barbara Dunn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1899-1911;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041749 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484758--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Dominika M. Wloch-Salamon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Environmental Sciences, Jagiellonian University, Krakow 30-387, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dominika%2BM.%2BWloch-Salamon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wloch-Salamon%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADominika%2BM.%2BWloch-Salamon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0040-1838" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dominika M. Wloch-Salamon</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dominika.wloch-salamon@uj.edu.pl" class="" data-icon-position="" data-hide-link-title="0">dominika.wloch-salamon@uj.edu.pl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katarzyna Tomala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Environmental Sciences, Jagiellonian University, Krakow 30-387, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katarzyna%2BTomala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tomala%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatarzyna%2BTomala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dimitra Aggeli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Stanford University, California 94305-5120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dimitra%2BAggeli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aggeli%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADimitra%2BAggeli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Barbara Dunn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Stanford University, California 94305-5120</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Barbara%2BDunn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dunn%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABarbara%2BDunn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7041-0035" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Barbara Dunn</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1899&amp;rft.epage=1911&amp;rft.atitle=Adaptive%20Roles%20of%20SSY1%20and%20SIR3%20During%20Cycles%20of%20Growth%20and%20Starvation%20in%20Saccharomyces%20cerevisiae%20Populations%20Enriched%20for%20Quiescent%20or%20Nonquiescent%20Cells&amp;rft.volume=7&amp;rft.issue=6&amp;rft.date=2017-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wloch-Salamon%2C+Dominika+M.&amp;rft.au=Tomala%2C+Katarzyna&amp;rft.au=Aggeli%2C+Dimitra&amp;rft.au=Dunn%2C+Barbara"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484758/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484758/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484758/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484758/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484758/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484758/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484758/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484758/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484758/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484758/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484758/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484758/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/6/1899" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/6/1899.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/6/1899.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/6/1899.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/6/1899.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
