<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/6/1683" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/6/1683.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/6/1683.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/6/1683.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/6/1683/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/6/1683.atom" />
<meta name="HW.pisa" content="ggg;7/6/1683" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Identification of Ganoderma Disease Resistance Loci Using Natural Field Infection of an Oil Palm Multiparental Population" />
<meta name="DC.Identifier" content="10.1534/g3.117.041764" />
<meta name="DC.Date" content="2017-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Tisné et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Multi-parental populations are promising tools for identifying quantitative disease resistance loci. Stem rot caused by Ganoderma boninense is a major threat to palm oil production, with yield losses of up to 80% prompting premature replantation of palms. There is evidence of genetic resistance sources, but the genetic architecture of Ganoderma resistance has not yet been investigated. This study aimed to identify Ganoderma resistance loci using an oil palm multi-parental population derived from nine major founders of ongoing breeding programs. A total of 1200 palm trees of the multi-parental population was planted in plots naturally infected by Ganoderma , and their health status was assessed biannually over 25 yr. The data were treated as survival data, and modeled using the Cox regression model, including a spatial effect to take the spatial component in the spread of Ganoderma into account. Based on the genotypes of 757 palm trees out of the 1200 planted, and on pedigree information, resistance loci were identified using a random effect with identity-by-descent kinship matrices as covariance matrices in the Cox model. Four Ganoderma resistance loci were identified, two controlling the occurrence of the first Ganoderma symptoms, and two the death of palm trees, while favorable haplotypes were identified among a major gene pool for ongoing breeding programs. This study implemented an efficient and flexible QTL mapping approach, and generated unique valuable information for the selection of oil palm varieties resistant to Ganoderma disease." />
<meta name="DC.Contributor" content="Sébastien Tisné" />
<meta name="DC.Contributor" content="Virginie Pomiès" />
<meta name="DC.Contributor" content="Virginie Riou" />
<meta name="DC.Contributor" content="Indra Syahputra" />
<meta name="DC.Contributor" content="Benoît Cochard" />
<meta name="DC.Contributor" content="Marie Denis" />
<meta name="article:published_time" content="2017-06-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Identification of Ganoderma Disease Resistance Loci Using Natural Field Infection of an Oil Palm Multiparental Population" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Multi-parental populations are promising tools for identifying quantitative disease resistance loci. Stem rot caused by &lt;i&gt;Ganoderma boninense&lt;/i&gt; is a major threat to palm oil production, with yield losses of up to 80% prompting premature replantation of palms. There is evidence of genetic resistance sources, but the genetic architecture of &lt;i&gt;Ganoderma&lt;/i&gt; resistance has not yet been investigated. This study aimed to identify &lt;i&gt;Ganoderma&lt;/i&gt; resistance loci using an oil palm multi-parental population derived from nine major founders of ongoing breeding programs. A total of 1200 palm trees of the multi-parental population was planted in plots naturally infected by &lt;i&gt;Ganoderma&lt;/i&gt;, and their health status was assessed biannually over 25&amp;nbsp;yr. The data were treated as survival data, and modeled using the Cox regression model, including a spatial effect to take the spatial component in the spread of &lt;i&gt;Ganoderma&lt;/i&gt; into account. Based on the genotypes of 757 palm trees out of the 1200 planted, and on pedigree information, resistance loci were identified using a random effect with identity-by-descent kinship matrices as covariance matrices in the Cox model. Four &lt;i&gt;Ganoderma&lt;/i&gt; resistance loci were identified, two controlling the occurrence of the first &lt;i&gt;Ganoderma&lt;/i&gt; symptoms, and two the death of palm trees, while favorable haplotypes were identified among a major gene pool for ongoing breeding programs. This study implemented an efficient and flexible QTL mapping approach, and generated unique valuable information for the selection of oil palm varieties resistant to &lt;i&gt;Ganoderma&lt;/i&gt; disease.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/06/01" />
<meta name="citation_mjid" content="ggg;7/6/1683" />
<meta name="citation_id" content="7/6/1683" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/6/1683" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/6/1683.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/6/1683.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/6/1683.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.041764" />
<meta name="citation_pmid" content="28592650" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="1683" />
<meta name="citation_lastpage" content="1692" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sébastien Tisné" />
<meta name="citation_author_institution" content="Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France" />
<meta name="citation_author_email" content="sebastien.tisne@cirad.fr" />
<meta name="citation_author" content="Virginie Pomiès" />
<meta name="citation_author_institution" content="Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France" />
<meta name="citation_author" content="Virginie Riou" />
<meta name="citation_author_institution" content="Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France" />
<meta name="citation_author" content="Indra Syahputra" />
<meta name="citation_author_institution" content="P.T Socfindo Medan, 20001, Indonesia" />
<meta name="citation_author" content="Benoît Cochard" />
<meta name="citation_author_institution" content="PalmElit SAS, 34980 Montferrier sur Lez, France" />
<meta name="citation_author" content="Marie Denis" />
<meta name="citation_author_institution" content="Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=A. Adeyanju;citation_author=C. Little;citation_author=J. Yu;citation_author=T. Tesso;citation_title=Genome-wide association study on resistance to stalk rot diseases in grain sorghum.;citation_pages=1165-1175;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Mol. Sci.;citation_author=J. R. Al-Obaidi;citation_author=Y. Mohd-Yusuf;citation_author=N. Razali;citation_author=J. J. Jayapalan;citation_author=C.-C. Tey;citation_title=Identification of proteins of altered abundance in oil palm infected with Ganoderma boninense.;citation_pages=5175-5192;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=P. Bajgain;citation_author=M. N. Rouse;citation_author=T. J. Tsilo;citation_author=G. K. Macharia;citation_author=S. Bhavani;citation_title=Nested association mapping of stem rust resistance in wheat using genotyping by sequencing.;citation_pages=e0155760;citation_volume=11;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_journal_abbrev=Biostatistics;citation_author=S. Banerjee;citation_author=M. M. Wall;citation_author=B. P. Carlin;citation_title=Frailty modeling for spatially correlated survival data, with application to infant mortality in Minnesota;citation_pages=123-142;citation_volume=4;citation_year=2003;citation_issue=1;citation_pmid=12925334;citation_doi=10.1093/biostatistics/4.1.123" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. M. Benson;citation_author=J. A. Poland;citation_author=B. M. Benson;citation_author=E. L. Stromberg;citation_author=R. J. Nelson;citation_title=Resistance to gray leaf spot of maize: genetic architecture and mechanisms elucidated through nested association mapping and near-isogenic line analysis.;citation_pages=e1005045;citation_volume=11;citation_year=2015;citation_pmid=25764179;citation_doi=10.1371/journal.pgen.1005045" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. Billotte;citation_author=N. Marseillac;citation_author=A.-M. Risterucci;citation_author=B. Adon;citation_author=P. Brottier;citation_title=Microsatellite-based high density linkage map in oil palm (Elaeis guineensis Jacq.).;citation_pages=754-765;citation_volume=110;citation_year=2005;citation_issue=4;citation_pmid=15723275;citation_doi=10.1007/s00122-004-1901-8" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=N. Billotte;citation_author=M. F. Jourjon;citation_author=N. Marseillac;citation_author=A. Berger;citation_author=A. Flori;citation_title=QTL detection by multi-parent linkage mapping in oil palm (Elaeis guineensis Jacq.).;citation_pages=1673-1687;citation_volume=120;citation_year=2010;citation_issue=8;citation_pmid=20182696;citation_doi=10.1007/s00122-010-1284-y" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. Bink;citation_author=M. P. Boer;citation_author=C. J. F. Ter Braak;citation_author=J. Jansen;citation_author=R. E. Voorrips;citation_title=Bayesian analysis of complex traits in pedigreed plant populations.;citation_pages=85-96;citation_volume=161;citation_year=2008;citation_doi=10.1007/s10681-007-9516-1" />
<meta name="citation_reference" content="citation_journal_title=J. Oil Palm Res.;citation_author=F. Breton;citation_author=Y. Hasan;citation_author=S. Hariadi;citation_author=Z. Lubis;citation_author=H. De Franqueville;citation_title=Characterization of parameters for the development of an early screening test for basal stem rot tolerance in oil palm progenies.;citation_pages=24-36;citation_volume=2006;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=H. Brun;citation_author=A.-M. Chèvre;citation_author=B. D. Fitt;citation_author=S. Powers;citation_author=A.-L. Besnard;citation_title=Quantitative resistance increases the durability of qualitative resistance to Leptosphaeria maculans in Brassica napus.;citation_pages=285-299;citation_volume=185;citation_year=2010;citation_issue=1;citation_pmid=19814776;citation_doi=10.1111/j.1469-8137.2009.03049.x" />
<meta name="citation_reference" content="citation_title=Evidence-based gene models for structural and functional annotations of the oil palm genome.;citation_author=K. L. Chan;citation_author=T. Tatarinova;citation_author=R. Rozana;citation_author=A. Nadzirah;citation_author=N. Azizi;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_journal_title=Test;citation_author=G. Claeskens;citation_author=R. Nguti;citation_author=P. Janssen;citation_title=One-sided tests in shared frailty models.;citation_pages=69-82;citation_volume=17;citation_year=2008;citation_doi=10.1007/s11749-006-0023-9" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=B. Cochard;citation_author=C. Carrasco-Lacombe;citation_author=V. Pomiès;citation_author=J.-F. Dufayard;citation_author=E. Suryana;citation_title=Pedigree-based linkage map in two genetic groups of oil palm.;citation_pages=1-12;citation_volume=11;citation_year=2015;citation_doi=10.1007/s11295-014-0804-3" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=J. Cockram;citation_author=A. Scuderi;citation_author=T. Barber;citation_author=E. Furuki;citation_author=K. A. Gardner;citation_title=Fine-mapping the wheat Snn1 locus conferring sensitivity to the Parastagonospora nodorum necrotrophic effector SnTox1 using an eight founder multiparent advanced generation inter-cross population.;citation_pages=2257-2266;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Planter;citation_author=R. M. Cooper;citation_author=J. Flood;citation_author=R. W. Rees;citation_title=Ganoderma boninense in oil palm plantations: current thinking on epidemiology, resistance and pathology.;citation_pages=515-526;citation_volume=87;citation_year=2011" />
<meta name="citation_reference" content="citation_title=Selection and Breeding.;citation_author=R. H. V. Corley;citation_author=P. B. Tinker;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=D. R. Cox;citation_title=Models and life-tables regression.;citation_pages=187-220;citation_volume=34;citation_year=1972" />
<meta name="citation_reference" content="citation_author=N. Cressie;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Cros;citation_author=L. Sánchez;citation_author=B. Cochard;citation_author=P. Samper;citation_author=M. Denis;citation_title=Estimation of genealogical coancestry in plant species using a pedigree reconstruction algorithm and application to an oil palm breeding population.;citation_pages=981-994;citation_volume=127;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Comput. Stat. Data Anal.;citation_author=L. Duchateau;citation_author=P. Janssen;citation_author=P. Lindsey;citation_author=C. Legrand;citation_author=R. Nguti;citation_title=The shared frailty model and the power for heterogeneity tests in multicenter trials.;citation_pages=603-620;citation_volume=40;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Mycopathologia;citation_journal_abbrev=Mycopathologia;citation_author=T. Durand-Gasselin;citation_author=H. Asmady;citation_author=A. Flori;citation_author=J. C. Jacquemard;citation_author=Z. Hayun;citation_title=Possible sources of genetic resistance in oil palm (Elaeis guineensis Jacq.) to basal stem rot caused by Ganoderma boninense--prospects for future breeding.;citation_pages=93-100;citation_volume=159;citation_year=2005;citation_issue=1;citation_pmid=15750739" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Durrant;citation_author=H. Tayem;citation_author=B. Yalcin;citation_author=J. Cleak;citation_author=L. Goodstadt;citation_title=Collaborative cross mice and their power to map host susceptibility to Aspergillus fumigatus infection.;citation_volume=21;citation_year=2011;citation_pmid=21493779;citation_doi=10.1101/gr.118786.110" />
<meta name="citation_reference" content="citation_title=Indications on sources of oil palm (Elaeis guineensis Jacq.) genetic resistance and susceptibility to Ganoderma sp., the cause of basal stem rot;citation_author=H. de Franqueville;citation_author=H. Asmady;citation_author=J. C. Jacquemard;citation_author=Z. Hayun;citation_author=T. Durand-Gasselin;citation_pages=420-431;citation_year=2001" />
<meta name="citation_reference" content="citation_title=Oléagineux;citation_author=J. P. Gascon;citation_author=C. H. de Berchoux;citation_title=Caractéristiques de la production d’Elaeis guineensis (Jacq.) de diverses origines et leurs croisements. Appl. À Sélection Palmier À Huile.;citation_pages=75-84;citation_volume=19;citation_year=1964" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. W. George;citation_author=P. M. Visscher;citation_author=C. S. Haley;citation_title=Mapping Quantitative Trait Loci in Complex Pedigrees: A Two-Step Variance Component Approach;citation_pages=2081-2092;citation_volume=156;citation_year=2000;citation_issue=4;citation_pmid=11102397" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=S. Gonen;citation_author=M. Baranski;citation_author=I. Thorland;citation_author=A. Norris;citation_author=H. Grove;citation_title=Mapping and validation of a major QTL affecting resistance to pancreas disease (salmonid alphavirus) in Atlantic salmon (Salmo salar).;citation_pages=405-414;citation_volume=115;citation_year=2015" />
<meta name="citation_reference" content="citation_author=P. Green;citation_author=K. Falls;citation_author=S. Crooks;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Mol. Plant Pathol.;citation_author=M. A. Gururani;citation_author=J. Venkatesh;citation_author=C. P. Upadhyaya;citation_author=A. Nookaraju;citation_author=S. K. Pandey;citation_title=Plant disease resistance genes: current status and future directions.;citation_pages=51-65;citation_volume=78;citation_year=2012;citation_doi=10.1016/j.pmpp.2012.01.002" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=E. O. Hama-Ali;citation_author=J. M. Panandam;citation_author=S. G. Tan;citation_author=S. S. R. S. Alwee;citation_author=T. J. Sheong;citation_title=Association between basal stem rot disease and simple sequence repeat markers in oil palm, Elaeis guineensis Jacq.;citation_pages=199-206;citation_volume=202;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Phytochemistry;citation_author=C.-L. Ho;citation_author=Y.-C. Tan;citation_title=Molecular defense response of oil palm to Ganoderma infection.;citation_pages=160-169;citation_volume=114;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C.-L. Ho;citation_author=Y.-C. Tan;citation_author=K.-A. Yeoh;citation_author=A.-K. Ghazali;citation_author=W.-Y. Yee;citation_title=De novo transcriptome analyses of host-fungal interactions in oil palm (Elaeis guineensis Jacq.).;citation_pages=66;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J. Oil Palm Res.;citation_author=A. Idris;citation_author=A. Kushairi;citation_author=S. Ismail;citation_author=D. Ariffin;citation_title=Selection for partial resistance in oil palm progenies to Ganoderma basal stem rot.;citation_pages=12-18;citation_volume=16;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Jansen;citation_title=Interval mapping of multiple quantitative trait loci.;citation_pages=205-211;citation_volume=135;citation_year=1993;citation_issue=1;citation_pmid=8224820" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=S. Jeennor;citation_author=H. Volkaert;citation_title=Mapping of quantitative trait loci (QTLs) for oil yield using SSRs and gene-based markers in African oil palm (Elaeis guineensis Jacq.).;citation_pages=1-14;citation_volume=10;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_author=D. Jiang;citation_author=H. Wang;citation_author=J. Li;citation_author=Y. Wu;citation_author=M. Fang;citation_title=Cox regression model for dissecting genetic architecture of survival time.;citation_pages=472-476;citation_volume=104;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. L. Kump;citation_author=P. J. Bradbury;citation_author=R. J. Wisser;citation_author=E. S. Buckler;citation_author=A. R. Belcher;citation_title=Genome-wide association study of quantitative resistance to southern leaf blight in the maize nested association mapping population.;citation_pages=163-168;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217757;citation_doi=10.1038/ng.747" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=Q. B. Kwong;citation_author=C. K. Teh;citation_author=A. L. Ong;citation_author=H. Y. Heng;citation_author=H. L. Lee;citation_title=Development and validation of a high-density SNP genotyping array for African oil palm.;citation_pages=1132-1141;citation_volume=9;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=AoB Plants;citation_title=Identifying the genes underlying quantitative traits: a rationale for the QTN programme.;citation_author=Y. W. Lee;citation_author=B. A. Gould;citation_author=J. R. Stinchcombe;citation_volume=6;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=Z. Luo;citation_author=Z. Piao;citation_author=X. Zhou;citation_author=T. Yang;citation_author=R. Yang;citation_title=An optimal parametric proportional hazards model for mapping heading time loci in rice.;citation_pages=37-44;citation_volume=192;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Montoya;citation_author=B. Cochard;citation_author=A. Flori;citation_author=D. Cros;citation_author=R. Lopes;citation_title=Genetic architecture of palm oil fatty acid composition in cultivated oil palm (Elaeis guineensis Jacq.) compared to its wild relative E. oleifera (HBK) cortés.;citation_pages=e95412;citation_volume=9;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=F. Morcillo;citation_author=D. Cros;citation_author=N. Billotte;citation_author=G.-F. Ngando-Ebongue;citation_author=H. Domonhédo;citation_title=Improving palm oil quality through identification and mapping of the lipase gene causing oil deterioration.;citation_pages=2160;citation_volume=4;citation_year=2013;citation_pmid=23857501" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol. Biochem.;citation_author=S. A. Nusaibah;citation_author=A. S. N. Akmar;citation_author=A. S. Idris;citation_author=M. Sariah;citation_author=Z. M. Pauzi;citation_title=Involvement of metabolites in early defense mechanism of oil palm (Elaeis guineensis Jacq.) against Ganoderma disease.;citation_pages=156-165;citation_volume=109;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Crop Prot.;citation_author=E.-C. Oerke;citation_author=H.-W. Dehne;citation_title=Safeguarding production—losses in major crops and the role of crop protection.;citation_pages=275-285;citation_volume=23;citation_year=2004;citation_doi=10.1016/j.cropro.2003.10.001" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=B. Parisseaux;citation_author=R. Bernardo;citation_title=In silico mapping of quantitative trait loci in maize.;citation_pages=508-514;citation_volume=109;citation_year=2004;citation_issue=3;citation_pmid=15150690" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=J. A. Poland;citation_author=P. J. Balint-Kurti;citation_author=R. J. Wisser;citation_author=R. C. Pratt;citation_author=R. J. Nelson;citation_title=Shades of gray: the world of quantitative disease resistance.;citation_pages=21-29;citation_volume=14;citation_year=2009;citation_issue=1;citation_pmid=19062327;citation_doi=10.1016/j.tplants.2008.10.006" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_author=W. Pootakham;citation_author=N. Jomchai;citation_author=P. Ruang-areerate;citation_author=J. R. Shearman;citation_author=C. Sonthirod;citation_title=Genome-wide SNP discovery and identification of QTL associated with agronomic traits in oil palm using genotyping-by-sequencing (GBS).;citation_pages=288-295;citation_volume=105;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Quenouille;citation_author=E. Paulhiac;citation_author=B. Moury;citation_author=A. Palloix;citation_title=Quantitative trait loci from the host genetic background modulate the durability of a resistance gene: a rational basis for sustainable resistance breeding in plants.;citation_pages=579-587;citation_volume=112;citation_year=2014" />
<meta name="citation_reference" content="citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=R. Singh;citation_author=M. Ong-Abdullah;citation_author=E.-T. L. Low;citation_author=M. A. A. Manaf;citation_author=R. Rosli;citation_title=Oil palm genome sequence reveals divergence of interfertile species in Old and New worlds.;citation_pages=335-339;citation_volume=500;citation_year=2013;citation_pmid=23883927;citation_doi=10.1038/nature12309" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=E. Sobel;citation_author=K. Lange;citation_title=Descent graphs in pedigree analysis: applications to haplotyping, location scores, and marker-sharing statistics.;citation_pages=1323-1337;citation_volume=58;citation_year=1996;citation_issue=6;citation_pmid=8651310" />
<meta name="citation_reference" content="citation_journal_title=Human heredity;citation_journal_abbrev=Hum Hered;citation_author=E. Sobel;citation_author=H. Sengul;citation_author=D. E. Weeks;citation_title=Multipoint estimation of identity-by-descent probabilities at arbitrary positions among marker loci on general pedigrees.;citation_pages=121-131;citation_volume=52;citation_year=2001;citation_issue=3;citation_pmid=11588394;citation_doi=10.1159/000053366" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=S.-S. Tee;citation_author=Y.-C. Tan;citation_author=F. Abdullah;citation_author=M. Ong-Abdullah;citation_author=C.-L. Ho;citation_title=Transcriptome of oil palm (Elaeis guineensis Jacq.) roots treated with Ganoderma boninense.;citation_pages=377-386;citation_volume=9;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=C.-K. Teh;citation_author=A.-L. Ong;citation_author=Q.-B. Kwong;citation_author=S. Apparow;citation_author=F.-T. Chew;citation_title=Genome-wide association study identifies three key loci for high mesocarp oil content in perennial crop oil palm.;citation_pages=19075;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Tisné;citation_author=M. Denis;citation_author=D. Cros;citation_author=V. Pomiès;citation_author=V. Riou;citation_title=Mixed model approach for IBD-based QTL mapping in a complex oil palm pedigree.;citation_pages=798;citation_volume=16;citation_year=2015;citation_pmid=26472667;citation_doi=10.1186/s12864-015-1985-3" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=K. Ukoskit;citation_author=V. Chanroj;citation_author=G. Bhusudsawang;citation_author=K. Pipatchartlearnwong;citation_author=S. Tangphatsornruang;citation_title=Oil palm (Elaeis guineensis Jacq.) linkage map, and quantitative trait locus analysis for sex ratio and related traits.;citation_pages=415-424;citation_volume=33;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Demography;citation_journal_abbrev=Demography;citation_author=J. W. Vaupel;citation_author=K. G. Manton;citation_author=E. Stallard;citation_title=The impact of heterogeneity in individual frailty on the dynamics of mortality.;citation_pages=439-454;citation_volume=16;citation_year=1979;citation_issue=3;citation_pmid=510638;citation_doi=10.2307/2061224" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=K. J. F. Verhoeven;citation_author=J. L. Jannink;citation_author=L. M. McIntyre;citation_title=Using mating designs to uncover QTL and the genetic architecture of complex traits.;citation_pages=139-149;citation_volume=96;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_journal_title=Aust. N. Z. J. Stat.;citation_author=H. T. Vu;citation_author=M. W. Knuiman;citation_title=Estimation in semiparametric marginal shared gamma frailty models.;citation_pages=489-501;citation_volume=44;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=X. Wei;citation_author=P. A. Jackson;citation_author=C. L. McIntyre;citation_author=K. S. Aitken;citation_author=B. Croft;citation_title=Associations between DNA markers and resistance to diseases in sugarcane and effects of population substructure.;citation_pages=155-164;citation_volume=114;citation_year=2006;citation_issue=1;citation_pmid=17047910;citation_doi=10.1007/s00122-006-0418-8" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=Z. Zhang;citation_author=S. Schwartz;citation_author=L. Wagner;citation_author=W. Miller;citation_title=A greedy algorithm for aligning DNA sequences.;citation_pages=203-214;citation_volume=7;citation_year=2000;citation_issue=1-2;citation_pmid=10890397;citation_doi=10.1089/10665270050081478" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=J. H. Zhao;citation_title=Mixed-effects Cox models of alcohol dependence in extended families.;citation_pages=S127;citation_volume=6;citation_year=2005" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Identification of Ganoderma Disease Resistance Loci Using Natural Field Infection of an Oil Palm Multiparental Population" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="twitter:description" content="Multi-parental populations are promising tools for identifying quantitative disease resistance loci. Stem rot caused by Ganoderma boninense is a major threat to palm oil production, with yield losses of up to 80% prompting premature replantation of palms. There is evidence of genetic resistance sources, but the genetic architecture of Ganoderma resistance has not yet been investigated. This study aimed to identify Ganoderma resistance loci using an oil palm multi-parental population derived from nine major founders of ongoing breeding programs. A total of 1200 palm trees of the multi-parental population was planted in plots naturally infected by Ganoderma , and their health status was assessed biannually over 25 yr. The data were treated as survival data, and modeled using the Cox regression model, including a spatial effect to take the spatial component in the spread of Ganoderma into account. Based on the genotypes of 757 palm trees out of the 1200 planted, and on pedigree information, resistance loci were identified using a random effect with identity-by-descent kinship matrices as covariance matrices in the Cox model. Four Ganoderma resistance loci were identified, two controlling the occurrence of the first Ganoderma symptoms, and two the death of palm trees, while favorable haplotypes were identified among a major gene pool for ongoing breeding programs. This study implemented an efficient and flexible QTL mapping approach, and generated unique valuable information for the selection of oil palm varieties resistant to Ganoderma disease." />
<meta name="og-title" property="og:title" content="Identification of Ganoderma Disease Resistance Loci Using Natural Field Infection of an Oil Palm Multiparental Population" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/6/1683" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Multi-parental populations are promising tools for identifying quantitative disease resistance loci. Stem rot caused by Ganoderma boninense is a major threat to palm oil production, with yield losses of up to 80% prompting premature replantation of palms. There is evidence of genetic resistance sources, but the genetic architecture of Ganoderma resistance has not yet been investigated. This study aimed to identify Ganoderma resistance loci using an oil palm multi-parental population derived from nine major founders of ongoing breeding programs. A total of 1200 palm trees of the multi-parental population was planted in plots naturally infected by Ganoderma , and their health status was assessed biannually over 25 yr. The data were treated as survival data, and modeled using the Cox regression model, including a spatial effect to take the spatial component in the spread of Ganoderma into account. Based on the genotypes of 757 palm trees out of the 1200 planted, and on pedigree information, resistance loci were identified using a random effect with identity-by-descent kinship matrices as covariance matrices in the Cox model. Four Ganoderma resistance loci were identified, two controlling the occurrence of the first Ganoderma symptoms, and two the death of palm trees, while favorable haplotypes were identified among a major gene pool for ongoing breeding programs. This study implemented an efficient and flexible QTL mapping approach, and generated unique valuable information for the selection of oil palm varieties resistant to Ganoderma disease." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
    <title>Identification of Ganoderma Disease Resistance Loci Using Natural Field Infection of an Oil Palm Multiparental Population | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Ov8d-uS-Soec2kMDsZvOKffEeaT-1vezbE_YGXOEoe8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484774","apath":"\/ggg\/7\/6\/1683.atom","pisa":"ggg;7\/6\/1683","ac":{"\/ggg\/7\/6\/1683.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/6\/1683.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1683"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1683"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/6\/1683":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PG\\\\\\\/\\\\\\\/M2fSREFw6IxsPVDMftno27efkY5aUaSLINBI1zTRfZDMiHrsCbwhXCfSLJCDXTrejiDY33w4WnlBIXjANNzuLeVVFh6AsMm3XiNwn7IpBgfFzDHaDSXGMrFQ+MQ2JyFWE3HjrYSsarnU5X61Q\\\\\\\/0Ah2QeQILH4AKNDs\\\\\\\/zJTPaPETZgn7GftdLeFQX3tZB6qxxuOFrPPbACnBX11PlGQKVK8vjB1onY1A4ZrQdK9VGcRE0uRonRs2WUYbTf+Npbooxw6vVVUwU7Ugpx7CdpwilSddplH5RDy4ZpuU+BhD2NW9RguMpHDsMICfLRvq4egoVhcul4daZlhxDEK7u0VfAncCdg05pu5vUq2IDdFv6a\\\\\\\/e16hDmqoEG\\\\\\\/zY2ikYbO9vABW2kF50EdOmd9USznvKf8\\\\\\\/NVHAwyGQxVBCxsT4B75QN7ZrIji1+X6wS2fLa+Dpjm2kXi1R\\\\\\\/8s47Lsa8mUzUdlOphxu9ERnnSyyHT+CAiZnVylsBAH1pMwTm8w+gY10+Jw5wASPcfyq7R1yNx9Oc0lsyQKTgQyPwr3qgmjj2guDJWs5D1Zeh7rQ0JOXIRPghfcRXy9kMgq\\\\\\\/AdsH71WLHRUNdmIHWiQHgyh7x54uaEACo1JorZh3cLXO7Mk6HJ7DQet9uoqC1PVsN8GMrcxlzLMBiOfF3WY9XPgHFbqP01Cw1DVPnyWzs\\\\\\\/cQQdr1AOzzU23ifUHJSaSeJ8qmI\\\\\\\/XmqIbI6gIdbGSQ8Ot+PxRhf2vtdvhRmiXnYKdklEriJjNyvecizUm\\\\\\\/k6YJW+USeLLTSPnSPJjkgx1FtWFEEPXNQmlDk2tXGpsyLTU438EzbhGSCJ6nQqNPNR9Gq6l1FI1lbSX1eqSBwcObVWEOivpuvXvLT\\\\\\\/4k5USP9aqKpaLB0Wy\\\\\\\/r2k2Njh9BgP9HuLHlnDd1pV43Qdf99ms5UOQxe\\\\\\\/1btqPMPz7JnkCI74hfqGaAJQp2Q1MoYR5iEeAipo33FpXr0yAVh8KfgB1mRTlfIxEmyZ86jgJmkRy04JtQgl3n2xQwDHt+5x4Kj+Dm83aBXJ76uRSpuRMTrMKHKG7ytiNjJMDnxgzXZKxKJ27S\\\\\\\/XU2sOEKsl3zWJ\\\\\\\/ok8TZ9rGevAcce5m\\\\\\\/RS0DZIisLReJ4Z2CUXk2\\\\\\\/DkgYS+UIW69T7Lgr3D38QuhwsgSYpsvVdiXSP6GKA7fCIFPMvudMzDc+s9CW46hL\\\\\\\/EKAhTDaqqfaQmZMEvsFW3stdq6YX+oSfO2vAj9TgZo4sKkPTGyZuTeaYwx+eMtbEzNve5QJzbZQhM0n\\\u0022,\\\u0022iv\\\u0022:\\\u00227o6ICJNXP4MqERE473aMSA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022eb4a10f6c59cc89e86b9e05462bd34d269289f21bf4b053c588f07e5f69da25d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RhJMW9lClrbi4nnjJFMGVLAsS7fzqYXyAKSoqy6CCT2izW3KMDQf9PEkptJ3H4aqnWR8uDpVqRq7IzQRePFVRY0PZ\\\\\\\/EhrOgHId7inDIGMkNsNAEK41ioJvWMQCkXC8kCraqwTX7WmTuXOdcFBLG0NlxbuhNMXAfiwhIF+6ID53lEB35QR4HkRtQflIw+9zXBVUcemFocQW985rVwXV7mMRsfRFW4JCypPI+cxLR2Xi6n\\\\\\\/XWKhc4XrKSMkunR8k5woBxlmB+6yCIG2zdzntxnIzjbSXno+lNuoBGhTxdRI8dq\\\\\\\/N\\\\\\\/nuSzB0JPvIX0sixhsQs4QWAWCVd0L2ew+p9PZTmqVH5GGqB0AGqG7Cep6V1niSsSjhH6gQB9MqnutEr0afSi6Frw1dmksPnoOb1XBB5yHcVM8oezLoS4t8Mhwab0+6qYC2esEhE01siLirpUiUT+\\\\\\\/OF0ewQkGcUG5ZgabEJYd3Buf5TEZovREOAVQjo9cTUTa2tqPr1m54T9feyDTocTHRejZeAFte36dUwMJJqHU+3kMgqSfvlYTpoOxQ4627RbAw2ivASlZqhhHlNIBauLZIp9KS7iRREhrnNzsKiICg4IHZpx1dHstIqBcv\\\\\\\/coZToclJD9IU4osX1r5D4KsFbR6+bJ3T0kpm2f\\\\\\\/lWs\\\\\\\/WkFqmlauBhCoSkgiG5urgTqcl6PwmIFnccxr\\\\\\\/3avVIBLF17eLP9G\\\\\\\/G2Kaq2WZw0Oh5Q971h7ch8tCs8zqKaT2CjhB2X\\\\\\\/hN2CGAHh\\\\\\\/JPioj0jZOGN4h+BvtpACEjG0njTW8OWj\\\\\\\/jtI9rjhr9gOXkpUXZzcbQ59Yl\\\\\\\/7KXDVdtozngDgoo+kG7rTy3HfJ1JgrPgLELq2q961DqUKjqwuntfLzlc4a0olFUbHeSvfJeC\\\\\\\/SsI3xTcmNq9hm7AQzqwcFcE1Nu4ptObedr15\\\\\\\/zXixQIJRod+9Q1UI8JbsvdZKbKxKPcPsInG56VYzsf85M5tNXambNm3Yf8z6JczL1B2ryEgTVMdpKsTNRGKroIREGvd8ohAmpe0FnV+eVspbWg8\\\\\\\/PNSzboVebW3w1udDyxCQmDukl1aJb4EKJzRgmGuSQRKTa\\\\\\\/z7\\\\\\\/VE9PhlcUUDxuj0AWoie1Izy6ziMBYpqbuEVQCotcgCBJi7AzPDYiI4VZMv+uMO2ir0BTsTT+IKYjuHbk7vms3Uh+7dm52x3FYoZ2eWbhV6waVwxvCnJMpzSEnneRAFyG+RcQysXpdjU93ZhRhtQvW0lVZqLOco0X7gZrx7brWM+fBdQOELJguCCSc9GJpgQEep+r7rgmuEQGZwoK0GLilnJtA7eydnx1UlwUBU9b2bV\\\\\\\/+JGEYskCQTV\\\\\\\/XMciukl9t7+44tyhUEZ0e\\\\\\\/Yz1o8FpzGMSMUiAakQnUGe2K358jIEIp8YeMde6xJDtsMQ1Nl\\\\\\\/+QPvdQl5PWancshGcqlCZqBM3k7\\\\\\\/aVbPQAowWZouWUn2WxMiQtEbBKI5ZCULSlxCquidiqGDs8E0ndFKoPc2bJTOBR7bza8FUuaH3ERCs+fdcRC3PjYGEzerOWme8SYK1qYbLFFo2k4OpW1x4LU0\\\\\\\/WIF6\\\\\\\/gEGc5cpe9wDU+y2dzIEXXtdOQ+du9quxY1C9Z9NUdJGEp+wUXCioMDyAN6wp\\\\\\\/0Q3hKJi5PIsexo90FG2CCSKNWK45UoiE\\\\\\\/raqCFFvWgcStRdBwdsjbezBEvJe8tpalEppPdmhLW8DBcghTb5Ufr5LFKGqEUDcW6onZvKU\\\\\\\/J8wVucHqWDcUJ1ZATFJJq7Fs3xLFoKLNbusosKp59JJ+YbXBbf7rH7rdihybGMrJVJsI9\\\\\\\/f+L7hM3W2J8EdH+AxK8PQew055OCnhAmhFz7LaTPcftbDzUXj4ol5QAy93mej1nnOMwZJrraP6EW7Yr\\\\\\\/UghzbWgOCJNYku4oH7UtWKlDCqZpDaM8VYGkRENcjfD5MMGIH2fZI0sqvehEiD\\\\\\\/V2RoT1zKOS7QZxWaTx\\\\\\\/iIbPdYPh\\\\\\\/vbRgptnEvH\\\\\\\/jBkRQjTOJ9c8+D1YNH2mp3iJVUi0PWJMv5G2r+ef4LMG99+8RRgoh1LOYS85mCTvGWDkz4woQ4p\\\\\\\/Ehx7czN+qGj6+ahuU+piU7GGglRCXy8CorjAAW73330gmrlpMGGw\\\\\\\/U+43tXPWogC6SwCO3LeTvvECVKifATta\\\\\\\/T8ju9vKcdvt1ghOgM3yGOvQ9YqIBxTyRg39m5lwSN9d1WNqxUflNW9zeD9o8GCzmBiOzvlSEzMrVK4gV7Qz5fW1gKRh+UE+HiCXUizLgw2cHwHxTQcF604oAP5xr+gXhmCzpSyMax4Q2XXT06klDF1\\\\\\\/An0sF59sGngaW2f4mUCD4\\\\\\\/sU2MM6s+OeI4eiAzKgYiP9D5NUu6OHIZW7w9Dw0Z4YY0Ft8z+vwk\\\\\\\/ALFM8TGIcXexh\\\\\\\/6tpOlmNQ6r9x1rBind9G+ms2aZBVom1kpRgWBKN67FQPO1\\\\\\\/qzRyHpQVrFvJdSpE91ke8aqJ6pa7yo\\\\\\\/9Jv0SDTe7DkoNw9zQbwf9ctyVfblidqHwDQEYAJlbI6lmS10OZWjq4KZWaYp83tF8g5Y9cRSQAkM\\\\\\\/U5y9MNtai4Ko1GE73JCpNA9lr2Q76HXRiwpgk1iKpho\\\\\\\/LGD+tehu5dq+rOFIyElnAUOUJvBictFsekF1z2c9OOC7Qdv3o64L0Mt+jEEBUXAEZoBuE1jp4769dMQ4BTQRWgUBvJQ\\\\\\\/hy\\\\\\\/+Wq8ZwFO3LPVI1ZLNgTmigaSyJPKrNnb2IP6DZ3pbgE7AFZLhCXKb+qzs3MgxNd5ECxau29AbD4Xi6mAYbLoylLpkgAd6tfQnBPzjWqRySbtV2n5JDaxtopQx4+vOWHsEyu8hTW9goRDBYrfPcwrQIlLuayxHSVAZt0WB1EYTHCh1rY5p32GeabnrwLOk6Ryvs259l9G05pisEMSVhGi\\\\\\\/nlEZ7P42\\\\\\\/zu6UUnrB7EjZPGC+Yf\\\\\\\/MsLBi9ZPKvzFqbBynfgdOC0ewQT6TJl4uBI8M70tFRetWQOY3Un+rU85drwdP8fa3bnx+Q9pH\\\\\\\/gn+xmnpm97u123whDYUykm3xA4PsWg5QjHrxiQo4TxfyLujefyipnCBTGPedPCub9i8SxT9+EggmPS9SPKyot5yJAjQkLDdFCMbXOoO0yXPn1k0qddaZqChLTw8AL7SIHjaCRaCQSnBzoreAuqC3NYhH4MhShUBOy8tYMW\\\\\\\/UkLwD7EJ9k8CFME0RgCrn1HhbT88T6WMirzFaSvg3IMBjRURmm4dVOCoe7jvkgaN9zkQ3kjlqrbP4lOxmdWnLqdXm0IklXnf+Ft58JlvVvl5XPnI1CIA+wXsaY+h6EF6rZxezpxevMmbERBhwJbOketBNkf4ko9aSompI9A1esTrGTzyor0UdTR4Bszt05TR+WavaXei1KbQ2ZwAzSrUy0OaTlxKTyN2d9UMfVRoZVgfVeW4nb0tAUyh9EHKNIDSqFNVYtoK8mK8zSv04byYs4gMxsw+XarG\\\\\\\/12aJreEnBulJpKj822BJJ7PICw7x5JNfMdPqmyj5ew5GYFYCENJhJ3hUc8wfYTmFaAM82KY0+VdQxaDAj6+1BWWcKfVY8LI9xNenxaF8C0jh5JMpb\\\\\\\/jTRY0RxcY\\\\\\\/tbJagjkE3b+PBQhhx2EnqzHEYyB2lsx3w6OkBDiWi1ohBuPrQ29BpqyGF1PbMf+qlsE6pUf9ZmoFx8ZIw7VVZ0O3bpDPTj\\\\\\\/VNuniRK4LACEKUdW6Kjhde2knpFVtYktalhZc61qo\\\\\\\/omYeZUwzRfTxeslrDX5UFe9Yo3hdPlOM0dyQ0B52cxqeYctexKUKjS13zojstNyUBHDnZQ\\\\\\\/+lmp6F2FKqDlCvfyAkvswQ95LQRck8kvuDNl6ODeJ9ZXCwGb\\\\\\\/+Ty2S8wGA8EunNGihZlc6nqinbC9dEo3XboDaCQEbiYin4YFLNQdFgO4YPgRHsaANmy3VaGnWhPkNZmKJ\\\\\\\/256W\\\\\\\/7pv+sbymk5KxIUAIfdxXwKzlanGlDgKBftE6r3VoRMn1gpPSaum\\\\\\\/4NVnlyIWHX01miO1NlrDRb48mslVOO+W4GrjWLE6xXlckGlZLAcyLbSK5VeuP0XEUDbdjBIdvWyBm3csrevQeQtQf6xN6PXC5WSlLKHkrKad611tXChzheM+jXvA+i1CtfU\\\\\\\/WSsrOQCioc0oFBor29qGvaRLco2iFlrS2GygPl+aJ74XahAlhIuOgqGIx49B9tjCkKXhLeS5NKH+J8pzNgu8Mt8IULhUjj8JVW05BZxAQ3gKivqXat+IbLMYyaTx7yLOPOg+J3B8Clv95X5fwZbhdUidF2TAxKfZTLtHxVDmdal4UjsfxbWjKTA0thYujssgqOrsaIywGK3PeOpzVvSHVB9vC47eUydFRTZiWHnhltDiqQB2XKloT2y646FAkQQf+\\\\\\\/1sBkiIMI\\\\\\\/9PxFVyI\\\\\\\/hndi9\\\\\\\/yY+Ssi0E7Zz\\\\\\\/qUVfWzvJ7NDC4Vvv+X6ihTvgQowh1r+Y8KP0GDGDr6JM8QFkCVOmg0peJFpeJ670I8Ntg3DKQLWQKyXCiizt63dGF2ouK8pf9PUVm1G\\\\\\\/O4RMHdg87E5n0N+Z8symOaCBt6E\\\\\\\/L2UinH9YdsreJpvSJclbAGZlz83Fo5EHaf3\\\\\\\/ePSBUyfTwnQP3A00TDD+fCFDJbkZanLTxilDQLzhCJSSQ4X6VK+3EOfWd6oPcOveSa\\\\\\\/ZO+yHUKwHZ+NGTjSkxgePNyRc8BdYVcUlFTLfcfv6KLsIm0fqeOYvZF4f\\\\\\\/j5ubfvYcrzPeqlQPtrwL2q7cvOPXjPLpdlCgr3AtiB7G6luMQMYRJofTIZ74XSRSy7GYG3dlXsayA5dhRw28bMueo8k2jNT0bCUAjOQ\\\\\\\/4i2oqjYQbkTs8\\\\\\\/PmHcg1JIUt8b6P6MyX781V\\\\\\\/bOsDv0ZWHWwkuWe8VJvXuzkcYC5mfBdTNkdX4QhK6ZJRM9TNmt0\\\\\\\/jp\\\\\\\/rk3\\\\\\\/NgZZekkvwXjTJtJ8b\\\\\\\/QfH+ZbsqgqWrlHFnGv+bfOBlX\\\\\\\/laNTzxbk30l8zfE1YuAWyCJPXx+4BIfRzjdyKeT1ryjPtPbUY+c5pgTLbX7mAjU2hFzEqq1ORzCVE6leCsY7XBkSBFpCKOGcDR4BlMXSeqLaDQbeCK9qqQ0NNObJ17B+CDyi8NSIYCtg6ArMfP0QFon71mUSb\\\\\\\/z6zUknxWTPo55nttaHqpcT40myImbIIqlLkEYX1A7lJbN3PwS9yrd\\\\\\\/XEiTK71UUi3eC\\\\\\\/hvFrPLXWaSM82KGlOt2KEZ65Jjny31lz8XA0314tszOD38NwzXfw9I7E\\\\\\\/4ohpCf4awloOqvtxi36aNCxplHKfw9k7Jc3nOkai7riJsQxxMvDZnWHdvExzygKXkcI\\\\\\\/h4+vlvPgmeTmrSEyN7PapjdKrG6ExqleF+KSrsyrL0I2uIHX+DuNAysRdrDbHeOxkLFN3soZQhfZmgn9esNFLK6CEXmQQCDp1jinGfdJOpiEz2y8mlEzv6PcvSQ13rR1SN6OmZoijVGp14ysNqWSl5MAxz8AGScLXNcJUXmeXHigMKxTryTZvpYb1C4AngPkwpJ+TL4+LUfSJnnk+z\\\\\\\/5JiFz\\\\\\\/f\\\\\\\/\\\\\\\/TQYv2brl7ZNjDF1ll0pWzYYNUpKO1\\\\\\\/72sKJF\\\\\\\/udm\\\\\\\/QgGSkOsGmMbH0lgRCeJVR1USavPsTd1Ps\\\\\\\/Kjr5ArgA5MFW5umHyGy3AaKYiQdh8yw2PwL+iy9a6G6rc11cekNrDCHjHbVIrnwaFFZ\\\\\\\/MIJ6KfXaxZds0P2Pbjq\\\\\\\/n9w+dKCXgxch3ePkqeRwF8SJ5Z2Vsqj+SkvBODmjRQKR6bcHZWnUfEsRG3B6btjA9n0Y\\\\\\\/e+onbXfpSuL35UdJ632WWtCoJQ+bBZHuKv4BxddA3h9gd6Zf3u\\\\\\\/n0c89kTREZPxQooS1CQeb\\\\\\\/j\\\\\\\/gurb76GQfjowZBZVwJMDZ1Sq3eE49TUhCukX\\\\\\\/nJYzsBEpWpnrAh8QFQ7JgbItxxyHLzVC35jxZWJW50YU8grAJOi3O9dFSEpW5XGjbFm3YFRNj58uYqv\\\\\\\/TTrvziU+VWK4wjSbdTs0GxOmMEC9SJTGUAYzXb5RaIyCLsjzrrJnWkAES70ehUpnO5sU+W3t1rMx4Yf1sNAXAk7y8BlWD8ERD+FXCOKfBqSPvBnW1OaZCk0MKtg\\\\\\\/C6CbsGn3r\\\\\\\/DfH7VGSlQcD1GkXch7I4EPEgKm6StTaLqqfqoYEflFpziZIRgBKswPvoLvMk39XiYRCsdb+ggQVFfRvBvig1jo\\\\\\\/XCZU8nJfkLWgBQL+nEBOUtJoH0satg8hm+Ds0VErQDU6HGtVkn6+EUTjZYXICySh6h5VU2mz+JAA04v5+WVhGRh3NsmLHeIq1G4cmuTWqrduosDPfSnDiwZD2ojbN0CGCRlwZWtAhMcCPWHtO99JIXdETG+5T\\\\\\\/SIpsbFjRR5sNw6vqL\\\\\\\/p7Bqz7h54e52hFopXgW7wRW0QNm5R9FesUpVpke4MNEP6yOnNlvtLYTziEBslmvBmArrSMRc79BfIfJ8VTrjOJQjYwRzg5Kv1S7BIYgDC2G5NtKKgFdt3jM1Oo2YnaCsm6xodeLk5Izg6oI8qPEG9zdQhelxE6Xf2zrPpMNDYSS0Qkuy89H+Fbjq1BKE6C6fXpg6jMZ+D6MW\\\\\\\/iIb39AV86gjVP7qK\\\\\\\/vuvpshCs8zgYYzeNkVWPUGJRc02tJBFyiwrBaI2oonN5LEmnSTRLGXyEMvm8GHwqVo5x8nOgNmTa8WGz4JPivK+ZTlmKscyUZjrvxr6CX+0hvDeDXVsB9qBAd7Wgkhhjb+tlbVXqGCU1Aw3QrzTT62JHvVE+tpUustZE\\\\\\\/QSJ7Yb0f+ForlXNnhGnl0D7NtALg3ZNc2TD2yPxraDvf9npq5Vr95tWdZxNTudPMzPCnB+21dzYRRIidLdyXO0VNhHXb9GYOkYVFcKB3ZXbHw9U3a3l8LdppIfa+hdHbOvzU6Orsy3rCz6jYFEcAAzk0\\\\\\\/iXpMM4o1O9vo4LKF3s1kMv0q1kNXUH8M7NCUGBgYU+YY99A4pGSqukkGNZDYVlLuyDDlcQz2fbv35PEzdYZB3GNBAB\\\\\\\/LqpHpHFn+pVKSYQWDvQW5x6sYCuXW+Wi4egMUis3Sn5TjVFV58oQOsE+GNDxNyuQ4hLeStah3kgYNpedbr2WACaPO3l8u1pfJYKPTl4OFOP+C9fe6T9e7bw3U+XLP0OoqMZOal2AHfI63CDVza+XH4CzvrqsSWtJm82jsG92b4dowM7A7KaN8OhBXORPMOKVOsc86wIPWCmSPSiP+NEg+Yds6BHuykHvn9oY4\\\\\\\/ljHnzr1VRhTBJZl5iW3QtpF0zvGXYjhGmBNiw45ah6osucLI9eySEz7Wmsj+TrxQ1o2G+Vmf9fLyw53qNwLwwHeuuuJqwFBb6swF+NYvFmhkbK0U6asHpwAWpfpBrGHrD6tmuKO+INSjDWnO5pAvI0zDMHQyxIa5U7JhAje\\\\\\\/601TqDwkW3Luu0i3fCS5S+THFIeB1tI4Yp4IIwF\\\\\\\/asXN4aZCW78TiXtey0jzI4JV4evYJq4n5eQYdo9iQZk0cpAywaRuZju7t7bJOFaFMJ0sSVLz78rtQgjbatkww37DpvgIjfBsl6M\\\\\\\/R9Xrp\\\\\\\/BuDv6l2RwZRo8k+REX7BiTrHfoix\\\\\\\/V5J\\\\\\\/TuAE0rGebiGNLv7FerDJBCYztVLrQYSAVVnUwdaxpWWVbf\\\\\\\/hynR1grxRBA18BUsHo5qj7+Pqb1dXeFaMaGuneh9Povl8TZuwK94Rsn5I\\\\\\\/w4U8sakeVwEbCpdshcKkc45lIpeGtD+0j8jIvoQRvZQltS31G0RgTP3hZpZ8o8\\\\\\\/QW6ZQTXvSwxuamvzuVpVsCdYKo5MG91WjIhf5qFPc39d9voYvsU47vfmYwL+E9tenTkb7s8BItArTgjsrnHegfsJ5k4tF0Q1+zAVazuwpkLicDJ0TKi4v\\\\\\\/0FRnlQVn1wruDMktvjaEulmtZ8xj+n3sp\\\\\\\/5oz4I5uVFCQNx3hYrrm5o0gPV1ha2DHC4fPEuy\\\\\\\/+4lm95UdvsfVVS5tgVVOy5dDrMeqt2j7QHxBGYwhQ5DSy59Ki5fRsOUW2vqhKWetlsFVYW2BGxjb93K0grK8SEfjqXCPLMIMsWcmwMvJ0gAN6emwNJOEAKNRBvIoBqBTHFuFsGtZjIFFNs3K+pjAYLFvT8MfK3u1waFvm81gzc1TXZKMvzC\\\\\\\/Et5\\\\\\\/++OE3OXu2QhBIkpis738jzdYvNDJkwyF9cJGE9ijo4sbeMp\\\\\\\/zSjrhf\\\\\\\/AHvn6A6wk+OZOAPn++jXEo9PKCisdatODNVttJ+YJcdlaYAsfo9Wny3yr8GGOQVf5QywOGKJ0\\\\\\\/hhaojNen74n7KPf9z+bE770Gw3sHXiA5SU932Hrh+2wuDdGdt6E2K0vKgo4qXeWhZHfZK5KIBA2HRJjFFv1sk0IPerhFLmlrfwhQpraTXBXh85RjZ9WrOvvudQh37y3L3Iy8wcoH\\\\\\\/hnppwAiRXByTKP05jwq4WSpbknSOlavosC8jEwL8cRqdYjDqC6M35Lj\\\\\\\/kuc\\\\\\\/kRCOmUJVeXPa3NWmVT2UXxlMjZUNlSoel7HEVaLIRc9glOFhN1DBr7AuTD+QzWMDVIuM4Lbhs7fJDsUln\\\\\\\/bDgl+7aNMqxtvpLCvXt00jj4zKpaA6FJ8GNIWYEccSwVCl2QU19Hh5UPUizW94D8IUVlT5EEe0O9YR6gOVE6T6pkcwRW8SbnpxziPTAzRYz\\\\\\\/6oUgmjB355oelpepa17xH99Rs2ZYTqCIdqtAFKD00Pt89lX63D8hvDsvTaG3g9i7rX3b6Iu5kRga7WjnKhRq6tjVWTaWrRp+\\\\\\\/WUgrcLEMKM8uo4IJW5dsTLkLMpSNz7h\\\\\\\/xV2ruk7jZsN1Oz3un+8\\\\\\\/+KKkKnzhzXKrMlrKJEX9WkfJfwXp59J3ibELscCY6Q1YYLp6bsFIkFvDB+Ee7Z5l+rwpfQqITqtaWPyKpdi+ZzCZJrLT6ddlGMI\\\\\\\/PlGBcmBKli1uy8dcxvOUnHi3kyhd3ZYakMUvzV0VP3cF\\\\\\\/NsYEL5xNU3msKdPVw8TQUI\\\\\\\/FEUbrJFSDqS2d\\\\\\\/vNQZreTp+6SGcuX5\\\\\\\/Kx06iT85OgnI3\\\\\\\/ffqedbtp13sCPEKKbrKXzi7FSyNOpS0f3l2F9p0Yi8XfLIpBDCz2SybSDt6wsocHYISznNoKdKeXY6Xr\\\\\\\/7iboSryU89WelIs9MINmQQVfFXTgCB3AMCOpIgu1ktCV7lXmvhtNYMKIXDKlidh\\\\\\\/rOvaTIMyJprCMFGkrC\\\\\\\/EaDzullUJTlCIjBz4sgod5dG2DOCPQyoqQqlrJOu3ANSTBzpRBuUs+qRAZgO06JIoYcTyqo1xeyRURCSYWy1z2FXJ+EN4yntkPmcFMFTyG+1O7U7Ri\\\\\\\/+pEksJmJ+TAyjbKy0nLUgED4v7QTVhXYa0NXObF37n35dWHNsOGeAXT4lpt9NjiTOmAbKVILgkamXAa4CnlOgllFJE7m\\\\\\\/QyZMeN3iJ6TyqUV3vpT1Mbe4MYLEmFWn05JdpUerXB+6ynGpbF4eHe4kNO4mFcBhRb9\\\\\\\/eL7SN6FIMJR+iJZJEVhp6Sylzt2llRFXbT4YMkOTSp6fSxjbN8ncruFixNzBqLDYDSjTzi7BRId40SCVW7BEiYlHdaxJ6yGQ9g8+g0CEc9sLb24s7SxkCVYFlFhmeGdgwGhaSzMFld\\\\\\\/czpiWHsIfqsv3xZX5iQtSCQgUteqigFsn4yt7156OMX7xdTNrG3gC2Airk86Z8JfofHiqFn8JZWWM5sX0xZKpiZUUSTvTXbQe2VxZmWxE+EOcpmcGyvhmsYavXlLajtS3jGPkonIy4TDOlQQ1xv6oKb32V5JuEeSMgORbqR34LrsUK9CTfBVTVwNp0MbLppRA+RxrHJjX0kkDCJEFeTMOuY6UISFnyzUqvJW5NqOe70R12Gd4q07SkOPUJzrKtzA8G4giUQAQTpXvYQ+qrOlf6rYzQL0nyD\\\\\\\/rykGQsq6FKgE3kpQhYFDAzDLxNBtsYzFdYhhLY4iD2V3tGCXhdX+pfxtYnsnCBIoG40uHcbk6psgXGu+yIr4hKkmBjBmHwCW7Nw8qz1W4G\\\\\\\/k0z2kIphhaNMg9rwycf5Uonrp5B\\\\\\\/srgbJQdXuiL8+uDBHE6bLdq9TWVDck9SjnsKvn+sdIOZ\\\\\\\/QkH2bQRR\\\\\\\/9YfYbXddXsZzsDL2fEdSzJykIUeKYmdh8IXsIvHO\\\\\\\/QE\\\\\\\/RhoHF6KP0Otois7izz3EI4RuWYWhikCoUCRjWhJITiHAGsiJ3JkMp+s3PHsErepQsF90tOhmtfXlGKx2DBKkLsQpwbYKMH8pQkTP4H3hlB8etzk6cUI9FOkg4\\\\\\\/L9v6ZoYTDEJb1HxlRIe9ZsENcAlz5RMR+J8qpgFC1ndcAcuBF5ZbwX3vHlCs7\\\\\\\/I1+vPnz93lCvDUbQ9SqBnkETsCQaiTE\\\\\\\/Snllydx1CyvtIxRpcjlBWWCBZlfLHbjeSwjAgNvVXQE0Ev\\\\\\\/2jvE0Z03w0nFZ0ywEKZhdIaWv03DatFPaeuLqmkm7AqmbRlmqScCSPzv9WxuTd+\\\\\\\/w+QNMtXGCX2Y5DEWx\\\\\\\/rZcZUcc+x7NcdRBBgFhYjupPz68lj3torWdGLw+5nYZ4GF7140nKofIinAzYofLKmvxDgKf9pedFagqabhmO\\\\\\\/O6f8LBqQ1cIPmUSqdEc9F5PC9VXerLnnISD+caeYBugqGyjqsxBjn8PVwmwIUAEmeAPCK\\\\\\\/DQpQUXZIcAgGUQznOasan2lO8KCE2UsT\\\\\\\/8QKWIqOiWKJ1voRkrqbQ8dP7V7t7m\\\\\\\/rZmnYNJbiqqbzNTeT4sp1G68S4eEWIDHayhXuBYEGMhc5fA6BQUy1ZU6RiKP2WgFiJSHxdX39bM3GIb6LCCy3LGBbvUEfQS5JrZdXIWOn5aRWDUvBZvZUuIBOECVkmoNUmmvioGp1tKOkN3mhv9pzM7dFC3QumiRJas6Jw6n8Bcuv\\\\\\\/NMFBq04j3NqtbM\\\\\\\/FcA4Dcy8LTEyY\\\\\\\/1Xqhv3DrsJjGJh+o+pTajLugkpsNowQeLi9dlqZcDgtDgPDTv+MxuGuukGHiSyiivXUje6TJi84ewL8UOOE8kkP1BXaQ+hF0RxjWg4yuf3XFFilN8f9JB9xfJ4m3IbYzoEfiqmGfdkQOnA1orZiAd\\\\\\\/9Uv89gZyzTMUCkClmtbxijX164C16hNiQx6zWzXlRu1j\\\\\\\/wm7xt7A4hQl9zBmmh0WeDZ+witMjLwgXlmFYJ9maFsoMIRyAIz3Xcdj1PhTPkC4Z7Fp2U+Y0mGcu2xnWmXVq7nNEx3Rtv7MeiTrONYs67ce2xNQFKDI9zvo1OeTngUy1ofe95GGzqEmH2AS6Ek0kEMimLkcoo+pyP05TsmUAo9sNlS1GUxueiH0fUoQgFKSlGWyx2K5\\\\\\\/nDzKUfsEubTR1uT05L0Wjm9zGaIa2oHhfci4wY0aBMOOnxTOCPxqlufOVBMB4kRCtCH1ipuD\\\\\\\/36v1mSILOnXM0QG91B4Y1hLu\\\\\\\/bEn1yy4x4xaKFuPl7dqGWWSnrnrZk6fErux4NP6ELXKb5cbrzSzBg11AFD9pxymqEDfi2pwBv\\\\\\\/2dC3wHc6lNk1AcF5X9JHEma5XdJO1zTW+5Ybc2sZeoKesDEzWmH8QtN+dxxY002pt0T23yadlNH6wzqYG8833MlgigB1Oo9zVBs5P8eKKv\\\\\\\/63ITTdXH9GTuBlQt+PqciDxuSRkPOuJyU435bio43CK6i9rnd6Z4SXVRGJTlj7smOyND\\\\\\\/aNmw0L+wNTZb9ViocXJDYZMRGVXbuRvrQpPJr33rGD9HvLi0iRaFH+lChGQ86ww5qO3q6oEeaAttd0zy73vAW67mRpjRKQJxCSd\\\\\\\/A6h5IF4fY72NINuyP21DDgFooo3WQd1ULXDUyT\\\\\\\/gtheD5lP5BRi5\\\\\\\/WGYw+8JK4F8US0a+LjVTiif5Rvr5GsdC\\\\\\\/WeQFo7+ELpkpbju6w4btR9AXTAfMiwAAfsvl7ca2nPIOJjogooBS\\\\\\\/N7d2lNrvbYflrMefOgvfBnrIoY\\\\\\\/KfptjthDzd3a+yhW+vhZM9uQMRWBTdMDQwkTuqYz+yFlY8dV55r+h0m\\\\\\\/0yKR8cYpma24zQSprH46tNj6fgz9BJOF5XiDWpMrkJ+RW4IC2AeHZAOpHuv3CBR4qbXJrUYt7K9V9P6urpcYHAktTllXGqvASY5zFKhtwxKHi1+RGCQw72zlhInjzqz6Y74WF9ccFfzfOUupA2Crno0dVzx\\\\\\\/9OABDvvwmDFjPtBk9ksdTq1cTLXo0hNnluyhqy16s+AEqdkqKxCe5ldtTkINFwfjCOwitf4TQGPAH++XgZKrj4Pw1p0Ns+msaJZWQpX8kC2QmxTUgeEPJJL5g1+8ddE3dod1TrrW+TEDKwVFtbmF+z6UjP79qhwqitoNTnsXY2ZOJUhxRkWQ2kedOq32y5ouGKJ4wbUZB2pmoYXQpuICuyYCPtf3IeEHmaC72\\\\\\\/KrTGvL\\\\\\\/8CkwurPzaVJwfmsjxQ+K6BMFRF9Cup3qmukd2CbeAcuA1VfN4SHxaO9OQOJaDB4BlPsSj7FsN\\\\\\\/nabLP+POWz6SmTk6hO8chzVtgME2AQMl3qgYfXyhb2mcna6rHUnbVpLU\\\\\\\/3IE+X9pEZclf1SnjJ2AUI0dEqfVQl9GxnTvhUmcyN95GnAXkYv6toiqAzDAQXqBiERlz+nDfjtERgyza5SzBZk9F8wuYDIZnavm3JpSgWYBPfiMW8kNygigiaIh2khCkURUDziwCB60bbS5qm1yjbThusoTnXq4v6wUeffx9JPOpOon7KObh5j+JsrqK1nq6+ESeaPyyHhHIJOvOaGkiqrbCU\\\\\\\/REW\\\\\\\/\\\\\\\/U6jXA+JBLqcoYNBIfBq4ELF4oo57r0PKlPS43gjrNVlq1GSDjB9T4\\\\\\\/SrjhcegaHShSZpVzCsMMBbTqLe1PtPeG+GDnTU6E0QM38i2m+GGuuk7AEi4JeJlQzqptOfR3Zd8HTcM++CkuY8T35dFXPWbjU4IfXN8hsmJR\\\\\\\/zW\\\\\\\/lUgiffHLSsevf8caQ2fohZ7CRamUcTVF5XidY5N5RZ6Vu+K8HfJjxBATcrRdWaUoFxBZZ9OOn8aHZKeYIkdt4Tr3HP0DbUCCv1Rmi+ECl6k1+IpTWW10\\\\\\\/jAFiOShFqPbdVMno3kgmqCFngUyv7ELpqv\\\\\\\/ojKypMwUrO9BC3FKUUCokbtpcyoEQ1AJSLX1VwnHhJe9l6DhPqy5cC24oeGFCaW9aTsZXwPsFsajSCrJ08xkeMirmjRkYfqDvjZLUR4ZrjzjVQ4vxLs\\\\\\\/D3Cpn7C+C94Y1fGHShyIBRsv0VjcAUXYaZEiuowmxzy0bHX2kPEhAxJKTtYK7yz31krjxKxV3SPT\\\\\\\/cPNnB4a1BAPiZA69HotDKWOGLmpSu0B23Mb3cSyB+PZZ6jsScmzNpXgjzqcwcoGCBZEo0k6zZ8RxAND8qmLePt155Zf7fKVMkmkquhqRT3613SW+Z19LG64Her7boaH6zGe8AF3dAe0sunaXBrkH1mTPrDF4+rD6tA7UdBvIebuZkkv7kqwBVllLBUMG\\\\\\\/+0mHtZM6Sdz7JZIGdYE6MMemiUzOmNdkDfTp3YIf65izICF4PUuyIvcKvR7w+zkv0DOvOS1cmYJh0QI6C25vT6922bzaIYo\\\\\\\/IQmNFBNnLsr95lLdLM0dOpvLszlTmMWSarOJVCy3brZpM20ZzP2k313U+zGwQn26PzRaQfLhKW3mfH3KlnLACF4DH9Ks+tQZqHU\\\\\\\/vCoJ1GG4NxX1TLGGdLxevct0SDIws8rkNUcH98as0E758E4iLIpkcRS8NCBQulw9OIlE3HTkutzX47OhM0ovB3iYLLRdEZVNaha56anu3d+PCol9BilfjPq4ZVysKk\\\\\\\/36S9HkSz7RB4j4ai4os2X\\\\\\\/5ZUh61f+2+I1kadw0HEtC+ByOWZaXp4UWqHIaaeva8+EYsLgevgUQatt\\\\\\\/T8vSZgAy4qzcfc3sf\\\\\\\/Vk83aQ17m6fios7fNv+YH4OgIZSn1Sc0Wh0kmfLSS+9OgfuK7TNo9CCEAIIM3lOL6hAcWIK8Uy1M6hYEl+CdOjDclm79nV96aFTu1QtnFCiNOVXygZJiI9rALEmBs8EdCEqsPHn8Omo2p2NmHnAnRIeKlvGaCBbvz+tcLaAurZLCMaptyxYSJNNi1nRZcuK40sOa\\\\\\\/6zhWocYG213FavdKQNvdzyoeqPsIGqFuFl3tufzyOC42u4Q9O2J5aDKgEdTRU683W7AtMhqlEF5IDmoEgeIVeAsyXpqhC3lwOtcVUv5KoXnxyl3RqDeiowo1gusgm4mSnMI\\\\\\\/rGhQSVYdGHPNiEBTQYxkyzL1XVm9G4oWm9y0wQgnxa2\\\\\\\/zPRe2niAt1ddJ9z\\\\\\\/VBN3NgfDj6IeQtUQZqQ3KJu3H48\\\\\\\/C7w9cLPu6EScfHE\\\\\\\/6kxQ7sG\\\\\\\/7vQ4PV1SRspiGlnLqVV1Nf5N+de\\\\\\\/Sf\\\\\\\/XWVCt1OvqPfQiduESxndbl8K30Y8P0cNYynXms9bZeyvGHkWLbZAYtRSxGQbTxePZQe0VO983bRMHOF7vIcjdKD8pDfUdcFvXzq3qbtXVcFjr4Tcj1OITID9L31qzlxFFJrcFx4qwrYMliUnLi7A24NJMCSqZwwJKon82gPX+BQwDSCDkYzWO7HCDe1MiDVvu7IdJPwNSyZqaoGXjvQJhFsQwrwStrN6h\\\\\\\/PopQ1L6vvsE5oMm5UPD3bNmtfdUzMp9CpgZ2b1Jxa6tifv0B\\\\\\\/eSyu0PKs0+ystp7HRXOTELrtRlxxVHt9m59VL5pN1OVY4He47tnhOeFNrLhrkFDtaMwuDGXO7S9XejtCe+pY9x86\\\\\\\/zMGvS4VzPHAlIrV50r3cfuNyi3tLu1z6lJTohcFMK1qsIC0LxtNvOdqScPEPLTEp\\\\\\\/4MORD9BhkkFuugIgO54QuOphhiMqmLs5iz+hzhSbhQmnRxeT3OMvD95KOTqgHhmlEsTUmjIDALyrfxlmIUsVc\\\\\\\/Jr7J5w6dHRyTtp\\\\\\\/f69l7unG6836VN5LAh+cYqCOrfAexdUDc+FkL0QB7qL3TOGGMTnul7GumiEwFlM10MaR4eDjqdGsIINvIR81dO5Il73XKCnJyry32gk9f1AistjfZGIMAN9OvnlAPbFjvucqL03G7ONzM5F5BnHifygJo\\\\\\\/yYLieVd1T\\\\\\\/ubugvPFuVhiewE6BJ8bbZDC1k+WIeUN\\\\\\\/WW5W3WhquKejGrM1SvEzpCpUXXCC9amgK2VkjBL15dzTAMQUqfi5N9e0Fdv1peu+k+kbiVCi7qrZX5wr8QEoMh2Egll7vJYIKpSKebdgZNEX1vuI28vYHzwdSvktQFYIMc70\\\\\\\/HU2nZUEC8EN5\\\\\\\/iJqR+\\\\\\\/pQ5F0cPvXZYcQCiaKsoePwT+ir2gVZi3hZLA\\\\\\\/opkFHmnROUn6NVRG\\\\\\\/kNplKAkgJsV9nvEk8Hr7moOobcam8xzqPH5U93QwnZWi1zxmFWSZwl2cHCm6Kh1ySK08Jk8XByfHxUAygm0NqlgXFI8Zo\\\\\\\/U\\\\\\\/W0YynbzLKM7mcTKt8B\\\\\\\/ePz24VRMWmEEhFMhUi3+\\\\\\\/WPq28DLwy\\\\\\\/z\\\\\\\/OBE9PUw\\\\\\\/u6yUXnyYJPgbJn63ij7cMYiJhuuOH3v8zv+aCHweuI6YnOp8U9fKfSwTOwtj\\\\\\\/Wi7BvnXhwZ5fmpo5Tbi2Gr82ZpK3o5b\\\\\\\/W4mp+GtE6dNzVR0f\\\\\\\/Rmj8FNIhrH0gEXnDNd6D2XIZP\\\\\\\/NLD9P3UQIYXdIKxkaZmC8es1wsybUVA9+CUy0vnRfs3gRK8xBxGLNji3Kh5VCX0l93uRFUg4udu1mC9wOqYjVxK6WOrQzbKcnYqvGXW4rtlYHxq4iGcuuHNsqWarwtuMtYVB7M3GpfNcRCR+J2zlU8emfKZ9MdLsEO18FHpnuJUwbQ1ybfV+StqJ1ByIBTDbumrGvZ+O9J8bOCBXWUxwN\\\\\\\/rsM6SjYKEs\\\\\\\/eyR\\\\\\\/yutnA6e+WASiEGulKfyjGep6Yz3El4S54n2RBgoM2WUH\\\\\\\/WGcZBtjCgNY8+0dfVNUt2pC5YJJl4BBCBayV5eLfAMYfNpzzFv3r7C4fu7LFXpPlM5sizSQDKD\\\\\\\/xMnfNxtRttn9YE4PXHItvcHYbv\\\\\\\/w6FmIxd60N0PDE8B7aWF5ZgzolRQY74dPnG859kQfMwpfwL607E1hiNlOlS0yzzc5PAD+HjcqkXjA8ZW4yGCzHUvrSGcCzlYcFHkqswOV3ruaFIWMoDegqdAyoybORqBvNy0F0v9jIGcuIroq3+04DthiktZEUPZrAleDEr5bVHHNfdMjcGJn2xuAWRsMhfIKhJHgP6sB439zDZiq9qkfEMGgHNYN\\\\\\\/3HMUNkW4GEGjB4Y7fsOtzPLEg8c58Fbr7TEBJtUtzj1LVFAV0xrskXfLErVVGTZTkMGMdPP3+om5OaIxPPdeu1nc9skWWdHTLKWUpFULx3y+KfeOKYsLMFjWt2wzQzYlUHtKGrDqSUCB5pa7Rd2pSxT7iFfSUcoBDUWe\\\\\\\/ftnaCtcYCpnamcntk6CDZgIWHNfyBJf+e9uTec8dS5T8luBfCAhAUSuLBM3eFT1QRfV5CXruneEQtQX5DfBy4zLjNPSeKdeg0z4vagwnoEkXQBGJ+22nKyCSAVTeKRtGvLB9BLGZ7n32Wi4cDZF6gfQRIi\\\\\\\/pvykMwJb21xQW6VerYgkqClPZKMAzg9o74N8VhrKAGZ2WO93xaT6\\\\\\\/E6ev50mvFKOfMazuCL4e5dr6ABMYERwp1Hb42EYAY4y+4wUrqrOIk8bU5N91FEY5I1UbUGy0D6PdLUaIiXF1\\\\\\\/up8m+6WgwVCklZc3Gc5UQRZIfCrYgfFKmoq3vZqfQ5fkCAbPW97sR7Oi+Hhc5EVVzkLDufVC\\\\\\\/cWvrG88+cAGH6W5rmoAkmFWpps3z1NGWlkyhKo4VI3A2Hh5JTrGUbxHOqff5Nom6jePkXBwjkMVxkFhQ6EzzZoqgp+Xopaiccm9OnqpUMlla8wfIvr3we2AA\\\\\\\/jUFq2TeEVmLEwSeMw8TeyIR\\\\\\\/\\\\\\\/abEmINONis2HneqqMxumC0g5kV8NSFAdkaHIj8aKikorhJQkwQQeNm9Subu2wauNQJw5+KBvsOKZRDDCeZWpNvTFrGVs7osOMQJVKQ+a3od85tXm1SrqjkmiCuRuaZVytJ\\\\\\\/eG6tPzkYu3YunvKIKSZGXvNAU3bTs8jIzVZkoLryZwDvt\\\\\\\/BFxcOTP4fRMkrSvH6e39DGOxnWskHDkpuQX\\\\\\\/b1IB2PfY942eSgHGLfGO9d8bqgvrLsHKpitHBsUiK3N5Exat3SKLLFE37srreSoLj3VWfZ3AptENkTqs67FpoTWi1CACxQw41UFMXGHHknbTfaipfemfpl7wjmi0SEK8QOyoUrvjcyzadCTBcisvSSjb5dy9g\\\\\\\/AMSrB8S7tVEwlENHZ8\\\\\\\/3yHF86bWVq7mTDwTlvUmBaSdifXuIZIC1khSrQiVnwKvFQT1lBBCD5nh64iViOLWgmG+GMnQZlP+k7\\\\\\\/KpZHIcRQXNz+WyjSjqs9BTlECfvRZc5YtpKh5Ji1+o66TEPR6sFAxtLI8MYjTRqARJa6PQ\\\\\\\/ggFIRA8S4mIrbYsCB6aYWf32X0YMbdFjLSYdgIO5qXNCr4+LZ9NdIou47gls7C3UDbxy+H09FRYPnao\\\\\\\/I0dKjuDBmxJWmJrSaMYTTNrW4zz0Ml1rdgMNffFbFYtxSnfiiDIszAMY8Ndyu4JWjXLVHw89uajX8OlAL6apNcrTGmiBjbWFegVvdRXBb3X9XddkuXUzswXDYW4FlBOUEGjRhROWPtgYTMTkhBjCnXiIz44YY5qipftpUp0SRIro7HvPzMCBenYGBLQTYebM0w3FG4CO\\\\\\\/1broCTpr9JHdK\\\\\\\/NEQqbng358rMsPvyPNTFU0z8sCqQ6NSgD1uujbHHMD7jue3BA4g\\\\\\\/BvmFo8QaXDidrG\\\\\\\/hwKn1mD8v4WXnbEM1\\\\\\\/ymEQhIYyXIe0r+gCH1Lx6GSFricT0XwPY9kv+sa4HKbi2IUi9emBpw1bHinUnG\\\\\\\/4jsnzXYL+x9J3yLdkJftlDqTjU26XHzVj8zwbkFAcX+yRp83qTg0ya+QQBGxpSDoBXpeHSCxzpv1C6IWWnZh5Pqd0JTnPnWCNg\\\\\\\/EBGX4wnw8cKrpdZIAIKAS3f\\\\\\\/w7eICrhHS6fXqTQNpCBH+WFI6KE6jSoQh57pagZpzkburPYW6HZbEbIL3PRgKlwG1yF1pD23o7+P3SOkPBWr4trCVUBsqNN4ir8Xz0MN5AhUcUppaKgaW9idzza1B4WSq2eQBKzwXi20Xjwwf5G+nNF9Nw5+zoz8klehnlvsEtOxyxu6cpyKsbnJWGyzsfrRYbrT2b11OhcoE4DG71eA+SN8zhey8RwFzpyNg9MuRaFpn7iXtFuGOvfI\\\\\\\/xHLuGly9\\\\\\\/l7Cjoo1KTLGkOPtdKDKsIsN1x\\\\\\\/nnXkZlj+QGlkEKMm2JTKj2078LTZr7BkiY8zMvykBk2maNu0xa62yNv2F2sXe4hQ8EzjsXe2TtBomZmsH63NtbNZzVdsaByOSyny0qYSDbarWnyFA3m0HPE1BPzPl4gUPfOLLA+FddxL0sw8VvqhLYSSywP5JdW8QpFdt1CqO3OKOOL2+kLEGVZSqXU4YOoNUtyUBgldg6TV5G95J3+ac6UEYPkV3nxCjh3FEs9X4K4betx+Sp6Bp0+B0l\\\\\\\/Q2At3voEBKV3HLNwNRdMOUzUdxwUTWJCjQlA5MXIFJLEIekLeQCmcWe4Nd8k4NbsgbMxeNZ9UAYWq4GGwnKKr1V6AbE50Ni2IDGyXuRMFUfkP8JcquBcDsvp7h8tZcIDgrCDACS3kbCYN1ml0yDkHuVMwb6LtLBbY4Aj60PFaWqcc17kD0cWj9BfhKRuYehsXBbSMBkVtM3rO4yZMRXlERDAdCmGHkivgzG1+1OjRAWtlspjUkxqGrogZO8vFV6r\\\\\\\/E\\\\\\\/T74XEbIuodem7Q7MqHHRIuFWWuhjxbE1rWNyJHdYluLUnPQN2yQzAN+diKJuV0cryK47zPIUVvtxHE2xaRfrkNEUoT6p5Nd2mqvFzIZ\\\\\\\/Iqk0Vpckhwj8dEPRyYTb+9JcYpRWQ0Gz63CNfVubXMClrrMUwcMRUDLzgho5R4KHhtFjz22ign6fJr997aYteTN8I9zAvLEm5ZzIgSav\\\\\\\/iOsetbV3ejeQnFA92glYkCJRPf0RRhWwRib4hbqPezzr4+\\\\\\\/zkm4L\\\\\\\/feAXmD9qieUCnUD\\\\\\\/HBRRonOoxmf7iJjMAFzqLTJ4UHUl\\\\\\\/5C+6iIKa0+goxgY\\\\\\\/tQ3x+NsDvU99\\\\\\\/U9TfOCXn6zathTNcO6WT7pegsm4GsXs8FBf7FpB3oOE1FfskVdYy6zriKj\\\\\\\/HhS8xqwnt2X5mWucZjMTMBWxM0hyov12YFQC9hkUi+S5mmuz1+q\\\\\\\/3x+IVm9XiuVdcYEhkNV3hAKbl7O7G781B75+K0XfefvXHUBuBWZZFvQl3w+7i2U7eQmlt\\\\\\\/fIAe5u8zayQMV5xLli4Ln9f+MsALlHWSk+S\\\\\\\/lBR6DwOtey8u+ohXn8XDP9LsdbxNJr9pvDqJ+CAUn\\\\\\\/tLrEPeJo+9YWzeot6kjci10JpfHuUiZ4AQiCcUfcPlCZjfuHawjl4u4vOp2eshckvyMmqiZo4VPk796YDOe8BTMIU3cs\\\\\\\/62BjXYFR\\\\\\\/lP73MGN5u6L8Eh58zDweWVGdtt2pfrNZ3w\\\\\\\/3dZ3VnAt6qmach2JAksyhO+aideDiM59AayRfGDP7Ewhpy2PN9K5GL7J7LqJ30oi1o8fMAl+XULUqXYlV5JyGno\\\\\\\/cU7MdOpqpkbRE1T\\\\\\\/KNfNUUjJqWIMCgSLVEM5ssnXRWAIgCbqPyr5hReILo8r6LCn7RFA7zFZCLm2W7YNBOCaDxFQdv8doHGx4fu9\\\\\\\/tdpTWE1ULaIBUvEj1v964JXPMU99Xm4g\\\\\\\/zF6JjEwyNvvfWGmue4RA9abbSYYqHNFj\\\\\\\/j7ubUPvCawcmTNYD1I0Bj+uKXd5YPznN3cpgIpqOhAItdCNlLAEpMgu998y01JX6QGzYlD4+xab9Bfs2W+nRqtQa9XfpLkreXaHyi4eXj7BIeOlsvGtDTXA8a9NRS\\\\\\\/XcHXdmMN05lTDlIH0veg5IX1esx3xFjI9WjAP90RM9csGa6doRF+wf6QQj+D92+TiS8LKtfmHYTM6SVbZ11Vim7jXov7wiiQ3BxPfw1+\\\\\\\/cZkvDPWMNw7MFJS15S9S7Wt1FoQh7YSaxX6a6AbUd\\\\\\\/kMjeseMf10qBytau9a3f8Gk1S7mBBxnsbSYAkjGK+musr6j78mZ+6zPviRVmN0qRzj02RMa2GVHRh5GO+QaPkKYIgJQs981rJ8v1\\\\\\\/PYSvgw5\\\\\\\/arKP0rHN7+yuFPHZDVJsfbJM13QVqup+7vSzbw6NWgGeXkp65ftHQnWwp6I\\\\\\\/wP8DVfGOiMXXQeko9kuGDqdcTVxUkeT73nxzpMjYhFPx5A8LdhSNvsF8TUu6gwX2EfmKWw2ys5AMEyIij9e95gEfjU\\\\\\\/u6kvqgc+S+wfC+jA2K6raA6vorK2XEm4PvIbQ\\\\\\\/IQ+Gf\\\\\\\/yfUeOjixNDWDadahyohn1SzHnk5FY+j9I2PPQELbmLKEQmu8mW0Yh3t5WgljQG3o+3ukMytkub+cbAe6TaqnLAJK3++0+PFMCbM4VAU5Q\\\\\\\/87rKLHYfHtugKe9QkM9xLQ4nLu\\\\\\\/6L0UkqLvbjKpYroXoqk8DLh8kcLk8FdKITp4uTICPygT00LYnQND3CPXtzCJWDTszpTwlzpiPMcLboSNGq\\\\\\\/tXotzk01hxiP10K9yPFqJrQUZehPnfOt+s5zyQs7rwnw\\\\\\\/WadaPJOspj97QmquxZP68ZdLeH8MIBfp+kM925hv9rDs21gi5oAIxMzr3JQ1x9hFTqcsN1sbaSC2VYS+mjZ864BnuR0TCdqjF6byyTHdjydU\\\\\\\/aKSz6IS8CC6iu5Jz5ASgNcwjlYQzkq2NcYpoFKaJDri+Qjwosk\\\\\\\/BWCpuv\\\\\\\/kXNXQT8\\\\\\\/Y0knTrSM3S510vkNeeh7T4Ph6O5lbIAddHOBrLq5PXCRafgrGOTtRKsU5G32RYrsT\\\\\\\/+JxgRUlKb\\\\\\\/DsdG0iTArkxQfXGW7YLI7xQsi0THY6J4bF1PIzlskDV60\\\\\\\/laKmwekOVItLAHR1t+8TxgyyXqNx8oa1comZSqOHpSsJp4q5diHvg1iWLX77jnqXS5K5YgH\\\\\\\/28nnw1Kn9SK1dvdV6P8+ABLZLemJUpGMJ9smLl1crGP1\\\\\\\/XUIDDpVv1sbP3SluXo4Zl1+62XLJZGWTGFpsistRHon+zgWj8ILAuIiXl\\\\\\\/XOk6HsaOOQeznoAeK9Nt644QlBnYRbKcPxc2x+qzeEl9TXTZzN+FyozNXtAXk67sGsRe2I7SeScophumi5AkSS70TpzWBD\\\\\\\/KC5gy7pwHOdHFSOPQ4R9I8RgFe5+SorIwQAhT4EjFziFGRF9RqFpgPqPLBIn5BooA8diJhY0s0hcUGo\\\\\\\/9JqfXjWTLvf58PvB1a48p0JidvxKDjAl+ZEYDMHDC9ApPZopgdppPJQiH2T+usiJdWBoPqcywfWgCNvLBeNZYeHCd+0ifYgD4M\\\\\\\/wCeMOpTsQpUA++zyFsaPEKMPP+DK3hNYCWda8G2QDwbkaADryLN1dhX1sx69LtDLTUg3V\\\\\\\/wbE66RMI2I1VAa48BHRbIL\\\\\\\/ix\\\\\\\/bkH+qoGUC4YsOvX4wAH1CQQLdpSHFWG7sICbZDsTqvZlHcW9d+lhvJWSdmlpceaJumcNpS6g3Uf2JMdK0aBtMt13yin2ofC8A1fmMoFBi6wH9HsTVvjqPmNue9X+kWLRqdI96pzzkDOZ0Cv1asdo0VcdbkIyI+ryWQcBaJCr6+SNHslfROuBYori4DiNcJ15Y76MJrnjV+alZZGkpIboqhFixNKTRviC5dlKCeTTUJvdvF4sAE2gya0NyA8gsJ9ZI66CrukLrfRke\\\\\\\/UsJK4cdn66DnTN0Od6bQcepKza6yNNVfuQMBfCb+BuTfjDqFSC0tVT4NlatoXvwGPmpEjnVhKE5FWj7kS4Rw\\\\\\\/K9U++OucAoelxrciNGNohPXysJc7ChR6xYp6eKhcX9LJJyBBBlUpxNrFzt5g1YilwOpJPWxnTr89MyCA2lCCqOMxQ46fLTa\\\\\\\/GtmjcF9wYBO3z24vr8TJLqux+KWG7VYE+rkkc\\\\\\\/b6bZldKgmTpAi5pjccmqaLa0qt7RDdLPIhs0F8BXEoWjsSPXN+MuZ6xWiIVArB845ZzulEKHM8pyQTzVpmmNfIUB0z1BK2eLXsqIfODGEvGhIfpZ2vmQoGr9t04UswfjusBYIOuXMCgn2YB7sX33I0r0ZwGseXJ3wCXeKhXZer+4MFylBxpGrTOmpclIIslUKngKwSpJi5J4YoD5796JaOZxTQoyx1Gokhre41+UkgkYen5\\\\\\\/ATBWUtEIGy6C4RGhpK5rsdu5F5UxhoHLI7YfPL96Y5ltQm6sFV0VUZ4gqaD8AVmL97OM2BbYBsYn9QwMZneLXhZJkBMSYYff9uktIxDmB5jKpyllb3ZxINkOiHsK5UVR+Mw84ZOTQ6hN90LZnRx+3U2sO\\\\\\\/lonpk1T9l5Qjxd3g3WDEse\\\\\\\/z3E5SjdysbFSQCYMvvF6tILp4nh9HeiRQulezi8Qqbeu\\\\\\\/5y+UrE\\\\\\\/dICL5Bqp+nPlKb\\\\\\\/fnbEabYcojD\\\\\\\/sm4jj4+BAJCvAcGRjCOMFfx69LgAqIXoZ1JOeYpwb\\\\\\\/\\\\\\\/DFGtUqMaLOP\\\\\\\/+MYOMEC4nuzWx+nd0XJSwutyTJSetwZRi08DSaGDMD1UZWs\\\\\\\/P+XbxT6ecMt3FYV6o1X9TBIv0Ud2gv9bFr000uNaby8v\\\\\\\/b7C4VDCbpBj+ahpbZScNHyiJ2KfDcNsiuDVrVKuLGtP63AZIakxKcex9LWNLRQe8Ttz+2dpZEAU0gtGIsmjNMaO5lxNq9MOf79E1cDXc91S88mg6s\\\\\\\/SfUE56esVH8ZRCod6uvw6XV5xTZLNQ60Or\\\\\\\/Kxj\\\\\\\/GPDHfvNrc\\\\\\\/lxVCy3SAqmbj5mx7uvbiM\\\\\\\/fCVInAH3ofAzAhpUxWB5yWQ29AFFi1TJTOIXdjThF1SLvpP3iCyV4y8qD26zbtmhGSiGeC5Z7hkyASOI1EuOGA6vaaoSluneow1MEuv6jXCSWk5B4qZeJ4YYWw5NjbpUX8H0Xl5uiPdUBJxobSWsEL7restbrH3gAe9jhpe\\\\\\\/yoKo25rpPtdQ1U8wDTy2DXMoCmsy+fGS8O8SiDnxkrWpEyC5t8R3Z9iXtCCGA32LbFyVnVqQBrIyD2cJbs1E3Jg\\\\\\\/GgEjE1X+ZtfJW4L\\\\\\\/lZSj9owRErk2hXhMN318B6G11vAY3m+BuYPH0dyQbp3HaW3XpKJMrFthlujssQDVYqKAKzOFyxXYXTHOTWraSqouHpFFCJJsGoAhmbfkUbGPAcmGlFpwclNx9uH1AJwEOMsnU70xppGI1l6I6A2ZRhUYma8n+0F+ibGDSN7ej6TmYB\\\\\\\/GWEJh6ZcgWsty9juQpsW5G9b3cdlrqSqus0\\\\\\\/s3VMYGc9xC7mq9\\\\\\\/Qo0NkCMmgrJvd7fezyjqCQuBzTCFkGdYwUjpxvYIs1syxLrRZChSbwnRWHD0ZKdHLPl75yZdlp8cvXzcc3k2ULl2mqO3G+H1i\\\\\\\/pxTQPwAf7DNuGQPUJb1hY8q4GU9SxgjMdluD26FxUO6IUZm6mhz2uALx8Mal9YgWnSI2eRnj5qOglm21QNm6dpG0NskSZEy6zm1qfCU2nalikh+LQ95mi1UOfwYHEM5qN7ov6pg5nJNCalQ0XMC3QtvmlDO6qnNCyfHgWiYIwaj+zpkY75RSsR6snHDprvtIMgA7lIO9q\\\\\\\/18QMVTFKVx4zkOuq626oUxjnrjnCP8T4u8BJ4VgDHiWNGWCj\\\\\\\/hZm+uJ1p4iuE0vWMiqfY0DYJlJuqzqWSXTdSt4B\\\\\\\/siMuWW3UvuiZ8alj7fRklp1mTVaXmtDVyVIqw1hnxr\\\\\\\/7n6+n319IFC7NZIKO+E0\\\\\\\/fqPZnGZvF4C6FwVA35DWAK2Ayzh3RHB3efZzUq6uTHryecFZUXytfZ\\\\\\\/4+TUsLBA8Hsi4m2cjN1STgamjbRsDjzwIwCKMjcVdbrl5BmWkIc4GS4cDFg\\\\\\\/JileHLpafC5x9oODAwtD2jHNaivSsIJgnVvFLKwTlS6WjwgrZFOP7P5X56rmNjWevUxXspc\\\\\\\/NuEiVZglRqhhYfgkxtE0VfRzNT5nxJmSZeIIaXcH4GrHY+gdUrG0vfozAg9m6WMdXboftvbnjG9vj7rL75cqbGMzyLDuM9AgxXF+1YfAStuIN9H8aP7ZbfYI7d32M5SRRKXuXVCqjPtNJMJGGYYI3uZYxbvIfsRfSVmC6AM4P7D7f489fKH6ocFHy6IFoYsBiJ8jLca5O4E07wGjpLhW8YCl9hakRcGe+He\\\\\\\/VpZYD65+fjBMVqk5f62CRRPTrtwI6cLa2U2kdai\\\\\\\/zfBADn1EFICQ0vzp9aZ14HVztmEcM8H0gR+77BtE3+JibFYhif+4JspIfMJKjlSSKbuqUIigXvzZ8SL3qrjUn5Rh+q1+O\\\\\\\/9u+xclMLAAoeihkP7rFFvkD8NDm3BeRTqLtUcDwKj41lAMFlnDZh4VpnS4ED2Z6tXMm9xoxtwxElCSYDcrX6B0mEZuKyzHcnue72lnI5ZiaJ\\\\\\\/yh21Sf32P9NjMGaUqflui2fmpK4MToyY16nWQrVP0Orif28Vq3B7gM0zUqrdcfGtfbL90sRqzNZrwAv73ysQawsir9CoZ4QbVvYVF51MbVIY3XvPWMtOQzw+l6VQF4XixXHNpglMQ\\\\\\\/4BV8q6Lb0rrgpaIVar8OAo52XESfpgK28XDhqiSpnmAiZfEjc8I3MvLCm3UAVrIG1823RDmh9\\\\\\\/ko9F67JbCPr5S9TKsPI1gX+0oqnZ8enYRMVaeMkAfsp\\\\\\\/b7C6qSbWVqoMEGd+ia+O9atYIx9mwLf7dclf+7Sw5Z6I3ENBKHWc64LxUeBJsKmGUuShTGQrRBYYKzLR0eUuLMu23iMi1R4jWxJNb2XQXc\\\\\\\/SuxxyquO6vrESugOL03Gq0fpoEbdw95OKv\\\\\\\/DO2jaJmMVFSXfdmbiCgMDlWJDCkws7Dxn\\\\\\\/lPhOJu9F1VVYRjyLqGjHbYbqN7qrr5OMNScBYhAwdAr\\\\\\\/2YzlaxGeeaXs9PMqZLXYPNIrjFbLgOvqam10OrnZIThWyeei5CVTZL\\\\\\\/0x5pIxwu0j2wPhRq0ddr1c2ybUR3rISbSamj9Zz6Mc\\\\\\\/f5DLsi3NX4UI8h2OUIOJp3wITIXypJydrqQdR2G9MRdtH6jfbIR2WPWS0a3grZ\\\\\\\/KcJluHoxx5TroLqXxhqYWfSgsDUWQFEPbjTgPJsX5lDO4Crtd2baqstp3tGHsUjqK3KdVYrimEpUo8h6qDQpbcUvOuSjpaopPdwm9qkhwccOpTuc\\\\\\\/\\\\\\\/8tMTmOtri4MHRSgM1sBTo1UgPPDFeh65fdOGP0RoV+4C+x2Q7vRpLYwz+bkzjyHY8M9sJTxXEgT6dVr7AEh2QPsQNyIkcO69NwKTKU\\\\\\\/zm1KsnFQmB\\\\\\\/rNu5HdmgtvCJdwe3vxJxJ6HQN9pCRs7vON6oyhCEcMaQI6teQ3npReHZAJe2VflktH+Fj8Oy3IHwOm3\\\\\\\/2O3lDzF9m2+g\\\\\\\/mgFbb\\\\\\\/ndYpg7BjeVLJ+6qfkV74zfUSPtmkY1hI2JHBKnE1hg\\\\\\\/ZyF8pw3YGk6XrbXwsbffTI6aIDcFC7a5MTQDnJXEv6V1BOFOltkoUU99bpiHlWWGXAjSkQudgJfvE2\\\\\\\/YxepqrDOTZmUEiUPhA8NEnN9B6kPFmsZtdZHdbshljNcO2grDx7K\\\\\\\/MxvxeJZ54THStgOhfg5jzqdLqy1N4jQ5QxjZUojxI7w\\\\\\\/LLqwNA6L6\\\\\\\/l\\\\\\\/Wf0HzOebSROcmzdB0pr3SiARB5oveeUACmc8hY+4NgguP+hRQkx+W0yaStZJu+i+Mlu1Ol6+xPmbap7sI+diwbpTl1\\\\\\\/5z70ekzUqTgYn3W5hJP11unjZ3lEfMMkY99Fe9Co4qu5pbzBRWOQ6bdRpW54wocz70PVSKiYJjuTh5UP0\\\\\\\/CsK63d8+SBitJbs1G\\\\\\\/qcgi0X8hrqvi4nfZQInRSeq2R1h0PoQxh5V4LC4LGGwn8JU0ASUcSTw3ma\\\\\\\/41vLbPg6Mi2aBiM+V1XOtdHrIAfNrCsvz1dbOP49WiR1Bxb+iPUQqOjsx\\\\\\\/K9qBuf6BUqDCqMnGbKpEuei0tPZrYdDzqcpatiOchegBLHI4nATgGQg8v\\\\\\\/IQ1xlJMyuadaCL5Ytb8NyGmhXxYSozshE+BV3qQzr3CPU\\\\\\\/9ndY2BUEik+Fowr+hcqmhxuRzxpQYEHviilLcDTzgViZNzoAOjK3Yf6Hp7UZjUdm2GDPEol2jDB4wkPof4A9hgYjASGWcBmLTdXnFaIdYvMTeP3WwnlHgZyW\\\\\\\/GHEahlBEK0gqPHN51PLfJhhil9Zbx7D\\\\\\\/A5tHyHxfA0l3fw3t9dS\\\\\\\/+YgoQVrroG12Be2DEQ8fpCwP5ODC9YHl6lr\\\\\\\/yNlJR5MKlbemxwP6LxigtcgyaplzU3RCRgaGFQ1viYH\\\\\\\/oLRwdOyhtiZ4L5ezbsT5339tj1plwfj+aeJMNHWiCOu69h1ssonTliuc1Nff4MD4DZWVgjH8GPCNLHV\\\\\\\/2i85FQ16IyKGVTy2Q1Ox1poLG8KzmjC17mVbTX2tStRAWX\\\\\\\/rlaiish0pHXRNDjpjT2be\\\\\\\/r8gBP+bylm3kf94wUfdwiw4SDVWb2s8s\\\\\\\/OnIksI2y0zvq1P7GYADW5f359wdzJRsUv1sc+NYWU7SVgZnLmTf9KT70VqQEXikXfI6S1PuvJCqjJ850aOjMMMSYGyvuM8+uzDHufe3ffY80wpf1N1UhpUgc8dMxVj\\\\\\\/9ypFmhS\\\\\\\/rNILqb2fb\\\\\\\/u7rvvUxXAhve88482tHhQT6rfBRFRHspRlQd+06MFKlFneDN8FruM92XYrapP7Sj7QLyKyoaqVZMQ5MXZgEsfTHL++\\\\\\\/ThItsLUeX11k3\\\\\\\/iFJ\\\\\\\/qAHh2IKndbd2z5GR\\\\\\\/28dd2HfMWW4XOwq4pJNb\\\\\\\/8o1Nu7ZKFi1Z7PgSZcEEI3QDh6z78tPJAkH2\\\\\\\/iisSSbheLZAqP7zI6kZtaF2eha\\\\\\\/crrLlmg+Hps\\\\\\\/cEs0QdSpIRSFGzLO5bKLtzGuQXZgx8W3EDj60x5WuMTYOIVOTwGxJeNhsxLCYSOkR9oTqtClGSRGdeF6IXbHvLa7+rSQ3ZpadMTKUoNGO6DhCXVBiqaLS4PFkEbHmsweuxIs7Ihlb8obB3YVnHXphcFpgZt8DVgYZ16yAMMm9l3OIRD\\\\\\\/wBO7+hUaekoyV8Zp7fsKQ4uXlyWmZAZGxEM6+2wnYbZcItRfJWj6KSXafR\\\\\\\/eVUKD7ExAKQT9xcdlKtSO6H\\\\\\\/uIvh6OHIEbSpxP\\\\\\\/2mNTqQJvT4gCMfsjK+KJocUCb4yRs+WOU\\\\\\\/IwUAcJTYi5irXfkr16rLdx1VXqCIGxoEwadtZ68mkl3RyecmHWN\\\\\\\/oxe6M5F3yevs588OgoIbU8rcIBt3KcDEsbvsakM47weQJ\\\\\\\/r\\\\\\\/1m84fM6K37kvEpg6N4tUSfJ\\\\\\\/VTgAGVwNippTyzjskrOjYzMOYA5bGjrmAFGaKj1\\\\\\\/VyiKfSS96x2H7ItvC1E\\\\\\\/7bLfPEim9WeLpQMYh6c5A6NFXt00y7\\\\\\\/RcgKcMgfjPshWJftzL3\\\\\\\/SFcyK\\\\\\\/Fj0j9pM5QgCyx2xjFRLgUDKKDMi3K+pJNNT\\\\\\\/YyQYIJQv1xWJz1BfjgsH13hSpkT1c29PBFZwk9cserTDJqUSKY0Dj4eEi9nxuC2i1oNaVol9rz6Gfy3CCjMGse02JEha+sQNhUIb6HS2LK1WwlH3IJjX2w4UliedQa1wA4KLa6iqBYBNx6fAa7Q+WfYEy+ECGk4JxjUnwOQ02aBE7kCegu9lFNJqSrvg8mq+1mv1WiHH2A2UxfeCsfJDwk6s8C+CdgFEbe4Z5pIYaci9o9ZpJrbVNVsGnGdQGLtvGNYWgLx6clO+VKoQw0hUoRBDULb8rNY+9hU+\\\\\\\/hv\\\\\\\/FdEy32wWN\\\\\\\/Tlk9\\\\\\\/Y+4boUmDjemCCOLUFJsGiLkBMQQ\\\\\\\/8v40P6LfX0vb84aMX96mEGF7J8uN\\\\\\\/k9KKIfipyAWqvW5QpIDh\\\\\\\/q3H0FjYQTKZadUQ9jRsSS6HWHSfttu\\\\\\\/T5vlpWSGN5NURqxjjq9XPOX235ARrFxQHDMoIhqrCzd2D\\\\\\\/jFDzklqzUZwam51XTW0DTVuK4SikdCiS7qjg6qrmw9Z4mYtEysmEgWZevb6+Z7hj68fEgPoHQmJ5FM61F18VMo0CQJzv90+yVraMShqCc8RMygvV7l490c50tGSJ+2hHhFar5XIXweK0jIfGarRDV0H7ukbQQxSEBd2rfTBvq0BKfWiKfBwvgoOg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022mYHJGKLPGBrkKxt6+rj+mA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d9c58091ae0988c4edaf59947c395a561b04f8729e0780fd8d7fa56c37c847de\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Neb\\\\\\\/Tq2Xsg04fNQMqVbsG1x9XFcUyUnAF\\\\\\\/xI4PxtLaXdxdloYs+DZy\\\\\\\/0NjJz6WOde\\\\\\\/Z4X2bhyW1XC+Umu73qjnrS+3eJGwP4YaZZglKzt4lFb953c4rD2wSZ1WcXygfsitzw6J0UWnd43eRfvAt+yaVqoZImljctAbi0kcSY8fJKPjbQCDJzkiwDCb1iTJHMs41OA02B3rinTFghqlJISPkMkWV\\\\\\\/AjNKa+E9Go2JWMzM9Q3Bj5GyiMS2euY3fB0DOajMAEln9wxywffbH2+HfFWPOlet0bzPOaXSHkcaTHmOqUrhljm8bBvt1SlFJzxnNLoxKyfVKMU2Mczi\\\\\\\/nw1obPlOByLo2bxjBY81VgGv5xkkcxz0sRROxaFThuUB0tt3J4yy9ICL4PH20AClVZ7D+\\\\\\\/flNWEH8PJX+\\\\\\\/jrvlGISm\\\\\\\/5kVMnDsEPilGxr+CsBWR\\\\\\\/2QOFcTxhv5SQiPn4liz8TXTlobYwHaKVtFAE7fHj9TltI947+RelmdgCpQ5sNge5W53LfN5luidtunBQN3aVr9e+k6W81H63ypx8XXaLY+v0gpYpY5SDxLdVxAqWlAhnFoJFFv\\\\\\\/L7ze31AC8m2rn231YGcW6dIF6Ti2bXE6coJ2KukRX\\\\\\\/dxZzvroeInCJ7UABlNa8CSG\\\\\\\/X2FzFYuqvsj+4lTcXzuswJqto73YSOGhgsBzz34720oBhxtUzYZlkdb5UdvNeuE9koQD6dIT\\\\\\\/VCHd5KOB7ITcfqG1m4pwqF+oDCdqzRIshNo1LRiAxyNgqnMmeG8mfrJWoJDpkQJY5020P92fEiY5CxhCf\\\\\\\/PcYtn3U4Tjs1GsN3wxq3hunDk9QG1RTqRkMvoTeFNC9ll4zs1GT7gYMSB85ZNUAIpk3MACmGuEvINy1L9hZqpPG+gmUwgeJo0BM+JAiG+xqHFsrzx8N98i47snst92NdkO8L00YRjnhS3V9TO6zKY0GjFZkzWBkDDs7fFG0FK2vNoSyigb8lOP5bdurWdFgf7weUt88yqrlUf5SGyBSS2F7Vv+52y2Fa81m9rnyrAdRuBr\\\\\\\/40FFyutpDpG27dHWSo8vD1xvQu6A\\\\\\\/8IzETUhpxuRj+HDED1SgAO0HWvJElSIvriLrId7PXEZtJ1pnzN5wgXedNF8N+2koOSgrq3nUN+REgtrtO3Ln71ojKMjk0ahyWLwWnAfM\\\\\\\/8bEot5l6VNmwHmdXnmUN+NIzY\\\\\\\/Z+8GARhaTE+MszZAzaMXFTujQZNOfVlhoLOjy03CBSAX68Bwb2YJbygpIRbMpG9et04X+TubvW1TV8ETnM+PnP1S8jsR+WL0q70cTkKeH\\\\\\\/qcfaxcaiPSyZUqfVh6NVl5Gz8RQgNIlfI5S3rapTC7GzxBu0SuN6IHJzJc7LeqZfOzjtsjXIqwevAN5WOrXZXNKNgOus3ZfuWroOeFd+7X1eKd5WCfvodiELGOLiRfbO0\\\\\\\/ZjCpwmvKBX3VOls3kYkHj4a\\\\\\\/+cil0xaN3kRJZX+k8ZFjlDsoTSQNgzkAgpTaInkNcmgWfd2rU9XJLCQno7eNUX9F4jDBnddc9Pt4ALOd3jzlbz4SoohO1btVyBl28B9oO379C0gx2MkkGAymTxneC2P95mzYXm6Qujdrpb81ewnZvjW3RGquHIAoynuuiIzm1M4Z48jix3Iz0HKNVfowB8RvTYjC76dvKwllAsph\\\\\\\/PKBqrHN6CXa06wZq6r5jnFK9ttbJuVSPyI5vTDqqrD8uJ8lzQmM91ex7rQhq3JUU6R5InzP7SymLJSGrxRihyX5+XunhyBh3MNsmWpQsj2Lah9X\\\\\\\/\\\\\\\/qNDDcs5K5+pC8pqFDsi4PuOkPB\\\\\\\/WtB23iRzPtOMzzv5+Fy3IKpiga14btkDxcAICVt0sFTVJM7b8WHLTPLdn8TujZvKitfbInx5anRSEAkQzmZQx+PRVN9ur\\\\\\\/Y47I8ngpRnrkE0a3lGpapgum6H5AAkxvMcEtfex6S9xAeHG4NWUlc2\\\\\\\/X7kz\\\\\\\/JpRjHjcbVYyYdwfemwEw68lhVxs6tLmOYqrapZg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022kvhlBKq308qHxVdNLzx0qA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022355b69b9b76720dac8534d768d95b9c005350761a200faea11d34828e3e93cec\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484774 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1683" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mGyVe8dyeTyD3G93w1uF925t0k3c2RV8Mw81QyJ0SKw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1683" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yQ6ob2YhoKIEc2P9g1sJUvprda6uQ_-gXa8xN9qFQT0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1665" title="Oas1b-dependent Immune Transcriptional Profiles of West Nile Virus Infection in the Collaborative Cross" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1693" title="Identification of Nitrogen Consumption Genetic Variants in Yeast Through QTL Mapping and Bulk Segregant RNA-Seq Analyses" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484774" id="node484774" data-pisa="ggg;7/6/1683" data-pisa-master="ggg;g3.117.041764" data-apath="/ggg/7/6/1683.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of <em>Ganoderma</em> Disease Resistance Loci Using Natural Field Infection of an Oil Palm Multiparental Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sébastien Tisné</span>, <span class="highwire-citation-author" data-delta="1">Virginie Pomiès</span>, <span class="highwire-citation-author" data-delta="2">Virginie Riou</span>, <span class="highwire-citation-author" data-delta="3">Indra Syahputra</span>, <span class="highwire-citation-author" data-delta="4">Benoît Cochard</span> and <span class="highwire-citation-author" data-delta="5">Marie Denis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1683-1692;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041764 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484774" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sébastien Tisné </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S%C3%A9bastien%2BTisn%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tisn%C3%A9%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AS%25C3%25A9bastien%2BTisn%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sebastien.tisne@cirad.fr" class="" data-icon-position="" data-hide-link-title="0">sebastien.tisne@cirad.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Virginie Pomiès </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginie%2BPomi%C3%A8s%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pomi%C3%A8s%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginie%2BPomi%25C3%25A8s%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Virginie Riou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginie%2BRiou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riou%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginie%2BRiou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Indra Syahputra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>P.T Socfindo Medan, 20001, Indonesia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Indra%2BSyahputra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Syahputra%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIndra%2BSyahputra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Benoît Cochard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>PalmElit SAS, 34980 Montferrier sur Lez, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beno%C3%AEt%2BCochard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cochard%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeno%25C3%25AEt%2BCochard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Marie Denis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie%2BDenis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Denis%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie%2BDenis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/6/1683" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484774" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484774/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1683.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484774" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484774/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1683.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484774" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484774/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1683.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484774" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484774/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/6/1683.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484774" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484774/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Multi-parental populations are promising tools for identifying quantitative disease resistance loci. Stem rot caused by <em>Ganoderma boninense</em> is a major threat to palm oil production, with yield losses of up to 80% prompting premature replantation of palms. There is evidence of genetic resistance sources, but the genetic architecture of <em>Ganoderma</em> resistance has not yet been investigated. This study aimed to identify <em>Ganoderma</em> resistance loci using an oil palm multi-parental population derived from nine major founders of ongoing breeding programs. A total of 1200 palm trees of the multi-parental population was planted in plots naturally infected by <em>Ganoderma</em>, and their health status was assessed biannually over 25 yr. The data were treated as survival data, and modeled using the Cox regression model, including a spatial effect to take the spatial component in the spread of <em>Ganoderma</em> into account. Based on the genotypes of 757 palm trees out of the 1200 planted, and on pedigree information, resistance loci were identified using a random effect with identity-by-descent kinship matrices as covariance matrices in the Cox model. Four <em>Ganoderma</em> resistance loci were identified, two controlling the occurrence of the first <em>Ganoderma</em> symptoms, and two the death of palm trees, while favorable haplotypes were identified among a major gene pool for ongoing breeding programs. This study implemented an efficient and flexible QTL mapping approach, and generated unique valuable information for the selection of oil palm varieties resistant to <em>Ganoderma</em> disease.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/plant-disease" class="hw-term hw-article-keyword hw-article-keyword-plant-disease" rel="nofollow">plant disease</a></li><li class="kwd"><a href="/keyword/multiparental-population" class="hw-term hw-article-keyword hw-article-keyword-multiparental-population" rel="nofollow">multiparental population</a></li><li class="kwd"><a href="/keyword/quantitative-disease-resistance-loci" class="hw-term hw-article-keyword hw-article-keyword-quantitative-disease-resistance-loci" rel="nofollow">quantitative disease resistance loci</a></li><li class="kwd"><a href="/keyword/survival-analysis" class="hw-term hw-article-keyword hw-article-keyword-survival-analysis" rel="nofollow">survival analysis</a></li><li class="kwd"><a href="/keyword/identity-descent" class="hw-term hw-article-keyword hw-article-keyword-identity-by-descent" rel="nofollow">identity-by-descent</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-3">Plant diseases cause high yield losses for almost all crops worldwide, thus constituting a major food security constraint in a context of high population growth and shrinking cropland (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Oerke and Dehne 2004</a>). The genetics of plant resistance to pathogens is thus an active research area to support the selection of resistant crop varieties in the framework of integrated pest management programs geared toward limiting the use of chemical pesticides. R-genes controlling qualitative disease resistance have been identified in plants mainly by using biparental populations derived from crosses between resistant and susceptible parents (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gururani <em>et al.</em> 2012</a>). Despite the efficiency of R-genes in the selection of resistant varieties, the sustainability of resistance could be improved by taking quantitative resistance loci into account (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Poland <em>et al.</em> 2009</a>), as shown empirically in studies on oilseed rape (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brun <em>et al.</em> 2010</a>) and pepper (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Quenouille <em>et al.</em> 2014</a>). Broadening the genetic diversity screened using association panels is a way to identify such QTL (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wei <em>et al.</em> 2006</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adeyanju <em>et al.</em> 2015</a>). Based on improved genetic and statistical properties (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Verhoeven <em>et al.</em> 2005</a>), multi-parental populations (MPPs) are increasingly being developed in many organisms (see the MPP collection <a href="http://www.genetics.org/content/multi-parental_populations">http://www.genetics.org/content/multi-parental_populations</a>), and their use for mapping quantitative resistance loci is promising (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kump <em>et al.</em> 2011</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Benson <em>et al.</em> 2015</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cockram <em>et al.</em> 2015</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bajgain <em>et al.</em> 2016</a>). Studies carried out in different contexts are needed to explore alternative experimental designs, appropriate statistical analyses, and to assess the potential of MPPs to gain insight into the genetic architecture of plant disease resistance.</p><p id="p-4">Oil palm (<em>Elaeis guineensis</em> Jacq.) is a perennial allogamous species of African origin. It is currently the world’s leading fat resource, accounting for &gt;30% of global vegetable oil production (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">USDA 2015</a>), with a constantly increasing demand (FAOSTAT, <a href="http://www.fao.org/faostat/">http://www.fao.org/faostat/</a>). Southeast Asia represents almost 90% of global production, but oil palm cultivation in this area is highly hampered by stem rot diseases, <em>i.e.</em>, basal (BSR) or upper (USR). BSR and USR are caused by <em>Ganoderma boninense</em>, a soil hemi-biotroph fungal pathogen that is also found in Africa and South America, where the disease is currently developing. <em>Ganoderma</em> disease is economically problematic when affecting ∼10–20% of palm trees, while 30–70% could be lost by the end of a typical 25-yr planting cycle, leading to substantial yield losses, premature replantation, and a waste of cropland (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Durand-Gasselin <em>et al.</em> 2005</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cooper <em>et al.</em> 2011</a>). The identification of genetic resistance in breeding material, based on field observations (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">de Franqueville <em>et al.</em> 2001</a>) or screening tests with seedling inoculation (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Idris <em>et al.</em> 2004</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Breton <em>et al.</em> 2006</a>), has led to the selection of partly resistant varieties. Information on the genetic architecture and molecular nature of <em>Ganoderma</em> resistance could help shorten the long oil palm breeding cycle (up to 20 yr), and enable its multi-criteria improvement using a marker-assisted selection strategy. All studies on <em>Ganoderma</em> disease to date have been based on seedling inoculation at the nursery stage: transcriptomic (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Tee <em>et al.</em> 2013</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ho <em>et al.</em> 2016</a>), proteomic (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Al-Obaidi <em>et al.</em> 2014</a>), or metabolomic (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nusaibah <em>et al.</em> 2016</a>) approaches have identified genes, proteins, and pathways affected by <em>Ganoderma</em> infection, while an analysis based on 58 simple sequence repeat markers found alleles associated with <em>Ganoderma</em> symptoms in resistant <em>vs.</em> susceptible families (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hama-Ali <em>et al.</em> 2015</a>). Field experiments are needed to assess broader genetic diversity in the agronomic context, but QTL analyses on oil palm crosses in the field are often not sufficiently effective because of population size and allelic segregation limitations (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Jeennor and Volkaert 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lee <em>et al.</em> 2014</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pootakham <em>et al.</em> 2015</a>). Recent alternatives, based on data from breeding programs involving multiple connected families, have yielded promising results on agronomic traits (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Billotte <em>et al.</em> 2010</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Tisné <em>et al.</em> 2015</a>).</p><p id="p-5">In this study, an oil palm multi-parental population (<em>Eg</em>9PP) derived from nine founders of an ongoing breeding program was used to map <em>Ganoderma</em> disease resistance loci. Natural <em>Ganoderma</em> infection was monitored in the field, and the infection status of the 1200 <em>Eg</em>9PP individuals were recorded over 25 yr to generate a unique dataset. An innovative statistical modeling approach was implemented to map genetic loci affecting the survival of <em>Eg</em>9PP individuals, by combining a Cox regression model used for survival time QTL mapping in animal (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Durrant <em>et al.</em> 2011</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gonen <em>et al.</em> 2015</a>) or plant (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Luo <em>et al.</em> 2013</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jiang <em>et al.</em> 2014</a>) species, with an identity-by-descent (IBD)-based variance component approach (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">George <em>et al.</em> 2000</a>) adapted to MPP design and with spatial effects modeling in order to control the confounding effects of <em>Ganoderma</em> epidemics. As <em>Eg</em>9PP parents are from genetic origins that are widely used in oil palm breeding programs, loci identified in this study provided highly relevant information to guide efficient marker-assisted selection for <em>Ganoderma</em> resistance.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genetic material</h3><p id="p-6">The oil palm (<em>Elaeis guineensis</em> Jacq.) multi-parental population (<em>Eg</em>9PP) was developed from 14 full-sib families generated by crossing five outbred parents of heterotic group A with four outbred parents of heterotic group B in an incomplete factorial design (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and see Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS1.xlsx">Table S1</a>). The group A parents belonged to the “Deli” population derived from four individuals planted in 1848 in Indonesia, and the group B parents belonged to “La Mé” (Cote d’Ivoire) or “Yangambi” (Democratic Republic of the Congo) populations (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Corley and Tinker 2003</a>). The <em>Eg</em>9PP parents are major founders of the PalmElit breeding program, a CIRAD subsidiary and leading oil palm breeding company (<a href="http://www.palmelit.com">www.palmelit.com</a>). This breeding program is shared and conducted with partners, <em>i.e.</em>, PT Socfin Indonesia (Indonesia) and INRAB (Benin). The 1200 <em>Eg</em>9PP individuals were planted in 1986 on the SOCFINDO estate (Medan, Indonesia), according to a randomized complete block design (RCBD) with at least five replications per family, each replication consisting of a plot with 15 full-sib individuals (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/FigureS1.tif">Figure S1</a>). Despite the highly favorable agro-climatic conditions for oil palm growing, this location is hampered by the presence of <em>G. boninense</em>—the causal agent of basal stem rot disease that naturally infected <em>Eg</em>9PP.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1683/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pedigree of the Eg9PP oil palm multi-parental population. Eg9PP (red boxes, n&#xA0;=&#xA0;1200) consisted of 14 full-sib families obtained by crossing parents from heterotic group A (blue boxes) and B (green boxes). The five parents from group A (D1&#x2013;D5) derived from four founders (DF1&#x2013;DF4) by partially known successive crosses and self-pollination (dotted lines). The number of individuals per family are indicated at the right side of the family name, with the number of sampled individuals in parenthesis (n&#xA0;=&#xA0;757). Shaded boxes represent the fully genotyped families that constituted the mapping population (n&#xA0;=&#xA0;604)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1576922106" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pedigree of the &lt;em&gt;Eg&lt;/em&gt;9PP oil palm multi-parental population. &lt;em&gt;Eg&lt;/em&gt;9PP (red boxes, &lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;1200) consisted of 14 full-sib families obtained by crossing parents from heterotic group A (blue boxes) and B (green boxes). The five parents from group A (D1&#x2013;D5) derived from four founders (DF1&#x2013;DF4) by partially known successive crosses and self-pollination (dotted lines). The number of individuals per family are indicated at the right side of the family name, with the number of sampled individuals in parenthesis (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;757). Shaded boxes represent the fully genotyped families that constituted the mapping population (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;604).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1683/F1.medium.gif" width="440" height="341"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/6/1683/F1.medium.gif" width="440" height="341"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1683/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1683/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484865" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Pedigree of the <em>Eg</em>9PP oil palm multi-parental population. <em>Eg</em>9PP (red boxes, <em>n</em> = 1200) consisted of 14 full-sib families obtained by crossing parents from heterotic group A (blue boxes) and B (green boxes). The five parents from group A (D1–D5) derived from four founders (DF1–DF4) by partially known successive crosses and self-pollination (dotted lines). The number of individuals per family are indicated at the right side of the family name, with the number of sampled individuals in parenthesis (<em>n</em> = 757). Shaded boxes represent the fully genotyped families that constituted the mapping population (<em>n</em> = 604).</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">Pedigree information was available for the La Mé parents, while the pedigree of the Deli parents was only known for a few (one or two) generations before the <em>Eg</em>9PP parents. The relatedness of the five Deli parents was estimated using pedigree reconstruction with MOLCOANC 3.0 software, as described in <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Cros <em>et al.</em> (2014)</a>, by using a larger Deli palm population (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Tisné <em>et al.</em> 2015</a>).</p></div><div id="sec-3" class="subsection"><h3>Phenotype acquisition</h3><p id="p-9">The <em>G. boninense</em> infection status was recorded biannually on the 1200 <em>Eg</em>9PP individuals over 25 yr, from the first year after planting (June 1987) to the uprooting of the genetic trial (June 2012). Symptoms were scored blindly on the basis of a six level scale, set up in the 1980s, and used since that time in the SOCFINDO plantations, with zero being a healthy palm tree and six a dead and fallen palm tree with the presence of <em>Ganoderma</em> fruiting bodies on the palm trunk (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/FigureS2.tif">Figure S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS2.docx">Table S2</a>). A seventh level represented palm trees absent at the visit time, or dead for reasons other than <em>Ganoderma</em> disease. Raw data were then curated, and the occurrence of two events was recorded with the time associated: the first <em>Ganoderma</em> symptom appearance (T1S, first observation of score 2–6), and the death of the palm tree due to <em>Ganoderma</em> disease (TD, first observation of score 6). Events of interest corresponded to T1S and TD, and the associated times were considered as survival times, <em>i.e.</em>, times from the start of the experiment to the time the event occurred (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS3.xlsx">Table S3</a> for phenotypic data). When the events were observed, the associated times were called exact or uncensored times, otherwise they were called censored observations.</p></div><div id="sec-4" class="subsection"><h3>Molecular data and genetic map construction</h3><p id="p-10">A total of 757 individuals out of the 1200 from <em>Eg</em>9PP were sampled before uprooting the genetic trial because of premature death or illegitimacy (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). In a previous study, 116 individuals from the reference family D2L1 were genotyped with 390 single sequence repeat (SSR) markers, and a first reference high density linkage map grouping 255 SSRs and 688 amplified fragments length polymorphisms (AFLP) markers was published (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Billotte <em>et al.</em> 2005</a>). The SSR markers, plus 26 SSR additional markers previously mapped in a pedigree-based linkage analysis that was recently published (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cochard <em>et al.</em> 2015</a>), were used to genotype 604 individuals from nine <em>Eg</em>9PP families, including D2L1, which constituted the mapping population (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Based on genotyping of the <em>Eg</em>9PP parents, and the genetic position of SSR markers, different SSR marker subsets were used depending on the family, to optimize the genome coverage and marker information level (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS4.xlsx">Table S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS5.xlsx">Table S5</a> for individual genotypes). A QTL region identified in the linkage analysis of the mapping population (<em>n</em> = 604) was investigated in further detail by genotyping all of the 757 sampled individuals from the 14 <em>Eg</em>9PP families with all of the SSR markers mapped in the region plus two markers (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS4.xlsx">Table S4</a>). DNA extraction, microsatellite amplification, and analyses were performed according to the protocol described in <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Cochard <em>et al.</em> (2015)</a>, and raw genotype data are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS5.xlsx">Table S5</a>.</p><p id="p-11">The Eg9PP genetic map was constructed with CRI-MAP version 2.507 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Green <em>et al.</em> 1990</a>). This program allows simultaneous analysis of multiple families and performs multipoint linkage analysis. The nine families of the <em>Eg</em>9PP mapping population were treated as unrelated families, and all individuals were considered as being of undetermined sex. Markers were grouped in separate input files based on the linkage group defined in the previously published genetic map (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Billotte <em>et al.</em> 2005</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Cochard <em>et al.</em> 2015</a>). Markers were ordered in each linkage group using several runs of the BUILD command on the most informative markers, then the remaining markers were placed in the framework obtained using the ALL command. The FLIPS command was used to check alternative marker orders at the different linkage group construction stages, using different values up to four, depending on the number of markers and length of the linkage group analyzed. The FIXED option was finally used to calculate the Kosambi genetic distances between markers (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/FigureS3.tif">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS4.xlsx">Table S4</a>). The genetic map obtained was drawn using MapChart software (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Voorrips 2002</a>).</p><p id="p-12">We attempted to infer a physical position for each marker mapped using BLASTn (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Zhang <em>et al.</em> 2000</a>). We used forward and reverse primer sequences for each marker as query sequences against the oil palm genome sequence as the target database (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Singh <em>et al.</em> 2013</a>). We filtered the result hits of BLASTn with a minimum <em>e</em>-value threshold of 0.5. The marker position was assigned easily when forward and reverse primers were found in the same genome location. In case of multiple hits on different chromosomes for the same primer pair, the marker position was assigned based on the hit with the greatest <em>e</em>-value. Inferred physical positions of markers are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS4.xlsx">Table S4</a>.</p><p id="p-13">Based on the marker physical positions, the physical positions of 210 predicted R-genes in the oil palm genome (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Singh <em>et al.</em> 2013</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chan <em>et al.</em> 2017</a>) were projected on the <em>Eg</em>9PP genetic map in order to identify colocalizations with <em>Ganoderma</em> resistance QTL (See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS8.xlsx">Table S8</a>).</p></div><div id="sec-5" class="subsection"><h3>Linkage mapping</h3><p id="p-14">The linkage mapping of <em>Ganoderma</em> disease resistance was performed using a combination of survival data analyses and the two-step variance component approach described in <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">George <em>et al.</em> (2000)</a>. The first step involved computation of IBD kinship matrices over a grid of genetic positions. The second step consisted of QTL presence tests over the grid of genetic positions by comparison of various models including random effect terms.</p><div id="sec-6" class="subsection"><h4>IBD matrix computation:</h4><p id="p-15">The variance component approach requires IBD kinship matrices as variance-covariance matrices for testing QTL effects at each genetic position. IBD analyses using SimWalk2 software (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sobel <em>et al.</em> 2001</a>) were conducted to estimate the IBD matrices formed by empirical kinship coefficients for each pair of <em>Eg</em>9PP individuals and their parents. Simwalk2 implements a Bayesian framework using a Markov chain Monte Carlo (MCMC) algorithm to select the most likely genetic descent graph depicting inheritance patterns within pedigrees (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Sobel and Lange 1996</a>). Empirical kinship coefficients were first calculated between the 604 fully genotyped <em>Eg</em>9PP individuals and their parents at every marker position and over a 3 cM interval grid, leading to 1006 evaluation points for QTL presence testing (available at <a href="https://doi.org/10.6084/m9.figshare.4780489.v1">https://doi.org/10.6084/m9.figshare.4780489.v1</a>).</p></div><div id="sec-7" class="subsection"><h4>Statistical modeling:</h4><p id="p-16">The Cox regression model (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cox 1972</a>) is a popular choice for analyzing the relationship between the survival outcome and explanatory covariates X. This semiparametric model has the following hazard function:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-2.gif"/></noscript></span></span> is a <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-3.gif"/></noscript></span></span> unknown vector of parameters reflecting the effects of covariates <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-4.gif"/></noscript></span></span> on survival, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-5.gif"/></noscript></span></span> is the time to the event or censoring, and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-6.gif"/></noscript></span></span> denotes the baseline hazard function. Note that no particular shape was assumed for the baseline function.</p><p id="p-17">In this study, the analysis of data based on natural field infection required taking into account the spatial component of the propagation of <em>Ganoderma</em> disease. The spatial (plot) information was included to model (1), as well as genetic (family and individual) information to avoid the genetic variance being absorbed by spatial variance due to the experimental design. One way of considering this information is to extend the model (1) to a mixed effect Cox model by adding one random effect with a covariance matrix based on pedigree, and one with a spatially structured covariance matrix based on the geographic coordinates of the plot. <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Vaupel <em>et al.</em> (1979)</a> were the first to introduce random effects (called frailties) in the survival model in order to group time-to-event data within <em>clusters</em>.</p><p id="p-18">Model (2) is then defined as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-7.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>with <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-8.gif"/></noscript></span></span> corresponding to the number of individuals, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-9.gif"/></noscript></span></span> is the design matrix relating the observations to the <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-10.gif"/></noscript></span></span> vector of family fixed effects, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-11.gif"/></noscript></span></span> the design matrix relating the survival outcome to the <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-12.gif"/></noscript></span></span> vector of spatial random effects <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-13.gif"/></noscript></span></span> at the plot level, and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-14.gif"/></noscript></span></span> is the design matrix relating the survival outcome to the additive genetic random effects <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-15.gif"/></noscript></span></span> with m the number of parents.</p><p id="p-19">Concerning the spatial random effects, a conditionally auto-regressive (CAR) structure (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Cressie 1993</a>) was used in order to model dependence among observations in neighboring plots (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Banerjee <em>et al.</em> 2003</a>). Where <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-16.gif"/></noscript></span></span> denotes the neighborhood of plot <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-17.gif"/></noscript></span></span> including the plots which share a common boundary, the distribution of <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-18.gif"/></noscript></span></span> is a multivariate Normal distribution defined by:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-19.gif"/></noscript></span></span>where α is the spatial autocorrelation parameter, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-20.gif"/></noscript></span></span> the adjacency matrix (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-21.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-22.gif"/></noscript></span></span>if <em>s</em> and <em>s</em>′ are neighbors, and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-23.gif"/></noscript></span></span> otherwise), <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-24.gif"/></noscript></span></span> is the spatial dispersion parameter, and <em>D</em> is a diagonal matrix with diagonal elements equal to <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-25.gif"/></noscript></span></span> <em>i.e.</em>, the number of neighbors for the plot <em>s</em>. The additive genetic random effects also follow a multivariate Normal distribution:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-26.gif"/></noscript></span></span>with <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-27.gif"/></noscript></span></span>being the kinship matrix based on pedigree information, and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-28.gif"/></noscript></span></span> the additive genetic variance.</p><p id="p-20">To save computation time in the QTL mapping step, the model (2) was used for predicting spatial random effects associated with each plot, and the spatial effects predictions <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-29.gif"/></noscript></span></span> were included as fixed effects in the following QTL models (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS7.docx">Table S7</a> for model (2) results). The model considered as null model for QTL mapping was then defined by:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-30.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-31.gif"/></noscript></span></span> is the <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-32.gif"/></noscript></span></span> design matrix relating the survival outcome for genotyped individuals to fixed effects (family and spatial effects). The null model defined in (3) was compared to the model (4) such that:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-33.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>with <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-34.gif"/></noscript></span></span> being the design matrix relating the survival outcome for genotyped individuals to the QTL random effect <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-35.gif"/></noscript></span></span> at the <em>p</em>th position on the genome. The QTL random effect at the <em>p</em>th position <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-36.gif"/></noscript></span></span> followed a multivariate normal distribution, with a variance-covariance matrix equal to the IBD kinship matrix <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-37.gif"/></noscript></span></span> at the <em>p</em>th position then being <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-38.gif"/></noscript></span></span></p><p id="p-21">A genome scan was performed to test the QTL presence by computing the log-likelihood ratio to compare models (3) and (4) at each p position. Best linear unbiased predictions (BLUPs) of QTL effect were computed for each <em>Eg</em>9PP individual using model (4) at the retained genetic positions (see next paragraph).</p></div><div id="sec-8" class="subsection"><h4>Log-likelihood ratio test:</h4><p id="p-22">The log-likelihood ratio test (LRT) was used to test whether frailty (QTL) variance was null at each p position. With a non-negative restriction on the variance component, the LRT is one-sided and has null value on the boundary of parameter space, so the classical likelihood ratio asymptotic chi-squared distribution theory is no longer valid. The distribution of LRT is then unknown under the null hypothesis H<sub>0</sub>, so an empirical approach by permutations (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Churchill and Doerge 1994</a>) and a theoretical approach were used to estimate significance levels. In the permutation approach, different LRT significance levels were calculated for each putative QTL based on 500 chromosome-wide permutations obtained by shuffling the identity of individuals in the data to break genotype-phenotype associations. For each permutation, the maximum LRT value obtained was recorded to construct the distribution of the maximum LRT value under the null hypothesis (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/FigureS4.tif">Figure S4A</a>). The empirical type I error rates associated with each LRT distribution were calculated so as to estimate the significance level of each LRT peak detected (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/FigureS4.tif">Figure S4B</a>). The theoretical approach was based on different works in shared frailty model framework (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Duchateau <em>et al.</em> 2002</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Vu and Knuiman 2002</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Claeskens <em>et al.</em> 2008</a>) that assessed the limiting distribution of LRT. The resulting distribution is the asymptotic 50:50 mixture of the <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-39.gif"/></noscript></span></span> and the <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-40.gif"/></noscript></span></span> chi-squared distributions of 0 and 1 degree of freedom, respectively, that was used to calculate <em>p</em>-values associated to each QTL detected.</p></div><div id="sec-9" class="subsection"><h4>Haplotype analysis:</h4><p id="p-23">QTL mapping identified a strong QTL in the top of linkage group 1, and this region was analyzed in further detail. Based on the genotyping of all the 757 sampled individuals from the 14 <em>Eg</em>9PP families with seven SSR markers covering 20 Mb (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS4.xlsx">Table S4</a>), the kinship coefficients were calculated between the 757 <em>Eg</em>9PP individuals and used to perform QTL mapping with this extended dataset in order to determine precise QTL location in the candidate region. Then, haplotype analysis was performed in the candidate region using SimWalk2 to identify the funder allele segregation in the Eg9PP individuals. Funder allele effects were then estimated including two fixed effects in model (3)—one for each heterotic group—in order to identify favorable alleles among the <em>Eg</em>9PP parents at the QTL peak closest marker.</p><p id="p-24">All analyses were performed with R software version 3.2.3 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">R Development Core Team 2012</a>) and the coxme package (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Therneau and Therneau 2015</a>), and R scripts, with example data files and documentation, are available at the github repository <a href="https://github.com/DenisMarie/Eg9PP_Ganoderma">https://github.com/DenisMarie/Eg9PP_Ganoderma</a>. The genome scan results are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS6.xlsx">Table S6</a>.</p></div></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-25"><em>Eg</em>9PP was uprooted in 2012, but <em>Eg</em>9PP families can be reproduced by crossing individuals derived from the self-fertilized progenies of <em>Eg</em>9PP parents. Requests for such genetic material should be addressed to PalmElit (<a href="http://www.palmelit.com">www.palmelit.com</a>). Raw genotype data are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS5.xlsx">Table S5</a> and IBD kinship matrices at <a href="https://doi.org/10.6084/m9.figshare.4780489.v1">https://doi.org/10.6084/m9.figshare.4780489.v1</a>. Phenotyping of disease events are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS3.xlsx">Table S3</a>. All R scripts, data file and documentation are available at <a href="https://github.com/DenisMarie/Eg9PP_Ganoderma">https://github.com/DenisMarie/Eg9PP_Ganoderma</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Variations in Ganoderma resistance in Eg9PP</h3><p id="p-26"><em>Ganoderma</em> disease resistance was assessed in 1200 <em>Eg</em>9PP individuals by biannually recording their infection status over 25 yr. At the end of the experiment, 58.5% of the individuals exhibited <em>Ganoderma</em> symptoms over this period, and 30.5% died due to <em>Ganoderma</em> disease (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> and see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS7.docx">Table S7</a>). These percentages ranged from 74.7 and 54.7% for infected and dead individuals, respectively, for the worst family D1L3, to 37.3 and 14.7% for the best families D2L2 and D2Y1 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a> and see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS7.docx">Table S7</a>). The percentages of individuals per family infected and dead at the uprooting of the trial were significantly correlated (<em>r</em> = 0.66, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>), but families with a similar percentage of infected individuals exhibited a substantially different percentage of death (<em>e.g.</em>, D3L3 and D2Y1, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). The times of the first <em>Ganoderma</em> symptom appearance (T1S) and death due to <em>Ganoderma</em> (TD) were modeled using Cox regression including spatial and family effects, and both effects were found to be highly significant (<em>P</em> &lt; 0.001). Risks per family ranged from 0.32 to 0.98 for T1S, corresponding to 3.1-fold lower risk for the best family (D2L2) compared to the worst family (D1L3) (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a> and see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS7.docx">Table S7</a>). For TD, risks per family ranged from 0.2 to 0.96, corresponding to 4.9-fold lower risk for the best family (D2Y1) compared to the worst family (D1L3) (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a> and see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS7.docx">Table S7</a>). In addition to this interfamily variance, large confidence intervals for risk per family indicated other sources of variance that could be captured by QTL effects in families (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS7.docx">Table S7</a>). Kaplan-Meier estimates of survival showed that the economically significant threshold of 20% affected palm trees was reached 15 yr after planting for the worst <em>Eg</em>9PP family as compared to 24 yr for the best (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1B</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1683/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ganoderma disease epidemics in the Eg9PP oil palm multi-parental population. (A) Relationship between the percentages of affected and dead individuals per family at the trial uprooting. Colors corresponding to families are indicated in the legend at the top left, with gray circles indicating the Eg9PP value (n&#xA0;=&#xA0;1200). Dotted lines represent the linear regression between the percentages of affected and dead individuals (slope&#xA0;=&#xA0;0.72, Pearson correlation coefficient r&#xA0;=&#xA0;0.66, p-value =&#xA0;0.009). (B) Survival curves for Eg9PP and Eg9PP families exhibiting maximum (D1L3) and minimum (D2Y1, D2L2) percentages of affected and dead individuals at the trial uprooting (color legend at bottom left). Survival estimates are plotted for two events, the time of the first Ganoderma symptom observation (T1S, solid lines), and death due to Ganoderma infection (TD, dotted lines)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1576922106" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Ganoderma&lt;/em&gt; disease epidemics in the &lt;em&gt;Eg&lt;/em&gt;9PP oil palm multi-parental population. (A) Relationship between the percentages of affected and dead individuals per family at the trial uprooting. Colors corresponding to families are indicated in the legend at the top left, with gray circles indicating the &lt;em&gt;Eg&lt;/em&gt;9PP value (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;1200). Dotted lines represent the linear regression between the percentages of affected and dead individuals (slope&#xA0;=&#xA0;0.72, Pearson correlation coefficient &lt;em&gt;r&lt;/em&gt;&#xA0;=&#xA0;0.66, &lt;em&gt;p&lt;/em&gt;-value =&#xA0;0.009). (B) Survival curves for &lt;em&gt;Eg&lt;/em&gt;9PP and &lt;em&gt;Eg&lt;/em&gt;9PP families exhibiting maximum (D1L3) and minimum (D2Y1, D2L2) percentages of affected and dead individuals at the trial uprooting (color legend at bottom left). Survival estimates are plotted for two events, the time of the first &lt;em&gt;Ganoderma&lt;/em&gt; symptom observation (T1S, solid lines), and death due to &lt;em&gt;Ganoderma&lt;/em&gt; infection (TD, dotted lines).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1683/F2.medium.gif" width="440" height="175"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/6/1683/F2.medium.gif" width="440" height="175"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1683/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1683/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484935" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child"><em>Ganoderma</em> disease epidemics in the <em>Eg</em>9PP oil palm multi-parental population. (A) Relationship between the percentages of affected and dead individuals per family at the trial uprooting. Colors corresponding to families are indicated in the legend at the top left, with gray circles indicating the <em>Eg</em>9PP value (<em>n</em> = 1200). Dotted lines represent the linear regression between the percentages of affected and dead individuals (slope = 0.72, Pearson correlation coefficient <em>r</em> = 0.66, <em>p</em>-value = 0.009). (B) Survival curves for <em>Eg</em>9PP and <em>Eg</em>9PP families exhibiting maximum (D1L3) and minimum (D2Y1, D2L2) percentages of affected and dead individuals at the trial uprooting (color legend at bottom left). Survival estimates are plotted for two events, the time of the first <em>Ganoderma</em> symptom observation (T1S, solid lines), and death due to <em>Ganoderma</em> infection (TD, dotted lines).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Linkage mapping of Ganoderma resistance loci</h3><p id="p-28">QTL analysis in the <em>Eg</em>9PP mapping population (<em>n</em> = 604) identified two regions associated with the time of the first symptom observation (T1S), and two with the time of death due to <em>Ganoderma</em> infection (TD), with no overlap between them (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The inclusion of spatial effects in the base model did not change the QTL results for T1S despite the reduced overall LRT level (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/FigureS5.tif">Figure S5</a>). On the contrary, three LRT peaks for TD were lost by adding the spatial effect (middle of LG1 and LG4, bottom of LG14), but a peak at the top of LG3 was increased (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/FigureS5.tif">Figure S5</a>). The strongest QTL was found for T1S at the top of LG1, with an LRT of 6.83 at a significance level (<em>α</em>) of 0.01 (T1S_1@9, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The variance associated with T1S_1@9 was 0.16, corresponding to a risk <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1683/embed/mml-math-41.gif"/></noscript></span></span> = 1.49-fold above the norm for ∼15% of the individuals, with a similar fraction of individuals having lower risk. The distribution of individual BLUP indicated the greatest risk segregation in families from La Mé parents L1, L2, and L3 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). The second QTL for T1S (T1S_1@159) was detected at the bottom on LG1, with a risk of 1.30 (<em>α</em> = 0.29), with risk segregation found mainly in families from parent D2 (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Both QTL detected for TD, TD_3@0, and TD_14@13 at the top of LG3 and LG14, respectively, had high risks (1.45) but low significance levels (<em>α</em> = 0.38 and 0.19, respectively), with a risk BLUP distribution indicating the greatest QTL segregation in families from parents D3 and D5 for TD_3@0, and L1 and L2 for TD_14@13 (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a> and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1683/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic determinism of Ganoderma resistance in the Eg9PP oil palm multi-parental population. (A) LRT testing the presence of a QTL for the time of the first Ganoderma symptom observation (T1S, solid lines) and death due to Ganoderma infection (TD, dotted lines) in the Eg9PP mapping population (n&#xA0;=&#xA0;604). Dashes on the x-axis represent markers on the genetic map. Labels above LRT peaks (Trait_Linkage group@position in cM) indicate QTL with an LRT below a type&#xA0;I error rate of 0.5. (B) BLUP distributions for random QTL effects (QTL names at top left of each panel) for the Eg9PP families of the mapping population (n&#xA0;=&#xA0;604, color legend at top right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1576922106" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic determinism of &lt;em&gt;Ganoderma&lt;/em&gt; resistance in the &lt;em&gt;Eg&lt;/em&gt;9PP oil palm multi-parental population. (A) LRT testing the presence of a QTL for the time of the first &lt;em&gt;Ganoderma&lt;/em&gt; symptom observation (T1S, solid lines) and death due to &lt;em&gt;Ganoderma&lt;/em&gt; infection (TD, dotted lines) in the &lt;em&gt;Eg&lt;/em&gt;9PP mapping population (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;604). Dashes on the &lt;em&gt;x&lt;/em&gt;-axis represent markers on the genetic map. Labels above LRT peaks (Trait_Linkage group@position in cM) indicate QTL with an LRT below a type&#xA0;I error rate of 0.5. (B) BLUP distributions for random QTL effects (QTL names at top left of each panel) for the &lt;em&gt;Eg&lt;/em&gt;9PP families of the mapping population (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;604, color legend at top right).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1683/F3.medium.gif" width="342" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/6/1683/F3.medium.gif" width="342" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1683/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1683/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484928" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Genetic determinism of <em>Ganoderma</em> resistance in the <em>Eg</em>9PP oil palm multi-parental population. (A) LRT testing the presence of a QTL for the time of the first <em>Ganoderma</em> symptom observation (T1S, solid lines) and death due to <em>Ganoderma</em> infection (TD, dotted lines) in the <em>Eg</em>9PP mapping population (<em>n</em> = 604). Dashes on the <em>x</em>-axis represent markers on the genetic map. Labels above LRT peaks (Trait_Linkage group@position in cM) indicate QTL with an LRT below a type I error rate of 0.5. (B) BLUP distributions for random QTL effects (QTL names at top left of each panel) for the <em>Eg</em>9PP families of the mapping population (<em>n</em> = 604, color legend at top right).</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484919/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484919/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">QTL associated with <em>Ganoderma</em> resistance in the <em>Eg</em>9PP oil palm multi-parental population</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Haplotype analysis of the major Ganoderma resistance QTL</h3><p id="p-36">T1S_1@9, the major QTL for <em>Ganoderma</em> resistance, was investigated in further detail by genotyping all sampled <em>Eg</em>9PP individuals (<em>n</em> = 757) with the five SSR markers in the region (LG1, 0–44.5 cM), by adding two additional SSR markers and performing IBD and haplotype analyses. IBD-QTL mapping on the augmented dataset confirmed the presence of a QTL for T1S on LG1, but the LRT peak position shifted to 0 cM on the mEgCIR1713 marker (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). While no QTL were found for TD in this region, a LRT peak was detected with the augmented dataset close to the mEgCIR3803 marker (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). Cox regression models including two fixed effects for haplotypes at the closest markers in each heterotic group indicated significant effects in group A (<em>p</em>-value &lt; 0.01) and group B (<em>p</em>-value &lt; 0.001) for T1S as well as for TD (<em>p</em>-values &lt; 0.01 and &lt; 0.001 for groups A and B, respectively). The four La Mé founder IBD alleles were present in this region in the three La Mé <em>Eg</em>9PP parents, with two recombination events in the T1S_1@9 region in founder meiosis (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). Marked differences in estimated risk were observed for individuals carrying alternative La Mé founder haplotypes: individual with an LF1 paternal allele (LF1_P, <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>) had a 2.1-fold lower risk of being infected by <em>Ganoderma</em> and 2.07-fold lower risk of dying due to <em>Ganoderma</em> compared to the worst allele (LF2_M). The survival estimates showed that the 20% affected palm tree threshold was reached 19.5 yr after planting for individuals carrying the LF2_M allele, as compared to &gt;23.5 yr for those carrying the LF1_P allele (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>). Moreover, this haplotype analysis allowed us to identify favorable alleles within the Yangambi and Deli genetic backgrounds associated with contrasted risks of <em>Ganoderma</em> infection and death (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/FigureS6.tif">Figure S6</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1683/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Haplotype analysis of Ganoderma resistance QTL T1S_1@9. (A) LRT testing the presence of a QTL at the top of linkage group&#xA0;1 for the time of the first Ganoderma symptom observation (T1S, solid lines) and death due to Ganoderma infection (TD, dotted lines) in the Eg9PP sampled individuals (n&#xA0;=&#xA0;757). Dashes on the x-axis represent markers on the genetic map. La M&#xE9; founder (LF1, LF2), and parent (L1 to L3) maternal (M) and paternal (P) IBD alleles are represented by different colors below the LRT lines. (B) Survival curves for the Eg9PP sampled individuals (n&#xA0;=&#xA0;757) according to the IBD allele they inherited from their La M&#xE9; parent. Survival estimates are plotted for two events, i.e., the time of the first Ganoderma symptom observation (T1S, solid lines), and death due to Ganoderma infection (TD, dotted lines)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1576922106" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Haplotype analysis of &lt;em&gt;Ganoderma&lt;/em&gt; resistance QTL T1S_1@9. (A) LRT testing the presence of a QTL at the top of linkage group&#xA0;1 for the time of the first &lt;em&gt;Ganoderma&lt;/em&gt; symptom observation (T1S, solid lines) and death due to &lt;em&gt;Ganoderma&lt;/em&gt; infection (TD, dotted lines) in the &lt;em&gt;Eg&lt;/em&gt;9PP sampled individuals (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;757). Dashes on the &lt;em&gt;x&lt;/em&gt;-axis represent markers on the genetic map. La M&#xE9; founder (LF1, LF2), and parent (L1 to L3) maternal (M) and paternal (P) IBD alleles are represented by different colors below the LRT lines. (B) Survival curves for the &lt;em&gt;Eg&lt;/em&gt;9PP sampled individuals (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;757) according to the IBD allele they inherited from their La M&#xE9; parent. Survival estimates are plotted for two events, &lt;em&gt;i.e.&lt;/em&gt;, the time of the first &lt;em&gt;Ganoderma&lt;/em&gt; symptom observation (T1S, solid lines), and death due to &lt;em&gt;Ganoderma&lt;/em&gt; infection (TD, dotted lines).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1683/F4.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/6/1683/F4.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1683/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1683/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484930" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">Haplotype analysis of <em>Ganoderma</em> resistance QTL T1S_1@9. (A) LRT testing the presence of a QTL at the top of linkage group 1 for the time of the first <em>Ganoderma</em> symptom observation (T1S, solid lines) and death due to <em>Ganoderma</em> infection (TD, dotted lines) in the <em>Eg</em>9PP sampled individuals (<em>n</em> = 757). Dashes on the <em>x</em>-axis represent markers on the genetic map. La Mé founder (LF1, LF2), and parent (L1 to L3) maternal (M) and paternal (P) IBD alleles are represented by different colors below the LRT lines. (B) Survival curves for the <em>Eg</em>9PP sampled individuals (<em>n</em> = 757) according to the IBD allele they inherited from their La Mé parent. Survival estimates are plotted for two events, <em>i.e.</em>, the time of the first <em>Ganoderma</em> symptom observation (T1S, solid lines), and death due to <em>Ganoderma</em> infection (TD, dotted lines).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-38">This study was based on data from monitoring natural field infection of an oil palm multi-parental population over 25 yr. This unique dataset provided an opportunity to map <em>Ganoderma</em> resistance loci in natural conditions according to an economic and efficient <em>in silico</em> approach, <em>i.e.</em>, based on data from breeding programs (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Parisseaux and Bernardo 2004</a>). Four <em>Ganoderma</em> resistance loci were identified by combining different statistical methods, including survival analysis, infection spatial modeling, and a variance component approach for QTL mapping.</p><div id="sec-16" class="subsection"><h3>Disease resistance loci mapping using the Cox regression model</h3><p id="p-39">The QTL mapping approach was developed in the framework of normally distributed quantitative traits. In this study, <em>Ganoderma</em> infection data were treated as survival traits, with censoring because of unobserved events. Such data are commonly found in human or animal studies in which most of the statistical methods were developed. <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Zhao (2005)</a> proposed a statistical approach using a mixed-effects Cox model to study familial effects in alcohol dependence. In salmon, disease resistance loci were mapped in complex populations using a variance component approach on binary survival data (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Gonen <em>et al.</em> 2015</a>). In plants, especially annual ones, survival analyzes are seldom performed because of the experimental potential, but methods have been developed to map QTL associated with flowering time in biparental rice (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Luo <em>et al.</em> 2013</a>) and barley (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Jiang <em>et al.</em> 2014</a>) populations. As suggested in <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Zhao (2005)</a>, QTL mapping was performed in this study using IBD kinship matrices, calculated with Simwalk2 (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Sobel <em>et al.</em> 2001</a>), as covariance matrices of the QTL random effect. This approach is flexible in terms of population design, and was found to be efficient in a former study on agronomic quantitative traits in a complex oil palm pedigree using a linear mixed effect model (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Tisné <em>et al.</em> 2015</a>). Modeling of a spatial effect to account for the variance associated with the natural inoculation of the palm trees was found to be highly significant, and changed the time of death mapping results. However, it is unclear whether it improved the mapping results without additional information, <em>e.g.</em>, from simulation studies. Further developments could improve the mapping method. Modeling QTL effects including two variance components, one for each heterotic group, would enable independent testing of the QTL presence in each genetic background and assessing the importance of nonadditive effects, <em>i.e.</em>, dominance or epistasis. A multi-QTL approach would also be worthwhile as the multiple QTL methods proved their superiority in estimating QTL effects and position (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jansen 1993</a>). This could be achieved using a variable selection approach in the Bayesian framework, as developed for quantitative traits (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bink <em>et al.</em> 2008</a>).</p><p id="p-40">The approach described in this study—flexible, efficient, with further possible extensions—could be interesting for studying disease resistance in plants, especially for perennials, which have limited experimental potential.</p></div><div id="sec-17" class="subsection"><h3>Genetic architecture of Ganoderma disease resistance</h3><p id="p-41">Plant disease resistance is commonly partitioned between R-gene mediated qualitative or complete resistance, and quantitative or incomplete resistance controlled by many quantitative resistance loci (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Poland <em>et al.</em> 2009</a>). In the present study, no QTL were found that completely prevented <em>Ganoderma</em> infection or death, which is consistent with field observations that could not highlight such complete resistance (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Durand-Gasselin <em>et al.</em> 2005</a>). Colocalizations of <em>Ganoderma</em> resistance QTL with oil palm predicted R-genes were observed (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1/TableS8.xlsx">Table S8</a>), but further work would be needed to validate these genes as candidate. Moreover, R-gene mediated complete resistance was observed almost exclusively with biotrophic pathogens, so it is unlikely that such resistance occurs in oil palm, with <em>Ganoderma</em> being an hemi-biotrophic pathogen (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ho and Tan 2015</a>). However, marked delayed effects on the time of the first symptom observation (T1S) and the time of death (TD) were observed among families (threefold to fivefold, respectively), with high variance associated with QTL noted for both T1S and TD. The different QTL patterns observed for T1S and TD, without any colocalizations noted, except at the top of LG1, was a striking result. This is consistent with the correlation at the family level between T1S and TD being looser than expected, indicating variation in the survival time when infected by <em>Ganoderma</em>. This could still be linked to the two stages of infection reported for <em>Ganoderma</em>, <em>i.e.</em>, biotrophic then necrotrophic, as it is known that both induce different immune responses that could lead to different genetic architectures for T1S and TD. <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Poland <em>et al.</em> (2009)</a> put forward six hypotheses on the mechanisms underlying quantitative resistance loci, with the first being the effect of pleiotropic genes regulating morphological and developmental phenotypes. Mapping of yield and morphology traits in <em>Eg</em>9PP (unpublished results) revealed colocalization of TD QTL with fruit weight and composition (TD_3@0) and stem height (TD_14@13), indicating possible pleiotropic effects as a basis of quantitative resistance to <em>Ganoderma</em>.</p><p id="p-42">This is the first study that identified the genetic determinism of <em>Ganoderma</em> resistance in the field, so the detected QTL need further validation in independent experiments, accompanied by a fine mapping approach to identify the underlying genes. Despite a marker density suitable for QTL mapping (one marker every 8 cM on average), the analysis of <em>Eg</em>9PP would benefit from denser marker sets, <em>e.g.</em>, SNP obtained by genotyping-by-sequencing (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Pootakham <em>et al.</em> 2015</a>) or high density genotyping array (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kwong <em>et al.</em> 2016</a>) that were recently developed in oil palm. Haplotype analysis of the T1S_1@9 region exhibited many recombination events (105 in 1.9 Mb), indicating that densifying <em>Eg</em>9PP genotyping in the candidate regions could lead to candidate gene identification. <em>Eg</em>9PP individuals were uprooted but leaf samples are still available, and acquisition of a denser genotyping is in progress. Although field experiments are certainly more relevant, nursery tests with inoculation are interesting to accumulate genetic and omic knowledge on <em>Ganoderma</em> resistance. Unfortunately, the only nursery-based genetic study to date (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Hama-Ali <em>et al.</em> 2015</a>) showed no overlap between field resistant QTL and SSR markers associated with <em>Ganoderma</em> disease incidence in the nursery despite some common markers and genetic backgrounds (Deli × Yangambi). The scope of the study, <em>i.e.</em>, involving 79 individuals from one tolerant and two susceptible families, genotyped with 58 SSRs, made it difficult to conclude on the transferability of the QTL results from the nursery to the field, and further research is needed to map resistance QTL based on nursery inoculated populations.</p></div><div id="sec-18" class="subsection"><h3>Contribution to Ganoderma resistance loci in oil palm breeding programs</h3><p id="p-43">The multi-parental population design allowed us to identify quantitative resistant loci among an extended genetic diversity, and to test their effects in various genetic backgrounds, which should enhance the transferability of results and the sustainability of the selected resistances. The most widely used oil palm breeding scheme was developed in the 1950s from a reciprocal recurrent selection (RRS) scheme to take advantage of heterotic effects on bunch production observed between group A and B genetic backgrounds, represented by Deli and La Mé/Yangambi parents, respectively, in the present study (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gascon and de Berchoux 1964</a>). Despite substantial differences in <em>Ganoderma</em> resistance levels among the three genetic origins assessed, with Deli being highly susceptible in comparison to Yangambi and La Mé material (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Durand-Gasselin <em>et al.</em> 2005</a>), allelic segregation was found in the Deli background for the major QTL T1S_1@9 (D1 and D5 parents), QTL T1S_1@159 (D2 parent), and QTL TD_3@0 (D3 and D5 parents). Hence, RRS would be efficient for both heterotic groups by selecting quantitative resistant alleles at different loci within genetic origins and combining them in A × B producer hybrids.</p><p id="p-44">Localization of <em>Ganoderma</em> resistance loci will facilitate multi-criteria improvement of oil palm, with the possibility of combining known favorable genomic regions, <em>e.g.</em>, for disease and yield related traits, while avoiding unintended unfavorable selection on the others. This is currently feasible since an increasing number of studies have reported QTL for agronomic traits such as bunch production and sex ratio (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ukoskit <em>et al.</em> 2014</a>; <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Tisné <em>et al.</em> 2015</a>), mesocarp oil content (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Teh <em>et al.</em> 2016</a>), fatty acid composition (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Montoya <em>et al.</em> 2014</a>), stem height (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lee <em>et al.</em> 2014</a>), and lipase activity (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Morcillo <em>et al.</em> 2013</a>). The selection of varieties resistant to a combination of diseases will be essential for this multi-criteria improvement, as <em>Ganoderma</em> is found throughout the oil palm cultivation area, combined with other devastating diseases like the <em>Fusarium</em> wilt in Africa, and bud rot in South America. Hence, the global genetic architecture of disease resistance in oil palm needs to be investigated, especially potential trade-off effects related to different pathogens, as shown for other species (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Poland <em>et al.</em> 2009</a>). The multi-parental population presented in this study, and the implemented QTL mapping approach, provide powerful tools for investigating such global genetic architecture of disease resistance in oil palm.</p></div></div><div class="section" id="sec-19"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank N. Marseillac and A. Berger for partly genotyping the population and N. Billotte for providing former genotyping results and collected samples. We acknowledge SOCFIN Indonesia (Indonesia) for planning and carrying out the field trials with CIRAD (France), and authorizing use of the phenotypic data for this study. This research was partly funded by a grant from PalmElit SAS.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-7"><p id="p-46">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041764/-/DC1</a>.</p></li><li class="fn" id="fn-8"><p id="p-47"><em>Communicating editor: J. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-05"><span class="received-label">Received </span>January 5, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-03-28"><span class="accepted-label">Accepted </span>March 28, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Tisné <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adeyanju</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesso</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide association study on resistance to stalk rot diseases in grain sorghum.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D5%26rft.spage%253D1165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Al-Obaidi</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohd-Yusuf</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Razali</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jayapalan</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tey</span>  <span class="cit-name-given-names">C.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identification of proteins of altered abundance in oil palm infected with <em>Ganoderma boninense</em>.</span> <abbr class="cit-jnl-abbrev">Int. J. Mol. Sci.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">5175</span>–<span class="cit-lpage">5192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BMol.%2BSci.%26rft.volume%253D15%26rft.spage%253D5175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bajgain</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rouse</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsilo</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macharia</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhavani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Nested association mapping of stem rust resistance in wheat using genotyping by sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0155760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0155760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.4" data-doi="10.1093/biostatistics/4.1.123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlin</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Frailty modeling for spatially correlated survival data, with application to infant mortality in Minnesota.</span> <abbr class="cit-jnl-abbrev">Biostatistics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft.stitle%253DBiostatistics%26rft.aulast%253DBanerjee%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D123%26rft.epage%253D142%26rft.atitle%253DFrailty%2Bmodeling%2Bfor%2Bspatially%2Bcorrelated%2Bsurvival%2Bdata%252C%2Bwith%2Bapplication%2Bto%2Binfant%2Bmortality%2Bin%2BMinnesota%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252F4.1.123%26rft_id%253Dinfo%253Apmid%252F12925334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvc3RzIjtzOjU6InJlc2lkIjtzOjc6IjQvMS8xMjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.5" data-doi="10.1371/journal.pgen.1005045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stromberg</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Resistance to gray leaf spot of maize: genetic architecture and mechanisms elucidated through nested association mapping and near-isogenic line analysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005045</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005045%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005045%26rft_id%253Dinfo%253Apmid%252F25764179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25764179&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.6" data-doi="10.1007/s00122-004-1901-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Billotte</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marseillac</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Risterucci</span>  <span class="cit-name-given-names">A.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brottier</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Microsatellite-based high density linkage map in oil palm (<em>Elaeis guineensis</em> Jacq.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">754</span>–<span class="cit-lpage">765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBillotte%26rft.auinit1%253DN.%26rft.volume%253D110%26rft.issue%253D4%26rft.spage%253D754%26rft.epage%253D765%26rft.atitle%253DMicrosatellite-based%2Bhigh%2Bdensity%2Blinkage%2Bmap%2Bin%2Boil%2Bpalm%2B%2528Elaeis%2Bguineensis%2BJacq.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1901-8%26rft_id%253Dinfo%253Apmid%252F15723275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1901-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15723275&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227714400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.7" data-doi="10.1007/s00122-010-1284-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Billotte</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jourjon</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marseillac</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flori</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">QTL detection by multi-parent linkage mapping in oil palm (<em>Elaeis guineensis</em> Jacq.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">1673</span>–<span class="cit-lpage">1687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBillotte%26rft.auinit1%253DN.%26rft.volume%253D120%26rft.issue%253D8%26rft.spage%253D1673%26rft.epage%253D1687%26rft.atitle%253DQTL%2Bdetection%2Bby%2Bmulti-parent%2Blinkage%2Bmapping%2Bin%2Boil%2Bpalm%2B%2528Elaeis%2Bguineensis%2BJacq.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1284-y%26rft_id%253Dinfo%253Apmid%252F20182696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1284-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20182696&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.8" data-doi="10.1007/s10681-007-9516-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ter Braak</span>  <span class="cit-name-given-names">C. J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Bayesian analysis of complex traits in pedigreed plant populations.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D161%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-007-9516-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-007-9516-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000254877800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breton</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hariadi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubis</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Franqueville</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Characterization of parameters for the development of an early screening test for basal stem rot tolerance in oil palm progenies.</span> <abbr class="cit-jnl-abbrev">J. Oil Palm Res.</abbr> <span class="cit-vol">2006</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BOil%2BPalm%2BRes.%26rft.volume%253D2006%26rft.spage%253D24%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.10" data-doi="10.1111/j.1469-8137.2009.03049.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brun</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chèvre</span>  <span class="cit-name-given-names">A.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitt</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Besnard</span>  <span class="cit-name-given-names">A.-L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quantitative resistance increases the durability of qualitative resistance to <em>Leptosphaeria maculans</em> in <em>Brassica napus</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DBrun%26rft.auinit1%253DH.%26rft.volume%253D185%26rft.issue%253D1%26rft.spage%253D285%26rft.epage%253D299%26rft.atitle%253DQuantitative%2Bresistance%2Bincreases%2Bthe%2Bdurability%2Bof%2Bqualitative%2Bresistance%2Bto%2BLeptosphaeria%2Bmaculans%2Bin%2BBrassica%2Bnapus.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03049.x%26rft_id%253Dinfo%253Apmid%252F19814776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03049.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19814776&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272344800027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatarinova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozana</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nadzirah</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azizi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evidence-based gene models for structural and functional annotations of the oil palm genome.</span> arXiv:1702.07246.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical threshold values for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.13" data-doi="10.1007/s11749-006-0023-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Claeskens</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguti</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">One-sided tests in shared frailty models.</span> <abbr class="cit-jnl-abbrev">Test</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTest%26rft.volume%253D17%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11749-006-0023-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11749-006-0023-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000254462600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.14" data-doi="10.1007/s11295-014-0804-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cochard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrasco-Lacombe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pomiès</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dufayard</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suryana</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Pedigree-based linkage map in two genetic groups of oil palm.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-014-0804-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-014-0804-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockram</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scuderi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furuki</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fine-mapping the wheat Snn1 locus conferring sensitivity to the <em>Parastagonospora nodorum</em> necrotrophic effector SnTox1 using an eight founder multiparent advanced generation inter-cross population.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2257</span>–<span class="cit-lpage">2266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D5%26rft.spage%253D2257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rees</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Ganoderma boninense</em> in oil palm plantations: current thinking on epidemiology, resistance and pathology.</span> <abbr class="cit-jnl-abbrev">Planter</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanter%26rft.volume%253D87%26rft.spage%253D515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1683.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corley</span>  <span class="cit-name-given-names">R. H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinker</span>  <span class="cit-name-given-names">P. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-source"><em>Selection and Breeding</em>.</span> <span class="cit-publ-name">Oil Palm Blackwell Sci. Ltd.</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Models and life-tables regression.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D34%26rft.spage%253D187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cressie</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <em>Statistics for Spatial Data: Wiley Series in Probability and Statistics</em>. <span class="cit-publ-name">Wiley-Intersci.</span> <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cros</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sánchez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cochard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samper</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Estimation of genealogical coancestry in plant species using a pedigree reconstruction algorithm and application to an oil palm breeding population.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">981</span>–<span class="cit-lpage">994</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duchateau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindsey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legrand</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguti</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The shared frailty model and the power for heterogeneity tests in multicenter trials.</span> <abbr class="cit-jnl-abbrev">Comput. Stat. Data Anal.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">603</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BStat.%2BData%2BAnal.%26rft.volume%253D40%26rft.spage%253D603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durand-Gasselin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asmady</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flori</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacquemard</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayun</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Possible sources of genetic resistance in oil palm (<em>Elaeis guineensis</em> Jacq.) to basal stem rot caused by <em>Ganoderma boninense</em>—prospects for future breeding.</span> <abbr class="cit-jnl-abbrev">Mycopathologia</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycopathologia%26rft.stitle%253DMycopathologia%26rft.aulast%253DDurand-Gasselin%26rft.auinit1%253DT.%26rft.volume%253D159%26rft.issue%253D1%26rft.spage%253D93%26rft.epage%253D100%26rft.atitle%253DPossible%2Bsources%2Bof%2Bgenetic%2Bresistance%2Bin%2Boil%2Bpalm%2B%2528Elaeis%2Bguineensis%2BJacq.%2529%2Bto%2Bbasal%2Bstem%2Brot%2Bcaused%2Bby%2BGanoderma%2Bboninense--prospects%2Bfor%2Bfuture%2Bbreeding.%26rft_id%253Dinfo%253Apmid%252F15750739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15750739&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.23" data-doi="10.1101/gr.118786.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durrant</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tayem</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Collaborative cross mice and their power to map host susceptibility to <em>Aspergillus fumigatus</em> infection.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1239</span>–<span class="cit-lpage">1248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.118786.110%26rft_id%253Dinfo%253Apmid%252F21493779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIzOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.6.1683.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Franqueville</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asmady</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacquemard</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayun</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durand-Gasselin</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Indications on sources of oil palm (<em>Elaeis guineensis</em> Jacq.) genetic resistance and susceptibility to <em>Ganoderma</em> sp., the cause of basal stem rot</span>, pp. <span class="cit-fpage">420</span>–<span class="cit-lpage">431</span> in <em>Cutting-edge Technologies for Sustained Competitiveness: Proceedings of the 2001 PIPOC International Palm Oil Congress</em>, <em>Agriculture Conference</em>, 20–22 August, 2001. <span class="cit-publ-name">Malaysian Palm Oil Board (MPOB)</span>, <em>Kuala Lumpur</em>, <em>Malaysia</em>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-other" id="cit-7.6.1683.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gascon</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Berchoux</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite> <span class="cit-pub-date">1964</span> <span class="cit-article-title">Caractéristiques de la production d’<em>Elaeis guineensis</em> (Jacq.) de diverses origines et leurs croisements. Appl. À Sélection Palmier À Huile.</span> <span class="cit-source">Oléagineux</span> <span class="cit-vol">19</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOl%25E9agineux%26rft.volume%253D19%26rft.spage%253D75%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Mapping quantitative trait loci in complex pedigrees: a two-step variance component approach.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">2081</span>–<span class="cit-lpage">2092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGeorge%26rft.auinit1%253DA.%2BW.%26rft.volume%253D156%26rft.issue%253D4%26rft.spage%253D2081%26rft.epage%253D2092%26rft.atitle%253DMapping%2BQuantitative%2BTrait%2BLoci%2Bin%2BComplex%2BPedigrees%253A%2BA%2BTwo-Step%2BVariance%2BComponent%2BApproach%26rft_id%253Dinfo%253Apmid%252F11102397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ni80LzIwODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baranski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorland</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norris</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grove</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mapping and validation of a major QTL affecting resistance to pancreas disease (salmonid alphavirus) in Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1683.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falls</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooks</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <em>Cri-Map</em>. <span class="cit-publ-name">CRI-MAP Doc</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.29" data-doi="10.1016/j.pmpp.2012.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gururani</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkatesh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nookaraju</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Plant disease resistance genes: current status and future directions.</span> <abbr class="cit-jnl-abbrev">Physiol. Mol. Plant Pathol.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BMol.%2BPlant%2BPathol.%26rft.volume%253D78%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pmpp.2012.01.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pmpp.2012.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hama-Ali</span>  <span class="cit-name-given-names">E. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panandam</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alwee</span>  <span class="cit-name-given-names">S. S. R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheong</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Association between basal stem rot disease and simple sequence repeat markers in oil palm, <em>Elaeis guineensis</em> Jacq.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D202%26rft.spage%253D199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">C.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Y.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Molecular defense response of oil palm to <em>Ganoderma</em> infection.</span> <abbr class="cit-jnl-abbrev">Phytochemistry</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytochemistry%26rft.volume%253D114%26rft.spage%253D160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">C.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Y.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeoh</span>  <span class="cit-name-given-names">K.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghazali</span>  <span class="cit-name-given-names">A.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yee</span>  <span class="cit-name-given-names">W.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">De novo transcriptome analyses of host-fungal interactions in oil palm (<em>Elaeis guineensis</em> Jacq.).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D66%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Idris</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kushairi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ismail</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ariffin</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Selection for partial resistance in oil palm progenies to <em>Ganoderma</em> basal stem rot.</span> <abbr class="cit-jnl-abbrev">J. Oil Palm Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BOil%2BPalm%2BRes.%26rft.volume%253D16%26rft.spage%253D12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Interval mapping of multiple quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJansen%26rft.auinit1%253DR.%2BC.%26rft.volume%253D135%26rft.issue%253D1%26rft.spage%253D205%26rft.epage%253D211%26rft.atitle%253DInterval%2Bmapping%2Bof%2Bmultiple%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F8224820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM1LzEvMjA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeennor</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volkaert</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mapping of quantitative trait loci (QTLs) for oil yield using SSRs and gene-based markers in African oil palm (<em>Elaeis guineensis</em> Jacq.).</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D10%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cox regression model for dissecting genetic architecture of survival time.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">472</span>–<span class="cit-lpage">476</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.volume%253D104%26rft.spage%253D472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.37" data-doi="10.1038/ng.747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kump</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belcher</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study of quantitative resistance to southern leaf blight in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKump%26rft.auinit1%253DK.%2BL.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D163%26rft.epage%253D168%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bquantitative%2Bresistance%2Bto%2Bsouthern%2Bleaf%2Bblight%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.747%26rft_id%253Dinfo%253Apmid%252F21217757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217757&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kwong</span>  <span class="cit-name-given-names">Q. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teh</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heng</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Development and validation of a high-density SNP genotyping array for African oil palm.</span> <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1132</span>–<span class="cit-lpage">1141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft.volume%253D9%26rft.spage%253D1132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stinchcombe</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identifying the genes underlying quantitative traits: a rationale for the QTN programme.</span> <abbr class="cit-jnl-abbrev">AoB Plants</abbr> <span class="cit-vol">6</span>: pii: plu004.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An optimal parametric proportional hazards model for mapping heading time loci in rice.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D192%26rft.spage%253D37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montoya</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cochard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flori</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cros</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic architecture of palm oil fatty acid composition in cultivated oil palm (<em>Elaeis guineensis</em> Jacq.) compared to its wild relative <em>E. oleifera</em> (HBK) cortés.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e95412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De95412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morcillo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cros</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Billotte</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ngando-Ebongue</span>  <span class="cit-name-given-names">G.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Domonhédo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Improving palm oil quality through identification and mapping of the lipase gene causing oil deterioration.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D2160%26rft_id%253Dinfo%253Apmid%252F23857501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23857501&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nusaibah</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akmar</span>  <span class="cit-name-given-names">A. S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Idris</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sariah</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pauzi</span>  <span class="cit-name-given-names">Z. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Involvement of metabolites in early defense mechanism of oil palm (<em>Elaeis guineensis</em> Jacq.) against <em>Ganoderma</em> disease.</span> <abbr class="cit-jnl-abbrev">Plant Physiol. Biochem.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%2BBiochem.%26rft.volume%253D109%26rft.spage%253D156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.44" data-doi="10.1016/j.cropro.2003.10.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oerke</span>  <span class="cit-name-given-names">E.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dehne</span>  <span class="cit-name-given-names">H.-W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Safeguarding production—losses in major crops and the role of crop protection.</span> <abbr class="cit-jnl-abbrev">Crop Prot.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BProt.%26rft.volume%253D23%26rft.spage%253D275%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cropro.2003.10.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cropro.2003.10.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220036000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parisseaux</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">In silico mapping of quantitative trait loci in maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">508</span>–<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DParisseaux%26rft.auinit1%253DB.%26rft.volume%253D109%26rft.issue%253D3%26rft.spage%253D508%26rft.epage%253D514%26rft.atitle%253DIn%2Bsilico%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci%2Bin%2Bmaize.%26rft_id%253Dinfo%253Apmid%252F15150690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15150690&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223097700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.46" data-doi="10.1016/j.tplants.2008.10.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balint-Kurti</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pratt</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Shades of gray: the world of quantitative disease resistance.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D29%26rft.atitle%253DShades%2Bof%2Bgray%253A%2Bthe%2Bworld%2Bof%2Bquantitative%2Bdisease%2Bresistance.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2008.10.006%26rft_id%253Dinfo%253Apmid%252F19062327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2008.10.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19062327&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263015000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pootakham</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jomchai</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruang-areerate</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shearman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonthirod</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide SNP discovery and identification of QTL associated with agronomic traits in oil palm using genotyping-by-sequencing (GBS).</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">288</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.volume%253D105%26rft.spage%253D288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quenouille</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulhiac</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moury</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palloix</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative trait loci from the host genetic background modulate the durability of a resistance gene: a rational basis for sustainable resistance breeding in plants.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1683.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Development Core Team</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <em>R: A Language and Environment for Statistical Computing</em>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.50" data-doi="10.1038/nature12309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ong-Abdullah</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Low</span>  <span class="cit-name-given-names">E.-T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manaf</span>  <span class="cit-name-given-names">M. A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosli</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Oil palm genome sequence reveals divergence of interfertile species in Old and New worlds.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">500</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D500%26rft.spage%253D335%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12309%26rft_id%253Dinfo%253Apmid%252F23883927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12309&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23883927&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323112400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sobel</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Descent graphs in pedigree analysis: applications to haplotyping, location scores, and marker-sharing statistics.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DSobel%26rft.auinit1%253DE.%26rft.volume%253D58%26rft.issue%253D6%26rft.spage%253D1323%26rft.epage%253D1337%26rft.atitle%253DDescent%2Bgraphs%2Bin%2Bpedigree%2Banalysis%253A%2Bapplications%2Bto%2Bhaplotyping%252C%2Blocation%2Bscores%252C%2Band%2Bmarker-sharing%2Bstatistics.%26rft_id%253Dinfo%253Apmid%252F8651310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8651310&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UM27100026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.52" data-doi="10.1159/000053366"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sobel</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sengul</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weeks</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Multipoint estimation of identity-by-descent probabilities at arbitrary positions among marker loci on general pedigrees.</span> <abbr class="cit-jnl-abbrev">Hum. Hered.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bheredity%26rft.stitle%253DHum%2BHered%26rft.aulast%253DSobel%26rft.auinit1%253DE.%26rft.volume%253D52%26rft.issue%253D3%26rft.spage%253D121%26rft.epage%253D131%26rft.atitle%253DMultipoint%2Bestimation%2Bof%2Bidentity-by-descent%2Bprobabilities%2Bat%2Barbitrary%2Bpositions%2Bamong%2Bmarker%2Bloci%2Bon%2Bgeneral%2Bpedigrees.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000053366%26rft_id%253Dinfo%253Apmid%252F11588394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000053366&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11588394&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171247900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tee</span>  <span class="cit-name-given-names">S.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Y.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdullah</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ong-Abdullah</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">C.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transcriptome of oil palm (<em>Elaeis guineensis</em> Jacq.) roots treated with <em>Ganoderma boninense</em>.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D9%26rft.spage%253D377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teh</span>  <span class="cit-name-given-names">C.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">A.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwong</span>  <span class="cit-name-given-names">Q.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apparow</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chew</span>  <span class="cit-name-given-names">F.-T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide association study identifies three key loci for high mesocarp oil content in perennial crop oil palm.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">19075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D19075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-web" id="cit-7.6.1683.55"><div class="cit-metadata unstructured">Therneau T. M., Therneau M. T. M., 2015 Package “coxme” Mixed Effects Cox Models R Package Version 2. <a href="https://CRAN.R-project.org/package=coxme">https://CRAN.R-project.org/package=coxme</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.56" data-doi="10.1186/s12864-015-1985-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tisné</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cros</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pomiès</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riou</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mixed model approach for IBD-based QTL mapping in a complex oil palm pedigree.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D798%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1985-3%26rft_id%253Dinfo%253Apmid%252F26472667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1985-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26472667&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ukoskit</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chanroj</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhusudsawang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pipatchartlearnwong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tangphatsornruang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Oil palm (<em>Elaeis guineensis</em> Jacq.) linkage map, and quantitative trait locus analysis for sex ratio and related traits.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">424</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D33%26rft.spage%253D415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-web" id="cit-7.6.1683.58"><div class="cit-metadata unstructured">USDA, 2015 Oilseeds: world markets and trade. <a href="http://www.thecropsite.com/reports/?id=4943">http://www.thecropsite.com/reports/?id=4943</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.59" data-doi="10.2307/2061224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaupel</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manton</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stallard</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">The impact of heterogeneity in individual frailty on the dynamics of mortality.</span> <abbr class="cit-jnl-abbrev">Demography</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDemography%26rft.stitle%253DDemography%26rft.aulast%253DVaupel%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D439%26rft.epage%253D454%26rft.atitle%253DThe%2Bimpact%2Bof%2Bheterogeneity%2Bin%2Bindividual%2Bfrailty%2Bon%2Bthe%2Bdynamics%2Bof%2Bmortality.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2061224%26rft_id%253Dinfo%253Apmid%252F510638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2061224&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=510638&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1979HK16900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verhoeven</span>  <span class="cit-name-given-names">K. J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Using mating designs to uncover QTL and the genetic architecture of complex traits.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D96%26rft.spage%253D139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000234843500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.61" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6IjkzLzEvNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vu</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knuiman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Estimation in semiparametric marginal shared gamma frailty models.</span> <abbr class="cit-jnl-abbrev">Aust. N. Z. J. Stat.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">489</span>–<span class="cit-lpage">501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BN.%2BZ.%2BJ.%2BStat.%26rft.volume%253D44%26rft.spage%253D489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.63" data-doi="10.1007/s00122-006-0418-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croft</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Associations between DNA markers and resistance to diseases in sugarcane and effects of population substructure.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWei%26rft.auinit1%253DX.%26rft.volume%253D114%26rft.issue%253D1%26rft.spage%253D155%26rft.epage%253D164%26rft.atitle%253DAssociations%2Bbetween%2BDNA%2Bmarkers%2Band%2Bresistance%2Bto%2Bdiseases%2Bin%2Bsugarcane%2Band%2Beffects%2Bof%2Bpopulation%2Bsubstructure.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0418-8%26rft_id%253Dinfo%253Apmid%252F17047910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0418-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17047910&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242154000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.64" data-doi="10.1089/10665270050081478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A greedy algorithm for aligning DNA sequences.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D7%26rft.issue%253D1-2%26rft.spage%253D203%26rft.epage%253D214%26rft.atitle%253DA%2Bgreedy%2Balgorithm%2Bfor%2Baligning%2BDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1089%252F10665270050081478%26rft_id%253Dinfo%253Apmid%252F10890397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/10665270050081478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10890397&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1683.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087833300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1683.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mixed-effects Cox models of alcohol dependence in extended families.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">S127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D6%26rft.spage%253DS127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/6/1683.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1665" title="Oas1b-dependent Immune Transcriptional Profiles of West Nile Virus Infection in the Collaborative Cross" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1693" title="Identification of Nitrogen Consumption Genetic Variants in Yeast Through QTL Mapping and Bulk Segregant RNA-Seq Analyses" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/6"><span>Volume 7 Issue 6, June 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/6.cover-source.jpg?itok=5_J-OKHc" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1683" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--FnVrNFZl5DWPcCTsVTgmVF8oOvjO-lvazD8Tg6eYA4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/6/1683" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/6/1683" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/6/1683" class="active" data-icon-position="" data-hide-link-title="0">Identification of Ganoderma Disease Resistance Loci Using Natural Field Infection of an Oil Palm Multiparental Population</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484774" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-IBHc2eMpaKYNXoo6CK8673N9vHub-zJx4lLlumnSdTs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1683.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/6/1683" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-pc2ONS40dbUsp_9wGRp2xDlD5w8LkU7KrrG8ZQpBcxc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/6/1683" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1683.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484774" id="node484774--2" data-pisa="ggg;7/6/1683" data-pisa-master="ggg;g3.117.041764" data-apath="/ggg/7/6/1683.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of <em>Ganoderma</em> Disease Resistance Loci Using Natural Field Infection of an Oil Palm Multiparental Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sébastien Tisné</span>, <span class="highwire-citation-author" data-delta="1">Virginie Pomiès</span>, <span class="highwire-citation-author" data-delta="2">Virginie Riou</span>, <span class="highwire-citation-author" data-delta="3">Indra Syahputra</span>, <span class="highwire-citation-author" data-delta="4">Benoît Cochard</span> and <span class="highwire-citation-author" data-delta="5">Marie Denis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1683-1692;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041764 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484774--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sébastien Tisné </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S%C3%A9bastien%2BTisn%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tisn%C3%A9%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AS%25C3%25A9bastien%2BTisn%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sebastien.tisne@cirad.fr" class="" data-icon-position="" data-hide-link-title="0">sebastien.tisne@cirad.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Virginie Pomiès </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginie%2BPomi%C3%A8s%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pomi%C3%A8s%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginie%2BPomi%25C3%25A8s%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Virginie Riou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginie%2BRiou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riou%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginie%2BRiou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Indra Syahputra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>P.T Socfindo Medan, 20001, Indonesia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Indra%2BSyahputra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Syahputra%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIndra%2BSyahputra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Benoît Cochard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>PalmElit SAS, 34980 Montferrier sur Lez, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beno%C3%AEt%2BCochard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cochard%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeno%25C3%25AEt%2BCochard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Marie Denis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie%2BDenis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Denis%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie%2BDenis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/6/1683" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/6/1683&amp;title=Identification%20of%20Ganoderma%20Disease%20Resistance%20Loci%20Using%20Natural%20Field%20Infection%20of%20an%20Oil%20Palm%20Multiparental%20Population" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/6/1683&amp;title=Identification%20of%20Ganoderma%20Disease%20Resistance%20Loci%20Using%20Natural%20Field%20Infection%20of%20an%20Oil%20Palm%20Multiparental%20Population" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/6/1683&amp;title=Identification%20of%20Ganoderma%20Disease%20Resistance%20Loci%20Using%20Natural%20Field%20Infection%20of%20an%20Oil%20Palm%20Multiparental%20Population" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/6/1683&amp;text=Identification%20of%20Ganoderma%20Disease%20Resistance%20Loci%20Using%20Natural%20Field%20Infection%20of%20an%20Oil%20Palm%20Multiparental%20Population" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/6/1683&amp;title=Identification%20of%20Ganoderma%20Disease%20Resistance%20Loci%20Using%20Natural%20Field%20Infection%20of%20an%20Oil%20Palm%20Multiparental%20Population" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/6/1683&amp;t=Identification%20of%20Ganoderma%20Disease%20Resistance%20Loci%20Using%20Natural%20Field%20Infection%20of%20an%20Oil%20Palm%20Multiparental%20Population" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/6/1683&amp;title=Identification%20of%20Ganoderma%20Disease%20Resistance%20Loci%20Using%20Natural%20Field%20Infection%20of%20an%20Oil%20Palm%20Multiparental%20Population" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/6/1683&amp;title=Identification%20of%20Ganoderma%20Disease%20Resistance%20Loci%20Using%20Natural%20Field%20Infection%20of%20an%20Oil%20Palm%20Multiparental%20Population" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484774/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484774" id="node484774--3" data-pisa="ggg;7/6/1683" data-pisa-master="ggg;g3.117.041764" data-apath="/ggg/7/6/1683.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of <em>Ganoderma</em> Disease Resistance Loci Using Natural Field Infection of an Oil Palm Multiparental Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sébastien Tisné</span>, <span class="highwire-citation-author" data-delta="1">Virginie Pomiès</span>, <span class="highwire-citation-author" data-delta="2">Virginie Riou</span>, <span class="highwire-citation-author" data-delta="3">Indra Syahputra</span>, <span class="highwire-citation-author" data-delta="4">Benoît Cochard</span> and <span class="highwire-citation-author" data-delta="5">Marie Denis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1683-1692;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041764 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484774--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sébastien Tisné </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S%C3%A9bastien%2BTisn%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tisn%C3%A9%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AS%25C3%25A9bastien%2BTisn%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sebastien.tisne@cirad.fr" class="" data-icon-position="" data-hide-link-title="0">sebastien.tisne@cirad.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Virginie Pomiès </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginie%2BPomi%C3%A8s%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pomi%C3%A8s%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginie%2BPomi%25C3%25A8s%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Virginie Riou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Virginie%2BRiou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riou%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVirginie%2BRiou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Indra Syahputra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>P.T Socfindo Medan, 20001, Indonesia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Indra%2BSyahputra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Syahputra%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIndra%2BSyahputra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Benoît Cochard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>PalmElit SAS, 34980 Montferrier sur Lez, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beno%C3%AEt%2BCochard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cochard%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeno%25C3%25AEt%2BCochard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Marie Denis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Improvement and Adaptation of Mediterranean and Tropical Plant Research Unit (CIRAD, UMR AGAP), Montpellier, 34398 France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie%2BDenis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Denis%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie%2BDenis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1683&amp;rft.epage=1692&amp;rft.atitle=Identification%20of%20Ganoderma%20Disease%20Resistance%20Loci%20Using%20Natural%20Field%20Infection%20of%20an%20Oil%20Palm%20Multiparental%20Population&amp;rft.volume=7&amp;rft.issue=6&amp;rft.date=2017-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Tisn%C3%A9%2C+S%C3%A9bastien&amp;rft.au=Pomi%C3%A8s%2C+Virginie&amp;rft.au=Riou%2C+Virginie&amp;rft.au=Syahputra%2C+Indra&amp;rft.au=Cochard%2C+Beno%C3%AEt&amp;rft.au=Denis%2C+Marie"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484774/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484774/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484774/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484774/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484774/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484774/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484774/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484774/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484774/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484774/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484774/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484774/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/6/1683" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/6/1683.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/6/1683.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/6/1683.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/6/1683.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
