<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/6/1721" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/6/1721.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/6/1721.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/6/1721.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/6/1721/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/6/1721.atom" />
<meta name="HW.pisa" content="ggg;7/6/1721" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Approaches in Characterizing Genetic Structure and Mapping in a Rice Multiparental Population" />
<meta name="DC.Identifier" content="10.1534/g3.117.042101" />
<meta name="DC.Date" content="2017-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2017 Raghavan et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Multi-parent Advanced Generation Intercross (MAGIC) populations are fast becoming mainstream tools for research and breeding, along with the technology and tools for analysis. This paper demonstrates the analysis of a rice MAGIC population from data filtering to imputation and processing of genetic data to characterizing genomic structure, and finally quantitative trait loci (QTL) mapping. In this study, 1316 S6:8 indica MAGIC (MI) lines and the eight founders were sequenced using Genotyping by Sequencing (GBS). As the GBS approach often includes missing data, the first step was to impute the missing SNPs. The observable number of recombinations in the population was then explored. Based on this case study, a general outline of procedures for a MAGIC analysis workflow is provided, as well as for QTL mapping of agronomic traits and biotic and abiotic stress, using the results from both association and interval mapping approaches. QTL for agronomic traits (yield, flowering time, and plant height), physical (grain length and grain width) and cooking properties (amylose content) of the rice grain, abiotic stress (submergence tolerance), and biotic stress (brown spot disease) were mapped. Through presenting this extensive analysis in the MI population in rice, we highlight important considerations when choosing analytical approaches. The methods and results reported in this paper will provide a guide to future genetic analysis methods applied to multi-parent populations." />
<meta name="DC.Contributor" content="Chitra Raghavan" />
<meta name="DC.Contributor" content="Ramil Mauleon" />
<meta name="DC.Contributor" content="Vanica Lacorte" />
<meta name="DC.Contributor" content="Monalisa Jubay" />
<meta name="DC.Contributor" content="Hein Zaw" />
<meta name="DC.Contributor" content="Justine Bonifacio" />
<meta name="DC.Contributor" content="Rakesh Kumar Singh" />
<meta name="DC.Contributor" content="B. Emma Huang" />
<meta name="DC.Contributor" content="Hei Leung" />
<meta name="article:published_time" content="2017-06-01" />
<meta name="article:section" content="Multiparental Population" />
<meta name="citation_title" content="Approaches in Characterizing Genetic Structure and Mapping in a Rice Multiparental Population" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Multi-parent Advanced Generation Intercross (MAGIC) populations are fast becoming mainstream tools for research and breeding, along with the technology and tools for analysis. This paper demonstrates the analysis of a rice MAGIC population from data filtering to imputation and processing of genetic data to characterizing genomic structure, and finally quantitative trait loci (QTL) mapping. In this study, 1316 S6:8 &lt;i&gt;indica&lt;/i&gt; MAGIC (MI) lines and the eight founders were sequenced using Genotyping by Sequencing (GBS). As the GBS approach often includes missing data, the first step was to impute the missing SNPs. The observable number of recombinations in the population was then explored. Based on this case study, a general outline of procedures for a MAGIC analysis workflow is provided, as well as for QTL mapping of agronomic traits and biotic and abiotic stress, using the results from both association and interval mapping approaches. QTL for agronomic traits (yield, flowering time, and plant height), physical (grain length and grain width) and cooking properties (amylose content) of the rice grain, abiotic stress (submergence tolerance), and biotic stress (brown spot disease) were mapped. Through presenting this extensive analysis in the MI population in rice, we highlight important considerations when choosing analytical approaches. The methods and results reported in this paper will provide a guide to future genetic analysis methods applied to multi-parent populations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/06/01" />
<meta name="citation_mjid" content="ggg;7/6/1721" />
<meta name="citation_id" content="7/6/1721" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/6/1721" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/6/1721.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/6/1721.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/6/1721.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.042101" />
<meta name="citation_pmid" content="28592653" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Population" />
<meta name="citation_firstpage" content="1721" />
<meta name="citation_lastpage" content="1730" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Chitra Raghavan" />
<meta name="citation_author_institution" content="Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines" />
<meta name="citation_author" content="Ramil Mauleon" />
<meta name="citation_author_institution" content="Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines" />
<meta name="citation_author" content="Vanica Lacorte" />
<meta name="citation_author_institution" content="Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines" />
<meta name="citation_author" content="Monalisa Jubay" />
<meta name="citation_author_institution" content="Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines" />
<meta name="citation_author" content="Hein Zaw" />
<meta name="citation_author_institution" content="Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines" />
<meta name="citation_author" content="Justine Bonifacio" />
<meta name="citation_author_institution" content="Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines" />
<meta name="citation_author" content="Rakesh Kumar Singh" />
<meta name="citation_author_institution" content="Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines" />
<meta name="citation_author" content="B. Emma Huang" />
<meta name="citation_author_institution" content="Discovery Sciences, Janssen R&amp;D, Menlo Park, California 94025" />
<meta name="citation_author" content="Hei Leung" />
<meta name="citation_author_institution" content="Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines" />
<meta name="citation_author_email" content="h.leung@irri.org" />
<meta name="citation_reference" content="citation_title=Amylose Content of Milled Rice;citation_title=Approved methods of analysis. Method 61â03.01;citation_pages=4;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. Alexandrov;citation_author=S. Tai;citation_author=W. Wang;citation_author=L. Mansueto;citation_author=K. Palis;citation_title=SNP-Seek database of SNPs derived from 3000 rice genomes.;citation_pages=D1023-D1027;citation_volume=43;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=N. Bandillo;citation_author=C. Raghavan;citation_author=P. A. Muyco;citation_author=M. A. L. Sevilla;citation_author=I. T. Lobina;citation_title=Multi-parent advanced generation inter-cross (MAGIC) populations in rice: progress and potential for genetics research and breeding.;citation_pages=11-15;citation_volume=6;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. M. Beissinger;citation_author=C. N. Hirsch;citation_author=R. S. Sekhon;citation_author=J. M. Foerster;citation_author=J. M. Johnson;citation_title=Marker density and read depth for genotyping populations using genotyping-by-sequencing.;citation_volume=193;citation_year=2013;citation_pmid=23410831;citation_doi=10.1534/genetics.112.147710" />
<meta name="citation_reference" content="citation_journal_title=Plant Prod. Sci.;citation_author=K. Cui;citation_author=S. Peng;citation_author=Y. Ying;citation_author=S. Yu;citation_author=C. Xu;citation_title=Molecular dissection of the relationships among tiller number, plant height and heading date in rice.;citation_pages=309-318;citation_volume=7;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. Fan;citation_author=Y. Xing;citation_author=H. Mao;citation_author=T. Lu;citation_author=B. Han;citation_title=GS3, a major QTL for grain length and weight and minor QTL for grain width and thickness in rice, encodes a putative transmembrane protein.;citation_pages=1164-1171;citation_volume=112;citation_year=2006;citation_issue=6;citation_pmid=16453132;citation_doi=10.1007/s00122-006-0218-1" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Gao;citation_author=M. Jin;citation_author=X.-M. Zheng;citation_author=J. Chen;citation_author=D. Yuan;citation_title=Days to heading 7, a major quantitative locus determining photoperiod sensitivity and regional adaptation in rice.;citation_volume=111;citation_year=2014;citation_pmid=25378698;citation_doi=10.1073/pnas.1418204111" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=Z. J. C. Gonzaga;citation_author=J. Carandang;citation_author=D. L. Sanchez;citation_author=D. J. Mackill;citation_author=E. M. Septiningsih;citation_title=Mapping additional QTLs from FR13A to increase submergence tolerance in rice beyond SUB1.;citation_pages=627-636;citation_volume=209;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Harushima;citation_author=M. Yano;citation_author=A. Shomura;citation_author=M. Sato;citation_author=T. Shimano;citation_title=A High-Density Rice Genetic Linkage Map with 2275 Markers Using a Single F2 Population;citation_pages=479-494;citation_volume=148;citation_year=1998;citation_issue=1;citation_pmid=9475757" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. E. Huang;citation_author=A. W. George;citation_title=R/mpMap: a computational platform for the genetic analysis of multiparent recombinant inbred lines.;citation_volume=27;citation_year=2011;citation_pmid=21217121;citation_doi=10.1093/bioinformatics/btq719" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=B. E. Huang;citation_author=A. W. George;citation_author=K. L. Forrest;citation_author=A. Kilian;citation_author=M. J. Hayden;citation_title=A multiparent advanced generation inter-cross population for genetic analysis in wheat.;citation_pages=826-839;citation_volume=10;citation_year=2012;citation_issue=7;citation_pmid=22594629;citation_doi=10.1111/j.1467-7652.2012.00702.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. E. Huang;citation_author=C. Raghavan;citation_author=R. Mauleon;citation_author=K. W. Broman;citation_author=H. Leung;citation_title=Efficient imputation of missing markers in low-coverage genotyping-by-sequencing data from multiparental crosses.;citation_volume=197;citation_year=2014;citation_pmid=24583583;citation_doi=10.1534/genetics.113.158014" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. E. Huang;citation_author=K. L. Verbyla;citation_author=A. P. Verbyla;citation_author=C. Raghavan;citation_author=V. K. Singh;citation_title=MAGIC populations in crops: current status and future prospects.;citation_pages=999-1017;citation_volume=128;citation_year=2015;citation_pmid=25855139;citation_doi=10.1007/s00122-015-2506-0" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Huang;citation_author=Q. Feng;citation_author=Q. Qian;citation_author=Q. Zhao;citation_author=L. Wang;citation_title=High-throughput genotyping by whole-genome resequencing.;citation_volume=19;citation_year=2009;citation_pmid=19420380;citation_doi=10.1101/gr.089516.108" />
<meta name="citation_reference" content="citation_title=Standard Evaluation System for Rice;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=H. Leung;citation_author=C. Raghavan;citation_author=B. Zhou;citation_author=R. Oliva;citation_author=I. R. Choi;citation_title=Allele mining and enhanced genetic recombination for rice breeding.;citation_pages=34;citation_volume=8;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=X. J. Li;citation_author=B. S. Yu;citation_author=G. C. Xu;citation_author=F. Y. Tan;citation_author=J. Y. Gao;citation_title=Analyzing quantitative trait loci for yield using a vegetatively replicated F2 population from a cross between the parents of an elite rice hybrid.;citation_pages=248-254;citation_volume=101;citation_year=2000;citation_doi=10.1007/s001220051476" />
<meta name="citation_reference" content="citation_journal_title=Breed. Sci.;citation_author=H. Lin;citation_author=M. Ashikari;citation_author=U. Yamanouchi;citation_author=T. Sasaki;citation_author=M. Yano;citation_title=Identification and characterization of a quantitative trait locus, Hd9, controlling heading date in rice.;citation_pages=35-41;citation_volume=52;citation_year=2002;citation_doi=10.1270/jsbbs.52.35" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. E. Lipka;citation_author=F. Tian;citation_author=Q. Wang;citation_author=J. Peiffer;citation_author=M. Li;citation_title=GAPIT: genome association and prediction integrated tool.;citation_volume=28;citation_year=2012;citation_pmid=22796960;citation_doi=10.1093/bioinformatics/bts444" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=I. J. Mackay;citation_author=P. Bansept-Basler;citation_author=T. Barber;citation_author=A. R. Bentley;citation_author=J. Cockram;citation_title=An eight-parent multiparent advanced generation inter-cross population for winter-sown wheat: creation, properties, and validation.;citation_pages=1603-1610;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. L. McNally;citation_author=K. L. Childs;citation_author=R. Bohnert;citation_author=R. M. Davidson;citation_author=K. Zhao;citation_title=Genomewide SNP variation reveals relationships among landraces and modern varieties of rice.;citation_volume=106;citation_year=2009;citation_pmid=19597147;citation_doi=10.1073/pnas.0900992106" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=S. G. Milner;citation_author=M. Maccaferri;citation_author=B. E. Huang;citation_author=P. Mantovani;citation_author=A. Massi;citation_title=A multiparental cross population for mapping QTL for agronomic traits in durum wheat (Triticum turgidum ssp. durum).;citation_pages=735-748;citation_volume=14;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=L. Pascual;citation_author=E. Albert;citation_author=C. Sauvage;citation_author=J. Duangjit;citation_author=J. P. Bouchet;citation_title=Dissecting quantitative trait variation in the resequencing era: complementarity of bi-parental, multi-parental and association panels.;citation_pages=120-130;citation_volume=242;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=H. Sakai;citation_author=H. Kanamori;citation_author=Y. Arai-Kichise;citation_author=M. Shibata-Hatta;citation_author=K. Ebana;citation_title=Construction of pseudomolecule sequences of the aus rice cultivar Kasalath for comparative genomics of Asian cultivated rice.;citation_volume=21;citation_year=2014;citation_pmid=24578372;citation_doi=10.1093/dnares/dsu006" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=W. Sannemann;citation_author=B. E. Huang;citation_author=B. Mathew;citation_author=J. LÃ©on;citation_title=Multi-parent advanced generation inter-cross in barley: high-resolution quantitative trait locus mapping for flowering time as a proof of concept.;citation_pages=1-16;citation_volume=35;citation_year=2015;citation_doi=10.1007/s11032-015-0202-z" />
<meta name="citation_reference" content="citation_journal_title=Breed. Sci.;citation_author=H. Sato;citation_author=K. Matsumoto;citation_author=C. Ota;citation_author=T. Yamakawa;citation_author=J. Kihara;citation_title=Confirming a major QTL and finding additional loci responsible for field resistance to brown spot (Bipolaris oryzae) in rice.;citation_pages=170-175;citation_volume=65;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. C. Schatz;citation_author=L. G. Maron;citation_author=J. C. Stein;citation_author=A. H. Wences;citation_author=J. Gurtowski;citation_title=Whole genome de novo assemblies of three divergent strains of rice, Oryza sativa, document novel gene space of aus and indica.;citation_pages=506;citation_volume=15;citation_year=2014;citation_pmid=25468217" />
<meta name="citation_reference" content="citation_title=Translational Genomics for Crop Breeding;citation_title=Applying genomics tools for breeding submergence tolerance in rice;citation_author=E. M. Septiningsih;citation_author=B. C. Y. Collard;citation_author=S. Heuer;citation_author=J. Bailey-Serres;citation_pages=9-30;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. Shao;citation_author=X. Wei;citation_author=M. Chen;citation_author=S. Tang;citation_author=J. Luo;citation_title=Allelic variation for a candidate gene for GS7, responsible for grain shape in rice.;citation_pages=1303-1312;citation_volume=125;citation_year=2012;citation_issue=6;citation_pmid=22772587;citation_doi=10.1007/s00122-012-1914-7" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=B. P. M. Swamy;citation_author=P. Vikram;citation_author=S. Dixit;citation_author=H. U. Ahmed;citation_author=A. Kumar;citation_title=Meta-analysis of grain yield QTL identified during agricultural drought in grasses showed consensus.;citation_pages=319-319;citation_volume=12;citation_year=2011;citation_pmid=21679437;citation_doi=10.1186/1471-2164-12-319" />
<meta name="citation_reference" content="citation_journal_title=Annals of Botany;citation_journal_abbrev=Ann Bot;citation_author=T. Toojinda;citation_author=M. Siangliw;citation_author=S. Tragoonrung;citation_author=A. Vanavichit;citation_title=Molecular Genetics of Submergence Tolerance in Rice: QTL Analysis of Key Traits;citation_pages=243-253;citation_volume=91;citation_year=2003;citation_issue=2;citation_pmid=12509344;citation_doi=10.1093/aob/mcf072" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Wang;citation_author=K. Wu;citation_author=Q. Yuan;citation_author=X. Liu;citation_author=Z. Liu;citation_title=Control of grain size, shape and quality by OsSPL16 in rice.;citation_pages=950-954;citation_volume=44;citation_year=2012;citation_issue=8;citation_pmid=22729225;citation_doi=10.1038/ng.2327" />
<meta name="citation_reference" content="citation_journal_title=Cell research;citation_journal_abbrev=Cell Res;citation_author=J. Weng;citation_author=S. Gu;citation_author=X. Wan;citation_author=H. Gao;citation_author=T. Guo;citation_title=Isolation and initial characterization of GW5, a major QTL associated with rice grain width and weight.;citation_pages=1199-1209;citation_volume=18;citation_year=2008;citation_issue=12;citation_pmid=19015668;citation_doi=10.1038/cr.2008.307" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=E. Yamamoto;citation_author=J. Yonemaru;citation_author=T. Yamamoto;citation_author=M. Yano;citation_title=OGRO: the overview of functionally characterized genes in rice online database.;citation_pages=26;citation_volume=5;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=E. Yamamoto;citation_author=H. Iwata;citation_author=T. Tanabata;citation_author=R. Mizobuchi;citation_author=J.-I. Yonemaru;citation_title=Effect of advanced intercrossing on genome structure and on the power to detect linked quantitative trait loci in a multi-parent population: a simulation study in rice.;citation_pages=50;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=M. Yano;citation_author=S. Kojima;citation_author=Y. Takahashi;citation_author=H. Lin;citation_author=T. Sasaki;citation_title=Genetic Control of Flowering Time in Rice, a Short-Day Plant;citation_pages=1425-1429;citation_volume=127;citation_year=2001;citation_issue=4;citation_pmid=11743085;citation_doi=10.1104/pp.127.4.1425" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. Yu;citation_author=W. Xie;citation_author=J. Wang;citation_author=Y. Xing;citation_author=C. Xu;citation_title=Gains in QTL detection using an ultra-high density SNP map based on population sequencing relative to traditional RFLP/SSR markers.;citation_pages=e17595-e17595;citation_volume=6;citation_year=2011;citation_issue=3;citation_pmid=21390234;citation_doi=10.1371/journal.pone.0017595" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=J. Yu;citation_author=J. Wang;citation_author=W. Lin;citation_author=S. Li;citation_author=H. Li;citation_title=The Genomes of Oryza sativa: a history of duplications.;citation_pages=e38-e38;citation_volume=3;citation_year=2005;citation_issue=2;citation_pmid=15685292;citation_doi=10.1371/journal.pbio.0030038" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z. Zhang;citation_author=E. Ersoz;citation_author=C.-Q. Lai;citation_author=R. J. Todhunter;citation_author=H. K. Tiwari;citation_title=Mixed linear model approach adapted for genome-wide association studies.;citation_pages=355-360;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208535;citation_doi=10.1038/ng.546" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Approaches in Characterizing Genetic Structure and Mapping in a Rice Multiparental Population" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="twitter:description" content="Multi-parent Advanced Generation Intercross (MAGIC) populations are fast becoming mainstream tools for research and breeding, along with the technology and tools for analysis. This paper demonstrates the analysis of a rice MAGIC population from data filtering to imputation and processing of genetic data to characterizing genomic structure, and finally quantitative trait loci (QTL) mapping. In this study, 1316 S6:8 indica MAGIC (MI) lines and the eight founders were sequenced using Genotyping by Sequencing (GBS). As the GBS approach often includes missing data, the first step was to impute the missing SNPs. The observable number of recombinations in the population was then explored. Based on this case study, a general outline of procedures for a MAGIC analysis workflow is provided, as well as for QTL mapping of agronomic traits and biotic and abiotic stress, using the results from both association and interval mapping approaches. QTL for agronomic traits (yield, flowering time, and plant height), physical (grain length and grain width) and cooking properties (amylose content) of the rice grain, abiotic stress (submergence tolerance), and biotic stress (brown spot disease) were mapped. Through presenting this extensive analysis in the MI population in rice, we highlight important considerations when choosing analytical approaches. The methods and results reported in this paper will provide a guide to future genetic analysis methods applied to multi-parent populations." />
<meta name="og-title" property="og:title" content="Approaches in Characterizing Genetic Structure and Mapping in a Rice Multiparental Population" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/6/1721" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Multi-parent Advanced Generation Intercross (MAGIC) populations are fast becoming mainstream tools for research and breeding, along with the technology and tools for analysis. This paper demonstrates the analysis of a rice MAGIC population from data filtering to imputation and processing of genetic data to characterizing genomic structure, and finally quantitative trait loci (QTL) mapping. In this study, 1316 S6:8 indica MAGIC (MI) lines and the eight founders were sequenced using Genotyping by Sequencing (GBS). As the GBS approach often includes missing data, the first step was to impute the missing SNPs. The observable number of recombinations in the population was then explored. Based on this case study, a general outline of procedures for a MAGIC analysis workflow is provided, as well as for QTL mapping of agronomic traits and biotic and abiotic stress, using the results from both association and interval mapping approaches. QTL for agronomic traits (yield, flowering time, and plant height), physical (grain length and grain width) and cooking properties (amylose content) of the rice grain, abiotic stress (submergence tolerance), and biotic stress (brown spot disease) were mapped. Through presenting this extensive analysis in the MI population in rice, we highlight important considerations when choosing analytical approaches. The methods and results reported in this paper will provide a guide to future genetic analysis methods applied to multi-parent populations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
    <title>Approaches in Characterizing Genetic Structure and Mapping in a Rice Multiparental Population | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__MRoKQICmX-Ek5ZIyBlwXzUtbKmCmIVqNnsCLjO1o65o__abNV7fJaqFhsjvAynloQDGCCstgzs17fQnVdDEt9-NA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"H0nY0fpkp_Th6JSyYmTQ0xvciSvj2fFu2didXw9ayB8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484782","apath":"\/ggg\/7\/6\/1721.atom","pisa":"ggg;7\/6\/1721","ac":{"\/ggg\/7\/6\/1721.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/6\/1721.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1721"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1721"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/6\/1721":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FMoC\\\\\\\/S0zvI39+zMDDR1txBVDOUwISYx1CoM25fEeuyuJ90j3u+oz3ZLpxI0O5+wCTT2zCfkC5HC2Y7T1\\\\\\\/15HYSsEMZ27RM6XF1c+l25EywJJEMD1ThMFCPg5ncDFAQxZ9otBdvaIaPaewOl5ILVZPw+86GMGbd0JWqU50DEi+7K3jaYIFNJy\\\\\\\/4A67BEUrc+nHKGvPDJ8Fh0XaluXNSKm9uq3DHGMm5xn\\\\\\\/HF\\\\\\\/x4PP3dlCVhevHJcQSvd6shC7eWrv9N3KzGopCy1lEK0Iwk3sY+VYwMmrH\\\\\\\/km0eeTGS\\\\\\\/PT91Y9H1hGMeRtkqaadUUh\\\\\\\/UbQtHRItBBTrMfc9xUzNFfTCm4TPl4rlzfR9TfUZ7ajtS0RbTn945jm6s4EB7b8BemgUWwPMTX0p7Kk5psdY\\\\\\\/qYB7Qrw8UefMkxaS0YZ8JmUnwWet+0HJ4CFyBswhcF+JjB1227xPSGXEt86eLDk8cRMM4GtW0ceyoz3DcS0I+BA3GGkx6bHl4VJfD6mJUMuc8pojRTGAz9BiDyrExqso7cIj0Mi1djUxiXdYEHezUp712WUO2ZN6UUT3FiwGUUVUSxO4sI27XFINgTSM\\\\\\\/Pkihg0Y1j5AMmY\\\\\\\/qGdI9fcckQp79iFINlrT1MLVCKFfROZ59Shvegn6q1PqM8SWDotULV1kLOVqwTgfHVBRcNUUSu1j76RTqg96xF3myGRVjCHtLTDrv3LiT1nyygNVz4B9hZczqRvMtCR4gIUfhrZfjt4euf4vUccX0v0d1zbPTpHkZrPVTNHtTIMqTgtX7IlNFSoc4XRZ39fgD2k8GRsjGddCdqhn+eOOCp5hDysjR+VJ4WAVtQc90PdZSM+TWfPE8A3JXzAHlZCwecnvwoWQtwag7L7FMHH4mjpJ\\\\\\\/jxTeZl6\\\\\\\/kRu5Hhkw7W2cnUILhKrCoMDyDyOtopFrUuo09TMo1ZEOKxCCMuvcna3CQ1PBK9XYPKyMUaiX5W44geABKD6LcuRRNJ2bY52z2ngx2LLSJOSq70LkC81OsqF8R1XvIdAEYwIpK3RSGU\\\\\\\/Hpn2l2L7ykOU0YY\\\\\\\/pfVA9jyN9yYW1XGx3sNWWq2Methr2OXhJwe4IJugDPdvxv7X2gDqcaLUDMPCZN1nEyqncTu3T5lK2zVxQ4UcGZpv\\\\\\\/Vr7Xe\\\\\\\/wlF9hyuSNoWuVNXLwItpc+X8SP7YCwNE5tz\\\\\\\/9Kr3mAmNrrtQB3P40UunraOgkm6a1vg31nTAGuk6Dy2kZrveYVPYuZAcefxB4YfQihWMK96BkC9HFqcOH3OORaWPiQUwdvGZix\\\u0022,\\\u0022iv\\\u0022:\\\u0022yS\\\\\\\/zXjlH7sSiERv3rLU+Dw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f0cc36b7c9ca76eddf09e027198b831406a1c2b2cb2645e256bf87fea55db871\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002214usjBP4\\\\\\\/+MzsPaKnIdSxTX3oJWaheUkVLESKWsJbw3TiDmKPZ36SKOxJbRprvUtKmt1t5AWZukY1EP\\\\\\\/\\\\\\\/5bNENZK7FMjmgtRwUPfqxXuKaWneRB3aPDCOoVCzMQrecMpPyymXFLHhhzeL1sDO74Yr0D8OF1syGwa7kGa09GfCm3gXNbMGotbunRiULn\\\\\\\/lLvQ\\\\\\\/9Vf1mVsZ1TyvZTXWPWZr1j+6FvhyTOorEg+ICWiCpNbGuuCvfc+9QfNEZUx6zfk8fVHifwIw4tAn4ihlO2QFkar2QgsTLeHoJqorB+ZuZJ2NxJNF\\\\\\\/3e+PQG+sSI0RY\\\\\\\/9QfHRl0qWmDg32iBe3QNx8THmjZOR3fxZGcYJklKm\\\\\\\/p6MaUhLNfe1PlM3NqbqZFMsFsxOHRrAk32dr9MGs9GGdIGR08beJ5OBjb6m9TnXvOezBPecCWXkdCtk0ULJkWKP2WmfpTD+XSxxWuiLBuRaKZllN2NDerthsbrcMH\\\\\\\/0YWnMj96b5DqauhcFzwIoGsI0yMDsjogzVFqh4LU+F2mpO\\\\\\\/TagkC+CPaE5G8L9jd11LWRgkc9yZw2AzWAm2ikJkWyYtw0YTXF4lBq28EnG7MVdBbG2jEKWF4\\\\\\\/1IJcx5Mo4rsaZwl3q5Ru79BTvecpQEEhpTCYWQPqAdpyU5MVVGc3yR+LOh\\\\\\\/du45VxMyDyi+CjT8vu1ZxrvqYyqfFaWt5KEEIuwozJoGaRB2BdwRScvYzz9rhvt3LfY4nYW07h4ZgWYsSTN1WNDlLHoFLqUWMjnZ0V4YukPGYR34tjFcl+wFXj\\\\\\\/VvGlmGPC8+NSxBuhNw6dg+ix9+9DzSp1tdTrIHmgV2IEoEEreAdfTepXgLa6d4ntMFi1sOAfnwYy8qMBnt7kE9DCKXZkuGidRiw6Tl7S6YYyhHkStaMa91yZuZTvhyKuEcnajRqxo1GJdg7Caj7dmNo9tB0u56FJtGmEjzeyMG1puED255zXbf0l4ln+sgrdt0zYsvRV2gWcukUhtF3YGU42L3BR4KLY\\\\\\\/d9nqKZv2jd7nhRpq\\\\\\\/kcgym32+EEURNftr2BSlcRah5ybXhNVxOWvQ2sqLv8QZJC1ofjFVvWgdMyQ99pkYFo45niQ0ah36kJjet36DlOhog922kVQWTLX8\\\\\\\/CVQ0pvuBFkaiY72nGnztZm1n4qneB9XGMQoQnbY\\\\\\\/laX8m7HyHsMNv1F0AQYA6meyuFmuV0rguslWbMoiMfTuWPzlsYH1PN5\\\\\\\/5pe6Ze2915K+0JI9XsEoQfY9Y00bYIjEAx043XpWySrvN9UZkRcLO6F67IPGZ8bOtlrksayAgtUwKrJhQ7e3mCg71TnXr\\\\\\\/952YBg49J8CIIlw3ksVwkH9G0+a3bgxmhQe0oKtSLzKgoOEbHdbOm5W4RijArJDgi6mIdBx55U4xqrDjocycqh48mb++kkKKOtM0sCXXZY9u4sW\\\\\\\/1FskdZW26R2++EPCJTmYR5VqGltMcQfsbL8mTGdkmWWWDZI\\\\\\\/8+KfQ4mV6LuNV+D2oXg2PnrrIdTOtTyg82KIPQ1mVARn+8PZoPAjR3C88M8qARyMAreekDlLzxvHw0XKelSDF9e0N5tsMfyjbBX7sQHRgn29ODKpINShSfgAyZAx5Y1UIl+toivRlDN9pkEMPL2xDRtVJfrp7WMOr+sHYOE2vyLtvOdfTyOTZmYE6X5OaVF47x1IfctAuE02vGvHIFGWXAAPGS+i0zBdFLr\\\\\\\/RLAXJeryMt\\\\\\\/B1fBNBiL9KxpnMMyiTuTnEWcloTaBGpt8Ne0vbeQvebsjqfwZaEvifaR3cYUL8UXahu4Cd3oOAbY0RLKqZo\\\\\\\/YiMRN1XKw\\\\\\\/wHr5wq0VEGVsoDx5Sx\\\\\\\/8fnQzsO146bkLInaYY\\\\\\\/\\\\\\\/ZiH3SK22GPPN\\\\\\\/Jq11RzcspAeMcbqbJXI984gMgJZIkWt\\\\\\\/CsbCoUOf9Fps53tYfpdtECub6XxrjNTJpDpkreR3vzQBQWcoLHO7U0We0JzCnafQpRKy3zrbXO7h+5JoHd8iMUYl5RO7kqYsmRI84Ovn6rY0go8hUTsJE\\\\\\\/r8qEioYpgf8OhXtC8uNCyFO5pGcwviIY09PfR7zTXPoaKFXLc\\\\\\\/URP\\\\\\\/h73mpdZnN4ODsrt0YsqigozR4k4WnRK2WclkZJSwELPsjIcpiXNF1iuoe6YunnKyxRzDLM8DHcy54eIVsrmelYc3PjTzCRe6ZUSsA0YGgz6YLk0C+0CyJ7sUU7wxrPJIUwKB8CCASQtZWUg0GbkU70sWF0P6v\\\\\\\/fVyhtykRa9PpWfHmVAcDjytQ+NQ1joOsxDrYANhRSckbWcAMCp3UUwVNjpskpx1RayFwYW\\\\\\\/lsd4Lca\\\\\\\/q63q\\\\\\\/uFs9209m8301+JWw5Q5ECPNhoJ+E9IggvLmCnpVHl8qY6TD8LdDxnszCCaLJseFnE2CTlLyvjalgJfJSovFL6WkrgMlS4uNHce171pgp5VRF0q7wAUU1lgLM\\\\\\\/Sg9XXZlbgfZR1Ig6cDrVTJPg8T1LkCZAsXu25Gqce+7JgipT61ElUZXTMjpEu4zXy+fSoLQoZ5kOTjs4C2Ko2A\\\\\\\/Lul+4YKmj9nac3lmA1Kpsyi\\\\\\\/XUuvg3o2WT4KUOOOCMHxS6Z7tpGOjudMLP0Q1JAezh5jR0P\\\\\\\/sBQZKqQ1+2UtNlifEG0KUCii6Goh\\\\\\\/2dJdQll\\\\\\\/pGz93L4J5btTmR+eSk7\\\\\\\/kqCdI\\\\\\\/RMK8NN4tNui52bRDwjfPLWgk96CtG2TYOemXUKjhRh0zR6EBqqqBDr9pSOS+2scYu5Q+HdOm+S2i7DvxSaLvdgAQw8qlKuIXLuQnsjaOh69\\\\\\\/1Xqu7WkqDEW2FH1PeMJVIYLVWpxyUa0e5vRciTmOzx0q90g0eHOR0Eguom3Ko5EHT6hksPqSblc8mQGf1S1zungV8kWoPMweiU6S8ZD9bqRgn5WBV4zftKrVs9dfrxmzBj3FdCkMH6qdszVSNOl60MBwoCsost9aaNr1Zshfd9x0DAQgT+bxpJZpqnUMk8aM6tRLPXle2sx6FM43Wr1E0neblIeYW0nYTPdL1goGzia\\\\\\\/QAiZ0ttzu4PvRDpxzL\\\\\\\/c9FGKWbn++0W9hwYrA3\\\\\\\/lSE42hO\\\\\\\/FlMDZ5xBDZSVJ8CXNDp0uf+nuYGWifmeyAxS1eZM1RlCRwcE\\\\\\\/oWFUb0dqOK\\\\\\\/qpQ7hxsrbiamhMqJutSm\\\\\\\/kx67LHktolk2ETm4zg0oT7SN7iFI+vxYHCDkVfpXN8hWolvwTFGQUW7i8H3FbM8E1PkzQf9D6\\\\\\\/pyAUHECRivsr+TrreFRqP3G6+C2ocwpxHujp+wpIWDScc60n8e+D\\\\\\\/v4XZLo5f2PWgTFOHzrWsOe4UJFQSx5mgTcfpbSe9Wm\\\\\\\/PFu\\\\\\\/OcbGrnRnrV58zNdh61cuBUyQGxAvnVPoUzEVVkratZSd88jBNbCzuxfF\\\\\\\/gYmKYBQ2OqpXKlaTO3hbSiafLufjPAo5ns3SckmfJF0NA3Uje8RzYl3uX8mpN8X1R2hj6lHETLIv19VGn+FJzFP4KC2oHqfXrZwy3wZVGAkIG1PMiyrHAFSTXD\\\\\\\/YjKqxJ1Hxo7kz1ww+pD13JaMwwhYgvCbA7eSKZzr2jRrCERZvtB8j\\\\\\\/z68h01fNoRFvKfzgxzMUktFdUGvMOqNSToMJ6s\\\\\\\/TJd8stDHm1rXIDwzPghM7TQCTu4a9nME+oMT+iShDHe0rV6chyk1QzXrNDRFINLJcnTMpsbhy6R7BgYLzAxoU4TJRT6xOSAyOb+p51Fm\\\\\\\/TQ2ZV6IwHCXq4phF+I9oTWCXfRNpOEBPDF4jYdieQV5da51IQuXdlL4lhDDudn3MIA41RgkpJtNx2CYgyEegWsLxuFc8O9iUo9T6mkoQDtjvLmx74l8DUAfwYWxJTzHQ55PPdRf\\\\\\\/yPvJa+7OZklFrUcepQ16cAvVTpCiN\\\\\\\/+Bf0dhr28+yhOE0DoSqmHWWxeBgJ5r38\\\\\\\/jFCM89ZrAu5ROfSA7ZROXcVGYRVbXEVenzqgTRLJV3VRAMKBsUnu9kssnJiIh7rmG3Emr6kHvH+cQUoMd5aHlx0cxG5+gUV47H5jSINxtalx0kAmoJnn7O8Z0Zb9cmkWHWhNV8SVmstaTIs5Qfz5X6ax5gFae1MaZoK7hu+mScmSjxSKeEQ67ZABrxkcTHFkm1eWvBB7RLnVUoeKHtb3CPc\\\\\\\/tTUVNqqXfrUCFY83QKz9jQr1QHJU5dj4LkfbkJdWGXhu6WEGkxXlOV00rMknneWD0IiMWgqImqc6tp9zW4C\\\\\\\/BWmakwa92bHYsL1jRLUYc7kELGc2GyBHdCcIwHTG1+NN2E0e7QRffVx4kaQ4e0hyOFrCsBSAynydBP8CcYtwDFLP2fWMVUV+0ii6PDaBjVB1y7\\\\\\\/JdZIYYC7rRZvpV+I4E0GHt8iqTDmYussfuMYhQ3OdxyeP+sDxkXeP7V13doSDtjFO5tuWOtothEJl1b8CADmCyH8OEPNqVuAXDl6yYta6KFoWnquJ69rNSDzcyWpWsIKHNpgoLPGCaVEEBOgy1A+z++s7LhH4tsmpu9Ib3NONDLiAVjLv+lM1n7INJIkv\\\\\\\/VL1xwg7IjUlY7uxL9+YFToLVQFSDYax1spEIN\\\\\\\/IDs5aLxWKNkTWKRa+\\\\\\\/S8E5b7rJuPfpqdsh83C9PlOMc7hGNrhedajYh2kb0DVwk2wq2hE+mmfJcBfELZWgYyU6tVgSF6AiyLZf42UXxgfNVkBwx\\\\\\\/yEw7rm0ZF5OUgtjIYpJj4KEsBouZahfVI0gFVREJ9KKjFufSokGM3QBvjWjhAvA1+9PVx6k\\\\\\\/8YGjAUsHh2NsIW5ICvzaX7GCu5mAzO+n7QoufjjcdiNmdN2NAsz37n+orj17B2ItycGP3UlYWgWNEFy9k\\\\\\\/B89AH2DHQ6qKVDwgqfS2m+W9\\\\\\\/B31pph41KCwTeSAjXLmhrdQfeFwGhZvmpdxfLy+t2R\\\\\\\/Uew7Zz1T2h1SDQYfDpCiKqtAcP10WatDr2A6eEZU6EoPfBuR8pXrDJ45\\\\\\\/mCWBGT4QsrSfBrlRMFI6ROdSk9zpCP5ikPa6nM6DIg\\\\\\\/7loGJLkQQ7boJM+yMlLopOTO5NdlP7Ttlt\\\\\\\/\\\\\\\/FPhWbYLLRdZZKWBedPcbyNAiHrCpZeRL9QTpX3vnWQyOY9dYNW65RA15EXql7zFL7BPnc8yME6q7DvOo5c7K1LMI5vMnGOqeGabIv+jo6Pp1z3kD4NUPt78Kz69QOyUhpppGfYtfA1X7IGuxeqagGbIs+uapghLW8M+Vw9L4VtqUnjktQ3Fao1\\\\\\\/gH\\\\\\\/MTca32oCRo9DIq3gAU1FHj0eiH\\\\\\\/+UFVhB\\\\\\\/FbhUoUMJ89c7ZG0a4ho6WWBkF5wqAJtXkW1RaLTUy97qx8QS4tMlMQ5VR5Uyr4cUFvDzXnLJOuULrAw6681r9RfTVpm8W50kkIaoNeuwLkwQamse1I\\\\\\\/BV9wWjeXWPDSs7rWdOhHFW2MuZ25UJRrtquUSIAtKNED\\\\\\\/zIx5I\\\\\\\/1D\\\\\\\/E8AYX4R\\\\\\\/zt8PWQOwa1TsWSeAI55ZnqftHn\\\\\\\/UC8Cqnq6FSMsFxrMndIoQ6K9uxtWpVYJiIUEv9jZjsgM+gFw0O7gjOaO0aCciApuyrlm+kEl0l++3zYcaliX28E8JyO3+E86zS7KfoAHzn3Q9LkjjxwTrt8toWtCEsYswUb2K7xJtvJIXzJpoRhKRdCjCH6Z5YyoBCYjghJa5cp1a7vWsLT0w6\\\\\\\/Sdzzp30UAkB\\\\\\\/E5hh9iN6u1FFsAog+Pn2zpKHDrq0VoINN27C31zN3IKxj+YnhoQrunwx4O9wU6jMhMhQyfy1ctUqOkDpuzxZbVK9uMAcd\\\\\\\/okcO9BVcqUBS8RYxfyXHr3HSi7ZhKiouuPTrP81CqD4RjS0Cnn3F5BZRg5DSwkjvek7hpwmfjVCZ7dcYHyYg\\\\\\\/SMIjEx8RREJNWrCCtQLWayE8iwVgTrTzWGm5Ra2yrmu1jEaX0sa+vIc9enTYz57xCS7xGs+CcN8E+B3D0VTMH6IJpG\\\\\\\/eMAEJ\\\\\\\/Twvtg0b60fl9oqV8URRr0Z8taT5gtbnPmUHctFxcL\\\\\\\/1dP8tQzmGUI+h8kVqL2BIrjOx9igG\\\\\\\/XCaLzATo67BVJKSBfo4uW9caoQyQVYKjs\\\\\\\/yJAF+z2t+VOJfhZe\\\\\\\/o4\\\\\\\/dtYT0a7VMEvGsjlPRroidYY9nkjuscI7Mglzfe\\\\\\\/VxPnGI\\\\\\\/jmxnkPQ79EmoE3oNroXnQclzd9pXx1epM14URIPVkHQFrD4E5Oc0CKgOMt9q57SW+20+G0t1SZbm9\\\\\\\/qTY8epmhxsn7v0FHlwXMnGXXP1XpdV59A7UpW\\\\\\\/Pe\\\\\\\/44wM9ZDdjm3F9yuf8JiOwqYbWOXF4Te84LSakWXMQezVhJNoixbJgK3D7G1QbCr+ahnUL86gUM4Rb64PZXCI9tPOsCC3r9\\\\\\\/SuX0hE601P97zbSmEAF7qrNjMt+idil4OWllCBZmbBNFB3cz9ojPqnZfoEilddRZBbzlrM8CAMjTGxXJ9gzsy7MyT3epS6S2\\\\\\\/Srwxr1D\\\\\\\/UbmybK91EE6dXcbvWJG9QJQpjcfZ3smWF+ZEPPtNafe5BGcE57jrORPYL3EC0V8DW6UJHe\\\\\\\/3UzkoKnwivNVAHbRUln+ywssLAkCVzui\\\\\\\/TUmvY\\\\\\\/9xCMqku6Q3auRlnADsQ5C3GDm7HWs7nMTewZ6Wmq8MNCQ1fss3oMuvLLuZUhfBEWSyKFYSf9JIefuML5vR3X7QHTpQClUg8qfJWAztP7v5PEpPBIx4lHKLQlBMeywdjQWI4G9IjObxwq9\\\\\\\/4gxv+O\\\\\\\/ISV3mg8n4kUfUwHtOfRwNOwL0QRe\\\\\\\/agM6fo+b31Xq0sNHK1obUZAZXLnvzz6zzkOKKBdAl21y\\\\\\\/4AYXajJ7F8DjjItX+bHnovjleaRC3Q298t2nU8WDuHyiSC4J3NvGbw9nzcNDWiyJtkkEmWy8voZl+w54iPaRrH8vxUmGHGLuYGgdE0YUSxE3CckJh5zaHyLTPu5zcSGL3ea6oXge\\\\\\\/IdLvvCcAag+ZE5kJdAf6BBdvRlqHG1QP0umANFrfIva4meyUxaKMuGcUVt1jkxIZt86h+rw7QK+PGKqemQYBdVr9vs6angpeQG\\\\\\\/HJcfgVElAkOR90m\\\\\\\/UVNJXcxSCJnBQWjpIfVJcS6qMptyt0gxueWiXoiqHzhmMoYDhwidwFF2JeSaQ1PGIMzOv5QR0vn1pVZelKlgizfUK+Pcwml\\\\\\\/Q1\\\\\\\/ZaJdiRf21l7TBy3ncXNem2J9UktT4UDUqYO3rSDXYaNo2kxuKTvgTqhWas8J6SrA3RWY+HYPzOLQn92KFV5Po47MFMpubnFBaYZqx4s1nkoI5bjujUWgM8F6XhejAQ\\\\\\\/DUGDwzIw2df31PFnKBWjEeQV9LD2vsDY6qrQdW\\\\\\\/OmRgc29OVFOzI6fc2xUsA2SB2g2wBmicF76YU3TNIzlRaLN6taC7yRLGVlL1rD4mlMN5mduWkrLxFyjwVTwyK7KOvhUtCZOAQWzdy2N3BBOhczLHLQMZdXQvcI9DGr\\\\\\\/GP2KhdIok0lOBMg\\\\\\\/sboa+fTqdCLpJghM6rUP9x+pdkzckkZGsr2JpJvqGCD\\\\\\\/zzrozt+ROTrvSX4Lsnmk+ndG3FNRS5a4mvjEvHOb9bMrJtpbzFuLwwdozVe+5ufxQTZYhc8U1EHAiMO1Rah55fTuWN1zVwwjyGj0BSFYqm+3kDxEslrXSA+hX6CoTNBg7R6xG6qX5JocFCmXBUPL0ZviVZgaO1I2wsa+wEQG\\\\\\\/9\\\\\\\/i9aYU1psxBrz3J45pzuDWcuWZTO0jPkOszV+mf7S0PpxmIdPsGo9yFku2LxB8Dev\\\\\\\/6XGguaWp2ai\\\\\\\/Be1QLNPzKXSWeT9wWS3p6JvUtYIzPz9cKfvuPoiE9bpE\\\\\\\/U2yqj83KlW3naQtukE8QcEGFzGNhGHm198a9LfU73SDJD901CC9ZSPm+TShZQDyGuOLnO2O8c+R75LFMWlPYx0l+m3QjpQibe8fhv7fq7Sb8af\\\\\\\/QmcAxjx8tzwYF\\\\\\\/+lZj76SXKEdSIppYC3m9NyZ6VJhIf\\\\\\\/P+\\\\\\\/Ab494We3G7hwMVlLAiZYLtcshj2rTSTqp2cisEDDnIf5Vv7SlCXloy5l91vO7UIeJ59Xc4\\\\\\\/qpaxbfwGawgtHplGqiXZLFQ8FGqBFtO5F9Z8zzzoa5JKooT+5UQusXyBDTQT4OnLgHX9T3YdiD6zNItIPwJQSAKStR2SaicDBePeAjkkIOhuDARTs1wjznOD+hwd\\\\\\\/YFqpw1Ce7ef8Cs\\\\\\\/9hh5iM1Gt\\\\\\\/DNInYb8sHwpWoOPo7NsC84575dw50HBwhl\\\\\\\/+Kboj+X+BZ17Or28tLwPN68IPny1ujN0Xx3dic7H5trKcONyjoGzsdu31lBXNxqVBvuaV\\\\\\\/Geyl4c6yWK9+4IWR+8L9xhssjWPTLLHTKFjOLOZPUPluXkMLYxbuqE\\\\\\\/uYmXYk\\\\\\\/V+89cGB3mvDCN0RNnnCuldJcbAyLu95a1Ao8rPrw4aMrKX\\\\\\\/QwLh4HFma4+B2MrqMvBD+7\\\\\\\/GoFjkPgv9z9vjpiCg\\\\\\\/77Vn\\\\\\\/oGX5y5MwmnBi3mPmnBYAMASfqIVKFh3W4640szfPCIeZxYlstEnwWLYyKyXvss2ZYzGJrUOYctMbeubCGRWlPq01\\\\\\\/dwCyP\\\\\\\/BYCZ4wsTQOImzN3R8cU+U11kvd370AjvTFr+QlINurGMIAHg7BjcPhDKhpw3zdgW\\\\\\\/uLAdEJbd4NKZKGVUFz3vBqDOp+ZDFXqLRQ6ybV3JdvewERQj7NIPi14aAXxPswOiwbAtFQGxuwwBiwnqDjCHrxMMSXHEOBneVlpxKeP\\\\\\\/Bb10ieQ5ArcNVKxPzADuL40Ld5u94g\\\\\\\/CnvJnQ4FPLdQmmvxpC9pFZ95efz5W8+YuCqswuZG2UTZFvm7vjbv7VVTfVnp9DLtZfUYrl4pVREJowBxOcH2lfBoF8+wZvpqgqCzwqUwPDDDv80Ja9bxitJyiD9f2BeRW6GNJNhi+ReCs4JMxAcnkw\\\\\\\/AJrw3wp1eoHNYS14dVlRSQF78oHakSM\\\\\\\/1oICmwP70bs5q8wSEvV+7BQYK1s3SwHOTiOqvpd80dHo4Q2xZiT6\\\\\\\/WAqYfnbC9J9AcaJCfppzyVEVl0M93CeN2SzHiH2lbro\\\\\\\/Bvv2lIX6uakIP\\\\\\\/uzAmODig90nntnxgUDo8mhYn2cXQz2ym84nLPjIzwzGuW7TN+rpbbtHCSkovYGY57eXAKcI4z+8z43j9FwiClU33CYMn6ZYa+1T9bwiVn+T55SpRTAIvvwhETWHQNEoX3tt5+MzTQ7HX7Z3pDw80E4gmhEWg\\\\\\\/dVLwQ+4++l3uo2S4frBt20OZxKFQ5\\\\\\\/Jh\\\\\\\/atckzYvNvTzMCk9ws9KF0XO9bTF+DPfjuQyoVQ+fRw+Sk4dX7nRIXCLHDBpb\\\\\\\/gTtOH0U5YaLnViJwiug8DadeO6naMIaoZzs9cCmYyMwjKPiH+HXNu7txXx8dc9GufEol8IIJMbCAglXSDBdHTjPKI43JrpX0uspz6rj8CGvcXGaJ0BXOy+k1aIf1DwSryauRWr1v6n9KqEKPob8mUInxJf4LElshaSvZkboJVKNb4xfo2lNjewoShVfXxSUz7ZtPmDjlraG02TCernzKqU59ynhlrWx1XWdUYL\\\\\\\/Old8tsmu3uA4gucJezLLp\\\\\\\/jhmhGFGVJkwJaIq4E2s2eLhVMADqSPZVTXtW3kMoPTnobMoZzxH4rezuj2Sfd3hMyD90dLSSYGKy68loLY+R1b9I\\\\\\\/1uX9mr+FsKSsy2B+Q2vChhO4AKiDWNX\\\\\\\/bmhCEE2QH1LSwLBLeNLUXV0TV7dkRjfZz7Q6M56TtYtOsLTWGe0KXujkPrQaAn0UFDfof3nm9NVD5yjmvhj7\\\\\\\/\\\\\\\/i+Vb3JW8ONyu1eMDqxPdX8c8A407vIoOGU6wz4D8hR+6emL0RafePvVSGiW+4ShPmuT0mZKbpJlZM6VAEpFQQ9NmgRf2QZDO4HeutUgGO5tC6KspIgWA2VKVegMlmUzLTlRfC6bXdx4MmV0fWl20A58bIJBn28TUHwT0ehbF3BY6AlYI7qHLsmgJLFmjNNNBt\\\\\\\/UIs\\\\\\\/1Rir9f5cFIbsjcFu2luxUvdqbZfPx5D2TYsOY8VbGsZNsV\\\\\\\/e+P+w042\\\\\\\/2rV76KJNdQW\\\\\\\/9ADBaO+QczUUusWhYWn7swWuVDkODZp3YWZpYLx8mBDsNgefV84RpsX6vUQn9acWmSnzmjjQFY8Q9x+lD35I88DLF0vDDk3ypzUFL68xuec65qeYRQadQjr1FAb9bwCFNM7awieyeECKpVqqJAT4T6y5fWX\\\\\\\/dMf4Gua4HDjbugtM6+NpxWJsWAi9+IcwqC2a6i9XQjqpkXhZ7b5cQiZZpiwUhis+1F4Ll5EKp7uIWZjHcl3wWAvzHO0LUfEo8xDKgNMTS+3x4+fRWDTBN9zBzn8FMQvVuhR3mmeULSZ3cDfx55zOMvahPBinPP0XA059z\\\\\\\/bMnEFjzWQTNNzz\\\\\\\/haCEbrA5gb1LLtacOsz0DaS8swgiS4VvZAyiDqENLzme5PcHu60Ev+p6PhPiQG7epiQa3nfUgws5f6Hz7SLte\\\\\\\/EnPX14Rnuat4LZZGfUH7FoeIeFUBRCdScIWzVSSRqJn4lmHbjWfQQvdFaWuH9AtESOe9hhmYr1m2nig\\\\\\\/nrTFmRlGw+BjsYUeSwIdvEtvbvLdYW0cAuCWoH\\\\\\\/5wF8jX+oAXgqdyhNEkEgL032AjTbpHd\\\\\\\/FkoBbEd\\\\\\\/Htjma2DJo0bYNnLQmYbh+jlLfyDeVpJ+C8jHgNwHWer1Cpc\\\\\\\/TKa+BT9kVLGJtTIBSv8Wi6ZVZyCfnF7Y1+8uiIc6gmjbeo2dLUvjJcdfNS\\\\\\\/lpESHJw\\\\\\\/rE+YANeBlPZ\\\\\\\/6F1zCrgOKql+9tiCTTSaxwDl7hr0tuVO\\\\\\\/Mk0puQbfAec\\\\\\\/T9jwd794KZ7GJXc3LImDxrPflCC68Z89RrZ\\\\\\\/T8mlXd3JeTIweB716keh78lzFfZgl9uQxmsen6cjFXGzie1Fj3WGm7Y88zDBd5RexMPl0m3ZndqNuVLrLdw2b6VN+LJZGeKG+qTVJIxR6V745DhS1YrSG6WIBBf\\\\\\\/mbhUqwtzy6nZ4kS2hMOLXFfuhFsiwG9aopWJu\\\\\\\/jZeO0+iKPOO\\\\\\\/ts5KNaCCf4LDDvxxgMnF+OseP1RF8R6OkN19lZ38w+FWnPFnZpIqT9NLKoJG3B8ndOQb+Sil+r+PBuydlUWe1uhxn7WDA5rVYQkUe6s8WUtOGuYdHDo\\\\\\\/A+npdFoVVq4rBwP1HZK8cEvVBGhM62g1Je3OjebfbXN51wyz2poU6PqModsuqcnRh0\\\\\\\/sK2+OpR9TshZQPXXN7\\\\\\\/T0VgHGuOmyo9Iu716idR+wCUOPIok7HZFmwtta+06LkAKDs+bZKeQoQY44TZgtqgLgk8JBPcSkVV0Oxg9pmiq1TM7qnMnXjg0MCNnUzzrGTgVhOR1jw7PyiEgG6G5J5pZbER63Q1j9Fhf+sGshAhCh0PpDHS6QuB+AVmUG+7CTZWyCkzY7h7hydkix8o6muFdjRYWtsi2xxQ\\\\\\\/2FmNBdG90gwZzgGvPvNZo0piWiT6+U3BAtRB36Jyg1PAF23beadxx3\\\\\\\/3dOpGO0sIe8bgTNPG5B+4TyiXLg7s8\\\\\\\/FcHisgSevzfDp7RdhboEGEQbExITbX6Q2xulrsDvAHnivv4eaKyecbntO4GOcJqf0hTYapL6V21+B5J3sZqN\\\\\\\/lA1BLKsLLp1WGIira7NsY+b0gvigpvp8c+2eJeevyPCAHg0pCwnkieFUqg0vXGTYTFV0suGMQc0JvUSOjiMd2HXP7jUJMMWa6uxKRk\\\\\\\/Bs\\\\\\\/SdmICE7WFLGDlDkxvTYevYmEQg6S4HQv+nFQo6dUc+mvHLNRKyfiE8xffUV4dCjVjGIdyc2zjSoLV7BbJ5luwvBIVsB6s0DaSp+hLSAghtsMxW312WLVl0vnZjvalhKnGjMYupbOkgULyUe+T2YPtka6t\\\\\\\/Ekvq9Oo7eVoHmIfEq+TdB38CiHFvNLeRrxWWVi+BbhQ7XjsF\\\\\\\/U3R4LKMj+ALeBZIYxOmKedffYN01lrLN1cawdFub+wb+tMCIWn3YkOffjr7vqdTSYPnXg9daMS7vfP1W7hycLJ4haDmt0ECe+eu9lafPyUlscm\\\\\\\/oMnGn8Ua\\\\\\\/z2JzUF6NUHzoCuDW8DOAo0+uOUai9HXVbiESa1tWzSsZPLhD9HhQB\\\\\\\/uZMrysoB5eBjCkMbLgHXtRK9\\\\\\\/\\\\\\\/k3Z9+zm8AIHAT1pE0Um59kGxSjeIon4FgUCFOLbnESfjSO1IisFUYqwPjg3VMnjCmaCkegqYjTXAMG\\\\\\\/Petn3AKSP+wbyOneQCOLkJgM0rQBe30j6BuexszuJsdVsh9KiuwKZmIFTgPtbXP0fdulZOU\\\\\\\/8Kv1\\\\\\\/Iw3M++6Okj\\\\\\\/2xVxDObyQOIDLqBnSgbqAAfYl7p47ohWm0mpqrXwTb\\\\\\\/4nXVpmWd1q5Wyk+qPCr9XcFsf\\\\\\\/46esMqUB+ql4wJVMPSFTzKf3ozZdRqJVKjpmKH0x2V5\\\\\\\/w5D0RuOk4bz\\\\\\\/Uz\\\\\\\/4mwmCj9qYoafgpk8cqXusvs+LKZMFWg8jpe2Lnkx5bb0N9bH7V55KCz3z+byg0w+xIaYeIa9ypWSBayS5XN3T8cT9sUMlRGAZ5m0QU5fDV4MEfqswy6F97Y0BQsUHz9SgM0nocTygiZYHrtOsCQN761mYSmGoAr5kTNvjK2It8m8yzTt7ACnxaHdPqhc4JI73BVfbX+M9JDNEiqSfNnvZQFIlWquJ+L2yXVxsMFFfaORqwecOSl1Zwd\\\\\\\/Tw+7VUr2bN4OvRRJa196Ojl7zNoteU8h+385gqQQke5m5+lHlqf9b11BsG\\\\\\\/VS8w16Q5xmT6M9EDAbP692NMpoFho7Rk6kBZ86D\\\\\\\/baz3dWe3cOWAne4eKrWYsSBisSVnv5die6bnyZwrWnRPPIrxq+Ri3q8s8JonkCYMposEzzrP9WSoPdIQ4ORn7zQ62SHG+hMk2GfYpWMkd+yNh6rmCJf7fAH\\\\\\\/aEQVZv43EvjyBvvMTEbzjpodw1mrrZVm7vYroefSnmZmzuTHH0AXUskjgOVVIu+LQ75\\\\\\\/IssFPwjbymQh++V8IoPZEC1J87bwC9EeuuslRyzM0Oi4lmb\\\\\\\/xLSH17GlQdTM5FWsPA0YS6BRdOMhpK3aZjalpdZCB15lqi8Ggh+X9FVl3IUcPZ6gxc0Zk32fL4MbXOTHOk3fjWlp0Vq8wZxLSMtwGFWvBWQE3Ho4dZ7qO0f6BUX+yqpZb21zE+C9Kn9KJwtmOq40pxkPbOi0uTL8IejHk948RqVpV1\\\\\\\/xVGqnScOI+VL\\\\\\\/qNnqqm\\\\\\\/zHHlOcnLY6YHnLZJIrKkmwoS1TzWvgvsUQgcgK45HjgFxvUEmcKSGJoajtfoTbaqhmgKrcJu9dqZh46dT95jweUkcFTVxmQEx4H0FeZ68J+9Wn+88ZS0PRWXjJ8EshiNPkiZKD8a1WnGmkmn3qN\\\\\\\/+PLUofRDYlaxBxztwhKBtB\\\\\\\/+AefF9caItsDkzVMZJxVLdBoMF5f6ywunzVWWqJJkEw45+hScvoKhKjrBd9y+hUArsV8qnNasWnMWsjcywh22BrnCE+WkO39vRSoOP63Xrdfqoplj9+K9WlIcMtJNa\\\\\\\/uUgDaxQ72ZNWTlBGa3Hpkal7v4Su3XExWLpZy5sSY3Y\\\\\\\/y2fWlnMSsco8cGfji2mwOkASMrIL++gWfXx77vINLmKw1AlOlAghCYGA1U4AL5j70pyYM8LhKvrKYKc\\\\\\\/c8yp4JzgYaVEjP+oIjxTG93tGoGCOqkGn8Q73HuqXuPiltMPwrpU+axbbi4bA2Ct8RgBCq5zY5ePdihxNjcdzcRLAif2Z3rzshE\\\\\\\/BJtWWkw\\\\\\\/Ca\\\\\\\/8b5WWtpqCtyn4hvtJosS5e0JD9tvCsULLLGyvpI3wPHv6S6sMg0s8Prl+obDBT6I6wtuZPCxTIcj9JEZXkwSIKVRjD\\\\\\\/vfk0bRZi94fhUKICWXJE+y7vU0PtHFrLkmsYwsHUAmVIXhto6QaRnKHqA\\\\\\\/OPez+M0xJkTTUUm5Y08AaDn\\\\\\\/iqFmZBe1L7NXW5+7CslhWRuyGuJt64yIFBgwVh7QEawW1KEtOktlOqmpGzryT+0wzZ+qCPUJ7UedQYueWADJ2R0dwXQYTxTJLENAcSd1Vy+bzKBMS+AJk10CvEaHN2gRG6KhBYM+gE\\\\\\\/1jQDILnAw5eBlEEBaaWBQpDFWOzWUVzUb3ji+hGM1Y\\\\\\\/qWoSqcCvcpApYLkZxM5MO05mywwcrmU7sl8Koby6nJIHpK7bA5+M1j3rsBXgzyRjEnBptz9V1k\\\\\\\/g4kG0tTDu95rSeAeqqMTUrOTg2mb5iIBCD\\\\\\\/l27+fThvVdCLGoJ19zP2ynQGiREtDxGmBHTFqqniZxduJlsPYufL\\\\\\\/73DlmTcYTjbe+9K3y1mOyXixWl2Ab\\\\\\\/njJz5J3qGxAEqaj\\\\\\\/uIS2wJvd4zdVg30Orlcs6vg\\\\\\\/AV3AD5E+64TtYsUuNhD\\\\\\\/LEToqeqkTZ55Tjtp2z7as294WcqjvnS\\\\\\\/K0ylNm5XMGYQQp1FFTYNozDYk3UwOoy0pdPharpr\\\\\\\/QHIcRoq9\\\\\\\/+lFGtH9gc+u7mqA6AeiH9amw73ipOLf7GknG7Uv\\\\\\\/wOgTCThzvhwEh70NRgWo8G0sZaBiKPUbnAW+sO1fFz0eEl8aVrONq1pCL5zTRd36jG7AUPAB52VX6POwqWJo0lVdNPB3nbN1sHWeovd49kCjdHnLAJvm2034JOvrU\\\\\\\/Sry30DphpP8vg7qZxDyql1cUJrf1Z8tmfxCgpoDJ28nT7Xo6XxrS3J\\\\\\\/nf0JSLNcEn\\\\\\\/J0tRFSShR8nbSnBYVAS5\\\\\\\/LLsKDwPPifNqzwYeUh8pYFSlq3ugopU+4Aj8sO3IMuZqz0J8dNf93fSv5OeZbkZKGpcnmzKZ8PmGesOlas1L1gozsd6ccSKF0jOm8I8EU6mQFjlqqSlb7MMYa6HaznXDZowo+d97mwrOi8gJYJmy5nven4lNdB5i0LiPWiQnTX6Vaoe5T9bwahNKqkQqmzVWElJxGmmYsvqHpxngcAUciayUI+K2Z3cPgebvG2WvJrZjcuc0\\\\\\\/ngdied6xLSTm9sF+VONIYjs9WAXjth9Qqn90aw\\\\\\\/n\\\\\\\/XwZ\\\\\\\/1ZBxBCc\\\\\\\/IUdbNA5Ds5g51n\\\\\\\/Hta81LLswv1knkSTeqh14ZIZ80v7KKKOHLmrb\\\\\\\/VM+kwAByAy03h5ekaJk0C\\\\\\\/5uaLjtKcAd41PVO348ezMGOpIQSo1PCUAtSetOgiBZN0fneaMORuzH8+WdnkHGdnTStAd+NPhRuroXHPSWgJUXDq0h8CqhKolc1FG\\\\\\\/fQG0FZGjLYB1KFw3Yhw5ceyS6VwIr\\\\\\\/x1hOvslTmG9TnsulEJ1K5yWybqSflkyqkjPNhNqu3n8GT1r+3nIcg7fcYUCT8Cer0z4wt0nKSw33hi7Dd2ZjUH8O3C9QffEmus\\\\\\\/H5udZ40zlgwhNp2ji3ONnVxKMDaqref3IGuhrQmbnxyXGYmmVt8uC45Hz9r7YDozCW+m136idIVNBgEuU9m8stnBJeqhc270HFvDCcwruIYOC1hvW4tfvt69WRqyZFe3isvR3hzpB+vYhkKPCUPJjGCwVjBKZ9tCqckPXPoNP7ODOhuqb1ofC40SMeYQ07WYvuyPdaYJDRe4LTmhty3tEEuPd5GGRkLBC\\\\\\\/6XjZpTMtMiobSW\\\\\\\/Nmc+xVA85eP3oJ7z0mfwjCb\\\\\\\/ZS516WhsfC9GKiIVakwgx0zG8BONL597IlLIyIN\\\\\\\/Z5fxQPszkuYOyC0F5c0pn2\\\\\\\/qa3ZwiUDytT7pNLq69YPsYhL\\\\\\\/13LCMStdzmmDVXWK1O+svKkTyqlSnHCdmOo89TKTRh3TpNEdmXwMIkrwwtSsvlAmKSzNpxWsJJ6nQFljQ5ZdMiAkyf\\\\\\\/nnl+3v5sTALapm8FEI4zIRIHpKTPWuoxVIMP2gLZf+WmYwcYBeQtbWh2sIRW8jviIVZ1\\\\\\\/TUByQd9b2NHmBkv\\\\\\\/U6WL8G1R0mRwHWIi8WclHfsKg3U\\\\\\\/iKg9OfWDETD65T6AkDQZzGEWbmCvMCxurUmZzSjTxYh9n\\\\\\\/ildJXmlLb\\\\\\\/4JtIxVIqThIO5QbNPV6iQMJGuS18ZEJmtAMbZClsG\\\\\\\/TOSWGMelcRtDW2ToW\\\\\\\/IiC61j5gD0nE420obU0cYObVGjLoP6viv0Poc8bDdzykz0uazP0bxX+ahhVNHgB2T3pKwLgIB5Qnz+o3ayf1U3Z3PcE1KJ1gD2VOd8MlouIqMewkAyGORCD0qg9zWkGtkrmn95UQd3hjxZxnrjqQV8RlUhHbL9Gl6k1EYV24GTjMEBMOahI4Mm1zi8uODBo8yLvIa6aEvLHz+CGusBYhAYc\\\\\\\/lAoJalUDslY68gUWz2Kk\\\\\\\/Q8pE80PD2DrX0eALuoUYNgrr\\\\\\\/02o\\\\\\\/l8enUsVnDVRSRvBDUwTINRUu3rErY8pM67y2YW+ftI5YrdRYEiFXZVlKJzfyv+zfKwFfE\\\\\\\/92vYc3BS1Ukwsu3WxZ1EWmiVHqT0M3knHqlCQuRQTbd1Oaj5\\\\\\\/7H5c2IRdLsrsj\\\\\\\/+R7amuisYbDfKELId2Dd+NLpjleC5eAOX22gW9y7zCTdToLEZJqbw7Dh2HXNGgQ\\\\\\\/N1a12uSYH+hDlJ2lXh6olWmUJ2PLZb2LjgqyVzno6WpmVR+C\\\\\\\/M7WeVKOaQEoYoJ0lq6Cc0OAt9kQcAozXc4X9TbQRHwbQTqRrrqPqK+f57PF8f\\\\\\\/Jk\\\\\\\/+QO3Dzkou5EWqxlNA4Roerw90lm\\\\\\\/juKUfsW8Dq4GscQYJdj9hJG0owomeOrSGlW1OF\\\\\\\/mU1LXsmSUAw6d6XeksXqwTdqY5TcphflHMf7UZgzAe8bRtXeu\\\\\\\/MgWETqxr3zQSN9bJeHjWwpUUsJGyA5XTEQGaTXcAZAINTXqv61T2YFiqmIXD99P0BftxTQmv8y1d2tMq4XI+yyRWjMDo+nWkupO2d+SX7vB\\\\\\\/L5o+H+MrTXfciSUgF25nWBKNTP4BEEr4hb+dvZhUVhZGdjl52AF6E4Pn58HtYZZT4G+kGN7MDJH4lIFit+QPbriPX3DHMewUHr4v9tnCbgywP0OKxOcdzBQIA5Dn4Bb7LzQ4PFAtfiYQo1SXMa7CKCUWz41LmrEZXHN+ZxxZDTB\\\\\\\/pJ\\\\\\\/SkfliHpwzSkfOVnD1pC+ZFD9xqc021rL9eCwvOfkEd0NYAfxnGjMkMjXPX+XWu5vqWi7KSwZ6Lx4vtHTxqn31wW2laYw8lF5gC0XLRY+yLpo3+diLf8MoPLSyQY9AWoIjgDn4rsqWpvAz7EjZbSyHEB3oENVuN7EHckC5286pzKBFOPJ6PTnUiNFJ1UnhZyOv0KQDqF\\\\\\\/NBpjIZUVaTRuJ2tI8FpwAXoMIP84lv2BkLMqkyt+hezwVSOcTnROloU\\\\\\\/V4\\\\\\\/DZ733FC0u99fy+cTKa90JKB3betMZcWS4g0O4evIwDNXHLHlD3cg3yXBWTGeqG9VmfquuMdlxLezNQwP8N\\\\\\\/f4tZ2JpF4FT9hSNrX+I8fMl+K75AnUyYiIlKIPGMh\\\\\\\/43mS1dZ5KZMEqCNgmOyl8FSHyBQ0K1fooa8UxUVwGjU\\\\\\\/GtEChplu5bBTAkb+V5EUf\\\\\\\/Ow3j7DDTYRxFDvWaCk5v7SjfUWZ+AtJr92jzJ74EjD0fcjtlYihwe5JhYb7I0PHVKKe2J7aKGc2RizGUVwZiz6O2Uhdmqp2PkYISAm9MKjFuT4uz9tgYrtjesvexGX9KIxGChF2jq77kxJkwvGVtX6AjKEA83EegGeYUkdUa7WsGjUW57RudE+bMzC9r+w8KMATz80Ua+HxUeifLBxtSBSSUjZsm+T\\\\\\\/9SoM1oZQCQaDs4mplYkPID9MLCyT4RtnmiU4ZWKokbsnUTSWARnjT4EBTGoi2FGNKU\\\\\\\/LnF\\\\\\\/0NvAV9VfRewehU04+u51BJDeWQuHibMbnZQxe\\\\\\\/a2ZWoxCAScdnQT4piIa\\\\\\\/T1dz+WCFWSgf23A4ZLfMM7524CO0A6k5UoZ8EYfvbLRqzMssRXvbZQt7\\\\\\\/\\\\\\\/l3Rud5U0sFhp\\\\\\\/bPO2CxqMyLwuGHBqaxNqcXJmR3C14mOBLb24248jLtFzk5OzN68caOZ7oTrBeG6xnTY20OdG\\\\\\\/Msd0lGd83P69UmZGiLV5w3E6BvxWnouTNoaRb3MyvJu4ZLfVfssjPLLwd0UcFOTVTeKWVz\\\\\\\/TfS+bNEtspWsx2LxbO7pXsQAWyQqoN3EqnEHLNxkB0iCYIpO9R20+\\\\\\\/FRzsTLYemNAiZsv8tGf30jtb1D7ghzEGfiym88rsb259IDkK7eF0usC9yJBMKIHZ18XdVphAnbOuDc7pkOPPv5WRCZiV1aUtnk\\\\\\\/QKAPuSzuuwa5ttBX\\\\\\\/uHCpJ+4t0CSsUzUZAA1oOjkUaXr6vbUc9eqOpGNBBTr+yiaajHQK\\\\\\\/oHXuyb6UfXE1jSlt4TKO\\\\\\\/9F8YLsqBPtNy1s\\\\\\\/c5WS+nUjTqMrymSwSB2GDWKoQ6mII8qQrDb4pkT9vFJbfVcgLW\\\\\\\/PhEo+kCrOMisxMEp88g06DX\\\\\\\/G8RouJ3vJeECvxubO+QUw6iUgClNDgeihQNYDNBNzgrDEX9iN641djH29o2kpc7t7E+lgH7hiSzI\\\\\\\/iXkX9IEg0wMMmjnI\\\\\\\/VRcx6mU1ufTbrUD4i\\\\\\\/29sfdtCC5ltlcE+yvt4gwC7X4\\\\\\\/F0ziRHoB\\\\\\\/l+p4K\\\\\\\/XXPiAp3rdSEXwWXdAVOpwlnbEcQDcORiPjuWj0QagH1Zx73CMwcAc6YZO+nFv7LPe7zWsGuWfV23Oo8\\\\\\\/EDLnexyfLvnMpn00KxPh6k5JUiR6iNLh4irw2HonOU8KNorDH0zYxiJkpJy56qURG+zprvjIzObuEQQ6Bbquj7M6pHsX9i\\\\\\\/fducGenCvlO2jq40uZt2W7U\\\\\\\/udDeGTWmXo2WUtCYGnGtT59lgvTyBPqb94pFQNAvYCxcaF1H8cj4pKM74MgxZKir37I1QeIhYqSl17YgvnIThhk9M7qdZLspe751Me5F3EM9dBidVyfxT6wVLmDMhgx9ChwEeqmdymbTABxwGanoF6NNCivPdVxMXo6SOQvQmpppggJ1i25Olff+D1\\\\\\\/eo4PP0ODW\\\\\\\/CYwRJ0D1t5W26FM8Kb1ksJcLtbdUUh3O6YVIM3nngeI79PQ\\\\\\\/rQS+BaGquXxKpaRbiKquhuKQs1RHSdwqAjU9tnJbfQpUfTRLknDAsWHFHMkWIvFk48MfmB3cMD\\\\\\\/seyj3Zk\\\\\\\/+fQRBXlZW0q8ltN4n9iw0nXAAeCYthV6tZKGUdlt9GSAsogdprRDpWIOZBUUdCSC4LWLGoNmpFP\\\\\\\/1nl\\\\\\\/1m2+WCg4G0e6le5zOvWcOy7NP4oVndlmn6sd3IVoo7kMT9FAHCdWrGYgLRxvqQU3cjrpoxl4PKzMn2tuHt3v6iwUNJCAL+ZTMYcoIk\\\\\\\/q9A9UcrFr9z2NtC+T6IACfzuI1QoJmoHuVTtHA9TDNMgoa5rNynPTCYK3jTDImAvYnJSxrCel5jWiIVuxrm+uEwGwRr5Zn5M2bw94QkAxv6XTnOI\\\\\\\/fY0Cihvu6CVHS9H7S3AeVJVq9kEyiXaoVTPiQI5ijkzQ3vX2gUWEoOcfcsMpMF0gTsDQZLwoL4OMUIJwzOUyxIsMlwP7eA\\\\\\\/cBk7APcd7QpJjiK3Q7fuzuj66eQbvSZQHAlE2kNQBqFRD6G+V34SOoGcbzwNRyDnhUD5aQap4bLywzmajM018CzorMKCLEjVmAbTl6LhxR2T0pX9ZgWPWcKqyCtwi+jcajNF3pJ244gf6gISQ2f0cZQjcpDhGGZ878dLjW\\\\\\\/DcQ3XauxXAQ2z3vxJCDZnyqbRetJYKPNiPwQ\\\\\\\/zljuZm36Y4P26GBnr1iyqED21PAOUhgsLQ\\\\\\\/1\\\\\\\/dXxQDBXhm11cR4URrofwzy9jweKuIM+9LScEZvQki52TTsvKBbe62mU7Pqr7rX\\\\\\\/WuA1E8lx6YD3213wlK6oNC0oPVnzcKhVB7guAiHr4TuO0+rVYC9RrXjgkt51kZfNeI8pJ1RrIMrySFfeUObxDzyIVrTxzKV+9zpbr\\\\\\\/mEvrdxwl\\\\\\\/4nNXF6UpZOaqWU+VesGweTlYNsqIs6ec\\\\\\\/uJlOi28lM8gSQQuXaZNefz10NQsMFO3BsuQn7YGWCpvI6wdGq9pJ\\\\\\\/x7SMnIZKYXHsd+QfD\\\\\\\/47wgYNxJX2Klr3Uho4gA2wZ1meijfhQiXwnacIQnVnsfRGhufGaq4rXUc0OE\\\\\\\/196LLR7OX6caKro+1lHGJZHzn1QI7ARj+az0zqQk1OG8I8qRYzlMOICs+eDyTLLKfyH6N8hTSWR0je1S3kw2xHQsYfMc6JnDSH6ulsiiy2BQppGd5hzCUmgOPaaSn53uGDLkWFXeKbcH19JnRFHkBtpzq6DGVB6lV+0sPoFAmSM4TX9lYKj05+7rz3pzfcsEv16mNDZm0jXxPbKM0QxmYSgA\\\\\\\/4bOp3x2xcUMKErllw\\\\\\\/xdphh+j95N0cMjuCPDSwE\\\\\\\/JgrHXh\\\\\\\/rgldu9n\\\\\\\/HOmmofa\\\\\\\/vPZPZdErSWK6hb9ejzHQ6ZcnVnBCRQ4turvXJYBEE\\\\\\\/ZtH18rzEZoPaTc2lgcJfv8GaFoEMJ6xqekoAUA03H\\\\\\\/GbuFgG5nVDx7sAD1oZF7xbq492ymo\\\\\\\/MVpT+DdzqJV6G259ECDoZv12WkaLthXcp5SUUgUVfiGZ\\\\\\\/CyhRkz6EZVtrG8oWYtwxOX49xFVCh1MjZvXCXa4pAmDHSkMoo0RT7KMqW+JpTH1g7EX3DN++vTcXI9xY7rqz2p4N2FATIQMazDkPukaKf9lAWLQXQF1gX3QazOqep58YyQC88FdiWRS62Gs8FvCiGMozUC1Yg6I0ydbjQjurFXJOgadT3tJRhr+92d\\\\\\\/3jlNaKGkwP4riJvVBQ06l9wK79aAsElI0E7bcXT7c2wxRvSp6bzJgikAWScaoLT\\\\\\\/ZA4XZH\\\\\\\/kLC9zzLXjv3A06B+QWlzkqXL+swCnnaRDB8h7+zLShMu1W\\\\\\\/OQucR8iA3FlS+J4mth6l5xmEI\\\\\\\/IDIHSPrRSLuI3TNN\\\\\\\/q4OHrPqoxSvk+cjjsx7kXo9jP\\\\\\\/8x+Q3Ist+LBtd5GCY5vetSN0NzxBJV4t+Eo8Qs9jMtf3+yT+CG+SU0xKCXiVGmW73IOV+JK4EDeA8mf5q3\\\\\\\/NoRij0o9h3iefIAAk5B1IHgXppGEi0PNMyZZgFMUfOp4D7P8Hv+IsEHMvbORhWyWtSaaRQaPBb5yTVrt1hdXAf6cPIkJqbCYjNutze68Q6pE+dvvA3tBA6VvCoAumWP4eFni4rUO\\\\\\\/ot8Esly+D1y8yfgmvdYADHgOdZgP5Z4cevIpCw++5XaYKCZ2C\\\\\\\/HgxQftgZNdgcT0qtoQSwKk1LFfdAzE5sdmOCkQ20+7+zj4QsWA8RqLPXx+bO\\\\\\\/7Hf3qQulqISBHvK9HwJFLOK2mpGzAqGakgG\\\\\\\/srjCM\\\\\\\/UKkkAV\\\\\\\/2mOrWe+RH0uKVmDg2c7OO\\\\\\\/o+Q3o8gfhfGA+dWcWRJZ\\\\\\\/XgBL\\\\\\\/gaAJjnTMTvpLBAvF437W3pT11lVCtEz260tHfNMMpeWQO\\\\\\\/JR9DrSopNaILfb6suXN01zR7Fkl9l1whsaGdLb1AmjFBSb3N8+xClvujIqvHQIaZu8UzmWDJki7sk+iB0cw4Ead0c5BreIOIhdPEVHb2UuGjqxEtFQySkyXcouP4iSvSVJCyASmqidy1IwrOxs1QsSzMlw\\\\\\\/iZVAltjYgn8VBi4MiA3SEGT7W5SKLHfJr3UQj2\\\\\\\/tCO3yz51dQux7\\\\\\\/xLfzquT2KzN7tQp+xzgHij+e8QYXczcr3xMwZ6PD2adgTOSXspF+DlID80c8DQ7qfPngUTgTkSLB3Ti+qk+uEH\\\\\\\/wsSMuprT\\\\\\\/i4EJQ97X5VjImXm4S8H4PajukrQAFbWqh1V08FWRdW0fE8twMcfP\\\\\\\/whgc7gWjN+Avxw2uXFR4paz+Klm0j2UHX+trlzm4Y0jzsgxAfVvkG\\\\\\\/ddWX9fefKvo1uPdj70nARIuAti0Ap9DZE4YhiTus6rCmlp4y5qPnwFsHGGT88fA8ulg6CjjhD7Dzjv5a0RpB97\\\\\\\/l7YrbFFELTwdRAeo4UrfpAZuqF5mTHviziOYrd2ZcEz8CgKm+igae2fsePYdPxK3Jubcf5Cq0giahZCMLi7p9XGkwJ5QIi\\\\\\\/hIOyoi6pqd4n5c2vTv7JFygJzCEYavlVTOdfB14BoG2vQ\\\\\\\/0q3L6Lb783lnfTgw6\\\\\\\/JuLZ0hib740VRh60IPgJy03D+5mrNno8SBZ\\\\\\\/6kp\\\\\\\/6iP8jteYmRVeTTh03M2XE3YMYW4VZ17eqAXDKd3XnVSnywvabzt6nTLygy2+4OgoR1MzmHKpaplxfysU1yxqSVwCkvjaHIorxrTRWVsV3KQRgkcRL04UrUh95yB6MCQq3RWCLQPCQdYT3L2ZUAH\\\\\\\/WTIZUegEsQx9EnsdCIWYT5JXMKvmZDnRlUW0QP5bpM9eJAVCyvIdnHENyOutlzUnI4JBe4YWQia7lX8gpnc5i+uefJaYdxeShGnyxoYC6K\\\\\\\/N\\\\\\\/5L+0819h4+m6Qy5VYAmBvjFV4DXPeKmYUR+ZZ6I206XfcyErn62MXkMEnSCp4MoQIIw9I05s4jHH7llBBmDpwviuz0zlFsfwHnoMYMGviSYRxLNQP5fYGte5dgBnzhC8hvD0sucLy9DENx1OwAOmXyCkCj\\\\\\\/Yhm7xmEfSOeauoaLXEoep\\\\\\\/X1EsLr4dqk+HbdrVGyDh6LBXwrCIKLJtskEoX4i7E01\\\\\\\/AoEZZ7J5ihdI3laxLBTFoA545tbDtT9VApBOJSVrxx1z6j2FFcbaf8JNhrGVBqj62i3hdeXYttN+Rlr1RjXDCZsJ9SrxmsfNdXKkpm06wnOYU3pRxskSdBoD\\\\\\\/Kdt29uBt23m+FcOfwjpNMivLcE5p0\\\\\\\/nPnxp1Mc1EKTihbix4M5e2uZxDEav2XexgPDHtiyVTXwe9\\\\\\\/vmvg8HYWmmJNxaENU8UGzWNJQYHkvb8PcxWVXCZ5fnkD66zY1\\\\\\\/\\\\\\\/14eYOvWpZiYpx8fae2JMajRgjePulcZ4xMWnw0+DLQrGasECwJ7GquP\\\\\\\/2OJVLwEq0EIeVi8XqPM2TY6aTVkG8IJHOd5gf8I8wbedULXp8c1n6YM43GSm\\\\\\\/cEblbbxRm\\\\\\\/nZAvA4wQLUUmZ+TpiLO4oUbpkpLKUV96WnxfVSnL46YszxDqq2UP1EFkenJ3CIvLG6ncaJC8FQut9o69Oxajr32Xfa0SonDdMIRaXJ06+zU4hUv1LNc9fcm9JfL1Msa9lc2UaKG7ewJxn2AZg6CatQlF\\\\\\\/3L2LERfuM2WP4zVo3bddASbBiw73aalqS320o4kzIyiWzuWShVeo6EyKwypq7yyDh2FdUOlkgj0i3NlBfeVOkncdChob+GoPdHToNZ3cb8GztyWGLZqi0uNRvgRZpY04Kplee9bxql7z7AecvTsIbajxhNcWpisI0tb2iCvKMe5nnyXzfcvIFrdCamUk+hdvNmWNenjeFEr767\\\\\\\/u90LteM+n+3xfsRtKAiOvgZiDI9gqDHGxDfRrQzAltEjTPF\\\\\\\/+apso1QGS2Z4TClYdSCbD15\\\\\\\/nauOBCGG5ffN\\\\\\\/FkKvpPJsqnBB1iKspvi8aho6iwJbmhG4j3FPN7p6hiOZr9KcOTzxmhJRZKtO132zPVn1PRAUDyAv3bHDPWoDSGCWlJlFf\\\\\\\/rkhAMCrXc+8qEcFWVOxEfRQNRMIs5ARcOADoOy2OdpCOtPdvPmWnZWE6BtaQF+ZYfvkInFdXnMjVHQEV49E6+0VPh14jmmtCfRrKH\\\\\\\/hdWalk4OMb8eEo\\\\\\\/wvNqPANesGGdACForxNgBI00FbcMg3V\\\\\\\/NuNGypO+AwOHG9kAHE6xZHH2QWNJaPJHj39L1VlyB0PrWXQiu\\\\\\\/R+lW7Oxen3AA4w8wuWH7\\\\\\\/hIuTRUryjQe9RH9L71rNqNvKvhGML+h9Q48XU9lyMNB6TuAZMf6yMGYPwOLezWZubfT9I\\\\\\\/8oyB3\\\\\\\/nwy2riJFYdsGeIdsto5zlw\\\\\\\/SHjc0B3BcKCQyYqecs0+D9xKquljYtKkVlElBDKJxn3MJmAyq1NwqCcxtpvkcXmgFT8MDDoBukAZ4hIxfqhEnb1yj\\\\\\\/BYDZPRIUC\\\\\\\/TjkTMV4lzd+aNJs+ZS\\\\\\\/jzY3npOTWRMfRih0LToT0nzGpoZl8D4FHe5FpgdsgDibzia\\\\\\\/1dlcsKM1+EOoXIxb85OAbeCRENyYJinMGHYI\\\\\\\/C4pYFs40iosIYNlJgIWPw6d\\\\\\\/HtSrG3jXgVS6idpQFGovQsZXo7lYb1KxgK0tEW852KqtkFKPQc7coI1C4lmUx2+jyBLQHLzWp\\\\\\\/q8lU5vM87YOjwXPbjNGV9YjgB45BicjbeMXcRvjmRGrh8VZXsPGNr5i2p5IwvJu+d4ju0OXneVH0x36csL2q+ndVBjiAeSP9r10P+KN7QnEQQ9m9SLEm8LLoC8R2whWlrWqc3hYQA114Xyv33iUishRDZ0LGUSUXiIT42N3\\\\\\\/jVjFiuchPb2SAEc2xS8f7O+YJ3Kd8k+5uq5w0lQSoUbOSp7Li6r2q3uRc1iNruR8QnaAaapZhRzcc8T9O+n1tnbu00KnIMUcD7ZxkwkCHCW\\\\\\\/aGUkYv7evrcwi234tkT2km5UPmfEr2fTvQYXs6W7E3Jp10IgG3BMVvAGqfMsE93NbzfQr5qpuKcP3cEnc7z6Xjtj338MpiSootGb+UbIKdCLoWAiRmXcLL5JTGagXGPAgh4kRiP5p3bn4OdKPK6E3wRS6IvC694HR21Xhi2K5aZOh6Z7kueqN66ilYEnD1K0WCM35fiQZV6Y\\\\\\\/23yKMo+Zm+NPdRMEkmeBnjWYCwc5J0lMCj3miZpoUssi1AWwuzORwCixiKHncqsc\\\\\\\/FBy2M8khhg+O2mcESX9LID4x5kkxxH2Qh52lwUnq1FAE5ugSx4VnGuhMRGo9kK4pu+Un62x4lyE9HR25sSQ4r4RT2zHQP064QRiDGoMnMw2FaFB26evWldZrgHcQLIUayLxN4\\\\\\\/1yhyA7ub0\\\\\\\/F\\\\\\\/2sklAbdRZ2pqGQXRbFtOAlrcJasJ3fXt90+YRGkqrS8krdPUxIGi6zvVvNU7Z66yTpH1q0rxtX4D0rCP2lSuYlAyNQcSHO5rDrx1iFrumbEed\\\\\\\/xfE0rWQl+idRRQoCluP5tPYfUaEm2HiqlG89pPme\\\\\\\/rf72UyFHPijre\\\\\\\/NhRvtfxmb0GqUWH+x+9sq1BCGHfdppSbvJuguyzmhG6gkn61SCRhMfghTThf41riliOyTa3ps913QBeClArJNAVNyBjKDs6kb+ywGdl9QcjShtyv7h\\\\\\\/LmaERrwmDyjfkxVARG90QJIKWvsUQ88E0JRw7gYpXrv0cGUwlsJzRSo+x7bCLEZL72FXudc7juOFq9kbJQw8k3vETYRzSVxqC5KmM53\\\\\\\/j4+5L3sk7Iwu0f7ely1NSv3d8L+jvnjIenejEJN\\\\\\\/uD\\\\\\\/Z4+NFjMNYC4Ef4WEaUVX+Id646X3IdDRk6zbSovCy1NrSIqWU9xPTabrzcL0l7gCkAUHisgIxFUvLapvwoPSDt4CHcvzSoPUide8NYfi\\\\\\\/R3xj+73fEPvt37x1bzIPi74NluaRMwLyh2bOoZqjEiOc9FZyBib6J+tj5AHC7o+PtJmfPlC3ybuFqjjmKY75yIRSdxPb9gWTsgu6kmdauB8KYHk6NHJaLYttdKzrmfdCpfZiGnD7+6BuPA5SAgvMVoqU4bEJKN9sFSLvhFU5pdHFbA1wla8nmrVTq9B\\\\\\\/Rd+hFTKX9H6\\\\\\\/c0XPTvb9x8pTjBGpfa7WksA0NH9xGi4mLbdIcmy+Ebx9U64ee5jE4rTfS7SbZuRKi4Sc4oji1HM8+v2OpaVJ1nSuQHRKORNEJMJrHj3v\\\\\\\/qtV4Cb1\\\\\\\/1Y+ch8Th1S5aBgFH+AJDytwj1bif00hh3yN6MUCnaLc1Zjamy8TZB0OUpAbQ0kb3QUuPMPxyA280+r1sTByFxVyBfbSTOcwXn9khxsIJr+mylNVfKmpY9GAlueZ3QE0DXYHiLhm7smSHQ4xsvPnmfCm5BX6+BrJaNFa3nqfi1Pikfqf93BYQ6tfBap\\\\\\\/H2hz1VAZWFumIp1b5v+PaIY3viRWdrFxutlgYafV223nGJrB1wxi2i64okLUtdR8j7g9d2yGCIRf69jkfeULAAD4aK9x8LwDmhHgICRpaqOuFc8d0\\\\\\\/p6+5pRp8EXa5bKlNnnXlLUu1eGCoLibey9ySYgtUd\\\\\\\/stshUYO59\\\\\\\/6k1yxrUOB\\\\\\\/W\\\\\\\/wfL2kQiuHQKffFVXIy3DiunzRP+hKW22+uYZlx8MGlYLhzA1muFJo9X84AHdlvHjHq9w2Hl2fj9FA9i93Gzw1E0\\\\\\\/oIx1pJOCy6B4vst+uU2W7+O5M\\\\\\\/S\\\\\\\/zbE3vilPkt+bccBApMhNAUmT3LoC2Ylg6ZmOM7+m8CIkuQAKAFxlkIIAf6SF2TIBs2bGWAH77NZKzbHeGg0PDld7XbkebJ8Qh0I0fr5db\\\\\\\/0w5N2n8WeSI9\\\\\\\/TSITB+QyRe\\\\\\\/GunOtKawvuODlfOozp1SFLe+2JOkuULygaoKmJAsNUrBFYMm6tsQwCZdKA+6TrK8W6JgndsDnkcQ1S+8NDueWpK1E3X8MI9WKI4zdkr\\\\\\\/It+DWOJLEp3nIsQo\\\\\\\/us6SWV60Bu64powBhWUgBuUn4m15zn7HNHwWEck6wrjXODgYFwXgGVD9Q6utnCnuI60wmMu1kJq3kD3jo9bG9jSUJlz5ZxAhxnOPnLEJuQzUeIO0XiNVCcPivgzBm+NsQUN2dSPJYLeHqtixsUndCj6B\\\\\\\/osmBUvQGnWmMcWg7HEieiR58aiWAsFT16aAbh0laRGqWOD5uSAk5LocL+SI+2tdqK9QCiLg63vZw2HiTciTUgDBECvXH3oq2xqORZO6T70OL6RoDgj80NYvPHXukADs+KA1CpxlZ6eSzQ8j\\\\\\\/eYWYqAvo4Wpngp30MobHGJBYRKnzx1\\\\\\\/MidJTtsM9al2W6JKJWoFK2c35hjhA9ghtVoC90rON9bMWkdymYHGg0TMF3nGin9ouKU46HCysrj8QPoCiiyWqJMguW7mESHbeHQpjVaGhkHdFU8CiulHTFBAaVGaOUmNz2iPM14VNSt3Z8L9xo8MjJmL5TMmhs28Ato43Asrqm3U0CqzBeHfBhjfRHFHhfMmSKDfjm3VLaOIWSnacDryKB1PjYLgii6qXWSH6HzwfNM562ed0TcwsJOA3XYRfxJvH20e9E\\\\\\\/kue5LV3VFhM+vMzqoaib4Pkm9UTgwKI3mBC7I\\\\\\\/n8Ea9rVaz0XancTRPHQvxXJAYcXO5TSi5HxApnxIxPgcHOIXJFoH2fjZJZg2vF4CZLNxEEQ0M\\\\\\\/JTV1cM+pjOxnvnY1Bzn\\\\\\\/rhRXYdZCeRuTH5pmkOsu502EpRSDKGcN\\\\\\\/9w+nddD0EFCH9SSVS3eLw2zYShi\\\\\\\/qWDFislTuCF4DDSNBXS2WcoCPdXMay7szz2zoOhvRojx2\\\\\\\/W7Vx0RYylCzSTan4UHXC4nlgNDcmnW\\\\\\\/nAdLaFiktubl2M3nsz5JfJtWkDD0guevnGMbep8QiHVZ4Jrg6fQEEucuOAwzKTpAqq4H+QuYg+LIDYNkBx45uZMQDvDbvzM00x8q2U2xVN9\\\\\\\/p8ecHGIzNCdZD+jZFr62D5sXv01WH05TxaBpr2IQUEHErCQB2if1FU4FbUC7Kq42DJZfx3S9NEOgEnhdvOXq+6Yx99C+yc8QhVERB98hPxlPn0MphHjGfFNu4q9+JW3yGARZN4oWwjFXzwNwSfNhdgO862b7n43A5llozETyK\\\\\\\/w0NKL6Pkol6r4tkn7SIVIP5ULiZm6whpQWiiDrqUmmJMwmqrI4IndbH2lvuGvr48auNlfwOAUl1S3usF4QzfCjDRQT0cvFoyY03VeG6A9nYTPPEPlCpIeMT1PVsMDXg3GHEukO25cL4t\\\\\\\/qXFzz9oMnI\\\\\\\/GUTQFhoeNY94oUamfyh0XX8ZnTX64EGO16UZTpjHTE4EUGKgzm6idE75Vbfc84GPPjo1IipLeoi2hWUvA\\\\\\\/4jisjEaYV9mrDcUIXdITre1fU6pEVA7jkWPGVYraie3\\\\\\\/3SaZgw2zNFFOHRPonukzm+3Sr5y5uRn3Kuk5IRxlHlO9N7w2nnCHma\\\\\\\/UigzW1Vth9963fcmyOaAZGuhLLyPwpvSEwnE+4Q3yuf0b5gruJmIwYXyinow5WA3dJ2uG9xufcQhwJVsSTpqclOr6BNvxIL48MgqoVplyLx\\\\\\\/JPau8S1h2W7zwBLI2dYIC43TAkUSLCnEQU6xDulggcGAqAcY5Gf+oyDO1yyZCOOYPWudjN9O4ZKVM3Hyc9CHLYAM414E3ffneIfuyIGCxJoCtjYWSUL\\\\\\\/qH3+EvLQzFLFfeOjtgkd9iANs2th0Hb0QWUWo6uH1ZjCjvowcflb+D7AETqEgHo\\\\\\\/p7sXQoIw7nA\\\\\\\/Hhv+Rwmz9hPGnehgq+GIsO9Uw0NIOfelMv0XNVb61BxM5GKWw7J2f4pAn78g==\\\u0022,\\\u0022iv\\\u0022:\\\u00229HE3o3aBpYg3s25vp0xsAw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022299037cf60ed44d1d0db9311cb7c7642905b33960de06ffa922481f0392a63d1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022P5xRkGQqoMyKuR4bLlZatqqUwkCZwnCPQji2qfpUVqVxv1X\\\\\\\/+qfBn6h30JIiQ11pXkltNctBxFqvnH4B8M5\\\\\\\/\\\\\\\/66ObfT92NqabL6eVbFG7BSIXuVWugpmSviBLf5sy1ST5Y\\\\\\\/kV2QSsamT5MIxotC+79cxJldtc\\\\\\\/+KwOAAMIPeYWEPfggoX9vbjo7bVIzCk2a0P2Ne2\\\\\\\/qDm7YDBvnAJQTWNDSUIHMRRcPupalogoYMY8+nQBqZmEciOR0dQp\\\\\\\/QGEGdif+wVdZ7575FErV6fDQVTgCQngziS29D1kKIiWQUrmGL7SkK34BQ+Hi+QmoPIPgBZ\\\\\\\/P5PDAZ2QJx1a\\\\\\\/PPiRqmXd8o2+FYaP\\\\\\\/MmVb+7gozNLvw\\\\\\\/RRnjBj04YBs7iH1nywdQBh6doa\\\\\\\/VCWd0kqdi1KLd6H+VSHv+1kWTJTX4uiUMlZQWhLX8Gv\\\\\\\/OxzFLAkTekmvJQvnn90V57ThbBYO+yjKlbhyy0BFPwb61lO7nOxwUcnh6LvcoRKAtfnhxRpIn3bdm\\\\\\\/cxuJz9vCG1JNbJsDKBh8BNTbXZr+p8IkWd9aLCPmlTIwCc1GTKfvlE79msNS6Q7UvUAlxFwI77OIwE7kLNRNvjQ21wsx8x8fcmCtsOwnpAzCwxcUwcnDAJ1VrBYoHdWdaHW0QSBY10YQ4fGq4mTrlNE\\\\\\\/1yg8Avoe8pqCQtx4TeAZYyKOcPDe8A+5KV7D7dSXgmjG1chjSk\\\\\\\/Z9Y+faeH3JuS6U1fiYAHtdyWcnP2aNJEm3CV7EAoyYYFUAdxcxfddF4XdXjyQhD1VJ5FxXTyxVR0APb\\\\\\\/BxXWLcM+iXtnoT3UCkI+EcxvzEsYcDE6jXVLOpNjBp\\\\\\\/l9xdD+B9vymKOYKYcaEGW9qte1us94qhc7ZCtKnpdjPouKAy3EW4teG1\\\\\\\/wK49+lm4S0o7QF7aMSF7+XSlh+nrKqzF0L8Gj4XSxQlz\\\\\\\/XIJ5larM2aG5kDZYFpP1sLT7Dli6j9MGy\\\\\\\/cI+N4i+9UdIbD0j60j0MpPZWDUxKgpdXYwew\\\\\\\/wy5jkxNtPlSyFzN+JD7dXxI1Bk+QWCXnU2H+yNfC4VILBf18pObtWTPj5wceenNm64TZZjmMQjwBnC5Cc269YwiG4mfsnqvEK37DCH8ul44dSzR1OqwEW\\\\\\\/rSfMpaFet86KruenrUT3xMO2P\\\\\\\/wXRgd+ExFeOz2cL6Ibpc1JF4yBYIa7C5I4iz9WRcKucq4F8mW5+z5hs0EC1pC5b3ICN4kaqQRzgXtPSBRNG0NEMwHd0dAbf8\\\\\\\/IO6FfbQSrc6yDd2+9pUBdJMHGCyvsDdYTyFJi6CMIriSS8YfMAW50dlD1+08GOj41attg59ikHGHmEZ6WNgiK+DjchpUlmRW1qh88ncwBV0hTYMHtCXQhzUlveuuijE0A7rVtTBbR+X19srDUAXtO9sOfXpkFymR93gkPD5dllvmHCJMRK3TAotj93Y9AKKvSSJyAtQMxb0EfXtieXx12UVBXUD1jsSHV1sQHMCPHoYAi1wAWVK3qKmsxvPua3X6XI+WwV355hZ6juTaAF0IhzcYD47iPw\\\\\\\/Az7vwjdkzvT1lWgaKO9OS7ASbq63fxIpV+NfgDD8B0AvZ2BHan793ygx2wy\\\\\\\/1Dznpn7tdvYnQ8ewxw73uCX3LKeXr\\\\\\\/yjOmhtgDppbnH\\\\\\\/Oq\\\\\\\/7SkF8vc60dG6nYGdhJ9WsNsL+A8JZqT7elIyt1DbG0eGDqHorPteCHovxFwWeIEk7APqRDb6+mr21b4egGy0dukZVsSnW9ZkB5LTi+N2SOOlu5U9u1uvyd5PUAfpQ9rb9QF3kiacyp4khGB1GHCBW+rP7dn1a61Xrh+vBTvljVfhWUTMVyiAyhkkm09aYPdRcInEW2\\\\\\\/opC5\\\\\\\/6sZr23lDTcTZ9IpVVYzNDg6G4ZfYVpKrL\\\\\\\/evyvD7BH8heHqyycl9DrW2GZ2mjauN5kxr0uHYx8V7PbV8PQ4UYvNeO43DHcXqiq94ID4ZmOmrL77KZgvSbleICiGMqLjPlpzrtCxYPP6B\\\\\\\/ieAEAOE2yA\\\\\\\/w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022UmJwZX\\\\\\\/o\\\\\\\/XQgSZw9XbK\\\\\\\/XA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226352ae564c6bdfc9bce7e7c80be6ca4663b91e5a0bcbbcada95e8141105f14c6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484782 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-population">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1721" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JPeIB6iaZ2ul2i9TeaeNSLuB8dTIxy2NpZlrXWpXGR4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1721" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3eseT-mXInXeLrDkR7ln_aHowaXK9PYaLyn0T7PisJg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1707" title="Pedigree-Based Analysis in a Multiparental Population of Octoploid Strawberry Reveals QTL Alleles Conferring Resistance to Phytophthora cactorum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1731" title="Whole-Genome Sequence of the Metastatic PC3 and LNCaP Human Prostate Cancer Cell Lines" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484782" id="node484782" data-pisa="ggg;7/6/1721" data-pisa-master="ggg;g3.117.042101" data-apath="/ggg/7/6/1721.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Approaches in Characterizing Genetic Structure and Mapping in a Rice Multiparental Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chitra Raghavan</span>, <span class="highwire-citation-author" data-delta="1">Ramil Mauleon</span>, <span class="highwire-citation-author" data-delta="2">Vanica Lacorte</span>, <span class="highwire-citation-author" data-delta="3">Monalisa Jubay</span>, <span class="highwire-citation-author" data-delta="4">Hein Zaw</span>, <span class="highwire-citation-author" data-delta="5">Justine Bonifacio</span>, <span class="highwire-citation-author" data-delta="6">Rakesh Kumar Singh</span>, <span class="highwire-citation-author" data-delta="7">B. Emma Huang</span> and <span class="highwire-citation-author" data-delta="8">Hei Leung</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1721-1730;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042101 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484782" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chitra Raghavan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chitra%2BRaghavan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghavan%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChitra%2BRaghavan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ramil Mauleon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ramil%2BMauleon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mauleon%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARamil%2BMauleon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Vanica Lacorte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanica%2BLacorte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lacorte%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanica%2BLacorte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Monalisa Jubay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Monalisa%2BJubay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jubay%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMonalisa%2BJubay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hein Zaw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hein%2BZaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zaw%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHein%2BZaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Justine Bonifacio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justine%2BBonifacio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bonifacio%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustine%2BBonifacio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Rakesh Kumar Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rakesh%2BKumar%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARakesh%2BKumar%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">B. Emma Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Discovery Sciences, Janssen R&D, Menlo Park, California 94025</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BEmma%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20BE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BEmma%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Hei Leung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hei%2BLeung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leung%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHei%2BLeung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:h.leung@irri.org" class="" data-icon-position="" data-hide-link-title="0">h.leung@irri.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/6/1721" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484782" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484782/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1721.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484782" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484782/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1721.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484782" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484782/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1721.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484782" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484782/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/6/1721.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484782" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484782/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Multi-parent Advanced Generation Intercross (MAGIC) populations are fast becoming mainstream tools for research and breeding, along with the technology and tools for analysis. This paper demonstrates the analysis of a rice MAGIC population from data filtering to imputation and processing of genetic data to characterizing genomic structure, and finally quantitative trait loci (QTL) mapping. In this study, 1316 S6:8 <em>indica</em> MAGIC (MI) lines and the eight founders were sequenced using Genotyping by Sequencing (GBS). As the GBS approach often includes missing data, the first step was to impute the missing SNPs. The observable number of recombinations in the population was then explored. Based on this case study, a general outline of procedures for a MAGIC analysis workflow is provided, as well as for QTL mapping of agronomic traits and biotic and abiotic stress, using the results from both association and interval mapping approaches. QTL for agronomic traits (yield, flowering time, and plant height), physical (grain length and grain width) and cooking properties (amylose content) of the rice grain, abiotic stress (submergence tolerance), and biotic stress (brown spot disease) were mapped. Through presenting this extensive analysis in the MI population in rice, we highlight important considerations when choosing analytical approaches. The methods and results reported in this paper will provide a guide to future genetic analysis methods applied to multi-parent populations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/multiparental-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations" rel="nofollow">multiparental populations</a></li><li class="kwd"><a href="/keyword/imputation" class="hw-term hw-article-keyword hw-article-keyword-imputation" rel="nofollow">imputation</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a href="/keyword/qtl-mapping" class="hw-term hw-article-keyword hw-article-keyword-qtl-mapping" rel="nofollow">QTL mapping</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-3">Traditionally, breeders and geneticists use biparental populations for mapping and varietal development. However, biparental populations have limited allelic variation as they only segregate for QTL that differ between the two parents. An alternate approach is to create multi-parental populations derived from elite parents, in which each line represents a combination of alleles inherited from multiple parents. Ideally, a diverse set of lines is generated simultaneously that can be readily used by breeders and geneticists. To achieve this, the selected parents must be (a) diverse in their traits, (b) good combiners, and (c) have elite features, which will result in new and favorable allelic combinations. Further, these diverse multi-parent lines serve as a genetic population to map QTL and to understand genetic interactions.</p><p id="p-4">MAGIC populations are now becoming more common in a number of crop species such as rice, bread wheat, durum wheat, barley, and chickpea. <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Huang <em>et al.</em> (2015)</a> provided a comprehensive review of the development and use of MAGIC populations, including brief descriptions of important populations under development. Breeders and geneticists have actively used and benefited from MAGIC populations, and several groups within the rice community are now also adopting this approach to develop trait-specific multi-parent populations (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bandillo <em>et al.</em> 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Leung <em>et al.</em> 2015</a>).</p><p id="p-5">Supported by the Generation Challenge program, the International Rice Research Institute (IRRI) initiated the development of multi-parent rice populations in 2008 to be used for both breeding and genetic studies. The four initial populations developed were: (i) MI (eight founders of the <em>indica</em> subtype); (ii) MAGIC PLUS (same eight founders as the MI population with two extra rounds of intercrossing); (iii) <em>japonica</em> MAGIC (eight founders of the <em>japonica</em> subtype); and (iv) the Global MAGIC (16 founders: the eight of the MI founders and the eight of the <em>japonica</em> MAGIC founders). The populations were generated from elite materials representing a diverse set of favorable traits. The MI population is the most advanced and has been studied in detail. At an early stage of population development (S4 equivalent to F5), <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bandillo <em>et al.</em> (2013)</a> mapped several traits on a small subset of the population, confirming that the lines were a well-recombined representation of the eight founders. Further, the populations were of interest to breeders due to the combination of their traits and favorable agronomic features leading to the selection of lines as prebreeding material. IRRI has since conducted several multi-environmental trials with a small subset of the MAGIC populations and continues to do so.</p><p id="p-6">The variations in the design of MAGIC populations in terms of the number of founders, the mixing combinations, and the recurrent crossing designs followed by selfing, will all have an impact on the structure of the final population. Theoretical and simulation studies in rice have identified the potential benefits and levels of mapping resolution in various designs, but these have not yet been measured in a real population (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Yamamoto <em>et al.</em> 2014</a>). For example, these simulations suggest that a minimum of six cycles of crossing is required to significantly improve the mapping resolution in a population derived from eight parents. As might be expected from theory, increases in the number of recurrent cycles will be accompanied by an increase in the number of nonrecombined genome segments, as well as an initial decrease in segment length. For a population like the MI, which has not undergone any recurrent crossing âpostmixingâ of the founder genomes, the expected number of nonrecombined segments in 800 lines derived from an eight-way cross is â¼150 per line if markers are evenly spaced at 0.1 cM (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Yamamoto <em>et al.</em> 2014</a>). For comparison, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Huang <em>et al.</em> (2009)</a> estimated â¼33 recombinations per recombinant inbred line (RIL) in 150 lines derived from a biparental cross.</p><p id="p-7">The predictions for the MAGIC structure based on high-density genotypes in simulation studies can now be verified using GBS data in these populations. Such low-coverage genotyping methods (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Elshire <em>et al.</em> 2011</a>) are popular due to their low costs and high throughput. However, they suffer from issues with missing data and genotyping error, the impacts of which are not yet well-characterized (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beissinger <em>et al.</em> 2013</a>). In particular, it is unclear whether a powerful statistical methodology can offset some of the issues by capitalizing on genomic structure and observed data. Methods for genetic analysis (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Huang and George 2011</a>) and imputation (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Huang <em>et al.</em> 2014</a>) based on haplotype probabilities, calculated from founder information and the pedigree, have been developed specifically for multi-parent populations. These methods depend on the high-accuracy estimation of which alleles are contributed from each parent (founder allele inheritance), which is affected by population size, founder genetic similarity, marker density, and genotypic data quality. However, how to balance trade-offs in marker density and data quality resulting from a fixed budget remains to be determined.</p><p id="p-8">Issues with genotyping and imputation will, in turn, have subsequent effects on QTL mapping analysis, introducing uncertainty and potentially identifying false positives. Hence, careful processing of GBS data is critical to correct for potential quality issues while maintaining the high density of desirable markers to identify regions of high LOD, define haplotypes, and characterize favorable founder alleles. Association mapping for several traits in the MI population have been performed (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bandillo <em>et al.</em> 2013</a>), from which several large-effect QTL were detected. However, interval mapping using founder haplotypes with dense data has not yet been investigated using the rice MAGIC populations. Both mapping approaches have been applied in other populations such as barley (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Sannemann <em>et al.</em> 2015</a>), tomato (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Pascual <em>et al.</em> 2016</a>), durum wheat (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Milner <em>et al.</em> 2015</a>), and wheat (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mackay <em>et al.</em> 2014</a>) and, in general, the combination allows the detection of QTL that might have been unaccounted for using one method alone.</p><p id="p-9">This article summarizes the efforts in examining the MI S6:8 population to support and explore how these complex populations can be analyzed. A rice multi-parent population was used to address common aspects of MAGIC populations in crops, <em>e.g.</em>, low-coverage GBS, imputation, mapping traits, and identifying favorable alleles or founder haplotypes. The research assesses whether theoretical predictions of recombination are confirmed in an experimental population. It also discusses the criteria for filtering, paying careful attention to the impacts of genotyping error and missing data. Allowing a minimal level of missing calls in founders and lines, haplotypes were defined based on founder probabilities. The missing data were then imputed, and the resulting dataset was used to determine the number of observable recombinations and to map the QTL for eight traits of interest. QTL for agronomic traits (yield, flowering time, and plant height), physical (grain length and grain width) and cooking properties (amylose content) of the rice grain, abiotic stress (submergence tolerance), and biotic stress (brown spot disease) were mapped using interval mapping (R/mpMap) and association mapping approaches (R/GAPIT).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>MAGIC population</h3><p id="p-10">The MAGIC population used for the analysis was derived from eight inbred elite founders of the <em>indica</em> subtype, which are widely adopted, high yielding, and tolerant to abiotic and biotic stresses. Detailed descriptions of the population can be found in <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Bandillo <em>et al.</em> (2013)</a>. In brief, the eight founders were intermated to derive 35 funnels with balanced contributions from each founder. The MI population was then derived by advancing 60 lines from each of the 35 eight-way F<sub>1</sub>s, resulting in a population consisting of 2100 lines. Approximately 2000 of these advanced intercross lines (AILs) are maintained, fixed at the S6 generation.</p></div><div id="sec-3" class="subsection"><h3>Data curation</h3><p id="p-11">When dealing with large experimental trials, collecting phenotype data can be challenging in terms of replications and plot size, especially in multi-environment trials for complex traits. At the same time, seed increase and purity should be maintained to allow researchers to trace the material of interest. Data curation is important as it enables breeders to extract AILs with the desired combination of traits. Currently, all data generated at IRRI on the MAGIC populations including pedigree information are stored in the IRRI database B4R (Breeding for Rice). We have also provided a large amount of data in the supplement, with a guide to the supplement found in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FileS1.docx">File S1</a>. The data pipelines move from generating barcode, field layout, and field data collection on tablets. The data collected include information on field operations. Raw genotype data is stored for the validation of QTL and for future applications of improved analytical methods.</p></div><div id="sec-4" class="subsection"><h3>Phenotyping</h3><p id="p-12">The population was phenotyped for various traits such as yield, flowering time, plant height, physical (grain length and grain width) and cooking properties (amylose content) of the grain, submergence tolerance, and brown spot disease. The above eight cases of QTL mapping results were selected to illustrate the mapping resolution. The trait analysis was conducted using PBTools (<a href="http://bbi.irri.org/products">http://bbi.irri.org/products</a>) and a broad-sense heritability, which is the proportion of the variation among entry means that is due to the variation in genotypic effects (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FileS2.docx">File S2</a>) for the traits, is reported.</p><div id="sec-5" class="subsection"><h4>Yield:</h4><p id="p-13">Yield trials were conducted for two dry seasons (2014 DS and 2015 DS) at IRRI under fully irrigated conditions using an augmented randomized complete block design. In the trials, the eight founders and four common checks [PSCBRC10 (IRRI 104), PSBRC 18 (IRRI 105), NSICRC222 (IRRI 154), and NSIC Rc132H (Mestizo 6)] were replicated three times. The number of hills harvested from each of the plots was recorded. Yield per plot (plot size: 2014 DS 96 m<sup>2</sup> and 2015 DS 2.16 m<sup>2</sup>) was converted to tons/ha using the formula below:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1721/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1721/embed/mml-math-1.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1721/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1721/embed/mml-math-2.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1721/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1721/embed/mml-math-3.gif"/></noscript></span></span>A one-stage linear model for augmented RCBD was used for analysis in PBTools (<a href="http://bbi.irri.org/products">http://bbi.irri.org/products</a>), treating genotypes as random with no relationships among lines. The BLUPs derived from the analysis were used for mapping. Details of the models used in the analysis are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FileS2.docx">File S2</a>.</p></div><div id="sec-6" class="subsection"><h4>Flowering time and plant height:</h4><p id="p-14">Flowering time and plant height were measured in the 2015 DS yield plots. The field trial was conducted in the 2015 DS at IRRI in fully irrigated conditions using an augmented randomized design. The MAGIC lines, eight founders, and four checks were grown across three blocks, with checks and parents repeated in each block. Each line was transplanted in two rows of 27 hills and the number of days from sowing to 50% flowering was recorded. Plant height was measured at maturity in the same trial as the flowering time. A single environment analysis was conducted using PBTools, treating genotypes as a random factor to obtain BLUPs for both flowering time and plant height. The linear model for augmented RCBD was used for plant height and flowering time, with additional details given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FileS2.docx">File S2</a>.</p></div><div id="sec-7" class="subsection"><h4>Grain physical and cooking properties (grain length, grain width, and amylose content):</h4><p id="p-15">Grain length, grain width, and amylose content (percentage of amylose by weight) were chosen to represent physical and chemical grain quality features. The grain length and width were measured using the FOSS cervitec, and amylose content estimation was based on the American Association of 643 Cereal Chemists Method 61â03 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">AACC 1999</a>). These grain traits were measured at IRRIâs Grain Quality and Nutrition Center. The grains of 1316 AILs and eight founders, harvested from lines grown in the field for seed increase in the 2012 wet season (WS), were used. Only a single sample from each plot was collected for quality assessment due to the high cost involved. The average grain length, average grain width, and percentage amylose content were used for mapping.</p></div><div id="sec-8" class="subsection"><h4>Submergence tolerance:</h4><p id="p-16">Submergence tolerance of the MAGIC lines was tested in the IRRI submergence ponds designed to hold water up to a depth of 1.5 m. The trial was conducted in a completely randomized block design composed of two replications. Seeds were directly sown at a high density (5 g/line) across six seedbeds per replicate. Checks with the <em>Sub1</em> QTL on chromosome 9 (Swarna-Sub1 and IR64-Sub1) and without (IR42, Swarna, and IR64) were used and represented in each of the seedbeds. The trial was subjected to complete submergence for 16 d, after which the water was drained. The survival of all lines was scored 7 d after drainage using a five-class visual score (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">IRRI 1996</a>).</p><p id="p-17">The nonparametric Friedman test, which is similar to the parametric repeated measures ANOVA, was used to derive the mean ranks using the Statistical Tool for Agricultural Research (STAR) software (<a href="http://bbi.irri.org/products">http://bbi.irri.org/products</a>).</p></div><div id="sec-9" class="subsection"><h4>Brown spot (Philippine isolate sm2):</h4><p id="p-18">Brown spot disease in rice is caused by the fungal pathogen <em>Bipolaris oryzae</em>. The fungus affects the plant at both seedling and adult stages, causing yield losses and negative effects to grain quality (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Sato <em>et al.</em> 2015</a>). The experiment was conducted in randomized complete block design with 36 blocks. All entries and founders were replicated three times and five checks were present in each block. Treatments and block were used as factors to derive BLUPs for QTL mapping. We tested the disease reaction of MAGIC lines, eight founders, and five checks (repeated in each block), to strain sm2, a local Philippine isolate of <em>B. oryzae</em>, at seedling stage. Fungal cultures were grown on potato dextrose agar plates and incubated at 25Â° for 10 d under alternating 12 hr in UV light and 12 hr in darkness to induce sporulation. The spores were scraped off and a spore suspension of 5 Ã 10<sup>4</sup> spores/ml was used to infect the seedlings. The inoculated seedlings were placed in a humid chamber overnight and maintained for 7 d in a cool room at 22Â°. In susceptible plants, this produced brown lesions on young leaves. The phenotyping was conducted in the greenhouse at IRRI. The most infected leaf was evaluated 7 d postinoculation. Images of diseased leaves were taken under a fluorescent lamp light box using a regular digital camera. The camera was at set distance from the object, in this case the diseased leaf or lightbox. Images of infected leaves were bulk analyzed using a customized protocol run on ImageJ software to measure the diseased leaf area. The total leaf area and the area covered by the lesion were measured using a color threshold based on which percent of diseased leaf area was estimated. A single environment analysis was conducted using PBTools, generating BLUPs. Details of the model used are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FileS2.docx">File S2</a>.</p></div></div><div id="sec-10" class="subsection"><h3>Genotyping</h3><p id="p-19">A total of 1316 S6:8 lines and nine replicates of the founder lines were genotyped using GBS (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Elshire <em>et al.</em> 2011</a>). Full details of the GBS pipeline approach can be found in <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Glaubitz <em>et al.</em> (2014)</a>. Briefly, it consists of wet lab processing of samples accompanied by an informatics pipeline prior to making SNP calls. The 96-plex ApeKI GBS protocol was used, wherein sets of 96 samples per lane were sequenced on an Illumina HiSeq. The GBS pipeline was run by the Philippine Genome Center of the University of the Philippines using TASSEL software version 3.0.169 (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Glaubitz <em>et al.</em> 2014</a>). The sequence reads were aligned to the reference genome Nipponbare sequence MSUv7 to derive the physical positions of markers. The raw GBS data files are extremely large and require specialized bioinformatics pipelines. Indeed, important considerations in running the GBS data pipeline include the transfer and storage of such data. An Amazon Elastic Cloud (Amazon EC2) instance was used in this study to enable uninterrupted analyses.</p></div><div id="sec-11" class="subsection"><h3>Marker filtering post-GBS pipeline</h3><p id="p-20">Postprocessing steps were applied to the genotype data, imposing various criteria, to extract multiple SNP sets in order to compare the effect of different numbers and quality of SNPs on detecting recombinations (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The SNP markers in founders and lines were filtered separately and the filtered SNPs common to both lines and founders were used for analysis. To filter the founders, replicates were first merged by ensuring that at least two had observed calls, with up to a quarter of the replicates allowed to have an alternate call. The alternate call within replicates of founders reflects genotyping errors. The most common allele was then taken as the founder call. Next, SNP markers with a minor allele frequency (MAF) of â¥ 0.125 (<em>i.e.</em>, one of the eight founders) at three levels of missing call rates across the eight founders (no missing calls, allowing up to two founders to have missing calls, and allowing up to six founders to have missing calls) were extracted. To filter the lines, a similar criterion was applied at two levels of MAF (0.125 and 0.05). The SNPs obtained at MAF = 0.125 were filtered, allowing only those with &lt; 30% missing data, while the SNPs obtained at MAF = 0.05 were further filtered allowing two levels of missing data (&lt; 20 and &lt; 30%). Finally, the filtered sets of SNP markers of founders and lines were merged in three combinations to obtain the markers present in both lines and founders. Marker data from the lines for the three sets of markers filtered in the founders only were also extracted. In total, six sets of filtered SNP data were examined.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1721/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Filtering for SNP markers from GBS marker data in founders and lines (post-GBS pipeline). The scheme shows the use of different criteria resulting in multiple sets of SNP marker sets, emphasizing the impact of filtering criteria on downstream results. GBS, genotyping by sequencing; MAF, minor allele frequency; Mindist, minimum distance between any two markers; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-835566639" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Filtering for SNP markers from GBS marker data in founders and lines (post-GBS pipeline). The scheme shows the use of different criteria resulting in multiple sets of SNP marker sets, emphasizing the impact of filtering criteria on downstream results. GBS, genotyping by sequencing; MAF, minor allele frequency; Mindist, minimum distance between any two markers; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1721/F1.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/6/1721/F1.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1721/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1721/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484804" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Filtering for SNP markers from GBS marker data in founders and lines (post-GBS pipeline). The scheme shows the use of different criteria resulting in multiple sets of SNP marker sets, emphasizing the impact of filtering criteria on downstream results. GBS, genotyping by sequencing; MAF, minor allele frequency; Mindist, minimum distance between any two markers; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Imputation</h3><p id="p-22">Genotypes were imputed using the âmpimputeâ function in R/mpMap (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Huang <em>et al.</em> 2014</a>). This method first estimates the probability that observed lines inherit alleles from each founder based on a pedigree-informed hidden Markov model (HMM), then audits the observed line genotypes for each founder to impute founder genotypes. Once the founders have been imputed, individual genotypes are imputed based on the updated HMM probabilities. Missing calls in the lines and founders were first imputed using the R/mpimpute function, and then the markers were filtered based on MAF. The imputed data were used for estimating recombination levels and interval mapping of multiple traits in the set of 1316 genotyped AILs.</p></div><div id="sec-13" class="subsection"><h3>Defining haplotypes and estimating founder probabilities</h3><p id="p-23">This study was interested in estimating the probability that a given region derives from each founder, as they can be used for defining haplotypes and for QTL mapping. In estimating these probabilities, several sets of SNP markers were used, with the attempt to saturate the genome with markers spaced no closer than 0.1 cM. Larger sets of SNPs are based on less stringent filtering criteria, hence, are of lower quality. In this study, six sets of SNP markers, which were derived using different criteria (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), were tested.</p><p id="p-24">For interval mapping, the map distance had to be represented in centimorgans, hence we used a linkage map derived from the physical map based on the conversion factor of 1 cM = 250 kb. The estimation of the equivalent centimorgan to bases was made using information from published high-quality, high-density genetic IRMI (accession ID irmi-2003, <a href="http://archive.gramene.org/db/markers/marker_view?action=view_map_set&amp;map_set_acc=irmi-2003">http://archive.gramene.org/db/markers/marker_view?action=view_map_set&amp;map_set_acc=irmi-2003</a>) and physical IRGSP map (available in Gramene as accession ID wig2005b; <a href="http://archive.gramene.org/db/cmap/map_set_info?map_set_acc=wig2005b">http://archive.gramene.org/db/cmap/map_set_info?map_set_acc=wig2005b</a>) of SSRs in the Nipponbare rice genome. SSRs in common between the genetic and physical maps (1224 in all) were determined, the respective coordinates (in centimorgan and base pair positions) tabulated, and the distances between SSRs (in centimorgan and base pairs) were computed for the entire genome. The average base distanceâcentimorgan ratio was computed between adjacent markers, per chromosome, and for the entire genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FileS3.xlsx">File S3</a>), resulting in the genome-wide ratio (â¼250 kb/cM) as the kilobaseâcentimorgan estimator.</p><p id="p-25">Founder probabilities were estimated at all markers using the âmpprobâ function in R/mpMap (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Huang and George 2011</a>) with program = qtl. The number of recombinations was then calculated at all marker positions (spaced &gt; 0.1 cM apart) using a forwardâbackward dynamic programming algorithm with a penalty of five for switching between founders (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FileS4.R">File S4</a>). The penalty acts as a disincentive to call recombination breakpoints; at its lowest values, we take the most probable founder at each genomic location, while as the penalty becomes large we expect the number of recombinations to decrease.</p></div><div id="sec-14" class="subsection"><h3>Mapping</h3><div id="sec-15" class="subsection"><h4>Genome-wide association mapping (GWAS):</h4><p id="p-26">GWAS mapping was carried out using R/GAPIT (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lipka <em>et al.</em> 2012</a>). The compressed mixed linear model (MLM) method was applied for detecting QTL associated with the trait (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Zhang <em>et al.</em> 2010</a>). A filtered set of 27,041 markers (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) across the 1316 MI lines was used for analysis. A kinship matrix based on the marker data was generated within the analysis. The default criteria implemented in GAPIT was used, with a significance threshold of 0.0001.</p></div><div id="sec-16" class="subsection"><h4>Interval mapping:</h4><p id="p-27">Interval mapping was conducted using the function âmpIMâ from R/mpMap. Simple interval mapping (SIM) was performed using BLUPs as response for the majority of the traits. The significance threshold for SIM was set to p-value &lt; 0.0001. Next, the function âfitâ was used to simultaneously estimate the effects of all QTL by fitting all the detected QTL in a single model. Note that this means that some QTL are no longer significant once all are included simultaneously. In this study, two separate final models, one based on percent variance and one based on p-value, were considered. The function âqindexâ was used to select QTL based on two criteria: (a) percent variance &gt; 2 and p-value &lt; 0.05, and (b) only p-value &lt; 0.0001 to include for fitting the final model. LD was estimated between flanking markers for QTL using the function âmpcalcldâ in R/mpMap (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Huang and George 2011</a>). This uses a multi-allelic measure for LD as described in <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Huang <em>et al.</em> (2012)</a>.</p></div></div><div id="sec-17" class="subsection"><h3>Data availability</h3><p id="p-28">The authors state that all raw data pertaining to the experiments has been submitted to the journal. A guide to supplemental files and raw data submitted to the journal can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FileS1.docx">File S1</a>. Raw genotype data can be found in <a href="http://snpseek.irri.org/_download.zul">http://snpseek.irri.org/_download.zul</a>. IRB is not applicable in our case. Downloadable files are available at URLs: Hapmap data (<a href="https://s3-ap-southeast-1.amazonaws.com/oryzasnp-atcg-irri-org/pub-data/MAGIC-Raw-genotype-data-Raghavan-2017.zip">https://s3-ap-southeast-1.amazonaws.com/oryzasnp-atcg-irri-org/pub-data/MAGIC-Raw-genotype-data-Raghavan-2017.zip</a>) and VCF data (<a href="https://s3-ap-southeast-1.amazonaws.com/oryzasnp-atcg-irri-org/pub-data/MAGIC-vcf-all-chromosomes.zip">https://s3-ap-southeast-1.amazonaws.com/oryzasnp-atcg-irri-org/pub-data/MAGIC-vcf-all-chromosomes.zip</a>).</p></div></div><div class="section results-discussion" id="sec-18"><h2 class="">Results and Discussion</h2><div id="sec-19" class="subsection"><h3>GBS data filtering</h3><p id="p-29">The postprocessing of GBS data was considered in detail given its tendency for genotyping errors. The SNP data of each founder replicate was compared against its 14 Ã sequence data in the 3 K panel (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alexandrov <em>et al.</em> 2014</a>). On average, the GBS calls and 14 Ã calls differed by 1.7%. This information was used to assume less than a 2% error in genotyping. The GBS raw data, those which were run through the standard GBS pipeline TASSEL 3, were used to generate SNP calls that served as the primary GBS dataset (as compared to the filtered datasets from postprocessing). The filtering of SNPs significantly reduced the original number of SNPs. Founder filtering resulted in three sets of SNPs (46,827, 68,826, and 88,083), MAGIC line filtering resulted in a further reduction of SNP markers (21,521, 24,800, and 33,056), and finally the merging of founders and lines resulted in three sets of common markers (16,702, 20,349, and 27,101), as shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>. SNP data were also extracted from lines corresponding to markers filtered in the founders only (46,827, 68,826, and 88,083) (see dotted line in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). The six datasets derived consisted of 16,702, 20,349, 27,101, 46,827, 68,826, and 88,083 markers. These filtering criteria will allow researchers to select an appropriate set of SNPs, keeping in mind that the stringency of criteria will provide a more accurate but smaller set of SNPs. Hence, the choice of thresholds will depend on the goals for analysis, as interval mapping may require fewer markers than fine-scale association mapping.</p></div><div id="sec-20" class="subsection"><h3>Imputation and marker spacing</h3><p id="p-30">To regain some of the information lost through filtering, the missing data for the founders and AILs were imputed. The function âmpimputeâ (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Huang <em>et al.</em> 2014</a>) has been shown to have nearly 100% accuracy in founder imputation and &gt; 90% imputation accuracy for a range of sample sizes, levels of missing data, and marker densities. At postimputation, SNPs with MAF &lt; 5% in the AILs and with a minimum intermarker distance of 0.1 cM were filtered out.</p><p id="p-31">As the filtering stringency decreased, the number of marker data points increased. In the first three data sets (16,702, 20,349, and 27,101), the marker data of lines were filtered for missing data and MAF at the first stage prior to imputation, while no filtering was applied to the line marker data in the following three datasets (46,827, 68,826, and 88,083) (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-32">In order to select a dataset for mapping, an intermediate filtering criterion was used. The largest set of markers was selected for lines where filtering was applied prior to imputing (33,056) and was combined with the largest dataset for founders (88,083), resulting in 27,101 markers common to lines and founders. It was noted that missing data were high because the replicated founder data (missing â¤ 6) were used for the first stage of filtering. Imputation of missing line and founder marker data followed by filtering for MAF &lt; 0.05 resulted in complete data (no missing data points) for 27,041 markers, which were used for GWAS. To extract markers for interval mapping, the 27,041 markers were binned such that no two markers were closer than 0.1 cM or 25,000 bp, which resulted in 6166 markers. All interval mapping performed in this study was conducted on this set of 6166 markers. On average, the intermarker distance was 0.24 cM; 66% of the markers had an intermarker distance of &lt; 0.2 cM and 98% of them were within 0.65 cM.</p></div><div id="sec-21" class="subsection"><h3>Founder probabilities and levels of recombination</h3><p id="p-33">Founder probabilities and levels of recombination were estimated using the imputed marker sets that were filtered for MAF (4800, 5486, 6166, 7391, 8445, and 9283) (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>). As the number of markers increases, the chance of observing genotyping errors and, hence, overestimating the number of recombinations estimated also increases. This trend is apparent at varying marker levels across the filtered data sets. To counteract the trend, this study introduced a penalty for shifting between founders and considered the effect of varying levels of penalty on recombination estimates (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). By filtering on the minimum distance between markers (mindist = 0.1 cM), the number of markers and consequently the number of estimated recombinations were reduced. It is important to note that the marker data are not uniformly dense since the maximum distance was not limited.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1721/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graph showing the trend in average number of observed recombinations per line of indica MAGIC (MI) population. The estimates are made at varying numbers of markers (x-axis) and penalty (Pen) levels. The x-axis indicates the number of single nucleotide polymorphism (SNP) markers used for estimating recombinations/marker sets (see Figure 1). Recombinations were estimated at a genotyping error probability of 0.3. AIL, advanced intercross lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-835566639" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Graph showing the trend in average number of observed recombinations per line of &lt;em&gt;indica&lt;/em&gt; MAGIC (MI) population. The estimates are made at varying numbers of markers (&lt;em&gt;x&lt;/em&gt;-axis) and penalty (Pen) levels. The &lt;em&gt;x&lt;/em&gt;-axis indicates the number of single nucleotide polymorphism (SNP) markers used for estimating recombinations/marker sets (see Figure 1). Recombinations were estimated at a genotyping error probability of 0.3. AIL, advanced intercross lines.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1721/F2.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/6/1721/F2.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1721/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1721/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484899" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">Graph showing the trend in average number of observed recombinations per line of <em>indica</em> MAGIC (MI) population. The estimates are made at varying numbers of markers (<em>x</em>-axis) and penalty (Pen) levels. The <em>x</em>-axis indicates the number of single nucleotide polymorphism (SNP) markers used for estimating recombinations/marker sets (see <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>). Recombinations were estimated at a genotyping error probability of 0.3. AIL, advanced intercross lines.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Between 63 and 71 recombinations were observed per AIL at the lowest penalty (3) and a probability of genotyping error of 0.03 using 6166 and 7391 markers, respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Also, the number of recombinations increased with the number of markers, and increasing the penalty resulted in fewer estimated recombinations. At all the penalty levels tested, the number of recombinations did not change much as the number of markers increased from 4800 to 6166, but a steeper increase in the following three marker sets (7391, 8445, and 9283) was observed. In this study, as the penalty increased, the slope decreased with increasing marker numbers. Ideally, one might expect the number of recombinations to stabilize as the genome is saturated with markers. As the least stringent criteria produced a set of 9283 markers, there is still scope for greater saturation. In comparison to these results, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Huang <em>et al.</em> (2009)</a> reported 33.8 recombination breakpoints per RIL derived from a cross between <em>Oryza sativa</em> 93-11 (<em>indica</em>) and Nipponbare (<em>japonica</em>).</p><p id="p-36">This study found slightly more recombinations than the simulations reported by <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Yamamoto <em>et al.</em> (2014)</a>, though there are a number of possible reasons for this difference. To saturate the genome (373,245,519 bp) (<a href="http://rice.plantbiology.msu.edu/annotation_pseudo_current.shtml">http://rice.plantbiology.msu.edu/annotation_pseudo_current.shtml</a>) with markers every 0.1 cM, â¼15,000 markers would be needed (assuming 0.1 cM equals â¼25,000 bp). With the current dataset, we were able to extract a maximum of 9283 markers. Simulation experiments (population size <em>n</em> = 800; genome size 1526.8 cM; bin 0.1 cM; cycles 0) reported â¼150 genome segments per individual (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Yamamoto <em>et al.</em> 2014</a>).</p><p id="p-37">Due to the increase in the number of recombinations observed beyond 7391 markers, we decided to select 6166 markers for mapping. This choice of core set was also based on the intermediate filtering criteria, and the fact that no SNP data were missing in the founders for these markers. While it would be interesting to see the effect of filtering and imputation on mapping, it would be best to investigate this in simulated data.</p><p id="p-38">Some of the issues we have seen with genotype errors, and increases in recombination breakpoints with additional markers, may be due to our use of a linkage map based directly on the physical map. Estimation of recombination (centimorgan distance) from physical genome size is best done in a proper mapping population, as the values are dependent on the genomes of the parents being used in the cross. However, comparative <em>O. sativa</em> genome sequencing publications (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Yu <em>et al.</em> 2005</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Schatz <em>et al.</em> 2014</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Sakai <em>et al.</em> 2014</a>), show that various rice cultivars have very high genome similarity (and chromosome colinearity) across major subpopulations (&gt; 95%), which gives us confidence in using a single estimation of recombination for <em>O. sativa</em>. The availability of a high-quality genome sequence in rice, and high-density genetic maps using sequence-based markers (SSR), have enabled a reasonable estimation of the physical-to-centimorgan distance conversion in rice. This process will be least appropriate in regions where the MI founders differ from the reference, and due caution should be exercised for any QTL that appear in such regions.</p><p id="p-39">In general, it must be kept in mind that the choice of filtering thresholds will influence downstream analysis. It is clear that larger populations and dense markers are required to detect small-effect QTL or recombinations, but this also entails a higher cost in genotyping. Lower-cost genotyping technologies, while producing larger numbers of markers, may be prone to more errors and, hence, will require more stringent filtering, which in turn reduces the number of markers. Statistical methods such as applying penalties and marker imputation as discussed above may correct this to some extent, but these must be used with caution.</p></div><div id="sec-22" class="subsection"><h3>Phenotype analysis and QTL mapping</h3><p id="p-40">The summary statistics of the traits in the population and founders are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS1.xlsx">Table S1</a>. Generally, in cases when there were contrasting alleles within the founders for traits, transgressive segregation was observed. For most traits, we observe a continuous range of values among founders, except in case submergence tolerance where only Sambha Mahsuri-Sub1 is tolerant.</p><p id="p-41">The imputed and filtered marker data were used for QTL mapping, with a linkage map generated from the physical map of the Nipponbare reference genome. The sequence length in the pseudomolecule release 7 (MSUv7) is 373,245,519 bp. This roughly converts to 1492 cM, which is comparable to map lengths reported earlier (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Harushima <em>et al.</em> 1998</a>). Similar procedures for interval mapping were followed for all eight traits considered, with BLUPs taken as response in the genetic model. We note that this may not be ideal in the case of low heritability traits, but our traits all have medium to high heritability. We report peak widths for both interval and association mapping, but note that these are not exactly comparable, since for association mapping we take the peak to include all markers significantly associated with the trait, whereas for interval mapping this is based on a 1 LOD drop-off. The average 1 LOD support interval of the QTL reported in this study is 2.83 cM (0.707 Mbp), the interval between flanking markers is 0.29 cM (0.072 Mbp), and the LD (<em>r</em><sup>2</sup>) between flanking markers is 0.85 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS3.xlsx">Table S3</a>).</p><div id="sec-23" class="subsection"><h4>Yield:</h4><p id="p-42">The BLUPs of yield for the founders ranged between 5.4 and 6.4 tons per hectare with Fedearroz 50, IR77186-122-2-2-3 (PSBRc 158), and IR64633-87-2-2-3-3 (PSBRc82) being the top three yielders. The BLUPs of yield for the MAGIC lines ranged from 4.4 to 8.6 tons per hectare, with nearly 20% (286 of the 1316) of the MI entries having greater yield than the founders. The high-yielding checks NSCIRc 222 (IRRI 154) and NSIC Rc132H (Mestizo 6, hybrid variety) yielded 6.7 and 7.9 tons per hectare, respectively, in the conducted trials. The heritability for this study was 54%. SIM detected four QTL on chromosomes 2, 3, 7, and 8. The phenotypic variance explained by the final model with percent variance &gt; 2% and p-value &lt; 0.05 was 10.13%, supported by a QTL on chromosome 3 (4.19 cM; 1.27 Mbp) and another QTL on chromosome 8 (104.62 cM; 26.2 Mbp) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FigureS1.docx">Figure S1</a>). The phenotypic variance of the QTL on chromosome 3 was 7.68% and that on chromosome 8 was 3.17%. The phenotypic variance explained by the final model with p-values &lt; 0.0001 was 10.13%, and was also supported by the two QTL on chromosomes 3 (4.19 cM; 1.27 Mbp) and 8 (104.62 cM; 26.2 Mbp). The results of SIM for 2014 DS or 2015 DS also detected these QTL on chromosomes 3 and 8.</p><p id="p-43">Association mapping detected significant associations on chromosomes 3 and 8. The SNP markers on chromosome 3 at positions 1,270,943 and 1,343,246 bp were the top two hits with p-values 1.74Eâ14 and 3.09Eâ14, respectively, and with <em>R</em><sup>2</sup> equal to 4.9%. On chromosome 8, one marker on position 26,573,952 bp was detected to be significant at p-value 1.14Eâ05 and with an <em>R</em><sup>2</sup> of 4.9%.</p><p id="p-44">The QTL on chromosome 3 (4.19 cM; 1.27 Mbp) was also detected as being significant for flowering time (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>). The QTL on chromosome 8 colocalized with the QTL that was reported to be linked to grain weight per plant and source activity (source QTARO: <a href="http://qtaro.abr.affrc.go.jp">http://qtaro.abr.affrc.go.jp</a>) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Yamamoto <em>et al.</em> 2012</a>). In this study, small-effect QTL for yield were detected, which will be validated. SIM also detected significant QTL on chromosomes 2 and 7 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>) that colocalized with the QTL for plant height qPH-2 (QTARO) and the QTL for 1000-grain weight <em>gw7</em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li <em>et al.</em> 2000</a>), respectively. Since yield is a complex trait, it may be a better approach to study the markerâmarker and markerâtrait interactions rather than to validate each small-effect QTL independently.</p></div><div id="sec-24" class="subsection"><h4>Flowering time:</h4><p id="p-45">The flowering time of the eight founders ranged between 85 and 101 d, with IR77298-14-1-2-10 flowering the earliest and Sambha Mahsuri-Sub1 flowering the latest. Flowering in the MAGIC lines ranged from 84 to 106 d. The heritability for flowering time was 49%. Eight significant QTL were detected using the SIM approach (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>). The final model with percent variance &gt; 2 and p-value &lt; 0.05 detected four QTL on chromosomes 1, 3, 6, and 7 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FigureS2.docx">Figure S2</a>), and explained 22.36% of the variance. The QTL on chromosome 3 (4.19 cM; 1.27 Mbp) accounted for 17.51% of the phenotypic variance. The final model with p-value &lt; 0.0001 detected QTL on chromosomes 3, 5, and 6, explaining 22.97% of the variance.</p><p id="p-46">Association mapping detected 73 markers associated with flowering time on chromosome 3. The SNP markers on chromosome 3 at positions 1,343,246 and 1,270,943 bp (which were also flanking markers detected by interval mapping) were the most significant markers with p-values of 3.57Eâ32 and 4.34Eâ32, respectively, and an <em>R</em><sup>2</sup> of 12.9%. The QTL detected on chromosome 3 (between 1,270,943 and 1,343,246 bp) colocalized with the <em>Hd9</em> (heading date) and <em>dth3</em> (days to heading/drought tolerance) QTL (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lin <em>et al.</em> 2002</a>). Association mapping also detected three markers (p-value &lt; 0.003) between â¼31.9 and 31.97 Mbp on chromosome 1, which are close to the QTL detected at 127.84 cM (31.97 Mbp). The marker at position 31,971,239 bp was the most significant and is also a flanking marker reported by SIM (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>).</p><p id="p-47">The QTL on chromosome 1 detected by SIM (153.1 cM; 38.29 Mbp) colocalized with the QTL reported for panicle/flower morphology and development as well as for plant height (<em>sd1</em>) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">McNally <em>et al.</em> 2009</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>). In this study, SIM detected a second QTL on chromosome 1 (127.84 cM between 31.9 and 31.97 Mbp) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>) which colocalized with a flower morphology QTL. The two QTL on chromosome 1 are separated by 25 cM (6.31 Mbp). The QTL on chromosomes 6 and 7 colocalized with the days-to-heading QTL <em>Hd3b</em> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Yano <em>et al.</em> 2001</a>) and <em>Hd7</em>, respectively (QTARO). A comparison of QTL for yield and QTL for flowering time shows the QTL in proximity (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS3.xlsx">Table S3</a>), suggesting possible correlations between the traits.</p></div><div id="sec-25" class="subsection"><h4>Plant height:</h4><p id="p-48">In this trial, the plant height of the MAGIC founders ranged from 86 (IR64633-87-2-2-3-3 (PSBRc82) to 112 cm (IR 4630-22-2-5-1-3), while the height of the lines ranged from 73 to 136 cm. The heritability for plant height was 70%. SIM detected 15 QTL (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>). The final model with percent variance &gt; 2% and p-value &lt; 0.05 was explained by the QTL on chromosomes 1, 3, 4, 5, 6, 7, 11, and 12 (shown in bold in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>) accounting for 26.19% of the variance. The final model with p-value &lt; 0.0001 explained 23% of the phenotypic variance, due to the QTL on chromosomes 1, 3, 5, and 11. The QTL on chromosome 1 (153.71 cM; 38.44 Mbp) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FigureS3.docx">Figure S3</a>) explained 9.8% of the variance and colocalized with <em>sd1</em> (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">McNally <em>et al.</em> 2009</a>). It should be noted that a second QTL was detected on chromosome 1 by SIM at 38.16 cM (9.55 Mbp).</p><p id="p-49">GWAS detected 61 markers on chromosome 1, including flanking markers from the interval mapping between 35 and 41 Mbp, to be significantly associated with an <em>R</em><sup>2</sup> of 17â18%. The second largest QTL, which explained 8.5% of the variance, was detected on chromosome 3 at 4.68 cM, â¼0.5 cM (0.12 Mbp) from the yield/flowering QTL. This QTL on chromosome 3 was also detected by GWAS with 17 markers significantly associated, although they were spread over 21 Mbp. There are possible interactions between the loci controlling yield, plant height, and flowering time. The QTL on chromosome 6 (56.83 cM; 14.35 Mbp) colocalized with a previously reported plant height QTL <em>qPH2-6-1</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cui <em>et al.</em> 2004</a>), which has also been indicated in an interaction between tiller number, plant height, and heading date.</p></div><div id="sec-26" class="subsection"><h4>Grain length:</h4><p id="p-50">The grain length of the founders ranged from 5.09 to 6.97 mm with IR77298-14-1-2-10, IR64633-87-2-2-3-3 (PSBRc82), and IR77186-122-2-2-3 (PSBRc 158) having long grains (6.6â7.00 mm). Grain length in the examined MAGIC population ranged from 5.1 to 7.3 mm. SIM detected 10 QTL. The final model with percent variance &gt; 2 and p-value &lt; 0.05 explained 49.05% of the variance, with the QTL on chromosomes 3 (65.68 cM, 16.64 Mbp) and 7 (98.73 cM; 24.70 Mbp) accounting for 31.51 and 13.66% of the phenotypic variance, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>). The QTL on chromosomes 3 and 7 were also fit by the final model with p-value &lt; 0.0001 and explained 46.73% of the variance (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FigureS4.docx">Figure S4</a>).</p><p id="p-51">The QTL on chromosome 3 was also detected by association mapping. There were 57 significant markers between 16.2 and 21 Mbp. This was also previously reported in an earlier study on a subset of the MI early generation (S4) population (<a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Bandillo <em>et al.</em> 2013</a>). This QTL is known to colocalize with the QTL for grain length (<em>qGL-3</em>), grain width (gw3.1), and length by width ratio (<em>qLWR-3</em>) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Yu <em>et al.</em> 2011</a>) (source QTARO). It is also located near the grain size QTL <em>GS3</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fan <em>et al.</em> 2006</a>). In the larger (1316 lines) S6:8 population, 53 markers at â¼24 Mbp on chromosome 7 were detected to be associated with grain length. This QTL on chromosome 7 has been previously reported to be linked to the grain width QTL <em>grb7-2</em> (source QTARO) or grain size (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Shao <em>et al.</em> 2012</a>).</p></div><div id="sec-27" class="subsection"><h4>Grain width:</h4><p id="p-52">The grain width of the founders ranged from slender (1.85 mm) to broad (2.5 mm). Sambha Mahsuri-Sub1 and IR77186-122-2-2-3 (PSBRc 158) both have slender grains compared to the broad grains of IR4630-22-2-5-1-3 and IR45427-2B-2-2B-1-1. In the MI population, grain width ranged from 1.7 to 2.8 mm. SIM detected 29 QTL. The final model with percent variance &gt; 2 and p-value &lt; 0.05 explained 57.18% of the variance based on 19 QTL (shown in bold in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>). Seven QTL on chromosomes 1, 2, 3, 5, 7, and 8 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>) were detected by the final model with p-value &lt; 0.0001 and accounted for 47.08% of the phenotypic variance.</p><p id="p-53">Association mapping detected 39 markers (4.3â5.6 Mbp; <em>R</em><sup>2</sup> 46.5%) on chromosome 5 that are associated with grain width. The marker at 5,391,586 bp, which is also the right flanking marker (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>) on chromosome 5, was the most significant. The QTL on chromosome 5 (4.9â5.4 Mbp) colocalized with <em>qGW-5</em> (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Weng <em>et al.</em> 2008</a>). GWAS also detected 49 markers on chromosome 7 (24.4â24.8 Mbp) to be associated with grain width. The QTL on chromosome 7 (97.6 cM; 24.42 Mbp) colocalized with the grain width QTL <em>grb7-2</em> (source QTARO) or grain size (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Shao <em>et al.</em> 2012</a>). This QTL explained the most phenotypic variance (16.58%). It is situated 1.13 cM (0.28 Mbp) away from the QTL detected for grain length in this study (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>). The QTL on chromosome 8 (105.95 cM; 26.54 Mbp) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>) was detected by both interval and association mapping (26.3â27.3 Mbp) approaches and colocalized with the grain width and yield potential QTL <em>GW8</em> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2012</a>). The QTL on chromosomes 5, 7, and 8 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FigureS5.docx">Figure S5</a>) have also been previously reported in the MI population (early generation S4) (<a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Bandillo <em>et al.</em> 2013</a>).</p><p id="p-54">SIM detected a fourth significant QTL on chromosome 3 at 97.21 cM (interval 24.48 and 24.53 Mbp), which colocalized with the grain length or width-related QTL (<em>qGL-3</em>, <em>gw3.1</em>, and <em>qLWR-3</em>) and with the QTL linked to grain weight (<em>gw3a</em>) and yield (<em>Yd3-13</em>) (source QTARO). GWAS (16.7â16.8 Mbp) and SIM detected a QTL for grain width on chromosome 3 at 64.33 cM (16.3 Mbp) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>) which is 1.35 cM (0.34 Mbp) away from the QTL for grain length (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>) detected in this study. The locations of the QTL on chromosome 3 for yield, flowering time, plant height, and grain length and width suggest possible links among these traits.</p></div><div id="sec-28" class="subsection"><h4>Amylose content:</h4><p id="p-55">Amylose content is a key cooking quality trait that determines consumer preference. The amylose content (percent by weight) in grains of the founders fell into a narrow intermediate type category between 19.8 and 24.6%. Amylose content in the rice grains of the MAGIC population ranged from 11 to 25%. SIM detected 12 QTL with the most significant QTL found on chromosome 6 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>). The final model with percent variance &gt; 2 and p-value &lt; 0.05 included five QTL (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>), explaining 53.62% of the phenotypic variance. Only one large QTL on chromosome 6 was fitted by the final model with p-value &lt; 0.0001, accounting for 49.92% of the phenotypic variance. The QTL on chromosome 6 which localized within the <em>waxy/qAC-6</em> locus (1.69â1.72 Mbp) alone accounted for 49.92% of the phenotypic variance (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FigureS6.docx">Figure S6</a>).</p><p id="p-56">Association mapping detected 123 significant markers on chromosome 6 between 0.36 and 2.8 Mbp, with a marker at 1,760,469 bp being the most significant. It is noted that this QTL (chromosome 6: 6.33 cM; 1.72 Mbp) was closely located to a QTL for flowering time (chromosome 6: 7.75 cM; 2.07 Mbp) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>). A second QTL on chromosome 6 (68.99 cM; 17.39 Mbp) detected by SIM colocalized with an amylose content QTL <em>amy6-1</em>, which is located in proximity to a plant height QTL (56. 83 cM; 14.35 Mbp) detected in this study. The QTL for yield, flowering time, and plant height have also been reported at this locus (source QTARO). Although several other (besides the ones detected on chromosome 6) QTL were detected in this study, only those on chromosomes 1 (154.39 cM; 38.61 Mbp) and 5 (detected only by SIM at 91.51 cM; 22.91 Mbp) colocalized with previously detected QTL associated with eating quality.</p></div><div id="sec-29" class="subsection"><h4>Submergence tolerance:</h4><p id="p-57">Sambha Mahsuri-Sub1 was the only tolerant founder used. The tolerant checks Swarna-Sub1 and IR64-Sub1 were used alongside the susceptible checks Swarna, IR64, and IR42 in the submergence trials. Only â¼7% of the lines were tolerant to 16 d of flooding (<em>i.e.</em>, survived 7 d after draining). SIM detected seven QTL on chromosomes 1, 3, 5, 7, and 9. The final model with percent variance &gt; 2 and p-value &lt; 0.05 detected QTL on chromosomes 7 and 9 and explained 23.97% of the variance, with the largest QTL being the <em>Sub1</em> QTL (22.44%) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>). The final model with p-value &lt; 0.0001 detected one large QTL on chromosome 9 at 27.15 cM (7.08 Mbp). The major <em>Sub1</em> QTL (chromosome 9: 27.15 cM; 7.08 Mbp) was mapped (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FigureS7.docx">Figure S7</a>) within a 1 LOD support interval 0.69 cM (0.17 Mbp) and 31.9 kb between markers at 7,045,612 and 7,077,542 bp. The QTL <em>SUB1A</em> is linked to culm/shoot growth trait (Source QTARO). A second QTL on chromosome 9 (81.94 cM; â¼20.7 Mbp), which colocalized with the QTL for flowering (Os<em>RRMh</em>) and drought tolerance/root length (<em>mrl9a</em>) (Source QTARO), was also detected by SIM in this study. The QTL detected by SIM on chromosome 1 (between 2,349,219 and 2,502,008 bp) colocalizes with <em>qSUB1.1</em> recently reported in a FR13A by IR 42 RIL (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gonzaga <em>et al.</em> 2016</a>).</p><p id="p-58">Association mapping detected 100 significantly associated markers, with the top 10 markers located between 6.1 and 6.19 Mbp on chromosome 9. The QTL on chromosome 7 (82.03 cM; 20.5â20.8 Mbp) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS2.xlsx">Table S2</a>) colocalized with the days-to-heading QTL <em>dth7.1</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gao <em>et al.</em> 2014</a>) (Source QTARO). <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Toojinda <em>et al.</em> (2003)</a> reported a QTL for submergence tolerance on chromosome 7 (19,256,914â19,257,039 bp). It was noted in this study that a small proportion of lines are submergence tolerant, which may be due to interactions between several QTL and the major <em>Sub1</em> QTL detected in this study. The submergence tolerance mechanism is associated with suppressed shoot growth and delayed flowering (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Septiningsih <em>et al.</em> 2013</a>). The MAGIC population would be ideal to understand the interaction between the detected QTL and to identify lines with different combinations of these QTL.</p></div><div id="sec-30" class="subsection"><h4>Brown spot:</h4><p id="p-59">The parents exhibited variation in resistance to sm2, an isolate of <em>B. oryzae</em>. IR77298-14-1-2-10 was the most susceptible while Sanhuangzhan-2, Sambha Mahsuri-Sub1, and IR4630-22-2-5-1-3 were resistant. The BLUPS in the AILs ranged from 7 to 30 (resistant to susceptible). The heritability for brown spot was 40%. SIM detected a large QTL on chromosome 12 (76.71 cM; 19.30 Mbp) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/FigureS8.docx">Figure S8</a>) and two others on chromosomes 4 and 8. Only the QTL on chromosome 12 was fitted by both the final model with percent variance &gt; 2 and p-value &lt; 0.05 and also the model with p-value &lt; 0.0001. The phenotypic variance accounted for by both the final models was 34.42%.</p><p id="p-60">GWAS mapping also detected the QTL on chromosomes 12 (19.3 Mbp) as highly significant (p-value = 3.62Eâ25). This QTL has been validated independently in other studies (biparental and 2 k diversity panel) at IRRI. The QTL interval (chromosome 12: 19,301,158â19,339,344 bp) colocalized with <em>qDLA-12-3</em> (QTARO), a QTL for leaf blast resistance. A second QTL on chromosome 12 was detected by SIM at 39.83 cM (10.08 Mbp), which colocalized with <em>qDLA-12-3</em> and is near the meta QTL yield under drought stress (MQTL) dty<sub>12.1</sub> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Swamy <em>et al.</em> 2011</a>). The QTL on chromosome 8 also detected by SIM colocalized with a drought tolerance QTL and a flowering QTL <em>qDTH8</em> (QTARO). Since brown spot disease is often observed in regions where there is water/soil stress, the QTL on chromosome 8 might be interesting to further investigate.</p></div></div><div id="sec-31" class="subsection"><h3>Conclusions</h3><p id="p-61">This paper reports on analytical methods applied to understand the genetics of the MI population in rice. It shows how GBS data can be used to generate large numbers of SNP markers as well as the importance of: (a) filtering criteria, (b) imputation of missing data, and (c) accounting for genotyping errors in downstream analysis. We have shown how filtering criteria affect estimated recombination numbers. Datasets derived from low filtering stringencies have high recombination numbers based on theoretical assumptions. These high levels could possibly be due to genotyping errors, which we accounted for in the analysis. Replicating founders for sequencing is useful to ensure high confidence of SNP calls. An alternative is to have deep sequence data for the founders, which may be used to impute SNP marker calls in lines to improve coverage. When dealing with large population sizes, GBS-generated data are the most cost-effective choice, but the savings in genotyping costs must be balanced with increased informatics requirements.</p><p id="p-62">Comparing significant QTL detected from both GWAS and interval mapping approaches were generally consistent for the detection of major genes or large-effect QTL. However, the methods differed for smaller-effect QTL, so using both approaches in combination may provide more strength of evidence for genomic regions to follow up in further validation experiments. Both new QTL and major QTL reported from independent studies were detected. Some QTL for multiple traits are within 2 cM of each other (highlighted in red in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1/TableS3.xlsx">Table S3</a>). However, in some of these cases, for example the QTL on chromosome 3 (4.19 cM; 1.27 Mbp) that is linked to both yield and flowering, the allelic effects of the founders are not alike. Further dissection of such loci, by characterizing MAGIC lines with varying allelic combinations, will help to clarify whether there are multiple QTL in this region or a single pleiotropic one. All results from our mapping effort will be moved forward for validation and marker development. The SNPseek database (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Alexandrov <em>et al.</em> 2014</a>) will be used to extract haplotypes in the region of interest.</p><p id="p-63">This work aims to capitalize on the primary advantage of MAGIC populations, namely that they combine favorable alleles from several different founders for multiple traits. For geneticists, this is an ideal situation to study QTL for multiple traits using one mapping population, although increases in the number of traits and QTL rapidly add to the complexity of analysis. A systematic coordinated effort in phenotyping is critical to optimally exploit the MAGIC population. However, phenotyping and data maintenance efforts will demand time and funds. Maximizing data collected from a single trial would be an effective approach by coordinating data collection for multiple traits among phenotyping groups.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-64">We acknowledge support provided by the Genotyping Services Laboratory (Michael Thomson and Christine Jade Dilla Ermita), the International Rice Research Institute (IRRI) for DNA extraction, the Genomic Diversity Facility, the Biotechnology Resource Centre, Cornell University for genotyping by sequencing services, and the Philippine Genome Center, University of the Philippines, Philippines (Venice Margarette Juanillas) for running the genotyping by sequencing pipeline. We acknowledge the support provided for population development and phenotyping by IRRI plant breeding teams (Mae Rose Maoirat, Jerome Carandang, Edilberto RedoÃ±a, Glenn Gregorio, and Nonoy Bandillo), IRRI biometrics (Alaine Gulles and Violeta Bartolome), and the Grain Quality and Nutrition Centre at IRRI.</p><p id="p-65">Author contributions: C.R., H.L., and B.E.H. wrote the manuscript; H.L., R.K.S., M.J., V.L., H.Z., and C.R. developed the population and conducted phenotyping; and C.R., B.E.H., J.B., and R.M. conducted genetic analysis.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-66">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.042101/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-67"><em>Communicating editor: J. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-05"><span class="received-label">Received </span>January 5, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-04-03"><span class="accepted-label">Accepted </span>April 3, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2017 Raghavan <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-book" id="cit-7.6.1721.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">AACC</span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Approved methods of analysis. Method 61â03.01</span>, pp. <span class="cit-fpage">4</span>. <span class="cit-source"><em>Amylose Content of Milled Rice</em></span>. 9th ed. <span class="cit-publ-name">American Association of Cereal Chemists</span>, <span class="cit-publ-loc">St. Paul, MN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexandrov</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mansueto</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">SNP-Seek database of SNPs derived from 3000 rice genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D1023</span>â<span class="cit-lpage">D1027</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D43%26rft.spage%253DD1023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bandillo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muyco</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sevilla</span>  <span class="cit-name-given-names">M. A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobina</span>  <span class="cit-name-given-names">I. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Multi-parent advanced generation inter-cross (MAGIC) populations in rice: progress and potential for genetics research and breeding.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">11</span>â<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D6%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.4" data-doi="10.1534/genetics.112.147710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekhon</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foerster</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Marker density and read depth for genotyping populations using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1073</span>â<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147710%26rft_id%253Dinfo%253Apmid%252F23410831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzEwNzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTcyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Molecular dissection of the relationships among tiller number, plant height and heading date in rice.</span> <abbr class="cit-jnl-abbrev">Plant Prod. Sci.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">309</span>â<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BProd.%2BSci.%26rft.volume%253D7%26rft.spage%253D309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-book" id="cit-7.6.1721.6" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species (L. Orban, Ed.).</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.7" data-doi="10.1007/s00122-006-0218-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title"><em>GS3</em>, a major QTL for grain length and weight and minor QTL for grain width and thickness in rice, encodes a putative transmembrane protein.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">1164</span>â<span class="cit-lpage">1171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DFan%26rft.auinit1%253DC.%26rft.volume%253D112%26rft.issue%253D6%26rft.spage%253D1164%26rft.epage%253D1171%26rft.atitle%253DGS3%252C%2Ba%2Bmajor%2BQTL%2Bfor%2Bgrain%2Blength%2Band%2Bweight%2Band%2Bminor%2BQTL%2Bfor%2Bgrain%2Bwidth%2Band%2Bthickness%2Bin%2Brice%252C%2Bencodes%2Ba%2Bputative%2Btransmembrane%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0218-1%26rft_id%253Dinfo%253Apmid%252F16453132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0218-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16453132&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236486600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.8" data-doi="10.1073/pnas.1418204111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title"><em>Days to heading 7</em>, a major quantitative locus determining photoperiod sensitivity and regional adaptation in rice.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">16337</span>â<span class="cit-lpage">16342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1418204111%26rft_id%253Dinfo%253Apmid%252F25378698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzQ2LzE2MzM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.9" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harriman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzaga</span>  <span class="cit-name-given-names">Z. J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carandang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackill</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Septiningsih</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Mapping additional QTLs from FR13A to increase submergence tolerance in rice beyond <em>SUB1</em>.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">627</span>â<span class="cit-lpage">636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D209%26rft.spage%253D627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harushima</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shomura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">A high-density rice genetic linkage map with 2275 markers using a single F2 population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">479</span>â<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHarushima%26rft.auinit1%253DY.%26rft.volume%253D148%26rft.issue%253D1%26rft.spage%253D479%26rft.epage%253D494%26rft.atitle%253DA%2BHigh-Density%2BRice%2BGenetic%2BLinkage%2BMap%2Bwith%2B2275%2BMarkers%2BUsing%2Ba%2BSingle%2BF2%2BPopulation%26rft_id%253Dinfo%253Apmid%252F9475757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ4LzEvNDc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.12" data-doi="10.1093/bioinformatics/btq719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">R/mpMap: a computational platform for the genetic analysis of multiparent recombinant inbred lines.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">727</span>â<span class="cit-lpage">729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq719%26rft_id%253Dinfo%253Apmid%252F21217121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNy81LzcyNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.13" data-doi="10.1111/j.1467-7652.2012.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">A multiparent advanced generation inter-cross population for genetic analysis in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">826</span>â<span class="cit-lpage">839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DHuang%26rft.auinit1%253DB.%2BE.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D826%26rft.epage%253D839%26rft.atitle%253DA%2Bmultiparent%2Badvanced%2Bgeneration%2Binter-cross%2Bpopulation%2Bfor%2Bgenetic%2Banalysis%2Bin%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2012.00702.x%26rft_id%253Dinfo%253Apmid%252F22594629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2012.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22594629&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307051700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.14" data-doi="10.1534/genetics.113.158014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauleon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Efficient imputation of missing markers in low-coverage genotyping-by-sequencing data from multiparental crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">401</span>â<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.158014%26rft_id%253Dinfo%253Apmid%252F24583583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzEvNDAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.15" data-doi="10.1007/s00122-015-2506-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">MAGIC populations in crops: current status and future prospects.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">999</span>â<span class="cit-lpage">1017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D999%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2506-0%26rft_id%253Dinfo%253Apmid%252F25855139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2506-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25855139&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.16" data-doi="10.1101/gr.089516.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">High-throughput genotyping by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1068</span>â<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089516.108%26rft_id%253Dinfo%253Apmid%252F19420380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA2OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-book" id="cit-7.6.1721.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">IRRI (International Rice Research Institute)</span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-source"><em>Standard Evaluation System for Rice</em></span>, Ed. 4. <span class="cit-publ-name">IRRI</span>, <span class="cit-publ-loc">Manila, Philippines</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliva</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Allele mining and enhanced genetic recombination for rice breeding.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D8%26rft.spage%253D34%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.19" data-doi="10.1007/s001220051476"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">F. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Analyzing quantitative trait loci for yield using a vegetatively replicated F2 population from a cross between the parents of an elite rice hybrid.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">248</span>â<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D101%26rft.spage%253D248%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220051476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220051476&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000088403800036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.20" data-doi="10.1270/jsbbs.52.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashikari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamanouchi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Identification and characterization of a quantitative trait locus, <em>Hd9</em>, controlling heading date in rice.</span> <abbr class="cit-jnl-abbrev">Breed. Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">35</span>â<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBreed.%2BSci.%26rft.volume%253D52%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1270%252Fjsbbs.52.35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1270/jsbbs.52.35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.21" data-doi="10.1093/bioinformatics/bts444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">GAPIT: genome association and prediction integrated tool.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2397</span>â<span class="cit-lpage">2399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts444%26rft_id%253Dinfo%253Apmid%252F22796960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTgvMjM5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bansept-Basler</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockram</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">An eight-parent multiparent advanced generation inter-cross population for winter-sown wheat: creation, properties, and validation.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1603</span>â<span class="cit-lpage">1610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.23" data-doi="10.1073/pnas.0900992106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McNally</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Childs</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohnert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genomewide SNP variation reveals relationships among landraces and modern varieties of rice.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">12273</span>â<span class="cit-lpage">12278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0900992106%26rft_id%253Dinfo%253Apmid%252F19597147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzMwLzEyMjczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milner</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maccaferri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mantovani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">A multiparental cross population for mapping QTL for agronomic traits in durum wheat (<em>Triticum turgidum</em> ssp. <em>durum</em>).</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">735</span>â<span class="cit-lpage">748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D14%26rft.spage%253D735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pascual</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauvage</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duangjit</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchet</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Dissecting quantitative trait variation in the resequencing era: complementarity of bi-parental, multi-parental and association panels.</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">242</span>: <span class="cit-fpage">120</span>â<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D242%26rft.spage%253D120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.26" data-doi="10.1093/dnares/dsu006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanamori</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arai-Kichise</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shibata-Hatta</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebana</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Construction of pseudomolecule sequences of the aus rice cultivar Kasalath for comparative genomics of Asian cultivated rice.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">397</span>â<span class="cit-lpage">405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsu006%26rft_id%253Dinfo%253Apmid%252F24578372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIxLzQvMzk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.27" data-doi="10.1007/s11032-015-0202-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sannemann</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathew</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LÃ©on</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Multi-parent advanced generation inter-cross in barley: high-resolution quantitative trait locus mapping for flowering time as a proof of concept.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D35%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-015-0202-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-015-0202-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsumoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ota</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamakawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kihara</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Confirming a major QTL and finding additional loci responsible for field resistance to brown spot (<em>Bipolaris oryzae</em>) in rice.</span> <abbr class="cit-jnl-abbrev">Breed. Sci.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">170</span>â<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBreed.%2BSci.%26rft.volume%253D65%26rft.spage%253D170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maron</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wences</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurtowski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Whole genome de novo assemblies of three divergent strains of rice, <em>Oryza sativa</em>, document novel gene space of aus and indica.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D506%26rft_id%253Dinfo%253Apmid%252F25468217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25468217&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-book" id="cit-7.6.1721.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Septiningsih</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collard</span>  <span class="cit-name-given-names">B. C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heuer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey-Serres</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Applying genomics tools for breeding submergence tolerance in rice</span>, pp. <span class="cit-fpage">9</span>â<span class="cit-lpage">30</span> in <span class="cit-source"><em>Translational Genomics for Crop Breeding</em></span>. <span class="cit-publ-name">John Wiley &amp; Sons, Inc.</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.31" data-doi="10.1007/s00122-012-1914-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Allelic variation for a candidate gene for <em>GS7</em>, responsible for grain shape in rice.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1303</span>â<span class="cit-lpage">1312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DShao%26rft.auinit1%253DG.%26rft.volume%253D125%26rft.issue%253D6%26rft.spage%253D1303%26rft.epage%253D1312%26rft.atitle%253DAllelic%2Bvariation%2Bfor%2Ba%2Bcandidate%2Bgene%2Bfor%2BGS7%252C%2Bresponsible%2Bfor%2Bgrain%2Bshape%2Bin%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1914-7%26rft_id%253Dinfo%253Apmid%252F22772587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1914-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22772587&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.32" data-doi="10.1186/1471-2164-12-319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swamy</span>  <span class="cit-name-given-names">B. P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vikram</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahmed</span>  <span class="cit-name-given-names">H. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Meta-analysis of grain yield QTL identified during agricultural drought in grasses showed consensus.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMallikarjuna%2BSwamy%26rft.auinit1%253DB.%2BP.%26rft.volume%253D12%26rft.spage%253D319%26rft.epage%253D319%26rft.atitle%253DMeta-analysis%2Bof%2Bgrain%2Byield%2BQTL%2Bidentified%2Bduring%2Bagricultural%2Bdrought%2Bin%2Bgrasses%2Bshowed%2Bconsensus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-319%26rft_id%253Dinfo%253Apmid%252F21679437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21679437&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.33" data-doi="10.1093/aob/mcf072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toojinda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siangliw</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tragoonrung</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanavichit</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Molecular genetics of submergence tolerance in rice: QTL analysis of key traits.</span> <abbr class="cit-jnl-abbrev">Ann. Bot.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">243</span>â<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BBotany%26rft.stitle%253DAnn%2BBot%26rft.aulast%253DTOOJINDA%26rft.auinit1%253DT.%26rft.volume%253D91%26rft.issue%253D2%26rft.spage%253D243%26rft.epage%253D253%26rft.atitle%253DMolecular%2BGenetics%2Bof%2BSubmergence%2BTolerance%2Bin%2BRice%253A%2BQTL%2BAnalysis%2Bof%2BKey%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcf072%26rft_id%253Dinfo%253Apmid%252F12509344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjg6IjkxLzIvMjQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE3MjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.34" data-doi="10.1038/ng.2327"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Control of grain size, shape and quality by <em>OsSPL16</em> in rice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">950</span>â<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DS.%26rft.volume%253D44%26rft.issue%253D8%26rft.spage%253D950%26rft.epage%253D954%26rft.atitle%253DControl%2Bof%2Bgrain%2Bsize%252C%2Bshape%2Band%2Bquality%2Bby%2BOsSPL16%2Bin%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2327%26rft_id%253Dinfo%253Apmid%252F22729225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2327&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22729225&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.35" data-doi="10.1038/cr.2008.307"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Isolation and initial characterization of <em>GW5</em>, a major QTL associated with rice grain width and weight.</span> <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1199</span>â<span class="cit-lpage">1209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bresearch%26rft.stitle%253DCell%2BRes%26rft.aulast%253DWeng%26rft.auinit1%253DJ.%26rft.volume%253D18%26rft.issue%253D12%26rft.spage%253D1199%26rft.epage%253D1209%26rft.atitle%253DIsolation%2Band%2Binitial%2Bcharacterization%2Bof%2BGW5%252C%2Ba%2Bmajor%2BQTL%2Bassociated%2Bwith%2Brice%2Bgrain%2Bwidth%2Band%2Bweight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2008.307%26rft_id%253Dinfo%253Apmid%252F19015668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2008.307&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19015668&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261817900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yonemaru</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">OGRO: the overview of functionally characterized genes in rice online database.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D5%26rft.spage%253D26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanabata</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mizobuchi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yonemaru</span>  <span class="cit-name-given-names">J.-I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Effect of advanced intercrossing on genome structure and on the power to detect linked quantitative trait loci in a multi-parent population: a simulation study in rice.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253D50%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.38" data-doi="10.1104/pp.127.4.1425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kojima</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Genetic control of flowering time in rice, short-day plant.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1425</span>â<span class="cit-lpage">1429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DYano%26rft.auinit1%253DM.%26rft.volume%253D127%26rft.issue%253D4%26rft.spage%253D1425%26rft.epage%253D1429%26rft.atitle%253DGenetic%2BControl%2Bof%2BFlowering%2BTime%2Bin%2BRice%252C%2Ba%2BShort-Day%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.127.4.1425%26rft_id%253Dinfo%253Apmid%252F11743085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTI3LzQvMTQyNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-book" id="cit-7.6.1721.39" data-doi="10.1371/journal.pone.0017595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Gains in QTL detection using an ultra-high density SNP map based on population sequencing relative to traditional RFLP/SSR markers (H. Matsunami, Ed.).</span> <span class="cit-source">PLoS One</span> <span class="cit-vol">6</span>: <span class="cit-fpage">e17595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DYu%26rft.auinit1%253DH.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De17595%26rft.epage%253De17595%26rft.atitle%253DGains%2Bin%2BQTL%2Bdetection%2Busing%2Ban%2Bultra-high%2Bdensity%2BSNP%2Bmap%2Bbased%2Bon%2Bpopulation%2Bsequencing%2Brelative%2Bto%2Btraditional%2BRFLP%252FSSR%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0017595%26rft_id%253Dinfo%253Apmid%252F21390234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0017595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21390234&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.40" data-doi="10.1371/journal.pbio.0030038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The genomes of <em>Oryza sativa</em>: a history of duplications.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253De38%26rft.epage%253De38%26rft.atitle%253DThe%2BGenomes%2Bof%2BOryza%2Bsativa%253A%2Ba%2Bhistory%2Bof%2Bduplications.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030038%26rft_id%253Dinfo%253Apmid%252F15685292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15685292&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-7.6.1721.41" data-doi="10.1038/ng.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todhunter</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Mixed linear model approach adapted for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">355</span>â<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D360%26rft.atitle%253DMixed%2Blinear%2Bmodel%2Bapproach%2Badapted%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.546%26rft_id%253Dinfo%253Apmid%252F20208535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208535&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1721.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/6/1721.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1707" title="Pedigree-Based Analysis in a Multiparental Population of Octoploid Strawberry Reveals QTL Alleles Conferring Resistance to Phytophthora cactorum" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1731" title="Whole-Genome Sequence of the Metastatic PC3 and LNCaP Human Prostate Cancer Cell Lines" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/6"><span>Volume 7 Issue 6, June 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/6.cover-source.jpg?itok=5_J-OKHc" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1721" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XO2e_Bgsfc9Us2SCBzbxgoTuqzkty__lnSJeqr_eP8w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-population" class="" data-icon-position="" data-hide-link-title="0">Multiparental Population</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/6/1721" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/6/1721" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/6/1721" class="active" data-icon-position="" data-hide-link-title="0">Approaches in Characterizing Genetic Structure and Mapping in a Rice Multiparental Population</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484782" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-KbHhlW7XM8g7TA9ylAZLtXy29hK8p0guvCXXw5rdTjM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1721.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/6/1721" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-B-3ss9fcfQnf70nmWh3JMWa6j5yLmcOmrPf-2rRp1F4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/6/1721" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1721.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484782" id="node484782--2" data-pisa="ggg;7/6/1721" data-pisa-master="ggg;g3.117.042101" data-apath="/ggg/7/6/1721.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Approaches in Characterizing Genetic Structure and Mapping in a Rice Multiparental Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chitra Raghavan</span>, <span class="highwire-citation-author" data-delta="1">Ramil Mauleon</span>, <span class="highwire-citation-author" data-delta="2">Vanica Lacorte</span>, <span class="highwire-citation-author" data-delta="3">Monalisa Jubay</span>, <span class="highwire-citation-author" data-delta="4">Hein Zaw</span>, <span class="highwire-citation-author" data-delta="5">Justine Bonifacio</span>, <span class="highwire-citation-author" data-delta="6">Rakesh Kumar Singh</span>, <span class="highwire-citation-author" data-delta="7">B. Emma Huang</span> and <span class="highwire-citation-author" data-delta="8">Hei Leung</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1721-1730;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042101 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484782--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chitra Raghavan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chitra%2BRaghavan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghavan%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChitra%2BRaghavan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ramil Mauleon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ramil%2BMauleon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mauleon%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARamil%2BMauleon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Vanica Lacorte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanica%2BLacorte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lacorte%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanica%2BLacorte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Monalisa Jubay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Monalisa%2BJubay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jubay%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMonalisa%2BJubay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hein Zaw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hein%2BZaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zaw%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHein%2BZaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Justine Bonifacio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justine%2BBonifacio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bonifacio%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustine%2BBonifacio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Rakesh Kumar Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rakesh%2BKumar%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARakesh%2BKumar%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">B. Emma Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Discovery Sciences, Janssen R&D, Menlo Park, California 94025</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BEmma%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20BE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BEmma%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Hei Leung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hei%2BLeung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leung%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHei%2BLeung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:h.leung@irri.org" class="" data-icon-position="" data-hide-link-title="0">h.leung@irri.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/6/1721" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/6/1721&amp;title=Approaches%20in%20Characterizing%20Genetic%20Structure%20and%20Mapping%20in%20a%20Rice%20Multiparental%20Population" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/6/1721&amp;title=Approaches%20in%20Characterizing%20Genetic%20Structure%20and%20Mapping%20in%20a%20Rice%20Multiparental%20Population" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/6/1721&amp;title=Approaches%20in%20Characterizing%20Genetic%20Structure%20and%20Mapping%20in%20a%20Rice%20Multiparental%20Population" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/6/1721&amp;text=Approaches%20in%20Characterizing%20Genetic%20Structure%20and%20Mapping%20in%20a%20Rice%20Multiparental%20Population" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/6/1721&amp;title=Approaches%20in%20Characterizing%20Genetic%20Structure%20and%20Mapping%20in%20a%20Rice%20Multiparental%20Population" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/6/1721&amp;t=Approaches%20in%20Characterizing%20Genetic%20Structure%20and%20Mapping%20in%20a%20Rice%20Multiparental%20Population" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/6/1721&amp;title=Approaches%20in%20Characterizing%20Genetic%20Structure%20and%20Mapping%20in%20a%20Rice%20Multiparental%20Population" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/6/1721&amp;title=Approaches%20in%20Characterizing%20Genetic%20Structure%20and%20Mapping%20in%20a%20Rice%20Multiparental%20Population" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484782/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484782" id="node484782--3" data-pisa="ggg;7/6/1721" data-pisa-master="ggg;g3.117.042101" data-apath="/ggg/7/6/1721.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Approaches in Characterizing Genetic Structure and Mapping in a Rice Multiparental Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chitra Raghavan</span>, <span class="highwire-citation-author" data-delta="1">Ramil Mauleon</span>, <span class="highwire-citation-author" data-delta="2">Vanica Lacorte</span>, <span class="highwire-citation-author" data-delta="3">Monalisa Jubay</span>, <span class="highwire-citation-author" data-delta="4">Hein Zaw</span>, <span class="highwire-citation-author" data-delta="5">Justine Bonifacio</span>, <span class="highwire-citation-author" data-delta="6">Rakesh Kumar Singh</span>, <span class="highwire-citation-author" data-delta="7">B. Emma Huang</span> and <span class="highwire-citation-author" data-delta="8">Hei Leung</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1721-1730;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042101 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484782--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chitra Raghavan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chitra%2BRaghavan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghavan%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChitra%2BRaghavan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ramil Mauleon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ramil%2BMauleon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mauleon%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARamil%2BMauleon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Vanica Lacorte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanica%2BLacorte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lacorte%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanica%2BLacorte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Monalisa Jubay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Monalisa%2BJubay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jubay%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMonalisa%2BJubay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Hein Zaw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hein%2BZaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zaw%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHein%2BZaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Justine Bonifacio </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justine%2BBonifacio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bonifacio%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJustine%2BBonifacio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Rakesh Kumar Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rakesh%2BKumar%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARakesh%2BKumar%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">B. Emma Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Discovery Sciences, Janssen R&D, Menlo Park, California 94025</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BEmma%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20BE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BEmma%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Hei Leung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, Manila 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hei%2BLeung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Leung%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHei%2BLeung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:h.leung@irri.org" class="" data-icon-position="" data-hide-link-title="0">h.leung@irri.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1721&amp;rft.epage=1730&amp;rft.atitle=Approaches%20in%20Characterizing%20Genetic%20Structure%20and%20Mapping%20in%20a%20Rice%20Multiparental%20Population&amp;rft.volume=7&amp;rft.issue=6&amp;rft.date=2017-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Raghavan%2C+Chitra&amp;rft.au=Mauleon%2C+Ramil&amp;rft.au=Lacorte%2C+Vanica&amp;rft.au=Jubay%2C+Monalisa&amp;rft.au=Zaw%2C+Hein&amp;rft.au=Bonifacio%2C+Justine&amp;rft.au=Singh%2C+Rakesh+Kumar&amp;rft.au=Huang%2C+B.+Emma&amp;rft.au=Leung%2C+Hei"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484782/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484782/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484782/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484782/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484782/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484782/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484782/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484782/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484782/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484782/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484782/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484782/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484763" id="node484763" data-pisa="ggg;7/6/1707" data-pisa-master="ggg;g3.117.042119" data-apath="/ggg/7/6/1707.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/7/6/1707" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Pedigree-Based Analysis in a Multiparental Population of Octoploid Strawberry Reveals QTL Alleles Conferring Resistance to <em>Phytophthora cactorum</em></span></a></div>
</cite>
</div></li><li class="last even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484773" id="node484773" data-pisa="ggg;7/6/1693" data-pisa-master="ggg;g3.117.042127" data-apath="/ggg/7/6/1693.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/7/6/1693" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identification of Nitrogen Consumption Genetic Variants in Yeast Through QTL Mapping and Bulk Segregant RNA-Seq Analyses</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Population" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Population</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/6/1721" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/6/1721.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/6/1721.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/6/1721.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/6/1721.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
