<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/6/1913" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/6/1913.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/6/1913.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/6/1913.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/6/1913/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/6/1913.atom" />
<meta name="HW.pisa" content="ggg;7/6/1913" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genetic Architecture of a Rice Nested Association Mapping Population" />
<meta name="DC.Identifier" content="10.1534/g3.117.041608" />
<meta name="DC.Date" content="2017-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Fragoso et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Describing the genetic diversity in the gene pool of crops will provide breeders with novel resources for varietal improvement. Nested Association Mapping (NAM) populations are uniquely suited for characterizing parental diversity through the shuffling and fixation of parental haplotypes. Here, we describe a set of 1879 rice NAM lines created through the selfing and single-seed descent of F1 hybrids derived from elite IR64 indica crossed with 10 diverse tropical japonica lines. Genotyping data indicated tropical japonica alleles were captured at every queried locus despite the presence of segregation distortion factors. Several distortion loci were mapped, both shared and unique, among the 10 populations. Using two-point and multi-point genetic map calculations, our datasets achieved the ∼1500 cM expected map size in rice. Finally, we highlighted the utility of the NAM lines for QTL mapping, including joint analysis across the 10 populations, by confirming known QTL locations for the trait days to heading." />
<meta name="DC.Contributor" content="Christopher A. Fragoso" />
<meta name="DC.Contributor" content="Maria Moreno" />
<meta name="DC.Contributor" content="Zuoheng Wang" />
<meta name="DC.Contributor" content="Christopher Heffelfinger" />
<meta name="DC.Contributor" content="Lady J. Arbelaez" />
<meta name="DC.Contributor" content="John A. Aguirre" />
<meta name="DC.Contributor" content="Natalia Franco" />
<meta name="DC.Contributor" content="Luz E. Romero" />
<meta name="DC.Contributor" content="Karine Labadie" />
<meta name="DC.Contributor" content="Hongyu Zhao" />
<meta name="DC.Contributor" content="Stephen L. Dellaporta" />
<meta name="DC.Contributor" content="Mathias Lorieux" />
<meta name="article:published_time" content="2017-06-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Genetic Architecture of a Rice Nested Association Mapping Population" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Describing the genetic diversity in the gene pool of crops will provide breeders with novel resources for varietal improvement. Nested Association Mapping (NAM) populations are uniquely suited for characterizing parental diversity through the shuffling and fixation of parental haplotypes. Here, we describe a set of 1879 rice NAM lines created through the selfing and single-seed descent of F&lt;sub&gt;1&lt;/sub&gt; hybrids derived from elite IR64 &lt;i&gt;indica&lt;/i&gt; crossed with 10 diverse &lt;i&gt;tropical japonica&lt;/i&gt; lines. Genotyping data indicated &lt;i&gt;tropical japonica&lt;/i&gt; alleles were captured at every queried locus despite the presence of segregation distortion factors. Several distortion loci were mapped, both shared and unique, among the 10 populations. Using two-point and multi-point genetic map calculations, our datasets achieved the ∼1500 cM expected map size in rice. Finally, we highlighted the utility of the NAM lines for QTL mapping, including joint analysis across the 10 populations, by confirming known QTL locations for the trait days to heading.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/06/01" />
<meta name="citation_mjid" content="ggg;7/6/1913" />
<meta name="citation_id" content="7/6/1913" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/6/1913" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/6/1913.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/6/1913.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/6/1913.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.041608" />
<meta name="citation_pmid" content="28450374" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="1913" />
<meta name="citation_lastpage" content="1926" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Christopher A. Fragoso" />
<meta name="citation_author_institution" content="Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut 06511" />
<meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511" />
<meta name="citation_author" content="Maria Moreno" />
<meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511" />
<meta name="citation_author" content="Zuoheng Wang" />
<meta name="citation_author_institution" content="Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut 06511" />
<meta name="citation_author_institution" content="Department of Biostatistics, Yale University, New Haven, Connecticut 06511" />
<meta name="citation_author" content="Christopher Heffelfinger" />
<meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511" />
<meta name="citation_author" content="Lady J. Arbelaez" />
<meta name="citation_author_institution" content="Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia" />
<meta name="citation_author" content="John A. Aguirre" />
<meta name="citation_author_institution" content="Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia" />
<meta name="citation_author" content="Natalia Franco" />
<meta name="citation_author_institution" content="Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia" />
<meta name="citation_author" content="Luz E. Romero" />
<meta name="citation_author_institution" content="Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia" />
<meta name="citation_author" content="Karine Labadie" />
<meta name="citation_author_institution" content="Commissariat à L’énergie Atomique et aux Énergies Alternatives, Institut de Génomique, Genoscope, 91000 Evry, France" />
<meta name="citation_author" content="Hongyu Zhao" />
<meta name="citation_author_institution" content="Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut 06511" />
<meta name="citation_author_institution" content="Department of Biostatistics, Yale University, New Haven, Connecticut 06511" />
<meta name="citation_author" content="Stephen L. Dellaporta" />
<meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511" />
<meta name="citation_author" content="Mathias Lorieux" />
<meta name="citation_author_institution" content="Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia" />
<meta name="citation_author_institution" content="Diversité, Adaptation, Développement des Plantes Research Unit, Institut de Recherche pour le Développement, F-34394 Montpellier, France" />
<meta name="citation_author_email" content="mathias.lorieux@ird.fr" />
<meta name="citation_author_email" content="m.lorieux@cgiar.org" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. Alexandrov;citation_author=S. Tai;citation_author=W. Wang;citation_author=L. Mansueto;citation_author=K. Palis;citation_title=SNP-Seek database of SNPs derived from 3000 rice genomes.;citation_volume=43;citation_year=2015;citation_pmid=25429973;citation_doi=10.1093/nar/gku1039" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Bioinformatics;citation_author=G. A. Auwera;citation_author=M. O. Carneiro;citation_author=C. Hartl;citation_author=R. Poplin;citation_author=G. del Angel;citation_title=From FastQ data to high confidence variant calls: the genome analysis toolkit best practices pipeline.;citation_pages=11101-11.10.33;citation_volume=43;citation_year=2013;citation_pmid=25431634" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=P. Bajgain;citation_author=M. N. Rouse;citation_author=T. J. Tsilo;citation_author=G. K. Macharia;citation_author=S. Bhavani;citation_title=Nested association mapping of stem rust resistance in wheat using genotyping by sequencing.;citation_pages=e0155760;citation_volume=11;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=H. Begum;citation_author=J. E. Spindel;citation_author=A. Lalusin;citation_author=T. Borromeo;citation_author=G. Gregorio;citation_title=Genome-wide association mapping for yield and other agronomic traits in an elite breeding population of tropical rice (Oryza sativa).;citation_pages=e0119873;citation_volume=10;citation_year=2015;citation_pmid=25785447;citation_doi=10.1371/journal.pone.0119873" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=F. Bourgis;citation_author=R. Guyot;citation_author=H. Gherbi;citation_author=E. Tailliez;citation_author=I. Amabile;citation_title=Characterization of the major fragance gene from an aromatic japonica rice and analysis of its diversity in Asian cultivated rice.;citation_pages=353-368;citation_volume=117;citation_year=2008;citation_issue=3;citation_pmid=18491070;citation_doi=10.1007/s00122-008-0780-9" />
<meta name="citation_reference" content="citation_title=International Rice Research Institute’s Contribution to Rice Varietal Yield Improvement in South-East Asia;citation_author=J. P. Brennan;citation_author=A. Malabayabas;citation_year=2011" />
<meta name="citation_reference" content="citation_title=A Guide to QTL Mapping with R/qtl;citation_author=K. W. Broman;citation_author=S. Sen;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=Ś. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. S. Buckler;citation_author=J. B. Holland;citation_author=P. J. Bradbury;citation_author=C. B. Acharya;citation_author=P. J. Brown;citation_title=The Genetic Architecture of Maize Flowering Time;citation_pages=714-718;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661422;citation_doi=10.1126/science.1174276" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=E. Callaway;citation_title=Domestication: the birth of rice.;citation_pages=S58-S59;citation_volume=514;citation_year=2014;citation_pmid=25368889;citation_doi=10.1038/514S58a" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=J. Chen;citation_author=X. Li;citation_author=C. Cheng;citation_author=Y. Wang;citation_author=M. Qin;citation_title=Characterization of epistatic interaction of QTLs LH8 and EH3 controlling heading date in rice.;citation_pages=4263;citation_volume=4;citation_year=2014;citation_pmid=24584028" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=M. Chen;citation_author=G. Presting;citation_author=W. B. Barbazuk;citation_author=J. L. Goicoechea;citation_author=B. Blackmon;citation_title=An integrated physical and genetic map of the rice genome.;citation_volume=14;citation_year=2002;citation_pmid=11910002;citation_doi=10.1105/tpc.010485" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=Z. Cheng;citation_author=F. Dong;citation_author=T. Langdon;citation_author=S. Ouyang;citation_author=C. R. Buell;citation_title=Functional rice centromeres are marked by a satellite repeat and a centromere-specific retrotransposon.;citation_volume=14;citation_year=2002;citation_pmid=12172016;citation_doi=10.1105/tpc.003079" />
<meta name="citation_reference" content="citation_journal_title=Annual review of plant biology;citation_journal_abbrev=Annu Rev Plant Biol;citation_author=A. Y. Cheung;citation_author=H.-M. Wu;citation_title=Structural and signaling networks for the polar cell growth machinery in pollen tubes.;citation_pages=547-572;citation_volume=59;citation_year=2008;citation_pmid=18444907;citation_doi=10.1146/annurev.arplant.59.032607.092921" />
<meta name="citation_reference" content="citation_journal_title=Nat. Plants;citation_author=P. Civáň;citation_author=H. Craig;citation_author=C. J. Cox;citation_author=T. A. Brown;citation_title=Three geographically separate domestications of Asian rice.;citation_pages=15164;citation_volume=1;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=G. Djedatin;citation_author=M.-N. Ndjiondjop;citation_author=A. Sanni;citation_author=M. Lorieux;citation_author=V. Verdier;citation_title=Identification of novel major and minor QTLs associated with Xanthomonas oryzae pv. oryzae (African strains) resistance in rice (Oryza sativa L.).;citation_pages=1-10;citation_volume=9;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=C.-H. Dong;citation_author=B. Kost;citation_author=G. Xia;citation_author=N.-H. Chua;citation_title=Molecular identification and characterization of the Arabidopsis AtADF1, AtADFS and AtADF6 genes.;citation_pages=517-527;citation_volume=45;citation_year=2001;citation_issue=5;citation_pmid=11414611;citation_doi=10.1023/A:1010687911374" />
<meta name="citation_reference" content="citation_journal_title=Journal of plant physiology;citation_journal_abbrev=J Plant Physiol;citation_author=Y. Feng;citation_author=Q. Liu;citation_author=Q. Xue;citation_title=Comparative study of rice and Arabidopsis actin-depolymerizing factors gene families.;citation_pages=69-79;citation_volume=163;citation_year=2006;citation_issue=1;citation_pmid=16360805;citation_doi=10.1016/j.jplph.2005.01.015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. A. Fragoso;citation_author=C. Heffelfinger;citation_author=H. Zhao;citation_author=S. L. Dellaporta;citation_title=Imputing genotypes in biallelic populations from low-coverage sequence data.;citation_volume=202;citation_year=2016;citation_pmid=26715670;citation_doi=10.1534/genetics.115.182071" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=E. W. Gachomo;citation_author=L. J. Baptiste;citation_author=T. Kefela;citation_author=W. M. Saidel;citation_author=S. O. Kotchoni;citation_title=The Arabidopsis CURVY1 (CVY1) gene encoding a novel receptor-like protein kinase regulates cell morphogenesis, flowering time and seed production.;citation_pages=1;citation_volume=14;citation_year=2014;citation_pmid=24387633;citation_doi=10.1186/1471-2229-14-1" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=H. Gao;citation_author=X.-M. Zheng;citation_author=G. Fei;citation_author=J. Chen;citation_author=M. Jin;citation_title=Ehd4 encodes a novel and Oryza-genus-specific regulator of photoperiodic flowering in rice.;citation_pages=e1003281;citation_volume=9;citation_year=2013;citation_pmid=23437005;citation_doi=10.1371/journal.pgen.1003281" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. J. Garris;citation_author=T. H. Tai;citation_author=J. Coburn;citation_author=S. Kresovich;citation_author=S. McCouch;citation_title=Genetic structure and diversity in Oryza sativa L.;citation_volume=169;citation_year=2005;citation_pmid=15654106;citation_doi=10.1534/genetics.104.035642" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. C. J. Godfray;citation_author=J. R. Beddington;citation_author=I. R. Crute;citation_author=L. Haddad;citation_author=D. Lawrence;citation_title=Food security: the challenge of feeding 9 billion people.;citation_volume=327;citation_year=2010;citation_pmid=20110467;citation_doi=10.1126/science.1185383" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=E. Guiderdoni;citation_author=E. Galinato;citation_author=J. Luistro;citation_author=G. Vergara;citation_title=Anther culture of tropical japonica× indica hybrids of rice (Oryza sativa L.).;citation_pages=219-224;citation_volume=62;citation_year=1992;citation_doi=10.1007/BF00041756" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Y. Harushima;citation_author=N. Kurata;citation_author=M. Yano;citation_author=Y. Nagamura;citation_author=T. Sasaki;citation_title=Detection of segregation distortions in an indica-japonica rice cross using a high-resolution molecular map.;citation_pages=145-150;citation_volume=92;citation_year=1996;citation_pmid=24166160;citation_doi=10.1007/s001220050106" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Harushima;citation_author=M. Yano;citation_author=A. Shomura;citation_author=M. Sato;citation_author=T. Shimano;citation_title=A High-Density Rice Genetic Linkage Map with 2275 Markers Using a Single F2 Population;citation_pages=479-494;citation_volume=148;citation_year=1998;citation_issue=1;citation_pmid=9475757" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Heffelfinger;citation_author=A. C. Fragoso;citation_author=M. A. Moreno;citation_author=J. D. Overton;citation_author=J. P. Mottinger;citation_title=Flexible and scalable genotyping-by-sequencing strategies for population studies.;citation_pages=979;citation_volume=15;citation_year=2014;citation_pmid=25406744;citation_doi=10.1186/1471-2164-15-979" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Heffelfinger;citation_author=C. A. Fragoso;citation_author=M. Lorieux;citation_title=Constructing linkage maps in the genomics era with MapDisto 2.0.;citation_year=2017;citation_doi=10.1093/bioinformatics/btx177" />
<meta name="citation_reference" content="citation_title=Novoalign;citation_author=C. Hercus;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=A. M. Ismail;citation_author=U. S. Singh;citation_author=S. Singh;citation_author=M. H. Dar;citation_author=D. J. Mackill;citation_title=The contribution of submergence-tolerant (Sub1) rice varieties to food security in flood-prone rainfed lowland areas in Asia.;citation_pages=83-93;citation_volume=152;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=T. Izawa;citation_title=The process of rice domestication: a new model based on recent data.;citation_pages=127-134;citation_volume=1;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Plant, Cell, and Environment (Print);citation_journal_abbrev=Plant, Cell, and Environment (Print);citation_author=D. H. Jeong;citation_author=S. Lee;citation_author=S. L. Kim;citation_author=I. Hwang;citation_author=G. An;citation_title=Regulation of brassinosteroid responses by phytochrome B in rice.;citation_pages=590-599;citation_volume=30;citation_year=2007;citation_issue=5;citation_pmid=17407537" />
<meta name="citation_reference" content="citation_title=Oryza: From Molecule to Plant;citation_title=Origin, dispersal, cultivation and variation of rice;citation_author=G. S. Khush;citation_pages=25-34;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=B. Kim;citation_author=S. M. Jang;citation_author=S.-H. Chu;citation_author=Y. Bordiya;citation_author=M. B. Akter;citation_title=Analysis of segregation distortion and its relationship to hybrid barriers in rice.;citation_pages=1;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. J. Kovach;citation_author=M. T. Sweeney;citation_author=S. R. McCouch;citation_title=New insights into the history of rice domestication.;citation_pages=578-587;citation_volume=23;citation_year=2007;citation_issue=11;citation_pmid=17963977;citation_doi=10.1016/j.tig.2007.08.012" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. L. Kump;citation_author=P. J. Bradbury;citation_author=R. J. Wisser;citation_author=E. S. Buckler;citation_author=A. R. Belcher;citation_title=Genome-wide association study of quantitative resistance to southern leaf blight in the maize nested association mapping population.;citation_pages=163-168;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217757;citation_doi=10.1038/ng.747" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. Lee;citation_author=J. Kim;citation_author=J. J. Han;citation_author=M. J. Han;citation_author=G. An;citation_title=Functional analyses of the flowering time gene OsMADS50, the putative SUPPRESSOR OF OVEREXPRESSION OF CO 1/AGAMOUS-LIKE 20 (SOC1/AGL20) ortholog in rice.;citation_pages=754-764;citation_volume=38;citation_year=2004;citation_issue=5;citation_pmid=15144377;citation_doi=10.1111/j.1365-313X.2004.02082.x" />
<meta name="citation_reference" content="citation_journal_title=J. Rice Res.;citation_author=Y.-S. Lee;citation_author=G. An;citation_title=Complex regulatory networks of flowering time in rice.;citation_pages=2;citation_volume=3;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Plant Biol.;citation_author=Y.-S. Lee;citation_author=G. An;citation_title=Regulation of flowering time in rice.;citation_pages=353-360;citation_volume=58;citation_year=2015;citation_doi=10.1007/s12374-015-0425-x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Li;citation_author=Y. Li;citation_author=Y. Shi;citation_author=Y. Song;citation_author=D. Zhang;citation_title=Analysis of recombination QTLs, segregation distortion, and epistasis for fitness in maize multiple populations using ultra-high-density markers.;citation_pages=1775-1784;citation_volume=129;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Planta;citation_author=D. Li;citation_author=C. Yang;citation_author=X. Li;citation_author=Q. Gan;citation_author=X. Zhao;citation_title=Functional characterization of rice OsDof12.;citation_pages=1159-1169;citation_volume=229;citation_year=2009;citation_pmid=19198875;citation_doi=10.1007/s00425-009-0893-7" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=X.-B. Li;citation_author=D. Xu;citation_author=X.-L. Wang;citation_author=G.-Q. Huang;citation_author=J. Luo;citation_title=Three cotton genes preferentially expressed in flower tissues encode actin-depolymerizing factors which are involved in F-actin dynamics in cells.;citation_volume=61;citation_year=2010;citation_pmid=19861654;citation_doi=10.1093/jxb/erp280" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. Lin;citation_author=H. Ikehashi;citation_author=S. Yanagihara;citation_author=A. Kawashima;citation_title=Segregation distortion via male gametes in hybrids between Indica and Japonica or wide-compatibility varieties of rice (Oryza sativa L).;citation_pages=812-818;citation_volume=84;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Planta;citation_journal_abbrev=Planta;citation_author=Z. Lin;citation_author=M. E. Griffith;citation_author=X. Li;citation_author=Z. Zhu;citation_author=L. Tan;citation_title=Origin of seed shattering in rice (Oryza sativa L.).;citation_pages=11-20;citation_volume=226;citation_year=2007;citation_issue=1;citation_pmid=17216230;citation_doi=10.1007/s00425-006-0460-4" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=S. E. Lincoln;citation_author=E. S. Lander;citation_title=Systematic detection of errors in genetic linkage data.;citation_pages=604-610;citation_volume=14;citation_year=1992;citation_issue=3;citation_pmid=1427888;citation_doi=10.1016/S0888-7543(05)80158-2" />
<meta name="citation_reference" content="citation_title=Constructing Genetic Linkage Maps with MAPMAKER/EXP Version 3.0: A Tutorial and Reference Manual;citation_author=S. E. Lincoln;citation_author=M. J. Daly;citation_author=E. S. Lander;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. P. Londo;citation_author=Y.-C. Chiang;citation_author=K.-H. Hung;citation_author=T.-Y. Chiang;citation_author=B. A. Schaal;citation_title=Phylogeography of Asian wild rice, Oryza rufipogon, reveals multiple independent domestications of cultivated rice, Oryza sativa.;citation_volume=103;citation_year=2006;citation_pmid=16766658;citation_doi=10.1073/pnas.0603152103" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=M. Lorieux;citation_title=MapDisto: fast and efficient computation of genetic linkage maps.;citation_pages=1231-1235;citation_volume=30;citation_year=2012;citation_doi=10.1007/s11032-012-9706-y" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=C. Lu;citation_author=K. Takabatake;citation_author=H. Ikehashi;citation_title=Identification of segregation-distortion-neutral alleles to improve pollen fertility of indica-japonica hybrids in rice (Oryza sativa L.).;citation_pages=101-107;citation_volume=113;citation_year=2000;citation_doi=10.1023/A:1003929312252" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Ma;citation_author=J. L. Bennetzen;citation_title=Rapid recent growth and divergence of rice nuclear genomes.;citation_volume=101;citation_year=2004;citation_pmid=15240870;citation_doi=10.1073/pnas.0403715101" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=O. C. Martin;citation_author=F. Hospital;citation_title=Two-and three-locus tests for linkage analysis using recombinant inbred lines.;citation_volume=173;citation_year=2006;citation_pmid=16489236;citation_doi=10.1534/genetics.105.047175" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=S. Matsushita;citation_author=T. Iseki;citation_author=Y. Fukuta;citation_author=E. Araki;citation_author=S. Kobayashi;citation_title=Characterization of segregation distortion on chromosome 3 induced in wide hybridization between indica and japonica type rice varieties.;citation_pages=27-32;citation_volume=134;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. Maurer;citation_author=V. Draba;citation_author=Y. Jiang;citation_author=F. Schnaithmann;citation_author=R. Sharma;citation_title=Modelling the genetic architecture of flowering time control in barley through nested association mapping.;citation_pages=1;citation_volume=16;citation_year=2015;citation_pmid=25553907;citation_doi=10.1186/1471-2164-16-1" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. McMullen;citation_author=S. Kresovich;citation_author=H. S. Villeda;citation_author=P. Bradbury;citation_author=H. Li;citation_title=Genetic Properties of the Maize Nested Association Mapping Population;citation_pages=737-740;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661427;citation_doi=10.1126/science.1174320" />
<meta name="citation_reference" content="citation_journal_title=Rice Today;citation_author=S. Mohanty;citation_title=Trends in global rice consumption.;citation_pages=44-45;citation_volume=12;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Ann. N. Y. Acad. Sci.;citation_author=S. Muthayya;citation_author=J. D. Sugimoto;citation_author=S. Montgomery;citation_author=G. F. Maberly;citation_title=An overview of global rice production, supply, trade, and consumption.;citation_pages=7-14;citation_volume=1324;citation_year=2014;citation_pmid=25224455;citation_doi=10.1111/nyas.12540" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=F. Ogut;citation_author=Y. Bian;citation_author=P. J. Bradbury;citation_author=J. B. Holland;citation_title=Joint-multiple family linkage analysis predicts within-family variation better than single-family analysis of the maize nested association mapping population.;citation_pages=552-563;citation_volume=114;citation_year=2015;citation_pmid=25585918;citation_doi=10.1038/hdy.2014.123" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Ouyang;citation_author=W. Zhu;citation_author=J. Hamilton;citation_author=H. Lin;citation_author=M. Campbell;citation_title=The TIGR rice genome annotation resource: improvements and new features.;citation_volume=35;citation_year=2007;citation_pmid=17145706;citation_doi=10.1093/nar/gkl976" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. L. Peirce;citation_author=K. W. Broman;citation_author=L. Lu;citation_author=R. W. Williams;citation_title=A simple method for combining genetic mapping data from multiple crosses and experimental designs.;citation_pages=e1036-e1036;citation_volume=2;citation_year=2007;citation_issue=10;citation_pmid=17940600;citation_doi=10.1371/journal.pone.0001036" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=S. Peng;citation_author=K. G. Cassman;citation_author=S. S. Virmani;citation_author=J. Sheehy;citation_author=G. S. Khush;citation_title=Yield Potential Trends of Tropical Rice since the Release of IR8 and the Challenge of Increasing Rice Yield Potential;citation_pages=1552-1559;citation_volume=39;citation_year=1999;citation_issue=6;citation_doi=10.2135/cropsci1999.3961552x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. A. Poland;citation_author=P. J. Bradbury;citation_author=E. S. Buckler;citation_author=R. J. Nelson;citation_title=Genome-wide nested association mapping of quantitative resistance to northern leaf blight in maize.;citation_volume=108;citation_year=2011;citation_pmid=21482771;citation_doi=10.1073/pnas.1010894108" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. C. Schatz;citation_author=L. G. Maron;citation_author=J. C. Stein;citation_author=A. H. Wences;citation_author=J. Gurtowski;citation_title=Whole genome de novo assemblies of three divergent strains of rice, Oryza sativa, document novel gene space of aus and indica.;citation_pages=1;citation_volume=15;citation_year=2014;citation_pmid=24393533;citation_doi=10.1186/gb-2014-15-1-r1" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Spindel;citation_author=M. Wright;citation_author=C. Chen;citation_author=J. Cobb;citation_author=J. Gage;citation_title=Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations.;citation_pages=2699-2716;citation_volume=126;citation_year=2013;citation_pmid=23918062;citation_doi=10.1007/s00122-013-2166-x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.;citation_author=K. Swarts;citation_author=H. Li;citation_author=J. A. Romero Navarro;citation_author=D. An;citation_author=M. C. Romay;citation_volume=7;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=M. Sweeney;citation_author=S. McCouch;citation_title=The complex history of the domestication of rice.;citation_volume=100;citation_year=2007;citation_pmid=17617555;citation_doi=10.1093/aob/mcm128" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=A. L. Szumlanski;citation_author=E. Nielsen;citation_title=The Rab GTPase RabA4d regulates pollen tube tip growth in Arabidopsis thaliana.;citation_volume=21;citation_year=2009;citation_pmid=19208902;citation_doi=10.1105/tpc.108.060277" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=M. Takano;citation_author=N. Inagaki;citation_author=X. Xie;citation_author=N. Yuzurihara;citation_author=F. Hihara;citation_title=Distinct and cooperative functions of phytochromes A, B, and C in the control of deetiolation and flowering in rice.;citation_volume=17;citation_year=2005;citation_pmid=16278346;citation_doi=10.1105/tpc.105.035899" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Tester;citation_author=P. Langridge;citation_title=Breeding Technologies to Increase Crop Production in a Changing World;citation_pages=818-822;citation_volume=327;citation_year=2010;citation_issue=5967;citation_pmid=20150489;citation_doi=10.1126/science.1183700" />
<meta name="citation_reference" content="citation_title=GigaScience;citation_author=The 3000 rice genomes project;citation_pages=1-6;citation_volume=3;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Biol. Conserv.;citation_author=T. Tscharntke;citation_author=Y. Clough;citation_author=T. C. Wanger;citation_author=L. Jackson;citation_author=I. Motzke;citation_title=Global food security, biodiversity conservation and the future of agricultural intensification.;citation_pages=53-59;citation_volume=151;citation_year=2012;citation_doi=10.1016/j.biocon.2012.01.068" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=Y. P. Wu;citation_author=P. Y. Ko;citation_author=W. C. Lee;citation_author=F. J. Wei;citation_author=S. C. Kuo;citation_title=Comparative analyses of linkage maps and segregation distortion of two F2 populations derived from japonica crossed with indica rice.;citation_pages=225-236;citation_volume=147;citation_year=2010;citation_issue=5;citation_pmid=21039459;citation_doi=10.1111/j.1601-5223.2010.02120.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Xu;citation_title=Quantitative trait locus mapping can benefit from segregation distortion.;citation_volume=180;citation_year=2008;citation_pmid=18957707;citation_doi=10.1534/genetics.108.090688" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=Y. Xu;citation_author=L. Zhu;citation_author=J. Xiao;citation_author=N. Huang;citation_author=S. McCouch;citation_title=Chromosomal regions associated with segregation distortion of molecular markers in F2, backcross, doubled haploid, and recombinant inbred populations in rice (Oryza sativa L.).;citation_pages=535-545;citation_volume=253;citation_year=1997;citation_issue=5;citation_pmid=9065686;citation_doi=10.1007/s004380050355" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Xu;citation_author=H. K. Dooner;citation_title=The maize aberrant pollen transmission 1 gene is a SABRE/KIP homolog required for pollen tube growth.;citation_volume=172;citation_year=2006;citation_pmid=16299389;citation_doi=10.1534/genetics.105.050237" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Yamamoto;citation_author=H. Lin;citation_author=T. Sasaki;citation_author=M. Yano;citation_title=Identification of Heading Date Quantitative Trait Locus Hd6 and Characterization of Its Epistatic Interactions With Hd2 in Rice Using Advanced Backcross Progeny;citation_pages=885-891;citation_volume=154;citation_year=2000;citation_issue=2;citation_pmid=10655238" />
<meta name="citation_reference" content="citation_title=The Pollen Tube;citation_title=The actin cytoskeleton in pollen tubes; actin and actin binding proteins;citation_author=E. Yokota;citation_author=T. Shimmen;citation_pages=139-155;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Yu;citation_author=J. B. Holland;citation_author=M. D. McMullen;citation_author=E. S. Buckler;citation_title=Genetic Design and Statistical Power of Nested Association Mapping in Maize;citation_pages=539-551;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202393;citation_doi=10.1534/genetics.107.074245" />
<meta name="citation_reference" content="citation_journal_abbrev=J Genet Genomics;citation_author=C. Zhang;citation_author=L. Guo;citation_author=X. Wang;citation_author=H. Zhang;citation_author=H. Shi;citation_title=Molecular characterization of four ADF genes differentially expressed in cotton.;citation_pages=347-354;citation_volume=34;citation_year=2007;citation_issue=4;citation_pmid=17498633;citation_doi=10.1016/S1673-8527(07)60037-X" />
<meta name="citation_reference" content="citation_journal_title=Rice Genet. News;citation_author=Z. Zhang;citation_author=P. Xu;citation_author=F. Hu;citation_author=J. Zhou;citation_author=J. Li;citation_title=A new sterile gene from Oryza glaberrima on chromosome 3.;citation_pages=26-29;citation_volume=22;citation_year=2005" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genetic Architecture of a Rice Nested Association Mapping Population" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="twitter:description" content="Describing the genetic diversity in the gene pool of crops will provide breeders with novel resources for varietal improvement. Nested Association Mapping (NAM) populations are uniquely suited for characterizing parental diversity through the shuffling and fixation of parental haplotypes. Here, we describe a set of 1879 rice NAM lines created through the selfing and single-seed descent of F1 hybrids derived from elite IR64 indica crossed with 10 diverse tropical japonica lines. Genotyping data indicated tropical japonica alleles were captured at every queried locus despite the presence of segregation distortion factors. Several distortion loci were mapped, both shared and unique, among the 10 populations. Using two-point and multi-point genetic map calculations, our datasets achieved the ∼1500 cM expected map size in rice. Finally, we highlighted the utility of the NAM lines for QTL mapping, including joint analysis across the 10 populations, by confirming known QTL locations for the trait days to heading." />
<meta name="og-title" property="og:title" content="Genetic Architecture of a Rice Nested Association Mapping Population" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/6/1913" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Describing the genetic diversity in the gene pool of crops will provide breeders with novel resources for varietal improvement. Nested Association Mapping (NAM) populations are uniquely suited for characterizing parental diversity through the shuffling and fixation of parental haplotypes. Here, we describe a set of 1879 rice NAM lines created through the selfing and single-seed descent of F1 hybrids derived from elite IR64 indica crossed with 10 diverse tropical japonica lines. Genotyping data indicated tropical japonica alleles were captured at every queried locus despite the presence of segregation distortion factors. Several distortion loci were mapped, both shared and unique, among the 10 populations. Using two-point and multi-point genetic map calculations, our datasets achieved the ∼1500 cM expected map size in rice. Finally, we highlighted the utility of the NAM lines for QTL mapping, including joint analysis across the 10 populations, by confirming known QTL locations for the trait days to heading." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
    <title>Genetic Architecture of a Rice Nested Association Mapping Population | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"p_xFHaGRvtitHem7y0zGuOMaWi9hF-pCZMeb-Y7IlCA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484771","apath":"\/ggg\/7\/6\/1913.atom","pisa":"ggg;7\/6\/1913","ac":{"\/ggg\/7\/6\/1913.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/6\/1913.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1913"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1913"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/6\/1913":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JZs\\\\\\\/\\\\\\\/dV\\\\\\\/F\\\\\\\/hcUshebidDOayqXTuL23Au5IWcWoV9PTqv5K4qbVz9q\\\\\\\/V+PePhulm\\\\\\\/0tWiEA7hvh6QVVAaok95qFPBpH9Jpjt3ga3WmSlvCpF9zxx0zCZaAnRZsUBevVKFyGVhDEmkprxXDKCz\\\\\\\/YI+O+FB1t80oeRPwySHO7MiPzlDb7qtvKzJwQRND50yewUxbqfJW3eLDPhRyz7\\\\\\\/xhxmNRbyZm1n30oWvqKJE4p1sw8e7ue8flT668av3YSPeu1p1VKNJcVrwqnvdInFhJFnUI3nBCaJXnSmqsnIf5p1VSPdWZgmgXNIwr\\\\\\\/xtmlfeEDBeAZp4AepiXXcM\\\\\\\/2rF0Jf1tQ2+d7NgQMVsWp6LAOHJbtKR7TRs5c8DzxraFc0x+mxLRSpKzAOyQYazy0Og+c1dUx2UWh6Iybb3ilo9Er54BaTKOWxzACPRpuBt8y4yujXXxw0B6IyhcxdoE8JfjUYpEkXod1dpkLirhFfc5\\\\\\\/77Xhxk43wNglmPRgTxh3R7mf6E\\\\\\\/R85rg8GaFOwGhCTQSKK9jZHK1djuu35u83lD0KRfVq0Z1Q5pE9R4XfOrTHmMf6B10Cw6E0fjUgWv0\\\\\\\/HA3wWvRsBwttUWbpnakP856\\\\\\\/b3UKbZ4uIlMALKqsp28L6nGfK2I1lDN0xLH5RWW5uqafTYTkR\\\\\\\/OchzuotCYrgrYxl56J7bMG8giTSrD3hzpff8gdpbIJ3+8Hxi3Vm\\\\\\\/v75OmqAKl9kXaOyLF6rPIveUFAW+texwB0Hqp2kOaBNOHqOCLwLXpR32eXat0j4J8+1OUiSj7v\\\\\\\/2nhj94Z3ZU5M1vYCuSqgRmxuc0GsYNcFoQAyGiFiCuBJOpGmdGb4Ck1rPWax9NZyzBGlDzGk2wEW5Ga4epnHVIrCl8IIxnvVUtdyvCHDuKbDP3ikvbaIiXchY3v1f2sy4nqElNDFe9RGPPIjtYUpJYPoWH6hLUmlcKHPlBf4ExT\\\\\\\/vkE7IoiQBv11nFVVw66+brraQg4I9bF4XD\\\\\\\/mjzdHcjJMK4bnYq+s1nUpHlDSNO9w2lVfWSgz5LWWkGFQkw8n\\\\\\\/+ou5WUzpKYqL60lVoOLsCcSzXJqmcOrZ\\\\\\\/go+8vMm7SpnmEwUYHdt5yjIZmAC5tbg2UpOeq\\\\\\\/QT\\\\\\\/\\\\\\\/deeDcGZJ055zGm2uQvOIYTvYLrR\\\\\\\/kRfEWBsqYuhkg+4FBHl89Ky8Hk7BPD8tlxuE8WmSpoTylxas7BvTGtlsZGwnNRGFickE9Ebw0xKKki\\\\\\\/2rQARiyl+zxjx+QaiaRTaYYrfDq3hWqg8xmL\\\\\\\/QR06oaGNoEs\\\u0022,\\\u0022iv\\\u0022:\\\u002217p2qqhr97B8hnPNRZ949w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002297e9ed53fd7c8a3736092ccb0f40f80736a3b80a486590ec86603634582bcc12\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224mBy+5CHEHTbP+\\\\\\\/lnW0Lt5ok+\\\\\\\/rXYhxaEDW7mct4gxckJFI7SaLCIfJDZqXYByUZrQG7b8YY1v1cXn20WPx+huuqTZDpUnq8W\\\\\\\/N7D\\\\\\\/JBOyyDx1LcwzUn76G4M+5oKXrZ6GSD7YdgnpN0gq\\\\\\\/vcVT0e5rvfnpGhHqFDdWjU6qGt65KKkoMmUUXujyFq0PX\\\\\\\/HRCU2kqbZ9PuOgp5KEVLTN\\\\\\\/FVA3Ddd5jqUQIhLYavLw6XcXnfp3trQKMlQ8IfUVrTvtKiJlHjB8lNiZpHcld5dHfm6zOfvX9889ncjQN8H6+vx6e+EQn+GKTfpMio\\\\\\\/PZPNPcFqkFIkHjVzO+A3yDtV3OiE\\\\\\\/1f88dpwCOEurSA2hSGLJrkS3Eo8bMJiwLO7vFn2SfNQ\\\\\\\/I1TKgHViH0iPu65c3oYC3Pl9kR\\\\\\\/4xzyjE0AsByem0w0SnSp9BLZD1dqrPX1cU697LuOrYnaEpMYkWFbijRLMXMBzS55jSfiCec2CY2v2+eB6JW+Vo3pwaFQ8UacHDI2HRd\\\\\\\/Z7ttAI+G5II4VYWneeIo1bsMbo3l\\\\\\\/yoWhYda1XPDqfUr9OoOPORwZ+sw\\\\\\\/13ev8QYNeFaaiHOSBE41VHFAcmD\\\\\\\/mIBlkOvF7mK6\\\\\\\/Pmi0349\\\\\\\/EASyuxQOvAk22OvAB6+Z0FhNeG9waq5o0MW+sMW8noywI3ZALSe7zoq5kvCxam7oXaXEmeypWk7i3vc6oBu2VKJbqNeI64Mk+TRa7wnQu\\\\\\\/65\\\\\\\/qPql2rSgBEwBAtxBn8y+Z\\\\\\\/K1xZMyK5Pka4DwCGJSibIVMgEg4r\\\\\\\/3xZoHW33btyEv2h\\\\\\\/mY6YWOkMVuQYA6nfMgY\\\\\\\/vvGlEt+GrPK3jTslW1b5C++y8kD4P2MMCb4J4cS1z25Z11odDf\\\\\\\/TS8a63AwqQCT4OZUDHHhP2NVgNlY6k5vuWaNR+dTrunoMkSYs3pt8G+XeXE40I7Xz0OqVFBndYzD56Lvbafy523CO\\\\\\\/x7eXqDOmXIn1NLuaw0+oRiK3aEBfoMdFC0iplc6ZiprItwtcqCKX6YJYWVA3rIPP2gKMQfoB1nnsTx6vCr7muqrIr+i7HPFPSUWVSGKwA2eGj07tWaaMFZaMckUd1VLxOGSW5Sn4fDmb5ObLU69FVG+eVQI6BIHfU5f8O9xdoC5qFp8th+YqIG1LGpZ+O06fei70k\\\\\\\/8aWomkkw01jvwGNtQGUq57wnlN6i8kvoyJLtUReWWkiVq4RG4Ro\\\\\\\/Am43hcjFsWe+6FYx7z2smDFapUA6o1KvEEOnkt+LuT\\\\\\\/RCFojvEejojuE\\\\\\\/gw5L\\\\\\\/cNuOwCFU1x6KKokKrrRxeIIIm9BqM5oQuswxObanr0byTOJE4veDdWsoWqYbitz+vKvHnq1\\\\\\\/\\\\\\\/Lykbhk9ap6hWRv\\\\\\\/BsIKrDb+5q0NtZ3gglQcTr2proDVAAzDeH7xx6mYxfWXZFX430VR9xuQtClpJDwpm3N6s7q4hw9NMpKYErYszPC5Dly3dWQWz7dTo\\\\\\\/IxNVdcSHkrYd\\\\\\\/bogjNMtxwOtQTuTn6Qk04+2ANuP+P7aPnc52+HtHcTGnuFF549Jtr1kWWeG4LrmVUUjY1CStnVVSpZB7+U6robuNx3iB7w\\\\\\\/XuANz\\\\\\\/IxIWd3XBmjSEYwmS4uKPsT8QcAHxR9rcSceUPMU6o81eo\\\\\\\/A5VUyhfIaRr60I1t5POVIv1J0JD66FdD0C5LEQrjXfjvocYdJ75BEC+XP1HswKs41ypAfOd1AziqR3m\\\\\\\/XCieX9UqKHxD8zlYg3BmedF3JgRAElnY6NT7Zzm\\\\\\\/js9VL9ap4kNYlcnnvbcrvJzLD8BqY\\\\\\\/k033+PqGrVkTpBsJ13\\\\\\\/GGZftVjfON58KLbHoCyr8kkMjHUSbpHpl4+rNSSKBYkvA7VSwxI3usRIGaiiTc87OEyGvIA9L\\\\\\\/D1N+K1CxINBkkFt9J1UDZP205dYu+Lfl9g4znKRAi0SjWgKrYPf8qrpOMjyqhdDxYkrLvSkzlutue8wl8fcqZCiT94LFl2Ed7v7u4i2\\\\\\\/HAAjb4J6Vay1gEuSGbLF5Tdn8eD\\\\\\\/ClA3UJkWdwHccwBzftxHD+GZqw7IlWYaYwddlth131savDTJj\\\\\\\/NPOnwPHk+mVnLGbrm0zQ2SjA\\\\\\\/k\\\\\\\/ORuuNDdj8XfownqBJvm+NhyK6P5hQf54c1ZHtu1aVfV+CCvgunhBRBI8WhS+uN5p7\\\\\\\/bg1S\\\\\\\/h4Etqr17P5q85MAyPhDAEPcCe4+mklMGTl6rcc0NJzA43dgXOXOC3Q6ZUDuFA+LdEhfURYVIP\\\\\\\/kRCVhWiwKCtB7x5MsbFYFNVjyVxd+gNIiYz5lnF7s5fS3+vV6vh2KbCCmfR6l5g5zTckerE5DS9eMwQiJbl9KEz1ZanIBMc14wqWWVaBQ+1DAyYYsjUkXq6MVrIyThX91ZkWXqnEsS\\\\\\\/Oobt8CK0DjVypssNLuBufpQGAahmhqgHUA5OuTVT\\\\\\\/E+rz7Ou49VJWvbtB01NGVxuGxCb4rXQ+trNmH1QcHCoBgXaRYpIsDySK244xRk7WTmm3BWPwxJuO2gj8zwvAS+x4aVhWgwP1bdHggRwBBV8h4NOTlL1ES9qMdz96\\\\\\\/H9I5c7TE0SrR\\\\\\\/hhpACq+R9Oiwq10vEbsugT8vy9cEnLzO39F7ArWw1mep3DwTkfC0Mp7GULOt0GP2zsnpXynEXcxCG7Pybl4cq8VXfiDR7ftdV3wumakZm6KRHLlSc+MjMIIScUf8WvxkX36WU8CEpEw2vQHwaqZ90GhFub4dw3PSkXnzca\\\\\\\/2zO8uRMKj\\\\\\\/\\\\\\\/hN95GbqKSLHWBOTimQiYiVUifhkKsyY\\\\\\\/ohu3qcEEpi3GZ3+b2wrqTsQxabTMTh+zEHdlJ3jXewulFto3tx7tB\\\\\\\/+gbtHbuCnxzo77xuUg7LSF5\\\\\\\/eWeO8ABi\\\\\\\/MVeVpx5lu2rw0aKs+Shy1Dv7Dxdtu9LUZoauDUtR+0JGSdosBv4YIjYJDhkOuEvdO2iiFqxLtgz6uPst6NlDUpUF8V7vAVCI3BdkqoEjbPAYlNVqcY\\\\\\\/X\\\\\\\/Ecc8vyqf0i304akhK\\\\\\\/N2\\\\\\\/2M4QG94uuwZq2EG4GDofx0lqjKOibNbmG1NVmGzTVYL1+YfX4dThrRDj05pKRn7VB3lOzyUCsaQ9YvFzTYFExzk8CrkhY6uXuwuz\\\\\\\/tz2VGTIDh101gAUAyE2qUwKaDK7KsThZIMNxw2URDIyfPazNcFlGBZ3vVTyjwdYqRQGS074VeiHxI3Ri22K2G7Sg4fmxM5rog8Ijujfz3VAAt43BWmeWfK3LiqGqLHm4BG6ppGxHouhxEvkIqcwkL6o8utMGwpYZbCA6X8eTPC2KcWc50AP6rAjdTWK5qhsd6MCFDFoHj3HzsjvpG5Qmr4kOaYsLyFZatV8K3ysqcsR6N2aZ00LTliVSwUJn2DQ9slAQ42Eum8H0HSlABjDVVCRKxtX2La5fMRMBFcaRMvddINnwUXWv8oBKekG83fuJq4FyKNu4ya6SS9mKW6+r\\\\\\\/AMJjHosJnP1p5gncz0c7GWt+zY5K5oOjebdUwOYTJN9j0xSXgNKZrI8JfG8m1X1t9uCRZe5iHThwWbLxQm\\\\\\\/VjE\\\\\\\/75ehEp4mTvM2m\\\\\\\/N3gr3JuOn7SnrNi8JZFUZvnOlKW7RfnAx8sxQmEeNYAYkyebSQeuhxYVkUkP3V1lOeXO9ZxZP3ZbGNEFyLmHUta0KcdPShRdJmWa6jzH2DUS8CXkiBw\\\\\\\/79iSpEy8lmrswoVkvCvfcydNhhWx2Vn7oGcPcdMizjH7\\\\\\\/FybBH27G19+SHAQKrsxnFEIalrqM63NB4bLvh\\\\\\\/qlPBSduPVBdWMbsCvTbPj3uAfXaC8OLVybRdrFWmhd1gZtJXnxVHE67twcIpGCwF9KnvmaTKQ8xN04NnYmDpfYcDltWdBAnOMHLrEUq61Hrb3oh0etBxDk9IbIuC2JF90X7W7rrlhBAvBmcfKdSq2WRbUmWM800D2uRsePI3bwOnlTwe9KpEzJQgCfMpO58MzeiKI5Pt\\\\\\\/HL5W0HfCVTqc3gbFk3boHkNwI38XtJTQvkaQOO6X0czjpXS4\\\\\\\/QAOnUHVTwW6qKyugMbzmEbE\\\\\\\/B\\\\\\\/b8HW60djK2FcxYIdGj89zwKya5XKNdGWMzUtTnb5sfkwlqFP4V8ljX6w8FJQ09HRpO9eCMCLp1+Qa1mGrR1JwdkWeqYjl5m9B4TQxdaYE0dTpn5RM24BaVrcvR+lr6d3M8i+LQJVmfLvyHs5v7yoU7F1Dt05wisvDHs1kbkuq40A0IxQaOaDN7RUMQ\\\\\\\/ZUMlPYsQG+EgTWBYdwXPGs1LhxLNwT5E9OLvn2z8zXUX4jfgcyoIrKjiWMxu0uhytQX3XUL0pWrVSE09H\\\\\\\/WkP2WLzHBD3z1mk4XJVfhFjXN6fpRrCwiLM04rwcjUrxcbfV7qM+ECu1MIh+QlOTZbw3EvDrrc7a4j0I0WQViwxC0wyCkDH844684xlM2FDQUF7l+i5COQSX6yWhUW\\\\\\\/HU7Qu\\\\\\\/Hsv0hxJANaOoHJs35DcoCuRXjr6zHUDxuUuFrOBCvdUXEESOuZgGOoWtHMWgC2o9nOKsr\\\\\\\/ZlOyu+NJjJOrTkygaXSTRR6Nzuz2AnY2WIDMWKBHpkyBzit6WGBaTowudz0OKXYfr03shj3TvrEw6xhnJyUXaYPxl2nlq\\\\\\\/eYfi08J6svEw3MlK+6G6igi+7yVy\\\\\\\/hJxnAFNFK\\\\\\\/t95T+bb9ZzuOh3X7hpNFO7E1xiv4Jnc+GG4TzC1gecRegnTpdpQ3DD3cTblesciUsaEPoDRge4H1y9Mef4MXhkPqQTKxoiPdBYQ1EuhLtDSe73kTx0nvhV8\\\\\\\/SeT4+aLl3qxpUtyQWohPMGhblWFUNSpqI1tMo2XeAQaGUhXraM9bO8rCx9H\\\\\\\/iELgdHRid18KmpD29pF69gBVUxOypI7MoC2hEp7irwMG9hKwRuEylgcCUoU46SJI9t7F7lsX\\\\\\\/ypvbl8di61aFasxN7nJ7IVZLhicGPYmT+VBPC0XT5ffAT0+5QLacGP90Qxv7ymcXn1lyYFLjeedfSoalGUN8g6eoNn5kJ6ZTQgFiTmuGG61pCZu6JRKwQP7Ms1rZoiso8CknNRYznoohrsDVqaWjspi5OG0j8wjoBAs7L1F2fdrq1wwIjjruZx5ErlcSNktXi2aW6\\\\\\\/FUdyWo3ZMJ7vdJrjluddtT2GT7qJ1tuf17rRAq404QH\\\\\\\/A2zr6nDOOPdVy3uXssA+UzHm1WburYZiQqawH\\\\\\\/DVPsb9IEkfSJQh+giRh2sqq7kzvYQU5TtrgIYT1le\\\\\\\/FRipaiuvPo5dMufm8uqra4o3cmi1Bjb8CFewP8IjmYdJRG86wtb9q3HdaIjK8WNuuIlHP2otqfsuIPTBuHAlQK4KI38lgdfKAWYCmBZl\\\\\\\/GTrZ+8wAY26Xqffd3d35yxAOpMoAjzkMJA7ST\\\\\\\/QAv4zAsrY\\\\\\\/8xYjJxQ3WvZWcqWT8rm\\\\\\\/L2CZasSxnLfjXYYhHutwmzJBCiJeP7wP\\\\\\\/m3c11xz9m3EhCNZQhHmBI8OEhoh231MhCSRQ+3RSto38CKcR0iPLeq2\\\\\\\/NIA0XQQHuF\\\\\\\/W81gvGCVARKR6R+8h17Gm7\\\\\\\/KpoXDO0GuWDNb7OSjGcyFglt3zH+lw+ZI3+eNmHqfjCErEsn8Ckz3O2qSJk3+qin4dZq+4BJ\\\\\\\/nBGFghodlkV48ImrpzlYvqy3DzYyEau\\\\\\\/fIpaZUOnFslItHIlAY2Wz18EDGD4XPQnmoBeHQbCM1eMuVbXbgTSHPTA0AWYa5zfhr0EXmaXdDvrEMEe6kXnptJJmfmbEgDwsh4onO8puY5SGhQihx2iWf4i0\\\\\\\/hPG4sTNOHW8Ia0BN55bj86n9XN\\\\\\\/8CDFqOfDuqBUGFjWC+OKbbxDVJ9xoF5npEFAXToCMVv5VWTUQ6sWFFvibsQBMVKyxix6PKqKJnlM4Ep96OpXJIzAyi1auauVs+hxIN1wmIsfwEboOanPC1xWq46zDPtQwrQJw+5MeL\\\\\\\/gDjx6F3G5VoYFQBiTYJ3OQIQoOjeg1WUQDVENdYSV5R\\\\\\\/dBd7oK0SUPC6+lTuE99aHfB9SuR4ct3oQHtlQFpp0+lgeXyf\\\\\\\/cRzy0FZIjarwa6tttSqgyTqb9ryhud1DCm++hEZJRQ3G\\\\\\\/+gHJaXLGqBJ9jidZiAjZs853fh06B318epHuCK4tupnc4dQLLeBVpBYW\\\\\\\/WLPyrvjtpU2XrpL5kwpyD2zELHIHmLKIFcQAWUn\\\\\\\/bPaRqJ607WgVNxwTZO7j+T2V+vjcQy0pPycTbSAQyHwPXjmOTQSUj7XvDPn+Swewb4RmVfmSRhVY0e4rOgLhjkVGAsH\\\\\\\/An0jl75fay4eC0i4NjyiTHwMrzrNc5eRJf1wecSO\\\\\\\/RY0p7n7EywkvOCnuhS\\\\\\\/ewV6Clv7it\\\\\\\/KDmg9XhxhfhYzA1aBi+ljKqfLzLm+pautLp4yUXmBbWuwW35e\\\\\\\/LuuV367U23nWQ5PaKKyBGll8cxS5n2Ts9NmUKpJ3wAuUqg7fVvPNVQoi4e9MnHKDHqldxv1KEY7cpfBjQ6brCxB5Xc7Gt7FpB1wMM3iuynTdFQXAJTXLeDtI48TiNxMhfq3YMRs5R3PBMUCGSCoLKISbZVfJfPlxn9ai0vZBtycuUWZGd4Dts+1Xql9jq1PecziswK2dg589LI4mrhsp1ZGKJbOnftDAn1hkbZygQOgEAsoitQ47p\\\\\\\/xYsNGnB7itkaovnjqj5\\\\\\\/7Yx3XmlioIw17\\\\\\\/HvN+pTVOcpK4qbB4xCwpK1orRWJip+SDYt+e0\\\\\\\/kHptjR9C5FM7fhK5xE04y\\\\\\\/kpK8kkGw5N5XpVBJk7c5XRb7zLpfKHMhxrk1TrIB+Mkt3JNhK634g2nFMM2TObvN4vRRJWZTne98VmjwchpLlOKGxL\\\\\\\/ySh8sqBtP15LuXD6vYQ9ZHmvgelUdnmxsKt4A4oEukKdZgtm6Pcq\\\\\\\/EfgzilkfYQ9RnoVPO79rZj\\\\\\\/cwswopQ32KTSSx8b+qhnRzJjrZE9bBnSG+79clw6xbjr0BGs9nlDyUoZn\\\\\\\/HdjmVEmdwGveIrAdaioWQmOZIe5OhntgnD9nQgShZDF1wk\\\\\\\/4beafsLbxHN782j6hAHd8Oy7rgJPSFR9tDeXTI5ZKSTyZM4RjW0p7+fZZkeJ6pqiyKS640sxijwjIqeqVD49F8R5xUJ4qM0cDKwv3TYX5eOIqyKnP32reDIZMgPzbkWjIPFj\\\\\\\/Or++THFBA5O00Zy94IhzX0vj4Kdt5rflJWamNzIrTnKVWFRDw5dTEFLeCmfN6O\\\\\\\/M5HtqtaCFgGnutAZ5G83a8L2YAvAi3UvjAte1RugYiYOgHDw5h42zZk5k1OUi8P5HXyRQO+fd2NnCo91XLl\\\\\\\/+QHY24zelE\\\\\\\/m4\\\\\\\/wZ\\\\\\\/yiw+m8C\\\\\\\/mgv17moa0RZyX3zrFERyX8qbri1Zu8BAU5J7CkwfMThShvejUXDHiIJ2xqmxxirvHuM03kOSkvJPDgG++F+URGf34OSeHJdmTWVmadkwilg+clYekm8YqoeGPujLa83hbuESCfLpDGa0dym7yFfOgvvLtJYgO5SSrVOZ+UEZQ8JeqzagRn7sPa+otS4+KU5uR\\\\\\\/9EIobAIN1I4FzRXV6tUcs9eEgyJSbbGOCcnIRpHvt\\\\\\\/T9sTs9uWDxarYU\\\\\\\/sTBbT5rjvUL1cRPN4NFYoOywWUWsyXIGMHB7nYtT4yrqA\\\\\\\/SGMLNjy3vW52QhZyl2Y\\\\\\\/hj3BX5bL7MsalHXmz5LWMusL53fT0QMeh0lycYFKXjmdNbCbXA07fLucLkPQBdrpDT0YlnIqYZLrDfay99dXbpEtVfyYoOmJJ2vTlFs59CRTlmbVQ7wVCU1D4bBcw2ynaymWGrmiRHpgIxhzaYa4w8xsihbZoFh63B6y4OZyc7zUPzN671qyWvJOTPjAAoY3z8NaZq1hSQPH6GCJ+wRCPjkIBffNaFTv44KZTkKnNwNxjQdF05W2+FgCklphRrn2tzNKkTrjWRQffOatjeRszuwUVPCQfxCiDL4XQQpbtG4Vjk07LQJM9RAj+Ufeg6Qn7w3PWpMUCEr392UCIZxV1NeJfSpJ6sapeOyoyTOXmYdHjvbUHIMKgWBd5RLaa3PG3atcNBc1A3dHPQQyeP4OhUU9Qs12wZMkES63gQgpx5BbtHpZ1Lku5L+6RBWKsGGVGqS+0M9kMY3iDD0Ajy21+0iWrD7uz7XnBY+0BIYh+wsCJVzUIIlASqgKsVC8LtSOFL3oWatUhD0ESdrSzraDNI2b1ZSA1zs5q1vUhCclz7rUvo4HbDDNQtwpG0QwHWEI7fWZjHAAOOnEwbtldJAKIna8zyfg9zrhZ1g4oreDSy\\\\\\\/f47MUqnnTo6VyrUrjoHHnMX8mEqmO9tkTzvkUGlpAFx6lxSNMgpiaqXTj8xE4XQtbeJ532ecoA\\\\\\\/cBccG64ofsP0A1ohWgfZfmiupIasyf0I9pdHhtWgJfrh041rnZ4mzTvHUs5bKY1TTdkuE4+Z98jBoE1ow2cTg5dtVVSVBLwQVQ55gskkD8UO4DPFeAV6DeE9XjlrybXwWu2koLxzRmPAphjiSeRk97vX9k8gy93kqaKVQt6TNsEQ4LbK9wUl1s4X8SUoDgAVcSIYE8yys1R7CB755Y7lndSbGGTmJHOeqNN8odC3m2hSVDUlpcb1uv8pe6ubzuspeeBCwH8Ul2JsmKCUE\\\\\\\/rLpcv1\\\\\\\/0ayN7Ux3zNHnZ1DbjYt1LahD377KWFBGGQqdu026gkIyu+6AK8qrbTyW8BxzkDge53ahix\\\\\\\/xT\\\\\\\/MqZQE+2PhEcuZSCKiDMzAfHnWj816kaoGoeNdFfSuun1B62ZNfGOIvEYyUhr9jX5euFrYWXw0dqQzqM54P7ZfjeMj\\\\\\\/UQrzHMhfXqQ8F+icM693OXhjwzvZO45VN1mzW7ioWak2smK3+gnT7Nm7bBU\\\\\\\/Tab0PYDY1vtfVRhhP4PgiAUDyMuv92wZQXZPupSSjgaXDmN9dpor4CKRKJefo6IMzmvZIwBR7bcFmKebHjO3GvCiUZMValtm6u6576eDRfvIuDNXgP1zKQ94ZZ0Nq4Z4VbFKNI0bP+Q5J5SOC2egr6tkB3nwv4GysItLl1ERpB95zhmGe20+uBWIxKTtIjxFNhxrW459pikn5mYsrgvvH4ELvd1TXegE+\\\\\\\/oLOcWHHN6iaRQ9nNlrNvBpbb94Ii9kPZMrHAq5w+QGL26IXF5axlc7PRVVMzM8BtfHH01mgIN0C9rTyuxYVFz2jsNsSRzkMvi1svYLYkb80Hx27W69LW+dg9zpPmw\\\\\\\/aeWv6xAF2l6QKh44kxgOmvB4y4l6Arz02tt6PlW3Ob9amEqapQ2vzvzn1EUrTBLBADeLCt0+nQstHaDrXwpARQjRjWLUWYmaFPxgg6GSQAUF1qWZpXy8GJVDnhRmAyVd7owomKsBF8ua6tDmukBzconudBL3RqTGZiBAJcL29kZ2dQ3KSxQGSmsePArtncE03LBUdt17syQa62OLrSbnkbO6S+1BH2ibEAhglXhSAoC66Zlh5oFEZPCezjdS4QK0dd0EN9E23+T5iEZrevp\\\\\\\/o\\\\\\\/CFI9j3FT5Q4E2zNzV0SO7PK0cCUBOsql6HCBvxYWqOlgU\\\\\\\/3ACyhhVZOTrP6OBq0JPzQAuLpKbqj1vewwXkIbI3Hs26WTZDMGOZFY0Op3qBQVa+rjixIrAX3r\\\\\\\/PIN6KDXhWdrnEHrB+4SAxcw\\\\\\\/+DoD\\\\\\\/VnlgJXNXH3TJXzJJS7KXl+CuKKDxtbM52lkGUxNUJHB7LxRFnMvza3fRIjMYifNTemiW0pMLR5kHPYfWkobtYWN+k5Q0m8FX3rXTicly74HB+2H48XNpK34LyzVR0KiNMoLKAtdvcZka9lcrYSXmGPuDXz5ESLShzZ2\\\\\\\/LMG1Qu91C1Ypfud6B8ygW\\\\\\\/02pJ5hKaeb+c\\\\\\\/uVemD0RowOB7+v608TAz2+MyqXVwGJ8n\\\\\\\/GJDX98YQLrq23baxFzcUMi95ku\\\\\\\/WI4cXGwIwOZWD4VFtBDi6WJWP\\\\\\\/cABuBqy0AEp41Qa7iQIFf+EEY+\\\\\\\/QWtxINePvuHzuxn89CmdVgOjPgw\\\\\\\/CmkDiSyrcqitzev+JmgFzXTsIfNhC5ihWyD4WVHvpDInDVtH2fjqvTXRLlxwyqGxA49DULgpYcsJmhIr4UWHKbN+c9xZ\\\\\\\/e8KBvPYo2FAd6gJUUwBWjzEYFzMuLq630596BIxdjc1o3DgJbDqhyVCAFAKPKmRi1HPW31SYps7uxKp5bcSAsS07QsroOr\\\\\\\/tTHeRd9nt0cVAxO2gL8NG0TiYdhe8viJwna3j0LMlP8bp+eV8sG5vJVykhrBkHJdLSV43m+Yz8xQ8HicWCsjAcL5GmuKvLhwzoYAW17NvulTBrZMIbg6ql9Lw54MLMjm1XyovZcjdP7UnE8mc7gPdlZeVXtd2f23QEziWF1lipN7gxBK0FPqrUZ1\\\\\\\/I1ZxNmFOOrSVOaUqRml0Ekw4L2GsU4664ugTKnQtiLO5UVpg0uWM1lzeCcsQjm70dnA4ibVaYQJI6u3N1yaaGFleFMuGiJ8xUy+TrJ9+FeRRHkBcC7buRSlvRZBfBdROy+ygGRePawIH1aDwyc1WCciFB\\\\\\\/FkgSwnPqCMIxCaEdGgDILIzaqLPiA8mAtitT7MB1sDzvd8+Qcv\\\\\\\/VJys3JVaQi2D\\\\\\\/MG3nW29bDiyPoRyoY\\\\\\\/ux8JyXGQTo8yxlMSAQRvzkKsJ2uMMn8k57q75ggV6SqhObl64yKvYIsO0PCXbM+Hdl7PIFGz+Ghm7KqDF1rxZu7qkevuKHqltaFeeY0b4emvVXMNiPj1qHfDHQJ9IR+igxs4DvRLyvM22Z0ZUn4pRoUBY+73tKUn\\\\\\\/jjHQCcBTG5fck6rxtzteydXaUkdOXwdXokwRTqELhYEwky\\\\\\\/0p\\\\\\\/Ez\\\\\\\/TgMYlVBIVIXV5V4zxXk23Am04E6rw+ByOI5GmmyeVxA3zL3p4jnGy5\\\\\\\/\\\\\\\/Nt8Rbo34z2PPgz0Rc85tqw2uHATIWnXttoJo8g3HXV66jNXrnvTjI+DBOOoNuB80JmJ2N9\\\\\\\/JAaXI44Ih8e0PIofYpBQa4wPCmYyCjbR0pmf5xAzgpsYbUVYWbzT4e+jRobq5MA+ofzIjhfgc2gKWbzEjizXeJoAqiYEzM2jurWyV8ZDhUlKrE+sETJLbJbqUrXI9o5nTg38CNVALKwAs5jPAANeEPekBlg+77AZPjx1mprvs+c5UHe6ICPIySJSDV+X4s1A1l+YjSptpC7QyX4RFHYM5gEFbVk9xgRffIDlvCC6l\\\\\\\/EH+v7PA4KYME6gGtJTx3kQ3I2PMn9NXvQ5kru2UPul6G5DGG5czKufFzT\\\\\\\/VWZlvNuTvAGs+vhlZCj8FzRdan8+F3F3YtnixCegZ4+xNs2b\\\\\\\/xc8Zq9T7B3n03lt8UPhNe2lRQW5vyoIjqHISFvAI5toYGbyRzyiT6JdGRMF7jEbbhQza4X7udtMg4zjYDCBrgYVWwVPnU8Ny2JTw1Hk4jLFSIsLahU7M6zC\\\\\\\/wvH4N0PfZTme9LqBRMPrLwQKKq\\\\\\\/ByKWjX074DDx90zH7mHw3VkDHsSw8JKmba0bpM7idRLhkL9RY7fmdGQJpBb10G\\\\\\\/T7+VQKuBbHK1\\\\\\\/KsalQLfm4\\\\\\\/fBe1aGV209PConvlMb5g2jlMck3FCfFTHVhukIeyd9Ck+uVaxIa5HEXPJ8kTSmi1A4oW4IvpDKCiVhxILV8\\\\\\\/WjM9jx2aK7lQM1sLVzS0lC0mHbyNFdxg\\\\\\\/xwq4Hi+cbJMpQxk7LWVuPpKImoHW5I3qoTrfICqAEf8aKhd08aeToDQYhCjuoYIMIS161cnfB3tevZGNXwnU84VW+82ngJvonYG4ICTQVXGAu7vNXdrfEOtQcGJVQjsdNLuyYfnzusylSEqVOTSeZNnOEka453LDWuuLAl+9cqfL0Hbm2DWUSzImkGjUofuNAF48WgZMJs49+typ1ZxvXkKKPFuTsIvFeZ45kXCv745WKq25107YwLF18WFfevA+nqr\\\\\\\/rjOgB5f+KeulHDtpN7kQoIzI8WBTXrdZwrbQCYY\\\\\\\/nN8mNjfgzH+XHtJ0qdRn+XnpB2b7z6PtK1ea55AKQtnOnm6Lfc3j4aol66MvWkoSavYl+cQWveBHqVwSji\\\\\\\/0u3as48g01suNOaXmHGmfAmqlbcf0hd7NM8dW33nwKYWeAgXCm\\\\\\\/uxJmUdz9O3p\\\\\\\/weI+w+j4hTxuUY\\\\\\\/s93x5iO+98xb8gZYqnDIq43+mcuLO4C0qVtdLoRCLma5Tb9uIc1m+dHEqKE7St5qn3wf2AGfOwcfO187HhuBXWi5byf+IjS4H1+6GdsGSCkVQiKMz4khPW0riLsuj2MfKIziIc7TaqmR3rYHJCu8lINJVhI+8SIN3S+\\\\\\\/a7W+ybJBzVvFRQuFOEwGll35kU3FEYI3FICfWBjDQUbXoEC8XGUbJhOGHDSSSRJHAu1qAaGSLxKZs4KCE+iU3Hkk5gZ4LDHz6qLfpINrEbMgIXR4GQl+BBO4hx76ZNNG6dRpolWVb2Tdm2u3OBmdBm7FlXidoheZl20jUG7MWhEgmoXUX1kmr08U0aSXbEJLMNaQetq1OuUtw78rYox\\\\\\\/2wgZMU6SLgTEmggkhxO4Xi7Px+47wQh3qcRg+UV4NzigB8JyoW16\\\\\\\/OyrgGyBjIW439xUL6E51JLHpZejZD5IMramXVj4M0MM2tPBhLL6K+nDlFKhfdn3kkKfp+JtOfV7a4d0gyXBVkJM3BqZY784AG3A37twKlmOnE73525OFTkBGkKGuPBxYQvj5rvkyueovQjvIK041sKmzHJR\\\\\\\/f6t0Jd0diVy+OFXZKLBvk\\\\\\\/C8U5lP\\\\\\\/4Y1zSBlw4r7NPAuRtoELJ5kd+jg23TAlY5jejrTc6sfOxeK9Q+nIyBkcLypgJuvAl26mAwbfMVNpuDnpK8WhfRrpjlk30tt6+nTPsJoFEFvojPQynUYIhEs4APlFxm0Z\\\\\\\/ph0gjDlm0eEzTIfBWrgkNPrkzXCv+dvAlnaczHxJO\\\\\\\/v\\\\\\\/sIR74Bm4rARlCb1PlZsL0g1mpL2B93dG49i+Pk5MjN6yM+de6EqdDV\\\\\\\/H1dzG5I72K6q4uzvhTM4NQQ9lSmJZPW5f5ovcEB+jXCRx07Hea2tkWWj58OKzvUBu0EWVTgwYxEABw6QAYBSIvGxsbT0+++vNtBgi+GnVRerXIDChZo+BoBcQoiH3XUw\\\\\\\/RMbaaVRgS+AV42k1yxNkTkQTOBN4KjJjfjSvyYZb4EmL2irGHNfwLhMvd\\\\\\\/6iip6l4zFTVSqfOdMeJRhjwn6B\\\\\\\/3caMSMizYkgF+zXRmpnRsRwQzWI1E42VZ\\\\\\\/85vNE1+TuQIQ+WoXHqA074Hw9VFIT7K4xGTuPAmIBJWfJJO0Gc9uBHbZo1u8USO6xx0g1H7VwEd7LBd0WlOCRt6+DsOUb\\\\\\\/rOpo3laa4I+dEPmtA+RAXzI9jelJbXe0JMvd0HL9jZ5CPPeev7bQNjmdyuuC\\\\\\\/ey6ZhyQ\\\\\\\/wOUOIfQp0raqrQIv5h8C4D9ZJvHbVzYa6Knq3IMmwTGM8xVDQNCwRVBBabeJ87REFpSD4axmAkS1CXCKnnMprNIBxGdNVRc5XqnY2IKyZ\\\\\\\/219J6FhuqIc0Lgup+oRE0uSuaBI6Q0YMQIq9IRVkOXNzhVby54DMI501pmIIVnXGdxAYTz6M9K0qvwyWNBDnOHZI+wZWw2ArLPlcAVKzBN0FX3g1\\\\\\\/OLCX0uAsa8w0hdq5qL3PJqOlHzUb3Ah1J462AT1e7aj0L79hxeBX9CksIucC5U\\\\\\\/fpVWv5pVydopqq8V+HBQv+Yq6E2lMtk2CU7vNJ1kXk3VGf55ODJTMZLvZa8AnwfSUf3b5glvq7l81BPM7pinN2PwqadTMleZkjafuF8SyUmOM\\\\\\\/Id\\\\\\\/fFr0ZsdmnvJ7zTCXOdeuogA959\\\\\\\/RI8oZYpDNnlA0UjZxojsBPg2lDP5c6aEfUFqF0GzGAL\\\\\\\/Ubtzl9SqF6hBp8RaZlhoaPzQnK7Q45P95XVS35E3tKuZGk4Qmc9Y\\\\\\\/FIdp58om\\\\\\\/kad\\\\\\\/pnZuRH7ZcVUKN9HpXR\\\\\\\/2kmD1sRegUhlErB7S\\\\\\\/jByp8Ym+q\\\\\\\/T0t0lK1+rW1CkQPTmc06AxDJ0MRRlMeYQ7GSEKWuH+V6m3eevU5krW7RICe1wYr6Y4xdORkXzFcAw1c2HsHVtpLxmse6Zrx5EoEepLOMw3JqKXE3+RlkFJZYuPJxdicwQAmnwX1o0gIS8ZIJ57u5pJuPUYXaWRmqRjbJ56rMOZJ89VlFq\\\\\\\/HH8ZLq+z\\\\\\\/57\\\\\\\/cCwhYW6fbRw2zLTFtFth0MeOcyKo02ua6B8uKd8zkbl1uKjb\\\\\\\/M4sbwg64EpgwMRqLg+T2+AbqR4ztNE7NoBK9EE+z1pq+ZGWOhum5C70pK8wR7fF1f26d1uKxwHLQUkDMjxlz1jPub3av5USYefCmoFpHu13VNjYv7Jj5mcR6kSLygaax82k+Ej15HJ3NPOVw1NGo4UdR\\\\\\\/KALhR\\\\\\\/ezDURjI4WvYZj0EcofFiOIevP6WWNG\\\\\\\/ziaKru4hmwT6m\\\\\\\/0bLvyurG+dHWdkecdsSKuQ+AZCbpo94ZTmASRezeI1kIMEL218T1\\\\\\\/\\\\\\\/\\\\\\\/Pini0cqOYAW9oQHyL9PAlFxSQDYD+6dhb2CnfyTSi8vV6\\\\\\\/6f04cLuWUU5gOgfKcKxSp0UrxsLber5jgXzUQ4eMj9G4ZvHCUeLSlqEHUlrlOfFIc4Vr2\\\\\\\/y2Ux3uQzfqYnb5hLSxHMt9TRIQ\\\\\\\/sxGjnEd6bcczjNZyX\\\\\\\/Imu+2f6vxN+BbQbY3EQpf0zrvnJQqs65tn\\\\\\\/e22F9jrrf7SQ1tZr6Yn3x9iiZVUdQMhuC9c2ax6msQpoVPpeY6TKfrn5KGGuClzLgfxD5e6qbMQyAhKLWgrwwNyGfKH7ov1ew7Vpu1d1y7pu5s9Njd8ZIvjYojadqMXzCZSGSDKBICpdf+wZIvFHBxvbiVUEKhUsR0\\\\\\\/0390sOV2rJjNighD5dz9hnzFS3axK0LwrMGT8gvZwS6rERj4WpfDaOCZt4wwfew2PTx3KDZyUlCR8kMD4OFUmdxgKANRH6LE\\\\\\\/z5HRdhOhMfP2llQvopXV3exDzpXG6EkAosMZ6ea4TgoElpGGozf1nTKY+2n5sNbknefQmzUH2+X2LW37n0Vg4zskCJSb4qt9O3xxza\\\\\\\/gDONb7VllExsoLYGUln0NUxLkzWYzuuTalKpqBV+Zz5DC3KQfD\\\\\\\/6z4kZGfGAWbwZJROyA6o6lMQwhE\\\\\\\/Gdw1ZN06fan4qAvdngd8Uua+p9+Io78gfQXhAFx1a5S5sN87T8cP+o\\\\\\\/pChBzbhse7nGLzxzkvhGscDG\\\\\\\/J6HjaS9sKCZkwmDrNmLmbgr9yhPAXhGROfz5W9y4PZnRHkrJbgstYQWwxFbeqLI\\\\\\\/H2jyOwySlF2WZIDNRR1aYlizyH6MAa\\\\\\\/TkvnKA3j66OEn7en5ITw\\\\\\\/3hzdhNDfq1m4SZdIYrx3qqssY905B8fwq7m6ywaj9FdL+nw038oQpVboWcfzHwXMlvFZ+khQN3JMKij1mnd5GKursGzuI6NH1w3zmeptW\\\\\\\/98M3VG24Mgb2\\\\\\\/EaIPdFhNFkd9Z7G7XlAZOcIc0apUxWwV7OmVG4bFAI8tUCP938fDlPTHiaRHAnHNsvBiDxHTADGBAGX8mQwgrgkrvLQNfKLzQFMpwOhAt\\\\\\\/NeEPGS9t4djbg\\\\\\\/YHGwzUgQlxfnUuNa43LwKN7I3ZPJbQhAX0wYVsnKTEkQgt0GsmLeC8dh1W4mvjDEpN0pSEuz7yYt4GxQAE5KPHpblTPWGVIYHGv+2m0nQQLeBlh1tTB5My4PnV7jbJdAV4aTGXWLunGpOK\\\\\\\/L9ZwHcl1n91uvJ21S6N\\\\\\\/XHKmu1HJqF8Zce+L5B8TR77IoByBVAIXC8aH2WwtNh1qIb0\\\\\\\/C+x4N5Z9o+AxE+VI\\\\\\\/SvvO5eLFCfi0wbUvnH372GVNWTI8pS0Qnm2TiiW73cM7iLLwb12MELVLv+tnqUJTLK3Rg8AIONnICGpPer40fUu0IUEdtWFoGJYUzRFklk5etP4J+V2ab7IM5Bblss3hBvjLtdbuSFlFNfb9x+V7hQApGYQqjgn6L5ZozS5kYYllPrIe4xmxv6PWyzr226S3i5D4gSBFm1uLgrUspk\\\\\\\/nLvA\\\\\\\/YOm5O1w\\\\\\\/9ZMCjhITANo6wsFYgBEQhW9TBKD3oaGY1geUaUZITpGHnWgz4FO+6mTD2FrYIgr2PFyyKzXvyuFcqTJBrYnMm\\\\\\\/spXKT2i7yMjdzznH1a8fdvy0qs61bJN3v9rx7QyHMMGLi2xXK2g9DUm\\\\\\\/Awnuzo7yANeKOzNlORpsD78yGbabMmilya2nyPfdjHJxqH5RdMECfna8Sq+B0KqnfYJNTCu1s45cMikSgPgdJk1HUcdWkfd56YbdDTARPGq\\\\\\\/IQMKlT68t\\\\\\\/xxEd9Wcy\\\\\\\/HWO114vx6odNEl08JJ7MXwUzLwhSksq6XdA5bnJaA5EEY3gEeqkGPPugGmip\\\\\\\/Fa91ay6b5pesnVD6qzzCuJ1WQ20NMc+ZG94Q7BW5lgAi++q1WMd9S0xiszfZ0Op1cNEj6XKp\\\\\\\/y0OzndeBV4HdWK3NqG6+zdKbW0jqiCI201\\\\\\\/qaaZcIsrGdn\\\\\\\/fxMUAjaz5Vk678RbZIhCnQBInQdGi09Fu01llIfgBZvlc1WliZKi5FVsNQpUTnXUUlmNPVGQMsMKVTI8AYReomcUTxRIhhFZ6C8na+N+j+WDuoD6RUZDL4xD14pUKEisbCZN8Qd9hXtzKCQu4kwHAL8D8jV5+fhiUmwMknd\\\\\\\/raykjxjgsYYbeKHDAPD8e4p0LQUfebjeA9\\\\\\\/prix\\\\\\\/guNzfgml9LBqey1oAO+8ALnD5LKYM+oloRgkspPLKH9\\\\\\\/WpLknZw209zshatuJoTpkbKrYVSVPdsxCTWjepIgQ8fOl9844NLt72\\\\\\\/GGliiU8r30bIQpAZTunx9R9IZjLn49CiwSlDdwwT+EIXqIT0\\\\\\\/t7V7xi9Izob23DvfTZ+srucHkVRj3zqPZcJSQtO0A7hRcpR9jS+ZNkybhizA7IM4xZk9ev639yTVP9nsdAUwfqUztWzC58HHvuoEO5EKZ6VhPxtnt065BC+JCofIuGm1t0dMJNfurxjInmfBmZ66msR3ItbeOXcMjFDjuLDUN10WxMmBGjPb4wTJ+GZ2xRYuhbdA\\\\\\\/lyp+JffLSyzqpbYEoI\\\\\\\/jW1g+Kw+qpRsn0q1J+5LQtQ2ia036QfLaYBr6AY3Kgx3pTlDLUzXsM7uMkpby06ECEgz\\\\\\\/ZfoShZFs1YJLWgKOhdWtt8oPkFnn1UXhwDHyst\\\\\\\/OygSiAjxcdyqpTzyB\\\\\\\/fm3Q2K43Zzdo+rK6gaAg5Kw5Hj35RejX9DE2nLrVKAkQJqElSXbfmqZEy6Qdli99wY\\\\\\\/sehQPcBm2is2OqSkrqL4OWz3ACzIND2p+up3acLYH9JfOMUrW4E9Ynvvl2omfqGVYygCJDgGblonxPIKKPsQMdJWwWFWqaxlZ7Pe6GVtYb+G1JtoYxpwmm4gDK6MDwJJB5k+WMemieHnwX9KhHExouJZtxmGBUwF\\\\\\\/ZAI0G\\\\\\\/Cxepk0evpFo6bfvZqfRBFLqARmiQh8rWhb80H237m+TSxgYjLJK9caHX9mweEUfuRYingtmqx2PIzTfvPv8QI7UfGw1aleLomfBLGLpzN2iBFdslLuqRNrboWgVdEgRty+Q1khXIEFq14GJ7OdA7dPnBrAgnnjDus1g5mGuTpj8RH7zpf6\\\\\\\/jQEZe4M9K+stGcVFz8KJ0FIUTPL9qxQW1GEf39WalWH9fN4dlwnK+xk5CA2HdPuyb951OiCcK+RmeZvO8MLwqXK1QheIAV71piQnf\\\\\\\/7DI12V0ljz30taGFwXYXAWEpevtLf3y0L8sVu12blypVCK9AIfI3vEPJFm9kxsS3oAe27O8cXpMzeYLLZ1IKBfLaY4fYBHbzFOKHVJDeM5h36Zr\\\\\\\/oa+onu\\\\\\\/uTQ\\\\\\\/dwrtCiQI1pp\\\\\\\/eho7ymVLlnEJO5sHlKpMRHT8SoKdHepmass2ZP1Hnn2UYgKg5SEheBMBmJpnZW+mCNlmlgI+8MqW4s0Q13xXaa57v5sgv0\\\\\\\/+vdF4KS+KPobSJ7ohOwGWk7oTBbpIbfWNt3q3\\\\\\\/3pNDvBf2QtntlwB2HP5mHIPnqiEhVNtvnU3NpileNRD9GNzNsSerbrEcwyz9NzANcoVtn1xmZJ6Q0ZOwlPXjzLrLS2TssBGObhQDujto6woaEaTGbUkQM5Op5sEDp7EuvvfVZpy493T2NvfobV2UmPGz839XktqaZ9l410YnGnVaLCks4QJaZj5V1FA+tj1yQHa30+2DzcQWbXEYe0HJKol4zyq1USCT+NHMBpMjekjBgHlvCxis0OTld9it\\\\\\\/VTiEH8L7VKDax\\\\\\\/gtdxe9sloUAzC\\\\\\\/A9gQlfmop\\\\\\\/A0BY5rFZGWCY2dY7aDd1P+82\\\\\\\/0Ui\\\\\\\/C9Yq5TNMiMCFMcYkSDJCRyjh8sDtk0N5\\\\\\\/U6eX3q00k5qv9SuyoqgtbHgbCcgntwZX56PDzCEhV8OTyq6+kd+GCtBeBpIPDPr8QkxDGWBx6zau0APOopSRduB43SIh7uxTVWmlkmXgeywcyClX4Ev1RqvxY2abDzsIDawde0jTBqrDYbzLWoAjntr3yp0uFCLHQ1ZZipHKiEENZnsrJxI\\\\\\\/plp9C62HN28kWL55kEIn942DsVFekdx6bS9rARP5pyuxqviQGIDSEMur8KRyjJc9gQENMMXGMFd6GpHUwDRfDhPOxtmGBAsSFHz2HOBIZJMoy+vHx8HwsmzQ7KeiK0nyh0R80\\\\\\\/3RZvRLTIbj0HvbaV8WAztnP6SrtIT\\\\\\\/d3X+u+PZRnbtJfUhUsUKAe2FTRmA+Agk4yvsqsBep6863Izb2+YgDc5c3676i36aBflsyNyITvF6WbUOpi+dNLgaroeZIz8bwxiM6LTznXag1PMviotaYJ0chD3f5WUVg13jSBAQSof9+4270tMTT4v0JaKlRw8Irl8GvfQBrrcGzbE6tEzbstKRA\\\\\\\/7SsElCSJkHpnNzv4SaQLUnR1LTH0QotZXzcFp+uq8z7tncSiascDPQAt7PEJqaj1f45l\\\\\\\/yw8qUNENNuiaekGXav75ZKUUp+SFrOKs1eWLGCAPPeyCYw25J9nII+1CKVbJTQyfMm82hZyJXZVPhpQs9XIEn+ziHRRxcreKZoH3g6dfWbUmi462EyQt5wUoVNGAqjMZEeAC1rzPv1aoc48wL3ULIDb2EVBkK9HHlPIVYKwXAJmLlCU7YvPHBOtavsgDKSs4\\\\\\\/ODjXEaBgTnXJ6Du8IGiKBRObmUNTFVmo1YRiIThCL1Ch0Zhqf7vrqLQDBDny2oXXBfPNzNSf0WOhokxceDPa\\\\\\\/uu5zlKk6RZ9VWZ6ddmQcJT5JAsnvKkBLIkAzeMnEpbDJddzMzefVuXN1eeA1Ij09yRfypLlFTXEeA02Ds3SbG+8JlvAC+peYaWeegtuAYPIZj+r5K+ZyfzFN5P+6Y6rBA6UOSZwvM3ZiLYGf4yqksLPJ\\\\\\\/6MQ7tWxcUbuXQWUsQx8PJw3E2KwhAvG\\\\\\\/bEAoc++u+7GD0dY32CnC5x3RvoeQtKW8Gvme+CNE9ERyz1i+xcX2GzsGOMCwjlC0eSwrXEVjSZwetd+\\\\\\\/20jURAxY21pV3HBS3Nezu8wYPQIJoDlcNCzJKvyWKmdC9Q8IUWtYt1DAIjysF2+0+DEQ7Cop6WT31Rz2wng7pO5\\\\\\\/ux\\\\\\\/qVcWHaqATzfag48AOP1VhTRBQh+\\\\\\\/Jx\\\\\\\/YdwaUSndERwHCa8ffQAC6QGIkTWuXabgYXd8AABkeLujk2i6OxxWbsiuao1aOgqbha+akYznV9ChGMV5OeETauMPUfi5DWTSgeMo8ZhjLYcff5Ol4KOYiSNFsvTA+tiksl4GU26fx3ySlsZYI8pvLgAmcv81rsOVQ2fXDk0qlrrbgcTzoBHxvxJXZiUaxg3uvL6roBZwySOPwEnMVuZDZ4gzeKOP1H7vUGBLe1O47bEWhDa7wIrXeElvShSet\\\\\\\/zyPv8iw97m59N+uv5uzlE32Z0hsfUKVjrTNufy8PjkEJLSHtjYy\\\\\\\/rGSWqFWZ\\\\\\\/bSAVf5A5lqjyzhcRnKocgcSyQLWnVVhR7V9mlxsgtp30DhN7oI+PYED+03knkrx4jM4gCbHK8mk+uQKUmMsBWBAWST2RmMQWbqfXwyFm8y8cfj0CREkoDGALYS56Lm53VV6D+nDbj\\\\\\\/bhJ1fT8dYX2Ju1\\\\\\\/YBHKmueXI5mnMK8QnJGfvE+PTP5BfNTPam14\\\\\\\/ZHlJb6zway7gUyPkGyAYTey1A0IYSpE3Ua3gZ+vMy7bugZ0Yy5sdoF5onwKD32YMkKntG8lEwiZT33COk0lEeudSb9+IwwawSrtdeE5rm2M6KRECweFL8NVuQAFkxu584kTaKcNJHK178TH7JJEKf1AnXLaRD6gxDwKG3A0lscWULPzkw2POpUBUmw+fW4sVd5kPauUL84kF0JY8ge9MEUctpIVelEA57GHgPIvGU2su4dci3712ymt9S96r\\\\\\\/6ly\\\\\\\/mXlPX5jQhT4iZEtn8hGEYnmlTj2je24fBDTGz0ke\\\\\\\/PZNPbFXH\\\\\\\/ygI3eat1OAOkg827cJpdA1zsbnXEYo7l\\\\\\\/c8rAPs0IMcIO+PxRK\\\\\\\/JjFUSfrTCnIXTTsn1ypJGEXoksEKLKrkyX+\\\\\\\/R\\\\\\\/Zm\\\\\\\/8A1M2+FnoAb\\\\\\\/7EZ+DvF+LxDP8YnbrFUHbqsC+24YGRTNy0HJFFDR7SySOIt7XEHz2xpP7Ddf\\\\\\\/2QBQQVhsJdUnuxfyRuRrVotJEjkNflG0HfoQeyFZJPcB\\\\\\\/F336Yxkzx4b5p0Zx0ULmEGk1FadV7ZX5PVIzSf5\\\\\\\/Me4TCiesHBCdkG9Jb4zL88QOo4knwYGf1fS\\\\\\\/7jdxMjsNAu8l2I9IcCf5uY3XKlrdgOj2QW5NCBkU\\\\\\\/UOYOEvZSQvltPtu\\\\\\\/FMYheJV1inqCNY8NQOQKyre02y\\\\\\\/3naTuV9LG7hPvbhllVhiprwvUrJrb2azKVfNlS9aMyK2o9QD56XCW2s6o8E5WPqPckaO\\\\\\\/cPMuiwUQv34wqL\\\\\\\/aiDGnErsU\\\\\\\/pB2M8yB6iVZCSF0P1zpuhtguuG78Mrq2OXBOjd89UY+X7sh5kjq2fmpTN27c\\\\\\\/XWVOte9k5nqkrNTtv2NjZiRthmbrvX0W7TNGhORY1QFPdQ0Y8FMhkpS0aMSMm1kjsWO2FBBdrkm1n9S5bZo2ZOsWpmZyxF7PRsx7LNETHxz5KnvYBB6fTqeYPqqAnMoloGJMZ07dfWtqV\\\\\\\/ESp0B3WMSnlHJ8OjebNp\\\\\\\/\\\\\\\/B0OEoKShE7up4g3dNpvHb1TDqz5MD7Wywj4UfiAbMsyEew5QfFm7HdILVot3YLtvDT0Cpr6YMDl5fyozHeiTJ4AoM3Wdm9lLL95GNqCMKhs5yVzMsO7YOFFhYr+eAhvb9VDoETYLenIS5lylvFD2dNtvxOwoWIZxqAAfQZM5\\\\\\\/vFAvjEi7RuHwTT1J17K6f9GOpdBM4FwQ498YPDdUBbqaN+hHVodc86idmwyd8zdM9CrhwUJA2jH418UBj\\\\\\\/t9hts4sDGNAqYmyENUfYHbOYH6xtasc2K4XRQo1NZ7fdytkm7vawNV7kTVsPiN22\\\\\\\/RbuYpdrSidZrJGPhz3kNS\\\\\\\/C4k8D9CRGR8iW2fn4QmmumvAojkPJJTQ\\\\\\\/eOt18aARsfE9dFOYTbIFBHd9ry+8KfBmVHhWzu26isIooOUDv7GdzIAFXcmAC3vk+9lSo6eA67Xfh5rWkHNPYKor8dTnn6bpaTsVTyebvKKIG1j\\\\\\\/YVdwLg7u\\\\\\\/4NUR+3RgnYi09Y0mI681LoNYuhScWUNtH1rxmlCMVf+DZVE\\\\\\\/tBIi8B4mJoFHUhsi9uVMsXmMW0TJvhDJiA03rB6\\\\\\\/aJZ\\\\\\\/sR0POy03l2NhtQR7TzFuu5A18SqrlPUZsbLHOPBBQFcJOvL8T5PBNLbR6vMKEEaUnCPowKSuXdTRwTa\\\\\\\/qHhZerD4lKcmBZj74AC0MExKNZTKJRxNSLwTo+o2BDrVR+afgXXKP07onTCyze1J9xK1Iqh\\\\\\\/raunWQDNGpFwtONsvWoowTQJsEP9ZAEOTQmGe\\\\\\\/ThJyai5\\\\\\\/RZiTJ4Tm6G4cIDC9Zy\\\\\\\/6WJ5JCxmWomS3b6xgkGZP7+RgHWYdiQHjFiEJIzlc1QrzVKwemhqsTk7OVIK2KhRgIKLpZVmV2j7yCna1UdtgAk+Hq5H6gieaKSAOSz55Tw9\\\\\\\/Iqp9OXn5a1QvbJ6SPFEqMskGnUAN48D8KRWcRWyPWXLX5XJ5iFKEbO6hXOYbga9xzY\\\\\\\/XRvAueqS5Aktkev2B8OYUFTAfGITH73W8zAn\\\\\\\/QTUo28h6DMh7JOVT\\\\\\\/FuQJEcPmtnYMjD+xILD+SmXOrjuN4IhieuN+14Rr27NmLBm6Ywv2Yju9PbSZ+v3Bf7jtuJGNOFkGf5SK6nz8i0w7O4OpXKcfNRr1Q5ckGv8j3RuDikGHQjG+9TFqxyvBgsTyjYZzLIkFoLKeKe+dS9\\\\\\\/HmvTbhemNjq3+JA6xduAXwShDXpZfumpXlsfm+DzMDL8EQf9GpWfbr8a6QGS8Brmdf4GQfQ3EkH8mXFtJayPKY4WV4T4xEvN9S++PpqAJ30p3D5dFdAutCHnncmLcMkDyk+xv45KTgaWrMhNUvJoW6\\\\\\\/USli4L2t\\\\\\\/0lV5U7icOSM5D+VpHbV87g\\\\\\\/FbsZa4PN7t\\\\\\\/GSF8D6ol5jp2c1nXVVDQ9nVN5giqTlfTLBWDDxH9hODP6Ha\\\\\\\/WeuSOCiZYHwLBOpYY0pcS3ZueVTymZ7V6AwjORTzGP0TPynr4qINlbwLMTOW\\\\\\\/ZyveRhO2lU6c+31i+6ZObno4dkNK+Xr0jnOx56JxL5bfW2a21YfVop0DLwyuBN+qKUr+xSlIy7mhwQKbDgnx\\\\\\\/8hvNtQcIkbL63lydP12smY84m7BVTsuJ2laFP1x3LZn9veZitRgu6bUbSlyKuJl8Kqe2rvsXOBhXe6WwM0OcvF\\\\\\\/9buQQO025ng69bscnHIfOtxHb6pd3oK0qm+vlK3GrhYwwS5HHjlxP\\\\\\\/zIG31VSUAIPU5bXwmVuxglAJRdpHaGBIJftZnrT9SVW3aeM5b8YZUiL6g8jZNqWdpVC7mMu6IamhpHl6\\\\\\\/oe3N4eL8\\\\\\\/UwFKZSWIeqiDVq2XGFsX6ahmKEh0NdCI7NY6a8MawtjgYI6hUsq8IAr5KPjcqqzF2vN8s9X2tSrcur9gSM5OKFv6IIQC3PLVmU5J952WRtrGJuotKobHtQBN08Mx+yu6ypykSCI0fQ17bV+Fi54szVD3dAAx0l7h5ktLY8yvTuEhB8rJRIKhLhIoj0c\\\\\\\/m3JDHTDSsnHbTdRyBKnoKR2xlhpJvpfAjQADSA5pClWD4IsCSc26B86hL8adm1NLa9nwhD2pdxbogTJcfIhmKIX0tO+AyWmESWgDFClMpoWJEgQGS4W43dBtohUJgsPboad3GhWWiVXi1kv8kZHWbCBXqi9nvzdxAkpPdviM4MY3AtbTaU51W+1SA41SSSUcwjHLzs+3Cfc07U2WNgZOXqDho8Qm\\\\\\\/36+wDEmQxbEa\\\\\\\/tiQguk77rpOc8h5MPnulqHX0DPyir12VnndoGNrieeaddop\\\\\\\/NinTFVhlFzeagICGSbP5vMWiSzu8wVKdvZoD0Nr\\\\\\\/sy5A58Mw4kxRoEV5USgLNgR8I\\\\\\\/0KTCXoio4+fo2XmogKFd3hhG\\\\\\\/c6OSz0ZmGtZwAoaqUTZsdP8b27QWr85ZAY62KcGh9Ixx+OkeuATgf76rOK7VEIVK3JHVoAppPrNLzE4U6trVsJWIRhMA6ku3Oe+0d9fN5\\\\\\\/T7kinssC\\\\\\\/+8ea5X2uB8x53U7EfMJqUTiMLicFuHG1VLlQCrborO02wQIf7YDbwi\\\\\\\/ILb08wevWZ71a8OWQZVWOwYOSRr93IF68Lbv\\\\\\\/HHpR04c00gotntIc+Z4yWJ4PlqIONEPZMOKGnlxBYoy19zVps8Db39NuikVBry05H3sZFsRUezOVzD4LVq2adBMEJNVtYRDp\\\\\\\/Qt7zbrZYLGnsm3Sxjmwy\\\\\\\/1gfIUTHQ7IIuUixhqNkAOC4K5YeEdo562fjEDhVOJyJcNdpoz8OBHYRrZYDQPRF5PV3WaYsxiOwWu6d1ZOVT9WRkoAx6JFmWSGY\\\\\\\/6Qsq+kk8R+8EZx1SmoB698gDCyO2u\\\\\\\/xMBW9SyC6dhY+CG5BS0yMQFEwifiVSW6ogpbZGQjbWw0T4uRJQIFpfQ4A\\\\\\\/4U0v5\\\\\\\/ioTOYaQ6f16SvxyxQxYko398Xn31nJ14wv11aVLkjiO7u3HnvVVUFnAjixkD7RrJa83t3RZrXTDME1PihajC09i7b+4XwNQmhMZ6fIl9Kc6BwsnWF56YUUx1K4GB+ga9BmWJVhRWtmWnomv9KmnhvJrvLk00P6Kosh6q+Mkapsxx\\\\\\\/bUQIx0xJ6V9yu03OlowBc6wzSnEAt0Q08XZbN0TvPvSfSGmzq6NJIW37YuxeYRfPYTp349yfS9yt6T9i89WfLSRy4j03xgMjqdrUMekf83Nry0duy0aZacXdKnHridL83dRQEs6rr8nl3zqrBmZQ941LEssAiYR7Gjv8m\\\\\\\/ZC3Y0nzN6SzDlxfPPMU0EzOg3vkxgUIdtiUUHTDWUlbMjTTzngwUuzmBzSQ7ftMqppErkluh+qwmwPuUOx\\\\\\\/tDjzG8VgZ77arMFFOG41vbrNOW3FOHiuZUFOOtU46KVvdM\\\\\\\/FGMvDQVso+fASdJJmHRRg5xX+6o9SRdlJZynxBqc5ngWsGVV8EL6YJeWlE1RCyStfXC84M3FZjmo3KM7Qb38Vw+RkZNgWemIR322MajGDQCeymzNdabTVFrW+6CHpU7QCEhY\\\\\\\/++UAfeMAYyv1KlH5PbAHeKAASO7eo18a9srpjuJuTr2ZxMBNxK6d5k\\\\\\\/lcDzxKKXwKRsavQotNdbLEHqoVLPlQ8OaEPFbQRbw2UNVj08AEQwVQ0URC47LbMtyt1k\\\\\\\/de9xR30Do6u0SsO4U7L6W0FoaMb75tFRuqPvFACeCxXY59I37ruyUyebnYw36YqpSiGeiebxhOlaYQPfrLvBBOagaCu5ZNLCHJ4YH46AcZ+dFTAVvUXlp3D1N+kc0psruey\\\\\\\/2hGBu7ld30x1YlWTrpQlufXYUpVqugSud1WsgkGWbeHVvtsXAl0tIkbfktb55Iyc36NzEMpKXHT2JUhOmKyjIOP9eljLPHwcZrq3DKpJu4MouFZClLVemhhYd3JZStOzUq45lSg0z0pmsPrUcge83SPGAq1dzSM9wO1YXBKoVT2vDkk8PZPIffVUqAdciQSv4NcYWWBVyOxBPuEqC9trbXuMaUAFUoBJ4x\\\\\\\/oFpSlWWxOU4rz7LHncNplVpaqvk4\\\\\\\/21SZvnb5vYbW7efo1\\\\\\\/xWd4+TCpgeZ41SoBszHIbWLNBDNcVZrtQvHwkcAv79xfEW1YSbJBQP6nwkcz9zpUt4zeULn1NUhoscA8RpvhJzmU83GTgPz0Za1mmwJq2rKMNAtjHCTgaYKHZfaRKblMgwWWEV\\\\\\\/hu9\\\\\\\/3qSpdh0VSWczEvUIdDPZMkipN6uQzMnfJJ\\\\\\\/Gllx6tEnVj2YtzrVlAOeMng3D5nzc5fhFphkjrXCsaMIypuJWBVX1IBJq0ztJ\\\\\\\/0ThK54WzJ9nXt\\\\\\\/x7U\\\\\\\/hcIhTFs46aOIX+\\\\\\\/7JtYF\\\\\\\/LvxRUoNqunpm2lNo9TGsCF4zLMEVJ2Y6X9fZCC+5g3wufFX9Vv0DUCkTM86+LwIvcjiqtS3FeRyrRZOryKkDdirLIuxGTVSFpCHkl3wLRX1mQAxc+rlb7mcUR7s6WINsNc3yVeVi9hqwNaMQyM27\\\\\\\/VqUjW7oq5tZPeSi9LErLADO1Uu3PnUxCF50YbHHNhxLLEpqsJq0MfPL+8VDmk46TZwtB4EgHhZbnG8TgZZj16qppdHuco72fREiEQa+ru4WjCPIiL1Wa8uzQcQP\\\\\\\/wqr7aYmpJMUQYXg8jeh\\\\\\\/UBa\\\\\\\/b4IxkNJnZ+06IbcNoUSwcV3+xtf9JV2zdSK5fN1b\\\\\\\/gmEIAkj7UmIQduzd6XtqFRaNhw4yx6E1pOF+oxJX0jxE4x72x0dUHoxSQfg1HETLYXSvBgfm8HSK3W6LfQ\\\\\\\/fVkvEYgLl\\\\\\\/Hgxxc0RwLSf7JQO9BOPQOZf+i286s8y22D9v9WXseSPqSQIbHdwqVUMWuchrTAH4fiOeeEgJMnOQnZKSvWZVSxS+glpIEgwiUWRoQAkQL350ZB9+\\\\\\\/V516zuujsICM8PT6VQIhmGZgFt16kkkLWf3YRktPJISG6AsDrs7VEPuTJ7F4Axnw2j1LCfVUwQousSY1OZwnbE8sDBP16BBGt5TrASGkj\\\\\\\/+fDK+idVNzDM3VprOLWgc+rU76t4A01l2YaTe2Dv6FSPUPEUEsze7pxzTpHfAdPPhjWcWU8loSTHn7OZX3A4frwHYw4110S64SpZq3cWuXNvBs1V3lUwAUQGyNgvJyaOEIs0tykqf4WLWor+rmAmdFC20tyfie8o77ZYU5iGyyzhZjnmxXS2+zO5HLDTOrnDOMR5fT17WXMQHwnpnOwHfU1Gyk2\\\\\\\/stEtsCVUznH5CWX4TrFrTfYy5hO0qGjIjfIF6T\\\\\\\/We\\\\\\\/a1OKBSIu78n91oHS3B4ntRnLOOWH8xFvnnkeUKc04u+W1EV70KfdietaEtOHeQ7BsXS\\\\\\\/VQ+Ckj7YfvZ5wdOJwnMJzkIYD0X\\\\\\\/AciFEYQMtuvDfRu7hGmPpqAr4u0Go+EYozkkZkv47BktvmS+qKz1a2fhj9lbUPQAhA0de8fEedHOnY5dT2VuxTuSXiA\\\\\\\/Csljyzj9KnrgH2bveDCjMYpaloDLAuex\\\\\\\/hYlL8eVzHEsevw1zLnz37QGmr7G4ObNDnpwKGDzfdzOuJBFc9jMYrqU4A0BKXICPtYvx2bwpVXA1QWlvHaCESvWZv12y8jWP0YfIp+G9kOUT7S2VW8P\\\\\\\/sxDXv6gvQb8ukJHwjuNYHmCiHVmqWqBNO0sKvXIzaERzNwabVeP6F6oyK+mQN2lo+MN8HjOsTN+BPGGvEBzpad68dCyPWKyLvZ8bNVJSGaY\\\\\\\/dJ22PUW6ifQj7IRw7uEXhetQFrDlZbcpLLHIcTIGp9tVv7cVAXoSgHZCPmzoMJYUy3KklMhm1CAWW\\\\\\\/R1ijYDn\\\\\\\/CT91jFFWlstZzwpwnO8h\\\\\\\/QwF\\\\\\\/cQNeB4IPfqyuFVld46K0zGlQ2J553mCUXQfh7M6QEwAROzhmcRq+zat1H4Vg\\\\\\\/jI52KK3aUkK043GqAaqHxaiS3bBlbXR8SnSi8tn4PSRkr8rT1cJmJjp\\\\\\\/EHI4Q2ovVBc4UUi\\\\\\\/5\\\\\\\/lnMmHCfTVtnCvCvFBgw1Kp6OvU4fUS+\\\\\\\/bgVLPjP6aopeP9dbo0n17s8jSr0Nm89T8Zx0vRSvrLjYT81UbLOf2ZR0h7UF46nL7z\\\\\\\/s5p7Je57UinNNCMlYBFFRiT8XEi8t6dvIalp9f9e1sw+dBSB2RRXPLbM3GV70D+qYGqGLwtHoPFYTMceqZH09bumGFiVd\\\\\\\/QLcmGdJqKeyfdu0VwNCXRPcBMu2ciqC1VVucqTBOk+9oO9cluocNI9pDBwhXwNA1r3\\\\\\\/WgzP05eIrPjZWEZ0jinIzq2XcP+\\\\\\\/r7OvwFsq5U\\\\\\\/HIdWwlU1NYHGbAYVtMEFT\\\\\\\/n0XcwscHgT0ykOdck2Xxx18Y8n5EHT660cCC3VX7BBNnfZDE7wng5hiMouEkTOTOSxil8v5THYz0JJBjaAEZqKsMzhBHSGnp+Ygwm\\\\\\\/hTPV4UM5dOPjAas8lFwActpjz0hY6V+WBUWOanxtpRqNV+eKBQ5ypLDqM6qz9aofVWSGGi+v55Qhv69PMvnId4l1k3tJD6i6iMn71JuaZR8DZTXJYtJnkaPwA7Ip7yRocI3oHXOPKd7Yy7ACBwajflWuQe8m+WaWU\\\\\\\/BglOrA6rPG3XqWPM2fjZhy5K\\\\\\\/p5LN50ZcFItLYYKXltCoPM\\\\\\\/UXN2DC7gxXATZs7Vdw3hBkfvRzYWiFZMdR8WiT0zPa8ssabHIf4sTlANs7N5ajDl+7YJRggH5F0TVyCop4Gq\\\\\\\/TI78PPsegZ4qD+tVGZX+LKpBNVF3oB5Mx2SRIbzs4YcsfFoE8NSR8MHe\\\\\\\/Ne4YoFq4UzdPDiIT\\\\\\\/YnzIl89Bnn82+p20oYtyerHiC28VHX\\\\\\\/QusOfki+tWq91cRkO+zQO2gUR4PI0NxL6FBUTAlK5T+nnQGb+s6CLZXPh+vELkPOl6MDq5jMMzrasouFh19MLcXMPmzjVnL6yfOqHBQmpJApboo9\\\\\\\/8LMW8eC6IZy6SEccB0WZl4UixEtb+z4SGdW7jy7WId8v+pXMG1BGCmmPhhrXSjavoeEY96EJq4lwIItnc8OdCwSvOeeQ+639mDKnkJ3xTQDgwGKRPPkAs6o51pyTJ06mmZejSsQ3G557QLKTIr0x8CxRexJS8eHuqeo6dKg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ECFPfw00loVPaTre8qdU2Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228d6352efc4c007485d7bb6ad007be7160235f9d51ebdcadd88e6ea847a6aae08\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dJ8AXBStyX2WCMdsCpkpze5w7CKBvzG91HvEMkcs7XSfvt2I1dcfPPfkk4LNv0fhNqrHBPkJEWRYCzTxlwYtnSM1kF\\\\\\\/x0aCNjjc08ZzuS+KS3Dif8xgEJSivkZE3R4YoOsKdXkSk37jNCY950wbgiXkV1Lm3F67vWrzMF6Sy8BDJvqRqSkugheGZ4hM0cHtqEr92x6LNwIgNjFiF\\\\\\\/3Hsk7G4s+hVSq5BiGRSdsyND4EnmbHmwcEYcwCP2ELwebBTzzdQUNMdNbFJSmQ9Bv2lhM+suORWWNuRMIGzAlW7AyR5aekB4U7nFcpbqbpQq8\\\\\\\/RFi92dF8G\\\\\\\/tx02L3PTImH0Te+ataxQmYPdS\\\\\\\/6Pur00uY1H69g5kzMortUMYS9Vbgp7lkXbessL9Jz+LpJ07WqGaPxaM5Q3tzvEYRxcthFUkIelQYFdzlYhm1QimgR08Yui\\\\\\\/UNnl0\\\\\\\/shYAjHvNLqhYcUV5fG5NNMy3slsX48J5bCDsmhYdOlE01TuqFN0KODodnAdD8Jhn0fCLELnRVM83+rMk+5moY6\\\\\\\/+Hxcgw5JZP3Lxcw1\\\\\\\/aTRld6ATPeD7HyKehJlAOvwS9Uvl9hB+82xoTyxc5pUghohjLgG4RAZ9+lirTNVeKIwbefDNJCBiO5VploPgs5ERasD2Hmgs6isUgKq2ftmk5sL68pJ4UBLII1ZYVIPmnXq+Ku6euPwp0gPkkz5IhZY23ILQ6FeFkn\\\\\\\/AMSdVtYnWpv4BXB7rVOb1qSawBwM9HG40nTG5nDmINmGbGmQEqNHxRKnX9oamxBI44WByQJVBKqLStVZ6MNQ40nmD8XU5sM4ZtggxlN51IE1FeZRkr9lgtmvWwmqQK5U\\\\\\\/a7O+exA+IF\\\\\\\/qwzU2S85H3hE4n9yyHdYXnOj5h6ZxeKSqvXaaYIlFxaE2j1II8F3IUVUJVgjGB3CEoXJMZbjSrptM1Xks9dfKJAfv7QRdwlCeUbLep3psPHRtqUYnB1SK4pBD\\\\\\\/T6GcD6v7mfXZRsPI2YWlYBf+8DIm5iKzuj+cwjRpNUMwlmh7HK5BWUVmuDgh6r7NSvn7CEJYwqTOXtL6L9jSt+p7uy+LZE0cK8UpJYxIh0EA1jOUnyzLzONnq3sZg6LButfqyHoA2+aebG3DlFRSKRRAoxMmvKCiJohtqTodmNbSof3ysH9jd4IiIHZU5BWCZ0S21tLkOFqfmu5FIpiM6iv9aQb6mnUI0R\\\\\\\/NXGCSrrDEE4KD3o99\\\\\\\/0tP4RQYWE6\\\\\\\/0t5dcrYDDzBfJiFdTLXtCvW5Q\\\\\\\/1O53lEGG7gdAVl3h24eYQUPMCnZ5nCoDqlSjy3XyaKfEfyywu4sd8j\\\\\\\/eaO2S2YLOsr57J1mlQXACS7QJifjbfbSDXaS0BkWVkBK6trEcEx7VFgz7Z1Gkcy5PfbSXufomXo3pB3f3Toessvn8K8rbsNbDbyw6g\\\\\\\/1GBVv5A2owcfnk0hL22\\\\\\\/Z3tvSH58dZjwEGa3Y++AfBorA6miCc0TJQFwwT4923ON13zkGC6UK5lWEdwF3J7FFEgBTpE\\\\\\\/Awr+UwmkSTuxKsSJldmtcSePmpIabkfwC5ju\\\\\\\/N1p3tA1px9RpV1VTVF8\\\\\\\/yA7UP\\\\\\\/pe2Hw1qNc9aqUFyb9wTUw+F+HKIvVj4HeYD8rGvbNbRATlyvV+5c71X4enDW6C11Roib463VHjjsXtRyg9G9W9F6aUxeqIswCDTbddaYsR+BXUM9EZ3DimYzgOd\\\\\\\/5JCMh6BQPqrB+\\\\\\\/O+ZZwZmwqMLecEgOF0I5CevlsiwGqraYvaFSzNI2SsH7RyDu3TEAlrAjX\\\\\\\/L86Pia5G0YmzRA2x\\\\\\\/X6hxk1lTCtVHlX7bwaQhw\\\\\\\/7gBsuvrQHfivooVNk1nKOxdHUEZBqCMgIpHlUoxPWcUVc7OzeksTRa37gIgrBwxaU2a9nEGk1CVC2N68nUNZyP9sSPARFE\\\\\\\/IydrlGxeZogJjxYuIT6\\\\\\\/iVVzYKepQ7AI\\\\\\\/A6H3PPiI4YJhKt7jzkk8+MuaIWZGrOXztXeLfVrPOQSq3Utw\\\\\\\/J1zRJrRl0GTOOA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022K9ar5pvK7PDSGJGNsKd1yQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e95cf0cf641986ec2a73688fe78516cd6df0b18716adf514ad865533b0ad1830\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484771 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1913" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-793_DoOgzblBOZZ_88gzjtE7wdrGIErQCzVH2wMVo0A" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1913" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZPS_kLxDFWIjPE4-Ud65a1r_uZM7etILw5AkdU2PMxY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1899" title="Adaptive Roles of SSY1 and SIR3 During Cycles of Growth and Starvation in Saccharomyces cerevisiae Populations Enriched for Quiescent or Nonquiescent Cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1927" title="A Chromosome-Scale Assembly of the Bactrocera cucurbitae Genome Provides Insight to the Genetic Basis of white pupae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484771" id="node484771" data-pisa="ggg;7/6/1913" data-pisa-master="ggg;g3.117.041608" data-apath="/ggg/7/6/1913.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Architecture of a Rice Nested Association Mapping Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christopher A. Fragoso</span>, <span class="highwire-citation-author" data-delta="1">Maria Moreno</span>, <span class="highwire-citation-author" data-delta="2">Zuoheng Wang</span>, <span class="highwire-citation-author" data-delta="3">Christopher Heffelfinger</span>, <span class="highwire-citation-author" data-delta="4">Lady J. Arbelaez</span>, <span class="highwire-citation-author" data-delta="5">John A. Aguirre</span>, <span class="highwire-citation-author" data-delta="6">Natalia Franco</span>, <span class="highwire-citation-author" data-delta="7">Luz E. Romero</span>, <span class="highwire-citation-author" data-delta="8">Karine Labadie</span>, <span class="highwire-citation-author" data-delta="9">Hongyu Zhao</span>, <span class="highwire-citation-author" data-delta="10">Stephen L. Dellaporta</span> and <span class="highwire-citation-author" data-delta="11">Mathias Lorieux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1913-1926;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041608 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484771" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher A. Fragoso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut 06511</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BA.%2BFragoso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fragoso%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BA.%2BFragoso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maria Moreno </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BMoreno%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreno%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BMoreno%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zuoheng Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut 06511</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biostatistics, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zuoheng%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZuoheng%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christopher Heffelfinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BHeffelfinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heffelfinger%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BHeffelfinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lady J. Arbelaez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lady%2BJ.%2BArbelaez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arbelaez%20LJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALady%2BJ.%2BArbelaez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">John A. Aguirre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BA.%2BAguirre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aguirre%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BA.%2BAguirre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Natalia Franco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2BFranco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Franco%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatalia%2BFranco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Luz E. Romero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luz%2BE.%2BRomero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Romero%20LE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuz%2BE.%2BRomero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Karine Labadie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Commissariat à L’énergie Atomique et aux Énergies Alternatives, Institut de Génomique, Genoscope, 91000 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karine%2BLabadie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Labadie%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarine%2BLabadie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Hongyu Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut 06511</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biostatistics, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hongyu%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHongyu%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Stephen L. Dellaporta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BL.%2BDellaporta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dellaporta%20SL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BL.%2BDellaporta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Mathias Lorieux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Diversité, Adaptation, Développement des Plantes Research Unit, Institut de Recherche pour le Développement, F-34394 Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathias%2BLorieux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorieux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMathias%2BLorieux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mathias.lorieux@ird.fr" class="" data-icon-position="" data-hide-link-title="0">mathias.lorieux@ird.fr</a>
<a href="mailto:m.lorieux@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">m.lorieux@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/6/1913" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484771" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1913.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484771" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1913.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484771" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1913.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484771" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/6/1913.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484771" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Describing the genetic diversity in the gene pool of crops will provide breeders with novel resources for varietal improvement. Nested Association Mapping (NAM) populations are uniquely suited for characterizing parental diversity through the shuffling and fixation of parental haplotypes. Here, we describe a set of 1879 rice NAM lines created through the selfing and single-seed descent of F<sub>1</sub> hybrids derived from elite IR64 <em>indica</em> crossed with 10 diverse <em>tropical japonica</em> lines. Genotyping data indicated <em>tropical japonica</em> alleles were captured at every queried locus despite the presence of segregation distortion factors. Several distortion loci were mapped, both shared and unique, among the 10 populations. Using two-point and multi-point genetic map calculations, our datasets achieved the ∼1500 cM expected map size in rice. Finally, we highlighted the utility of the NAM lines for QTL mapping, including joint analysis across the 10 populations, by confirming known QTL locations for the trait days to heading.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/qtl-mapping" class="hw-term hw-article-keyword hw-article-keyword-qtl-mapping" rel="nofollow">QTL mapping</a></li><li class="kwd"><a href="/keyword/computational-biology" class="hw-term hw-article-keyword hw-article-keyword-computational-biology" rel="nofollow">computational biology</a></li><li class="kwd"><a href="/keyword/genetic-maps" class="hw-term hw-article-keyword hw-article-keyword-genetic-maps" rel="nofollow">genetic maps</a></li><li class="kwd"><a href="/keyword/nested-association-mapping" class="hw-term hw-article-keyword hw-article-keyword-nested-association-mapping" rel="nofollow">nested association mapping</a></li><li class="kwd"><a href="/keyword/plant-genomics" class="hw-term hw-article-keyword hw-article-keyword-plant-genomics" rel="nofollow">plant genomics</a></li></ul><p id="p-3">Currently, one in eight people is estimated to be suffering from malnourishment, primarily in developing countries (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Tscharntke <em>et al.</em> 2012</a>). Advances in plant breeding and biotechnology, including increasing the quality and diversity of germplasm, will play an important role in reducing malnourishment and improving food security (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Tester and Langridge 2010</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brennan and Malabayabas 2011</a>). Domesticated rice is the staple of over half of the global population, comprising 50% of the daily caloric intake of 560 million Asians experiencing undernourishment (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Mohanty 2013</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Muthayya <em>et al.</em> 2014</a>). The future of food security, including rice agriculture, faces incredible challenges in the form of climate change, population growth, and increasing standard of living (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Godfray <em>et al.</em> 2010</a>). For example, many areas of lowland tropical Asia suffer from erratic flooding, a trend only to increase with climate change. The introduction of submergence-tolerance into rice varieties vastly improved rice agriculture in many impoverished areas prone to flooding, reaching 3.8 million Asian farmers within 3 yr of its introduction (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ismail <em>et al.</em> 2013</a>). This is a prime example of how identifying new sources of valuable agronomic traits can improve germplasm and, therefore, food security.</p><p id="p-4">Domesticated Asian rice, <em>Oryza sativa</em>, has been subjected to distinct trajectories of domestication and cultivation, resulting in separate reservoirs of genetic diversity. Domestication of the two major clades of rice, the <em>japonica</em> and <em>indica</em> subspecies, began ∼9000 yr ago, in the river valleys of East Asia and (concurrently or possibly afterward) across the foothills of the Himalayas (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Khush 1997</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Garris <em>et al.</em> 2005</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Londo <em>et al.</em> 2006</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kovach <em>et al.</em> 2007</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lin <em>et al.</em> 2007</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Sweeney and McCouch 2007</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Izawa 2008</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Callaway 2014</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Civáň <em>et al.</em> 2015</a>). <em>Tropical japonica</em>, a <em>japonica</em> group, is descended from <em>japonicas</em> brought south to tropical Asia and Indonesia, whereas <em>temperate japonicas</em> were adapted to temperate Asia (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Khush 1997</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Kovach <em>et al.</em> 2007</a>). <em>Indica</em> rice can be divided into <em>indica</em> and <em>aus</em> groups, although there is evidence that <em>aus</em> is descended from a separate wild population (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Civáň <em>et al.</em> 2015</a>). However, the divergence between <em>indica</em> and <em>japonica</em> predates domestication, as each subspecies resulted from sampling separate wild populations that diverged 0.44 MYA (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ma and Bennetzen 2004</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Kovach <em>et al.</em> 2007</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Schatz <em>et al.</em> 2014</a>). As a result, tropical rice diversity has at least two major sources, one found in the <em>indicas</em> and the other in <em>japonicas</em>. Therefore, creating admixture populations between <em>indica</em> and <em>tropical japonica</em> varieties may result in novel combinations of tropical agriculture-adapted alleles. Moreover, traits found in <em>japonicas</em>, especially in <em>tropical japonicas</em>, such as blast resistance, robust panicles, extensive root architecture, and fewer yet sturdier stems, are just some of the features that could be beneficial in the development of improved <em>indica</em> rice varieties (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Guiderdoni <em>et al.</em> 1992</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Peng <em>et al.</em> 1999</a>).</p><p id="p-5">A central goal of mapping populations in plants has been to identify the genetic architecture of agronomic traits present in diverse germplasm. Mapping populations can be used to create novel combinations of parental alleles, fix parental alleles, break apart haplotypes, and to test additive or dominance effects. NAM populations, in particular, feature the development of a series of parallel Recombinant Inbred Lines (RILs), each representing a different “diversity donor” parent crossed with a common “reference” parent (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Yu <em>et al.</em> 2008</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">McMullen <em>et al.</em> 2009</a>). Initial F<sub>1</sub> crosses are self-pollinated by single seed descent for several generations to nearly reach complete homozygosity. Despite extensive recombination, parental alleles and haplotypes become fixed. The power of this population design is through the presence of two sources of recombination: (1) the shuffling of parental alleles over several generations through segregation and genetic recombination, and (2) historical recombination of haplotypes present in the various diversity donors. This combination allows for joint mapping of traits across multiple NAM-RILs, greatly increasing the accuracy and precision of QTL discovery, especially when combined with high-density genotyping. Detection power is increased when weak QTL signals in NAM lines (possibly representing QTL with small genetic effects) are accumulated from several populations into a stronger signal. Statistical methods such as joint stepwise regression (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Buckler <em>et al.</em> 2009</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Ogut <em>et al.</em> 2015</a>) and Fisher’s method (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Peirce <em>et al.</em> 2007</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>) have been used for joint mapping and pooling test statistics from population-specific QTL analyses.</p><p id="p-6">NAM has been designed primarily for maize (<a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Yu <em>et al.</em> 2008</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">McMullen <em>et al.</em> 2009</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Li <em>et al.</em> 2016</a>), but also in other cereals such as wheat (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bajgain <em>et al.</em> 2016</a>) and barley (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Maurer <em>et al.</em> 2015</a>). This approach has resulted in the mapping of QTL for traits such as flowering time (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Buckler <em>et al.</em> 2009</a>) and leaf blight resistance in maize (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kump <em>et al.</em> 2011</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Poland <em>et al.</em> 2011</a>), stem rust resistance in wheat (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bajgain <em>et al.</em> 2016</a>), and flowering time in barley (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Maurer <em>et al.</em> 2015</a>).</p><p id="p-7">Moreover, NAM lines may also function as an archive for genetic diversity. In the maize NAM populations, major heterotic groups in the United States and China have been represented by the careful selection of founding parental lines (<a id="xref-ref-82-3" class="xref-bibr" href="#ref-82">Yu <em>et al.</em> 2008</a>; <a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">McMullen <em>et al.</em> 2009</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Li <em>et al.</em> 2016</a>). The NAM populations generated from these founding lines are a resource for maize breeders to identify genetic diversity that has not yet been applied in breeding programs.</p><p id="p-8">In this current study, we developed and characterized rice NAM lines to facilitate the identification of beneficial <em>tropical japonica</em> diversity, and incorporated these traits via an elite <em>indica</em> bridge variety (IR64) for future breeding initiatives. Ten <em>tropical japonica</em> diversity donor parentals were selected to be crossed with IR64 <em>indica</em> and create 10 recombinant inbred populations of ∼200 lines each for a total of 1879 NAM lines. The allelic segregation and recombination patterns in the 10 populations were described to aid future trait mapping and breeding endeavors. As a demonstration of the usefulness of our populations for mapping traits, we explored the genetic architecture of a well-characterized agronomic trait and show, through joint analysis, that combining QTL test statistics from multiple populations can provide additional mapping precision in rice NAM lines.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Generation of the NAM population</h3><p id="p-9">A NAM population using <em>tropical japonica</em> diversity donors was constructed according to the protocol of the original NAM design for maize (<a id="xref-ref-82-4" class="xref-bibr" href="#ref-82">Yu <em>et al.</em> 2008</a>; <a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">McMullen <em>et al.</em> 2009</a>). IR64, an elite International Rice Research Institute (IRRI) <em>indica</em> line with a complex pedigree, was selected to be the common parent and a representative of the <em>indica</em> subspecies (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The diversity donor parentals were selected (1) for resistance and physiological traits and (2) to represent the genetic diversity present in <em>tropical japonica</em>. Azucena was chosen in particular for plant physiological traits such as days to heading; panicle and root architecture; grain traits such as aroma, shape, size, and zinc and iron content; tolerance to drought and aluminum; and resistance to <em>Striga</em>, <em>Xanthomonas oryzae pv. oryzae</em>, <em>Rice yellow mottle virus</em>, and <em>Magnaporthe grisea</em>. The IR64 × Azucena population, developed by Institut de Recherche pour le Développement (IRD), France, has been previously genotyped and described by Simple Sequence Repeats (SSRs) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bourgis <em>et al.</em> 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Djedatin <em>et al.</em> 2016</a>) and Genotype-by-Sequencing (GBS) (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Spindel <em>et al.</em> 2013</a>). The other nine diversity donors were chosen to complement mapping of the above traits in IR64 × Azucena crosses, and to expand upon the polymorphism between IR64 and Azucena. These additional diversity donors were selected from a phylogeny generated from microsatellite genotyping of representative <em>tropical japonica</em> lines in the International Center of Tropical Agriculture (CIAT) rice collection exhibiting drought resistance (E. Torres, unpublished data). <em>Tropical japonica</em> parentals were then chosen to represent as many individual clades of this phylogeny as possible.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Nested Association Mapping (NAM) population design. The greater NAM population is comprised of 10 separate Recombinant Inbred Line (RIL) populations, with each population being derived from a cross between a diversity donor parent and an IR64 indica parent common to all 10 populations. Each RIL population consists of roughly 200 replicates of the initial cross. Therefore, there were 1879 total lines in the greater NAM population (&#x223C;200 lines &#xD7; 10 RIL populations). After the F1, each line was self-pollinated for nine generations (IR64 &#xD7; Azucena) or six generations (all other crosses) by single-seed descent (SSD). Selfing by SSD facilitates the shuffling of parental haplotypes through recombination and the fixation of haplotypes through homozygosity. Each selfing generation reduces heterozygosity by half, so a F7 NAM line would yield a heterozygous proportion of 0.56 = 0.016. This figure is adapted from Yu et al. (2008)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2025222528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Nested Association Mapping (NAM) population design. The greater NAM population is comprised of 10 separate Recombinant Inbred Line (RIL) populations, with each population being derived from a cross between a diversity donor parent and an IR64 &lt;em&gt;indica&lt;/em&gt; parent common to all 10 populations. Each RIL population consists of roughly 200 replicates of the initial cross. Therefore, there were 1879 total lines in the greater NAM population (&#x223C;200 lines &#xD7; 10 RIL populations). After the F&lt;sub&gt;1&lt;/sub&gt;, each line was self-pollinated for nine generations (IR64 &#xD7; Azucena) or six generations (all other crosses) by single-seed descent (SSD). Selfing by SSD facilitates the shuffling of parental haplotypes through recombination and the fixation of haplotypes through homozygosity. Each selfing generation reduces heterozygosity by half, so a F&lt;sub&gt;7&lt;/sub&gt; NAM line would yield a heterozygous proportion of 0.5&lt;sup&gt;6&lt;/sup&gt; = 0.016. This figure is adapted from Yu &lt;em&gt;et al.&lt;/em&gt; (2008).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1913/F1.medium.gif" width="299" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/6/1913/F1.medium.gif" width="299" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484932" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Nested Association Mapping (NAM) population design. The greater NAM population is comprised of 10 separate Recombinant Inbred Line (RIL) populations, with each population being derived from a cross between a diversity donor parent and an IR64 <em>indica</em> parent common to all 10 populations. Each RIL population consists of roughly 200 replicates of the initial cross. Therefore, there were 1879 total lines in the greater NAM population (∼200 lines × 10 RIL populations). After the F<sub>1</sub>, each line was self-pollinated for nine generations (IR64 × Azucena) or six generations (all other crosses) by single-seed descent (SSD). Selfing by SSD facilitates the shuffling of parental haplotypes through recombination and the fixation of haplotypes through homozygosity. Each selfing generation reduces heterozygosity by half, so a F<sub>7</sub> NAM line would yield a heterozygous proportion of 0.5<sup>6</sup> = 0.016. This figure is adapted from <a id="xref-ref-82-5" class="xref-bibr" href="#ref-82">Yu <em>et al.</em> (2008)</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">Each of the 10 <em>tropical japonica</em> diversity donors were crossed with the common <em>indica</em> parent IR64. About three hundred F<sub>2</sub> individuals were derived from each F<sub>1</sub> cross. Whenever possible, all 300 samples were taken from a single F<sub>1</sub> plant, with additional F<sub>1</sub> plants used only when necessary. Approximately 3000 F<sub>2</sub> plants were advanced to the F<sub>7</sub> generation by single-seed descent, and to the F<sub>10</sub> generation for the IR64 × Azucena population. Due to environmental conditions in the field and to partial genetic sterility in <em>indica</em> × <em>japonica</em>-wide crosses, some of the plants exhibited reduced fertility at different generations. As a result, just under 200 NAM lines were created for each lineage (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In total, 1879 NAM plants were generated. Outcrossing was minimized in each generation by covering inflorescences with pollination bags prior to anthesis. <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> indicates the population numbers for each of the 10 sets of RIL populations that were created to generate the larger NAM population.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484870/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484870/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Descriptive statistics of 10 IR64 <em>indica</em> × <em>tropical japonica</em> diversity donor recombinant inbred populations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>GBS libraries</h3><p id="p-18">Modified flexible and scalable GBS (fsGBS) libraries were prepared according a published protocol (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Heffelfinger <em>et al.</em> 2014</a>). The Heffelfinger fsGBS protocol, which uses blunt-end restriction enzymes and employs standard Illumina Y-adapters, facilitates greater multiplexing through dual-indexed barcodes and discourages concatamer formation. Approximately 200 ng of genomic DNA was digested with <em>Rsa</em>I, a 4 bp restriction enzyme, to achieve high marker density and maximize the number of restriction fragments for Illumina sequencing. Illumina libraries were paired-end sequenced at the Yale Center for Genome Analysis. Based on read coverage, a subset of samples was selected for additional resequencing to achieve a minimum coverage threshold for all NAM lines.</p></div><div id="sec-4" class="subsection"><h3>Genomics dataset protocol</h3><p id="p-19">GBS paired-end reads were aligned with Novoalign (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hercus 2012</a>) to version 7 of the Nipponbare reference genome (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Ouyang <em>et al.</em> 2007</a>). Variant calling was performed with GATK (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">DePristo <em>et al.</em> 2011</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Auwera <em>et al.</em> 2013</a>), and a custom algorithm was used for filtering raw variant calls and filtering imputation results, as described in <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Heffelfinger <em>et al.</em> (2014)</a>. Variant calling and imputation was performed on a population-by-population basis (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). LB-Impute (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fragoso <em>et al.</em> 2016</a>) was first used to impute missing parental markers that were sequenced in the offspring. A second round of parental variant filtering was executed to remove any heterozygous parental markers and to confirm that each marker was polymorphic between the parentals. Next, offspring imputation was performed based on the imputed parental genotypes (parameters for all of the above programs are described in Supplemental Material, Note S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>). LB-Impute was chosen for its ability to perform parental imputation and for its high imputation accuracy in regions of low-coverage residual heterozygosity. The final imputed dataset represents high-confidence markers present in each population with low-confidence markers left as missing.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Stages of processing GBS variant data. After variant calling with GATK, variant calling data (vcf format) undergoes four stages of processing. (A&#x2013;D) represent chromosome 1 of the IR64 &#xD7; Azucena population, with each row representing each NAM line, and each column a marker. Red markers are homozygous for the IR64 state, blue for Azucena, yellow heterozygous, and black missing. The first stage (A) involves variant filtering with custom software described by Heffelfinger et al. (2014), (B) parental imputation and filtering with LB-Impute, and (C) offspring imputation with LB-Impute. Since LB-Impute may leave variants adjacent to transitions in parental state as missing, BP-Impute is applied in (D) to infer the most likely breakpoint location (red box). Genotypes may be left as probabilities or can have discrete genotypes assigned through least squares. BP-Impute is designed to prepare datasets for joint trait mapping and genetic map construction; NAM lines with excessive heterozygosity are also removed. In (E) phenotypes are regressed on each marker, then permuted phenotypes are also used to calculate locus-specific P values. The locus-specific P values can be used in joint analysis. (F) and (G) represent the entire LB-Impute dataset for the IR64 &#xD7; Azucena and IR64 &#xD7; CT10035-26-4-2-M NAMs, with each concentric circle depicting a separate NAM genome. For (G), blue is the homozygous CT10035-26-4-2-M state. GBS, genotype-by-sequencing; NAM, Nested Association Mapping." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2025222528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Stages of processing GBS variant data. After variant calling with GATK, variant calling data (vcf format) undergoes four stages of processing. (A&#x2013;D) represent chromosome 1 of the IR64 &#xD7; Azucena population, with each row representing each NAM line, and each column a marker. Red markers are homozygous for the IR64 state, blue for Azucena, yellow heterozygous, and black missing. The first stage (A) involves variant filtering with custom software described by Heffelfinger &lt;em&gt;et al.&lt;/em&gt; (2014), (B) parental imputation and filtering with LB-Impute, and (C) offspring imputation with LB-Impute. Since LB-Impute may leave variants adjacent to transitions in parental state as missing, BP-Impute is applied in (D) to infer the most likely breakpoint location (red box). Genotypes may be left as probabilities or can have discrete genotypes assigned through least squares. BP-Impute is designed to prepare datasets for joint trait mapping and genetic map construction; NAM lines with excessive heterozygosity are also removed. In (E) phenotypes are regressed on each marker, then permuted phenotypes are also used to calculate locus-specific &lt;em&gt;P&lt;/em&gt; values. The locus-specific &lt;em&gt;P&lt;/em&gt; values can be used in joint analysis. (F) and (G) represent the entire LB-Impute dataset for the IR64 &#xD7; Azucena and IR64 &#xD7; CT10035-26-4-2-M NAMs, with each concentric circle depicting a separate NAM genome. For (G), blue is the homozygous CT10035-26-4-2-M state. GBS, genotype-by-sequencing; NAM, Nested Association Mapping.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1913/F2.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/6/1913/F2.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484994" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Stages of processing GBS variant data. After variant calling with GATK, variant calling data (vcf format) undergoes four stages of processing. (A–D) represent chromosome 1 of the IR64 × Azucena population, with each row representing each NAM line, and each column a marker. Red markers are homozygous for the IR64 state, blue for Azucena, yellow heterozygous, and black missing. The first stage (A) involves variant filtering with custom software described by <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Heffelfinger <em>et al.</em> (2014)</a>, (B) parental imputation and filtering with LB-Impute, and (C) offspring imputation with LB-Impute. Since LB-Impute may leave variants adjacent to transitions in parental state as missing, BP-Impute is applied in (D) to infer the most likely breakpoint location (red box). Genotypes may be left as probabilities or can have discrete genotypes assigned through least squares. BP-Impute is designed to prepare datasets for joint trait mapping and genetic map construction; NAM lines with excessive heterozygosity are also removed. In (E) phenotypes are regressed on each marker, then permuted phenotypes are also used to calculate locus-specific <em>P</em> values. The locus-specific <em>P</em> values can be used in joint analysis. (F) and (G) represent the entire LB-Impute dataset for the IR64 × Azucena and IR64 × CT10035-26-4-2-M NAMs, with each concentric circle depicting a separate NAM genome. For (G), blue is the homozygous CT10035-26-4-2-M state. GBS, genotype-by-sequencing; NAM, Nested Association Mapping.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">The missing, low-confidence markers were often present in transition regions of recombination breakpoints. Therefore, a second algorithm, Breakpoint Imputation (BP-Impute, Note S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>), was developed to improve the precision of genotyping in regions containing recombination breakpoints. BP-Impute calculates a Markov chain across the missing regions, with chains in either direction constrained to the flanking parental state. The transition probability was the proportion of recombinant genotypes across each interval. Emission probabilities were the binomial probabilities of emitting genotypes given read coverage and the constrained parental state. The probabilities from both chains were normalized to sum to one, and the weighted average genotype is thus summed from these probabilities.</p><p id="p-22">In each BP-Impute population dataset, a limited number of lines with excess heterozygosity (&gt; 10%), representing potential outcrossing, were removed so that estimations of recombination would be more representative of the rest of the population. The resulting dataset was comprised of genotype probabilities, with each data point representing the probability of a genotype being homozygous for the diversity donor allele. The genotype probabilities were used for measuring segregation distortion and trait mapping. For measuring local recombination rates and the genetic map, discrete genotypes were assigned to genotype probabilities by applying least squares through a separate, custom R script (BP-Impute, Note S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>).</p></div><div id="sec-5" class="subsection"><h3>Parental sequencing, clade assignment, and allele frequency analysis</h3><p id="p-23">The 11 parental lines were sequenced as part of the International Rice Genomic Initiative (IRIGIN), a France Genomique project by whole-genome Illumina paired-end sequencing at an average coverage of 35 ×. Whole-genome sequencing (WGS) datasets of parental lines were subject to the same read alignment, variant calling, and variant filtering as the GBS data, but with slightly different parameter settings (Note S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>). The filtered WGS variant data were trimmed by randomly selecting a marker within bins of 5 kb. For this set of 72,193 markers, the SNP-Seek database (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexandrov <em>et al.</em> 2015</a>) was queried to extract the trimmed marker data for the (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">3000 Rice Genomes Project 2014</a>). In total, 49,431 markers and 2656 lines were extracted from SNP-Seek. Only markers that were homozygous in the NAM parentals were selected from the SNP-Seek marker set in order to exclude false heterozygous regions arising from poor read mapping in repetitious genomic regions. Lines annotated as <em>indica</em>, <em>temperate</em> <em>japonica</em>, and <em>tropical japonica</em> were merged with the NAM parental dataset and were imputed using the FILLIN algorithm (Tassel 5 version 20160428) (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Swarts <em>et al.</em> 2014</a>). Then, the imputed dataset was filtered for missingness (lines and markers with missingness &gt;0.1 were removed, then the set of complete, nonmissing markers was extracted), and pruned for linkage disequilibrium (LD) (markers in bins of 100 kb with pairwise correlation &gt;0.3 were removed). The final merged, imputed, complete, and LD-pruned dataset consisted of 2655 lines (including the 11 NAM parentals) genotyped at 7152 markers. Principal component analysis (PCA) was performed on this dataset, using the prcomp R library, to elucidate the placement of the NAM parentals in the greater population structure of rice.</p><p id="p-24">The parental contribution of IR64:DD alleles was determined for each marker in the NAM populations using the BP-Impute dataset. Statistical evidence of deviation from the 1:1 expected segregation ratio was measured with a χ<sup>2</sup> goodness of fit test.</p></div><div id="sec-6" class="subsection"><h3>Recombination analysis</h3><p id="p-25">The genetic map for each population was calculated from the LB-Impute (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Fragoso <em>et al.</em> 2016</a>) datasets (imputed with the –keep flag), with three different methods. The –keep flag was used so that both unimputed and missing variants remain in ambiguous regions surrounding recombination breakpoints. In method 1, multipoint genetic maps were calculated with Mapmaker/EXP3.0 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lincoln <em>et al.</em> 1993</a>). The LB-Impute datasets were filtered for unique markers, and multipoint maps were calculated with Lincoln and Lander error detection method (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lincoln and Lander 1992</a>) both on and off. In method 2, two-point maps were estimated from full imputation of the LB-Impute data with argmax from R/QTL (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Broman <em>et al.</em> 2003</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>). The argmax function was used to impute the LB-Impute datasets in order to further resolve breakpoints. The Mapmaker and R/QTL analyses were performed in MapDisto 2.0 (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Lorieux 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Heffelfinger <em>et al.</em> 2017</a>). These three calculations were compared with the genetic maps generated with method 3 based on BP-Impute datasets.</p><p id="p-26">In method 3, the assign genotypes function was used to identify breakpoints in BP-Impute datasets and reach full imputation. Then, the genetic distance between each pair of adjacent markers was calculated through two-point analysis. The recombination rate per meiosis between adjacent markers, <em>r</em>, was calculated with the unbiased estimate for RILs self-pollinated by single seed descent, as described by Martin and Hospital, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1913/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1913/embed/mml-math-1.gif"/></noscript></span></span>, where <em>m</em> is the number of recombinants among <em>N</em> RILs (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Martin and Hospital 2006</a>). Transitions into heterozygous states from homozygous variants, or vice versa, were either set as missing (weighted as 0, with all other transitions weighted as 1) or were weighted as 0.5 (with all other transitions weighted as 1). This was performed to examine the effect of heterozygosity on the genetic map sizes, while taking into account the theoretical fixation of heterozygous regions as homozygous after infinite selfing generations. From estimates of <em>r</em> with all methods, the Kosambi mapping function was used to measure the genetic distance between each pair of adjacent markers.</p><p id="p-27">A joint genotype dataset was also created to compare the joint genetic map size with individual population genetic maps. The union of all markers in the 10 populations was taken, imputed with the R/QTL algorithm argmax function (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>) (double recombinant probability set as 0.005), then the joint two-point map was calculated with the <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Martin and Hospital (2006)</a> <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1913/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1913/embed/mml-math-2.gif"/></noscript></span></span> estimate in MapDisto 2.0 (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Lorieux 2012</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Heffelfinger <em>et al.</em> 2017</a>).</p><p id="p-28">The Gaussian kernel method was then applied to smooth the BP-Impute two-point genetic distances for analysis of local recombination rate. The kernel method was chosen to reduce the effects of any possible errors introduced through genotyping or breakpoint imputation. A similar analysis performed by Spindel was used as a guide (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Spindel <em>et al.</em> 2013</a>). For each measurement of genetic distance, a Gaussian (normal) density function with σ = 0.5 was centered on the physical position, in mega bases, of the corresponding marker. The kernel was set to 0 where markers were off the queried chromosome, and the density was then renormalized so it would sum to 1. Each marker pair’s two-point genetic distance was then recalculated according to the weights determined by the kernel.</p></div><div id="sec-7" class="subsection"><h3>Trait mapping</h3><p id="p-29">The number of days to heading (the emergence of the rice inflorescences) since sowing date was selected to demonstrate the utility of the NAM design for trait mapping in individual populations, and jointly across all 10 populations. Days to heading was phenotyped at CIAT between 2011 and 2013, with sowing dates in January, February, March, May, June, July, and November. All populations were phenotyped for the trait; two lines were excluded from QTL mapping for erroneous data entries.</p><p id="p-30">The genotype probability dataset from BP-Impute (each genotype represented by the probability of representing the homozygous, diversity donor state) was used for trait mapping. With single marker linear regression, each marker in each population was tested for the null hypothesis of no additive genetic effect at the locus (the <em>y</em> intercept as the best explanation of the phenotype). The alternative hypothesis was that the genotypes explain some variance of the phenotype. The <em>F</em> statistic value (henceforth referred to as the <em>F</em> value), from the R lm function, was stored for each marker. The <em>F</em> value is asymptotically equivalent to the LOD score (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>) and is convenient for its easy and rapid extraction from the lm function. Linear regression was used to fit the null and alternate hypotheses so that continuous genotype probabilities may be used. Given the saturation of the genetic map, use of interval mapping (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lander and Botstein 1989</a>) was not required.</p><p id="p-31">The methodology of Peirce regarding permutation, linear interpolation, locus-specific <em>P</em> values, and joint analysis (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Peirce <em>et al.</em> 2007</a>) was applied through custom R scripts written for trait mapping in the 10 populations. In order to facilitate downstream significance testing, the phenotypes were permuted 1000 times, and each permutation was regressed on the markers. In order to calculate locus-specific <em>P</em> values, the probability of encountering an <em>F</em> value as large as the observed value in the 1000 permutations was determined for each marker.</p></div><div id="sec-8" class="subsection"><h3>Joint analysis for trait mapping and allelic frequency</h3><p id="p-32">For <em>F</em> value peaks jointly segregating in the QTL analysis, and regions of segregation distortion present in multiple populations, joint analysis was used to pool test statistics from the respective populations. To this end, Fisher’s combined probability test (Fisher’s method) was applied to <em>P</em> values from χ<sup>2</sup> goodness of fit tests (1:1 parental allele segregation) and to locus-specific <em>P</em> values from trait mapping. According to Fisher’s method, the natural logs of <em>P</em> values were summed for each of the markers in the local joint marker set. This sum, when multiplied by −2, has a χ<sup>2</sup> distribution with degrees of freedom equal to twice the number of summed log <em>P</em> values under the null hypothesis. In order to create the joint marker set for these analyses, <em>P</em> values (segregation distortion) or <em>F</em> values (trait mapping) were linearly interpolated by physical position. This allowed for each population to have a test statistic at the same set of positions representing the region of interest. Joint analysis was performed on a targeted basis at these specified regions.</p><p id="p-33">For the joint trait mapping analysis, genome-wide adjusted <em>P</em> values were calculated. This refers to the proportion of pooled <em>P</em> values obtained from random permutations that are as extreme as the observed pooled value. For each of the populations subject to pooling, locus-specific <em>P</em> values were calculated for every permuted <em>F</em> value at a given marker. A locus-specific <em>P</em> value was then randomly selected at every marker, from each population, to create a random pooling. This permuted pooling was performed 1000 times, and the genome-wide adjusted <em>P</em> value was estimated by the proportion of the permuted pooled values larger than the pooled observed data. The threshold for significance was the 95th percentile of permuted poolings.</p><p id="p-34">In order to create a support interval for the joint QTL peaks, the consensus support interval among population-specific 1.5-<em>F</em> value or 3-<em>F</em> value support intervals [comparable to the 1-LOD, 1.5-LOD, or 1.8-LOD interval commonly used (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Peirce <em>et al.</em> 2007</a>; <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>)] were calculated. This method was used as an approximation to a true C.I. comparable across populations. A similar approach was used for support intervals surrounding segregation distortion loci, except the support intervals were only calculated for the pooled χ<sup>2</sup> values, not for each individual population-specific value.</p><p id="p-35">All analyses, including studies of recombination, segregation distortion, and trait mapping, were performed with custom R scripts. These programs were tested and run with R version 3.3.2 (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">R Core Team 2014</a>).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-36">NAM parental and offspring germplasm and genotypes are available by request. Genotypes are offered through a material transfer agreement (MTA) from Yale University via stephen.dellaporta@yale.edu and the germplasm through a MTA from the International Center of Tropical Agriculture, via the corresponding author. LB-Impute (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Fragoso <em>et al.</em> 2016</a>) and BP-Impute are available through a license on the Dellaporta Laboratory Github site <a href="https://github.com/dellaporta-laboratory">https://github.com/dellaporta-laboratory</a> and MapDisto v2 (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Lorieux 2012</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Heffelfinger <em>et al.</em> 2017</a>). </p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Selection of NAM parentals</h3><p id="p-37">The 10 <em>tropical japonica</em> diversity donors and IR64 <em>indica</em> common parent were genotyped through 35 × WGS. We explored the genetic relationship between the NAM parental lines and the general diversity found in the <em>indica</em> and <em>japonica</em> rice subspecies in the 3000 Rice Genomes Project (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Rice Genomes Project 2014</a>). PCA on this dataset revealed three clusters of rice lines, corresponding to the <em>indica</em> subspecies, and <em>temperate japonica</em> and <em>tropical japonica</em> groups of the <em>japonica</em> subspecies (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The first principal component, representing 48% of the variance in the dataset, described the <em>japonica–indica</em> axis in rice. The second principal component, representing 3% of the variance, coincided with the separation between <em>temperate</em> and <em>tropical japonica</em>. IR64, the <em>indica</em> common parent, clustered with the other <em>indica</em> lines on the far right of the <em>indica</em> axis. Although all of the diversity donors were chosen as <em>tropical japonica</em> lines, two of these lines were located closer to <em>indica</em> along the <em>japonica–indica</em> axis (PC1). Azucena and ITA164 clustered as expected with <em>tropical japonica</em> at the far left of the <em>japonica</em> axis, whereas CT8556-37-1-3-1-M and CT10035-26-4-2-M were located within the <em>indica</em> cluster closer to the right of the axis.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal Component Analysis (PCA) of 2644 indica and japonica lines and 11 Nested Association Mapping (NAM) parental lines. PCA was performed on 1789 indica, 371 temperate japonica, and 484 tropical japonica lines of the 3000 Rice Genomes Project merged with 11 NAM parental lines. The first principal component (PC), demonstrating an indica&#x2013;japonica axis in the dataset, describes 47% of the total variance. The second PC describes the differences between the two japonica subtypes and 3% of the total variance. IR64, the indica common parent, clusters with indica, while the diversity donors appear on a gradient of indica&#x2013;tropical japonica admixture. Two diversity donors in particular, CT8556-37-1-3-1-M and CT10035-26-4-2-M, cluster with indica, potential evidence admixture in the pedigree of these two lines. Azucena and ITA164, on the other hand, cluster tightly with tropical japonica." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2025222528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Principal Component Analysis (PCA) of 2644 &lt;em&gt;indica&lt;/em&gt; and &lt;em&gt;japonica&lt;/em&gt; lines and 11 Nested Association Mapping (NAM) parental lines. PCA was performed on 1789 &lt;em&gt;indica&lt;/em&gt;, 371 &lt;em&gt;temperate&lt;/em&gt; &lt;em&gt;japonica&lt;/em&gt;, and 484 tropical &lt;em&gt;japonica&lt;/em&gt; lines of the 3000 Rice Genomes Project merged with 11 NAM parental lines. The first principal component (PC), demonstrating an &lt;em&gt;indica&#x2013;japonica&lt;/em&gt; axis in the dataset, describes 47% of the total variance. The second PC describes the differences between the two &lt;em&gt;japonica&lt;/em&gt; subtypes and 3% of the total variance. IR64, the &lt;em&gt;indica&lt;/em&gt; common parent, clusters with &lt;em&gt;indica&lt;/em&gt;, while the diversity donors appear on a gradient of &lt;em&gt;indica&#x2013;tropical japonica&lt;/em&gt; admixture. Two diversity donors in particular, CT8556-37-1-3-1-M and CT10035-26-4-2-M, cluster with &lt;em&gt;indica&lt;/em&gt;, potential evidence admixture in the pedigree of these two lines. Azucena and ITA164, on the other hand, cluster tightly with &lt;em&gt;tropical&lt;/em&gt; &lt;em&gt;japonica&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1913/F3.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/6/1913/F3.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484947" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">Principal Component Analysis (PCA) of 2644 <em>indica</em> and <em>japonica</em> lines and 11 Nested Association Mapping (NAM) parental lines. PCA was performed on 1789 <em>indica</em>, 371 <em>temperate</em> <em>japonica</em>, and 484 tropical <em>japonica</em> lines of the 3000 Rice Genomes Project merged with 11 NAM parental lines. The first principal component (PC), demonstrating an <em>indica–japonica</em> axis in the dataset, describes 47% of the total variance. The second PC describes the differences between the two <em>japonica</em> subtypes and 3% of the total variance. IR64, the <em>indica</em> common parent, clusters with <em>indica</em>, while the diversity donors appear on a gradient of <em>indica–tropical japonica</em> admixture. Two diversity donors in particular, CT8556-37-1-3-1-M and CT10035-26-4-2-M, cluster with <em>indica</em>, potential evidence admixture in the pedigree of these two lines. Azucena and ITA164, on the other hand, cluster tightly with <em>tropical</em> <em>japonica</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Population summary</h3><p id="p-39">The 1879 NAM lines were sequenced and genotyped with a customized GBS method and informatics pipeline of variant calling, variant filtering, and imputation. Ten genotype datasets were created, one for each <em>tropical japonica</em> diversity donor × IR64 <em>indica</em> recombinant inbred population. Genotyping metrics describing these populations were summarized in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. Genomic DNA from the NAM lines was digested by <em>Rsa</em>I to create GBS libraries (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Heffelfinger <em>et al.</em> 2014</a>), multiplexed according to population membership, and subjected to paired-end Illumina sequencing. Multiplexed sequencing reads were next deconvoluted by molecular barcoding, aligned to the Nipponbare reference genome version 7 (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Ouyang <em>et al.</em> 2007</a>), then further processed through variant calling, variant filtering, and imputation (see <em>Materials and Methods</em> for details). The first round of imputation resulted in ∼99% of variants being imputed via LB-Impute (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Fragoso <em>et al.</em> 2016</a>); a final stage of imputation was required to genotype markers surrounding recombination breakpoints to create a complete dataset (BP-Impute, Note S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>) for each population. These complete datasets were used for trait mapping and genetic map construction.</p><p id="p-40">Dense genotyping was achieved in all 10 populations (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). There were 1879 total lines, ranging from 184 to 190 per population. Average coverage per filtered variant in each population ranged from 1.94 to 2.56. The range of LB-Impute imputed variants ranged from 5,454,213 (among 188 lines) to 16,094,325 (among 188 lines). The greatest average heterozygosity was in IR64 × CT10035-26-4-2-M, at 5%. BP-Impute removed any line with &gt; 10% heterozygosity, so after this processing step, the number of lines in each of the 10 populations ranged from 177 to 186.</p></div><div id="sec-13" class="subsection"><h3>Distribution of parental alleles</h3><p id="p-41">We examined whether each genomic region of the diversity parent was present in at least one member of the respective NAM subset. In all 10 NAM populations, there was no fixation of IR64 reference parent alleles at any genotyped location (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>). Diversity donor alleles were present at all sites, indicating a full capture of genetic diversity from our <em>tropical japonica</em> parental lines.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Segregation distortion in chromosome 3. Chromosome 3 exhibits strong segregation distortion, in the direction of favoring the IR64 allele, for two loci. One locus is specific to IR64 &#xD7; CT10045-5-5-M-1, the other loci has a shared signal in six populations. The left panel is the color code key; a heatmap from blue to red representing the position of the respective diversity donor on the japonica&#x2013;indica axis from principal component (PC) 1. The center panel features the diversity (div.) donor allele proportion at each marker (top) with the dashed line across 0.5 reflecting the expected segregation ratio. The bottom plot on the panel shows P values from a &#x3C7;2 test of the 1:1 null hypothesis. The right panel focuses on the two loci of distortion encountered on the chromosome. Full lines represent the population(s) used in the joint analysis and dashed lines were excluded populations. For the population-shared locus, the P values were pooled via Fisher&#x2019;s method. The black line over the peaks shows the 1.5-&#x3C7;2 support interval for the peak, the gray line is the 3-&#x3C7;2 support interval. Asterisks reflect potential gene candidates for distortion factors. They were chosen for their previously described role in pollen tube growth in rice or other plants. There were 16 genes in the 1.5-&#x3C7;2 support interval for the population specific locus and two genes for that of the shared locus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2025222528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Segregation distortion in chromosome 3. Chromosome 3 exhibits strong segregation distortion, in the direction of favoring the IR64 allele, for two loci. One locus is specific to IR64 &#xD7; CT10045-5-5-M-1, the other loci has a shared signal in six populations. The left panel is the color code key; a heatmap from blue to red representing the position of the respective diversity donor on the &lt;em&gt;japonica&#x2013;indica&lt;/em&gt; axis from principal component (PC) 1. The center panel features the diversity (div.) donor allele proportion at each marker (top) with the dashed line across 0.5 reflecting the expected segregation ratio. The bottom plot on the panel shows &lt;em&gt;P&lt;/em&gt; values from a &#x3C7;&lt;sup&gt;2&lt;/sup&gt; test of the 1:1 null hypothesis. The right panel focuses on the two loci of distortion encountered on the chromosome. Full lines represent the population(s) used in the joint analysis and dashed lines were excluded populations. For the population-shared locus, the &lt;em&gt;P&lt;/em&gt; values were pooled via Fisher&#x2019;s method. The black line over the peaks shows the 1.5-&#x3C7;&lt;sup&gt;2&lt;/sup&gt; support interval for the peak, the gray line is the 3-&#x3C7;&lt;sup&gt;2&lt;/sup&gt; support interval. Asterisks reflect potential gene candidates for distortion factors. They were chosen for their previously described role in pollen tube growth in rice or other plants. There were 16 genes in the 1.5-&#x3C7;&lt;sup&gt;2&lt;/sup&gt; support interval for the population specific locus and two genes for that of the shared locus.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1913/F4.medium.gif" width="440" height="201"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/6/1913/F4.medium.gif" width="440" height="201"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484950" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">Segregation distortion in chromosome 3. Chromosome 3 exhibits strong segregation distortion, in the direction of favoring the IR64 allele, for two loci. One locus is specific to IR64 × CT10045-5-5-M-1, the other loci has a shared signal in six populations. The left panel is the color code key; a heatmap from blue to red representing the position of the respective diversity donor on the <em>japonica–indica</em> axis from principal component (PC) 1. The center panel features the diversity (div.) donor allele proportion at each marker (top) with the dashed line across 0.5 reflecting the expected segregation ratio. The bottom plot on the panel shows <em>P</em> values from a χ<sup>2</sup> test of the 1:1 null hypothesis. The right panel focuses on the two loci of distortion encountered on the chromosome. Full lines represent the population(s) used in the joint analysis and dashed lines were excluded populations. For the population-shared locus, the <em>P</em> values were pooled via Fisher’s method. The black line over the peaks shows the 1.5-χ<sup>2</sup> support interval for the peak, the gray line is the 3-χ<sup>2</sup> support interval. Asterisks reflect potential gene candidates for distortion factors. They were chosen for their previously described role in pollen tube growth in rice or other plants. There were 16 genes in the 1.5-χ<sup>2</sup> support interval for the population specific locus and two genes for that of the shared locus.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">We did find evidence of shared and population-specific signals of segregation distortion when examining parental contributions in the 10 NAM populations. The proportion of diversity donor to reference parent alleles deviated from the expected 1:1 in several locations. Depending on the diversity donor of a respective NAM population, some of the population exhibited shared or unique trends in the segregation ratio. There were shared signals of IR64 alleles being favored in chromosomes 3, 7, and 9. In these regions, many of the populations experience diversity donor allele proportions &lt; 15% (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>).</p><p id="p-44">Applying Fisher’s method to pool <em>P</em> values from χ<sup>2</sup> tests of a 1:1 segregation ratio confirmed segregation trends among the NAM populations (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and Figure S1 and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>). For populations exhibiting similar trends in segregation distortion, the natural logs of χ<sup>2</sup> <em>P</em> values were summed for each of the markers in the joint marker set.</p></div><div id="sec-14" class="subsection"><h3>Identification of segregation distortion loci</h3><p id="p-45">The strongest evidence of segregation distortion was found on chromosomes 3, 6, 7, and 9 (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>). These regions all experienced distortion favoring the IR64 allele. There existed a population-specific locus on chromosome 6 and shared peaks on chromosomes 7 and 9. The minimum diversity donor allele frequencies for markers at these peaks ranged from 0.07 to 0.15. Finally, only chromosomes 1, 4, and 6 presented substantial evidence of segregation distortion in the direction of the diversity donor allele, with extended regions of distortion for one of the 10 populations.</p><p id="p-46">There were two peaks of segregation distortion on chromosome 3; one was specific to IR64 × CT10045-5-5-M-1 and the other was shared between six populations (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a> and Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>). The 1.5 χ<sup>2</sup> interval for the population-specific peak was 94,269 bp, containing 16 genes centered at 7.6 Mb. At this population-specific peak, the minimum diversity donor allele frequency at a genotyped marker was 0.13. Two genes in this interval that could potentially contribute to pollen tube growth include the <em>ras</em>-related protein LOC_Os03g13860 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cheung and Wu 2008</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Szumlanski and Nielsen 2009</a>), and the <em>actin depolymerizing factor</em> LOC_Os03g13950 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dong <em>et al.</em> 2001</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Feng <em>et al.</em> 2006</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Zhang <em>et al.</em> 2007</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Li <em>et al.</em> 2010</a>). Expanding support to 3-χ<sup>2</sup> did not greatly increase the interval.</p><p id="p-47">In the shared peak, the 1.5-χ<sup>2</sup> support interval was 16,015 bp, containing two genes centered at 13.3 Mb. The minimum diversity donor allele frequency at the pooled peak was 0.10. One of the two genes in this interval is a rice homolog of the maize <em>aberrant pollen transmission 1</em> gene, which influences the speed of pollen tube germination in maize (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Xu and Dooner 2006</a>). Increasing the support to 3-χ<sup>2</sup> expanded the support interval to 173.5 kb.</p></div><div id="sec-15" class="subsection"><h3>Recombination analysis</h3><p id="p-48">To assess the accuracy of our genotyping and imputation methods, we surveyed the number of recombination events in each NAM line, and used this information to calculate total genetic maps for every population. Across all NAM lines, the average number of recombination events was 18.9 with a SD of 10.9 (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>). When we examined the average number of recombination events in the 10 populations, ANOVA suggested that the differences between the population means were significant, with an <em>F</em> value of 4.5 and a <em>P</em> value of 6.8 × 10<sup>−6</sup>. The population with the greatest average number of recombination events was IR64 × CT10006-7-2-M-2, with 21.6 events. The fewest average number was IR64 × Azucena, at 16.1 events.</p><p id="p-49">Of the three genetic map construction strategies, multipoint Mapmaker/EXP3.0 (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Lincoln <em>et al.</em> 1993</a>) produced the largest maps, averaging across the 10 populations, at 2177.0 cM (± SD 352.8 cM) without error correction, and were significantly shortened with error correction, at 1494.2 cM (± SD 218.2 cM) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). R/QTL argmax (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Broman <em>et al.</em> 2003</a>; <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>) imputation produced an average two-point map size of 1554.5 cM (± SD 161.7 cM). BP-Impute resulted in 1210.2 cM (± SD 91.2 cM) without counting heterozygous–homozygous transitions and increased to 1430.8 cM (± SD 94.4 cM) with counting transitions (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>). The population with the greatest map with BP-Impute and heterozygosity was IR64 × CT10006-7-2-M-2 at 1612.0 cM and the smallest was IR64 × CT10035-26-4-2-M at 1255.8 cM.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic map sizes of 10 NAM populations. Genetic maps were calculated following (i) method 1, multipoint analysis with MapMaker/EXP3.0 with error detection off and on; (ii) method 2, two-point analysis with MapDisto on data imputed with the R/QTL argmax function; and (iii) method 3, two-point analysis on BP-Impute datasets, without and with integration of heterozygous&#x2013;homozygous transitions. The expected map size in rice is represented by the horizontal dotted line, at 1521.6 cM (Harushima et al. 1998). The 10 population means are shown with &#xB1;1 SD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2025222528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic map sizes of 10 NAM populations. Genetic maps were calculated following (i) method 1, multipoint analysis with MapMaker/EXP3.0 with error detection off and on; (ii) method 2, two-point analysis with MapDisto on data imputed with the R/QTL argmax function; and (iii) method 3, two-point analysis on BP-Impute datasets, without and with integration of heterozygous&#x2013;homozygous transitions. The expected map size in rice is represented by the horizontal dotted line, at 1521.6 cM (Harushima &lt;em&gt;et al.&lt;/em&gt; 1998). The 10 population means are shown with &#xB1;1 SD.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1913/F5.medium.gif" width="440" height="221"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/6/1913/F5.medium.gif" width="440" height="221"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484790" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-50" class="first-child">Genetic map sizes of 10 NAM populations. Genetic maps were calculated following (i) method 1, multipoint analysis with MapMaker/EXP3.0 with error detection off and on; (ii) method 2, two-point analysis with MapDisto on data imputed with the R/QTL argmax function; and (iii) method 3, two-point analysis on BP-Impute datasets, without and with integration of heterozygous–homozygous transitions. The expected map size in rice is represented by the horizontal dotted line, at 1521.6 cM (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Harushima <em>et al.</em> 1998</a>). The 10 population means are shown with ±1 SD.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">In the joint population dataset, where all NAM lines from the 10 populations were imputed together by R/QTL argmax (<a id="xref-ref-8-7" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>) as a single dataset, the original size of the union of markers was 170,000. This value was collapsed to 50,079 markers by removing loci with no recombination with other loci. Then, after R/QTL argmax imputation, the dataset consisted of 50,006 complete markers with no missing values. The two-point genetic map (Martin Hospital estimate for <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1913/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1913/embed/mml-math-3.gif"/></noscript></span></span>) produced a total map size of 1348.7 cM for the joint dataset (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>).</p><p id="p-52">Although the genetic maps calculated from imputed data of each population globally approximated the map size for rice, there were local variations in recombination rate. Each of the 10 populations demonstrated deviations from the expected average recombination rate [1 cM per 0.244 Mb, or 4.1 cM:Mb (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chen <em>et al.</em> 2002</a>)], with local hotspots being both unique and shared among the NAM subsets (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). As expected, all populations experienced a decrease in recombination at the centromere (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cheng <em>et al.</em> 2002</a>; <a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Ouyang <em>et al.</em> 2007</a>). However, throughout the remainder of the genome, the cM:Mb ratio remained close to the expected value of 4.1 cM:Mb (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Chen <em>et al.</em> 2002</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gaussian smoothed ratio of the genetic and physical maps. Two-point genetic distances were calculated for each pair of adjacent markers, for all populations, using method 3 with integration of heterozygous&#x2013;homozygous transitions and the Kosambi mapping function. A Gaussian kernel with &#x3C3; = 0.5 was placed over each marker to smooth the two-point genetic and physical distances; then the ratio was taken and plotted. The horizontal line is the expected ratio, at 1 cM per 0.244 Mb or 4.1 cM:Mb, per Chen et al. (2002). The vertical dotted line is the approximate location of the centromere, as defined by the presence of centromere-specific retrotransposons described by Cheng et al. (2002). Chr, chromosome; PC, principal component." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2025222528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gaussian smoothed ratio of the genetic and physical maps. Two-point genetic distances were calculated for each pair of adjacent markers, for all populations, using method 3 with integration of heterozygous&#x2013;homozygous transitions and the Kosambi mapping function. A Gaussian kernel with &#x3C3; = 0.5 was placed over each marker to smooth the two-point genetic and physical distances; then the ratio was taken and plotted. The horizontal line is the expected ratio, at 1 cM per 0.244 Mb or 4.1 cM:Mb, per Chen &lt;em&gt;et al.&lt;/em&gt; (2002). The vertical dotted line is the approximate location of the centromere, as defined by the presence of centromere-specific retrotransposons described by Cheng &lt;em&gt;et al.&lt;/em&gt; (2002). Chr, chromosome; PC, principal component.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1913/F6.medium.gif" width="378" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/6/1913/F6.medium.gif" width="378" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484793" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-53" class="first-child">Gaussian smoothed ratio of the genetic and physical maps. Two-point genetic distances were calculated for each pair of adjacent markers, for all populations, using method 3 with integration of heterozygous–homozygous transitions and the Kosambi mapping function. A Gaussian kernel with σ = 0.5 was placed over each marker to smooth the two-point genetic and physical distances; then the ratio was taken and plotted. The horizontal line is the expected ratio, at 1 cM per 0.244 Mb or 4.1 cM:Mb, per <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Chen <em>et al.</em> (2002)</a>. The vertical dotted line is the approximate location of the centromere, as defined by the presence of centromere-specific retrotransposons described by <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Cheng <em>et al.</em> (2002)</a>. Chr, chromosome; PC, principal component.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Joint QTL analysis</h3><p id="p-54">Days to heading, defined as the number of days from sowing until the emergence of inflorescences, is a complex trait of agronomic significance. The short arm of chromosome 3 features four genes, the CCCH-type zinc finger gene LOC_Os03g02160, <em>early heading date 4</em> (<em>Ehd4</em>) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gao <em>et al.</em> 2013</a>), MIKC-type <em>MADS-box 50</em> gene (<em>OsMADS50</em>) LOC_Os03g03070 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lee <em>et al.</em> 2004</a>), <em>DNA-binding with one finger 12</em> (<em>OsDof12</em>) LOC_Os03g07360 (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Li <em>et al.</em> 2009</a>), and <em>rice phytochrome B</em> (<em>OsPhyB</em>) LOC_Os03g19590 (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Takano <em>et al.</em> 2005</a>). Our goal was to determine whether a joint mapping approach in the NAM population could reveal the complex genetic architecture in this region. The entire NAM population was phenotyped for a number of traits including days to heading. The global mean of days to heading, among all NAM lines, was 91.86 d, with a SD of 6.69 d (Figure S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>). The greatest mean days to heading, for an individual population, was IR64 × CT10035-26-4-2-M at 99.90 d. The fewest mean days to heading was IR64 × Azucena at 88.82 d. ANOVA suggests that differences between the population means were statistically significant, with an <em>F</em> value of 55.00 and a <em>P</em> value &lt; 2.2 × 10<sup>−16</sup>.</p><p id="p-55">The joint analysis of days to heading QTL (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>) revealed three joint peaks that correspond closely with <em>Ehd4</em>, <em>OsMADS50</em>, and <em>OsDof12</em>. The peak that aligned with <em>OsMADS50</em> had the smallest <em>P</em> value encountered through joint permutation testing; <em>Ehd4</em> was located adjacent to a nearby subpeak of <em>OsMADS50</em>. <em>OsDof12</em>, located further downstream, aligned to the second highest peak. The <em>OsPhyB</em> gene coordinates were positioned beneath the 17.79 (−log10; 1.25) 95th percentile of permuted pooling replicates. In order to identify gene candidates, the MSU Rice Genome Annotation Project (<a id="xref-ref-62-4" class="xref-bibr" href="#ref-62">Ouyang <em>et al.</em> 2007</a>) was consulted for gene locus locations and the BLAST (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altschul <em>et al.</em> 1990</a>) tool was used to search for homologs in other plant species.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Joint and individual population QTL analysis of days to heading on the short arm of chromosome 3. (A) Locus-specific P values from each population were pooled through Fisher&#x2019;s method. The positions of three out of four known photoperiod genes are represented by vertical lines on the x-axis. The horizontal line is the 95th percentile of permuted poolings. (B) The locus-specific P values for each population; the proportion of single marker regressions with permuted phenotypes yielding an F value as extreme as the observed F value. Since there were 1000 permutations, the minimum P value is 10&#x2212;3. The y-axis in all subplots are the &#x2212;log10 P values, on a linear scale, with the horizontal line reflecting the 10&#x2212;3 minimum P value. The x-axis is the same as (A), with the vertical lines representing the four genes. (C) The peak corresponding to OsMADS50, also green in (A), is further examined in individual populations. The colored lines are the 1.5-F value support intervals; the 3-F value support intervals are in gray. The summations of the support intervals, indicating consensus between the individual support intervals, are located above. The 1.5-F value consensus is the solid line, and the 3-F value consensus is the dotted line. The vertical lines are the positions of Ehd4 and OsMADS50. The color coding in (B) and (C) for the 10 populations is consistent with the other figures, and reflects the principal component analysis in Figure 3. QTL, quantitative trait loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2025222528" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Joint and individual population QTL analysis of days to heading on the short arm of chromosome 3. (A) Locus-specific &lt;em&gt;P&lt;/em&gt; values from each population were pooled through Fisher&#x2019;s method. The positions of three out of four known photoperiod genes are represented by vertical lines on the &lt;em&gt;x&lt;/em&gt;-axis. The horizontal line is the 95th percentile of permuted poolings. (B) The locus-specific &lt;em&gt;P&lt;/em&gt; values for each population; the proportion of single marker regressions with permuted phenotypes yielding an &lt;em&gt;F&lt;/em&gt; value as extreme as the observed &lt;em&gt;F&lt;/em&gt; value. Since there were 1000 permutations, the minimum &lt;em&gt;P&lt;/em&gt; value is 10&lt;sup&gt;&#x2212;3&lt;/sup&gt;. The &lt;em&gt;y&lt;/em&gt;-axis in all subplots are the &#x2212;log10 &lt;em&gt;P&lt;/em&gt; values, on a linear scale, with the horizontal line reflecting the 10&lt;sup&gt;&#x2212;3&lt;/sup&gt; minimum &lt;em&gt;P&lt;/em&gt; value. The &lt;em&gt;x&lt;/em&gt;-axis is the same as (A), with the vertical lines representing the four genes. (C) The peak corresponding to OsMADS50, also green in (A), is further examined in individual populations. The colored lines are the 1.5-&lt;em&gt;F&lt;/em&gt; value support intervals; the 3-&lt;em&gt;F&lt;/em&gt; value support intervals are in gray. The summations of the support intervals, indicating consensus between the individual support intervals, are located above. The 1.5-&lt;em&gt;F&lt;/em&gt; value consensus is the solid line, and the 3-&lt;em&gt;F&lt;/em&gt; value consensus is the dotted line. The vertical lines are the positions of Ehd4 and OsMADS50. The color coding in (B) and (C) for the 10 populations is consistent with the other figures, and reflects the principal component analysis in Figure 3. QTL, quantitative trait loci.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1913/F7.medium.gif" width="440" height="337"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/6/1913/F7.medium.gif" width="440" height="337"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1913/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484955" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-56" class="first-child">Joint and individual population QTL analysis of days to heading on the short arm of chromosome 3. (A) Locus-specific <em>P</em> values from each population were pooled through Fisher’s method. The positions of three out of four known photoperiod genes are represented by vertical lines on the <em>x</em>-axis. The horizontal line is the 95th percentile of permuted poolings. (B) The locus-specific <em>P</em> values for each population; the proportion of single marker regressions with permuted phenotypes yielding an <em>F</em> value as extreme as the observed <em>F</em> value. Since there were 1000 permutations, the minimum <em>P</em> value is 10<sup>−3</sup>. The <em>y</em>-axis in all subplots are the −log10 <em>P</em> values, on a linear scale, with the horizontal line reflecting the 10<sup>−3</sup> minimum <em>P</em> value. The <em>x</em>-axis is the same as (A), with the vertical lines representing the four genes. (C) The peak corresponding to OsMADS50, also green in (A), is further examined in individual populations. The colored lines are the 1.5-<em>F</em> value support intervals; the 3-<em>F</em> value support intervals are in gray. The summations of the support intervals, indicating consensus between the individual support intervals, are located above. The 1.5-<em>F</em> value consensus is the solid line, and the 3-<em>F</em> value consensus is the dotted line. The vertical lines are the positions of Ehd4 and OsMADS50. The color coding in (B) and (C) for the 10 populations is consistent with the other figures, and reflects the principal component analysis in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. QTL, quantitative trait loci.</p><div class="sb-div caption-clear"></div></div></div><p id="p-57">Of the individual population locus-specific <em>P</em> values (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>), IR64 × Azucena and IR64 × CT10035-42-4-4-M experienced the minimum possible <em>P</em> value (10<sup>−3</sup>) for entire interval containing the three jointly segregating genes, <em>Ehd4</em>, <em>OsMADS50</em>, and <em>OsDof12</em>. The other populations either demonstrated weaker peaks or troughs in between the three genes. No population exhibited clear evidence for a peak at the <em>OsPhyB</em> gene.</p><p id="p-58">The <em>F</em> values differed greatly among the populations (Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>). IR64 × Azucena had the greatest <em>F</em> values by far, with a maximum of 69.79, and a complex QTL peak structure in the vicinity of <em>Ehd4</em> and <em>OsMADS50</em>. IR64 × CT10035-42-4-4-M had the second highest <em>F</em> values and a similar QTL peak structure. Other populations, despite demonstrating clear peaks in the permutation analysis, had much smaller <em>F</em> values as compared to IR64 × Azucena.</p><p id="p-59">The joint peak corresponding to <em>OsMADS50</em> was subjected to more in depth analysis to better understand the individual contributions of each population to the joint peak (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7C</a>). The analysis interval was chosen to avoid inclusion of the minor peak near <em>Ehd4</em>. The greatest <em>F</em> value for each population, between 0.82 and 1.90 Mb on chromosome 3, was determined, and the 3 and 1.5-<em>F</em> value support interval was overlaid on the joint peak. Each set of support intervals was then summed to calculate a consensus interval. The two summations were plotted above the individual support intervals in <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7C</a>, with the 1.5-<em>F</em> value as a solid line and the 3-<em>F</em> value as a dotted line. The <em>OsMADS50</em> gene is upstream of the 1.5-<em>F</em> value consensus (85,226 bp) within this interval. IR64 × Azucena was the primary driver of this consensus region. Homeobox genes and protein kinases are found in this region, including a receptor-like protein kinase LOC_Os03g03280 homologous to <em>CURVY1</em> that controls flowering time in <em>Arabidopsis</em> (MSU rice BLAST: <em>e</em> score 10<sup>−160</sup>, top query coverage 96.32, top Id 38.57) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gachomo <em>et al.</em> 2014</a>). From the 3-<em>F</em> value support intervals, the consensus region is 235,514 bp and contains <em>OsMADS50</em>.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-60">We have developed and characterized the genetic structure of 10 recombinant inbred populations of rice that combined for a total of 1879 NAM lines. These populations represent a genetic resource for mapping traits relevant to tropical agriculture and for the study of the genetic properties of <em>indica</em> × <em>tropical japonica</em>-wide crosses. Through the power of joint analysis, we show that regions of complex genetic architecture or segregation distortion can be finely dissected and characterized. By applying newly developed variant filtering and imputation methods, and combined with the improved fsGBS method (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Heffelfinger <em>et al.</em> 2014</a>), our results combine both dense genotyping and accurate detection of recombination events, without the trade-off of excessive erroneous recombination events. The average map size of our BP-Impute datasets, 1430.8 cM (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>), deviates 90.8 cM from the expected map size of 1521.6 cM (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Harushima <em>et al.</em> 1998</a>). A previous estimate of the genetic map of the IR64 × Azucena RIL population was 1862 cM, an overestimate of the expected map size by 331 cM (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Spindel <em>et al.</em> 2013</a>).</p><p id="p-61">It is likely that the BP-Impute genetic maps (method 3) are slightly shorter (1430.8 cM ± SD 94.4 cM) than expected, because BP-Impute assumes that there is only one recombination event within an ambiguous interval and it is possible that in some intervals there are multiple recombination events. R/QTL (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Broman <em>et al.</em> 2003</a>; <a id="xref-ref-8-8" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>) imputes the missing regions via the Viterbi algorithm (method 2), and may introduce additional breakpoints absent in the BP-Impute datasets. This results in the larger average two-point genetic maps for R/QTL, at 1554.5 cM (SD ± 161.7 cM). Mapmaker/EXP3.0 (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Lincoln <em>et al.</em> 1993</a>), with multipoint genetic maps (method 1), produced the largest maps at 2177.0 cM (SD ± 352.8 cM) without error detection (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Lincoln and Lander 1992</a>). Method 1 was shorter than method 2 and comparable to method 3, at 1494.2 cM (SD ± 218.2 cM) with error detection (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The large disparity between the error-corrected datasets indicates that many of the variants within the ambiguous and unimputed regions, if not checked for accuracy, may induce erroneous recombination results.</p><p id="p-62">The joint dataset has a smaller genetic map size (1348.7 cM) than the average of the 10 populations (1430.8 cM) (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> and Tables S2 and S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>). Since the joint dataset was imputed with R/QTL argmax (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Broman <em>et al.</em> 2003</a>; <a id="xref-ref-8-9" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>), residual heterozygosity was imputed as homozygous. This results in false homozygous genotypes and the smaller genetic map. Therefore, the joint dataset has limitations in genotype accuracy and the number of detected recombination events that the individual population datasets do not possess. However, because the joint dataset contains genotypes from all populations in a single flat text file, it facilitates an initial joint QTL mapping survey.</p><p id="p-63">The greatest strength of using the BP-Impute dataset is the preservation of residual heterozygosity. Unlike R/QTL argmax, which imputes heterozygous regions as homozygous, LB-Impute and BP-Impute parse heterozygous regions and impute their breakpoints. This could be significant in the rice NAM datasets, as many NAM lines had residual heterozygosity (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Including the residual heterozygosity in the genetic map calculations allowed comparison with methods 1 and 2, which assume infinite number of selfing generations in the RILs, and increased the average map size from 1210.2 to 1430.8 cM (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>).</p><p id="p-64">Should the 3000 Rice Genomes Project data (<a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Rice Genomes Project 2014</a>) have been available during the creation of these NAM lines, an improved experimental design would have recruited parental lines more representative of <em>tropical japonica</em> diversity and with less <em>indica</em> admixture. A resource such as the 3000 Rice Genomes Project is essential to ensure that a panel of parental lines fully maximizes the genetic diversity found in a phylogenetic group of rice. Our PCA (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) indicates that some of the diversity donors (CT8556-37-1-3-1-M and CT10035-26-4-2-M in particular) appear to have been heavily admixed with <em>indica</em>. As long as the admixed lines harbor traits of interest also present in the other, unadmixed parentals, the admixed lines warrant inclusion in the founder panel to facilitate joint mapping analysis. However, if the primary goal is to create a diversity archive as well as a mapping population, admixture between the diversity donors and the common, reference parent should be avoided.</p><p id="p-65">The 10 populations described in this study are well suited for detailed study of segregation distortion, on an individual or joint population basis. As an example, we focused on two regions on chromosome 3. Chromosome 3 is known to contain a multitude of segregation distortion loci, especially in the context of <em>indica</em> × <em>japonica</em> crosses—which are known to segregate for multiple sterility genes—such as <em>ga2</em>, <em>ga3</em>, and <em>S34</em> (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lin <em>et al.</em> 1992</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Harushima <em>et al.</em> 1996</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Xu <em>et al.</em> 1997</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Lu <em>et al.</em> 2000</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Matsushita <em>et al.</em> 2003</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Zhang <em>et al.</em> 2005</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wu <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kim <em>et al.</em> 2014</a>). In many <em>indica</em> × <em>japonica</em> populations, such as F<sub>2</sub> (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Harushima <em>et al.</em> 1996</a>; <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Xu <em>et al.</em> 1997</a>), BC (<a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Xu <em>et al.</em> 1997</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Kim <em>et al.</em> 2014</a>), and RIL (<a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">Xu <em>et al.</em> 1997</a>), the <em>indica</em> allele tends to be favored in regions of segregation distortion. In our study, we showed that the majority of segregation distortion loci favor the IR64 <em>indica</em> allele in multiple populations (Figure S1 and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>), whereas there were <em>japonica</em>-favored regions on chromosomes 1, 4, and 6 that favored one population. In order to demonstrate the ability of the NAM populations to identify gene candidates for distortion factors, both within an individual population and jointly across several, we further inspected two loci on chromosome 3. Gene candidates identified by this study may contribute to the architecture of previously known segregation distortion on chromosome 3.</p><p id="p-66">The first population-specific segregation distortion peak for IR64 × CT10045-5-5-M-1 was found at 7.6 Mb. A second, shared segregation distortion region among six populations was found at 13.3 Mb on chromosome 3 (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). The 1.5-χ<sup>2</sup> support intervals for these two regions from population-specific and pooled χ<sup>2</sup> tests were 94,269 bp (population-specific, at Chr3:7.6 Mb) and 16,015 (shared, at Chr3:13.3 Mb). Under the assumption that there is only one shared distortion factor in this interval, we examined the support interval for the pooled χ<sup>2</sup> values. In the vicinity of 13.3 Mb, the pooled 1.5-χ<sup>2</sup> support interval included two genes, one of which, LOC_Os03g23030, is homologous to the maize pollen gene <em>aberrant pollen 1</em> (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Xu and Dooner 2006</a>). The close proximity of this pollen gene homolog to the peak distortion signal makes it a gene of interest for further examination of a potential role in gametic selection or sterility in rice.</p><p id="p-67">For the IR64 × CT10045-5-5-M-1-specific interval at 7.6 Mb, there are 16 genes within the 1.5-χ<sup>2</sup> support interval. At least two are of special interest for their role in pollen tube formation. One, LOC_Os03g13860, is a ras-related protein. Ras-related proteins are in the family of Rab GTP-binding proteins (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Cheung and Wu 2008</a>), and the Rab GTPase <em>RabA4d</em> in <em>Arabidopsis</em> regulates pollen tube growth (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Szumlanski and Nielsen 2009</a>). A second gene in this interval is the Actin Depolymerizing Factor (<em>ADF</em>) LOC_Os03g13950. The actin cytoskeleton is a crucial component of pollen tube elongation (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Yokota and Shimmen 2006</a>).</p><p id="p-68">The NAM datasets also proved useful tool for complex trait mapping. We examined the QTL architecture of the first 15 Mb of chromosome 3, for the trait days to heading, both in individual populations and jointly (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7</a> and Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1/FileS1.docx">File S1</a>). The short arm of chromosome 3 is especially rich with photoperiod genes and days to heading QTL (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lee and An 2015a</a>,<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">b</a>). Four genes in particular have been well described. The CCCH-type zinc finger protein, early heading date 4 (Ehd4), has been shown to upregulate activity of the florigen genes <em>Ed3a</em> and <em>RFT1</em> (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Gao <em>et al.</em> 2013</a>). The MIKC-type MADS-box protein (OsMADS50) is a flowering activator, and interacts with the <em>OsGI</em>–<em>Hd1</em>–<em>Hd3a</em> flowering pathway (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Lee <em>et al.</em> 2004</a>). The <em>DNA-binding with one finger 12</em> gene (<em>OsDof12</em>) is a transcription factor that controls the expression of <em>Hd3a</em> (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Li <em>et al.</em> 2009</a>). <em>Rice phytochrome B</em> (<em>OsPhyB</em>), one of three phytochrome genes found in rice (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Takano <em>et al.</em> 2005</a>), has also been implicated in photoperiod monitoring and is also found in this interval. In our study, we examined population-specific and shared QTL peaks in the vicinity of these four genes.</p><p id="p-69">Significant QTL peaks adjacent to three of the four known photoperiod genes in this region were identified. For one of those genes, <em>OsMADS50</em>, we further investigated whether there may be other genes of interest in this region besides <em>OsMADS50</em>. We calculated the support intervals for each individual population, and examined the region of their greatest consensus. When 1.5-<em>F</em> value support intervals were measured for each individual population, it appeared that there was a disjoint QTL structure in this region, with the interval of greatest consensus being an 85,226 bp region downstream of <em>OsMADS50</em>. This consensus support interval contains a protein receptor-like kinase with homology to <em>Arabidopsis</em> <em>CURVY1</em> (MSU rice BLAST: <em>e</em> score 10<sup>−160</sup>, top query coverage 96.32, top Id 38.57) implicated in flowering time control (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Gachomo <em>et al.</em> 2014</a>). When the 3-<em>F</em> value support intervals were used, an expanded consensus interval of 235,514 bp containing <em>OsMADS50</em> was identified. Notably, in a previous genome-wide association study of days to heading in elite tropical rice lines, a peak association signal was also detected 800 kb downstream from <em>OsMADS50</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Begum <em>et al.</em> 2015</a>). The region surrounding <em>OsMADS50</em> may therefore contain additional gene candidates that have not yet been described.</p><p id="p-70"><em>OsPhyB</em> did not reach significant QTL thresholds in joint analysis, and no population exhibited strong marginal effects. This lack of signal is interesting because of <em>OsPhyB’</em>s previous implication in flowering time control by repressing flowering under long day conditions (<a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Takano <em>et al.</em> 2005</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jeong <em>et al.</em> 2007</a>) and the gene’s polymorphism between the <em>tropical japonica</em> diversity donors and the <em>indica</em> common parent. Reasons for a lack of QTL signal may include both environmental and genetic factors, in addition to reduced detection power brought about by segregation distortion. The NAM lines were grown and phenotyped in Cali, Colombia, at a latitude of just 3.42° N, where days and nights are nearly equal all year round. This is likely to reduce the effect of long day flowering inhibition by <em>OsPhyB</em>. For genetic causes, nonadditive and epistatic effects have been shown to mask single QTL (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Yamamoto <em>et al.</em> 2000</a>). Finally, segregation distortion has been shown to reduce QTL detection power (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Xu 2008</a>), and <em>OsPhyB</em> exists in a region of segregation distortion on chromosome 3. In future studies of the NAM populations, phenotyping will occur in a variety of environments, and nonadditive genetic effects will be investigated to further explain the genetic basis of complex traits such as days to heading.</p><p id="p-71">The genotypes and germplasm used in this study are freely available for use in research. All imputation software is available online. Any auxiliary scripts used to process the data are also available upon request. Parental, offspring, and breakpoint imputation used here have also been included as part of the latest release of MapDisto 2.0 (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Lorieux 2012</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Heffelfinger <em>et al.</em> 2017</a>).</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-72">François Sabot (Institut de Recherche pour le Développement) facilitated the inclusion of the NAM parents in the list of IRIGIN (International Rice Genomic Initiative) lines. Edgar Torres and Silvio James Carabali [International Center for Tropical Agriculture (CIAT)] helped with initial F<sub>1</sub> crosses of the NAM lines. Andrés Gutierrez (CIAT) performed Simple Sequence Repeat analysis of the NAM parental lines. Yingchun Tong (Yale University) provided crucial technical assistance in the creation of flexible and scalable Genotype-by-Sequencing genome libraries. This project was supported by National Science Foundation awards 1444478 and 1419501, and by Biomedical Informatics Research Training at Yale (project T15 LM 007056). Computational analyses and data processing were performed on the Yale University High Performance Computing Cluster supported by National Institutes of Health grants RR19895 and RR029676-01. NAM population development was funded by the Generation Challenge Program and the Global Rice Science Partnership. </p><p id="p-73">Author contributions: C.A.F. performed read alignment; variant calling; variant filtering; LB-Impute; BP-Impute; retrieving, merging, and imputing 3000 Rice Genomes Data; principal component analysis; recombination analysis; genetic map calculation; segregation distortion analysis; joint segregation distortion analysis; cleaning phenotype data; joint trait mapping; drafted the manuscript; and created figures. M.M. was responsible for the development, coordinating, and training for NAM library protocols. Z.W. was responsible for statistical advice, specifically regarding BP-Impute. C.H. developed LB-Impute, IR64 × Azucena calibration, and the VarFilt program. L.J.A. developed NAM lines in the field. J.A.A., L.E.R., and N.F. developed NAM library prep. K.L. developed whole-genome sequencing libraries and sequenced NAM parentals. H.Z. was responsible for statistical advice, especially with joint analysis methodology, and edited the manuscript. S.L.D. advised and coordinated genomics and computational biology, training for NAM library prep, and edited the manuscript. M.L. advised, conceived, and coordinated the NAM populations, genomic analysis, computed multipoint maps, and edited the manuscript. The authors declare no competing financial interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-7"><p id="p-74">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041608/-/DC1</a>.</p></li><li class="fn" id="fn-8"><p id="p-75"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-02-06"><span class="received-label">Received </span>February 6, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-04-14"><span class="accepted-label">Accepted </span>April 14, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Fragoso <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.1" data-doi="10.1093/nar/gku1039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexandrov</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mansueto</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">SNP-Seek database of SNPs derived from 3000 rice genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D1023</span>–<span class="cit-lpage">D1027</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1039%26rft_id%253Dinfo%253Apmid%252F25429973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0My9EMS9EMTAyMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xOTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.2" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Auwera</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">del Angel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>  <span class="cit-article-title">From FastQ data to high confidence variant calls: the genome analysis toolkit best practices pipeline.</span> <abbr class="cit-jnl-abbrev">Curr. Protoc. Bioinformatics</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">11.10.1</span>–<span class="cit-lpage">11.10.33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BBioinformatics%26rft.volume%253D43%26rft.spage%253D11101%26rft_id%253Dinfo%253Apmid%252F25431634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25431634&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bajgain</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rouse</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsilo</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macharia</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhavani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Nested association mapping of stem rust resistance in wheat using genotyping by sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0155760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0155760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.5" data-doi="10.1371/journal.pone.0119873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Begum</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lalusin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borromeo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregorio</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide association mapping for yield and other agronomic traits in an elite breeding population of tropical rice (<em>Oryza sativa</em>).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0119873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0119873%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0119873%26rft_id%253Dinfo%253Apmid%252F25785447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0119873&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25785447&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.6" data-doi="10.1007/s00122-008-0780-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourgis</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guyot</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gherbi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tailliez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amabile</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Characterization of the major fragrance gene from an aromatic japonica rice and analysis of its diversity in Asian cultivated rice.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBourgis%26rft.auinit1%253DF.%26rft.volume%253D117%26rft.issue%253D3%26rft.spage%253D353%26rft.epage%253D368%26rft.atitle%253DCharacterization%2Bof%2Bthe%2Bmajor%2Bfragance%2Bgene%2Bfrom%2Ban%2Baromatic%2Bjaponica%2Brice%2Band%2Banalysis%2Bof%2Bits%2Bdiversity%2Bin%2BAsian%2Bcultivated%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0780-9%26rft_id%253Dinfo%253Apmid%252F18491070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0780-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18491070&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257585200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1913.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brennan</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malabayabas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source"><em>International Rice Research Institute’s Contribution to Rice Varietal Yield Improvement in South-East Asia</em></span>. <span class="cit-publ-name">Australian Centre for International Agricultural Research</span>, <span class="cit-publ-loc">Canberra, Australia</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1913.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>A Guide to QTL Mapping with R/qtl</em></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.9" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xOTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.10" data-doi="10.1126/science.1174276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetic architecture of maize flowering time.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D714%26rft.epage%253D718%26rft.atitle%253DThe%2BGenetic%2BArchitecture%2Bof%2BMaize%2BFlowering%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174276%26rft_id%253Dinfo%253Apmid%252F19661422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.11" data-doi="10.1038/514S58a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Callaway</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Domestication: the birth of rice.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">514</span>: <span class="cit-fpage">S58</span>–<span class="cit-lpage">S59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D514%26rft.spage%253DS58%26rft_id%253Dinfo%253Adoi%252F10.1038%252F514S58a%26rft_id%253Dinfo%253Apmid%252F25368889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/514S58a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25368889&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterization of epistatic interaction of QTLs LH8 and EH3 controlling heading date in rice.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">4263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D4%26rft.spage%253D4263%26rft_id%253Dinfo%253Apmid%252F24584028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24584028&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.13" data-doi="10.1105/tpc.010485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Presting</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbazuk</span>  <span class="cit-name-given-names">W. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goicoechea</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackmon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">An integrated physical and genetic map of the rice genome.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.010485%26rft_id%253Dinfo%253Apmid%252F11910002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE0LzMvNTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.14" data-doi="10.1105/tpc.003079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langdon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ouyang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buell</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional rice centromeres are marked by a satellite repeat and a centromere-specific retrotransposon.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1691</span>–<span class="cit-lpage">1704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.003079%26rft_id%253Dinfo%253Apmid%252F12172016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE0LzgvMTY5MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xOTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.15" data-doi="10.1146/annurev.arplant.59.032607.092921"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.-M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Structural and signaling networks for the polar cell growth machinery in pollen tubes.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bplant%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BPlant%2BBiol%26rft.aulast%253DCheung%26rft.auinit1%253DA.%2BY.%26rft.volume%253D59%26rft.spage%253D547%26rft.epage%253D572%26rft.atitle%253DStructural%2Band%2Bsignaling%2Bnetworks%2Bfor%2Bthe%2Bpolar%2Bcell%2Bgrowth%2Bmachinery%2Bin%2Bpollen%2Btubes.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.59.032607.092921%26rft_id%253Dinfo%253Apmid%252F18444907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.59.032607.092921&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18444907&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256593200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Civáň</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Craig</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Three geographically separate domestications of Asian rice.</span> <abbr class="cit-jnl-abbrev">Nat. Plants</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">15164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BPlants%26rft.volume%253D1%26rft.spage%253D15164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.17" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Djedatin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ndjiondjop</span>  <span class="cit-name-given-names">M.-N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanni</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorieux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdier</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identification of novel major and minor QTLs associated with <em>Xanthomonas oryzae</em> pv. <em>oryzae</em> (African strains) resistance in rice (<em>Oryza sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.19" data-doi="10.1023/A:1010687911374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">C.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kost</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">N.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Molecular identification and characterization of the Arabidopsis AtADF1, AtADF5 and AtADF6 genes.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DDong%26rft.auinit1%253DC.%2BH.%26rft.volume%253D45%26rft.issue%253D5%26rft.spage%253D517%26rft.epage%253D527%26rft.atitle%253DMolecular%2Bidentification%2Band%2Bcharacterization%2Bof%2Bthe%2BArabidopsis%2BAtADF1%252C%2BAtADFS%2Band%2BAtADF6%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010687911374%26rft_id%253Dinfo%253Apmid%252F11414611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010687911374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11414611&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168354600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.20" data-doi="10.1016/j.jplph.2005.01.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Comparative study of rice and <em>Arabidopsis</em> actin-depolymerizing factors gene families.</span> <abbr class="cit-jnl-abbrev">J. Plant Physiol.</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bplant%2Bphysiology%26rft.stitle%253DJ%2BPlant%2BPhysiol%26rft.aulast%253DFeng%26rft.auinit1%253DY.%26rft.volume%253D163%26rft.issue%253D1%26rft.spage%253D69%26rft.epage%253D79%26rft.atitle%253DComparative%2Bstudy%2Bof%2Brice%2Band%2BArabidopsis%2Bactin-depolymerizing%2Bfactors%2Bgene%2Bfamilies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jplph.2005.01.015%26rft_id%253Dinfo%253Apmid%252F16360805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jplph.2005.01.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16360805&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234908600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.21" data-doi="10.1534/genetics.115.182071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fragoso</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heffelfinger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dellaporta</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Imputing genotypes in biallelic populations from low-coverage sequence data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.182071%26rft_id%253Dinfo%253Apmid%252F26715670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAyLzIvNDg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.22" data-doi="10.1186/1471-2229-14-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gachomo</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baptiste</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kefela</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saidel</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kotchoni</span>  <span class="cit-name-given-names">S. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The <em>Arabidopsis CURVY1</em> (<em>CVY1</em>) gene encoding a novel receptor-like protein kinase regulates cell morphogenesis, flowering time and seed production.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D14%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-14-1%26rft_id%253Dinfo%253Apmid%252F24387633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-14-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24387633&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.23" data-doi="10.1371/journal.pgen.1003281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fei</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Ehd4</em> encodes a novel and <em>Oryza</em>-genus-specific regulator of photoperiodic flowering in rice.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003281%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003281%26rft_id%253Dinfo%253Apmid%252F23437005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23437005&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.24" data-doi="10.1534/genetics.104.035642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garris</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tai</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coburn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic structure and diversity in <em>Oryza sativa</em> L.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1631</span>–<span class="cit-lpage">1638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.035642%26rft_id%253Dinfo%253Apmid%252F15654106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8zLzE2MzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.25" data-doi="10.1126/science.1185383"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Godfray</span>  <span class="cit-name-given-names">H. C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beddington</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crute</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haddad</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Food security: the challenge of feeding 9 billion people.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">812</span>–<span class="cit-lpage">818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1185383%26rft_id%253Dinfo%253Apmid%252F20110467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Ny84MTIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.26" data-doi="10.1007/BF00041756"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guiderdoni</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galinato</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luistro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vergara</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Anther culture of tropical japonica× indica hybrids of rice (<em>Oryza sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D62%26rft.spage%253D219%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00041756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00041756&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.27" data-doi="10.1007/s001220050106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harushima</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurata</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Detection of segregation distortions in an indica-japonica rice cross using a high-resolution molecular map.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D92%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050106%26rft_id%253Dinfo%253Apmid%252F24166160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24166160&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harushima</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shomura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A high-density rice genetic linkage map with 2275 markers using a single F2 population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHarushima%26rft.auinit1%253DY.%26rft.volume%253D148%26rft.issue%253D1%26rft.spage%253D479%26rft.epage%253D494%26rft.atitle%253DA%2BHigh-Density%2BRice%2BGenetic%2BLinkage%2BMap%2Bwith%2B2275%2BMarkers%2BUsing%2Ba%2BSingle%2BF2%2BPopulation%26rft_id%253Dinfo%253Apmid%252F9475757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ4LzEvNDc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.29" data-doi="10.1186/1471-2164-15-979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffelfinger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fragoso</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Overton</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mottinger</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Flexible and scalable genotyping-by-sequencing strategies for population studies.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">979</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D979%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-979%26rft_id%253Dinfo%253Apmid%252F25406744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-979&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25406744&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.30" data-doi="10.1093/bioinformatics/btx177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffelfinger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fragoso</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorieux</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Constructing linkage maps in the genomics era with MapDisto 2.0.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btx177</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1913.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hercus</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source"><em>Novoalign</em></span>. <span class="cit-publ-name">Novocraft Technologies</span>, <span class="cit-publ-loc">Selangor</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ismail</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">U. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dar</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackill</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The contribution of submergence-tolerant (Sub1) rice varieties to food security in flood-prone rainfed lowland areas in Asia.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D152%26rft.spage%253D83%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Izawa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The process of rice domestication: a new model based on recent data.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D1%26rft.spage%253D127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Regulation of brassinosteroid responses by phytochrome B in rice.</span> <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">590</span>–<span class="cit-lpage">599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.stitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.aulast%253DJeong%26rft.auinit1%253DD.%2BH.%26rft.volume%253D30%26rft.issue%253D5%26rft.spage%253D590%26rft.epage%253D599%26rft.atitle%253DRegulation%2Bof%2Bbrassinosteroid%2Bresponses%2Bby%2Bphytochrome%2BB%2Bin%2Brice.%26rft_id%253Dinfo%253Apmid%252F17407537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17407537&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1913.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khush</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Origin, dispersal, cultivation and variation of rice</span>, pp. <span class="cit-fpage">25</span>–<span class="cit-lpage">34</span> in <span class="cit-source"><em>Oryza: From Molecule to Plant</em></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Germany</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jang</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">S.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bordiya</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akter</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Analysis of segregation distortion and its relationship to hybrid barriers in rice.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D7%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.37" data-doi="10.1016/j.tig.2007.08.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kovach</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweeney</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">New insights into the history of rice domestication.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">578</span>–<span class="cit-lpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DKovach%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D23%26rft.issue%253D11%26rft.spage%253D578%26rft.epage%253D587%26rft.atitle%253DNew%2Binsights%2Binto%2Bthe%2Bhistory%2Bof%2Brice%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.08.012%26rft_id%253Dinfo%253Apmid%252F17963977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.08.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17963977&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251533600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.38" data-doi="10.1038/ng.747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kump</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belcher</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study of quantitative resistance to southern leaf blight in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKump%26rft.auinit1%253DK.%2BL.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D163%26rft.epage%253D168%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bquantitative%2Bresistance%2Bto%2Bsouthern%2Bleaf%2Bblight%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.747%26rft_id%253Dinfo%253Apmid%252F21217757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217757&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.40" data-doi="10.1111/j.1365-313X.2004.02082.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Functional analyses of the flowering time gene <em>OsMADS50</em>, the putative <em>SUPPRESSOR OF OVEREXPRESSION OF CO 1/AGAMOUS‐LIKE 20</em> (<em>SOC1/AGL20</em>) ortholog in rice.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">754</span>–<span class="cit-lpage">764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DLee%26rft.auinit1%253DS.%26rft.volume%253D38%26rft.issue%253D5%26rft.spage%253D754%26rft.epage%253D764%26rft.atitle%253DFunctional%2Banalyses%2Bof%2Bthe%2Bflowering%2Btime%2Bgene%2BOsMADS50%252C%2Bthe%2Bputative%2BSUPPRESSOR%2BOF%2BOVEREXPRESSION%2BOF%2BCO%2B1%252FAGAMOUS-LIKE%2B20%2B%2528SOC1%252FAGL20%2529%2Bortholog%2Bin%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2004.02082.x%26rft_id%253Dinfo%253Apmid%252F15144377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2004.02082.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15144377&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221451800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Complex regulatory networks of flowering time in rice.</span> <abbr class="cit-jnl-abbrev">J. Rice Res.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BRice%2BRes.%26rft.volume%253D3%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.42" data-doi="10.1007/s12374-015-0425-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">Y.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Regulation of flowering time in rice.</span> <abbr class="cit-jnl-abbrev">J. Plant Biol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BPlant%2BBiol.%26rft.volume%253D58%26rft.spage%253D353%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12374-015-0425-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12374-015-0425-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Analysis of recombination QTLs, segregation distortion, and epistasis for fitness in maize multiple populations using ultra-high-density markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1775</span>–<span class="cit-lpage">1784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D1775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.44" data-doi="10.1007/s00425-009-0893-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Functional characterization of rice OsDof12.</span> <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">229</span>: <span class="cit-fpage">1159</span>–<span class="cit-lpage">1169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.volume%253D229%26rft.spage%253D1159%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00425-009-0893-7%26rft_id%253Dinfo%253Apmid%252F19198875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00425-009-0893-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19198875&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265540100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.45" data-doi="10.1093/jxb/erp280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">G.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Three cotton genes preferentially expressed in flower tissues encode actin-depolymerizing factors which are involved in F-actin dynamics in cells.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferp280%26rft_id%253Dinfo%253Apmid%252F19861654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjc6IjYxLzEvNDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikehashi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanagihara</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawashima</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Segregation distortion via male gametes in hybrids between Indica and Japonica or wide-compatibility varieties of rice (<em>Oryza sativa</em> L).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">812</span>–<span class="cit-lpage">818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D84%26rft.spage%253D812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.47" data-doi="10.1007/s00425-006-0460-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffith</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Origin of seed shattering in rice (<em>Oryza sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">226</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.stitle%253DPlanta%26rft.aulast%253DLin%26rft.auinit1%253DZ.%26rft.volume%253D226%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D20%26rft.atitle%253DOrigin%2Bof%2Bseed%2Bshattering%2Bin%2Brice%2B%2528Oryza%2Bsativa%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00425-006-0460-4%26rft_id%253Dinfo%253Apmid%252F17216230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00425-006-0460-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17216230&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246566700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.48" data-doi="10.1016/S0888-7543(05)80158-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lincoln</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Systematic detection of errors in genetic linkage data.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">604</span>–<span class="cit-lpage">610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLincoln%26rft.auinit1%253DS.%2BE.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D604%26rft.epage%253D610%26rft.atitle%253DSystematic%2Bdetection%2Bof%2Berrors%2Bin%2Bgenetic%2Blinkage%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0888-7543%252805%252980158-2%26rft_id%253Dinfo%253Apmid%252F1427888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0888-7543(05)80158-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1427888&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JW35200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1913.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lincoln</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-source"><em>Constructing Genetic Linkage Maps with MAPMAKER/EXP Version 3.0: A Tutorial and Reference Manual</em></span>. <span class="cit-publ-name">Whitehead Institute for Biomedical Research</span>, <span class="cit-publ-loc">Cambridge, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.50" data-doi="10.1073/pnas.0603152103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Londo</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">Y.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">K.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">T.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaal</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Phylogeography of Asian wild rice, <em>Oryza rufipogon</em>, reveals multiple independent domestications of cultivated rice, <em>Oryza sativa</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">9578</span>–<span class="cit-lpage">9583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0603152103%26rft_id%253Dinfo%253Apmid%252F16766658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzI1Lzk1NzgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.51" data-doi="10.1007/s11032-012-9706-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorieux</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MapDisto: fast and efficient computation of genetic linkage maps.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1231</span>–<span class="cit-lpage">1235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D30%26rft.spage%253D1231%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-012-9706-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-012-9706-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.52" data-doi="10.1023/A:1003929312252"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takabatake</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ikehashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Identification of segregation-distortion-neutral alleles to improve pollen fertility of <em>indica-japonica</em> hybrids in rice (<em>Oryza sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D113%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1003929312252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1003929312252&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.53" data-doi="10.1073/pnas.0403715101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Rapid recent growth and divergence of rice nuclear genomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">12404</span>–<span class="cit-lpage">12410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0403715101%26rft_id%253Dinfo%253Apmid%252F15240870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzM0LzEyNDA0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.54" data-doi="10.1534/genetics.105.047175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">O. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hospital</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Two-and three-locus tests for linkage analysis using recombinant inbred lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047175%26rft_id%253Dinfo%253Apmid%252F16489236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTczLzEvNDUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matsushita</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iseki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuta</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Araki</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Characterization of segregation distortion on chromosome 3 induced in wide hybridization between indica and japonica type rice varieties.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D134%26rft.spage%253D27%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.56" data-doi="10.1186/1471-2164-16-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Draba</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnaithmann</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Modelling the genetic architecture of flowering time control in barley through nested association mapping.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-16-1%26rft_id%253Dinfo%253Apmid%252F25553907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-16-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25553907&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.57" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xOTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.58" data-doi="10.1126/science.1174320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeda</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic properties of the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcMullen%26rft.auinit1%253DM.%2BD.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D737%26rft.epage%253D740%26rft.atitle%253DGenetic%2BProperties%2Bof%2Bthe%2BMaize%2BNested%2BAssociation%2BMapping%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174320%26rft_id%253Dinfo%253Apmid%252F19661427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohanty</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Trends in global rice consumption.</span> <abbr class="cit-jnl-abbrev">Rice Today</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">44</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2BToday%26rft.volume%253D12%26rft.spage%253D44%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.60" data-doi="10.1111/nyas.12540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muthayya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugimoto</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montgomery</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maberly</span>  <span class="cit-name-given-names">G. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An overview of global rice production, supply, trade, and consumption.</span> <abbr class="cit-jnl-abbrev">Ann. N. Y. Acad. Sci.</abbr> <span class="cit-vol">1324</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.volume%253D1324%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnyas.12540%26rft_id%253Dinfo%253Apmid%252F25224455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nyas.12540&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25224455&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.61" data-doi="10.1038/hdy.2014.123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ogut</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bian</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Joint-multiple family linkage analysis predicts within-family variation better than single-family analysis of the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">552</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D552%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.123%26rft_id%253Dinfo%253Apmid%252F25585918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25585918&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.62" data-doi="10.1093/nar/gkl976"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ouyang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The TIGR rice genome annotation resource: improvements and new features.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">D883</span>–<span class="cit-lpage">D887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl976%26rft_id%253Dinfo%253Apmid%252F17145706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNS9zdXBwbF8xL0Q4ODMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.63" data-doi="10.1371/journal.pone.0001036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peirce</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A simple method for combining genetic mapping data from multiple crosses and experimental designs.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e1036</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeirce%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253De1036%26rft.epage%253De1036%26rft.atitle%253DA%2Bsimple%2Bmethod%2Bfor%2Bcombining%2Bgenetic%2Bmapping%2Bdata%2Bfrom%2Bmultiple%2Bcrosses%2Band%2Bexperimental%2Bdesigns.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001036%26rft_id%253Dinfo%253Apmid%252F17940600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0001036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17940600&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.64" data-doi="10.2135/cropsci1999.3961552x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cassman</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Virmani</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheehy</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khush</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Yield potential trends of tropical rice since the release of IR8 and the challenge of increasing rice yield potential.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1552</span>–<span class="cit-lpage">1559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DPeng%26rft.auinit1%253DS.%26rft.volume%253D39%26rft.issue%253D6%26rft.spage%253D1552%26rft.epage%253D1559%26rft.atitle%253DYield%2BPotential%2BTrends%2Bof%2BTropical%2BRice%2Bsince%2Bthe%2BRelease%2Bof%2BIR8%2Band%2Bthe%2BChallenge%2Bof%2BIncreasing%2BRice%2BYield%2BPotential%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1999.3961552x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1999.3961552x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000084787700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.65" data-doi="10.1073/pnas.1010894108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide nested association mapping of quantitative resistance to northern leaf blight in maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">6893</span>–<span class="cit-lpage">6898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1010894108%26rft_id%253Dinfo%253Apmid%252F21482771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE3LzY4OTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-web" id="cit-7.6.1913.66"><div class="cit-metadata unstructured">R Core Team, 2014 The R project for statistical computing. Available at: <a href="http://www.R-project.org/">www. R-project. org/</a>. Accessed: October 31, 2014.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.67" data-doi="10.1186/gb-2014-15-1-r1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maron</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wences</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurtowski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole genome <em>de novo</em> assemblies of three divergent strains of rice, <em>Oryza sativa</em>, document novel gene space of <em>aus</em> and <em>indica</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-1-r1%26rft_id%253Dinfo%253Apmid%252F24393533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-1-r1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24393533&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.68" data-doi="10.1007/s00122-013-2166-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cobb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gage</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2699</span>–<span class="cit-lpage">2716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2699%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2166-x%26rft_id%253Dinfo%253Apmid%252F23918062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2166-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23918062&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero Navarro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Novel methods to optimize genotypic imputation for low-coverage, next-generation sequence data in crop plants.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">7</span>. Available at: <a href="https://dl.sciencesocieties.org/publications/tpg/abstracts/7/3/plantgenome2014.05.0023">https://dl.sciencesocieties.org/publications/tpg/abstracts/7/3/plantgenome2014.05.0023</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.70" data-doi="10.1093/aob/mcm128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sweeney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The complex history of the domestication of rice.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">951</span>–<span class="cit-lpage">957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcm128%26rft_id%253Dinfo%253Apmid%252F17617555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjk6IjEwMC81Lzk1MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xOTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.71" data-doi="10.1105/tpc.108.060277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szumlanski</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Rab GTPase RabA4d regulates pollen tube tip growth in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">526</span>–<span class="cit-lpage">544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.108.060277%26rft_id%253Dinfo%253Apmid%252F19208902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvNTI2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.72" data-doi="10.1105/tpc.105.035899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takano</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inagaki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuzurihara</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hihara</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Distinct and cooperative functions of phytochromes A, B, and C in the control of deetiolation and flowering in rice.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">3311</span>–<span class="cit-lpage">3325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.105.035899%26rft_id%253Dinfo%253Apmid%252F16278346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMi8zMzExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.73" data-doi="10.1126/science.1183700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tester</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Breeding technologies to increase crop production in a changing world.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">818</span>–<span class="cit-lpage">822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTester%26rft.auinit1%253DM.%26rft.volume%253D327%26rft.issue%253D5967%26rft.spage%253D818%26rft.epage%253D822%26rft.atitle%253DBreeding%2BTechnologies%2Bto%2BIncrease%2BCrop%2BProduction%2Bin%2Ba%2BChanging%2BWorld%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1183700%26rft_id%253Dinfo%253Apmid%252F20150489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Ny84MTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.74"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">The 3000 rice genomes project</span> <span class="cit-pub-date">2014</span> <span class="cit-article-title">GigaScience</span> <span class="cit-vol">3</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.75" data-doi="10.1016/j.biocon.2012.01.068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tscharntke</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clough</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wanger</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Motzke</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Global food security, biodiversity conservation and the future of agricultural intensification.</span> <abbr class="cit-jnl-abbrev">Biol. Conserv.</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BConserv.%26rft.volume%253D151%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biocon.2012.01.068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biocon.2012.01.068&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000306042600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.76" data-doi="10.1111/j.1601-5223.2010.02120.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ko</span>  <span class="cit-name-given-names">P. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Comparative analyses of linkage maps and segregation distortion of two F2 populations derived from japonica crossed with indica rice.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DWu%26rft.auinit1%253DY.%2BP.%26rft.volume%253D147%26rft.issue%253D5%26rft.spage%253D225%26rft.epage%253D236%26rft.atitle%253DComparative%2Banalyses%2Bof%2Blinkage%2Bmaps%2Band%2Bsegregation%2Bdistortion%2Bof%2Btwo%2BF2%2Bpopulations%2Bderived%2Bfrom%2Bjaponica%2Bcrossed%2Bwith%2Bindica%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-5223.2010.02120.x%26rft_id%253Dinfo%253Apmid%252F21039459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-5223.2010.02120.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21039459&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.77" data-doi="10.1534/genetics.108.090688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Quantitative trait locus mapping can benefit from segregation distortion.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">2201</span>–<span class="cit-lpage">2208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090688%26rft_id%253Dinfo%253Apmid%252F18957707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC80LzIyMDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.78" data-doi="10.1007/s004380050355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Chromosomal regions associated with segregation distortion of molecular markers in F<sub>2</sub>, backcross, doubled haploid, and recombinant inbred populations in rice (<em>Oryza sativa</em> L.).</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">253</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DXu%26rft.auinit1%253DY.%26rft.volume%253D253%26rft.issue%253D5%26rft.spage%253D535%26rft.epage%253D545%26rft.atitle%253DChromosomal%2Bregions%2Bassociated%2Bwith%2Bsegregation%2Bdistortion%2Bof%2Bmolecular%2Bmarkers%2Bin%2BF2%252C%2Bbackcross%252C%2Bdoubled%2Bhaploid%252C%2Band%2Brecombinant%2Binbred%2Bpopulations%2Bin%2Brice%2B%2528Oryza%2Bsativa%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs004380050355%26rft_id%253Dinfo%253Apmid%252F9065686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s004380050355&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9065686&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WM03600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.79" data-doi="10.1534/genetics.105.050237"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dooner</span>  <span class="cit-name-given-names">H. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The maize aberrant pollen transmission 1 gene is a SABRE/KIP homolog required for pollen tube growth.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.050237%26rft_id%253Dinfo%253Apmid%252F16299389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8yLzEyNTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yano</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Identification of heading date quantitative trait locus Hd6 and characterization of its epistatic interactions with Hd2 in rice using advanced backcross progeny.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">885</span>–<span class="cit-lpage">891</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DYamamoto%26rft.auinit1%253DT.%26rft.volume%253D154%26rft.issue%253D2%26rft.spage%253D885%26rft.epage%253D891%26rft.atitle%253DIdentification%2Bof%2BHeading%2BDate%2BQuantitative%2BTrait%2BLocus%2BHd6%2Band%2BCharacterization%2Bof%2BIts%2BEpistatic%2BInteractions%2BWith%2BHd2%2Bin%2BRice%2BUsing%2BAdvanced%2BBackcross%2BProgeny%26rft_id%253Dinfo%253Apmid%252F10655238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU0LzIvODg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1913.81"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yokota</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimmen</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The actin cytoskeleton in pollen tubes; actin and actin binding proteins</span>, pp. <span class="cit-fpage">139</span>–<span class="cit-lpage">155</span> in <span class="cit-source"><em>The Pollen Tube</em></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.82" data-doi="10.1534/genetics.107.074245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic design and statistical power of nested association mapping in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D539%26rft.epage%253D551%26rft.atitle%253DGenetic%2BDesign%2Band%2BStatistical%2BPower%2Bof%2BNested%2BAssociation%2BMapping%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074245%26rft_id%253Dinfo%253Apmid%252F18202393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5MTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.83" data-doi="10.1016/S1673-8527(07)60037-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Molecular characterization of four ADF genes differentially expressed in cotton.</span> <abbr class="cit-jnl-abbrev">J. Genet. Genomics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BGenet%2BGenomics%26rft.aulast%253DZhang%26rft.auinit1%253DC.%26rft.volume%253D34%26rft.issue%253D4%26rft.spage%253D347%26rft.epage%253D354%26rft.atitle%253DMolecular%2Bcharacterization%2Bof%2Bfour%2BADF%2Bgenes%2Bdifferentially%2Bexpressed%2Bin%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1673-8527%252807%252960037-X%26rft_id%253Dinfo%253Apmid%252F17498633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1673-8527(07)60037-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17498633&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1913.84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A new sterile gene from <em>Oryza glaberrima</em> on chromosome 3.</span> <abbr class="cit-jnl-abbrev">Rice Genet. News</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">26</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2BGenet.%2BNews%26rft.volume%253D22%26rft.spage%253D26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/6/1913.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1899" title="Adaptive Roles of SSY1 and SIR3 During Cycles of Growth and Starvation in Saccharomyces cerevisiae Populations Enriched for Quiescent or Nonquiescent Cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1927" title="A Chromosome-Scale Assembly of the Bactrocera cucurbitae Genome Provides Insight to the Genetic Basis of white pupae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/6"><span>Volume 7 Issue 6, June 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/6.cover-source.jpg?itok=5_J-OKHc" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1913" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9pbyKrKvo_fAJ-9zlOA4Sk8wou_VFwh7CmAECDZylDo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/6/1913" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/6/1913" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/6/1913" class="active" data-icon-position="" data-hide-link-title="0">Genetic Architecture of a Rice Nested Association Mapping Population</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484771" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-V1uhdpi3ZLbbSKtVh-tIWzz-VVN2dCmEobLH_Bxpo3Q" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1913.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/6/1913" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-72y2Jl4UlaP3GHDu1eivLjN6A8GAf2GkJ2h-4XbO-I4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/6/1913" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1913.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484771" id="node484771--2" data-pisa="ggg;7/6/1913" data-pisa-master="ggg;g3.117.041608" data-apath="/ggg/7/6/1913.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Architecture of a Rice Nested Association Mapping Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christopher A. Fragoso</span>, <span class="highwire-citation-author" data-delta="1">Maria Moreno</span>, <span class="highwire-citation-author" data-delta="2">Zuoheng Wang</span>, <span class="highwire-citation-author" data-delta="3">Christopher Heffelfinger</span>, <span class="highwire-citation-author" data-delta="4">Lady J. Arbelaez</span>, <span class="highwire-citation-author" data-delta="5">John A. Aguirre</span>, <span class="highwire-citation-author" data-delta="6">Natalia Franco</span>, <span class="highwire-citation-author" data-delta="7">Luz E. Romero</span>, <span class="highwire-citation-author" data-delta="8">Karine Labadie</span>, <span class="highwire-citation-author" data-delta="9">Hongyu Zhao</span>, <span class="highwire-citation-author" data-delta="10">Stephen L. Dellaporta</span> and <span class="highwire-citation-author" data-delta="11">Mathias Lorieux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1913-1926;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041608 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484771--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher A. Fragoso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut 06511</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BA.%2BFragoso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fragoso%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BA.%2BFragoso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maria Moreno </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BMoreno%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreno%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BMoreno%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zuoheng Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut 06511</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biostatistics, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zuoheng%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZuoheng%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christopher Heffelfinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BHeffelfinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heffelfinger%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BHeffelfinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lady J. Arbelaez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lady%2BJ.%2BArbelaez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arbelaez%20LJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALady%2BJ.%2BArbelaez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">John A. Aguirre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BA.%2BAguirre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aguirre%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BA.%2BAguirre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Natalia Franco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2BFranco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Franco%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatalia%2BFranco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Luz E. Romero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luz%2BE.%2BRomero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Romero%20LE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuz%2BE.%2BRomero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Karine Labadie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Commissariat à L’énergie Atomique et aux Énergies Alternatives, Institut de Génomique, Genoscope, 91000 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karine%2BLabadie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Labadie%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarine%2BLabadie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Hongyu Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut 06511</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biostatistics, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hongyu%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHongyu%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Stephen L. Dellaporta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BL.%2BDellaporta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dellaporta%20SL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BL.%2BDellaporta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Mathias Lorieux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Diversité, Adaptation, Développement des Plantes Research Unit, Institut de Recherche pour le Développement, F-34394 Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathias%2BLorieux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorieux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMathias%2BLorieux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mathias.lorieux@ird.fr" class="" data-icon-position="" data-hide-link-title="0">mathias.lorieux@ird.fr</a>
<a href="mailto:m.lorieux@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">m.lorieux@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/6/1913" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/6/1913&amp;title=Genetic%20Architecture%20of%20a%20Rice%20Nested%20Association%20Mapping%20Population" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/6/1913&amp;title=Genetic%20Architecture%20of%20a%20Rice%20Nested%20Association%20Mapping%20Population" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/6/1913&amp;title=Genetic%20Architecture%20of%20a%20Rice%20Nested%20Association%20Mapping%20Population" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/6/1913&amp;text=Genetic%20Architecture%20of%20a%20Rice%20Nested%20Association%20Mapping%20Population" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/6/1913&amp;title=Genetic%20Architecture%20of%20a%20Rice%20Nested%20Association%20Mapping%20Population" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/6/1913&amp;t=Genetic%20Architecture%20of%20a%20Rice%20Nested%20Association%20Mapping%20Population" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/6/1913&amp;title=Genetic%20Architecture%20of%20a%20Rice%20Nested%20Association%20Mapping%20Population" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/6/1913&amp;title=Genetic%20Architecture%20of%20a%20Rice%20Nested%20Association%20Mapping%20Population" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484771/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484771" id="node484771--3" data-pisa="ggg;7/6/1913" data-pisa-master="ggg;g3.117.041608" data-apath="/ggg/7/6/1913.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Architecture of a Rice Nested Association Mapping Population</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Christopher A. Fragoso</span>, <span class="highwire-citation-author" data-delta="1">Maria Moreno</span>, <span class="highwire-citation-author" data-delta="2">Zuoheng Wang</span>, <span class="highwire-citation-author" data-delta="3">Christopher Heffelfinger</span>, <span class="highwire-citation-author" data-delta="4">Lady J. Arbelaez</span>, <span class="highwire-citation-author" data-delta="5">John A. Aguirre</span>, <span class="highwire-citation-author" data-delta="6">Natalia Franco</span>, <span class="highwire-citation-author" data-delta="7">Luz E. Romero</span>, <span class="highwire-citation-author" data-delta="8">Karine Labadie</span>, <span class="highwire-citation-author" data-delta="9">Hongyu Zhao</span>, <span class="highwire-citation-author" data-delta="10">Stephen L. Dellaporta</span> and <span class="highwire-citation-author" data-delta="11">Mathias Lorieux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1913-1926;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041608 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484771--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christopher A. Fragoso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut 06511</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BA.%2BFragoso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fragoso%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BA.%2BFragoso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maria Moreno </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BMoreno%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moreno%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BMoreno%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zuoheng Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut 06511</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biostatistics, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zuoheng%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZuoheng%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christopher Heffelfinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BHeffelfinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heffelfinger%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BHeffelfinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lady J. Arbelaez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lady%2BJ.%2BArbelaez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arbelaez%20LJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALady%2BJ.%2BArbelaez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">John A. Aguirre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BA.%2BAguirre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aguirre%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BA.%2BAguirre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Natalia Franco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2BFranco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Franco%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatalia%2BFranco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Luz E. Romero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luz%2BE.%2BRomero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Romero%20LE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuz%2BE.%2BRomero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Karine Labadie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Commissariat à L’énergie Atomique et aux Énergies Alternatives, Institut de Génomique, Genoscope, 91000 Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karine%2BLabadie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Labadie%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarine%2BLabadie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Hongyu Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut 06511</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biostatistics, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hongyu%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHongyu%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Stephen L. Dellaporta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut 06511</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BL.%2BDellaporta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dellaporta%20SL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BL.%2BDellaporta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Mathias Lorieux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rice Genetics and Genomics Laboratory, International Center for Tropical Agriculture, Cali 6713, Colombia</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Diversité, Adaptation, Développement des Plantes Research Unit, Institut de Recherche pour le Développement, F-34394 Montpellier, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathias%2BLorieux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorieux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMathias%2BLorieux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mathias.lorieux@ird.fr" class="" data-icon-position="" data-hide-link-title="0">mathias.lorieux@ird.fr</a>
<a href="mailto:m.lorieux@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">m.lorieux@cgiar.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1913&amp;rft.epage=1926&amp;rft.atitle=Genetic%20Architecture%20of%20a%20Rice%20Nested%20Association%20Mapping%20Population&amp;rft.volume=7&amp;rft.issue=6&amp;rft.date=2017-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Fragoso%2C+Christopher+A.&amp;rft.au=Moreno%2C+Maria&amp;rft.au=Wang%2C+Zuoheng&amp;rft.au=Heffelfinger%2C+Christopher&amp;rft.au=Arbelaez%2C+Lady+J.&amp;rft.au=Aguirre%2C+John+A.&amp;rft.au=Franco%2C+Natalia&amp;rft.au=Romero%2C+Luz+E.&amp;rft.au=Labadie%2C+Karine&amp;rft.au=Zhao%2C+Hongyu&amp;rft.au=Dellaporta%2C+Stephen+L.&amp;rft.au=Lorieux%2C+Mathias"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484771/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484771/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484771/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484771/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484771/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484771/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484771/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484771/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484771/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484771/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484771/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484771/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/6/1913" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/6/1913.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/6/1913.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/6/1913.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/6/1913.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
