<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/6/1643" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/6/1643.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/6/1643.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/6/1643.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/6/1643/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/6/1643.atom" />
<meta name="HW.pisa" content="ggg;7/6/1643" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Beavis Effect in Next-Generation Mapping Panels in Drosophila melanogaster" />
<meta name="DC.Identifier" content="10.1534/g3.117.041426" />
<meta name="DC.Date" content="2017-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 King et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="A major goal in the analysis of complex traits is to partition the observed genetic variation in a trait into components due to individual loci and perhaps variants within those loci. However, in both QTL mapping and genetic association studies, the estimated percent variation attributable to a QTL is upwardly biased conditional on it being discovered. This bias was first described in two-way QTL mapping experiments by William Beavis, and has been referred to extensively as “the Beavis effect.” The Beavis effect is likely to occur in multiparent population (MPP) panels as well as collections of sequenced lines used for genome-wide association studies (GWAS). However, the strength of the Beavis effect is unknown—and often implicitly assumed to be negligible—when “hits” are obtained from an association panel consisting of hundreds of inbred lines tested across millions of SNPs, or in multiparent mapping populations where mapping involves fitting a complex statistical model with several d.f. at thousands of genetic intervals. To estimate the size of the effect in more complex panels, we performed simulations of both biallelic and multiallelic QTL in two major Drosophila melanogaster mapping panels, the GWAS-based Drosophila Genetic Reference Panel (DGRP), and the MPP the Drosophila Synthetic Population Resource (DSPR). Our results show that overestimation is determined most strongly by sample size and is only minimally impacted by the mapping design. When &lt; 100, 200, 500, and 1000 lines are employed, the variance attributable to hits is inflated by factors of 6, 3, 1.5, and 1.1, respectively, for a QTL that truly contributes 5% to the variation in the trait. This overestimation indicates that QTL could be difficult to validate in follow-up replication experiments where additional individuals are examined. Further, QTL could be difficult to cross-validate between the two Drosophila resources. We provide guidelines for: (1) the sample sizes necessary to accurately estimate the percent variance to an identified QTL, (2) the conditions under which one is likely to replicate a mapped QTL in a second study using the same mapping population, and (3) the conditions under which a QTL mapped in one mapping panel is likely to replicate in the other (DGRP and DSPR)." />
<meta name="DC.Contributor" content="Elizabeth G. King" />
<meta name="DC.Contributor" content="Anthony D. Long" />
<meta name="article:published_time" content="2017-06-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="The Beavis Effect in Next-Generation Mapping Panels in Drosophila melanogaster" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;A major goal in the analysis of complex traits is to partition the observed genetic variation in a trait into components due to individual loci and perhaps variants within those loci. However, in both QTL mapping and genetic association studies, the estimated percent variation attributable to a QTL is upwardly biased conditional on it being discovered. This bias was first described in two-way QTL mapping experiments by William Beavis, and has been referred to extensively as “the Beavis effect.” The Beavis effect is likely to occur in multiparent population (MPP) panels as well as collections of sequenced lines used for genome-wide association studies (GWAS). However, the strength of the Beavis effect is unknown—and often implicitly assumed to be negligible—when “hits” are obtained from an association panel consisting of hundreds of inbred lines tested across millions of SNPs, or in multiparent mapping populations where mapping involves fitting a complex statistical model with several d.f. at thousands of genetic intervals. To estimate the size of the effect in more complex panels, we performed simulations of both biallelic and multiallelic QTL in two major &lt;i&gt;Drosophila melanogaster&lt;/i&gt; mapping panels, the GWAS-based &lt;i&gt;Drosophila&lt;/i&gt; Genetic Reference Panel (DGRP), and the MPP the &lt;i&gt;Drosophila&lt;/i&gt; Synthetic Population Resource (DSPR). Our results show that overestimation is determined most strongly by sample size and is only minimally impacted by the mapping design. When &amp;lt; 100, 200, 500, and 1000 lines are employed, the variance attributable to hits is inflated by factors of 6, 3, 1.5, and 1.1, respectively, for a QTL that truly contributes 5% to the variation in the trait. This overestimation indicates that QTL could be difficult to validate in follow-up replication experiments where additional individuals are examined. Further, QTL could be difficult to cross-validate between the two &lt;i&gt;Drosophila&lt;/i&gt; resources. We provide guidelines for: (1) the sample sizes necessary to accurately estimate the percent variance to an identified QTL, (2) the conditions under which one is likely to replicate a mapped QTL in a second study using the same mapping population, and (3) the conditions under which a QTL mapped in one mapping panel is likely to replicate in the other (DGRP and DSPR).&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/06/01" />
<meta name="citation_mjid" content="ggg;7/6/1643" />
<meta name="citation_id" content="7/6/1643" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/6/1643" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/6/1643.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/6/1643.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/6/1643.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.041426" />
<meta name="citation_pmid" content="28592647" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="1643" />
<meta name="citation_lastpage" content="1652" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Elizabeth G. King" />
<meta name="citation_author_institution" content="Division of Biological Sciences, University of Missouri, Columbia, Missouri 65211" />
<meta name="citation_author_email" content="kingeg@missouri.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9393-4720" />
<meta name="citation_author" content="Anthony D. Long" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California Irvine, California 92697" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=C. Alonso-Blanco;citation_author=J. Andrade;citation_author=C. Becker;citation_author=F. Bemm;citation_author=J. Bergelson;citation_title=1,135 Genomes reveal the global pattern of polymorphism in Arabidopsis thaliana.;citation_pages=481-491;citation_volume=166;citation_year=2016;citation_pmid=27293186;citation_doi=10.1016/j.cell.2016.05.063" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. Altshuler;citation_author=J. Hirschhorn;citation_author=M. Klannemark;citation_author=C. Lindgren;citation_author=M. Vohl;citation_title=The common PPARgamma Pro12Ala polymorphism is associated with decreased risk of type 2 diabetes.;citation_pages=76-80;citation_volume=26;citation_year=2000;citation_issue=1;citation_pmid=10973253;citation_doi=10.1038/79216" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. L. Aylor;citation_author=W. Valdar;citation_author=W. Foulds-Mathes;citation_author=R. J. Buus;citation_author=R. A. Verdugo;citation_title=Genetic analysis of complex traits in the emerging collaborative cross.;citation_volume=21;citation_year=2011;citation_pmid=21406540;citation_doi=10.1101/gr.111310.110" />
<meta name="citation_reference" content="citation_journal_title=Mol. Dissection Complex Traits;citation_author=W. D. Beavis;citation_title=QTL analyses: power, precision, and accuracy.;citation_pages=145-162;citation_volume=1998;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=W. D. Beavis;citation_author=D. Grant;citation_author=M. Albertsen;citation_author=R. Fincher;citation_title=Quantitative trait loci for plant height in four maize populations and their associations with qualitative genetic loci.;citation_pages=141-145;citation_volume=83;citation_year=1991;citation_pmid=24202349;citation_doi=10.1007/BF00215714" />
<meta name="citation_reference" content="citation_author=K. W. Broman;citation_author=S. Sen;citation_pages=1-399;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Brzyski;citation_author=C. Peterson;citation_author=P. Sobczyk;citation_author=E. Candes;citation_author=M. Bogdan;citation_title=Controlling the rate of GWAS false discoveries.;citation_volume=205;citation_year=2017;citation_pmid=27784720;citation_doi=10.1534/genetics.116.193987" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Chen;citation_author=J. D. Storey;citation_title=Relaxed significance criteria for linkage analysis.;citation_volume=173;citation_year=2006;citation_pmid=16783025;citation_doi=10.1534/genetics.105.052506" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=R. Cheng;citation_author=M. Abney;citation_author=A. A. Palmer;citation_author=A. D. Skol;citation_title=QTLRel: an R package for genome-wide association studies in which relatedness is a concern.;citation_pages=66-66;citation_volume=12;citation_year=2011;citation_pmid=21794153;citation_doi=10.1186/1471-2156-12-66" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=E. J. Chesler;citation_author=D. M. Gatti;citation_author=A. P. Morgan;citation_author=M. Strobel;citation_author=L. Trepanier;citation_title=Diversity outbred mice at 21: maintaining allelic variation in the face of selection.;citation_pages=3893-3902;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_journal_title=R News;citation_author=D. Clifford;citation_author=P. McCullagh;citation_title=The regress function.;citation_pages=6-10;citation_volume=6;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_title=The Genome Architecture of the Collaborative Cross Mouse Genetic Reference Population;citation_pages=389-401;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. A. Cubillos;citation_author=L. Parts;citation_author=F. Salinas;citation_author=A. Bergström;citation_author=E. Scovacricchi;citation_title=High-resolution mapping of complex traits with a four-parent advanced intercross yeast population.;citation_volume=195;citation_year=2013;citation_pmid=24037264;citation_doi=10.1534/genetics.113.155515" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011" />
<meta name="citation_reference" content="citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. A. Flint-Garcia;citation_author=A.-C. Thuillet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=S. M. Romero;citation_title=Maize association population: a high-resolution platform for quantitative trait locus dissection.;citation_pages=1054-1064;citation_volume=44;citation_year=2005;citation_issue=6;citation_pmid=16359397;citation_doi=10.1111/j.1365-313X.2005.02591.x" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. M. Gatti;citation_author=K. L. Svenson;citation_author=A. Shabalin;citation_author=L.-Y. Wu;citation_author=W. Valdar;citation_title=Quantitative trait locus mapping methods for diversity outbred mice.;citation_pages=1623-1633;citation_volume=4;citation_year=2014;citation_doi=10.1534/g3.114.013748" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=H. H. Göring;citation_author=J. D. Terwilliger;citation_author=J. Blangero;citation_title=Large upward bias in estimation of locus-specific effects from genomewide scans.;citation_pages=1357-1369;citation_volume=69;citation_year=2001;citation_issue=6;citation_pmid=11593451;citation_doi=10.1086/324471" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. N. Hirschhorn;citation_author=M. J. Daly;citation_title=Genome-wide association studies for common diseases and complex traits.;citation_pages=95-108;citation_volume=6;citation_year=2005;citation_issue=2;citation_pmid=15716906;citation_doi=10.1038/nrm1583" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Huang;citation_author=S. Richards;citation_author=M. A. Carbone;citation_author=D. Zhu;citation_author=R. R. Anholt;citation_title=Epistasis dominates the genetic architecture of Drosophila quantitative traits.;citation_volume=109;citation_year=2012;citation_pmid=22949659;citation_doi=10.1073/pnas.1213423109" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. Huang;citation_author=A. Massouras;citation_author=Y. Inoue;citation_author=J. Peiffer;citation_author=M. Ràmia;citation_title=Natural variation in genome architecture among 205 Drosophila melanogaster genetic reference panel lines.;citation_volume=24;citation_year=2014;citation_pmid=24714809;citation_doi=10.1101/gr.171546.113" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=X. Huang;citation_author=M.-J. Paulo;citation_author=M. Boer;citation_author=S. Effgen;citation_author=P. Keizer;citation_title=Analysis of natural allelic variation in Arabidopsis using a multiparent recombinant inbred line population.;citation_volume=108;citation_year=2011;citation_pmid=21368205;citation_doi=10.1073/pnas.1100465108" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. G. King;citation_author=C. M. Merkes;citation_author=C. L. McNeil;citation_author=S. R. Hoofer;citation_author=S. Sen;citation_title=Genetic dissection of a model complex trait using the Drosophila synthetic population resource.;citation_volume=22;citation_year=2012;citation_pmid=22496517;citation_doi=10.1101/gr.134031.111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. G. King;citation_author=S. J. Macdonald;citation_author=A. D. Long;citation_title=Properties and power of the Drosophila synthetic population resource for the routine dissection of complex traits.;citation_volume=191;citation_year=2012;citation_pmid=22505626;citation_doi=10.1534/genetics.112.138537" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. G. King;citation_author=G. Kislukhin;citation_author=K. N. Walters;citation_author=A. D. Long;citation_title=Using Drosophila melanogaster To Identify Chemotherapy Toxicity Genes;citation_pages=31-43;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236447;citation_doi=10.1534/genetics.114.161968" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=E. G. King;citation_author=B. J. Sanderson;citation_author=C. L. McNeil;citation_author=A. D. Long;citation_author=S. J. Macdonald;citation_title=Genetic dissection of the Drosophila melanogaster female head transcriptome reveals widespread allelic heterogeneity.;citation_pages=e1004322;citation_volume=10;citation_year=2014;citation_pmid=24810915;citation_doi=10.1371/journal.pgen.1004322" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. X. Kover;citation_author=W. Valdar;citation_author=J. Trakalo;citation_author=N. Scarcelli;citation_author=I. M. Ehrenreich;citation_title=A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in Arabidopsis thaliana.;citation_pages=e1000551-e1000551;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593375;citation_doi=10.1371/journal.pgen.1000551" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Liu;citation_author=Q. Zhou;citation_author=L. Dong;citation_author=H. Wang;citation_author=F. Liu;citation_title=Genetic architecture of the maize kernel row number revealed by combining QTL mapping using a high-density genetic map and bulked segregant RNA sequencing.;citation_pages=915;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=A. D. Long;citation_author=S. J. Macdonald;citation_author=E. G. King;citation_title=Dissecting complex traits using the Drosophila synthetic population resource.;citation_pages=488-495;citation_volume=30;citation_year=2014;citation_pmid=25175100;citation_doi=10.1016/j.tig.2014.07.009" />
<meta name="citation_reference" content="citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. F. C. Mackay;citation_author=S. Richards;citation_author=E. A. Stone;citation_author=A. Barbadilla;citation_author=J. F. Ayroles;citation_title=The Drosophila melanogaster Genetic Reference Panel.;citation_pages=173-178;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22318601;citation_doi=10.1038/nature10811" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. McClellan;citation_author=M.-C. King;citation_title=Genetic heterogeneity in human disease.;citation_pages=210-217;citation_volume=141;citation_year=2010;citation_issue=2;citation_pmid=20403315;citation_doi=10.1016/j.cell.2010.03.032" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. McMullen;citation_author=S. Kresovich;citation_author=H. Sanchez Villeda;citation_author=P. Bradbury;citation_author=H. Li;citation_title=Genetic Properties of the Maize Nested Association Mapping Population;citation_pages=737-740;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661427;citation_doi=10.1126/science.1174320" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome J.;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. K. Pritchard;citation_title=Are rare variants responsible for susceptibility to complex diseases?;citation_pages=124-137;citation_volume=69;citation_year=2001;citation_issue=1;citation_pmid=11404818;citation_doi=10.1086/321272" />
<meta name="citation_reference" content="citation_author=D. A. Roff;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_author=D. O. Siegmund;citation_author=N. R. Zhang;citation_author=B. Yakir;citation_title=False discovery rate for scanning statistics.;citation_volume=98;citation_year=2011;citation_doi=10.1093/biomet/asr057" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=S. l. Slager;citation_author=J. Huang;citation_author=V. j. Vieland;citation_title=Effect of allelic heterogeneity on the power of the transmission disequilibrium test.;citation_pages=143-156;citation_volume=18;citation_year=2000;citation_issue=2;citation_pmid=10642427;citation_doi=10.1002/(SICI)1098-2272(200002)18:2&lt;143::AID-GEPI4&gt;3.0.CO;2-5" />
<meta name="citation_reference" content="citation_title=PLoS Genet.;citation_author=K. R. Thornton;citation_author=A. J. Foran;citation_author=A. D. Long;citation_title=Properties and modeling of GWAS when complex disease risk is due to non-complementing, deleterious mutations in genes of large effect.;citation_pages=e1003258;citation_volume=9;citation_year=2013;citation_pmid=23437004;citation_doi=10.1371/journal.pgen.1003258" />
<meta name="citation_reference" content="citation_title=Comparison of different approaches to interval mapping of quantitative trait loci;citation_author=H. F. Utz;citation_author=A. E. Melchinger;citation_pages=195-203;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. F. Utz;citation_author=A. E. Melchinger;citation_author=C. C. Schön;citation_title=Bias and Sampling Error of the Estimated Proportion of Genotypic Variance Explained by Quantitative Trait Loci Determined From Experimental Data in Maize Using Cross Validation and Validation With Independent Samples;citation_pages=1839-1849;citation_volume=154;citation_year=2000;citation_issue=4;citation_pmid=10866652" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Xavier;citation_author=S. Xu;citation_author=W. M. Muir;citation_author=K. M. Rainey;citation_title=NAM: association studies in multiple populations.;citation_volume=31;citation_year=2015;citation_pmid=26243017;citation_doi=10.1093/bioinformatics/btv448" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Xu;citation_title=Theoretical Basis of the Beavis Effect;citation_pages=2259-2268;citation_volume=165;citation_year=2003;citation_issue=4;citation_pmid=14704201" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=B. Benyamin;citation_author=B. P. McEvoy;citation_author=S. Gordon;citation_author=A. K. Henders;citation_title=Common SNPs explain a large proportion of the heritability for human height.;citation_pages=565-569;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20562875;citation_doi=10.1038/ng.608" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Yang;citation_author=N. A. Zaitlen;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_author=A. L. Price;citation_title=Advantages and pitfalls in the application of mixed-model association methods.;citation_pages=100-106;citation_volume=46;citation_year=2014;citation_pmid=24473328;citation_doi=10.1038/ng.2876" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Beavis Effect in Next-Generation Mapping Panels in Drosophila melanogaster" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="twitter:description" content="A major goal in the analysis of complex traits is to partition the observed genetic variation in a trait into components due to individual loci and perhaps variants within those loci. However, in both QTL mapping and genetic association studies, the estimated percent variation attributable to a QTL is upwardly biased conditional on it being discovered. This bias was first described in two-way QTL mapping experiments by William Beavis, and has been referred to extensively as “the Beavis effect.” The Beavis effect is likely to occur in multiparent population (MPP) panels as well as collections of sequenced lines used for genome-wide association studies (GWAS). However, the strength of the Beavis effect is unknown—and often implicitly assumed to be negligible—when “hits” are obtained from an association panel consisting of hundreds of inbred lines tested across millions of SNPs, or in multiparent mapping populations where mapping involves fitting a complex statistical model with several d.f. at thousands of genetic intervals. To estimate the size of the effect in more complex panels, we performed simulations of both biallelic and multiallelic QTL in two major Drosophila melanogaster mapping panels, the GWAS-based Drosophila Genetic Reference Panel (DGRP), and the MPP the Drosophila Synthetic Population Resource (DSPR). Our results show that overestimation is determined most strongly by sample size and is only minimally impacted by the mapping design. When &lt; 100, 200, 500, and 1000 lines are employed, the variance attributable to hits is inflated by factors of 6, 3, 1.5, and 1.1, respectively, for a QTL that truly contributes 5% to the variation in the trait. This overestimation indicates that QTL could be difficult to validate in follow-up replication experiments where additional individuals are examined. Further, QTL could be difficult to cross-validate between the two Drosophila resources. We provide guidelines for: (1) the sample sizes necessary to accurately estimate the percent variance to an identified QTL, (2) the conditions under which one is likely to replicate a mapped QTL in a second study using the same mapping population, and (3) the conditions under which a QTL mapped in one mapping panel is likely to replicate in the other (DGRP and DSPR)." />
<meta name="og-title" property="og:title" content="The Beavis Effect in Next-Generation Mapping Panels in Drosophila melanogaster" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/6/1643" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="A major goal in the analysis of complex traits is to partition the observed genetic variation in a trait into components due to individual loci and perhaps variants within those loci. However, in both QTL mapping and genetic association studies, the estimated percent variation attributable to a QTL is upwardly biased conditional on it being discovered. This bias was first described in two-way QTL mapping experiments by William Beavis, and has been referred to extensively as “the Beavis effect.” The Beavis effect is likely to occur in multiparent population (MPP) panels as well as collections of sequenced lines used for genome-wide association studies (GWAS). However, the strength of the Beavis effect is unknown—and often implicitly assumed to be negligible—when “hits” are obtained from an association panel consisting of hundreds of inbred lines tested across millions of SNPs, or in multiparent mapping populations where mapping involves fitting a complex statistical model with several d.f. at thousands of genetic intervals. To estimate the size of the effect in more complex panels, we performed simulations of both biallelic and multiallelic QTL in two major Drosophila melanogaster mapping panels, the GWAS-based Drosophila Genetic Reference Panel (DGRP), and the MPP the Drosophila Synthetic Population Resource (DSPR). Our results show that overestimation is determined most strongly by sample size and is only minimally impacted by the mapping design. When &lt; 100, 200, 500, and 1000 lines are employed, the variance attributable to hits is inflated by factors of 6, 3, 1.5, and 1.1, respectively, for a QTL that truly contributes 5% to the variation in the trait. This overestimation indicates that QTL could be difficult to validate in follow-up replication experiments where additional individuals are examined. Further, QTL could be difficult to cross-validate between the two Drosophila resources. We provide guidelines for: (1) the sample sizes necessary to accurately estimate the percent variance to an identified QTL, (2) the conditions under which one is likely to replicate a mapped QTL in a second study using the same mapping population, and (3) the conditions under which a QTL mapped in one mapping panel is likely to replicate in the other (DGRP and DSPR)." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
    <title>The Beavis Effect in Next-Generation Mapping Panels in Drosophila melanogaster | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"9RwhM8ZHxCD6O48G8EI8yBOM2s15-LCFUFc9xbk0oKE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484760","apath":"\/ggg\/7\/6\/1643.atom","pisa":"ggg;7\/6\/1643","ac":{"\/ggg\/7\/6\/1643.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/6\/1643.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1643"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1643"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/6\/1643?ijkey=90eb2a5b82398a3db6dadbfec690b8901e6165ee\u0026keytype2=tf_ipsecsha":true,"\/content\/7\/6\/1643":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zdRAe50o64RWH0mHR1tbt1kRgzEiaChqlydlyi1AIfit8ovtpuRwp80oT7K4DEYfpAIsDv\\\\\\\/qum\\\\\\\/Z0pIf0QzXkluyahAnG90sHoO\\\\\\\/OA2lNQjW4XkG0P0TEdOLGBJLjIWrRzFZbyb7JQaMy+ZCbgC5G0JDHfXF1U+FraVt5+lsjA1WnUeWY\\\\\\\/J3Rb7GganB90i7x7VdFfIm9iAwSBliBdgzN32Qmjn+g0IGDcOcmO8vY4QUwuIVVKvn+QW0Q+x+dhK\\\\\\\/0SMkRhRykdoVD+Xu5jEqOOvbi3uv\\\\\\\/BadJP58iCNGSkLdXXUgxMVXKOb9WjjC1NznU419fA1TGsgUNKkX\\\\\\\/vadn73FAjklzCC3nh8jfTJfxrl6HorDo32yH8rEocg2Wngdu4rXN267qKNOfJcnzVeTU1SDybovy8wKr5bN9KzdF3UMtihRm0JPva6L7iAyOQA4MT+8E3+6aaxnT6O+euR9DL\\\\\\\/fgn1hm\\\\\\\/R47hsD4P+pyv7TbH\\\\\\\/tAnmYbZlUMXNmKbhh2EwLeEtyRIXlp\\\\\\\/1Kljvq9ql\\\\\\\/fxSl1scFKINHEcvlG\\\\\\\/7Qgw3eYMIwWkuW0bZFIVGv9KXmCHfNfYugClapUnpGvE\\\\\\\/4nWR\\\\\\\/4tY02YoRWQTqZaxtmCVgPli6suSoBblDwQP2ijYiPHssw08Nhzd7KL+comNunBezqyXNd8oxY8tzvylj+S1PVjHcaD99QbWiqZjG56dsbMskJyA9Hxm0S5VK2fMipTDYFBJHuvhc3xhuIvBdnzYswdLjBUv\\\\\\\/qEfSSdJEhhdxRKGGC0wfUSNLl5fG4c7OycSFpdOX\\\\\\\/wlr5FDMO1H880faMqLI0fKewJ1cxqNpzkhs+vY0upbR7lBaxZh0\\\\\\\/VMk5nzHF++jATioaKERgtRCC1+1qUPOzIIA0m\\\\\\\/6AbFbQykaGbkcjuUUQFOjjL6+3MRX\\\\\\\/70dppVrUEjd3xicP+cJgSLna8LxlVMrVy+zV50qij9NvVEtnSoSMj\\\\\\\/EvkLfMIjg7aNj8K8PJj4kDXPaDv8BEEbnhDDUIXZKAfP8hXDWzU6ZtfFrux+OeTgzKlE\\\\\\\/oMjGEfFUf25yd25GbiQgOG9eMYopXXNcVC\\\\\\\/s6NFrQTZJD\\\\\\\/uzcgGW6EFKZewxR5MqGRfp9yLRZ5qRzP5wjr0n74E47iIyP86lHDeBS8md7T5ZbR2ALocHhLPGE7YxCA5\\\\\\\/qwoYezq9kCOr616Tkl2jmgcUXZcGdcSEow71RBZCGjHsPi\\\\\\\/6hYH4UjWsWbgnOlJvRsTD9dxZM+i71qVBkhtLma+fbQwlZyEZw3dWeHRUjvmy\\\u0022,\\\u0022iv\\\u0022:\\\u0022WBLO38PUl6248\\\\\\\/I7NJI4dQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d24f012ff5e7ed5c3865170ac54209ec646c4d0cb6f702fdf9d8600fa2384318\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022INvvfu\\\\\\\/9niyaM8J4hqmp1BONUgDkzC84gxiJS\\\\\\\/sr4DceG2Z+6545wk2jD4kO4a0fnw+qDFc9PLiy4NFoccQb3Rty+daopxDbaDPywJtH1VW4c\\\\\\\/GDtPmqcNn3XcHsvgDS+wGZ\\\\\\\/NY8mw4Ps0\\\\\\\/3zpS9NN8nDWBtSFAEfRrJtNJAbLgb7tfdLxZTeIHnZLt82aDnOUNqG8bXCdojPTd7GtwORQuQKZrVvUL050yjrzCQY7UfPWxD5eX0e4h+N83D9U15uHdmalCp0GCnlpaROkCNwuBYxSiqo0RTDumsK8FQ53ZR4CacV2o4WkWhZUf+aXtHL8GP\\\\\\\/kZ+JywekRb2RHV5x\\\\\\\/UX1KmpjIm7NMMtaKr9kX\\\\\\\/lyrV4GX6XRv\\\\\\\/OiYPr+8cPPLaISJmzwVsC7jjlCSdeygxaH5tLBDMbBXFByQpKxnPzqkmiFmnVm9AeOSjTJB+O27sH26YEzslexsoGs7Obc0xsyGQ2R6rPeq4ZHHpDStCxnHqoZpG8KUntnGHDqRLAK3dLYmDFvLs1rgylr3jKteH155GGpzVJ\\\\\\\/Re45T3RxYR7PhUSty7dBn8AbieYsmAK6c+bLf+VaB++7U0A1WNFzWh8UW66OyRnFgSUXb\\\\\\\/WxyMdG6RlGBUyyGPudKg3\\\\\\\/0prLH664EpZD8lOoDomGPMvRm\\\\\\\/\\\\\\\/Vv88Hp6y6Fw2n91ltP+kiBWo+mW91gBMtZa1QX0XLe6FXfK+zKe8Bi3uBK3WypFr8EU041xhCIiwW8tSwBAUkE7P3IxuJbtupfDL1QwCENB0QBEyb\\\\\\\/Gk0y0cLZ9tWUp18LdwMLJCrpi6+DYes9uxSK2q1gXUqW7jywI8KfxP1LX9ZI0Puwh19iP73OWZBK1QPhfEx3w\\\\\\\/AC5ZB2ujEyE3yU6F1R45oIxm8ABxppk\\\\\\\/bK5uCFnbsGzKO86LMF\\\\\\\/VhEly9MqnYbt4zKfFRUaJfGv20Pu90L5s1VoGMo+\\\\\\\/UcJRq6Vc16VXII0PDa1ZD9AUFZ+FHlzS3rZywv24PIAoM5xJKXT4qADUaLOW8c5ZQ2+yGIKGqpLEaJX5nMUChCdkuC92IIaM6Ztlla5j3c6AuXi5UGAVIXXS2hOTNK7XG+v01pK4R61ltiyZES88bC1OMn\\\\\\\/elrk5VEoOb6SQOQ\\\\\\\/PhT6n5NwqCjyhLQ+DyZIENzv324IKQlunf4P+MhVfzAdldCXueocBGbuG7HLsHEHuuzTN2YfNCfiFv8X4LjnQ9ss0Vr281YUMedhIjUgpwGrwIKTtDrq07t8cQeIbp1cbl8wApIRShb5IBhsnP7Zym2MeDnRJPJufxzDXGLf7elrBLZqhHrzkRlFHgPWHhBFZF\\\\\\\/dntTwhuDN7RWWZCjlxFcCrGw7vmYKUOxGsS3HaQFPBiDIZ4d15EduhgjWfRxgV22KSd+5ztVbGa9SbVbGuBfVIXRQ400sYjBM6E3ieD3Asl4DnBFmwHe940HbDfMvfxu8UMvnLQJd\\\\\\\/Bw3Njzj+hmW8aUnj1TWMObTwVWXBkQhBThiZV6qrlyfvXjZ+sI+kYcCYincfi0bJchCLh9cD8dAqisqnGBEYqk8SL+5xgNiz4Y9t1APeUjL5l26JY8QcIAlILC2sDCqNz3pPWEWl5TCTliUmoV\\\\\\\/tmHbfL\\\\\\\/E8dMI44FF1dLz\\\\\\\/pqADr1f8mZCNHiHkXTdpvAvb345pY\\\\\\\/pyhmHcEi\\\\\\\/uN8vaZVsIexQmeXoajF2tr0TmvvykNIJu5wd\\\\\\\/god+161AnAnFGwXS6r4d3Vtx3Ts2tT19bvS\\\\\\\/wgDG5H4Mh7TeX23tYSN8gcNysPaBI1ZgtAtnZ2MgHYL3ad+pOesOb6wmVUkTIhta1qLJAUKhxs0Z4K8M0D1lsWGjRdaBwTB3Braq1g1jHVfF7DKnhyBqZ1\\\\\\\/4y\\\\\\\/+h0nZXNvaILDGDtfTjVTjVkpgjZ9ZKY2Sln9pS6ErovdT5CT3TpPGjGbv9nmuIZzVSLpWzQK3IVqH046\\\\\\\/zS5yivbtzK4S+ycFOPbNgXcEwoErw9VeTyKtJ04LrAEkCVID\\\\\\\/7xlFEg1G0Hdb4UYSzQLxeSJub6gmgSr+pWvbNPqWa+zeOWQoGNA\\\\\\\/ZPxcdTRiztae6T+8\\\\\\\/YfGSE1pyehxBNHypEZEXS\\\\\\\/T61AmqqrrpFKqkCYhAZgoj9cIKP\\\\\\\/4uBcTopURuViKwjRrDSd\\\\\\\/lAiPtNv5BKVuW2JjCJkpfP6s\\\\\\\/tmdvTUtedhW5zfDAj6g5qDZpN6dgnEmAJXUUxGTRocU7\\\\\\\/l\\\\\\\/tCqe7bGKNbJ6CdgKdcssA7xPehNt3k8PxHi7gXbnkmSyUwXQliBzCzMGr8UEydK79OG+mVS\\\\\\\/bLHLIw+6MgsaHvVjbrgCVnyE5eie\\\\\\\/vtzDE86f0KR7FFLJZU4Wd+C4\\\\\\\/B\\\\\\\/\\\\\\\/BmGfAGSMxvwvuouxTCc\\\\\\\/IpDnknYsk4gU5yQ6RYgBeQP5PdfYwITNS8lqaAFr0vl\\\\\\\/sTuCZWXhzE3qhyHubsqBJ\\\\\\\/H1pbWgqCuI60NBKSCgE4c\\\\\\\/7hx1b6PEKk6Hsz\\\\\\\/2uxV8vDs2JSfaS7RsfgnbSI7VvaQnQhcxr8tFWycNK1himsEp\\\\\\\/tHQbynY9l4TmQxO5Oi+fuMXo+r+530cB8\\\\\\\/1ZhJ9A5jTztt4fRlP587q5WHPiOdVD828HNUSRFfQwyXu4FB2aJLRHne9AS6TU\\\\\\\/GRPYDGcV0vZzoIXp\\\\\\\/kFjyHkBsNfpCjFEvMDWZoqGbcuhDIVu7BBypZYZSIaETZgHzjEpfNS229C5Bs546jcfTlGu2ZLob1vJUDqdnd92gOO9QG7rgbzMulJqu8Jk+7r2N0KNzVEEhQHqTbtv8+672GO4\\\\\\\/ASoFZH6ve5amCn79XWmzkvMV4eyJR42jz\\\\\\\/3TjqA2puDprAWbTrB5oZw2j\\\\\\\/CxL1MUm8SSdzARuJ2l+7WQ+9C9TwHbwctx49eZZH\\\\\\\/H8yVvgZrrClCmlZsHNur1TAIjorOqnSKGN11nztxKTus1NKZgmFB1C1\\\\\\\/MIlHB\\\\\\\/7huFU0uRpLMLS2vLpqximFrSgYmtbrWQuyYegpdC\\\\\\\/Tt477kW17AvBncZiun5HaXFJwnk02a9VDamLkwwQ1hRYwSN4TPP+SMdL6xC\\\\\\\/iVeNBb22YhkAoU+0YaSeGZ6J3Jcaj6lQPu\\\\\\\/ZaGBtbG9nSRcVOuyN77SQHP69+lDi\\\\\\\/VknZ7YbotavEE1\\\\\\\/ITxtzD5Wc\\\\\\\/qYyWfhTuAvHBTyu1J8T8LbGZBVifUivyWgU8JFMQNT14TCwH59GJKAzP27Qunj3W5yjzso7sq1hPVxWpuX34kui5AFuz5BMKvwJDDyyT5Vt0cORk9OZwSRQUiKkGbeMB\\\\\\\/dMWmSlRPUg86ovgcjPEWWJ09CSrG0otqybFZ3BWRabl1obS+kyah0lhLSTeuTeFrDDXY0bQucBQ\\\\\\\/WCKxehmH2FNOtSohxCj1Tv5y0hrsXVkQZZ+cKVYPriZTTOyEUCj+xjkDldaieLmfpZk3zzvvzSkPCe2YHIF0Ztvjthq8fXmoS9Wr0xMqgL7MxFrNC9NddUCeB0e7BqjX7Sx2+dCzIGsELc7YVzXvWCzNtECoBODmLWXsHZ02MtBuW22pRk895GtGvqgbMCZ8+IYHDzCCbsJe3CRyvkqvDJ5mqqD\\\\\\\/NQB1scWHOSAw2O3Sck2oyIAyjYiYZ93CiTXdnYpb3t6LU0c1b9U7gMnKSpn2BFu6jh4pazlfUkCPPr2HMwC74yIOFGLlOM4HTnnNBKhn8d2QHjcdgVCWrageIhtqGDc\\\\\\\/8h+HY4rkEyPx7mgCN+1YxoNEcO4KBJ1CwqWYq\\\\\\\/8HvAWcxi0SkxokQm6ElKmsKTPcbuwkcNq4fWPlcc6eTd6228OPJQqvm+VEH2NFTpfJfGZ1HtIfE503QMKALC1wvfgXnLF5b\\\\\\\/RM5l2sk6USrifedzEof4vJ1fiufQSfxjJe\\\\\\\/Fa29zrAp3zCYGjWy+YoSeerCySZBbTs+5\\\\\\\/rjXWGBlMygdI4fWjrFR8nMquoVIhTzf7ku5Ym3DXkHteWr+PFakGfguru3m7l7J5\\\\\\\/vshZcBXr4LWSYngzy4Et9yreXXKBYfA9DDOOb2TPfJH\\\\\\\/cqIP94yVhJOgr37qwtF\\\\\\\/2d0Cn\\\\\\\/lJBrRkM\\\\\\\/7xsxyV0xQ\\\\\\\/\\\\\\\/A37VP1pTEAc+48jZw+pc+Ct9CPiShCwY8TFGH4Z6q+JjHNg+kTqpm\\\\\\\/ADtztpw6d8LTn9+eHeax0crMnWlmRGco1ybyedOpH4pSdui\\\\\\\/m9pKAKAcpXsIono9s4lghsqhz3VeYAe0\\\\\\\/KN7rDm710UHv4EHSCzotkBBTZW8lt+WJNcxghsxuE8MI8ywwNw69h2zFE7rW6ibNkYcPmAxf9ypJMD2SsXnKMvNtapGTQv90aVqjGtRTvnVGkdD4qG2FzFX6swj20q9QhbBBpS6xqmRYWgOg6PqbfXSIORMEp2+uMqRzuZMgqQoBdT\\\\\\\/HyOHGg3tSYOldhFTU3ArOau2iJU+oNI3arStIK9I5I\\\\\\\/SoTibOauoE6yD7aPB7LlAEevMfSM3NZ+nuHQBa2ValSW5hqgVozHeqk9zkjxiXMPtvk6gE8yKOJF1xbr\\\\\\\/tPujnxN33n86nsaerWgpR9lYokvzO0mS1nlaLRxHfKyZ8yGo\\\\\\\/BMlyHrlaGSjXGp1+hAUOFAfH6myblRuY6K6+5ihetVIiOnpSYSnRpLlgv38rnxekBfIACwcwHVQ6XTh4t8o+4jat\\\\\\\/H2RXKgqoVMsfcM\\\\\\\/BtPIsXTRjoSnZg+8ETd7vCzgGh0AfSpwivava+us5BZUsHLPqed4k2XAEE99hj0Qo+MiTUdg9w0PAkqbnqXqBuhVsgEpZ89wlTPp2lZK\\\\\\\/YY5iytyOJymHdyfHdOkriIlgJ+x9mto5h95i6xSDfRlnf3FN3gcnEIWB23Iq2QyBZgkg+zz81\\\\\\\/shaF4OelUuZkuyvTpT5NnlG7Dl7BL11913XANwk1lsIZLcdBsliacHCWJOFI4kZV919ANT6cDnvZXUY94VolFP7AWPSQ3D1DnS2\\\\\\\/o3sPX39eqIFRqXYmqQQZz2bB\\\\\\\/yovdHhqitfyzXXy1wyhtHLUU7xOc51EQhSxcxcNF3YsioQHdUPo7T0Dqc7YGFKkGCG7++yxS\\\\\\\/M5EfeMREgVZm6eY3wFgQlRlWb1lMYuI+J7v9JNJCX6Dp5cQnzF40Li7bwpxuTIDuV6E9tqZ1rYc8A9WlOqVRcpPA9Kd93iimU0hVmh7ss7zMDgQuwOcaMmYLqyol\\\\\\\/c\\\\\\\/Lccc1h3AaHXgw7w0SOriVCXmqo15FYttNRfftRpVPqWjgnJPX6525byZwwcawozY3lO5g7lNrvXFnCTt2eMGAzUN7HSOgIsA7s8cBqqHZzahXgWUwy2YVvuU1Wv\\\\\\\/UehhBfkUtPYg2TuKLs30n5LE0eRGVSqKeGOWNpabii6ntXNNZ23s1zyMAhOtnbcCwp1DMcGhQWo40nDKALoDwSONoAN\\\\\\\/RCHZZdRYhkIj3+6l9PZxkX9nmLyFpVVzZder58PJi\\\\\\\/\\\\\\\/cqkbmRVi1VPocRx5RRGYPxG1C79lS7GhxmYpZrJ7bqAeTUHIbEXkXDOfHx7DltQ4wlo+Z8R9erXTQid2+tOBm\\\\\\\/JUg7LGOFq1fdj7kJ+bIBGZH7gPPRYzn9hEQa3K\\\\\\\/bH7UNDnZlRt2loGZUxpMtSrDlzhOB1TF2jtM6sLV74FUKxUo+N0dV8MwttKKm+bocxD673c0MRUph+4XZXpwZCsOuOmzYRQG\\\\\\\/t1DAzi47RdHWLMv3rvHIgbLprQ7datISRB31wH0P0YA8cE\\\\\\\/YxKlz+S8y3glYRS4L2Yn\\\\\\\/Djs7vzz4bv4cO8lKQDWA3shEmbxXS8vPzsmWyxpUCJggsLINKKir5Fedra8uC1z8f4aeP4Y8MMqQEF1aMB5CKQVvgAclnbmKTPcEMijHkvMmyCEvggr7oa3f2j6zpUQ358fJnBkCYEI1WzsoHQlzmQmY0F\\\\\\\/lTpMQ\\\\\\\/ixZL\\\\\\\/A53GkCaIszscE5NipFPOy8KN\\\\\\\/SN+T+URLFW74KZjy2+OdQlrbLjPBNA6QvMMAOBcz2yoarVDfJtJDboh508MahzKthyezanEor7JAjjO3gYB+FGJ3qJM0\\\\\\\/FHipeymDQe6M9K+tI+CRVtVRF4H1QnOkPeUzh0Pz6zlegGpkaG+1BnsTNKopRSj3VhO6HWzcKci1RJU8VvgJcByvMKXvPpcjR2Uq\\\\\\\/CXgd3knSlWIXHb8ovAxX8CsiHhb4nxdt4CmJnLpJ7zOpaAp9YMXw4TysdTeI0AaMAu\\\\\\\/8kCcLY4odTbIuzJDYr5UdGMi2ZV1MDVgI3ejqXaqNZsjxqjl4y7keVR9Z95LlyqTSpEN5n7LdX2qFVdU9faOPyDB5eyuU1ao04MH9RTa8Jv8YAd1maSyCdBEu8qhOHLg8bBHeoN0P69HTBFPOMHv7RZmYZCjCWw+SqWgcJOjYDqSab\\\\\\\/wO6tsBmcQWyvPQAYF2YfOeCkwYngTTbvGlDPCybQomuzFdumPA4YcjdYYBTAVyWu29jl66Y+P+2fb4OnYEI6DKiS6FEsPk77r\\\\\\\/oYA1atFVwLf+BCZPo4\\\\\\\/ApZRPyIE5q6YtfLiVDZpmgOgEkRqOODrijvvZbYB+7aoHGiPIugYg\\\\\\\/5YHEbE8kQw08nujBYiYcWD5QZwgF9rsqgG3iQ3IudbmWVeHvA+U7sH1ModrX6fKGGgz1JSt4MfBW5BIqhFUy3\\\\\\\/tqKin8E52a0e6XsMZY+JZT5ialVIPcFUjogpstrG5fQ3hEbZqNGvOgFFLWJV+fnw\\\\\\\/lObWvw\\\\\\\/UwHm+uehI34SFPQSUX88qlpt5tyioYv7BZK7nLRKtR6fLG+vVkYYNOyQ+STliQDAeKUNK6QoJKMKwUQSASVkoal15omqaVnmbGxMDbcyx1Za+t+HZnvHYK+mX01ZSem5TkI69hVSi+2M69xCjfCvg84ImOFUERqNpc87lQh4dXNS2F1IiBy+iGUpLHLT3r2TNgUGvni8imSGCdj6UkovMgHZT54jWTqWIEhzU7PHwKggFc6oUpNK5nB\\\\\\\/0JYvtcL4G+4sdu0eZgcJnGT0t3gav11QqCY9sI0fus08yzFUEfjPapqHIMpe+PByeYjfeLWZOeArbOPTt8IKOZrsWAGAsPblieBw81+7sfKn\\\\\\\/an2TrC1heFIatf6a3FTmGyZi4ow9lY\\\\\\\/2N4ViLox8saYPSRyzxESQ+CMnNE6rhsIZ6kt09\\\\\\\/goGauP7Fjo7LgXNHqpSZXTjII5GjzvjeEx\\\\\\\/4zLkt91gcylY193l3oIye9Dj5P34X3O8M5BYiO85Gcw6zyfpfn5gyUDmH26CppxvU6kGOXQ4t6JwMX7qk6kofypd\\\\\\\/i9AJlcWeWv5jLJdSsR4Jc3bvgAQEhEhDBRgHlAeeaZeLiG\\\\\\\/Ss2aqdj8DeCCmcX0Z\\\\\\\/MS5pfTeOnmZQkgptJudXfDWzO9x23nVWyH7xj1gnvjj6lrb\\\\\\\/q9fCvIfSO78PXl\\\\\\\/7VkjiTsfuMoi\\\\\\\/sgqMGFndhZu3E7BNmRmxr656LOyIAw9NwB1n5QTwMMnM3T+ZiTmhPZNtg0qQXyFVj7f\\\\\\\/zoB6S2m4B936vhyP3wz0RZMjQlBB215Axu\\\\\\\/EWkny8XhUawoR0D4vnkakapPQ4nkZGFLXIj56CWfk6KsoRv5suUNY46CHcZ0X14fbssfz3jJwKxdtJw\\\\\\\/I0Z2CDC9dd7eWeWz8o\\\\\\\/XyD70FSzQvp7o+WikBilE0s1oxKdeVZwO20uJA3\\\\\\\/4VSmzEM9wWl58O6oEnJjaUOcWlYATosS374rCZDlwdheH\\\\\\\/crHmbu2PdhvaIMBt+sZX2OPU2ubzY\\\\\\\/mPnipTj0KPar\\\\\\\/chD\\\\\\\/6sopPIPOEv3IVkvuxs7KDbb3SK9sby\\\\\\\/CYnV9yGZoX38QiQt4eMBcPJd4ThnZHh7ddXKHg9W\\\\\\\/ISBDwI87BcoFdTFBRWNRLoRCPO1QDGMU+A4HU4WtL7npl41QWWlaxcRuk9G02liWgi\\\\\\\/2KLLw11ChoLKOQSAs+lb5vhJ+QXnmQAiHIRnzlFrbKfyNuc4Xzuyi1WG27s6scwIabPZZrGIF+7SsxY\\\\\\\/Uh1osPDSJAQzCaiGsriC02uKExaNQ0dAJxy+J3qKI0COaPwXpBQwUqsQg4NxbwUu3qUYqnGSPGTxBbaYqQFoz11PoipCAguXYEGusatlrsxlKBLAKpwwGTd9BKeDUJdPzIoZmM4v4DJ9LGlDZ28SDZWu2lLEeEFGMwNzy+y0eNSfDT7x4GRn3Yb3LX2Kvti0SYqXj6Ec0AHn6H4tKsWzSweoBiMs7a0HNGRNF0TEQNHsKia3Soh64wEE\\\\\\\/wxFIUsGVvOBaU+3kKHiI4LogNVyuyqPT9wZgslWSwOU7O64OlI4bxZihEvEcE\\\\\\\/64M+DpMRAzikHKpi2bWZQidQBXoqFCGpI85oqU1qyIdEW1reOUiPd3qadLQ0aR44Zz8StVu9ZiTq7ALQQA5pFjoSR1Qk95usCuDPQ0nZWvrl7nPOKvpY43egIs2ZfJvZZtcwwnmjkZdg8JB0dRabiBaCBKAvrWgl3k5K5g6hvjNZJ5qob2i6wFYD8JO9plhDZOJT2ERVetZUCJSsnvhH\\\\\\\/nWkPA0PUmFzT97HQ25XHL+f3dgVl\\\\\\\/ktEcrZhEYsaL+jBR\\\\\\\/FadZJp62DD2GOrDQNx+kYy9FhsfjTlJQlBZt55UyrPBlu19RbYTgLh9TCke6v5r9Vea6QK\\\\\\\/ylH9lSPDX3\\\\\\\/MhjA17osleNiBOQrU22FGyZ8ufP86l+l28UEX0e9KvXpKazWcTV3lCY8CEdtiDJ8il1IV5Unh\\\\\\\/ZzesFcuDQJMby\\\\\\\/v1onuoRVO9H3n8KNUz3RiWMm1\\\\\\\/Pzj\\\\\\\/rSYi3Whd6vltXUJPDZxN3S7x4yiqyYsyhSP\\\\\\\/jlhHFm5NPvDX5DzcLWIl8xes2aROfnIdNXUse+YJlYmKdQ83dR9A\\\\\\\/POsEm78FmhGsFwKSbx4LxNYi+GH5\\\\\\\/PBuStZPFEgLE77Q4sVIq4c+cm64LjVgznfiFsSLTQrDJIwKuYQ6ogFL7F5sLzFKO7EqD8RZF8Js2QpOQSiOTr3l8kOAPCmrYNpOKmuaLR2RU2OkWdx6xQyGcmQJq8Wx3mUwQRGag1\\\\\\\/SlpgJl3Pv0pKe3iq\\\\\\\/4cRNbjptS5zqFz+9SaDyvdoNaJQDuMsyabpd5KhlJqzr0Ay0EXTKKrl8AvM4831YVYR6\\\\\\\/ljScVUFmCHVehn\\\\\\\/k7uX5kFFNQtuzQKp9+96wuo8j7d8aRAZdyU84Jz0XjPvk1L2i\\\\\\\/QH+VsYIw3CcAGn2M\\\\\\\/sSmu\\\\\\\/V+lGNskozb9fncuYYAHPzya6cI2FHWduC\\\\\\\/h+OHWc77ECcGFlKaSmeVM28AuuK3VN\\\\\\\/wlXupoqP4oGTpEj\\\\\\\/srnVpbYd3v6qlMMitoclH6apeMDJVu+uff3pl33gJ4KPZHxz+P58YvwvpHKdexJswimVyNrWmHZ+YZS4jJvD\\\\\\\/8tobNsaRFeN1Ybh6ozrIg9NXJtrxEZHM49ZrnhJNPTmbGJD9pPLa4nXNweuDRpcg6isjdKHXVVHTRS5UukMCx8eAgFYuvztx1+enYhiKhXGlVC3bHsQQ55SIhtXX2O4zKF\\\\\\\/Tnp3pxgmtruQvEFNOAM+DStTiAqrPwKDaY+rM5T0N5lh6d\\\\\\\/0mtCZmFask1s1HD65PI2UV31s9W0yltJ2wnFir\\\\\\\/HtVsKYOxIpiw1eoNpHqhEPz75nagwkbkuffcbDgfgVIbjLr8dkoFvXc0HRXa28MEh4unByLpl1MX3r5yOfHDUnguJPAw2GPiGKze35T6GqEITY9yLagTZSABSkWEoOOKdcT83Atr6nVxqlp9MKM0l+2f1dLkjdb8oTuiENKQBLMLXy10PdFRxVnBd0a\\\\\\\/aY9HvlTUfLO7LWRG5iVl3y+La5JG58VYH7JAHFWvTm8\\\\\\\/18+eiwP6k0ahIU0L6daZjy15953V6OYzGIlmgTxwgvBsymMcpjkdYBXSBJiM94g8s0fd6wg3niyf30nHslmjV7fEFqtYmpCP3WJn+uHFlCAdxglcffs88NR6pK3vuzZlckijTjrlgi1VU0pzTLY1kdZ5RwSG4VrYDNjmh1W5V3xek1kfeSA\\\\\\\/nl+ZPY5FNEkpjC\\\\\\\/NReMVU2ufnJ86UwJz9fRy7pmageiRT2kp6b\\\\\\\/VvP\\\\\\\/9b3+qeixCthygZooDaUZTHh\\\\\\\/hvNV5pqd7RzJEdprOPDmlpatHZXlN8ICw53vHUo0CQLhzChCfZr+FaSsNmQG1X3r0YukZ\\\\\\\/OY42BeFUtGp3OlC3JaV9spJZ\\\\\\\/ahMN9RB\\\\\\\/MuxLLoP7rKE9IWwlLWf1AsuPsgwT1L4Kz6Ly0p4X+AkjR\\\\\\\/lyersUjHGpt5lKdSxqol23j8P8mTU3IB7eM8Dkzc6X4C5YhFybyLGqqPikdTLkXmsIhVhXGdsjAZfq8mEczxSAw3WE7lCfXbErt2Uf9aFly8Fqi1LsHmTIkeaEnCp+uk6MpOujUHmhZFAl+xnAiLQp\\\\\\\/qjnBm6B3V8aAkqllxJ1Q3D5WBLMGwtHzmaT8rexWXyeFCNpbRnQXReEKegZxQylLzdCbXhSjw\\\\\\\/DYAIeEyLDP0+bq1V5+5E97AZoWWyIuj9fUa3n\\\\\\\/GFa8ADtuCqibkGzYse4jyHFS0iGIwYODYnuJoQRfp1Ls\\\\\\\/CwIz4ZmgDaDZnv6abYdfZNxSVlpvJ9du5uOx+8s7SNjd14sTDeo7hXtP14TOvnY6iv2O71y1JkrDSBDMWU5evqFj442bRcYp9G4pWwgvRo1uUEIshG\\\\\\\/rcRquWIdEMZQU8a91GqswuCWs5aoUZfKom+gMtF7+XmixmN7xkT6sNkb77rE2AwQyS9q9PSqle8eJfLZ3+IGl7J41FNdo4ZKTrZS5UzJanjcwM6ViDbjdFPQpcp211osoODhMl8ixrj2WuQLzUaNmrTZMpRDk8vsJRtpM2ay587dAdcuPlTC27WcU+hj6idcuKQEk7ehAR+gi70vYJEWa8\\\\\\\/zjEfrHR3xc4J6xnMsiSXFf11xusB9fXm1PDaxEWsJfG\\\\\\\/CyMfBpK1XtSouAFofenZjuEWb43p0rEwGJefLnjoZjArgyCm8TYa79dcgvxjJ8QYwwrKrvdAEiHVz0OfXa621jRUdL6uWyt48bQ\\\\\\\/EoMmtuqTHQWd5av7LsID0ecFv52388bQiy5No1ajOT7FHe9PzmFCmarWwOsn6y2ignxCu0iYq3dDVT4fWxt7SC7jPjCD7t8EIG7D5sVcgl9T0tXz2HJQXY4+AdXUoJ+apEQOxTl\\\\\\\/C2n7iW0kq5hDvLThPfM6GjRCBsV1auNkWwCXymWBEsX97WTab\\\\\\\/aG2clow8yrWbqOPKZ0dkCJTDG96uy7l6nflECab06MVL3UXt1UOpC+45Y7\\\\\\\/dysY351lUNtcqdBppkxHAOgpcv5RpCDCIediDxGchlotX\\\\\\\/Sn9teg3ftho0gzEutlbIdT6Jxelc00v8GoBKqMh1c+2EKkwhPYsDOVRIxvkhdCwVSzEJEnmK6GhfkG2IgfTAZcUClDBB50iwSajhG\\\\\\\/U+qNyWgHDSa\\\\\\\/3xo8ovcDtI\\\\\\\/Qn6vmPT+Oe3NJgD7gk44HUVr\\\\\\\/Rx73RJthJa2Sg4Uzs4VgbyNJk5\\\\\\\/oA1keJ\\\\\\\/QI8KMbStzK3cKHjR6IEmH5E9p1qjYWW2v\\\\\\\/HVHmCjeElnsHIkIr+tadVo2qrWak09zOnGzLRBCCMxcA98ScSOmXxYEGA5Tk3UxrP49umULd4NHrGOhMCpKNdRl7oZE5Wk1gsITIOlf1vc3MNPfIrg\\\\\\\/dOYpsCE3zH7SFGfwsQ09xaQeswQNCCvJ2i0\\\\\\\/\\\\\\\/y1ZfYwroA4IkwbWqQBpYTVGW4YCGGqchujbdORhnSJmiyBCiDGDNKtYH7Zk6jGskdkProeUP7R3SiBZ3yBTI8PamXjyJHa\\\\\\\/HHhFAmE\\\\\\\/InG\\\\\\\/qxbYg65Q7964r0LEyRNEutOAXPhpaToSXPvMB\\\\\\\/HBhRkBL5F+nXq9njH6Hqz\\\\\\\/xxFxsfxcP1Eo6TUZgD6P766MdBfvugr5dt8z\\\\\\\/KIwdiRvBOqvbmU3+MeVxXjuTXKd5ayEsbnmOhRyto62nNyng9ORGXlT5N\\\\\\\/BRU48BGpP\\\\\\\/xh+VwG25wBysS\\\\\\\/KOSK6\\\\\\\/KN\\\\\\\/IjpWzXzFiKXol4AQpapv23X3jsQcD2fN4tBOJx29o1gmQ6w+An86r93qwrJ9GBTcA2L6piCK5Z4iAfSnP0liRbAdFZiohptbT1OP6S3kxBIWWTZQXemTyA6Z\\\\\\\/NBU39UXfKBXi9wqEQOHGLAHAoY2v8QI18dqiKbsyuodGXVpaT+s7bfa+iPxbyCZXqPag0TDYWxkrw13B3dkYozWndS1X2ZYAZXYbsgSxM9CVnEVEUTNAGeuTar94QIgu5feGt6Wjt9BShISc\\\\\\\/MXHnuk+ugWeDlD9Fa3Qwvqn0kz+HZdHcwlAWKIcPKI\\\\\\\/\\\\\\\/vlxxCF9VDae8wPVGy6hCfNTM55LAYCQgi92hxymcIsKhbLiKi52P\\\\\\\/GMJEo3gnp9WHBoARZB\\\\\\\/xnD1Hi8JPHFazSc9Zq\\\\\\\/4Oayqa2FIa8bpAcFeMzXQYRTVxzZAyL5OMal8Xh8G7fQTEbweeKWmIzkQrre6pb0p88A4NJfg+3Ut1B0tjXn476mvhDrTZkjBALpNi6fogCaKpVO2DvtqNhjsPU6jEgTfh+a4Hn9L0QyVezZp82qNyh2R0DCVuIs8TJrqfJbFWcmGhW\\\\\\\/BofQRXXYr5GbpYrjZMYpX55E7XQPIX\\\\\\\/29WEUFSxzEx39Q3v7opyoXkOMDm6ElileW+v5pEav6ds+S0hwyp\\\\\\\/kwunqGrJnIltbpNWKfzQ5K5CXGMof51Srdrv2Fgunx5BLQmGV+RiCqzrZ2zCucSBJqV\\\\\\\/S4+Ft+srugHQNtBgqt9e6aAMp\\\\\\\/PyztFwlzEpK+wOOck0ZbSxfo\\\\\\\/Refu7o5ptruRNAVdVnw4LZ2FHk02ow5bKxeNeAZJduytskQstErw1R6Vh\\\\\\\/pn8gGZ54ieV\\\\\\\/g+pr44kay3hBJFSNQdSWl1pfLOuq6e19h0W3t+2+UAAJMZOkcE68cW6upm0Vm40h0uxlxtJNHVBiGB\\\\\\\/R1MgrepbbORYwQIF9mnBL5mOzjfAhbpahc\\\\\\\/bNMHPd3U\\\\\\\/dk49o5Xf18aaToZIc90rfn0ZfhsmZ87rdLeu4nd7SCnsWGnY2uuVwq+mjz3zS2QSVCBQ9CT7gnJrrncLK9sRFZjxDkDAWyKhj1JoIFH9P39ubCeDPDe+AiQHQRSVMRB4IceUl6pUUo8RebvoUSAu9O\\\\\\\/XH3KyRu7o2M+SIU8aTYJgI884\\\\\\\/zugry84KHH5+DiXctu7zMlBc4g7bgN2WDyVBkSpty4R549DaseoUWEkBEkB5pM0dkrmgUvm07qD3vgAhY79HRFn5gDtcXzEiJQ17og5OaXEt6sTbh8jZtWDM3sak4xRCY8ZOm6SvAbfGhi0x+hZos35q43+wlXBrQXZdtkipmGPNLDHbi8QUuUevdEDGmlZ4EWBgudxWhbZvvzAxNMQk56vHliirKzWhBU1b6G3Mj8o696mKo+uakycMOzMMiv4v8GxlENEKTVT15wubP+t8Q8mnVk5Ao0eD48xAWY8wpFA0SviT+Hk6MqM5\\\\\\\/WFLMIPcpVAeSfIMohWDiLuD+CZFJBNUUPoQPa+LnjhUL0J+hgQ63YCSzje+0phQUKRUmLQVDp3eHfmGu05tm64\\\\\\\/ekeBmtj5jqHF5KQvQoIO\\\\\\\/lhosbQdpsngeACxY7MtZIkTePfutCztwv3wt71JNFJ++ripE5XfQQbreK0Y3li6CncsGWwdnbd2LkIEEPxkBTIb0i3G7Od0RwNw6FK9jPma+bp\\\\\\\/OI\\\\\\\/kjRfNLKlNUW0ZiKfh5m6+ZCVJlAVjMFI1DgsPGSu83vYdMmnKcLZ45Nv8BuhrfU8g5uDuaRncsQO8Ch5HDQ0u2itbLqhm5jpxG83ZhnY9hu6i2A47wfBhcY3LEh\\\\\\\/hZidPHBz3+asc6ZzdhxoDsIJICVLu50GPjoDyXgo9J5j5xxqhx7OxlgDe76K5dN3uJ2LKaZg9qXSMdPXaQuQ1qfgHPCmSuONe4lc8JEk3RAnPoY8vQlaJ\\\\\\\/2z3ZbyeTuHr\\\\\\\/BuqHbGa6YZW2KHEtxHMdxp3w4iFVA\\\\\\\/TroHfjEboSmzJoe2yyimovIrH+vPeZ8+TEASkDmG+uA12UqDyn6KQNlTCZPlK2yc1qGrVOuHpIkTwpvunPeYAcXMF9QxmR8ddWj026+0zLvvBkRiS3cMvOemfsroDiOayNTWU1ps0AozZ8mN6UKSN3bZ91TjT+Uepb0Z8N0dAnEHB3MrficoBc5\\\\\\\/Ie+4aVf+JlZvf33v+hKdBNLaUhu7hx6uCew0Z7xTB\\\\\\\/81eOhaMnPdrh\\\\\\\/12HHz3+01BpmYCrPr6VJB0s8ONhcm7YjIJjk2sEHaBhZTfQ9urAJzIsbv7Po+3FFs3lHurwEvoMr0gdSijnw2zkxbrWIBNxhLKvDEBwKyALUnYC1r+cmBJO2FAg3Cwjf9c1HGiRRChx2H5jUBoJ0TGN33TWo72Q+Ep5ZndCwC6yoxZANR\\\\\\\/\\\\\\\/u0B01bCvHXZDuM77ASFRHLIwB44xZxnvKv6u1EQR7z79sk4suLM9vMbKnRmtNgIhVuxGQDgCVJaYAdPTczc1rnRlx12Miz6kSGyNAydIh9kmF0cr3lV0xhh2wqCMReW5DDp0Tk0Xolgxasz2AfqnQ57Kaf+vo4bd4XqE8upkxtg0UdFTUBul5kcUJW993yy11gz9wSDYAhhugNN\\\\\\\/Fw8OdW1U257aC2GiDF8ebMKtRgcGLEmljXAxdlzqErjHxH0gHxWN8TCVbQOY0ICAi0cU1r2UYy\\\\\\\/r8ReoH+mbjYNSnd4DxqqEfGz7n9uQH+A+NKs6xZOLHX21rurR4qdy6wjbPfvu0nFBRZshmChsi5gvoHrunA2GvZxlygeeO0+YaDFkJNDbl7u6KY1RCD4g6uO+iDercgjrhZKQu\\\\\\\/Nmcs++TLYLih+5lL5eGI9x\\\\\\\/i8DkbdImflQBwxExivQZe0TE+yL4eL+YQwC01HZnR6AxzqlVITtP1\\\\\\\/JpSnaTtqk+xa2eOV\\\\\\\/k4JFVAlp6iTDEKLualWk1CzcgKpB8WaE4sW5GQBFcq24HQKlrW\\\\\\\/f8cXU3R8pzSHX5+1nBbEBXP5Vm1t4jCfRqHrG9W3AKhMnYuqUvTrAx0d3f\\\\\\\/XXwVuLGVOymvK+GdhATm4bwOicGksBoDdyvMqfRAqHetbK4Rve8gzEDH9b14rseHLvZJtBAksoKVFaR+6c8ee2c12zPBGP8ns8ytPeVvSEdfGAWr9d2n29nyNpKvxK3m+e8GW4nmlp0KDgJn1rs5dmJWZ3g+LasC3yg2tdt9Rr0mZXuRbs0DTrxADnr0viTRtnUzulW\\\\\\\/pntF125kIcLvIHG0Wg692apnhyMjn0nS3hJRrn3+oRegvn6DQlu5rzIIGKqvnJHrZuprmmAYYt4OepiU3sTLCIX59chqwRrqZNSFTz2v0AqkbmHRwb8xr9nO8waIUKxsMdxn0BiKr3U2rv4U9pUqvOgN8UKoFw7MWnCbEvEJME23iuywju88QLTM4lDXJS0euJQy3vKT8cQed++RIuDjMIyYxesRXqZwhC\\\\\\\/w1kMZ0KaqyGNNb633XQ4oMKzTKJpio2tUAf0vqEI4dFWcDe09ceDqZXPyrHvnhGqVhGH5Ew2PSs\\\\\\\/As1sa7wJHGjGX1igTeUffVOktR\\\\\\\/IgK1SZpH2DWFmgDLGfGyZpb7JPsXRQ5DNtyhf\\\\\\\/aU7Z0bpajjypsX5jFpek7IHOm8eiM6aQe56hRuNPUWyJcTXz6uU3l4Ly0Q7t681EzkmrwhgiiFUp9VIhRrpXQKefZIaoQXfEsRu5DcoH+fYBzHX0h28qPu8wC8xE8ZDt81s8uSqbHrJOmS8wsSlQMiGV52Jc0S5sEpbAl8wafXXByrohJKqvuMdPIYoyJRAZkRO6FzLlkmAIFbklT9Vb32to\\\\\\\/WyxJPjjOB0aMVptaJAo9wnRWLC2Ov0MCDlbX+cqYGHRqjC8r3Tm1UzpflOH4A3R2+Xi+qOmcW\\\\\\\/8jy8Ln30fFHfDAK9919NkSJezY0is0rMb9YagRXGfja9aDbv0GFgFZQuNyFm\\\\\\\/56veB2+ZBGALJDXUTMOW41zciL+VmMMjt9UwYWOsVxBdQrtVllGdqr\\\\\\\/oOiAVM1K+J\\\\\\\/zEEkcz6B3ZDQa+o\\\\\\\/j8fgaVUATKQkO3ZmoE\\\\\\\/30dyvtJOhU768lzZQOSrptywlkEs3LFMsAI9FwewaVdft4Oo5LNmocXDgCKWgjpR0egH75by9aR6WBL+OT78EDpzrSrDaY5yMlXJosLyuPNMbv6nNeRZyiEwh7Tw9PwqGRwI4L9FAhZ2lJbDp810qJ\\\\\\\/hldwIk7Vimj5i40RFk7VfSdOtEdhQXpOA8Ea2UitKDK\\\\\\\/pWFNh\\\\\\\/IW6oiLiJ47cbFaV0vxpz4XlIF20D0PeCIWKHxeEnmnVHA8IHMFO2ZFUsilKjUGz4yBp6N+RFYIM2X0hOuHwvUhI34vcQvQRisZHDnW2uXMF3E9S3yD61lQV1hZ4zVJzsGAGi63p2lJVQWG7gtdbAgH\\\\\\\/dbi72NcdD2Iif5mfIMhT8b56uWm+2CLJDzqe6sWvkBvgJG8GkN3sQlBR8vbpzB0x7qo3qVBP5GBKcs43FIU9Hast7Y4yelnOVj9ARAGLFornAO7PzBxRqBoXDq1tQT\\\\\\\/jTtpE7fjcVQ6\\\\\\\/oANs4bf78KFTzjDLynOjJWGhrUv3nvoGi1isHm+wOuDJLy2uhM3StR5QbUjXpgdH+Y9k9jKHlKO1UKdzADbrZt+p4oMxAhjddtMfr9HUn+6WuHjimy03R2a8bTKXLdaBiPkz42xq+orn18C5OR7s0mqu6A+Wbe8T9D+x7ST1OADdtAgWPliMVD4+umiGikkyk+qvYhi2Y\\\\\\\/WkU2akYx1IhpPnXx7t0kkDxjOQEZJG0Mq+fkA4MhIfoI2F4dfrSC2IP5Hyz7eLCY\\\\\\\/KuXuIPW5ggLD2KTk\\\\\\\/SChUehsBugQ0V4vgEEENoy8XxZweLUpo9GaBoOwnsuAAXMRO9rb9l3o91O3KwB6tzIDLKFpJovieSqSQkHggAZX5D6BdhSIuk4wvTtx29Fo8sK0v8gK\\\\\\\/yI1lOiTHKw6qCcuOPDBveYAzoxDOcmMs54ZsZzH0EvS+xTHOyzj01xuYRMFYG3iLqHAAzq6gdo50Tif3FYW\\\\\\\/+YgU6YyAb6Jtd8Le+J\\\\\\\/5UGp7lP8Caz4FVzw8RQweUCsqfohmz+\\\\\\\/qU6sK3lGujIuYvxxf8oRLCQ\\\\\\\/jw9Z\\\\\\\/MRUUJRkrrQDQkiFzyYQpjnOPFPkFZcHGV88fw6j+dpW5hsqzcTpsNWgmX4EuDNssK1HnsPr9siJRh+WPGQbJGe0DpYMPFYpZ2QC9ipQlh0kNYxK5O3t30PJxbigr6qKR1lcm2VU5N\\\\\\\/8c7bwNvCWeexgAtzeVn88t3mg9Je8Rct5mtOUUkiIx6U1DD+fa2qclsqyKG5P76YgxXYjkawpHQxlsNigSN6wcEIApw7uwPWXBnkItWt8i66RlNInoYl5HAbdiDcQkrTPyaAu5rZVDYF4Dfijaa9Sj0pOJcAoQvgkXqYlJzADRyk77tsmXbOXozEPTnuZeQbZ0ndZSApOtl8Du1HP7ll499dJVFCb8FjhfcZupPf+Oqpeaqa77tB6dp8sk6kdgvLHOFvgQwKZiIsas9GABhzJQK+qS+6yDgryOvVKSssXEYCkO4kR4xWLsS+\\\\\\\/5rms2aRyDBrY+KB8UkUStXiEd6d+FdH32xhj4LTZwi\\\\\\\/LyXIafdFEGpdE61X09A2PgK7DBQqjiYwkIhkVstTM0eJTsDFLCpfuKuESHbzn9RXyOJx\\\\\\\/v8LymscK+VVDtSnVpZB2nZBlhsN48xDTSg9h4fwRLlG9UF0FC1pAhic1ba\\\\\\\/vsEhv3RdC\\\\\\\/v\\\\\\\/rf72Mj0iG50sUygp3GSO1VHQeVnZr3ZnUgh6bAvrMj\\\\\\\/hOHurpudpj0iI+t8RmEJXWzBr86ONKpZzrdxs0f0kqEESoC2x7QeT0v0qtWhiOYQ2JflMKpWcUTMUB1U7xmcKOa8I1eqpbLjb09KRCHnrlBaJZvWup5\\\\\\\/Xl+Bv000Q4GUvr6\\\\\\\/MBP20vbvWUGsbC4DOkFeoznLVqElnM\\\\\\\/HoDBtZg6FFWWBTnI1+RcrymwYFRb0wOHp62TmzPI5VIiU4HnZ1BQVFuGzXOrvUy4hXS\\\\\\\/SCS7iIGg9bBRAbJCOA6bWOHgi6Icmj6jgN+ZIiUsN5FmKDxjGdggHUghKHmM0yYaBK8azDPErdvKC8J0Q1\\\\\\\/KH0wCQo9hnPvyEq1mjsozh5XmAm87P50hAgsMrhIPIC5Z43hMpot6cFDnIhe6iMWnMsjkhqPau5PJQjhZAn\\\\\\\/BylwZK2Z2l9TT3NCx3MHbKoGxzuIrrUOxXw0nbfDZF7\\\\\\\/dKec6bjJvJApWMuKIkMh\\\\\\\/HgycMDZgeu7CGSerXsrW5vmtQltSNU1JWJ2SBw1PrrUMuETqND6ubxTYSURYhe4ntl1Lrwq3CRDiwR2tB9l25F28p7mqonbcCHVDY0d9wBHdV68\\\\\\\/6cX8A5CnVzA6eBgtobVFqps45ft\\\\\\\/CDp5T5z\\\\\\\/71ETeqN\\\\\\\/F+VEqmsKNxN0x89kuMHpMRatVdmbueWRHs8gEjMWxJ6tvyBOmbXvJbdutlLp3Jv4gLu53pRR+7mLVmwH8t1GCqq6yGMm04bQQCHmZKO1lj2kF9R0Vil++ZqmIjt4NZaGaVGqZ+jGel8693syBB5wymAdvbZY8qPBGU7cQitJz\\\\\\\/RMYd7bi0uLZWjYW47uWGDVWZbZW64o83sMn620jzR7iAHOj+\\\\\\\/eEcQqVCcXfC1dmDvDi3lJvCDoC+ZXceprkluuV48uZ+1TOJrSxznUxMl10maFARlqAhICHn3ecZvwJgwJ1Yjy+1SW0W4nOG1uuIh2P0m3mBiO4VnHeFL0y9IU7S0zV8d1C3QFw87yZs31upy5H3r0XPD2OIMbs3RVVAu0XoRCiECr9yDdfGxyHiCYDqAZidur5\\\\\\\/Lbi3Ra\\\\\\\/RFH6y4wPBzd3e1u9KyWMEkuym9ynoVF6LDIbhUAXJr83ewE0HrPMhpqzRKYo\\\\\\\/S0ZOoBhBYkyGnSmQRfEM5\\\\\\\/RPJ1oG4SUISmAYkxVzx5jRn\\\\\\\/3+rfA6HH\\\\\\\/AFxUjO2UTYW\\\\\\\/p9y0TxFw0Zo063xMycJxC+rFOKfwsv1JgI0pMB7Zhw4ToOz3uG\\\\\\\/ImFro16oAXZ6YN3uzOrQUtOI1jXX1Qffi1zIH7ATEm3vVEOgkfUmhur5WxY6Gk0WIZqVP5O+4yfk8onmw8GTRKbpoVkMD3gjfRvtn8A5myi6slp9otbLCGz76EDtCrm9yxxN40N+MMd7z3QKaVa66HYFnA+JPLyX1X6PEpT5AL2juU6YS\\\\\\\/UfaY9eQF0r4t+04KSYMWxWWjXz18k1+6Gk3Qr91LUGipQ+JnyGDkYA7Sme1oS2v4RtaEsmloNE9aKCbGmbTCUd7lK7tRPHr9oYojDkZDeNGK26g4wQAKRwqPyxKj6NcZ+N4USTfJqk6\\\\\\\/W5nNPhHSCC3RosEjreOCHCHGjqNplyEjEwfL\\\\\\\/+vVNagijMmVIxF21yVIuxL2JZkk01\\\\\\\/AuB0KzNPzGc8IMIWt9ES1Wcqtt1D2J\\\\\\\/l\\\\\\\/QoaOKhOwLOc8NVZ96wu3Aj+PcM4+vPCZOKzHCDv9JWhps4x7gtmrNEAAhz1NhS8Tu7BJqEWgnKcIX4MpioZwm5LysW+fYnst6iB6NK0gNdX895nlloJqqWQnBLePcy5Vts4Dpz6CXg8xraydKlhgddCPcGUF75ltLk+7tOnVObDCCKC3GWrf984vuzJEIX2R9Vj69AJCkf\\\\\\\/YoYznRgb6xeGwU6P21rxnPwVRHKrn7xIv8TTZbQ7Tf0cmDwCsSJa\\\\\\\/16hNp96VyRvO5rcYm0CdQ5o2ges9u6Iim8OeBm9WmgDPkGPu1lcrx0IiyqkBLuaRH3pSri3OP3LVb9+UD\\\\\\\/jfWiKdVGzovQpMafnU5KFKXRjf0gs5\\\\\\\/U7oY6W0GkDTwWmX2oBcfrpd0n7ZD0OeBQnYf5b9wW4vaC6xWrSqIAIWH4n0kko3svhAwC7T0ZrDu6mjQlZ9J62YCoSCNLCVzmlgwE+9Y9B54M3sRPaDn8MG942WUp\\\\\\\/vFQp1x6C7GddeXeu3skf5q4LKhWlwO+eLgi8jQuw7gaVGvmuv39EyyANxS8djqlRBStOZRIy9R0ZFGo0oka1VDyGzDEzLQ\\\\\\\/nfi8x4zMII8o72+9zO8kC3tKq9oxpyEL\\\\\\\/YQLIpyZ9GiVTLfyJ+QmpoV+ebJvZ1tyEzscdOk+lyiwyEcFB1J5\\\\\\\/r6pMv0iSjNhmjAiXd+ruIabakeyv9kwJinYWos1xnd3Z6p+mggWmS2qWBEkG3iqeA9WNeRQvmAOcNy5LQA7qVCRSRTEpVisCd0QD9GKaJerJKaZ1gucOMp3BUqKQ+7wRib4ynBZuAD9jPSr9\\\\\\\/BnU7vRSTQ2aU2mfvifhPFPS+6x+Ji6JqaT06cap2Od9NuVfJmXjAc18tcgUeIDT2GT4lGvNgIby\\\\\\\/mzsUtPVH8tUdWadSfScsfkGQqNn5t20ON9xOZQfaoSiwcsmudPb7VQf+EGfnW7jA+YjRu52tm+pWwrepI7aYhzP00Xd5\\\\\\\/B4MLFAPL9OwkUaqrI\\\\\\\/VmSHmhB4eOSr0GIR+3I0CFnO3XihIgvnao3msM3tkwJs0NX9NznB8dy7kDcBPAwmoNkl7YupgrzC1LuoxQSNR78yS8vwGr+QLtrKWSgWAlS3P4X72JBj9Ij6+PBmzzJy+eH2+uwZqX\\\\\\\/qMSu4o05mSbDYPKh0ySVP63SuPWbJAqtSn3LBzimBxt2Hiuirfo7d3U0XGlF88N4ttpwuZeHzT1XyZmWnWjOoMHRxQNayAf7ZyYMuRfvHssPKve4gG9pIoo78ZhxxYi4TbqvbKPhhWVNQPy9K5ljUoTcvmOt4PNA\\\\\\\/2vmltwgNZu6Cj92zuKy43kCESWwvDT8GAuwS\\\\\\\/WV3nW3SPd748wzM05Dtqct7u6lvzb26gxd8eyVfC34TS9Iy49K90TuJveannZuDMc74VnGhHSbskQR89fI4+ZQIuXlatTk6Fnl\\\\\\\/nTRBORX0HeQ9xiToTzjCoYxvqYntgl3FDSGPAywykI9vvEsqq+5wMHUPaq5+oedQmhXDb9uFea\\\\\\\/rLUCTlu3qZat8bgdbu8ez06lo316AWOYlRgZrrp1agHl1jSQJotrJ92BG5S1Ozc\\\\\\\/W4PK2RI0+IkMsGcprosGeEpWZGHrOep+4IJAYNrvwH83ASSa1OaHfMsYVdhtTJgYx0FFSIBdSvzEH5L4S+dE2dVhQPl8Qg\\\\\\\/v\\\\\\\/k7MlaDk9BP26kIK3jV19djhJymEVSqf683\\\\\\\/WBCMuikS\\\\\\\/U7XGMem19nWyjvpinkEvPIDG5Ke21Az3XaBsmKl155Otc8HDOZa09tF+pRh83Jvmnljv7WBYXdk9wW32CDkEwUnDzlXOT6Q5yaRf3LDvoSJNadIcHx9oEygylZP1+HqbEgExaZAJnaZta0eV0MFJP9XM50yGq8ZCTs3dnyqLAACwf6gZE+WyoHHDd43AYFp1Q8kQ8JpxInyouCeZg9id+5XfK7\\\\\\\/sAAAS9wLs8X9L\\\\\\\/CCAEpp2j0P28Qojba8MhCE3MrUWcYxP\\\\\\\/vWg\\\\\\\/4iF0s\\\\\\\/f8WOmQE0+FM1141IMwKOC5xPNN80cgk1C9cbdotF354oypwydr4iNg8LeW2NCNUFTYOxBsARZq97qaiZTU7fzFR2oEtm\\\\\\\/RydoFkLHR6jDEXyt\\\\\\\/chwU6cYLiV\\\\\\\/h0A2L+pYA6RnckeGK9cEFdHfowudcMM+kRjb3mSw4os3xtEGLeQ5byUtWdAneFM+UoN9ChLdDMlEhBlswanuCdRMxH5wSCh0cXleyonlSRyQdMhogdjz3+Cw2IU6MVb0avYrHhWeowoBsBAKC3\\\\\\\/lwaw7Nnl+EKuRzfS3o3irpVng4WP3tpauAzUsyQimBqwYNsUfsyNTJQm3oDBt4AWBrY8lyg+d2d\\\\\\\/27jmOdrz\\\\\\\/nnltpjvJDkB7E\\\\\\\/G0UiT\\\\\\\/SFQb8H7DUqFryMusiTgnHJlkxJls2sz7pFevQzmnCGDHSHU+EONcS\\\\\\\/dLl\\\\\\\/12mcuMtRWNl12Fpy1QuCFQd4aXkH1inFCsqQg\\\\\\\/8iQfqEPMqrZ2W\\\\\\\/uURY9DgaycnvKxGXBCv\\\\\\\/s\\\\\\\/bX72UdVVUD1W6Q6EbAITqXTEINneKZw9vyqsOW2kutRpdGeyzCkh0hkv3szacutWL1AWfu9i+vs6pQfCR0gtbok6JnOqBS+PzVuq4LH\\\\\\\/crFNvkNG7ciDfwnXd7t1vdwyTSwPM\\\\\\\/UjnXfAaLO\\\\\\\/uRRizdmMR1+p+KZuc4nGqbL+hRECD1s+IcGhcu\\\\\\\/yQnPXVNfb7qFbROP05C3AO8rPKCEk8tMkzK5ua8u\\\\\\\/hgzc37zUe5tk6BpYd0clXxzZWOlJpDHZ+4lik7gHParHPx9ok5sGJQ41oK0P+ZqSPNbHCo\\\\\\\/mDeJWD8nTs3E5TjEZ6l1RlQNnAPEJtkGPW+5DZtcD0QbDHGgOasK5Ugqpw4CMelPHtuHls\\\\\\\/VKNSlj3CchIpZ1V3VVeAXyHUoZFcsesHwpz27gNw6KgC0s0269SIRLCy8EdSr62wgvkuw2robdpDOupRKyu2sjTp+yjHHPME7uQCPzJTeIxOlk\\\\\\\/HMVb43A7NFJAU137H4qk4Ij0b7u\\\\\\\/VE0jqvNt8tI5PGhsEB+V88H24VtYIWuTkU1tXGzwq7lhnncaKl3lH6uTuCp3Vz7\\\\\\\/p2R3NAwaWxhX515B3eVgSZwME5JsK8g18oGZvxkUC8z4OwYi72mnUVyks6k84Dhe5sVTwq0g6cB3d9s42jGG\\\\\\\/eswl9Ss5bF7wxnOpHjGnk0VpJ16bAXNfAVi1Ao1bnOt9thGSYiUQMV3WNV8sMA+ASRjCrmJUSvfuye9bjp1iyfMrqLNAbhvdxJhzZ+ZdNcRPWFwPSODRJJnGsvSrf3Es8kbDEiF0opICD2mJHyinJQmfKcv+fgRp34tD6Fy05ufm75xGwcQECsghurmRLY+yO3F1sKaVoqZTh3QolZyj3ZKeF3W9mQqJCzThzfUxt0KgYBr\\\\\\\/StO+e6RilE7s7Js+u6pLv4ooGME7wKO6sbk6dCmqYBWyJ5Vk9RVsvC7rBxK04xZKY79KR3FNDJN45DIJXoS2+v4FMpnPjdfj6UV5TIQNEiLf48IbgzIDWNERJFLq4UBxRDke\\\\\\\/HuuitQJI7vOk9Po0ImkpZsjEFZJhnghahfrIGdbyacUzkuARU9HP8aZ04LCMPXAvpPgVfPdr\\\\\\\/MOsvTl14dGH7GpzN064hAHkz3Ap6Jp9\\\\\\\/g4gAZ81nNqPi54TPisuCbObKgdeE+Jbbd2ep\\\\\\\/4cS9P+7X972ZrcsDo\\\\\\\/y8DxPUmDqvijLRsqQYGCycYxVnadakSvrqkenIAeAHbUliK+u4qp2QcpMuvqEHG4kRMB2HSTk0rEESlirEGnrIj6oooiXrZ0k3VePJiLT\\\\\\\/KiMN4eQMzcHFVuHZpIj5E8dey5NN5TT7iyiaM7ImsH0NNway3z5z3bwSKjVcxeJSq8tEmAk7IfqpLgKEf7e2WBWOE+pMOL+7DsU5Xsg\\\\\\\/7cwA\\\\\\\/HcjmjFzLjO0Zvttlr5H595MvdNKAMV859sDTaTwktqcZhCLkfblnBccmXdtGZnG4QTpxPvy116QJ83WrCwwv1olUtnWG26k6sJ\\\\\\\/S2QGT7jLDpvRsGuUmhTQM8PEM6AuTSTnm+QsmZMSj3p93keYQ6rIUNOPGdLvPJW\\\\\\\/+2Aga3T1+CEhlnGoaCLcRx\\\\\\\/SRnr7zctOmglP+SNNnVW9Ar98ygVwaF4B52pb9yT5lVM850DxjEGRxJDEax4BOyhvZQrSXf7gDfs6Um26wyn5ZSov5azi7vPGxqoHKHKZWUV+DvfSlVFv1fmS53Y9DTPAXIU9ICnpg3DzWV5i8P8gC+S9gi6dG7T0hXQUC5fId2Z9bRMd5q\\\\\\\/CvIm3IVGidkO6k0UMtL0jBId\\\\\\\/nazzWt688XAAIbekhm2eHxB+45TAchhB\\\\\\\/WfTBn6WHDd2zNsV8K0ml1NCU\\\\\\\/zGJ+tO2NTJ1oWLn9WCE0J01UAGgbndiD7hE01EO0SVcipfrSCdWmhdwj7bD0YmNNV8pMvAeOkgR7at6cZn\\\\\\\/jIMDRdrw82akGpdWKfTcReORJiL1y9S\\\\\\\/PdBD+vrXVspyKCS3uKHhD3apwt6yI11mAQz+yS\\\\\\\/eV9ewyPsILUr65LGi+wNeGMkt7u112r+PHehWhlhwEvCVhgxI+zeZ+XE3QrtZHE9VIWqqvF\\\\\\\/Xercmf58TXCp8tCLLiLTZbFkJodMP7\\\\\\\/2RFMIA0TAcYbiLcQhyBsj6yLxytINkyhaSnim2xsZ1ffuC5UDMJT85SKh8k5yEZ6x2EzB5g10q+bFXAtLkJv\\\\\\\/eCIJdZ2OSqzHqgTfQCIBFpuTkspqJchF1iVdenxK5QdocOUgZeMX+6pYbwC4qmnbc7+EIF8QwfQbDxk1iWzT7gGnPvvF6ZMFa2b1C\\\\\\\/eoSZ4DEcRe3rHnHfratmOZi++c8186rkMkxNTiaQhlF8mxaRRF4EExg1nvTPqDprBG1lklVVEHDb2h9DWp0TdBGo5y6wDGDTjMcINB2gjOlh2VGsNZjbaI7Uero2wQL\\\\\\\/4HHhj6C6zPxVEEhFJEH80c94XI3fiAUiV8d7wFU7dKLd9ZZgV02PrC3AXgo3yVNBXNOrdk7IT7SZPVBpIyqpGFISMqglnbSa7WCVeClwKN3e2IZ5gP8DSDII4ZBCqv1hI0N5n7AAj9+f9f\\\\\\\/HSAffp4iiS9w55NTnclCHnVAbUacyw+7\\\\\\\/FcCLRdpZDi9VnNSiREU6M5Nhhzv04\\\\\\\/3rSV+5VAzTfB4gTfDph0GiegAbptY1CjBHhjFdtHthuk8xyR9nzO6P4FduYTnAmxSOIDsRpw3\\\\\\\/2dTqBuJ8LzCdqeXChfR0y4BBUZNtoGDGk0RiECs7PFM+bIAodu5HImpSlaB4xlCzhnQ3PQHAHTcmKIJ5YT4gK4NGqKptl0qcTEPk393PaYwz6VopoCQG1GgVptEl9zMjrPj+KqanMq828mPQeGopRosiYTngR\\\\\\\/bo4kcqkiCV2RZMEhxt\\\\\\\/TFBIPQb4TSOvIRjGMdyKjCfoTakgxshexT6SBo6P09YwfZpBqfS+HrlJpByQ5r8e1qAEh5UWo3G2H7JnP7e6CHpk\\\\\\\/wOD5N+AR9VqxT5HhhdE55aq\\\\\\\/cwBnFcjPZBy7aEv2S4ulbfCAx9tX+zCEnK6wnR\\\\\\\/ArS1V0\\\\\\\/G4oo7q4EE90RB5BYCDyw0zGRuaqiKfHudmbz2CF+xiJ7a0ODyxyJx5RtuovABlpGL37XHEmzpPz\\\\\\\/hFg8i1t6pU4B\\\\\\\/V4gcKFmzxO4wQk\\\\\\\/u6+rHNh4pSy1B3rEJ3\\\\\\\/VuoHJwcNlOvbbR0xDP8F+jrFJxCwCxFtVGr4h6RNhkdK91R8lHUrDQKwYyYyRnv4T5ybmJAfHkYPd7WCTyemyNqmCHhPBAOIOQxb+yZfipEDCsdVLTK\\\\\\\/hxH1o+koyKlR+C1XNkoTRtlNueVORb1SC3G83xAhRwS7yiTcfBt2+CoCirH16AELcUEm++ACCSXWPOXsIl84bQGvkcbd3dyw\\\\\\\/zCWybliB3bMEa08otYMFA+ByLhDIOgpF5I0oyMQkwQaSU8zgXw5WHz274Oi63X2Ast5iuXOM+p6aWb7ONQ0NlWHn8o37tuvEOOAWQroq7+Xb2e1FrjTu3+EPspjvF78lp5gg322igWw92szdB4Ru+S5yvYiqGutHGJokS29646g\\\\\\\/IIf8oV+TfJZJYSKSZlf1u6Ru7onu74VPhwVHmm3NfMD5r868Bqyvn7hVUKqj1779d44P4yyvtTTdcJ30CFSz8mv3HFYAnJrGdIJSVXfw7oztDKEb9fh5j1yZzIjlpnuM83h455UXXnYh+EQZXg1lShshVdniMA\\\\\\\/XOlhrOoLDIEthyFdasuKj2etYTyIapXC6HSRM\\\\\\\/T9iZFoHBKR6s4ZkUZ87dh9Ln1Nzt\\\\\\\/tOCWRjavt1y\\\\\\\/FiVGd8NTBwQMOBHlucLd4yiBZ4bvBSitqQaMuZwH7SzHFf61k7oteCMomY+EZ6zARsrK0Q\\\\\\\/ogklqNRM2vxONJERp1lkLj9EFjdsRYXvyYL2l\\\\\\\/5BCXQ6lsxby\\\\\\\/yoLEacd5GFORCiYTL7jk2MKVJMH+YZsAiOHs5ml1mO3xGMkYjsIes5IR9ZmFHPDHy0vqcfmpa5k8oJuDNAaDkTlPhsBXDDzkKRc\\\\\\\/MnF9IXDZN6ZqUPrTbHYI7nUi0F+XPdk4limhYNEVwg1guLhWCCduTOEcX632a6Skoepguy6VzCmcvRnj78Vs8IJ9ucZH431IXXaRku+5\\\\\\\/6CugWJBxAx3Hue+cxsXmzglKnrlOjM3z7NKXR2UGit7J5fA908RW5Q+JLttTrzEInVfkm1kF9T1W9w+hKUDCc\\\\\\\/5NJdvzrJYEAWUbddpujW5mT3yNKTPBxXWpelfpZhMt1aEUBpG1yO9295XZy8nxagoSYh6qtkQDpZ8YrClbdyNtWwr2alDpFlUO8IcOzhprB8cx6J\\\\\\\/GTyirDYyUqTSylwFnWR6Z5ANDRhKh0e6p3RtgOuet2SR22za4AHK+ks+9XNQSp8F+t8uzKLf8mlXRBbeYgblVgTIRQSygsAqxriq82hcjud1GOvDKhzYHDMOV69QSFE3S8W5FxfVuo4YFgTkxRo7nEBtb4rr1INcCbrPwrUDA5GjD5wMIEaom3wgTvYXNpJz7eY2sRTx06lAeCEHRj0ffKcC6P7UURqJd2PGIYLURn5AduGtG5PHqU9BWBmpEj6Z8FFVsJHo9qZUftpTXdS++x9vfr9Xz\\\\\\\/g+XeLi7x0oaps2+IRMQc4RMdgtn+CCs+pz+WAzRsIZOs21eqikzXU4vjuvGhLx8f+QeCfFoDN2FDAXrqCo22Ox3raZI4A14SCdHtznKB9bxc6lpUDHmoHaQIHbocu6pvRS11jW0DZSAXwJugm34bSTwHU9NkGyP0rzMdn4vBxeZpKysS4N8ROxc18leUJ0XtxJZQ65I4\\\\\\\/GyBdffqMigR+SvhYpA8VlcfcoyvjxMwGe9QryEwohpOBEgHMGgM7wZcHox1WYuPhyc2eLaR\\\\\\\/mDxExIt10CTXpe1pXejFkud6IsYCLn4DMKU3KWIrn6GzAKW+kaZXcJhve\\\\\\\/\\\\\\\/kg5RqFXaU7Bfbab6R65ysBBsMyScHsSDXdHIvE37qBHy88wV9yrSYWmhL8+dqr9vrIbUyxNkIRNhreKWe4V7jeEGVrSolSv31PdX3bPVpMQbHFmhS2mFW+B\\\\\\\/KWFeSzo5PWhs+iEU88pG4RKQyfR1Plr4+F6EBVCubQAOhOSDVA+6nxV2CdFUkcVBbAPWdYhxQkOmADdnavMRjISWCEDIlzsyl0DRJDy0ha6FQ9nL+UJdAMAARXloGOnLMwxojpehTVrXaiJAKpt8PSYiuzR4CKa\\\\\\\/fpY8+bldRc7m0NCdDNfPj+KlzPTESwPwPnBYqKcEBtu\\\\\\\/vj6Ru3CC7T4r9Ck6Jtzc9kb\\\\\\\/Bwnpb\\\\\\\/oBcfRLaMFR7fgsq55Qx5ta+hUUovfP4WHGhcb+45+s376JlcYsj+Q2tQpKLC4XijGD0FVpSo8RVEBNDK7RZEaSIPkP2Hfw1cW\\\\\\\/u6mQH9TjwxMna421FiCOAhKueZ2cxq\\\\\\\/tJdlqzKnckWt9kUxKoUETzvkNeBS398yjqhJbJ6rNL9RXtqa0ASrItiOsnnPV8wem3VepwnRJ8ko4WQqFT\\\\\\\/seXxlrEQHfKzh\\\\\\\/vdjYgLn0diBOm70HVePSP+i+X36wFkZAt\\\\\\\/uPw80uN8rL293C7Uso7LD6zk4D2puKkgeIs5A8bYs+ahWMi15OhKMCrT6dCDMKD1+kJjDCoC\\\\\\\/Ig4YJGulyiNpI2G8Z7jO5nEQr0UcMfBpjJ2bA0Aw4uYnEcCE\\\\\\\/a2d9J8jZ5S155aOwj4M5ZkFwu+rIroblKyAEcw4H4ISmXKfpExAtmZYGkk3x0DAgfN4S0zxLe2DOpLD+5rAh4MugHZybtE\\\\\\\/6o1+5c8\\\\\\\/TLZPYAgFUnLoEcWW7tMuCJdi89Dhcr1IkkrrCwRUWoxOdhFKODl7Vxt\\\\\\\/miEWUYZevWW7IWgPqRJFJ2KNBF0dejO8\\\\\\\/1TMoUBWHdyA3cM4KyEpJpRwqS5cA5hNj9fuY4bh79XNIZkhvbF8mZHvW2yjJi1slfy+96nzdeIjuXSo0w3\\\\\\\/eKr3KpzBKlWmf89xyRGWiJnEbGOlp2vJuurG5mtbMdEg570k6Rh2\\\\\\\/787urUsgMsc2TSfVDmVUpYFClcRDjj3ifGTKfuCzmV+qNW1\\\\\\\/KPmd5cEEjrJjt7O\\\\\\\/BWWwiEZ9XfrK1NqJrQq6h0qPL+NNMHXm\\\\\\\/GiJpg0iQkHucIrLUoQeQEJMv9vWsf7oaQRcnuYtwDvzeKTUPxX3\\\\\\\/YIdb2j7XF\\\\\\\/NV+eMmnAvwTTNnwR+atfs3oDG2b1UROTuAWC\\\\\\\/0xNG4cmShXM+1TYRrVhHOBlnelwtfhz9oEDrsilG8O2fI6vF6bC+vEcFoJjizuU2GLPhrrWukwFSbqSg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022joh3jvEFlGBlkPUFa0mVRg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e0de8d7e0e4368f3bf61da6c62d398f524a42f6452cbb490aa21c274d612988a\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RLHDCXapwc2v85Qm6Ujz6H8Dcfavc3FXGt6I73bS05OmLThRgEaGSZfP+Uw6quCPebzwh7OeZn6tMmcRI6NmyNsK3EnJ9gqrpcQzXR5IIXNpLrlgrR2GkHlwouApcNily1z27yeAsWVxMTjdUAuy\\\\\\\/FY03zJFzI6mFCpxEikuqvJpupJBkB3rrCT\\\\\\\/JoV0c2OQgBgdMP4AbJyhtyaRvJYPQg8OFNkClFVgpCS\\\\\\\/oCD5KQzjX3jVFmVVECOTASkjRFPoJitYIUNLa0MRcJkEkyA+D8Yw4prTlLI5zxztEP4xy7QzUg9ZRuUvRlVf4+vtGrlqAw\\\\\\\/YLEXAqTnFxlvksaexFBXk1yrvbdz6E1\\\\\\\/eIblg4CasJ9\\\\\\\/ZaTtxj0PCeGebvRmmG7DquKyGTWWSV\\\\\\\/L9vGq1mOhHcHf3IB9zRfKTUxvCyeZfV+qOa1\\\\\\\/1lRkqAx3\\\\\\\/NfkEKw5o77wsjV2CHs2f\\\\\\\/+UYo6XlPfBrmO0QaXbvXdS+hOBTqSk0L2CCTQmi4\\\\\\\/0uLP2Y4EC+GaW8bY6rletswT\\\\\\\/jU2N66I2OFJTSIOAoP5uhDFdSXFZqq9M4NW8bGiDJFsWb\\\\\\\/yHuWCAd3RXt\\\\\\\/atTBn7VJWOH1TZAY8E0sKXYBaopTfWpRfLRNHGurN76+XlbboC0wU9fWM+agJq5X28g1tN2dfpTMSYhG20LzJQSosMQ4o22b0nBNesouRgl3aFdjORXFmgn93q22MG3f7sAYku1hpc9gWh\\\\\\\/FDtb1mmYaE7ceIrr0y080rkfa3RwHo0meMv6iOuiUmZpjHOewbu+O7GK2TxGCMJ7O2lyfwafHH0e0wYM7QxeLjXbxygHw9VSIWLrG9+VrX8255UQuvRXM9ZKGG5FUwqGwXGIsNRhgKItoVPO531qsr0SVfUBDAm7dPpizzflvk+d3mZpnwEOcX8SkwXp5Gun6niWScyomF0fEvB9BUVZrC5GueoFchBK8Fgc+vZ8Dd3\\\\\\\/EdVM9f8aJ52JpxK3F4SZHZU9C5GZJ8Gdf0S+dy\\\\\\\/5S\\\\\\\/WCTMeyx+ztYXDd2DDkMQ9UrmwZHodJWGMCXpoZzRYbL3\\\\\\\/mYC4oVNgNvTe77yu7sMX4NdAzZPiHFB0dnhG1n63wRs9swK0m23mGVpGTngjoj7j43lmpqhJZVXNMwMdnEGt1rUVvTk\\\\\\\/Y22EJsovk4dy0v1i+Lnip+KTZwy6Vd3xTaxTCjkekL9zg2Jx4Vbdn8Ef+L5H5yyoFSFxckD9jCV4Fx9YuHWz+EECcxY6Cr7HWJj97WtYL143V6ryRKgyKr1yg27sSMlY3BHKTDusGJtIGty7lZ29xMg16fp4P9arqUV162oavmGFmaPhedrgMdj5TLoXDLtY7X6FbxCqIh\\\\\\\/rmoC2P8sq7Wnc4pSPFYdw5jIi4Y22Z0GMcKLleHYARWBkE1F8d5FHcWZ8lCN9f\\\\\\\/+7o9q7Nizko\\\\\\\/b4lQO+btmS2MYiXFmbw5UNg4PO1OB49XWgWgplNteaaWqFOEyeCuN3BuZp4unbbBTwZRv73myKULrrUrQKHMY\\\\\\\/XayHHr5kLHHkGVzsgH2gseTPCAyYuHiBb3pEms84+6HJXA892RfR2vzL9BTZEpBRfMtdJfuz2SN0JQU+ReC2atw5mF0G7TYB7G7F9Bl7hyR48RVf\\\\\\\/w9w5C+KAN0+wG8vAz9yeH41PhJqbBl+gB2Wg82IAMJfG2lq82aTISV3gJ6q4\\\\\\\/JUZeKbdEFH2RgOlck684TMKErANdciL7QPlpjC52oqhc10p5QUKaRJf+epjgbz2DV+SVN4vW6TRYliJ\\\\\\\/961AXkEFHVvWIMr\\\\\\\/8ivd2lzYC5GGYb21VS2Qp2qr1sq5MMGGpUECS9usRoj0cusFIr7OqhMv7rnurN0RBfrx9kfqQWEqn8bIrnwjySyjlIoON6DQGZI0CPUagPTNoye8OJmK+GOKMUv1piMvKA4FoX1+vkbWKFqrBmsbx8hL6Q+VZ9XxHFCCZkQsflfn1DOVEH3eauvhyqOLe604\\\\\\\/T6\\\\\\\/x828xyaRiptmA0DPBqYJeeLivdcArYv9iTrSfMlz3nj7BzsTDno7OBs\\\\\\\/GAr8HGaoIb+KSnroFc\\\\\\\/tl+FScHZyk7gev0bjXKzBnYmDntrxKT\\\\\\\/YCZlVZ0rMSgZ5MXFa35AVxMpEKic\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/Zd07pomElVSOnRtmnk6eQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002271a478c84d6456dbbfe48f6eebd33eae2e186ff7ba9cf12155c69aa838a7a85c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484760 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1643" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-92NIugF_4YwWISVy9HzuaptQCmHeyqsh26nxyfvSoIw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1643" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TWb3pkYCUMt19QxdfSyNtKSGFCo9DA3ckPZKvzmKoY4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1631" title="Loci Contributing to Boric Acid Toxicity in Two Reference Populations of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1653" title="Allelic Variation in the Toll-Like Receptor Adaptor Protein Ticam2 Contributes to SARS-Coronavirus Pathogenesis in Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484760" id="node484760" data-pisa="ggg;7/6/1643" data-pisa-master="ggg;g3.117.041426" data-apath="/ggg/7/6/1643.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Beavis Effect in Next-Generation Mapping Panels in <em>Drosophila melanogaster</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9393-4720" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elizabeth G. King</span> and <span class="highwire-citation-author" data-delta="1">Anthony D. Long</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1643-1652;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041426 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484760" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elizabeth G. King </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Biological Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BG.%2BKing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=King%20EG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BG.%2BKing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9393-4720" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elizabeth G. King</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kingeg@missouri.edu" class="" data-icon-position="" data-hide-link-title="0">kingeg@missouri.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anthony D. Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of California Irvine, California 92697</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BD.%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnthony%2BD.%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/6/1643" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484760" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484760/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1643.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484760" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484760/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1643.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:484760" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:484760/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1643.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484760" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484760/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/6/1643.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484760" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484760/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">A major goal in the analysis of complex traits is to partition the observed genetic variation in a trait into components due to individual loci and perhaps variants within those loci. However, in both QTL mapping and genetic association studies, the estimated percent variation attributable to a QTL is upwardly biased conditional on it being discovered. This bias was first described in two-way QTL mapping experiments by William Beavis, and has been referred to extensively as “the Beavis effect.” The Beavis effect is likely to occur in multiparent population (MPP) panels as well as collections of sequenced lines used for genome-wide association studies (GWAS). However, the strength of the Beavis effect is unknown—and often implicitly assumed to be negligible—when “hits” are obtained from an association panel consisting of hundreds of inbred lines tested across millions of SNPs, or in multiparent mapping populations where mapping involves fitting a complex statistical model with several d.f. at thousands of genetic intervals. To estimate the size of the effect in more complex panels, we performed simulations of both biallelic and multiallelic QTL in two major <em>Drosophila melanogaster</em> mapping panels, the GWAS-based <em>Drosophila</em> Genetic Reference Panel (DGRP), and the MPP the <em>Drosophila</em> Synthetic Population Resource (DSPR). Our results show that overestimation is determined most strongly by sample size and is only minimally impacted by the mapping design. When &lt; 100, 200, 500, and 1000 lines are employed, the variance attributable to hits is inflated by factors of 6, 3, 1.5, and 1.1, respectively, for a QTL that truly contributes 5% to the variation in the trait. This overestimation indicates that QTL could be difficult to validate in follow-up replication experiments where additional individuals are examined. Further, QTL could be difficult to cross-validate between the two <em>Drosophila</em> resources. We provide guidelines for: (1) the sample sizes necessary to accurately estimate the percent variance to an identified QTL, (2) the conditions under which one is likely to replicate a mapped QTL in a second study using the same mapping population, and (3) the conditions under which a QTL mapped in one mapping panel is likely to replicate in the other (DGRP and DSPR).</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/complex-traits" class="hw-term hw-article-keyword hw-article-keyword-complex-traits" rel="nofollow">complex traits</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila-melanogaster" class="hw-term hw-article-keyword hw-article-keyword-drosophila-melanogaster" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila melanogaster</em></a></li><li class="kwd"><a href="/keyword/beavis-effect" class="hw-term hw-article-keyword hw-article-keyword-beavis-effect" rel="nofollow">Beavis effect</a></li><li class="kwd"><a href="/keyword/qtl-mapping" class="hw-term hw-article-keyword hw-article-keyword-qtl-mapping" rel="nofollow">QTL mapping</a></li><li class="kwd"><a href="/keyword/gwas" class="hw-term hw-article-keyword hw-article-keyword-gwas" rel="nofollow">GWAS</a></li><li class="kwd"><a href="/keyword/dspr" class="hw-term hw-article-keyword hw-article-keyword-dspr" rel="nofollow">DSPR</a></li><li class="kwd"><a href="/keyword/dgrp" class="hw-term hw-article-keyword hw-article-keyword-dgrp" rel="nofollow">DGRP</a></li><li class="kwd"><a href="/keyword/multiparental-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations" rel="nofollow">multiparental populations</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-3">One of the most fundamental questions about the genetic basis of complex traits is: how much does each individual locus contribute to the total genetic variance that exists for a phenotype (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Falconer and Mackay 1996</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Roff 1997</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lynch and Walsh 1998</a>)? Are there few loci contributing substantial amounts or many small loci, each with marginal contributions? Obviously, answering these questions requires the ability to estimate the magnitude of the contribution of a locus (<em>i.e.</em>, QTL) to a phenotype. However, this straightforward goal is complicated by the Beavis effect [first described in <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Beavis <em>et al.</em> (1991)</a> and <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beavis (1994)</a>], the well-known phenomenon that when the percentage of variance explained (hereafter PVE) by a locus is estimated only for statistically significant loci, the PVE will be overestimated on average, and in some cases severely so. Both <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Utz and Melchinger (1994)</a> and <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Beavis (1998)</a> performed simulations demonstrating this bias for classic two-line cross designs. They demonstrated that the severity of the bias increases with decreasing power, thus the bias is expected to be greatest when sample size is low and/or when the true contribution of a locus is small. <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Xu (2003)</a> analytically derived an equation predicting the severity of the bias for a backcross QTL mapping design, which agreed with the original simulation results of <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Beavis (1998)</a>. Thus, the statistical properties of the Beavis effect have been well characterized for traditional two-line QTL mapping designs; however, it is not known how precisely these results apply to modern mapping approaches.</p><p id="p-4">Modern, “next-generation” mapping panels differ in several key ways that could influence the statistical properties of the Beavis effect in those panels. Here, we focus on GWAS panels and MPPs. GWAS perform single marker association tests from sets of individuals or lines, associating the genotype at a locus with the phenotype. The bias associated with GWAS has been noted previously (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Göring <em>et al.</em> 2001</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hirschhorn and Daly 2005</a>) in human studies. In model system GWAS, tests for association can be carried out at millions of SNPs in panels of a few hundred inbred lines (<em>e.g.</em>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Flint-Garcia <em>et al.</em> 2005</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mackay <em>et al.</em> 2012</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alonso-Blanco <em>et al.</em> 2016</a>). MPPs are more similar to traditional QTL mapping designs, but involve a multi-way initial cross between founder strains followed by multiple generations of intercrossing (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kover <em>et al.</em> 2009</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Aylor <em>et al.</em> 2011</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">King <em>et al.</em> 2012a</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">b</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cubillos <em>et al.</em> 2013</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Long <em>et al.</em> 2014</a>). In these populations, mapping involves associating the haplotype at a locus with the phenotype of interest. The number of possible haplotypes at a locus is equal to the number of contributing parents in the MPP, leading to more complex models with more d.f. than the single degree of freedom test typically carried out in a genetic association study. One major difference in modern panels is the frequency distribution of alleles in these panels. The frequency of the alleles at a causative locus is known to affect the power to detect it, with maximal power when alleles are at 0.5 and reduced power as one allele becomes rare (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Slager <em>et al.</em> 2000</a>). In traditional two-line intercross designs, allele frequencies rarely deviate much from a frequency of 0.5. However, the expected frequency distributions for GWAS panels and MPP panels are quite different (<em>e.g.</em>, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kover <em>et al.</em> 2009</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Aylor <em>et al.</em> 2011</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">King <em>et al.</em> 2012a</a>). In GWAS-based panels, alleles are at their natural allele frequencies and thus, there are many alleles at low frequency. In MPPs, alleles are expected to be at a frequency of 1/<em>n</em>, where n equals the number of parents. However, particularly when there is a longer crossing period, drift and selection will cause true frequencies to deviate, in some cases substantially, from expectation (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">King <em>et al.</em> 2012a</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chesler <em>et al.</em> 2016</a>). In addition, modern mapping panels have higher marker density, and thus less missing genotype information. In both types of panels, genome information is increasingly approaching complete genome information, with full resequencing of all lines in GWAS panels and full resequencing of parent lines coupled with dense genotyping of recombinant inbred lines (RILs) in MPPs. All of the above factors have the potential to influence the degree of bias in estimating the contribution of individual QTL to the genetic variance.</p><p id="p-5">When the Beavis effect is substantial, there is not an expectation of high concordance among different mapping studies. One would not necessarily expect to map the same QTL in replicated experiments or in different mapping panels. Further, for successfully mapped QTL, one might expect estimated effect sizes to differ, perhaps considerably. In the past, a failure to appreciate this reality has led to results that are consistent with one another being incorrectly described as contradictory (<em>e.g.</em>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altshuler <em>et al.</em> 2000</a>). In addition, in some cases, complex hypotheses are invoked to explain inconsistent results for the same loci that could also be explained by the Beavis effect (<em>e.g.</em>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Huang <em>et al.</em> 2012</a>). Therefore, it is valuable to clarify the expectations for the validation of mapped QTL within the same mapping populations and between different mapping populations.</p><p id="p-6">In this paper, we focus on two next-generation mapping panels used widely in the <em>Drosophila melanogaster</em> community, the GWAS-based DGRP and the MPP DSPR. We simulate QTL that account for different true PVE and map these simulated QTL for different sample sizes in both panels. We then quantify the resulting bias in the estimates of the PVE by significant QTL. Finally, we discuss the factors that influence the likelihood of replicating a QTL in a second mapping study of the same design, as well as the ability to replicate mapped factors between resources.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Mapping populations</h3><p id="p-7">We used two major community resources for genetic mapping in the <em>D. melanogaster</em> system, the DGRP (<a href="http://dgrp2.gnets.ncsu.edu/data.html">http://dgrp2.gnets.ncsu.edu/data.html</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Mackay <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Huang <em>et al.</em> 2014</a>) and the DSPR (<a href="http://FlyRILs.org">http://FlyRILs.org</a>; <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">King <em>et al.</em> 2012a</a>,<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">b</a>), to perform simulations. All simulations and analyses described below were performed in R (version 3.3.1; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">R Core Team 2016</a>) and all code is publicly available via GitHub (<a href="https://github.com/egking/QTLbiasSIM">https://github.com/egking/QTLbiasSIM</a>). The same set of code with nearly all raw and intermediate data files is available at Zenodo: <a href="http://doi.org/10.5281/zenodo.438140">http://doi.org/10.5281/zenodo.438140</a>.</p><p id="p-8">For the DGRP, we used the DGRP Freeze 2.0 release, consisting of 205 inbred lines (<a href="http://dgrp2.gnets.ncsu.edu/data.html">http://dgrp2.gnets.ncsu.edu/data.html</a>) created from wild-caught females in Raleigh, NC. All lines have been resequenced, and genetic variants have been called. Details about the formation and sequencing of this mapping population are available in <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Mackay <em>et al.</em> (2012)</a> and <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Huang <em>et al.</em> (2014)</a>. Briefly, the DGRP lines were created by collecting over 1000 mated females and establishing isofemale lines, which were then subjected to 20 generations of full-sib mating. To quantify genetic similarity between lines, we used the A.mat function in the rrBLUP package to obtain a global kinship matrix (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Endelman 2011</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Poland <em>et al.</em> 2012</a>). We included only positions with a call in 80% of lines and a minor allele frequency &gt; 5%. Subsequently, we dropped one member of each set of lines whose coefficient of coancestry exceeded 0.25. A total of 20 lines were dropped, giving a final set of 185 lines.</p><p id="p-9">For the DSPR, we used the data from the DSPR release 4 (<a href="http://FlyRILs.org/Data">http://FlyRILs.org/Data</a>), which consists of two populations (pA and pB) of RILs, each created from an eight-way, 50 generation cross. Following the crossing phase, lines were subjected to 20–25 generations of full-sib mating. The two populations of RILs are essentially replicates; each was created from a different set of eight founder lines with the exception of a single founder line that is shared between the two populations. The founder lines have been fully resequenced and all RILs have been genotyped using RAD markers. These data inform a hidden Markov model (HMM) that infers the probability that each segment in each RIL is derived from each founder line, producing near complete genome information for all RILs (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">King <em>et al.</em> 2012b</a>). All data associated with the DSPR lines are available here: <a href="http://FlyRILs.org/Data">http://FlyRILs.org/Data</a>. Complete details about the formation and genetic characterization of the DSPR are available in <a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">King <em>et al.</em> (2012a</a>,<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">b</a>). For our simulations below, we used only the pA RILs and expect that the results would apply equally to pB RILs.</p></div><div id="sec-3" class="subsection"><h3>Simulated phenotypes</h3><p id="p-10">For the DSPR, we used the set of HMM probabilities (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">King <em>et al.</em> 2012b</a>) and founder genotype calls to impute full genotype data for each RIL corresponding to the probability of a given SNP flavor at each location for each RIL. For a given RIL at a given position, the probability it harbors the reference allele is given by:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-1.gif"/></noscript></span></span>where <em>F<sub>i</sub></em> is the genotype for the <em>i</em>th founder (0 = alternative allele, 1 = reference allele) and <em>P<sub>i</sub></em> is the probability the RIL harbors the <em>i</em>th founder genotype at the position. We then dropped any positions with a minor allele frequency &lt; 2.5%. We also excluded positions that were outside the set of genotyped positions in the RILs, where we therefore do not have a haplotype assignment from our HMM. At each position where we simulated a QTL, we used the binomial distribution to assign either the reference or alternative allele to each RIL using the probabilities calculated above. Since founder allelic states are inferred with near 100% accuracy for most loci (<em>i.e.</em>, the <em>F<sub>i</sub></em>’s), and similarly at most positions in the genome in any given RIL the founder state is also highly certain (<em>i.e.</em>, the <em>P<sub>i</sub></em>’s), for any given RIL at any given position the Prob(REF) is close to zero or one. In fact, over all RILs and SNPs only 4% of Prob(REF) inferences are between 2.5 and 97.5%. We simulated sample sizes of 100, 185 (to match the DGRP), 500, and 878 (the size of the full pA panel).</p><p id="p-11">In the DGRP, we dropped any positions with a minor allele frequency &lt; 2.5% in the resource or that was missing from &gt; 20% of lines. We then used the kinship-based EM method implemented in the A.mat function in the rrBLUP package to impute any missing genotypes. In addition, when selecting sites to simulate QTL, we excluded positions that were near major inversions (Release five coordinates = <em>2L:</em> 0.4–14.9 Mb, <em>2R</em>: 9–18 Mb, and <em>3R</em>: 6–27 Mb). We considered sample sizes of 100 and 185 (the size of the full DGRP panel).</p><p id="p-12">We then simulated both biallelic and multiallelic QTL for different values of the PVE by the QTL (5, 10, and 20%) in both mapping populations. In the biallelic case, we simulated a QTL at a given position by randomly choosing a SNP and adding environmental variance to correspond to the given PVE by the QTL to generate a phenotype. We generated a set of random normal deviates <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-2.gif"/></noscript></span></span> to correspond to environmental variance for each effect size where <em>z</em> = the percent of the phenotypic variance explained by the QTL and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-3.gif"/></noscript></span></span> is the genetic variance at the QTL. We also simulated a gene-based multiallelic case (c.f. <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Thornton <em>et al.</em> 2013</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Long <em>et al.</em> 2014</a>). For each gene, we chose three random SNPs within the interval of that gene ± 1 kb on either side. We then assigned a multiallelic additive genotype for each line by summing across the three SNPs (<em>e.g.</em>, 000 = 0, 001 = 1, 011 = 2, 010 = 1… 0.111 = 3). We then added environmental variance as above to create phenotypes. In most cases, we performed 1000 independent simulations, simulating a single QTL in each at randomly chosen loci. However, when power is low, few QTL are detected, producing a small sample size to estimate the average PVE. Therefore, to ensure a reasonable estimate of the PVE by detected QTL, we performed additional simulations whenever the number detected was &lt; 30 (the number of simulations is given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484844/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484844/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Power (% QTL identified) and % validated in the DGRP and DSPR for different sample sizes and true PVE by the simulated QTL</span><div class="sb-div caption-clear"></div></div></div><p id="p-17">In both the DSPR and DGRP, the fact that all individuals of a given line are genetically identical allows for multiple measures per line to reduce environmental error, effectively increasing the PVE by the QTL. All our simulations only considered a single QTL, thus <em>V</em><sub>e</sub> should be thought of as a measure of the true random environmental noise plus the genetic variance due to all QTL in gametic phase linkage equilibrium with the focus locus. In both the DSPR and DGRP, linkage disequilibrium in the ILs/RILs only extends over small genetic intervals, so for major QTL this is a reasonable assumption. In the DGRP, linkage disequilibrium decays rapidly, reaching background levels within hundreds of base pairs (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Mackay <em>et al.</em> 2012</a>). In the DSPR, linkage extends further but decays to background within ∼2 Mb (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041426/-/DC1/SupplementalFigures.pdf">File S1</a>).</p></div><div id="sec-4" class="subsection"><h3>Mapping simulated QTL</h3><p id="p-18">When performing genetic mapping, it is necessary to account for relatedness between lines, particularly when genetic similarity varies substantially between different sets of lines. However, including the causative marker in the calculation of the kinship matrix is known to produce overly conservative mapping results (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Yang <em>et al.</em> 2014</a>), which we confirmed with our simulations (Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041426/-/DC1/SupplementalFigures.pdf">File S1</a>). To avoid this effect, we used the leave-one-chromosome-out (LOCO) method and computed separate kinship matrices leaving one chromosome arm out at a time (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Yang <em>et al.</em> 2014</a>). When testing a given marker, we used the corresponding kinship matrix that does not include that chromosome arm. Other approaches to mapping that could be implemented to avoid overly conservative results are the NAM-R (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Xavier <em>et al.</em> 2015</a>) R package or the BayesCpi function in the gdmp (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdel-Azim 2016</a>) R package. For the DGRP, we used the A.mat function in the rrBLUP package to obtain these kinship matrices (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Endelman 2011</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Poland <em>et al.</em> 2012</a>). For the DSPR, we estimated the kinship matrix using the haplotype inferences stemming from the HMM as the proportion of the genome identical-by-descent between any pair of RILs, using ∼11,000 regularly spaced positions on a genetic scale. For any given pair of RILs (a and b), the proportion of the genome identical by descent is given by,<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-4.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-5.gif"/></noscript></span></span> is the probability the <em>a</em>th RIL harbors the <em>i</em>th founder genotype at the <em>j</em>th position, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-6.gif"/></noscript></span></span> is the probability the <em>b</em>th RIL harbors the <em>i</em>th founder genotype at the <em>j</em>th position, and <em>n</em> is the number of positions. The DGRP consists of lines derived from a wild-caught population. Thus, haplotype blocks are short and haplotype information is not estimated. The typical model fit is a biallelic association test at each marker. We adapted the assoc.map function in the DOQTL package (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gatti <em>et al.</em> 2014</a>) to the DGRP data to perform a linear mixed model. This function relies on the methods developed for the QTLRel package (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cheng <em>et al.</em> 2011</a>) and the regress package (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Clifford and McCullagh 2006</a>) to account for kinship (using LOCO kinship matrices) and the Matrix eQTL algorithm for computational efficiency. The model at a given locus is as follows:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-7.gif"/></noscript></span></span>where <em>y<sub>i</sub></em> is the phenotype of the <em>i</em>th line, <em>x<sub>i</sub></em> is the genotype of the <em>i</em>th line (0 = aa, 2 = AA), <em>b<sub>i</sub></em> is the vector of SNP effects, <em>g<sub>i</sub></em> is the vector of genetic effects to adjust for kinship, and <em>e<sub>i</sub></em> is the vector of residuals. For each simulated QTL, we fitted this model at each variant within a 10 Mb (± 5 Mb) region centered on the causative SNP. We performed this localized mapping instead of genome-wide scans to reduce the computational effort involved in performing the mapping. Because we simulated a single QTL for each simulation and are only interested in the estimated PVE and significance at that QTL, we can perform mapping for only this small region.</p><p id="p-19">In the DSPR, the test is for association between the founder haplotype at a given position and the phenotype (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>). We adapted the fast.qtlrel function in the DOQTL package (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Gatti <em>et al.</em> 2014</a>) to the DSPR data to fit a linear mixed model, which relies on the methods developed for the QTLRel package (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Cheng <em>et al.</em> 2011</a>) to account for kinship (using LOCO kinship matrices). The model at a given locus is as follows:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-8.gif"/></noscript></span></span>where <em>y<sub>i</sub></em> is the phenotype of the <em>i</em>th individual, <em>p<sub>ij</sub></em> is the probability the <em>i</em>th line has the <em>j</em>th haplotype at the locus, <em>b<sub>ij</sub></em> is the vector of effects for the <em>j</em>th haplotype, <em>g<sub>i</sub></em> is the vector of genetic effects to adjust for kinship, and <em>e<sub>i</sub></em> is the vector of residuals. For each simulated QTL, we fitted this model at regularly spaced 10 kb intervals [see <a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">King <em>et al.</em> (2012a</a>,<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">b</a>)] for a 10 Mb region (± 5 Mb of the nearest position to the causative SNP).</p><p id="p-20">Performing thousands to millions of tests across the genome results in a requirement to adjust the threshold for significance to correct for performing multiple comparisons. Traditionally, QTL studies have controlled the family-wise error rate (FWER), the probability of one or more false positives occurring genome-wide, at 5%. The threshold corresponding to a 5% FWER is calculated by performing permutations and determining the maximum genome-wide LOD score for each permuted dataset. The FWER corresponding to 5% is the threshold at which there is only a 5% chance of observing at least one false positive (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Churchill and Doerge 1994</a>). Controlling the false discovery rate (FDR) (the number of false positives/the number of total positives) is also a popular approach. However, several studies (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chen and Storey 2006</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Siegmund <em>et al.</em> 2011</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brzyski <em>et al.</em> 2017</a>) have demonstrated that underlying linkage structure can make applying FDR to genome-wide tests problematic. Linkage is expected to produce similar signals across several positions, such that a single signal is expected to produce several significant “hits” nearby for both true and false positives. This dependency can alter the estimation of FDR. If positive signals can be characterized into independent signals, permutations can inform the false positive rate. We use the false positive rate here, not the FDR, as the estimate of FDR is dependent on the number of positives detected in a given genome scan. We performed 2000 genome scans for randomly generated phenotypes with no simulated QTL for each sample size. We used these scans to estimate both the threshold corresponding to a FWER of 5% and the expected number of false positives for a range of significance thresholds. To estimate the number of false positives, we first identified all peak positions for a given genome scan. Then we removed any peaks that were nearby a higher peak. We used different distances for the DSPR and DGRP given the two populations have very different linkage disequilibirum structure. For the DSPR, if a peak was within 2 cM of a higher peak, it was eliminated. For the DGRP, the same was true if a peak was within 0.5 cM. We then recalculated the Beavis effect for different <em>P</em>-value thresholds to determine how the Beavis effect relates to power and the expected false positive rate.</p><p id="p-21">We considered a QTL successfully mapped if a LOD score within the 10 Mb interval surrounding the causative site exceeded the significance threshold. We estimated the PVE by the QTL from the LOD score: 1−10<sup>−(2/</sup><em><sup>n</sup></em><sup>)*LOD</sup> (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>) at the peak position in this interval.</p></div><div id="sec-5" class="subsection"><h3>QTL validation</h3><p id="p-22">We performed additional simulations to consider the likelihood of validating the significant QTL for each parameter combination described above within mapping populations. For parameter combinations where we initially identified significant QTL in over 100 simulations, we randomly selected only 100 to map again; for all others, we selected all significant QTL. We generated a new set of phenotypes for the same PVE and sample size, and performed mapping again as described above within the same mapping population.</p><p id="p-23">We also performed simulations to assess the likelihood of validating a hit in a second mapping population (<em>i.e.</em>, validating a DSPR hit in the DGRP or vice versa). We limited these simulations to a set of our parameter combinations and only considered the biallelic case. For mapped QTL in the DGRP, we considered a sample size of 185 and a PVE of 5 and 10%. All significant QTL for those parameter combinations were then simulated in the DSPR for a sample size of 185 and 878. In the DSPR, we considered mapped QTL at a sample size of 185 and 878 and a PVE of 5 and 10%. These QTL were then simulated in the DGRP at a sample size of 185. We simulated QTL as above but with the effect of the locus set by the initial mapping population. Thus, at each SNP, we generated a set of random normal deviates <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-9.gif"/></noscript></span></span> to correspond to environmental variance for each effect size where <em>z</em> = the percent of the phenotypic variance explained by the QTL in the initial mapping population and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1643/embed/mml-math-10.gif"/></noscript></span></span> is the genetic variance at the QTL in the initial mapping population. This method simulates QTL in the second mapping population with a different PVE whenever the allele frequency is different. For example, a QTL with a 5% PVE in the DGRP will be simulated in the DSPR with a higher PVE if the minor allele frequency at that SNP is higher in the DSPR and vice versa.</p><p id="p-24">When validating QTL, the search space is localized instead of genome-wide. Therefore, we calculated the threshold corresponding to a FWER of 5% when mapping is done only for a small region. Using our randomly generated phenotypes with no QTL, we calculated the FWER threshold by randomly choosing a position and considering only a ± 5 Mb region surrounding that position. We then used this threshold to determine the likelihood of validating a given QTL.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-25">All simulations and analyses described below were performed in R (version 3.3.1; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">R Core Team 2016</a>) and all code is publically available via GitHub (<a href="https://github.com/egking/QTLbiasSIM">https://github.com/egking/QTLbiasSIM</a>). The same set of code with nearly all raw and intermediate data files is available at Zenodo: <a href="http://doi.org/10.5281/zenodo.438140">http://doi.org/10.5281/zenodo.438140</a>.</p></div></div><div class="section results-discussion" id="sec-7"><h2 class="">Results and Discussion</h2><div id="sec-8" class="subsection"><h3>Estimating the percentage variance due to a QTL in the DGRP and DSPR</h3><p id="p-26">As has been shown previously (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Utz and Melchinger 1994</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Beavis 1998</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Xu 2003</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>), the PVE by significant QTL is overestimated in all except the most highly powered cases and the overestimation is most severe for the lowest sample sizes in both the DGRP and the DSPR (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Power is lower in the DGRP compared to the DSPR (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), resulting from a more stringent <em>P</em>-value threshold due to the fact that many more tests are performed in the DGRP, both in actual tests performed (∼11,000 <em>vs.</em> ∼2.4 million) and in the functional number of independent tests based on the size of haplotype blocks and the extent of linkage across the genome. However, the thresholds corresponding to the PVE are similar between the two populations, leading to similar overall magnitudes of the Beavis effect.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The percent variance explained by significant QTL in the DSPR (A&#x2013;C) and the DGRP (D&#x2013;F) for different sample sizes. The true percent variance explained by the QTL is shown by the solid line [(A and D) = 0.05, (B and E) = 0.1, and (C and F) = 0.2]. Each point corresponds to a single significant QTL. Note that different parameter combinations lead to different numbers of detected QTL and that lower powered conditions result in fewer observations. Boxplots are overlayed with the central line corresponding to the median and the box encompassing the first and third quartiles. DGRP, Drosophila Genetic Reference Panel; DSRP, Drosophila Synthetic Population Resource; QTL, quantitative trait loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-623097679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The percent variance explained by significant QTL in the DSPR (A&#x2013;C) and the DGRP (D&#x2013;F) for different sample sizes. The true percent variance explained by the QTL is shown by the solid line [(A and D) = 0.05, (B and E) = 0.1, and (C and F) = 0.2]. Each point corresponds to a single significant QTL. Note that different parameter combinations lead to different numbers of detected QTL and that lower powered conditions result in fewer observations. Boxplots are overlayed with the central line corresponding to the median and the box encompassing the first and third quartiles. DGRP, &lt;em&gt;Drosophila&lt;/em&gt; Genetic Reference Panel; DSRP, &lt;em&gt;Drosophila&lt;/em&gt; Synthetic Population Resource; QTL, quantitative trait loci.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1643/F1.medium.gif" width="440" height="311"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/6/1643/F1.medium.gif" width="440" height="311"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484929" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">The percent variance explained by significant QTL in the DSPR (A–C) and the DGRP (D–F) for different sample sizes. The true percent variance explained by the QTL is shown by the solid line [(A and D) = 0.05, (B and E) = 0.1, and (C and F) = 0.2]. Each point corresponds to a single significant QTL. Note that different parameter combinations lead to different numbers of detected QTL and that lower powered conditions result in fewer observations. Boxplots are overlayed with the central line corresponding to the median and the box encompassing the first and third quartiles. DGRP, <em>Drosophila</em> Genetic Reference Panel; DSRP, <em>Drosophila</em> Synthetic Population Resource; QTL, quantitative trait loci.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">In the DSPR, where the haplotype encompasses the complete multiallelic effect, there is little difference between the biallelic and multiallelic cases. In contrast, in the DGRP, each marker is tested separately. Therefore, the actual contribution of individual SNPs is lower than the overall effect at the QTL. Thus, in the multiallelic case in the DGRP, the overestimation, when it occurs, is even greater than it appears. In the highest powered case, the multiallelic case underestimates the effect at the QTL as a whole, because it is estimated from only a single contributing SNP. Association tests that attempt to consider all SNPs within a given gene, such as burden tests, may better estimate the contribution of multiple SNPs; however, several possible methods exist that vary in their power under different genetic architectures (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Thornton <em>et al.</em> 2013</a>).</p><p id="p-29">There is a small, but consistent difference between the mapping populations in the magnitude of the bias, with a larger bias in the DSPR. We hypothesize that the difference is due to the additional terms in the haplotype-based model and potential overfitting. The Beavis effect stems from conditioning on only significant QTL to estimate the PVE by QTL. If we instead estimated the PVE for all our simulated QTL at their true locations, regardless of whether they were found to be significant, the estimates are expected to be normally distributed and centered on the true magnitude. At the smallest sample sizes, both the DGRP and the DSPR deviate from this expectation, but the DSPR deviates much more severely (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The DSPR model estimates an effect separately for each haplotype, but our simulated QTL are produced from fewer groups. For example, when the true effect is at a single SNP, just two groups are necessary. In addition, as sample size decreases, it becomes more likely that there will be little or in some cases no representation of some founder haplotypes at a given position. These haplotypes are nonetheless included in the model and this may contribute to an overfitting effect. However, we also note that the additional bias in the DSPR compared to the DGRP is quite small compared to the overall bias (∼2%; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The percent variance explained by all simulated QTL at the true location in the DSPR (A&#x2013;C) and the DGRP (D&#x2013;F) for different sample sizes. The true percent variance explained by the QTL is shown by the solid line [(A and D) = 0.05, (B and E) = 0.1, and (C and F) = 0.2]. Each point corresponds to a single QTL. Boxplots are overlayed with the central line corresponding to the median and the box encompassing the first and third quartiles. DGRP, Drosophila Genetic Reference Panel; DSRP, Drosophila Synthetic Population Resource; QTL, quantitative trait loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-623097679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The percent variance explained by all simulated QTL at the true location in the DSPR (A&#x2013;C) and the DGRP (D&#x2013;F) for different sample sizes. The true percent variance explained by the QTL is shown by the solid line [(A and D) = 0.05, (B and E) = 0.1, and (C and F) = 0.2]. Each point corresponds to a single QTL. Boxplots are overlayed with the central line corresponding to the median and the box encompassing the first and third quartiles. DGRP, &lt;em&gt;Drosophila&lt;/em&gt; Genetic Reference Panel; DSRP, &lt;em&gt;Drosophila&lt;/em&gt; Synthetic Population Resource; QTL, quantitative trait loci.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1643/F2.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/6/1643/F2.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484817" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-30" class="first-child">The percent variance explained by all simulated QTL at the true location in the DSPR (A–C) and the DGRP (D–F) for different sample sizes. The true percent variance explained by the QTL is shown by the solid line [(A and D) = 0.05, (B and E) = 0.1, and (C and F) = 0.2]. Each point corresponds to a single QTL. Boxplots are overlayed with the central line corresponding to the median and the box encompassing the first and third quartiles. DGRP, <em>Drosophila</em> Genetic Reference Panel; DSRP, <em>Drosophila</em> Synthetic Population Resource; QTL, quantitative trait loci.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The average estimated percent variance explained by significant QTL vs. the true percent variance explained by those QTL for different sample sizes (colors) and mapping populations (symbols). Only simulated biallelic cases are shown. The 1:1 line is plotted in gray and represents where points should lie if estimated parameters match the true parameters. DGRP, Drosophila Genetic Reference Panel; DSRP, Drosophila Synthetic Population Resource; QTL, quantitative trait loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-623097679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The average estimated percent variance explained by significant QTL &lt;em&gt;vs.&lt;/em&gt; the true percent variance explained by those QTL for different sample sizes (colors) and mapping populations (symbols). Only simulated biallelic cases are shown. The 1:1 line is plotted in gray and represents where points should lie if estimated parameters match the true parameters. DGRP, &lt;em&gt;Drosophila&lt;/em&gt; Genetic Reference Panel; DSRP, &lt;em&gt;Drosophila&lt;/em&gt; Synthetic Population Resource; QTL, quantitative trait loci.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1643/F3.medium.gif" width="440" height="349"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/6/1643/F3.medium.gif" width="440" height="349"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484964" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">The average estimated percent variance explained by significant QTL <em>vs.</em> the true percent variance explained by those QTL for different sample sizes (colors) and mapping populations (symbols). Only simulated biallelic cases are shown. The 1:1 line is plotted in gray and represents where points should lie if estimated parameters match the true parameters. DGRP, <em>Drosophila</em> Genetic Reference Panel; DSRP, <em>Drosophila</em> Synthetic Population Resource; QTL, quantitative trait loci.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Overall, the discovery of bias is not a new result, but it is worth noting that direct estimates of the PVE by QTL at low sample sizes are nearly wholly uninformative of the true effect, <em>i.e.</em>, the estimated PVE bears no relation to the true PVE (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Despite this fact, the PVE by QTL are routinely reported for mapping studies with small sample sizes (<em>e.g.</em>, <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Aylor <em>et al.</em> 2011</a>; <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Mackay <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">King <em>et al.</em> 2014a</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Liu <em>et al.</em> 2016</a>), and there is not an associated indicator of uncertainty such as a standard error to aid interpretation. Cross validation, which uses one portion of the data to identify significant QTL and the remaining data to estimate the PVE, has been shown to be an effective strategy to avoid bias in the estimation of the PVE attributable to QTL (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Utz <em>et al.</em> 2000</a>). However, this strategy is not routinely employed given that the full data set cannot be used to assess significance and there is a desire to maximize power. We advise explicitly acknowledging the lowest PVE that could be detected at a given sample size and the potential bias when cross validation is not employed.</p></div><div id="sec-9" class="subsection"><h3>Significance thresholds and the Beavis effect</h3><p id="p-33">As described above, the Beavis effect results from choosing only the factors identified as significant, and thus choosing PVE at the tail of the overestimated side. Our results are thus dependent on the stringency of the significance threshold used. The above results controlled the family-wise error rate at 5%, which is a reliable but conservative strategy. Using more liberal criteria for significance will decrease the Beavis effect, but of course will also increase false positives at the same time. This trade-off is similar to the trade-off between the rate of false positives and false negatives. The Beavis effect can be reduced and power can be increased but only at the cost of increasing the false positive rate (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) The mean expected false positive rate for different P-value thresholds for the DSPR (red) and DGRP (black), with a sample size of 100 shown in light colors and the full panel size shown in dark (185 for DGRP and 878 for DSPR). (B) The mean estimated PVEexplained for different P-value thresholds for a true PVE of 10% and a sample size of 185. DSPR is in red and DGRP is in black. (C) Power for different P-value thresholds for a QTL with a PVE of 10% and a sample size of 185. DSPR is in red and DGRP is in black. DGRP, Drosophila Genetic Reference Panel; DSRP, Drosophila Synthetic Population Resource; PVE, percentage of variance explained; QTL, quantitative trait loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-623097679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) The mean expected false positive rate for different &lt;em&gt;P&lt;/em&gt;-value thresholds for the DSPR (red) and DGRP (black), with a sample size of 100 shown in light colors and the full panel size shown in dark (185 for DGRP and 878 for DSPR). (B) The mean estimated PVEexplained for different &lt;em&gt;P&lt;/em&gt;-value thresholds for a true PVE of 10% and a sample size of 185. DSPR is in red and DGRP is in black. (C) Power for different &lt;em&gt;P&lt;/em&gt;-value thresholds for a QTL with a PVE of 10% and a sample size of 185. DSPR is in red and DGRP is in black. DGRP, &lt;em&gt;Drosophila&lt;/em&gt; Genetic Reference Panel; DSRP, &lt;em&gt;Drosophila&lt;/em&gt; Synthetic Population Resource; PVE, percentage of variance explained; QTL, quantitative trait loci.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1643/F4.medium.gif" width="205" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/6/1643/F4.medium.gif" width="205" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484978" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">(A) The mean expected false positive rate for different <em>P</em>-value thresholds for the DSPR (red) and DGRP (black), with a sample size of 100 shown in light colors and the full panel size shown in dark (185 for DGRP and 878 for DSPR). (B) The mean estimated PVEexplained for different <em>P</em>-value thresholds for a true PVE of 10% and a sample size of 185. DSPR is in red and DGRP is in black. (C) Power for different <em>P</em>-value thresholds for a QTL with a PVE of 10% and a sample size of 185. DSPR is in red and DGRP is in black. DGRP, <em>Drosophila</em> Genetic Reference Panel; DSRP, <em>Drosophila</em> Synthetic Population Resource; PVE, percentage of variance explained; QTL, quantitative trait loci.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">One of the major difficulties in assessing the potential true PVE from the estimated PVE, and how the bias might change with the significance threshold, is the lack of knowledge about the true distribution of effect sizes. For example, the probability that a given estimated PVE discovered in an experiment is mildly biased <em>vs.</em> severely biased is highly dependent on the underlying distribution of the true PVE of all causative variants. This distribution is typically assumed to be exponentially or γ distributed, with an abundance of low PVE loci and very few high PVE loci. As we have shown, the severity of the bias depends strongly on sample size, but it will also depend on this distribution of true PVE. For example, even if sample sizes are relatively high (<em>e.g.</em>, ∼1000), if the vast majority of causative variants contribute 1% or less to the phenotype, the resulting bias will generally still be quite severe, because power declines with decreasing PVE (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Utz and Melchinger 1994</a>; <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Beavis 1998</a>; <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Xu 2003</a>; <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>).</p></div><div id="sec-10" class="subsection"><h3>Prospects for validation of putative QTL</h3><p id="p-36">Within a mapping population, validation attempts typically involve attempting to map the same locus using a different experimental design, such as crossing many lines to create an outbred population, phenotyping this population, and resequencing a set of individuals from the tails of the distribution. Another method is crossing the lines to a standard line to perform phenotyping. When these methods are employed and there is little concordance between significant hits, a tempting conclusion is the presence of genetic background effects. Here, we simulated a QTL with identical parameters twice to determine the degree that one would expect replication in ideal circumstances. In this case, the likelihood of validating a true QTL could potentially be calculated by assessing power with our local region FWER. However, given a significant QTL in one study, the same genome-wide multiple testing criterion may not need to be applied if mapping is done locally. The power to validate a given QTL with our local region FWER threshold is shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. It is also possible that individual loci may be more or less likely to replicate depending on other factors affecting power, such as frequency or the location on the chromosome. To account for this, we explicitly simulated new phenotypes for the loci deemed significant and performed mapping again. The resulting probability of validating QTL within populations is given in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, though we note that in some parameter combinations with low power this probability is based on few loci. In most cases, the power to validate our previously identified loci is higher than considering all our simulated loci at the lower threshold, indicating that factors such as allele frequencies also influence the likelihood of validating a given locus. In many realistic parameter combinations (<em>e.g.</em>, a PVE of 5% and a sample size of a few hundred), the likelihood of validating a QTL is &lt; 50% in both mapping populations. Furthermore, in many complex traits, the PVE of most causative loci are expected to be &lt; 5%. Therefore, the failure of validation attempts within populations should not be taken as evidence for epistasis in the absence of additional evidence.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484859/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484859/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Power (% QTL identified) to validate QTL in the DGRP and DSPR for different sample sizes and true PVE by the simulated QTL</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">Between populations, several additional factors besides a lower threshold will strongly influence the likelihood of validating a QTL in a second mapping population (<em>i.e.</em>, validating a DSPR hit in the DGRP or vice versa). First, a causative allele that is not segregating in the second population will obviously have no chance of being validated. In the DGRP and DSPR, there are a total of ∼2.5 million SNPs that passed our criteria, which excludes any SNP with a minor allele frequency of &lt; 2.5% (see <em>Methods</em>). Of these, 60% are unique to the DGRP, 34% are shared between the two populations, and 7% are unique to the DSPR (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5a</a>). Thus, for biallelic QTL in these populations, only 34% of loci have a nonzero chance of being replicated. However, in the case of multiallelic QTL, where there are several SNPs within a causative QTL that affect the phenotype, the same precise SNPs do not necessarily need to be present for a QTL to replicate. There is a growing body of evidence in support of the hypothesis that mapped QTL can be multiallelic. For example, in the DSPR, there is evidence multiple alleles exist at causative QTL for large effect traits such as the <em>Adh</em> locus (<a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">King <em>et al.</em> 2012a</a>) and large effect <em>cis</em> eQTLs (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">King <em>et al.</em> 2014b</a>), a pattern that has also been found for human diseases [reviewed in <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">McClellan and King (2010)</a>] and is supported by theoretical models (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Pritchard 2001</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Thornton <em>et al.</em> 2013</a>). Second, for an allele with a given additive effect, the allele frequency of the causative SNP will influence the PVE by the allele in the second population. In other words, if an allele with a given effect in one population is lower in frequency in the second population, it will explain a lower percentage of the variance. For the set of SNPs shared between the DGRP and the DSPR, the minor allele frequencies are correlated, but the relationship is not strong (<em>r</em> = 0.19, <em>P</em> &lt; 0.001). Alleles are equally likely to be at increased frequency in the DSPR as in the DGRP, with an average difference of 14% (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5b</a>). Of course, this applies only to the set of shared SNPs in the two populations; there is a higher frequency of rare alleles in the DGRP but these alleles are unique to the DGRP. Finally, not only will the frequency of the causative locus vary, but the presence and frequency of all other causative variants will also vary, influencing the heritability of the trait and relative contributions of the loci. For a typical complex trait, the number of these variants is expected to be high. For these reasons, the expected PVE for a given locus in a second mapping population is essentially an unknown quantity. Thus, in the most general sense, the potential for validation is best informed by the power at our lower local region FWER for various PVE and sample sizes (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Density plot of the frequencies of SNPs in the DGRP and DSPR. Different colors denote SNPs unique to each population and shared in the two populations. (B) Histogram of the differences in the MAF for the set of SNPs shared between the DGRP and the DSPR. The black vertical line denotes zero; to the left, the MAF in the DSPR is higher than in the DGRP; and to the right, the MAF is higher in the DGRP. DGRP, Drosophila Genetic Reference Panel; DSRP, Drosophila Synthetic Population Resource; MAF, minor allele frequency; QTL, quantitative trait loci; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-623097679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Density plot of the frequencies of SNPs in the DGRP and DSPR. Different colors denote SNPs unique to each population and shared in the two populations. (B) Histogram of the differences in the MAF for the set of SNPs shared between the DGRP and the DSPR. The black vertical line denotes zero; to the left, the MAF in the DSPR is higher than in the DGRP; and to the right, the MAF is higher in the DGRP. DGRP, &lt;em&gt;Drosophila&lt;/em&gt; Genetic Reference Panel; DSRP, &lt;em&gt;Drosophila&lt;/em&gt; Synthetic Population Resource; MAF, minor allele frequency; QTL, quantitative trait loci; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1643/F5.medium.gif" width="440" height="169"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/6/1643/F5.medium.gif" width="440" height="169"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1643/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484858" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-41" class="first-child">(A) Density plot of the frequencies of SNPs in the DGRP and DSPR. Different colors denote SNPs unique to each population and shared in the two populations. (B) Histogram of the differences in the MAF for the set of SNPs shared between the DGRP and the DSPR. The black vertical line denotes zero; to the left, the MAF in the DSPR is higher than in the DGRP; and to the right, the MAF is higher in the DGRP. DGRP, <em>Drosophila</em> Genetic Reference Panel; DSRP, <em>Drosophila</em> Synthetic Population Resource; MAF, minor allele frequency; QTL, quantitative trait loci; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">However, for the set of shared SNPs in the DGRP and DSPR, it is useful to estimate the likelihood of replicating a locus between populations. For a subset of parameter combinations, we performed simulations to determine the power to identify a “hit” from the DGRP in the DSPR and vice versa (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Note that these simulations assume that the PVE of the locus is determined by the effect of the locus and its allele frequency and is not influenced by changes in the allele frequencies of other loci. For the same sample size, the power to validate between populations is similar, given that the SNP exists in both populations. The power to validate between populations for the full sample size in the DSPR is lower than the power to validate within populations (compare <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Despite the fact that it is equally likely for an allele to increase or decrease in frequency, power is already high for the full sample size. Thus, SNPs at increased frequency in the DSPR compared to the DGRP (increasing the PVE) do not increase power substantially but SNPs at decreased frequency in the DSPR compared to the DGRP (decreasing the PVE) do reduce power, lowering overall power. Overall, as is generally the case in mapping studies, QTL would be expected to validate between populations when the SNP is common (increasing the likelihood it exists in another population), the PVE is high, and/or the sample size in the validation population is high.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484900/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484900/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Power (% QTL identified) to validate QTL between populations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Conclusions</h3><p id="p-46">Here, we have confirmed the behavior of the Beavis effect in two major modern <em>Drosophila</em> mapping populations, the multiparent DSPR and the GWAS-based DGRP. We have shown that the expected overestimation of the PVE by QTL is similar in the two populations and is strongly determined by sample size. Some of the severity of the Beavis effect stems from the requirement of using a high significance threshold that results from performing mapping at thousands to millions of positions. At one extreme, the false positive rate can be tightly controlled giving high confidence that a mapped QTL is a true positive; however, the Beavis effect will be most severe in this circumstance. Using relaxed significance thresholds may decrease the severity of the Beavis effect but will also increase the likelihood that any given QTL is a false positive. Approaches that use the entire set of SNPs to estimate the variance explained by all markers (<em>e.g.</em>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Yang <em>et al.</em> 2010</a>) avoid both of these issues by not conditioning on specific loci as being significant. However, these methods are therefore unable to pinpoint specific loci as being potentially causative, instead describing the overall variance attributable to all genetic factors. Finally, we have shown that the power to validate a mapped QTL both within the same mapping population and between mapping populations is not necessarily high, even given the relaxed significance criteria that can be used for a focused scan. A lack of validation is generally the more likely scenario and should always be treated as an absence of evidence, not evidence of absence.</p><p id="p-47">The core issues that we have discussed here—the PVE by QTL, power to detect QTL, and the prospects for validation of QTL—are relevant to any mapping population. One potential limitation to our simulations is that they only consider a single causative locus for each simulated phenotype. Our results are expected to be generalizable to any given mapped QTL, but we are not able to address issues that arise when mapping multiple QTL in a given experiment, including estimating the overall PVE explained by all QTL and the potential bias associated with this estimate. Qualitatively, our results in the DSPR should provide a rough guide for other MPPs (<em>e.g.</em>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">McMullen <em>et al.</em> 2009</a>; <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Aylor <em>et al.</em> 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Huang <em>et al.</em> 2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Cubillos <em>et al.</em> 2013</a>) employing haplotype-based mapping. In addition, while power is quite different in the DSPR and DGRP, the bias in PVE in the two resources is quite similar for similar sample sizes. This result suggests that our results, and those found previously (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Utz and Melchinger 1994</a>; <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Beavis 1998</a>; <a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Xu 2003</a>; <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Broman and Sen 2009</a>), for the bias in PVE may apply very broadly across mapping designs as well, though this issue deserves a more thorough investigation of additional mapping panels and designs in various species.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">This work was supported by National Institutes of Health (NIH) grants F32 GM099382 and R01 GM117135 to E.G.K., and by NIH grants OD010974 and GM115562 to A.D.L.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-11"><p id="p-49">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041426/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041426/-/DC1</a>.</p></li><li class="fn" id="fn-12"><p id="p-50"><em>Communicating editor: K. W. Broman</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-01-03"><span class="received-label">Received </span>January 3, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-03-16"><span class="accepted-label">Accepted </span>March 16, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 King <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-7.6.1643.1"><div class="cit-metadata unstructured">Abdel-Azim, 2016 Gamal. gdmp: genomic data management. R package version 0.1.0. Available at: <a href="https://CRAN.R-project.org/package=gdmp">https://CRAN.R-project.org/package=gdmp</a>. Accessed: February 1, 2017.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.2" data-doi="10.1016/j.cell.2016.05.063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alonso-Blanco</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrade</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bemm</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergelson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">1,135 Genomes reveal the global pattern of polymorphism in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.05.063%26rft_id%253Dinfo%253Apmid%252F27293186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.05.063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27293186&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.3" data-doi="10.1038/79216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirschhorn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klannemark</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindgren</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vohl</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The common PPAR gamma Pro12Ala polymorphism is associated with decreased risk of type 2 diabetes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D76%26rft.epage%253D80%26rft.atitle%253DThe%2Bcommon%2BPPARgamma%2BPro12Ala%2Bpolymorphism%2Bis%2Bassociated%2Bwith%2Bdecreased%2Brisk%2Bof%2Btype%2B2%2Bdiabetes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F79216%26rft_id%253Dinfo%253Apmid%252F10973253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/79216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10973253&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089078000022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.4" data-doi="10.1101/gr.111310.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds-Mathes</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdugo</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic analysis of complex traits in the emerging collaborative cross.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1213</span>–<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111310.110%26rft_id%253Dinfo%253Apmid%252F21406540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.6.1643.5"><div class="cit-metadata unstructured">Beavis, W. D., 1994 The power and deceit of QTL experiments: lessons from comparative QTL studies, pp. 250–266 in <em>Proceedings of the Forty-Ninth Annual Corn and Sorghum Industry Research Conference</em>, American Seed Trade Association, Chicago, IL.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">QTL analyses: power, precision, and accuracy.</span> <abbr class="cit-jnl-abbrev">Mol. Dissection Complex Traits</abbr> <span class="cit-vol">1998</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BDissection%2BComplex%2BTraits%26rft.volume%253D1998%26rft.spage%253D145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.7" data-doi="10.1007/BF00215714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albertsen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fincher</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Quantitative trait loci for plant height in four maize populations and their associations with qualitative genetic loci.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D83%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00215714%26rft_id%253Dinfo%253Apmid%252F24202349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00215714&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24202349&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1643.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <em>A Guide to QTL Mapping with R/qtl</em>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.9" data-doi="10.1534/genetics.116.193987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brzyski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sobczyk</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Candes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bogdan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Controlling the rate of GWAS false discoveries.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.193987%26rft_id%253Dinfo%253Apmid%252F27784720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjA1LzEvNjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.10" data-doi="10.1534/genetics.105.052506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Relaxed significance criteria for linkage analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">2371</span>–<span class="cit-lpage">2381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.052506%26rft_id%253Dinfo%253Apmid%252F16783025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My80LzIzNzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.11" data-doi="10.1186/1471-2156-12-66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skol</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">QTLRel: an R package for genome-wide association studies in which relatedness is a concern.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DCheng%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.spage%253D66%26rft.epage%253D66%26rft.atitle%253DQTLRel%253A%2Ban%2BR%2Bpackage%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bin%2Bwhich%2Brelatedness%2Bis%2Ba%2Bconcern.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-12-66%26rft_id%253Dinfo%253Apmid%252F21794153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-12-66&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21794153&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strobel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trepanier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Diversity outbred mice at 21: maintaining allelic variation in the face of selection.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3893</span>–<span class="cit-lpage">3902</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D3893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical threshold values for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clifford</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCullagh</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The regress function.</span> <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">6</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D6%26rft.spage%253D6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.15" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the collaborative cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCollaborative%2BCross%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D389%26rft.epage%253D401%26rft.atitle%253DThe%2BGenome%2BArchitecture%2Bof%2Bthe%2BCollaborative%2BCross%2BMouse%2BGenetic%2BReference%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.16" data-doi="10.1534/genetics.113.155515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scovacricchi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution mapping of complex traits with a four-parent advanced intercross yeast population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155515%26rft_id%253Dinfo%253Apmid%252F24037264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExNDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1643.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <em>Introduction to Quantitative Genetics</em>. <span class="cit-publ-name">Pearson, Prentice Hall</span>, <span class="cit-publ-loc">Harlow</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.19" data-doi="10.1111/j.1365-313X.2005.02591.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thuillet</span>  <span class="cit-name-given-names">A.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Maize association population: a high-resolution platform for quantitative trait locus dissection.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1054</span>–<span class="cit-lpage">1064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DFlint-Garcia%26rft.auinit1%253DS.%2BA.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D1054%26rft.epage%253D1064%26rft.atitle%253DMaize%2Bassociation%2Bpopulation%253A%2Ba%2Bhigh-resolution%2Bplatform%2Bfor%2Bquantitative%2Btrait%2Blocus%2Bdissection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2005.02591.x%26rft_id%253Dinfo%253Apmid%252F16359397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2005.02591.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16359397&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233776900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.20" data-doi="10.1534/g3.114.013748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabalin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative trait locus mapping methods for diversity outbred mice.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1623%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.013748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.114.013748&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.21" data-doi="10.1086/324471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Göring</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terwilliger</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blangero</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Large upward bias in estimation of locus-specific effects from genomewide scans.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1357</span>–<span class="cit-lpage">1369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DG%25C3%25B6ring%26rft.auinit1%253DH.%2BH.%26rft.volume%253D69%26rft.issue%253D6%26rft.spage%253D1357%26rft.epage%253D1369%26rft.atitle%253DLarge%2Bupward%2Bbias%2Bin%2Bestimation%2Bof%2Blocus-specific%2Beffects%2Bfrom%2Bgenomewide%2Bscans.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F324471%26rft_id%253Dinfo%253Apmid%252F11593451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/324471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11593451&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172086900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.22" data-doi="10.1038/nrm1583"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirschhorn</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome-wide association studies for common diseases and complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHirschhorn%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D95%26rft.epage%253D108%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bfor%2Bcommon%2Bdiseases%2Band%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm1583%26rft_id%253Dinfo%253Apmid%252F15716906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm1583&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15716906&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226721300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.23" data-doi="10.1073/pnas.1213423109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbone</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anholt</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Epistasis dominates the genetic architecture of Drosophila quantitative traits.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">15553</span>–<span class="cit-lpage">15559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1213423109%26rft_id%253Dinfo%253Apmid%252F22949659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM5LzE1NTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.24" data-doi="10.1101/gr.171546.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massouras</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ràmia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural variation in genome architecture among 205 Drosophila melanogaster genetic reference panel lines.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.171546.113%26rft_id%253Dinfo%253Apmid%252F24714809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTE5MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.25" data-doi="10.1073/pnas.1100465108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulo</span>  <span class="cit-name-given-names">M.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Effgen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keizer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Analysis of natural allelic variation in Arabidopsis using a multiparent recombinant inbred line population.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">4488</span>–<span class="cit-lpage">4493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1100465108%26rft_id%253Dinfo%253Apmid%252F21368205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzExLzQ0ODgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.26" data-doi="10.1101/gr.134031.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merkes</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeil</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoofer</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Genetic dissection of a model complex trait using the Drosophila synthetic population resource.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1558</span>–<span class="cit-lpage">1566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.134031.111%26rft_id%253Dinfo%253Apmid%252F22496517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMTU1OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.27" data-doi="10.1534/genetics.112.138537"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Properties and power of the Drosophila synthetic population resource for the routine dissection of complex traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.138537%26rft_id%253Dinfo%253Apmid%252F22505626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvOTM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.28" data-doi="10.1534/genetics.114.161968"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kislukhin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walters</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Using Drosophila melanogaster to identify chemotherapy toxicity genes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKing%26rft.auinit1%253DE.%2BG.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D43%26rft.atitle%253DUsing%2BDrosophila%2Bmelanogaster%2BTo%2BIdentify%2BChemotherapy%2BToxicity%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161968%26rft_id%253Dinfo%253Apmid%252F25236447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk4LzEvMzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.29" data-doi="10.1371/journal.pgen.1004322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanderson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNeil</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Genetic dissection of the Drosophila melanogaster female head transcriptome reveals widespread allelic heterogeneity.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004322%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004322%26rft_id%253Dinfo%253Apmid%252F24810915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24810915&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.30" data-doi="10.1371/journal.pgen.1000551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kover</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trakalo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scarcelli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A multiparent advanced generation inter-cross to fine-map quantitative traits in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKover%26rft.auinit1%253DP.%2BX.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000551%26rft.epage%253De1000551%26rft.atitle%253DA%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2Bto%2Bfine-map%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000551%26rft_id%253Dinfo%253Apmid%252F19593375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593375&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic architecture of the maize kernel row number revealed by combining QTL mapping using a high-density genetic map and bulked segregant RNA sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">915</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.32" data-doi="10.1016/j.tig.2014.07.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">E. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dissecting complex traits using the Drosophila synthetic population resource.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">488</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D30%26rft.spage%253D488%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.07.009%26rft_id%253Dinfo%253Apmid%252F25175100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.07.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25175100&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1643.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <em>Genetics and Analysis of Quantitative Traits</em>, <span class="cit-publ-name">Sinauer</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.34" data-doi="10.1038/nature10811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Drosophila melanogaster genetic reference panel.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DThe%2BDrosophila%2Bmelanogaster%2BGenetic%2BReference%2BPanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10811%26rft_id%253Dinfo%253Apmid%252F22318601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22318601&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.35" data-doi="10.1016/j.cell.2010.03.032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClellan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">M.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic heterogeneity in human disease.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMcClellan%26rft.auinit1%253DJ.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D210%26rft.epage%253D217%26rft.atitle%253DGenetic%2Bheterogeneity%2Bin%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.03.032%26rft_id%253Dinfo%253Apmid%252F20403315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.03.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20403315&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276738400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.36" data-doi="10.1126/science.1174320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez Villeda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic properties of the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcMullen%26rft.auinit1%253DM.%2BD.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D737%26rft.epage%253D740%26rft.atitle%253DGenetic%2BProperties%2Bof%2Bthe%2BMaize%2BNested%2BAssociation%2BMapping%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174320%26rft_id%253Dinfo%253Apmid%252F19661427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.37" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Genome J.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BJ.%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.38" data-doi="10.1086/321272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Are rare variants responsible for susceptibility to complex diseases?</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">124</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D69%26rft.issue%253D1%26rft.spage%253D124%26rft.epage%253D137%26rft.atitle%253DAre%2Brare%2Bvariants%2Bresponsible%2Bfor%2Bsusceptibility%2Bto%2Bcomplex%2Bdiseases%253F%26rft_id%253Dinfo%253Adoi%252F10.1086%252F321272%26rft_id%253Dinfo%253Apmid%252F11404818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/321272&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11404818&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170108100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-other" id="cit-7.6.1643.39"><div class="cit-metadata"><cite><span class="cit-comment">R Core Team, 2016 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL <a href="https://www.R-project.org/">https://www.R-project.org/</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1643.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roff</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <em>Evolutionary Quantitative Genetics.</em> <span class="cit-publ-name">Chapman &amp; Hall</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.41" data-doi="10.1093/biomet/asr057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siegmund</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yakir</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">False discovery rate for scanning statistics.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">979</span>–<span class="cit-lpage">985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252Fasr057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvbWV0IjtzOjU6InJlc2lkIjtzOjg6Ijk4LzQvOTc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE2NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.42" data-doi="10.1002/(SICI)1098-2272(200002)18:2&lt;143::AID-GEPI4&gt;3.0.CO;2-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slager</span>  <span class="cit-name-given-names">S. l.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vieland</span>  <span class="cit-name-given-names">V. j.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Effect of allelic heterogeneity on the power of the transmission disequilibrium test.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DSlager%26rft.auinit1%253DS.%2BL.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D143%26rft.epage%253D156%26rft.atitle%253DEffect%2Bof%2Ballelic%2Bheterogeneity%2Bon%2Bthe%2Bpower%2Bof%2Bthe%2Btransmission%2Bdisequilibrium%2Btest.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291098-2272%2528200002%252918%253A2%253C143%253A%253AAID-GEPI4%253E3.0.CO%253B2-5%26rft_id%253Dinfo%253Apmid%252F10642427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1098-2272(200002)18:2&lt;143::AID-GEPI4&gt;3.0.CO;2-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10642427&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085224600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1643.43" data-doi="10.1371/journal.pgen.1003258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foran</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Properties and modeling of GWAS when complex disease risk is due to non-complementing, deleterious mutations in genes of large effect.</span> <span class="cit-source">PLoS Genet.</span> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003258%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003258%26rft_id%253Dinfo%253Apmid%252F23437004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23437004&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1643.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Comparison of different approaches to interval mapping of quantitative trait loci</span>, pp. <span class="cit-fpage">195</span>–<span class="cit-lpage">203</span> in <em>Biometrics in Plant Breeding : Applications of Molecular Markers</em>, edited by <span class="cit-ed"><span class="cit-name-surname">van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed cit-collab">European Association for Research on Plant Breeding</span>. <span class="cit-publ-name">CPRO-DLO</span>, <span class="cit-publ-loc">Wageningen, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Bias and sampling error of the estimated proportion of genotypic variance explained by quantitative trait loci determined from experimental data in maize using cross validation and validation with independent samples.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1839</span>–<span class="cit-lpage">1849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DUtz%26rft.auinit1%253DH.%2BF.%26rft.volume%253D154%26rft.issue%253D4%26rft.spage%253D1839%26rft.epage%253D1849%26rft.atitle%253DBias%2Band%2BSampling%2BError%2Bof%2Bthe%2BEstimated%2BProportion%2Bof%2BGenotypic%2BVariance%2BExplained%2Bby%2BQuantitative%2BTrait%2BLoci%2BDetermined%2BFrom%2BExperimental%2BData%2Bin%2BMaize%2BUsing%2BCross%2BValidation%2Band%2BValidation%2BWith%2BIndependent%2BSamples%26rft_id%253Dinfo%253Apmid%252F10866652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NC80LzE4MzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.46" data-doi="10.1093/bioinformatics/btv448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rainey</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">NAM: association studies in multiple populations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3862</span>–<span class="cit-lpage">3864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv448%26rft_id%253Dinfo%253Apmid%252F26243017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzEvMjMvMzg2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xNjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Theoretical basis of the Beavis effect.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2259</span>–<span class="cit-lpage">2268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D165%26rft.issue%253D4%26rft.spage%253D2259%26rft.epage%253D2268%26rft.atitle%253DTheoretical%2BBasis%2Bof%2Bthe%2BBeavis%2BEffect%26rft_id%253Dinfo%253Apmid%252F14704201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS80LzIyNTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTY0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.48" data-doi="10.1038/ng.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benyamin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEvoy</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henders</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common SNPs explain a large proportion of the heritability for human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DCommon%2BSNPs%2Bexplain%2Ba%2Blarge%2Bproportion%2Bof%2Bthe%2Bheritability%2Bfor%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.608%26rft_id%253Dinfo%253Apmid%252F20562875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562875&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279242400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1643.49" data-doi="10.1038/ng.2876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Advantages and pitfalls in the application of mixed-model association methods.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2876%26rft_id%253Dinfo%253Apmid%252F24473328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24473328&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/6/1643.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1631" title="Loci Contributing to Boric Acid Toxicity in Two Reference Populations of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1653" title="Allelic Variation in the Toll-Like Receptor Adaptor Protein Ticam2 Contributes to SARS-Coronavirus Pathogenesis in Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/6"><span>Volume 7 Issue 6, June 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/6.cover-source.jpg?itok=5_J-OKHc" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1643?ijkey=90eb2a5b82398a3db6dadbfec690b8901e6165ee&amp;keytype2=tf_ipsecsha" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5QLchoLZMywAwkna7lPR7dVJUi2Qw8wLpPfBV8LBHYo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/6/1643" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/6/1643" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/6/1643" class="active" data-icon-position="" data-hide-link-title="0">The Beavis Effect in Next-Generation Mapping Panels in Drosophila melanogaster</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484760" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-IDITXFJMhZTooJ6J66Z0HWnSPj92nX4pd7_kACjCkbM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1643.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/6/1643" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-J5AuSvBxZDdBdG2ogvoMlCtcc56Fdi7xOBVCB7vkwrE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/6/1643" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1643.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484760" id="node484760--2" data-pisa="ggg;7/6/1643" data-pisa-master="ggg;g3.117.041426" data-apath="/ggg/7/6/1643.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Beavis Effect in Next-Generation Mapping Panels in <em>Drosophila melanogaster</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9393-4720" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elizabeth G. King</span> and <span class="highwire-citation-author" data-delta="1">Anthony D. Long</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1643-1652;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041426 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484760--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elizabeth G. King </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Biological Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BG.%2BKing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=King%20EG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BG.%2BKing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9393-4720" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elizabeth G. King</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kingeg@missouri.edu" class="" data-icon-position="" data-hide-link-title="0">kingeg@missouri.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anthony D. Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of California Irvine, California 92697</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BD.%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnthony%2BD.%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/6/1643" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/6/1643&amp;title=The%20Beavis%20Effect%20in%20Next-Generation%20Mapping%20Panels%20in%20Drosophila%20melanogaster" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/6/1643&amp;title=The%20Beavis%20Effect%20in%20Next-Generation%20Mapping%20Panels%20in%20Drosophila%20melanogaster" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/6/1643&amp;title=The%20Beavis%20Effect%20in%20Next-Generation%20Mapping%20Panels%20in%20Drosophila%20melanogaster" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/6/1643&amp;text=The%20Beavis%20Effect%20in%20Next-Generation%20Mapping%20Panels%20in%20Drosophila%20melanogaster" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/6/1643&amp;title=The%20Beavis%20Effect%20in%20Next-Generation%20Mapping%20Panels%20in%20Drosophila%20melanogaster" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/6/1643&amp;t=The%20Beavis%20Effect%20in%20Next-Generation%20Mapping%20Panels%20in%20Drosophila%20melanogaster" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/6/1643&amp;title=The%20Beavis%20Effect%20in%20Next-Generation%20Mapping%20Panels%20in%20Drosophila%20melanogaster" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/6/1643&amp;title=The%20Beavis%20Effect%20in%20Next-Generation%20Mapping%20Panels%20in%20Drosophila%20melanogaster" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484760/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484760" id="node484760--3" data-pisa="ggg;7/6/1643" data-pisa-master="ggg;g3.117.041426" data-apath="/ggg/7/6/1643.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Beavis Effect in Next-Generation Mapping Panels in <em>Drosophila melanogaster</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9393-4720" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elizabeth G. King</span> and <span class="highwire-citation-author" data-delta="1">Anthony D. Long</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1643-1652;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041426 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484760--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Elizabeth G. King </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Biological Sciences, University of Missouri, Columbia, Missouri 65211</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BG.%2BKing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=King%20EG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BG.%2BKing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9393-4720" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elizabeth G. King</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kingeg@missouri.edu" class="" data-icon-position="" data-hide-link-title="0">kingeg@missouri.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anthony D. Long </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, University of California Irvine, California 92697</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anthony%2BD.%2BLong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Long%20AD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnthony%2BD.%2BLong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1643&amp;rft.epage=1652&amp;rft.atitle=The%20Beavis%20Effect%20in%20Next-Generation%20Mapping%20Panels%20in%20Drosophila%20melanogaster&amp;rft.volume=7&amp;rft.issue=6&amp;rft.date=2017-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=King%2C+Elizabeth+G.&amp;rft.au=Long%2C+Anthony+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484760/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484760/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484760/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484760/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484760/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484760/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484760/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484760/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484760/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484760/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484760/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484760/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/6/1643" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/6/1643.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/6/1643.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/6/1643.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/6/1643.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
