<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/6/1995" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/6/1995.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/6/1995.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/6/1995.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/6/1995/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/6/1995.atom" />
<meta name="HW.pisa" content="ggg;7/6/1995" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic-Enabled Prediction in Maize Using Kernel Models with Genotype × Environment Interaction" />
<meta name="DC.Identifier" content="10.1534/g3.117.042341" />
<meta name="DC.Date" content="2017-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Bandeira e Sousa et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Multi-environment trials are routinely conducted in plant breeding to select candidates for the next selection cycle. In this study, we compare the prediction accuracy of four developed genomic-enabled prediction models: (1) single-environment, main genotypic effect model (SM); (2) multi-environment, main genotypic effects model (MM); (3) multi-environment, single variance G×E deviation model (MDs); and (4) multi-environment, environment-specific variance G×E deviation model (MDe). Each of these four models were fitted using two kernel methods: a linear kernel Genomic Best Linear Unbiased Predictor, GBLUP (GB), and a nonlinear kernel Gaussian kernel (GK). The eight model-method combinations were applied to two extensive Brazilian maize data sets (HEL and USP data sets), having different numbers of maize hybrids evaluated in different environments for grain yield (GY), plant height (PH), and ear height (EH). Results show that the MDe and the MDs models fitted with the Gaussian kernel (MDe-GK, and MDs-GK) had the highest prediction accuracy. For GY in the HEL data set, the increase in prediction accuracy of SM-GK over SM-GB ranged from 9 to 32%. For the MM, MDs, and MDe models, the increase in prediction accuracy of GK over GB ranged from 9 to 49%. For GY in the USP data set, the increase in prediction accuracy of SM-GK over SM-GB ranged from 0 to 7%. For the MM, MDs, and MDe models, the increase in prediction accuracy of GK over GB ranged from 34 to 70%. For traits PH and EH, gains in prediction accuracy of models with GK compared to models with GB were smaller than those achieved in GY. Also, these gains in prediction accuracy decreased when a more difficult prediction problem was studied." />
<meta name="DC.Contributor" content="Massaine Bandeira e Sousa" />
<meta name="DC.Contributor" content="Jaime Cuevas" />
<meta name="DC.Contributor" content="Evellyn Giselly de Oliveira Couto" />
<meta name="DC.Contributor" content="Paulino Pérez-Rodríguez" />
<meta name="DC.Contributor" content="Diego Jarquín" />
<meta name="DC.Contributor" content="Roberto Fritsche-Neto" />
<meta name="DC.Contributor" content="Juan Burgueño" />
<meta name="DC.Contributor" content="Jose Crossa" />
<meta name="article:published_time" content="2017-06-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic-Enabled Prediction in Maize Using Kernel Models with Genotype × Environment Interaction" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Multi-environment trials are routinely conducted in plant breeding to select candidates for the next selection cycle. In this study, we compare the prediction accuracy of four developed genomic-enabled prediction models: (1) single-environment, main genotypic effect model (SM); (2) multi-environment, main genotypic effects model (MM); (3) multi-environment, single variance G×E deviation model (MDs); and (4) multi-environment, environment-specific variance G×E deviation model (MDe). Each of these four models were fitted using two kernel methods: a linear kernel Genomic Best Linear Unbiased Predictor, GBLUP (GB), and a nonlinear kernel Gaussian kernel (GK). The eight model-method combinations were applied to two extensive Brazilian maize data sets (HEL and USP data sets), having different numbers of maize hybrids evaluated in different environments for grain yield (GY), plant height (PH), and ear height (EH). Results show that the MDe and the MDs models fitted with the Gaussian kernel (MDe-GK, and MDs-GK) had the highest prediction accuracy. For GY in the HEL data set, the increase in prediction accuracy of SM-GK over SM-GB ranged from 9 to 32%. For the MM, MDs, and MDe models, the increase in prediction accuracy of GK over GB ranged from 9 to 49%. For GY in the USP data set, the increase in prediction accuracy of SM-GK over SM-GB ranged from 0 to 7%. For the MM, MDs, and MDe models, the increase in prediction accuracy of GK over GB ranged from 34 to 70%. For traits PH and EH, gains in prediction accuracy of models with GK compared to models with GB were smaller than those achieved in GY. Also, these gains in prediction accuracy decreased when a more difficult prediction problem was studied.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/06/01" />
<meta name="citation_mjid" content="ggg;7/6/1995" />
<meta name="citation_id" content="7/6/1995" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/6/1995" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/6/1995.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/6/1995.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/6/1995.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.042341" />
<meta name="citation_pmid" content="28455415" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="20" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1995" />
<meta name="citation_lastpage" content="2014" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Massaine Bandeira e Sousa" />
<meta name="citation_author_institution" content="Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, 13418-900 Brazil" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7887-9543" />
<meta name="citation_author" content="Jaime Cuevas" />
<meta name="citation_author_institution" content="University of Quintana Roo, 77019 Chetumal, Mexico" />
<meta name="citation_author" content="Evellyn Giselly de Oliveira Couto" />
<meta name="citation_author_institution" content="Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, 13418-900 Brazil" />
<meta name="citation_author" content="Paulino Pérez-Rodríguez" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, CP 56230 Montecillos, 56230 D.F., México" />
<meta name="citation_author" content="Diego Jarquín" />
<meta name="citation_author_institution" content="Agronomy and Horticulture Department, University of Nebraska, Lincoln, Nebraska 68583-0915" />
<meta name="citation_author" content="Roberto Fritsche-Neto" />
<meta name="citation_author_institution" content="Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, 13418-900 Brazil" />
<meta name="citation_author" content="Juan Burgueño" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 D.F., México" />
<meta name="citation_author" content="Jose Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 D.F., México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Akdemir;citation_author=J.-L. Jannink;citation_title=Locally epistatic genomic relationship matrices for genomic association and prediction.;citation_volume=199;citation_year=2015;citation_pmid=25614606;citation_doi=10.1534/genetics.114.173658" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=Y. Beyene;citation_author=K. Semagn;citation_author=S. Mugo;citation_author=A. Tarekegne;citation_author=R. Babu;citation_title=Genetic gains in grain yield through genomic selection in eight bi-parental maize populations under drought stress.;citation_pages=154-163;citation_volume=55;citation_year=2015;citation_issue=1;citation_doi=10.2135/cropsci2014.07.0460" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.;citation_pages=210-223;citation_volume=84;citation_year=2008;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_issue=2;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=J. Crop Improv.;citation_author=J. Crossa;citation_author=P. Pérez;citation_author=G. de los Campos;citation_author=G. Mahuku;citation_author=S. Dreisigacker;citation_title=Genomic selection and prediction in plant breeding.;citation_pages=239-261;citation_volume=25;citation_year=2011;citation_issue=3;citation_doi=10.1080/15427528.2011.558767" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. Crossa;citation_author=D. Jarquín;citation_author=J. Franco;citation_author=P. Pérez-Rodríguez;citation_author=J. Burgueño;citation_title=Genomic prediction of gene bank wheat landraces.;citation_pages=1819-1834;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=M. Maccaferri;citation_author=R. Tuberosa;citation_author=J. Burgueño;citation_title=Extending the marker × environment interaction model for genomic-enabled prediction and genome-wide association analysis in durum wheat.;citation_pages=2193-2209;citation_volume=56;citation_year=2016;citation_issue=5" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. Cuevas;citation_author=J. Crossa;citation_author=O. Montesinos-Lopez;citation_author=J. Burgueno;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_title=Bayesian genomic prediction with genotype × environment interaction kernel models.;citation_pages=41-53;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Cuevas;citation_author=J. Crossa;citation_author=V. Soberanis;citation_author=S. Pérez-Elizalde;citation_author=P. Pérez-Rodríguez;citation_title=Genomic prediction of genotype × environment interaction kernel regression models.;citation_pages=1-20;citation_volume=9;citation_year=2016;citation_issue=3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Gianola;citation_author=J. B. van Kaam;citation_title=Reproducing Kernel Hilbert Spaces Regression Methods for Genomic Assisted Prediction of Quantitative Traits;citation_pages=2289-2303;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430950;citation_doi=10.1534/genetics.107.084285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=R. L. Fernando;citation_author=A. Stella;citation_title=Genomic-assisted prediction of genetic value with semiparametric procedures.;citation_volume=173;citation_year=2006;citation_pmid=16648593;citation_doi=10.1534/genetics.105.049510" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=D. Gianola;citation_author=K. A. Weigel;citation_author=N. Krämer;citation_author=A. Stella;citation_author=C. C. Schön;citation_title=Enhancing genome-enabled prediction by bagging genomic BLUP.;citation_pages=e91693;citation_volume=9;citation_year=2014;citation_issue=4;citation_pmid=24722227;citation_doi=10.1371/journal.pone.0091693" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146;citation_volume=52;citation_year=2012;citation_issue=1;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=N. Heslot;citation_author=D. Akdemir;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.;citation_pages=463-480;citation_volume=127;citation_year=2014;citation_issue=2;citation_pmid=24264761;citation_doi=10.1007/s00122-013-2231-5" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. Du Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_issue=3;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jiang;citation_author=J. C. Reif;citation_title=Modeling epistasis in genomic selection.;citation_volume=201;citation_year=2015;citation_pmid=26219298;citation_doi=10.1534/genetics.115.177907" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=M. López-Cruz;citation_author=J. Crossa;citation_author=D. Bonnett;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_title=Increased prediction accuracy in wheat breeding trials using a marker × environment interaction genomic selection model.;citation_pages=569-82;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Massman;citation_author=H. J. G. Jung;citation_author=R. Bernardo;citation_title=Genome-wide selection vs. marker-assisted recurrent selection to improve grain yield and stover-quality traits for cellulosic ethanol in maize.;citation_pages=58-66;citation_volume=53;citation_year=2013;citation_issue=1;citation_doi=10.2135/cropsci2012.02.0112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=G. Morota;citation_author=D. Gianola;citation_title=Kernel-based whole-genome prediction of complex traits: a review.;citation_pages=1-13;citation_volume=5;citation_year=2014;citation_pmid=24567736;citation_doi=10.3389/fgene.2014.00001" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=G. Morota;citation_author=M. Koyama;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=D. Gianola;citation_title=Predicting complex traits using a diffusion kernel on genetic markers with an application to dairy cattle and wheat data.;citation_pages=17;citation_volume=45;citation_year=2013;citation_issue=1;citation_pmid=23763755;citation_doi=10.1186/1297-9686-45-17" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=S. Pérez-Elizalde;citation_author=J. Cuevas;citation_author=P. Pérez-Rodríguez;citation_author=J. Crossa;citation_title=Selection of the bandwidth parameter in a Bayesian kernel regression model for genomic-enabled prediction.;citation_pages=512-532;citation_volume=20;citation_year=2015;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. M. González-Camacho;citation_author=J. Crossa;citation_author=Y. Manès;citation_author=S. Dreisigacker;citation_title=Comparison between linear and non-parametric regression models for genome-enabled prediction in wheat.;citation_pages=1595-605;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. M. Gonzalez-Camacho;citation_author=J. Crossa;citation_author=Y. Manes;citation_author=S. Dreisigacker;citation_title=Comparison between linear and non-parametric regression models for genome-enabled prediction in wheat.;citation_pages=1595-1605;citation_volume=2;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=P. Pérez Rodríguez;citation_author=J. Crossa;citation_author=K. Bondalapati;citation_author=G. De Meyer;citation_author=F. Pita;citation_title=A pedigree reaction norm model for prediction of cotton (Gossypium sp.) yield in multi-environment trials.;citation_pages=1143-1151;citation_volume=55;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=C. Saint Pierre;citation_author=J. Burgueño;citation_author=J. Crossa;citation_author=G. Fuentes Dávila;citation_author=P. Figueroa López;citation_title=Genomic prediction models for grain yield of spring bread wheat in diverse agro-ecological zones.;citation_pages=27312;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. Sukumaran;citation_author=J. Crossa;citation_author=D. Jarquín;citation_author=M. Lopes;citation_author=M. P. Reynolds;citation_title=Genomic prediction with pedigree and genotype × environment interaction in spring wheat grown in South and West Asia, North Africa, and Mexico.;citation_volume=7;citation_year=2017;citation_pmid=27903632;citation_doi=10.1534/g3.116.036251" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Unterseer;citation_author=E. Bauer;citation_author=G. Haberer;citation_author=M. Seidel;citation_author=C. Knaak;citation_title=A powerful tool for genome analysis in maize: development and evaluation of the high density 600 k SNP genotyping array.;citation_pages=823;citation_volume=15;citation_year=2014;citation_issue=1;citation_pmid=25266061;citation_doi=10.1186/1471-2164-15-823" />
<meta name="citation_reference" content="citation_title=Interbull Bulletin;citation_author=P. M. VanRaden;citation_pages=33-36;citation_volume=37;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. V. Wimmer;citation_author=T. Albrecht;citation_author=H. J. Auinger;citation_author=C. C. Schön;citation_title=synbreed: framework for the analysis of genomic prediction data using R.;citation_pages=2086-2087;citation_volume=28;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=X. Zhang;citation_author=P. Pérez-Rodríguez;citation_author=K. Semagn;citation_author=Y. Beyene;citation_author=R. Babu;citation_title=Genomic prediction in biparental tropical maize populations in water-stressed and well-watered environments using low-density and GBS SNPs.;citation_pages=291-299;citation_volume=114;citation_year=2015;citation_issue=3;citation_pmid=25407079;citation_doi=10.1038/hdy.2014.99" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic-Enabled Prediction in Maize Using Kernel Models with Genotype × Environment Interaction" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="twitter:description" content="Multi-environment trials are routinely conducted in plant breeding to select candidates for the next selection cycle. In this study, we compare the prediction accuracy of four developed genomic-enabled prediction models: (1) single-environment, main genotypic effect model (SM); (2) multi-environment, main genotypic effects model (MM); (3) multi-environment, single variance G×E deviation model (MDs); and (4) multi-environment, environment-specific variance G×E deviation model (MDe). Each of these four models were fitted using two kernel methods: a linear kernel Genomic Best Linear Unbiased Predictor, GBLUP (GB), and a nonlinear kernel Gaussian kernel (GK). The eight model-method combinations were applied to two extensive Brazilian maize data sets (HEL and USP data sets), having different numbers of maize hybrids evaluated in different environments for grain yield (GY), plant height (PH), and ear height (EH). Results show that the MDe and the MDs models fitted with the Gaussian kernel (MDe-GK, and MDs-GK) had the highest prediction accuracy. For GY in the HEL data set, the increase in prediction accuracy of SM-GK over SM-GB ranged from 9 to 32%. For the MM, MDs, and MDe models, the increase in prediction accuracy of GK over GB ranged from 9 to 49%. For GY in the USP data set, the increase in prediction accuracy of SM-GK over SM-GB ranged from 0 to 7%. For the MM, MDs, and MDe models, the increase in prediction accuracy of GK over GB ranged from 34 to 70%. For traits PH and EH, gains in prediction accuracy of models with GK compared to models with GB were smaller than those achieved in GY. Also, these gains in prediction accuracy decreased when a more difficult prediction problem was studied." />
<meta name="og-title" property="og:title" content="Genomic-Enabled Prediction in Maize Using Kernel Models with Genotype × Environment Interaction" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/6/1995" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Multi-environment trials are routinely conducted in plant breeding to select candidates for the next selection cycle. In this study, we compare the prediction accuracy of four developed genomic-enabled prediction models: (1) single-environment, main genotypic effect model (SM); (2) multi-environment, main genotypic effects model (MM); (3) multi-environment, single variance G×E deviation model (MDs); and (4) multi-environment, environment-specific variance G×E deviation model (MDe). Each of these four models were fitted using two kernel methods: a linear kernel Genomic Best Linear Unbiased Predictor, GBLUP (GB), and a nonlinear kernel Gaussian kernel (GK). The eight model-method combinations were applied to two extensive Brazilian maize data sets (HEL and USP data sets), having different numbers of maize hybrids evaluated in different environments for grain yield (GY), plant height (PH), and ear height (EH). Results show that the MDe and the MDs models fitted with the Gaussian kernel (MDe-GK, and MDs-GK) had the highest prediction accuracy. For GY in the HEL data set, the increase in prediction accuracy of SM-GK over SM-GB ranged from 9 to 32%. For the MM, MDs, and MDe models, the increase in prediction accuracy of GK over GB ranged from 9 to 49%. For GY in the USP data set, the increase in prediction accuracy of SM-GK over SM-GB ranged from 0 to 7%. For the MM, MDs, and MDe models, the increase in prediction accuracy of GK over GB ranged from 34 to 70%. For traits PH and EH, gains in prediction accuracy of models with GK compared to models with GB were smaller than those achieved in GY. Also, these gains in prediction accuracy decreased when a more difficult prediction problem was studied." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
    <title>Genomic-Enabled Prediction in Maize Using Kernel Models with Genotype × Environment Interaction | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__HLkFhTtFYKr9UM35ZkeLEJf0AA-RJsk2jDIDlTr7P6E__db-Yi0lrH1NHADEOMB-20Dvr8Qs6F9pwOIMQF9BHr1w__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__qoHMxai33VxgVvIHOVQupamFpg1jn-ylrf2gIvUolBs__Fp-C6VyxAewt6D0gYbv6aOnifLuGF1NoN4YO_dqmh1I__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"95TNufVJPYyWaKdQWwnYTLKgAbDyRdAOrx4lgFZ9eG4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"misc\/ui\/jquery.ui.accordion.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"ggg;7\/6\/1995"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"ggg;7\/6\/1995"},{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1995"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1995"}],"nid":"484776","apath":"\/ggg\/7\/6\/1995.atom","pisa":"ggg;7\/6\/1995","ac":{"\/ggg\/7\/6\/1995.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/6\/1995.atom","jcode":"ggg"}},"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/6\/1995":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xkuN3Syrom0gBHJEBprcsxs3zXFPUStvxyuZOSMCCccaeHqbH\\\\\\\/g4SdRnUuncmA\\\\\\\/aIkSW64GuTDyxvPtHpvZfcAIpV61M1ylsRF5rxatbCsi4yI839sVA0x8K7JwtW2px+kj5\\\\\\\/j2mPKZlNmnGSPnJYodWVODKxcpx4XA0agNKV9JFcW+oM+cgzyIJK56GILyNPqg4C\\\\\\\/6kUc\\\\\\\/0nyW+yeRDTVJMEQCH5qsg7LcgPXs\\\\\\\/mw\\\\\\\/hjozgenzIeU+m+ZUV1y9bm1ETrE2b6+UFkA+BIPoRcqYylKzUBF6IiKKE16agOpX\\\\\\\/DBUyD0jD5tbLxwNCgVAtySf3L68wJUt+JH8zwbXRVVR+RiKCZS7ysxMNiG7\\\\\\\/nzVqSz\\\\\\\/pgTcvlHRSIKUzNf8nR7l5ESVaNNdqlCVfLiaHcrafa+noRX23kTSTmPawc+RqSR8KRQmLKlxHNKMwxW1emFZdoKuYD1\\\\\\\/aXTst1jfoL9RjImiw7Bog33f7ZgMj6FoKR9XeYmVltH+Q56FUEGlGakflx1RY8mekMco0WZyXU0fVUh3l\\\\\\\/4Pt0ViLh1i8EciHvLGIcNJXFX12cL7\\\\\\\/xLnMRpN9lhtMkRqB7wL8XfDwjpP2R2d1zEzv4oX+k9P3JdEvTpDLShQ1qLhLr8fP0KQRrbS1tc4S8+JAF58BBiSdsd8PxhmjH7FE+vcUSr11zG+6oRNZmz\\\\\\\/BsLyk8b0Xei5NpkfWqzT1UN2s8dHAWxdZZjEPRYExVXvUt4Hj4E25g76TfxSRy8Uo7ovyBfUvJXf2ccM9feBgsyjI7uDYsMDO\\\\\\\/Bo\\\\\\\/XdPaJTJwnIcuWIjDH8fSEcrqUOBADxQKKoKDc+XbD6sjMjVBDaxGNBg2ugemvr4Ghm5b9tfZN49fEpMPo5y1hc7DIzg+6sqimWeyRtmnLDUj9s\\\\\\\/lfCM\\\\\\\/TmVfay1r4Ad0tKIUTQq5WT27ALR\\\\\\\/zn5uK6v9wzmdM21PIIUuXFyype9cb3gfxhHHB4m+hi\\\\\\\/ZHLhxlK7ULxuw55t0tvY\\\\\\\/ET1YutTDUpre2UqOJQxxb7InaXLxrHp8W1O\\\\\\\/nvCbRlg0F2zuNYVwYzzn8air5Ix9glNK0dHgLBXhf1swMjg759jy9HXi+GF57OYeC9VaR2SNgiB2wJ0GTSFCiZH6c28NpkPwHpBJ\\\\\\\/p0KwMx5L1omJKg6PK\\\\\\\/8yb2PGX89vn1NFjfN2oQtVWxK4vibNZLRUcavWAxcigfHPKQ\\\\\\\/defO+ATJ\\\\\\\/LrI6NSY8r7F1bofhIUQ4FCjduk1loGqNdRoSmgVPbLgHQuUCvgnkyKrb2+Pj4\\\\\\\/UW\\\\\\\/fZ\\\u0022,\\\u0022iv\\\u0022:\\\u00220Logg7IbHA2bYrTAbECPWg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002278a64079cc6242becaa26d9241aae1d5a0cad4786bbb60ceb12e7ba389021a8f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229gB3ZbSdWxRPlGuol5mb55KUIBBMV6IbAKTllnvvMHWGtRv\\\\\\\/LM1BADKEy9C7MKrioUgpr9QYvFc99hgIBUdywYHgPqm\\\\\\\/engWNZJ3BY0QAkKFdxvBR5FXpbXVIVM9OieCc5eclqEXo9mRiCsOkkFF535geyGjbCJb4VlXWD6wjEot\\\\\\\/v4P3SQirIjKl5nYgclF4tyCRUsz9Q7HPuLnaKU2UPA8GL8CSjKsIc8PfZu2BYkCYZIwL1ZF84DgsUb4oYUnFl+ESpdEMq2dbPSL6u9hclWK8ffeobnRRgrUoO334jHaBr+Dj4TTc+vNxRDRlSIXW3kWxVJTu71ZOutAYW2XYEULBkQe3iql+bh4bPWCuiwgaXniEfZbZgECkJCMT+4yeP2dTvd44ILD9l6CfCjV9saJLfgkt9EDHlrRAYUMXXbOgr0yKSDKz+734A\\\\\\\/bhSjm3j3VxZzZsbM4auI0Eh2HVYK18mMo7tru2ifCPpccyUTaMdRFFFi118VcvjueQCfg3b0qQuLMT1Q6ZLNpgYCfeY\\\\\\\/fFCNsehTILrnXkQq1tsEygIiBdgwbMdocFu9esIva7tRcsFczeLVmUO57IZ7BFrd3R\\\\\\\/ZYzDNLpi\\\\\\\/YNktVVUzidVBwk7eAaliCFRDBGH05kZNicw8uyeBnTFN2QY6rcFmtTqkPWeJ9YHhSlNojDHEzudzoBvcKrJb2gX5MogvZp73KF0za2jj\\\\\\\/WDYQuju6VoGL9YdLMmlP3KN98Of2rYcMedRCAE2iGVAhhtdXDtYZHlJiM0UaHkTngD3aRswpBeUc1t+efnX9+\\\\\\\/0louKg3G9lfr54WTtPyMKpKut86PFKLiR23dKri0DRRU8F4NRV1U7Ym0aneB3Vka6O0aikIu8Q3rFbEWf1dAJ68M\\\\\\\/Iut2\\\\\\\/WgWhcIOsXLXjNSy44IZXhsto6jTr6e9GFNaZwNId3+MziP3jq1ICef00EuIjqrh5R\\\\\\\/PAsa+wxxsfRgayGdHWGUm21mzQIr9Sb+1kQNkq6G6OsOi5EGntgXT4oiYtQ12zv6\\\\\\\/EvSsXHYrORuJ\\\\\\\/7AmoPt1ZUYyYhCQ6DSrYZLvu7wRGuhZIZY6mjcdcTBTfJGda7o96q4LGhqiEQQyFeNeYi5XJDlDz5Ocxn\\\\\\\/BaU\\\\\\\/NaZetCPhtmMSpYXj+pKkb5ACDB39RpultOO8KV7UTFMKcyQiCDtRmOzvQFG9a0UJiIce2nYYqX73w6F6HoZRANsyIo5bdxIbnEXTM+h5soN6eV4LXFXEhrDiZcA2W4Pu1jR5vxZ9vKGQDB6JOPiKu9WYYfG2s8a1Qwk+7IEuxLaWbEhPLYoKTxjNo6qR\\\\\\\/xR8n\\\\\\\/h\\\\\\\/O\\\\\\\/IqfMzYb+zFHNerZT+5CM7yB+LgpRLX67efJrRhI+dtOBCkawlJ2HVlI8AFiD\\\\\\\/\\\\\\\/2UYzf49UHQTExF1EPig0wZnipydrOKqvxx2Sa\\\\\\\/dSqxJsL\\\\\\\/7oHe7qQfpjiGG+jzikPJ58DWJEquioN6i4ieoVcSO4FBh1TigoIsYunlCGwoc5342+S2ZevjdxL5kk5HYeBbEFH7CK3EAt9r4GRbR+BIhGQtIThI1+xj8Wo78RwYZwheFAVYzwISnrvpoQW9J6PboUb4mtCa0wEY92Uan6EXrNCvNiDULwygegtlET\\\\\\\/F07aofrTy\\\\\\\/jFI4irN9ZxTfsggImHnrcpWUQ63KaxdfsbimMRCmV+w7xKT1aGkp6YszzuPctW6kUWiope4NCk\\\\\\\/cJTkhVrXJ2KKkiFv2WTfyc8dsN\\\\\\\/NaWxE4nrUNYaTDV6ygBglP3EaaZM7NYJRYDxIogYZPJNoPvPgtZF9lbYrs+JtI2cCLsDbN2zOkL301Zw28YvaKLDvN\\\\\\\/IHiNdD8MGlaYwgxumBJdWv7SkEfyOYaextG\\\\\\\/tOSEdiyGLHAw+sI9dKTVLcRbJksktQJsNHIOcAdulk78NSRhj2EmYKHZeoAnFXD3UR\\\\\\\/wHzRonBNmX15uigspvXbp4aZpdgGsxjkK1AExVwdBWh\\\\\\\/XX80M0sS5xWKZNKrPrkXABACV8bCqy4vzXsD3PXK7sXVvjps1CtNNCU5OgEFJoSSm5Y2FlTY7ZIS8myChHIV\\\\\\\/0aVhpnxCrNmAnC4RawDS5bP+KYDrCLE\\\\\\\/zfjM4qlstZSmxzYXMGynpXN0iF32fmj8+p6jUgxcrfFNJHH4fHFJSsXIpux\\\\\\\/9vtXl8xUs+2qJ46Ofd4zqu3FWyIkXFO0uqQQelj2+xvzLipmp3Wv9Xyp0TdX4HMvXcgqp9NU8MYuNMwKXi2THOSwIuzNzQnb6QRirHDC0hdmfOCk8FTwZZdJFNFwBY4VachmhAQEDG4ESBwKQkOu1JB77Orlft6x7PyYsMKF0FCfWVY5QHAQIPmyJ\\\\\\\/6Rfn0UoYTfiMhUpzaIu5x6zPXzQUW41a2Dy+Um\\\\\\\/AEkMtJ96yYOwgG21mopEtVruhjHOukSsQH4Sd2meOf99qIwYPDYpMdISK613+xOcTswKboW2LLHBJg9Tt10ILmZv3EBtgyxzZb9yhT96Bqp94tJXZy4Km0kQe4wTr3zhiMXqJ75wUD7QhS3GA0zMjHnzuGqVg8\\\\\\\/YxTFrhtKImyZ9HFbfHF5WyTkoOmdP3Xs7rs0fxXHgD5DNBhhq4oAmWRZJlwzlp83yiK\\\\\\\/uTJocijx5gwf3DxSn+yBjSWHdEdRC2x\\\\\\\/lQMBTqRJ0Duyg5dppcP1pPrDtUJj5oQ9sgiu\\\\\\\/2qOAF3s6mmxU\\\\\\\/hScpTkGeoA+q\\\\\\\/VbeJXQdWKeO8llciTEIjqTvBSv\\\\\\\/u6ROJ9LJ\\\\\\\/iAGJCN+fL3cv3GEe68dBSYOKJC46TM18Q37JNDN5vDDcne8Z0OD6jNMt6L0aFDWiolK5xHzZXMujjYEBe8ocyB008hWTVp2kKRJG5JE0Vd5GTqTYVi04JJOpcaVMQSVSZuokihTuXGVbrAb2h39LQkA86B\\\\\\\/NdMb9ChTBeAUCSWxUTVqkPMtOqKzntd2YhnYlkTdIgZg8Y4uRVy4wSYO7OCw1I+ZbgXYd+0LhUl0DAo+57H9WvVS9RdCr7wWW4DEGD8v+HTUztvzPSzJjPQ6qkxxrvyf3VGsJ3fBADe73iliYtpA1oyD\\\\\\\/UzMCCoavtgxW+lRs3P9yxL4UAp3C60BJ5Lq+Lk2vdgb2yhUL5CAD5dn6ku9Jt+MHT9ltvPq\\\\\\\/53Sva\\\\\\\/11KUyp9VrXPQG32Y+9OLu4cVGJVACy84zzA3rytQulJcf3bIw6eikJ8glRDkXsvLHBUnv1kFOL3LQADriVINpVMDjd2OyKX7pO+UwNvZpx+CJ6PXLbavKxEQEah0eBp7IeiK8QFO2T72btbsyWAS8jH0vfiRZDyjtH\\\\\\\/b81viDDtTdCMGpG7YTTJR2DczYr5hiMyUW3KLjUKBwh7uHT8DCuZQLuIILdGjFnmRW2D67ig\\\\\\\/AYfGa13Xha2jpk6XvysFgu88fRjqUCyxZzz3vBXNRRFRtP4+LTclM7mDWnri7t8uysLvxala6fxc4ofsKxzqLbTY93H0ywM5BOmLsLEv+hYoki5b\\\\\\\/69OTXFz5SWpNF4HG1y7FpDNy\\\\\\\/U+4Tde\\\\\\\/jHXdM5nkWg2qaSlGLz4y+IEDdQzXXf+JcYdmj3+247QVWwwSkr90pAAUxyjlJoyLifOkXU6SWnSdwEWgq+zGqeUWv7eXl8jCFsr1HaQa9eBTr2rm\\\\\\\/ahkc+2nMDspux\\\\\\\/pKzViM5oW1l7MYVO\\\\\\\/D\\\\\\\/B\\\\\\\/Wwc5we8jVwRSzGyy8U836pqjMg17e3cnijH0wVBLj\\\\\\\/fboDDySfiNMpBoZeRiYsvuA6TlpkqaMDglWMWK7\\\\\\\/sT5u2F3uUfxj3B+BV17vF5eWIDOhv4yt53tvVGy3YcMFaMDtlC5el82FZZ7ZkImbVThWWJNdcHw9Lcr4nRKb6Ho4DW\\\\\\\/4oVbWWvpzDvaZ5nw8mrmBgtmIInVSiFE\\\\\\\/9UrTFkQLZtAM5e1b+i49p5Dj10H9A5FKmIsBKVZwREeTw1GR5T+UWK\\\\\\\/ktt\\\\\\\/yml6N+VC8DJG\\\\\\\/glqVpLg9h3bNU5d+uJrF\\\\\\\/644B+HQTeJsQfyfMGApqvGjE7coKkyuWcPTUbFn55nr8X6BgiY\\\\\\\/xuiE+91hpe2mBi1cPhe6yY2RtvG7l5Ip13Uz86qtzVcAQm6pNJIW985Lasb1LSpnytaZBzxWJWWr9eWX7+QPxttDJyu69NyJOVEptO++XS87Is\\\\\\\/IpG98wSNeUh1xbFrRcji4o18AHlsJg38XxEJzXprtd0AlriN1wcdeCJ81lQz\\\\\\\/D0n8L5q\\\\\\\/1oZ+LVdX11jtm\\\\\\\/3KocIZVOld8k3Ngb0hV9qh6fSxE5WxtnbXik1y6eKYyE30W50SRr22V3rNX9Ry3snZ8E0SpgxiyvYV\\\\\\\/yAVpDLmiOvCbJdRKM3HuZ5OnJuXIXRMwbfukgytpjK7iHXEK2fYK0Z0ba8nPUKQuSQ9yv\\\\\\\/o+KJ9U5vYnk4Ewrdwdmxm5jTNbI29+xbC9Ds664D1372fpYW6SkVpFUYex9JGkkOmNb58CF2bDFcGKzoC3wg7PrZiIWvRgmNkyld3hk1pm+bUX83q9QUwPYI8qHNLvWCGx9aT7Q\\\\\\\/TtUWOR\\\\\\\/JuTXL7dKfHMe\\\\\\\/AVLoUh4PPc56vco9NVNh5khxVBczUZot1pEMM5lupnQcupHZiuPID8mocO3CAnnEoFNm4Q6wGllTFb9XC7SXldhd8JnVJF5gdlkBFELM64HPkc63HYErDQHjhVraykyOHTkqnhrDb8xMte3yYuKdY9ZlKdYACXx4U+UH7B4glR44qummLYMgR3o9Iu3kbnvzPq6QrTHtJgMHOkYFbRNfzHMgVYFnC0Z5Se2jKvZV+S+x9KGt0ICXwS7P3AX3wOETbASG\\\\\\\/rp78oGkGa6YElzOwJd0TksEZeJrAAn490KIft6oMf\\\\\\\/cw7T4r1JjO6o8ONrMGwQktukGecOovem8GnLJ5ktpwVg7TzA7JVmXU9dd6VK2gukoq7xA5pRJdVyXw94WF0X98Ql4EYKHtS3I0IkDYh+XjT+tIM120TJ0WWwoBOT1aNnJYfmvPbhFn79oRWNaerS7iyVtNM68hJTpp8yTpbvgJHfO+Z5yP7pD0ERXe1AKOfW\\\\\\\/ZspSRok73tg4nUtOi60mJ25j7tDhpGX3FkUSAq5gBfS2G7bMjjhaDdoujpOVip5SxtVjHcNj\\\\\\\/I3GuXIQ699a\\\\\\\/J5XCL8SuOxUI5ou24wNRa\\\\\\\/gKbc8KgPm0bODdnTh1PMAZ\\\\\\\/LkILSVdhRSue\\\\\\\/M0nPEtTuLt7WyngSXWZVZg45WUExv1Ducv\\\\\\\/9OTo4LC+5plWcYFuRHgqht5\\\\\\\/zuDlXBvK62NTXhXT\\\\\\\/LCOJ16kqTkIc8evCx4Ri5NzYqHYYyc9Kpgveemn\\\\\\\/7VO2MaBhuPjiBbSxWIVzxVOOVx+npC+NIRKZ0lAA4z20O2poc110tdRYhEhtivWbB9DAvBABINnHABv1Q2nK5rQwLAKFQzhkWZ9sdRwB\\\\\\\/Idm89G0hq0H9RKIlbbbB+ah0o3PVSa\\\\\\\/ZtDToaFJuM1umwyZaJm2pK34yJwFFMMJzoHJPpIb0xxFg4GeG8Nt6Jz3wRt3DL05lZXsB+HFPtcURX43jBjiLKlJwfmMReQ4EDRZKWrbMb41rVtYUTMPq3tDcRzOoMrdW3bJMQE+3v+dmirN6Y+AhFkTmcupI8yvtsUHKSAs6W07HCp0Q4Z3lw5yBXGq+9Bcc+x4PKAD+IYUSxQRXL1blS\\\\\\\/eamgBdkJSLZMf2klHpkTbXKmYCCKE+2FhMQP3z8aD1F8jjf4sBniLbO3cB6y751Kn3eT5MA45JJKTbHMvDYseH5MxnXeBqUVL3uXob5jEBrZAlWECkoMyoe7DZllOpuhBmyAK0YfWCIdGyywGHGi+WBHOCyvlrBptiOvkFFGbtktw3knJMx72zAshmBrtiK36uhBo6I6kURiUN78iku+BZqyuIZP66KDRi8QW07pIfKO\\\\\\\/Aw1\\\\\\\/wI2ef2Y520dH8B\\\\\\\/cncLqJoSoRE0NDU6annF3czBTeZc1L1s\\\\\\\/5mkPPU3WQyTKzD5EJbEkFdcQQCQKzTRtTAtnwtpkFA9WU\\\\\\\/JImy1EOpwvb\\\\\\\/UZb+hypvsHGnb2IugUETov\\\\\\\/rdMddtMquFa0nzq80iloMGyTNyudscm0Tyq\\\\\\\/naWAGtxQHEjkt+xSnfz9PUgKjisxGZ+uRgdUKlapv6UkF4FjS3cfl0dePyv9SjOQAvDa8LiRNddQPEFtku1MoCLR1n+mwBAU46koDuxXfIr1TsTFLT52Utr3XrAMjpq4rwpG4uflwsxXKmDWJg7N9K4w0FjC1lcFdxEX\\\\\\\/PgIijAQco1G4tPnKTfA4WimJsW8LPYqTiPl7eKoloiywccSZETH10ECGpmC2YZH0iceM9Fc1h8uKPZlC\\\\\\\/fdFVuoaOS3mjMNvUhOgee3kwt1GKW5PoMIXjyrbl\\\\\\\/eEYPnYW080zFmzGRkHlLqBJ9UXbGrOK5kCAGqcIw7O9Ce7KwfqAmLa5WpW4sAYSMMCBPzWvRLD\\\\\\\/TeDH7pNsbzptItDR6ZuRVvmi+lGYrtQHDOSWAvb05oBkfGKcOfhvfJvVLT89SoP6nIncxuVotJrRkxZkMs8VmDHTZpia3nlR1I2jQAU+ErWEo7t5vmme0j6NzSecRoXyyP9\\\\\\\/PqPKoq8TEi3WqBxKmCCRMfeWmi2OKrEEUntxThSyHmgzxsvQHTjpZheo1lMqzVBW4LMFuow08kkAJOpW6lAORmjpwF+3GW29z2yKSGUqYPN5QCLZFelIWihjrUyJBAplSoB+K\\\\\\\/bLabWs0hKe02F7RjJ8+RQ5LdtVZmRoJJU7+a1wLQ5pnKoBmK62An4YnLHgU6o7ZsMf1fk9MnZUl27i+DJUEKc\\\\\\\/XynWP+nO\\\\\\\/sWtPIOO\\\\\\\/jsyXdA3qq8AhhIT0BfWEi6mib2WcGCu+T1TQA825CCuMH543aqNanDciwxasliRrZ5\\\\\\\/R2wV6D4RtgxfGTQkFbG7U2ZYsIpwQl0Q5DJ+98Rl5FkqZnTZ2XsrvfWLR+Figwyc3wobclK154iFe4RCEntTBsPMdtL56r2pKwa3ivPIxwy8gnhdcdLQHST7Qt+\\\\\\\/jGRGkE8uJNRHi\\\\\\\/sTEUr6Y+v1j4UnBBBOgZC3+doqENnmO9VUe57NG9dO0+\\\\\\\/xylkSwbtmOyUzYOOWoK1LanxWTyd+XtGJqsNkKjFExojG68ip6kCNIa7r5yrJLt6gEGB3Q7f63cnY0l4SnAH68XAqu8nKv08kCsoymlIgvnRr5dL48KytlgBmBdnkt5xLrUx0zEIB3YrCN6u5ahnMpTTHdzCmEWEPQ\\\\\\\/RPGiA7pK4hHBzn+I1a9cLLyZfPEP2zfXbrFJqeSo4lJIB2cJdyaNCSEHSfZDOwwz90zj1QM1C3telb5LphuPo\\\\\\\/8aYWiWzLH3Sp5wtq7ky7kEGr04pPU8vRHJxI0rPAMZPqXlJ73uS0cGLsYjIUzMQUzdBuqThL7e+aNqZ80EXdRJPBWlnLmiyrZ+wzSZWhOTxfo2VV61HpGvucsD+Ytwpa36l6RkzbSK\\\\\\\/3pzX1ya2CQut6Lud1Y9q0hNOILZpkNs1\\\\\\\/wTh4EdM8D2U\\\\\\\/uyNjo5PDPp5x1UGbmiqNpVndFKeivzVcJ0mCYa9n7WaH9Pmq2d5csOfnLuvp7W2zEafrltqhKS31OcT03y2QQt+AG3wsBbeM2JhjcaxpDF8KR8KbinXTve8avimzhQc89MTFS2cFU\\\\\\\/bLSZAsBCAJEdsPhknHD06Kw\\\\\\\/igS5uKr1bPBMAnU3+QJZ1+vzS6b25zNugPFZB0qVWwPzOcNsJid+e2GXlkuVklQL0g+9DiLODahy+QahkCKisGhmJ5YuaL7nscy\\\\\\\/bYNJhDqvCSSlwIG\\\\\\\/6hjmBwgLY3pbsP01qG8GWOdM1KSPJTuXKDzPW00++dcX21yraAYWHLj0WNKLlYBjQSGCwGfmBSBajjqWRDl0nj4npRn3A7QqyBA1r3dhmXvnxUUvGtODlU473ik10iK85F\\\\\\\/HzOqdp7wITczrxwfQ1zU8YebQbcoE61gsVP6dsG1yRUxWqr4GqcTOIhpPPgbkQglbsIr9UUkwD6nU1geLZXRz6pXTiWqrzkgcu\\\\\\\/XjuFjmNBQ9e4ckQMiJzNMJDE9KrMCtwaBWgXLaIVO5EQNwckcpSvAil8KC7JVuuuXkGSeo9NIl33FZgA+I9q16EzDg64YO\\\\\\\/eg9c4Iy+7ChGR36w4D2musiSnQdvU+anPvNR3M\\\\\\\/0\\\\\\\/0GRG5y2z6UHUlAL3WBX12Da9UBPkamX1JgqIOAUwQa\\\\\\\/w1HnDBgCwJcC95GxUdBCM9nuOliN6hTbPuW\\\\\\\/89gfYSR9zJ59Hc3u\\\\\\\/Sq6wWkyksihvJGVC3MUtopbja\\\\\\\/XCW+blc5maXPgsoyyo1Lyvs1n6DbCoQhhsW5zm\\\\\\\/tR7vDR7SqU752QcdFIvH7vRtmSngvuY+rP9PwwKeYPUrIhYpaDQHP1O2t4rj8xapYPNS2fbTDE6Y9DgmdWX6xDJF8AewCBsxLvsCp5Ij310YDROd19cRczdREDXyGqWC2A4iDrbIyYibt1CrXWmL+FZmk1iquBoRgidTgdZATgj6+\\\\\\\/vDxQavTZACQGESOuwAnLOf64J7s7Tx1HpeFtBEoEnHKmhGPWHAeujTQ1d\\\\\\\/lwuWmF7oIILBFBsZnHwwY5E9tB7a6WrojFMrfvn5EUdwVc5e1Gi6jAQjNfWCBtYJywXK80GXpZHR29DUacafn221gytSpgGHIcEWjjDZIvYZ\\\\\\\/uk+oK71TjmTXpZZuGxyfudnj3vEBQzdR2osz58eAo8CKe6zciP+MLWThm51+oPo0Imab5nxzI1RLsnwLzOoWE1riNPJLAODpXGf4W0GyGhvgUrnLzhK22GYXcdcKsXxX\\\\\\\/iH4GALac01xKAgrGQOnfVD0aTXnvqf95T5PabXy3aLwHxSz62PbbPPXDyY0WqSZNNOtszWzY\\\\\\\/8VpggxjCYj7hQj61B4UTkUeql99gs+B51e7b3+6JZh2+h3numrtES1WY\\\\\\\/7KNRSPgeyKedNldXhh6XX1vKnZXRe8Z8uSpmjrd2TXbIpdwAQfbZZVLw+Jhj6HEL9EdKO8pk8Ue2DNV9+8LpeBt\\\\\\\/GgSOSNsmcpdKwEYjjKYc566Vice+1783ANOPfO2OxDiLYOz5yy6PW7UBUcClTCqQn7DocCXmXpsqrtOtvitOgkmifHbxuFLCXgllfxGD5Wn6iPseBWdDW16SFb18PrDxzSB5KanJeDU4AAXvE2qJrsKXPFAC9rIxvfUEx\\\\\\\/Mc203MGOVb\\\\\\\/TEsKf06Bet+AC4X50TNXoIMktzPFsTqx6DhVx780cNszzwuGCWCHQEuxPXLrcM2WMt\\\\\\\/7bARiFvB9dvMRUoK0NqC4chLIb73LcJWDsUnLpRVHHWmFcs78U4wGo0xQbnD3Rvlv5z89mn+U2gO246XM2LzUzqvpaKitbI0dD+J3ndg8Bfe5LVztOtnuy0yzcfN3uy3D355R9ZBzA1SGxpD8wVtSyUazO49iXC389nXy\\\\\\\/g5QDsqhBfmM+E2CsmWsB9wnrP2K2wo9mHAlxBXt8nWFJBw7W\\\\\\\/xAeJPdsQLUdYIWh0VqFinf8Klm4vEArEKXU6gMxpJ3h2gfNarjgtVL6W8Bclf8W08KASaoXDlv4rwNojzFFHER9cxsaZFyozwraRg3ujNoMDHIvpdrLFeRpVfhq+S83r25EmkNURBsonxKYUUkFMIqj+kheOyzeqk\\\\\\\/XF1YFb8sYW9lp8Ms2TCXabHhQxf4FzoQYx\\\\\\\/uNPnOK40r4Zutmh97i01o5zAVudSAnJBYU7jbyDH8mm10KBRxYlEjeik1xP5EXKqhnFUif4hlDmSNRX64C0BFXaHRm9BCti45UeDeatVlVDjidaD2K6fSdfD2GPimM4x3ljW1j3MJ+Fck+DRUkSaqvlx\\\\\\\/kh+C72QHv0NtiWthruPFdSKiBzmNKNycmilIT4yJbJu0z1b5CAMEfhImu3flyKL+4ftFaS7X+sFbZriGRgrvBdvB3w\\\\\\\/vnuHjIJt3JEZCswDr3uo9EXJYm+cryGgJrm3rtKsOeyQ4LIAFy7ZFLDhw5AB7h+YDX27Jw726Oky\\\\\\\/QLeoPMtfsBDlR\\\\\\\/\\\\\\\/Pgo\\\\\\\/lRtDcLMr3D2OFj+PBCOXGYU7o\\\\\\\/3Xa4X38Xijq7A1GEQ8Yr5rJDDHoFyymMnVYYErOnaU+Q+\\\\\\\/YRMXSRVZthBCzktSdX8uIVE3FeylJU\\\\\\\/fQBdTdEAuq96ihFZ3\\\\\\\/wbfZwOulhkj+\\\\\\\/XM\\\\\\\/WT+QvhlgAXz6vtO0sFaDuN+Yx1LzDamOLNNrUl4DuuzkiMpNS2zRY1N0NBlcCNINA4q6nAFTGtDPUVlscGfeEXKH+WAjNmP5d2on4sHMT1vil2htjRPdbiOWS0ZU7kUhumbAr5dCgKjiYWR2eEQ0nkXag+wtc4iXn1rJ5pvGl4jVUA4mWAbm66qzuvYYgAi3CfrLjLeDtXQ3QRaicXaxuRWk4Tdrx2GDSQo53LmSGANX3z5RaIPMSbjTXbznrj+VcDpfkOGyPgpTpoxMBftEi9PWEh+0RcSxmwpk5m7E5CJAvp6HS\\\\\\\/eN5mJQe\\\\\\\/rBZwbVkyw3eQquw9cBxJmK8Q6Lakg8bgqAdDfOBXReZszLwQgyFa00jiP56isWfi05ZwvaKS6W0Y7EdT3CKxAb20DXQ7taQ7joaE1PZ4dDJl5vDUK9HTTMXTSpb1p7RpnatGiMAKDRcYju8fz0YWLEIla6vcNQyFGvVUKvnuJvRg76NCnxk1t40GhPSlgdPYsQaWVptCIQdrPsHyHNALi8ukM9AM8H1GI\\\\\\\/9X3iKw5nqKzeUhm1UdzWvuGPakbeA0EccSZh\\\\\\\/b0BFgH4Yqvss8LmdLOccgozuCxWXiTFdo\\\\\\\/jRrFGqB1TRPu8xW9M58Qt3EOTFqrZ9eT3Hqv+g9gHiIu7qSviKAJl57143WvjmviKnNSfuY7Pbm0Nm+iBJ2BsEVcr5Sw5uT2MJ6wvH18NmEmDDyArKLeo5Lxwhn5qnOemvvCiEXU0t0Ra7\\\\\\\/3arZKyJGVnKiyiIInwie4A24SilC7z4q6nD2FNvK73SqafV0wohpgI2JpnPxHMzkr4sQPVUMJ3rbM7kR4UZRiBImbBNiIO7KNJocQQrRMIi\\\\\\\/G3LXG+GVixygbfZ49eVeNjQ0bwGddFnN3R\\\\\\\/ut2arfp9LJ3MyOjBh+Ky2jGBfGnqgrA0o\\\\\\\/xw2fdtamfH102aNWgN7oH3FjuQ8IPjT4sdXmYZ69A7CVZUkLePalVqR6zI\\\\\\\/cWvQ0bS3\\\\\\\/nW4hMAY6RGgqhkvDAUX9FIhfEcMc2ic\\\\\\\/NICmdF53RjCddSPyIz41tqzyWGt17E11CkAS79dJ6ETJJQm5PRUv4jov+LFmLK2sHtn5E6v\\\\\\\/mPlEokVpEiuwAoe\\\\\\\/qTT\\\\\\\/QRpDaShC7dNewA8RiSE5k2lzGf\\\\\\\/j+76P8MB9F+MD+YnhRX3p7z1aIx11n5LpuZvcOq7DdvqQehU9Hdj89OT+YqkBuLymDDJxji80x5Xci1Co99CeDx3XDHeKyWSDovx+xYmMBAu6nz\\\\\\\/xdDJkAAQd5bY2yABRNuBSswurnoP9Qeoi\\\\\\\/\\\\\\\/SVrPSrOKehs1cw4U008GYLgNj\\\\\\\/6S+Opoaxp9OLE2cqaf0VhMNsaputH97qsytkG+TX4svMoBoRi885BANLT8\\\\\\\/wnwD5kx\\\\\\\/1TvjBcPFivboPW2+ODT5ZoCuXkfCgfXvWqKdm4kbX8RgMQsQiqFIVkBwaDsfT1BynrgccQJ50tP\\\\\\\/+Ua\\\\\\\/To05CAqyOIvgv\\\\\\\/lgoSYIksltStTUSXMFnmnHY7rRIdJwO161uKYuKXThuheG74rtWTLTMHwDF5ayMzqcDCa\\\\\\\/j1oGjJwoNnERkDMV\\\\\\\/YoM861yFEpssXZGiOSctxTyhSyElWx5ySnmMW6Wmo+AOz0VlcWThScvOLPZ6PlQzoXtrwzwVdCMs4Ve+iP9NDAIJiUE1oa9Uez6Z24MUR6GFkJEyyQQIS8Y285dnBxHYWXjhhHTdCzB6hwzo1oj7ja0kH8jfG8tUW0EXvCtmwSSG69czZd7PwbJ7AezazS6qKGsUIj\\\\\\\/Z66EZlkD5+h0xXCyHIRs7nYFBM9xf+e26Hg4Sr340HFIFX\\\\\\\/kllNJw2m7q6bBRrD7tDVY2ueLmok3EUqMoP1VrN0v5DoZoWOM8aFenOHTXqIxdecV8QuxQWaYU9ZD6Xu8KJurturbcSWGTH2gtiCrR3+qR2tktpaHed2gw\\\\\\\/iNazvTaPMACqU7X\\\\\\\/yrKeYhRDCWY0P0fyqBKhA\\\\\\\/1dQjjpN6xtFdYdlnBDczqFfgdLxm4XdNgbnrCy5tq\\\\\\\/hPzEHNDrHvR4PrXt6TdNvzINgv95A0ULM9E3hIsyhjCzVvuUFOry4R18wI4idRJXLOCXgvVFGGexasgg1GDbw3Rm33XPFCAbyGmJj0CzPjV6CHwCarLtb3jFukHoUvh56NW8MXXIHmzSK5TQGE6Mhl+KqMFfIoWtJu9lE\\\\\\\/9ACE8MuBvBXVDSnIlxGNE8uWDcel7LAb3YHLZU9C8GIfN3qiwUpUFq9TwYZnrhVqHswqwI4KV9SITUeXQV8WnBG4bUbD8Uiqqh+uAABIrl8uWE2pE5cT4F6YAZsgG8lN7MsG88DrvYJ9XzxypZwIPele4Rw3T+U9B0VaeQ3V0Ij1JWBq3PEfbGjLENABEDVMMKwsO+5hUEqsAPCydFvGq8Q7IO2rvg5BH+FTm0e+bDWrsC5Tq8pEsnPmlEkynh+7x6bKFNcgqnYLRc2r9vEQkJ++qXniTKb3SePh+GIFIsACwBW\\\\\\\/fIjmSwJQeAgkN6lDq677pTNPHB9kSlKFAEjina6\\\\\\\/NBTY6I71DvQ8FNJQPzeWV174HU+dZlrWYtwkGnXiBn2YgI0LrJw6DmOxZ6IFgNYaD5bvdF+juA6AKX5iO3B8UyF+BODlRHm90r96JUZ5Bm1W+GJatp7bgwttsR93HnWFd2K3RmEf199XZ0pW8jNDKYTDdeaV2RIbmQmYSOdRIOC95RvsBJLej35sPE0lolaIDq\\\\\\\/AopogLXMZqwtCHXGyZ7PhU0oZesdvbCGv+fRdNMIvO2Gg\\\\\\\/UFBs+refocu0CkSXJt9rubBFE+1+ONdoPzfYSe5UFj6Kb3k+xFVx0nLUTrVJ94AuvzpVUJad5aK3nd5rlulQCrb7iSSkF\\\\\\\/cEyyFnNrIzMGVk\\\\\\\/KHHgcwYDyOuUnuuYKjoqp5hqPiMXXmfbVn9Ml3Fix9ThXFCAy7ZBOzL6tqqNOhFO36jXuLyUuFWe0deqwtzAxCBAJYg9Ngw\\\\\\\/X1C3zvTvFJXZKltaZt7n2\\\\\\\/9jdD2ysRk2RLlCZGFbH\\\\\\\/F0kH2aI0GUslwEwCLwFecMf4LheWf+2aiLsae9IEd9QJnOW3MiWaKBT2Dpg7cENxv400TU0RWaoZHfzXUz34694+OalUKgjPpcOy0+FD+S57Cc0xpVWnpOACnEhyE2qmaQG77\\\\\\\/tAz42\\\\\\\/pfJimch7to4BkYgw+PEEoHp7pRH9FaYJgFTIjgQ6Ac7ng5rmPb+NPGMcNC18wZofc8at3Wzj0CEY6CuKmXADavMGfyb8EhzBR0HIwxHdw0UvwzEaCUs1uPr4Q0RujSAxMx4ySOjD9IF6U7N676ULm1uKD9slOcEeYZAhcyzMAMtYC7gak\\\\\\\/ppuRjMLXz44kW1DmWfAHwdghdjNErb+WIHlfZLBi9Eg6KVczQz9EAKwkCgSQ4M4EY7bkfViCTt7T40G9BeNRbtm9xgJOZRztaARYXBQmiVbi03Ijir9R5JlTfZPU52j207YQmb6Avl4J3XSSbXlQwPqEKdQHEvZO0GRDls9U0BgBa56XWPXI1UpL9qfi5ZRrpRPID\\\\\\\/Vom0p+YhsdTNIcU7xxkSPnq8cJSEv0BMBl+CAxtYz9P\\\\\\\/ElqbYEw8BD6CjmV8RUbfK8ElfMvwyQCvIwGjLklFFnqrpmQfYXQwkUF6fyA3sY1YJhsqr4ywkX2l29fW\\\\\\\/AOz973uvVAPiIga4PKu25sxxN7AqLyH762J8DgWYhTHzBNC59CpgWh\\\\\\\/ITI4yd7KGyI+5hywlXB9IGV8ALwnINzkSlQ83EcbcKM8fBmWAgPW1phTHIESCOmY5Lmg46WG3RtNtdNBriux2FTnlmXdpecUy\\\\\\\/5c2wV7BoHfJMeVHnk\\\\\\\/jUw6CST8Pq16ao1wfO\\\\\\\/I4MRrnfjCg5ol1S+Xui\\\\\\\/mD79BveXqg8MYIAZd1fpryTqqOXLsqR7JBo8tAlJdNdYSbMHoWK0OFbLlfoD27wZmL7mMz1TR9Cgh60UUvVn4XN\\\\\\\/8ScYyc0SZbjCYns5hIRZlM\\\\\\\/4qZ95GkGGtQlUiTAkxEppRwohHkEIQK\\\\\\\/u3XPYuYsXI7EGRddyth1JgdgkeW8\\\\\\\/GUgidqXo5jvS0jc1gYMeVqOIFnxm13z3hEcFevo5l3kXJxST4JCypTGFi0o7Qm3oZMPK39vKGnq7gOg3BdMCflsG\\\\\\\/Dgw8z6DCRc6Gkuc4fOMplUGP3b3zw46j30VDA6v5XY7E\\\\\\\/FPCzBC0F9hEW\\\\\\\/VN3lHRWKNAZk1P9Sup1\\\\\\\/IZVYwF1Fvz\\\\\\\/5KE8RcGoD0kkx2piHprIS0KVUoj1I1tQ5TQ8I8bTQv94Js4vBy5ZFGQR4aiQ6qz6dtfEjUTZG6jTI6HqinN4nZd1ocQcqfb41za8XdXiW4zFScwz5GSTCig2M3RqqkDiEh32Ggu0lcGIxGcPWtyx0k\\\\\\\/OItUu30\\\\\\\/gi3\\\\\\\/GycyktTzQsloRo6aoPOCsN7TduWHrXvausG9O9PIpuwS4IQvc\\\\\\\/l+1wgia7TzKirZ2u0n3EyxH22vvqWHAcnW9V\\\\\\\/h2X6bEf08RLjPz6zDb1ciAphCp50mYo1lCKNzxRXurwFrVbbqtBfSP0WR+BhFbwI9m1Ti9+FwFlSt99yxxtK\\\\\\\/V+ojjSuhUulb3F\\\\\\\/Zeq\\\\\\\/JR9g6YfxziyWMHqEQF5OILAE5gQDjqboVNOJrGwrq49EqgpHiiIvvXV+73qLUX4lnthYJTm0OXRP2E1z69fHKjkiVZ10\\\\\\\/REfq9+sx4IvfsijBVUSA42JP9labz+dRDHb8o1RkhbzYBj5FSI9pCBUyJGoxDxS0jE1OsQLPVNO7t2J0HYLcShqDfCdHpIR+MWJvloRFsGzZ0PfR8P5Z1eA9AMO7cVjAiMBbcTvew37JF43vQVdf2RKrd3Kf+0YRTE4mb0qc1HaRBLNHgkKc53aZyUvqWKI5BMxCc94Z2UuoZguH5hn408PRWFWiWzc9dVAe+KsI\\\\\\\/jX7VnT1m6hTKODBrW8lL\\\\\\\/30BtZTTjQMKfBBp3mTJp0W6FkbsFIMyUibnVD6+I+Tn\\\\\\\/yR2bZbRnkJ1jnKCq3UpYI6y8ee4qNqwSjOpComcMOw9YDLejMJGqiDRJexzyIANFlmhdqHKA+wQINrDNZq1Pe\\\\\\\/g500LvYb36SjU\\\\\\\/7zDrshvVmkfRctjIHMKfM8uMAlvgzWSY4v2C7bT806xHstvSXfRj4jf9mChdueDbZr7HyClLrjyEsCTDIpRN+KSF6JUxAqmZkwX4Iw8oFGy9JGgTxsLnBEZ12TYEsvSyFCkV2T0UxhdhkBxb4HSKzMuUOnyFLWxwgkjLfT89Jl113sndry2hhVOH14CQD8iGEYHfck7L0RUKkL26hJKiR5QAj3J4u856b2ysy4KdqOG07FXrjMbJovpSa0\\\\\\\/yTQDy8UD27l6Rll3uCSVOV6gbh35nHjWz30h3F\\\\\\\/7c3ZdsygMSr1DDjkYOUjy\\\\\\\/+PviflbjtTkBwMWb0qvIGCgXsmiJy2JqhAOaNs6jnNTp3KhCaS5Mp3owlQCURdBgH5JOOsMT09Da7lBpRydLvcjyFqLH3lYAu+7X9+L28lpwXnUPNVA+DLOIC+U\\\\\\\/w7jg4u98ZDykL2hXeWEOUBD7ydp158TyCGWuLYSLGeYQbU7kWziEiDkV7wYQPTLHFjpR+W6XgbJUAwBgvHRYlhBWow8Z1oieGypo\\\\\\\/xcz7ffzUgWSFrrzR7ICL\\\\\\\/6VNqkaT2MqhoEQnjKE2KhDDOEKn\\\\\\\/bYdYm5zurv0CEdWzFa2x0x4I84aEIzchczF+igybFLItPjVMGWz\\\\\\\/zltTVGSSn116tse2LTrajv9deSw5VnEpe726TeClQGEQyLzbG\\\\\\\/4gjVGzQY68mQ0vDxUR3XJZStjoe8c32QY7jbJ2XpbNu8FI4MT3HZxG1FaVfNHXMbHD2InHT+ckFlcD\\\\\\\/1plSl8FHpVoXVguuJLUAd7tFLcESYFovUYOR33AvpK5o9oTZsAowM2NN+RU9tqOFrvBqxZDbiMfhxcIhDHKVgBAf9Har4W3mdY\\\\\\\/ehqsBZhBhuCfmX80lhR07A3QQpMj1xyy1eMAfejt9ju\\\\\\\/qRFS6gOSSlsdfgNWmlHEcOEniEqYv2KjSiukhjnpt9gnR9ic3VpjYher8giv7ISQryMzfXGDUmvsoMT751YUJipzvVq8dUpE6M\\\\\\\/hRcm\\\\\\\/p2DnqFSzuBQzME6VdV+TStR47W0JMoiUBmzHaV3kb5CT3G85hUGSLwDn5dzgnicoG6Aq2GUyRoe\\\\\\\/hKl59y0akA7PNU61nMs0aKQEdoGyBjZUBBqqaSfAvhTliIT5geBKWK6U\\\\\\\/hMjtQn1M9fqi9Qsb38epbWwzbRsu\\\\\\\/iV2gi\\\\\\\/+iKPQmDf3g5s4tFMa\\\\\\\/G5uhJVcfjXAj1l71FLFT02amq7J605U9zgkgpGpkZTVxRbSsP8For\\\\\\\/vlc5JT4\\\\\\\/MKEPa2poXAr3zGhBz7T9AW8YuZCGZ1pAw1yJLIzayk59EQcqPKyPvOu9x6CMKmdKLl0VnI2DT8TVqGFySTdlx3BRnCJJVf0\\\\\\\/CDUHJh9RU\\\\\\\/Mcq4kBZs1cW+7cQZJHLH0WpgAJyFlLNLuzzJRIxC+lJzm8SLGeO7OAvph4lN7v3Io7cGWPWQLVnxsjdmFmWec3fNiHkoMCozIbHLbhzSUzEPU3ehrHYrho+40S\\\\\\\/6kV1QrPn+X\\\\\\\/VAOsrUrGrYmApe4k\\\\\\\/HThTbpCd37aZ0frCVBGT6PXNwnTupuOiEGE6k6fxAsMwVtaw7sm2cVaPPoqouydm6wk2ocFiDloVVJwOmfSoYhgZ3bsTATsm3G354F9lBs9sfk7GvZgdfQMo7yqoSk5z0gMi2Xokl9w6butYmkbA+xy\\\\\\\/lzAKhrxQS4\\\\\\\/7KNecjp0afCXy4XuRFp8iBLOw54mLuLV3uBLaeN+oRHE6RlaOWm2TTnQTVhGZskLekL67QB4Xvz\\\\\\\/Ju+Wlp8DVhe3HFylVoBiKYQnPE11ilHNCML3wgjxi0a3H9WJVHF2Gj\\\\\\\/MOnILvBgTl66Gi872CPROhq1igj8x63fyfW4g\\\\\\\/Tm+Dw7PHPV25ms8H5itsjLaVEGZJyi9bZrWyHM0TnVxroF5tpyIxrwkpOe0zcv4pVraVAcScKT+5m9nAt9YT\\\\\\\/rOOOTRppSBL7dsbgOnArQDG\\\\\\\/\\\\\\\/rKd5OOID7asCIH2oT4c6tdl+Oi9F0ecRVZEH6k3BsQ8sPnWzHwUx1NB6h1zOG6YCayi+ZlMSgFH12ZjxzudKBK+Z4Z5VPj0u87lqqfGBsHNloVqsF6N52wPqlSLVSl1hJiYM8TV158QSAG5NQHCyikoAM3+zompn6I4+lNWa71vah3Ncm4G3aXEn0zRe1EJRdZnPsRlVh1WNSy3Xh3WojLeCqV7Ory233lKmOTBPhFhlKvWMOc3XkDvp9JYbaZqbc6kTA2T2ga1M79OoTqYBfUjuQ6iq4lMFRa3buNSNDHVX8jyOYsxs7QIIbDfREkV\\\\\\\/+5pFIjKfyarkLKe6G5DPQXXTJoqcDeRyDwDEO2kwZgP41jQ8+xNJHVgZtREQiRNO5+oAZQtSHG\\\\\\\/TlkXbMGkpQJrRdKgvCtY70hBvTdlct9NyWH4hmPE54p+jE95RKUxr4P4DhGtzxoehl09sdwpy3JXNJYLu5HRg3Q+RJdMAYY5Lz0raLeE+2ldTEVwEnBE723RyoFkBF67DoC+a9baEFAePsESNNp5GazcfMsR2iUtgBN6v8r6Za7gxwRChwMwpjTtZ14dixST1k08jfSiw5bn5plA5+nEF2Eh9i4wV6sYDev8Cq0ldbnOdIg6JC6t923e4vJUixG+xlMNiRFwQDxooUfoCzV\\\\\\\/P3s3uxj5rUtN5JfMRzMfZy4rR8OWj0WJN2oCxJguHU8YZ1hCQhL8EAlC1PkjuYIXN3NKoNzVOyl\\\\\\\/J828xeWxhz9KU6J9ZSnZDdCpNZbK6jSn388lqgcL2wHHUa9yZQwfXcQnG+yplJNH7Jq4UntPG\\\\\\\/OYX0Ay7vyaKbWNxdsJdNpbT+MC\\\\\\\/ezvluUhaE1AFlt5fLv0DDXvcMQRgLVFYO90QlfBKCfFbD+ULNNwtkWY\\\\\\\/oELC3BXM1y3NUi5gKPF7BG4CEfpLc7O1JhCg1mM9VXZ2Obdq4iddpOWKwgvETubFPkJY2sBGhRnYme53r26Q8Lav9lly59kRcdEDmVwvMagakVvU2CIJE3cosfbXJbG3Yuud6eDbaoOSe5IPH07px2SgFqEGgja9wLaNH\\\\\\\/2bP1Zp7S5rqeYjZIroAE2cG7\\\\\\\/1ZLC57VXz0aZurVVlaNbxyGk8tyjQuE75vdPS6\\\\\\\/SCje2BMwoXhhr3mTQiWuAJ3rX4x7K5uKlnrnD2ytLj39CH1bMcbyddwbXvyvj60c47CLyaIUxywUdscFT7yTcHC1HrBCL0zc1aj3g5iYgW+og4RJ9CaZZpW2iTEJ0L4AKCPKA58Yl341bGXy\\\\\\\/\\\\\\\/BswPzmJDK9JmU6933JOLpOrcYPmeCa7xlsJt4YmuVRQ0EjixT6gdteF1XcMpKVOpJx8hMlTJqWf2t\\\\\\\/PkKqK40TQrFb+s7HM\\\\\\\/kq9GTD9WwOA43+GFuU0\\\\\\\/iUMiSRXdK6VCtRegVXt721sIFyYMkn8IXSJ7Wtth3U8ff8j8r1MgIFtkCDAB3qKRJLO21qT3uHz5CiBcDXabxbQERRSF95iS1i1c5PGSV4Nh2UYtDzdG1opQbSMqJsYJrIu1gnOJTF0ofX7ILM3unib\\\\\\\/oD\\\\\\\/K3r0WYf1Rq9n6ii1v3Ww9YuQU6nvzcd3sEbYiN6tUivP4U8cv6e+FlGUV8PXv8Kwn4CpNKvzgdgWpmbz8Kk\\\\\\\/\\\\\\\/AmeHca1TxCMA9oIhPqPKGaMcMxOQXMaiyT9z\\\\\\\/h27njBrzmmB1pAOKpBZS0aQtbsyGO5lr8qVPt4P3k9JpmTh6fqEmrRW3R9lqIbZ0\\\\\\\/D6AjeBrPbAA55XJ8LVxzo6jy5WY4ZDSs86mrdSlw6\\\\\\\/4EhIzxTIqhjeioP\\\\\\\/63zL1F48SkmXXn1YJwI2XB5DuPh8h8+0TonycLusa6p7wtXw5YpDHj6yYqES76DYV+3ypCvEYlHSaG4zLrZhd2+VRf2LqEjXv32L0N9haElS73FP2NcJfWtJ7Z6sDCJIkiXI16oxrh4fBrRNhK2SCDjrgoQjxwL1uCI73WJ+MAWHkQxJNdy2x982dFznxkZV1hgqRl7ISKtUg6tI8ULXtyVgRdiSwWwCUwDzgZRvMJi76Tq9OoYU\\\\\\\/bp524ibAaXHeqJlde+Rv5U2xNOY5ReryOjZ8pQtQB0\\\\\\\/GH1ZoG4isnOFazBngDk9aaFi3SrhXyPBPJygURkITRuJRsH7zPqZF5qgnmI9QrIr1LNokAAbLSEv+WNGiqYnX6j6G5wR8MbtQUYYbeb179tJ6Au8w6XzSfaoFcJx93P8Euqx50UjDwYlzlrWc1fr4SBVX6Be+yTo7ESGEBK8qjl5tAqMJlcGQ3hnynm1Pm+KfgTLc6AyPJc765BzA2Ee8NF1\\\\\\\/kiNjq9w8NJiWeWxDrBOZ9OJpCjJ8z\\\\\\\/km2yVMeYi8wS\\\\\\\/tVGeD7GEp8TNxSuYQsluvMHpEtEfaZcGwjUBC\\\\\\\/Y6vIgvGSaUS\\\\\\\/W5IxNy6xXNRoVgb4ztpJiEHchtMwdMLaXjvcNpCgULHR9RivJX7aiBPrWqf4ntp0EK7utKUR1lAtlAneycvl4dMY55zMzENLYQWuVsaR34F4LUtKbudvXVTtQTHoEEHee+f9muj4zl8qH91\\\\\\\/QipXMMeJspPU\\\\\\\/3qObKyE924mSFnj879pnHHz93ryQGtTKz9AHlVVZULqUsRCIZFgMDFHvflRaVWH5VtQGMJTfDZ5VD7gNf35e9p7f\\\\\\\/toNVl8Bdr5H8PVFeSUmZcUqkqYEnGmrj+OUS44lloLMbvc+O6NS\\\\\\\/VtZVwwtiOhVoblJgrxDGo1+t9ICLpw+y01L0TuQbQWtPTuBResIVcaGScfEGWM5TDphhxgEEHs\\\\\\\/cVzLgJ9uYMWwPLET2tsTzf1YGVzunbbZpKWkI9ZflXnWVYjY9BGw\\\\\\\/9IPIYnOe02by+b5vZ30HggZYI+GOZPozG0CEZKag1rPtzG0a5CQj0obPMvkaLW3WtuflQ99lnCWKCegKPgbS4KmRKPEuikK3v5o7W+a7Sq7\\\\\\\/G8FlWTZed\\\\\\\/1Eyk+PiO2I0wxzWCcBBi2QZsA\\\\\\\/O0AC7qNldMtY3fu8Fdm47wxIhfEjdPkfnHZeYcUoK2306k\\\\\\\/k87T2it9JUlK65GSKpxBBFOFf5vdDMnSha165hXsb0FdKf1sH3Y6BbtBuGZ2rqvcUqCzufgn32pAb2DGSujttj0ALEzNI\\\\\\\/\\\\\\\/V3I3ub\\\\\\\/eIYkdTQNkopYLHrY6x8JOTLtAWMDBTEGXptlegw4rkwXY6Z8CNe9bqpZYa\\\\\\\/Rk6A1CDICsppayv37T3q2\\\\\\\/FB6oKgE31PSMalhsY41wVqyhzSAOinfZp460\\\\\\\/iZuX3bT\\\\\\\/8Di5gZmVgMi25sYswlYgrWuOGGs4oiIzxv5ywRGgutL7d+tQF0CMiqoiWXJwtyIJ2m6z1sDPO45MNLHv169uxWnNHUMjkNoJOubKG5gHn548xfwwqcK\\\\\\\/w93E6I10vj4U8Cuk6qC4j\\\\\\\/kkS9XmN57p\\\\\\\/2eW0xPz7r0fQqirycNHZgKEvWDInGIjsa1HYN8pebVEM+SOCkNp9fVL2agYqsxjf9+qbmWas30Vk99twSEPQXssbG056V6mHGapsHqIGhxRqeeAIngvIVroN7g3eVskgF1FCR+u0RKI+guc2HhYKYZYblkKJpPxdtdQ6AZSYdjlKSk9jVQCpOZjLkB0Ei4aCOcE+x40PAJqxeN3NXSKxiY0N38S\\\\\\\/IzgDTZl4+pVymYT\\\\\\\/s+8XwY4e\\\\\\\/ZbWL\\\\\\\/sxXtfn04cIjjNe65XEQbSY3Catyil366sG5z8Y+rEZIk8gDrjaMBTfE+Tt5frCvP0XURoSU1lWYpatJaqtRpmLwO9akoVF3+KsaXBYxO7p+7MO4LBA5eGpkNeK6KTy1XcI\\\\\\\/ClZ7Y\\\\\\\/lYAEmX98cZJ27bJzpe7WEG7S2jYMRB6vq+ykKNN+0rknno8gbt7wldpSnF0kSWSHgApGZzCxXr88fOQU5St3KP8jOkc5BNOO2JoQpVMIaWyEYqHfBah4VXXidf51Bdh0MkrgHwms91n8EjhARCZr6F6BtEDS2COjOj3qcdtLNGm1ZjTJmeglhHT2fIYGpREIMyTU3pRILJZSijgQ\\\\\\\/i2hAs76XPgmtSaCUmOyY\\\\\\\/nptZzJKU6I71mrusIQwAD0Ay\\\\\\\/lbe\\\\\\\/ptqJLRt47F2rIA4crIb7HhKoEj1vDrkP\\\\\\\/JT5VOKs5Jt3PDzCP0JE0CHtf8uGua83RFRgu5JnMJqhqCkIY4D7K\\\\\\\/Ns\\\\\\\/4IMTwBz\\\\\\\/fASWiSN6KNgBOp20NWbsuPKiD5IFm97qJb2+SVHAFJ4ZSWMXauie\\\\\\\/AqWDtalin7FqvtsjoI73nEfSGJf5MAlyetLybpO1+De6KKv9nd5LPt1MCteG3PhOfWno6tU5MFScyZawcHU8FqqezGd+UveD716Zfij6MV\\\\\\\/QnfD1nMjW49u6cfipUo5XjXDOfOVx\\\\\\\/yQ8Vl3AK06QuXMxf29o8jyHwQcRRSnjKbyLCWN\\\\\\\/jVbFWu6hWi6MOoahMr66QWkw\\\\\\\/EjSPHxI3B0dk5PO4AcqpTAZqssGUlNKwrWR2N1Csn6s9te2QW34UP1DG1eyXDJy0+mzH\\\\\\\/lZtiLUMocU905hqpMKFihpth9XReHXAbT\\\\\\\/\\\\\\\/7p8iJ28izdWH6ZheHI5iCYZP6AXI98iYZ2Lkpov0vzGnZBHV2b7\\\\\\\/JJKV+qFI7QXVfQ8WfCy0Gr9HP\\\\\\\/N8LSqBcLk0Nus07jPtP7eG0wS+gKtjSbiv3JdHtvBQqNb9eU56kXGBo2GMVXBkXn1uERtKGb7hnMOPC3GiDerCz9LqfHwCMyv3I0VTAefEmSY4cCMnqUN1KoKWol8sR2keRLGiOTO4zoBZysm5DIdylAFaQkOelf13EdcQHZbrRCCCUCdxNPs0878Urb\\\\\\\/srt5O50hMsl63aVsAYSLQckwtv3xObXJKhY8YQYbk4BSip7XtqWv6PpT6IKaBuArresgPWxFH\\\\\\\/opSCLuwjDepjWQTEDGVt6Kgf6TywpIx2RbNlNRYvrEOWtcelT8MyPQBPO+16lVqT51+CEoytPnmGzSsEVW7iHs9j7GYlexGVZmBabm+v+3rSsOM4kG\\\\\\\/AdWpTVy5kRqMnq0yx63kgbELS2EFmXR9ZZx7vfH6fQ3jkGQSyFbSSY6Ai6oQshCvl6vqnGRxGPOSDo4tF67dpj1QEqKL79iIjltw5fMtMt\\\\\\\/ah1smIKUjKSDE0OzSD2uP6f8ppb+tBVd+CFLcrgjkUP6TnW9HQkyrcSjhLqGKSy8JlFkjczPls\\\\\\\/ZyhUA1kkmOUELZh3ARCCqVJUX1f2f37kc6tqbS0GcXNa56U1Tpt3gsZRatrWvOyJokHB3HUyAuoCYMgq30CSTVfQ8jeIt8OdO7V7sg99iQZDikavGsYtQJpItD\\\\\\\/345ALivNlUg8vuQL8dxxmsymy1nrbXSsDi2bfLbg9sOcKA6\\\\\\\/d8AofRuNnm9b3\\\\\\\/sGsyiwswzvw8y0BKCp60ITm1tmrKXRzcpbJN42EyG93IaXWB94pxJ9s\\\\\\\/sXKzo0Y8UsP6kwSTFUugweuk\\\\\\\/lsL7l69cvSDOhEPJEPAqzfZUJPo5I0oRMLIMcoc1P8WUrUnLcFtCzUEoqRs9W25X49yALy2wSeh65vkhQVArdGwlc4h4DYryF9ahpjXCqXNu139DrhAPzdt7oV9ClYR0L\\\\\\\/UIbmo5K+o\\\\\\\/UnrWlMRtI+nGgGVeythh5MwiQAa0zNJg41kiBSfOC2Fz2aFrYCbHpj6nZD4G0HrWub1OHz+3KiKwT4oPdcuiaRU8l4NBC\\\\\\\/6bY6QPz2p5WdKeq9zC\\\\\\\/bnXmnQnPFuI27vfSh3QdPxjJmrqZGTSWL\\\\\\\/yOKpBnBodgHJCE\\\\\\\/BFGZKXpiF4qZcUjKjK+ExYO0Bd4T3COC4YW\\\\\\\/8kOM5vvgH+rD\\\\\\\/XLOcDj675wI3EP9JkL2txUBtD3m\\\\\\\/NwpbOr0NZ4GmYOptHNuR0AOt+eqP+psYGmYaCLKCZfmtJk4HdkT\\\\\\\/slmg7YJ37tvTc23W5mC+2CvkCc6QrE1u6l3ZKSoK0WuWpG+6fYtCJWW24sDlwzdobUzk7Q5h6yrKZJGHF2UpPu6NwQ28VUvVgpx6rVfFAbZ\\\\\\\/wlXenbMJEUW6CrmJvGGRZs2WEhv6Xl\\\\\\\/l07F03U9TJmpHxPBBfOuiAY+zR4dBZjXgxVstsn8iE4tp4UEBLNjUngK6Bplcmu4rWeQiFuOMvRO5zpHDo8\\\\\\\/Yi0rJzP9Pc+3yRZOALAmJCqqJvsc1g07zBwYNpMAb7GNh3UWDM5fLictVAGR8ICxNYs6hR8p+X9wjeWH\\\\\\\/KlnSCmTYFUGcI5RAa3ahFWpQzZkJLDAevdeSm0b392hFw0vUPPbP4fbwo+yXsHCgerGfvA+EKtLEWLWg+p0cGI\\\\\\\/Bx4Ki+pU3mH7bgy8aCk0gBtlSO3h6KgFu8c0\\\\\\\/5yw7ji0JClXsRV15Abw5Grf2GTBO6lecMenri6Q6rGFD\\\\\\\/wFBB6UXioory\\\\\\\/i44XTh1vQOJzTXsV\\\\\\\/g1s78gkMYvzgGM4XGB1U02hSOkGcJJ7RFW+duNUGOQ+Czs2aE1BjbIfT0XwyqNGE\\\\\\\/v4XqGUjOg1GQeoZkKpGgiqyPTJdHyztlCpf21tM1P+iJhN3LrE\\\\\\\/CftJAhAEzDSivg04jsaSfYnjrjVAIaAEyujo58\\\\\\\/kkqJOb\\\\\\\/JcR0f5wfaiTaq23YdJ4MleJvF7pg+W672TJu9u5AqST07SBW1EyTh1Yz7JLvBA+6sT0WIihjD3pIDYSloRMnI2uk9mh85jbS\\\\\\\/Y6MnqZOtzAv62aR\\\\\\\/4LtaJv7AaCxhfn7g1oQrcDYcwQJp9keu6FIPfSmpF3vJTgAxeR5D08acu+zmjL18QxRiPODXgUZI+eeeMUN8mWJxRmvt+MMDBMd73dFJ5VIsbYcuWhLfARYTxSHOf38dFaAY4Gui9iDaaGGhEnA2OU+NiJsBKPm1Jlx5+HwVXMcG8UTqEfmHPXKmTqEOhLNybL6+80n4PYChvDCfp7D2oqK+4nu+5avtmRRwrfTnQU5Tpv1PFUUpiwyeWsHEM0thLuspBGNoWKQyog+68iT53cYT7vbBusF5PrqmGj6KedrNfwpsFVhtTbkqTqBF4smJYoscVnl3Wx1TvjYzwUwAHmWEGtPj\\\\\\\/cHWSNda7U8UBonTwr7HN4AhhtkdJH10y5uy0+9grh84kartXHmHND\\\\\\\/+cj4DDbvpqvYdpfs31wXtp0Ja2YQp3QQ1uAT58VfF6U3chPb9qDOzVBMtMfHfY8M6CrPbt6wYQcD0tzXs4Zg4MM8QXmMIdWNW+rj4O5LPLdFszVdL7lSL2f6zPsEhRE6X\\\\\\\/FGlsZwjmYby2Ml5mkzVvccyk0xVEnubzCZYKCrPWXCEhieQUpSdQVWuPb7cB1mZfmndJSt3ajprAel5lKiAcb6PK40JmckmW+KQXz4z1UGQM65KLefq\\\\\\\/N\\\\\\\/nVfnHwSiJo1tES16bYxWrLi8FYu7uiHIchwavVKHKIODqjmL7dWkbkLts4qB6ooiugEQ2On60C05+F0FA9IRwzYfY9RZfbLdInCOD0UxYdtN4Sag91xbWLoCOqh4Px5DpQpGjIVlh8gytkhO6Q\\\\\\\/mmQdagS1YHXkwBuImcq+eDT8msvTdQGyZw4FAY8m7\\\\\\\/e23t8C0hSvTsqA0kCv+fLx9RoTD\\\\\\\/5O+h31wX\\\\\\\/Xt5tVCZlA\\\\\\\/wke4bUeb4AcXvX609gdMrqmvcLqxv0AwbS18KMTDnn500yz0yJi46\\\\\\\/hr3ltm6avA+Z0IjyHwLLl\\\\\\\/H0Hb44HBTQ9m99bSRzt4wMVMRZUejZIzMEY0eM36O00cYODA8Q5iplUHol3YxPnXyBSGi35Nz3+ant95XyrTVFw2CbSQ\\\\\\\/loy5WI+UG++OqEJpe5qr\\\\\\\/cQUIcTzWvf0QZGdPFEv5XZa3ZFC2LzLKYStEvY8+TrXHnMTzJr7+YI8pChuf46HuXtTIrwsPL4KALZfF+tMOd3GtdeZMZO96RC6w6T1rLU7NRgz3s\\\\\\\/FpAgRuego2PTNi8WXWqVcBZGmqrQhzw4+cSdCyBoZTH3bcHteLz7a4U4gjNNnqP1oKUPlPHNPZee5SIP9a9LP3E9Cjb0Qh8pXB9mQAWN0i1gWvtOrRPLUpbZlQ2kLh8znKKmisI4SgD7RyWbYDY3yIFZVUcjimJq7YunKEsSTvsYGQgcWhWAAUT18ESeSzlcUwD7LcmplUhkeQxPtUO4kUbzKW9OzBl2mPsdtnGWw+9sHqF4yuDDIJACqvcfEgs8Dg1kM3oUBj52BMo4M25HSNacSwlP0REMVpzHV5sGmfksIkTmiqC1OFSKtdxYHlaLAm0pOaNjqA7QjwWpCmwVgWNw\\\\\\\/rHuHIpGezvAac++hZL54RuE9mYUyqDhAqk5lA8QFcd22YllZE33R\\\\\\\/dHyIiXKFsIP\\\\\\\/roUCrXC339ZRopvLOU6g5+MTrbYZDKiP9\\\\\\\/bVdgMLnwba6Fv5arUEetkoTmystfhW21nAGrFpEzs\\\\\\\/lWOXGODgXuoJxvOHJpQl5TJCkYU5hkcucZWU47g6LZEYXLlWV0gwTTJciicVAOB92Y5RkD2uwtJmC5Ct\\\\\\\/2c6Sj5vqmb7mkNH7vgkujJAnUOtXff3RpcQuCDjjBWHwU+Z9ovsPVJrNWGSo3ri+qrJLXLxdv5nUDGEyVq4be5POxh8Fzigeh4e4HuUpMkuMzaNzOZpFGsVypXNbWI2CM\\\\\\\/QvLvYP2gCyT6de8jquK2PAeAjnjczl3cV47ZK8qBWLi+FgQhxmDfcCQxFCGhuLlC5GdQNVVXjYcv+PVSNAw41VgIwtC3WKkjffS1VO\\\\\\\/15and1TZ06o3YiJUYIfd32+PXSu00fqXyxKJf2c9GE47fa0Yva92laimlL+r\\\\\\\/VD5qp1V+QRBN7nRIGzilTa3KEkLhzNTccY\\\\\\\/+VW58yhbkSp2MHnMPimwkSCz9ybEauTdR9+TQCzbrU0BzSh\\\\\\\/OF0HT3KtsJ0ItVhwC8iVNSVN8S\\\\\\\/IwBao3zOfFuFDK6eJf2EyANa5KC6GdFKxvE4Qp\\\\\\\/jcuojh5kQ75D5Q5mBSkVr1at8uqOjoOoIAPb6tzDZa+V7MoqKt4H4BglvuHmx5aoX7KJLTBomwwkehjGlKzZj6TAIlPf\\\\\\\/XoxWfDrNbEItY1yY19H6mSZhFHq5EHKswdW5\\\\\\\/IOlMVpQh584JBtjhvekVY8SQmAKiR1R5mojp1NEsxMBK5ZjO4XaOaX\\\\\\\/4A7V\\\\\\\/OmDxJ1Eb61+wZ4t\\\\\\\/8Ez4aHpz5PtNINHTMxgOEq\\\\\\\/ahngivfDz7WgHQXApX+eu0EKrpKrCjqPM0r68k+4tmxO8KJ\\\\\\\/LboRD8JMvIn7kj7xEfPuUdN0pGOnGF1xH5TcOIHCLer0OtvKncm4QHui7W04l9qx2cQgOzRUJ8okjNFToI45MprAv154BpudCw24YKIXn2SLZjsISXJn3YDN5qnFjVNsPwUM\\\\\\\/eEiDaIDXNEUlKTIotd1W2qTvOigl0rqwxZ0xWYDGVIjUCdCFWB185iHdWMawP\\\\\\\/6rmhaCDMWWHQQlcIK98S77aYOR0HRjrk8aJQ7kJ4I2Kfiqnl3Y15jlYKdtCoM0BVSLzmZshb34KxMqkyj0t5jMn2D5ngENXdvXMTA9luueM6o4j7XcoN7aLrC4jGv\\\\\\\/al7KnQwQZu+Ba6OsvgTo4BzYopepBNFQVOvq9xtveBPO0MlEoALlpYzdnEkwFKtOOtIpMCNAVj0n4O+Xoh41DtHpRXnryleY3n93NWg0ABram67u5xqSrq4uV3DyVy9Ko43GD86vtTzwx6NmOes2Bl79OsoZiPBNStYNAk8QCnolwUh\\\\\\\/drdwGrt3BDQO909LeSy3s9c06EAabGkhiP7AZovGy+eMYqz+m4wkmNjvbXI0QwdWwsp0rSFBFTznChN6zjKz6fhPsOfLiRIFepLc3kAgpVA2\\\\\\\/PNvhJEZKrAyCDyv8eqN7harjjLsRXzoT6EgnR+kruvA2h3\\\\\\\/zKo7fZpnYkWQVkc9W7+C4LhzSeAmb902eUjuLJ5RhRG8X7ek6fNOJK5\\\\\\\/vq6GEcKiD\\\\\\\/JH1NhCZrTSOmhNbpMjfjSHkqUjp3MdYezCUHj\\\\\\\/pFx4oW1uuOD8rIk6YBwiwStdtj6GCoFfZcBXMrcfD+ev\\\\\\\/BOcd44yDfk9wGlPk9Si47YFSDPPYsMJ4HQeHGhx0hwkMXXE5YyN9\\\\\\\/TpATYT7Oos4hY7GWjgmTaKLyZ4nSKx1RlH+AZS7He2irvfzKf7RlqTlPeAgXTASL6vejnkti3pQgVn7U9EBAcHzWtrpo+EYDkbckgPFx4VQIrCVwe0\\\\\\\/8iX0e3sgR27576yWAvQUI1tO8v7BB3aVQXXgIszLWxLinsreSb7R2dotdFa1xb28F93kWNmergurxrz10fvQlzDHdqlo3b3m0EcyCbt3hRW\\\\\\\/c++HlVtM8OCjBQq9AexvdG4qkVB\\\\\\\/TlA7h968YllmIN3ba11LOJKIyyvra33J+FvBsSfinym81AZtwQLZOpEBT2pCy\\\\\\\/yR\\\\\\\/hMknWKgQExFxP45QStsAI+w723vg8GE\\\\\\\/3rx5\\\\\\\/YeZ4vGjOL+3r+WFbWiJcok9GrFaGLV4DhnGi6GSBCvj9y0LVNsyNDTHVW0jpBNIMWPMqpaP36\\\\\\\/U1w2R82Eqaqp44o3x5tsz3oo1R2kkiNW7wjM3A9NoEsN21A2GZKlJPg1sNE\\\\\\\/gjPS\\\\\\\/a4apW+vRXSfXJ7V9aam2binVDQDHHOzXC38sy4YfxTJuCYtvSzGEyLQgtvxUdrPn1xD7vQ9EFnYOHEym5HhsxBT3G2bjb4b5ZkDlZLJi5dFDp4kAuW+iwLyl47DPtiDHqczx36p41x3xa8Uv4Afro3j7QGLKx\\\\\\\/pd8VtVMOUkpYfjPR1+bvQKmWuAngkJCjwnRFNSZO9PubC8kYUsHcImO9pC0lnrBk8s95BXF6SxzXdLVkaHvnhMYJpI7O2ekHdridGwobL3UqwK27YnedWc3Ti8C3cAUSfH7bGI4DTXh15cJx+dMwQCom8teiHVtdAqyk\\\\\\\/zoMbRVNoVHMjIpQCGXE0ae6uZMw\\\\\\\/2rqCpU10xgGl1bw4VQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022erj7Jf6eOwyFgl54XbQIYQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b732b9cd764780acc3f25631abbf614ceac1381aef4fb26be1becf6192177ba3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229+s1vOhae0rDMIUqA9ole\\\\\\\/Er7n99svx3iuLHAa2HwlEwF44onJks2Fb7ErjYWsvFTy81+LQiVUV31Qilh43XceX0L0Xn\\\\\\\/u4tdPIefkkWWI1sg5TXVpXPJ+SYDKXgbHsZAKx4MJ10bo3rNyyqhxPh6vLZ17G+oa0MBkr9pG+fKnkh6ykOrmvVSEXbCt3r85N+uWEV5o89uALzdgYHzLdqsW1lDuKqVW2TJBhkudfFWJOFW184p3zeJcWitZH7Lzj7Oh8\\\\\\\/2fB6+doScOtVeD8H\\\\\\\/diDzzePQAPGKhxdwCZLZZOtqYBXgeorM9A4u7vFoHMvUovF3BDOaw3lKRAIogkQFYpqW5TQy0xvrGOVSdgUmIa37mBPL79h8yF9zlPyXobApwCpbkIq+JAcbG0BSPtENBpHKcg5Z1CoXVwYbtMoG7Zy8eGKwKzFVelCCSf7\\\\\\\/fMlqamnTuJPeiXQRM1T5do6vt9LrLAGAsL\\\\\\\/Lyg9rVhDH\\\\\\\/VykY5CE0awH3Zz+mON8HvblDIRmP0UgtvoZvHA+zy3ZCx2Tcx\\\\\\\/9osExvjumHTdUzIChpdmXouZMU9IiAZdlfDyCCHU5DKzG+cIyHkZ4ADddOD+s62oSt1gdB7gWIvNnu+Y07AQwgEy+2vKKMOG+\\\\\\\/dEdwVU9HvWyyKAHmTAjhm6MVcHp9wcqVqU3QTw+Vl2p0EZdiNYahuNxc9wqniZ5i6x3jpqAxToTvIhwbY15UDIIl\\\\\\\/2LiRIPkJhbBrHvSmNNBAZ29C8EeG4PK\\\\\\\/FXLgPUEaNzSVK05BzXXfRg+2LOXieOp28M49obLwwdAOs4BkWzHMfepXpICnCUtxaVNdaBkaWbGfjvNr\\\\\\\/sy6TAb9ZTN572\\\\\\\/MbaYpvgRdN6R7mQHLGeL1NBVNHSd2CpTrNZEWPvpRZevD\\\\\\\/l3tCi7bdeX+r+850nNlOiopgPCKDZ4ni5\\\\\\\/dJVbeQ7JvGqLwTXVvlehqL2g\\\\\\\/Mo37R8NjEkZ\\\\\\\/M9uXhkCtweNuG61MY1V7GbrWlYQ\\\\\\\/1dbnpiHw+OzJOZie33kPC0Uoz1stWJcGHYuFXgYwjLwZ56b6w73FJykF6vFs55\\\\\\\/vdPKBH76I8dawunucssf6R13WJKsTd2NhIvxH2jkMU5QPocSf7AVoER4wop+t7oloWpsg4hr4OUEQgYoSfG40vj03YlgulxEmD9kPBWIBncX64AR5PBspi8smAs6b7dVnA8MGIk6ptPudOiQQmhTTTdLSjC913SEGBOY\\\\\\\/ecsd98FAO8snk7s\\\\\\\/Du98gpl3bhVySfiuKlDzMy59kN31lQdGZttKltLKLUHZriHzEv5hlfSclawBjOeh5MHNgr6cIR4qatcNubt7HeEUSoWXAOPyYzhoDV7YCzeOT\\\\\\\/5pJGqOb+i2rXVhBMgMGu2sLfVSDYVhKtSgWtpCEoPskmA1vBc7+2JwfkBeUZpCahHvYWvfwOy9OTJPpVIRBm72rCai2osoq8zAUwm8ky5bUNJ2YUKWphQLMSS3c2NU2X1fBPneN4V+nqCS5kgTtOp5fSiqcMMx6M69DPvUm75ha2KbXTk5wKDnVh2sf2WJQSQdtlfndu9MGdmbaLE+aTxarJYNJWd9w32siW89kgfZr6I5HAeTio6e1e0+v3T0KbC3512FmhXQFVSAruVA1l6e8HtwgaFnjNLVebPaYsbZERyeykyw2xTbZc1B5+fAraqge8pd1P29si\\\\\\\/JxR827hwMS1pMZMIZ+y82+THtITU\\\\\\\/gZ4fDRBBydoDfylbr2yxE5EzmaloUcrVHJeXv3kZi5eRZR9lRJKjeqqK++6sIEnBC5LxHp\\\\\\\/H8x4MRyz6OPzC3aqT1mgfyb6AuBYX6hp18EBuZSJnwM7UBHkEVy2vdt2Z2NlerfXaFSdHiwFtawsHhA8YXfav2xr+tyYe9ZhRDf+oWpHFiPKVLyovJfr2+0N+tVhLedEF6XENn4n\\\\\\\/Kkt1XWGH8ucXIrJgIjMDw66dioiv9dMxISC3ogGZ6ouFzfZtCer69nxVHsiYyc9gtZbtOCGPOUG2R+OWXjA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022YkV8w\\\\\\\/5+HeNdumZ+OS87sg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a720c937c7abcbf5b05cac34bbf544a0bc3034fa22cd8225aa85ea3b64bc1fe0\u0022}"},"ajax":{"edit-submit--5":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484776 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1995" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-slGzI7jsAGfvk8yER6daXjZDcoZ9VNV2OMB72c4p8uM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1995" method="post" id="highwire-search-quicksearch-form-1" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--3">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--3" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--7" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-W4D6I6ZRMDZzC1T-IGtA90Ovo37abZVh8mHMnC9FkVg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_1" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-3" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1985" title="HDAC Inhibitors Disrupt Programmed Resistance to Apoptosis During Drosophila Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2015" title="Insight into the Recent Genome Duplication of the Halophilic Yeast Hortaea werneckii: Combining an Improved Genome with Gene Expression and Chromatin Structure" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484776" id="node484776" data-pisa="ggg;7/6/1995" data-pisa-master="ggg;g3.117.042341" data-apath="/ggg/7/6/1995.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic-Enabled Prediction in Maize Using Kernel Models with Genotype × Environment Interaction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7887-9543" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Massaine Bandeira e Sousa</span>, <span class="highwire-citation-author" data-delta="1">Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="2">Evellyn Giselly de Oliveira Couto</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Diego Jarquín</span>, <span class="highwire-citation-author" data-delta="5">Roberto Fritsche-Neto</span>, <span class="highwire-citation-author" data-delta="6">Juan Burgueño</span> and <span class="highwire-citation-author" data-delta="7">Jose Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1995-2014;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042341 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484776" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Massaine Bandeira e Sousa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, 13418-900 Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Massaine%2BBandeira%2Be%2BSousa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bandeira%20e%20Sousa%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMassaine%2BBandeira%2Be%2BSousa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7887-9543" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Massaine Bandeira e Sousa</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University of Quintana Roo, 77019 Chetumal, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Evellyn Giselly de Oliveira Couto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, 13418-900 Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evellyn%2BGiselly%2Bde%2BOliveira%2BCouto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Oliveira%20Couto%20EG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvellyn%2BGiselly%2Bde%2BOliveira%2BCouto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Colegio de Postgraduados, CP 56230 Montecillos, 56230 D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Agronomy and Horticulture Department, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Roberto Fritsche-Neto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, 13418-900 Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BFritsche-Neto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritsche-Neto%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoberto%2BFritsche-Neto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/6/1995" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484776" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484776/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1995.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484776" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484776/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1995.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484776" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484776/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/6/1995.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484776" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484776/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Multi-environment trials are routinely conducted in plant breeding to select candidates for the next selection cycle. In this study, we compare the prediction accuracy of four developed genomic-enabled prediction models: (1) single-environment, main genotypic effect model (SM); (2) multi-environment, main genotypic effects model (MM); (3) multi-environment, single variance G×E deviation model (MDs); and (4) multi-environment, environment-specific variance G×E deviation model (MDe). Each of these four models were fitted using two kernel methods: a linear kernel Genomic Best Linear Unbiased Predictor, GBLUP (GB), and a nonlinear kernel Gaussian kernel (GK). The eight model-method combinations were applied to two extensive Brazilian maize data sets (HEL and USP data sets), having different numbers of maize hybrids evaluated in different environments for grain yield (GY), plant height (PH), and ear height (EH). Results show that the MDe and the MDs models fitted with the Gaussian kernel (MDe-GK, and MDs-GK) had the highest prediction accuracy. For GY in the HEL data set, the increase in prediction accuracy of SM-GK over SM-GB ranged from 9 to 32%. For the MM, MDs, and MDe models, the increase in prediction accuracy of GK over GB ranged from 9 to 49%. For GY in the USP data set, the increase in prediction accuracy of SM-GK over SM-GB ranged from 0 to 7%. For the MM, MDs, and MDe models, the increase in prediction accuracy of GK over GB ranged from 34 to 70%. For traits PH and EH, gains in prediction accuracy of models with GK compared to models with GB were smaller than those achieved in GY. Also, these gains in prediction accuracy decreased when a more difficult prediction problem was studied.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genotype%C3%97-environment-interaction-g%C3%97e" class="hw-term hw-article-keyword hw-article-keyword-genotype&#xD7;-environment-interaction-g&#xD7;e" rel="nofollow">Genotype× Environment interaction (G×E)</a></li><li class="kwd"><a href="/keyword/genomic-best-linear-unbiased-predictor-gblup-linear-kernel" class="hw-term hw-article-keyword hw-article-keyword-genomic-best-linear-unbiased-predictor-gblup-linear-kernel" rel="nofollow">Genomic Best Linear Unbiased Predictor (GBLUP) linear kernel</a></li><li class="kwd"><a href="/keyword/gaussian-nonlinear-kernel" class="hw-term hw-article-keyword hw-article-keyword-gaussian-nonlinear-kernel" rel="nofollow">Gaussian nonlinear kernel</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Genomic selection (GS) arose from the need to improve the prediction of complex traits based on dense marker information. It was first proposed in animal breeding (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Meuwissen <em>et al.</em> 2001</a>), and then applied to plant breeding (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">2011</a>). Using GS, genomic breeding values are estimated as the sum of marker effects for unphenotyped individuals in the testing population. Several genomic prediction models were developed and applied to simulated and real plant breeding data (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bernardo and Yu 2007</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Massman <em>et al.</em> 2013</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">2013</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pérez-Rodríguez <em>et al.</em> 2013</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beyene <em>et al.</em> 2015</a>). In general, these studies showed good prediction accuracies for complex traits evaluated by means of random cross-validation (CV) partitions of the data.</p><p id="p-4">Genomic-enabled prediction models were originally developed for a single trait in a single environment. However, multi-environment plant breeding trials are routinely conducted to estimate and take advantage of genotype × environment interaction (G×E). Therefore, to implement GS strategies in plant breeding, G×E needs to be estimated, modeled, and predicted. When genetic marker information is used for computing associations between individuals through the Genomic Relationships Matrix (GRM) G (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Habier <em>et al.</em> 2007</a>), this model is also referred to as Genomic Best Linear Unbiased Predictor (GBLUP) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">VanRaden 2007</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">2008</a>). <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Burgueño <em>et al.</em> (2012)</a> extended the GBLUP methodology to incorporate and model G×E effects. The Bayesian model of <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> (2014)</a> is another GBLUP extension that introduces main and interaction effects of markers and environmental covariables via covariance structures. <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Heslot <em>et al.</em> (2014)</a> proposed using crop modeling for assessing genomic G×E. In general, studies have shown that modeling G×E can give substantial gains in prediction accuracy (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Heslot <em>et al.</em> 2014</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> 2016a</a>,<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">b</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> 2016a</a>,<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">b</a>).</p><p id="p-5">Recently, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a> proposed a GBLUP prediction model that explicitly models G×E and marker × environment interaction (M×E) where marker effects and genomic values are partitioned into components that are stable across environments (main effects) and others that are environment-specific (interactions). The model of <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a> has advantages and disadvantages; the advantages are: (i) it can be easily implemented using existing software for GS, for example, BGLR (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">de los Campos and Pérez-Rodríguez 2016</a>); and (ii) it can be implemented using both shrinkage methods (a ridge-regression type estimator) and variable (marker) selection methods. In this case, the M×E model can be employed not only for GS but also for genome-wide association analyses to identify genomic regions that contribute to stability and to interaction effects (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2016b</a>). Furthermore, in terms of reducing the models’ residual variance, the M×E model outperformed the more traditional single-environment and across-environment models for complex traits (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> 2015</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2016b</a>). However, the M×E model is more efficient when applied to sets of environments that have positive correlations. This limitation arises because the genetic covariance between any pair of environments is represented by the variance of the main effect, which forces the covariance between pairs of environments to be positive (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> 2015</a>).</p><p id="p-6"><a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">VanRaden (2008)</a> first suggested models using a standard linear kernel, where the GBLUP is a linear model characterized by parameters related to additive quantitative genetics theory. However, complex traits are affected by nonlinearity effects between genotypes and phenotypes due to complex interactions among genes (<em>i.e.</em>, epistasis) and their interaction with the environment. <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gianola <em>et al.</em> (2006)</a> proposed nonparametric and semiparametric methods to model the relationship between the phenotype and markers that are available within the GS framework. The nonparametric methods are capable of accounting for small complex epistatic interactions without explicitly modeling them.</p><p id="p-7">Therefore, the semi-parametric Reproducing Kernel Hilbert Space (RKHS) reduces the dimensions of the parametric space and also captures small complex interactions among markers (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gianola and van Kaam 2008</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">de los Campos <em>et al.</em> 2010</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Morota and Gianola 2014</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gianola <em>et al.</em> 2006</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">2014</a>). The RKHS method uses a kernel function to convert the marker matrix into a set of distances between pairs of individuals (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Heslot <em>et al.</em> 2012</a>). Recently, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jiang and Reif (2015)</a> formulated a model with explicit epistatic effects of markers and proved that this model is equivalent to RKHS with Gaussian kernel, thus demonstrating that this model captures epistatic effects among markers. Several studies confirmed the advantage of using RKHS regression to increase prediction accuracy by capturing nonadditive variation (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">de los Campos <em>et al.</em> 2010</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Morota <em>et al.</em> 2013</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Pérez-Rodríguez <em>et al.</em> 2013</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Morota and Gianola 2014</a>).</p><p id="p-8">Recently, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> (2016a)</a> compared methods that applied the G×E interaction GS model of <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a> using a linear kernel (GBLUP) and a nonlinear Gaussian kernel with the bandwidth estimated by an empirical Bayes method proposed by <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pérez-Elizalde <em>et al.</em> (2015)</a>, and a Kernel Averaging method, or multi Gaussian kernels (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">de los Campos <em>et al.</em> 2010</a>). <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> (2016a)</a> evaluated these methods using single-environment and multi-environment G×E interaction models to show the higher prediction ability of the Gaussian kernel models with the G×E model <em>vs.</em> the linear kernel with the G×E model. The most flexible Gaussian kernels captured major and complex effects of markers in addition to their interaction effects.</p><p id="p-9">In genomic-enabled prediction, linear models consider genetic values as linear combinations of marker effects; therefore, GBLUP with G×E can also be fitted with the nonlinear Gaussian kernel. Similarly, the model of <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> (2014)</a> can also be used with the linear GBLUP kernel, and with the nonlinear Gaussian kernel (GK). However, so far, the model of <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> (2014)</a> has not been used with the GK, and its prediction accuracy has not been compared with the G×E with the GBLUP kernel of <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a>, or with the model of <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> (2014)</a> across environments, or when including G×E with the GBLUP kernel.</p><p id="p-10">Therefore, the main objectives of this research were: (1) to study the prediction accuracy of the multi-environment, single variance G×E deviation model (MDs) of <a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> (2014)</a> used with the GK method (MDs-GK) and the prediction accuracy of the multi-environment, environment-specific variance G×E deviation model (MDe) of <a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a> fitted with the GK method (MDe-GK), and to compare them with the prediction accuracy of their counterpart models using the linear kernel GBLUP (GB) method (MDs-GB and MDe-GB), and (2) to compare the accuracy of the four previous models with the accuracy of the single-environment, main genotypic effect model (SM), and the multi-environment, main genotypic effect (MM) of <a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> (2014)</a> using GB and GK methods (SM-GB, SM-GK, MM-GB, and MM-GB).</p><p id="p-11">Two data sets of Brazilian maize hybrids (HEL and USP) genotyped with dense molecular markers, and phenotyped in different environments with different numbers of hybrids per environment and different traits, were used to compare the prediction accuracy of those eight model-methods.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Phenotypic experimental data</h3><p id="p-12">This study considered two hybrid maize data sets. The first data set is from the Helix Seeds Company (HEL), while the second is from the University of São Paulo (USP). The HEL data set consists of 452 maize hybrids obtained by crossing 111 pure lines (inbreds); the hybrids were evaluated in 2015 at five Brazilian sites: Nova Mutum (NM), (13° 05′ S, 56° 05′ W, 460 m above sea level) and Sorriso (SO) (12° 32′ S, 55° 42′ W, 365 m above sea level) in the state of Mato Grosso; Pato de Minas (PM) (18° 34′ S, 46° 31′ W, 832 m above sea level), and Ipiaçú (IP) (18° 41′ S, 49° 56′ W, 452 m above sea level) in the state of Minas Gerais; and Sertanópolis (SE) (23° 03′ S, 51° 02′ W, 361 m above sea level) in the state of Parana. The experimental design was a randomized block with two replicates per genotype and environment. The phenotypic and genomic data on inbred lines are credited to Helix Seeds Ltda Company.</p><p id="p-13">Data from USP consist of 740 maize hybrids obtained by crossing 49 inbred lines. The hybrids were evaluated at Piracicaba and Anhumas, São Paulo, Brazil, in 2016. The hybrids were evaluated using an augmented block design, with two commercial hybrids as checks to correct for micro-environmental variation. At each site, two levels of nitrogen (N) fertilization were used: Ideal N (IN) and Low N (LN) for a total of four artificial environments (P-IN, P-LN, A-IN, and A-IN). The experiment conducted under ideal N conditions received 100 kg ha<sup>−1</sup> of N (30 kg ha<sup>−1</sup> at sowing and 70 kg ha<sup>−1</sup> in a coverage application) at the V8 plant stage, while the experiment with low N received 30 kg/ha of N at sowing. Each plot was 7 m in length, with 0.50 m spacing between rows and 0.33 m between plants.</p><p id="p-14">There was an imbalance in both data sets because not all hybrids were evaluated in all locations (incomplete field trials). The main traits of the two data sets were GY (grain yield in ton per hectare), PH (plant height in centimeters), and EH (ear height in centimeters). Plant height was measured from the ground to the flag leaf and EH from the ground to the base of the ear. The empirical distribution of the three evaluated traits (GY, PH, and EH) were symmetric in most of the environments (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). For the HEL data set, PH and EH were evaluated in three environments. Broad sense heritability (repeatability) was computed using the standard formula based on plot means:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-1.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-2.gif"/></noscript></span></span> is the variance of the hybrids, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-3.gif"/></noscript></span></span> is the variance of the hybrids × location interaction and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-4.gif"/></noscript></span></span>is the residual error variance and <em>s</em> and <em>r</em> are the number of environments and replicates in each environment, respectively.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box plot of grain yield (ton per hectare) (GY), plant height (centimeter) (PH), and ear height (centimeter) (EH) for (A) HEL data set and (B) USP data set. Environments for the HEL data set are IP, Ipia&#xE7;&#xFA;; NM, Nova Mutum; PM, Pato de Minas; SE, Sertan&#xF3;polis; and SO, Sorriso. Environments for the USP data set are: P-LN, Piracicaba low nitrogen; P-IN, Piracicaba ideal nitrogen; A-LN, Anhumas low nitrogen; and A-IN, Anhumas ideal nitrogen." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2060048635" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Box plot of grain yield (ton per hectare) (GY), plant height (centimeter) (PH), and ear height (centimeter) (EH) for (A) HEL data set and (B) USP data set. Environments for the HEL data set are IP, Ipia&#xE7;&#xFA;; NM, Nova Mutum; PM, Pato de Minas; SE, Sertan&#xF3;polis; and SO, Sorriso. Environments for the USP data set are: P-LN, Piracicaba low nitrogen; P-IN, Piracicaba ideal nitrogen; A-LN, Anhumas low nitrogen; and A-IN, Anhumas ideal nitrogen.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1995/F1.medium.gif" width="309" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/6/1995/F1.medium.gif" width="309" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484828" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Box plot of grain yield (ton per hectare) (GY), plant height (centimeter) (PH), and ear height (centimeter) (EH) for (A) HEL data set and (B) USP data set. Environments for the HEL data set are IP, Ipiaçú; NM, Nova Mutum; PM, Pato de Minas; SE, Sertanópolis; and SO, Sorriso. Environments for the USP data set are: P-LN, Piracicaba low nitrogen; P-IN, Piracicaba ideal nitrogen; A-LN, Anhumas low nitrogen; and A-IN, Anhumas ideal nitrogen.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Genotypic data</h3><p id="p-16">The USP and HEL parent lines were genotyped with an Affymetrix Axiom Maize Genotyping Array of 616 K SNPs (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Unterseer <em>et al.</em> 2014</a>). Standard quality controls (QC) were applied to the data, removing markers with a Call Rate ≥ 0.95. The remaining missing data in lines were imputed with the Synbreed package (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wimmer <em>et al.</em> 2015</a>) using the algorithms from the software Beagle 4.0 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Browning and Browning 2008</a>). The hybrid genotypes were obtained by genomic information of the parent inbred lines. Markers with Minor Allele Frequency (MAF) of ≤0.05 were removed. After applying QC, 52,811 and 54,113 SNPs were available to make the predictions in the HEL and USP data sets, respectively.</p></div></div><div class="section other1" id="sec-4"><h2 class="">Statistical models</h2><p id="p-17">Statistical models for genomic predictions taking into account G×E were proposed by <a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> (2014)</a> and <a id="xref-ref-24-8" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a>. The Jarquín model incorporates genetic information from molecular markers or from pedigree (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pérez-Rodríguez <em>et al.</em> 2015</a>), and/or from environmental covariates, whereas the López-Cruz model decomposes the marker effect across all environments and for each specific environment (interaction).</p><p id="p-18">In this study, four statistical prediction models were fitted to both data sets to study their prediction accuracy using random CV schemes (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). We also compared the prediction accuracy of two kernel regression methods in the four models. The models were: a single-environment, main genotypic effect model (SM), a multi-environment, main genotypic effect model (MM) (<a id="xref-ref-22-9" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> 2014</a>), a multi-environment, single variance G×E deviation model (MDs) (<a id="xref-ref-22-10" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> 2014</a>) and a multiple-environment environment-specific variance G×E deviation model (MDe) (<a id="xref-ref-24-9" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> 2015</a>). The two kernel regression methods were: the linear kernel GBLUP (GB) method used by <a id="xref-ref-22-11" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> (2014)</a> and <a id="xref-ref-24-10" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a>, and the Gaussian kernel (GK) method proposed by <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> (2016a)</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484987/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484987/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Model, kernel method, and abbreviation of the model-method combination</span><div class="sb-div caption-clear"></div></div></div><div id="sec-5" class="subsection"><h3>The single-environment, main genotypic effect model (SM)</h3><p id="p-20">The SM model fits the data from each environment separately and takes into account the main effect of genotypes. In matrix notation the model is written as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-9.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-10.gif"/></noscript></span></span> is the response vector, and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-11.gif"/></noscript></span></span> represents the observations in the <em>i</em>th line <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-12.gif"/></noscript></span></span> in each environment; <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-13.gif"/></noscript></span></span> is the general mean; <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-14.gif"/></noscript></span></span> is the incidence matrix that connects the random genetic effects with phenotypes; <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-15.gif"/></noscript></span></span> is the random genetic effects for each environment, and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-16.gif"/></noscript></span></span> the residual random effects for each environment. The SM model (1) assumes that the distribution of the <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-17.gif"/></noscript></span></span> vector is multivariate normal with mean zero and a covariance matrix <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-18.gif"/></noscript></span></span> that is, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-19.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-20.gif"/></noscript></span></span> is the genetic variance component of <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-21.gif"/></noscript></span></span> in the <em>j</em>th environment and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-22.gif"/></noscript></span></span> is a symmetric, positive semi-definite matrix, denoting the variance–covariance of the genetic values constructed from the genomic molecular markers. It is also assumed that the errors <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-23.gif"/></noscript></span></span> in each environment are independent with homogeneous variance, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-24.gif"/></noscript></span></span> and distributed as <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-25.gif"/></noscript></span></span> (where <strong><em>I</em></strong> is the identity matrix, and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-26.gif"/></noscript></span></span> is the residual variance in the <em>j</em><sup>th</sup> environment). Therefore, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-27.gif"/></noscript></span></span> is an approximation of the true unknown genetic values, and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-28.gif"/></noscript></span></span> captures the residual genetic effects that were not explained by <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-29.gif"/></noscript></span></span> plus other nongenetic effects that approximate the errors. Note that matrix <strong><em>K</em></strong> may have different dimensions depending on the number of lines evaluated in each environment.</p></div><div id="sec-6" class="subsection"><h3>The single-environment, main genotypic effect model with GBLUP (SM-GB) and Gaussian kernel (SM-GK)</h3><p id="p-21">For SM model (1), matrix <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-30.gif"/></noscript></span></span> can be constructed using the linear kernel <strong><em>K</em></strong> = (<strong><em>XX</em></strong>′/<em>p</em>) (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">de los Campos <em>et al.</em> 2013</a>) proposed by <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">VanRaden (2007</a>, <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">2008)</a> for estimating the GBLUP, where <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-31.gif"/></noscript></span></span> is the standardized matrix of molecular markers for the individuals, of order <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-32.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-33.gif"/></noscript></span></span> is the number of markers (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-22">When markers have a more complex function, the GK has proved to be more effective for capturing complex cryptic interactions between markers, thereby improving the prediction accuracy of the model (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> 2016a</a>). The entries of the Gaussian kernel are computed as <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-34.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-35.gif"/></noscript></span></span> is the Euclidean distance between the individuals <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-36.gif"/></noscript></span></span>th and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-37.gif"/></noscript></span></span> given by the markers; <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-38.gif"/></noscript></span></span> is the bandwidth parameter that controls the rate of decay of <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-39.gif"/></noscript></span></span> values (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pérez-Rodríguez <em>et al.</em>, 2012</a>; <a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> 2016a</a>). In this study, GK takes the form <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-40.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-41.gif"/></noscript></span></span> and the median of the distances is used as a scaling factor (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2010</a>). <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">de los Campos <em>et al.</em> (2010)</a> described the theory of the Gaussian kernel in the context of the kernel averaging method for the RKHS.</p></div><div id="sec-7" class="subsection"><h3>The multi-environment, main genotypic effect model (MM)</h3><p id="p-23">One multi-environment model considers the main fixed effects of environments, as well as the random genetic effects across environments<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-42.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-43.gif"/></noscript></span></span>is the response vector and <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-44.gif"/></noscript></span></span> is the vector of observations of the lines <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-45.gif"/></noscript></span></span> in the <em>j</em>th environment <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-46.gif"/></noscript></span></span> The fixed effects of the environment for the data used in this study are modeled with the incidence matrix of the environments <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-47.gif"/></noscript></span></span> where the parameters to be estimated are the intercept for each environment (<span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-48.gif"/></noscript></span></span>) (other fixed effects can be incorporated into the model), and incidence matrix <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-49.gif"/></noscript></span></span> connects genotypes with phenotypes for each environment. It is assumed that the random vector of genetic effects <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-50.gif"/></noscript></span></span> across environments follows a multivariate normal with mean zero and a covariance matrix <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-51.gif"/></noscript></span></span> that is, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-52.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-53.gif"/></noscript></span></span> is the variance of the main genetic effects across environments, and <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-54.gif"/></noscript></span></span> In this case, the resulting model is equivalent to the across-environment GBLUP model of <a id="xref-ref-24-11" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a> and <a id="xref-ref-22-12" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> (2014)</a>. When in the MM model of (2) <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-55.gif"/></noscript></span></span> is used with <strong><em>K</em></strong> = (<strong><em>XX</em></strong>′/<em>p</em>) (GBLUP), the model is the GBLUP across environments (MM-GB), whereas if <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-56.gif"/></noscript></span></span> is used with the GK, the model is the GK across environments (MM-GK) (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-8" class="subsection"><h3>Multi-environment, single variance G×E deviation model (MDs)</h3><p id="p-24">This model extends (2) by adding the random interaction effect of the environments with the genetic information of the lines <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-57.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-58.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>It is assumed that the vector of random effects of the interaction <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-59.gif"/></noscript></span></span> has a multivariate normal distribution, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-60.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-61.gif"/></noscript></span></span> is the Haddamar product operator and denotes the element-to-element product between two matrices in the same order (<a id="xref-ref-22-13" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> 2014</a>), <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-62.gif"/></noscript></span></span> is the variance component of the interaction, the <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-63.gif"/></noscript></span></span> matrix is defined as before, and <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-64.gif"/></noscript></span></span> is the vector of the main genetic effects that follows a multivariate normal with mean zero and a covariance matrix <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-65.gif"/></noscript></span></span> that is, <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-66.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-67.gif"/></noscript></span></span> is the variance of the main genetic effects and <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-68.gif"/></noscript></span></span></p><p id="p-25">Therefore, under these conditions<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-69.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-70.gif"/></noscript></span></span> represents the kernel constructed from the molecular markers of the lines in the <em>j</em>th environment. Just as in model (2), matrix <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-71.gif"/></noscript></span></span> is used in the variance–covariance for <strong><span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-72.gif"/></noscript></span></span></strong> of model (3), and is also a component of the variance-covariance of <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-73.gif"/></noscript></span></span> Then, kernel matrix <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-74.gif"/></noscript></span></span> can be constructed using the linear kernel (MDs-GB) or the GK (MDs-GK) (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-9" class="subsection"><h3>Multi-environment, environment-specific variance G×E deviation model (MDe)</h3><p id="p-26">The model of <a id="xref-ref-24-12" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a> separates the genetic effects of markers into the main marker effects across all environments and the specific marker effects in each environment. Thus, this model in matrix notation is<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-75.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>For the MDe model (4) of the vector, <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-76.gif"/></noscript></span></span> represents the main effect of markers (across all environments) with a variance–covariance structure similar to those used in models (2) and (3), that is, <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-77.gif"/></noscript></span></span> However, as pointed out by <a id="xref-ref-24-13" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a> and <a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> (2016a)</a>, <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-78.gif"/></noscript></span></span> is common to all <em>s</em> environments, and the borrowing of information among environments is generated through kernel matrix <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-79.gif"/></noscript></span></span> On the other hand, <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-80.gif"/></noscript></span></span> represents the specific effect of the markers in environments (or the effects of the interaction) with a variance–covariance structure different from model (3), that is, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-81.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-82.gif"/></noscript></span></span> is:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-83.gif"/></noscript></span></span>Note that matrix <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-84.gif"/></noscript></span></span> can be expressed as the sum of <strong><span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-85.gif"/></noscript></span></span></strong> matrices, and the effects given by <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-86.gif"/></noscript></span></span> are specific for the <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-87.gif"/></noscript></span></span>th environment; thus, it has a normal distribution with mean and variance equal to <strong>0</strong>, except for the <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-88.gif"/></noscript></span></span> environment, which has a variance–covariance matrix <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-89.gif"/></noscript></span></span> The importance of these two terms (<span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-90.gif"/></noscript></span></span>) of MDe model (4) is given by the corresponding variance components that are estimated from the data. Kernel matrix <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-91.gif"/></noscript></span></span> is used in the components of <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-92.gif"/></noscript></span></span> while kernel matrix <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-93.gif"/></noscript></span></span> is used in the component of <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-94.gif"/></noscript></span></span>both, <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-95.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-96.gif"/></noscript></span></span> can be used with linear kernel (MDe-GB) or with the Gaussian kernel (MDe-GK) (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-10" class="subsection"><h3>Estimating variance components using full data analyses</h3><p id="p-27">The four models—SM, MM, MDs, and MDe fitted with GBLUP and GK methods—were used on the entire HEL and USP data sets for all the traits. To fit the models, the phenotypic data were centered and standardized (<em>i.e.</em>, each phenotypic data point was centered by subtracting the overall mean of all environments and standardized by dividing by the sample standard deviation across all environments). These analyses were performed to derive estimates of variance components. The posterior variance components resulting from the residual effects, genetic main effect, and genetic environment-specific effects of the four models described above for three traits and for environments in HEL and USP data sets were computed and reported. Since the data were standardized for GBLUP method, the summation of the variance components approximates 1.</p></div><div id="sec-11" class="subsection"><h3>Assessing prediction accuracy by random CV</h3><p id="p-28">The prediction accuracy of the single-environment model-methods was assessed using 50 random partitions, organized in five folds with 10 random partitions each, with 80% of the hybrids comprising the training (TRN) set, and the remaining 20% of the individuals comprising the testing (TST) set. This was performed separately in each environment, as the single-environment models were fitted separately for each environment. For this validation procedure, all the parameters of the models (including variance components resulting from residual effects, and genetic effects) were re-estimated from TRN data in each of the TRN-TST partitions</p><p id="p-29">For the multi-environment models, the prediction accuracy of the model-methods for predicting the pattern of missing values in each environment was generated using two different CV designs (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Burgueño <em>et al.</em> 2012</a>). The random CV1 design mimics the prediction problem faced by breeders when newly developed lines have not been evaluated in any environment; in this case, 20% of the lines were not observed (unphenotyped) in all the environments and had to be predicted. The random CV2 design mimics a prediction problem when lines are tested in incomplete field trials (sparse testing), where some lines are evaluated in some environments but not included in other environments. For these approaches, we assigned 80% of the observations to TRN and 20% to TST. None of the lines to be predicted in the TST are in the TRN set. For this validation procedure, all the parameters of the models (including variance components resulting from residual effects, genetic main effects, genetic × environment interactions effects, and environment-specific effects) were re-estimated from TRN data in each of the 50 random TRN-TST partitions. Note that CV2 design can only be applied to multi-environment model-methods (MM, MDs, and MDe), but not to single-environment model-methods (SM). Also, for SM the random CV is a CV1 but applied to only one individual environment (site).</p><p id="p-30">For each TRN-TST partition, models were fitted to the TRN data set, and prediction accuracy was assessed by computing Pearson’s product–moment correlation between predictions and phenotypes in the TST data set within environments. The same TRN-TST partitions were used to assess the prediction accuracy of each of the models. Thus, 50 correlations were computed for each model and trait; the mean and SDs of these 50 correlations are reported.</p><p id="p-31">Therefore, adjusting models for full data, making inference on the parameters, and assessing predictions in TRN-TST partitions were based on 30,000 samples collected from the posterior and predictive distributions after discarding 5000 samples as burn-in.</p></div><div id="sec-12" class="subsection"><h3>Software</h3><p id="p-32">The aforementioned models can be implemented using the R (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">R Core Team 2016</a>) package Bayesian generalized linear regression (BGLR) (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">de los Campos and Pérez-Rodríguez 2016</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Pérez-Rodríguez and de los Campos 2014</a>). Appendix A gives the R codes used in the models included in this study when the numbers of individuals in different environments vary. It should be pointed out that, in previous studies using the <a id="xref-ref-22-14" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> (2014)</a> and <a id="xref-ref-24-14" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a> models, the number of entries in each environment was the same; thus, the model fitting and the random CV partition scheme were simpler than in this case, where there are different numbers of entries in different environments.</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-33">The phenotypic and genotypic data for the maize hybrids included in this study can be found at <a href="http://hdl.handle.net/11529/10887">http://hdl.handle.net/11529/10887</a>. Each HEL and USP data set contains the data corresponding to each GB and GK kernel, as well as phenotypic data for each trait (PH, EH, and GY) in each environment.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Descriptive statistics</h3><p id="p-34">Box-plots of GY, PH, and EH in each environment for the two data sets are depicted in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>. Most of the distributions of the traits in environments had a symmetric distribution. For the HEL data set, environments NM and PM produced better GY, while environment SE had the lowest GY (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). Traits PH and EH showed similar trends, but were measured in only three environments: IP, PM, and SE. Data from USP had environments with soil nutritional stress, and the results showed that environments with ideal nitrogen conditions (P-IN and A-IN) had better potential for all traits (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>).</p><p id="p-35">All pair-wise environments for all traits in the HEL and USP data sets had high and positive correlations (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). As expected, for PH and EH, the correlations were higher than for the complex trait GY. In all environments, GY, PH, and EH showed high heritability in the HEL data set: GY ranged from 0.60 to 0.86, PH varied from 0.72 to 0.91, and EH ranged from 0.69 to 0.89. For the USP data set, heritability values were high for PH and EH in all environments (from 0.60 to 0.85 and 0.66 to 0.91, respectively). For the USP data set, trait GY had an intermediate value of 0.42 for sites Anhumas and Piracicaba with ideal nitrogen (IN) and lower values for Piracicaba and Anhumas under low nitrogen conditions (LN) (0.22 and 0.19, respectively).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484962/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484962/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Heritability and phenotypic correlations among five environments for grain yield, and three environments for plant height and ear height for the HEL data set</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484882/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484882/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Heritability and phenotypic correlations among four environments for grain yield, plant height, and ear height for the USP data set</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Estimating variance components</h3><p id="p-37">The distribution of the residuals after fitting the eight model-method combinations to the two data sets was approximately normal (data not shown). Phenotypic (and marker) data were standardized; thus summation of variance components approximated 1 for GB models. Deviations may be due to the high degree of imbalance in the number of hybrids in different environments, especially for data set HEL.</p></div><div id="sec-17" class="subsection"><h3>HEL data set</h3><div id="sec-18" class="subsection"><h4>Single-environment:</h4><p id="p-38">For the SM model for the three traits in each environment, the estimated residual variance components for the GK method were smaller than those for the GB method (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). In contrast, the variance component of genetic effects for each environment increased for the SM-GK model-method as compared to the genetic effects of model-method SM-GB.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484904/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484904/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">HEL data set</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h4>Multi-environment:</h4><p id="p-40">For the MM model, the residual variance components for MM-GK were smaller than the residuals for MM-GB for all traits, whereas, for the genetic effects, the opposite occurred: the variance component of MM-GK was higher than that of MM-GB. For the MDs models, the residual variance components for MDs-GK were always smaller than the residual variances for MDs-GB, whereas the opposite occurred for the genetic main effect and genetic interaction effect (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>).</p><p id="p-41">The results of the MM and MDs models indicated that the inclusion of the interaction term (G×E) induced a larger reduction in the estimated residual variance for GY; for traits PH and EH, these reductions in residual variance were smaller than those found for GY from the MM and MDs models. For the MDe model, the residual variance components of MDe-GK were smaller than those of the MDe-GB for all traits, whereas the variance components for the genetic main effect and genetic environment specific effect were higher for the GK models than for the GB model. The genetic main effect values are related to the correlations between environments, which are positive and range from medium to high. For the variance component associated with the genetic interaction effect, the values are lower because they decreased G×E in environments with a positive correlation. In general, the fit of MDs is slightly worse than the fit of MDe models because the genetic interaction effect component has a small effect on all traits, but mainly on EH and PH (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>).</p></div></div><div id="sec-20" class="subsection"><h3>USP data set</h3><div id="sec-21" class="subsection"><h4>Single-environment:</h4><p id="p-42">For all traits, the variance components of genetic effects were higher when using the GK than the GB method in all environments and traits for the single-environment, main genotypic effect model (SM) (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). The residual variance for SM-GK was always smaller than the residual for SM-GB for all traits.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484945/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484945/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">USP data set</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h4>Multi-environment:</h4><p id="p-45">The residual variance components of the MM and MDs models for all traits were very similar, as were most of the variance components for the genetic main effects. For all the environments and traits, the residuals from the GK were smaller than the residuals from the GB for the MM, MDs, and MDe models. The variance components of the genetic main effects of MDs-GB and MDe-GB were consistently smaller than the genetic main effect of model-methods MDs-GK and MDe-GK. Also, the variance components of the genetic environment specific effects of the GK models were higher than those of the GB models for all traits (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>).</p></div></div><div id="sec-23" class="subsection"><h3>Prediction accuracy of the models with GBLUP and GK methods</h3><p id="p-46">Results of the prediction accuracy of the eight model-method combinations for the random CV2 of the two data sets are given in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>, and <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>. Results of the prediction accuracy of the eight model-method combinations for random CV1 of the two data sets are given in Table B1 and Table B2 in Appendix B. Note that, for the single-environment models, prediction accuracy was assessed by 50 random partitions of the data into 80% TRN and 20% TST.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="HEL data set. Correlation between phenotypes and prediction values (average of 50 random CV partitions) for single-environment, main genotypic effect model with GBLUP kernel method (SM-GB), and single-environment, main genotypic effect model with GK method (SM-GK) for: (A) five environments (horizontal axis) for grain yield, (B) three environments for plant height, and (C) three environments for ear height. Environments are: Ipia&#xE7;&#xFA; (IP), Nova Mutum (NM), Pato de Minas (PM), Sertan&#xF3;polis (SE) and Sorriso (SO). Error bars show SD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2060048635" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;HEL data set. Correlation between phenotypes and prediction values (average of 50 random CV partitions) for single-environment, main genotypic effect model with GBLUP kernel method (SM-GB), and single-environment, main genotypic effect model with GK method (SM-GK) for: (A) five environments (horizontal axis) for grain yield, (B) three environments for plant height, and (C) three environments for ear height. Environments are: Ipia&#xE7;&#xFA; (IP), Nova Mutum (NM), Pato de Minas (PM), Sertan&#xF3;polis (SE) and Sorriso (SO). Error bars show SD.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1995/F2.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/6/1995/F2.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484810" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-47" class="first-child">HEL data set. Correlation between phenotypes and prediction values (average of 50 random CV partitions) for single-environment, main genotypic effect model with GBLUP kernel method (SM-GB), and single-environment, main genotypic effect model with GK method (SM-GK) for: (A) five environments (horizontal axis) for grain yield, (B) three environments for plant height, and (C) three environments for ear height. Environments are: Ipiaçú (IP), Nova Mutum (NM), Pato de Minas (PM), Sertanópolis (SE) and Sorriso (SO). Error bars show SD.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="HEL data set. Mean correlation between observed and predictive values (average of 50 random CV partitions, CV2) for multi-environment, main genotypic effect model GBLUP kernel (MM-GB), multi-environment, main genotypic effect GK (MM-GK), multi-environment, single variance G&#xD7;E deviation model GBLUP kernel (MDs-GB), multi-environment, single variance G&#xD7;E deviation model GK (MDs-GK), multi-environment, environment-specific variance G&#xD7;E deviation model GBLUP kernel (MDe-GB), multi-environment, environment-specific variance G&#xD7;E deviation model GK (MDe-GK) for (A) five environments (horizontal axis) for grain yield, (B) three environments for plant height, and (C) three environments for ear height. Environments are: Ipia&#xE7;&#xFA; (IP), Nova Mutum (NM), Pato de Minas (PM), Sertan&#xF3;polis (SE) and Sorriso (SO). Error bars show SD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2060048635" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;HEL data set. Mean correlation between observed and predictive values (average of 50 random CV partitions, CV2) for multi-environment, main genotypic effect model GBLUP kernel (MM-GB), multi-environment, main genotypic effect GK (MM-GK), multi-environment, single variance G&#xD7;E deviation model GBLUP kernel (MDs-GB), multi-environment, single variance G&#xD7;E deviation model GK (MDs-GK), multi-environment, environment-specific variance G&#xD7;E deviation model GBLUP kernel (MDe-GB), multi-environment, environment-specific variance G&#xD7;E deviation model GK (MDe-GK) for (A) five environments (horizontal axis) for grain yield, (B) three environments for plant height, and (C) three environments for ear height. Environments are: Ipia&#xE7;&#xFA; (IP), Nova Mutum (NM), Pato de Minas (PM), Sertan&#xF3;polis (SE) and Sorriso (SO). Error bars show SD.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1995/F3.medium.gif" width="440" height="402"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/6/1995/F3.medium.gif" width="440" height="402"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484881" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-48" class="first-child">HEL data set. Mean correlation between observed and predictive values (average of 50 random CV partitions, CV2) for multi-environment, main genotypic effect model GBLUP kernel (MM-GB), multi-environment, main genotypic effect GK (MM-GK), multi-environment, single variance G×E deviation model GBLUP kernel (MDs-GB), multi-environment, single variance G×E deviation model GK (MDs-GK), multi-environment, environment-specific variance G×E deviation model GBLUP kernel (MDe-GB), multi-environment, environment-specific variance G×E deviation model GK (MDe-GK) for (A) five environments (horizontal axis) for grain yield, (B) three environments for plant height, and (C) three environments for ear height. Environments are: Ipiaçú (IP), Nova Mutum (NM), Pato de Minas (PM), Sertanópolis (SE) and Sorriso (SO). Error bars show SD.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="USP data set. Mean correlation between phenotypes and predictions (average of 50 random CV partitions) for single-environment, main genotypic effects model with GBLUP kernel method (SM-GB), and single-environment, main genotypic effects model with GK method (SM-GK) in four environments (horizontal axis) for (A) grain yield, (B) plant height, and (C) ear height. Environments are: Anhumas ideal Nitrogen (A-IN), Anhumas low Nitrogen (A-LN), Piracicaba ideal Nitrogen (P-IN) and Piracicaba low Nitrogen (P-LN). Error bars show SD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2060048635" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;USP data set. Mean correlation between phenotypes and predictions (average of 50 random CV partitions) for single-environment, main genotypic effects model with GBLUP kernel method (SM-GB), and single-environment, main genotypic effects model with GK method (SM-GK) in four environments (horizontal axis) for (A) grain yield, (B) plant height, and (C) ear height. Environments are: Anhumas ideal Nitrogen (A-IN), Anhumas low Nitrogen (A-LN), Piracicaba ideal Nitrogen (P-IN) and Piracicaba low Nitrogen (P-LN). Error bars show SD.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1995/F4.medium.gif" width="440" height="256"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/6/1995/F4.medium.gif" width="440" height="256"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484848" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-49" class="first-child">USP data set. Mean correlation between phenotypes and predictions (average of 50 random CV partitions) for single-environment, main genotypic effects model with GBLUP kernel method (SM-GB), and single-environment, main genotypic effects model with GK method (SM-GK) in four environments (horizontal axis) for (A) grain yield, (B) plant height, and (C) ear height. Environments are: Anhumas ideal Nitrogen (A-IN), Anhumas low Nitrogen (A-LN), Piracicaba ideal Nitrogen (P-IN) and Piracicaba low Nitrogen (P-LN). Error bars show SD.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="USP data set. Mean correlation between observed and predictive values (average of 50 random CV partitions, CV2) for multi-environment, main genotypic effect model GBLUP kernel (MM-GB), multi-environment, main genotypic effect Gaussian kernel (MM-GK), multi-environment, single variance G&#xD7;E deviation model GBLUP kernel (MDs-GB), multi-environment, single variance G&#xD7;E deviation model Gaussian kernel (MDs-GK), multi-environment, environment-specific variance G&#xD7;E deviation model GBLUP kernel (MDe-GB), multi-environment, environment-specific variance G&#xD7;E deviation model Gaussian kernel (MDe-GK) in four environments (horizontal axis) for: (A) grain yield, (B) plant height, and (C) ear height. Environments: Anhumas ideal Nitrogen (A-IN), Anhumas low Nitrogen (A-LN), Piracicaba ideal Nitrogen (P-IN), and Piracicaba low Nitrogen (P-LN). Error bars show SD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2060048635" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;USP data set. Mean correlation between observed and predictive values (average of 50 random CV partitions, CV2) for multi-environment, main genotypic effect model GBLUP kernel (MM-GB), multi-environment, main genotypic effect Gaussian kernel (MM-GK), multi-environment, single variance G&#xD7;E deviation model GBLUP kernel (MDs-GB), multi-environment, single variance G&#xD7;E deviation model Gaussian kernel (MDs-GK), multi-environment, environment-specific variance G&#xD7;E deviation model GBLUP kernel (MDe-GB), multi-environment, environment-specific variance G&#xD7;E deviation model Gaussian kernel (MDe-GK) in four environments (horizontal axis) for: (A) grain yield, (B) plant height, and (C) ear height. Environments: Anhumas ideal Nitrogen (A-IN), Anhumas low Nitrogen (A-LN), Piracicaba ideal Nitrogen (P-IN), and Piracicaba low Nitrogen (P-LN). Error bars show SD.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/6/1995/F5.medium.gif" width="440" height="425"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/6/1995/F5.medium.gif" width="440" height="425"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/6/1995/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/484827" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-50" class="first-child">USP data set. Mean correlation between observed and predictive values (average of 50 random CV partitions, CV2) for multi-environment, main genotypic effect model GBLUP kernel (MM-GB), multi-environment, main genotypic effect Gaussian kernel (MM-GK), multi-environment, single variance G×E deviation model GBLUP kernel (MDs-GB), multi-environment, single variance G×E deviation model Gaussian kernel (MDs-GK), multi-environment, environment-specific variance G×E deviation model GBLUP kernel (MDe-GB), multi-environment, environment-specific variance G×E deviation model Gaussian kernel (MDe-GK) in four environments (horizontal axis) for: (A) grain yield, (B) plant height, and (C) ear height. Environments: Anhumas ideal Nitrogen (A-IN), Anhumas low Nitrogen (A-LN), Piracicaba ideal Nitrogen (P-IN), and Piracicaba low Nitrogen (P-LN). Error bars show SD.</p><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484894/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484894/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">HEL data set</span><div class="sb-div caption-clear"></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484927/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484927/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">USP data set</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-24" class="subsection"><h3>HEL data set</h3><div id="sec-25" class="subsection"><h4>Single-environment:</h4><p id="p-55">Results for CV2 for GY showed that the prediction accuracy of the SM-GK model-method was higher than for SM-GB in all environments (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). For GY in environment IP, prediction accuracies were 0.64 for SM-GB and 0.73 for SM-GK, while in environment PM, prediction accuracies were 0.68 for SM-GB and 0.74 for SM-GK (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>). The percent change in accuracy for single-environment GB <em>vs.</em> single-environment GK for GY ranged from 9 to 32% for the PM and NM environments, respectively. For trait PH, prediction accuracies were high: &gt;0.65 for SM-GB and &gt;0.69 for SM-GK. For trait EH, prediction accuracies were &gt;0.64 for SM-GB and &gt;0.67 for SM-GK; the increase in prediction accuracy ranged from 3 to 6% for traits EH and PH (<a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>).</p></div><div id="sec-26" class="subsection"><h4>Multi-environment:</h4><p id="p-56">For GY, the best model-methods for CV2 were MDs-GK and MDe-GK for the five environments, followed by MDs-GB in the IP environment, MM-GK in the NM, PM, and SO environments and by MDs-GB and MDe-GB in the SE environment (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). For trait GY, the percent change in accuracy for MM-GB <em>vs.</em> MM-GK was high, ranging from 9% in PM to 48% in SE environment; for MDs-GB <em>vs.</em> MDs-GK, the % change ranged from 13% in PM to 47% in the SO environment; and for MDe-GB <em>vs.</em> MDe-GK, the % change ranged from 13% in PM to 49% in SE environment (<a id="xref-table-wrap-6-4" class="xref-table" href="#T6">Table 6</a>). These results show that, for trait GY, the increase in prediction accuracy was high using the GK method. For less complex traits PH and EH, the percent change of GK over GB for all models showed a small increase in prediction accuracy, with the percent change ranging from 1 to 6%; there was no difference for trait EH in the IP environment.</p><p id="p-57">For GY, models MDs-GK and MDe-GK were, for CV2, the models that had the best prediction accuracy in all environments, with values ranging from 0.58 in SE to 0.81 in the IP environment. These values were higher for a complex trait like GY (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). For the PH trait, MDs-GK and MDe-GK were the best models, with a prediction accuracy of 0.81–0.82 for all environments; for trait EH, these values ranged from 0.77 in IP to 0.81 in PM. In general, MDs models had similar results across environments.</p><p id="p-58">Prediction accuracy for random CV1 decreased (Table B1, Appendix B) as compared with those computed for CV2 for all traits and models. For example, for environment IP, MM-GB in CV2 gave a correlation of 0.56, whereas for CV1, prediction accuracy was 0.48, that is, an 8% decrease in accuracy, whereas, for environment NM, the decrease is ∼10%. Similar decreases in accuracy from CV1 to CV2 were found for other traits and model-method combinations. However, for GY, models with GK showed a greater increase in prediction accuracy over models with GB under CV1 (Table B1, Appendix B) than the increases achieved by those models under CV2 (<a id="xref-table-wrap-6-5" class="xref-table" href="#T6">Table 6</a>). The % change in prediction accuracy of CV1 <em>vs.</em> CV2 for models with GK <em>vs.</em> GB did not change for traits PH and EH.</p></div><div id="sec-27" class="subsection"><h4>Summary of results:</h4><p id="p-59">Prediction accuracy of trait GY in the five testing environments increased from 9 to 48% for models with method GK compared to models with method GB. For GY, increases in prediction accuracy of environments from multi-environment models MM, MDs, and MDe with GK <em>vs.</em> MM, MDs, and MDe with GB tended to be higher than those achieved by the single-environment model (SM-GK <em>vs.</em> SM-GB). For traits PH and EH in three environments, the increase in prediction accuracy of method GK over method GB ranged only from 0 to 6%, and increases in prediction accuracy from multi-environment models MM, MDs, and MDe with GK <em>vs.</em> MM, MDs, and MDe with GB tended to be lower than those achieved by the single-environment model (SM-GK <em>vs.</em> SM-GB).</p><p id="p-60">Although prediction accuracy for all model-method combinations decreased in CV1 <em>vs.</em> CV2, the % change in the prediction accuracy of models with GK <em>vs.</em> models GB did not change and even increased for GY.</p></div></div><div id="sec-28" class="subsection"><h3>USP data set</h3><div id="sec-29" class="subsection"><h4>Single-environment:</h4><p id="p-61"><a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a> shows the correlations between observed and predicted values for all models in the USP data set. In general, for traits GY, PH, and EH, all prediction accuracies of the single-environment models were similar using the GB or GK methods (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). In these cases, the percentage change in accuracy for SM-GB <em>vs.</em> SM-GK was 7% (GY) in environment P-LN and 5% (PH) in environment A-LN. For GY, the prediction accuracy in the A-IN environment (0.40 for SM-GB and SM-GK) was greater than in other environments. Gains in prediction accuracy of SM-GK over SM-GB for trait GY in this data set are much lower than those achieved in the HEL data set.</p></div><div id="sec-30" class="subsection"><h4>Multi-environment:</h4><p id="p-62">Between the GB and GK methods, there were large increases in prediction accuracy using the GK method, especially for traits GY and PH (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). For trait GY, the MM model gave a high percentage change in accuracy for GB <em>vs.</em> GK ranging from 34 to 59%. In the MDs model, the percentage change ranged from 35 to 65%, and for the MDe model, the percentage change ranged from 37 to 70% (<a id="xref-table-wrap-7-3" class="xref-table" href="#T7">Table 7</a>).</p><p id="p-63">For PH, the increase using the GK method was similar to those found for the GY trait. The percentage change for the MM model with GB <em>vs.</em> GK ranged from 38% (A-IN and P-IN) to 57% (A-LN). For the MDs models, the percentage change ranged from 37% (A-IN) to 61% (A-LN), and, for the MDe models, the percentage change ranged from 37% (A-IN) to 51% (A-LN). For trait EH, the increase was smaller, ranging from 15 to 23% for the GBLUP <em>vs.</em> the GK method. The MM and MDs models showed similar correlations when compared using the same kernel method. Models MDs-GK and MDe-GK had a clear and sustainable increase in prediction accuracy over their counterparts using the GB kernel, MDs-GB, and MDe-GB.</p><p id="p-64">The prediction accuracy of random CV1 decreased (Table B2, Appendix B) compared to the accuracy obtained with CV2 for all traits and models. In general, compared with other models, models MDs and MDe decreased the accuracy of CV1 <em>vs.</em> CV2 more; this was more pronounced for GY in some environments than in other environments. Also, the decrease in the accuracy of CV1 <em>vs.</em> CV2 was higher for GY than for PH and EH.</p></div><div id="sec-31" class="subsection"><h4>Summary of results:</h4><p id="p-65">The prediction accuracy of trait GY in four environments increased from 34 to 70% for multi-environment models MM, MDs, and MDe with GK <em>vs.</em> MM, MDs, and MDe with GB. These gains in accuracy were much higher than those achieved by single-environment SM-GK over SM-GB (from 0 to 7%). Similar patterns were found for traits PH and EH, where the increase in prediction accuracy of method GK over method GB ranged from −2 to 5% for the single-environment model, SM, and drastically increased for multi-environment models MM, MDs, and MDe with GK <em>vs.</em> MM, MDs, and MDe with GB, ranging from 15 to 57%.</p><p id="p-66">Prediction accuracy for all model-method combinations decreased in CV1 <em>vs.</em> CV2 and the % change in accuracy of models with GK <em>vs.</em> models with GB decreased for all three traits. The increase in accuracy of models with GK <em>vs.</em> models with GB did not occur under the CV1 design.</p></div></div></div><div class="section discussion" id="sec-32"><h2 class="">Discussion</h2><p id="p-67">Several studies have documented the benefits of using a nonlinear Gaussian kernel in multi-environment models for capturing small complex interactions among markers, and increasing prediction accuracy (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Gianola <em>et al.</em> 2006</a>, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">2014</a>; <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Pérez-Rodríguez <em>et al.</em> 2013</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Pérez-Elizalde <em>et al.</em> 2015</a>; <a id="xref-ref-10-8" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> 2016a</a>,<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">b</a>). In this research, genome-enabled prediction accuracy was studied in two different hybrid maize data sets using multi-environment models with the GBLUP and GK methods. In this context, we proposed including the GK in the model of <a id="xref-ref-22-15" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> (2014)</a>, which accounts for environment information without and with interaction terms. Prediction accuracy was obtained using traits with different genetic architecture and heritability values that ranged from low to high.</p><div id="sec-33" class="subsection"><h3>Prediction accuracy differences in datasets, methods, CV designs and G×E</h3><p id="p-68">We compared the single-environment model with two G×E models. The first G×E model can accommodate environmental covariables (<a id="xref-ref-22-16" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> 2014</a>) that were not available in this study; the second G×E model (<a id="xref-ref-24-15" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> 2015</a>) accommodates environment main effects and environment-specific variances; both models used the GB and GK methods proposed by <a id="xref-ref-10-9" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> (2016a)</a>. These two G×E models have demonstrated good prediction accuracy when used in genomic-enabled prediction studies (<a id="xref-ref-24-16" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> 2015</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Zhang <em>et al.</em> 2015</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> 2016a</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Saint Pierre <em>et al.</em> 2016</a>). We found that GK methods improved the prediction ability of all single-environment and multi-environment models for CV2 design for all traits in HEL and USP datasets. Prediction of all traits with multi-environment models incorporating the G×E term, and the GK method gave better prediction accuracy (especially for complex traits such as GY) than the other model-methods. The increases in prediction accuracy using models with GK under random (CV2) are also reflected under random cross-validation 1 (CV1) for HEL data set, but, to a lesser extent due to the difficulty of borrowing information of unobserved (unphenotyped) lines in all environments. Similar decreases in prediction accuracy were found by <a id="xref-ref-24-17" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a>, when attempting to predict wheat lines in untested (unobserved) environments under the CV1 random partition scheme.</p><p>Furthermore, for trait GY, differences in CV1 <em>vs.</em> CV2 for GB and GK methods for the two data sets might also be due to other factors such as:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-70">Differences in dataset repeatability, higher in HEL data sets (0.81, 0.60, 0.86, 0.69, and 0.78 for each of the five environments) than in USP (0.22, 0.42, 0.19, and 0.42 for each of the four environments) (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, respectively) due to better quality of the replicated trials in HEL than the unreplicated trials in USP;</p></li><li id="list-item-2"><p id="p-71">More opportunity to borrow information from the close relatives hybrids in HEL (genetic diversity = 0.175) than from the less related hybrids included in USP dataset (genetic diversity = 0.372). For <em>h</em> = 1 (bandwidth parameter) the GK is a direct function of the squared Euclidean distance between hybrids based on markers (<span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-117.gif"/></noscript></span></span>). Thus, for a set of unrelated hybrids <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-118.gif"/></noscript></span></span> large and <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-119.gif"/></noscript></span></span><strong>I</strong> (identity matrix), <em>i.e.</em>, the GK weight more heavily the within (own) hybrid information compared to the between hybrid (from relatives) information than GB (the marker information is of no use) (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">de los Campos <em>et al.</em> 2010</a>). On the other hand, for a set of related hybrids then <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-120.gif"/></noscript></span></span> 0 and <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1995/embed/mml-math-121.gif"/></noscript></span></span><strong>1</strong> (matrix of ones), <em>i.e.</em>, marker information given by the GK weight equally or less the own hybrid information compared to information from relatives given by GB. Therefore, we speculate that, for related hybrids of HEL dataset, GK gives heavier weights to the within hybrid as well as between hybrids than the GB method does, that is why for GY, CV1 and CV2 for the multi-environment models gave very good increase in prediction accuracy over the GB;</p></li><li id="list-item-3"><p id="p-72">For CV2, G×E modulates both values (within and between hybrid information); hybrids in USP (doubled the genetic diversity) less related than those from HEL data set, then it is expected negligible values between hybrids, and similar values assign by GK and GB to within hybrid information; for CV2, predictions accuracy depends, to a great extent, on the correlations between environments.</p></li></ol><p>In summary, for CV2, for the prediction accuracy of GY in the HEL dataset, the method GK weights own, and between hybrid, performance as well as the relationship between environment (G×E modeled by MDs y MDe), whereas, for USP, GK weights only the own hybrid performance and the relationship between environment becomes more important that in the HEL data set.</p><p id="p-73">In the studies of <a id="xref-ref-24-18" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a> and <a id="xref-ref-10-10" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> (2016a)</a>, the data were balanced in the sense that all the individuals were included at the same time in all the environments. On the contrary, in this study, we had a great deal of imbalance because different numbers of maize hybrids were included in different environments; this was more pronounced in the HEL data set than in the USP data set, and different R scripts were necessary for implementing the model–method combinations.</p></div><div id="sec-34" class="subsection"><h3>Prediction accuracy using linear and nonlinear kernel methods</h3><p id="p-74">According to <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Gianola <em>et al.</em> (2014)</a>, GK has better predictive ability and a more flexible structure than GBLUP. Another point is that GK can capture nonadditive effects between markers. <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Jiang and Reif (2015)</a> evaluated maize data sets and investigated whether the prediction accuracy across connected biparental families can be increased by modeling additive × additive epistasis; the authors found that the prediction accuracy of RKHS (including epistasis) was superior to that of GBLUP (ignoring epistasis). Our study clearly shows a large increase in predictive ability when the G×E model and the GK method are combined for all traits, but mainly for GY (both data sets) and PH (USP data set). This indicates that, to increase predictive ability, it is important to consider the nonadditive effect (<em>i.e.</em>, epistasis) as the genetic relatedness across connected populations.</p><p id="p-75">There are different choices for computing kernel functions: for example, linear kernel matrices incorporate only additive effects of the markers, polynomials kernels of different orders might incorporate different degrees of marker interactions, and the Gaussian kernel function uses complex epistatic marker interaction (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akdemir and Jannink 2015</a>). In GS, additive kernels of the GBLUP type are employed when predicting breeding values, whereas, when attempting to predict genetic values, GKs would be more appropriate. <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akdemir and Jannink (2015)</a> demonstrated that epistatic marker effects in local regions of the chromosome with low recombination are stable through generations, and offer the opportunity to exploit epistasis for improving genomic-enabled prediction accuracy; the authors defined local kernels in regions of the genome and calculated separate kernels for each region. The results shown in this study clearly showed the benefit of exploiting these local epistatic effects captured in the GK, and their interaction with environments.</p><p id="p-76">The flexible structure of the MDs and MDe models is important, especially when combined with kernels that capture nonadditive effects, as previously proven. In general, in the two maize data sets, the MDs-GK and MDe-GK models had better prediction accuracy than other models. The increase was not higher due to the occurrence of intermediate-to-high positive correlations between the analyzed environments, resulting in low G×E. These results corroborate those obtained by <a id="xref-ref-24-19" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a> and <a id="xref-ref-10-11" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> (2016a)</a>, who observed that the intensity of the environmental correlation is related to the proportion of the genomic variance explained by the genetic main effects of markers across environments and genetic-specific effects of markers in environments.</p></div><div id="sec-35" class="subsection"><h3>Better fit of the G×E GK models</h3><p id="p-77">For GY of the HEL data set, the better fit of the model-method MDe-GB over the MDs-GB and MM-GB models is evident in their residual variance components: 0.227, 0.230, and 0.336, respectively. Similarly, for GY, the better fit of model MDe-GK over the MDs-GK and MM-GK models is also evident in their residual variance components: 0.093, 0.107, and 0.273, respectively. These trends in the residuals of the models are also found in traits PH and EH. The residuals of GK models were lower than those of the GBLUP models, indicating the better fit of the nonlinear <em>vs.</em> the linear kernel methods.</p><p id="p-78">For GY of the USP data set, similar trends in the residuals of model-method MDe-GB <em>vs.</em> the MDs-GB and MM-GB models were found: 0.840, 0.836, and 0.864, respectively. Similar clear trends in the residuals of these models were found for traits PH and EH. These patterns are also clear for the residuals of GK model-methods MDe-GK, MDs-GK, and MM-GK for trait EH, but not for trait PH.</p></div><div id="sec-36" class="subsection"><h3>Prediction accuracy using multi-environment models</h3><p id="p-79">In this study, all pairwise correlations between environments were high and positive. This is important, because the G×E model of <a id="xref-ref-24-20" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> (2015)</a> has the limitation of better and more efficient prediction when applied to subsets of environments that have positive and similar correlations (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2016b</a>; <a id="xref-ref-10-12" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> 2016a</a>). In positively correlated environments, the main marker effects are the most influential components when predicting genetic values; their variance component is high, and this produces better prediction accuracy than the single-environment model (<a id="xref-ref-24-21" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> 2015</a>; <a id="xref-ref-10-13" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> 2016a</a>). Environments with intermediate or high positive correlations indicate little G×E interaction. Thus, the model reacts by reducing the specific marker effect. A correlation that is negative or close to zero implies strong G×E interaction, making it difficult to predict one environment based on information from another. With this, the model reduces the main effect of the marker, and increases the specific effects. To work around the limitations of the G×E models, <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016b)</a> developed multi-environment Bayesian genomic models that allow an arbitrary genetic covariance structure between environments, because an unstructured covariance matrix was used, and its parameters were estimated from the data.</p><p id="p-80">In this study, the prediction of multi-environment models was assessed by applying the CV strategy called CV2 in <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Burgueño <em>et al.</em> (2012)</a> and (<a id="xref-ref-10-14" class="xref-bibr" href="#ref-10">Cuevas <em>et al.</em> 2016a</a>), where some lines are represented in some environments but not in others. This CV2 validation strategy performed better than CV strategy CV1 (where lines have not been evaluated in any field trials) proposed by <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Burgueño <em>et al.</em> (2012)</a>, when applied in multi-environment models (<a id="xref-ref-22-17" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> 2014</a>; <a id="xref-ref-24-22" class="xref-bibr" href="#ref-24">López-Cruz <em>et al.</em> 2015</a>; <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2016b</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Saint Pierre <em>et al.</em> 2016</a>).</p><p id="p-81">When introducing interaction effects, models MDs and the MDe showed increases in prediction accuracy for GY in most environments for the HEL and USP data sets over the single-environment model. Furthermore, models with GK had superior prediction accuracy than GB models. For traits PH and EH in both data sets, not much increase in prediction accuracy of MDs and MDe models was achieved over the single-environment model. These results were expected because the genetic architecture of traits PH and EH is less complex than that of GY and less influenced by environmental factors. When main marker effects and interaction effects are introduced in the model using covariance structures, this improves prediction accuracy.</p><p id="p-82">The increase in accuracy when the interaction term for complex traits was included concurs with the findings of <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> (2016a)</a>, where increases in prediction accuracy were achieved by including dense molecular markers and G×E in a set of Mexican and Iranian landraces. <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Zhang <em>et al.</em> (2015)</a> also used multi-environment models incorporating G×E, and obtained an increase in prediction accuracy of several maize biparental populations. Similarly, <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Saint Pierre <em>et al.</em> (2016)</a> evaluated spring wheat lines and introduced other environmental covariates in the Jarquín’s model, showing that the prediction models gave better predictions using random CV. Recently, results of extensive analyses of spring wheat trials across international environments conducted for several years in South and West Asia, North Africa, and Mexico showed a consistent increase in the genomic prediction accuracy of the MDs-GB model over the MM-GB and SM-GB models (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sukumaran <em>et al.</em> 2017</a>). One limitation of the data used in this study is that only 1 yr was available for assessing the genomic-enabled accuracy of the various models.</p><p id="p-83">Our results show that predictions with medium-to-high accuracy for GS programs can be expected in environments with low-to-high heritability. Despite the low heritability of GY in P-LN and A-LN environments, the results were similar to results in other environments that have medium-to-high heritability. But, in general, for the USP data set, prediction accuracy of all target traits under stress conditions was lower than under ideal nitrogen conditions, while in stress environments, heritability was lower than under ideal nitrogen conditions. These results agree with those obtained by <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Zhang <em>et al.</em> (2015)</a>, who found lower genomic prediction accuracy under water stress conditions than under ideal conditions, especially for complex traits such as GY.</p></div><div id="sec-37" class="subsection"><h3>Conclusions</h3><p id="p-84">Incorporating the GK method into the model of <a id="xref-ref-22-18" class="xref-bibr" href="#ref-22">Jarquín <em>et al.</em> (2014)</a> increased prediction accuracy as compared to the model used with the linear kernel GBLUP; these results were found in both data sets with the models used in this study: MM across environments, MDs, and MDe. Other results of the application of eight model-method combinations between four models (SM, MM, MDs, and MDe) and two kernel methods (GBLUP and Gaussian kernel) in two extensive maize data sets show that (1) genomic models incorporating G×E interaction had higher prediction accuracy than single-environment models; and (2) models with nonlinear Gaussian kernel had higher prediction accuracy than models with linear kernel GBLUP. The model-method combinations with the highest prediction accuracy were MDe-GK and MDs-GK. Results of this study indicated that by employing appropriate statistical genomic-enabled prediction models the researchers and plant breeders can improve the prediction of hybrids that were not evaluated in several environments. Random CV2 mimics sparse (incomplete) testing that allow to safe resources to the breeding program while improving prediction accuracy. Further research is required to compare the genomic-enabled G×E kernel prediction models used in this study with the models recently developed by <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Cuevas <em>et al.</em> (2016b)</a>, who studied genomic-enabled G×E models by means of Kronecker products applied to unstructured covariance matrices. It is also necessary to develop efficient computing software for fitting the structures considered in the models of this study while maintaining the inferential advantages of the Markov Chain Monte Carlo.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-190">We thank two anonymous reviewers and the Associate Editor for the time invested in reading and reviewing this article. Their comments and suggestions significantly improved the quality and readability of the manuscript.</p></div><div class="section app" id="app-1"><h2 class="">Appendix A</h2><p id="p-85">In Box A1, we provide simplified scripts that can be used to obtain the GBLUP and GK matrix, to be used in the multi-environment models used in this study. The example uses the following R-object:</p><p id="p-86">- X (n×p) a genomic marker matrix. The line IDs can be retrieved using either rownames(X) or SNP IDs in colnames(X).</p><p id="p-87">In the examples used to fit MM, MDs and MDe models, an R-object is necessary as matrix (pheno_geno) compound by three vectors of Location_id, Germplasm_id and Y, where:</p><ul class="list-simple " id="list-2"><li id="list-item-4"><p id="p-88">pheno_geno$Location_id vector is compound by environment information</p></li><li id="list-item-5"><p id="p-89">pheno_geno$Germplasm_id vector is compound by germplasm names</p></li><li id="list-item-6"><p id="p-90">pheno_geno$Y is a numeric and standardized vector with yield records in all environments</p></li></ul><p id="p-91">In Box A2 we give simplified scripts that can be used to fit models MM, MDs or MDe</p></div><div class="section app" id="app-2"><h2 class="">Appendix B</h2><div class="boxed-text" id="boxed-text-1"><span class="boxed-text-label">Box A1.</span> <h3>An Example of How to Obtain GBLUP (GB) and GK Matrix</h3><p id="p-92">X&lt;-scale(X,center=TRUE,scale=TRUE)</p><p id="p-93"># G-BLUP kernel (GB)</p><p id="p-94">GB&lt;-tcrossprod(X)/ncol(X)</p><p id="p-95"># Gaussian Kernel (GK)</p><p id="p-96">dist&lt;-as.matrix(dist(X))^2</p><p id="p-97">GK&lt;-exp(-dist/median(dist))</p></div><div class="boxed-text" id="boxed-text-2"><span class="boxed-text-label">Box A2.</span> <h3>An Example of How to fit MDs model with cross-validation scheme CV2</h3><p id="p-98"># Loading objects needed for MM, MDs and MDe models</p><p id="p-99">library(BGLR)</p><p id="p-100">env&lt;-c(1,2,3,4) ### Choose environment</p><p id="p-101">K&lt;-GK #### Choose kernel GK or GB</p><p id="p-102">model&lt;- “MDs” ### Choose model “MM”or”MDs”or”MDe”</p><p id="p-103">CV&lt;- 2 ### Choose CV=1 (CV1), CV=2 (CV2)</p><p id="p-104">nEnv&lt;-length(env)</p><p id="p-105">pheno_geno1&lt;-pheno_geno[pheno_geno$Location==env[1],]</p><p id="p-106">i&lt;-2</p><p id="p-107">while (i &lt;=nEnv){</p><p id="p-108">pheno_geno1&lt;-rbind(pheno_geno1,pheno_geno[pheno_geno$Location==env[i],])</p><p id="p-109">i&lt;-i+1</p><p id="p-110">}</p><p id="p-111">envID&lt;-as.factor(pheno_geno1[,1])</p><p id="p-112">IDs&lt;-as.character(unique(pheno_geno1$Germplasm_id))</p><p id="p-113">IDy&lt;-pheno_geno1$Germplasm_id</p><p id="p-114">Y &lt;- as.numeric(pheno_geno1[,3])</p><p id="p-115">Y &lt;- scale(Y,center=TRUE,scale=TRUE)</p><p id="p-116">names(Y)&lt;-IDy</p><p id="p-117">nEnv &lt;- length(env)</p><p id="p-118">K&lt;-K[IDs,IDs]</p><p id="p-119">phenol_geno1$ Germplasm_id&lt;-factor(x= phenol_geno1$ Germplasm_id,levels=rownames(K),ordered=TRUE)</p><p id="p-120">Zg &lt;- model.matrix(∼factor(pheno_geno1$Germplasm_id)-1) # genotype design matrix</p><p id="p-121">Ze &lt;- model.matrix(∼factor(pheno_geno1$Location)-1) # environment design matrix</p><p id="p-122">K1 &lt;- Zg%*%K%*%t(Zg)</p><p id="p-123"># CV scheme fit</p><p id="p-124">set.seed(12345)</p><p id="p-125">nFolds &lt;- 5</p><p id="p-126">partitions &lt;- 10</p><p id="p-127">if (CV == 1) {</p><p id="p-128">mfold &lt;- matrix(NA, ncol = partitions, nrow = length(IDs))</p><p id="p-129">sets &lt;- matrix(NA, nrow = nrow(pheno_geno1), ncol = partitions)</p><p id="p-130">for (s in 1:partitions) {</p><p id="p-131">mfold[, s] &lt;- sample(1:nFolds, size = length(IDs), replace = TRUE)</p><p id="p-132">for (i in 1:nrow(pheno_geno1)) {</p><p id="p-133">sets[i, s] &lt;- mfold[which(IDs == pheno_geno1$Germplasm_id[i]), s]</p><p id="p-134">}</p><p id="p-135">}</p><p id="p-136">}</p><p id="p-137">if (CV == 2) {</p><p id="p-138">sets &lt;- matrix(NA, nrow = nrow(pheno_geno1), ncol = partitions)</p><p id="p-139">for (s in 1:partitions) {</p><p id="p-140">for (i in IDs) {</p><p id="p-141">tmp = which(IDy == i)</p><p id="p-142">ni = length(tmp)</p><p id="p-143">tmpFold &lt;- sample(1:nFolds, size = ni, replace = ni &gt; nFolds)</p><p id="p-144">sets[tmp, s] &lt;- tmpFold</p><p id="p-145">}</p><p id="p-146">}</p><p id="p-147">}</p><p id="p-148">trn_tst&lt;-matrix(nrow=nrow(pheno_geno1),ncol=nFolds,NA)</p><p id="p-149">NA.y &lt;- matrix(Y, ncol = nFolds, nrow = length(Y))</p><p id="p-150"># Fitting MM model</p><p id="p-151">if (model==”MM”) {</p><p id="p-152">ETA&lt;-list(ENV=list(X=Ze,model=”FIXED”),</p><p id="p-153">Grm=list(K=K1,model=”RKHS”))}</p><p id="p-154"># Fitting MDs model</p><p id="p-155">if(model==”MDs”){</p><p id="p-156">ZEZE&lt;-tcrossprod(Ze)</p><p id="p-157">K2&lt;-K1*ZEZE</p><p id="p-158">ETA&lt;-list(ENV=list(X=Ze,model=”FIXED”),</p><p id="p-159">Grm=list(K=K1,model=”RKHS”),</p><p id="p-160">EGrm=list(K=K2,model=”RKHS”))}</p><p id="p-161"># Fitting MDe model</p><p id="p-162">if (model==”MDe”){</p><p id="p-163">ETA&lt;-list(ENV=list(X=Ze,model=”FIXED”),</p><p id="p-164">Grm=list(K=K1,model=”RKHS”))</p><p id="p-165">for(k in 1:nEnv){</p><p id="p-166">ZEE&lt;-matrix(0,nrow=nrow(Ze),ncol=ncol(Ze))</p><p id="p-167">ZEE[,k]&lt;-Ze[,k]</p><p id="p-168">ZEEZ&lt;-(ZEE%*%t(Ze))</p><p id="p-169">K3&lt;-K1*ZEEZ</p><p id="p-170">ETA[[k+2]]&lt;-list(K=K3,model=’RKHS’)</p><p id="p-171">}}</p><p id="p-172">COR&lt;-matrix(0,nrow=nEnv,ncol=(nFolds*partitions))</p><p id="p-173">for (s in 1:partitions) {</p><p id="p-174">NA.y &lt;- matrix(Y, ncol = (nFolds), nrow = length(Y))</p><p id="p-175">for(i in 1:ncol(trn_tst)){</p><p id="p-176">trn_tst[,i]&lt;-ifelse(sets[,s]==i,’tst’,’trn’)</p><p id="p-177">NA.y[,i][(trn_tst[,i]==”tst”)]&lt;-NA</p><p id="p-178">A &lt;- as.matrix(NA.y[,i])</p><p id="p-179">fm &lt;- BGLR(y=A,ETA=ETA,nIter=10000,burnIn=1000,thin=2)</p><p id="p-180">for(g in 1:length(env)){</p><p id="p-181">tst1 &lt;- which(is.na(fm$y[envID==env[g]]))</p><p id="p-182">COR[g,((s-1)*nFolds+i)] &lt;- cor(Y[envID==env[g]][tst1],fm$yHat[envID==env[g]][tst1])</p><p id="p-183">}</p><p id="p-184">}</p><p id="p-185">}</p></div><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484812/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484812/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table B1</span> <span class="caption-title">HEL data set</span><div class="sb-div caption-clear"></div></div></div><div id="T9" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484868/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484868/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table B2</span> <span class="caption-title">USP data set</span><div class="sb-div caption-clear"></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-14"><p id="p-191"><em>Communicating editor: J.-L. Jannink</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-02-17"><span class="received-label">Received </span>February 17, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-04-28"><span class="accepted-label">Accepted </span>April 28, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Bandeira e Sousa <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.1" data-doi="10.1534/genetics.114.173658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Locally epistatic genomic relationship matrices for genomic association and prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">871</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.173658%26rft_id%253Dinfo%253Apmid%252F25614606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzMvODU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.2" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genome-wide selection for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.3" data-doi="10.2135/cropsci2014.07.0460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mugo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarekegne</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic gains in grain yield through genomic selection in eight bi-parental maize populations under drought stress.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">154</span>–<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D154%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.07.0460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.07.0460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">84</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">210</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D84%26rft.spage%253D210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.5" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.6" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.7" data-doi="10.1080/15427528.2011.558767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahuku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection and prediction in plant breeding.</span> <abbr class="cit-jnl-abbrev">J. Crop Improv.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">239</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCrop%2BImprov.%26rft.volume%253D25%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15427528.2011.558767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15427528.2011.558767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franco</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span>. <span class="cit-article-title">Genomic prediction of gene bank wheat landraces.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D1819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maccaferri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuberosa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Extending the marker × environment interaction model for genomic-enabled prediction and genome-wide association analysis in durum wheat.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">2193</span>–<span class="cit-lpage">2209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D56%26rft.spage%253D2193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-Lopez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>. <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Bayesian genomic prediction with genotype × environment interaction kernel models.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D7%26rft.spage%253D41%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soberanis</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Elizalde</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic prediction of genotype × environment interaction kernel regression models.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">3</span>) <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-web" id="cit-7.6.1995.12"><div class="cit-metadata unstructured">de los Campos, G., and P. Pérez-Rodríguez, 2016. BGLR: Bayesian generalized linear regression. R package version 1.0.5. Available at: <a href="http://CRAN.R-project.org/package=BGLR">http://CRAN.R-project.org/package=BGLR</a>. Accessed: August 10, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.13" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.14" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.15" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.16" data-doi="10.1534/genetics.107.084285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Kaam</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reproducing kernel Hilbert spaces regression methods for genomic assisted prediction of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2289</span>–<span class="cit-lpage">2303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2289%26rft.epage%253D2303%26rft.atitle%253DReproducing%2BKernel%2BHilbert%2BSpaces%2BRegression%2BMethods%2Bfor%2BGenomic%2BAssisted%2BPrediction%2Bof%2BQuantitative%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084285%26rft_id%253Dinfo%253Apmid%252F18430950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIyODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTk5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.17" data-doi="10.1534/genetics.105.049510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stella</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic-assisted prediction of genetic value with semiparametric procedures.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1761</span>–<span class="cit-lpage">1776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049510%26rft_id%253Dinfo%253Apmid%252F16648593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE3NjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTk5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.18" data-doi="10.1371/journal.pone.0091693"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krämer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stella</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Enhancing genome-enabled prediction by bagging genomic BLUP.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">e91693</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De91693%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0091693%26rft_id%253Dinfo%253Apmid%252F24722227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0091693&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24722227&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.19" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTk5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.20" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.21" data-doi="10.1007/s00122-013-2231-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">463</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2231-5%26rft_id%253Dinfo%253Apmid%252F24264761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2231-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24264761&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.22" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du Cheyron</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.23" data-doi="10.1534/genetics.115.177907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Modeling epistasis in genomic selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177907%26rft_id%253Dinfo%253Apmid%252F26219298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNzU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">López-Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Increased prediction accuracy in wheat breeding trials using a marker × environment interaction genomic selection model.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D5%26rft.spage%253D569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.25" data-doi="10.2135/cropsci2012.02.0112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">H. J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide selection <em>vs.</em> marker-assisted recurrent selection to improve grain yield and stover-quality traits for cellulosic ethanol in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">58</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D58%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.02.0112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.02.0112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTk5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.27" data-doi="10.3389/fgene.2014.00001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Kernel-based whole-genome prediction of complex traits: a review.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00001%26rft_id%253Dinfo%253Apmid%252F24567736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24567736&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.28" data-doi="10.1186/1297-9686-45-17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koyama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Predicting complex traits using a diffusion kernel on genetic markers with an application to dairy cattle and wheat data.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-45-17%26rft_id%253Dinfo%253Apmid%252F23763755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-45-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23763755&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Elizalde</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Selection of the bandwidth parameter in a Bayesian kernel regression model for genomic-enabled prediction.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">512</span>–<span class="cit-lpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D20%26rft.spage%253D512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.30" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE5OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manès</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>. <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparison between linear and non-parametric regression models for genome-enabled prediction in wheat.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1595</span>–<span class="cit-lpage">605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D1595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manes</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>. <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparison between linear and non-parametric regression models for genome-enabled prediction in wheat.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1595</span>–<span class="cit-lpage">1605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D1595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bondalapati</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Meyer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pita</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A pedigree reaction norm model for prediction of cotton (<em>Gossypium</em> sp.) yield in multi-environment trials.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-web" id="cit-7.6.1995.34"><div class="cit-metadata unstructured">R Core Team, 2016. <em>R: A Language and Environment for Statistical Computing</em>. R Foundation for Statistical Computing, Vienna, Austria. Available at: <a href="https://www.R-project.org/">https://www.R-project.org/</a>. Accessed: August 10, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saint Pierre</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuentes Dávila</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Figueroa López</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction models for grain yield of spring bread wheat in diverse agro-ecological zones.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">27312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D27312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.36" data-doi="10.1534/g3.116.036251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sukumaran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite> <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction with pedigree and genotype × environment interaction in spring wheat grown in South and West Asia, North Africa, and Mexico.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.036251%26rft_id%253Dinfo%253Apmid%252F27903632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMi80ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzYvMTk5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.37" data-doi="10.1186/1471-2164-15-823"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Unterseer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haberer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A powerful tool for genome analysis in maize: development and evaluation of the high density 600 k SNP genotyping array.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D823%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-823%26rft_id%253Dinfo%253Apmid%252F25266061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-823&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25266061&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.6.1995.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> Genomic measures of relationship and inbreeding. Interbull Annual Meeting Proceeding <span class="cit-source">Interbull Bulletin</span> <span class="cit-vol">37</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterbull%2BBulletin%26rft.volume%253D37%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.39" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schön</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">synbreed: framework for the analysis of genomic prediction data using R.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2086</span>–<span class="cit-lpage">2087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D28%26rft.spage%253D2086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1995.41" data-doi="10.1038/hdy.2014.99"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic prediction in biparental tropical maize populations in water-stressed and well-watered environments using low-density and GBS SNPs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">291</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D291%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.99%26rft_id%253Dinfo%253Apmid%252F25407079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.99&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25407079&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1995.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/6/1995.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1985" title="HDAC Inhibitors Disrupt Programmed Resistance to Apoptosis During Drosophila Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/7/2015" title="Insight into the Recent Genome Duplication of the Halophilic Yeast Hortaea werneckii: Combining an Improved Genome with Gene Expression and Chromatin Structure" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/6"><span>Volume 7 Issue 6, June 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/6.cover-source.jpg?itok=5_J-OKHc" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1995" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-n44cR3TuAmm5jdnQ_qjFlzv8a_52dcql9neLmxb9pBQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/6/1995" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/6/1995" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/6/1995" class="active" data-icon-position="" data-hide-link-title="0">Genomic-Enabled Prediction in Maize Using Kernel Models with Genotype × Environment Interaction</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484776" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-kuuEHNvkaBNT_kD8JztniQOgV8nyQOqM7GfUyp6-_GE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit--2" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1995.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/6/1995" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--5" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-eLDqDbXV7iGj9YKqGMejAXIlgv59a_S9Y7ENnf3sqSI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/6/1995" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1995.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484776" id="node484776--2" data-pisa="ggg;7/6/1995" data-pisa-master="ggg;g3.117.042341" data-apath="/ggg/7/6/1995.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic-Enabled Prediction in Maize Using Kernel Models with Genotype × Environment Interaction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7887-9543" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Massaine Bandeira e Sousa</span>, <span class="highwire-citation-author" data-delta="1">Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="2">Evellyn Giselly de Oliveira Couto</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Diego Jarquín</span>, <span class="highwire-citation-author" data-delta="5">Roberto Fritsche-Neto</span>, <span class="highwire-citation-author" data-delta="6">Juan Burgueño</span> and <span class="highwire-citation-author" data-delta="7">Jose Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1995-2014;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042341 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484776--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Massaine Bandeira e Sousa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, 13418-900 Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Massaine%2BBandeira%2Be%2BSousa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bandeira%20e%20Sousa%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMassaine%2BBandeira%2Be%2BSousa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7887-9543" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Massaine Bandeira e Sousa</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University of Quintana Roo, 77019 Chetumal, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Evellyn Giselly de Oliveira Couto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, 13418-900 Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evellyn%2BGiselly%2Bde%2BOliveira%2BCouto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Oliveira%20Couto%20EG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvellyn%2BGiselly%2Bde%2BOliveira%2BCouto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Colegio de Postgraduados, CP 56230 Montecillos, 56230 D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Agronomy and Horticulture Department, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Roberto Fritsche-Neto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, 13418-900 Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BFritsche-Neto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritsche-Neto%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoberto%2BFritsche-Neto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/6/1995" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/6/1995&amp;title=Genomic-Enabled%20Prediction%20in%20Maize%20Using%20Kernel%20Models%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/6/1995&amp;title=Genomic-Enabled%20Prediction%20in%20Maize%20Using%20Kernel%20Models%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/6/1995&amp;title=Genomic-Enabled%20Prediction%20in%20Maize%20Using%20Kernel%20Models%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/6/1995&amp;text=Genomic-Enabled%20Prediction%20in%20Maize%20Using%20Kernel%20Models%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/6/1995&amp;title=Genomic-Enabled%20Prediction%20in%20Maize%20Using%20Kernel%20Models%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/6/1995&amp;t=Genomic-Enabled%20Prediction%20in%20Maize%20Using%20Kernel%20Models%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/6/1995&amp;title=Genomic-Enabled%20Prediction%20in%20Maize%20Using%20Kernel%20Models%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/6/1995&amp;title=Genomic-Enabled%20Prediction%20in%20Maize%20Using%20Kernel%20Models%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484776/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484776" id="node484776--3" data-pisa="ggg;7/6/1995" data-pisa-master="ggg;g3.117.042341" data-apath="/ggg/7/6/1995.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic-Enabled Prediction in Maize Using Kernel Models with Genotype × Environment Interaction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7887-9543" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Massaine Bandeira e Sousa</span>, <span class="highwire-citation-author" data-delta="1">Jaime Cuevas</span>, <span class="highwire-citation-author" data-delta="2">Evellyn Giselly de Oliveira Couto</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Diego Jarquín</span>, <span class="highwire-citation-author" data-delta="5">Roberto Fritsche-Neto</span>, <span class="highwire-citation-author" data-delta="6">Juan Burgueño</span> and <span class="highwire-citation-author" data-delta="7">Jose Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1995-2014;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.042341 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484776--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Massaine Bandeira e Sousa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, 13418-900 Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Massaine%2BBandeira%2Be%2BSousa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bandeira%20e%20Sousa%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMassaine%2BBandeira%2Be%2BSousa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7887-9543" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Massaine Bandeira e Sousa</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jaime Cuevas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>University of Quintana Roo, 77019 Chetumal, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jaime%2BCuevas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cuevas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJaime%2BCuevas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Evellyn Giselly de Oliveira Couto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, 13418-900 Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evellyn%2BGiselly%2Bde%2BOliveira%2BCouto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Oliveira%20Couto%20EG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvellyn%2BGiselly%2Bde%2BOliveira%2BCouto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Colegio de Postgraduados, CP 56230 Montecillos, 56230 D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Agronomy and Horticulture Department, University of Nebraska, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Roberto Fritsche-Neto </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genetics, Luiz de Queiroz College of Agriculture, University of São Paulo, Piracicaba, 13418-900 Brazil</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BFritsche-Neto%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritsche-Neto%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoberto%2BFritsche-Neto%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jose Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), 06600 D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jose%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJose%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1995&amp;rft.epage=2014&amp;rft.atitle=Genomic-Enabled%20Prediction%20in%20Maize%20Using%20Kernel%20Models%20with%20Genotype%20%C3%97%20Environment%20Interaction&amp;rft.volume=7&amp;rft.issue=6&amp;rft.date=2017-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bandeira+e+Sousa%2C+Massaine&amp;rft.au=Cuevas%2C+Jaime&amp;rft.au=de+Oliveira+Couto%2C+Evellyn+Giselly&amp;rft.au=P%C3%A9rez-Rodr%C3%ADguez%2C+Paulino&amp;rft.au=Jarqu%C3%ADn%2C+Diego&amp;rft.au=Fritsche-Neto%2C+Roberto&amp;rft.au=Burgue%C3%B1o%2C+Juan&amp;rft.au=Crossa%2C+Jose"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484776/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484776/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484776/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484776/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484776/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484776/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484776/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484776/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484776/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484776/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484776/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484776/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/6/1995" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Statistical models</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-32" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix B</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/6/1995.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/6/1995.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/7/6/1995.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
