<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/6/1833.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/6/1833.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/6/1833/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/6/1833.atom" />
<meta name="HW.pisa" content="ggg;7/6/1833" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Variational Bayes Genomic-Enabled Prediction Model with Genotype × Environment Interaction" />
<meta name="DC.Identifier" content="10.1534/g3.117.041202" />
<meta name="DC.Date" content="2017-06-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Montesinos-López et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="There are Bayesian and non-Bayesian genomic models that take into account G×E interactions. However, the computational cost of implementing Bayesian models is high, and becomes almost impossible when the number of genotypes, environments, and traits is very large, while, in non-Bayesian models, there are often important and unsolved convergence problems. The variational Bayes method is popular in machine learning, and, by approximating the probability distributions through optimization, it tends to be faster than Markov Chain Monte Carlo methods. For this reason, in this paper, we propose a new genomic variational Bayes version of the Bayesian genomic model with G×E using half-t priors on each standard deviation (SD) term to guarantee highly noninformative and posterior inferences that are not sensitive to the choice of hyper-parameters. We show the complete theoretical derivation of the full conditional and the variational posterior distributions, and their implementations. We used eight experimental genomic maize and wheat data sets to illustrate the new proposed variational Bayes approximation, and compared its predictions and implementation time with a standard Bayesian genomic model with G×E. Results indicated that prediction accuracies are slightly higher in the standard Bayesian model with G×E than in its variational counterpart, but, in terms of computation time, the variational Bayes genomic model with G×E is, in general, 10 times faster than the conventional Bayesian genomic model with G×E. For this reason, the proposed model may be a useful tool for researchers who need to predict and select genotypes in several environments." />
<meta name="DC.Contributor" content="Osval A. Montesinos-López" />
<meta name="DC.Contributor" content="Abelardo Montesinos-López" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="José Cricelio Montesinos-López" />
<meta name="DC.Contributor" content="Francisco Javier Luna-Vázquez" />
<meta name="DC.Contributor" content="Josafhat Salinas-Ruiz" />
<meta name="DC.Contributor" content="José R. Herrera-Morales" />
<meta name="DC.Contributor" content="Raymundo Buenrostro-Mariscal" />
<meta name="article:published_time" content="2017-06-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="A Variational Bayes Genomic-Enabled Prediction Model with Genotype × Environment Interaction" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;There are Bayesian and non-Bayesian genomic models that take into account G×E interactions. However, the computational cost of implementing Bayesian models is high, and becomes almost impossible when the number of genotypes, environments, and traits is very large, while, in non-Bayesian models, there are often important and unsolved convergence problems. The variational Bayes method is popular in machine learning, and, by approximating the probability distributions through optimization, it tends to be faster than Markov Chain Monte Carlo methods. For this reason, in this paper, we propose a new genomic variational Bayes version of the Bayesian genomic model with G×E using half-t priors on each standard deviation (SD) term to guarantee highly noninformative and posterior inferences that are not sensitive to the choice of hyper-parameters. We show the complete theoretical derivation of the full conditional and the variational posterior distributions, and their implementations. We used eight experimental genomic maize and wheat data sets to illustrate the new proposed variational Bayes approximation, and compared its predictions and implementation time with a standard Bayesian genomic model with G×E. Results indicated that prediction accuracies are slightly higher in the standard Bayesian model with G×E than in its variational counterpart, but, in terms of computation time, the variational Bayes genomic model with G×E is, in general, 10 times faster than the conventional Bayesian genomic model with G×E. For this reason, the proposed model may be a useful tool for researchers who need to predict and select genotypes in several environments.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/06/01" />
<meta name="citation_mjid" content="ggg;7/6/1833" />
<meta name="citation_id" content="7/6/1833" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/6/1833" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/6/1833.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/6/1833.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/6/1833.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.041202" />
<meta name="citation_pmid" content="28391241" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="6" />
<meta name="citation_num_pages" content="21" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1833" />
<meta name="citation_lastpage" content="1853" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Osval A. Montesinos-López" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, 28040, México" />
<meta name="citation_author" content="Abelardo Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Jalisco, México" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author" content="José Cricelio Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), 36240 Guanajuato, México" />
<meta name="citation_author" content="Francisco Javier Luna-Vázquez" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, 28040, México" />
<meta name="citation_author" content="Josafhat Salinas-Ruiz" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, Campus Córdoba, Km. 348 Carretera Federal Córdoba-Veracruz, Amatlán de los Reyes, 94946, México" />
<meta name="citation_author" content="José R. Herrera-Morales" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, 28040, México" />
<meta name="citation_author" content="Raymundo Buenrostro-Mariscal" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, 28040, México" />
<meta name="citation_reference" content="citation_journal_title=Anim. Sci. J.;citation_author=A. Arakawa;citation_author=M. Taniguchi;citation_author=T. Hayashi;citation_author=S. Mikawa;citation_title=Variational Bayesian method of estimating variance components.;citation_pages=863-872;citation_volume=87;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Adv. Neural Inf. Process. Syst.;citation_author=H. Attias;citation_title=ADVANCES IN NEURAL INFORMATION PROCESSING SYSTEMS;citation_pages=209-215;citation_volume=12;citation_year=2000;citation_issue=1–2" />
<meta name="citation_reference" content="citation_journal_title=Bayesian Anal.;citation_author=P. Carbonetto;citation_author=M. Stephens;citation_title=Scalable variational inference for Bayesian variable selection in regression, and its accuracy in genetic association studies.;citation_pages=73-108;citation_volume=7;citation_year=2012;citation_issue=1;citation_doi=10.1214/12-BA703" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_issue=2;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=G3 Bethesda;citation_author=J. Crossa;citation_author=Y. Beyene;citation_author=S. Kassa;citation_author=P. Pérez;citation_author=J. M. Hickey;citation_title=Genomic prediction in maize breeding populations with genotyping-by-sequencing.;citation_pages=1903-1926;citation_volume=3;citation_year=2013;citation_issue=11" />
<meta name="citation_reference" content="citation_journal_title=G3 Bethesda;citation_author=J. Cuevas;citation_author=J. Crossa;citation_author=O. A. Montesinos-Lopez;citation_author=J. Burgueno;citation_author=P. Perez-Rodriguez;citation_title=Bayesian genomic prediction with genotype × environment interaction Kernel models.;citation_pages=41-53;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Cuevas;citation_author=J. Crossa;citation_author=V. Soberanis;citation_author=S. Pérez-Elizalde;citation_author=P. Pérez-Rodríguez;citation_title=Genomic prediction of genotype × environment interaction Kernel regression models.;citation_pages=1-20;citation_volume=9;citation_year=2016;citation_issue=3" />
<meta name="citation_reference" content="citation_title=Advanced Mean Field Method—Theory and Practice;citation_title=Graphical models and variational methods.;citation_author=Z. Ghahramani;citation_author=M. J. Beal;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=T. Hayashi;citation_author=H. Iwata;citation_title=A Bayesian method and its variational approximation for prediction of genomic breeding values in multiple traits.;citation_pages=1-14;citation_volume=14;citation_year=2013;citation_issue=1;citation_pmid=23323762;citation_doi=10.1186/1471-2105-14-1" />
<meta name="citation_reference" content="citation_journal_title=Bayesian Anal.;citation_author=A. Huang;citation_author=M. P. Wand;citation_title=Simple marginally noninformative prior distributions for covariance matrices.;citation_pages=439-452;citation_volume=8;citation_year=2013;citation_issue=2;citation_doi=10.1214/13-BA815" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. Du Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_issue=3;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Li;citation_author=M. J. Sillanpää;citation_title=Estimation of quantitative trait locus effects with epistasis by variational Bayes algorithms.;citation_volume=190;citation_year=2012;citation_pmid=22042575;citation_doi=10.1534/genetics.111.134866" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Li;citation_author=M. J. Sillanpää;citation_title=A Bayesian nonparametric approach for mapping dynamic quantitative traits.;citation_volume=194;citation_year=2013;citation_pmid=23770698;citation_doi=10.1534/genetics.113.152736" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=B. A. Logsdon;citation_author=G. E. Hoffman;citation_author=J. G. Mezey;citation_title=A variational Bayes algorithm for fast and accurate multiple locus genome-wide association analysis.;citation_pages=58-58;citation_volume=11;citation_year=2010;citation_issue=1;citation_pmid=20105321;citation_doi=10.1186/1471-2105-11-58" />
<meta name="citation_reference" content="citation_journal_title=G3 Bethesda;citation_author=M. Lopez-Cruz;citation_author=J. Crossa;citation_author=D. Bonnett;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_title=Increased prediction accuracy in wheat breeding trials using a marker × environment interaction genomic selection model.;citation_pages=569-582;citation_volume=5;citation_year=2015;citation_issue=4" />
<meta name="citation_reference" content="citation_title=Developments in Probabilistic Modelling with Neural Networks – Ensemble Learning. Neural Networks: Artificial Intelligence and Industrial Applications;citation_author=D. MacKay;citation_pages=14-15;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=G3 Bethesda;citation_author=O.A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=J. Crossa;citation_author=F. H. Toledo;citation_author=O. Pérez-Hernández;citation_title=A genomic Bayesian multi-trait and multi-environment model.;citation_pages=2725-2744;citation_volume=6;citation_year=2016;citation_issue=9" />
<meta name="citation_reference" content="citation_journal_title=Am. Stat.;citation_author=J. T. Ormerod;citation_author=M. P. Wand;citation_title=Explaining variational approximations.;citation_pages=140-153;citation_volume=64;citation_year=2010;citation_issue=2;citation_doi=10.1198/tast.2010.09058" />
<meta name="citation_reference" content="citation_journal_title=J. Comput. Graph. Stat.;citation_author=J. T. Ormerod;citation_author=M. P. Wand;citation_title=Gaussian variational approximate inference for generalized linear mixed models.;citation_pages=2-17;citation_volume=21;citation_year=2012;citation_issue=1" />
<meta name="citation_reference" content="citation_title=Monte Carlo Methods;citation_author=C. P. Robert;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=G3 Bethesda;citation_author=J. Rutkoski;citation_author=J. Poland;citation_author=S. Mondal;citation_author=E. Autrique;citation_author=J. Crossa;citation_title=Predictor traits from high-throughput phenotyping improve accuracy of pedigree and genomic selection for yield in wheat.;citation_pages=2799-2808;citation_volume=6;citation_year=2016;citation_issue=9" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. E. Teschendorff;citation_author=Y. Wang;citation_author=N. L. Barbosa-Morais;citation_author=J. D. Brenton;citation_author=C. Caldas;citation_title=A variational Bayesian mixture modelling framework for cluster analysis of gene-expression data.;citation_volume=21;citation_year=2005;citation_issue=13;citation_pmid=15860564;citation_doi=10.1093/bioinformatics/bti466" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=E. Yamamoto;citation_author=H. Matsunaga;citation_author=A. Onogi;citation_author=A. Ohyama;citation_author=K. Miyatake;citation_title=Efficiency of genomic selection for breeding population design and phenotype prediction in tomato.;citation_pages=202-209;citation_volume=118;citation_year=2017;citation_issue=2" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Variational Bayes Genomic-Enabled Prediction Model with Genotype × Environment Interaction" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<meta name="twitter:description" content="There are Bayesian and non-Bayesian genomic models that take into account G×E interactions. However, the computational cost of implementing Bayesian models is high, and becomes almost impossible when the number of genotypes, environments, and traits is very large, while, in non-Bayesian models, there are often important and unsolved convergence problems. The variational Bayes method is popular in machine learning, and, by approximating the probability distributions through optimization, it tends to be faster than Markov Chain Monte Carlo methods. For this reason, in this paper, we propose a new genomic variational Bayes version of the Bayesian genomic model with G×E using half-t priors on each standard deviation (SD) term to guarantee highly noninformative and posterior inferences that are not sensitive to the choice of hyper-parameters. We show the complete theoretical derivation of the full conditional and the variational posterior distributions, and their implementations. We used eight experimental genomic maize and wheat data sets to illustrate the new proposed variational Bayes approximation, and compared its predictions and implementation time with a standard Bayesian genomic model with G×E. Results indicated that prediction accuracies are slightly higher in the standard Bayesian model with G×E than in its variational counterpart, but, in terms of computation time, the variational Bayes genomic model with G×E is, in general, 10 times faster than the conventional Bayesian genomic model with G×E. For this reason, the proposed model may be a useful tool for researchers who need to predict and select genotypes in several environments." />
<meta name="og-title" property="og:title" content="A Variational Bayes Genomic-Enabled Prediction Model with Genotype × Environment Interaction" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/6/1833" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="There are Bayesian and non-Bayesian genomic models that take into account G×E interactions. However, the computational cost of implementing Bayesian models is high, and becomes almost impossible when the number of genotypes, environments, and traits is very large, while, in non-Bayesian models, there are often important and unsolved convergence problems. The variational Bayes method is popular in machine learning, and, by approximating the probability distributions through optimization, it tends to be faster than Markov Chain Monte Carlo methods. For this reason, in this paper, we propose a new genomic variational Bayes version of the Bayesian genomic model with G×E using half-t priors on each standard deviation (SD) term to guarantee highly noninformative and posterior inferences that are not sensitive to the choice of hyper-parameters. We show the complete theoretical derivation of the full conditional and the variational posterior distributions, and their implementations. We used eight experimental genomic maize and wheat data sets to illustrate the new proposed variational Bayes approximation, and compared its predictions and implementation time with a standard Bayesian genomic model with G×E. Results indicated that prediction accuracies are slightly higher in the standard Bayesian model with G×E than in its variational counterpart, but, in terms of computation time, the variational Bayes genomic model with G×E is, in general, 10 times faster than the conventional Bayesian genomic model with G×E. For this reason, the proposed model may be a useful tool for researchers who need to predict and select genotypes in several environments." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/7/6/1833" />
    <title>A Variational Bayes Genomic-Enabled Prediction Model with Genotype × Environment Interaction | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FVH3nQbokaj_fF_ZXdebffXIWzF-Z62Ey7o5G_--qV4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"484769","apath":"\/ggg\/7\/6\/1833.atom","pisa":"ggg;7\/6\/1833","ac":{"\/ggg\/7\/6\/1833.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/6\/1833.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1833"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/6\/1833"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/6\/1833":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UDLw4wKzY8U6rX4QJgrBUIzW\\\\\\\/LhXY7XAbUtj82HWpmVW4n7Urhpbag3R6VKz49JBP9OZR3Tc+CjnC5YFROcLPJPhrqj6+Q2W97yt5XcC3xvNCCPnDLJVcwIo2gwANIWQQgb4kbhHmw4T3FsJ7FHMOaCJ5aqBAUh9bD9iDYnoFk3RSDqEpFvWcEJrD2CXQO4oCQj32g8i4arsk5LBv4cEekbRq1qDvtwPOAdIXDMU05AvuSjhOQvKBonXFiof2AuzAjBoGpgn32GGZVni0LqA\\\\\\\/7LLp+krZ\\\\\\\/aw4fAgM17m0KTXjz+qehFsADD+huXesGwMStzwJos7YyZCser4\\\\\\\/YWlfn1om3cSqz67nKPqPiF2Xr3blee+9aDsjenO35lZINr33BnPsGx48D1rNIW0KtXj0RiC\\\\\\\/hjtADWiuuJtMbfU+0GRrdjlaAS9yV7QMawF6PHmQ3mhgGxJJzNCrdTavUBdsQYggZy2zQtunrNVwBSoCG3xqbL1giwKR0TtCfRKU3IwTe+aIC\\\\\\\/ZwdTP7C+Qb\\\\\\\/DiJkaHAGqYfuKL3PlvKO2QR2yhSmnkKF5ew3D4NSNGSD7smxGbIu6MpJGyMcVeEBZaZoy2NnwZtzZangrH5P6eUK5c6B29T9jTzimOtH2o2szl40LkhI5ihUg9Vhoa45MBx+ila3laQOpILz+WVm2JMGh44B+mjGen2cqoj+wLert9hPNzoJmz0BdsqKqbvirtK4Xi\\\\\\\/P1SIADzZfoPMNWzzAgeH52V2L7tVPcE8jsaPUeJzCTpPqLTKIMPyrDdXqu3+ey1cE\\\\\\\/Db2NtXeS+hkjdKo1LChXge9W\\\\\\\/qtGBgBvlRoyWZj5GlPkK5s8vaOBYJndgys3ZfAyFFmgHwJd0ChdC\\\\\\\/uurtp7QTaNzTjnPxqMJnR4Oaxd9uQB32nc8id0RtVBmbldBqZCxd7knYoh2zw+h19y33BaLZY\\\\\\\/xbzJhPAEnb9+6hKDVq+WnTK1kaGJS9oiALzu\\\\\\\/WaiT5emkYs1\\\\\\\/PBHCRqh1yvQ0izYt5MfnEZEIqWVO4x2VIjWS5a4oa5QG6B+UvpOUXT168VXDXmQHyhYujnuq9jvRUPEVSh6YutSPQnILJwGHyULkfJP1c9M6MZ7V\\\\\\\/SL9uPAaqzu4\\\\\\\/ys6P8Sz7sbUGRslbwOGwjMc7m9tO5CECOt5PdEzW8BKRftmbXjFWuRiiouxtBOQUum9jkzDmolks5ohodgSfjq\\\\\\\/1CVWZjrwPEpwqfGw5syctc65rA3C2zfoNCIEthYytwpbCUGTNtfkvVqY94V2L7vJFh3OEJIq\\\u0022,\\\u0022iv\\\u0022:\\\u00225rzAXGFNi+s1XhUXv\\\\\\\/SdEg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e7cbcd1cd66f6c08e72b945844bb85412343c0fc5c84a0c666852b979a2a1df5\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uLPlid3eFg3InwEOdtnhpBzUQctveRBkbEOeIjm\\\\\\\/tpBMECnRcRYn7N9qEAlFFleER7MgUP19Tk7oS6VtE6DPDOJB5e9lzuZRR\\\\\\\/M6qpVmfu7ATIo1EijjcmuFFGW4VDQKQ6HZNemC4986vUO3\\\\\\\/ktJuaoVAzm11wz2sNxrcaT78p7gQ5ke2cQzrd5T2NW6hBqCsH5xpD+bFXkE36gNBIRCIZkSALfSo6PRUgDjb3Zh7GLy26YWCXuImc79HmLCwN3PcsP+4\\\\\\\/L93EOobATwwjKTkzVtRXRpJ79aCsRPt8muPEgJZ8wHSiFyTMcKMsZGzbqVfc25K+p+AE33wSK6vi8CtyVF+iD8ZFsliFYrHEuBeSATUGzX95S1qW7gCTIVDptZ+XMcoh5kIRmDY2Bnb4nfW4vPItqzx19OyZ2a5QjMXouYjG6szK3rkU8qp6UnpugrkkaLeTJzBsb\\\\\\\/9tAcMtxj3pNjkTC+ynxr4drf57igwIC56PdzAUTcVj+\\\\\\\/FZCioi0Q+C08HeOExZf55PGGgIl3DEczlpDhFn\\\\\\\/wJZwyVFoUoLGk2gOOnE\\\\\\\/4Qzj7NMwigQ43XVnJ2QQ2\\\\\\\/+\\\\\\\/uxwJO3UOYR3Q8OUSQ7iIwW81mEnxkiYRahE9IQOagsED7seJxYzK7+gkmlsq+ovLcjs5\\\\\\\/\\\\\\\/5PK8kTVqKNi+DkgvUqqglNLz84PD67T3bbPWk8mGrROb9eE7syFVoMYSQB2MzxupUL2jDuKNYGnv29ksTVnkRfF\\\\\\\/ATPmymEaPKKBCHz9bTXK6YkHHUhyQBjd8K8MPCUfpZc0m9o2CCyBCi4oL4U9gh3twAGEbpqTdtu7D6VA35H\\\\\\\/UxRy9k0CvZ5RH5GlPmTCnIInFdXOFULYZxzuE8rCVrXgg+4pFCkymx5R1PhyJXVxo9eCSi6uMRu++iqHHICmTLmMfUTXfJwFLoIJ4vIarFDOQpkorrl\\\\\\\/yUH\\\\\\\/Lk8sMmWiS+Z1GrnsD7iuWxapxWC5FF+7DJ+PDt9+llprXnF1VPZ180gmHwkGbOTTQqp9jPEc3tOL6BvaYCEzOF8czwKDvsOGV0tNjoqZG3yxd\\\\\\\/iSrxi1emVcB872vEZ+qjYCdHYbhskPAK0Ul7VfA0K0\\\\\\\/hSKdheZ3vuK7QtE4l95vEZcmuYZXg96gUALNGpXfOmCR1vMeg7iM9WhfprUxuVed6tjQN0nl118zNckALIVzQIpjsuAgltnm2jrNOra5IIPNxC\\\\\\\/6vwA7TY+vKuS2EZNraXBi3wzj9BxEY8ENH0uCOd7L5VGHb3qI17lxYrX5ZK1riZNNblCJ6nZ5zyeAM+2pvH26yEzHyxPyPhs6FUDOsoouwIMuoBUe0adMD2WKaQQlEOOo5uqQn\\\\\\\/Gg7m5DThpDgR4NN6Jfgzyfs66P8I3LTM4x8V05LqCVeZeL7JXMURW6WBhZunJdyjbtD90oT3B7XNpve1TzLm8dNg9d3xuFc71snzB7WpeJYfeoA8PqFnyRFQGCavFsw0OHO7LaMWFl1IrYo542dfEzY\\\\\\\/p2didfmybGpUJlFmpzXZ\\\\\\\/RIsatsTiaa2Wt4aZiHET2TIl\\\\\\\/5bS3N4mU02DbVilKBsJ\\\\\\\/GSewaWSMwNac9oLj\\\\\\\/BytNxaHlDTIWz5t7ee5bSIMK7HYhwn5WfZiPpy6eLpz1rGtRsmiJQgXm\\\\\\\/SQyQSdraPr5GOx\\\\\\\/4liTTGIfIpF8AJuOsreKs91tl13CAgug\\\\\\\/epV4iAlxelcZ5JHYBoPQfgt3epsQbHhVF+sLpDPca3LL1jUepQIssZXG17i0XZJd0kL9dF07nFWqgC2nih9dEKerYgJ8+nPv4plKJL4CUQE\\\\\\\/e6cvBFvXHAT1y7KF0nihjumzPoUrUGDfEoUNkrJBrSBu5ZFl0ZcWC8RhDqcQA3JdT6dIySbLKQ4ggFN+OX1CafAYx1jSX1A9n8PsOO6IeOvvfVNBv4hUjdn4vjRmvECN4k1yD1J+hbk0NInL6\\\\\\\/0GrIF\\\\\\\/uaT1fo\\\\\\\/ok4w\\\\\\\/l4OOkazKP1RMs0wH0ootBB2VEHSmx4PCX0KjR1bv1678Yj7HXqlTyitcSKO3zuLoKX2f+MS56eexunODL6Ui56uWMmYaBL8j+\\\\\\\/7QzjNwkQwopjo97vupvJGNCTqaL8Qttn5Dn7FVk+S1rwbTZcQltm+jq96ZrDegBnMOXF\\\\\\\/DIDDMBzHrhW5KU8r6gZ8G2dRiskeshWFlxrXnbcR+rqd4rdPnwZuvPjcgP0QZNZ01KdCWofCZ7L1KfZyvS3S+FhRck4UU6V1VkHyP3IWuwMlrmRShcHwRweBBkzV5ML0wnPqyhS5D88jMK\\\\\\\/Ct8lkh3qzJkG5KPb48REh73UGZcKouq5am\\\\\\\/Pe\\\\\\\/4UtaSoxanEAkuGJphtaroMNWkf07Vp8UKxihJdNdFld+5gSo\\\\\\\/lw86\\\\\\\/wuCcPDHyfuBGhxNyQtARc47p3RffB6Wd5WMpMXG1t9Sk9qHW1C\\\\\\\/YrIMtU2H5OSd2Kdngio0D49\\\\\\\/2DlyNTgLj\\\\\\\/zFxHXI1iMP9FqYE4VGEXUFT2B7kyARCY\\\\\\\/8uJJvIrgcn7A19\\\\\\\/Eu8JglfOaDkfVVvglxbjTgUFd+tYeaIM\\\\\\\/ngAGhv2vPPRyMikat0tIcb\\\\\\\/trzTN+863ig12CIYBEJSfalvcLwOLojH8x4PfaYlwKaoPE4gXUU4g0RjSP\\\\\\\/JgKTsyB6vvpEAxkjXX5lTu7CS3onabs2UbmUgGFewv304ftPNBIHUyURLSlK1NHTjgCTG7+wglGorROIhaxg4w\\\\\\\/3zfaytpnLVTBCdSWwP45LX7Ukbp55kFcXZsXkXVRx8wiskMJJP3HQZkOYUUZ1ymciRnQEKeFeBMJxYEUEfDoVAodjzuc6szAKTXCTH9ng\\\\\\\/8APJgHXHelKrtgUDRvdHe\\\\\\\/hwqc3aW92dV0ZEOpGqjuUL7xzG+v++hWfwckpoNj5Uh\\\\\\\/CPpYwitEshtoPnRRbUSirqiKTV7uGdjA+InIMnnTtuGKBH2shmFFj7Ng3OxHtWnCab6w4muSUTzJEz3neeBpoItOvqPEn69SWWYrxNrRal7JTgxuJBeoMU2V3LcDcY3sxnFLTRer8ncqPpzKUsHebczNOy1fDmnxhIzaj\\\\\\\/EQ0mCKMr+fOdyVJZjOwmDQjxoIC8VfODtYHa4btBLg6f5uNkV8fm0IdgMlooQbqmY5dHRwQmeenKyjoxSiUjFTJeRuarJA5nyGNBlTkTtIp6eEYRhohTa++fEyAmK1doQiGOL8tm7b6wsAJ3o0P0zfmbCE9IXTiHFTwGm1\\\\\\\/G9vFPUNdH8YBB\\\\\\\/dorlyBpGG+IBpVX8FfwnEoptXCwHd+klPxf7HIo1EnCUNLsIVpEVGACjKwx99dQI7Y45ehLg8fI0sz70zKVysyFP7mNpSSgbaDOKlIqErpNF1Vkd5YhHdl1q2h9\\\\\\\/LPbj1dNDFm0y4adAj2qceVuXjKGHW5f3V8c3hdUk4O9FSlSinG\\\\\\\/6J4ZkWCxjggOTMU0OCdvUtIJOsmXvVe5UdQYmEjMUpTqBQV89gR3H\\\\\\\/+poGCK1oHXzAlOufwKDc4\\\\\\\/Ct8awsXjK\\\\\\\/uW2DgT+aWwqYC\\\\\\\/RjvJTvEZhIgHlPhFf6RBnbxeVq6feQE9LprHQXcOpZD5USybkcufiMl7IQ3JmpuASt4V\\\\\\\/0Po8IW2tEF6t3oekxKyXpn8WvzIMuQ0FHuEJZN6A91YpBf31QeUGMdsxWDJXglyY7d7NBas0RTHDZyccaeu4MwakNwOEc3kC7YX0x31cC68g0AldKxGQ19sNJ6G7VYwG8\\\\\\\/LkStcUh9qPnpmPRAkyLh6IuAsNyxBU\\\\\\\/PPEpfSwrX+07VdvRvpvPjY1vcyIjV87eZZPTj7IHgp+oO4glCiSpiGl5PungDCJOpfTXW5NbAZF8QQRparh3gx3zHCLjz+t78tEvNEm09Yq9biHNELaLhHT9qwU2bNey\\\\\\\/cM69T\\\\\\\/24mZZGxk6utXhupjlHCHdpxWymxJ\\\\\\\/\\\\\\\/oMTfuSOGEi355c58O7T9qUuX9RaeqqHYYR+c8Y\\\\\\\/64EdnLJzVSEoXU31VxO0TK1bXiO+8RgJgOS7qGlSSICjlfQmOLRLqO6hVmMaarcbb+ZCQLMxy+jmaAhPkZeWJ751kGS43V5m4UaLewEYOqwWzXvwoQlRbWfikM\\\\\\\/F0gMdyW0sl1tIt+Mbf1ZxyykwlRvPYnOq9xppxR0nKjCCWEKnMwFHzQI7P1Fi0n0H4j1zmiunM9s51CjRqTssmCq9CG4ddsdboG8exgo7+6h3dFNhPdq5Mgzk3tNTpYvgIAMv2Q12DZfQvaxBZ3CkOEcGDjTBP7FqeatHbUHK5MeU5dnjsw\\\\\\\/5PvlTjyo4qEEc9phAl0VBNTK8qzkT8TZDApUY1y2DAg1iqOlPpa5S0vGmWlpxS4aKa57hfXfF5JFN0he7RWJVTmsphQAGmhtuHwgW3CfNzaNChgEoT2s67pFCWDKy8RdsAOAoI6Np0\\\\\\\/q1Nn\\\\\\\/uvr8fo42\\\\\\\/bHkAtGpCu6rPvDpbWRlbGdHqfPy0qPPmnYmz2I80Zd4ywSN538Q+ESLKUdb7SGEykeF4IITJcmzFt2sQjnovHru\\\\\\\/55HlKVkpA1ctJJ1KaivWUQ698ic85xMZxRIGVK8VCwGKPXyR5FBHrRUp2ofKgCjpRs81KM05s+88To8rRTBqWsY0h7YGkLxf7feT8Xt0wzns13zjVM6o8a0haYJvbVC0ioNMaFHxWYFP7f5H3vIGlIVX1u9H52BdsFIlL+nZXQZjhBK3k9eqv0Dv\\\\\\\/O3bVebxvVkpr9iOnFejNwqvLhvAG7+LxYF2LlzFv9PN89TkOb3Z7hFCAUSuFRG4M32eEWP8KxixNoSO7VR0nvD03U7BXTAT+ZUsnoX0QiD5PYG7AGxHaB2VS9kjep+lJOVq0d2cqV+\\\\\\\/NL1gHdAPkbyl09Qgc6u1qm4+fmaQt0l3Rew1wdzDS6SPngNAph+D8JY3v812gcTR0JQ7s3jLrDePsiyB4L9rQgA5VzvcCV+Na6sWMhJV2J5tHgM\\\\\\\/DFYH09C0dWZFrkxWfKq5\\\\\\\/zQP2suARaiMMi55DLmiOxYwntcrATGfTcneY0y1OUNzGvM7lgtZG\\\\\\\/mC1DRm9M8Xbe8LrsZ+usr8wp75gPGF7hNDeR3L\\\\\\\/qZPqNKGCNuf6enGgRE1uh3k5Zq+Fs8w7NGMoSfQDcKq6xN\\\\\\\/p9ahL2DTVmiK8pwj3Ozt3Hab2x8Y2CwnGHA16FHDowObvvsBzGnmvJ0f95hICEHWLyU5SmuqYaTx\\\\\\\/q8BiQu+sNRCuiBBbvgOGJiC1VpOMzOvTejMSp90V66RMNShU\\\\\\\/YDQ43UksSFOL9zB0MdqvtmHWmITee+AZ8VjcbsKKbsQ\\\\\\\/Jq5pU+wjDg7lSebCl3zddHZwZLwCx0vp36SNWlu2n2eHF8KgJTSb\\\\\\\/Lbp0MKlI8VfWopNL25uV6geVX8e7Zc0mIvNQrNJUUGrzosSgoWQu7FewlTNOJ6kJ2hsAxiK8yZK7YSkpOAoiy70hK6bVOx8PPzFvJ7BvxxGN5Kcqg3ZTl2B3h7bLqB6BpzzoIU8Hal3LR7tnIDq9fVSyqPG1WtXCLZp+2G\\\\\\\/h8toqJX8KAdZOYLIvEOclbNJEiA5qxSOVMkVK9wZTX4sMjF+8QvA80c41Iuj4Y7q2iJVlkzBiVDJvJEwkoejTCeTZ8TcjyWp0uHFBm4WdgXq65IQgkzy8r4opP0X6iOSitXgqTyNslte4tJcgLHrd7z8LhfVSBZ5bXGPhl38BYgIWHaoNvaXxhWkd6vMWEHA0qyEA0gN2Cwl4J6X3z5AKXTAU7GXLWXJVzepSHM8hK5O6wxP0syW+s+FQh1cQPIssgCMPEnSlxLtkXFSjLFC7Og84cCfiThfXVVnmQBKDeJgDY1Y1auylXgRK8JxvsLqv2I0ezWzMfONmSagAl+62jYOYYVQaifCzWWkjfKXqnQQbHmEf57bGSEpnjGRxglsmyo4sTND8AlR3dTu0U6KedAyRTqk+XrvNok70zK+J4BDA0JUlPGcTSV7pPFzmJBEUjtFnbbTH4VBWC5fQZlmH+Tc6XiWUIn9Xc5uwULYBdshDrAw2GVnf0nhJ72oddVnMneF2ID9+9bKiNLXH+JUMGXgzK\\\\\\\/qdy\\\\\\\/NaR1jx65IpjJeOWss1G9Savhs6imMuYg7oxCVEWG4rKt4PxomYSWf1K42hq\\\\\\\/i7xlGQcj2y28iaPO0E+TofyWhjKxx3WnavAkjomFJzgAJRUOUnpLQwFa\\\\\\\/2Upl8ci0RqNLGexbXNOIJs5ufh0oiNLPHw3qjYJ3Z5mL3c+B\\\\\\\/Za8R1p6lUi2f0BuqPIiZ0QT38w0gC8gIwpAJOQ3PIVlBMz4otXDKQgc8j4tEQ5skKjSfJdD2X2Jjcu4dz\\\\\\\/nen\\\\\\\/QjSM4nbY45\\\\\\\/8Qf1gxSkJVMb7K4krIAWs66YHEq\\\\\\\/wmfkpYQpM9HRGpGlnogd9i+tjfqmdVfBvZCVNcQJVnuXtIRhNIMSUPiuT6pFv6+YavDp18lQMh1koqjbUBlA9vUAR5qfb5zmd4vbObcjqz4lpY4ip0hgINTtpjwxfmFD5cmnWVPqqsXyZd7F9ZKqtb9Xktp1h5aD5mZHbYhChkNAA+KsrHke9ZaVNyseFHKwUp234sNnbvbWaKQEMGulQ+PEbbFH2RIGeiJi6cW99rtV1+ni7BfsC4NBtJxdqYSE9FGoXW7T\\\\\\\/IC4CoiF4zDrRt9cRGUU\\\\\\\/dJ0Lq7ORk8Imz7zuVuiV\\\\\\\/RSbFpvZm6Z3F3atjhxdLr0pL819w1\\\\\\\/ecTS5kq551DatGw3\\\\\\\/U8uJyYiIHDGbQD2j1c9ygKbkq7bWR+NU1rPP5gh63bGiPJb6CKA6PmSrUUDthaZluikQZ+NGvQ\\\\\\\/GtTbPl0CLL8QIVNT6sA0vLl7pEv9OWHCJauu\\\\\\\/i\\\\\\\/rrqtg8YuhpD6sRkpjTuBnh1FDdnH0Xg61pFbjizkNbnP6WiRgfg4qmcDis9yLPhgkOrLIq3T+7sjjrycNA8dRftD\\\\\\\/q0C6ojHi\\\\\\\/UTeJ9EPkBn3PbiqO3XGCRtO6RDYl8YKSBK4WR2ktx+IIeJZhfWJEpA+QLPpTzxcf17DGgdtOcspIWd3koDU7uH5SNCwz3O4TRfhhZ+J9DS96QBJKkgU+42H4bhZnY+48EJCuDiT3Bvqp9\\\\\\\/6dTYnvm\\\\\\\/aFu+uLXaRburMaPKOQJxZGBnChPPpT8prAOb3hf8GOaOpf2RmxUjOuvdZq\\\\\\\/o2Ic6iAPkaQKvTzP4Orqjx\\\\\\\/4lPgLcAxJ+QpUfU3oDsYSnb+oRkkijvcmn7Lh57EEMZkA9kWehdXb1n0rUhBJExvQ68ROvV2KkS03N9KHyCVwIqIXdQQaTE1Rp\\\\\\\/jiNMNaL7m3bIaTyBmC0DMBgnXra7li+eNjlXsMbJqaOW+Z6v\\\\\\\/40jl6Ui9SWrORRjmD+CvMGc\\\\\\\/tOppH4gc3kS13BZgCXfY0MzA\\\\\\\/3eU57w6rUqnyv\\\\\\\/5ZMLWFG1L761DLTHyIHvwJLSKREvNqfORGkcI17XElqNzLi5e2Nc\\\\\\\/XlcbvO2NfOM9aKIOKz+llkhc4YWBkwQT+hgzAFORU3lCoFHWnvLfpuOXimC+0es91Y3ZbYEacycL5ZQsPSiQmKWRoB6f0411AVURGPv5fUGlWbU2Dl6I4HdANaE30oviFEZ3fO2guGiETplz+A4JdleftAYA2N3xfTr4JzFejgdXAdPXwE4j+INMMfer+WzjTxvb+B3I2+ymYSNHKRhfzdQZcQwDNU\\\\\\\/lb1Y8EvETYQPo2cQAp7WOdd+OTcxuiRm+xRWPayUhV9rGO6WEiGQGTwBzOBAvW\\\\\\\/JI\\\\\\\/nvBR5sillxZaJPSgiKTF8KfF26AGZnCZEGMzkEXCxXuicBX1wwMmth4zeip9Nxd3mOJSKbAjEJI+B8bmsc8xoLmNf\\\\\\\/rp86PPLLGlKyHDB+7YFvztLhPkq1EFWK\\\\\\\/VQgMXIu8p\\\\\\\/QwVTjC1I4U0u+gTeWWEfucT7aLoWiEZ9TrEopV9K5bPsUraWuR+\\\\\\\/PPHY15\\\\\\\/12c\\\\\\\/SEBIMqsl+Q5duzS\\\\\\\/4KTCCutJQHUogDnC1ua\\\\\\\/auGxLLMHw5Rjk8W\\\\\\\/CYUBIFTwVccroNs0NpqTpKq\\\\\\\/wNXl38YSrFRj\\\\\\\/baLH98MfDeMd+aamszbmoPT3Bk0PP6ZBT4HmcNawrTGYBYlt0ipBDniTHxdBc\\\\\\\/tD9wDK5O\\\\\\\/CruAsQpN+IOrwK8kxis6DRNwjN5URYnJ4R69839AFa41XOjKuNh7izG9VDLqiwVf0UZ778a1T4b5m5l6nnj6yKZZw7aLy9uL15T1gcRk2qYCQA8bfXWd\\\\\\\/I+QAaTMykmX0TFxUcRdhlz182Ebpp+Kv+8S8WlkLZMiJ510fBVLvEAaCNcvMqJocQJB4cGVGIuyXWMi3QHLT2\\\\\\\/o0zCssf7wD8LZqQYontCXNjTkslnvGgVyvL5FbXn+7gH2uinklWGLWNbDbgpiQlIl3u0bQSs8ovffofuGjth+\\\\\\\/tcQaOedZDJcYkAT51S6ayCDf2MaQel3OtOqqORvGY75bzPwI3cJtiQHyPnT3EpWMgGVumsP71EC1Miefzm9s7fbB5LmgzAKY\\\\\\\/aZtPKi05uk9fm2s3\\\\\\\/ZTtAN+m\\\\\\\/4V4IBXLmnZNmEeduXvhPih+\\\\\\\/zM\\\\\\\/rAkXfD4qq7cFH4dyl+eGs1cCxpS1JQpf59VBAlm1ZPYp9aS8Pqcj52o+XcBnP\\\\\\\/iGrOqJRnwBADZqn7KJifBvfAfCANwHdml\\\\\\\/yBWFcEZqiZr85m2lV\\\\\\\/owYtllqA6zQThVNtr+\\\\\\\/idCS24dxNc2rY6QEZ+Frg0WcKIVHbevgy07VECLgYltfpAluLMTFQhxULU\\\\\\\/JtKLQeqB1Wqir504Rq\\\\\\\/yVsIK6qxdNvVc7XFKymCVXhZvj5pQ4NQBaFTnb8X6OcLX6oEGyZuIhqtSsU4WQF7z7kCflifQzxOUMSrLQKe9zyCBFBGI97q5uwtvtBJcemJuVokmgqf68PgCDieqvLOo24wOxUhbZ6bKdLhX\\\\\\\/ND5H32KheJdvS7gHd812WU1rR2Yq4srsEA3uhWZjEebMFpaiGYeDrNT88VTg8HrD5Uw+suh\\\\\\\/LIF6uKeTYd8cs4HRp2+qwF9XihZSzNOjP1I6QZWr9QfdjBKbE+0U0+oi9UObgHyZXA+J\\\\\\\/CCmeC71on+EI2PDfhASeY+B3QgpTms94DUbpg8Ifr3jR3ALZkVJGMRJBD7EdDbq6wkSBE7WkDWcQXQPem6NfS1poeIvHxeMN8qpRZsQpAIGtXSeH1rhPV1e6KiCvRT9DlJ4vy0Nmx1JdY7RAeOr+LrK21LvXj\\\\\\\/j\\\\\\\/WAwUc4sLnXYkVtMxpJAgftP7yXzwr9OJLjX8LxDcqYkTw2wk17j2E\\\\\\\/4AwJ2DF7sb6u2cej4WUxyb1Ay34b2eynEBNmGczdHjIWxlCTv4Cs16jSoEgIl3xs8sq5OEk0w8OxiXszBt2UGUl7R5IRaNwTiq8KbUnzhUBeE0eJpm8vJ309D01Pmw5lCQgkdHgB1\\\\\\\/MGIkFrqT4i0wgbBCOjU+\\\\\\\/NzuX7leaUDHVUTZTN+lG8SszrKVAbNKGPNK2+CVUfkxzgYH8vB\\\\\\\/mUFiCFY9Uz1tcZAjl+RlFnA4InaLJq+wiu6DTcb3+awl8HQGoaQhJC8jLirV5rwEoIRqR8VDyzH+K8kluGqiptrIbNaLb5+GOww+f6nd\\\\\\\/WcoAz3b6tvfc3RzwVdCRtqvVZ8fczGF7CJx0032KaCcKB9RvASwC0Jl3c5XsY\\\\\\\/K6+3tZax4tqQh2TZ5pMEoheOliSlXeqgV7DkuR+JKgRz3O0wlXWVux+PYTelHumiKauz\\\\\\\/aCll3LUoVP1JHNDUkOeyLM6QONzMmmOrkb43\\\\\\\/dv9VUNI569080DHuliXa92qmogkJqT+ojQLCiZFk6ItA2zq+Eut0wYW02ZVkq4Y4z+a7X5Pf29y5iGMKXt3fxciZG2ujFeNxd+7ucP3bH3hKfC3nBDVcqqGsEomcmiG8LeCoM47FGTGNFEuG6D0pvaR2l81MTb6r3km6218bJIVTRI7PH1nXqzAOh2hVfVrlP6rIoUt7GlaCVrVZEDdarVdu0cdOcN26HfG7+kW8WTt0PlOPMpHwdCUB4Hd+HzArcZjtyFoSy0WcKFUPkFlB+lqTs\\\\\\\/6Y8rCl1JKiJ+4ufySLUjG4O1OhQ97HFLotMycPXfqjR9b0MQTeJGQ62o4QSrpVPl6sRl4eWIGtv+VW2WeXRkaQhtrPq\\\\\\\/SU8ahYrEynORN\\\\\\\/4Z3Nj6+BuwJ1uniwitVQaTZt\\\\\\\/gtDaRLQBqJdC1uukacWmONJGFJYN6mUUQmzwqCBzk6pYBYE0o4T1oHMc39rrFRnjRHJ8f7qB4j5cocu4\\\\\\\/rPyRQs6Nd2G2OqX1uRpuwdot3brFt5B7snS2R19nSJo4E0FCiaE8LSsObr+d5zNw1Lez1HzGVWK1Mi+hqUB8Yr+5IjIQCx2yS3nS+eyadPG8G4x\\\\\\\/o8+FicMkSxQ1tUI02GCfpzNTu+dW2iv+9b2MbWDEp6r5zoNUw6F7Pl8RFdqN5FWX906OTkJrTeJdUrScmD1pNFb4cy5Xl0NOzWMm2BC8wG5XK7RMepPqxstkIO7KjtBJXil2UgTsh4KVfRgPz0X1g67gonY3JbzBvf2u6pv6PLHVEEwWUKZq5D9km3GEvemBfHotGsdZlAvPLDlYMOhCspit\\\\\\\/T3oaS+MFv6G+\\\\\\\/bSKjs5+QJJ2k4qT0AIN0ThFsxWJt4\\\\\\\/gv9PSDlnHLuJkXuHz4rTjGEulJ5g6o5zbHCM4sf73s88LEsOdNENhr8frR+0SowkD868rBspQgJOWu1e4eE5gssTgQFB8oiFFoqB7oQpgzq6f6PIuSrA6GZ0DSMeXDrDqeTcfwHysd3h1CSaKyQqyVTXLfb7e11MsL+VvGojzJ1xxZ1DIsuiozW0oz53in+JQK0gWXCtK68YxMwec66K5G9I8dkTwS8CBaeoUsas9aGo1yBju7D7dZEDmfxhOgTnx3DR0Xp1s+s+fi+FY0NrAPLHxdTtERk+2YxqMdklJPbzTOhareIQCaU5yfe6y+mwJee4IEvQOcUHFbnTKyh5JR2kPPDpG1NAVAYvwNy6l4QFnfYTMrvW7uJL9nk9Q3htypFKUC4WQAQ45X6kjYFwfLb59Rp9Mgk3QFMF985owUs2SV5EqRViDywHA9SCdujMLm59gnx+mdeexAISDawONP0SJryw\\\\\\\/LUFhg7R6LD3Rx+t5jvPrm76ok9CZ6WIvV1w24OhPqfuW2ATnhs52EURQbBjGB\\\\\\\/AVKYQCkOIAQZAOBrgoCIOs+a\\\\\\\/ITsmp4zAwitUuKChn7Nj245nUdZxUjZC9kNE47KxcrKlyXi2dTdsTIG4\\\\\\\/3vTbn2OyRxLUKXYZRqfMy+m5p1NqG\\\\\\\/wE+jUWrbKw9i73oCBg5I5URvqkGTjlv4lBNrn8CYs73OcXoKwNLjSyIfXXOrG\\\\\\\/Pyqec5+HM0INAyQfP3BJfqvdn3Bx9LE43uAm3TnpPNvfXcx1AHQrnNeBYqr5Gx9OEZVfohVWUCtAzsExyx8qdWLs8sZ\\\\\\\/a2PkVjjuqlyHLg0LglQAVfLX4xrR6rjoKfJo5W52AITrVIYb\\\\\\\/xTvx6\\\\\\\/eXydP4b4mwlOjS8U1aeqrWlgoKaznm5QGI4ceHQ5wCNglueFld8mUY2e8\\\\\\\/NyDajTSP2QngV\\\\\\\/t4XGBsop0\\\\\\\/irMD+f\\\\\\\/cNA9PcmH92Ik695bw8WEKB3i7ZzVMSL0Qtefa2x7XW89FyT97RvIOilDjQMIhSpcsUWZLHSbIubhaLMxyVeqpwp4AbxPfV2nE9ZOneLz6SB\\\\\\\/w7k+1J2WlD1yubBblGgmtFeN+kVRfQiuqijghujIzSDl32PXsvT3OhdXEpaOsC3XmqGMdhZnvidVhiAAdP\\\\\\\/oZj047MG2UnLJbDm9p5txSvTAU48ud046XA3ahvc8e6e+XpOhQOsbmaeqn6ELRESrXE2hkyduHYJsOWcqv61QPHH8OdZ8yO0nSmvsZAOF6gzPunLquOMyoWndqgQ+D1DPgHqvrhozCLjsJiKOBZW5FlVeAmaN9NbSbtPKCkJwpCdfkO72sXYYF1j6hf\\\\\\\/ROwlamfbpZAPLhcmCbCO3M1BEckqJ7A80olJekkzIsfBTW4D9jAvvezfATZgLybPciGL6M6u9QGDyYQm0\\\\\\\/c40BhQJ4AcYoZ6DcX0zafdRo3rOVlAWvszizs9cfq2XPO9\\\\\\\/kPiBiPDicM2XABDAZ9HVXJlV551JZMweXZTmRlZiTnisk81YmVSGD9cuwSaJJ6sRPAf4xZGBnzxebM\\\\\\\/HUVa1Breoq8CIizgrazAxas3y9u8Z6xvJs8LgWIAL6X8AwWX4+BXECeLvDCZHAiVT728CDTSAiXfA4\\\\\\\/aIoYeTAr5Lp376gxEa+TeNCzgBbjcx\\\\\\\/t1+unzqgOUbr7zxX+oZRbZR8sKaV7d0dxWn0DzIIBH29gA6eIDL9IJmRz2zJtJ7R0S2rIxjvfXQdZzcpkDDU1M3QdraIRvSvf\\\\\\\/8cnIKpEHuA30rXTT1F2DDHHFjdbvIGaVuLvAWn+pN6TBYZhAD7sb6b\\\\\\\/uPY+GtowNdvVeGPHsg0rwOMUD\\\\\\\/\\\\\\\/R53uJYCBldQw4CyIh7rkj2v8aC8mQ9U7HKjx+S4VJ99rbnkbL+hqvJw8iq306zE5FBeWCcC8JDt1pUfb8qZ25I8mnXxZ1hyjPFI6G0Mqp9mQ8KX+tZWFqmFOgq+fl3X+PcHCds0z+vwQzCug42lg5uLBXtiwdIBND4PR5hCMAzVQbF6fg\\\\\\\/Imsp6Xz9QBj5y\\\\\\\/MMFU5+nyKnzXSiDmjM3dK+XiWYMH5Jua3XWDMnCbfk7yl0aPeMmGgATlHPwAOpcijOsrSRp5ImbzVWjwyFuOTvQ3Wlke759XbbgYyHjZT8CQZ0GzZn0eD72hwJpwas33qC33j140gUopRss8\\\\\\\/v1wmwNAm+Ezioq+NKZu8AAufK1OG7J8bH7qIND8suw0SHrLyk2V+IdYoBztAQLbF5d5u+l3Zlp7GMsR1qOV0BHBlonpdcD+jowS6Cg86mXNAgHVOH3NUpZVkMT1Pg4f\\\\\\\/W4hhuifB7rY7TlVp7fmSOCl9ITZrHem0ekWxpQNdjf+AFWupnE4WqO9J3xkQvzuAjoF18fw+XhYi0nIJDPv38nclajQV3JbAbVqcXY9gB6tYQCRVXf5DCaSAp55sMb78CB\\\\\\\/P1yCkCWscZgRREt+YuNTUDKNljcJgZasloGR3j8zCLTZvpyV0YhyAh7yEJJ2B0py5kKevATA+fOLViMt\\\\\\\/KG+UBgJweQHCx5uaXgyVa9dY5gWcz0NUaOQtlkqM94ndqGae3UDWRUJR4H68pQaKT88PqzMKAQ8VvbEUBCHwWZL7KtEHM0dG0bgxcpbWwScjGtzwpCaI0MBcPgQ0NpDGsgn\\\\\\\/Or8NTSNxbPwmADkD9wSqu9cxS3Np10CF1NLXSV0+g5\\\\\\\/WoOFG+8\\\\\\\/8aQCXoUH8c1re2wkwZhI0pf6npyyyqbYtcRjhIeG4cu\\\\\\\/rkr9PAVBUjpEvcVKaUWbADaN28mZA5c1OdmK3PQp57C9iDNmz0skWzqRTQHcPabFO7Fun40IhLJ3wEjD9Ke1UJwy+S8xYtlLyxvTGMKWXkHeSiha9X1Yu9Ty520nMmFLIXdq+jfwSu5PabR\\\\\\\/OpVkcWQ5dMG9XMpTEq+5WaclES5KyaeU8vT3TbU4nF0+FkcYutPoZ2FfUTjN\\\\\\\/LJreNK3QWXRLJh+0kVjvFQz0R2wbFMFYf2CdXacrPfmQgr9UR75ec9uOFcKyaYbB48js1Rxe4Tc0mbv8QvOjdt+IdHWWfOkLdM\\\\\\\/dbbl73d0AAn7sTLdGT+QhI6k9XqwMss++KeuKRbI7i37QCd7RdOhKWbmvpxGyQMQRILhyMiSzE\\\\\\\/Mo4XrGg7roj2gSpE7SMp338lCYE097Nkaeec4mpGvqNVb2+6okh15bIo0lbHn3b52qBXsRjopYx5YIsWUZv4rhU2cwfRuaCHg7Sc1LsEAYwG8MtqKxwqp0nR5apXxnhSUt\\\\\\\/5kDEVZYRJ80C1eXSo4v40VLHI8XZW7WfFNCKH9zfIlKp2NwMyY4211dlgj2wSmQtuNT5v6nIQcrHdg9NxWQdb+ZmBLbS6o7EujupmuIroaQfE\\\\\\\/Kj9dQUsp8JV1mEefn9xHwHI8HnCWCOVUt0yIAQhV52vNivCZOikTDa7ZLo9XB39i9245psR\\\\\\\/t7KE\\\\\\\/l+JZc9vobzYsXJVvSZFWNa3zNV1tPctcVJkdppc5GSZBPlhBbSsRHaIn7IUYq0zwRTEStXKhgbPiquQAqlliyxshRi+f3UkVbreHVRboc3XMttsROUx8bg9yf7gvBNbustdJjf2SgP\\\\\\\/h6X4F3LwbUjv\\\\\\\/D7uIxLyfsjgLP0S2+\\\\\\\/+efmLnvPpXwY2mcfzoQsiEhi1DtFCGKj6+TIyUBu+dBqA7QJcTcS34UodsIln8iczsym78yXeKCi6KtykLVuwZkN40AAIYUjDRQS\\\\\\\/jebTcQ6ZJDK5kL0BqsEtGLV9UO3BIGBEiLkFDbCxOymbpAYgelJ8Eu0xejuxeqSM7ga7U3UL81LufoJWccXOzZDxN6xJz6Ps1QEGkHORxqCWuq6IR7herhh0e0ma53MJ1SXxLeUYzleORWBZ7JjbV44NXDMVLBYexId6Mt2VaIYgEpY0cp\\\\\\\/MgHptPdssQnLztrhtmhsmAUqT03j6d3E5L4enDypl8KnqfOgV8wtwWXDp\\\\\\\/1o8Y7ddxd7QxIEaYXa52TEfKBJQUKoVCHXeOTKj9nP\\\\\\\/9yoiXLZpX5XyGi1Q9ZqchGxeyjbRDPoX9xfQXn79LP8QFxnZWE6gwU+RgtmH2ha61wxM+jkLIqRp8g66Q2DzmMSuho8XXt+MwZuNdBOACIkCXaeboxZmaHS93dUMHGhy8MJzpbilDitm42Mewrw5MjRukTCWB5Ts8S7E1nlPvj+C2YMVweTqfQ\\\\\\\/Cv3mWSEoEyYVlsobDCVCO\\\\\\\/WWdf+p3O+XfXtWlkEOjfs4AOjMk2mQugdsvksYLe+P2nHkMVbnsp0MmI1WNHnSYXMHZuE7TsaLL0334w\\\\\\\/9R6ZQw6T9bJA9\\\\\\\/UIBaFUekVDqPFPp5s7V3XVeo+OTKDsLh66LGOCrCnSKNf27gthdodWiShPr+LJcqPYJ\\\\\\\/VCEbtX0rDSW2fGn0iA3baDG4wOVF7aTAL5N\\\\\\\/kbvoi78CSj+nZPbfEiPBL2UBHK31ZNgJltvzrKz2GQVOqxUmOREl0vPzNz7\\\\\\\/+SWP\\\\\\\/KGhNy5BNucOeAzb+sXlg0iH3q\\\\\\\/2cd29nmuEg2XmmvNRfyv+Q2vzaQ1ACSMc15\\\\\\\/gJ40AvKOobNVsWIcp+02X+2Da+\\\\\\\/BXo\\\\\\\/lWqe6gk3P3Ce8TwkLBh56o7EV1bMaj6mQuyFJHmt\\\\\\\/cY0kkOi1UoN1q7KDnOVXkjAdSwXTlfnvUCNAAlcqyx3Xkmc2VyvcYFXehBfFW3DmxaCjK9k5NDQgjYM\\\\\\\/Dv\\\\\\\/a8xWtDAO9Loe5sYa+P0j8aqQgcSx7Mf3yMqU1aI1OoK14AH7Bc0lsh7yfwePE8+tjT4DPogXKkpSKOj6p8U9CIv\\\\\\\/YhsrfVihDLl5K2ORhDhnoxybSGTISaQGDDrrB+CyC31b8L7B0kuGankGI3Vgd6+3AoA9u1y7kA8167pcqCmz4l1BnJowZ0nq69fs5Oyuz2N54YD5V0NXm1i5QRC5ipWo3Z5IfpyZhsZGRE6GG6d0iaDGpJN2pdSosfq2+LypUfmYcLacmQrq2pVVU3pbcHI0kDwIeSvFGhMJchhOOeetEiHvwTlmbiIQqsDc2TH8q4La8vvaglXNhw07S7FzVPf68l6Tngt707moJASLiyMV\\\\\\\/xdqbQjYBXFOtvJo7wbLKscT0\\\\\\\/kDsM1XJDkaOXgv6DQ4bLFkC9y\\\\\\\/KbJISG2jsXDAFvWMS3cVyxUxLbxqV4357bJraE6ZP9qb3rPHznis8kKtiF7w+JDUyTwHetCi3JrYkmL4chTqEq3olQ3f15dc\\\\\\\/GKY+lntJxOjHamY4AXnuS\\\\\\\/YIX6LCIinrQrpirhTs\\\\\\\/XexcxrlHP32RHS5\\\\\\\/EXS1whdF0+AAE4qrKZj+LK5Dby4uD4FOaR8Va6qwZHubeBJfVBY5y5j93yl5+Xt5PE8J5I5INXo2aFKmCmm1\\\\\\\/hvGWsUOIJoM9tY7ydWMDt9i38dJ9vGdWgxXgcfNhxjPIstg9srM\\\\\\\/OSHxRfl+YYdZphC8Jd+A8aDWN9WA7XYk0HJCZSVBMC88\\\\\\\/IcrkD\\\\\\\/k2Z97U+92yTzkj+58pnkP+MY8H61rCG+C+\\\\\\\/q5IeCSZ0tyswYmwXc\\\\\\\/d6Heu8hQOoAehgntA41T\\\\\\\/z9vIX\\\\\\\/N6cKJjyax1BEUcnzTGqRsCPDj0xwpQcrbPv45TZ6tLAo2BeuthBfq7NGS42FpYsL2jPItHF4+vbao2THdwCVVg+DsZQ\\\\\\\/VN8EiQhxyNZTpGcBBRaEE6giVVw\\\\\\\/\\\\\\\/A+9RuP3ruHZrfjDsxaIldyCKJOz982sfeBJ8fIogpVuAdWQqlc+fK7Rs4YZBm+6CszIAkkinoKC4txwbqyRkIw27EfzYDSrI1fhIK8qn2pN+dYyBdzUlXrPn9993kF8MVkgRAehm4kP8WzE5eTz\\\\\\\/sxL8tC9yCPXN7Eh1uK3fxfJgVQ9SA2JjNxFGCnuiGaxcH2qAEFTyzPtMmS7pANlEwYzfQxbZ\\\\\\\/8609dtxI2KjNAALiIkYoJ4jP\\\\\\\/hknZNy3Pb1LSs2kKP\\\\\\\/nkL0mUp3l5m3GKxoU1NJ5LXipfKu73ynnmwvXsDNrBUNt1hu8i0sn2DUa\\\\\\\/VozShsr20POXz0sKXZeWMPQFp3YEjVog39QtlB5Fg2MtQn+HbNUjoEyQZC4hxPnSwZvSewDCdIrahL\\\\\\\/3c7NjY9g+9eBX9XsID2yA4HWyfGXtmv6wWh9q+TQ5MwGT4VW3WRK7tCTjikQnTOTUayjLWNGzA78cvHwyr4G\\\\\\\/+XMpttssePt+0Vx+6TDo4DvbxnuNvsimCxkeZWH808hva7227uOd8Nra9R5jn5g2TIT5X3udNwhsGUVHdsLxx47Un+uo2ZD2XspbeX43ndxX6c6hEquE\\\\\\\/01t5vwGtqFlJwl5f69tY2tK8XRgLRgdMk7+FJSAvMJ9M7Rqmi4GK1RNIZDz+o3xZpImOH+oIBLSoqpMzfUn8dGvRmaLcWD\\\\\\\/pivpVaH3+djwQqXhC3f0rIfmh0XOP3lriQKHvrr9DhoZ+SNlxP3651J477CR9w9VLNE2Vw4dFVpMMh2LT8nVdkDRQJjYAgtAiv1WEiUcmkmpq4McKVgl4+gYyFr7dadclJHgNjT9BXG5Y3\\\\\\\/Q18S37YCVlceOEq2n1lFzqPqmmqx6wTaUfgDsJyHI\\\\\\\/BnTA7noK8CGKxHE\\\\\\\/8dxqJ43vknfz9IUeHy5frw8N72wzLtGZQbUU3t+1rYFMlP0fcSB+JE0k5qXP67bQ8JRGk73m5cy96usF2\\\\\\\/JE4dJA2ntep4PXNNobGLI02ux1wzjoMd1140Lp1k5eIexKHwW2t5dClZO+0xkbzgEzjI4xyoSqtiHUV8NXvhYdYhwwA0mamrcBkLes32YmXBHVPOrNvikdAONGGcuquX8u8T\\\\\\\/pYTnmPJeAkActo1Ek2iEgCQrCuai4\\\\\\\/K3tVzvwzWOKqCFchBY\\\\\\\/Jn9uPs3dHKcBMrX8NWCIN0437MDOndx\\\\\\\/rA\\\\\\\/5M8RkBc0q38f\\\\\\\/Q+sd3nNAVe869elGrFbkujuwbreBugyzo9uyhHrnZDjBmjDpS66rIJT65Spi7xWNExWm60CtMKbPqnTTL8bSE8ik\\\\\\\/wlhTX8vgcI0W6j4Sv09ZLCXIL7HyBZDy1TiYxLdKa7dFLkXBwhuGxZWAJXBEi2YQ4Asd8d7d1eIepqsuw1Skr8jZGuJH8yd2QQ8fOG+83bQ1giPjJd+d7Na5nSELy6g6JZVCxufbbyinmbuwlXJc\\\\\\\/V8WnPxxNOGlTOvTzZ+6TtO1P7Pik8Jz6QKn+rNnoO71aTpXHionWFNXJoy8IHF7uqpTwUYlk+\\\\\\\/3SOpvDD7ccEGEih2o\\\\\\\/DtZhUzUfKdFNhSoqZS8WnkZxK8e41sVs0jm17GMeVn05czBi639T4dpjyjNqPsX4bjg1EDywKeG+DOW0i2sL0vp6A1vide5Vg8gyPjcMcRpPgT8zVIk1wvS\\\\\\\/vX\\\\\\\/9qmZRhVpsLo4krZOAK0tMrcocTZ86mKXBnNgmB4DNI525Rf4tIrCdO21Y+RqCjQCydz4uTjwcFy7GS00FdOq3mN7bFPuNvSypqjMhcoaZbkrRyMEdCRMnibxNmqa0jMa3ssqoCCWlrZA3pnb33LxU9I3U+28oayFRDw\\\\\\\/zEnS1UIPcCLTvdTQ59IaLS\\\\\\\/Z5r1beA4rNBFxTlH+tA5AzlTIAsIcbwRvUsk+pBCtsSDXqFvW02SaQJyEsnTSSNwHxZf2xuNRWv0Hdo4hhSuCMsGMtY4aAxJvTpSzWXn0LcN8nWMYOdUm9pYsc2l4fcoKCtbK35sTjyP47U+b4wOTEphp59yzQdT9UqztFIhE97VYAUn32h4OJJI5xAzx+jKO6tjj93ueyW15E95sleE07walNqlW7kq9TDMg4rnGOtQtLzxkifY42oDfrNYjGnV5gYFk3qdxcB4rQ1Z40aiOdN9u7xi6OSVziY4jgr3Judktlc1Tlb5ewFB7rj0aQCaHWLtFiPOIFMxhp73S0C36juyEJZYWtsjW406W7Nd4O+fDiSYTuPSWdUz0mi\\\\\\\/TMwMpYmabZoqxcYczLLH3alCyvgkTMnxZfe5tDu7CHgpJX9Q5FkSOx1WP3ald6o\\\\\\\/ka64LL9w+SdSjEzr1vtLI7Hvy4ENmRuCcKOdx\\\\\\\/uM\\\\\\\/3T8LyVo7BzIXRf3fLeGBxgCfkznJHbF0GqjDHGvs64X5gR05da\\\\\\\/A\\\\\\\/3cyG2PEE2K+GrH+dfH7PfQ3nBObvVIe0cPPBfpJzoK3s5ndtNzeoSPFqt4eSiPg2AfcZmY8cFHgJERmapFYYkXtfUSxhg3eVGR60XBLn15DGw2juj5juSLtD3WvENf+hI9x2Gint3ZgRokiCt\\\\\\\/MW+p0U4Y4HW5ndf9W4h0dlfgu01764kc3R995TZXaA5Vwyit8FENLWvLg6vT4jkm84I5VSKvRCiUqoQIY4d+cf5eR9HLufpyJjfTh9+H18oqgLnHT1aTZigr3mIaRp8pCjXcCZYehWbTgXbXLrXmEMPZgvXDEzAuJe7s29uv61s+aPrf1O9C4459xxTk6NAjOjiAF3LxQxt1DuJ2gRc13s8Ac5qEeGTftcL1gbuovnmrxaMRn+XBj0X1R0v4X7mKx1tDAOEghT3TeD+6Wm5VLrjZ6J4QXVyKc0H8P0VL0iVv7VqHaXF2sASGuUxJeF5ZnFqCUUxe8xC2nXb0aFfnFJ4x3vAbCJ387Warq+0WopcBozxacjdQZ8VI1b4Zik27Ra+JkcVswTkxKwfnCvQMNiYE293+F0slnvpg85+rvuGZUfG\\\\\\\/yKHcu1pYx6Pf1c9IicFTsrcmirp2dg1g90w3Mklyx31pyCyC8xnxHqXXZbg1nZi3nOZWcqlGu\\\\\\\/UmHnQRaDmoPR6QP4koyNeJ\\\\\\\/PE4RFa7fNUMLSpjvEx919\\\\\\\/m2\\\\\\\/MN4AOTSnb4idvIDR5kjCEEn1TrLLc2SJLD4IUDe8abE51FkPPhbZBJKzmamUecYwbSY0E+9Vd76dHwoOUPP3AhKizZ2lHgT4NhK007kJwqGWGXsz9q47Qmby+Gs43QM2oAUKC+AXf2LybwzCdXPqdTde2xI1AZaZF986M5VIKkT5d\\\\\\\/KRf4x3EG904tZC+gN5S2nU9GEq5H3qppKlwJ83TzQtUtDFOBYdJ0nog80qCxlAMq00QwpzKW9WqAfTHYMm4nlb1ByLM6biwWEr6HXo3fAcJZtigiN9tNwbIvHyO3OZ6tk01pqt5jYPhEJAU8jr\\\\\\\/XIhDLaBzJ0RJrd1BE7Ij0Jkh8Ug3VqDxsvdnRvBdb\\\\\\\/qq6snjbdH\\\\\\\/QzOA9oZWrTuYJb4sMGvT5AVE04ZEbxUEZNd+8DSt7AKqkrL1pimBvFqQqopVSjDTtmky3fZ4fznEYr\\\\\\\/NXoMJrEBWOltiU\\\\\\\/e\\\\\\\/iSweqLmPOkaegrNkxQmfm9itYjTksNBrC0MhZh7kl8qmE9fxN058ZcahLw9KY\\\\\\\/S4IU\\\\\\\/oXvSoEp9+Y910rYb+Pn5h6Abkv1JyFMQhcemYad44l0C7Gwm+qjOEWtxrIIFl8p\\\\\\\/YBpD\\\\\\\/ftL4Edz\\\\\\\/MHMd0GdAFB8NaLSGjGNQViAlI8tUK\\\\\\\/2b14oxT+3Gk4v8TfcopCTAt1FNlSuXN6\\\\\\\/YhPGMPTekYvXKzmLep+mF4b7YDqJtH24ke+XZ5FZTd1BeQ8rKFsYHMaFXKnI8bUjmeKX1hR\\\\\\\/fjFKqhrGZtlHSR+ctDHo+Thrti3HtgoRPKeL7fY3byLNYDDDq+TrKgWCpYMJwA+L5VwDruJ64VeDdz8+hci1BPnryfj4PFK142CuFbzjrYMRhn4x1\\\\\\\/2V6shL25XckMskRxj5Tk2FUVeN91RZtUCb9QecHPyP4VDf9p5Wljb1tUu5WpxZpCLKvfwOwFpg4mjyxvBYtdsRjsji0ftm0KpN6MFJRI4c8JQIYW6BMj2wVvRlkowv3k28e5FP5ZdzXOEoJQWvQG9jSrRJbr2WJHAoWSTuapsNtbGhnOuByCeFpHlFUccxE0E1CT1zarCit+3vnYlE\\\\\\\/lYj35NkhAS\\\\\\\/fq0Bu2zE0ZmKhlOxaAsbBtMJpoDQU8kyq0jYPEirVa7xaogoyW9\\\\\\\/E0rw1RDnEAWBXsYJoEqSFx9WJXNJrcQP6OYfvrtwB7Gse4pweXni6x3nGoolKNsvJcGgdtoipNjiribocjcBsjPYwaLKF8Ck6lUtjoaKM+iNIA\\\\\\\/SFNc\\\\\\\/J+vOFZo9forZcve55L2zv7Y5pFkkyDFTSTgUwKOAlr7k25cCZ28O9EgMh7pqdypQNPIoHiDdJ+4xVp9awhF3YdDe0lMhnHqhOq5zrgIWLJSr7JWWv8xu3dBUxy4axOeMwAVjg3gljRlC37lHbBN7vZO8fxm9SEwlWC9\\\\\\\/ZdmqKrLokGG2FpQeKzvyiw0HRglMBU+aHXAf6TRYUOAQKECjR\\\\\\\/D+hWcXdK79JT+kaGW80ZTbBN1enKTCIGtD5+g6dpe6gGj3xJ+Y\\\\\\\/iM\\\\\\\/jsY6bOnpa5QlDKXmhFGsXBZzMBA12bi5XM8M6NGXt2yT8xkgy0lOSKyAZkaBiIKOSFDNrcGljTQgIYyhByjHwpELTosANlxHABRvqOdt+LapbB4eJ+KJKlpe8xlDWlnpA8nB29dCIGu9RLCgTcW4iAs5h3cffsU3nPsVawqUUpsn3SmdUbN\\\\\\\/zAmVJc+3sQ\\\\\\\/dJLt7BWG0wq6RxjCcTEafyhd3L\\\\\\\/zGudMNzDRLZ41QlkX5P\\\\\\\/czE0lG\\\\\\\/IVxZlIJ7E4t7IUlcYKUwrMtx7D+BhXjQWqMfkMiT\\\\\\\/wCA5N\\\\\\\/CRV3gCeVFY5TqJy8jWGXwoUCEykXZ\\\\\\\/4uREySfUcKhGVxBJd8+tXWDqdMCmSQLTUdUzZhtJo5AFjYBUErTahLOxw9rmEHVw5dwJcKCE168igrKGrMCSvt5GCzzooTSQGDXLMBkOldw0el+4UuWFxe+N7PGhzkOVt5T\\\\\\\/oHyvNouDLzkBgxZYYWTMsnnPDzciqsQRumLKXo1MeyZs5nhuHOMs4O757Xm8suHWt627852S7BVoDlHrXooIryTkpNmKgV\\\\\\\/v8TsGXlQxLKptxVlGgNIOv7bnuKccIMezbaHC1LYZXRxclrs8QL+SIE5eYm5ChxiB3+f+R3XhqW4EMEBZe4m+sJusuLscDvtaRkZubV8EvR1G6etU3E57k4hYQMHgk\\\\\\\/KbTLciV+AWHqOT17vJo0QGV9nSh2AELaaj9OKFqsk4CPNjlB6TLVu9\\\\\\\/0DLQSI9ucfpwEohX464rrlWD8DK3YdCj0xK13Hy7x+93AytruBSM14b+zC2RtU3RE6b2mKYLuu89sHpqzZGLAvq4elwBQh8S\\\\\\\/zF3fcK9EZtOd54w\\\\\\\/Kaj5RHBqRSevHpNUO4+BqHb1E+hguAvryQt6MamJkL\\\\\\\/ePAiwt0PaXtcpAxWJ+hjT529APRNbPntT0ZRZXKzKP5dzuzjme9eQa5K8ofn6\\\\\\\/Fk2veLsePXbvAL1KRBOVugbVjna8zBnORzZ2f2Sn2LubnuaTNtCX5plyEfVac79NKl09ZwfDs0br13iBeBPqzQTTU+sQusxsPYoMiW2pi\\\\\\\/5W3PapHLmrCYnSwJuvK1MVg6TcTUSwrKHItC5tHfwXxS1SS5zGOwOMsFimau+JZW494llXFfc4QtiHeu3OxFJek7OipEqwBHJywgAVzPJYkMf2VFx0MJTAzX8OJ99eP0MM6s2RIFIniN9HdX6YbOtBr36kHzFYHnDqkeaoAQeQ2BoQewOcjCTb6fnMVNMBA4DdRr5U0YWfHKwdH7pKjJRm4hiNNvQXtYU9W8rQaDo1Jc78\\\\\\\/efFC2RpufAljqpIdo3luBXgCUCs9p5j6v0fEA1Get8+tsePFHadTd68ZdVdVXfXqXG9iBNdIMfhbgwQ\\\\\\\/pNwsBcLAKqygE7\\\\\\\/MJKGQt9G94z4yBIVmN0cGftbxVuVFXzvNT9d9CYLNYqbaLDnNqftYw84av5DlrRZEBCXK8CLIz7Z1uFUNscBAOAG9vjoDEjrk3bkr5vYZ2oCV+Opn0n9KM5hflQ2cxg2ZSE\\\\\\\/rcJsfSgU11JeUWrE\\\\\\\/Oj1H6cvVEjFtaXJtNaL9Os+rhmPtfsKnOFxf1q8viABZSbd2UIIFo4fNoTD0Gue4c0lAsQFGRFyT4T1Oan4ELu\\\\\\\/4FGt8u5utRgvwY+sHqLuxGVQISCC3\\\\\\\/jaqqRZN1\\\\\\\/+LZVQwUwhGroJiPMFlTBrBEe1QChMzu7PcYgm8GKIwVlKvrgnhdiA58EigHwPlhCD+HE8gu2pHccYbjLO+Cx1OrUah4dqXU9MpHkktS3zOREjnqOZiiluHp\\\\\\\/pZa96tqqTt8KB9ZwI3Nd+N7h7Iv8dfz+j9gIvHu+KrE56gCZFtPWagOWoTLDFQKk+whYhca546XqVvWIUpcVzDoOitjsiQ7oCOQQdvNQRc2MfR\\\\\\\/MRj1hG5MkxFEyC2KHqnl+AXP+v8ZloP3O1FzaeyQbyz\\\\\\\/Yj\\\\\\\/5wTGWyvAFwycNh1KJXwDCVdTshA7jjjje9Fj68Hgw3o9D0kMzdtb7ue76HsmXC9QVlYkF8x668ZxfqnZ1A8ADnYRK6fWe8bX03EBFKXwtrsSNt5AMPQmOkrKEOKEL3NYdtgmHfZB7OqED6Tdqma3JXN71qRhV3CI+hf+wsIXeUWKtO4qVhsjA7vFZ\\\\\\\/Clmmopkqm9Qgd9Xa2U6wdBUPAj6cnuaMhZzXgRKfuP3fmnV6qrOsBmEjqO2cXSRr9rZ+ar05L8Nu0jcCUZNBnRkVhKodTY05FyGkDB8\\\\\\\/xuwZbaCULeL\\\\\\\/Pxvh8nO5JmU4N+XGObdnF5oosccFXUyJpkoaJORg2JUjt\\\\\\\/DNk1W7jKuhJ4EZh6G5gr3kZDWl+KUmxEMvtROI35JEtmRkoEOvCcu3qV4jmX9L7bGtXoOXC3dwx4B6asBON1XjLCH1NAgkKjmKIQS5cJwBabiTujJQiSKzM623OVOXTCt0+fbb+OO0tS5FLWwwP0qe12sR+JGpoQtKYhA9I4YJlCvUjT9GNpRugjrio8qVQeDj6W5cwnI0MnTw9iWYpgDAwzTF\\\\\\\/5Y36rYTBXEwZv5otqDqJObiKAWA3PRgt8OJfl7JjhY9o9YuqsF9nfQQgR7coWzdgGJ5Vx2wW8FfF52czPMv1cxdV95mSG1FHu4pnge\\\\\\\/21E\\\\\\\/RR3Rg26+g4diAPMdX6pBkhf44TfL7lebokQ2upkneJzbRLD1dq45\\\\\\\/DBKFiPkC8dhGRWmhi40vi83Fj4NYunRgJUEJsQqa7WNE9jnWM7n6D+Q0AjRoyFMi0fwDCA9sqk5W447peLaeH1qJRVLHM9yXitDxuFpD\\\\\\\/ENopMampctiGwwWkz1FUvhQNLGWfjb32dO49hrvi\\\\\\\/DcyAbGfp184gp1MnD+OwfCFO\\\\\\\/BjaAWhV1dXpr0NOx12LHr2q1Hx5wzB\\\\\\\/Yvui+yuwGAVy+np5pxawj2Qikal+7yQBL2uJurJDr38dD+zs\\\\\\\/vGA3kS+F8jXuG+wO7VDjWj+LOtV9fg645GhYAnWlyGSX7I0tT\\\\\\\/hHMVYKigqy4BLqFHlzu5qRUnTyO2H1PGaCxEvqyA4Io7o17NRY9wZxoz2VO5GF3OR9eIrV4+lVIZczCk40fLpRQ2HXiOrYjBdKONCj4yFzk3wdfBrLsuIayRq8YE8nh+cXRnEWFvZIxamA\\\\\\\/N+xScqMANcxbDAZajkKxeWe91rYfaG7wnZdf5t9rI+5WvYJsMqXla7\\\\\\\/XSuhIh7qe\\\\\\\/N3A3b1IghwPMg0DOkqYyqmH5tWvkcPoJepD6+kSqgp353C1EOBHVDvcEeGJwI8\\\\\\\/cjt7rXjIbAWzC7yPpcg0sizosnrdwWFVJEHA1\\\\\\\/i+K984rsiSBGnY4B4LYdYwAPu0GYDfkO3er7oDAcgnU0jPV7kuudonfFv8pEzNUGNGxcqdUc60DykzF7HRPooF4vAftJh37rj0xQjTeYTnHPj7F9+HsCIoDvJ5z\\\\\\\/4\\\\\\\/imLmsT950Swx02e0O4q665Ltj1ivLhVxHcOOxzkUHOW7hYkN4wsMR9uEY3E9WvgN16AuwWf3LlpcLmxGYurgQRLeKfkuO1lqDMCmdDQw6s\\\\\\\/janzgTMKL1cv+AGUxi3q1XpF4QF+ORKPDToaTrc+Bn8rMCjx4Mwho1Mv9w4JEi6Gy04umckLyujMQ3kczR0ALuQwtAAK2SIuvQWe\\\\\\\/IJAyNo34KRVrtz4\\\\\\\/c3zYqwiMz4xIQeduL5Ot6Sz9e5awH6HjXqEOx7WjvM+i\\\\\\\/SSf8cZCXeP76\\\\\\\/4glGgnWlCO64RNfXlOjcnBl2TBHurpZX\\\\\\\/TL8ess7TRrbvSi7ObsmXZL8U+xz\\\\\\\/p+diZjwr\\\\\\\/rzad6UYqwVSIDwDdzfAktmBYLtCiNJTSWfhoa9huYksSiJhNOoMqp\\\\\\\/kiiovrLT8aM5xn6ATHdYQiiuyC+3XeniV0RbaVXy1AiPGividtnz6Sf0QEmS2y79Umm0W9nqDoA1bzS95CytrNZtDbjA6a\\\\\\\/WhsMbj6ZWev44NbbT6XHXBNaZnqwcpjbQCA8q82YivMuKF83PRjHRi5XWsl88kRgJB+v8XlQg0AVbG60BJjrnGbYLjVNrRDSvSqN+03oSEip69I5as5XJV2hdtHyV2s6cZU6BhprKsS8WU4aclQkDJaS4HbLpibonzOgZ9y5+jgjtCfo45tzD\\\\\\\/RRX0knf21HCoJGkYfdlvR2FFBZYeUj0Z6YYhynyEM7JoLuKylS8hGChf22df0sEg9CKvzLtadAkZHdrgO27zZlzC9f+1OKdUHTRfe2sYtGJj0iXQONf5a1f8flK\\\\\\\/NcCBs2BuBjpffImdBOnE\\\\\\\/mWJt\\\\\\\/e2RJ3Yp2MHVzJktooktJyWIkqegELovVsTGjnG9tOiQG9wPE70Yfsgn6p7ekq+GoCScFhEgBcyGH72AFy6mwKgz3VixOSv0rLe6JUvEQ0WxRI6Zm91rXsVSgx8URCIoGsREzessggNwRS\\\\\\\/\\\\\\\/HUIZpuN02xVv2seMl2AQwPN\\\\\\\/Di\\\\\\\/UScLfWibVIbytHKSGWjRAsrL4CuKKSipovUJOSG\\\\\\\/mb2CH1X0McgHTS8GxpiFgYS7l\\\\\\\/eSCEU13N3Bh5oXqbhalEpaIiBFrhVwrywdvDcqpYPkD3o1nB5pTtQ\\\\\\\/LBGt\\\\\\\/HimqmSPwTwmHdaQRP+m26JmR2SGFe6Ch2i7miXUmqko0hPRja0yWAwskxU5I9wL9KAqt4NT4trvvKlgcLfV7oUBqgf4RPJNwvlwFt32+vyef4XaTWNMmXqblUrs94G97awWlRvo6LuD6u+g3Z82PxouZdS9Keh7gbF3xN1Wg6AfEsDncBkWLG+LK\\\\\\\/5VdqD87z3uSJ2mg9Kp33AB0QHy23X2JZYsea4h+RwL6tu9m5Bn6nzKjCY0bB3O3xHfiKBw1DQ5+9O9wYvDeFw0zYkQcpIe5yL1n4622YPmZ+5OpIrAUa842WSILjGl01OX5Ino00VLACSTUNjBxqCMbUjFNNCVboC5FhdjcTQngySJI9HFQfO0xhcJznkeq6ysdoKAQ9Y9ac+gXP3M2zmDGO2Bf4mXQTJrz\\\\\\\/uTrXGiXJN8Z\\\\\\\/HttWrWupwJrCtjvxuU1YtxS3EJZtMF1EzvaYFTKM0uhhEjDbP7XbEGOrs+oLqpu6s3Xdqetc\\\\\\\/qPuK85RxxqI+GKt4Q6x1vIlENB5TsEK0VmazvqsJBFaaN9fSUSsjNj6wYHIU8EJ6nKT4ZgGLL2v1ukLDS\\\\\\\/zhwJe+IZEcb1tPW8nhzSCFeSAHRs6zZaV7Mx+dNuZ69wLt7fu48QIOjhAPCg61GCwJ1WCNBzC3foO0xwy8ovJqwkng7JqH\\\\\\\/ujTjiBqxYLtrg2YDlUAjlcAn2Tja+qcjJPpZYrCeuKu7LRiyPmQ4k8g8bH4ARZ5fMx0V9xfhHhSJOSTCSNu0x+L7tHO+QzxhDIy9RiSvvtuNAmv\\\\\\\/zinRO9FbvW54VYABzSZicH4gDEA+dsj4Hr6Futds+\\\\\\\/I4xNRpix7ggsm+LX82rdKplGXbmGbGIHxjVzT7hPFpJn9W5m7Mx2SCp3ellG2mmfgy6uAhx\\\\\\\/HxqyKfIg\\\\\\\/Hh\\\\\\\/NxXqPIxAuTYJJ0YwtLrH+kEVRFB+Pdddw+Fb5tzjdtSEDSLs3r+wCiAluFE0NkFXoARrUSAYwCRMvEKXr0p5\\\\\\\/uToxRJeA0AWg4982rROtpBesmnQocbhbBkYOUUaQtqo0bWvUOpWL5i8ZgfBVi2tzCfdUmzPBR9iYhQRKcEi5pokQ605AETWEg\\\\\\\/SpDGcaT\\\\\\\/aprdjfE0C7+9JuRoNReEgx2oCCqLDjBvs1tIJla4KLEo8Of1tWUgfyb3z8kz5H4MCKotir\\\\\\\/x\\\\\\\/ucYAs9oJIqV3QUDMz6m2mrmkMPWTyA96+qo9r6EZlf+521UF0AA4nx94sf\\\\\\\/l6j2UVecqi1bMTwt52pPzj8AHFdKtsArt02DGaGFKTuyr8G8ZSgKT6fFtKUZJdO0iHNP85\\\\\\\/gBV0pnFd66R+sqRPz091Lbz64hVdIf9FArDRsg5gggR161xQ+UJ2zpCsmiS0LGKvNoyiors1UK0NQSNdeU7u5q+J87xM8zL61CM4V681hG6LfAktE1AKkIFu75AWd+RXW9a1krqonRctDfnI4vnNV2hqUClhl8D5LLvbo3I01wTLffRGJmRTPuRwNYJGviYw18ujFrGzFOSWjQ9nVr\\\\\\\/O2bp+q+CoPEzPD3v8J9rKG2qFJrINncssteUXumpAQtHgL14e3CcCh8vhLphWmy8BpscbNvtz6Ib+Witx019TkFbFxDQsEhNlumxNrTzTMs1Ww7iZB\\\\\\\/J5MFYXouYJLL\\\\\\\/MS08GlMFHN2UK6swfP9Wyy82vfcdnQajL9CgwxoVmpi8dnybBtGfhKw3\\\\\\\/1QxWcYjhcBzU+JBQqpTfp0CRbrdAhPIU6ke3YkKAI0hYII3JS7kBJXs8P1QZbryMdPr5AdZ\\\\\\\/mbXqrY+EmKKByEU9F5YcHWcFNhakw+YVaTmp3X0Z10PN42DWM3F7Uj+Ah3EpnusUvb0JRIeAjAO2lYEN5rMq9CrlQvz2yQoFjRZzVFReBu4rA8YDktxq72tsaeeWN6ipWPKnZ2ElVOcUJBMLO2W7Nk64lpFpMM3kaX9WkizBpdS9+afipSYl2\\\\\\\/cAJ7XX1D5Aeky3EYz\\\\\\\/3pdF8WTQ8sAczOyqqdKwpwAoLbJpwoaT6aVgaxQ6RkfoM90CgAdi5vUeibFQkrpPnPIxWy5QhzfUyTRla\\\\\\\/MGRC1AM0inzzOfGB9X7TjPIB3lXcIndMUKgm\\\\\\\/MUyWVXv1vMZv8LjA3m3fuB93dqYLr3NLDh\\\\\\\/nfyM3TUZdMcN2dD7L1Jdvz5VvRKyBLhJD6rxhfiMWlNWhl5m2UST6\\\\\\\/GI85SgYeyjEWrntd8vJ\\\\\\\/KSJ8iAU06rGx01wrXDoQv98+uqTgKw9o8z4Cj0YxOdMln2X\\\\\\\/9eJjvTG4QXLQ\\\\\\\/lveFKe44cpHY88S1YBDNtC8TH3rV\\\\\\\/cs7OtaKLYtbJ+OqvjV45tg6ustu1v+IqwqDcRmvSK5YZYNG3oP6aCrKmRLaR4HdaCqzc4TC73WIb1BCPrH6WrPUoACsx2XDMeMxDJHNiv6CKMZBlReOXEzD6edCKjM0nwYLktUzJoAukLBRgLEsqtQBNGimUecPiNqMcwjRGN\\\\\\\/VslfShL+P+wvt6m9SMQnjGMIBNlB3TgmAxWbtNZYYvRoSSRrtAFYC2In0M\\\\\\\/zukRjspnFIcYWktSs2zDWnxlntMf5iKuE1T9zd9f80SYRUyHFHge+wBP5ePJSQu5OP\\\\\\\/s5K+7HPWpgV6R8d9tqo4DiKjeP\\\\\\\/rO\\\\\\\/coitOx2dhJbpQoK1cL8ZcYSNPl\\\u0022,\\\u0022iv\\\u0022:\\\u00225uUr4\\\\\\\/AFY8RYx7zhNl\\\\\\\/tdw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227984ca2c2c78e68e457ee43fa90a789dc1572b03b3ebafd6422afcf998253ebe\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223lth3VkrIeqKGp\\\\\\\/nLF6URUwS9AjOIGsKfViyaUaGd4Hp\\\\\\\/QKnxzNDqn1Q6bMOSHSN4G7Li72Ku1NxZI79MGcYaNOzP+c27wGgtozfiF2nEKwhYh8c+Np3Il11nF1bFlRxZFUaJxMJUCs9uvZT8oPktbpqbryaJULvvgTfLfdelhN8p5gA688jhiSzyZugEr\\\\\\\/JwLO2En4YoWZ0a6lurBD7fn7\\\\\\\/S\\\\\\\/xxHHafAgvdymXN4+yj5e0ak\\\\\\\/zrCLFosvZt635\\\\\\\/GRlZeNcfhgGTScOogLcAuSb8cBjsFGRAUPyaUxiYSQRMKLMY35N3WUJEnVadpPlOkg26Kwo+VR7Q6N76vKtbt6mjZ5h4lrBSbwJiluJ7\\\\\\\/bYkHkRh+zS4BdQ4r6UM5sWqjjf+zWazipGIR6FVc2IBr+fDQZhNfPHcktNDTGBdmZlpXh7ik0VxMKIOTCQDnRUooNn3UL9KDe8iLA+nz+06tUeMPhUpEqmXIZL+B\\\\\\\/rjo5l8\\\\\\\/urnhKtZ5n3NZdFhilcZwrWF1HdLVSepOrohnSIjfTTEEjIRtLFhjCI0eGegM4Trp78v6ijrr2CjTFthISex\\\\\\\/u0CPGmAtf++FR2PJBXGBQcxEAv\\\\\\\/g6ey0aCGgCw+RGM9+FZTkJUTThoG2uOorpk15zQDBO9M5WRt3kItO2Kk1mo5EaFZ+eonGsjrQ4E727jmGA+HtT0Xm0qMnbOGA4bGjMSQVL7ksxfn9cDHDedP6of4undCjDbuoxyzPfHPhNe1EIYpNeCWkiwZ7if+A24ewrIEDNy7LYhglfcyFvag4e6+KZ+ZT9uvD2lPESzu+IdNF\\\\\\\/PpTHpiWqOrsT3zjhUJULVZdmLzA0aHI9us1m6d4aFpMy1ZCuKTHlrR7xAV0YYOuITN7r+LNANNT+XlyJUTeb9MYbGafaNRq1PTRhrlMoSIllAgeQCVCj9QX2CXowvw9PxcZgyt\\\\\\\/H8\\\\\\\/Zb3FqcBveubBUfo9mKyf9D5AZfCs2vrSFcdB2fqCinCn+fK7KSHlJIy7gWbWeoXbXKoyKKrHExAb\\\\\\\/HW1yBTX\\\\\\\/DG2a7TMARxIwtJUSy8U6\\\\\\\/cjvx4uhOBINubLCG\\\\\\\/ylaS1Lun+8ZgtvW6q5VIVAftzQfH6Ya8oBaYTKM9BDma7arl99KejAVTIbm3hK5gR\\\\\\\/JTE5vozJaddjMGWkqFD6xiJ1erIW+FNICrNvatBay3sKzUbgtH4u4LhrDhWqsIKE2YNwjzNCRg3jaQVGmqIaSsLzWk1GQl6iEWXHfaNTjOqHO0HoNhTFWMK6CjQ94576kcYmFY93JdtiTGksKKnC5M8GJKZcmq25FD56q7pqBXUumlB7msrzYTYuncjpoujhnXdYTSdksE\\\\\\\/Sd9jXB\\\\\\\/\\\\\\\/93P35E8QQb06AopBOOA5o+SFU\\\\\\\/eIUpB6Qb3JvNojKFYBAsI7vPRMWCaEOiOLD30YrkHNi58nvXtrsiF2Ow1i6g0koMsC\\\\\\\/tOYmxSqaL1bEsLcw1SsGWx5FHqLCsq+iJJ7lACyDJHSN7SGfAHndbN3YJ5jC0DVYGPkSvj0tOZvmcqcaN7xIaVXzfYMfaOZBKBO8+FB6YFuAhpBn9Dlz3\\\\\\\/MdvAocubQ+47+M4alFh5ra5MY2KbL8FPBFygY3QUfNMkQqpzTptDTkkB9ekTO8jSJp6y55nu5ZkFDi\\\\\\\/\\\\\\\/urct2PSYxHs7wn474Ht3GQYsiVrsr527\\\\\\\/3O\\\\\\\/DtwNYn1UXUklyF6IRqsWwH4uKu\\\\\\\/uf0eWFmkNcruATdF++8hG4s9mXXtRNT9gEouPnA6\\\\\\\/UHEtuNVU9K0Dy\\\\\\\/Fp3eM95qKxvYr8s23C0HSEvU5MJuH31N0mHLWI3MVaOEdWYQBZ7XF0dMeXLxs6BLdHyzjGyi8c3x1OVwsYOiYr1Hg+riEU\\\\\\\/wcoOw5fH5wYtP6Lw5zUNFb4AuCG5cDqtZynvOqg5nj0IpRD7nJIFlAFhOJqJEz6ay\\\\\\\/6KBd\\\\\\\/ftV6ALskQg5+xWtQkzks\\\\\\\/WIpmQv8O9RPTsjczPYZaN0kTZxoOf4EJURETvhEEqu2LX8+0WF1AwWu7Da7+jxkfpM53NPn8sESd15U=\\\u0022,\\\u0022iv\\\u0022:\\\u0022GhbWxKDVm2fVTjezAcuijw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002277a3bb7ce85908a0903c45adb24b708e6e944ac6f41bc891c0edb77008be2e06\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-484769 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1833" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5LdmFKsQQskAquLGKHhx3SMmgtubDFmERonk32CrSws" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1833" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AMp5uBF0WKfODqiTttcpwDLGrUFmwniKcoRnE49HXOk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1819" title="Genome-Wide Transcription and Functional Analyses Reveal Heterogeneous Molecular Mechanisms Driving Pyrethroids Resistance in the Major Malaria Vector Anopheles funestus Across Africa" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1855" title="Genome-Enabled Prediction of Breeding Values for Feedlot Average Daily Weight Gain in Nelore Cattle" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484769" id="node484769" data-pisa="ggg;7/6/1833" data-pisa-master="ggg;g3.117.041202" data-apath="/ggg/7/6/1833.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Variational Bayes Genomic-Enabled Prediction Model with Genotype × Environment Interaction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">José Crossa</span>, <span class="highwire-citation-author" data-delta="3">José Cricelio Montesinos-López</span>, <span class="highwire-citation-author" data-delta="4">Francisco Javier Luna-Vázquez</span>, <span class="highwire-citation-author" data-delta="5">Josafhat Salinas-Ruiz</span>, <span class="highwire-citation-author" data-delta="6">José R. Herrera-Morales</span> and <span class="highwire-citation-author" data-delta="7">Raymundo Buenrostro-Mariscal</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1833-1853;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041202 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484769" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">José Cricelio Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), 36240 Guanajuato, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCricelio%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BCricelio%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Francisco Javier Luna-Vázquez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BJavier%2BLuna-V%C3%A1zquez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luna-V%C3%A1zquez%20FJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BJavier%2BLuna-V%25C3%25A1zquez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Josafhat Salinas-Ruiz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Colegio de Postgraduados, Campus Córdoba, Km. 348 Carretera Federal Córdoba-Veracruz, Amatlán de los Reyes, 94946, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josafhat%2BSalinas-Ruiz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salinas-Ruiz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJosafhat%2BSalinas-Ruiz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">José R. Herrera-Morales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BR.%2BHerrera-Morales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Herrera-Morales%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BR.%2BHerrera-Morales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Raymundo Buenrostro-Mariscal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raymundo%2BBuenrostro-Mariscal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buenrostro-Mariscal%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaymundo%2BBuenrostro-Mariscal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/6/1833" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:484769" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:484769/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1833.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:484769" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:484769/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/6/1833.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:484769" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:484769/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/6/1833.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:484769" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:484769/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">There are Bayesian and non-Bayesian genomic models that take into account G×E interactions. However, the computational cost of implementing Bayesian models is high, and becomes almost impossible when the number of genotypes, environments, and traits is very large, while, in non-Bayesian models, there are often important and unsolved convergence problems. The variational Bayes method is popular in machine learning, and, by approximating the probability distributions through optimization, it tends to be faster than Markov Chain Monte Carlo methods. For this reason, in this paper, we propose a new genomic variational Bayes version of the Bayesian genomic model with G×E using half-t priors on each standard deviation (SD) term to guarantee highly noninformative and posterior inferences that are not sensitive to the choice of hyper-parameters. We show the complete theoretical derivation of the full conditional and the variational posterior distributions, and their implementations. We used eight experimental genomic maize and wheat data sets to illustrate the new proposed variational Bayes approximation, and compared its predictions and implementation time with a standard Bayesian genomic model with G×E. Results indicated that prediction accuracies are slightly higher in the standard Bayesian model with G×E than in its variational counterpart, but, in terms of computation time, the variational Bayes genomic model with G×E is, in general, 10 times faster than the conventional Bayesian genomic model with G×E. For this reason, the proposed model may be a useful tool for researchers who need to predict and select genotypes in several environments.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/variational-bayes" class="hw-term hw-article-keyword hw-article-keyword-variational-bayes" rel="nofollow">variational Bayes</a></li><li class="kwd"><a href="/keyword/multi-environment" class="hw-term hw-article-keyword hw-article-keyword-multi-environment" rel="nofollow">multi-environment</a></li><li class="kwd"><a href="/keyword/genome-enabled-prediction" class="hw-term hw-article-keyword hw-article-keyword-genome-enabled-prediction" rel="nofollow">genome-enabled prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-3">Nowadays, genomic-assisted selection (GS) is a very attractive breeding tool for selecting candidate genotypes early, using only dense molecular markers without the need to phenotype the validation population for several complex traits. However, to take advantage of all the available information (markers, multiple-traits, multiple-environments, etc.), more efficient statistical methods are desirable for genome-enabled prediction models. Markov Chain Monte Carlo (MCMC) methods provide an effective means of dealing with complex models, and are widely used in genomic prediction models because they allow simulation of high-dimensional distributions to arbitrary levels of accuracy. However, MCMC methods are computationally expensive, especially in the presence of intractable multivariate integrals in posterior densities and likelihood (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ormerod and Wand 2012</a>).</p><p id="p-4">For the above reasons, the variational Bayes method (VBM), which is very popular in computer science and the machine learning community, has been adopted by the statistical science community, because, although variational approximations kill some accuracy of the MCMC methods, they offer vast improvements in terms of computational speed and time. The VBM facilitates approximate inference for the parameters in complex statistical models, and provides fast, deterministic alternatives to Monte Carlo methods (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ormerod and Wand 2010</a>). Most VBM applications are found in the machine learning community, and there is enough empirical evidence that the VBM tends to be faster than classic methods such as MCMC sampling. Also, this method is easier to scale to large data—it has been applied to problems such as large-scale document analysis, high-dimensional neuroscience, and computer vision, as well as in computational biology, robotics, speech recognition, marketing, optimal control, reinforcement learning, statistical network analysis, astrophysics, and the social sciences (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Blei <em>et al.</em> 2016</a>). However, the VBM has been studied less rigorously than MCMC, and its statistical properties are less understood (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Blei <em>et al.</em> 2016</a>).</p><p id="p-5">In GS and prediction, we have found some applications for the VBM, for: (1) clustering analysis of gene-expression data (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Teschendorff <em>et al.</em> 2005</a>); (2) conducting genome-wide association studies (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Logsdon <em>et al.</em> 2010</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Carbonetto and Stephens 2012</a>); (3) predicting genomic breeding values in univariate and multiple traits (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hayashi and Iwata 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Yamamoto <em>et al.</em> 2017</a>); (4) estimating quantitative trait locus effects (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Li and Sillanpää 2012</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">2013</a>), and (5) estimating variance components in the context of animal science (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arakawa <em>et al.</em> 2016</a>).</p><p id="p-6">Most VBMs applied in genomic-enabled prediction use a simple linear model of the form <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-1.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-2.gif"/></noscript></span></span> is the phenotype of the <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-3.gif"/></noscript></span></span> individual, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-4.gif"/></noscript></span></span> is the sample mean, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-5.gif"/></noscript></span></span> defines the genotype of the <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-6.gif"/></noscript></span></span> marker, and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-7.gif"/></noscript></span></span> is an error term normally distributed with mean zero and variance <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-8.gif"/></noscript></span></span> (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Logsdon <em>et al.</em> 2010</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Carbonetto and Stephens 2012</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Yamamoto <em>et al.</em> 2017</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Li and Sillanpää 2012</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Teschendorff <em>et al.</em> 2005</a>). Exceptions to the use of simple linear models in VBM were studied by <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Hayashi and Iwata (2013)</a>, who modeled multiple traits without effects of environments and without interaction terms; by <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Arakawa <em>et al.</em> (2016)</a>, who used a model for univariate responses with mixed effects without interaction terms; and in the research of <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Li and Sillanpää (2013)</a>, who used a model for functional regression without random effects and without interaction terms.</p><p id="p-7">Based on the above, the purpose of this study was to propose a univariate genomic VBM that takes into account genotype × environment interaction (G×E), and could be an alternative to the existing Bayesian and non-Bayesian models with G×E described by <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Jarquín <em>et al.</em> (2014)</a> in the reaction norm model. Since animal and plant breeding programs today perform genetic evaluations of large numbers of lines, or animals in many environments, for several traits and genotyped with thousands (or millions) of genetic markers, complex Bayesian models may take days or weeks to do this, and, as a result, the existing models become untenable.</p><p id="p-8">For this reason, in order to retain all the useful features of the Bayesian genomic models with G×E at a reduced computational cost, we propose, within the general framework of variational Bayes, a method for estimating the univariate genomic model with G×E, which we call the variational Bayes multi-environment (VBME) model. This means that the VBME will recast the problem of computing posterior densities by the Gibbs sampler of the conventional Bayesian methods with G×E—referred to here as the Bayesian multi-environment (BME) model—as an optimization problem using the VBM, which consists of introducing a class of approximating distributions of the latent variables, then optimizing some criteria to find the distribution within this class that best matches the posterior (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Carbonetto and Stephens 2012</a>). Also, it is important to point out that the newly proposed method uses, as a prior for all standard deviations (SDs), a half-t distribution that is a high noninformative prior distribution (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Huang and Wand 2013</a>). We derived the full conditional and variational posterior distribution of the VBME model, explained its implementation, and applied it to eight experimental genomic (maize and wheat) data sets with two main objectives: (1) to compare the accuracy of the VBME with that of the standard BME model; and (2) to compare the computational processing time of VBME with that of BME.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>The variational Bayes method using the density transform approach</h3><p id="p-9">The most common variants of the VBM are the density transform approach and the tangent transform approach. We will explain the basis of the density transform approach; readers interested in understanding the tangent transform approach can consult <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Ormerod and Wand (2010)</a>. The density transform approach consists of approximating the posterior densities through other densities for which inference is more tractable. In Bayesian statistics, the inference problem is to compute the conditional distribution of unknown parameters given the observations <em>p</em>(<strong><em>θ</em></strong>│<strong><em>y</em></strong>) = <em>p</em>(<strong><em>y</em></strong>,<strong><em>θ</em></strong>)/<em>p</em>(<strong><em>y</em></strong>); the denominator is known as the marginal distribution of the observations or model evidence, <strong><em>θ</em></strong>∈<strong>Θ</strong> is the parameter vector, and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-9.gif"/></noscript></span></span> is the joint likelihood of the data and model parameters. The essence of the density transform approach is to approximate the posterior density <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-10.gif"/></noscript></span></span> by a function <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-11.gif"/></noscript></span></span>, for which the <em>q</em>-dependent lower bound <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-12.gif"/></noscript></span></span>on the marginal likelihood is given by<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-13.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>Expression (1) is more tractable than the marginal likelihood, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-14.gif"/></noscript></span></span> Indeed, the maximization of <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-15.gif"/></noscript></span></span> with respect to <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-16.gif"/></noscript></span></span> is equivalent to minimizing the Kullback-Leibler divergence between <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-17.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-18.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-19.gif"/></noscript></span></span> This is because the marginal likelihood satisfies<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-20.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>which can be derived through Jensen’s inequality, and the equality holds if, and only if, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-21.gif"/></noscript></span></span> almost everywhere (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Ormerod and Wand 2010</a>). To achieve tractability, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-22.gif"/></noscript></span></span> is maximized over a restricted, more manageable, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-23.gif"/></noscript></span></span> class of densities.</p><p id="p-10">Here, for the implementation, we chose, as restrictions for the <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-24.gif"/></noscript></span></span> class of densities, the <em>mean field variational family restriction</em>, where the latent variables are mutually independent, and each is governed by a distinct factor in the variational distribution, that is, <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-25.gif"/></noscript></span></span> factorizes into <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-26.gif"/></noscript></span></span> for some partitions <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-27.gif"/></noscript></span></span> of <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-28.gif"/></noscript></span></span> Each <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-29.gif"/></noscript></span></span> is governed by its own distribution <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-30.gif"/></noscript></span></span>, and, in principle, each variational factor can take on any parametric form that is appropriate for the corresponding random variable. In the optimization process, these variational factors are chosen to maximize the evidence lower bound given in Equation (2). One advantage of using the mean field variational family restriction over <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-31.gif"/></noscript></span></span> is that it is possible to obtain explicit solutions for each product component in terms of the others when conjugate priors are used. These lead to an interactive scheme for obtaining simultaneous solutions. The solutions rely on result 2.1.</p><p id="p-11">Result. 2.1. Let <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-32.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-33.gif"/></noscript></span></span> be continuous random vectors with joint density <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-34.gif"/></noscript></span></span> Then<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-35.gif"/></noscript></span></span>is attained by <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-36.gif"/></noscript></span></span></p><p id="p-12">One disadvantage of the mean field variational family restriction is its tendency to underestimate the variability of parameter estimates; it is also not a good option when the parameters under study have a high degree of dependence, which causes degradation in the resulting inference. Conversely, if the posterior dependence between parameters is weak, then the product density restriction could lead to very accurate approximate inferences (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Ormerod and Wand 2010</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Blei <em>et al.</em> 2016</a>). When conditional conjugate priors are used, given the other variational factor components, the optimal densities of each variational factor, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-37.gif"/></noscript></span></span> are proportional to the exponentiated expected log of the full conditionals obtained with Gibbs sampling, that is,<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-38.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-39.gif"/></noscript></span></span> are the known full conditionals in the MCMC literature, and the expectations on the right-hand side of Equation (3) are computed with respect to the density <em>q</em>(<strong><em>θ</em></strong>)/<em>q</em>(<strong><em>θ</em></strong><em><sub>j</sub></em>). From this equation, an interactive scheme can be used to obtain the optimal densities. Equation (3) allows obtaining all the variational posteriors for implementing the VBM using mean variational approximation.</p></div><div id="sec-3" class="subsection"><h3>Statistical model</h3><p id="p-13">Since we wanted to develop a variational Bayes version of the BME model, we used <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-40.gif"/></noscript></span></span> to represent the normal phenotype from the <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-41.gif"/></noscript></span></span>th replication of the <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-42.gif"/></noscript></span></span>th line in the <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-43.gif"/></noscript></span></span>th environment <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-44.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-45.gif"/></noscript></span></span> represents the number of replicates of each line in each environment under study. Therefore, the total number of observations is <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-46.gif"/></noscript></span></span> Therefore, the BME model is<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-47.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-48.gif"/></noscript></span></span> represents the effect of <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-49.gif"/></noscript></span></span>th environment, <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-50.gif"/></noscript></span></span> represents the genomic effect of <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-51.gif"/></noscript></span></span>th line, and is assumed as a random effect, <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-52.gif"/></noscript></span></span> is the interaction between the genomic effect of the <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-53.gif"/></noscript></span></span>th line and the <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-54.gif"/></noscript></span></span>th environment, and is assumed a random effect, and <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-55.gif"/></noscript></span></span> is a random error term associated with the <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-56.gif"/></noscript></span></span>th replication of the <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-57.gif"/></noscript></span></span>th line in the <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-58.gif"/></noscript></span></span>th environment.</p><p id="p-14">In matrix notation, the model given in Equation (4) including all the information is expressed as:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-59.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-60.gif"/></noscript></span></span> is the response vector of order <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-61.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-62.gif"/></noscript></span></span> is the design matrix of environment effects of order <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-63.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-64.gif"/></noscript></span></span> is the regression coefficient vector of order <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-65.gif"/></noscript></span></span> associated with the environment effects, <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-66.gif"/></noscript></span></span> is the design matrix of order <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-67.gif"/></noscript></span></span> associated with the random effects of lines, <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-68.gif"/></noscript></span></span> is the vector of random effects of lines of order <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-69.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-70.gif"/></noscript></span></span> is the design matrix of order <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-71.gif"/></noscript></span></span> associated with the interaction random effects between lines and environments, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-72.gif"/></noscript></span></span> is the vector of interaction random effects of lines and environments of order <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-73.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-74.gif"/></noscript></span></span> is the vector of random error terms of order <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-75.gif"/></noscript></span></span>. Then, we assumed that <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-76.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-77.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-78.gif"/></noscript></span></span> also that this vector is statistically independent, where <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-79.gif"/></noscript></span></span> denotes the genomic relationship matrix, and was calculated as suggested by <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">VanRaden (2008)</a>. <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-80.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-81.gif"/></noscript></span></span> denotes a Kronecker product, where <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-82.gif"/></noscript></span></span> is an identity matrix of order <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-83.gif"/></noscript></span></span> which indicates that we are assuming independence between environments.</p></div><div id="sec-4" class="subsection"><h3>BME model</h3><p id="p-15">The joint posterior density of the parameter vector of the BME model is given by<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-84.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>which is not a known distribution that is easy to simulate, so we will use MCMC techniques to obtain samples of this by means of the Gibbs sampler. The notation <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-85.gif"/></noscript></span></span> represents a random variable, <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-86.gif"/></noscript></span></span>, that has an inverse Gamma distribution with shape <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-87.gif"/></noscript></span></span> and scale <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-88.gif"/></noscript></span></span> parameters <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-89.gif"/></noscript></span></span> with density function <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-90.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-91.gif"/></noscript></span></span> The following prior specification is given:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-92.gif"/></noscript></span></span>It is important to point out that the hierarchical priors given to the SDs induce Half<span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-93.gif"/></noscript></span></span> priors to achieve arbitrary high noninformativity in this parameter (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Huang and Wand 2013</a>). Note that the use of this type of priors of our proposed model for genomic-enabled prediction is different than the priors used in the context of the existing Bayesian univariate models with G×E. Details of all full conditional distributions derived from the Gibbs sampler for implementing these BME and VBME models are given in Appendices A and B, whereas the R code for implementing the VBME model is provided in Appendix D.</p></div><div id="sec-5" class="subsection"><h3>Variational version of the BME model</h3><p id="p-16">As an alternative to the Gibbs sampler for estimating the parameters, in this section, we will derive the optimal densities under the mean field variational family restriction, assuming the following factorization of <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-94.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-95.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>The difficult problem of finding the exact posterior (6) became less difficult, <em>i.e.</em>, it now consists of finding an approximate parametric posterior <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-96.gif"/></noscript></span></span> with moments (<em>i.e.</em>, parameters). Inference then reduces to finding a density <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-97.gif"/></noscript></span></span> that minimizes a measure of dissimilarity between <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-98.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-99.gif"/></noscript></span></span> This can be achieved by maximizing the log of the evidence lower bound that is an approximation of the BME model using Equation (6) with respect to (the moments of) <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-100.gif"/></noscript></span></span> For details, see <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">MacKay (1995)</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Attias (2000)</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ghahramani and Beal (2001)</a> and <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bishop (1999)</a>.</p><p id="p-17">Because in the Bayesian formulation of the model given in Equation (5), we use conditional conjugate priors over parameters of the corresponding partition of <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-101.gif"/></noscript></span></span> in the factorization assumed for <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-102.gif"/></noscript></span></span> in Equation (7), all the optimal distributions (Equation 3) of the VBME model of each variational factor, <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-103.gif"/></noscript></span></span> belong to the same family as the full conditional distributions for the BME model. Because the same Bayesian model (BME) was converted to its variational counterpart (VBME), both models are comparable. The derivation details of all these densities are given in Appendix A.</p><p id="p-18">In summary, the iterative scheme used to obtain the optimal density <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-104.gif"/></noscript></span></span> over the class of densities given in Equation (7), is given by the following algorithm. Before presenting the algorithm, we provide some basic notations. We will use <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-105.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-106.gif"/></noscript></span></span> to denote the mean and variance of <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-107.gif"/></noscript></span></span> distribution. For vector parameter <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-108.gif"/></noscript></span></span> we use the analogously defined <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-109.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-110.gif"/></noscript></span></span> Therefore, the expected values for the inverse of the parameters <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-111.gif"/></noscript></span></span> used were: <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-112.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-113.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-114.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-115.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-116.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-117.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-118.gif"/></noscript></span></span> respectively, while, for the scale parameters, we will use the notation <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-119.gif"/></noscript></span></span> Therefore, <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-120.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-121.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-122.gif"/></noscript></span></span> represent the scale parameters of the variational posterior distribution of <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-123.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-124.gif"/></noscript></span></span> respectively. Also, <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-125.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-126.gif"/></noscript></span></span> is the prior of the beta regression coefficients.</p></div><div id="sec-6" class="subsection"><h3>Algorithm for implementing the VBME</h3><p id="p-19">Initialize: <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-127.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-128.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-129.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-130.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-131.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-132.gif"/></noscript></span></span>Cycle:<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-133.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-134.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-135.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-136.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-137.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-138.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-139.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-140.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-141.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-142.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-143.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-144.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-145.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-146.gif"/></noscript></span></span>until the change in the lower bound, <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-147.gif"/></noscript></span></span> between iterations is less than a tolerance value specified <em>a priori</em>. For our choice of approximating distribution (see Appendix B), in each cycle of the algorithm, <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-148.gif"/></noscript></span></span> has the following analytical expression:<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-149.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>To obtain (8), several substitutions were made for simplification; it is valid only for the VBME model proposed. In this algorithm, the updates of the parameters should be as given in the order proposed for the algorithm.</p></div><div id="sec-7" class="subsection"><h3>Model implementation</h3><p id="p-20">Recall that the hyperparameters of the prior distributions are values given by the user in the Bayesian paradigm. In this case, the hyperparameters used for the BME model given in the previous section were: <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-150.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-151.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-152.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-153.gif"/></noscript></span></span> All these hyperparameters were chosen to lead weakly informative priors. The implementation of both models (BME and VBME) was done in the R statistical software (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">R Core Team 2016</a>). The full conditional and posterior distributions given in Appendix A were used for the BME and VBME models, respectively. Also, for the VBME model, an R code is given in Appendix D. The BME model was implemented with the MCMC method with 40,000 iterations with a burn-in of 20,000 and a thinning of five; therefore, 4000 samples were used for inference.</p></div><div id="sec-8" class="subsection"><h3>Assessing prediction accuracy</h3><p id="p-21">Prediction accuracy was assessed using 20 training (trn)-testing (tst) random partitions; we used this approach because one can obtain as many partitions as one needs with a replicated trn-tst design. The implemented cross-validation (CV1) mimics a situation where lines were evaluated in some environments for the trait of interest, but some lines are missing from all the other environments. We assigned 80% of the observations to the trn set, and the remaining 20% to the tst set. Of the variety of methods for comparing the predictive posterior distribution to the observed data (generally termed “posterior predictive checks”), we used the Pearson correlation. Models with higher correlation values indicate better predictions. Under the BME, the predicted observations were calculated as <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-154.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-155.gif"/></noscript></span></span> are estimates of <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-156.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-157.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-158.gif"/></noscript></span></span> these estimates correspond to the posterior mean of <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-159.gif"/></noscript></span></span> collected Gibbs samplers to estimate the predicted values after discarding those used for the burning period, while under the VBME, <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-160.gif"/></noscript></span></span> as described in the algorithm for implementing the VBME.</p></div><div id="sec-9" class="subsection"><h3>Experimental data sets</h3><p id="p-22">Here, we present the information on the data sets used for implementing the proposed models. A total of eight data sets were used (three maize and five wheat).</p><div id="sec-10" class="subsection"><h4>Maize data sets 1–3:</h4><p id="p-23">A total of 309 doubled haploid maize lines was phenotyped and genotyped; this is part of the data set used by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em> (2013)</a> and <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Montesinos-López <em>et al.</em> (2016)</a>, which comprised a total of 504 doubled haploid lines derived by crossing and backcrossing eight inbred lines to form several full-sib families. Traits available in this data set include grain yield (Maize_GY), anthesis-silking interval (Maize_ASI), and plant height (Maize_PH); each of these traits was evaluated in three optimum rainfed environments (Env1, Env2, and Env3). The experimental field design used in each of the three environments was an alpha lattice incomplete block design with two replicates. Data were preadjusted using estimates of block and environmental effects derived from a linear model that accounted for the incomplete block design within environment, and for environmental effects.</p><p id="p-24">The genomic data were obtained through genotyping-by-sequencing (GBS) for each maize chromosome; the number of markers after initial filtering and the number of markers after imputation were summarized in <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em> (2013)</a>. Filtering was first done by removing markers that had &gt;80% of the maize lines with missing values, and then markers with minor allele frequency ≤0.05 were deleted. The total numbers of GBS data were 681,257 single nucleotide polymorphisms (SNPs), and, after filtering for missing values and minor allele frequency, 158,281 SNPs were used for the analyses. About 20% of cells were missing in the filtered GBS information used for prediction; these missing values were replaced by their expected values before doing the prediction.</p></div><div id="sec-11" class="subsection"><h4>Wheat data sets 4–5:</h4><p id="p-25">The first two wheat data sets came from a total of 250 wheat lines that were extracted from a large set of 39 yield trials grown during the 2013–2014 crop season in Ciudad Obregon, Sonora, Mexico (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Rutkoski <em>et al.</em> 2016</a>). The trials were sown in mid-November and grown on beds with five and two irrigations plus drip irrigation. Days to heading (Wheat_DTHD) were recorded as the number of days from germination until 50% of spikes had emerged in each plot, in the first replicate of each trial, while plant height (Wheat_PTHT) was recorded in centimeters.</p><p id="p-26">For the first two wheat data sets, GBS was used for genome-wide genotyping. Single nucleotide polymorphisms were called across all lines using the TASSEL GBS pipeline anchored to the genome assembly of Chinese Spring. Single nucleotide polymorphisms were extracted and markers were filtered so that percent missing data did not exceed 80 and 20%, respectively. Individuals with &gt;80% missing marker data were removed, and markers were recorded as −1, 0 and 1, which indicate homozygous for the minor allele, heterozygous, and homozygous for the major allele, respectively. Next, markers with &lt;0.01 minor allele frequency were removed, and missing data were imputed with the marker mean. A total of 12,083 markers remained after marker editing.</p></div><div id="sec-12" class="subsection"><h4>Wheat data set 6:</h4><p id="p-27">This data set, from CIMMYT’s Global Wheat Program, was used by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> (2010)</a> and <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cuevas <em>et al.</em> (2016a</a>,<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">b</a>) and includes 599 wheat lines derived from 25 yr (1979–2005) of Elite Spring Wheat Yield Trials (ESWYT). The environments represented in these trials are four agroclimatic regions (mega-environments). The phenotypic trait considered here was grain yield (Wheat_Yield1) of the 599 wheat lines evaluated in each of the four mega-environments. The 599 wheat lines were genotyped using 1447 Diversity Array Technology (DArT) markers generated by Triticarte Pty. Ltd. (Canberra, Australia; <a href="http://www.triticarte.com.au">http://www.triticarte.com.au</a>). Markers with a minor allele frequency (MAF) (0.05) were removed, and missing genotypes were imputed using samples from the marginal distribution of marker genotypes. The number of DArT markers after edition was 1279.</p></div><div id="sec-13" class="subsection"><h4>Wheat data sets 7–8:</h4><p id="p-28">These two data sets were described and used by <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lopez-Cruz <em>et al.</em> (2015)</a> and <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cuevas <em>et al.</em> (2016b)</a> for proposing a marker×environment interaction model. The phenotypic data consisted of adjusted grain yield (tons/hectare) records collected during three evaluation cycles of different inbred lines evaluated in different environments, and denoted as Wheat_Yield2 and Wheat_Yield3. The environments were three irrigation regimes (moderate drought stress, optimal irrigation, and drought stress), two planting systems (bed and flat planting), and two different planting dates (normal and late). All trials were planted using a lattice design with three replicates in each environment at CIMMYT’s main wheat breeding station in Cd. Obregon, México. The phenotype used in the analysis was the Best Linear Unbiased Estimate (BLUE) of grain yield obtained from a linear model applied to the alpha-lattice design of each cycle-environment combination. The trait Wheat_Yield2 had 693 wheat lines evaluated in four environments, and the trait Wheat_Yield3 included 670 wheat lines evaluated in four environments. Genotypes were derived using GBS technology, and markers with a MAF of 0.05 were removed. All markers had a high incidence of uncalled genotypes, so we applied thresholds for incidence of missing values, and focused on maintaining relatively large and similar numbers of markers per data set. After editing the missing markers, we had a total of 15,744 GBS markers for analyzing traits Wheat_Yield2 and Wheat_Yield3.</p></div></div><div id="sec-14" class="subsection"><h3>Data availability</h3><p id="p-29">Phenotypic and genotypic data on the eight data sets included in this study can be downloaded from <a href="http://hdl.handle.net/11529/10907">http://hdl.handle.net/11529/10907</a>.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><p id="p-30">The results of this research are given in two sections. The first section compares the parameter estimates and implementation time under both models (BME and VBME). The second section compares the prediction accuracy of both models in each of the eight data sets under study.</p><div id="sec-16" class="subsection"><h3>Comparing parameter estimates and the implementation time of models BME and VBME</h3><p id="p-31"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> gives the parameter estimates under both models (BME and VBME) of the <strong><em>β</em></strong> coefficients, variance components (<span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-161.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-162.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-163.gif"/></noscript></span></span>), Pearson correlation between the observed and predicted phenotypes of the full data sets, and the time in minutes required to implement each model on the data sets under study. In general, the beta coefficient estimates obtained under the BME and VBME models are very similar, but there are substantial differences in the estimates of variance components, given that a little more than half of them were lower under the VBME model as compared to those estimates obtained under the BME. Also, in seven out of eight data sets, the Pearson correlation between observed and predicted genotypes was larger under the BME, while in seven out of eight data sets, the implementation time was ∼10 times lower under the VBME than under BME.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484976/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484976/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Parameter estimates, implementation time and Pearson correlation between the observed and predicted values for the complete data sets under the BME and VBME models</span><div class="sb-div caption-clear"></div></div></div><p id="p-33"><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows details of the comparison between the proposed VBME model <em>vs.</em> the BME model. The smallest difference between the beta coefficients was observed in <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-171.gif"/></noscript></span></span> for data set Wheat_Yield1 (1%), while the largest difference was observed in <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-172.gif"/></noscript></span></span> for data set Wheat_Yield2 (25.3%); the estimates of the BME model were lower. With regard to the variance components, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> shows that the smallest difference was observed in <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-173.gif"/></noscript></span></span> for Wheat_Yield3 (3.8%), while the largest difference was also in <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-174.gif"/></noscript></span></span> for data set Maize_PH, where the estimated variance component of the VBME was 5.68 times higher than the estimated variance component of the BME model.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484977/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484977/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Relative comparison using the full data sets between the BME and VBME calculated as parameter estimate under BME minus parameter estimate under VBME divided by parameter estimate under BME</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">In terms of the Pearson correlation, in seven out of eight data sets, the best correlations were observed in the BME model; the minimum difference (1.6%) was observed in data set Wheat_Yield3, while the maximum difference (15.4%) was observed in Wheat_PTHT. Finally, in terms of implementation time (in minutes), in seven out of eight data sets the VBME model was faster than the BME model, with a minimum difference of 16.9%, and a maximum difference of 98.2%. The computer used to implement both models for each of the eight data sets (reported in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) was a Windows operating system with processor Intel(R) Core(TM) i7-4610M CPU @ 3.0 GHz under a 64-bit operating system.</p></div><div id="sec-17" class="subsection"><h3>Comparing the prediction ability of models BME and VBME</h3><p id="p-36"><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> presents prediction accuracies for both models resulting from the 20 trn-tst random partitions implemented for each of the eight real data sets under study. <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> gives the prediction accuracies for each environment in each data set. In general, the prediction accuracies of the BME are slightly higher than those of the VBME model. However, it is interesting to point out that, even under the VBME, the prediction accuracies are comparable to those found under the BME.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484864/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484864/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Prediction accuracy of the BME and VBME models resulting from the 20 trn-tst random partitions implemented</span><div class="sb-div caption-clear"></div></div></div><p id="p-38"><a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> shows that in five out of eight data sets the predictions under the BME model were the best. Also, the lowest difference in the five data sets in which the BME model was the best was 4.2% in favor of the BME in Env1 in data set Wheat_Yield2, while the largest difference was 52.9% in favor of the BME in Env4 in data set Wheat-Yield1. For the three data sets in which the VBME was the best, the smallest difference was 1.2% in Env2 in data set Wheat_DTHD, and the largest difference was 138.1% in Env1 in data set Wheat_Yield1.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/484841/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/484841/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Relative comparison of the prediction accuracies of the BME and VBME models calculated as APC under BME minus APC VBME divided by the APC under BME</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-40">We proposed and implemented the variational Bayes version of the BME model. We found that the <em>β</em> coefficients were estimated in a similar way in both models (BME and VBME), but the variance components were considerably underestimated in the VBME model. However, in terms of prediction accuracy, the BME was a little better than the VBME model. The VBME model is a quick and a good approximation to the BME model. These results were expected, since, under MCMC methods, we estimate the parameters using the mean posterior from samples of the target distribution (posterior distribution), while, in the second approach for estimating the parameters, we also use the mean, but from the approximated target distribution, here obtained under the VBM and called variational posterior distribution. The above is documented in some papers that reported that the VBM showed less accuracy than its MCMC counterpart. This is due to the fact that variational Bayes methods approximate the target posterior distribution by a factorized distribution that has minimum KL distance to the target posterior. The factorized distribution we used was the mean field variational family restriction, which makes strong independent assumptions to achieve great flexibility and scalable optimization at the price of low accuracy and underestimation of posterior variances (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Blei <em>et al.</em> 2016</a>). In contrast, MCMC methods such as the BME model have a high computational cost with the advantage that they generate samples directly from the target posterior distribution that are guaranteed to converge asymptotically to the true posterior distribution. The new proposed VBME model for univariate phenotypes with G×E interaction is a step ahead of the models proposed until now, in the context of genomic-enabled prediction models proposed by other authors and described in the <em>Introduction</em>. In this paper, we propose a genomic VBM model using a half-<em>t</em> priors distribution, and give clear theoretical details of the derivation of the full conditional and posterior distributions, and account for G×E interaction.</p><p id="p-41">Advances in genetic engineering allow collecting and storing millions of markers (SNPs, etc.) from plant and animal genomes, and many breeding programs are interested in improving many correlated traits simultaneously. For these reasons, in genomic selection there is a great need to use genome-enabled prediction models that take into account all available markers, traits, and environments under study to improve the prediction of genotypes early in time for multiple correlated traits. However, due to the large amount of input information for Bayesian methods (such as those used in the BME model), these statistical models and methods might become intractable, or impose a very high computational cost in terms of time. For these reasons, and because it offers a significant improvement in computational speed at the cost of a small loss in prediction accuracy, we believe that the proposed VBME model is an attractive alternative to the BME model.</p><div id="sec-19" class="subsection"><h3>Advantages and disadvantages of the variational model (VBME)</h3><p id="p-42">As already mentioned, the variational model (VBME) was proposed due to the need to improve the speed of the BME. For this reason, we propose the iterative algorithm given in the section <em>Algorithm for implementing the VBME</em>, which approximates the BME model in a reasonable way. Results show that the VBME method: (1) is sometimes more flexible than its Bayesian counterpart; (2) offers a reasonable approximation to the posterior distributions of all model parameters; (3) most of the time is much less demanding in computational time than the BME built using MCMC techniques; and (4) maintains most of the advantages of Bayesian inference. However, the VBME has some limitations: (1) the approximation cannot become more accurate by increasing computational time; (2) part of the VBME’s advantage in computing time is because the model is simple; however, this algorithm needs to be adapted to cases that are different from the one used here; (3) the VBME, like all variational Bayes methods, does not guarantee that it will converge to a global maximum; (4) the convergence time depends strongly on the level of tolerance desired and the nature of the data; and (5) it has been documented, and was observed here, that variational models underestimate some covariance parameters. However, the time reduction depends strongly on the tolerance level: the lower the value, the longer the convergence time. For this reason, although our results show a significant reduction in time, this gain can be reduced substantially if the chosen tolerance level is smaller. Our empirical results when implementing the VBME in eight real data sets show that a tolerance level of 0.00001 is enough, because when we increased it to 1E−8, no significant gain in prediction accuracy or accuracy in parameter estimates was observed. Also, since in developing our VBME, we used the mean field variational family restriction that approximates <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-182.gif"/></noscript></span></span> by the product of <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-183.gif"/></noscript></span></span> latent independent variables to gain tractability, this may underestimate the variability of parameter estimates when there is a considerable degree of dependence. Another important disadvantage of the VBM is that deriving the set of equations (variational posterior and lower bound) used to iteratively update the parameters often requires a large amount of algebra compared with deriving the comparable Gibbs sampling equations. This is evident in Appendix A for our proposed VBME model, which, despite being a simple model, required a lot of algebra and notation to obtain the variational posterior distribution and the corresponding lower bound. This amount of algebra needs to be done each time for every specific model.</p><p id="p-43">To take advantage of the proposed model, we provide some guidance on when to use the VBME or the BME. The BME (an MCMC method) tends to be more computationally intense than the VBME (a variational method) but guarantees that exact samples from the target density are (asymptotically) produced (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Robert 2004</a>). While the VBME does not guarantee exact samples, only approximate ones (that is, it can only find a density close to the target), most of the time it is faster than MCMC methods. Since the VBME is based on optimization, it can more easily take advantages of stochastic optimization and distributed optimization. For these reasons, the VBME is expected to be better suited to larger data sets than the BME and for scenarios where we want to quickly explore many models. Also, <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Blei <em>et al.</em> (2016)</a> point out that the VBME is preferred when the posterior of a mixture model admits multiple modes, since MCMC techniques are not an option for this scenario, even with small data sets.</p><p id="p-44">Finally, we believe that the VBME method, as proposed here, is a useful addition to the existing arsenal of genome-enabled prediction models; it is also an alternative to the BME model, since it considerably reduces implementation time. Additionally, this research opens a new branch of methodologies for fitting genomic models to genomic data that need novel methods to take advantage of the huge amount of data available nowadays. For these reasons, we agree with <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Ormerod and Wand (2010)</a>, who pointed out that the usefulness of variational approximations increases as the size of the problem increases, and Monte Carlo methods such as MCMC start to become untenable. We also believe that our research contributes significantly to researchers’ understanding of the VBM, because, in developing the proposed VBME, we provided all the details of the derivation of the variational posterior distributions needed to derive the q-specific evidence lower bound for building the algorithm, and the R scripts for implementing the proposed VBME model. This information gives the scientist the basic elements needed to develop new models under the variational Bayes approach. It is also important to point out that we provide all the details for deriving the full conditionals needed for implementing the BME model since the proposed model uses Half-<span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-184.gif"/></noscript></span></span> priors that produce noninformative variance components (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Huang and Wand 2013</a>).</p></div><div id="sec-20" class="subsection"><h3>Conclusions</h3><p id="p-45">In this paper, we propose a new alternative to the BME via variational Bayes, which we called the VBME model. The proposed method underestimated the variance components and similar estimates of the beta coefficients were found; in terms of prediction accuracy, the VBME had lower prediction accuracies than the BME. However, according to our results, the loss in prediction accuracy is compensated for by the significant gain in implementation time, since the VBME is much faster than the BME. However, it is important to point out that if a very low tolerance value is selected, the gain in terms of implementation speed will vanish. For this reason, to take advantage of the proposed VBME, a reasonable tolerance value should be chosen that guarantees the quality of the parameter estimates and predictions, as well as a significant reduction in implementation time.</p><p id="p-46">We also believe that the proposed model should be tested on many real data sets to obtain more empirical evidence of its performance; this evidence would allow us to extend the proposed variational Bayes model to take into account nonnormal responses (binary, Poisson, negative binomial, etc.), for multiple traits and multiple-environment models (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Montesinos-López <em>et al.</em> 2016</a>), which would enable scientists to make a more precise selection of candidate genotypes at early crop stages. Also, models that avoid the product density restriction can be proposed. Finally, the proposed VBME model is a useful addition to the existing arsenal of genome-enabled prediction models, and an excellent alternative to the BME model, since it produces competitive predictions at a lower computational cost.</p></div></div><div class="section app" id="app-1"><h2 class="">Appendix A</h2><div class="section" id="sec-21"><h3>Derivation of full conditional and variational posterior distributions for the BME and VBME models</h3><div id="sec-22" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-185.gif"/></noscript></span></span></h4><p id="p-47"><span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-186.gif"/></noscript></span></span>where:<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-187.gif"/></noscript></span></span></p></div><div id="sec-23" class="subsection"><h4>Variational posterior for <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-188.gif"/></noscript></span></span></h4><p id="p-48"><span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-189.gif"/></noscript></span><span class="disp-formula-label">(A.11)</span></span>where<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-190.gif"/></noscript></span></span></p></div><div id="sec-24" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-191.gif"/></noscript></span></span></h4><p id="p-49"><span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-192.gif"/></noscript></span></span></p></div><div id="sec-25" class="subsection"><h4>Variational posterior for <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-193.gif"/></noscript></span></span></h4><p id="p-50"><span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-194.gif"/></noscript></span></span></p></div><div id="sec-26" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-195.gif"/></noscript></span></span></h4><p id="p-51"><span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-196.gif"/></noscript></span></span></p></div><div id="sec-27" class="subsection"><h4>Variational posterior for <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-197.gif"/></noscript></span></span></h4><p id="p-52"><span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-198.gif"/></noscript></span></span></p></div><div id="sec-28" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-199.gif"/></noscript></span></span></h4><p id="p-53"><span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-200.gif"/></noscript></span></span>where<span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-201.gif"/></noscript></span></span></p></div><div id="sec-29" class="subsection"><h4>Variational posterior for <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-202.gif"/></noscript></span></span></h4><p id="p-54">Defining <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-203.gif"/></noscript></span></span>, the variational posterior distribution of <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-204.gif"/></noscript></span></span> is given as<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-205.gif"/></noscript></span></span>where<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-206.gif"/></noscript></span></span></p></div><div id="sec-30" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-207.gif"/></noscript></span></span></h4><p id="p-55"><span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-208.gif"/></noscript></span></span></p></div><div id="sec-31" class="subsection"><h4>Variational posterior for <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-209.gif"/></noscript></span></span></h4><p id="p-56"><span class="disp-formula" id="disp-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-210.gif"/></noscript></span></span></p></div><div id="sec-32" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-211.gif"/></noscript></span></span></h4><p id="p-57"><span class="disp-formula" id="disp-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-212.gif"/></noscript></span></span></p></div><div id="sec-33" class="subsection"><h4>Variational posterior for <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-213.gif"/></noscript></span></span></h4><p id="p-58"><span class="disp-formula" id="disp-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-214.gif"/></noscript></span></span></p></div><div id="sec-34" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-215.gif"/></noscript></span></span></h4><p id="p-59"><span class="disp-formula" id="disp-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-216.gif"/></noscript></span></span>where:<span class="disp-formula" id="disp-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-217.gif"/></noscript></span></span></p></div><div id="sec-35" class="subsection"><h4>Variational posterior for <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-218.gif"/></noscript></span></span></h4><p id="p-60">Defining <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-219.gif"/></noscript></span></span> the variational posterior distribution of <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-220.gif"/></noscript></span></span> is given as<span class="disp-formula" id="disp-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-221.gif"/></noscript></span></span>where<span class="disp-formula" id="disp-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-222.gif"/></noscript></span></span></p></div><div id="sec-36" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-223.gif"/></noscript></span></span></h4><p id="p-61"><span class="disp-formula" id="disp-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-224.gif"/></noscript></span></span></p></div><div id="sec-37" class="subsection"><h4>Variational posterior for <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-225.gif"/></noscript></span></span></h4><p id="p-62"><span class="disp-formula" id="disp-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-226.gif"/></noscript></span></span></p></div><div id="sec-38" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-227.gif"/></noscript></span></span></h4><p id="p-63"><span class="disp-formula" id="disp-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-228.gif"/></noscript></span></span></p></div><div id="sec-39" class="subsection"><h4>Variational posterior for <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-229.gif"/></noscript></span></span></h4><p id="p-64"><span class="disp-formula" id="disp-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-230.gif"/></noscript></span></span></p></div><div id="sec-40" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-231.gif"/></noscript></span></span></h4><p id="p-65"><span class="disp-formula" id="disp-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-232.gif"/></noscript></span></span></p></div><div id="sec-41" class="subsection"><h4>Variational posterior for <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-233.gif"/></noscript></span></span></h4><p id="p-66"><span class="disp-formula" id="disp-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-234.gif"/></noscript></span></span>with <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-235.gif"/></noscript></span></span></p></div><div id="sec-42" class="subsection"><h4>Full conditional for <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-236.gif"/></noscript></span></span></h4><p id="p-67"><span class="disp-formula" id="disp-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-237.gif"/></noscript></span></span></p></div><div id="sec-43" class="subsection"><h4>Variational posterior for <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-238.gif"/></noscript></span></span></h4><p id="p-68"><span class="disp-formula" id="disp-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-239.gif"/></noscript></span></span></p></div></div></div><div class="section app" id="app-2"><h2 class="">Appendix B</h2><div class="section" id="sec-44"><h3>Expression of the lower bound of the log likelihood for the variational Bayes model</h3><p id="p-69"><span class="disp-formula" id="disp-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-240.gif"/></noscript></span><span class="disp-formula-label">(B1)</span></span></p><p id="p-70">The first term in (B1) is<span class="disp-formula" id="disp-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-241.gif"/></noscript></span></span></p><p id="p-71">The second term in B1 is<span class="disp-formula" id="disp-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-242.gif"/></noscript></span></span></p><p id="p-72">The third term in B1 is<span class="disp-formula" id="disp-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-243.gif"/></noscript></span></span></p><p id="p-73">The fourth term in B1 is<span class="disp-formula" id="disp-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-244.gif"/></noscript></span></span></p><p id="p-74">The fifth term in B1 is<span class="disp-formula" id="disp-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-245.gif"/></noscript></span></span></p><p id="p-75">The sixth term in B1 is<span class="disp-formula" id="disp-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-246.gif"/></noscript></span></span></p><p id="p-76">The seventh term in B1 is<span class="disp-formula" id="disp-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-247.gif"/></noscript></span></span></p><p id="p-77">The eighth term in B1 is<span class="disp-formula" id="disp-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-248.gif"/></noscript></span></span></p><p id="p-78">The ninth term in B1 is<span class="disp-formula" id="disp-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-249.gif"/></noscript></span></span></p><p id="p-79">The tenth term in B1 is<span class="disp-formula" id="disp-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-250.gif"/></noscript></span></span></p><p id="p-80">The eleventh term in B1 is<span class="disp-formula" id="disp-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-251.gif"/></noscript></span></span></p><p id="p-81">The twelfth term in B1 is<span class="disp-formula" id="disp-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-252.gif"/></noscript></span></span></p></div></div><div class="section app" id="app-3"><h2 class="">Appendix C</h2><div class="section" id="sec-45"><h3>Alternative version of the algorithm for implementing the VBME</h3><p id="p-82">For the VBME algorithm given here is an alternative version that ignores the priors of the components <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-253.gif"/></noscript></span></span> since the variance component <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-254.gif"/></noscript></span></span> was fixed. Also, the components <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-255.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-256.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-257.gif"/></noscript></span></span> were ignored since the components <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-258.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-258.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-259.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-259.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-260.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-260.gif"/></noscript></span></span> were replaced by the hyperparameters Sb1, Sb2 and Sbe. To provide informative priors for these parameters, you can use Sb1=0.25*var(y)*<span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-261.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-261.gif"/></noscript></span></span>, Sb2=0.25*var(y)*<span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-262.gif"/></noscript></span></span> and Se= 0.5*var(y)*<span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-263.gif"/></noscript></span></span>.</p><p id="p-83">Initialize: <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-264.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-264.gif"/></noscript></span></span><strong>,</strong> <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-265.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-265.gif"/></noscript></span></span><strong>,</strong> <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-266.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-266.gif"/></noscript></span></span><strong>,</strong> <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-267.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-267.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-268.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-268.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-269.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-269.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-270.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-270.gif"/></noscript></span></span>.</p><p id="p-84">Cycle:<span class="disp-formula" id="disp-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-271.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-271.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-272.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-272.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-273.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-274.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-274.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-275.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-275.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-276.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-276.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-277.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-277.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-278.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-279.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-279.gif"/></noscript></span></span>until the change in the lower bound, <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-280.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-280.gif"/></noscript></span></span> between iterations is less than a tolerance value specified a priori. In this case, the corresponding <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-281.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-281.gif"/></noscript></span></span> has the following analytical expression:<span class="disp-formula" id="disp-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-282.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/6/1833/embed/mml-math-282.gif"/></noscript></span></span></p></div></div><div class="section app" id="app-4"><h2 class="">Appendix D</h2><div class="section" id="sec-46"><h3>R Code for Implementing the VBME Model</h3><p id="p-85">#####Set the working directory##################################################</p><p id="p-86">rm(list=ls())</p><p id="p-87">setwd(“C:\\Variationa Bayes Version\\VarBayes.Univariate”)</p><p id="p-88">####Data###########################</p><p id="p-89">load(“DATASET1.Wheat_GY.RData”)</p><p id="p-90">Env1=rep(1,599)</p><p id="p-91">Env2=rep(2,599)</p><p id="p-92">Env3=rep(3,599)</p><p id="p-93">Env4=rep(4,599)</p><p id="p-94">Entry=c(1:599)</p><p id="p-95">Env.All=c(Env1,Env2,Env3,Env4)</p><p id="p-96">Entry.All=c(Entry,Entry,Entry,Entry)</p><p id="p-97">YY.N=matrix(Y,ncol=1,byrow=F)</p><p id="p-98">Hybrids=cbind(Env.All,Entry.All,c(YY.N+0))</p><p id="p-99">colnames(Hybrids)=c(“Env”,”Entry”,”y”)</p><p id="p-100">Hyb=as.data.frame(Hybrids)</p><p id="p-101">##########Matrix of incidences#####################</p><p id="p-102">X=model.matrix(∼0+as.factor(Hyb$Env))</p><p id="p-103">Z1=model.matrix(∼0+as.factor(Hyb$Entry))</p><p id="p-104">svda&lt;-svd(G)</p><p id="p-105">D&lt;-diag(svda$d)</p><p id="p-106">d_Inv=(1/svda$d)</p><p id="p-107">D_Inv&lt;-diag(d_Inv)</p><p id="p-108">U&lt;-svda$u</p><p id="p-109">tV&lt;-t(svda$v)</p><p id="p-110">LL=U</p><p id="p-111">Z1=Z1%*%LL</p><p id="p-112">Gg=D</p><p id="p-113">Z2=model.matrix(∼0+Z1:as.factor(Hyb$Env))</p><p id="p-114">y=Hyb$y</p><p id="p-115">betav=c(colMeans(Y))</p><p id="p-116">tX=t(X)</p><p id="p-117">tZ1=t(Z1)</p><p id="p-118">tZ2=t(Z2)</p><p id="p-119">I=ncol(X)</p><p id="p-120">n=nrow(X)</p><p id="p-121">J=ncol(Z1)</p><p id="p-122">S_invB=diag(I)</p><p id="p-123">d2_Inv=rep(d_Inv,I)</p><p id="p-124">SigmaB=Sigmab1=Sigmab2=0.2</p><p id="p-125">Sig.e=0.1</p><p id="p-126">vb0=vb1=vb2=ve=2</p><p id="p-127">data.set&lt;-data.frame(Hyb)</p><p id="p-128">K=1</p><p id="p-129">####Lower bound of the log of the marginal likelihood (logp(y;q*)################</p><p id="p-130">logpyq&lt;- function(y,Mu.q.b0,Mu.q.b1,Mu.q.b2,Sig.q.b0,Sig.q.b1,Sig.q.b2,Bq.Sig.b0,Bq.Sig.b1,Bq.Sig.b2,Bq.Sig.be,X,Z1,Z2,Gg,vb0,vb1,vb2,ve){</p><p id="p-131">n.T=length(y)</p><p id="p-132">J=dim(Gg)[1] #############Number of lines#######################################</p><p id="p-133">I=dim(X)[2] ############Number of environments######################################</p><p id="p-134">K=1 ######Number of replications######################################</p><p id="p-135">Inv.Gg=Gg</p><p id="p-136">###############Expected values##################################################</p><p id="p-137">Mu.q.inv.sig.be=(0.5*(ve+n.T))/Bq.Sig.be</p><p id="p-138">Mu.q.inv.sig.b0=(0.5*(vb0+I))/Bq.Sig.b0</p><p id="p-139">Mu.q.inv.sig.b1=(0.5*(vb1+J))/Bq.Sig.b1</p><p id="p-140">Mu.q.inv.sig.b2=(0.5*(vb2+I*J))/Bq.Sig.b2</p><p id="p-141">Inv.IG.D2=diag(d2_Inv)</p><p id="p-142">tX=t(X)</p><p id="p-143">####Lower bound of the log of the marginal likelihood (logp(y;q*)################</p><p id="p-144">log.lik=0.5*log(det(Sig.q.b0))+0.5*log(det(Sig.q.b1))+0.5*sum(log(diag(Sig.q.b2)))-0.5*(vb1+J)*log(Bq.Sig.b1)-0.5*(vb2+I*J)*log(Bq.Sig.b2)-0.5*(ve+n.T)*log(Bq.Sig.be)}</p><p id="p-145">YMatrix=matrix(y,ncol=4,byrow=F)</p><p id="p-146">beta_zero=c(apply(YMatrix,2,mean,na.rm=TRUE))</p><p id="p-147">Mu.q.b0.sv=rep(1,I)</p><p id="p-148">Mu.q.b1.sv=rep(0.1,J)</p><p id="p-149">Mu.q.b2.sv=rep(0.1,I*J)</p><p id="p-150">Sig.q.b0.sv=diag(I)</p><p id="p-151">Sig.q.b1.sv=diag(J)</p><p id="p-152">Sig.q.b2.sv=diag(I*J)</p><p id="p-153">Bq.Sig.b0.sv=Bq.Sig.b1.sv=Bq.Sig.b2.sv=Bq.Sig.be.sv=1</p><p id="p-154">vb1=vb2=ve=5</p><p id="p-155">Sb1=0.25*var(y)*(vb1+1)</p><p id="p-156">Sb2=0.25*var(y)*(vb2+1)</p><p id="p-157">Se=var(y-X%*%beta_zero)*(ve+1)</p><p id="p-158">Inv.Gg=D_Inv</p><p id="p-159">n.T=length(y)</p><p id="p-160">tX=t(X)</p><p id="p-161">tZ1=t(Z1)</p><p id="p-162">tZ2=t(Z2)</p><p id="p-163">LL0=logpyq(y=y,Mu.q.b0=Mu.q.b0.sv,Mu.q.b1=Mu.q.b1.sv,Mu.q.b2=Mu.q.b2.sv,Sig.q.b0=Sig.q.b0.sv,Sig.q.b1=Sig.q.b1.sv,Sig.q.b2=Sig.q.b2.sv,Bq.Sig.b0=Bq.Sig.b0.sv,Bq.Sig.b1=Bq.Sig.b1.sv,Bq.Sig.b2=Bq.Sig.b2.sv,Bq.Sig.be=Bq.Sig.be.sv,X=X,Z1=Z1,Z2=Z2,Gg=D,vb0=vb0,vb1=vb1,vb2=vb2,ve=ve)</p><p id="p-164">LL0</p><p id="p-165">tolerance&lt;-c(0.00001) # Tolerance to declare convergence</p><p id="p-166">converged=FALSE</p><p id="p-167">maxIter=1000</p><p id="p-168">itnum=0</p><p id="p-169">while(!converged){</p><p id="p-170">itnum=itnum+1;</p><p id="p-171">######Updating the parameters of interest########################################</p><p id="p-172">if (itnum==1) previous.LL0&lt;-LL0</p><p id="p-173">if (itnum&gt;1) previous.LL0&lt;-current.LL0</p><p id="p-174">Mu.q.inv.sig.be=(0.5*(ve+n.T))/(Bq.Sig.be.sv)</p><p id="p-175">Mu.q.inv.sig.b0=1/10000</p><p id="p-176">Sig.q.b0=solve((diag(I))*Mu.q.inv.sig.b0+Mu.q.inv.sig.be*(tX%*%X))</p><p id="p-177">Mu.q.b0=Sig.q.b0%*%(Mu.q.inv.sig.be*(tX%*%(y-Z1%*%Mu.q.b1.sv-Z2%*%Mu.q.b2.sv))+((diag(I))*Mu.q.inv.sig.b0)%*%beta_zero)</p><p id="p-178">Mu.q.inv.sig.b1=(0.5*(vb1+J))/Bq.Sig.b1.sv</p><p id="p-179">DDD1=1/(d_Inv*Mu.q.inv.sig.b1+Mu.q.inv.sig.be*I)</p><p id="p-180">Sig.q.b1=diag(DDD1)</p><p id="p-181">Mu.q.b1=Sig.q.b1%*%(Mu.q.inv.sig.be*tZ1%*%(y-X%*%Mu.q.b0-Z2%*%Mu.q.b2.sv))</p><p id="p-182">Mu.q.inv.sig.b2=(0.5*(vb2+I*J))/Bq.Sig.b2.sv</p><p id="p-183">d22_Inv=c(d2_Inv)</p><p id="p-184">DDD2=1/(d22_Inv*Mu.q.inv.sig.b2+Mu.q.inv.sig.be)</p><p id="p-185">Sig.q.b2=diag(DDD2)</p><p id="p-186">Mu.q.b2=Sig.q.b2%*%(Mu.q.inv.sig.be*tZ2%*%(y-X%*%Mu.q.b0-Z1%*%Mu.q.b1))</p><p id="p-187">Bq.Sig.b1=0.5*(t(Mu.q.b1)%*%(Inv.Gg)%*%Mu.q.b1+sum(diag(Inv.Gg%*%Sig.q.b1)))+Sb1</p><p id="p-188">Inv.IG.D2=d22_Inv</p><p id="p-189">Bq.Sig.b2=0.5*(t(Mu.q.b2)%*%(diag(Inv.IG.D2))%*%Mu.q.b2 +sum(diag(diag(Inv.IG.D2)%*%Sig.q.b2)))+Sb2</p><p id="p-190">ym0=(y-X%*%Mu.q.b0-Z1%*%Mu.q.b1-Z2%*%Mu.q.b2)</p><p id="p-191">Bq.Sig.be=0.5*(t(ym0)%*%(ym0)+sum(diag((tX%*%X)%*%Sig.q.b0))+sum(diag(I*Sig.q.b1))+sum(diag(Sig.q.b2))+Se)</p><p id="p-192">####Lower bound of the log of the marginal likelihood (logp(y;q*)################</p><p id="p-193">log.lik=0.5*log(det(Sig.q.b0))+0.5*(sum(log(DDD1)))+0.5*(sum(log(DDD2)))</p><p id="p-194">-0.5*(vb1+J)*log(Bq.Sig.b1)-0.5*(vb2+I*J)*log(Bq.Sig.b2)-0.5*(ve+n.T)*log(Bq.Sig.be)</p><p id="p-195">current.LL0&lt;-log.lik</p><p id="p-196">relErr&lt;-abs((current.LL0/previous.LL0)-1)</p><p id="p-197">if (itnum&gt;=maxIter) {</p><p id="p-198">converged=TRUE</p><p id="p-199">print(“Warning:maximun number of iterations exceeded.”)</p><p id="p-200">}</p><p id="p-201">if(relErr&lt;tolerance) converged=TRUE</p><p id="p-202">Bq.Sig.be.sv=c(Bq.Sig.be)</p><p id="p-203">Mu.q.b1.sv=Mu.q.b1</p><p id="p-204">Mu.q.b2.sv=Mu.q.b2</p><p id="p-205">Bq.Sig.b1.sv=c(Bq.Sig.b1)</p><p id="p-206">Bq.Sig.b2.sv=c(Bq.Sig.b2)</p><p id="p-207">##############Estimates of variance components##################################</p><p id="p-208">Sig.b1=(Bq.Sig.b1)/(0.5*(vb1+J-2))</p><p id="p-209">Sig.b2=(Bq.Sig.b2)/(0.5*(vb2+I*J-2))</p><p id="p-210">Sig.e=(Bq.Sig.be)/(0.5*(ve+n.T-2))</p><p id="p-211">cat(’Iteration:’, itnum, ’\n\n’)</p><p id="p-212">cat(’Current LL0: ’,round(current.LL0,3), ’\n’, ’Previous LL0: ’,</p><p id="p-213">round(previous.LL0,3), ’\n’, ’Tolerance: ’, round(relErr,3), ’\n’,</p><p id="p-214">’Converged: ’, converged, ’\n\n’)</p><p id="p-215">}</p><p id="p-216">###########Printing parameter estimates#########################################</p><p id="p-217">Par.estimates=rbind(Mu.q.b0,Sig.b1,Sig.b2,Sig.e)</p><p id="p-218">Par.estimates</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-219"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-03-10"><span class="received-label">Received </span>March 10, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-04-05"><span class="accepted-label">Accepted </span>April 5, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Montesinos-López <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arakawa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taniguchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mikawa</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Variational Bayesian method of estimating variance components.</span> <abbr class="cit-jnl-abbrev">Anim. Sci. J.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">863</span>–<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BSci.%2BJ.%26rft.volume%253D87%26rft.spage%253D863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Attias</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A variational Bayesian framework for graphical models.</span> <abbr class="cit-jnl-abbrev">Adv. Neural Inf. Process. Syst.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">1–2</span>): <span class="cit-fpage">209</span>–<span class="cit-lpage">215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BNeural%2BInf.%2BProcess.%2BSyst.%26rft.volume%253D12%26rft.spage%253D209%26rft.atitle%253DADVANCES%2BIN%2BNEURAL%2BINFORMATION%2BPROCESSING%2BSYSTEMS%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.6.1833.3"><div class="cit-metadata unstructured">Bishop, C. M., 1999 Variational principal components, pp. 509–514 in <em>Proceedings Ninth International Conference on Artificial Neural Networks</em>, <em>ICANN’99</em>, Institute of Electrical and Electronics Engineers, New York.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-web" id="cit-7.6.1833.4"><div class="cit-metadata unstructured">Blei, D. M., A. Kucukelbir, and J. D. McAuliffe, 2016 Variational inference: a review for statisticians. arXiv. Available at:: <a href="https://arxiv.org/abs/1601.00670">https://arxiv.org/abs/1601.00670</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.5" data-doi="10.1214/12-BA703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carbonetto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Scalable variational inference for Bayesian variable selection in regression, and its accuracy in genetic association studies.</span> <abbr class="cit-jnl-abbrev">Bayesian Anal.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">73</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBayesian%2BAnal.%26rft.volume%253D7%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1214%252F12-BA703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/12-BA703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000303602600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.6" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in maize breeding populations with genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">G3 Bethesda</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1903</span>–<span class="cit-lpage">1926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BBethesda%26rft.volume%253D3%26rft.spage%253D1903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-Lopez</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Rodriguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Bayesian genomic prediction with genotype × environment interaction Kernel models.</span> <abbr class="cit-jnl-abbrev">G3 Bethesda</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BBethesda%26rft.volume%253D7%26rft.spage%253D41%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soberanis</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Elizalde</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic prediction of genotype × environment interaction Kernel regression models.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1833.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghahramani</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beal</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Graphical models and variational methods.</span> In <span class="cit-source">Advanced Mean Field Method—Theory and Practice</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Saad</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Opper</span>  <span class="cit-name-given-names">M.</span></span>. <span class="cit-publ-name">MIT Press</span>, <span class="cit-publ-loc">Cambridge, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.11" data-doi="10.1186/1471-2105-14-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A Bayesian method and its variational approximation for prediction of genomic breeding values in multiple traits.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D14%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-1%26rft_id%253Dinfo%253Apmid%252F23323762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23323762&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1833.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.12" data-doi="10.1214/13-BA815"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wand</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Simple marginally noninformative prior distributions for covariance matrices.</span> <abbr class="cit-jnl-abbrev">Bayesian Anal.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">439</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBayesian%2BAnal.%26rft.volume%253D8%26rft.spage%253D439%26rft_id%253Dinfo%253Adoi%252F10.1214%252F13-BA815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/13-BA815&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.13" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du Cheyron</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1833.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.14" data-doi="10.1534/genetics.111.134866"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimation of quantitative trait locus effects with epistasis by variational Bayes algorithms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.134866%26rft_id%253Dinfo%253Apmid%252F22042575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzEvMjMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.15" data-doi="10.1534/genetics.113.152736"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sillanpää</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A Bayesian nonparametric approach for mapping dynamic quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">997</span>–<span class="cit-lpage">1016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152736%26rft_id%253Dinfo%253Apmid%252F23770698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzQvOTk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy82LzE4MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.16" data-doi="10.1186/1471-2105-11-58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Logsdon</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mezey</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A variational Bayes algorithm for fast and accurate multiple locus genome-wide association analysis.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLogsdon%26rft.auinit1%253DB.%2BA.%26rft.volume%253D11%26rft.spage%253D58%26rft.epage%253D58%26rft.atitle%253DA%2Bvariational%2BBayes%2Balgorithm%2Bfor%2Bfast%2Band%2Baccurate%2Bmultiple%2Blocus%2Bgenome-wide%2Bassociation%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-58%26rft_id%253Dinfo%253Apmid%252F20105321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-58&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105321&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1833.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Increased prediction accuracy in wheat breeding trials using a marker × environment interaction genomic selection model.</span> <abbr class="cit-jnl-abbrev">G3 Bethesda</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">569</span>–<span class="cit-lpage">582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BBethesda%26rft.volume%253D5%26rft.spage%253D569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1833.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacKay</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-source"><em>Developments in Probabilistic Modelling with Neural Networks – Ensemble Learning. Neural Networks: Artificial Intelligence and Industrial Applications</em></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">London</span>. pp. <span class="cit-fpage">14</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O.A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toledo</span>  <span class="cit-name-given-names">F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Hernández</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A genomic Bayesian multi-trait and multi-environment model.</span> <abbr class="cit-jnl-abbrev">G3 Bethesda</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">2725</span>–<span class="cit-lpage">2744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BBethesda%26rft.volume%253D6%26rft.spage%253D2725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.20" data-doi="10.1198/tast.2010.09058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ormerod</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wand</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Explaining variational approximations.</span> <abbr class="cit-jnl-abbrev">Am. Stat.</abbr> <span class="cit-vol">64</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">140</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BStat.%26rft.volume%253D64%26rft.spage%253D140%26rft_id%253Dinfo%253Adoi%252F10.1198%252Ftast.2010.09058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/tast.2010.09058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000278250500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ormerod</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wand</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gaussian variational approximate inference for generalized linear mixed models.</span> <abbr class="cit-jnl-abbrev">J. Comput. Graph. Stat.</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">2</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BGraph.%2BStat.%26rft.volume%253D21%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-web" id="cit-7.6.1833.22"><div class="cit-metadata unstructured">R Core Team, 2016 <em>R: A Language and Environment for Statistical Computing</em>. R Foundation for Statistical Computing, Vienna, Austria. Available at: <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-7.6.1833.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">C. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source"><em>Monte Carlo Methods</em></span>. <span class="cit-publ-name">John Wiley &amp; Sons, Ltd.</span>, <span class="cit-publ-loc">NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Autrique</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Predictor traits from high-throughput phenotyping improve accuracy of pedigree and genomic selection for yield in wheat.</span> <abbr class="cit-jnl-abbrev">G3 Bethesda</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">2799</span>–<span class="cit-lpage">2808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2BBethesda%26rft.volume%253D6%26rft.spage%253D2799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.25" data-doi="10.1093/bioinformatics/bti466"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teschendorff</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbosa-Morais</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenton</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caldas</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A variational Bayesian mixture modelling framework for cluster analysis of gene-expression data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">13</span>): <span class="cit-fpage">3025</span>–<span class="cit-lpage">3033</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti466%26rft_id%253Dinfo%253Apmid%252F15860564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTMvMzAyNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvNi8xODMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.26" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F6%2F1833.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.6.1833.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsunaga</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Onogi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohyama</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyatake</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Efficiency of genomic selection for breeding population design and phenotype prediction in tomato.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">118</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">202</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D118%26rft.spage%253D202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/6/1833.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/6/1819" title="Genome-Wide Transcription and Functional Analyses Reveal Heterogeneous Molecular Mechanisms Driving Pyrethroids Resistance in the Major Malaria Vector Anopheles funestus Across Africa" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/6/1855" title="Genome-Enabled Prediction of Breeding Values for Feedlot Average Daily Weight Gain in Nelore Cattle" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/6"><span>Volume 7 Issue 6, June 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/6.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/6.cover-source.jpg?itok=5_J-OKHc" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (6)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/6/1833" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MYl3RGPLOjef5KkNWxk3cxpAAgyym3qCnyBXHNxDHZI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/6/1833" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/6/1833" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/6/1833" class="active" data-icon-position="" data-hide-link-title="0">A Variational Bayes Genomic-Enabled Prediction Model with Genotype × Environment Interaction</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/484769" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-sPdwA1VhnHI-m--OGfjkRN7-sCK_RS_VuQdc2HgyPVM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1833.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/6/1833" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ztvaHbrOdxI1Iqd2IsjmA8UvTHTMkIBrmlR2U46CIzg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/6/1833" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/6/1833.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484769" id="node484769--2" data-pisa="ggg;7/6/1833" data-pisa-master="ggg;g3.117.041202" data-apath="/ggg/7/6/1833.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Variational Bayes Genomic-Enabled Prediction Model with Genotype × Environment Interaction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">José Crossa</span>, <span class="highwire-citation-author" data-delta="3">José Cricelio Montesinos-López</span>, <span class="highwire-citation-author" data-delta="4">Francisco Javier Luna-Vázquez</span>, <span class="highwire-citation-author" data-delta="5">Josafhat Salinas-Ruiz</span>, <span class="highwire-citation-author" data-delta="6">José R. Herrera-Morales</span> and <span class="highwire-citation-author" data-delta="7">Raymundo Buenrostro-Mariscal</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1833-1853;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041202 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484769--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">José Cricelio Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), 36240 Guanajuato, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCricelio%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BCricelio%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Francisco Javier Luna-Vázquez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BJavier%2BLuna-V%C3%A1zquez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luna-V%C3%A1zquez%20FJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BJavier%2BLuna-V%25C3%25A1zquez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Josafhat Salinas-Ruiz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Colegio de Postgraduados, Campus Córdoba, Km. 348 Carretera Federal Córdoba-Veracruz, Amatlán de los Reyes, 94946, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josafhat%2BSalinas-Ruiz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salinas-Ruiz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJosafhat%2BSalinas-Ruiz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">José R. Herrera-Morales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BR.%2BHerrera-Morales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Herrera-Morales%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BR.%2BHerrera-Morales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Raymundo Buenrostro-Mariscal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raymundo%2BBuenrostro-Mariscal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buenrostro-Mariscal%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaymundo%2BBuenrostro-Mariscal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/6/1833" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/6/1833&amp;title=A%20Variational%20Bayes%20Genomic-Enabled%20Prediction%20Model%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/6/1833&amp;title=A%20Variational%20Bayes%20Genomic-Enabled%20Prediction%20Model%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/6/1833&amp;title=A%20Variational%20Bayes%20Genomic-Enabled%20Prediction%20Model%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/6/1833&amp;text=A%20Variational%20Bayes%20Genomic-Enabled%20Prediction%20Model%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/6/1833&amp;title=A%20Variational%20Bayes%20Genomic-Enabled%20Prediction%20Model%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/6/1833&amp;t=A%20Variational%20Bayes%20Genomic-Enabled%20Prediction%20Model%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/6/1833&amp;title=A%20Variational%20Bayes%20Genomic-Enabled%20Prediction%20Model%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/6/1833&amp;title=A%20Variational%20Bayes%20Genomic-Enabled%20Prediction%20Model%20with%20Genotype%20%C3%97%20Environment%20Interaction" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/484769/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="484769" id="node484769--3" data-pisa="ggg;7/6/1833" data-pisa-master="ggg;g3.117.041202" data-apath="/ggg/7/6/1833.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Variational Bayes Genomic-Enabled Prediction Model with Genotype × Environment Interaction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">José Crossa</span>, <span class="highwire-citation-author" data-delta="3">José Cricelio Montesinos-López</span>, <span class="highwire-citation-author" data-delta="4">Francisco Javier Luna-Vázquez</span>, <span class="highwire-citation-author" data-delta="5">Josafhat Salinas-Ruiz</span>, <span class="highwire-citation-author" data-delta="6">José R. Herrera-Morales</span> and <span class="highwire-citation-author" data-delta="7">Raymundo Buenrostro-Mariscal</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">June 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 6 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1833-1853;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041202 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node484769--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430, Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biometrics and Statistics Unit and Global Wheat Program of the International Maize and Wheat Improvement Center (CIMMYT), 06600 México D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">José Cricelio Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), 36240 Guanajuato, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCricelio%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BCricelio%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Francisco Javier Luna-Vázquez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BJavier%2BLuna-V%C3%A1zquez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Luna-V%C3%A1zquez%20FJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BJavier%2BLuna-V%25C3%25A1zquez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Josafhat Salinas-Ruiz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Colegio de Postgraduados, Campus Córdoba, Km. 348 Carretera Federal Córdoba-Veracruz, Amatlán de los Reyes, 94946, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josafhat%2BSalinas-Ruiz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salinas-Ruiz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJosafhat%2BSalinas-Ruiz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">José R. Herrera-Morales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BR.%2BHerrera-Morales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Herrera-Morales%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BR.%2BHerrera-Morales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Raymundo Buenrostro-Mariscal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raymundo%2BBuenrostro-Mariscal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buenrostro-Mariscal%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaymundo%2BBuenrostro-Mariscal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1833&amp;rft.epage=1853&amp;rft.atitle=A%20Variational%20Bayes%20Genomic-Enabled%20Prediction%20Model%20with%20Genotype%20%C3%97%20Environment%20Interaction&amp;rft.volume=7&amp;rft.issue=6&amp;rft.date=2017-06-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval+A.&amp;rft.au=Montesinos-L%C3%B3pez%2C+Abelardo&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Montesinos-L%C3%B3pez%2C+Jos%C3%A9+Cricelio&amp;rft.au=Luna-V%C3%A1zquez%2C+Francisco+Javier&amp;rft.au=Salinas-Ruiz%2C+Josafhat&amp;rft.au=Herrera-Morales%2C+Jos%C3%A9+R.&amp;rft.au=Buenrostro-Mariscal%2C+Raymundo"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/484769/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/484769/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/484769/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/484769/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/484769/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/484769/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/484769/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/484769/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/484769/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/484769/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/484769/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/484769/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/6/1833" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix B</a></li><li class="odd"><a href="#app-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix C</a></li><li class="even"><a href="#app-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix D</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/6/1833.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/6/1833.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/7/6/1833.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__CK8qNg7zHnCPfipcSU0qjCyIiw2pe-WKGx2V0TgizGw__wprDSLLH9CuacxJkxX2DuWt0EaauZA-f_LGjovfY3aw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
