<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/5/1201" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/5/1201.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/5/1201.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/5/1201.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/5/1201.atom" />
<meta name="HW.pisa" content="ggg;6/5/1201" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Massive Amplification at an Unselected Locus Accompanies Complex Chromosomal Rearrangements in Yeast" />
<meta name="DC.Identifier" content="10.1534/g3.115.024547" />
<meta name="DC.Date" content="2016-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Thierry et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Gene amplification has been observed in different organisms in response to environmental constraints, such as limited nutrients or exposure to a variety of toxic compounds, conferring them with specific phenotypic adaptations via increased expression levels. However, the presence of multiple gene copies in natural genomes has generally not been found in the absence of specific functional selection. Here, we show that the massive amplification of a chromosomal locus (up to 880 copies per cell) occurs in the absence of any direct selection, and is associated with low-order amplifications of flanking segments in complex chromosomal alterations. These results were obtained from mutants with restored phenotypes that spontaneously appeared from genetically engineered strains of the yeast Saccharomyces cerevisiae suffering from severe fitness reduction. Grossly extended chromosomes (macrotene) were formed, with complex structural alterations but sufficient stability to propagate unchanged over successive generations. Their detailed molecular analysis, including complete genome sequencing, identification of sequence breakpoints, and comparisons between mutants, revealed novel mechanisms causing their formation, whose combined action underlies the astonishing dynamics of eukaryotic chromosomes and their consequences." />
<meta name="DC.Contributor" content="Agnès Thierry" />
<meta name="DC.Contributor" content="Varun Khanna" />
<meta name="DC.Contributor" content="Bernard Dujon" />
<meta name="article:published_time" content="2016-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Massive Amplification at an Unselected Locus Accompanies Complex Chromosomal Rearrangements in Yeast" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Gene amplification has been observed in different organisms in response to environmental constraints, such as limited nutrients or exposure to a variety of toxic compounds, conferring them with specific phenotypic adaptations via increased expression levels. However, the presence of multiple gene copies in natural genomes has generally not been found in the absence of specific functional selection. Here, we show that the massive amplification of a chromosomal locus (up to 880 copies per cell) occurs in the absence of any direct selection, and is associated with low-order amplifications of flanking segments in complex chromosomal alterations. These results were obtained from mutants with restored phenotypes that spontaneously appeared from genetically engineered strains of the yeast &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; suffering from severe fitness reduction. Grossly extended chromosomes (macrotene) were formed, with complex structural alterations but sufficient stability to propagate unchanged over successive generations. Their detailed molecular analysis, including complete genome sequencing, identification of sequence breakpoints, and comparisons between mutants, revealed novel mechanisms causing their formation, whose combined action underlies the astonishing dynamics of eukaryotic chromosomes and their consequences.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/05/01" />
<meta name="citation_mjid" content="ggg;6/5/1201" />
<meta name="citation_id" content="6/5/1201" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/5/1201" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/5/1201.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/5/1201.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/5/1201.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.024547" />
<meta name="citation_pmid" content="26945028" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1201" />
<meta name="citation_lastpage" content="1215" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Agnès Thierry" />
<meta name="citation_author_institution" content="Institut Pasteur, Unité de Génétique Moléculaire des Levures, CNRS (UMR3525), Sorbonne Universités, UPMC, Université Paris 06 (UFR927), F-75724 CEDEX 15, France" />
<meta name="citation_author" content="Varun Khanna" />
<meta name="citation_author_institution" content="Institut Pasteur, Unité de Génétique Moléculaire des Levures, CNRS (UMR3525), Sorbonne Universités, UPMC, Université Paris 06 (UFR927), F-75724 CEDEX 15, France" />
<meta name="citation_author" content="Bernard Dujon" />
<meta name="citation_author_institution" content="Institut Pasteur, Unité de Génétique Moléculaire des Levures, CNRS (UMR3525), Sorbonne Universités, UPMC, Université Paris 06 (UFR927), F-75724 CEDEX 15, France" />
<meta name="citation_author_email" content="bdujon@pasteur.fr" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_author=G. M. Adamo;citation_author=M. Lotti;citation_author=M. J. Tamas;citation_author=S. Brocca;citation_title=Amplification of the CUP1 gene is associated with evolution of copper tolerance in Saccharomyces cerevisiae.;citation_volume=158;citation_year=2012;citation_pmid=22790396;citation_doi=10.1099/mic.0.058024-0" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=F. W. Alt;citation_author=R. E. Kellems;citation_author=J. R. Bertino;citation_author=R. T. Schimke;citation_title=Selective multiplication of dihydrofolate reductase genes in methotrexate-resistant variants of cultured murine cells;citation_pages=1357-1370;citation_volume=253;citation_year=1978;citation_issue=5;citation_pmid=627542" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. L. Araya;citation_author=C. Payen;citation_author=M. J. Dunham;citation_author=S. Fields;citation_title=Whole-genome sequencing of a laboratory-evolved yeast strain.;citation_pages=88-88;citation_volume=11;citation_year=2010;citation_pmid=20128923;citation_doi=10.1186/1471-2164-11-88" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=J. E. Barrick;citation_author=R. E. Lesnki;citation_title=Genome dynamics during experimental evolution.;citation_pages=827-839;citation_volume=14;citation_year=2013;citation_pmid=24166031;citation_doi=10.1038/nrg3564" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: A flexible trimmer for Illumina Sequence Data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. J. Brewer;citation_author=C. Payen;citation_author=S. C. DiRienzi;citation_author=M. M. Higgins;citation_author=G. Ong;citation_title=Origin-dependent inverted-repeat amplification: a replication-based model for generating palindromic amplicons.;citation_pages=e1002016-e1002016;citation_volume=7;citation_year=2011;citation_issue=3;citation_pmid=21437266;citation_doi=10.1371/journal.pgen.1002016" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=C. J. Brown;citation_author=K. M. Todd;citation_author=R. F. Rosenzweig;citation_title=Multiple duplications of yeast hexose transport genes in response to selection in a glucose-limited environment;citation_pages=931-942;citation_volume=15;citation_year=1998;citation_issue=8;citation_pmid=9718721;citation_doi=10.1093/oxfordjournals.molbev.a026009" />
<meta name="citation_reference" content="citation_journal_title=Soc. Biol.;citation_author=M. K. Burke;citation_title=How does adaptation sweep through the genome? Insights from long-term selection experiments. Proc. R.;citation_pages=5029-5038;citation_volume=279;citation_year=2012;citation_doi=10.1098/rspb.2012.0799" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. P. Byrne;citation_author=K. H. Wolfe;citation_title=The yeast gene order browser: combining curated homology and syntenic context reveals gene fate in polyploidy species.;citation_volume=15;citation_year=2005;citation_pmid=16169922;citation_doi=10.1101/gr.3672305" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S.-L. Chang;citation_author=H.-Y. Lai;citation_author=S.-Y. Tung;citation_author=J.-Y. Leu;citation_title=Dynamic large-scale chromosomal rearrangements fuel rapid adaptation in yeast populations.;citation_pages=e1003232;citation_volume=9;citation_year=2013;citation_issue=1;citation_pmid=23358723;citation_doi=10.1371/journal.pgen.1003232" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. Despons;citation_author=P. V. Baret;citation_author=L. Frangeul;citation_author=V. Leh-Louis;citation_author=P. Durrens;citation_title=Genome-wide computational prediction of tandem gene arrays: application in yeasts.;citation_pages=56-56;citation_volume=11;citation_year=2010;citation_pmid=20092627;citation_doi=10.1186/1471-2164-11-56" />
<meta name="citation_reference" content="citation_journal_title=Comptes rendus biologies;citation_journal_abbrev=C R Biol;citation_author=L. Despons;citation_author=Z. Uzunov;citation_author=V. L. Louis;citation_title=Tandem gene arrays, plastic chromosomal organizations.;citation_pages=639-646;citation_volume=334;citation_year=2011;citation_issue=8-9;citation_pmid=21819945;citation_doi=10.1016/j.crvi.2011.05.012" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. R. Dettman;citation_author=N. Rodrigue;citation_author=A. H. Melnyk;citation_author=A. Wong;citation_author=S. F. Bailey;citation_title=Evolutionary insight from whole-genome sequencing of experimentally evolved microbes.;citation_pages=2058-2077;citation_volume=21;citation_year=2012;citation_issue=9;citation_pmid=22332770;citation_doi=10.1111/j.1365-294X.2012.05484.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=J. de Visser;citation_author=G. M. Arjan;citation_author=J. Krug;citation_title=Empirical fitness landscapes and the predictability of evolution.;citation_pages=480-490;citation_volume=15;citation_year=2014;citation_pmid=24913663;citation_doi=10.1038/nrg3744" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. Dujon;citation_author=D. Sherman;citation_author=G. Fischer;citation_author=P. Durrens;citation_author=S. Casaregola;citation_title=Genome evolution in yeasts.;citation_pages=35-44;citation_volume=430;citation_year=2004;citation_issue=6995;citation_pmid=15229592;citation_doi=10.1038/nature02579" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. J. Dunham;citation_author=H. Badrane;citation_author=T. Ferea;citation_author=J. Adams;citation_author=P. O. Brown;citation_title=Characteristic genome rearrangements in experimental evolution of Saccharomyces cerevisiae.;citation_volume=99;citation_year=2002;citation_pmid=12446845;citation_doi=10.1073/pnas.242624799" />
<meta name="citation_reference" content="citation_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=N. El-Mabrouk;citation_author=D. Sankoff;citation_title=Analysis of gene order evolution beyond single-copy genes.;citation_pages=397-429;citation_volume=855;citation_year=2012;citation_pmid=22407718;citation_doi=10.1007/978-1-61779-582-4_15" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Fogel;citation_author=J. W. Welch;citation_title=Tandem gene amplification mediates copper resistance in yeast.;citation_pages=5342-5346;citation_volume=79;citation_year=1982;citation_issue=17;citation_pmid=6291039;citation_doi=10.1073/pnas.79.17.5342" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. A. Gaines;citation_author=W. Zhang;citation_author=D. Wang;citation_author=B. Bukun;citation_author=S. T. Chisholm;citation_title=Gene amplification confers glyphosate resistance in Amaranthus palmeri.;citation_volume=107;citation_year=2010;citation_pmid=20018685;citation_doi=10.1073/pnas.0906649107" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=D. Gresham;citation_author=M. M. Desai;citation_author=C. M. Tucker;citation_author=H. T. Jenq;citation_author=D. A. Pai;citation_title=The repertoire and dynamics of evolutionary adaptations to controlled nutrient-limited environments in yeast.;citation_pages=e1000303-e1000303;citation_volume=4;citation_year=2008;citation_issue=12;citation_pmid=19079573;citation_doi=10.1371/journal.pgen.1000303" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sc.;citation_author=D. Gresham;citation_author=R. Usaite;citation_author=S. M. Germann;citation_author=M. Lisby;citation_author=D. Botstein;citation_title=Adaptation to diverse nitrogen-limited environments by deletion or extrachromosomal element formation of the GAP1 locus.;citation_volume=107;citation_year=2010;citation_pmid=20937885;citation_doi=10.1073/pnas.1014023107" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=J. Hose;citation_author=C. M. Yong;citation_author=M. Sardi;citation_author=Z. Wang;citation_author=M. A. Newton;citation_title=Dosage compensation can buffer copy-number variation in wild yeast.;citation_pages=05462;citation_volume=4;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Karin;citation_author=R. Najarian;citation_author=A. Haslinger;citation_author=P. Valenzoela;citation_author=J. Welch;citation_title=Primary structure and transcription of an amplified genetic locus: the CUP1 locus of yeast.;citation_pages=337-341;citation_volume=81;citation_year=1984;citation_issue=2;citation_pmid=6364141;citation_doi=10.1073/pnas.81.2.337" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Cell Biol.;citation_author=W. P. Kloosterman;citation_author=E. Cuppen;citation_title=Chromothripsis in congenital disorders and cancer: similarities and differences.;citation_pages=341-348;citation_volume=25;citation_year=2013;citation_pmid=23478216;citation_doi=10.1016/j.ceb.2013.02.008" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. P. Kloosterman;citation_author=L. C. Francioli;citation_author=F. Hormozdiani;citation_author=T. Marchall;citation_author=J. Y. Hehir-Kwa;citation_title=Characteristics of de novo structural changes in the human genome.;citation_volume=25;citation_year=2015;citation_pmid=25883321;citation_doi=10.1101/gr.185041.114" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=J. O. Korbel;citation_author=P. J. Campbell;citation_title=Criteria for inference of chromothripsis in cancer genomes.;citation_pages=1226-1236;citation_volume=152;citation_year=2013;citation_pmid=23498933;citation_doi=10.1016/j.cell.2013.02.023" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=R. Koszul;citation_author=S. Caburet;citation_author=B. Dujon;citation_author=G. Fischer;citation_title=Eucaryotic genome evolution through the spontaneous duplication of large chromosomal segments;citation_pages=234-243;citation_volume=23;citation_year=2004;citation_issue=1;citation_pmid=14685272;citation_doi=10.1038/sj.emboj.7600024" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Koszul;citation_author=B. Dujon;citation_author=G. Fischer;citation_title=Stability of large segmental duplications in the yeast genome.;citation_pages=2211-2222;citation_volume=175;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=V. Leh-Louis;citation_author=L. Despons;citation_author=A. Friedrich;citation_author=T. Martin;citation_author=P. Durrens;citation_title=Pichia sorbitophila, an Interspecies Yeast Hybrid, Reveals Early Steps of Genome Resolution After Polyploidization;citation_pages=299-311;citation_volume=2;citation_year=2012;citation_issue=2;citation_pmid=22384408;citation_doi=10.1534/g3.111.000745" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_pages=589-595;citation_volume=25;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence alignment/map (SAM) format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. E. Libuda;citation_author=F. Winston;citation_title=Amplification of histone genes by circular chromosome formation in Saccharomyces cerevisiae.;citation_pages=1003-1007;citation_volume=443;citation_year=2006;citation_issue=7114;citation_pmid=17066037;citation_doi=10.1038/nature05205" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=R. Martinez;citation_author=M. T. Latreille;citation_author=M. Miranda;citation_title=A PMR2 tandem repeat with a modified C-terminus is located downstream from the KRS1 gene encoding lysyl-tRNA synthetase in Saccharomyces cerevisiae.;citation_pages=149-154;citation_volume=227;citation_year=1991;citation_issue=1;citation_pmid=2046655;citation_doi=10.1007/BF00260720" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=I. Milne;citation_author=G. Stephen;citation_author=M. Bayer;citation_author=P. J. Cock;citation_author=L. Pritchard;citation_title=Using Tablet for visual exploration of second-generation sequencing data.;citation_volume=14;citation_year=2013;citation_issue=2;citation_pmid=22445902;citation_doi=10.1093/bib/bbs012" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=L. Morales;citation_author=B. Dujon;citation_title=Evolutionary Role of Interspecies Hybridization and Genetic Exchanges in Yeasts;citation_pages=721-739;citation_volume=76;citation_year=2012;citation_issue=4;citation_pmid=23204364;citation_doi=10.1128/MMBR.00022-12" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=H. Muller;citation_author=A. Thierry;citation_author=J. Y. Coppee;citation_author=C. Gouyette;citation_author=C. Hennequin;citation_title=Genomic polymorphism in the population of Candida glabrata: gene copy-number variation and chromosomal translocations.;citation_pages=264-276;citation_volume=46;citation_year=2009;citation_issue=3;citation_pmid=19084610;citation_doi=10.1016/j.fgb.2008.11.006" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. A. Nieduszynski;citation_author=S.-I. Hiraga;citation_author=P. Ak;citation_author=C. J. Benham;citation_author=A. D. Donaldson;citation_title=OriDB: a replication origin database.;citation_volume=35;citation_year=2007;citation_pmid=17065467;citation_doi=10.1093/nar/gkl758" />
<meta name="citation_reference" content="citation_journal_title=J. Comput. Biol.;citation_author=S. Nurk;citation_author=A. Bankerich;citation_author=D. Antipov;citation_author=A. A. Gurevich;citation_author=A. Korobeynikov;citation_title=Assembling single-cell genomes and mini-metagenomes from chimeric MDA products.;citation_pages=714-737;citation_volume=20;citation_year=2013;citation_pmid=24093227;citation_doi=10.1089/cmb.2013.0084" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Payen;citation_author=R. Koszul;citation_author=B. Dujon;citation_author=G. Fischer;citation_title=Segmental duplications arise from Pol32-dependent repair of broken forks through two alternative replication-based mechanisms.;citation_pages=e1000175-e1000175;citation_volume=4;citation_year=2008;citation_issue=9;citation_pmid=18773114;citation_doi=10.1371/journal.pgen.1000175" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. Payen;citation_author=S. C. DiRienzi;citation_author=G. T. Ong;citation_author=J. L. Pogavhar;citation_author=J. C. Sanchez;citation_title=The dynamics of diverse segmental amplifications in populations of Saccharomyces cerevisiae adapting to strong selection.;citation_volume=4;citation_year=2014;citation_pmid=24368781;citation_doi=10.1534/g3.113.009365" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. Rancati;citation_author=N. Pavelka;citation_author=B. Fleharty;citation_author=N. Noll;citation_author=R. Trimble;citation_title=Aneuploidy underlies rapid adaptive evolution of yeast cells deprived of a conserved cytokinesis motor.;citation_pages=879-893;citation_volume=135;citation_year=2008;citation_issue=5;citation_pmid=19041751;citation_doi=10.1016/j.cell.2008.09.039" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdottir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. C. Schwartz;citation_author=C. R. Cantor;citation_title=Separation of yeast chromosome-sized DNAs by pulsed field gradient gel electrophoresis.;citation_pages=67-75;citation_volume=37;citation_year=1984;citation_issue=1;citation_pmid=6373014;citation_doi=10.1016/0092-8674(84)90301-5" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. J. Stephens;citation_author=C. D. Greenman;citation_author=B. Fu;citation_author=F. Yang;citation_author=G. R. Bignell;citation_title=Massive genomic rearrangement acquired in a single catastrophic event during cancer development.;citation_pages=27-40;citation_volume=144;citation_year=2011;citation_issue=1;citation_pmid=21215367;citation_doi=10.1016/j.cell.2010.11.055" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. H. Sturtevant;citation_title=A Case of Rearrangement of Genes in Drosophila;citation_pages=235-237;citation_volume=7;citation_year=1921;citation_issue=8;citation_pmid=16576597;citation_doi=10.1073/pnas.7.8.235" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=A. Thierry;citation_author=V. Khanna;citation_author=S. Créno;citation_author=I. Lafontaine;citation_author=L. Ma;citation_title=Macrotene chromosomes provide insights to a new mechanism of high-order gene amplification in eukaryotes.;citation_pages=6154;citation_volume=6;citation_year=2015;citation_pmid=25635677;citation_doi=10.1038/ncomms7154" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. M. Torres;citation_author=T. Sokolsky;citation_author=C. M. Tucker;citation_author=L. Y. Chan;citation_author=M. Boselli;citation_title=Effects of Aneuploidy on Cellular Physiology and Cell Division in Haploid Yeast;citation_pages=916-924;citation_volume=317;citation_year=2007;citation_issue=5840;citation_pmid=17702937;citation_doi=10.1126/science.1142210" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=E. M. Torres;citation_author=N. Delphoure;citation_author=A. Panneerselvam;citation_author=C. M. Tucker;citation_author=C. A. Whittaker;citation_title=Identification of aneuploidy-tolerating mutations.;citation_pages=71-83;citation_volume=143;citation_year=2010;citation_issue=1;citation_pmid=20850176;citation_doi=10.1016/j.cell.2010.08.038" />
<meta name="citation_reference" content="citation_journal_title=Planta;citation_author=M. M. Vila-Aiub;citation_author=S. S. Goh;citation_author=T. A. Gaines;citation_author=H. Han;citation_author=R. Busi;citation_title=No fitness cost of glyphosate resistance endowed by massive EPSPS gene amplification in Amaranthus palmeri.;citation_pages=793-801;citation_volume=239;citation_year=2014;citation_pmid=24385093;citation_doi=10.1007/s00425-013-2022-x" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. W. Welch;citation_author=S. Fogel;citation_author=G. Cathala;citation_author=M. Karin;citation_title=Industrial yeasts display tandem gene iteration at the CUP1 region.;citation_pages=1353-1361;citation_volume=3;citation_year=1983;citation_issue=8;citation_pmid=6621529;citation_doi=10.1128/MCB.3.8.1353" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=J. Wieland;citation_author=A. M. Nitsche;citation_author=J. Strayle;citation_author=H. Steiner;citation_author=H. K. Rudolph;citation_title=The PMR2 gene cluster encodes functionally distinct isoforms of a putative Na+ pump in the yeast plasma membrane.;citation_pages=3870-3882;citation_volume=14;citation_year=1995;citation_pmid=7664728" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=D. R. Winge;citation_author=K. B. Nielson;citation_author=W. R. Gray;citation_author=D. H. Hamer;citation_title=Yeast metallothionein. Sequence and metal-binding properties;citation_pages=14464-14470;citation_volume=260;citation_year=1985;citation_issue=27;citation_pmid=3902832" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=M. Zarrei;citation_author=J. R. McDonald;citation_author=D. Merico;citation_author=S. W. Scherer;citation_title=A copy number variation map of the human genome.;citation_pages=172-183;citation_volume=16;citation_year=2015;citation_pmid=25645873;citation_doi=10.1038/nrg3871" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=C. Z. Zhang;citation_author=M. L. Leibowitz;citation_author=D. Pellman;citation_title=Chromothripsis and beyond: rapid genome evolution from complex chromosomal rearrangements;citation_pages=2513-2530;citation_volume=27;citation_year=2013;citation_issue=23;citation_pmid=24298051;citation_doi=10.1101/gad.229559.113" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Y. Zhao;citation_author=P. K. Strope;citation_author=S. G. Kozmon;citation_author=J. H. McCusker;citation_author=F. S. Dietrich;citation_title=Structures of naturally evolved CUP1 tandem arrays in yeast indicate that these arrays are generated by unequal nonhomologous recombination.;citation_volume=4;citation_year=2014;citation_pmid=25236733;citation_doi=10.1534/g3.114.012922" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Massive Amplification at an Unselected Locus Accompanies Complex Chromosomal Rearrangements in Yeast" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
<meta name="twitter:description" content="Gene amplification has been observed in different organisms in response to environmental constraints, such as limited nutrients or exposure to a variety of toxic compounds, conferring them with specific phenotypic adaptations via increased expression levels. However, the presence of multiple gene copies in natural genomes has generally not been found in the absence of specific functional selection. Here, we show that the massive amplification of a chromosomal locus (up to 880 copies per cell) occurs in the absence of any direct selection, and is associated with low-order amplifications of flanking segments in complex chromosomal alterations. These results were obtained from mutants with restored phenotypes that spontaneously appeared from genetically engineered strains of the yeast Saccharomyces cerevisiae suffering from severe fitness reduction. Grossly extended chromosomes (macrotene) were formed, with complex structural alterations but sufficient stability to propagate unchanged over successive generations. Their detailed molecular analysis, including complete genome sequencing, identification of sequence breakpoints, and comparisons between mutants, revealed novel mechanisms causing their formation, whose combined action underlies the astonishing dynamics of eukaryotic chromosomes and their consequences." />
<meta name="og-title" property="og:title" content="Massive Amplification at an Unselected Locus Accompanies Complex Chromosomal Rearrangements in Yeast" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/5/1201" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Gene amplification has been observed in different organisms in response to environmental constraints, such as limited nutrients or exposure to a variety of toxic compounds, conferring them with specific phenotypic adaptations via increased expression levels. However, the presence of multiple gene copies in natural genomes has generally not been found in the absence of specific functional selection. Here, we show that the massive amplification of a chromosomal locus (up to 880 copies per cell) occurs in the absence of any direct selection, and is associated with low-order amplifications of flanking segments in complex chromosomal alterations. These results were obtained from mutants with restored phenotypes that spontaneously appeared from genetically engineered strains of the yeast Saccharomyces cerevisiae suffering from severe fitness reduction. Grossly extended chromosomes (macrotene) were formed, with complex structural alterations but sufficient stability to propagate unchanged over successive generations. Their detailed molecular analysis, including complete genome sequencing, identification of sequence breakpoints, and comparisons between mutants, revealed novel mechanisms causing their formation, whose combined action underlies the astonishing dynamics of eukaryotic chromosomes and their consequences." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
    <title>Massive Amplification at an Unselected Locus Accompanies Complex Chromosomal Rearrangements in Yeast | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"o6VDd5c2M5cRHzB3GEf5RtDrX9D5o1jh6bFAAzXmv0w","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460353","apath":"\/ggg\/6\/5\/1201.atom","pisa":"ggg;6\/5\/1201","ac":{"\/ggg\/6\/5\/1201.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/5\/1201.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/5\/1201"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/5\/1201"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/5\/1201.abstract?sid=d73770b8-eb31-44b4-89d5-c21b8c0c162e":true,"\/content\/6\/5\/1201":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bLfNevXKMuHyECw5wehdCTWTcuSg120oKTXPknABIkJh6cThhtCtV\\\\\\\/JJgVT8+ZcJhbu+3ApxsXuyhj+1WAZtNyKlhioxpu3wyi4IO5BWZXfB4Va0z1LEGw3h8\\\\\\\/Khj5doxCO3hSQN4jf6IpOoJcoT2s1nluJQTuJu5YfQqz9AWdpge1YDv+z6kxQgdcp2q6yC1QihRxdje4RY0QP5m7AmECl8ThjbaFdnSVOSBisEqMPDl4q70fOvigaoMsUS8zwW4och5Ck50VLyXTO4EpKMUOWs1VTOZazv\\\\\\\/NuMXxXtjOK5Ejxl5JgH2vp6sbSJfGUSe88p3eeuxsTtCZVKoILlYM2N7huWjZSIAFkJxHioND9dueZ5Dh1aDB34vK4CvfK1ZGMUMJI2AVtCHBEfd3TJJmuSSPK4GR+lucJdoY7c06YdT5TjDOsbvrvXnqKrfb7D+i6DXh06hrV2E19RV1FT\\\\\\\/zNLlo0S5Wv\\\\\\\/pdpAagaNlmfoalU4NaRTXPpqCNG9SKe38JkM4xEQFlSMDXt9nf\\\\\\\/bRlVkmgxUx6gaapS7qM87ImVUF56LW10njIaOGIWIVUSuhLGv0NcM+YhSvB54GWXxGVLaivcK4Hr6PodJcrk1HvGpN6uDdbfH0iYhWBqDifT0ZNT4d6PC3vndMV8qrjqIGpJ8PK4ksqWiKz7hfkEIjvWGovguWNugC1+VWQ8jw4PkIA0FqpEhcrxd878O7ubCja8p9TiiicQ90EncYyISkTHY+twNTyx1WJbLLz12KL7kTNirzpTzUxDYKKuniQh9ArdcTgDFSJSXbuZz7+NjQ1wtxhhod09CQ0ED1ukVrJ7kycf5UdENsXtxUfVPFmxLR4nKajrmpid8jos5PDBJN1e9SKf2cV+lJtY5\\\\\\\/\\\\\\\/CijXaC2yH2UQkgTD8URAT2CEA8wB8vyZhTeua8M29kof+7bKhAlRx58Hdqubs801OxTML5n2q8jxOgqN47jZ4+zmJG\\\\\\\/2C0v5u\\\\\\\/l9hEElG5CJzD\\\\\\\/W01UUFf1u\\\\\\\/oFgktgXUEyZjYRyIFxfnKvXkEn\\\\\\\/0VGXTfGE4C3tqzVghy8GuvYvzgxGvlU2DcicLvLtB4P9WyLaLIX9jOaAfIY\\\\\\\/hv02N8yyV8csMU3\\\\\\\/WSIFGlg1WC5oFcb6Klh19l1FGdIp31ykJAC1MPbUoFI3yeOBrk8NFnpGMrfWM2QgpdAunh7iLbfVhmYaMBBUljEcLSAFSLRsK4VG0T29yQmMv+jex1jC3GsjcnPmG0EwlAlPHXOeaCbISd9VuE0AOvYBJohnnd8mvzAGeg\\\u0022,\\\u0022iv\\\u0022:\\\u002291NlUogFiSG\\\\\\\/jMZtJ2HpGw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229e9930dc48804ca17688de84b0f7bd960082ca9589347536a71a7a21752d85db\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220+GecJtTUjm45HnhDSrKgvQtxAq\\\\\\\/ZDppv8Ri1YYNhbkYH3PCTP1ygrIbv6qHrbg7vA8PZuviArouhBhLNftXXvwUCgO3oziCHYOqwzVcd+90pn6qeR3KcNeE59Jot\\\\\\\/BnNlxBHPmnoj1fI7iAx16plzjtgDEBvxwSaiJBr2QImoggUlO\\\\\\\/WSf+aowrbM4swSEFb2QH7Gz1AGOj+QStHIuSpONEm\\\\\\\/A8b+HqMqvRU1QFdcKX0YSQBdNxs1HE9\\\\\\\/ED\\\\\\\/MWNMsHDnhjDFAygM7utPwFnLbW8f2EntDLOuHt+nFoCaq388mMjSgqKJl4b3s14m98oSY2j3bbU+MTLzkpKga21ZQ1xC2daN5D6dZeVO10Q70Jlmrmg+LHMq3VHOJUfGuL0XDMtCzmn7W2bra9q0v9DOlPw8RfF4eJgr\\\\\\\/\\\\\\\/rYqgKD5oaG7zsf\\\\\\\/ltoZ+5GRK1XSfOLp7KKlv7HVIk+pvvnmR3uDMvJvV4Ejw\\\\\\\/iVBuAvWkLTFta7m4FKzngXVsuBGVDWosA+wQ5fKEEmjYMs+1xW4PMKKxbPmvWllhoy3qFA4aDRg5rOcSMJuxv8VNnvmep2llkMW4iYbHhDpubkGr4IqmqnoPvuhoreU2+bFwFnrgI+k8SrWCeMNV5wJaOTk3975odxdN2rPVHbcmMCTVudZDXlztuO5EBSQccwfd+y6fj4d4n+JIQ5MzdNH9JIJriNU95B+zddvYnM+i8FLg6ThLXknAhoaVq41dSMVoWuDNd\\\\\\\/X0esFZ1JAL0nh9Xk2vVfDLax8OqnaDIRWmp0AeSoaB29t0EUqZYSerosjrzXRdx5JyPxGcagVQsYXp69RefzlyJ8zcGGPSP75AtPfpaz9kXmn\\\\\\\/ZekjEAju2P7hqZnWNBG\\\\\\\/NlBkcbi\\\\\\\/5vsUbRjPdYY8wVlBl3AtZLfbhhOStATgSD28mTgDFpUTY9Umevat00kX+CU9eJ8uBq4lJOXjVcChVYv8vpC6uiMPlXLzLWIwyYNeTJVthfjOPplks2lq3WDmMeQuWH3MU9cVhTEornAXN6mo7TZfQxWyRnxiBD0Gsix2lzrIgBxKhhMrlINOylahB5N0nkCC1weNW50bc4pJtRFG+dvYaLbVRNq7uxq6GeYNWLbslRZeUQrT6EABXyzEwWJt3UH3Bc\\\\\\\/xNqKBj7XQO2r45s\\\\\\\/oc9DZmHcNhsR3133g8RgwF+iT3YNRsUkGR9x1w7tE18w\\\\\\\/CtJKSqdUeN5eFJ+AGBEeqz4cAKgBk6ayhyu\\\\\\\/Fqhgsvd\\\\\\\/LCVdevjIGxmEQEdArm7ygG1tgPCyBiJZDbLrMocx+2pEixrdi8qrKPhqmIisQ\\\\\\\/slslIb6PWM6W15hFSA3Trb\\\\\\\/4YcFz3WKYj0ZYLo1MnTjIngGW5HiEhAqEj2tMy6btuX7TeByGVU0FxFf6LJOidxDV7FpH2G7lcMcJkGcc0yhC\\\\\\\/LlLUv+Y6TwZ86lYXrR2K08k+ViRIa4bHIj00C2+m02oLfs4U5WeT9kC4Kgl3HXO2bOsiZ5MBNgjTjSQbBbNNSvwk5J\\\\\\\/u1suAD+f3kK68zbHAXqnqydfq4yM2tNWZR4++QNA3qWFDztSV4a539Xa8nsDN5TjzV\\\\\\\/BlLDoEM8FY++5Q2nF\\\\\\\/Cuko4m1DMH1jRHD5HYLdM4WUbQ3pDUZVbAgx7kgbmeLb1FQZgcvt9GyoNTWHlVzU4SejmjIdQrG8TClSTfVtwi8Z0DWUexhry6OhAIwEvfwmT9mN0z2xF3MOQS28l5vx4cJ+tT\\\\\\\/LKxUPOqneaHh\\\\\\\/84O9E2Om4gn652kVkduBF7bUQBi0yHRRygncg5xNbdonY2sYtRvRO5U\\\\\\\/xh1d7B9RI0amfwDYnz+AxuoDtyZ3zYX4jMGiAX\\\\\\\/9ezISifijS\\\\\\\/ba1FYjmGTZvxToiqW3uDfVzgoQ0MDN1i8bKjJsSk0uMGnxQLSdhYE71cWzdqLyd3TwtWHS1wFl1KeLuY3TeGX8TvPGRWUonbpUd4Od80oR3oCmQlopqCWkGUo1cSlHe5ana52oAn3O7L7I4Zck49lFajrQOnjqdXTlVGl6dbSiDNk\\\\\\\/MClu9Oziz3oLvO2R5FugfjQ9FSG3iehESZTZvXN8\\\\\\\/7pv4Ntsx2hYrFvaWiNamAmOdQ\\\\\\\/1CgJo8GPFzKN1fS6qGbAMIZDE73PIOPh\\\\\\\/b0w4T1eyv3TJbMrUp1KG+XyvhHwdUrVP8xN5NyfHThX5JvvHaCVwcXfkPk7FSDAR81Mlv19zfOoSiUYQaVjim2CH9Uj6chAbhCJYrKTTKX3Jlgbej3pF+otWMoSB7H+uanMpvli0\\\\\\\/bGtlOrcyfbxcFvh+rO8KoZAqP8Ad6jggkgh6VRxkmDuBiRxYiEdoraFWi9eheQjUGHONs1aMNqN+NGVWQ+z1i34DDhcv+K07z\\\\\\\/qX4o1GN2tn\\\\\\\/TIB9XKb2Mws7AnRQd9RhY23L+d\\\\\\\/50nFtEFAhjaUKQKikXJjPWzQKsxHdFv+nhysg5tvkFw1SQkfUIKvY1hglSCcNQoIL11czMvKdglm5VfDdszObWR4mc9R6j0bwDWFz5UodTRP\\\\\\\/gLgRCBNupeftBP3M6lUAKgd8uBzCEvIgC9MRzsySp9iWyvnwsf7NXh5Qju33km9u+pQs63mzxaInBzUNvGUpT1XUXvFncseFvVe7m4v31gDXzOfCQp6NAfgX9mbojo5qjTDGDMRAcTzhUxEI\\\\\\\/HOAGvrdyGUe33Sj0B9geXMwIxyFcll6prtGiG+OmOtNhSEruLFPUlnbWi13fPVJW\\\\\\\/SHXJUbIKKzt4cwFJ4in\\\\\\\/vJGKt0dD3ktqmrgCTrp5AttjWQXX3o\\\\\\\/PFYA8ZptZIZnr3FNzkzGypp1f9dMIyv4ex6iSAwfVGyBFNY6ZpcvebxW8K5\\\\\\\/UrlSbL6VdrQ588inQyXeLyRz7cy6GMElIlp91z5Z+2uSxTm9DTIeRkKkO8XZmVA5Y7ipAiNsWGFlBsd8pWdcBInzsrwYuqDQfYxDE29h78pMofDGcWm3vCHVpbu9QexiNsivHu6cOomxukuY\\\\\\\/dT9qgPtZR43QM63LWAMPArubnvV0NfyzcJYrY8DuBdeBi3+S\\\\\\\/j2mssAdZBRqBi8WmRBy8kIcTWngbGkyW4Ro+c4j7VK13tjMqhqywdF+PWMdsjuvZRKfjZBGrtGtahEuF+hJPQcLNS1WMAH4s6hzBA4G4sY8H5UKJVA+SvBGmjQ\\\\\\\/NfUxw5wi0OCjhS4gJ3wMJDumm1SkH3twrlRevILXgAFeKDtqnGORA8xdoDNBfjQzNcmRn9sIzp+wp4KsoHrzbFxP+rkt+4j7bPmb60RYbacfdhzCt\\\\\\\/WuUqJdsc\\\\\\\/EsheDSmPe3J3ZZQEHwZ5njOSM5fCqEg+me2HJNx++ZeldFu6A17ksrsg7WUjI2NCOqIfFXyYfvALi+xa5YaST3Gaui5BgEewrwbjtVujkECtmi7nY3MfAvM+BOtner\\\\\\\/pJ1XEWIyKTAoWaySXoMbfk\\\\\\\/nzes\\\\\\\/nQr1YwN49H\\\\\\\/H43n\\\\\\\/vSizB8FPib1h43b+1ErbCWvUx9GyXJ0Li1eTlTgbtQ+E82TxcMr7NwQdqSQYTg0eNOWqSDUg\\\\\\\/E9Zf3iv9SBFJLacBNHR0aOXVGAahy8275M6Gjgipa9PyNJY3eljzA1sZAxq\\\\\\\/go+IfsxsyjU52dYo5ZtZri01Ex1BhbQomdyBe3R6TmNHl5KP72DEz2WdwpvxubEKwukpm6RxGovm3eybeKYwqPIURFz3wq7aGl7RgKf5cKPOst1CvxWbqN3OnD8xnEr5QL346rqOdQY+Jcfs9ccfEZwSSYtyMmmxg4LOTt10iabHTlDKaZW8O6oFmhWTpGI4GXzhll\\\\\\\/2bU6HbRdq4MvN7ikKXqKX2FmcHE+1kEQUUPE3XP7htdDq11J3h0SgYh4HI3DWD05Nsbxa8ETlU7+ZqWgzMc8Zr1hUYiEthnGUn2uC+9M6p+r4NG3LftSuVEYV+95CWwYtc2DJ0pELrwn6y2DBAck72OnECzJfjiHmZLZGAJO6I7VQPZTP8Q2TR1MoPJDaBLhzraKvzsHZIRODJ47oH42wYZT\\\\\\\/YOMnGnQl0hL4JiKRY\\\\\\\/H5KTMIDOjX7hl8P\\\\\\\/LrPnibtfP\\\\\\\/FyRQyK7WSPSckJZJgZHrJxKnoAYv\\\\\\\/WvMO+yQzTk3oRQHXmWFwPSSNKOcn+XUBB+GdnpTAzmcr8UVHItuQar1sScgDLdML5onZhHJcJBm68fvZiNdZxrc5hC73AXluCW0Y1MlB8adLNhYy2BOR3u6hJ3PDh5tXUVLcNdq\\\\\\\/N7OXLOmqmAyiR9hzMWdg1GYTYanu5x6hkAxRNZMeVoyqGqSeG+xht1ecU44Ffdcm42HQFKk+ao+ItpLKSairepbiG5XSc3q+q5Utu2vug0cGdg5xsd1vq3V1VGK\\\\\\\/03emj9N96pxD8XqOxxyb8q1cUVgyjaUNN6TCWtLeQVznKkXr\\\\\\\/cz4kAYg50SH0OTWmOgvPTskLk+js\\\\\\\/eHPqt4TgBni+blBy0bW9px2y5jxSq8L0VkJWCQCV0w4TeIZeGM45Rkg1BqGMB0ncMc8uv3kOtDfoD1Yocl63\\\\\\\/abu6pvlh8McA\\\\\\\/1mD+kimzHy4tGab1KPSqONKlJ\\\\\\\/HM9ZUhGPB0jRQPatri9vtl179W4HaSlBnfNmPRuwcpJVaIMjMHlv\\\\\\\/nwSSQZqWLyfCIYccocMW5mXkfupEI0KabAoGbSnWsfbUc\\\\\\\/c5WbuTEH+yEk\\\\\\\/2v2l6yV953D+CodgTgkEZPU6r82nSx1Kg2OxuI7YF9514rjqXD1UL8DzSnhuswP8udxIrOS6H6ruJlUB3ee9TvlJdSykexNSjuKeKFo4a1MNm\\\\\\\/saPVCVnEVc2xNg5eDtivIDeHAEK0yozqcVQO9bggU0vXDHUqxIplZBBT9YRg0PyxuXIzE0mwvVDWWhtb1ec3kScyaabfA4I1uSML9oTsOrfNdbfmFaQu52HQB1vpxR4xt8RI8Zo2aLpvOxtm1HEW897FLcv3XYnn+drZOX3ixMHYeYdcSnkmnmhRnzKAl9g7AamxMe8UABpK\\\\\\\/KDjKUfkg\\\\\\\/TNI7jV5L7H\\\\\\\/3sxnGaBnKzULpNi0Lqm\\\\\\\/11+1Sg\\\\\\\/\\\\\\\/0+sEy261IIfWNu2ShrZFE7Imek+5JK\\\\\\\/E\\\\\\\/reAYoIr\\\\\\\/PS5wXyrradzzxoBQkFGsiDbHrRxt6hyeK\\\\\\\/z97ooYaYcqJGYQ+m63BhwQeKK\\\\\\\/5jpZbv3CYH3FUsbZKCltPati+xQiVCPxJw7AOXOrnFqxojSmlCp5Uam4FEkNOGPEwwxx76PJIttP4VKVPy20EOEwBA+8jKsyy0MJFdRHEBdGtndFqJXTQT+WhLIoy2SXrxZw92SPnRpxGVi03Yst5qTDwhUG9Uj3CDJ1ArqJjIQ+A0JhUxjObaY7fNH3btFdKKYQ1dNRMEq\\\\\\\/g5Sd\\\\\\\/NvIzokA0tFZdlFmpLg4G9RaUGrJg8Vgz52ciZ2jE4XhF1pYFBpshBSUlvwo98VDQzf+btVX0qO8fXOl754I1ax+uAMacalclCj2ObhM3QYldgY6nljQ\\\\\\\/JYcUU8ZdDiU6PuMlWO\\\\\\\/vKXoP+vcL2\\\\\\\/Mq3wISDJeQUEpGaOJxltnDU8vxnZGIVBjoiwYvcvdzyy6febglcEdpT\\\\\\\/TLJH9xHYmSVkPtoFSjpQY3+oOube7VmX3O8WiWrFHUbA5iIxHIkDC8zJTBepMHgwpTTfFeezSQc+fHxcfzvlt1pnEafisqFiHVjA3TSMX7dBPvP9csIi5ANzIIUSQnuhUbmllQ3PPBJnR112Q0tqdzrqcAbe4wAxH1A50ygdr1P75pAYXx34fKD6t0zPI0IlmXL6cMe0QUnDY97tZBjUpiPjSE3kC6PB+YbpupCoGftGECiustWYKU+o7nqy2WhLLzU5jsCBVS1tb+HAY5EJWoVlRNYaOQAmp+zvJtF4SxEFIe1\\\\\\\/Gawvkwp8WKH669JiLp6TGSbNljlioEerNlMLIM9oFQnjrc5TYxEIMVR0Ee5HXFzI753ZJ\\\\\\\/x\\\\\\\/\\\\\\\/V+yn0ODxqZbumnKnOSIOzzpAmVqJA7dLmljslr9rtyDoO9JqbGUnl\\\\\\\/BSr+GZeneYLtUNVIhba7vSdJFOyylPnOWY82FOn9f5SUrZ3GWQLBr9HHWUQLSgSUXrITa4PeOykMrL3ntl4ECtl1c7ax0TI6gr5Xl7cb1Me\\\\\\\/GjPS9nt2RPKkQZWygsOGNesORAQuJmzn1aoHFnpwPtxf9kZ+8IhgJft0NCFuU11T2JuN4GG4\\\\\\\/O0dKUner8sdrsxC3Jo+i+lKf++wbRnPoWJ37CBnoeUX917DPnF96n0WbG+ZweNngLERUeayJ6nrezwVnj8fjA0c1W0tjg2Rqu3A5Qi\\\\\\\/u2Zi578f7NyEI3ZEcEpEkUS03qMom9HISAeu3lq92Vf6zxaR12YIR2lw\\\\\\\/WkRdB5cBJGDONOeXl\\\\\\\/27Srye0Ety56wzfln1N8Mag\\\\\\\/S+tVeYJ7jhVdL5cE6MAPY56LMgB5EzWdp4HpdIOe81ku3UMvX8KaOuhkO1mZnaV7Bs\\\\\\\/juwhK1OYRqWa3X5DUIidyJHV60nceDQwpKYnK5V9PzlmbXOJ+NmdP\\\\\\\/NlGX9cwMLXLquQbCrcBOLjJC9YvMvywr7xoFTwR5bwiLPcrXEf3xUqHhaquTvj49pFEIaTNmGPiSYjWGH5omcRRXY7JphxRCVWFgGw8kBg9nVMIQvNh0IXkTVwdu8vCDBgiNRAgkqD3ZShtiTLgTabSWgb6+VB4QtuPq\\\\\\\/MZspb1GRsRUR7sxm\\\\\\\/4jPcKh+VkNmYx0EdMVRAnLnVzjKCg4TG8whX1O31zBXhMyFdxKBv18XdURMIneN5DMtJU\\\\\\\/5XTlgilBEqYV+2z8Om2jTDL1M3LEN+vSdYDI\\\\\\\/WlDaQqt9xlQqO8Ylx+zl78YRe0Na9h4wrS4wHH4kmpjT+E32Z5DEPLQh\\\\\\\/jxtmhBoxah87cGagkiqtXaJD5SG7+XMl3fmiX5C07rMrWiTJYCPwchzadraYk2qKVI3U9yyDU\\\\\\\/upKeEfzmQ1WD2c7EWlo9BwhpyQMF6\\\\\\\/gNzT520f1a4SRBDwsFjIe4oyxyT\\\\\\\/sf+MP5\\\\\\\/YYivTCiv6VU3t6CJNqfnDCofk0FKzPLY4VzYXvL2mycCWM6t7BOaVjEOp8q\\\\\\\/CKoHgiCklN9jkNFJAGjEVH59rcJuSorsG\\\\\\\/fiTHhHCCcVDzmH\\\\\\\/m64yh04P15hVgfNDOu\\\\\\\/9HGcfJTX9M3F7I1yu8yhNSxRgkExgGuD29C796uG8Nm8g9osJM9nJyT6crQ3H+fJZ4knnOU\\\\\\\/dbLiYv9tY+SDb0rr2oltSitszm+GWmaqbbIkV0jPzc+\\\\\\\/WffhCkjYRD43nJBhep4Q7u9tDSpvPR697hNtrOL75BlBypLFRWnFqzIpaCajsyfSuKv9oOEjWoSx8mrS2GyCz8y\\\\\\\/zHSpBP\\\\\\\/asLkqKUsSjIUC6crKQzLHhup6OWn4I2AohlriFiBFYS6t4UqTwtNBB1tpvGRiKfEl\\\\\\\/RYb6Q2hMdAkyMjqEvaXjUnBDcSpxv+b86qHijLLyDxMOyAb9IQZS6OkoyzvezXbtzlSxIyLZKUhuK30sxehjmA3PKXeBsKTXJdQ76N9I4dEZLzzsiEI7SxBjjvFQHEoHUk173VeuqSzhexyo1ZKqdrh8YjC2FMZH5hL\\\\\\\/l\\\\\\\/rrTAkjRPV3ETkz4DgJgCxFNZOeQ0guNVOFXIhqdzlI2LI8K1psVsY6FM2QGiMV1\\\\\\\/7Jouw1+OOtukJKUa1FTVR6n9hFjSkBE81cm2KoO8Eg4q6A9qrvhF5I7zYIJCrpK7Ef8afTfbgWV+J1U0g0b8\\\\\\\/rzvdsLvOl8jE3nt0OOWtjIaKLER6Fvv7q2VboN0jRtVfWZqRe7DIDx07624VMGYJ2wwtRZ2u1V+E0ShdM\\\\\\\/exsSMOi+9mh85OpEw71z00SuWI2On5JG5YdkL8Ty7DnD1xDcYSxCElfzn1oauDdidqflyuKFal3zA9eIeIxZfNLd6KCD60BkQZDjONUwahsBpisN1A9wVE3OXUcrXTaHX1TD0RWHboJi1dMnHkkW2sIG+zd8F15Gn7JeNW84cVOndyZQiI1XfDqOKEizjFFGWXl9YNLyB8uMCNNFzSYcaCy4BEtVVtULGNjSiazqfL6hl\\\\\\\/fkGrGWxHX2td3WdVShRTmPAqwzhliHsqCjU4MBu8MqfrSpYhpHtPZGHqPzuNyTTx1zJeQmHTbK51P4im3PHGAuvrENDh+9jDlma2rW8VEOk73LBgWtnRROPifJZYTBqmCxK\\\\\\\/Mx\\\\\\\/3qm2+ik6xGJcWJDlSicUHVBUdaEGmlE5R2KpkwGND4J2TcRR0cNynDCa6jVvsrb\\\\\\\/3KuXmecqqrEvdG9sylS3yPgwehWXcA10uWVfkQ\\\\\\\/ld52PWmeJi\\\\\\\/Or2y0OAVnowj0Y8BrRsQ6dDvc+k0VOmuLLEIZq56vEQAA8se0ljQrpIRXiaxdwJg2KQOSUauu4qNe4gtTe9a20q69o1+fuQCYD5Gq5SsbQgXDZIZI5430SvHYTAdjAmgI3AXJmLu1hBLLk\\\\\\\/7RwZYxIlovX1emi7vufQLNUjJNBHhiyjmCmf5siXSVSH3ECWSxXsySbzvnfoF19t4NtVMV9n09PUW1i4UAcZwsobw+b8Ifi9FvqZtP4EmHZBBxBh9cjGfo0sS1g4lsV55PHnTalybLc0Bo+fQ+cevuX0mzoBMaNoQGF9bEXPq2veBzul0VTktRlrkZioJRoV5+PXtq3uWyBmuFtxgdoUarBvCskTu5UOI87TmYb72WHg7a94HOFdJM3XbIpDirWhzyZAQvlbhtFsZvrCgTzi9EkD8GkM0Krhlsun3Lq8uEbXevISIT1AjRZ9ecNW+K95iYu5bwdYD\\\\\\\/9wAvGF+EBreWnUdwdHSJ6oktgeZi0de2BLHf5SOBUoFGSw4L+jqkgMdMXU\\\\\\\/l\\\\\\\/PrqtErZSFX+iX2KWxP78J3583wbYje5hl7ImVkzpT13myn0if36SR5FGiIKIwhV+LJLYQm2ptwWjfpp7uebqF4q5IU5BYftM1AuUq9PwgcjqUlCmC51\\\\\\\/KD2nIcjkgZAm4Zjn3xXpRlQ8C7cIp6VF1jc+jRdW5B+S+esN80TbyC4LBb4eQKkOj\\\\\\\/+vBqdi8k4bgRQviYvx8\\\\\\\/dU\\\\\\\/\\\\\\\/j1os1KVBL5ySZ+2nhPS2MQhN4qUnkFWkoXWJbOODoGQIA1BLens2AjHn+uwM8z03cSAnqAoA89ySkL9A1Y9PwQokvPpoHUpQAXNksjmDT1f2SqRtOXKX0wOOWK3Y80ZUu4IY1HiwMkTLpx50Un4q1nVQIIaXqgjBbl0xIxZX17Nv2iRh+4TqgVrzKRiGZKpkm16gMMeYsKYDbc6NSISzFbclBIR\\\\\\\/9agbPI86vgF\\\\\\\/oz64CfR8HsR51ewmtzRxHzBhGhbVTIfIOK7rI1QxaS2Qpjg4JfGmirBVja5680yDOR1SAusMIHD8e8koo2CvqYWweafqxpX9AGi5sD5k7ABmtno+MyXHVhFOKbCr8PEemQc7RC71x8ceTc+IAaP1CikrKT9TZgIwUt69CeWFZDLU\\\\\\\/Pcjrzo\\\\\\\/o9Grx01w5f\\\\\\\/dwkctCADx0bdWs+nbP2chfBPR31OOgUahhQJqEpC8QShJCtwaLKQrXMF3D6zDBgYF1hbGZWABInCUd4mzJh6BFBihtILTGIzloKW1TqVfnQXbM1fzpkxjy4kL6bb+D5mFqiBn9JAS9g+x++12mfyI0Nnj6KxCP9KBr\\\\\\\/SGhPSRPVcvMJhhIwDvjiVEjQMqgmZGFWYFJr68LnXIIIJXhmjG6hJ4wN02E+wFJT4eT9ku\\\\\\\/7Uyi7q6M\\\\\\\/NB25Pq1f4MP2wHmog2YI0+tsvvLUWvust2kGwh1elYCGNZ08YFToR9\\\\\\\/4CabscGRXVmo3tevH5pZPK1unaEHr6kMPku1h+0aXPBPHlQb\\\\\\\/WTVeoTDqZrO73qZhPpEVht0uSx+\\\\\\\/dqZBhr2d9mbPTxJ2GVNUMmS4PeX32qPe2LjalP30jAI922skKYMvxQ4BIIH1Em71QTAFgvuHYJCnZgyrqBg5iXlGYT84e\\\\\\\/kb0antGpM6Xm5v\\\\\\\/CO5ofqa47dNnMY3BwszDFlpBn9kfZgAP9O03l\\\\\\\/\\\\\\\/Ze8MD7QoKZG1uGrEYMr65P33kf\\\\\\\/NzFOLFUcFD9WCiL5S2zOrhcn+o01hsXR6Xcd9DYXvSrl1IDge4veNFxxkBBY5PT3RrNXG5jPSPwJd4dKhiE2KqebO5\\\\\\\/SMs85bRBH+Z7MTUZBMqNLx7f0btnrWg90ppDa2zXfzTaMR110tLECz7w\\\\\\\/mCXYVS5mGs9wq03ujPZd16bM+RyjHs36QqHqoLIVUSwz9LxTiQtFJJp39lIimddOliEV8YonLqfJfmYuucGnGQM9vooAO2BHLrmK411pNe\\\\\\\/6L5cnZuMAHuuOhA5okbZzXvX2NWyGMsl2z+WEHz7SDgMaNKHQ+dHh9TINgmNk6nTKdqLc8i84sEfYpOVjVvCIP3\\\\\\\/dL3qReyIMNHK7zWVtoMPbqDi4UCm7kQEwd+UMXtP9RuNFJnxXyeTNuU+kD\\\\\\\/HaK\\\\\\\/SJx25ANvip5hKxcdFyPC1FonaIhjpPVn+3RQnYrIjTQHllYExxGSOx512xCBFkjExcjmnnsTxLin\\\\\\\/+j27pKQnNSFDWrbz3Oph4HgoAdaBCCv3m6FQGdgY9qEgTT7O919jsYFVxYUn\\\\\\\/DL37Eb+wWSDvvnQ\\\\\\\/shu6WxaPQNUB22TasbVyyKctrkMpDASE32oXcHM0dWdlZwwoRdVGrgq0zC76wwa5Wy7e+n+AF6gO9YkJfhp1aPZAqhiggrvtFvBi3rkSWUviMmaVu\\\\\\\/db8M14f3oNalJ\\\\\\\/0NLMdR1ksxzL9MNmse+1J1eZsZ+KYcI7bl8pCvFKsog0NqpGLg6Uy5W4g7UFxmZIAEUhktLkKWZEPwEkXwiMFTUukbS3jTVO4p+zaEdwMVOXlNtLzu4qGC33h8T\\\\\\\/Qx9Wgfn4H6xUmTsZkW4hkyI9WfhHssRioXjda3vaO\\\\\\\/PF+cUDxvSCRmVurQxC\\\\\\\/pApoljhzLS0ZppjWKoIDYC7lRmBpI0nPqHfCuo+BJ5YlUMGKcdRq\\\\\\\/yzcouzxg9W8kcNVBluoYmGtlg0CvEgYXHzeJsPw0H4Hs4HjlyBjsqGRizZZ6j4BZxYc8P+BSu+uC+1lg7CFuZzhtr26jWGBwMVaqCtYucpAIicfV+7CfSUY6HAaQ6xS9ALemQAvgZ7f\\\\\\\/r0aR9BHLTRl\\\\\\\/thi6Yg8+hHMPdrWNrPtjIimEc1E5HI2cpO0UoFwnpoNMVGSjXmKqk8NZnk8y7h3HfsiQCO8n0GuDohlLhD\\\\\\\/aO6YO+gVzwYIhvTNX9JbwR6aEhnAozgcSnPlA4NYqz88e1xQL+u8iAdXeIFs+Mgn0exeTzFv6SuRqaRowGB+dikjoS+DdWf6Yxr+1qFPpzU2bhPj7bT5vfX00JgN05iq5PXwGLWPo7ltmYaZrmKeE3eg\\\\\\\/Rg7YXp13\\\\\\\/82mA2uP3nGTR02T+D+3Wq5iyne7kYmm2+mucZcqVvcIDM6xrFB+\\\\\\\/lwzQOfHSg+ZMQIP9Yp6JXB+Nx\\\\\\\/ttWbgm7+Yd5\\\\\\\/ZmRohmQpouJUJIE4Mtt07VX6+R52QvJLlrxEs0XDzdcVGh1btAzXnhF62ljYW6Pic3pjlwhi96Gup77ylD9arxiesxkNYAomugFkQgf3hUYJqxk1LEteSdRi7288KKrGyqbH+kSXJjBxGiIGWZCMBMOj4qE9VDMR5bz8RU3SaIaJpS1jaao6brolYd0OzPHDLhNmuLSDg+rHNAoyHmi5+QEKK6HoCJeT5oCyOC0xoiQd7vgNTbTt8+6WMoK5auxjAmDZXxw44B3TzQ8TB+YvO4k\\\\\\\/C8NqfkIb8tzjyDYGBNe3M\\\\\\\/YzLi7pLgOiQa6xvtpZ7\\\\\\\/2ETJJQrM5XOVrlCFxUwURYtyhCY3FtmEgVPZPLfTJmxMDruAHzbrfj5xbmxE54qgbFemLOp9tScXl+H2hFbR70yVLpw\\\\\\\/htPRThbmHU4c789BqAFPb2AFs18twePYGxlNdEW1gLVebnAnMBjec3F1Tpi8vGuzRvTkWq7JOaOePC1dnLwPfPzvLsemjXTmDi8hA0b9qnp3l4jVv8zBfr8Fra+QLRO5fcccKBozXLglXs0K\\\\\\\/l5USg+5w\\\\\\\/iHhsvKBAf2YfYWHQOMmIEE\\\\\\\/9V0n0nnctcJZcXE1KXVHEtCD7hq8hNnCAfVjwUEAILYJiD\\\\\\\/FtOuNwIeCljJWYRZKh\\\\\\\/SvuyEd4fOZC4AeOchwfSwqtM2Z3RAhktpmTqfVjpTPqcP6fZZwmwLyh0BxapgqZR6pw25WBkktlzwP3wQD96C\\\\\\\/5bPTcY82YRbsyJGO9zrp0MTsSOit5frkseU0ZEReqzZgnWzX1b6f2Rv9XiwWNZVDAhh2n2J4SiEmtzR6FCUaLmPqPZPsdTtBeH+5OBppn\\\\\\\/tueyKLQ2wOTxlbexNtz8LW0GxHaC6dGWUUpNK4Z+qgKeOCqDuw9FnHa\\\\\\\/e7oU5Wmvy6aUF5pcU2snZj+ZhhbmSddv21pUL9vgpD6aPXVtzUcMBrUWC8z93lwlHWech8ei6NOGpIBzUtfvVSxNTFVIDo0b1h0n0\\\\\\\/KPmHwMUsA4+qCbmtrhkRlODkKrG+Cvd4ON4Pif1gSQLIgMOJsKjQGJMiQmCWXdmdXhHfiPzfD\\\\\\\/6svtCQo4eimDI\\\\\\\/jdPWZTpzHyR9mqFPE6ImfaR2KkXo2uUPXnXO2tZ+4nWwXV5H6GvJzoGb4G72L5kjLf75rJEsa9Agd0mG8bNpjsAxWUE1pcCVUBByw0i1cb774n4dJ+1fOnjbwXm8iu+PqYEbz0kAf9+afYL+egYFDaEkMo3Dtkk6lTsjzlBghne2iNC5GXXooqdB5mBSMBiYbCp\\\\\\\/7OTKcmRNd41o96IYhRNgeoW+7VN9v0T2qQejG5obESr1vJwllkAJIL0eUtZIFO0UzMyhzFPzOtwb+GZJERkXzjRTQkCvwa7E\\\\\\\/U3MMlUDoqeO\\\\\\\/V9uNz9NDUvDf90yjuwsX5QK92OYeqQ6LUUDQnoY\\\\\\\/95vMRLVajObmSgbx7iwAJGR75Kncd\\\\\\\/keHbAwYa0kb0XLQYSUhqeNjpIfcnkUNhmqTAhnl888J8iPpYlHjU8QvxoS6UNiOSvy2w4MAnJZJaFw9xPtlKOk65mbkyM5WLSaz4qOjfcUMhNTy0QWREN+p3iUYHljSotpK352d5yACrc3vgeJTdSmu2Zo5qxRb7ROSCxZ2BuGYp9dvMMFsbC43NAOo8PDEu7bAZ8k08Xr5NF+ernTQs9XUtNufxhrqArTot7Z43Y03GKIPYl8wRYcTm40507XwO5TUlRu4mbS5\\\\\\\/1N6Gmx+CSRj6yzfkEiGU2J0xpLKzlqT++csQgMd7ZcGN54qTIrYfe+jRiY+Sj\\\\\\\/+wmj2ngwq4WihY8DGL9aBwD4RNzYbRTcF2jh7GOaWy9n+7Aq05fdn8xrd\\\\\\\/0YS0ZcUCizMdGGbsA5JW\\\\\\\/P5waqF70OnsK\\\\\\\/3Roy5qIFaBbAwGwh8B6C7MmeS2vqXj0RU20kRoouCp9wR\\\\\\\/Wrn6Y6gEEgtKd420npWkrPkl377DPwHiEIm1IvgjprB5xlbQZTMbVj2tV7TAnJZzlI6bvkzdXMLrPbEMAmxT6UrsYpn8VhUAeeXeyJB1Rr5Cno1gk5RQ3y1edCiIVQUUu8QkREjScpQdYn4Np1rn2KZkb5B0SX2Q3ntUVVQliPWqDHBzcDY5D1RzofUG949QPDqtNammshkofrlDmDJmsM58ELyna6CRp4e3Gsgy4tpYknZxhk7MXWUMm6TrLahlPtNBUKcwxF3yjIlfLxfntpbxRnHZTxwMVHyXVEUTPx6CdoX47Fu7pj\\\\\\\/oR0CXYz1J1bDhHpK0ccPB6m0Iq3VGaqFQ2oRf8Mb+9OiwKo3dngYuszWKlbZ61OVFgaZjBeaDmPAiOElR+4iSh9+u\\\\\\\/TaY7XrI5BPXYE734HSPjsRwP89ew43nBJR5wYW6pYkq\\\\\\\/eulwZo1zr26VMuJM4ZhyFWmHnEEOKzsjtc7RuTDsmK9emweUzix8V\\\\\\\/DXJpvP7\\\\\\\/d5D6PG3b1MvUQTbjgvOVu4LxhxHmy5erYIgfMulbCqcQzw0UHcX320YxMvQj0lucstT5fquDgoYyAKTFCZXMEbLx8F28FF3j4Dk+M\\\\\\\/sbOHSicMt6AoRLIOKW09FemyfI7+pNdjurNOLLXWDzo+vsA6xcE79aoNzAgzO6yTnGerKvfBS+pTswuC2ZyC8ITsURQp3wavCmYddM7A\\\\\\\/D2uSV+oxWmfp9co1GX\\\\\\\/c9kVPppjeUqxQLi4H3eCyUqwMe46AEUqb+MkMo0LfscntNNn1Dhs0m52cc+qltCNInbKQyhq74T96hdGWWGae\\\\\\\/RzNi4h3KSWreXfIK66aN\\\\\\\/tnZbyRtAXadZsSOYsU0NZ0hm+cDQN\\\\\\\/TijyGkS8V9YIahaj6or7tJ+3hUAqvH9KhgmgfqmUXGJRL+GRu9+\\\\\\\/wXx2erigNGzQPmECBLwWDQD\\\\\\\/TuvrrZQxHY72fx6VxljHUd5reiuludxLORGJBFAn\\\\\\\/yQ9uFgcwvLtcZksOGHNvvm+l8g6p89CuyPPkP45awj1bfdZb03Q9vNEwYoKcbxOAzdv2LiosQF6dnypNJE+fJY9O7\\\\\\\/DqvBNdciX9Aier4PhTFo7KQtdCaAfrewpdtyNOpoKu9jIkgQNRaGTQVca3W\\\\\\\/RI9miENItPdM7utFSYd+VbMCeVgdTz3ynHJpoVGR6W7fs9k3xYadSEtd8OwUKveF\\\\\\\/+ATg7Dxt8hR7DcIhowjZHqLy5HPdPSvVbkXDCIFwJ1sukTi\\\\\\\/Xqgp7i0V9ZUufGQwwGZL2tuqthSUcxZGkASnsG7uwUx\\\\\\\/WWMKeqUvJQebk5DVwu7zvlCifu6s31mtEOwDcxt54JIF8YQfsFFMs33qKAH7E6HVstP7p9P9+UAoXCZ6raWaq1XlbJON2d+0ykndLnwgEi5hs\\\\\\\/ScXFBAqJwcTK+DtcZyW901VjA0GwILvf\\\\\\\/hPLOx12iE2eBMeJOm8nRPFIBAmiI7VPojwbQCetcgCFXSSJ9RoiFW2cdnTHyTNrrGY+yVFvZZUjXW9FDRJg\\\\\\\/xcMw0YfkZeJmbQ\\\\\\\/qV\\\\\\\/g0e\\\\\\\/U0\\\\\\\/dWRdXrPMHx3GQHVawBlUsZhSsMfOxYA7AElmGYsbqQ0PXmgzWikDNxtGTmKUNC3ZwBNVd\\\\\\\/k5BDuQHmdzxF8djOBlHnOPJjlLHjufzJJ3p+PKFVCaqwGx4uKILXHYAX05MpRw0b1G25vrQnx\\\\\\\/NGuVtOwqewdpJ+hdRZgYYqt4iir+qbOTWUOIiy9Cs45MyoQeydo1kudKwg9UsAhnrTynzyan7\\\\\\\/07sOhxFQOxRDsvKuuld3BCZnmZbuPF6GQddBQsL7GVmEfrXxHeW5kKzYwZYDds7dzWSw0rqk874qWrTBknfc2VOLQHmwt7A5O+VKk7r14tdJxBvGkdmM006qXhXZdZ6YiC+QCI3iV17oJl02vD1JFCdSRN7qkDh46Wr1ib7+7njH\\\\\\\/2KM2ASNV+410UEkG8tdmwoO2iB6T8wsWEQReXSYVUdEnUcVP4DX2v\\\\\\\/mB5rs8E1VudokSBlsWEGhSCgcrhxFAdsdULaBLTMvvi25IUe+CLCAr745sh2t56D7BjaLgX3d7JX7E+mDeMPnSWkyVHZUbO8eFuUm3KvC3zHRzEuECjT\\\\\\\/r8a5\\\\\\\/LCz\\\\\\\/1yirxaD7FDOlM43nRHZ+6+dch7XZWIZGQ81BnwEr2sgOqOrn5W0B69qQoXoe9\\\\\\\/HDSG4m0qoim6yh4okXHAURCiTEQ7LWPdedDl7NeB8t72LWE4oLPENG6\\\\\\\/ELTvooKG2KFQnk98cWZxhZFvTEhb9uIggsO72eZXAsi9K0H\\\\\\\/WLUOKlYUPXiFm9bkx39Yl8LFgM1iffdtcF49\\\\\\\/etWdkuylFwLqKNlQLejH08b873q0LGp4NwGYGlumJNeJn1yEUjpl9gjqqOr7A0Hd8Y5XBO9NsXf8LPmpwgaMk+TKozABuWpFkgr3uPOpiVbzb4\\\\\\\/yc8vXcMYpyMyUJGkAuxH\\\\\\\/27q2aJsWERGPNU2fiSMrVVtc7n4aqx\\\\\\\/glxgJAFBIBbzEloXN0LBfSiHzo3yc4EJnHyeFICgaVWGcQKAO5NrMjP85k4\\\\\\\/ebY\\\\\\\/O7WOdbDejm0qgr3SFE8mmLUtFSbyyY897RTLewzhJCNqDZtpoW2dbwXYtsZUDpOAhQJumc9gz8INWG2xXk9K8HKKy+2UPEO66iLOr49wH\\\\\\\/itnvPjRlEtb5adoODuG3HqGFf5R6In2Bmn\\\\\\\/aydgPhnkKnNc1m6IHUuB9lpM651LP0otobiXmj5tq8PcVukipveXB2AG6RpepqGOs0qrJhlOy1AOKB9PHicIagUp+nRoRKfniMx\\\\\\\/\\\\\\\/kV85x+b01zHXabSBrdKJOLmhXqq+ZD\\\\\\\/2OWbJB8vHPCmaxeB9srllrn89OrbkTjG66g3o906S+uqrFOR34XG6VXeEMqVn4d8+53PHxg2m\\\\\\\/niFI7dsZBHf7VSIqVQaJqUHcX1X1bOPCZzZaSHeCWSoDDYJTKdyq87hszPbCukJSH6GnP\\\\\\\/4ww4bS9Xt6LX8Bg49LX3Db3GkEpxO67sw51r5ie7r3L71KJQH4K3DvzW3\\\\\\\/NBmMrdQfwjyF7ll2vElmP\\\\\\\/NEC5aT004\\\\\\\/YOqHhY3D9Xu8J2wMiG3gwjHdEooMo1Jj9lYU8xOXAsGJ37cKykW5RxNnSX1Uua\\\\\\\/RvzaEr5Lez+VcZ1ryu73pboJlo4gEJArxCUCfgDCHs1rA1dqbGdu\\\\\\\/j6j2+Nypl6UUww63\\\\\\\/W+8AARlY2kZICrj68tXS\\\\\\\/UbJeQFHwIvVuf3VXDgR4Rcpd0I7WKQDCACRQEZjhKH5sRBvB+4OnvOsOP2GC+v5\\\\\\\/4lXGRfA03w4MNyuDdoTjZjFhveiIWBIfQhLxH92GT7O8dqa0Qx6KY8vSs0olKnxUWWO52No2COoYG2hRw39hv65WjVGNW4+nWfEPKzA+gwrK+aW0kU9ad8ahY3EG9OM2mSi4a\\\\\\\/FYM1WgtpEOQwePlF79AnEtkIlvZMBPRTkVAtgvd1Xoq3VmiJ4fYS8twGaU3lA6592AL0tVF\\\\\\\/dhk9xYcfTxyykxryChVRBKNeKquAUsn\\\\\\\/FfCEXikxZajUrzoge3cnqQzaf4HLClHxkaySP3WhFQvwWRsRaF8+HJ6nqDH4xhmqbwyXQx5cdxhrrESkzY9Dwh0kGsHNYuiAJptHx789fRKjcqlZzRv00h07lAPr7AsPOCRiLgfDKJSsAOheJh9Xq8YbBkcFa8YZivVZZHCHrp5mbTyKFrB8M+wAyRtvhn\\\\\\\/nmNfcjZjUpusm0+UDbFi\\\\\\\/Q+iOTltDh8nsnH3R9fEvutcWwKPCcR6jFROdTSgNsyxqsJyEyYnkOydgsmE8b1QiXcXwPYpWWF0cSd3TMlsWnuaYkpha\\\\\\\/wprrSbNu0lnqeggNiqpXKLwT9pIDkV\\\\\\\/565hscr6PVJdIz3yPUOiWbiGVezKedZlyVTdQI2UbTxhRbl78+cWbN0YEvFeCXCSsnjbpsoQZBZlWZlRxZm8RsXmiIflCh9z8Z7nexx3mnyftCV\\\\\\\/NdyVc5LA+qbhdE12HrfD8GLthFebPJxb0m8gInxYAZKpu2nvKxCsKsifDmB2VFnkjFWpJ5j1jwouUQg4vMevj\\\\\\\/MoWVh+kNa2xh44Gc9ORVJ6HQLqCaopUTNMwAlYcWh56zylzNqMYoJV3jrxsNbucMwkSofPmxuTHjm80IHQ9pw6MaQeVbnROpMrakl4vlQz\\\\\\\/yfcvH7ReqnYFPrhsH7DA6+ijMTKoivVBS70IBYfa2uZqiG106L9k2jJjx4CqcDQmJxNnCC8iU+a2U3LB9i8I7ARLZeRUudUlid8f45ow2EoKZR3kX\\\\\\\/IECXC9mu3XPd\\\\\\\/C79q0OxK+jPcau34oA43keYCdE\\\\\\\/44XWT5LjXlfqyqauuG6TwdXLOCCTp0lSuQCKuS4STvwVzLU1pO5+98qCkKUY6zzBxJ7hnozQ9Kd6FhCJm13nPMzXg+stwG3HfiNVmtbLA\\\\\\\/JxSZCGMML+FwLKyGe4uLbEx5gyAX8wqTe\\\\\\\/tVTGbSS\\\\\\\/j1Olr9SovqEBp9sCRNaI5EJ5q9dbbDfItGPuipwu8EdlwFtsfJyql1o29i\\\\\\\/aNMXy\\\\\\\/zXVxupduCtkKTege263Hz99dkWfmn+\\\\\\\/v5Jl72JP9mAXXaABmdGdQU+utaAMmb7H\\\\\\\/RloPTcw+whvq1gEGJdMowMaHnixPkdose76UJvgPCr15QKkuI2Wfesljkamz9UaLl6n17POaHhdplIT0sCHRlEUGkpx5LwHlSiaiN25e7pv83+S8XIU3cvmPaBghtjUMQufdeQ+Rv8twaq0L+cMt+J5JUeuVgBMaFzOUsCdZP3Xhzi2+L3mORYhMN9EApFIX6jxFgOUU4yitRDfMJfztaFf3bFVmGuUpcownyRIAEvI8MJV3fP3RclvvNCjPPysEIL9oTTWPWeCFjM7YuH7USykitwgfF6VXM+jQvmfQKCAcMuM7C53vdpK+3zS5cnFZKiscYKlnj0MXFW6WwgVaXCRyeiWZMxl4raFKZoK7vi799aPvBoNijIlCtK835XfftQWRlcWiw+g\\\\\\\/JNztQfg\\\\\\\/\\\\\\\/uOjbD87UVehT2C65tilEgxWZLTtABgSGb+QbxkAlhGxcZxAQCVGyTqFnfEF7aEalSSqM\\\\\\\/GBECZUYmkXw0\\\\\\\/+2PY28psUId\\\\\\\/tgQ\\\\\\\/3yGKXoAhBeb9bxUg5ZbCDCafG0Zj+nknWkRR5eAxRWhFAxCpA3uBTyLgyVumQxrg0YOFXMlRghYKxqEK\\\\\\\/bMt2SEsqxD8ZAEHrGJfUpYpaAPmI1ZrOP3QiuLlyUzUfqAuzRHq2ShwlYKKMby9k03Y4N92dQBtdoptW59bHAScRRlApOPaC5HUAJESbINAe09fSjq+BiNzm9b8H1rkI4DkY2d5t9FZfXWNer0V1+ymaFAXQuANZUEskfL1pWnnMr6LLuttSoQ6j3J2bpmkarhSf0Kv2xS47+0aLNSVcxoDa9ZODBi5XxotbZsEi04xnImZ0eqK5aZMQOl0mLa1yp5f3B\\\\\\\/l4hfFYkQW5NqUaJiYjC\\\\\\\/eTYh8kfIHxNRmbT7JmuNf7PwNdhCD0MVs6QFP8AFeCLZlgIpnnemHut0NLk\\\\\\\/iT58hSO7YVc3AFXZB6RW289vx7Nt6ydbXaCHrz4M4n\\\\\\\/rp1O6Msxk4B7utFhDJGri4mNgNRprFoWFw+PNKFOucefaQX4\\\\\\\/HSXBovP0Tp1nqrymW52ChHHu6LeGkl6Z66g\\\\\\\/YLiaAQG6tnQHyswqLORf\\\\\\\/Optt56y\\\\\\\/QqMBEOWFck8E1wHXHgK+MDjOrsaYcyo88Vdf1XHi8U3haLYvrMfjz3GLGoXgztk2a\\\\\\\/oZf1KsJZhiIHmlKu5s6SIlle2p39MnFB+gQFTBIXIFqXQhj1IEv3tYonoTNqaFQ6LgMr9vYnm\\\\\\\/zdisv40b7N+gLUZVgy4ZN1IK0LI0\\\\\\\/P1pGhrGJBuilOc2F\\\\\\\/rgK5abowtFhupGNL8i86VuTn6Md3FRLdfkpfGKV6FFR+ldlog2A078nNF8MW+8LyQlgvSZXrVAMPy2TPwd4YP9H0QuuQt3og0b36xBCu05k7LP+RTE58FoAwVFRNqw6jsAAYDss0J4dUjTTTKk7jW++qFD+H0ybMQEN9\\\\\\\/\\\\\\\/gtHPNTHSVsOsC\\\\\\\/5g8dWSkNaS6WL\\\\\\\/eMAKqngJHdAwrnXcxkGzZ4DkOsDRKkj4ImRWNOoxgtWprhC5YnfsG4r4WrcRiPYOTXKytwYzUf5F9XfWLbN+JK6sEiKBrNlh2hXQwBtQ+PS8+Aa1yppdQLaV\\\\\\\/V3AhIwlTK46Q8QsE4fXV0PAofHErH7rLSRjDc+zaSYAz9X70jS6KKE+CYZjMQFslgyO4wDc2+NQwodG22EAWYwslo0J+5hCeimRuKYQBTspatwE1qqiGPUNTVA98IS+0eQ056Rltlhgm6+fpoZLAOg6pBL9Ybj82LzBOkSr4RqdjdR\\\\\\\/GYo6kVNHVP0wQ2ZtyzfwTwRenDtQV8MYgSnuSjZu\\\\\\\/5eCBeE7zEh\\\\\\\/zExkEl6am\\\\\\\/fiDmzxYYLOS6b9\\\\\\\/a9Pex5vWE7HOT5mCwou\\\\\\\/I265Vh9YuGazYUR3z80vwXh\\\\\\\/NjYb6BWAiSvr3eZrFPAIUfuVFSUDAfFsCiyFcYUzaSE56QJI0nKyUlnoqcll89lu4ZO42YDlP+7zBZxybTROespltDJFDQuZzmirGM6UtYgvZAs82uS7tdww+4nFEVxNeh0y5sTyYKjMI1YVcgRbwktR7IWkTYkOJCylS\\\\\\\/FjSHJc5FPwO0MdXCiQO\\\\\\\/RU8Ejxg5jBDTAcO0IfI0jDhGBRX\\\\\\\/3WOlvhZt7H2Vh\\\\\\\/kVnS3HSnVm5U3vHOXCDprJzeQ\\\\\\\/iEKzaKrFVllBiHkuhb0CVnr0gHUJCiUWnu4\\\\\\\/CsIZ27OS2ewsZs7a3nawyASzmBKhRYhHhvGRPueOp4ztzXmUIo3naPGUcBdkxYYmXO0zORmVSJXruaKmdVfNDEZRjuo9qCjHdu2iDkKqr9QUg9onzOZxqtQsCt9wZzTp5qMXMyNzyxfOTqwPvW6z7NDt600XpfY3V1QZOrH+YvlBBecJsPScn26jQ8cYsJRqEV\\\\\\\/38XzhQcgxQiFBA8RflWJwwDf9IpMjcb+dyQJBDN+b6YPEfTeMJ\\\\\\\/ERCWo5pNNb8YKVF1qp7JPAQFY89tt\\\\\\\/SDAcUrLGgMdvZeza4xYoRGOSON1TJ6a3R6OfV5GNoMjMF1hfMkdH0qCVtb+RtlYlXWILosKHQCY4g6QqKSj2qgwEx6ROAAISrr9eqbcyKfZ0+QJpv1jc7JEkZrgWUoZdy\\\\\\\/JzDuf2YlsI0F1TXpQ0zJAtx0ZYNbE51ldWEW7BHFaw5DDwXG5ZkjtiWiBqjJGWVPLEtSi9H6wiNZ+6YZweqdfpeS9KiGb4GjsDixvY+ofZTDoGEYUWRcDWOsTfC\\\\\\\/U3ON\\\\\\\/ZcWbGnriuLthTI7OvueKmm4fofynXZCBcuDyvuTZaKfUTbTXjGA97YyRsFL9es9WoSTvbnOs\\\\\\\/L9K7KY18G1N4lspWibqYPmLzE+GhjZkkkN13M+efinLdPYEwLpvguYgnrZ8P7qyRhA2GCcxgBMxc0uay5yqSAs3ZWabLMcjCmWi5s\\\\\\\/g5iYHIlLxuvXGhqqZaacBNUh1ctxSZyqLDOu\\\\\\\/xwaZHfKluHNzUc9S1cRCP08zaqfcVuNR0JdqN0hwLycCF4u4roXu8iMdoj45A8sJrT4Wksy9+FZV\\\\\\\/tUhNpWfssizfql51bj2MCJPhoev0CeSiIcib84QE4hg\\\\\\\/F10kN6rq6keH4CL9u9UTlZLIYMPqQaXxUo9FXx7l+\\\\\\\/pExiC6z1Ztd8KyIKgD1nzhuv1NopeKH+zjjVkHsgyAsvcKWCn0rjVDzBUMCyDdcWWxPXmdxzemCgstB8RxiiXOIDZB4M4AHUFEDyz9FU26BhS648NJQHYIANt3i88bA10iWrABZnxcPpRHKoNJMpPT0EFjTTw8YxxS9L2RMOq1Xf9ncK+HzebCgc7IaizYb1P\\\\\\\/gyspgoztT6qroxlP8TAXUYsgptGnc+Xxif\\\\\\\/6map2euOO+A6w627RJA+1k\\\\\\\/eilGwRuUiwlAFEJ5tp825B8JKuIkCf6GXscKvAGoJ98I3kW8MP3oN5xMT\\\\\\\/vz7p7OBqck71qk11jam6j2qg6nzvRRExrpFpeL1wrnaclvNSwYr2nM64b4JDUWtX5xW\\\\\\\/XYUKe9HHMQjJ\\\\\\\/FssbsoeEv1bkp7UKf6eyujPMiGjDFkm7rcgFO22GH4XxjOoeYoIhwqGKloGR2z851k7I5hQCtf2YhTA3KYgUCLHusbKG8y16ISgAHroRD059kleXeWOYzjW6aB+ywUKY7zr+PJJ\\\\\\\/h9j9+Au065dTETpaDINYWC7CHbGKqVD7Xd79tz51S4d+alBYA5J4rToAtUwQnTSQUC3EZ6CoKC92laI\\\\\\\/xwvNTRfH24auIfvFlg\\\\\\\/E+M7Ut9VdVFa5wgWH9OvXHPKAH9Hjz6sPjgUVAOer1QoBcDZDyXDyEWbet3Wuy8bx50iFpu8eNgeGj+lvvFMSfd7fSoidDZ3m5pTZed6ySc3OxVstSI1GWL9GB5gq0ATY5nDhlTSH8ucoPeg+0\\\\\\\/T3\\\\\\\/xc2Z0hOV6T4uWsm2Ne+v8y\\\\\\\/eMirPJ01nFvDafqMhUBxg5g6jda+2UdeSgF\\\\\\\/bdOJxmPYp6tHdrX2fY5uMwoEfjHYrpMGPSAqJS5oMoxCiaLYiiv7QLGxPU7d5btv3\\\\\\\/7X52lOnNFdVgK5U3OkiCyfmzvepM+Rh1mTrO1JWJohhlwJN7VjLmLpD1CT0GFjsKty5zbnKAR+FaJqm0go8gjUEFnNuTtJLQgFA\\\\\\\/8eo7qYpw3EDNcS8NgVLa8D2gKDthR3iuwk8uzbU7TwNbNyKkHl9V+l1IQ+jT24CsEwYm3H+HxXxiQc3HoaQ6VBGzL0wu7Gy0DsUDrbUsQMFr0ne59sftLDX8f6GSK18j1vOKJfG1NqHdlEAWxHwYIHaIBZ1LYAG6dLJYtdVH9I\\\\\\\/KeSbP7bEg0mfbFH40homefpC0p3BW0FJCg8uunZOcok+8+xTZjewIQStt\\\\\\\/EGusyezTtT8jene0QKSSk605MWmP0K2TxbDJXS6ru2CdIYZIYEGFDt2TVVK4DJylGiDuoM3bWaY2QKB8YL\\\\\\\/9LXIN6zH\\\\\\\/K6XqQK+9PUKi9VxCdrA3z2G\\\\\\\/BUZ22tB5wDzJOUhDcW6bu\\\\\\\/mFarkRxAB7z8Fh14JdqiGmO9krsXaKOOJdXmx91KHFVfqZre8FL49ZaHawRlQ\\\\\\\/d+\\\\\\\/FTCgkh\\\\\\\/SKtUJY0P7qocC1M6G4tbMEV4ZUSN7v0WdXC7gqB607U76\\\\\\\/D0i3MtzLP7M\\\\\\\/aI4VYacj4jIi0nEvroJRpH2EcdwpRpQBhgA8SZyZIZcpXB03V48zQR8x1Gvb1bGJW+euCz4sxi1wn1JQHk0u3E+vbO2k7fyoakG22O7brJ+bO9NrqrhpSR2My5GHvoYbSvSwSXPJ0V+2UYe0jmgBUnSdkKT4ZN4ch4X3rWQcqkaIzcDja8dNYafBhoCTtWyVOYYIha6CAID\\\\\\\/HXy5fjv05WwgmhZUdPhggLMxhdecXKilAvnq0m6+Kkdh1mdvC9heuSxloeS8nakGBNVcg32oHrEqTHhSCY3OYz96gPDFnwaXJde7wEInpgbzIgfmLPHxYZJOHTtehQkUHc4umuuvbUGzQrxXcYtzojXQ+fEZp8kjHLkihwznhmnhIHBp62W68AjFV4pWKMCL3lMI\\\\\\\/6tAo7Ftg2JTln72cOpLfiN2Kc+1KYCiUQCnS5PWptG2w\\\\\\\/xJkkOkJ8rkjDxNXZg4gLIqUQswD5buifigJmlAizemnulbM6DjGHDUdBBwi4P1+7OU361kkHYEmpPjv1J37g8M2T83Brw7TY9n2nbK\\\\\\\/0k8nKJ0OmEm8+KiIfEC\\\\\\\/YrXhQzE50QuelmICYbT5beP2QbR3JJ+ZUH\\\\\\\/UKBYLud+CjBOw44\\\\\\\/BHRXXfNyQsxXqte5mp0Isfm29AIOSGtMjyLIIIV8d+Tdc2f+QjAFvmTtRSURXCcF99Fkr4JUXgSmGna3MOk5dppdiLY25+3SUqRgbdD\\\\\\\/ChP7wkw71ysnBqFrcJvSZgt2W0ozvBDMzI5od6mqxqzBWsiHI5DIG9uFdFKRupb4UagBALUOhjjUaLc4QZQSNHhkEWSW0pznQBFrUVOO2j+6\\\\\\\/o06uwFUN3dOzcHX7\\\\\\\/Mjno\\\\\\\/IcVSBvmJbi+zDYer3ULaFjNoLevseQhqTteXJQjA9uvfFbpv5UM\\\\\\\/apab\\\\\\\/yj8ZRfMgTethF5rVHL7+dbxO2vLa67NVXUmGFwpjpT141VicZP1CuoWhx2StRytSbvzPD4A2jBx1riYxdKjQPSsv6U4VdcvtY7FI7rQOF+Wdto+DOaitpitbeOpEXDYK\\\\\\\/PdlQOwdpn+bzMULLTFyUd2TAUG7jeU+juXB\\\\\\\/YQgsXjDIanee6nHhKEubR6QjUHXBunAPqU8TZ9pzE2zgjXIN6wiIRD5KM9vBH684EIBDOyqeO8bTEvuS6AsZ+LlkwJIRs\\\\\\\/A1mZ2S7YksjAx9mluCiVFTbqZAtKji0DWXItlCgfDsBY8gxFGJwBsANGhjrPYNSxdhNcQL\\\\\\\/E10dVlLsp9BB9w9pApsll\\\\\\\/Eaq+EfmfNmkEsbWa8rSSLTthGmQ2uxSsoKtoyRL4AI3\\\\\\\/b+GrZ\\\\\\\/7Rjtjt+J3uXrXeVjxnhCeeSsmbgMdQIWfemXLb2wbt1aZFRRJ46YPCzD4hb6FwZ8Vw0jzJidx0nSJ7MZ062uFDzA1dQ8wFVJDQwaxqX4ksTR5xAmaZxfkeS3xc+rcXqeNOkZhRABS9VQjmlKSv4jdaii3wUqBGKnTxByhqPhYvDx1cXgvRm0KQtCJI1yDRQ5iJTC7aJrZ2E5czS4inwd2hI42x8bUbnNBprW4NqYtxrB8atTD+1qHqb2g\\\\\\\/MXcjkcCbw8RohbgjFjc7izPV2QWxOFmveaU7Q7gGrwBQfi3slCXv733E4g09MxbX4MpttSWzon2mPqJ1acj2+9w16Fpn5SlgeZwOMeT1cFcl4L4bZaYgMJKErH83+S7wuSzXefygG9jU6THvcDLLsV86+V6dbE84Bzo78cpwALxvZzG4apwXAg4r6B8CEv6KfDQcqBpysowUlpwdUujrmHn0ueYLP0zmoYSzJtHCS+O2qQ\\\\\\\/L0ZuIMr0+PU6S5cyy15yP8n+CPbnU6A1RiVBYaoMKy4Kffp12+0rpP17scLm3LFK21eN4GvHgHmKyyARy2Klyz3yoemuoV550\\\\\\\/Q1SzFBq+8nLY7+MZv9PLzkx8lDadjzwinnXHxlAWRnK36Iidatp7KaGTT746L7krKwA\\\\\\\/7IxzSCCBXk+9Fs8Ya8fSknfU7k3LzBkjoTZaonV0dnfQrlCq2XEI4JXA2uykjj1hZ6dvgURk2p3UA9QI2NEL1IcL90Sshk0h3tCR7rfGFCX68ShXPhc1NooCWqAbJ8y2hGVvMin5j3IFYu8GvX3ry+jbW0AuQSWtZcGWR53bXwsqCNhGsuBTSy2Xb6Im29qrJG8w3cg7LKozSuaTYkFaFCzCPmaixE9dkmuO+XaQ0BUsz5dlWS5ObWFqzAqvpjNneiPG0JEqeZClsQZyTkKxhRL0ITrPvmbUR3Zfg2BeOQh1cawCzt56nyK00Z9MWx3F5\\\\\\\/Fg+pQa75ras8vXdKKpppXm2qzWw8ze3vhcfwKy9qKR0kmH6usQQ3dFHEQTosJax5fuGSbusA5+BFp3uTDWfJ\\\\\\\/xZ\\\\\\\/o8Sv0Jok12cATIJais0QXsw+ePBIslaJ2LW0uz6ERLuUzZNJMWiE5ggcYrNnz6Oad04ExU3y5ZCh84beK\\\\\\\/HQSQvqi5FwxJyIguLSnQCbjOQlKohiDN6EJUjmdBW4Ie01DwT0y8z3vsyYU0+IihGbuVRK04z3lzlIGUew98SzBolyNPwf4SZLpthDL0MnYBwLNra+hy9rM9JyRjXmKTy8yH2Zyr1isyaW2lY1nqdlu7rKJdaVKrYc4aVBsr4IK\\\\\\\/iblTfPuvGd5OhcQPPkgNWodzToDT8CJxudDwSiIiGswxueDg0nRyZGdIuuDB7vW8YJ50o482fkFC3suHyfx\\\\\\\/hnzsVIIn2p58YNyKxl0zppAZU0+gmVsrEV+tB1EzJgIKNSLH9dBPQN7J0AnkixuwwhZOMsD6eBhPzq+lg3rvpyWxLbP4tjuU27IEQ\\\\\\\/P4qPwbFXpCBlV0go0dR90kyVxrwZYtOIeLbBFZNm4izKxYZicfFYkYOkx0Yp1LJUtegDwiN+LkyN7019f2Uwx12OO0xwlCWKW5B1JaexazbbNwFV3TSxrBO461BXdtmmh4DeErNsjA5YE9F3tEMcdtMZ43m3JmO+nuWqXXiEGjLqcRphXTjk9kw6H08FjK7miditxavtI1wNwpqh\\\\\\\/lTGLCKUImZG9TFZDtsLNrolKmzF0Ca2I1mZm8As2msy4dVkg6d3gh0aJzpXVtE7X+QCzw4IBmvpCdmTpxykMPMI5Kt5QTmAZTl5RWorACbLc2htnjC47LUUzXcljgAdwYDNPMyYaa+LG0JFYWGzsjovXeTkVPPAZAuFCiIGHKMnZ7+nDmWUKlp02Zo6acGS7N+\\\\\\\/o9WOSez0lycG\\\\\\\/hvd5iSuFBXf23S97i9Uguni0As8fgEIJFz+IZN1whAEXxOoMLHL812Z2Dm2Q0WjbmTG4LgcXY3lD+KJl7WH11tKoeaI9enKenisYiDTUcqlwwUZamvX2d3laKfifSKlO8C444Nk9jdkPeJKnZ8Y5cxax8s66wNfrbu2R+Pd0KZ7XnDhMXhykI2I6GJTKxOKbdc47\\\\\\\/SR+pfZudnLU9WF8lkFF69wx2RlLWNu8p5xoK2PzRJ5O+940sXPfFicR7XIf087LH+\\\\\\\/X9z3YdcyJpUyKBPMWtrPFsljtCvSwoaPgD39EaPzTz9gWXYfvmimqixUeoYInkuumHJkQmxwjhCmRGUifpqozC7FAnSKs0ix2LmTq46Wkq1TJmu5mN1QJNcF7B9j9e\\\\\\\/mqNkv5R0B188UgFtskAjbt8jQkUZ\\\\\\\/SU\\\\\\\/pzpphJFANpijY966fordjhvrVe5xNQE1y9Rg39MLpYjApZk13KBZO1SVp9\\\\\\\/rLDI67cMeCCjUVH3Wc0OYMrruZQEsst0RTe\\\\\\\/JKh6WY15Vhei8XLJaAtws0\\\\\\\/5zXsCL3mXd+aPcR7FWoj9\\\\\\\/amkLe+w8H28DtU4wiGIMLq9z1cAmrrqn7mOTC7hgO96LZUH4yVrnNiLFSTbWVgIQQgfRjL5LCmfJHuENoq9b9KvU6aHvvhNKpxBXA3jIA+D+RbQYtpV1z\\\\\\\/3TweUndthTHRKuTX5MKgNNl7iKKt76kVGjrOJWG\\\\\\\/7Gwk6s5E6UdAI44+7t7VFu91g3\\\\\\\/v2QSv3ipZfHiqcF03JPUSk7a9J0hbnGpQdh0ImsPoSSQjLIuIsiJkLMCteD4elii8oXZDZ02xzJLJeIuuvQgrT6Z8HspSLDHWy8PUZ5SyA\\\\\\\/skyY5rImYw\\\\\\\/yLtaeY3WyietkvSasfKPITiz+JTvEZCT0suEduaQY3iR76ujvvOuYMnnwET1S8UnS3W5iemr\\\\\\\/16sEp8rLbiSkk8NzESacK0H4K+p5SruSC8NnkK8+ImBlhd661F65fkbxf9tPCym+1xm3wlo1UlKFiBekO3K8idrBhlze+IjbD9cuYNJSlr3zI4NYi3PoIJCqbKYtGFkD63+4f53jr3i1IAW2dDTWcnOsK908BWIETGimQ0+l71gd1Rs9+d\\\\\\\/+6cWETn5oveI9p+T9NNjCLf1JM5SJTmTCJDjggPSmTtgXtQkO0N3C5ycuAKMRG7FZ0VsOymgOdnTHrBt8mNvlCXQ4alvZvMmjSscbMePEbeWdHwjyt0wOhNL0OkH74\\\\\\\/grrmZ4DGH9B9XFP53zOy8LDCxrobEssKwg8cMDkRBPJGzCqxjBfVfOIvlkB5S3gf00IOe+ixkyhIrTYGQJNIv1cH\\\\\\\/IPbcAfwg55pvBWvNZYUrnq4oThdZbfvrB3g+EEZAV9vbB\\\\\\\/i0rKkkyN\\\\\\\/ysvG0DjsctSqMZHbWXv7OuC7obDxKrcWRY8obA616e1kIBiPPOhy48FpsIo6d+L+wX30YTYcauYLQH9hxR08F7BB9yDiQRv7itjVHiJo\\\\\\\/AHJKfuw4XFGWf5lAedGxOozyNfpbAg3rUpLETbRK6L9\\\\\\\/zqtk1XPbPITc45Q3u2SyfCMYQbjLmyiG++P5dYyXxuqxyH+vL\\\\\\\/4Mdv8fgOGnuzRaLSrthv7HrmOwC6YkhKoNeKpR2F2f7rtNrPjf6uthwdp8zdwNEM+N0HrAbnyoUbI4mdeBI1sfxTucZsWNb42NkvpANXJjOj4DBHHhyEY1zsRc\\\\\\\/6YSJhhknK8bpgAjxkiyy83PGdVZyDVFCQmGvN7TrmVKXTGLN1E0rk\\\\\\\/bdZ7Cskf94rMgEnrf5FbVundGSK4\\\\\\\/wkfe7M7wkYw6Q4\\\\\\\/Kj\\\\\\\/f9j6bSTpohZwsZmucCPDrAH315ekPYkQPsx7ao9qf1EpNCw57r74\\\\\\\/x\\\\\\\/SFrLZO1EM0q38P\\\\\\\/nYSH77hIhXraUx8\\\\\\\/Vo2TwuS0N0ABNT27TaqilCC6+BHwfyjc9eHyLnJek9hqzxpshD5LRFRKJ\\\\\\\/Ah02LDlksC\\\\\\\/cFHg1PuNcXf9ZB+sbeRXckm2CDqz0OZTG+ptVUjVnOfFRfPoG9HnPBXY8G+blLGM\\\\\\\/EYGyHQa2aAzzMkI2JNjm5I1JYbnVJNLdwVXPC2iVMJ8MyIn+cD5\\\\\\\/IA0j6p6QqB8fUOWeeJmHBKpcd4KcytyPDHYZCqNVFWe5wFGrB7MMs+3t+Sit2ePNPFzmG0yFpCkcpJG9\\\\\\\/BXO8TagF4rQP1EGzBqrgw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022N3VE92t\\\\\\\/P3kIRpUJzrtIgw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ba375e4a9f975d7b198fdd7f4817d8beab40cae8b161a983248dc2928dfe3ea1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tX61LnEUUzMInlH1ezgMbK1w9kdTV1Ugkg\\\\\\\/CKY0paLuOUT7FXJlm33aEUzCkSVWOT9XVzU5OvB7wgy+KOt9JEX9v5JNxVS\\\\\\\/nxlami9jQ\\\\\\\/S4SuAgHiaoxjA7\\\\\\\/LB\\\\\\\/vTLUeq93Csnvq7Bv2WwNw1\\\\\\\/EYQYLN8ndm88svs0MD2jn3qXmz0\\\\\\\/NO\\\\\\\/262Uc5MSGj9hCnAJyUuRAcG\\\\\\\/SMjGR\\\\\\\/OB458WKTBGJyop4hnwyGIm2nZr4GP05OBOFuGxc6d6Q1N6GYvg2IMGFQTsIDV24TXWIOmYAL+o7R3gsf3rT9HSOqoksP+zmTnkFssK+bK1oeggRvi7YhO0E4Tb2r7Ou4Y4WWhOsx85oywC374DIhrY0tbVpq\\\\\\\/xAT8OiW742RujbTfghL4PKR4uO\\\\\\\/d7\\\\\\\/m9tRdLxvOrGraKkN7mi01WEQKW4x\\\\\\\/LMHZFkAiJz757wqD+j6JcZCfn1M9rOuQXAbRu96t92GyRJp\\\\\\\/JPZgv2NR0SKs8Q4j0xr0ExuWdpiBOFoKD37wHJGfQSQXz+FYu5kvA3r6mGXJZ3u034LsCFFm0I7uqDSLoy0TNcb1nNrhTRgaXz\\\\\\\/eZmQ6Jhi3DtwBwpqK3JsS9xKZykSJ\\\\\\\/ekPg3Jf60WlVua7snL58GjWYzNZ9Kloz3+N4wGWkRAhKDyPQUTRU2Bh28gcHX3WQnazjNRIYHfEm0F\\\\\\\/W+rgYZEuQH1G+4qMh\\\\\\\/5RsjO1ROtUE65CCpFQsAwHhgcEJ4fAbsfQecR6T+5MyQb\\\\\\\/8EALp8wM0Yek6rgMROuYkJQQrts13Ic9tXLudvY9egX\\\\\\\/eYHNk9rQZFMvsXQ4saRF9qdVE7mcbAPyHGZHwUVH72yZfjeViNg9RgupBSM00WCrZyPYDsnqLX+0RZUi6XgB+Sc2fIdzuCA4fd95oFyod0xpvXsq1UE+dI\\\\\\\/svggsz3zTp\\\\\\\/0vd1ZX0Y3YrlLA8eWoHY9WzBDYzm6LDWZADlrhDTL4YIsTkN3nxB1WyXUTfxHdHeSn+dLxyIBFS9gI+gERg8KBqhiNR60iyfmc9bXRQ1CaSaESheie2QAIxAQk\\\\\\\/5DOBnYICN7NXPxkEHw+PNBNe+21Fm7ugtv2J2vFkNwcrGrP1pEah3zDo5RU+l9ZPigyuJftxlg4uQsCaZk6LSACFwGSkOCH1WePQgCCUsSUCMif\\\\\\\/yQ5AjIHW5PV7MywQaBAXxbI8zyS7sTBx1L8eZIgVGOuXVp6SofPWR4YiZX8Mc2O4P46C4urr9I+fnNE45xDsDxfigRM\\\\\\\/iU9kfxJhI\\\\\\\/BKtiO75C9x7KfttUQ8+Cpzjev7r\\\\\\\/q7OWunqe4nTxGDGV7E+NQb2oADUdb7YMBL1OYCaJ3tpvIxtRRuklPj7J1+rMxSX4aDb9bo59a3qGI4v0h\\\\\\\/WW\\\\\\\/DRk+shSqCNG7CgcjIWX6W48EGU\\\\\\\/2gzdxYaVQsx9HhEVtuYcGnt7u\\\\\\\/Kff1yCbigkDNqb872TtdyEWKLTreYYN1H51reFhDi9XQ4LN+D4T32MipvQRDxhKHqVu9IsIND2V7fkOlU6BQuJ8nGDP5ii991eNoR3Irfxz3pzf9rW0yn8AYg+l\\\\\\\/\\\\\\\/xJgMv8Bxs6DtM2JW+dxy6eI5s37K+YIgbOqpTwJ6ZLxWDqPo1iP5jryNg0Gz5\\\\\\\/1t7h9W5xA0jLdpowYn3DL8hINuc9mIuom5RIQ+ALp\\\\\\\/OngfaY8u5C6Me\\\\\\\/2uXrrqhANaTrywjtWXA1Afjtb2SsNfbgK0nawjlrLvARbtszX8bwp86+LPSdME+8rQYmhFkFo8B7jejw5onjZffNwToG5nLaSUmZrP\\\\\\\/RQnDRUAC97K3ITW7qnkYrFjGJbynxChx0cbjD49hspXRcVbI7A9+iP4Evvmo0QCxE8WmnoKDqGjTy2br3Q6aS8ddY\\\\\\\/JwnM06O7a0M5UNcS1\\\\\\\/k+fKgvNOO\\\\\\\/tuOKA3H3DFCTVtMMg\\\\\\\/RWcQx9oYeLlIt3IWX7MaySAYmzBP6wXuzJJrunBfYhTYz8RXkYu+z6W9SODW2qeyBOqFX6IDa4k+iEbYEfvyrpEyrMHIaxjG2MuJlpKQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022J9SIZAd9rKkbbZTQ1SEqHQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002233433261483e0b6a5a70fac95c0b3c40175b8d209c4681544beed3312bdcb97d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460353 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1201" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cly4U4A8jZERBzPTEOm7-8WdPU6gT2wj6UIiAnDFoD4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1201" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sWfr1rAn3UAl1P58hfbMo0OPDT6fyHSxZG4R9YogHqk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/5/1191" title="Comparative and Transcriptome Analyses Uncover Key Aspects of Coding- and Long Noncoding RNAs in Flatworm Mitochondrial Genomes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/5/1217" title="The Genetic Linkage Map of the Medicinal Mushroom Agaricus subrufescens Reveals Highly Conserved Macrosynteny with the Congeneric Species Agaricus bisporus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460353" id="node460353" data-pisa="ggg;6/5/1201" data-pisa-master="ggg;g3.115.024547" data-apath="/ggg/6/5/1201.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Massive Amplification at an Unselected Locus Accompanies Complex Chromosomal Rearrangements in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Agnès Thierry</span>, <span class="highwire-citation-author" data-delta="1">Varun Khanna</span> and <span class="highwire-citation-author" data-delta="2">Bernard Dujon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1201-1215;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024547 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460353" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Agnès Thierry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institut Pasteur, Unité de Génétique Moléculaire des Levures, CNRS (UMR3525), Sorbonne Universités, UPMC, Université Paris 06 (UFR927), F-75724 CEDEX 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Agn%C3%A8s%2BThierry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thierry%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAgn%25C3%25A8s%2BThierry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Varun Khanna </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institut Pasteur, Unité de Génétique Moléculaire des Levures, CNRS (UMR3525), Sorbonne Universités, UPMC, Université Paris 06 (UFR927), F-75724 CEDEX 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Varun%2BKhanna%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khanna%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVarun%2BKhanna%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bernard Dujon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institut Pasteur, Unité de Génétique Moléculaire des Levures, CNRS (UMR3525), Sorbonne Universités, UPMC, Université Paris 06 (UFR927), F-75724 CEDEX 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernard%2BDujon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dujon%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABernard%2BDujon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bdujon@pasteur.fr" class="" data-icon-position="" data-hide-link-title="0">bdujon@pasteur.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/5/1201" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460353" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460353/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1201.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460353" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460353/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1201.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460353" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460353/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1201.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460353" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460353/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/5/1201.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460353" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460353/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Gene amplification has been observed in different organisms in response to environmental constraints, such as limited nutrients or exposure to a variety of toxic compounds, conferring them with specific phenotypic adaptations via increased expression levels. However, the presence of multiple gene copies in natural genomes has generally not been found in the absence of specific functional selection. Here, we show that the massive amplification of a chromosomal locus (up to 880 copies per cell) occurs in the absence of any direct selection, and is associated with low-order amplifications of flanking segments in complex chromosomal alterations. These results were obtained from mutants with restored phenotypes that spontaneously appeared from genetically engineered strains of the yeast <em>Saccharomyces cerevisiae</em> suffering from severe fitness reduction. Grossly extended chromosomes (macrotene) were formed, with complex structural alterations but sufficient stability to propagate unchanged over successive generations. Their detailed molecular analysis, including complete genome sequencing, identification of sequence breakpoints, and comparisons between mutants, revealed novel mechanisms causing their formation, whose combined action underlies the astonishing dynamics of eukaryotic chromosomes and their consequences.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/trna-synthetase" class="hw-term hw-article-keyword hw-article-keyword-trna-synthetase" rel="nofollow">tRNA synthetase</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/cup1-amplification" class="hw-term hw-article-keyword hw-article-keyword-cup1-amplification" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">CUP1</em> amplification</a></li><li class="kwd"><a href="/keyword/macrotene-chromosomes" class="hw-term hw-article-keyword hw-article-keyword-macrotene-chromosomes" rel="nofollow">macrotene chromosomes</a></li><li class="kwd"><a href="/keyword/quasi-palindromes" class="hw-term hw-article-keyword hw-article-keyword-quasi-palindromes" rel="nofollow">quasi-palindromes</a></li><li class="kwd"><a href="/keyword/disomy" class="hw-term hw-article-keyword hw-article-keyword-disomy" rel="nofollow">disomy</a></li></ul><p id="p-5">Structural changes and copy number variations are frequently observed alterations of eukaryotic chromosomes, with potentially important evolutionary consequences as well as possible deleterious effects that are able to determine pathological processes in humans (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kloosterman <em>et al.</em> 2015</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zarrei <em>et al.</em> 2015</a>). In the yeast <em>Saccharomyces cerevisiae</em>, low-order amplifications (mostly duplications) of large chromosomal segments are often observed in evolutionary experiments aimed at studying adaptation to limiting environmental conditions (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dunham <em>et al.</em> 2002</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gresham <em>et al.</em> 2008</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">2010</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Araya <em>et al.</em> 2010</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Payen <em>et al.</em> 2014</a>) or the recovery from artificial gene dosage imbalance (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Koszul <em>et al.</em> 2004</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Libuda and Winston 2006</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Payen <em>et al.</em> 2008</a>). These events generate a variety of topological forms, including intra- or interchromosomal segmental duplications, the formation of additional chromosomes (neochromosomes made by the junction of two or more duplicated segments), or independent episomes in circular or linear forms. Higher order gene amplifications were also observed in cultivated mammalian cells exposed to methotrexate (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alt <em>et al.</em> 1978</a>) or in plants exposed to glyphosate (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gaines <em>et al.</em> 2010</a>) with, in the latter case, multiple copies dispersed on the different chromosomes. In <em>S. cerevisiae</em>, expansions of tandem gene arrays were observed long ago at the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus on chromosome VIII in response to exposure to toxic copper salts (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Welch <em>et al.</em> 1983</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Karin <em>et al.</em> 1984</a>) and, more recently, at the <em><a href="http://www.yeastgenome.org/locus/S000002751/overview">HXT6</a>-<a href="http://www.yeastgenome.org/locus/S000002750/overview">HXT7</a></em> locus on chromosome IV in response to glucose limitation (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brown <em>et al.</em> 1998</a>). If dozens of short tandem arrays (typically two to three gene copies) scatter normal genomes of all yeast species (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Dujon <em>et al.</em> 2004</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Despons <em>et al.</em> 2010</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">2011</a>), larger arrays are also occasionally encountered in natural yeast genomes, often showing polymorphic size variation in populations as illustrated, for example, by the <em><a href="http://www.yeastgenome.org/locus/S000002447/overview">ENA1</a>-<a href="http://www.yeastgenome.org/locus/S000002446/overview">ENA2</a>-<a href="http://www.yeastgenome.org/locus/S000002445/overview">ENA5</a></em> locus encoding a P-type ATPase sodium pump in <em>S</em><em>. cerevisiae</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Martinez <em>et al.</em> 1991</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wieland <em>et al.</em> 1995</a>) or by genes encoding α-1,3-mannosyl transferases in the pathogenic yeast <em>Candida glabrata</em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Muller <em>et al.</em> 2009</a>).</p><p id="p-6">In addition to these phenomena, high order amplifications of large chromosomal segments, leading to significant chromosome size expansions (macrotene chromosomes), were recently discovered in phenotypically restored revertants from genetically engineered <em>S. cerevisiae</em> strains (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Thierry <em>et al.</em> 2015</a>). In these strains, the replacement of essential tRNA synthetase genes (RS genes) by their orthologs from another, distantly related yeast species (<em>Yarrowia lipolytica</em>) resulted in severe fitness reduction due the insufficient charge of the pools of tRNA molecules by their cognate amino acids. Experimental evidence indicated that the multiple <em>in loco</em> repeats formed in macrotene chromosomes were the result of single-step massive accidental events during DNA replication, rather than selection-driven, successive illegitimate recombination between copies. In humans, complex genomic rearrangements resulting from single catastrophic events, designated chromothripsis, were also recently identified in cancer cells or in cells of patients suffering from congenital developmental disorders (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stephens <em>et al.</em> 2011</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kloosterman and Cuppen, 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Korbel and Campbell 2013</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zhang <em>et al.</em> 2013</a>). In such cases, pieces from a shattered chromosome arm are stitched together in apparently random order, orientation, and number, producing deletions and duplications. However, to our knowledge, no direct relationship has been reported so far between large chromosomal rearrangements associated with moderate copy number variation and massive, selection-driven gene amplifications. The results presented here establish such a relationship and suggest a mechanism for the formation of such grossly altered chromosomal structures.</p><p id="p-7">Using our previously constructed transgenic <em>S. cerevisiae</em> strains (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Thierry <em>et al.</em> 2015</a>), in which the essential Asn (asparagine) tRNA synthetase gene (<em><a href="http://www.yeastgenome.org/locus/S000001061/overview">DED81</a></em>) was replaced by its ortholog from <em>Y. lipolytica</em>, we now report the spontaneous formation of novel chromosomal structures bearing massive amplifications of the 2 kb-long <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus (up to over 800 copies per cell), associated with complex low-order amplifications and rearrangements of large chromosomal segments affecting one or two distinct chromosome(s). Contrary to previous reports (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Welch <em>et al.</em> 1983</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adamo <em>et al.</em> 2012</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2013</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zhao <em>et al.</em> 2014</a>), the amplification of the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus in our mutants is not only much greater (by at least one order of magnitude) but is also not selected for, based on copper resistance. The concomitant formation of massively amplified <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> loci with the low-order amplifications of large chromosomal segments, which include the Asn-RS transgene on which the phenotypic selection is based, illustrates how catastrophic alterations occur in normal eukaryotic chromosomes. We report here the detailed analysis of such events and propose underlying mechanisms based on genome comparisons between different mutants in two independent evolutionary experiments.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Culture conditions</h3><p id="p-8">Yeast strains were grown on YPD medium (Yeast extract 10 g/L, Bacto peptone 10 g/L, glucose 20 g/L, with or without 25 g/L bacto-agar as needed) at 30°, unless otherwise indicated.</p></div><div id="sec-3" class="subsection"><h3>Sporulation and parental strain construction</h3><p id="p-9">Tetrads from diploid strains BYAT580-0 and BYAT580-200 (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Thierry <em>et al.</em> 2015</a>) were micromanipulated using Singer MSM equipment. Ascospores were inoculated on thin YPD agar medium and incubated at 30° for 3 days, following which the agar was placed on top of new YPD plates to ensure sufficient nutrient availability for slow-growing strains, and further incubated for several days (see Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS1.pdf">Figure S1</a>). Selected haploid segregants were immediately grown in patches on YPD medium for a limited number of generations. Cells were stored at –80° (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-4" class="subsection"><h3>Pulsed-field gel electrophoresis (PFGE)</h3><p id="p-10">Blocks of native chromosomes were prepared from agarose-embedded yeast cells according to standard methods (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Schwartz and Cantor 1984</a>). PFGE was run on 1% agarose gels, in 0.25 × TBE buffer at pH 8.3 at 12°, and 5 V/cm for 65 hr on a Rotaphor (Biometra) with an alternating field angle of 120° and pulse ramps of 300–100 sec (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) or 180–80 sec (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, B–E</a>, <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, and <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>), with linear accelerations. PFGE of <em>Eco</em>RI or <em>Bam</em>HI digests were run on 1% agarose gels, in 0.25 × TBE buffer at pH 8.3 at 12°, and 6 V/cm for 30 hr on a Rotaphor (Biometra) with an alternating field angle of 110–95° and a pulse ramp of 100–20 sec with logarithmic acceleration (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Southern blots of PFGE were hybridized using digoxigenin-labeled probes, according to the manufacturer’s protocol (Roche, <a href="https://lifescience.roche.com/shop/en/fr/products/detection-of-dig-labeled-probes">https://lifescience.roche.com/shop/en/fr/products/detection-of-dig-labeled-probes</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pulsed-field gel electrophoresis (PFGE) of evolved mutants. Electrophoresis was conducted as described in Materials and Methods. Left: ethidium bromide fluorescence, size scales in Mb calibrated from migration of natural S. cerevisiae chromosomes. Right: hybridization with indicated probes. (A) Evolved mutants from strain BYAT3C. (B&#x2013;E) Evolved mutants from strain BYAT8A (plus parental strain BYAT521 and mutant BYAT711 used as controls). Note that BYAT722 was probably a mixed colony. (Top box) Approximate positions of probes along chromosomes VIII and XVI (coordinates in kb, see Table S3 for details). [C], normal chromosome VIII; [D], normal chromosome XVI; [M], macrotene chromosomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-409920684" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pulsed-field gel electrophoresis (PFGE) of evolved mutants. Electrophoresis was conducted as described in &lt;em&gt;Materials and Methods&lt;/em&gt;. Left: ethidium bromide fluorescence, size scales in Mb calibrated from migration of natural &lt;em&gt;S. cerevisiae&lt;/em&gt; chromosomes. Right: hybridization with indicated probes. (A) Evolved mutants from strain BYAT3C. (B&#x2013;E) Evolved mutants from strain BYAT8A (plus parental strain BYAT521 and mutant BYAT711 used as controls). Note that BYAT722 was probably a mixed colony. (Top box) Approximate positions of probes along chromosomes VIII and XVI (coordinates in kb, see Table S3 for details). [C], normal chromosome VIII; [D], normal chromosome XVI; [M], macrotene chromosomes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1201/F1.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/5/1201/F1.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468167" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Pulsed-field gel electrophoresis (PFGE) of evolved mutants. Electrophoresis was conducted as described in <em>Materials and Methods</em>. Left: ethidium bromide fluorescence, size scales in Mb calibrated from migration of natural <em>S. cerevisiae</em> chromosomes. Right: hybridization with indicated probes. (A) Evolved mutants from strain BYAT3C. (B–E) Evolved mutants from strain BYAT8A (plus parental strain BYAT521 and mutant BYAT711 used as controls). Note that BYAT722 was probably a mixed colony. (Top box) Approximate positions of probes along chromosomes VIII and XVI (coordinates in kb, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/TableS3.pdf">Table S3</a> for details). [C], normal chromosome VIII; [D], normal chromosome XVI; [M], macrotene chromosomes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Evolutionary experiments</h3><p id="p-12">Subclones from parental strains were inoculated in YPD liquid medium and grown at 30° by serial transfer for a total of ∼30–35 successive generations (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS1.pdf">Figure S1</a>). Each transfer was made in 2 L fresh medium with inoculums of 10<sup>9</sup> cells. Under such conditions, each culture represented ∼8 generations (<a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Thierry <em>et al.</em> 2015</a>). After each culture, cells were diluted and plated on YPD solid medium to monitor the growth and morphology of the resulting colonies. Faster growing mutants (larger colonies) were selected for molecular analysis.</p></div><div id="sec-6" class="subsection"><h3>DNA sequencing</h3><p id="p-13">Total DNA purified from yeast cultures was fragmented using a Bioruptor (Diagenode) and libraries were prepared using the TruSeq DNA sample preparation method (Illumina, TruSeq cluster SR v3 and TruSequation 50 cycles SBS v3). Sequences were read on HiSeq2000 (Illumina) at high coverage (384–834 ×).</p></div><div id="sec-7" class="subsection"><h3>Sequence analysis</h3><p id="p-14">Bioinformatics analysis followed the pipeline illustrated by <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS5.pdf">Figure S5</a>. Regions of problematic base calling (deviation from Chargaff’s rules) were visualized by FastQC v0.10.1 (Babraham Bioinformatics, <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>) and trimmed off using Trimmomatic v0.30 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolger <em>et al.</em> 2014</a>). Trimmed reads were then aligned against reference sequences of the 16 chromosomes plus the mitochondrial DNA of <em>S. cerevisiae</em> S288c (Genbank <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_001133&amp;atom=%2Fggg%2F6%2F5%2F1201.atom">NC_001133</a> to <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_001148&amp;atom=%2Fggg%2F6%2F5%2F1201.atom">NC_001148</a>,  GenBank <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_001224&amp;atom=%2Fggg%2F6%2F5%2F1201.atom">NC_001224</a>, PLN 06-DEC-2008) and the Asn-RS gene of <em>Y. lipolytica</em> (<em>YALI0E05005g</em>), using the single end mapping mode of BWA v0.6.2 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li and Durbin 2009</a>) with default parameters. Output SAM files were converted to BAM files using SAMtools v0.1.18 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li <em>et al.</em> 2009</a>). Sequencing coverage at each position was computed by BEDtools v2.17.0 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Quinlan and Hall 2010</a>) and compared to median coverage over the entire genome (normalized to 1 for haploid cells) to calculate copy number per cell. When necessary, coverages were smoothed using 1500 nucleotide (nt) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) or 5000 nt sliding windows (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS2.pdf">Figure S2</a>). Copy number plots were drawn using R version 3.1.2 (<a href="http://www.R-project.org/">http://www.R-project.org/</a>). Tablet (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Milne <em>et al.</em> 2013</a>) and IGV <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">(Robinson <em>et al.</em>, 2011)</a> were used to visualize alignments between the BAM files and reference sequences and to screen for the possible presence of nucleotide substitutions and indels at critical loci (see <em>Discussion</em>). In order to identify novel junction sequences corresponding to breakpoints, all unmapped reads against the reference sequence (extracted using SAMtools v0.1.18 with the option “view -f 4”) were compiled into a SAM file, converted to a Fastq file using the Picard v1.81 “SamToFastq.jar” tool (<a href="http://picard.sourceforge.net/">http://picard.sourceforge.net/</a>), and then submitted to reassembly using Spades 3.0.0 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nurk <em>et al.</em> 2013</a>) with the kmer option “-k 21,51,71.” The resulting contigs with sufficient sequence coverage (corresponding to &gt; 0.5 copy per genome) were manually inspected to identify possible breakpoints against the reference sequence.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Amplicons identified from genome sequencing coverages. (A) Copy number variation along chromosome VIII, as determined from sequence coverage of evolved mutants (see Materials and Methods). For mutant numbering, refer to Table S1. Copy numbers (ordinates) were normalized to 1 (for haploid) using median coverage of other chromosomes (see Figure S2). Curves were generated using R version 3.1.2 (http://www.R-project.org) and smoothed over 1500 nucleotide sliding windows (fluctuations correlate with GC %). (B) Copy number variation along the left-end segment of chromosome XVI. Horizontal colored bars symbolize amplicons (see text and Figure 5). Abrupt curve drops correspond to the Y. lipolytica Asn-RS gene (purple diamond) absent from the S. cerevisiae reference sequence. Pink oval: centromere. Light green background highlights the CUP1 locus (cut for drawing clarity, see Figure 3). Central windows zoom to the 13 kb-long right subtelomeric segment of chromosome VIII beyond coordinate 540 kb (curves smoothed using 200 nucleotide sliding windows, see text for interpretation). Asn-RS, asparagine tRNA synthetase; Chr, chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-409920684" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Amplicons identified from genome sequencing coverages. (A) Copy number variation along chromosome VIII, as determined from sequence coverage of evolved mutants (see &lt;em&gt;Materials and Methods&lt;/em&gt;). For mutant numbering, refer to Table S1. Copy numbers (ordinates) were normalized to 1 (for haploid) using median coverage of other chromosomes (see Figure S2). Curves were generated using R version 3.1.2 (http://www.R-project.org) and smoothed over 1500 nucleotide sliding windows (fluctuations correlate with GC %). (B) Copy number variation along the left-end segment of chromosome XVI. Horizontal colored bars symbolize amplicons (see text and Figure 5). Abrupt curve drops correspond to the &lt;em&gt;Y. lipolytica&lt;/em&gt; Asn-RS gene (purple diamond) absent from the &lt;em&gt;S. cerevisiae&lt;/em&gt; reference sequence. Pink oval: centromere. Light green background highlights the &lt;em&gt;CUP1&lt;/em&gt; locus (cut for drawing clarity, see Figure 3). Central windows zoom to the 13 kb-long right subtelomeric segment of chromosome VIII beyond coordinate 540 kb (curves smoothed using 200 nucleotide sliding windows, see text for interpretation). Asn-RS, asparagine tRNA synthetase; Chr, chromosome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1201/F2.medium.gif" width="349" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/5/1201/F2.medium.gif" width="349" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468029" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-15" class="first-child">Amplicons identified from genome sequencing coverages. (A) Copy number variation along chromosome VIII, as determined from sequence coverage of evolved mutants (see <em>Materials and Methods</em>). For mutant numbering, refer to <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/TableS1.pdf">Table S1</a>. Copy numbers (ordinates) were normalized to 1 (for haploid) using median coverage of other chromosomes (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS2.pdf">Figure S2</a>). Curves were generated using R version 3.1.2 (<a href="http://www.R-project.org">http://www.R-project.org</a>) and smoothed over 1500 nucleotide sliding windows (fluctuations correlate with GC %). (B) Copy number variation along the left-end segment of chromosome XVI. Horizontal colored bars symbolize amplicons (see text and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Abrupt curve drops correspond to the <em>Y. lipolytica</em> Asn-RS gene (purple diamond) absent from the <em>S. cerevisiae</em> reference sequence. Pink oval: centromere. Light green background highlights the <em>CUP1</em> locus (cut for drawing clarity, see <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Central windows zoom to the 13 kb-long right subtelomeric segment of chromosome VIII beyond coordinate 540 kb (curves smoothed using 200 nucleotide sliding windows, see text for interpretation). Asn-RS, asparagine tRNA synthetase; Chr, chromosome.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of the CUP amplifications. (A) Map of the CUP1 locus on chromosome VIII deduced from the reference S288c sequence. The 2 kb repeat units are delimited by the vertical dotted red lines. Gray arrowhead boxes: protein-coding genes, void light blue boxes: replication origins (ARS810). The CUP1 genes are colored in green and the antisense transcripts RUF5-2 are materialized by the dark blue arrows. Sizes indicated for EcoRI and BamHI restriction fragments correspond to the parental strains used in this work, containing 16 copies of the repeat unit. (B) Copy number variation over the CUP1 locus, as determined from sequence coverage of evolved mutants. Same legend as Figure 2. (C) Pulsed-field gel electrophoresis (see Materials and Methods) of restriction digests of total DNA from evolved mutants. Top left: ethidium bromide fluorescence, size scale in Mb calibrated from migration of normal S. cerevisiae chromosomes (size markers lane). Right: hybridization with the CUP probe (below: shorter exposure of selected area to facilitate measurements of hybridizing bands, size marks repeated on right). Control strain: BYAT521. [N]: restriction fragment of normal CUP1 locus; [E] restriction fragments of expanded CUP1 loci. Bottom left insert: migration of hybridizing bands of evolved mutants was reported on the calibration curve (abscissae: migration in cm, ordinate: size in kb) for precise size determination (see Table 2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-409920684" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of the CUP amplifications. (A) Map of the &lt;em&gt;CUP1&lt;/em&gt; locus on chromosome VIII deduced from the reference S288c sequence. The 2 kb repeat units are delimited by the vertical dotted red lines. Gray arrowhead boxes: protein-coding genes, void light blue boxes: replication origins (ARS810). The &lt;em&gt;CUP1&lt;/em&gt; genes are colored in green and the antisense transcripts &lt;em&gt;RUF5-2&lt;/em&gt; are materialized by the dark blue arrows. Sizes indicated for &lt;em&gt;Eco&lt;/em&gt;RI and &lt;em&gt;Bam&lt;/em&gt;HI restriction fragments correspond to the parental strains used in this work, containing 16 copies of the repeat unit. (B) Copy number variation over the &lt;em&gt;CUP1&lt;/em&gt; locus, as determined from sequence coverage of evolved mutants. Same legend as Figure 2. (C) Pulsed-field gel electrophoresis (see &lt;em&gt;Materials and Methods&lt;/em&gt;) of restriction digests of total DNA from evolved mutants. Top left: ethidium bromide fluorescence, size scale in Mb calibrated from migration of normal &lt;em&gt;S. cerevisiae&lt;/em&gt; chromosomes (size markers lane). Right: hybridization with the CUP probe (below: shorter exposure of selected area to facilitate measurements of hybridizing bands, size marks repeated on right). Control strain: BYAT521. [N]: restriction fragment of normal &lt;em&gt;CUP1&lt;/em&gt; locus; [E] restriction fragments of expanded &lt;em&gt;CUP1&lt;/em&gt; loci. Bottom left insert: migration of hybridizing bands of evolved mutants was reported on the calibration curve (abscissae: migration in cm, ordinate: size in kb) for precise size determination (see Table 2).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1201/F3.medium.gif" width="440" height="377"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/5/1201/F3.medium.gif" width="440" height="377"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468313" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-16" class="first-child">Analysis of the CUP amplifications. (A) Map of the <em>CUP1</em> locus on chromosome VIII deduced from the reference S288c sequence. The 2 kb repeat units are delimited by the vertical dotted red lines. Gray arrowhead boxes: protein-coding genes, void light blue boxes: replication origins (ARS810). The <em>CUP1</em> genes are colored in green and the antisense transcripts <em>RUF5-2</em> are materialized by the dark blue arrows. Sizes indicated for <em>Eco</em>RI and <em>Bam</em>HI restriction fragments correspond to the parental strains used in this work, containing 16 copies of the repeat unit. (B) Copy number variation over the <em>CUP1</em> locus, as determined from sequence coverage of evolved mutants. Same legend as <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. (C) Pulsed-field gel electrophoresis (see <em>Materials and Methods</em>) of restriction digests of total DNA from evolved mutants. Top left: ethidium bromide fluorescence, size scale in Mb calibrated from migration of normal <em>S. cerevisiae</em> chromosomes (size markers lane). Right: hybridization with the CUP probe (below: shorter exposure of selected area to facilitate measurements of hybridizing bands, size marks repeated on right). Control strain: BYAT521. [N]: restriction fragment of normal <em>CUP1</em> locus; [E] restriction fragments of expanded <em>CUP1</em> loci. Bottom left insert: migration of hybridizing bands of evolved mutants was reported on the calibration curve (abscissae: migration in cm, ordinate: size in kb) for precise size determination (see <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-17">Strains are available upon request (athierry@pasteur.fr). DNA sequence data for BYAT711, BYAT725, BYAT721, BYAT722, BYAT723, BYAT724, BYAT726, BYAT727, and BYAT729 have been deposited in the European Nucleotide Archive (<a href="http://www.ebi.ac.uk/ena/data/view/PRJEB8937)">http://www.ebi.ac.uk/ena/data/view/PRJEB8937)</a> under the accession codes ERS688167, ERS688168, ERS688169, ERS688170, ERS688171, ERS688172, ERS688173, ERS688174, and ERS688175, respectively.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Extensive chromosomal size variation and disomy in evolved mutants</h3><p id="p-18">The delicate construction of the transgenic parental strains used in this work and the selection of evolved mutants are detailed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FileS1.pdf">File S1</a> and summarized by <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS1.pdf">Figure S1</a>. In brief, two haploid strains, <em>BYAT3C</em> and <em>BYAT8A</em>, in which the essential Asn tRNA synthetase gene (<em><a href="http://www.yeastgenome.org/locus/S000001061/overview">DED81</a></em>) on chromosome VIII was replaced by its ortholog <em>YALI0E05005g</em> from <em>Y. lipolytica</em> (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/TableS1.pdf">Table S1</a>), were used to start the experiments after having verified their normal genomic structures (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/TableS2.pdf">Table S2</a>). The two strains were meiotic segregants of previously characterized diploid strains (<a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Thierry <em>et al.</em> 2015</a>) and exhibited extremely reduced growth rates on rich glucose medium, as expected. They were immediately stored at –80° after the smallest possible number of generations after meiosis to prevent the accumulation of undesirable mutants before the start of the experiments. Evolution experiments were carried out by serial transfer in rich glucose medium from subclones of each parental strain (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS1.pdf">Figure S1</a>). Eight evolved mutants were isolated from each experiment by picking the largest colonies among the heterogeneous populations that rapidly formed during cultivation. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/TableS1.pdf">Table S1</a> lists the parental strains and evolved mutants analyzed in this work, with their origin and genotype.</p><p id="p-19">Evolved mutants were first analyzed by PFGE, showing surprising results (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Instead of the normal chromosome VIII migrating at its 563 kb size (as observed in the control), each mutant showed two bands hybridizing with the <em>YALI</em>-Asn-RS probe; an unexpected result given the fact that the original strains were haploid and that all mutants remained haploid (see below). Furthermore, the two evolutionary experiments gave strikingly different results. All eight mutants from strain <em>BYAT3C</em> showed a similar pattern, as if an earlier mutational event had propagated unchanged in the entire population (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>). Their shorter band corresponded in size to the normal chromosome VIII, whereas their longer band (note the more intense hybridization) migrated at ∼1090 kb, <em>i.e.</em>, nearly twice the size of the normal chromosome VIII, a figure reminiscent of the macrotene chromosomes previously described (<a id="xref-ref-47-6" class="xref-bibr" href="#ref-47">Thierry <em>et al.</em> 2015</a>). Mutant BYAT711 was selected for all subsequent analyses (see below), assuming that the other mutants were identical. In contrast, the eight mutants from strain <em>BYAT8A</em> showed extensive diversity, and none were similar to the first set of mutants (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>). They all lost the normal size chromosome VIII and, instead, exhibited two larger bands (again reminiscent of macrotene chromosomes) hybridizing with the <em>YALI</em>-Asn-RS probe (note the intense hybridization in both cases), ranging in size from 890–1530 kb, <em>i.e.</em>, 1.6–2.7 times the size of the normal chromosome VIII (for precise sizes, refer to <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468210/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468210/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Size of hybridizing bands of undigested chromosomal DNA on pulsed-field gel electrophoresis</span><div class="sb-div caption-clear"></div></div></div><p id="p-23">As a first attempt to identify the nature of these novel macrotene chromosomes, we used a probe (YH240) marking the right arm of chromosome VIII at coordinates 239–240 kb (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/TableS3.pdf">Table S3</a>). The results, shown in <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a>, indicated similar hybridization intensities for the normal size chromosome VIII and the macrotene chromosome in BYAT711, suggesting disomy. In contrast, for all mutants derived from <em>BYAT8A</em>, only the larger of the two macrotene chromosomes hybridized with the probe, indicating that, although larger than normal, the smaller one did not contain the entire chromosome VIII.</p></div><div id="sec-11" class="subsection"><h3>Amplifications of chromosomal segments deduced from whole-genome sequences</h3><p id="p-24">To examine the unexpected genomic structures of our evolved mutants, we sequenced each of them at high coverage and analyzed the sequences as indicated in <em>Materials and Methods</em>. When mapped against the reference S288c genome, sequencing coverage proved homogenous over entire genomes, indicating an absence of copy number alteration except for chromosome VIII (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS2.pdf">Figure S2</a>). In addition, a short segment of chromosome XVI corresponding to the left subtelomeric regions was also amplified in seven of the eight mutants of <em>BYAT8A</em> (BYAT725 remained normal). Therefore, we concluded that our mutants remained haploid, like their parental strains, but exhibited segmental amplifications. Only mutant BYAT711 showed complete chromosome VIII disomy, as judged from double sequencing coverage of its right arm beyond the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus (consistent with the hybridization of both bands with probe YH240, see <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1C</a>). Detailed examination of the amplified regions along chromosomes VIII and XVI revealed a set of interesting amplifications, out of which several amplicons could be precisely defined (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-25">For mutant BYAT711, in addition to disomy, the sequencing coverage over chromosome VIII further increased in the segment between the centromere and the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus (for the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus, see next paragraph). Detailed examination of sequence reads in this segment using Tablet (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Milne <em>et al.</em> 2013</a>) suggested the presence of two partially overlapping amplicons (VIII-D and VIII-E), each duplicated, thus raising total copy numbers to three (coordinates 116.5–133.8 kb) and then four (from coordinate 133.8 kb to the left side of the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus). These amplicons are large (93.5 kb and 76 kb for VIII-D and VIII-E, respectively), and include the Asn-RS locus among numerous flanking genes. The copy number increase from one to four of the <em>Y. lipolytica</em> RS gene is sufficient to explain the phenotypic recovery of mutant BYAT711 compared to its parent (as previously determined in similar experiments by <a id="xref-ref-47-7" class="xref-bibr" href="#ref-47">Thierry <em>et al.</em> 2015</a>).</p><p id="p-26">In contrast, for all mutants derived from <em>BYAT8A</em>, the right part of chromosome VIII after the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus remained haploid (again consistent with hybridizations using the YH240 probe, see <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1C</a>), whereas the left arm showed double sequence coverage and the interval between the centromere and the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus was further amplified by the presence of extra copies of amplicons VIII-D and VIII-E, as above (except for strain BYAT725, where the latter was replaced by amplicon VIII-F with a distinct left boundary, see below). Precise amplicon limits were again defined by Tablet analysis. Additional copies of these amplicons raised copy numbers to three (coordinates 116.5–133.8 kb, as above) and five for the segment between coordinate 133.8 kb and the left side of the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus (see next paragraph for the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus). These numbers correspond to two additional copies in excess of haploid number for amplicon VIII-E (VIII-F in the case of BYAT725), and one additional copy in excess of haploid number for amplicons VIII-C and VIII-D. Careful examination of sequence coverage along chromosome VIII in these mutants also revealed that their right 13 kb-long subtelomeric segments after <em>YHRC</em>δ<em>16</em> (from coordinate 549.5 kb to the telomere) were entirely deleted (central inserts in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Note that interpretation of this result was complicated by the fact that an exactly identical sequence (except for the absence of the <em><a href="http://www.yeastgenome.org/locus/S000001259/overview">IMD2</a>)</em> exists in the right subtelomeric segment of chromosome I. Consequently, sequence reads originating from chromosome I were erroneously mapped to chromosome VIII, obscuring the deletion (note the coverage lower than one copy and the complete absence of reads corresponding to <em><a href="http://www.yeastgenome.org/locus/S000001259/overview">IMD2</a></em>). Finally, the 56.5 kb-long left subtelomeric region of chromosome XVI was also observed with a total sequencing coverage of three in the seven mutants of this series bearing amplicon VIII-E (but not in BYAT725 bearing amplicon VIII-F), suggesting two additional copies of amplicon XVI-A in excess of haploid number (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>). Surprisingly, the seven mutants from BYAT8A sharing amplicons VIII-E and XVI-A appeared indistinguishable from one another by sequence coverage, whereas they showed macrotene chromosomes of varying sizes (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1</a>), suggesting that the differences originated from the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> amplifications.</p></div><div id="sec-12" class="subsection"><h3>High-order multiplication of the CUP amplicon</h3><p id="p-27">Sequencing coverage over the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus showed the astonishing result of ∼100 (mutant BYAT711) to ∼450-fold excess (mutants BYAT724 and BYAT727) compared to haploid level (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). To interpret this result, is it useful to recall that two copies of the 2 kb repeat unit containing the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> gene and its flanking sequences were arbitrarily included in the reference genome sequence of strain S288c. Our evolved mutants must, therefore, bear ∼200–900 copies of the 2 kb <em>CUP</em> amplicon (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) compared to 16 copies for the parental strains (8 × coverage over the two loci of the reference sequence, not shown). The <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus was known to vary in repeat number between strains in relation to tolerance to copper salt toxicity (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Welch <em>et al.</em> 1983</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2013</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Zhao <em>et al.</em> 2014</a>, see <em>Discussion</em>) but, to our knowledge, the data obtained here exceed previous numbers by at least one order of magnitude.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468235/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468235/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Numbers of <em>CUP</em> amplicons in evolved mutants</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">In order to examine the relationship between the multiplication of the <em>CUP</em> amplicon, not directly selected for in our experiments (copper salts were never used), and the low multiplicity of the large amplicons bearing the <em>YALI</em> Asn-RS gene selected for in our experiments, we first hybridized the PFGE-separated chromosomes of evolved mutants with a <em>CUP</em> probe (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1D</a>). In each mutant from BYAT8A, the two macrotene chromosomes showed high hybridization intensities with the probe, indicating that amplifications of the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> repeats occurred <em>in loco</em> within each altered chromosome VIII copy. For BYAT711, the macrotene chromosome also showed high hybridization intensity with the CUP probe, also indicating <em>in loco</em> amplification, whereas the normal-size chromosome VIII showed normal hybridization intensity (comparable to the control), suggesting the absence of amplification.</p><p id="p-30">The absence of <em>Eco</em>RI and <em>Bam</em>HI sites within the 2 kb <em>CUP</em> repeat unit (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a>) offered us a direct method to determine the number of loci containing <em>CUP</em> amplicons within the macrotene chromosomes and, for each locus, to directly measure the number of repeats. A normal band migrating at ∼35 kb (<em>Eco</em>RI) or ∼40 kb (<em>Bam</em>HI) and hybridizing at normal intensity with the probe was observed in each evolved mutant (as well as in the control strain), indicating that the natural <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus was preserved in all mutants with the same number of repeats (16) as their parental strains (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3C</a>). However, much larger restriction fragments, with more intense hybridization signals, were also observed in the mutants. Their migration on PFGE indicated sizes ranging from ∼250–670 kb (for precise measurements, see <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). For BYAT711, only one such band of ∼250 kb (<em>Eco</em>RI) was observed, whereas two such bands, ranging in size from ∼315–670 kb, were present in the other mutants (the two bands comigrated in BYAT721). From these data, we concluded that a single new <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus containing ∼124 repeats was present within the <em>macrotene</em> chromosome VIII of BYAT711, either in place of the normal <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus or in addition to it (the mutant is disomic and it is impossible to tell whether the hybridization level of the normal band corresponds to one or two copies). For the other mutants, we concluded that a single new <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus was present in each of the two <em>macrotene</em> chromosomes, again either in place of the normal <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus or in addition to it (so long as at least one normal locus remained present in order to account for the presence of the normal band). From size measurement of restriction fragments, these new <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> loci must contain from ∼157–335 repeats (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-31">It is perhaps interesting to note that the copy numbers of the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> amplicon, estimated from the <em>Eco</em>RI and <em>Bam</em>HI digests, were systematically lower (by roughly a third) than those deduced from sequencing coverage. Beside intrinsic technical biases that remain to be quantified, the difference may suggest the presence of extra copies of the CUP amplicon in our mutants, independent from the chromosomes, which may result from an intrinsic instability of the expanded <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> loci in macrotene chromosomes (tandem repeats are bound to loop-out by recombination in <em>S. cerevisiae</em>). The presence of hybridizing material in the slots and the important smears observed in both <em>Eco</em>RI and <em>Bam</em>HI digests (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3C</a>) support these hypotheses. Quantification of the DNA material in the smears from hybridization signals was, however, impossible given the use of nonradioactive probes. Despite this numerical uncertainty about copy number, our evolved mutants have much higher numbers of <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> repeats than any previously described <em>S. cerevisiae</em> strains, raising questions about the formation and structure of these novel macrotene chromosomes.</p></div><div id="sec-13" class="subsection"><h3>Novel junctions identified from whole-genome sequences</h3><p id="p-32">All of the above data converge to indicate the presence of grossly altered chromosomal structures, the precise mapping of which is necessary to understand their origin. In a first step in this direction, we searched for novel junctions in the sequences of evolved mutants that could be indicative of chromosomal breakpoints. To do so, all unmapped reads of each mutant were assembled as described in the <em>Materials and Methods</em>, and new contigs with sufficient numbers of reads were compared to the reference sequence. To our surprise, we discovered quasi-palindromic sequences corresponding to the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus in all mutants. Each quasi-palindromic sequence was supported by a very large number of sequence reads (633–1494), leaving no possible doubt about their existence (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS3.pdf">Figure S3</a>). Three distinct sequences were found in total, each specific to one or to a series of mutants (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Two of them (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, A and C</a>) correspond to the junction between the two opposite DNA strands within <em><a href="http://www.yeastgenome.org/locus/S000001098/overview">RSC30</a></em> or more likely within one copy of <em><a href="http://www.yeastgenome.org/locus/S000001096/overview">YHR054c</a></em>, leaving a single strand loop of 38 nt (mutant BYAT711) or 63 nt (all mutants of the BYAT721–729 series, except BYAT725). These junctions occurred without sequence homology and probably correspond to an inverted template switching during the replication of one or a few repeated copies of the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus. The third quasi-palindromic sequence, only observed in BYAT725, corresponds to the junction between two short (16 bp), nearly identical sequences in opposite orientation within the 2 kb repeat unit, the first one within <em><a href="http://www.yeastgenome.org/locus/S000001098/overview">RSC30</a></em> (or <em><a href="http://www.yeastgenome.org/locus/S000001096/overview">YHR054c</a></em>) and the second one upstream of the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> gene itself (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). These two motifs are separated from each other by 745 nt on the reference sequence, leaving a nonpalindromic segment of equivalent size in the junction. We have verified that each quasi-palindromic sequence was not only specific to one type of mutant (not found in others) but also absent from genomes of other strains previously sequenced at high coverage (BYAT580-345, BYAT581-345, and BYAT583-345, <a id="xref-ref-47-8" class="xref-bibr" href="#ref-47">Thierry <em>et al.</em> 2015</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Novel junction sequences within the CUP amplicon. The map of the CUP1 locus in the reference sequence is reproduced on top of each panel (same legend as Figure 3) with thick red arrows to locate the novel junctions discovered in each evolved mutant from whole-genome sequencing (reproduced in gray shade in the second repeat unit, for memory). For sequence reads across junctions, refer to Figure S3. The resulting palindromic structures formed are cartooned at the bottom of each panel (flanking units are in opposite orientation, red arrows). (A) The quasi-palindromic sequence identified in BYAT711 joined the two opposite DNA strands in YHR054c. Red letters: palindromic sequence, green letters: 38 nucleotides-long nonpalindromic interval. (B) The quasi-palindromic sequence identified in BYAT725 occurred between two 16-nucleotides-long, nearly identical inverted sequences in YHR054c and ARS810 (red letters, sequence differences between the two loci in blue). These two sequences are separated from each other by 745 nucleotides on the reference sequence. (C) The junction sequence identified in mutants BYAT-721, -722, -723, -724, -726, -727, and -729 joined the two opposite DNA strands at the beginning of YHR054c. Red letters: palindromic sequence, green letters: 63 nucleotides-long nonpalindromic interval. Note the single mismatch within the palindromic sequence (blue letters). CN, total copy number of the junction sequence in the cell as determined from sequence coverage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-409920684" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Novel junction sequences within the CUP amplicon. The map of the &lt;em&gt;CUP1&lt;/em&gt; locus in the reference sequence is reproduced on top of each panel (same legend as Figure 3) with thick red arrows to locate the novel junctions discovered in each evolved mutant from whole-genome sequencing (reproduced in gray shade in the second repeat unit, for memory). For sequence reads across junctions, refer to Figure S3. The resulting palindromic structures formed are cartooned at the bottom of each panel (flanking units are in opposite orientation, red arrows). (A) The quasi-palindromic sequence identified in BYAT711 joined the two opposite DNA strands in &lt;em&gt;YHR054c&lt;/em&gt;. Red letters: palindromic sequence, green letters: 38 nucleotides-long nonpalindromic interval. (B) The quasi-palindromic sequence identified in BYAT725 occurred between two 16-nucleotides-long, nearly identical inverted sequences in &lt;em&gt;YHR054c&lt;/em&gt; and ARS810 (red letters, sequence differences between the two loci in blue). These two sequences are separated from each other by 745 nucleotides on the reference sequence. (C) The junction sequence identified in mutants BYAT-721, -722, -723, -724, -726, -727, and -729 joined the two opposite DNA strands at the beginning of &lt;em&gt;YHR054c&lt;/em&gt;. Red letters: palindromic sequence, green letters: 63 nucleotides-long nonpalindromic interval. Note the single mismatch within the palindromic sequence (blue letters). CN, total copy number of the junction sequence in the cell as determined from sequence coverage.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1201/F4.medium.gif" width="291" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/5/1201/F4.medium.gif" width="291" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468058" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Novel junction sequences within the CUP amplicon. The map of the <em>CUP1</em> locus in the reference sequence is reproduced on top of each panel (same legend as <a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a>) with thick red arrows to locate the novel junctions discovered in each evolved mutant from whole-genome sequencing (reproduced in gray shade in the second repeat unit, for memory). For sequence reads across junctions, refer to <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS3.pdf">Figure S3</a>. The resulting palindromic structures formed are cartooned at the bottom of each panel (flanking units are in opposite orientation, red arrows). (A) The quasi-palindromic sequence identified in BYAT711 joined the two opposite DNA strands in <em>YHR054c</em>. Red letters: palindromic sequence, green letters: 38 nucleotides-long nonpalindromic interval. (B) The quasi-palindromic sequence identified in BYAT725 occurred between two 16-nucleotides-long, nearly identical inverted sequences in <em>YHR054c</em> and ARS810 (red letters, sequence differences between the two loci in blue). These two sequences are separated from each other by 745 nucleotides on the reference sequence. (C) The junction sequence identified in mutants BYAT-721, -722, -723, -724, -726, -727, and -729 joined the two opposite DNA strands at the beginning of <em>YHR054c</em>. Red letters: palindromic sequence, green letters: 63 nucleotides-long nonpalindromic interval. Note the single mismatch within the palindromic sequence (blue letters). CN, total copy number of the junction sequence in the cell as determined from sequence coverage.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Given the high numbers of CUP repeats in evolved mutants, it was particularly interesting to determine the copy number of each novel junction (deduced from sequence read coverages, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS3.pdf">Figure S3</a>). We found the BYAT711 junction present at normal haploid coverage (0.96 ×), indicating a single occurrence for the entire genome, a figure to be related to the presence of a single expanded <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus in this genome (on the macrotene chromosome). In contrast, the two other junctions (in BYAT725 and the BYAT721–729 series) were present at nearly double coverage (see <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>), suggesting two occurrences per genome in these mutants, again correlated to the presence of two expanded <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> loci (one in each macrotene chromosome). This correlation strongly suggests the presence of one quasi-palindromic junction within each expanded <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus, the rest of the locus being made of direct tandem repeats of the 2 kb CUP amplicon, an important point for consideration when elucidating the origin of the novel chromosomal structures formed (see below).</p><p id="p-35">In addition to the <em>CUP</em> junctions, another novel junction sequence was also discovered from complete genome sequencing of mutant BYAT725 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS4.pdf">Figure S4A</a>). It joins a telomeric repeat (C<sub>1-3</sub>A)<sub>n</sub> to nucleotide 123,222 within <em><a href="http://www.yeastgenome.org/locus/S000001050/overview">YHR008c</a></em> (the left boundary of amplicon VIII-F). This sequence has a copy number of only 0.63 (as judged from sequence coverage), which is probably an underestimate given the difficulty of mapping sequence reads including short telomeric repeats. The fact that this junction could only be found in mutant BYAT725 is consistent with the presence of amplicon VIII-F in this mutant only, and offers independent confirmation of its existence (compared to VIII-E). Other junction sequences expected from the structures of macrotene chromosomes (see below) fall within δ elements, excluding their possible reassembly from shotgun sequence reads. One of them, joining <em>YHRW</em>δ<em>7</em> with <em>YHRC</em>δ<em>3</em> could, however, be directly verified by sequencing a PCR amplification product from mutant BYAT711 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS4.pdf">Figure S4B</a>).</p></div><div id="sec-14" class="subsection"><h3>Topological structures and origin of macrotene chromosomes</h3><p id="p-36">With the above data, we were able to reconstitute the structures of the macrotene chromosomes in the different evolved mutants (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Mutant BYAT711, which contains a single macrotene chromosome in addition to a normal chromosome VIII, represents the simplest case. The double sequencing coverage over the entire chromosome VIII reference sequence (see <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>) indicates that the <em>macrotene</em> chromosome contains an entire copy of chromosome VIII, plus additional segments being made of extra copies of amplicons VIII-D and VIII-E (one of each), and of the 220 kb-long new <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus. The back-to-back junction between amplicons VIII-D and VIII-E within their inverted δ boundaries (demonstrated by direct sequencing, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS4.pdf">Figure S4B</a>), and the quasi-palindromic junction within the expanded <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus, argue in favor of an inverted region inside the macrotene chromosome, as illustrated by <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>. Four equivalent structures of this type remain logically possible with presently available data, depending upon the relative location of the expanded <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus <em>vs.</em> the original one (arbitrarily shown to the left on <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>) and the relative orientation of amplicons VIII-D and VIII-E between the two <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> loci (see <em>Discussion</em>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Deduced structures of macrotene chromosomes in evolved mutants. (A) Summary of amplicons with their sequence coordinates along chromosomes (in kb) and remarkable elements at their boundaries (yellow arrowhead boxes). Internal double arrows indicate orientation relative to the reference sequence. Pink tightening: centromere; black box: CUP1 locus, purple diamond: Y. lipolytica Asn-RS gene. (B) Scheme of the normal and macrotene chromosomes in mutant BYAT711. Gray cylinders, original chromosome sequences; colored boxes, extra copies of amplicons. Novel junctions at breakpoints are indicated by rectangles (thick borders, experimentally determined sequences; dotted borders, hypothetical). Gray, quasi-palindromic junctions within the CUP amplicon (see Figure 4 and Figure S3); yellow, &#x3B4; elements (see Figure S4). The macrotene chromosome drawn represents one of the four logical possibilities, depending upon the orientation of the novel VIIID-VIIIE segment and the relative positions of the normal and expanded CUP loci. (C) Scheme of the two macrotene chromosomes in mutant BYAT725. Same legend as B. Rectangle with red arrow: junction with telomeric repeats (see Figure S4). (D) Scheme of the two macrotene chromosomes in mutants BYAT721, -722, -723, -724, -726, -727, and -729. Same legend as B. All mutants share the same topological structures but differ from one another by the sizes of the two expanded CUP loci (insert, sizes in kb; see Table 2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-409920684" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Deduced structures of macrotene chromosomes in evolved mutants. (A) Summary of amplicons with their sequence coordinates along chromosomes (in kb) and remarkable elements at their boundaries (yellow arrowhead boxes). Internal double arrows indicate orientation relative to the reference sequence. Pink tightening: centromere; black box: &lt;em&gt;CUP1&lt;/em&gt; locus, purple diamond: &lt;em&gt;Y. lipolytica&lt;/em&gt; Asn-RS gene. (B) Scheme of the normal and macrotene chromosomes in mutant BYAT711. Gray cylinders, original chromosome sequences; colored boxes, extra copies of amplicons. Novel junctions at breakpoints are indicated by rectangles (thick borders, experimentally determined sequences; dotted borders, hypothetical). Gray, quasi-palindromic junctions within the CUP amplicon (see Figure 4 and Figure S3); yellow, &#x3B4; elements (see Figure S4). The macrotene chromosome drawn represents one of the four logical possibilities, depending upon the orientation of the novel VIIID-VIIIE segment and the relative positions of the normal and expanded &lt;em&gt;CUP&lt;/em&gt; loci. (C) Scheme of the two macrotene chromosomes in mutant BYAT725. Same legend as B. Rectangle with red arrow: junction with telomeric repeats (see Figure S4). (D) Scheme of the two macrotene chromosomes in mutants BYAT721, -722, -723, -724, -726, -727, and -729. Same legend as B. All mutants share the same topological structures but differ from one another by the sizes of the two expanded &lt;em&gt;CUP&lt;/em&gt; loci (insert, sizes in kb; see Table 2).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1201/F5.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/5/1201/F5.medium.gif" width="332" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468316" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Deduced structures of macrotene chromosomes in evolved mutants. (A) Summary of amplicons with their sequence coordinates along chromosomes (in kb) and remarkable elements at their boundaries (yellow arrowhead boxes). Internal double arrows indicate orientation relative to the reference sequence. Pink tightening: centromere; black box: <em>CUP1</em> locus, purple diamond: <em>Y. lipolytica</em> Asn-RS gene. (B) Scheme of the normal and macrotene chromosomes in mutant BYAT711. Gray cylinders, original chromosome sequences; colored boxes, extra copies of amplicons. Novel junctions at breakpoints are indicated by rectangles (thick borders, experimentally determined sequences; dotted borders, hypothetical). Gray, quasi-palindromic junctions within the CUP amplicon (see <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS3.pdf">Figure S3</a>); yellow, δ elements (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS4.pdf">Figure S4</a>). The macrotene chromosome drawn represents one of the four logical possibilities, depending upon the orientation of the novel VIIID-VIIIE segment and the relative positions of the normal and expanded <em>CUP</em> loci. (C) Scheme of the two macrotene chromosomes in mutant BYAT725. Same legend as B. Rectangle with red arrow: junction with telomeric repeats (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS4.pdf">Figure S4</a>). (D) Scheme of the two macrotene chromosomes in mutants BYAT721, -722, -723, -724, -726, -727, and -729. Same legend as B. All mutants share the same topological structures but differ from one another by the sizes of the two expanded <em>CUP</em> loci (insert, sizes in kb; see <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Deduction of the structures of the other mutants was more complicated because each exhibited two macrotene chromosomes, one of which containing the entire chromosome VIII (except its 13 kb-long right subtelomeric segment, see <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>) and the other entirely missing all sequences at the right of the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus. All these mutants showed two expanded <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> loci in addition to the normal one (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3C</a>) and two quasi-palindromic junctions within the CUP repeat unit (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4, B and C</a>), strongly suggesting the presence of one such junction in each expanded locus. In mutant BYAT725, the junction between the left boundary of amplicon VIII-F and a telomeric (C<sub>1-3</sub>A)<sub>n</sub> sequence (demonstrated on <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS4.pdf">Figure S4A</a>), suggests that the two extra copies of amplicon VIII-F were joined to the expanded <a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a> loci, serving as neo-telomeres for the two macrotene chromosomes as illustrated in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>. On the shorter macrotene chromosome, this structure is linked to the left part of chromosome VIII before the normal <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus. On the longer macrotene chromosome, the same structure is associated with the extra copy of amplicon VIII-D, itself replacing the deleted 13 kb-long right subtelomeric segment (see <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>). This last junction, involving the δ<em>3</em> and δ<em>16</em> elements, is only hypothetical; the two others are experimentally supported.</p><p id="p-39">The structure of all other mutants could be deduced from similar principles (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5D</a>), only assuming a novel junction between the extra copies of amplicons VIII-E and XVI-A, both present in duplicates (see <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>). The presence of δ elements in appropriate orientation at the boundaries of these amplicons supports this hypothesis. In this structural model, the differences in size between the macrotene chromosomes of the different mutants (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) are entirely accounted for by the differences in size of their expanded <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> loci, hence the copy numbers of the CUP amplicon (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5D</a> and <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-40">The formation of such complicated chromosomal structures remains, of course, difficult to imagine. In order to identify possible intermediate steps in their formation, we analyzed several subclones of each parental strain before the evolution experiments (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS1.pdf">Figure S1B</a>). The results proved to be highly interesting (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). All five subclones of <em>BYAT3C</em> and all four subclones of <em>BYAT8A</em> (including subclones numbered “one,” used to initiate the evolutionary experiments in both cases), revealed two bands hybridizing with the <em>YALI</em> Asn-RS probe, one corresponding in size to the normal chromosome VIII and the other to a macrotene chromosome. In other words, disomy and the formation of a first macrotene chromosome occurred very early in the haploid clones after meiosis. Both bands hybridized with the CUP probe as well as with the two probes on its left (YH120) and right (YH240) sides. None hybridized with the chromosome XVI probe. Subclone one of <em>BYAT3C</em> is indistinguishable from evolved mutant BYAT711, in agreement with the homogeneity of the eight evolved mutants of this experiment (see <a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1A</a>). The mutational event(s) at the origin of these evolved mutants must, therefore, have taken place very early during the cultivation of strain <em>BYAT3C</em>. Note, however, the different macrotene chromosomes in subclones three and five compared to subclones one, two, and four. The intense hybridization with the CUP probe suggests that massive amplification occurred in all of them (the DNA of subclone three was unfortunately partly degraded).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pulsed-field gel electrophoresis (PFGE) of parental subclones. PFGE was run according to Materials and Methods. Left: ethidium bromide fluorescence, size scales in Mb calibrated from migration of natural S. cerevisiae chromosomes. Right: hybridization with indicated probes. Refer to Figure 1 and Table S3 for probe definition and to Figure S1B for origin of subclones of parental strains BYAT3C and BYAT8A. Subclones numbered &#x201C;one&#x201D; (bold red) were used for evolutionary experiments. (a), degraded DNA sample; [C], normal chromosome VIII; [D], normal chromosome XVI; [M], macrotene chromosomes; [M0], initial macrotene chromosomes (see Figure 8)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-409920684" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pulsed-field gel electrophoresis (PFGE) of parental subclones. PFGE was run according to &lt;em&gt;Materials and Methods&lt;/em&gt;. Left: ethidium bromide fluorescence, size scales in Mb calibrated from migration of natural &lt;em&gt;S. cerevisiae&lt;/em&gt; chromosomes. Right: hybridization with indicated probes. Refer to Figure 1 and Table S3 for probe definition and to Figure S1B for origin of subclones of parental strains BYAT3C and BYAT8A. Subclones numbered &#x201C;one&#x201D; (bold red) were used for evolutionary experiments. (a), degraded DNA sample; [C], normal chromosome VIII; [D], normal chromosome XVI; [M], macrotene chromosomes; [M&lt;sub&gt;0&lt;/sub&gt;], initial macrotene chromosomes (see Figure 8).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1201/F6.medium.gif" width="264" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/5/1201/F6.medium.gif" width="264" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468155" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-41" class="first-child">Pulsed-field gel electrophoresis (PFGE) of parental subclones. PFGE was run according to <em>Materials and Methods</em>. Left: ethidium bromide fluorescence, size scales in Mb calibrated from migration of natural <em>S. cerevisiae</em> chromosomes. Right: hybridization with indicated probes. Refer to <a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/TableS3.pdf">Table S3</a> for probe definition and to <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS1.pdf">Figure S1B</a> for origin of subclones of parental strains BYAT3C and BYAT8A. Subclones numbered “one” (bold red) were used for evolutionary experiments. (a), degraded DNA sample; [C], normal chromosome VIII; [D], normal chromosome XVI; [M], macrotene chromosomes; [M<sub>0</sub>], initial macrotene chromosomes (see <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">The four subclones of BYAT8A showed similar patterns (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>), also suggesting that early mutational event(s) of a similar nature took place during cultivation of strain <em>BYAT8A</em>, generating a macrotene chromosome in addition to a normal size chromosome VIII (disomy). In this case, however, the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus was not amplified (as judged from hybridization intensity), an important indication of possible underlying mechanisms (see <em>Discussion</em>). The clear differences observed between the PFGE pattern of subclone one, and those of the evolved mutants derived from it (see <a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1</a>), demonstrates that the expansion of the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus and the replacement of the normal chromosome VIII copy by a second macrotene chromosome corresponded to subsequent mutational events, which occurred during this evolutionary experiment. The differences between BYAT725 and the other mutants indicate that at least two independent mutational events of similar type must have taken place during this evolutionary experiment.</p></div><div id="sec-15" class="subsection"><h3>Stability of amplified structures and macrotene chromosomes</h3><p id="p-43">To characterize this important point, we examined eight subclones from each evolved mutant (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). According to the protocol used (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS1.pdf">Figure S1C</a>), each subclone has underwent ∼25 additional generations on YPD medium from the original mutant. The result showed complete stability for BYAT711 within the limit of the experiment (eight subclones indistinguishable from one another and from the original mutant), whereas some instability was observed for the mutants derived from <em>BYAT8A</em>. In BYAT725, the large macrotene chromosome (M2) appeared more stable than the small one (two reductions, one expansion). The other mutants seemed to differ from one another but, in total, the large macrotene chromosomes appeared less stable (seven reductions and one expansion) than the small ones (two or three reductions, one expansion). Mutants BYAT723 and BYAY729 looked completely stable within the sampling limit tested. Overall, the different mutants appeared to be sufficiently stable to transmit their complex structures unchanged to the majority of their progeny (57/72) after ∼25 generations in rich medium. We did not determined the structures of the variant subclones, but it is possible that the differences in size correspond to instability within the new expanded <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> loci. Most variants corresponded to shortening of the macrotene chromosomes, but two cases of size expansion were also visible (BYAT725 M1 and BYAT721 M2, see <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Stability of evolved mutants. Pulsed-field gel electrophoresis (see Materials and Methods) of original mutant (lane 0, compare to Figure 1) and its subclones after &#x223C;25 additional generations on YPD medium (lanes 1&#x2013;8). Left: ethidium bromide fluorescence, size scale in Mb calibrated from migration of natural S. cerevisiae chromosomes. Right: hybridization with probe YH120 (see Figure 1 and Table S3). (A) mutant from BYAT3C. (B and C) Mutants from BYAT8A. [C], normal chromosome VIII; [M], macrotene chromosome; [M1] initial size of shorter macrotene chromosome; [M2] initial size of longer macrotene chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-409920684" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Stability of evolved mutants. Pulsed-field gel electrophoresis (see &lt;em&gt;Materials and Methods&lt;/em&gt;) of original mutant (lane 0, compare to Figure 1) and its subclones after &#x223C;25 additional generations on YPD medium (lanes 1&#x2013;8). Left: ethidium bromide fluorescence, size scale in Mb calibrated from migration of natural &lt;em&gt;S. cerevisiae&lt;/em&gt; chromosomes. Right: hybridization with probe YH120 (see Figure 1 and Table S3). (A) mutant from BYAT3C. (B and C) Mutants from BYAT8A. [C], normal chromosome VIII; [M], &lt;em&gt;macrotene&lt;/em&gt; chromosome; [M&lt;sub&gt;1&lt;/sub&gt;] initial size of shorter macrotene chromosome; [M&lt;sub&gt;2&lt;/sub&gt;] initial size of longer macrotene chromosome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1201/F7.medium.gif" width="287" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/5/1201/F7.medium.gif" width="287" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468188" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-44" class="first-child">Stability of evolved mutants. Pulsed-field gel electrophoresis (see <em>Materials and Methods</em>) of original mutant (lane 0, compare to <a id="xref-fig-1-15" class="xref-fig" href="#F1">Figure 1</a>) and its subclones after ∼25 additional generations on YPD medium (lanes 1–8). Left: ethidium bromide fluorescence, size scale in Mb calibrated from migration of natural <em>S. cerevisiae</em> chromosomes. Right: hybridization with probe YH120 (see <a id="xref-fig-1-16" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/TableS3.pdf">Table S3</a>). (A) mutant from BYAT3C. (B and C) Mutants from BYAT8A. [C], normal chromosome VIII; [M], <em><span class="underline">macrotene</span></em> chromosome; [M<sub>1</sub>] initial size of shorter macrotene chromosome; [M<sub>2</sub>] initial size of longer macrotene chromosome.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-45">The first example of a rearrangement altering gene order along a chromosome was discovered long ago in <em>Drosophila</em> (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sturtevant 1921</a>), and the importance of this phenomenon and of gene copy number variations in the evolution of normal or altered genomes is now fully recognized (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">El-Mabrouk and Sankoff 2012</a>). Thus, the extensively reshuffled chromosomal structures observed in the present study represent yet another example of these wider mechanisms, but have novel aspects worth considering for our general understanding. First, the very high genetic amplifications obtained (up to 880 copies of the 2 kb-long <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> repeat unit) did not correspond to the phenotypic selection applied. The <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> gene encodes a metallothionein (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Winge <em>et al.</em> 1985</a>). Among yeasts, it is uniquely found in <em>S. cerevisae</em> and related <em>Saccharomyces sensu stricto</em> species (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Byrne and Wolfe 2005</a>) and, thus, likely corresponds to a recent horizontal acquisition (from an as yet unidentified source) or <em>de novo</em> gene formation. Its increased copy number has been known to confer increased resistance to copper salt toxicity for a long time (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fogel and Welch 1982</a>; <a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Welch <em>et al.</em> 1983</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Chang <em>et al.</em> 2013</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Zhao <em>et al.</em> 2014</a>); however, copper salts were not used in our experiments. The selection was based on the transgenic Asn-RS gene, encoding a tRNA synthetase and located 70 kb upstream of the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus. This gene was found to rise to a maximum of only five copies per cell; this was the result of large segmental duplications and chromosomal disomy, and not single gene amplification. Second, evolved mutants with restored fitness were obtained by the formation of grossly altered chromosomal structures, generally considered as phenotypically deleterious or prone to induce chromosome instability, such as aneuploidy, the presence of large inverted repeats, excessive chromosome length, or numerous tandem repeats. The effects of aneuploidy have been extensively examined in yeast (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Torres <em>et al.</em> 2007</a>). Frequently observed in mutants affected in the cytokinesis motor (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rancati <em>et al.</em> 2008</a>), aneuploidy is generally compensated for by lower gene expression in natural yeast strains (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hose <em>et al.</em> 2015</a>), and mutations in several genes or intergenic regions were previously reported to increase aneuploidy tolerance (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Torres <em>et al.</em> 2010</a>). We have verified the sequences of these genes and intergenes in all our evolved mutants, but could not find nucleotide substitutions or indels in any of them. We, therefore, consider that the chromosome VIII disomy observed is intrinsically linked to the mechanism of formation of the macrotene chromosomes, rather than being the result of phenotypic selection.</p><p id="p-46">Compared to classical adaptation experiments in which gradual fitness increase results from sequence alteration or low level amplification directly affecting the few genes of interest (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Burke 2012</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dettman <em>et al.</em> 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barrick and Lenski 2013</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">de Visser <em>et al.</em> 2014</a>), the phenotypic restoration obtained here from severely altered genotypes proceeded by catastrophic events operating at the chromosomal scale (we have verified the absence of point mutation in the transgenic Asn-RS locus in all evolved mutants studied). Contrary to chromothripsis (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Zhang <em>et al.</em> 2013</a>) observed in cancer cells, the altered chromosomal structures observed here do not correspond to massive fragmentation (no additional breakpoints beside those mentioned in <em>Results</em> were observed across the genomes of our evolved mutants), but can be interpreted by the interactions between a few anomalous events during replication and/or segregation of chromosomes, including the formation of quasi-palindromic junctions associated with the massive tandem expansion of flanking repeats. The resulting structures are complex macrotene chromosomes that are sufficiently stable to propagate unchanged over dozens of successive generations, despite the expected problems mentioned above.</p><p id="p-47">The process by which these complex structures form is very interesting. Although not directly demonstrated experimentally, a coherent model can be proposed after detailed examination of the different chromosomal topologies observed in the different evolved mutants and their parental strains (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>). The model postulates the formation of an intermediate macrotene chromosome VIII, based on classical segmental duplication mechanisms (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Koszul <em>et al.</em> 2004</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">2006</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Payen <em>et al.</em> 2008</a>), nearly twice the size of the original chromosome VIII, in addition to the concomitant disomy of that chromosome. Such a structure was observed in the early subclones of the parental strain BYAT8A prior to starting the evolutionary experiment (see <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>), indicating a very early mutational event. How the disomy and large segmental duplication are mechanistically related remains unclear, but it should be noted that a similar association was previously observed at the origin of a macrotene chromosome IV in diploid cells (<a id="xref-ref-47-9" class="xref-bibr" href="#ref-47">Thierry <em>et al.</em> 2015</a>). In this intermediate macrotene chromosome VIII, the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus is not amplified (two normal loci coexist) and the Asn-RS gene is present in three copies per cell, a number likely sufficient to partially restore growth fitness (see <a id="xref-ref-47-10" class="xref-bibr" href="#ref-47">Thierry <em>et al.</em> 2015</a> for discussion of phenotypes). The beauty of the model is that, at this stage, it postulates a unique molecular mechanism to explain the diversity of all of the final structures observed: the accidental formation of a quasi-palindromic junction within one or the other of the two normal <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> loci. The absence of, or the limited, sequence homology at these junctions (see <a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>) suggests accidental template switching events during replication. The presence of only one such junction in each amplified <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus suggests that this event is the triggering step for the massive amplification of the flanking copies, which may simply result from a rolling-circle type of mechanism generated by the interaction between two replication bubbles, as previously suggested (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brewer <em>et al.</em> 2011</a>; <a id="xref-ref-47-11" class="xref-bibr" href="#ref-47">Thierry <em>et al.</em> 2015</a>). The presence of a replication origin (ARS810) within each <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> repeat (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nieduszynski <em>et al.</em> 2007</a>), and the fact that the parental cells are severely stressed (<a id="xref-ref-47-12" class="xref-bibr" href="#ref-47">Thierry <em>et al.</em> 2015</a>), probably contribute to increase the frequency of such events. The rolling-circle amplification may eventually terminate in a segmental duplication of the chromosomal segment flanking the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus (itself limited by the possible interaction between δ elements as a means to regenerate a continuous chromosome, <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>). Depending on whether the initial template switch in a <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> repeat occurs in the left or the right <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus (which should have equivalent probability), two distinct topological situations follow that account for the mutants observed. In the first case, an internal deletion occurs, keeping the right chromosome end intact. The result is the structure observed in mutant BYAT711 (and likely in all other mutants of the BYAT3C evolutionary experiment, see <a id="xref-fig-1-17" class="xref-fig" href="#F1">Figure 1</a>). In the second case, the equivalent deletion eliminates the right chromosome end, necessitating compensation via the addition of a new telomeric structure. The junction with the telomeric (C<sub>1-3</sub>A)<sub>n</sub> repeats in BYAT725, or with a segmental duplication of the left subtelomeric region of chromosome XVI in other mutants, fulfills this need (see <a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Model for the formation of macrotene chromosomes with massively amplified CUP1 loci. Colored symbols of amplicons and chromosomal segments as in Figure 5. Step 1: The model postulates the formation of a first intermediate macrotene chromosome (equivalent to M0 in Figure 6) concomitant with disomy (shown between brackets). This macrotene chromosome corresponds to a long segmental duplication between two &#x3B4; elements (top blue line), hence contains two normal CUP1 loci (black rectangles) and two copies of the asparagine tRNA synthetase gene (purple diamonds). The chimeric &#x3B4; formed at the junction is symbolized by the red arrowhead box. Step 2: A quasi-palindromic junction is accidentally generated (red stars) by a template switching event within one or the other normal CUP1 locus, initiating a cascade of amplifications affecting the CUP1 locus and its flanking segment (amplicon VIII-B) terminated by interactions between &#x3B4; sequences (see Discussion). If the left CUP1 locus is affected (case 1), the process may be terminated by the interaction between the two cooriented &#x3B4; elements drawn, generating an internal deletion and placing amplicons VIII-D and VIII-E (green and blue rectangles, respectively) in the opposite orientation (this is the topology observed in mutant BYAT711, see Figure 5B). If the right CUP1 locus is affected (case 2), the same mechanism results in the same &#x3B4; element interacting with another structure to form a new telomere, hence deleting the distal part of the initial macrotene chromosome (demonstrated in Figure 2A). This interaction may involve amplicon XVI-A (as demonstrated in Figure 2B for mutants BYAT721, 722, 723, 724 726, 727, and 729, shown here) or the direct junction with a telomeric sequence (as demonstrated in Figure S4 for mutant BYAT725, not shown here). In both cases, a third step is required to transform the intermediate with one macrotene chromosome (under brackets) into the final structures observed in mutants having two macrotene chromosomes (see Discussion for mechanisms)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-409920684" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Model for the formation of macrotene chromosomes with massively amplified &lt;em&gt;CUP1&lt;/em&gt; loci. Colored symbols of amplicons and chromosomal segments as in Figure 5. Step 1: The model postulates the formation of a first intermediate macrotene chromosome (equivalent to M&lt;sub&gt;0&lt;/sub&gt; in Figure 6) concomitant with disomy (shown between brackets). This macrotene chromosome corresponds to a long segmental duplication between two &#x3B4; elements (top blue line), hence contains two normal &lt;em&gt;CUP1&lt;/em&gt; loci (black rectangles) and two copies of the asparagine tRNA synthetase gene (purple diamonds). The chimeric &#x3B4; formed at the junction is symbolized by the red arrowhead box. Step 2: A quasi-palindromic junction is accidentally generated (red stars) by a template switching event within one or the other normal &lt;em&gt;CUP1&lt;/em&gt; locus, initiating a cascade of amplifications affecting the &lt;em&gt;CUP1&lt;/em&gt; locus and its flanking segment (amplicon VIII-B) terminated by interactions between &#x3B4; sequences (see &lt;em&gt;Discussion&lt;/em&gt;). If the left &lt;em&gt;CUP1&lt;/em&gt; locus is affected (case 1), the process may be terminated by the interaction between the two cooriented &#x3B4; elements drawn, generating an internal deletion and placing amplicons VIII-D and VIII-E (green and blue rectangles, respectively) in the opposite orientation (this is the topology observed in mutant BYAT711, see Figure 5B). If the right &lt;em&gt;CUP1&lt;/em&gt; locus is affected (case 2), the same mechanism results in the same &#x3B4; element interacting with another structure to form a new telomere, hence deleting the distal part of the initial macrotene chromosome (demonstrated in Figure 2A). This interaction may involve amplicon XVI-A (as demonstrated in Figure 2B for mutants BYAT721, 722, 723, 724 726, 727, and 729, shown here) or the direct junction with a telomeric sequence (as demonstrated in Figure S4 for mutant BYAT725, not shown here). In both cases, a third step is required to transform the intermediate with one macrotene chromosome (under brackets) into the final structures observed in mutants having two macrotene chromosomes (see &lt;em&gt;Discussion&lt;/em&gt; for mechanisms).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1201/F8.medium.gif" width="349" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/6/5/1201/F8.medium.gif" width="349" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1201/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468261" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-48" class="first-child">Model for the formation of macrotene chromosomes with massively amplified <em>CUP1</em> loci. Colored symbols of amplicons and chromosomal segments as in <a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5</a>. Step 1: The model postulates the formation of a first intermediate macrotene chromosome (equivalent to M<sub>0</sub> in <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>) concomitant with disomy (shown between brackets). This macrotene chromosome corresponds to a long segmental duplication between two δ elements (top blue line), hence contains two normal <em>CUP1</em> loci (black rectangles) and two copies of the asparagine tRNA synthetase gene (purple diamonds). The chimeric δ formed at the junction is symbolized by the red arrowhead box. Step 2: A quasi-palindromic junction is accidentally generated (red stars) by a template switching event within one or the other normal <em>CUP1</em> locus, initiating a cascade of amplifications affecting the <em>CUP1</em> locus and its flanking segment (amplicon VIII-B) terminated by interactions between δ sequences (see <em>Discussion</em>). If the left <em>CUP1</em> locus is affected (case 1), the process may be terminated by the interaction between the two cooriented δ elements drawn, generating an internal deletion and placing amplicons VIII-D and VIII-E (green and blue rectangles, respectively) in the opposite orientation (this is the topology observed in mutant BYAT711, see <a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5B</a>). If the right <em>CUP1</em> locus is affected (case 2), the same mechanism results in the same δ element interacting with another structure to form a new telomere, hence deleting the distal part of the initial macrotene chromosome (demonstrated in <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2A</a>). This interaction may involve amplicon XVI-A (as demonstrated in <a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2B</a> for mutants BYAT721, 722, 723, 724 726, 727, and 729, shown here) or the direct junction with a telomeric sequence (as demonstrated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1/FigureS4.pdf">Figure S4</a> for mutant BYAT725, not shown here). In both cases, a third step is required to transform the intermediate with one macrotene chromosome (under brackets) into the final structures observed in mutants having two macrotene chromosomes (see <em>Discussion</em> for mechanisms).</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">The only question left with this model is why mutants corresponding to the second case (amplification of the right <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus seen in the BYAT8A mutants) have two macrotene chromosomes, while mutant BYAT711, which corresponds to the first case, keeps a normal chromosome. The presence of five copies of the Asn-RS gene as opposed to four may provide some phenotypic advantage but the effect is probably minor given our previous data (<a id="xref-ref-47-13" class="xref-bibr" href="#ref-47">Thierry <em>et al.</em> 2015</a>). In addition, this hypothesis does not explain why mutant BYAT711 is stable. We prefer to think that the basic reason for replacing the normal chromosomal copy with a second macrotene chromosome (with identical topology to the first) is the difficulty of maintaining two chromosome homologs with distinct telomeres. To our knowledge, this idea has not previously been proposed before but may warrant further study. The correction of the normal chromosome to a copy of the macrotene chromosome (with duplication of the amplified <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> locus) is not difficult to imagine if one thinks about the long-range loss-of-heterozygosity phenomena observed in chromosomes of yeast hybrids (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Leh-Louis <em>et al.</em> 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Morales and Dujon 2012</a>). The opposite correction is obviously possible, but it would result in a cell with two normal chromosome copies expected to have a severely deleterious phenotype (only two Asn-RS genes).</p><p id="p-50">The gene amplifications observed here largely surpass previously known figures of tandem gene arrays in number and, as such, may be regarded as experimental curiosities. However, recent unpublished data of the 1002 yeast genomes sequencing project (<a href="http://1002genomes.u-strasbg.fr">http://1002genomes.u-strasbg.fr</a>) reveal the existence of a number of natural <em>S. cerevisiae</em> isolates exhibiting amplified <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> loci, with total copy numbers approaching those of our mutants, suggesting that the same mechanism may operate in nature (J. Schacherer, personal communication). The fact that such events occur in the absence of direct phenotypic selection (the massive amplification of the <em><a href="http://www.yeastgenome.org/locus/S000001095/overview">CUP1</a></em> gene did not increase the tolerance of our mutants to copper salts compared to parental strains with 16 copies, data not shown), suggests that they do not generate an important fitness cost, a phenomenon that has already been mentioned for gene amplification in plants (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Vila-Aiub <em>et al.</em> 2014</a>), and which is further documented here by the important stability of our mutants.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">We thank Laurence Ma, Christiane Bouchier, and all members of the genomic platform of Institut Pasteur for genomic sequences, Sergi Leon-Bernabeu for sharing unpublished data, and our colleagues of the Unité de Génétique moléculaire des levures for fruitful discussions. This work was supported in part by contract DYGEVO from Agence Nationale pour la Recherche programme blanc 2011 section SVSE6.</p><p id="p-52">Author contributions: A.T. performed the experimental work and contributed to interpretation. V.K. performed sequence data analyses and contributed to interpretation. B.D. coordinated the project and wrote the paper. The authors declare no competing financial interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-7"><p id="p-53">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024547/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-54"><em>Communicating editor: A. M. Dudley</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-11-06"><span class="received-label">Received </span>November 6, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-02-20"><span class="accepted-label">Accepted </span>February 20, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Thierry <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.1" data-doi="10.1099/mic.0.058024-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adamo</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lotti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamas</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brocca</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Amplification of the <em>CUP1</em> gene is associated with evolution of copper tolerance in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">2325</span>–<span class="cit-lpage">2335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.058024-0%26rft_id%253Dinfo%253Apmid%252F22790396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.058024-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22790396&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309852800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alt</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kellems</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertino</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schimke</span>  <span class="cit-name-given-names">R. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Selective multiplication of dihydrofolate reductase genes in methotrexate-resistant variants of cultured murine cell.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">253</span>: <span class="cit-fpage">1357</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DAlt%26rft.auinit1%253DF.%2BW.%26rft.volume%253D253%26rft.issue%253D5%26rft.spage%253D1357%26rft.epage%253D1370%26rft.atitle%253DSelective%2Bmultiplication%2Bof%2Bdihydrofolate%2Breductase%2Bgenes%2Bin%2Bmethotrexate-resistant%2Bvariants%2Bof%2Bcultured%2Bmurine%2Bcells%26rft_id%253Dinfo%253Apmid%252F627542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJqYmMiO3M6NToicmVzaWQiO3M6MTA6IjI1My81LzEzNTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTIwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.3" data-doi="10.1186/1471-2164-11-88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araya</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Whole-genome sequencing of a laboratory-evolved yeast strain.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAraya%26rft.auinit1%253DC.%2BL.%26rft.volume%253D11%26rft.spage%253D88%26rft.epage%253D88%26rft.atitle%253DWhole-genome%2Bsequencing%2Bof%2Ba%2Blaboratory-evolved%2Byeast%2Bstrain.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-88%26rft_id%253Dinfo%253Apmid%252F20128923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20128923&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.4" data-doi="10.1038/nrg3564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrick</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesnki</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome dynamics during experimental evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">827</span>–<span class="cit-lpage">839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D827%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3564%26rft_id%253Dinfo%253Apmid%252F24166031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24166031&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.5" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: A flexible trimmer for Illumina Sequence Data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–2120.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTUvMjExNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.6" data-doi="10.1371/journal.pgen.1002016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DiRienzi</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Origin-dependent inverted-repeat amplification: a replication-based model for generating palindromic amplicons.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBrewer%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De1002016%26rft.epage%253De1002016%26rft.atitle%253DOrigin-dependent%2Binverted-repeat%2Bamplification%253A%2Ba%2Breplication-based%2Bmodel%2Bfor%2Bgenerating%2Bpalindromic%2Bamplicons.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002016%26rft_id%253Dinfo%253Apmid%252F21437266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21437266&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.7" data-doi="10.1093/oxfordjournals.molbev.a026009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenzweig</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Multiple duplications of yeast hexose transport genes in response to selection in a glucose-limited environment.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DBrown%26rft.auinit1%253DC.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D931%26rft.epage%253D942%26rft.atitle%253DMultiple%2Bduplications%2Bof%2Byeast%2Bhexose%2Btransport%2Bgenes%2Bin%2Bresponse%2Bto%2Bselection%2Bin%2Ba%2Bglucose-limited%2Benvironment%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026009%26rft_id%253Dinfo%253Apmid%252F9718721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE1LzgvOTMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.8" data-doi="10.1098/rspb.2012.0799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">How does adaptation sweep through the genome? Insights from long-term selection experiments. Proc. R.</span> <abbr class="cit-jnl-abbrev">Soc. Biol.</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">5029</span>–<span class="cit-lpage">5038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoc.%2BBiol.%26rft.volume%253D279%26rft.spage%253D5029%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2012.0799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2012.0799&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.9" data-doi="10.1101/gr.3672305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The yeast gene order browser: combining curated homology and syntenic context reveals gene fate in polyploidy species.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1456</span>–<span class="cit-lpage">1461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3672305%26rft_id%253Dinfo%253Apmid%252F16169922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMC8xNDU2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.10" data-doi="10.1371/journal.pgen.1003232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">S.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">H.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">S.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leu</span>  <span class="cit-name-given-names">J.-Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dynamic large-scale chromosomal rearrangements fuel rapid adaptation in yeast populations.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e1003232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003232%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003232%26rft_id%253Dinfo%253Apmid%252F23358723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23358723&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.11" data-doi="10.1186/1471-2164-11-56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Despons</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baret</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frangeul</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leh-Louis</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrens</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide computational prediction of tandem gene arrays: application in yeasts.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDespons%26rft.auinit1%253DL.%26rft.volume%253D11%26rft.spage%253D56%26rft.epage%253D56%26rft.atitle%253DGenome-wide%2Bcomputational%2Bprediction%2Bof%2Btandem%2Bgene%2Barrays%253A%2Bapplication%2Bin%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-56%26rft_id%253Dinfo%253Apmid%252F20092627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20092627&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.12" data-doi="10.1016/j.crvi.2011.05.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Despons</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uzunov</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Tandem gene arrays, plastic chromosomal organizations.</span> <abbr class="cit-jnl-abbrev">C. R. Biol.</abbr> <span class="cit-vol">334</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComptes%2Brendus%2Bbiologies%26rft.stitle%253DC%2BR%2BBiol%26rft.aulast%253DDespons%26rft.auinit1%253DL.%26rft.volume%253D334%26rft.issue%253D8-9%26rft.spage%253D639%26rft.epage%253D646%26rft.atitle%253DTandem%2Bgene%2Barrays%252C%2Bplastic%2Bchromosomal%2Borganizations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.crvi.2011.05.012%26rft_id%253Dinfo%253Apmid%252F21819945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.crvi.2011.05.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21819945&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.13" data-doi="10.1111/j.1365-294X.2012.05484.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dettman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodrigue</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melnyk</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolutionary insight from whole-genome sequencing of experimentally evolved microbes.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2058</span>–<span class="cit-lpage">2077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DDettman%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D21%26rft.issue%253D9%26rft.spage%253D2058%26rft.epage%253D2077%26rft.atitle%253DEvolutionary%2Binsight%2Bfrom%2Bwhole-genome%2Bsequencing%2Bof%2Bexperimentally%2Bevolved%2Bmicrobes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05484.x%26rft_id%253Dinfo%253Apmid%252F22332770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05484.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22332770&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302932500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.14" data-doi="10.1038/nrg3744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Visser</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arjan</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krug</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Empirical fitness landscapes and the predictability of evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">480</span>–<span class="cit-lpage">490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D480%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3744%26rft_id%253Dinfo%253Apmid%252F24913663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24913663&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.15" data-doi="10.1038/nature02579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrens</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casaregola</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome evolution in yeasts</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">430</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDujon%26rft.auinit1%253DB.%26rft.volume%253D430%26rft.issue%253D6995%26rft.spage%253D35%26rft.epage%253D44%26rft.atitle%253DGenome%2Bevolution%2Bin%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02579%26rft_id%253Dinfo%253Apmid%252F15229592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15229592&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222356800036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.16" data-doi="10.1073/pnas.242624799"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badrane</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferea</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Characteristic genome rearrangements in experimental evolution of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">16144</span>–<span class="cit-lpage">16149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.242624799%26rft_id%253Dinfo%253Apmid%252F12446845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjUvMTYxNDQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTIwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1201.17" data-doi="10.1007/978-1-61779-582-4_15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Mabrouk</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankoff</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Analysis of gene order evolution beyond single-copy genes</span>. <span class="cit-source">Methods Mol. Biol.</span> <span class="cit-vol">855</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DEl-Mabrouk%26rft.auinit1%253DN.%26rft.volume%253D855%26rft.spage%253D397%26rft.epage%253D429%26rft.atitle%253DAnalysis%2Bof%2Bgene%2Border%2Bevolution%2Bbeyond%2Bsingle-copy%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-61779-582-4_15%26rft_id%253Dinfo%253Apmid%252F22407718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-61779-582-4_15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22407718&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303131100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.18" data-doi="10.1073/pnas.79.17.5342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fogel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Tandem gene amplification mediates copper resistance in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">5342</span>–<span class="cit-lpage">5346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFogel%26rft.auinit1%253DS.%26rft.volume%253D79%26rft.issue%253D17%26rft.spage%253D5342%26rft.epage%253D5346%26rft.atitle%253DTandem%2Bgene%2Bamplification%2Bmediates%2Bcopper%2Bresistance%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.79.17.5342%26rft_id%253Dinfo%253Apmid%252F6291039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzkvMTcvNTM0MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.19" data-doi="10.1073/pnas.0906649107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaines</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bukun</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chisholm</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene amplification confers glyphosate resistance in <em>Amaranthus palmeri</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">1029</span>–<span class="cit-lpage">1034</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0906649107%26rft_id%253Dinfo%253Apmid%252F20018685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzMvMTAyOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.20" data-doi="10.1371/journal.pgen.1000303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenq</span>  <span class="cit-name-given-names">H. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The repertoire and dynamics of evolutionary adaptations to controlled nutrient-limited environments in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGresham%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De1000303%26rft.epage%253De1000303%26rft.atitle%253DThe%2Brepertoire%2Band%2Bdynamics%2Bof%2Bevolutionary%2Badaptations%2Bto%2Bcontrolled%2Bnutrient-limited%2Benvironments%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000303%26rft_id%253Dinfo%253Apmid%252F19079573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19079573&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.21" data-doi="10.1073/pnas.1014023107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usaite</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Germann</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisby</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Adaptation to diverse nitrogen-limited environments by deletion or extrachromosomal element formation of the <em>GAP1</em> locus.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sc.</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">18551</span>–<span class="cit-lpage">18556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSc.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1014023107%26rft_id%253Dinfo%253Apmid%252F20937885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQzLzE4NTUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hose</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yong</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sardi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newton</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Dosage compensation can buffer copy-number variation in wild yeast.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">05462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D4%26rft.spage%253D05462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.23" data-doi="10.1073/pnas.81.2.337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Najarian</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haslinger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valenzoela</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Primary structure and transcription of an amplified genetic locus: The <em>CUP1</em> locus of yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKarin%26rft.auinit1%253DM.%26rft.volume%253D81%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPrimary%2Bstructure%2Band%2Btranscription%2Bof%2Ban%2Bamplified%2Bgenetic%2Blocus%253A%2Bthe%2BCUP1%2Blocus%2Bof%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.81.2.337%26rft_id%253Dinfo%253Apmid%252F6364141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI4MS8yLzMzNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.24" data-doi="10.1016/j.ceb.2013.02.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kloosterman</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuppen</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Chromothripsis in congenital disorders and cancer: similarities and differences.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BCell%2BBiol.%26rft.volume%253D25%26rft.spage%253D341%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ceb.2013.02.008%26rft_id%253Dinfo%253Apmid%252F23478216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ceb.2013.02.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23478216&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.25" data-doi="10.1101/gr.185041.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kloosterman</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francioli</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hormozdiani</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchall</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hehir-Kwa</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Characteristics of de novo structural changes in the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">792</span>–<span class="cit-lpage">801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.185041.114%26rft_id%253Dinfo%253Apmid%252F25883321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzYvNzkyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.26" data-doi="10.1016/j.cell.2013.02.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korbel</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Criteria for inference of chromothripsis in cancer genomes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1226</span>–<span class="cit-lpage">1236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D152%26rft.spage%253D1226%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.02.023%26rft_id%253Dinfo%253Apmid%252F23498933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.02.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23498933&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316192500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.27" data-doi="10.1038/sj.emboj.7600024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koszul</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caburet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Eukaryotic genome evolution through the spontaneous duplication of large chromosomal segments.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">234</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DKoszul%26rft.auinit1%253DR.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D234%26rft.epage%253D243%26rft.atitle%253DEucaryotic%2Bgenome%2Bevolution%2Bthrough%2Bthe%2Bspontaneous%2Bduplication%2Bof%2Blarge%2Bchromosomal%2Bsegments%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600024%26rft_id%253Dinfo%253Apmid%252F14685272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIyMy8xLzIzNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koszul</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Stability of large segmental duplications in the yeast genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">2211</span>–<span class="cit-lpage">2222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D175%26rft.spage%253D2211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.29" data-doi="10.1534/g3.111.000745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leh-Louis</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Despons</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durrens</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>Pichia sorbitophila</em>, an interspecies yeast hybrid, reveals early steps of genome resolution after polyploidization.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DLouis%26rft.auinit1%253DV.%2BL.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D299%26rft.epage%253D311%26rft.atitle%253DPichia%2Bsorbitophila%252C%2Ban%2BInterspecies%2BYeast%2BHybrid%252C%2BReveals%2BEarly%2BSteps%2Bof%2BGenome%2BResolution%2BAfter%2BPolyploidization%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000745%26rft_id%253Dinfo%253Apmid%252F22384408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjIvMi8yOTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTIwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D25%26rft.spage%253D589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.31" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence alignment/map (SAM) format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.32" data-doi="10.1038/nature05205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Libuda</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Amplification of histone genes by circular chromosome formation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">443</span>: <span class="cit-fpage">1003</span>–<span class="cit-lpage">1007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLibuda%26rft.auinit1%253DD.%2BE.%26rft.volume%253D443%26rft.issue%253D7114%26rft.spage%253D1003%26rft.epage%253D1007%26rft.atitle%253DAmplification%2Bof%2Bhistone%2Bgenes%2Bby%2Bcircular%2Bchromosome%2Bformation%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05205%26rft_id%253Dinfo%253Apmid%252F17066037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05205&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17066037&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.33" data-doi="10.1007/BF00260720"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latreille</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miranda</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">A <em>PMR2</em> tandem repeat with a modified C-terminus is located downstream from the KRS1 gene encoding lysyl-tRNA synthetase in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">227</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DMartinez%26rft.auinit1%253DR.%26rft.volume%253D227%26rft.issue%253D1%26rft.spage%253D149%26rft.epage%253D154%26rft.atitle%253DA%2BPMR2%2Btandem%2Brepeat%2Bwith%2Ba%2Bmodified%2BC-terminus%2Bis%2Blocated%2Bdownstream%2Bfrom%2Bthe%2BKRS1%2Bgene%2Bencoding%2Blysyl-tRNA%2Bsynthetase%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00260720%26rft_id%253Dinfo%253Apmid%252F2046655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00260720&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2046655&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.34" data-doi="10.1093/bib/bbs012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milne</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cock</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Using Tablet for visual exploration of second-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">193</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs012%26rft_id%253Dinfo%253Apmid%252F22445902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYmliIjtzOjU6InJlc2lkIjtzOjg6IjE0LzIvMTkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.35" data-doi="10.1128/MMBR.00022-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morales</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolutionary role of interspecies hybridization and genetic exchanges in yeasts.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DMorales%26rft.auinit1%253DL.%26rft.volume%253D76%26rft.issue%253D4%26rft.spage%253D721%26rft.epage%253D739%26rft.atitle%253DEvolutionary%2BRole%2Bof%2BInterspecies%2BHybridization%2Band%2BGenetic%2BExchanges%2Bin%2BYeasts%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00022-12%26rft_id%253Dinfo%253Apmid%252F23204364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3Ni80LzcyMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.36" data-doi="10.1016/j.fgb.2008.11.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thierry</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coppee</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouyette</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hennequin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic polymorphism in the population of <em>Candida glabrata:</em> Gene copy-number variation and chromosomal translocations.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">264</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DMuller%26rft.auinit1%253DH.%26rft.volume%253D46%26rft.issue%253D3%26rft.spage%253D264%26rft.epage%253D276%26rft.atitle%253DGenomic%2Bpolymorphism%2Bin%2Bthe%2Bpopulation%2Bof%2BCandida%2Bglabrata%253A%2Bgene%2Bcopy-number%2Bvariation%2Band%2Bchromosomal%2Btranslocations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2008.11.006%26rft_id%253Dinfo%253Apmid%252F19084610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2008.11.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19084610&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.37" data-doi="10.1093/nar/gkl758"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nieduszynski</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiraga</span>  <span class="cit-name-given-names">S.-I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ak</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benham</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donaldson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">OriDB: a replication origin database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">D40</span>–<span class="cit-lpage">D46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl758%26rft_id%253Dinfo%253Apmid%252F17065467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE0OiIzNS9zdXBwbF8xL0Q0MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.38" data-doi="10.1089/cmb.2013.0084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nurk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bankerich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antipov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korobeynikov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assembling single-cell genomes and mini-metagenomes from chimeric MDA products.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BBiol.%26rft.volume%253D20%26rft.spage%253D714%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2013.0084%26rft_id%253Dinfo%253Apmid%252F24093227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2013.0084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24093227&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.39" data-doi="10.1371/journal.pgen.1000175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koszul</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Segmental duplications arise from <em>pol32</em>-dependent repair of broken forks through two alternative replication-based mechanisms.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPayen%26rft.auinit1%253DC.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253De1000175%26rft.epage%253De1000175%26rft.atitle%253DSegmental%2Bduplications%2Barise%2Bfrom%2BPol32-dependent%2Brepair%2Bof%2Bbroken%2Bforks%2Bthrough%2Btwo%2Balternative%2Breplication-based%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000175%26rft_id%253Dinfo%253Apmid%252F18773114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000175&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18773114&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.40" data-doi="10.1534/g3.113.009365"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DiRienzi</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pogavhar</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The dynamics of diverse segmental amplifications in populations of <em>Saccharomyces cerevisiae</em> adapting to strong selection.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.009365%26rft_id%253Dinfo%253Apmid%252F24368781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvMy8zOTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTIwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.41" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.42" data-doi="10.1016/j.cell.2008.09.039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rancati</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pavelka</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleharty</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noll</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trimble</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Aneuploidy underlies rapid adaptive evolution of yeast cells deprived of a conserved cytokinesis motor.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DRancati%26rft.auinit1%253DG.%26rft.volume%253D135%26rft.issue%253D5%26rft.spage%253D879%26rft.epage%253D893%26rft.atitle%253DAneuploidy%2Bunderlies%2Brapid%2Badaptive%2Bevolution%2Bof%2Byeast%2Bcells%2Bdeprived%2Bof%2Ba%2Bconserved%2Bcytokinesis%2Bmotor.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.09.039%26rft_id%253Dinfo%253Apmid%252F19041751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.09.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19041751&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261183900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.43" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.44" data-doi="10.1016/0092-8674(84)90301-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Separation of yeast chromosome-sized DNAs by pulsed field gradient gel electrophoresis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSchwartz%26rft.auinit1%253DD.%2BC.%26rft.volume%253D37%26rft.issue%253D1%26rft.spage%253D67%26rft.epage%253D75%26rft.atitle%253DSeparation%2Bof%2Byeast%2Bchromosome-sized%2BDNAs%2Bby%2Bpulsed%2Bfield%2Bgradient%2Bgel%2Belectrophoresis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252884%252990301-5%26rft_id%253Dinfo%253Apmid%252F6373014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(84)90301-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6373014&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984SR71400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.45" data-doi="10.1016/j.cell.2010.11.055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenman</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bignell</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Massive genomic rearrangement acquired in a single catastrophic event during cancer development.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DStephens%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D144%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D40%26rft.atitle%253DMassive%2Bgenomic%2Brearrangement%2Bacquired%2Bin%2Ba%2Bsingle%2Bcatastrophic%2Bevent%2Bduring%2Bcancer%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.11.055%26rft_id%253Dinfo%253Apmid%252F21215367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.11.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21215367&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285993600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.46" data-doi="10.1073/pnas.7.8.235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1921</span> <span class="cit-article-title">A case of rearrangement of genes in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSturtevant%26rft.auinit1%253DA.%2BH.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253D235%26rft.epage%253D237%26rft.atitle%253DA%2BCase%2Bof%2BRearrangement%2Bof%2BGenes%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.7.8.235%26rft_id%253Dinfo%253Apmid%252F16576597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo3OiI3LzgvMjM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.47" data-doi="10.1038/ncomms7154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thierry</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khanna</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Créno</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lafontaine</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title"><em>Macrotene</em> chromosomes provide insights to a new mechanism of high-order gene amplification in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">6154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D6154%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms7154%26rft_id%253Dinfo%253Apmid%252F25635677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms7154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25635677&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.48" data-doi="10.1126/science.1142210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokolsky</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boselli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Effects of aneuploidy on cellular physiology and cell division in haploid yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>: <span class="cit-fpage">916</span>–<span class="cit-lpage">924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTorres%26rft.auinit1%253DE.%2BM.%26rft.volume%253D317%26rft.issue%253D5840%26rft.spage%253D916%26rft.epage%253D924%26rft.atitle%253DEffects%2Bof%2BAneuploidy%2Bon%2BCellular%2BPhysiology%2Band%2BCell%2BDivision%2Bin%2BHaploid%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1142210%26rft_id%253Dinfo%253Apmid%252F17702937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTcvNTg0MC85MTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTIwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.49" data-doi="10.1016/j.cell.2010.08.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delphoure</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panneerselvam</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identification of aneuploidy-tolerating mutations.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTorres%26rft.auinit1%253DE.%2BM.%26rft.volume%253D143%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D83%26rft.atitle%253DIdentification%2Bof%2Baneuploidy-tolerating%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.08.038%26rft_id%253Dinfo%253Apmid%252F20850176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.08.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20850176&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282362000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.50" data-doi="10.1007/s00425-013-2022-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vila-Aiub</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goh</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaines</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Busi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">No fitness cost of glyphosate resistance endowed by massive <em>EPSPS</em> gene amplification in <em>Amaranthus palmeri</em>.</span> <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">239</span>: <span class="cit-fpage">793</span>–<span class="cit-lpage">801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.volume%253D239%26rft.spage%253D793%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00425-013-2022-x%26rft_id%253Dinfo%253Apmid%252F24385093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00425-013-2022-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24385093&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333125000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.51" data-doi="10.1128/MCB.3.8.1353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welch</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fogel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cathala</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Industrial yeasts display tandem gene iteration at the <em>CUP1</em> region.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1353</span>–<span class="cit-lpage">1361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DWelch%26rft.auinit1%253DJ%2BW%26rft.volume%253D3%26rft.issue%253D8%26rft.spage%253D1353%26rft.epage%253D1361%26rft.atitle%253DIndustrial%2Byeasts%2Bdisplay%2Btandem%2Bgene%2Biteration%2Bat%2Bthe%2BCUP1%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.3.8.1353%26rft_id%253Dinfo%253Apmid%252F6621529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjMvOC8xMzUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyMDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wieland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nitsche</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strayle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steiner</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudolph</span>  <span class="cit-name-given-names">H. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The <em>PMR2</em> gene cluster encodes functionally distinct isoforms of a putative Na+ pump in the yeast plasma membrane.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">3870</span>–<span class="cit-lpage">3882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D14%26rft.spage%253D3870%26rft_id%253Dinfo%253Apmid%252F7664728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7664728&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RQ66900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winge</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielson</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamer</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Yeast metallothionein. Sequence and metal-binding properties.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">260</span>: <span class="cit-fpage">14464</span>–<span class="cit-lpage">14470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DWinge%26rft.auinit1%253DD.%2BR.%26rft.volume%253D260%26rft.issue%253D27%26rft.spage%253D14464%26rft.epage%253D14470%26rft.atitle%253DYeast%2Bmetallothionein.%2BSequence%2Band%2Bmetal-binding%2Bproperties%26rft_id%253Dinfo%253Apmid%252F3902832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNjAvMjcvMTQ0NjQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTIwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.54" data-doi="10.1038/nrg3871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zarrei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merico</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherer</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A copy number variation map of the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D16%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3871%26rft_id%253Dinfo%253Apmid%252F25645873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3871&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25645873&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1201.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.55" data-doi="10.1101/gad.229559.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leibowitz</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pellman</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Chromothripsis and beyond: rapid genome evolution from complex chromosomal rearrangements.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2513</span>–<span class="cit-lpage">2530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DZhang%26rft.auinit1%253DC.-Z.%26rft.volume%253D27%26rft.issue%253D23%26rft.spage%253D2513%26rft.epage%253D2530%26rft.atitle%253DChromothripsis%2Band%2Bbeyond%253A%2Brapid%2Bgenome%2Bevolution%2Bfrom%2Bcomplex%2Bchromosomal%2Brearrangements%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.229559.113%26rft_id%253Dinfo%253Apmid%252F24298051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI3LzIzLzI1MTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTIwMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1201.56" data-doi="10.1534/g3.114.012922"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strope</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozmon</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Structures of naturally evolved <em>CUP1</em> tandem arrays in yeast indicate that these arrays are generated by unequal nonhomologous recombination.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2259</span>–<span class="cit-lpage">2269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.012922%26rft_id%253Dinfo%253Apmid%252F25236733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjQvMTEvMjI1OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjAxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/5/1201.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/5/1191" title="Comparative and Transcriptome Analyses Uncover Key Aspects of Coding- and Long Noncoding RNAs in Flatworm Mitochondrial Genomes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/5/1217" title="The Genetic Linkage Map of the Medicinal Mushroom Agaricus subrufescens Reveals Highly Conserved Macrosynteny with the Congeneric Species Agaricus bisporus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/5"><span>Volume 6 Issue 5, May 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/5.cover-source.jpg?itok=8SH0hKOw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1201.abstract?sid=d73770b8-eb31-44b4-89d5-c21b8c0c162e" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--ieFoca-IhRy2wTubx0zPhu6KriYPlV9RiCqP4WZELA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/5/1201" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/5/1201" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/5/1201" class="active" data-icon-position="" data-hide-link-title="0">Massive Amplification at an Unselected Locus Accompanies Complex Chromosomal Rearrangements in Yeast</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460353" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-8VxIkB8wJTBjrjitNtQGKXSAxyq9bILjdnXV2EkVxE8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/5/1201.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/5/1201" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-JE2DK98X-wisG26AI5EHedZA5iqnnVueoVFKt_pveh0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/5/1201" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/5/1201.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460353" id="node460353--2" data-pisa="ggg;6/5/1201" data-pisa-master="ggg;g3.115.024547" data-apath="/ggg/6/5/1201.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Massive Amplification at an Unselected Locus Accompanies Complex Chromosomal Rearrangements in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Agnès Thierry</span>, <span class="highwire-citation-author" data-delta="1">Varun Khanna</span> and <span class="highwire-citation-author" data-delta="2">Bernard Dujon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1201-1215;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024547 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460353--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Agnès Thierry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institut Pasteur, Unité de Génétique Moléculaire des Levures, CNRS (UMR3525), Sorbonne Universités, UPMC, Université Paris 06 (UFR927), F-75724 CEDEX 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Agn%C3%A8s%2BThierry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thierry%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAgn%25C3%25A8s%2BThierry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Varun Khanna </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institut Pasteur, Unité de Génétique Moléculaire des Levures, CNRS (UMR3525), Sorbonne Universités, UPMC, Université Paris 06 (UFR927), F-75724 CEDEX 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Varun%2BKhanna%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khanna%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVarun%2BKhanna%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bernard Dujon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institut Pasteur, Unité de Génétique Moléculaire des Levures, CNRS (UMR3525), Sorbonne Universités, UPMC, Université Paris 06 (UFR927), F-75724 CEDEX 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernard%2BDujon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dujon%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABernard%2BDujon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bdujon@pasteur.fr" class="" data-icon-position="" data-hide-link-title="0">bdujon@pasteur.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/5/1201" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/5/1201&amp;title=Massive%20Amplification%20at%20an%20Unselected%20Locus%20Accompanies%20Complex%20Chromosomal%20Rearrangements%20in%20Yeast" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/5/1201&amp;title=Massive%20Amplification%20at%20an%20Unselected%20Locus%20Accompanies%20Complex%20Chromosomal%20Rearrangements%20in%20Yeast" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/5/1201&amp;title=Massive%20Amplification%20at%20an%20Unselected%20Locus%20Accompanies%20Complex%20Chromosomal%20Rearrangements%20in%20Yeast" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/5/1201&amp;text=Massive%20Amplification%20at%20an%20Unselected%20Locus%20Accompanies%20Complex%20Chromosomal%20Rearrangements%20in%20Yeast" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/5/1201&amp;title=Massive%20Amplification%20at%20an%20Unselected%20Locus%20Accompanies%20Complex%20Chromosomal%20Rearrangements%20in%20Yeast" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/5/1201&amp;t=Massive%20Amplification%20at%20an%20Unselected%20Locus%20Accompanies%20Complex%20Chromosomal%20Rearrangements%20in%20Yeast" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/5/1201&amp;title=Massive%20Amplification%20at%20an%20Unselected%20Locus%20Accompanies%20Complex%20Chromosomal%20Rearrangements%20in%20Yeast" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/5/1201&amp;title=Massive%20Amplification%20at%20an%20Unselected%20Locus%20Accompanies%20Complex%20Chromosomal%20Rearrangements%20in%20Yeast" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460353/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460353" id="node460353--3" data-pisa="ggg;6/5/1201" data-pisa-master="ggg;g3.115.024547" data-apath="/ggg/6/5/1201.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Massive Amplification at an Unselected Locus Accompanies Complex Chromosomal Rearrangements in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Agnès Thierry</span>, <span class="highwire-citation-author" data-delta="1">Varun Khanna</span> and <span class="highwire-citation-author" data-delta="2">Bernard Dujon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1201-1215;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024547 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460353--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Agnès Thierry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institut Pasteur, Unité de Génétique Moléculaire des Levures, CNRS (UMR3525), Sorbonne Universités, UPMC, Université Paris 06 (UFR927), F-75724 CEDEX 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Agn%C3%A8s%2BThierry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thierry%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAgn%25C3%25A8s%2BThierry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Varun Khanna </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institut Pasteur, Unité de Génétique Moléculaire des Levures, CNRS (UMR3525), Sorbonne Universités, UPMC, Université Paris 06 (UFR927), F-75724 CEDEX 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Varun%2BKhanna%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Khanna%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVarun%2BKhanna%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bernard Dujon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institut Pasteur, Unité de Génétique Moléculaire des Levures, CNRS (UMR3525), Sorbonne Universités, UPMC, Université Paris 06 (UFR927), F-75724 CEDEX 15, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernard%2BDujon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dujon%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABernard%2BDujon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bdujon@pasteur.fr" class="" data-icon-position="" data-hide-link-title="0">bdujon@pasteur.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1201&amp;rft.epage=1215&amp;rft.atitle=Massive%20Amplification%20at%20an%20Unselected%20Locus%20Accompanies%20Complex%20Chromosomal%20Rearrangements%20in%20Yeast&amp;rft.volume=6&amp;rft.issue=5&amp;rft.date=2016-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Thierry%2C+Agn%C3%A8s&amp;rft.au=Khanna%2C+Varun&amp;rft.au=Dujon%2C+Bernard"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460353/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460353/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460353/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460353/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460353/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460353/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460353/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460353/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460353/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460353/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460353/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460353/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/5/1201" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/5/1201.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/5/1201.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/5/1201.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/5/1201.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
