<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/5/1417" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/5/1417.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/5/1417.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/5/1417.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/5/1417.atom" />
<meta name="HW.pisa" content="ggg;6/5/1417" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A High-Density SNP Genetic Linkage Map and QTL Analysis of Growth-Related Traits in a Hybrid Family of Oysters (Crassostrea gigas × Crassostrea angulata) Using Genotyping-by-Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.116.026971" />
<meta name="DC.Date" content="2016-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Wang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Oysters are among the most important species in global aquaculture. Crassostrea gigas , and its subspecies C. angulata , are the major cultured species. To determine the genetic basis of growth-related traits in oysters, we constructed a second-generation linkage map from 3367 single-nucleotide polymorphisms (SNPs) based on genotyping-by-sequencing, genotyped from a C. gigas  ×  C. angulata hybrid family. These 3367 SNPs were distributed on 1695 markers, which were assigned to 10 linkage groups. The genetic linkage map had a total length of 1084.3 cM, with an average of 0.8 cM between markers; it thus represents the densest genetic map constructed for oysters to date. Twenty-seven quantitative trait loci (QTL) for five growth-related traits were detected. These QTL could explain 4.2–7.7% (mean = 5.4%) of the phenotypic variation. In total, 50.8% of phenotypic variance for shell width, 7.7% for mass weight, and 34.1% for soft tissue weight were explained. The detected QTL were distributed among eight linkage groups, and more than half (16) were concentrated within narrow regions in their respective linkage groups. Thirty-eight annotated genes were identified within the QTL regions, two of which are key genes for carbohydrate metabolism. Other genes were found to participate in assembly and regulation of the actin cytoskeleton, signal transduction, and regulation of cell differentiation and development. The newly developed high-density genetic map, and the QTL and candidate genes identified provide a valuable genetic resource and a basis for marker-assisted selection for C. gigas and C. angulata ." />
<meta name="DC.Contributor" content="Jinpeng Wang" />
<meta name="DC.Contributor" content="Li Li" />
<meta name="DC.Contributor" content="Guofan Zhang" />
<meta name="article:published_time" content="2016-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A High-Density SNP Genetic Linkage Map and QTL Analysis of Growth-Related Traits in a Hybrid Family of Oysters (Crassostrea gigas × Crassostrea angulata) Using Genotyping-by-Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Oysters are among the most important species in global aquaculture. &lt;i&gt;Crassostrea gigas&lt;/i&gt;, and its subspecies &lt;i&gt;C. angulata&lt;/i&gt;, are the major cultured species. To determine the genetic basis of growth-related traits in oysters, we constructed a second-generation linkage map from 3367 single-nucleotide polymorphisms (SNPs) based on genotyping-by-sequencing, genotyped from a &lt;i&gt;C.&amp;nbsp;gigas&lt;/i&gt;&amp;nbsp;×&amp;nbsp;&lt;i&gt;C.&amp;nbsp;angulata&lt;/i&gt; hybrid family. These 3367 SNPs were distributed on 1695 markers, which were assigned to 10 linkage groups. The genetic linkage map had a total length of 1084.3&amp;nbsp;cM, with an average of 0.8&amp;nbsp;cM between markers; it thus represents the densest genetic map constructed for oysters to date. Twenty-seven quantitative trait loci (QTL) for five growth-related traits were detected. These QTL could explain 4.2–7.7% (mean&amp;nbsp;=&amp;nbsp;5.4%) of the phenotypic variation. In total, 50.8% of phenotypic variance for shell width, 7.7% for mass weight, and 34.1% for soft tissue weight were explained. The detected QTL were distributed among eight linkage groups, and more than half (16) were concentrated within narrow regions in their respective linkage groups. Thirty-eight annotated genes were identified within the QTL regions, two of which are key genes for carbohydrate metabolism. Other genes were found to participate in assembly and regulation of the actin cytoskeleton, signal transduction, and regulation of cell differentiation and development. The newly developed high-density genetic map, and the QTL and candidate genes identified provide a valuable genetic resource and a basis for marker-assisted selection for &lt;i&gt;C.&amp;nbsp;gigas&lt;/i&gt; and &lt;i&gt;C.&amp;nbsp;angulata&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/05/01" />
<meta name="citation_mjid" content="ggg;6/5/1417" />
<meta name="citation_id" content="6/5/1417" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/5/1417" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/5/1417.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/5/1417.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/5/1417.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.026971" />
<meta name="citation_pmid" content="26994291" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1417" />
<meta name="citation_lastpage" content="1426" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jinpeng Wang" />
<meta name="citation_author_institution" content="Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, Shandong, China 266071" />
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China 100039" />
<meta name="citation_author_institution" content="Laboratory for Marine Biology and Biotechnology, Qingdao National Laboratory for Marine Science and Technology, Qingdao, Shandong, China 266071" />
<meta name="citation_author_institution" content="National and Local Joint Engineering Laboratory of Ecological Mariculture, Qingdao, Shandong, China 266071" />
<meta name="citation_author" content="Li Li" />
<meta name="citation_author_institution" content="Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, Shandong, China 266071" />
<meta name="citation_author_institution" content="National and Local Joint Engineering Laboratory of Ecological Mariculture, Qingdao, Shandong, China 266071" />
<meta name="citation_author_institution" content="Laboratory for Marine Fisheries and Aquaculture, Qingdao National Laboratory for Marine Science and Technology, Qingdao, Shandong, China 266071" />
<meta name="citation_author_email" content="gzhang@qdio.ac.cn" />
<meta name="citation_author_email" content="lili@qdio.ac.cn" />
<meta name="citation_author" content="Guofan Zhang" />
<meta name="citation_author_institution" content="Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, Shandong, China 266071" />
<meta name="citation_author_institution" content="Laboratory for Marine Biology and Biotechnology, Qingdao National Laboratory for Marine Science and Technology, Qingdao, Shandong, China 266071" />
<meta name="citation_author_institution" content="National and Local Joint Engineering Laboratory of Ecological Mariculture, Qingdao, Shandong, China 266071" />
<meta name="citation_author_email" content="gzhang@qdio.ac.cn" />
<meta name="citation_author_email" content="lili@qdio.ac.cn" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=K. R. Andrews;citation_author=J. M. Good;citation_author=M. R. Miller;citation_author=G. Luikart;citation_author=P. A. Hohenlohe;citation_title=Harnessing the power of RADseq for ecological and evolutionary genomics.;citation_pages=253-92;citation_volume=17;citation_year=2016;citation_issue=2;citation_doi=10.1016/j.jembe.2006.12.007" />
<meta name="citation_reference" content="citation_title=Encyclopedia of Measurement and Statistics;citation_title=Pearson product-moment correlation coefficient;citation_author=J. O. Anthony;citation_author=D. Larry;citation_author=L. L. Nancy;citation_pages=751-756;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. P. J. Brooks;citation_author=K. B. Storey;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=267;citation_year=1988;citation_issue=1;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Archives of Biochemistry and Biophysics;citation_journal_abbrev=Archives of Biochemistry and Biophysics;citation_author=D. Buestel;citation_author=M. Ropert;citation_author=J. Prou;citation_author=P. Goulletquer;citation_title=Reevaluation of the &amp;quot;glycolytic complex&amp;quot; in muscle: a multitechnique approach using trout white muscle.;citation_pages=13-22;citation_volume=28;citation_year=2009;citation_issue=4;citation_pmid=2973767;citation_doi=10.1016/0003-9861(88)90002-1" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. E. Campanella;citation_author=H. Y. Chu;citation_author=P. S. Low;citation_title=Assembly and regulation of a glycolytic enzyme complex on the human erythrocyte membrane.;citation_pages=813-2407;citation_volume=102;citation_year=2005;citation_issue=7;citation_doi=10.2983/035.028.0410" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_volume=22;citation_year=2013;citation_issue=11;citation_pmid=15701694;citation_doi=10.1073/pnas.0409741102" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Chakravarti;citation_author=L. K. Lasher;citation_author=J. E. Reefer;citation_title=A maximum-likelihood method for estimating genome length using genetic-linkage data.;citation_pages=3124-182;citation_volume=128;citation_year=1991;citation_issue=1;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. D. Chu;citation_author=S. T. Kaluziak;citation_author=G. C. Trussell;citation_author=S. V. Vollmer;citation_title=A maximum likelihood method for estimating genome length using genetic linkage data.;citation_pages=175-182;citation_volume=23;citation_year=2014;citation_issue=7;citation_pmid=2060775" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=J. B. Cole;citation_author=B. Waurich;citation_author=M. Wensch-Dorendorf;citation_author=D. M. Bickhart;citation_author=H. H. Swalve;citation_title=A genome-wide association study of calf birth weight in Holstein cattle using single nucleotide polymorphisms and phenotypes predicted from auxiliary traits.;citation_pages=1863-3172;citation_volume=97;citation_year=2014;citation_issue=5;citation_doi=10.1111/mec.12681" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=E. Cotter;citation_author=S. K. Malham;citation_author=S. O’Keeffe;citation_author=S. A. Lynch;citation_author=J. W. Latchford;citation_title=mortality of the Pacific oyster, Crassostrea gigas, in the Irish Sea: the influence of growth, biochemistry and gametogenesis.;citation_pages=3156-21;citation_volume=303;citation_year=2010;citation_issue=1–4;citation_pmid=24612794;citation_doi=10.3168/jds.2013-7409" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=L. Degremont;citation_author=E. Bedier;citation_author=P. Boudry;citation_title=Mortality of hatchery-produced Pacific oyster spat (Crassostrea gigas). II. Response to selection for survival and its influence on growth and yield.;citation_pages=8-29;citation_volume=299;citation_year=2010;citation_issue=1–4;citation_doi=10.1016/j.aquaculture.2010.02.030" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=21;citation_volume=6;citation_year=2011;citation_issue=5;citation_doi=10.1016/j.aquaculture.2009.11.017" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. Fishman;citation_author=A. J. Kelly;citation_author=E. Morgan;citation_author=J. H. Willis;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=159;citation_year=2001;citation_issue=4;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Graner;citation_author=A. Jahoor;citation_author=J. Schondelmaier;citation_author=H. Siedler;citation_author=K. Pillen;citation_title=Construction of an RFLP map of barley.;citation_pages=250-256;citation_volume=83;citation_year=1991;citation_issue=2;citation_pmid=24202366" />
<meta name="citation_reference" content="citation_title=Molecular Cloning: A Laboratory Manual;citation_author=M. R. Green;citation_author=J. Sambrook;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Marine biotechnology (New York, N.Y.);citation_journal_abbrev=Mar Biotechnol (NY);citation_author=X. Guo;citation_author=Q. Li;citation_author=Q. Z. Wang;citation_author=L. F. Kong;citation_title=Genetic mapping and QTL analysis of growth-related traits in the Pacific oyster.;citation_pages=218-226;citation_volume=14;citation_year=2012;citation_issue=2;citation_pmid=21932055;citation_doi=10.1007/s10126-011-9405-4" />
<meta name="citation_reference" content="citation_journal_title=Marine biotechnology (New York, N.Y.);citation_journal_abbrev=Mar Biotechnol (NY);citation_author=X. M. Guo;citation_title=Genetic mapping and QTL analysis of growth-related traits in the Pacific oyster.;citation_pages=218-226;citation_volume=1;citation_year=2009;citation_issue=3–4;citation_pmid=21932055;citation_doi=10.1007/s10126-011-9405-4" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=I. G. Gut;citation_author=G. M. Lathrop;citation_title=Duplicating SNPs.;citation_pages=251-790;citation_volume=36;citation_year=2004;citation_issue=8;citation_doi=10.1111/j.1753-5131.2009.01014.x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. A. Hackett;citation_author=L. B. Broadfoot;citation_title=Duplicating SNPs.;citation_pages=789-790;citation_volume=90;citation_year=2003;citation_issue=1;citation_pmid=15284844;citation_doi=10.1038/ng0804-789" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=B. Hansson;citation_author=M. Akesson;citation_author=J. Slate;citation_author=J. M. Pemberton;citation_title=Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.;citation_pages=33-38;citation_volume=272;citation_year=2005;citation_issue=1578;citation_pmid=12522423;citation_doi=10.1038/sj.hdy.6800173" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=D. Hedgecock;citation_author=G. M. L. Perry;citation_author=M. L. Voigt;citation_title=Linkage mapping reveals sex-dimorphic map distances in a passerine bird;citation_pages=2289-2298;citation_volume=272;citation_year=2007;citation_issue=1578;citation_pmid=16191642;citation_doi=10.1098/rspb.2005.3228" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. Hedgecock;citation_author=G. Shin;citation_author=A.Y. Gracey;citation_author=D. Van Den Berg;citation_author=M.P. Samanta;citation_title=Second-generation linkage maps for the pacific oyster Crassostrea gigas reveal errors in assembly of genome scaffolds.;citation_pages=S267-2019;citation_volume=5;citation_year=2015;citation_issue=10" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Hubert;citation_author=D. Hedgecock;citation_title=Linkage maps of microsatellite DNA markers for the Pacific oyster Crassostrea gigas.;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=26248981;citation_doi=10.1534/g3.115.019570" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Hubert;citation_author=E. Cognard;citation_author=D. Hedgecock;citation_title=Linkage Maps of Microsatellite DNA Markers for the Pacific Oyster Crassostrea gigas;citation_pages=351-362;citation_volume=288;citation_year=2009;citation_issue=3–4;citation_pmid=15454548;citation_doi=10.1534/genetics.104.027342" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Hubert;citation_author=B. Higgins;citation_author=T. Borza;citation_author=S. Bowman;citation_title=Development of a SNP resource and a genetic linkage map for Atlantic cod (Gadus morhua).;citation_pages=172;citation_volume=11;citation_year=2010;citation_doi=10.1016/j.aquaculture.2008.12.006" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. Huvet;citation_author=A. Gerard;citation_author=C. Ledu;citation_author=P. Phelipot;citation_author=S. Heurtebise;citation_title=Development of a SNP resource and a genetic linkage map for Atlantic cod (Gadus morhua).;citation_pages=191-191;citation_volume=15;citation_year=2002;citation_issue=1;citation_pmid=20307277;citation_doi=10.1186/1471-2164-11-191" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=W. Jiao;citation_author=X. Fu;citation_author=J. Dou;citation_author=H. Li;citation_author=H. Su;citation_title=High-resolution linkage and quantitative trait locus mapping aided by genome survey sequencing: building up an integrative genomic framework for a bivalve mollusc.;citation_pages=45-101;citation_volume=21;citation_year=2014;citation_issue=1;citation_doi=10.1016/S0990-7440(01)01148-2" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. B. Jones;citation_author=D. R. Jerry;citation_author=M. S. Khatkar;citation_author=H. W. Raadsma;citation_author=K. R. Zenger;citation_title=A high-density SNP genetic linkage map for the silver-lipped pearl oyster, Pinctada maxima: a valuable resource for gene localisation and marker-assisted selection.;citation_volume=14;citation_year=2013;citation_pmid=24107803;citation_doi=10.1093/dnares/dst043" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. Kamps-Hughes;citation_author=A. Quimby;citation_author=Z. Y. Zhu;citation_author=E. A. Johnson;citation_title=Massively parallel characterization of restriction endonucleases.;citation_pages=810;citation_volume=41;citation_year=2013;citation_issue=11;citation_pmid=24252414;citation_doi=10.1186/1471-2164-14-810" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. R. Knox;citation_author=T. H. N. Ellis;citation_title=Excess heterozygosity contributes to genetic map expansion in pea recombinant inbred populations.;citation_volume=162;citation_year=2002;citation_issue=2;citation_pmid=23605040;citation_doi=10.1093/nar/gkt257" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. D. Kosambi;citation_title=Excess Heterozygosity Contributes to Genetic Map Expansion in Pea Recombinant Inbred Populations;citation_pages=861-873;citation_volume=12;citation_year=1943;citation_issue=2;citation_pmid=12399396" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=D. Lallias;citation_author=S. Lapegue;citation_author=C. Hecquet;citation_author=P. Boudry;citation_author=A. R. Beaumont;citation_title=AFLP-based genetic linkage maps of the blue mussel (Mytilus edulis).;citation_pages=172-349;citation_volume=38;citation_year=2007;citation_issue=4;citation_doi=10.1111/j.1469-1809.1943.tb02321.x" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=S. Launey;citation_author=D. Hedgecock;citation_title=AFLP-based genetic linkage maps of the blue mussel (Mytilus edulis).;citation_pages=340-349;citation_volume=159;citation_year=2001;citation_issue=1;citation_pmid=17559551;citation_doi=10.1111/j.1365-2052.2007.01611.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Leitão;citation_author=R. Chaves;citation_author=S. Santos;citation_author=H. Henrique Guedes-Pinto;citation_author=P. Boudry;citation_title=High Genetic Load in the Pacific Oyster Crassostrea gigas;citation_pages=255-265;citation_volume=343;citation_year=2007;citation_issue=2;citation_pmid=11560902" />
<meta name="citation_reference" content="citation_journal_title=Aquat. Living Resour.;citation_author=F. Le Roux;citation_author=M. Gay;citation_author=C. Lambert;citation_author=M. Waechter;citation_author=S. Poubalanne;citation_title=Comparative analysis of Vibrio splendidus-related strains isolated during Crassostrea gigas mortality events.;citation_pages=251-258;citation_volume=15;citation_year=2002;citation_issue=4;citation_doi=10.1016/S0990-7440(02)01176-2" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_issue=14;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_issue=16;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Marine biotechnology (New York, N.Y.);citation_journal_abbrev=Mar Biotechnol (NY);citation_author=L. Li;citation_author=X. Guo;citation_title=AFLP-based genetic linkage maps of the pacific oyster Crassostrea gigas Thunberg.;citation_pages=26-36;citation_volume=6;citation_year=2004;citation_issue=1;citation_pmid=14564534;citation_doi=10.1007/s10126-003-0001-0" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=L. Li;citation_author=J. H. Xiang;citation_author=X. Liu;citation_author=Y. Zhang;citation_author=B. Dong;citation_title=Construction of AFLP-based genetic linkage map for Zhikong scallop, Chlamys farreri Jones et Preston and mapping of sex-linked markers.;citation_pages=63-73;citation_volume=245;citation_year=2005;citation_issue=1–4;citation_doi=10.1016/j.aquaculture.2004.12.015" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=Q. Li;citation_author=W. G. Liu;citation_author=K. Shirasu;citation_author=W. M. Chen;citation_author=S. X. Jiang;citation_title=Reproductive cycle and biochemical composition of the Zhe oyster Crassostrea plicatula Gmelin in an eastern coastal bay of China.;citation_pages=752-759;citation_volume=261;citation_year=2006;citation_issue=2;citation_doi=10.1016/j.aquaculture.2006.08.023" />
<meta name="citation_reference" content="citation_journal_title=Fish. Sci.;citation_author=Q. Li;citation_author=Q. Z. Wang;citation_author=S. K. Liu;citation_author=L. F. Kong;citation_title=Selection response and realized heritability for growth in three stocks of the Pacific oyster Crassostrea gigas.;citation_pages=643-648;citation_volume=77;citation_year=2011;citation_issue=4;citation_doi=10.1007/s12562-011-0369-0" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. Li;citation_author=M. He;citation_title=Genetic mapping and QTL analysis of growth-related traits in Pinctada fucata using restriction-site associated DNA sequencing.;citation_pages=e111707;citation_volume=9;citation_year=2014;citation_issue=11;citation_pmid=25369421;citation_doi=10.1371/journal.pone.0111707" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Lien;citation_author=L. Gidskehaug;citation_author=T. Moen;citation_author=B. J. Hayes;citation_author=P. R. Berg;citation_title=A dense SNP-based linkage map for Atlantic salmon (Salmo salar) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns.;citation_pages=615-615;citation_volume=12;citation_year=2011;citation_pmid=22182215;citation_doi=10.1186/1471-2164-12-615" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_title=Fine mapping of the pond snail left-right asymmetry (chirality) locus using RAD-Seq and Fibre-FISH.;citation_author=M. M. Liu;citation_author=J. W. Davey;citation_author=R. Banerjee;citation_author=J. Han;citation_author=F. Yang;citation_volume=8;citation_year=2013;citation_issue=8" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. Lu;citation_author=Y. Y. Sautin;citation_author=L. S. Holliday;citation_author=S. L. Gluck;citation_title=The glycolytic enzyme aldolase mediates assembly, expression, and activity of vacuolar H+-ATPase.;citation_volume=279;citation_year=2004;citation_issue=10;citation_pmid=14672945;citation_doi=10.1074/jbc.M303871200" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=L. Luo;citation_author=Y. M. Zhang;citation_author=S. Xu;citation_title=A quantitative genetics model for viability selection.;citation_pages=347-355;citation_volume=94;citation_year=2005;citation_issue=3;citation_pmid=15536483;citation_doi=10.1038/sj.hdy.6800615" />
<meta name="citation_reference" content="citation_journal_title=Anticancer Research;citation_journal_abbrev=Anticancer Res;citation_author=S. Mazurek;citation_author=H. Grimm;citation_author=S. Wilker;citation_author=S. Leib;citation_author=E. Eigenbrodt;citation_title=Metabolic characteristics of different malignant cancer cell lines.;citation_pages=3275-3282;citation_volume=18;citation_year=1998;citation_issue=5A;citation_pmid=9858895" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=H. A. Orr;citation_author=M. Turelli;citation_title=The evolution of postzygotic isolation: accumulating Dobzhansky-Muller incompatibilities.;citation_pages=1085-1094;citation_volume=55;citation_year=2001;citation_issue=6;citation_pmid=11475044;citation_doi=10.1554/0014-3820(2001)055[1085:TEOPIA]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Aquat. Living Resour.;citation_author=S. Patrick;citation_author=H. Arnaud;citation_author=L. M. Olivier;citation_author=R. Daniel;citation_author=G. Philippe;citation_title=A comparative field study of growth, survival and reproduction of Crassostrea gigas, C. angulata and their hybrids.;citation_pages=243-250;citation_volume=15;citation_year=2002;citation_issue=4;citation_doi=10.1016/S0990-7440(02)01175-0" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=L. V. Plough;citation_title=Environmental stress increases selection against and dominance of deleterious mutations in inbred families of the Pacific oyster Crassostrea gigas.;citation_pages=3974-3987;citation_volume=21;citation_year=2012;citation_issue=16;citation_pmid=22747636;citation_doi=10.1111/j.1365-294X.2012.05688.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. V. Plough;citation_author=D. Hedgecock;citation_title=Quantitative trait locus analysis of stage-specific inbreeding depression in the Pacific Oyster Crassostrea gigas.;citation_volume=189;citation_year=2011;citation_issue=4;citation_pmid=21940682;citation_doi=10.1534/genetics.111.131854" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=L. V. Plough;citation_author=G. Shin;citation_author=D. Hedgecock;citation_title=Genetic inviability is a major driver of type III survivorship in experimental families of a highly fecund marine bivalve.;citation_pages=895-910;citation_volume=25;citation_year=2016;citation_issue=4;citation_doi=10.1111/mec.13524" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. A. Poland;citation_author=P. J. Brown;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=W. Pootakham;citation_author=P. Ruang-Areerate;citation_author=N. Jomchai;citation_author=C. Sonthirod;citation_author=D. Sangsrakru;citation_title=Construction of a high-density integrated genetic linkage map of rubber tree (Hevea brasiliensis) using genotyping-by-sequencing (GBS).;citation_pages=367;citation_volume=6;citation_year=2015;citation_pmid=26074933" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=P. Ren;citation_author=W. Z. Peng;citation_author=W. W. You;citation_author=Z. K. Huang;citation_author=Q. Guo;citation_title=Genetic mapping and quantitative trait loci analysis of growth-related traits in the small abalone Haliotis diversicolor using restriction-site-associated DNA sequencing.;citation_pages=163-170;citation_volume=454;citation_year=2016;citation_doi=10.1016/j.aquaculture.2015.12.026" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=P. M. Richards;citation_author=M. M. Liu;citation_author=N. Lowe;citation_author=J. W. Davey;citation_author=M. L. Blaxter;citation_title=RAD-Seq derived markers flank the shell colour and banding loci of the Cepaea nemoralis supergene.;citation_pages=3077-3089;citation_volume=22;citation_year=2013;citation_issue=11;citation_doi=10.1111/mec.12262" />
<meta name="citation_reference" content="citation_journal_title=Applied Statistics-Journal of the Royal Statistical Society Series C;citation_author=J. P. Royston;citation_title=The W test for normality.;citation_pages=176-180;citation_volume=31;citation_year=1982;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Sakamoto;citation_author=R. G. Danzmann;citation_author=K. Gharbi;citation_author=P. Howard;citation_author=A. Ozaki;citation_title=A Microsatellite Linkage Map of Rainbow Trout (Oncorhynchus mykiss) Characterized by Large Sex-Specific Differences in Recombination Rates;citation_pages=1331-1345;citation_volume=155;citation_year=2000;citation_issue=3;citation_pmid=10880492" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=C. Sauvage;citation_author=P. Boudry;citation_author=D. J. de Koning;citation_author=C. S. Haley;citation_author=S. Heurtebise;citation_title=QTL for resistance to summer mortality and OsHV-1 load in the Pacific oyster (Crassostrea gigas).;citation_pages=390-399;citation_volume=41;citation_year=2010;citation_issue=4;citation_pmid=20096029" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=C. W. Shao;citation_author=Y. C. Niu;citation_author=P. Rastas;citation_author=Y. Liu;citation_author=Z. Y. Xie;citation_title=Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder (Paralichthys olivaceus): applications to QTL mapping of Vibrio anguillarum disease resistance and comparative genomic analysis.;citation_volume=22;citation_year=2015;citation_issue=2;citation_pmid=25762582;citation_doi=10.1093/dnares/dsv001" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_title=Candidate gene polymorphisms and their association with glycogen content in the Pacific oyster Crassostrea gigas.;citation_author=Z. C. She;citation_author=L. Li;citation_author=H. G. Qi;citation_author=K. Song;citation_author=H. Y. Que;citation_volume=10;citation_year=2015;citation_issue=5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Singer;citation_author=H. Perlman;citation_author=Y. L. Yan;citation_author=C. Walker;citation_author=G. Corley-Smith;citation_title=Sex-Specific Recombination Rates in Zebrafish (Danio rerio);citation_pages=649-657;citation_volume=160;citation_year=2002;citation_issue=2;citation_pmid=11861568" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=P. Singh;citation_author=M. Salih;citation_author=J. J. Leddy;citation_author=B. S. Tuana;citation_title=The muscle-specific calmodulin-dependent protein kinase assembles with the glycolytic enzyme complex at the sarcoplasmic reticulum and modulates the activity of glyceraldehyde-3-phosphate dehydrogenase in a Ca2+/calmodulin-dependent manner.;citation_volume=279;citation_year=2004;citation_issue=34;citation_pmid=15199064;citation_doi=10.1074/jbc.M402282200" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. D. Tan;citation_author=C. L. Wan;citation_author=Y. F. Zhu;citation_author=C. Lu;citation_author=Z. H. Xiang;citation_title=An Amplified Fragment Length Polymorphism Map of the Silkworm;citation_pages=1277-1284;citation_volume=157;citation_year=2001;citation_issue=3;citation_pmid=11238411" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2008" />
<meta name="citation_reference" content="citation_title=MapQTL @ 5, Software for the Mapping of Quantitative Trait Loci in Experimental Populations;citation_author=J. W. Van Ooijen;citation_year=2004" />
<meta name="citation_reference" content="citation_title=JoinMap @ 4, Software for the Calculation of Genetic Linkage Maps in Experimental Populations;citation_author=J. W. Van Ooijen;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. P. Van Tassell;citation_author=T. P. Smith;citation_author=L. K. Matukumalli;citation_author=J. F. Taylor;citation_author=R. D. Schnabel;citation_title=SNP discovery and allele frequency estimation by deep sequencing of reduced representation libraries.;citation_pages=247-252;citation_volume=5;citation_year=2008;citation_issue=3;citation_pmid=18297082;citation_doi=10.1038/nmeth.1185" />
<meta name="citation_reference" content="citation_journal_title=Molecular biology reports;citation_journal_abbrev=Mol Biol Rep;citation_author=C. Wang;citation_author=H. X. Wang;citation_author=Y. Li;citation_author=B. Z. Liu;citation_title=Identification of a fructose-1,6-bisphosphate aldolase gene and association of the single nucleotide polymorphisms with growth traits in the clam Meretrix meretrix.;citation_pages=5017-5024;citation_volume=39;citation_year=2012;citation_issue=4;citation_pmid=22160517;citation_doi=10.1007/s11033-011-1298-9" />
<meta name="citation_reference" content="citation_journal_title=J. Shellfish Res.;citation_author=H. Y. Wang;citation_author=L. M. Qian;citation_author=X. A. Liu;citation_author=G. F. Zhang;citation_author=X. M. Guo;citation_title=Classification of a common cupped oyster from Southern China.;citation_pages=857-866;citation_volume=29;citation_year=2010;citation_issue=4;citation_doi=10.2983/035.029.0420" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. F. Wang;citation_author=H. G. Qi;citation_author=L. Li;citation_author=H. Y. Que;citation_author=D. Wang;citation_title=Discovery and validation of genic single nucleotide polymorphisms in the Pacific oyster Crassostrea gigas.;citation_pages=123-135;citation_volume=15;citation_year=2015;citation_issue=1;citation_pmid=24823694;citation_doi=10.1111/1755-0998.12278" />
<meta name="citation_reference" content="citation_journal_title=Electron. J. Biotechnol.;citation_author=J. P. Wang;citation_author=L. Li;citation_author=H. G. Qi;citation_author=X. D. Du;citation_author=G. F. Zhang;citation_title=RestrictionDigest: a powerful Perl module for simulating genomic restriction digests.;citation_year=2016;citation_doi=10.1016/j.ejbt.2016.02.003" />
<meta name="citation_reference" content="citation_journal_title=J. Ocean Univ. China;citation_author=Q. Z. Wang;citation_author=Q. Li;citation_author=L. F. Kong;citation_author=R. H. Yu;citation_title=Response to selection for fast growth in the second generation of Pacific oyster (Crassostrea gigas).;citation_pages=413-418;citation_volume=11;citation_year=2012;citation_issue=3;citation_doi=10.1007/s11802-012-1909-7" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. A. Ward;citation_author=J. Bhangoo;citation_author=F. Fernandez-Fernandez;citation_author=P. Moore;citation_author=J. D. Swanson;citation_title=Saturated linkage map construction in Rubus idaeus using genotyping by sequencing and genome-independent imputation.;citation_pages=2;citation_volume=14;citation_year=2013;citation_pmid=23324311;citation_doi=10.1186/1471-2164-14-2" />
<meta name="citation_reference" content="citation_journal_title=Aquacult. Res.;citation_author=R. D. Ward;citation_author=L. J. English;citation_author=D. J. McGoldrick;citation_author=G. B. Maguire;citation_author=J. A. Nell;citation_title=Genetic improvement of the Pacific oyster Crassostrea gigas (Thunberg) in Australia.;citation_pages=35-44;citation_volume=31;citation_year=2000;citation_issue=1;citation_doi=10.1046/j.1365-2109.2000.00388.x" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=Z. Q. Xia;citation_author=M. L. Zou;citation_author=S. K. Zhang;citation_author=B. X. Feng;citation_author=W. Q. Wang;citation_title=AFSM sequencing approach: a simple and rapid method for genome-wide SNP and methylation site discovery and genetic mapping.;citation_pages=1;citation_volume=5;citation_year=2014;citation_doi=10.1038/srep09939" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Z. Xu;citation_title=Quantitative trait locus mapping can benefit from segregation distortion.;citation_volume=180;citation_year=2008;citation_issue=4;citation_pmid=18957707;citation_doi=10.1534/genetics.108.090688" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. F. Zhang;citation_author=X. D. Fang;citation_author=X. M. Guo;citation_author=L. Li;citation_author=R. B. Luo;citation_title=The oyster genome reveals stress adaptation and complexity of shell formation.;citation_pages=49-54;citation_volume=490;citation_year=2012;citation_issue=7418;citation_pmid=22992520;citation_doi=10.1038/nature11413" />
<meta name="citation_reference" content="citation_journal_title=Aquacult. Int.;citation_author=X. X. Zhong;citation_author=Q. Li;citation_author=X. Guo;citation_author=H. Yu;citation_author=L. F. Kong;citation_title=QTL mapping for glycogen content and shell pigmentation in the Pacific oyster Crassostrea gigas using microsatellites and SNPs.;citation_pages=1877-1889;citation_volume=22;citation_year=2014;citation_issue=6;citation_doi=10.1007/s10499-014-9789-z" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_title=Genetic map construction and detection of genetic loci underlying segregation distortion in an intraspecific cross of Populus deltoides.;citation_author=W. C. Zhou;citation_author=Z. X. Tang;citation_author=J. Hou;citation_author=N. Hu;citation_author=T. M. Yin;citation_volume=10;citation_year=2015;citation_issue=5" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A High-Density SNP Genetic Linkage Map and QTL Analysis of Growth-Related Traits in a Hybrid Family of Oysters (Crassostrea gigas × Crassostrea angulata) Using Genotyping-by-Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
<meta name="twitter:description" content="Oysters are among the most important species in global aquaculture. Crassostrea gigas , and its subspecies C. angulata , are the major cultured species. To determine the genetic basis of growth-related traits in oysters, we constructed a second-generation linkage map from 3367 single-nucleotide polymorphisms (SNPs) based on genotyping-by-sequencing, genotyped from a C. gigas  ×  C. angulata hybrid family. These 3367 SNPs were distributed on 1695 markers, which were assigned to 10 linkage groups. The genetic linkage map had a total length of 1084.3 cM, with an average of 0.8 cM between markers; it thus represents the densest genetic map constructed for oysters to date. Twenty-seven quantitative trait loci (QTL) for five growth-related traits were detected. These QTL could explain 4.2–7.7% (mean = 5.4%) of the phenotypic variation. In total, 50.8% of phenotypic variance for shell width, 7.7% for mass weight, and 34.1% for soft tissue weight were explained. The detected QTL were distributed among eight linkage groups, and more than half (16) were concentrated within narrow regions in their respective linkage groups. Thirty-eight annotated genes were identified within the QTL regions, two of which are key genes for carbohydrate metabolism. Other genes were found to participate in assembly and regulation of the actin cytoskeleton, signal transduction, and regulation of cell differentiation and development. The newly developed high-density genetic map, and the QTL and candidate genes identified provide a valuable genetic resource and a basis for marker-assisted selection for C. gigas and C. angulata ." />
<meta name="og-title" property="og:title" content="A High-Density SNP Genetic Linkage Map and QTL Analysis of Growth-Related Traits in a Hybrid Family of Oysters (Crassostrea gigas × Crassostrea angulata) Using Genotyping-by-Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/5/1417" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Oysters are among the most important species in global aquaculture. Crassostrea gigas , and its subspecies C. angulata , are the major cultured species. To determine the genetic basis of growth-related traits in oysters, we constructed a second-generation linkage map from 3367 single-nucleotide polymorphisms (SNPs) based on genotyping-by-sequencing, genotyped from a C. gigas  ×  C. angulata hybrid family. These 3367 SNPs were distributed on 1695 markers, which were assigned to 10 linkage groups. The genetic linkage map had a total length of 1084.3 cM, with an average of 0.8 cM between markers; it thus represents the densest genetic map constructed for oysters to date. Twenty-seven quantitative trait loci (QTL) for five growth-related traits were detected. These QTL could explain 4.2–7.7% (mean = 5.4%) of the phenotypic variation. In total, 50.8% of phenotypic variance for shell width, 7.7% for mass weight, and 34.1% for soft tissue weight were explained. The detected QTL were distributed among eight linkage groups, and more than half (16) were concentrated within narrow regions in their respective linkage groups. Thirty-eight annotated genes were identified within the QTL regions, two of which are key genes for carbohydrate metabolism. Other genes were found to participate in assembly and regulation of the actin cytoskeleton, signal transduction, and regulation of cell differentiation and development. The newly developed high-density genetic map, and the QTL and candidate genes identified provide a valuable genetic resource and a basis for marker-assisted selection for C. gigas and C. angulata ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
    <title>A High-Density SNP Genetic Linkage Map and QTL Analysis of Growth-Related Traits in a Hybrid Family of Oysters (Crassostrea gigas × Crassostrea angulata) Using Genotyping-by-Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"-KGJ7A1GuquGjRM8sHUahPqwwsbzwSZ1DdakkWJo5KM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460134","apath":"\/ggg\/6\/5\/1417.atom","pisa":"ggg;6\/5\/1417","ac":{"\/ggg\/6\/5\/1417.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/5\/1417.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/5\/1417"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/5\/1417"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/5\/1417.figures-only":true,"\/content\/6\/5\/1417":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022F2OT0KD1zf+JiDOjedITdIH9Dadf\\\\\\\/Pcr6SPEhEb0lUt1TAEiIrDsGxJUBRtSa57t1iF5SDlCpVUuuteu5fpGO4dc2t\\\\\\\/V85k+OABBVVa7wsQVvjFajSRaAyesu4Pi\\\\\\\/hUgCjK819kTeuswqkYYo69jpwgqgxjibsDRmn5FGTPheGJtIQBduTqOQ4Z4X2GHHvQuqahTFRJCg1kJTVzaMVqRo9vMYpjaX6YzxVLJnJ7VHuIE2StkQKzT75CBweRd739a5zWjUY6cEIpuyHRCDwIgkPTxx8lWx8kHKFMj\\\\\\\/zL1VkPmmvHFPfnWiMHIlbGyEanrlSdPwDuzhsAYWIJhIt0W+yNfRemaSeVR4hmcXP20tkf7ugxQdqCqTHwJEAKLiKa8npx3KyIsQ9aHC+oeebpPGHRM24od7YOoO9+8oFFSbsCN+Oak9b3au\\\\\\\/NXNOgF\\\\\\\/59ahfMyhG5rn1rbaXlHgJq3RAvh3\\\\\\\/M7rpmQIKpKt8MXQDD5u8NEair67o6BJGLUHGvgOk1YrE29Glh8goWNz62oj1gO9tUA1AeQdqg7SUOv2soBjzrL2uzl4C9TqKar81SL\\\\\\\/1HpmZHTjgFFvcrDXgAm5FauDKRGhUA+b8+lo2VTj\\\\\\\/827rf3w98Qi4j0HVYJEaOl7aX\\\\\\\/ILl9qQpKW0CvE10c+MXPJIYS7sfCX6TXhMz72CCc\\\\\\\/ra6rg04+GdYO\\\\\\\/ZQnqR5hQUPBFa4O\\\\\\\/tllZhyeRiIDElPqys2vWypBacKs+4tTHvdQqIbw2T4AQgTj8M+rtwR0S4V9j0lqO3XQYTlV08D1yqxDp8l3QFQaeJsjB49ht404IX6TLQX\\\\\\\/9of2y93+RRHffoZtliUpWHK+ELyiYKMnpQOzWtzIQU48f45\\\\\\\/\\\\\\\/KwvZna2nro6\\\\\\\/FCOZJHptefuYkgrF8geRIrT43p8Ay9nAsUGg2MdhFTi7O66oAuEtO236BaVKw1o+w0nMH\\\\\\\/uosKvHcT5PtN8dvJlhf8iejl6e59x+N\\\\\\\/ajuS5DVupAm6fIeQY1BpB8N6aljW2u7yU4Ysyg\\\\\\\/\\\\\\\/UWOkZSfPVEaCC9eMhvjQ4kVbUjLBxN3lf51XJPJKbDOXBOHonr8frSA9dvWzXskCtFVoFxPd2Fsgq\\\\\\\/WlOiFHjwAsuLMwB6VDT0QX8k\\\\\\\/gpz26hy\\\\\\\/h3maXsJp1s2lmY1UhxvdJdAS+Q9Fh7dzkViABo6nIUXTik0yr+GnlXXl1vM959HhG\\\\\\\/XVcsd69aBm4wbxb5lKT7AfHDnY8pZvF0Rgb5pY3zKFBOunPPuz4AjejsXP4WLKfQsbGlZbOLPA9g2Tr\\\u0022,\\\u0022iv\\\u0022:\\\u0022aui1j+scp+zyP7XSb+YAdQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229fae5d1a8bef15379e931b0eff8844ee3ed1dc5ec4e5d4c7f1bdbf53c275f32c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Ofia9Avk2FlG5BhkOBR63R83bSl7TVfktipVegfPGV2B1ZqXou2nuq5mvila38mBxE\\\\\\\/568PgJMtyjLi\\\\\\\/C2NRfvmNuLq57zP\\\\\\\/hPeyZFqHkkl9IcvbZe7RrgQvuQOw91TcHbWzPstBbYlPrP\\\\\\\/M5Cwm8peZ5zTxMdASYiUiv8ee2wtzqA5cu1luIzCyut2VF3hLJ1hu2jj9w\\\\\\\/5PILk6KsDzu9S\\\\\\\/o7ipV3N6c5zGfYFU9bT8y35JkjBg1nIk+9RXSfc9nPiEuy2N4TPySz9FZGEB36fx9dqDIkRvtURCnx62xRlpkjrnwQdVYLwILT6M2qmP6AmwgavLZ0i5HfXD1F02UrkSfHnHS0tYK1du7AiHYT8KSP9OFPY38rLj\\\\\\\/q\\\\\\\/n4uMLPqtmxZU+Jf5jSN0oTu5sYJItAfDyZS3cBuCDbNApLb+bTcfyTrfn43NCeyUMk\\\\\\\/hBf6H4\\\\\\\/neBUGfwwVt06pv3RjFu+boO\\\\\\\/B1OodHIPseEK8NPx\\\\\\\/sT64LPvoG50MZ2mnyeL7gbOdNxw\\\\\\\/2Xe7aUrOmkbrGgtJx7rCTHuc8a4QNkwhmCZDjlUPyw7KG\\\\\\\/OqDyjYv3jtBSu1VbopKM1HwctAnWmqm0YnC0bGayJLtRZwFktss1BeUU587e5rzqcXNDVYenJhAb3SxHPugjR5HIptTxcmwhalGklzGbbckWX5WLpSoddDqEDZzEChNpdjhy7W0B0yS3fCSp3AJBjpb5tjhNdUbyA33eCQCiKicGhq1bHqu+IrdRj2igwnWt1xCP+k5bnOR99zNK6910tmBpqNgDl1wUbJjbNX7trv4J+rgtzziSeA8+jkcyHK9kFP3AACWGOTCHD4MkrF71UI0NdeoYPnPv8mlsedmP7Y4\\\\\\\/l2mZH5p5K9VG+445hY1Kj2nvIZ++lyVzIwJfm8JXQFL7XvwFOKw8HNMn4xaGXcI4XNJlYBW4owbB4KRTyNkL5t\\\\\\\/PZHNw9ZAa41sMXfpJPS59xnAO6CvYg2uEqz+dxCVdG6gfhqARUs2gRKcTTmTBX7cbMF+mUeTwtVLMXVpf\\\\\\\/Fe7Pna5hEHGKtQAENP62Fo9RMn9vjWFgSI\\\\\\\/GbI5EonbUfeM373DWM6GLroIkRzHw7ZVBq4cmqhmZykbYliaHX9XRuOlsuEvqsnhSyrQVARXWh9VXJN20Ej08uj+sUW3HtktHyLy+gfiGR+KrCL6Ra1fh\\\\\\\/sBn8pbbVwh09SzJXkpp60T0fzePXPxuA1kgA67s4n+7AID0ywjZst2y69ayEVxpPvJ3rb1zwTtxiEF4Mz2z\\\\\\\/u6czSpTYeTREhFDGQSxqvm5hmon00wwCH9wjbOIdgUTxntVZuZyhylTCjFK3\\\\\\\/8I9RTDtmtGGquexWnbWk28BbqJk3\\\\\\\/Wr+hgZXfqDc4yrp6u+inxHvQHyTAi9ur61rWY6p31jI2Cnf8qCApXI\\\\\\\/eM\\\\\\\/N+MNHs6GfaEqKHeRZ3mIWkEo+PTajyQzUmbClbOtwrQFNobyMn85MdXNRM2Pg8hwWBKtEtB6m7wA1SgHQI3BAdp9ZtiCleBYFS0L61tlqzsoHoo52TYr2qJwWnle9nYkxh+GSzc7gnSV0ibBBCP5ggDc+ao7kvGRrfeHezKjFXTkH2kFPGNcog0NjGK4m+mQsjwObJZzLWrKjLOlE3DX3eYAKXnfXq+IRlLegDYuRyICm088s24IR4qwjOVcuiVG\\\\\\\/OCfVJXROdW8mkOQ2p4u6ybCSShPg26zk0O2\\\\\\\/FRq1+0mXLiJx5DSi4G60h76ojOKMRg05j+7q+8byM3WxJiN0R4LlZjaFHb38xRWD1fGQAinv1aPnp8NyfyCye35xRKcBUWwPPHaKOrBVeR9i68g+ZSZuqIdgVqc0HdaJ8sbnbqTp0mn\\\\\\\/W8pw0HdFnBlqkTmrzg7c9fJyZXSonQLOOu81Sr0ZCyiBoSqu12yjBSQROXhv7p\\\\\\\/l3fxEtNFihvwsSnwofc8gpUfkMw6uurFTCO3KRQ82PVwb5ZS\\\\\\\/2Pw1u6j6pxbO9qGasEX9Z617ocd4LqGpitPQP55AwNMizjOmq41OVjQdVcFH9\\\\\\\/f1qMwCKS2Nl2MXW522plNdTnGtV0Nj45bfQG0l7N7mMDZ\\\\\\\/N3plvzpRuNQIEs4xqgfAx+I1iv8yI7j8EePCcrUDkgJ0VqkUNZXqdJEaBQ2JhbxvjbBBDsTF1JI+uUNKzwJQj+8YPY6u4yr6v+pfL56PhUi7ozOinGvN1HxJ2p+0o5Yw5yAz+HA5Ue+9HdDlq+ckmgiOdsRnIlnSN2OiZhOGQbbn6MJByvbAJVz\\\\\\\/HctOx7m3XCvcHs9ysUpA1Bi51D4lNRYus8yiP8WGg61t4wfnLxQq0BZ+7FZiQTkyqu3hrzWazwA\\\\\\\/OAP2L1IX\\\\\\\/HL2MDQMQikMXUBnJM69KQjLrp3Qi64mX21bV5eBzZLRPrP87lqM8IfOxBawDlUtys0cfS89h9Kx1zdLx4tRY3LyCPLIz9f05elnL2LdOTGF5MpnwUHUGJCO6G9n5S1DgEMJHdti2aEuHsJs\\\\\\\/6t3i5Y1uL2dz1ItDhp68NhMq3o8TnDbobqWRD1snATvy5n2RwvBgIz0pnj+qu7KOTqftCeSfR9qKrSGKKpBI2aOYB6VUkCOSufBWm6D3Ei5Mqt9VD7JNgYIaCQNtwBokKulwsleXaeS8B24eg6baW5tW6KbW7KjsP5mNGysGzz9z3vyXrMD09IGCfJ1MP2mlV7\\\\\\\/W0HUWxYiYZqo7yeDI93NJFeomhMVmLC+8efeh9IUsX0p0UynFQJ6LhGN8Zf+ZSRiwKFiUf8+S5hqgIcnvbxKgh3KeMHJQhS9L6dUf+pAY+50ekrrnNej1TFRIdhnyTUZxQscT4Yx6Ct7E56o8xbgF10NIgQwAgMbLGN9eqSBifvp1Vt1IGw18CXMBYGzIBCMkR4VeeGl2QABAyCYWTzbRJYb5zduKpiHPXjyqxGTWWQgsHH\\\\\\\/KRtZZ5qmc8jZglHjqQdPS2zCc4JmQR8wNRhyW0DuRl+mLbYaUaotu5sHxp9g4+6FrAn5163kmouUdgnhWeO3Dol26Igl71KAKnj6f7rsk3TLHKWSfrcT5XYTXeYr\\\\\\\/pANBUFGHm4EDm0IMjEwK1I+iag75DN44gbUDeb79SAfSqDHyJAiv2yoIX43HgnQho8EH55trW9\\\\\\\/kSxUdKW6dVP2epITKS3NwGua8qI8dIp0fDWt7rXu7c6uCxNtmYnr+NcGVeTERsUdK0OjVSSXTEhaALX7gJLQ7TX48kKfNTG7I+z5x2\\\\\\\/a6i9VA925IVpvOn55YTb3REq4mPbvys8doRgltrh4QbSq\\\\\\\/3EVZfCx8qEbpltybpB6g2I9wEz5zGO6yEINF5sGiyqapNvpXxzF0vmPGgo73+LlKXm4ZPpno\\\\\\\/VOluYoPHHfVCNbP2WJEsPhptyTsRoUoDwksGWoSm5tLG+ly2RzjkAj2WSfIdw97Jc5zSXFAhzyz0DYk9KNkhIFMCuZF80\\\\\\\/YkPsxNlSt4mQMj\\\\\\\/KXIDzktxehXY2fn36rl5BX9QfJvK39ZUYFb+EXcKSRCVEqJEEqwdlXsMvV+PCKEJLZh4fsVjOVARlVBCaip9F+8GQjouYKqhu+kgmuD2kXZAcEqp3muioo7UAXMlS6GTb3JtU9gZS1bah\\\\\\\/PPWgk94lG5ppY4GMDCREDBaPr+mQT1XK5w6hh278F+NjEfTRZC+gshCV\\\\\\\/TH\\\\\\\/JZzAJq5025Qrs0vYyvmRf9OimkTmcAljyPo2vzQk5xHynKchguvZs5HVHYdTH+seb6uR5OAjW8zXAxg5cgnHKk9djwBTqIi5PGY6rSSD5l37gAXgwvSSEgSSyXbehfP+EcEB58MUA37k8JoDvs1NwxcaIlL8c4dvcp6LI3Oq8si7eE5kRQzN94lHb5qbws4UmmZgxT5dJVqWOv+Lsv4lZaW2PMlLRSTCfGXZd3IYrLWsfSMEx54bdjeoVCdeOg0F9190seOuNulZ4tOW0m1Rx6CfhaTY2dJECvfgQF2uthAyDWyYazG+pfZDYHjaspMUE1RN7wICUMq+7znhGeY5WgDSpgreW2qzaS7xtJNXltJa\\\\\\\/x\\\\\\\/Qyf8\\\\\\\/3LQu6zjzGkMXCP4Lo5G2nAbEAnTfbk\\\\\\\/J6bDHfF+OrGgNjuvV482ztmeRxosTh1NsVxLPg7+TltdXKIQlM+FEYBIobDNWfS3L9FXjxY9I8wA5eYMU4mAUSnbQD7Zk+mvqfn33CpgrlU2MRf8NHs5dm92a1pwpxHmxdjJF6\\\\\\\/4IFR+Vd7pin31M5IBznRFwDySCKquBNjuyCphEGmcbSQ1J0zaMIJ2XTl8sXYj3VBkmUZOdItnhatVgR1k6u0z5R+3PxJ\\\\\\\/9f6pegvroHb+xqbvAS7DB58KwdKRzaMYB2Eu+uyi37ERGd3sQ7Cp7vzFl9FiQ4Q+Buj99og1e+vrvpXkgeff3HD5OlPJd2uNUAQ78aZks8JtjledcFW79xHGVMt4pLi1nPvzAPHOIu\\\\\\\/TMKsHD4SrypK8ypHHnwFjXQoFJAyaRzDUmSg858sed\\\\\\\/YAU9QLv2ug0Y23CCK97bZKRNZzYVAYcmhTn5w1keUyc8RXUaAvNpOLSY\\\\\\\/4UZtML3OSCE56Lp4zvgT07B4u9ur2nUSDD62b7576bp+FiEZD1L2Y8chbtWUz+Ud5WE\\\\\\\/566NrsZNGV56UQaxdbgVGqb0kwUJ5VyCi40LxmwHn0vTWe9uaGHw2hWXJAJEKwz42B2uwWPGGtPwZ+o9gPZA1XYskos0XxC4tJF++z3Uu3TO1r5fORBV05HFNSYu5J79QH4V+ZnpZGReEPci\\\\\\\/NpIxtd\\\\\\\/QDYts0hnu79iwJttt\\\\\\\/ifVqHjC9DMgKQ\\\\\\\/pvIo4iu23rlsmB31vK40x5RQglAGO9ZUlHNzqVr6EcHFOxRF6jl7oKUndtOqNNMbsTwvrDSgMkxpGCdHFn2D+M9NHTp3+5Emk+hAUvHesTynrInacfsElfdalsBhT9eq8qIxemE2eemqFl5emebSjCyhjI08wLRfzATpaGZWwsh\\\\\\\/iQDaHDO26Abx16Q80CHfgta5moPliQYyP28NeInGVCV0VzxvAEBVvIiCUs6O\\\\\\\/Vam+QUIImFTm6sakONiBwwLEWTTBiV7kdc9UVkDGVblaZbrS+cNcEB6QkgMWtFETB3+qrSF0S8ibHo2Fp2NrLVshSNJgghzciZh6p2+5z\\\\\\\/bY0Z5vqfk48Ey2F+3fh8bgq8ZeQRdFDJzps5e\\\\\\\/Lpsna3xvZePmFc6mz8hy5BTr2iXTa\\\\\\\/SrDC0xNlVDmUfeVH93oAT4isvxphiqfBr8AkoUD0YvUoY9HjwrHRBIqgpMul1xbFOwdeQmkwJIWtA9mfkInJm8RIJuyt61fXhGppzHCwd+vkLKxaaospb9XQ6NsZiiCsVtG0gn0rECJvY18nLSJp1fPPAiTLflIERSDecgZJu4JFBBzx1EZjKkYe5n+QRMlXK2zjzSeZp0Dglr1OtwcXhMNebfvJxlV2p0qWJI\\\\\\\/E1V5AXbXfzP+kGBpp0jn9SwLo6Q444umf9NnUGD7LhARybeHsdv30p+9fGdq7svYwU2g4e9snDBQspZ6npslhGOKnUtNeO8eqdeYAC3Dl70SDFScB3eZsZp43d36vacv+yKeGJ8NlVILsYZGiN0U2UzVSRkpDBx+31yGLon+OT\\\\\\\/XFXaSqgDMz90lLeRIyOkPu4WvdMdlPEP2iUtUbAKVfKYS+8dgda0cMhp1eQwd5j5ZhlN4einLkeVd1DW+DbR1FMqvBz1Hm2xmgMxt4s95o\\\\\\\/q1s1K2jTnC2g9CpBy3Ih+xvUHZnR0rg8SlLDw4j9xxwmALuu8N3Oa16fGOEmRMhYayToEwiFLZuhSCvsb5E\\\\\\\/0SFW\\\\\\\/TRuEkimsZUELngskUJ\\\\\\\/OA5ba8nkXFRSr2J4uyYSfvNuCeKl7RP3FqJ3r1bMasoOFnJCA9xb1IQBwJC3qfiE8bR1mzGF84ZgI\\\\\\\/VudL\\\\\\\/yJn4nXhQKZYCtWw4bMwUs3tNh4Qkv28tbvLrEohbowmJT4X316ISHvvXk6zS0G4h1Hj5pZiSHRUcjhHld9Uk7l0rcQQWmFHIgA1Dp8+7bQZ7FY86LeiKzm7KlEnzzFZZ02fglocRtTRjE04dNUZSIPGw+AxTLMYfuzZ37wnmGR+3IVVz0c8S6jxEalejuw+4XvoWjW4QwlGKxLtoeWMgQKi9+6l3OlBVCH7DjN4K5vyZWBU+i+4FKkwITj45MgkKn2HpWoLi7DDLL5dHPw8SHCl06JGfDSofm+c02pY9jJh4AMo5FhlBR\\\\\\\/C8F907Oxau5lDSyHK605szVaB\\\\\\\/w3KoPNq2GkpdSshfNe0byJutuRr7gzM+7xvbuKBQpl8XELGj4JR8tQO6axUANUy0RQE\\\\\\\/qopIcYYbv638opj4AnU3g1TrhNukATUpjBulJFppj7bQ7fzMligM4Y9ZAupBj\\\\\\\/YGa2XGsK3d986yazanSq4rzDYOJ+oRWDN9XvvWGcuqWk6q4FbSsU0hvm3oICPQyLx475JwLmlG86ig95e+GGMbmsgbqPFE\\\\\\\/YazTSbdaBljcoMNP\\\\\\\/X03SaVZYUoJYrJmgTTnm7Z3lvSd4pfa4YoIhVku0hKtjKn5UC\\\\\\\/zFROtoCEv0j+\\\\\\\/DN6722lAfZOzFnpzWQXticF\\\\\\\/8gy33cDj4\\\\\\\/xjydYYDcYAK2icdAR5E9mYvLO1F9WsdjRNsYaifJUGYyaZa8POhpsAdWVDvq\\\\\\\/h5txdwSePF+97YSGHJNrXJQdwQM7kd7Ppi6Jvaqgix\\\\\\\/CTRgCrA6LZ8x4BfZNA8ScR+5blXRVU7rayiQzfx4lje77mTYt7dMpgcGTFxAw0AFcy\\\\\\\/e9Z2yVFm+X7AhIvVcMhkrLaQUf76WOEkUzs8W9R8Jiz\\\\\\\/rjvbEDPbt25DgX\\\\\\\/Gt7LR0\\\\\\\/B+o+E5BhSul7e+q0lMNFbUi320SnMOzKOyCduURf1GFQKObRNuNMHg2990wZUlHElS6he+iXzoRxsMjFXEmhcRvCTvAxDVION5TkuH33Dd2UdBe3LRC+u72dxEpxulGYvqx2CleqVZXS2B5Vvc3FIGOju3dEha1uoYLVgFgnAsHn7EUVXFdkn4tO7mr2M+G+7HPJzFeVwCl2OfzqBqp0iaXU8IMieIw3u6yaBEd9rrTdhvarEX4gi3wkkjG4jx6f0QQMBBoICm7LhxqrArFNasUvF+TCgE4igxDITEfTA3u\\\\\\\/0kQFbVpTj1\\\\\\\/\\\\\\\/onUgazZHzDNO6KXjSmWCyB\\\\\\\/+BlbFW4EJclOAb1+Xqe7Z2lxYe5Ds6PpnHSPMYB9dd3EG7LxousYoHnvDK3RzlH++jyjjLIp4eePjTHoH2A0UPg3LE2Phrc72oEh4qvcqhukWeI6Kc+b\\\\\\\/FEWS5THnJupHGkcR9Z7eR\\\\\\\/ULuQBw8Z5\\\\\\\/tKeLA1uHHz\\\\\\\/pxTtC1unnQJxM7J7omjOKt2NdxqWidLjAWklD9lA3XIxcfDRQtgiYgKPP5YAnT2n+dAfWbHWDipjBUZ8P2MMVUsoM2+rM8Owh6I875iLd66xtxhkKxRpOwe0ml0jBILbTB\\\\\\\/vDEimAITScNBysLtzEU6s74mIMVssSoZZRl++EqQICQsxYvPv7+WDpHw\\\\\\\/DaRFJV1n3DLofInZzUu\\\\\\\/Z9UDMmRGVMFSXBT3BkYr+FEnAitfPTTFqcfsbktdhSFENYDUn9tdxYcg6prsy2hVmQ8Jnwsx5h8s8HxCxc5DKHMHN6tKvn\\\\\\\/ytXrj0nI6nEp8GInOXWpA1N\\\\\\\/rgTOGwALOTxdBNPE\\\\\\\/g+S2WGHRI9mZT3DZBp5hTlQG8\\\\\\\/5TPyWHDWXoM+83uLbn2lDB\\\\\\\/InlJAjROuC227Cd1H6VEnxD236sDdgbxqPFvZUr1YZmG8c389Rwlp6e9gNXUR0u5HtZy4BAbD8TDy0MKqjEO\\\\\\\/340R5EFG6yo\\\\\\\/htNERocOY26s4xfVCkI0L0dSPAb6A8PqF0cnD3kAfdylzfRvxaVedit9JLw6erwmYu1q+lGlx3XtMh\\\\\\\/M65ZkrfdqqEVOgyG3q9cMt52VAD9g0YpBEUOR2gzr+xIX6sfmQmSOr6IjtPKcmkYgLhWWljW0uq0ERZww9XoM5ScQKJJkjqzf5ubP0JDc3v2VWi4O9SlA+eQ+tf1vH2KbxcYXAmUs27ClYoMKZWGjKxN1EmlnPV8ZV1TI+MvZ4pA7pOLAvsIS1gLmEW\\\\\\\/7Bdlol\\\\\\\/3KpeSCfd8en1egiGMkf68PZDNcxvfGKmBC1Vno19lAGGbhURh86FR05EbEjU139BC9evv03FlVVbl7k\\\\\\\/jvHwyJZLBca7+gLbftGODwDm6AfyM2fszFGOZiloHIF6Q8jBB86iP+1gmsHqR0rkCfegDV68473RTpFslu58tESLNAzgRdC1tYDiKiKS3xLceWpin9OK6Gav2XWEtIy7z6fzS+K6LQXurgNVZv8noCIqiA6ONhvYfkvm1BswgXxMoSVFCgsm5+U61JjoogcJ3rUDOsn3wkM4Ms+Efui1LWETp3rWrYTmUlDUg1nyMpHWIB8gRNuHK\\\\\\\/b3mmUb0sboDK4\\\\\\\/r5H8D31N6RHuW88M9qWb6xQDhPLZJI\\\\\\\/2wMOuf0ki5gqhzC8eoaUGKyzeRCNjGOu1eIv4yMkciIbDV9BMXiViPMbwzQoihCsLxUqZUavoelXrYfXZ0sRpXk7wiOOMvk8dR3WelcTmP+UPg2chQTEouibtSZdJPRJYevxmrIixb+V6H22s8Lce2FAs2OWOuklI68Io9yu6De7yEzJoH63EgpqEebLswGdmE4X1yvDU6p023uBcjPyBgMofRUzSOKrW1+qKACazHkVCLjg0FDutWBL51P8KKylVRVtuw+c+SpoyjUXUxFxLdq3HwUg6g0mv559dqb0Dhu35RqBDR+qf\\\\\\\/5qKh6OND\\\\\\\/fSBgpsUlrmpHrzs8hXONXW8dGUvbV5nQYyJ6Mi9V4w5m66abbwjKC3BctPWZlgG2yKyGvNd2zooIkwNvvJ2vNG\\\\\\\/FTn84TnSrScljgbo0S+L0c+7LqCQSMQLAXMd2TwKj9Ccmi9dw9hhBuYac8zjlXD6v\\\\\\\/1Ej3rlVCyonr1edO8InbrzAV+tL89zBJccbGZJFwJNRGj9RoBfL69SHvSP0dIQe+a0X\\\\\\\/KZKm2c3t93SocIooseU1AfdJ2zKQzux2sySodKJ\\\\\\\/XIJ+e7RLMIM3lJlmIYvCXx5O2r5lvCM0\\\\\\\/MrFcQQjeRWeWqrduKf8xH88uxGT\\\\\\\/tUnLkoWYwGTLKAzz7kqRlWWwTVGVdJv44CzdsH7chnvH\\\\\\\/A7UHWgW2HbcTC\\\\\\\/VbDI5\\\\\\\/vFZOTn+7e0VXb2qS\\\\\\\/tVvIgJHg9rDYEk\\\\\\\/XqiV94PIs8dztOMihJB1SrAOgxPErp9hr+mb4hr0q8Kd34eHPyAvf945Xff+zXtOp37j6ZTwOpJQKT6ZPcWaDvkiB\\\\\\\/oBespiREqkXyEsoSg0ElN\\\\\\\/FaRCmaRe5pSnjCPSL3s4xtth+YHnXWvrtfNBHDcTzFe7e+Hs1YGuTjCwi6LiwXD1TnRUEviSzh\\\\\\\/Oku8g3fs8zXhNiw9d5wE7MuI4h5yHJJ59K\\\\\\\/fGKduTK+l+sqfSttWDoM6mXAxH3395Pig4PATU3v0T7+Iyil3lCOVriRTjsdOqSwgoQZVJ918xF\\\\\\\/P7cR91VG1TF9g7LMzhEFTzmDUh3UZncC4bdGROxjTiFQCQ8LbMYXserf99HjWi+C\\\\\\\/1Az3ViLrjPBbVOaeFWw2X84tC2wqGfiJJ7AKbmWW9pe5n2zaCThdbga3EdqkVpSrUAPH6lDNrmZFJ9dRt9yURaRpzhXe3YkxDpruQsP0AsE3jT4ohkMf3rcpiDgCUunYOaFJZXzpMlGkwfixFkNc16hMoDqV\\\\\\\/23OcsJWDNt6YrNHrCaBDLb3+PGVn3MMkhCkhKXYJ1OLwoiz5Y+uF1ot6i03QVQMr+5eHIcNtsfgxy5Fgfxb6UZ6blDY8CPPq3+clETpkMvZ5x6pgZ0QAdTw+E\\\\\\\/RKBf67xmpzYjvL8AioJQviMUIdoFw32E8p6gK\\\\\\\/YIl5LrH79mvUolANkBwbviUcEleXwDl6TuLozC4g8DwCPxF4hLUvPAF5ClKWB\\\\\\\/O5plKIFXnzSxtSySUlFadtc+ocFeEg7mh8cKm\\\\\\\/Mi8KiQCAQOwDVcFcwViGlHT0AiLni\\\\\\\/MAuaDP9LGynOLCfkqvtyBqRh1f14qYmWN2orUbHCdYH2wfhEny2VIdGWFUHH6KbREmXvRS53KwzP5Ws0zNNGcjbmBnjy\\\\\\\/Wt6G92l3ROtslz2c8N6lgIJ2K3GxLJukxUKcoevfPqeA2KbadfNbGwp7wTa8VN4kPszgmj1ZCOIbieeBOrxm07eLnwtK3hz5\\\\\\\/B15fxTH2pl32tCkNXBZD8lGEyW\\\\\\\/qw48kPOO\\\\\\\/driOHp\\\\\\\/gsqJdPOh4nApeCgouFj9lEt3Kg3zaBEQ5etbxQ3zF6jevhMDJSp5aLGQaCLpeUxeSgXTcpnIFmJEhpI3J6m4+QO0Pi7He2SVXEC7wIUiwVRr1NHTNZ3AuBWgrE1xfySY0qqtHyk2OfYCvYF08uRwUwpJ9Fi6SHRQVuCbhC2igFvwYTGlAdAeVRG7yp4ovp774RX5Bj+9UQnrfrDYP8Z4kMiKF9dz\\\\\\\/IO\\\\\\\/5AEAGwCu6W52jEfCb75n\\\\\\\/dctIa1c0q0w2uRcinHrL74dHxTaiCHcHGR9ozWzbFBVZall7cknHBFu3tI1kcUHj55C8KgaRSNxUf\\\\\\\/+4UCQ4UB4FslHztxxiQKFkHPfcuREYG777Oo4Pd6ZM\\\\\\\/MyMWOvT\\\\\\\/\\\\\\\/c6V7P9ZNPwedhDRrH8NeSa6it6cgUXFvFcsbOgrhh4MgOdcIBzFC4n8x6I+Z2ZgBeJVepyliJP4oP8zJJeaLh17zt2xf3uR3Tr6+qsX+9t1D9zIEdf1wMLi2Lcz4jtRZtYGdl89+0Vb3Jwn61w2DvtGavDsPtqhRxsDtkVppk7ftLuRaYBeCvutzjrmnqWKgdpIvRQbrPnNmCa983RvpkpnY3uDXCaA1NOOqKNtTMEXjHaV0dZFr1pDPFc5rmMWyF23lFn9adGvnK+6C1RKsmXjG7FdUUNuMMh+iWvQ2pAiSyQhvPacPwEGRIf0OjZjj8QtG4k6Y0joj\\\\\\\/5jds3QKlIp4UyvBZ3Mn+8BMc\\\\\\\/Mb\\\\\\\/kvgDfS1yYdrO5zvBv0S4H2Rzda\\\\\\\/Vx4t3fWGAdix5wD\\\\\\\/yi8TmC5sQYXpaYsib9++BRIHSKCeV7Q0FZuO74mr0CS69ZiiGsyPFhr2+0riEZo31sp8WNZA+Od9CowRibETz6+4dgFvuweTRz4XoR+lTpJ0IrQ84AZ9J\\\\\\\/qSm6z9zc2iUJhpK1YK5qxB1b\\\\\\\/FO571hLVjEcRM6wFPC3n7um1qL+mcS5mEtK2RJC24DCIb3eU4r8ASG6TRUlCZMh4WMa7YNiFjInhmz4ceGbN5HCrnO48k6LTH9UOode82o\\\\\\\/Sa5HEAAICKvxfpDcj\\\\\\\/BF86hdvVk9YHUhGGs0mLxXcKIMJN7K09JzAi12w\\\\\\\/lie2gkN8O61uQxVCpDn3N3RlKTAOGtd8hBEOLybetckJmWvep0+dfmX5Suzjps6r0DwhQFyXPNdtCaSCxuq2wvp1QFQkMInizsZDpDeElaPLax8XhhOk14+aAJfg\\\\\\\/tViyFadU8KECEYIOSjRGup9sHG5zZOTEIQ\\\\\\\/gRGKJsA2YI6m3OZ2R3KtpZOaHSox5Gz5uPJIRt0+lKljX23t6pLiuU0wl5k6EmuPtkTPXNq\\\\\\\/QEZbYs1gKy3AKPJyg1I1SzUh70V3GN9fyLwEKWb26x7BjY1jBbOC\\\\\\\/wzG+EPOB0uSpKM4XGoYtDtrJcu3DhQ9MCcrbaTisi2xFXRGhWJqJ\\\\\\\/uUcdvT+l4PWqPTzKsq+xiYC8nLSlneCyMi87vNACDyeYiPilvIgQ01qHdJwCwoD+G5sJoV2hj\\\\\\\/tEKgOn2tHgl9LrgW2K+VMvne46F0j0RXfMWwqjIIseQEwFShTAjNN3pGu1ypw3IOmj6LR7tg5tbEhYW6NuYeN8kREX8Qe2A0c2J4lUKRntjQBba04lj0WEWEs2tB0I3qT8IUN\\\\\\\/K7e6L\\\\\\\/59PbEuzEaLzpTR84nDqiXJ3dkc9n45KRdSlP1Hz5aA5uVTedjRp2PFkVUPF6LzdVHR42TExXaBbJzN3sIwBAZgZ0WudG2l0KPygm9oCPS2zieED3K0qlHYj\\\\\\\/GdsnhJ2bO3+kFPrI4vQ4JgbiSzY\\\\\\\/MN6Gl6dlV3RImpFzjpRc3xHLgbtHOyvPr1KNZWUIghlKz2FnDn4mJHy8iIOYxPD\\\\\\\/dd+elFJBS2DTu7ThBklT8McgHMJDuWR7ZC22kw2k5D+wwT50W9mSGmUcSAMMPJFl+fdLrXdpzTH2ipXOcKol4Ru8M8pxOSyZOTRHRwJGLM72nNF4pGI8SVGA6Sd9Yq6Ng9bQtPDbzd1WbInLfH5i3xjVjN1Wq60msxvi28+As7OquKrRHZT4uvcfEZX8CBXj1ZxWkK8tCKkl84\\\\\\\/2h5WMkxuoL52INIs2wtt\\\\\\\/DXNwp+I9aZAn7aVJmO1JAFEX4WEQwRkWs0JbxdLaZnAEz84hoyAll8xDboDAIzrfvi8ECGC85QNRCd2Mio3+01\\\\\\\/a3eApdeKxmbU1kByISXFECh1fpH2NeRAdz0WBEJHz\\\\\\\/551eT4Boqu\\\\\\\/BgrTjnH\\\\\\\/ir\\\\\\\/ACuGa+jEIGfoFONJ9cCTSqzZeWN5Y9k7w9nnMO3Nxlpqo8AQKDC8Eszq5xb6tbK+1x6Yxke2z5CCEb1EjtlHtF5ON0pgyXhWfvWLIrJ5or8I0p9ajzghV+Yuo2KwGDBCKviqVxOqgVvdAEiIga8yskRzLyDWO9aShefAguN3P4mVZjcr0qeCaZNXSiImlM3iX9wbb6J8fEwKsjDnIAl6ZNcIvxi1u8cib3mj1s4Eg9bSFHAkWPSfhmnZ7ny5TSwVhQ3\\\\\\\/Tth38XH+WLn3Iry9jAiSYTWkvz1hFgW5nLcVy5ZZqZRVwOTThAYCsCCtl3g1IeH4p8u\\\\\\\/sP4an06WCjLeKuW4Kg8x8evLM8Xe18V4zRbAj2RdBD9BtfRxg+50ppkOkIYHD\\\\\\\/c6m11dsZ43PTOpQ7LyrgN6MJ6HM4JfMSI+\\\\\\\/rjNOv9pA5sWKHHZdk\\\\\\\/Ubq3mCR\\\\\\\/c\\\\\\\/YFpBiTIvV6Kk5lclhVeTA2nZb6t\\\\\\\/QijN+ixcXy3kXLNrHN+ltgB8YndcNj0PJjfkIfya2faP5H3HaW\\\\\\\/plVkTazYiKBk4ITC\\\\\\\/n30YCCfukkVyXk9KQ8e86Cb4F81jpzhQjFDA\\\\\\\/yUwFL6jnrnVeC6Y18UocBtXQuzbsuCD2b+tmNH3DQnaRSueqPFfttR4fLF6DIYun\\\\\\\/52Epo6Kue7fzNzP0cbxazLxtxSzlGYSw7s4VmJxENJONUMXgAADjf5evcmXVLU3ocLfls2ZCI5b3S+EkFQEhJRWa13lMpU1ajvWbKoknENRD8vxwWPopi7alTh+Not8EL+UGwt6WGUhaYBWKWNONMRy2Q\\\\\\\/MdhoYg7GXO9HjMIjeWCylZVVLQLCRPc5TA5gnMX10zlDy3fBtMI7lWdDRCLN3er0lUzsJOQ9V08R5MB24eMT71V4j3gyOH8TBrHcd47oNVN\\\\\\\/5c3vbmgFobiLkHc\\\\\\\/2MoPZHUjg42bxgcSeFM9TCM+ESqfZE2DqqB\\\\\\\/dJt\\\\\\\/JSmUpKDqAf5HP76NPkwPOKvMCA1IAh7NiKfkke3U563nV6tun3xaO11mbdOLXS78zyIX51O5DsPMTU9ofOblQfYiGVgbhKKOFmqezxbRxrGyDCJVVTHWTfRgdTSqLX1weOlJ+SY6d3KbwzKaAo3bl2Kv0wHkouOfzzGXJTa14dJRTZMZMDEhVg1yQ\\\\\\\/OyIsgyOqiT4wrFHalICMZNKX\\\\\\\/dVz81QcWf0ouXNj5vnaASgS+NXQkcKpZAkSBVQVagMeKrPqXPD6ZrujyeuvyhVMbsbRlu8OerVpF6yronW3JlWb3cf2R\\\\\\\/UKq8NO\\\\\\\/j6TuJWPciA9OlG6Zt705KkHDRKNaxxqov+y7jL3ETUzreEIKVXKkXh2Yc84bX7yqcSRCi0pMB6UgT3uNK7tSupOKOZvQ8rEgZvt+ehM\\\\\\\/BRw1kPXfmabLxfTzug5VtBvFZWt5m6+BdLDwy0mp+CEyxsHVXhJiW0bsOb4RpTwWVcymP6IHvsBdb3Jh3fTtbVVBqIyEkP2QzSZtQsNekxaRvrFtVddAHNHiMfcpOlVAUeKHwWJAb8ptRvyCig6ogZe2A6KnmOzR70KPDOfUVsw2HPlgsbBqrncy0NVngsuYlFlUg7dG5x6jpScUSsOzkOIj05WcEdWDeJ6LeWoY+\\\\\\\/Ojx2WUjXW4HaBQBe3+98oL72eOEU8ho0zzT20w8HgjtZQXWwHwHv0S3ReLyfZycpwwXJ0b6gdzuOz3XuwJJ7Tr8L39yU6enDb9SdT63CltIh5DiedQfarrTrwmgIqICz5sH86rOQIr4DtGGIUlH+n\\\\\\\/nApJ5m3jMgQX7QPfRDYqvXOu7kWx+NkDA26kXqKG8J1IH\\\\\\\/1olX6KtWwq8E3gAsRdYAwDkvM\\\\\\\/QMCDnLArEB4ovR6lA9BGDkeO24V32\\\\\\\/ZMq4q+EVBj77zNAgcQ9taEshcxfE+YrO0RR3VjiidnOdy4KcnQPGQMmf2AJJ769NEsM5YimVviwhOPyZwM6XYEflwBI2HtCTDR4Id2H8i6EYEdMsqyAPEq\\\\\\\/cBE1lzf7FeIetoBFd6rurYG3\\\\\\\/7aPlL9B\\\\\\\/0qjKHzhkegGZ7Jpx+zpbGWN4GUUGs2\\\\\\\/pifJ5VRgYkINTbEbEfmUSCfOMgwjcNJ2xpi8P4\\\\\\\/UIWLr5QWWSw3YMEK+8Le12nr\\\\\\\/j46jII4HRXZCqUWMdKXXnfbo2FFkZK5KvrnLDfoo6BSrF1QhbqryRq+cEHm0a2lStx5xSUA50QGDKdGI0t\\\\\\\/mr96kfr6D8zyVwn2Ggze8Yi0BQJrjTWaEq2YLAs1u5Xg3IQY\\\\\\\/Ys1uDgDfgB+dzMDh\\\\\\\/qyU4J5WjE0OWhc20KpwYdLGKPjvbGoV17YKdbNlGD46lKydeXKV91eHB3CqtZ\\\\\\\/jkSDSTx6GRWAb0OWRv0vFJlCgleKMSLHufUA7hJ5+JKrTsPFYHOIbOUHaJHYmwEjnZK09S88iGeXOIAKwxFxdi5rVNl9l5pv0zN0qf5I1q48kSZIjrC26T\\\\\\\/GKL3JszuedHQmx6fujYd1fUhf6s1WF6V6VdiPuTi20X61CUaZQrNxxPqo\\\\\\\/yoSAGcSiI+ZaezMzmWtM1eXTr4t8HG5uQ6n5WlIvMNQp8Garmz9OcR1tXHxiQnXDMQbWOBthDf0Mn11pVdZkR7xyQOPNDfn8l757yAJk2o58SZjdwagqhfdU3g+nlgD2PdTIPsFoh2e4S9YWMz0gR6H1kSFAMRCXH1ODDCr3JJNxboa63zXfb0eGQyOz+r8yNfXd2BtSF+HPlixZobYTCpz08wi\\\\\\\/xsaNJRjDb+DwWEXECWLtk61QbahFLnJHV956D8\\\\\\\/yKQv\\\\\\\/zcm1DY4qOkUeWrRPwAPCoSFXTXiVRRfSt6jEYmKGMOCezrCtXwd9BHmmtsn7112HY8jGv5tVaiWybGhdr+JduJrk0zmAjaZYFqR9audK6VStqzQ5u\\\\\\\/8TsoQSAgoiNTSwmI22+Td0iIxOqGJYnB6lMpmYr835WB0ehN87hAllcDp64vVGv71EaCi586aUuUg\\\\\\\/ApAgJd2WrVZEo6cYjzrAadrA2npMAgX1nBFXL3axfcUjWt43iRwWQWjUCym5fkFYkTiGgjbqwz7H9RgYzD8w\\\\\\\/XWlktXPdErgbvKN50kuQaa65k0UnUEO8U3g3hEmedrtN7GnHdCCFYedNh5R2EmxHksW\\\\\\\/jr4K97YLI\\\\\\\/rNYM2FNrazTPNknG2vwcemKXqw0xZ5gaxCi6zGB7B1hnx7Qx5UGEBD+PQ0vjFSt3pKclNQGxcs6VENDqr2ew1mLlpz\\\\\\\/DP6nDoaxONv4KXoYExy2kibjpN5Zo4fdDtLrEpg3yKFI1oNOUDB8X2aApvNoFILTaHT4osps9SnPCT1w0WQK3kZMqlytwKk5Cve9zzopJsXVHLuMZbzZdzy1eDTXPw14LTN8y1y\\\\\\\/Uwca3oH4PHWe3lCEixHBJykmfIMofbBjTYRQE+BB0CyTsJ\\\\\\\/MQ0NE4T3mjm8MMB\\\\\\\/w6LH+wOE2NO35S1LfgIAiXgkdzYgGdASuxUTVhG+6IRmOBUi\\\\\\\/nOAYIThUm2zKXl+Pr\\\\\\\/UyYTtr1JoXtLe6OknGGUVsUwZmcdr87lxTYhzJnoOdFlQmVsa0PUh1Y3VDg9T7Ss08HdnI9YprFzq\\\\\\\/7euvKFRjdDVHUYQn+uorMUagGJsxiCYan3YvXvq2q54jqaI21HgN986AmB5w4EL1ZdZooS6Kpn\\\\\\\/FJcgpJNivd6bbb9xfpFjxKYoUbJ0YRfeguobRcuWsT1N6UFZN8zjHCyX5eO\\\\\\\/Tal4+zfgmY3M4opcpEzG6mw7TgrDNmWIqhM\\\\\\\/iUWEP8Ac1QBqwUMrVjn9goN5U1F8aH1+Gl5qNFyHv8vv7INUAFz3wm0+Axg77ASK0sQbG7+UeEk4MPCEZRH69JIyBm59dFllnNNaesXmbzcmkgFY34NjZdH0fp7b3GrqeHpOpugPAbLGyzX488GnCLX3+dxFlaqbhaMyD8RnErGBF5QdYYzI2fu+8KhyEbjMNc5FsK9VKJunVurV9nbJraruqEfF9fFHg5DZlh+pabRauQvKhQqj64uvYVaEivhLbi0ZN3kNPmxHx\\\\\\\/Pn8vTuGy\\\\\\\/Bx7v7zBzIkVu04fgQwuPoJBSxUvk\\\\\\\/hmmAKwHukfk32jFiRX54OBP2HSyNkoUpHKPoVkgo2Y8L8iK6lwa9buh35yaSdO4CrV\\\\\\\/y+q+rvNrrZeR7fA9Uc9BA5mQ\\\\\\\/5jtx1fKU\\\\\\\/Jbw0oJdrhoq5yCoiGddI70FocR799Mvh\\\\\\\/PGA5oTmfvmKZfIIENKoD0DkY5PaNE2gjVE8GcSQC\\\\\\\/LqZ+baERaqrB7jTtujgex5ItsNiPOcrp7Hq2ckjJfvmOm098p7xn4e5EKMqrQDM34Q42+jf3vxBDXeogNcLF3I\\\\\\\/fURq\\\\\\\/Ak54mQnzCfBlpKLRSQTGibAfMUG2NW0BTsyadKQ4zQueI0KRnvkxUTC+6V0MnX7H4thL+acsE59RDOm9B7KquAO7c0OQY7C1nQiDmFn\\\\\\\/bRYh8rugInhoIbh6LfglbGapnkQbFKjBlRDLEBXEW0hhnftDLEUJiiEeeRMLeexk+R9eyvpl\\\\\\\/D8VbKHZMOK2151VawUPdfOUqTBeg1Iys1tk0v\\\\\\\/hEOvXs\\\\\\\/ikeBxUwEcsrBbEmt9IBG1r6x0Ub42Qp+emw0elRaWhRxIuNak8ADbUc8TZahwegDfvXPCmmE0Zp\\\\\\\/gvhb38NLx\\\\\\\/ZtpFPojCDCx+Aqw9oeDbotDZ8vNqUkywyj8Uz7Yja821Bvvx2m3KXPxkrOSL3FHF+523kw75L8Q34H\\\\\\\/2flp2D4178ccpsESX9hyI8C\\\\\\\/OpUMOrttWCj+TkM0uZPxixopbw4E8ME1id30p1D8LFTG1S0\\\\\\\/+nJ1nm+oKI\\\\\\\/YB75pa17Kj5wLEDFCcglnwXrOiVQH5TRYnldqruz4kqEzy9kH6QXuKAt8teLoPEKpCyJH4giaysZ4cR8A6saYU7ahMHWw5LUjwQjQPeNIZ7aBXyvJ2eqBFz6Me6hMxKP8Y67st+FdzEXFUqBpjkjhhh4Tm1lUIp32Zsb5XmopqBcPEBl43fnIZ+gzGD4dDYNTjrcOAujkoKKavz4Ram714SO+Jzcsq2B5gnegX2LS9WmIZuYByGdLjruzoYTmY4IEPlA7cWYwZwiGZTfksSQuwJDjzalii6GRCxgi2vZdMMNh04HxyHJY8H15WQf\\\\\\\/Njq2djLRRbhQIgUmC6kiHURIbNRVIPznfWg3RpLsvS3LofcxzhGuh8A46ZOaX\\\\\\\/ZTlNoQV39iOziLjBL7mWSQGL2YKqFoL9NDvl7kYczPm3gBU6Cu0uqE8RO4RhrtKb+M0xpb4Yo1BZYJFPI6fKOqLiKewbR81douT95IpbvESUoIHVXb+GljpCIMDnSpp9O8e+FWp9v6YjTQP1WXe7mCeEm5+9aJaDATpEtk9HyP6Go7aSzTwUK51yW7Zri6p2oeRkjO6XZgw0b5oY+bGst\\\\\\\/thojve\\\\\\\/YmZ+draZRU7SsEZI3WnnKt6KMCYlT5EBB\\\\\\\/Wnw7NKpfjLkMno2sscaupMtGZ3O8W2C0s4Up\\\\\\\/5Ply+KU2nON\\\\\\\/XLrLO7QQRKamhg\\\\\\\/UUHtMBiKJycfKhajH5YvwMFcv+uZgi7oPGPk\\\\\\\/ImWfXb\\\\\\\/KOrCWOiTmF\\\\\\\/yoVvnFK9I3ECcoHVSyGQaKc1+6BPT\\\\\\\/EwGVJknPcMTSY9uYXBoHFgB2yUYVi2eJomb7P1MMx7C03842XS+GqjonE+NDvNfNLdQyTU0h29VKvCpWWQChteC+66lwA53b\\\\\\\/Pj7YMtHKlO8lvngVTk2U6lBxd+YyfnNJhLwv4vUsIJtOEm6B70iMIq9nR6WLb6KnFhX4DWqk0ZWos3TkA\\\\\\\/NOTJA4gks4n5rEq89IOm8kl09ORnAp5BQj0iv9w0GqcXWiZ6IqKXRULRi5b8HPvIVF3pSWFEbY01zBS4J7PUfY8mQfr+BFgoQaeO2pDPChdQ89wFgwVp9NaWl6ys5G9+voUS2sp1WA32rBilely\\\\\\\/clTnlZoz3eNUNkI3UBGnGPxCHWU16HgZPdaUQy7CbUWlIoBe8pAwUiloSRdxGlg\\\\\\\/ryi7sbGSLyPJx\\\\\\\/VmaVe5qiQhnvdS4b+B1d\\\\\\\/TNVlIFwOiWbLVzTHZrHzmjd+9z8C\\\\\\\/sF0LbCcgy8raNpzixkHZtE6cEa1oKrugjDOMnaPCZGVAGVM6qBW9aPnu9AeX+S2o4pQf+G1ZEikzN74GB7MP8zBoi9D8tbXbeOZO2uvl0f7FEOpjoC3RZ1kBCThamWAQZ+qXEj5THifEUX7itovV1QDxOd+CCTyIrgbkkkeGWNAMvfSxy2m+krnCdR0vtQo+HSNaZvwEpgzrp4bxeg6DO0A5Z9d9G4G7gKpLeIXNPceWmiJzE\\\\\\\/HblayTermgFAzDAQBfd9lGWH62gI5KAlR1lhFFGuAckUyJTj1DcYkU6jbFknS+ucHFuwcIbKw2LmPVr0Oj7pSp3oKSyzM8Ql7LuViZxhABC2v0OoMIFmXiQny46LbdH4XlffPDvaCrlxqHD6bFLc0jm4LqJLrJxuYRnXXCIGQnMq7BIEk2OUDFbXjrNHzi+qvDkZxjqAtu1e8EXEy0UaR\\\\\\\/8EBhsB3IWbkV4yZMaa98pL\\\\\\\/J4QgFviry6e\\\\\\\/XS1tsPmEsHM0iAo4Tq5T\\\\\\\/Ys4mcab3wEHTnm8jN5P63SwpbmXvyimXElZ8Z60Rnw2CtnkUPOMJyqOs2L\\\\\\\/EvDeQmLOW2M5WLE1jidkBs7OJ0gj7OENMNkZmqjm8BnKvX04FFlH15Im4EPMo5Grx4ndxdRccWEpJdFPS1t1WVW7QYlmr4wk2A7abEhgAUsDVyqWvyhK3Nw4JuvJu\\\\\\\/ofO3c3fXQN0tcWd4OO3j7u\\\\\\\/MF0S2Zo1UaEFl989zdZyx4YsQfbO8X247pDdvMtP65RlqoWgT0j9vRYYB70KXOAaE1Wtb3deqlhkYJ\\\\\\\/vU83WDWul9rZT6Q\\\\\\\/8K\\\\\\\/e1vC\\\\\\\/zR9R4FE7tRQ8QDyCnhMrhYurbhD0ra3m5hMxXrMuHo8dN672N5yT9DB6jA9P2Dj1+o4sNYP7\\\\\\\/G8wZ9C320tYtZkwrxKh6C7u3KCG7Lmlo8S9amvE0U4qKhkSH7LqPTze3bWKkspcdWZqANahlygwlxOYF4Shucq+otEgxakH489IuZqT\\\\\\\/5RfmWiNIQsYCJifU7TF4uE+tsKF2Xl2vQZFida4Fa9+eRciK95PymrxDxkZwYLROGWVyomQkNOoKisn1H1WbECsyBzTHiGhABF8DiiOZwJxZi2+ybcJFpr5kheujPCtxaHcr9clN\\\\\\\/21qMDCkSZPKoJqFU4pu03G9REsQplsvEcdHdw9r+8ykhZTpm+\\\\\\\/4l34W8kPhhTfJgiVk559i6D8UBYXLdGq8wW5rXZEM\\\\\\\/NwsR4ocDSNMIDokkw9LUSa63GlppUZkuvZEcmnS4iyzwMzGXlEDqPMqEbuOLIvnZtjIFxi9iespl11qX3evLy9R81N383mM2iMLIKCXiDrkuhybnOxGlFymRMXMSSJVp+i7rDntUrjLBWKn7VhGWfYsQvG17FAkEGT6k2u\\\\\\\/jwcxcfxlx3D8ag59PFUxUsfF4578X0OOyMvt9U5ZTSjhg4XuaD1Jchp1dXWt7af8tRzVPA1bymNOXZsIsxSwrbg09el84xYnYrBCK6cHrcalVauka36xiklTczdcA+98DyII\\\\\\\/Zp0WYvt1u0afSiWa74yzWZrU3dOCBh\\\\\\\/+hspWslReGDD1Flhqfrpn32OF8eZ7ejDscopaBnHm9XmKp7T\\\\\\\/FaJZKK7sgkAEzNBvbyfdH3H89bYPQv+iUgnHkXrVWHpn6Tf0q\\\\\\\/Fce3K6glxnvPfnXTZEK0MJZMyDyF8NAUcdya4+ykjsqkdwQDLTjfyCc\\\\\\\/anAwUi8PUe+UqValSASGhg3a+lnIE2BEcN2mLH0wgXi2TI3cs6zjHAkkgkkWxs5i1eUu6hz5HQa7ph\\\\\\\/Ix\\\\\\\/e8Wjq6rEsK86MmFtCJptd9DOePIHVOetriFl02yDpSijRs9A5rb44GbSUeaLLqW9x\\\\\\\/lC1yNzaziNjahuhn6JIVWCtlMs44iQgS2QlXP6b9VbjIgMRmaUv6xdvs00hEHxCuFq\\\\\\\/Rp7IdBnQ+4cU4J0DiJMDBUGLOUTrQtMt0lUcWhAdCCUwS6eKRr2iVP+49TFCiOEYgQpfG4NTOAg1LCHhWEQtjA+hHsqFiFo9Yb5qJthP8JetUmQ0Bydpljo34i0MZdJtWSKjy\\\\\\\/iSbM4MQILIwwToR8abQGrbh2rytb0nrNgDY5r+5NEgLwFskhZx8JmYJFpHi9eak3U03JfEmtVFguwOPwR0Hgl\\\\\\\/YAlhVd43RDJTeCK2i30nvtfAaj+Kf6N\\\\\\\/u+jmpy6DQq40UFwy+\\\\\\\/lJnTsktZxb+S6nwfHFAbRoQh323npVK+qKn5iSgoTSeUd9Ydw8wzUgbzbFJGIkUS4XMwierPsMvZfpnsEdhVzah60trKcHS75iojkDDobOFhhTOpbaTrxlHDPGWQdWFhxGQs1Qp4wLIbN\\\\\\\/DTyocqSbdXnbWgrcqYr0VJN+NdkCMI8YK6XFHDBGNpkEtmXt4xTiIXbRueJmPod4NEFBRNHiYwK\\\\\\\/nh35eh+TGLLvl1ktisVyniEGm71uXoOb1pq7iLJVnJx\\\\\\\/2Y+7Ib9\\\\\\\/WQi7PXoSxAs0JxDZELwKNz\\\\\\\/ph+Gw7kz4yisiwxtzUaKxeG+JpcgorJoyFCQjbZYzVJATqQMYxry2lx+FE0HqD9ojcW8+IuX7kuBwkhnWLOHFbIidIpA+VmY+rgryUOVeZ67xRi69ihFrDuhp1xXfUJv5fAYxIA20g23LcJWrZ6XH1JXt14AtZeWYL58SIfF4aCXfH\\\\\\\/poz87RCvPBkj5UT1krTFG4VenDLiYVHtFOJdBWYHT4drB8w5+QeufNSShjqd\\\\\\\/asJXUudmWXS2NLbNVD\\\\\\\/CuT\\\\\\\/B5ImVlyqNF94YNQ5rm1Cb08WWD6Xe1oiQ4Akm5cJzD4uLmLFmwF152nTNEDivWCr6eS59e1g6SvGXh6tpIft63FdoGoEix2h97qeFBNEq8swVckIV56SkJQ9toaFYBU2\\\\\\\/+gVQxS9jia4z273pH25qgU127kifJ6JkmYT2IuI3nAqVw9xwQhhpuIF4DIFhNH+TNv1sxp1SrmUkH6F3r5EudHDJZpSF+xfG\\\\\\\/huzk7Q5MfLUxj6JLAx64utZ+XVTiyDEcuhNQ1UIuxO\\\\\\\/nGxjBwxqrIKJJWrgpgD3c23TICgRa+mHv9T8ikQ\\\\\\\/LN11xTQeSVrY2819XwgJ\\\\\\\/TLm5EkjQDmjCHLZXzdbmOBPaAQY39DyhIwbSd8Dqx9\\\\\\\/2kl\\\\\\\/dVO5jCN3BVuK3KsIWpbJGe+rS4RxnGYJqV3XVhkonh5PCcuBqXkcIVg1cwOD7qQXP6aH7zp7f2ndFfkG\\\\\\\/v2HH+l\\\\\\\/d9B4XOdRICfq+VPO7u6Dwpw\\\\\\\/5hB+OY2535Pao2YqLLd6GW+Ew2s\\\\\\\/o1GWlEN0Q8WIpL0s1cnO9r4NqIqFJB5xoB7GT6GOb0b0U5GSEce36JHWBQCsAbFX\\\\\\\/7ZAio5j4v5+WTmcDoJ3qT2zGls6wiFvQo7+fmZH5f8ufen9bqL8B230MlK1k+OxYOdL95mp2bikKrJXsZ8vuSGRRbNP\\\\\\\/gtiabm5hQZL0d84j61DZoAu7OD2IXOLIegIjIOv4A3qOfVW48j+EFXepj1KcQ8\\\\\\\/wvbc\\\\\\\/SwYACq0kPIk2mwtRsWJt0H0OY1fwuV9vCgCj+3hxMIoNT4SwIy1WQjgxrYDyIx6\\\\\\\/yX7Jaaj3qckDfI6ql\\\\\\\/zqGvskCTHK9kfMmTKuQDKA\\\\\\\/y6Z5CdQEAg4qn7YsDYedJsTFdgSTRc12ecBhjFWmnTzO6zpWBDdlA1n6ghqgnq2ys0K8IClMOQH\\\\\\\/eDx2LM7Uh87f5VZJC9uxlGLoyk5ynWLMQ5L28wuKYGZG6XdGMtrOMH9RnqA2UmK2fRJ8dQTGu00iVVVNUBgxWfwt3TpRbRy87UsFmwC5ri4K8niWOJHaKLJKDhGGt0toiiqiNaVv38XriLbWUb831+lXVm62S6pUlPFPhoSUnklfTrJFwDcJI48fZf3uaLOgNxGlakRcIiqB9xL3Nnrf\\\\\\\/T5A\\\\\\\/eIH47VwJ54fxboJ+9pGVVZLKBvXuUo91xM0KauJXFZnGfdZCaX8lLxDLKt+5s3NUezm4\\\\\\\/c9es9E8AyJpHI4N8c4nOZpNLXAAvN1SJiS\\\\\\\/iXE5hg5ve65y6vV8AMZd2RmBLahlCE8V2QLEMyVT2KTBjVHOVv8v+Wgsg9+2AJwCE56COm2VcUGFIF6GZSMEDx3ajP91JCjpq9gJn46La\\\\\\\/0mbyfGK2uq4c1v8\\\\\\\/MHN4S09X3x5QXSFQboah0nJkTFaWiuyv0KaeXFqWIaXfd+tygmzAheghEF+d7hhzcY0cSgsPfYGB9rQfYuODnrVHhpi3boW2j00jFsDLjEWPxXad7gxKOWnPtKJTGLn5GJEQY2019FadxhYf2tGMYHlucLQCd2JMii0BPcMhrFuI0mN\\\\\\\/HKPhz2rQ4PPz\\\\\\\/1wo1ichaXHfuKx8E1\\\\\\\/XOO\\\\\\\/aqplAcRxXgWPoG7\\\\\\\/AirLiNSz4n8M7YR9my1FLiWYddJiNl7hp\\\\\\\/ZKK7BW6OYJ9jh3gYa1DREjhizgE6ML\\\\\\\/skbgHvX7OedefCbSqvz\\\\\\\/Ls1DA4LVFOuEZ+Toiqe1eTgSJXvyIGeLImb8Vxtlbhc4eanS+8sB8slM+TeEJm6cVUL3NGr\\\\\\\/3zVztrfaa20QL+7E0zpSMNzpbOi6qUuXwlScdlK6828l2y9CoAZYmvAYkbq2SG5UWloS9fkbmuPv6tb0nLmnbWZ9KKwx65F3Y2iRFsI\\\\\\\/NQLAdm9K2LsrhXVFQGnuFyj4DJbknPjASE\\\\\\\/sPF07GxBJg0URMsVhx9nNnuo7kMd\\\\\\\/nDeM5PZhtHNFZzkT9Y0KwfG7Ax3KJV810O4iqgPT44povlI0gFpPYAI035AtnZL2PqRmnMkBG7UldMhC17Cas6ODXmR6UHzpWRKPuGTa5tb6U+XJtHOog7Z18LplJ70yPXRYqkYWi8972GBJeVYuRB2ElGMBYgid1v2Di8hORi77s9pNFaXzArN2PLX9A0KRcPlv0m3pNm66Ft7QQ202CWgNp81VZRR32a5ns\\\\\\\/DFRGjYH\\\\\\\/\\\\\\\/0B0KXiUJIjf2INqvVTqvKWFS5foCaJvCqE9YEbN79S+DQKMgzIYkSGbsjyg3bblDmAvnFlxUcgdPDxq5DRsXE0wHJNqWkJqLI4DYkMTDrbkox6C4TFKWMv4dR+ejJWdXv5K23QF0TcALtruEWVOaZvuFfD5lZv3Jmy7NrpjjM\\\\\\\/rGzYUitMQV7TpuECS6Qea2RAVSEYDcNnHtk4I1uQaCgly2adxWL4dnCvfthGpodCL6rDkJk97wqlmGsN\\\\\\\/LACmU+pgGjOc4ZKjpGsiZMtJI6874qbWy1XY2pNudGFECNPclrKnv8H4x2a5HJmfcvp2RvgGPCCaU9jc\\\\\\\/bMFcB8cr6oOMh9Nwi+GGin6nqxciyq0WVLd8p2xsyhMc7E5bHAuzXU6MX6W8l6Uhm8ERcOhXIbLnAnWo\\\\\\\/f+9GmJb2hvnqZ2gl7VadJ2VNHWB8LJeRyhnpaNXOqRel+3i+iQqQswf2Q\\\\\\\/oM3jg7rY19Hgo3KJA6PdAtcmitmd4r+EPGWimtC2gYs7WmxO2dQJgMA3gUJLfUrWc2EVxWEf0qQfvjpI8Ijj5kFpa5nxv5ymcPFedRksxLu9xVyo9zQlPBdb6EW7ZJ7D3bY\\\\\\\/Le1eCiY02TigZCCGXhnMVwsLpFVrnRZlEdgufYoE81lqDGA6cNGWzn5QL1xrXeiGkwOL+BUBkLPgiRx9u7BRA8ufT4WcRYAWXEG6xJ3MmpdPqO7rphux3x5VMkgWMNmL0pHAruW60Gmy8RnXW6pErqaaDS+RJeXDE3fGCrCvEcUVcqUT1iB5\\\\\\\/pilqt9renY7tGh\\\\\\\/vZvR0XooGzeCCAMrBbOimFMWMZMthYJ3P46T4LuzUFA6NKUyT+i2VYfsPNRpwQwgPkXWBTzg3YBVe7B6pp+5Cv6WZBbUXke6D4h53JMX+5USVuxgJ2JM5lz05jT\\\\\\\/OsplBEPKx3qzTudW04pXFncK7ppyoSRgrQ33sypQHLvFy719PoCbrRXelJvSFF+Gr2vn6yw2mEeN4PV+rqIv2LIlg5VorpIZJzMdYropnNPSMOhlanotu6T7dUbvkVenCRvtKkAuk2GRW8rdd34IisN58iLwAbwEHLsw1eBUBU\\\\\\\/qUXvPHl5oIosbdiUdiUyEKVLp8yVEf5bPICLN3OFwMCOzhAVYqshtiOHtACnuwWm4Fu4ceM9EOlueikBdQx\\\\\\\/bYIn0SL+bRu1kMnhXbfsF+6HIo7CPPLsWJu0fhDndKBcEWqpjUWjpfipk51rvI50\\\\\\\/S5gjMes+vfCxcTFd8jQOdj3\\\\\\\/X78Ei3BXAnqWk6Dl3xkkf8BCJQIocqPuvekGoxSTLY2m5iseAcksTNOvD6pymSuA7SpGqcOh6KzEod2nBgrNfcBBViROtkOaoR5OCDNV4MhkSDK6+f7+g\\\\\\\/teaHpBcmsvlU4PuMlAkLRxvfe1sSuhOGCvvyGRkRs\\\\\\\/KT9XmXk\\\\\\\/YtfC3RkgVw8k3OV2Ov7OmNapsC+9uPbJdWfVQfbVwnwcnVhS7WooEe3MjHnduTplBsaJjqi1PPWkYNWbUOYv2XEOb\\\\\\\/IFuMfrou3R9\\\\\\\/zZBhUyGvf4N9WpkQ21ndIHZOp\\\\\\\/NaWY9qsrzxi8YPB3IzJCjDxbnC\\\\\\\/SwLUSF7vyi4FfpMCMTQcojc+ni\\\\\\\/zIKeWw\\\\\\\/Bedt5Nov7M5GLMRXj4zGlVqnx1DMpzISpJk5jjTDLPJsQ2DdxYd+7onMbojmXvaaKJMVx2ZG9gxX\\\\\\\/kPrp7yZbtFRb3HxFoteoTJ92YbbLdn07iUb0Ye53dlyBwKO6vjmJvt4j9d9hJjDAOJHxCJSLMdLAI0CcfGq\\\\\\\/QcyYdvxhP5lCDRxQW6o8V+DdvETLMMiZP6Eo8DO3VxuVUTYUXmDP4wXeqRNEcrb\\\\\\\/JG\\\\\\\/fI6wY3G1PVL5v2KbS072ULRDipg9oUa9UdcuwODFWigRzWI96k7SDGFEARsbtW\\\\\\\/u+qJqUlSs3MUqXZx72FL+RtvRzmnonrJdIq3n\\\\\\\/6Snop7YdCQpGb0zPn09ymH0FtwNbYHZaPm87yuY8de5RYfrqgukAyFm\\\\\\\/2j15j0iM\\\\\\\/NNZzTDKX894J01yU3\\\\\\\/gJssAxYYsySkQQEpKMdihEc043hz7YOrWchaauhBfGrBIqGwAkUfh9ZzqtNvT6Cqb\\\\\\\/P+UTWxyYVNpOmXdq4+S9VXqQJ8HGFhG8fFseT3oiIhRmHfs4YRaw2y4U93VuVhMPgVNUG0PEE+DxwqPF7vc23LJDWRHxDfExFlaHLP9nG21UBcltt5GBsoYIZH7ISpa+uHcc+hqWjMHxevITEq55QjWMCqd1fn6oqDYNe3WgKJoiRay7LmigrVI6HthSwu9MhXXoyqfKhrObcN7IlwlXh6s4nC5CKldUKWzkFkxlPPJ3ZJoUFWvZb8uwIQDNPmAjHR4Pg7aKnRvB2uYSoD2mXin5w8\\\\\\\/MlaAiTY3fZXxJS2++AmscNxqEovSp8BrzQCEuJcBMAiuABD8lRrbgp2NSHFkrXn9I9WIubyMpAIl9lWTwpxkW2PJUYwxf9WVwBoXT3DupnR4e7g2NVSkw6500\\\\\\\/L1QMbw96W8qfDq95BwHzMQhSrWVnS+9OWHM2xkZ4b063yOQIt6uKvPzVdW9peecuTkL1MIpU2UgGxuiA9nanihBvP6mgG8hryMGeDUIeNmj9KqPY0Aag\\\\\\\/Df13HRZ8sfvJ2y4Wqc1AwTZ+k\\\\\\\/DVL5gHSZx2HX7u0IhzUwNaJ74YbZwE7K9cGyh3\\\\\\\/0Dd9j7x\\\\\\\/FI93k5CPNtlh1PkLnIsAqxGLKcLytHBbjd9MgZZE9wyVhJg8wZc78tQQ8ZX7Z3FhKKlRFj12RA24meq+Z\\\\\\\/iRKrBi5OpkHmwx7Fo+eVbcquDTgs2PxiCtQ758134HqnOZGBxD30g5mJQz91ZJ6VBsLrQ2N1jDzhcYqsSXjCcV5cTjdJujOYG83KlVxqqwjHjPiS3xK2EwJroN6D\\\\\\\/EVVLp1HiGfzr2no8QvCdhaTdJZD8jTaBbRtL2+5xk8IQKkf2MJdhyxhpzgqbokBq044az3ozPKpGDufNmM6F\\\\\\\/RWOpR9Sth\\\\\\\/QSQzTkLR1sJtyUUC8PVEOFyLfiIuKX5QpJdmsieegjRAeVJ4hwXQuDU9h7Acol1aBjKtf71ETwmgXr31ynryEVRfR1\\\\\\\/USGrpAYBVmo\\\\\\\/BtSESU5p3eWofKxORgPIbahHil9fU8TrEsxG0y8sckS3DAYuES7gNQ1uiPOC1Qyt9e6ZkQp7yLQsVXRDVOZE7d045wkHXYGbozmzqjNgNNKLKC72WSMopB6EuFs0xdF1j4s1d8evuDKgVkR\\\\\\\/0KID6R7eiFzIFbawAhGwTqhsv65nbzh7RG1bo\\\\\\\/ZHq8pOqZwWUD\\\\\\\/irfHDvoM0Nbn8r8KAi+K0F91kTDfReCj+Pql4KQfnDbvOyPFWIQyZ0PKHy05Lm7AzJXKXmsxhpf90F55xzxW7RdPC+mDZfkwIGhyK3xx\\\\\\\/5RHT+VmX9hkVLPhJfi8WjrbpL7HH5hmpWPDlE0zJdoXxUGtiX+B0rPwSI6FqKeGtcXFNgc0Ii\\\\\\\/Ee237QGe0ksUd\\\\\\\/klj9fAIAKiSaiWaFFuO9pF0HvxH3scJQ0iL5D200mMx7I9YCFIA182RUwLG2rfvFkdJsb3vJWgt4Vlh9EWafC7pPfbucZTUF66gcJZMaCfyVROv3iHen9kLvxXrjDYnOhptZiLKl3jPST43y4NDiao\\\\\\\/GXgcpHB47Xs7Y+t22J9NIm6e4PckrDxjnx01PQISeEIkE4WLK9u2FI8YFLH33VHDwUQ7VMJfOJwn4U5Yz4RtGtoopX92wK9E84\\\\\\\/ZoIZ6PVJBj8teyphAJQHIu93uZX7qblzyS1eci\\\\\\\/nJfONJ7sqLpyxvp4tuoKn9pPW1h7ZwO2lm+KZ1vlnNAXKlEmwaPsFkfsZeb+iQbZwo5qYJJ6fKc2iBfYrpJQkz0zch7\\\\\\\/yV7oLFoHqI5haEo0cHbc6vctMSNQwdhGYQM1\\\\\\\/qfdm\\\\\\\/FH2gOL6uQAu0+Y\\\\\\\/ZEnKnGsR1DHMRwmGq3EGZHV9dc5P39SxNIQwFZ4BOz38b0BJ12L6xSZvhMLwWRxJKxufLhh2udVClBPF5Aqu4Qxjtmq+RJrNIiee6AtpO0ECFVCX9CfAr63o28l21NtYYi419Sm+k0SooIAYntjBwtS\\\\\\\/6tdsR7YH65q0YAaoxUDdUlm8XsznjBEUrPoL2hnprj3ym0MUTZmzUIbjXKScMLl+PVf6zhwppNUXeLwLwIrttnAYhliJJO6\\\\\\\/96voLKObKAo2TzkaxIbmjf5x89y6Il\\\\\\\/4xtkMq7i3FzmYslL3NpD45\\\\\\\/2inYeIjMPns01DDk9svj4SviygNx30oxiLkbjBu1dwtNcmQjAbM8PXxPmoFOTmSjcUdWO4CVtGClew34fzYQH0ECyvotI6MVRutKVXpEhaq2hjIiRRTG19N0tUwuttlC+mJqaqNHftSZgmczOkap+Iik2OZhV20dwd\\\\\\\/u\\\\\\\/CaDrYmQM0xVt1b3g6GAvoR\\\\\\\/K6jk9wOi3yjmqu8LHWoNru6xndZKmTXZuKjMtdBC2SOsLVi+beG2lqyOoRS07PoFnN4MPa62GHkL31NCQyjdtl5ahQAhjR\\\\\\\/SUzfB1dLyDFIPGtx0bLEpbTFEYwVHon\\\u0022,\\\u0022iv\\\u0022:\\\u00226NXt948ShYC9HEgK6Hdg\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222442e2af612232c00df47d8a7d0fa3702a66e39ae41edf398cd10b1a9dbc1f65\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DmV8sNV8YF2RC7y6HDsk7StA1AG\\\\\\\/iunviBAF+Iqb5Rrt9G5u3szbktHYV8wdmbN4TZXcEXkRR0eBslqBTBD474venIP+iXt0Ml\\\\\\\/vDiTpS7fkAEtwN1FMT9fBOhXxUxK2R++0UeA1je7qlwYtk8McgSb+fWDsUeSdcb9rujrket+4lwwnz77k0mYi8U3ncrXsPBQ9uWHJryBbIUTVRMIGRhBAk4Y5h9r+kV0JyXC6liDHnBvsFVSQebOCSjsBCWZ+GJm5lKgBsk2MiwjyVIoTu3FDVJYodRRKQoI2xk\\\\\\\/XeKVk5yWnaH9M+qnNQMPSI8x+KMhjMAtORdhNQIiR5ubbPeM2brXpZ1vGv9XXXNR8a9+9tNece8wePLW\\\\\\\/MAD3Q2BgZbvfGwCOHM17BtgQEf8ScZ+rgfD9zCHYuoaUpr5F8nJZVY9nBvNJVD6J1iMZfTkBKH4LIuz3q\\\\\\\/v7Xh4yKGeZehZFTYLvkswL4rx9L7ExplRXtW3iifJnzSWY8MFZUAHWNbIAYj5tg+4a1qUYkDMvqD0BfcnL07N4SsLxtw7Y5DeGo3GIayJQcezgRoROnBzdmNggOsoDXAloSxtIVUroP4OROrou8FJNn2ish6l2upGKZ2hYQkkYNY1qYdJcBC35Cy+TL7ozSZuPSkqXWG3JqRCdpx9sCZVYHQMM0QnYSxKd+LwIMOgjUCvosL0Z3dEDjqn9Z0M2cxZI9Bjxw3aFuNCKjCbj81x15M\\\\\\\/VJgUlzFfyRqthDNBXzk0p\\\\\\\/rpH5am9d+7RA8hu+L7F2VBMvLAFFvZHmtmy6YJ8JcggjGgdUmXRTqPR3Ol+IM3Sf1ztcxwWvspr+g2uaxBBDowSy6JrD7\\\\\\\/RqjzIG4HcsBqjZuXvA7WL3uLrfQkSYkJXd1pwfhHxiYnlOJRoT\\\\\\\/h0tXyDZZ2nZkycQcSnq6cdupDbJxeji3QpuKA3NxnTzIZcn0XIYj6ZV3IMDt\\\\\\\/9Knbul2nNRRXZBEZPhzmR3fD+OBZIk\\\\\\\/HQv5hYaOeyav9h1Uj6cvwttksOZolqQ4RzFB2zLi14rK++FraVPZKV4W4GsmthKx6Hg1pd3QXVvQIEzmINUIsOUdlUvxziMOTbiIkAhtoIZeHh1jLNU2688HBwc0gJz5KN+P0b8a90YToJ4fcgpVHZP00AKn18k1zFhFTNZSLkFRhQ9Op7GDuX6KiYr2fwQT6U0u+d8Nx4S+MMlrlUMv2NIbHm2C35uJ9ysddwd4tU3qTcO8Z+r4QLh5RxTTKi0IzBA5PHg12l8\\\\\\\/VPmjJNdBVFEslfGJRSTxJdwVEEPurkXST01P\\\\\\\/3C2WEp+\\\\\\\/cTetq2jH1cr6eisLxENa\\\\\\\/EJIgl7vw\\\\\\\/YcH4OVzaWpXIRzMa1Lj+bAM1GEUyjRvM\\\\\\\/qf8gDZrsxKnFsLTvByYDiKxRUtHRJ8WJfcNJw\\\\\\\/FeHfW78CBMNVg1xznXOqECbA+DO5Ovk\\\\\\\/88MrNu2\\\\\\\/hPHaggI\\\\\\\/YcQD7fEzsITHbdSdnPEsu1zv2USTE5+2zYH5qSa8IICmSwAcUDuqIuzcpTApjGXip\\\\\\\/BzfnqZgrH6d05c9sNwdQr0kaQTqjOauL9Hm6dy\\\\\\\/rX+JtltPOa75YjewADT1f3hvYZKo5AS2NAHjdY6ppipbnwZhZLv+hDfkS1c7\\\\\\\/UZ\\\\\\\/vgfsT3J2UiHVSV8kcH48i2\\\\\\\/28xdVrtQujdoFshLsP4fXq6aq5LRR64VkSy+7LFqMAXBo4TtZv0LB1azM91yUdd3aYpMDsqZcp62frHtX0W99wqKIAjCYWitQMRtzPMa30zgNLtttJnzJVK7wTl2Y8kUy6+Uu6cPjKL3dKteaVFuQZIebZBIB0k7tMac07izLJOnoh8VmvUVzjhIvAG9KJJpbd3r8oAgF1Q0U4X0ClhDLVL9vOdZ+lCR9l4Zms4JzuY4pt1J4TWrfgLuCZr+4RDcMrUTEvugnooOUnkHvm6ssvJV3swspgmPJQUpXfb7s7dIzJZ+v8fYc\\\\\\\/tGuWPTF09AJDB2v2Y\\\\\\\/a0lEf4O0VpI3dl\\\\\\\/0vXNDDFNGitpUGXWBkilNakMQz2G1TX4AYO2buvyMEq6iz9w4dOhH\\\u0022,\\\u0022iv\\\u0022:\\\u0022xf1BisXbkmDetvZWHDUTtg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220e42f58ec7024ef463f50c2413ad0e03d79ee8e31fc764f51cd241d2f435c0b6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460134 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1417" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RhOVvIo02od6hXiktrjhGf2FoCS7QpAb7PmHmPR-cTk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1417" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LB5N0P5NA3UOJW7hzTkF2FQaz8h22rR-Cg3bk8TNQY8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/5/1409" title="Genetic Background, Maternal Age, and Interaction Effects Mediate Rates of Crossing Over in Drosophila melanogaster Females" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/5/1427" title="Quantitative Assessment of Eye Phenotypes for Functional Genetic Studies Using Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460134" id="node460134" data-pisa="ggg;6/5/1417" data-pisa-master="ggg;g3.116.026971" data-apath="/ggg/6/5/1417.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density SNP Genetic Linkage Map and QTL Analysis of Growth-Related Traits in a Hybrid Family of Oysters (<em>Crassostrea gigas × Crassostrea angulata</em>) Using Genotyping-by-Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jinpeng Wang</span>, <span class="highwire-citation-author" data-delta="1">Li Li</span> and <span class="highwire-citation-author" data-delta="2">Guofan Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1417-1426;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.026971 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460134" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jinpeng Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>†</span>University of Chinese Academy of Sciences, Beijing, China 100039</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory for Marine Biology and Biotechnology, Qingdao National Laboratory for Marine Science and Technology, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>§</span>National and Local Joint Engineering Laboratory of Ecological Mariculture, Qingdao, Shandong, China 266071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinpeng%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJinpeng%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Li Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>§</span>National and Local Joint Engineering Laboratory of Ecological Mariculture, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Laboratory for Marine Fisheries and Aquaculture, Qingdao National Laboratory for Marine Science and Technology, Qingdao, Shandong, China 266071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALi%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gzhang@qdio.ac.cn" class="" data-icon-position="" data-hide-link-title="0">gzhang@qdio.ac.cn</a>
<a href="mailto:lili@qdio.ac.cn" class="" data-icon-position="" data-hide-link-title="0">lili@qdio.ac.cn</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Guofan Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory for Marine Biology and Biotechnology, Qingdao National Laboratory for Marine Science and Technology, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>§</span>National and Local Joint Engineering Laboratory of Ecological Mariculture, Qingdao, Shandong, China 266071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guofan%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuofan%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gzhang@qdio.ac.cn" class="" data-icon-position="" data-hide-link-title="0">gzhang@qdio.ac.cn</a>
<a href="mailto:lili@qdio.ac.cn" class="" data-icon-position="" data-hide-link-title="0">lili@qdio.ac.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/5/1417" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460134" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460134/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1417.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460134" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460134/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1417.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460134" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460134/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1417.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460134" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460134/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/5/1417.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460134" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460134/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Oysters are among the most important species in global aquaculture. <em>Crassostrea gigas</em>, and its subspecies <em>C. angulata</em>, are the major cultured species. To determine the genetic basis of growth-related traits in oysters, we constructed a second-generation linkage map from 3367 single-nucleotide polymorphisms (SNPs) based on genotyping-by-sequencing, genotyped from a <em>C. gigas</em> × <em>C. angulata</em> hybrid family. These 3367 SNPs were distributed on 1695 markers, which were assigned to 10 linkage groups. The genetic linkage map had a total length of 1084.3 cM, with an average of 0.8 cM between markers; it thus represents the densest genetic map constructed for oysters to date. Twenty-seven quantitative trait loci (QTL) for five growth-related traits were detected. These QTL could explain 4.2–7.7% (mean = 5.4%) of the phenotypic variation. In total, 50.8% of phenotypic variance for shell width, 7.7% for mass weight, and 34.1% for soft tissue weight were explained. The detected QTL were distributed among eight linkage groups, and more than half (16) were concentrated within narrow regions in their respective linkage groups. Thirty-eight annotated genes were identified within the QTL regions, two of which are key genes for carbohydrate metabolism. Other genes were found to participate in assembly and regulation of the actin cytoskeleton, signal transduction, and regulation of cell differentiation and development. The newly developed high-density genetic map, and the QTL and candidate genes identified provide a valuable genetic resource and a basis for marker-assisted selection for <em>C. gigas</em> and <em>C. angulata</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/single-nucleotide-polymorphisms" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphisms" rel="nofollow">single nucleotide polymorphisms</a></li><li class="kwd"><a href="/keyword/reduced-representation-sequencing" class="hw-term hw-article-keyword hw-article-keyword-reduced-representation-sequencing" rel="nofollow">reduced-representation sequencing</a></li><li class="kwd"><a href="/keyword/distortion-segregation-ratios" class="hw-term hw-article-keyword hw-article-keyword-distortion-of-segregation-ratios" rel="nofollow">distortion of segregation ratios</a></li><li class="kwd"><a href="/keyword/missing-data-imputation" class="hw-term hw-article-keyword hw-article-keyword-missing-data-imputation" rel="nofollow">missing data imputation</a></li><li class="kwd"><a href="/keyword/growth-related-traits-candidate-genes" class="hw-term hw-article-keyword hw-article-keyword-growth-related-traits-candidate-genes" rel="nofollow">growth-related traits candidate genes</a></li></ul><p id="p-3">Originating in East Asia, the Pacific oyster (<em>Crassostrea gigas</em>) has been introduced to many countries around the globe, and has become an important species in aquaculture in many countries including the USA, Canada, Australia, France, and South Africa, due to its rapid growth rate and high productivity (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guo 2009</a>). Of the many cultured aquatic animals, the worldwide production was highest for <em>Crassostrea gigas</em> for many decades; in 2013, global production was 555,994 metric tons, with an estimated value of approximately US$ 1.3 billion (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Food and Agriculture Organization 2015</a>). In China, it is a dominant species in the intertidal zone, and a major cultured species on the northern coast. In contrast, <em>C. angulata</em>, a subspecies of <em>C. gigas</em> (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Wang <em>et al.</em> 2010</a>), is widely distributed and cultured on the southern coast of China. The two subspecies present differentiation in growth rate when cultured in the same environment (L. Li <em>et al.</em>, unpublished data) and the similarity of genic regions of them exceed 90% (H. G. Qi <em>et al.</em>, unpublished data). The two subspecies can hybridize, and the hybrids are fertile (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Huvet <em>et al.</em> 2002</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Patrick <em>et al.</em> 2002</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Leitão <em>et al.</em> 2007</a>). Farming of both <em>C. gigas</em> and <em>C. angulata</em> is confronted with problems, including a decline in growth rate, germplasm degeneration, and summer mortality (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Le Roux <em>et al.</em> 2002</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Buestel <em>et al.</em> 2009</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cotter <em>et al.</em> 2010</a>). Several traditional selective breeding programs have been launched to increase growth rate and decrease summer mortality (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Ward <em>et al.</em> 2000</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Degremont <em>et al.</em> 2010</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Li <em>et al.</em> 2011</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Wang <em>et al.</em> 2012b</a>). However, traditional selection is time-consuming and labor-intensive. Marker-assisted selection (MAS) is becoming popular in crop and aquaculture breeding programs for its high efficiency. Construction of a linkage map and mapping of trait-related quantitative trait loci (QTL) is required as a foundation for implementation of MAS.</p><p id="p-4">Several linkage maps have been constructed for the Pacific oyster using amplified fragment-length polymorphism (AFLP) markers, or combinations of AFLP and microsatellite markers (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Li and Guo 2004</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2012</a>), microsatellite DNA markers (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hubert and Hedgecock 2004</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hubert <em>et al.</em> 2009</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Plough and Hedgecock 2011</a>), and a combination of microsatellite markers and single-nucleotide polymorphisms (SNPs) (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sauvage <em>et al.</em> 2010</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Zhong <em>et al.</em> 2014</a>). However, only a limited number of QTL have been identified for economic traits, such as five QTL peaks for oyster weight on four chromosomes (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hedgecock <em>et al.</em> 2007</a>), three QTL for growth-related quantitative traits (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2012</a>), and two QTL for glycogen content (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Zhong <em>et al.</em> 2014</a>). However, these linkage maps were constructed mainly based on low-density microsatellite DNA markers and AFLP markers, which limited the resolution for QTL mapping. Recently, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hedgecock <em>et al.</em> (2015)</a> constructed higher-density, second-generation linkage maps for the Pacific oyster based on 1085 coding SNPs generated by expressed sequence tag (EST) sequencing, as well as 66 microsatellite DNA markers, in five mapping families with an average total map length of 588 cM, a mean of 584 markers per map, and an average marker spacing of 1.04 cM.</p><p id="p-5">Compared with micro-array based and PCR-based SNP genotyping methods, such as high-resolution melting (HRM) (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Wang <em>et al.</em> 2015</a>), reduced representation library (RRL) sequencing is a more economical and efficient way to develop abundant SNP markers for construction of a dense linkage map (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Van Tassell <em>et al.</em> 2008</a>). Popular RRL methods include restriction site-associated DNA sequencing (RAD) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baird <em>et al.</em> 2008</a>) and genotyping-by-sequencing (GBS) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Elshire <em>et al.</em> 2011</a>), which utilize restriction endonucleases to generate DNA fragments flanking the restriction sites of these enzymes. These methods are widely used in model and nonmodel species, including marine bivalve species, such as <em>Pinctada fucata</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Li and He 2014</a>) and <em>Chlamys farreri</em> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jiao <em>et al.</em> 2014</a>).</p><p id="p-6">To make full use of interspecies variations, a number of hybrid families of <em>C. gigas</em> and <em>C. angulata</em> were constructed to dissect the genetic basis for the economical traits, and to accelerate molecular breeding. In the present study, one hybrid family was used to screen the GBS-based SNPs. A high-density genetic map containing a number of GBS-based SNPs was constructed, QTL associated with growth-related traits were detected, and candidate genes for growth-related traits were identified using the oyster reference genome (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Zhang <em>et al.</em> 2012</a>). Considering the high similarity of genic regions of <em>C. gigas</em> and <em>C. angulata</em>, we intend to detect QTL and candidate genes that have high chance of being used in both these subspecies. These data will help to promote genetic dissection of important economic traits and subsequent MAS for oysters.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Ethics statement</h3><p id="p-7">No specific permissions were required to conduct experiments with Pacific oyster in China. All oysters used in this study were cultured in a fishery farm and cannot cause environmental harm.</p></div><div id="sec-3" class="subsection"><h3>Mapping family and measurement of growth-related traits</h3><p id="p-8">The mapping family was constructed in a hatchery in Laoshan, Qingdao, China (N36°12′, E120°41′) in the summer of 2012. The male parent was a 2-yr-old hybrid of <em>C. gigas</em> and <em>C. angulata</em> produced and cultured in Nan’ao Island, Shantou, China (N23°25′, E117°01′). The female parent was a 2-yr-old wild <em>C. gigas</em> individual from Jiaonan, Qingdao, China (N35°44′, E119°56′). The mating of these two parents generated an F1 cross. Both the parent oysters were dissected manually, and their gonads were rinsed into 1 liter of seawater. The gill tissues of the two parent oysters were sampled and stored in 70% alcohol for DNA extraction. The sperm and eggs were activated separately in seawater for 30 min and 1 hr before fertilization, respectively. The larval progeny were reared in a 70-liter barrel in the hatchery, for 1 month, until all individuals adhered to the adhesion substrate. The progeny were then cultured on a farm located in Jiaonan, Qingdao. When the progeny were 14 months old, the shell height, shell length, shell width, gross weight, and soft tissue weight of each of the 106 progeny were measured. Correlations between these traits were evaluated by Pearson’s correlation coefficient (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anthony <em>et al.</em> 2007</a>) using the <em>cor</em> statistics function of R (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">R Core Team 2008</a>). For each offspring, gender was determined, and the gill tissue was sampled and frozen in liquid nitrogen for subsequent DNA extraction. Student’s <em>t</em>-test was used to compare the five growth-related traits between female and male progeny.</p></div><div id="sec-4" class="subsection"><h3>Genotyping-by-sequencing</h3><p id="p-9">DNA was extracted from the parent oysters using a standard phenol/chloroform extraction method (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Green and Sambrook 2012</a>). Following extraction, the DNA was examined by electrophoresis to confirm its integrity. The DNA concentration was quantified using Qubit 2.0 (Thermo Fisher Scientific, Waltham, MA). The Perl package RestrictionDigest (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Wang <em>et al.</em> 2016</a>) was used to determine the appropriate enzyme combination for digesting the genomic DNA and the <em>Eco</em>RI–<em>Hin</em>fI combination was selected. GBS sequencing libraries were constructed according to <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Poland <em>et al.</em> (2012)</a>, with slight modifications. First, 200 ng of DNA from each oyster was double-digested with <em>Eco</em>RI [New England Biolabs (NEB), Ipswich, MA] and <em>Hin</em>fI (NEB) in CutSmart buffer (NEB) at 37° for 2 hr. Second, the digested DNA fragments were ligated to two kinds of adaptors using T4 DNA ligase (NEB) in T4 ligase buffer: adaptor 1 ends with an overhang matching that produced by <em>Eco</em>RI, and adaptor 2 ends with an overhang matching that produced by <em>Hin</em>fI. The adaptors were quantified using Qubit 2.0 to equalize the adaptors used for every individual. Nineteen barcodes, 4–9 bp in length, located within adaptor 1 were designed to distinguish different individuals. Third, DNA fragments from the second step with different barcodes were pooled, and DNA fragments from the two parents and the 106 progeny were mixed to generate six pools. Fourth, the pooled samples were separated on a 2% agarose gel, and DNA fragments of 400–600 bp (including adaptors) were purified using a QIAquick Gel Extraction Kit (Qiagen, Hilden, Germany). Fifth, the selected DNA fragments were amplified by PCR using NEBNext High-Fidelity 2 × PCR Master Mix (NEB) with the following procedure: 95° for 30 sec, 62° for 30 sec, and 68° for 30 sec for 16 cycles. Sixth, the amplified products were purified using Agencourt AMPure XP beads (Beckman Coulter, High Wycombe, UK). Single-end 100 bp (100SE) sequencing of the purified DNA fragments, in the region adjacent to adaptor 1, was performed on the Illumina HiSequation 2500 system (Illumina, San Diego, CA) in accordance with the manufacturer’s recommendations.</p><p id="p-10">The Stacks software was used to analyze the sequencing data and to genotype DNA variations in the mapping family (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Catchen <em>et al.</em> 2013</a>). First, raw reads were allocated to each individual by running “process_radtags” in Stacks; during this procedure, reads with incorrect barcodes or low quality were discarded, to guarantee the read quality. Second, reads of each individual were aligned to the Pacific oyster reference genome oyster_v9 (GenBank accession no. GCA_000297895.1) by using BWA (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Li and Durbin 2009</a>) with parameter “aln –n 2 –M 3;” the alignment result files in Sequence Alignment/Map (SAM) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Li <em>et al.</em> 2009</a>) format were used for subsequent analysis. To decrease the negative impact of repetitive sequences on genotyping, uniquely aligned reads were extracted by run the bash command “grep -v ’XT:A:R’ ” for subsequent analysis. Third, “ref_map.pl” in Stacks was used to genotype markers. The maximum-likelihood SNP model was used to call genotypes and the genotype likelihood ratio test significance level was set as 0.05. The minimum depth of coverage to report a stack was set as 3. The map type was set as “CP.” The output file type of genotypes was set as “joinmap.” Custom Perl scripts were used to filter output.</p></div><div id="sec-5" class="subsection"><h3>Linkage map construction</h3><p id="p-11">The whole RAD locus was used as a single marker in map construction. Linkage map construction was performed according to the strategy developed by <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Ward <em>et al.</em> (2013)</a>. First, maternal and paternal linkage maps were constructed separately using JoinMap 4.0 (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Van Ooijen 2006</a>). Markers in the lm × ll segregation type (the female parent is heterozygous, and the male parent is homozygous) were used to construct the female linkage map, and markers in the nn × np segregation type (the female parent is homozygous, and the male parent is heterozygous) were used to construct the male linkage map. For construction of each sex-specific linkage map, markers were grouped into linkage groups under the cross-pollination (CP) population type. The threshold independence logarithm of the odds (LOD) score was set to 5.0–7.0 for the female map, and 5.0–9.0 for the male map. Then markers were initially ordered along the linkage groups using the maximum likelihood algorithm of JoinMap 4.0 for each sex-specific linkage map. Once the markers were given an initial order along the linkage groups, imputation of missing data and genotyping errors was performed using Maskov (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Ward <em>et al.</em> 2013</a>). The imputation parameters in Maskov were set to E = 3, and the maximum amount of missing data were set at 40%. After the imputation, imputed markers were again input into JoinMap 4.0, and the map distances were calculated using Kosambi’s mapping function (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kosambi 1943</a>). Then, the imputed markers from the sex-specific maps were used to construct the sex-average linkage map. The LOD score was set to 6.0–9.0. Map distances were calculated using Kosambi’s mapping function. Observed genome length (Goa) was taken as the total length of all markers on each linkage group. Two methods were used to estimate the expected genome length (Ge) of the linkage map: (1) Ge1 was calculated by multiplying the length of each genetic linkage group by (m + 1)/(m – 1), where m is the number of loci in each genetic linkage group (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chakravarti <em>et al.</em> 1991</a>). (2) Ge2 was calculated by adding 2s (s is the average interval of the genetic map) to the length of each linkage group, to account for chromosome ends (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fishman <em>et al.</em> 2001</a>). The average of Ge1 and Ge2 was used as Ge. Genome coverage (Coa) was denoted by Goa/Ge. For each linkage group, the average spacing of markers was calculated by taking the mean value of all interval distances between adjacent markers. For the whole linkage map, the average spacing of markers was calculated by taking the mean value of all linkage groups’ average spacing of markers.</p></div><div id="sec-6" class="subsection"><h3>QTL detection</h3><p id="p-12">Prior to QTL mapping, growth traits were tested to determine whether they were normally distributed via the Shapiro–Wilk test (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Royston 1982</a>) using R. If the trait did not show a normal distribution, the values were transformed to show a normal distribution by power transformation. MapQTL 5.0 (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Van Ooijen 2004</a>) was used to map growth-related QTL. The interval mapping and multi-QTL mapping (MQM) methods were used to map QTL. For each trait, the interval mapping method was first used to reveal general information on candidate QTL. Then, the MQM method was used for every linkage group, under the condition that the loci with the largest LOD scores in the other linkage groups were treated as cofactors. The LOD scores in each linkage group were fixed after the cofactors in all other linkage groups had been searched. Permutations were executed to determine the genome-wide significance threshold of the LOD scores (<em>P</em> &lt; 0.05), with the number of permutations set at 10,000. LOD peaks bigger than LOD threshold were identified as QTL. The location of each QTL was determined according to the location of its LOD peak and the surrounding region. Candidate genes for growth-related traits were identified by mapping the corresponding markers in QTL regions to the reference genome, followed by retrieval of the corresponding gene ID from the gene annotation file.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-13">Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/FileS1.pdf">File S1</a> contains histograms for the five growth-related traits. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/FileS2.pdf">File S2</a> contains boxplots of comparisons for the five growth-related traits between female and male progeny. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/FileS3.pdf">File S3</a> contains charts of the 10 linkage groups on the sex-average map. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/FileS4.pdf">File S4</a> contains charts of the distribution of distorted markers on the 10 linkage groups on the sex-average map. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS1.xlsx">Table S1</a> contains values of five growth-related traits and gender for every progeny of the mapping family. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS2.xlsx">Table S2</a> contains an analysis of correlations between the five growth-related traits in the mapping family. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS3.xlsx">Table S3</a> contains the details of the sequencing and genotyping results of the six genotyping-by-sequencing libraries of the mapping family. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS4.xlsx">Table S4</a> contains a summary of SNPs identified in the mapping family. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS5.xlsx">Table S5</a> contains the genotypes of the 1695 markers mapped. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS6.xlsx">Table S6</a> contains the sequences of the mapped markers. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS7.xlsx">Table S7</a> contains the markers and their positions on each of the 10 linkage groups on the sex-average map. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS8.xlsx">Table S8</a> contains the comparison of genetic distances and expected recombination counts before and after the imputation for the sex-specific maps, by linkage group. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS9.xlsx">Table S9</a> contains the lengths and numbers of the markers for the sex-specific and the sex-average maps, by linkage group. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS10.xlsx">Table S10</a> contains the proportion of distorted markers for each linkage group on the sex-average map. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS11.xlsx">Table S11</a> contains the IDs and symbols for the annotated genes in the QTL regions.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Phenotyping of growth-related traits</h3><p id="p-14">Shell height ranged from 3.7 cm to 9.9 cm, with an average of 7.6 cm (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS1.xlsx">Table S1</a>). Shell length ranged from 2.6 cm to 5.8 cm, with an average of 4.5 cm. Shell width ranged from 1.8 cm to 3.2 cm, with an average of 2.4 cm. Mass weight ranged from 1 g to 7.6 g, with an average of 4.6 g. Soft tissue weight ranged from 0.2 g to 1.9 g, with an average of 1 g. Pearson correlation coefficients between growth-related traits ranged from 0.397 to 0.852 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS2.xlsx">Table S2</a>). Mass weight and soft tissue weight showed the highest correlation (0.852), and shell height and shell width showed the lowest correlation (0.397). Not all growth-related traits showed significant correlations; hence, they were not integrated into a single composite trait, but were analyzed separately for QTL mapping. Values for shell length, shell width, mass weight, and soft tissue weight followed a normal distribution, but values for shell height did not (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/FileS1.pdf">File S1</a>). The shell height values were transformed for normality by taking the exponential: <em>y</em> = exp(<em>x</em>^2/1000), where <em>x</em> represents trait values before transformation and <em>y</em> represents trait values after transformation. Gender affected growth-related traits in this mapping population (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/FileS2.pdf">File S2</a>). Female progeny showed significantly larger values for five growth-related traits than male progeny. Male progeny showed larger variance for five growth-related traits than female progeny; this indicated that gender should be treated as a covariant when mapping QTL.</p></div><div id="sec-10" class="subsection"><h3>Genotyping-by-sequencing</h3><p id="p-15">Digestions of the Pacific oyster reference genome with different enzyme combinations were simulated with RestrictionDigest, and <em>Eco</em>RI–<em>Hin</em>fI was selected to digest the genomic DNA for library preparation. Simulated <em>Eco</em>RI–<em>Hin</em>fI digestion revealed that the digested fragments covered 6.84 Mb of the genome, 48.9% of which were located in intergenic regions, and 51.1% in genic regions, with 43.3% in introns and 7.8% in exons. This simulated distribution was concordant with the genomic region proportions found in the reference genome (<a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Zhang <em>et al.</em> 2012</a>), indicating that digestion with <em>Eco</em>RI–<em>Hin</em>fI would generate an even distribution of digested fragments across the genome. To identify genomic variation in the mapping population, <em>Eco</em>RI–<em>Hin</em>fI RRLs were constructed. To ensure sufficient read depth at potential variation loci, deep sequencing of both parents and all progeny was performed. Details of the sequencing and genotyping results are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS3.xlsx">Table S3</a>. In summary, 100SE sequencing of six GBS libraries generated 664.3 M of raw reads with a GC content of 43%. After filtering the low-quality reads, 473.9 M (71.3%) clean reads remained. Cleaned reads were mapped against the Pacific oyster reference genome with BWA. The total mapped base lengths covered by the ∼90-bp short reads from the parental <em>Eco</em>RI–<em>Hin</em>fI libraries was approximately 6.58 Mb, representing 1.2% of the published reference genome sequence. Of the 6.58 Mb mapped, 47.5% were located in intergenic regions, and 52.5% were located in genic regions, with 38.2% located in introns and 14.3% located in exons. The GC content of the mapped regions was 36.2% for intergenic regions, 39% for genic regions, 35.2% for introns, and 49% for exons.</p><p id="p-16">In total, the Stacks software generated 65,031 RAD loci in the mapping family, of which 15,783 (24.3%) were polymorphic. These polymorphic RAD loci contained 40,445 SNPs, with an average of 2.6 SNPs per locus. Relative to the reference genome, we identified a total of 40,445 single nucleotide substitutions, among which transitions (A ↔ G or C ↔ T) accounted for 48.1%, and transversions (A ↔ C, A ↔ T, C ↔ G, or G ↔ T) accounted for 51.9% (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS4.xlsx">Table S4</a>). Transitions between C ↔ T and A ↔ G appeared to be most prevalent, with each representing approximately 24.3% and 23.8% of the total polymorphisms. In contrast, C ↔ G transversions were the least common type of change, representing only 5.7% of the total polymorphisms. We categorized the SNPs according to their location on the reference genome. More than half of the GBS-based SNPs were distributed in intergenic regions (50.2%), whereas 35.2% of SNPs were distributed in introns, and 14.6% of SNPs were distributed in exons.</p><p id="p-17">Segregation analysis of the polymorphic RAD loci showed that 8743 segregated in the progeny, and these were used in the subsequent linkage analysis. Of these genotyped RAD loci, 4041 were present in the lm × ll configuration, 3635 were in the nn × np configuration, 631 were in the ef × eg configuration (three alleles, both parents are heterozygous), 281 were in the ab × cd configuration (four alleles, both parents are heterozygous), and 155 were in the hk × hk configuration (two alleles, both parents are heterozygous). High missing values were observed among the genotyped RAD loci: the missing ratio ranged from 0.9% to 99.1%, with an average of 61.6%.</p></div><div id="sec-11" class="subsection"><h3>Genetic linkage map</h3><p id="p-18">Following cosegregation analysis, 1694 RAD loci were mapped on the linkage map (mapped markers) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/FileS3.pdf">File S3</a>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS5.xlsx">Table S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS6.xlsx">Table S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS7.xlsx">Table S7</a>); the other 7049 RAD loci could not be mapped. A total of 3367 SNPs was located on the mapped markers, with an average of 1.9 SNPs per marker. Of the mapped markers, 930 were in the lm × ll configuration, 553 were in the nn × np configuration, 150 were in the ef × eg configuration, and 61 were in the ab × cd configuration.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1417/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The sex-average and the sex-specific linkage maps, constructed using a hybridized family of Crassostrea gigas and C.&#xA0;angulata. The sex-average linkage groups are named A1&#x2013;A10. The female linkage groups are named F1&#x2013;F10. The male linkage groups are named M1&#x2013;M10. The two linkage groups on the female map corresponding to A3 on the sex-average map are denoted F3.1 and F3.2, and represented as F3. The two linkage groups on the male map corresponding to A3 on the sex-average map are denoted M3.1 and M3.2, and represented as M3. M6.1 and M6.2 represent M6. F8.1 and F8.2 represent F8." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1665475556" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The sex-average and the sex-specific linkage maps, constructed using a hybridized family of &lt;em&gt;Crassostrea gigas&lt;/em&gt; and &lt;em&gt;C.&#xA0;angulata&lt;/em&gt;. The sex-average linkage groups are named A1&#x2013;A10. The female linkage groups are named F1&#x2013;F10. The male linkage groups are named M1&#x2013;M10. The two linkage groups on the female map corresponding to A3 on the sex-average map are denoted F3.1 and F3.2, and represented as F3. The two linkage groups on the male map corresponding to A3 on the sex-average map are denoted M3.1 and M3.2, and represented as M3. M6.1 and M6.2 represent M6. F8.1 and F8.2 represent F8.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1417/F1.medium.gif" width="440" height="252"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/5/1417/F1.medium.gif" width="440" height="252"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1417/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1417/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468230" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">The sex-average and the sex-specific linkage maps, constructed using a hybridized family of <em>Crassostrea gigas</em> and <em>C. angulata</em>. The sex-average linkage groups are named A1–A10. The female linkage groups are named F1–F10. The male linkage groups are named M1–M10. The two linkage groups on the female map corresponding to A3 on the sex-average map are denoted F3.1 and F3.2, and represented as F3. The two linkage groups on the male map corresponding to A3 on the sex-average map are denoted M3.1 and M3.2, and represented as M3. M6.1 and M6.2 represent M6. F8.1 and F8.2 represent F8.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468200/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468200/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Length, number of markers, and average distance between markers for sex-average and sex-specific maps, as well as the female:male ratio, by linkage group, constructed using a hybridized family of <em>Crassostrea gigas</em> and <em>C. angulata</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-20">Initial sex-specific maps were constructed using the original markers, which contained missing data. The average number of expected recombination events per linkage group was 6.7 for the female linkage map, and 3.4 for the male linkage map (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS8.xlsx">Table S8</a>), indicating double recombination events due to missing values and erroneous marker genotypes. An imputation was performed using Maskov, to impute missing values and remove suspected erroneous genotypes. Following the imputation, the average number of expected recombination events per linkage group was 2.6 for the female linkage map, and 1.7 for the male linkage map, a reduction in recombination of 61.2% and 50%, respectively. For the genetic distance of each linkage group, a mean of 65% reduction for the female map, and a mean of 52.7% reduction for the male map was observed after the imputation, respectively. The imputed markers in the lm × ll and nn × np configurations, together with the original mapped markers in the ab × cd and ef × ef, configurations, were used to construct the sex-average map.</p><p id="p-21">For the sex-average map, 1694 markers fell into 10 linkage groups (the linkage groups were named A1–A10) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS9.xlsx">Table S9</a>). For the female linkage map, 1102 markers fell into 12 linkage groups (the linkage groups were named F1–F10) (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS9.xlsx">Table S9</a>). For the male linkage map, 689 markers fell into 12 linkage groups (the linkage groups were named M1–M10) (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS9.xlsx">Table S9</a>). Analysis of common markers between the sex-specific and sex average maps revealed the correspondence of their linkage groups (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS9.xlsx">Table S9</a>). Each of LG A1, A2, A4, A5, A7, A9, and A10 on the sex-average map corresponded to a single linkage group on the sex-specific map. LG F3.1, F3.2, and LG M3.1, M3.2 corresponded to LG A3 on the sex-average map. LG M6.1, M6.2 corresponded to LG A6 on the sex-average map. LG F8.1, F8.2 corresponded to LG A8 on the sex-average map. For the sex-specific map, linkage groups corresponding to the same sex-average group were considered the same linkage group, and the sum of the length of the linkage groups represented the total length of the corresponding linkage group. For example, for the female map, the length of LG F3 was the sum of LG F3.1 and LG F3.2 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The sex-average map spanned a total genetic distance of 1084.3 cM, with an average spacing of 0.8 cM and a Coa of 98.7%; the length of each LG ranged from 94.7 cM (LG A9) to 134.3 cM (LG A1), with an average genetic length of 108.4 cM. The female map spanned a total genetic distance of 1034.6 cM, with an average spacing of 1.3 cM and a Coa of 97.5%; the length of each LG ranged from 70.6 cM (LG F9) to 121.4 cM (LG F1), with an average genetic length of 103.5 cM. The male map spanned a total genetic distance of 744.8 cM, with an average spacing of 1.7 cM and a Coa of 95.2%; the length of each LG ranged from 45.6 cM (LG M4) to 101.6 cM (LG M1), with an average genetic length of 74.5 cM.</p><p id="p-22">We observed an overall female-to-male recombination ratio of 1.39:1 (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The largest recombination ratio between sexes was found for linkage group 4, with a female:male ratio of 2.40:1, and the smallest recombination ratio between sexes was found for linkage group 9, with a female:male ratio of 1.07:1. Localized differences in recombination rates between the sexes were observed. High recombination rates were found near the centromeric region on LG F1, F2, F4, F5, F6, F7, F8.1, F8.2, and F10 on the female map, and high recombination rates were found near the telomeric regions on LG M1, M2, M4, M5, M7, M8, M9, and M10 for the male map (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-23">Of the 1694 markers on the sex-average map, 836 (49.4%) showed significant (<em>P</em> &lt; 0.05) segregation distortion (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS10.xlsx">Table S10</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/FileS4.pdf">File S4</a>). For the sex-average linkage map, LG A8 and LG A5 presented the most severe segregation distortion, with 77.2% and 73.5% of markers distorted, respectively. LG A10 presented the lowest segregation distortion, with 12.4% of markers distorted. Distorted markers tended to be clustered, rather than randomly distributed across a linkage group (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/FileS4.pdf">File S4</a>). For the distorted markers in the lm × ll type (423), 183 showed homozygote deficiency while 240 showed heterozygote deficiency. For the distorted markers in the nn × np type (238), 120 showed homozygote deficiency, while 118 showed heterozygote deficiency.</p></div><div id="sec-12" class="subsection"><h3>QTL mapping analysis of growth-related traits</h3><p id="p-24">A total of 27 QTL was identified, including seven for shell height, three for shell length, 10 for shell width, one for mass weight, and six for soft tissue weight (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). These 27 QTL were distributed across eight linkage groups on the sex-average map, except for LG A3 and LG A6. More than half (16) of the QTL were clustered in their respective LGs. A major cluster containing five QTL (qGRT-8, qGRT-9, qGRT-18, qGRT-19, and qGRT-20) was detected in the narrow region between 67.3 and 76 cM on LG A8. On LG A2, another cluster containing four QTL (qGRT-13, qGRT-21, qGRT-22, and qGRT-23) was detected within a small region (59.3–66.5 cM). Three other clusters were situated between 44.4 and 47.4 cM of LG A1, which contained two QTL (qGRT-11 and qGRT-12), between 4.7 and 11.6 cM of LG A4, which contained three QTL (qGRT-1, qGRT-2, and qGRT-14), and between 44.3 and 53.9 cM of LG A7, which contained two QTL (qGRT-4 and qGRT-5). LG A8 contained the most QTL, with six, and LG A10 contained the fewest QTL, with one. Of the five growth-related traits, four had QTL in LG A8, which indicated that LG A8 shows enrichment for genes regulating growth. In particular, qGRT-7 for shell height and qGRT-6 and qGRT-25 for soft tissue weight were at the same location on LG A8. The phenotypic variance explained by these QTL ranged from 4.2% to 7.7%, with an average of 5.4%. The low phenotypic variance explained by these QTL indicated that no major loci (explaining &gt;20% of the total variation) were detected. For shell height, the seven QTL explained 35.3% of the total phenotypic variance. For shell length, the three QTL explained 20.1% of the total phenotypic variance. For shell width, the 10 QTL explained 50.8% of the total phenotypic variance. For mass weight, the QTL explained 7.7% of the total phenotypic variance. For soft tissue weight, the six QTL explained 34.1% of the phenotypic variance. The LOD scores of the QTL varied from 3.4 to 5.1, with an average of 4.1. A number of genes related to growth-related traits were identified from the higher-density map based on the Pacific oyster genome assembly. As shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1/TableS11.xlsx">Table S11</a>, 38 annotated genes were identified in the QTL regions. Among the annotated genes, two were found to be growth-related (<em>agl</em> and <em>fbp1</em>), which function as key factors in carbohydrate metabolism, and others were found to participate in the assembly and regulation of the actin cytoskeleton (<em>avil</em>, <em>fmn2</em>, and <em>specc1l</em>), signal transduction pathways (<em>prkg1</em>, <em>dusp6</em>, and <em>grk1</em>), and the regulation of cell differentiation and development (<em>tbata</em> and <em>megf8</em>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468008/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468008/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Characteristics of quantitative trait loci for growth-related traits</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><div id="sec-14" class="subsection"><h3>Genotyping-by-sequencing in oysters</h3><p id="p-25">RRL sequencing methods coupled with next-generation sequencing technologies are widely employed to genotype SNPs in vertebrates and plants (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andrews <em>et al.</em> 2016</a>), but only a limited number of studies utilize them to call SNPs in mollusks (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Liu <em>et al.</em> 2013</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Richards <em>et al.</em> 2013</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chu <em>et al.</em> 2014</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Li and He 2014</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Jiao <em>et al.</em> 2014</a>). In this study, GBS was first used to screen for high-quality, abundant SNPs for construction of a genetic map for oysters. <em>Eco</em>RI and <em>Hin</em>fI were selected for the construction of RRLs; restriction digestion produced DNA fragments of 270–470 bp, which were sequenced for SNP screening. The mapping locations of the GBS reads and the distribution of the SNPs located on them were generally in accordance with the results of <em>in silico</em> simulations, indicating high consistency between simulated and actual digestions. This result indicated that <em>Eco</em>RI and <em>Hin</em>fI could generate evenly distributed markers, and constituted a good restriction enzyme combination for the GBS analysis of oyster. An enrichment of GBS reads in exons was observed, which may result from using two methylation-sensitive enzymes to construct the libraries (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Pootakham <em>et al.</em> 2015</a>). Deep sequencing of GBS libraries was performed to reduce the negative effect of nonuniform distribution of sequence reads, which mainly originated from inaccurate DNA quantification in the GBS analysis. A total of 15,783 polymorphic RAD loci was identified in the mapping population, for which 40,445 SNPs were detected. Deep sequencing of each locus, at 91.5-fold for the male, 63.4-fold for the female, and 13.1-fold to 143.2-fold for the progeny, guaranteed the accuracy of genotyping.</p><p id="p-26">Despite the deep sequencing of the RRL libraries, a high proportion of missing values (low consistency of RAD loci among individuals) was observed among the genotyped markers, which was unexpected. Several factors may explain the high missing data ratio. The first factor may be the unstable fidelity of the restriction enzymes used in the library construction. The activity of the restriction enzymes can be affected by sequence context, enzyme concentration and buffer composition. Unsuitable conditions may lead to either ineffective cleavage at the cognate restriction site or relaxed specificity allowing cleavage of degenerate ‘star’ sites (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kamps-Hughes <em>et al.</em> 2013</a>). The second reason may be differences in methylation patterns between the two parents and the progeny, which would lead to inconsistencies in enzyme cut-site locations among individuals, resulting in the generation of inconsistent tags (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Xia <em>et al.</em> 2014</a>). The third reason may be the genomic differences between <em>C. gigas</em> and <em>C. angulata</em>. The fourth reason may be the nonuniform distribution of sequence reads among different individuals resulted from inaccurate quantification of DNA for restriction digestion and differentiated PCR amplification efficiency. Although high missing value ratio was observed, nearly 1600 RAD loci showed missing value ratio smaller than 30%, which could be used to construct linkage map. The application of GBS in oysters could significantly improve the throughput of SNP genotyping and reduce the cost of genetic map construction and population genetic analysis.</p></div><div id="sec-15" class="subsection"><h3>Linkage mapping</h3><p id="p-27">Of the 8743 genotyped RAD loci, 1694 were successfully mapped to the linkage map while 7049 failed to be mapped to the map. The low ratio of RAD loci that can be assigned into the linkage map is also observed in other linkage mapping studies (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Li and He 2014</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ren <em>et al.</em> 2016</a>). In this study, the main reason for the unsuccessful mapping of the 7049 RAD loci may be their high missing ratio (with an average of 61.6%). Of the mapped 1694 markers, an average missing ratio of 26.8% was observed, which was comparable to other GBS studies (<a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Ward <em>et al.</em> 2013</a>). It was reported that missing values and typing errors in the data would reduce the proportion of correctly ordered map (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hackett and Broadfoot 2003</a>). To reduce the negative effects of missing values and typing errors on map order, we used Maskov to impute missing values and correct typing errors for genetic markers (<a id="xref-ref-76-4" class="xref-bibr" href="#ref-76">Ward <em>et al.</em> 2013</a>). The successful imputation reduced linkage group length, and decreased the recombinant count for each linkage group on the sex-specific maps, indicating that the quality of the sex-specific maps was improved. The sex-average map built with the imputed markers thus represents reliable marker order and map distance.</p><p id="p-28">Compared to the previously published linkage maps of the Pacific oyster constructed with first-generation DNA markers (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Li and Guo 2004</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Hubert and Hedgecock 2004</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Hubert <em>et al.</em> 2009</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Sauvage <em>et al.</em> 2010</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Plough and Hedgecock 2011</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2012</a>; <a id="xref-ref-81-3" class="xref-bibr" href="#ref-81">Zhong <em>et al.</em> 2014</a>), the genetic map in this study contains more markers and has higher density. More recently, <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hedgecock <em>et al.</em> (2015)</a> constructed second-generation linkage maps with high density for the Pacific oyster, mainly based on 1085 coding SNPs generated by EST sequencing, as well as 66 microsatellite DNA markers, with an average marker spacing of 1.0 cM. This map represented a substantial improvement on previously reported first-generation maps. In the present study, we constructed a second-generation linkage map based on SNPs generated through GBS, which is more efficient in terms of cost and time than microarray based and PCR-base genotyping. The sex-average map consisted of 1695 tags, with an average interval of 0.8 cM.</p><p id="p-29">The number of linkage groups (10) on the sex-average map corresponded well to the haploid chromosome number of oysters (10). Although both the female and male maps contained 12 linkage groups, they corresponded well with the 10 linkage groups on the sex-average map. The discrepancy between the number of linkage groups and the haploid chromosome number observed for the sex-specific maps was also observed in previously published oyster linkage maps with relatively low marker number (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Hubert and Hedgecock 2004</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Li and Guo 2004</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2012</a>). The split between the linkage groups is unexpected in terms of the high marker number and the small average interval in this study. It appears that, for the sex-specific maps, some markers that could have integrated the two linkage groups that aligned with a single linkage group on the sex-average map were missing. Potential reasons for the missing markers are as follows: 1) the gap may correspond to a genomic region that is homozygous in the parents; 2) the gap may correspond to a genomic region with high recombination rates but low marker density (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Li and Guo 2004</a>).</p><p id="p-30">Both overall and localized differences in recombination rates between the sexes were observed in the present study. In both vertebrates and invertebrates, female maps are usually longer than male maps (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hansson <em>et al.</em> 2005</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lien <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jones <em>et al.</em> 2013</a>). The same phenomenon was also observed in <em>C. gigas</em>. <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Hubert and Hedgecock (2004)</a> observed a female-to-male recombination ratio of 1.25:1, and <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Li and Guo (2004)</a> observed a female-to-male recombination ratio of 1.36:1. In contrast, <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Hedgecock <em>et al.</em> (2015)</a> observed no significant differences between sexes in the recombination rates, which suggested that sex-specific recombination pattern varied among different mapping families. In this study, the overall female-to-male recombination ratio was 1.39:1, which was in accordance with that in previous studies. In addition to the overall recombination rate difference between the sexes, localized differences in recombination rates were also observed: high recombination rates were found near the centromeric region on the female map, and high recombination rates were found near the telomeric region on the male map. In other aquatic animals, such as rainbow trout (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sakamoto <em>et al.</em> 2000</a>), zebrafish (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Singer <em>et al.</em> 2002</a>), and <em>Pinctada maxima</em> (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Jones <em>et al.</em> 2013</a>), a similar pattern was also observed. The mechanism of overall and localized recombination rate differences remains unclear. However, several possible causes have been proposed: first, the germ cells developed in different environments; second, meiosis was initiated at different time points; third, pairing and synapses of the homologs at meiosis differed between oocytes and spermatocytes. Further studies are required to reveal the mechanism governing the differences in recombination rates between the sexes.</p></div><div id="sec-16" class="subsection"><h3>Segregation distortion</h3><p id="p-31">Segregation distortion is a common phenomenon in many species, and the proportion of distorted markers varies across different species (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Graner <em>et al.</em> 1991</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Tan <em>et al.</em> 2001</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li <em>et al.</em> 2005</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lallias <em>et al.</em> 2007</a>). In this study, we observed 49.4% distorted markers in the outbred cross, which is comparable with previous studies. For the Pacific oyster, segregation distortion is observed in both inbred and outbred crosses. The rate of distorted markers in inbred crosses ranges from 17% to 66% (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Launey and Hedgecock 2001</a>; <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Sauvage <em>et al.</em> 2010</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Plough and Hedgecock 2011</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Plough 2012</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Hedgecock <em>et al.</em> 2015</a>), and the rate in outbred crosses ranges from 18.8% to 68% (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2012</a>; <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Hedgecock <em>et al.</em> 2015</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Plough <em>et al.</em> 2016</a>). Segregation distortion has been reported to be caused by zygotic viability selection, duplicated genes, transposable elements, and unusual meiotic segregation distortion (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Knox and Ellis 2002</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gut and Lathrop 2004</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hubert <em>et al.</em> 2010</a>). <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Launey and Hedgecock (2001)</a> demonstrated experimentally that a high genetic load of deleterious recessive genes, which resulted in strong zygotic selection during the larvae stage, was a cause of segregation distortion in the Pacific oyster. Another factor leading to the high proportion of distorted markers observed in this study may be the hybridization between <em>C. gigas</em> and <em>C. angulata</em>. Hybrid incompatibility is a key biological factor causing uneven transmission of alternate alleles, which is frequently caused by disrupted genetic interactions among loci of parental lineages, resulting in the nonrandom elimination of particular allelic combinations (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Orr and Turelli 2001</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Zhou <em>et al.</em> 2015</a>). The ratio of markers presenting heterozygotes inviability (54.2%) is slightly bigger than that of markers presenting homozygotes inviability (45.8%) in this mapping population, which is in accordance with previous finding that heterozygotes are inviable in outbred families (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Plough <em>et al.</em> 2016</a>).</p><p id="p-32"><a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Hedgecock <em>et al.</em> (2015)</a> reported that the effect of distorted markers on map construction is not universal, and is likely to vary within and among linkage maps. Considering that distorted markers can be potentially helpful in the detection of QTL (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Xu 2008</a>), and that discarding them could potentially remove massive amounts of information and decrease genome coverage (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Luo <em>et al.</em> 2005</a>), we retained the distorted markers for map construction despite their potential negative effects on the map. The uneven distribution of the distorted markers may suggest that marker distortion was not caused by technical limitations or other typing errors. In this study, the distorted markers were used in constructing the linkage map, as they represented nearly half of all mapped markers and revealed genomic regions subject to viability selection.</p></div><div id="sec-17" class="subsection"><h3>QTL for growth-related traits and associated genes</h3><p id="p-33">The high density of the genetic map allowed us to perform QTL fine mapping of economically important growth-related traits. In the present study, 27 QTL associated with growth-related traits of oysters were found to be distributed in eight LGs. Interestingly, more than half of the QTL were concentrated within narrow regions (clusters) in the LGs, which indicated clustering of genes regulating growth in oysters. In previous studies, a limited number of QTL for growth-related traits were detected based on the first-generation genetic maps with low density. For example, only three QTL were identified for two principal components, each of which explained 0.6%, 7.5%, and 13.8%, respectively, of the phenotypic variance based on a genetic map constructed using SSR and AFLP markers, and no linked candidate genes were identified (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Guo <em>et al.</em> 2012</a>). In the present study, the phenotypic variances explained by these QTL were relatively low (with an average of 5.4%), which indicated that no major loci were detected and the growth-related traits were regulated by many genes of small effect. The low phenotypic variances explained by detected QTL were also observed in the disease-resistance QTL mapping (5.1–8.38%) of the Japanese flounder (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Shao <em>et al.</em> 2015</a>). The identified QTL explain relatively low phenotypic variations, and more fine-mapping QTL based on multiple families are required before they can be utilized in the molecular breeding practice.</p><p id="p-34">In the present study, within the QTL regions, 38 genes were identified, and two were previously reported to be associated with growth. The glycogen debrancher enzyme encoded by <em>agl</em> is important for degradation of glycogen, the major reserve for glucose storage; it is known to play a central role in providing energy for maintenance and gametogenic development (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Li <em>et al.</em> 2006</a>). <em>Agl</em> was also identified as being associated with glycogen content in the Pacific oyster (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">She <em>et al.</em> 2015</a>). The fructose-1,6-bisphosphatase encoded by <em>fba</em> is a key enzyme in glucose metabolism, and is essential for cell growth (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brooks and Storey 1988</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Mazurek <em>et al.</em> 1998</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Singh <em>et al.</em> 2004</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Campanella <em>et al.</em> 2005</a>). It catalyzes the reversible cleavage of fructose-1,6-bisphosphate to glyceraldehyde-3-phosphate and dihydroxyacetone phosphate (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lu <em>et al.</em> 2004</a>). One SNP in <em>fba</em> has been identified to present a significant correlation with growth traits in <em>Meretrix meretrix</em> (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Wang <em>et al.</em> 2012a</a>). In addition to <em>agl</em> and <em>fba</em>, several genes participating in the assembly and regulation of the actin cytoskeleton (<em>avil</em>, <em>fmn2</em>, and <em>specc1l</em>) were also identified within the QTL regions. A genome-wide association study of calf birth weight in Holstein cattle using SNPs revealed the enrichment of the “Regulation of actin cytoskeleton” pathway, as defined by the Kyoto Encyclopedia of Genes and Genomes, indicating an association between growth-related traits and actin cytoskeleton regulation in bovines (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cole <em>et al.</em> 2014</a>). The QTL mapping results reported here also revealed a correlation between growth-related traits and actin cytoskeleton regulation in oysters. Other genes identified in QTL regions included <em>prkg1</em>, <em>dusp6</em>, and <em>grk1</em>, which participate in signal transduction, and <em>tbata</em> and <em>megf8</em>, which participate in regulation of cell differentiation and development. These genes are valuable candidate growth-related genes, and warrant further investigation to confirm their involvement.</p></div><div id="sec-18" class="subsection"><h3>Conclusion</h3><p id="p-35">In this study, abundant SNPs were successfully obtained via GBS. A high-density linkage map was constructed for a hybrid mapping family of <em>C. gigas</em> × <em>C. angulata</em>. A total of 27 QTL was detected for five growth-related traits and 38 associated genes were identified, which will provide valuable genetic resources and the basis for MAS for both <em>C. gigas</em> and <em>C. angulata</em>.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-36">We thank Huaiping Zheng of Shantou University for providing the male parent of the mapping family. We thank Kai Song and Xuedi Du of the Institute of Oceanology, Chinese Academy of Sciences, Qingdao, for their advice on the project. The authors are grateful for the support of the National Basic Research Program of China (973 Program, No. 2010CB126402), the National Natural Science Foundation of China (31530079), the Earmarked Fund for Modern Agro-industry Technology Research Systems (CARS-48), and the Taishan Scholars Climbing Program of Shandong.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-37">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.026971/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-38"><em>Communicating editor: W. S. Davidson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-01-13"><span class="received-label">Received </span>January 13, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-03-12"><span class="accepted-label">Accepted </span>March 12, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Wang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.1" data-doi="10.1016/j.jembe.2006.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Harnessing the power of RADseq for ecological and evolutionary genomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">81</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D343%26rft.spage%253D253%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jembe.2006.12.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jembe.2006.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000245690500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1417.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anthony</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larry</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nancy</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Pearson product-moment correlation coefficient</span>, pp. <span class="cit-fpage">751</span>–<span class="cit-lpage">756</span> in <span class="cit-source">Encyclopedia of Measurement and Statistics</span>, edited by N. <span class="cit-ed"><span class="cit-name-surname">Salkind</span></span>. <span class="cit-publ-name">SAGE Publications, Inc.</span>, <span class="cit-publ-loc">Thousand Oaks, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.3" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.4" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">S. P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">K. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Reevaluation of the glycolytic complex in muscle—a multitechnique approach using trout white muscle.</span> <abbr class="cit-jnl-abbrev">Arch. Biochem. Biophys.</abbr> <span class="cit-vol">267</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">13</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.5" data-doi="10.1016/0003-9861(88)90002-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buestel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ropert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goulletquer</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">History, status, and future of oyster culture in France.</span> <abbr class="cit-jnl-abbrev">J. Shellfish Res.</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">813</span>–<span class="cit-lpage">820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchives%2Bof%2BBiochemistry%2Band%2BBiophysics%26rft.stitle%253DArchives%2Bof%2BBiochemistry%2Band%2BBiophysics%26rft.aulast%253DBrooks%26rft.auinit1%253DS.%2BP.%26rft.volume%253D267%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D22%26rft.atitle%253DReevaluation%2Bof%2Bthe%2B%2526quot%253Bglycolytic%2Bcomplex%2526quot%253B%2Bin%2Bmuscle%253A%2Ba%2Bmultitechnique%2Bapproach%2Busing%2Btrout%2Bwhite%2Bmuscle.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0003-9861%252888%252990002-1%26rft_id%253Dinfo%253Apmid%252F2973767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0003-9861(88)90002-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2973767&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.6" data-doi="10.2983/035.028.0410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campanella</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Low</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Assembly and regulation of a glycolytic enzyme complex on the human erythrocyte membrane.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">2402</span>–<span class="cit-lpage">2407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D28%26rft.spage%253D813%26rft_id%253Dinfo%253Adoi%252F10.2983%252F035.028.0410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2983/035.028.0410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.7" data-doi="10.1073/pnas.0409741102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0409741102%26rft_id%253Dinfo%253Apmid%252F15701694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAyLzcvMjQwMiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xNDE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.8" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakravarti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lasher</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reefer</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">A maximum-likelihood method for estimating genome length using genetic-linkage data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">128</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">175</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaluziak</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trussell</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vollmer</span>  <span class="cit-name-given-names">S. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Phylogenomic analyses reveal latitudinal population structure and polymorphisms in heat stress genes in the North Atlantic snail <em>Nucella lapillus</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1863</span>–<span class="cit-lpage">1873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChakravarti%26rft.auinit1%253DA.%26rft.volume%253D128%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D182%26rft.atitle%253DA%2Bmaximum%2Blikelihood%2Bmethod%2Bfor%2Bestimating%2Bgenome%2Blength%2Busing%2Bgenetic%2Blinkage%2Bdata.%26rft_id%253Dinfo%253Apmid%252F2060775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI4LzEvMTc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzE0MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.10" data-doi="10.1111/mec.12681"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cole</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waurich</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wensch-Dorendorf</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bickhart</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swalve</span>  <span class="cit-name-given-names">H. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A genome-wide association study of calf birth weight in Holstein cattle using single nucleotide polymorphisms and phenotypes predicted from auxiliary traits.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">97</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">3156</span>–<span class="cit-lpage">3172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D23%26rft.spage%253D1863%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.11" data-doi="10.3168/jds.2013-7409"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cotter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malham</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Keeffe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latchford</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">mortality of the Pacific oyster, <em>Crassostrea gigas</em>, in the Irish Sea: the influence of growth, biochemistry and gametogenesis.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">303</span>(<span class="cit-issue">1–4</span>): <span class="cit-fpage">8</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D97%26rft.spage%253D3156%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2013-7409%26rft_id%253Dinfo%253Apmid%252F24612794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2013-7409&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24612794&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000334655600064&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.12" data-doi="10.1016/j.aquaculture.2010.02.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Degremont</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bedier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boudry</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mortality of hatchery-produced Pacific oyster spat (<em>Crassostrea gigas</em>). II. Response to selection for survival and its influence on growth and yield.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">299</span>(<span class="cit-issue">1–4</span>): <span class="cit-fpage">21</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D303%26rft.spage%253D8%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2010.02.030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2010.02.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000278152100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.13" data-doi="10.1016/j.aquaculture.2009.11.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D299%26rft.spage%253D21%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2009.11.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2009.11.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.14" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fishman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A genetic map in the <em>Mimulus guttatus</em> species complex reveals transmission ratio distortion due to heterospecific interactions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1701</span>–<span class="cit-lpage">1716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-web" id="cit-6.5.1417.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Food and Agriculture Organization (FAO)</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-comment">World aquaculture production of fish, crustaceans, molluscs, etc., by principal species in 2013. Available at: <a href="ftp://ftp.fao.org/fi/stat/summary/a-6.pdf">ftp://ftp.fao.org/fi/stat/summary/a-6.pdf</a>.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahoor</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schondelmaier</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siedler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Construction of an RFLP map of barley.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">83</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">250</span>–<span class="cit-lpage">256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D83%26rft.spage%253D250%26rft_id%253Dinfo%253Apmid%252F24202366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24202366&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GU45600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1417.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sambrook</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source">Molecular Cloning: A Laboratory Manual</span>, <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.18" data-doi="10.1007/s10126-011-9405-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic mapping and QTL analysis of growth-related traits in the Pacific oyster.</span> <abbr class="cit-jnl-abbrev">Mar. Biotechnol. (NY)</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">218</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMarine%2Bbiotechnology%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.stitle%253DMar%2BBiotechnol%2B%2528NY%2529%26rft.aulast%253DGuo%26rft.auinit1%253DX.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D218%26rft.epage%253D226%26rft.atitle%253DGenetic%2Bmapping%2Band%2BQTL%2Banalysis%2Bof%2Bgrowth-related%2Btraits%2Bin%2Bthe%2BPacific%2Boyster.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10126-011-9405-4%26rft_id%253Dinfo%253Apmid%252F21932055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10126-011-9405-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21932055&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.19" data-doi="10.1007/s10126-011-9405-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Use and exchange of genetic resources in molluscan aquaculture.</span> <abbr class="cit-jnl-abbrev">Rev. Aquacult.</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">3–4</span>): <span class="cit-fpage">251</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMarine%2Bbiotechnology%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.stitle%253DMar%2BBiotechnol%2B%2528NY%2529%26rft.aulast%253DGuo%26rft.auinit1%253DX.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D218%26rft.epage%253D226%26rft.atitle%253DGenetic%2Bmapping%2Band%2BQTL%2Banalysis%2Bof%2Bgrowth-related%2Btraits%2Bin%2Bthe%2BPacific%2Boyster.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10126-011-9405-4%26rft_id%253Dinfo%253Apmid%252F21932055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10126-011-9405-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21932055&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.20" data-doi="10.1111/j.1753-5131.2009.01014.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gut</span>  <span class="cit-name-given-names">I. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lathrop</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Duplicating SNPs.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">789</span>–<span class="cit-lpage">790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D1%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1753-5131.2009.01014.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1753-5131.2009.01014.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.21" data-doi="10.1038/ng0804-789"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broadfoot</span>  <span class="cit-name-given-names">L. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">90</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DGut%26rft.auinit1%253DI.%2BG.%26rft.volume%253D36%26rft.issue%253D8%26rft.spage%253D789%26rft.epage%253D790%26rft.atitle%253DDuplicating%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0804-789%26rft_id%253Dinfo%253Apmid%252F15284844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0804-789&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15284844&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222974000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.22" data-doi="10.1038/sj.hdy.6800173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akesson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slate</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pemberton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Linkage mapping reveals sex-dimorphic map distances in a passerine bird.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">272</span>(<span class="cit-issue">1578</span>): <span class="cit-fpage">2289</span>–<span class="cit-lpage">2298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DEffects%2Bof%2Bgenotyping%2Berrors%252C%2Bmissing%2Bvalues%2Band%2Bsegregation%2Bdistortion%2Bin%2Bmolecular%2Bmarker%2Bdata%2Bon%2Bthe%2Bconstruction%2Bof%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800173%26rft_id%253Dinfo%253Apmid%252F12522423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12522423&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181165800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.23" data-doi="10.1098/rspb.2005.3228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">G. M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voigt</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mapping heterosis QTL in the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">272</span>: <span class="cit-fpage">S267</span>–<span class="cit-lpage">S268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DHansson%26rft.auinit1%253DB.%26rft.volume%253D272%26rft.issue%253D1578%26rft.spage%253D2289%26rft.epage%253D2298%26rft.atitle%253DLinkage%2Bmapping%2Breveals%2Bsex-dimorphic%2Bmap%2Bdistances%2Bin%2Ba%2Bpasserine%2Bbird%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2005.3228%26rft_id%253Dinfo%253Apmid%252F16191642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMzoiMjcyLzE1NzgvMjI4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xNDE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gracey</span>  <span class="cit-name-given-names">A.Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Den Berg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samanta</span>  <span class="cit-name-given-names">M.P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Second-generation linkage maps for the pacific oyster Crassostrea gigas reveal errors in assembly of genome scaffolds.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">2007</span>–<span class="cit-lpage">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D272%26rft.spage%253DS267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.25" data-doi="10.1534/g3.115.019570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Linkage maps of microsatellite DNA markers for the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">351</span>–<span class="cit-lpage">362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.019570%26rft_id%253Dinfo%253Apmid%252F26248981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTAvMjAwNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xNDE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.26" data-doi="10.1534/genetics.104.027342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cognard</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Centromere mapping in triploid families of the Pacific oyster <em>Crassostrea gigas</em> (Thunberg).</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">288</span>(<span class="cit-issue">3–4</span>): <span class="cit-fpage">172</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHubert%26rft.auinit1%253DS.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D351%26rft.epage%253D362%26rft.atitle%253DLinkage%2BMaps%2Bof%2BMicrosatellite%2BDNA%2BMarkers%2Bfor%2Bthe%2BPacific%2BOyster%2BCrassostrea%2Bgigas%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.027342%26rft_id%253Dinfo%253Apmid%252F15454548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY4LzEvMzUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzE0MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.27" data-doi="10.1016/j.aquaculture.2008.12.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borza</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Development of a SNP resource and a genetic linkage map for Atlantic cod (<em>Gadus morhua</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D288%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2008.12.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2008.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000264622300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.28" data-doi="10.1186/1471-2164-11-191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huvet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ledu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phelipot</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heurtebise</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Is fertility of hybrids enough to conclude that the two oysters <em>Crassostrea gigas</em> and <em>Crassostrea angulata</em> are the same species?</span> <abbr class="cit-jnl-abbrev">Aquat. Living Resour.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">45</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHubert%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.spage%253D191%26rft.epage%253D191%26rft.atitle%253DDevelopment%2Bof%2Ba%2BSNP%2Bresource%2Band%2Ba%2Bgenetic%2Blinkage%2Bmap%2Bfor%2BAtlantic%2Bcod%2B%2528Gadus%2Bmorhua%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-191%26rft_id%253Dinfo%253Apmid%252F20307277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20307277&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.29" data-doi="10.1016/S0990-7440(01)01148-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-resolution linkage and quantitative trait locus mapping aided by genome survey sequencing: building up an integrative genomic framework for a bivalve mollusc.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">85</span>–<span class="cit-lpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft.volume%253D15%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0990-7440%252801%252901148-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0990-7440(01)01148-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.30" data-doi="10.1093/dnares/dst043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jerry</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khatkar</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raadsma</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenger</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A high-density SNP genetic linkage map for the silver-lipped pearl oyster, <em>Pinctada maxima</em>: a valuable resource for gene localisation and marker-assisted selection.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdst043%26rft_id%253Dinfo%253Apmid%252F24107803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjc6IjIxLzEvODUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTQxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.31" data-doi="10.1186/1471-2164-14-810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kamps-Hughes</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quimby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Massively parallel characterization of restriction endonucleases.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D14%26rft.spage%253D810%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-810%26rft_id%253Dinfo%253Apmid%252F24252414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-810&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24252414&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.32" data-doi="10.1093/nar/gkt257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knox</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">T. H. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Excess heterozygosity contributes to genetic map expansion in pea recombinant inbred populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">861</span>–<span class="cit-lpage">873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt257%26rft_id%253Dinfo%253Apmid%252F23605040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS8xMS9lMTE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzE0MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosambi</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">The estimation of map distances from recombination values.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKnox%26rft.auinit1%253DM.%2BR.%26rft.volume%253D162%26rft.issue%253D2%26rft.spage%253D861%26rft.epage%253D873%26rft.atitle%253DExcess%2BHeterozygosity%2BContributes%2Bto%2BGenetic%2BMap%2BExpansion%2Bin%2BPea%2BRecombinant%2BInbred%2BPopulations%26rft_id%253Dinfo%253Apmid%252F12399396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYyLzIvODYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzE0MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.34" data-doi="10.1111/j.1469-1809.1943.tb02321.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lallias</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapegue</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hecquet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boudry</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">AFLP-based genetic linkage maps of the blue mussel (<em>Mytilus edulis</em>).</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">340</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D12%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1943.tb02321.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1943.tb02321.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.35" data-doi="10.1111/j.1365-2052.2007.01611.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Launey</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">High genetic load in the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">255</span>–<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DLallias%26rft.auinit1%253DD.%26rft.volume%253D38%26rft.issue%253D4%26rft.spage%253D340%26rft.epage%253D349%26rft.atitle%253DAFLP-based%2Bgenetic%2Blinkage%2Bmaps%2Bof%2Bthe%2Bblue%2Bmussel%2B%2528Mytilus%2Bedulis%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2007.01611.x%26rft_id%253Dinfo%253Apmid%252F17559551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2007.01611.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17559551&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leitão</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaves</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henrique Guedes-Pinto</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boudry</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Interspecific hybridization in oysters: restriction enzyme digestion chromosome banding confirms Crassostrea angulata · Crassostrea gigas F1 hybrids.</span> <abbr class="cit-jnl-abbrev">J. Exp. Mar. Biol. Ecol.</abbr> <span class="cit-vol">343</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">253</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLauney%26rft.auinit1%253DS.%26rft.volume%253D159%26rft.issue%253D1%26rft.spage%253D255%26rft.epage%253D265%26rft.atitle%253DHigh%2BGenetic%2BLoad%2Bin%2Bthe%2BPacific%2BOyster%2BCrassostrea%2Bgigas%26rft_id%253Dinfo%253Apmid%252F11560902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzEvMjU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzE0MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.37" data-doi="10.1016/S0990-7440(02)01176-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Roux</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gay</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waechter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poubalanne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Comparative analysis of <em>Vibrio splendidus</em>-related strains isolated during <em>Crassostrea gigas</em> mortality events.</span> <abbr class="cit-jnl-abbrev">Aquat. Living Resour.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">251</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquat.%2BLiving%2BResour.%26rft.volume%253D15%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0990-7440%252802%252901176-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0990-7440(02)01176-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.38" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xNDE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.39" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xNDE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.40" data-doi="10.1007/s10126-003-0001-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">AFLP-based genetic linkage maps of the pacific oyster <em>Crassostrea gigas</em> Thunberg.</span> <abbr class="cit-jnl-abbrev">Mar. Biotechnol. (NY)</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">26</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMarine%2Bbiotechnology%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.stitle%253DMar%2BBiotechnol%2B%2528NY%2529%26rft.aulast%253DLi%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D26%26rft.epage%253D36%26rft.atitle%253DAFLP-based%2Bgenetic%2Blinkage%2Bmaps%2Bof%2Bthe%2Bpacific%2Boyster%2BCrassostrea%2Bgigas%2BThunberg.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10126-003-0001-0%26rft_id%253Dinfo%253Apmid%252F14564534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10126-003-0001-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14564534&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.41" data-doi="10.1016/j.aquaculture.2004.12.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiang</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Construction of AFLP-based genetic linkage map for Zhikong scallop, <em>Chlamys farreri</em> Jones et Preston and mapping of sex-linked markers.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">245</span>(<span class="cit-issue">1–4</span>): <span class="cit-fpage">63</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D245%26rft.spage%253D63%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2004.12.015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2004.12.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000227403600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.42" data-doi="10.1016/j.aquaculture.2006.08.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirasu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">S. X.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Reproductive cycle and biochemical composition of the Zhe oyster <em>Crassostrea plicatula</em> Gmelin in an eastern coastal bay of China.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">261</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">752</span>–<span class="cit-lpage">759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D261%26rft.spage%253D752%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2006.08.023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2006.08.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.43" data-doi="10.1007/s12562-011-0369-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Selection response and realized heritability for growth in three stocks of the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Fish. Sci.</abbr> <span class="cit-vol">77</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">643</span>–<span class="cit-lpage">648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFish.%2BSci.%26rft.volume%253D77%26rft.spage%253D643%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12562-011-0369-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12562-011-0369-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.44" data-doi="10.1371/journal.pone.0111707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic mapping and QTL analysis of growth-related traits in <em>Pinctada fucata</em> using restriction-site associated DNA sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e111707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De111707%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0111707%26rft_id%253Dinfo%253Apmid%252F25369421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0111707&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25369421&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.45" data-doi="10.1186/1471-2164-12-615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gidskehaug</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A dense SNP-based linkage map for Atlantic salmon (<em>Salmo salar</em>) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLien%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.spage%253D615%26rft.epage%253D615%26rft.atitle%253DA%2Bdense%2BSNP-based%2Blinkage%2Bmap%2Bfor%2BAtlantic%2Bsalmon%2B%2528Salmo%2Bsalar%2529%2Breveals%2Bextended%2Bchromosome%2Bhomeologies%2Band%2Bstriking%2Bdifferences%2Bin%2Bsex-specific%2Brecombination%2Bpatterns.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-615%26rft_id%253Dinfo%253Apmid%252F22182215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22182215&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Fine mapping of the pond snail left-right asymmetry (chirality) locus using RAD-Seq and Fibre-FISH.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">8</span>): e71067.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.47" data-doi="10.1074/jbc.M303871200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sautin</span>  <span class="cit-name-given-names">Y. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gluck</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The glycolytic enzyme aldolase mediates assembly, expression, and activity of vacuolar H+-ATPase.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">279</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">8732</span>–<span class="cit-lpage">8739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M303871200%26rft_id%253Dinfo%253Apmid%252F14672945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNzkvMTAvODczMiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xNDE3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.48" data-doi="10.1038/sj.hdy.6800615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A quantitative genetics model for viability selection.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">94</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">347</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D94%26rft.spage%253D347%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800615%26rft_id%253Dinfo%253Apmid%252F15536483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15536483&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mazurek</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimm</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leib</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eigenbrodt</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Metabolic characteristics of different malignant cancer cell lines.</span> <abbr class="cit-jnl-abbrev">Anticancer Res.</abbr> <span class="cit-vol">18</span>(<span class="cit-supplement">5A</span>): <span class="cit-fpage">3275</span>–<span class="cit-lpage">3282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnticancer%2BResearch%26rft.stitle%253DAnticancer%2BRes%26rft.aulast%253DMazurek%26rft.auinit1%253DS.%26rft.volume%253D18%26rft.issue%253D5A%26rft.spage%253D3275%26rft.epage%253D3282%26rft.atitle%253DMetabolic%2Bcharacteristics%2Bof%2Bdifferent%2Bmalignant%2Bcancer%2Bcell%2Blines.%26rft_id%253Dinfo%253Apmid%252F9858895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9858895&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077376000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.50" data-doi="10.1554/0014-3820(2001)055[1085:TEOPIA]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turelli</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The evolution of postzygotic isolation: accumulating Dobzhansky-Muller incompatibilities.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">55</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1085</span>–<span class="cit-lpage">1094</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BA.%26rft.volume%253D55%26rft.issue%253D6%26rft.spage%253D1085%26rft.epage%253D1094%26rft.atitle%253DThe%2Bevolution%2Bof%2Bpostzygotic%2Bisolation%253A%2Baccumulating%2BDobzhansky-Muller%2Bincompatibilities.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282001%2529055%255B1085%253ATEOPIA%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F11475044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2001)055[1085:TEOPIA]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11475044&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169903000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.51" data-doi="10.1016/S0990-7440(02)01175-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patrick</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnaud</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olivier</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daniel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A comparative field study of growth, survival and reproduction of <em>Crassostrea gigas</em>, <em>C. angulata</em> and their hybrids.</span> <abbr class="cit-jnl-abbrev">Aquat. Living Resour.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">243</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquat.%2BLiving%2BResour.%26rft.volume%253D15%26rft.spage%253D243%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0990-7440%252802%252901175-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0990-7440(02)01175-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.52" data-doi="10.1111/j.1365-294X.2012.05688.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plough</span>  <span class="cit-name-given-names">L. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Environmental stress increases selection against and dominance of deleterious mutations in inbred families of the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">3974</span>–<span class="cit-lpage">3987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DPlough%26rft.auinit1%253DL.%2BV.%26rft.volume%253D21%26rft.issue%253D16%26rft.spage%253D3974%26rft.epage%253D3987%26rft.atitle%253DEnvironmental%2Bstress%2Bincreases%2Bselection%2Bagainst%2Band%2Bdominance%2Bof%2Bdeleterious%2Bmutations%2Bin%2Binbred%2Bfamilies%2Bof%2Bthe%2BPacific%2Boyster%2BCrassostrea%2Bgigas.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05688.x%26rft_id%253Dinfo%253Apmid%252F22747636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05688.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22747636&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306897500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.53" data-doi="10.1534/genetics.111.131854"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plough</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative trait locus analysis of stage-specific inbreeding depression in the Pacific Oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.131854%26rft_id%253Dinfo%253Apmid%252F21940682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4OS80LzE0NzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTQxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.54" data-doi="10.1111/mec.13524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plough</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedgecock</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic inviability is a major driver of type III survivorship in experimental families of a highly fecund marine bivalve.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">895</span>–<span class="cit-lpage">910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D895%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.55" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pootakham</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruang-Areerate</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jomchai</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonthirod</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sangsrakru</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of a high-density integrated genetic linkage map of rubber tree (<em>Hevea brasiliensis</em>) using genotyping-by-sequencing (GBS).</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D367%26rft_id%253Dinfo%253Apmid%252F26074933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26074933&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.57" data-doi="10.1016/j.aquaculture.2015.12.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">W. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Z. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic mapping and quantitative trait loci analysis of growth-related traits in the small abalone <em>Haliotis diversicolor</em> using restriction-site-associated DNA sequencing.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D454%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2015.12.026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2015.12.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.58" data-doi="10.1111/mec.12262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaxter</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RAD-Seq derived markers flank the shell colour and banding loci of the <em>Cepaea nemoralis</em> supergene.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">3077</span>–<span class="cit-lpage">3089</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3077%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Royston</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">The <em>W</em> test for normality.</span> <abbr class="cit-jnl-abbrev">Applied Statistics-Journal of the Royal Statistical Society Series C</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">176</span>–<span class="cit-lpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BStatistics-Journal%2Bof%2Bthe%2BRoyal%2BStatistical%2BSociety%2BSeries%2BC%26rft.volume%253D31%26rft.spage%253D176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzmann</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozaki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A microsatellite linkage map of rainbow trout (<em>Oncorhynchus mykiss</em>) characterized by large sex-specific differences in recombination rates.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1331</span>–<span class="cit-lpage">1345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSakamoto%26rft.auinit1%253DT.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D1331%26rft.epage%253D1345%26rft.atitle%253DA%2BMicrosatellite%2BLinkage%2BMap%2Bof%2BRainbow%2BTrout%2B%2528Oncorhynchus%2Bmykiss%2529%2BCharacterized%2Bby%2BLarge%2BSex-Specific%2BDifferences%2Bin%2BRecombination%2BRates%26rft_id%253Dinfo%253Apmid%252F10880492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS8zLzEzMzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTQxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sauvage</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boudry</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heurtebise</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">QTL for resistance to summer mortality and OsHV-1 load in the Pacific oyster (<em>Crassostrea gigas</em>).</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">41</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">390</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DSauvage%26rft.auinit1%253DC.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D390%26rft.epage%253D399%26rft.atitle%253DQTL%2Bfor%2Bresistance%2Bto%2Bsummer%2Bmortality%2Band%2BOsHV-1%2Bload%2Bin%2Bthe%2BPacific%2Boyster%2B%2528Crassostrea%2Bgigas%2529.%26rft_id%253Dinfo%253Apmid%252F20096029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20096029&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.62" data-doi="10.1093/dnares/dsv001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide SNP identification for the construction of a high-resolution genetic map of Japanese flounder (<em>Paralichthys olivaceus</em>): applications to QTL mapping of Vibrio anguillarum disease resistance and comparative genomic analysis.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">161</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsv001%26rft_id%253Dinfo%253Apmid%252F25762582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZG5hcmVzIjtzOjU6InJlc2lkIjtzOjg6IjIyLzIvMTYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzE0MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">She</span>  <span class="cit-name-given-names">Z. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Que</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Candidate gene polymorphisms and their association with glycogen content in the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">5</span>): e0124401.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perlman</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corley-Smith</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Sex-specific recombination rates in zebrafish (<em>Danio rerio</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">649</span>–<span class="cit-lpage">657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSinger%26rft.auinit1%253DA.%26rft.volume%253D160%26rft.issue%253D2%26rft.spage%253D649%26rft.epage%253D657%26rft.atitle%253DSex-Specific%2BRecombination%2BRates%2Bin%2BZebrafish%2B%2528Danio%2Brerio%2529%26rft_id%253Dinfo%253Apmid%252F11861568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYwLzIvNjQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzE0MTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.65" data-doi="10.1074/jbc.M402282200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salih</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leddy</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuana</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The muscle-specific calmodulin-dependent protein kinase assembles with the glycolytic enzyme complex at the sarcoplasmic reticulum and modulates the activity of glyceraldehyde-3-phosphate dehydrogenase in a Ca2+/calmodulin-dependent manner.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">279</span>(<span class="cit-issue">34</span>): <span class="cit-fpage">35176</span>–<span class="cit-lpage">35182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M402282200%26rft_id%253Dinfo%253Apmid%252F15199064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzkvMzQvMzUxNzYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTQxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Y. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiang</span>  <span class="cit-name-given-names">Z. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">An amplified fragment length polymorphism map of the silkworm.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1277</span>–<span class="cit-lpage">1284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTan%26rft.auinit1%253DY.-D.%26rft.volume%253D157%26rft.issue%253D3%26rft.spage%253D1277%26rft.epage%253D1284%26rft.atitle%253DAn%2BAmplified%2BFragment%2BLength%2BPolymorphism%2BMap%2Bof%2Bthe%2BSilkworm%26rft_id%253Dinfo%253Apmid%252F11238411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny8zLzEyNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTQxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1417.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1417.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source"><em>MapQTL @ 5</em>, <em>Software for the Mapping of Quantitative Trait Loci in Experimental Populations</em></span>, <span class="cit-publ-name">Kyazma B.V.</span>, <span class="cit-publ-loc">Wageningen, Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1417.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source"><em>JoinMap @ 4</em>, <em>Software for the Calculation of Genetic Linkage Maps in Experimental Populations</em></span>, <span class="cit-publ-name">Kyazma B.V.</span>, <span class="cit-publ-loc">Wageningen, Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.70" data-doi="10.1038/nmeth.1185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Tassell</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">SNP discovery and allele frequency estimation by deep sequencing of reduced representation libraries.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">247</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DJones%26rft.auinit1%253DG.%2BM.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253D247%26rft.epage%253D252%26rft.atitle%253DSNP%2Bdiscovery%2Band%2Ballele%2Bfrequency%2Bestimation%2Bby%2Bdeep%2Bsequencing%2Bof%2Breduced%2Brepresentation%2Blibraries.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1185%26rft_id%253Dinfo%253Apmid%252F18297082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18297082&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253777900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.71" data-doi="10.1007/s11033-011-1298-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Identification of a fructose-1,6-bisphosphate aldolase gene and association of the single nucleotide polymorphisms with growth traits in the clam <em>Meretrix</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Rep.</abbr> <span class="cit-vol">39</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">5017</span>–<span class="cit-lpage">5024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Breports%26rft.stitle%253DMol%2BBiol%2BRep%26rft.aulast%253DWang%26rft.auinit1%253DC.%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D5017%26rft.epage%253D5024%26rft.atitle%253DIdentification%2Bof%2Ba%2Bfructose-1%252C6-bisphosphate%2Baldolase%2Bgene%2Band%2Bassociation%2Bof%2Bthe%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bwith%2Bgrowth%2Btraits%2Bin%2Bthe%2Bclam%2BMeretrix%2Bmeretrix.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11033-011-1298-9%26rft_id%253Dinfo%253Apmid%252F22160517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11033-011-1298-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22160517&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.72" data-doi="10.2983/035.029.0420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Classification of a common cupped oyster from Southern China.</span> <abbr class="cit-jnl-abbrev">J. Shellfish Res.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">857</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BShellfish%2BRes.%26rft.volume%253D29%26rft.spage%253D857%26rft_id%253Dinfo%253Adoi%252F10.2983%252F035.029.0420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2983/035.029.0420&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.73" data-doi="10.1111/1755-0998.12278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Que</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Discovery and validation of genic single nucleotide polymorphisms in the Pacific oyster <em>Crassostrea gigas</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">123</span>–<span class="cit-lpage">135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12278%26rft_id%253Dinfo%253Apmid%252F24823694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12278&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24823694&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.74" data-doi="10.1016/j.ejbt.2016.02.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">X. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">RestrictionDigest: a powerful Perl module for simulating genomic restriction digests.</span> <abbr class="cit-jnl-abbrev">Electron. J. Biotechnol.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.ejbt.2016.02.003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectron.%2BJ.%2BBiotechnol.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ejbt.2016.02.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ejbt.2016.02.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.75" data-doi="10.1007/s11802-012-1909-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Response to selection for fast growth in the second generation of Pacific oyster (<em>Crassostrea gigas</em>).</span> <abbr class="cit-jnl-abbrev">J. Ocean Univ. China</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">413</span>–<span class="cit-lpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BOcean%2BUniv.%2BChina%26rft.volume%253D11%26rft.spage%253D413%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11802-012-1909-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11802-012-1909-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.76" data-doi="10.1186/1471-2164-14-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhangoo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Fernandez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Saturated linkage map construction in <em>Rubus idaeus</em> using genotyping by sequencing and genome-independent imputation.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-2%26rft_id%253Dinfo%253Apmid%252F23324311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23324311&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.77" data-doi="10.1046/j.1365-2109.2000.00388.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">English</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGoldrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nell</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genetic improvement of the Pacific oyster <em>Crassostrea gigas</em> (Thunberg) in Australia.</span> <abbr class="cit-jnl-abbrev">Aquacult. Res.</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">35</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquacult.%2BRes.%26rft.volume%253D31%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2109.2000.00388.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2109.2000.00388.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.78" data-doi="10.1038/srep09939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">Z. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">B. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">AFSM sequencing approach: a simple and rapid method for genome-wide SNP and methylation site discovery and genetic mapping.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep09939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep09939&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.79" data-doi="10.1534/genetics.108.090688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Quantitative trait locus mapping can benefit from segregation distortion.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">2201</span>–<span class="cit-lpage">2208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090688%26rft_id%253Dinfo%253Apmid%252F18957707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC80LzIyMDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTQxNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.80" data-doi="10.1038/nature11413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The oyster genome reveals stress adaptation and complexity of shell formation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">490</span>(<span class="cit-issue">7418</span>): <span class="cit-fpage">49</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZhang%26rft.auinit1%253DG.%26rft.volume%253D490%26rft.issue%253D7418%26rft.spage%253D49%26rft.epage%253D54%26rft.atitle%253DThe%2Boyster%2Bgenome%2Breveals%2Bstress%2Badaptation%2Band%2Bcomplexity%2Bof%2Bshell%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11413%26rft_id%253Dinfo%253Apmid%252F22992520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22992520&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1417.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309446800030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.81" data-doi="10.1007/s10499-014-9789-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">X. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">QTL mapping for glycogen content and shell pigmentation in the Pacific oyster <em>Crassostrea gigas</em> using microsatellites and SNPs.</span> <abbr class="cit-jnl-abbrev">Aquacult. Int.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1877</span>–<span class="cit-lpage">1889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquacult.%2BInt.%26rft.volume%253D22%26rft.spage%253D1877%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10499-014-9789-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10499-014-9789-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1417.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">Z. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic map construction and detection of genetic loci underlying segregation distortion in an intraspecific cross of <em>Populus deltoides</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">5</span>): e0126077.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/5/1417.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/5/1409" title="Genetic Background, Maternal Age, and Interaction Effects Mediate Rates of Crossing Over in Drosophila melanogaster Females" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/5/1427" title="Quantitative Assessment of Eye Phenotypes for Functional Genetic Studies Using Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/5"><span>Volume 6 Issue 5, May 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/5.cover-source.jpg?itok=8SH0hKOw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1417.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PTdnzCaSSQfQZM7WRU4ftS_EExdaFJm1POkuESlDj7A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/5/1417" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/5/1417" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/5/1417" class="active" data-icon-position="" data-hide-link-title="0">A High-Density SNP Genetic Linkage Map and QTL Analysis of Growth-Related Traits in a Hybrid Family of Oysters (Crassostrea gigas × Crassostrea angulata) Using Genotyping-by-Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460134" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-QHbdY5-0KoYgHMKxUUeM3ljOu5rQhU_oC23ylawZ_EA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/5/1417.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/5/1417" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-7KTIskpsab8w7tIzE_apVxhnCFqpv0awQOyrg7_V7fI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/5/1417" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/5/1417.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460134" id="node460134--2" data-pisa="ggg;6/5/1417" data-pisa-master="ggg;g3.116.026971" data-apath="/ggg/6/5/1417.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density SNP Genetic Linkage Map and QTL Analysis of Growth-Related Traits in a Hybrid Family of Oysters (<em>Crassostrea gigas × Crassostrea angulata</em>) Using Genotyping-by-Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jinpeng Wang</span>, <span class="highwire-citation-author" data-delta="1">Li Li</span> and <span class="highwire-citation-author" data-delta="2">Guofan Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1417-1426;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.026971 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460134--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jinpeng Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>†</span>University of Chinese Academy of Sciences, Beijing, China 100039</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory for Marine Biology and Biotechnology, Qingdao National Laboratory for Marine Science and Technology, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>§</span>National and Local Joint Engineering Laboratory of Ecological Mariculture, Qingdao, Shandong, China 266071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinpeng%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJinpeng%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Li Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>§</span>National and Local Joint Engineering Laboratory of Ecological Mariculture, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Laboratory for Marine Fisheries and Aquaculture, Qingdao National Laboratory for Marine Science and Technology, Qingdao, Shandong, China 266071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALi%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gzhang@qdio.ac.cn" class="" data-icon-position="" data-hide-link-title="0">gzhang@qdio.ac.cn</a>
<a href="mailto:lili@qdio.ac.cn" class="" data-icon-position="" data-hide-link-title="0">lili@qdio.ac.cn</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Guofan Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory for Marine Biology and Biotechnology, Qingdao National Laboratory for Marine Science and Technology, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>§</span>National and Local Joint Engineering Laboratory of Ecological Mariculture, Qingdao, Shandong, China 266071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guofan%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuofan%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gzhang@qdio.ac.cn" class="" data-icon-position="" data-hide-link-title="0">gzhang@qdio.ac.cn</a>
<a href="mailto:lili@qdio.ac.cn" class="" data-icon-position="" data-hide-link-title="0">lili@qdio.ac.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/5/1417" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/5/1417&amp;title=A%20High-Density%20SNP%20Genetic%20Linkage%20Map%20and%20QTL%20Analysis%20of%20Growth-Related%20Traits%20in%20a%20Hybrid%20Family%20of%20Oysters%20%28Crassostrea%20gigas%20%C3%97%20Crassostrea%20angulata%29%20Using%20Genotyping-by-Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/5/1417&amp;title=A%20High-Density%20SNP%20Genetic%20Linkage%20Map%20and%20QTL%20Analysis%20of%20Growth-Related%20Traits%20in%20a%20Hybrid%20Family%20of%20Oysters%20%28Crassostrea%20gigas%20%C3%97%20Crassostrea%20angulata%29%20Using%20Genotyping-by-Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/5/1417&amp;title=A%20High-Density%20SNP%20Genetic%20Linkage%20Map%20and%20QTL%20Analysis%20of%20Growth-Related%20Traits%20in%20a%20Hybrid%20Family%20of%20Oysters%20%28Crassostrea%20gigas%20%C3%97%20Crassostrea%20angulata%29%20Using%20Genotyping-by-Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/5/1417&amp;text=A%20High-Density%20SNP%20Genetic%20Linkage%20Map%20and%20QTL%20Analysis%20of%20Growth-Related%20Traits%20in%20a%20Hybrid%20Family%20of%20Oysters%20%28Crassostrea%20gigas%20%C3%97%20Crassostrea%20angulata%29%20Using%20Genotyping-by-Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/5/1417&amp;title=A%20High-Density%20SNP%20Genetic%20Linkage%20Map%20and%20QTL%20Analysis%20of%20Growth-Related%20Traits%20in%20a%20Hybrid%20Family%20of%20Oysters%20%28Crassostrea%20gigas%20%C3%97%20Crassostrea%20angulata%29%20Using%20Genotyping-by-Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/5/1417&amp;t=A%20High-Density%20SNP%20Genetic%20Linkage%20Map%20and%20QTL%20Analysis%20of%20Growth-Related%20Traits%20in%20a%20Hybrid%20Family%20of%20Oysters%20%28Crassostrea%20gigas%20%C3%97%20Crassostrea%20angulata%29%20Using%20Genotyping-by-Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/5/1417&amp;title=A%20High-Density%20SNP%20Genetic%20Linkage%20Map%20and%20QTL%20Analysis%20of%20Growth-Related%20Traits%20in%20a%20Hybrid%20Family%20of%20Oysters%20%28Crassostrea%20gigas%20%C3%97%20Crassostrea%20angulata%29%20Using%20Genotyping-by-Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/5/1417&amp;title=A%20High-Density%20SNP%20Genetic%20Linkage%20Map%20and%20QTL%20Analysis%20of%20Growth-Related%20Traits%20in%20a%20Hybrid%20Family%20of%20Oysters%20%28Crassostrea%20gigas%20%C3%97%20Crassostrea%20angulata%29%20Using%20Genotyping-by-Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460134/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460134" id="node460134--3" data-pisa="ggg;6/5/1417" data-pisa-master="ggg;g3.116.026971" data-apath="/ggg/6/5/1417.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density SNP Genetic Linkage Map and QTL Analysis of Growth-Related Traits in a Hybrid Family of Oysters (<em>Crassostrea gigas × Crassostrea angulata</em>) Using Genotyping-by-Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jinpeng Wang</span>, <span class="highwire-citation-author" data-delta="1">Li Li</span> and <span class="highwire-citation-author" data-delta="2">Guofan Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1417-1426;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.026971 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460134--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jinpeng Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>†</span>University of Chinese Academy of Sciences, Beijing, China 100039</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory for Marine Biology and Biotechnology, Qingdao National Laboratory for Marine Science and Technology, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>§</span>National and Local Joint Engineering Laboratory of Ecological Mariculture, Qingdao, Shandong, China 266071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinpeng%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJinpeng%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Li Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>§</span>National and Local Joint Engineering Laboratory of Ecological Mariculture, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Laboratory for Marine Fisheries and Aquaculture, Qingdao National Laboratory for Marine Science and Technology, Qingdao, Shandong, China 266071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALi%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gzhang@qdio.ac.cn" class="" data-icon-position="" data-hide-link-title="0">gzhang@qdio.ac.cn</a>
<a href="mailto:lili@qdio.ac.cn" class="" data-icon-position="" data-hide-link-title="0">lili@qdio.ac.cn</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Guofan Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Key Laboratory of Experimental Marine Biology, Institute of Oceanology, Chinese Academy of Sciences, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory for Marine Biology and Biotechnology, Qingdao National Laboratory for Marine Science and Technology, Qingdao, Shandong, China 266071</div><div class='author-affiliation'><span class='nlm-sup'>§</span>National and Local Joint Engineering Laboratory of Ecological Mariculture, Qingdao, Shandong, China 266071</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guofan%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuofan%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gzhang@qdio.ac.cn" class="" data-icon-position="" data-hide-link-title="0">gzhang@qdio.ac.cn</a>
<a href="mailto:lili@qdio.ac.cn" class="" data-icon-position="" data-hide-link-title="0">lili@qdio.ac.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1417&amp;rft.epage=1426&amp;rft.atitle=A%20High-Density%20SNP%20Genetic%20Linkage%20Map%20and%20QTL%20Analysis%20of%20Growth-Related%20Traits%20in%20a%20Hybrid%20Family%20of%20Oysters%20%28Crassostrea%20gigas%20%C3%97%20Crassostrea%20angulata%29%20Using%20Genotyping-by-Sequencing&amp;rft.volume=6&amp;rft.issue=5&amp;rft.date=2016-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wang%2C+Jinpeng&amp;rft.au=Li%2C+Li&amp;rft.au=Zhang%2C+Guofan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460134/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460134/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460134/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460134/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460134/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460134/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460134/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460134/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460134/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460134/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460134/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460134/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/5/1417" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/5/1417.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/5/1417.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/5/1417.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/5/1417.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
