<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/5/1313" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/5/1313.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/5/1313.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/5/1313.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/5/1313/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/5/1313.atom" />
<meta name="HW.pisa" content="ggg;6/5/1313" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Selection in Multi-environment Crop Trials" />
<meta name="DC.Identifier" content="10.1534/g3.116.027524" />
<meta name="DC.Date" content="2016-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Oakey et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection in crop breeding introduces modeling challenges not found in animal studies. These include the need to accommodate replicate plants for each line, consider spatial variation in field trials, address line by environment interactions, and capture nonadditive effects. Here, we propose a flexible single-stage genomic selection approach that resolves these issues. Our linear mixed model incorporates spatial variation through environment-specific terms, and also randomization-based design terms. It considers marker, and marker by environment interactions using ridge regression best linear unbiased prediction to extend genomic selection to multiple environments. Since the approach uses the raw data from line replicates, the line genetic variation is partitioned into marker and nonmarker residual genetic variation ( i.e. , additive and nonadditive effects). This results in a more precise estimate of marker genetic effects. Using barley height data from trials, in 2 different years, of up to 477 cultivars, we demonstrate that our new genomic selection model improves predictions compared to current models. Analyzing single trials revealed improvements in predictive ability of up to 5.7%. For the multiple environment trial (MET) model, combining both year trials improved predictive ability up to 11.4% compared to a single environment analysis. Benefits were significant even when fewer markers were used. Compared to a single-year standard model run with 3490 markers, our partitioned MET model achieved the same predictive ability using between 500 and 1000 markers depending on the trial. Our approach can be used to increase accuracy and confidence in the selection of the best lines for breeding and/or, to reduce costs by using fewer markers." />
<meta name="DC.Contributor" content="Helena Oakey" />
<meta name="DC.Contributor" content="Brian Cullis" />
<meta name="DC.Contributor" content="Robin Thompson" />
<meta name="DC.Contributor" content="Jordi Comadran" />
<meta name="DC.Contributor" content="Claire Halpin" />
<meta name="DC.Contributor" content="Robbie Waugh" />
<meta name="article:published_time" content="2016-05-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic Selection in Multi-environment Crop Trials" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection in crop breeding introduces modeling challenges not found in animal studies. These include the need to accommodate replicate plants for each line, consider spatial variation in field trials, address line by environment interactions, and capture nonadditive effects. Here, we propose a flexible single-stage genomic selection approach that resolves these issues. Our linear mixed model incorporates spatial variation through environment-specific terms, and also randomization-based design terms. It considers marker, and marker by environment interactions using ridge regression best linear unbiased prediction to extend genomic selection to multiple environments. Since the approach uses the raw data from line replicates, the line genetic variation is partitioned into marker and nonmarker residual genetic variation (&lt;i&gt;i.e.&lt;/i&gt;, additive and nonadditive effects). This results in a more precise estimate of marker genetic effects. Using barley height data from trials, in 2 different years, of up to 477 cultivars, we demonstrate that our new genomic selection model improves predictions compared to current models. Analyzing single trials revealed improvements in predictive ability of up to 5.7%. For the multiple environment trial (MET) model, combining both year trials improved predictive ability up to 11.4% compared to a single environment analysis. Benefits were significant even when fewer markers were used. Compared to a single-year standard model run with 3490 markers, our partitioned MET model achieved the same predictive ability using between 500 and 1000 markers depending on the trial. Our approach can be used to increase accuracy and confidence in the selection of the best lines for breeding and/or, to reduce costs by using fewer markers.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/05/01" />
<meta name="citation_mjid" content="ggg;6/5/1313" />
<meta name="citation_id" content="6/5/1313" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/5/1313" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/5/1313.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/5/1313.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/5/1313.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.027524" />
<meta name="citation_pmid" content="26976443" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1313" />
<meta name="citation_lastpage" content="1326" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Helena Oakey" />
<meta name="citation_author_institution" content="Division of Plant Sciences, University of Dundee at the James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK" />
<meta name="citation_author" content="Brian Cullis" />
<meta name="citation_author_institution" content="National Institute for Applied Statistics Research Australia, University of Wollongong, NSW, 2522, Australia" />
<meta name="citation_author" content="Robin Thompson" />
<meta name="citation_author_institution" content="Rothamsted Research, Harpenden, Hertfordshire AL5 3JQ, UK" />
<meta name="citation_author" content="Jordi Comadran" />
<meta name="citation_author_institution" content="Department of Cell and Molecular Sciences, The James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK" />
<meta name="citation_author" content="Claire Halpin" />
<meta name="citation_author_institution" content="Division of Plant Sciences, University of Dundee at the James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK" />
<meta name="citation_author_email" content="c.halpin@dundee.ac.uk" />
<meta name="citation_author_email" content="r.waugh@dundee.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1808-8130" />
<meta name="citation_author" content="Robbie Waugh" />
<meta name="citation_author_institution" content="Division of Plant Sciences, University of Dundee at the James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK" />
<meta name="citation_author_institution" content="Department of Cell and Molecular Sciences, The James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK" />
<meta name="citation_author_email" content="c.halpin@dundee.ac.uk" />
<meta name="citation_author_email" content="r.waugh@dundee.ac.uk" />
<meta name="citation_reference" content="citation_journal_title=Transactions on Automatic Control.;citation_author=H. Akaike;citation_title=New look at statistical-model identification.;citation_pages=716-723;citation_volume=AC19;citation_year=1974;citation_doi=10.1109/TAC.1974.1100705" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueno;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_title=ASReml R-reference manual;citation_author=D. Butler;citation_author=B. Cullis;citation_author=A. Gilmour;citation_author=B. Gogel;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Comadran;citation_author=B. Kilian;citation_author=J. Russell;citation_title=Natural variation in a homolog of Antirrhinum CENTRORADIALIS contributed to spring growth habit and environmental adaptation in cultivated barley.;citation_pages=1388-1392;citation_volume=44;citation_year=2012;citation_issue=12;citation_pmid=23160098;citation_doi=10.1038/ng.2447" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Perez;citation_author=D. Gianola;citation_author=J. Burgueno;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=J. Crop Improv.;citation_author=J. Crossa;citation_author=P. Perez;citation_author=G. de los Campos;citation_author=G. Mahuku;citation_author=S. Dreisigacker;citation_title=Genomic selection and prediction in plant breeding.;citation_pages=239-261;citation_volume=25;citation_year=2011;citation_doi=10.1080/15427528.2011.558767" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=B. Cullis;citation_author=B. Gogel;citation_author=A. Verbyla;citation_author=R. Thompson;citation_title=Spatial analysis of multi-environment early generation trials.;citation_pages=1-18;citation_volume=54;citation_year=1998;citation_doi=10.2307/2533991" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=B. Cullis;citation_author=A. Smith;citation_author=N. Coombes;citation_title=On the design of early generation variety trials with correlated data.;citation_pages=381-393;citation_volume=11;citation_year=2006;citation_doi=10.1198/108571106X154443" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. Da;citation_author=C. Wang;citation_author=S. Wang;citation_author=G. Hu;citation_title=Mixed model methods for genomic prediction and variance component estimation of additive and dominance effects using SNP markers.;citation_pages=e87666;citation_volume=9;citation_year=2014;citation_pmid=24498162;citation_doi=10.1371/journal.pone.0087666" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=I. J. M. de Boer;citation_author=I. Hoeschele;citation_title=Genetic evaluation methods for populations with dominance and inbreeding.;citation_pages=245-258;citation_volume=86;citation_year=1993;citation_pmid=24193467" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. Hickey;citation_author=R. Pong-Wong;citation_author=H. Daetwyler;citation_author=M. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=Z. A. Desta;citation_author=R. Ortiz;citation_title=Genomic selection: genome-wide prediction in plant improvement.;citation_pages=592-601;citation_volume=19;citation_year=2014;citation_pmid=24970707;citation_doi=10.1016/j.tplants.2014.05.006" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=A. Gilmour;citation_author=B. Cullis;citation_author=A. Verbyla;citation_title=Accounting for natural and extraneous variation in the analysis of field experiments.;citation_pages=269-293;citation_volume=2;citation_year=1997;citation_doi=10.2307/1400446" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=Z. Guo;citation_author=D. M. Tucker;citation_author=D. Wang;citation_author=C. J. Basten;citation_author=E. Ersoz;citation_title=Accuracy of Across-Environment Genome-Wide Prediction in Maize Nested Association Mapping Populations;citation_pages=263-272;citation_volume=3;citation_year=2013;citation_issue=2;citation_pmid=23390602;citation_doi=10.1534/g3.112.005066" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. Fernado;citation_author=J. Dekker;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. Fernando;citation_author=D. Garrick;citation_title=Genomic BLUP decoded: a look into the black box of genomic prediction.;citation_volume=194;citation_year=2013;citation_pmid=23640517;citation_doi=10.1534/genetics.113.152207" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_title=Plant breeding with genomic selection: gain per unit time and cost.;citation_author=E. L. Heffner;citation_author=A. J. Lorenz;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_volume=50;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=J.-L. Jannink;citation_author=H. Iwata;citation_author=E. Souza;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy for grain quality traits in biparental wheat populations.;citation_pages=2597-2606;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2011.05.0253" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=E. L. Heffner;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Genomic selection accuracy using multifamily prediction models in a wheat breeding program.;citation_pages=65-75;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome.2010.12.0029" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=J. Jannink;citation_author=A. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice.;citation_volume=9;citation_year=2010;citation_pmid=20156985;citation_doi=10.1093/bfgp/elq001" />
<meta name="citation_reference" content="citation_journal_title=Trends Biotechnol.;citation_author=E. Jonas;citation_author=D.-J. De Koning;citation_title=Does genomic selection have a future in plant breeding?;citation_pages=497-504;citation_volume=31;citation_year=2013;citation_pmid=23870753;citation_doi=10.1016/j.tibtech.2013.06.003" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=A. M. Kelly;citation_author=A. B. Smith;citation_author=J. A. Eccleston;citation_author=B. R. Cullis;citation_title=The Accuracy of Varietal Selection Using Factor Analytic Models for Multi-Environment Plant Breeding Trials;citation_pages=1063-1070;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.08.0540" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=B. Lado;citation_author=I. Matus;citation_author=A. Rodriguez;citation_author=L. Inostroza;citation_author=J. Poland;citation_title=Increased genomic prediction accuracy in wheat breeding through spatial adjustment of field trial data.;citation_pages=2015-2113;citation_volume=2;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=D. Ly;citation_author=M. Hamblin;citation_author=I. Rabbi;citation_author=G. Melaku;citation_author=M. Bakare;citation_title=Relatedness and genotype × environment interaction affect prediction accuracies in genomic selection: a study in cassava.;citation_pages=1312;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.11.0653" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Massman;citation_author=H.-J. G. Jung;citation_author=R. Bernardo;citation_title=Genomewide selection verses marker-assisted recurrent selection to improve grain yield and stover-quality traits for cellulosic ethanol in maize.;citation_pages=58-66;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.02.0112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. R. Munoz;citation_author=M. F. R. J. Resende;citation_author=S. A. Gexan;citation_author=M. D. V. Resende;citation_author=G. de los Campos;citation_title=Unravelling additive from nonadditive effects using genomic relationship matrices.;citation_volume=198;citation_year=2014;citation_pmid=25324160;citation_doi=10.1534/genetics.114.171322" />
<meta name="citation_reference" content="citation_journal_title=Biotechnol. Biofuels;citation_author=H. Oakey;citation_author=R. Shafiei;citation_author=J. Comadran;citation_author=N. Uzrek;citation_author=B. Cullis;citation_title=Identification of crop cultivars with consistently high lignocellulosic sugar release requires the use of appropriate statistical design and modelling.;citation_pages=185;citation_volume=6;citation_year=2013;citation_pmid=24359577;citation_doi=10.1186/1754-6834-6-185" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=H. D. Patterson;citation_author=V. Silvey;citation_author=M. Talbot;citation_author=S. T. C. Weatherup;citation_title=Variability of yields of cereal varieties in U. K. trials.;citation_pages=238-245;citation_volume=89;citation_year=1977" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=H. Piepho;citation_author=J. Ogutu;citation_author=T. Schulz-Streeck;citation_author=B. Estaghvirou;citation_author=A. Gordillo;citation_title=Efficient computation of ridge-regression best linear unbiased prediction in genomic selection in plant breeding.;citation_pages=1093-1104;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.11.0592" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=A. Smith;citation_author=B. Cullis;citation_author=R. Thompson;citation_title=Analyzing variety by environment data using multiplicative mixed models and adjustments for spatial field trend.;citation_pages=1138-1147;citation_volume=57;citation_year=2001;citation_issue=4;citation_pmid=11764254;citation_doi=10.1111/j.0006-341X.2001.01138.x" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=A. Smith;citation_author=B. Cullis;citation_author=R. Thompson;citation_title=The analysis of crop cultivar breeding and evaluations trials: an overview of current mixed model approaches.;citation_pages=1-14;citation_volume=143;citation_year=2005;citation_doi=10.1017/S0021859605004995" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=T. R. Solberg;citation_author=A. K. Sonesson;citation_author=J. a. Wooliams;citation_author=J. Ødegard;citation_author=T. H. E. Meuwissen;citation_title=Persistence of accuracy of genome-wide breeding values over generations when including a polygenic effect.;citation_pages=53-53;citation_volume=41;citation_year=2009;citation_pmid=20040081;citation_doi=10.1186/1297-9686-41-53" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Stranden;citation_author=D. Garrick;citation_title=Technical note: Derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit;citation_pages=2971-2975;citation_volume=92;citation_year=2009;citation_issue=6;citation_pmid=19448030;citation_doi=10.3168/jds.2008-1929" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc., B;citation_author=R. Tibshirani;citation_title=Regression shrinkage and selection via the LASSO.;citation_pages=267-288;citation_volume=58;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=BMC Proc.;citation_author=C.-L. Wang;citation_author=P.-P. Ma;citation_author=Z. Zhang;citation_author=X.-D. Ding;citation_author=J.-F. Lui;citation_title=Comparison of five methods for genomic breeding value estimation for the common dataser of the 15th QTL-MAS workshop.;citation_pages=S13;citation_volume=6;citation_year=2012;citation_pmid=22640547;citation_doi=10.1186/1753-6561-6-S2-S13" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=V. Wimmer;citation_author=C. Lehermeier;citation_author=T. Albrecht;citation_author=H.-J. Auinger;citation_author=Y. Wang;citation_title=Genome-wide prediction of traits with different genetic architecture through efficient variable selection.;citation_volume=195;citation_year=2013;citation_pmid=23934883;citation_doi=10.1534/genetics.113.150078" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zhong;citation_author=J. Dekker;citation_author=R. Fernado;citation_author=J.-L. Jannink;citation_title=Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a Barley case study.;citation_volume=182;citation_year=2009;citation_pmid=19299342;citation_doi=10.1534/genetics.108.098277" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Ser. A Stat. Soc.;citation_author=H. Zou;citation_author=T. Hastie;citation_title=Regularization and variable selection via the elastic net.;citation_pages=301-320;citation_volume=67;citation_year=2005;citation_doi=10.1111/j.1467-9868.2005.00503.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Selection in Multi-environment Crop Trials" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection in crop breeding introduces modeling challenges not found in animal studies. These include the need to accommodate replicate plants for each line, consider spatial variation in field trials, address line by environment interactions, and capture nonadditive effects. Here, we propose a flexible single-stage genomic selection approach that resolves these issues. Our linear mixed model incorporates spatial variation through environment-specific terms, and also randomization-based design terms. It considers marker, and marker by environment interactions using ridge regression best linear unbiased prediction to extend genomic selection to multiple environments. Since the approach uses the raw data from line replicates, the line genetic variation is partitioned into marker and nonmarker residual genetic variation ( i.e. , additive and nonadditive effects). This results in a more precise estimate of marker genetic effects. Using barley height data from trials, in 2 different years, of up to 477 cultivars, we demonstrate that our new genomic selection model improves predictions compared to current models. Analyzing single trials revealed improvements in predictive ability of up to 5.7%. For the multiple environment trial (MET) model, combining both year trials improved predictive ability up to 11.4% compared to a single environment analysis. Benefits were significant even when fewer markers were used. Compared to a single-year standard model run with 3490 markers, our partitioned MET model achieved the same predictive ability using between 500 and 1000 markers depending on the trial. Our approach can be used to increase accuracy and confidence in the selection of the best lines for breeding and/or, to reduce costs by using fewer markers." />
<meta name="og-title" property="og:title" content="Genomic Selection in Multi-environment Crop Trials" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/5/1313" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection in crop breeding introduces modeling challenges not found in animal studies. These include the need to accommodate replicate plants for each line, consider spatial variation in field trials, address line by environment interactions, and capture nonadditive effects. Here, we propose a flexible single-stage genomic selection approach that resolves these issues. Our linear mixed model incorporates spatial variation through environment-specific terms, and also randomization-based design terms. It considers marker, and marker by environment interactions using ridge regression best linear unbiased prediction to extend genomic selection to multiple environments. Since the approach uses the raw data from line replicates, the line genetic variation is partitioned into marker and nonmarker residual genetic variation ( i.e. , additive and nonadditive effects). This results in a more precise estimate of marker genetic effects. Using barley height data from trials, in 2 different years, of up to 477 cultivars, we demonstrate that our new genomic selection model improves predictions compared to current models. Analyzing single trials revealed improvements in predictive ability of up to 5.7%. For the multiple environment trial (MET) model, combining both year trials improved predictive ability up to 11.4% compared to a single environment analysis. Benefits were significant even when fewer markers were used. Compared to a single-year standard model run with 3490 markers, our partitioned MET model achieved the same predictive ability using between 500 and 1000 markers depending on the trial. Our approach can be used to increase accuracy and confidence in the selection of the best lines for breeding and/or, to reduce costs by using fewer markers." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
    <title>Genomic Selection in Multi-environment Crop Trials | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"H2xYNQlkf81WWnJwbpE_edbPP5WbY9JuJzxey9hbaoo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460359","apath":"\/ggg\/6\/5\/1313.atom","pisa":"ggg;6\/5\/1313","ac":{"\/ggg\/6\/5\/1313.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/5\/1313.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/5\/1313"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/5\/1313"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/5\/1313.long":true,"\/content\/6\/5\/1313":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BBxGD6Ji6Dj1Wzomea0\\\\\\\/XCANy4Pc9SqZ81\\\\\\\/mWUIX\\\\\\\/iIW3JWFdCZxN69U7F5mvX3Ls0XwuX8lgcUqVu7i0bB\\\\\\\/osHJjTYwL+R3+ZEMSate0px5NGGsxCuR5kNCKpEoepWxnnyzzg9I2bnYHjnlUgZHfNPzcPpPsvojBy2J8G2WqIoqdh2Vm3a5dNA5nruQNISsHzIsaLHzkFbGLT6TB1bHU2BfbP0yxcWZjiNU4vCwk79TRDQP49mMviX1BUcPZswwI\\\\\\\/ZyQ3OJhOE6bUuNwMiHLnb\\\\\\\/buFtwWI5fOV1xp9u\\\\\\\/Nf1EEK9B34S9YKmWY1G3yt\\\\\\\/TL5BSoG9croki54\\\\\\\/+9vSAUDtaic0hBiu9d1r4vXzNhc5w49A83jwtn3rkEG2j2PpMT\\\\\\\/sqpE7UNroTQwSw+m25L6z2uA8hpNQ7lYq9K34pKHKcXtFudYd9FdgaZwtIm\\\\\\\/9WUvQI4ssw993MyJ6PZ0KDwLj3\\\\\\\/wdagInNGyaHXARkLmZfO21T\\\\\\\/MZ+KuztW3D6iieNQkSgUnU+Z+pLE\\\\\\\/gPowfroOTGKoq0djKF\\\\\\\/ZdKLUsO329Sfx0zw7iwyPCOJrJ8qpUqHcAaB1FNNQIpaMtEzc8pCgE8Z60yb9IKxXUCV6CEfuaGNylalgfuoqRmXSgLh\\\\\\\/QCaMb4+mYIt72zBjqGtR3ULfmMa3H56olisTMhZexVdeiKIdzzRjX8yO3lHiCw8ZHI3jYjw96\\\\\\\/tVfad\\\\\\\/o4SGhFjrcjB8wqpblGNhCDdah6ucfxATtNg\\\\\\\/8G0GpH2yrIfi9kBe4O5cuSuQLAU8cnjqXno4F39Ge3nZuG+50rEG\\\\\\\/wKyjSvYJmqTXo4vI+Jt\\\\\\\/nxpryz2WUVel\\\\\\\/7CkQ6cMIRS0j+wAqWEyPyLscZGnO\\\\\\\/ySLo9syXLSuRxfAReNlN9L\\\\\\\/YxQF9VOmxssbQR51C+br\\\\\\\/zTGvPLZKvmzDD4K9GboK+eVw4vPZsmrGPQBM\\\\\\\/a3hJY4AiR9ZylJUgpaVr8NhC+yMo7Yhqyf930RRi34h3TFVJEkGxfqPLT+aH006\\\\\\\/7oom+e14JZE60zr0JeJQxVGCu+t\\\\\\\/hXdBftxqNyZHSkPbcrhtlxIjwMZ943w9EZhHSWrrrDPcjzbPBYk9DjkWTdv9Vpu7NJ59OrMx7DQnzIqzqUbTT+p84X+jSwt8FjSX5f30xqyr9sJ3svjfnK1Gj6CxvbXwCjEFBTxSy9bu07SOQvCPI05eskp\\\\\\\/YPrOSpZC23C2aw4RbQwWB9Lv+YdkxWZKkqFQOAuJeZMYxbBlv1gr\\\\\\\/j4wNwBwBOOaUIPwEpH\\\\\\\/oCsm0fqpU\\\u0022,\\\u0022iv\\\u0022:\\\u0022Q18gTyEd7u3jcvPdt2frCA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ac16102ce4a12b9ed2b41afde8b9908954722536bbcd8873afd25cc4c23ad39b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Klm54pAZFexWpeIJI3cn4v4Icx4jVu29trHu95WiXZhDXSzDmAhcF4ax9V6nH7c6j3ac8Wd44J3On\\\\\\\/aCiuchNkGKNrndmzdmu+5Po\\\\\\\/urWRIzhtrlIvTiabrru5W1+YBxrOCEuGCkYiNKTmU9FO5tCKRqyy10Til33t\\\\\\\/UXah6hW3Lt6NU04BbmXCZR4ogSbnvejw6UNnpDPiKbW3Uy6gUv4lscrBzrPLifZzty4Ci3\\\\\\\/DB5xx6RZkF8uUsBtCSt7iOJfRhNPSQKw3AIg7uUvsS2UTlFLwM1DpfgvAYdLILFDNI6vu47TRMoRdk7tE2JlmXhutWpQ2OZD7Q60UemW7aadOKMd8SCWHkdbQ6BmvooYojggU8GOWF6sIZyN+8Dy8MPoZsZ708RAPjEhs1aqmRUWhlm+QoDrRKtzJy80nAq6XaZ16nzL9moBkZbCJBnQJF4Oe0T+svseiJZYaHLIHsaw3XE+\\\\\\\/G+YtPQBEdleLAbmRdYji809pa761YbUcKJKruHz\\\\\\\/r3HWndRH+8tG5GnOxcfmC5+k8IQCP8MV+E058tQWwqZf2GZ+csiZDYM3o6+2URBN\\\\\\\/o42W5Xnde7NZDNLACNVTCBTOTQidFi7Zy6TVC1c8\\\\\\\/DJ1oQLAeltDKBsmZlNAsD4+UmZBaZIofpOyfltVDLFCUusXb0wFChDAXgyUTn2rtzYpJAtGHHGjavj9yD5m+CXnOWQdt2p+DGCTh1eY1vi22nFVN6+ATIbY\\\\\\\/2PzBc4oWa6xsKNU9JzoRj2EXyYdCl64m5jQtfBp\\\\\\\/P3N5d+3sv+DXd\\\\\\\/x\\\\\\\/sCN41lxVWHHFbvpZ\\\\\\\/jBBUf6dwGjZ0Ubnn5Lb9aeQciSapBuFJeceJogZHAzvP\\\\\\\/eflo5U2ko1lhi1gE4xItbu9HO46tGjMbgHXuuRWPw+h9vFV7nQQnPutQlE3xne0sgNR0PocvOVKMiX8wDQUM8etl\\\\\\\/y4AsXnT8bkp3kLvUAma8p1ppZcnpn7ruzOCg7KTT3Y45WTBQHfYbfbfyMg\\\\\\\/U7MXnVH0PotvD6Fzt81NEod8QW7IZeVVu\\\\\\\/tq\\\\\\\/kcP1Ma3bGw85DFP\\\\\\\/mehiBQO36x1KnFsblYMEQ3cXjQtLo\\\\\\\/W5lmC0rgHfGPV1ePnzKJXM1TT01hM42TOh\\\\\\\/N\\\\\\\/LZc6CQdotwdk6L0HA3CN6oZcRbzhoiK70ZDejkj+eVRR+jUS5C3RE2UHlQtn0WtZHetUv7LOyaKjemaBxRsvBBNGdvnTVU1vlHfj5tDl2hEPjohbcNax74TuIEHhiGzYl6EgFyUyf6299x\\\\\\\/KuagOUeYAnjbHjL3\\\\\\\/C2bS9uBNc137gkAJ6XnnOjsFIQUBMLaN0rUTzZHVgLDDcjGRmgkJdcjaZixcHmXJy5mKtpURRdmsfbQyqjdi2FHp020WVr+CbHrO4weqqGbMokeg064vnDR1yScIMCP7mWviI5LAfPmhpwmLJgM3A4B980WcVBgyAuoByr7WOezDT8ywHBtkOZFxKxqmZPuGUN+JV0YdoStWzzfs1Gl+ta4+aKEoQK0L1y9K60qRoo2PXRj8D6IqmQutEmrQJWhHeI2SYEiDZcCS29nm2MvNouIjatd0vDF5QyDBMe6UdRnfP64D+\\\\\\\/SZD2P0RArm87WiAtHS\\\\\\\/U2JaUqiGUkHlA7TOif+WRPjdgIZH0\\\\\\\/7gV\\\\\\\/u6PuGS86W0eqgYv6tCtEGEtd3kpoM2PIzc6UJ3mpHkosGzFns690imi9JP+NasKZeddSUdjF3pJPDX454SavmQavkfQvGupQr8bGZDYy6GD\\\\\\\/TSEhfYAtHk9kJJ57FHKUWVSJfnY0YmIC\\\\\\\/ofWPuEa\\\\\\\/aYp+2OTcg475B94IB1HKEw3Gf7CF+smOxMnK63ao5T6\\\\\\\/DqA0Fpw2LvtdXF3wGbcX7ghm\\\\\\\/gtqX\\\\\\\/i68Ds021xJ\\\\\\\/2LtFjYLwBfSefIrq\\\\\\\/\\\\\\\/iZySAuBbTpDr3RkuE498Dvf7xy8fz+mGjj34BSGkn6Xas7\\\\\\\/V3QAOLyVsYBgY2cnWXA4htcpKQwJ9Tuol7xSP8mbX4lfAM00LdwZAugk\\\\\\\/fo94PdxUQTPcCv2hFFdiC4Rtkw4rDrCdPE1VIqyBvZ1X0bWlgR68FJsmvP+cuy9SWZcKVp17424RDx2S4ENylwxmR2Hf8IFvnAh2V68IFEE7EmrMVAHVF95jkjZAr8m3PKLsRg2C4BAYglcUQa5IRDuAJzJFnwJhVCAQcWnZLTwHZ81ZVSB77LGQlnPqekwl58cb2hIi2BuccKLLtXvD+hZpCxuT5LtH+zyApucRlwjiFnEFcdeFXmJCXM9O1Xt82Ed4vsSC\\\\\\\/wv3bg+en6rIcw2dFdBOPcy72kGdABCeFlM7FBl\\\\\\\/VCgRdGfB6UQFDGZZ3LLuWI2arvu79\\\\\\\/FVvnPBGx+qMzVanpS8kwY2qJ71LWTIMxXYvSOl2vmfwWHcE1xPk1b3Zh7lZTT94b2A3D7LpcT\\\\\\\/TcjgQPxRpnTE6LNlqCi9MlkqbUkYg9h2SnZyWFbToxILHJVSw8pF7FunsWHQAxV5bqAWHaUGiFUDOHQlX7DqS2xKp1\\\\\\\/LvQUHuC+vcofPvlimJCGI6a5d5f3UATFkq7GDXEmI6YWTxw8oQXv5cq1Att68b8rAW1eDfhFsZ2ohke\\\\\\\/QnI\\\\\\\/PWc5uCoVv58o7xb0tEPeJBQbwOPvigk\\\\\\\/ohrgUAKWO0nnGEPgscI7qJH2zbNWBsq+KyRKgT19PUCR4sg4NfYBvGLoXKbf7d1GfiggVMFfQFk6nfbtGNB0cmSYhulqehXFebBaEa6fwLa7cz\\\\\\\/vCiBVFU2xJtNscm1yKkICA4WUvb3sUxbdMHvRWWWEH+1o4RR1fMnSZ5SvoOydalGevfVvJllYujJi7RkNCLOgh\\\\\\\/MJx0hFrQAlGX4fw81zNgo8OJU2pIkK\\\\\\\/Gk7R5Jpfo4o8nwJ8GqX81k0VAMDuiZudGqauWXD\\\\\\\/6+5TqjeOeXaHizH81\\\\\\\/qISM44NCvknCuJeyrhPa3\\\\\\\/zR8ntHzvIEwDLsvlF3OVKSrrs3tOWJLjl14QNy6XEXul7aU6px48hWwhmHv203NXw6j7u7+97I4DClscX1q2AD5iQ+j0s7Z\\\\\\\/YmSa0kYBrGSz619c7TjucfKakwN+x\\\\\\\/nNtSJZQuTDNTEdvmgLGvNbBpPQILf1bgTq2uUn6Cj03TUEoy12fF2VbtDKqU7frdTKta0b3IdWcxGcAp91ZMKfglAEIEksz5WAYSYkwdZuxP5yJDIpg+y3F4FNex5bSOxyI8Wm9uXvybp9WYBCZLIJUO9pDnoQy8z08lK8TEHD\\\\\\\/DJZ85384eBdRkguPcV\\\\\\\/NZgvKb7xbm2AGFL5Ido07jitv3CJ6G1yHsUuOJ5\\\\\\\/gqbaxDOD8KMiCB2ZPRBp3zUPN64QfqmdRsCbLzEt+qUFUYqMj91F9k3LSFuPeXaf8jVmgz9U3eudmNNw2cqyD0B00NKwTAsYmsEc8b2erTlIJ1XekP2R4lMgQDKRN0snZ2PtdUm6Da8AiQcCkxLLtNrIsWnhU65PWG5KAvSBwOSQw+AVc8b0HPiSrTz8XuMM4dxWHTUAOaCCgqz2hbf4X4wkR7bWYtxSXHltIKvoiL6fs3a19VGNSqE4b4cbrGjLe730YlcBPOrpWzXMJA9WwwkfMgejMJDa7pNTppS0wqppfjZBPhxpYBsnNRzePiE5K9iDM6bMQt5RRa7KfSWnnprpgJb8vs5rz3OXOJFWN7VgoHf2XsZ91SWKKq8hK\\\\\\\/cvdFfkdOganD8q9JPVzX5+DkBkLmYrD3LQz27+375xwjBwlLXqXS8S4B1yzditbVXngrelV5L\\\\\\\/KzBiE1V61nMGbo\\\\\\\/8Gb2INNaKOcOnYVEAAmkPLk6vRreiHBdeNac15VsjdqFg6\\\\\\\/aRBHcXNhyZuJxZAMuKSokNv7FRm+WK7EwcOXhxkHNonxuzBd\\\\\\\/lReHchrtQzhFmHCQ9o1FYUGgsl\\\\\\\/iUHOBc7WzbS+qbE8\\\\\\\/XofD5dAjWpdlv6Lbx06fvtCNwY0UyEImyImisrteP1nzxX3MffEH1F4L1rEqGMA6\\\\\\\/rwBX9m823mQaDYKAytZd50PUfdfTnjsjytPRuMnts4DNQeqtu0uM6h0YO5WFSoOZm+lFHn9pYZ4gGb5OMlZbfwVczjGOzuZhjq\\\\\\\/f6Abq5+d2cZEnQcCheNQLpvbcJX922k6dnVLsfDrImaQVBSc7G\\\\\\\/NnLcKJVpWCZzO2Cw+JjUdIIOlJlXmaWPFKNGooCgrjDh7vfZbMc1QbW4GUt8RCV6h+CyzhenUQ2d3Ce3wyzj9EPkFj4tOfDoZ3pfdCSoeGvMnsypo69emkq8abw2spuJkEL\\\\\\\/7mOSHb5INo+Q0xmrU48PtrGghihMixD3PzvxyjD6Jp0lIg80ZhXcYlBH2ZBI3S66\\\\\\\/ZaKAxwldN9RfqRNvPwnr2U5hagXID7FkrI1EFPxAw+zIL2m9LjqMzi\\\\\\\/36DYd6WjeENiyxDP0m92IS2ah2dvtACZxB9RxZE+zzlRm3zjySjHAFsYsIqlekSwEVZAvjcIPy7NZ10Vz3\\\\\\\/+iwDsjfbaAf+HzaWV5gO89V+Dv9wWs\\\\\\\/2wJIp1senfF8cYEvp4YGpu7mV4XrOCEjuPtFlXAg8kmclT4+LQFdGbtBLZzovrYh+9bEn6kMutCYC2fsBAmG35OPVW\\\\\\\/9YWjIOtALXbFKp\\\\\\\/FBz0Cz2M+WXkb9XTzPT\\\\\\\/k+uIIf4BKQut2WFzAgOiQF4azBP0VuGAnh4zPTADvEMB5bvtVQrtmRuADQp8kZDapC05uJtqpTvx8O7YkRNaBZ73zzT3saGXUgKk8MUE6bJWhjKYBsEQjh2VXDkw6tTFeogfWFV62ZbVryDGrZwx88opd6opElQ2yAh08T4ElZKFmw6TBedgdIgiyOiyKJI7ZFSf4my9wfmsJObJfM1oGh6JGkM1mrtC6oOq377od0JXrix0+mxTz8qphEihEzMyrprkqSxobYVMZgfg3wGhUUD7EP5hou7bBR92OuQyHhdW\\\\\\\/vR5jRyd84qbqJZ0MuK+HXb3BbXfDv4kf0CNUIFqBLZPgVnzzGSzYHBBShcriOaSQ6iLQ6awQY\\\\\\\/RTV4NcVdaB5tu6lGAAMJsAOFwF6nkBYW1\\\\\\\/XTK2hCXHK7Zkn3wwTS1HRkZSshKS2mI46q2VDFXY0WfLUpdfP5I\\\\\\\/TSLQ54nG8a5PZt5rJjrTRACdtha5sEWmZ2enHfm4tVEjn7cuvTtYCfIGcj5s2TLZ1RmyCtNgNH35hgnlUyzzLm6blgmB\\\\\\\/jPTbZtE6hSs49LJWgFtAvgkwVh0vvX+yzHBBk3OSjs4L90\\\\\\\/llUDcw69FBw1JBllzCS7GNI2th3mpPOYuZpGTpP3csrg4u41Kr6zn1ZShIzQSiKsL0k8ESCFIxrbaGnWlcK+lc3EzmF52jppikssLcXxz0zG4aMScdI0qN\\\\\\\/Aj6U0\\\\\\\/ypN6uOh51Q9aaIB9caSdH+41NJxzYrtNFyUETfHFNJg+13oC+pdX9\\\\\\\/2qbDgXmE3zReUA\\\\\\\/QoeDtrsctoc6sOWvyM78jp2PEQt++O\\\\\\\/CLuk5XZTCFgB\\\\\\\/YCEF+hH8AKJvUW0Easjjbxnan6FaODr9dOJlTSkuPQRrjaRxY\\\\\\\/cjpBfjlV\\\\\\\/tQvniLzt2YexwsCCjT0xpX6iF+dSpLUsSlfwQGFobs1i0rmQKseROFY50TMtl8glQENHoZ5o2gPwxPS4aFXFPRLtFQwON7CK0OjPQwdHterIfKbHphJNavZ24vTRFqdBUZuIrPMXda8GxRMpWWulI1auWPlh6Tgtd6lybyaU+Pus4vi3WB50IfVzVcb92eMldqq3J9vSH2pefFuWgUeBD6GYsOTUvVEUUf4MEAEUcZrNE6dstIpR6TTZWwVz7gNrg3jcgNIgPuxDoD2fJTqil6q3KYRi7lF09iHZdJ6vhQ6biN\\\\\\\/x9MNRW0rddYbUJ4g2UjHqviwsfSi4jnw7FXJhXFPWBBV7c9kyWli\\\\\\\/+E8ExgRTu96f7bvJga6Fb6dNlYGANhW+BKycD2PP6O9MFVuox3r7VEhUE1dpOKeUw\\\\\\\/eOuk7YDBmnzUtioVDgbD5J+4CJa8yBGle31DK3lyR7ro89X7Xj1FIIEJwf2cbiQ1AEgCdFBp+oGYYihOmlTG9ZWbhzuaN+oEfXcPKQU+BupvckecdGOyaFKZKaXAGkGrTpQbSlUIIpm8jMiTBYrJN6Jmq10DaDpiNQVTpcTzPHFhlcn9thYIm54gqjwsG5rlHeP7Gf\\\\\\\/8xCgmoCJfZ6bPEPkeg6jzGGL8W01CcAXZ\\\\\\\/+3MUla9nL1l5+GRHichqUe8rjkAP9Vwd32l2R8ONmBOfTg4FePqDLicrrdQAzqx+vYAVUJSmDlcLhm4HCuSkpHo1ko5YOQ2CnqDa0ZCFPqFRXsuwt5YNuI4O40R3Zr+F8ZJHI+8shhOXyUFKhENf\\\\\\\/zq58deSZnAiScI0bJXms6xVq3wQlgnJVhEVcQT9pylUMUQbw\\\\\\\/OKcZsl9Q2REluQL6jRuXxuHejJCFdKYEf7UYeVM9WR6GTBgo3Ns+icsFIgyRvxNbBMm+5+zrp24UdFaLNDGKjpJS7LBIEz4VsnU6+VWZp5EI5qYKp\\\\\\\/t\\\\\\\/UzuFOfbtF1+j+wt4TFnMNekrQrPz7HQQH8GLSob7IfFevs4mo+\\\\\\\/Vz\\\\\\\/xh8kbOLX6XhknMQFT9wAWm7wEXJWVkcAtdXsixawvoyIhlZtvT93sSAWQC8iw7L2VH36KTnlfeeyOESqJtJ\\\\\\\/CgIfpDIv6N\\\\\\\/MpCeXhaiFSyNmdIsP55a3Uc1\\\\\\\/05yvb6zaPRYsKaFgq7exRfnRPI8GjknXtsFeKH9T5kZxGhqT4nHpHqbmKLC6uikrKNnJpEmPW1JrT8BGKaU9cOKy3Tf\\\\\\\/iSy\\\\\\\/EkdxszIM\\\\\\\/9MBFzqxuBvtbGmI\\\\\\\/TK+UHJ0EPvheZuodrgn5atM0Oa37auvXghMS9nzr9Uony8bapHBRNmew13MLzzUpcl6faGyY+IwGlPKryXV\\\\\\\/GMG2dJY4g8MiPNdDWqPQz8s+U0Q+KYu613gBKfKNX7GoHtUmyc\\\\\\\/ErpBRf\\\\\\\/iDSSoqyO4NDPzNTOhniNOlZHSzf+01mvC42eNanIzwT6qpoBrZOeQMUfu6Rm2uTOJdMt4nJxj\\\\\\\/UkCd\\\\\\\/qQKrRwPARbzilpXyfW8bpoVggpdvVj7LL8SvzToWekwDqXzFecDxcnlOzpyW9gHUs\\\\\\\/gloBNS7XUIqvJaM2Zk7qWI+kPplJ17gbflaDEZ1UyG+2wKig1vSDK1l9+BbY4OHFXu5w9EeeS22AFtQSm0gxF4TsYn\\\\\\\/oSh3U6z185iYrFf+HY3pd2MJGagt0b4uBggN5i2Hb6K+EItECz01GaRimY5037fbb\\\\\\\/bdcxWBvavW0EuLGQg17zGsaLj9v7ZelnuyJIkfqoXigi7KdhvzvZM\\\\\\\/8E4z+3yno4r8UcWHYaLrUxozw0Ra0Jgmu4IfZETud3wukklLmj5AQk40tgY+9ofbs+L1ZHGY\\\\\\\/IPGl5x5Kue17mihA5RnGUn56qrmzzLT4NNL9zb57lCCY0HMlIhRki17f69EiIfO9jU\\\\\\\/iQeI+TXjovi0RfyKzgG8dm\\\\\\\/SAn9sGvhzMgIrEpyzIJL6Tn2p9YR7V20IhO1Np0BTB1CrU7mh0LD8AKuSa7BAGgvFRYutXvhjI4fKwSqEfYvqBOjgQ8rpIqu\\\\\\\/3A88r7ieYettz3Qs7tL\\\\\\\/+byUmI3z8ykk61sYghmVReuKoxaf19kEnzhUEhnbGs3RTKPzAthqvyq0OiHRHDHLPMcjeVTFBjVEY3h+HWtwqMMOc\\\\\\\/g3LxQoid5JgJ0H9Z\\\\\\\/HCtmvP4Qnp93aWMmwTxlRNeCY9gIZBvDy7YaRrPuayHmqtGH7VmEO0UO3JkwOlY7+tclsRvD8N2gWHaOuLePTX8tjYvAssAr5UqO8zWkI7qEZSlDNGZKgMbPgAuZOqLYGOfZK\\\\\\\/\\\\\\\/SDhOvePCj+NIVtlJ8w72+mPWrfm9bfhNs03KTtdrJfLfdefskRgMR0Afsxr4JWE0IyoEbsdsHH71IeEjUtwiEuNN8NXFXR6jSZu+Y\\\\\\\/oQ4XFDon+lmb2UK9vpZKRvqx8J+U1PCweanPwcCJ80hHCsElRCTHaUmb4oszytVRX2smgwy6BNsM8P4oHc7KLaBfwNnTEPHdukAZhG4iyIq9XiZkCzs3yKG5H7au1eW6BYE4b8n3vvxob3UC7WsSIpaW7RObZoLMZ4CyPZpZoXKazWk3XkHRIKCmxE\\\\\\\/ovh67B15FSDUvd7vpoDKefC\\\\\\\/T3HzRHazavd8SzY19po6MQuDUR5Zm6yXiPGVt08cDqCUmMLdPQM4mcmpqgFQVxy2kDn80cLWZzCNy\\\\\\\/lZcxQnNZniLg+IwtzMhUqDWTJUZGmXrEVwFzaYveNDqn5uqifNbfr3xw0Uq33ujVbN2tb+3LDiiqa+x676lMa1ag\\\\\\\/hfXVfn\\\\\\\/SuA6vSIsAoR\\\\\\\/HOIeZw0YnJfvtJDX3bE1PFYgW0oyg2T7sjWZDdi3iK10\\\\\\\/PmM5byvFgzSxw7kjK1Z9ucMPXSBbtpPlU2zYlwVn5suXDPqcnSuGR\\\\\\\/FdiZk5aap\\\\\\\/5n2rlwsk3B2qj5ik86pdROqjD598npsqxNfXOUHXH\\\\\\\/JY1QlKMiZmef9oEvDa4rAzksWVUUb2+d85T1a5sspsXP\\\\\\\/LmFO4hcvn9L628pnbLofreyf6xBI\\\\\\\/gZpCwva\\\\\\\/1UM20q\\\\\\\/Z4BQd5KTWGNgr\\\\\\\/\\\\\\\/NuPht4UOwK9ptf\\\\\\\/sZiIlBvUfUsaRQso8Hr4DgiMukldmKjkPT9Zp+ijkdXCZVXUGOiSNqb0w4rU68Elwk+qutEaRZ0sLqPE5Asee2lKAYMXes3mmSzbZVJw5CIoEqjEXUsuEzwJpel5LgNuLuHKk86Fq8Jj2lzrqPgKYtyVki\\\\\\\/nBmEmFoFfcpDPoWKzvJuQ407LHl5ly1kdB6fqOKAnXb0oTM+V1ODjofbDku5usKdgQOzmTqbne\\\\\\\/6GtMJr+zqEY2JlQX\\\\\\\/g+qDNtA+F1mIsEpVrYianBe5FLmkoxpgZhbLVBmJZTL6q9Ml\\\\\\\/nvYyEaRJ020IhIxeIK752tCVEd326JwngQ2RA45pQrEdOd6lQIhNiHB\\\\\\\/UrxLOiyD0f6UJdDkXB4K1brtWasW8zR5mQdmGtSSsYT5bA10eRmpjkbnLsFsT1hfizaqkQbTKNg0v5YdU0nrPjK4bYWmPybI9XZZREufiRzbsnzeCbo5RZyhIpduRiaD9fUOx5N7OsPqIGgATnUeKmStEns79eIK\\\\\\\/XgLO3Nf\\\\\\\/Y\\\\\\\/1Ljqv7oRk\\\\\\\/\\\\\\\/br+qQcROgzGEbFfjyo1QjK4gy3t\\\\\\\/wHPt6F8RJHt2D8HKypTQxuf37CRaITylukcnped0sdNEGLYJ0b8i1TyvSnPYhQ9tgJjxugi6U8wDFg43ZV6FRXM1wuhMAfQ41fc8EqUPY9w\\\\\\\/5Fa1c52cVgsOicGZiKlH3hNncSObxnCVTx3lSzGgr8NH5ouixX0iVU0OzdsGP4DnPXAhnoUYnWnt6T1YB7Tm+zVUfg6Ndcff0J0mOjxMcUl\\\\\\\/vYDC6J5qAWesafOZPqEkFMW6Clwgm2khtUNZNQzDPR4ZrdPXyoNqYI6GkLNKAWyPLSoN0VQ\\\\\\\/Et8W7SWnaVeu5fJHmLReRT6czMjLBpjxfSYCpcEpiRCMeBGx6dPYKCJQKvVTcHfbCNfcZ3vcqMssOQWnJ9Wp5dUUxJqKgW2Wdysyx7B4rIM+c+L6TqxLxAJ4Qbu230QZEbo8eci87Vt\\\\\\\/66ytcdCKQ2EQfC0yhyAZl5lvntAfIOe\\\\\\\/z60QtQpr+acoN8dSrRGLoZMpHpc516RdBPnihiNqNwfDbqn1Olf3Oqu8n4CH8Pg1GSgHttapFjy8XeS8YyVmF2xhCf9mF2y8eFWU5to8gq1BQCdcp\\\\\\\/grVzCYTFuKVSsdckTZIznRsNqKSw+rZyMWjFmjo2U5k6MqcmJxOZCbyatc0grDnskfCCfzeP2VfUXmGfFPmfncvpH8YMdX1kE3agZt8KTWaZ9oSoKsBjhvgcprCTo8efCbCU8LAf8IwFOgDnJGQK2fdt\\\\\\\/os9rI09kYU9hpU4+0cAzO\\\\\\\/Di\\\\\\\/kG7Wa5E9B5qYeL9SAHbNHv9NTF3KP2z7CDnccCx8VvvgB4bp\\\\\\\/syQBtMmUGnv62gzcEdpRv39L6v8tAcfC1z0Fv1324IHkcfYSNfM2WNRVogDG50YBipjcw1hJSXCtMjKTAwxc7DEg0vR+XgoTWm+sy+fZd4N1VHs9jjeeR4SKiAOpy5uWtfhDNfmxe+yOkDJzrzVgw0MlPp8jlRP9RUBhXUATwDpO7UV0aYe1ihtpwiZa\\\\\\\/Fnw9YbYchkZczNQ69sLwywQpw72mYAk6zjFHU7TUnsie8EvWTKROgeSOkO56wOHut1keRasVQBSxSK19z4jiUy21LbWpsx\\\\\\\/x\\\\\\\/r0shBkuZ1WN3fNcXEP9dr\\\\\\\/3TRmaGPKhAJbJl8idWpVw64iowUGZvOzVEWiwkgDzL47qU4hrHuu6CqUHoGphZU++EkLPJlLJdxnldJZHJ1BNQX2fa5zooPMAjjTZC+xN+R5drpvTXiuqGYjIU4egwosbHvlkk3w1u4Wt\\\\\\\/dOW\\\\\\\/DIlZphRgU+pqkOkH7eRVGe0UeyxduIOPk1hJKAb\\\\\\\/Qy3K0YgBKtZ4Utq4dzXRCQmo7hQ2oDfV7pov15fVEeI0cAEGz7ZH1tGE22KxNFzhieXjQxgxedu2fDSMKYJxtdFsdqfK9Rn16eE879n+EbGGul4IqNWLcQxIGTNdSYYwr9EdmgYz2SGVdbFpUurvKQMp02gbWMihaiCBRtf+qPJgVZbR\\\\\\\/J0Qn+B7FlmoAIrq6tcNYLesj9F1v4nV7QxPueL\\\\\\\/JAKp\\\\\\\/MwCB\\\\\\\/sq6bLfwnaWzkSa5k\\\\\\\/d3izLae5IKAtxP8Tl5PXfjeUXJQCc0wSQ6VQOAiWCea85ZSLPF9etHExUbPNujEWzkjOwM+VgDi3gERpeGtSdev33TXV6jmEOZqNkFqPeFX1MQxQRcbObo6t11Ip1BCO\\\\\\\/yz167ajBCRl8a04OEw+bhWTV+mMmQ\\\\\\\/VD\\\\\\\/uxhma9H+2wcwOPS4OptwuOEIeobu20F7YgklaLJJo3iVQWaujqj\\\\\\\/s6vv01hMyFWPXwz7qTaRR3pBuUKPoFIXwCWpkRLUtIzT1fbJzjqq\\\\\\\/l7oEOUBNf+B3K4wUvRFo5hUjAajN6BaPDNwmiRlLnWjLGNzf0H93HAUN2JVeIpHuZsizDB23dRZHNQyUyra5zE578XqGyYZY5wEimyG8mMGWwAoidXSeCz0ZlDzNNstPO2OsTfGz6oUcPAjsyLM4QWcE3\\\\\\\/h1wm3M\\\\\\\/aRaCSg9pPu3xNARBK7mFWiLPejgDXccOGYUHzW3IDG\\\\\\\/h0FZdZfDNXOokvjWHlym4ppFlgKWNL7FZ3IkuN6tdb7070euzMcadMDCgYMv8OGrcImWbTM8TJFNcwyrGMxys0ih81qMosfyQOAaxjeENosdZRpFONAX6hVKPMIzdoLf3nnngWkrm6AepuUEXrD6JslCDamUxg1HOA+BJZS9y++6396bmrf6WS0uArlP0lJqbd4B4n1wYwblNaVhxbVvzjhXXGSlqOJ83LTVPJw3t53icBtLT\\\\\\\/rs8waFDwzec3\\\\\\\/xTcUVseInDCRhA5AIL6s5NHhu8gEhHGMiH\\\\\\\/FAVtmZUeOnrGXhLalWWxAB9alFccDdO3js7VciXppnGGxqnCWM4LK2p7aNFWbI0Tl3P42YFtNkpwk5yp2\\\\\\\/9Y86mSTOesS2XhVxVRFWSrCGWAfR+Dq8+pVCOG1VIUaSAAbNFPscUl8Qjlas+bOHEolseLmRlDtwKLGQa0CMzzy1AFDqIcCREOUbav6imuzBWUmV\\\\\\\/4aBLsO7FCX+KedHg8s59WWD1ZSmFdusHwY+mOxjOCwXYY0fIpgtI0CgK8\\\\\\\/FVsAfl36+U5B77Vxx5uiB36cbUyh+V2CG+dUGCo4rVerVVbtWC\\\\\\\/cCYm7ORwpkEExN9MkXM48PxzIsALNr4HfmFUQ4X1wbW9YtK2CTVMdqs3Q9kCE\\\\\\\/cgLfa5kijAusccMOANV12elaF7\\\\\\\/3TRd7IzGEwlO3qPDP1A5QxE9i+whdpjXSfEdek5zv4P6Vr1+8PaiHFgXKtqsXhmuu+A\\\\\\\/rkGIMlymnEuIC311qrUxfkGRAvv+hOlcHavP+5vZi24fOa6qKnbwUS\\\\\\\/TEton+ueFT56RaW3eFFYPM+OI5buAtNCd7hEQSwqc7VfSp4zX84muiQssp\\\\\\\/++EQabsVmNihVm1TSCT927shwN\\\\\\\/CPdTkkmsQppMJQSFzjavFQrMXt+Xly+YbYHXX4tfzFQcwPcABnqfLORLk8y\\\\\\\/zvheqAv5i7hXc3S3+\\\\\\\/ZQXHIwjDQYcn1iQ8RyxLeiO8GGVrIRW\\\\\\\/lMHo61aHq4NudueBgfHX4h3VN5l9GWqHBEERL+fhjhfRSHNWMiDXsiT4eqUUf965rnKzdLML9Rdv4K60+ZBTW+VZA7qbF5pC1JT1FJqXOkRGvWL2e79f9RovAfs5kbqGmhVc+Bl9Qcr1n7ZgvBURadyf+U1Dy1Akt3mn8ybkEVMJUJ33Cs9XoBuFQNWSS5zpSzQOu6pigcdNM5Jyw7cnf8dxeD8+rPMOcoozuCrZgq0K1YbybbKDkL4TPQd6KIKML7oB+rTIVGAlT9rsA6RhWewiU6bTYQ7ZJt56PoGuupOLZKK+YvGrEysiziWnR84QpNGPM\\\\\\\/8T8BoGVui0z7d0LwYf9iYVlZ3lrr3HMzTQJQKO2C0DQF2u+PXYXdi\\\\\\\/bev3AvpQOXSDSmDV9fHZE+JtgtDKbFhtzJuaOKUSFn14J2pFpvCP++DKvAabzwzaONFaFthklBl7hBSPQkDrrFy09Rk0paYDHtl7aj6mN5VoCAWq0rRoL5232kjiax8oLdmMJhRVHnUXJ87J6YEA1O+qLmCfeCusj5qwnzwft86+0KOF\\\\\\\/7RYnkozY5Dg6zZPVIQ3Qy7uqX1DCC2QkDvENW2xykeLnxi4hxO8Um\\\\\\\/eT87SfU8CqjKO6C9XBd3bypBGshZOzmqVMqq3E5iOqk\\\\\\\/9CMTWu1aVf2YzK3qtV75CYZyzKquF3kCkFTZMzqDoNe+6RYa4cY\\\\\\\/BcQCzN8X9YjQD3PDE24EtlI3MaRJybtNi\\\\\\\/\\\\\\\/\\\\\\\/sIa\\\\\\\/mcmd\\\\\\\/jUIYqSwTLt3zEAocXkvj8sYXIyN\\\\\\\/b\\\\\\\/peb\\\\\\\/0nvXncRSzz+7H\\\\\\\/cOayP0\\\\\\\/PoyCe44SYAj4s8vpu5KaUcuD4xaPtkSyrC3j33lnop2GlkeTwxKkCYjjLy0ooQzbLDy+5QRQInNo5WadOgKJ4CUIYbGTBcsRhvwejBoSjPHkY3Gmj8YiPnqDcC11C1+KrGICUq9WE5JY0bxHSeEmV1Jg+fd55dPgfIfsMujf\\\\\\\/RfxwlVEImjSELxbu5Pcy3R6uTHUXPdoVV10Q0o45\\\\\\\/BNKOTsr3ScOzuIBqioB33IaSebm7fOCoD6wirwFlIIa+fBFjZ8023e6J5F4HOztuBaBjLowOOdoH+qz3gCATRy6vGtgN7f\\\\\\\/uZeZQ2Xx\\\\\\\/hNDS9GeqWyVGo3qUo8RwyTCuW+r2TknMo8PFSmtx6pp5FBIl7IgBmiiGLEcNd+Ehh76OQ2mJT2r7rh2j+swxE0FZ\\\\\\\/WoM4esBYLebSBK7rRZsvoTzd9Wp7STJa1bps3WTkC\\\\\\\/F+c1WBNafg916igfnW5AwEmxEBeZZ\\\\\\\/Ug9sBJx6sICHYLEqQbA5XA8eyNdTubfr6gv+QPbeHF8Ysj0xcvwjNakUWpqSLVvVMpXDYKgN4Uw2iiVkSP83zfzEuviaBieF\\\\\\\/w8bqB6k6je6aTSHG9f1oFjM2+7n\\\\\\\/EejAONX\\\\\\\/Dhsnop\\\\\\\/Uc+\\\\\\\/k9zNJCqImJfgcZRmCzzSoEkkj2xgVY5\\\\\\\/DRKjRqucC2LG0bkYPrkThnmlxFDqLIAOCs6Zf+\\\\\\\/d2m2jbpCr0xmXnpoKF\\\\\\\/z7Rm+L9RaxAtBl\\\\\\\/R05mpLGknExjEdALXunSt3+9gdT58Bkp2hrdqpc3QWGpGxZMJjII1F7ZmDei9O228IyR91cM3v7Vlu72QfxVSDPyu3g2a\\\\\\\/9mF+BMnt42iIo9a4BL1LYjCYn4keRZgE+zZ7AKqLTyQJEtndDGRNnHD8rTop\\\\\\\/7MbFm\\\\\\\/H0v01qNbqQb1iN8bBnNfAHiwBXG4hyHpKEUPhticjni+9aWp\\\\\\\/juF4rt8flonnaGDEvVRPiP\\\\\\\/mzls+Gvbje7Jka81BP0jPt\\\\\\\/wBAq08DwplQFYaEAyW38sKYHM\\\\\\\/V\\\\\\\/4adrNQOAv+WFDl33D0L8ehF7Kf8UMt\\\\\\\/D1t\\\\\\\/XGNQxFsntDZX8gJThKShCJd323WzFF5bgSJM8wG729MIHRHpOeBwUxEYfa8EK81wJda5MX6w0rGnSoefdy1mJXtx5HZ7W69IhJZ8dB07zAqUWHVr1\\\\\\\/oeC45+cHFPI4nfg\\\\\\\/L4atNOqEXj89QczdssgLeadRfECkJNDujHa1uQQQcWxkcl5chOd3iSBcAouOmfLB730x9TT1qKbpP2+kmyszeoyenlaUBcstb+06+PcVmvZClPdcz6ZfQoxhcXitw\\\\\\\/8UoED0DIMRJKMdy7ceMnVL14BkFTZDswISO58XsZlNmALaTpD6CVjNx6Q2PUkha37b97jCvqAIz0piQp12WPD0gBHxzcJgDaGBGV7sQWt2qR5+ZOKZpIH65vF9fevuv6cdonu4wyamjtZnDWC2Q38wV36kZ56SrNKWkKVj4LnSzmHWURspX6xlwQN+yKB\\\\\\\/LiFAN+Ic9Z\\\\\\\/xdpPhbh6w4Y1z1jxv+7EcU7lyZUSiBrkEuBZE7RxOcaIjZ3qMsSUm8FMeRF+MjnFDeejOy6GGPUlQvXwCTg2dv+X70LZ4xrSd+jmd1LdncJdrCkfOvRgmMSVd618\\\\\\\/6Xd5CWhQybAKZYsHb7NgrUip8+XTK81v48lsZHjg9C0OzwChiGdibXiKu\\\\\\\/020PZWBog0NdXPlnYB6cHNSsqjhgnS\\\\\\\/oKXyCRzzmZKdMJuXBpYOOvjekl+kkueI8ULuxrb75OfHbkpB+owwSLd5C1Mgn8F0qptyps2jmo9+TAAciK9w27ir9Re+w3eClRfOWsc7TUi9CAqReDaFKtTPdmYTIL9B55Gkyuhk0CA5GSUslceg2AWadyhVwRhzXM1YEXvBpGh9rkieyZBgnRb2RtvCxQIjeLwmnoTcW6N1GVm1XU6CcecP655Eg1zAIuIsG0X2aLrKrEp8wL\\\\\\\/IZN0h4sImN6WpKCRmfxpH7UZ+a7KDVas3wfSm8CoQ6u+rz+IroBEv5SSgOUjMfRfwwg0CuuSygGvMhZS2GB91c8dKLNTXYTQwapMjw60NnunO+gNNOERst4Dt+Bc5Cp5oUV5yaLitZrdPXJgMdXSahbLNtq9g5s6IRSRNFDkmqYJKNv93H8khhIUYhW6K39WJupzbgZa+Wk8eq24kNmaWxhpQCeGnyYNWvP4xaMLXpq3tvZ5+peUtSAl0rMbLaO5O9ToDM2254XFqrGnnYyZuKOpnPiDOhweMz4gSr8Q+qRYSvWcxk8MjgZ4sdtxOiUfPqfKGJCJTXTg+3WydWpdr8s8s4qF4DcdwzwGl0GY4XKEG0xwGF+kBi6V8YWhQZNcg2igkTFg3hG7KBodK9HO5SZbGigftJTfYR15JoRp3L72Vl+SNlZkEbgAcbsDOHH73t9YTUTKdLGWQv2BCk3sYYsMEaaPaRVtxOT4Arj0UQ\\\\\\\/kZx+CAeiWnpaa3BwWEanLgyEqOLVf6RLnQN9\\\\\\\/YA\\\\\\\/F\\\\\\\/gtM7nVbYMxCizubgZ3KMUpml3jAskZWVs0QrmtKFSMAGl4RTRzbvkgK\\\\\\\/K8nekqaPAQ1hRPXFVt6eRVCsjAySYBAmFna+fSa6UgMa1A6pEq2DHASNKf71RFaHKliOL7AZ\\\\\\\/WqCMKoGDNH0J07+DzJzR6m1xnmAY5Te7NtfdAC2FktJ6rkn18JdQA32+gEj\\\\\\\/S+ClQATRMQllSn5m6B\\\\\\\/\\\\\\\/w83xhd3Q1USlb1GDgPCf7NTBcpANNY9g59hWIuZyZKn0P\\\\\\\/omlgX08PHGcn\\\\\\\/OlzCsYdWNmaCTCUVXTiOsLBab7pbzsT+NPWRR1Rr4g189unqx2vmKbdz4oJwFvbNjryeamQQQPF1wvd3fqsl02n3lo9J13CSnf7gat49nFgjbts5c3wjS0Z8cM+XCloKVFX2+qXek44TUF+c00XLRTBZZmvrKoMg9ZBsMWL07zIt3FHE2C6Vm7D1nH+ts6nsqGuVcyjNpzgb6PTzVk7x\\\\\\\/SdYRH7WnuZmKL3YbSk4ZRxyEaaejOSZa9zFiieuJkG8Y8juZXNnh1+VXbv436Qgs8QYZLah5vIs7jatqifA9hPn+cwkhW4dlQnphX2rGJ4gru8EsdUuci7j2J0OocODoiykpykXCFIO04WIgscXRT0UkBc49T2D4CSJvabxq56UZo1gUOZZ3pP1G\\\\\\\/gRYRukKV5JxGSW8S6bBqvy8lE7wS0mHx+IAK3HU6+QE8sAsGyVzNdgxLThyzqXQG\\\\\\\/WVu8Ar93j78iWuuUGr5bL8HCBQgZnZDMJeszJ48au0eDMFVfuds8Zs2\\\\\\\/xtXVkhqmY6Bs2UEOmnHpeIxkWYkOU4SaDvAaZoZor7MqFus2Bggf\\\\\\\/2CoYNMibAaD4Ub2SCbxors97yfgQ1K5\\\\\\\/u\\\\\\\/G4FdxPHJb4Axo\\\\\\\/kIOm2ExRHI8GEny1g\\\\\\\/a4HWVoNZ1+NK\\\\\\\/XC2QtBNNq3Oppno8n7C7v6BVl6h3urTTcqtmigR9JFmssVEzeWoiqp4RVLFOQao8oWTlODrH759nAURkNNMZnxXIC2lJ\\\\\\\/xoSGDh\\\\\\\/+UCvs8BrYWZerOsxh+RI22KI7tWq176pWw\\\\\\\/FTsosWPMOURuSSXyh2vtxphaV94XzqzK1ykKvmcNvvHAn1bCpOrze+LnloHcaV6boDU3uxRtsDkMj7boVU++JStZZn8AfwWLNNsoQCSuALu+VNpLKB6PH0HTOEo6kUqBEmfNAO3FY40csow+FTUKyrjWUHP\\\\\\\/qrOADNMEpa06Kq07YwMIKtOFMvSw3mG5XN0k3mPp4gaDTAsGCiXNBMz0pbpmbSv3vi7KflzY6Rxkw215ATTamFvBC60KrBvg9u42sQrnj7h6mZNcgFWjIrV1n0X17V14AurNwU2hk8IlCsiuAUwTRoiTeer9nDvgmoy02OXpW8k9bfgohT\\\\\\\/BXAZKdnxHeRcM50uHVVGQbUnKKDqVOMpD+aZ8cSCGzUrH0tKsiJPLSvBcE1EiADgjOcMNrxtfrD9lvDaTIddgDrX5XA57x4+rnCwYFi2K5r5I+Ws0iJDyQUikVn88RsA4viqk\\\\\\\/LAN5ZnGHeh9cnVF7IaQRleFYbeghdVDhm3LfQgVTEAvHz0S0VoI\\\\\\\/Xorxg8MtlKcj5LDz1XENhqLN9U69CRJeqRlZzmAtlLexJ5w5dz\\\\\\\/5BVAP4Dkd8SdGAzUz0jlA+0z1zltDYlEif6tSKnZLgTr1McsyHRVEhSNR3jyPHCb6IBNcy1by8EjQt0Jbrj6TeaWgCiY\\\\\\\/udRhbMxBLnO4BCvMNWBcEwzztdRlkKCr2BrtVuO2zmdyFx64vUqB1Jkyg3iQe2GoJirB8tlavS+7nziJMQmTdoF+T9C1VnqJlRV2nlaC7s\\\\\\\/Q29xX260erJPOOVMLDCNAuEkrT3QG1zgLJYnRptkquJkE7lOZXSunLOBYXjBVyEH0Z4Ygi9tEoGbedoQxJX2IWisCI9IFZLXtEvmmLXR8E7CgL1Yj9B2ttoxlppiXlqgHHfulzq2Zh3aKq0pCk7Fhv7H+o7Iy2fX+KYyLNVBjnCLux98\\\\\\\/uGsJwyYpxIIRJIPFFQ1u8eezjvyP8f32NJtAnTt7McTRecoc2J0ob9r7prGr1YHwxwRtI+hvFznZ1UygUEFqo15Moqv28d+3HuO5VSXUP8D0pZVLdpgOOsqo0aGfAYGH8r1zPNsNkcwUQyLMxnd+c0+n5uB7PiOZNdnP8l\\\\\\\/S6bY8WH5G3TJxzzoSR74UEEONBTjSvmK8syEGSTmVaCmj4vupYvy1QcpxE02jJSX2hGtSkYio1jQUrGxA0pAZOiur8EGHXof53Yv8805RDC4viKPhiiylqVxWJHpa96uv\\\\\\\/h8Ilj0uyA5OSLDKAOr0XGXB8KX8L0Zaqa0kUQMbg96NzTM0alQcARi9+lboc1uMo4DQCvOP4UWvlcM6kkIEnqJfjuNs+eFri\\\\\\\/BVjNPfEsrQj7IOX9XpatnzXuCLZGJBbO0qZFq1G93TbvnbK+VAZOlxHbGjteqzhxrXT1E6GfnII+sOH8SemCZns7xP7mUS+mStSZrDYY2erK1mmiHWc1fJWayiD464sz7yO2XEPHHnRGtb+htcH1ZBzFJG7TxtUAhJdsZuzbd4vzDi6kC7MeVxdEYnU1eqCIviPINfZBatpWS3m1rQ5tuEml41jF76DRR\\\\\\\/xTu7EXy8rYAL7mfI9VI4Mm\\\\\\\/V65iAXMcw58LqgUCJT8TD7ScstfYM8Pbf7DUT5udBQaVZyGr7gBT5o8mqKcCrJtdYlBtYFUEd+9pRyCTbYegzmCTczqs8YPASXXFjO4+ZIqORsIuFtXD+js8itJhZSyhrYsR+9k5CUyvgrh6FE9JUox\\\\\\\/dEzde1WODiCaWiU0lVxR5WvB3WCtHQJLUbJADtfcEXXbV7eY\\\\\\\/lseuUKpGmof3j9pLAxSVfc5IizyzvYGkc9KruZ++gLCy\\\\\\\/ohQ9ntEUYivQvJ60yLIryH1iY\\\\\\\/LN0VqfWDr\\\\\\\/eLtHI7N5K2sqOChhkQp79FcFRoyAjjflQglTYasYea30bZl\\\\\\\/fqMPW3x4X5AuQMcbaTwq1hkxURhLOic6DFzg7wdEces2n+2mR6MZfdpE5o9mbv2jpqQncBmyKi1zHa7LxbU52tX1Lrd6EEhz7Cu3iO9dEMuK1CytWQEfBUbtn6sWwmIlzy8kbRPuurUqUe60\\\\\\\/2vSY48zDf+S6Gynq1ANJtpF5iH16x2mSxDATXp3\\\\\\\/72JTUaiPsUXdF8JjlQRQoF+A3V2BThorUspMMtLODTMImeI136Eut\\\\\\\/cYstIwyaNrpHW2XCRxH1\\\\\\\/maJr2OlPJ0bZsShcRhzJidhK9FAqUftnw6BBS6nx5X2uLD8GmrmflHa0uyWCWysvlLkQtO8eii5z7Iv0iLljBo9phO7mmjtvxY6OI3zYXhN8aoF+ttqAr434YhHX6Tku2VSyYUuVtxuRiS53ayNO2i1yxKh6cjXFfmVEXpUlCiydjntZFhYjtaQ4afmsbDkK+NNGPvcRfzJiwArtAAwnvbNskrHJm8bqg7mID6nq0LeODynwJdN4eluBHT1fdzAHuo0MDIzRBmJeAxSmSw3IVwIGhAfem44UxfHbKTgTcKaqP3\\\\\\\/p7v6Hy\\\\\\\/FBG\\\\\\\/vUKOwNPOQVCkL4PFVpElpacw9pnF4rWLn1s1Q\\\\\\\/wVhcUa+fZpAsYxeF6J9jfcCHQ1DUXQfZ5C16afPAtV5acpTudG\\\\\\\/Kiq732WnueDz897tontgfYM7J0G\\\\\\\/3iBJ3KA16VE6Vsj7cd3X0kmqak+PkLqz+TEkGQXe+tkts8I8J7Oq0hAlMyEuhJQyd\\\\\\\/MNKU8Y3hU5tbajLym1hNDwnTsbDwscxd7lrwNhlk5LE8IIWwfYBNOhoB9yxkltDiixyyV5g5bUO\\\\\\\/69in\\\\\\\/LuvVagbvfKIoXAH4kRgqy9fNAOMKrkuJagGpqPErdYMuvYw\\\\\\\/KMcTbUAvqXi+DB02ItX6niSeR8CUIqhqBrEWUkRJRwapelfs8GrvL7dgecsOlwFaCyPfT+UnoA8gFOjGec5VWUcj+YCnUvSf+K1CGZ+sevujFqmRgkp3zd7YGhK3HlJGYpaHlBi2dCqNPqGqzDHkNamqQei\\\\\\\/HuuWxxsktc+5LO1od+JKJ9n8Pl2DJRp8lvatFgCBlHy1NRf9+83dHPTapgqbW4CIgUPyMnCIqHbv\\\\\\\/XH\\\\\\\/3etzVHaM\\\\\\\/gMNI\\\\\\\/guaFs89os874JaCw\\\\\\\/G0PfEdRnB9u4WEVKMSkntnvvSdP1RMq2q1ODc9ApfQLwj3ezB4YIe3AJKwNAiWj0qfDklM3p7ttZLupG8ZTCg3C2wQ4SH1J1QeJSELY5RO8utK6UBvuT7I6Wve4kiVyd\\\\\\\/mm0ZZoR1Mw2LK2rcwxbNNxoVC0\\\\\\\/TvPuKQoaPf8JLrnfNcBRtIlthnbKkY9jLTi0JwScFStnVrzY05jTCxxTIE0+g99LG2vz7QvOP\\\\\\\/AODWCh6pcfsyBklTD\\\\\\\/yl0YYWJcNIbe3zg6zuP1EN8tszjxzSWSrCY8x3G+VNOBR9Xmz\\\\\\\/iqcWKodI\\\\\\\/c1u2vaaenKuZmlHt6UTPrMaeViD+StmmqWO8Zp+WLdK8FmXtf1BefJsc\\\\\\\/NJXrUgdJfb99\\\\\\\/xKr8LhSPAl+GZcNw9p8UKKwZYUSuGphWAayJiNzGn0uKoGTjjEoTMFzQq8ih2UHWsIpMbI9vobCVo3JMnCx6uWm4l0jVPb3Krbz1AfAgHtH6KdD+Z+H48ZME0S\\\\\\\/5ET0LXjK867JJo4cyFI5rCF1BQD9C7xXLalfzresUHNyCjZfx9YVZs\\\\\\\/D03pq4Od8zU2ATO1WsvoQ8+zavlFAnHzGs9lbTEevIzK7vQzeWG3OnyoB1KmKhD3dVVfxrP\\\\\\\/p+p+UYGa1NBf3Er3+ZXMJSuzdp8d7taRAQ6J6N8+0MLOHdNn1tJuTBwvywUhnR\\\\\\\/dUxAsf37VDvOnZlbHbd5rJB7xUOCjjVR24mymQbxgmGOl9D3\\\\\\\/CfzrVhrtNObHEhl9cO3r43OuBMM8Bg+T8iYxyh3KlTIc+ix7xcKIMDco6FEgiXiutKDoxPyEqstgrbeGIcpboZkONqA+dUnBRMfUPBVQFcIC\\\\\\\/3VfbJzk90s6lJVgmgHgRzL6SWXLD0D9PiRJMZ7J7WotD5Pl4AFMhnRDPwDR7wA6wD\\\\\\\/1Z\\\\\\\/xzKJMWCwikYpsHtLOtxFEMEHKwwnJyrpGD\\\\\\\/pEIUyL1YFa0FR4JpojKg\\\\\\\/DN55UhjX3Kyd4uuy0Qk\\\\\\\/iV3NfgYxU5XEMpW8\\\\\\\/SdUSAuG2nLXcj1xl45I7tRrt7QUDjAawsLf78x\\\\\\\/ocCBVcHKrjfUvvKRShwkIfBoy5VYpvMVh8xQ4JY7l6tquMLYL7VZ5PRg\\\\\\\/q3RP25C2QqyPv40kRRPaEChQdzkAUVNzpCmRvNSkAC+EFaUnBeLAFF9SL6F\\\\\\\/imeQMcMpZy0CCgfEIQdr1VvptxRSGFKdiYbRfdFTBY60ZNhX4DdDSWxuBc3Wlo0w8773q2nXoqux08WflYjdINVTTWWggd6zHJq9oCf3X9POrKEJcYrBVkzcXAC0e4ZLGMWxW0b8aYLzWQ0d9mgfOc06AkxKpXLGHrMGgnnwaoygwcucqnW978n1cbtEbHWMXMtahhTe972zYS+I31Zmd83HHBCtg\\\\\\\/k05Ozzvk5x19WvSqWtAKBocp5cgpLdYDdyWtIL35uGWGTxoByP3CZOZGBfkvF6o\\\\\\\/6eNoHLN45F8ocvDGxOlzlFhc1tHt5G3+uTZsIsY0J8e8oo49QxHxJPPEd+8jfamv1EwVNbRy0t3zYwCyGFfao151dk1qE8KE8cHMJH4OhtcFGbu08Ydp4hSEEa\\\\\\\/iSENzyKBTpfKvDajGyq0WBLf4xmRtfrao3ebwNwCZTW6pKitbL9QpvKjfanlITg6exDJZYrl5iRWmhzguKVtYBLDPmhiuMa0shJ2QXicYSl6\\\\\\\/pTKH4biuAbl9RAjCESdNEJKdWwF4BZEUlxwPoBaIys6diV1RroE5nyg4KXQ2c5bKUlArSt2LlpEXhbZoDWzutoOy\\\\\\\/qX8L+Vr2C\\\\\\\/xIAqqeyth2ac49tYwHkbVp1cO1XLCjGcvelyffGvki\\\\\\\/p5\\\\\\\/37qJXJv5TF35250LfbxnQnaBk+Ihje6JfB33kl9uoJ1OsUzJMpEY9sd5q46fiwcEHJP+5m\\\\\\\/Ymsbxfzs9lcZXPtUU9vC7rZo\\\\\\\/7AG9Yas6TorEzUwxZBofZGk\\\\\\\/F\\\\\\\/Bd3gLiPlmuNK5xiBvZVye7NagKuSS6qInoW2h3Mhlsv8SyLsdVYHkL2dfahOrvlexStIE2n8hFccUfhelbEm\\\\\\\/mmCSBHar9BC2STGCFDvf0zDfA4NQGT\\\\\\\/AXv30EZ1LxfUfoJIUe1m7gHMCoNsnd1SajnV49y86FnrT7Yxa2Leg7wqFOQ5CZQ3KUlEApt+bOi4IdI3gMNsLJTZX3oS\\\\\\\/cd17T3+04VKbpLMcORbh6tZAL2ClbFVCTTGjMyaPYic65kuaaBf4+zBdp2C3YOxp7Yn27bRCOIxhqcqzBpK\\\\\\\/0ZxPv1YGwFd7zlzFFk57lXW0P0N8LMeKTm9mE9o9tnCKwIO8a6Th5Y1AfLjud2b4V2ovPBTie\\\\\\\/T3dMDHw1DojU39Z1ZfRsq8Q4jybLgHbBcPjUTBYsyXUzOw9Mlu4d5tlGgF0O5RP9Pimu8XxjMoiohNHn3CwMD28YVKvl8VXTUhN5VSs5BHHweSXvnVaxkypI4y4kmfgTQ9lbkIr6UBjr1xuQNQREsjd7cWybJYOTgs78iMsAvH\\\\\\\/\\\\\\\/jxHZjEhDXSiYri1C+srzm47ucal2Kb9IkvDqSq0tP6tGf86bOa58VVvhzJDn4kfoqmq\\\\\\\/FV+l3yKM3ePwS3w6X4Sl23NfEWxXEfiFo\\\\\\\/WxmhRDvoci6guWmogoxKFRBDvsKl+PShQlWF8smqkJKrDs6J0AkN6JfCIY90oRyqedlF6CBTud96o2SccIXuBUBqd+yCOfNrIMtYidLwzqerzWXS7wiXPSA4r2\\\\\\\/+s+j0okJjJtVYIirZR0rgpFpOtpJj7BKAz\\\\\\\/YrvO4jNzZyqBsOwJwgXeFtzfevwSY\\\\\\\/iAWmIfGNzo+CTWoWa80GXO1HzIPcHQizI02cp8IBkMXWTl2QHNR8zQkBg\\\\\\\/X26QPVskotXHEoH2XgQM0r1p1Ljdc3gWQ1Yjdpgkit02FYD7J4FRKhUYPDxXenjZPMHMfG0fG\\\\\\\/ZGLrU35MfFBRPtGvmGvwq1dVbubwerDKSulcxbLC5AHhlmIMJoO\\\\\\\/beTtNABUsEslBIm2DHV6K4IT\\\\\\\/edoh7F8nLQfu0PfViY59D62te2sPxqlkWkHcwOLyixu5rw7M3nIw1+lU0b9+J3ZCNKbxa14PO5lSDh25tGDXdWkfJprEdFmv9xsSziRUEjn1aSdD36ZN8tEn7ah3eF\\\\\\\/CVQNXiIXdKTHuMXbhbljmk4CsS37O1AzHM\\\\\\\/eJreKUphGkJJntjmDY2\\\\\\\/QUWO9MCqp7M863kZuko\\\\\\\/Qip+4ou2dITYSA7HM6MwJSf033KICd1Zt0SjCdomoV7bDc2HLNMgFX\\\\\\\/rKxdkJBC9NE2d0gUrAkt8efqakAaQVnolzQE5FmpaNw3MN2cmDJ8\\\\\\\/kvzFSHz7ZmpBjYA45v04Xuf2elgFBkaDqk8l7Wel8HTv\\\\\\\/EsYT87JwEya0YfQu0iM48D42U8mAMLbiAIZwqKLgBTbtMbD3PfMOtJ3QfHikt9bOJeVXbn\\\\\\\/SGq9sDkx1QwkNETnT5gxLPTdX++exf1UeGm5vZY8YKvglwKFwJOawreDfG5EF\\\\\\\/D3wOf20Iz53igC0C9tq6C2O14f\\\\\\\/lsLLaYORmHC0r7mN4ys5RktY\\\\\\\/TbogwcLrhxFrThPuEk6iEmCTEYF1h3+G7BIz9jF\\\\\\\/1p9qjHoPCozYpKsKmPAan2UdrSyy4V1+PUogj\\\\\\\/ZmBsiUlECAE54Isu6ugi\\\\\\\/C2fHBX00HCaBWOfA7o4O9kZyyfLqnlttcv9SKOrUNwWmbGWWGSY+MiyuC2UjhJPGRHm+NYiUIOmbzMIiHMMb601tGB2um6ZjJqiwoP5sXZUHKWXd8jicq48sEm72+SYm+v857KaaKS0iDccvDW3ro\\\\\\\/0XpHiysKPm+AaWXKuMVKazgM3pRmCnuJt+lT1xtO9ii1VCecSUgIalBnSCAJ9vGbfvBCkGZB\\\\\\\/nFFJE8NSTSEyPYx4QOvm7SD\\\\\\\/eY02kH4e3qdJ8T3y6RmMGecXT6G9X8eAf9yin63celgJzhCSFy1moG6Fa\\\\\\\/gM\\\\\\\/XPtrYWxgIM\\\\\\\/QfNW5kYM2ab2BID4CEfWRwP6kXujhS\\\\\\\/HsGRi2HQkyG1Iq4grvBBFcsq3viUKPrpRZdOEwdVxnSsPRNGnzOslXrGFEOPwG35FrPrnXZs56qbNdyIcCLieba4fZl03MPvNG\\\\\\\/fLXeblCb9HqjqC8tKGUEnYXxwMqMUgr+dQYV1\\\\\\\/cvtTazrFyqOatOk\\\\\\\/gaOEQUOVku8IjectnBu6yAhEais7+TGu6MN2kwGSHCzNsw1U\\\\\\\/2yls1Mj8gAw+pg3vJcoBmbyauCEA2AyvThhi78XyYOiuMf3S\\\\\\\/E4yCPWbPrdCqJ8BQT6uSLIfzHcWJ+55Bp8KwJqO1k62ua9k4rq2z4jZKWpOCQ8Y2KHbcSsgrz5qrd1knb9IauqUy8uXVhHv0UUDOEDWPfOpvvRywctWdBBojnTcRsbrE0jpLHGFxnVkgdrHpxmUlDi56Vk1fq5Do1boZN8tYu5HrXZwYNhReb24X2GaCYkf\\\\\\\/IsnikAnuOQjV8aSbuGNcktXrPw419hbqmwFYDXp1p8nP5kbX5xdGLp6jwrIZGHicBAwp10vSpZUuJcB7aWcLk5IGmb4FsiJgY0T2HCicYvcuKlMRAvYf+isv0iNQjxma9jNS641xkZfe5bUF5pMfoDfbHP4OUtCo0KAX77ncXbQTfr6dRHurMw\\\\\\\/32nc4aw1H8rbyryLRVtAB3EYFJo1zpBa0Zngi2G6dq2plgojyesxP7jOeinNSdFlF5BL8ZAnSTGX0ATMHHBWtTcVnELcofGzUvKl0qNnxKuETrwQc0aBXBkTsTLk3g3FCPBumEgaTL8GDD7sC3jMMvvrvq6hTrxUlTxoOuddLgR+jyc1UIAInCfsdh6foXJ9QvOXrWuJbhUdxlAB+N7QtvSt2UTo69UUwnAzLyguUgeRVQiVzDyrYkjmazD3ZnlNQnTiqWrdXQnT1AFGSzskMPYnq5\\\\\\\/3ftmVwwnBRgyvWmcRcfq0As77itxwALy2Yg011PBkfuaW+2TrdJ77eUCQeJtgOBVxQK4aOENcrkECpoCIOy6TPRQU5THo5rWfJbTe1PKSmTrVG8YFtU7DAAjmlNELEofU\\\\\\\/vDK6zY3sEvYkKZEqYL\\\\\\\/nQkfEAA1YRN\\\\\\\/PqNcDZMt75yWTI8uiX\\\\\\\/Wt15ALRgvPYl26Jcxt89498ql3wMIM8VRLyASaHpJ8+4k8qGb3rblIeGSA0z7g+YB3lG0Q3mznJvCeySopp+Dd5lEwZsT467RBtJpjrQsRPPfCnThOvSpXhUFzi+PnRRMB87FpR7tNm5bvO0OltXG9oLvN3LDJw3BIUOrhjsmxBilaPwdXyJOK+afNkXITV\\\\\\\/j+FrxkCkCSvDPgR+6jM1VtIotMjBZfHrZ4w1DobaZTFq\\\\\\\/OkbUXGY2VKT8cMwoRaVg3GAeKyQf1FQq6eRrIiqOYCOylwswoarg6m0Xe0idbKj1CvVPqKYTKByyVSPvoXcY1HLGig5nhQ+pHeeiE\\\\\\\/5Ry+3mHhiQWBwDzLfbh2GIXvvg2iB8Wohxefa6uuTsDcXbSizeGvahJzx+ubqCC9JkeEyqjxpr45qg8hiYlQa50AnajXB2vtWFvIpGtrppVM5CjpPrfhtuY9FGXzg0xej8aeT5awtfmNsVQbzatdWM0So6OOV1PvXiuB67jsbpOkpsQF8GxVOUI+fQDauNk63ZBqgE4KrG7zZXCcFUFfoQJeGOOTQY9OGon85zpeJPwRGhowD4RDO\\\\\\\/TbKNtyECXu87Bg9mU3v2PkNRHG9CfqZojtYDYxCsK8k3fvQB79WwKJgnlxCgaVtPx6uN558BF0qHMIgn\\\\\\\/c+S22XXmoQ4VnkSqrFh\\\\\\\/alNvvSmVB4CTnH\\\\\\\/tgS7r7blW1YvNZkY1eZlIDOkNHqv8ZlpDB+jsuW0qCYpBEX+xygo5DH9qajw4WlfMl1ethG6QxdaqzcuQiKndpixlhpOiRLaVqe4IufzJWH8JEDiDoxmiIrm4ND3fieQg\\\\\\\/l5To6NU3bpIqCuRuum1mc1USmWTTmbVevVuZTyg5ZYaVf8hP1or+IG5Kowk1ABTwYpLC\\\\\\\/X5mMknrx6+uA+mj8HwxA+OB6IrhrEoDcfA4BpWUGSdgpaD0IRTaWym6sGZCMugLQY22cbJhxPGrRSoqzuIv5BaXLZLehs0V0YswDN5CB1DX4XztZe8DGIk64SXkcCjfDfq94rG5nBhUR4lu7OB\\\\\\\/8o4wjLfKjwNIizHYdvXK56Q2a7Emuh48bM0Qp0c\\\\\\\/J0E2NftfHcZGerAlSOEBdNhSJDCXp1UlnTS\\\\\\\/iBRhNRIg4j+6gpPD0snrrwH\\\\\\\/8u46mUy6sOo1lr+KpFU06oReHmPM0yg7GBqcwi+gELfqfjm+aN28vIcLpQ0wWFJwsXx4QtdZZyMSlEiJ4fx7HrJQGoeWfeEyOw77V\\\\\\\/Pw61UlV\\\\\\\/RaRWAOIHTsadsRUwLCWvBIh48Y8iKPMbl5dHup3f5V1h7wIXwZnc0ImS\\\\\\\/afDAJR5zPaYtU5Eohqh0vE2Co9zEZABVvBLayZJfyvfz5p1\\\\\\\/9vE4a2OQppRoGAg4cOt4FR\\\\\\\/DEO3nKgP1ULBtT9yUICp3+FZOsEtjoE41Dt8rHKjqCdiENt5\\\\\\\/ocKaL0AKfYcsQYaceMCjQQhdcb7zYcopR+zf\\\\\\\/iptNMFl3F15BuoXgWFyvsd5ErGSTe3S69RWBeAFov+PUCYP8Qy9ImoZZ73EtxYA8J87aw41n82yn7zGJe\\\\\\\/A5nyJ\\\\\\\/ikguAlYDbdF20MViw+HQyyGBKysPGXt7F5O66WnUSJ+E+u5GShf4J1a6\\\\\\\/84NwK0P0TZf4FeGmw+v9ORYX5f\\\\\\\/ZiaUisaF\\\\\\\/vCna1ePlh3d9vpmPSj73wbzEHnMIvpYinuk+8+\\\\\\\/\\\\\\\/EsArux6g+X0LPsU\\\\\\\/\\\\\\\/dQoanRsSqfuaMyKcJNRo0lsYJBpVqigL6vZwNk0peO0SxutTybyMf0V+BYPpbt9IXFD0H5h6xLBwS0\\\\\\\/efEEgNLnD7e623y\\\\\\\/p87hU4Mm0mNjY3+k19pQKKGLORXYzsJV08HohBsOALKsfP3Q87FNuBCCYqvpke5dIeID\\\\\\\/LMXsocoPB98nkrqfDFMwDLoFc4Bs042jXZTPm505KfIjt\\\\\\\/OreuuS737c1lsrAA47TUrYSwTO4Dd6BEoVoP4gkscwjgu+Hs5BEASWU2xBFQr9Vj9qo261pfoM+4uxgYFFNdehUlEfCf0h\\\\\\\/EaXG8F3sx3cz516btHfXBLmT87u3vaEPlzeq05ots1hLEFQsFWlO0wBcLF29iQDOiNYYWf8CW3Z3kQ8d\\\\\\\/SKlgfFLt3pu8KFjO1FmywLHFR0rL\\\\\\\/CbkX9JL57U2a0MPESrt9hw8dozFIWYVcs6jSj9NZxErJlRui3G+ufPwRhKjyu\\\\\\\/jdzf8gsTgcajTYThRZUnCq1K64E39Z4lOfUHJHgH3nvXUfhpOmCV4IlMLlFeLlYkLZVvhhmVU+59nDH4LSaF6nWQnHQTq\\\\\\\/YyltOERkxbTgXdZR2P3VUuJzUfuCckPjUBD++XK\\\\\\\/3s478N3Wbu4Ez2WYx+iBuqRr3v8BlOyIMNM8CLGaMt0\\\\\\\/\\\\\\\/IlG2hF6F81Ed3+T2Rh9HlA8eTwH8MFkNfCZ8mI+uHyKCo\\\\\\\/oJT6OKb5VMnhU1I7le+XmxlVZKfLCFax2DPAsBRgKLsWY2RF7cr6R6RpUyVHcV9OmYxffWNTr\\\\\\\/0IQg1L714TDhm4NDuPRztUwZ28cP\\\\\\\/nDX84DTxbjrWghKN9YedaXULzLsVuvcTGwi39wn39eVi1C6zd\\\\\\\/VImuw8MS5kWqPCskAMQaJX1WoZoAicROqBFxozKl63w3giTmyNAgb05MUnQV1OEGaXVLjN4XS068IWwhMWmm3LXxObUOzwV4XfG2rqVOu6TLxagCrrJQX5uM1yCI01ReycFQ7pGredoBnzYnD128WW50FKWk+hLSnggg2jkMNibTfelc0nGyMuAwWqVlZwAAyPy3vFdbim3UvSvuL1KObvo4AGIox6cBoI2ZaTrce\\\\\\\/Vb5q\\\\\\\/JBKVx2nuk1NM9zfssxUXrJZWigzzEEi9cqSwKNCXWC+PBgMkm\\\\\\\/N4T7uXy2b83c\\\\\\\/2kAozDrcKRwV72kCQuTQlTVXimJnaLoeWCZ4BPORxXSVHWu9xMHN0t0jBvM4Y4Xc4DuaYudmXLyKS+msHaG2T4nex5KG9Oaj2k+osG\\\\\\\/9c9WtEHElUJmwgCJ0rlZMq8B\\\\\\\/\\\\\\\/5gMxGEgPzIUJ3iYpCcGywKNdiTCMTMf7zOJugxR8bE3uzQVOlwNbWre5gjOh29xcC+V0guA8EMbOO0ZPhSj5NXiHu1KP7OMWQEebllx\\\\\\\/Zobkwa9ApbREaT2RQfD3BOLkjI0SdGMYc7P2NUQuNaXVEf48aQ\\\\\\\/PCzUBk2qCcJHLlZ34YPv0YLVUslKbEEVKOt0D\\\\\\\/tyo\\\\\\\/91Qnmh9rgsDQD46qDaaF8beoEuEhNviHWveen2XuRrhoyUhTn7mYyDIhzsDi1zDvpPT5HDY5i0+HKir\\\\\\\/z0t0HVhjnlCpMzTXwRpnnjObmeR\\\\\\\/1Gve1hWTOBU3W2iK10LbONMfRkDtemajQ7TGe3z1PYBHvqsArk1fLGnYjPEFSjMKK1QVz1jxJgokEzTww2o\\\\\\\/rdEFKSpgSU=\\\u0022,\\\u0022iv\\\u0022:\\\u0022jUU5a7x1KOviGeJoWMYzeg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022be43b77eddf10231e6d5e1289d953585b4d1965436c1baaac3d2982994ebc751\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022swV2Loj01e\\\\\\\/0I3p9XjNvHkkxFckIBftW1rGPy74hlP2kW4m21hLBfysKhldv4xIV22RI54jPEOTF8YrFSpjvm1dGIoHsif80A5XGsMW6Cn\\\\\\\/2rwChmy5Gafs91o7jC+3QSwRBSoUhmyOmk2stKralcrjb5sttR0hXBEO8UMXu7KUn0WD814wWf5MciC7\\\\\\\/VzAoUEYVtqjG5m3qnyKaUiRHBrkU8JxLIcnfCRJQxzJKEHRaL1LjKVllvlmULRH5UHkayXHQ3qmjVw\\\\\\\/zuNxHHtXDQfjGfW3PPz+DgFFH3LQOstf93ONNgzMra+vMj4rJufgRcm4NyBFi29ud\\\\\\\/ywmwAc\\\\\\\/3hwyoikGdEOhFzRRe\\\\\\\/i7m6AXZcRu+RvZXLbbidUorxj6ulbuUgPXCkLX4Ezm0uK\\\\\\\/3Xd9KzedI+6fKdShC5cyxElryhfsdJsLjBRwS++vxQgzwaxrQRdwZD8bYchUbWmaztPug0VNUThoOTubmy9Wv6RUe5W7lWR3Hpi8AYEYXuGA4fGYrGdNgCRApJ5XOC7pVLBt67yWEl1rCY5O\\\\\\\/Yc81w2B0IYh3pguhTf0OIZEnEs3mIr+Y9RslAjMfqBZWCFVlsVd+DM9IvpVcRTZwY4r6PjzKOPQEeTylixGjll1SFwt4wbVjoDphhQj\\\\\\\/IAg9aZNVWyhF+hF4xA3gX+D9X+xQ4qkiajkCd1tBF758m5roIo6Ngcpk+PDXZJKPMD6UQOU1i2CFvMpF0YkIaIhbrETaztp2xsFAHQMWoSvoIfhxBCve85\\\\\\\/S8FmXrRxgycjL92nJle+Bgcz1igXJN1wEAtj2zvFqVf7s\\\\\\\/tsbmF+QwzikcGhPFvyHp8N2nV+ms6lpMQkhfpec4xs8MN\\\\\\\/moBFVTKCuAERi45avKq9bT7vARRjEfh2hbkeG815gF+iNc7alITxZxFfLia5ZtqLfSN63NbQFEAaioId9\\\\\\\/OQn6p0E\\\\\\\/tc7LsLxdAH9Q\\\\\\\/M5Sph1tjyNx4dnVJmz\\\\\\\/DH1QpWCDJb1GoS8Kff+PhiHwiV7OI\\\\\\\/oOwiZ7LPrq5jGUbW78Uk0l2Bv2yTF7C8E2qOt2UZomfDBzESsNVcC0KUpN7++MjdsreKiXx+B7MiS7q1QV\\\\\\\/qHES0ejlVQVHhEwiBJGFNh+ORcKBPvY0FovPHp7HtTiKvPJIAKP4xJCy8gGoTmmtemhmQSBuwpS0i7G2\\\\\\\/o8sOI\\\\\\\/hCDKU\\\\\\\/2pSt48NRYTepGgCDKI4bqIVS4q1T8wMftTobxLI8j2VWmhVIWwaAitAY1tNiPqvJC5RWzNz8YScjvHpFrfH27Kbx0dtVktsSm7EnhckPf0NWrqNlPvsDWd1\\\\\\\/wCnxKoKryi0W+nHNPHpdOVfgpNznvgeZ\\\\\\\/nomwgWYRdAlCUIcRpvKYuWRTRhNkWL1iYen87OIe5OXsq0V44Nqxz0ymt80jYKpqL8uFVchk37FAFkF4UVvl16p8FfN6D4kgT+JIorYhIPInV0N60atiwRPiZJPYnOGadWI5fyAKRve19mcbh8cBPuZerb9uOqgX2zy0qE1GIu7Dr9B62aFby\\\\\\\/qyUqX+jhDSnF2MDy9voqp\\\\\\\/0q4zF2iM0kvu0Ai4T\\\\\\\/YHM6GQyP0NZ9BZ2ULjseMw4IwetASbR9ealoMPeKC\\\\\\\/8YyduVFZlp9ArjoYscOIjAts9j9LTSqBIAxH9gRkhUvt6\\\\\\\/KzhfmDfSvRX9j5lzQxMVFC1Tdgd\\\\\\\/RQE3v6ws3PA0SY7C8rPV8FBua9mCRUV\\\\\\\/nsO2EEaLcVz3Sd\\\\\\\/ypzZGgTm1CIc\\\\\\\/ymPhZCROYRjTSuafC8tsspj31JWRCLHQQhhpQAaXYwhChBSpJQKOtOwAnaVy7Qz28NGWEIokPT4JfqyZzh2SI4oe7pn+b\\\\\\\/t6+\\\\\\\/e+xs\\\\\\\/JiFejKtEGiKk45d9i+\\\\\\\/Th6Ddk67gzZg3D5SZUlBCJH29MziMd0BZ\\\\\\\/Bctg03fhum8sym2FFQlA6r5bWeauN1cN7SN0bI4onx4x8hQHRssapZYv5IijDBg\\\\\\\/QlR8bMunSHmxAhyIjCtDaxJ5179YBaMSmOCzRB+5EDjzcGjzd+UJjHl\\\\\\\/nvZDh2OyWuRzhYO10fiXowd0WX\\\\\\\/E09TgEAxU3NbI6TgOp+kO+akDLbxD8e9OUs3V0hQbD62nGuoLPYrlc+3Af5AkKfaw1nnDCspJRNEIBiybIDJ\\\\\\\/myw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022owNrKkMabFrZHy4pkOE3Gw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fbba6dccbb19d36b72d9ec08609401e674b08982a7d35c966788368f88e44077\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460359 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1313" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BhbA3eNdCEp5vLNLHbNwgsckvKnNfC8EgavHmuUsTVM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1313" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cuptDCwb3ReH_v2ZU0vAneJiR-9E1DzMl09R1BNqHCc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/5/1305" title="Characterization of Chicken MMP13 Expression and Genetic Effect on Egg Production Traits of Its Promoter Polymorphisms" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/5/1327" title="The SEB-1 Transcription Factor Binds to the STRE Motif in Neurospora crassa and Regulates a Variety of Cellular Processes Including the Stress Response and Reserve Carbohydrate Metabolism" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460359" id="node460359" data-pisa="ggg;6/5/1313" data-pisa-master="ggg;g3.116.027524" data-apath="/ggg/6/5/1313.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Selection in Multi-environment Crop Trials</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Helena Oakey</span>, <span class="highwire-citation-author" data-delta="1">Brian Cullis</span>, <span class="highwire-citation-author" data-delta="2">Robin Thompson</span>, <span class="highwire-citation-author" data-delta="3">Jordi Comadran</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-1808-8130" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Claire Halpin</span> and <span class="highwire-citation-author" data-delta="5">Robbie Waugh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1313-1326;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027524 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460359" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Helena Oakey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Plant Sciences, University of Dundee at the James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helena%2BOakey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oakey%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelena%2BOakey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Brian Cullis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>National Institute for Applied Statistics Research Australia, University of Wollongong, NSW, 2522, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BCullis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cullis%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BCullis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Robin Thompson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rothamsted Research, Harpenden, Hertfordshire AL5 3JQ, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robin%2BThompson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thompson%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobin%2BThompson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jordi Comadran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Cell and Molecular Sciences, The James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jordi%2BComadran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Comadran%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJordi%2BComadran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Claire Halpin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Plant Sciences, University of Dundee at the James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BHalpin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Halpin%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClaire%2BHalpin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1808-8130" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Claire Halpin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:c.halpin@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">c.halpin@dundee.ac.uk</a>
<a href="mailto:r.waugh@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">r.waugh@dundee.ac.uk</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Robbie Waugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Plant Sciences, University of Dundee at the James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Cell and Molecular Sciences, The James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robbie%2BWaugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waugh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobbie%2BWaugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:c.halpin@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">c.halpin@dundee.ac.uk</a>
<a href="mailto:r.waugh@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">r.waugh@dundee.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/5/1313" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460359" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460359/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1313.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460359" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460359/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1313.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460359" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460359/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1313.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460359" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460359/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/5/1313.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460359" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460359/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Genomic selection in crop breeding introduces modeling challenges not found in animal studies. These include the need to accommodate replicate plants for each line, consider spatial variation in field trials, address line by environment interactions, and capture nonadditive effects. Here, we propose a flexible single-stage genomic selection approach that resolves these issues. Our linear mixed model incorporates spatial variation through environment-specific terms, and also randomization-based design terms. It considers marker, and marker by environment interactions using ridge regression best linear unbiased prediction to extend genomic selection to multiple environments. Since the approach uses the raw data from line replicates, the line genetic variation is partitioned into marker and nonmarker residual genetic variation (<em>i.e.</em>, additive and nonadditive effects). This results in a more precise estimate of marker genetic effects. Using barley height data from trials, in 2 different years, of up to 477 cultivars, we demonstrate that our new genomic selection model improves predictions compared to current models. Analyzing single trials revealed improvements in predictive ability of up to 5.7%. For the multiple environment trial (MET) model, combining both year trials improved predictive ability up to 11.4% compared to a single environment analysis. Benefits were significant even when fewer markers were used. Compared to a single-year standard model run with 3490 markers, our partitioned MET model achieved the same predictive ability using between 500 and 1000 markers depending on the trial. Our approach can be used to increase accuracy and confidence in the selection of the best lines for breeding and/or, to reduce costs by using fewer markers.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/multi-environment-trial" class="hw-term hw-article-keyword hw-article-keyword-multi-environment-trial" rel="nofollow">multi-environment trial</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/random-ridge-regression" class="hw-term hw-article-keyword hw-article-keyword-random-ridge-regression" rel="nofollow">random ridge regression</a></li><li class="kwd"><a href="/keyword/gebv" class="hw-term hw-article-keyword hw-article-keyword-gebv" rel="nofollow">GEBV</a></li><li class="kwd"><a href="/keyword/barley" class="hw-term hw-article-keyword hw-article-keyword-barley" rel="nofollow">barley</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-5">Whole genome prediction (WGP) uses genotypic information in the form of molecular genetic markers to predict individual phenotypic performance, and has utility in livestock and crop breeding. For a particular population, WGP associates a phenotypic value with each molecular marker allele, which is consequently known as a marker effect. The sum of the marker effects (which relate to the alleles present in an individual’s genotype) is a predictor of their phenotypic performance, and is known as a genomic estimated breeding value (GEBV). In ‘Genomic Selection’ (GS), the GEBV is used to select the best parents for breeding, or to predict the performance of progeny using only genotyping data without the need for phenotypic screening. In order to derive GEBVs for prediction, an initial ‘training population’ is phenotyped and genotyped to estimate marker effects. GEBVs can then be calculated for individuals descended from, or related to, the training population (a ‘validation population’) that have not been phenotyped, but for which genotypic information is available. This saves both time and the costs associated with phenotyping in a breeding program. Since GS uses all genetic markers to calculate the GEBV, it potentially captures all of the loci that influence a trait. This distinguishes GS from more traditional marker assisted selection (MAS), where a few diagnostic markers are used to follow the inheritance of specific loci influencing a trait, and marker assisted recurrent selection (MARS), where only a subset of significant markers are used to select for quantitative trait loci (QTL) in a given population. The potential of GS to accelerate crop improvement due to shorter generation times and the avoidance of phenotypic evaluation has been established (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jannink <em>et al.</em> 2010</a>), and shown to outperform MAS (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Heffner <em>et al.</em> 2010</a>) and MARS (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Massman <em>et al.</em> 2013</a>).</p><p id="p-6">Since 2001, when <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Meuwissen <em>et al.</em> (2001)</a> compared least square, ridge regression best linear unbiased prediction (RR-BLUP) and two Bayesian approaches (BayesA and BayesB) for GS in animal breeding, there has been an increase in the number of methods available (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2013</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Desta and Ortiz 2014</a>), and widespread uptake, particularly in dairy cattle breeding, where official GEBVs are published (<a href="http://www.interbull.org">http://www.interbull.org</a>). These GS methods, however, differ in their predictive ability (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2013</a>) and suitability for specific applications. Crop populations may require different GS methods to those of animals due to the potential presence of extensive linkage disequilibrium (LD), population substructure, and agronomic performance traits which are often influenced by many QTL of small effect (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wimmer <em>et al.</em> 2013</a>). In a simulated data set based on actual marker data from barley, RR-BLUP was found to be more accurate than BayesB (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Zhong <em>et al.</em> 2009</a>), and has been recommended for crop improvement applications (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Heslot <em>et al.</em> 2012</a>). <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Wimmer <em>et al.</em> (2013)</a> compared the performance of four commonly used methods; RR-BLUP and BayesB (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Meuwissen <em>et al.</em> 2001</a>), LASSO (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Tibshirani 1996</a>), and the elastic net (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Zou and Hastie 2005</a>). They also found good performance with RR-BLUP, and recommended its use in crops. RR-BLUP has the advantage over Bayesian approaches of being easily implemented and quick.</p><p id="p-7">Despite these advances, GS is only starting to be adopted in crop breeding. Some of the reluctance to adopt GS may be due to the additional modeling challenges of crop improvement scenarios. Suitable models need to accommodate data from replicate plants of the same line, the influence of spatial variation in the field, and potentially different ‘environments’ if the crop is trialed in several locations or multiple years. These variables introduce significant genotype by environment interactions (G × E) and new methods are needed that consider G × E effects, as well as nonadditive effects, and the crop-specific (in)breeding cycle (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jonas and De Koning 2013</a>).</p><p id="p-8">The reasons for considering spatial variation in crop breeding activities are obvious. Every trial (or environment) will have considerable sources of nongenetic variation such that even the position of a line in the field will impact its phenotypic response. Allowing for spatial variation through appropriate trial design and analysis will ensure that more accurate genetic effects are revealed (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gilmour <em>et al.</em> 1997</a>). In GS, this is also true; the accuracy of genomic prediction in RR-BLUP is improved after adjusting for spatial variation using moving-means as a covariate in the model (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lado <em>et al.</em> 2013</a>).</p><p id="p-9">Similarly, consideration should be paid to the fact that crop lines are often assessed in a multi environment trial (MET), <em>i.e.</em>, in different geographic locations, seasons, or years, in order to determine performance stability across environments (<em>i.e.</em>, G × E). In GS, G × E is an important component of genetic variability (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">2011</a>). A MET in a GS context is therefore an important extension as it allows the examination of marker by environment (M × E) interactions, and, in particular, the identification of markers whose effects are stable across environments (trials), as well as those that are environment-specific. As RR-BLUP involves fitting a linear mixed model, the incorporation of a MET extension is straightforward, and improvements in prediction when using two stage approaches to MET analysis have already been shown (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Burgueno <em>et al.</em> 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Guo <em>et al.</em> 2013</a>).</p><p id="p-10">Capturing nonadditive effects in genomic selection is more complicated because the genomic relationship matrix described by the markers and used in RR-BLUP captures not only additive genetic relationships at QTL but also LD and cosegregation information (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Habier <em>et al.</em> 2013</a>). A first step toward modeling nonadditive effects in GS is to include pedigree information that captures a polygenic effect. Pedigree information included in a BayesB model in animal GS marginally improved the accuracy of selection, and reduced bias, which is important when marker effect estimates are used over multiple generations (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Solberg <em>et al.</em> 2009</a>). A small improvement in crops has also been shown (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Crossa <em>et al.</em> 2010</a>). <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Burgueno <em>et al.</em> (2012)</a> explored the inclusion of pedigree information in RR-BLUP MET models, and found that it improved prediction accuracy for individual lines in some circumstances, but not others. However, compared to use of pedigree, inclusion of both additive and nonadditive marker-based, or realized genomic relationship matrices further improves prediction of breeding values (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Munoz <em>et al.</em> (2014)</a>.</p><p id="p-11">Most methods of crop GS use a two-stage analysis. First, data from individual replicated plants or plots are used to derive the line means. This allows software already developed for animal studies, which cannot handle replicates, to use the means for RR-BLUP in the second stage. However, a two-stage approach biases marker effects, and induces heterogeneous residual variances and residual correlations that are not completely eliminated by a weighted analysis (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2013</a>). A single-stage approach that uses individual plant or plot data, includes replication, and accounts for spatial variation and randomization-based terms (<em>e.g.</em>, blocking factors), would be preferable because it would not have the difficulties associated with a two-stage approach. Incorporating data from individual plant or plot replicates would have additional advantages for GS. Markers may not capture all the genetic variation contributing to the phenotype, particularly if the number of markers used for prediction is low. Including replicates allows the total genetic effect due to lines to be partitioned into the genetic effect due to markers, and a residual genetic effect not captured by markers, which will include nonadditive genetic effects. This approach is possible without the need for pedigree information (which is not always available), would be more encompassing than a polygenic effect, and would not require the calculation of additional matrices that can induce dependency between variance components. In addition, separating out the nonadditive genetic effects from the model residual variance should increase the accuracy and predictive ability of the additive genetic effects compared to two-stage analyses.</p><p id="p-12">In this paper, we propose such a single-stage approach to the analysis of multi-environment data by fitting a single linear mixed model that extends single trial RR-BLUP analysis. Marker and M × E interactions are incorporated as terms in the model, and form the basis for the analysis of METs. In this GS approach, the marker and M × E interaction terms are assumed to be random using RR-BLUP, and the approach extends RR-BLUP analysis by partitioning the term for genetic variation into marker and nonmarker (or residual genetic) variation using raw data from individual replicates, rather than line means. We first outline the proposed method that we refer to as <em>partitioned</em> RR-BLUP for MET, and then illustrate its use on a real data set with appropriate comparisons to nonpartitioned, or <em>standard</em>, models and single trial RR-BLUP analysis, both within a single-stage approach.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Description of motivating example</h3><p id="p-13">The new approach is illustrated with an example data set of the phenotypic trait ‘height’ in an association mapping population of cultivated barley. During two consecutive years (2010 and 2011; referred to also as trials), spring barley lines were grown in pots, in the field, within a polythene tunnel, with each pot containing one plant (from one line). In each trial, the pots were arranged in a spatial row-column design with five replicate blocks, where the replicate blocks correspond to biological replicates. In the 2010 trial, 648 lines were planted with pots arranged in 405 columns by eight rows, with each replicate block consisting of 81 columns by eight rows. In the 2011 trial, 856 lines were planted with pots arranged in 535 columns by eight rows, with each replicate block containing 107 columns (see <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Oakey <em>et al.</em> 2013</a> for further details). There were 639 lines common to both trial years. The lines were predominantly European elite cultivars of two-row spring barley. At full maturity, the height of each plant was measured in centimeters from the base of the plant to the top of the main stem. The software CycDesigN 4.0 (VSN International) was used to generate the design each trial year.</p><p id="p-14">A set of 7864 high-confidence, gene-based single nucleotide polymorphism (SNP) markers, incorporated into a single Illumina iSelect assay (Illumina Inc.), was used to genotype DNA extracted from 477 lines (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Comadran <em>et al.</em> 2012</a>), including 459 lines grown across both years, one line from 2010 only, and 17 lines from 2011 only. Nonpolymorphic SNPs, and SNPs with more than 20% missing values, were removed. For each marker, individuals were coded as 0 (homozygous minor allele), or 2 (homozygous major allele). The population consists of lines that are derived via single seed descent and should be homozygous; a heterozygous marker within a particular individual suggests that there is an error with the calling of the marker, thus these heterozygous markers were coded as missing. Missing values were imputed using the R package impute (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hastie <em>et al.</em> 2014</a>). Markers that were heterozygous and failed to converge to either 0 or 2 after several iterations were discarded, because this suggests the marker itself may not be appropriate for use. This resulted in a set of 4654 homozygous markers with a minimum minor allele frequency of &gt;5% and map positions available for analysis. The final set of 3490 markers used in the analysis was a subset of the 4654 markers, and had no two markers identical in terms of their qualitative coding across the lines. Pedigree information on the lines was unavailable. The phenotypic and genotypic data are available in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS2.zip">File S2</a>.</p></div><div id="sec-3" class="subsection"><h3>General form of the new model for whole genome prediction</h3><p id="p-15">A general form of the new model is now presented. This is a single linear mixed model that incorporates marker and M × E interactions as terms in the model with appropriate variance-covariance structures to allow for correlation between trials. In this new approach, the genetic variation is partitioned into marker and nonmarker (or residual) variation through the inclusion of all raw data in the model. In addition, spatial trends and design and randomized factors can be easily incorporated in the model. The new model is referred to as the <em>partitioned</em> RR-BLUP for MET. We use the term ‘trial’ to denote different environments, which for our example data set represents different years.</p><p id="p-16">Consider a data set consisting of <em>v</em> lines and <em>s</em> trials. The new mixed model for whole genome prediction can be developed as follows<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-2.gif"/></noscript></span></span>is the vector of response across each of the <em>s</em> trials,<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-3.gif"/></noscript></span></span> is the vector of response for trial <em>t</em> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-4.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-5.gif"/></noscript></span></span> is the number of observations (pots) in trial <em>t</em>, <strong>τ</strong> is a vector of fixed terms, consisting of an overall mean performance for each trial, as well as trial specific global or extraneous spatial terms, for example, linear row or linear column effects, and <em><strong>X</strong></em> is the associated design matrix, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-6.gif"/></noscript></span></span> is the vector of random line effects of the <em>v</em> lines in each of the <em>s</em> trials with design matrix <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-7.gif"/></noscript></span></span>, and has the general form,<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-8.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span><strong>M</strong> is the (<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-9.gif"/></noscript></span></span>) matrix of <em>v</em> lines by <em>p</em> SNP markers, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-10.gif"/></noscript></span></span> is the (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-11.gif"/></noscript></span></span>) identity matrix, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-12.gif"/></noscript></span></span>is the vector of <em>p</em> random SNP marker effects in each of <em>s</em> trials and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-13.gif"/></noscript></span></span> is the vector of <em>v</em> random residual genetic effects in each of <em>s</em> trials, and represents line variation (and therefore genetic variation) that has not been accounted for by the markers; ⊗ is the kronecker product.</p><p id="p-17">Let <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-14.gif"/></noscript></span></span> take a general form<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-15.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>Where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-16.gif"/></noscript></span></span> is a (<span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-17.gif"/></noscript></span></span>) matrix, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-18.gif"/></noscript></span></span> is a vector with var(<span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-19.gif"/></noscript></span></span>) = <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-20.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-21.gif"/></noscript></span></span> is a (<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-22.gif"/></noscript></span></span>) matrix for <em>k</em> factors, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-23.gif"/></noscript></span></span> is a vector with var<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-24.gif"/></noscript></span></span>) = <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-25.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-26.gif"/></noscript></span></span> is the (<span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-27.gif"/></noscript></span></span>) marker genetic variance matrix across trials.</p><p id="p-18">Let <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-28.gif"/></noscript></span></span> also take a general form<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-29.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>Where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-30.gif"/></noscript></span></span> is (<span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-31.gif"/></noscript></span></span>) matrix, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-32.gif"/></noscript></span></span> is a vector with var(<span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-33.gif"/></noscript></span></span>) = <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-34.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-35.gif"/></noscript></span></span> is a (<span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-36.gif"/></noscript></span></span>) matrix for <em>l</em> factors, <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-37.gif"/></noscript></span></span> is a vector with var<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-38.gif"/></noscript></span></span>) = <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-39.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-40.gif"/></noscript></span></span> is the (<span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-41.gif"/></noscript></span></span>) residual genetic variance matrix across trials.</p><p id="p-19">Thus with <em>s</em> trials, the genetic variance matrices <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-42.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-43.gif"/></noscript></span></span> are both (<span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-44.gif"/></noscript></span></span>) matrices, each with <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-45.gif"/></noscript></span></span> parameters to be estimated.</p><p id="p-20">Let <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-46.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-47.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-48.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-49.gif"/></noscript></span></span> are (<span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-50.gif"/></noscript></span></span>) and (<span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-51.gif"/></noscript></span></span>) matrices of <em>k</em> and <em>l</em> factor loadings for each trial, respectively</p><p id="p-21">then<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-52.gif"/></noscript></span></span>and<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-53.gif"/></noscript></span></span>The vector <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-54.gif"/></noscript></span></span> consists of sub vectors <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-55.gif"/></noscript></span></span>, where the subvector <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-56.gif"/></noscript></span></span> corresponds to the <em>i</em>th random term. The corresponding design matrix<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-57.gif"/></noscript></span></span> is partitioned conformably as <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-58.gif"/></noscript></span></span>. The subvectors are assumed mutually independent with variance <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-59.gif"/></noscript></span></span>. The subvectors include random terms for describing spatial trends in individual trials, such as random row, random column, or spline terms. The residual vector <strong>ε</strong> has variance <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-60.gif"/></noscript></span></span>, a block diagonal matrix of <em>s</em> blocks, <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-61.gif"/></noscript></span></span>.</p><p id="p-22">In crops, the modeling of spatial trends in field trials is crucial (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gilmour <em>et al.</em> 1997</a>), and the model above enables the addition of these trends where necessary. Furthermore, trial-specific design or randomization-based terms such as blocking factors can also be included in the model (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cullis <em>et al.</em> 2006</a>).</p><p id="p-23">Thus the line term <em><strong>g</strong></em> reflects the total genetic variation partitioned into additive variation as described by the markers and residual genetic or nonadditive variation, the fixed <strong>τ</strong>, random <em><strong>u</strong></em> and residual <strong>ε</strong> terms reflect the design and conduct of the trials, and as such provide the underlying structure for nongenetic variation.</p><div id="sec-4" class="subsection"><h4>Special cases of the general form of g:</h4><p id="p-24">The general form of <em><strong>g</strong></em> (Equation 2) shows the details for partitioning the total genetic variation. There are two special cases of <em><strong>g</strong></em>. A <em>phenotypic</em> model can be fitted by letting <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-62.gif"/></noscript></span></span>, so that the <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-63.gif"/></noscript></span></span> marker term is omitted, and <em><strong>g</strong></em> represents total genetic variation (assuming unrelated lines) as described by the phenotypic information. A <em>standard</em> RR-BLUP model for GS can be fitted by letting <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-64.gif"/></noscript></span></span>, so that <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-65.gif"/></noscript></span></span>, the residual genetic term, is omitted; here, <em><strong>g</strong></em> represents additive genetic variation as described by the markers. The <em>standard</em> model reflects the most common current practice in GS, where only the markers are included in the model. These two additional models will be used as comparators to the new <em>partitioned</em> model.</p></div><div id="sec-5" class="subsection"><h4>Special cases of the general form of <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-66.gif"/></noscript></span></span>:</h4><p id="p-25">From the general form of <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-67.gif"/></noscript></span></span> (Equation 3), special cases of <em>s</em>, the number of trials, and <em>k</em>, the number of factors, can be considered. By varying <em>s</em>, single and multiple environment trials are encompassed, and by varying <em>k</em> different structures for the variance-covariance matrix of <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-68.gif"/></noscript></span></span> can be considered (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Varying <em>k</em> enables an appropriate form of the var<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-69.gif"/></noscript></span></span> to be established to describe the correlation structure between trials, and may vary depending on the data set. A similar table of special cases of the form of <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-70.gif"/></noscript></span></span> (Equation 4) can also be constructed if <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-71.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-72.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-73.gif"/></noscript></span></span>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468169/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468169/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of the special cases of the general form of <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/T1/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/T1/embed/mml-math-74.gif"/></noscript></span></span><sup><em><sup>a</sup></em></sup></span><div class="sb-div caption-clear"></div></div></div><p id="p-31">For a single trial <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-102.gif"/></noscript></span></span> is a main marker term, with <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-103.gif"/></noscript></span></span>. Models for multiple environment trials are now discussed.</p><p id="p-32">The simplest model for more than one trial is the diagonal (DIAG) model, where <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-104.gif"/></noscript></span></span> is a main marker term in each of the trials. The <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-105.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-106.gif"/></noscript></span></span> has off-diagonals for all trials assumed zero. The DIAG model therefore assumes a separate marker variance for each trial, and no marker covariance between trials, and is equivalent to fitting each trial separately.</p><p id="p-33">In the compound symmetry (CS) model, <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-107.gif"/></noscript></span></span> is a main term for markers, and <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-108.gif"/></noscript></span></span> is an interaction term for the markers and trials. All trials have the same marker variance, and all pairs of trials have the same marker covariance, so that the <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-109.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-110.gif"/></noscript></span></span> is (<span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-111.gif"/></noscript></span></span>) matrix of ones and <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-112.gif"/></noscript></span></span>.</p><p id="p-34">For the <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cullis <em>et al.</em> (1998)</a> (CS+DIAG) model, <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-113.gif"/></noscript></span></span> is a main marker term, and <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-114.gif"/></noscript></span></span> is a term for the interaction of the markers and trials. This model assumes the same marker covariance for pairs of trials, and a separate marker variance for each trial. Thus, the form of <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-115.gif"/></noscript></span></span> is the same in the CS and (CS+DIAG) models; however, in the latter, <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-116.gif"/></noscript></span></span>. In this model, the covariance between pairs of trials is assumed to be not greater than the variance of the individual trials.</p><p id="p-35">An unstructured (US) model allows different marker variances and covariances between trials, so that <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-117.gif"/></noscript></span></span> is an interaction term of the markers and trials, and no main marker effect is fitted. The<span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-118.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-119.gif"/></noscript></span></span> has diagonal elements that are the marker variances for the individual trials, and off-diagonal element that are the marker covariance between trials. As the number of trials increases, the US model becomes over parameterized, making it difficult to fit.</p><p id="p-36">Multiplicative models have been shown to work well in practice in MET analysis (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Smith <em>et al.</em> 2005</a>), and are viable alternatives to the unstructured model. In fact, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kelly <em>et al.</em> (2007)</a> found that the factor analytic model with <em>k</em> factors (FA<em>k</em>) of <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Smith <em>et al.</em> (2001)</a> was preferred over an unstructured model because it improved the predictive accuracy of the line empirical BLUPs.</p><p id="p-37">In a GS situation, a factor analytic model with a main marker term and <em>k</em> factors (FAM<em>k</em>) may be more appropriate than a FA<em>k</em> model that excludes this term. This is because a main marker term represents QTL that are common and stable across trials (in the absence of an interaction between trials and markers) and the marker × trial term will give information on QTL that are trial or environment specific.</p><p id="p-38"><a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Smith <em>et al.</em> (2001)</a> showed that a FAM<em>k</em> model is equivalent to a factor analytic model with (<em>k</em>+1) factors, where the first set of loadings are constrained to be equal. For a FAM<em>k</em> model, we let <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-120.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-121.gif"/></noscript></span></span>, with var<span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-122.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-123.gif"/></noscript></span></span> is a vector of line scores with <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-124.gif"/></noscript></span></span>, then <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-125.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-126.gif"/></noscript></span></span> is a (<span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-127.gif"/></noscript></span></span>) matrix of loadings, and <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-128.gif"/></noscript></span></span> is a diagonal matrix with the diagonal elements referred to as specific variances. The approach of including a main marker term is in contrast to a phenotypic model for estimating genotypic values, where, in crop field trials, the main line term is usually excluded. Notice when <em>k = 0</em>, the FAM<em>0</em> model is equivalent to the CS+DIAG model. The special cases of the general form of <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-129.gif"/></noscript></span></span> shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, can be fitted in each of the three models, <em>phenotypic</em>, <em>standard</em>, and <em>partitioned</em>, which reflect different forms of <strong><em>g</em></strong> (Equation 2).</p></div><div id="sec-6" class="subsection"><h4>Computational efficiency:</h4><p id="p-39">If the number of markers exceeds the number of individuals, we can fit<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-130.gif"/></noscript></span></span>The analysis will now be dependent on the number of lines rather than on the number of markers, and therefore will reduce the dimensionality of the model, making it more computationally efficient (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Stranden and Garrick 2009</a>).</p><p id="p-40">The estimation of variance parameters is by residual maximum likelihood (REML). Given estimates of the variance components, empirical best linear unbiased predictors (E-BLUPs) were obtained for random terms from the mixed model equations as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-131.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-132.gif"/></noscript></span></span> is the vector of genomic breeding values in each trial.</p><p id="p-41">Here, we use <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-133.gif"/></noscript></span></span>as in <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Piepho <em>et al.</em> (2012)</a>, where <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-134.gif"/></noscript></span></span> represents a realized genomic relationship matrix. <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Meuwissen <em>et al.</em> (2001)</a> used <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-135.gif"/></noscript></span></span>, where <em>p</em> is the number of marker (locations), and <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Habier <em>et al.</em> (2007)</a> used <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-136.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-137.gif"/></noscript></span></span> is the allele frequency at marker locus <em>q</em>. Omission of the scalar term will not affect the conclusions of the analysis.</p><p id="p-42">If <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-138.gif"/></noscript></span></span> is of full rank, then<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-139.gif"/></noscript></span></span>for <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-140.gif"/></noscript></span></span></p><p id="p-43">Thus, if<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-141.gif"/></noscript></span></span>Then, the E-BLUPs obtained from the mixed model equations are<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-142.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>and<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-143.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>If <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-144.gif"/></noscript></span></span> is of full rank, then<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-145.gif"/></noscript></span></span>For our data set, the number of markers exceeds the number of individuals, therefore the <em>standard</em> and <em>partitioned</em> models were fitted using the computationally efficient approach. All models were fitted in ASReml v3.0-1 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Butler <em>et al.</em> 2009</a>) for R v3.2.0 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">R Core Team 2015</a>). Instructions for completing all the analysis shown in the paper can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS1.pdf">File S1</a> along with supporting data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS2.zip">File S2</a>) and R scripts (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS3.zip">File S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS4.zip">File S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS5.zip">File S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS6.zip">File S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS7.zip">File S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS8.zip">File S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS9.zip">File S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS10.zip">File S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS11.zip">File S11</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS12.zip">File S12</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS13.zip">File S13</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS14.zip">File S14</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS15.zip">File S15</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS16.zip">File S16</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS17.zip">File S17</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS18.zip">File S18</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS19.zip">File S19</a>).</p></div></div><div id="sec-7" class="subsection"><h3>Heritability</h3><p id="p-44">The calculation of the generalized heritability in complex linear mixed models is not straightforward (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cullis <em>et al.</em> 2006</a>). Here, the generalized heritability for each trial is calculated from the phenotypic model (where <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-146.gif"/></noscript></span></span>) as <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-147.gif"/></noscript></span></span> where <em>a</em> is the average pairwise prediction error variance of line effects, and <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-148.gif"/></noscript></span></span> is the genetic variance of trial <em>t</em> (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Cullis <em>et al.</em> 2006</a>). The R code for calculating the heritability is in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS19.zip">File S19</a>.</p></div><div id="sec-8" class="subsection"><h3>Cross-validation</h3><p id="p-45">Initially, the <em>phenotypic</em>, <em>standard</em>, and <em>partitioned</em> models were fitted using the <em>full</em> data set to enable the most appropriate MET form of <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-149.gif"/></noscript></span></span> (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), the term representing marker or additive variation, and <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-150.gif"/></noscript></span></span>, the term representing nonadditive or residual genetic variance, to be established for use in the cross-validation. A single trial analysis was also investigated in the cross-validation represented by the DIAG form. We generally denote the <em>phenotypic</em>, <em>standard</em>, and <em>partitioned</em> models with specific forms of <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-151.gif"/></noscript></span></span> as E<sub>FORM</sub>, S<sub>FORM</sub>, and P<sub>FORM</sub>, respectively [where FORM is either DIAG, US, CS, CS+DIAG, or FAM<em>k</em> (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>)].</p><p id="p-46">To establish which MET form of <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-152.gif"/></noscript></span></span> is superior, and to undertake initial comparisons between the three models, the Akaike information criteria (AIC) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akaike 1974</a>), or log-likelihood ratio test (if the models were nested), were used. For example, the <em>phenotypic</em> and <em>standard</em> model are nested within the <em>partitioned</em> model when the form of <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-153.gif"/></noscript></span></span> is the same, and can be compared using the log-likelihood ratio test. However, models with different forms for <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-154.gif"/></noscript></span></span> need to be compared using the AIC. The AIC was calculated as twice the number of random parameters minus twice the log-likelihood (the number of fixed parameters was ignored in this calculation as this was constant over the models). For the AIC, lower values indicate superior models. Once the form of <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-155.gif"/></noscript></span></span> for the MET analyses has been established, the cross-validations could proceed.</p><p id="p-47">The cross-validation involved randomly dividing the lines in the data set (as evenly as possible) into 10 groups, which were used across the single and MET analyses so that consistency and comparability was maintained as much as possible (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468190/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468190/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Number of lines with marker information in the groups used in the cross-validation</span><div class="sb-div caption-clear"></div></div></div><p id="p-51">Three different cross-validations were examined, and were defined by the number of groups in the validation and training sets. These were CV10, CV20, and CV40, where approximately 10, 20, and 40% of lines, respectively, were included in the validation set (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>), with the remaining lines used as the training set. Iteration across all combinations was investigated. For example, for the CV20 cross-validation, two groups (the validation set) were omitted in any one iteration. To cover the possible combinations of two of the 10 groups, a total of 45 iterations were necessary. The R scripts with details of the random division of the data and group combinations for CV10, CV20, and CV40 are in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS3.zip">File S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS4.zip">File S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS5.zip">File S5</a>, respectively.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468010/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468010/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of validation and training groups in three cross-validations</span><div class="sb-div caption-clear"></div></div></div><p id="p-54">Using the training set, marker effects were obtained under each model (<em>standard</em> and <em>partitioned</em>), each cross-validation (CV10, CV20, and CV40), and each scenario (single trial 2010, single trial 2011, and MET). The marker effects (Equation 6 and Equation 7) were used to predict the GEBVs of the lines in the validation set. The DIAG form of <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-156.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-157.gif"/></noscript></span></span> was used to generate marker effects equivalent to analyzing each trial separately, with each trial having one set of marker effects. For the MET analyses of both trials, different forms of <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-158.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-159.gif"/></noscript></span></span> (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>) were initially investigated as described previously, with the most appropriate forms used in the cross-validation. For the CS, CS+DIAG, and FAM<em>1</em> forms of <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-160.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-161.gif"/></noscript></span></span>, there were three sets of marker effects. These were: the main marker effects, representing markers that are stable across both trials, and marker effects from each trial, which represent the additional marker by trial interaction. A total marker effect for each trial was obtained by adding the main marker effect to the marker by trial interaction effect. For completeness, the US form, which produces only total marker effects for each trial, was also initially investigated, although, as we are interested in MET models with main marker effects, the cross-validation focused on the most appropriate of the CS, CS+DIAG, and FAM<em>1</em> forms.</p><p id="p-55">In simulations, the true breeding value is known, and the GEBV calculated in the lines of the validation set can be compared to the true breeding values to determine the predictive ability and the accuracy and precision of the GEBV. As real data were used here, the true breeding value was unknown. In the absence of pedigree information, the genotypic values (GV) of the lines in the validation set were used as the comparator. These were calculated from a <em>phenotypic</em> model, based on all the lines with marker information, where lines were assumed to be unrelated. The GVs from this <em>phenotypic</em> model reflect total genetic effects, and were calculated using single trial and MET analyses. In crops, breeders are most often interested in the commercial potential of lines, so the adoption of the genotypic values (which are the total genetic values of each line) as the comparator to GEBV of validation lines reflects this crop breeding strategy. Linear regression models were fitted in which the response was the observed GVs of lines in the validation set, and the explanatory variable was the GEBVs of those same lines calculated using the marker effects. The cross-validation investigated the performances of the <em>partitioned</em> and <em>standard</em> RR-BLUP models in terms of their predictive ability measured by the R-squared value, and the accuracy and precision of the GEBV measured by the mean square error (MSE) from the linear regression models. The <em>standard</em> and <em>phenotypic</em> models were compared using the same training sets.</p><p id="p-56">Comparisons were investigated between the GEBVs and GVs, where the same model (single trial or MET) and marker terms were used to derive both the GEBV and GV. A further comparison was made between the GEBVs of the main effect from the MET model to GVs from a single trial analysis of each trial year. The R script for performing each cross-validation (CV10, CV20, and CV40) under each scenario (single trial 2010, single trial 2011, and MET) can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS6.zip">File S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS7.zip">File S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS8.zip">File S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS9.zip">File S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS10.zip">File S10</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS11.zip">File S11</a>.</p></div><div id="sec-9" class="subsection"><h3>Implementing GS with a lower density of markers</h3><p id="p-57">In addition to using all the markers to predict the GEBV of lines in the validation set, we explored a low-density GS approach, where a subset of <em>random</em>, rather than significant, markers and their effects were used to predict the GEBV of lines. This was explored for CV10 only (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-58">For each of the 10 training sets of lines, subsets of markers of increasing size, <em>x</em>, from <em>x</em> = 10 to <em>x</em> = 3490 (all markers) were chosen at random. For each size <em>x</em>, and each training set of lines, the markers were chosen at random, and resampled to provide 200 different random combinations of size <em>x</em>. The performance at each subset of markers was measured as the average of the regression coefficients, and mean square error over the 200 random combinations of size <em>x</em>. For each <em>x</em>, each scenario and model was compared using the same random marker subsets and the same training set of lines. R scripts can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS12.zip">File S12</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS13.zip">File S13</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS14.zip">File S14</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS15.zip">File S15</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS16.zip">File S16</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS17.zip">File S17</a>.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-59">All data necessary for confirming the conclusions presented in this manuscript are provided in Supplemental Material.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Height phenotypic data in multi-environment trials</h3><p id="p-60">An association mapping population of two-row spring barley lines was grown in a randomized spatial row-column design in two consecutive trials in years 2010 and 2011 (<em>i.e.</em>, two environments), with five replicate plants of each line. At full maturity, the height of each plant was measured. The raw mean heights for lines grown in both years, and for which we have marker information, are shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The plant height was slightly elevated in 2010 as opposed to 2011; the mean height in 2010 was 94.8 cm, and in 2011 was 87.9 cm. The correlation between the means of the line heights across the years was high at 0.76. These data were subsequently used, as described below, to develop our new GS model, and to predict, based only on the complement of lines with molecular markers present in each training set, the GEBV for height of lines that were in each validation set.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1313/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation of mean heights of lines in the 2010 and 2011 trials. The datapoints represent only the lines with marker data that were grown in both years." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1309847294" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlation of mean heights of lines in the 2010 and 2011 trials. The datapoints represent only the lines with marker data that were grown in both years.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1313/F1.medium.gif" width="440" height="401"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/5/1313/F1.medium.gif" width="440" height="401"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1313/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1313/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468229" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-61" class="first-child">Correlation of mean heights of lines in the 2010 and 2011 trials. The datapoints represent only the lines with marker data that were grown in both years.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Comparison of models for whole genome prediction</h3><p id="p-62">Initially, the <em>partitioned</em>, <em>standard</em>, and <em>phenotypic</em> models were fitted (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>) using the full data set, enabling the form of <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-162.gif"/></noscript></span></span> (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>), the term representing marker variation, and <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-163.gif"/></noscript></span></span>, the term representing nonadditive or residual genetic variance, to be established for use in the cross-validation. All models included a random term for replicate block for both trials; an additional random term was also included in the analysis to account for spatial variation present between columns in 2011.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467994/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467994/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Summary of the models fitted to the full data set</span><div class="sb-div caption-clear"></div></div></div><p id="p-74">Log-likelihood ratio tests comparing <em>standard</em> and <em>partitioned</em> models with the same form of <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-166.gif"/></noscript></span></span> (<em>e.g.</em>, S<sub>DIAG</sub> <em>vs.</em> P<sub>DIAG</sub>, S<sub>CS+DIAG</sub> <em>vs.</em> P<sub>CS+DIAG</sub>, etc.) were significant (<em>P</em> &lt; 0.001), suggesting the <em>partitioned</em> models are superior to the <em>standard</em> models (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). In addition, all the MET models had lower AIC than the single trial year models, suggesting that the MET models were superior (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). For two trials, the factor analytic models (E<sub>FAM</sub><em><sub>1</sub></em>, S<sub>FAM</sub><em><sub>1</sub></em>, and P<sub>FAM</sub><em><sub>1</sub></em>), and the unstructured models (E<sub>US</sub>, S<sub>US</sub>, and P<sub>US</sub>), respectively, are identical, and therefore the US results are shown with the FAM<em>1</em> model in <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>. Given that the factor analytic model has been shown previously to improve the predictive accuracy of the line empirical BLUPs, that factor analytic models are easier to fit than an unstructured model (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kelly <em>et al.</em> 2007</a>), and given that we are primarily interested in models that fit a main marker term, the US model is not discussed further.</p><p id="p-75">For the <em>phenotypic</em>, <em>standard</em>, and <em>partitioned</em> MET models, the compound symmetry form (E<sub>CS</sub>, S<sub>CS</sub> and P<sub>CS</sub>) had the highest AIC, suggesting that this model was not a good choice in comparison to the other forms. The CS+DIAG forms (E<sub>CS+DIAG</sub> and P<sub>CS+DIAG</sub>) had similar AICs to the factor analytic models (E<sub>FAM</sub><em><sub>1</sub></em> and P<sub>FAM</sub><em><sub>1</sub></em>). However, the factor analytic model of the <em>standard</em> model (S<sub>FAM</sub><em><sub>1</sub></em>) showed a lower AIC than the CS+DIAG form (S<sub>CS+DIAG</sub>). Examining the REML estimates of the variance components (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>), it is clear that the CS+DIAG model does not necessarily fit as well as the FAM<em>1</em> model, as the covariance between years has been constrained to be equal to the 2011 trial variance. However, as the variance component estimates of the CS+DIAG form (E<sub>CS+DIAG</sub>, S<sub>CS+DIAG</sub>, and P<sub>CS+DIAG</sub>) and the FAM<em>1</em> form (E<sub>FAM</sub><em><sub>1</sub></em>, S<sub>FAM</sub><em><sub>1</sub></em>, and P<sub>FAM</sub><em><sub>1</sub></em>) were similar for this data despite these constraints, both these forms were investigated further in the cross-validation. It is worth noting that <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a> does not show the complete set of possible <em>partitioned</em> models. For the <em>partitioned</em> model, we have explored only models with the same form for the <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-167.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-168.gif"/></noscript></span></span> terms. It is, however, possible to include different forms for each of <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-169.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-170.gif"/></noscript></span></span> terms, for example, the former could take a compound symmetry form, and the latter a factor analytic form. When fitting the FAM<em>1</em> model, the full parameterization required five parameters (one for the main effect, two loadings, and two specific variances), two more than the three required. Thus, when fitting the FAM<em>1</em> model, the two specific variances were constrained to be zero. <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Smith <em>et al.</em> (2001)</a> discuss the parameter constraints necessary for FA<em>k</em> models with <em>k</em> &gt; 1. Thus based on the results of the log-likelihood ratio tests, AIC, and estimates of the model variance components, for the MET analysis of trials in the cross-validation, we explore only the CS+DIAG and FAM<em>1</em> forms for the marker and residual genetic effects. From the variance components of the diagonal form of the phenotypic model, the heritability of the trials was calculated as 0.90 in 2010 and 0.75 in 2011, so both trials have high heritability. The difference in heritability between years reflects a greater contribution of environment to the variation in 2011, and consequently a lower proportion of variation that is genetic in that year. A comparison between the variance of the <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-171.gif"/></noscript></span></span> term from <em>phenotypic</em> and the <em>partitioned</em> DIAG models (E<sub>DIAG</sub> and P<sub>DIAG</sub>, respectively) enabled an estimate of the proportion of genetic variation accounted for by the markers, which was 0.75 for trial 2010, and 0.73 for trial 2011. The R code for determining the proportion of genetic variation is in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1/FileS19.zip">File S19</a>.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468178/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468178/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">REML estimates of variance components of the models fitted (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a>)</span><div class="sb-div caption-clear"></div></div></div><p id="p-80">The variance components of the residual terms in each year were higher in the <em>standard</em> models compared to the equivalent forms of the <em>phenotypic</em> and <em>partitioned</em> models, particularly in 2011. The total genetic variation is therefore not accounted for in this model, perhaps as expected, given that the marker effect should reflect just additive genetic variation; the proportion of the genetic variation not being accounted for by the markers thus has contributed to the enlarged residual term. Given these limitations, the <em>standard</em> models were not taken forward for cross-validation.</p></div><div id="sec-14" class="subsection"><h3>Cross-validation of selected models</h3><p id="p-81">The lines were randomly divided into 10 equivalent groups to facilitate comparative cross-validations, where the majority of lines were used as the training set, while 10%, 20%, or 40% of lines constituted the CV10, CV20, and CV40 validation sets, respectively (see <em>Materials and Methods</em> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-82">The cross-validation focuses on comparing the <em>partitioned</em> and the <em>standard</em> RR-BLUP model in each of three scenarios, which are each trial year separately (P<sub>DIAG</sub> and S<sub>DIAG</sub>, respectively), and in a joint MET analysis of both years. From the initial fitting of the MET models just discussed, we examine only the models where the form of <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-173.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-174.gif"/></noscript></span></span> takes either CS+DIAG (P<sub>CS+DIAG</sub> and S<sub>CS+DIAG</sub>, respectively), or FAM<em>1</em> (P<sub>FAM</sub><em><sub>1</sub></em> and S<sub>FAM</sub><em><sub>1</sub></em>, respectively). When fitting the <em>standard</em> and <em>partitioned</em> models of the factor analytic form (S<sub>FAM</sub><em><sub>1</sub></em>, P<sub>FAM</sub><em><sub>1</sub></em>) in ASReml, specific variances were set to zero as in the full data set, and the variance components of S<sub>FAM</sub><em><sub>1</sub></em> were used as starting values for the variance components of P<sub>FAM</sub><em><sub>1</sub></em>.</p><p id="p-83">Various comparisons between the predicted (GEBVs) and observed (GVs) results were made, and the performance of the <em>partitioned</em> and <em>standard</em> RR-BLUP models in terms of predictive ability of the markers was measured by the R-squared value (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>), while the accuracy of line estimates (GEBVs) was measured by the mean square error (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>). Both of these were averages across the iterations (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468282/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468282/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Average <em>R</em>-squared<sup><em><sup>a</sup></em></sup> (SD) of partitioned verses standard RR-BLUP model for different cross-validation (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>), models (<a id="xref-table-wrap-4-8" class="xref-table" href="#T4">Table 4</a>) and effects used to generate the GV and GEBV</span><div class="sb-div caption-clear"></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468253/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468253/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Average mean square error<sup><em><sup>a</sup></em></sup> (SD) of partitioned verses standard RR-BLUP model for cross-validation (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>), models (<a id="xref-table-wrap-4-9" class="xref-table" href="#T4">Table 4</a>) and effects used to generate the GV and GEBV</span><div class="sb-div caption-clear"></div></div></div><p id="p-92">In all three different CV evaluations [CV10, CV20, and CV40 (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>)], the <em>partitioned</em> model showed a higher R-squared and lower MSE than the <em>standard</em> model, indicating that the predictive ability and accuracy of line estimates (GEBVs) were superior in the <em>partitioned</em> model. This supports the finding of lower log-likelihoods and superior fit of the <em>partitioned</em> models as compared to the <em>standard</em> models when the full data were used (<a id="xref-table-wrap-4-10" class="xref-table" href="#T4">Table 4</a>). The R-squared decreased, and the MSE increased, as the number of lines in the training set decreases (going from CV10 to CV40) for the equivalent model (<em>i.e.</em>, within the <em>partitioned</em> models, or within the <em>standard</em> models), which was expected, as predictions were based on fewer lines. It should be noted that, except where the 2011 marker effects from the P<sub>DIAG</sub> model were used to generate the GEBVs (Comparisons 2 and 3, <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>), the R-squared in the <em>partitioned</em> model in CV40 (where 40% of the lines were in the validation set) was similar to, or higher than, the R-squared of the <em>standard</em> model in CV10 (where 10% of the lines are in the validation set), suggesting the <em>partitioned</em> model was superior even when reducing the number of lines upon which the predictions are based. There was only a small compensating increase in MSE between the <em>standard</em> model in CV10, and the <em>partitioned</em> model in CV40, with the same exception of the 2011 marker effects from the P<sub>DIAG</sub> model (Comparisons 2 and 3, <a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a>), and also the main marker effects from the P<sub>FAM</sub><em><sub>1</sub></em>, which showed larger increases (Comparison 12, <a id="xref-table-wrap-7-3" class="xref-table" href="#T7">Table 7</a>). These results shows that the <em>partitioned</em> model provides the best predictions of the height of lines in the validation set.</p><p id="p-93">The results of the CV10 cross-validation were considered in more detail, bearing in mind that the other cross-validations (CV20 and CV40) reflect similar patterns. Initially, results where the equivalent of a single trial analyses was used to generate both GEBVs (P<sub>DIAG</sub> and S<sub>DIAG</sub>) and GVs (E<sub>DIAG</sub>) were examined. Using the same trial year to generate both the GEBVs and GVs (Comparisons 1 and 3, <a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>) gave the best results. The single trial analysis from 2010 showed a higher predictive ability than that from 2011 for the <em>partitioned</em> model (0.461 <em>vs.</em> 0.334, <a id="xref-table-wrap-6-4" class="xref-table" href="#T6">Table 6</a>), and for the <em>standard</em> model (0.404 <em>vs.</em> 0.323, <a id="xref-table-wrap-6-5" class="xref-table" href="#T6">Table 6</a>). This difference between the predictive ability in each year was initially surprising; given that there was a large overlap of individuals, the percentage of variation explained by the markers was similar, and the observations were highly correlated across the 2 years. The <em>partitioned</em> model showed a 5.7% and 1.1% improvement for 2010 and 2011, respectively, in predictive ability over the <em>standard</em> model.</p><p id="p-94">If the opposite trial years were used to generate the GEBVs, then predicting 2010 GVs using the marker effects generated in 2011 (Comparisons 2, <a id="xref-table-wrap-6-6" class="xref-table" href="#T6">Table 6</a>) had a higher predictive ability than the opposite combination (Comparisons 4, <a id="xref-table-wrap-6-7" class="xref-table" href="#T6">Table 6</a>), for both the <em>partitioned</em> and <em>standard</em> models. <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ly <em>et al.</em> (2013)</a> suggested that G × E, which cannot be accounted for in a single trial, reduces the ability to make predictions. These results suggest that the 2011 heights had a higher environmental component than those observed in 2010, making prediction of the GEBVs from 2011 more difficult. We would therefore expect to be able to predict trial year 2011 better if a marker by trial interaction effect was included in a MET model, and this is exactly what was found, as discussed below.</p><p id="p-95">The next comparisons are the MET analyses, where the GEBVs and GVs are based on a total effect found by summing the main or overall marker effect and the trial year marker effects. The first thing to note is that for 2010 the CS+DIAG form for <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-175.gif"/></noscript></span></span> had around a 4–6% higher predictive ability than the FAM<em>1</em> form in both the <em>standard</em> and <em>partitioned</em> models (Comparisons 5 and 7, <a id="xref-table-wrap-6-8" class="xref-table" href="#T6">Table 6</a>). For 2011, the results were similar between the forms of <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-176.gif"/></noscript></span></span> (Comparisons 6 and 8, <a id="xref-table-wrap-6-9" class="xref-table" href="#T6">Table 6</a>), with around a 0.1% difference in predictive ability between the FAM<em>1</em> form and the CS+DIAG form, with the latter showing only a slightly lower predictive ability, but a lower MSE. This suggests that the CS+DIAG form was superior to the FAM<em>1</em> form for this data set. Exploring different forms for <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-177.gif"/></noscript></span></span> is clearly an important step in determining the best model for GS.</p><p id="p-96">The results for predictive ability were similar between the MET analysis and single year analysis for 2010 (Comparisons 5 and 1, 0.462 v 0.461, <em>partitioned</em> model, <a id="xref-table-wrap-6-10" class="xref-table" href="#T6">Table 6</a>), but the MET shows a lower MSE (Comparisons 5 and 1, 7.35 v 7.47, <em>partitioned</em> model, <a id="xref-table-wrap-7-4" class="xref-table" href="#T7">Table 7</a>). For 2011, the results of the MET were clearly superior over the single year analysis (Comparisons 6 and 3, <a id="xref-table-wrap-6-11" class="xref-table" href="#T6">Table 6</a>), at 11.4% higher (0.448 v 0.334) for the <em>partitioned</em> model and 7.2% (0.395 v 0.323) for the <em>standard</em> model. This suggests that using 2 years’ data greatly improved the accuracy of the GEBVs by up to 11.4% for this data set. The benefit of including the marker by trial interaction effect was apparent in 2011 in particular, the year in which the results from the single trial analysis suggested that the environment had a large influence on the results. The greater environmental influence in 2011 is consistent with the lower heritability and reduced genetic influence on plant height compared to the 2010 trial (see previous section). Greater environmental stress may also explain the lower mean height of lines in 2011.</p><p id="p-97">Finally, we compared the GEBVS derived from the main marker effects in a MET analysis with the GVs derived from the single year analysis. Again the CS+DIAG form was superior to the FAM<em>1</em> form for <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1313/embed/mml-math-178.gif"/></noscript></span></span> in terms of predictive ability, and we therefore concentrated on the former (Comparisons 9 and 10, <a id="xref-table-wrap-6-12" class="xref-table" href="#T6">Table 6</a>). There was a reduction in predictive ability if the year specific marker effect was omitted when calculating the GEBVs, particularly for 2011 (0.448 <em>vs.</em> 0.336, Comparison 6 and 10, <a id="xref-table-wrap-6-13" class="xref-table" href="#T6">Table 6</a>), where the environmental influence was higher. However, despite this reduction, a similar predictive ability to a single trial analysis was maintained, and the main effect still had a higher predictive ability than using the marker effects from the opposite year. For example, it is 3.5% higher in 2010 (Comparisons 9 and 2, 0.442 <em>vs.</em> 0.407), and 1.7% higher in 2011 (Comparisons 10 and 3, 0.336 <em>vs.</em> 0.319), with correspondingly lower MSE.</p></div><div id="sec-15" class="subsection"><h3>Implementing GS with a lower density of markers</h3><p id="p-98">A low density GS approach was considered, where, for each of the training sets of lines in CV10, the predictive ability of subsets of random markers was investigated. The results of the MET analyses and single trial analyses across the subsets of random markers were compared in each trial year (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> for trial 2010 and trial 2011, respectively). Both plots showed that the <em>partitioned</em> models have a much steeper incline within the subsets of random markers containing less than 500 markers than the <em>standard</em> models, with the graphs flattening out more than the <em>standard</em> models as the marker numbers in subsets increase. This suggests that, for lower numbers of markers, the predictions were more accurate and reliable in the <em>partitioned</em> model, and therefore fewer markers were required to obtain similar predictions. This would tend to support the finding that the line estimates based on markers from the <em>partitioned</em> model were more accurate (lower MSE, <a id="xref-table-wrap-7-5" class="xref-table" href="#T7">Table 7</a>) than those of the <em>standard</em> model. The horizontal lines in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> show the single trial year analysis of the <em>standard</em> model using data from the same year. For trial 2010, all of the <em>partitioned</em> models had superior predictive ability across the entire range of marker subsets (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>), with the main improvements in predictive ability coming within the first 1000 or so markers. In the <em>standard</em> RR-BLUP model, although there was an initial improvement in the predictive ability, this was less intense, and mostly small and steady over all of the latter marker subsets. This means that we can achieve the same predictive ability with the MET model as with the <em>standard</em> RR-BLUP model using around 1000 markers, or around 2490 markers less than in the full higher-density GS. In 2011, the <em>partitioned</em> models had superior performance over the <em>standard</em> single year model for lower numbers of markers, with the <em>partitioned</em> MET model reaching the same level of prediction with only 500 markers.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1313/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of partitioned vs. standard RR-BLUP model of CV10 (Table 3) for different forms (Table 1) and comparisons (Table 6) for trial year 2010 across a range of subsets of random markers. The horizontal line is maximum predictive ability of the standard single trial year analysis for 2010. Each subset represents the average results from 200 different sets of random markers, the comparisons across analyses are on the same subsets of random markers. MET, multi-environment trial analysis; STY, single trial year analysis; part, partitioned model; std, standard model; total, main marker effect&#xA0;+&#xA0;marker by trial interaction effect; main, main marker effect; same, same year used for prediction (2010 GEBV used to predict 2010 GV); opp, opposite year used for prediction (2011 GEBV used to predict 2010 GV); C, see Comparison as per Table 6 for more detail." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1309847294" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of &lt;em&gt;partitioned vs. standard&lt;/em&gt; RR-BLUP model of CV10 (Table 3) for different forms (Table 1) and comparisons (Table 6) for trial year 2010 across a range of subsets of random markers. The horizontal line is maximum predictive ability of the &lt;em&gt;standard&lt;/em&gt; single trial year analysis for 2010. Each subset represents the average results from 200 different sets of random markers, the comparisons across analyses are on the same subsets of random markers. MET, multi-environment trial analysis; STY, single trial year analysis; part, &lt;em&gt;partitioned&lt;/em&gt; model; std, &lt;em&gt;standard&lt;/em&gt; model; total, main marker effect&#xA0;+&#xA0;marker by trial interaction effect; main, main marker effect; same, same year used for prediction (2010 GEBV used to predict 2010 GV); opp, opposite year used for prediction (2011 GEBV used to predict 2010 GV); C, see Comparison as per Table 6 for more detail.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1313/F2.medium.gif" width="440" height="397"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/5/1313/F2.medium.gif" width="440" height="397"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1313/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1313/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468087" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-99" class="first-child">Comparison of <em>partitioned vs. standard</em> RR-BLUP model of CV10 (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>) for different forms (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>) and comparisons (<a id="xref-table-wrap-6-14" class="xref-table" href="#T6">Table 6</a>) for trial year 2010 across a range of subsets of random markers. The horizontal line is maximum predictive ability of the <em>standard</em> single trial year analysis for 2010. Each subset represents the average results from 200 different sets of random markers, the comparisons across analyses are on the same subsets of random markers. MET, multi-environment trial analysis; STY, single trial year analysis; part, <em>partitioned</em> model; std, <em>standard</em> model; total, main marker effect + marker by trial interaction effect; main, main marker effect; same, same year used for prediction (2010 GEBV used to predict 2010 GV); opp, opposite year used for prediction (2011 GEBV used to predict 2010 GV); C, see Comparison as per <a id="xref-table-wrap-6-15" class="xref-table" href="#T6">Table 6</a> for more detail.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1313/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of partitioned verses standard RR-BLUP model of CV10 (Table 3) for different forms (Table 1) and comparisons (Table 6), for trial year 2011 across a range of subsets of random markers. The horizontal line is maximum predictive ability of the standard single trial year analysis for 2011. Each subset represents the average results from 200 different sets of random markers, the comparisons across analyses are on the same subsets of random markers. MET, multi-environment trial analysis; STY, single trial year analysis; part, partitioned model; std, standard model; total, main marker effect&#xA0;+&#xA0;marker by trial interaction effect; main, main marker effect; same, same year used for prediction (2011 GEBV used to predict 2011 GV); opp, opposite year used for prediction (2010 GEBV used to predict 2011 GV); C, see Comparison as per Table 6 for more detail." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1309847294" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of &lt;em&gt;partitioned&lt;/em&gt; verses &lt;em&gt;standard&lt;/em&gt; RR-BLUP model of CV10 (Table 3) for different forms (Table 1) and comparisons (Table 6), for trial year 2011 across a range of subsets of random markers. The horizontal line is maximum predictive ability of the &lt;em&gt;standard&lt;/em&gt; single trial year analysis for 2011. Each subset represents the average results from 200 different sets of random markers, the comparisons across analyses are on the same subsets of random markers. MET, multi-environment trial analysis; STY, single trial year analysis; part, &lt;em&gt;partitioned&lt;/em&gt; model; std, &lt;em&gt;standard&lt;/em&gt; model; total, main marker effect&#xA0;+&#xA0;marker by trial interaction effect; main, main marker effect; same, same year used for prediction (2011 GEBV used to predict 2011 GV); opp, opposite year used for prediction (2010 GEBV used to predict 2011 GV); C, see Comparison as per Table 6 for more detail.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1313/F3.medium.gif" width="440" height="391"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/5/1313/F3.medium.gif" width="440" height="391"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1313/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1313/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468107" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-100" class="first-child">Comparison of <em>partitioned</em> verses <em>standard</em> RR-BLUP model of CV10 (<a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>) for different forms (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>) and comparisons (<a id="xref-table-wrap-6-16" class="xref-table" href="#T6">Table 6</a>), for trial year 2011 across a range of subsets of random markers. The horizontal line is maximum predictive ability of the <em>standard</em> single trial year analysis for 2011. Each subset represents the average results from 200 different sets of random markers, the comparisons across analyses are on the same subsets of random markers. MET, multi-environment trial analysis; STY, single trial year analysis; part, <em>partitioned</em> model; std, <em>standard</em> model; total, main marker effect + marker by trial interaction effect; main, main marker effect; same, same year used for prediction (2011 GEBV used to predict 2011 GV); opp, opposite year used for prediction (2010 GEBV used to predict 2011 GV); C, see Comparison as per <a id="xref-table-wrap-6-17" class="xref-table" href="#T6">Table 6</a> for more detail.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-101">In this paper, a method for genomic selection is proposed for the analysis of multi-environment crop trials. The method differs from other methods in a number of ways.</p><p id="p-102">The method uses raw data observations at the plot, or pot, level rather than line means, thus incorporating line replication. This enables the total genetic variation due to lines to be partitioned into variation due to markers, and residual genetic variation, where the latter accounts for any genetic variation unexplained by the markers. The GEBVs, which are based on marker effects, are assumed mostly to reflect additive effects. The residual genetic effect therefore should capture nonadditive effects. In inbred lines, the nonadditive effect will represent epistatic effects. However, in noninbred lines, other nonadditive effects will include dominance, inbreeding depression and homozygous dominance effects, the covariance between additive and dominance effects, and epistatic effects (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de Boer and Hoeschele 1993</a>).</p><p id="p-103">Previous studies (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Solberg <em>et al.</em> 2009</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Burgueno <em>et al.</em> 2012</a>) have included pedigree information, which captures a polygenic effect as a way of accounting for nonadditive effects, and more recent studies (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Da <em>et al.</em> 2014</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Munoz <em>et al.</em> 2014</a>) have used marker-based relationships to separate additive and nonadditive variation. <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Da <em>et al.</em> (2014)</a> only included additive and dominance relationship matrices. <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Munoz <em>et al.</em> (2014)</a> included, in addition, first-order epistatic relationship matrices (additive by additive, dominance by dominance, and additive by dominance); however, their approach has the disadvantage that the resulting genetic effects are nonorthogonal, and there is dependency between some of the estimates of the additive and nonadditive variance components, as the first-order epistatic relationship matrices they form are based on Hadamard products of the additive and dominance relationship matrices. This means that the matrices may not be capturing all nonadditive variation. Also, given the number of relationship matrices necessary to account for nonadditive variation, extension to a MET model may be difficult. In contrast, the residual genetic effect representing nonadditive effects found in our model is more encompassing and less restrictive, and should capture all nonadditive effects. It is, however, worth noting that, because of the flexibility of the linear mixed model, additional relationships matrices may also be added to our model if required. For example, if pedigree information was available, a single polygenic effect could be added, or it may be possible to further partition our residual genetic effects by the addition of further genomic relationship matrices, for instance, a genomic dominance matrix could be added in the case of hybrid crops.</p><p id="p-104">Recent studies have shown that maximum prediction was reached when the breeding value was based on both additive and nonadditive effects (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Da <em>et al.</em> 2014</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Munoz <em>et al.</em> 2014</a>), and <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Ly <em>et al.</em> (2013)</a> notes that considering only the additive component may underestimate prediction accuracy. Here, we have only investigated the use of the additive proportion of the total genetic effect as described by the markers for the prediction of breeding values where, for future lines, <em>only</em> marker information is available. We found that partitioning the total genetic variation into marker and residual genetic variation, and using the improved predictive ability of the marker additive genetic effects for future predictions, gave more accurate estimates even for the single trial analysis than was the case if the total genetic variation was not partitioned (<em>i.e.</em>, when fitting the <em>standard</em> model, which excluded the residual genetic and therefore nonadditive, variation). Improvements of up to 5.7% in predictive ability were found. Further work is required to determine whether improved prediction can be achieved by using the total genetic effect (as opposed to the total genetic effect from markers) of a line from the <em>partitioned</em> model for phenotypic evaluation. The value of using the total genetic effect (additive plus nonadditive) for phenotypic evaluation would depend on the impact of the nonadditive proportion. If the nonadditive proportion of variation was reasonably high, the use of the total genetic effect for phenotypic evaluation would be important. However, it is worth noting the total genetic effect does not reflect the potential of a line as a parent as nonadditive effects are not inherited, although it may better predict the commercial viability of a line and may be useful in determining which lines to take forward from a breeding program for elite development.</p><p id="p-105">Using the raw data in a single stage approach has the added advantage that it allows spatial variation to be incorporated into the analysis, enabling joint estimation of all effects, genetic and nongenetic. This is the preferred option, as the precorrection of data necessary in a two stage approach can have undesirable consequences, such as biased estimates of marker effects, and induced correlations between residuals (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2013</a>).</p><p id="p-106">It was evident from the analysis that the <em>standard</em> model had an inflated trial residual (error) term compared to the <em>partitioned</em> model. The <em>partitioned</em> model enabled the additive variation due to markers to be estimated, and it was found to account for approximately 75% of the total genetic variation in each of the trials. In the <em>standard</em> model, the inflation of the trial residual term, while apparent, was not sufficient to explain all of the unaccounted nonadditive genetic variation. This suggests that, in the <em>standard</em> model, some of the omitted nonadditive genetic variation is incorporated into the estimate of the genetic variation due to the markers, and they do not therefore reflect purely additive variation. This perhaps explains why, in the <em>standard</em> model, the marker effects are not as good for prediction of GEBVs, and this model showed higher MSE of prediction. The outcome is that estimates of marker effects from the <em>partitioned</em> model should be less biased, be more likely to reflect additive variation, and therefore lead to better estimation for future prediction.</p><p id="p-107">The single trial RR-BLUP <em>partitioned</em> model was extended to enable a multi-environment approach to analysis. Analyzing trials under the <em>partitioned</em> RR-BLUP model in a MET setting extends a phenotypic MET model in an intuitive manner. In the GS model, we implicitly included both a main effect for markers, and a marker by trial interaction term. We found that a main marker effect in an analysis of trials in a MET was a useful predictor, even when a strong marker by trial interaction was present. These main marker effects seem to reflect a more ‘stable’ proportion of the marker, and were shown to have a predictive ability slightly superior to a single trial analysis. Presumably, the addition of more trials would improve the robustness of the main marker effect to G × E.</p><p id="p-108">The RR-MET (<em>partitioned</em> and <em>standard</em>) performed well particularly where there was a larger environmental influence. In the example data set, in the 2011 trial, improvements of the MET over the single trial analysis of as much as 11.4% were seen, probably due to the inability of a single year analysis to account for the marker by trial variation. The poorer performance found when using 2011 marker effects to predict 2010 supports the observation of <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Ly <em>et al.</em> (2013)</a> that the presence of G × E reduces the ability to make predictions in locations where no evaluations have previously been done. MET analysis uses the correlation between trials to improve prediction of lines (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Smith <em>et al.</em> 2005</a>). Where the environmental influence was lower, as in the 2010 trial, the <em>partitioned</em> single trial model performed well, with similar predictive ability to the <em>partitioned</em> MET model, but with lower MSE in the latter. As in <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Guo <em>et al.</em> (2013)</a>, gains here are attributable to the more accurate estimates of trial-specific marker effects through utilizing genetic correlations. In our cross-validation approach, we excluded validation lines from both trials, and found that, in 2011, where environmental influence was large, the MET was superior to the single trial analysis. <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Burgueno <em>et al.</em> (2012)</a> and <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Guo <em>et al.</em> (2013)</a> also looked at MET verses single trial analysis and included cross-validation schemes (referred to as CV1 in both papers), which also excluded validation lines from all environments. Our findings are contrary to those of <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Burgueno <em>et al.</em> (2012)</a>, who found no improvement in predictability of a MET over a single trial analysis, but support the findings of <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Guo <em>et al.</em> (2013</a>), who found similar average gains in prediction accuracies of up to 10%.</p><p id="p-109">In terms of using a subset of the markers in a low density GS approach, similar predictive ability could be gained with a much lower number of markers in the <em>partitioned</em> MET model, particularly in 2011. The results suggest that the <em>partitioned</em> model increased the accuracy of the marker effects with further gains to be had by using the MET model, particularly if the environmental influence on results is high (as in 2011). When examining the value of predictive ability of random subsets of markers, some subsets were superior to others (results not shown). Examining markers that are consistently found in the highly predictive subsets may lead to suitable choices for a MARS approach, and this may be worth exploring when considering the practical and optimal use of markers in GS in crops. Finally, as <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Heffner <em>et al.</em> (2011a</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">2011b</a>) have found, reducing the number of lines in a training population decreases the predictive ability. However, our observations suggest that the <em>partitioned</em> model goes some way to alleviating this effect.</p><p id="p-110">In summary, the <em>partitioned</em> MET model used here is a single linear mixed model that incorporates trial, residual genetic-by-trial interactions, and trial-specific field and randomization-based terms, in a random RR-BLUP setting for marker effects and their interaction with trials. The MET model offers a viable, flexible addition to the GS tool box.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-111">The authors thank everybody who helped with the planting, harvesting, and sampling, particularly Reza Shafiei, who supervised much of the phenotyping, and Nicola Uzrek, who played a major role in coordinating plant and seed maintenance and health. Thanks to the Biotechnology and Biological Sciences Research Council (BBSRC), who financially supported this work through the BBSRC Sustainable Bioenergy Centre (BSBEC) grant number BBG0162321, and to the Scottish Government Centre of Excellence in Climate Change (CXC, 2011–2016). C.H. is a Royal Society Wolfson Research Merit Award holder.</p><p id="p-112">Author contributions: H.O. designed the field phase in 2011, developed the statistical models and conducted the statistical analysis, interpreted the statistical data, and drafted the manuscript. B.C. and R.T. developed the statistical models, and assisted with the statistical analysis. J.C. designed the field phase in 2010, assembled the plant material, and coordinated sowing and growing of both the 2010 and 2011 trials. R.W. provided the marker data used for genetic analysis. C.H. and R.W. jointly conceived and won the research funding to conduct the work, and oversaw design and implementation of both seasons’ experimental data collection; both contributed to writing the manuscript. All authors read and approved the final manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-36"><p id="p-113">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027524/-/DC1</a></p></li><li class="fn" id="fn-37"><p id="p-114"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-10-03"><span class="received-label">Received </span>October 3, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-03-05"><span class="accepted-label">Accepted </span>March 5, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Oakey <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.1" data-doi="10.1109/TAC.1974.1100705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akaike</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">New look at statistical-model identification.</span> IEEE <abbr class="cit-jnl-abbrev">Transactions on Automatic Control.</abbr> <span class="cit-vol">AC19</span>: <span class="cit-fpage">716</span>–<span class="cit-lpage">723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransactions%2Bon%2BAutomatic%2BControl.%26rft.volume%253D19%26rft.spage%253D716%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTAC.1974.1100705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TAC.1974.1100705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.2" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1313.3"><div class="cit-metadata"><ol class="cit-ed-list"><li><span class="cit-ed"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite> (Editors), <span class="cit-pub-date">2009</span> <span class="cit-source"><em>ASReml R-reference manual</em></span>, <span class="cit-publ-name">VSN International Ltd.</span>, <span class="cit-publ-loc">Hemel Hempstead, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.4" data-doi="10.1038/ng.2447"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, L. Ramsay, N. Stein <em>et al</em>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">Natural variation in a homolog of <em>Antirrhinum</em> CENTRORADIALIS contributed to spring growth habit and environmental adaptation in cultivated barley.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1388</span>–<span class="cit-lpage">1392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DComadran%26rft.auinit1%253DJ.%26rft.volume%253D44%26rft.issue%253D12%26rft.spage%253D1388%26rft.epage%253D1392%26rft.atitle%253DNatural%2Bvariation%2Bin%2Ba%2Bhomolog%2Bof%2BAntirrhinum%2BCENTRORADIALIS%2Bcontributed%2Bto%2Bspring%2Bgrowth%2Bhabit%2Band%2Benvironmental%2Badaptation%2Bin%2Bcultivated%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2447%26rft_id%253Dinfo%253Apmid%252F23160098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2447&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23160098&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.5" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEzMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.6" data-doi="10.1080/15427528.2011.558767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahuku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection and prediction in plant breeding.</span> <abbr class="cit-jnl-abbrev">J. Crop Improv.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCrop%2BImprov.%26rft.volume%253D25%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15427528.2011.558767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15427528.2011.558767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-2" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.7" data-doi="10.2307/2533991"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Spatial analysis of multi-environment early generation trials.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D54%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2533991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2533991&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.8" data-doi="10.1198/108571106X154443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombes</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">On the design of early generation variety trials with correlated data.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D11%26rft.spage%253D381%26rft_id%253Dinfo%253Adoi%252F10.1198%252F108571106X154443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/108571106X154443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000242477500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.9" data-doi="10.1371/journal.pone.0087666"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Da</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mixed model methods for genomic prediction and variance component estimation of additive and dominance effects using SNP markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e87666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De87666%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0087666%26rft_id%253Dinfo%253Apmid%252F24498162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0087666&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24498162&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Boer</span>  <span class="cit-name-given-names">I. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoeschele</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Genetic evaluation methods for populations with dominance and inbreeding.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D86%26rft.spage%253D245%26rft_id%253Dinfo%253Apmid%252F24193467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24193467&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.11" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEzMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.12" data-doi="10.1016/j.tplants.2014.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desta</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection: genome-wide prediction in plant improvement.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D592%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2014.05.006%26rft_id%253Dinfo%253Apmid%252F24970707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2014.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24970707&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.13" data-doi="10.2307/1400446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Accounting for natural and extraneous variation in the analysis of field experiments.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D2%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1400446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1400446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.14" data-doi="10.1534/g3.112.005066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basten</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accuracy of across-environment genome wide prediction in maize nested association mapping populations.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DGuo%26rft.auinit1%253DZ.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D263%26rft.epage%253D272%26rft.atitle%253DAccuracy%2Bof%2BAcross-Environment%2BGenome-Wide%2BPrediction%2Bin%2BMaize%2BNested%2BAssociation%2BMapping%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.005066%26rft_id%253Dinfo%253Apmid%252F23390602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvMi8yNjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTMxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.15" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernado</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTMxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.16" data-doi="10.1534/genetics.113.152207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic BLUP decoded: a look into the black box of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152207%26rft_id%253Dinfo%253Apmid%252F23640517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEzMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-other" id="cit-6.5.1313.17"><div class="cit-metadata unstructured">Hastie, T., R. Tibshirani, B. Narasimhan, and G. Chu, 2014 impute: Imputation for microarray data. R package version 1.36.0. Available at: <a href="https://bioconductor.org/packages/release/bioc/html/impute.html">https://bioconductor.org/packages/release/bioc/html/impute.html</a>. Accessed: April 21, 2014.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant breeding with genomic selection: gain per unit time and cost.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: 1681–1690.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.19" data-doi="10.2135/cropsci2011.05.0253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic selection accuracy for grain quality traits in biparental wheat populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">2597</span>–<span class="cit-lpage">2606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D2597%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.05.0253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.05.0253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000295839200031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.20" data-doi="10.3835/plantgenome.2010.12.0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic selection accuracy using multifamily prediction models in a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome.2010.12.0029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome.2010.12.0029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.21" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.22" data-doi="10.1093/bfgp/elq001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: from theory to practice.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felq001%26rft_id%253Dinfo%253Apmid%252F20156985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYmZncCI7czo1OiJyZXNpZCI7czo3OiI5LzIvMTY2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEzMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.23" data-doi="10.1016/j.tibtech.2013.06.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jonas</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Koning</span>  <span class="cit-name-given-names">D.-J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Does genomic selection have a future in plant breeding?</span> <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D497%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2013.06.003%26rft_id%253Dinfo%253Apmid%252F23870753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2013.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23870753&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.24" data-doi="10.2135/cropsci2006.08.0540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eccleston</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The accuracy of varietal selection using factor analytic models for multi-environment plant breeding trials.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1063</span>–<span class="cit-lpage">1070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DKelly%26rft.auinit1%253DA.%2BM.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1063%26rft.epage%253D1070%26rft.atitle%253DThe%2BAccuracy%2Bof%2BVarietal%2BSelection%2BUsing%2BFactor%2BAnalytic%2BModels%2Bfor%2BMulti-Environment%2BPlant%2BBreeding%2BTrials%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.08.0540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.08.0540&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lado</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matus</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inostroza</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Increased genomic prediction accuracy in wheat breeding through spatial adjustment of field trial data.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">2015</span>–<span class="cit-lpage">2113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D2015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.26" data-doi="10.2135/cropsci2012.11.0653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabbi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melaku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakare</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Relatedness and genotype × environment interaction affect prediction accuracies in genomic selection: a study in cassava.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">1312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D1312%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.11.0653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.11.0653&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.27" data-doi="10.2135/cropsci2012.02.0112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">H.-J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomewide selection verses marker-assisted recurrent selection to improve grain yield and stover-quality traits for cellulosic ethanol in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D58%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.02.0112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.02.0112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTMxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.29" data-doi="10.1534/genetics.114.171322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Munoz</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gexan</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Unravelling additive from nonadditive effects using genomic relationship matrices.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.171322%26rft_id%253Dinfo%253Apmid%252F25324160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE3NTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTMxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.30" data-doi="10.1186/1754-6834-6-185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oakey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shafiei</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uzrek</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of crop cultivars with consistently high lignocellulosic sugar release requires the use of appropriate statistical design and modelling.</span> <abbr class="cit-jnl-abbrev">Biotechnol. Biofuels</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnol.%2BBiofuels%26rft.volume%253D6%26rft.spage%253D185%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1754-6834-6-185%26rft_id%253Dinfo%253Apmid%252F24359577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1754-6834-6-185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24359577&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.5.1313.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silvey</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weatherup</span>  <span class="cit-name-given-names">S. T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Variability of yields of cereal varieties in U. K. trials.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">238</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D89%26rft.spage%253D238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.32" data-doi="10.2135/cropsci2011.11.0592"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estaghvirou</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordillo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficient computation of ridge-regression best linear unbiased prediction in genomic selection in plant breeding.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1093</span>–<span class="cit-lpage">1104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1093%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.11.0592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.11.0592&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-6.5.1313.33"><div class="cit-metadata unstructured">R Core Team, 2015 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. Available at: <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-2" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.34" data-doi="10.1111/j.0006-341X.2001.01138.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Analyzing variety by environment data using multiplicative mixed models and adjustments for spatial field trend.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">1138</span>–<span class="cit-lpage">1147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DSmith%26rft.auinit1%253DA.%26rft.volume%253D57%26rft.issue%253D4%26rft.spage%253D1138%26rft.epage%253D1147%26rft.atitle%253DAnalyzing%2Bvariety%2Bby%2Benvironment%2Bdata%2Busing%2Bmultiplicative%2Bmixed%2Bmodels%2Band%2Badjustments%2Bfor%2Bspatial%2Bfield%2Btrend.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0006-341X.2001.01138.x%26rft_id%253Dinfo%253Apmid%252F11764254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0006-341X.2001.01138.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11764254&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174956800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.35" data-doi="10.1017/S0021859605004995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The analysis of crop cultivar breeding and evaluations trials: an overview of current mixed model approaches.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D143%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021859605004995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021859605004995&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.36" data-doi="10.1186/1297-9686-41-53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wooliams</span>  <span class="cit-name-given-names">J. a.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ødegard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Persistence of accuracy of genome-wide breeding values over generations when including a polygenic effect.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DSolberg%26rft.auinit1%253DT.%2BR.%26rft.volume%253D41%26rft.spage%253D53%26rft.epage%253D53%26rft.atitle%253DPersistence%2Bof%2Baccuracy%2Bof%2Bgenome-wide%2Bbreeding%2Bvalues%2Bover%2Bgenerations%2Bwhen%2Bincluding%2Ba%2Bpolygenic%2Beffect.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-53%26rft_id%253Dinfo%253Apmid%252F20040081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-53&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20040081&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.37" data-doi="10.3168/jds.2008-1929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stranden</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Technical note: derivation of equivalent computing algorithms for genomic predictions and reliabilities of animal merit.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">2971</span>–<span class="cit-lpage">2975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DStranden%26rft.auinit1%253DI.%26rft.volume%253D92%26rft.issue%253D6%26rft.spage%253D2971%26rft.epage%253D2975%26rft.atitle%253DTechnical%2Bnote%253A%2BDerivation%2Bof%2Bequivalent%2Bcomputing%2Balgorithms%2Bfor%2Bgenomic%2Bpredictions%2Band%2Breliabilities%2Bof%2Banimal%2Bmerit%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1929%26rft_id%253Dinfo%253Apmid%252F19448030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448030&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Regression shrinkage and selection via the LASSO.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc., B</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%252C%2BB%26rft.volume%253D58%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.39" data-doi="10.1186/1753-6561-6-S2-S13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">P.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X.-D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lui</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparison of five methods for genomic breeding value estimation for the common dataser of the 15th QTL-MAS workshop.</span> <abbr class="cit-jnl-abbrev">BMC Proc.</abbr> <span class="cit-vol">6</span>(<span class="cit-supplement">Suppl 2</span>): <span class="cit-fpage">S13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BProc.%26rft.volume%253D6%26rft.spage%253DS13%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1753-6561-6-S2-S13%26rft_id%253Dinfo%253Apmid%252F22640547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1753-6561-6-S2-S13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22640547&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1313.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.40" data-doi="10.1534/genetics.113.150078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehermeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auinger</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide prediction of traits with different genetic architecture through efficient variable selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150078%26rft_id%253Dinfo%253Apmid%252F23934883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzIvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEzMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.41" data-doi="10.1534/genetics.108.098277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernado</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a Barley case study.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098277%26rft_id%253Dinfo%253Apmid%252F19299342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEzMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1313.42" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Regularization and variable selection via the elastic net.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Ser. A Stat. Soc.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSer.%2BA%2BStat.%2BSoc.%26rft.volume%253D67%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/5/1313.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/5/1305" title="Characterization of Chicken MMP13 Expression and Genetic Effect on Egg Production Traits of Its Promoter Polymorphisms" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/5/1327" title="The SEB-1 Transcription Factor Binds to the STRE Motif in Neurospora crassa and Regulates a Variety of Cellular Processes Including the Stress Response and Reserve Carbohydrate Metabolism" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/5"><span>Volume 6 Issue 5, May 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/5.cover-source.jpg?itok=8SH0hKOw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1313.long" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IyUVzeu4Fabpbll4GTgcrFcFbz3ziY-wk-DTiB9gQ-4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/5/1313" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/5/1313" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/5/1313" class="active" data-icon-position="" data-hide-link-title="0">Genomic Selection in Multi-environment Crop Trials</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460359" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-bPsrLqoqD7DUO4s3ROTd3HAjW66jK6TeE9DBBqT9HBo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/5/1313.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/5/1313" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-kyGJd9uvlyAbmF2AKVSYkBQuK3GCWifqv6NQDuKCxEA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/5/1313" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/5/1313.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460359" id="node460359--2" data-pisa="ggg;6/5/1313" data-pisa-master="ggg;g3.116.027524" data-apath="/ggg/6/5/1313.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Selection in Multi-environment Crop Trials</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Helena Oakey</span>, <span class="highwire-citation-author" data-delta="1">Brian Cullis</span>, <span class="highwire-citation-author" data-delta="2">Robin Thompson</span>, <span class="highwire-citation-author" data-delta="3">Jordi Comadran</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-1808-8130" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Claire Halpin</span> and <span class="highwire-citation-author" data-delta="5">Robbie Waugh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1313-1326;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027524 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460359--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Helena Oakey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Plant Sciences, University of Dundee at the James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helena%2BOakey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oakey%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelena%2BOakey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Brian Cullis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>National Institute for Applied Statistics Research Australia, University of Wollongong, NSW, 2522, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BCullis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cullis%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BCullis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Robin Thompson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rothamsted Research, Harpenden, Hertfordshire AL5 3JQ, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robin%2BThompson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thompson%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobin%2BThompson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jordi Comadran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Cell and Molecular Sciences, The James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jordi%2BComadran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Comadran%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJordi%2BComadran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Claire Halpin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Plant Sciences, University of Dundee at the James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BHalpin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Halpin%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClaire%2BHalpin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1808-8130" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Claire Halpin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:c.halpin@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">c.halpin@dundee.ac.uk</a>
<a href="mailto:r.waugh@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">r.waugh@dundee.ac.uk</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Robbie Waugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Plant Sciences, University of Dundee at the James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Cell and Molecular Sciences, The James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robbie%2BWaugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waugh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobbie%2BWaugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:c.halpin@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">c.halpin@dundee.ac.uk</a>
<a href="mailto:r.waugh@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">r.waugh@dundee.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/5/1313" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/5/1313&amp;title=Genomic%20Selection%20in%20Multi-environment%20Crop%20Trials" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/5/1313&amp;title=Genomic%20Selection%20in%20Multi-environment%20Crop%20Trials" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/5/1313&amp;title=Genomic%20Selection%20in%20Multi-environment%20Crop%20Trials" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/5/1313&amp;text=Genomic%20Selection%20in%20Multi-environment%20Crop%20Trials" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/5/1313&amp;title=Genomic%20Selection%20in%20Multi-environment%20Crop%20Trials" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/5/1313&amp;t=Genomic%20Selection%20in%20Multi-environment%20Crop%20Trials" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/5/1313&amp;title=Genomic%20Selection%20in%20Multi-environment%20Crop%20Trials" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/5/1313&amp;title=Genomic%20Selection%20in%20Multi-environment%20Crop%20Trials" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460359/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460359" id="node460359--3" data-pisa="ggg;6/5/1313" data-pisa-master="ggg;g3.116.027524" data-apath="/ggg/6/5/1313.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Selection in Multi-environment Crop Trials</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Helena Oakey</span>, <span class="highwire-citation-author" data-delta="1">Brian Cullis</span>, <span class="highwire-citation-author" data-delta="2">Robin Thompson</span>, <span class="highwire-citation-author" data-delta="3">Jordi Comadran</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-1808-8130" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Claire Halpin</span> and <span class="highwire-citation-author" data-delta="5">Robbie Waugh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1313-1326;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027524 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460359--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Helena Oakey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Plant Sciences, University of Dundee at the James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Helena%2BOakey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oakey%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHelena%2BOakey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Brian Cullis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>National Institute for Applied Statistics Research Australia, University of Wollongong, NSW, 2522, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BCullis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cullis%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BCullis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Robin Thompson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Rothamsted Research, Harpenden, Hertfordshire AL5 3JQ, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robin%2BThompson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thompson%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobin%2BThompson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jordi Comadran </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Cell and Molecular Sciences, The James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jordi%2BComadran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Comadran%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJordi%2BComadran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Claire Halpin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Plant Sciences, University of Dundee at the James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claire%2BHalpin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Halpin%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClaire%2BHalpin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1808-8130" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Claire Halpin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:c.halpin@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">c.halpin@dundee.ac.uk</a>
<a href="mailto:r.waugh@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">r.waugh@dundee.ac.uk</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Robbie Waugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Division of Plant Sciences, University of Dundee at the James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Cell and Molecular Sciences, The James Hutton Institute, Invergowrie, Dundee DD2 5DA, Scotland, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robbie%2BWaugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waugh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARobbie%2BWaugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:c.halpin@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">c.halpin@dundee.ac.uk</a>
<a href="mailto:r.waugh@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">r.waugh@dundee.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1313&amp;rft.epage=1326&amp;rft.atitle=Genomic%20Selection%20in%20Multi-environment%20Crop%20Trials&amp;rft.volume=6&amp;rft.issue=5&amp;rft.date=2016-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Oakey%2C+Helena&amp;rft.au=Cullis%2C+Brian&amp;rft.au=Thompson%2C+Robin&amp;rft.au=Comadran%2C+Jordi&amp;rft.au=Halpin%2C+Claire&amp;rft.au=Waugh%2C+Robbie"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460359/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460359/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460359/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460359/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460359/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460359/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460359/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460359/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460359/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460359/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460359/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460359/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/5/1313" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/5/1313.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/5/1313.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/5/1313.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/5/1313.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
