<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/5/1287" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/5/1287.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/5/1287.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/5/1287.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/5/1287/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/5/1287.atom" />
<meta name="HW.pisa" content="ggg;6/5/1287" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Conflation of Short Identity-by-Descent Segments Bias Their Inferred Length Distribution" />
<meta name="DC.Identifier" content="10.1534/g3.116.027581" />
<meta name="DC.Date" content="2016-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Chiang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Identity-by-descent (IBD) is a fundamental concept in genetics with many applications. In a common definition, two haplotypes are said to share an IBD segment if that segment is inherited from a recent shared common ancestor without intervening recombination. Segments several cM long can be efficiently detected by a number of algorithms using high-density SNP array data from a population sample, and there are currently efforts to detect shorter segments from sequencing. Here, we study a problem of identifiability: because existing approaches detect IBD based on contiguous segments of identity-by-state, inferred long segments of IBD may arise from the conflation of smaller, nearby IBD segments. We quantified this effect using coalescent simulations, finding that significant proportions of inferred segments 1–2 cM long are results of conflations of two or more shorter segments, each at least 0.2 cM or longer, under demographic scenarios typical for modern humans for all programs tested. The impact of such conflation is much smaller for longer (&gt; 2 cM) segments. This biases the inferred IBD segment length distribution, and so can affect downstream inferences that depend on the assumption that each segment of IBD derives from a single common ancestor. As an example, we present and analyze an estimator of the de novo mutation rate using IBD segments, and demonstrate that unmodeled conflation leads to underestimates of the ages of the common ancestors on these segments, and hence a significant overestimate of the mutation rate. Understanding the conflation effect in detail will make its correction in future methods more tractable." />
<meta name="DC.Contributor" content="Charleston W. K. Chiang" />
<meta name="DC.Contributor" content="Peter Ralph" />
<meta name="DC.Contributor" content="John Novembre" />
<meta name="article:published_time" content="2016-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Conflation of Short Identity-by-Descent Segments Bias Their Inferred Length Distribution" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Identity-by-descent (IBD) is a fundamental concept in genetics with many applications. In a common definition, two haplotypes are said to share an IBD segment if that segment is inherited from a recent shared common ancestor without intervening recombination. Segments several cM long can be efficiently detected by a number of algorithms using high-density SNP array data from a population sample, and there are currently efforts to detect shorter segments from sequencing. Here, we study a problem of identifiability: because existing approaches detect IBD based on contiguous segments of identity-by-state, inferred long segments of IBD may arise from the conflation of smaller, nearby IBD segments. We quantified this effect using coalescent simulations, finding that significant proportions of inferred segments 1–2 cM long are results of conflations of two or more shorter segments, each at least 0.2 cM or longer, under demographic scenarios typical for modern humans for all programs tested. The impact of such conflation is much smaller for longer (&amp;gt; 2 cM) segments. This biases the inferred IBD segment length distribution, and so can affect downstream inferences that depend on the assumption that each segment of IBD derives from a single common ancestor. As an example, we present and analyze an estimator of the &lt;i&gt;de novo&lt;/i&gt; mutation rate using IBD segments, and demonstrate that unmodeled conflation leads to underestimates of the ages of the common ancestors on these segments, and hence a significant overestimate of the mutation rate. Understanding the conflation effect in detail will make its correction in future methods more tractable.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/05/01" />
<meta name="citation_mjid" content="ggg;6/5/1287" />
<meta name="citation_id" content="6/5/1287" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/5/1287" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/5/1287.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/5/1287.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/5/1287.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.027581" />
<meta name="citation_pmid" content="26935417" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1287" />
<meta name="citation_lastpage" content="1296" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Charleston W. K. Chiang" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095" />
<meta name="citation_author_email" content="chiang82@ucla.edu" />
<meta name="citation_author_email" content="jnovembre@uchicago.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0668-7865" />
<meta name="citation_author" content="Peter Ralph" />
<meta name="citation_author_institution" content="Department of Molecular and Computational Biology, University of Southern California, Los Angeles, California 90089" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9459-6866" />
<meta name="citation_author" content="John Novembre" />
<meta name="citation_author_institution" content="Department of Human Genetics, University of Chicago, Illinois 60637" />
<meta name="citation_author_email" content="chiang82@ucla.edu" />
<meta name="citation_author_email" content="jnovembre@uchicago.edu" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=A. Albrechtsen;citation_author=T. Sand Korneliussen;citation_author=I. Moltke;citation_author=T. van Overseem Hansen;citation_author=F. C. Nielsen;citation_title=Relatedness mapping and tracts of relatedness for genome-wide data in the presence of linkage disequilibrium.;citation_pages=266-274;citation_volume=33;citation_year=2009;citation_issue=3;citation_pmid=19025785;citation_doi=10.1002/gepi.20378" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Albrechtsen;citation_author=I. Moltke;citation_author=R. Nielsen;citation_title=Natural selection and the distribution of identity-by-descent in the human genome.;citation_volume=186;citation_year=2010;citation_pmid=20592267;citation_doi=10.1534/genetics.110.113977" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=A fast, powerful method for detecting identity by descent.;citation_pages=173-182;citation_volume=88;citation_year=2011;citation_issue=2;citation_pmid=21310274;citation_doi=10.1016/j.ajhg.2011.01.010" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=Detecting identity by descent and estimating genotype error rates in sequence data.;citation_pages=840-851;citation_volume=93;citation_year=2013;citation_pmid=24207118;citation_doi=10.1016/j.ajhg.2013.09.014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=Improving the accuracy and efficiency of identity-by-descent detection in population data.;citation_volume=194;citation_year=2013;citation_pmid=23535385;citation_doi=10.1534/genetics.113.150029" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=High-resolution detection of identity by descent in unrelated individuals.;citation_pages=526-539;citation_volume=86;citation_year=2010;citation_issue=4;citation_pmid=20303063;citation_doi=10.1016/j.ajhg.2010.02.021" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Haplotype phasing: existing methods and new developments.;citation_pages=703-714;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=21921926;citation_doi=10.1038/nrg3054" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. D. Campbell;citation_author=J. X. Chong;citation_author=M. Malig;citation_author=A. Ko;citation_author=B. L. Dumont;citation_title=Estimating the human mutation rate using autozygosity in a founder population.;citation_pages=1277-1281;citation_volume=44;citation_year=2012;citation_issue=11;citation_pmid=23001126;citation_doi=10.1038/ng.2418" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=N. H. Chapman;citation_author=E. A. Thompson;citation_title=A model for the length of tracts of identity by descent in finite random mating populations.;citation_pages=141-150;citation_volume=64;citation_year=2003;citation_issue=2;citation_pmid=12948676;citation_doi=10.1016/S0040-5809(03)00071-6" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. K. Chen;citation_author=P. Marjoram;citation_author=J. D. Wall;citation_title=Fast and flexible simulation of DNA sequence data.;citation_volume=19;citation_year=2009;citation_pmid=19029539;citation_doi=10.1101/gr.083634.108" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=E. Y. Durand;citation_author=N. Eriksson;citation_author=C. Y. McLean;citation_title=Reducing pervasive false-positive identical-by-descent segments detected by large-scale pedigree analysis.;citation_volume=31;citation_year=2014;citation_pmid=24784137;citation_doi=10.1093/molbev/msu151" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=R. C. Griffiths;citation_author=P. Marjoram;citation_title=Ancestral inference from samples of DNA sequences with recombination.;citation_pages=479-502;citation_volume=3;citation_year=1996;citation_issue=4;citation_pmid=9018600;citation_doi=10.1089/cmb.1996.3.479" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Gusev;citation_author=J. K. Lowe;citation_author=M. Stoffel;citation_author=M. J. Daly;citation_author=D. Altshuler;citation_title=Whole population, genome-wide mapping of hidden relatedness.;citation_volume=19;citation_year=2009;citation_pmid=18971310;citation_doi=10.1101/gr.081398.108" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=A. Gusev;citation_author=E. E. Kenny;citation_author=J. K. Lowe;citation_author=J. Salit;citation_author=R. Saxena;citation_title=DASH: a method for identical-by-descent haplotype mapping uncovers association with recent variation.;citation_pages=706-717;citation_volume=88;citation_year=2011;citation_issue=6;citation_pmid=21620352;citation_doi=10.1016/j.ajhg.2011.04.023" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Gusev;citation_author=M. J. Shah;citation_author=E. E. Kenny;citation_author=A. Ramachandran;citation_author=J. K. Lowe;citation_title=Low-pass genome-wide sequencing and variant inference using identity-by-descent in an isolated human population.;citation_volume=190;citation_year=2012;citation_pmid=22135348;citation_doi=10.1534/genetics.111.134874" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=L. Han;citation_author=M. Abney;citation_title=Identity by descent estimation with dense genome-wide genotype data.;citation_pages=557-567;citation_volume=35;citation_year=2011;citation_issue=6;citation_pmid=21769932" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Hum. Genet.;citation_author=L. Han;citation_author=M. Abney;citation_title=Using identity by descent estimation with dense genotype data to detect positive selection.;citation_pages=205-211;citation_volume=21;citation_year=2013;citation_pmid=22781100;citation_doi=10.1038/ejhg.2012.148" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. Harris;citation_author=R. Nielsen;citation_title=Inferring demographic history from a spectrum of shared haplotype lengths.;citation_pages=e1003521;citation_volume=9;citation_year=2013;citation_pmid=23754952;citation_doi=10.1371/journal.pgen.1003521" />
<meta name="citation_reference" content="citation_title=Oxford Surveys in Evolutionary Biology;citation_title=Gene genealogies and the coalescent process;citation_author=R. R. Hudson;citation_pages=1-44;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Kong;citation_author=G. Masson;citation_author=M. L. Frigge;citation_author=A. Gylfason;citation_author=P. Zusmanovich;citation_title=Detection of sharing by descent, long-range phasing and haplotype imputation.;citation_pages=1068-1075;citation_volume=40;citation_year=2008;citation_issue=9;citation_pmid=19165921;citation_doi=10.1038/ng.216" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. R. Nelson;citation_author=D. Wegmann;citation_author=M. G. Ehm;citation_author=D. Kessner;citation_author=P. St Jean;citation_title=An abundance of rare functional variants in 202 drug target genes sequenced in 14,002 people.;citation_volume=337;citation_year=2012;citation_pmid=22604722;citation_doi=10.1126/science.1217876" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. F. Palamara;citation_author=T. Lencz;citation_author=A. Darvasi;citation_author=I. Pe’er;citation_title=Length distributions of identity by descent reveal fine-scale demographic history.;citation_pages=809-822;citation_volume=91;citation_year=2012;citation_issue=5;citation_pmid=23103233;citation_doi=10.1016/j.ajhg.2012.08.030" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=P. F. Palamara;citation_author=L. Francioli;citation_author=G. Genovese;citation_author=P. Wilton;citation_author=A. Gusev;citation_title=Leveraging distant relatedness to quantify human mutation and gene conversion rates.;citation_pages=775-789;citation_volume=97;citation_year=2015;citation_pmid=26581902;citation_doi=10.1016/j.ajhg.2015.10.006" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. E. Powell;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Reconciling the analysis of IBD and IBS in complex trait studies.;citation_pages=800-805;citation_volume=11;citation_year=2010;citation_issue=11;citation_pmid=20877324;citation_doi=10.1038/nrg2865" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=P. Ralph;citation_author=G. Coop;citation_title=The geography of recent genetic ancestry across Europe.;citation_pages=e1001555;citation_volume=11;citation_year=2013;citation_pmid=23667324;citation_doi=10.1371/journal.pbio.1001555" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. A. Tennessen;citation_author=A. W. Bigham;citation_author=T. D. O’Connor;citation_author=W. Fu;citation_author=E. E. Kenny;citation_title=Evolution and functional impact of rare coding variation from deep sequencing of human exomes.;citation_volume=337;citation_year=2012;citation_pmid=22604720;citation_doi=10.1126/science.1219240" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. A. Thompson;citation_title=Identity by Descent: Variation in Meiosis, Across Genomes, and in Populations;citation_pages=301-326;citation_volume=194;citation_year=2013;citation_issue=2;citation_pmid=23733848;citation_doi=10.1534/genetics.112.148825" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=O. Zuk;citation_author=E. Hechter;citation_author=S. R. Sunyaev;citation_author=E. S. Lander;citation_title=The mystery of missing heritability: Genetic interactions create phantom heritability.;citation_volume=109;citation_year=2012;citation_pmid=22223662;citation_doi=10.1073/pnas.1119675109" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Conflation of Short Identity-by-Descent Segments Bias Their Inferred Length Distribution" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
<meta name="twitter:description" content="Identity-by-descent (IBD) is a fundamental concept in genetics with many applications. In a common definition, two haplotypes are said to share an IBD segment if that segment is inherited from a recent shared common ancestor without intervening recombination. Segments several cM long can be efficiently detected by a number of algorithms using high-density SNP array data from a population sample, and there are currently efforts to detect shorter segments from sequencing. Here, we study a problem of identifiability: because existing approaches detect IBD based on contiguous segments of identity-by-state, inferred long segments of IBD may arise from the conflation of smaller, nearby IBD segments. We quantified this effect using coalescent simulations, finding that significant proportions of inferred segments 1–2 cM long are results of conflations of two or more shorter segments, each at least 0.2 cM or longer, under demographic scenarios typical for modern humans for all programs tested. The impact of such conflation is much smaller for longer (&gt; 2 cM) segments. This biases the inferred IBD segment length distribution, and so can affect downstream inferences that depend on the assumption that each segment of IBD derives from a single common ancestor. As an example, we present and analyze an estimator of the de novo mutation rate using IBD segments, and demonstrate that unmodeled conflation leads to underestimates of the ages of the common ancestors on these segments, and hence a significant overestimate of the mutation rate. Understanding the conflation effect in detail will make its correction in future methods more tractable." />
<meta name="og-title" property="og:title" content="Conflation of Short Identity-by-Descent Segments Bias Their Inferred Length Distribution" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/5/1287" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Identity-by-descent (IBD) is a fundamental concept in genetics with many applications. In a common definition, two haplotypes are said to share an IBD segment if that segment is inherited from a recent shared common ancestor without intervening recombination. Segments several cM long can be efficiently detected by a number of algorithms using high-density SNP array data from a population sample, and there are currently efforts to detect shorter segments from sequencing. Here, we study a problem of identifiability: because existing approaches detect IBD based on contiguous segments of identity-by-state, inferred long segments of IBD may arise from the conflation of smaller, nearby IBD segments. We quantified this effect using coalescent simulations, finding that significant proportions of inferred segments 1–2 cM long are results of conflations of two or more shorter segments, each at least 0.2 cM or longer, under demographic scenarios typical for modern humans for all programs tested. The impact of such conflation is much smaller for longer (&gt; 2 cM) segments. This biases the inferred IBD segment length distribution, and so can affect downstream inferences that depend on the assumption that each segment of IBD derives from a single common ancestor. As an example, we present and analyze an estimator of the de novo mutation rate using IBD segments, and demonstrate that unmodeled conflation leads to underestimates of the ages of the common ancestors on these segments, and hence a significant overestimate of the mutation rate. Understanding the conflation effect in detail will make its correction in future methods more tractable." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
    <title>Conflation of Short Identity-by-Descent Segments Bias Their Inferred Length Distribution | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CV7XfbeJkSHJpjrQeev9QTFSnYLffpcrgnfZjgD9WV0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460355","apath":"\/ggg\/6\/5\/1287.atom","pisa":"ggg;6\/5\/1287","ac":{"\/ggg\/6\/5\/1287.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/5\/1287.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/5\/1287"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/5\/1287"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/5\/1287":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/qSESTAdE+HISuWqA5qy4mmKVDyUnlLJ+9HIzl6UfegY4NXJSjapjoOzoyyRJSUpoLWgHEJqCjSbE8n2zzUwrePcnu6UrjMQroyXsHbFsQhDl+uNNSdc9ChHdcnhwV4uRrxYxRMmtoIhv8ElHGnCxmwky+XfPgwkaVbouC+FGbTG2fJt\\\\\\\/ZymjRjBrGWRWehp7i\\\\\\\/Ch4HRwYpXwULAr6d57vzcSBtXSwMlNf29WpQ5rzteUUDikxBrk1fRfaDVCtpqfW16yskQQzcl9z6E3kJyj52f8vsFBZp2hxZTMPbGcdbVoEihr0gHhtLm6j3Zqhn2+xofePzBL35ztxpbX1l2gEOlbQtljzsJhMebn3pu9rvLPaovcPlYaAwnuqW6XzQaKXbf8jFV9no2quoe4iQoggPgle6+89e5Q4SAt+T2KFR\\\\\\\/PYXmiWMknT+9yp83wi2z8eXE7PK9Hsw3l2ef8nq8bhjMaAqVHw4fDsVYre\\\\\\\/0zkySLaiuA2Vpjm3jqDkuHTUFvOr\\\\\\\/v5QJSOL7FgijF8f3bYXoCaAuVxv\\\\\\\/EZn3a9BsH96oZpOBRhleAie+oI5f9BAdP8FxsKlYjGH5qnfjZAYgKtqmUHAtxeifwOqk\\\\\\\/r2doFJwPjhy\\\\\\\/hHRcSM0e4bKzyy+lM9WUpvwNTwxfjuuU1z1yoOVIIGUr\\\\\\\/JK0\\\\\\\/7d9mFTeX9Bdh+Sy1vadRScMTJTHR\\\\\\\/Pgu3aFqV6jy9ida4JqRyKTi5q+jFlDY+7JNkdMlejontc9yNXcfBp2MqDZwoaYYAKJvdmVZMN4wV4P+r7oEyjbkg1V9dGEyjeCzNR1kLwuqxkXzKMAYbPuEZTdhr0Nw4ncGSVztH6y1kBkk1YzvWmSpzoJHoHZYNWmmtEnxceggwSFkON6pTUFY+ReHbad6VY6QVRZpqcbES46nBc4vUqVGed0LhEFcxL4gIdD3BBD6QcE2iidf+FhTPSgrK3okOVy7hWqP2w0VmSJgCTuYMbEGQFjXe59V9CI62i9Dez8TymRVUCDDaFi3lK6VU6LKpo\\\\\\\/nqNM9FIu5RJg\\\\\\\/Vy3PMLNygKmrmlTBNWX7FMx68BUMmETqGgGxPTxCeP3s6\\\\\\\/+ePHTdSYBciVK+aBKh\\\\\\\/CZRtD0zabIwsBns2D2Uto9WNRMRrwxGxP5Q0ZPK8z28uWtUe6NTNoxRccu1P8afBlaPm1pu0YYybcudr2Ku1r+bXH7Cp1JW1RHqRqjZ9SjxCkyMaeDEYx5RDwmjsHdGOZ2XHRzIOa8cST68d4tbuRmThLcOguM7tSJM\\\\\\\/0iiRLfMWuNPJ+\\\u0022,\\\u0022iv\\\u0022:\\\u0022WFLZFdkEjw\\\\\\\/1cpNiYIHW0w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220acfa1cfa1498f2a7ea8d1485312b132c0bad67f6adaaf16db53aa9832e43077\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227nXmXUlWaioQh9RGkV8TPe2hhON36GB3T9DUFz2OwwHrtxIyxWvcjmxwY1VKMHf7tocX0jFaAkRiVGwMWftglcw4msezhALYatozhEWKgH6ikWnym3WMhJBzVBziMJPiqHeinwz4FUB+Wdeug1xEx2dCjaYJOBqyeh9THOL1ESwgwgkaCXtprecNGIDAkg\\\\\\\/Ot6y4cih8uaaUBML4OOI8E6XSLCQvTy3CdRgXtapGb6bMgIi7ShVNgvLyLEvy5fhyxAM1v6MS+NoQuk\\\\\\\/OOvm7qhcC7MIaHtof63GGWivI2YqshbDX7njfY0Gk9jyUX9Na3lQ2460cXmXmN8YqVOmt2Yvbi36as61LMQX1s92rlLX+w9CRUvlqqoNwnlmclzmi7UnqGrj9rrkZXhUUeDt9tnDGmf1tfywH8Dul3mMgJoQzWJMckEMxfAIM5uF4xNd61yxWpcVg3RXeYCaOdEmHI+\\\\\\\/sduQP+X5v39omVo0X5sLS061rpGmdc4t5WXutcflDwH3P+yLhs8NU70DI2w3xBCb66m2wbf9GctT81uUfGLGxbkdkyQoVCshCiWvIAZzBfXpiFiaX+FImU5yt9C2aBptMm3dtRLtFRoae74JsMvZwGSsWS\\\\\\\/C5mZeYxruqyFHTb7kLb2AAUkLun\\\\\\\/j1vaJdhiaT7sj9Kh\\\\\\\/uTMZaOW+H\\\\\\\/Nn7OMjJ7NJ5wGZIXR5yCaJRKcyZsWaWSNQUaz+rMyYbllrnBHpFXRbFDAmrCzGkg8KGu5NlP8T5x8v14NgFRjITKRwGLso9tmm\\\\\\\/5CPutrBk1ECPISq1PmizZq3ZOUduhHWPHYeqXSx7Dzr\\\\\\\/Z1pu6ntj2W6wNRTV+WLXiO7vFeD9Dmr+v3zZwDDe2ip5mKZqQhgbqlYq+09Fj43RaU5DIP7htlF1m+oZrVutZFfILGfb8dA6CPh3ldtdAFs5hL4wxx7uSwDYIPK2KhJ+KnmFVt2TPGY5gIrhkNah4\\\\\\\/KahRp2G7MNkPmoBtwmf2HtQs7sAf8psLP7OKVWwGCtVla\\\\\\\/Ib9GqGmFSQgevCA\\\\\\\/cs3wm1YGN6gKBjp7AuZGMJYwV0XfUMaPR18XiL4tZSFdp1pv0rwJUxwVwZh2hsU4QCXSt\\\\\\\/QPXUTPk3UbTjLySdqHL2kZhQYQ3ATSAaSR+vYaPA1khZqCVzaN14zY+X6Se8ZaNxBAjvq982AzQkZTtaapGDnt8CROOzU+B8ydj3\\\\\\\/Is2Ot9l9XSjGsiXrtg3UUZR0v+oQBS7LmjF0QBMNcXw8+ZELbzRaMZ+UapUCNMDy9HlHMWMKUjxv\\\\\\\/ZRtaBKFci6Qk++ysGFtTrc1BGbC9yA3PxlQCkjnaKIA7AWeL0GacWr22PBP\\\\\\\/U05SculXds8+apKrWm4TXaBeo+vZPSB2s8f0niLqx84Fz8qhi7O3iIpZSFykCW\\\\\\\/uQEDOBQrPjYHHEvaq4Ous7B0xp7FJ8QcOCzsGv+hLeoiZSELTkwnWcCZMhQmaxnOdXQ8rnV64tUaBUoExZhP1uytL\\\\\\\/LeW9iVSSDn\\\\\\\/dylTiZa\\\\\\\/JdH57PamyFHJEqijwHBwYx8vHvKnn41k+jdNfTPuRdArScllJMPrqnzYyOhoaQtxZSS43tVSR1fd\\\\\\\/XoMH7y+Xf5q9S+akSzv5b7IfgxdLFW\\\\\\\/CIcsZMoyMG3GhtVyxUVNtEg1Dy6wOkZKexTr801ERxZhkjP55ytIQQa2WZmeHLZ6VDsZKcYF5DoMzEoJoy27qgpNyoUUI3A74rhC8luBqisPvvkefsDAyyFTbMX0TyPjrCsGR7lBCs8caw5dBkUAr3vU9Bc1uKkZKkYQIzstzcATDQr+77YcleZV2wrWBv\\\\\\\/ThJuCMz6VA50zgkaij9tOHCCbtNJpFj7zpaxzUnXNYuudDoZPtftuzQQhdUiQml4GXw3MU\\\\\\\/lS7r+h9Q7voKX5Za1NAwFTVoGxHoR7COB3nHLIy9N0ahcC5r3HyaeqwezC1m72gvjQpn4yyG0PiXMKU0RANXMzFZPDNSBVECq1zRn7Rl2X\\\\\\\/O\\\\\\\/iQR1jAzPUcWy63A6F2LdTYuHUtJoEhJMJybAMVkHif70v\\\\\\\/tlcDU88vjhG7Z+lcCIJAxORMcrxoosIYe3pXaEcyJFTdUb7PouVCLzWzn0axvTOwucmfCELrHbazszqS6vWN6UVtBoX+6\\\\\\\/kQpEobU3g82C4x2tVC9atvtZdgG3NvQEQh5KWFhTABd2q\\\\\\\/QbCjuyJjaGAEunrOGYQKidZa7SPVSVKER0+Fp2QMFQ37Kyd52wvmVG3BD73ZpkbvM4Vo7CGeLMYFSZro92361t2NiEd+c+4KYX9ViSXdJ4+Y7h7GLJfxI6AKnHzE5443ivqs+QMiQ+nhn3pagalgxgG2QJRXVhj0umWttXr509EXef4Zup1og7k40ExcrnL6fc7naZ7mqovQxFjm6R3obApVlLq13\\\\\\\/qqYl8n69GVcIMjd+yijwE17LWcRwSkXAEkuJxb9TDkyzWwHYIsmtl6KKlxfPyNRfk15tQqunP2emT0tPw\\\\\\\/ZrTGxSYmCrulBNIN1X+65L611qXt0M4kV3j4\\\\\\\/fV8EYMJFRC3XdJ6t7Kudmb8A0exUbesjJKPDeLiGadYkomgS+xK8zJ6JMoXNpk8bKpRGgECdO09kqeSlBRt0eGHdjEfstaBla29QGL2dWSAb9QpYj638Soj1utAtUnrLZ4\\\\\\\/ca2oMqEtkNVN\\\\\\\/YwKQSMfY5WlpayP3UbmVoSAVrzv1td92lnl89ofUk2C2nwyc8+GxLLNZiYF8U5Hmw2UJPqxyvxv4BXzwsTmiYDLj\\\\\\\/PhyvJeIWwg13NsyQP9MFRNBMh1+vq\\\\\\\/bg473W33BNVI+Qxxj\\\\\\\/PbMafg3q9nX3SfYwoFRLPrOPCFE4UaEfqBlLqiSBexYVod9UcBZaQ7om44AtVVMp1SHzrbCePeoL1Bd7Fhp6b+DVVH1ui5IjB5OidWR+iCzdJAjS+Jri35j5QsILXHRMpMwgwS6XBIfRjQJzqs\\\\\\\/vv3scvvjhuvHZ4WfUSB+WJFNAWb3vUr\\\\\\\/Dcc55UPqFngAGb2RQrQDJrFm+LiPI411\\\\\\\/in0rA\\\\\\\/aw+Z3ife1KxxhszHMvU\\\\\\\/hwnQLEhZIp2WoIKIjVLm14qFJjjlVVkd0NeZ5SXg0Qxjnkskexz751mjBbzLivp8qf39u\\\\\\\/lLCz3fDtJf82iCZO6VetY9JF4Kjm7WLJRUkxMczo4y3+2jDXKMXB95rKA1AppIP3kniq5MwObhG8DuRO9cWuuENcwBo3p9JOKrTTbYKIbdpzPK\\\\\\\/1uxfTXX83CulD6ZVzFLvBTMRRrsRR5HTJBRgzTzWRg4aV+x7\\\\\\\/r42ofbNW7BiAtKBmrbviV4qtVGc9hiN7JvWibcT63JaHSbzv5W5fBqISAswcDMzSg3T4XUodyX6aThwwrI4eOKIOIT0p7ZPnzEIPq0HbMsiNPvIick5rE1gYuX4Qh8A5qKGtlgamB4vMa04pd1jSr5BFs2BLLvQME\\\\\\\/eGunpw7lXZYHb6aepORRgkBL\\\\\\\/EvQnUG1hrCRcfF6LOd5snQfUh5xatL5U6lMB7JQGTBRnm\\\\\\\/riD+V0qhI0zoPCo\\\\\\\/\\\\\\\/EnA6YOMMfAqBJcjcIf43Z0aSaSWbCJlq7OCW7T8d4MqbHXSxDz35dgtXVxKDd3jF3hOkV2yI4JwvBqIPLA+vj6Ov8kmSTB4OwhGB4IG3VRP7P4D+WJT3QVLA4wWRuOBb9qYKsiwKzgUK5aGoWhRsAJUmZFmzFRh5cUiokuWBceE9bYpwfm4LgKisklYGDkb2Hh1uRXDA7kFsPATRb95Xa\\\\\\\/oYkSdQeSoLhrGhSg2TPNqwY+7vupF7punBhZmjofG16Q3VpRjBl82cjlz7+JCpBS7iaqhz6aVF3GGAu5IU3ZGOTwItBjMir7u1ONY\\\\\\\/FjewOEzVSb0C+NUgEN3+geNqplE5TMkId5bm1\\\\\\\/wKi279Apbc43zUjw1LGHzdrSow7rGJ922NoPv2oGAu\\\\\\\/jPRwfYwE3O2yxRupLltZmbbxlYq3dZO\\\\\\\/\\\\\\\/w\\\\\\\/CdyghnytE0w6F6TSS8e+EJLgGtCSBpyWhvYB7YuEYzFUV0EXgH0xmtFs1F4J5u2f7XQZL3tJodj38AM\\\\\\\/xG7+1aRDwH+KOMQbyosWVVknrYN9ZGd4blSGEd3yniB4Km63j\\\\\\\/vmdqnwebjo+eGFLffwDZgMtHJEDuAnG8QO\\\\\\\/ZdUuRhMAcRR5GAK\\\\\\\/C6Kd46MoLd8+uUZEgjR\\\\\\\/m9M4+WUJSNzg\\\\\\\/OmiCXaoYTe+L+MYak7SecXs138gq97enHgHuSR2LDDiwPDjVKLNyIZ3LMdexpcPF4b+BBWb71fsZjDbA6NKch2i\\\\\\\/Qx2m3YoNxtSiWCU0oRhbdBj0tjKLUdimkGtEYBUQ7BQceGoNJr9+Q4poKpi\\\\\\\/SV2yv9KbwYpevH7k05Sf9r6uG6Fs2WvdVxWwioqu\\\\\\\/TSz+ocwHMjnY5ixOnJ67lc6AtXd0Mm7obCEN67k38CrrD6NyPBMSojFiDMaFrGoo3VpGG3AQ7kZ8xQtE1OlTh+zcb3\\\\\\\/kRLzMFmJtIRshx6yjdWxAtcqU26DPKyPH\\\\\\\/AUq9GYgtp79ITqGWexmg2gYWV2RAyTDKvsxxGiMv499bQt\\\\\\\/yTcR2WlEEpahFL3zvVkh04cfEUp3upI0LeFkCI7KmGO4YeGfO6Jj6K++XbqBOQ1sPsZax\\\\\\\/EQOI+pcy3TVXqlMgjlhaSgeep8xwz+Ut+0JAxWtmCFCRHysXX\\\\\\\/FidaOCHR1iA9P\\\\\\\/AuA5565i1OpHtxG3YR\\\\\\\/PR0w92WOOuhkJU0S3AqcPF3dKC616n9YiTorW4wjuFXGsedHSvs9hYQkD4qprrA8PzIw8LZTKdzYN6drdc+OBm5szEhTMZIi0tAp3IxsTcD+qC0sCWQNghZ0dl5eAfgnyNQqs2wkQB4VH9VPYMDG7qWBxajk5upwM1lNXmgcZy55mSdiuaJf+UV5kltC1YnfqvD4owP6u\\\\\\\/6dq8tWVI5mFBZe3aC+B8jf8a6CIh+51B4E2iH+PM+uaNaSP96iEnv3ugF0Pfx4Q1ppDpLayktD51Wecy1M0u\\\\\\\/gvMpTYNGtR77dLtMmUohZwls0eaXvO2yx9WN3+r4yw0y2uxM3dJ\\\\\\\/Ykm2fOAQ7c\\\\\\\/OaycSxB0VQK9AQMON7VxgvNHt8M73icj6PlygzSYKFQR2L8rBbcHC\\\\\\\/NmDSu7hUlATQF7oMaQ+UvMces06MLbT+f4EV\\\\\\\/K\\\\\\\/3hOu\\\\\\\/yzQ4ErXmeXyop6INfvLf2NZPvVGv9LQQ2SBm10FyQc62PpnTRsAbau1+Msj+que7xBgid+PVL8DIR0JwiMBJyj56qFQ7y1XC9lMFgpQ4ewpuvCFt1+8B95rcywwmujbHG7kQbbNsDheu2fJo5QHpMQa0XcjJBsvj5PKJZnKfOEhSHHgKm1YOzYovWBfkqFX\\\\\\\/y93Fj2pazLdpGrMXXuJKetdWJitHSudQ256NXQBSJF12BcIoZnZlHvUHDmi36P7rgmaJK7lfVUI1JDop0gtA4naIpfYHX2LOLts3UwmhLsGSwHwDcHFVH6WpzsjbELHDVjggdZ5DqxE1wZseuH1BqDHzcOzcE4urRG6AlTxYXLb4pzoxFyrPT+tmg\\\\\\\/R\\\\\\\/WAoKP5GjF9o0zuWwKWsyvzFEuXAvT7AvZAGPJDIpdGY3+dUg67AH33tUDB8ESJ8ivRBGgGk6+aaDIe5NAnawmtlLgzxsp18VbpzCUJ3hnfrRo28p09OX2QSdarppuTuV+3sNoB\\\\\\\/haB8o7AlwEbzDTJcGWKSpNcNBiZFDu6v2ZQZsZpRoomwpc\\\\\\\/hMjBghmAHEjxRUEqENkQxb5t79xmueuDBpv4BSnaekThffblMzztKFjgxwCaIEMHLORXziO0cGMLVSRT0GKRuagU7vSZq1LhEL5A1O8AVno2Sn\\\\\\\/IQqVgneAQJWlElz0+5Ifq\\\\\\\/M74t1TI1qp+Q\\\\\\\/km9OwYkJPoBXdu5TjhQl1QUfotkG7zoiRSC0e4cWKEWMgEA\\\\\\\/3jUcZDj2PEBZBe+RMqxmVQfO07RRzHzCfPxqX7M3YYpHB7on1h\\\\\\\/AJzM5RkvS\\\\\\\/Jv6g4KvP5r46gf3YY+vj\\\\\\\/YQttHJLqPwFn0QIwPtQamRueqPJNflrT55OKc2EXz\\\\\\\/ZcY9EuHQxo2GfIBc1NU\\\\\\\/q0z2wuO0qYw20JNXnE72eJbylKnTdQBMEtwBpNlKVHD\\\\\\\/JGLCdkNQhkGpVDzVHIhlpVJjqsITklHwP6VBJahXDKLLdfIcNUqfGVzaUcBjfdvjQStuif\\\\\\\/sUKCUYFNxeguFzdcM5mqlFB95yys79PKIuQWbg6\\\\\\\/DyqJ5FzgVkL6DkXb2IZm6scNDCx9UYSFhsG6Dv6BX3cQJO8djLZ1vkxBBzK8nqC4GK9bi5NHRslzl4BkT9XqptoPWIGpomoX9JZBKkdKcRoqJYC8tzlL3JA6UUN+CAIAlHdHcySx21q2WbvdFqi4JuA5QRFXiKRpIeI3sb7iXAHCVj1rgYS2x8YGhgTd11ECW1eCf3qTE+XSIdj4P3YR2d77SNUlERacO\\\\\\\/\\\\\\\/3TUw9I3Tb9iLoJxxDbH+QtUCbuln5\\\\\\\/IqVr7\\\\\\\/jkizpb0kn105S5vgvOrCWyYHYu5HxAHDP7nswN+QtP7ACYiRrPHJ5F9N7pctp0YC3oaAuo\\\\\\\/0p+wb1dbTnx0C4DZ7j4LQaczA9maNU576JW8gMXnw3Z6fi2fpUATKOiQQZrfz8LGKBe052kzex4pedR1XPRMNEEi9b1xDAeGXn9d33dUd4vGqOAq1qPFK8HbroW1mnxq+xGQDyFtaladBg09BM+PBGApOw8qs4EPlGIs7EEbLzprgiay3AXfNES3smC4zhDEmXc+vWp04PlSDVQFZbb\\\\\\\/HVb5g4Ayl6xOyVfSgzslkznm\\\\\\\/m\\\\\\\/cX1qnLTmoWiabUQ51e6Wj0NcQrP+Uma\\\\\\\/VCoMIUl0dVBdIQ\\\\\\\/CeoVHG9ICG0zOMD2JpfGFNat3mkVV3z8Z2qxGgsVXjWOJaNibC2gRqZsiE6vBsYZbvblsNJ+lFmB+\\\\\\\/Z+HwKI11r0UvOiv3LoCPq0cdvnyCin\\\\\\\/WxOicA6EI8QhE0ad5L3yLrOXKB2wRnlm1V7rBoDO4RFrpfXbqDpI+y38hxCRvbQgCoDZm\\\\\\\/6upGwQBHdFTGE94wA9qqaPMYKg1KNMHnwZRm8H9DU5uuZHFC6qB8AM7PKUFoFxvgTNP3UhAUAxh8YwIXkoDz7Hy72bjyy2rnOPyBKYin8\\\\\\\/fh7tPA5KiznvLQxqWrN+Aqv6ZIFKbYa1s\\\\\\\/lGCkQ4E262SadZ2xmMYHJHgcGzF7GCyPdLtvMEpH0JtwpSrdeFkScqgEg5QcItYUSaZeLGkMMOgZzZAFwCo8eEBtPVH3QRKGd8PMmZBmn1lunLvMQuoJwdnaPcM6M5y98g6ZwpMf\\\\\\\/DlE+xxpX+xllBQdoWMO2gES6a3G3T6t86nCqQ2bqbiU03IK9HVYk533YkZvfNwK9djJvWH7T0OISLlyDmhdBzopHC3k6S3e4gJ6RPpTyKYI52sp2PTHk0GfbTS9PTxz4wGfwUUclzSHX+jNO\\\\\\\/YK7cK7Odv49w4oey\\\\\\\/97gndV2P5EEeWgrTUy12fioF99yaAjD+2IEq\\\\\\\/ypAnZ2JoqVYa+yNghGFfPOgmNtQLxGsYiA0hgUilM1D8br2VX+yNE5UdIkFvIlW2bgyQpImnmffZj65QrLtr3ZXr52NFb5CEw1RDnWxFNi2zT9sK3ve3QcpGEK3E4sLnzPWTAS4\\\\\\\/zLPrSQtUwkTObK3OYcNLSbF0T+mIEvf+I+Mfvns0SufMrqHRkXTNlWiDirkF67IEUgVa7bZTSXSNtMRViY122itxyITvDVHjUwCDMrupgPsozP0HRRVD4pH8VpgUCFUgbe\\\\\\\/EfZTcALZdbS5XVo9QubC1fqQGivgrVxQzsF6gIR4u9DujBRM68j6o0yR6kNop9KP8CW9ndGqYLI4Xfz1OerM+RPT0y4+UifSnN\\\\\\\/VvIVIIsqTIPhf4YUteul0aQvDcznR573R3PsD1oop\\\\\\\/Xr1MGCUAG+8XjbISEV42cBHehslo7P\\\\\\\/\\\\\\\/dKWEBHIeTg4CncI+cQ3TaBLO4HqGpfcwBDMlE4Qp7kFNixSWl7tRKV216q4iySiN+PD81v+2PVz2\\\\\\\/BDTc0\\\\\\\/twpw6487b8gB3G3fDaIUvtg6YGRi5gYNnSMB5b4XGrddqVNdriwQKFjbT1hvtor47TAd4jBIwXBroZL9adzINpbkNTi+G8eLOIclPpJkWmc4aWK\\\\\\\/aGMsYuqOc6YrKQGisDN03XQVaDLAEYSPmrFDYKcDrChu3taAMou2pCFXqhA6RSUfJ53iF5hDAyRg0SIJDCgYfFlwZl3oLkWdGNHiivnxP8phPIjSJT8wSHh2FU3JSOlZPD5P1NH\\\\\\\/CoqzdkBjkdxjrrnlNkwik9rgVQfvoMwm4udGF1IeX9gRkFPrpLRDB2uW9mjK7j0SvbYqbFYGhhd6vHxGksvmKPcx2BwUVXFAsYRhdFzl5FSW3jb0wv4ClM\\\\\\\/xX9woA7ZO2NRVv8qRDGVK+m414zEL6oVFBDpdHvlvQKU8cxE0GhQTlJiOTlqsmVDVvGb\\\\\\\/zdVNrYg3KdWZfXU5GbHMFJ6kFW0H18qvvaBGejjEofqr7ceQazWtsrjLglRbf+fY3WfKNSxLt7GFSS+d6\\\\\\\/kkItOLzzzm9tJyKVN9TREM693P3mJoD3N3\\\\\\\/hW0d0MPa06i0UDuaNlQyL9oU+Yja980oo0xIVLhbwClieuYQDHzvQ5E6d6DZoAInUw6TPHViPWBpOX2ulSXbkrcpEP+QPZbawGdH5Wi1cehMmyeUbULm+NPzzF6NgCvZkiwz+9XpUHMSrkz7xNWXZiyAbfLopOixO5dotsR7DrFJ6NeNJbg59Zo9Ngs7ylt1686VeMMFhOJhcOClbBCkdT+fAXzQKwN7UtIN21EPYmuOkeaknYWApLqYzuBTZBPRZzh3lP3C+8i\\\\\\\/IiPTFyTIaEXyxFmEAjDO84fzfclafF7KDEOOUeh2aDZ+lX\\\\\\\/gAXqL9dmQ2HpWuWxbPn1WkW5fhe3DX9Sz4jxTkR1Ax8QVq2vFvBo7slfb6D1kEfhaLdXg7c5VcT7c5MwhTPnV+65aAUNLYS4\\\\\\\/q3Fclfq0ede8lbbAq5grJCjNOBXfqNMRXZXm2aB5BbxPP\\\\\\\/WHC4gyVyntjhtdB6nGgwN3yzyv5cOVvHjQoPlvy+77MatDV2ySTWhq\\\\\\\/INyKRDbaLuu\\\\\\\/JXtEN9uxPVWKmWkGkOlZ3XEdlaeZ0uyvNJ0wm8y7mrErwtFGFxa8\\\\\\\/\\\\\\\/\\\\\\\/EQFwWD8gcvh03xTPNDxl\\\\\\\/Qpv73+1jS4RBEj8HwS\\\\\\\/V7uJt3JrEafmARv4zAFyMOHK3ARnDbbckswBI2DTacniPzVBLG7Ez4hi6HWhxFdpuWpV52xVAM2K3kGBjKH9fDdLwBWPrXmWuXOcHdIvoy4sB8lngqPlGwrgtD\\\\\\\/lsqJZwcxCwSJoa7vWUlrWzkM2vYNvML7P0\\\\\\\/\\\\\\\/vAXHjjV5+goDRGUBlcnihrRhfL+QLFmOzEKWpIVEtfqPrmIO+ZRtCRe\\\\\\\/r8L5VRPoqGhKs5fn+gMIj5P0rnZhcr5YjGqL8iQt+MHftXXu9DdXNUNWGewDbca98\\\\\\\/2rPMbfxIqGCi+yYWl+oDHlHXTAC3YEpp4is4Di27yC+NsLIFS3wnLu6JfNwln1yFHUGoDVCDqswPBhtojVkmyEW4nsv1cxNo5obpHN4vgFcJiUL42SutZOfKHomhmVTmeH5+2RZ3YIPs0YvvoA2vsMWXm+AVEsnqQb1VMEKGwu0nwv4yRl1Ma0FFv3RzOk\\\\\\\/YmTLWTf\\\\\\\/MgjrUYCU4gjihTv282eMkCygYIIZ+Wd3k0ccyHYbhaLnCUhIL8cpTxJ0xBNb0lHOnHqrgXcIE562aHo8jgZlt6+UUgCkvgVGFVrSHaON4IKHSRZUU5xm4ekdYh3HCQ\\\\\\\/3YfKwcLAJa3\\\\\\\/tziKKgI7pVMnTZ5ae2cSD1nfcwY2h6ikhNJrZG1cB80tJFza5+FMF7nFaBplGkIpEWX6Tjfwiv0VRqbxvLKgE6N\\\\\\\/POmpg0Wayf1qL5COJAJHNe4gNaWxnSnJVgHYCF7LAqewYhKoPJbSN7tVqaGzzazGe7IT3UrCVAJNBCjS6CGxv6rkQpxeq4twsmEPxNmZRHp6uxuLSvCpY51ii0NdeXvqPeXBjn9HRFlAKePapR4KyIMDCIiYOltOdgFyoj7n3hrnYnj+l83CdZ1rf\\\\\\\/XP1PSMszPZIqdd6nnE02yr\\\\\\\/kR9GK2LGGl4NGF+g8HiFMmWSPgfjIh2GC04Q7nJokXsIGW3k1IW3XH5bgxlhjR\\\\\\\/qS55wX6hvO3ydCb+24hWdAUpCG5oxcPUofGJXLcbmVNN2y43GzqFadyUzqpkvPCyGjkB7rykdFU1E78RzXec8GiABvaYTy5JOYXVjo0a17sDKRL41I\\\\\\\/u7auWV2yiuQg4zCMBEiSpEHu1G0B\\\\\\\/coCpXaGmQ\\\\\\\/aQ+FesYBsu\\\\\\\/YI4\\\\\\\/WFqsP00IxPjITe7Z5hyuiYSHVQVegHGsSx6Glx+ff3pzCiNmlZVTsjZ0+mG7bMdV\\\\\\\/Azx5DdJNnb4xxoksr4U6xeBYAY4cxuBjnvQtqs7mSAree410ugb7GNXlGUUo5teANbDkw0iTahLnADLXinBWOmS6mvoBfQVPrlLgKiFxzluvzOeD5kZWsQYzL\\\\\\\/hjziPu6JlPxDtIh4Ouermnr\\\\\\\/zuQy8OlrXyMlXjqk53N\\\\\\\/T0E8Ur7mLEvSTgEPyLxZxkfzbqAt8G5ZfWhv5G5x5gOl1tgZPFNHW5vscpExN3xyns5H3EbpPzaX48SHMk6TDGJHKQE1o9iu8p+dfu+ux+89Y7cFd238oCRIfC141z4GUrDsdRMFw7Y+C\\\\\\\/ue0FFDomWLV2oxFfM4JZnvj\\\\\\\/4Uy6uNTtwcf0ePHVoAMbnjf6zuQcrAYG54tKhvU8hxz3flUWtD2SmE0u1SdxL3nTRD0S4SNYDNGlajRDyP3gnbNAJOF0crjnbiuwgz4p2sAdXF8G\\\\\\\/4FPC5kJ74V2auUzXhrITcF510UkefTzAhkqV2cEFmy+\\\\\\\/qpszZ8DaSwrkeu3h6FGWaI5ROBhlhr\\\\\\\/SgVYz5hHxfALebhSI4RoZKEAmZ+ra4T3v\\\\\\\/h+iMMhzfhxkLOw7\\\\\\\/NB9lKMo2q3xkNmbdmfyQqPnl19h4wgOMffnYcPpDnrBzsqKyqoPSb4Y1xY5vcy+dnVaXS0d1X+GLCL28V115eAJGyVBqqqgRIgKLaw6dBEN2iiyVR9vS3XPgkWxa7r8wppudMGK9jMk7rAdz6flS8Ioeltp\\\\\\\/ElpuKg1LFaM9XINlYpsDkYsJn0VHitzR+fwxg66xJ7ISwQdaGZf7IgALFmYmcrXeDGmi7MgOM4PPobWJ8zJVk7Eb06eGRHHMKiMR9Ihld1lHEA9I\\\\\\\/Gi8t0+rDEXwZjwh49l6KX2TyUJdLyWRYAkZ7pNi4iKqdKe6xp6bVgEhezdMRdotoJTTFdkAh7aa+dZ3d3ONnPXCdYTdKpzYpQnFBuxVAyhIV+fa8p0Kw5c6QfbnD3ueRmSt8nY\\\\\\\/SgmdWcZtRHmiXT1ope5tlyFN465XhKD94DWbpBJP903QI9t5p2JGcG\\\\\\\/68s0qHmvuBPT6ATUV7SOjnTo4DViuJiyuyGsylIdRuRmX+drUbQ5X03t7dJJrNNZNOgLIuVLHn0KnsgkP76c10vzsnJVpCxuUVKIUq0FSWRmEHem4xsmvtSZcAZXmH7ecJiRyTU1lMFxLzGp\\\\\\\/+Cch+Fc0pk2mwdZpGIziZdXMVQRHTOW24U955ZGFqM7h\\\\\\\/oSJe1CnuW67yUS8yn9gvcBzG\\\\\\\/FlcTtgr+HgxrGktzn+VA+dttkzaTuByvlUCxkzqjNeyUYOm4nPHbRFk0riEOgLtYRbZK41d9jku8BolT2wcjIxLsRFrdPI6zjuu5biTEbqmWRedahFO6fV0Gu2i+BHF1LcehRtMfts90IQK92nfz739be2k6mjQqdAyZ1cRLXT6D5l6MGHhQGBpf+uNo9Gh+SvRUleJk8eTeQJw3kgkrGudeBpG7U7kLR2s7B8F+QG5PGEBlJPsgGhjiGTossbJLzkf0ej2vp0BkeyjD2qfCglim5OHH7SdiAvEwrCHzCSQc64Meq5TQGaq5FyQlD2ZBkyF+q0xoHOetc+dxe7kQlr2JTjI19N8eiOHwDc8\\\\\\\/sBaBCoqYfaofZOXEi0Z+VkcVZ5PrZZ1Wr+yND3zGZv\\\\\\\/4QM7N22y2ubs0S2ySL8ZYKJlWHlZhTgP+161uo5ug45o1dYYCAKHeng76v8uWoXLsVBuCLq+x8KxgAJkJ2T4ad4xmFmMYpQLjbwqqsBjnzXJaXNWEFWxXq2IX\\\\\\\/\\\\\\\/bnSFcZlRH6FI7BfWA+rO4BJFyUyraqe\\\\\\\/9kdgH\\\\\\\/K1LynO2CiuKAzqYofuByZKPIAptoPStfqoAnGSFsoCjm+YtNtufgOW1gHDcHxj6p7W1Vjl2wgyIdqsyvVAvYlH520N2UeuP7LAq1M29Vwwq5Q5q5uX5kkC2U+38RbRD9JYQkZxvFj+aGE+lxcl8VcetiCtoAuZSqk4+5ykgr0V8plUITo4fXl+WZ7LNkYhRx4Za6pmtzgKIXOBHN0HlRjoeK7jiSizq8MNo\\\\\\\/n5lJIKd8+NmCQ6nDUx6q71Axix49JD\\\\\\\/uQu9m+qAyr9\\\\\\\/jqtusHTKdVg6dkDwCQXTSI+cm\\\\\\\/gtJeh2BGFipsWVZ8Cgp9duWBXjb2n\\\\\\\/004z3ku6JknFh2SVgX6Giv\\\\\\\/w9gB0Hp4WKnUa03FvjX\\\\\\\/gjPuEewX8DC5p+EV2h3c\\\\\\\/6dZ6Uh2ENMc8zmdK9THXdYBeo+swdYhjxEv5tryyfTAfU9jpXLWkXjmfiCgalWzOukXcD667NWrbrvW738QDsWzrlqhE0HZYhJJJ24YaCK0u4GNELjTRR8by6MvPs3o\\\\\\\/T\\\\\\\/EB3t5xJqL\\\\\\\/n8y0vk9cuBVqnrKjnsZ9ozRLjARZKX8HZiuJachTvyOTQ6HxSbi2h9aXTu2Rv1+9vREpzhFtzuB8S9ts5rZim59Ecs69U0s6Si+oh+XK++LAg44+HXtRxh+pf\\\\\\\/nn7nej77btjUFkcyVJGf176BFvyny8eppBn8+kCDQSZti\\\\\\\/KDX\\\\\\\/6FiWkImT9YH+gs+QRtd\\\\\\\/IeDcgd2UUpKLRjoKympIeNlRSxfVxFfyCppMXwzCNWcsQoJh8jZfV1VlcSTDsVMsJmrVPGa92LMx1AFKmv3T6I8vC03jrldQVDIsVQ\\\\\\\/U39LBW2zyKycJwcBN2F3Sw6mkmGXTTv4OFgjHev\\\\\\\/DoLIjQcgWWJq3VnHJi2V9fpVCZjWsZlvZczmp6jFMw2t7DoLhRQyToyueFxlhF6Ubp\\\\\\\/Ze+zk\\\\\\\/WnbiqqH28eMLpTWG5sDH33WeoKsCHi4UFBUYc3x1NlpVo9nhvplMa5H8D66qTsQXl7ts76je+FY+WP1Q4zbf1gkhSki3SshvVILP59DxHuCRcm4\\\\\\\/kWKRsjK7VsaxcK5Qjg4NgPt4oX34n0ekSBqMuiVvQYInydK+5YNngUfUXdiEdvC1Akr7dffuhOopHf3wXbm14BRfMroahjEN8LtJDZ3bHOyoZiAHW2PnbapWEkqrpRZUHDnRIcgjlG9XKqZmg8RVTL72zYLnbYMSJ7tlA+jepgQMEk9vcNLToqm6rRXimcrAiPSl0lzE5olRLCdgYsVu\\\\\\\/Qqd6FWUksSRYdaXYbc4YlkhAUxsCQkN30dI38XqpbO+cmiFaG2y3cHs+HFJFGx18NrWaIS7LfYyX9TiKvdq43V9XRi7T4SS07ldhIWsi+\\\\\\\/2l42FrDzJYXVBCM053tZwvHe6bHMaspMCwFUazZ2Au1HF\\\\\\\/HJaB\\\\\\\/TYYiv6xtecy\\\\\\\/\\\\\\\/vGaBUdZgUd9+Ce\\\\\\\/Ijd4shB0+2xthW5xiX0a7Vgb84qqAd99s5RqJWjsqNB8gMfyxos6tQJQ83fFvWNZ0ggNZLhmAgxKAVVxXONjjUr2kACdyHzJu4u5P3g3yL4w2JegGPAJpgcfavfBX2HGkVtBvkx6xvrlneI4mTLg\\\\\\\/CqzPNG5OszzLCdmZ0IyaDchicZ8IkjA\\\\\\\/YyfUNsbFRJYiAqNykNUCpQe5HjvXcQTLJocLLirxJpS7zjReQIDnIKMOHAnVpW\\\\\\\/oBl3Ws8hf29HXRtoIe8oqhKzNQ2Yu8GUqxQpPm0XCnPj\\\\\\\/7cnSEwnheQdBTp\\\\\\\/cO4tQpMcOQjKPEEKCdUWdUTm\\\\\\\/r4w4XJ1AkgWlZ6ZFZGWn4ldeWq8vLyv3vvnBIG7pvhKLi4evW4u1HAoeDAy+eGRWrOshBeX33VrRgt3bAW2hzRg0NeiGWyCqSRF6ptFX8+2rB\\\\\\\/\\\\\\\/fx3t2fo6OGLBQyiDWexdISAmG3SQ7MvYYkXAAPFV47YTFTvT7d1TJ5QU+wrKxUOs92g3jb6Av5BVfxMU1oh65qB6Dcqcsj6FvpbIVxX86MRb98K+GHG0Ed38V2p8vWe28dAqk7sB6ulcwuOQuRAWt3I8pQCfyjFgxsiMebPm2byisTN1I\\\\\\\/s06ky7qBd\\\\\\\/yPC3E6oj1\\\\\\\/fTWt+rDRUDT2Ku4es8C8trDS5l3\\\\\\\/jmD25XIWt2ficzBvrNftTZ\\\\\\\/bDDPbDmw1cfVLp5BnW13eb9hp8laJTE82az7fdo1gXhH8wiv77QIvtd7c9siwp\\\\\\\/i4Ynhuj+eVsSRFeoBsDyUfnpqBfetwwiQt\\\\\\\/2Nbm9UiMepGJTJuKkygHzDIpzGPysD8xVuWB6qOFPc0QLN7SZ533VEFCCLXVsDioOTJwcCGJNIpUlp6Q7shfSg4R0Yfwhc+d5O3vDMiBBnji6LDZXYQbOvfRlZq9Z\\\\\\\/sKF6yW24YhYYXUcAKXqICZGdqhSwF8bbBaQlWxBBj\\\\\\\/XE1lkLGmQKIbAWfZEXWjPDl\\\\\\\/t5Qw53fj6\\\\\\\/SyQrHvLGWu7ixQYcXjx98KlT8X6fdyecA3PqrgunTmr6Em7A0J\\\\\\\/KFDaRRxQWJEzh+eCCc2cCM2yDIn0iWdHT+0zMzmUT6ss+AIYKFNjxQlbQXCS8d3OM8BMsdJO1qkxoJnmgEGjlLX6boDQ7cpp17lZzv2rBe7IykNOA60As9VgikVSRAqx9bk+5an\\\\\\\/Bh+l\\\\\\\/9kVDwA5Mx+dmsjkbvoIyczHwC+Syh0KAy0iKoWHtDt22qCyyRPrDJvT0+RxksxgCX+RcoYkmRXJdWCNX9ihpUW01ZvqRE\\\\\\\/I5GVk7RMVdEH8w3ZBC64rBONPhITRT5TlgrSalbQqWn77rGNICiuIRgXTcYgq8zx4Ky3Mjp6LZW5iXPTXL1SdX0WgzkbhzoluoJyWelUvAi9IbrCvYRziN50wgn1v3GtDuBYlyItaRPdDgRV6XErAr9AppG4zzV\\\\\\\/J2+RozAuBCDxSW2fm0yAc2LfYnaneGcMYH3TZXGZ1kFA7+eMfY2REwX17JiLX6\\\\\\\/Lv9Bkg\\\\\\\/dWG2JVP0HfA+2UOxosJEFePQYT2A5+GmTkgMmDQ2sENJ0oy8sGEt3T+N0g+57PyVOcAsDO53c\\\\\\\/47mlPthJWOCFKK\\\\\\\/rwAu+JU8\\\\\\\/baPmdMwJsFk4Q2CI97qNZDP8nAI8anOHlQSv8dXKfEB9w70UOiETTssxNdzLtonuzW4zO5K1drIpBFe5RwiIUvTb9c1Lq5egH+NPllqPsijwCDGjJNVCBrmXs94gj385Hi9VFAtl1EI3G21NoT9qs3DxLr3Rf0tDbM+p8VMZMtHaREH3qGGegdxTcI1KRByiQ\\\\\\\/umaIh8YevTS6qXIjHuhxxW0ZQxJX9xItMRPnc0KkLBQFDL+u1D+MyEVt9hJhKoRT\\\\\\\/stW7mE0XH+q1sWXxcwAFvux84e4\\\\\\\/R85D5J0jDlEIwqqlqTb3w9wb0wTUFN55gsNYeu8v90ItCVI7EGtrWfUEk1EOy0AiQEqH0VrKjYa6EKBrt9Q3ShAuxsYhq7zZm0u3eoPdVE+NMDqr74FJiIQoMO+FT+g1MejQdlY91FNlRjwpIzbzIZueUjlWolD6dOL7NaAT\\\\\\\/9G6goeSTS8WuSLWTzz5Hnwo+3OJlVEho0qvhyO4rnEoUimtO8bomX5UyCYJUROAtomVZXvRHbi9OARygqPlam16pbwJLfiGbUv9y9+DEbXRgyj+J6Md6sZVT38+b3rqXvAQNL6MEjLWFPYhq0A8DgRfzXFKa0yfjzNJb9zWWTOmmH57Z5aDvKcuGwXKwsj7idDJ\\\\\\\/Xrl0LPwBwCo67tCeegzYiblHoXNTpywva2FJ\\\\\\\/GkHUGvwo8OazyeE+tLtlITiILjhs6AkdbwGv8gCoC\\\\\\\/P0hI1s4YfPtVOekwFwOodN+v99lY3027x2hsEk2lsdNQGfd8fZOUkQ7TCLFxHvYAfhmaFaCHvkkiVpZOTD3U9evBA9zIv1UgOyt8U79UVMbYJe9iKIzQvWFcPBX3mffXu1KEG4f3T1eGWeu9cB1JBIr\\\\\\\/9S9+k1\\\\\\\/gfSDlx1kebdGkxMOmiLh5JFLVEcN63XIGM6+ZHF4SF6ZwWxMd9wCwnyOxKFwRw8aeSnCOiG94Q4UB7KR6azz6HY8pTqyTdx6nhvExWdjRtm+KOw2U+G9fLIa0Z7dycOJF2TdXoOLlXoZq\\\\\\\/2OucT8mjBfO+OpjKcGLOO3CimuDQ3Hp2Ny2GSQnSK+cgO8Bt4byN4EpAnlwqVA\\\\\\\/vQ1TKerrPdyvSFEc5qsta1+VUTdntocOfP5wEqcoQgiVfe2xU8\\\\\\\/omoT67l768KTDSnkn0Dlt+WNqI4THZwBXY+xsWjacGIcD3dfSUKKwL1GHmjEkp4Dmkg8RS9LSRcwDchz+vg4xC7JbMb\\\\\\\/P+66UwkRvs6sKih2ulTpzHqyUqBcH3ATLzY39RBvOsqqFJdNjMYYKl5kyfqyEQi4Ahv0Ubgjg4HtaRd5+9rVWLFSbnH5DLOrXLaRa+B5p+phj16mIlCVLnLmh9oynE+sLcvqjW\\\\\\\/ZVjKX\\\\\\\/AMlENyOSixiXDfP7mKp9L2flxzfnOp094HMSZGgZt02twdrqS0i3acxUBt9X3BEfjBvDWqrE9O06atA1zMjikLrP+ZpM92nxSo3b5LsZKrgpnti8IlM72i9xB9ERN0VgtAFtEFLAxTMD25Z9Fl8vF565y8etj21cQB2hwI9nzBJIPY8tKbXfIADSK4PVVmXcHemctv63oqvr39KRO9OmvuZJgeGLvyZ\\\\\\\/MreorLf+i\\\\\\\/+0ajuhqiq9iwZWyVPsXx+RvaSkMtBSNC\\\\\\\/BuYFXB4JqA\\\\\\\/IzWi1f6C22U3ci9dWyh1XMWE0859l9EkjpjR+gmjeFNoVT48G4MB508O2jlQQecnca81YGL7RY4QjhpWmCZ1mNSf1hRRT9Ar8B5QOZ3jqu94faaPi6Uji7RRGg\\\\\\\/TVLF8gGC8c49oVxewtIT8OsPok0DVXPgB5ja9VKkt88f7nw4zjComzcZ\\\\\\\/R70e0\\\\\\\/aaRip+ON2WJhe6DgptRiPZzrmlLM0qD30njOtGdkdd0RFRdn9VEjmoX\\\\\\\/Apgf+R7mc+TflfLub1aREG2hMyzBjD6JJnFAJeFK06ByXLivHkceNvmi6+dIjmtsw4bfu+cjsbyluVWtDXXlpRwipEeV\\\\\\\/kvBCV0dteS5+bUCR0NIEdOBKZSjfULLz578d94O1p0BzE0O5QQhAf2sHRsxIdui4N3DbzC208o1FuYrPjMLEyHkUJCo0WtEdvnnSs43axndF4L+s2QMH5FHyobZhrFlGy\\\\\\\/zm\\\\\\\/j89zXojlai+UEpABQ6PriwYOautCLATDjRwBNJ6KX8+rw3qa07FEWfsBilpi5GGwOMFqvF9Ak4ELUtkZCuOpqAxNXV7evsdTEPabPe972WijLtyStMw3\\\\\\\/DBd3pPiUvRD206X8CHss1J+\\\\\\\/DPUykEWeyTKaAKmYOQxMbP\\\\\\\/+o3Ek6FmDU9qsbsWYUeZctbOXxq7HkEW\\\\\\\/rlRoVs3CkESWy5WBcET5j6+4UlcbYiNYUYHCCDv5xJRUFOqG0\\\\\\\/nlPB1VQtQ+fwumnMWodaK\\\\\\\/HcgYrSoOlvWuHX97ZekF\\\\\\\/6L+jE0\\\\\\\/qHokpAEmIFhvyA5KlGqQHg8db4\\\\\\\/dy6lnjnDrVh6EWH63FFTZilv\\\\\\\/j4oehhsmdABuoEbF5cuhNESiLXxhvXd\\\\\\\/3d80JRRLqaMvthDecbTyWSCY\\\\\\\/FCSkbnQysaeJjGfNkT4+PvZCj5NJmTh94nQHLbHO3ic+Ez3\\\\\\\/mt0+VI1jaArlwuJpkBC9IpWOQXyL6RL4ZQMtzBfH9IQ012DS99nXW0sPCPucYgAZZut\\\\\\\/NBdKTVgzcOfspEIVFTuoDYqW\\\\\\\/rJIfAswNwhLBZTRvjYwRYwuMdevPdqVWdCZkpNjvIyGDg0IfB1ZSy8SJ7bsR+syjS++AW\\\\\\\/3Q5Cb5KcF4PfkPGFrlVfEkoddwxThzyOaW6HKwg7bfD7vYQGIgkAbPTPHLzD\\\\\\\/LxX30guosSfgkI7AFkb2rcppyHOhWOi6hy76TwHaIx+h4MPAl4H1ZPigSSWLO5D0+IAQ99jtMus05ruRbmIFobymn9fUONo7tHJI2vJyEEABTGCEoDzJSRx6Go+ohVT0mFmngk1MxseiI571+EYgiKXYnF+ChNIAystR2KM5YSZCKvMRucG6fn2X+V8fxAT+TKzT2P29VWxVYRA\\\\\\\/23MqP75+fv+pMx1WK\\\\\\\/f7\\\\\\\/PSlkKQXAiaHJx7oS2JdF+eY90pBxlotukYO1Vad3CzDxNRAYRExGw9Fb\\\\\\\/FEsnICqEbv8J77Lx3g+eXZ5XvoGdIyfbYPCyD9qR2H\\\\\\\/xQ5eIlig+g68G2tjCfWELAbKqSndRRDf7t7LEjYm0SjwqWjoqf9wbYLUQghH49+LG+AzoVLqYk4o9kRjSL+GkT\\\\\\\/BmBvQhAZttGH8KFGVwi\\\\\\\/zIPbplVw\\\\\\\/ElE\\\\\\\/i0MqcYKYNQu8Kgchh7Elf39tZh8eZY5wKzzOPfBO5DvsqWoDlQEmOBC85FsTBlmVVdWDaMBi7e07Z16mZKAgyjSE7zR4JP4rWAip2MzuLvy6UdETtngiJivTAY6q0Bq3rgGQyFoDgwiGYR\\\\\\\/hdFkzxT4YEcdwByndRdv3bEpSosu\\\\\\\/NXtcqZkF7jkntO5oi1ly8WVJh2FQ9Vtk3YDk8ucZjFurorli2yzNGCO89\\\\\\\/g3es5MolmPrFkYVD2SkeYCdZj2sdYShzzAIDTOWllV9Ri+8GqAYCaqKPqjdOADZJuxv32j8Vn7PIf8saJO5y6FQ\\\\\\\/WGy\\\\\\\/lL6654GZl8en7CnHN8ZxHUD5\\\\\\\/\\\\\\\/VFIZugCg8qYjhNLJ0WThJCTC2d8KhIEXyifHz\\\\\\\/3UAWTmOKTHO\\\\\\\/6Rw3b3crFR7SYB+H\\\\\\\/epJYM1m9baN\\\\\\\/fFTilebDHxfL3c4S4VjeXqLB\\\\\\\/GqSbdgUAuQc7EN3tv8fRq2ggZJQtaUqDHXx8iXq7acgcoszVyL6Q5uc9Evqjgs2ybHClq1KiycM3GN7x2sm5\\\\\\\/7E\\\\\\\/V7QBhIKfDhnmQBMri+X50o33K7NVYjTyx73c9FMYOjmESMq2veEfp30596UCYeFygTwIswfRbiC7GIT6umyObndOo7sI+7edTaZCuikVpdOZbZ1k6B7A\\\\\\\/5gUIV0XNkNGDKKXdwz5lU+CwopXxmq3Mr3Alb20tbU6Qj6JRWdIXSALPTXK3SUxWO8lwoVL\\\\\\\/L9al5N4eFj1vCrnZjinIuTGdbFP7CC6ird0qw5a\\\\\\\/YP4H3VovhmMxN4S71XIrx4Z0guIRjQHo7E\\\\\\\/rWMmC4xWKjed4Gp\\\\\\\/C7JzcYFUthkP9j+ND2mPNJv\\\\\\\/W96NYYQp4G8\\\\\\\/Wz\\\\\\\/Ka5U5rL4oQJFLr5UG6+ne98cINgwaYiZ9xtCTXvJcRd6Wzl55yF3RgqqJjJhNEQ56Wm3TlwFCedolJ60RPKepG0PNLOxuzFfPHJQ+viMb5Tj4\\\\\\\/FS4FpmPhD2ggqah8q\\\\\\\/AnZJUQfPH485IZe7vGJM8aw1KF1l4tisXPNsVZYf+6z3dGfSKAHfJQxhZcBYONEXxzjYX1MXfFOnKKpFfuReNquG9amYPQpf1A2fXtfUmUaj9IK+9E9LkLP2rkiFZ+riFudhjelenLzPBW4Fy0Hkmhra34iXnCPQK9m3v4gicUOH1OJF27vVPyJbdMStmsY0OkOgv6uQbNmcjS4aQwqF78yDHQQAsQgff+iNaqhuEB9vECumfqXGxkBcc8uEHNmWNQBxvxsT9eiozJxK7CKmMM8kFoG5MkH6uNeDPD04eSI7qHU\\\\\\\/QUOQz6+HVo2DMuRp8eoVvGrySNznU4OHd8ks4yvHdCtjl+ZvSG\\\\\\\/ZcygtP24w9JL6aDkhR3EyQd5PMrSAOFFUHo775SPk5onXrjyw8VvPJ5q8LcFX+uCE\\\\\\\/Juj0xs9o1wHP2B3J7JPUSMOk8raq4pbbqwdNX1mHGcVBL5qIdewYODasbpwyLNTqBIXtEpnAinbE\\\\\\\/GlYXwtaP26PzFB0GazuYds2QqNAAeFqyW96IGXqL5KPZ6wxehJYcwyhiqy+BCeIWV0Dngz2wltOCnv0gfZLB3MXmrS7N\\\\\\\/knrDB2ugQ2\\\\\\\/rTaBXclZrXCc+QXlQdZaq1GJMej20erLFBJgi2cj6EZcFfMs6b7jasCncefKRnC4ktGJeBHBPFllqBorHGTnOsSUYvOgTnvV0XES\\\\\\\/trEbhP1Hwkj5qvF\\\\\\\/8OhZl8KJzMFlPM7D7zdo3XgR48Jwt+0NYR2ZDLPrqiPHDGGSPzVH\\\\\\\/7AkKUjmgXWwm7LHd4Jpq6SqbYDP6NRW9buUQgnpKHPWINPkA3g605NFHfWCq32NiVH4JzeYGIrA1HXek7kHAI4M+8FdXWFd1LBWMa6apDGapDczacjsOmyoc1AOy\\\\\\\/NyDV3kWKn76ilNxj8CoInqR35EEJMB5qHCnHNzSGZPqBC9slIF4cR9hnP\\\\\\\/\\\\\\\/5cAAp71LAB3leOAJSkXNfBwdBpD8xqyrwpqdo8kBMFUUlxYoZSyCRo4pVhbqIyiKF9poshmugLai+J+L7tG5pso4IjifBrm94vY0Y8D4ZOr\\\\\\\/HKowoJhssxl9pz0NmRtth\\\\\\\/1eYOjfisxjzg3txxowWIxUW\\\\\\\/eidwdlJqWeTU93E5iD9YHC2mt+qxmzW8Qd6ww8QgMYcb1PRPnICCeogLtXR1SszhugroJ50fWqAuXCnDCxX2Ql\\\\\\\/\\\\\\\/5TEswOMfpHtmCnBP1QY7JqJ6qyp1MhlLoyiu1pIi\\\\\\\/LeKrIq2PCMY2L50+683irv15Cs9cwYKakAY9Cp3s14zSXk4cZ+xKIf5WfzjDQG18ii+iACUNyJ+CZNVtloHqfTbuLwwMHWCzbu1tlmeAtwRqmYkAD8av9Rfe9HChqtuZ+ize1roDu+rY3WrlKbWsFWlGdIY4FwDi3Gs\\\\\\\/vM+LevZvpvu+C9bP\\\\\\\/v2Xy1LmqrAwouT5pKc+3sW\\\\\\\/Mba2yqq8DtnFdjwhhjfnmz+cHR7Z4VbHvv4qQkUUl+dHEpDc4+7jAeu0VOIJxukAHW\\\\\\\/dP5pNUUaVLD9hxB2RXTFHZRvCMGsLHE2CuJPsK+xtIdOhUAyo7AKozOgr0PEVldSMKwVi81wc51EsPTICb0tc00qpJNtWkONogonCi9QZxozO3VsWsnf0U25UkcHYrAfb8QuF7vbxnnDZbfp1zKK\\\\\\\/AsGVIOJwP31wKTEPV53vXcPpgYeLKmIWTNWEGGkzPZSMfAtIxtGvwYsMijQhEFL6d7IzVP5kE2WPko6n86U5ao4jhxPuVGOaX0RkM2jQOdEyUSSQGukTN3\\\\\\\/nU6clwm\\\\\\\/cO5MuTISRktVwOnvkREBUbFSOGx3KO\\\\\\\/Y2QN70xKmls0UT\\\\\\\/ME1NRRzm3psPhhhvyCdp90e3ENG4uI2k5H4emepxMuCswRvgFl4UpYJyYOStE2ad5ADW3b9PLJoESEv7cB2UYyNfFXPNUrw4mhforJzSOqUHPT8HlaKrpEzHnG9x1auiOa\\\\\\\/VpmsquetfBAjYzFpZ2o\\\\\\\/yxz5Q3GuKLTkv+nkeJpY205Q9CtBh9W+X9bQTwI6hb2a8OSbRf6Bcwy1dy3rjfiSSErSrLTg7KQEbDliuFKwJLLiSq+ccSyuH3c0EjNAPKpxHgnVNjTdkOSpJRx\\\\\\\/1w6WJ3Y9beIwknhdCJnfDO\\\\\\\/mcJUEPHKtD4u0j2uu4cW4RDGVQJIeUM9bwrg4SfvhqXqVt95G+W7SWgIvvPiXHhODHpPVXdqkE4QBYSb0vnqYNohl\\\\\\\/UZV3yAgBHZyw+mvJkgHg3Cr62eSfxBo3soABLjjudf4WEP\\\\\\\/MDt6dXWM5xWEYbxHJsYIfL7QOZlBTb8abp3DHXZx+E0PygR0JDqgu1O603sgwW2t56dxs0WTryX4rS7lo\\\\\\\/hsMFLTwGeZo3vOzdQd\\\\\\\/8jM0DH9sqTmIH19dp\\\\\\\/GzdVfIsFHMNTC5qGZX4VRh+MvCEq9JxjPSOK8ew2yBHABMYyS0z\\\\\\\/08uUawFgBq5LTBtkN0rUYMntoIUkSlx0Ey9nnai+M3\\\\\\\/jtMtpZLdpUS4Nr6yi99u6Gu\\\\\\\/tVt2p+KVtvU9hvFwFF64C28njDv7\\\\\\\/3EH3busYGoGXI9XMuny2LND1b4jInyzkR2MMTcMKUgfdNErWLutWWDhs8F2k+Ps5knLfr8dfuiMbaNkRain23y7bw5mVJAqCio\\\\\\\/LCreTmEFB6DugfbqRqAl54pEbXJegC1JehFF2TbUWAaxh0esWtHPnEGrC1V5YOpEqNN2HJ6knsSEmg5dLL2x48N6d8oI5lXkryQJ17bDkPblPvJVHestqYhR\\\\\\\/rmIb2y\\\\\\\/+CSehU0TOWlm0hRAVXqXyeEHkivNmwluNUOs1JQSlKKPKjYqeV2zgp\\\\\\\/BmYwvBs3wXrhwH88ng5CwI8wAivMu0rnClkaIRd20B6TfVEb8nGHrBdCh92WTEF3hV71a7IUGxwkg49MSQMs9AcJUNPdRW88dffZb6gwIvr6lJmIcUS5f+C6FpglSyrBZ3YvupCuvoCBcZbZ15YSa+eoc1ZZZcoOoDdykLumisvYMoRoeCBWVd9qnAH+XpaaMVEPThXrCvhnd2i47xwlw2yYDgVRulz\\\\\\\/4LY2YCaX4XCWbUCNi9IfV2wrLEV9Uwm+Y1r+O43LHDILndEeCSXPTbY\\\\\\\/Wk86HsM6iWCqO1K+2pvpn\\\\\\\/au0omRIIKIR3oDQFh8qG\\\\\\\/3FTyc6cJCdPcpcMGNuW9PViNWK7fkd6WNF0Kl+\\\\\\\/kOIkiEui4\\\\\\\/LeyZoemDUAXRu7Rabi4YOocGsZ38uc5lmpwaCRJ6Uh55K\\\\\\\/wA0+MK7+FseddH3STg4tjcmDHClbPhNyvao1QUg7Vh21bx91ZT9jWoDVdkzvCZssd+ix0t9yKnrVIOY6EE1VueHdVDaglXDoQ6xiuMZi0PL28LvQLf2NbaQ20qq6bncK8dSGKGwJ98x5gnT1iFNn6AKF0RVS3YINb4R0hEJfqiBgK4\\\\\\\/Z48BETZ8BRfS+JXsQq0eodvUyXQ6KXkerxBSUVwzFwFbCImxZ4ZS\\\\\\\/14fiBBYS5l3ZBxFJ152OniyTqIg6dChy\\\\\\\/VkdIbL0La0A+gwFPJuN1N0BXk\\\\\\\/SziGCpa8Ie1PrCNGbHGnXAolLYbBQWbOt8TAD\\\\\\\/sax\\\\\\\/co2iM3HfCI3c\\\\\\\/RA2FESJDo38O5CXEIJ28jqC8B9Z+W74dnlqKHiNu9OtDPrT8+SfZQCsK1bGiwO40heS2ByWm6yqA+1GiYfVg\\\\\\\/5+EW6oiUa6nUGqKc7Q91tDsjUe+LgKRDzCQouAppv6sxdvlvE638XrwrIEFcZDJ50eSzUIkjBHMkyLbVAl4lvPU0zAzs7QDkhbakNDcxleuCv3s1eKNSB8xvhzW25zVNjQl3pxhNPnjaRarK7bJR8S6yOyflbovn7iY3YdRR6JxaLHpungOdGcpCLWZ6cHcyS4NA9JC6fouZUeNyabnsLN9MHDBYSBSsJg5l0XRYsxY8muqJMGpWeVrt1w\\\\\\\/Nfw59M5Pef4h7anEUapQ1BXbIiHBEdizmd3DksSzIGiDDfsV8q0XoKzJkpnC1bOoLwbBwnmmk+9A9B\\\\\\\/Ay+D8yS4PzCUPN54uIP+30JmaNfBbm+dOnWRlvpDZ0La355siH\\\\\\\/PFKkfpjSgs3GGPgBaMRbrSRK+X4eELaNDflH5OuvkKD4KQdz8fglEkVs7xBUkFXp2pk9dZJltcHNBR3yBgLFDebVmPCTbcSjdLucSEzlhwQ2XCOnzhtQaR\\\\\\\/7v5hhGFA+Fv0V8xCKkuwHaTTqRYC\\\\\\\/Kzuce2XTrW63eL2f47Wa5H9XEpJQifSDaj1V\\\\\\\/rJWP4VgIr8VRwl5mVHyN2Kxsq6BuO9Uckm9uEr0fKsLqa66ScagWvv6wKyjUOeKGIIjp2oV4reaUBzBc55ZXnHN1F3uSUCWzXAN+fCYT2Sbn2ajcV6INHMuLys1jllKmcHqfgCA8cvZkBVQmHto4oHF\\\\\\\/Qi3457oyc8ZpJdpUhH4\\\\\\\/ugaZ4S9FcejbtNp7KwJCWDW23\\\\\\\/OgTeYW5q5uVTAzegd3PdbnK493sZK4ximixiushQ+DumBd0y0sFkZSOvbo2pbuvlMNOE9S9JigvO4HeUp1oSPb4qU37eojt8ytzJkVifdW\\\\\\\/8VafLUrD7vmk+silXewCyYtQ56A9qIzhlipmqTNFbelPB4P8fh1qvjPdiSbNI7LBtNsBuN7SspKG+dOhuu2Hk9PlK6BN4r3pL0HSjGDJmXwaVg\\\\\\\/qI5S+fcQVYlk3kquI4pGfDQDqtR++OtTDWwpFgl5jlG2+FEj01bRpVwmXLu8EBjo\\\\\\\/o4IP2EPOoevXQXDlrGSGHSHl3qDIM2\\\\\\\/0YSxlH5iC0e5tipREDSwUJw\\\\\\\/ZumhCR8dNl2U7oKsbEPHAjbasAUohgqIM5KfQb1drNozE1\\\\\\\/1Yg\\\\\\\/rnY+2MBYJwnhKUA8BxDXKAxZFQ5YMowheKa9HpCnxpYyfFSqik52CpIDuE4flWsUvMzTr4WRDteSn2p6wl+qmDm5EeqG2byoD95oJ1+K1zwLlhFXLbMO10WLJl7aRBVOoTriBmKdXGDh9EMX5raqJSlYEx7gUOt+ooQXxFSv1RruqO0coRA\\\\\\\/AbsBBRqQHgGvtZ+ypqS2XDz3HW2cQNCSm41X5R5TY59OI0iJMKpucIG6a+TVvmgzwDNkDKlLT4nhn0m0w63XrslcJOTOWNl8BmKSg394iJoISdtIxXAWZ\\\\\\\/cSKMTqB5WLH6Y2h7gqQUx8ceb92HdEe\\\\\\\/2LDMYXkNoQcwNCtKRNc25xQjbI2Z+oSmOqpLtI8yTFZ1Cd8nsYpkRUlm6XVMqLrxN0kOaxITm5kyVCF8XU42ZmhKM+CO9VuxGXI+mOH++c0ppaNiByDV5Gb0L54QRlgsN9F50C4OUZ8FgeYhuJcqklZ8Inhx+P3pXctYxBLBPbUI0bUMSDvC\\\\\\\/VfmiyJDA0r5tlJhuV78Pawre8wx66itNEMit+sfPbiU\\\\\\\/ruyxR0Twn7HRtzj7X+pdDq1M8O8TaSPVWQ+xmQkyov10DiQAfXnhRJ4U1u+wA3qg+lMvjUK55abEfKkmGMf6aATQpKcqvNIaIHpr5NklQYyLFGGqkh++HWvtOelKL9y5qLuluAXj4JJO5fTQ2kay8k7bfOY7GuTP\\\\\\\/yA5kiiCh2Y8gfwN7RnV1N\\\\\\\/WFKNdVx3iex\\\\\\\/Hqr1YR89Vugmt9twCIiOzd0v3VQwE392h6MZKO8JXKfI1wYD9GDoM3BmT4uK5fHgscOfkJ2cj4ICZjd0UrUviXc36z6lqVsQyAp4c5Sj\\\\\\\/pNdLkUi4jz1E425WHYzNIqlqw7BdZbhZusELHyWjUw03iHCONLOEwluL0CJ414EjUUeRh5BL01Rw5ZlpZNeqthyOdtORn5grHxNx5jjvKW5k84znUJWad6\\\\\\\/bFBV2u9Z8MJ0GzXHiWZHF8E2iKM+\\\\\\\/Mf+GVqyY5dUd4dAo\\\\\\\/RjW0vKPZnLj6OMvE+5pTSRh5t5dxuTNERG1Eo6Ycm3qyQO1bx5OqW6NafOEZNjqenaWougGJPEWrUWhqUuTrwxifbCxRMpEK2q7XMSInPHmQZirdR0pF50Sdb6tR0ECURM4Sd98WajWhmVSs80yA3R8xhKEfuiC59VJB\\\\\\\/MPiJaC\\\\\\\/XZ0jdcBuk0L\\\\\\\/lgK1JSHC2RF5hGqKSsEddWdrni+70oJ\\\\\\\/knkQFd3ppnPf4GAzP158\\\\\\\/mQ4hrCUjXk3t+o755VnXYlDgKKSx2YdhdZC+e0hwFOMkNrhuNVLHp9hrQRzR8JIOYYC1oUEJPG7ZIOU9sA8JawjerVLdogzOrvNqciMzH4Qrd4ewY8rQKaNoQntTrTYFd12TyC2ICZKqOmbp0D9MoefM3GHFPwm1VuoUl7ReKO9aFE7Zh8H3oZqhE93k6HKXYUKJUsZ74zedvx1kqJ3wPa5PdwseQGUmNp98tK4dBzSQgAu107QhuH56NR85Ft9W0OkGyCrJvZSPtgaWkKe69O2rG+OA7YNT78p2j0nFxpipDSwUNT92OdLPsAElDGyqjPIMyuJEHKA4FrVLDdNtexAYCi9M0jAqh1GznI7q6YmeI2j548bwqhcGwWB6DxcmgZ0yo4FD0Dk6Ktx6hnnFSN+RbNkSxPgrawVJZzjIpj5KqsksSjJO0+njXhR\\\\\\\/KGArCsupptPBZppOEhHK4t\\\\\\\/SUGqG\\\\\\\/jYbsHtrrMpVjgJ5Hc5t0cltypWa9NRBD0MVB2pulBc3IKGVkL6LKMjb8rPDba6MbALXEhQaR1ZRhW+BWOFD\\\\\\\/\\\\\\\/fPbwLb0cACtJp4\\\\\\\/l6QiHAS\\\\\\\/txLluA0bbcC42mJuAMXaygbN7CijRzGL1qJY4BEeChDfx2ROmO8LTXu6+3pgNd39H\\\\\\\/4bZ1UX6GZEQKjjtmY8nByoyjClVBl+Zak0VH+OEJmn2K\\\\\\\/nSUry4OsVgZKavkodLRTMy07C8j6VYWc6v58VVuesn9UvcRKXr0mLaM\\\\\\\/YJlakVEDy9l58b8pHkfqgMnL6AHYn06pv2J+bHV9CMLP7sL5sIU4x0VJXuFtvDqX1LxsAkg93MTZKopyyHvIQ0qhK0rHyknpnP7lGG1IFvHCXIQmJWFS4KaB3xL+vtFLh\\\\\\\/vBOT0rLF19CF6nIb9Vw6ZnOCAEgGCw7Ev\\\\\\\/blVSlr2COwwi3X6ar9GnJ3TOS0SrBZ9QcQbUb3K5tytTX84AaPyDJP9TqXtTbnywYStPlAyof7QR1ear0406OPZFgPq5meljFA\\\\\\\/doRP5yuYJTwN7k\\\\\\\/cL56V5xkxqgYbnH8xNtTbEkyp1hyQy\\\\\\\/VNLqfxwCoEQTo2hTZLgK2A6XZhuGrPenSStaa7XlnNJnKrRqTdxsFqg0KsAMAhqglMu58LAXIQd90Eq6\\\\\\\/VnC8RRm+aYFCj\\\\\\\/1YWs50A1ndlHysZVu8ATPXd9MxupctRsPbgMwSE4EE+F+4TrsfrSAwhK39DgIqpZKMm41Thf3qcnmVRxA7LnhUS6cCMAG0OBApjJ36yUurfbeK4QcTM4GD8oWNXbyqeIZII4pcVbICcODtk8RpDf6mmsXb0OVQA8\\\\\\\/uZKxd0cgESmBYHzpN7WKEogomeXc3OZKLzlrN193n0MHhhU+tKhQWvaqmmCSl1K9ChmQRxiFHy9b0yVjI2RE4LmqSYDAqbPhEibVrm35FOon0ee5yUth2wGEHm+tXFXoLISpmdzY52NKIbhOduy436ISr60AXMtBD9xN4DN1M+MBWS3mYVqk2lEeD6DjtU8NVQBxJde3wKjPvRUhORA2u+etP\\\\\\\/qF5546HbdDG1AwLJugROzUSZDVLdi3+6TbU4VJqKbV8X8zjsabZLfkMX4+cDuFTd0xHSSGkunfkyJMkz92ZfNUVNIlE7vS+IXmjfgRczZvK7zRHbbocDkOrupKrvo19CZ4Cq09jqNZMQLQuOt+WSAmKLaRPCRz+SVjV8bMrz\\\\\\\/CRWGHAdqCmT5oiv8x9I6jH64KCMRBLMiREeKn\\\\\\\/pF9zaKn6\\\\\\\/TWFUDlgPz8ANwavTO2rlsmhEUldg3RhJ4wuCDpAaKcV2TBdzSHWwYyOI0vqh4wO7oRAMgiEhmTVVZteOPPFx2k6BuHL2R9pwjNtLBhEIJ6h2+mV6m8BMP2bKRwB97sWnc7hK1Z\\\\\\\/eaRi6Xa1xz4SXcGTBINsPk22SGz9swYCfQQFAlw+2oy\\\\\\\/MYarUSZh4ygADGSyDnC02ervEF8lMug0ilzVVLbvfo73vzFHer29HDjl8ASk4PIc+DaGedg1C8P92mV\\\\\\\/7uuxJkco4v3mzpQW2h34uhoWfPy3ZX8l4+v2k3nBAm0\\\\\\\/XI03ufbZsshp313lhz7Rg7onyKFmciT4olbhVAeH8qeC6MWNiqywl4M8Z5hcK9O1oOzWorlqoNvSMaZAgasljFecqXuH95FQ\\\\\\\/3QG7zvhUr\\\\\\\/ifCV2Yv\\\\\\\/ivfn5\\\\\\\/yuad6JFLoHMi2FDhaHNaGgQhUDfDSE5sIT0rd3f5dIYh2xgeHGrD1iTiGMBwIfgyDv8KSQvxbBAQep6r\\\\\\\/5If7YQ8mz1HnqHdxIY3+jrmrpPEM2vvTavzVVDREUPaQSkE\\\\\\\/NTS+s3s3eEM3J3zzyg\\\\\\\/Fspew==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QFFfJURx\\\\\\\/Jpu2afXcL6pZg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220433698fc73511b147b2947ad4df9112435ec503a2612b9c1bae5b00bb1b63ad\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221dbzyxqOS8Z3YVjnJG\\\\\\\/a51ll3oYwZ324yK9S9NJeTXQ3TSc6pf5eVO+wMScVm6PkD3+330iw0Ak8akPfdSsOg5FqL3YrA2v7t3MvUY8T0keINp+gaSdcbjYF5b9MlpZU0YvRg3PSrNpegLfShMVxV0+TiW6fwcH\\\\\\\/bqumKh+kX+fP1qJTILHMrtiRiBffj9ADt\\\\\\\/5LsFuadjPsTynJTxV28Mlmq0vogewy+maJXBnWQws7\\\\\\\/WySiwNTMK8sjxNKDTAeM3bE2NAgSR78FLihBRRQZbAjOu1RZiGEnQaatzk3gZqDOUqk97yC4i3x4mMDixKRzEqU3artLYYxJcewzM7y4Ls7mla48qgQDMNecfjFkU8HF4+nuzu7KdER0uxFAeNjSFsaI+jb89srQJd9KwJ13YBX1z6taDU48CYf1UInZKfxAdUWPZJw1+W56K73JOMFYH\\\\\\\/KGWBWdjUZcMYTwdmfCAS5ZbpFHkd92LsaUkloD4PVvurc4CQQdJ0MziV71a+PnAZOYQ+O8HbgfDRwsD52hZfudqEpzXlyot\\\\\\\/0LJ+B3AR54kMS\\\\\\\/u6NGNeIgOjb9E3S6tQVCZZ\\\\\\\/gS\\\\\\\/WCuronB1xGvDiwRnnjbh1UxbUTvuOowvgK4gIIBklUP4PTVqpJhCC+e0UJUoSJ7AZZockKT2FVQBSQcEIk9a7CIOPAkhUEi8Cu2KjcSW4EQx2wp6yHb9bco7pUOOjMFjyBpCZha5+nD4fZQO4KFHnvu6\\\\\\\/e9HXmFD9ULpzQx37g2UXij3vx1AFBFHEEJ\\\\\\\/CCU\\\\\\\/ScsUWHAbjT1ymAGbZbhT49SmoldQePfVs4f1uGuUenESaXT8hOijEoRzmvl7o5Vih9nUeiOjljR8Nd6eIjEUgBJ7cvEG+5G+oqrI+\\\\\\\/l+csiw0zBW8x+eDCX0wKpcWAk5UmmjlRH1tqfp6v2GcI04PhTKg8ROM5yJvKnXFx54aEyTMNVc6pKJbdIjVCwr1zYVuOQLR+MXmAQ0UsIqZcOCJf4wUbzB15wfhmBesyp1u1E2y0fhxTeQKa4KVKeciFAEvAyKWowhCf8ZwcCkxNKG1J6Byq56Jt+r1By\\\\\\\/aI4GZCb2n0Tg+M50uTe9RRuQGsPWzQAqpBIMXJ8S0Vs9YoOhrmrwJdDmpXFede7fV4LIQ4Myu+L5VS5iHLTIapXt6mkxykoo4oYFhrPU520APHQkqXjr7uBTZkWxqhcboCglpL19zREoVsPR7TQhg9\\\\\\\/93uzw01ri6acKUERCNyEupRgjCJPOofNkHhRhaT5h3qr843L+VZaxS92qekT9B8at7SjK\\\\\\\/tShDLBLGb+rzBI49JCZYi9Qa8KXw\\\\\\\/4269AdqgZ\\\\\\\/euRFFi8yQzLdfh9aYZrSNi4dAfUjpex6Qfs0NAsEwuqKH60bliIQmUjXo2tyY9Kzt18SsUctOejloGcIlXgNSmb5ef0zpyYWxxkq6Y+niLVB9brTWgMszk6hIo6OxPh+VG9xWGKD\\\\\\\/h2PkQvTZ3SJHa7W0Cb1B9+tjlq\\\\\\\/wf\\\\\\\/M6T+cQ+Lfz39gUd2DXQUTWxb1Naxf6lYJi7A4cGlZitaFX0ihaBR6x49Byd+ufs5muo2O3\\\\\\\/KAGQWVeAHKvn2AGcvQAPJCU9i1BWdKi45EvYKW62utlStgiMU5elrgZIRCcrYWJvR902YgrdSaGacffXUM+dl3wdsl0LyH1CV6gQp8zCocrqWLBJ5j6NDLkb7lu69sC6s3NkFSvItwhi8V2zeksoxoqdkxTVDXOwulylQu4ebE8UBmPacGHVfti7vGBVqGDF8p6DI\\\\\\\/FmGDVPWU6EgnocWCMGxKVEZKkyH4pNdCHtpTNh5of1DKXpL+dA5XR6FxEFvMz1ReCrWxk8wSfErz1uGFV1I9Hn7YWdDr97u\\\\\\\/qPLqRlGQY5\\\\\\\/4A7WDOyGljZE1884bdi8BtC7P7K4Hd5DPQdIR07U8C2Mvo5MhGakKiVuB3Dh9YK1lMb1bazBMjyDyUaHUgFl9fyN4ttdh8FBjwkk9q1CrN5cgv6lKnaEvm2RgA\\\\\\\/IqhP6CXYw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022AlsuhuYs8eXKAOA8alBvgg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002249f2f0aae2665159fee6a90d58c66b1b27a0d4ba960e9f9861d6afb880b316b3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460355 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1287" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-lMcYSmOLZWssDZScVI8GT3SKBXCG6GYYHsCnmHcxJsg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1287" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GhIFnhV1lwxCykTJQq-grMNUo83jyXhcLABaeAtlFZI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/5/1277" title="Chicken gga-miR-103-3p Targets CCNE1 and TFDP2 and Inhibits MDCC-MSB1 Cell Migration" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/5/1297" title="Mapping Challenging Mutations by Whole-Genome Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460355" id="node460355" data-pisa="ggg;6/5/1287" data-pisa-master="ggg;g3.116.027581" data-apath="/ggg/6/5/1287.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Conflation of Short Identity-by-Descent Segments Bias Their Inferred Length Distribution</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0668-7865" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charleston W. K. Chiang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Ralph</span> and <span class="highwire-citation-author" data-delta="2">John Novembre</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1287-1296;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027581 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460355" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charleston W. K. Chiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charleston%2BW.%2BK.%2BChiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chiang%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharleston%2BW.%2BK.%2BChiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0668-7865" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charleston W. K. Chiang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chiang82@ucla.edu" class="" data-icon-position="" data-hide-link-title="0">chiang82@ucla.edu</a>
<a href="mailto:jnovembre@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">jnovembre@uchicago.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter Ralph </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Computational Biology, University of Southern California, Los Angeles, California 90089</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BRalph%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ralph%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BRalph%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Ralph</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John Novembre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Human Genetics, University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BNovembre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Novembre%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BNovembre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chiang82@ucla.edu" class="" data-icon-position="" data-hide-link-title="0">chiang82@ucla.edu</a>
<a href="mailto:jnovembre@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">jnovembre@uchicago.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/5/1287" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460355" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460355/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1287.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460355" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460355/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1287.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460355" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460355/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1287.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460355" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460355/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/5/1287.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460355" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460355/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Identity-by-descent (IBD) is a fundamental concept in genetics with many applications. In a common definition, two haplotypes are said to share an IBD segment if that segment is inherited from a recent shared common ancestor without intervening recombination. Segments several cM long can be efficiently detected by a number of algorithms using high-density SNP array data from a population sample, and there are currently efforts to detect shorter segments from sequencing. Here, we study a problem of identifiability: because existing approaches detect IBD based on contiguous segments of identity-by-state, inferred long segments of IBD may arise from the conflation of smaller, nearby IBD segments. We quantified this effect using coalescent simulations, finding that significant proportions of inferred segments 1–2 cM long are results of conflations of two or more shorter segments, each at least 0.2 cM or longer, under demographic scenarios typical for modern humans for all programs tested. The impact of such conflation is much smaller for longer (&gt; 2 cM) segments. This biases the inferred IBD segment length distribution, and so can affect downstream inferences that depend on the assumption that each segment of IBD derives from a single common ancestor. As an example, we present and analyze an estimator of the <em>de novo</em> mutation rate using IBD segments, and demonstrate that unmodeled conflation leads to underestimates of the ages of the common ancestors on these segments, and hence a significant overestimate of the mutation rate. Understanding the conflation effect in detail will make its correction in future methods more tractable.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/identity-descent" class="hw-term hw-article-keyword hw-article-keyword-identity-by-descent" rel="nofollow">identity-by-descent</a></li><li class="kwd"><a href="/keyword/coalescent" class="hw-term hw-article-keyword hw-article-keyword-coalescent" rel="nofollow">coalescent</a></li><li class="kwd"><a href="/keyword/human-genetics" class="hw-term hw-article-keyword hw-article-keyword-human-genetics" rel="nofollow">human genetics</a></li></ul><p id="p-3">In the present study, we consider a genomic region to be shared identically by descent (IBD) between a pair of individuals if the region was coinherited from a common ancestor without any intervening recombination. This and related concepts of coinheritance and IBD have been useful in numerous applications. For example, identifying shared haplotypes forms the basis of several imputation (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gusev <em>et al.</em> 2012</a>) and phasing (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kong <em>et al.</em> 2008</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">S. R. Browning and B. L. Browning 2011</a>) methods; the frequency of IBD within and between cohorts allows the detection of natural selection and trait-associated loci (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Purcell <em>et al.</em> 2007</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Albrechtsen <em>et al.</em> 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gusev <em>et al.</em> 2011</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Han and Abney 2013</a>); and contrasting the number and length of IBD segments enables inferences of past demographic histories (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Palamara <em>et al.</em> 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ralph and Coop 2013</a>). The concept of IBD has also been used in recent efforts to estimate key human genetic parameters such as the mutation rate (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Campbell <em>et al.</em> 2012</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Palamara <em>et al.</em> 2015</a>) and trait heritability (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Zuk <em>et al.</em> 2012</a>).</p><p id="p-4">Traditionally, the concept of regions or segments sharing IBD has been defined with respect to a set of founder individuals, such as in a pedigree (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Thompson 2013</a>). More recently, attention has been paid to IBD segments among pairs of individuals in populations without known pedigree. Since all homologous regions trace their ancestry to a single common ancestor, and since the definition we adopted is with respect to recombination, not mutation, each individual is IBD with every other individual at every position in the genome. In random samples from unstructured populations, most of these IBD segments shared by a pair of individuals are short relative to the spacing of polymorphic sites (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chapman and Thompson 2003</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Powell <em>et al.</em> 2010</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Thompson 2013</a>). For this reason, only particularly long and hence recent segments of IBD can be reliably identified. In practice, the segments studied are the subset of all IBD segments that exceed a particular length threshold which, on the basis of estimated error rates in humans, is often chosen to be between 1–2 cM (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">B. L. Browning and S. R. Browning 2011</a>). However, this choice could depend on the behavior of the specific algorithm one uses and the research question one seeks to answer.</p><p id="p-5">A number of computer programs exist to detect IBD segments using high-density array data from population samples. Beagle IBD (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">S. R. Browning and B. L. Browning 2010</a>), RELATE (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrechtsen <em>et al.</em> 2009</a>), IBDLD (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Han and Abney 2011</a>), and PLINK (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Purcell <em>et al.</em> 2007</a>) detect IBD segments based on a probabilistic hidden Markov model for IBD status to determine the posterior probability of IBD at a genomic location. Probabilistic models tend to be computationally intensive (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">B. L. Browning and S. R. Browning 2013b</a>), thus it is often unfeasible to apply them to large population datasets. On the other hand, programs like GERMLINE (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gusev <em>et al.</em> 2009</a>), Beagle fastIBD (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">B. L. Browning and S. R. Browning 2011</a>), and Refined IBD (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">B. L. Browning and S. R. Browning 2013b</a>) adopt a dictionary approach to detect IBD, followed by varying degrees of probabilistic assessment of IBD segments to improve accuracies. These dictionary approaches are scalable to large sample datasets. These methods identify IBD segments by looking for long stretches of near-identical sequences (identity-by-state, or IBS). This is unambiguously useful; but to make use of these data in a model-based framework, some applications then assume that such segments, if long enough, have been uninterrupted by intervening recombination, and thus coinherited from a single common ancestor.</p><p id="p-6">This assumption, as we will examine in the current manuscript, does not always hold. It is clear that long segments of IBS could be due to the conflation of two or more shorter IBD segments, especially using diploid data, where the two IBD segments could be shared between different pairs of haplotypes. This error in the estimated length will in turn lead to errors in the estimated ages of common ancestors and bias downstream inferences. For example, if unmodeled, conflated segments would be inferred to derive from an erroneously recent common ancestor, thus biasing estimates of population size and migration rate. Similarly, and as we will show, an approach to estimate the mutation rate using the length of IBD segments would erroneously assign younger ages to conflations of older segments and thus overestimate the mutation rate.</p><p id="p-7">Conflation of neighboring IBD segments is one mechanism leading to misestimation in IBD calling. <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">B.L. Browning and S. R. Browning (2013b)</a> have previously examined the problem of overestimation of IBD segment lengths in IBD calling algorithms, implicitly grouping misestimation due to conflation with other factors such as genotyping errors. To help distinguish sources of error, we refer to “conflation” as the conflation of two neighboring long segments, resulting in a large error in estimated length, and “minor endpoint errors” as any other error that leads to a small error in estimated length. From the standpoint of the calling algorithm, at the end of a nonconflated segment the statistical evidence of IBD should decrease as sequence mismatches accumulate in regions where two individuals are very distantly related; eventually the evidence of IBD falls below the necessary threshold and the called segment is terminated. This is what we consider a minor endpoint error, which could be alleviated by simply removing ends of called IBD segments or more sophisticated statistical modeling in subsequent analysis. On the other hand, at the end of a conflated segment, the statistical evidence for IBD actually increases as the algorithm considers markers in the neighboring IBD segment, and continues until the end of the second segment. Thus, the problem of conflation of segments that we study here is one that will not disappear with increasing marker spacing or accuracy, as it is driven by the rate at which true segments of an appreciable length are spatially clustered in the genome. Indeed, it is this conflation that may have pushed the summed length of IBD in a region over the length threshold necessary for detection. Furthermore, this kind of error, as we show here, is more problematic for shorter segments near the calling threshold and in diploid organisms with imperfect phasing. Though researchers in the field recognize that IBD segment distributions are biased and often compensate for these effects via simulation or numerical procedures, our aim is to explicitly highlight and characterize this potential mechanism for the biased length distribution of IBD segments.</p><p id="p-8">To characterize the problem, we used coalescent simulations where we know the precise ancestral recombination graph [ARG (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hudson 1991</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Griffiths and Marjoram 1996</a>)] of the simulated sample. At all positions of a simulated sequence, the ARG relates individual haplotypes via a local genealogical tree. As in <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">B. L. Browning and S. R. Browning (2013b)</a>, we identify true IBD segments between two individuals as those stretches of contiguous sequence on which the time back to the most recent common ancestor between the two does not change (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A schematic relating IBDARG and IBDcalled segments. The cartoon shows the alignment of four haplotypes belonging to two diploid individuals. Across the region multiple ARGs exist to relate the four haplotypes in a tree. Across the first IBDARG region (orange, IBDARG,1), the two middle haplotypes have recent, unchanging, local graphs for the entire region. Across the second IBDARG region (red, IBDARG,2), two different haplotypes have recent, unchanging, local graphs for the entire region. The two IBDARG segments happens to occur near each other such that the entire region may be detected by algorithms based on long stretches of sequence similarity (IBDcalled segment). ARG, ancestral recombination graph; IBD, identity-by-descent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2109848378" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A schematic relating IBD&lt;sub&gt;ARG&lt;/sub&gt; and IBD&lt;sub&gt;called&lt;/sub&gt; segments. The cartoon shows the alignment of four haplotypes belonging to two diploid individuals. Across the region multiple ARGs exist to relate the four haplotypes in a tree. Across the first IBD&lt;sub&gt;ARG&lt;/sub&gt; region (orange, IBD&lt;sub&gt;ARG,1&lt;/sub&gt;), the two middle haplotypes have recent, unchanging, local graphs for the entire region. Across the second IBD&lt;sub&gt;ARG&lt;/sub&gt; region (red, IBD&lt;sub&gt;ARG,2&lt;/sub&gt;), two different haplotypes have recent, unchanging, local graphs for the entire region. The two IBD&lt;sub&gt;ARG&lt;/sub&gt; segments happens to occur near each other such that the entire region may be detected by algorithms based on long stretches of sequence similarity (IBD&lt;sub&gt;called&lt;/sub&gt; segment). ARG, ancestral recombination graph; IBD, identity-by-descent.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1287/F1.medium.gif" width="440" height="327"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/5/1287/F1.medium.gif" width="440" height="327"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468203" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">A schematic relating IBD<sub>ARG</sub> and IBD<sub>called</sub> segments. The cartoon shows the alignment of four haplotypes belonging to two diploid individuals. Across the region multiple ARGs exist to relate the four haplotypes in a tree. Across the first IBD<sub>ARG</sub> region (orange, IBD<sub>ARG,1</sub>), the two middle haplotypes have recent, unchanging, local graphs for the entire region. Across the second IBD<sub>ARG</sub> region (red, IBD<sub>ARG,2</sub>), two different haplotypes have recent, unchanging, local graphs for the entire region. The two IBD<sub>ARG</sub> segments happens to occur near each other such that the entire region may be detected by algorithms based on long stretches of sequence similarity (IBD<sub>called</sub> segment). ARG, ancestral recombination graph; IBD, identity-by-descent.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">We then simulated genotype data, from which we inferred IBD segments using published algorithms, and contrast the algorithm-detected IBD segments to the true IBD segments determined from the ARGs. We show that, under various demographic models appropriate for modern humans, a significant proportion of the detected IBD segments 1 cM or longer (predominately due to those between 1–2 cM long) are composed of at least two subsegments. As an example of how the conflation can lead to practical problems, we analyze the behavior of a potential estimator of the <em>de novo</em> mutation rate using IBD segments. We observed accurate estimates of the input mutation rate when true IBD segments were used, but overestimates of the mutation rate by ∼10–40% using inferred IBD segments, depending on the amount of conflation in the dataset. Our analysis focused on using one of the most recent IBD calling algorithms for array data, Refined IBD (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">B. L. Browning and S. R. Browning 2013b</a>), but we also tested a number of other algorithms designed to detect IBD segments from array data, and demonstrate that the conflation problem is faced by all algorithms tested.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Coalescent simulations</h3><p id="p-11">We simulated five 20 Mb regions of 2000 haplotypes each using MaCS (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2009</a>). To mimic genetic data of actual populations, we simulated under a demographic model previously estimated for a European population (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Nelson <em>et al.</em> 2012</a>). Specifically, we modeled a population with constant size: 12,500 diploid individuals up until 17,000 generations ago, 24,500 between 3500–17,000 generations ago, 7700 between 368–3500 generations ago, and exponentially expanding at a rate of 0.017 per generation (gen)to a present day size of 4 × 10<sup>6</sup>. We assumed constant mutation rate (1.2 × 10<sup>−8</sup> per bp per gen) and recombination rate (1 × 10<sup>−8</sup> per bp per gen). The MaCS command used is given in the <em>Appendix</em>.</p><p id="p-12">To generate phased sequence data from the MaCs output, we paired the genotypes from two randomly chosen simulated haplotypes. We introduced no missing data or genotyping errors. For our parameter settings, MaCS produced an average of 255,587 (ranged from 254,984 to 256,125) variants per simulated regions. To generate genome-wide (unphased) array data, we down-sampled the sequencing data to match the marker density and allele frequency spectrum typical of array data. The marker density was 5800 markers per 20 Mb region, which extrapolates to approximately 715,000 markers across the entire set of autosomes for which recombination rate estimates are available (approximately 2450 Mb). This density is typical of a genome-wide 1 M array after quality controls and filtering on common variants. The minor allele frequency spectrum was empirically matched to that found on an array, based on the array data from European individuals (<em>i.e.</em>, approximately a flat distribution for variants &gt; 5% frequency). Markers with minor allele frequency &lt; 5% were dropped from the array data. In total, an average of 5833 (range 5721–5944) variants were retained for the simulated array data for each simulated region. We next used simulated array data to call IBD segments with a number of available IBD calling algorithms, and then used the simulated sequencing data to infer mutation rate (below).</p><p id="p-13">In addition to the plausible demographic model from <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Nelson <em>et al.</em> (2012)</a>, we also tested a number of other demographic trajectories: the European and African demography from <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Tennessen <em>et al.</em> (2012)</a>, the same European model but without the second, rapid, exponential growth phase, as well as a constant size population of 100,000 individuals. Detailed descriptions of the demography and MaCS commands used for these scenarios can be found in the <em>Appendix</em>.</p></div><div id="sec-3" class="subsection"><h3>Detecting IBD<sub>called</sub> segments</h3><p id="p-14">The main algorithm of IBD detection tested here is Beagle’s Refined IBD (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">B. L. Browning and S. R. Browning 2013b</a>), though we also tested a number of other algorithms designed to detect IBD segments from array data, namely fastIBD (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">B. L. Browning and S. R. Browning 2011</a>), GERMLINE (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gusev <em>et al.</em> 2009</a>), IBDLD (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Han and Abney 2011</a>), and PLINK (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Purcell <em>et al.</em> 2007</a>).</p><p id="p-15">We followed the recommendations of the authors of Refined IBD/Beagle and called IBD segments in our simulated dataset using Beagle 4.0 (version 1399) with <em>ibdtrim</em> and <em>overlap</em> parameters set to typical marker density in 0.15 cM and 1.5 cM windows in the dataset, respectively. We used the default LOD score of 3. Only segments greater than 1 cM were retained for analysis. To avoid artifacts due to the edges of simulated regions, we also removed segments that overlapped the boundary of the simulated region. In total, we were left with 5459 IBD segments greater than 1 cM in length across the five simulated regions (between 1036–1157 segments per region).</p><p id="p-16">For comparison with IBD segments called by other algorithms, we also used GERMLINE (v.1.5.1), fastIBD (v.3.3.2), and IBDLD (v.3.004.4) to call IBD segments (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Gusev <em>et al.</em> 2009</a>; <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">B. L. Browning and S. R. Browning 2011</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Han and Abney 2013</a>). We applied each method to only a single simulated region to reduce the amount of computation.</p><p id="p-17">To estimate IBD segments using GERMLINE, we first subjected the simulated data to 10 iterations of phasing by Beagle. Following previous performance comparisons of GERMLINE to Refined IBD, we required the minimum length of the segment to be 1 Mb (–min_m = 1), and seed size for exact matching to be 32 (–bit 32). However, we set –err_hom and –err_het to be 0, as we introduced no genotyping errors in our simulation and higher values could spuriously lead to more apparent conflations. We also used the haploid mode (–h_extend), which allows GERMLINE to utilize the haplotype phase information, and should improve performance (<a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">B. L. Browning and S. R. Browning 2013b</a>). Default parameters were otherwise used. Increasing the value of bit parameter decreased the number of segments detected but produced otherwise qualitatively similar results (data not shown). We also used –haploid on the computationally phased data and obtained qualitatively similar results (data not shown). We did not test GERMLINE in haploid mode with perfectly phased data (from simulation) as it is unrealistic in practice. In total, we detected 786 IBD segments greater than 1 cM from one simulated region.</p><p id="p-18">Default settings were used to estimate IBD segments using fastIBD. Following the authors’ recommendations, IBD detection was conducted 10 independent times and then combined, retaining only segments supported by at least two independent runs and at least one segment reaching a confidence score below 10<sup>−10</sup>. We dropped approximately 1.3% of the segments that would otherwise be merged by criteria adopted by others who have similarly analyzed IBD segments detected by fastIBD (<a id="xref-ref-5-8" class="xref-bibr" href="#ref-5">B. L. Browning and S. R. Browning 2013b</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Ralph and Coop 2013</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Durand <em>et al.</em> 2014</a>), as we found that much of the need for merging may have come from genotyping errors or missingness, which are not allowed in simulations here. A total of 558 IBD segments greater than 1 cM from one simulated region were detected.</p><p id="p-19">Our initial run with IBDLD, using all 1000 diploid individuals for one simulated region, did not complete within 72 hr. Therefore, we randomly selected a subset of 500 diploid individuals for our evaluation. We used the method GIBDLD (–method GIBDLD) and chose to output only segments with length &gt; 1000 kb (–length 1000); otherwise, default settings were used. A total of 643 IBD segments were detected.</p><p id="p-20">We also tested PLINK [v1.07, (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Purcell <em>et al.</em> 2007</a>)]. As recommended, we first thinned the dataset so that no pair of SNPs within a window of 100 markers (moving at windows of 25 markers) would have an r<sup>2</sup> &gt; 0.2. However, estimation of IBD segments on the thinned dataset, using default settings, detected no segments. Less stringent thinning, increased marker density, a lowered minimum number of SNPs, or minimum segment length did not significantly improve detection of IBD segments using PLINK (data not shown). This is consistent with the lower power to detect IBD segments by PLINK as observed by others (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">S. R. Browning and B. L. Browning 2010</a>), and PLINK was thus not further evaluated.</p></div><div id="sec-4" class="subsection"><h3>Calling ground-truth IBD<sub>ARG</sub> segments from simulated data</h3><p id="p-21">We next used the simulated ARGs to determine the true IBD segments. MaCS reports the ARG as a sequence of genealogical trees along the simulated region, so that neighboring trees differ if a recombination event has occurred somewhere in that genealogical tree. Such an event may not affect the IBD status of a given pair of diploids, if it does not involve any of their four haplotypes. An IBD<sub>ARG</sub> segment shared by a pair of diploids is then defined as a continuous stretch of genealogies with unchanging age to common ancestor (<em>i.e.</em>, no recombination since the common ancestor) between at least one pair of the four pairwise haplotype configurations. This may conflate segments on which there has been an ancestral recombination but both common ancestors lived at the same time; but the chance of this is very small. We then aimed to identify all IBD<sub>ARG</sub> segments between all pairwise comparisons of diploid individuals in the simulation. However, it is computationally challenging to enumerate all such segments in an ARG, so we implemented two approximations to make the problem computationally tractable. First, we sampled a genealogy only every 0.01 cM (or 10 kb) of the simulated dataset, greatly reducing the number of genealogies that need to be processed. Second, we extract only IBD<sub>ARG</sub> segments less than 3000 generations old, greatly reducing the number of IBD segments to follow and keep in memory. Together, these approximations allowed us to obtain nearly all segments with length ≥ 0.2 cM at a cost of uncertain boundary resolution within 0.01 cM, as none of &gt; 5.7 million segments have age very close to 3000 generations (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/TableS1.pdf">Table S1</a>). While our approach may not detect all IBD<sub>ARG</sub> segments in the simulated dataset (especially the very short ones), we can ensure that all detected IBD<sub>ARG</sub> segments in this manner will be true IBD segments, free of any conflation effects.</p><p id="p-22">These simulations produced rates of IBD somewhat lower than published, empirical estimates. Our main simulation adopts a European demography (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Nelson <em>et al.</em> 2012</a>) that assumed a panmictic population experiencing an explosive recent expansion reaching an effective population size of 4,000,000. We detected 4208 IBD<sub>ARG</sub> segments &gt; 1cM among 1000 simulated individuals, which translates to an IBD rate of 8.42 × 10<sup>−3</sup> segments per pair of individuals per 100 Mb region. For comparison, <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Ralph and Coop (2013)</a> used fastIBD and detected ∼1,900,000 (IBD<sub>called</sub>) segments &gt; 1 cM genome-wide in a population of ∼2200 general Europeans. This translates to an IBD rate of approximately 29.95 × 10<sup>−3</sup> segments per pair of individuals. The difference in rates is likely due to unmodeled local population structure, which increases recent coalescence rates relative to a randomly mating population, as well as due to other differences such as imperfectly modeled demography, marker density, genotyping errors, and false positives of the calling algorithm.</p></div><div id="sec-5" class="subsection"><h3>Estimating mutation rate based on IBD segments</h3><p id="p-23">There has been recent interest in estimating mutation rates based on principles of IBD sharing to complement existing approaches from trios and phylogenetic analyses (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Campbell <em>et al.</em> 2012</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Palamara <em>et al.</em> 2015</a>). Here, we adopted a simplistic estimator of mutation rate based on IBD, similar in spirit to that used in <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Campbell <em>et al.</em> (2012)</a> with autozygous segments, assuming that the phase and demographic history of the sample are known and that no genotyping or sequencing errors exist. We will show that without taking the conflation phenomenon into consideration, our mutation rate estimates will be biased. However, we note that the studies referenced here have utilized external information, such as pedigrees and trio-based phasing, to implicitly overcome the conflation problem; see <em>Discussion</em>.</p><p id="p-24">Differences between IBD segments must have occurred more recently than the common ancestor giving rise to the IBD. Therefore, an estimator of the mutation rate μ is:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1287/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1287/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>I</em> is the set of all IBD segments, and for a given IBD segment <em>i</em>, <em>m(i)</em> is the number of sequence mismatches on the two IBD haplotypes, <em>L<sub>seq</sub>(i)</em> is the total sequenced region (= the length of the IBD segment, <em>L<sub>IBD</sub>(i)</em> (if completely sequenced), and <em>T<sub>IBD</sub>(i)</em> is time since the common ancestor in generation.</p></div><div id="sec-6" class="subsection"><h3>Simulating conflated segments to test the impact of conflation on mutation rate estimator</h3><p id="p-25">Based on all of the IBD<sub>ARG</sub> segments with length &gt; 0.2 cM, we first estimate the distribution of IBD segment ages given segment length in the simulation. As suggested by visual inspection, we binned lengths to 0.1 cM resolution, and for each length bin, fitted a mixture of two γ distributions to the true distribution of ages in that bin using an EM algorithm implemented in the R package mixtools (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/TableS2.pdf">Table S2</a>). In practice, this distribution will change depending on the underlying demography of the population and will likely be difficult to estimate. However, for the purpose of our illustration here, we assumed that the demography is known.</p><p id="p-26">Then, to simulate conflated segments in a more efficient manner than using the coalescent, we sampled IBD segment lengths from the apparent length distribution and assigned each sampled IBD segment as a conflated segment or a nonconflated segment, depending on its length and the appropriate proportion of conflation given its length observed in the simulations. For nonconflated segments, we sampled an age, <em>T<sub>UC</sub></em>, from the age distribution given its length, <em>L<sub>UC</sub></em>, using the mixture γ (parameters given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/TableS2.pdf">Table S2</a>). The number of sequence mismatches in each IBD segment were then sampled from a Poisson distribution with mean <em>2L<sub>UC</sub>T<sub>UC</sub>μ</em>, where μ is the mutation rate and set to 1.2 × 10<sup>−8</sup>.</p><p id="p-27">For conflated segments, we make the simplifying assumption that the entire segment of length <em>L<sub>C</sub></em> is composed of only two subsegments of length <em>L<sub>C1</sub></em> and <em>L<sub>C2</sub></em>, with no gap. <em>L<sub>c</sub></em> is sampled from the apparent length distribution (with conflation), <em>L<sub>C1</sub></em> is sampled from the true length distribution of IBD<sub>ARG</sub> segments, and <em>L<sub>C2</sub> = L<sub>C</sub> – L<sub>C1</sub></em>. For each pair of subsegments, we then sampled ages <em>T<sub>C1</sub></em> and <em>T<sub>C2</sub></em>, and sampled numbers of sequence mismatches as above.</p><p id="p-28">For both conflated and nonconflated segments, we also sampled the apparent age based on the apparent length (the full length for nonconflated segments, and the sum of the two lengths for conflated segments). These will be the estimated <em>T<sub>IBD</sub></em> used in Equation 1. We generated 1000 sets of 5000 IBD segments (roughly the number of segments &gt; 1 cM detected by Refined IBD across the five 20 Mb regions), and report the median, fifth and 95th percentile estimates of the mutation rate.</p><p id="p-29">Note that our strategy here is aimed at examining the potential impact of conflation, and conflation alone, on our mutation rate estimator in a conservative manner. As such, we assumed that there are no genotyping or sequencing errors, the phase and the demography are known, and that there is no misestimation of the IBD segment boundaries and no gaps between the conflated segments. In practice, these are all issues that can further confound the analysis.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-30">Codes used for analysis in this paper are available upon request.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>The prevalence of subsegment conflations among called IBD segments</h3><p id="p-31">Across the simulations, Refined IBD called a total of 5459 IBD<sub>called</sub> segments. A random subset of 250 IBD<sub>called</sub> segments, together with how they are composed of IBD<sub>ARG</sub> segments, is displayed in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>; the entire dataset is displayed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/FigureS1.pdf">Figure S1</a>. Some of the subsegments completely overlapped with the longest IBD<sub>called</sub> segment due to diploidy, but 22.4% of the IBD<sub>called</sub> segments with length &gt; 1 cM include portions of IBD<sub>ARG</sub> segments that are shorter but at least 0.2 cM long, 17.5% of the IBD<sub>called</sub> segments have their length extended by at least 0.2 cM (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). When extended, the length of the IBD segments is extended by an average of 0.31 cM. One illustration of a conflated segment is shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, where between the four possible pairs of haplotypes between two diploid individuals, two separate segments were inherited from two sets of recent, but different, common ancestors that are in close proximity to each other, leading the algorithm to call the entire region IBD.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The prevalence of subsegments among algorithm-detected IBD segments. Each of a set of 250 randomly chosen IBDcalled segments detected by Refined IBD is represented by a vertical bar. The IBDcalled segments are sorted along the x-axis according to the detected length in decreasing order. For each IBDcalled segment, the longest intersecting IBDARG segment is shown in yellow. The second longest intersecting IBDARG subsegment, if present, is shown in blue. The overlap between the two longest IBDARG segments, if any, is shown in olive green. The remainder of the detected region is clumped in black. For each IBDcalled segment displayed, we also show the number of subsegments &gt; 0.2 cM detected in simulation using the vertical axis on the right. For completeness, we also display cases where the second IBDARG segment is completely overlapping the longest IBDARG segment, in which case it would not confound the calling algorithm. See Figure S3 for results based on IBDcalled segments detected by GERMLINE, fastIBD, and IBDLD. ARG, ancestral recombination graph; IBD, identity-by-descent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2109848378" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The prevalence of subsegments among algorithm-detected IBD segments. Each of a set of 250 randomly chosen IBD&lt;sub&gt;called&lt;/sub&gt; segments detected by Refined IBD is represented by a vertical bar. The IBD&lt;sub&gt;called&lt;/sub&gt; segments are sorted along the x-axis according to the detected length in decreasing order. For each IBD&lt;sub&gt;called&lt;/sub&gt; segment, the longest intersecting IBD&lt;sub&gt;ARG&lt;/sub&gt; segment is shown in yellow. The second longest intersecting IBD&lt;sub&gt;ARG&lt;/sub&gt; subsegment, if present, is shown in blue. The overlap between the two longest IBD&lt;sub&gt;ARG&lt;/sub&gt; segments, if any, is shown in olive green. The remainder of the detected region is clumped in black. For each IBD&lt;sub&gt;called&lt;/sub&gt; segment displayed, we also show the number of subsegments &gt; 0.2 cM detected in simulation using the vertical axis on the right. For completeness, we also display cases where the second IBD&lt;sub&gt;ARG&lt;/sub&gt; segment is completely overlapping the longest IBD&lt;sub&gt;ARG&lt;/sub&gt; segment, in which case it would not confound the calling algorithm. See Figure S3 for results based on IBD&lt;sub&gt;called&lt;/sub&gt; segments detected by GERMLINE, fastIBD, and IBDLD. ARG, ancestral recombination graph; IBD, identity-by-descent.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1287/F2.medium.gif" width="440" height="247"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/5/1287/F2.medium.gif" width="440" height="247"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468116" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">The prevalence of subsegments among algorithm-detected IBD segments. Each of a set of 250 randomly chosen IBD<sub>called</sub> segments detected by Refined IBD is represented by a vertical bar. The IBD<sub>called</sub> segments are sorted along the x-axis according to the detected length in decreasing order. For each IBD<sub>called</sub> segment, the longest intersecting IBD<sub>ARG</sub> segment is shown in yellow. The second longest intersecting IBD<sub>ARG</sub> subsegment, if present, is shown in blue. The overlap between the two longest IBD<sub>ARG</sub> segments, if any, is shown in olive green. The remainder of the detected region is clumped in black. For each IBD<sub>called</sub> segment displayed, we also show the number of subsegments &gt; 0.2 cM detected in simulation using the vertical axis on the right. For completeness, we also display cases where the second IBD<sub>ARG</sub> segment is completely overlapping the longest IBD<sub>ARG</sub> segment, in which case it would not confound the calling algorithm. See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/FigureS3.pdf">Figure S3</a> for results based on IBD<sub>called</sub> segments detected by GERMLINE, fastIBD, and IBDLD. ARG, ancestral recombination graph; IBD, identity-by-descent.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The conflation effect as a function of the length of IBDcalled segments. (A) The complementary cumulative distribution functions (i.e., 1-CDF) for the total length extended due to subsegments &gt; 0.2 cM (except for the longest IBDARG segment). The distributions are also stratified by four levels of length: Between 1&#x2013;1.25 cM, between 1.25&#x2013;1.5 cM, between 1.5&#x2013;1.75 cM, and &gt; 1.75 cM. The conflation effect is generally driven by segments &lt; 1.75 cM in detected length. (B) The biases in estimated length due to subsegments and end point errors as a function of the estimated length. We binned all IBDcalled segments in 7 bins: [1, 1.2), [1.2, 1.4), [1.4, 1.6), [1.6, 1.8), [1.8, 2), [2, 2.2), [2.2, 20), and for each bin examined the average length extended (from both ends) beyond the longest IBDARG segment found in the called region due to either a subsegment &gt; 0.2 cM (blue), or other minor endpoint errors and gaps between subsegments (black). Each data point is plotted on the x-axis at the median length of the bin. ARG, ancestral recombination graph; CDF, cumulative distribution function; IBD, identity-by-descent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2109848378" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The conflation effect as a function of the length of IBD&lt;sub&gt;called&lt;/sub&gt; segments. (A) The complementary cumulative distribution functions (&lt;em&gt;i.e.&lt;/em&gt;, 1-CDF) for the total length extended due to subsegments &gt; 0.2 cM (except for the longest IBD&lt;sub&gt;ARG&lt;/sub&gt; segment). The distributions are also stratified by four levels of length: Between 1&#x2013;1.25 cM, between 1.25&#x2013;1.5 cM, between 1.5&#x2013;1.75 cM, and &gt; 1.75 cM. The conflation effect is generally driven by segments &lt; 1.75 cM in detected length. (B) The biases in estimated length due to subsegments and end point errors as a function of the estimated length. We binned all IBD&lt;sub&gt;called&lt;/sub&gt; segments in 7 bins: [1, 1.2), [1.2, 1.4), [1.4, 1.6), [1.6, 1.8), [1.8, 2), [2, 2.2), [2.2, 20), and for each bin examined the average length extended (from both ends) beyond the longest IBD&lt;sub&gt;ARG&lt;/sub&gt; segment found in the called region due to either a subsegment &gt; 0.2 cM (blue), or other minor endpoint errors and gaps between subsegments (black). Each data point is plotted on the x-axis at the median length of the bin. ARG, ancestral recombination graph; CDF, cumulative distribution function; IBD, identity-by-descent.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1287/F3.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/5/1287/F3.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468013" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">The conflation effect as a function of the length of IBD<sub>called</sub> segments. (A) The complementary cumulative distribution functions (<em>i.e.</em>, 1-CDF) for the total length extended due to subsegments &gt; 0.2 cM (except for the longest IBD<sub>ARG</sub> segment). The distributions are also stratified by four levels of length: Between 1–1.25 cM, between 1.25–1.5 cM, between 1.5–1.75 cM, and &gt; 1.75 cM. The conflation effect is generally driven by segments &lt; 1.75 cM in detected length. (B) The biases in estimated length due to subsegments and end point errors as a function of the estimated length. We binned all IBD<sub>called</sub> segments in 7 bins: [1, 1.2), [1.2, 1.4), [1.4, 1.6), [1.6, 1.8), [1.8, 2), [2, 2.2), [2.2, 20), and for each bin examined the average length extended (from both ends) beyond the longest IBD<sub>ARG</sub> segment found in the called region due to either a subsegment &gt; 0.2 cM (blue), or other minor endpoint errors and gaps between subsegments (black). Each data point is plotted on the x-axis at the median length of the bin. ARG, ancestral recombination graph; CDF, cumulative distribution function; IBD, identity-by-descent.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An illustrative example of the conflation effect on the mutation rate estimator. For a particular IBDcalled segment of length 1.145 cM, we show the distribution of IBDARG subsegment age (y-axis) as a function of position (x-axis, between 9.45&#x2013;10.59 Mb of a 20 Mb simulated region). Each of the four different pairwise haplotype configurations between the two diploid samples is illustrated with a different color. The simulated haplotype numbers are displayed in the upper right hand corner. The vertical dashed lines demarcate the 10% segment length from both ends of the segment that one could remove from analysis due to the uncertainty in estimating the ends of the IBDcalled segments. The age of each subsegment is plotted as a step function of its length. In this case, the IBD region is dominated by two long segments of IBD, one between simulated haplotypes 1865 and 654, another between simulated haplotypes 911 and 654. (There is actually a third, very short, segment of recent coalescence between simulated haplotypes 911 and 654 that is not obvious here.) Regions that do not produce long IBD segments can be clearly seen with the deep coalescences. In this case, the predominate IBD haplotype should be between haplotypes 1865 and 654, but the conflation with a neighboring IBD haplotype between haplotypes 911 and 654 led to the estimation of a single long IBD segment. ARG, ancestral recombination graph; IBD, identity-by-descent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2109848378" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;An illustrative example of the conflation effect on the mutation rate estimator. For a particular IBD&lt;sub&gt;called&lt;/sub&gt; segment of length 1.145 cM, we show the distribution of IBD&lt;sub&gt;ARG&lt;/sub&gt; subsegment age (y-axis) as a function of position (x-axis, between 9.45&#x2013;10.59 Mb of a 20 Mb simulated region). Each of the four different pairwise haplotype configurations between the two diploid samples is illustrated with a different color. The simulated haplotype numbers are displayed in the upper right hand corner. The vertical dashed lines demarcate the 10% segment length from both ends of the segment that one could remove from analysis due to the uncertainty in estimating the ends of the IBD&lt;sub&gt;called&lt;/sub&gt; segments. The age of each subsegment is plotted as a step function of its length. In this case, the IBD region is dominated by two long segments of IBD, one between simulated haplotypes 1865 and 654, another between simulated haplotypes 911 and 654. (There is actually a third, very short, segment of recent coalescence between simulated haplotypes 911 and 654 that is not obvious here.) Regions that do not produce long IBD segments can be clearly seen with the deep coalescences. In this case, the predominate IBD haplotype should be between haplotypes 1865 and 654, but the conflation with a neighboring IBD haplotype between haplotypes 911 and 654 led to the estimation of a single long IBD segment. ARG, ancestral recombination graph; IBD, identity-by-descent.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1287/F4.medium.gif" width="377" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/5/1287/F4.medium.gif" width="377" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468156" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">An illustrative example of the conflation effect on the mutation rate estimator. For a particular IBD<sub>called</sub> segment of length 1.145 cM, we show the distribution of IBD<sub>ARG</sub> subsegment age (y-axis) as a function of position (x-axis, between 9.45–10.59 Mb of a 20 Mb simulated region). Each of the four different pairwise haplotype configurations between the two diploid samples is illustrated with a different color. The simulated haplotype numbers are displayed in the upper right hand corner. The vertical dashed lines demarcate the 10% segment length from both ends of the segment that one could remove from analysis due to the uncertainty in estimating the ends of the IBD<sub>called</sub> segments. The age of each subsegment is plotted as a step function of its length. In this case, the IBD region is dominated by two long segments of IBD, one between simulated haplotypes 1865 and 654, another between simulated haplotypes 911 and 654. (There is actually a third, very short, segment of recent coalescence between simulated haplotypes 911 and 654 that is not obvious here.) Regions that do not produce long IBD segments can be clearly seen with the deep coalescences. In this case, the predominate IBD haplotype should be between haplotypes 1865 and 654, but the conflation with a neighboring IBD haplotype between haplotypes 911 and 654 led to the estimation of a single long IBD segment. ARG, ancestral recombination graph; IBD, identity-by-descent.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Comparing segment length biases due to conflations as a function of the called segment length, we see that the contribution due to conflation of subsegments &gt; 0.2 cM is greater than that due to minor endpoint errors and gaps in the calling (defined here as the regions of the called segment that are not attributed to a subsegment &gt; 0.2 cM) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). The minor endpoint and gap errors are relatively independent of the length of the IBD<sub>called</sub> segment length, while biases due to conflation are concentrated among the shorter segments (&lt; 1.75 cM, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, A and B</a>). Discarding 0.1 cM from either end of the IBD<sub>called</sub> segments improved the calling slightly, reducing the proportion of conflated segments to 19.9% (12.9% of segments have their length extended by at least 0.2 cM), with an average of 0.26 cM increase in length when a segment is extended. However, the improvement is mostly due to reducing the minor endpoint errors, with little impact on the errors due to conflation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/FigureS2.pdf">Figure S2</a>). Together, these results suggest that a significant contributor to the biases in the length of IBD<sub>called</sub> segments is the conflation of subsegments that are themselves stretches of IBD inherited from a relatively recent, though different, common ancestor. The conflation error also differs from the typical minor inaccuracies of detecting endpoints, in that the overlap with a true segment does not improve if we discard the ends of the called segment.</p><p id="p-36">While we have thus far focused on IBD<sub>called</sub> segments produced by the Refined IBD program, our observation should not be specific to Beagle, but is more generally applicable to any IBD calling algorithm so long as these programs depend solely on sequence identity for inference. To test this hypothesis, we also used fastIBD (<a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">B. L. Browning and S. R. Browning 2011</a>), GERMLINE (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Gusev <em>et al.</em> 2009</a>), and IBDLD (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Han and Abney 2013</a>) to call IBD segments in each of our simulated regions. We observed the same conflation effects across these algorithms, though different algorithms were affected to different degrees (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/FigureS4.pdf">Figure S4</a>).</p><p id="p-37">Finally, the conflation effect is also not specific to the demography we simulated. In addition to the European-like human demography estimated in <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Nelson <em>et al.</em> (2012)</a>, we also simulated under both the European and African demographic histories published in <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Tennessen <em>et al.</em> (2012)</a>, an alternative European-like model without the most recent exponential growth phase, as well as a constant size population. Though different demographic scenarios are affected by the conflation events to different degrees, the conflation effect is evident in all cases (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/FigureS5.pdf">Figure S5</a>).</p></div><div id="sec-10" class="subsection"><h3>The rate of conflation of true IBD segments in the simulated ancestral recombination graphs</h3><p id="p-38">Though we have observed significant conflation among IBD segments called by a number of algorithms, it remains possible that this is particular to the technical aspects of the calling algorithm, and avoidable somehow. Thus, we next characterized the rate of conflation events between true IBD segments identified from the simulated ancestral recombination graph. We define a conflation event in our simulated data as when two IBD<sub>ARG</sub> segments, each of length at least <em>w</em> cM, separated by a gap no greater than a small number 0 (two subsegments overlap) or 0.01 cM, together account for an end-to-end length of at least 1 cM. The simplest model of IBD segment conflation is that, given the length distribution determined by the coalescent time distribution, observed long IBD segments are uniformly and independently distributed along the genome. To test this, we simulated 100 replicate datasets by permutation where we assigned each segment randomly (with replacement) to two diploid individuals uniformly across the genomic regions.</p><p id="p-39">We observe that the conflation rate increases substantially as increasingly smaller IBD<sub>ARG</sub> segments are allowed (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/FigureS6.pdf">Figure S6</a>). When counting conflation events due to segments as small as 0.2 cM, the rate of conflation (2.66 events per 1000 pairs of individuals per 100 Mb) is actually on the same order of magnitude as the true IBD rate &gt; 1 cM in length (8.42 events per 1000 pairs of individuals per 100 Mb) in the simulated dataset. When compared with the simulated replicates, the levels of conflation are comparable (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>), suggesting that the majority of the observed conflations are due to the random independent placement of IBD segments along chromosomes. Therefore, any potential correlation in long segment placement induced by the haploid coalescent or the pedigree structure appears minimal.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Conflations of shorter IBD segments will bias the length distribution. (A) For each bin of segment length range, we calculated the rate at which two IBDARG segments in our simulations, both within the length range, are adjacent and together constitute an end-to-end length of at least 1 cM (i.e., maximum gap sizes = 0 cM and combined length &gt; 1 cM). The blue dot is the actual value observed in simulation. The boxplot shows the variance around the observed value by randomly sampling from the observed segment length distribution but randomly assigning the location of a segment and sample IDs 100 times. (B) The biased length distribution if each conflated IBDARG segment is counted for its conflated length rather than the two true lengths. Note that the apparent length of each conflated segment is due to conflation of two IBDARG segments, independent of any imprecision due to algorithm calling. Dotted line is the true length distribution if each conflated segment can be resolved based on the coalescent genealogy. Inset shows the comparison between the biased length distribution and the true length distribution in log scale. For results based on a maximum gap size of 0.01 cM, refer to Figure S6. ARG, ancestral recombination graph; IBD, identity-by-descent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2109848378" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Conflations of shorter IBD segments will bias the length distribution. (A) For each bin of segment length range, we calculated the rate at which two IBD&lt;sub&gt;ARG&lt;/sub&gt; segments in our simulations, both within the length range, are adjacent and together constitute an end-to-end length of at least 1 cM (&lt;em&gt;i.e.&lt;/em&gt;, maximum gap sizes = 0 cM and combined length &gt; 1 cM). The blue dot is the actual value observed in simulation. The boxplot shows the variance around the observed value by randomly sampling from the observed segment length distribution but randomly assigning the location of a segment and sample IDs 100 times. (B) The biased length distribution if each conflated IBD&lt;sub&gt;ARG&lt;/sub&gt; segment is counted for its conflated length rather than the two true lengths. Note that the apparent length of each conflated segment is due to conflation of two IBD&lt;sub&gt;ARG&lt;/sub&gt; segments, independent of any imprecision due to algorithm calling. Dotted line is the true length distribution if each conflated segment can be resolved based on the coalescent genealogy. Inset shows the comparison between the biased length distribution and the true length distribution in log scale. For results based on a maximum gap size of 0.01 cM, refer to Figure S6. ARG, ancestral recombination graph; IBD, identity-by-descent.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1287/F5.medium.gif" width="440" height="177"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/5/1287/F5.medium.gif" width="440" height="177"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1287/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468191" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-40" class="first-child">Conflations of shorter IBD segments will bias the length distribution. (A) For each bin of segment length range, we calculated the rate at which two IBD<sub>ARG</sub> segments in our simulations, both within the length range, are adjacent and together constitute an end-to-end length of at least 1 cM (<em>i.e.</em>, maximum gap sizes = 0 cM and combined length &gt; 1 cM). The blue dot is the actual value observed in simulation. The boxplot shows the variance around the observed value by randomly sampling from the observed segment length distribution but randomly assigning the location of a segment and sample IDs 100 times. (B) The biased length distribution if each conflated IBD<sub>ARG</sub> segment is counted for its conflated length rather than the two true lengths. Note that the apparent length of each conflated segment is due to conflation of two IBD<sub>ARG</sub> segments, independent of any imprecision due to algorithm calling. Dotted line is the true length distribution if each conflated segment can be resolved based on the coalescent genealogy. Inset shows the comparison between the biased length distribution and the true length distribution in log scale. For results based on a maximum gap size of 0.01 cM, refer to <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/FigureS6.pdf">Figure S6</a>. ARG, ancestral recombination graph; IBD, identity-by-descent.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">To further investigate this, one can also assign each pair of conflated segments as being in <em>cis</em> or <em>trans</em> based on whether the pairs involved 2, 3, or 4 of the possible four haplotypes: a pair is <em>cis</em> if both IBD segments were shared by the same two (out of the four possible) haplotypes, implying that the segments descend from recent but different common ancestors; the pair is <em>trans</em> if the pairs involve three or four of the possible haplotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/FigureS7.pdf">Figure S7</a>). In theory, <em>trans</em> conflations should only be problematic to a haploid-based IBD detection algorithm if phasing is imperfect, and thus represent the amount of conflation problems that can be avoided with better phasing. We find that approximately 80% of the conflations occur <em>in trans</em> rather than <em>in cis</em> (2.13 events <em>vs.</em> 0.498 events per 1000 pairs of individuals per 100 Mb). As there are three times as many possible <em>trans</em> arrangements as <em>cis</em>, the proportion predicted by independent placement, 75%, is within the confidence interval.</p><p id="p-42">The observed rate of conflation leads to a noticeable bias in the length distribution of apparent IBD segments with length greater than 1 cM (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/FigureS6.pdf">Figure S6</a>). Specifically, approximately 27% of the apparent IBD segments between 1–1.2 cM long are due to conflation.</p></div><div id="sec-11" class="subsection"><h3>Evaluating the impact of conflation on a mutation rate estimator</h3><p id="p-43">We evaluated the impact of the biased length distribution of IBD segments due to conflation on a mutation rate estimator. Given accurately defined IBD segments and perfect sequence and phase information, the sequence mismatches between a pair of individuals in the IBD segment reveals mutations that have arisen since the common ancestry of the two IBD haplotypes. Therefore, we devised a simplistic mutation rate estimator based on observed sequence mismatches between a pair of IBD haplotypes and a conservative heuristic to generate simulated data (<em>Methods</em>).</p><p id="p-44">When we applied the mutation rate estimator on a set of nonconflated segments &gt; 1 cM (<em>i.e.</em>, all IBD segments are called perfectly), we would estimate the same mutation rate (1.201 × 10<sup>−8</sup>/bp/gen; fifth and 95th percentiles = 1.174 × 10<sup>−8</sup> and 1.228 × 10<sup>−8</sup> /bp/gen, respectively) as the value used to simulate the data (1.20 × 10<sup>−8</sup> /bp/gen). When the estimator is applied on a set of conflated segments, all composed of two subsegments, we would estimate μ to be 1.647 × 10<sup>−8</sup> /bp/gen (fifth and 95th percentiles = 1.596 × 10<sup>−8</sup> and 1.702 × 10<sup>−8</sup> /bp/gen, respectively), which is a 37% inflation. When we applied the estimator to a mixture set composed of ∼34% conflated segments (as would be predicted by <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>), the estimated μ is 1.311 × 10<sup>−8</sup> /bp/gen (fifth and 95th percentiles = 1.287 × 10<sup>−8</sup> and 1.337 × 10<sup>−8</sup>/bp/gen), a more modest 9% inflation. As one increases the length cut-off for IBD segments for analysis, the median estimate of μ becomes less inflated, though still higher than the true value due to the persistent low level of conflation, and with a wider confidence interval due to fewer IBD segments for analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/TableS3.pd">Table S3</a>).</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-45">A number of population genetic analyses focus on the information provided by the IBD segment length distribution. So far, nearly all applications assume that each detected IBD segment is derived from a single common ancestor. However, the observed IBD length distribution is often biased, and extensive efforts are undertaken to empirically correct the observed distribution in order to ensure the validity of the downstream conclusions. In this manuscript, we studied this core assumption, and examined through simulations the phenomenon of conflation of shorter segments. Our characterization of this phenomenon shed light on one of the mechanisms by which the IBD length distribution is biased in practice.</p><p id="p-46">We have shown that using simulated SNP array data, under a reasonable demography for western European humans, and irrespective of the choice of IBD-calling algorithm examined, an appreciable proportion of called IBD segments shorter than 2 cM are composed of conflations of shorter, more distantly related, subsegments. For instance, of segments &gt; 1 cM inferred using Refined IBD, greater than 22% have the IBD segment length extended due to other short subsegments &gt; 0.2 cM; most of these were extended by 0.2–0.5 cM. The rate of conflation is reduced to ∼6% for longer IBD segments (&gt; 2 cM) (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>). In our simulations, chance conflations often arose between independent, shorter segments (<em>e.g.</em>, two of sizes 0.5 cM); this creates more segments near the shorter end of our detection threshold (1 cM) than longer because the true segment length distribution is so strongly peaked near zero (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>). Furthermore, we have shown that the conflation errors are unlike typical minor endpoint estimation errors, which would be improved by removing ends of the segments from downstream analysis (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1/FigureS2.pdf">Figure S2</a>). However, we note that the severity of the conflation problem may differ in real data, as other factors such as the marker density and the underlying, unobserved population demography could also be in play.</p><p id="p-47">The conflation effect described here could potentially in part explain the recent observation of pervasive false-positive IBD segments detected in pedigrees (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Durand <em>et al.</em> 2014</a>). In <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Durand <em>et al.</em> (2014)</a>, the authors noted that over 67% of the IBD segments they identified with GERMLINE between a child in a trio to an unrelated individual are not observed between either of the parents in the trio to the same unrelated individual. This false positive effect is also exacerbated near the algorithm’s length threshold for making a call. Conceivably, at least a subset of these false positive events is not due to blatant errors in the calling algorithm. Instead, some of the IBD segments called in the child-to-other comparisons could be due to conflations of IBD segments, independently inherited from the two parents. As a result, the full-length IBD segments are not observed in either of the parents. However, also note that implementation differences of the pipelines utilizing GERMLINE could cloud direct comparisons. For example, <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Durand <em>et al.</em> (2014)</a> used different length thresholds as used here, and did not use the haploid mode of GERMLINE, which is expected to improve performance.</p><p id="p-48">The failure to account for the biased length distribution of IBD segments due to conflation would lead to errors in downstream analysis, which we demonstrated with a simplistic estimator of the mutation rate based on IBD segments. While we focus only on the issue of conflation, other hurdles exist to estimating mutation rates from IBD segments, such as estimating the age of the MRCA or dealing with imperfect computational phasing. Indeed, utilizing external information (such as focusing on autozygous segments or using trio-phasing) to overcome these issues, as well as modeling of genotyping errors, are the focus of recent attempts to estimate mutation rate using the IBD concept (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Campbell <em>et al.</em> 2012</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Palamara <em>et al.</em> 2015</a>). However, the external information used by these studies is not always available, and would reduce the sample sizes available for analysis. As such, there remains room for improvement.</p><p id="p-49">Other types of analyses that rely heavily on the length and age distribution of IBD segments are demographic inferences based on IBD segments. For example, <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Ralph and Coop (2013)</a> devised a nonparametric approach to infer the number and age of recent shared ancestors between populations based on IBD sharing between populations. This provides estimates of distributions of the ages of IBD segments given their lengths, but analytic difficulties inherent to demographic inference may make these quite noisy. <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Palamara <em>et al.</em> (2012)</a> avoided this by fitting demographic models parameterized with few parameters. In both cases, it would be preferable to directly model conflation of short segments, although both carefully corrected the observed length distribution of IBD segments to reach robust conclusions. <a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Ralph and Coop (2013)</a> focused only on IBD segments longer than 2 cM, which we show could remove the majority of conflation effects, and further modeled the power, false-positive rates, and length misestimation of IBD detection. Instead of using a hard cut-off, <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Palamara <em>et al.</em> (2012)</a> used an approach that iterates between demographic inference and IBD detection, until they converged on a set of parameters for GERMLINE that minimized the difference between called and true IBDs from simulation specific to the data at hand, prior to the formal demographic inference procedures. Alternative approaches using IBS directly rather than IBD can also be taken, as the length of IBS segments can be more precisely obtained given high-coverage sequencing data (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Harris and Nielsen 2013</a>). It will be interesting to investigate if a theoretical framework contrasting IBS and apparent IBD segments can predict the rate of conflated segments within a dataset.</p><p id="p-50">Though we focused on an IBD detection framework using high-density array data, as the field develops algorithms aimed to detect even shorter segments due to the improved density of markers in the era of whole genome sequencing [<em>e.g.</em>, programs like IBDseq (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">B. L. Browning and S. R. Browning 2013a</a>)], it seems imperative to note and reevaluate the potential problem of conflation. We found that ∼75–80% of the conflated segments are the result of pairs of segments existing <em>in trans</em>, suggesting that improvements in phasing may greatly reduce, but not completely eradicate, biases due to conflated segments. Therefore, in addition to improvements in phasing, considerations of the conflation effect and external information such as availability of other individuals in the pedigree could all contribute toward improved accuracy in IBD detection and interpretation.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">The authors thank Kirk E. Lohmueller and Alexander Platt for helpful discussions and critical reading of the manuscript. C.W.K.C. is supported by a National Institutes of Health (NIH) Ruth L. Kirschstein National Research Service Award for postdoctoral fellows (F32GM106656) and T32NS048004 from National Institute of Neurological Disorders and Strokes. J.N. is supported by NIH grant R01 HG007089 and R01 GM108805. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institute of Health.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-13"><h3>MaCS Commands for Simulations</h3><p>The primary demographic model used throughout the manuscript is one based on deep resequencing of 202 genes across 14,000 European individuals (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Nelson <em>et al.</em> 2012</a>): the population has ancestral size of 12,500 up until 17,000 generations ago, 24,500 between 3500–17,000 generations ago, 7700 between 368–3500 generations ago, and exponentially expanding at a rate of 0.017 per generation to a present day size of 4 × 10<sup>6</sup>. The MaCS command used is:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-52">macs 2000 20000000 -T -t 0.192 -r 0.16 -G 0.017 -eN 0.000023 0.001925 -eN 0.00021875 0.006 -eN 0.0010625 0.003125</p></li></ul><p>All input parameters are scaled by 4N<sub>0</sub> for MaCS, with N<sub>0</sub>, the present day effective population size, equaling 4 × 10<sup>6</sup> in this case.</p><p id="p-53">Moreover, we simulated under a number of other demographic scenarios, primarily based on those published by <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Tennessen <em>et al.</em> (2012)</a>. These include: the European and African demography from <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Tennessen <em>et al.</em> (2012)</a>, a European model but without the aggressive second exponential growth phase, as well as a constant size population. Specifically, the African demography was modeled with a population of size 7310 until 1480 generations ago, 14,474 between 205–1480 generations ago, and then exponentially growing at rate 0.0165 per generation until reaching a present day size of 424,000. For the European demography, we modeled a population with ancestral size 7310, that has a population size of 14,474 between 2040–5920 generations ago, 1861 between 920–5920 generations ago, after which the population experienced exponential growth at rate 0.003 per generation until 204 generations ago, when the growth rate increases to 0.0195 per generation to reach a present day size of 496,500. The alternative European model without the aggressive second exponential growth phase is similar, but reaching only a present day size of 9300. Finally, the constant size population has a constant size of 100,000. The MaCS command for these simulations are:</p><p id="p-54">Tennessen European model (N<sub>0</sub> = 496,500):</p><ul class="list-simple " id="list-2"><li id="list-item-2"><p id="p-55">macs 2000 20000000 -T -t 0.023832 -r 0.01986 -G 0.0195 -eG 0.000102718 0.00307 -eN 0.000463746 0.00374823 -eN 0.00102719 0.02915204 -eN 0.002980864 0.01472305</p></li></ul><p id="p-56">Tennessen African model (N<sub>0</sub> = 424,000):</p><ul class="list-simple " id="list-3"><li id="list-item-3"><p id="p-57">macs 2000 20000000 -T -t 0.020352 -r 0.01696 -eG 0.0 0.016475022 -eN 0.000120873 0.034136873 -eN 0.003490568 0.017240607</p></li></ul><p id="p-58">Tennessen European alternate model (N<sub>0</sub> = 9300):</p><ul class="list-simple " id="list-4"><li id="list-item-4"><p id="p-59">macs 2000 20000000 -T -t 0.0004464 -r 0.000372 -eN 0.0 1.0 -eG 0.005510753 0.003074847 -eN 0.024758065 0.20010753 -eN 0.05483871 1.55634409 -eN 0.159139785 0.78602151</p></li></ul><p id="p-60">Constant size population model (N<sub>0</sub> = 100,000):</p><ul class="list-simple " id="list-5"><li id="list-item-5"><p id="p-61">macs 2000 20000000 -T -t 0.0048 -r 0.004 -eN 0.0 1.0</p></li></ul></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-63">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027581/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-64"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-01-22"><span class="received-label">Received </span>January 22, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-02-29"><span class="accepted-label">Accepted </span>February 29, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Chiang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.1" data-doi="10.1002/gepi.20378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sand Korneliussen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moltke</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Overseem Hansen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Relatedness mapping and tracts of relatedness for genome-wide data in the presence of linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">266</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DAlbrechtsen%26rft.auinit1%253DA.%26rft.volume%253D33%26rft.issue%253D3%26rft.spage%253D266%26rft.epage%253D274%26rft.atitle%253DRelatedness%2Bmapping%2Band%2Btracts%2Bof%2Brelatedness%2Bfor%2Bgenome-wide%2Bdata%2Bin%2Bthe%2Bpresence%2Bof%2Blinkage%2Bdisequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20378%26rft_id%253Dinfo%253Apmid%252F19025785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20378&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19025785&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264689500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.2" data-doi="10.1534/genetics.110.113977"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moltke</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Natural selection and the distribution of identity-by-descent in the human genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.113977%26rft_id%253Dinfo%253Apmid%252F20592267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzEvMjk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.3" data-doi="10.1016/j.ajhg.2011.01.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A fast, powerful method for detecting identity by descent.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DB.%2BL.%26rft.volume%253D88%26rft.issue%253D2%26rft.spage%253D173%26rft.epage%253D182%26rft.atitle%253DA%2Bfast%252C%2Bpowerful%2Bmethod%2Bfor%2Bdetecting%2Bidentity%2Bby%2Bdescent.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2011.01.010%26rft_id%253Dinfo%253Apmid%252F21310274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2011.01.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21310274&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.4" data-doi="10.1016/j.ajhg.2013.09.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Detecting identity by descent and estimating genotype error rates in sequence data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">840</span>–<span class="cit-lpage">851</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D93%26rft.spage%253D840%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2013.09.014%26rft_id%253Dinfo%253Apmid%252F24207118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2013.09.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24207118&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.5" data-doi="10.1534/genetics.113.150029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Improving the accuracy and efficiency of identity-by-descent detection in population data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150029%26rft_id%253Dinfo%253Apmid%252F23535385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvNDU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.6" data-doi="10.1016/j.ajhg.2010.02.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">High-resolution detection of identity by descent in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">526</span>–<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D86%26rft.issue%253D4%26rft.spage%253D526%26rft.epage%253D539%26rft.atitle%253DHigh-resolution%2Bdetection%2Bof%2Bidentity%2Bby%2Bdescent%2Bin%2Bunrelated%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2010.02.021%26rft_id%253Dinfo%253Apmid%252F20303063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2010.02.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20303063&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276716800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.5.1287.7" data-doi="10.1038/nrg3054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Haplotype phasing: existing methods and new developments.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D703%26rft.epage%253D714%26rft.atitle%253DHaplotype%2Bphasing%253A%2Bexisting%2Bmethods%2Band%2Bnew%2Bdevelopments.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3054%26rft_id%253Dinfo%253Apmid%252F21921926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921926&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.8" data-doi="10.1038/ng.2418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chong</span>  <span class="cit-name-given-names">J. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malig</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumont</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimating the human mutation rate using autozygosity in a founder population.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1277</span>–<span class="cit-lpage">1281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCampbell%26rft.auinit1%253DC.%2BD.%26rft.volume%253D44%26rft.issue%253D11%26rft.spage%253D1277%26rft.epage%253D1281%26rft.atitle%253DEstimating%2Bthe%2Bhuman%2Bmutation%2Brate%2Busing%2Bautozygosity%2Bin%2Ba%2Bfounder%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2418%26rft_id%253Dinfo%253Apmid%252F23001126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23001126&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.9" data-doi="10.1016/S0040-5809(03)00071-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A model for the length of tracts of identity by descent in finite random mating populations.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DChapman%26rft.auinit1%253DN.%2BH.%26rft.volume%253D64%26rft.issue%253D2%26rft.spage%253D141%26rft.epage%253D150%26rft.atitle%253DA%2Bmodel%2Bfor%2Bthe%2Blength%2Bof%2Btracts%2Bof%2Bidentity%2Bby%2Bdescent%2Bin%2Bfinite%2Brandom%2Bmating%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0040-5809%252803%252900071-6%26rft_id%253Dinfo%253Apmid%252F12948676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0040-5809(03)00071-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12948676&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.10" data-doi="10.1101/gr.083634.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marjoram</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and flexible simulation of DNA sequence data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.083634.108%26rft_id%253Dinfo%253Apmid%252F19029539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMTM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.11" data-doi="10.1093/molbev/msu151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLean</span>  <span class="cit-name-given-names">C. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Reducing pervasive false-positive identical-by-descent segments detected by large-scale pedigree analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2212</span>–<span class="cit-lpage">2222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu151%26rft_id%253Dinfo%253Apmid%252F24784137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMxLzgvMjIxMiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.12" data-doi="10.1089/cmb.1996.3.479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marjoram</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Ancestral inference from samples of DNA sequences with recombination.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DGriffiths%26rft.auinit1%253DR.%2BC.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D479%26rft.epage%253D502%26rft.atitle%253DAncestral%2Binference%2Bfrom%2Bsamples%2Bof%2BDNA%2Bsequences%2Bwith%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.1996.3.479%26rft_id%253Dinfo%253Apmid%252F9018600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.1996.3.479&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9018600&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996WE51500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.13" data-doi="10.1101/gr.081398.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Whole population, genome-wide mapping of hidden relatedness.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">318</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.081398.108%26rft_id%253Dinfo%253Apmid%252F18971310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzIvMzE4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.14" data-doi="10.1016/j.ajhg.2011.04.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salit</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxena</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">DASH: a method for identical-by-descent haplotype mapping uncovers association with recent variation.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">706</span>–<span class="cit-lpage">717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DGusev%26rft.auinit1%253DA.%26rft.volume%253D88%26rft.issue%253D6%26rft.spage%253D706%26rft.epage%253D717%26rft.atitle%253DDASH%253A%2Ba%2Bmethod%2Bfor%2Bidentical-by-descent%2Bhaplotype%2Bmapping%2Buncovers%2Bassociation%2Bwith%2Brecent%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2011.04.023%26rft_id%253Dinfo%253Apmid%252F21620352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2011.04.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21620352&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.15" data-doi="10.1534/genetics.111.134874"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramachandran</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Low-pass genome-wide sequencing and variant inference using identity-by-descent in an isolated human population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">689</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.134874%26rft_id%253Dinfo%253Apmid%252F22135348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNjc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identity by descent estimation with dense genome-wide genotype data.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DHan%26rft.auinit1%253DL.%26rft.volume%253D35%26rft.issue%253D6%26rft.spage%253D557%26rft.epage%253D567%26rft.atitle%253DIdentity%2Bby%2Bdescent%2Bestimation%2Bwith%2Bdense%2Bgenome-wide%2Bgenotype%2Bdata.%26rft_id%253Dinfo%253Apmid%252F21769932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21769932&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.17" data-doi="10.1038/ejhg.2012.148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Using identity by descent estimation with dense genotype data to detect positive selection.</span> <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D21%26rft.spage%253D205%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2012.148%26rft_id%253Dinfo%253Apmid%252F22781100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2012.148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22781100&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.18" data-doi="10.1371/journal.pgen.1003521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Inferring demographic history from a spectrum of shared haplotype lengths.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003521%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003521%26rft_id%253Dinfo%253Apmid%252F23754952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23754952&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1287.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Gene genealogies and the coalescent process</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">44</span> in <span class="cit-source">Oxford Surveys in Evolutionary Biology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Futuyma</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Antonovics</span>  <span class="cit-name-given-names">J.</span></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.20" data-doi="10.1038/ng.216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frigge</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gylfason</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zusmanovich</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Detection of sharing by descent, long-range phasing and haplotype imputation.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKong%26rft.auinit1%253DA.%26rft.volume%253D40%26rft.issue%253D9%26rft.spage%253D1068%26rft.epage%253D1075%26rft.atitle%253DDetection%2Bof%2Bsharing%2Bby%2Bdescent%252C%2Blong-range%2Bphasing%2Band%2Bhaplotype%2Bimputation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.216%26rft_id%253Dinfo%253Apmid%252F19165921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19165921&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258761200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.21" data-doi="10.1126/science.1217876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegmann</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehm</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Jean</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An abundance of rare functional variants in 202 drug target genes sequenced in 14,002 people.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1217876%26rft_id%253Dinfo%253Apmid%252F22604722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzcvNjA5MC8xMDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTI4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.22" data-doi="10.1016/j.ajhg.2012.08.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palamara</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lencz</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pe’er</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Length distributions of identity by descent reveal fine-scale demographic history.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">809</span>–<span class="cit-lpage">822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DFrancesco%2BPalamara%26rft.auinit1%253DP.%26rft.volume%253D91%26rft.issue%253D5%26rft.spage%253D809%26rft.epage%253D822%26rft.atitle%253DLength%2Bdistributions%2Bof%2Bidentity%2Bby%2Bdescent%2Breveal%2Bfine-scale%2Bdemographic%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2012.08.030%26rft_id%253Dinfo%253Apmid%252F23103233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2012.08.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23103233&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.23" data-doi="10.1016/j.ajhg.2015.10.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palamara</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francioli</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Genovese</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilton</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Leveraging distant relatedness to quantify human mutation and gene conversion rates.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">775</span>–<span class="cit-lpage">789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D97%26rft.spage%253D775%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.10.006%26rft_id%253Dinfo%253Apmid%252F26581902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.10.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26581902&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.24" data-doi="10.1038/nrg2865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Reconciling the analysis of IBD and IBS in complex trait studies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">800</span>–<span class="cit-lpage">805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPowell%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D800%26rft.epage%253D805%26rft.atitle%253DReconciling%2Bthe%2Banalysis%2Bof%2BIBD%2Band%2BIBS%2Bin%2Bcomplex%2Btrait%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2865%26rft_id%253Dinfo%253Apmid%252F20877324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2865&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20877324&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.25" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.26" data-doi="10.1371/journal.pbio.1001555"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ralph</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The geography of recent genetic ancestry across Europe.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1001555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D11%26rft.spage%253De1001555%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001555%26rft_id%253Dinfo%253Apmid%252F23667324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001555&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23667324&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1287.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.27" data-doi="10.1126/science.1219240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tennessen</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigham</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Connor</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution and functional impact of rare coding variation from deep sequencing of human exomes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">64</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1219240%26rft_id%253Dinfo%253Apmid%252F22604720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzcvNjA5MC82NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.28" data-doi="10.1534/genetics.112.148825"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identity by descent: variation in meiosis, across genomes, and in populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DThompson%26rft.auinit1%253DE.%2BA.%26rft.volume%253D194%26rft.issue%253D2%26rft.spage%253D301%26rft.epage%253D326%26rft.atitle%253DIdentity%2Bby%2BDescent%253A%2BVariation%2Bin%2BMeiosis%252C%2BAcross%2BGenomes%252C%2Band%2Bin%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.148825%26rft_id%253Dinfo%253Apmid%252F23733848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvMzAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1287.29" data-doi="10.1073/pnas.1119675109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuk</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hechter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunyaev</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The mystery of missing heritability: Genetic interactions create phantom heritability.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1193</span>–<span class="cit-lpage">1198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1119675109%26rft_id%253Dinfo%253Apmid%252F22223662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA5LzQvMTE5MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjg3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/5/1287.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/5/1277" title="Chicken gga-miR-103-3p Targets CCNE1 and TFDP2 and Inhibits MDCC-MSB1 Cell Migration" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/5/1297" title="Mapping Challenging Mutations by Whole-Genome Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/5"><span>Volume 6 Issue 5, May 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/5.cover-source.jpg?itok=8SH0hKOw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1287" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Kwhax9iT6bk-z3zK6jpbBxPHPH14qcR1S0FLlEWBczM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/5/1287" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/5/1287" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/5/1287" class="active" data-icon-position="" data-hide-link-title="0">Conflation of Short Identity-by-Descent Segments Bias Their Inferred Length Distribution</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460355" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Yu2ChPukC18Hb5NgXG-1AJ92n3ZR1xx_aoZCq2d1jmQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/5/1287.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/5/1287" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-yNQosF23ZK0pPngZ0S_r_IzTPq9vSe-fUmEnSbLKfl8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/5/1287" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/5/1287.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460355" id="node460355--2" data-pisa="ggg;6/5/1287" data-pisa-master="ggg;g3.116.027581" data-apath="/ggg/6/5/1287.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Conflation of Short Identity-by-Descent Segments Bias Their Inferred Length Distribution</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0668-7865" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charleston W. K. Chiang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Ralph</span> and <span class="highwire-citation-author" data-delta="2">John Novembre</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1287-1296;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027581 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460355--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charleston W. K. Chiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charleston%2BW.%2BK.%2BChiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chiang%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharleston%2BW.%2BK.%2BChiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0668-7865" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charleston W. K. Chiang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chiang82@ucla.edu" class="" data-icon-position="" data-hide-link-title="0">chiang82@ucla.edu</a>
<a href="mailto:jnovembre@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">jnovembre@uchicago.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter Ralph </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Computational Biology, University of Southern California, Los Angeles, California 90089</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BRalph%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ralph%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BRalph%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Ralph</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John Novembre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Human Genetics, University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BNovembre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Novembre%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BNovembre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chiang82@ucla.edu" class="" data-icon-position="" data-hide-link-title="0">chiang82@ucla.edu</a>
<a href="mailto:jnovembre@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">jnovembre@uchicago.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/5/1287" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/5/1287&amp;title=Conflation%20of%20Short%20Identity-by-Descent%20Segments%20Bias%20Their%20Inferred%20Length%20Distribution" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/5/1287&amp;title=Conflation%20of%20Short%20Identity-by-Descent%20Segments%20Bias%20Their%20Inferred%20Length%20Distribution" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/5/1287&amp;title=Conflation%20of%20Short%20Identity-by-Descent%20Segments%20Bias%20Their%20Inferred%20Length%20Distribution" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/5/1287&amp;text=Conflation%20of%20Short%20Identity-by-Descent%20Segments%20Bias%20Their%20Inferred%20Length%20Distribution" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/5/1287&amp;title=Conflation%20of%20Short%20Identity-by-Descent%20Segments%20Bias%20Their%20Inferred%20Length%20Distribution" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/5/1287&amp;t=Conflation%20of%20Short%20Identity-by-Descent%20Segments%20Bias%20Their%20Inferred%20Length%20Distribution" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/5/1287&amp;title=Conflation%20of%20Short%20Identity-by-Descent%20Segments%20Bias%20Their%20Inferred%20Length%20Distribution" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/5/1287&amp;title=Conflation%20of%20Short%20Identity-by-Descent%20Segments%20Bias%20Their%20Inferred%20Length%20Distribution" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460355/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460355" id="node460355--3" data-pisa="ggg;6/5/1287" data-pisa-master="ggg;g3.116.027581" data-apath="/ggg/6/5/1287.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Conflation of Short Identity-by-Descent Segments Bias Their Inferred Length Distribution</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0668-7865" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Charleston W. K. Chiang</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter Ralph</span> and <span class="highwire-citation-author" data-delta="2">John Novembre</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1287-1296;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027581 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460355--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charleston W. K. Chiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of California, Los Angeles, California 90095</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charleston%2BW.%2BK.%2BChiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chiang%20CW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACharleston%2BW.%2BK.%2BChiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0668-7865" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Charleston W. K. Chiang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chiang82@ucla.edu" class="" data-icon-position="" data-hide-link-title="0">chiang82@ucla.edu</a>
<a href="mailto:jnovembre@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">jnovembre@uchicago.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter Ralph </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Computational Biology, University of Southern California, Los Angeles, California 90089</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BRalph%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ralph%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BRalph%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9459-6866" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter Ralph</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John Novembre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Human Genetics, University of Chicago, Illinois 60637</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BNovembre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Novembre%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BNovembre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:chiang82@ucla.edu" class="" data-icon-position="" data-hide-link-title="0">chiang82@ucla.edu</a>
<a href="mailto:jnovembre@uchicago.edu" class="" data-icon-position="" data-hide-link-title="0">jnovembre@uchicago.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1287&amp;rft.epage=1296&amp;rft.atitle=Conflation%20of%20Short%20Identity-by-Descent%20Segments%20Bias%20Their%20Inferred%20Length%20Distribution&amp;rft.volume=6&amp;rft.issue=5&amp;rft.date=2016-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chiang%2C+Charleston+W.+K.&amp;rft.au=Ralph%2C+Peter&amp;rft.au=Novembre%2C+John"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460355/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460355/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460355/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460355/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460355/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460355/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460355/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460355/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460355/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460355/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460355/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460355/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/5/1287" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/5/1287.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/5/1287.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/5/1287.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/5/1287.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
