<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/5/1153" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/5/1153.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/5/1153.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/5/1153.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/5/1153/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/5/1153.atom" />
<meta name="HW.pisa" content="ggg;6/5/1153" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Does 3D Phenotyping Yield Substantial Insights in the Genetics of the Mouse Mandible Shape?" />
<meta name="DC.Identifier" content="10.1534/g3.115.024372" />
<meta name="DC.Date" content="2016-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Navarro, Maga. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We describe the application of high-resolution 3D microcomputed tomography, together with 3D landmarks and geometric morphometrics, to validate and further improve previous quantitative genetic studies that reported QTL responsible for variation in the mandible shape of laboratory mice using a new backcross between C57BL/6J and A/J inbred strains. Despite the increasing availability of 3D imaging techniques, artificial flattening of the mandible by 2D imaging techniques seems at first an acceptable compromise for large-scale phenotyping protocols, thanks to an abundance of low-cost digital imaging systems such as microscopes or digital cameras. We evaluated the gain of information from considering explicitly this additional third dimension, and also from capturing variation on the bone surface where no precise anatomical landmark can be marked. Multivariate QTL mapping conducted with different landmark configurations (2D vs. 3D; manual vs. semilandmarks) broadly agreed with the findings of previous studies. Significantly more QTL (23) were identified and more precisely mapped when the mandible shape was captured with a large set of semilandmarks coupled with manual landmarks. It appears that finer phenotypic characterization of the mandibular shape with 3D landmarks, along with higher density genotyping, yields better insights into the genetic architecture of mandibular development. Most of the main variation is, nonetheless, preferentially embedded in the natural 2D plane of the hemi-mandible, reinforcing the results of earlier influential investigations." />
<meta name="DC.Contributor" content="Nicolas Navarro" />
<meta name="DC.Contributor" content="A. Murat Maga" />
<meta name="article:published_time" content="2016-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Does 3D Phenotyping Yield Substantial Insights in the Genetics of the Mouse Mandible Shape?" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We describe the application of high-resolution 3D microcomputed tomography, together with 3D landmarks and geometric morphometrics, to validate and further improve previous quantitative genetic studies that reported QTL responsible for variation in the mandible shape of laboratory mice using a new backcross between C57BL/6J and A/J inbred strains. Despite the increasing availability of 3D imaging techniques, artificial flattening of the mandible by 2D imaging techniques seems at first an acceptable compromise for large-scale phenotyping protocols, thanks to an abundance of low-cost digital imaging systems such as microscopes or digital cameras. We evaluated the gain of information from considering explicitly this additional third dimension, and also from capturing variation on the bone surface where no precise anatomical landmark can be marked. Multivariate QTL mapping conducted with different landmark configurations (2D &lt;i&gt;vs.&lt;/i&gt; 3D; manual &lt;i&gt;vs.&lt;/i&gt; semilandmarks) broadly agreed with the findings of previous studies. Significantly more QTL (23) were identified and more precisely mapped when the mandible shape was captured with a large set of semilandmarks coupled with manual landmarks. It appears that finer phenotypic characterization of the mandibular shape with 3D landmarks, along with higher density genotyping, yields better insights into the genetic architecture of mandibular development. Most of the main variation is, nonetheless, preferentially embedded in the natural 2D plane of the hemi-mandible, reinforcing the results of earlier influential investigations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/05/01" />
<meta name="citation_mjid" content="ggg;6/5/1153" />
<meta name="citation_id" content="6/5/1153" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/5/1153" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/5/1153.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/5/1153.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/5/1153.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.024372" />
<meta name="citation_pmid" content="26921296" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1153" />
<meta name="citation_lastpage" content="1163" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nicolas Navarro" />
<meta name="citation_author_institution" content="Biogéosciences, UMR CNRS 6282, Univ Bourgogne Franche-Comté, EPHE, PSL Research University, F-21000 Dijon, France" />
<meta name="citation_author_email" content="nicolas.navarro@ephe.sorbonne.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5694-4201" />
<meta name="citation_author" content="A. Murat Maga" />
<meta name="citation_author_institution" content="Division of Craniofacial Medicine, Department of Pediatrics, University of Washington, Seattle, Washington 98105" />
<meta name="citation_author_institution" content="Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7921-9018" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=D. C. Adams;citation_title=A method for assessing phylogenetic least squares models for shape and other high-dimensional multivariate data.;citation_pages=2675-2688;citation_volume=68;citation_year=2014;citation_pmid=24899536;citation_doi=10.1111/evo.12463" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. D. Aguirre;citation_author=E. Hine;citation_author=K. McGuigan;citation_author=M. W. Blows;citation_title=Comparing G: multivariate analysis of genetic variation in multiple populations.;citation_pages=21-29;citation_volume=112;citation_year=2013;citation_pmid=23486079" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=P. S. Anderson;citation_author=S. Renaud;citation_author=E. J. Rayfield;citation_title=Adaptive plasticity in the mouse mandible.;citation_pages=85;citation_volume=14;citation_year=2014;citation_pmid=24742055;citation_doi=10.1186/1471-2148-14-85" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. Andolfatto;citation_author=D. Davison;citation_author=D. Erezyilmaz;citation_author=T. T. Hu;citation_author=J. Mast;citation_title=Multiplexed shotgun genotyping for rapid and efficient genetic mapping.;citation_volume=21;citation_year=2011;citation_pmid=21233398;citation_doi=10.1101/gr.115402.110" />
<meta name="citation_reference" content="citation_author=D. Aneja;citation_author=S. R. Vora;citation_author=E. D. Camci;citation_author=L. G. Shapiro;citation_author=T. C. Cox;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Biological Reviews (Cambridge; Print);citation_journal_abbrev=Biological Reviews (Cambridge; Print);citation_author=W. R. Atchley;citation_author=B. K. Hall;citation_title=A model for development and evolution of complex morphological structures.;citation_pages=101-157;citation_volume=66;citation_year=1991;citation_issue=2;citation_pmid=1863686;citation_doi=10.1111/j.1469-185X.1991.tb01138.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=W. R. Atchley;citation_author=A. A. Plummer;citation_author=B. Riska;citation_title=GENETICS OF MANDIBLE FORM IN THE MOUSE;citation_pages=555-577;citation_volume=111;citation_year=1985;citation_issue=3;citation_pmid=4054609" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Evol. Dev.;citation_author=L. Boell;citation_title=Lines of least resistance and genetic architecture of house mouse (Mus musculus) mandible shape.;citation_pages=197-204;citation_volume=15;citation_year=2013;citation_pmid=23607303;citation_doi=10.1111/ede.12033" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=L. Boell;citation_author=D. Tautz;citation_title=Micro-evolutionary divergence patterns of mandible shapes in wild house mouse (Mus musculus) populations.;citation_pages=306-306;citation_volume=11;citation_year=2011;citation_pmid=22008647;citation_doi=10.1186/1471-2148-11-306" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=L. Boell;citation_author=S. Gregorova;citation_author=J. Forejt;citation_author=D. Tautz;citation_title=A comparative assessment of mandible shape in a consomic strain panel of the house mouse (Mus musculus)--implications for epistasis and evolvability of quantitative traits.;citation_pages=309-309;citation_volume=11;citation_year=2011;citation_pmid=22011306;citation_doi=10.1186/1471-2148-11-309" />
<meta name="citation_reference" content="citation_journal_title=Dev. Genes Evol.;citation_author=L. Boell;citation_author=L. F. Pallares;citation_author=C. Brodski;citation_author=Y. Chen;citation_author=J. L. Christian;citation_title=Exploring the effects of gene dosage on mandible shape in mice as a model for studying the genetic basis of natural variation.;citation_pages=279-287;citation_volume=223;citation_year=2013;citation_pmid=23563729;citation_doi=10.1007/s00427-013-0443-y" />
<meta name="citation_reference" content="citation_journal_title=Medical image analysis;citation_journal_abbrev=Med Image Anal;citation_author=F. L. Bookstein;citation_title=Landmark methods for forms without landmarks: morphometrics of group differences in outline shape.;citation_pages=225-243;citation_volume=1;citation_year=1997;citation_issue=3;citation_pmid=9873908;citation_doi=10.1016/S1361-8415(97)85012-8" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.;citation_author=K. W. Broman;citation_author=T. P. Speed;citation_title=A model selection approach for the identification of quantitative trait loci in experimental crosses.;citation_pages=641-656;citation_volume=64;citation_year=2002;citation_doi=10.1111/1467-9868.00354" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Front. Zool.;citation_author=P. A. Bromiley;citation_author=A. C. Schunke;citation_author=H. Ragheb;citation_author=N. A. Thacker;citation_author=D. Tautz;citation_title=Semi-automatic landmark point annotation for geometric morphometrics.;citation_pages=61;citation_volume=11;citation_year=2014;citation_doi=10.1186/s12983-014-0061-1" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Cande;citation_author=P. Andolfatto;citation_author=B. Prud’homme;citation_author=D. L. Stern;citation_author=N. Gompel;citation_title=Evolution of multiple additive loci caused divergence between Drosophila yakuba and D. santomea in wing rowing during male courtship.;citation_pages=e43888-e43888;citation_volume=7;citation_year=2012;citation_issue=8;citation_pmid=22952802;citation_doi=10.1371/journal.pone.0043888" />
<meta name="citation_reference" content="citation_journal_title=Hystrix;citation_author=A. Cardini;citation_title=Missing the third dimension in geometric morphometrics: how to assess if 2D images really are a good proxy for 3D structures?;citation_pages=73-81;citation_volume=25;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=F. Chevalier;citation_author=C. L. Valentim;citation_author=P. T. LoVerde;citation_author=T. J. Anderson;citation_title=Efficient linkage mapping using exome capture and extreme QTL in schistosome parasites.;citation_pages=617;citation_volume=15;citation_year=2014;citation_pmid=25048426;citation_doi=10.1186/1471-2164-15-617" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. A. Churchill;citation_author=R. W. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=M. L. Collyer;citation_author=D. J. Sekora;citation_author=D. C. Adams;citation_title=A method for analysis of phenotypic change for phenotypes described by high-dimensional data.;citation_pages=357-365;citation_volume=115;citation_year=2015;citation_pmid=25204302;citation_doi=10.1038/hdy.2014.75" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Cox;citation_author=C. L. Ackert-Bicknell;citation_author=B. L. Dumont;citation_author=Y. Ding;citation_author=J. T. Bell;citation_title=A new standard genetic map for the laboratory mouse.;citation_volume=182;citation_year=2009;citation_pmid=19535546;citation_doi=10.1534/genetics.109.105486" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=V. Debat;citation_author=A. Debelle;citation_author=I. Dworkin;citation_title=Plasticity, canalization, and developmental stability of the Drosophila wing: joint effects of mutations and developmental temperature.;citation_pages=2864-2876;citation_volume=63;citation_year=2009;citation_issue=11;citation_pmid=19624729;citation_doi=10.1111/j.1558-5646.2009.00774.x" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=A. G. Drake;citation_author=C. P. Klingenberg;citation_title=The pace of morphological change: historical transformation of skull shape in St Bernard dogs;citation_pages=71-76;citation_volume=275;citation_year=2008;citation_issue=1630;citation_pmid=17956847;citation_doi=10.1098/rspb.2007.1169" />
<meta name="citation_reference" content="citation_journal_title=Ann. Appl. Stat.;citation_author=I. L. Dryden;citation_author=A. Koloydenko;citation_author=D. Zhou;citation_title=Non-Euclidean statistics for covariance matrices, with applications to diffusion tensor imaging.;citation_pages=1102-1123;citation_volume=3;citation_year=2009;citation_doi=10.1214/09-AOAS249" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Dupuis;citation_author=D. Siegmund;citation_title=Statistical Methods for Mapping Quantitative Trait Loci From a Dense Set of Markers;citation_pages=373-386;citation_volume=151;citation_year=1999;citation_issue=1;citation_pmid=9872974" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. Dworkin;citation_author=J. A. Anderson;citation_author=Y. Idaghdour;citation_author=E. K. Parker;citation_author=E. A. Stone;citation_title=The effects of weak genetic perturbations on the transcriptome of the wing imaginal disc and its association with wing shape in Drosophila melanogaster.;citation_volume=187;citation_year=2011;citation_pmid=21288875;citation_doi=10.1534/genetics.110.125922" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=M. Emília Santos;citation_author=C. S. Berger;citation_author=P. N. Refki;citation_author=A. Khila;citation_title=Integrating evo-devo with ecology for a better understanding of phenotypic evolution.;citation_pages=1-12;citation_volume=2015;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Magnetic resonance imaging;citation_journal_abbrev=Magn Reson Imaging;citation_author=A. Fedorov;citation_author=R. Beichel;citation_author=J. Kalpathy-Cramer;citation_author=J. Finet;citation_author=J.-C. Fillion-Robin;citation_title=3D Slicer as an image computing platform for the Quantitative Imaging Network.;citation_pages=1323-1341;citation_volume=30;citation_year=2012;citation_issue=9;citation_pmid=22770690;citation_doi=10.1016/j.mri.2012.05.001" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Flint;citation_author=E. Eskin;citation_title=Genome-wide association studies in mice.;citation_pages=807-817;citation_volume=13;citation_year=2012;citation_issue=11;citation_pmid=23044826;citation_doi=10.1038/nrg3335" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=P. Franchini;citation_author=C. Fruciano;citation_author=M. L. Spreitzer;citation_author=J. C. Jones;citation_author=K. R. Elmer;citation_title=Genomic architecture of ecologically divergent body shape in a pair of sympatric crater lake cichlid fishes.;citation_pages=1828-1845;citation_volume=23;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=T. E. Galesloot;citation_author=K. van Steen;citation_author=L. A. L. M. Kiemeney;citation_author=L. L. Janss;citation_author=S. H. Vermeulen;citation_title=A Comparison of multivariate genome-wide association methods.;citation_pages=e95923-e95928;citation_volume=9;citation_year=2014;citation_pmid=24763738;citation_doi=10.1371/journal.pone.0095923" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=H. Gao;citation_author=T. Zhang;citation_author=Y. Wu;citation_author=Y. Wu;citation_author=L. Jiang;citation_title=Multiple-trait genome-wide association study based on principal component analysis for residual covariance matrix.;citation_pages=526-532;citation_volume=113;citation_year=2014;citation_pmid=24984606;citation_doi=10.1038/hdy.2014.57" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. Gnirke;citation_author=A. Melnikov;citation_author=J. Maguire;citation_author=P. Rogov;citation_author=E. M. LeProust;citation_title=Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing.;citation_pages=182-189;citation_volume=27;citation_year=2009;citation_issue=2;citation_pmid=19182786;citation_doi=10.1038/nbt.1523" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.;citation_author=C. Goodall;citation_title=Procrustes methods in the statistical analysis of shape.;citation_pages=285-339;citation_volume=53;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. M. Gray;citation_author=M. D. Parmenter;citation_author=C. A. Hogan;citation_author=I. Ford;citation_author=R. J. Cuthbert;citation_title=Genetics of rapid and extreme size evolution in island mice.;citation_volume=201;citation_year=2015;citation_pmid=26199233;citation_doi=10.1534/genetics.115.177790" />
<meta name="citation_reference" content="citation_journal_title=Hystrix;citation_author=P. Gunz;citation_author=P. Mitteroecker;citation_title=Semilandmarks: a method for quantifying curves and surfaces.;citation_pages=103-109;citation_volume=24;citation_year=2013" />
<meta name="citation_reference" content="citation_title=Modern Morphometrics in Physical Anthropology;citation_title=Semilandmarks in three dimensions;citation_author=P. Gunz;citation_author=P. Mitteroecker;citation_author=F. L. Bookstein;citation_pages=73-98;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=J. Guo;citation_author=X. Mei;citation_author=K. Tang;citation_title=Automatic landmark annotation and dense correspondence registration for 3D human facial images.;citation_pages=232;citation_volume=14;citation_year=2013;citation_pmid=23870191;citation_doi=10.1186/1471-2105-14-232" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=B. Hallgrímsson;citation_author=W. Mio;citation_author=R. S. Marcucio;citation_author=R. Spritz;citation_title=Let’s face it–complex traits are just not that simple.;citation_pages=e1004724;citation_volume=10;citation_year=2014;citation_pmid=25375250;citation_doi=10.1371/journal.pgen.1004724" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=M. J. R. Healy;citation_title=Rao’s paradox concerning multivariate tests of significance.;citation_pages=411-413;citation_volume=25;citation_year=1969;citation_doi=10.2307/2528798" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. Hodges;citation_author=Z. Xuan;citation_author=V. Balija;citation_author=M. Kramer;citation_author=M. N. Molla;citation_title=Genome-wide in situ exon capture for selective resequencing.;citation_pages=1522-1527;citation_volume=39;citation_year=2007;citation_issue=12;citation_pmid=17982454;citation_doi=10.1038/ng.2007.42" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. Houle;citation_author=D. R. Govindaraju;citation_author=S. Omholt;citation_title=Phenomics: the next challenge.;citation_pages=855-866;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21085204;citation_doi=10.1038/nrg2897" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Huang;citation_author=Q. Feng;citation_author=Q. Qian;citation_author=Q. Zhao;citation_author=L. Wang;citation_title=High-throughput genotyping by whole-genome resequencing.;citation_volume=19;citation_year=2009;citation_pmid=19420380;citation_doi=10.1101/gr.089516.108" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=B. Huber;citation_author=A. Whibley;citation_author=Y. L. Poul;citation_author=N. Navarro;citation_author=A. Martin;citation_title=Conservatism and novelty in the genetic architecture of adaptation in Heliconius butterflies.;citation_pages=515-524;citation_volume=114;citation_year=2015;citation_pmid=25806542;citation_doi=10.1038/hdy.2015.22" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. R. Jones;citation_author=J. M. Good;citation_title=Targeted capture in evolutionary and ecological genomics.;citation_pages=185-202;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13304" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Khila;citation_author=E. Abouheif;citation_author=L. Rowe;citation_title=Evolution of a novel appendage ground plan in water striders is driven by changes in the Hox gene Ultrabithorax.;citation_pages=e1000583-e1000583;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19649305;citation_doi=10.1371/journal.pgen.1000583" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C. P. Klingenberg;citation_title=Evolution and development of shape: integrating quantitative approaches.;citation_pages=623-635;citation_volume=11;citation_year=2010;citation_issue=9;citation_pmid=20697423;citation_doi=10.1038/nrg2829" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=C. P. Klingenberg;citation_author=N. A. Gidaszewski;citation_title=Testing and quantifying phylogenetic signals and homoplasy in morphometric data.;citation_volume=59;citation_year=2010;citation_pmid=20525633;citation_doi=10.1093/sysbio/syp106" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=C. P. Klingenberg;citation_author=L. J. Leamy;citation_title=Quantitative genetics of geometric shape in the mouse mandible.;citation_pages=2342-2352;citation_volume=55;citation_year=2001;citation_issue=11;citation_pmid=11794792;citation_doi=10.1554/0014-3820(2001)055[2342:QGOGSI]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. P. Klingenberg;citation_author=L. J. Leamy;citation_author=E. J. Routman;citation_author=J. M. Cheverud;citation_title=Genetic Architecture of Mandible Shape in Mice: Effects of Quantitative Trait Loci Analyzed by Geometric Morphometrics;citation_pages=785-802;citation_volume=157;citation_year=2001;citation_issue=2;citation_pmid=11156997" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. P. Klingenberg;citation_author=L. J. Leamy;citation_author=J. M. Cheverud;citation_title=Integration and Modularity of Quantitative Trait Locus Effects on Geometric Shape in the Mouse Mandible;citation_pages=1909-1921;citation_volume=166;citation_year=2004;citation_issue=4;citation_pmid=15126408;citation_doi=10.1534/genetics.166.4.1909" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=W. J. Krzanowski;citation_title=Between-Groups Comparison of Principal Components.;citation_pages=703-707;citation_volume=74;citation_year=1979;citation_doi=10.2307/2286995" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I.-Y. Kwak;citation_author=C. R. Moore;citation_author=E. P. Spalding;citation_author=K. W. Broman;citation_title=A simple regression-based method to map quantitative trait loci underlying function-valued phenotypes.;citation_volume=197;citation_year=2014;citation_pmid=24931408;citation_doi=10.1534/genetics.114.166306" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. B. Langlade;citation_author=X. Feng;citation_author=T. Dransfield;citation_author=L. Copsey;citation_author=A. I. Hanna;citation_title=Evolution through genetically controlled allometry space.;citation_volume=102;citation_year=2005;citation_pmid=16009935;citation_doi=10.1073/pnas.0504210102" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=L. J. Leamy;citation_author=C. P. Klingenberg;citation_author=E. Sherratt;citation_author=J. B. Wolf;citation_author=J. M. Cheverud;citation_title=A search for quantitative trait loci exhibiting imprinting effects on mouse mandible size and shape.;citation_pages=518-526;citation_volume=101;citation_year=2008;citation_issue=6;citation_pmid=18685568;citation_doi=10.1038/hdy.2008.79" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=M. C. Ledur;citation_author=N. Navarro;citation_author=M. Pérez-Enciso;citation_title=Large-scale SNP genotyping in crosses between outbred lines: how useful is it?;citation_pages=173-182;citation_volume=105;citation_year=2009;citation_pmid=19844266" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. R. Linnen;citation_author=Y.-P. Poh;citation_author=B. K. Peterson;citation_author=R. D. H. Barrett;citation_author=J. G. Larson;citation_title=Adaptive Evolution of Multiple Traits Through Multiple Mutations at a Single Gene;citation_pages=1312-1316;citation_volume=339;citation_year=2013;citation_issue=6125;citation_pmid=23493712;citation_doi=10.1126/science.1233213" />
<meta name="citation_reference" content="citation_journal_title=IEEE transactions on medical imaging;citation_journal_abbrev=IEEE Trans Med Imaging;citation_author=J. Liu;citation_author=W. Gao;citation_author=S. Huang;citation_author=W. L. Nowinski;citation_title=A model-based, semi-global segmentation approach for automatic 3-D point landmark localization in neuroimages.;citation_pages=1034-1044;citation_volume=27;citation_year=2008;citation_issue=8;citation_pmid=18672421;citation_doi=10.1109/TMI.2008.915684" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. Liu;citation_author=T. Shikano;citation_author=T. Leinonen;citation_author=J. M. Cano;citation_author=M.-H. Li;citation_title=Identification of major and minor QTL for ecologically important morphological traits in three-spined sticklebacks (Gasterosteus aculeatus).;citation_volume=4;citation_year=2014;citation_pmid=24531726;citation_doi=10.1534/g3.114.010389" />
<meta name="citation_reference" content="citation_journal_title=Front. Physiol.;citation_author=A. M. Maga;citation_author=N. Navarro;citation_author=M. L. Cunningham;citation_author=T. C. Cox;citation_title=Quantitative trait loci affecting the 3D skull shape and size in mouse and prioritization of candidate genes in-silico.;citation_pages=1-13;citation_volume=6;citation_year=2015;citation_pmid=25688210;citation_doi=10.3389/fphys.2015.00001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Manichaikul;citation_author=J. Dupuis;citation_author=S. Sen;citation_author=K. W. Broman;citation_title=Poor performance of bootstrap confidence intervals for the location of a quantitative trait locus.;citation_volume=174;citation_year=2006;citation_pmid=16783000;citation_doi=10.1534/genetics.106.061549" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Manichaikul;citation_author=J. Y. Moon;citation_author=S. Sen;citation_author=B. S. Yandell;citation_author=K. W. Broman;citation_title=A model selection approach for the identification of quantitative trait loci in experimental crosses, allowing epistasis.;citation_volume=181;citation_year=2009;citation_pmid=19104078;citation_doi=10.1534/genetics.108.094565" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=B. McCane;citation_title=Shape variation in outline shapes.;citation_volume=62;citation_year=2013;citation_pmid=22993142;citation_doi=10.1093/sysbio/sys080" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. G. Mezey;citation_author=D. Houle;citation_author=S. V. Nuzhdin;citation_title=Naturally segregating quantitative trait loci affecting wing shape of Drosophila melanogaster.;citation_volume=169;citation_year=2005;citation_pmid=15520257;citation_doi=10.1534/genetics.104.036988" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. R. Miller;citation_author=J. P. Dunham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_author=E. A. Johnson;citation_title=Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.;citation_volume=17;citation_year=2007;citation_pmid=17189378;citation_doi=10.1101/gr.5681207" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=L. R. Monteiro;citation_title=Multivariate regression models and geometric morphometrics: the search for causal factors in the analysis of shape.;citation_volume=48;citation_year=1999;citation_pmid=12078640;citation_doi=10.1080/106351599260526" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genomics Hum. Genet.;citation_author=R. Mott;citation_author=J. Flint;citation_title=Dissecting quantitative traits in mice.;citation_pages=421-439;citation_volume=14;citation_year=2013;citation_pmid=23834320;citation_doi=10.1146/annurev-genom-091212-153419" />
<meta name="citation_reference" content="citation_journal_title=Ann. Zool. Fenn.;citation_author=F. Muñoz-Muñoz;citation_author=D. Perpiñan;citation_title=Measurement error in morphometric studies: comparison between manual and computerized methods.;citation_pages=46-56;citation_volume=47;citation_year=2010;citation_doi=10.5735/086.047.0105" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=C. L. Olson;citation_title=Comparative Robustness of Six Tests in Multivariate Analysis of Variance.;citation_pages=894-908;citation_volume=69;citation_year=1974;citation_doi=10.2307/2286159" />
<meta name="citation_reference" content="citation_journal_title=Psychol. Bull.;citation_author=C. L. Olson;citation_title=On choosing a test statistic in multivariate analysis of variance.;citation_pages=579-586;citation_volume=83;citation_year=1976;citation_doi=10.1037//0033-2909.83.4.579" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=M. Olson;citation_title=Enrichment of super-sized resequencing targets from the human genome.;citation_pages=891-892;citation_volume=4;citation_year=2007;citation_issue=11;citation_pmid=17971778;citation_doi=10.1038/nmeth1107-891" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=L. F. Pallares;citation_author=B. Harr;citation_author=L. M. Turner;citation_author=D. Tautz;citation_title=Use of a natural hybrid zone for genomewide association mapping of craniofacial traits in the house mouse.;citation_pages=5756-5770;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12968" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=L. F. Pallares;citation_author=P. Carbonetto;citation_author=S. Gopalakrishnan;citation_author=C. C. Parker;citation_author=C. L. Ackert-Bicknell;citation_title=Mapping of craniofacial traits in outbred mice identifies major developmental genes involved in shape determination.;citation_pages=e1005607;citation_volume=11;citation_year=2015;citation_pmid=26523602;citation_doi=10.1371/journal.pgen.1005607" />
<meta name="citation_reference" content="citation_journal_title=Pattern Recognit.;citation_author=P. Perakis;citation_author=T. Theoharis;citation_author=I. A. Kakadiaris;citation_title=Feature fusion for facial landmark detection.;citation_pages=2783-2793;citation_volume=47;citation_year=2014;citation_doi=10.1016/j.patcog.2014.03.007" />
<meta name="citation_reference" content="citation_journal_title=J. Anat.;citation_author=C. J. Percival;citation_author=D. K. Liberton;citation_author=F. Pardo-Manuel de Villena;citation_author=R. Spritz;citation_author=R. Marcucio;citation_title=Genetics of murine craniofacial morphology: diallel analysis of the eight founders of the Collaborative Cross.;citation_pages=96-112;citation_volume=228;citation_year=2016;citation_pmid=26426826;citation_doi=10.1111/joa.12382" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. K. Peterson;citation_author=J. N. Weber;citation_author=E. H. Kay;citation_author=H. S. Fisher;citation_author=H. E. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. Prud’homme;citation_author=C. Minervino;citation_author=M. Hocine;citation_author=J. D. Cande;citation_author=A. Aouane;citation_title=Body plan innovation in treehoppers through the evolution of an extra wing-like appendage.;citation_pages=83-86;citation_volume=473;citation_year=2012" />
<meta name="citation_reference" content="citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=Conf Proc IEEE Eng Med Biol Soc;citation_author=S. M. Rolfe;citation_author=L. G. Shapiro;citation_author=T. C. Cox;citation_author=A. M. Maga;citation_author=L. L. Cox;citation_title=A landmark-free framework for the detection and description of shape differences in embryos.;citation_pages=5153-5156;citation_volume=2011;citation_year=2011;citation_pmid=22255499" />
<meta name="citation_reference" content="citation_author=S. Schlager;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=A Statistical Framework for Quantitative Trait Mapping;citation_pages=371-387;citation_volume=159;citation_year=2001;citation_issue=1;citation_pmid=11560912" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. Shifman;citation_author=J. T. Bell;citation_author=R. R. Copley;citation_author=M. S. Taylor;citation_author=R. W. Williams;citation_title=A high-resolution single nucleotide polymorphism genetic map of the mouse genome.;citation_pages=e395-e395;citation_volume=4;citation_year=2006;citation_issue=12;citation_pmid=17105354;citation_doi=10.1371/journal.pbio.0040395" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Stephens;citation_title=A unified framework for association analysis with multiple related phenotypes.;citation_pages=e65245;citation_volume=8;citation_year=2013;citation_pmid=23861737;citation_doi=10.1371/journal.pone.0065245" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=J.-I. Suto;citation_title=Identification of multiple quantitative trait loci affecting the size and shape of the mandible in mice.;citation_pages=1-13;citation_volume=20;citation_year=2009;citation_issue=1;citation_pmid=19067046;citation_doi=10.1007/s00335-008-9154-5" />
<meta name="citation_reference" content="citation_journal_title=J. Anat.;citation_author=D. L. Swiderski;citation_author=M. L. Zelditch;citation_title=The complex ontogenetic trajectory of mandibular shape in a laboratory mouse.;citation_pages=568-580;citation_volume=223;citation_year=2013;citation_pmid=24111948;citation_doi=10.1111/joa.12118" />
<meta name="citation_reference" content="citation_title=Using Multivariate Statistics;citation_author=B. G. Tabachnick;citation_author=L. S. Fidell;citation_year=2013" />
<meta name="citation_reference" content="citation_title=Quantitative Genetics in the Wild;citation_title=Evolutionary potential and constraints in wild populations;citation_author=C. Teplitsky;citation_author=M. R. Robinson;citation_author=J. Merilä;citation_pages=190-208;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Valdar;citation_author=L. C. Solberg;citation_author=D. Gauguier;citation_author=S. Burnett;citation_author=P. Klenerman;citation_title=Genome-wide genetic association of complex traits in heterogeneous stock mice.;citation_pages=879-887;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16832355;citation_doi=10.1038/ng1840" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. S. Workman;citation_author=L. J. Leamy;citation_author=E. J. Routman;citation_author=J. M. Cheverud;citation_title=Analysis of Quantitative Trait Locus Effects on the Size and Shape of Mandibular Molars in Mice;citation_pages=1573-1586;citation_volume=160;citation_year=2002;citation_issue=4;citation_pmid=11973311" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=B. Yalcin;citation_author=J. Nicod;citation_author=A. Bhomra;citation_author=S. Davidson;citation_author=J. Cleak;citation_title=Commercially available outbred mice for genome-wide association studies.;citation_pages=e1001085;citation_volume=6;citation_year=2010;citation_pmid=20838427;citation_doi=10.1371/journal.pgen.1001085" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=S. M. Yezerinac;citation_author=S. C. Lougheed;citation_author=P. Handford;citation_title=Measurement error and morphometric studies: statistical power and observer experience.;citation_volume=41;citation_year=1992;citation_doi=10.1093/sysbio/41.4.471" />
<meta name="citation_reference" content="citation_journal_title=Front. Zool.;citation_author=R. Young;citation_author=A. M. Maga;citation_title=Performance of single and multi-atlas based automated landmarking methods compared to expert annotations in volumetric microCT datasets of mouse mandibles.;citation_pages=33;citation_volume=12;citation_year=2015;citation_pmid=26628903;citation_doi=10.1186/s12983-015-0127-8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. B. Zeng;citation_author=J. Liu;citation_author=L. F. Stam;citation_author=C. H. Kao;citation_author=J. M. Mercer;citation_title=Genetic Architecture of a Morphological Shape Difference Between Two Drosophila Species;citation_pages=299-310;citation_volume=154;citation_year=2000;citation_issue=1;citation_pmid=10628989" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Does 3D Phenotyping Yield Substantial Insights in the Genetics of the Mouse Mandible Shape?" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
<meta name="twitter:description" content="We describe the application of high-resolution 3D microcomputed tomography, together with 3D landmarks and geometric morphometrics, to validate and further improve previous quantitative genetic studies that reported QTL responsible for variation in the mandible shape of laboratory mice using a new backcross between C57BL/6J and A/J inbred strains. Despite the increasing availability of 3D imaging techniques, artificial flattening of the mandible by 2D imaging techniques seems at first an acceptable compromise for large-scale phenotyping protocols, thanks to an abundance of low-cost digital imaging systems such as microscopes or digital cameras. We evaluated the gain of information from considering explicitly this additional third dimension, and also from capturing variation on the bone surface where no precise anatomical landmark can be marked. Multivariate QTL mapping conducted with different landmark configurations (2D vs. 3D; manual vs. semilandmarks) broadly agreed with the findings of previous studies. Significantly more QTL (23) were identified and more precisely mapped when the mandible shape was captured with a large set of semilandmarks coupled with manual landmarks. It appears that finer phenotypic characterization of the mandibular shape with 3D landmarks, along with higher density genotyping, yields better insights into the genetic architecture of mandibular development. Most of the main variation is, nonetheless, preferentially embedded in the natural 2D plane of the hemi-mandible, reinforcing the results of earlier influential investigations." />
<meta name="og-title" property="og:title" content="Does 3D Phenotyping Yield Substantial Insights in the Genetics of the Mouse Mandible Shape?" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/5/1153" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We describe the application of high-resolution 3D microcomputed tomography, together with 3D landmarks and geometric morphometrics, to validate and further improve previous quantitative genetic studies that reported QTL responsible for variation in the mandible shape of laboratory mice using a new backcross between C57BL/6J and A/J inbred strains. Despite the increasing availability of 3D imaging techniques, artificial flattening of the mandible by 2D imaging techniques seems at first an acceptable compromise for large-scale phenotyping protocols, thanks to an abundance of low-cost digital imaging systems such as microscopes or digital cameras. We evaluated the gain of information from considering explicitly this additional third dimension, and also from capturing variation on the bone surface where no precise anatomical landmark can be marked. Multivariate QTL mapping conducted with different landmark configurations (2D vs. 3D; manual vs. semilandmarks) broadly agreed with the findings of previous studies. Significantly more QTL (23) were identified and more precisely mapped when the mandible shape was captured with a large set of semilandmarks coupled with manual landmarks. It appears that finer phenotypic characterization of the mandibular shape with 3D landmarks, along with higher density genotyping, yields better insights into the genetic architecture of mandibular development. Most of the main variation is, nonetheless, preferentially embedded in the natural 2D plane of the hemi-mandible, reinforcing the results of earlier influential investigations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
    <title>Does 3D Phenotyping Yield Substantial Insights in the Genetics of the Mouse Mandible Shape? | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"GP0wRtK4_O5W8BDlLXNDDwzu8dZBXnUsgWE3Y3rnp3o","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460361","apath":"\/ggg\/6\/5\/1153.atom","pisa":"ggg;6\/5\/1153","ac":{"\/ggg\/6\/5\/1153.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/5\/1153.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/5\/1153"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/5\/1153"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/5\/1153?cited-by=yes\u0026legid=ggg;6\/5\/1153":true,"\/content\/6\/5\/1153":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wfrCDR55dYRVWh9pstlQilU+R+\\\\\\\/w\\\\\\\/c2vKGvpSNMxLPAg\\\\\\\/zbSWPKiODbJ+qRp7aa20bVv\\\\\\\/neELng8XqIYbx660FF68jAtRcPAaaHcVsVNvdmaOyj8rt2xgbXQGtk9RvYIuh5wnvNCWAhit6suM3pYtMgYuDlhrrwXWO95Yaud0hMQfhqX41UlyRfvEKCBNVbQ7ba\\\\\\\/GZSrlovEYu6\\\\\\\/dje1BP4z2\\\\\\\/K\\\\\\\/U4cf1t0s0NeMg1orHo+Ja+d04hDgACLZ1sRM+pFpz951flHccmCaDpbkFnvCpSiyOpLTgHq+Xbx4V39QpvV9YmVmipsOzRFn1d0lDc1iTiNkaQgV9sEg84i5b6uP\\\\\\\/E+t+\\\\\\\/UV8upPc4KUQ9+vQICf3bZEF3K7eE31MPNl0g7V2f6srb7USrMwfChLcKZZ9pjyUu9pDN7JzLC4a44qaqrA2M6xuOxEsGPFhi4WvOjCpFWSJx+YFmBSiX+Jh8r0xND6jRrrQ\\\\\\\/2LdY8YTbMUkXxzTN+gUS08FZOQ9Sb3HLz9i0TeilFA2j9LLAIEZcqejvIs+3nS19+bzdJi+TlcsGC0rW40FESg4qo9ZooQop1460zxoIh98mKQ7JdCN5TrNQLGdRxKwhLAdVzp6U2X5XF6v2E+8clMBHViHdcPsN8n5Pn\\\\\\\/vnhu0906yY2N2WqUjAGsqeUsj9krSyRNH93upUxHIcWwOb+zIJPm1m6KgtHxOL\\\\\\\/TqMNuDM2lB4pj4iAXmeA2X4MP0sotnJud3Ofy\\\\\\\/fquRsDLCIdR14Vwf3rk3ZjuG7AGaow90xY7Iwf\\\\\\\/aoQgNuT+l7lYfj3Lp1SvqZw\\\\\\\/xrnzIlz04CHRGKuoeVpJ7YjkBRYYVAFYLkHCVpoUvttSjD3Mp4ZTwJLhQFsYqfGnLKgQVjPo0Zko1uSXiFbEingjsrNKSJs0jYlqQM\\\\\\\/TMseCLB2FZ74DuJg3o7sUrOG27nvuQjKwHtJr3qvQqgGCFu7REWear8KUsyOLHT6mCKIiXmpKf+g9OcBQ2Bh4dGOqwr0YV8wYLxM1t4mGutEbaT4hmev3gyJiHPUXtfzEi9ImQdetVsj1KC0b6hUPmfsp3AraPHXB6IsySVjLWAlTreFBuC8ztYWOoaTqkbn0Jik18vD0O\\\\\\\/MdWvvJOvPA6W1\\\\\\\/C4wXcq0HVGzW7HUKtmI7UqMOG8WvsfwljZYKiL1H2EshxJapqv2gQ2PKnDHr3dNnM53yG4cg7xKbl1l0gdGsxwnqMDbEKc+0w8m6CtbCmjGHEPhgXGbTzluc2K1lIV73Vt0YMRoeck82Gnvr3zZ7\\\u0022,\\\u0022iv\\\u0022:\\\u0022IXJOqA2F\\\\\\\/dKkSE7yrrmh7Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222f4be01dc27d42687c1727c63c2a9cf210f564c43b3009e8d0e49678637c3abe\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rBrmckTFHrKY6hvPLQUDA0D69T6yfoLeaOV0s\\\\\\\/my\\\\\\\/pJJzSuIrY\\\\\\\/+fk9OekmQnj1+E5g2cw4jBK45sBbmJ7YUl9Dag5q71yxmMcAKbLdeLZktg81dk\\\\\\\/Ik2+fM\\\\\\\/G6oYoooXhlcQupHVVJmET1Mi1Zhp2lbHISWfqMRqSmc2sUND+C0SHij42aA1n5N8OxEzM0lflafY5Auq+nCGWYjgP86\\\\\\\/h0hc8d3XgvdyelacZUnTuytHa9J1cRtgnHTZMoSoP\\\\\\\/DcGPO4cfZA35M\\\\\\\/uLo9c0fgQFyQUs1LQ\\\\\\\/h0TjoO+mj41zMRw91b7gydKI3iJyRv8M\\\\\\\/fOs7aTRsTxPdd5gDU3G50kyGIZIsDVnMrhM5MyHZQukV9EYuqmnm+CG4D5nZeXrPnEgW5RaMzbjDNNSGPGddB10SObtapusn6+YpXyDeJvGEafK0X\\\\\\\/+mRCZl9suwlub\\\\\\\/4lN663WuYES8bM6g0bZS\\\\\\\/zRP88pG3AbUFmRaPOP1kUsNjvcrvsoJrpxafBXztguyYzIeY6e0ZzUhdqcy6NBloXzTx6AIeFiuwPoOPYmqKVg7eOKA17a42pMlh3vgJororOdI7IX6vsGYaXLDKFH4cYyv8B60wqLIH73DebwESKqsvG7pm7cRQHI1lakuF5y5NWMgLh2BhdTAD7j7bqMO35f4PbkcwhHKgw+3Uy2WBwCN8IoEU8j8bgsukuem7d8ouYNL\\\\\\\/\\\\\\\/2ueHB4HkEKveE1fCmx357J81gYYUgU9n5qbVgReBEny9UtCnjBv3FpKtujPDY7Wi+5Sw0QgihMAPrSPxgh7KDH1XIlWAw4gIQWet8F\\\\\\\/fH7LhgczY\\\\\\\/wvDSWK8Rug2ecSPIPmtkMO0rpMT1VkHIsbDkZgxmdPkVu5EhKFJgfPMNTUQFGLMdyyQiwSqhphJrE+R442sdBj15scJqjSbFM4pMoHaVM1OUD8wNwELkmcVm74\\\\\\\/ABfZOwVlO66\\\\\\\/mlDcyux1aAgOtaIeXPjA31fJFI2ESCbuDkNg\\\\\\\/ZReW9iqu2MQbs94J7DR5X7AHeMt+8wHE0LnY7Y5KZMKBpTzMRuwnHaRKFLvQAlLX6CMnIIcbfCyn2hRZerjpYK7gsFyMQCmUNmGgIxVZ4cem8com35IKqlgwTg2KBoNYbSdrShDiG2CkRwdikfbJBmmoJBSq4k8KrLc9\\\\\\\/w1wrP0ZPIYxZupCa4+jc2q9MdbHHd2VbtqwvlusC9h9HazxMyWbMs\\\\\\\/g3Hg\\\\\\\/VVwXqr8HTlHiw25P5Gy5AQY7eYwFSxe9uI3hNQZKaowcaRXPlxJ979ddW9yLhy4RRb\\\\\\\/BmbNbxnLPs5MHGo\\\\\\\/xCpuWurP8XNTM3uFJ3BH5hp3GdBQgeI2mygO9hR64gUuE0kywe20UY8\\\\\\\/dfWStA2kg6kScAuOmPcmfQiBg6Qma4\\\\\\\/p+Bfv0cOcB9b3jUkItpPp4FT4vlhF7ozBv+MdKN+QDcW+yaG6dw+w4A6jUSzRHsQtl9FW\\\\\\\/FUUgNUGfs7KbWsTFDXngf\\\\\\\/rKX05l78XunJ4iy20aPFvLqGO1C3l0N2zJdEqh6wvsXwpYWcgYt56H2f+UwgxBcfYktz8B7WzPN6gX\\\\\\\/RX+An3fLboXZj1VZJrVRCGhQktdMdNNQEpUt\\\\\\\/Taz\\\\\\\/+NtU3Xq9ROGhEjM2+Mp8zB4FfpHxg2yAPd7d1wmCyKyp9jw7egcXWIN\\\\\\\/pIO0k7DqDJGdxJYPeqtOJIjoUUAIoTZ61VODADrJK+XVIcBNa4SPesr5w\\\\\\\/7sp0flzMBfZoMUGur5xbkWAhr8DQNpkqut6uQTfYLyDPAEsdb1RKtmlUTN\\\\\\\/uzf\\\\\\\/WYyqf2rriJv+YWOkR0ejPoPTdVGUOudAAjdrZmPWBUE+7GDJtD0nQnWxJiKcsdKBfkHns6jZBDy4XxUGwf+xAUD9SSsuKY9XO9oY+OO9xUeNdppsKjf\\\\\\\/MX3msxqMT86DNA3dg1l7HVRNMNgNVXSXV7B1y0XESOutv9yryM0NZtx0YZj7g9t248ldUUUk9sAfKIw6Cytj6g4kJxAzsceMrSnRkLvP9g3dq3lbY2nFXOXUSva3eogPpcIDD3goIKoZ2JPnwi6cbsqBfl1z1Z+5LW1sbiliAoHMLMmgm8fy83ngGW\\\\\\\/6iTDBQYPanw\\\\\\\/cBmsaM92wQK\\\\\\\/Jpp6EIzlihFobXtFadpxQXG0LPEX+41v4gfi036SYViSMNEQ6Uh94KATMs\\\\\\\/gsz5DEFTOnifQ7iX+GtyFbLujqfoLJezZnaI5Q5tywZZ3Vum4JfDiy1WHURgAYPTvNviNTE8jGUohWyooDcEnGR0I9FAEZKjOUyJUm8tZQFLbBXtXgDaydtetoyi3orxJAmyjxRR9c8u\\\\\\\/LWPqDTr\\\\\\\/g1mBZb3Ohfzwi6OLcVFcgRA7QIWP7PDm7alNFbaKWJXog4MG6nv9IKnF\\\\\\\/8Qt3VruSeFBn6AWyu96JblcrZNjzw8ZdHyilcQwqCIfA1ugst33mveXmMbesfzgEXOsPa5ZSoQ3cs4jiVQFoavKr7zDukg3msH3HUGZpfpBxyLh5vYbgdWceleKR530iLvfO8FAFY5yqomi1Au\\\\\\\/QA\\\\\\\/4HkQDa+9SvEQrKuqVveaO8Lvg4xJhPXT8iylqVp2uS8ipxl4wvgtU62fHC4Aa4rNSdrDUJsEdcy3kgyuHBY9T\\\\\\\/6gHbGRJz2Pn2hL3jamU2zxEQPBhsdHWmlGxadezFZNpxoz\\\\\\\/NU00pZ7Zy8PvYYD1F2QgtGHgKr09iLMvN3H2bqCjc37QmhIdPTyYwX7zslSUNk2cIsoSfZ6oa4OMo\\\\\\\/E4QAAX\\\\\\\/Dz1w3KywdWg7hqJOfl+V90a6MPNCxzsS368YlVkGF4YyO6B2B6p5kRduPagx9YTTJ61E+OnGoQWReiq091BDn1+yYCg3dtCveQlHEtEW3IJNqlfvt7evhs1DAP9M0YBRYZinyDRgznLfFmOG9dlnGJ9Nh1\\\\\\\/FJ\\\\\\\/KaIN1rVeZdpOswa0ZBDMdMDgzcCISDQeKbLY\\\\\\\/lIsv8pt+UzQcNfDECjT\\\\\\\/\\\\\\\/i\\\\\\\/zZvrL6t5YhYcUzHMwuuFmCRb+pRME2XfkPEiaseuPThg4Giilhbte8WZ+Zi9Xc4\\\\\\\/WJBCIvUMf3dDm3bBvbc6f0AUwDQxLWhmJ3cmZh0wZ39cXcu6kUYShPnb31kOAYEw\\\\\\\/X+LC6kn87XFHOPnk+K5OqA5Wx\\\\\\\/ZWXL7rE\\\\\\\/plwUOQVh6dGHJQE4Ma\\\\\\\/cukUlk0eOmHd6jHIkLF1nhjz5HIxG89hdzUOZZezAEZx6rZDO3h9CYnfc6eUMuNfKDOM0oFvNmT8gK36RC8duD4oJ8eCWL5CpQ+FQSy0IMH8SGdfCynvdLpI0SEPruMxpIE54yXdGPr+6slr4wmuMR38u46HqkgW\\\\\\\/Watj0wbc1uHKKCASsQbLOvrqgvRAC6i61D5ipOLUGhNZdgthEZMWvdqZ1re9bIP6MINHOxgeHvYhRn5\\\\\\\/x6vKz1pXlq0nzY9MFH\\\\\\\/OKTq28p9S3epvMCqkFRqrLrvOhnyiH2UE0gr9wcF4WV+2QfJRlHxMf7c3LKDNl+HMhHvm5bgLp0Y\\\\\\\/5mMqHs63\\\\\\\/KC639ylWG0BQJv\\\\\\\/ZbAb45kizS+V2pwXgQfmLQPw2D2dVIPYNqBVAnhGPpD3zvoYOLuQdPTFWkb8Q6W2EO\\\\\\\/7jqVI\\\\\\\/wu\\\\\\\/BpHGeyMRQOJqm+IrKocJaCS0G2YHT1J\\\\\\\/aNi9aY5E\\\\\\\/XpwfWja4Dtx1Dly1oEEFNRnwHOm4OY1DtXQnhtmJpvPjV+Jz6Zb8R4k251AcSegjELTEBnGn2a\\\\\\\/EZvPEZsowlVO5lQ2PRTJ8Cw5wzTzE7mEfsbMjhWNJdyGjf0M8O+v7D+H+7p77XTLL2C7PRwj3uPRfxY7xABi9wRA3YZrgkahdyymxe9TYZAyuCAY4338Ow87J\\\\\\\/lRTFhpUL1S3EZhzrOqwilF6izZM\\\\\\\/GNGs0g8OFeNGQe8Zc9fBTo1pfyNh6BavnyyInOy5n2ZBYaukOe\\\\\\\/Yh+Mxioj5RCcVknLHd4Gv\\\\\\\/VK0JLUI14TSFHl5W+eeT\\\\\\\/L6hywWnfLgFPhZYKA6u39LF5IAZTzWBy0Tg+mumHSo0xd0\\\\\\\/5OEApz9V6tGHifxv9wJrokCizg7\\\\\\\/vwQnwwhxz1i4X8gytIaQVRrxedbQBrTg\\\\\\\/AVwHqvD0vv3gf1cK\\\\\\\/JqrUEh8SETQctcLjRHHizJHygh3axiAPcP\\\\\\\/DJDd7QRPNAv5ZvE78ZVC+4TXFJwhvJLlhioVRMFbAeDqvNt2zvHttqhUThBhNOoM6Jr6cQ65xcYfzglpVfhqQdvmWzkyguF455Z1eaiK+O2JBHgs8tLETTjXcxjUEG7sjpjAYEalzJ6yalqPurKml+IrseGAGRBSt+j3zJTmMNMiLyuKGoQWLuPOx2dMoqzuGJII0TFfwMkgFaZjIO6jG7C5VrWMnxzU8onNzZkQixYAIU0l9Ub+HY9A85oj12DcydMpT9ZFjHS5PGCPHGjRfHK5uGSmYzo9XeHG4elY1o3MSDLuukdjYwWWhBK72vTDeY5u+z+0aNwymJPsMMFSO0s262oYmNO3aPpysbhDSfHuoyZAqLDAOjySDpIEpkZrNNu92oFmD1VF7vJ9FntRVpvUUA0CCf03aNlEXEgwPniV7m8ZY1GIqwENPQVnDvghek9ryk87BYoewVxxkXtSjWX4HTNLnEE8j6NsrFvhz4WLYJzxXZXnXi\\\\\\\/gxRpZBOecmvTBeUlUl+W4AIKLvOnrmb814RLg2VHQsYZgFdIIb\\\\\\\/ouGwXDh4WmVUlWQ8oU0V7YECG9jzyFX0UI9OY3+E3yFoEdvaMkUZuSlFFaF9LTE4sScWJ\\\\\\\/CXHuiSuOCLbl6YKorYtu5jn7p8FDthKDZxHJL1MaX2VtoJxTTOzThEPms4qTabnFPlhTzl3OzQNcRG\\\\\\\/6CyZFy1\\\\\\\/BYgsC9KGw1LwU06Dhl4cHYou0p5AFKKZny\\\\\\\/D+WaEfc+xfogQ0zhtf5OwTVXe8fTxXO0gjcW8zRFrSpCS6Mx0YRtz4VD7secVjPHshXXr5xnNQb11bDz4yrShrTttKDB2S1AFh1LZ1i\\\\\\\/ImcxLIytQ\\\\\\\/YrtAcxE1aGDDGJu7T9XhLuBk9OhCQVmjnUdyIp7VwsvTQaX5VKadZPd8MPHyTwdKnuD\\\\\\\/PRVx1+KB4\\\\\\\/EXpGWFR1CtCJQKWM\\\\\\\/Sr8STftxpRQVS5zgHH4Syi8lYlgJGd1PGXEmqp2dV7xrASmqWgU\\\\\\\/getceiDw21v8ECt1LqKjI5aw6ATPbcHDLRJEjMxUbI4E4A9JrI8y94m2nTUpelaLkrRRu0R7pPW0e9kJ8DUf5ZckmXAkrQl6UFDk0A8fb+LVlny10zTU6ZDxHZLORtHp1rgxZeucWSV32VtxQUx2l2s61iKC+IFoKnVFPrFPQhH\\\\\\\/udf+f5mh+yEDxm7OsJY9KuR7pQo9\\\\\\\/8Xxg5WQQjW4DtuIHo4N14CCPlE0P6C0UnD9otUV439nuafX7mGsycfVRyrh8iDbVkNxnVeO121Lcwn9+Z36XtNHG4MQC5qGCC\\\\\\\/G5TQ5Vg\\\\\\\/08e+ZriuI7Iq30of4KLH96e3ZvXXYDYY6EQTVt+gyRz56TSjYO8hayWcekw+1ozex1w\\\\\\\/y2HdfYinbvdcDYHWYegwWXXZ9AgtlpQ9gRjR8NQjn18r+edy2kk87W0e0Lv8FhekhQpn3xq8Ealrm0m9EjE4MZCLia9OMDbrmqmGU42jmUbBPZN57rOzRPuFPOGRZU5\\\\\\\/ydj8GPcJilImPBFzOj2IVG7ryAAumLzQY8o6l1tBvTXZQo+LANc4kCkwGnLIPbMz4ulkezxqVOISrd9rG8FuH83kChosDPoicutJPUE5tqO+oOBoW9kTdHQEfOGU4iUQDJlchaeC+NYp4Q08ODh5LgUBbcfE901YBNZ5U2dZDywa70C7r3139ve9412dhBctYZaq8Bh6+jzJrZ5WDx7tp0nv5vzJywwcoXi+4zEQ6KZ+wKjq+6g31YGX7N1PylO7Sro3f4aE66lV9N10pE\\\\\\\/yWwKyk4m8YZ1OQ7QzlQliGfJT8y0g9eEKCC54OBDv1nZaCwEKxN+AfUeGQx+RclL90oXxNEB1nLXyfZdM9+YYU+J2D0dhj4S5SQY0V5kcPf6I1neyZYE1pr0DPehaKhoFWkTDQGLk9zgLxHHP94ffcvLCg9kRrTyw7qRi4XyVbPfTvbr7P+AkGE\\\\\\\/+9mQou5gHy4zZeHcmpKwaAjgzaeCR0R4es8Yw9lcSiGtsDwO1l0\\\\\\\/pQXVX6UDJQ3xbP6aD2rZ7wImZtBYmFYzSp21+5rGDsltIXl8YBTquhADc1JmMQCue7EPOlbmWXBFmz2t8B9PbJjjgQOcTKa16b1XqQUccDcCIRmNZRTpfBIaLs+v\\\\\\\/TdwEtqoggZysTv6nHIWFnFm0v8Dzcy+LZbbnVkQwyTYQeeIJUDpNG13Hy1CpfrPJvDToAopcvMYmvBuztW54TEV8oxEjyqZqg9p86A7+\\\\\\\/uB5MGkHj2oE7ZvSfnQ0peW+WEAlwaDoWIZpNvOfJhG6XlVJsmz2Hj2sPA83Dqid6CqndtiOyDutl3v6RZ6VQXFD1gg8wvpSSnvXEpAbekAvY9jrobqVByPBpvzU\\\\\\\/ZkvuS5aavzci4Z3QEltzUqVQMG5iwUqSkhBkKIGoZ8tkn2TXfa6uvTJXJfi45U4CND1en2ecTISCLXHv8Vv++9OUe+Xu99jfe5aRMJu4WFWGltd0XQpPOVPJvNEf4CG1zorb5yoN\\\\\\\/+Pa6Tbrj5JF2FeD\\\\\\\/dnOzhBqkd7OJnfM1afWBVd7bL\\\\\\\/Mpxu1jkIbnxrIohZ7yXJ5q2O2H1sEbksiK41dbzS3f2zwNNzD1fG92NpVQSvZHb65+mIpm3X53Dh8zTtc2lm56E9s1BxrWIFz8nts+0Bxe4h9Uvwc6sqq3\\\\\\\/yTQYdAsS68\\\\\\\/tquoRo\\\\\\\/g9+sK4mcx9s9+if+hvNM8TZnO6Zw9Sw4WQHqLXJORAnIsFEjZJ7uQwyGrkRIDIcPScnV\\\\\\\/Dn88ZcvK9KeB4EqFmp17C+Lnw6W5Sa4OQ0CbY5FrBiwH7R5B7grI8BM99HzbNlXkRIbHcJZ092luoadfLDczJMR8soANnI9\\\\\\\/BKLCb0\\\\\\\/Y9ktGyk4Wv+sdfxQOAmibX6vPebft4hqosvzhOms3i5tahtqZal3udgzaVPT6f+cy28fFVDLTMEDELyG4wwrWnAasgR2n7MtnVc0+CwOgn9LTHtzqNHP4EkMPIYb2ondZ9bNWAIRWLVbaCX3TwikfvWnQ0\\\\\\\/H0FkIG7iZCeqUF0nQbJ1ZJpYOaGq6iQVrs+0yipXk7L2YqUpjtTiKseL+XaIyhZLJUkQIVJ\\\\\\\/bwcRuJ8FZatxPrfh2PsDUEJWJWtV8Z8E2AtD2xiC43giQmcZpIZEhV\\\\\\\/nrWy5LK9LibQbfSyJF0650JXTWhzX4lOzO2QGIvtcDyk6XL6RA9B+fxmDlM8yJ4D0vBc6zbdrZJUgvkMphV1yc2xkc6AsjDmCAONMUx9683u+3C4ZUB+0BRqSjnjRUbepTgqiv7\\\\\\\/yG\\\\\\\/zGNSSJqEVHMHYxon76j2j\\\\\\\/4W+S2dAQMuiDG3ZYvsm8YBbxo3hLqnWhzUSXttxL\\\\\\\/v5HGNDRUMH1deXugnuU3gKp0aYfRJBUcmja+i7zYTrcBD26ljgYgxM66wIXFJ6EINvStHePgXI4elPS894XzsDTxh3nZqQ1WBnNqx+tgeeSgrZEnp5fD3o\\\\\\\/OZ8TG9v4O+C7aJeDAJVu07LIQ6AjUz\\\\\\\/OhrSyg8gzOrJDIpiieUmRSSmMOfQabGmPwCzdPhUdcK5kAOlHsgg06rl5Powv3xHw5NsFhTcPP0zh2LKn09tFRA98Wt2tsYEhactLIRXRF5G0zIgsYJbtEFzqaIY9z+5CTSQKjiQLT\\\\\\\/vbelX\\\\\\\/TkY0wCvcJ04iOWhGb0yXqzKpW9kZqL589UZ4wWz7f6vl7FmUL5ibrgvAYyZL2q4kd6B7ePqME0znZYdS9tcpazyhsATS75tH3DjCrIFPs3ouvGW6G9sihiKwsuiKhMc1uld1HuLt3dyDbadW5cUgM0dMz5beDQYqW3rC55OSmzfvSIYuEmxR1fu0xnWQPTbhxu9CmbnBe1xiB15CCqCCeha3ye8LKIZuhtVy27bUYb07FUuyquTXhWFUuMkhoPLltnD6oiYjeY7HT6AYj6Cx0kvb+eSvDw7rXGYUBX2IPThDBUaISt1xELRaegrqiJZ8ajB9mG1NPfhNyRgfftjxi0nGUhtnYynxwjKgxuuvtMeePbPr9NAP0rqOlCDnPLDc8iHqLHbfHKeA9mXVG72C6b6TwwqwhXuOfveCQkBNDqUpP829p6ryrgOpKu4NUayuo0PrhNq5Lh5HkF7ewzxFIfqQ7Bt9vN1oKfN8L9piZnryEOA6K7TL8WhfMeGFg9F6ugpJkaEurGgg0mBP1ILgdcqUbqod9kg8YOUQHtRXPyH7NE7bFp4JmrL8OlCV7tn2qleIcluwjFEncFJXSa48XKcxR1CwBEROD6SIyxh9WYCWT0C+I0mAG6QNvDrNFPdxJrjR9Whu72xz0mZn2E0w10A\\\\\\\/YtbsSZmCGWPbHRT7N\\\\\\\/XgSOWH5gOFClMkIoR4FovvWlsNyDlfrDLH8virxdJ\\\\\\\/59fJB9lKqPNn+lVzTeBztQPHfN3oEzcPQoMVQMnnyQiyTqm+xzSy4tBJ5C7z3pANDWjhinOSbTGYymbQoo\\\\\\\/vucg5Q8r0rehM8LWXvVQWOrzWXINhqT1gV4LKRRMId37yOxCDGNbhnlRevdkrk7umlPENrwEuzZv43\\\\\\\/AUv6l7VcZlY0ITA0sOB+gM4P\\\\\\\/jBWYFOCmmjCMZ7VHyHiLUF4QbEc0PD29F7X242q\\\\\\\/vsYNfO6cy5fFrVjS9W634lGkzhfUi5o0arX+2oZifcHgMBChz8L7g\\\\\\\/cVCWk2DFvc1SDGqVqST5bW97ACfuBY6JtbNipmX6JHn51uRZ1rMpaWSgc5Hb6Tjmlv4DJFvy5gFP6BQMTKion\\\\\\\/+nfNgsRD54BjvbXaXFRmRbOWpHevigvpbCyY0sSBI\\\\\\\/nGMhNqL6L+1KI6iEK9NpSrompX9wMSX97pRX6A+mnmP2VXuxhli3MG3cx40Mn83ORau5nk\\\\\\\/SygWsGVAIVh6SMPwHhcOLbdgSW2VVjUyoNLxPWlQXqu0bCAda\\\\\\\/XDu\\\\\\\/as\\\\\\\/nJZwPSbJrBAa\\\\\\\/n2pnuWY8OAI5i3\\\\\\\/HF8U\\\\\\\/O5LIJT5cU2bzI2n+SDnP2EorISxu2VWWw3MD5mbt4IwhI1OxB7+E5wk5cRtesoTkZqATWQt9iSHAMdNsTNP3Ml7nBVl24npe2i1L5PeqG+BiFEADg5QcYZeNTTj8Ro3CDlK1XP70IiOvcwz\\\\\\\/mBTBP6BAr40C0ts86zi2mdRROi8GUhEYTe\\\\\\\/szkABMK9oA6X8+ibAitRasLvlC+ZT8eYWW7Tctvc+MFe6o6VZXrVMm\\\\\\\/KW8g3EiuBD970pqaqN+HQjf62iZpqbwiu5GFDnztkdTz5Dm7p6jJxd6dKkVmFfR45wFUiVk2+sNZcoOefvg2m1q3FJXGAjiQBkqPGwGBivty6B++DkfD2k6XogXd7gaMu4Pbhjt8Jq\\\\\\\/9e2MGW8fgobVf3TbU8Oj0Q9LDp39k7faKJttwZ\\\\\\\/VBou92dD5twJBIBX1\\\\\\\/NBndHQbNBI0PmuroeZl9l0WuDfrX\\\\\\\/PUEtCaPGqGgEUNteKEdh9fgSx3bzaGm49XlEewq53ZIq3F43sHQ1+kMJdtMmwsRk2HZOumRKV8Pe2NqYVJMkdu6L0IBWuSFZrQLEx+PwPs3xtNOoXX9rQru47isoX\\\\\\\/ZAQ28AsZMDsuHp9xQjfY6jwmpdPE25M12h61W+IkzBCMGvxmiEg5m+Cfwr3OMcYDzK7iUOKW80M6vXtfsjIaDh4Tp\\\\\\\/3zxGBrqD8YHDlN5zfwgbsAGrQdkvoapkdK3IQXeSmYBJwFldWvnp\\\\\\\/oGEpEhdlwkepHJ6BW4VE7rW4X1WfykaUQzdMNTDE87sSr6Gtojg514vwvhs3qfX1HKO6Rtv1tAZjfbVz6jC5jmJ+ZzbJxmRYdVvuaGBNwo0Ne2vKQflD5eCzBRyAUBdIc5C1I6qr9duj46Jnhfj+fqpa5fBpJ9L76XThSQM9ApjiGc7i013bUW0A0TNHfY5+XSg9iGWx5GjkgOkZ90r+S0nCVbhMmmZXsLGbjPAd97FuBYgL14npOw\\\\\\\/8wvXEwqFXfl2OELMqCRwqaIQlV050vQLdkBsLxBo7iuUVhzj4ZmXw276drL5fLIzeqYZ2ZcKq4sFBqjBT2Tr+jw3stcTe0IFr6VzTtKbZHwCmQrk9JkusmFOfI70lov+ReraKHa1v3y+sPex8xoDoSkcCFVY2U5yyuNY35v2X9RaYi\\\\\\\/srGdQKQxrAFoRC3VA9GHq8HCXap+Y4OlKwjl4iXNT\\\\\\\/u4b7qsCXOhFo4YQLsZXARNA\\\\\\\/xb\\\\\\\/l7cZ7f6BDrZalJkEeTMnUQ9rbh+tF1SQGaZ+QP7xX5gEs7Z2jrdwSqHhMJZV6+GivPR\\\\\\\/tfQjSlHdKDVwuuRLBnJxWihMt2M9qJTviXAtpkVYsSOMxsEqEFA6KRX4boxzzRMUMZC2LbzaNeTyJEnBEhPanMcPLOjnnMozmByFTkTdLLXSgnzLjxIiVQrpMu0tOmfU+\\\\\\\/yx812U10Ae+MiYEGl0bZtpKz9GzqPO3yOshn432NahepHyyJrJAN2dDzpdbPXmNspAZG0oTMPv67hOuIBT3pTOxUehUUgQmbNZwzx3xIxbugnGlYEuCpgbb1KIAtvZ4lAkTRbe16qKaZxFS7FI0O4pZqb4vN8EmfR6iD67sUReY2aOMS5KW7IrXnRTE011rW0Hsppyv7I3vvwjzl3AkR1svWAuOv4smmh7BaMyAnvUNUtvBb6lCE02vEDV5ez+sGpTEme1hBeMaNYLByZ0iGxGY9Q4efNh2s6vzWDP\\\\\\\/lbBieH3KzDcJb38jbDP7EO6FGdBlknHmXsoHJalMfGmRdbuLIClFOn7X14YejpRZlgkloqbtRdU8XSV8lu7+imP41NKKg0ZVw36lQ1\\\\\\\/l9LEk2zneLeL+mFTuL5KrdAwRJVo2nCqWi3VVgoobo9rE7+7EhMoOQmht7T7A8m0akdqOcOFRjhw2YYrXGJOpwZxtGzEi2t1nHdv8mGsKrPa4pKXFfO5OKRknRKaMFeQhXY3E\\\\\\\/Z0vhUZXaMFMeeWvP8fPsEYoiN7ozZCFS+NSQZ0YrUIcBpDyb5yn+a+2YO+elSJWhHtj5Kjp5Qva13z1k4iDXMLvN11Zx1YxziJH9R8gPs0KR\\\\\\\/yPyZu1w6RqVUo4i7Ux0Nq0OPtdMSOhxY5GOe59NgiImPtby79td4H9Ds+jgZG\\\\\\\/UA\\\\\\\/WMBQ8DwWy6sBVjBMl7GCaUBdZq+3vdGkDZCY+ZIw6fcqjDwZ+i+rcFzMTRvfUFhktHDZDKg2puVo5seR7hcFV35BI+luFtBPxOk8P41Vrcpaz6r9P2Is5HuzwF3V1cbBmGYPtlwi3zj+eEwiHfWaISyc5ocU22y\\\\\\\/JzMqslQmgcpu3kPWO\\\\\\\/\\\\\\\/f8XW2DVuEENfLpocfo7gBT6P7sCXEaGStxVsR4wzgoNF3p8OymTUN3zJrKndPM3R1vpCb5f8RYjsxBIkJKcQ5FZFh2OnvDwjLZNvaBIxKSPUcSJh6vRNY5OZk\\\\\\\/PpM3ae9D+xtvXEubxb6wh6xc\\\\\\\/AHwGIDHg1dKpsB1n2PGVDetIe+C1zKW9L0kqKtyJUjnSIT4VwfQ1kmdQ4QoKF7gQ\\\\\\\/zEfcvjsMlRUDxj83Bb7ztMsq89WLJWwj3F3Ts5x++\\\\\\\/Au2SLs+k15O3R6DI3dJpqtwwKdWFg3CCqfSZFWOWdTSa5cmWIeSst4y8+yLVrHulzMZ6mDLGEzJWzGP0OiFUI38g6HM8uhPzk\\\\\\\/wmuUf6WryOrTJEbd+lwpfIB8cxIXI2nMxF2dR51cG4bD4MEwj4t1zx5EopFaeR9sHEFUU4vcS3Pb+xIXYucmpZbvkD8NkvAG2mNAy6ijtLrAo6TwEz23JqYgWiznKV96Ln\\\\\\\/q3bsjxdplmB638ekfTQ9TyHkl0VKczp6PuaRAWgHFz2dBbm25HBePbvG6FFkUTaE0FljxbSHtoYGNZ7FGfK4tOEvQ\\\\\\\/4MGoP0pL1OjNUiBJ4uLqt2yQ6Byt+pVBiJ+vf4FDhV9kfhBGZ9JYP+\\\\\\\/Skw+0BDt7Nz\\\\\\\/TnOFZ2A24g0obGD3uU62n7p2Z6W8gAZSipLu0ZxIFUQ4qns3I9c0dl3qhwMlElhfpvOWrsnWt17KuGw8d8xmtjdP2YFeYtxXQHvoWdJGwPaWVhhyGKq6h3p+GY8HtT+fGhbK2cYtmp8aH9Ius8zLuGIhHyHlZlzD8QTcmV25bGdr31SNLXW\\\\\\\/EsqesdC5shzsBj1gGE4tTU+Y2FrRFnj5USAP+j0AK\\\\\\\/0eUApTKOdJuux+jKsew2jtDYVyN+myKTB1gPN\\\\\\\/opkRa\\\\\\\/x3JssORwskweVHEosw2h4e5vFDbKwb7WG2D\\\\\\\/3iu+\\\\\\\/nwMMdd\\\\\\\/zsUDRc20I7q8GwzvkfE4OsQYmANDrpdC6Rfv15Vod5SutWl0UWGdtWGame67z88u5lNHQ0igsbdz1PJ9WH8UZCGnenCec+cyTxKnBuiOrKGB0Pl466cyQPUufV6+4p2hyLFgxbmb2R\\\\\\\/i0dJ8+joloMUQRIL3awJ1JXanRErXS0qkTg6ElLjC6fHQ8t4ASLbH5E9C1BEkpVomkCKOzHz2ZoRo\\\\\\\/TouO7CD6JeYJvzpRR6NpbOXVeCwY7rJq17Y3OhZOJ9U1FiSnpsXSdQWUMoXfuUHWx7gWC\\\\\\\/IbnFApK7oqn266FOOImY1tlEJN+Lahk\\\\\\\/WFbHg4oSmDQdis7D++HA\\\\\\\/TDYjAhZ1nW4UlLAnfRC\\\\\\\/eq3MlB69acmGlDXIpkT7Yv19vS2dlQ9f9fug+5b\\\\\\\/eGR0EsdeO5tHUzhrZiEOUL\\\\\\\/fMHTGlosoy\\\\\\\/j7HXwYVzZzIJMFv4tbMVc4UcL8qqDyoH1zSN6jdK38ViVLIDZknBAI3EmBFQcUcNdiQYq6GKRlcoAiR9Z2MIA5r0gZ2DAXe7okqL3K5O9dvlSEalDWiaUKYQUBNYAvJhB2H+ztRjbdE2PMyfvTJbEqoByDtbFp99Q6uA0jSAMn8+lQ4bSRgJAFNv4\\\\\\\/0NmtDtAKuTri0H0LxMmjvSgDC8Gou7HoPQOZUZivudSu3NQXqr4qnKLjahgbtoxDZSWq+EBvMHm7XORQ4JkQ7RX3\\\\\\\/vXaGcSSbw0rVOGapjZty2CrFWENRFMDj9\\\\\\\/6fr6Y2WaUAs+ItZPsXqFNz6roxvHzeJCPvKBbesxADkUYUhA9UqOrF3A5QMuowg3pIy00UQt0UphAb+MRR9YpSPidyA4vVwRkCUhXe9eTptOhGnBagmQ9n1in0iQtrlMLsfdR0sXrcYNRWEueMhbtlWl0bAqym6IjDRqKiW6dXgq2ga\\\\\\\/73BC9HWel4eeLbXhRKsgRLvrN2IkJIfYb8qIbk1YLSACYExKLEiJ4e9mbEW\\\\\\\/kLuEZVgfz05MJjYZOq9VJTK8HzmuQJMHpouJM7gEu6kk+RjjGmRzlpwLW49IWunCUJKA8gJsfwlLCuT4iBu+pH186BHROSetagJOGFkc7fXVGfCkkhvSBfcc7bPyrWtwvebHbRv2yVvsh0q52CW2PMxXEFEhoS0ghpJ9K4xMoAA3IxmT1UR\\\\\\\/Vdgjw11+LiULmvkef2jr0Q0btjQH7R32UqG1oaw\\\\\\\/Dm3s+9UDoSNmQ8hKQjvYCz3pKjfxJEUZ\\\\\\\/iHrtxg6nP9VFScBIL\\\\\\\/pK27TGrgGPWwSaHCGFtDgwe3NhLVHM9nq+M+MDz6lGgroBqq6TgC4OW\\\\\\\/IkOHW+nJhVv6tDCeSXJX98DVY7hIPejlblVlIaxk5hnV4QjS9NdzyO4da5XVlKqW6Rbu\\\\\\\/VuDT0Hj8oV\\\\\\\/FCQi5gXpthpeTrzR3GO+E4jRLjwP9GibmP6ix9I1t9PdXa1zOvR57MJTMr9TYmANIH79I4bnJZMvyCvCWCH4aJv+19FEIeCGIkCmeWyYrjFtTRh2kWbuWdGI0RmjpbEDG8Q4rOgkNfkStD2roLb22uTKoNfman0dtf5JeTOzE2qhLDDBMXsA6teNAX\\\\\\\/VGTzmV84H3Ns7\\\\\\\/qQUvgXLfznWgB7PE2ZZ1fu9z4ONDHfmwZ9CZt0Kyh5yp3GBZmJq\\\\\\\/HGoqoIpJhB6ihkVPQlj1G23raUZ8lxluot7RVjWqV2BTrPgHwMMdZENbMZFqB2l8C2xtDIxxda5b1bK9B1ZD2SWe95y5Q5S6jYwdUtzh0WaYvrHcMgbBGiPEkx7cxrcPAqlgzxWjReF2lZw0kRn53PdOeR7O6MAFUjD+YJ1luuLRe+Ds0qgV5GLDKE8AJnFGPSstBGTX1q7TEXsi7xsNwKFxsybVORNEy1wdJHYp7deC9Q6BdFx5VXBHp2qk\\\\\\\/ArPez\\\\\\\/8H1zmXirEmVLgRMCq60HIB4\\\\\\\/AwMsl2+JdUsO+0b\\\\\\\/l9q7j3zbwh9tFx3SfRhXD9ZZyJTgZnzXQOBcQp+b4ohOcVFiUCvBHfCrEJCCl3diT9mhyV9ctWiBfpaRYaZ5XVucUt59T5lkoWgxD1Gy6IDqXyC\\\\\\\/7p99kNnBQCRLCUUuZmA+DKgnIPgMOS1AVySNqIP6EBl\\\\\\\/+SEjg4Jt+1CDLiInSCYjaK6l5i+wmyOEVUNu4y\\\\\\\/1GtChCq\\\\\\\/c6yy+xZPsWiu8JKAZL49RCNZSND\\\\\\\/zIEJTWfYQHfgk19KuAW2qEbp9wwfFVSXwX21WrUCxD\\\\\\\/qBmp686CgThPPdUnp0WRuNr0RjdIvVrEn+OCpjL7XOEQZtRi0HyOgPxcae3jSXNgdjHtfdjfB21oYxidwutYshnFHwjkqEfBHjNt5X8UWS1WyBq3bhdle+0MqNaRa0Tz3tSYR0MdLQdheZ6Atkx3FtP+a8dM9xb4pgD3DXgSvRKHwlN58leF9Jed5x1ZLGivADH5qyeRY+ssUoMK0dKGkmLxzax5H2ZkIuW82gW5XfwVaFklUXuT7mHoAoUgl7CltfPKjhPptP7Pp+dC1S8sZSC2uPS0r4WojSKzv1VYd58h9uT\\\\\\\/8VdWneide0YYyxqCHJBPE1MtvMtR8Op8QxRvxTa71DC+R5aEUeKzrefTWW51RQVkCVRHb7zeVs7bCIshjwx\\\\\\\/FcTkm2Ibxnug9KD5gkDh8vRmONTC8sURjNCSK09Xhz549kQgEogRKZelSj68WHAddJFVRuXmNGmIi8EPgSacS5JCMG0iE4Lsk5OJB9KMYdEwklTE4R5QG3DhpKz17PdtMw0ciL+HWHCEMJpCgPgWCWzpa4Rv\\\\\\\/akROSieN0P76\\\\\\\/CwQPJWIiKV8UFuMj8rSoH3SCk4zbo2bFlw59sb2snO9VDKOBrqaPUoR19IgOz6hEAlT7hgkTZPeP60RAj1gxECjgfDvM011USwGrsJTVuS5vNy+Bzk6\\\\\\\/5o\\\\\\\/M8ykkJL2QbZqjTtaleehU5A6KrCyvV8Wt5486kiGzuIRPsO9dMANFiU1cNXi\\\\\\\/RnpAK1yb3VDMtlZCCgdSnvTPjOwNDDWAhJb271V31p401w5n43TNeDwYkhd4UwwCLIykkRKhDNYtm6Xi\\\\\\\/vQG4xe5r5yfYDGL8e0LGxTnQ2FQeAQBGyoZqgzE7sm3ia77HOQkf18CLMdlwdkA9HZ2Qm79nPrv0IyHInoRWl+iEp4nQWKySjrYhO5Lk9Q0CE\\\\\\\/5XDoKpU2rMPUU2ChCeNh05m47djxt9QoQVWT4HkNyVH60QOeo80BgjG6y4a0zC4pER+WV4A+4++3N8+O\\\\\\\/5To4REesxNECO\\\\\\\/qW\\\\\\\/GQMZZLx4if0TP5p0wsClv7GpPoKj7KWZrKq3SYb9oa4yapvhw0R3EHrHU4cXW2WwCtjDYsIXOnElQBIbb9HqDMyOmqTCbmJu3Nj1mcH\\\\\\\/66kQYApCaHfjTYp0XWREhYjFyOp6dSF42jqnUVoEFitn3ZosscTLaa56CdV0aOhdoWqU8dLy0fwgkkgW6s7q0BmCexLJZY+wCMgfcfrpGmN1yPCU6tmF1GIR7sY7e07E9eQN8OQK3znwrb8mVr\\\\\\\/3b5YItRXQzA879SW73lyR\\\\\\\/0rRDxN8k55L5QSRg7rfDhzMIPwADgpIz2X6XbtFtVpUzZKpnOw1ExOmfS\\\\\\\/GwACLVOjWzooapNpt6A4ScRuPGQVQ\\\\\\\/6m0m986yC3\\\\\\\/iNWEC4m4mco5YJ9eMTKopYMPrW8OSo7HnL2J6yoykyAMWCogsjlC16tPh2eJGzRZSSE06p14HcS\\\\\\\/hgRR1FN347fKwiTglZIJ8d5eh5oNOIUmF5PLJm73Yls3NOV7+bxtg8AOqWckCYcpR7NnuARyQb5AFD6qYpYatyvNU8ygjNObfkjLRzvrqi9ji1CGSOjW5oQ6NMY4jW4U2LrcsTU3Jk6YTIY7E1iZWuI4XVNk3rk\\\\\\\/R+W7r8mmIDhaNfgVjpt\\\\\\\/ljMMMqMcL2FLX8K8cCa9TuvOGyi9ku7VB7DSKJs6biWBrHRdfE1LhANdjJuIosRRUiiMb+MOuiBn6OLrSPOfswefJ46YO5XL27DUlnLbjwppxlCGW\\\\\\\/oltlfhxVntPMZU0UuIPVUoBtU4yizXzYySNkaGug1ZO\\\\\\\/2WHVAMbR0B1XngNSEGSAdaJ3AT6yClL+P93t3pmTPQxySlsVqHk0oV7IX7SgTtH1t76JTjfRJAcDgXY7EW3JdQ3mtbm22BEpTP80gjIKiMGF8+QIg8xZFg8Wzpq4Qdn08bscwTx5U\\\\\\\/HV51K9DIt5blpawOKtmxego+qsEvJTHWwy7w5NFzvZIRLPXQpEJ9dv2q5WlWJsZJ2yQC+dDbmNneksg0lA3Lr\\\\\\\/04gwv7uRG1VAWviNYOJ9l8+twpkGiz1n4TrFNHAIFAjYkmrrLVOakkC9g+JOF7hwF3J+6sh3JPsikcV3lLmU+kCenetFc2EgYrKUFmDLIuAkxuUh4C2heApUOWZf6z\\\\\\\/ZPj4FQicy38WbJ9kwFjbyp1zzRj0UHSAjXgw3tJ7AiIJp8yeL9nyNZWW5nAznEjFz8E\\\\\\\/Dh9omYwASYMpeYmPY9U2s47etTsyAtJJRPiW+n0KRHsu5ukVzoOFg8MQ9Ov5+Rp4pVtF0O6V8I4kIhykvHL4sacn7Zv8pD3CwJyTYe3\\\\\\\/TnnwwhDQ1ZkLR220qOPl8zIhQSZhPYH3huRdeLvaf9PKzCbGfNATgJCr43gIw3hSEc3LZS+e8w6g720RcV46s4Sq896pUwWDIef062PxRUFCSY3wnBw+Dxk+onFpWVeh5H3fUTFr5ZukxwSfnnrVp3EwVbtZiYv16x8e9sd06SPenEj3l42q0E3zd\\\\\\\/PTMtjqMSplUqlv3dbOjMUtAwiOE1f5LtvJRLJfSHdfOjU3++xHI\\\\\\\/cckoGD0exwDb311HTMH1pwtS3LB152\\\\\\\/f5chG9A+ZFt\\\\\\\/bm5lWQ34FTGHc8KdHLbyEJ3pbhn5j0\\\\\\\/+szEm6J5TmUycrFbBpnohZYSEIxsoF1W6Afx\\\\\\\/j0Jx8xYQu8LrGV2rwwwA1Be88HT1EBkWl+sR\\\\\\\/azI7LayFyPqInRP9yJsBNgx9qgXyBX95fjL4nS\\\\\\\/1IScefYDqfg3qvvYb8S2YtaFQppMM8zA7fCccACMlBfnHQHaPD94WzeRrgsVtzs6pSlwDUBNMc5I5Awsh3Oa5VdoDMoRWtbFhqndPPoPXiXmjErsBcmQZallZq8qB19d0Qd32ABE97o+iRXg\\\\\\\/hSGycd1JnadWONs1qWOY7P8qcr2Zv9qHQUonUIYEIFp5pI6ErDFpyA747Gf0dSgxEsjf2tRb1n9Q9Ym0HAcF0tyumEAV97aUGV5WmpYligE5e7sBhMz4uV7PwFtDbGFHncHw0U61qk6f4D9nZ8GS\\\\\\\/e76EKJGZGjYvxrIf8P9h0iY6R8swVyVYtMWN5OxYD76ytjOWPMeYy1LYNREVlo1O+mYQLPDI4oaiBMUpK4FZYcxOws0FP489ZhoJ9UMXibnsgE7fm\\\\\\\/v1+Jo5dcNpqDZ\\\\\\\/8jqr+\\\\\\\/lDg9Su59DMBJDg83A6Qw7eLlSfgBXgKdyp+AiOXMC\\\\\\\/3n2JMT7rRGzAfKa+Y5\\\\\\\/HInw6PmJiULAFiAgZimFhn0SfwFn3cz85ng4wDbqayfUUWbG5\\\\\\\/sy\\\\\\\/xP4H3un0b+qLvH7Imh\\\\\\\/s4\\\\\\\/SIG4\\\\\\\/sH605WnDY7fABwnd02qAsa14H1dag+zsLx96zB6hK3BxSLL2cfd+xSlzlAxCTPUUjTXOziv1DM1oHHXV+7yxmioCU9drI2XVQXL04c4v0f72H9fvFeMcgnmUP17w+ZqvwsgJSvadsrfpAjnkCbPpzt2Q0YFSTowBtucwTguLsI5PBo3+4Ea5ahQqJVkLVCwzsX5gCsYc5d+AQ1AE5emrz34MsJtdt1YEIZvR\\\\\\\/h\\\\\\\/QFh2Jc8huuIXp0L4eZGJPjTuWBOucdzuFrQuOuT225mPDemQqOLIcmGeF\\\\\\\/jc7bCyjEw8+1QlE4gqPkbq3iB1Hiei8kL0\\\\\\\/DqRTNILHtFkEmimLrfF6ZwN\\\\\\\/HyQqpaMJXleLdtBTT3VT7FfZ+OcE+X7t2MtgkQTZTPvDq0PPHTvLrOhKKA13P8bpR02q8KlJjN16G2ki3o25dvL\\\\\\\/3QUSMJPRCpt7ZfqGZkyU0C4rOEXdwIwiqhB3L8b\\\\\\\/9b0FhpVjy0tkLpb2RkTn4G2UzSWtJR8S71oyuU6RQciji7exofEbieCgR+buJqg1K8k\\\\\\\/GYIW8Z4MdZTAOxHTujn53+SKRNUOMycM4PLlvahc9X7ItliZXWvOl6K\\\\\\\/dY63xk50\\\\\\\/DdKgG9KtPUTof4wLBI\\\\\\\/8OFBgBwXyQy9VQcgukH741y+M5eX+r2qNe0ZSDYtAox3PRml+3h+B7tcVVMkQLcgInECb5jnSRWN4bdY2MEtv2FqYwXaiOqzEaNsA3cxhyzffQseuqGkyti+hfC+jGEtULTE\\\\\\\/2cnreoyELGy3+Om9xdLSyBGZVnAOH7uc\\\\\\\/3lI2RYEHzSwHSX6G+GqRO0iBHVw5FS28xpwdNuVERcY\\\\\\\/qxIRnrZtL++2jN7zsCRcBDrdyTjrAzdlxaWQUE6Yw+Ey\\\\\\\/i7OClfFEaiEYTIGJHYjsO7evm2ZdmaG1qahzEJolE9dDHpIBxh3gkYI2GJRvtUpE0uZWSYedeoMaOXwJ0LzEMbcVpAOlVsP\\\\\\\/HCTp5Vpyou\\\\\\\/sjqX05ixYa\\\\\\\/sxVxsP5wd9DTeQ2vm2fmlUb5n4cBLF3ZnjRq4VAfbOemXrI8D2F84TqzKkjqguC+89n2BR1ao2XJkkl26or2lCwGC7MBa+278Mw5axCu1hjX0ZaQdBdvs3LXQ\\\\\\\/S9naPTZdGskI\\\\\\\/1\\\\\\\/6XbRt3Wfq6at3JPrkI0LR+48iciPLMJuIZfABh5zIJmZwgZkok4dWbjb+9k9+Fnpbp2DgBgLV1t38iBx1q\\\\\\\/NQMo61LNWrsdqhd1sRg8bARhTTDcHlQNMVXAaWqUb2ljDRVe7WJqBDWh3gvLoLAPq6tnB1NG\\\\\\\/2mrG5InoZyyEVGq6cYTvL\\\\\\\/z36Ry+S0SL\\\\\\\/PNfTUQcMZx5iV33yIXZiL58ilNvwK9WAt6BZHJgz2zN3\\\\\\\/TxKaYdKTrfKveO+y0RvxFCs\\\\\\\/in\\\\\\\/0MwjZjXrSCWpRNCCW044Yefia9PwN70phYws2gyFdrrKjo2JZ0LGalgzsS6DImA4+cGTAWK9\\\\\\\/\\\\\\\/+xgZSWUz8DUiHSKnb5qDIH+j5IzQdX793Q6uBbNTp6J3gX9nIGltX2zsD7ForJ09u0awxYiFhRqbCtgwSVf78LtByx7SkaGEHi7xLFahJ\\\\\\\/DcDbc7WmiL8FfRswdrB0WOMbFvE5EBNCz6FoJ231vFFHbbsiXl10eWJoPFV4FBeU8CzHCPi2wBUVeNZiPwMVij9RBGcO6QmljgU4c2PdSFYcdsH7AmijQ+aDB2EEQosArPwKenUScffTxKpfJ13ypy0NAKxsfmw57ZDF\\\\\\\/P6QEZNk4UU21OXRZsK5JnoD8xPEgNuW7BZLdrwfH0S4EXv8fvuqGSAmfzZG8H8EfF3Av9QlZiYMazd1+Ov8Cc4UoY4KISrMgBtLYIbb\\\\\\\/vmnlYWQjQT7LHRnoDmTobmb83ckKXelUkKJJb\\\\\\\/Ly9OZdYPIkKN0PVU\\\\\\\/nUM4AmCpL6MzYEjkrs\\\\\\\/g4cMCpUykarFLKQ+V5LOqzpfl5rephr1RvZ\\\\\\\/PPq5t0gJlsjmnMKXz8LRD61UYrCsX73E\\\\\\\/W45Xa3kASm90gHka7NCEf8lfReBFscBS7IksB\\\\\\\/STcf+o6Wdxs+s3sUHfnMKxx\\\\\\\/DbfCLvppaS9AAKZrKPR\\\\\\\/QOjsK2Txl01\\\\\\\/G6VGPhNKOv9pMIxkgclw9rGQ01VZRVlVVGCmYh7RVg6LzQDFZW4mM\\\\\\\/vk5elPxM+rtxkYnp8Sqa+35WL7R5omlxDxJYKHH4EoZTSiPw8yTSY57beDrtHk28r0SOtgfY4\\\\\\\/b4kZ5Zg133+40MpcqIWnfU0voVYFK8MZKUj441JqFetYN1y0eyzlAzt\\\\\\\/vI\\\\\\\/QQdVSQdYsY0ZE9ezrRkJFjpstQq7N62MpWaRNBBGlOzsLbCAbt1tYowQBDAk4WqQ\\\\\\\/4zpUfZOr9I3BXjKTQGODqeAPkwjoorLqqDo5aFvGvk4owLvQG9sQ7Uv8z3F0stCZjLEiLQC\\\\\\\/CVv7tnB28ceMSR9Nonl+vCAAdVHChLXp9cwVSvbFYr53OboGbDH0Hf4Z3dynqnn\\\\\\\/7b9y3NCWz0pXBrb\\\\\\\/Yrh3f7oNFlIVPOO8mjSNP3u7R6PxR2VCxO0FRvsVfnNfqx+W3r02QxoHleEqJVMHRlOcoxk\\\\\\\/q3n6ad7ntNHc2p8mLh9wu1nhFra6eMjnaiWdckR5aK2Zz5JEYVzA1dGPupYtYrgm1RacnyGGKiLReufNe5x4atN8ywQ\\\\\\\/gJcBANWybygbHSV95RmVtwLkHqHtN0c5BsCUDEwNdt6n3CteFGBvvok2z\\\\\\\/Ziw27QS48vGQX9bX5k0cUBbUfymVj3SGQfH1ixRWmLPXkfpMBkNV+QrytAeYkmlJ7SQcYGc4KayQxXsjQAHm+tRQUN7IOKvjgaJYoV7878ABZA2GWbK18+4i\\\\\\\/5RF1z5rBCg4oyVMx0eqpC2QcZMagvCw8tPXS2Fz5wp3F7hHYi0agppwwQUlkTHE8LQbQoDKoGbIGOUVnNQa3jjZToylyqV+KbP6UOamg15H0UIJjjtOslhksvr5lL2nbujroRzcpGH6QXLHmXwHgXSFTXL6d5sNK+9juQT4pOM6hyMi8lZSOm2jbWxEds6D0lOa82k\\\\\\\/dpWUDwE35h8mwMLcydgAlyCydU73NcsEXwl8CjMMHrtmOYdnJA4gA11y7lyPj0OHTLWRiyKGEe1jZFHrWF7Kq+E+DdU7Hx6QU1dIwL3V0fHgh7CJ6bCjsKd21z+LlKM80wGrYEq3p2C5DV6lk0fuJNwEbzL0JvUF+dpr+zWKoqYrSUIZhWt2c1GS+vbdH7i5wFbCB6tunR9pcVXWB42E2EG4HkFPtdin+KApBz06VZOdHHhOLSZhelv+ZUJ\\\\\\\/nfg2yXYIoW\\\\\\\/xhNZRllGkDxWqOyreHDLGXIDptYoQjeOffz7trUR1unX17aNI14ayZAlZ1y3btvJwvP3blFsF92ptRW8w\\\\\\\/z29NvdojiDLh\\\\\\\/54lR\\\\\\\/m+qKmoBeyf8mYEdgK3QZ6LVDmusx53kfft5ODFxWw1fHvZe9VBL76y3QCkoeue7Nr8O71vg2j5i1GFLZiWTHdCVbqpQexs4Ig8CXG64hTzBrI928t+tnWW33U16us936GS5A4x7+lDhf31r0UaRhZNFrMGPpxdpK5PbCA5OQ4aTFRyBf4sYjb03Exs0OI5byuSXK7FMzUgTemyj0IgVApBiLLnT66Ye3KXgRgNXkgmq6sBZtdoHnS5GSeKtkJ9cMO7XCAp4LSXjeGq5KS27yZis5oQpCjNnRKAXdpxzDN5w7XdDW+L39qbGHoDyKMx\\\\\\\/NiavXmaOGJYh1w1czdWOgK2r\\\\\\\/zzxIUppeNjx29YxywtRtN\\\\\\\/837MDE9RcT9U9Uxlu5KoJDONYlcOmpYEBLB8D9TBvCvgcjaBLg+ZhYcKYkjf+fFDEOW5o2z\\\\\\\/vcyv2mPsh\\\\\\\/R0tBmyd\\\\\\\/1QdjCN6KQGUe6cma3+4nGitmDqP8QplPNKT9QVfMzuIybw8P\\\\\\\/pLuKq1SwZKsMyNZP38zNrH80oKIk+cVv44M67QNyzd0Ps3XyQqkkQju3AIWDXQpauNVhFXAqwU2pLaCQg+yoRs+4RGFF7GrC1wqSbfzSKcY4jS7h6Fj4jLDaADLAPyx6UZIDzcwcE8QlZZaRz83tVGGKioufdlt9bez0AkDYHToMcHCbYxvXyHZpOY8qcziW9rI61MxEZB8hUFLq2GiP+XOLHblzHgcbA+p1wbOOj2IC40J3TZbwLKi81BtliCnV6fgtx7AWuKlxrFAzE+BbTLZz6U3dbv98VL6IOjRGaii\\\\\\\/01tyxcuRxo9yalz+TLAl7d1+6nErfc2ktq9VzV3UwAdjnRqA5RSguj3t5o4EGv3m2MlZ6th4v8oBzOgVg5JhvCxaqKrOl9tF1o8f6gvi6TwGIArQaO4djFSFEA2c\\\\\\\/8v76NvnAB2enbdAyPO5PkliflrPbv1I4TY9c68tygJBcNrbGTUKvlgv9ltIMtM+VGpVOZ1iMtW8DwO2EmS+0DRY2Wv4qR3j6PMRDUVXM+OIZ3cDGnS5M1E0p1ySBAhMHwJZRTCK3bRORZiWJCKNiG3Oq3dWQmVJcMapADKsEjWdlYkWnSCje\\\\\\\/V1EfbVOiEyCDn0rMWVBPQQxr5i94Pea7advWlWH0c5u72mxnp+maCQpGuLnhIlt87QFUc0bmcaU2puokATwlvc+t61uFudXrxX3KjfFIea2CTo6uQWfKD3WcFMlJwoC\\\\\\\/4qfNDbLRihm5wKWsbuxctK5OvRi6hEIc6xbK4Udf65yZf0RsCh6WDFICm\\\\\\\/tyO5DZhwdPjSG\\\\\\\/TZNA9x6iYWzzEsAkjhbfbQ3kae+Rb3KZkXABDmKyb5cqcR53wAxBnpSkAwrn\\\\\\\/9cqn\\\\\\\/XRj68OEiMpUwIZZdyfDVsxlTZL47hl7dpWBDyjdCdD9jjSnsM0gu4y+xL0q0riFwo0zSHSDYOp30ujH6dCQL7yyLXHR++2Hp2nY74+ek5dvr6G\\\\\\\/39u6xzk9QkeJlZbExV\\\\\\\/RuGpg4EAYbgeWP+jFsH0ieKJhdEdKg6SPKj75wU5PjjHjits7D1h\\\\\\\/8JHoTPnPrL7Up+2pvtfwxOOgobdQdRwkr5zGO2+zQF3GYO1IcGwoFc6liqCM9M5G4Jig6+O4kekMqfc07DO93uS6SlUkgzkRCnO5jqRcJYKVoybyQtIEr5lL84pE\\\\\\\/XJGoAnMSDokC8OottTieB9YKBCLN49+\\\\\\\/KCq1nvJX\\\\\\\/3gbi32zDnIS+5U8atEeJqdgeCDIV8oi+hovOMOv06ccA8p\\\\\\\/B9fEIPqn\\\\\\\/xbg5l45YnwTYPTBvvRHT2ouX0dqoWt2CuIGIj4j1HGvCLn22jhLptZkE8Eu0hMyfFiIVyI0ECLdyCch4IrFEkIUe8CQyVFtUjjHUUUwIzjaMDXNYIypiZWceCQkcbbEyEXRpBEQCof56ltpzFfsUFVzv5+ICCUtyfq8031Lpl7GlJCM4VSLd8NzffwkbStw4QfGgs5dQs6RCn9Z6kZjGkrYPRV50KttwcVphvPksH81hPAT3vwptp9vqmL3S+g3W+bZtbZBFiSGMRyVcvD5KaJu3MPWEubVHgAM3uwnwwDf+QBJW\\\\\\\/Ff+zUk1hFjzbhmLoKNfxAoPH1YQjGl8As\\\\\\\/ya92fY8n1oqZWbxwBGIjZVAPvFpadSYSoqCC8YU5kftEmzsWv41Uz52PA28l9ja\\\\\\\/TmmsIlzJK+ltt9U1DkE4acONyZiAQj\\\\\\\/Z1wmhcsv2gykCgU8S42tZTpe7DtMzy+44prE1u5P5toOHjf\\\\\\\/gd4PTBjkdE\\\\\\\/+gkJ9rM1OGuJYPA5Ve7USCrRwShsbcfaR4FWhtVEP8f9D\\\\\\\/Zsx6saZbACLYlkORbEHioCT845ZLpXVToACY0P+PvjL+Ecyk+HZ\\\\\\\/ez5qG8nNPALGkrkqJQ9wRph9sMPnnYQRlp\\\\\\\/hBPbWVVZX5N2H9bYswWKjmsYDG33z5Yn2G0wKl0boDAEMZbWJx4+ttrLAfBC+xkDZhJ8\\\\\\\/qBRIklOhPxAP6reNOEjeaqDLiVFWdJlGiVMXOoDbLzq+GM2ieXweZ6h\\\\\\\/Rv7Y+y6WdeZvmqiDc0QHVt877adh8h4UDIVS47iUXHDNXD6KWYR9BpdjlExLTnjGm7BQzUWe\\\\\\\/kUKkuTQecqfIH8nMZsSdg9wrQt+QrGt9L0Bwre8FdXHp8XA9KHMv9rNgbxWqOJZYss\\\\\\\/rAb9QIzYkO\\\\\\\/dEEF37ICwpTc7us5ns9lXCrgPTm5F4be4j\\\\\\\/slUD\\\\\\\/eg6\\\\\\\/m14PqnMO4DgXl2DORzPiGToXbHHSlKFN\\\\\\\/8ifBCECmwrkKPoregJ+U7vUKjrTGTX+LUWMbm9aL+42s63z4ASrH0fAkgp\\\\\\\/tBAMVq7Db5LGXlCMyCeyy3aVamx8dtMQVMXXIavP\\\\\\\/J0twRcvrgC5uz+f9Kyw\\\\\\\/goKpnTzeo5MJOxbCmOWiypcRgLOmWlgcMkbDCzu4kdf3ITN4Zd+BnBlX0GclIxSCEu1OL+15iGlMRnAsSvJABpxAFRvJjq1kfRBcb4QV7dUje6kx6fYFlYuc69kTVS5Xu8ZY1HPuf0LgOoe6i98QgDAg46pzsaC4lLDFQrV9slFsiiKvwGfZw0E0IwfqednN0t+n8Ek7hNil8c9wPl+dmAxuHHM4Ygid0REMR5JrdNSLtQJYWQPbO2xL6w\\\\\\\/McUw6DjmnSIlmBhbkKjvq5yXG8U9yGeio7mygynDvqjeAzzhQrM9VlDSQ15gUhZOUkJy+Yb0q5jNiNglTq\\\\\\\/wwCEDAcUxzv7CApK+S\\\\\\\/RaoKpwXK\\\\\\\/SqJycYEvIl9VABIzRXaO+pdynH3D8xymhpPj7hOmpsh2FZ2Pvj21O\\\\\\\/q\\\\\\\/JP\\\\\\\/gbvDp8MLJBM5m8J+wOES6DFwa42y8I\\\\\\\/3molHkzEPimea5YSz00TItkyiqY0KVnnNB5z1Dr6HEdPbfXgpNa\\\\\\\/xX5ZVSJWiYA5wPrq0thi3WKWmNDzTU6DE4OEkIN67dvCk4oQCuksCzYIapgONw979qDSA6yUPwxPdVZLsYt1vtTG7YuAGZjp9mZi92Au0oPyhDRS+XLqEZzEfj9PIf+UlByny5f+wd8pEGKVOrt4cV\\\\\\\/0qwd\\\\\\\/2HkwL0F3mLpKiIkcZTQfgbZQOgYAS75vDviuLQkREJfd4Lu0kn7YW4v50zZfC669DXAUMCRY8VVHicCTM\\\\\\\/C9jHgukS3rZ0ABHoCpyl+b\\\\\\\/igw5qt5PRSAyzXbmGWSI2LDm3A0Wa4\\\\\\\/XPoBmxyLqxzrg9ljiw6pgspfsjEKh3z9GVd9wdJz3odKbPpYFYe0Sy1S+3zbAt7gFopaIKnQFPO9qf2e7Sx2MFlhWe7JIAm1MwpwnqvYBDBQO3C3LRwppg1t4PCCyCg\\\\\\\/nrZbqX8h6fCzeP9RgjS4QWcxgLg\\\\\\\/uzGO22CMCM8GkpOrPA6W8W2d6N6FfSypLW1zI\\\\\\\/DeS1YSK\\\\\\\/zjYc7X8CKz8SlDLXh4aW3M4DtLqWi4FdhaFxvTLQ7furX+1+MlM9Fzr2noZZxnsQJTRjnMf5DtGESEk1QlFkmWSTj2UOjU5AH9sVU3xuCISjzqMP6JyFCFphylmYiGTeZsah1uAt+ToK92LkFq\\\\\\\/3gVi4WOFl3B97oub2u4jyXUaYHt9hJFhvOSMbodpl8N8qUq91bga++D77M8OHnbpntLltPbp6iBqEq+yPJMiIasI7ccoFsjsLF6W06cVAJg6y2MPLorCHHrbDBm7k6FOJYrebbv2gmnSDu3oJPnw8qDPH1wtjuZ2sCDV0VnZZymK8C5PdQSYvkkrh5Bn2+gog1KUnO++WWVYIz8YQU2sOpmajBYm6J7khd1dJchH\\\\\\\/YBMzudG4AYdAXTDLJ1xbphfhj2pcIsH\\\\\\\/mw7qzpTm7Xfpm07hh475WnNqGqWX6RfKZeYwKeW8sgOkVpYLkChaQv3ZMTnN11ReB3hOL1QEItBzZB9+7NxK4hxUZl9IOWEmzTGgWkXw3P0r0n9T+0UjC\\\\\\\/WoFlC7oWDdxx+Xch2dYdYn2v+ckBgc8JXZogdrHfJUSpcj18R6BCxEUv7alKuM9stdAQlKZkxRnY\\\\\\\/SxLbx8vz+8troZekehU12ixQKx868T4vsRI56PD9Jn1UN8h+xmU6unGcVtPjxrExxlWNqo0EzYUTN9qj8LpA\\\\\\\/KvV\\\\\\\/edHAgUP7vB7FFGpmJHOLHDU6CcFEsAJBc9fYAYY60MoUwtUjDDKILraiemKeLSn4OPBavGLE7NC0XW0RC8b4DxAPLxB75O4ks9JaGJ\\\\\\\/+026in3VQ8AlQGTJ2OWuhyYY3+rmYgsrbGFKMH4jyNxe9hoMDHdMIKyqSloO14xxfA2yJ2P3U701ap4vSVD6BNS5ATWPrY+xeSHRsh9hlc7504iOu2027XFuS7VwxZmB3djNGBREhXS0mMZyUrqWTWPrVHqHZGHVbZjV6\\\\\\\/SAB5gaga8pnVRxkt5m\\\\\\\/wjAf320aAxespRhPLqzhXJWAX5WEKxWJIRW\\\\\\\/RvGo0GDfLPmokOkVtny1\\\\\\\/+fDTtiNhQkvPkL7xyDmGypzplVvOYfBDz36ophZxCUmgywnm+2IunOL\\\\\\\/idW8p\\\\\\\/MDU6Tw6m+zRqKkOicINEMySPhAHkCwJ\\\\\\\/WwDlJVni79cJJw2P4gjGruFBGNy2R28WJ9+dcm8LRBkxWJswATjZ60hZAXYoNmntjluD8TBjjG3mW17GKnQgaCjLr72DGm8e9Ypnn7A1VVWeDUjDDdvG1caQp\\\\\\\/8\\\\\\\/nXXiuilSAf1UhviDR6P+R61KMERPH+\\\\\\\/hZdH4rCdf+lFE9a4yUcqd9wNbQAJwFq8OUaL\\\\\\\/dL+qd6trfi5Qh8cl25jgedo9Osp6Fu66GFEndjf7aut6jhe+afl0BVJjrJRoLXHE8ls3LAwBgdJBTVwERu4Q7aT9JLx9OmEESEvymqhI7FIZNRmI81mrTTZYOzhkYzc6sRgcRvyWm\\\\\\\/tOIXUmy0S5zDeoeGTGGlQCeIDeHO\\\\\\\/4dG53qoDY9YttqTi08uk296eNt7Fcx3zl1YPNsA+f0mo4Vsa8nO26avXnjXFJGikaR8ItzHmCf0sod8+2OAfI5k5vaQNzvJhdrbbQ5tx1SseZ1o3pg45bs7S+kRbncQLpJY7L1dIrATpYXULgmbC0tj92FRKodMa4QJxOFjkDY6w76KP\\\\\\\/pyZZTTredF9VHwA1vcIKPkv\\\\\\\/h1I1fBKQ3q7MkUyRZOBVzKQ\\\\\\\/EwwFdCngQP77i5Si8NAcMxf9Fx35Wqn0LcPKFaXSkoADxYGgeogr\\\\\\\/YJr1lgLntrf\\\\\\\/NoImYqFcwXRBnSBwiaSLY89R8eemYzHqgjKD++QOGss4iL3VsDxYcycz0XwWeSYoZuphmjJmultwV0mPQeFcIGpuQW0M4lzsTGrUBShTlLRNVi1ScurbvHUh\\\\\\\/wzQjwto7QlDRZSQXG9\\\\\\\/PauR\\\\\\\/di0x7OzUyjzFB72d7fVBGlaHyjJt3UVd8+D+5O6yLcMHGyCe+2tnlzuOGYJ6xt6sAE0YTiRLRaht9N\\\\\\\/V3mrnew3rYKG+ZkL+P3HnCvx46Rh6Kq0s99OX9QyPv3JOxs5yNW5nv8KxQANEPH3DSF5yS7InNtel+AMhRNQ+9Ifi5vEb7enN0jC6SMDtY13ywTm\\\\\\\/PI11xqzon5GazU6wkQEzQu5nrmLdV3x8PWl0obwyWx13w5bJ3XoroW7JfbeJy10pNuBkXX4eCCdd\\\\\\\/WcODpALVkNDO2xw3b12w1T81\\\\\\\/oVwFwPaOkJ0XbqRGCAakb9TaAAOZpbWb+JvfzdXxkmy2O0QCYAKkHWKCRII7TqB4rT0\\\\\\\/qZaaqpjlYlkGl8R5i1OxYNx0kJWLpugJf0bGNmg6vjCIVXGIV1zL6yUh1YUaoCgkb\\\\\\\/JxcwE3pOqS5OVtj8GZCYi9OkLfRw+4MIa23mp3OAc3G+XxbqnHBAqO1FYf7J3Tdg80U5sBnBBIgXGicDaRLE9GFcEMIaHDSIU8qPROzbm1L2RBWTxXCr48trjcCDVlVsFVXUC8dhhg+beHiEv5nOTFo6J5pWfp0gMaM058h9GNRRNJzetGC\\\\\\\/0gNk+tdCjVRyhbeN+EudLdx+bpS81\\\\\\\/aYhjrBDCYuOKquO8hGShA8A0wwQ0fJkaI9iyjWBD+1vLutjp5ps+0TAh2UYUOXzUaEqhZ00V4WtDgdkGQ5JIpQqV+ETRIca72asMU8O4xipOkElM+OuAHyDrELb0UpYadub\\\\\\\/GNUXi46ujdcVhgv0TAJm0WlXawYDirtgOxwOjI0T11N+YJyNr\\\\\\\/Uua3GM\\\\\\\/enEctV6R8hNoFRO6QzRO4Nu9xq175LuIAycUHYvYmsXqin\\\u0022,\\\u0022iv\\\u0022:\\\u0022WRv9uzOnlhjL6VhEJJE1xQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ec6f4de601581fa0ee9f55b05a38b1c3d8c6de4414a649856a41283953bb4acb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XkrkBoUQ0p65\\\\\\\/ujjThQ5rxt2jmfe1mXK5eYqsh9Jls\\\\\\\/t80GrtZYrBu1KGjzeXhmhhWeTJi\\\\\\\/03zcpfDOvXehX5UgNw78\\\\\\\/kv1t2oBIoiCbOpw5krYuovpKyB1PFez3aZHNfXjq8w5ZT45Kkd5bGgn2\\\\\\\/ye03Jl34zBZK\\\\\\\/sxWrB8smv3NYDzalWe6vJBDexwUIOYV4iPCQtT8zNdK9qIPPys15h\\\\\\\/WIe6VaekY\\\\\\\/KCP5TP\\\\\\\/CzQtuhh8J6afIlM\\\\\\\/MKovLhWCeokmGm6bz995oU3m+OHkFCoRgg3CbpovdjNn+QFwKd8hM6dmlccuREbDOSQXYTCagl1bHadRAiVLtiHY+Qgr1Wez3vBfN1QdGfh4yYBNS\\\\\\\/GFvpdJ5Vneg7lvPOH9rq169ZKEwIYzbIhNE7iRqyFKTnHuD6fMKTtnmh9m1QXHWsiv2g9InZlpHuqPjx\\\\\\\/K0Qapy6T\\\\\\\/MyU2qGcPjz9amI1D70qoBX724bYEez+z+V2pVWWROpp2oqJ7nlc7t55k\\\\\\\/TepJBABcvbQ\\\\\\\/hyltPyDdWU95i1BxEy+z+0zvqJGmqpAiKPzRiBSm2Cf60gwCjaGK8FLdEgTPXqgBm1MtwUkBEAD\\\\\\\/pKoFsQ8KplZ28IAkJgF0GZjkiRFRHvt\\\\\\\/sXSCbKiTd0Q+PymW2lr+IkvyUGGGFoNaomdeR9FEujFQf97ME6H1wYua0FQGp5okoX7yKO7EcRk76y1cvpLYp\\\\\\\/M0CihurGYRFo6htiVMTFPXxHwjhgFP4VjiP0mPa9eQsHGM8KsXR9ztzTY7EpqnscmKEtarplBCE5DiZvB3DcRgEFAlUh5tnTsvTyvG\\\\\\\/Jvth+hpLMl8mpcqpi70EDw+l7e0dnx\\\\\\\/0gmhTcWr2yvLOCd4uv\\\\\\\/RbLNGSfz9Rp\\\\\\\/1L8u4MglxMuCfR9M6dL+9u+T7+2LjTezOIbLsvixqV0+EiUXmDNpVyfYkWoJBHryXwDJsfnV8CCToG6dnecE6YAVjnu4SPhtniovU0IUR4fsu1NJjFMi+We29lssPCCf0uhwX3TlcbH3u\\\\\\\/u+DvkqrMpgX7AmyDQF0UWVvAull0WLrIvNJTjYykEkDxDnQiHBFxs6O3NgHzwct19xdkBl0RB9hEcNFGEvKE17EnGc9NxVEIHlTCCW7wSyX2r8ph6PtOfYKVtuT+jpIxfXbaIGpe84MANpa7vxgFWEeo6H0MfJ160J3vTO9BD6FKEMitOV4uUMkrucdX7+usZKOwh1Zty6VJgWALPj7FrRXSrABSPfk2gphZmKtXnFmEzY+GoGpC1nP4i4MvyL27fPL81fu04Z\\\\\\\/mz3StujPZQ7XBdBpPqZ+wV+xSNh+nBzKtAGmoXxDhAHm6gMC4UpX3FiUF82jVERHlfNro6LMz0RTGNf09lLUFXBXsQ6HYtQk7kVUBFA\\\\\\\/WLITMCZh+FI2kDhXpgoNnzEkGSeL3mN8CYCWeSk7KV2arYpdjrIy0lTtSlYqgdxusmL6VeZqlv97WVSpSGbFnIYkRMp0LsttPf4czbDRbK6sx0XLYu5hSjoLXmD0CpiqveHM+yS39H4QOqtlZBhZQrZsHZS\\\\\\\/hvSDG5FSwZReC3wM4dOa0p6nhiEKuU5J8J0HuxUtokWzT97Wt0Uw6PZMQ44LmDZ5aUgIKaoOlQwCBtjiLXgIbqrofxZw1eDYPQ7k6uwjXlh5ceYVNzdjIs2ZfaqLnaHo4nEjvRBB39JWLO70zN8eu8XHWYvtNR5M1NfJihX7evUbxpp81KxtpHiIEHwXasTAp3xFX39pvu9nJa1VIPlc9Vx0FuX2S350MJWNwp3z\\\\\\\/+A5SIGqfKu4OAGoSs4cU3pTUQ+ftXZ9P1ATgR9pPdEbbEnrS\\\\\\\/yIJ7U2LumwUHwEufbAX55rQwUqbLrAcVrsVSPEmJcRQhSyrAzpTf8AEbGFaOzutuw\\\\\\\/A7p2ruebPUKhxTvKJVuFVUZTQwmNO2gc0qnaKZTQ2WNJFP10lO\\\\\\\/YbvaEb0ukrrYr92aI57jG4zSqe43YOZQxIg5+Uhr0+rXbf+5f0GzEqp1eolgzP1MCysyc387SqQ6p8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Jjo2hMYxcCONUuGAFoPS2Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022680fae213a70cc855abc78d2cac7417071f835cb7fe6ee95e3717a1d22bc8c58\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460361 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1153" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-EcJ8-qB6DFX1fbwLfyDfgMeCKXi9fLxmIHjXQp7w7WU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1153" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Y8BLc2F6rFZIqUazKEJGW_CBUlQM--bQf4Dloz-qg6c" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/5/1141" title="The Genetic Basis of Baculum Size and Shape Variation in Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/5/1165" title="Genomic Bayesian Prediction Model for Count Data with Genotype × Environment Interaction" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460361" id="node460361" data-pisa="ggg;6/5/1153" data-pisa-master="ggg;g3.115.024372" data-apath="/ggg/6/5/1153.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Does 3D Phenotyping Yield Substantial Insights in the Genetics of the Mouse Mandible Shape?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5694-4201" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicolas Navarro</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7921-9018" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>A. Murat Maga</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1153-1163;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024372 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460361" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicolas Navarro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biogéosciences, UMR CNRS 6282, Univ Bourgogne Franche-Comté, EPHE, PSL Research University, F-21000 Dijon, France </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BNavarro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Navarro%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicolas%2BNavarro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5694-4201" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicolas Navarro</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nicolas.navarro@ephe.sorbonne.fr" class="" data-icon-position="" data-hide-link-title="0">nicolas.navarro@ephe.sorbonne.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">A. Murat Maga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Craniofacial Medicine, Department of Pediatrics, University of Washington, Seattle, Washington 98105 </div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=A.%2BMurat%2BMaga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maga%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AA.%2BMurat%2BMaga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7921-9018" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for A. Murat Maga</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/5/1153" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460361" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460361/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1153.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460361" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460361/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1153.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460361" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460361/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1153.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460361" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460361/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/5/1153.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460361" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460361/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">We describe the application of high-resolution 3D microcomputed tomography, together with 3D landmarks and geometric morphometrics, to validate and further improve previous quantitative genetic studies that reported QTL responsible for variation in the mandible shape of laboratory mice using a new backcross between C57BL/6J and A/J inbred strains. Despite the increasing availability of 3D imaging techniques, artificial flattening of the mandible by 2D imaging techniques seems at first an acceptable compromise for large-scale phenotyping protocols, thanks to an abundance of low-cost digital imaging systems such as microscopes or digital cameras. We evaluated the gain of information from considering explicitly this additional third dimension, and also from capturing variation on the bone surface where no precise anatomical landmark can be marked. Multivariate QTL mapping conducted with different landmark configurations (2D <em>vs.</em> 3D; manual <em>vs.</em> semilandmarks) broadly agreed with the findings of previous studies. Significantly more QTL (23) were identified and more precisely mapped when the mandible shape was captured with a large set of semilandmarks coupled with manual landmarks. It appears that finer phenotypic characterization of the mandibular shape with 3D landmarks, along with higher density genotyping, yields better insights into the genetic architecture of mandibular development. Most of the main variation is, nonetheless, preferentially embedded in the natural 2D plane of the hemi-mandible, reinforcing the results of earlier influential investigations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/3d-geometric-morphometrics" class="hw-term hw-article-keyword hw-article-keyword-3d-geometric-morphometrics" rel="nofollow">3D geometric morphometrics</a></li><li class="kwd"><a href="/keyword/multivariate-qtl-mapping" class="hw-term hw-article-keyword hw-article-keyword-multivariate-qtl-mapping" rel="nofollow">multivariate QTL mapping</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/mus-musculus" class="hw-term hw-article-keyword hw-article-keyword-mus-musculus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Mus musculus</em></a></li><li class="kwd"><a href="/keyword/mandible-shape" class="hw-term hw-article-keyword hw-article-keyword-mandible-shape" rel="nofollow">mandible shape</a></li></ul><p id="p-3">Geometric morphometric methods based on landmarks offer a convenient statistical framework to conduct quantitative genetic analyses of the shape of complex morphological structures such as the skull and mandible (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Klingenberg 2010</a>). Among model systems in the genetics and development of complex traits, the mouse mandible is probably one of the most extensively used (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Atchley <em>et al.</em> 1985</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Atchley and Hall 1991</a>). The primary reason for this success is its intermediate level of complexity (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Klingenberg and Navarro 2012</a>) between rather complex models, such as the skull (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Hallgrímsson <em>et al.</em> 2014</a>), and simpler models, such as the <em>Drosophila</em> wing (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Dworkin <em>et al.</em> 2011</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Debat <em>et al.</em> 2009</a>) or other insect appendages (<em>e.g.</em>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Khila <em>et al.</em> 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Emília Santos <em>et al.</em> 2015</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Prud’homme <em>et al.</em> 2012</a>). One historical reason, and probably a major constituent of that success, is the relative flatness of this bone, allowing rudimentary 2D imaging techniques to be used effectively.</p><p id="p-4">Genetic architecture, imprinting effects, integration, and modularity of the mouse mandible have been investigated in fairly high detail using geometric morphometrics (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Klingenberg <em>et al.</em> 2001</a>, <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">2004</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Leamy <em>et al.</em> 2008</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Suto 2009</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Boell <em>et al.</em> 2011</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">2013</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Boell 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Boell and Tautz 2011</a>). One common assumption in the majority of these studies is the approximation of the mandible to a 2D shape based on a photograph of either the medial or the buccal sides. The 2D imaging of 3D shapes is well known to incur information loss and errors due to object projection (see <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Cardini 2014</a>, for a review of this source of error). This flattening may represent a major factor of variation in the sample because of its intricate interaction with the positioning of the object and its shape itself. Nonetheless, for years it has been a common practice in the community to collect data in 2D, mainly because of technical availability and feasibility as well as processing speed compared to 3D data. Despite potential consequences, the 2D projection error inherent to this practice has rarely been explicitly assessed in morphometric protocols compared to other kind of digitization and observer errors that are routinely controlled (<em>e.g.</em>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Muñoz-Muñoz and Perpiñan 2010</a>; <a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Yezerinac <em>et al.</em> 1992</a>). For the mandible, the sole study that we are aware of reports that 2D approximation is actually accurate for marmot data (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Cardini 2014</a>). With the increased availability and reduced cost of both surface and volumetric high-resolution imaging, it is becoming more and more feasible to conduct studies using 3D landmarks, thus reducing the risks of issues related to artificial 2D flattening.</p><p id="p-5">At the same time, in genetics, there is growing interest in deciphering the genetic architecture of within-population variation and local adaptation using genome-wide association studies in either natural populations or outbred stocks (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Mott and Flint 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Flint and Eskin 2012</a>). In such populations, variants segregate at variable frequencies and linkage disequilibrium is in the order of a few dozen kb (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Yalcin <em>et al.</em> 2010</a>), and this molecular variation needs to be captured. Nowadays, dense SNP maps obtained from genotyping arrays that are commercially available, or from diverse genotyping-by-sequencing techniques such as whole-genome resequencing (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Huang <em>et al.</em> 2009</a>), RAD-seq (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Peterson <em>et al.</em> 2012</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Miller <em>et al.</em> 2007</a>), multiplexed shotgun sequencing (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Andolfatto <em>et al.</em> 2011</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cande <em>et al.</em> 2012</a>), or targeted capture (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Jones and Good 2016</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Linnen <em>et al.</em> 2013</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Olson 2007</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Chevalier <em>et al.</em> 2014</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Hodges <em>et al.</em> 2007</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Gnirke <em>et al.</em> 2009</a>), may yielded thousands to millions of SNPs even in species lacking a reference genome (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Elshire <em>et al.</em> 2011</a>). This structure of molecular variation requires the use of large sample sizes that are in the order of a few thousands in the best case scenario of average minor allele frequencies (<em>e.g.</em>, <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Valdar <em>et al.</em> 2006</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Navarro and Klingenberg 2007</a>). Failure to reach these high sample sizes will undoubtedly result in difficulties with reaching significance and being able to make decisions between noise and signal in associations (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Ledur <em>et al.</em> 2009</a>). These studies may finally come up with mixed results, mapping the main players but accounting only for a low fraction of the total genetic variance, suggesting a large amount of missing heritability (<em>e.g.</em>, <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Pallares <em>et al.</em> 2014</a>).</p><p id="p-6">Whether the imaging modality is 2D or 3D, the process of acquiring landmark coordinates has remained predominantly the same over the last 30 years, <em>i.e.</em>, tedious, manual expert annotation of the anatomy. Moreover, the third dimension presents unique challenges (<em>e.g.</em>, dealing with the accurate projection of 3D structures on a 2D medium like a computer screen and/or artifacts relating to the orthographic or perspective rendering of the specimen), and the amount of time required to access the accuracy of landmarking (<em>i.e.</em>, making sure the selected landmark is actually located on the specimen, not an artifact of the 3D rendering angle) clearly counterbalances the gains by a dramatic cost on the actual feasible sample size. Landmarking in 3D requires 10–60 sec/landmark depending on the expertise and software used (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Bromiley <em>et al.</em> 2014</a>). These numbers are also in the order of time required for the digitization of a complete mandible in 2D. Modern phenomics need large sample sizes to follow high-throughput genomic technologies and questions of modern genetics (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Houle <em>et al.</em> 2010</a>). A variety of alternatives to manual expert annotation exist or are currently under development depending on the field and the imaging modalities used (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Perakis <em>et al.</em> 2014</a>, <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">2010</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Liu <em>et al.</em> 2008</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Aneja <em>et al.</em> 2015</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Guo <em>et al.</em> 2013</a>). For instance, attempts have been made to annotate landmarks on CT scans of new specimens using a machine learning algorithm applied to an initial training set created by experts (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Bromiley <em>et al.</em> 2014</a>), or using registrations of whole surfaces or volumes (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Rolfe <em>et al.</em> 2011</a>), sometimes coupled with multiple atlases, allowing precision comparable to manual editing (<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Young and Maga 2015</a>). Most approaches are actually based on the dense registration of whole surfaces or volumes and therefore encapsulate homologies at the level of the whole structure. In such phenotyping protocols, constraining the registration of the whole structure with some expert annotation improves point correspondence by ensuring homologies of some anatomical features (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">McCane 2013</a>). The semilandmark approach (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Bookstein 1997</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Gunz <em>et al.</em> 2005</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Gunz and Mitteroecker 2013</a>) employs such points on curves and surfaces for which homology conditions are relaxed. These landmarks do not have a one-to-one correspondence but quantify anatomical regions where precise manual annotation is not feasible or possible. The technique uses true landmarks to anchor the homology, and the optimal placement of these points is obtained by sliding them locally on the surface until either the Procrustes distance or the Bending energy is minimized (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Gunz and Mitteroecker 2013</a>).</p><p id="p-7">Here, we want to revisit some of the QTL studies of mandibular shape in mice using landmarks acquired in 3D. We chose the mandible as our structure of interest because it is a well-studied model system, on which several QTL mapping studies of geometric shape have already been conducted in the past in inbred intercrosses (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Klingenberg <em>et al.</em> 2001</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Leamy <em>et al.</em> 2008</a>). Since none of these studies have associated 3D data for their samples, we turned to a new mouse backcross between A/J and C57BL/6J that we recently reported (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Maga <em>et al.</em> 2015</a>). We wanted to reassess mandibular shape genetics using this new dataset and 3D phenotyping, and evaluate the gain of information (if any) from the third dimension, given that mandible flattening seems at first an acceptable compromise for large-scale study. As a secondary goal, we wanted to increase the phenotypic coverage of the mandible using a template of semilandmarks that was tied into the already collected expert landmarks and assess whether any further benefit was obtained from dense phenotypic coverage.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Experimental design and statistical shape analysis</h3><p id="p-8">All aspects of the experimental design, genotyping, the rationale for mapping shape loci using multivariate techniques, and a complete development of the multiple QTL mapping approach used are detailed in an open paper (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Maga <em>et al.</em> 2015</a>). Briefly, skulls of 433 (A/J×C57BL/6J)×A/J 28-day-old individuals were microCT scanned at 18 mm spatial resolution, and genotyped from liver tissue at 882 informative autosomal SNPs using the Illumina medium density linkage panel. After phenotyping and removing six incomplete specimens (see below for specific details relative to the mandible), a full generalized Procrustes analysis (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Dryden and Mardia 1998</a>) was performed on these 3D landmarks using the R/Morpho package (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Schlager 2015a</a>), and then multivariate shape QTL mapping was done using the R/shapeQTL package (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Navarro 2015</a>) of R statistical software (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">R Core Team 2015</a>). All animal protocols were approved by the University of Washington’s Institutional Animal Care and Use Committee.</p></div><div id="sec-3" class="subsection"><h3>3D phenotyping: manual landmarks and simulated 2D phenotyping</h3><p id="p-9">Thirteen 3D landmarks from the right mandible (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) were acquired twice from 3D renderings of original image stacks of the complete skull using 3D Slicer (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Fedorov <em>et al.</em> 2012</a>, <a href="http://www.slicer.org">http://www.slicer.org</a>). The sets were averaged as the best estimate of the landmark location. These landmarks correspond to the classical set of ∼15 landmarks used in previous studies of mouse mandible genetics (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Atchley <em>et al.</em> 1985</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Klingenberg <em>et al.</em> 2004</a>; <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Leamy <em>et al.</em> 2008</a>). We initially acquired more landmarks but we found some gross or systematic errors on several of them, which were consequently removed. On the remaining set of 13, no systematic error between landmarking sessions was found (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-1.gif"/></noscript></span></span>), and the percentage of measurement error was 4.38% and ranged from 2.6–7.55% per landmark.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mandibular 3D surface, landmark, and semilandmark templates. Light blue and gray dots are manual 3D landmarks. Small green dots represent the semilandmark template. The two gray landmarks and the small purple dot were used as the initial reference plane for 2D flattening prior to optimization." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426311044" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mandibular 3D surface, landmark, and semilandmark templates. Light blue and gray dots are manual 3D landmarks. Small green dots represent the semilandmark template. The two gray landmarks and the small purple dot were used as the initial reference plane for 2D flattening prior to optimization.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1153/F1.medium.gif" width="397" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/5/1153/F1.medium.gif" width="397" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468184" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Mandibular 3D surface, landmark, and semilandmark templates. Light blue and gray dots are manual 3D landmarks. Small green dots represent the semilandmark template. The two gray landmarks and the small purple dot were used as the initial reference plane for 2D flattening prior to optimization.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">For a fair comparison with existing results based on 2D imaging, and to better evaluate the gain from the third dimension independently of the gain of denser genotyping, we artificially flattened the mandibular 3D landmarks. To do that, we first aligned each mandible to its main axes, then chose three landmarks to start (dark gray and purple landmarks in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>): the lower point of the angular process, the lower end of symphysis, and an additional landmark (purple landmark) at the top of the inner ridge meeting the molar alveolus. Positions of the first two were refined iteratively based on the innermost vertex of the mesh along the normal to the plane they defined with the third landmark. These three landmarks defined an approximate natural plane on which the mandible would lay. All 13 landmarks were then orthogonally projected according to the normal to this plane.</p></div><div id="sec-4" class="subsection"><h3>3D phenotyping: semilandmarks</h3><p id="p-12">Right mandibles were segmented from the articulated heads. Image voxel resolution was reduced from 18 mm to 36 mm to make further image processing and computations more feasible. We applied a global threshold to remove nonbone material followed by a watershed algorithm to fill any gaps, since watertight meshes are critical for generating semilandmarks on the bone surface. A 3D Gaussian filter with <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-2.gif"/></noscript></span></span> was applied to reduce the noise in voxel data. Along this process, six specimens were identified with incomplete mandible scans and removed from the analysis.</p><p id="p-13">Because the manual landmarks were annotated on the original full-resolution, articulated mouse heads, small differences may exist between the 3D surface and the 3D landmarks due to the image processing pipeline employed. Therefore, we back-projected the averaged landmarks onto the mandibular surface based on the shortest Euclidean distance between the landmark and the 3D mesh. This ensured that the manual landmarks also existed on the hemi-mandible meshes generated for this analysis.</p><p id="p-14">A template of uniformly distributed semilandmarks was generated using poisson-disk sampling on the closest individual to the mean shape of 3D landmarks. As an alternative, targeted templates using curves and patches may have been developed to focus on specific anatomical features like ridges on the surface (<em>e.g.</em>, masseteric ridge) or borders (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Swiderski and Zelditch 2013</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anderson <em>et al.</em> 2014</a>), but our aim was to model the bone surface densely. Points lying on the incisors or molars were also manually identified and removed from the template. In the end, we retained 579 semilandmarks (green landmarks in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) in addition to our 13 expert-annotated landmarks. This template was transferred onto new samples by thin-plate splines based on these 13 landmarks.</p><p id="p-15">Semilandmarks were further slided based on the bending energy and back-projected on the actual mesh surfaces after the sliding relaxation (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Gunz <em>et al.</em> 2005</a>). Both the expert-annotated landmark and the semilandmark data were subjected to two independent full GPA. The described procedure made use of the <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-3.gif"/></noscript></span></span> function from the <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-4.gif"/></noscript></span></span> package (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Schlager 2015b</a>) and the closemeshKD, placePatch, and 3Dslider functions of the <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-5.gif"/></noscript></span></span> package (<a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">Schlager 2015a</a>).</p></div><div id="sec-5" class="subsection"><h3>Shape QTL mapping</h3><p id="p-16">The effect at the locus <em>l</em> was estimated using the multivariate linear model <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-6.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-7.gif"/></noscript></span></span> is the value of the covariate <em>c</em> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-8.gif"/></noscript></span></span> is the probability of the QTL genotypes given the flanking markers <em>M</em> for individual <em>i</em>. These probabilities were computed using <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-9.gif"/></noscript></span></span> (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Broman <em>et al.</em> 2003</a>). The effects β are the <em>q</em>-dimensional effect of the covariate <em>c</em> (<em>i.e.</em>, log of the centroid size, gender and direction-of-cross) or of the genotype <em>j</em> representing the direction and magnitude of the shape change of the overall configuration of landmarks within the shape space.</p><p id="p-17">A forward/backward algorithm was used for multiple QTL model searching. This procedure drops and refines positions of additive QTL without any prior knowledge of their number per chromosome, and compares models based on a penalized LOD score (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Broman and Speed 2002</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Broman and Sen 2009</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Manichaikul <em>et al.</em> 2009</a>), <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-10.gif"/></noscript></span></span>. The forward search was repeated up to a model γ including 50 QTL. The penalty <em>T</em> for each additional QTL was evaluated using the 1000 permutations approach (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Churchill and Doerge 1994</a>). One classic inferential approach in geometric morphometrics uses the sum of the residual sum of squares (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Goodall 1991</a>). Here, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-11.gif"/></noscript></span></span> scores are derived from the Pillai’s trace, a classic multivariate statistic, which makes use of covariances and is proven to be fairly robust in a variety of situations (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Olson 1974</a>, <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">1976</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Tabachnick and Fidell 2013</a>). However, it is important to note that recent approaches to decipher the basis of adaptation or speciation using RAD-seq or similar techniques (see <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Jones and Good 2016</a>, for a review) in nonmodel organisms may have fairly small sample sizes compared to the dimensionality of phenotypes (<em>e.g.</em>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Huber <em>et al.</em> 2015</a>). Such <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-12.gif"/></noscript></span></span> studies might gain in robustness by focusing just on variances, dropping the cost of estimating covariances, as suggested also in other contexts (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams 2014</a>). Our multivariate approach for mapping multiple QTL is very similar to the one developed for the mapping of function-valued traits, with <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-13.gif"/></noscript></span></span> based on the sum of the residual sum of squares (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Kwak <em>et al.</em> 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Gray <em>et al.</em> 2015</a>). Bayes credible intervals of QTL were computed from the <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-14.gif"/></noscript></span></span> profile (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Dupuis and Siegmund 1999</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Sen and Churchill 2001</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Manichaikul <em>et al.</em> 2006</a>).</p><p id="p-18">The magnitude of shape changes associated with each QTL was expressed first in units of Procrustes distance as the norm of the additive vector <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-15.gif"/></noscript></span></span> (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Klingenberg <em>et al.</em> 2001</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Workman <em>et al.</em> 2002</a>). Also, the amount of variation accounted for, given all the other QTL and covariates, was reported as the percentage of total Procrustes variance (%SST in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1/TableS2.pdf">Table S2</a>), a statistic routinely used with multivariate linear models (<em>e.g.</em>, <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Monteiro 1999</a>). We also reported the effect size as a percentage of variance accounted for in the specific direction defined by the additive vector <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-16.gif"/></noscript></span></span> in the shape space (%SS proj Scores in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1/TableS2.pdf">Table S2</a>). For that, we defined a new shape variable, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-17.gif"/></noscript></span></span>, corresponding to the shape variable most associated with the shape changes defined by <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-18.gif"/></noscript></span></span> and containing both the effect and the residuals in that specific direction (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Drake and Klingenberg 2008</a>). The proportion of those projection scores explained by the QTL <em>j</em> is then the ratio of variance between the <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-19.gif"/></noscript></span></span> and the scores <strong>v</strong>. Its expectation with backcross and unlinked QTL is <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-20.gif"/></noscript></span></span>.</p><p id="p-19">QTL-based <strong>G</strong> matrices were compared across the three approaches using only the <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-21.gif"/></noscript></span></span>-coordinates of the 13 manual landmarks that were common across the set of matrices to be compared. Many approaches for matrix similarity have been used in the literature for comparing <strong>G</strong> matrices across populations (<em>e.g.</em>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aguirre <em>et al.</em> 2013</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Teplitsky <em>et al.</em> 2014</a>) or between levels of variation (see for example <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Debat <em>et al.</em> 2009</a>, for contrasting canalization and developmental stability). Overall distance between <strong>G</strong> matrices was assessed with the root Euclidean distance (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Dryden <em>et al.</em> 2009</a>), <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-22.gif"/></noscript></span></span> with the matrix square root <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-23.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-24.gif"/></noscript></span></span> is the spectral decomposition of <strong>G</strong>. Then, the similarity between <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-25.gif"/></noscript></span></span> was measured as their angle, which was compared to <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-26.gif"/></noscript></span></span> pairs of random vectors to assess the significance of whether two <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-27.gif"/></noscript></span></span> were more similar than pairs of random vectors (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Klingenberg and Leamy 2001</a>). To extend this comparison beyond <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-28.gif"/></noscript></span></span> and pairwise comparisons, we computed the common subspace <strong>H</strong> across the three <strong>G</strong> (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Krzanowski 1979</a>). The matrix <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-29.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-30.gif"/></noscript></span></span> corresponds to the first <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-31.gif"/></noscript></span></span> eigenvectors of <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-32.gif"/></noscript></span></span>. We chose <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-33.gif"/></noscript></span></span> based on the cumulative amount of variance accounted for (<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-34.gif"/></noscript></span></span>). Briefly, the spectral decomposition of <strong>H</strong> provided axes maximizing the similarity among matrices <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-35.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-36.gif"/></noscript></span></span> is the angle between an eigenvector and the subspace <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-37.gif"/></noscript></span></span>. An upper bound of Δ is the number of matrices to compare. Angles <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-38.gif"/></noscript></span></span> quantified how different each eigenvector of <strong>H</strong> are from the <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-39.gif"/></noscript></span></span> eigenvector of <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-40.gif"/></noscript></span></span> (<em>e.g.</em>, see <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Aguirre <em>et al.</em> 2013</a> for a more detailed treatment). Finally, we compared the three datasets based on their difference in terms of heritability. With multidimensional traits, heritability is also multidimensional (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Klingenberg 2010</a>), meaning that beyond an overall amount of heritable variance there are also directions across the shape space accounting for a varying degree of heritable variation. The multivariate analog to <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-41.gif"/></noscript></span></span> is <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-42.gif"/></noscript></span></span>, where <sup>−</sup> stands for the Moore-Penrose generalized inverse. Its spectral decomposition <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-43.gif"/></noscript></span></span> provided these directions in the shape space maximizing heritability (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Klingenberg and Leamy 2001</a>).</p><p id="p-20">These shape features <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-44.gif"/></noscript></span></span> as well as any eigenvector of <strong>G</strong>, <strong>P</strong>, or QTL effects <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-45.gif"/></noscript></span></span> might then be amplified and visualized as a colorized 3D surface using thin-plate spline mapping from the mean shape and its mesh model, and by computing the signed distance between these extrapolated meshes. This visualization procedure made use of tps3d and meshDist functions from the <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-46.gif"/></noscript></span></span> package (<a id="xref-ref-88-3" class="xref-bibr" href="#ref-88">Schlager 2015a</a>).</p></div><div id="sec-6" class="subsection"><h3>Comparison with previous mapping data</h3><p id="p-21">Mandible shape QTL have already been assessed in several studies using 2D landmarks on F2 mice from a LG/J×SM/J intercross (<a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Klingenberg <em>et al.</em> 2001</a>, <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">2004</a>) or on F3 mice from the same cross (<a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Leamy <em>et al.</em> 2008</a>). Despite missing confidence intervals for the discovered QTL, the latter was kept for comparison, because it represented a follow-up study on the former two with more markers and more individuals. A genome-wide association study using the first generation of wild-caught mice from a hybrid zone (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Pallares <em>et al.</em> 2014</a>) was also included in comparisons. It differs from the three others by its use of outbred hybrid mice, 3D data, but smaller sample size (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468025/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468025/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Study design and descriptive statistics for confidence interval on QTL positions</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">The closest proximal and distal markers given for the confidence intervals in earlier studies were converted to the current genetic map (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Cox <em>et al.</em> 2009</a>) using the Jackson Laboratory’s Marker Query Tool. The SNPs used in the mapping from F3 of the LG/J×SM/J intercross (<a id="xref-ref-60-6" class="xref-bibr" href="#ref-60">Leamy <em>et al.</em> 2008</a>) were converted to the Cox map using the Mouse Map Converter tool of the Jackson Laboratory using the SNP IDs. Six SNPs were not recovered from their IDs in the conversion. Genomic positions in the NCBI build37 for these SNPs were known from the updated heterogeneous stock data (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Shifman <em>et al.</em> 2006</a>; <a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">Valdar <em>et al.</em> 2006</a>). They were then converted from those NCBI build37 genomic coordinates to the Cox map using the Mouse Map Converter tool. The genomic positions of loci discovered in <a id="xref-ref-79-4" class="xref-bibr" href="#ref-79">Pallares <em>et al.</em> (2014)</a> were converted from the GRCm38 coordinates to the Cox map using the same tool.</p><p id="p-27">Some markers used in LG × SM studies (<a id="xref-ref-55-6" class="xref-bibr" href="#ref-55">Klingenberg <em>et al.</em> 2001</a>, <a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">2004</a>) are now known to be syntenic and do not localize to a specific location in the genome. Overall, three QTL were removed from the comparison, and left or right positions of the confidence interval were imputed with values from the central position for three others. The central positions of two other QTL were treated as missing because the new map positions fell outside the confidence interval, and the closest flanking marker was several dozen cM away. All these imputations led to slightly underestimated descriptive statistics on confidence intervals from previous studies. Such bias is nonetheless against any evidence of gain in the precision of QTL locations with more recent data.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-28">Genotypes and phenotypes are available from <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1/FileS1.zip">File S1</a> as a cross object readable by <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-47.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-48.gif"/></noscript></span></span>.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Mandible shape variation</h3><p id="p-29">Variation from expert-annotated 3D landmarks was dispersed with 22 out of 32 principal components (PC) with a variance higher than 1%, and explaining 94% of the total Procrustes variance. Two first PCs explained about 13% each (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Interestingly, the interpolated shape changes (using thin-plate-spline) associated with PC1 show strong correlated deformations in muscle insertion regions where no landmarks are digitized to actually capture genuine variation in these regions. Shape variation in 2D was dispersed with 18 out of 22 PCs with a variance higher than 1%, explaining 97% of the total Procrustes variance and the first three PCs explaining between 10–14% each. Based only on the <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-49.gif"/></noscript></span></span>-coordinates of the 13 landmarks, major axes of variation are more similar than two random vectors (<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-50.gif"/></noscript></span></span>) once the permutation between PC1 and PC2 on 3D landmarks, which account for almost the same amount of variance, is controlled.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal component (PC) analyses of mandible shape variation. (A) Percentage of variance explained by the first 22 PCs from phenotypic covariance matrices together with the visualization of shape changes associated with PC 1 from 3D landmark and semilandmark datasets. In the visualization, darker colors (red or blue) represent interpolated shape changes diverging from the mean shape. Green color represents unchanged shape of the mandible compared to the mean shape. (B) Percentage of variance explained by the first 22 PCs from G matrices based on discovered QTL (quantitative trait loci)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426311044" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Principal component (PC) analyses of mandible shape variation. (A) Percentage of variance explained by the first 22 PCs from phenotypic covariance matrices together with the visualization of shape changes associated with PC 1 from 3D landmark and semilandmark datasets. In the visualization, darker colors (red or blue) represent interpolated shape changes diverging from the mean shape. Green color represents unchanged shape of the mandible compared to the mean shape. (B) Percentage of variance explained by the first 22 PCs from &lt;strong&gt;G&lt;/strong&gt; matrices based on discovered QTL (quantitative trait loci).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1153/F2.medium.gif" width="348" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/5/1153/F2.medium.gif" width="348" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468294" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-30" class="first-child">Principal component (PC) analyses of mandible shape variation. (A) Percentage of variance explained by the first 22 PCs from phenotypic covariance matrices together with the visualization of shape changes associated with PC 1 from 3D landmark and semilandmark datasets. In the visualization, darker colors (red or blue) represent interpolated shape changes diverging from the mean shape. Green color represents unchanged shape of the mandible compared to the mean shape. (B) Percentage of variance explained by the first 22 PCs from <strong>G</strong> matrices based on discovered QTL (quantitative trait loci).</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Semilandmark variation was most dispersed with 426 nonnull eigenvalues from which 19 PCs, with a percentage of explained variance higher than 1%, explained 79% of the total Procrustes variance, the first two explaining 13% and 14% each. We chose to use the first 71 PCs that accounted for 95% of the total variance. Based only on fixed landmarks, the main axis of phenotypic variation is very similar to PC1 from 2D landmarks (<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-51.gif"/></noscript></span></span>), or to PC2 from the 3D landmarks (<span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-52.gif"/></noscript></span></span>).</p></div><div id="sec-10" class="subsection"><h3>Covariate analysis</h3><p id="p-32">The main effects of centroid size, gender, and directionality of the cross on mandible shape were found to be significant (<em>p</em> &lt; 0.0001) but no significant interactions among them were found. Altogether, they explained 4.5% of the total Procrustes variance. The direction-of-cross was the major effect in this sample, explaining 2.5% of the total Procrustes variance (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1/TableS1.pdf">Table S1</a>). Covariate results with 2D shapes were similar to the 3D data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1/TableS1.pdf">Table S1</a>). Results from the multivariate linear model of the three covariates were similar to those from the 3D manual landmark only, with the covariates explaining 5.5% of the variance altogether.</p></div><div id="sec-11" class="subsection"><h3>2D embedding of shape variation</h3><p id="p-33">3D shapes from manual landmarks were oriented according to the reference 2D plane allowing the decomposition of each effect according to the antero-posterior and bucco-lingual axes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The variation of the 3D landmarks on each PC in the bucco-lingual direction is highly variable (light gray amount in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). However, major axes of variation (PC1 to PC10; ∼73.6% of the total Procrustes variance) are embedded mainly in the antero-posterior/dorso-ventral plane of the mandible (black and dark gray), with very little of this variation in the bucco-lingual direction (4.4%). Accordingly, only 9–13% of covariate effects (size, sex, or direction-of-cross) were along this bucco-lingual axis. Shape variation from the semilandmarks was again mostly embedded in the flat plane with only 5.9% of the variation in the bucco-lingual direction, and covariate effects were mostly within this plane (∼90% of the effect).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentage of shape changes within the -plane (black) and along the z direction (gray). The proportion of shape effects &#x3B2; that lie along the  dimension (m= 1, 2, or 3) is the sum of the squared effects over the k landmarks on the  dimension normalized by the norm of the effect, . The figure may be understood as the proportion of changes that are embedded in the flat plane () or that get out of this plane (z). Such parametrization is sensical here, despite the fact that shapes are invariant to rotation by definition, only because they were specifically oriented according to this specific coordinate system. (A) Principal components (PC), covariate, and QTL (quantitative trait loci) effects for 3D landmarks. (B) Principal components, covariate, and QTL effects for semilandmarks. Covariates are noted CS for the log of the centroid size, X for the direction of the cross, and G for gender." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426311044" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Percentage of shape changes within the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-53&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;-plane (black) and along the &lt;em&gt;z&lt;/em&gt; direction (gray). The proportion of shape effects &lt;em&gt;&#x3B2;&lt;/em&gt; that lie along the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-54&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; dimension (&lt;em&gt;m&lt;/em&gt;= 1, 2, or 3) is the sum of the squared effects over the &lt;em&gt;k&lt;/em&gt; landmarks on the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-55&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; dimension normalized by the norm of the effect, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-56&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The figure may be understood as the proportion of changes that are embedded in the flat plane (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-57&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) or that get out of this plane (&lt;em&gt;z&lt;/em&gt;). Such parametrization is sensical here, despite the fact that shapes are invariant to rotation by definition, only because they were specifically oriented according to this specific coordinate system. (A) Principal components (PC), covariate, and QTL (quantitative trait loci) effects for 3D landmarks. (B) Principal components, covariate, and QTL effects for semilandmarks. Covariates are noted CS for the log of the centroid size, X for the direction of the cross, and G for gender.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1153/F3.medium.gif" width="440" height="166"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/5/1153/F3.medium.gif" width="440" height="166"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468056" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Percentage of shape changes within the <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F3/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F3/embed/mml-math-53.gif"/></noscript></span></span>-plane (black) and along the <em>z</em> direction (gray). The proportion of shape effects <em>β</em> that lie along the <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F3/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F3/embed/mml-math-54.gif"/></noscript></span></span> dimension (<em>m</em>= 1, 2, or 3) is the sum of the squared effects over the <em>k</em> landmarks on the <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F3/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F3/embed/mml-math-55.gif"/></noscript></span></span> dimension normalized by the norm of the effect, <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F3/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F3/embed/mml-math-56.gif"/></noscript></span></span>. The figure may be understood as the proportion of changes that are embedded in the flat plane (<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F3/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F3/embed/mml-math-57.gif"/></noscript></span></span>) or that get out of this plane (<em>z</em>). Such parametrization is sensical here, despite the fact that shapes are invariant to rotation by definition, only because they were specifically oriented according to this specific coordinate system. (A) Principal components (PC), covariate, and QTL (quantitative trait loci) effects for 3D landmarks. (B) Principal components, covariate, and QTL effects for semilandmarks. Covariates are noted CS for the log of the centroid size, X for the direction of the cross, and G for gender.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>QTL mapping of mandible shape</h3><p id="p-35">In all three cases (2D, Manual 3D landmarks, and Semilandmarks), the three covariates (log of the centroid size, gender, and direction-of-cross) were included in the QTL mapping as additive covariates. Multivariate QTL mapping identified between 17 QTL for 2D landmarks, 19 QTL for 3D manual landmarks, and 23 QTL for the 3D semilandmark analysis. All autosomes but two (chromosomes 18 and 19) harbor at least one, and in some cases up to three, QTL (chromosome 11) depending on the phenotyping method used. Summaries of the positions of the discovered QTL (location, nearest marker, and the confidence intervals) are provided in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> for semilandmarks. Summaries for all three cases are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1/TableS2.pdf">Table S2</a> and plotted on <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. The median widths of the confidence intervals were 9 cM for both 3D landmarks and their 2D projections, and 5 cM for the semilandmark dataset, and three quarters of the confidence intervals were smaller than 17.3 cM, 20 cM, and 9.2 cM, respectively, for these three datasets (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Thirteen QTL were replicated across the three approaches, all 2D QTL were replicated in 3D but some were split in two with the semilandmark data or were not captured, eight were only mapped with the semilandmarks, and two only with the 3D landmarks (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1/TableS2.pdf">Table S2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468173/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468173/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Closest SNP, confidence interval, and protein-coding gene content of semilandmark QTL</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL from 2D, 3D, and semilandmark analyses. Results from earlier studies from the LG/J&#xD7;SM/J intercrosses (Klingenberg et al. 2001, 2004; Leamy et al. 2008) and from the Pallares et al. (2014) GWAS (genome-wide association study) are plotted in the gray boxes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426311044" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;QTL from 2D, 3D, and semilandmark analyses. Results from earlier studies from the LG/J&#xD7;SM/J intercrosses (Klingenberg &lt;em&gt;et al.&lt;/em&gt; 2001, 2004; Leamy &lt;em&gt;et al.&lt;/em&gt; 2008) and from the Pallares &lt;em&gt;et al.&lt;/em&gt; (2014) GWAS (genome-wide association study) are plotted in the gray boxes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1153/F4.medium.gif" width="324" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/5/1153/F4.medium.gif" width="324" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468206" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-40" class="first-child">QTL from 2D, 3D, and semilandmark analyses. Results from earlier studies from the LG/J×SM/J intercrosses (<a id="xref-ref-55-7" class="xref-bibr" href="#ref-55">Klingenberg <em>et al.</em> 2001</a>, <a id="xref-ref-56-6" class="xref-bibr" href="#ref-56">2004</a>; <a id="xref-ref-60-7" class="xref-bibr" href="#ref-60">Leamy <em>et al.</em> 2008</a>) and from the <a id="xref-ref-79-5" class="xref-bibr" href="#ref-79">Pallares <em>et al.</em> (2014)</a> GWAS (genome-wide association study) are plotted in the gray boxes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Effect sizes of QTL were small (∼1% of total Procrustes variance; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1/TableS2.pdf">Table S2</a>) but explained on average ∼12% of the variance in the specific direction defined by the QTL effect (regression projection scores). Replicated QTL between 2D and 3D data explained a similar percentage of total Procrustes variance (Wilcoxon test: <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-58.gif"/></noscript></span></span>), but 3D effects explained a significantly greater percentage of variance in the specific direction of the QTL (projection scores) than 2D effects (Wilcoxon test: <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-59.gif"/></noscript></span></span>). According to shape variation and covariate effects, QTL act mainly in the 2D plane (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). However, the contribution of the third dimension may be as high as 30% for some QTL with the manual landmarks or the semilandmarks (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). It is important to note that these QTL were not detected in the 2D analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1/TableS2.pdf">Table S2</a>). Visualization of QTL effects from 3D landmarks only or semilandmarks are provided in supplementary <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1/FigureS2.pdf">Figure S2</a>.</p></div><div id="sec-13" class="subsection"><h3>Comparison of QTL-based G matrices</h3><p id="p-42">Altogether, these QTL accounted for 14.6% (2D), 15.1% (3D), and 16.9% (semilandmarks) of the total Procrustes variance. From those, 12.8% (3D) and 14.4% (semilandmarks) of the genetic variance are along the third dimension. Overall differences among the three <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-60.gif"/></noscript></span></span> matrices measured by the root Euclidean distances <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-61.gif"/></noscript></span></span> are low, the semilandmark matrix appearing as the most different (<span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-62.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-63.gif"/></noscript></span></span>). In agreement with this observation and similarly to <strong>P</strong> matrices, <strong>G</strong> matrices presented a very similar eigenvalue profile among the three approaches (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) with <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-64.gif"/></noscript></span></span> explaining about 20% of the variance. Here, there was also a permutation between the two first PC in the 3D landmarks compared to the two other datasets. Once this permutation was controlled, these three <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-65.gif"/></noscript></span></span> were more similar than random vectors <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-66.gif"/></noscript></span></span> based only on the <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-67.gif"/></noscript></span></span>-coordinates of the 13 fixed landmarks for the two first comparisons and on <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-68.gif"/></noscript></span></span>-coordinates for the last one.</p><p id="p-43">The Krzanowski’s common subspace analysis confirmed these observations. Angles between the first seven eigenvectors of the common subspace <strong>H</strong> and the <em>q</em> PCs accounting for 90% of the variance for each matrices (<span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-69.gif"/></noscript></span></span> for 2D landmarks or 10 for the <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-70.gif"/></noscript></span></span> based on 3D landmarks or semilandmark datasets) were small (δ ranging from 1.6–16°). Their associated eigenvalues were very close to their maximum value of three (Δ ranging from 2.997–2.859). This means that the common subspace may be almost perfectly recovered from linear combinations of the <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-71.gif"/></noscript></span></span> eigenvectors of any of the <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-72.gif"/></noscript></span></span> matrices. The semilandmark <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-73.gif"/></noscript></span></span> matrix appeared again to be the quickest to diverge, underlying the additional information we get from semilandmarks even if they were not explicitly taken into account in the construction of the common subspace.</p><p id="p-44">Multivariate heritabilities are systematically greater with 3D than 2D data (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) but in the same order ranging from 0.55–0.01. Semilandmarks show dimensions with a heritability ranging from 0.76–0.15. The shape changes associated with the most heritable dimension were very different to the one observed on 3D landmarks only (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Such discrepancies seem easily explained as the strong changes appeared to map on mandible surface and ridges where no manual landmarks could be easily captured.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multivariate heritabilities of shape dimensions. The  are the eigenvalues of the  matrix. The shapes correspond to the shape changes associated with the first dimension of  for the 3D landmark or the semilandmarks. For the significance of the color see Figure 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1426311044" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Multivariate heritabilities of shape dimensions. The &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-74&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; are the eigenvalues of the &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-75&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; matrix. The shapes correspond to the shape changes associated with the first dimension of &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-76&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for the 3D landmark or the semilandmarks. For the significance of the color see Figure 2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1153/F5.medium.gif" width="440" height="313"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/5/1153/F5.medium.gif" width="440" height="313"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1153/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468150" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-45" class="first-child">Multivariate heritabilities of shape dimensions. The <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F5/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F5/embed/mml-math-74.gif"/></noscript></span></span> are the eigenvalues of the <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F5/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F5/embed/mml-math-75.gif"/></noscript></span></span> matrix. The shapes correspond to the shape changes associated with the first dimension of <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F5/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/F5/embed/mml-math-76.gif"/></noscript></span></span> for the 3D landmark or the semilandmarks. For the significance of the color see <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-46">Previous studies using QTL mapping of mandible size and shape in mouse have relied typically on 2D landmarks and sparse sampling of the genome using microsatellite markers (<a id="xref-ref-55-8" class="xref-bibr" href="#ref-55">Klingenberg <em>et al.</em> 2001</a>, <a id="xref-ref-56-7" class="xref-bibr" href="#ref-56">2004</a>) or a few hundred SNPs (<a id="xref-ref-60-8" class="xref-bibr" href="#ref-60">Leamy <em>et al.</em> 2008</a>). Our primary purpose in this study was to validate QTL responsible for the variation in mandibular shape observed in mice using 3D phenotyping with denser genotyping than previously attempted, and assess the gains (if any) from the consideration of the third dimension.</p><p id="p-47">We detected marginally fewer QTL than studies based on the LG/J×SM/J intercross (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). This might be expected as we fitted a model with all QTL simultaneously, whereas the LG/J×SM/J intercross studies modeled the genetics of the trait at the chromosome-wide level and used sample sizes about three times bigger. Effects of sample size and/or of allele frequencies seem striking in the study of <a id="xref-ref-79-6" class="xref-bibr" href="#ref-79">Pallares <em>et al.</em> (2014)</a>, where only 10 loci were discovered but with greater precision, thanks to their use of an outbred population and a dense SNP map. Overall, about half of the 23 QTL we discovered had already been reported in the literature. Only one region on chromosome 15 seemed to be replicated across the three different crosses (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). Another GWAS locus on chromosome 16 was replicated in our cross. Two QTL previously detected with imprinting due to maternal genetic effects (<a id="xref-ref-60-9" class="xref-bibr" href="#ref-60">Leamy <em>et al.</em> 2008</a>) were also replicated here. At the time of the submission, a 3D geometric morphometric study using about 700 laboratory outbred mice, 80,000 SNPs, and univariate association mapping of the main principal components was published (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Pallares <em>et al.</em> 2015</a>). They mapped seven loci for mandible shape with great precision and their main finding, which was related to the <em>Mn1</em> gene, seems to be replicated here based on gene content (SH8 on chromosome 5 in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-48">It may initially appear that not much difference exists between the mapping results based on 3D manual landmarks and their 2D projections, due to their significant overlap (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). However, the benefit of adding the third dimension is demonstrated by the reduction in the confidence intervals of the QTL estimates. The median CI estimate for our 2D projection results was 11 cM. Yet, when we conducted the mapping on 3D landmarks, extreme CIs were marginally reduced and median CI was down to 6.3 cM when we used semilandmarks. The difference was even more striking when we considered the total length of the QTL intervals, which dropped by 120 cM, representing a 38% drop in the number of known protein coding genes with 75% of QTL covering at most 146 protein coding genes (358 with 2D and 271 with 3D landmarks). The 2D projection data yielded very similar results to the one of 13.3 cM obtained from the remaining 24 QTL reported in earlier studies (<a id="xref-ref-55-9" class="xref-bibr" href="#ref-55">Klingenberg <em>et al.</em> 2001</a>, <a id="xref-ref-56-8" class="xref-bibr" href="#ref-56">2004</a>) despite a marked difference in molecular marker density (10-fold).</p><p id="p-49">The consideration of the third dimension leads to the discovery, with 3D landmarks, of two additional QTL on chromosome 1 and 14, both QTL covering a few mandible-annotated genes (<em>Disp1</em>, <em>Hhat</em>, <em>Ifr6</em>, and <em>Kat6b</em>). Adding one dimension to each landmark (<em>i.e.</em>, 10 independent dimensions to the shape space) increases detection power. This increase in power is not just related to bigger effect sizes or higher dimensionality, which could be more costly than useful in some cases (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Healy 1969</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Adams 2014</a>). The two additional QTL actually have 26% and 32% of their effects along these additional dimensions. Thus, the specific consideration of the third dimension in the mapping is clearly very informative in these cases. The relaxed constraints on homology of semilandmarks and their ability to densely model the surface of bone led to the discovery of eight additional QTL. Three of them show between 20–30% of their effect on the third dimension. Three of those additional QTL cover one to four mandible-annotated candidate genes: <em>Lef1</em>, <em>Rere</em>, and in the same QTL <em>Akap13</em>, <em>Kif7</em>, <em>Serpinh1</em>, and <em>Folr1</em>. The overall additive genetic variation (<strong>G</strong>) is consistent between 2D and 3D data, both in terms of the amount of total variation captured and the directionality of major variations. Beyond an overall similarity with other data, semilandmarks capture original shape features related to the bone surface, and those features drive the pattern of multivariate heritability (<span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5/1153/embed/mml-math-77.gif"/></noscript></span></span>).</p><p id="p-50">Multivariate approaches have been repeatedly shown to be more powerful than analyzing individual PCs (or univariate traits) independently in various contexts (see for instance in GWAS, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Galesloot <em>et al.</em> 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Gao <em>et al.</em> 2014</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Stephens 2013</a>). However, this univariate PC approach is commonly used, at least for operational reasons (<em>e.g.</em>, <a id="xref-ref-79-7" class="xref-bibr" href="#ref-79">Pallares <em>et al.</em> 2014</a>, <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">2015</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Percival <em>et al.</em> 2016</a>, with geometric morphometric data). An incidental result of our study is a plea for the use of a fully multivariate approach with shape data, not only because shape is a single multidimensional trait (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Klingenberg and Gidaszewski 2010</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Collyer <em>et al.</em> 2015</a>). Major phenotypic PCs appear to have only minor components of variance on the third dimension, whereas some QTL present up to a third of their variation in that specific direction. Prior selection of shape variables based on phenotypic PCA will reduce effect sizes by a third for those QTL, thus reducing power. We have chosen to reduce the dimensionality of the semilandmark data using such a technique. However, we kept most of the total variance (95%) while discarding a large amount of nonnull dimensions (355). Our assumption is that the remaining variance was only random variation. Such noisiness is inherent to the registration process of semilandmarks. This assumption leads us to not reestimate the effects and their associated effect sizes on the complete shape, as doing so will only marginally change our estimates. However, this reestimation of QTL effects on the complete shape may be good practice when the QTL detection is, for some reason, done on a strongly reduced shape space (<a id="xref-ref-79-8" class="xref-bibr" href="#ref-79">Pallares <em>et al.</em> 2014</a>, <a id="xref-ref-80-3" class="xref-bibr" href="#ref-80">2015</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Liu <em>et al.</em> 2014</a>; <a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Zeng <em>et al.</em> 2000</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Langlade <em>et al.</em> 2005</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Mezey <em>et al.</em> 2005</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Franchini <em>et al.</em> 2013</a>). Major QTL may be detected with the first PCs only, but they also have pleiotropic effects on additional dimensions.</p><p id="p-51">In conclusion, the congruence of our results pleads for robustness of our knowledge on the genetic architecture of the mouse mandible, built over a few decades and initially based on 2D imaging techniques. There are many benefits of performing 2D morphometrics in large phenotyping programs, which can be summarized as the simplicity of the techniques and the time saved, but they come at the price of accuracy. Despite inherent difficulties and workloads that may impede the broader use of 3D techniques, information can be gained even from fairly flat structures like the mouse mandible. Once these technical and cost difficulties have been overcome, it appears that making the most of new technologies by opting for denser phenotyping is worth the supplementary technical expertise that is required, and may provide some new insights on the genetics of shape.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">We acknowledge Erika Jessett for her help in image processing, Sarah Park for her help in DNA sample preparation, and we would like to thank two anonymous reviewers for their constructive comments. We acknowledge the following institutions and endowments for sponsoring portions of this research: a National Institutes of Health/National Institute of Dental and Craniofacial Research Pathways to Independence award to A.M.M. (5K99DE021417-02), and Conseil Régional de Bourgogne (CRB) funding to N.N. (CRB PARI Agrale 5 Faber 2014-9201AAO047S01518). </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-9"><p id="p-53">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024372/-/DC1</a></p></li><li class="fn" id="fn-10"><p id="p-54"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-10-30"><span class="received-label">Received </span>October 30, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-02-16"><span class="accepted-label">Accepted </span>February 16, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Navarro, Maga</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.1" data-doi="10.1111/evo.12463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A method for assessing phylogenetic least squares models for shape and other high-dimensional multivariate data.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">2675</span>–<span class="cit-lpage">2688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D68%26rft.spage%253D2675%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12463%26rft_id%253Dinfo%253Apmid%252F24899536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12463&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24899536&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguirre</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hine</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGuigan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blows</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparing G: multivariate analysis of genetic variation in multiple populations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D21%26rft_id%253Dinfo%253Apmid%252F23486079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23486079&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000328751100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.3" data-doi="10.1186/1471-2148-14-85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Renaud</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rayfield</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Adaptive plasticity in the mouse mandible.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D14%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-14-85%26rft_id%253Dinfo%253Apmid%252F24742055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-14-85&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24742055&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.4" data-doi="10.1101/gr.115402.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erezyilmaz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mast</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multiplexed shotgun genotyping for rapid and efficient genetic mapping.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">610</span>–<span class="cit-lpage">617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.115402.110%26rft_id%253Dinfo%253Apmid%252F21233398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzQvNjEwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.5.1153.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aneja</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vora</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camci</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> Automated Detection of 3D Landmarks for the Elimination of Non-Biological Variation in Geometric Morphometric Analyses. 28th IEEE International Symposium on Computer-Based Medical Systems (22–25 June São Carlos, Brazil).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.6" data-doi="10.1111/j.1469-185X.1991.tb01138.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atchley</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">B. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">A model for development and evolution of complex morphological structures.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.stitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.aulast%253DAtchley%26rft.auinit1%253DW.%2BR.%26rft.volume%253D66%26rft.issue%253D2%26rft.spage%253D101%26rft.epage%253D157%26rft.atitle%253DA%2Bmodel%2Bfor%2Bdevelopment%2Band%2Bevolution%2Bof%2Bcomplex%2Bmorphological%2Bstructures.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-185X.1991.tb01138.x%26rft_id%253Dinfo%253Apmid%252F1863686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-185X.1991.tb01138.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1863686&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FQ80000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atchley</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plummer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riska</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Genetics of mandible form in the mouse.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAtchley%26rft.auinit1%253DW.%2BR.%26rft.volume%253D111%26rft.issue%253D3%26rft.spage%253D555%26rft.epage%253D577%26rft.atitle%253DGENETICS%2BOF%2BMANDIBLE%2BFORM%2BIN%2BTHE%2BMOUSE%26rft_id%253Dinfo%253Apmid%252F4054609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTExLzMvNTU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.8" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP Discovery and Genetic Mapping Using Sequenced RAD Markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>–<span class="cit-lpage">e3377</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.9" data-doi="10.1111/ede.12033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boell</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Lines of least resistance and genetic architecture of house mouse (<em>Mus musculus</em>) mandible shape.</span> <abbr class="cit-jnl-abbrev">Evol. Dev.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BDev.%26rft.volume%253D15%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fede.12033%26rft_id%253Dinfo%253Apmid%252F23607303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ede.12033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23607303&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.10" data-doi="10.1186/1471-2148-11-306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boell</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tautz</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Micro-evolutionary divergence patterns of mandible shapes in wild house mouse (<em>Mus musculus</em>) populations.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DBoell%26rft.auinit1%253DL.%26rft.volume%253D11%26rft.spage%253D306%26rft.epage%253D306%26rft.atitle%253DMicro-evolutionary%2Bdivergence%2Bpatterns%2Bof%2Bmandible%2Bshapes%2Bin%2Bwild%2Bhouse%2Bmouse%2B%2528Mus%2Bmusculus%2529%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-11-306%26rft_id%253Dinfo%253Apmid%252F22008647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-11-306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22008647&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.11" data-doi="10.1186/1471-2148-11-309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boell</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregorova</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forejt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tautz</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A comparative assessment of mandible shape in a consomic strain panel of the house mouse (<em>Mus musculus</em>)–implications for epistasis and evolvability of quantitative traits.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DBoell%26rft.auinit1%253DL.%26rft.volume%253D11%26rft.spage%253D309%26rft.epage%253D309%26rft.atitle%253DA%2Bcomparative%2Bassessment%2Bof%2Bmandible%2Bshape%2Bin%2Ba%2Bconsomic%2Bstrain%2Bpanel%2Bof%2Bthe%2Bhouse%2Bmouse%2B%2528Mus%2Bmusculus%2529--implications%2Bfor%2Bepistasis%2Band%2Bevolvability%2Bof%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-11-309%26rft_id%253Dinfo%253Apmid%252F22011306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-11-309&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22011306&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.12" data-doi="10.1007/s00427-013-0443-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boell</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pallares</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brodski</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christian</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Exploring the effects of gene dosage on mandible shape in mice as a model for studying the genetic basis of natural variation.</span> <abbr class="cit-jnl-abbrev">Dev. Genes Evol.</abbr> <span class="cit-vol">223</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BGenes%2BEvol.%26rft.volume%253D223%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00427-013-0443-y%26rft_id%253Dinfo%253Apmid%252F23563729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00427-013-0443-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23563729&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.13" data-doi="10.1016/S1361-8415(97)85012-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bookstein</span>  <span class="cit-name-given-names">F. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Landmark methods for forms without landmarks: morphometrics of group differences in outline shape.</span> <abbr class="cit-jnl-abbrev">Med. Image Anal.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMedical%2Bimage%2Banalysis%26rft.stitle%253DMed%2BImage%2BAnal%26rft.aulast%253DBookstein%26rft.auinit1%253DF.%2BL.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D225%26rft.epage%253D243%26rft.atitle%253DLandmark%2Bmethods%2Bfor%2Bforms%2Bwithout%2Blandmarks%253A%2Bmorphometrics%2Bof%2Bgroup%2Bdifferences%2Bin%2Boutline%2Bshape.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1361-8415%252897%252985012-8%26rft_id%253Dinfo%253Apmid%252F9873908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1361-8415(97)85012-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9873908&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1153.14"><div class="cit-metadata unstructured">Broman, K. W., and S. Sen, 2009 <em>A Guide to QTL Mapping with R/qtl</em>, Springer, New York.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.15" data-doi="10.1111/1467-9868.00354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">T. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A model selection approach for the identification of quantitative trait loci in experimental crosses.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">641</span>–<span class="cit-lpage">656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D64%26rft.spage%253D641%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-9868.00354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.16" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.17" data-doi="10.1186/s12983-014-0061-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bromiley</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schunke</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ragheb</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thacker</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tautz</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Semi-automatic landmark point annotation for geometric morphometrics.</span> <abbr class="cit-jnl-abbrev">Front. Zool.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BZool.%26rft.volume%253D11%26rft.spage%253D61%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12983-014-0061-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12983-014-0061-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.18" data-doi="10.1371/journal.pone.0043888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cande</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prud’homme</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stern</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gompel</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution of multiple additive loci caused divergence between <em>Drosophila yakuba</em> and <em>D. santomea</em> in wing rowing during male courtship.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e43888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DCande%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De43888%26rft.epage%253De43888%26rft.atitle%253DEvolution%2Bof%2Bmultiple%2Badditive%2Bloci%2Bcaused%2Bdivergence%2Bbetween%2BDrosophila%2Byakuba%2Band%2BD.%2Bsantomea%2Bin%2Bwing%2Browing%2Bduring%2Bmale%2Bcourtship.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0043888%26rft_id%253Dinfo%253Apmid%252F22952802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0043888&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22952802&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cardini</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Missing the third dimension in geometric morphometrics: how to assess if 2D images really are a good proxy for 3D structures?</span> <abbr class="cit-jnl-abbrev">Hystrix</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHystrix%26rft.volume%253D25%26rft.spage%253D73%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.20" data-doi="10.1186/1471-2164-15-617"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chevalier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valentim</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LoVerde</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient linkage mapping using exome capture and extreme QTL in schistosome parasites.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D617%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-617%26rft_id%253Dinfo%253Apmid%252F25048426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-617&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25048426&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical threshold values for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.22" data-doi="10.1038/hdy.2014.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collyer</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekora</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A method for analysis of phenotypic change for phenotypes described by high-dimensional data.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.75%26rft_id%253Dinfo%253Apmid%252F25204302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.75&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25204302&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-2" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.23" data-doi="10.1534/genetics.109.105486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackert-Bicknell</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dumont</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A new standard genetic map for the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1335</span>–<span class="cit-lpage">1344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.105486%26rft_id%253Dinfo%253Apmid%252F19535546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi80LzEzMzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.24" data-doi="10.1111/j.1558-5646.2009.00774.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Debat</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Debelle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dworkin</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Plasticity, canalization, and developmental stability of the Drosophila wing: joint effects of mutations and developmental temperature.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">2864</span>–<span class="cit-lpage">2876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DDebat%26rft.auinit1%253DV.%26rft.volume%253D63%26rft.issue%253D11%26rft.spage%253D2864%26rft.epage%253D2876%26rft.atitle%253DPlasticity%252C%2Bcanalization%252C%2Band%2Bdevelopmental%2Bstability%2Bof%2Bthe%2BDrosophila%2Bwing%253A%2Bjoint%2Beffects%2Bof%2Bmutations%2Band%2Bdevelopmental%2Btemperature.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2009.00774.x%26rft_id%253Dinfo%253Apmid%252F19624729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2009.00774.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19624729&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271031900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.25" data-doi="10.1098/rspb.2007.1169"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klingenberg</span>  <span class="cit-name-given-names">C. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The pace of morphological change: historical transformation of skull shape in St Bernard dogs.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DDrake%26rft.auinit1%253DA.%2BG.%26rft.volume%253D275%26rft.issue%253D1630%26rft.spage%253D71%26rft.epage%253D76%26rft.atitle%253DThe%2Bpace%2Bof%2Bmorphological%2Bchange%253A%2Bhistorical%2Btransformation%2Bof%2Bskull%2Bshape%2Bin%2BSt%2BBernard%2Bdogs%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2007.1169%26rft_id%253Dinfo%253Apmid%252F17956847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMToiMjc1LzE2MzAvNzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1153.26"><div class="cit-metadata unstructured">Dryden, I. L., and K. V. Mardia, 1998 <em>Statistical Shape Analysis</em>, Wiley, Hoboken, New Jersey.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.27" data-doi="10.1214/09-AOAS249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dryden</span>  <span class="cit-name-given-names">I. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koloydenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Non-Euclidean statistics for covariance matrices, with applications to diffusion tensor imaging.</span> <abbr class="cit-jnl-abbrev">Ann. Appl. Stat.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1102</span>–<span class="cit-lpage">1123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BAppl.%2BStat.%26rft.volume%253D3%26rft.spage%253D1102%26rft_id%253Dinfo%253Adoi%252F10.1214%252F09-AOAS249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/09-AOAS249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dupuis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegmund</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Statistical methods for mapping quantitative trait loci from a dense set of markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDupuis%26rft.auinit1%253DJ.%26rft.volume%253D151%26rft.issue%253D1%26rft.spage%253D373%26rft.epage%253D386%26rft.atitle%253DStatistical%2BMethods%2Bfor%2BMapping%2BQuantitative%2BTrait%2BLoci%2BFrom%2Ba%2BDense%2BSet%2Bof%2BMarkers%26rft_id%253Dinfo%253Apmid%252F9872974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUxLzEvMzczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.29" data-doi="10.1534/genetics.110.125922"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dworkin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Idaghdour</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The effects of weak genetic perturbations on the transcriptome of the wing imaginal disc and its association with wing shape in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">1171</span>–<span class="cit-lpage">1184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.125922%26rft_id%253Dinfo%253Apmid%252F21288875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ny80LzExNzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.30" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emília Santos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Refki</span>  <span class="cit-name-given-names">P. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khila</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Integrating evo-devo with ecology for a better understanding of phenotypic evolution.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">2015</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft.volume%253D2015%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.32" data-doi="10.1016/j.mri.2012.05.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fedorov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beichel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalpathy-Cramer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fillion-Robin</span>  <span class="cit-name-given-names">J.-C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">3D Slicer as an image computing platform for the quantitative imaging network.</span> <abbr class="cit-jnl-abbrev">Magn. Reson. Imaging</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1323</span>–<span class="cit-lpage">1341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMagnetic%2Bresonance%2Bimaging%26rft.stitle%253DMagn%2BReson%2BImaging%26rft.aulast%253DFedorov%26rft.auinit1%253DA.%26rft.volume%253D30%26rft.issue%253D9%26rft.spage%253D1323%26rft.epage%253D1341%26rft.atitle%253D3D%2BSlicer%2Bas%2Ban%2Bimage%2Bcomputing%2Bplatform%2Bfor%2Bthe%2BQuantitative%2BImaging%2BNetwork.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mri.2012.05.001%26rft_id%253Dinfo%253Apmid%252F22770690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mri.2012.05.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22770690&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309946000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.33" data-doi="10.1038/nrg3335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association studies in mice.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">807</span>–<span class="cit-lpage">817</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFlint%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D807%26rft.epage%253D817%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3335%26rft_id%253Dinfo%253Apmid%252F23044826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23044826&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Franchini</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fruciano</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spreitzer</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elmer</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic architecture of ecologically divergent body shape in a pair of sympatric crater lake cichlid fishes.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1828</span>–<span class="cit-lpage">1845</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D1828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.35" data-doi="10.1371/journal.pone.0095923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galesloot</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Steen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiemeney</span>  <span class="cit-name-given-names">L. A. L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vermeulen</span>  <span class="cit-name-given-names">S. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A Comparison of multivariate genome-wide association methods.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e95923</span>–<span class="cit-lpage">e95928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De95923%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0095923%26rft_id%253Dinfo%253Apmid%252F24763738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0095923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24763738&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.36" data-doi="10.1038/hdy.2014.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multiple-trait genome-wide association study based on principal component analysis for residual covariance matrix.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">526</span>–<span class="cit-lpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D113%26rft.spage%253D526%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.57%26rft_id%253Dinfo%253Apmid%252F24984606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.57&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24984606&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.37" data-doi="10.1038/nbt.1523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnirke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melnikov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogov</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LeProust</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGnirke%26rft.auinit1%253DA.%26rft.volume%253D27%26rft.issue%253D2%26rft.spage%253D182%26rft.epage%253D189%26rft.atitle%253DSolution%2Bhybrid%2Bselection%2Bwith%2Bultra-long%2Boligonucleotides%2Bfor%2Bmassively%2Bparallel%2Btargeted%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1523%26rft_id%253Dinfo%253Apmid%252F19182786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19182786&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263209000025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodall</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Procrustes methods in the statistical analysis of shape.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D53%26rft.spage%253D285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.39" data-doi="10.1534/genetics.115.177790"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parmenter</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hogan</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ford</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuthbert</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetics of rapid and extreme size evolution in island mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177790%26rft_id%253Dinfo%253Apmid%252F26199233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzEvMjEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gunz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitteroecker</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Semilandmarks: a method for quantifying curves and surfaces.</span> <abbr class="cit-jnl-abbrev">Hystrix</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHystrix%26rft.volume%253D24%26rft.spage%253D103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1153.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gunz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitteroecker</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bookstein</span>  <span class="cit-name-given-names">F. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Semilandmarks in three dimensions</span>, pp. <span class="cit-fpage">73</span>–<span class="cit-lpage">98</span> in <span class="cit-source">Modern Morphometrics in Physical Anthropology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Slice</span>  <span class="cit-name-given-names">D. E.</span></span> <span class="cit-publ-name">Kluwer Academic/Plenum Publishers</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.42" data-doi="10.1186/1471-2105-14-232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mei</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Automatic landmark annotation and dense correspondence registration for 3D human facial images.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D14%26rft.spage%253D232%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-232%26rft_id%253Dinfo%253Apmid%252F23870191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23870191&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.43" data-doi="10.1371/journal.pgen.1004724"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallgrímsson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mio</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcucio</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spritz</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Let’s face it–complex traits are just not that simple.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004724%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004724%26rft_id%253Dinfo%253Apmid%252F25375250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004724&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25375250&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.44" data-doi="10.2307/2528798"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Healy</span>  <span class="cit-name-given-names">M. J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Rao’s paradox concerning multivariate tests of significance.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D25%26rft.spage%253D411%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2528798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2528798&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.45" data-doi="10.1038/ng.2007.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodges</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xuan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balija</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molla</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide in situ exon capture for selective resequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1522</span>–<span class="cit-lpage">1527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHodges%26rft.auinit1%253DE.%26rft.volume%253D39%26rft.issue%253D12%26rft.spage%253D1522%26rft.epage%253D1527%26rft.atitle%253DGenome-wide%2Bin%2Bsitu%2Bexon%2Bcapture%2Bfor%2Bselective%2Bresequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2007.42%26rft_id%253Dinfo%253Apmid%252F17982454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2007.42&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17982454&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251267400023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.46" data-doi="10.1038/nrg2897"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Govindaraju</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Omholt</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Phenomics: the next challenge.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">855</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHoule%26rft.auinit1%253DD.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D855%26rft.epage%253D866%26rft.atitle%253DPhenomics%253A%2Bthe%2Bnext%2Bchallenge.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2897%26rft_id%253Dinfo%253Apmid%252F21085204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2897&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085204&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284359000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.47" data-doi="10.1101/gr.089516.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-throughput genotyping by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089516.108%26rft_id%253Dinfo%253Apmid%252F19420380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA2OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.48" data-doi="10.1038/hdy.2015.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whibley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poul</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Conservatism and novelty in the genetic architecture of adaptation in Heliconius butterflies.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D515%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.22%26rft_id%253Dinfo%253Apmid%252F25806542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25806542&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.49" data-doi="10.1111/mec.13304"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Targeted capture in evolutionary and ecological genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D185%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13304&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.50" data-doi="10.1371/journal.pgen.1000583"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khila</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abouheif</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowe</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution of a novel appendage ground plan in water striders is driven by changes in the Hox gene Ultrabithorax.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000583</span>–<span class="cit-lpage">e1000589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKhila%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000583%26rft.epage%253De1000583%26rft.atitle%253DEvolution%2Bof%2Ba%2Bnovel%2Bappendage%2Bground%2Bplan%2Bin%2Bwater%2Bstriders%2Bis%2Bdriven%2Bby%2Bchanges%2Bin%2Bthe%2BHox%2Bgene%2BUltrabithorax.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000583%26rft_id%253Dinfo%253Apmid%252F19649305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000583&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19649305&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.51" data-doi="10.1038/nrg2829"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klingenberg</span>  <span class="cit-name-given-names">C. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evolution and development of shape: integrating quantitative approaches.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DKlingenberg%26rft.auinit1%253DC.%2BP.%26rft.volume%253D11%26rft.issue%253D9%26rft.spage%253D623%26rft.epage%253D635%26rft.atitle%253DEvolution%2Band%2Bdevelopment%2Bof%2Bshape%253A%2Bintegrating%2Bquantitative%2Bapproaches.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2829%26rft_id%253Dinfo%253Apmid%252F20697423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2829&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20697423&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281115600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.52" data-doi="10.1093/sysbio/syp106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klingenberg</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gidaszewski</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Testing and quantifying phylogenetic signals and homoplasy in morphometric data.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyp106%26rft_id%253Dinfo%253Apmid%252F20525633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjU5LzMvMjQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.53" data-doi="10.1554/0014-3820(2001)055[2342:QGOGSI]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klingenberg</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leamy</span>  <span class="cit-name-given-names">L. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Quantitative genetics of geometric shape in the mouse mandible.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">2342</span>–<span class="cit-lpage">2352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DKlingenberg%26rft.auinit1%253DC.%2BP.%26rft.volume%253D55%26rft.issue%253D11%26rft.spage%253D2342%26rft.epage%253D2352%26rft.atitle%253DQuantitative%2Bgenetics%2Bof%2Bgeometric%2Bshape%2Bin%2Bthe%2Bmouse%2Bmandible.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282001%2529055%255B2342%253AQGOGSI%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F11794792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2001)055[2342:QGOGSI]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11794792&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172947000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1153.54"><div class="cit-metadata unstructured">Klingenberg, C. P., and N. Navarro, 2012 Development of the mouse mandible: a model system for complex morphological structures, pp. 135–149 in <em>Evolution of the House Mouse</em>, edited by Macholán, M., S. J. E. Baird, and J. Pialek. Cambridge University Press, Cambridge.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klingenberg</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leamy</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Routman</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic architecture of mandible shape in mice: effects of quantitative trait loci analyzed by geometric morphometrics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKlingenberg%26rft.auinit1%253DC.%2BP.%26rft.volume%253D157%26rft.issue%253D2%26rft.spage%253D785%26rft.epage%253D802%26rft.atitle%253DGenetic%2BArchitecture%2Bof%2BMandible%2BShape%2Bin%2BMice%253A%2BEffects%2Bof%2BQuantitative%2BTrait%2BLoci%2BAnalyzed%2Bby%2BGeometric%2BMorphometrics%26rft_id%253Dinfo%253Apmid%252F11156997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU3LzIvNzg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.56" data-doi="10.1534/genetics.166.4.1909"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klingenberg</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leamy</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Integration and modularity of quantitative trait locus effects on geometric shape in the mouse mandible.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1909</span>–<span class="cit-lpage">1921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKlingenberg%26rft.auinit1%253DC.%2BP.%26rft.volume%253D166%26rft.issue%253D4%26rft.spage%253D1909%26rft.epage%253D1921%26rft.atitle%253DIntegration%2Band%2BModularity%2Bof%2BQuantitative%2BTrait%2BLocus%2BEffects%2Bon%2BGeometric%2BShape%2Bin%2Bthe%2BMouse%2BMandible%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.4.1909%26rft_id%253Dinfo%253Apmid%252F15126408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ni80LzE5MDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.57" data-doi="10.2307/2286995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krzanowski</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Between-Groups Comparison of Principal Components.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">703</span>–<span class="cit-lpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D74%26rft.spage%253D703%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2286995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2286995&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979HS22400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.58" data-doi="10.1534/genetics.114.166306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kwak</span>  <span class="cit-name-given-names">I.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spalding</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A simple regression-based method to map quantitative trait loci underlying function-valued phenotypes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">1409</span>–<span class="cit-lpage">1416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.166306%26rft_id%253Dinfo%253Apmid%252F24931408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny80LzE0MDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.59" data-doi="10.1073/pnas.0504210102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langlade</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dransfield</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copsey</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evolution through genetically controlled allometry space.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">10221</span>–<span class="cit-lpage">10226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0504210102%26rft_id%253Dinfo%253Apmid%252F16009935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzI5LzEwMjIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.60" data-doi="10.1038/hdy.2008.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leamy</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klingenberg</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherratt</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A search for quantitative trait loci exhibiting imprinting effects on mouse mandible size and shape.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">518</span>–<span class="cit-lpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DLeamy%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D101%26rft.issue%253D6%26rft.spage%253D518%26rft.epage%253D526%26rft.atitle%253DA%2Bsearch%2Bfor%2Bquantitative%2Btrait%2Bloci%2Bexhibiting%2Bimprinting%2Beffects%2Bon%2Bmouse%2Bmandible%2Bsize%2Band%2Bshape.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2008.79%26rft_id%253Dinfo%253Apmid%252F18685568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2008.79&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18685568&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260993500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ledur</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Enciso</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Large-scale SNP genotyping in crosses between outbred lines: how useful is it?</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D105%26rft.spage%253D173%26rft_id%253Dinfo%253Apmid%252F19844266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19844266&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.62" data-doi="10.1126/science.1233213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linnen</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poh</span>  <span class="cit-name-given-names">Y.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">R. D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Adaptive evolution of multiple traits through multiple mutations at a single gene.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLinnen%26rft.auinit1%253DC.%2BR.%26rft.volume%253D339%26rft.issue%253D6125%26rft.spage%253D1312%26rft.epage%253D1316%26rft.atitle%253DAdaptive%2BEvolution%2Bof%2BMultiple%2BTraits%2BThrough%2BMultiple%2BMutations%2Bat%2Ba%2BSingle%2BGene%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1233213%26rft_id%253Dinfo%253Apmid%252F23493712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzkvNjEyNS8xMzEyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.63" data-doi="10.1109/TMI.2008.915684"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nowinski</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A Model-Based, Semi-Global Segmentation Approach for Automatic 3-D Point Landmark Localization in Neuroimages.</span> <abbr class="cit-jnl-abbrev">IEEE Trans. Med. Imaging</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1034</span>–<span class="cit-lpage">1044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2Btransactions%2Bon%2Bmedical%2Bimaging%26rft.stitle%253DIEEE%2BTrans%2BMed%2BImaging%26rft.aulast%253DLiu%26rft.auinit1%253DJ.%26rft.volume%253D27%26rft.issue%253D8%26rft.spage%253D1034%26rft.epage%253D1044%26rft.atitle%253DA%2Bmodel-based%252C%2Bsemi-global%2Bsegmentation%2Bapproach%2Bfor%2Bautomatic%2B3-D%2Bpoint%2Blandmark%2Blocalization%2Bin%2Bneuroimages.%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTMI.2008.915684%26rft_id%253Dinfo%253Apmid%252F18672421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TMI.2008.915684&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18672421&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.64" data-doi="10.1534/g3.114.010389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shikano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leinonen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cano</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.-H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identification of major and minor QTL for ecologically important morphological traits in three-spined sticklebacks (<em>Gasterosteus aculeatus</em>).</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.010389%26rft_id%253Dinfo%253Apmid%252F24531726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvNC81OTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.65" data-doi="10.3389/fphys.2015.00001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maga</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cunningham</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Quantitative trait loci affecting the 3D skull shape and size in mouse and prioritization of candidate genes in-silico.</span> <abbr class="cit-jnl-abbrev">Front. Physiol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPhysiol.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffphys.2015.00001%26rft_id%253Dinfo%253Apmid%252F25688210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fphys.2015.00001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25688210&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.66" data-doi="10.1534/genetics.106.061549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manichaikul</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupuis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Poor performance of bootstrap confidence intervals for the location of a quantitative trait locus.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061549%26rft_id%253Dinfo%253Apmid%252F16783000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc0LzEvNDgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.67" data-doi="10.1534/genetics.108.094565"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manichaikul</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moon</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A model selection approach for the identification of quantitative trait loci in experimental crosses, allowing epistasis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.094565%26rft_id%253Dinfo%253Apmid%252F19104078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MS8zLzEwNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.68" data-doi="10.1093/sysbio/sys080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCane</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Shape variation in outline shapes.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">134</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsys080%26rft_id%253Dinfo%253Apmid%252F22993142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjYyLzEvMTM0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.69" data-doi="10.1534/genetics.104.036988"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mezey</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nuzhdin</span>  <span class="cit-name-given-names">S. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Naturally segregating quantitative trait loci affecting wing shape of Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">2101</span>–<span class="cit-lpage">2113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036988%26rft_id%253Dinfo%253Apmid%252F15520257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzIxMDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.70" data-doi="10.1101/gr.5681207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">240</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5681207%26rft_id%253Dinfo%253Apmid%252F17189378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzIvMjQwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.71" data-doi="10.1080/106351599260526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monteiro</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Multivariate regression models and geometric morphometrics: the search for causal factors in the analysis of shape.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F106351599260526%26rft_id%253Dinfo%253Apmid%252F12078640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo2OiJzeXNiaW8iO3M6NToicmVzaWQiO3M6ODoiNDgvMS8xOTIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.72" data-doi="10.1146/annurev-genom-091212-153419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dissecting quantitative traits in mice.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenomics%2BHum.%2BGenet.%26rft.volume%253D14%26rft.spage%253D421%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-091212-153419%26rft_id%253Dinfo%253Apmid%252F23834320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-091212-153419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23834320&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326658500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.73" data-doi="10.5735/086.047.0105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz-Muñoz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perpiñan</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Measurement error in morphometric studies: comparison between manual and computerized methods.</span> <abbr class="cit-jnl-abbrev">Ann. Zool. Fenn.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BZool.%2BFenn.%26rft.volume%253D47%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.5735%252F086.047.0105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5735/086.047.0105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-web" id="cit-6.5.1153.74"><div class="cit-metadata unstructured">Navarro, N., 2015 shapeQTL: shape QTL mapping experiment with R. R package version 0.2. <a href="https://github.com/nnavarro/shapeQTL">https://github.com/nnavarro/shapeQTL</a>. Accessed: January 28, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1153.75"><div class="cit-metadata unstructured">Navarro, N., and C. P. Klingenberg, 2007 Mapping multiple QTLs of geometric shape of the mouse mandible, pp. 125–128 in <em>Systems Biology Statistical Bioinformatics</em>, edited by Barber, S., P. D. Baxter, and K. V. Mardia, Leeds University Press, Leeds.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.76" data-doi="10.2307/2286159"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Comparative Robustness of Six Tests in Multivariate Analysis of Variance.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">894</span>–<span class="cit-lpage">908</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D69%26rft.spage%253D894%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2286159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2286159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1974V289200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.77" data-doi="10.1037//0033-2909.83.4.579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">On choosing a test statistic in multivariate analysis of variance.</span> <abbr class="cit-jnl-abbrev">Psychol. Bull.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPsychol.%2BBull.%26rft.volume%253D83%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1037%252F%252F0033-2909.83.4.579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1037//0033-2909.83.4.579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976BY35200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.78" data-doi="10.1038/nmeth1107-891"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Enrichment of super-sized resequencing targets from the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DOlson%26rft.auinit1%253DM.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D891%26rft.epage%253D892%26rft.atitle%253DEnrichment%2Bof%2Bsuper-sized%2Bresequencing%2Btargets%2Bfrom%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth1107-891%26rft_id%253Dinfo%253Apmid%252F17971778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth1107-891&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17971778&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250575700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.79" data-doi="10.1111/mec.12968"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pallares</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harr</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tautz</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Use of a natural hybrid zone for genomewide association mapping of craniofacial traits in the house mouse.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">5756</span>–<span class="cit-lpage">5770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D5756%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12968&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.80" data-doi="10.1371/journal.pgen.1005607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pallares</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbonetto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gopalakrishnan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackert-Bicknell</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Mapping of craniofacial traits in outbred mice identifies major developmental genes involved in shape determination.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005607%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005607%26rft_id%253Dinfo%253Apmid%252F26523602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005607&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26523602&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1153.81"><div class="cit-metadata unstructured">Perakis, P., G. Passalis, T. Theoharis, and I. A. Kakadiaris, 2010 3D facial landmark detection &amp; face registration, Tech. Rep., University of Athens, Greece.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.82" data-doi="10.1016/j.patcog.2014.03.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perakis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theoharis</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kakadiaris</span>  <span class="cit-name-given-names">I. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Feature fusion for facial landmark detection.</span> <abbr class="cit-jnl-abbrev">Pattern Recognit.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">2783</span>–<span class="cit-lpage">2793</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPattern%2BRecognit.%26rft.volume%253D47%26rft.spage%253D2783%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.patcog.2014.03.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.patcog.2014.03.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.83" data-doi="10.1111/joa.12382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Percival</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liberton</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel de Villena</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spritz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcucio</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetics of murine craniofacial morphology: diallel analysis of the eight founders of the Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">J. Anat.</abbr> <span class="cit-vol">228</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnat.%26rft.volume%253D228%26rft.spage%253D96%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjoa.12382%26rft_id%253Dinfo%253Apmid%252F26426826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/joa.12382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26426826&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.84" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prud’homme</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minervino</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hocine</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cande</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aouane</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Body plan innovation in treehoppers through the evolution of an extra wing-like appendage.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">473</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D473%26rft.spage%253D83%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-other" id="cit-6.5.1153.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2015</span> R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. URL <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rolfe</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maga</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A landmark-free framework for the detection and description of shape differences in embryos.</span> <abbr class="cit-jnl-abbrev">Conf. Proc. IEEE Eng. Med. Biol. Soc.</abbr> <span class="cit-vol">2011</span>: <span class="cit-fpage">5153</span>–<span class="cit-lpage">5156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DConf%2BProc%2BIEEE%2BEng%2BMed%2BBiol%2BSoc%26rft.aulast%253DRolfe%26rft.auinit1%253DS.%2BM.%26rft.volume%253D2011%26rft.spage%253D5153%26rft.epage%253D5156%26rft.atitle%253DA%2Blandmark-free%2Bframework%2Bfor%2Bthe%2Bdetection%2Band%2Bdescription%2Bof%2Bshape%2Bdifferences%2Bin%2Bembryos.%26rft_id%253Dinfo%253Apmid%252F22255499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22255499&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-other" id="cit-6.5.1153.88"><div class="cit-metadata unstructured">Schlager, S., 2015a Morpho: Calculations and visualisations related to Geometric Morphometrics. R-package version 2. 3.0. Available at: <a href="https://github.com/zarquon42b/Morpho">https://github.com/zarquon42b/Morpho</a>. Accessed: September 10, 2015</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-other" id="cit-6.5.1153.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlager</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> Rvcg: Manipulations of Triangular Meshes Based on the ’VCGLIB’ API. Available at: <a href="https://github.com/zarquon42b/Rvcg">https://github.com/zarquon42b/Rvcg</a>. Accessed: September 10, 2015</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.90"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A statistical framework for quantitative trait mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSen%26rft.auinit1%253DS.%26rft.volume%253D159%26rft.issue%253D1%26rft.spage%253D371%26rft.epage%253D387%26rft.atitle%253DA%2BStatistical%2BFramework%2Bfor%2BQuantitative%2BTrait%2BMapping%26rft_id%253Dinfo%253Apmid%252F11560912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzEvMzcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.91" data-doi="10.1371/journal.pbio.0040395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shifman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copley</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A high-resolution single nucleotide polymorphism genetic map of the mouse genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DShifman%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De395%26rft.epage%253De395%26rft.atitle%253DA%2Bhigh-resolution%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bgenetic%2Bmap%2Bof%2Bthe%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040395%26rft_id%253Dinfo%253Apmid%252F17105354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17105354&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.92" data-doi="10.1371/journal.pone.0065245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A unified framework for association analysis with multiple related phenotypes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e65245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De65245%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0065245%26rft_id%253Dinfo%253Apmid%252F23861737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0065245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23861737&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.93" data-doi="10.1007/s00335-008-9154-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suto</span>  <span class="cit-name-given-names">J.-I.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of multiple quantitative trait loci affecting the size and shape of the mandible in mice.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DSuto%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D13%26rft.atitle%253DIdentification%2Bof%2Bmultiple%2Bquantitative%2Btrait%2Bloci%2Baffecting%2Bthe%2Bsize%2Band%2Bshape%2Bof%2Bthe%2Bmandible%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-008-9154-5%26rft_id%253Dinfo%253Apmid%252F19067046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-008-9154-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19067046&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.94" data-doi="10.1111/joa.12118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swiderski</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zelditch</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The complex ontogenetic trajectory of mandibular shape in a laboratory mouse.</span> <abbr class="cit-jnl-abbrev">J. Anat.</abbr> <span class="cit-vol">223</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnat.%26rft.volume%253D223%26rft.spage%253D568%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjoa.12118%26rft_id%253Dinfo%253Apmid%252F24111948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/joa.12118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24111948&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1153.95"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tabachnick</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fidell</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">Using Multivariate Statistics</span>, <span class="cit-edition">Ed. 6th</span>. <span class="cit-publ-name">Pearson Education, Inc</span>, <span class="cit-publ-loc">Boston</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-book" id="cit-6.5.1153.96"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teplitsky</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merilä</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evolutionary potential and constraints in wild populations</span>, pp. <span class="cit-fpage">190</span>–<span class="cit-lpage">208</span> in <span class="cit-source">Quantitative Genetics in the Wild</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Charmantier</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Garant</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Kruuk</span>  <span class="cit-name-given-names">L. E. B.</span></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.97" data-doi="10.1038/ng1840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solberg</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gauguier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burnett</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klenerman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide genetic association of complex traits in heterogeneous stock mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DValdar%26rft.auinit1%253DW.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D879%26rft.epage%253D887%26rft.atitle%253DGenome-wide%2Bgenetic%2Bassociation%2Bof%2Bcomplex%2Btraits%2Bin%2Bheterogeneous%2Bstock%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1840%26rft_id%253Dinfo%253Apmid%252F16832355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16832355&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.98"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Workman</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leamy</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Routman</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Analysis of quantitative trait locus effects on the size and shape of mandibular molars in mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1573</span>–<span class="cit-lpage">1586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWorkman%26rft.auinit1%253DM.%2BS.%26rft.volume%253D160%26rft.issue%253D4%26rft.spage%253D1573%26rft.epage%253D1586%26rft.atitle%253DAnalysis%2Bof%2BQuantitative%2BTrait%2BLocus%2BEffects%2Bon%2Bthe%2BSize%2Band%2BShape%2Bof%2BMandibular%2BMolars%2Bin%2BMice%26rft_id%253Dinfo%253Apmid%252F11973311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MC80LzE1NzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.99" data-doi="10.1371/journal.pgen.1001085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicod</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhomra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Commercially available outbred mice for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001085</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001085%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001085%26rft_id%253Dinfo%253Apmid%252F20838427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838427&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.100" data-doi="10.1093/sysbio/41.4.471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yezerinac</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lougheed</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handford</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Measurement error and morphometric studies: statistical power and observer experience.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252F41.4.471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjQxLzQvNDcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.101" data-doi="10.1186/s12983-015-0127-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maga</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Performance of single and multi-atlas based automated landmarking methods compared to expert annotations in volumetric microCT datasets of mouse mandibles.</span> <abbr class="cit-jnl-abbrev">Front. Zool.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BZool.%26rft.volume%253D12%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12983-015-0127-8%26rft_id%253Dinfo%253Apmid%252F26628903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12983-015-0127-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26628903&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-6.5.1153.102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercer</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genetic architecture of a morphological shape difference between two Drosophila species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DZeng%26rft.auinit1%253DZ.-B.%26rft.volume%253D154%26rft.issue%253D1%26rft.spage%253D299%26rft.epage%253D310%26rft.atitle%253DGenetic%2BArchitecture%2Bof%2Ba%2BMorphological%2BShape%2BDifference%2BBetween%2BTwo%2BDrosophila%2BSpecies%26rft_id%253Dinfo%253Apmid%252F10628989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU0LzEvMjk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/5/1153.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/5/1141" title="The Genetic Basis of Baculum Size and Shape Variation in Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/5/1165" title="Genomic Bayesian Prediction Model for Count Data with Genotype × Environment Interaction" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/5"><span>Volume 6 Issue 5, May 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/5.cover-source.jpg?itok=8SH0hKOw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1153?cited-by=yes&amp;legid=ggg;6/5/1153" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-glruuwBtVnF_0RiwoL2CmUS2Sy33hYkgCX5InJyombc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/5/1153" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/5/1153" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/5/1153" class="active" data-icon-position="" data-hide-link-title="0">Does 3D Phenotyping Yield Substantial Insights in the Genetics of the Mouse Mandible Shape?</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460361" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Kk2jaJrfzig5jRN-bI20hffgL8cUBmeCad4ndJJo7Ho" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/5/1153.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/5/1153" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-WQUNVqY16ts-SJ95GFb0DhjIlEV01bPwl6gO-R7CqVA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/5/1153" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/5/1153.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460361" id="node460361--2" data-pisa="ggg;6/5/1153" data-pisa-master="ggg;g3.115.024372" data-apath="/ggg/6/5/1153.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Does 3D Phenotyping Yield Substantial Insights in the Genetics of the Mouse Mandible Shape?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5694-4201" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicolas Navarro</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7921-9018" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>A. Murat Maga</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1153-1163;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024372 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460361--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicolas Navarro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biogéosciences, UMR CNRS 6282, Univ Bourgogne Franche-Comté, EPHE, PSL Research University, F-21000 Dijon, France </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BNavarro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Navarro%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicolas%2BNavarro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5694-4201" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicolas Navarro</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nicolas.navarro@ephe.sorbonne.fr" class="" data-icon-position="" data-hide-link-title="0">nicolas.navarro@ephe.sorbonne.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">A. Murat Maga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Craniofacial Medicine, Department of Pediatrics, University of Washington, Seattle, Washington 98105 </div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=A.%2BMurat%2BMaga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maga%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AA.%2BMurat%2BMaga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7921-9018" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for A. Murat Maga</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/5/1153" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/5/1153&amp;title=Does%203D%20Phenotyping%20Yield%20Substantial%20Insights%20in%20the%20Genetics%20of%20the%20Mouse%20Mandible%20Shape%3F" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/5/1153&amp;title=Does%203D%20Phenotyping%20Yield%20Substantial%20Insights%20in%20the%20Genetics%20of%20the%20Mouse%20Mandible%20Shape%3F" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/5/1153&amp;title=Does%203D%20Phenotyping%20Yield%20Substantial%20Insights%20in%20the%20Genetics%20of%20the%20Mouse%20Mandible%20Shape%3F" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/5/1153&amp;text=Does%203D%20Phenotyping%20Yield%20Substantial%20Insights%20in%20the%20Genetics%20of%20the%20Mouse%20Mandible%20Shape%3F" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/5/1153&amp;title=Does%203D%20Phenotyping%20Yield%20Substantial%20Insights%20in%20the%20Genetics%20of%20the%20Mouse%20Mandible%20Shape%3F" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/5/1153&amp;t=Does%203D%20Phenotyping%20Yield%20Substantial%20Insights%20in%20the%20Genetics%20of%20the%20Mouse%20Mandible%20Shape%3F" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/5/1153&amp;title=Does%203D%20Phenotyping%20Yield%20Substantial%20Insights%20in%20the%20Genetics%20of%20the%20Mouse%20Mandible%20Shape%3F" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/5/1153&amp;title=Does%203D%20Phenotyping%20Yield%20Substantial%20Insights%20in%20the%20Genetics%20of%20the%20Mouse%20Mandible%20Shape%3F" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460361/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460361" id="node460361--3" data-pisa="ggg;6/5/1153" data-pisa-master="ggg;g3.115.024372" data-apath="/ggg/6/5/1153.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Does 3D Phenotyping Yield Substantial Insights in the Genetics of the Mouse Mandible Shape?</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5694-4201" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicolas Navarro</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-7921-9018" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>A. Murat Maga</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1153-1163;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024372 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460361--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicolas Navarro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biogéosciences, UMR CNRS 6282, Univ Bourgogne Franche-Comté, EPHE, PSL Research University, F-21000 Dijon, France </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BNavarro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Navarro%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicolas%2BNavarro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5694-4201" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicolas Navarro</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nicolas.navarro@ephe.sorbonne.fr" class="" data-icon-position="" data-hide-link-title="0">nicolas.navarro@ephe.sorbonne.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">A. Murat Maga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Craniofacial Medicine, Department of Pediatrics, University of Washington, Seattle, Washington 98105 </div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Developmental Biology and Regenerative Medicine, Seattle Children’s Research Institute, Seattle, Washington 98101</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=A.%2BMurat%2BMaga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maga%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AA.%2BMurat%2BMaga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7921-9018" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for A. Murat Maga</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1153&amp;rft.epage=1163&amp;rft.atitle=Does%203D%20Phenotyping%20Yield%20Substantial%20Insights%20in%20the%20Genetics%20of%20the%20Mouse%20Mandible%20Shape%3F&amp;rft.volume=6&amp;rft.issue=5&amp;rft.date=2016-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Navarro%2C+Nicolas&amp;rft.au=Maga%2C+A.+Murat"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460361/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460361/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460361/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460361/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460361/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460361/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460361/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460361/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460361/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460361/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460361/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460361/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/5/1153" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/5/1153.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/5/1153.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/5/1153.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/5/1153.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
