<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/5/1297" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/5/1297.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/5/1297.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/5/1297.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/5/1297.atom" />
<meta name="HW.pisa" content="ggg;6/5/1297" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mapping Challenging Mutations by Whole-Genome Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.116.028316" />
<meta name="DC.Date" content="2016-05-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2016 Smith et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Whole-genome sequencing provides a rapid and powerful method for identifying mutations on a global scale, and has spurred a renewed enthusiasm for classical genetic screens in model organisms. The most commonly characterized category of mutation consists of monogenic, recessive traits, due to their genetic tractability. Therefore, most of the mapping methods for mutation identification by whole-genome sequencing are directed toward alleles that fulfill those criteria ( i.e. , single-gene, homozygous variants). However, such approaches are not entirely suitable for the characterization of a variety of more challenging mutations, such as dominant and semidominant alleles or multigenic traits. Therefore, we have developed strategies for the identification of those classes of mutations, using polymorphism mapping in Caenorhabditis elegans as our model for validation. We also report an alternative approach for mutation identification from traditional recombinant crosses, and a solution to the technical challenge of sequencing sterile or terminally arrested strains where population size is limiting. The methods described herein extend the applicability of whole-genome sequencing to a broader spectrum of mutations, including classes that are difficult to map by traditional means." />
<meta name="DC.Contributor" content="Harold E. Smith" />
<meta name="DC.Contributor" content="Amy S. Fabritius" />
<meta name="DC.Contributor" content="Aimee Jaramillo-Lambert" />
<meta name="DC.Contributor" content="Andy Golden" />
<meta name="article:published_time" content="2016-05-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Mapping Challenging Mutations by Whole-Genome Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Whole-genome sequencing provides a rapid and powerful method for identifying mutations on a global scale, and has spurred a renewed enthusiasm for classical genetic screens in model organisms. The most commonly characterized category of mutation consists of monogenic, recessive traits, due to their genetic tractability. Therefore, most of the mapping methods for mutation identification by whole-genome sequencing are directed toward alleles that fulfill those criteria (&lt;i&gt;i.e.&lt;/i&gt;, single-gene, homozygous variants). However, such approaches are not entirely suitable for the characterization of a variety of more challenging mutations, such as dominant and semidominant alleles or multigenic traits. Therefore, we have developed strategies for the identification of those classes of mutations, using polymorphism mapping in &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; as our model for validation. We also report an alternative approach for mutation identification from traditional recombinant crosses, and a solution to the technical challenge of sequencing sterile or terminally arrested strains where population size is limiting. The methods described herein extend the applicability of whole-genome sequencing to a broader spectrum of mutations, including classes that are difficult to map by traditional means.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/05/01" />
<meta name="citation_mjid" content="ggg;6/5/1297" />
<meta name="citation_id" content="6/5/1297" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/5/1297" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/5/1297.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/5/1297.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/5/1297.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.028316" />
<meta name="citation_pmid" content="26945029" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="5" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1297" />
<meta name="citation_lastpage" content="1304" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Harold E. Smith" />
<meta name="citation_author_institution" content="National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author_email" content="smithhe2@niddk.nih.gov" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3517-5973" />
<meta name="citation_author" content="Amy S. Fabritius" />
<meta name="citation_author_institution" content="National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Aimee Jaramillo-Lambert" />
<meta name="citation_author_institution" content="National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Andy Golden" />
<meta name="citation_author_institution" content="National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8599-2031" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. Abe;citation_author=S. . Kosugi;citation_author=K. . Yoshida;citation_author=S. . Natsume;citation_author=H. . Takagi;citation_title=Genome sequencing reveals agronomically important loci in rice using MutMap.;citation_pages=174-178;citation_volume=30;citation_year=2012;citation_issue=2;citation_pmid=22267009;citation_doi=10.1038/nbt.2095" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Arribere;citation_author=R. T. . Bell;citation_author=B. X. . Fu;citation_author=K. L. . Artiles;citation_author=P. S. . Hartman;citation_title=Efficient marker-free recovery of custom genetic modifications with CRISPR/Cas9 in Caenorhabditis elegans.;citation_volume=198;citation_year=2014;citation_pmid=25161212;citation_doi=10.1534/genetics.114.169730" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Symp. Quant. Biol.;citation_author=J. E. Barrick;citation_author=R. E. Lenski;citation_title=Genome-wide mutational diversity in an evolving population of Escherichia coli.;citation_volume=74;citation_year=2009;citation_pmid=19776167;citation_doi=10.1101/sqb.2009.74.018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. R. Birkeland;citation_author=N. . Jin;citation_author=A. C. . Ozdemir;citation_author=R. H. . Lyons;citation_author=L. S. . Weisman;citation_title=Discovery of mutations in Saccharomyces cerevisiae by pooled linkage analysis and whole-genome sequencing.;citation_volume=186;citation_year=2010;citation_pmid=20923977;citation_doi=10.1534/genetics.110.123232" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. P. Blumenstiel;citation_author=A. C. . Noll;citation_author=J. A. . Griffiths;citation_author=A. G. . Perera;citation_author=K. N. . Walton;citation_title=Identification of EMS-induced mutations in Drosophila melanogaster by whole-genome sequencing.;citation_volume=182;citation_year=2009;citation_pmid=19307605;citation_doi=10.1534/genetics.109.101998" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. E. Bowen;citation_author=K. Henke;citation_author=K. R. Siegfried;citation_author=M. L. Warman;citation_author=M. P. Harris;citation_title=Efficient mapping and cloning of mutations in zebrafish by low-coverage whole-genome sequencing.;citation_volume=190;citation_year=2012;citation_pmid=22174069;citation_doi=10.1534/genetics.111.136069" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=B. Bowerman;citation_title=The near demise and subsequent revival of classical genetics for investigating Caenorhabditis elegans embryogenesis: RNAi meets next-generation DNA sequencing.;citation_volume=22;citation_year=2011;citation_pmid=21960050;citation_doi=10.1091/mbc.E11-03-0185" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Brenner;citation_title=THE GENETICS OF CAENORHABDITIS ELEGANS;citation_pages=71-94;citation_volume=77;citation_year=1974;citation_issue=1;citation_pmid=4366476;citation_doi=10.1051/medsci/20031967725" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=A. A. Connolly;citation_author=V. . Osterberg;citation_author=S. . Christensen;citation_author=M. . Price;citation_author=C. . Lu;citation_title=Caenorhabditis elegans oocyte meiotic spindle pole assembly requires microtubule severing and the calponin homology domain protein ASPM-1.;citation_volume=25;citation_year=2014;citation_pmid=24554763;citation_doi=10.1091/mbc.E13-11-0687" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. Doitsidou;citation_author=R. J. . Poole;citation_author=S. . Sarin;citation_author=H. . Bigelow;citation_author=O. . Hobert;citation_title=C. elegans mutant identification with a one-step whole-genome-sequencing and SNP mapping strategy.;citation_pages=e15435-e15435;citation_volume=5;citation_year=2010;citation_issue=11;citation_pmid=21079745;citation_doi=10.1371/journal.pone.0015435" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=S. K. Dutcher;citation_author=L. Li;citation_author=H. Lin;citation_author=L. Meyer;citation_author=T. H. Giddings;citation_title=Whole-Genome Sequencing to Identify Mutants and Polymorphisms in Chlamydomonas reinhardtii;citation_pages=15-22;citation_volume=2;citation_year=2012;citation_issue=1;citation_pmid=22384377;citation_doi=10.1534/g3.111.000919" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. L. Ferguson;citation_author=H. R. . Horvitz;citation_title=The multivulva phenotype of certain Caenorhabditis elegans mutants results from defects in two functionally redundant pathways.;citation_pages=109-121;citation_volume=123;citation_year=1989;citation_issue=1;citation_pmid=2806880" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Fire;citation_author=S. . Xu;citation_author=M. K. . Montgomery;citation_author=S. A. . Kostas;citation_author=S. E . Driver;citation_title=Potent and specific genetic interference by double-stranded RNA in Caenorhabditis elegans.;citation_pages=806-811;citation_volume=391;citation_year=1998;citation_issue=6669;citation_pmid=9486653;citation_doi=10.1038/35888" />
<meta name="citation_reference" content="citation_author=E. Garrison;citation_author=G. . Marth;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=O. Hobert;citation_title=The Impact of Whole Genome Sequencing on Model System Genetics: Get Ready for the Ride;citation_pages=317-319;citation_volume=184;citation_year=2010;citation_issue=2;citation_pmid=20103786;citation_doi=10.1534/genetics.109.112938" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. Hodgkin;citation_title=A genetic analysis of the sex-determining gene, tra-1, in the nematode Caenorhabditis elegans.;citation_pages=731-745;citation_volume=1;citation_year=1987;citation_issue=7;citation_pmid=3428597;citation_doi=10.1101/gad.1.7.731" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Hodgkin;citation_author=T. . Doniach;citation_title=Natural Variation and Copulatory Plug Formation in Caenorhabditis elegans;citation_pages=149-164;citation_volume=146;citation_year=1997;citation_issue=1;citation_pmid=9136008" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. Homer;citation_author=B. . Merriman;citation_author=S. F. . Nelson;citation_title=BFAST: an alignment tool for large scale genome resequencing.;citation_pages=e7767-e7767;citation_volume=4;citation_year=2009;citation_issue=11;citation_pmid=19907642;citation_doi=10.1371/journal.pone.0007767" />
<meta name="citation_reference" content="citation_journal_title=Methods;citation_author=P. J. Hu;citation_title=Whole genome sequencing and the transformation of C. elegans forward genetics.;citation_pages=437-440;citation_volume=68;citation_year=2014;citation_pmid=24874788;citation_doi=10.1016/j.ymeth.2014.05.008" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. V. Irvine;citation_author=D. B. . Goto;citation_author=M. W. . Vaughn;citation_author=Y. . Nakaseko;citation_author=W. R. . Mccombie;citation_title=Mapping epigenetic mutations in fission yeast using whole-genome next-generation sequencing.;citation_volume=19;citation_year=2009;citation_pmid=19423874;citation_doi=10.1101/gr.089318.108" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. Jaramillo-Lambert;citation_author=A. S. . Fuchsman;citation_author=A. S. . Fabritius;citation_author=H. E. . Smith;citation_author=A. . Golden;citation_title=Rapid and efficient identification of Caenorhabditis elegans legacy mutations using SNP-based mapping and whole genome sequencing.;citation_volume=5;citation_year=2015;citation_pmid=25740937;citation_doi=10.1534/g3.115.017038" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. M. Kramer;citation_author=R. P. . French;citation_author=E. C. . Park;citation_author=J. J . Johnson;citation_title=The Caenorhabditis elegans rol-6 gene, which interacts with the sqt-1 collagen gene to determine organismal morphology, encodes a collagen.;citation_pages=2081-2089;citation_volume=10;citation_year=1990;citation_issue=5;citation_pmid=1970117;citation_doi=10.1128/MCB.10.5.2081" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Kulkarni;citation_author=D. C. . Shakes;citation_author=K. Guevel;citation_author=H. E. . Smith;citation_title=SPE-44 implements sperm cell fate.;citation_pages=e1002678-e1002678;citation_volume=8;citation_year=2012;citation_issue=4;citation_pmid=22570621;citation_doi=10.1371/journal.pgen.1002678" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. A. Labed;citation_author=S. . Omi;citation_author=M. . Gut;citation_author=J. J. . Ewbank;citation_author=N. . Pujol;citation_title=The pseudokinase NIPI-4 is a novel regulator of antimicrobial peptide gene expression.;citation_pages=e33887-e33887;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22470487;citation_doi=10.1371/journal.pone.0033887" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=R. A. Laitinen;citation_author=K. Schneeberger;citation_author=N. S. Jelly;citation_author=S. Ossowski;citation_author=D. Weigel;citation_title=Identification of a spontaneous frame shift mutation in a nonreference Arabidopsis accession using whole genome sequencing.;citation_volume=153;citation_year=2010;citation_pmid=20388671;citation_doi=10.1104/pp.110.156448" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=S. T. Lamitina;citation_author=S. W. . LâHernault;citation_title=Dominant mutations in the Caenorhabditis elegans Myt1 ortholog wee-1.3 reveal a novel domain that controls M-phase entry during spermatogenesis;citation_pages=5009-5018;citation_volume=129;citation_year=2002;citation_issue=21;citation_pmid=12397109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. W. LâHernault;citation_author=D. C. . Shakes;citation_author=S. . Ward;citation_title=Developmental genetics of chromosome I spermatogenesis-defective mutants in the nematode Caenorhabditis elegans.;citation_pages=435-452;citation_volume=120;citation_year=1988;citation_issue=2;citation_pmid=3197956" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. . Handsaker;citation_author=A. . Wysoker;citation_author=T. . Fennell;citation_author=J. . Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=W. S. Liau;citation_author=U. . Nasri;citation_author=D. . Elmatari;citation_author=J. . Rothman;citation_author=C. W. . Lamunyon;citation_title=Premature sperm activation and defective spermatogenesis caused by loss of spe-46 function in Caenorhabditis elegans.;citation_pages=e57266;citation_volume=8;citation_year=2013;citation_pmid=23483899;citation_doi=10.1371/journal.pone.0057266" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=X. Ma;citation_author=Y. . Zhu;citation_author=C. . Li;citation_author=P. . Xue;citation_author=Y. . Zhao;citation_title=Characterisation of Caenorhabditis elegans sperm transcriptome and proteome.;citation_pages=168;citation_volume=15;citation_year=2014;citation_pmid=24581041;citation_doi=10.1186/1471-2164-15-168" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. E. Mains;citation_author=I. A. . Sulston;citation_author=W. B. . Wood;citation_title=Dominant maternal-effect mutations causing embryonic lethality in Caenorhabditis elegans.;citation_pages=351-369;citation_volume=125;citation_year=1990;citation_issue=2;citation_pmid=2379819" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Minevich;citation_author=D. S. . Park;citation_author=D. . Blankenberg;citation_author=R. J. . Poole;citation_author=O. . Hobert;citation_title=CloudMap: a cloud-based pipeline for analysis of mutant genome sequences.;citation_volume=192;citation_year=2012;citation_pmid=23051646;citation_doi=10.1534/genetics.112.144204" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. H. Morgan;citation_title=THE ORIGIN OF NINE WING MUTATIONS IN DROSOPHILA;citation_pages=496-499;citation_volume=33;citation_year=1911;citation_issue=848;citation_pmid=17774436;citation_doi=10.1126/science.33.848.496" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=T. H. Morgan;citation_title=THE ORIGIN OF FIVE MUTATIONS IN EYE COLOR IN DROSOPHILA AND THEIR MODES OF INHERITANCE;citation_pages=534-537;citation_volume=33;citation_year=1911;citation_issue=849;citation_pmid=17817675;citation_doi=10.1126/science.33.849.534-a" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=P. J. Muhlrad;citation_author=J. N. . Clark;citation_author=U. . Nasri;citation_author=N. G. . Sullivan;citation_author=C. W. . Lamunyon;citation_title=SPE-8, a protein-tyrosine kinase, localizes to the spermatid cell membrane through interaction with other members of the SPE-8 group spermatid activation signaling pathway in C. elegans.;citation_pages=83;citation_volume=15;citation_year=2014;citation_pmid=25022984" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Paix;citation_author=A. . Folkmann;citation_author=D. . Rasoloson;citation_author=G. . Seydoux;citation_title=High efficiency, homology-directed genome editing in Caenorhabditis elegans using CRISPR-Cas9 ribonucleoprotein complexes.;citation_volume=201;citation_year=2015;citation_pmid=26187122;citation_doi=10.1534/genetics.115.179382" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. Sarin;citation_author=S. . Prabhu;citation_author=M. M. . OâMeara;citation_author=I. . PeâEr;citation_author=O. . Hobert;citation_title=Caenorhabditis elegans mutant allele identification by whole-genome sequencing.;citation_pages=865-867;citation_volume=5;citation_year=2008;citation_issue=10;citation_pmid=18677319;citation_doi=10.1038/nmeth.1249" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=G. Saxer;citation_author=P. Havlak;citation_author=S. A. Fox;citation_author=M. A. Quance;citation_author=S. Gupta;citation_title=Whole Genome Sequencing of Mutation Accumulation Lines Reveals a Low Mutation Rate in the Social Amoeba Dictyostelium discoideum.;citation_pages=e46759-e46759;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23056439;citation_doi=10.1371/journal.pone.0046759" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. S. Seidel;citation_author=M. V. . Rockman;citation_author=L. . Kruglyak;citation_title=Widespread genetic incompatibility in C. elegans maintained by balancing selection.;citation_volume=319;citation_year=2008;citation_pmid=18187622;citation_doi=10.1126/science.1151107" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=K. A. Swan;citation_author=D. E. . Curtis;citation_author=K. B. . Mckusick;citation_author=A. V. . Volnov;citation_author=F. A. . Mapa;citation_title=High-Throughput Gene Mapping in Caenorhabditis elegans;citation_pages=1100-1105;citation_volume=12;citation_year=2002;citation_issue=7;citation_pmid=12097347" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=O. A. Thompson;citation_author=L. B. . Snoek;citation_author=H. . Nivjeen;citation_author=M. G. . Sterken;citation_author=R. J. . Volkers;citation_title=Remarkably divergent regions punctuate the genome assembly of the Caenorhabditis elegans Hawaiian strain CB4856.;citation_volume=200;citation_year=2015;citation_pmid=25995208;citation_doi=10.1534/genetics.115.175950" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Wang;citation_author=M. . Li;citation_author=H. . Hakonarson;citation_title=ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.;citation_volume=38;citation_year=2010;citation_pmid=20601685;citation_doi=10.1093/nar/gkq603" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Y. Wang;citation_author=J. T. . Wang;citation_author=D. . Rasoloson;citation_author=M. L. . Stitzel;citation_author=K. F. . OâConnell;citation_title=Identification of suppressors of mbk-2/DYRK by whole-genome sequencing.;citation_volume=4;citation_year=2014;citation_pmid=24347622;citation_doi=10.1534/g3.113.009126" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. R. Wicks;citation_author=R. T. . Yeh;citation_author=W. R. . Gish;citation_author=R. H. . Waterston;citation_author=R. H. . Plasterk;citation_title=Rapid gene mapping in Caenorhabditis elegans using a high density polymorphism map.;citation_pages=160-164;citation_volume=28;citation_year=2001;citation_issue=2;citation_pmid=11381264;citation_doi=10.1038/88878" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zuryn;citation_author=S. . Le Gras;citation_author=K. . Jamet;citation_author=S. . Jarriault;citation_title=A strategy for direct mapping and identification of mutations by whole-genome sequencing.;citation_volume=186;citation_year=2010;citation_pmid=20610404;citation_doi=10.1534/genetics.110.119230" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mapping Challenging Mutations by Whole-Genome Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
<meta name="twitter:description" content="Whole-genome sequencing provides a rapid and powerful method for identifying mutations on a global scale, and has spurred a renewed enthusiasm for classical genetic screens in model organisms. The most commonly characterized category of mutation consists of monogenic, recessive traits, due to their genetic tractability. Therefore, most of the mapping methods for mutation identification by whole-genome sequencing are directed toward alleles that fulfill those criteria ( i.e. , single-gene, homozygous variants). However, such approaches are not entirely suitable for the characterization of a variety of more challenging mutations, such as dominant and semidominant alleles or multigenic traits. Therefore, we have developed strategies for the identification of those classes of mutations, using polymorphism mapping in Caenorhabditis elegans as our model for validation. We also report an alternative approach for mutation identification from traditional recombinant crosses, and a solution to the technical challenge of sequencing sterile or terminally arrested strains where population size is limiting. The methods described herein extend the applicability of whole-genome sequencing to a broader spectrum of mutations, including classes that are difficult to map by traditional means." />
<meta name="og-title" property="og:title" content="Mapping Challenging Mutations by Whole-Genome Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/5/1297" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Whole-genome sequencing provides a rapid and powerful method for identifying mutations on a global scale, and has spurred a renewed enthusiasm for classical genetic screens in model organisms. The most commonly characterized category of mutation consists of monogenic, recessive traits, due to their genetic tractability. Therefore, most of the mapping methods for mutation identification by whole-genome sequencing are directed toward alleles that fulfill those criteria ( i.e. , single-gene, homozygous variants). However, such approaches are not entirely suitable for the characterization of a variety of more challenging mutations, such as dominant and semidominant alleles or multigenic traits. Therefore, we have developed strategies for the identification of those classes of mutations, using polymorphism mapping in Caenorhabditis elegans as our model for validation. We also report an alternative approach for mutation identification from traditional recombinant crosses, and a solution to the technical challenge of sequencing sterile or terminally arrested strains where population size is limiting. The methods described herein extend the applicability of whole-genome sequencing to a broader spectrum of mutations, including classes that are difficult to map by traditional means." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/5.cover-source.jpg" />
    <title>Mapping Challenging Mutations by Whole-Genome Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"AU-S83jSgc3QJoUvMrFiyXhHI17fen0Mzk_eKzPAnF0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460362","apath":"\/ggg\/6\/5\/1297.atom","pisa":"ggg;6\/5\/1297","ac":{"\/ggg\/6\/5\/1297.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/5\/1297.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/5\/1297"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/5\/1297"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/5\/1297":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nbg6XeCoGx6gXH7SBbosjmcxqDAGUaI1Zb7yR1jNFlzhMN1uvZ1uC6ToPQtKP38Gyls0X4Y3FU\\\\\\\/2hEBhkByJU1Wq4OhjIS1taIKcGWlMCNAeSG7rjsM3J6N3id5F7LVfl6WcIQexoX4caPzJo35rmb9tJXqpGTMmGK0ruN3NqnPPAMZzOqi23zsF9eA33SRDrLZMejKDmcGclr7b\\\\\\\/dXO\\\\\\\/DX5XcsnfgtbcfMqHQM+O5ptR2TZAUuPIIlrjz5G7pINhCjU8PZwVtFNkNzCWaPLRzvGy48u+bOovy7K1OAs+VjcQZR0VuZxzzvePrW0L5AHSsf8jaArKGaOD5XfwuvhrkRjEA3JgXlBvle1lbhp1MHPUq51cI2hQqYo7+n5nCVAE9ts+Qecxi+NZSpVv+2UYuhrfEsEbBrfV5cQa0ha4k9a8l4kMdAiIb8qldLVxC2cZ1py4HcLAaU7yQzIT2m\\\\\\\/0Ppdi\\\\\\\/we6RVMiqN1G4kSRb7Nj75qAx4kcJN9ipegIPJhyYGulH12PRUtYAvQ7ZoWfQFJ7UtPWZn7rV8M2HifwzLLjU1kyWLACbViymIZedcdReZecO41viF0cRDY791IMPRa4QSEX2z8BSN2oMjFR\\\\\\\/RAG2WCtRmBxIN9RxUzbrdNhOrPwlkeyZYmA2RcyBjwIKb8sDrhl4F9Kekjd3bDnLxuYjhbIm+jux6LLYphd6Q+MuKxlDp8OT\\\\\\\/CUCBnD8QrA7eRBfEZemFBpm7TnIGit82gYSOUYgekJmgQUBqlgPsFRuuiWVeH3ooX31UCIwrYf8p\\\\\\\/y03wjf9W16BXMTnOpvbV5acvVKXjGqgEwzZ7yYLbkfxZOgwjXPCE2k19a4TUY40nmvWgSgrPK9C7cy5bR4ELLGfl3bK1K4+k9iBOf\\\\\\\/mFZ6N6KvoiysFdlMhgdnUnXhoCu1nAfFTFfoFQlbHa5HR3mQh8OmuxRd+Hkom1oTqhcs7KqPZbYPy64vEA+5hQ5Wsdb3ueFia\\\\\\\/kf9SIKRl08SPHzU5sfQDDVU4iU+qvmmFt\\\\\\\/9LETbjTOxEiekQbJX1KK3ZCp96++a\\\\\\\/sMlGjipWqNrn6aGcr8q0r\\\\\\\/eQolmj\\\\\\\/9atnzjsYk1h6kosDrtN8Gtglb\\\\\\\/6tqoZLZSYgQe+jaBbmBVM7QGucMMMbTTl7j3ny9nRtmyLi06Y5tqCNJa59sOFJPf90rnPNkxtfJWZQiGIyixKepbIUvbFFDLIFOt+Gjt8zgeZINcYQSFhVSTUWxFuS6pfL+lFfOqd7GxMsH7HrLXm4dlAU00bo\\\\\\\/a86XzMHn6q\\\u0022,\\\u0022iv\\\u0022:\\\u0022oOrdDQqY7II0yDJ5OtdmMQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e4b54599ef49f62b0be96fdf550cdb32ebaa0f270aed268250dc5b7fa136787f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002226bsL2C9AMG\\\\\\\/ocSBe4XeJG6tT4h8p6XjnXJw5sH9MBQp5rUynVZaSN7a\\\\\\\/LJvrmP5C1dZKxiZuaQL0Sxyg7vppkCi\\\\\\\/Gs9pgf5U\\\\\\\/Ozv6naQeMLArLmF7sTozjd34Mk5oxj7UHxxySgOjyp4e51RUca5cN\\\\\\\/b7N\\\\\\\/ZkMGPe6tNdnjw+NHO488zWTd9sikkQeGGCCjDsDfYT3WvpHhLeuKZ8i+REDHQTRo2ASa3RY75DCNPWBbNn0E1\\\\\\\/TR6qzwmZkFUL1BmNHfEWEhzB5OAgDGNzkfM3NgLamfqYdVIgwJ2ND3pHRQMnHo8kI0I5KozGI98Zsx8PaDiIwPobvATb5tLmtmE63\\\\\\\/wpPRZqVhaO\\\\\\\/cSa1QPyWj\\\\\\\/HczOsbX+dcACYy6mEJrRueAlB16ZirjslLx0TnA8xqgSi7rf0VE6ZPiUU1oTZ4gvTve6Fp4ko7xSCNlSE2klE1uM\\\\\\\/kpeajSFbSJuMXE1rfK0fo5xzbOv9X0zgROOtjqH069XvN3ZI8b3i0nc6MCLOg3mCzWRDsGjimgLruTTdTdBRtZigzP2mFbo12U9kl4VI0Pgz7iTb0I+pyS2JW5PScFF4sXjbCx3kqycTetg+v2ah3fTyCHXI7JGW3bo1uIO4\\\\\\\/smcpv8WqE0Ri4vcNxo1pG\\\\\\\/4UuTtmhyemf7F4B5XrO10To+RYQNIWm4+KPsajpxjFX6GhBwwHcVFTOmjd7LJbgxiMu1tpkgnb7tQNXJZoshtKvr2aSPKvKCyZLskf1UvDbPpq2iiaLLuYm1btyt1NqL5o3kNO3gNzTYsX8nD6QFJFIxGJ1clZCeMC54R+\\\\\\\/8JdVyfJcURfIQrEYLb\\\\\\\/Ul4nYITtuPgpRP4\\\\\\\/md0CKbcEeTPj0xBnP41lnO28PATXTrJJvh2+AFa93HJLED+YtU4J+h5WIGz6fs4jY0HIOpowR6QSTudgStTKdIB0Ib+PyoKmUm9dSfMrkGMSHwnSIGaml\\\\\\\/S2vJTitqecwHeBNG5ht1\\\\\\\/l9WRSSkE5ooBogRM7cYdsLbeHg7sOQZ6RZcTzKX7qB38dO4jpHeGNh8y1oZI3\\\\\\\/2s+iej8YUjgZBI7h3z2p70DiMCB4p5K\\\\\\\/LpqTxFu2wJfvq6C65Sn+XKj72LasZf2K\\\\\\\/pKEzLA859hPAS+q4h1A3ouVhBlUbe2dVcUk7JgdZJ6\\\\\\\/Evw1iiJ6K+1IcLvEcSBNB9udaAu7+KUWtvrnwp3rW86sq\\\\\\\/Wjm0zLkcHTkD5Yc5sKLgR8AjQW0VSThcS7kRqkWm2TbvZHtQed1bApAe6aZOU3zmW2br4P4f++VE1NtV3Pd3t1xuDfhu6qVE+I9Y3H+2A97996bjvoJ4o+7cP4bfM1m9O3zRwbVUPoxyLvKnBC3nMNlzoAF5iYR4drbDYtYT0zwin\\\\\\\/dQe0PgSGofTqRXeoRpmfp1d9TxeqLWWDR8x+nbrQBWolxc9Sf0Qbs2+4VwJZGyxc43fkow+qDs\\\\\\\/wui0hrK6ZkEsN5M93Fhw5eQ+5izPGR9u\\\\\\\/sD5KXewNLijooK6mgtGPAvzVmP2OwVs5jr4Wq3UrHsj3A323cMDA0APbKzV2xz40ESwPDjQJ9LoliuzmCRVpT3objrPjzXb74YCEE0GUDn+UrOQDTePwNrO9P4LbLyHY9+xCCniLNMqwczjAdfxRalua5TQh315OurgSuQC9NnSEaTG4niN850xgMxJsvuHDGvmmViWGXSWakbdEfa0CH2\\\\\\\/XrGnko4y0j\\\\\\\/ZSr7HrykBRcQPNeVZMInG4FxxehdShP5xfPofscQhvEc\\\\\\\/ps1bQfGSLMcKf06Gy3wiLb1PLth\\\\\\\/mNx2i4bQT\\\\\\\/RjlsSTT9pt0igoifkapSiLu+EojIiNfidcbi7eUCjFI3sdU9UPKD5e3b4O4g2YxJwZSQ2VWOvSAsai5\\\\\\\/UYdPw5+rADAQIVCl3lQyeoRAFFS\\\\\\\/bjefudoe2HhvggFlbSqZNj96UvgxkSTqicRnXADP+bnMM526QIS4qulci\\\\\\\/+dosDRytLWAos3ALWX6xrZDLAf2VlEadPxYgPFu+DCgNg421B149gJsEK+WFJsIoBv7afp8IawuXRhgjulDIqDfjT+3TENqNHhK5Q2o4c1mcmvWWSWSV2sfmhWzrSEAUOq4rpsOoORTszCufDl\\\\\\\/KiouwUQbs0SICp6nP7\\\\\\\/AwVJm2kxxvnIsiJwXC38oYSJM40UdR8Z4aQ1ts7jBljR8pBnYL\\\\\\\/OEGSz6TpdpIc9BrDg2\\\\\\\/rIwg9hghxksn2R7okEuKhm3vn4RovzCo4imdNdkLYojVipjr7GQ8SLD+THs7BlmA81X6PdGv+Jzm\\\\\\\/N5vf6nw53ZtDQMLnLy15vi0q+IfrB2YdedG9JDSBmY5wSM\\\\\\\/OEUceUctqpeL5fOA9lyxgaRUO9b5QZgG\\\\\\\/6X3OZAXcXB1ApZXt0oUpgYCbt0Vu\\\\\\\/Bbaeyu3Zs\\\\\\\/AV2RHZkjFy7qvz4S91LzbfQpBzOlGQFRU\\\\\\\/YNyEmZfCouqPGVSPU3Zu\\\\\\\/J92cdk6CvUwFz31yssC59UGnjeQQasASiollqOUGqqtSDcmiz+IL50THvVnu6f5KTB5Lp6C9Kp03oADGXiPQOcEqRY94qJ\\\\\\\/2+ZihD683eWdIKeqBSGtoZ+eOBdFPSdrC34RrIfYb5Xv+ZPvNm2GoDAb1FpZBWbB51g6Xh945ChYN9t24DNn7Lq6CHcDnq0KgFY72\\\\\\\/VBjNum0fyfDD7i2CibYNRGrPt9YaYDMIs7lFs+JlfrIXpbtmfTE4vDLDhSJMkP9Yb+yFfaRGic6wB4MBES5UmGoxDIkfAj4ZVhXTU83Gb8qffBPGzvcX6Fmy14HxEBrvbjrW4Fg2s0iUUmltdqflYvi7sxWj2h7spbunUKTmnDgkqX8nN45dGIkPIfYr+eDJuqgFers3IMZVNfwLEHQFWmFDRzomU\\\\\\\/uNC1zJQQdLl957OW9BPQg7n\\\\\\\/nd4Uo6CiaHBj0fRPAVhauiegqIGWjUG2CuRzXOb3FJ2BYFEYUBWcvkX9VVDC\\\\\\\/u\\\\\\\/BTq\\\\\\\/WiUfR6p6Y9Dn\\\\\\\/ugWIQrNuli8o1BXI7JlkxiTADcgjpmpMftK9Kn0y+smtA\\\\\\\/ipJKXkI55tytkc345fJrsBkzfGLm0eMd81zLxhjfl2tkNe\\\\\\\/UWKPtYivJdaz9wBkmz8\\\\\\\/CXGOiStxcP19f8jx4NhYNf9DqjO\\\\\\\/Y9wrI9rl7U1nvPDtEsYs\\\\\\\/AInidFc+NaBhGpKehV0OZlM8MTzlAskiQqv8Sgyhessqlsx26+zkbIntLyMt7gfwtmWChU37i2muYmbaJKfepBARIJmzC5JpMkv26mJ+qAlcYU7AKf7YoEekrFmu+UcjmjX7cU8Q+wQ6doLi8lJwQWATnR5ih5EfURBNWb86cD\\\\\\\/asc4pALNA6yyPT83ry3ilWHriECcCHF0ERUd1Yn\\\\\\\/xdDmkRJ2271MgL8SAZjEegOumUUDM52nI4TwSbioDPpKc4k8yEvD6DaFB4qq0LTiIPZznwWS\\\\\\\/mdYnFaPAawUkATssshf\\\\\\\/Oe+3MF28oce5hkAjFQEzItQqGuTpgxRQK1KpON5F14uB\\\\\\\/PRJG9kV\\\\\\\/ZqaryEGOAXR+ceQy0Ui8qpXNL\\\\\\\/HcQwSLqTco1D6yZ5DVRpnni8C+o\\\\\\\/ANUcY6bxIHuyoQlbXeyO0aatCU03TG8ESubW1eEVScYnEP++\\\\\\\/UQZe9I9RQr3lDjiyKT2ormTVbrx6y0NmpwjaWlB9VCSSaVhaQaazvnc2rHaVFzL\\\\\\\/xBz7RmVu7e019A15aAwn7iHMu+LK\\\\\\\/rSt6a6x8H2MyE7IasYnxL5Zee38hXmYs6hVXEUsWpqSMhaUq5feHywv02SaQwAMcTZAbSRntDlh9J65TBewzoRMhudy69oh8HiF\\\\\\\/4UwvQ5P89jXfLaMu\\\\\\\/8CRYb4ZveS+Z6CqVCShE6PvOuMrrmPqC0GnZBmn0owAQAg+RfjHfpp\\\\\\\/W4xcaSMsTUBXavuFgmue8E+gjM8SPMMqh1QGnJnauSggc8X9sY3qmFApqQ29g\\\\\\\/hscAlYGTUMbLboKugJoUeH\\\\\\\/jAcdVd39uvl6b+aP1vEDnuthMjUTk7r6pRVwZ2Q4t1ZfeILTwQ+YWnOenwttiUliQor7bHvsttx29GwRiDFtV1slkJ41Xti4PBTsonIIDncPdQDIP0pLXmgCT0JRVQ49ed\\\\\\\/aLL0FrvTMOjmwAT9bP9raIneqADnd37ntbzJ6xIOD3wZkU0DRHGYGZgGJXATQm65cYjNhmFA1LbMGJW9o\\\\\\\/CoITmULlaw2MsRJmC2o1Z8wBdQchd1\\\\\\\/eL1XzaqtXF0GJnG+z5nB1Xwt1iCOeeixXIB9GFAamgQLkyBSqDTKja4ZA+zfnGYTw0Wto8fdkNcYge6NIrNw4ZX\\\\\\\/aaYajPdlOFfRwjyWwlEzKkPlKuD+P\\\\\\\/SF\\\\\\\/gtpJCHhWRgm4bfid961N2Os5AqKpen\\\\\\\/kGkTHdG+pj05abEvEtzFhQAL5rAp\\\\\\\/dkQjUP7b5wYaEcPRdFjP\\\\\\\/MfE2ekRm1sRZ2fEE9nRy5LQqmesoOaeirY0V5n+x6XRNioAYfdJVEGmKu39hh+nXaGJLFUhOSrCHXA9PrTTUwCPRHr7wmIE8mCTj5JmUZUno280MQjBIhddntpH5tF66wgbYjr24raCZmbof5SDTK9w2XZsDbxFRBbN6xwFya4aw5U8Z2zcBmXsR096b82mYyUEr65xx5HlFPx\\\\\\\/Sb9KA2+1BsQi35XazqspyEaJwjMtrG7KSNhMa9ERxJsR\\\\\\\/Q\\\\\\\/oLreNPbqHw4Yi+Xd2WYippuvEYfrKdXN5XwsowvqFurqgbewMtRiDXfxrNTwKlCOerc8jv0gdr5K7LtAWPFctpulA8wYKGHYkSJoOoid00yIRLRANkf9Di3\\\\\\\/Kl3SpbZLqJ+6ISCMMFeQk6Wn9wOhM1\\\\\\\/O4qQNlQLE16NS\\\\\\\/GPpF5B3EMrfYeMwOAN4wx4UnLPP9HBQubdiBFBxLBaaumpPf35g53ZAtt+04V52slxE\\\\\\\/9YnDR1z8K8+V6S5KMCr1YZuRfrjjqOdbJ14Yal7cMly1fydgmvDpfLJPzDw5INgSwcehi6Bb1BODLT+3noyeTERf5ywZbAaYXrYprfj\\\\\\\/UJb+M3owWaWMVYICV0RNOVSnRJ+3se60hnWE2FJG4LVQMf0XhQ\\\\\\\/aPSTuFf4WvBGprgaN0kLiUKY12stHHF\\\\\\\/NdMbPqNiIX67frie2AyMrfMQcXoWHK2waWXyBm3HL8iMmkiPvC95zry8J0XnglCDlmZ02Ug\\\\\\\/2zKQJvqqx4J+ct1Xwnlfw8k3N90NqptOc+ANZMe4u48F1oek9+Szo4cfq20R50gGlgLRQcygjapXy3rtVw4niqs+F20Am\\\\\\\/uvnxn6u1NfclBSqZcOTPAvvRmle0m3iYuRTf5uN5pedBgqhqtvDRSP6oTsPHk\\\\\\\/LmMhRVOZ+JQZG+u96RcYnnUHDH07gHZZrwoLyzyxOCftTqwrJiyA0vVRXQEnKxkYhWsKB1IxuRIhwyhdrHYCbwIH65DW794RZLzYw+Kees2Sqx60mSYq8kIE6VVBhywRK9gsHebcZQRSp2C8r\\\\\\\/JVulusobiNO4fJnyx0\\\\\\\/ZsZRgIw2+LIyH6qjBeSLYg8Uc8y3T\\\\\\\/8JLYbN8JzQam3U6ddVQW+1dKlQNeO7ZMjSBH81r1dqI0CNfUE+8fHTZqYA8YkuxRckoi63L7jPdnnCtFl9qQf5wOlJ5e8LOKF4HYdtLnoMMLl\\\\\\\/29gJTmSDa+NNmgGNW6fSZIJZeygy2bHHVP\\\\\\\/0y2orskA1A6vXb+g4bbtDgyttUPAErD1KYzF9pDZZluQKe5yitUnUKxmNb65Ogwn2VdRckmSf7EDSuzZ9+b9tRZ\\\\\\\/yxurzQB\\\\\\\/7yiFPQmTIFfTexDqeKQ20Un9SMTOjOadx1u9jJp+DPa5h2o1FZZQ1ILmG1sGbsWhDO7ju+M\\\\\\\/eaq4lO3u4IiB+WmV4q1Fmpg8yd\\\\\\\/YMPAXhj\\\\\\\/51VRvzwP6o7mUhpBvRX\\\\\\\/PqV+7OGaVXc9fEWbCDa0tzka6vpNxAi0HTVKKYfD7xFeCKF3iSYwT3f4kS+EeN4fgr6cp0Od9DWqDqy+O1x1SSkUkfKuIf9AqRl0cjuzcZ7NqiJ4Pxv82Ha61tD66EdAS0Sc7DE9g2866nPdg7vhJsUwS0AWPaV25rKg2ZLOX6eaGW\\\\\\\/TSJyrNnBIWEAuVA\\\\\\\/jbfxtPbPF9Xzd49bBIJO1zK5IdUvw+9hYssHOZ5eXTPCd7G6xiM7PSj6y2sS1KP\\\\\\\/BU5D7NtzXpYyvbRFDP6F5XUy7C6vONNFZs8h85Pr08\\\\\\\/0uGlokjeEFD8gTlSyBSXptg5GPxdhjpyyeEXCIVbdXErwtD+GD9V6r5AemssgDqYm6I1nFQhew+5+zLztd77aL5lJ2ltwEZysYzbmwWvUqrWNzthGMs0EM4UZpqWELPYUCFStqwjYyGYPKT+weRwzNc7OeTBDlVHEsLMT2wRTpEWHOaBocK5BNYEhL2PhsHwX+R6abblXI5diwTJHic3QbS24XrhTEUzLwzTtFWzigQqwRMctpT5Ff11ctJghmQxNIl7cgn3HojIq1q7sNgLm75xnkXPt3ly4BFlXEv6WALwpGA5dUQX7KVxC8nBCo4DS+09Wzimr9qtLZPnwRDUXeLrIWUXkLTqXDEuJ66ZpO0otWDHJhUZWM+b\\\\\\\/Ju7r32pcjhmfjeQpgYAqz4vQRccylnjst5+AcOV\\\\\\\/bzv4KwUMIpdV5ygPWJWwumIEcGq9E1vPJlLBTaCopXZPZfTjOwTF5BAr05Bd1wuu9lcGmCWLCXBGFlxtfoggGbvUVSxvrDto9OqlWuynjjgZkDFC3LPL9b3k2l0NiDZHYTz9FoqlIqANdWfWW4rs2wN5WMc0pKnTfa1jQrtwvpzAxqs53P73oU2qXxVKnMCk11sxxoq27l7sV23gKt0Ir0lx07z0xltLXlSslrnnVHtYBKvEfi\\\\\\\/Dgwm621eDxSP7RloRQwWKHQK2meX4Ag15rfmuRElfRRVfZodn2KzPzFc1NhaHhYRYBVtvfelMFJTBCoHOWwNYxDJgN5SnNIqIgZ\\\\\\\/M2ppd3LYRf4Ue3BNtqcv\\\\\\\/Bvj+94Rnhx65Sfxuo2B5nBRJAYHI+jCUamz74tuDSd79YbnOKNQtnXOa+8C2EbU\\\\\\\/Z7MA029DhEoHOwWhE625yWe5NicFKab7BGLG0X\\\\\\\/sW13tOPDw+q7msmPKwAJOrDGn7KTyNPAom6SQfem+JQBl0I8yjZTJEbHHsP+Q7Bb1D7f+fg2lKmRW6iKtWCy5o9BLh9nuMQ0D+G6m6YhFhdoL0wV9sF1DEoL5SZmlstPLBcOkTXDNf8X0XdULjGhuEHCgKUxsXwvd2q8FqLaQCrIfaouiNcVO++iSo2FBeQ9HLd9wyBfLR0tP\\\\\\\/bU\\\\\\\/Wa8AU5TVC\\\\\\\/2\\\\\\\/CoKCGu7jZd26Aknf6DKkcQiq5v4whKzEzWE04fBXaecRgVIUb+MzTwN461LEg8xA3+SL+fsh\\\\\\\/yhGvUwjxSJmJbQicsOureeuGXnMhXvNPaTS62oW57nlBL4iaRqZ7PPYMFbHT9sykb9F0C\\\\\\\/CeSvJur6xksHPV\\\\\\\/+YYBJNVi8wePt+0dqUlyYxb+7Cf9CaOIgtOsBLbVY9srS8055GQcalXfrrSQkIJqKO07KTOGD81JEmmFnPbXtxQKTAVBOETewG3Q0plH2MvOd2LES5+CZyruIsCUdhG8yFSAlxVT5Bdr2Z2dMCWDfCDxaUwYhD6dGrhQAcuLulwnRb+eQGkk+n\\\\\\\/r1DzHnAb5I+6NoxQQ7tlQNApcwWrBzm2epwabs4cSIX1YWJvDU66P6Kd9ONRTC0Ltfjw6jg3XdIEihcA\\\\\\\/NadHWouJQ4gW\\\\\\\/OfgY7IGbabkxn8ZvIiCtZu+KwRP43N25TL4G5sSmnF6jdb6ng0L32iIBUPrUOeSFoAY4cgJ+VIRYTOY3MpRz\\\\\\\/URfjudXwInl0JDuzBqvMaBCi4XrAy0yh13FXZqnQUuVuAeXLB9TSN5fnznwdCrfNako4JeAACZtK5YGHux2Zgt5YDOOxJXcxquODI+U9KrgTQDQaSio0bYx9gdA52OuvfatnSsE\\\\\\\/R+OO3y5SAl4DSu1uv05z0pilaFVPhvVwyu+jcrtKEbNkJUmnJfWzyeDjeZpaxDDHNhrPkEQbkngMx9gJRst+o4gldpu5693gGIjnYUs+48FaC1vx6039sonF6XyjXRgxP2\\\\\\\/0nPvFeQ5ja3mdJRKDqrfCSSdeun3HXKl3IQcwutKIbiV\\\\\\\/qMh8MQNTOlM7XDO7su0PxdVFsGFV5xVJdF3U0vl416hS85QOJsJS+uwrvU9mTSGX8N3eUVzCYF9trdw1nuDwVMk1IPxdCU12uCzxSUsTmiq7mWBn\\\\\\\/kSRzD2\\\\\\\/0Zv3eQpgbdcUfHsEt52OwVibtqzxzcbP1CQo7nUjxE7d8dblyVauwKwEQySV0bJOAMNvuoCeuIUfz2bKhU8rUQvNJQJynU2mI2l2OazzAPBmZR1iLfzJuC8OCSIG5y\\\\\\\/cCdyrA0DXb3qmsJIr+9T4\\\\\\\/qMYP8t+1H+LS71k3dbxv+AnzCxVl425U\\\\\\\/G6MxoWtCrbcOVLcdbJ8LqAATSEkG0ZzbeEtXpT7ZnUzTMLj3LYtIkUdrUAayKXgFReDCcqL8rGHX2koNnu4JpvhJQXxVKzZ\\\\\\\/IFzLVPxeRoPZnITi7lBvdSYGwnvX+xkbl7C\\\\\\\/dLYjdz\\\\\\\/0H1xRMhxgtNRPUiAOdnONKe8zM9I8E2Oekx0lvDneKR1i9AFEBWvjIm7QsSA1N7X5BMxMpW4Ayf\\\\\\\/5fe9SYngtYuksoOg16vTwkTyqKHRTv7pnwZLgrb\\\\\\\/1MWhl0EgWYWZIzQc\\\\\\\/v\\\\\\\/uRFsrG00xcC0ut8PTZBrw17F6akenBocy39xFZ9b1HhGNpBqrWV6JUbf9ZRJvelG4exnGQLDziAp8nptT07S4H3LBdYH7CRP1ZEWw5o43wSnWkaRft5joQHIKrL49CbgrOXxLXaD\\\\\\\/T1pfjerSTdxI1tLC8M5fXHIWRB2xi77NQkETMHnES7iBIoBTY0HL1QG1JhCzkR4ujZyeqeEyDYG886bSDBAeXAGF8ZjsV+aUG35NVN\\\\\\\/RQNsQ+hF1kr84dgkYmpBNyuKw3T3spKIiicj1g3xuvWwZzLsiyOmQWz+Gn9pMNYv+wCJa2BUsQ3b6OKwCjWRg4gcifilaJ+1Qdfagkoaqlo9tS0XFslIh3UorpwiAuDrYW\\\\\\\/ZAyfxxmgWAbGYzokmrxN9PL+rpvotdCucugu6pvarkotijhBs50aXDzoRoreCE0IT5ClS2X7vl87cbrgbzmebXNxF6V3HZgdDLjzftSK28hNpzp0NQNIhmQgI6kwRuzEDvtw3TxQxdweGjTjRg+dFktUid6QcAfEVwKvjLlfAhQDkM0idifZpMMaTxJ8iM2IojrNtIwShyAOoRHz0ldFsOj44yTyTZk26WtOjnGUE5ThJev3GIEnLTfCSyfwzrveaIK8MxHsufY2FqU1invn0MUnr3JDsyKhFq9hv7cQQojoLXH6wuHYHsdQAnKX1wTMRGw202RFGBtRZUJ8o3OiS526Rs5oCwtI8j5xlPbc8ounUAo5n+qfVAqTw0sVdQek1QN1EJ0q+WJU8kBbAnUcoU7NSAKuu3\\\\\\\/GTdofh1wZfQ0DkkEcew1TZqD7dnsbboIIJ0NGFQ+y2xp4mapJp13RI0sRhpzDZ0Vys0mNBvHcAfMwjkU1x+7IiOvjy6fIZWlvdv36w1DcnxlIGzgL4qIx0k7f0H+afNwO0uBq82DqhTPvflpxE6V\\\\\\\/VSEUVDEbyDz0ZrOesqNNr6PmkAclTvba2oCTwnYJMz7VAVNRUeeNP46SCGG2Pk75mBqlVcFkHwaZXYsjRhfSsQdLIcter2\\\\\\\/Q63y+WNkD6GIuLBQT1SCIEKhJKu\\\\\\\/qO5ma\\\\\\\/\\\\\\\/yuMFyuj3HAO\\\\\\\/8qPHVGUM\\\\\\\/nv6BCUcYHy8SR+SSP4sxWaM9j7a9OZV0X4y7L+ddS+s7JG+oCl4odBv1oqVp2hPuOQihN2Iw2\\\\\\\/7XxondGNyPvgXMTMdDPyHc\\\\\\\/h9WbdcFHTbEAmv9ximYOZsnHMBaxgioXSV7HZn7nzyaFmi3KE++e0F75C6m4c3LW\\\\\\\/p4jb8iNmkVx2FhbOapyxf7Bd9uwais4\\\\\\\/x2RJSuwGRRnirgKMp1CVc8KZSE05yfZfOcG3Ilh3Yc1hdfm4fBoYhrckHQO+HEK8rj\\\\\\\/CveOEXx8KuDzEL7pZH3FhZtOBggWsc0VYXLFPxQAU5izV1Xvd5e5LkqGAMk2dceINjVuJkV3c1\\\\\\\/5buUgmN7MZbXEUC8ZgC8a0\\\\\\\/Fez3lQlDaPRX1zmxRhbMQg6uaq6mAiRc\\\\\\\/q0rmcib0UNvTLFKl44bZvdvlR5W\\\\\\\/CmYjfknDHQd\\\\\\\/R89svA6fDe7mhUrbA9yRuB3UaQ4YUYGlaNAlJbiwGSlZLmwy46GE\\\\\\\/mVEOIgGKk8nS2K0nR6xuKcTkayt15aHwKXag51eaEnAmEaMVT6EMtC3yYHIP3fnbbQ5FrbEIMDyB\\\\\\\/w6ogs93JTlgkr16GJny9nhABUVKIV9v9Zu3S26jfAaf4kbYqVFpzp7\\\\\\\/TkFyNH4CRiyL5IfiWANQin\\\\\\\/Co\\\\\\\/O26SqfObDbkfeQO7X7CgMsSAkP6GbDqWRkJ4qRXU2K7W5sO0g\\\\\\\/lne4UzoSP4icFLPCBdM8uNAUlBNYFoBQuvZM4uZ+1MHkygP\\\\\\\/8NgKxte\\\\\\\/DfDiKTG4ez8sU\\\\\\\/HUqUwNJ9tIMy0EnrBqwb8thWeyWRQ65daF5k1ApbA502CVKoeBqT1Q+jWO+An7c4H+YrDJ+x3Leuz1gkU28Ocr7Ru7QEXQV4\\\\\\\/5\\\\\\\/\\\\\\\/xCP6TQ2ZztFIwjzR1d4n573z+SYC7TkZXHKrXvhMNGtQnsOOZCeEYcYx27GOQECoWDmlbATogCu3lREg9dID+igLmvay25sF\\\\\\\/AT7iC0dqovnE91Wckurz92ubkiyJpyyVC7gflHBejEEZ\\\\\\\/C8bLjUhBVoB5jdSIV97v1hLwx5+jsSXzfzg3IYEG6ZvgwJIZjN8BExV19Y5M7vHn9gvLe46iEss4\\\\\\\/cuRcb\\\\\\\/SZ\\\\\\\/AJjQwR2mwW9q7Z1N3spztNAvVRd1WqvG2Xje3\\\\\\\/zor+WtfDtvwm8zcvM1ufcX9OShwIpBOyURil9jo4+GLrIuYwVvMEUtK0A22AAm+syPri91i6ffJq8p3sf6rJdC8+oMtL0DfHmVwX6OsbYEhj9FsBChJnL+xjZT\\\\\\\/9+Zs6OtsTP\\\\\\\/3A67TgMzmN0+9XaHqxIY9jBoQ9Rs8TkgIN+briW4Srh96UFKYhuILvvRFPRHWiPfRdnTusXv+uW54PK54ecZl7FJKJ2Nbj21Ss2ary+9lPvSAzgFVhMOmdWKQ7LQE+9fbcS43Jl75gL+ar9HfTAcEQltSdnkvmbSKOY8MhiCKO8AlT0WGr08acuegSoH\\\\\\\/I8FcmGOeeh9pWF36gVB\\\\\\\/2i+Ye2AmbFsJJxI2ovtCN42pxlwhzlTtvwmZOzNoiwAAxNhtXh9j7xx0qIVasMCH18BBJ8hvD7+M5es79VXFTOPqaB5fte\\\\\\\/wWZO\\\\\\\/UYFXCE\\\\\\\/ORvl3yca3VBjZbH+CEPj8K\\\\\\\/9ofo6WXKAguBhGOVBNYCrlQHATRxJpyd1OAmYb7XnqZ8EMwqBb6YW\\\\\\\/RsCmfFkRhDuGLXOAGOrlFzgSyZ\\\\\\\/GLDBCU9LX9dzMFd71QnltHlFzyVsOrI9S9JsGXg0EYZqdNxPVmA0lSHjSXpOsnhZnH6g7lwO1gZJjnzT8YQJs\\\\\\\/PW53e8NlLpVA9xvYa+Ud6MnVni2Lv730g1mNRyFV\\\\\\\/9xBsl\\\\\\\/qFl0+JPX\\\\\\\/y2X\\\\\\\/n98cWnwlSjI7F3aCT8\\\\\\\/qNxkTxWDbf6gv+jFAGRISWnhoINeW1QrI3nbc62zduURRwHvmb9EBOPgEEoR3DcymEYTr4TK36YwTinxRbVFReD+XTMHPj313dcwzQtMdGVWw+r+rGAYRwLgGWk23n5E06htJ1RZD5s49z6xM3ZR36QG2XE4V3KVLUO47VQG+QNkpzPcclrG3Iv42nY\\\\\\\/F6GFu2gRinWbsFEqCWmtRGXoJoH36dYPe30+Y\\\\\\\/f2Wmz5mAxwKgQrvjsnJ1bmAKDzaxTpey8KtXz0NLfvZKqvjAqsO+6KCK+CDcrMX+yIWsoZJM1qOcyJ6WIp07Lf6p71GjBbgV1wrUk34qTvfVcZ0I8zkpCNewONnpIHDr+wPitVB9ZmQZ4lWkjiDAFevAL4nZfHToNLlEww42YQAb\\\\\\\/poOP8WHEawjj6dADA77FlsUDGKhZUrWHTvjeDxX1tam32Q4S9iO93uSChtMdEoYnOi8YyzNn52uOcopPWqfRqiFqpp4wZVolWHwVF+fiskEfcZLKynSfdAsD4gwGWBw5LYcTo5ge1jLj8YvRLZ0coEPpzH2nH1G7Cz+koBZvkivMgSEVRv6U\\\\\\\/LKwIc3h9VpW4MLUpWz8Gc1bjYVFTYURBtWov4ppqc1goWqUAvvX795iMXCk2680rm5r7TwUyEkEQ6JRPzrn4FW09SCfZpi5iipHNSTtga89GNiUmT4yODtM360ypyAJNCjNL87CQCLJZHXnI2pQ\\\\\\\/nvJnMm0t1A4W6m7p7wqW6KckqAxHZgckO5jKHsK5SLpk2OrAU5KwDoHQMWrlxp0Dp96HG1SnZiEXP9iNGr8lFrAV5loiuGo49YxyXqgamFkC8KAuRD5A4gE7pCc1ChKYiKMaY+4PQ\\\\\\\/++ZZCSpINCLHv+krA+b3PWVo3R7wQf+A\\\\\\\/fhxVYw+ve\\\\\\\/QJ7rh5cVJPacZ4P8c1hsstRtFvHZQcasnei04jSXdwdrgxbh4z3t0RVGgOaMU6iocuvmFniIsKJBSUOGYFMPu8U0GdTpFHwMHME6ZsfvZmF0UUPaFWhH+P68BQx\\\\\\\/mdG0o0942L30gofXlcHfwiBEw23tJl9b26cON+qNApqF5uXe4M0RUeo4x9xVflrgXAEBPPbsYWN06\\\\\\\/+lzHeTxtDtXNU7ELcK5N1w1p16CmAB0CyWcC6790Y6FZ6NVDFFdgMFaqbPmmSLPadYmKZMoxo3ae8xDD+7t50WoTaKyqgsgqblTAlO36\\\\\\\/AUyJs6J4mViMChYXJv8w5A0EfPrycFpB8a1QFkGQv35vz0GLoyY27Rfp7iG72K1zPWJ9a5wdbzjpReScshAlVfT+r9Y+WCCLQFpHGEihtRSvACEvLCzh4pfhWeFagAoU\\\\\\\/dB37NQcbkDLjB4hRSLxOubMZSnxojkqW8efB+qvA5kzlPycfbnJ3J1ndt\\\\\\\/cR4gmnDC0avvNUE5T6BATv4dospk9PpW4HJ92sFJg9BJ+jR3K33SzbxfCpz77ob2IlkbIHHET+ciztiKimDR2SJHggg98+Uw+C1OFbhIxAVgx61davBsHShusx0JTzWaG4rhR66NXLudoJGYRcItoGbwW6nzoKubF9SUP8F8tuq0KVdh1b4Jv2LDVb\\\\\\\/3hpHUrwYzdXJTd+X23NBe0AEqAtmBUOiachAzu8FlxEKyBtUa9CQY+QwTB0usz\\\\\\\/DlyghKnLbqHkKoNudjm73MDFluAMqDKz8iEqx\\\\\\\/dG8MpXHmp+\\\\\\\/ubePELioLCmL1cgz2FA0yMUOWHc\\\\\\\/VdI\\\\\\\/Ivd6zf9JwaAMSdff0EYvkqZlIvulZDAbUGEFeIhe1\\\\\\\/ORuHppK+vFFRIC2WTPpHuKFvdbYjhUnGvND6P9stLPqjC58yEEg\\\\\\\/Ozjng6FJABbcYnmcAQFz+b0uSyHKgHX\\\\\\\/VzWcH9Ot1o5zw4L6pT+E7TDu7biUrYqfGdaVfDw4KM+NUkWTK2nGEYDp2wJexMIWdBrQhkh6VFfX7w\\\\\\\/eBU4\\\\\\\/0VlNWo7MyfgRgBRIi2uugSC5OqG9RCPX5ncO9aGoNE6kb5SiSEmE\\\\\\\/BicX6kh0Kce9dITSY6Szkdf+NZtH9BiXuIVheVSV2UKDFgBvR\\\\\\\/1IGS\\\\\\\/wFS7DgE\\\\\\\/+kPXt4NGvOLp5QQEG4oQLlowq6jGEAGi\\\\\\\/00fsnay+Cwr6Ant\\\\\\\/zzYNK1oJIsUOnsXlsnyS7gJ\\\\\\\/uybLqxy5uIT8FTWFkyfpyTs9pNhOO4GKXJBJiuyD3FYJuJ9ZT5jk3kXynTozQNzlW4d0C1h\\\\\\\/rXccdjb7\\\\\\\/m56Et5oN77wcms3kiJkK1pTiUdpS1OyH\\\\\\\/dOslmWH88bXTBHJVK\\\\\\\/2jag7G4IDZFXpcZuvy8NkT4hwq7F5P+JIwlfr\\\\\\\/oWBhyFzufnuXbt0G0KO1mH8qFYJaPY2s20ZAFmV5TVtVUn68ZrOSpVQTxsPIzmwHCmuZzKzqLXy+SluIaiefHhCHuS52g\\\\\\\/jri7TJNd79Wm2ZZQWHWraCafgv2frjzmAS+HUhrtrfG\\\\\\\/b8xhfzPkYVC4ebktWzO03\\\\\\\/ODpNDy7GmILX4Kb0GsV+OAh\\\\\\\/ieoNh8KwsrBvvD6MHDSGo1zWLeGrvZ5CguDC6YaPa5K80cVLlL5gCe85LONQStYhMS3Oc+6LyXw+sHFP7fE8iQtMIDKa6zYUCGBDlMv46BlSvF8J7+4wjK1EkshaAH7cNzf1A\\\\\\\/0TSaR1j\\\\\\\/OAeiOZmm\\\\\\\/2CErqL1lkwPui+fh7uk7rf5MdcrimNoO46ize8CZ7Fk8KTnmMtUZUq6q2iTju1yTqnOhApOIImtDkVi4tt2C5TV4dICaKp0YWWplm4xz5PtmW49FzHLnWhDmxrlwuJzIWnkUMNAAjpPKzz2EJ+G86ASKM595btB2x5Y40Sty0NlmJPFexOZg\\\\\\\/JCGJrrTDVEYJsyVbi8XrsKxLbgB8YwT\\\\\\\/2vE4Op6Y40SWitjHMbup5UnLuziwY+tK31Wn9LnZ9OHP6jOtZGOKbNhQDslojMP51khqLJIR6JsiLm\\\\\\\/jsKnzzNo2t9BzmXDqoWHqthBFeelmPfnYKaRwUpLv8wdfIVXv3SEu6yUfbKtr27Eaue1kvaO3bjE51FT0vCMa7ryBYOsS0q1y2sLo1HgRFdXuygw9QfPPb3xoOmKlz6L1EO8O\\\\\\\/Vi9bJghfHd37kr6l6p\\\\\\\/5DWwGrGPkXL3iWS92ZwdYck3xJqn\\\\\\\/d\\\\\\\/BK9v6ryCmY4fJWatj6j8ddjks3NsDJxLAEmuI8MG2WVVasN\\\\\\\/+KcfCIdFAKRJ0hfPeeeaUmPfI1vnGgvkFadDLUiS\\\\\\\/7hfwNHkd0xAGLT\\\\\\\/vfg+5wczegY6cJkCkuSgd5Tr4br0PZYUZ6WumMBUVXc\\\\\\\/+AguOH3C7x2nQOttSqlONuR5tn6z4B2+GeinMh3VfAUtWaGMAi5FpYPvBhkVUMV\\\\\\\/tNYzKxkOUe6A4R052VRq2taCnKtgsQpAcp8ItDVRVQsBcChQHc2Pxh+sEaSPdVT9NTWJG7jR7FxRHwucEco1VWE730HJ\\\\\\\/NYVVis7P6pJV4E3S0vOK4MBzujrTXoshm1rZj\\\\\\\/NwOl6I0iOKEniQaGK+hloJFbDyBtUhVUCtt7VKISPgwggWf8meZPUH9+aVv1CNrntZuJ7lJ+gV\\\\\\\/t+mg60ruwyUNWbwZPP9SA7hLMapS7NEPlDUKdNMQWkpOyCu9p1MlH8zdW6\\\\\\\/j5ChtaI5mA51LCZQNLdvjPdsfR12Qmla08up36z2A2CyhqaNoKTBPS5d1fyRhPsm8Qn5yEaikKKShsogDNKNm2pOShF7wQqS8GtdNitnAn7TI2unMCIuZiJsrPhrQaB8sGyhdlNMTB7BvKQtlibSxLLb\\\\\\\/1mycovm\\\\\\\/VtYAKUmEDxHOdmVvvpxKiUjbpaqnYOaaqYyvdRzKzxQvw\\\\\\\/OKBSciuMtZO31zzNtR1AcCqxUzwVUZNutfHoP1jxZLDHEVkWmioQUp5PNfdrugfyRwpNg94yaH3fU70EKedA19tzDKJS4hTE78IZoTEacn+neM3EjGo5C8ZDOS9ZhKtA3xlTrBPX+9pxwmWhoF7ciaLwWhO2XAxTNNYdI7lGd\\\\\\\/7jnktJGgg\\\\\\\/tUJlfO1S8mqjNUumnlhn2ARCeTsxkC5+0oiXtDsh4oh9j8JufInrU3PfZkSTU71BMioWTyUs309i2Rs9YPGhASakPf2jlq3BxWX2QLsuvefjmRSDPMR\\\\\\\/\\\\\\\/Lplyq0rg9x9uyPg15+Ba7\\\\\\\/8XxyCSCEZGeVl+o1NYjoxLqtkIgFCUYUzNj2PHTRXIG2RPbBvIKZYZ0DUWAwv3rTO\\\\\\\/FeDD9d\\\\\\\/d3tzAxNuhO1sy9sbwhXE4OoAHLPnl6Ex0w4r3tdrAbrGjV1r2uqv0vI64i+eqHAF\\\\\\\/\\\\\\\/JYSsJJVj2fy+KhPrn\\\\\\\/2xaZAMCWsSsJP3DuYJK4j+py6VXV+78s9BPdsOE4qqqTXDq2CRsPro8T\\\\\\\/qDF7G08FHkg4egtF1XdWhx\\\\\\\/uDWPryPnVM4q6jWr6uYxtBPNt0W60XhDSyRmnGrpLRCV\\\\\\\/IYlTg6H3K50ntLV3J5xYHkyzRy8+DGMy0MV\\\\\\\/L01zumTehfWFPV82UZLR\\\\\\\/iMSlic461stbb37We9Wwmf7VRHKm0CUSEf6DKWYhiiaz5fLE4lXrdRfM8CgUoKRKEnFKVYDoAdu3\\\\\\\/TlHm+0OdhptsiiEgCBU5DIhfii7wtSCd9NL29rxXj7G9xZD7J8IdtKcZv6uFuzfkkC2MWLkbxp4S6fiU5OaQkEMp+tiVvNNqtSc3fSI+ueOViLa5uIPCP5JYT\\\\\\\/rnL5P35aZMEcAqOfYShjl1SEFOyuWAWo+IglY75WvpRNggZAhX6IM7PbMZU27vC57QCHz47IW6ARzOL5UwQh+ewnnuI6edzcm4vElL4N2h782WI1fJYpwTJp7kDJ5MYhYzYxtDtQqr9iLlHNoIvnNahk\\\\\\\/+PMwKWlquT9lS4OKp3LhP9l2TwWzg0pRMgTo414ITM1lGeff4+6jlsjw4YyzdjbAuXkDRLsZM8C6p1y7HLwKC9Xdfwzl40mBb52JQY1Nb9OCrQERDRGynrpCRuXyg5ICDxAREvsAqn+1eC7Ylfozexe7KghJycdiQI70zXdShwCkST+tU\\\\\\\/UBEo7pV1jj9IfoHyR4IVEBP18kH1kEzQMu1aN41QmF6kD6YlX7iBG34tT5A7KvwoeWEOUCu3SbvNtnei6zBOxbUhQYUjKYFDa30ojf3TpU\\\\\\\/0MkKjITtx6Vx3uybmzR00ku\\\\\\\/\\\\\\\/995huzWbOSiX1DVGD9AD5zI+NLAMtvjVlK6xt7xDqQVU5KGlWurc3i7chNGNimcMv45Ks+5biGhCkQzbsXTF5p1t2H+GHshGtgVq22Iv3FuwzwpFGxOIyUSC+HUOk1gDMRc5HDr9mE9LoXGEolhfBnP8h2kwI48Nc9xU7zd2vO3fN1zoUtKyZ2cevdXfXi+xfyVWZB2ucAnMYEfKgdTdpSE6HqOCLErF5+0ZzgiLkD+\\\\\\\/1wD0b7Ftc++h51r3BFDZWd3+VtBJ7L5Ha3iW8yACPlXoRXVTZFkJw4lQ1J8LAaWF+Jz4bUIFkm+WdMt438hQtVzl8GUwywwCiKFUc1bRNcdKGmNl\\\\\\\/LSF1VH9Z+PMQJbQSTV3jtJ6q6579pRSkWnbTtg0\\\\\\\/hd1+5WwDd5nrFP637s3dJMozgIZhc+3IJTvTyYe4g95N36c3\\\\\\\/r2VDpt4RSZUu\\\\\\\/JdEVAhOJ6\\\\\\\/WhiQUKqC65fGImdXdAbOcOTAP5WZxDrMQXsnMOxBHvbNrsJpOaeMx9u5Rn55iLg0hff0EVNFjJ+MkBn0Lal\\\\\\\/Rf4xOBVO6WOU+tCa4bA\\\\\\\/yeQf4uc1E4mbrABjgOb3jZrUY1R2VPem6uYrLqFUSC32izPdFXYkzAWZFUgJQuc7Uvd5eQ7CXgfbBNHeS3mBaTlpM1R2JpseXTYT6ZSfv5E8J3LxK1QK10JST\\\\\\\/4D5Agry9oSrIHgmUuMksKwGOO5vSqXPz\\\\\\\/CUhFz0vdesv3huBiSWTs1meFXQNkUcvxfaD2wWy0XNLub2\\\\\\\/dRaedY8bFeD3wscFvhrEMK6urGpHKnaehnyjbgM12tTVMhKQHDsZ6OLKN9tuAyJ0tB4XB5fK8LWkrl1QMbW904l2qKp+IhMrMmqJbDhKwWrqwEavZOWM3U98mffbRTIzWbTyyYb9PyknKUrjZT3GknT5fgGimGZRPhnhieWft2dX8oE09wn7M36lBOuMKhwQozT9yk4wpJzZ+3Dr+JqA13qwmGcL9YlLMkvD4EvgC+j8luDyXviiqnT6szhfjmrdqW1tCVdJQRUflQknCo2C5HWTgleoI53Hr0eMHRFB3QBtT29DkVsctFpqPGzK42WztGS5ZxSYQTvcM0ujaImtgJHzQwM5WuL0SFjkCIhpDI7C7GiEK6agQXQDEeJT50u0wk\\\\\\\/ThjlA28M+UXdfqvx6nFme6SP8RXWeuRHkZRJblwez406AikRwgyUh\\\\\\\/esnXoAvPvOvAw\\\\\\\/VDtDn6DVfZFRXwZ0ukd5i3vjbcTBjIYXb7T5oR5ABrxugbttf95dsgTp3vNb3diDaceGs9+wW5gV9MvlSNsoLfBb8duR8hk57VmrDzTZEDmmfK4g7NQ+upVU6gPV\\\\\\\/WE\\\\\\\/bM0aswk2LHJ9jWsBi8uCsH88VKDMekzs9FIso6x3aAsi1l6dXcGn4YGcRvPnnZLL7qryipSvwAbdcuzSvSNf9WcFlZXeMkXVAtZQrDkMb2ozYq4kGkBPNM63kYeczK\\\\\\\/2Zd0cQA9SlkzKe\\\\\\\/p9YYZKH12TbSNicMCuiphAeOBLTeFFWTgFwqGcUzLzDs\\\\\\\/I8kPVak3AUn1Cgh\\\\\\\/5mznwqiNLCqDc4QUuSOjslG4Dh+kWqUr49oSFehnp6FebSKvGxm7+Ad9jCUjdwzldnS5GGKLSldTgQqKJlws\\\\\\\/DD72txOak97ULgK+HfSpxcpcbs4wl0OvHJqZ\\\\\\\/onDyrB8SkfPMQwRm3dKYk1hx4zONGqisyocPOsAVL3pg04UjaY\\\\\\\/7NH4nOu7vDC+c6pWBJ8DAiAA\\\\\\\/Xsd+pgw1n0wtm38IwQb6ma1bqiyZ8HshTcND3MkzCnHNT6\\\\\\\/s++bHJQ\\\\\\\/1BUY83XxA8j51kEOUQjWSQ3msUxb4Jmq137jAgaybbt2vGFkQpEImiNpZ+718OUunoyOyB\\\\\\\/ubb0XVYinSXdMOQbljCH9mt3n\\\\\\\/pj5ZRbDihX08lnvoJkpy1j4b+WyzCa4DFWK98TAUxJTW0OXr92c5uNU7\\\\\\\/9TvZWj7dmLLalh88sylW5qEWAQQZgfow92BMC40YJur3VDMPOdm+lwQtLXNNHWF3+xAiS2obMrpLLhkDqXKMV6aevmqAI5X5AeYzCbcIz1fNTTmygS8kGhoJRJR8QF8tBnbsSysZyqwHTdpX3oTsWFrTA+sgPBb18OVu9QUkkvT\\\\\\\/EcxASUr2FURwdsx61wqdmhKyCWM5TnEX00ndQSYBcF\\\\\\\/WeYblYIbSFj29GeEwxmKEgnVNl5kLd\\\\\\\/FmCinBLpX6t+M5Hpl7C6A6C8ykVywSZHZG\\\\\\\/+o02k6Qj9lMJkDssogdz+Nq9ZAJllmuUUHYtFPf91yE4kCR17hUKzgP7bL+\\\\\\\/IcDlpaGsP50xBW3IkAfoNC3YzZonGCdku8p0GaiKFrWebgU3T7Jyfk1IlZNXtziAKg2oZjXzmgIFRC2HnRqaUzDpp\\\\\\\/z3ccTVV9z6Bu0o+jrAE0sOlXg8VZRiIN2D6FGf5SD4X16uluX3UmnKPIcw5kZED7GD6Q7fn\\\\\\\/VJBbMY9fE\\\\\\\/8LeXLDFnk+PNrlI4SUv+iT4yAxqXoNL\\\\\\\/8+3MxyN3ARAKmqTHTjsdbLb9opn5AfZ1O2NQiLWDKTzmGBih9ejNyWF9SYF5jvchJioy1VY5GEeDwwLgHBsfZmFm4Eeh222tXiwD1\\\\\\\/DWsbuldFBecyn07ufJMjLWVBycf7ogqfd8boZWS\\\\\\\/Kg4Nx\\\\\\\/qmpm\\\\\\\/XWqhytQylozGikrFmQeg5FXvzSvKMru64XcnV+tQbUQ952xvyxTQ3KP++tOtw0WLQOhj5ok5ukkpqEP7bdHAzEIlQJSj\\\\\\\/lg0sFozqPbHIz0vg4sqrlV\\\\\\\/UlXdJe\\\\\\\/To4OE8+qXhYR\\\\\\\/Wx518ltt9it1ennNvel\\\\\\\/aPv3Q7HJl0VV+Crerv5UnLRi99PNzYDwEHgzQOyjYff1WcYjv79jEK7CaT4l4Bpy7EOfEbPNPvYNFj3Elcd2D55U6GoT7s3EQOP1P4O+JrSm29FpZssZNCtXzu7cTArcgksTfO6EIuYE5od+IDf\\\\\\\/uVJgbvyc6\\\\\\\/VNE7D2X37xs1oGVzIhyweyJN2C\\\\\\\/YBPpr5atLsXSPU5dFBy4lMbdKga1BfQEyiiUvNT9iMjkxz0KfT5xgVpm4XGalmjIABdg4Vil\\\\\\\/vYgKnBRIMxOcNKgqtNRM9HqatYGt\\\\\\\/mu9a7WDKmdPxZ5axEzTTyBxKlbagA6n+CvkAtOJjCDOdMpKATwXlL5K8RAZQDlrSNxm2JpC+5D52VVb2+qIrCnJeBcCPCfsS\\\\\\\/WkWBBSBuCwGpfIQIF\\\\\\\/9KZf6VIzaEBZntna6vSWNDHLE61g7Ob8p1521Gfybdar\\\\\\\/hHRddSGT3vMrvOF6VxfmK6d4YGCkBUDDxaN3APu0iRl2dKZSHVv\\\\\\\/sNPMdXv8z9hSU+lPJoBJ4LolJzVi3d13+RIZQ5WFcZvMVVlIPQ1OQoX4UI1kncNGG0jsXsaRF\\\\\\\/Q8rVww85InTdZ53E4jWd9E33fy9+etzo9efbUOlhMsxXeu0RrjZ6033v1nFyfr1VZuAh8b8el5q2tQFk+T2e+LTa1HG81mVqADwvp2cQGc3NmELem1FOxJR8WtmjZAjfvuG6TsfVtc+a159TeRBHCZDF+zPlRYLVg7DZPjVCRLvm8uT65H1Leijxom6ow8HAswZk6cHW30M8V+lXXQl1hOgO+wmMqys3J0RhkEmdt3tt5rUG2eolG24yPUk3b3ms9qZxQ2\\\\\\\/WnKrc7pPoX6TVmWsj8YJLy4XyPqV4nhIzs8GGpkV7W1WSi+aErs6JBkcnY1hsDi75F1ch+8\\\\\\\/Yjt+0oY4EYysV4BTJfXxnZ+tj8t1l6EXpaC1KbtwB9s+P1QOsLMHIRzCLDO4rMTLffbVlk1jMmtt9lF2p\\\\\\\/wSxIq01UlkD5wrB7k4AKmexOPGINVMc+Zkuw5CMXsSofBEL02gY3xrbAL6qZn3HsdVZvCm0cX1aUHn8mpSE79xPPJX4aOFPTNP6ecANU3VskhAOblBvmyEyALlEa848jY0owZKkpANCbm0fFraIiHNrzDqttdV79w6c+cI\\\\\\\/gRbKunEXes+T5ArlTr+r\\\\\\\/UADX+oBpKwaGCNK8W7Be+Ve4KCUmdIFwXAQkalY7BHfNkrM7E\\\\\\\/Dxhrcgu91rceXVaRMlQ5JiFKn+dP45jQ1ZveY0hnHW827qV\\\\\\\/UbkKIGEJ8u63J7Fc13TJW81l6KRu1ejZXtm+uCHTuONSUbhnKxFt6q2i6ZzFWI9tSjPbmA2yxt244Ju7SvwBXgo1KsLMGjjMHL4DOcVMHtl+ME7qwAQi3xsIUtQkwzcNZqh3A3LHU1hII4l8LKT4pnPjf\\\\\\\/T7syupRWwXXV0zn4Wv\\\\\\\/1v8rY6bEYGowFttPrNegDXUR1pHgAvc+mv5vRDHq3mzYJpxJDzz2g1doKGsVojvZQeWcb+nnmnflmapA6oKZrziqUkfSvmRXeo05fajTcNs1iJ5L7yOmV6jQlz8RCCuBnDgTegTEfccXTXpc4XsNLN5bFmA41ewWptXOCBIaCcOxByZGr2W1BVpBMcyQAd7lNheyuTOoskYWmgvDQwXRgJmXt9vYeW8kqeCbLzedV27rxSOjXVWpZe2bMKTBGG\\\\\\\/V6Ax2+mhUlRfPXUJvOghIZm512+IQq5i8giB36I7ctHfFfwYjytFkhPSqhBLtVOr11XihnkLDoKt\\\\\\\/ECaJWiVCOUMC0L5+4ZHoqu+ENQ5\\\\\\\/K+QXIwS1XQ9lzwJsBNeu4S7DHp1KgBYHxFjTbbdMOokaWC2pKP5CoBCWTnLsA4uji74464RyldrQnAfVr2EeuMnef2rdLyM5N15OM0rYByP32Xmw4kS+WqhA6WTnJ8rVd7x9mi9CtWTIAQQgThY83mWdYutlPWCYWyl4RUwC9vfIZtisEm+zt1rALquXa\\\\\\\/n5d1ShZVmHDOkevtMvRKUDTvR1\\\\\\\/zN5ZC2td9t\\\\\\\/fAZ+Qf\\\\\\\/WdlwJImIlMxWOW5+fxFVYXd+KUm6uHzW5InOIy4GWsIdW4yQS8m82zBBfCQ0zcTkLnR9IYQerqq9VRTP0pyuVrKSutEHnNdjoC\\\\\\\/h8WqRIn6gotgK4ZTS4i44pGRtrloRL71xQ\\\\\\\/NW+DfyxewkZfnSAqvhZuKETY1i4pXqWT9B4KtjBjk+lE0Zqa99E+gaUlFya1Weil\\\\\\\/gU5HS\\\\\\\/j7O5Bmnsrj2uViyAPlWY\\\\\\\/lrr4NCzldPr5vacnVyx4C4cpGz741qsF0Hh9C4i3khCCDT4ZrvKFYjWNfSPQLv1Qj7PSoXdveUEuygiDmdgAE\\\\\\\/EKo38O9JZFcdDUl9qpvrTCzgwuUFaJ0gHr9k2hWbE0TeimKKw5Owlv7DFAV8fij4fadQGIJCfvcZP8l12\\\\\\\/tbp+XG3dyOygUa7UompJRsHBpFN6eFDpDEBWaPy35RUcg4NvQGoW55Cp8KQQHqxFvQlaCVszUrTj2PvEuoV9Lm1WEAA6BhTejbLQKvAyXEPSY00p2glCC124mniOaol7RXElfuLqgNPQPSwCVjbXgmMfX\\\\\\\/c21nm6LWMskVlAdfWt7ktbOLTCVixRiBEbOUgM4v8Ap44V+ucU5dyb9dDqNpaAv+oIAm+3pVlGJSuBxAIlA2cKmcuauVqJf4aKCo\\\\\\\/AkSftDXZM1rSnrjdXyXyazID+VASRoiatxABQe7AnSsAUOAVJOfC0Fik\\\\\\\/C8AvQV969jmI0\\\\\\\/hzm9UzPjC82fQ8zN78SduUOLbP8lvhWalaHRgibjSha6p+zRhXk9\\\\\\\/6GwNTR65VMGb4Je2LCOav7yPr2vxCyMwoAQCLF\\\\\\\/NeEQ0T5fEme1ANrvedDRZ1w0DJFdaCszww8pMdQbFs6VtLPRPViRa21idzXRwkx107wZhkZATd6CGjMtozYLqoENc4IQAqZfUn4GGvvR52y++\\\\\\\/\\\\\\\/Fq4SwqUDRUAhuHmI1vg6AM2YUkLZ7UiMKzPkbKnfEcJc\\\\\\\/lgvVnIaqahcpcoA\\\\\\\/tHDmrzBk2xwHstpQ49AQ996b5SNTAL7qWr8MTGsRaMvNv1ID2c8LIoV\\\\\\\/qZ10PisiyEdEu10CmxVKlSaw9QWXOaqxOtSxPlrUWIKUZlFS6UL2vmY9ApMBM6C5rVEpfvmM7HT5YJiUgXzjEjxywSIyKtgN3uYZfmOrYXkPJOwnuEIiGegFhvK\\\\\\\/sR8UbrGov5Ta3DwakHH29wo6NTALUit4zqbRF6QGrmzL7iTcfdvUspyn8zmtiqU3NX9UM8fIqo2x0LGPsNCW7\\\\\\\/7OJvnpw1Tonk5y6gDYiTXdzyTGjCiwHeBA63iiRYEb3tvz9XbEN+hUD7pRYyJirbi1KBWvzpoQfDcr989bNlgfwfJwf8XmK73T2ScL0uXWFcp3HHVMz8XTWZujrVcyPotGwzdidTt3e6agm5iLtneV1TofMyyTZq5mHih8O\\\\\\\/iQjfugsPpfNSE\\\\\\\/r6ooTw7kaDylZCSV9H2XOkbhQ2rahRmMgVDICnPYHBywoOoFDJOG0E+Yps8GeImooU2Jd+tggsat7B1mPgBfovLG+L8oM7XouH30dTut32nBfMGusq\\\\\\\/MYGzChF8M9lw9n8kTYQTJgS4H1Wju5f0kRC8E0ZBuRpbhcfqLeaN6AQ8bcN752wwlSj0gAjgiEfrq5jgzbJUAdyYE8E1audkQnaubDYLS2xKx2KaC5NJka7aOAD\\\\\\\/hbxPeLFPfXNE3YAU9aVxnA+ElqEBEH4mtyaq+3A3e8Fq7VUSS1oDmzGVKuzbgZy5933vQD+fOqthsmS\\\\\\\/NX67aWUS8QmXiCbk8SnXId3rnPkEEvHJQZMNfQdYEUfNh3an9FqqF0V9ut6e93aM13dffRcEe3YbE4Erouip61RTO9Cu+ZlvjBYK7pB\\\\\\\/fQN0rS6BVvPjdUBB1gzvcMxYdq9dDJaE5mfug\\\\\\\/qr0utXWAFnU2HIB3OKK0NfASlyyygwS+\\\\\\\/y46PLbrlyqQau8qnBcinvVhgdp4Kuf\\\\\\\/0Wg\\\\\\\/keHSQuN27XDEX6sbOCsObd\\\\\\\/i+PG+gIkF2husXS3owVHNI\\\\\\\/+KWq0bc6O+IwQrmJNhRaKcx1a6an+gUPWTCZ175hoq0netaM6oJwVDauG2bsyL+0E0LgWNBsmMsbPbGhyllL8Ld2gseL\\\\\\\/6zlbvrLFmDrBOOma7RAX7H1ymVR4dIB\\\\\\\/dgH67QAvrYkeCxfOCL11AMyYjkFswybPjD18vt1e929ana\\\\\\\/pold36u2\\\\\\\/sAxp26+hzvV8Xk\\\\\\\/f5LGBFs9SxshbBBgsphYFF\\\\\\\/h6u5n56LYtDDdvAQqb83gsw2TfkYH7rnVDkr+oYVx9uKeS9ZMiMtfkvgRU7dfIw6f+c6Emn1FHcRMB8nFya2jMMCxBLKK3DAJqfD\\\\\\\/OCMacBg+7msfTMR\\\\\\\/BXXp0e5E1td7jO2VMITv36VNqJj7gHT\\\\\\\/TB2JCJ9JIzyNAUQK8NKwyDNrXNYlVE+bQScp44\\\\\\\/BzekYCtxav1+VMF8z\\\\\\\/hJqYqVH8x1KWKAdTUVzdQaI\\\\\\\/dwTS11eqlQkEOLfXL6oL62J0uiO6mb8WNsHgWdba49beuDJhyKfOWNfWaDu46EjFex7M2zatLwE23dQKMDYsVeWEQlDlYSAyWKwkwoa9ePY48Nlz2ny77mUBjHVD3kBz5sKh5S3wGyf\\\\\\\/ftrmiUk+RkaXFBStD35YKiSyr4A\\\\\\\/hoiJ1aGUIXlhD3pXm4CdPhBxmnv7Ru+pR0sYkdkK9MjUiOA3AAAe0jF7dcQcg\\\\\\\/GcLnWpkb9uzigtLjlCB5LjQUqaoHjaIw5IML9gkbuO7YO2XW7uV5mJkWb21pfs8yaS2uGTwYfdW8Ytv6d0u1hwSch0+W8tHlTsUZgLSZl4+0LRZoKJ6S3neKje\\\\\\\/NIXWGemzW2jpzU2xUdKT8xDZjF5tHxhkhguthjVUhUyGXmEoQ0SGN6LwH4G4OE2V5GDjfhSCOGBI2Bm\\\\\\\/DtfkLJ2+6V8LVvlZv5UelKQeJdojc+RY0V0Uem6jAjGNFb0q0o\\\\\\\/7weNAnguz43BrNqDtl2Q9sE4V8l3SbjKxa3AM0+X4xbyPgms1R98LgNs4fc2OaSsafashjArZNXlQMJmXyC7c8M3GttjKhCy7j4hRhD7LSEw1qsk4+pA9m7JubO6VHo4ox4EMBJWENY8alGdvDxprW03TVdLPqKsjdmzmvUFrvFXK56q2knZM0\\\\\\\/v76\\\\\\\/TMMTssFkhU7oL2aW0w6YHXbibAD3AaeqU8zNnk1E2q3gHUBcZbhFsUTkhUW3r+JO19LEqiR98Sxl2cHo6sovQNphuxetjcVmlUx+U0J22HMj2DVXJbT64Iv\\\\\\\/d5L7tYNnXiXyU4sqjLOKCy+mb8iLlSCDUnbHpKEYYstO3FQPris+\\\\\\\/F8nf6QfxKvAetxQS+MRHZDGJyv5+o5dZTa+f748FdSjWGUQHkUFAIDIUbI248dOC9fusi9frY8MoKVUUHPueZ6TBzcErB7Zcw8K2sSPy3A6lXGJJETCpqKxBWDRe8bHochrk05AYU3fdXLUWO3PCYqDJPLtF2hGPBihH5mWKghWVRhWjhuH2WwSY1iyqyI558udHp4oGWouMCpwijSqEFbZvwA1rvmYQ5g\\\\\\\/NCeFxU3vWfFUtEi7MRM3mokD778EgJ1\\\\\\\/ZkynkLR59Wo2akPrg1iZP6GoFlWNNNpDXsjgCA+yogdOesLLLGfavYo8i+ZPYZSYKv8nFCBYjDfyhRT1fR+hjF\\\\\\\/n\\\\\\\/CZF4iI6ZZQll6ogcYe5B1IdJUzMS+zsMJuFfYttzCsvliPddyfnoN7XxqW8nfGGYIBIvUji3MT5TCW0tps4dzBohmVZAj9rbZL9ZboDAFLCUvJTTArO1MPzeDW8X88TP6dyM1pke2f0Wj3naMmtjHuZzdIOxx2tZKi+zSltGq0v0HrgUstmonsSf1I69V5Ye8\\\\\\\/WfR94Cz8byGZAS0Cjkv3GAygbVn7Oxvk5OHe69pRp6pm\\\\\\\/m0oQCNwcDtNeBNPtAjW9ZGWtODLatPoMq6vD3WKA0Oe9hNQ9ujA8nsI5epMR1J9gL+dYyyuBc6KclkyrRdEBD1NPY2CKq+8zxps5yb9c7iJnlBybYOJlzCPvqHekhBJ+EaA9VwX+c9BhYhwkfp3scx8N+lSi0W8VzhBDj2rFeiQ\\\\\\\/5OEzbL7il\\\\\\\/Sy73a6REBM2gYsBhHA5JNkXyQ8gbJhrfXS5WKm9ENcV5Dkf9F81rihyDhR4syrqlrS\\\\\\\/8pQMGHDpCVRll3R03Ck3j\\\\\\\/kYY7McpjFGTiIDt0rWE9Rrd1oxKh4XM+eOodwskBGDGgLc\\\\\\\/xCvzMW5rdyzvSn0v8YhEQHQWll71eYFxFyuQemeygD6thDHlFgiQYyvx9igOLT627yGLAbWMBo+\\\\\\\/xJBTpkMSISzZs35WR7d\\\\\\\/6ruIVEGf4zYHsGV5wUFpvMy6P99KcmEGnqf1ddy7TRc5QISGUgT1Az2yD9GXNKD1I0hltTPUoET2oCjShjRz5+\\\\\\\/CjvyA1cUHq4l4vAi3mvokeuX5RwkqAZzXb9NMzNGfBEqn5c0TMIrkHyYilm4vPRWPEziM9jp+6xH9U\\\\\\\/GEL5Uq3tZ1212F64SD9UIrpLJpQ7\\\\\\\/TAx0InL84qCfSpSZfhJvHZ8+OKqABc5kNkA2faxiXet912qBlp+mMuHA3Mr\\\\\\\/R\\\\\\\/l8l6aByHFp4a6vcgXA87JC7DwIJWJQ05eTlDGVKmLwTTdQsZUa9O0H8YOE191lo8kjKxm1hwZyGeMb5o4hO+lBAR4GSt1BojSbMXtWEM7l7BXI1SyEiPLD9GyUxWMiFKY4TpO\\\\\\\/zHy\\\\\\\/qVH7JIr1llHD5CWqBBgnwZpiYOGW\\\\\\\/uBQPEKwRP1b0J9aMcOT7UAAiMNhhwodCjIRLR2FgzwvRfSlFht4XkMAi5NgJLPczTQYCOg6y4ODjoyjlo0vIvX6Dp\\\\\\\/qRpXmP2LfvVlE3Guj2jxgab\\\\\\\/+InzP0n82YT8vlvOZ6eoNvsHB0KnOKEy+SJe3DyCiVJB8rbN1Y5\\\\\\\/J+H3D3w2+hBwZpBhYubMGv3zYHuF7FgB5viZvc6YU\\\\\\\/oCRQlErcQAbzvv54aTr8UUJEprs66YgJAbxEXYrksM\\\\\\\/gfd6DHbCWILHWiFxmcV95YXXKcgrDaMIczCEncXWrjnSDiiLR8SQojbwODTKdbD8p7XFkgl8vPLyBoCu3225JUQAvdtNMztapJ0fbG2wUM+3xUgWrqPN8SAtdIcF674m5I\\\\\\\/wcOpNqd6KMMkrxOo+0Jl4WBX1AA5rSZKOQvtZcfVy19CZbXDuE+j9cr0WrlPe95IkXFlM1zYNjCLfnFfCBwP53YFtAlvpNmjoQ2rPc+XPwQDIdlEGEuyt9Ixqunzfvqx6dGPKr5XFGjYHduDDOeTGgYrByO0XP07zBkofW1c0LKKqBmuntPR5\\\\\\\/uE+C5v4eOodn\\\\\\\/eQAkZ2X0FMO2Ec4zSQ23jopAbZNOdsxDwIGBFeGmdyoTD+p6nP\\\\\\\/KNtkQq\\\\\\\/c6Ho10q+KMjrXeRGIedERuPQGkv3AKDMVY2SE7ljJ+OaVuQ2IaD4FABL+fycSnD7aNn+bzLgS+W6s\\\\\\\/eW5w0l1A0Vk4oVanCIJsviP\\\\\\\/yUuo+2kwOOdEohgPx8gMsde9z2VmuR2ytB6fQ7yTPf47hgPeztRjEndQMMxP\\\\\\\/U+BYwO\\\\\\\/z2zhYhtDUHWjxfMqcbVayOocCdazm4LHqseOD0uD4z5bapkdawgJdAKYIaSp2QZ6a9SEcoItWVIU+6U6JeMBlTSShq5xbguVq3rKSV5Vjtt8CP0IA9YsQ7rf8OoaDs\\\\\\\/TIfYaNjnLIp0qg21dCkKmbjrVwMpXBB3Akft1az\\\\\\\/qw2UuPv3\\\\\\\/ccuIqWnJZrc9wvyGKZOeuWtU74GNq70hkoPfCycUfrb4rNK0lVo\\\\\\\/fwAtf4baZrpL2E\\\\\\\/hWQGFpw1U0kLOCRnnojucUYvT5d1WGr\\\\\\\/ghAnYwyxxFGVbmFraDo\\\\\\\/eUXssQr33sk\\\\\\\/MM83aGG89yih4GU8mOM+pjvFhcTNK+YrJQIrLugpdDsWYhxkEkOpwfz2HH0NvF1iPOOgw0ZxpA+iXaBiXgrB0pOEv8\\\\\\\/HJ459W6s9BjAfIb9peFoOxzv2\\\\\\\/Fw30I1e2ek6eEMc29FMx00pib6j0O1wnQqFXqKin255RWQPu0riuPPyatoxzosjz9yl7h5jpG4w9ViaM7gyLtO9K2ZiIci0teT+7joLljcDx+7c+3OJMcrz3bq83AmmQzXGAt\\\\\\\/9Vyk7dhPaP53DLfqRqTl5tgHHtcT\\\\\\\/BZd1vl0FBaYawuvT3hJmrRL8v3pAB1eRdQw9K\\\\\\\/L2\\\\\\\/O5YcnAQIZovf7pq3lveJHcSpJ0QYUUI+dCPWgUxuLAquQ6xH\\\\\\\/vZDs7pGQmoLL4v0Z2P+4s4Y\\\\\\\/t0JFUba\\\\\\\/E1P60XO7B0YYy5zNnvMQTkwXn+eRfnk5gtdHbCmASVhhOKlnvA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ijJemlaRhpg4DiqxxRIbnQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022eae77eb993e33b00932d56739d38c765c986fd1cb6ba601e09c64a3d38cd7f7c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022KUEZfQIHOo+dRRlb071uqLLbkdKbQhQf5R19Ll3IH0xvz70DyBYfvFDxO918X0gbaOIGjrcfxeSgHm7Qk4sMPaGMXRtSfvWEB9WQA5+3A643ygHFdDhM1jpNP036vdoLhZkycLe\\\\\\\/bSsx77SBw1QVUpP49OBRzu4B7Cra1mzR5hMUufmS8CBvgqqngC+lE0Vr8xaLwDlw8DM9u5PvqDWn3RywkaD3koBlH3eNjnhq77jxCY\\\\\\\/eDnvedrOc4RcYI9pqxMv79wP\\\\\\\/hnJ6T9jNfp6PDXpM0z4YNDmHvnvAPl2uYEuRpZexoDvgQgYzwx+vcrCDjNaKDuXo9zfr3o1Qjt2zTvvW9HXZbczqqb8j2fRAuHDjGjkC9xYsZBBLHXRsx36XZ+LUNPAKvkaqCVvKpke6aA\\\\\\\/I4cenjPJtCk468ENfEisL35AYFaU6bEnW3CaZmZkis39P8Xz1N+N\\\\\\\/\\\\\\\/7Zm9glYXhTj7qh9U+hsgB2sifja4yBQfqGAmdEB6YjcC9zU+aebr39+aKJssaWED3SJLjLcllanpKEL98vKHrdpV88WwYE84Mb93aG8muyu9xCCed54eyypIk8JGgJVm\\\\\\\/q53ez8eTL8aJhld4bceOERnR0AD0x0jbzOVsdkPd8W50B+v5y97Z2lLypAk2NXecEw+DBfU1wX4TcVdbvoaJPk881uWinZ0Z7PRlXGqxtrYT31mIdaFe5izJqEZAakAyL4PvtwQEG5Ux1fK2ld4uSiA6vOSSTiA2\\\\\\\/SE7ME3vMW+AvvlR4xdTZgMLUNB9kD4jtZzK+0ogJ7K5t7GBNq7LNJK4O61+LqRHu4Zel1hPICbVaw9b+Bl2nOxlCj1VZRbkMN8LXD9ZX5ObrDa6Z6UDnGvDVH5zoJN30WVnlOdC\\\\\\\/LDOlqSKkkNRwVxDL3mKYMibVq2EgezU19XQGkNXrzxCbAl7u7ZUcniYxYTHklaH8c+1Ya1ATYBdjI13CaZzVDo+YTuUwG7u72M50APCRaLXkH1uZMrJtfgbOM5M+gWBNn4csvvRoP6MKb63lKCYiFIY4VqiPs9VcCeXISln9FIH3W5jDzxeFycDj3YZF\\\\\\\/SiUuchOYmiGpyJEcd4nOFq3xzaYpgwiwqn02XRq9kNzMzZKNoRdg2gKDIHx5\\\\\\\/gBrgnhlCd3CZwxccd3i9AbHyrpocDEx28U4x8R065w4pX4GfsRhA0z7wOOtpPblnI4bZMrfZwbzcAVM62XjVKfCRao+lgYwNM+XaLBYzieTLUxM5Enn9rhu+HZ114laktWbNzoSASeTUGPowYQQnpI6d2vdFWDrofbTIzCb\\\\\\\/kaGurMqoPNZ1E8R9GKXDx1THktYm3AgxcH59EbW6l4D\\\\\\\/k33q+6x4nx0gQSFdT9jkSrhBUpmkgWWU8yLKRoI9I8kKF8NAibG6szgcYcZbda4SDOFCroq8EWGzvhOx3DMsaWASsh6euzw3bLkxc9cCDfFI8n6fI31iQLhlAuwskE5p5hQvt\\\\\\\/zZ3LTgY2Bc0u2NUPVlNW90gQXvMQLS052tuwW88btGlRG6rBToab48jwQKnoXqbU+t0n0KMellta6CIMey\\\\\\\/riEfohLE7qTOigBJpXwZq0LPPmOJ6VXo2BeP5UQFlzVKKezwDptyy6Jjol4xitE\\\\\\\/JtMv6GrUtrDa+sxsuYZRld+4E7onSedjFilWfscW10cdLyAJ6Na6L7lDM8cqv2IEuy14o4XTIlGtzG2ldCKbrpNbi9UoLd3z96K5Xr4N2VVLMh9d6NtuDeyWCOD1eST8YgiESbkfe3xoxEzKWgGTvux8L8ybutd19sRlgYF5ykdqPy4yWznqKKtvFEpmirF\\\\\\\/WB+urBmCS0C9iNhIEQpArmjXrG2r1d7LKtJ3AVXd01GdQiBM905HxVgoWBECSWAKB249Yq88Fes8oEO2gqqPPXn9Rlc8580vMDp8HaRZAbrm\\\\\\\/cLEscD9VjhpYWdqdeu2Gtnf\\\\\\\/fMIw7ayy6Z3aSfUNDv3n24u7StH3q3mUPE1ZF9PTSGGeFHYKQZA==\\\u0022,\\\u0022iv\\\u0022:\\\u00228uwe2EX4gdbzGjGoy+Qq0Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002222c8c7c00eef88843d584e5455d028452f7c4e5781ef6d69a14d3b85c97093e5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460362 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1297" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VMl8sQNXNx8dWnrl-u7eBVTL48nVSjadBgcsphcbIHk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1297" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-30XWS3Tts82C9XcAP2W0hG5tlrpQBSL7TxvkqgPWVQ8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/5/1287" title="Conflation of Short Identity-by-Descent Segments Bias Their Inferred Length Distribution" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/5/1305" title="Characterization of Chicken MMP13 Expression and Genetic Effect on Egg Production Traits of Its Promoter Polymorphisms" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460362" id="node460362" data-pisa="ggg;6/5/1297" data-pisa-master="ggg;g3.116.028316" data-apath="/ggg/6/5/1297.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Challenging Mutations by Whole-Genome Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3517-5973" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Harold E. Smith</span>, <span class="highwire-citation-author" data-delta="1">Amy S. Fabritius</span>, <span class="highwire-citation-author" data-delta="2">Aimee Jaramillo-Lambert</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8599-2031" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andy Golden</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1297-1304;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028316 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460362" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Harold E. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harold%2BE.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20HE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHarold%2BE.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3517-5973" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Harold E. Smith</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:smithhe2@niddk.nih.gov" class="" data-icon-position="" data-hide-link-title="0">smithhe2@niddk.nih.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amy S. Fabritius </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BS.%2BFabritius%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fabritius%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmy%2BS.%2BFabritius%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aimee Jaramillo-Lambert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aimee%2BJaramillo-Lambert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jaramillo-Lambert%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAimee%2BJaramillo-Lambert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andy Golden </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andy%2BGolden%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Golden%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndy%2BGolden%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8599-2031" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andy Golden</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/5/1297" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460362" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460362/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1297.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460362" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460362/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1297.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460362" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460362/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/5/1297.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460362" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460362/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/5/1297.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460362" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460362/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Whole-genome sequencing provides a rapid and powerful method for identifying mutations on a global scale, and has spurred a renewed enthusiasm for classical genetic screens in model organisms. The most commonly characterized category of mutation consists of monogenic, recessive traits, due to their genetic tractability. Therefore, most of the mapping methods for mutation identification by whole-genome sequencing are directed toward alleles that fulfill those criteria (<em>i.e.</em>, single-gene, homozygous variants). However, such approaches are not entirely suitable for the characterization of a variety of more challenging mutations, such as dominant and semidominant alleles or multigenic traits. Therefore, we have developed strategies for the identification of those classes of mutations, using polymorphism mapping in <em>Caenorhabditis elegans</em> as our model for validation. We also report an alternative approach for mutation identification from traditional recombinant crosses, and a solution to the technical challenge of sequencing sterile or terminally arrested strains where population size is limiting. The methods described herein extend the applicability of whole-genome sequencing to a broader spectrum of mutations, including classes that are difficult to map by traditional means.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/forward-genetics" class="hw-term hw-article-keyword hw-article-keyword-forward-genetics" rel="nofollow">forward genetics</a></li><li class="kwd"><a href="/keyword/variant-detection" class="hw-term hw-article-keyword hw-article-keyword-variant-detection" rel="nofollow">variant detection</a></li><li class="kwd"><a href="/keyword/complex-alleles" class="hw-term hw-article-keyword hw-article-keyword-complex-alleles" rel="nofollow">complex alleles</a></li><li class="kwd"><a href="/keyword/snp-mapping" class="hw-term hw-article-keyword hw-article-keyword-snp-mapping" rel="nofollow">SNP mapping</a></li></ul><p id="p-3">The advent of next-generation sequencing technology has transformed our ability to identify the genetic variation that underlies phenotypic diversity. It is now feasible to perform whole-genome sequencing (WGS) in a matter of days and at relatively low cost. By comparing the data to an available reference genome, one can determine the complete constellation of sequence variants that are present in the sample of interest.</p><p id="p-4">WGS has proven a boon to experimental geneticists who utilize classical, or forward, genetics (<em>i.e.</em>, random mutagenesis and phenotypic screening) in model organisms. Although those techniques have been employed for more than a century (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Morgan, 1911a</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">1911b</a>), determination of the mutation responsible for the observed phenotype remains the rate-limiting step in most cases. Mutation identification by WGS compares favorably to traditional techniques, such as linkage mapping and positional cloning, in terms of speed, labor, and expense (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hobert 2010</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bowerman 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hu 2014</a>). Consequently, the technology has been adopted in a wide variety of model species, including <em>Caenorhabditis elegans</em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sarin <em>et al.</em> 2008</a>), <em>Drosophila melanogaster</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blumenstiel <em>et al.</em> 2009</a>), <em>Escherichia coli</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barrick and Lenski 2009</a>), <em>Schizosaccharomyces pombe</em> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Irvine <em>et al.</em> 2009</a>), <em>Arabidopsis thaliana</em> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Laitinen <em>et al</em>. 2010</a>), <em>Saccharomyces cerevisiae</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Birkeland <em>et al.</em> 2010</a>), <em>Dictyostelium discoideum</em> (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Saxer <em>et al</em>. 2012</a>), <em>Chlamydomonas reinhardtii</em> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dutcher <em>et al</em>. 2012</a>), and <em>Danio rerio</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bowen <em>et al</em>. 2012</a>).</p><p id="p-5">A great advantage of forward genetic screening is the freedom from constraint on the types of alleles recovered; the only requirement is that the mutation produces the phenotype of interest. That property stands in contrast to techniques for reverse genetic screening, such as RNA interference (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fire <em>et al.</em> 1998</a>), in which the molecular target is known but the ability to modify its activity is limited to a reduction of gene function. Although that category of mutation (reduction or loss of function) represents the class most commonly recovered in forward genetic screens, it is also possible to generate alleles that increase the level or activity of the gene product, reverse normal gene function, or produce novel activity. However, the expanded repertoire of variants accessible by random mutagenesis is accompanied by an increasing complexity in analysis. Although forward genetic screens are typically designed to obtain recessive alleles, they can also recover dominant, semidominant, and even multigenic mutations. The latter categories of alleles are challenging to map by traditional methods, such as linkage to morphological or molecular markers, due to the complexities of zygosity (for dominant and semidominant alleles) or the dependence of the phenotype upon independently segregating loci (for multigenic traits). As a practical consequence, preference is given to recessive, monogenic alleles, and the tools for mutation identification by WGS are tailored to that end.</p><p id="p-6">We reasoned that the data produced by WGS, coupled with the appropriate mapping crosses, should greatly enhance our ability to identify causative variants among the less genetically tractable categories of mutations. To that end, we have utilized a polymorphism mapping method previously developed for mutation identification in <em>C. elegans</em> (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wicks <em>et al.</em> 2001</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Swan <em>et al.</em> 2002</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Doitsidou <em>et al.</em> 2010</a>), and evaluated various strategies for mapping dominant, semidominant, and two-gene mutations. We have also adopted a method to prepare sequencing libraries from small amounts of genomic DNA, and demonstrate its utility in strains where sample recovery may be limiting. Finally, we have performed analysis of WGS data from traditional recombinant mapping crosses, useful for mutation identification in legacy strains or in cases where polymorphism mapping may not be suitable. Together, those approaches extend the application of WGS to the identification of various types of mutations.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-7">All <em>C. elegans</em> mutant strains were derived from the wild-type <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> Bristol strain and contained one or more of the following alleles: <em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene">rol-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00248869;class=Variation">su1006</a>) II</em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00002997;class=Gene">lin-8</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00089281;class=Variation">n111</a>) II</em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00002998;class=Gene">lin-9</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00089282;class=Variation">n112</a>) III</em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00021771;class=Gene">spe-48</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00087785;class=Variation">hc85</a>) I</em> (formerly identified as <em><a href="http://www.wormbase.org/db/get?name=WBGene00004962;class=Gene">spe-8</a></em>), <em><a href="http://www.wormbase.org/db/get?name=WBGene00004964;class=Gene">spe-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00087793;class=Variation">hc104</a>) V</em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00001073;class=Gene">dpy-11</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00143044;class=Variation">e224</a>) V</em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00006808;class=Gene">unc-76</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00143591;class=Variation">e911</a>) V</em>. Hawaiian strain isolate <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hodgkin and Doniach 1997</a>) was used for polymorphism mapping. CRISPR-mediated gene editing to create <em><a href="http://www.wormbase.org/db/get?name=WBGene00021771;class=Gene">spe-48</a>(<a href="http://www.wormbase.org/db/get?name=WBVar02145476;class=Variation">gd11</a>)</em> was performed as described previously (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Paix <em>et al.</em> 2015</a>), using <em><a href="http://www.wormbase.org/db/get?name=WBGene00001072;class=Gene">dpy-10</a></em> as a co-CRISPR marker (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arribere <em>et al.</em> 2014</a>). CRISPR reagents are listed in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028316/-/DC1/TableS1.pdf">Table S1</a>. Worms were propagated using standard growth conditions (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brenner 1974</a>). Fertility assays were performed as described previously (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kulkarni <em>et al.</em> 2012</a>).</p><p id="p-8">Sequencing libraries were constructed using the TruSeq DNA sample prep kit v2 for large-input gDNA samples or TruSeq ChIP sample prep kit for low-input gDNA samples (Illumina, San Diego, CA). Briefly, gDNA samples were sheared by sonication, end-repaired, A-tailed, adapter-ligated, and PCR-amplified. A detailed protocol for obtaining sheared gDNA from small numbers of animals can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028316/-/DC1/FileS1.pdf">File S1</a>. Libraries were sequenced on a HiSeq 2500 (Illumina, San Diego, CA) to generate single-end 50-bp reads, and yielded &gt;20-fold genome coverage per sample. The reference genome used for alignment was obtained from WormBase version WS220 (<a href="http://www.wormbase.org">www.wormbase.org</a>). For Hawaiian single-nucleotide polymorphism (SNP) mapping, we used a list of 112,061 annotated Hawaiian SNPs (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Wicks <em>et al.</em> 2001</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Swan <em>et al.</em> 2002</a>; D. Spencer and R. H. Waterston, unpublished data) from WormBase version WS220. The SNP data in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> were determined with a pipeline of BFAST (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Homer <em>et al.</em> 2009</a>) for alignment and SAMtools (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li <em>et al.</em> 2009</a>) for variant calling. All other SNP data were obtained using BBMap (Bushnell 2015) for alignment and FreeBayes (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Garrison and Marth 2012</a>) for variant calling. Duplicate reads were removed after alignment and, unless otherwise indicated, at least three independent reads were required for a variant call. ANNOVAR (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wang <em>et al.</em> 2010</a>) was used for annotation. For polymorphism mapping, SNP frequencies with LOESS regressions were plotted against chromosome position with R (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">R Core Team 2015</a>) using the same parameters reported for CloudMap (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Minevich <em>et al.</em> 2012</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Polymorphism mapping of recessive mutations. A. Mapping strategy. Only the mutation-bearing chromosome is diagrammed. The strain containing the mutation of interest (asterisk, in red) is crossed to the polymorphic mapping strain (Haw, in green). F2 progeny homozygous for the mutation are pooled for sequencing. B. Mapping data. The proportion of reads corresponding to each Hawaiian SNP is plotted against the physical map. The green line is a LOESS regression of the SNP frequency. Shown is a representative plot used to map the APC subunit mutation such-1(ax2010) at chromosome III, position 11.5 Mb (indicated by red asterisk) (Wang et al. 2014). C. Mapping data from the same strain of an unlinked chromosome (chrV), for comparison." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2105307666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Polymorphism mapping of recessive mutations. A. Mapping strategy. Only the mutation-bearing chromosome is diagrammed. The strain containing the mutation of interest (asterisk, in red) is crossed to the polymorphic mapping strain (Haw, in green). F2 progeny homozygous for the mutation are pooled for sequencing. B. Mapping data. The proportion of reads corresponding to each Hawaiian SNP is plotted against the physical map. The green line is a LOESS regression of the SNP frequency. Shown is a representative plot used to map the APC subunit mutation &lt;em&gt;such-1(ax2010)&lt;/em&gt; at chromosome III, position 11.5 Mb (indicated by red asterisk) (Wang &lt;em&gt;et al.&lt;/em&gt; 2014). C. Mapping data from the same strain of an unlinked chromosome (chrV), for comparison.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1297/F1.medium.gif" width="375" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/5/1297/F1.medium.gif" width="375" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468105" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Polymorphism mapping of recessive mutations. A. Mapping strategy. Only the mutation-bearing chromosome is diagrammed. The strain containing the mutation of interest (asterisk, in red) is crossed to the polymorphic mapping strain (Haw, in green). F2 progeny homozygous for the mutation are pooled for sequencing. B. Mapping data. The proportion of reads corresponding to each Hawaiian SNP is plotted against the physical map. The green line is a LOESS regression of the SNP frequency. Shown is a representative plot used to map the APC subunit mutation <em>such-1(ax2010)</em> at chromosome III, position 11.5 Mb (indicated by red asterisk) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wang <em>et al.</em> 2014</a>). C. Mapping data from the same strain of an unlinked chromosome (chrV), for comparison.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>Data availability</h3><p id="p-10">Sequence data are available at the NCBI Sequence Read Archive (BioProject accession number PRJNA305991).</p></div></div><div class="section results-discussion" id="sec-3"><h2 class="">Results and Discussion</h2><div id="sec-4" class="subsection"><h3>SNP mapping strategy</h3><p id="p-11">We employed a previously described one-step method for simultaneously mapping and identifying candidate mutations in <em>C. elegans</em> by WGS (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Doitsidou <em>et al.</em> 2010</a>). The parental strain bearing the mutation of interest is mated to a highly polymorphic Hawaiian strain isolate, which contains â¼370,000 SNPs (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Thompson <em>et al.</em> 2015</a>). For mapping, we used a subset of â¼110,000 (roughly one per kbp) previously annotated SNPs (see <em>Materials and Methods</em> for details). Recombination between the parental and Hawaiian chromosomes occurs in the heterozygous F1 hermaphrodites, which are allowed to self-fertilize. Those F2 progeny that exhibit the mutant phenotype (mutant F2s, for short) are selected and pooled for sequencing. The positions and frequencies of the Hawaiian SNPs are plotted on the physical map; the SNPs are present in â¼50% of the reads at unlinked loci, but are absent in the interval where the mutation resides (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, AâC</a>). The same data are used to identify novel homozygous sequence variants within the mapping interval as candidate mutations. The strategy has been used successfully to identify a number of recessive mutations (<em>e.g.</em>, <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Doitsidou <em>et al.</em> 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Labed <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Liau <em>et al.</em> 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Connolly <em>et al.</em> 2014</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Wang <em>et al.</em> 2014</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jaramillo-Lambert <em>et al.</em> 2015</a>). With the exception of the strains that contain linked morphological markers (see below), we utilized the Hawaiian SNP mapping method described here for the characterization of different categories of mutations.</p></div><div id="sec-5" class="subsection"><h3>Dominant and semidominant alleles</h3><p id="p-12">For recessive alleles, homozygosity in the mutant F2s produces a gap in the Hawaiian SNPs that defines the mapping interval. Dominant alleles, however, yield a mixture of heterozygous and homozygous mutant F2s in a 2:1 ratio. Consequently, the frequency of Hawaiian SNPs at the mutant locus is only slightly reduced, from 50 to 33%, with a concomitantly small increase in the candidate allele frequency to 67%. Therefore, neither the mapping interval nor the causative mutation is clearly defined.</p><p id="p-13">We considered two different strategies to address the problem (diagrammed in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). For the first strategy, reverse mapping, wild-type F2s are picked and pooled for sequencing. The dominant allele is excluded from the population, so the Hawaiian SNP frequency rises to 100% in the mapping interval. An additional sequencing sample, from the mutation-bearing parental strain, is required to identify candidate mutations in that region. The second strategy involves the screening of F3 progeny from individual mutant F2s, to discriminate homozygous (100% mutant F3s) from heterozygous (75% mutant F3s) F2 animals. The homozygotes can then be pooled, sequenced, and analyzed in the same manner as recessive mutations. F3 screening has been suggested previously (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Doitsidou <em>et al.</em> 2010</a>) but unsupported by data, so we chose to evaluate its performance relative to reverse mapping.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Polymorphism mapping of dominant mutations. A. Mapping strategies. The strain bearing the dominant mutation (asterisk, in red) is crossed to the mapping strain (haw, in green). For reverse mapping, wild-type F2 progeny are pooled for sequencing. For F3 screening, progeny from individual mutant F2s are screened to identify homozygous mutants, which are pooled for sequencing. B. Reverse mapping data for rol-6(su1006). Plot is shown for chromosome II, which contains rol-6 at position 8.7 Mb. C. Mapping data for F3 screen. Chromosome II, with rol-6 indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2105307666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Polymorphism mapping of dominant mutations. A. Mapping strategies. The strain bearing the dominant mutation (asterisk, in red) is crossed to the mapping strain (haw, in green). For reverse mapping, wild-type F2 progeny are pooled for sequencing. For F3 screening, progeny from individual mutant F2s are screened to identify homozygous mutants, which are pooled for sequencing. B. Reverse mapping data for &lt;em&gt;rol-6(su1006)&lt;/em&gt;. Plot is shown for chromosome II, which contains &lt;em&gt;rol-6&lt;/em&gt; at position 8.7 Mb. C. Mapping data for F3 screen. Chromosome II, with &lt;em&gt;rol-6&lt;/em&gt; indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1297/F2.medium.gif" width="440" height="385"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/5/1297/F2.medium.gif" width="440" height="385"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468214" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-14" class="first-child">Polymorphism mapping of dominant mutations. A. Mapping strategies. The strain bearing the dominant mutation (asterisk, in red) is crossed to the mapping strain (haw, in green). For reverse mapping, wild-type F2 progeny are pooled for sequencing. For F3 screening, progeny from individual mutant F2s are screened to identify homozygous mutants, which are pooled for sequencing. B. Reverse mapping data for <em>rol-6(su1006)</em>. Plot is shown for chromosome II, which contains <em>rol-6</em> at position 8.7 Mb. C. Mapping data for F3 screen. Chromosome II, with <em>rol-6</em> indicated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">We tested the two mapping strategies using the well-characterized <em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene">rol-6</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00248869;class=Variation">su1006</a>)</em> mutation. The <em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene">rol-6</a></em> gene encodes a cuticular collagen, and the <em><a href="http://www.wormbase.org/db/get?name=WBVar00248869;class=Variation">su1006</a></em> allele produces an easily identifiable dominant roller (<a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000645;class=Phenotype">Rol</a>) phenotype (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kramer <em>et al.</em> 1990</a>). For the reverse mapping strategy, we picked 100 wild-type F2 progeny from the mapping cross. The mapping plot produced an elevated frequency of SNPs on a single chromosome (chrII) with a peak that encompasses <em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene">rol-6</a></em> (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). For the F3 screen, we picked 120 individual F2 rollers and identified 36 homozygotes that produced 100% <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000645;class=Phenotype">Rol</a> progeny. Polymorphism mapping revealed a reduction in SNP frequency on chromosome II, with a gap that encompasses <em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene">rol-6</a></em> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). Our results demonstrate the validity of the two methods for mapping dominant mutations.</p><p id="p-16">We note that the mapping interval from reverse mapping is not as clearly demarcated as for F3 screening, because the latter approach benefits from a threshold effect for SNP detection. First, low-frequency SNP detection is limited by read depth; for 20-fold genome coverage, the average limit is 5% (1/20). Second, the presence of sequencing errors in the data produces a large number of low-frequency false-positive SNP calls. Filtering by frequency removes those false positives, but also removes <em>bona fide</em> low-frequency SNPs. Consequently, a discontinuity is produced in the SNP plot (in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>, note the absence of SNPs at the bottom of the plot between 0% and â¼5%) and creates a gap that defines the mapping interval. By contrast, the reverse mapping method is unaffected by low-frequency SNPs, and produces a continuous plot that approaches the maximum frequency (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>, top). To better visualize the peak for reverse mapping, we calculated the normalized frequency of homozygous SNPs in 0.5 Mb intervals (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028316/-/DC1/FigureS1.pdf">Figure S1</a>; see legend for details). A histogram of those values indicated that the peak SNP frequency corresponds with the location of <em><a href="http://www.wormbase.org/db/get?name=WBGene00004397;class=Gene">rol-6</a></em>. We conclude that reverse mapping is at least as accurate as F3 screening for dominant alleles.</p><p id="p-17">The two methods that we describe for dominant mapping provide different advantages for the user. F3 screening is more cost-effective; only a single sequencing sample is needed for both mapping and mutation identification, whereas reverse mapping demands two sequencing samples. However, F3 screening requires that the mutant phenotype be observable in two sequential generations, and many classes of mutations do not fulfill that criterion. Phenotypic categories unsuitable for F3 screening include embryonic or larval lethality or arrest, sexual transformation leading to infertility, gametogenesis defects, and male-specific phenotypes. Dominant mutations have been reported for several of those classes: <em><a href="http://www.wormbase.org/db/get?name=WBGene00003209;class=Gene">mel-26</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00054442;class=Variation">ct61</a>)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00003183;class=Gene">mei-1</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00054417;class=Variation">ct46</a>)</em> result in maternal-effect embryonic lethality (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mains <em>et al.</em> 1990</a>), 21 alleles of <em><a href="http://www.wormbase.org/db/get?name=WBGene00006604;class=Gene">tra-1</a>(gf)</em> produce sterile females (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hodgkin 1987</a>), and six alleles of <em><a href="http://www.wormbase.org/db/get?name=WBGene00006940;class=Gene">wee-1.3</a></em> cause sperm-specific sterility (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lamitina and LâHernault 2002</a>). Mutations such as those are refractory to mapping by F3 screening but accessible by reverse mapping.</p><p id="p-18">In principle, semidominant alleles should be more amenable to analysis than dominant alleles, since heterozygous animals exhibit an intermediate phenotype that can be distinguished from homozygotes. In practice, many phenotypes (such as body length or lifespan) are phenomena with a continuous range of values, and it may not be possible to assign the genotype of individuals unambiguously based on the observed characteristic. Therefore, we recommend the same F3 progeny screening strategy used for dominant alleles, which should prove similarly sufficient to distinguish the zygosity of individual F2 animals. The reverse mapping strategy might also be employed, but only in cases where the wild-type and heterozygous phenotypes are fully resolved.</p></div><div id="sec-6" class="subsection"><h3>Two-gene synthetic interactions</h3><p id="p-19">The category is defined by the requirement for mutations at two loci to produce the phenotype of interest. Although more typically isolated as genetic enhancers, it is also possible to obtain such double mutants by unbiased forward screening. One of the earliest screens in <em>C. elegans</em> for cell lineage mutants recovered the recessive <em><a href="http://www.wormbase.org/db/get?name=WBGene00002997;class=Gene">lin-8</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00089281;class=Variation">n111</a>)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00002998;class=Gene">lin-9</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00089282;class=Variation">n112</a>)</em> pair of alleles that define the synthetic multivulva (synMuv) class A and B categories (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ferguson and Horvitz 1989</a>). Each of those mutations is phenotypically silent in isolation, but the double mutant exhibits multiple ectopic pseudovulvae (<a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000700;class=Phenotype">Muv</a> phenotype).</p><p id="p-20">We chose the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002997;class=Gene">lin-8</a></em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00002998;class=Gene">lin-9</a></em> pair as a test case for the application of WGS to the identification of two-gene synthetic interactors. We selected 100 F2 <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000700;class=Phenotype">Muv</a> progeny from the mapping cross and pooled them for sequencing. Two chromosomes contained a biased distribution of SNPs, with mapping intervals that correspond to the positions of <em><a href="http://www.wormbase.org/db/get?name=WBGene00002997;class=Gene">lin-8</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00002998;class=Gene">lin-9</a></em> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, AâC</a>). The gap in SNPs flanking <em>lin</em>-8 is masked by a high local density of polymorphisms, but evident at higher resolution (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). We note that the same method has been used to map the double mutation <em>ot628</em>, obtained in a screen for loss of neuronal identity (O. Hobert, personal communication). We conclude that WGS and polymorphism mapping can be applied successfully to map two-gene traits.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Polymorphism mapping of lin-8; lin-9 double mutations. SNP mapping plots. A. Chromosome II, which contains lin-8(n111) at position 3.1 Mb. B. Enlargement of the chromosome II interval from 2.0 to 4.0 Mb. C. Chromosome III, which contains lin-9(n112) at position 8.9 Mb." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2105307666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Polymorphism mapping of &lt;em&gt;lin-8&lt;/em&gt;; &lt;em&gt;lin-9&lt;/em&gt; double mutations. SNP mapping plots. A. Chromosome II, which contains &lt;em&gt;lin-8(n111)&lt;/em&gt; at position 3.1 Mb. B. Enlargement of the chromosome II interval from 2.0 to 4.0 Mb. C. Chromosome III, which contains &lt;em&gt;lin-9(n112)&lt;/em&gt; at position 8.9 Mb.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1297/F3.medium.gif" width="440" height="403"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/5/1297/F3.medium.gif" width="440" height="403"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468066" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Polymorphism mapping of <em>lin-8</em>; <em>lin-9</em> double mutations. SNP mapping plots. A. Chromosome II, which contains <em>lin-8(n111)</em> at position 3.1 Mb. B. Enlargement of the chromosome II interval from 2.0 to 4.0 Mb. C. Chromosome III, which contains <em>lin-9(n112)</em> at position 8.9 Mb.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Sequencing from small samples</h3><p id="p-22">Many categories of mutations produce terminal phenotypes (such as lethality or sterility) that effectively limit the sample size. Standard WGS library preparation calls for microgram quantities of input gDNA, which can only be obtained from large populations. However, protocols have been developed for library construction that require only low nanogram amounts of DNA, so we evaluated the suitability of low-input libraries for mutation identification from small numbers of worms.</p><p id="p-23">A major concern is that the sequence coverage obtained by using low-input library preparation might prove insufficient for our application. Accurate mutation identification requires data that encompass the entire genome. The potential for coverage bias from small amounts of input DNA might preclude detection of some variants. To address that concern, we compared the data produced from a standard library preparation using 1 Âµg of gDNA (referred to as the large sample) to those generated from a low-input library prep protocol using only 2 ng (hereafter, small sample) of the same gDNA. We selected the highly polymorphic Hawaiian strain as our test case, to ensure a large number of validated SNPs for identification. We used equal numbers of sequence reads for the two data sets and compared the alignment metrics.</p><p id="p-24">First, we determined the fraction of the genome that was covered by at least three reads (the minimum read depth required for a variant call in our analysis) in the large and small samples. We observed little difference (&lt;0.25%) in genome coverage between the two libraries (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>), indicating that the small sample does not introduce substantial gaps in coverage. Note that our coverage calculations are conservative; repetitive sequences constitute â¼5% of the genome and, by filtering reads that mapped to multiple loci, those sequences were excluded. Next, we considered more subtle evidence for biased coverage. The small amount of input DNA might reduce the complexity of that library relative to the large sample, producing an increase in the fraction of duplicate reads as well as a larger variance in the per-base depth of coverage. We did observe differences in the degree of read duplication (large, 15.8%; small, 39.1%) as well as the depth of coverage distribution between the two libraries (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>), consistent with reduced library complexity in the small sample. However, the impact of those differences on SNP detection was minimal. After removing the duplicate reads from each sample, we performed variant calling and compared the lists of annotated Hawaiian SNPs that were identified in each data set. The numbers and identities of the SNP calls were essentially the same between the large and small samples, with greater than 99% overlap (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>). We conclude that libraries constructed from small amounts of input are sufficient for accurate mutation identification.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Coverage for large vs. small-input Hawaiian samples. A. Number of bases in the reference genome covered by at least three uniquely mapped reads. B. Read depth distribution for the large (blue) and small (red) input samples. C. Venn diagram indicating the number of Hawaiian SNPs identified in each sample." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2105307666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Coverage for large &lt;em&gt;vs.&lt;/em&gt; small-input Hawaiian samples. A. Number of bases in the reference genome covered by at least three uniquely mapped reads. B. Read depth distribution for the large (blue) and small (red) input samples. C. Venn diagram indicating the number of Hawaiian SNPs identified in each sample.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1297/F4.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/5/1297/F4.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468160" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Coverage for large <em>vs.</em> small-input Hawaiian samples. A. Number of bases in the reference genome covered by at least three uniquely mapped reads. B. Read depth distribution for the large (blue) and small (red) input samples. C. Venn diagram indicating the number of Hawaiian SNPs identified in each sample.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We proceeded to obtain gDNA from a small population of worms with a terminal mutant phenotype. Our test strain contained the recessive <em><a href="http://www.wormbase.org/db/get?name=WBVar00087785;class=Variation">hc85</a></em> allele, which produces a sperm-specific defect that renders them sterile (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">LâHernault <em>et al.</em> 1988</a>). The mutation was originally assigned to the <em><a href="http://www.wormbase.org/db/get?name=WBGene00004962;class=Gene">spe-8</a></em> complementation group. However, more recent data dispute that assignment; <em><a href="http://www.wormbase.org/db/get?name=WBVar00087785;class=Variation">hc85</a></em> complements the sperm-specific sterility of another <em><a href="http://www.wormbase.org/db/get?name=WBGene00004962;class=Gene">spe-8</a></em> allele, and targeted sequencing of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00004962;class=Gene">spe-8</a></em> gene from the strain bearing <em><a href="http://www.wormbase.org/db/get?name=WBVar00087785;class=Variation">hc85</a></em> failed to identify a molecular lesion (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Muhlrad <em>et al.</em> 2014</a>). To address that discrepancy, we used the Hawaiian mapping cross and recovered gDNA from a pool of 50 hand-picked sterile F2 animals. SNP mapping defined the interval between positions 1.7 and 2.7 Mb on chromosome I (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). That interval is distinct from the position of <em><a href="http://www.wormbase.org/db/get?name=WBGene00004962;class=Gene">spe-8</a></em>, which is located on the same chromosome at position 0.1 Mb.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Polymorphism mapping from small input sample. A. Mapping data for spe-48(hc85) allele. Mapping interval (red bracket highlighting SNP gap at 1.7&#x2212;2.7 Mb) differs from position of spe-8 (blue arrow at position 0.1 Mb). B. Physical map of candidate mutations in the spe-48(hc85) mapping interval. Position, gene name, and amino acid change are indicated. C. Fertility data from the CRISPR allele spe-48(gd11). Mean progeny counts with standard deviations (n = 10 hermaphrodites) are shown for the indicated genotypes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2105307666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Polymorphism mapping from small input sample. A. Mapping data for &lt;em&gt;spe-48(hc85)&lt;/em&gt; allele. Mapping interval (red bracket highlighting SNP gap at 1.7&#x2212;2.7 Mb) differs from position of &lt;em&gt;spe-8&lt;/em&gt; (blue arrow at position 0.1 Mb). B. Physical map of candidate mutations in the &lt;em&gt;spe-48(hc85)&lt;/em&gt; mapping interval. Position, gene name, and amino acid change are indicated. C. Fertility data from the CRISPR allele &lt;em&gt;spe-48(gd11)&lt;/em&gt;. Mean progeny counts with standard deviations (n = 10 hermaphrodites) are shown for the indicated genotypes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1297/F5.medium.gif" width="440" height="294"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/5/1297/F5.medium.gif" width="440" height="294"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468140" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-27" class="first-child">Polymorphism mapping from small input sample. A. Mapping data for <em>spe-48(hc85)</em> allele. Mapping interval (red bracket highlighting SNP gap at 1.7â2.7 Mb) differs from position of <em>spe-8</em> (blue arrow at position 0.1 Mb). B. Physical map of candidate mutations in the <em>spe-48(hc85)</em> mapping interval. Position, gene name, and amino acid change are indicated. C. Fertility data from the CRISPR allele <em>spe-48(gd11)</em>. Mean progeny counts with standard deviations (n = 10 hermaphrodites) are shown for the indicated genotypes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">We confirmed the validity of the SNP mapping data by determining the molecular identity of the <em><a href="http://www.wormbase.org/db/get?name=WBVar00087785;class=Variation">hc85</a></em> allele. Four genes within the mapping interval contained novel, homozygous, nonsynonymous mutations (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). Previous analyses determined that virtually all Spe genes exhibit sperm-specific expression (<em>e.g.</em>, see <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kulkarni <em>et al.</em> 2012</a>), so the four candidates were prioritized by that criterion. Recently published RNA-Seq data (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ma <em>et al.</em> 2014</a>) indicated that only one of those four genes, <em><a href="http://www.wormbase.org/db/get?name=WBGene00021771;class=Gene">Y51F10.10</a></em>, is detectably expressed in sperm. Therefore, we engineered the observed <em><a href="http://www.wormbase.org/db/get?name=WBGene00021771;class=Gene">Y51F10.10</a></em> missense mutation, a threonine-to-isoleucine substitution at amino acid 272, into the wild-type strain via CRISPR-mediated gene editing (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Paix <em>et al.</em> 2015</a>). The engineered allele, designated <em><a href="http://www.wormbase.org/db/get?name=WBVar02145476;class=Variation">gd11</a></em>, recapitulates the recessive Spe phenotype of <em><a href="http://www.wormbase.org/db/get?name=WBVar00087785;class=Variation">hc85</a></em>. Self-fertility is normal for heterozygous <em><a href="http://www.wormbase.org/db/get?name=WBVar02145476;class=Variation">gd11</a>/+</em> hermaphrodites but low for homozygous <em><a href="http://www.wormbase.org/db/get?name=WBVar02145476;class=Variation">gd11</a></em> hermaphrodites, and fertility is restored by mating to wild-type males (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). Complementation testing of the Spe phenotype revealed the <em><a href="http://www.wormbase.org/db/get?name=WBVar02145476;class=Variation">gd11</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBVar00087785;class=Variation">hc85</a></em> mutations to be allelic, as self-fertility of <em><a href="http://www.wormbase.org/db/get?name=WBVar02145476;class=Variation">gd11</a>/<a href="http://www.wormbase.org/db/get?name=WBVar00087785;class=Variation">hc85</a></em> hermaphrodites is low (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>). Because the <em><a href="http://www.wormbase.org/db/get?name=WBGene00021771;class=Gene">Y51F10.10</a></em> gene is distinct from <em><a href="http://www.wormbase.org/db/get?name=WBGene00004962;class=Gene">spe-8</a></em>, we have assigned the new gene name <em><a href="http://www.wormbase.org/db/get?name=WBGene00021771;class=Gene">spe-48</a></em> to <em><a href="http://www.wormbase.org/db/get?name=WBVar00087785;class=Variation">hc85</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBVar02145476;class=Variation">gd11</a></em>.</p></div><div id="sec-8" class="subsection"><h3>Marked mutant mapping</h3><p id="p-29">The examples described above require mating to the polymorphic Hawaiian strain to identify the mapping interval. However, there are numerous cases where that cross may not be suitable. First, there are known genetic incompatibilities between the Hawaiian strain and the standard laboratory <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> Bristol strain (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Seidel <em>et al.</em> 2008</a>). The incompatibility loci produce biased segregation ratios in the F2 population, such that mutations in the vicinity of the loci are difficult to map (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Minevich <em>et al.</em> 2012</a>). Second, the genetic diversity of the polymorphic strain might include modifiers of the phenotype of interest, thereby confounding the isolation of phenotypically identifiable animals needed to generate the mapping information. Finally, the investigator may wish to take advantage of available recombinant lines obtained from crosses to morphologically marked strains. Those include legacy strains that were mapped using two- or three-factor crosses, as well as lines constructed to link an easily visible marker to the mutation of interest.</p><p id="p-30">A map-free method of mutation identification has been developed (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zuryn <em>et al.</em> 2010</a>), in which homozygosity of linked variants after extensive backcrossing defines the mutation interval. An extension of that approach, using bulk segregant analysis, was subsequently reported (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abe <em>et al.</em> 2012</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Minevich <em>et al.</em> 2012</a>). We propose a conceptually similar strategy for the analysis of mutations linked to a morphological marker (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). Those strains contain two sources of novel genetic variation that are absent from the wild-type strain: (1) <em>de novo</em> variants produced during mutagenesis, and (2) pre-existing variants in the morphologically marked strain. A single backcross of the marked mutant strain to the wild-type is used to determine the mapping interval. Nonrecombinant F2 segregants are homozygous for both marker and mutation, and any variants between those loci. Sequencing of the nonrecombinant F2s and comparison to the wild-type strain sequence is used to discriminate common <em>vs.</em> novel homozygous variants; a plot of the latter reveals a cluster that defines the mapping interval. Note that the strategy is not predicated on knowledge of the physical location or molecular identity of the linked morphological marker allele, nor does it require accurate determination of the recombination frequency between the marker and the mutation of interest.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping of mutations linked to markers. A. Mapping strategy. The strain bearing the marker mutation (a, in red) and linked mutation of interest (asterisk) is crossed to wild-type (blue). F2s homozygous for both mutations are pooled for sequencing. B&#x2212;C. Mapping data for dpy-11 spe-10 and spe-10 unc-76, respectively. Homozygous variants (&gt;80% variant call) unique to the mutant strain are plotted against the physical map. Red points, 100% variant calls. D. Reciprocal mapping. Data from B and C are combined, and average variant fraction is plotted against the physical map. Red line, LOESS regression of the average variant fraction." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2105307666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mapping of mutations linked to markers. A. Mapping strategy. The strain bearing the marker mutation (&lt;em&gt;a&lt;/em&gt;, in red) and linked mutation of interest (asterisk) is crossed to wild-type (blue). F2s homozygous for both mutations are pooled for sequencing. B&#x2212;C. Mapping data for &lt;em&gt;dpy-11 spe-10&lt;/em&gt; and &lt;em&gt;spe-10 unc-76&lt;/em&gt;, respectively. Homozygous variants (&gt;80% variant call) unique to the mutant strain are plotted against the physical map. Red points, 100% variant calls. D. Reciprocal mapping. Data from B and C are combined, and average variant fraction is plotted against the physical map. Red line, LOESS regression of the average variant fraction.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/5/1297/F6.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/5/1297/F6.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/5/1297/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468220" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-31" class="first-child">Mapping of mutations linked to markers. A. Mapping strategy. The strain bearing the marker mutation (<em>a</em>, in red) and linked mutation of interest (asterisk) is crossed to wild-type (blue). F2s homozygous for both mutations are pooled for sequencing. BâC. Mapping data for <em>dpy-11 spe-10</em> and <em>spe-10 unc-76</em>, respectively. Homozygous variants (&gt;80% variant call) unique to the mutant strain are plotted against the physical map. Red points, 100% variant calls. D. Reciprocal mapping. Data from B and C are combined, and average variant fraction is plotted against the physical map. Red line, LOESS regression of the average variant fraction.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">We evaluated the strategy using the <em><a href="http://www.wormbase.org/db/get?name=WBGene00004964;class=Gene">spe-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00087793;class=Variation">hc104</a>)</em> mutation marked with <em><a href="http://www.wormbase.org/db/get?name=WBGene00001073;class=Gene">dpy-11</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00143044;class=Variation">e224</a>)</em>. The former confers temperature-sensitive, sperm-specific sterility (Spe phenotype), and the latter produces short, fat animals (<a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000583;class=Phenotype">Dpy</a> phenotype). We crossed the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001073;class=Gene">dpy-11</a> <a href="http://www.wormbase.org/db/get?name=WBGene00004964;class=Gene">spe-10</a></em> strain to wild-type males, and picked 50 Spe Dpy F2s grown at the restrictive temperature for sequencing. After variant calling, we removed those that were also present in the wild-type sample and identified novel homozygous variants (minimum 15 reads, &gt;80% variant call). The 80% threshold accommodates errors in the sequence data as well as mistakes in selecting the desired F2s, at the expense of including some nonhomozygous variants. The distribution of homozygous variants was clearly enriched on chromosome V (73 of 87 total, or 84%). We plotted those variants and highlighted the <em>bona fide</em> (100%) homozygous calls (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>, in red), and observed a cluster between 5.0 and 14.5 Mb (a span that encompasses <em><a href="http://www.wormbase.org/db/get?name=WBGene00001073;class=Gene">dpy-11</a></em> at 6.5 Mb and <em><a href="http://www.wormbase.org/db/get?name=WBGene00004964;class=Gene">spe-10</a></em> at 10.4 Mb). We conclude that, in the case of strains bearing marked mutations, a single backcross is sufficient for mapping.</p><p id="p-33">Use of a marked mutation offers some advantages over the previously described non-Hawaiian mapping strategies. It employs only a single round of backcrossing, which is less laborious than the multiple rounds of backcrossing necessitated by the initial map-free method of <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Zuryn <em>et al.</em> (2010)</a>. It also requires less effort than bulk segregation analysis: only the marked segregants are screened for the mutant phenotype, which reduces the number to be screened by â¼75%. That feature is particularly useful for mutations that are difficult to score or exhibit low/variable penetrance. Our strategy also foregoes the additional work and expense of sequencing the background strain that is used for filtering in bulk segregation analysis (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Minevich <em>et al.</em> 2012</a>), albeit with the disadvantage of retaining background mutations that are not causative. However, our method was designed specifically with legacy mutations in mind, to accommodate cases where background filtering may not be suitable (<em>e.g.</em>, the provenance of the marked mutant strain is uncertain, the background strain is no longer available, or the mutant strain has drifted significantly since its original isolation). In instances where multiple strains are analyzed from the same mutant screen, it would still be possible to filter background variants that are present in more than one strain.</p><p id="p-34">A disadvantage of marked mutant mapping is that the mapping interval must span both loci, which can produce an unacceptably large mapping interval if the marker and mutation are not tightly linked. In some cases, it might be preferable to use only recombinant F2 segregants to provide higher-resolution mapping. However, screening for F2 recombinants may entail substantially more work. In the example provided, the desired recombinant (<em><a href="http://www.wormbase.org/db/get?name=WBGene00001073;class=Gene">dpy-11</a> <a href="http://www.wormbase.org/db/get?name=WBGene00004964;class=Gene">spe-10</a>/+spe</em>-10) is morphologically wild-type but sterile, and the map distance between the two loci is 2.54 cM. To obtain the 50 animals needed for small-input library construction as above, it would require screening of â¼4000 (50/0.5 Ã 0.0254) wild-type F2s. Thus, the tradeoff between mapping resolution and additional screening will be dictated by the particulars of the mutant phenotype.</p><p id="p-35">Higher-resolution mapping can also be obtained in cases where reciprocal recombinants are available from three-factor crosses with flanking markers. Individual homozygous mutant lines containing either the left or right marker are sequenced separately, and the frequencies of novel variants in each sample are plotted on the physical map as above. A contiguous cluster of homozygous variants present in both samples defines the mutation interval. The endpoints are delimited by variants that are homozygous in only one sample.</p><p id="p-36">We assessed the utility of reciprocal mapping with a <em><a href="http://www.wormbase.org/db/get?name=WBGene00004964;class=Gene">spe-10</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00087793;class=Variation">hc104</a>) <a href="http://www.wormbase.org/db/get?name=WBGene00006808;class=Gene">unc-76</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00143591;class=Variation">e911</a>)</em> strain (the latter mutation produces uncoordinated, or <a href="http://www.wormbase.org/db/get?name=WBPhenotype%3A0000643;class=Phenotype">Unc</a>, movement). After mating with wild-type males, we picked 50 Spe Unc F2s grown at 25Â° for sequencing. First, we mapped homozygous variants as above. The plot revealed a biased distribution on chromosome V (52 of 69 total variants, 75%), with a cluster between 8.0 and 16.5 Mb that contains <em><a href="http://www.wormbase.org/db/get?name=WBGene00004964;class=Gene">spe-10</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00006808;class=Gene">unc-76</a></em> (located at 15.1 Mb; <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>). For reciprocal mapping, we used the homozygous variants identified in either the <em><a href="http://www.wormbase.org/db/get?name=WBGene00001073;class=Gene">dpy-11</a> <a href="http://www.wormbase.org/db/get?name=WBGene00004964;class=Gene">spe-10</a></em> sample or the <em><a href="http://www.wormbase.org/db/get?name=WBGene00004964;class=Gene">spe-10</a> <a href="http://www.wormbase.org/db/get?name=WBGene00006808;class=Gene">unc-76</a></em> sample, and calculated the average variant fraction by combining the data from both samples. The map plot produced a smaller interval than either single sample, with well-defined endpoints at 8.6 and 14.5 Mb clearly indicated by the LOESS curve (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6D</a>). The utility of the strategy should be weighed against the effort and expense of sequencing an additional strain, but it may be worthwhile when the markers are far from the mutation of interest and/or validation of candidate alleles is challenging.</p></div><div id="sec-9" class="subsection"><h3>Conclusions</h3><p id="p-37">WGS has transformed our ability to identify mutations obtained from forward screening. We describe a variety of methods for applying the technology to types of mutations â dominant and semidominant alleles, synthetic interactors, and terminal phenotypes â that pose particular challenges to identification. We demonstrate the validity of those methods by the identification of both previously known as well as new mutations, and confirm the latter by independent criteria. Although our analyses were limited to <em>C. elegans</em>, the strategies can be generalized to any species in which polymorphic isolates are available for crosses and should prove useful for researchers who wish to apply the power of WGS to the investigation of alleles that are difficult to identify by other methods.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">We thank members of Michael Krauseâs lab and the Baltimore Worm Club for fruitful discussions, and Sevinc Ercan for sharing the small sample gDNA isolation protocol. Some strains were provided by the Caenorhabditis Genetics Center, which is funded by the National Institutes of Health (NIH) Office of Research Infrastructure Programs (P40 OD10440). This study was supported by the Intramural Research Program of the National Institutes of Health, National Institute of Diabetes and Digestive and Kidney Diseases, and is subject to the NIH Public Access Policy.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-39">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028316/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028316/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-40"><em>Communicating editor: D. S. Fay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-01-04"><span class="received-label">Received </span>January 4, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-03-01"><span class="accepted-label">Accepted </span>March 1, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2016 Smith <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.1" data-doi="10.1038/nbt.2095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosugi</span>  <span class="cit-name-given-names">S. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshida</span>  <span class="cit-name-given-names">K. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natsume</span>  <span class="cit-name-given-names">S. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takagi</span>  <span class="cit-name-given-names">H. </span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genome sequencing reveals agronomically important loci in rice using MutMap.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">174</span>â<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DAbe%26rft.auinit1%253DA.%26rft.volume%253D30%26rft.issue%253D2%26rft.spage%253D174%26rft.epage%253D178%26rft.atitle%253DGenome%2Bsequencing%2Breveals%2Bagronomically%2Bimportant%2Bloci%2Bin%2Brice%2Busing%2BMutMap.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2095%26rft_id%253Dinfo%253Apmid%252F22267009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22267009&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.2" data-doi="10.1534/genetics.114.169730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arribere</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">R. T. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">B. X. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Artiles</span>  <span class="cit-name-given-names">K. L. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartman</span>  <span class="cit-name-given-names">P. S. </span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Efficient marker-free recovery of custom genetic modifications with CRISPR/Cas9 in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">837</span>â<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169730%26rft_id%253Dinfo%253Apmid%252F25161212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzMvODM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.3" data-doi="10.1101/sqb.2009.74.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrick</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genome-wide mutational diversity in an evolving population of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">119</span>â<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BSymp.%2BQuant.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fsqb.2009.74.018%26rft_id%253Dinfo%253Apmid%252F19776167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjg6Ijc0LzAvMTE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.4" data-doi="10.1534/genetics.110.123232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birkeland</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">N. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozdemir</span>  <span class="cit-name-given-names">A. C. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyons</span>  <span class="cit-name-given-names">R. H. </span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weisman</span>  <span class="cit-name-given-names">L. S. </span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Discovery of mutations in <em>Saccharomyces cerevisiae</em> by pooled linkage analysis and whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">1127</span>â<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.123232%26rft_id%253Dinfo%253Apmid%252F20923977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni80LzExMjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTI5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.5" data-doi="10.1534/genetics.109.101998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blumenstiel</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noll</span>  <span class="cit-name-given-names">A. C. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">J. A. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perera</span>  <span class="cit-name-given-names">A. G. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walton</span>  <span class="cit-name-given-names">K. N. </span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Identification of EMS-induced mutations in <em>Drosophila melanogaster</em> by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">25</span>â<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101998%26rft_id%253Dinfo%253Apmid%252F19307605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTgyLzEvMjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTI5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.6" data-doi="10.1534/genetics.111.136069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henke</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegfried</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warman</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Efficient mapping and cloning of mutations in zebrafish by low-coverage whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1017</span>â<span class="cit-lpage">1024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.136069%26rft_id%253Dinfo%253Apmid%252F22174069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC8zLzEwMTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTI5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.7" data-doi="10.1091/mbc.E11-03-0185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowerman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">The near demise and subsequent revival of classical genetics for investigating <em>Caenorhabditis elegans</em> embryogenesis: RNAi meets next-generation DNA sequencing.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3556</span>â<span class="cit-lpage">3558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E11-03-0185%26rft_id%253Dinfo%253Apmid%252F21960050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xOS8zNTU2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.8" data-doi="10.1051/medsci/20031967725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span>â<span class="cit-article-title">The genetics of <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">71</span>â<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBrenner%26rft.auinit1%253DS.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D94%26rft.atitle%253DTHE%2BGENETICS%2BOF%2BCAENORHABDITIS%2BELEGANS%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fmedsci%252F20031967725%26rft_id%253Dinfo%253Apmid%252F4366476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiNzcvMS83MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-6.5.1297.9"><div class="cit-metadata unstructured">Bushnell, B., 2015âBBMap short-read aligner, and other bioinformatics tools. Available at: <a href="http://sourceforge.net/projects/bbmap/">http://sourceforge.net/projects/bbmap/</a>. Accessed: July 10, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.10" data-doi="10.1091/mbc.E13-11-0687"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Connolly</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osterberg</span>  <span class="cit-name-given-names">V. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">S. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">M. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">C. </span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title"><em>Caenorhabditis elegans</em> oocyte meiotic spindle pole assembly requires microtubule severing and the calponin homology domain protein ASPM-1.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1298</span>â<span class="cit-lpage">1311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E13-11-0687%26rft_id%253Dinfo%253Apmid%252F24554763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI1LzgvMTI5OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.11" data-doi="10.1371/journal.pone.0015435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doitsidou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">R. J. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarin</span>  <span class="cit-name-given-names">S. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bigelow</span>  <span class="cit-name-given-names">H. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O. </span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title"><em>C. elegans</em> mutant identification with a one-step whole-genome-sequencing and SNP mapping strategy.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e15435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDoitsidou%26rft.auinit1%253DM.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De15435%26rft.epage%253De15435%26rft.atitle%253DC.%2Belegans%2Bmutant%2Bidentification%2Bwith%2Ba%2Bone-step%2Bwhole-genome-sequencing%2Band%2BSNP%2Bmapping%2Bstrategy.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015435%26rft_id%253Dinfo%253Apmid%252F21079745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21079745&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.12" data-doi="10.1534/g3.111.000919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dutcher</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giddings</span>  <span class="cit-name-given-names">T. H.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Whole-genome sequencing to identify mutants and polymorphisms in Chlamydomonas reinhardtii.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">15</span>â<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DDutcher%26rft.auinit1%253DS.%2BK.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D15%26rft.epage%253D22%26rft.atitle%253DWhole-Genome%2BSequencing%2Bto%2BIdentify%2BMutants%2Band%2BPolymorphisms%2Bin%2BChlamydomonas%2Breinhardtii%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000919%26rft_id%253Dinfo%253Apmid%252F22384377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjIvMS8xNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvitz</span>  <span class="cit-name-given-names">H. R. </span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">The multivulva phenotype of certain <em>Caenorhabditis elegans</em> mutants results from defects in two functionally redundant pathways.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">109</span>â<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFerguson%26rft.auinit1%253DE.%2BL.%26rft.volume%253D123%26rft.issue%253D1%26rft.spage%253D109%26rft.epage%253D121%26rft.atitle%253DThe%2Bmultivulva%2Bphenotype%2Bof%2Bcertain%2BCaenorhabditis%2Belegans%2Bmutants%2Bresults%2Bfrom%2Bdefects%2Bin%2Btwo%2Bfunctionally%2Bredundant%2Bpathways.%26rft_id%253Dinfo%253Apmid%252F2806880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzEvMTA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.14" data-doi="10.1038/35888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fire</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montgomery</span>  <span class="cit-name-given-names">M. K. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kostas</span>  <span class="cit-name-given-names">S. A. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Driver</span>  <span class="cit-name-given-names">S. E .</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Potent and specific genetic interference by double-stranded RNA in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">391</span>: <span class="cit-fpage">806</span>â<span class="cit-lpage">811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFire%26rft.auinit1%253DA.%26rft.volume%253D391%26rft.issue%253D6669%26rft.spage%253D806%26rft.epage%253D811%26rft.atitle%253DPotent%2Band%2Bspecific%2Bgenetic%2Binterference%2Bby%2Bdouble-stranded%2BRNA%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35888%26rft_id%253Dinfo%253Apmid%252F9486653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35888&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9486653&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072089500057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-web" id="cit-6.5.1297.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garrison</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marth</span>  <span class="cit-name-given-names">G. </span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>âHaplotype-based variant detection from short-read sequencing. arXiv: 1207.3907v2.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.16" data-doi="10.1534/genetics.109.112938"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">The impact of whole genome sequencing on model system genetics; get ready for the ride.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">317</span>â<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHobert%26rft.auinit1%253DO.%26rft.volume%253D184%26rft.issue%253D2%26rft.spage%253D317%26rft.epage%253D319%26rft.atitle%253DThe%2BImpact%2Bof%2BWhole%2BGenome%2BSequencing%2Bon%2BModel%2BSystem%2BGenetics%253A%2BGet%2BReady%2Bfor%2Bthe%2BRide%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.112938%26rft_id%253Dinfo%253Apmid%252F20103786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg0LzIvMzE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.17" data-doi="10.1101/gad.1.7.731"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodgkin</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">A genetic analysis of the sex-determining gene, <em>tra-1</em>, in the nematode <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">731</span>â<span class="cit-lpage">745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHodgkin%26rft.auinit1%253DJ%26rft.volume%253D1%26rft.issue%253D7%26rft.spage%253D731%26rft.epage%253D745%26rft.atitle%253DA%2Bgenetic%2Banalysis%2Bof%2Bthe%2Bsex-determining%2Bgene%252C%2Btra-1%252C%2Bin%2Bthe%2Bnematode%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1.7.731%26rft_id%253Dinfo%253Apmid%252F3428597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiMS83LzczMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodgkin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doniach</span>  <span class="cit-name-given-names">T. </span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Natural variation and copulatory plug formation in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">149</span>â<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHodgkin%26rft.auinit1%253DJ.%26rft.volume%253D146%26rft.issue%253D1%26rft.spage%253D149%26rft.epage%253D164%26rft.atitle%253DNatural%2BVariation%2Band%2BCopulatory%2BPlug%2BFormation%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Apmid%252F9136008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ2LzEvMTQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.19" data-doi="10.1371/journal.pone.0007767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Homer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merriman</span>  <span class="cit-name-given-names">B. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">S. F. </span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">BFAST: an alignment tool for large scale genome resequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e7767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHomer%26rft.auinit1%253DN.%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253De7767%26rft.epage%253De7767%26rft.atitle%253DBFAST%253A%2Ban%2Balignment%2Btool%2Bfor%2Blarge%2Bscale%2Bgenome%2Bresequencing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0007767%26rft_id%253Dinfo%253Apmid%252F19907642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0007767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19907642&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.20" data-doi="10.1016/j.ymeth.2014.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Whole genome sequencing and the transformation of <em>C. elegans</em> forward genetics.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">437</span>â<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%26rft.volume%253D68%26rft.spage%253D437%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2014.05.008%26rft_id%253Dinfo%253Apmid%252F24874788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2014.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24874788&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.21" data-doi="10.1101/gr.089318.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Irvine</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">D. B. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaughn</span>  <span class="cit-name-given-names">M. W. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakaseko</span>  <span class="cit-name-given-names">Y. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mccombie</span>  <span class="cit-name-given-names">W. R. </span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Mapping epigenetic mutations in fission yeast using whole-genome next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1077</span>â<span class="cit-lpage">1083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089318.108%26rft_id%253Dinfo%253Apmid%252F19423874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA3NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.22" data-doi="10.1534/g3.115.017038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaramillo-Lambert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchsman</span>  <span class="cit-name-given-names">A. S. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fabritius</span>  <span class="cit-name-given-names">A. S. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. E. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golden</span>  <span class="cit-name-given-names">A. </span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Rapid and efficient identification of <em>Caenorhabditis elegans</em> legacy mutations using SNP-based mapping and whole genome sequencing.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1007</span>â<span class="cit-lpage">1019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.017038%26rft_id%253Dinfo%253Apmid%252F25740937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjUvNS8xMDA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.23" data-doi="10.1128/MCB.10.5.2081"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">French</span>  <span class="cit-name-given-names">R. P. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">E. C. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. J .</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">The <em>Caenorhabditis elegans rol-6</em> gene, which interacts with the <em>sqt-1</em> collagen gene to determine organismal morphology, encodes a collagen.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2081</span>â<span class="cit-lpage">2090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DKramer%26rft.auinit1%253DJ%2BM%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D2081%26rft.epage%253D2089%26rft.atitle%253DThe%2BCaenorhabditis%2Belegans%2Brol-6%2Bgene%252C%2Bwhich%2Binteracts%2Bwith%2Bthe%2Bsqt-1%2Bcollagen%2Bgene%2Bto%2Bdetermine%2Borganismal%2Bmorphology%252C%2Bencodes%2Ba%2Bcollagen.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.10.5.2081%26rft_id%253Dinfo%253Apmid%252F1970117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEwLzUvMjA4MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.24" data-doi="10.1371/journal.pgen.1002678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kulkarni</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shakes</span>  <span class="cit-name-given-names">D. C. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guevel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. E. </span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">SPE-44 implements sperm cell fate.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKulkarni%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253De1002678%26rft.epage%253De1002678%26rft.atitle%253DSPE-44%2Bimplements%2Bsperm%2Bcell%2Bfate.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002678%26rft_id%253Dinfo%253Apmid%252F22570621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22570621&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.25" data-doi="10.1371/journal.pone.0033887"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labed</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Omi</span>  <span class="cit-name-given-names">S. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gut</span>  <span class="cit-name-given-names">M. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewbank</span>  <span class="cit-name-given-names">J. J. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pujol</span>  <span class="cit-name-given-names">N. </span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The pseudokinase NIPI-4 is a novel regulator of antimicrobial peptide gene expression.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e33887</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLabed%26rft.auinit1%253DS.%2BA.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De33887%26rft.epage%253De33887%26rft.atitle%253DThe%2Bpseudokinase%2BNIPI-4%2Bis%2Ba%2Bnovel%2Bregulator%2Bof%2Bantimicrobial%2Bpeptide%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0033887%26rft_id%253Dinfo%253Apmid%252F22470487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0033887&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22470487&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.26" data-doi="10.1104/pp.110.156448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laitinen</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jelly</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Identification of a spontaneous frame shift mutation in a nonreference Arabidopsis accession using whole genome sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">652</span>â<span class="cit-lpage">654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.110.156448%26rft_id%253Dinfo%253Apmid%252F20388671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTMvMi82NTIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTI5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamitina</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LâHernault</span>  <span class="cit-name-given-names">S. W. </span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Dominant mutations in the <em>Caenorhabditis elegans</em> Myt1 ortholog <em>wee-1.3</em> reveal a novel domain that controls M-phase entry during spermatogenesis.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">5009</span>â<span class="cit-lpage">5018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DLamitina%26rft.auinit1%253DS.%2BT.%26rft.volume%253D129%26rft.issue%253D21%26rft.spage%253D5009%26rft.epage%253D5018%26rft.atitle%253DDominant%2Bmutations%2Bin%2Bthe%2BCaenorhabditis%2Belegans%2BMyt1%2Bortholog%2Bwee-1.3%2Breveal%2Ba%2Bnovel%2Bdomain%2Bthat%2Bcontrols%2BM-phase%2Bentry%2Bduring%2Bspermatogenesis%26rft_id%253Dinfo%253Apmid%252F12397109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTI5LzIxLzUwMDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTI5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">LâHernault</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shakes</span>  <span class="cit-name-given-names">D. C. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">S. </span></span></li></ol><cite>, <span class="cit-pub-date">1988</span>â<span class="cit-article-title">Developmental genetics of chromosome I spermatogenesis-defective mutants in the nematode <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">435</span>â<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DL%2527Hernault%26rft.auinit1%253DS.%2BW.%26rft.volume%253D120%26rft.issue%253D2%26rft.spage%253D435%26rft.epage%253D452%26rft.atitle%253DDevelopmental%2Bgenetics%2Bof%2Bchromosome%2BI%2Bspermatogenesis-defective%2Bmutants%2Bin%2Bthe%2Bnematode%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Apmid%252F3197956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIwLzIvNDM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.29" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J. </span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>â<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.30" data-doi="10.1371/journal.pone.0057266"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liau</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasri</span>  <span class="cit-name-given-names">U. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elmatari</span>  <span class="cit-name-given-names">D. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothman</span>  <span class="cit-name-given-names">J. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamunyon</span>  <span class="cit-name-given-names">C. W. </span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Premature sperm activation and defective spermatogenesis caused by loss of <em>spe-46</em> function in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e57266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De57266%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0057266%26rft_id%253Dinfo%253Apmid%252F23483899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0057266&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23483899&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.31" data-doi="10.1186/1471-2164-15-168"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>  <span class="cit-name-given-names">P. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y. </span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Characterisation of <em>Caenorhabditis elegans</em> sperm transcriptome and proteome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D168%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-168%26rft_id%253Dinfo%253Apmid%252F24581041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-168&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24581041&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mains</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sulston</span>  <span class="cit-name-given-names">I. A. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">W. B. </span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">Dominant maternal-effect mutations causing embryonic lethality in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">351</span>â<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMains%26rft.auinit1%253DP.%2BE.%26rft.volume%253D125%26rft.issue%253D2%26rft.spage%253D351%26rft.epage%253D369%26rft.atitle%253DDominant%2Bmaternal-effect%2Bmutations%2Bcausing%2Bembryonic%2Blethality%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Apmid%252F2379819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI1LzIvMzUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.33" data-doi="10.1534/genetics.112.144204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Minevich</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">D. S. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blankenberg</span>  <span class="cit-name-given-names">D. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poole</span>  <span class="cit-name-given-names">R. J. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O. </span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">CloudMap: a cloud-based pipeline for analysis of mutant genome sequences.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1249</span>â<span class="cit-lpage">1269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144204%26rft_id%253Dinfo%253Apmid%252F23051646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzEyNDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTI5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.34" data-doi="10.1126/science.33.848.496"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1911</span><span class="cit-comment">a</span>â<span class="cit-article-title">The origin of nine wing mutations in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">496</span>â<span class="cit-lpage">499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMORGAN%26rft.auinit1%253DT.%2BH.%26rft.volume%253D33%26rft.issue%253D848%26rft.spage%253D496%26rft.epage%253D499%26rft.atitle%253DTHE%2BORIGIN%2BOF%2BNINE%2BWING%2BMUTATIONS%2BIN%2BDROSOPHILA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.33.848.496%26rft_id%253Dinfo%253Apmid%252F17774436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTA6IjMzLzg0OC80OTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTI5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.35" data-doi="10.1126/science.33.849.534-a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1911</span><span class="cit-comment">b</span>â<span class="cit-article-title">The origin of five mutations in eye color in <em>Drosophila</em> and their modes of inheritance.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">534</span>â<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMORGAN%26rft.auinit1%253DT.%2BH.%26rft.volume%253D33%26rft.issue%253D849%26rft.spage%253D534%26rft.epage%253D537%26rft.atitle%253DTHE%2BORIGIN%2BOF%2BFIVE%2BMUTATIONS%2BIN%2BEYE%2BCOLOR%2BIN%2BDROSOPHILA%2BAND%2BTHEIR%2BMODES%2BOF%2BINHERITANCE%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.33.849.534-a%26rft_id%253Dinfo%253Apmid%252F17817675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTI6IjMzLzg0OS81MzQtYSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muhlrad</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">J. N. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasri</span>  <span class="cit-name-given-names">U. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">N. G. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamunyon</span>  <span class="cit-name-given-names">C. W. </span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">SPE-8, a protein-tyrosine kinase, localizes to the spermatid cell membrane through interaction with other members of the SPE-8 group spermatid activation signaling pathway in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253D83%26rft_id%253Dinfo%253Apmid%252F25022984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25022984&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.37" data-doi="10.1534/genetics.115.179382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paix</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Folkmann</span>  <span class="cit-name-given-names">A. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasoloson</span>  <span class="cit-name-given-names">D. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seydoux</span>  <span class="cit-name-given-names">G. </span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">High efficiency, homology-directed genome editing in <em>Caenorhabditis elegans</em> using CRISPR-Cas9 ribonucleoprotein complexes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">47</span>â<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179382%26rft_id%253Dinfo%253Apmid%252F26187122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjAxLzEvNDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTI5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-web" id="cit-6.5.1297.38"><div class="cit-metadata unstructured">R CORE TEAM, 2015. R: a language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. <a href="http://www.R-project.org">http://www.R-project.org</a>. Accessed: July 10, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.39" data-doi="10.1038/nmeth.1249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prabhu</span>  <span class="cit-name-given-names">S. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">OâMeara</span>  <span class="cit-name-given-names">M. M. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">PeâEr</span>  <span class="cit-name-given-names">I. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O. </span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title"><em>Caenorhabditis elegans</em> mutant allele identification by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">865</span>â<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSarin%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253D865%26rft.epage%253D867%26rft.atitle%253DCaenorhabditis%2Belegans%2Bmutant%2Ballele%2Bidentification%2Bby%2Bwhole-genome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1249%26rft_id%253Dinfo%253Apmid%252F18677319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18677319&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259650100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.40" data-doi="10.1371/journal.pone.0046759"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saxer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Havlak</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quance</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Whole genome sequencing of mutation accumulation lines reveals a low mutation rate in the social amoeba Dictyostelium Discoideum.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSaxer%26rft.auinit1%253DG.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46759%26rft.epage%253De46759%26rft.atitle%253DWhole%2BGenome%2BSequencing%2Bof%2BMutation%2BAccumulation%2BLines%2BReveals%2Ba%2BLow%2BMutation%2BRate%2Bin%2Bthe%2BSocial%2BAmoeba%2BDictyostelium%2Bdiscoideum.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046759%26rft_id%253Dinfo%253Apmid%252F23056439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046759&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23056439&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.41" data-doi="10.1126/science.1151107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L. </span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Widespread genetic incompatibility in <em>C. elegans</em> maintained by balancing selection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">589</span>â<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151107%26rft_id%253Dinfo%253Apmid%252F18187622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2My81ODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTI5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swan</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curtis</span>  <span class="cit-name-given-names">D. E. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mckusick</span>  <span class="cit-name-given-names">K. B. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volnov</span>  <span class="cit-name-given-names">A. V. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mapa</span>  <span class="cit-name-given-names">F. A. </span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">High-throughput gene mapping in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1100</span>â<span class="cit-lpage">1105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DSwan%26rft.auinit1%253DK.%2BA.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D1100%26rft.epage%253D1105%26rft.atitle%253DHigh-Throughput%2BGene%2BMapping%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Apmid%252F12097347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEyLzcvMTEwMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNS8xMjk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.43" data-doi="10.1534/genetics.115.175950"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nivjeen</span>  <span class="cit-name-given-names">H. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sterken</span>  <span class="cit-name-given-names">M. G. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volkers</span>  <span class="cit-name-given-names">R. J. </span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Remarkably divergent regions punctuate the genome assembly of the <em>Caenorhabditis elegans</em> Hawaiian strain CB4856.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">975</span>â<span class="cit-lpage">989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.175950%26rft_id%253Dinfo%253Apmid%252F25995208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzMvOTc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.44" data-doi="10.1093/nar/gkq603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakonarson</span>  <span class="cit-name-given-names">H. </span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq603%26rft_id%253Dinfo%253Apmid%252F20601685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOC8xNi9lMTY0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.45" data-doi="10.1534/g3.113.009126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. T. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasoloson</span>  <span class="cit-name-given-names">D. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stitzel</span>  <span class="cit-name-given-names">M. L. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">OâConnell</span>  <span class="cit-name-given-names">K. F. </span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Identification of suppressors of <em>mbk-2/DYRK</em> by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">231</span>â<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.009126%26rft_id%253Dinfo%253Apmid%252F24347622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvMi8yMzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzUvMTI5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.46" data-doi="10.1038/88878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wicks</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">R. T. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W. R. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterston</span>  <span class="cit-name-given-names">R. H. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plasterk</span>  <span class="cit-name-given-names">R. H. </span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Rapid gene mapping in <em>Caenorhabditis elegans</em> using a high density polymorphism map.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">160</span>â<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DWicks%26rft.auinit1%253DS.%2BR.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D164%26rft.atitle%253DRapid%2Bgene%2Bmapping%2Bin%2BCaenorhabditis%2Belegans%2Busing%2Ba%2Bhigh%2Bdensity%2Bpolymorphism%2Bmap.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F88878%26rft_id%253Dinfo%253Apmid%252F11381264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/88878&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11381264&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F5%2F1297.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169011800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.5.1297.47" data-doi="10.1534/genetics.110.119230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuryn</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Gras</span>  <span class="cit-name-given-names">S. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jamet</span>  <span class="cit-name-given-names">K. </span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarriault</span>  <span class="cit-name-given-names">S. </span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">A strategy for direct mapping and identification of mutations by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">427</span>â<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.119230%26rft_id%253Dinfo%253Apmid%252F20610404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzEvNDI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi81LzEyOTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/5/1297.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/5/1287" title="Conflation of Short Identity-by-Descent Segments Bias Their Inferred Length Distribution" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/5/1305" title="Characterization of Chicken MMP13 Expression and Genetic Effect on Egg Production Traits of Its Promoter Polymorphisms" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/5"><span>Volume 6 Issue 5, May 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/5.cover-source.jpg?itok=8SH0hKOw" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/5/1297" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JMiPgxwJpu15tQAdGNkEexNiJsuSg3PUlpAcZL6ABpg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/5/1297" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/5/1297" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/5/1297" class="active" data-icon-position="" data-hide-link-title="0">Mapping Challenging Mutations by Whole-Genome Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460362" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ANDTNzsfq_EiswJ9Cq8k8mGYm3KG4hPgugKT1Of2gFs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/5/1297.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/5/1297" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-vr0de1THk0B8y_UaVBGRPJcCw_KpQS1CeKUxe9rEfQE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/5/1297" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/5/1297.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460362" id="node460362--2" data-pisa="ggg;6/5/1297" data-pisa-master="ggg;g3.116.028316" data-apath="/ggg/6/5/1297.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Challenging Mutations by Whole-Genome Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3517-5973" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Harold E. Smith</span>, <span class="highwire-citation-author" data-delta="1">Amy S. Fabritius</span>, <span class="highwire-citation-author" data-delta="2">Aimee Jaramillo-Lambert</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8599-2031" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andy Golden</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1297-1304;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028316 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460362--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Harold E. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harold%2BE.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20HE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHarold%2BE.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3517-5973" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Harold E. Smith</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:smithhe2@niddk.nih.gov" class="" data-icon-position="" data-hide-link-title="0">smithhe2@niddk.nih.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amy S. Fabritius </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BS.%2BFabritius%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fabritius%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmy%2BS.%2BFabritius%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aimee Jaramillo-Lambert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aimee%2BJaramillo-Lambert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jaramillo-Lambert%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAimee%2BJaramillo-Lambert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andy Golden </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andy%2BGolden%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Golden%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndy%2BGolden%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8599-2031" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andy Golden</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/5/1297" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/5/1297&amp;title=Mapping%20Challenging%20Mutations%20by%20Whole-Genome%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/5/1297&amp;title=Mapping%20Challenging%20Mutations%20by%20Whole-Genome%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/5/1297&amp;title=Mapping%20Challenging%20Mutations%20by%20Whole-Genome%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/5/1297&amp;text=Mapping%20Challenging%20Mutations%20by%20Whole-Genome%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/5/1297&amp;title=Mapping%20Challenging%20Mutations%20by%20Whole-Genome%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/5/1297&amp;t=Mapping%20Challenging%20Mutations%20by%20Whole-Genome%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/5/1297&amp;title=Mapping%20Challenging%20Mutations%20by%20Whole-Genome%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/5/1297&amp;title=Mapping%20Challenging%20Mutations%20by%20Whole-Genome%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460362/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460362" id="node460362--3" data-pisa="ggg;6/5/1297" data-pisa-master="ggg;g3.116.028316" data-apath="/ggg/6/5/1297.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Challenging Mutations by Whole-Genome Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3517-5973" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Harold E. Smith</span>, <span class="highwire-citation-author" data-delta="1">Amy S. Fabritius</span>, <span class="highwire-citation-author" data-delta="2">Aimee Jaramillo-Lambert</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-8599-2031" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Andy Golden</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">May 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1297-1304;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028316 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460362--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Harold E. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harold%2BE.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20HE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHarold%2BE.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3517-5973" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Harold E. Smith</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:smithhe2@niddk.nih.gov" class="" data-icon-position="" data-hide-link-title="0">smithhe2@niddk.nih.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Amy S. Fabritius </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BS.%2BFabritius%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fabritius%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmy%2BS.%2BFabritius%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aimee Jaramillo-Lambert </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aimee%2BJaramillo-Lambert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jaramillo-Lambert%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAimee%2BJaramillo-Lambert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Andy Golden </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andy%2BGolden%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Golden%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndy%2BGolden%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8599-2031" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andy Golden</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1297&amp;rft.epage=1304&amp;rft.atitle=Mapping%20Challenging%20Mutations%20by%20Whole-Genome%20Sequencing&amp;rft.volume=6&amp;rft.issue=5&amp;rft.date=2016-05-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Smith%2C+Harold+E.&amp;rft.au=Fabritius%2C+Amy+S.&amp;rft.au=Jaramillo-Lambert%2C+Aimee&amp;rft.au=Golden%2C+Andy"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460362/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460362/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460362/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460362/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460362/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460362/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460362/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460362/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460362/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460362/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460362/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460362/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/5/1297" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/5/1297.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/5/1297.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/5/1297.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/5/1297.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
