<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/8/2385" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/8/2385.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/8/2385.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/8/2385.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/8/2385.atom" />
<meta name="HW.pisa" content="ggg;6/8/2385" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Population of Deletion Mutants and an Integrated Mapping and Exome-seq Pipeline for Gene Discovery in Maize" />
<meta name="DC.Identifier" content="10.1534/g3.116.030528" />
<meta name="DC.Date" content="2016-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Jia et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="To better understand maize endosperm filling and maturation, we used γ-irradiation of the B73 maize reference line to generate mutants with opaque endosperm and reduced kernel fill phenotypes, and created a population of 1788 lines including 39 Mo17 × F2s showing stable, segregating, and viable kernel phenotypes. For molecular characterization of the mutants, we developed a novel functional genomics platform that combined bulked segregant RNA and exome sequencing (BSREx-seq) to map causative mutations and identify candidate genes within mapping intervals. To exemplify the utility of the mutants and provide proof-of-concept for the bioinformatics platform, we present detailed characterization of line 937, an opaque mutant harboring a 6203 bp in-frame deletion covering six exons within the Opaque-1 gene. In addition, we describe mutant line 146 which contains a 4.8 kb intragene deletion within the Sugary-1 gene and line 916 in which an 8.6 kb deletion knocks out a Cyclin A2 gene. The publically available algorithm developed in this work improves the identification of causative deletions and its corresponding gaps within mapping peaks. This study demonstrates the utility of γ-irradiation for forward genetics in large nondense genomes such as maize since deletions often affect single genes. Furthermore, we show how this classical mutagenesis method becomes applicable for functional genomics when combined with state-of-the-art genomics tools." />
<meta name="DC.Contributor" content="Shangang Jia" />
<meta name="DC.Contributor" content="Aixia Li" />
<meta name="DC.Contributor" content="Kyla Morton" />
<meta name="DC.Contributor" content="Penny Avoles-Kianian" />
<meta name="DC.Contributor" content="Shahryar F. Kianian" />
<meta name="DC.Contributor" content="Chi Zhang" />
<meta name="DC.Contributor" content="David Holding" />
<meta name="article:published_time" content="2016-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Population of Deletion Mutants and an Integrated Mapping and Exome-seq Pipeline for Gene Discovery in Maize" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;To better understand maize endosperm filling and maturation, we used γ-irradiation of the B73 maize reference line to generate mutants with opaque endosperm and reduced kernel fill phenotypes, and created a population of 1788 lines including 39 Mo17 × F2s showing stable, segregating, and viable kernel phenotypes. For molecular characterization of the mutants, we developed a novel functional genomics platform that combined bulked segregant RNA and exome sequencing (BSREx-seq) to map causative mutations and identify candidate genes within mapping intervals. To exemplify the utility of the mutants and provide proof-of-concept for the bioinformatics platform, we present detailed characterization of line 937, an opaque mutant harboring a 6203 bp in-frame deletion covering six exons within the &lt;i&gt;Opaque-1&lt;/i&gt; gene. In addition, we describe mutant line 146 which contains a 4.8 kb intragene deletion within the &lt;i&gt;Sugary-1&lt;/i&gt; gene and line 916 in which an 8.6 kb deletion knocks out a &lt;i&gt;Cyclin A2&lt;/i&gt; gene. The publically available algorithm developed in this work improves the identification of causative deletions and its corresponding gaps within mapping peaks. This study demonstrates the utility of γ-irradiation for forward genetics in large nondense genomes such as maize since deletions often affect single genes. Furthermore, we show how this classical mutagenesis method becomes applicable for functional genomics when combined with state-of-the-art genomics tools.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/08/01" />
<meta name="citation_mjid" content="ggg;6/8/2385" />
<meta name="citation_id" content="6/8/2385" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/8/2385" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/8/2385.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/8/2385.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/8/2385.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.030528" />
<meta name="citation_pmid" content="27261000" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2385" />
<meta name="citation_lastpage" content="2395" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Shangang Jia" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588" />
<meta name="citation_author" content="Aixia Li" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588" />
<meta name="citation_author" content="Kyla Morton" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588" />
<meta name="citation_author" content="Penny Avoles-Kianian" />
<meta name="citation_author_institution" content="USDA-ARS Cereal Disease Laboratory, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author" content="Shahryar F. Kianian" />
<meta name="citation_author_institution" content="USDA-ARS Cereal Disease Laboratory, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author" content="Chi Zhang" />
<meta name="citation_author_institution" content="School of Biological Sciences, Center for Plant Science and Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588" />
<meta name="citation_author" content="David Holding" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588" />
<meta name="citation_author_email" content="dholding2@unl.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Anders;citation_author=W. Huber;citation_title=Differential expression analysis for sequence count data.;citation_pages=R106-R106;citation_volume=11;citation_year=2010;citation_issue=10;citation_pmid=20979621;citation_doi=10.1186/gb-2010-11-10-r106" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Anders;citation_author=P. T. Pyl;citation_author=W. Huber;citation_title=HTSeq – a Python framework to work with high-throughput sequencing data.;citation_volume=31;citation_year=2015;citation_pmid=25260700;citation_doi=10.1093/bioinformatics/btu638" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. P. Blumenstiel;citation_author=A. C. Noll;citation_author=J. A. Griffiths;citation_author=A. G. Perera;citation_author=K. N. Walton;citation_title=Identification of EMS-induced mutations in Drosophila melanogaster by whole-genome sequencing.;citation_volume=182;citation_year=2009;citation_pmid=19307605;citation_doi=10.1534/genetics.109.101998" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=Y. T. Bolon;citation_author=W. J. Haun;citation_author=W. W. Xu;citation_author=D. Grant;citation_author=M. G. Stacey;citation_title=Phenotypic and genomic analyses of a fast neutron mutant population resource in soybean.;citation_volume=156;citation_year=2011;citation_pmid=21321255;citation_doi=10.1104/pp.110.170811" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=R. A. Dante;citation_author=B. A. Larkins;citation_author=P. A. Sabelli;citation_title=Cell cycle control and seed development.;citation_pages=493;citation_volume=5;citation_year=2014;citation_pmid=25295050" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=J. R. Dinges;citation_author=C. Colleoni;citation_author=A. M. Myers;citation_author=M. G. James;citation_title=Molecular Structure of Three Mutations at the Maize sugary1 Locus and Their Allele-Specific Phenotypic Effects;citation_pages=1406-1418;citation_volume=125;citation_year=2001;citation_issue=3;citation_pmid=11244120;citation_doi=10.1104/pp.125.3.1406" />
<meta name="citation_reference" content="citation_journal_title=Hereditas-Genetisk. A;citation_author=J. Dumanovi;citation_author=M. Denic;citation_author=L. Ehrenber;citation_author=K. Bergstra;citation_title=Radiation-induced heritable variation of quantitative characters in wheat.;citation_pages=221-227;citation_volume=62;citation_year=1969" />
<meta name="citation_reference" content="citation_journal_title=Aust. J. Agric. Res.;citation_author=D. B. Egli;citation_title=The role of seed in the determination of yield of grain crops.;citation_pages=1237-1247;citation_volume=57;citation_year=2006;citation_doi=10.1071/AR06133" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=F. Gavazzi;citation_author=B. Lazzari;citation_author=P. Ciceri;citation_author=E. Gianazza;citation_author=A. Viotti;citation_title=Wild-type opaque2 and defective opaque2 polypeptides form complexes in maize endosperm cells and bind the opaque2-zein target site.;citation_volume=145;citation_year=2007;citation_pmid=17827273;citation_doi=10.1104/pp.107.103606" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=X. Guo;citation_author=K. Ronhovde;citation_author=L. Yuan;citation_author=B. Yao;citation_author=M. P. Soundararajan;citation_title=Pyrophosphate-dependent fructose-6-phosphate 1-phosphotransferase induction and attenuation of Hsp gene expression during endosperm modification in quality protein maize.;citation_volume=158;citation_year=2012;citation_pmid=22158678;citation_doi=10.1104/pp.111.191163" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. R. Holding;citation_author=B. G. Hunter;citation_author=T. Chung;citation_author=B. C. Gibbon;citation_author=C. F. Ford;citation_title=Genetic analysis of opaque2 modifier loci in quality protein maize.;citation_pages=157-170;citation_volume=117;citation_year=2008;citation_issue=2;citation_pmid=18427771;citation_doi=10.1007/s00122-008-0762-y" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. R. Holding;citation_author=B. G. Hunter;citation_author=J. P. Klingler;citation_author=S. Wu;citation_author=X. Guo;citation_title=Characterization of opaque2 modifier QTLs and candidate genes in recombinant inbred lines derived from the K0326Y quality protein maize inbred.;citation_pages=783-794;citation_volume=122;citation_year=2011;citation_issue=4;citation_pmid=21076810;citation_doi=10.1007/s00122-010-1486-3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. G. Hossain;citation_author=O. Riera-Lizarazu;citation_author=V. Kalavacharla;citation_author=M. I. Vales;citation_author=S. S. Maan;citation_title=Radiation Hybrid Mapping of the Species Cytoplasm-Specific (scsae) Gene in Wheat;citation_pages=415-423;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=15454553;citation_doi=10.1534/genetics.103.022590" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Agric. Biol.;citation_author=L. Hu;citation_author=J. Guo;citation_author=Y. Zhao;citation_author=L. Zhu;citation_author=Y. Huang;citation_title=Molecular characterization and evolutionary analysis of the putative caleosin gene family in maize (Zea mays).;citation_pages=261-268;citation_volume=16;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=M. G. James;citation_author=D. S. Robertson;citation_author=A. M. Myers;citation_title=Characterization of the Maize Gene sugary1, a Determinant of Starch Composition in Kernels;citation_pages=417-429;citation_volume=7;citation_year=1995;citation_issue=4;citation_pmid=7773016;citation_doi=10.1105/tpc.7.4.417" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=M. M. Johri;citation_author=E. H. Coe;citation_title=Clonal analysis of corn plant development. I. The development of the tassel and the ear shoot.;citation_pages=154-172;citation_volume=97;citation_year=1983;citation_issue=1;citation_pmid=6840396;citation_doi=10.1016/0012-1606(83)90073-8" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. C. Koboldt;citation_author=Q. Zhang;citation_author=D. E. Larson;citation_author=D. Shen;citation_author=M. D. McLellan;citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.;citation_volume=22;citation_year=2012;citation_pmid=22300766;citation_doi=10.1101/gr.129684.111" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Maydica;citation_author=B. Lazzari;citation_author=C. Cosentino;citation_author=A. Viotti;citation_title=Gene products and structure analysis of wild-type and mutant alleles at the Opaque-2 locus of Zea mays.;citation_pages=253-265;citation_volume=47;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.;citation_volume=27;citation_year=2011;citation_pmid=21903627;citation_doi=10.1093/bioinformatics/btr509" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate long-read alignment with Burrows–Wheeler transform.;citation_volume=26;citation_year=2010;citation_pmid=20080505;citation_doi=10.1093/bioinformatics/btp698" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=L. Li;citation_author=D. Li;citation_author=S. Liu;citation_author=X. Ma;citation_author=C. R. Dietrich;citation_title=The maize glossy13 gene, cloned via BSR-Seq and Seq-walking encodes a putative ABC transporter required for the normal accumulation of epicuticular waxes.;citation_pages=e82333;citation_volume=8;citation_year=2013;citation_pmid=24324772;citation_doi=10.1371/journal.pone.0082333" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=X. Li;citation_author=Y. Song;citation_author=K. Century;citation_author=S. Straight;citation_author=P. Ronald;citation_title=A fast neutron deletion mutagenesis-based reverse genetics system for plants.;citation_pages=235-242;citation_volume=27;citation_year=2001;citation_issue=3;citation_pmid=11532169;citation_doi=10.1046/j.1365-313x.2001.01084.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Liu;citation_author=C. T. Yeh;citation_author=H. M. Tang;citation_author=D. Nettleton;citation_author=P. S. Schnable;citation_title=Gene mapping via bulked segregant RNA-Seq (BSR-Seq).;citation_pages=e36406-e36406;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22586469;citation_doi=10.1371/journal.pone.0036406" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Mascher;citation_author=M. Jost;citation_author=J. E. Kuon;citation_author=A. Himmelbach;citation_author=A. Assfalg;citation_title=Mapping-by-sequencing accelerates forward genetics in barley.;citation_pages=R78;citation_volume=15;citation_year=2014;citation_pmid=24917130;citation_doi=10.1186/gb-2014-15-6-r78" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=C. M. McCallum;citation_author=L. Comai;citation_author=E. A. Greene;citation_author=S. Henikoff;citation_title=Targeting Induced Local Lesions IN Genomes (TILLING) for Plant Functional Genomics;citation_pages=439-442;citation_volume=123;citation_year=2000;citation_issue=2;citation_pmid=10859174;citation_doi=10.1104/pp.123.2.439" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=D. R. McCarty;citation_author=A. M. Settles;citation_author=M. Suzuki;citation_author=B. C. Tan;citation_author=S. Latshaw;citation_title=Steady-state transposon mutagenesis in inbred maize.;citation_pages=52-61;citation_volume=44;citation_year=2005;citation_issue=1;citation_pmid=16167895;citation_doi=10.1111/j.1365-313X.2005.02509.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. C. Miller;citation_author=N. D. Obholzer;citation_author=A. N. Shah;citation_author=S. G. Megason;citation_author=C. B. Moens;citation_title=RNA-seq-based mapping and candidate identification of mutations from forward genetic screens.;citation_volume=23;citation_year=2013;citation_pmid=23299976;citation_doi=10.1101/gr.147322.112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Naito;citation_author=M. Kusaba;citation_author=N. Shikazono;citation_author=T. Takano;citation_author=A. Tanaka;citation_title=Transmissible and nontransmissible mutations induced by irradiating Arabidopsis thaliana pollen with gamma-rays and carbon ions.;citation_volume=169;citation_year=2005;citation_pmid=15371348;citation_doi=10.1534/genetics.104.033654" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=K. J. Nordström;citation_author=M. C. Albani;citation_author=G. V. James;citation_author=C. Gutjahr;citation_author=B. Hartwig;citation_title=Mutation identification by direct comparison of whole-genome sequencing data from mutant and wild-type individuals using k-mers.;citation_pages=325-330;citation_volume=31;citation_year=2013;citation_pmid=23475072;citation_doi=10.1038/nbt.2515" />
<meta name="citation_reference" content="citation_journal_title=Dev. Biol.;citation_author=R. S. Poethig;citation_author=E. H. Coe;citation_author=M. M. Johri;citation_title=Cell lineage patterns in maize embryogenesis: A clonal analysis.;citation_pages=392-404;citation_volume=117;citation_year=1986;citation_doi=10.1016/0012-1606(86)90308-8" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=A. Rahman;citation_author=K. Wong;citation_author=J. Jane;citation_author=A. M. Myers;citation_author=M. G. James;citation_title=Characterization of SU1 Isoamylase, a Determinant of Storage Starch Structure in Maize;citation_pages=425-435;citation_volume=117;citation_year=1998;citation_issue=2;citation_pmid=9625695;citation_doi=10.1104/pp.117.2.425" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=R. H. Ramirez-Gonzalez;citation_author=V. Segovia;citation_author=N. Bird;citation_author=P. Fenwick;citation_author=S. Holdgate;citation_title=RNA-Seq bulked segregant analysis enables the identification of high-resolution genetic markers for breeding in hexaploid wheat.;citation_pages=613-624;citation_volume=13;citation_year=2015;citation_pmid=25382230;citation_doi=10.1111/pbi.12281" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=P. A. Sabelli;citation_author=R. A. Dante;citation_author=H. N. Nguyen;citation_author=W. J. Gordon-Kamm;citation_author=B. A. Larkins;citation_title=Expression, regulation and activity of a B2-type cyclin in mitotic and endoreduplicating maize endosperm.;citation_pages=561;citation_volume=5;citation_year=2014;citation_pmid=25368625" />
<meta name="citation_reference" content="citation_journal_title=Breed. Sci.;citation_author=Y. Sato;citation_author=K. Shirasawa;citation_author=Y. Takahashi;citation_author=M. Nishimura;citation_author=T. Nishio;citation_title=Mutant selection from progeny of gamma-ray-irradiated rice by DNA heteroduplex cleavage using Brassica petiole extract.;citation_pages=179-183;citation_volume=56;citation_year=2006;citation_doi=10.1270/jsbbs.56.179" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. Schierenbeck;citation_author=D. Ries;citation_author=K. Rogge;citation_author=S. Grewe;citation_author=B. Weisshaar;citation_title=Fast forward genetics to identify mutations causing a high light tolerant phenotype in Chlamydomonas reinhardtii by whole-genome-sequencing.;citation_pages=57;citation_volume=16;citation_year=2015;citation_pmid=25730202;citation_doi=10.1186/s12864-015-1232-y" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=K. Schneeberger;citation_title=Using next-generation sequencing to isolate mutant genes from forward genetic screens.;citation_pages=662-676;citation_volume=15;citation_year=2014;citation_pmid=25139187;citation_doi=10.1038/nrg3745" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=K. Schneeberger;citation_author=S. Ossowski;citation_author=C. Lanz;citation_author=T. Juul;citation_author=A. H. Petersen;citation_title=SHOREmap: simultaneous mapping and mutation identification by deep sequencing.;citation_pages=550-551;citation_volume=6;citation_year=2009;citation_issue=8;citation_pmid=19644454;citation_doi=10.1038/nmeth0809-550" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. M. Settles;citation_author=D. R. Holding;citation_author=B. C. Tan;citation_author=S. P. Latshaw;citation_author=J. Liu;citation_title=Sequence-indexed mutations in maize using the UniformMu transposon-tagging population.;citation_pages=116-116;citation_volume=8;citation_year=2007;citation_pmid=17490480;citation_doi=10.1186/1471-2164-8-116" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=K. Shirasawa;citation_author=H. Hirakawa;citation_author=T. Nunome;citation_author=S. Tabata;citation_author=S. Isobe;citation_title=Genome-wide survey of artificial mutations induced by ethyl methanesulfonate and gamma rays in tomato.;citation_pages=51-60;citation_volume=14;citation_year=2015;citation_pmid=25689669" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdottir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.;citation_volume=14;citation_year=2013;citation_pmid=22517427;citation_doi=10.1093/bib/bbs017" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=B. J. Till;citation_author=S. H. Reynolds;citation_author=C. Weil;citation_author=N. Springer;citation_author=C. Burtner;citation_title=Discovery of induced point mutations in maize genes by TILLING.;citation_pages=12-12;citation_volume=4;citation_year=2004;citation_pmid=15282033;citation_doi=10.1186/1471-2229-4-12" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Trapnell;citation_author=L. Pachter;citation_author=S. L. Salzberg;citation_title=TopHat: discovering splice junctions with RNA-Seq.;citation_volume=25;citation_year=2009;citation_pmid=19289445;citation_doi=10.1093/bioinformatics/btp120" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=C. Trapnell;citation_author=A. Roberts;citation_author=L. Goff;citation_author=G. Pertea;citation_author=D. Kim;citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks.;citation_pages=562-578;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22383036;citation_doi=10.1038/nprot.2012.016" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=M. Trick;citation_author=N. M. Adamski;citation_author=S. G. Mugford;citation_author=C. Jiang;citation_author=M. Febrer;citation_title=Combining SNP discovery from next-generation sequencing data with bulked segregant analysis (BSA) to fine-map genes in polyploid wheat.;citation_pages=14-14;citation_volume=12;citation_year=2012;citation_pmid=22280551;citation_doi=10.1186/1471-2229-12-14" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=E. Vollbrecht;citation_author=J. Duvick;citation_author=J. P. Schares;citation_author=K. R. Ahern;citation_author=P. Deewatthanawong;citation_title=Genome-wide distribution of transposed dissociation elements in maize.;citation_volume=22;citation_year=2010;citation_pmid=20581308;citation_doi=10.1105/tpc.109.073452" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=J. C. Wallace;citation_author=M. A. Lopes;citation_author=E. Paiva;citation_author=B. A. Larkins;citation_title=New Methods for Extraction and Quantitation of Zeins Reveal a High Content of {gamma}-Zein in Modified opaque-2 Maize;citation_pages=191-196;citation_volume=92;citation_year=1990;citation_issue=1;citation_pmid=16667246;citation_doi=10.1104/pp.92.1.191" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=G. Wang;citation_author=F. Wang;citation_author=G. Wang;citation_author=F. Wang;citation_author=X. Zhang;citation_title=Opaque1 encodes a myosin XI motor protein that is required for endoplasmic reticulum motility and protein body formation in maize endosperm.;citation_volume=24;citation_year=2012;citation_pmid=22892319;citation_doi=10.1105/tpc.112.101360" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=G. Wang;citation_author=J. Zhang;citation_author=G. Wang;citation_author=X. Fan;citation_author=X. Sun;citation_title=Proline responding1 plays a critical role in regulating general protein synthesis and the cell cycle in maize.;citation_volume=26;citation_year=2014;citation_pmid=24951479;citation_doi=10.1105/tpc.114.125559" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=J. L. Wu;citation_author=C. J. Wu;citation_author=C. L. Lei;citation_author=M. Baraoidan;citation_author=A. Bordeos;citation_title=Chemical- and irradiation-induced mutants of indica rice IR64 for forward and reverse genetics.;citation_pages=85-97;citation_volume=59;citation_year=2005;citation_issue=1;citation_pmid=16217604;citation_doi=10.1007/s11103-004-5112-0" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=T. Yin;citation_author=D. Cook;citation_author=M. Lawrence;citation_title=ggbio: an R package for extending the grammar of graphics for genomic data.;citation_pages=R77-R77;citation_volume=13;citation_year=2012;citation_issue=8;citation_pmid=22937822;citation_doi=10.1186/gb-2012-13-8-r77" />
<meta name="citation_reference" content="citation_journal_title=International journal of radiation biology;citation_journal_abbrev=Int J Radiat Biol;citation_author=R. Yoshihara;citation_author=Y. Hase;citation_author=R. Sato;citation_author=K. Takimoto;citation_author=I. Narumi;citation_title=Mutational effects of different LET radiations in rpsL transgenic Arabidopsis.;citation_pages=125-131;citation_volume=86;citation_year=2010;citation_issue=2;citation_pmid=20148698;citation_doi=10.3109/09553000903336826" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=L. Yuan;citation_author=Y. Dou;citation_author=S. F. Kianian;citation_author=C. Zhang;citation_author=D. R. Holding;citation_title=Deletion mutagenesis identifies a haploinsufficient role for gamma-zein in opaque2 endosperm modification.;citation_volume=164;citation_year=2014;citation_pmid=24214534;citation_doi=10.1104/pp.113.230961" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zuryn;citation_author=S. Le Gras;citation_author=K. Jamet;citation_author=S. Jarriault;citation_title=A strategy for direct mapping and identification of mutations by whole-genome sequencing.;citation_volume=186;citation_year=2010;citation_pmid=20610404;citation_doi=10.1534/genetics.110.119230" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Population of Deletion Mutants and an Integrated Mapping and Exome-seq Pipeline for Gene Discovery in Maize" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="twitter:description" content="To better understand maize endosperm filling and maturation, we used γ-irradiation of the B73 maize reference line to generate mutants with opaque endosperm and reduced kernel fill phenotypes, and created a population of 1788 lines including 39 Mo17 × F2s showing stable, segregating, and viable kernel phenotypes. For molecular characterization of the mutants, we developed a novel functional genomics platform that combined bulked segregant RNA and exome sequencing (BSREx-seq) to map causative mutations and identify candidate genes within mapping intervals. To exemplify the utility of the mutants and provide proof-of-concept for the bioinformatics platform, we present detailed characterization of line 937, an opaque mutant harboring a 6203 bp in-frame deletion covering six exons within the Opaque-1 gene. In addition, we describe mutant line 146 which contains a 4.8 kb intragene deletion within the Sugary-1 gene and line 916 in which an 8.6 kb deletion knocks out a Cyclin A2 gene. The publically available algorithm developed in this work improves the identification of causative deletions and its corresponding gaps within mapping peaks. This study demonstrates the utility of γ-irradiation for forward genetics in large nondense genomes such as maize since deletions often affect single genes. Furthermore, we show how this classical mutagenesis method becomes applicable for functional genomics when combined with state-of-the-art genomics tools." />
<meta name="og-title" property="og:title" content="A Population of Deletion Mutants and an Integrated Mapping and Exome-seq Pipeline for Gene Discovery in Maize" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/8/2385" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="To better understand maize endosperm filling and maturation, we used γ-irradiation of the B73 maize reference line to generate mutants with opaque endosperm and reduced kernel fill phenotypes, and created a population of 1788 lines including 39 Mo17 × F2s showing stable, segregating, and viable kernel phenotypes. For molecular characterization of the mutants, we developed a novel functional genomics platform that combined bulked segregant RNA and exome sequencing (BSREx-seq) to map causative mutations and identify candidate genes within mapping intervals. To exemplify the utility of the mutants and provide proof-of-concept for the bioinformatics platform, we present detailed characterization of line 937, an opaque mutant harboring a 6203 bp in-frame deletion covering six exons within the Opaque-1 gene. In addition, we describe mutant line 146 which contains a 4.8 kb intragene deletion within the Sugary-1 gene and line 916 in which an 8.6 kb deletion knocks out a Cyclin A2 gene. The publically available algorithm developed in this work improves the identification of causative deletions and its corresponding gaps within mapping peaks. This study demonstrates the utility of γ-irradiation for forward genetics in large nondense genomes such as maize since deletions often affect single genes. Furthermore, we show how this classical mutagenesis method becomes applicable for functional genomics when combined with state-of-the-art genomics tools." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
    <title>A Population of Deletion Mutants and an Integrated Mapping and Exome-seq Pipeline for Gene Discovery in Maize | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"A9nK8m205UkvjVsKBMeG7yo_gbI1FNwGdRhfZ0HnaG8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460437","apath":"\/ggg\/6\/8\/2385.atom","pisa":"ggg;6\/8\/2385","ac":{"\/ggg\/6\/8\/2385.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/8\/2385.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2385"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2385"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/8\/2385":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XqJikWAgbLE9nbRBBz6+ktr6XkOO0slzCihimmkQf26bKgKVeY4eru6GV\\\\\\\/i+Jt+k4yHnIPcrPeHx2DiV98dooVK3UznGPcjid5Eo8WXmJ2cWxjHYCBTbiMKmc7q5GRk5Ng73ShKnWN4Ux6mSxunSZr9t76u3l4u\\\\\\\/fyBMRlyMz6d7RwrwxQzl+yaXl3dQwS4KVcyBTmqxZ71o2b+PvM09rrpia0VotSgxifT7NBcq0VuVLM56aE0UUQJX5r2MZuaZOpau2GyoiUv4RiswIMn6jL6Qp0hKCkf\\\\\\\/84qyVcJ2GjDJ+MovsTO33lZWDx5wlC0\\\\\\\/uMXt3jeWud1hsNfyBvX8YYcNAZuxlkZRSI6o7K8lKU6WsXVOKBhr1CUlNEUKYGw0JFZ75mynSK8q1RG\\\\\\\/rFfO3bm2F8VNDpO8cuszbi9GtD+Xs1qzohyL9xw6JGjTH7BYrWWHX4K8spqi04RCrnX6xrI37fUmV3PU8fOLjdt9Lwp42vHwT79SWVmgbuAcTD3TDxbGsGYFwpqnvw9BD2G\\\\\\\/BRXbKooG0cBFOEAgkscbPUZpskHOnPTd5OPsvjIcwvr4UNDZjYQ2BfujZ32I+vlTDggfh1qMQ9AVwg9DkCE\\\\\\\/5EDPoaQT+BcvcVxOormVl2d\\\\\\\/CwxvksLR09PAEpIEhGehMFW2YMW0i4EunrEvRPIGowY2ZLmOUeccINhXLrooOpHUihyGm03U5LaOOEoQ3zNJgR9o7JJqihBvQmj\\\\\\\/IzNH5Yb6o4jij3WGZ+GWnXdiUa71ite7eSYvPCGBR2s3FfWt3M866CZ3WgAnTW+3cE689\\\\\\\/7ilnTj2Fl05oOzFSfVWrcPMZqLuadXtDMN\\\\\\\/fFGAUC0i+Mrd\\\\\\\/yFzpcAIjAoFe2LyygFkezm03F8gyVhl1\\\\\\\/te5LBjak2AI1\\\\\\\/LeGG1ql7SO7\\\\\\\/0TUOZ+mndh\\\\\\\/n6IEvHSzCeKUv27LVMPIpEgXuYp30hN2krLz\\\\\\\/dFvfdHdr0S1tUXeRLTGkDrmPRcaUWbUY1OYsHpDEvZdgVnxSskCNf0E7F2\\\\\\\/390i1BiCC1OyJ\\\\\\\/Vlpm3jBAsfqjZ7JbUp8V8b+PANb09HC5Mgg6XvaC3vYG4nQmdlMzpqg93rS5Iv3JtldFw6EoxaAhjgyzW4nv8N1ToAH8fWaQBShZJq6OOgWVF4aGaC3V3nQ0Hvi2gy7DrjGVQiF8t\\\\\\\/FXQw\\\\\\\/CS\\\\\\\/fNXuv3uJZkvzAqau5rw3Etis1aWZOylqeYBjyWUTtrRA5GEuGKItgRx7INxi6Q\\\\\\\/489oQiRwGXEoUV0kRH+rBS2wTLT1HRmxBu\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/jwyvsPjFiZoI1T10tZJhQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e05326a24b54383511aba3a4b92aa0f1877ebefd9b4579528fe10fad53038890\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226RK9AsBp5kCjgpCLXV3NqWjTQwWm3mrSHb68625w\\\\\\\/OhkZc343NJXnsCOOBPuWGPwj5FkNnVy0mu6ioVpNaETzSEjVQyX8UyWvO+jEBLeVEH01TacCzMDLRzfvzTt7zSPw2b9K5HVmbq6S0ZUcaIFGI+sBsBVJIQl06hSp\\\\\\\/ZN2pimu6rrY\\\\\\\/09yvKRbhceE\\\\\\\/uxzB0tPu8uwe+XOY88wwn1vvLTCe4pV5QpF5\\\\\\\/kuy8W1YS3+zkPf+6H5qinCqVMSRhEpmRVy99zLlHEThMScK032G67v4kaE689UehYYq7PiL\\\\\\\/XG2I67XCrtoSZbK3wPq4yze8MX6hxsiXLoX+Uhclj1R7i+csP9NkJrtVc6XZdHXTBYMdlFVEVIntrVHjkytoG5\\\\\\\/Ha9mEvaqO7R2mrLbkCt45Ue0ACJ+KQe1Ps3geN+TTrnF3sHsvQtKYRzbaaw+a13vSJdgkgQvTheQEFS4HNT49TYatyIaty00huOr7j9OfeeSQRAm4M7y05BrLIJSc4cFHY1uc5gJ8PjSiKM8fXgkIc9uzGQNQuMZYY\\\\\\\/vLecOuDUVGoNgCvUOgJIDS2Jemj+I5u4EuMk0YQ\\\\\\\/D0rtwAjcqV0fuY1SjJvc6yErLqNI5p05RPjI44FqY0a2I44mS9b4MPL\\\\\\\/ZZDT1Pp2O\\\\\\\/k0e2Fu0utz98D6YAHqok\\\\\\\/tEMobtd3zAkDwnBXFkEh134GL+EwvQqhMPRvZDSty\\\\\\\/2mZovwhlJCkbOKBUEY6iMB2HBivuNVtZUPc1G7HNBzTbUkx7NIQ7gNwqM25LvlpH7RAsreGnStkOYTQxgWOtVBcvk1iDlLlf1waCpLuHtwMmqd2dMlflzLi7ctEZwv8YC1lkaqqT0ad4ECpGm1abnmm\\\\\\\/EsNFDUIGjrk2MYFdHhK6IqT\\\\\\\/IhyC2qUbdv+EzD+pCVc1uWZusS\\\\\\\/vGJgabyxhK48rx1PGazHWgaPx1jbrMJ0xDlf0JT\\\\\\\/qFJbuxjOUO6iwLKWEL68Xqa5SiJWsS7BvG6GnjlFt6wz48oAEj8jHi3q2V59zlbE9bfFmvGuwNDSuf2qWs3ab\\\\\\\/UdO64saSKDLKFukg9qYLjWwQmOd7qOdt3s+DkZiUNfZPLuUeBx0GS8a\\\\\\\/fsJLlNx8WuSpd5KOkXoEuBU1ouCOQ95ixlicqhI8hfTPw6Cr\\\\\\\/qECXrGy4PeNbIfDRxkoPlsIBnySgqbX9QgU\\\\\\\/JXvuOEDVnFOkXJM7sBQ0MomX8jYNGDb0SfvqP48v0oBE5\\\\\\\/M9\\\\\\\/5mR0elvNbX+47SDvEnSqAW4efz3Fr\\\\\\\/H8Sa9D75UKyod9jfKQXgM+HDA2P7E0hZ9C2JzLWc5eJEeTFWqdgDNbSn0sf+nVpXxNsHqMEKbjWuOWhbMu7GDqa7uH4LiRDR3CaPWATNoAIzXUq9dnX3Y\\\\\\\/Ek7Gj57Ocs2SvsMJb4GPT0bD5x2yUkB8ZGHipeSPcmsM8Zw6\\\\\\\/oO+lwbW+M3Lgv+Rr6HtkchKCzzpsre92IPEFMN36E9cu0V+Bcr603aJnXPcfafhnEdtaJdNFEeHQZJl+Ev942S0rCGrcdpbxbIoTBvl8u8577Hjeq4ibZB01emwF4zd29UNlw+Djj6LkCC9gBAe7DT+U2GVYFXzb0IgNbQkQbCgfQ+DiCi071XLzkPhFhWIf\\\\\\\/jc5ZxAx5eOCvqIyumz3291qGMj9w37M7rofyMbjFYQl3ezjAf\\\\\\\/oFH1BwDfAUbpq0qtyWB5DXZyG6ZRQpRqdYaAZSPa\\\\\\\/GU7TgxMIgcsmj47wOXCxEnSwT9q3\\\\\\\/E8p15groraSyst2i50L9QsoIf4s3Ysh1LBXGl0Tx0wJqyQN\\\\\\\/4IyFeSNFZKOnwpODDfszjxKMftvwVG4YMEsJ3tzpJu2XsysRCAFerPKJ9Lv52E0SkptqIad+3enFzw4Xb1MncbRGqs83zlKgwN86OnD0TqsxuayBBtFVD\\\\\\\/unABlKqboz8WXj+NP0joBPU2E5FdDkIigXMzIgiuuc4hxrr6mc2MO41HhlzKPfbNpgUdsPp5ggsFGlvu+GYQPkeoEFDux3xUdBR\\\\\\\/uHwy7NEg8\\\\\\\/SZ0BllqjLlPV+TgFQlOrJ8XhT1jKE1zo+H+0i44t\\\\\\\/zIAUs6Grc\\\\\\\/yQR58ojeNSNz8SYvMYSWeaP0NApms2kZo2zBqN9Fg9tH\\\\\\\/zDze+LEUAJx\\\\\\\/NyDk+Ci2+AEKPjhx0tLX9hhZ3VQcpCLswhHW5UPRBxuAosdW84p0YcJ4ZwLXS0Z1XdK063oQavhbBsVGL\\\\\\\/QD69RdtFxoXCEU\\\\\\\/B1LKDtw6uynQ\\\\\\\/5S3Lie3EKZsOssHCtMQl4xfyR9RmMb9rpXNxtyYORru59rw9BOxf3ql62RStXgEBcL1Pc6q1TCNkqOQAnqUW217uFRyl6O\\\\\\\/DtNz95q2cq4JSjVTgr8u8ac6CD+gnHS6pVIjEBGdtudHsKtmwtQHeYTeRHIU9a0Qw\\\\\\\/6QcDTUSqP7QBZefcdZg05XnoFXyiKqUoTILSjTA7KsZocAZr\\\\\\\/XTUYEgND9QWTF+KjpqtyoJnUc+3BUMFe4wOpC7l2qvALeSPI+FEGL3or6b4HMybKVqT\\\\\\\/QzSADmWaAx7itHTtRbD5ccYXMdGWFzy6SKkfcxTsa\\\\\\\/YLMZVAIMiJzxxhVgQ88+ZMEJEtIQJvDHHcVFbgUzf04Mz3OzxgJKSFAXBLU1rIHHgDDxQfJ\\\\\\\/bBo6Y+pgnaZhGAnUiXPcjwZAu4TYKkni8kB0pqNTGb67lN4rjY4YNfz2Nz4P9UVSR3Br85tJKNsS5\\\\\\\/n6xODtvd5xRVPN2PtA4flWvlSkhS03B2OBwzA1Ty8EOSXflombBsiVYE0J4xKxDxE\\\\\\\/X\\\\\\\/zUyMok3XFWH3\\\\\\\/dJTT4MJDtlxNlxbpL44tMVQ+T2S+F+ZuUcHL2LfU8c70vf1050p+0D+yrMcrNbSQDfzqWVKf+Qer6TOcd98JODtjIX5tsQOZACXhCzdQSSnFDPv9vpVt9Cy0s6zub8QgSkTnwr0kSERmJQ19rFhsTjOqKXxRSK6Ax4BUSAHF7o5Tl+HvQ7yfQWdP9VNgnPpXhyWOg\\\\\\\/Ripz8Usp2Yui6AVzbnMy8zz7BZFYju52X9YtDFK5L37N5S78Fr0WeA7CHyEW0eqNgcwywQdIoZnt\\\\\\\/26VwxB91e4dcz1FMIpPTNO1oGPxC4g3HKThoPYzNJCgOTbPytQqZJPwi6ti1PosH5ZldF+EERBaaG4qN7fUgfjc5fNVaQYmrmeNEOaYUb4mx1\\\\\\\/N3bc5b4s+bMmQ63Ss9yJPTpFtu87lOgOo05cb5BnYtSQx71g\\\\\\\/4SW4v6Z70d2165AbxY8wjsFR9I6IRljhLLyWM9SkKqxiqjqW5SIT+f8Q\\\\\\\/SwiU9aJ3Vkf6tqbTSdmPBCW5yDXPETnZa1zkzRsp3qi3Mcs7dl61cAyX8LFchrFRVoZiiPmXl3ZvPCUK7wEL2najN70QEmbV0bwPOYNDQ3m0iOPE7m+9gTAbpGDvLKgopq\\\\\\\/dI4IRrIcp0EJrAEnRAC8cn5ugGF18XrabnPJdlTf2om48+bKsGl6hL3SzhFGk8mGVp6bkg90olhR\\\\\\\/73aGg2v8dqSLP\\\\\\\/9lZHB6nDCnnrITo1rbDBfDMWzpyFUJ5g+Voy6g6CTk9fU3y4A0MSatzP+TNB4er8+jCq+jSaYwCFxCE+Jph\\\\\\\/3I8q3mJcjXh8WJ4Gu5IfidkTqhLCwrUMvF0vymP+3qy10Q+Wjo\\\\\\\/iIllkyaZPh9hF9cA1wcQ1vcYj29TsIGAl\\\\\\\/qgP\\\\\\\/p0euiPhO0TzBOlqlzlCsTr\\\\\\\/z0CmkTFxv8i0sJx6z\\\\\\\/JYrzEI6230F1ORtXabqkjBxw1gtVDiQdlItUsT4\\\\\\\/kNObWwIbq9xfuJckCBJYAIMfJIjNcFZj01sEm5fMBnB+nlKk9HvTgxZBqi\\\\\\\/F0WwJnAvk\\\\\\\/VebSdBkNpoojdDsYVbHsJbhTVthQbghPh0o9vTbkrFuBgeV10ME1GfcpLt4jK0P44IjokRJkQmuJhzJ8gPKbMZBJO8hPFq7a+S55eJKmvbIZqkoWgpvYg8MfPqHFerBzdsguk4N7F8ZNCzqnqMAYwC5ERa1AJrbCV1DgNFWT8clWJqxR6fqmTsd8qQOIVZ+abhYEFI8t50fajrH\\\\\\\/08gs6AH5f9FKOuVlq\\\\\\\/TDSDtJueYuAQjXZ+Nm+qKzfrKvq+cCwGjkMMThqhcptctMPqly0XPzeG5tbhu38tmkJPWsMIepxf6VtHXNNig7FSaSm7r\\\\\\\/Z8CVGQyeQ5qLTALP401nqutprXSIhZMfwgwQjTY90i4TVH2SZhKVdpBfUwMSPq85RkohflOU4xKhvne5JJiZkgGesenC\\\\\\\/QiBP\\\\\\\/29C06yznXo4U2I3Y5yX5ipNzmDD6oJUWffudZcgCB6w946uSEoVwaY\\\\\\\/NVAwdqXmLgxTOjKphP6yy6UevFu51HGf2nnbmB7l1E+\\\\\\\/IViT1sWnrWQaGMzonuApMqZgcHtNr1Dpdwu1jhh0w7ggxdlrRfeKaN\\\\\\\/GJhooMAIQRntc1liQIkAXro1NbgSwn7ARcuSiI6prA5uyL0kRKhG8jBoxstrUMGUOQQZyAuB8Cfv4ukLfbdiJMlj8atTouMxD+SO4b6PKv9VLwgQG7e\\\\\\\/WmVYwCT6UVMkGcSewiXLpnipx0B+McdnyBrc2J7faL\\\\\\\/Blh\\\\\\\/5DlOTjCo6+pccowmQ9BgxGNUoIX8b87VEkQfAEbYFMcKDavMZyLXDOlKB5xYaHmmxn3pGxdcoaPqrB3YMaYYWSEabGv5IF\\\\\\\/qHJfEj7RaDnMnqTzGgg2v28fZ7UlGi\\\\\\\/yEhjuDvjBWqfzRbd0+1X6VbwtRaMtz7qBeKMO8i9Gq+dD7Lde3rbL\\\\\\\/y0XThTXDz9tzZ\\\\\\\/MMWnxaupNEUlfJL5yOG5H8o1unFC3RttRUOHAGoJfCNm1EE8DQ+YXJ3GTTZWGhr\\\\\\\/i5+c2xZMa1u1uULRcJuDWh88wKf1u557CzZZOpf4qMhUt2miuAUQLZCr3dM6Vpk04A4XsWPDx4IPvUQqtowwMZ3jr++X721eNEuNP985OsP0+TUawFHwFebBq0jNeC3JeLoF8xlJ7xKIvFHQmFa2XYkfFe7s7DTSmk+zlVWtJ3I\\\\\\\/KUbeXJqgG2SXWUhzCZvzIALKNR\\\\\\\/Gz1Gk40XqejO2I6MIgpsQtpaIXzT\\\\\\\/waGpyv+ydOH7LQQemzipNaTtsLr3M9LbeRCBQ6Q9GYXoR3VcbRnes0hTaecpRLgQUiI7UBzSiySSA5U1q1JanPEn4zT4AtvwwupHYf8BMJPoYRwW1Tf51CQXxJrhygU+eIy\\\\\\\/vkTR6748Zf7s0orVBF0g8T\\\\\\\/LBapCMhWYvhdXnlo1eIdkH1LVY2EKLpj+Cl9P4HAgQjaQLNP3GIFEc6b+JEjM9BCQ+o9eOuCqNnvJ5rsn+IpA09qCj\\\\\\\/Y0habaloHfvkVTDclkEywcXoDIj+k\\\\\\\/ge+FZRPg4wyKp\\\\\\\/cujnFGUDbjRClYJ2k3Pbn29mdPE6O8bC3Adaxe928jpuNPqG+3zHUhuCQs3ieiCFtAe6MNrvGxELBVOIeE\\\\\\\/Uh42ZynRxyt6iFavYZs8YZPsh169f7\\\\\\\/WEPprLBdQP6iJHikBJXM2u+oU+6kjoQ9+7KJfgaKmhAjcrkz9N8UdN3SWXTFfdqB\\\\\\\/fyOVPRAHNW4VINgqVzcAGxcQOFC\\\\\\\/d5tYx72KXe5O8tf0P3zBS1lM8PAK3HAhE+T+dFqVVN2F1k9\\\\\\\/ff82GHI0Hf2gYZoN22Ss2g+aokZL4wDzYF0XSuxmY48pToiYxrv1kI0likT+BHdZ5GXC\\\\\\\/mENUlRJPNwFkGTJzJegkoCQYhaP7VKaxMCdfG58EnJJBgST9rVMpft6tmknJAu3LiQ6A5iIZ5FdVCs4hA5Z8pkDgHeCuzto5TZxeQaMEx81DpiX+UiI1aCu\\\\\\\/E778b088fq6XoF1kJOgGoJ7tlP7OI8GJb6TXNrn+d+P5f\\\\\\\/DXwaLDvqvnG+r5d50Izjta+WS2mKF70ROgsRdrBNpglF8vEyBmagBb\\\\\\\/3LnoBxEiTqzITwJUuXVlTkkg49tcnl0sfL1+ippEwCWFiSUtp2wJK28g0apzh3v37zt\\\\\\\/rN2wb04hN0npVrq6lBQD6KownJfwTdOUxykAt8OOaszIS2bK7ZQZ53tXCJWljGuPmpcBfyj9J\\\\\\\/3AfQOA2B8uvEPHHDatVTfxC3nUiLN1f9IFkPEy90qy32xxjPArS8YR+TYSaj0iVlUa4QuLwCvVy4Kc8BumgbTOkva2s\\\\\\\/c8lU6IXKSSi84eqabM9Ohezm2jjoIlKpzLf9owQeld25NzTUAXWjhJQo86syZWne4iRNtHwSfZS7ITZaNl5i42kdIZUba3UX\\\\\\\/n+4ZxlY72bKUAHm8FFIWRnYsmKGdxPNyrIfE4DMAbvDX7d756VYzAAwbzP4e8o2A0z58CC79eeCUAOETJz9OuyfOisRnR9rmVF6ER+vQJiVuUH5VUMh40F00kgsVfo+2kQmlOzGxIDLwn37Ql3LZm6\\\\\\\/dsuPXR5MVAEdSmHdHcmwqP50DbPGcFT6\\\\\\\/ztylL+jxsIxZayr1FoCbXRXPumf8h1ov2Z2Au7KvhrSimSBPZK+BgRtAPSHJbfQZLGHI0v9Gmy81L\\\\\\\/TrI9O\\\\\\\/kPK1JTn7+QBjxHjwuYTRwcoNmBe2oFfsXNadGPfDO6ddiP0ZFdf\\\\\\\/3f2ybWhaJhm99MlSMGvUFqhSczTT6IduyO6gSKljIGRPZzR1eKVvIFNlvP8a6sFObXdDEdjoXoiV+CrJbPsuw60dr7bTwVBoSaptC92kr8mgS7OYruGfp7pc9JblTZffR6UXREpmVPy6kLquFVMSCKvQN+dUTWam1dWiDTqC\\\\\\\/tYzGWey1q9TrN9uF+RWPx8moMh6eGKSEqKNs0yGYRwlFdvM+47SHijHsR\\\\\\\/gBAW\\\\\\\/GI9urkBlAcksG4mLxcCWP9pPvYsTI5FKndhN1KAE2xHGfb7R\\\\\\\/NJfzBPUVSq\\\\\\\/5Cxy2VwuovABWqkTG2auOHjHFZnwzGhOguXLk0tBp15\\\\\\\/4qEzFkT6Mda\\\\\\\/vfD8fJYKuBKWOMMTwEwwaNoeeFUJMExvMGNRretrlQOvjg\\\\\\\/3a6dN7db1VlIq+U\\\\\\\/+4UOACUKhR7Sv78xKmU\\\\\\\/ucHgmVh+04fuTY4JzkTZtJxbIWaBH7nezm29eVZD0Hdl9K+XYbV6XO1rxTxCEJbZLWPbnb9zKGArdSToyxIwc1JR3+P3JNBb5Ha7rnd7WvgPMb9E1JtOMIEsisO3EjYqFTEsZvmYoBui5JKc61PMEKfzol0y9X87n0zcZLi0HEvaUUvzkS1+nxDWWBoDJVE1+lQWDVCbLLHuP75yXXsB3YLiXDUf\\\\\\\/1bN5SiV5nWHHc5SQAyi8AisvNRk1hT\\\\\\\/FfSioy0Ej2Yd8wKdzpztPMcQ8l+lIaFT9J+aCxHcsXH2Dj3IeNvyMJF\\\\\\\/T2pXIxzu6RreTutQ76BQ\\\\\\\/eKBgGRjLgqdhLJFWK\\\\\\\/zfZpIYfgcT2XwN2Ci2UNmRZtv8Xuc7+tSwwo8BOLoGBrSFNPGQ38cioLkhbuiuFh9zBVlvPQwVpGEbKcAR7m4cNSh40Q3rCHt2ofVy8b7OjTSCB4wQL9ajAxhGJ7xZYd745r5EflBsHXKAXeou09djK1H6eVuLm0e0XgAvBllem85pB6utKh8DD1AROAXJjDkumxGqLepKwDnZBnNak08n0nuQOVepZ\\\\\\\/87dHp14j6\\\\\\\/+p\\\\\\\/j4sZ3jQjAZLvRkOKs1LJ5JMYAyUWTi9YPnvTsxJak8eLtN6RvO1\\\\\\\/6juDqQVxgOA8t5mHkso6HS+cw9ec4TFnk6I2miXFKTdSNyxMoq+n\\\\\\\/u++7jw2bGeH1RF+s9e1XjCvareElbgXJS8NOrYtDcAuJkj8i95bjLq+dbJIg9I1puNQK9dAtRhSAKiG1i3BGAJhxNIyJlVj6D0Z2ipUmeRPsFkXSEAh5H\\\\\\\/AVwfGPcQ\\\\\\\/2R8Wjic3rtfgfaM8OfJ5Zmb8juGG3wB9dQI2I4lk\\\\\\\/lDZToiEoUuxH+\\\\\\\/\\\\\\\/SfeWD4Tzn2wJB+yLzRZBsf4DeckCVpZbB5O\\\\\\\/OfVKWomQUUuCXZQVJU1oJam3bfNXwsqVqDlF4hbErOi5q5Aig3k7bTrUEkziRPfBLqAtLkrvuwP8gLID5ANyfyt7WmY\\\\\\\/f1MyFW86\\\\\\\/Ro1UJb0eNb8E6o5bGHXwW1i83UtOHT3atkbmK4qxh1\\\\\\\/idRvyalZ+3y2aiICL6F86vZjyIF9n44xlp1Yr9bx1lyEpGNipVJdl8DhhHBB8ob15fap6vSgOvf2pz0KRbyn9TWydihFbT+GFvP0oA3gA6GP5rUB+MXzQYpg0le6qtBgWEgQo0MMce5Db\\\\\\\/6vmJICXqrmKS32blKcwotF8h8g1s8q7jeoOcZERN6X5U5TzzQXrmp3GZQlGYh3EOjnuojuVwkosPkLp6Y+7SGnKaHN8Zw4GCvOkzJOP6j\\\\\\\/7UN75m9Y8DxHNGf9DaI9g\\\\\\\/i4lAOiv5vLEjm8njrd5sacaKs+4vFsXR7H\\\\\\\/XQTWh2eVtsC13SO5acHQKcaSXNs+myMh95WgsmCnZ\\\\\\\/CEz+mxni6AlVQt9sNOQc7JO7a6JBfB\\\\\\\/nqFkoMOgzh7lj+Mq17Eb5t7QUpfnexE2vb4MOxSnp8KMp91R+P\\\\\\\/jFO0QPH\\\\\\\/lw5Zvi3eGqJMtUEXZvM\\\\\\\/AmD8bPyqqA+Yjcc4Hli9WjuRRgKBdWm3eOkGl8hh3UzRzciAWalRGFiYJwozLerNpULONnpJgyex+By7gsJA0d0zjfDILoJLX+WyhpgIqc47ghlWkNp8COHpcVe3zGe1Fe850UzMDXNEaINaQiNqZGuvt\\\\\\\/BWGrE4m4dp8rgVkVUR6PScr8FOXIQW3aQ2NkgnXlUUlIFmVfT0JDQovJm9p9eQg6VYFzi+2iUXwcGbxyJNL9rtZEwRo+EtQbpR6AM\\\\\\\/Y85rnIKbxKicjAqKIWNvkTIph7WPLdVUbnMBcRILzFPXtM3LhlJKWOIbCDyrZXBJNYEJVq9hsLLEuKqLJtMPSZlAA+Q\\\\\\\/\\\\\\\/ZUfevEPTJFhULIfQ7XSTu\\\\\\\/m3EbwfTazsqwfO\\\\\\\/eZjqRCfnLpoqsHpZdwXsuqXNwDGf9ygUgigZBy7xMwi9\\\\\\\/Ms5rBaIk4SDgoRid5nTvKm6VF4OCjt8et5dz5IOgi\\\\\\\/7TwyF7fG4kLDzzVQO6MrzzklWSPxQoE40Fi3uqmuI5mgFBzQAiYwnh2BFHHfBj+SlwKvFGfF3sO9N0S1k79oMVWo0gmTGP9HjiORR1RCRNNA0lb0NohoeTRaqHoWUtYqMtM+ETaLFzJM\\\\\\\/rlRQONuvFWNK0s4pA07WqJKNYPtIpUaGKFwdki\\\\\\\/k9PthPZDrKt9S5xoITamB8Gmt1nPdW2VpKpF5D9jDPJshPq5TCLJJj8C5LFP55wFh8KTDUXOmCv1T2QIL5zjn6By5zIn8jvrIE1J+913ItupWSkMb17rD+dVUpF0ZLWmcm7dAHyzfaF0+BmSJ8lKbZXDypor+LzsJu9ilazEoo7FwiSVTTN8m1a2TUiG3bSNmll6O19qSFJDGkZbOX2yt2Fo\\\\\\\/1ykSmrli0sy1xhqlcqEqkM5nGxM5RJ\\\\\\\/zyLouDJuWEYYutH42YxYOZ9532bG4lU1CPpypX2tS0mk3B6PcBdsk3tMK\\\\\\\/z7\\\\\\\/NgFN3ZT6a8DAk2mEyQnvynk8slCSj0oBQofzfGm92zGAoZZyqcuCr6kJwfW41lN4VSicyOAbQwawRP3gj2+hKZjQvmxdgp\\\\\\\/xqLs4wC2qMqeawUtWflIIfz183yZ\\\\\\\/jorldl4W6fF9DIlC8ib8jpL8utR2bDjIZTAE1Pfh+41fIQ6v1vJyfHPdfznS1fRFCzRZ6bkXzCNVcyIVWTJARI7NhF9Jh70e8FSg+yAkOhaiKEsx0oDrxxigmc8v502JB0bLi19i28tW6Vzznk\\\\\\\/qJ4Zhx\\\\\\\/mql2z\\\\\\\/9WSs\\\\\\\/49WfR3\\\\\\\/ixo31u++Qq\\\\\\\/tZeIfq2rO8Jbe1a3K7STqGcFqk\\\\\\\/6\\\\\\\/PKUIxC4axHlkmVFTXcaCaxqPgFoVHivfJmb4zMa2TJuVCPa+oPOQ+IJqAShS3FDEyAnFsOCiJfI5ajHEVP2+1swKpRVLIGXrd1IgFhLHj8JiVcv7zRdtl\\\\\\\/b+tN8iWszDOhIuCTpheY+aFK6BSvgr8EZqyqUn6gEJ4i2NdCAv3uTrwXv3SmgSB9MYXPGHULZqPKvOiXpK6OGsLCyF371mHFZNQzAhSPF\\\\\\\/dTqTOebh\\\\\\\/mgG8r8+uOyMqlFanhGIQ\\\\\\\/gFL7scHtsLOkwltbIUpWauqRxWYCwWIGRyg2\\\\\\\/ueliE9KQOKu8G\\\\\\\/BSxkjKRFJ6gv070Wr1uUmdP48AJ+4U0KpMK4iqWQIKLjK9WSEDuVjH9R9IVQJnb3skd6BDmCemTKveS65dkOJUiCYRb4wn3thfbZhxveqH08cFLxUEnsFixRHFzoEP87KGvN+vtwQy2yXBWmd2RlWhx7gHbPcLI78Uug\\\\\\\/o29mLa25zbyIianqb0VMiHmkLvAy80DJ1v3Q7a7ceI0VjxPMnKWH\\\\\\\/xqV26FtmKNxKjOB47uoQneX8KSwz5EUjVQ\\\\\\\/Obeyv+22tfGoZbX4OtHuyGCuIJWgVczsx\\\\\\\/h4p1oni6m2H9JglQVgoqR9WRclRujG62\\\\\\\/b47Hg0AQKCi9XyytU1R7gbzaxw+2RsP5p+eaHfMJ9hS5JKviO3BOgh7GrtEWDXOMlD1kjwE4rAlrAT9aDu3BBMMDKHiJIsv6T+6hmo3zyPbcx6jl74EXkgWDioQodq9vUFLlsdsujbqmtwqakpAvhT3cLmO\\\\\\\/QfWvkC3tGCTUX8FLUzbeOsn42\\\\\\\/U07qQif8L0T\\\\\\\/\\\\\\\/WnLqODiLP2sbaZIdt+LYxJxZVr3TtwPmiR\\\\\\\/gxKApvnuMm8\\\\\\\/H4Xopd6gf5LqK0kr4SBGmaRvUtVGcc+iIl7oJLlQm6i5MnwPj24wnhBt5C02O8UBolcUMIEjGqfZzkstr8gMkjt4O6oIDh35uDgH8SxmKTLbv1VGTqGPKkbwG3jFgVBphZA3varPJhHasuEZ79lHgeXD5ZvW+TvCqkBJl1K5edslU0a+3N0NDWHF4d0Z5SRqEd6LhIYv6qNEs2iHK9Og\\\\\\\/PGumUn5jCIgDDOmSKLrdjwX\\\\\\\/CoIaCUNZxubseJNubdpodiLMEw5TMM8xQ7wD4RjyO9RA+jFZX26lvmCHJ3WjCtVaJp0hOvK2flUItQgxcAdmgkcLEn2cMiX6jU1+IuwlzbkHxPE+YHCx0tdD4kvYD2RYs0W9nZnU\\\\\\\/3rFOmvTHAO\\\\\\\/Go8JZhfQM8gkSU\\\\\\\/9er0ZijiNoHzLsWHSK4nNWjEPcg+2IS8qXp+BBvCXJHfLFOQ3Na1hSWqjzbsf4YzQmyEL78QxPnU4Q7agjjKILE+XtPlfB2wzfYoJCWTWlUbEMwKEEX1nYqNdMxsFQe+WXukshs7Ty8yzuX1svmr52EOQrNxUFBOF9Dtb2uwuhWD3j6wBnC4BD8Csu5iQNUcjT7+30H+052qXObrCTIiMeWDzzPMKxoyYJCeKuADNMBHyDdyHHKZ7A\\\\\\\/arupHiK1QTkkVGhpkOIqIXg51Ajcg828VMGjrnfzDrnivcBKIXx616mdWicHXyGCpRtvkhQBvEFZlpJMUnukrqPbu\\\\\\\/lJ0ce55aeauRDZd+undNkRIals7pCrkAstpGCLEZYrFfqOknX\\\\\\\/IxL6Yxn0WYRCSbAqOiwzkrcX9oq5aQm9PCQXtRrxNld+rxa7N1Arn8nksaR0b7RhArH36+csc91ExRaFN4YrA+njc2cUGY4\\\\\\\/DGLm1YssRTIW8FiTJ6DNNjz+6QQeEiDT57f4rIRnLFzQ80iF\\\\\\\/ha7AsnpwmkIQercuMU5lW6WnOyQ1DZk5uHPiUx8byB7hNmj15sxhOzBrjAPeXsCoxGs7TmzEvmFgrPx3oIIA+ckRasxNzquCYK63bZ\\\\\\\/6K83XanPXMpxFQnE4U39IoDmUL0BTb5qivafCFDPqGBUL7SO4M+7GD6wRAkOvg7qXJbmDKEi6SewJ7v8rBhoLPA61QfS1QG2677qsMCTjKn8YLdRGo1Tn+PKN6aDKHvxAexCUfd3aXqfxUxboBbbMixb83+xcuo1q0lK7IOMEfc2ee6CanLGdmUUhW1ND6T+MGTzrqvbwPPwYRnbbBAGsF0Muo+hLGTGDbkmZ9f66t2x2SS45PT4829YJI0gVaxLT+AQFPdBLILOGs+qiRyNiiJB5JWCgqbOln1KmbEyFk1k77fM4ll3Zjr\\\\\\\/bXgEHnSrmvBcrO8GmQ77VaVry111KDfD+SDzXH1bmfbFuAF1dMJ7juty2yK6MLV\\\\\\\/HKTlQOBTHEUv+B9lHLzg4Kovnd5rjXEN6+6PI0SGJFSK6xy+H618yJR9QC07fK1nn7PFiwLUWEwTkYPT+mpM6mwnGOKR0RnIAoo1qgKRwO7qX2QiC8eLTb4DlGTJNPagm1u3Buri5uYSiXFsTebuZf+wUTDQWT1ubXgjWKfb20jF3s2KY64FVxBM6oz3unqDBBqCwwIkLFV1+mcmEx2wA7\\\\\\\/DiIYX+nb20jQNoGaPfWy\\\\\\\/6xInyuDWZ\\\\\\\/HWtKuClRvhL7aqAADKf865+sGy\\\\\\\/vGTTKpIDNTxQVnV\\\\\\\/w+7KzDWcXsJYmKBdDM85JB5SknRVoOkNKAwZRlCGQgcrspNDeA6YUompL9mSdPAzZi\\\\\\\/Kj6twLlC9Q21u14LtDX8KpMwoyidwP7JLEazy74QzB9M1UXLYXu29ygEVU2UnpEYmj7N6IdKusW7rpmxlwkLn8erHAh4RPX4SoLJEvMfUQ1rUbdaGaPuSU4iGThrB0U7pqBM2kndG8mL\\\\\\\/0xjpLiME2eF3RlbbCq95ZxL+z2VVVzrdUI8gY0yTOArhvA\\\\\\\/FT7sqLsvP2POtHrKlmHnQ9TW5YyDuDAUtErJLUyQj50R1caoqfLzjYiLzL4nt7ZO+nwXcjkJJImlWim6N0GFIBAnj9TnR66IyF4nnWm8OwfL\\\\\\\/UBv8xRjA2Ikndf9Qa+6y\\\\\\\/0+W9JZM9QNaUWPp4GfwjSw1yLpvArDVXpbIwRpwha4za4jqXFT5PpGj52q3ICGl+dJX5yt1DSLCD\\\\\\\/lvA4VHJv1Fp5IiK6lc7+DzOT5pQAzSon9vdrlIiZ4OJHN8IH4gSHV4KmHpVuqfg8gpN8r2D4jJmcFgc41eA\\\\\\\/1\\\\\\\/hxXufBLifdvSpRqGIPDaJEWxxhu7WCBM3+Is5wDxI3owr4eVfm9G2Tob4LicFvDwa8m3Ne7L6uAXkZPVsvpFH+089yOyAdxeaSK6Ouu22GiSk0lcu1lEiyc7S3ltidAKkqNPItO2IwYrxeUrnaa8eO4zqdRGveHHFBX\\\\\\\/\\\\\\\/NIYxc9T9PXB6cRilvKd2w4Zsrr\\\\\\\/hAYgieO3Ob\\\\\\\/1Z8qKFQmMOxwZHLFa4\\\\\\\/VCKZ2WYgIzLvem0i60QGvIv3GAvOfrFYRE724zx1jVjj5JC4KkIFp0b3APG0wnMQTzLBYkgg9SpFAIBeps2RexfVhzjFv+OO1FcpJcUit4Do6mVDTOldfnKa0NhytLSe3Q02m0+ljMAb5Sb5mbaPtL0g7hEZA\\\\\\\/Eff+1kDgrGd7MWQDNwHSN06P6EuG2M+w8hyKXBxZMWOXzhHUNHXGVNU5T+2wHiwOhKxKQU87dfy4Ao5i89bo9vAGDB8uRzGYDyxErBz8R0uVwCKavdtE8SQlcGex3R7HztQv3myzP69w5nKPSOK9BoSAY8DgfRV9ggi6Qy6XCDYC6U2QS7KI\\\\\\\/Xf2pkHAkiKrSRuXgJrSfqjzVSElQAi1R9bEY+hvVCTB9pEN6qmuoOjLvMwZDQbQ82D0AXJi30xoSBia\\\\\\\/Pmik\\\\\\\/FR9tdtLEwmHtn43bRDNCflUFxBDdjalhh08ZOg3XU4sptyV3Bh7ZZZpNE10UReZiewSyhHt2tt1yUW5dYqwM3xoP9rshte1R3ZdF9R8g7CYumEBs1WJjT7kVrTNC81IgM7c\\\\\\\/6qNtWMNfHwtys0vRh+IqW3s1iFJt95Y\\\\\\\/bD8BfPBbXNoij1sC32E4mNdPqGPq1oB9IVp4\\\\\\\/A7GPX8652TNvy\\\\\\\/sEEOTvkWVMEjrkKs4z7qUtp2REHo5d4n2GRiZC2Cp0m\\\\\\\/33Qwe\\\\\\\/YE8pit3VAVP4oENtfw+Gpwlx9BRms0oLUbMGrmCN+KlQFSD4h5ePzFtyfQkQ2j4GcC3qtrKB2BqW27li4gYDDGp1x6VHkTkDSq5lcHkbZw40EiYpi5ltlNtoHCp6TGSrMbMfzTuqjz\\\\\\\/Uj5fUrDyeM8HUbWg+a8EOxn0QnGEROZfe0ZdUywFrAwpAQyDK66eajfdo\\\\\\\/yY7t9EKnpjPTueMghbVNRs62J\\\\\\\/7Z2yuOokAFmabLXeUWu5RTqYrUMczhDls6723Oxi8nSZED4\\\\\\\/dI7a+a8Eaz4uI\\\\\\\/PzZ60\\\\\\\/y8nfJBi6YefACLAf+Sg65+TSzbesNvuDizCgGheS7Tk2WQpjg3FXCeXsdUoUp79Ef1bO1QB9NbgFXRJlU8\\\\\\\/sPCM057L5HmGvaCw58LIZAn\\\\\\\/owHpl3KYXOnYwLZ8\\\\\\\/31qlkd9M1eD+kugscJaHQTn1xd1ZPzSoGwse0Hxouavr+DEHnihqQYcLJIin6uqASk3jorG19khr\\\\\\\/mE6Swtcy9MNyHAue2\\\\\\\/YmUZLGD7IxIFOo525j\\\\\\\/WGzaBb0HCHYJDSS+K4EOilqmlxrb4\\\\\\\/4pV493qJ5LXBqXwPuDxZFkZ8UW\\\\\\\/pcAC2uBld6c3GHQuiZlYaNsrOtlCDctabo9m+yH8irzGjFB7k\\\\\\\/SIY+79iOgaFvf1Sa5gVKK2HiCA704cszu3hphWjzwGmvGLwkS0UB2E+lqykf1TqFFzoCZLmGLD4gZhFNEc1gsuqrc3DafOtSMBtrthI4Z1GAzQsdATojl2r5jhqXq5\\\\\\\/XuDMqzx5nbzUcmmbAlJBC31aKCADZ7TuMbyN2gLEZPAaKAQrtsQksyo5ZgUZRDLBUTQ+buOf3ZFDLseffupQy5OCOqyeT5OxV4Db0mtWeshf0eCzPMkaMz9NyDMGgUpYy6bNm47DCntqGt7+ajsotxLZhzm7pHfF+L8fLD4zZiVts6dEC0OAAj\\\\\\\/BvIa0kKt8OsOwHv+oqZhGf8Q7Ur6JJJfZCZ2w+JtCmJXTComYtaIej9ZT\\\\\\\/2qI3rQmpDJnngQhdHNaiQcDmUEcA0sQQt95dzW29zGcThGZ\\\\\\\/aEbh8k95XdveQERWqROpzs4KDwQv+HOXHry4tu6KaV5MQKF2dOSZAfu11kls6hc\\\\\\\/yjBE5aTI9g+71teafag45i+E5A+ESi7wI5fxRl0DBQ4FMxP3crEQy5M8NRX2kLgEXnaLD8PvVytLmjrJWX3pCAou9IC\\\\\\\/VUrpm5h7uxC\\\\\\\/Dypz7WzRLCWtArzRyjPAzOHTBZU5l2iH7N5fCHGIzSry7T6KZlQNAFW3kH0ptACXcHZs4IOI2JVqC\\\\\\\/BGHRZsMUXHX1WTZhgNA0SDYAq2sgxG5urm8VGNv8ddXZnuxm11\\\\\\\/Yi1bGsMDGehlpSr\\\\\\\/5tBVzQo10hdVyPWg7V5j40BYieLGg3DSuSo9ftg8MJsudC2HsRyPJpg2iu4PtJl578TMtCK\\\\\\\/Fr32sg6aK3t4QbURyV9KO9TmQpJbSILeQykCBapGoq2YVFvhbTZ3alxZLvVIshcA7l4+InGT2qIPnSOJ\\\\\\\/y\\\\\\\/AqkZfOIxgYXQtQvC0x4yUdAK1Jp6+PCU+d+WCLPA54vrjsrm500NEOBcb1yu0jEWq54zl\\\\\\\/2yhJxYF4D5EgHDuPTkOXAr60LVNSVH063NMtZgkw2BSoocwTTQo6vQtVrVpZaeYWbJVD50b7jlYurX33mCaSaF4i8wrQ62iZvCR95JWIFAMU63tYFn4S4jKuQHiM7rEU+wrW2hlDP8es63iwmZjMy7guO2+ZFRkvfpv2O4JdEqp1SQ+Rj3o4FYJ7DZgbkDbU1HZcaL+JqLGgV6kyAvsM1p7JW8VpObkhEI80NbhSLuEAwLFPRrPFI+2E1uPxDHn+kPtqe\\\\\\\/yo3k+Rlh4Ywt5FlHN25dgKt9oyJItDYbJ+oNks8w9gGATaOrmlCo5MaMDx29bDwjl5WKqgBiAfQBYax3XWT2EbIHOcjttXEdUE1\\\\\\\/Ql6wd2\\\\\\\/aEUmcdD5+L3nBUGNAp+0YhdABD1K2+QPTsup2SStfqzhGDMA\\\\\\\/NqE7Y2J225AcAApoOflLQk02taBMw1gfW7mB+92JgjR9mtn9pRC\\\\\\\/Wi9JVl272ATQMUKFr3pJUOItPwZe07VRnWmCgjVXRIz7vhhKraBIbXIzXarHN9tqg6Qy\\\\\\\/Y6yKCQYgTXmwbIkmGuhoWrfQJZjk9FRbkUAZBSMQwSqECb+XCb2WBqP7ghvoYx1lqwoZBHpPLCWFWJG3Mi0Jqk2FVPow5u2DeuyTgW3IHOm91jlSpz952TGsc5fVgrStut0Z0OHnFr0yu7BedCglMn3iynbxPYzKEYCIvVJdlq\\\\\\\/\\\\\\\/krF2Jy\\\\\\\/3gWsOek7URno7uByIZkAvGWy00gPItB7rU0T1WSXVDdWEa+2ltS1pnWWE5ZmDeIF9RWoguChUYRp\\\\\\\/wI+7UmDKf1fxFZ1uqXATPMF4vKycVoC6xFa8EfELBzHIRbvxP3x3AW4xXJaiIUh9zBNypbxKap7fuik\\\\\\\/3hmmoBaRLu089i7Z9iICcmnOeDZR9wJMFxuCvYpSXalwSBII49T8gOACBM1Tj4rZUS1jQWLOBiI2qdSnIxcpDy3h3lilyHXR6Qg731kSoHgT7A3EcZFhhIG1JgHcA9yuOqZ8Gvz\\\\\\\/+Lc0u2pYZd7wuUkr49eWW+ftil4UIM2IjEOGogiuAyVXJwAh9daMnc415ioPM2VbKHIkl+H6nmgGAdhUSNKrq5LEt+4cVQ6Ajg1FrjQsE4pSQyJfum\\\\\\\/XyVuJkVtH0NOisvsbMyYd63e9YEq6MHtJzH5E9ud\\\\\\\/UREFmlLOlF4fmV53wYxzFOGHKzheZbXG80mW27pBvwv+AEYrlMyWuZSvbNmBunZwyGv2hDf6IkvXt6ug4fZu7xvreiJU\\\\\\\/+MIN1iXUAa3dRzzFb7I7xWXdvFB1M+Jo4jJP1o0NFzZj+QJOuiH6uWciRUSNn+wZg65lEPkn4tDtVNhgFq5HAXop2FOB\\\\\\\/EEaHx2H3vxo\\\\\\\/wqFRJm9\\\\\\\/x6shdpV1+X9J9sKWfwtHFY6J5XlURCuVxwMnLcBToXJ2dF4oCi+tNVXgdqvu2gc1STFjSm3t1cAISLbdWlV48d3hqy777VutI2RFPWMw9sHImpqhziJYeFIrKQnQBaYCxQjEjlZXTSoRd5xSCIE9YiOPDNyu7nEQB44GbOfdF1dtaSM8OJPh0pq+QBVDLuBd7FqyRgftV2+ZO3H7zZ45GsKv6MHJ4JQVlAYKSrZVT27uGylm58v\\\\\\\/FT4JNqgj9fcN1S1wI5eILgQQtLKsZZy7GKCxDMPTm+eFMq\\\\\\\/X8u7+fKCtiV6irIgQg+hbhKC61oe9PHcKV3Dc6MRko2B9GSZjPmekTbQPZDq\\\\\\\/O0W8Bm0hxNyzMqClDbrvHuXEyGH2ttDnv7NwUV8XVLcKttX8psHgOg\\\\\\\/Eq1H56FplxLY78XC+4HgS3x1zY9EZyMIf9a0BOpTk21qJX2OodjEcdBwiLjIzzuihn01PE4OSliQkI\\\\\\\/5NGrYkA7ycLZ6BGg9izc\\\\\\\/XM7ZNQjiJM2wCdMfuzf9zYF6YijEdSGAyEnTFf69090Gt8vyF6IdWD4L45zLHtH7rsxvnqPMkiTn+YufG4axB6wLPS6l75H5ywqeo7bUpjAsbIac57HqH3IF+SQe2grOt\\\\\\\/NzgUMdk2l2vkmpebjbFcjNsr0VuXHmOBWZ5T6c9m0gWL3UGivc+f6JBsg4VwhgK2pijbE08hFisRIU59Uv4BDsxaQhYqtnhG27K7unaafB4WL4tes3GafBYc2iw3kTdbRgzOtCeEnCOuuif1g1p1Nlr4qnUR8AuLvZcJWWP3yn23T9pXHzgIoA0hxN+D59LjKd8d0OiIWIuU4nj\\\\\\\/nXiPssofK9ngrjmZ4SSk+p0apKP7M15jSnwUSohMGo+JJRranbSUhw\\\\\\\/ZN3Y511tG6Q9bXxqMx1lnNuEZbN+B1jRpWLNlXLeUpD4rebw62MMQGF7XZ2xz6\\\\\\\/JME8BHD48pVpzjR7u0wz66qjblp0Y1xBzjfD\\\\\\\/AqcF76yMvg72KdTdANIs35SPMy\\\\\\\/lYbQw7PyHQ22\\\\\\\/Gd\\\\\\\/hdcYz1NjSv\\\\\\\/RT1SR0WBEDb4tFQJJX1ThPgrG0p7JBIhxup++X3GKsj151RtPGf3pGb7DBjxgYrXbvYyv3xW3hO91QW9Fo6xgX5ZcEhyETndFNkqKr506AZiJLn1+bdWaPTNBh7YO+4OYU48caf6cLYCXsa3ZeL5LHVHl5cqgPqhpp2Zi3LTblErkK+y9KFsHpuxTICEbn0rp2aA6ah49TM+1eZ2hG5fJxxNVr2U71xYGv70jMkO1U8s1GcgIE5+gwpBm4y8x1JGXt2kIqbdEy\\\\\\\/huWp048srSrCsrna19UpqXuXtrNOJyh1uxAFyrgT+yIiYBKbRmIjw0VejjPU1hLdF31UmZ7H8nbLeSanQOHka3DM5ATipwgghfr2xdEsu3AcGgkZ6U96Rv2P88bkdctmjKTfURWK26kP4+qZzcatZBeYJclTXOxnq74ALQHLiAQ39lsWX9bBuXuhqDWSRg0e3tA\\\\\\\/SvaTUFI0znu3I4h4kdUe6AQ9KEYChpEOyw4z6tRIYMa5DWHlakWFBAevbY5U0oDG5jpCNHHI3FiKL0j5sljDeM\\\\\\\/DcfGlwZQSgL8GX3eN\\\\\\\/3LuChUCbfRZnX45Jw3L\\\\\\\/BNdlK7QzzOMYvNYsAKRtMNnoZQgNM08PiNJ+U65oBxmFfYmTOZAD0qWyVDOqb12cAU8GkVg8fKNXaOtspG+jJAgrOfL3\\\\\\\/\\\\\\\/clYqzAU2b083ljCl56GchknI7m+nzt6wtAUBAH54Cy\\\\\\\/S0X3YE8+4vIaCZypf85QSxnVjhCD4aZEuwtmaBlcR2nsN++YBH0OtYakfW6hVzIOirRK8+2IciCIYKQLwUgg\\\\\\\/Eipgq+VhtjQyYi\\\\\\\/OswISMCaXnFL5gp0GaL9xQjtjXZH5t2Kqs2SlteWMFfqkx3eapKDzWTGRBB\\\\\\\/4tctjjmPCBOSe+KXpSz7QzrC+bIXGnbKW5SKSC8vQwwoE+tGcX\\\\\\\/7mus94ykIIv5rZm4B4kWrY0OaEruP\\\\\\\/OF\\\\\\\/0EV7U+xuzfc1tYwOK61XS+rH1mdlGKXDbdC0r8w5t\\\\\\\/xnUJsp9yHgWbbp07GOWKMUhOqWru0a5lnO5fmbYvRSwRHUkI\\\\\\\/rIWMCwZbuAy3JZQhh+NJw1balQLXd3v7vEwa0JL3k4AnoB1mqI\\\\\\\/++NZXy0wtx1LTUpEZXhu97KMlMTOFEUit8hPcUCsQkUotU1hlD8AgZJ5h0lDY6BilfxvV+fRZ08ctAODXExfRt98mFMjr9+ZTl\\\\\\\/SfCPCpKNaYZ1cXuY1PXAMczJtCTRLYTAp+WpfmJGS6ZeOvfBVgrETpABAx56x5AiqghVcHWwjjS4QagvDQ9IA69wgWsx1SUD8pKeBkUDICdCGvdJCKvEU2ahgAjVIvVl39Hpx3T6VAdeB2X7QCOswXegHR7pJ0eA2m+HdR\\\\\\\/T+lC\\\\\\\/BeierfrEMxkCgAKDp5cL344e1b4hKoc0Ay2CVAQeoXkdn0pXzc6pCvwIRCE\\\\\\\/HrDi\\\\\\\/\\\\\\\/ThZ7H1uG2MOkc3EEWeNgu97lxe2WmCm8DuHaQVUqe8rAEh+9Yvz7vtGhbbZoN\\\\\\\/phxccOyxxlh9a97rEZ8Bqg600Y04\\\\\\\/G8S+ezwxSuycD4Y\\\\\\\/k5KuGg7Hr+QhTI8biIiYifJi2WtQMkivJoWmwC1M5UwpSKcC+K5EzszLM2O85hi4W1+2g3tZ\\\\\\\/iI10dyk604LEFryP6crGWB+JmrpYU3NUyh\\\\\\\/29dQRJeAdtKWHMVoqGFPt7WWVwhrs7DtQEAdlYcP27QVLafXk39+nYVzmWVU1+YibwwxEj6\\\\\\\/VcpOmZErDaaUTkxTAIPo\\\\\\\/bNhZYqzZw0mZntbT98dvK0zwRIHm144LRvsztuBdAILfQIm8st9yEPIvFXMrDsor17VzG2wnGd7SUIq6+YRIiDh2eGUjaKaZfQuuiF+SIPxlkwED6qaDu1cCfnJfCHfYM6U6qxpsJktUVWKiMaWihf6BgXBGbpvXrIHL0cw0hzM\\\\\\\/36in7mjzKow0glrGtZxyU00gWhWfn26jU6WSb91NYDym\\\\\\\/dDaukeU+i5gIPgX+KMKeoPGTIMmApC86Ir6iQ3pjfJ2W1HmMXANgOVpKiFGOwF15vyTbKuT0BKPWh8Qi09QgXVpKbT9JC2Zida81CgFa2M\\\\\\\/eUkuZUMQ3LRZkS565A03RNwOAx8gdSXsuUyywXyrnvRuhz8OfsFmYkE4QPqqHBx368Iwi4ziX+ennO4FCXMURpG8JRxFrfgU3+HmV5yhA7SEz84ntzFWPpXKIx+85h3\\\\\\\/cmi08g7bhDGSdd62BfIU8U5wSf7WH0RazJ0XrHteOC+oBRdAdvgkslC6TrJRhPMRzl8RZMhuupHT8X3G5nG8NeUSAsDv5AL4tTN68dpmKP8IayzOdCpEH4kjfSpS37\\\\\\\/Lpk9fwr+QMj5v3N8NhD7GT5A5brvau\\\\\\\/BVRDdu8qD5CL62RR3Xtw22mcFtwO0kj3\\\\\\\/zdhcO3RbP3xKwfaCkdjqRHdz+LAFTtN1OEm5n622L6ef+HDI3T2RxW0UDT8oZc3AD8A37xaCFMt6jjqBTEPxwfLKvD4\\\\\\\/DVzJfzQm9eGBZpgZsN9K5+am3FO7+E9I6IFpsdHeTCsN8L7KV8QfwceroeTVzlL0UDDVr30IvTPvGG4Ct\\\\\\\/GOZKMOjqG2V8yAdA0hxd\\\\\\\/hw3Cyeco\\\\\\\/Myxxm\\\\\\\/z8qgyHiiTmTwgStiH+2tUVf90pRlOKem6TFLYCHgzzUe38De3E\\\\\\\/rRjxPUUqbXCUNKn8lDISuSTbATCepyrbSpyCFnO5gWDqmxlZBYf8fFw7CD2BG0x8iz8jx9iXNDP8chlyp0fKJXXtebW2ouqMZTUWCW+pDPVYbTVrCKYclMdhAznrkYZplTsGofG35RQvgzNoFl6JcOftQ4YMoDrLZz7iRCRYPcFlWqqV\\\\\\\/EXl\\\\\\\/bNm\\\\\\\/4PnYn1n0iEjhAqu+yaWl9\\\\\\\/ocrgt1tgqPr940AJ4xqY3KF7bWi2fhYKkMQcDsYPnAHdai5echn6biakccN6zX6E9QlEHQFUGhmRkMxxJjwCie0b1D9WMYid2\\\\\\\/vp4zK4XYY10dLZeIa6QbhT2Q+g76izC94ZrvGdzngAQDRXPDPyRSnbpN6QGREqL1XFpmYF2lDBakwHGbNmHYiSFxPEgzSCt6pwBk8upSGzSELOJ9em6aoJ9cvvlwh8YCHgGgeo5RoSD7hKf340XZ3XkOOxvyTPhUaMYz5Ri+OMl2kaRwPUt63fxBgM7vGMns6dO3KW+NFOoff1U4QHjGoEiLOZq2vVKM7oG6rzjwlDiJ7CKI1Wq4EIJhFX2ujSNDsoJV9cmncDrCc4s1CdVyDTQXSfKGJ6svO9c\\\\\\\/3jOVEQd9PQ5MnDmEMldNCx4h+H31dKplv6O5blvIhZJh\\\\\\\/n4sDirT8vr5jDorcIxvEhX58uipY5mko6lmmIg6BMbXYDyXGQqesJn1TqV7onD\\\\\\\/Nx15NYHNjv756H+WiUqYRRCmqduoq7ctjQQrQ\\\\\\\/\\\\\\\/XUJ3YknEkkq9gDvSBC8YwRJAaXxVfi7u\\\\\\\/G+2Mi4eoWrmMDJbvBId9hLlwEdpo00K0FblSJPtLl5HsFcrxJK5PIP81KVD3PdcMaIVZAWK9Zsyljkjev8akzsE7OPt8la9biyg8UaFUB31IzMYXuG3bkvzleunHZGLr9P0+XwCh8TAGOeHEQ6DC3yBMwVD5W12suSl\\\\\\\/KPhl6vujzktap9H47pJq5UCFFNlSee90hUhP2ENW7RqM+GKT7O+UUVA144yuuwVMwlQieBcLUidUf0ZVhPdXFGPdNUtedYb+qwCMobPhkoDpAGCfVEQ1eh8BccVtljuLqTXtJzmDk47gm0U1F7q+\\\\\\\/BXu9is+M66bpqzdLqDHj1FFQ6Vb2Jlk8JLtZYl1AmDSvH7WUSIyVgso\\\\\\\/h8PC\\\\\\\/3J7D6KULdT5+Bw9XMXqKL7tNluUe70hoJpQq5IEevP5MXH4CK2rEd1zorhEsfDDg4Hmlw8EafB2MNRMic01NsL0fXCdu7KQmubE0AUS9qfvgOmCuVwN2WlwHJYpE2uXshZz+Ocnw5CyOLRToCPQBXFkCz9i130PrtaKJCscmD\\\\\\\/\\\\\\\/5WC7Blyat47mqQ13dgYC49FTlQDVYMdXsErS2Mls1MlwwWjtcYVHXpFiWS459DoO9ekHnPaHJKtIDz8fM8z1UY619vZsafWQ6pUF\\\\\\\/l9C7QVwEicl2MWfOJYAa7k\\\\\\\/JyTBIhf84zBxvMVMEsbX50qjBCsfJCJxC6UqvtO0QwBpvU7PJF88yHew5\\\\\\\/A\\\\\\\/KQRXhQH6OG+MM8dW+3tFZYqTiTVuMtcJXLmbxU57eMNyHeHtE2SmdXDvrY5Kfcar\\\\\\\/rmHxoEmzXLqFdNjBMa8hOK64qemT2dp9GNFf+gvKkst8LoolazHEnxL12WgRH\\\\\\\/1AIfk5WFjRgjyNdkkgZsPMyCFREm9MYaqmbQiH+OOvglRzuzIGcRGUBOqd+9IvdT7bX8ixPXFRbUjvI9nHcbxmd\\\\\\\/7jybgX22CLpUnqZZ+TpofCO1q0DrNf4jmJCSnvdhjRozEaqA7qB2sgGVdWmyi1Wi17XcAgcQk9+lERkx2vM8\\\\\\\/McXerY7tp0fQcGUFcvBj0mAwSyiyTOMIFoXyW+7ykTq6cPSI3Fb4xXQe\\\\\\\/csflN1MQddH3oTAIYdx2T5j3fwRhxV6qStqdu7D3GS2KUf9gAk2OANl96jGVpSzZ21J6156PRUrkJJIEkpKBFxqY69dq4qQ9Bn5PeSjWSN+biMy7H7WHAX6Cbi5cXnd5Wrn87xChOr7tihTqPBl1Du7nF0yuBVAyqdpxW99M9zxbaEm5wv7OJ5GyXLakXDHmloAbwbvTEFZo7hhai1CBVxqQEwnZAwfz\\\\\\\/ONk1TH0MUPi9yGt4WaJTIVIhvTDs512ngKHg5V3fizvvIkiknbJWtxu4JLN+CndOiubYxBJWApIfTG4BuS50ULgGKgHu5h9Bz571wK3fUou0cTDDYAHjZpLr1FdgaPpqwYGPZuCDDP+8fFSgfwiuWPExBYQF06INg28Hl\\\\\\\/ULkhCoB4bp0BAEO3NttlOex\\\\\\\/E\\\\\\\/12J7\\\\\\\/qSucPwTrjes3I0VBlkmrjDp9HKbY68m4ECu6ZQ25\\\\\\\/yMQiDWomuGkxXWMeGSot+xmvsZV1RqV5Fe4nGkPYFSs3hbPdEPgIcHYw+QmjYQquSdhAi6w6DSaMipwNrfrd60TyV1RRltaLfssAnyLAEowdAeqWGn9As9JVHnJFzmUCr2cL8rhlqjA7FIygNSpImPvbSsJL6uLH62lwpiL88RHETeRpQsfLTtvykW2gCFHwXkJESb7tVjEg8CZ+cb5hN6gjjRnUlUjif9Hnuf0w38tSxEcxoY\\\\\\\/onwF43PkB0vw8ByINieCaO529SmOp6S4IQoP4evZ3z7r1xJAbevjzD8VdnrzZA79wdVN4bY6oxwM+BKupuSAqQZwQ3SYPXqgcT2DdQdq29DnLdYeL+E\\\\\\\/LbiadG+HUFcGig2EFzVM0YKVa8ksLC2+4BExuta8xK8XzV5TizegBuk5NfyDieA4\\\\\\\/8Qed86li2BN2yZfRW4tF1+IIbejrgYJ0dAcrvVJXtktul95cc99UiqhtUUTi7GfLqnZ33dbYD3Pc0SRfiQXe1hA13VSpIVEwlMZordEJnncaT\\\\\\\/GdEfB67Ng8s3mdSvOkS7HFqAyQcpnkOGB3\\\\\\\/Ev5i30oDA1bC1y+AbCXa1bF30yPYa+EvAGUZ8JvgL0+4EHK0csJHGaQaofNYOgz4f7KKx4M7JTaGCe8wSJIn7FFslopYYzFH9NBooDDJ5Bvg\\\\\\\/GcnFaqB0VnQMtsmUHoePaAYsFPFAdQFAij1dKP0zwXcZqplvS8S1sXwQMZ7Q6w6gmhyKsOIKkOkOHPLfj1Et6jzO4zyvqtxMJc1gVe762S3O9Vs6lFmnloO7LVfyEHc31GRWk50fqh3kKY8OTtIqWzg634SlMuyFNmddyPKb65b7ldAK1VDeQLf2N2bvuIThK1\\\\\\\/LyaRFpe\\\\\\\/SXJ0zazNTGAlDeREVJJ\\\\\\\/MR47\\\\\\\/Rd8DF7rypdtxCzzVEXwpFXdPxGWpOAtL2q0qwbs6OoTVSKVS9uAPS4gNyP4CpOGeGJ0jl4LtJVsMstJkjgnA56XlyPpa5opWUzvK025RQbhzzsSrki3LDweWUS9JTS+RS0cnH7MHIm8jrFjNziFJQIpgua5l964yy\\\\\\\/idc7tfJi0L+n5vRUffXR7VNPuCQgwpI6C2Tuu+c2cYPVcz\\\\\\\/acXvF\\\\\\\/1Oi4XsJrOmLKvr\\\\\\\/OfMHTGjaVWY6u+tcOtWnvqoAKwqeUsJimc+\\\\\\\/NwNfU0bBDxLg3GmohbnCTJoDmklXnaSBi55Vz0vf7fWwRKRo6E7kcIUrzGrELi+m1I20zkB8pP0+AVG7JKAbA\\\\\\\/\\\\\\\/fmvnUmOgpVlgKhuhnj377ZlBkGw8xqek4uAOGwGQXe7qD6oJCxsb4kR37UNuNOC4GCvOUjSTC8js2\\\\\\\/c6OWC+ysIysMNZSOjh6vtiSCR1n\\\\\\\/fzIQfTfNhmvpS\\\\\\\/FZoxlNyY6cZiLIB32jKFmVzWeC+VXkLN3J1WyWrZ+DYwbEIMehBW\\\\\\\/ljhHDP9z7yZyK2kqs0D8WOKth+RPze6+g3sgUPLQHX81bUyALII24bhDgMGDvDN\\\\\\\/4kj0WXy0fptz6IMDO1FOqk8L2KuOW89yoroD3Gq4l4J0VrgUQoESA43TvW\\\\\\\/uGtMlAZ5pCANcb5fvVJ3gcy5DcR3S5LjJX7U57jTBIG\\\\\\\/YK9D\\\\\\\/JUdslMmOoQM9BubFCxHV0NALXFD5lAu6J5uQVrRmOCpd3hmc5QiSeD91A2U+Hwo2ow7uIaPBKJghT7k3jWmxNRrLfFYiQbv0dK890RfCN8kvE9z7Yj4Y\\\\\\\/ltBL0Ap6XgUL4X+7iMvNBTjUs6fcDiPlXVinTL3Us2vF9MD142\\\\\\\/Vy0oj2APVFYh30\\\\\\\/wYftOTCV6elP+wJulaZzv24A4O7G9SKbrJpPjXcRPuWgz4u7XaRcSWv3QzgLme4g5mYsw8mePdXfowhFQiSevfPofbOeInBieW1pCBg4DSTUYcAOI80lGrNeqYR+Mtvp9IOVFfyVnO5hl2JJxe3FlM7r1ZqMmLtG0770KZbxiSwQiBXetvgrv9q8tIgGMeYkYfVYDtCVLBuoFh9Q6tVP+Sc+g9DrOIKJKw\\\\\\\/j+C9fx+cr7aLy1ASE1i2ITzOsGoluAmPDLoF2HOPztvwI3z6bSEQ3Eiyx43G5kT+yu+kxhWYN41Jp2BBUT1qxEyAlkAVglNO+97iEeVRdON\\\\\\\/A7RGIV3jJIzPgCgaS5Ly+hfHXajnotUbt0c4K\\\\\\\/2HJ+5bk1Inhei5VMQtaTA11O0wR5RZ5UC2YtG5xIW7g\\\\\\\/IvLa52lVhucsEgJ8zvEmcgW\\\\\\\/\\\\\\\/l34LH2QA4NRmVCtVAVMgFUz2XBOPxCgFEkxYpoSHgc+cgWlXkbwgOpneGWMNWW+eSh0VeCaXSkg7JMhCXbrrJPuIJxP4Y7GDeyal5EFlLfCChEfAoKq2btb2iSx0bGnlxfrklo\\\\\\\/5TopuvXju5YA\\\\\\\/LEq6fcRoAOneOBMU\\\\\\\/QPsJyfswpVwfecaUoMA3HhOeK0FUvie8sXFx+CTw6\\\\\\\/os0cgrJudZm7rz1K\\\\\\\/L6pWA64T07F+\\\\\\\/ucEC7EAOMCZ7CxsoLFYp2nsI+iD9Mc5grcPUt0sCom6tBXGwq4hhm0fwfT1UtN3o9OohLI32c5n1qUtwGq8DA1mAIveaI9CtnYifvSPyAgz+1I7KNim8wLYzDHgNyd3xOxzeDD8KuBgS5xaRTUdVCxyhGHX\\\\\\\/db2bVw3fp0Ghvfnjzr5n7rKf8jARYFMdL7IAcPwLqWC70PZyjh2Od3DR8ApJ4Xau\\\\\\\/4\\\\\\\/u8JgPMOjRly3PZOW7VpVvxIMtDqKh5JqG2QgfhsfoyUAhZGlSfhArNEWeMj7FN4dyoFlfjmQfIfe3jhK6c+5qnU1UjS2z4VueYM4h2jw\\\\\\\/FaHGX02GfSFD49JLKAbU3\\\\\\\/p02ky5nKDA15v9P1MP0O\\\\\\\/jIUQyCp+xL0yTq5khYn3SAY558cFlg2+FbBogT38rZREgjMkmofyIgKF6hfxmbQ7PFW2f5utPDxJ+N1ce0Pe+Eny9aORmV8n2PUK0kW8Xn25ljk96dCW4ovCMgyqnshNgbv2QXRSSWaJJfKpsyUdcMG2X0vnuNl9gifuX3eSYlXsi8em7J0n0V1E9uhYT1\\\\\\\/FXjKKe6ehY8pyVsiHK\\\\\\\/3IBL9Sss32GRaSjXYqxrhe4o4jQWqQAE3XrFjMNNq6rjzJuPta1U5GTpTTWYRaORWTNlXf9qNUKlOGPuMZI8lSSFMikd8SOnlqTSaFFXgivxChlMYQUzfrfUV6SpQJRhjF5MukEAQXJghh9G4q\\\\\\\/ZuGaHbj4IZ+9NhbNIMJVZpaVExBNF5+GEtRwPzCKIWgX1wreWNVSsDZxcZT7stxXSm1WSK5SIL+FfobADvGiLOJvUsB8WCxCkgob0Cka+NtZim9mcROfZVqVJspg+VcnyouZnNEa\\\\\\\/9cWu9cvkPGEUltGMHvlTOeNnhxUM2FMkKUmcRfevDPfbjm2c6A\\\\\\\/DMPILY+KCIpQ6bz0E3Rzap9vVMWA1XDdZ5nyq01pmab8t1+4YNcL8dxhlDtoTXoK7+tjmU16tfQMaFdpemdwfOWGU3peYL8nMHNtM91hn9w3OdNUpr+BbouIlrAKrLsd5X7geIO+AkrAHc0rdA7g0idg18Gcib8m6\\\\\\\/0nSeOUCl6vw8tdfSAlQtE2Vaf\\\\\\\/Jl0MWgRACHe867GHTqPwq3jyPJsBMBLiUPWD4OM5KDuaJbeaBXf\\\\\\\/rJ3Of1RqZ02ZDzr2Ql3GOPEK28JX2n9Oeo8NIZItEcKUtT8kOS4qqxZ4xWZhIhBjBZu4MMVxUuUBcTU7HNVLyhSTsO6A6cQJKPHPr6439klp3pL2tFyrmlRB+Pi57IwerugEso5Pr6Bfowyk+6CRob9Qn6izTzeGpjZg00WU6wGuUl0ttCavMDkk8tcpb1nqd5oafZ2LFNR04UR5fNXDMTU\\\\\\\/r13NJAFQumljwYG9quU76TT1GJVj\\\\\\\/tbXvAH1cq\\\\\\\/sOxtUfGYpdZ2vDCxDUyrGC4TzPUI0+Fjh+stZ4qe4icYdXFBepZUEcus7oeM0FkavsDkz2ASoZZ71aZXFkDoZ8XaQkx8jLGXBTVFxY48Wr\\\\\\\/rdmeg\\\\\\\/QB\\\\\\\/gKEX3uJg43huarTfgz3s4GqcP3Ew\\\\\\\/2MjhiunO82gGiuNB5GChpsE3C+DMWGBLdHsHr9P65fqRuFvs\\\\\\\/8AWTtBBK0C9wCEVdA7EP2OGYtJBCgqU5zjc2CPfFkBn3KpDVvSDMMEjRu8tmHgQ+4gqP2pZSpZw03vBLL0cLDlNs1O8ot185AGZpcSsd9ATNCByj1nEi7wXRZ1MVwNafYFylKI9gGQ1tSYH23Jx5aA146QR6TvAii7zSYxH\\\\\\\/uqGG9CJWjkla5BLOAxxK0PCDv71jVEvII3ySGX7xfpujoIqQ84y2JGlOwoqvBD8KlhSFapW9seacdDH3Sr5bQdhQCzs\\\\\\\/L5I1oZQ2cdDnOxAu0SYQnBCSMu\\\\\\\/GZoiGwSKTHt90zTheN87OuD5otpb5gxDi6XdmmsC90S2Zt1rThcZwFa7B4wDUQ4DAv6IJt3l+UU610PfqEGcXWrDunK6RJjSpQamSbTyugW0XXXr5uIimFejJ40bVYnNMCsAIbIGjYaqy7867uG9YNzlmT7XEBcVPvSllWODRHTYsAVg6WjenOstTCSOKkl+j7tlfyPX\\\\\\\/gRItftXo7Cv+h+M3j2QljTqRFyoUkVRyMvwuvjoXS+liehW\\\\\\\/rZBpXbk9KahgJKfsmRtylvwKY\\\\\\\/QGrbD4GTDvEdIEoBb9iFoUB6OZqIEUhsDJr9IMx8plgRIYiauckCtl2ZsS3Qv3HfmI7rZhfBeyVXFxijz+hUxk02L1lWSXGbMsjQTZe4aTbuEd4MElNIBs32O8qVSMqtM\\\\\\\/kxRtgxLgAn4JFNdS1guhibHZ9f4tl7db1VNNKUOlzAPR8HQvXThFGAG1buVIfCpl3aYVwzVLbc9LGKvK6Lv0GzlU56jsjQU1eUxKoUMNaR1rJgRDknj3CQPt79\\\\\\\/GSFMXf6YD5iWL\\\\\\\/TUaBWMYajEhcQOxPW+o+eryGx4XIn0BfLJGeTRy2deeq9\\\\\\\/8aul0v0wMVvkC40TsvVK\\\\\\\/VavgOvZWigZcX\\\\\\\/gZ73or7WUcaj4FPWE\\\\\\\/09V107CzpwOXOm76Q8VRCeAk\\\\\\\/uSu19XxwxBAN36ZddCQhcc3PJjiyfb4MCSeQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/18GnQvBqyZAbcst+lEnag==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022585d4b9082f8b70f7078fbbfed3bae11f5efcca22072773f1c548f72ebc6ea76\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224NYsXCSEEbfS2362qoyn7+FA9QeQKNUJNcWsB0JhDtfr1k1rsG05+vizTCjs+wtEs8MXXQehaEV38mmSbbY+t4v4HPgQyJr8Y3ik1+LYjmiwPOg4RAyWSmwEwjQvet7\\\\\\\/6cOotfPKtE5A47HBZRAAigA9poUFygJ4SIpXNLNJAv0UTPGcAzVb6f28DoU8xorB4oHkqsorAesrLtO1XUYMy0WIXxU850Gl6MvCNgJgS2IioM6rP\\\\\\\/K\\\\\\\/faixH5SnbMNWCKtwqzcMqS6pv+dq\\\\\\\/FKOHRaPRxjfPV2LnbVgqjvc1Ckh6LDhGwN74yyGbhwNImHJdgru83uP0iHz2zygkd1eZFtN51FdG5Nd31ACLk3Ic7b4RB2y5CJozzkwcYQ6HQZOtm7tH81OkaL3\\\\\\\/5FtObmTzeNo763i5iFZdUjEz5QxeFqKeBF8hR6wRTQ57SBxe9pfBo3kXrXMXBSsodHL7iAvhu\\\\\\\/NnOYEg1Q7cRmUDsraAJ9iYgRWQXhOZmrJT8Ag9TuT0Rl17H\\\\\\\/VI1JbsiozcSHJzdRmDbclvb677\\\\\\\/qejbpuJiCBx2\\\\\\\/iTXAd0xY5Qy6TlHrLdAiJoOY23pHwFaRtTwfMQ92H2xSYCqkJoNo7ghMqhjnLJ0Wz1RoLgNUBx7luJ+lQmQwaKmFq0TKWW7y1JiXbC7tw4oQADbcPOycs9MZ21a6YwVd0kqcbhCIdXu7pAE\\\\\\\/eE9K8BUex0hhtFPo7Sg5Yil5DSLe3KLZgav7717pothjgiHaQSrtHLVQ2pb6lRIT8GyRupkHoeqr4yRtIGSyyddtYJw9V\\\\\\\/5nPRe9t0MUdZwrvMqUi+49DlAD5oHXuxFQ4Axmq7b7KPkGlH6tWWba9C+BSq4B+CCGStWkY+SiZWaTXpQ9vV5r49ecQ5qTG5w\\\\\\\/YsySCNSeEvlUCcnWaNDhLgr2eaViqaM7fal8fTF7aumK6yUxNhro9MjYKruSOsXSbnZCD4VnibkKt\\\\\\\/l36WYUhV7H2v8ZYRCnKoxKqVkH2zxYC8eBgX0t2R4lXiA8vVm8tNrOh3+\\\\\\\/Rr0Vy5loe9g6nACttvgNscoGiT2HSI3LS5VFTkgChXu3XJuzBCQ+CFja5HPOfCpjJ11SQEplcYiqqgmCromG2uiJ3nNDtWzFWVkfMNZ\\\\\\\/qTziG7pofI43d55DaGSEM+IkeWfWBrNgurZejNS2yuo0Y3dfJkZ9FpQpmTQJWXgTQoS+ocuqP56xaJX7HfQgqNJS\\\\\\\/7BQqxVEmqAir63cXl2Dn1l6cIJD+ksyu+R12TC0sQJaJWxvTO3pETNYfYhkAs\\\\\\\/EG3ste9Qs5p9AQFe+WZZYt62fSacMdQk84IboS2fNPlPm86wLTIMapzXNsIJTHqWwuP+e53qxPi\\\\\\\/5gveTNcEzuoTk7Svugiuf4o0NeAPVPnicBSATQenitQ6+mocu8MhiG6L0mRLQ+RAQMwqBNMNwyKDiCsmLQCvR2Hy4FDOlwWmNw5p83dC+aTofwcXpuQW3BaVl1UJA4Uv9OHP9JjakNp5yeJBti2l06YEVAXcfVfMHoKA47\\\\\\\/m\\\\\\\/EgWVSFzgE+wi+vNr4ga15LFYHY+IjU2HMFXSeh9mv81fsuuAJdy\\\\\\\/dPuEVFHxbN3VMW0Hrmw6nwOzPS\\\\\\\/V9vO9u8cwbeNulj7nDAVhvFSrnm5MKNBwb3LRLCsNSXK4ZW956\\\\\\\/4rHgiZ1XNhb8bt0kv5neGLXXG+XcZmTLg50OXcU\\\\\\\/Cnw50UfoBU2VaOD0vjxH+tHrwpkV7sL8NMu8I3U2\\\\\\\/2ndZ\\\\\\\/paGdHsMSNgk3ULj67UfS+H8r6v9j2sjBHHDb\\\\\\\/ocC0TOOXKOVyx\\\\\\\/UY9KiuhSTbtbamxJStYgo\\\\\\\/wYvRPfGFLD0ln5qv5ATkFtKZQwZhQT2Vqmd8N2MYeMEfJk5VLxChC2UMpppmMFMuZsN5sWICGNkQfGXDHOjUVcmZs2BkNTF4wxIiHax4ZKHj4u6QO0CbCMx6a0m4SXrMgxvLx1p5eUQaS6LN+PQfst9MLUnuHyzo7MUPxl7B37mmMy7K7+3RtsSTNA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022P7kX3l5HXnlEhwSkgi8cXQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223760d19c72068a73a50c32548674dca414461c1166df8a50b4334ae19b33b126\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460437 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2385" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-evTJ3donRn_QZQYsFRyq_O_h4XSrhlGbcV4X-SjpvgM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2385" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zyw7c6HNaEEcI2ADJ8BjCwRaqQKlWbtjNpvtz-hm5NI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2375" title="Septate Junction Proteins Play Essential Roles in Morphogenesis Throughout Embryonic Development in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2397" title="A Genome-Wide Screen for Dendritically Localized RNAs Identifies Genes Required for Dendrite Morphogenesis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460437" id="node460437" data-pisa="ggg;6/8/2385" data-pisa-master="ggg;g3.116.030528" data-apath="/ggg/6/8/2385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Population of Deletion Mutants and an Integrated Mapping and Exome-seq Pipeline for Gene Discovery in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Shangang Jia</span>, <span class="highwire-citation-author" data-delta="1">Aixia Li</span>, <span class="highwire-citation-author" data-delta="2">Kyla Morton</span>, <span class="highwire-citation-author" data-delta="3">Penny Avoles-Kianian</span>, <span class="highwire-citation-author" data-delta="4">Shahryar F. Kianian</span>, <span class="highwire-citation-author" data-delta="5">Chi Zhang</span> and <span class="highwire-citation-author" data-delta="6">David Holding</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2385-2395;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030528 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460437" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shangang Jia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shangang%2BJia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jia%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShangang%2BJia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aixia Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aixia%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAixia%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kyla Morton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyla%2BMorton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morton%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyla%2BMorton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Penny Avoles-Kianian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS Cereal Disease Laboratory, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Penny%2BAvoles-Kianian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Avoles-Kianian%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APenny%2BAvoles-Kianian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Shahryar F. Kianian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS Cereal Disease Laboratory, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shahryar%2BF.%2BKianian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kianian%20SF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShahryar%2BF.%2BKianian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Chi Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Biological Sciences, Center for Plant Science and Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chi%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChi%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">David Holding </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BHolding%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holding%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BHolding%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dholding2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">dholding2@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/8/2385" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460437" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460437/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2385.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460437" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460437/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2385.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460437" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460437/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2385.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460437" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460437/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/8/2385.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460437" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460437/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">To better understand maize endosperm filling and maturation, we used γ-irradiation of the B73 maize reference line to generate mutants with opaque endosperm and reduced kernel fill phenotypes, and created a population of 1788 lines including 39 Mo17 × F2s showing stable, segregating, and viable kernel phenotypes. For molecular characterization of the mutants, we developed a novel functional genomics platform that combined bulked segregant RNA and exome sequencing (BSREx-seq) to map causative mutations and identify candidate genes within mapping intervals. To exemplify the utility of the mutants and provide proof-of-concept for the bioinformatics platform, we present detailed characterization of line 937, an opaque mutant harboring a 6203 bp in-frame deletion covering six exons within the <em>Opaque-1</em> gene. In addition, we describe mutant line 146 which contains a 4.8 kb intragene deletion within the <em>Sugary-1</em> gene and line 916 in which an 8.6 kb deletion knocks out a <em>Cyclin A2</em> gene. The publically available algorithm developed in this work improves the identification of causative deletions and its corresponding gaps within mapping peaks. This study demonstrates the utility of γ-irradiation for forward genetics in large nondense genomes such as maize since deletions often affect single genes. Furthermore, we show how this classical mutagenesis method becomes applicable for functional genomics when combined with state-of-the-art genomics tools.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/maize-functional-genomics" class="hw-term hw-article-keyword hw-article-keyword-maize-functional-genomics" rel="nofollow">maize functional genomics</a></li><li class="kwd"><a href="/keyword/bsr-seq" class="hw-term hw-article-keyword hw-article-keyword-bsr-seq" rel="nofollow">BSR-seq</a></li><li class="kwd"><a href="/keyword/exome-seq" class="hw-term hw-article-keyword hw-article-keyword-exome-seq" rel="nofollow">exome-seq</a></li><li class="kwd"><a href="/keyword/opaque" class="hw-term hw-article-keyword hw-article-keyword-opaque" rel="nofollow">opaque</a></li><li class="kwd"><a href="/keyword/causative-mutation" class="hw-term hw-article-keyword hw-article-keyword-causative-mutation" rel="nofollow">causative mutation</a></li></ul><p id="p-3">There are a variety of existing resources for forward and reverse genetics in maize (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">McCarty and Meeley 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Settles 2009</a>). These are based on EMS (ethyl methane sulphonate), transposons, and radiation-induced physical alterations to DNA. EMS mutagenesis creates mostly point mutations and often results in knock-down alleles that are useful for studying gene function. With the availability of the complete maize genome and the ability to map genes through techniques such as BSR-seq (bulked segregant RNA-seq) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> 2012</a>, 2013), map-based cloning of EMS mutants in maize is feasible. The Targeting Induced Local Lesions in Genomes (TILLING) approach (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">McCallum <em>et al.</em> 2000</a>) has been developed for high-throughput identification of EMS point mutations in a variety of plant species including maize (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Till <em>et al.</em> 2004</a>). Transposons, which create a high frequency of knock-out alleles, have been used in a variety of forward mutagenesis schemes and have led to various reverse genetics resources such as the Mutator-based Uniform Mu project (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">McCarty <em>et al.</em> 2005</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Settles <em>et al.</em> 2007</a>) and an <em>Ac</em>/<em>Ds</em> based system (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Vollbrecht <em>et al.</em> 2010</a>). Proving successful cloning through additional alleles is often a bottleneck in the completion of mutant cloning procedures and so existing and new mutant collections, such as the one described here, increase options for both forward and potentially reverse genetics in maize.</p><p id="p-4">Though transposon insertions are useful for creating nulls, mutations can also be leaky. Likewise, EMS mutations often result in knock-down alleles. Physical deletion of genes, as achieved through irradiation, is one way to remove this ambiguity. Radiation has also been used for inducing physical deletions of genes and this has been shown to be a viable system for PCR-based reverse genetics in <em>Arabidopsi</em>s (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2001</a>). Although deletion mutants were obtained for over 80% of targeted loci from an <em>Arabidopsis</em> population of more than 50,000 lines, such near saturation based on random deletions would be a major challenge in plants with much larger genomes such as maize. However, deletion mutagenesis has potential as a tool for investigating biological processes of interest, such as embryo and endosperm development and other general aspects of vegetative and reproductive plant development. In bringing together genomic sequencing of the maize exome and BSR-seq, a bulked segregant mapping approach based on quantitative typing of SNPs derived from RNA-seq reads (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> 2012</a>, 2013), we are able to appraise the frequency and size of deletions.</p><p id="p-5">The potential of γ-irradiation for reverse genetics has been investigated (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wu <em>et al.</em> 2005</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sato <em>et al.</em> 2006</a>). Although the mutagenesis rate is lower than with EMS mutagenesis, γ-irradiation results in a higher proportion of knock-out mutations (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Sato <em>et al.</em> 2006</a>). The dosage of radiation must be carefully optimized to maximize the mutagenesis rate while allowing an acceptable level of viability of the M1 plants. Low and moderate doses cause a relatively high proportion of useful mutants with normal yield (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dumanovi <em>et al.</em> 1969</a>) while high doses induce a higher proportion of large deletions containing genes essential for viability (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Naito <em>et al.</em> 2005</a>). In an <em>Arabidopsis</em> irradiation experiment, the majority of large deletions were not transmitted from the M1 to the M2 generation and were assumed to result in deletion of genes required for seed germination, gamete development, or other aspects of reproductive development (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Naito <em>et al.</em> 2005</a>). However, in large genomes with lower gene density, the tolerance for large deletion may be higher. The chromosomal alterations in <em>Arabidopsis</em> included point mutations, translocations, insertions, small 1 to 4 bp deletions and large deletions ranging in size from kilobases to several megabases. In a rice mutagenesis experiment, the rate of mutation was ∼1 mutation per 6.2 Mb (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Sato <em>et al.</em> 2006</a>). With current genomics technologies, radiation mutagenesis has greater potential for studying gene and genome function. To this end, a large fast neutron population in soybean was created for functional genomics of agriculturally important traits (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolon <em>et al.</em> 2011</a>).</p><p id="p-6">We have demonstrated the utility of γ-radiation for functional genomics in maize by knocking out the 27 kD γ-zein gene, the major modifier of the <em>opaque-2</em> phenotype in quality protein maize (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Yuan <em>et al.</em> 2014</a>). Here, we describe the use of γ-radiation to create a collection of novel heritable seed mutants in the B73 maize reference genetic background. We prioritized opaque kernel and reduced kernel size mutants that did not substantially affect kernel viability or vegetative development. The other main objectives were to appraise this mutagenesis method for functional genomics by investigating the occurrence of kilobase scale deletions that are most likely to affect single genes as well as to develop a method to rapidly and affordably map and identify causative deletions. To identify causative mutations in the induced mutants, we built a complete pipeline, bulked segregant RNA and exome-capture sequencing (BSREx-seq), by combining BSR-seq (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> 2012</a>) and exome-seq (exome sequencing), plus genomic PCR and RT-PCR verification. We generated a new allele of the <em>o1</em> mutant (line 937) and used it as an example to prove that our pipeline can successfully map the gene and identify the causative deletion within the mapping interval. Significantly, the deletion in mutant 937 is entirely contained within one gene thus supporting this method for maize functional genomics. We describe examples of mapping causative mutations in other mutants some of which correspond to very strong deletion candidates. We also describe a new allele of the <em>sugary1</em> mutation which also harbors an intragenic deletion that we identified with exome-seq alone. We developed an improved BSR-seq analysis pipeline, which was optimized to display causative deletions as their corresponding gaps within the linkage peak. Our analysis highlights the potential for this mutant resource for identifying genes involved in various aspects of seed and whole plant development using the BSREx-seq pipeline.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Radiation dosage testing and γ-radiation mutagenesis</h3><p id="p-7">Freshly harvested B73 maize seeds were equilibrated to 13% moisture as previously described (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hossain <em>et al.</em> 2004</a>) and treated in batches of ∼50 seeds with various dosages of γ-radiation (15 to 35 Gy). After irradiation, the seeds were immediately planted in the greenhouse in potting soil. Plant survival was determined as the proportion of seedlings compared to nonirradiated control seeds 1 month after sowing. Radiation dosage was set at 25 Gy based on a 75% survival rate and was used for full scale mutagenesis of 4000 seeds. These M0 seeds were planted in the field at the University of Nebraska East Campus farm in summer 2012.</p></div><div id="sec-3" class="subsection"><h3>Propagation, screening for kernel phenotypes, and creation of mapping populations</h3><p id="p-8">M1 ears were harvested from M0 plants. Recessive mutant phenotypes are not visible in the M1 ears due to the already physically separate ontogeny of the tassel and ear founder cells in the embryonic maize shoot apical meristem (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Johri and Coe 1983</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Poethig <em>et al.</em> 1986</a>). However, potentially dominant phenotypes were noted. All M1 ears were propagated to the M2 families (10−20 ears per family) in 2013 and 2014 and families were scored for segregating phenotypes including opaque, small kernel, small kernel opaque, narrow, thin, rough surface, partially empty pericarp, shrunken, white, fully empty pericarp (EP), viviparous (VP) (precocious germination), severe defective kernel (DEK). Six seeds from lines carrying a possible mutant phenotype, with the exception of EP, VP, and DEK classes which are invariably lethal, were tested for germination (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Plants which germinated and survived past the seedling stage, thus having reduced likelihood of nonseed specific pleiotropic effects, were propagated to adulthood in the greenhouse and self-pollinated and their pollen used to generate an F1 outcross to Mo17. In some cases M2 ears were not obtained but the more reliably obtained F1 was propagated to the F2 (5−6 plants) in all cases. Where a mutant phenotype was penetrant in the B73/Mo17 hybrid background and not specific to the B73 genetic background, this was displayed as a segregating phenotype in all F2 ears which were large, healthy due to hybrid vigor, and thus easy to reliably screen (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS1.xlsx">Table S1</a>). A small fraction of confirmed mutants (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>) were selected for BSR-seq based on the F2 phenotype and the ability to germinate M2 seeds for the exome-capture experiment.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469067/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469067/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Viability, heritability tests, and production of B73 F2 mapping populations</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Opaque and reduced-fill kernel phenotypes. (A) Opaque phenotypes were shown on light box. Normal phenotype and mutant phenotype are shown for kernels from B73 &#xD7; Mo17 F2 segregating ears. (B) 937/o1 and 146/su1 allelism tests." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-887290274" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Opaque and reduced-fill kernel phenotypes. (A) Opaque phenotypes were shown on light box. Normal phenotype and mutant phenotype are shown for kernels from B73 &#xD7; Mo17 F2 segregating ears. (B) 937/&lt;em&gt;o1&lt;/em&gt; and 146/&lt;em&gt;su1&lt;/em&gt; allelism tests.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2385/F1.medium.gif" width="265" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/8/2385/F1.medium.gif" width="265" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468934" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Opaque and reduced-fill kernel phenotypes. (A) Opaque phenotypes were shown on light box. Normal phenotype and mutant phenotype are shown for kernels from B73 × Mo17 F2 segregating ears. (B) 937/<em>o1</em> and 146/<em>su1</em> allelism tests.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>BSR-seq</h3><p id="p-10">Approximately 30 normal-type (wild-type and hemizygous) seeds and 30 mutant seeds from selected segregating F2 ears were planted in small pots and grown in the greenhouse for 2 wk. Approximately 5 cm<sup>2</sup> of leaf tissue was sampled from each plant and pooled to produce normal-type (N) and mutant-type (M) pools (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The plant number in one BSR-seq pool varied from 21 to 30, and the pools with less than 30 plants resulted from less than 100% germination (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS2.xlsx">Table S2</a>). Total RNA was extracted, DNase1 treated, and repurified using a previously described method (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Holding <em>et al.</em> 2011</a>). Libraries were prepared from RNA-seq by the University of Minnesota Genomics Center using TruSeq RNA v2 kits and samples were bar-coded so that each HiSequation 2500 lane had 10 samples (five mutants each with a normal-type and mutant-type pool). Inserts of about 200 bp were selected by gel size. Sequencing was conducted on a 50 bp paired-end run on the HiSequation 2500 using rapid chemistry.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="BSREx-seq workflow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-887290274" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;BSREx-seq workflow.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2385/F2.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/8/2385/F2.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468928" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-11" class="first-child">BSREx-seq workflow.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Exome sequencing</h3><p id="p-12">Genomic DNA was extracted from leaf tissue from B73 wild-type and M3 single mutant 14-d-old leaf material according to a previous method (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Holding <em>et al.</em> 2008</a>). High-quality DNA was prepared by using Qiagen DNA clean-up kit, after its concentration and integrity were checked using a NanoDrop and agarose gel analysis. Purified DNA was sent out for exon capture and HiSeq sequencing at the University of Minnesota Genomics Center. The Maize_ISU_RNG_Zeaome_EZ_HX1 SeqCap EZ kit (Roche) enriched ∼110 Mb exon regions (<a href="http://xtalks.com/Exome-Sequencing-for-Crops.ashx">http://xtalks.com/Exome-Sequencing-for-Crops.ashx</a>), which covered 39,621 genes and 190,641 exons in B73 annotation (version: AGPv3.25), and was used to purify mutant samples with four bar-coded samples per column. After the exon capture step, the samples were further pooled so that each HiSequation 2500 lane contained the exome fraction from eight mutants. Sequencing was conducted on a 125 bp paired-end run on the HiSequation 2500 using v4 chemistry.</p></div><div id="sec-6" class="subsection"><h3>SNP/indel calling for BSR-seq and exome-seq</h3><p id="p-13">All raw short reads were preprocessed by trimming off low-quality nucleotides on individual reads and/or removing low-quality whole reads with Trimmomatic v0.32 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bolger <em>et al.</em> 2014</a>). The minimum average quality score was set at 28 and the minimum length was 45 bp for BSR-seq and 100 bp for exome-seq data, respectively. The preprocessed high-quality reads (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS2.xlsx">Table S2</a>) were then mapped against the B73 reference genome (AGPv3.25) using Bowtie2 v2.2.4 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Langmead and Salzberg 2012</a>) with default parameters. VarScan v2.3.7 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Koboldt <em>et al.</em> 2012</a>) was employed for SNP/indel calling from the pair-wise pileup files, the aligned base-pair information at one chromosomal position listed in one line, generated by Samtools v0.1.19 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li 2011</a>) after merging, sorting, and indexing the mapped reads. The minimum read depth for a SNP/indel call is five. During SNP/indel calling, a given position was considered as a SNP/indel if there were only two alleles and the frequency of each allele was more than 20% of the depth at this position.</p></div><div id="sec-7" class="subsection"><h3>Expressed gene counts and assembly</h3><p id="p-14">Preprocessed BSR-seq reads were also aligned to the B73 reference with TopHat (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Trapnell <em>et al.</em> 2009</a>), and numbers of reads mapped were counted using HTSeq v0.6.1 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anders <em>et al.</em> 2015</a>) based on the B73 annotation (AGPv3.25). For pair-wise comparisons, differentially expressed genes were identified by using DEseq (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anders and Huber 2010</a>) for <em>P</em>-value calculation. Transcriptome assembly was implemented by Cufflinks (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Trapnell <em>et al.</em> 2012</a>) after mapping reads to the reference genome with TopHat, and assembled transcripts in normal and mutant pools were compared to each other with Cuffcompare.</p></div><div id="sec-8" class="subsection"><h3>Linkage analysis for BSR-seq</h3><p id="p-15">In the BSR-seq analysis, F2 plants of B73 mutant × Mo17 are expected to consist of three genotypes, AA, Aa, and aa, among which AA and Aa plants show the normal phenotypes, but aa plants show the mutant phenotypes. The goal of BSR-seq is to identify the genomic area that is closely linked to aa alleles. Therefore, in this area, two alleles in the mutant pool are homozygous and both are from B73. During BSR-seq analysis, the sequence variants between Mo17 and B73 worked as markers to identify the linkage regions which show the linkage disequilibrium in the whole genome. After steps of BSR-seq read mapping against the B73 reference genome and SNP/indel calling for all reads from both normal and mutant pools, the SNPs/indels with more than five reads in both normal and mutant pools, only two alleles, and either allele having &gt;20% depth were kept. One allele is from B73 and the other from Mo17. For a chromosomal position having this kind of SNP/indel, if its Mo17 allele, <em>i.e.</em>, the one from the Mo17 origin rather than the B73 one has a fraction from 50% (Aa) to 100% (AA) in the normal pool, and it is homozygous in the mutant pool, <em>i.e.</em> a pure B73 allele without any sequence variants, then this position was considered as having a positive SNP/indel. For all positive SNPs/indels (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS2.xlsx">Table S2</a>), a sliding-window of 100 kb with a step of 10 kb was used to calculate the number of positive SNPs/indels per window, where a high number indicates a linkage peak, closely linked to a causative mutation. The numbers of positive SNPs/indels in sliding-windows were displayed with R and Bioconductor package “ggbio” (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Yin <em>et al.</em> 2012</a>) in the whole genome. In the linkage peak region, high numbers of positive SNPs/indels in sliding-windows, the causative deletion would be identified with combining the exome-seq data analysis. At the same time, a large read-free region in either normal or mutant pool, possibly due to the sequencing bias, gene expression regulation, or γ-radiation, was not considered for positive SNPs/indel discovery, but would be considered as a candidate for DNA deletion if its expression was uniquely found in the normal pool underneath the linkage peak.</p></div><div id="sec-9" class="subsection"><h3>Deleted exon identification using exome-seq</h3><p id="p-16">Deleted exons, which are mutations most likely caused by γ-radiation, were identified from exome-seq analysis. Preprocessed exome-seq reads were aligned to the B73 reference genome with BWA mem (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li and Durbin 2010</a>), a mapping tool designed for long reads, and depth for each chromosomal position was achieved by Samtools. The accumulated depth (<em>D</em>) for a given exon was the sum of sequencing depth of all positions in this exon, and the normalized depth of an exon was calculated by the equation: (10<sup>9</sup> × <em>D</em>)/(<em>LD</em><sub><em>a</em></sub>), where <em>D</em><sub><em>a</em></sub> is the total accumulated depth of the whole genome and <em>L</em> is the exon length. An exon was considered as a deleted one if the normalized accumulated depth for this exon is more than 30 in B73 but 0 in the mutants.</p></div><div id="sec-10" class="subsection"><h3>Filtering out the common mutations from background</h3><p id="p-17">Due to the nonidentical nature of different seed stocks of the same inbred in use by maize researchers, the B73 background of all mutants generated in this population has common genetic variation from the B73 reference genome which is shared in all mutants. This variation causes false positives and will be identified by searching common mutations among two or more mutants. Since the chance of two mutants having γ-radiation-caused deletions with identical coordinates is essentially 0, such common deletions among mutants cannot result from the mutagenesis and cannot be causative. Indeed, our BSR-seq analysis showed that no causative candidate mutations were found in the same genome locus in any pair of mutants. To identify these mutations from the background, all discovered mutations, including deleted exons, and SNPs/indels, were scanned for the occurrence in all mutants. The mutations with occurrences of two or more were discarded, and the remaining ones consisted of the mutant-unique variations (likely not causative) and mutations induced by γ-radiation which are potentially causative. The unique deletions were double-checked in Integrative Genomics Viewer (IGV, version 2.3) (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Thorvaldsdottir <em>et al.</em> 2013</a>) to exclude the bias caused by sequencing and exon capture steps.</p><p id="p-18">Searching for the causative deletion in each mutant was conducted using BSR-seq alone, exome-seq alone, and BSR-seq and exome-seq together. The Perl script developed for BSREx-seq data analysis is freely downloadable on our website (<a href="http://sysbio.unl.edu/BSREx/">http://sysbio.unl.edu/BSREx/</a>).</p></div><div id="sec-11" class="subsection"><h3>Zein analysis and western analysis</h3><p id="p-19">Total protein as well as zein and nonzein fractions were extracted from mature and developing endosperm according to the previous method (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wallace <em>et al.</em> 1990</a>). Western analysis of the OPAQUE-1 protein was conducted according to the previous method (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Guo <em>et al.</em> 2012</a>). The O1 antibody was a gift from Rentao Song and was used at 1/500 dilution.</p></div><div id="sec-12" class="subsection"><h3>Genomic PCR and RT-PCR verification</h3><p id="p-20">For causative gene candidates, we wrote a Perl script to extract the sequences from B73 genome reference, and used Primer3 to design batch primers with melting temperatures of around 60°, PCR product size range of 50 bp to ∼500 bp (the optimized 200 bp), and primer size range of around 25 bp. We also called Blastn program to estimate the primers’ specificity across the whole genome, to make sure the primers will produce unique PCR product with leaf genomic DNA or complementary DNA (cDNA) as template. A step-moving strategy with a window size of 500 bp and a step size of 200 bp was used to generate the best primers inside genes or exons. All primers in this study are included in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS3.xlsx">Table S3</a>. The Perl script (GPPD, Genomic PCR Primer Design) is deposited on our website (<a href="http://sysbio.unl.edu/BSREx/">http://sysbio.unl.edu/BSREx/</a>).</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-21">Seeds from all lines are available on request. The BSR-Seq and exome-seq data have been deposited in the Short Reads Archive (SRA) database (<a href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>) of National Center for Biotechnology Information (NCBI) with the accession number of SRP067758. </p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Identification of maize kernel mutants in a B73 deletion population</h3><p id="p-22">Four thousand mutagenized B73 maize kernels (M0) were propagated to the M1 ears. After loss of plants due to failed germination, lethal somatic effects, and male or female sterility, 1788 variably filled M1 ears were recovered. During the summers of 2013 and 2014, one row (10−20 plants) of each family were propagated to the M2 ears which were then screened for segregating seed mutant phenotypes. Although it was anticipated that many putative mutant phenotypes would be environmental and not heritable, 266 families showing potential genetically caused variation were tested (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). All mutant classes, except DEK, EP, and VP, were tested for seed and seedling viability and those that survived to the reproductive stage were self-pollinated to assess heritability and outcrossed to Mo17 to create an F1 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS1.xlsx">Table S1</a>). The F1 was then self-pollinated to make an F2 mapping population. In some cases, the F2 was the first visible confirmation of the heritability of the phenotype where the M2 plant self-pollination did not produce an M3 ear. Five mutants, including lines 1039-<em>o2</em> (opaque-2 control mutant), 937 (full size opaque kernel), 1115 (small opaque kernel), 1554 (small, thin kernel with variable opacity), and 883 (full size opaque kernel), for which both BSR-seq and exome-seq data were generated, were selected for demonstrating the BSREx-seq pipeline (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS2.xlsx">Table S2</a>). Mutant 146, which exhibited a shrunken kernel phenotype, was included in the exome-seq experiment although an F2 mapping population was not yet available for BSR-seq. Mutant 916 (small, thin partial-filled opaque kernel) was only subjected to BSR-seq and not exome-seq. Kernel phenotypes are shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>. In the mutants tested, the F2 ears visibly segregated 1/4 mutant, and chi-square tests showed that the ratios were in accordance with mendelian inheritance (1:3, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS4.xlsx">Table S4</a>). Normal (wild and hemizygous/heterozygous types) and mutant seeds from segregating F2 ears were planted in the greenhouse, and a single young leaf from each of 30 plants was collected and pooled for RNA extraction. Meanwhile, the nonhybrid M2/M3 seeds were planted as well as wild-type B73, for leaf DNA extraction in exome-seq (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p></div><div id="sec-16" class="subsection"><h3>BSREx-seq identified the causative o2 deletion in the control mutant 1039-o2</h3><p id="p-23">To test the limits of the power of exome-seq and BSR-seq to identify a causative deletion, we used a test line 1039-<em>o2</em> created by crossing a random B73 M0 plant (line 1039) to W64A<em>opaque2</em> (<em>o2</em>), self-pollinating the M1/F1 and selecting for homozygous <em>o2</em> seeds in the segregating M2/F2 ears. Since this mutant is a genomic mosaic of both B73 and W64A, the resulting presence/absence variation from pure B73 presents a rigorous test of both BSR-seq and exome-seq. The resulting B73/W64a hybrid had a full size opaque phenotype (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS1.pdf">Figure S1A</a>) caused by the homozygous <em>o2</em> mutation. 1039-<em>o2</em> mutant exhibited typical <em>o2</em> opaque phenotype and zein and nonzein profiles characteristic of <em>o2</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS1.pdf">Figure S1B</a>).</p><p id="p-24">During BSR-seq data analysis for 1039-<em>o2</em>, about 455,000 high-confidence SNPs/indels from all 10 chromosomes were retrieved (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS2.xlsx">Table S2</a>), and their density was shown as orange bars in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS1.pdf">Figure S1C</a>. After using sliding-window analysis, the peaks of positive SNPs/indels, indicating the location of linkage disequilibrium, were plotted against their physical coordinates in the B73 reference genome (red bars in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS1.pdf">Figure S1C</a>). In 1039-<em>o2</em>, positive SNPs/indels intensively appeared on chromosome 7. The highest linkage peak was identified as around 12.5 Mb, indicating linkage disequilibrium with causative mutation in the <em>o2</em> gene (GRMZM2G015534, chr7: 10,798,621–10,800,828).</p><p id="p-25">After exome-seq reads in 1039-<em>o2</em> were mapped to the B73 reference genome, the read abundance in each exon was analyzed, and many read-free exons, mainly due to the variation between B73 and W64A, were detected and named as “absent exons” (blue bars in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS1.pdf">Figure S1C</a>). Exome-seq analysis for 1039-<em>o2</em> showed that the genomic region with the highest linkage peak does not have any absent exons, while they are abundant in the neighboring regions surrounding the peak. The number of absent exons and other genomic variations are much greater than in other pure B73 mutants and these genomic variations evenly distributed across the genome, suggesting the mosaic genome context. This is consistent with the significant differences expected between the W64A genome and the B73 reference genome, and this genomic variation may have been made more pronounced by the fact that the exome-capture kit does not include W64A sequence and the exome-captured reads for the W64A genome could not always be well aligned to the B73 reference genome.Despite the challenge for identifying causative deletions with many absent exons in a non-B73 genome using exome-seq, in principle most of which are false positives, we identified the known 25 bp causative deletion inside the <em>o2</em> gene in mutant 1039-<em>o2</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS1.pdf">Figure S1, C and D</a>). The NCBI BLAST search showed the <em>o2</em> gene sequence in exome-seq is the same as the reference sequence (AJ491298.1) of o2-T allele (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lazzari <em>et al.</em> 2002</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gavazzi <em>et al.</em> 2007</a>) in W64Ao2, and our 1039-<em>o2</em> mutant was homozygous for the 25 bp deletion.</p></div><div id="sec-17" class="subsection"><h3>BSR-seq identified strong candidates for causative deletions in mutants 916 and 1554</h3><p id="p-26">We tested whether BSR-seq alone can identify the causative deletion for mutant 916. The linkage peak is at the genomic position 59.3 Mb of chromosome 5, in which the causative deletion is expected to be located (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS2.pdf">Figure S2A</a>). Since γ-radiation mutagenesis can induce large deletions (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Yuan <em>et al.</em> 2014</a>), which could span longer than one gene, we searched for genes with high read abundances in the normal pool (wild type and hemizygous) and no reads in the mutant pool. Gene GRMZM5G856929 (5:59,832,359–59,833,908) showed a very significant difference in read abundance between normal and mutant 916. Without exome-seq, extending forward and backward across the chromosome 5 to a ∼250 kb region surrounding this gene, five genes were treated as deletion candidates for further genomic PCR verification, among which two end genes on the chromosome 5 were shown not deleted due to being expressed in the mutant and the others were deletion candidates. Finally, genomic PCR showed that two genes (GRMZM5G856929 and GRMZM2G017081) were confirmed deleted, and the other three neighboring genes were not (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3C</a>). The deletion region was estimated to be of ∼8.6 kb from 59,832,359 to 59,840,942 on chromosome 5, within a linkage gap of 0.97 Mb (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). It was noted that GRMZM2G017081 is annotated as <em>Cyclin-A2</em>. Cyclins, which mainly consist of A-, B-, and D-type, are the activating units of CDKs, and formation of cyclin–CDK complexes is essential for kinase activity in cell cycle regulation. It was suggested that the <em>proline responding1</em> gene mutation in an opaque mutant can result in down-regulation of cyclins, which caused cell cycle arrest at the transition from G1 to S phase, suppression of cell proliferation, and finally the seedling lethality phenotype (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2014</a>). A similar phenotype of small and thin partial-filled opaque kernels was seen in mutant 916. <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dante <em>et al.</em> (2014)</a> and <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sabelli <em>et al.</em> (2014)</a> described core cell cycle factors that play important roles in the regulation of the cell division cycle during endosperm development (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Dante <em>et al.</em> 2014</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Sabelli <em>et al.</em> 2014</a>). It is likely that the <em>Cyclin-A2</em> gene is the causative gene mutation in 916, with its embryo specific expression (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS3.pdf">Figure S3</a>), while the other gene GRMZM5G856929 was shown to have leaf-specific expression based on the expression results of MaizeGDB website (<a href="http://www.maizegdb.org/">http://www.maizegdb.org/</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="BSR-seq linkage analysis identified causative gene candidates in mutants 916 and 1554 using expression ratio. Expression difference overlapped with linkage peaks on chromosome 5 in mutant 916 (A) and on chromosome 10 in mutant 1554 (D). Red bars: numbers of positive SNP/indel in sliding-window of 100 kb with a step of 10 kb; purple bars: fold-change of gene expression values between mutant pool and normal pool based on BSR-seq reads; black bar: centromere. Y axis scale for positive SNPs/indels ranges from 0 to 23 and 31 in mutants 916 and 1554, respectively. (B) An enlargement of the highest linkage peak region, 58&#x2013;62 Mb, was shown in mutant 916, in which a gap (&#x223C;0.97 Mb in blue) contains the causative deletion. (C) PCR verification of a causative deletion containing two genes on chromosome 5 in mutant 916. Normalized numbers of read counts of normal type (N) and mutant (M) were shown. Expression fold-changes (FC) defined as N/(M + 0.05) and P-values calculated by R package DESeq, were used to search for deleted exons. The verified deletions are noted with an asterisk. (E) The mapping coverage of the gene GRMZM2G139797 in exome-seq and BSR-seq are visualized." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-887290274" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;BSR-seq linkage analysis identified causative gene candidates in mutants 916 and 1554 using expression ratio. Expression difference overlapped with linkage peaks on chromosome 5 in mutant 916 (A) and on chromosome 10 in mutant 1554 (D). Red bars: numbers of positive SNP/indel in sliding-window of 100 kb with a step of 10 kb; purple bars: fold-change of gene expression values between mutant pool and normal pool based on BSR-seq reads; black bar: centromere. Y axis scale for positive SNPs/indels ranges from 0 to 23 and 31 in mutants 916 and 1554, respectively. (B) An enlargement of the highest linkage peak region, 58&#x2013;62 Mb, was shown in mutant 916, in which a gap (&#x223C;0.97 Mb in blue) contains the causative deletion. (C) PCR verification of a causative deletion containing two genes on chromosome 5 in mutant 916. Normalized numbers of read counts of normal type (N) and mutant (M) were shown. Expression fold-changes (FC) defined as N/(M + 0.05) and &lt;em&gt;P&lt;/em&gt;-values calculated by R package DESeq, were used to search for deleted exons. The verified deletions are noted with an asterisk. (E) The mapping coverage of the gene GRMZM2G139797 in exome-seq and BSR-seq are visualized.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2385/F3.medium.gif" width="224" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/8/2385/F3.medium.gif" width="224" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468936" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">BSR-seq linkage analysis identified causative gene candidates in mutants 916 and 1554 using expression ratio. Expression difference overlapped with linkage peaks on chromosome 5 in mutant 916 (A) and on chromosome 10 in mutant 1554 (D). Red bars: numbers of positive SNP/indel in sliding-window of 100 kb with a step of 10 kb; purple bars: fold-change of gene expression values between mutant pool and normal pool based on BSR-seq reads; black bar: centromere. Y axis scale for positive SNPs/indels ranges from 0 to 23 and 31 in mutants 916 and 1554, respectively. (B) An enlargement of the highest linkage peak region, 58–62 Mb, was shown in mutant 916, in which a gap (∼0.97 Mb in blue) contains the causative deletion. (C) PCR verification of a causative deletion containing two genes on chromosome 5 in mutant 916. Normalized numbers of read counts of normal type (N) and mutant (M) were shown. Expression fold-changes (FC) defined as N/(M + 0.05) and <em>P</em>-values calculated by R package DESeq, were used to search for deleted exons. The verified deletions are noted with an asterisk. (E) The mapping coverage of the gene GRMZM2G139797 in exome-seq and BSR-seq are visualized.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">In mutant 1554, one gene GRMZM2G139797, which had significantly differential read abundance between normal and mutant in the F2 generation, fell within the BSR-seq linkage peak region on chromosome 10 (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS2.pdf">Figure S2B</a>). This suggested that, at a distance of 72 kb from the BSR-seq linkage peak, the gene GRMZM2G139797 with gene name ZmCLO2b (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), which has a large number of reads in the normal pool and no reads in the mutant pool, is a strong candidate for the causative gene. This gene encodes a caleosin family member, and likely plays a role in seed oil body formation (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hu <em>et al.</em> 2014</a>) which would be expected to directly affect embryo development more than endosperm development. Thus, characterizing reduced seed fill mutants also can shed light on processes involved in carbon partitioning and communication between embryo and endosperm during development. It was noted that exome-seq analysis found no gene deleted in the highest linkage peak region, and ZmCLO2b itself is not deleted (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3E</a>). This may suggest a causative mutation in the intergenic regions close to this gene, although three deleted exons and one small deletion were found in other regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS2.xlsx">Table S2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468925/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468925/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Causative deletion candidates identified by BSREx-seq</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Exome-seq alone identified the causative deletion in mutant 146</h3><p id="p-33">Exome-seq was applied to mutants 146, 1039-<em>o2</em>, 1554, 1115, 937, and 883 (not 916). For mutant 146, after exome-capture, the HiSeq sequencing results showed that more than 189,000 out of a total of 190,641 Ensemble maize annotated exons had at least five reads. Since more than 99.2% of exons were captured, it was possible to search for the gene/exon absence. The deleted exons and other mutations in the mutants were identified by comparing mutants and wild-type B73. The detected variation consisted of γ-radiation introduced mutations and individual noninduced variations, and the noninduced variations were identified by searching common ones (occurrence ≥2) among all mutants, including 105 deleted exons, 21,368 small indels, and 225,632 SNPs. The removal of this common genetic variation significantly reduced the identified causative mutation candidates.</p><p id="p-34">Exome-seq analysis of mutant 146 identified one deletion (∼4.8 kb) on chromosome 4 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS4.pdf">Figure S4A</a>), containing 10 exons (exons #2–#11) in GRMZM2G138060 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS4.pdf">Figure S4B</a>), the <em>SUGARY-1</em> gene (<em>SU1</em>), plus 11 small indels. The deletion region (4804 bp) was estimated to extend from 41,397,178 to 41,401,983, where the left break point was located inside the exon #2 body (41,397,089–41,397,274) and the right break point broke the intron between exons #11 and #12. It indicated that nine whole exons (exon #3–#11) were deleted in the <em>SU1</em> gene, and part of exon #2 was lost.</p><p id="p-35">The 79 kD protein encoded by the <em>SU1</em> gene functions as an isoamylase-type starch-debranching enzyme for starch biosynthesis and accumulates in endosperm (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rahman <em>et al.</em> 1998</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dinges <em>et al.</em> 2001</a>). The <em>su1</em> mutant has increased sucrose concentration, decreased concentration of amylopectin, and a shrunken kernel phenotype (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">James <em>et al.</em> 1995</a>). The same kernel phenotype was found in the mutant 146 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). To test whether the identified deletion in <em>SU1</em> gene was causative, we did reciprocal crosses between mutant 146 and a reference <em>su1</em> mutant (407D), and confirmed that their offspring had the shrunken kernel phenotype (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>) and so they do not complement each other and are thus allelic.</p></div><div id="sec-19" class="subsection"><h3>Exome-seq identified a large deletion under the BSR-mapping peak in mutants 1115 and 937</h3><p id="p-36">The combination of BSR-seq and exome-seq were used to identify causative deletions in mutants 1115 and 937 and this illustrates that deletions can be large and include many genes in the case of 1115 or relatively small and affect only a single gene in the case of 937. In mutant 1115, deleted exons identified by exome-seq data scatter on three chromosomes (3, 5, and 7), while the BSR-seq linkage peak resides only on chromosome 3 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS5.pdf">Figure S5A</a>). The linkage peaks on chromosome 3 are distributed widely, and the highest peak is at the genomic site 137,660,000 with a distance of ∼10 Mb to a large deletion, which suggests that this large deletion interrupted the random chromosomal recombination and resulted in a wide linkage peak. Therefore, the combination of exome-seq and BSR-seq analysis results revealed a large deletion (∼1.7 Mb) on chromosome 3 between 125,369,536 and 127,036,462 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS5.pdf">Figure S5, B and C</a>), which covers 86 exons from 26 genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS5.xlsx">Table S5</a>). Although dissecting which gene causes the phenotype and whether or not the phenotype results from the loss of more than one gene is not straightforward, one can infer the causative deletion is on chromosome 3.</p><p id="p-37">Unlike the large deletion in mutant 1115, a short intragene deletion was found in mutant 937. A sharp linkage peak on chromosome 4 in mutant 937 suggested a strong linkage of the opaque phenotype to the causative gene at genomic position around 174.9 Mb (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS2.pdf">Figure S2C</a>), where the highest linkage peak was shown. Exome-seq analysis identified a deletion underneath this linkage peak. The deletion resides within the <em>OPAQUE-1</em> (<em>O1</em>) gene (GRMZM2G449909), and results in the loss of six exons (#13–#18). According to the BSR-seq reads, <em>O1</em> was expressed in leaf of both normal-type and mutant 937 (raw reads count ratio, 407:381), but no reads were mapped to the six deleted exons in mutant 937. The deletion break points on two ends of the deletion were estimated at 176,912,920 inside the intron of exon #12–#13 and 176,919,560 inside the intron of exon #18–#19, resulting in an ∼6.6 kb deletion (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Cufflinks was used to assemble transcripts in wild type and 937 and confirmed that the six exons were deleted in the expressed <em>o1</em> gene (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS6.xlsx">Table S6</a>), and found two wild-type transcripts, GRMZM2G449909_T1 and GRMZM2G449909_T2. Furthermore, TopHat alignment results showed that both transcript variants were found in normal F2 plants (wild type and hemizygous), while only the mutant transcript without the six exons was present in the mutant (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of causative deletion candidate in mutant 937 by combining BSR-seq and exome-seq. (A) The current BSR-seq analysis and two other BSR-seq methods, Bayesian (Liu et al. 2012) and BFR (Ramirez-Gonzalez et al. 2015), all mapped the causative gene to chromosome 4 in mutant 937. Enlargement of a sharp linkage peak (red), 172&#x2013;178 Mb, and a gap (&#x223C;1.98 Mb) covering the 6203 bp deletion (blue) spanning six exons of O1 gene inside the linkage peak were shown. Black bar: centromere. Y axis scale for positive SNPs/indels ranges from 0 to 20. (B) The alignments of exome-seq reads (gray) around the causative deletion, including six exons (#13&#x2013;#18, in black), are shown in the coverage and read windows. The alignment of BSR-seq reads showed two transcript variants, one for normal type (black), and the other (red) for mutant 937 without six exons of o1 gene. (C) Western blot shows two protein variants with different lengths: wild-type and mutant truncated proteins. Hem, hemizygous-type kernel with both wild-type and mutant truncated proteins; Hom, homozygous wild-type kernel with only wild-type proteins; Mut, mutant-type kernel with only mutant truncated proteins." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-887290274" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identification of causative deletion candidate in mutant 937 by combining BSR-seq and exome-seq. (A) The current BSR-seq analysis and two other BSR-seq methods, Bayesian (Liu &lt;em&gt;et al.&lt;/em&gt; 2012) and BFR (Ramirez-Gonzalez &lt;em&gt;et al.&lt;/em&gt; 2015), all mapped the causative gene to chromosome 4 in mutant 937. Enlargement of a sharp linkage peak (red), 172&#x2013;178 Mb, and a gap (&#x223C;1.98 Mb) covering the 6203 bp deletion (blue) spanning six exons of &lt;em&gt;O1&lt;/em&gt; gene inside the linkage peak were shown. Black bar: centromere. Y axis scale for positive SNPs/indels ranges from 0 to 20. (B) The alignments of exome-seq reads (gray) around the causative deletion, including six exons (#13&#x2013;#18, in black), are shown in the coverage and read windows. The alignment of BSR-seq reads showed two transcript variants, one for normal type (black), and the other (red) for mutant 937 without six exons of &lt;em&gt;o1&lt;/em&gt; gene. (C) Western blot shows two protein variants with different lengths: wild-type and mutant truncated proteins. Hem, hemizygous-type kernel with both wild-type and mutant truncated proteins; Hom, homozygous wild-type kernel with only wild-type proteins; Mut, mutant-type kernel with only mutant truncated proteins.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2385/F4.medium.gif" width="440" height="188"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/8/2385/F4.medium.gif" width="440" height="188"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469021" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Identification of causative deletion candidate in mutant 937 by combining BSR-seq and exome-seq. (A) The current BSR-seq analysis and two other BSR-seq methods, Bayesian (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> 2012</a>) and BFR (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Ramirez-Gonzalez <em>et al.</em> 2015</a>), all mapped the causative gene to chromosome 4 in mutant 937. Enlargement of a sharp linkage peak (red), 172–178 Mb, and a gap (∼1.98 Mb) covering the 6203 bp deletion (blue) spanning six exons of <em>O1</em> gene inside the linkage peak were shown. Black bar: centromere. Y axis scale for positive SNPs/indels ranges from 0 to 20. (B) The alignments of exome-seq reads (gray) around the causative deletion, including six exons (#13–#18, in black), are shown in the coverage and read windows. The alignment of BSR-seq reads showed two transcript variants, one for normal type (black), and the other (red) for mutant 937 without six exons of <em>o1</em> gene. (C) Western blot shows two protein variants with different lengths: wild-type and mutant truncated proteins. Hem, hemizygous-type kernel with both wild-type and mutant truncated proteins; Hom, homozygous wild-type kernel with only wild-type proteins; Mut, mutant-type kernel with only mutant truncated proteins.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Comparison of BSR-seq analysis algorithms in mutant 937</h3><p id="p-39">Our BSR-seq analysis pipeline was optimized to produce a sharp mapping linkage peak close to the causative deletion, and had the ability to detect a gap in the BSR-seq linkage peak caused by the deletion. The original BSR-seq analysis algorithm developed in maize by <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> (2012)</a> used the empirical Bayesian approach to calculate the probability of meeting a 1:2 ratio of mutant to wild-type alleles in the nonmutant pool, in a 50-SNPs window (<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> 2012</a>). We optimized the strategy, and directly calculated the positive SNP number in a physical 100 kb window with a moving step of 10 kb. Our BSR-seq analysis algorithm was compared to the previous Bayesian (<a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> 2012</a>) and bulk frequency ratio (BFR) (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Trick <em>et al.</em> 2012</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Ramirez-Gonzalez <em>et al.</em> 2015</a>) methods for mutant 937, to evaluate their performance. All three BSR-seq analysis methods showed linkage peaks for the causative deletion on chromosome 4 (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>), but the BFR method resulted in suboptimal results with increased noise and many false positives (BFR ≥ 1). Using a 50-SNPs window (<a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> 2012</a>), the Bayesian method reduced the noise by smoothing the peak, but the physical coordinates of linkage peaks were lost. In our method, the sliding-window played the same smoothing function and the density of SNPs was kept. Therefore, a significant gap (175–177 Mb), corresponding to the deletion of six exons within the <em>o1</em> gene, inside the BSR-seq linkage peak in mutant 937 was shown as well. This ∼1.8 Mb gap could also result from low SNP density (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS2.pdf">Figure S2C</a>). In addition, as evidence of the relationship between SNP density and linkage peaks, the correlation of high SNP density across chromosome 4 with small linkage peaks suggested their weak linkage to the causative gene region, and this is not evident on other chromosomes. The moving steps and windows of positive SNPs/indels in BSR-seq analysis were evaluated, and for a kilobase-sized deletion, a megabase-sized window would be too large to show the linkage gap by causative deletion (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS6.pdf">Figure S6</a>). The gap covering the causative deletion in mutant 937 was estimated to range from 1.1 Mb to 2,076,580 bp, while the gap was ∼1 Mb in mutant 916. A minimum window size of 100 kb and step size of 10 kb is needed to plot the gaps caused by causative deletion in mutants 937 and 916.</p></div><div id="sec-21" class="subsection"><h3>Precise characterization of o1 deletion in mutant 937</h3><p id="p-40">To verify the deletion in mutant 937, primers were designed inside the exons and introns (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS3.xlsx">Table S3</a>), and the six deleted exons were all validated based on genomic PCR results, with two intron regions deleted as well (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5, A and D</a>). The BSR-seq and exome-seq results allowed broad definition of the deletion, and clone sequencing provided the base pair resolution. Long genomic PCR was used to cover the junction site in mutant 937, and five products were obtained by amplifying across the junction in the mutant that gives no product in wild type (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). Sequencing of these PCR products showed the precise genomic coordinates of two break points at 176,913,222 and 176,919,426, so that the actual deletion size is 6203 bp (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5E</a>). This showed good agreement with the estimation of break points (176,912,920 and 176,919,560) in BSREx-seq analysis. It was interesting that a nucleotide residue of adenine (“A”) was left between two break points, which might be formed by DNA repair.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic PCR and clone verification of deletion in O1 in mutant 937. (A) Exon/Intron column is for the region where primers were designed. Mapped read depth coverage in exome-seq was normalized to compare wild-type B73 and mutant 937 (B73:937). Fold-changes (FC) of read depth coverage between B73 and mutant 937 and P-values were used to search for deleted exons. The verified deletions are noted with an asterisk. (B) Five long PCRs were conducted to cover the junction site, and PCR product sizes are 7584 bp, 8331 bp, 8713 bp, 9046 bp, and 11,327 bp for Region 1&#x2013;5. No bands in B73 result from genomic regions too long for PCR. (C) cDNA was synthesized and RT-PCR was conducted to compare B73 and 937 and showed an &#x223C;600 bp difference. Clone sequencing confirmed the PCR product sizes are 864 bp in B73 and 222 bp in 937. (D) The causative deletion covered six exons inside the total 39 exons of O1 gene. (E) Clone sequencing of genomic PCR products in 937 confirmed the 6203 bp deletion inside O1 gene and the two break points at 176,913,222 and 176,919,426 based on maize genome v3.25. (F) cDNA clone sequencing identified the transcript variant without the six deleted exons, and exons #12 and #19 were linked directly. The O1 gene resides in the negative strand, and was shown in coordinates for the positive strand from 5&#x2032; end to 3&#x2032; end. (G) Protein domains in O1. The deletion from position 497 to 710 was in the head domain. Protein domain positions were from Wang et al. (2012)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-887290274" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic PCR and clone verification of deletion in &lt;em&gt;O1&lt;/em&gt; in mutant 937. (A) Exon/Intron column is for the region where primers were designed. Mapped read depth coverage in exome-seq was normalized to compare wild-type B73 and mutant 937 (B73:937). Fold-changes (FC) of read depth coverage between B73 and mutant 937 and &lt;em&gt;P&lt;/em&gt;-values were used to search for deleted exons. The verified deletions are noted with an asterisk. (B) Five long PCRs were conducted to cover the junction site, and PCR product sizes are 7584 bp, 8331 bp, 8713 bp, 9046 bp, and 11,327 bp for Region 1&#x2013;5. No bands in B73 result from genomic regions too long for PCR. (C) cDNA was synthesized and RT-PCR was conducted to compare B73 and 937 and showed an &#x223C;600 bp difference. Clone sequencing confirmed the PCR product sizes are 864 bp in B73 and 222 bp in 937. (D) The causative deletion covered six exons inside the total 39 exons of &lt;em&gt;O1&lt;/em&gt; gene. (E) Clone sequencing of genomic PCR products in 937 confirmed the 6203 bp deletion inside &lt;em&gt;O1&lt;/em&gt; gene and the two break points at 176,913,222 and 176,919,426 based on maize genome v3.25. (F) cDNA clone sequencing identified the transcript variant without the six deleted exons, and exons #12 and #19 were linked directly. The &lt;em&gt;O1&lt;/em&gt; gene resides in the negative strand, and was shown in coordinates for the positive strand from 5&#x2032; end to 3&#x2032; end. (G) Protein domains in O1. The deletion from position 497 to 710 was in the head domain. Protein domain positions were from Wang &lt;em&gt;et al.&lt;/em&gt; (2012).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2385/F5.medium.gif" width="251" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/8/2385/F5.medium.gif" width="251" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2385/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468996" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-41" class="first-child">Genomic PCR and clone verification of deletion in <em>O1</em> in mutant 937. (A) Exon/Intron column is for the region where primers were designed. Mapped read depth coverage in exome-seq was normalized to compare wild-type B73 and mutant 937 (B73:937). Fold-changes (FC) of read depth coverage between B73 and mutant 937 and <em>P</em>-values were used to search for deleted exons. The verified deletions are noted with an asterisk. (B) Five long PCRs were conducted to cover the junction site, and PCR product sizes are 7584 bp, 8331 bp, 8713 bp, 9046 bp, and 11,327 bp for Region 1–5. No bands in B73 result from genomic regions too long for PCR. (C) cDNA was synthesized and RT-PCR was conducted to compare B73 and 937 and showed an ∼600 bp difference. Clone sequencing confirmed the PCR product sizes are 864 bp in B73 and 222 bp in 937. (D) The causative deletion covered six exons inside the total 39 exons of <em>O1</em> gene. (E) Clone sequencing of genomic PCR products in 937 confirmed the 6203 bp deletion inside <em>O1</em> gene and the two break points at 176,913,222 and 176,919,426 based on maize genome v3.25. (F) cDNA clone sequencing identified the transcript variant without the six deleted exons, and exons #12 and #19 were linked directly. The <em>O1</em> gene resides in the negative strand, and was shown in coordinates for the positive strand from 5′ end to 3′ end. (G) Protein domains in O1. The deletion from position 497 to 710 was in the head domain. Protein domain positions were from <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> (2012</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">cDNA sequencing was used to examine <em>O1</em> gene transcript variants in mutant 937. RT-PCR identified a ∼600 bp difference between the products of wild-type B73 and mutant 937 (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>), confirming the deletion and two transcript variants. Clone sequencing showed the six deleted exons were not present in the 937 mutant transcript, and the boundaries of exons 12 and 19 (176,920,129 and 176,909,023) were linked directly (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5F</a>). The cDNA difference between wild-type and mutant 937 was the concatenated lengths of six exons (642 bp). cDNA clone sequencing showed the deletion did not change the open reading frame, and predicts a truncated O1 protein with 214 amino acids deleted out of the total length of 1521 amino acids (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5G</a>). The western blot confirmed the existence of two protein variants in normal-type and mutant 937 (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a>), which confirmed the two transcript variants in BSR-seq analysis (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a>). Since the deletion did not change the open reading frame, two thirds of normal kernels are expected to show the presence of both the full-length 173.4 kD protein and a detectable level of the truncated 148.9 kD protein indicating their hemizygous nature. Homozygous wild-type kernels only contain the 173.4 kD protein while homozygous mutant kernels only contain the 148.9 kD protein. The 214 deleted amino acids occur in the myosin head domain, which enables myosin to bind the filamentous actin to move inside the cell using ATP hydrolysis (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2012</a>). Thus, the deletion likely impairs O1 function as a myosin XI motor protein (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2012</a>) although it is an intriguing possibility that residual function remains, especially since partial complementation may be evident from the allelism test to the <em>o1</em>-reference allele (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>). The progeny kernels, while opaque, show a thin layer of vitreous endosperm.</p></div><div id="sec-22" class="subsection"><h3>Limitation of BSREx-seq</h3><p id="p-43">Although the ability to identify functional mutations by BSREx-seq was demonstrated, mutations in extragenic regulatory regions are undetectable for the exome-seq technology. For example, mutant 883 showed very good BSR-seq linkage results on chromosomes 8, but no deleted exons or differentially expressed genes between normal and mutant types were found under the linkage peak (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS2.pdf">Figure S2D</a>). Since γ-radiation induces random mutagenesis, and the gene-free regions are much longer than coding regions, it is expected that many deletions will be entirely within gene-free regions and that some of these will be causative. In such cases, alternative strategies must be employed to identify functional variations under the linkage peak.</p></div></div><div class="section discussion" id="sec-23"><h2 class="">Discussion</h2><div id="sec-24" class="subsection"><h3>A novel mutational resource for functional genomics of maize seed development</h3><p id="p-44">Both endospermic and nonendospermic seeds develop in three overlapping phases (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Egli 2006</a>). These are a phase of cell division and minimal weight gain after fertilization, a seed filling stage when cells expand, endoreduplicate, and assimilate storage compounds and a final stage of maturation when desiccation and dormancy occur. Mutations that interrupt early stages may be more likely to cause lethality during embryogenesis or during the earliest stages of seedling growth as a result of failed or insufficient differentiation or maturation of cotyledon or endosperm tissue. Indeed, more than half of small kernel and opaque mutants identified in this population display seed or seedling lethality and as such, it is impossible to assign seed-specific functions to their causative genes. During the late seed development phase, mutants that fail to desiccate or become dormant are also often lethal and we have also identified many mutants which appear to be in this category. However, a substantial proportion of maize seed mutants in this population appear to be affected in the midstages of kernel filling, nutrient assimilation, and endosperm maturation, and often have little or no adverse effect on embryo viability or adult vegetative development. Such mutants are useful for identifying true seed-specific control factors. γ-radiation was used to create this population since it is known to create whole gene deletions, but since relatively little is known about the range of deletion sizes, its value as a functional genomics tool was not known. We not only created a unique biological resource for studying seed and whole plant development, but also showed that, although some mutations are too large to be realistically useful for functional genomics, some mutations affect single or small numbers of genes.</p><p id="p-45">In an <em>Arabidopsis thaliana</em> experiment involving high-dosage γ-radiation, most of the identified deletions were large (up to 6 Mb), and the majority of these were not transmitted to progeny due to the loss of genes essential for viability (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Naito <em>et al.</em> 2005</a>). Both nucleotide substitutions and small deletions induced by γ-radiation were found in different plant species, such as <em>Arabidopsis</em> (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Yoshihara <em>et al.</em> 2010</a>), tomato (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Shirasawa <em>et al.</em> 2015</a>), and rice (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Sato <em>et al.</em> 2006</a>). In our maize mutagenesis population, only large and medium-sized deletions were confirmed or implicated as causative candidates, and no small indels and SNPs were thought to be causative. For example, three small indels and 17 SNPs with high confidence in mutant 937 were unique compared with B73 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/TableS2.xlsx">Table S2</a>), but none of these small variations corresponded to the BSR-seq linkage peak at 174.9 Mb of chromosome 4.</p><p id="p-46">Despite the apparent extensive generation of small mutations, our confirmed or strong candidates for causative mutations were large- or medium-scale deletions, such as the 1.7 Mb deletion in mutant 1115. More importantly, a single gene deletion was confirmed as the causative mutation in mutants 937 and 146 (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>). This indicated that our optimized γ-radiation dosage makes a good balance between causing deletions and maintaining high-level viability. Although the deletions in 937 and 146 are still of several kilobases, the extensive number and size of introns resulted in these deletions being entirely within a single gene in both cases. In some cases where we have a precise mapping interval of linkage peak but no apparent exonic deletions in that interval such as in mutants 883 and 1554, we have to consider induced mutations within introns or in regulatory sequences.</p></div><div id="sec-25" class="subsection"><h3>Improved BSR-seq not only maps the causative mutation but can highlight the deletion within the mapping peak</h3><p id="p-47">BSR-seq combines the convenience of bulked segregant analysis with RNA-seq using RNA extracted from pooled mutant and nonmutant plants. Furthermore, using quickly and easily obtained seedling leaf tissue RNA, causative genes can be mapped even if they are not expressed in leaf tissue, using SNPs mined from surrounding genes that are expressed (<a id="xref-ref-25-9" class="xref-bibr" href="#ref-25">Liu <em>et al.</em> 2012</a>, 2013). This is advantageous because seed mutant phenotypes of interest in the current collection may arise from loss of genes with seed-specific expression and the need to isolate RNA pools from developing kernels would add considerable time and expense since it requires large-scale field propagation, ear sampling, and storage. In fact, all the causative gene candidates in this study were shown (using publicly available expression data) to have high-level expression in embryo, endosperm, and whole seed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1/FigureS3.pdf">Figure S3</a>).</p></div><div id="sec-26" class="subsection"><h3>BSREx-seq for forward genetics</h3><p id="p-48">Whole genome scanning by exome-seq has the potential to detect the presence/absence of genes, exons, and subexon regions. However, depending on the dosage, γ-radiation mutagenesis might induce too many mutations, which could reduce its applicability for forward genetics. Using BSR-seq alone also has some limitations, and using expression difference between wild type and mutant is unlikely to unequivocally identify a causative gene. Kernel-specific, nonpleiotropic mutant phenotypes may arise from mutations in genes not expressed in leaf tissue. Furthermore, small deletions inside causative genes do not necessarily alter their expression level measured by read counts. Both of these factors make it difficult to infer the causative gene by using BSR-seq alone. Therefore, in this study, we developed BSREx-seq by combining BSR-seq and exome-seq. By scanning the mapping interval for deletions, in the small number of mutants so far studied, we have been able to confirm or strongly suggest candidates in three.</p><p id="p-49">Mapping-by-sequencing is a relatively new method in forward genetic screens for identifying causative mutations behind phenotypes of interest (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Schneeberger 2014</a>). In addition, direct comparison of resequenced whole genomes can identify candidates for causative mutations (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Nordström <em>et al.</em> 2013</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schierenbeck <em>et al.</em> 2015</a>). For example, exome sequencing of phenotypic bulks identified a candidate gene in barley (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mascher <em>et al.</em> 2014</a>), and though expensive and more labor intensive, whole genome sequencing (WGS) was able to map the causative mutation through a pooled mapping population (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Blumenstiel <em>et al.</em> 2009</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Schneeberger <em>et al.</em> 2009</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Zuryn <em>et al.</em> 2010</a>). BSR-seq was shown to have similar mapping intervals to WGS-based bulk segregant analysis (BSA) (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Miller <em>et al.</em> 2013</a>). With cost-effective sequencing of exon-enriched genomic DNA compared to WGS-based BSA, BSREx-seq has the ability to pool and multiplex increasingly large numbers of different mutants, and give flexibility to fit the various situations in different plant species. Combining BSR-seq mapping using the transcriptome with exome DNA sequencing represents an efficient and cost-effective approach to identify causative mutants in complex genomes.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">The authors thank Patrick Schnable and his laboratory for designing the Maize_ISU_RNG_Zeaome_EZ_HX1 SeqCap EZ array and allowing its use prior to publication. The authors thank Rentao Song for the gift of the O1 antibody. The Hiseq sequencing and exome-capture were performed by the University of Minnesota Genomics Center. The authors thank Lingling Yuan for early assistance with field work and Ying Ren for useful discussions. The computing work in this project was supported by the Holland Computing Center of the University of Nebraska. This work was supported by the UNL Center for Plant Science Innovation Program of Excellence and the UNL Department of Agronomy and Horticulture and by the Agriculture and Food Research Initiative competitive grant no. 2013-02278 of the United States Department of Agriculture National Institute of Food and Agriculture.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-51">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030528/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-52"><em>Communicating editor: J. D. Faris</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-04-22"><span class="received-label">Received </span>April 22, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-05-25"><span class="accepted-label">Accepted </span>May 25, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Jia <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.1" data-doi="10.1186/gb-2010-11-10-r106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Differential expression analysis for sequence count data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAnders%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253DR106%26rft.epage%253DR106%26rft.atitle%253DDifferential%2Bexpression%2Banalysis%2Bfor%2Bsequence%2Bcount%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-10-r106%26rft_id%253Dinfo%253Apmid%252F20979621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-10-r106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20979621&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.2" data-doi="10.1093/bioinformatics/btu638"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyl</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">HTSeq – a Python framework to work with high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu638%26rft_id%253Dinfo%253Apmid%252F25260700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIzMS8yLzE2NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.3" data-doi="10.1534/genetics.109.101998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blumenstiel</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noll</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perera</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walton</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of EMS-induced mutations in <em>Drosophila melanogaster</em> by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101998%26rft_id%253Dinfo%253Apmid%252F19307605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTgyLzEvMjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.4" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTUvMjExNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.5" data-doi="10.1104/pp.110.170811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolon</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haun</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stacey</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Phenotypic and genomic analyses of a fast neutron mutant population resource in soybean.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">240</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.110.170811%26rft_id%253Dinfo%253Apmid%252F21321255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTYvMS8yNDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dante</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larkins</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabelli</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cell cycle control and seed development.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D5%26rft.spage%253D493%26rft_id%253Dinfo%253Apmid%252F25295050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25295050&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.7" data-doi="10.1104/pp.125.3.1406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dinges</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colleoni</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Molecular structure of three mutations at the maizesugary1 locus and their allele-specific phenotypic effects.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1406</span>–<span class="cit-lpage">1418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DDinges%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D125%26rft.issue%253D3%26rft.spage%253D1406%26rft.epage%253D1418%26rft.atitle%253DMolecular%2BStructure%2Bof%2BThree%2BMutations%2Bat%2Bthe%2BMaize%2Bsugary1%2BLocus%2Band%2BTheir%2BAllele-Specific%2BPhenotypic%2BEffects%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.125.3.1406%26rft_id%253Dinfo%253Apmid%252F11244120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTI1LzMvMTQwNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dumanovi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenber</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergstra</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Radiation-induced heritable variation of quantitative characters in wheat.</span> <abbr class="cit-jnl-abbrev">Hereditas-Genetisk. A</abbr>. <span class="cit-vol">62</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas-Genetisk.%2BA%26rft.volume%253D62%26rft.spage%253D221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.9" data-doi="10.1071/AR06133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Egli</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The role of seed in the determination of yield of grain crops.</span> <abbr class="cit-jnl-abbrev">Aust. J. Agric. Res.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">1237</span>–<span class="cit-lpage">1247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAust.%2BJ.%2BAgric.%2BRes.%26rft.volume%253D57%26rft.spage%253D1237%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR06133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR06133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000242166300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.10" data-doi="10.1104/pp.107.103606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gavazzi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lazzari</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciceri</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianazza</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viotti</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Wild-type opaque2 and defective opaque2 polypeptides form complexes in maize endosperm cells and bind the opaque2-zein target site.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">145</span>: <span class="cit-fpage">933</span>–<span class="cit-lpage">945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.107.103606%26rft_id%253Dinfo%253Apmid%252F17827273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNDUvMy85MzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.11" data-doi="10.1104/pp.111.191163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronhovde</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soundararajan</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Pyrophosphate-dependent fructose-6-phosphate 1-phosphotransferase induction and attenuation of Hsp gene expression during endosperm modification in quality protein maize.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">917</span>–<span class="cit-lpage">929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.191163%26rft_id%253Dinfo%253Apmid%252F22158678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTgvMi85MTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.12" data-doi="10.1007/s00122-008-0762-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holding</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbon</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ford</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic analysis of <em>opaque2</em> modifier loci in quality protein maize.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHolding%26rft.auinit1%253DD.%2BR.%26rft.volume%253D117%26rft.issue%253D2%26rft.spage%253D157%26rft.epage%253D170%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bopaque2%2Bmodifier%2Bloci%2Bin%2Bquality%2Bprotein%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0762-y%26rft_id%253Dinfo%253Apmid%252F18427771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0762-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18427771&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257125700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.13" data-doi="10.1007/s00122-010-1486-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holding</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klingler</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Characterization of <em>opaque2</em> modifier QTLs and candidate genes in recombinant inbred lines derived from the K0326Y quality protein maize inbred.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">794</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHolding%26rft.auinit1%253DD.%2BR.%26rft.volume%253D122%26rft.issue%253D4%26rft.spage%253D783%26rft.epage%253D794%26rft.atitle%253DCharacterization%2Bof%2Bopaque2%2Bmodifier%2BQTLs%2Band%2Bcandidate%2Bgenes%2Bin%2Brecombinant%2Binbred%2Blines%2Bderived%2Bfrom%2Bthe%2BK0326Y%2Bquality%2Bprotein%2Bmaize%2Binbred.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1486-3%26rft_id%253Dinfo%253Apmid%252F21076810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1486-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076810&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.14" data-doi="10.1534/genetics.103.022590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hossain</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riera-Lizarazu</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalavacharla</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vales</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maan</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Radiation hybrid mapping of the species cytoplasm-specific (<em>scs<sup>ae</sup></em>) gene in wheat.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">415</span>–<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHossain%26rft.auinit1%253DK.%2BG.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D415%26rft.epage%253D423%26rft.atitle%253DRadiation%2BHybrid%2BMapping%2Bof%2Bthe%2BSpecies%2BCytoplasm-Specific%2B%2528scsae%2529%2BGene%2Bin%2BWheat%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.103.022590%26rft_id%253Dinfo%253Apmid%252F15454553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY4LzEvNDE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Molecular characterization and evolutionary analysis of the putative caleosin gene family in maize (<em>Zea mays</em>).</span> <abbr class="cit-jnl-abbrev">Int. J. Agric. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BAgric.%2BBiol.%26rft.volume%253D16%26rft.spage%253D261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.16" data-doi="10.1105/tpc.7.4.417"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Characterization of the maize gene sugary1, a determinant of starch composition in kernels.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DJames%26rft.auinit1%253DM.%2BG.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D417%26rft.epage%253D429%26rft.atitle%253DCharacterization%2Bof%2Bthe%2BMaize%2BGene%2Bsugary1%252C%2Ba%2BDeterminant%2Bof%2BStarch%2BComposition%2Bin%2BKernels%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.7.4.417%26rft_id%253Dinfo%253Apmid%252F7773016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjcvNC80MTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.17" data-doi="10.1016/0012-1606(83)90073-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johri</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coe</span>  <span class="cit-name-given-names">E. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Clonal analysis of corn plant development. I. The development of the tassel and the ear shoot.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">154</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DJohri%26rft.auinit1%253DM.%2BM.%26rft.volume%253D97%26rft.issue%253D1%26rft.spage%253D154%26rft.epage%253D172%26rft.atitle%253DClonal%2Banalysis%2Bof%2Bcorn%2Bplant%2Bdevelopment.%2BI.%2BThe%2Bdevelopment%2Bof%2Bthe%2Btassel%2Band%2Bthe%2Bear%2Bshoot.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252883%252990073-8%26rft_id%253Dinfo%253Apmid%252F6840396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(83)90073-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6840396&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983QS13100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.18" data-doi="10.1101/gr.129684.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129684.111%26rft_id%253Dinfo%253Apmid%252F22300766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTY4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.19" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lazzari</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cosentino</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viotti</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Gene products and structure analysis of wild-type and mutant alleles at the <em>Opaque-2</em> locus of <em>Zea mays</em>.</span> <abbr class="cit-jnl-abbrev">Maydica</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMaydica%26rft.volume%253D47%26rft.spage%253D253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000180968700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.21" data-doi="10.1093/bioinformatics/btr509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2987</span>–<span class="cit-lpage">2993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMjEvMjk4NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.22" data-doi="10.1093/bioinformatics/btp698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and accurate long-read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp698%26rft_id%253Dinfo%253Apmid%252F20080505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi81LzU4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2385.23" data-doi="10.1371/journal.pone.0082333"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The maize glossy13 gene, cloned via BSR-Seq and Seq-walking encodes a putative ABC transporter required for the normal accumulation of epicuticular waxes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e82333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De82333%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0082333%26rft_id%253Dinfo%253Apmid%252F24324772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0082333&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24324772&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.24" data-doi="10.1046/j.1365-313x.2001.01084.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Century</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Straight</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronald</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A fast neutron deletion mutagenesis-based reverse genetics system for plants.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DLi%26rft.auinit1%253DX.%26rft.volume%253D27%26rft.issue%253D3%26rft.spage%253D235%26rft.epage%253D242%26rft.atitle%253DA%2Bfast%2Bneutron%2Bdeletion%2Bmutagenesis-based%2Breverse%2Bgenetics%2Bsystem%2Bfor%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313x.2001.01084.x%26rft_id%253Dinfo%253Apmid%252F11532169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313x.2001.01084.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11532169&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170466800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.25" data-doi="10.1371/journal.pone.0036406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nettleton</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene mapping via bulked segregant RNA-Seq (BSR-Seq).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e36406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLiu%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De36406%26rft.epage%253De36406%26rft.atitle%253DGene%2Bmapping%2Bvia%2Bbulked%2Bsegregant%2BRNA-Seq%2B%2528BSR-Seq%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036406%26rft_id%253Dinfo%253Apmid%252F22586469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22586469&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.26" data-doi="10.1186/gb-2014-15-6-r78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jost</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuon</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Himmelbach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Assfalg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mapping-by-sequencing accelerates forward genetics in barley.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">R78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253DR78%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-6-r78%26rft_id%253Dinfo%253Apmid%252F24917130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-6-r78&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24917130&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.27" data-doi="10.1104/pp.123.2.439"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCallum</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Targeting induced local lesions in genomes (TILLING) for plant functional genomics.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DMcCallum%26rft.auinit1%253DC.%2BM.%26rft.volume%253D123%26rft.issue%253D2%26rft.spage%253D439%26rft.epage%253D442%26rft.atitle%253DTargeting%2BInduced%2BLocal%2BLesions%2BIN%2BGenomes%2B%2528TILLING%2529%2Bfor%2BPlant%2BFunctional%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.123.2.439%26rft_id%253Dinfo%253Apmid%252F10859174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMjMvMi80MzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-6.8.2385.28"><div class="cit-metadata unstructured">McCarty, D. R., and R. B. Meeley, 2009 Transposon resources for forward and reverse genetics in maize, pp. 561–584 in <em>Handbook of Maize</em>, edited by J. L. Bennetzen and S. Hake. Springer, Berlin.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.29" data-doi="10.1111/j.1365-313X.2005.02509.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarty</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Settles</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latshaw</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Steady-state transposon mutagenesis in inbred maize.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DMcCarty%26rft.auinit1%253DD.%2BR.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D52%26rft.epage%253D61%26rft.atitle%253DSteady-state%2Btransposon%2Bmutagenesis%2Bin%2Binbred%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2005.02509.x%26rft_id%253Dinfo%253Apmid%252F16167895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2005.02509.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16167895&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231952000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.30" data-doi="10.1101/gr.147322.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obholzer</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megason</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moens</span>  <span class="cit-name-given-names">C. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RNA-seq-based mapping and candidate identification of mutations from forward genetic screens.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.147322.112%26rft_id%253Dinfo%253Apmid%252F23299976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIzLzQvNjc5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.31" data-doi="10.1534/genetics.104.033654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naito</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kusaba</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shikazono</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Transmissible and nontransmissible mutations induced by irradiating <em>Arabidopsis thaliana</em> pollen with gamma-rays and carbon ions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.033654%26rft_id%253Dinfo%253Apmid%252F15371348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY5LzIvODgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.32" data-doi="10.1038/nbt.2515"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordström</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albani</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutjahr</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartwig</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mutation identification by direct comparison of whole-genome sequencing data from mutant and wild-type individuals using k-mers.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DNordstrom%26rft.volume%253D31%26rft.spage%253D325%26rft.atitle%253DMutation%2Bidentification%2Bby%2Bdirect%2Bcomparison%2Bof%2Bwhole-genome%2Bsequencing%2Bdata%2Bfrom%2Bmutant%2Band%2Bwild-type%2Bindividuals%2Busing%2Bk-mers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2515%26rft_id%253Dinfo%253Apmid%252F23475072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2515&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23475072&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.33" data-doi="10.1016/0012-1606(86)90308-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poethig</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coe</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johri</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Cell lineage patterns in maize embryogenesis: A clonal analysis.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">392</span>–<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BBiol.%26rft.volume%253D117%26rft.spage%253D392%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252886%252990308-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(86)90308-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1986E396600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.34" data-doi="10.1104/pp.117.2.425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rahman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jane</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">M. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Characterization of SU1 isoamylase, a determinant of storage starch structure in maize.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DRahman%26rft.auinit1%253DA.%26rft.volume%253D117%26rft.issue%253D2%26rft.spage%253D425%26rft.epage%253D435%26rft.atitle%253DCharacterization%2Bof%2BSU1%2BIsoamylase%252C%2Ba%2BDeterminant%2Bof%2BStorage%2BStarch%2BStructure%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.117.2.425%26rft_id%253Dinfo%253Apmid%252F9625695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMTcvMi80MjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.35" data-doi="10.1111/pbi.12281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramirez-Gonzalez</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segovia</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bird</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fenwick</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holdgate</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">RNA-Seq bulked segregant analysis enables the identification of high-resolution genetic markers for breeding in hexaploid wheat.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">613</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D13%26rft.spage%253D613%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12281%26rft_id%253Dinfo%253Apmid%252F25382230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25382230&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabelli</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dante</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon-Kamm</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larkins</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Expression, regulation and activity of a B2-type cyclin in mitotic and endoreduplicating maize endosperm.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D5%26rft.spage%253D561%26rft_id%253Dinfo%253Apmid%252F25368625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25368625&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.37" data-doi="10.1270/jsbbs.56.179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishio</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mutant selection from progeny of gamma-ray-irradiated rice by DNA heteroduplex cleavage using <em>Brassica petiole</em> extract.</span> <abbr class="cit-jnl-abbrev">Breed. Sci.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBreed.%2BSci.%26rft.volume%253D56%26rft.spage%253D179%26rft_id%253Dinfo%253Adoi%252F10.1270%252Fjsbbs.56.179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1270/jsbbs.56.179&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000239230400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.38" data-doi="10.1186/s12864-015-1232-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schierenbeck</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ries</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogge</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grewe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weisshaar</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fast forward genetics to identify mutations causing a high light tolerant phenotype in <em>Chlamydomonas reinhardtii</em> by whole-genome-sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D57%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1232-y%26rft_id%253Dinfo%253Apmid%252F25730202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1232-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25730202&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.39" data-doi="10.1038/nrg3745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Using next-generation sequencing to isolate mutant genes from forward genetic screens.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">662</span>–<span class="cit-lpage">676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D662%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3745%26rft_id%253Dinfo%253Apmid%252F25139187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3745&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25139187&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.40" data-doi="10.1038/nmeth0809-550"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Juul</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SHOREmap: simultaneous mapping and mutation identification by deep sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">550</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchneeberger%26rft.auinit1%253DK.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253D550%26rft.epage%253D551%26rft.atitle%253DSHOREmap%253A%2Bsimultaneous%2Bmapping%2Band%2Bmutation%2Bidentification%2Bby%2Bdeep%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth0809-550%26rft_id%253Dinfo%253Apmid%252F19644454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth0809-550&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19644454&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268493700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-6.8.2385.41"><div class="cit-metadata unstructured">Settles, A. M., 2009 Transposon tagging and reverse genetics, pp. 143–159 in <em>Molecular Genetic Approaches to Maize Improvement</em>, edited by A. L. Kriz and B. A. Larkins. Springer-Verlag, Berlin.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.42" data-doi="10.1186/1471-2164-8-116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Settles</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holding</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latshaw</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Sequence-indexed mutations in maize using the UniformMu transposon-tagging population.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSettles%26rft.auinit1%253DA.%2BM.%26rft.volume%253D8%26rft.spage%253D116%26rft.epage%253D116%26rft.atitle%253DSequence-indexed%2Bmutations%2Bin%2Bmaize%2Busing%2Bthe%2BUniformMu%2Btransposon-tagging%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-116%26rft_id%253Dinfo%253Apmid%252F17490480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17490480&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirakawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunome</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabata</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isobe</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide survey of artificial mutations induced by ethyl methanesulfonate and gamma rays in tomato.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D14%26rft.spage%253D51%26rft_id%253Dinfo%253Apmid%252F25689669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25689669&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.44" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesirov</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYmliIjtzOjU6InJlc2lkIjtzOjg6IjE0LzIvMTc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.45" data-doi="10.1186/1471-2229-4-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Till</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weil</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burtner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Discovery of induced point mutations in maize genes by TILLING.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DTill%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D4%26rft.spage%253D12%26rft.epage%253D12%26rft.atitle%253DDiscovery%2Bof%2Binduced%2Bpoint%2Bmutations%2Bin%2Bmaize%2Bgenes%2Bby%2BTILLING.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-4-12%26rft_id%253Dinfo%253Apmid%252F15282033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-4-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15282033&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.46" data-doi="10.1093/bioinformatics/btp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TopHat: discovering splice junctions with RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp120%26rft_id%253Dinfo%253Apmid%252F19289445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNS85LzExMDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.47" data-doi="10.1038/nprot.2012.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">562</span>–<span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D562%26rft.epage%253D578%26rft.atitle%253DDifferential%2Bgene%2Band%2Btranscript%2Bexpression%2Banalysis%2Bof%2BRNA-seq%2Bexperiments%2Bwith%2BTopHat%2Band%2BCufflinks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2012.016%26rft_id%253Dinfo%253Apmid%252F22383036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2012.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22383036&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.48" data-doi="10.1186/1471-2229-12-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adamski</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mugford</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Febrer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Combining SNP discovery from next-generation sequencing data with bulked segregant analysis (BSA) to fine-map genes in polyploid wheat.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DTrick%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.spage%253D14%26rft.epage%253D14%26rft.atitle%253DCombining%2BSNP%2Bdiscovery%2Bfrom%2Bnext-generation%2Bsequencing%2Bdata%2Bwith%2Bbulked%2Bsegregant%2Banalysis%2B%2528BSA%2529%2Bto%2Bfine-map%2Bgenes%2Bin%2Bpolyploid%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-12-14%26rft_id%253Dinfo%253Apmid%252F22280551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-12-14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22280551&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.49" data-doi="10.1105/tpc.109.073452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vollbrecht</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duvick</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schares</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahern</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deewatthanawong</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide distribution of transposed dissociation elements in maize.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1667</span>–<span class="cit-lpage">1685</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.073452%26rft_id%253Dinfo%253Apmid%252F20581308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIyLzYvMTY2NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.50" data-doi="10.1104/pp.92.1.191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paiva</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larkins</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">New methods for extraction and quantitation of zeins reveal a high content of γ-zein in modified <em>opaque-2</em> maize.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DWallace%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D191%26rft.epage%253D196%26rft.atitle%253DNew%2BMethods%2Bfor%2BExtraction%2Band%2BQuantitation%2Bof%2BZeins%2BReveal%2Ba%2BHigh%2BContent%2Bof%2B%257Bgamma%257D-Zein%2Bin%2BModified%2Bopaque-2%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.92.1.191%26rft_id%253Dinfo%253Apmid%252F16667246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo4OiI5Mi8xLzE5MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.51" data-doi="10.1105/tpc.112.101360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>Opaque1</em> encodes a myosin XI motor protein that is required for endoplasmic reticulum motility and protein body formation in maize endosperm.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">3447</span>–<span class="cit-lpage">3462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.112.101360%26rft_id%253Dinfo%253Apmid%252F22892319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMzQ0NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.52" data-doi="10.1105/tpc.114.125559"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Proline responding1 plays a critical role in regulating general protein synthesis and the cell cycle in maize.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2582</span>–<span class="cit-lpage">2600</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.125559%26rft_id%253Dinfo%253Apmid%252F24951479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI2LzYvMjU4MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.53" data-doi="10.1007/s11103-004-5112-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lei</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baraoidan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bordeos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Chemical- and irradiation-induced mutants of indica rice IR64 for forward and reverse genetics.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DWu%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D59%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D97%26rft.atitle%253DChemical-%2Band%2Birradiation-induced%2Bmutants%2Bof%2Bindica%2Brice%2BIR64%2Bfor%2Bforward%2Band%2Breverse%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-004-5112-0%26rft_id%253Dinfo%253Apmid%252F16217604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-004-5112-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16217604&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232498000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.54" data-doi="10.1186/gb-2012-13-8-r77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">ggbio: an R package for extending the grammar of graphics for genomic data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">R77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DYin%26rft.auinit1%253DT.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253DR77%26rft.epage%253DR77%26rft.atitle%253Dggbio%253A%2Ban%2BR%2Bpackage%2Bfor%2Bextending%2Bthe%2Bgrammar%2Bof%2Bgraphics%2Bfor%2Bgenomic%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-8-r77%26rft_id%253Dinfo%253Apmid%252F22937822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-8-r77&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22937822&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.55" data-doi="10.3109/09553000903336826"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshihara</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hase</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takimoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narumi</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mutational effects of different LET radiations in <em>rpsL</em> transgenic <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Int. J. Radiat. Biol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bradiation%2Bbiology%26rft.stitle%253DInt%2BJ%2BRadiat%2BBiol%26rft.aulast%253DYoshihara%26rft.auinit1%253DR.%26rft.volume%253D86%26rft.issue%253D2%26rft.spage%253D125%26rft.epage%253D131%26rft.atitle%253DMutational%2Beffects%2Bof%2Bdifferent%2BLET%2Bradiations%2Bin%2BrpsL%2Btransgenic%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.3109%252F09553000903336826%26rft_id%253Dinfo%253Apmid%252F20148698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3109/09553000903336826&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20148698&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.56" data-doi="10.1104/pp.113.230961"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kianian</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holding</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Deletion mutagenesis identifies a haploinsufficient role for gamma-zein in <em>opaque2</em> endosperm modification.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.113.230961%26rft_id%253Dinfo%253Apmid%252F24214534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjQvMS8xMTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2385.57" data-doi="10.1534/genetics.110.119230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuryn</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Gras</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jamet</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarriault</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A strategy for direct mapping and identification of mutations by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.119230%26rft_id%253Dinfo%253Apmid%252F20610404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzEvNDI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/8/2385.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2375" title="Septate Junction Proteins Play Essential Roles in Morphogenesis Throughout Embryonic Development in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2397" title="A Genome-Wide Screen for Dendritically Localized RNAs Identifies Genes Required for Dendrite Morphogenesis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/8"><span>Volume 6 Issue 8, August 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/8.cover-source.jpg?itok=40Kz9Oeq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2385" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZOeJGL23JJ35ovZKIbVOeNmSFn1KuRxQamwybeMvR0A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/8/2385" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/8/2385" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/8/2385" class="active" data-icon-position="" data-hide-link-title="0">A Population of Deletion Mutants and an Integrated Mapping and Exome-seq Pipeline for Gene Discovery in Maize</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460437" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-TmHqlD3-nzvg06JrrY17rXg1MH-RVfWmNu73GZkMuKA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2385.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/8/2385" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-xES1LhXWHuVdlnM8hBa2DYgjlrFbbbYA5SNdC7FPQDs" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/8/2385" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2385.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460437" id="node460437--2" data-pisa="ggg;6/8/2385" data-pisa-master="ggg;g3.116.030528" data-apath="/ggg/6/8/2385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Population of Deletion Mutants and an Integrated Mapping and Exome-seq Pipeline for Gene Discovery in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Shangang Jia</span>, <span class="highwire-citation-author" data-delta="1">Aixia Li</span>, <span class="highwire-citation-author" data-delta="2">Kyla Morton</span>, <span class="highwire-citation-author" data-delta="3">Penny Avoles-Kianian</span>, <span class="highwire-citation-author" data-delta="4">Shahryar F. Kianian</span>, <span class="highwire-citation-author" data-delta="5">Chi Zhang</span> and <span class="highwire-citation-author" data-delta="6">David Holding</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2385-2395;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030528 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460437--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shangang Jia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shangang%2BJia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jia%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShangang%2BJia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aixia Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aixia%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAixia%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kyla Morton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyla%2BMorton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morton%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyla%2BMorton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Penny Avoles-Kianian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS Cereal Disease Laboratory, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Penny%2BAvoles-Kianian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Avoles-Kianian%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APenny%2BAvoles-Kianian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Shahryar F. Kianian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS Cereal Disease Laboratory, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shahryar%2BF.%2BKianian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kianian%20SF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShahryar%2BF.%2BKianian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Chi Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Biological Sciences, Center for Plant Science and Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chi%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChi%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">David Holding </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BHolding%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holding%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BHolding%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dholding2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">dholding2@unl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/8/2385" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/8/2385&amp;title=A%20Population%20of%20Deletion%20Mutants%20and%20an%20Integrated%20Mapping%20and%20Exome-seq%20Pipeline%20for%20Gene%20Discovery%20in%20Maize" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/8/2385&amp;title=A%20Population%20of%20Deletion%20Mutants%20and%20an%20Integrated%20Mapping%20and%20Exome-seq%20Pipeline%20for%20Gene%20Discovery%20in%20Maize" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/8/2385&amp;title=A%20Population%20of%20Deletion%20Mutants%20and%20an%20Integrated%20Mapping%20and%20Exome-seq%20Pipeline%20for%20Gene%20Discovery%20in%20Maize" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/8/2385&amp;text=A%20Population%20of%20Deletion%20Mutants%20and%20an%20Integrated%20Mapping%20and%20Exome-seq%20Pipeline%20for%20Gene%20Discovery%20in%20Maize" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/8/2385&amp;title=A%20Population%20of%20Deletion%20Mutants%20and%20an%20Integrated%20Mapping%20and%20Exome-seq%20Pipeline%20for%20Gene%20Discovery%20in%20Maize" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/8/2385&amp;t=A%20Population%20of%20Deletion%20Mutants%20and%20an%20Integrated%20Mapping%20and%20Exome-seq%20Pipeline%20for%20Gene%20Discovery%20in%20Maize" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/8/2385&amp;title=A%20Population%20of%20Deletion%20Mutants%20and%20an%20Integrated%20Mapping%20and%20Exome-seq%20Pipeline%20for%20Gene%20Discovery%20in%20Maize" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/8/2385&amp;title=A%20Population%20of%20Deletion%20Mutants%20and%20an%20Integrated%20Mapping%20and%20Exome-seq%20Pipeline%20for%20Gene%20Discovery%20in%20Maize" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460437/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460437" id="node460437--3" data-pisa="ggg;6/8/2385" data-pisa-master="ggg;g3.116.030528" data-apath="/ggg/6/8/2385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Population of Deletion Mutants and an Integrated Mapping and Exome-seq Pipeline for Gene Discovery in Maize</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Shangang Jia</span>, <span class="highwire-citation-author" data-delta="1">Aixia Li</span>, <span class="highwire-citation-author" data-delta="2">Kyla Morton</span>, <span class="highwire-citation-author" data-delta="3">Penny Avoles-Kianian</span>, <span class="highwire-citation-author" data-delta="4">Shahryar F. Kianian</span>, <span class="highwire-citation-author" data-delta="5">Chi Zhang</span> and <span class="highwire-citation-author" data-delta="6">David Holding</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2385-2395;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030528 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460437--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shangang Jia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shangang%2BJia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jia%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShangang%2BJia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aixia Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aixia%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAixia%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kyla Morton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kyla%2BMorton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morton%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKyla%2BMorton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Penny Avoles-Kianian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS Cereal Disease Laboratory, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Penny%2BAvoles-Kianian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Avoles-Kianian%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APenny%2BAvoles-Kianian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Shahryar F. Kianian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>USDA-ARS Cereal Disease Laboratory, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shahryar%2BF.%2BKianian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kianian%20SF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShahryar%2BF.%2BKianian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Chi Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Biological Sciences, Center for Plant Science and Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chi%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChi%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">David Holding </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, Center for Plant Science Innovation, Beadle Center for Biotechnology, University of Nebraska, Lincoln, Nebraska 68588</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BHolding%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holding%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BHolding%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dholding2@unl.edu" class="" data-icon-position="" data-hide-link-title="0">dholding2@unl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2385&amp;rft.epage=2395&amp;rft.atitle=A%20Population%20of%20Deletion%20Mutants%20and%20an%20Integrated%20Mapping%20and%20Exome-seq%20Pipeline%20for%20Gene%20Discovery%20in%20Maize&amp;rft.volume=6&amp;rft.issue=8&amp;rft.date=2016-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jia%2C+Shangang&amp;rft.au=Li%2C+Aixia&amp;rft.au=Morton%2C+Kyla&amp;rft.au=Avoles-Kianian%2C+Penny&amp;rft.au=Kianian%2C+Shahryar+F.&amp;rft.au=Zhang%2C+Chi&amp;rft.au=Holding%2C+David"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460437/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460437/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460437/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460437/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460437/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460437/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460437/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460437/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460437/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460437/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460437/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460437/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/8/2385" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/8/2385.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/8/2385.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/8/2385.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/8/2385.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
