<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/8/2329" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/8/2329.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/8/2329.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/8/2329.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/8/2329/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/8/2329.atom" />
<meta name="HW.pisa" content="ggg;6/8/2329" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions" />
<meta name="DC.Identifier" content="10.1534/g3.116.031443" />
<meta name="DC.Date" content="2016-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Jarquin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The identification and mobilization of useful genetic variation from germplasm banks for use in breeding programs is critical for future genetic gain and protection against crop pests. Plummeting costs of next-generation sequencing and genotyping is revolutionizing the way in which researchers and breeders interface with plant germplasm collections. An example of this is the high density genotyping of the entire USDA Soybean Germplasm Collection. We assessed the usefulness of 50K single nucleotide polymorphism data collected on 18,480 domesticated soybean ( Glycine max ) accessions and vast historical phenotypic data for developing genomic prediction models for protein, oil, and yield. Resulting genomic prediction models explained an appreciable amount of the variation in accession performance in independent validation trials, with correlations between predicted and observed reaching up to 0.92 for oil and protein and 0.79 for yield. The optimization of training set design was explored using a series of cross-validation schemes. It was found that the target population and environment need to be well represented in the training set. Second, genomic prediction training sets appear to be robust to the presence of data from diverse geographical locations and genetic clusters. This finding, however, depends on the influence of shattering and lodging, and may be specific to soybean with its presence of maturity groups. The distribution of 7608 nonphenotyped accessions was examined through the application of genomic prediction models. The distribution of predictions of phenotyped accessions was representative of the distribution of predictions for nonphenotyped accessions, with no nonphenotyped accessions being predicted to fall far outside the range of predictions of phenotyped accessions." />
<meta name="DC.Contributor" content="Diego Jarquin" />
<meta name="DC.Contributor" content="James Specht" />
<meta name="DC.Contributor" content="Aaron Lorenz" />
<meta name="article:published_time" content="2016-08-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The identification and mobilization of useful genetic variation from germplasm banks for use in breeding programs is critical for future genetic gain and protection against crop pests. Plummeting costs of next-generation sequencing and genotyping is revolutionizing the way in which researchers and breeders interface with plant germplasm collections. An example of this is the high density genotyping of the entire USDA Soybean Germplasm Collection. We assessed the usefulness of 50K single nucleotide polymorphism data collected on 18,480 domesticated soybean (&lt;i&gt;Glycine max&lt;/i&gt;) accessions and vast historical phenotypic data for developing genomic prediction models for protein, oil, and yield. Resulting genomic prediction models explained an appreciable amount of the variation in accession performance in independent validation trials, with correlations between predicted and observed reaching up to 0.92 for oil and protein and 0.79 for yield. The optimization of training set design was explored using a series of cross-validation schemes. It was found that the target population and environment need to be well represented in the training set. Second, genomic prediction training sets appear to be robust to the presence of data from diverse geographical locations and genetic clusters. This finding, however, depends on the influence of shattering and lodging, and may be specific to soybean with its presence of maturity groups. The distribution of 7608 nonphenotyped accessions was examined through the application of genomic prediction models. The distribution of predictions of phenotyped accessions was representative of the distribution of predictions for nonphenotyped accessions, with no nonphenotyped accessions being predicted to fall far outside the range of predictions of phenotyped accessions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/08/01" />
<meta name="citation_mjid" content="ggg;6/8/2329" />
<meta name="citation_id" content="6/8/2329" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/8/2329" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/8/2329.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/8/2329.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/8/2329.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.031443" />
<meta name="citation_pmid" content="27247288" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="2329" />
<meta name="citation_lastpage" content="2341" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Diego Jarquin" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Nebraska 68583-0915" />
<meta name="citation_author" content="James Specht" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Nebraska 68583-0915" />
<meta name="citation_author" content="Aaron Lorenz" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108" />
<meta name="citation_author_email" content="lore0149@umn.edu" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals.;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=N. Bandillo;citation_author=D. Jarquin;citation_author=Q. Song;citation_author=R. Nelson;citation_author=P. Cregan;citation_title=A population structure and genome-wide association analysis on the USDA Soybean Germplasm Collection.;citation_year=2015;citation_doi=10.3835/plantgenome2015.04.0024" />
<meta name="citation_reference" content="citation_title=Soybeans: Improvement, Production, and Uses;citation_title=Genetic diversity in soybean;citation_author=T. E. Carter;citation_author=R. L. Nelson;citation_author=C. H. Sneller;citation_author=Z. Cui;citation_pages=303-416;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=C. M. Fox;citation_author=T. R. Cary;citation_author=R. L. Nelson;citation_author=B. W. Diers;citation_title=Confirmation of a seed yield QTL in soybean.;citation_pages=992-998;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.10.0688" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Funatsuki;citation_author=M. Suzuki;citation_author=A. Hirose;citation_author=H. Inaba;citation_author=T. Yamada;citation_title=Molecular basis of a shattering resistance boosting global dissemination of soybean.;citation_volume=111;citation_year=2014;citation_pmid=25468966;citation_doi=10.1073/pnas.1417282111" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=D. J. Garrick;citation_title=Genomic BLUP decoded: a look into the black box of genomic prediction.;citation_volume=194;citation_year=2013;citation_pmid=23640517;citation_doi=10.1534/genetics.113.152207" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Hickey;citation_author=S. Dreisigacker;citation_author=J. Crossa;citation_author=S. Hearne;citation_author=R. Babu;citation_title=Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation.;citation_pages=1476-1488;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.03.0195" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=D. Hoisington;citation_author=M. Khairallah;citation_author=T. Reeves;citation_author=J.-M. Ribaut;citation_author=B. Skovmand;citation_title=Plant genetic resources: What can they contribute toward increased crop productivity?;citation_pages=5937-5943;citation_volume=96;citation_year=1999;citation_issue=11;citation_pmid=10339521;citation_doi=10.1073/pnas.96.11.5937" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=J.-Y. Li;citation_author=J. Wang;citation_author=R. S. Zeigler;citation_title=The 3,000 rice genomes project: new opportunities and challenges for future rice research.;citation_pages=8;citation_volume=3;citation_year=2014;citation_pmid=24872878;citation_doi=10.1186/2047-217X-3-8" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=C. F. H. Longin;citation_author=J. C. Reif;citation_title=Redesigning the exploitation of wheat genetic resources.;citation_pages=631-636;citation_volume=19;citation_year=2014;citation_pmid=25052155;citation_doi=10.1016/j.tplants.2014.06.012" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_title=Adding genetically distant individuals to training populations reduces genomic prediction accuracy in barley.;citation_pages=2657-2667;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.12.0827" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=S. R. McCouch;citation_author=K. L. McNally;citation_author=W. Wang;citation_author=R. S. Hamilton;citation_title=Genomics of gene banks: a case study in rice.;citation_volume=99;citation_year=2012;citation_pmid=22314574;citation_doi=10.3732/ajb.1100385" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Plant Genet. Resour.;citation_author=R. L. Nelson;citation_title=Managing self-pollinated germplasm collections to maximize utilization.;citation_pages=123-133;citation_volume=9;citation_year=2011;citation_doi=10.1017/S147926211000047X" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=D. Sehgal;citation_author=P. Vikram;citation_author=C. P. Sansaloni;citation_author=C. Ortiz;citation_author=C. S. Pierre;citation_title=Exploring and mobilizing the gene bank biodiversity for wheat improvement.;citation_pages=e0132112;citation_volume=10;citation_year=2015;citation_pmid=26176697;citation_doi=10.1371/journal.pone.0132112" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Q. Song;citation_author=D. L. Hyten;citation_author=G. Jia;citation_author=C. V. Quigley;citation_author=E. W. Fickus;citation_title=Development and evaluation of SoySNP50K, a high-density genotyping array for soybean.;citation_pages=e54985;citation_volume=8;citation_year=2013;citation_pmid=23372807;citation_doi=10.1371/journal.pone.0054985" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Q. Song;citation_author=D. L. Hyten;citation_author=G. Jia;citation_author=C. V. Quigley;citation_author=E. W. Fickus;citation_title=Fingerprinting soybean germplasm and its utility in genomic research.;citation_volume=5;citation_year=2015;citation_pmid=26224783;citation_doi=10.1534/g3.115.019000" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. D. Tanksley;citation_author=S. Grandillo;citation_author=T. M. Fulton;citation_author=D. Zamir;citation_author=Y. Eshed;citation_title=Advanced backcross QTL analysis in a cross between an elite processing line of tomato and its wild relative L. pimpinellifolium.;citation_pages=213-224;citation_volume=92;citation_year=1996;citation_pmid=24166170;citation_doi=10.1007/s001220050116" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=F. Technow;citation_author=A. Bürger;citation_author=A. E. Melchinger;citation_title=Genomic Prediction of Northern Corn Leaf Blight Resistance in Maize with Combined or Separated Training Sets for Heterotic Groups;citation_pages=197-203;citation_volume=3;citation_year=2013;citation_issue=2;citation_pmid=23390596;citation_doi=10.1534/g3.112.004630" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="twitter:description" content="The identification and mobilization of useful genetic variation from germplasm banks for use in breeding programs is critical for future genetic gain and protection against crop pests. Plummeting costs of next-generation sequencing and genotyping is revolutionizing the way in which researchers and breeders interface with plant germplasm collections. An example of this is the high density genotyping of the entire USDA Soybean Germplasm Collection. We assessed the usefulness of 50K single nucleotide polymorphism data collected on 18,480 domesticated soybean ( Glycine max ) accessions and vast historical phenotypic data for developing genomic prediction models for protein, oil, and yield. Resulting genomic prediction models explained an appreciable amount of the variation in accession performance in independent validation trials, with correlations between predicted and observed reaching up to 0.92 for oil and protein and 0.79 for yield. The optimization of training set design was explored using a series of cross-validation schemes. It was found that the target population and environment need to be well represented in the training set. Second, genomic prediction training sets appear to be robust to the presence of data from diverse geographical locations and genetic clusters. This finding, however, depends on the influence of shattering and lodging, and may be specific to soybean with its presence of maturity groups. The distribution of 7608 nonphenotyped accessions was examined through the application of genomic prediction models. The distribution of predictions of phenotyped accessions was representative of the distribution of predictions for nonphenotyped accessions, with no nonphenotyped accessions being predicted to fall far outside the range of predictions of phenotyped accessions." />
<meta name="og-title" property="og:title" content="Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/8/2329" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The identification and mobilization of useful genetic variation from germplasm banks for use in breeding programs is critical for future genetic gain and protection against crop pests. Plummeting costs of next-generation sequencing and genotyping is revolutionizing the way in which researchers and breeders interface with plant germplasm collections. An example of this is the high density genotyping of the entire USDA Soybean Germplasm Collection. We assessed the usefulness of 50K single nucleotide polymorphism data collected on 18,480 domesticated soybean ( Glycine max ) accessions and vast historical phenotypic data for developing genomic prediction models for protein, oil, and yield. Resulting genomic prediction models explained an appreciable amount of the variation in accession performance in independent validation trials, with correlations between predicted and observed reaching up to 0.92 for oil and protein and 0.79 for yield. The optimization of training set design was explored using a series of cross-validation schemes. It was found that the target population and environment need to be well represented in the training set. Second, genomic prediction training sets appear to be robust to the presence of data from diverse geographical locations and genetic clusters. This finding, however, depends on the influence of shattering and lodging, and may be specific to soybean with its presence of maturity groups. The distribution of 7608 nonphenotyped accessions was examined through the application of genomic prediction models. The distribution of predictions of phenotyped accessions was representative of the distribution of predictions for nonphenotyped accessions, with no nonphenotyped accessions being predicted to fall far outside the range of predictions of phenotyped accessions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
    <title>Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"jX4jm1-9LO7JQQtxOXMgQXTKiDcx6hgmMVM8BohaSD8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460251","apath":"\/ggg\/6\/8\/2329.atom","pisa":"ggg;6\/8\/2329","ac":{"\/ggg\/6\/8\/2329.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/8\/2329.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2329"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2329"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/8\/2329":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Z2gJ\\\\\\\/v09gHRTZosMCf5wqVEI4mC4SHLVo2mL3BXDtohXjHfKnAA3oHTipSvOOVYHA6jjFFXSAwO2BeZovyGCSsFl7T6fEXCzzpwXj4rCbHxD5zCEbuyC7da53qHGtUzv00vQ4W0o3Yb8UM+umpE9j6tNb409cJ304gNYjHgm9uHUptapDWL64HQieLPYBDae1MDxrHYPwjitQ14gDz05Hov4PvlNGp9NBBceoU7aZOHULUFIJy\\\\\\\/GqcNzr6WVYVFS17pZDGF2yFEuR5QKnzW316WEFf5eSrfuCKkWP8KWBmbZJZQWq2FF6QLc7RIMIMBWc6XxH7llD\\\\\\\/78Qt9VrkoNmm4IplIidKONTUYF7BStu1MiwLqkGZqRBQ0EvAWsjflFRHIs3\\\\\\\/hQlcdNkiBQneK34ZTTOGRQKEO0jFh8A1Zj1le5PBfQpSKNrino68quVS7OZE5R68BIWBcM0rbgRkv1eVEGLgfFG0IH+jJ62bps742G5cedJlZRdZ5bqt\\\\\\\/Pjr3XBtjdvCeAMUicXyPAR+C9LEMUyhareijnMCO6cI9C9UQcvJu8LfY1+RT2lvKZ6hdQFFCIVrDEwnmMmepETgmEzOCbhw5aVh9KTbQ2V7zwf2nQ0QdOW203Ko\\\\\\\/cGpQ5K3\\\\\\\/lLWYc0a2x9gVm1fMg9OWEb6iZkOqrbMklwLjUSoxwyGjKuacJ8N9l3k\\\\\\\/3h\\\\\\\/jesbMRkcTIgt2bSTZSs9jJ57wymkTu1j\\\\\\\/fVWvSzB6YEtCT7rXFoR0r++6H0QNu3Tl9R4wieGEaU4Zl\\\\\\\/1m69Pl8NmGlq3Bk5wauquWzvnVFke73R8yna3eBt9Lc5og\\\\\\\/VJT4bAFThhELl7354I5PJ2Oxvt0U5MlYK+LYJ5uQ31Q0OU78iVQZVB4NsMfOsKQZma1VMeWhk1Q6JBEBjqKjrXBFZ1Y2+0QePJEekJDmdB2BDzboBhc5Gn9xDkThgZjFVRlHf9Jk0M6ML9Ag770r1Ihb6cc62PjD8wCITFLsw4YwZUNqcpNMmHM7hM3zgols23N+wq14V1RiB0oXH+l7alkTEKTGL4rEt7auwdZyAZ3YpVcqlndSpDez9yx92XJ0Y4h+OnA4HU4hD6DTh\\\\\\\/n34cg0fzqToU7NNwKGu2FZvX+I5ypS+yh8AIkcyWnBUpLnFA12dHaYwH3I13\\\\\\\/0xCn+UqWmuEsZYgmQpFbkT1rWiesQguceHZUuFdN3mvafbQKVh78v4\\\\\\\/VPC\\\\\\\/VzziMrAQHoUayM23vfmIJPzVygmqqEKe+WLCr6xXxhmbUTQKSB9lWYbrUIfcJR\\\u0022,\\\u0022iv\\\u0022:\\\u0022NRgqiKjVMVeICcemWFFK7A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222b715a3ee0d31e85dc27d39d000e7639ed1b6b2507e8d85b57ebde4f75826ac6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226XynJ+uW7l2GZ55qBcjC3BEie7Wd0xLG4JMK1aNEExT7zXCtSUshv8mnGycflkyTuorSA0zAheajID\\\\\\\/OeLX+ygB3PmL34H+mHrse\\\\\\\/Dq2mNuGqMeLN5SLgCP5hIWbu4RENj8bnSxQWdbBiDa5cjb64m1UUb6VMsYLPHORDk5pprzM8j0zhXtR\\\\\\\/tONCaVeOZ9GcErsom7WmsFhZNan01XUMl8YBSQEIdOryyIlO4mzz4CMAYd8EdbOFgyz78gqRieO+FAgtTAbsGnYM9TYO2HCsvePMELIAURJlJHbo8aW3bDMo8IlAT4xvdQeJbj4GhC\\\\\\\/lVcogwHaAGt4n9ulpmrHO5ROWYpdS\\\\\\\/dS4seRPTu8WerAfkcrBKCJraTlWWMr\\\\\\\/1ab+JIW2kN9RzPO16m2G9GtHKIklYkn8F8jXqTGmtmux3rPN44ORWBj5V9MAvD3VxzYu3PXGhlpQ+ptkAUFqYbxxfIw1exOM8FxIAx3dO6pa9\\\\\\\/tr4gTZVQmzwsp\\\\\\\/yp+DNVca\\\\\\\/jNvi\\\\\\\/PYS3jxqYx2UcQYANxTrhHaZRzbQwda6AwHCclq1wHk36r1G3CTRWBoUK51FhnjEi33pOFWCO3dNuVxQ+NOgawQwI94a\\\\\\\/WYi65xBBAPFs2OT5gBIq5cS5rjJAiQJKgiRUBWjy0\\\\\\\/pgMnpYviNZBluge+41D4fg\\\\\\\/F7IRcN7m7w8c3TM\\\\\\\/IMwLpEeB8wk+L3rO3LcfWzyEJpRZnaQWumH8PC87GGX8HxjbVmHI20Krldle61GPihPgkE3yX34XDuAx938v2vKIr91Kl56huBNTrjAr5YvfPgeyWm4DX\\\\\\\/4UcUtCJVQEJrCH8Eceqql4viWgLNu1x3n1uSIG+4J3gTT0OgRPd2lGehutEztCccc3J\\\\\\\/9Bwyg\\\\\\\/XgQVmQwLeWxMfHMLZFnHIjsabhmaELNqlQ2QqzLWKTNzb56MVplpriY+TqnfL3TvukzEoSXAiigULf9TgZbGF7IO6Dw\\\\\\\/2mQDIV7efrPjxsHZJFFwwsysf71ZDoRmef1xJQXSxIYIG7oSTsg4AS7IeTQ4sh2BDq79YfPaC0wwa7fOfxX1xFT9piJ7u6dyYx7C4h4aaM+gUEwqjnH1O2HF68teOam1Evq+mamoIc62Cv7AcKyC3yASjQ3G1q1aZ3uTkb0hZgOWtfdBWqkXFPv4kJ\\\\\\\/BgruKoaZ7GDls2bixgjqmfZzly0bawf5Aco58iUq7ZE96IBlO1n2MTC4H7zoYfDUVZCGZQ8jINZm5AG6eHNLOCQEQT63eAGn30lAUIPLmBnTmRPcv7sz0bLyfDRga3GsK1Yh3tuTA2I74o5wNl7cClfTYgHFAUmjBVrIVkhlUB5rB0OdCsPopFZdfee9Hrfu4umezpMClJofrjui2RwPpsd+FrkCA9VbZRuxX\\\\\\\/YmxfAy87oE+Pgkh1o6fCPDThORvXORQFA1tzp4cb7j5VBSCNtib1tVywkGYVQsvOE3TKt+C30uEofp4H6t9cl0KAc78U3w3k7uCg27SJnV\\\\\\\/5O5xxIeViFGd98nzCCcG4PVQTweYCX+XulQm1GN+j0EHwTW4wyliLubEEAOZ8q7OCe5XcoK4pLMWehr5k8B0mw0PKZiOUFao6bQqFJZmainxoZvzzbTVrRc678e6wDq\\\\\\\/sgim0cGJECitWep\\\\\\\/1qSU35X1\\\\\\\/fhJwLAoMc4JmxoRZeO1Gta5koUGuHCGePhImSRk4R9DdV0ZU6qiJ7Gtw7pEhkssmiGQMtp\\\\\\\/xGx5kiBQiIS85pFEv\\\\\\\/WQlAD\\\\\\\/2VCGU53+hN6xUHM3AquQCtYNDOhKGpTujYeMfS1MqJxHv33B7Q8SkTOQXFPZnu8+6m8J6sxHvurwKaRHUYMcwdNpeUO4wPvQZM76ZgZSKxj+H1dz6DAXW\\\\\\\/3VqAbA4z7wT7jevAwo9F7W7OtbWw2njFqL0YI4OflQDTzrP+EPkAByR7gW99FR+eybA3nvWgRBOcV+jU6mtPBm8NPmSH+IVC9qb+pKAsU3R09GBma9u+IgS50F4fH+pvYpecNRnc3DUgWeaQZwVPoHFH0kaXtORjPTLsRWjm394MUtseN84UpYAB+MoR3S33lWlZxESfzqiX+cAWDgB+f2QwkAoR8ToZi0FNF9mPt9S9jd0mGzyQRhh+cg9T386ckmxoMcVW8GFeD9P4yHrWGmKtpq++5Urf5EX6fsgxe\\\\\\\/ckiteiKCvexfAzbeIejJpsh\\\\\\\/i0IvMT3UnT5hvDcDMd1mnal412BXus5eWtnSjd9o2b4C8FDF4eYzZdh24MlU8bx8tYhn5vfplSCT+N+8R1Xln1HaQhGTWzQsMHqJasDE89uDcSaQSXl4Hyb+\\\\\\\/xKO+RyvA\\\\\\\/lrHzmHnbym1\\\\\\\/Y3syuqxzmxfdfcj3SQZ0RQ0eaRuJXXJ+UHcE3a\\\\\\\/qUZN1fLzCCMvKdPWgRbnU5CF5NWz3vej5S1ZzXZ\\\\\\\/RS+N6vYqF9kNsVZdK3QPjmFmzMjVY2cDsnuo1uJ67pWEIZmkTJ\\\\\\\/LWXthSOxQ0GOhYt4\\\\\\\/rkDj4ucMXuM\\\\\\\/Cs1KFF76va5W9kVs+51St383itMbP4KSUSA4cEHUZiaPe3DGQC8a29LGZJU4rwyGgI6Tp+IlgNS+8pwDX1rmi+VVUcMMdLnB97X3hKPmr80lgFOutK4FrUZpMdQE2+nu2OISY7\\\\\\\/8\\\\\\\/Hfo0NPKJRtRDoBS5i1+jYtJ4Ac5LhiH2PldL\\\\\\\/JTYP60PLNgnFq92pa0mMTypiXagdhLnhFHZn9VHfSJ7LnmvQJKIm4Huz3NecwNG8JxrMcVe7SxPEqrKYON0oMuaWAStLQFCWvDaeOtb9F99En4u5mKjwco6vPCOmbXUA5Vjcqe8fPBc7swZnVfRS5e\\\\\\\/oAyvHaKx8L\\\\\\\/muoQdLoJZQtApyth6+Ei15Dzw35LcHn+kuJJSJatoNhCUvV+R+Ra1Q9U2vohwSCTZtmkteCHvwmW4JkSNuNVaHGOFLoAvheDtZCjGPYqQs+cr52RlPeDk1+ziuyk4u8POwrNPXHZ7I4IQOeydIt7jMvdZb80+YsDQ6eJW2Hqzs5n44dEm4oVwgIV94R5GDW\\\\\\\/Xra+eeiSB0UKi4Rov01bSqlEgzPNZf\\\\\\\/2AS11AJz+YrjbIIeG17angdwf1ofrPHxKvVucTi6Y4mYevqmCUYHpYg6OUcnz271pUQRTfjVvmays3GALu4NWYNh2Sd2eqlIrMZf5ickWWXN00xgzBTTtkixQ20VhurB39Rbvij7SNgEwnTkbPkJYiIVmivjYAkmUkHycqyDzodBojgO0r8Pl5pmDfjW1YStQvWZx9ebAwKi5b6w7FDfOWkfuoOwngJwvldhbc2Zi2cBVU6ACi5diPkN5nzWvI+r+qAnqEH48UIegG+QldISuZRUT9e7Lw38gzkG9nCrpWYnzBUnMDwciZcPzq++O45T9PTIoxmgdHai0JjeFN3WYbW2ZQt2SI6Mckp3ZVFOg1yH2bDCx752bu2yXYMM1vylhaR9EMC\\\\\\\/zDQdoq8AloT80okNyczEHLNWqL1g7Ay\\\\\\\/4pkJeFWMtL4YGSgSbXV2KZf75fJqMtnQ+frzBzI599ARwboX5LEZJRBnjhxwU5u9l5rmuvK+24+HsN0A+uJxBddO26QyG0Kv6eIMSIQLp2AlqsPoXdWWV\\\\\\\/xhRkDp55M7HedC6pWkuS9CIlzTxwgjVrMcOOPuIu11S0C3WdkpjJJyXrjvd00D\\\\\\\/v74C\\\\\\\/Zm97dd+uomI7C\\\\\\\/jqsYz1g3RvLHkeSguPP1skbnSAKMiCY+36D7W6+VDa1vGYKa37ZdDp3IE2IchGAW8zfoXc2uoMI0ujFre0NBvrzt4boG4nNMMgtuQaXBioGqVP5cAMBwRU2OpD7P1ek8xFORZOPB\\\\\\\/3YtpWSJd8IJ8rxqIHy6wQksGTBwnR2k9IVuzMA+G4rSP+faJhCMpM4MiR3YouFREpzkJmn7vE2iblBIWDis6e3dfrbHonavsV5Q4kejiWjjLe3uoaxNMfBb7Nl7z\\\\\\\/KvPRyCIW6yJLFrETduAZj8n61Z2Myr4KHp7nFNJtcyujK0tleXRQtn0UPNY48mocwm0XTcaHGWGX6kOBWyLKnmRY2Ac1qxZb5f7f6KFDgTOk5phw5H0DSiD2KXY0\\\\\\\/Prv16WmbnZFGW9GUplQtkfuJkjd99w5VnGTV0NgNPm8fmEtN1RbzPMV90TBDQxGY7Uk6W72rrQfcCV8scdgi0Iu9j6hdCtPn78pByPTnAdbIGZSqD\\\\\\\/Q8mDLv6RXxuxJeeuM9xtSu4iEvPv7GslMEKLmatrz9+BPoWsNjADa94KY0T8fqlzZj1TKCgy6w3aai7HZQbXdUBqiCopcKNlh6xMjRnfk4uHGYo\\\\\\\/1bn1FKOwmyZDm6eXYY2iBV6mcCwAAD+7pxHmZbghwbl\\\\\\\/0luf4XLi\\\\\\\/PNf+lNnCBKFQkaImARgTaqxWrDbuPlidAds4QLumN171ZyXa4SeB+4VTmAD3OxhuHroANxG6Ny0N61u62j91iV8EI2Z1kOEEcC\\\\\\\/fgK4g05wvDN4RljEQfs7ZCwTPmJX1tG3x8MwzSNX4BXPJH9t7tOHKXQWSpA5bTpT5jWvOk7JghpOFzjMrRXpbWv\\\\\\\/xfrtdU1fBN7NTc2S+Tq6mo5tGC+mNEQ\\\\\\\/X56O9BlS2v8KC+tq2e8g+hEvvUl1TVR4pHi88qviipF7ZIrLK1AO\\\\\\\/KWyZKZvwjCmRoTNGvyMP25jQGXcHWo0hzbEhAklinoefzNdNVyE+Z8quaZpiE4ZXTHusPjs7PI\\\\\\\/dBfrVp0BID60XZPodxVZx8KdAOFLJe+MtUZOI\\\\\\\/2hurXR0tNMN0gOtXQqxunBBMqyQdOwhoD3m8tBNb8j1yQH3EVJMdbd6Tl3eHWP3FrTFBl9cB8J4rDzLuaj2WgknSSjp7aZskhn9qlc0GNFSSRSTHMlBUxoQhNDwYUrykVgYA4EzBfaHrk1E+pu4OejRLK9ye+uWwAIJgbiyMcYGDEsqqWZ23tjVQYE0iOqluNwQOGJypGIytvHpWkMODOUR5gOXyvj036cCCpx0bPfm+We4diNw1qvrdwXN48jPbD29+zErPHzUU9pThm7qv1i3FVQVPeVmuuTgDSO3LdTcczrJ6JksC+KoX\\\\\\\/0IkfJsIASDId1WX3GW40zjIEseWqOojgfqeHWzTzA1qQAVfNTCZKkBshhSjrghNcc31PL3D+KaoqBsFu4IVTPuxMmcQeRsxWS2m8DEo9F02BuuOdwFCOsjoJcZcBuOHjlavnxVXq1bwBK6vCGjlStgL+r8tkpZJL9S0C\\\\\\\/bXMk53GpTXgDSy\\\\\\\/CtbQQVw6UzM6UASfS4bHhU53vmP6OcAH1suOsjw6I4\\\\\\\/+B0KAZDqmEBWTcp3axWybBTn4sJ+tYkVr5BN3mkSP7SfDANEe+elebZBtb58icIgfC8GwnP4LVRE06VXbr1YVr6mInD+4YQ0yL85EbcUQ32kF3nva7ARTMuvaUQKYSnwZXRSByo6t6amoXo0M6O8HQLtd1QGIQSAbwIgGyv3k1SLjiFia3JIb5L6W+BRo65Gkj77JcX8zTmBFEKEIY+feEe\\\\\\\/1NxzTkFbM2Q+8G8p4AmhXfkNENicwniFCX2zO1KvGkOCQ+vefNN8Ikqdh2lmFxKqtLyJfB+JSdb3PoC5oOi6uAVA\\\\\\\/yFLNxqXbb7MwIg+rYelR6kYJEMtj8mjNwxRoAxcbxFVo5mmxdp6HplgAFQZORKruEWK9gpssyTdT9fEhoR2bVfuUKT2cZle+ELsNcLHPOgZ6SJDmOoARg3KBF4f+JeW6EpU+qN3E\\\\\\\/99qLU3FSL5g+fX5iVIncGzL2XjMueNslREambw07yinxj3S0UR6JC8clXGElTihY\\\\\\\/2qjg4FtHMkhIGzwm+bOHw4oI40ETv5oraAAofGEydlZj\\\\\\\/Ji5vhlgj9JqlwAEiZZLUuO68\\\\\\\/EGGXN7IbkZiqCgQAfX9p+\\\\\\\/9icvcYD3xKbODpXUBlt3QgTrZKrTk\\\\\\\/z4UiTBWhUV1sMoa2\\\\\\\/izqqTm\\\\\\\/M41jzSCxPkfutcgqg3vDf1MS9qAek9eiT\\\\\\\/t1\\\\\\\/BzZjshCskktKS2O5SLn9zrpm5+BX4c3FOo\\\\\\\/bglr7gn8pjfn7HvCVXZVqwqcR9z3Bx0Hc0T9b+ye5w3nQLdoyq6dk1JGS3ybVipakWOQzI2sX9RuQ71ppbW\\\\\\\/ZkvX8673dH3PPOZ7ZnBGq+jyKqTy+nmvhwtCS5iq9yPJvXBT15YFvN+O7Q\\\\\\\/fPmCmbDE2cvZz0sQNIH9cOa5XNH24wS3TpCnY3bJvDffDVdxrxR35EnwvGgh2Rb4kN3dJ\\\\\\\/RC\\\\\\\/D290twcaek7Uyo1Zj1UuAxQ2zZpQLqxyX8HYZO5qUjHcOPViZ1oZP4g9OIZB83RsHxJY+46mlMMB+dZGpbLPEukkrJG2q6A8ROrQQK9j+Z0tVzrCC5OzOtA3ubnVu2miiKEM7RDwbL7vbNbT\\\\\\\/k+CDC\\\\\\\/9H3Wc3d18kAtu53GJ\\\\\\\/ZpZE3uDfn\\\\\\\/QBqxkfkohP2wu3kRptnSHTznbcOZUd9M1mYwDUig5qc4wb\\\\\\\/AE4dNBRFIsLM2WYfRn2NsYloRxNrxkBavdVWtcRwXGqDoDUp7+K\\\\\\\/SVkO1LbJbImlz2kG7ac3or\\\\\\\/c3NXbqcNcyHC5KyZz53qLt91ecM\\\\\\\/dkSrPSGzU\\\\\\\/qU3Ryccbstk\\\\\\\/l33xUrthSVurBuNJTTuCd0LROMaddc3VVXxzz5ZmFQ7kXqNbh+68HOk6cHHR0Fl16xSwQsbk+jFOVJ+p+6waqCCGh0FVdsIydfdLoVMh3f6K0ckxRsMmdR80nwTlQv4AXr4jA9koe87oWsFa0LHn5uyzArue6bpqxd5XsBzujVj0Urci42rmzZkvl4Qq8peuCtsOMjiInb8jwjYvbsRvSONeKpqC\\\\\\\/lXJjIxtip4bOKWDxtHrWipfo7EocfukV\\\\\\\/VnqnAtKoTFS9hK6AfSUlLgjx7861R2wsl2SBTmtWLmvKORpMIujsSeDsvQyTaQwobRjUHa18TGK3\\\\\\\/k3Gb2xc9DhTqLqVqHee8V3cOHIz8ggo4byU+bEhnrOghU0xATp0xi47my+3UKqP1qS5MigEqsb\\\\\\\/T9SOOFcyVk0tHk6DYohioTqxnhqxGxJEiP51CXmBfYb6R+Z39xy+sgSa0QHk2lpVBcHmzI42Kd160jpmXzMR5rGUR\\\\\\\/vOyawlhbaI5+uR\\\\\\\/HYx9J2kG+Eo\\\\\\\/XDDU37buBLb9\\\\\\\/+jFz6U+A8Bz4rhQdwCAoiZAp6PczDHxxVE\\\\\\\/Cjhi8xJR\\\\\\\/uUbcTvVDOqe0yHjvg3gFhZ6XelI6L3oobYc9cHB6yIOvEibel2T48ybIqSrM3nYlTeHM3833zJH4iaQm4AQW1hJCQU3JojwKSZskMh6GSL+tfGNFkfTlplYCRMBYiLONQ5NuCLNwWm5dX8BKFQN4ar8jfhVbEvvZzSBqmDcf7Ubcczs4onJXOH8o8RN4Gc1NH2K+TFolQCkqS5YaqRQ46txjoDB8OMl6+V8UqOwUwz4NCwbAtBaMFjultjOgFyv6mURKAk6u89Y6FsehAtJ3HWR8mKxO\\\\\\\/Scx43L+qbNMZ44dlq5Rn7PpDI6fbNS2mTKCs0J6HFuicbBVGltGB+\\\\\\\/v19FsdUKTeUxh7mHn5KMQpGEeczhMyID9MVzYzkYl\\\\\\\/hn1O\\\\\\\/F9ODB8vHIjfB6Bf95cthtKnmxXb8+pxK6+K95hZqj+mp8\\\\\\\/Fa4lxPT81P4FbqpN4adTMhSl7BxWOLbxtEUj+laOjDI\\\\\\\/4sTU2Yj6yebq0z8b4AEemEgROFXltszk+9WrcobLAfBg5KW9pmqaXZX5d7PHLptlTIMGzsmv2WxaG6ell0H2p0hA9DbE+lyWKJXVcChfQDmOun3jmCru\\\\\\\/GqmQvLftoVC60arbs1\\\\\\\/1a6nQN0lBx4mneBvOEotP8gvmwcfZmTFuriHxEKGb8RAXw2tPh4\\\\\\\/ZnjgSlMeVQhPV2IomnQoUz\\\\\\\/KZLNu01v7bPQ3tbQCztB8gczN7+cUqBOmzpxZdVBxe\\\\\\\/NowlrKkuOvx2BRGG7J11o5WrU2dZ26FgsWGH\\\\\\\/Aifaee5Cbqw4nDeL0n9eujOeZyIahYRCMmXqPtPkHDggFlsza2ssDEPV9iw5i1656k5HLwyuZmPIZhGkN6dAtGviGrNRW2HRoEywPs1UEwy1+hrFMFQ+xf\\\\\\\/\\\\\\\/pPIvf2BA6JA1lP5le4HHz++ytzMqwQsNsSdrxWOoMs9jk8TJmv07Gpm0Z53NYpyM58YJCpdwluXTJStq0uxD7IrNHxRgWEhyJZBfLONFtI1BoqHIdserXXwxJwnQRAdp2fNqMlkcNg9cTsgEsvGtaFUoWuCLvoomBAAk6fXrnkJMrXbmVvtfkVBUSxH78Xr+RS38egyPjTJAHq0fopRGBsJd8pPEr+In7BLh1buidqPL+5e7zvmwWnph9FpqnG+d5TAPzW4z8SWwzxJImbEdvxQ9rksb9+FsaZSb1UoXNnYZwLt2g2Z8G15ggpHNXEp2uz1imDivFuLhnjC1q9NL84Fo9MQ8gQHgR4G4W+6WiY+O6mK8kuGCiJtze9Ayn+4l\\\\\\\/AFPIG\\\\\\\/th1YVy++lCKfibEKO5OO+vfq6bqctf44fXOK\\\\\\\/a1xS6F7a3vayH0RSTWUOlaQAHjXhh6eXv7qWPQMKlHhXJfGkivhGFTUBcpWnZRnQYx30S7Pdhze0hDir0w6LpfDLvEjhardy5TTUixfuq0g8SAAvx21bZvQHEN809wki5GYS1t6f+sGa0z5\\\\\\\/kbKYQJcpFiZIup1Ztms4UpLGmUhPUBVKc8gWUiyeeQ5dxuHKzm\\\\\\\/WdzRkJ2HMFWSSvyK3dTqPt+\\\\\\\/DvincsebLGIis+y2gSxrAjUuuxz8mr5QDhypsVsjaElBsXJ+gvSH7rlU1VfCV90DXRu1FnRAdFVd+rBgoEMrRbSwHM2c99GLft\\\\\\\/WNh9tLQS9T59AROXqEbWP99yw2tpHx1z5hYoVHq8lafYnphDS7ammeCmHu5StSK84MBiShJ9QBJ\\\\\\\/4uu6lWCIWvhLxl0icDxuIKwB6H0K0x1IBH0KCtzbPTWr9qdMQ76S23eTRIn0bM3ldYcko4Ui3mcaY7QLe4WQzkqFXLJF2KVMIL19HX9MVCcGWg\\\\\\\/UVTlpfY33bk4stNa8y5tvDPWNHf5ZFWbxXv9wANEYiHovcJ420S9p9VywiGbtGKceHm8JjLfLKl7T5lc8dxc2hqcICiciWabMDh7LVEHeEoLKpVVq6gnbhRA8srElhxkkSJx0DojeHQRRnyRVk22R4xc30K1q\\\\\\\/wmUVPKStRDidsjt5gf1vPK+0dJJGM4N503Vd9DIVcreo+3xkGvw0vQeyHjgVMjATkW2r7wxAywRKNjDQWcVrKA642kSMvbK5Dkex72xNn4rGM29WgzGqHfvz6PrMOzNzQ4jj0LcLlMp7OmjcQF8gtR6gUYKvzrcEQgU2EbNNSZj\\\\\\\/TkZoPWwdR90e7txKRoh4j1od\\\\\\\/u2WrJjN0yZE88jyhCK7spaqrtiEs1YkRYn288SRtP2MozbCWMjQm1csXGZDt7YyyWUr56glO6ASw\\\\\\\/Hfgh2tQvDlD42CaS82QsmfQ5UesATtkg2sGCM\\\\\\\/qW386J62YYXRlNPrCR+Al5ezlff7\\\\\\\/sTkXR1xIxWAOxw9cONp5M8soLlTpjXtU+2wyvL2Gc8gN2D+zGwhUwOCgi+C2hkev701R+JF\\\\\\\/tVabgtHOcUZcjjjPuvRSCaBRpGtrB4pG3pBkscqJJPNZC\\\\\\\/XcdY1BUrYkysQGNK6FFBkI4KcuTKhaXoQcMsLuh0qjisM3N\\\\\\\/YJxqgymKINUAkZDXibMT7gsQa5C9CvBpVCkl8dpRxbW\\\\\\\/K9FHHRyRM4RjXVnkiHEW0EsFzOcV3GkYfNdMf6ObbPsmyvvlRVZSUkYd08PWMRv5fusmKMhsNgm6xMkclKCeMJv0iSuGy632K1GOndR7xa1hLQm7\\\\\\\/N49GWrXE7rQOAdCIKskMSeRO8VeZQ4SImVY\\\\\\\/kYMVIemd004Gn+XEFXtT5SqaohiB\\\\\\\/lXoQdkPaoRLKKyYPXrRfwoH\\\\\\\/XC0nrcTHAr9nJ+a22HzBB00tupgbnV03ZZ15B33BNeO6CEAgELwGKFBnm0SbUG7xS+PzWK9YJ9DzX5nEQsC+8WCjccttYM9pNLkjkkei0IFfU6Ywebv8CPjzQ6bIO4pN20aS2mC69jdp33HNIU3TvCAi6ostKT9OiLYULLrE2MwwdlDijrLhk9QLJ2HwQG\\\\\\\/7OnqVeol973dylAVT7qr9I8yOtljOM5Ziw1HIswCcjcVam4TJe7S7Op1jCYVGJ1Xyu6+LfCKeIPs8soe3WsAAx4wlekkY2hTOC\\\\\\\/ycJT5CsteU2wY5LrKja+B5PWRmnZy0fNbXIDq6wwNqenj1lV5AN078Dn38BJxTtCOfDcCM39MiQ53CrcU1mY7l3XThSSC3PefHZVmIz5PMDnx+VT6+G40miC6nQb\\\\\\\/LVCJZ16X88rFYWtWqIeZD+bydFxmy371oyyugpeul0NIKyobaeKlGYNKpknzs81TOI8iX86vJ\\\\\\\/07IZ0df528EPSYcB5AIZy1HYymC8bsahdwdXmtnwpoZrgHgkIAlJzky9TAp72TvNQmfELzGVc6dSoG\\\\\\\/+QwdsoHR0I3BxdZNMfHx\\\\\\\/NRmGKUX7uyUCU1ocYysI5UoKNZivztSd1eEvgzJv6mCCUxKoSGnxmAWbLa0u9G5nzmlsvrrtD3C+XxtFMqKoGKogarJ2S8yIj2Hs1M42vjyfMWsBnjG4VxLkNiG2s+r6uE+bPzhqkoDIJY24QWgmbujBjKucL3RZno7trEVF5O3mOjSDZjJpwmoXhXwGJEjBMwlU+wZgEJMG6Di539aH7s9hblHTuuXAZ5dFfpdhZYIJ5E29pSn1oWgfILtRL0zsxOvuFvDgA535cToVSvKsWjmBZUHjxWbQdwJT43MJqfI10mPmY+i8U3Z\\\\\\\/vIXV9rSf5ht9+nehO56LFKagc9J2FmU1IBhGPuSOQ18GYb1fHCq3FXhfzA+YRSIfgRxdCPiOOMz9IMeEGBAeZp28VDVIARiTBw7l09Ao1LwsFSN0Tk9a5OZSGA\\\\\\\/w6ISPyhuCuTz9RPz7DIljvdSGvI0k4kBIQ3ERf8ez8JVLQclDG\\\\\\\/djUT8QMdzv4+mdvZBqVi1EmHzY73qRbYkWnuCP2Pv0iVn9\\\\\\\/QXCJvvL1WLFd3F0+5lwzyUm59du6gJZF1JkT2wlMBH1JPzjv7F1g1KWIbg042ucy5lPVqc\\\\\\\/vHfVcb8GFwQMmqGYDW7BTL8r86\\\\\\\/km5JJiaevN\\\\\\\/Kj746S4T0KCRhnVDUpDE0vsXxxbo2c27az\\\\\\\/jMa+GO95uravN3XWe1PK5tpmP3KAzj3KcrK3FLhZrs6gQq58eYzfjkQb0DvfDvEqBXCqzw3c1DN0rMHQD\\\\\\\/\\\\\\\/QILpbSJa06hXCAFN6ZDB0k+puB6FxQ5iH+\\\\\\\/WrK9l7\\\\\\\/hztR8OZcII2zW1aQarCFpVPrtGZuGMIJ8aJMe8riSZnjS6cosoxwq8a2GaLh+oLfvtmS+moBRruvzQVADH+Y5a9OXce22NoZwspWAAMwdevH7aoUD3pVnTLf9Ogr4pNEt6x26bHjWMjRs\\\\\\\/A1ZGtqS0j2hG\\\\\\\/8krVf1C0KXC2W9tVj5f6o94fnt9r3wtOkkOBDIwlzxi1kkw3+yNC9WFsGFTatiuJOYEcGv6yys82FKZQsHgpKR8T7CwFKHxkj3YK+Gz7EQkwayQir9cid6g4HlcZj1rD5KoJRyAl3zXFdCHNBM1g8WvpLB1p7t1LW9Zmy7fE5knc40YuWiOkCLBhy4eok3uFbCjtXMN1fJZym4mSDtMjG7Ke8BCXwiQGDF4fIxGPCyvo8If+rgCucmgBqX+YquHCrF82xEAvAxmmjOxyvjKbqo+SMVMTdaJQ2fpMBvya\\\\\\\/IjCsR1TTTg3P9e0dBojKGWaQP\\\\\\\/kbZQXoap1ydBZKQmyDcEtENYrFSgGKAy+nuPor98iPtJmWiPuP0dNJs3PiHnLaMqb47xHwtbZAhw0oFBRbvTKCnEFpuB7vJGeujGf5+3s7MXRzAYJ4\\\\\\\/eJ72FwUvXHm9Hk4bH4tYvQWMyDZnXUA97HSuHPe2QvH+QtfAm9h3iqZUSUVNb+v\\\\\\\/OndnWQJTB+oypxrB2pVODcph682OlpUEjmlCxPsoLlYCDolIhZbvmlFi1VrDV\\\\\\\/\\\\\\\/HbODIjmAQ0NuKZgFOX4mKVkqBwXAuiaBEx1xD+YxBhqVfgc5HUmgKpETJTGWXxcWXGTAZePQzDTJNipkG7DlCWV7z30tvERIYoO+sABQrRZxqd\\\\\\\/iLJ7NR\\\\\\\/rq9N26gcckg51g+Ugtyp7xEA6PX\\\\\\\/5WY2TKQ9w2PTLMyFuZl3G+TityKRW+uAM0X9FQ2ejKMYHY+nFJtEZPn1KW161YuTL6ZZZ\\\\\\\/YQCdBCBt3Ue0KqNcIQhAQV\\\\\\\/WAAbPSADIZu3v0IZsRpaajEWU5beodDWbLPJDoroFch3QUthaQnfdujSnmr+a\\\\\\\/mPinLJHzkuZSbwSTSOZQ2Awfrl3RAWXMyrxc70Y1TxsWI1FrO\\\\\\\/N+MOkjQurMlks\\\\\\\/blEOW1ocWrVnIF9HOpVD2H0NYaT339tXRXOikIqHRrpoEd2Ud+X05ZIYbg7AP\\\\\\\/G+BWKFNt8QqXLoMIpl8xh7saMFob1CNqXP\\\\\\\/dXIT02QILeMwFoxOQkJ8mruvxHoEHlE36t2FatZjIuKuARuT95cOKZ\\\\\\\/S9fWhetRm+5KpUc\\\\\\\/vJBgTSkgdnQ2kFMFKHtSh6sIhuxpC4G65+u9rx0jmIyuHiG9bu20TK0IxB69D9NL4LoA3AW\\\\\\\/nCZZj5XwL8xwcJJGhAfLz7ktQy+ehD6s+S0ZsZqJsppLkaigiHLjz4W9u5AbOsP+Bm795bVftvuEuq0S7GHvJTkwFO\\\\\\\/XeDNqsIEZCSP7mzjWxM+IcKE5xtaGwiy3jnW1WOJ9E+XI7DD3dqPHyvF8oVKqi5lDuoTOvjcEE6yNJeymVI\\\\\\\/cUnf7MiImDqlWt5F\\\\\\\/uPEpTLvaBfyBIkogH\\\\\\\/Aul4MppKWj7rihb+5YRQ5fpLuIiQqCqL6MTvbyql9k4MaFy980m33RsBXUr8aMUo+S4mK2R6ciE88s80WSEVJ4Fb34omq\\\\\\\/jscM2OD9Hr75ninanO+T5LptANcFk65spG0lXNLsEnYN0Qw4r40C0Bs2ckoWzcyX+rAZ0BpD\\\\\\\/8GvXpzcwjzshueElF9xAFf0xlfwsi8EdWtXhNrakeCSyXxpYcHv3KHVgy8E19A4OAsLid3ozYq6lb8Id4XZabx72SXJAHGgDvbs5XVU8XKqwa86ODtzzEklGxwjGnEAOSFLz0xKaxIcSE+vW2u1spVFS6usK8xpQoAveJu8dt3uibewDXYhdg9XwfUEQYB6FxqV9BcC5IKp0FQfebmNRpmeOlqFbiCDSOmzS23zXXZxnLQthtPMszzswtL9+COpuvPIUW8yYuLy2d8QuKXP9TImcLmpKywr0AGFuxdjJZf4BP8fjvLCNuta\\\\\\\/zQmRJXrwUFOT+yVwhWzaMcY1e3xp75JBe6T104BFWTpJfHKUvdTL2o6m\\\\\\\/9+9wHVdqZvU61H6BEeF51oBO8PyRGHeLjcFRyHw79iKBOQsx2E7XAoOU85NmqQ+QXGA5KsczsgxSa5U3HTKjsv3g8SAJMtzcm7chd5fS0Enj2Nw5ITP4F7k1wUvmJGqY3grdacWFQqKqNKWl61bSLqJEM8baqUXa4A7VbUMiHhZ9\\\\\\\/2w1rypc2UvDGQMcyY6p8szer0k6YaCK7eBlDqYAN8\\\\\\\/Kw6ZWyLVOf1D8Ji01rfVW+blgEjV61I6sYhs157YY1KdRjDNdoGfwRJPg7cddtcPR9AtYqsCj8pWs3oa0g0k7FRIlcphHUbOoRlTTYqeXZ50NTDw8+fhWwpJmbgHvF79fjqdRDxz\\\\\\\/s+qhO3+0O9q+8azclTo5rf9InNYdy3NLXslHcGouTWlFkqVMkunPzfoDFubD9Omd+NMbCODjlgjtRejat8zLdg8IGsKV\\\\\\\/9PqwwjKHpm7ivjjQq9bgDjpIb4PuBNr72CSGBCW9coOjC53+VTSPiB0wK7P0My5WWkpbyWSLhwbNFUV9zo4Bqp5zqUUVoh71ABxc6N\\\\\\\/hJccRAwx4kWrQyVZbCttXxy9UF\\\\\\\/KMEnaIM9\\\\\\\/rz6uivDCnSO++\\\\\\\/eM6Ak6oDpCM\\\\\\\/NMLLYUB3V\\\\\\\/aeM6+1NNuE6RJzmlt4OIYRdTomGzzm\\\\\\\/CI426qBAFTySghvr0lXJndPTdonT2XQq4OvBnalhLYgO58jcmAYeRAYYml6JjssSI+ukHKTaOImNVMxsub0E\\\\\\\/IqUHOETv5nyDZH7a9afG3ZGQ8cobbQHS0V3Ai\\\\\\\/mT4bao5GoqK+ytU8RdDG\\\\\\\/UZyZsI9c4boomN7f7s9dI6S+Pmj498lz6bIzZNewkctOvWJTq+RG1BQZaVaHKiWFMlKUs1qTeeNhrw0hf9wjS3XQqU2cOvm3k3tFdKGkwknu548fCngVykI69vrySya6Y+AyEwC6Jd4xcDVjBMc\\\\\\\/C0QlxCtr\\\\\\\/s2g7GnGCz87Y0g7WOJW9jSfaHJBTA4RJ8cwbKujeI\\\\\\\/LVmJRFsR+kJL0yqNP+LWNxGpGYqAegkeKHyrinXzWFKyKQZfAoD056ibqmI7Du6+K9oAEYcK9ZiOQYOGCedqD7ZXSoxjBxKXR0oXTKvLuMZPtNs7tMPjSJ+Oqo\\\\\\\/Wg6ypMNeVQJ3E7eTrwfsBRKHfZyjAS81kKsbMpiZvS+8i04vkhdJCg\\\\\\\/yT7+\\\\\\\/ECsXDq0GuZchYcbKxPIthbjtEWYbPQ0+GpB5pnzu9aAkH2\\\\\\\/t6yvD4wCrw1IgkqtBBlzVMJ3\\\\\\\/mK5a4GpE2rIGp\\\\\\\/l62knczk7oZqoiTyKyUdHn\\\\\\\/Rq0nABBbSRRMn5+rn47hcKNALnyA1fkj926J5027Zbd2LxSH4TwChfLFGYRSaQ2S6sRa+XNLewXrSQO4aWlk3iew6dUUSKyXhYvHH0OxdND7jw9TSw56447mSud6tc5bZodedlgnbCynRtGSLmdkZqIQNJ1nq5lJK7ykRd0HEEliI1VQ\\\\\\\/NnImhA5LXd02Kj6ClOABuydra68HrwA4MC9Nu\\\\\\\/tfT92gzpXGsRm90XtL9Wfw7CcPl6\\\\\\\/7i+3qjk8uWNFmFgBeu21ZEAs\\\\\\\/A49satuT8eYh76Rp8uBbfLK0854+ndfp3baldJ4GySLM2Nqf+vIiVt8kqd+vqWV6na8RDS46iQfZFT0p+ngJm4bmO87I5sogVPx50zRP6PrYlIcMShgl3fmqCMvPsPD4tYVRnQEeepQXebNEXZIKmZ0kJk\\\\\\\/3rUlPTr+774n+Qd7LFn8xRpszo6jAgqmfc6XScIzBj61vUITBz+cfOD7SHrqB8HKMI6Zd3YN8uBSOncYOHpDePCjZyOjQL9zpaSYINNSO\\\\\\\/hn+piscStzzSzrDSaE5nZFrtZpARt3MG8ZR1IDxgt35xDtTy9D\\\\\\\/T7kY6bVBjjSTsLJI4w3qJl6hZlRB+akFnum6NEMfVPdC4Q+4QLO50HaOdo3GQAAAwQJ+TcNNwKAf2t+Cql5uBsbu7+GIFp\\\\\\\/e7fjgh+oBdbiO1j6Foz18FolhJYIfeiRbdZSQoiQ7zkr2HdA3KIi6N+jkbO3LxcIvh6BILpG\\\\\\\/BIRmn2tK9KEFOyPg5BiqSUAXmSyNw03iLioB\\\\\\\/iL14uTMQ1yqsNEtaUw1BPZmdPpxIr9+fdqD9sCjCz4PsGb5zcdGG96JaMYHX\\\\\\\/pjbUyT94PZ35bK6tK38bBXNM9q3luqdx0FC7Va00FVYGtAznZ\\\\\\\/UTBPrxGPB1Iy3TEXyYhwfULECqAKhYKAuyxSbjX+NXmt9i6uyZrhFEjuMQQIvpX7ZfRQPdBusyOd9wAQfr3Oc4EWzVTwdEuxWR14NDNLDSBxztUf2rIkzXniLxZuXM4sD9gcXpShum+QJVUHyZxugEhEX+jF1eawsLqqFN+G6qT32thZZGEiKhOwtenNhPQzsUK\\\\\\\/ZPx0DdVP27gjqTHdJ6ilvrbELfHAlHHLD34LrazHV+PUIfoj3XjqUInXQb27W3S7L2sD6LnkbG63Z\\\\\\\/B\\\\\\\/6FarI9XKra1fNejKA1fGcLgVxIDA\\\\\\\/sKUJ5jRPjdx1o2upf5HoHtu+XD6oZSpGtQJ8JMktH\\\\\\\/LLbPaezH7Yn604h7LqD5+EEyv4dV2snm2HRwyvP7P7KkjjRmlMiimgSJk5QBO6Q5oQhd74sFTNi4k4HVh8zHq0Qr7LN0JwZO\\\\\\\/VgdiHUNSIbz0ZPQ+UJ+tPRgwkpw\\\\\\\/xmGhH9his+fv28WMav0+NUpLHFMx9YnorcTv+zPJv1qHE0b\\\\\\\/sxUiTKSgQU+dYz4txohACeoloBypSoCe2c6UlpwMbMdSD7K0APVplXnn9ZOIAQaToqGMKxJPut4A4ZKC7PNCnzydOlwLtbKNGw7gjw3vj6VC3XJU1d7oM5pPlx58Br+oNnPPu3G\\\\\\\/cRT82ZGhqZKugTFgunTvlPrZqjwMb6Q92kFT5mm4u4qS\\\\\\\/r+syXsBStwbWdbeuTDi3IH+0VXWfT46mYZdpWPf0Ud+7w93RA2maxcxOnBakmAv6MeNR7X2iCGpYrctwBXRmrJ4U1NlIYVFVfZLIuInX4ke\\\\\\\/HmJYWA0hm+ORpfgB3ZjCpiKIuPKndDCsMYI7WcWDoegL\\\\\\\/gDk60qRTg6qv5XZarGCkoE36iRat07tITkUpfjVE2ZG\\\\\\\/zZY\\\\\\\/5CV1Py8vY5xE+ge4ioSkIQ2CA9Tanncx8eZbvHBAJ2ncG1tR0FSDDvIV+u58CRU+fez4LwjYMFLiGy330hKb9Y02xrXnQtS1PdFsdSoZ\\\\\\\/mdKv0uk1HDTjqnXgR6U2H7Wpb8eEkj3m+A6tTp61PCAFg1X4NlK7X9mR41olgkffhm7lZGMuqQL7B\\\\\\\/QXn\\\\\\\/j3+z\\\\\\\/jZMOpdbfWMjMWzeZSvFSxli3X45HD06uBNZTumyEobxkHkBsQuJtBKHes4YIxKfLx+tdUza1U1Ujwd31XiXYCi1xhBnFSpddEnalqfXDkM\\\\\\\/y9O6ROLTT6yaPf3K3uTejS0ACX7hb0yTAJ1aY2vdhJoCtnl531avv7J7ZbnG7oFKQFgyu2PEAauy4hq5O78hKsADuEM+tIDQ1+m8zg9gCRWmn6NuaXB1\\\\\\\/mcoiTF6hlcJs6ZYPVAHSJwh0GNmIP50IxdYgOtlk1Mfx71DBOaF8EW8zXBDfdbRhBArBtcCV17Z\\\\\\\/0olV0hUdHsaWjzhjhL6CRSGtYlHkCfwDEtF4YUUqq8KN92a4Z13vBzMmmKq8CShwa3fePh26PsBSflSJkcdag4KglnzS+geEcSiFxC2xA0sYL4\\\\\\\/H7u\\\\\\\/6Jb9IBQYekCFF4XV15BIvVwJ3IEDZJE42mVhbSH21ybT7wKPkNiHAyrlUfTFUXiz4lLSvwYufC7V3DR2WtOla\\\\\\\/OHAYfYOOiQ\\\\\\\/yhCkLej8PZ390GANIpbpBjJ6rYn6zZWa+XpSFm4d8g\\\\\\\/fMFPvcgFVYdC8zC7xAhrvm9mfx4gOt7sxBTd70XifS+oOkTJTLbI916lI9KoPyyL7V3sZJWRt1RIUckrADQ05P1UCDL8rs14iGfvD6jQ0J6Hl4qfxbTWYhqxov6CGp+6qWa19h+tf3bEV1syqaJ+Trm3+VZlPz3Q0XTXLD20S1jsgryyw4RrGdI2iZgQ6fuIy4TKVrRIImePpUdV9nyxkVgW2xf4cb5nRGZEKlWZ6AH6OTbGvcAvQCofx8y7w8L\\\\\\\/z9QxZlebJH0UF2UzCsGB\\\\\\\/DUc6f6anVwn8MUI3YRDhtxZ5XMfSvdAz0YibNXhJsMlcaezBhYl5MUFq05b+TLw\\\\\\\/I6hAcn5rQjGOyARtF9AEOK7\\\\\\\/QFXJfzAGm37tiVUXOXCSbfgKcJLklTunSNgCZVbezSOPQWLazT3mDefOKnw81tUJk2q1NzfglLNi7EWXc+HHWONEQ+f558RIF6QtucwTfgElV7HHamDSgZ9ZSugJg6f7HPl8C7VFp\\\\\\\/RhaQfUt0EI5xyy\\\\\\\/gGhP7NavUeDe73+Sw71cZng4FMsFJ1\\\\\\\/\\\\\\\/h+uZ4tPFYpzu6AqfvJF3L7W\\\\\\\/QHVWjOQ0PWOSMcjTLvm4oS9Ms\\\\\\\/7Fy2FGoyHPbx7fBfGVPq8FOWAFxCjKToh+CtKp6Zc84e5aYbfOJIiI8BgMhtOIF3YYTyJ6myp1t5QYDBqY6ITmUM709Wqnu+9nPpNzu7KKiH02ZyDAEEhkBwAMoyfJmgLJaTdT9MZ\\\\\\\/OefR02tTVFgE2UYX7wtH2n\\\\\\\/ZJXThbjJF2lye5PPRV7vC4LTN2Y8uSM585zm8cQGEWDFBIEVf6a3y7uRLmRt3z3nMk9UQacOxeOklcAR4jl639raTCz5UmkIJMDYFMlKx2mZg62QYh4wvbAUvbdxFbcHkgsB3YYn7AeGWgzYcxFOa3dahNzZSFVWKeJ5XQ0PpDtzarQdoe8BPHPAS\\\\\\\/tt7yrEVAR3AQde6BnhOfK4cilB2mJYcRZjO8DxOFauFjJ+nprMXkLIDv1lHVPdhpXDPcgL22y6eq99tImp5w+\\\\\\\/ZnkkTkGmte0adXFTvGDhaCQMaSIS3WcIOSEkCrFkyNEe336H7YNcS+X8Bs7Wv7Lhh\\\\\\\/9hilc2bXBj7gHChCv3jMr48M\\\\\\\/obuNWOUzUKsFG9jviPknFp6\\\\\\\/fVxuoSWIP+FpvutB9a6Xrm0N4JRqb8yhC9+CHULayTDAi8pOI2gX0XUqehsz\\\\\\\/BboYoM+k0TUSHIDjF6r4HT7NFExfjUcGZBcGPpW350KgShAX9A5SP7+nO51oGxYSW\\\\\\\/AQ0P6KJeZxU5APV9rs9CKwUQ1ICSuHLrzBy3RNN9FKzRDGOSRAreqDQFe2jUFFEPOumzCVxYtWNPGgM7dYZZf6m7kboDgBIPSmZkZygeM6UhLGGhf4l3m2t0fAsTBe36Jebj6hAHh0vppVjQjJtBC85xMeA+5iRJMFBJmW8QWSRCZlDae35APsscsH2m7e7FI7j28npJCVGWpQP0nXIKQhiE60K8pJY7s8774z35WsRr\\\\\\\/zhcDpPW979NSgj\\\\\\\/HsbXaEkAwfEqkf\\\\\\\/ziG5tJ7r\\\\\\\/UG8AVnrf3MXm3L4G1Fr0Rpadl7BRg++FM7QjTizitt15b0gxxDlU9mTTeVWuiiu1QUwu8UUYHhuiCT+7V6u9C+U9vVZJ1vbknjTZtDSfcxqCN9n2jox5TNpNH6mIN+XbduVhLlfcgNgaS5orbTfOdavWjnwZnd410p78iLN5UZEm7nK1OVSRfhX6RldcmXLs5Fjl\\\\\\\/BFoBHf82w89Rwri\\\\\\\/4oMOvPh1FPbMhoWscKzN5owmok+rY0MyhT9XrL0mpt10ck6yk9HNmvOKGVJx3yhspYxs8qGzth68iOHiYgW+Sjh9LxgsozcGr7Cp7MEJek0wWzSRWfCL9fUakgMAi5Ydb1PnHr0TisUIfPfFqmlT508bkFyJV61FdPdLdVEh7MLTRTUPkKjpvB4c2wASrhEdgziKQ9Gk903SMbUEvFvjyRcTMsgDUC04bNJXsZE2PbQq4GUcCoR2TMjbhHt9t5AJYKh8bVDaPRWBMIBmMUbd6qIDxYriD1jJkuwmOhW3uf7Ep+2kH2rCs0WfFa6lRN1arArwW5VmoSLUVXMPDLQM2xMzQzgoipj4Z4gnqNbI0DN0DXH2sMR4ryVNnzYG2AEMZDyCuZLvdRC9SSNQMSxcn0PzLgUBfQ4MBTF+EkpxyVthRlpI4aaHH6IGux5IT03oUIgsa4swKASMQXTMxEf5vYsr7btjpd9tTVLlJmsHV\\\\\\\/geMverSKiRMeiwuY+zAVcl2NIDFtWPyvMlhWK8ttyeSU\\\\\\\/2RWe4OzusZnqeaLu\\\\\\\/\\\\\\\/04xYdvzZpGBKAXz1vCaepv3bHjluZSLlwI6oO9pT5Mj4Dt21oWTAznwInrFS+9F8siLRdEHbUHD4Hvcci\\\\\\\/n5pGqPEqQWAz0FE41Dg1hGrhe+cLzoTmIVnC768BL6Z\\\\\\\/KgjxSEKe4wwVOq8ojXAXDy\\\\\\\/De7YZdLxpb++VTUxLHzhnImFeEpZbn0tRG4PHHxe+mX2SgrNd879OrCZF10sr9CMtdbUu1CFEWWP4qP5N6AsafntxYoM0RD57iKtstGmgzhMhnzvyVZtOqPEuQ9QhUP2YGfIeMStL1MU\\\\\\\/YniPq9s6c40uEin9lXhYUZcQDKg\\\\\\\/KeUHq4Ew1VSYx4hpJYOeC\\\\\\\/9Fx1tr\\\\\\\/zzNqONSv3\\\\\\\/XBj\\\\\\\/jP6QjOuiFCGSiKg5rGvhhohyEnDqzKHziBYtAHlVS7vIlNS1x3qElDeEcJ6H83CmqTOSx3AyA47ATA1zoFhIypH5D\\\\\\\/mVrl3xY1dirsFRiGIdjtIcCiETeH29YsU5nmmqqAE30ccssausn9T7+\\\\\\\/pZJi83l6pmiGiSXwGcEW0+\\\\\\\/ZciL2qu6n+S4VV4sFaTeLOvOFWQo+AGcL09ex8sBc6udZWsM7C766Yxcq8muf7v8QEBDWayRQ3FqjFyw3opVLp+Ak3JCiVt47XspMC6SdtlV602jmD\\\\\\\/e4FDvfK7lkblmKonnVFcYZE5KK1pW4Si+AJX8kVk9vF8lSsUrtuM3mq4oS3T0nW\\\\\\\/fDLZhJ2jxvCPjpqUCT8iooOs\\\\\\\/ySREu08vS53NxDLQ3m1tzT0HuD9sFk4L7PAoZfGEFsPmF4deSNSSlGlqjNgr7LgXRtGPxh+jQpOMXAmHeoD0O1FcGgvoCya28ZBgNTE9l9MYIUBtu0\\\\\\\/LY+ZD56onAKs6lxp9h5KPMwEWTBmlFtzkrzzvZcoDhX+kHOhcbRoCkSRQwz3zlnxy+d5rCbWnwvpz2NrlS7167wo4KJB8sV4FAGeymwUOIYVXH5kKJS\\\\\\\/lQIcOLfnyOG2fNVEtFMsZYsVXjvHkKRSVyhCEYSF7vxmAngUmYDu\\\\\\\/bv0dkYtP0WjWaa0A6xm2FMUWnObPZ0sZK0HROv6EyW\\\\\\\/6HuSoFac76rfPqPRJF+l\\\\\\\/GazVI3tBr5iPzD3YwH3Iar1\\\\\\\/mvGaSxgWAM+75pRQ81nDYre+umvL9Gk4XWa9rjL7V0IaOxEn3SNwj+FZDoCO6eF1GyD8mNvphYBjAqTuU5BZJ68qmBt8AW59YmS85GBBjWz3mTpL4IkvjhBo0SNKhXiERio\\\\\\\/rsLXnjpBRjxRCijLMcZVMgbMeoKUwgi6IziH0phBASgGK4eHMOBOckzTtPpgAV1CeCSgisiI2FoWq4wBVP304qepxwjyPa6L6YOx3E1mGjajolGj7VHCV5oQlZxYOtO5dJy101axlSk2zD1V82GpOQTS5romZ+hO2PJSsMS5UbefAZ5BHwAAN57DoYT\\\\\\\/JTFlHOPjmZf8SbkgvYlI3W6laba+tvTzqazq+4tlCAML8b5BLRqJOMo2IdcWU1E6CgG5G\\\\\\\/P3Gepvt2ac5vqoUVIeHKU9B8edHQpLHWkA8iVspMuR0RvwA5ZtoboJKesqSzA2pJUwSIA7in10hbLd+MMYDhCE7LYt6e6f\\\\\\\/S3Pn3mxLp2w0Zx77DoILg6\\\\\\\/cnfprmuuZ\\\\\\\/kXnegOFXqs4F8I5jEPQ4X2GlaWfP0PIVu3B6IFo+CTTCGyki9amQPiD3+DLbGQfdaFWtzSbweJ\\\\\\\/MeSfvxozAjZrqQNAT4iS\\\\\\\/C2e0gH2YBVerIySLbGttK+F+EZ7HeYoYI53jVGWMpk5Tn4RvQvYR\\\\\\\/B5NiIqBqG4e42sNRDUb8aDKqk+oEY5bSepMFS\\\\\\\/WgLdu+uHpgE631vl6b3LbPxFGjpVkf5ITHzdsM+dCv57ucrJvNcNMGyxvtGI28SHqhaSAJ6oaowW5QRU2CpDKdRFhQa51KkdMq\\\\\\\/7UaZjtTRHxm\\\\\\\/AokXNIkBrJPZGcmNSQd+6QC4vjdiGN2qdX2QlkUg7tUBonwJTBQIOyn934Ns4fYiBWrGus0o3w7VwkbZfF3b3\\\\\\\/4hL64QFru+ExEV\\\\\\\/6UcDk8c2t+a2b9CMN9pALEvRxKHHx84yP8ML8kpUgcRUf36H+4NFoR6MQ4X808x9bsMrtOMtr8gv0z+PS\\\\\\\/dDXHeFCFrU1acFp4RQOJj\\\\\\\/JRskfG2KZJ4W\\\\\\\/Ntc9KxUObX5VWM8fdiveZ78zuFpj7kz1\\\\\\\/qp5MaTKmJvu\\\\\\\/RYpT8tHUSnvR0xgpzVR6VWzqFVuXwXLazYq37TBmJ\\\\\\\/5nfJ4ykpAM3Pa1wR\\\\\\\/KI1MifbvyZMzV1\\\\\\\/lpzhPEcQfEeObexC4Tp9BdxLtrz+fqq2nl5TMoNGXfwxFgp2Vr6iYxQkYqlf4Z47Fg+KWNc1G3cB5VZB\\\\\\\/I3EdxVyY6NeBRwrqz7rpLYheOlKS5+nda6WBTjOkU\\\\\\\/Dvs+8QmBa5CVqcC9ftukeqL1VMRibu0S57KoOeRjls+lMAnu+wvqvlf6NnVPfOBG0a5b\\\\\\\/KS7fgydTwjAnbltI8DyAmO5Fo6fITSr2i0SEZmXkugN\\\\\\\/Zby59phRIGOyynadAd9iKxjdhk3TCKUeX8PWuvw3dOc8B6EZGlnMerPjBDwNdGhtdzlx0d+4VGBiWOzBEsf2cfyRDgDKezDJ8mqfdpPAIpYFRSUwMUMHi5vWhlm+bTnNzGekGOKXoefUXenwoPJKTAQhpOt6o4WnWqdP5gNrGdAc7lhb2UdbLXGV+PrzcuyHMxt7PAyzIkhEYDAqM0JCFSDluLYJ3LCkznjYbf9RflLkcEZufNfJw7vZ\\\\\\\/cSzqzEAs11p2ga0aj6JVev8b\\\\\\\/DaoTvge7BLDvVnRRL5VE4HzOTn15VeC0PLfKrZeiAV\\\\\\\/XMkEfU2NyLEEHdP2ctsImELAqs03fwptFQfBKdRp9uxrVUaIKeKoy2+rEkSKStUNIIsGlAwVxumOn\\\\\\\/DuzFbFeZmKRwUmjne4YJQLLqRj7On\\\\\\\/yZRwBwTjUVelptJ68+NHi0okkEXl9RQFX5RbEh+Wsw3nMi4umRmZybXacdNJvgF3FnknLBIgEbI1cl4vYQl2Zp\\\\\\\/BmV2RuBILVIAqgX1SQ8yEuuMyfgXSRl1uu7tjFBE2xysnVN7X2RwQJvpS0OngeGORys4i7SnDkn3AW+RKV9MgIxdqYeGpwnRft+AaFrbxQHA65NairMMU4SiKJUXaBb4xvkm8eWoo7xlubMvT\\\\\\\/p0zh0hZSvvxnfXOzDhZ2yXDX\\\\\\\/L5qSAHiHLWGafiC0zasqKcsB+KNfUrf2fduGWlU5BmIlfDS4BjHO3tmmWaDLlRHdwUqyyBNiSMFhF+yS+KiM7T7ihh\\\\\\\/otAVpzslVwiC0HeZf9ZQ11qjQ3X2rVtSMd0ffWLniTQ1kztleLPSE0kdyOYb1WHiLGHF4Z3V0AQikgsx+6xvC0+UBTOX++5QJ+KRWoBBwrud8bqvoC+UzJBRZ3ItslEw6Vu8nvQy\\\\\\\/LC+KFE8MzCNhKyIoaW3QP1\\\\\\\/+oll47UYSUr\\\\\\\/I1FdcmjzbgQErYgOHzJMUT1YcbPZ6uWfHAWkXC0anRerHBl0rWozJynaGGSJ\\\\\\\/Cp1+B19eMj5KRiRKG2wSPB59J\\\\\\\/aRQnwdPsJHZ+bV4qI\\\\\\\/9noTsdFrD3DdO\\\\\\\/WdZ+oecuXTU+jm640L\\\\\\\/7SQLB67FWaN8ULBfCYlON9ZhXfxshTwUc14iFnSyAE977e1shZ3Y4GIhWfXdh3\\\\\\\/N9CuAbIS+VO\\\\\\\/pck+GG+kAjVZRunmfOhIGXUe3m4Qw0EX+tvHx4HGj7GwdR+OeIREBzanYE4e86gPFWIvcvQE9FVw2+B3yhwPSREYzM5e\\\\\\\/8uW8N0R53T+0rXpUPZdpzKApr\\\\\\\/T0Nu0daliBcgYgI2ifOg7\\\\\\\/IlenWOJgqxEicRAubncAvxdlJcYc8hLlg0rfIRXpmh55lIjKspHZGyRndcbvvvF7PAESyCCrnZ3IbJ38CmRg0vECf+ZV+UcnGbTbooHM24BRDPXTqx5r1kB9DgSXim29GqZc70pcxY1CewzxG1foCE7rHIsd2nIprUsANWT0idpY1Rt8MDs9MPkvjn3RcfGq5iICu85jPLPICxw\\\\\\\/rzGcl0n8QODgh1024CVgJlw9VyXzPT\\\\\\\/POumvDJ+fT6tU2WHDmkeppk6a+RsguTZXl16wY7gzVBdFPMSfUtE2pZkhMbjizk7V9uBk0ZBUxvRRroy5wDmGszfU0olEb5ZqakPmP2Weli\\\\\\\/Xk9XZtKHR1I\\\\\\\/xBmgSIdT19Ya4qQ\\\\\\\/BcWv8MCjMMQ5mbk4+Gc1fqpSbh4rIhnXa07DGSdroeEAEtEkcGrToXywqqTDj1+Yf1NNl0fomfCcIFkEFkydOVEpJWULkXx9FhatR2g73+PdCrRFWQP0xU9perqaAcQdcgzGWP2HX5H61Qnt9NOCgYQlxnLJPyjmJTYz9g0R+9wJXbtwYBXl3NczJ38CwwqykenY6jQMOu8sETuXW0z4sZ7qG+04267Cpp69TjucXsDxTg372mckSFxHt1r3wgf7h4UYHApEIqMR8uTnb5QdLeL79PHl8ftR+YJATIPSGI9wiGZ\\\\\\\/5iWB7QdqX36GYGLr\\\\\\\/J\\\\\\\/OWwIjKb6IieqHkDsvRawt5LSoEyFUC\\\\\\\/UbyLeEnRnJwJQgxKh4EAJ47MmnYd0ll7\\\\\\\/Mcaa\\\\\\\/updg0c5GaIdRSvrBDiO+71wUyHaiBr0PFz2yxm4qmZmPh\\\\\\\/+FGV5nI3V2B9H\\\\\\\/kbO4SvPaxjftP2VQJz8fMNCoBeCCQjUAVf8U4r5o\\\\\\\/WRT0p3s0kOL1lvUFT8uObBpSV2Tg+uxn246scK8B581qfFihTqxNe2NFM2CRjOtHK31Ajo\\\\\\\/nxhXrrxKskOVvLTkgPmWRgVXGNqOuFZ3LxKwOpxGnFfdSUHYb8V2az8q8HXpBSF+PmBH1bCAM5eK8IUk5eOB54yvnDnlF41T2a0XV0TVttKFgejzPqFZkLywSF0VxyOdBevv+ehmw9BonADnR94w8G\\\\\\\/zy2v5Bh8KuU1mBP0h3o26xI8p4GX9DtB6n7Clt7f4Bg1afVx8g99UJyX\\\\\\\/+jd9zHOoQkgJHIsPaIjWwPr+IDUNbknsrpOuvJLpl2VULlpoa8E5DhQpSzGfiPIOwUXazeZOKnGFnJjsdc7OlB854c4E6WdE1IhujMWJingEyW0RkFJeuTD6WShfJVZOvLtGU8RiGkPUIKNi+Ru43OKSqw0ZY94j\\\\\\\/81CyOvKZ9IHvcy\\\\\\\/h1LQ6lZ69diTFCNJLxMxjDeSGqNmRZ+Rt3+0gChHevpH6cCaJiXg6debgR8EF6zYy5LgDnvZlSmn3R8v4y+sThBY5Dgf957N++H4kKln0NsqB3FLk919N90VRZHH0qu+SnywMMFesqu3PCh6Xv986U7NOLuwfk8sWn0Ta1zMCs9UPPBelYfFseXaeMpwPf5Mqt95DprXeUn1LMTflIU6X6pto8wrQ5zNE8Yg3haOIulaWTANFZwRoLIb3pdl1XQWf\\\\\\\/0S\\\\\\\/lenFJwUSZ+WXGKgoVWLfS0lXzt3uVYgWZX1y9oo\\\\\\\/jqFJ45TL8qfwRWSDmeA1qhlrnXMlj8CQDP2QLJ2N\\\\\\\/RLGaT4U+7HjLiRXTbMqb+26cL+2nk3J3LYGpaaogOSbaO5nwJALdm8M7AFQmoXYAEyc2MG0zGc\\\\\\\/PBelcr1PeJgFGKs0n1uNeSZ46RGLU+M9bllPcZALecPH7HOl8ulSBvnU2Uk09mFM3bJvRFI3x3NqRTtaQh8DEsyyZ+26vIpWhS1EM4lfrq5t4SjqFIgvJiiYmey91owIqSM9WQ+0B2etgGn+8awW31+Bfnr7nr85ZkHsYiUbYWbfGjudfKWkCOl6kqj3l\\\\\\\/mgRWFv8LFUPFex1k3bNy9wtGczxkVhERHy4Dpr3fwCXIfC2f0H9ORAAhM3IRjJDWyBEBPzZMehW60xyc8Vnz2QyqwKYQl9gvClrvbZrs\\\\\\\/+AYqDXGXRTvUNiVSVALKPDRhK+YG39nH2E6ks+WrXWspAnEsHySlCUXvV6MoS0q2hyScGdEaRp2u+BlHSrmBnbysFN35qiy9jObtmolJZIyzdvgpdA5708Gi7EGFtX58UDTARqLUH1WI9stQBa1Yv+vIa8VXBZ8FHzXpjBJCUWylwGhIgEesaXFGU4utD6LGYyMmNYlt1x9mCeJANyovSonmH3nLkD4nuduQT9YhdYmF7KrhnYz+phly0Plvq3DdoPeHaY2kb8Fz15owrxkR9chGykyiiEUqVwfJ6wqSqSwXAKMmqMGs2cn4hNrEz6ZBRdIPohD3DHN0VGzl0iRl6qeJB3Ysnok\\\\\\\/pFy56R0bHy2kxw2ShVndn8mNsM58mvadR04MCOeCIvif6wCrgmKeamyDRk55mwSCXnAYqlybHMI2X+dM5zGE0w\\\\\\\/0QJqG76DHwX5gJUYPiIHh9F\\\\\\\/ROtIvmXz7zZ0NxXhV7iLP5D6n0meLGvnU9\\\\\\\/HHydWufK1RveJQe3hijCOvKx6nLYrnlB75EmT1+TC+J\\\\\\\/adET52XeZ3SSQuYPvh7zfyAEbeuDB+mcxkYiUBFZwteBI\\\\\\\/yWuqFj9yLfl8pweX\\\\\\\/wSpO76dSr\\\\\\\/KWShSQ+51XWUL8JFgYsS3UpMpK6OEFinz4PmWAQgr8raqk3obBNyGMCFnXPcURh8DEKIrU6KShkzUKYBLx77NB0w2MFhMACStGBF2+7ovipzgM9j46KNgUxqC0pKK9eghGSELzPBrrHG02k\\\\\\\/9\\\\\\\/fQOuwzOe3pLVFzI9\\\\\\\/nmbPZvWFa\\\\\\\/aT1DT9CKX\\\\\\\/xsAezcWRcvqKjOJsZo6A+vzbsytDV3QNCnu8kbMKpSue0LE0Eu+9+\\\\\\\/QDKkfpf3hqjkVIgvqyVCDaw5cEGvD7c9ltTMo8rxdhtW9N3jB3PsjmAfixCh2tiNqKeC+tO9U4rHTcuNcAHS3uRdEdHVqA1sYEJ7pX9Cj4iR4cp\\\\\\\/6nt+VFGDANOQe1Rbs4LApoNmJOSDRHF+2E41pUTXI5j\\\\\\\/2F\\\\\\\/oPW3d6E2PrA1NCyJ6bXJpL1X\\\\\\\/W2A+MJCU2igKFvkHaFvoqOmt6bucMdtYDM3FDn7K3a0odtLVb6mCsbFivCALDghSdWdBLlVXrQKc0eIlp9NCpn\\\\\\\/U65UnHmD+cBz8Yz4hZIZSD9xy2uXebxfte8DPuCGLep3lyZ34tatnzx23RU7H1+TcNyNrgZBFbT7ptPXRMLm4SazIGWRz\\\\\\\/+Oo2XCZdVrRpjwvrSW8GBkNUk1ajdbv\\\\\\\/ZKR3s7LF5JbKW+AdkCW8e7Pq+Nhf7EKaW2W3B\\\\\\\/MVzWw6TzDfxUEhziH28EjgPNp7\\\\\\\/T6KHwBVs7nmK8a\\\\\\\/9LZkDUFSsEuFSWQOmnOgUSStqcKz620tv3vSX5avknXsynU2RVYT80snoIrtifnBbhw\\\\\\\/mmN+gS4T3mLmVDqM+U3QWbEqEPdNAzAo37AAa6z5jiv6M6wIlpYTma3Tjyo7MncqAaz+SUh1HEML025SRJCbUYzqEWVcW4a\\\\\\\/sG887blVJvqw69WLR8yx\\\\\\\/R6Nc0Jmf54vsmGnl2trx4uB71IuCWLAywggF0hGypAGBXzXbpdFaT5DNZV7nlKQgtjfpW0TR7\\\\\\\/9CxPL9QD\\\\\\\/mIo0DNgj+y\\\\\\\/Vo7PfKCTCOGbq47iQJCbOQb\\\\\\\/JFhACqMfGDEqiq5EeTuSVrvkbu+kDWKZmIYmCzdZooJtXcMKatXxpaW7dQnRqKEFjbY+pcPy9jF5C4H3J1FOQjdpLD+W5TcdzPCFTqCqfE0z5daXJpOlQhVs86bgK42ddB6Og+wFWuTi3zrebtWKXPgEdk+NmC4qy2\\\\\\\/BVHgmcc52cWTB3CHl5q+BnjG7LtwK5YQd3RrnwIAhEgpgeIGRrwk8VEiHaiOg9qATqCFZP+HJHIYIu2VsIpeJX389bMcPIVTVe3Mt70\\\\\\\/ULLHLqUMr4jgQDBIeXOinGicRO9DfFEuq6gfvOPa00Utf8RrA1OIZEVFdQ1xxhmqCBICMFEvgi0nWfzsYRRGyX4uBpaG\\\\\\\/j0RgzwUl1Cf7UjZySTp9B0FmsomwQWeRTUlvLnwIOlaFnODXY9vnVx5BRbQVw9AxHzGYpD8pduN1Z8kpT0eYnJ1iJ7YRHnxLkoJtQe\\\\\\\/mG+CCP8PBPsN6SC7z5Bdk9ubJN+MP8kBBxmc2KGDz7x1OXJlZe2zwUvBpWazlIRaYgTUsr53E2oLXmcI0EI+wA4W62T4p43ap2jEHAFJH6k7F0eOv8D5Rko0rT+2ujFAowgGBV1Hgw\\\\\\\/5iCTdKTShuC6QqPNZ+OjQ3r0sCddhKNnhtuNlcI6Fj0i7bbPrhzncuIGugtaeaEi9R+qbUaoZQ7mSCiibb0lBjqCfwVXWp4f\\\\\\\/Lneq3gBCSAwrKd\\\\\\\/SdF34fiZxrxSxOpHPW5owji4PzIhWj6RETzBAPWaFdLb27tUgbycRzzftosN9mkKR6NxzvIUo34pC2pTf8WD8XR9I2fE88M\\\\\\\/HwPBUgD\\\\\\\/lOyn31BcRafxFNACrM9UNsJFGOs9cGw4n7l7uLxw99BZzEBomhjdIBzM\\\\\\\/eNwhMIoz7I7\\\\\\\/3uNJNwXeNZsGiuE7EtS3Wgy0tOgNhn8dagPhYFMQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022GFAgM9bRmPF+x9baVGkWUw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002273ee50d2714b51d9cb5c6de5fa695ac7a2f17467ba63eb464f72e1adee6f2eeb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OeJkQJQXVzvQUH0Quc6LyEi\\\\\\\/1fFa1\\\\\\\/W4yZd11YalAA7urQUN4fAcD44n6Rm0YaO3zXDqlyFsqgXNs+evzXDkTrF3A5V\\\\\\\/nQgTldTvCsiIlJDB1Mmz\\\\\\\/ZL+TihHJuxQkILlOrC8qkMvCPLCLVlCffAXxjz4Y2bv97QzprEIEQWhg15m8r+lpg7ZGzaUSSHhIhR62F0E9W0JIc0KDbXHD3ohy3mstndb1W97kIoE\\\\\\\/n2Giq0v+8jsmv6kdbGMlUjEFQlMPhNegJJAUTdulhx0cGn+sqnWi4mRtTnbCwCif6ANwD0VxDotmKWn4i1i5mDDH8TEcowmQG5g4qRAKE7IHPSD\\\\\\\/XvkK+91FEfytqjir75vrwDVAOlkNPu+gSX2xW19x1eC32XocTLOLu0I7jXL202B2q46\\\\\\\/zhqyF\\\\\\\/weGAPpf8SCzfH866zDuNet7OdB\\\\\\\/5M7QJ8TkgoK\\\\\\\/z7O9\\\\\\\/xiCzUuI2BebW6n8NbcURsnEHNRr7wjZCKDDOLkVr3gqc1rQg+1GFUMbY67919wpIVSBdrEI0nirtQC3aKGYG1ny6Y+L1Nts\\\\\\\/p2WtZjd6bMyRgRKGGLltA8nFnigXlhFdBcMqaxg\\\\\\\/UrResG0dIa8fAqegz8sn5Ylk1VLBPVeUmdaehhM+b6xGAEEdXZtZejPHoayE4Xbo746uXs\\\\\\\/49BLyBw3s14CBvrJvD\\\\\\\/ReND4y\\\\\\\/ZCfk46fZrGnm1x+5wgYjRgwcQweHqu81ZEmFf+AxRJCsJY\\\\\\\/n5YGUjYEE3tswNF6jxSlefwGOH5QlJ4fkh+GiEi2aHbGCa+7eNpBKG2KrlG2IB+iBJksmHXmRP1JzvjfO0Txu52f2DeZcrHjdBTeQi7siLXYMK0b7NcJGC1uKMmILzpODkMmWZ\\\\\\\/yOIEkQmNZ\\\\\\\/Q1Suj84IfXuf8hnFbGyiamFfnSyZdz50dk4moPtiUtG1jdKPOXCozX6y0K0cEM+WEGt6w3kucMDyqxV+OVdg+nYsYQdaquSVlEcprn9gu0mKxdrwsmx4Fj1hVwwDcS3IB+Vu80CGO7jBG7fV83Fk0nhU8lYUoF2fAzqImsKJCo7o8gZF\\\\\\\/8LSNoEENjE9e3BB1VpKvBMsTEDvXkj8s+3il9I\\\\\\\/jGkiifkF8ecmhHB6XfwZjaAtNabxLyTj2azxLHjPvJSQ7ZniWY3Mq\\\\\\\/sIa3rYfHFBaD5I+84IMm5BrhrtHp5a7yGF2lm4kDRyeUDar6dE\\\\\\\/rLdk5qo+Pu3u8u5SV1aaSQ3G5UHpW5Fg9Noq2Gd\\\\\\\/4bTJEZrGXTLJCT91F0g4DU5mBZameCF41y921di+hGsBIk3m03CuunQsJ3GZXtzjct8zoaF8x3o7f2\\\\\\\/aBHYoyG0olyMu44EAjgwRCDN3bbWF2qugFrXwCEVAP9ZnXAgF7cflFJwDFIA\\\\\\\/yQb9JACtJ7U\\\\\\\/+\\\\\\\/jqRvAfyCs5o9C9vOVmS4wALsWd9LemG\\\\\\\/SXS38ISGcLDCfOQz7u0JghmKdJ9R9RMAYWjvLq\\\\\\\/rYo5bcyxNbekok+Gj7TYU9n5znb0YLRR2J5guSThSlZ+hiqMw0k\\\\\\\/qmLYhHLM+xlTWcbI1B2deDbMvxKTzbyOhazKeCluPEWgZLe9BPSEhruu+yzuEpB7oTrNUHLoVN1LUxDHVdyYhwePzurc5Uc72vMORnkW8+oWEwM0Zwo1I4Tc3IFwmVxX7uAGzw8NLvmlM+AhJ2pcbxOhffbcuu1Pg5ZvbqybkP8uy5XTaNc9g\\\\\\\/rmI9ixi2PCoFrh9KffnVfOo0Z4jNRsIbMzEOf3Uj5PaCJTbdd5v5YhWeiiYuZ2I6vqMB0+cpsTKe9xdMc4IvGkaqQB1I6cGBMm8uwsmN2VT+APcqM9Njj7sG3ZFlnW5ECQat0MYi+CWhfUwEHToykiDKd2ATYp4puB3mVjcoYqHqVUIufJn\\\\\\\/GzSIMdJiNkRDVqx\\\\\\\/sRtVuY9XwGUGtO8VJn+bkJxK4hNDsurSut3lZF8cZ3ZzHUNOgBYFbJSM0RhEWGFo4cT8IPDRzpwqk1Qk\\\\\\\/tZst6GanZJqmBeXuqYXtEIdGw==\\\u0022,\\\u0022iv\\\u0022:\\\u00223pWubfTLclO44q8kv6WGAQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225bf1d4a7e0aac362604cbc3b84ea6d4adc84f7e98ffa07782eccf8a224c56efc\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460251 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2329" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-i66S78WcHqvCxQFIUUggJZEsb9B6S7Nn_Yf8fSZguE4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2329" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PT0NggSijUc1C1brxzOd4de6e2GoO65F7usrmqrViU0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2319" title="Genetic Regulation of Transcriptional Variation in Natural Arabidopsis thaliana Accessions" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2343" title="A Genome-Wide Association Study for Regulators of Micronucleus Formation in Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460251" id="node460251" data-pisa="ggg;6/8/2329" data-pisa-master="ggg;g3.116.031443" data-apath="/ggg/6/8/2329.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Diego Jarquin</span>, <span class="highwire-citation-author" data-delta="1">James Specht</span> and <span class="highwire-citation-author" data-delta="2">Aaron Lorenz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2329-2341;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031443 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460251" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Diego Jarquin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarquin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarquin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarquin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James Specht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BSpecht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Specht%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BSpecht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aaron Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lore0149@umn.edu" class="" data-icon-position="" data-hide-link-title="0">lore0149@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/8/2329" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460251" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460251/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2329.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460251" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460251/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2329.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460251" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460251/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2329.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460251" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460251/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/8/2329.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460251" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460251/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The identification and mobilization of useful genetic variation from germplasm banks for use in breeding programs is critical for future genetic gain and protection against crop pests. Plummeting costs of next-generation sequencing and genotyping is revolutionizing the way in which researchers and breeders interface with plant germplasm collections. An example of this is the high density genotyping of the entire USDA Soybean Germplasm Collection. We assessed the usefulness of 50K single nucleotide polymorphism data collected on 18,480 domesticated soybean (<em>Glycine max</em>) accessions and vast historical phenotypic data for developing genomic prediction models for protein, oil, and yield. Resulting genomic prediction models explained an appreciable amount of the variation in accession performance in independent validation trials, with correlations between predicted and observed reaching up to 0.92 for oil and protein and 0.79 for yield. The optimization of training set design was explored using a series of cross-validation schemes. It was found that the target population and environment need to be well represented in the training set. Second, genomic prediction training sets appear to be robust to the presence of data from diverse geographical locations and genetic clusters. This finding, however, depends on the influence of shattering and lodging, and may be specific to soybean with its presence of maturity groups. The distribution of 7608 nonphenotyped accessions was examined through the application of genomic prediction models. The distribution of predictions of phenotyped accessions was representative of the distribution of predictions for nonphenotyped accessions, with no nonphenotyped accessions being predicted to fall far outside the range of predictions of phenotyped accessions.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/soybean" class="hw-term hw-article-keyword hw-article-keyword-soybean" rel="nofollow">soybean</a></li><li class="kwd"><a href="/keyword/genetic-diversity" class="hw-term hw-article-keyword hw-article-keyword-genetic-diversity" rel="nofollow">genetic diversity</a></li></ul><p id="p-3">The foundation of plant breeding is genetic diversity, yet the success of modern scientific plant breeding is leading to an erosion of the very genetic diversity it relies upon as farmers discard landraces in favor of genetically improved and uniform cultivars derived from a limited ancestral base. This genetic erosion increases vulnerability to agricultural insect and disease epidemics, as well as diminishes gains from breeding and selection. Germplasm collections serve as an important source of variation for germplasm enhancement; that variation sustains long-term genetic gain in breeding programs. A stunning number of accessions—7.4 million—is being maintained <em>ex situ</em> by plant germplasm collections worldwide, also referred to as gene banks (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">FAO 2010</a>). The largest number of accessions belongs to wheat, with ∼856,000 accessions held, followed by rice with nearly 774,000 accessions (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">FAO 2010</a>). The USDA National Plant Germplasm System (NPGS) alone holds &gt; 574,000 accessions for 14,965 species as of June 2015, ranging from 53,525 accessions for rice to 165 accession for quinoa (<a href="https://npgsweb.ars-grin.gov/gringlobal/query/accessionsbysite.aspx">https://npgsweb.ars-grin.gov/gringlobal/query/accessionsbysite.aspx</a>).</p><p id="p-4">The identification and mobilization of useful genetic variation from germplasm banks for use in breeding programs is clearly a necessity, not only for sustaining current rates, but also for increasing future rates of crop genetic improvement (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Sehgal <em>et al.</em> 2015</a>). Nevertheless, there is evidence that these collections are woefully underutilized. In 2004, Carter and coworkers estimated that among ∼45,000 unique soybean accessions maintained in germplasm collections worldwide, only 1000 have been used in applied breeding programs (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Carter <em>et al.</em> 2004</a>). Beneficial alleles for traits like yield have been mined from exotic and wild germplasm (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Tanksley <em>et al.</em> 1996</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Fox <em>et al.</em> 2015</a>), and breeders accept that landraces and exotic germplasm likely contain alleles that could enhance their germplasm, even for intensely selected traits, such as yield. However, efficiently mining such large germplasm collections with little knowledge on accession breeding values and the distribution of favorable alleles for complex traits like yield is a huge challenge, yet selecting exotic parents for yield improvement is just as critical as selecting elite parents.</p><p id="p-5">Plummeting costs of next-generation sequencing (NGS) is revolutionizing the way in which researchers and breeders interface with plant germplasm collections. It is possible that all accessions held worldwide will be densely genotyped using NGS technologies. Some present examples of wide-scale genotypic characterization of the germplasm collections include the genotyping by sequencing of the CIMMYT maize collection (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hearne <em>et al.</em> 2015</a>) and the sequencing of 3000 rice genomes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Li <em>et al.</em> 2014</a>). This information will greatly benefit the selection of accessions for breeding and genetics research. Using genomic data, accessions could be selected that contain specific alleles of desired effect (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">McCouch <em>et al.</em> 2012</a>), or all accessions representing all allelic variations at particular loci (such as maturity) could be selected. An allele-focused approach could be replaced or augmented by a genomic prediction approach to predict the breeding value of each accession held in the collection (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Habier <em>et al.</em> 2007</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">VanRaden 2008</a>). Such predictions on breeding value, especially when compared to some well-known adapted checks, would greatly increase the value of germplasm collections by giving breeders a means to identify those accessions (of the thousands that are available) meriting their attention (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Longin and Reif 2014</a>).</p><p id="p-6">The USDA Soybean Collection dates back to 1895, with record keeping formally starting in 1898. A large share of the accessions (∼5000) was collected as part of the expedition of P. H. Dorsett and W. J. Morse in Asia between 1924 and 1932 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Nelson 2011</a>). The USDA Soybean Germplasm Collection (hereafter referred to as the Collection) is one of the most intensely used germplasm collections in the world, and the most intensely used in the NPGS (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Nelson 2011</a>). Remarkably, the entire collection has been genotyped with 50K single nucleotide polymorphisms (SNPs; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Song <em>et al.</em> 2015</a>), creating a tremendous resource for understanding the distribution of genomic variation in the Collection and how it relates to phenotypic variation.</p><p id="p-7">We assessed the usefulness of the genomic and phenotypic data collected on 9171 records from the Collection for developing genomic prediction models to evaluate the genetic value of accessions held in the collection for the complex, yet economically important, traits of protein, oil, and yield. Moreover, we investigated factors affecting prediction accuracy such as training set composition, both in terms of subpopulation membership and trial locations. Our results are the first report on using comprehensive, extensive data gathered over time by the curators of a germplasm collection for making genomic predictions that will help breeders select accessions in a more rational manner.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Phenotypic and genotypic data</h3><p id="p-8">The USDA Soybean Germplasm Collection contains ∼18,500 accessions of <em>G. max</em>. The phenotypic data used in this study was obtained from the USDA Soybean Germplasm Collection evaluations conducted periodically to characterize newly acquired accessions for basic morphological, agronomic (including yield), and seed quality traits. Data from 25 trials were analyzed (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Dates of the data sets range from 1963 to 2003 and locations include Urbana, IL; St. Paul, MN; Lexington, KY; and Stoneville, MS. The majority (5731) of accessions was evaluated in just one trial, 2976 accessions were evaluated in two trials, 50 accessions were evaluated in three trials, 11 accessions were evaluated in four trials, and three accessions in five trials. Accessions originally classified as maturity group 0 were evaluated mostly in St. Paul, with a small number evaluated in Urbana. Classification of these accessions has now been refined to include 00 and 000 classification. Maturity groups (MG) I–III were evaluated predominantly in Urbana, with some MG I and II evaluated in St. Paul. MG IV were evaluated in Lexington, Urbana and, to a small extent, Stoneville. Accessions belonging to MGs V–IX were evaluated in Stoneville, with the exception of seven MG V accessions being evaluated in Urbana in 2001–2002 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). All trials were blocked by MG. The 1MN63 and 1IL64 trials included two replications planted within the same year. All other trials also included two replicates, but replicates were planted in two separate years. Field plots comprising the trials conducted between 1963 and 1966 were two rows per entry, 2.4 m long and 1 m apart, except for 1MN63 in which row spacing was 0.90 cm. Starting in 1980, trials consisted of four-row plots to minimize competition effects. Rows were 3 m long and 0.75 m apart at planting and end-trimmed to 2.4 m long. The only exceptions were the 1989–1990 trials in St. Paul and Urbana, where rows were planted to be 4.7 m long but later trimmed to 3.2 m. Data were collected only on the center two rows.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468846/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468846/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">List of trials including accessions from the USDA Soybean Germplasm Collection</span><div class="sb-div caption-clear"></div></div></div><p id="p-9">Protein and oil were also measured using seeds of accessions stored in cold room of the Urbana maintained Collection. This dataset is named SOYBEAN.CHEMICAL.NB.2009 and consists of 2721 samples. Soybean samples were sent from the Collection to St. Paul, MN where they were ground and scanned by NIR (Foss 6500) at the University of Minnesota. All accessions included in this set were also grown and phenotyped as part of other trials.</p><p id="p-10">The traits analyzed for this study were seed yield, oil, protein, lodging, and early shattering. Seed yield was measured as the machine harvestable seed weight per plot adjusted to 13% seed moisture and expressed as Mg ha<sup>–1</sup>. From 1963 to 1966, protein concentration was determined using the Kjeldahl method, and oil concentration was determined using the Butt extraction method. From 1981 and beyond, oil and protein concentrations were determined using near-infrared reflectance on ground samples. Lodging is rated on a scale of 1–5, with 1 given to plots with 100% erect plants, and 5 given to plots with prostrate plants. Early shattering is scored at harvest on a 1–5 scale, where 1 = no shattering, 2 = 1–10% shattering, 3 = 10–25% shattering, 4 = 25–50% shattering, and 5 = greater than 50% shattering. More detailed trait descriptions and information on methods of measurement can be found at <a href="https://npgsweb.ars-grin.gov/gringlobal/descriptors.aspx">https://npgsweb.ars-grin.gov/gringlobal/descriptors.aspx</a>, then choose “Soybean” from the dropdown menu.</p><p id="p-11">In addition to the phenotypic data routinely collected by the USDA and collaborators on the collection, an independent data set on MGs I–V PIs was obtained to serve as an additional validation set. These data were collected by J. E. S. at the University of Nebraska in 2003 and 2004. Briefly, 101 accessions were selected from a larger set of ∼1500 accessions on the basis of acceptable lodging, seed shattering, disease resistance, and overall appearance. Most of these 101 accessions belong to MGs II and III. They were evaluated in field trials under two water regimes, dryland and full irrigation, at Lincoln, NE. Plots were arranged in a randomized complete block design with four replications per water regime. Replications receiving the same water treatment were blocked together in the field. Plots consisted of two rows 0.76 m apart and 2.90 m long. Plots were machine harvested, and seed yield was adjusted to 13% seed moisture. Protein and oil concentration were measured using near-infrared reflectance spectroscopy. For use here, the data were divided into four water-regime–year combinations. A linear model was fit to each dataset separately to calculate estimates of broad-sense heritability on an entry-mean basis. The linear model included rep (fixed) and accession (random).</p><p id="p-12">The original genotype data set consisted of 52,041 SNPs scored using the Illumina Infinium SoySNP50K BeadChip as described by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Song <em>et al.</em> (2013</a>). The SNP data are publicly available at <a href="http://www.soybase.org/dlpages/index.php">http://www.soybase.org/dlpages/index.php</a>. SNPs with greater than 80% missing scores and minor-allele frequencies &lt; 0.01 were removed from the data set, leaving 38,452 SNPs for analysis and genomic prediction model training.</p></div><div id="sec-3" class="subsection"><h3>Subpopulation assignment</h3><p id="p-13">The effect of predicting across and within subpopulations was investigated. Previous research found that country of origin and MG explain only a small proportion of the subpopulation structure (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bandillo <em>et al.</em> 2015</a>). Accessions were clustered using ADMIXTURE (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexander <em>et al.</em> 2009</a>) to objectively assign accessions to more genetically differentiated subpopulations. ADMIXTURE provides model-based estimations of ancestry based on multi-locus genotype data. A number of subpopulations, K, is defined by the user. Each individual is assigned a membership probability to each subpopulation. For this study, the conversion from membership probabilities to discrete subpopulation memberships was accomplished by assigning each accession to the subpopulation that it had the highest probability of belonging to. Determining the value of K was accomplished using the differences from the estimated 10-fold cross-validation (CV) errors obtained from ADMIXTURE for successive K-values (ΔCV). Although the election of an optimal number of subpopulations is not a critical objective of this research, the K value at which ΔCV plateaued was chosen.</p></div><div id="sec-4" class="subsection"><h3>Models</h3><p id="p-14">The Bayesian models here presented include genetic and nongenetic (or structural) covariates. The nongenetic covariates were included to remove, as much possible, the phenotypic variance generated by environmental and population structural factors such as location and maturity group. Since all models have the same linear predictor form, at this point, only the general structure is shown and further specifications will be given to stress differences among models.</p><p id="p-15">The linear predictor can be written as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-2.gif"/></noscript></span></span> is the overall mean common to all phenotypes, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-3.gif"/></noscript></span></span> is the effect of the <em>j</em>th trial (for <em>j</em> = 1,..,26); <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-4.gif"/></noscript></span></span> is the effect of the <em>k</em>th maturity group nested in the <em>j</em>th trial; <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-5.gif"/></noscript></span></span> is the additive genetic effect of the <em>i</em>th accession modeled using whole-genome markers; and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-6.gif"/></noscript></span></span> is the residual. Residuals are assumed to be independent and identically distributed (IID) following a normal distribution with mean zero and variance <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-7.gif"/></noscript></span></span>. Since the effects of the maturity group are expected to change in accordance with the latitude of the trial locations, these effects were considered as nested within trials. Flat priors were given to the trial and maturity group effects to approximate fixed effects in maximum likelihood estimation.</p><p id="p-16">The additive genetic effect of the <em>i</em>th accession is modeled as a linear combination of random marker effects represented by <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-8.gif"/></noscript></span></span>, where <em>p</em> is the number of markers, <em>x</em><sub>il</sub> is indicator variable for the <em>l</em>th marker scored on the <em>i</em>th accession, with <em>b</em><sub>il</sub> being the marker effect. The election of the prior distribution of the random terms enables the model to perform different actions with respect to the treatment of these marker effects as described below. A comprehensive review of the five popular models used for genomic selection (GS) can be found in <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">de los Campos <em>et al.</em> (2013</a>), but a very brief description follows:.</p><div id="sec-5" class="subsection"><h4>Genomic best linear unbiased prediction (G-BLUP):</h4><p id="p-17">A convenient reparameterization to reduce the computational burden is given by considering <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-9.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-10.gif"/></noscript></span></span>. From the properties of the multivariate normal distribution (MVN) <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-11.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-12.gif"/></noscript></span></span> an <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-13.gif"/></noscript></span></span> symmetric matrix whose entries are given by <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-14.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-15.gif"/></noscript></span></span> is the estimated allele frequency at the <em>l</em>th marker. This matrix is known as the genomic relationship matrix (GRM) whose entries describe genomic similarities among pairs of accessions. The posterior mean of g is the best linear unbiased predictor of g, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-16.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-17.gif"/></noscript></span></span> is obtained via restricted maximum likelihood (REML) methods.</p></div><div id="sec-6" class="subsection"><h4>Bayesian least absolute angle and selection operator (LASSO):</h4><p id="p-18">The structure of this model is very similar to that from the <em>Bayes A</em> model; however, the marker-specific prior variances are assumed IID exponential, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-18.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-19.gif"/></noscript></span></span> in this case is a regularization parameter that controls the shape of the exponential prior distribution. The marginal prior distribution of the marker effects becomes a double exponential distribution (DE).</p></div><div id="sec-7" class="subsection"><h4>Bayes B:</h4><p id="p-19">Bayes B is a variable selection model allowing some proportion (π) of marker effects to be null and the remaining (1 – π) to be non-null. This is captured with a mixture density:<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-20.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-21.gif"/></noscript></span></span> as the proportion of markers with null effect. With this consideration, the marker-specific prior distributions of the non-null marker effects are a scaled inverted chi-squared distribution, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-22.gif"/></noscript></span></span>. The prior distribution of the marker-specific variance parameter for the non-null proportion of marker effects is similar to the one used in Bayes A. To completely specify this model, a beta prior distribution is assumed for the proportion parameter such that <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-23.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-24.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2329/embed/mml-math-25.gif"/></noscript></span></span>. All of these assumptions result in the marker effects having a marginal prior distribution comprised of an IID mixture of a point mass at zero, and a scaled-<em>t</em> distribution.</p><p id="p-20">Hyperparameters for all models were set using the rules described in <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">de los Campos <em>et al.</em> (2013</a>). All models were implemented using the BGLR package (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Pérez and de los Campos 2014</a>).</p></div></div><div id="sec-8" class="subsection"><h3>Cross-validation schemes</h3><p id="p-21">A series of CV schemes was designed to assess the usefulness of genomic predictions for selecting accessions as well as optimizing the construction of genomic prediction training sets. To accomplish the latter goal, several different grouping criteria for splitting the data were used in order to create variable training-testing relationships. The first grouping criteria involved splitting the data by trial (<em>i.e.</em>, 26 trials for oil and protein; 25 trials for yield). A second grouping criteria used genetic criteria to split the entire population of accessions into nine subpopulations as described above. Finally, the training-testing sets were grouped by geographical location, which in this case was defined by state in which the evaluation trials were conducted (<em>i.e.</em>, MN, IL, or MS). The KY data were dropped from this analysis since only one trial was conducted in KY.</p><p>Four CV schemes were applied to each grouping criteria. Each CV scheme mimicked the problem of predicting accessions without data. To accomplish this, all phenotypic records of any accession in the validation set was removed from the training set before model training. Each CV scheme is described individually.</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-23">Leave-one-accession-out within groups (<em>One/Group</em>): within each group (<em>i.e.</em>, trial, subpopulation, state) each accession is predicted, one at a time, using as the training set the data from the remaining accessions in the same group. This procedure is repeated until all accessions in the group are predicted. To assess predictive ability, observations and predictions are compiled and correlated for each group separately.</p></li><li id="list-item-2"><p id="p-24">Leave-one-accession-out across groups (<em>One/All</em>): this CV is the same <em>as One/Group</em> except training sets consists of data from all groups rather than only a single group.</p></li><li id="list-item-3"><p id="p-25">Leave-one-group-out (<em>Group/All</em>): here, each group is predicted using a training set consisting of data from the other groups only. The training set does not include data from the group comprising the validation set.</p></li><li id="list-item-4"><p id="p-26">Group-by-group (<em>Group/Group</em>): a whole group is predicted using the information from another, single group. This procedure is repeated for all possible combinations.</p></li></ul><p>A schematic of these CV schemes is displayed in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A diagram of the four cross-validation schemes used to validate genomic predictions. Each of the colors represents a different group. Groups are comprised of trials, states, or genetic clusters (see Materials and Methods). Arrows point from the training set to the validation set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-760892060" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A diagram of the four cross-validation schemes used to validate genomic predictions. Each of the colors represents a different group. Groups are comprised of trials, states, or genetic clusters (see &lt;em&gt;Materials and Methods&lt;/em&gt;). Arrows point from the training set to the validation set.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2329/F1.medium.gif" width="440" height="239"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/8/2329/F1.medium.gif" width="440" height="239"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468961" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-27" class="first-child">A diagram of the four cross-validation schemes used to validate genomic predictions. Each of the colors represents a different group. Groups are comprised of trials, states, or genetic clusters (see <em>Materials and Methods</em>). Arrows point from the training set to the validation set.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Predictive ability was assessed using Pearson’s product-moment correlation coefficient on the vectors of genomic predictions and observed phenotypes adjusted for trial and MG effects. Confidence intervals were computed using the bootstrap procedure with 10,000 bootstrap replicates.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-29">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><p id="p-30">An initial assessment of predictive ability for oil, protein, and yield was made by performing <em>Group/All</em> CV and evaluating predictive abilities among the 25–26 trials with the five models described above. Average predictive abilities were moderate to very high for most trial and trait combinations (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, and Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS1.xlsx">Table S1</a>). For oil, predictive ability ranged from 0.46 to 0.92, with a median across trials of 0.69. Predictive abilities for protein were lower, especially on the low end, ranging from 0.29 to 0.92 with a median of 0.56. Genomic prediction for accession yield, typically the most difficult trait to predict, was more successful than expected, yet highly variable, ranging from 0.17 to 0.79. The median predictive ability for yield was 0.64. Predictive abilities among the three models were compared for each trial and trait combination. Very little to no difference among models was observably evident (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). For this reason, G-BLUP was exclusively used for all subsequent analyses.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictive abilities for oil, protein, and seed yield for each of the 25 trials. Predictions were made using the Group/All cross-validation scheme and three different models: genomic best linear unbiased prediction (G-BLUP), Bayes B, and Bayesian LASSO. The black bars display the 95% C.I." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-760892060" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Predictive abilities for oil, protein, and seed yield for each of the 25 trials. Predictions were made using the &lt;em&gt;Group/All&lt;/em&gt; cross-validation scheme and three different models: genomic best linear unbiased prediction (G-BLUP), Bayes B, and Bayesian LASSO. The black bars display the 95% C.I.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2329/F2.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/8/2329/F2.medium.gif" width="429" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468883" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Predictive abilities for oil, protein, and seed yield for each of the 25 trials. Predictions were made using the <em>Group/All</em> cross-validation scheme and three different models: genomic best linear unbiased prediction (G-BLUP), Bayes B, and Bayesian LASSO. The black bars display the 95% C.I.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">It is important to remember that the validation phenotypes were adjusted for MG effects, and thus variation explained by the genomic prediction model is independent of any variation between maturity groups, and the predictive abilities calculated reflect the ability to predict within maturity groups. Moreover, reported predictive abilities are correlations between predictions and phenotypes corrected for MG, with no adjustment being made for the heritability of the validation phenotypes. Since the validation phenotypes are imperfect estimates of true additive genetic values, the reported predictive abilities are likely downwardly biased estimates of the true prediction accuracies being defined as the correlation between the predictions and true breeding values.</p><p id="p-33">Since most of the accessions analyzed were unimproved landraces, an important consideration is the degree to which variation in lodging and shattering influence variation in machine harvestable seed yield. Shattering is a genetically simpler trait compared to yield (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Funatsuki <em>et al.</em>, 2014</a>), and genomic prediction models trained using data from landraces might be simply predicting shattering rather than purely seed yield. An analysis of the phenotypic data did reveal that machine harvestable grain yield was negatively correlated with shattering and lodging, with mean correlation coefficients being –0.27 and –0.21, respectively (data not shown). In order to eliminate the influence of shattering and lodging on variation in seed yield, shattering and lodging scores were fit as fixed covariates both in the G-BLUP model and to calculate adjusted seed yield phenotypes for validation. Predictive ability was calculated as it was for <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> using the 8517 records with available shattering and lodging scores. Predictive abilities for yield were reduced as expected when variation for lodging, shattering, or both was removed through the use of covariates (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS2.xlsx">Table S2</a>). The average reduction in seed yield predictive ability, expressed as a percentage of the original predictive ability, was near 10% when either lodging or shattering were accounted for (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS2.xlsx">Table S2</a>). When both traits were fit as covariates, the reduction in predictive ability was 23% on average across trials (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS2.xlsx">Table S2</a>). Predictive ability was not reduced at all, or very little, in some trials, whereas in others it was reduced by as much as 47%, indicating shattering and lodging affected seed yield to different degrees across trials. A similar analysis was performed on maturity date, but maturity date had a negligible effect on seed yield predictions after correction for MG effects (results not shown).</p><p id="p-34">The value of training genomic prediction models for prediction of accession performance was further evaluated by using an independent set of MG II and III accessions evaluated in multiple environments, each with four replications, for the measured traits of oil, protein, and seed yield. Entry-mean heritabilities were high due to the highly replicated design, ranging from 0.83 on average for yield to 0.91 on average for protein and oil. Genomic predictive abilities, on average, were 0.58–0.59 for protein and oil, and 0.49 for yield (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). These values are somewhat lower than the predictive abilities estimated for the GRIN trials, perhaps because the Nebraska trials included less genetic variability because accessions were preselected on the basis of agronomic performance. They were, however, similar to the predictive abilities for seed yield in the GRIN trials when seed yield values were adjusted for shattering and lodging. This result indicates that the genomic prediction models can still discriminate among relatively poor and good performing accessions within sets previously selected for agronomic performance. A comparison was made between the genomic predictive ability and correlations between the data available from the GRIN trials, and the phenotypes collected in the highly replicated Lincoln, NE, trials. Genomic predictive ability was consistently better than the GRIN phenotypes for protein and oil, although the confidence intervals did overlap (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). For yield, the two methods were similar for three of the four Lincoln, NE, trials, and genomic prediction was numerically better than the phenotypic data in the fourth comparison (DRY–2003; <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). This result suggests that the genomic predictions are at least as good as the phenotypic data in GRIN, and thus may be a useful tool for choosing among the nonphenotyped accessions held in the Collection or newly collected accessions.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468848/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468848/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Genomic predictive abilities using the Lincoln, NE, data as validation data, and correlations between the phenotypic data available in GRIN and the Lincoln, NE, data</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">A key question when using predictions for accession selection relates to the enrichment of the selected set. While correlations are a good indicator of how successful genomic predictions could be used for this purpose, we desired to look directly at this by calculating the frequency of “selected” accessions observed to be better than the mean or in the bottom 10% based on actual field trial data. The top 10% of accessions were chosen on the basis of their genomic predictions using G-BLUP and <em>Group/All</em> as described above. Shattering and lodging were not adjusted for in this analysis, as we assumed breeders would want select accessions with high machine harvestable seed yield. We found that, on average, a high percentage of accessions among the top 10% based on predictions were observed to be better than the trial mean. This value was 89% for oil, 80% for protein, and 88% for yield (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). In the case of yield, 100% of the selected accessions were observed to be better than the population mean in five trials. Another key outcome would be the avoidance of poorly performing accessions. The top 10% based on predictions very rarely included accessions observed to be in the bottom 10%. The average observed frequency across trials was only 0–2% depending on the trait (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). A frequency of 0% was observed for more than half the trials. This result indicates that predictions can very effectively eliminate the worst- performing accessions.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468898/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468898/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Percentages of accessions among the top 10% of accessions based on predictions observed to be in the bottom 10% or greater than the mean based on phenotypic data from each listed trial</span><div class="sb-div caption-clear"></div></div></div><div id="sec-11" class="subsection"><h3>Trials</h3><p id="p-38">The effect of combining <em>vs.</em> separating trial data was evaluated by performing within-trial CV (<em>One/Group</em>), between-trial CV (<em>Group/All</em>), and combining data across all trials (<em>One/All</em>) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-39">Within-trial predictive abilities were moderate to high for all traits, being greater than 0.60 in most cases for oil and yield (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS1.xlsx">Table S1</a>). Predictive abilities were slightly lower for protein. Only a very subtle improvement was observed when data across all other trials was added to the training set (<em>One/All</em>), with differences ranging from 0.04 (protein) to –0.01 (yield) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS1.xlsx">Table S1</a>). While, on average, there was very little difference, the range across trials was considerable, and it appeared that there were benefits to including all data in the extreme cases. In the case of the 3IL84 trial, for example, it was observed that predictive ability could be increased from 0.31 to 0.81 for oil and 0.44 to 0.75 for protein when data were combined across all trials compared to a within-trial training set only. On the negative side, we observed that predictive ability was decreased by 0.05 for oil and 0.06 for protein, in the case of IL66 and IL945 trials, respectively. It appeared that, for protein and oil, benefits to combining across trials were much more dramatic compared to any reductions in predictive ability (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS1.xlsx">Table S1</a>). The differences between <em>One/Group</em> and <em>One/All</em> were more uniformly distributed in the case of yield, with a reduction of 0.06 for MN945 and a gain of 0.14 for 2KY81.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictive abilities for oil, protein, and seed yield for each of the 25 trials. Predictions were made using the One/Group, One/All, and Group/All cross-validation schemes. The black bars display the 95% C.I." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-760892060" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Predictive abilities for oil, protein, and seed yield for each of the 25 trials. Predictions were made using the &lt;em&gt;One/Group&lt;/em&gt;, &lt;em&gt;One/All&lt;/em&gt;, and &lt;em&gt;Group/All&lt;/em&gt; cross-validation schemes. The black bars display the 95% C.I.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2329/F3.medium.gif" width="306" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/8/2329/F3.medium.gif" width="306" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468920" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-40" class="first-child">Predictive abilities for oil, protein, and seed yield for each of the 25 trials. Predictions were made using the <em>One/Group</em>, <em>One/All</em>, and <em>Group/All</em> cross-validation schemes. The black bars display the 95% C.I.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Using data from the same trial(s) in both training and validation sets creates the unrealistic advantage of including the trial-specific G × E effects contained in the validation data. Because the exact same environmental conditions specific to individual trials would not be observed again, a better assessment of the usefulness of these GRIN training sets for predicting future trial performance would be attained using the <em>Group/All</em> CV. The <em>Group/All</em> CV correlations were very close, on average, to the <em>One/Group</em> and <em>One/All</em> CV correlations (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS1.xlsx">Table S1</a>), indicating that the sheer volume of data can overcome any lack of shared G × E effects.</p><p id="p-42">A trial-by-trial CV (<em>Group/Group</em>) results in highly variable predictive abilities. In many cases, the predictive abilities between trials was zero, but, in some cases, the predictive ability reached as high as 0.90 (<em>e.g.</em>, oil, 2MN81 predict 5MN90) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS3.xlsx">Table S3</a>). The average predictive ability for the <em>Group/Group</em> CV was 0.49 for oil, 0.30 for protein, and 0.45 for yield, which is far less than the predictive abilities observed using <em>Group/All</em> CV. This illustrates the expected advantage of combining data across many trials to form a training set.</p><p id="p-43">By ordering the trials by state, it is apparent that the northern locations of MN and IL predicted one another relatively well as compared to be predictive ability between MS and the northern locations. This pattern was more prominent for yield (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS3.xlsx">Table S3</a>).</p></div><div id="sec-12" class="subsection"><h3>States</h3><p id="p-44">Given the pattern observed when predicting between trials conducted in different states, we desired to look at this more closely by setting up a CV based on trial geographical location. The distribution of data points across states is as follows: 4047 records from 11 IL trials; 1339 records from seven MN trials; 3258 records from six MS trials (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The MN trials consisted of only MG 0–II accessions with the majority (72%) belonging to MG 0. The IL trials were predominantly comprised of accessions from MGs I–IV, with &lt; 1% being from MGs 0 and V. The MS trials consisted only of accessions from MGs IV–IX.</p><p id="p-45">As expected, a training set including data from the state being predicted (<em>One/Group</em> or <em>One/All</em>) performed much better than training sets not including data from the state being predicted (<em>Group/All</em>) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). A key question we wanted to address with this analysis was whether training sets should be created by dividing data among states, or if a universal training set including all data—regardless of state—would perform just as well or better. Little to no differences were observed between these two CV schemes for any trait and state combination (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). This finding suggests that predictive abilities are not improved by maximizing training set size by combining across states, nor are they reduced by including data from environments as different as MS when predicting relative performance of early MG accessions in MN. A similar pattern was observed when variation for lodging and shattering was removed through inclusion of covariates.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictive abilities for oil, protein, and seed yield for each of the three states. Predictions were made using the One/Group, One/All, and Group/All cross-validation schemes. The black bars display the 95% C.I." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-760892060" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Predictive abilities for oil, protein, and seed yield for each of the three states. Predictions were made using the &lt;em&gt;One/Group&lt;/em&gt;, &lt;em&gt;One/All&lt;/em&gt;, and &lt;em&gt;Group/All&lt;/em&gt; cross-validation schemes. The black bars display the 95% C.I.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2329/F4.medium.gif" width="317" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/8/2329/F4.medium.gif" width="317" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468824" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-46" class="first-child">Predictive abilities for oil, protein, and seed yield for each of the three states. Predictions were made using the <em>One/Group</em>, <em>One/All</em>, and <em>Group/All</em> cross-validation schemes. The black bars display the 95% C.I.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468736/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468736/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Predictive abilities for oil, protein, and yield estimated using state as the grouping factor</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Clusters</h3><p id="p-48">The ADMIXTURE analysis suggested the presence of nine genetic clusters within the population of accessions used for this study (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/FigureS1.pdf">Figure S1</a>). A visual inspection of the principal component analysis plot displayed in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> suggests that the diversity <em>within</em> clusters varies, and structure <em>among</em> the clusters exists, with some clusters being more closely related than other clusters. The proximity of clusters to one another can be partially explained by MG. Clusters four, five and eight are comprised mainly of early maturity groups (0–II), whereas early and medium MGs appear in Cluster 1 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS4.xlsx">Table S4</a>). Clusters 2, 3, 6, 7 and 9 belong to medium and late MGs. Most clusters include good representation of at least three MGs except for cluster 3, which is dominated by MG VIII.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A three-dimensional plot of accession values for principal components one, two, and three. The centroid of each cluster is indicated by an empty sphere. The spheres containing numbers label each centroid by its corresponding cluster designation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-760892060" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A three-dimensional plot of accession values for principal components one, two, and three. The centroid of each cluster is indicated by an empty sphere. The spheres containing numbers label each centroid by its corresponding cluster designation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2329/F5.medium.gif" width="440" height="339"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/8/2329/F5.medium.gif" width="440" height="339"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469028" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-49" class="first-child">A three-dimensional plot of accession values for principal components one, two, and three. The centroid of each cluster is indicated by an empty sphere. The spheres containing numbers label each centroid by its corresponding cluster designation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">In general, predictive abilities were lower for the <em>Group/All</em> scheme based on cluster compared to the <em>One/Group</em> and <em>One/All</em> schemes (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Without correction for shattering and lodging, the <em>One/All</em> scheme tended to produce the highest predictive abilities, although the difference between <em>One/Group</em> and <em>One/All</em> were very small. Correcting seed yield for shattering and lodging produced a different outcome where the <em>One/Group</em> scheme was markedly better for four of the nine clusters (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> and <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). A pattern between predictive ability and relationship between clusters was not readily apparent. The only consistent result was the poor predictive ability of cluster 3, which was expected based on its limited size and variation. These results combined indicate that within-cluster information is the most valuable information. We tested whether compiling a training set by only including related clusters improved predictive abilities. To do this, clusters 1, 4, 5, and 8 were grouped, and clusters 2, 3, 6, 7, and 9 were grouped. Grouping clusters by genetic similarity did not improve predictions compared to the universal <em>One/All</em> scheme (data not shown).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predictive abilities for oil, protein, and seed yield for each of the nine genetic clusters (CL). Predictions were made using the One/Group, One/All, and Group/All cross-validation schemes. The black bars display the 95% C.I." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-760892060" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Predictive abilities for oil, protein, and seed yield for each of the nine genetic clusters (CL). Predictions were made using the &lt;em&gt;One/Group&lt;/em&gt;, &lt;em&gt;One/All&lt;/em&gt;, and &lt;em&gt;Group/All&lt;/em&gt; cross-validation schemes. The black bars display the 95% C.I.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2329/F6.medium.gif" width="317" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/8/2329/F6.medium.gif" width="317" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468857" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-51" class="first-child">Predictive abilities for oil, protein, and seed yield for each of the nine genetic clusters (CL). Predictions were made using the <em>One/Group</em>, <em>One/All</em>, and <em>Group/All</em> cross-validation schemes. The black bars display the 95% C.I.</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468979/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468979/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Genomic predictive abilities using the One/Group, Group/All, and One/All cross-validation schemes for oil, protein, and seed yield. Data was grouped by genetic cluster.</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Prediction of nonphenotyped accessions</h3><p id="p-54">A total of 8771 accessions housed in the Collection has been phenotyped at least once in the 26 trials (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS4.xlsx">Table S4</a>), but no phenotypic data were available for 7608 accessions from GRIN when this study was designed. Genomic predictions were calculated for the nonphenotyped accessions using the full training set (<em>i.e.</em>, all clusters, all environments) in order to assess differences in distributions between phenotyped and nonphenotyped accessions. Phenotyped accessions were predicted with the <em>One/All</em> cross-validation scheme. More specifically, we wanted to know if any nonphenotyped accessions would be predicted to be superior to the phenotyped accessions. Substantial differences were not observed with the predictions of the nonphenotyped accessions falling within the range of the phenotyped accessions (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). Nevertheless, using information in the form of genomic predictions will help breeders choose among those accessions that have no accompanying information, opting for those that would be expected to be above average for yield, protein, and oil, and thus avoiding those accessions predicted to be inferior for these traits. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS5.xlsx">Table S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1/TableS6.xlsx">Table S6</a> contain genomic predictions for the phenotyped and nonphenotyped accessions.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scatter plot of genomic predictions for grain yield vs. the sum of oil and protein. The intercept of each trait included in the prediction to place values back on the original trait measurement scale. Phenotyped accessions are represented by the blue density cloud, and nonphenotyped accessions are represented by the red circles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-760892060" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Scatter plot of genomic predictions for grain yield &lt;em&gt;vs.&lt;/em&gt; the sum of oil and protein. The intercept of each trait included in the prediction to place values back on the original trait measurement scale. Phenotyped accessions are represented by the blue density cloud, and nonphenotyped accessions are represented by the red circles.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2329/F7.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/8/2329/F7.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2329/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468841" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-55" class="first-child">Scatter plot of genomic predictions for grain yield <em>vs.</em> the sum of oil and protein. The intercept of each trait included in the prediction to place values back on the original trait measurement scale. Phenotyped accessions are represented by the blue density cloud, and nonphenotyped accessions are represented by the red circles.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-56">Crop germplasm collections hold valuable genetic diversity to help protect society against the genetic erosion of agriculturally important species for which only a limited number of genotypes are actually cultivated at any given time. It is imperative that these collections exist as dynamic, utilized sources of variation rather than as “gene morgues”, as they are sometimes referred to (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hoisington <em>et al.</em> 1999</a>). One obstacle to utilization is reliable phenotypic characterization of collections, as phenotyping collections consisting of tens of thousands of accessions can be difficult and expensive. High density genotyping of entire germplasm collections, however, has become more feasible than thorough phenotyping, even with the advent of phenomics platforms. This study demonstrated that historical data on accessions held in collections, when combined with high density SNP data, can be used to develop predictive models for important and complex traits of soybean. Genomic prediction models explained an appreciable amount of the variation in accession performance in independent trials, with correlations between predictions and observations reaching up to 0.92 for oil and protein, and 0.79 for machine-harvestable seed yield. Predictive abilities for seed yield were reduced when variation for lodging and shattering was accounted for. Nevertheless, estimates of prediction accuracy calculated using data from a highly replicated, independent trial of only accessions with previously determined acceptable performance (<em>i.e.</em>, minimal shattering and lodging) also gave an optimistic outcome for using predictions to assist in the selection of superior accessions. Based on a comparison of predictions and observed field performance in each trial, a soybean breeder could select the top 10% of accessions based on genomic prediction of yield, and expect 88% of the selected accessions to be better than average for yield. This example demonstrated that genomic predictions can be used to enrich field trials of accessions with accessions that perform better than a randomly selected set. Looking at the extremes, we found that the top 10% for each trait rarely contained accessions that performed in the bottom 10% according to actual trial data, indicating that using predictions very effectively eliminates the accessions that hold little promise, ultimately saving field resources to evaluate more of those that do hold promise.</p><p id="p-57">Compiling historical data on accession evaluations conducted across four states going back to 1963 provided us with a very large training dataset consisting of over 9000 accessions. Soybeans adapted to different latitudes belong to different MGs. The trials used as a source of data ranged from trials conducted on early MGs in Minnesota to late MGs conducted in MS. We explored the optimal use of such a large and diverse training set for calibrating genomic prediction models. Our results can be summarized by two basic findings. First of all, the population and target environment being predicted should be well represented in the training set. The poorest predictive abilities were observed when we attempted to predict between states, or between genetic clusters. Second, genomic prediction training sets appear to be very forgiving to the presence of data from diverse geographical locations and genetic clusters. It was surprising to observe that adding data from very different geographical locations had no effect on predictive ability. For example, the prediction of performance in MN environments was not affected by the presence of training data collected in MS on MG VII–IX accessions. This may partially be an artifact stemming from the tendency of accessions from similar MGs to genetically cluster, and the partitioning of MGs across the states used for evaluation. In BLUP, information from closer relatives is weighted more heavily, while less weight is given to information from distant relatives (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">de los Campos <em>et al.</em> 2013</a>), meaning data from MS was probably weighted less heavily in the prediction of early MG accession performance in MN.</p><p id="p-58">Building a training set by adding accessions from different and diverse genetic clusters did not improve nor harm predictive ability when the goal was to predict accession performance within a single cluster. One exception included the prediction of yield corrected for shattering and lodging across diverse clusters. Our general results are not consistent with results from barley that suggested that the addition of unrelated individuals to a training set can potentially reduce predictive ability (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lorenz and Smith 2015</a>), but they are consistent with results in maize where training sets were formed by combining data across heterotic groups (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Technow <em>et al.</em> 2013</a>). The underlying reasons for the neutral effect of adding genetically distant individuals to the soybean accession training sets could relate to the flow of information from historical LD and pedigree relationships to prediction accuracy (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Habier <em>et al.</em> 2013</a>). In the barley case (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Lorenz and Smith 2015</a>), where there is substantive family structure and a high degree of relatedness among lines from the same breeding program, it is likely that pedigree relationships, captured by G, are the predominant source of accuracy. The addition of less related individuals can reduce the accuracy provided by this source of information (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Habier <em>et al.</em> 2013</a>). In the case of the soybean germplasm collection, where many individuals do not share close pedigree relationships, and where common ancestors likely go back many generations, the predominant source of accuracy is likely historical LD. The large training populations and high marker densities may have allowed the capturing of this information (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Habier <em>et al.</em> 2013</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hickey <em>et al.</em> 2014</a>), offsetting any possible detrimental effect on the genetic relationships source of information.</p><p id="p-59">In conclusion, this study demonstrates that historical data collected as part of plant germplasm collection characterizations can be used to develop predictive models to help breeders select accessions for introgressing useful genetic variation. We found that, in the case of the soybean germplasm collection, these models are robust to the inclusion of diverse sources of data, but training sets should include data from populations and environments representative of the target populations and environments. This data has already been collected and made freely available, and therefore nothing is preventing the use of these models for enhancing utilization of this genetic resource. Genomic predictions might also be used to develop trait-specific “core collections” that could be used for deeper phenotyping for detailed studies on physiological mechanisms and high-resolution QTL mapping. It is anticipated that the genomics revolution will create similar data resources for germplasm collections of other agriculturally important species, and that genomic prediction will serve as a key tool for making practical use of the genomic data.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-60">This study was funded by a grant from the North Central Soybean Research Program titled “Acceleration of Soybean Yield and Composition Improvement through Genomic Selection”. We are grateful for their support.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-61">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031443/-/DC1</a></p></li><li class="fn" id="fn-7"><p id="p-62"><em>Communicating editor: S. A. Jackson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-11-11"><span class="received-label">Received </span>November 11, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-05-23"><span class="accepted-label">Accepted </span>May 23, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Jarquin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.1" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.2" data-doi="10.3835/plantgenome2015.04.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bandillo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cregan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A population structure and genome-wide association analysis on the USDA Soybean Germplasm Collection.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2015.04.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2015.04.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2015.04.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-6.8.2329.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sneller</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic diversity in soybean</span>, pp. <span class="cit-fpage">303</span>–<span class="cit-lpage">416</span> in <span class="cit-source"><em>Soybeans: Improvement</em>, <em>Production</em>, <em>and Uses</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Boerma</span>  <span class="cit-name-given-names">H. R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Specht</span>  <span class="cit-name-given-names">J. E.</span></span>. <span class="cit-publ-name">Crop Science Society of America</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.4" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-other" id="cit-6.8.2329.5"><div class="cit-metadata unstructured">FAO, 2010 The second report on the state of the World’s plant genetic resources for food and agriculture. Food and Agriculture Organization (FAO), Rome, Italy.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.6" data-doi="10.2135/cropsci2014.10.0688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cary</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diers</span>  <span class="cit-name-given-names">B. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Confirmation of a seed yield QTL in soybean.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">992</span>–<span class="cit-lpage">998</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D992%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.10.0688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.10.0688&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.7" data-doi="10.1073/pnas.1417282111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Funatsuki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirose</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inaba</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Molecular basis of a shattering resistance boosting global dissemination of soybean.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">17797</span>–<span class="cit-lpage">17892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1417282111%26rft_id%253Dinfo%253Apmid%252F25468966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzUwLzE3Nzk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.8" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjMyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.9" data-doi="10.1534/genetics.113.152207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic BLUP decoded: a look into the black box of genomic prediction.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">597</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152207%26rft_id%253Dinfo%253Apmid%252F23640517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.8.2329.10"><div class="cit-metadata unstructured">Hearne, S., J. Franco, J. Chen, C. P. Sansaloni, C. D. Petroli <em>et al.</em>, 2015 Genome wide assessment of maize genebank diversity; synthesis of next generation technologies and GIS based approaches. Proceedings of Plant and Animal Genome XXIII, San Diego, CA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.11" data-doi="10.2135/cropsci2013.03.0195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hearne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evaluation of genomic selection training population designs and genotyping strategies in plant breeding programs using simulation.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1476</span>–<span class="cit-lpage">1488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1476%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.03.0195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.03.0195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.12" data-doi="10.1073/pnas.96.11.5937"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoisington</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khairallah</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reeves</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribaut</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skovmand</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Plant genetic resources: what can they contribute toward increased crop productivity?</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">5937</span>–<span class="cit-lpage">5943</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHoisington%26rft.auinit1%253DD.%26rft.volume%253D96%26rft.issue%253D11%26rft.spage%253D5937%26rft.epage%253D5943%26rft.atitle%253DPlant%2Bgenetic%2Bresources%253A%2BWhat%2Bcan%2Bthey%2Bcontribute%2Btoward%2Bincreased%2Bcrop%2Bproductivity%253F%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.11.5937%26rft_id%253Dinfo%253Apmid%252F10339521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTYvMTEvNTkzNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.13" data-doi="10.1186/2047-217X-3-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeigler</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The 3,000 rice genomes project: new opportunities and challenges for future rice research.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D3%26rft.spage%253D8%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-3-8%26rft_id%253Dinfo%253Apmid%252F24872878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-3-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24872878&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.14" data-doi="10.1016/j.tplants.2014.06.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longin</span>  <span class="cit-name-given-names">C. F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Redesigning the exploitation of wheat genetic resources.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">631</span>–<span class="cit-lpage">636</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D631%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2014.06.012%26rft_id%253Dinfo%253Apmid%252F25052155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2014.06.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25052155&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000343359900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.15" data-doi="10.2135/cropsci2014.12.0827"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Adding genetically distant individuals to training populations reduces genomic prediction accuracy in barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">2657</span>–<span class="cit-lpage">2667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D2657%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.12.0827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.12.0827&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.16" data-doi="10.3732/ajb.1100385"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McNally</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomics of gene banks: a case study in rice.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1100385%26rft_id%253Dinfo%253Apmid%252F22314574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6Ijk5LzIvNDA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjMyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.18" data-doi="10.1017/S147926211000047X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Managing self-pollinated germplasm collections to maximize utilization.</span> <abbr class="cit-jnl-abbrev">Plant Genet. Resour.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenet.%2BResour.%26rft.volume%253D9%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS147926211000047X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S147926211000047X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.19" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.20" data-doi="10.1371/journal.pone.0132112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sehgal</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vikram</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sansaloni</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierre</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Exploring and mobilizing the gene bank biodiversity for wheat improvement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0132112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0132112%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0132112%26rft_id%253Dinfo%253Apmid%252F26176697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0132112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26176697&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.21" data-doi="10.1371/journal.pone.0054985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quigley</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fickus</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Development and evaluation of SoySNP50K, a high-density genotyping array for soybean.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e54985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De54985%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054985%26rft_id%253Dinfo%253Apmid%252F23372807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23372807&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.22" data-doi="10.1534/g3.115.019000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyten</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quigley</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fickus</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fingerprinting soybean germplasm and its utility in genomic research.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1999</span>–<span class="cit-lpage">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.019000%26rft_id%253Dinfo%253Apmid%252F26224783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTAvMTk5OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.23" data-doi="10.1007/s001220050116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandillo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eshed</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Advanced backcross QTL analysis in a cross between an elite processing line of tomato and its wild relative <em>L. pimpinellifolium</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D92%26rft.spage%253D213%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050116%26rft_id%253Dinfo%253Apmid%252F24166170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24166170&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TZ82500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.24" data-doi="10.1534/g3.112.004630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bürger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction of northern corn leaf blight resistance in maize with combined or separated training sets for heterotic groups.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DTechnow%26rft.auinit1%253DF.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D197%26rft.epage%253D203%26rft.atitle%253DGenomic%2BPrediction%2Bof%2BNorthern%2BCorn%2BLeaf%2BBlight%2BResistance%2Bin%2BMaize%2Bwith%2BCombined%2Bor%2BSeparated%2BTraining%2BSets%2Bfor%2BHeterotic%2BGroups%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004630%26rft_id%253Dinfo%253Apmid%252F23390596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvMi8xOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjMyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2329.25" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2329.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/8/2329.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2319" title="Genetic Regulation of Transcriptional Variation in Natural Arabidopsis thaliana Accessions" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2343" title="A Genome-Wide Association Study for Regulators of Micronucleus Formation in Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/8"><span>Volume 6 Issue 8, August 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/8.cover-source.jpg?itok=40Kz9Oeq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2329" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VtmMOhpuLcAEdzcW97xgcU5ofLYdfQT5eCVQiUX8Y3E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/8/2329" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/8/2329" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/8/2329" class="active" data-icon-position="" data-hide-link-title="0">Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460251" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_iPdd-6oYPc1u9jEyfYfH0oC-LiWf-Y3_pswKFBhqqg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2329.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/8/2329" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-QulyFje-10xs3InIrJAlau7msao0f0otpQvfo2OtzOA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/8/2329" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2329.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460251" id="node460251--2" data-pisa="ggg;6/8/2329" data-pisa-master="ggg;g3.116.031443" data-apath="/ggg/6/8/2329.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Diego Jarquin</span>, <span class="highwire-citation-author" data-delta="1">James Specht</span> and <span class="highwire-citation-author" data-delta="2">Aaron Lorenz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2329-2341;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031443 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460251--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Diego Jarquin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarquin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarquin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarquin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James Specht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BSpecht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Specht%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BSpecht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aaron Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lore0149@umn.edu" class="" data-icon-position="" data-hide-link-title="0">lore0149@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/8/2329" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/8/2329&amp;title=Prospects%20of%20Genomic%20Prediction%20in%20the%20USDA%20Soybean%20Germplasm%20Collection%3A%20Historical%20Data%20Creates%20Robust%20Models%20for%20Enhancing%20Selection%20of%20Accessions" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/8/2329&amp;title=Prospects%20of%20Genomic%20Prediction%20in%20the%20USDA%20Soybean%20Germplasm%20Collection%3A%20Historical%20Data%20Creates%20Robust%20Models%20for%20Enhancing%20Selection%20of%20Accessions" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/8/2329&amp;title=Prospects%20of%20Genomic%20Prediction%20in%20the%20USDA%20Soybean%20Germplasm%20Collection%3A%20Historical%20Data%20Creates%20Robust%20Models%20for%20Enhancing%20Selection%20of%20Accessions" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/8/2329&amp;text=Prospects%20of%20Genomic%20Prediction%20in%20the%20USDA%20Soybean%20Germplasm%20Collection%3A%20Historical%20Data%20Creates%20Robust%20Models%20for%20Enhancing%20Selection%20of%20Accessions" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/8/2329&amp;title=Prospects%20of%20Genomic%20Prediction%20in%20the%20USDA%20Soybean%20Germplasm%20Collection%3A%20Historical%20Data%20Creates%20Robust%20Models%20for%20Enhancing%20Selection%20of%20Accessions" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/8/2329&amp;t=Prospects%20of%20Genomic%20Prediction%20in%20the%20USDA%20Soybean%20Germplasm%20Collection%3A%20Historical%20Data%20Creates%20Robust%20Models%20for%20Enhancing%20Selection%20of%20Accessions" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/8/2329&amp;title=Prospects%20of%20Genomic%20Prediction%20in%20the%20USDA%20Soybean%20Germplasm%20Collection%3A%20Historical%20Data%20Creates%20Robust%20Models%20for%20Enhancing%20Selection%20of%20Accessions" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/8/2329&amp;title=Prospects%20of%20Genomic%20Prediction%20in%20the%20USDA%20Soybean%20Germplasm%20Collection%3A%20Historical%20Data%20Creates%20Robust%20Models%20for%20Enhancing%20Selection%20of%20Accessions" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460251/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460251" id="node460251--3" data-pisa="ggg;6/8/2329" data-pisa-master="ggg;g3.116.031443" data-apath="/ggg/6/8/2329.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Diego Jarquin</span>, <span class="highwire-citation-author" data-delta="1">James Specht</span> and <span class="highwire-citation-author" data-delta="2">Aaron Lorenz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2329-2341;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031443 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460251--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Diego Jarquin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarquin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarquin%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarquin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">James Specht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BSpecht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Specht%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BSpecht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Aaron Lorenz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BLorenz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lorenz%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAaron%2BLorenz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lore0149@umn.edu" class="" data-icon-position="" data-hide-link-title="0">lore0149@umn.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2329&amp;rft.epage=2341&amp;rft.atitle=Prospects%20of%20Genomic%20Prediction%20in%20the%20USDA%20Soybean%20Germplasm%20Collection%3A%20Historical%20Data%20Creates%20Robust%20Models%20for%20Enhancing%20Selection%20of%20Accessions&amp;rft.volume=6&amp;rft.issue=8&amp;rft.date=2016-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jarquin%2C+Diego&amp;rft.au=Specht%2C+James&amp;rft.au=Lorenz%2C+Aaron"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460251/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460251/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460251/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460251/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460251/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460251/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460251/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460251/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460251/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460251/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460251/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460251/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/8/2329" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/8/2329.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/8/2329.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/8/2329.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/8/2329.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
