<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/8/2553" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/8/2553.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/8/2553.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/8/2553.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/8/2553/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/8/2553.atom" />
<meta name="HW.pisa" content="ggg;6/8/2553" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Using Sequence Variants in Linkage Disequilibrium with Causative Mutations to Improve Across-Breed Prediction in Dairy Cattle: A Simulation Study" />
<meta name="DC.Identifier" content="10.1534/g3.116.027730" />
<meta name="DC.Date" content="2016-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Berg et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Sequence data are expected to increase the reliability of genomic prediction by containing causative mutations directly, especially in cases where low linkage disequilibrium between markers and causative mutations limits prediction reliability, such as across-breed prediction in dairy cattle. In practice, the causative mutations are unknown, and prediction with only variants in perfect linkage disequilibrium with the causative mutations is not realistic, leading to a reduced reliability compared to knowing the causative variants. Our objective was to use sequence data to investigate the potential benefits of sequence data for the prediction of genomic relationships, and consequently reliability of genomic breeding values. We used sequence data from five dairy cattle breeds, and a larger number of imputed sequences for two of the five breeds. We focused on the influence of linkage disequilibrium between markers and causative mutations, and assumed that a fraction of the causative mutations was shared across breeds and had the same effect across breeds. By comparing the loss in reliability of different scenarios, varying the distance between markers and causative mutations, using either all genome wide markers from commercial SNP chips, or only the markers closest to the causative mutations, we demonstrate the importance of using only variants very close to the causative mutations, especially for across-breed prediction. Rare variants improved prediction only if they were very close to rare causative mutations, and all causative mutations were rare. Our results show that sequence data can potentially improve genomic prediction, but careful selection of markers is essential." />
<meta name="DC.Contributor" content="Irene van den Berg" />
<meta name="DC.Contributor" content="Didier Boichard" />
<meta name="DC.Contributor" content="Bernt Guldbrandtsen" />
<meta name="DC.Contributor" content="Mogens S. Lund" />
<meta name="article:published_time" content="2016-08-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Using Sequence Variants in Linkage Disequilibrium with Causative Mutations to Improve Across-Breed Prediction in Dairy Cattle: A Simulation Study" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Sequence data are expected to increase the reliability of genomic prediction by containing causative mutations directly, especially in cases where low linkage disequilibrium between markers and causative mutations limits prediction reliability, such as across-breed prediction in dairy cattle. In practice, the causative mutations are unknown, and prediction with only variants in perfect linkage disequilibrium with the causative mutations is not realistic, leading to a reduced reliability compared to knowing the causative variants. Our objective was to use sequence data to investigate the potential benefits of sequence data for the prediction of genomic relationships, and consequently reliability of genomic breeding values. We used sequence data from five dairy cattle breeds, and a larger number of imputed sequences for two of the five breeds. We focused on the influence of linkage disequilibrium between markers and causative mutations, and assumed that a fraction of the causative mutations was shared across breeds and had the same effect across breeds. By comparing the loss in reliability of different scenarios, varying the distance between markers and causative mutations, using either all genome wide markers from commercial SNP chips, or only the markers closest to the causative mutations, we demonstrate the importance of using only variants very close to the causative mutations, especially for across-breed prediction. Rare variants improved prediction only if they were very close to rare causative mutations, and all causative mutations were rare. Our results show that sequence data can potentially improve genomic prediction, but careful selection of markers is essential.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/08/01" />
<meta name="citation_mjid" content="ggg;6/8/2553" />
<meta name="citation_id" content="6/8/2553" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/8/2553" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/8/2553.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/8/2553.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/8/2553.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.027730" />
<meta name="citation_pmid" content="27317779" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="2553" />
<meta name="citation_lastpage" content="2561" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Irene van den Berg" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark" />
<meta name="citation_author_institution" content="Génétique Animale et Biologie Intégrative (GABI), French National Institute for Agricultural Research (INRA), AgroParisTech, Université Paris Saclay, 78350 Jouy-en-Josas, France" />
<meta name="citation_author_email" content="irene.vandenberg@unimelb.edu.au" />
<meta name="citation_author" content="Didier Boichard" />
<meta name="citation_author_institution" content="Génétique Animale et Biologie Intégrative (GABI), French National Institute for Agricultural Research (INRA), AgroParisTech, Université Paris Saclay, 78350 Jouy-en-Josas, France" />
<meta name="citation_author" content="Bernt Guldbrandtsen" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark" />
<meta name="citation_author" content="Mogens S. Lund" />
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark" />
<meta name="citation_reference" content="citation_journal_title=Anim. Prod. Sci.;citation_author=D. Boichard;citation_author=F. Guillaume;citation_author=A. Baur;citation_author=P. Croiseau;citation_author=M. N. Rossignol;citation_title=Genomic selection in French dairy cattle.;citation_pages=115-120;citation_volume=52;citation_year=2012;citation_doi=10.1071/AN11119" />
<meta name="citation_reference" content="citation_journal_title=Journal of applied genetics;citation_journal_abbrev=J Appl Genet;citation_author=M. H. Braunschweig;citation_title=Mutations in the bovine ABCG2 and the ovine MSTN gene added to the few quantitative trait nucleotides identified in farm animals: a mini-review.;citation_pages=289-297;citation_volume=51;citation_year=2010;citation_issue=3;citation_pmid=20720303" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=R. F. Brøndum;citation_author=E. Rius-Vilarrasa;citation_author=I. Strandén;citation_author=G. Su;citation_author=B. Guldbrandtsen;citation_title=Reliabilities of genomic prediction using combined reference data of the Nordic Red dairy cattle populations.;citation_pages=4700-4707;citation_volume=94;citation_year=2011;citation_issue=9;citation_pmid=21854944;citation_doi=10.3168/jds.2010-3765" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=R. F. Brøndum;citation_author=G. Su;citation_author=L. Janss;citation_author=G. Sahana;citation_author=B. Guldbrandtsen;citation_title=Quantitative trait loci markers derived from whole genome sequence data increases the reliability of genomic prediction.;citation_pages=4107-4116;citation_volume=98;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. B. Cole;citation_author=G. R. Wiggans;citation_author=L. Ma;citation_author=T. S. Sonstegard;citation_author=T. J. Lawlor;citation_title=Genome-wide association analysis of thirty one production, health, reproduction and body conformation traits in contemporary U.S. Holstein cows.;citation_pages=408-408;citation_volume=12;citation_year=2011;citation_pmid=21831322;citation_doi=10.1186/1471-2164-12-408" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=H. D. Daetwyler;citation_author=F. S. Schenkel;citation_author=M. Sargolzaei;citation_author=J. A. B. Robinson;citation_title=A Genome Scan to Detect Quantitative Trait Loci for Economically Important Traits in Holstein Cattle Using Two Methods and a Dense Single Nucleotide Polymorphism Map;citation_pages=3225-3236;citation_volume=91;citation_year=2008;citation_issue=8;citation_pmid=18650300;citation_doi=10.3168/jds.2007-0333" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=H. D. Daetwyler;citation_author=A. Capitan;citation_author=H. Pausch;citation_author=P. Stothard;citation_author=R. van Binsbergen;citation_title=Whole-genome sequencing of 234 bulls facilitates mapping of monogenic and complex traits in cattle.;citation_pages=858-865;citation_volume=46;citation_year=2014;citation_pmid=25017103;citation_doi=10.1038/ng.3034" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=A. I. Vazquez;citation_author=R. Fernando;citation_author=Y. C. Klimentidis;citation_author=D. Sorensen;citation_title=Prediction of complex human traits using the genomic best linear unbiased predictor.;citation_pages=e1003608;citation_volume=9;citation_year=2013;citation_pmid=23874214;citation_doi=10.1371/journal.pgen.1003608" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. P. W. de Roos;citation_author=B. J. Hayes;citation_author=R. J. Spelman;citation_author=M. E. Goddard;citation_title=Linkage disequilibrium and persistence of phase in Holstein-Friesian, Jersey and Angus cattle.;citation_volume=179;citation_year=2008;citation_pmid=18622038;citation_doi=10.1534/genetics.107.084301" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=T. Druet;citation_author=I. M. Macleod;citation_author=B. J. Hayes;citation_title=Toward genomic prediction from whole-genome sequence data: impact of sequencing design on genotype imputation and accuracy of predictions.;citation_pages=39-47;citation_volume=112;citation_year=2014;citation_pmid=23549338;citation_doi=10.1038/hdy.2013.13" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Erbe;citation_author=B. J. Hayes;citation_author=L. K. Matukumalli;citation_author=S. Goswami;citation_author=P. J. Bowman;citation_title=Improving accuracy of genomic predictions within and between dairy cattle breeds with imputed high-density single nucleotide polymorphism panels.;citation_pages=4114-4129;citation_volume=95;citation_year=2012;citation_issue=7;citation_pmid=22720968;citation_doi=10.3168/jds.2011-5019" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Mapping genes for complex traits in domestic animals and their use in breeding programmes.;citation_pages=381-391;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19448663;citation_doi=10.1038/nrg2575" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. Grisart;citation_author=F. Farnir;citation_author=L. Karim;citation_author=N. Cambisano;citation_author=J.-J. Kim;citation_title=Genetic and functional confirmation of the causality of the DGAT1 K232A quantitative trait nucleotide in affecting milk yield and composition.;citation_volume=101;citation_year=2004;citation_pmid=14983021;citation_doi=10.1073/pnas.0308518100" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=B. J. Hayes;citation_author=P. J. Bowman;citation_author=A. C. Chamberlain;citation_author=K. Verbyla;citation_author=M. E. Goddard;citation_title=Accuracy of genomic breeding values in multi-breed dairy cattle populations.;citation_pages=51-51;citation_volume=41;citation_year=2009;citation_pmid=19930712;citation_doi=10.1186/1297-9686-41-51" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=J. K. Höglund;citation_author=G. Sahana;citation_author=B. Guldbrandtsen;citation_author=M. S. Lund;citation_title=Validation of associations for female fertility traits in Nordic Holstein, Nordic Red and Jersey dairy cattle.;citation_pages=8;citation_volume=15;citation_year=2014;citation_pmid=24428918" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Howie;citation_author=C. Fuchsberger;citation_author=M. Stephens;citation_author=J. Marchini;citation_author=G. R. Abecasis;citation_title=Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.;citation_pages=955-959;citation_volume=44;citation_year=2012;citation_issue=8;citation_pmid=22820512;citation_doi=10.1038/ng.2354" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=C. Hozé;citation_author=S. Fritz;citation_author=F. Phocas;citation_author=D. Boichard;citation_author=V. Ducrocq;citation_title=Efficiency of multi-breed genomic selection for dairy cattle breeds with different sizes of reference population.;citation_pages=3918-3929;citation_volume=97;citation_year=2014" />
<meta name="citation_reference" content="citation_title=Residual maximum likelihood estimation of (co)variance components in multivariate mixed linear models using average information.;citation_author=J. Jensen;citation_author=E. A. Mäntysaari;citation_author=P. Madsen;citation_author=R. Thompson;citation_pages=215-236;citation_volume=49;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. S. Khatkar;citation_author=P. C. Thomson;citation_author=I. Tammen;citation_author=H. W. Raadsma;citation_title=Quantitative trait loci mapping in dairy cattle: review and meta-analysis.;citation_pages=163-190;citation_volume=36;citation_year=2004;citation_issue=2;citation_pmid=15040897;citation_doi=10.1051/gse:2003057" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=M. S. Lund;citation_author=G. Su;citation_author=L. Janss;citation_author=B. Guldbrandtsen;citation_author=R. F. Brøndum;citation_title=Invited review: Genomic evaluation of cattle in a multi-breed context.;citation_pages=101-110;citation_volume=166;citation_year=2014;citation_doi=10.1016/j.livsci.2014.05.008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. M. MacLeod;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=The effects of demography and long term selection on the accuracy of genomic prediction with sequence data.;citation_volume=198;citation_year=2014;citation_pmid=25233989;citation_doi=10.1534/genetics.114.168344" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. K. Matukumalli;citation_author=C. T. Lawley;citation_author=R. D. Schnabel;citation_author=J. F. Taylor;citation_author=M. F. Allan;citation_title=Development and characterization of a high density SNP genotyping assay for cattle.;citation_pages=e5350-e5350;citation_volume=4;citation_year=2009;citation_issue=4;citation_pmid=19390634;citation_doi=10.1371/journal.pone.0005350" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=U. Ober;citation_author=W. Huang;citation_author=M. Magwire;citation_author=M. Schlather;citation_author=H. Simianer;citation_title=Accounting for genetic architecture improves sequence based genomic prediction for a Drosophila fitness trait.;citation_pages=e0126880;citation_volume=10;citation_year=2015;citation_pmid=25950439;citation_doi=10.1371/journal.pone.0126880" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. Pérez-Enciso;citation_author=J. C. Rincón;citation_author=A. Legarra;citation_title=Sequence- vs. chip-assisted genomic selection: accurate biological information is advised.;citation_pages=43;citation_volume=47;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=L. R. Porto-Neto;citation_author=W. Barendse;citation_author=J. M. Henshall;citation_author=S. M. McWilliam;citation_author=S. a. Lehnert;citation_title=Genomic correlation: harnessing the benefit of combining two unrelated populations for genomic selection.;citation_pages=84;citation_volume=47;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L.-A. Raven;citation_author=B. G. Cocks;citation_author=B. J. Hayes;citation_title=Multibreed genome wide association can improve precision of mapping causative variants underlying milk production in dairy cattle.;citation_pages=62;citation_volume=15;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=J. Dairy Sci.;citation_author=G. Sahana;citation_author=B. Guldbrandtsen;citation_author=B. Thomsen;citation_author=L.-E. Holm;citation_author=F. Panitz;citation_title=Genome-wide association study using high-density single nucleotide polymorphism arrays and whole-genome sequences for clinical mastitis traits in dairy cattle1.;citation_pages=7258-7275;citation_volume=97;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Using Sequence Variants in Linkage Disequilibrium with Causative Mutations to Improve Across-Breed Prediction in Dairy Cattle: A Simulation Study" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="twitter:description" content="Sequence data are expected to increase the reliability of genomic prediction by containing causative mutations directly, especially in cases where low linkage disequilibrium between markers and causative mutations limits prediction reliability, such as across-breed prediction in dairy cattle. In practice, the causative mutations are unknown, and prediction with only variants in perfect linkage disequilibrium with the causative mutations is not realistic, leading to a reduced reliability compared to knowing the causative variants. Our objective was to use sequence data to investigate the potential benefits of sequence data for the prediction of genomic relationships, and consequently reliability of genomic breeding values. We used sequence data from five dairy cattle breeds, and a larger number of imputed sequences for two of the five breeds. We focused on the influence of linkage disequilibrium between markers and causative mutations, and assumed that a fraction of the causative mutations was shared across breeds and had the same effect across breeds. By comparing the loss in reliability of different scenarios, varying the distance between markers and causative mutations, using either all genome wide markers from commercial SNP chips, or only the markers closest to the causative mutations, we demonstrate the importance of using only variants very close to the causative mutations, especially for across-breed prediction. Rare variants improved prediction only if they were very close to rare causative mutations, and all causative mutations were rare. Our results show that sequence data can potentially improve genomic prediction, but careful selection of markers is essential." />
<meta name="og-title" property="og:title" content="Using Sequence Variants in Linkage Disequilibrium with Causative Mutations to Improve Across-Breed Prediction in Dairy Cattle: A Simulation Study" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/8/2553" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Sequence data are expected to increase the reliability of genomic prediction by containing causative mutations directly, especially in cases where low linkage disequilibrium between markers and causative mutations limits prediction reliability, such as across-breed prediction in dairy cattle. In practice, the causative mutations are unknown, and prediction with only variants in perfect linkage disequilibrium with the causative mutations is not realistic, leading to a reduced reliability compared to knowing the causative variants. Our objective was to use sequence data to investigate the potential benefits of sequence data for the prediction of genomic relationships, and consequently reliability of genomic breeding values. We used sequence data from five dairy cattle breeds, and a larger number of imputed sequences for two of the five breeds. We focused on the influence of linkage disequilibrium between markers and causative mutations, and assumed that a fraction of the causative mutations was shared across breeds and had the same effect across breeds. By comparing the loss in reliability of different scenarios, varying the distance between markers and causative mutations, using either all genome wide markers from commercial SNP chips, or only the markers closest to the causative mutations, we demonstrate the importance of using only variants very close to the causative mutations, especially for across-breed prediction. Rare variants improved prediction only if they were very close to rare causative mutations, and all causative mutations were rare. Our results show that sequence data can potentially improve genomic prediction, but careful selection of markers is essential." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
    <title>Using Sequence Variants in Linkage Disequilibrium with Causative Mutations to Improve Across-Breed Prediction in Dairy Cattle: A Simulation Study | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RPsIH_W4actfGikZQanEfTPL1ifXex9O8QKViXOvzAA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460268","apath":"\/ggg\/6\/8\/2553.atom","pisa":"ggg;6\/8\/2553","ac":{"\/ggg\/6\/8\/2553.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/8\/2553.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2553"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2553"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/8\/2553.figures-only":true,"\/content\/6\/8\/2553":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022jfsTConhsMlpHrHqdUq9h9o7QyeHSQMjajVOec1OlkMq\\\\\\\/lNAYOtLqt6VnPtA6SgdgQbKVJHPU9GWYnajexFOGQdYics0BXAMjz5s6MYgldWK+OzytWGMoRVNRl1Ko0rX7U7Wv4qdLCICNGQMDXgYddYHgSVQyBVvduNcoTyAkHS2i0sACgw5gJk9fDHwwdJqDPmk+UvzVA98d2G5mcojFP92cWlSjHAeQ\\\\\\\/NBfAumBN+BdMnL3D4u8uaZMHJqP3bY31ooT7jnauGfP73isk+98iNpRkx8epZ3OcpmsaFrR0kPl9w85I8Q6Hf57WlyqtEC+fx+gDrHISmeZBK+KJkl+6hetNstQQggZrkeV75kNiBLtoureNcriMCJ6sW9fAxafCCj6LDlv22Oi0iATAJS55npVX0tzTW2EIH6EZ9PPJzcRxolb1wbzceUurE6kb9kTS52TdObUdlXn\\\\\\\/5H2Xq0lV5e5Y\\\\\\\/y6lqlfPVvmu2FZLhh9eXl0Wq8L172AgyT+u+CWyxd9RkEBYC2QGKmDW5XvunVfETdr8sCZxnKKpxbbuq1TExxlxftCCsbj9KI\\\\\\\/9\\\\\\\/tXKQgg1h8dRHiYJkAAM6yUyNHeeNHgwU56NNY8tHtebKCB1XbUPJRV9IUQaIKBP\\\\\\\/7+mWBXH++uHFlgR6zU87t\\\\\\\/yhyzJrgHPn6eFy9zJraO6w\\\\\\\/9sVOmGOb8KG4+bmVifYrfwYGuM6phaMO32DihbC2TCYgScXaoIcgkKrMn9Z054BoaPgCA1FEyZpRV3K1Ii+gEhm5siAQjAQVB512005UDxAZd8Ape2mzZwFTRZWJnTVxEH\\\\\\\/kz382ST7lmOro5v\\\\\\\/UfvnO\\\\\\\/RjyYCAamNHwv8zaj71F71zlVcusvGOAAI+S8mFVKc8VSqcHLwD9xJkzorr5Xi9T+qVZv\\\\\\\/7Sw6bTNgY+A8JJrMjriIJpmHsCiIdV2eg0\\\\\\\/0Y6nmFRv4FNqU2u\\\\\\\/VD1azHyO9373ltNoL+tJLfBK8ssE\\\\\\\/Z84g0cYF1Mvq3v7h5KOGWqKqoqx6YnRb267FGJFUzKzPREMKUr1tGY9eigXm7IQtGZRHWBnO1U9sIq06MKTadn1N4Vgu5qOIrNxOY5EGSA3f6+bN5rkOWVcWdAkYeFd\\\\\\\/VwsOWvzYVcTd0ho9u8OtS472ySGflJvLDaXaWg\\\\\\\/UUxh3iTAdvsybFTA4HxkeVjcBb4Oxo0Wg4lp8yPNm9Ch0MJNxni3fPHXqZo5MCGcoc\\\\\\\/zqG0u8ZU\\\\\\\/JnSUJ8+GQtexnap7eJ4JOlGAdqLngJQBzs8Af7enby+7x8Kf1SR\\\u0022,\\\u0022iv\\\u0022:\\\u0022hga3A5INSLBUSeSFOmJf9Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228b77dca8cfb072429fa4c40ea63e6a00eee345fbdb623d2733ba93729f2c9991\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022A9YQbrn2FxUzX39PS+cckpkf2Qg+TaxeJxy1G\\\\\\\/VXQEzzwgBdaCt8j06OReYfWnJnfP0BRezKl2HbwL6BRTpUymryH93ZAnJviK907twxvcxf5HjRYMv8kxRP1IWJJu98VWM+VMp3IqYZmk\\\\\\\/Nm+2bGwFsRjrh0Cv15B+S5Jt4nJ12su4ad8Kf\\\\\\\/UV8\\\\\\\/eGg0VpD1LtcAJm+6zaJYcYAdp+bZkCaY4uWzL\\\\\\\/RAuJrGQhRimQtwUtWBv1gW6fXFLdEx\\\\\\\/x8pzX3we65X5sLuxC+4c2VjBu5fI2ZRaxp2cZgXchBHmFpY4DuI4ikGcOODG7r7eLTGiMLyAp+49jXid2E6COEVlyT1EkhTefrsJe6enwBLWu8Wzyx59EzgPJ4joyZe6BX+A5S8zCdORvpFFh4TzQzVebF0XnGpBns7mmwZBC0QjRckNYkGRkwZF6L4yDQPx3vx1uUnXxbkf\\\\\\\/iR5BtYtO\\\\\\\/ktKsmhlNNHYQzcQeWDJsIx+yWi\\\\\\\/SipwmSCVbLWOP3o6083weyL81e5PXxtXIpt2rFEX+JNf0H0efvlpsvQZjthADf3rGla+uSjrQXGBJj8Kvob28HI2L0zW0Gy1RwZAxVlNKa7MQVPaRXnQKDgCUxVAjSlfjTeMM\\\\\\\/mhtqLfAy1urqxPb5CU+qp5vsYauS+MUIeUJShDId4U4UnlEOvfkabrz9EYcOWrm3bae06C6D1+26p5LVacAkVBkzZSwhH6YIZpApfcaYZmbsYjHB04O\\\\\\\/yAq6g9rt1me\\\\\\\/0mfmO5zVbVC1wOZfagluFteuPC0xCc8Ix\\\\\\\/z85iAdNdC\\\\\\\/QzwOjPy7OnVsJqy61pygk4XzVsv9hD8\\\\\\\/\\\\\\\/IMTY7LvWAIw5JCJHPeQ3pe9vyOkehEf7EEFv7LNF4pzcwoRBHoMbE+kJpOYg6IrfxXfgBqBXweEQ57opsat1cA7ZQu7fntn6piT72oezS5dNHNy1S3uxjejY09yflpv77ukT0dOWF\\\\\\\/L00Y\\\\\\\/DjvJzf2s9qtqE1oypfMkb84sEKnyy5kjKtBD7zHFIuSvWMsYjUlN2u4BPziE31OSDeqjq4UNeRjC6Y2BVatV+ejdHN1aVwAe0BPDKNu4ecrfoXvr4OZ5v4X8fI9DwjnQNFukynefjUef80dYHdqhXkthBVSLZmeEdTGTz3LfwSzf+kNKaVarbMD0v0uoDew3NYe0BviAkNZybL5uRmQygTmk1oSLcwIZrUAgWGzpCk8BXm11VH3ADB4GtHnrs2f7Pon4ZWEuniX33Naclv8034eKzzSfrXyzF19D+RaVOg1nhbHPq5PwBZAp2hRB3AAJrXHnIVRotoGw5p\\\\\\\/pOE24pqPVBJaU+5BRdI4TMOt0qSkvUQrnaXy7cs6Pp0uS+YoN+zzLtGbnaNKmIvQ6okS\\\\\\\/TT7m9z5QrWAWEantO+0JmRpnx+n\\\\\\\/5jqQP+ht+T+MX83qxykGCs\\\\\\\/NeRQ4yj5VzXe\\\\\\\/anCEKn1dm6+h7T0yGKgDbwWQFSmgd4ZSXK0IkHFogW\\\\\\\/82lWjUPYdaznkfPdw\\\\\\\/o6UNNSUReompv2IYifxO7XvqHHXjUMlr+rMpxTWfUrbnZ4geGkbnri6dQxJFzu8RdJu57lHFsG\\\\\\\/BzBGbNXx7kEyCQn4toiyiCPrF+5B348HziIwHfNvDxoEdk\\\\\\\/y575K36eIg+3HgnaQaSaBwnsc8Yw7halLjfJLCoZRX7NpFxpaTWo3Kgngvj61dYfEoOzEFRa4SvcZhQk2oEH3UbqtI528WqdPzxtWuJxaXDFMOmVCFn2F4ZgKnry9UUMK\\\\\\\/XiWepqubrBYfCs\\\\\\\/8v5dPtwD\\\\\\\/ABRHI8ZOWsZkvRjN6GOxdobyM1dBGEQoncbi58ml+KnwjEQUWyEzFK4yNMFUdXem0XwBMFXVlYpQTv0\\\\\\\/qcg7wMKfoMeTgcCR5aLQvT+L68n17Fw4y\\\\\\\/UeSOXrpS1iJtUiEL7xKHFx8qyPAlJ3\\\\\\\/hizQEU\\\\\\\/Cq1vEE4DQK7\\\\\\\/3YSlrLxbnMkxqd+duSb3\\\\\\\/nlsZsQpFPu+bD1c4rtjQO8WLYCMW5yjiL2Yl0elIECvuYkjls6ddlsvZDaNcM7o\\\\\\\/zWlO9j1zYq9SUt6TSRX2iEt+eOxFBNe5B9xkOiEN7wnqcQuyN8u29BQfWQAARx4qr5qAEsAEtHYgyicbSQPGe5K2lwynReBweibXhsnw6RzEfYWnUNeWNZCTEwvgx1hgoQHdNZXv6CpAD2nuP8Zfg3BdG5S+PJukuehitGwkid6TFPMGiCi8hzVfVn8yBNct5QpZzhmZNfxqBmwGPDnTbdyFITajOMBzhYScSSn\\\\\\\/lDDoDg6u8rXSf92CYvkcz4bqb0KyvTq\\\\\\\/p4CaZBKwBpB1Nnu5IweSWBcA2qHP\\\\\\\/55Q3YlirMkhJCiQqyJFmNoFQNT+KEADezlO+8ZLfjBVrapVmxuFmNISHfi54mh6WWwVOG\\\\\\\/dLivDOoP48rAAu4AI9tuPgTa\\\\\\\/YeZMFfYBtmEIYZZXCddSJj6isQY1prbIg1zTtUsTCgetuHmmS0TiwZPHvkRwsIgMlbU4hS6tl5GAbFUV7fEoih6M0PWZwAN8NwDIxloo5z7EVLf41AUVrlVml\\\\\\\/NHnASU8p94JmIAGc95bl1UheZr+YrrKmgeFsW0PZHtjOTLbJLmOWu2vuoS26x9A3Zzax6vEL6K+VXrEmGZ83AoBZO3hyEjHK0BlUtpwUDoq9qAMGss8JFnOM45yChyJNthlhyCuebDvEpZmpxtaXmOH2D7wGoIQRtX\\\\\\\/P0yh3y2+AN1RmpW9u2+6lpkNdAJtJaXIi1JkpVkk9n8p2zDs8ZhYEZXdp64FdwwzTM5zSBPE3BtFj4Va\\\\\\\/RIRTmvuD\\\\\\\/CoRtSWyYeglL6oMyeCAa2KtO5GBl3eeUZFLdU9oO99Gjg+XPgYBy08l93SVA6Zy0fhSA\\\\\\\/CeLRBwA18jfZQxPAVf1WkSRzeUz450bNcCz+qFz4uwpEhlaUGWiIrBE07umxk0vV\\\\\\\/qBPGePji\\\\\\\/Y+qpZiNadgs99QrE7uInNz6BefBlkAbiRgdIB5oH2NPlTqOgvVQxb5cbG4N3YkmZo87ihW+cX1RWu69fen3EmOKJCzg0yl3WsNaDGE\\\\\\\/m8BTMS6bk+jbteC8otn6r7d\\\\\\\/OhaRS3CHzHpQ\\\\\\\/mKzks9e7ZhY6DxtN1ezF2Us7BQjE3NxrWx3XEmf1Fn0bJvDconFwpgDhWZ94eD3lY\\\\\\\/w4+XvRy3A5t1Wn0kov5IZHVJv4xcGVOADcRI65uniK8FJkfYbdSTxoMmie1JJgA3xOf+5RwcE\\\\\\\/HEbx7uf66hfe8j5NdKPQ+d25+lV+WEyrvZm3R\\\\\\\/vvtgZ6BKsbzlRgVSiDG5SpHfoPQtlUareY0Gl36q7Cdt3glZIDErb6QWMJ4v5Msee5rBgkuunZa7Ko3+Sdro1e5BTQ964mEqBQjQ9dBNi4v1sE8dDmwYaTCTwvrZwhYdSo\\\\\\\/5kuoND0iwBxhILua7nPEvyVr7aMi9zH6hvoLzj6Q7TsW0ZI4N1hMyRFBGBI8tOzJPMINs\\\\\\\/qG0JJrsKp6RchcGOMfQECgBQ3f\\\\\\\/fxS0eqdUeQJexsR16uUHOiRRkbHMp2UhX8wvKhRRvQxV00yyWYP6h20tnSPQYUeNQDWA22t3h1ZLuVnMP36M+oBKMhN4vKsrsRiECjFnlGHTWxtNMic9uUDV79QXPktk1taZZdRvw1Kn5tzynfWl5sE1v1P2bDzSwomsLM0rLLvTedj4w6QiOhd7eb9Sbp3+5OThQX5DXH7xAUNuLoAv8bCrbtNNSI1qwHRlR+K5nnY7bsl4MWv0KVNdYbAyjcRAuLpkC+SXytvSxiBWAvgtTMxinMDIgpcyyT3UXYtYvSAENdkp93DFM3wKRd1eLHfBxVy1tF1ajeN+yt99sZ0\\\\\\\/y+J6fqFLL6CUFONafoUqR7s5Cs2HdJtsH0kGrwCQbcN8fgr2pvAxjfbrqSkpTzU0pB9xjf9BhMdakoyo8vnwgibYxl6B4qbQ8nGmd73VgTTrsLxELINwUO0ZswFSdcnLUm8hPEHtHBoMf6sCG6n7VBdfwvoKtuCuOixqCneOns\\\\\\\/avcmfyVEfFve86guZeQJtNrUYn45PJW3d5wNi2Ar8ctMuNTuifJprlsUfA5SQTQq2vC0HLtPNLJphUDwsN23OHSGAYOt\\\\\\\/A2skRwzw+wvbNY+j2OWiH5Ya4LPfQFnbZ5XIXKa6Vvnkyeen37VgP0hp++C485md4PiL1tR8HX5CXH0ziZ\\\\\\\/JGvG5D40mma\\\\\\\/1\\\\\\\/CAt0Q15rpckjzltdTj\\\\\\\/gwAXrUA0KQGG1GyvDVC2x5K7vvBo9tLlasH9wfuVBQDNrtmbvTZh7CQla3RpdssLW4DRnylaRYn+wyP\\\\\\\/7BH8mBX6AGkdToM\\\\\\\/6ZL0W0pmoQq+6HDdfQXHtV3IphCn5HJ0I+\\\\\\\/1uVa7E8E4GFxeQgabRuJz0bOJyROx5U7pdeO5\\\\\\\/ddEFZjybmvlqFudBWuoBeKVPVkxu3aFdZnfaikt5F9yrosKCdmw+A40gPPv0687l2flf+4h\\\\\\\/3mjCYSbAd2kz7Lr\\\\\\\/FKBjCSf9KSzFKDZ87ve0jAarm+mSHn02NiREq3X2xZ9NEpVRlxD6+TIukr8nb+Cr31lGbWs6u35WcZCrB7AU6A6vnh4qCtgwvY\\\\\\\/6AjncY\\\\\\\/SdDJoJe2fBYZhbvctaAxlcbmi9a07RHd69f0m4CY2+mPxUAGkiL\\\\\\\/6E\\\\\\\/jyI7j+pod5WGl46YqxS6Ob0tiRbTBgG17aSb3RZ60z59vuKMfiU66aYF0vLBc0k5cg9hcqCSDdc2CKY+0UT\\\\\\\/9bBZoo+Q7VFN5lnf6LNh4yztOf1f73JcfsQnCH7ssi7iEo5InINONTBifJXO1bMZRtflUlMA5dRBTYUJn8\\\\\\\/udc4XOp50cot4GvqMW+AbdbU+RhGcdtQOqhCUOijgnKBbhGvACr4ZIMgwuKE7THJhwKmxhWvmTMyWpatDlNqv+0VqfTcbbz7r\\\\\\\/KlReMDvhg6fHEDQ5rmakBUzOKN8dljZDT\\\\\\\/r3Y6ClHtYA6Ypw3P7rwc2ejinl3cZzz4+AeaU\\\\\\\/9dvLceLUdiHCNUm3h7wIYH6Kbcq1vjP+mGW4mW8aj9xITjs06WcsJvA7NBQ6mXn4pQtDphuz7eEPVv3DCRNEZ9nZEK6h8mrGWDhZZ8J5fv5cjPQjxvLQBbp5ZaKibCCO9SnnpY7RrP\\\\\\\/PqWy0hihzbxJmMmt2BmoB66A2OEJG1fQXai4dW+\\\\\\\/k1rPBDLMYR0dcZ5KgjJZ7D5InH1l6akqs1se6IYmg\\\\\\\/R10+ni6itgzv2rrB6HQPSzqO5HvnGYh+yJ6Z9ZuP5V6dq8s2QkQYF9sKVb8nhP2bbMDyvRbJTVzjE304+afOi8S74XF9WujZ2ubzSavb3RoO5eJXFUNSykOW6Pcv1gvPn8AGMI5g7rALISTt0VDWQsE2EZgKAV9hQJj6cYKGLlIU4JETHhJn5Ha4sMG75l9xvFaSp6xgSlvOmbjUcxSx6T0JQ04HJsvkuM8uQhQqHHxc\\\\\\\/c25MbYAjWos4ihTi6sBB2vwYUa\\\\\\\/HfpGN3bB6oF+M4BPIrDfIMdNimBWMPE6kE23\\\\\\\/XjlkoMTfrLzr6QmJQ6TJjf22JOEBp+JCdcr4tlMH9l8yaxNwGeH+yzVyxfbgx+SjEdr6hiUj5ePY2OhgpH3V6W8qIMMwJCiEkHs5uAgoFObLxPPDdqh0fm6ISx1bUXxB4CqpabdUkZHSbi99K6jVhtgHwap7xf6P7WABt9aRIRLs29ErYhmIBFNYVprbfKNHF42fo9JW7T\\\\\\\/PzoLOzF8hihPXJGBuaj\\\\\\\/RqaFmLuKhW1bSLq0eWBLcoUSO1vhEEr9EL\\\\\\\/0GaEK0uuU4h5dUAmIxwyhBIE1NLPimsPvZTlGGxPL2EIOSRBStT+o9hA2nFo3MSneN0PhHoz7HQt0ZsV6GrLbGT8hLyIm+Fb2d1nB5GizkjjbAy4G6b2x7mgmvwFNc1nzQh520lAq8kaUKArboIcUoJRnAP6mQsl04jPAyRDnKY4sBWxBOG0VBj1SU5rfj6EDhic1mNKQduiEi83Gm2uT9wU5nlMuIARIOWjBhXZS6BU4A2ht+1VvpK50k8IcPY7+9R+CS2rGQx3kdEH\\\\\\\/j0+YrwmKco6RGft+tAGF8qnhHT8xZM82tG\\\\\\\/I8InqajDlD8p37M79prxBp7ndVDia2SfJ3VzTjGE935mBRqGJ9WjTs8o07DPhQl\\\\\\\/AGIm9PoVnWWhvJiM+tJuGjLM9rqsLgXhDbOpkEqHU\\\\\\\/s0N3RHKsCpP6LQgE1sZY1ZoHAyV2uvKXguwzUfzlYidHFcEiqlfGpXHqUavbngpyQmuj1mxcmh0\\\\\\\/sJ2Fq90Ywe245IX9sV7VzxDejSLeQCgcsVsnxr4PiCNUWpRYVlpxWnTWwgs77ypa0Dz16Lsrtloz1sfUribIvnaL39EFRPM1QceBHHcKF5T\\\\\\\/HAMaqNE96U48EXreYjSZHmIZ0Tey119lCzllb47RqH5JAijy0V29q7mCzJfMtK09AB+li\\\\\\\/ImvUjwvFwX3IM5LNJ3PfZ8cJuFPWQ9JnfIVeAawQCLraEeWQkpQ7rkNjEq\\\\\\\/Yc1RB1cn1wEdDx3E743rqE7jTGx7S8YjOcQTRNKT8n6Nh8qWPGW1COUPbvDFKRPDgGYZFHTeIyhDb24IiVl6TyUkP0g2qzQ8lAKiusDmPESIaoFaPO9rclVAvK0GNZvQ8JAG4MEKfCNjkDLd7Cwnxy8o2lYlcY+UTcaH5O1NS4gCOIjwJZ2kzgN8eLuB1VF2b5v66atHcE7nHL6ZUA\\\\\\\/XbtHX0nyZMoHqZKghuC34QpshSyxlrzsEqdy8sivTts0nUXLIRY0mcYxbijZOAMga1YWcyyjn\\\\\\\/sN5iygMMQVMI6jA6SoMFkuPLLX9L37mRen\\\\\\\/Hhonljni7WcrN0cBnVSHufxTEWSPt7RJlNL7fDWtnol\\\\\\\/HkVLRflTrqqU15wVz7nnoeW+MJ85LsYKE4sOJLVqwsUm9eY5hM4kkj8vPDyfNdeaMr0gNGTOMmSufDL6gbrLkTwT5V9yHAvyXRtPBCjDA9jfw9cp7osQLFS71wOINmM1Ice2aJQqebiVaj09xjNCBPBx3M9h6U8jQSx8TXXvAvOP4GzmtDU5eKsxSmNN\\\\\\\/9M9DNRBgg8F\\\\\\\/e5B5Zn6AvpMEYtS\\\\\\\/6pxGC5VwTS1IjN83hvv1SdoblD2N8LOwufoLDTWVjk2mTF5ZFkcDkHSGrm1yg80EuyMXGDiE0hvXk12E0L5FWgcG+kQ1vwDCOm0rtxZk3OMcKPiFbDo3x0002VhHIDjapf89KawhJ8hUbYGiZvt3cQ2LqBRxzAjZFf+RR6iydWUYmY9rCxI3plP79CT1A1vcF7\\\\\\\/k+R9gVnw+iMm4lVHBQ3L3kTrJFEmG\\\\\\\/wms3f7m\\\\\\\/25bmTKZ7j7NoZewlqw5D5WR\\\\\\\/VWlguyFmKcE9yDCkPayxJhvGjPLSRdyjxPA1JhaY\\\\\\\/irlzQKAPlfpt8HRQmdK5BgS8u0nGxn1RpXANlsrbKGPme1xj3mFesK6g12btLWZO6HqSUZ+jOUJ9AH5vKcJt0NqSK6WhTMrnA7Jc1Gefm2WKc6amgbEwtKmWf1Hw5\\\\\\\/S5I3pM5w26Bmn7HHpivhF3t7vnXEuNRIARLaK9GRW3zUFyoAcKZB8IUtxsIQ3z5fl16K+3jb95pvNHDRCbGgfBpjb649QFIe4n0sc3D\\\\\\\/j3K4vYaA1VyKIs4P38lOQ9zFEOafIVazTVeI8RyYzmHopCGMjjwPNUMyVYKnSCSOjSMJOnuoTswP5LMBQ4pr1G0lzUXmU2iTxm3pUCq1NJZAe8p7KnRWJG\\\\\\\/vtDtLcKCoN73eVG9ivoS\\\\\\\/zFLQ+Q8DUu+gklm+n8MjSr1CxnQyt4E5qy71wug8RJ226Q+ykPT5sBXdi3Upr4xIKriLGqJmINSRt6y5A\\\\\\\/WohBqs168hiu\\\\\\\/ZiZllZF3udTSuHdnSHR\\\\\\\/jLw4J\\\\\\\/05Fvp\\\\\\\/huh1A9DfzaOY9savpG6OqWzRgp+vENsvQM5Pgk9bZUZzG5lk\\\\\\\/7eul7DoNyHfXYdpil3Xe9WAFYPSuXPjt3NimmCAt00jTesWHawMxdIeqHaLbx4gBeqJHYRYISrvvdTuxoACWq\\\\\\\/o22WLgafphvhVQw0PfH87nyExn\\\\\\\/mkX6m+z9EhF0cm4Gc1didndewySWWF+jd6tLJjdVFLN1SqYDpNVaOO7gCDS8+I\\\\\\\/EDUWSxpW38e9Y4DU4NmX\\\\\\\/R15YAUPuuBFDv+80T7SEBGHP7rU5nTvHSQ5zo2jkVUHOredO780wh1Ahn8M7mNlatOll\\\\\\\/KkIBGWRoeKRH1PIxy9b13WhkxxvjGDbIvD4lekRxISH+\\\\\\\/6AvEdxAxTykTW9bzii9MlAlGxu2iVcVMyGdUQZ5xHTPNHIohIchxYVpCWubTDZ5Q+tUTBtC2UedixfOUewzEhitdLmNq7SB6xriXNCJcYWy4gOtvr9LRHZBH\\\\\\\/NxXL1nWZZ4sxZ1DZ6cTWEOLCP7oY\\\\\\\/vZykzT0teTG3IJbkpaZo5udba8WffaQTohPohVDKURAOoV0SzMaPJmiTTC9aDBZszdvc8q1wzbHiDoE3Nm40JRoTVWsTb7wF8mOSKDgm9Hv1x++go81rtftBnSum7bbTrTg5KE5xfJdSdQFB+qUrfzloHSzOlMCNTVapqI7e6UZJRPQ9cFZWq3DYlrOI6M+W82ZK12P5T8esoY36XgOBtiqgnNZIjcW1YcGhpx0i75U9iGfOO+W8tn1QyGQXR66wU\\\\\\\/fwXF7o3zu4zlvxO1pKNYcCIoHWR9FK13SA\\\\\\\/k\\\\\\\/HB5JUpiPwAi2WNyo8vRHDdmrbcKw7+wnYQ5T8iYjUM2BZavi3+DM0MOI5Ltu4IgH5VLG2Nyeos3MSFRJUCmbysGy0aq946oSy79CG4leHQeGE3zj70y\\\\\\\/3JH6NhhI250NC0VE+LnWU3wJR+qf+7S++uHOXLvuqkr0oiAvdFtjKPKGDkhDME\\\\\\\/ZOtpssBOnzQoWU04JqEsGti7xWtzWiXUS3AUP+zbQ837O2Dkv83sqc5E1aaTHVwsEEx+6VZttZURvNmPkOGf22SWsUiOAWekWODu2do4QGZDQaLtOjjVoNm2+8EOjm57cv7Ck5gbT32RWEYNOpCOmpqYj1VwcJs0yxvBn0TuKczRh1rlwZgBUD87DAWYiYnCBGUN9pAgJKyH2m7ddHhsjFbni3rn5T9gdNMmSuB+T2+maX0bPCtVM65XWCNNxCeg2vwBD90TYbhA0hiIUTJ4BycpshDFfGlbUQTV\\\\\\\/bQ9WShI\\\\\\\/d7VfuHikzncxkBJlpv50LEqcA+nr\\\\\\\/8wdSej6J\\\\\\\/S5xBBDlzSGENb87m+Kt28Mgmwpo3Jvl6eO1fUqk0x5QJB2RhI3H1fDzCCFQgYZd7d3u3fYvRgXxEs3gN7SV4PusfdetkzEyiFRH6G77qtdTS1fEoKU4e3L1nKMf3Wrasee0cE3VWy8TVc56\\\\\\\/Q1CLzpWW+sZC27fpTbA8rPGNfwYAUBFERytJo126EmwPu65SnWUbtd\\\\\\\/HgfZLVb2QoSQ07t5ZW6ylMs1lA3Sh5ruq7ROtYikp4Svr0FWeI1iiEulpctpA4aUedueZddFz7QnYD\\\\\\\/+0\\\\\\\/YTX8gmvZomIdBYRcNsp1Of+JrZoHPSGSllibBauggjakhcdxj82vRZw3wSzqZBgdwXiCXmLLDIoLFfuVgzamslpyvkYxtZwhUVFu8yN6CI0ECjOZb3Gq5tRhRBSutWBzRn0Ta62gK\\\\\\\/z3L4TVKsnUY\\\\\\\/Q2pxEC2oX4uNM2FrE6t8\\\\\\\/tpBST3jGx+Owwr8anJQNxkki\\\\\\\/nMaCe+I915OW5I\\\\\\\/BZYGt47ylUAXo\\\\\\\/GZaldNolW\\\\\\\/5sswmqIwRazZWkN761PAQqipkU6u2hB1gOnNGc\\\\\\\/\\\\\\\/oaIQhoxKMCuLLpWL3LTB679OMoTeRLk4JpUNn5S47K4UW5Rn+b+TNFMYkPfhnZnKO8JZJpBtYuoABo7aKm1mWYkOyPc+o180T\\\\\\\/\\\\\\\/\\\\\\\/1oPLtSredCS2AJSpDsRBrwglxsIqGJTr9j24Mt8kU9VRcotN9YAt5R54XG8BMpW54QXxkrF13yKf1R2tFZ\\\\\\\/TTQjBsUl6rdeXXb5PLbH\\\\\\\/hDgsVDfxHaq9HhWY883UJU5TCA0dyqkA2pCrPxrbXJjoaxeqLm3eQ\\\\\\\/RyVk\\\\\\\/lslDn9auuo9P10ajVHQnS20Ao3UQZqPDFFjHfPww9pP8Y55kpCD3sRTxEh0z\\\\\\\/6ZdIgKNFEfc4R13vfQGemGxRKUU6SA+ZdN+BmI6unzmO8uE3sn6Ir8hDCpiy4TfdwTHg+I0P9Ha2W5cCUsUpBxyI52LW\\\\\\\/dW\\\\\\\/yY6oyuRNTpw+ia1ArSU5JERoOmwAz3GjgHX930AU92JE4twXdziejDbS07geOd7AoBI+41RrD+VY4fI9NhLGoRiZ5n4n\\\\\\\/iqjv7yyjAvVKU1KZDu9aLZWYR+gwENQY0omQ1EttkCb2NIVwYgz516vh0CPTYV5tdHTroRGgyJSGvyN2ecX2JauV\\\\\\\/FdbfGWwcmNPUgEh6U9bcMZPWLU9Cx9zBcaFso40qH1z7VlrSc5zygiTDK\\\\\\\/P0E03A9IkXbAfXxD7c9LVuUe\\\\\\\/vnhCZf7tWPg6JCdnF7+C0ie9Rb23y35wOJYaSlTmB7l5zuSw3bCqDF8f2b0Gsl1gu3B6KUdTpntW2ZUA\\\\\\\/6hlfz6ktweyjpWr8J63cYn5vUYOPDBD7nBDPRocuSZ0eYyrC976\\\\\\\/Ku1SodSjPK91R3tZVCWLRWkCM2CZrLknx\\\\\\\/r4MWaNjzRAsYJo4MYDsUvxXmUDyqEZSs3hPEJDPyRu9w0jDKzTpHOEJofC8eDAaAyMOce8O4m1G5GgdFcoO\\\\\\\/Ul7houAsxVo0O0gci2MIIWaL9ob24v5LgqIdbwou3Vg3Y7O+R4hbmDy1HIUIqOhpSWQ3h0yIe752Kbbd2fKSPUyo872E4E5DF2objLeydG+nSpNWc0xfAssnp4FJUVuU+Tu546ko9JD+yIZuVPx7wxhuHrbm0Ig71ir6vEyQt+G7DzvFh77eZ6C4SOtnjmQ15DFcUf4ukYe+HrLRBtb6AEtKBm5TUC+WhWAZFX+5I3y6j43KqOV9JNGlq3DnJ9dgyqMK\\\\\\\/dw0KghXd8Q6Xy0jvVawa7B6gOhA9RhSSrxPozrin89VeGGm612eRyeCPypmt6xvnkjqAPGtH0WEcUVZvP9eLA1VKGgbK\\\\\\\/2XiC6\\\\\\\/CiNmXiAYwKjvu4EBIDPQf6mWL4L4GsSY0zjBwum0mnUnswgu+modXv8qP6QVuja+RkIK8v4x1boM8OwFTxedXwj+AHIRis0oHsbBe8nEqYW4jgS0z+enJyw1yT0JgLlql2qmsWQzyB5ATMIbScpb1weWmTAY65v6odpA6EKY2MZ\\\\\\\/NeikEUhArPzn3MimVImkXlu41jBekJr1VhsTT1PvfY7ASz9N6QGkgEWio8qIXGDDQQQ5mmRHbbQxP1un9WPv8beBxT7\\\\\\\/75of99tnzq64xkeInIrNDcAQzNDHe3RoHjbQeief2y6cymrZF5hGJCgLPu\\\\\\\/6\\\\\\\/+NGgKnbfGp5Eejqgew1b27V1UBrrdUaV3KAekgWTAS8eHdAzRDAzuNHZwVOyF9AelZbBpagw78bDnDKMn\\\\\\\/ETiKImYqHMhpU6yUmlffus7up4bpBqDgoLVrciAcaYmjnWDBIqjvspSEjIgKbPkScCXvJVosfBXKvaB1U4JgDrxaKzhdQDGyaS2sKQByKCQt\\\\\\\/iCvbodlV3hLrWVfy+DIb1t2lioqZWHGag\\\\\\\/6EMK8H\\\\\\\/p3VWm5ZXtFo5Q0RldWdBWi7aYQvlNlz5cFmKz8t+BL\\\\\\\/PQSh9sADL8FVB5AkQ8RQ\\\\\\\/EZZ4t5fFF2\\\\\\\/a2+y\\\\\\\/bgobOIMXOh45uVSMqKYTlvcGMXejXtM6L1sX9bBd70NNyVQX8EFVXWuXG5DwOHw5nF98VSLEzb8jD0xzUC3ynvQVJvKqT7a0nyUeoQBAaCko62G8qf7OpewtJ+z3qjuN2QvGzy9zMW0yHForaHT+0OWst5\\\\\\\/8s3EPhrqR32DmKGq1gw4H6xDy+PDzS2EAbjpNN2MlRqN5arRV6YUrlf8YwpAhq+\\\\\\\/h8hrk3cp\\\\\\\/t0B+Q4hY8Dhr81shqRtyAjG2OzO2IuE2FhdOK0b4jyV6fQr7J6Jiza2hlU2stYVhyOgqUKcE2hk\\\\\\\/YQm91i\\\\\\\/JJxFn8lP9aoRfuimmdCI0Tpg7\\\\\\\/5NIvsRopsaGW3TDYoKKuWoTABSLeFT1V3fhGbpYLeFB5yj0VuUId1kS4Eb0VzFhwK0RiSwEbt9UVBpkNQMD0b0wYwdUEfafMWejiqqhVJlsOSSZPNAcW\\\\\\\/cB4TVEIPGdBr8p+sTriGWJeHVMZ4z4JDO401HZ3axAfyqDILGYHAyRaZH594P7fpSgDpsvgfuDN\\\\\\\/QRDkAbHRMYSTejkOsEYWs0au15waDJ9t+eVyN1PruW\\\\\\\/TIfo\\\\\\\/VXYTcE7flPg7K4dWmwmRrfoGZmN4VvcM6sNjzb\\\\\\\/EYyhO1rSw6ZMa48fagmZyUuoO2wTJ78jAHiADEcvOmEnnDOxkrSQ4HjZvhmHaBsGNgVN5FOW0sA86Idv2OI1USBVaAurF3pB4BDtYBx7cfMO9nt1jRjoabGOgwFxIU1xYzN7wpCu5ZO64+UQhnjyXc\\\\\\\/gWxd+cRT3+Cfgey9vSJYA5mwEQ7hfoGm\\\\\\\/IoPJOfo3Sm1efANaY9obEP0ZNhFcQQjV\\\\\\\/BaM+8dqyOLEjoWouxiXlJdZn9x1CZxpKkkjgW553OnpofeSvCFdNz1DuxcQ+PnyKX\\\\\\\/2hClonZzCAJEkVobPxm06HXfT4o74Pcq9CldPBpz36JZlRLrh9h2wde2qnSfrBCpOQX5uaShWbKdW8YpiHCpEpjQJbahkDq1qicdzZEvkKO5JYIM\\\\\\\/+KvH5Xa3kLgHo5bG1uUOspcv0rYeSdGvoKzIlv7Z38LwRiHColLQNea6liTu66smvYzS8rlr6stOcUH+bCcb9gZ9ZSrt9x07QfcNrim0ZsuFq7gv9EGjASghJQxcVhpSGFuZ1K2BfVUhIHF+YWlQVz2sKlb1BL0sI3JgAmiIYMcgLB4wJaf7K3n\\\\\\\/lfUbD\\\\\\\/U5OO5iPle1IaRhKAGQ0jLvAhQl0AaQCrJP9wTkrT67JeeBsnsUPIGhvB3cTTynKQvaL0N+j73MfvGeHkA5N38MZbQFFs3Hzf1HQQjg4waSYTr2rkssBMyWWBO3QHd+idjCweea3Oe6gLWhnSSnZXrkdvVEYrFzpB0aOCKMvsql2GWSORMKyIrlE1OMcuLQWMW77P7kiCPBbSpc+MHhMTFWvs3ijcA7LV2tqVUYxRNfqIYVGulQ7pyeQMiKb0z8qdYAD4e66s+4rVO4T2M+lhl\\\\\\\/NUpt1xzu+76lIKfi1wRV2vAZh3mr6E5sF9XnLkh2Lhfsehfyp7tYh5vyJ8I+NHS05+CFr\\\\\\\/vPYyD9VsmBDN+wzv3h5uRwkwS1xa5DgoWUj+zAmDmCud0iLLtfCcngCo0MwN+iyRjkgGqycm1CafPY7PHYWiBCuAduiJPZh+yGcYD3Sbhb+DJ9YjSlvwwRb4aEYvtd6jEIMOEjlCiO5OpuDlwnwQmIWMeRU+avBDp\\\\\\\/jNeoKltbNJb4t90yH9pyiXcAOXOc+4mdEPqK8dAHMpM2sP9UjOaLtdspfCatYnGJQpO8AD+D9DKTtS\\\\\\\/38WutE9oTOGwDOsYUKAvgyxYkq1RwdxaENEO7+lQsIoo4E5jx1auQJ251AFSOBIj2hlRSo04ZNCoKFUyyVCy4W7n9nvGXu\\\\\\\/KOKKD0p6LHZZzgF1t8Ea4jftUo7LYorEsoq\\\\\\\/xK498MZh7j\\\\\\\/EJMVwUP8zfxiVuXhau3+ux3B9Im+PIFO\\\\\\\/lV62UIFLa\\\\\\\/9sXr1LhITXCFsUWF8OZaOa7SfP6IWX+gNbCyS8aYAlr2RuW8j0MQZn+Sh3R7x35Tj1lDXQK8Ws48GNbsm3PM\\\\\\\/+eTdK+VX2MCevy6mqMmuG66h7slTsm+5JFyELQ4vrqZO16qB1e1\\\\\\\/c2yBS26hXVKZKArI3Gh4L1W554epbbTXV1PlKGtuW9Wnl8Z1uo21mPNO5Aha+6AUwj5fjtrvfsSJyPs+swTc1e\\\\\\\/VkTeCci2\\\\\\\/RY4T48eJMDvTYn7QChNAudbC0ApecGGC3K36uRGnC40zJa97DTybS2ne7NUbK+FbbFtb\\\\\\\/n+DxTINz2I4l+DPJg6NifKR+0B5S7\\\\\\\/RXrUe1n\\\\\\\/fYFGVvnXF2Bhsy+h\\\\\\\/hoLNuV2PM03gd1+RLJ3jQSe8su7Iv\\\\\\\/CGq7Cjx5KS7yGvLK7TjS+pu02BwhhzuP8KLrshywHlS7tvBA4yXgq0M2+h\\\\\\\/TgFJEB5AZNEIlhxoS0ARS3YKfvlM\\\\\\\/4eD8MDYKokwm1b7rGOzWTNQSZzfR8Urg6QAU9JZL1geRITEFSd7racpJtchZortL96xgxoAJKFYtwhdZHH81nxmWdAk4+SziZRmLQgTPAaSCTlaVK1aXfF3eHNm1XzkAP48O5fOOjBVkMbBFUCs5o7Um7MY2l4O+PccAMb\\\\\\\/rJtr0oMCPmL2dQntCoVmRCkqQgVePtuPbXb8KiukYEn5Ik6Itf\\\\\\\/vjMZYjAhHJ1xjwu6U1WiZFXUYYQYG06CDNHRVxhx73XbRY3Ehcgx1yyPtgrHlD+OjZssFSItJZs1CjrtgPeWQCq5IwJBkQmBSF9lGsBX0LWVSaaBRvNF94jPh54I9+MCshj9qJNL1GrXrEE4PYhEnFG8d9cWEsNgd3tlqhM0Z8uA\\\\\\\/MurSy4yjL+j91iHWiMN0vBvjml3Qf7hAGdAU1GNug3U61Z93WjO12+i0+\\\\\\\/Ifh3H2Qdr1gdjPAc0\\\\\\\/DGZyX6eyuL2e0M0SIZsRclBZVayP9fIp\\\\\\\/jrv4Rt8aDIiwKbuwiRTXr1eNCo0R3GpMhg2MtDmjrqdNOeQzdX41If\\\\\\\/C4X6B3xyXHI\\\\\\\/iejT4kptkY4uTAXIIv8t+80D07a9TjwsYAbGL1SVQW+ILwcTKmIGQDbMLQki7iWd5QVbjEW\\\\\\\/cZGdWgfR+K7Z82S3a49XmyLucdhXI87xzH7Asknn2fOnvZbwpSvCQON5dJWyepMW741BggxzLvs8OR+Tk22341QuEyfQgsM\\\\\\\/\\\\\\\/Y+Y8cYDK38cUMn1yybp6hYH2UHkMPcibQE0pT9DnVvbM9plKLgFGB5duz4ThcrpZMPaVQK+LRpXSUuPvBXc0agr2V\\\\\\\/nPpoiVB172H8jgcwHolxcDF+3j3vzCh7F5ObmAHVyYCMh85QK5L4Us5EOkkri7Yh1wu6zTL9VWJqQOOUwgqWfEqLrOvvDFmI3YCEcB8s6amYKFPJpuX2hCHSauEqCWJkxxIJOac8oHen6ppNmywtDKmwvBh0Mp066ol\\\\\\\/iKiAAUahW829TIXBdh5OpzZk9ZkHKfXhk5exAMyQxl48rHzBmM58peKMRBm9NB1KPp9nvehHfk8m6QXbGusWU64UfgXGG3m3UPWilcavy6lfRJgGA4uQwcgDaH7oh0K0Zhbm5eeEH0+qIzP+SH2VtElC7DIM8qhtzTVoWAk502R6lNNGgX4jfdu9\\\\\\\/MCMAX\\\\\\\/R76RUONkTximhMNnESylnCrtQc21Cl+A6q23VvGQcVU7awtXHQAB8eLNFAs2NS5S3hMxxut5LVVq\\\\\\\/jkH1Vo8nQIy\\\\\\\/IyB4IouFX7qCyynoo9WbxbRpWKWh78z9Utk5\\\\\\\/6FJcWxUm9i5RfH5I6mUS0nIcFtPydnIEpO\\\\\\\/BpuuNdzEeaKsJ41zuvhCfz1vc340j76ID015aXg1MnARf2iYJeP6MTmAETp4Ri0nW8Jk5XCUjVFCx1CfWT6BojP9gngcEfZP53aivMpq2fKhlRlOiAZODhUuCqcmQWlSMJZT1pBOMKCM3BUQmPf6HS4IS\\\\\\\/my+yRC+Wpho0qRkql5Ba7Jj5weElWM\\\\\\\/nI9VsHBnh5zDqJKenj87atR1QiB26GscxdQuMzzw2mkipb+FBH5XpJTAfl0z19ZYt0m\\\\\\\/StbM0IbGJgGdZoMVCnXimAZrRPDwcGifpQBtpWNRXEG+XBAVF6VhQzZcMkJO8MDFGtBynGTQpkcq\\\\\\\/4PgI7mVlifYL6DT8XUBg7jCmzJhxfFXn5Z144Gf8vlKR6hJQ+VqgBNUkwagf5pfTsu\\\\\\\/EFJl5L1iNpCROd5uCigMBnu+NjY9IsUzrm6tJWQyS7DKdDj1jKRWA7\\\\\\\/wmODtY2BIGg349\\\\\\\/oRaZ4NLxjN8GbBGEPgxsE4\\\\\\\/5lVpsqwgUMxFW5vRHdeWkU73XQOvR7Y1At03qlnKFBOf1EQE\\\\\\\/mK9Lh0pGrVQApMHr9qRc6aUt9hMeel23q4qz3EAxIJ4tyokh13L\\\\\\\/\\\\\\\/8UEJAVh2nDw0MNIjznEUMbiKYQ\\\\\\\/G5hIUcmOTjKz0BZNKDSfw1maBJM2l7RIEiV9aTjfHHszuIRUXzmFUtp+X8FXj7lJ\\\\\\\/Sj\\\\\\\/87w2bv3HJvPLul1MXX8G0M0hC7951Lqx43HVZ2ED40LKIXRjjc5amsP488ERWMgM38IXmElNr5U6KmUf6rr8QUyuEwb8DX7SWp8mrWlvO+BcW1jTq75wuMSmEMT4TE3s8LKAtAogp3Rs9LDo2\\\\\\\/2TC1oGm\\\\\\\/PEwshRer9MqjIhGS\\\\\\\/Umh9zt3rQJ2c0QJMykSmrPIWnGaHJCeq6XPgKvb2KZu1NykkcuzxshJj865FPl+4yE\\\\\\\/jDZyWbG9J86UAJlPwONrad8UCekK8qQImKMezt7KovMELSKyPYNWAJvJrxsQcFI+uSqcdMHlRGOdAb0AhlMr\\\\\\\/mMLwedTUF6f0b7os6ZI5lk9Qtj1WRLyfWP5998wKalXwf3MD4bakNtC7joMjzazPLB\\\\\\\/wqYr4JpT\\\\\\\/k9sgtlyQLw8fkbeBbEYQ03B3KG0gPJ+HphOdGsqOzq\\\\\\\/oaz9WRAlMQKnE60EBE6b24ZdLvhx511sHiRdbBh6pn9z+2GYw\\\\\\\/RZJ302vsNhZiL5kY+P9lSE0y2pBZxQVH9\\\\\\\/tbQzFL5sLoO8x5yG2\\\\\\\/Q51Afivsxx9dexc8Mt90z9lx\\\\\\\/B\\\\\\\/O5tGGcFfx4HA6za\\\\\\\/rt4K6P7f87qMf+L7IsrlkShM+e+Gpsds8ymPlkVzSJ9CzzWTP1s1cm8bfCdoJGLxZ2BaNcfgaCvdOxZV5u\\\\\\\/iEqP5sFLIUMwpGRC4YNofZanYI9X1rKe98+nRB\\\\\\\/vJFpwXcjTU6L\\\\\\\/IfoocC5kn8nzzQjaAbzk8Jr0kOG\\\\\\\/+nBYfN+lCdTd\\\\\\\/Ck6cX33KTYwP2EBKbDOcbCHHpokndwRUKO4P2pq6Kul9kii9Sv125ZrGBSn2MLhQCAkUSATdfW6bGq3rRVEWohKTGsvX+q4kfwsH3QdVV8Q4tBn0SU7qRSKzegqGVcPlNomi7Op1li0qksz8rile\\\\\\\/ZxvaSKwyTvq8zr8aLK5Mf+I5jEmWHP6MMFdNV4d2Z8wVNnMM7PgfwRk41eqZFsd60wlVAnVIREAYQNQv+GbPeVjVQ59Y7yxGbkjg8TRsuiFopBrNLhIejX7xxvCNuwPVOMhKKhiJrb4Iw+OGP9fKAaXxFc9UVI49DhqN4dqrXOg2hZdizUftEMt2Sb5fb88lXbmvsndqFytbcliTCztCex590rJtLo3TIGV3mWZvSds7sp7KekaCaleTKLzhIDVBOxn0+t9eO4V\\\\\\\/de6X0Wz7lkXixnE8okZXJOjNf72QjFPDslL\\\\\\\/F7z6bWFlrgZVBCFxsfK3B+ptCEkXRhnE7MkbCG3PAQ86hG84eL4dpvTESXN2wRp\\\\\\\/n\\\\\\\/0xO1O5y+iFndapuQRWARtuhdsbWd1xuSAYIcivWEOdyk4p9Uv4SXJbqEg3ragCNCh42aAk1upBr5KkK42HhajYYfUf7sQl3KFd9d\\\\\\\/BWvvGmwSWwFIJaC5Jb78eWKRsjzZvl+se84VFV4wq4ohXbasT0FFsN4dslxDfoj9mjO6ilhfl7JQ18xlYOliskgmfPRqIe97DB5sPVFJY9KhUb7ZraU1CxYrBJU0siXOWWQ0x0PQ3bD3oOCwCuuOJdMJqoUeGu\\\\\\\/D+o\\\\\\\/UxIxYm7kwCnLiLFHHa+HFU8CghZocr2XcYGkdQpMI18IyitgK16phpzk6MLKmDIbpzdWuU0op4BwhLN1C\\\\\\\/K6xb89HN5z0IN1sEaG6KZmwqy2R0DMLVe4wAnKfx9uZ+TYTWn8\\\\\\\/HVARxQ5UWQSsf4vi7lofy65Tpca7qXm4kmy+0P9kpeaLY71SltEr0XU3crMyo65Lg9TXwEbt++JHOAw+LzV57Q32akfWn7BR\\\\\\\/9V02\\\\\\\/RMwnAha4uUGj58njTd6oIzSS6NMI6Fbgw2vCsVhYtuR7Oi9QSTgnrcWDCovRUKVF3Wg7DVL4A8m\\\\\\\/AiWb6Sff1laI+zB3toad3RM6E28LyQynsfcMdpRzJymHGDlZvzPoTxaBMCC5S8iEFGKKYD79+3eSh5jikdkbn1fZ0\\\\\\\/ZVzfiUi3rAAkGi4rJiBSe1\\\\\\\/HnoUpD2vnMg1tjAcLJ9hbQORDF3DkKDR1GhDa2oM61AdGi+8wU9GhgAZxiHbxHN7a\\\\\\\/+KLmj+MuQbrOUccducl8DdpjNS0OhAmc8LkCiwShLoJgQU1RNe2hSr7aJanaHlhRnwM2sAjolfGQFO2mU6qNk4XWEfqtgBHCSdr2UXYjSjLOLhJDKPpxs38DxnrwY5xd2TITmKI9QYK684fZyHU5Q9a3n\\\\\\\/\\\\\\\/A6AlcpxQwT1zc5mX9abKAUVuh4AZpVROaPFSB69K4jSo9r2g6ng0QC\\\\\\\/8AaJUgWbLXzPgcNxM7EmkUy2NQu0QdCz+ZG30LcVzNATiIzjzB1CjvpxnvcqlXnu5\\\\\\\/Pla2lqNpMmBFwouLnR05Xm\\\\\\\/Z4afRG0VO9eMStYCYac8PJqYhiQ5KT1AUuZin1zLy9laO4Cg8wyw\\\\\\\/PgY4TQK8Z3bTDmYUjX6CouFvtV9e8MeFWK0nVuAW1fzwYGKppKV3kqltIYgJugfNQRIYPxjRKJJUsOLRchfIcl+CJN\\\\\\\/uyXsXEbEhB2MFOcLZmZHBmPMDEKiq+k0Z\\\\\\\/gTggsEJPW5km+RZqw0ozaEPWSP\\\\\\\/UY9oRjfILJWmNltcPcdlvJQ2ZSKLNTQ7KaT7fDaF21B9\\\\\\\/FvcQ9ePHCzmJfkqkN5Yp\\\\\\\/wZABhzKKQmZ3ErxpEkyLID2zMalgaRJFyU34CqwhLMjfi0+PQm\\\\\\\/wdprrtUfbLmQFtQCus4q8PA51BdLN8gjurzixYXQk0QPumkVPFTo1xIREDiJADDFqgGKaeKKkh\\\\\\\/No4RhaxyJN8tjmr8d4myLYPMlfmDjz6tPmtgqRIQrlEGSTjrH8jYP5LHMfjJui9JM3hj8DX\\\\\\\/0\\\\\\\/hZ+BJse2HJrLmI8Vy7rWp2fi7VZgcHzhK1dOJ1fabkuC6CR1+xgeD2Jc4kWTjAXBzWC+D\\\\\\\/ubXedi8zp9NFIXlJJsv7m4Ydto5KNibN9AT33dyjN8vxxK51mPOuaV42UF2yp\\\\\\\/f70NL+1TNFW4KM0xpmi0qDVl+pRi955KTsw\\\\\\\/at3KSR2zMOgn28A88b+GpT7aD9n1FtukF7lp38qf6X30Vv7cRk65lIANwMsCuqEN277BC\\\\\\\/0jjnFdgDzHd8DjomP5O6dDv9HWwZ+3Wnw7c6qqfaeTBj9A5A3qEl7upuYEsecIciNwy4x046\\\\\\\/az5AkOSBF6gVRVUbyKNYasveEJIPqtLVUcVIqGR6kqRjH6MUtLOy4A9t6d+adYNyVMwLpbcEmJMaD+2HOMIb2a9o\\\\\\\/uwmwTYfHoJe+WqhwAmIoKvNDSz6ySBZ0lF1ySGN2blQWFneAUheRt33BlGfBKE62t8gI+5NZe8dsR9qx6eSUCN1\\\\\\\/2YjhQG5BTqOs+VJUXln0nN+EjyfleIv9ktUW1RLeP2qKRjUVZq5iKz+9t0AhRhv+\\\\\\\/A8M1FZGsfho9NDJOvYnmJ0sY0l4Culy7JxKZXmNbbNUOyadGA1ixMcnl8Kb1qKb4EYD1V7EyAIvAeD6IIYp5Toc4+uoVLZnzPrbjpZPL9248Mhs8bB4ZfpkptbqxE4NtaCQnBeIpo8EMmmOmxKAi+TdaIR1BKnyi65K+mgdZpf9oJdvOrdlem\\\\\\\/gfCNjcLzSxujLIaa1pbet1pdvOVLX2kMMk44H3QqjIieyhk8ZZ19\\\\\\\/DLnKFfG93T0CkmVCR4vaAFNNdjfSsb6fzue17HniYR4xjfEjU0Hnj2iQU69\\\\\\\/tt7rOko3n1ViNEjhi61FZBR6IQK5PuP2t3O5yZln1AbZkpOdBJvUN8B78Dy4WBQs4piKIlFaeqjUPcSAi3umUfuVXUWBiI1QHDdL\\\\\\\/ZhZZpWpamF0PwlB0c8dxX7S1CoD1cPuwyogBlo1CbJhuaEDlfVihidLVubuOW63mYkPprjHwWJZFl6cQg3IewPxJecuxtRtRkm\\\\\\\/hUkkuD9DaHVBdbLbREISrhR7Kxsc4Sqh6WGKbukGJH86tqafOgzt1lQcm70nPNbicyjHJoEynAZ1N1+IbBLOz1ICzptF9kNl\\\\\\\/bDiZhpVe\\\\\\\/86wpsS1DDytbTX7YBiX87IOzgOntlDn8hF\\\\\\\/qMH2TCkc83kFocUdkSdMnm37zM6St6DK4s9lC4uQI+AczJcvKCN52iD1O\\\\\\\/8ndzigeH+zQ4sFqK7c2eNNa1R0YtGrAePj9BwP2LEbFJl9oeGRH7xFsMZDCLWEGo\\\\\\\/JA17sv\\\\\\\/WDNSxJ8LxrOTXkoUi\\\\\\\/n72BbXhMOqcFY7mzHxGaf80CN75FQ8bzvzM8EoI3Nl4UBovkYcIeF\\\\\\\/twIkJuNolRJxMVGxcWuHPTeL9Upth98fQSwBCPwuSBTckq7vbX+WTqIExmjRsyxSJ4oilx7bovNn6CUTIVplEsZy0KN06DVEPi31Lnv8rdPhe98nwWd0\\\\\\\/mtfV6yQ3zDz342EaEHIdvevmz\\\\\\\/3z4OQnWp8WF2DEyshTOBFqrnPNFs\\\\\\\/CEdaxXnLF4h87rP9XNbz3iL4cQfiX8VbYz3D2D9mrYOR1aDerEyb7ws0IN+YkPmJJNn4pgOsmfmNqAhZxKjVItAnNbMLJ+9Qd5wr45RFBCHZBHUKLhWlb4P1lZsi5lBElafyRPOmKCbe6YkR1aRf+W1C6EN7BesGlVKk9jU5FakTvpAzHFM6rPikg93UQPQJ+FdYvaB2E+gRkwHz0dTCLGd6V8Dz0Kb3Qe76Yc56oxMAcuv5pLYH551AMh+l1ASXwdVZq937CxwnrCmjrWAB5FwVno6LhGDbnXrlTUi3zRKOZ\\\\\\\/QB29iiKljjWYddQS0dMbxwZHbzbyrUgNZh7fNr7sTQPAN6I91BTa1jaddqgoA8MdgNSfbL7Yhzm47XpT3p70JpMFhskn6\\\\\\\/agGvRKR7mxMpUtBIhWknMo2qerxEel+S3PF9zMADcg8fWti\\\\\\\/htrn9CXZbiZnSjU5zzj5TmJsK6YO9+HUUm7aA42rz+DmRpDjuLgZbThWbWzsPVnIGkZzpMEewyZ5z5ihXdj+nQxqAvK\\\\\\\/eK7vhySj9t4coGemTHA46BOUpa9dAP4GuLvg3C2ejxMGZnQtU8Z2CMvz95bvc1+ezT3kd8rLw0ChQTM6la8u5glQNvMAJB8biFh1mLbO+FYoUAhO\\\\\\\/Ug0XpZLebuft918p01MsMZDIxrAENKXHVsY13XJHAXGsq58FvBpPqW2axlns9XHKE965+x3zJ9bocv247Lxnxg6hoI9hhs1U4ZWi3mRQwKwjXR9s7sLdg74Se7+\\\\\\\/w3JWnoyEh7IS0lJEw86BctuEZ47uV4H95aMFQTkHHfOKyJGZTE1xWYzqxeeuDZvdkrft0WIixwKSWYbYV1WwKd0lePgd26jBEKsUVc6AdaY5gjhKWkfstWKRhYrS8KETNNZ3CbMYdMacb9H6AOFDeTQ4UiktJdhaeV6NBbD9JfS7gVwRpYuACYGeWIvkAYJIPFcjEV78r1TyUN3MLnYPxi\\\\\\\/6b9icfafn25wsSITWXFID3I+lR4gCVO8rNkruCR4wbia7mzkrDEnPUCEWbA2+LVPscyZRruMicZgYdWIiaajHaLQYNOCXzG4ETy4wF0eWCR3xh3cRijx8glFjRujrk4UfTy5YG6MLIBYYp2kyJUw2VVVKZsk7FD77JdVWCZwUrNAX3\\\\\\\/SXSmfMhRfpdUPKNKBho7Ra2aNA1pXTtGZpa6x\\\\\\\/M0DO5qZZMq24pHZITutJFY0l2k5N6HbKn7dq9vIEc6O93UCBkG0pVWdfFq1RaQL0He9qfjUIJtj2fxevja7+p1pQqI3yGLfNWnkvw2BUqVYjpFIdgDxqk3ZK\\\\\\\/znjCY6gAtiB4vib3WJ01efyb8UWa9wZMV4+2YqzeEo8gQ5GSh5TK1Z90zdDEHHL1WXA\\\\\\\/MO+pNNR74DhNLafsZW0m3YnshC6LE9LA\\\\\\\/y9r5unJAra8xJPikU1nwZb2vmtGwrCjTYv0mNdcnOlZOt2rOIrI718wzBDGcmwPgfoGPXD1cYSiQ1jSSurnnqOK4u87K\\\\\\\/WcTYuhP2N+vT4s8gM44izE80twtC+ra7Oc5Y6r2tQUoUpcuntvSUJ+EIwiE0dvgGCzfdkxwkcGwej95VQGJb1PUgUiCU+d0SW\\\\\\\/J6\\\\\\\/xkoLqQvM3Qx6kZlhUPRY7pYJkI2xjIsd0efwExtNpFcVZ7ReJyxrln8CcDgSPXcBPSV9dCn2RFyIrUZUM2VkfJA7NBF2ZMQFLwhfZ8zRof\\\\\\\/V9YrO9GQC4AmMggbIRuTmc8GSSe5TbOCUlgcOWUnlUHwUYm79IT1JRJrcaFCFFmhTEfuqXH4s2mXuRMWRfRjUHET7+Xe3iGaAW6rKUdvtQVxfS\\\\\\\/t5UhEgLHKZvvs9DspFIvz5RhAQoJFpLzJGE4f7+irg9SWiXSESI9l8F\\\\\\\/qWNL+eNoy7eqaAq0ZcSdqQ1seuEljy8ZPmNgzwVWjYa16GP4\\\\\\\/LQ2iXKOaKl+4zrdq9eOly\\\\\\\/99XLQvxJE3ZUha\\\\\\\/ILkVaqjH2uJvTcjBI9joHYvYFDlJFzVDV9ta7K48K5G\\\\\\\/xjTn+olK8ER6gSCRZLza9PvykAdu5lZTkAHrtvFZQ9hiiFQP5r\\\\\\\/H+BMq8Slqk46IcPO5CxZN9qD4rcwmkPwg9ZTjUI421vk5gXpgWpIPjPszbeOU6tJpBD4R97HBgNeR7F4tV43d3E3Y1McI8ku0JkPICWsOjvPCiWzIW0dMWwsxjFjdZWDNWMHa+nZ+5Ef7MsUVXXvrvDSSLKOD\\\\\\\/DZkcfBSocMbYtD4RRP0xEWGlVeYQhEN1M3duxZ1x0K1MBiqzCZbFEqsz6LovlEBCbgYKAkht7JXqv856PFmBqxahmajz5qMP6oAXSh4d1qlrbaQnwoeqnsyJsL6n\\\\\\\/Llgb6n1yo\\\\\\\/TOsmeMDRC5ng9jbHM\\\\\\\/uMXBeX5S0t\\\\\\\/tutZPw\\\\\\\/umuQ7l2dYjxLlqc1TgZwsELbFw\\\\\\\/oq7wBPZ70UqcgivRua7\\\\\\\/cLRqxo75ZUZ9MAYmnG+mc8HCf1wmfxFSkhi+DlmFJt6jX94WRxKY2D\\\\\\\/9IK+tF3bJlquF9dQYuM0TJQfsPe\\\\\\\/xNJcHOozf2tYiQUoXbSO0MbdH6k2tC0MLFqYwr+ZFXaXKU1zw1J8U+c\\\\\\\/VeKxSSnzxhm2J\\\\\\\/e8CdzVkKltcueZWVUG\\\\\\\/uZa0k5gKT6X0gl5KvafYZhNXaxFyi+2k1bhkYBitmMhwhJFuLVgGqbItwj9wu8m6OCIeYVaIqbWd+JZDObgNewEjXuR2EE9vPlJhanfyKspadkwZoqZzReVDJu5iSiR3MMhdCw15bUJ2At+voSha67efhxLhglpUZ0\\\\\\\/Ve3VhcnNhK9Mn7cwbEUtk3aEBRUXc3a9XcVDod0csyQ34V9HE\\\\\\\/qX6CdDcQ3CyQSOKyjBOU4nBZpfHVsNws6D\\\\\\\/ULpey2TADDC7OKLWjOJLERd9urhcQKQgqILEwgkUYFgES7ZngB6DMIzJiAus5NEe5XUqh2fqKHIXHrlmwsJ3MuBodID6t3Q5xv\\\\\\\/w3RBdZ0PPjZJT6RCea1tBGs3QGt83TVb6uId75zTVvs9FRJatfc9kv34+oFvurcc3ohHwDi7G9KqeRl0sydrxk0Dup5ca3Sxks4liiGgArwrU4\\\\\\\/wWA2A1nEkPFf7\\\\\\\/bDVs8CUKaPd04u6knPAfvEhlqpN5kUa9dRQbXn+wkUIK0x00BtPleDaDALLeOnABqzjsK3lqovAWcx8MKCZYZX3cfd4XpXOFx7kfFCvUbCXRZi7aAB8yHV1mpy4nG73\\\\\\\/iunI+e6lLEedUMxY1ZPDtOGUO4j7bVU5TUksmIL17qrDaMJKJ6PCXf7ajUvzCvTNFkEhXI5DKKxz9AHjMRx49nLpNNy2kwyLrbU1tXLt5Ia+KW2W98+AQKkAvVvIlBL4+ycVaeBfA08fAZ\\\\\\\/\\\\\\\/UqZpj7I9vQTKO0AXSuVWbWj69RmZHqLNpqCRglRQkfEyyzCfEdadChxxDJZ5Hds5E3Y1O1amIWD7ZGgJFyzEyUUnOPUQeUpBBylM7TUxRynm8rOtBLQ3GSTj9FM\\\\\\\/+4KMwtQvo2OwSfGbGcoJLWoOdVtEJgHxHETjbD3m1+Dpcd4plZEqi8IQwNHgrMW5Ml3bGTCieFO6SNoRdUmVK+4DjEkTsTnKNXc2zZzIcsjXk\\\\\\\/+HijYRQum5tDYlUiRo1hB5W\\\\\\\/3raIT2KgObSopTYlAzN2HJEV96Y799PhWS6LampHbhS5Tk3m++99gvaDhSzhpSbevjZ55b2JvoWo\\\\\\\/wldebnhuDuL\\\\\\\/mHYW12BTOUhwhwwbet4KIZlW1e5NpF1oc2SfzYNE61Uu1yDUKQxBGePkrxSGdVU1wzL0sUb\\\\\\\/3qwnetex7GGRZheeMipQO8FJL9672MBUCMvG5PS+UMfeQu388+kjP4bg2Gn\\\\\\\/HBLxpjs9kLOTwE4z7UILMqfZimrOXkBmgncoDoSyZ24haoheYa2RW\\\\\\\/UJMy\\\\\\\/HnS8ePCf8LWNKGaolpgjVVIGgKxcU3735LK3IVvR6lZLLUkU3E0K9A8iJOoQ46Ug\\\\\\\/1XuyRqo3pQN1Txc7P5nmMplQz3LH+ceFmiAbq7\\\\\\\/Y7TT6zAqcoJWi2INaXvI589UiyvZcpbFaP6t6xy4\\\\\\\/ZR959u8TIUa8gcj8c8crgWXBnQo2GIbHjjuRDlOK8H1wBa9X0HTi+rA+V5c0zQEiWtiL3lYRfin41kVLXuNDlJTjWoPzvGOwkgC4gqb\\\\\\\/j\\\\\\\/RxjBS5ZoRE8+TamkvvYXEW\\\\\\\/1DAPYMydDDeOXzN0SzPNduVsOzaiq5WVfa8U09RVRIZ2K0rgwpjr3EZkoSsGIfYsVcl5xt2mu4VckczxkgRa8p+6f4HvJGNyIWFkVeQGtcscMoYLnJlr6bzRLYVQnXRsBalqFh0lFxkCPnueZHAvi4wFsVJ+mBVWinJVoV03fxA1y7MoS+U18qtzKjsVIPI1AM\\\\\\\/pQUc\\\\\\\/a8Zu40ETQeFdehH9tndrWh2vkdGx2lHZM1+0j\\\\\\\/zrf85pFj2\\\\\\\/XEWv2r7ZmN2uHDbjWmv78wfMc8rtt33DY++nU7u+JCmoz5HenNgdmIkrDD39NbtdRRZDUFu1spAr8x5cR1yp5B5lHfZAcpSKdshjSpvUwsvGD2K+0BLBQdbxCJ\\\\\\\/qMMYltmOr\\\\\\\/rVkxffnkNul3x5CwwrUq696Io09\\\\\\\/KRC9xq4nLGaWeB3ZYQ8cgOq3trRPJFSE9F9AhSG8pn+UF5FmXwr4YQRAVpswFu62w5C+WyeZjdWhxSG\\\\\\\/opnxlvBJ\\\\\\\/rwyNn6BXOJZKeGr9EWSNuCJZNiQx76cFKmQzfwWK\\\\\\\/dfU2WW5KShEaxed0CTDbUuSRsbVeJvUKNHSpn7zSVj1qQYQ9PmVW0jHMEVeUO\\\\\\\/dYDA8ZaiCtTOqEHMfPGBWpadKo14SgDG91wZFBAGTTFy+Xieq5M3z\\\\\\\/QkO3tNUyntNwfyQoP0jCLDbW1Vmu8LOtpEDhm1y8gZDYVtqpSl+XnTTn\\\\\\\/AtU2+c5QUXMBH+Ecu6X25JEzFPtOfXq\\\\\\\/0DYVmR7ZOJ7ZyFojU1uyUynOBaklm8GORy2cH+k6j4h9uJ6fxTAo1SrOuep1RfxunMzRE5fhegwO2IEeFtZmzq4unmeO27KPWfUHCZV7OBOz5Ewg6VKYQZiUQUo9wiNuF4rZu\\\\\\\/RlrrVxSVwpYf24XAW8bK9cngpqrIQHQkjbYH+J14ieEDzkU\\\\\\\/Jc2fKMBIPPezPKRZsm\\\\\\\/w7\\\\\\\/ROJjBPIexsy10Q6cksprHVASb5x6+EiiRceabxoYeAZ8nUdXl91L3K8g0X8XPTXj0UbKv+LOTuVMsRwCVsEglkYg88ZWLZS2Asmyil+u0gsDEygdGFiMF6Sv0mAQkfSM7l6PkeVRG4hGTd3dX4NakBIU4Xb+oJSKyjjZhWUpYtnvMRucPQg\\\\\\\/C01zyosC37VB6CPbAuYK5lNmeJItu+EWE44EYWhz5m+fBpvUYjZjqxPBhkWA9Hyt6esVo8CpDXqO47P7xk9Y9SIma+mXWTiyuJRc9DEymbsZQLpOCXKtMdBGXJ5Qli38G6iiemPMpCDnT\\\\\\\/FesAwE3gj4LUrPORtYuLMptChg1UpbQC7OqC4hGekrS51mpwLCKph3j8MxS5s5kzQjPhOQo2jZx+rzzOmfdLWQT4q4J66IFDfgbbi9qAZkWyQLbXK9OOuRllfNEWahSo6x+Ego46gVcv3gJFcgzWERSzZPi9aUW\\\\\\\/GwiruLXcH8CJVUXFFDn57QvDb0zZOFJGZIpzwD2TLlJbDPJrM1CEyXx6bG037C+CBobTHg6KaJlqhYRozCraeLAKup3srChxkIeNCHNFiE3rz43xboDb5u\\\\\\\/EJudc3fHvZYe54xDkXIlo1tSc8N3p\\\\\\\/YuPig+R3cPFBPJsX\\\\\\\/QNRAzS9DIduek7D67LitnQEo4LFu7Sv6hPKtturvpy43zSffBpJQA9gnbEF5M+fb9XEmwbnkq1Dfk8APqBF0h2npE8mnU4FstzwCawwMw8qEFtNjxf98pFq6UljcayahF3fqCdMzZAbGAXveyu1fLIzCaAElYJyFm3N13HAV6pNehAV4hCvugvN+m6cPvqQX9\\\\\\\/IznMA5XqmqRVTZMLjj75YqtHzq36uJM9Gk+rOW62g1kxePiXGpHcF8kBam3iCvWrnOYpDkFNvdUrA1my9IHivempajp2kTtj2Md4qMnAz90zIcGUqMlCGHGWnxc63Za\\\\\\\/OlaGGL5xjRK75CjjIinJImM+QZyOy6qx7dzIGG2V2LyQXlaP7hZVdW1hpSHyj0oAvaUXOGHSwZNSNZ\\\\\\\/GHUlgeCiZptvtosmrv\\\\\\\/uac7Jj\\\\\\\/Cy6T+QwF9J+zDOlOElSx\\\\\\\/ERG1RWFz9N1s3fDJKVXQ+69FV9fmQMQTeFiZZYL\\\\\\\/84HqDAGqeg+TyLi7nfJ5l03kqk1G5dnS3OBCnwxiWJxI0lgeaqHoW7w5ma7v5fhy5Pgqn+CA80Y9bVjFTuSSZNwujBK8HcQ7nbL3kMHg+FsN2gEOc5m+G6JhCsDzwhtgknOzWMA3do+gQFfp6HfBdYs7hlex2KnE2SIcWMVxzWvII3zxur4PDB+WkdSP\\\\\\\/n9\\\\\\\/8NgWQ3t7k1x6pFTmighAwMZPP3dNdEw7jJfgD7lVy0KL7eGFuWpESG991wbmG6A\\\\\\\/7wc8NzlkNa+JizzNcG7seQ4s9gtgMsk\\\\\\\/yl30khzQiyo6KUz5EdRabQ\\\\\\\/aHjpsB4hs46FVZDnixW16kTYjPWg8BQBUCdFrqHWrHKD1XwMNbTXpNg\\\\\\\/dcfxGFyvyehAbby3oiZ7EnkICnAe6YiqBCzf3T61Hx9\\\\\\\/33U0iEsNuLq0x6vHCuI5c4nFwqVvRGWE1ANTjdDb8Sc5hOZm8\\\\\\\/guu5Vcm\\\\\\\/ukqYaaxvt\\\\\\\/m0JMBpDMVQmaASSBgP8AWZU3D+0gwD0MN9RhMIaIl0g10XTPhATv0br2Xe4VfOZ5EwSjMO4EalMnuUypEyRuWQOyRz\\\\\\\/63\\\\\\\/wSK2jksrO6jmsVy4sgcx9\\\\\\\/pZWeOpCeC5vP9rg5r6eEd5QTU7FsefxcoYNiAvv5MM1irObJ\\\\\\\/n4SbEe2tB\\\\\\\/UaJa6sw+yVePwEJb3zFhC+1eHv6p5BT\\\\\\\/SBNTVLb0z1LznuvF3QA3kZgQumoSSrXpRUAITp79KJfyNWBfsiPadcG2henJKFBjY4sJ5bmzFRleGyVzAIN8N\\\\\\\/r22+0Ly1cpfolt3V51yxFz3o7PDUQMdpvh\\\\\\\/z0gGXX4xWZaprCgeZNcAWqsT1FQSWbi+\\\\\\\/oDccoIFVdxyJNlL0p7JIN+CZwkm\\\\\\\/+J5rjpWtTZl1AHFLDAlDEWCBveXlYZ+tSxuQ7VGMSDpOFMi0h\\\\\\\/NKbUbn5LdEhaxfwEiCSbQxoJP6TZyyHz17I3jeF2SqFUVHJOD4uiOF53N95X\\\\\\\/bJC4SMFngHKtITKrcyJnE06z+uOpwfTFdyHskGYRwZRuCM+hB0ui8n1m5P6FOE4P7PXy6bj4C4czYlV1PxmhKvgoOF1ys3IqT7EUfyF+pak5I5ILG\\\\\\\/UTzVKltWAPUVOlxabnA5YsfwBP5X6jxJpi9\\\\\\\/pgBN+Sz2L5GW\\\u0022,\\\u0022iv\\\u0022:\\\u0022BhKYzQR84vLQZ4D+\\\\\\\/mHEog==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227bccfb7d7608816c9e37c0324b83e87b14db442b29772ce6a97099bf23584587\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Vt8Be+oMCArWMWXp6hT+1ivPAd85+sUMkgRQVwtnXHMbOEsWrHZxIFAPje3c6BnVWI5ULoXUyfqpGm4i2AM12Rjyy8zNhaCIY\\\\\\\/2Lh4Ots2gW+5lK+mJz+fvOq\\\\\\\/HJO7omN3E0FrRlw5kwYVEkJykwFaZJ0UMk9a3N1UPdowbWML88TN6DE5ZKoYDjxVDT+RqZI5P3iYKGx+prxxcmhCja8Q5UEN7BgDFW6mcXtMebobqt251Y3cuK07Ts\\\\\\\/3NGug0I5ika+KessoY\\\\\\\/W+8DxDOecGpZBvEiU+xy2G\\\\\\\/ORe5cLzZhassDl0y2CZmQ2BDcPFgsecbcAKILpx6ZsMcAJ1KZdPCkFKVvWg\\\\\\\/UDMCCngxA8q+twP9owwBYhR7odnftgNUEjH7ssbNgGY60bhb6Znj5b5zU1LAy1QWykan7DuPWTRsk6J6RF+u8MIQTL+wgkBsqfaa7aELV0uD4Iz4x0VkIH4bfldP9fmzDtXSKPUa5x5QYkOYJbdATZ3vGrw6r2lC6iYL8aSfxSc4MPQM0R5KHFePARDtFXG0qtkqElXhDy8ElPmBsupdC+yXnth0paKVhFzv7S51c06h8pHr3BjuF0HgYLjeW+E4Cv2FuuNjcT71nvn6jjcRl4aCNqMbT96ffz+Xz\\\\\\\/VjuFgEcA+PVEmsY3Fv46ZzJDFgNdfHMhyPMqDDou\\\\\\\/xvJ4koUhB4BB10K4JIEJnrT76h4xrxKyo5xiax85ruGmvDy\\\\\\\/4Fw3ei1BdpWL18KkFfVpr96vlHEV64eEEWJTo7J9Pvc2uxiFpNXr7fTDHI63J9ZU2Vnm0Wa2y\\\\\\\/PrBJkHznJ8FCs6jWfgNNlsLs6Alskgc79hDxj\\\\\\\/zVngUSOscvAl7hsTuTMY6cl7jTsMVpYGx61nTVR14QQfWFizLvKx03KyRViuSD\\\\\\\/UISXXqAVT+15uuo\\\\\\\/+1xSEb406P5BVKjW19V+ce0akJDHWIwkqKUQDhl0252EFWOvytAaJplWJoIRFvHH+JR0CGmYD+27XgNjNTnFL5kr85Qz4B0q9DWdAo7ELzkOKzSr7ACw122jS1\\\\\\\/6fWXXBRkOgH73gZFFMb8H7ApBqe6dTUxqio1MuMo1wmycBqe5ExXFaeg8ORRl+MdZ2TQT\\\\\\\/FVhhhySeXUN0rBHMziDSDnh0o6e9Xys5wyB1FyOXxi2HoA+mQd4wvYl1rTLEHpa3TfK3AYyCa4ZXGUJQM7lYHg12Dza1pScADRyRaDx70Y6w+2fTulVLbxd+VY2zdchwU5Vla9MwZsRdacddN3LU4I5qwDEL+UrB08RNQzKI3G1uQCINCpnKgAdh6XlrzVuHcCNJUkoZtHqTz+4Zva7lJcO6lpu3S5rfIBoHtqZ7gGxuN3sD8eY6yrUFrcXiBCBorqrsAU0OHyH16awoDorihK6R3apOAGrGJb2zhc7eV7iv3fT4E80zTjAmPO0s+3M208rZrp7Y6dvVqdKsCFav9J\\\\\\\/TTIDLkT9+F569PmFuCKUDVq2N1tDU2AlG6IDjiKCF5nKYXnMzojYu+tDRQwSS+xk6wyh34MlB9ma11o9sTQ9xC8wiwpSgE3EoyZlHL\\\\\\\/vPq5Xape2bPvBLShKfuqawuttNAZmR59BN4QyPpExn3CDFDsn29eTMK6HkgGCvCkdwZOECchtdD8Y\\\\\\\/it5N0EQS6f9AmwJod6srmtRnvGcYDjUlg5OpcPZA1qQ1PYnskaHHvEv\\\\\\\/EtaxiTsOI1hjmGCZuy\\\\\\\/EOdu++bfb90XcWRI0lHNmn9Fg0hv35Gy6r0Hv63sZAPiA0x41jT8xGOpuscoa90e0ideJSKUHJaupo8dbK7qOkX7xUGQpnVEPehHFqSfvu0fAKEplG8YPnLzNPYAXa+ScBKOK0Kw\\\\\\\/\\\\\\\/Ys8r5UpKBvCEN7UbATtpNsTDhJyBCcyf8m3URQPUL+Sg9T6D35RLs9odmchdsQaHGUJhZcuz76L\\\\\\\/2sHcacQvNWdE0WlfLVAhF5kK9gLgpbEEjvbq8nYCCRkRLI4yFXoPpKvCYBcSg0lVfJQxiAGtDYtBGE+lKdC7q58z8qfevAwUMP1QTGOHVAdm4yR9qNWGX\\\u0022,\\\u0022iv\\\u0022:\\\u0022brt3WDfQwPD2DLlrxmnMLg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022adb7db280a44d96d5342a7818d701e739a16f542a6df2856e2f1f657fc3ddcfd\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460268 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2553" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4Z0i1gAQUlLvc3stqHCw5GeXOqk32xnSKuPtPHtoKUI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2553" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Up_UYx8q9dfvi29IuUMBUr_sm76QOFE4tqvfB_IP0jA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2543" title="A Meiotic Drive Element in the Maize Pathogen Fusarium verticillioides Is Located Within a 102 kb Region of Chromosome V" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2563" title="Variation and Evolution in the Glutamine-Rich Repeat Region of Drosophila Argonaute-2" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460268" id="node460268" data-pisa="ggg;6/8/2553" data-pisa-master="ggg;g3.116.027730" data-apath="/ggg/6/8/2553.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Using Sequence Variants in Linkage Disequilibrium with Causative Mutations to Improve Across-Breed Prediction in Dairy Cattle: A Simulation Study</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Irene van den Berg</span>, <span class="highwire-citation-author" data-delta="1">Didier Boichard</span>, <span class="highwire-citation-author" data-delta="2">Bernt Guldbrandtsen</span> and <span class="highwire-citation-author" data-delta="3">Mogens S. Lund</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2553-2561;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027730 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460268" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Irene van den Berg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Génétique Animale et Biologie Intégrative (GABI), French National Institute for Agricultural Research (INRA), AgroParisTech, Université Paris Saclay, 78350 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Irene%2Bvan%2Bden%2BBerg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20den%20Berg%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIrene%2Bvan%2Bden%2BBerg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:irene.vandenberg@unimelb.edu.au" class="" data-icon-position="" data-hide-link-title="0">irene.vandenberg@unimelb.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Didier Boichard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Génétique Animale et Biologie Intégrative (GABI), French National Institute for Agricultural Research (INRA), AgroParisTech, Université Paris Saclay, 78350 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Didier%2BBoichard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boichard%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADidier%2BBoichard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bernt Guldbrandtsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernt%2BGuldbrandtsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guldbrandtsen%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABernt%2BGuldbrandtsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mogens S. Lund </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mogens%2BS.%2BLund%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lund%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMogens%2BS.%2BLund%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/8/2553" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460268" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460268/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2553.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460268" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460268/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2553.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460268" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460268/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2553.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460268" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460268/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/8/2553.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460268" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460268/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Sequence data are expected to increase the reliability of genomic prediction by containing causative mutations directly, especially in cases where low linkage disequilibrium between markers and causative mutations limits prediction reliability, such as across-breed prediction in dairy cattle. In practice, the causative mutations are unknown, and prediction with only variants in perfect linkage disequilibrium with the causative mutations is not realistic, leading to a reduced reliability compared to knowing the causative variants. Our objective was to use sequence data to investigate the potential benefits of sequence data for the prediction of genomic relationships, and consequently reliability of genomic breeding values. We used sequence data from five dairy cattle breeds, and a larger number of imputed sequences for two of the five breeds. We focused on the influence of linkage disequilibrium between markers and causative mutations, and assumed that a fraction of the causative mutations was shared across breeds and had the same effect across breeds. By comparing the loss in reliability of different scenarios, varying the distance between markers and causative mutations, using either all genome wide markers from commercial SNP chips, or only the markers closest to the causative mutations, we demonstrate the importance of using only variants very close to the causative mutations, especially for across-breed prediction. Rare variants improved prediction only if they were very close to rare causative mutations, and all causative mutations were rare. Our results show that sequence data can potentially improve genomic prediction, but careful selection of markers is essential.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/across-breed-prediction" class="hw-term hw-article-keyword hw-article-keyword-across-breed-prediction" rel="nofollow">across-breed prediction</a></li><li class="kwd"><a href="/keyword/sequence-data" class="hw-term hw-article-keyword hw-article-keyword-sequence-data" rel="nofollow">sequence data</a></li><li class="kwd"><a href="/keyword/genomic-relationships" class="hw-term hw-article-keyword hw-article-keyword-genomic-relationships" rel="nofollow">genomic relationships</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-4">For accurate genomic prediction, it is important to have a large reference population (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Goddard and Hayes 2009</a>). Therefore, combining reference populations of several breeds could improve across-breed prediction. The level of linkage disequilibrium (LD) across breeds is, however, substantially lower than within breed. Therefore, with the current 50K and high-density (HD) single nucleotide polymorphism (SNP) chips, combining reference populations results in increased prediction accuracy only for closely related breeds, such as different Nordic Red populations (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brøndum <em>et al.</em> 2011</a>), while, for more distant breeds, such as Holstein and Jersey, the accuracy of across-breed prediction is low (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hayes <em>et al.</em> 2009</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Erbe <em>et al.</em> 2012</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lund <em>et al.</em> 2014</a>). Increasing the SNP density from 50K to HD was expected to increase prediction accuracy across breeds, because, at the marker density of the HD chip, LD between causative mutations and markers should be sufficient for across-breed prediction (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de Roos <em>et al.</em> 2008</a>). In practice, however, only small or no differences between prediction accuracies using the 50K or HD were found (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Erbe <em>et al.</em> 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hozé <em>et al.</em> 2014</a>). By containing causative mutations as well as variants in very high LD with the causative mutations, sequence data could potentially make accurate across-breed prediction feasible. The increasing number of whole-genome sequences available will enable the use of sequence data in breeding programs. With the 1000 Bull Genomes Project (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Daetwyler <em>et al.</em> 2014</a>), a large reference population of sequences is available for imputation of SNP chip genotypes to whole-genome sequence, which can subsequently be used for genome-wide association studies and genomic prediction.</p><p id="p-5">While most variants in the sequence have a low minor allele frequency (MAF), the variants on the SNP chips are selected to have high MAF (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Matukumalli <em>et al.</em> 2009</a>). Differences in MAF between markers and causative mutations would limit the maximum LD achievable between markers and causative mutations. The abundance of low MAF variants in the sequence data could suggest that at least some of the causative mutations have low MAF. Using sequence data to predict these variants could therefore increase prediction accuracy, either by inclusion of the causative variants, or by achieving a stronger LD with low-MAF sequence variants near the causative mutations and the causative mutations themselves. <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Druet <em>et al.</em> (2014)</a> found that using sequence data for genomic prediction leads to a higher prediction accuracy for rare causative mutations.</p><p id="p-6">Sequence data does not, however, necessarily contain all causative variants for various reasons, including filtering and imputation errors. Furthermore, even if all causative variants were present, and even if a trait is very polygenic, the number of variants directly affecting that trait is only a very small proportion of all variants present in the sequence. Therefore, the majority of sequence variants used for genomic prediction will be, at most, in imperfect LD with the causative mutations. As a consequence, the prediction reliability will be lower than the prediction reliability if only causative mutations were used. The reduction in prediction reliability as a consequence of imperfect LD between causative mutations and markers can be quantified by the regression of genomic relationships at markers used for prediction on genomic relationships at causative mutations (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> 2013</a>). We used this regression to investigate the potential benefits of using sequence data for within- and across-breed prediction in five French and Danish dairy cattle breeds.</p><p id="p-7">In this paper, we focus on the influence on across-breed prediction of LD between causative mutations and markers. While LD is an important factor, the accuracy of across-breed prediction is also influenced by other factors. Only part of the causative mutations segregating within breed are segregating across breeds (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Raven <em>et al.</em> 2014</a>), and causative mutations that do segregate across breeds can have different effects in different breeds, due to, <em>e.g.</em>, epistasis and dominance. Therefore, results in this paper should be interpreted as maximum gains that could be achieved by using sequence variants near the causative mutations.</p><p id="p-8">The objective of this study was to use actual sequence data from representative individuals of different breeds to investigate the potential benefits of sequence data for prediction of genomic relationships compared to the use of 50K and HD data, focusing on the loss in prediction reliability due to the use of markers in imperfect LD with the causative mutations. Because the advantages of using sequence data are expected to be larger in populations with low LD than in populations with high LD, within- and across-breed predictions were compared in several scenarios. We simulated several scenarios, varying the number of causative mutations, and the MAF of the causative mutations. Different sets of markers were subsequently used, with sequence variants on varying distances from the causative mutations, varying the MAF of the markers, or variants from the 50K and HD SNP chips. First, we validated the formula proposed by <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> (2013)</a> to estimate the loss in prediction reliability for across-breed prediction. Subsequently, we applied this formula to a wider range of scenarios and breeds.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-9">Two datasets were used for different parts of the study. Both datasets consisted of sequence data on chromosome 1. The first set (SEQ, Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027730/-/DC1/FileS1.zip">File S1</a>) contained nonimputed sequence data of 122 Holstein (HOL), 27 Jersey (JER), 28 Montbéliarde (MON), 23 Normande (NOR), and 45 Danish Red (RDC) bulls. These bulls were the first bulls to be sequenced in France and Denmark, and were major ancestors contributing to each breed. While all JER and RDC bulls were Danish bulls, and all MON and NOR were French, HOL bulls originated from different populations. There were 41 French, 22 Danish, 22 American, eight Canadian, eight German, three Italian, two Swedish, and one Finnish HOL bulls. Variant calling was done using GATK (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">DePristo <em>et al.</em> 2011</a>). There were 1,475,541 biallelic SNP and indels on chromosome (BTA) 1 segregating in at least one of the five breeds. Causative mutations were simulated by randomly sampling 100 or 250 variants, either out of all variants (C<sub>100</sub> and C<sub>250</sub>) or out of all variants with a MAF below 0.10 (C<sub>100LM</sub> and C<sub>250LM</sub>).</p><p id="p-10">The second set (IMP, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027730/-/DC1/FileS2.zip">File S2</a>) consisted of imputed sequences of 1230 JER and 961 RDC bulls. Imputation was done in two steps, using IMPUTE2 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Howie <em>et al.</em> 2012</a>), first from 50K to HD, and subsequently to full sequence. More details on the imputation can be found in <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Höglund <em>et al.</em> (2014</a>). After filtering out variants with an IMPUTE2 INFO score below 0.9, there were 1,150,905 variants segregating on chromosome 1. The majority of these variants were segregating in both JER and RDC, while this was not the case for the variants in the SEQ data. To make the dataset comparable, only a subset of the data were used for analyses, containing 247,141 variants for which the percentages of variants segregating in both JER and RDC, in JER but not in RDC, in RDC but not in JER, and neither in JER nor in RDC, were equal to those in SEQ. For the individuals in IMP, phenotypes were simulated as following. First, 50, 100, or 250 variants were randomly selected from all 247,141 variants as causative mutations for scenarios C<sub>50</sub>, C<sub>100</sub>, and C<sub>250</sub>, respectively. The effects were then drawn from a normal distribution<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-1.gif"/></noscript></span></span>. Subsequently, true breeding values (TBV) were obtained by summing the effects of the causative mutations, and a random error was added to the TBV to obtain a phenotype, so that the heritability = 0.8.</p><p id="p-11">For the SEQ dataset, markers were selected according to one of the following scenarios. In scenarios P<sub>50K</sub> and P<sub>HD</sub>, markers were either all variants on BTA1 present on the Illumina BovineSNP50 Beadchip, or on the Illumina BovineHD Beadchip. In scenarios P<sub>50KC</sub> and P<sub>HDC</sub>, only the closest variant of each causative mutation present on the 50K or the HD SNP chip were selected. In the remaining 16 scenarios, selected markers were all markers present in two 1-kb intervals on both sides of each causative mutation. The distance between the beginning of the intervals and the causative mutations (<em>d</em>) was 1 base, 1 kb, 5 kb, 10 kb, 25 kb, or 100 kb, and the intervals contained either all variants (P<sub>SEQ</sub><em><sub>d</sub></em>), or only the variants with a MAF ≥ 0.10 (P<sub>SEQ</sub><em><sub>d</sub></em><sub>HM</sub>) present in the intervals. Each of these scenarios was repeated 50 times. For the IMP scenarios, the analysis was restricted to the P<sub>50K</sub>, P<sub>HD</sub>, P<sub>50KC</sub>, P<sub>HDC</sub>, P<sub>SEQ1kb</sub>, P<sub>SEQ10kb</sub>, and P<sub>SEQ25kb</sub> scenarios, with 20 repeats per scenario. An overview of the subsets of variants used from which causal loci and markers were selected is given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> gives an overview of all different scenarios.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468919/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468919/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Subsets of variants</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2553/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Description of the different scenarios. Scenarios varied according to the number (c) and minor allele frequency (MAF) of the causative mutations, and to the nature (from sequence or from chip), MAF, and distance to causative mutations of prediction variants." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-510937996" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Description of the different scenarios. Scenarios varied according to the number (&lt;em&gt;c&lt;/em&gt;) and minor allele frequency (MAF) of the causative mutations, and to the nature (from sequence or from chip), MAF, and distance to causative mutations of prediction variants.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2553/F1.medium.gif" width="440" height="297"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/8/2553/F1.medium.gif" width="440" height="297"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2553/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2553/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468921" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Description of the different scenarios. Scenarios varied according to the number (<em>c</em>) and minor allele frequency (MAF) of the causative mutations, and to the nature (from sequence or from chip), MAF, and distance to causative mutations of prediction variants.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">For each scenario, two genomic relationship matrices (<strong>G</strong>) were constructed, using either the causal loci or the markers following <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">VanRaden (2008)</a>:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-2.gif"/></noscript></span></span>where <strong>G</strong> is the genomic relationship matrix at causative mutations or markers, <strong>Z</strong> is a standardized genotype matrix with the causative mutations or markers, and <em>p<sub>j</sub></em> the allele frequency of the alternative allele for locus <em>j</em>. For individual <em>i</em> and locus <em>j</em>, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-3.gif"/></noscript></span></span>, with <em>n<sub>ij</sub></em> equal to the number of alternative alleles (0, 1, or 2). Allele frequencies were computed from genotypes of all individuals used to construct the genomic relationship matrix.</p><p id="p-15">The genomic relationship matrices were constructed for all individuals in each breed, and for each pairwise combination of breeds. Only variants segregating in at least one of the breeds were used to construct the genomic relationship matrices. Subsequently, the loss in prediction reliability resulting from the use of markers rather than the causal loci was quantified following <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> (2013)</a>:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-4.gif"/></noscript></span></span>where the difference in reliability between prediction using markers in imperfect LD with the causative mutations (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-5.gif"/></noscript></span></span>), and the prediction (<em>R<sup>2</sup></em>) if markers were in perfect LD with the causative mutations, is quantified by a reliability factor (RF). RF was either obtained (RF<sub>O</sub>) by estimating <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-6.gif"/></noscript></span></span> and <em>R<sup>2</sup></em>, or predicted as RF<sub>P</sub> = <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-7.gif"/></noscript></span></span>. The <em>b</em> in the RF<sub>P</sub> is the regression coefficient of the genomic relationships at markers on the genomic relationship markers at the causative mutations:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-8.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-9.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-10.gif"/></noscript></span></span> are the genomic relationships between individual <em>n</em>+1 and individuals 1 to <em>n</em> at the markers and the causative mutations, respectively, and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-11.gif"/></noscript></span></span> a random residual. First, <em>b</em> was computed for each individual. Subsequently, RF<sub>P</sub> was computed within replicate, using the <em>b</em> averaged across individuals. Finally, RF<sub>P</sub> was averaged across replicates.</p><p id="p-16">To estimate RF<sub>O</sub>, the IMP dataset was split into training and validation sets, using either JER to predict genomically estimated breeding values (GEBVs) of RDC, or prediction of JER GEBVs using phenotypic information of RDC. GEBVs and variance components were estimated using Average Information Restricted Maximum Likelihood (AI-REML) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jensen <em>et al.</em> 1997</a>) as implemented in DMU (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Madsen and Jensen 2013</a>):<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-12.gif"/></noscript></span></span>where μ was the overall mean, <strong>Z</strong> an incidence matrix linking phenotypes with the vector that contains the additive genetic effects <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-13.gif"/></noscript></span></span>, and <strong>e</strong> a vector with random residuals <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-14.gif"/></noscript></span></span>. The reliability was estimated as the squared correlation between GEBVs and TBVs. The genomic heritability was computed as <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2553/embed/mml-math-15.gif"/></noscript></span></span>.</p><div id="sec-2" class="subsection"><h3>Data availability</h3><p id="p-17">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3>Genomic relationships within and across breeds</h3><p id="p-18"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the genomic relationships using HD markers within and across breeds. HOL bulls originated from several countries, and within breed relationships were smaller than in the other breeds. Genomic relationships between other breeds were larger than with HOL, except for some of the RDC bulls. Within RDC, some of the individuals were highly related, while others were less related, which confirms the admixed nature of the breed.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2553/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic relationships within and across breed, using HD markers. HOL, Holstein; JER, Jersey; MON, Montb&#xE9;liarde; NOR, Normande; RDC, Danish Red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-510937996" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genomic relationships within and across breed, using HD markers. HOL, Holstein; JER, Jersey; MON, Montb&#xE9;liarde; NOR, Normande; RDC, Danish Red.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2553/F2.medium.gif" width="431" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/8/2553/F2.medium.gif" width="431" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2553/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2553/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468838" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Genomic relationships within and across breed, using HD markers. HOL, Holstein; JER, Jersey; MON, Montbéliarde; NOR, Normande; RDC, Danish Red.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Variants segregating across breeds</h3><p id="p-20">The percentage of simulated causative mutations that were segregating in each breed, and shared between each combination of two breeds, is shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The largest number of causative mutations segregated within HOL and RDC, 78% and 77%, respectively, when causative mutations were selected out of all variants, and 64% and 62% when causative mutations were selected out of all variants with a MAF below 0.10. The smallest percentages were observed in JER, with 54% and 32% without and with restriction on MAF, respectively. HOL and RDC shared the largest number of causative mutations, and, for each breed, the percentage of shared causative mutations was highest with HOL and RDC and lowest with JER.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468865/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468865/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Sharing of causative mutations between breeds</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Reliabilities for different sets of markers</h3><p id="p-22"><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> shows the reliabilities, heritabilities, and RF estimated by cross-validation, and compares the observed RF (RF<sub>O</sub>) with the RF predicted by the formula (RF<sub>P</sub>). Reliabilities were very high for the causative mutations, ranging from 0.73 for prediction of RDC with a JER reference population with 250 causative mutations, to 0.95 for prediction of JER with a RDC reference population with 50 causative mutations. Reliabilities were substantially lower for all sets of markers. The highest reliabilities using markers were obtained with P<sub>SEQ1kb</sub>, and decreased when the distance between markers and causative mutations increased. Reliabilities equaled 0.38, 0.23, and 0.09 for P<sub>SEQ1kb</sub>, P<sub>SEQ10kb</sub>, and P<sub>SEQ25kb</sub>, respectively, when averaged across breeds and number of causative mutations. Reliabilities were lowest for P<sub>50K</sub> (0.06) and P<sub>HD</sub> (0.10), averaged across breeds and number of causative mutations. Using only the markers closest to the causative mutations rather than all 50K or HD markers increased reliability, especially for P<sub>HDC</sub>, with reliabilities of 0.11 and 0.30 for P<sub>50KC</sub> and P<sub>HDC</sub>, respectively.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468962/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468962/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Reliabilities and observed and prediction reliability factors for different sets of markers</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Heritabilities for different sets of markers</h3><p id="p-24">Estimated heritabilities were very close to the true heritability when prediction was based on the causative mutations. Averaged across scenarios with different number of causative mutations and breeds, the heritability was estimated to be 0.79. Using different sets of markers, the highest heritabilities were obtained with P<sub>50K</sub> and P<sub>HD</sub>. Here, heritabilities were 0.77 averaged across scenarios with different numbers of causative mutations and 0.78 averaged over breeds. When sequence intervals near the causative mutations were used for prediction, heritabilities decreased when the distance between causative mutations and prediction intervals (<em>d</em>) increased. Heritabilities ranged from 0.61 for prediction of JER from RDC for scenario C<sub>50</sub>P<sub>SEQ25kb</sub> to 0.81 for prediction of RDC from JER for scenario C<sub>250</sub>P<sub>SEQ1kb</sub>. The lowest heritabilities were obtained for P50<sub>KC</sub> and P<sub>HDC</sub>, with 0.56 averaged across scenarios with different numbers of causative mutations, and 0.64 when averaged over breeds.</p></div><div id="sec-8" class="subsection"><h3>Observed and predicted loss in reliability</h3><p id="p-25">Averaged across sets of markers and breeds, the difference between observed and predicted RF equaled 0.06, 0.06, and 0.09 for the scenarios with 50, 100, and 250 causative mutations, respectively. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> show the reliability, RF<sub>O</sub> and RF<sub>P</sub> as a function of the number of causative mutations. While RF<sub>P</sub> increased when the number of causative mutations increased, the reliability and PF<sub>O</sub> decreased. Averaged across scenarios and breeds, RF<sub>P</sub> equaled 0.19, 0.21, and 0.26 for C<sub>50</sub>, C<sub>100</sub>, and C<sub>250</sub>, respectively, compared to an RF<sub>O</sub> of 0.23, 0.23, and 0.17. Consequently, the loss in reliability was overestimated with 50 and 100 causative mutations, but underestimated with 250 causative mutations. With 50 causative mutations, the predicted RF minus the observed RF (RF<sub>P-O</sub>) ranged from –0.13 with P<sub>HD</sub> to 0.03 with P<sub>50KC</sub> for prediction of JER, and from –0.07 with P<sub>HD</sub> to 0.09 with P<sub>50KC</sub> for RDC. In the scenarios with 100 causative mutations, RF<sub>P-O</sub> ranged from –0.12 for PHD to 0.04 to P<sub>50KC</sub> in JER, and from –0.07 to 0.11 for RDC. With 250 causative mutations, RF was underestimated only with P<sub>50K</sub> and P<sub>HD</sub>, and prediction of JER. RF<sub>P-O</sub> ranged from –0.04 with P<sub>50K</sub> to 0.09 with P<sub>SEQ25kb</sub> for JER, and from 0.06 for P<sub>HD</sub> to 0.19 for P<sub>50KC</sub> for RDC.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2553/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Reliability factor (RFP) as a function of the distance between causative mutations and intervals in prediction scenarios for within- and across-breed prediction. HOL, Holstein; JER, Jersey; MON, Montb&#xE9;liarde; NOR, Normande; RDC, Danish Red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-510937996" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Reliability factor (RF&lt;sub&gt;P&lt;/sub&gt;) as a function of the distance between causative mutations and intervals in prediction scenarios for within- and across-breed prediction. HOL, Holstein; JER, Jersey; MON, Montb&#xE9;liarde; NOR, Normande; RDC, Danish Red.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2553/F3.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/8/2553/F3.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2553/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2553/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468999" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Reliability factor (RF<sub>P</sub>) as a function of the distance between causative mutations and intervals in prediction scenarios for within- and across-breed prediction. HOL, Holstein; JER, Jersey; MON, Montbéliarde; NOR, Normande; RDC, Danish Red.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Influence of the distance to causative mutations for within- and across-breed prediction</h3><p id="p-27"><a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> shows RF<sub>P</sub> for the C<sub>250</sub>P<sub>SEQd</sub> scenarios as a function of <em>d</em> for within- and across-breed prediction. While both within and across breeds, RF<sub>P</sub> decreases when <em>d</em> increases, this decrease was larger across than within breeds. For example, within HOL, RF<sub>P</sub> decreased from 0.91 to 0.86 when <em>d</em> increased from 1 base to 25 kb, while for across-breed prediction in HOL and MON, RF<sub>P</sub> decreased from 0.60 to 0.28 with the same increase in <em>d</em>. Within-breed, RF<sub>P</sub> was highest for MON and lowest for HOL, and, across breeds, RF<sub>P</sub> was highest for breed combinations with RDC. The sharpest decrease in RF<sub>P</sub> was between 1 base and 25 kb. For example, between JER and MON, RF<sub>P</sub> dropped from 0.65 to 0.28 when <em>d</em> increased from 1 base to 25 kb, and further decreased to 0.10 when <em>d</em> was 100 kb. All breeds and across-breed combinations showed the same pattern, and more or less a parallel decrease in RF<sub>P</sub>.</p></div><div id="sec-10" class="subsection"><h3>MAF</h3><p id="p-28">The relation between RF<sub>P</sub> and MAF of causative mutations and markers is shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. RF was lower for causative mutations with a low MAF than for scenarios with both low and high MAF causative mutations. When causative mutations contained both low and high MAF variants, prediction was always slightly better when prediction was based only on high MAF markers, while in the scenarios with only low MAF causative mutations, RF<sub>P</sub> was generally higher when both low and MAF markers were used. The difference was larger with markers within 25 kb of the causative mutations.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2553/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The influence of the MAF of causative mutations and markers on the reliability factor (RFP). Cc_PSEQd, no restriction on MAF for either causative mutations or markers; CcLM_PSEQd, MAF causative mutations &#x2264;&#xA0;0.10 (no restriction for markers); Cc_PSEQdHM, no restriction for causative mutations, MAF markers &gt;&#xA0;0.10; CcLM_PSEQdHM, MAF causative mutations &#x2264; 0.10, MAF markers &gt; 0.10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-510937996" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The influence of the MAF of causative mutations and markers on the reliability factor (RF&lt;sub&gt;P&lt;/sub&gt;). Cc_PSEQd, no restriction on MAF for either causative mutations or markers; CcLM_PSEQd, MAF causative mutations &#x2264;&#xA0;0.10 (no restriction for markers); Cc_PSEQdHM, no restriction for causative mutations, MAF markers &gt;&#xA0;0.10; CcLM_PSEQdHM, MAF causative mutations &#x2264; 0.10, MAF markers &gt; 0.10.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2553/F4.medium.gif" width="440" height="315"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/8/2553/F4.medium.gif" width="440" height="315"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2553/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2553/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468822" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">The influence of the MAF of causative mutations and markers on the reliability factor (RF<sub>P</sub>). Cc_PSEQd, no restriction on MAF for either causative mutations or markers; CcLM_PSEQd, MAF causative mutations ≤ 0.10 (no restriction for markers); Cc_PSEQdHM, no restriction for causative mutations, MAF markers &gt; 0.10; CcLM_PSEQdHM, MAF causative mutations ≤ 0.10, MAF markers &gt; 0.10.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-30">We quantified the loss in prediction reliability due to the use of markers in imperfect LD with causative mutations for different simulation scenarios, both within and across breeds. For some scenarios, we estimated prediction reliabilities, while for others, we used a formula to approximate the loss in prediction reliability. The loss in reliability quickly increased with the distance between markers and causative mutations, especially across breeds, and was influenced by the MAF of both causative mutations and markers.</p><div id="sec-12" class="subsection"><h3>Distance between causative mutations and markers</h3><p id="p-31">All sets of markers resulted in large decreases in prediction reliabilities compared to prediction using only the causative mutations. This is in line with results from a simulation study by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Pérez-Enciso <em>et al.</em> (2015)</a>, where prediction accuracies decreased rapidly when variants in lower LD with the causative mutations were included. Using only causative mutations for genomic prediction is, however, not realistic, since only a few causative mutations have been identified (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Grisart <em>et al.</em> 2004</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Braunschweig 2010</a>). Alternatively, numerous quantitative trait loci (QTL) have been identified for various traits (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Khatkar <em>et al.</em> 2004</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Daetwyler <em>et al.</em> 2008</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cole <em>et al.</em> 2011</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Höglund <em>et al.</em> 2014</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Sahana <em>et al.</em> 2014</a>), and can be used for genomic prediction. Therefore, we tested scenarios that used sequence intervals near the causative mutations, and increased the distance between the intervals and the causative mutations. While using these sequence intervals resulted in substantial decreases in reliability compared to prediction using the causative mutations, these scenarios did result in higher reliabilities than those obtained using all markers from SNP chips. The reliability rapidly decreased when the distance to causative mutations increased, showing the importance of mapping precision. Several studies using real data have shown that increases in prediction reliability can be obtained using variants associated with QTL (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Boichard <em>et al.</em> 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brøndum <em>et al.</em> 2015</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ober <em>et al.</em> 2015</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Porto-Neto <em>et al.</em> 2015</a>).</p></div><div id="sec-13" class="subsection"><h3>Marker density</h3><p id="p-32">The accuracy of genomic prediction is highly dependent on the level of LD between markers and causative variants. Therefore, sequence data can potentially increase prediction accuracy because it contains variants in high LD with the causative mutations. Increasing the marker density adds markers closer to, and in higher LD, with the causative mutations. At the same time, however, markers at a large distance of the causative mutations are added. Therefore, the genomic relationships at markers will be close to the genomic relationships at all loci rather than the genomic relationships at the causative mutations. As a consequence, increasing the density of the SNP markers from 50K to HD did not increase the reliability. Reliability increased only when the markers further away from the causative mutations were discarded, because genomic relationships at markers then approach the genomic relationships at causative mutations. This is in agreement with <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Erbe <em>et al.</em> (2012)</a>, who found no difference in prediction accuracy when the marker density was increased from 50K to HD using the GBLUP model. However, when Bayesian variable selection models were used, both <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Erbe <em>et al.</em> (2012)</a> and <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hozé <em>et al.</em> (2014)</a> found a small increase in prediction accuracy for HD markers compared to 50K markers. In a study using simulated sequence data, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">MacLeod <em>et al.</em> (2014)</a> found increases in prediction accuracies for sequence data compared to HD data when Bayes R was used. GBLUP (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">VanRaden 2008</a>) uses all markers to construct a genomic relationship matrix, and, therefore, using all markers from the 50K or HD chips will result in a genomic relationship matrix close to the genomic relationship matrix at all loci, while the Bayesian variable selection models (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Habier <em>et al.</em> 2011</a>) assume the majority of markers have no effect. By including only markers in high LD with causative mutations in the prediction models, Bayesian models result in covariance structures of phenotypes that are closer to those at causative mutations than GBLUP.</p></div><div id="sec-14" class="subsection"><h3>Prediction of RF</h3><p id="p-33">For some scenarios, we compared the RF estimated by cross-validation with the RF predicted by the formula of <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> (2013)</a>. While the differences between RF<sub>P</sub> and RF<sub>O</sub> are in the same range as those observed by <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> (2013)</a>, both RF<sub>P</sub> and RF<sub>O</sub> were influenced by the number of causative mutations, but in opposite direction. When the number of causative mutations increased, RF<sub>P</sub> increased, while RF<sub>O</sub> decreased. Consequently, RF<sub>P</sub> overestimated the loss in prediction reliabilities in the scenarios with 50 and 100 causative mutations, while it is supposed to be a measure of the minimum reduction in prediction reliability. Therefore, with the SEQ data set, we focused on the scenarios with 250 causative mutations, where RF<sub>P</sub> could be used as an approximation for the minimum loss in reliability. RF<sub>P</sub> is based on the genomic relationships at causative mutations and markers. When the number of causative mutations increases, genomic relationships at causative mutations become closer to the genomic relationships at all loci, and therefore, closer to any set of randomly selected markers, resulting in a higher RF<sub>P</sub>. On the other hand, in scenarios with fewer causative mutations, prediction reliabilities may be higher because more information is available to accurately estimate their effect. Originally, RF<sub>P</sub> was developed with reference to a homogeneous population (<a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> 2013</a>), and applying it to across-breed prediction may have resulted in larger deviations from RF<sub>O</sub> than it would have had for within-breed prediction. RF<sub>P</sub> compared genomic relationships matrices at markers and causative mutations, and therefore, the way the genomic relationship matrix is constructed is likely to influence RF<sub>P</sub>. In the across-breed scenarios, we used the allele frequencies of both populations to center the genotypes. Alternatively, we could have centered the genotypes based on the allele frequency of each population, which would have resulted in different genomic relationships, which would in turn have influenced RF<sub>P</sub>.</p></div><div id="sec-15" class="subsection"><h3>MAF</h3><p id="p-34">The SNPs on the 50K and HD chips are selected to have a higher MAF than most sequence variants (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Matukumalli <em>et al.</em> 2009</a>). As a consequence, their predictive ability will be better for common variants than for rare variants. If causative mutations are mainly variants with low MAF, using sequence data for the prediction of causative mutations could increase prediction accuracy. <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Druet <em>et al.</em> (2014)</a> found that using sequence data increases prediction accuracy particularly for rare causative mutations. In our results, including low MAF variants in the prediction intervals increased RF<sub>P</sub> only if all causative mutations had low MAF and the variants were very close to the causative mutations. When the distance between intervals and causative mutations increased, including low MAF variants still resulted in a slightly higher RF<sub>P</sub> across breeds, while within-breed RF was higher when low MAF variants were excluded. When causative mutations included both low and high MAF variants, excluding low MAF variants always resulted in a higher RF<sub>P</sub>. The LD between a rare variant and a causative mutation will generally be high if only the causative variant is rare and the distance between both variants is very small. Therefore, the advantage of using rare variants for genomic prediction might be limited, especially when GBLUP is used. Furthermore, for across-breed prediction, the number of rare variants shared between breeds is expected to be low, due to their low frequency, their increased fixation risk due to drift, and their generally younger age compared to high MAF mutations.</p></div><div id="sec-16" class="subsection"><h3>Across-breed genomic prediction</h3><p id="p-35">LD within breeds is larger and conserved over longer distances than across breeds (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">de Roos <em>et al.</em> 2008</a>). As a consequence, RF<sub>P</sub> was lower and decreased faster with an increasing distance across breeds than within breeds. Differences in LD within breeds, and in the amount of variants shared and the LD preserved across breeds, can explain differences between the different breeds and different breed combinations. HOL used for the analysis consisted of different Holstein populations, originating from several countries, while for the other breeds sequences originated from one country. This most likely explains why RF<sub>P</sub> for HOL was lower than for the other breeds.</p></div><div id="sec-17" class="subsection"><h3>Limitations and further studies</h3><p id="p-36">In our study, we only focused on the influence of LD between markers and causative mutations on the prediction of genomic relationships at causative mutations. In practice, there are many other factors influencing the accuracy of genomic prediction that we did not take into account. Across-breed prediction requires, besides high LD between causative mutations and markers, causative mutations to be segregating across breeds, and to have similar allele substitution effects in different breeds. In the across-breed scenarios, causative mutations were sampled from all causative mutations segregating in at least one of the breeds. In practice, the proportion of QTL segregating across breeds may be smaller than those presented in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Raven <em>et al.</em> 2014</a>). Furthermore, by predicting genomic relationships rather than breeding values, we did not consider scenarios where allele substitution effects vary across breeds, due to, <em>e.g.</em>, dominance and epistasis. Our results suggest that inclusion of sequence variants near causative mutations can increase across-breed prediction. Because some of the assumptions made in the simulation of the phenotypes are likely to be violated in real populations, our results should be interpreted as maximum gains that could be obtained by using variants in close LD with the causative mutations. In practice, gains would be expected to be smaller, due to factors other than LD limiting across-breed prediction. Further studies are required to investigate the effect of the proportion of causative mutations segregating across breeds, and differences and genetic heterogeneity on the accuracy, and advantage, of sequence variants for across-breed prediction.</p></div><div id="sec-18" class="subsection"><h3>Conclusions</h3><p id="p-37">Our results show that sequence data can potentially improve genomic prediction. While using the full sequence directly for prediction is not likely to improve prediction, using only variants close to the causative mutations could substantially increase reliability. This was especially the case for across-breed prediction, where reliabilities dropped quickly when the distance between causative mutations and markers increased. Using rare sequence variants improved prediction only when they were very close to rare causative mutations. To exploit the potential advantages of sequence data for genomic prediction, careful selection of markers is essential.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">We would like to thank the anonymous reviewers and the associate editor for their recommendations that improved the paper. French sequence data originated from the Cartoseq project funded by l'Agence Nationale de la Recherche (ANR) and Apisgene (ANR10-GENM-0018). I.B. benefited from an Erasmus-Mundus fellowship and a grant by Apisgene, within the framework of the European Graduate School in Animal Breeding and Genetics. This research was supported by the center for Genomic Selection in Animals and Plants (GenSAP) funded by The Danish Council for Strategic Research.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-39">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027730/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027730/-/DC1</a></p></li><li class="fn" id="fn-6"><p id="p-40"><em>Communicating editor: G. A. de los Campos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-01-28"><span class="received-label">Received </span>January 28, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-06-08"><span class="accepted-label">Accepted </span>June 8, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Berg <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.1" data-doi="10.1071/AN11119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boichard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Croiseau</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossignol</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in French dairy cattle.</span> <abbr class="cit-jnl-abbrev">Anim. Prod. Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BProd.%2BSci.%26rft.volume%253D52%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAN11119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AN11119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000301064800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braunschweig</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mutations in the bovine ABCG2 and the ovine MSTN gene added to the few quantitative trait nucleotides identified in farm animals: a mini-review.</span> <abbr class="cit-jnl-abbrev">J. Appl. Genet.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bapplied%2Bgenetics%26rft.stitle%253DJ%2BAppl%2BGenet%26rft.aulast%253DBraunschweig%26rft.auinit1%253DM.%2BH.%26rft.volume%253D51%26rft.issue%253D3%26rft.spage%253D289%26rft.epage%253D297%26rft.atitle%253DMutations%2Bin%2Bthe%2Bbovine%2BABCG2%2Band%2Bthe%2Bovine%2BMSTN%2Bgene%2Badded%2Bto%2Bthe%2Bfew%2Bquantitative%2Btrait%2Bnucleotides%2Bidentified%2Bin%2Bfarm%2Banimals%253A%2Ba%2Bmini-review.%26rft_id%253Dinfo%253Apmid%252F20720303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20720303&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.3" data-doi="10.3168/jds.2010-3765"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brøndum</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rius-Vilarrasa</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strandén</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guldbrandtsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reliabilities of genomic prediction using combined reference data of the Nordic Red dairy cattle populations.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">4700</span>–<span class="cit-lpage">4707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DBrondum%26rft.auinit1%253DR.%2BF.%26rft.volume%253D94%26rft.issue%253D9%26rft.spage%253D4700%26rft.epage%253D4707%26rft.atitle%253DReliabilities%2Bof%2Bgenomic%2Bprediction%2Busing%2Bcombined%2Breference%2Bdata%2Bof%2Bthe%2BNordic%2BRed%2Bdairy%2Bcattle%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3765%26rft_id%253Dinfo%253Apmid%252F21854944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3765&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21854944&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brøndum</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahana</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guldbrandtsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Quantitative trait loci markers derived from whole genome sequence data increases the reliability of genomic prediction.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">4107</span>–<span class="cit-lpage">4116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D98%26rft.spage%253D4107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.5" data-doi="10.1186/1471-2164-12-408"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cole</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonstegard</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawlor</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association analysis of thirty one production, health, reproduction and body conformation traits in contemporary U.S. Holstein cows.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DCole%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D12%26rft.spage%253D408%26rft.epage%253D408%26rft.atitle%253DGenome-wide%2Bassociation%2Banalysis%2Bof%2Bthirty%2Bone%2Bproduction%252C%2Bhealth%252C%2Breproduction%2Band%2Bbody%2Bconformation%2Btraits%2Bin%2Bcontemporary%2BU.S.%2BHolstein%2Bcows.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-408%26rft_id%253Dinfo%253Apmid%252F21831322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-408&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21831322&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.6" data-doi="10.3168/jds.2007-0333"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schenkel</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sargolzaei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome scan to detect quantitative trait loci for economically important traits in Holstein cattle using two methods and a dense single nucleotide polymorphism map.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">3225</span>–<span class="cit-lpage">3236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DDaetwyler%26rft.auinit1%253DH.%2BD.%26rft.volume%253D91%26rft.issue%253D8%26rft.spage%253D3225%26rft.epage%253D3236%26rft.atitle%253DA%2BGenome%2BScan%2Bto%2BDetect%2BQuantitative%2BTrait%2BLoci%2Bfor%2BEconomically%2BImportant%2BTraits%2Bin%2BHolstein%2BCattle%2BUsing%2BTwo%2BMethods%2Band%2Ba%2BDense%2BSingle%2BNucleotide%2BPolymorphism%2BMap%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0333%26rft_id%253Dinfo%253Apmid%252F18650300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0333&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18650300&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257876000032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.7" data-doi="10.1038/ng.3034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capitan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pausch</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stothard</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Binsbergen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole-genome sequencing of 234 bulls facilitates mapping of monogenic and complex traits in cattle.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">858</span>–<span class="cit-lpage">865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D858%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3034%26rft_id%253Dinfo%253Apmid%252F25017103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25017103&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.8" data-doi="10.1371/journal.pgen.1003608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klimentidis</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Prediction of complex human traits using the genomic best linear unbiased predictor.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003608%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003608%26rft_id%253Dinfo%253Apmid%252F23874214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874214&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.9" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.10" data-doi="10.1534/genetics.107.084301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Roos</span>  <span class="cit-name-given-names">A. P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spelman</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Linkage disequilibrium and persistence of phase in Holstein-Friesian, Jersey and Angus cattle.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1503</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084301%26rft_id%253Dinfo%253Apmid%252F18622038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE1MDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjU1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.11" data-doi="10.1038/hdy.2013.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Druet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macleod</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Toward genomic prediction from whole-genome sequence data: impact of sequencing design on genotype imputation and accuracy of predictions.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.13%26rft_id%253Dinfo%253Apmid%252F23549338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23549338&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.12" data-doi="10.3168/jds.2011-5019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goswami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Improving accuracy of genomic predictions within and between dairy cattle breeds with imputed high-density single nucleotide polymorphism panels.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">4114</span>–<span class="cit-lpage">4129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DErbe%26rft.auinit1%253DM.%26rft.volume%253D95%26rft.issue%253D7%26rft.spage%253D4114%26rft.epage%253D4129%26rft.atitle%253DImproving%2Baccuracy%2Bof%2Bgenomic%2Bpredictions%2Bwithin%2Band%2Bbetween%2Bdairy%2Bcattle%2Bbreeds%2Bwith%2Bimputed%2Bhigh-density%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bpanels.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-5019%26rft_id%253Dinfo%253Apmid%252F22720968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-5019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22720968&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305363100062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.13" data-doi="10.1038/nrg2575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping genes for complex traits in domestic animals and their use in breeding programmes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D381%26rft.epage%253D391%26rft.atitle%253DMapping%2Bgenes%2Bfor%2Bcomplex%2Btraits%2Bin%2Bdomestic%2Banimals%2Band%2Btheir%2Buse%2Bin%2Bbreeding%2Bprogrammes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2575%26rft_id%253Dinfo%253Apmid%252F19448663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19448663&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.14" data-doi="10.1073/pnas.0308518100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grisart</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farnir</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karim</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cambisano</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.-J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic and functional confirmation of the causality of the DGAT1 K232A quantitative trait nucleotide in affecting milk yield and composition.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">2398</span>–<span class="cit-lpage">2403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0308518100%26rft_id%253Dinfo%253Apmid%252F14983021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAxLzgvMjM5OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yNTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.15" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the Bayesian alphabet for genomic selection.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.16" data-doi="10.1186/1297-9686-41-51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Accuracy of genomic breeding values in multi-breed dairy cattle populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D41%26rft.spage%253D51%26rft.epage%253D51%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bbreeding%2Bvalues%2Bin%2Bmulti-breed%2Bdairy%2Bcattle%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-41-51%26rft_id%253Dinfo%253Apmid%252F19930712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-41-51&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19930712&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Höglund</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahana</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guldbrandtsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Validation of associations for female fertility traits in Nordic Holstein, Nordic Red and Jersey dairy cattle.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253D8%26rft_id%253Dinfo%253Apmid%252F24428918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24428918&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.18" data-doi="10.1038/ng.2354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchsberger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast and accurate genotype imputation in genome-wide association studies through pre-phasing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%26rft.volume%253D44%26rft.issue%253D8%26rft.spage%253D955%26rft.epage%253D959%26rft.atitle%253DFast%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bin%2Bgenome-wide%2Bassociation%2Bstudies%2Bthrough%2Bpre-phasing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2354%26rft_id%253Dinfo%253Apmid%252F22820512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820512&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hozé</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phocas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boichard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ducrocq</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficiency of multi-breed genomic selection for dairy cattle breeds with different sizes of reference population.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">3918</span>–<span class="cit-lpage">3929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D97%26rft.spage%253D3918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mäntysaari</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Residual maximum likelihood estimation of (co)variance components in multivariate mixed linear models using average information.</span> J. Indian Soc. Agric. Statistics <span class="cit-vol">49</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.21" data-doi="10.1051/gse:2003057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khatkar</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tammen</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raadsma</span>  <span class="cit-name-given-names">H. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Quantitative trait loci mapping in dairy cattle: review and meta-analysis.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DKhatkar%26rft.auinit1%253DM.%2BS.%26rft.volume%253D36%26rft.issue%253D2%26rft.spage%253D163%26rft.epage%253D190%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Bmapping%2Bin%2Bdairy%2Bcattle%253A%2Breview%2Band%2Bmeta-analysis.%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A2003057%26rft_id%253Dinfo%253Apmid%252F15040897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:2003057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15040897&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188936300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.22" data-doi="10.1016/j.livsci.2014.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guldbrandtsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brøndum</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Invited review: Genomic evaluation of cattle in a multi-breed context.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2014.05.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2014.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.23" data-doi="10.1534/genetics.114.168344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacLeod</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The effects of demography and long term selection on the accuracy of genomic prediction with sequence data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1671</span>–<span class="cit-lpage">1684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.168344%26rft_id%253Dinfo%253Apmid%252F25233989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE2NzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjU1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-web" id="cit-6.8.2553.24"><div class="cit-metadata unstructured">Madsen, P., and J. Jensen, 2013 A user’s guide to DMU. 1–32. Available at: <a href="http://dmu.agrsci.dk/DMU/Doc/Current/dmuv6_guide.5.2.pdf.">http://dmu.agrsci.dk/DMU/Doc/Current/dmuv6_guide.5.2.pdf.</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.25" data-doi="10.1371/journal.pone.0005350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matukumalli</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawley</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allan</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Development and characterization of a high density SNP genotyping assay for cattle.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e5350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMatukumalli%26rft.auinit1%253DL.%2BK.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253De5350%26rft.epage%253De5350%26rft.atitle%253DDevelopment%2Band%2Bcharacterization%2Bof%2Ba%2Bhigh%2Bdensity%2BSNP%2Bgenotyping%2Bassay%2Bfor%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005350%26rft_id%253Dinfo%253Apmid%252F19390634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19390634&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.26" data-doi="10.1371/journal.pone.0126880"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magwire</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlather</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accounting for genetic architecture improves sequence based genomic prediction for a <em>Drosophila</em> fitness trait.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0126880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0126880%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0126880%26rft_id%253Dinfo%253Apmid%252F25950439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0126880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25950439&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Enciso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rincón</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Sequence- <em>vs.</em> chip-assisted genomic selection: accurate biological information is advised.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D43%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Porto-Neto</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barendse</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henshall</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McWilliam</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehnert</span>  <span class="cit-name-given-names">S. a.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic correlation: harnessing the benefit of combining two unrelated populations for genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D84%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raven</span>  <span class="cit-name-given-names">L.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cocks</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multibreed genome wide association can improve precision of mapping causative variants underlying milk production in dairy cattle.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D62%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sahana</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guldbrandtsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomsen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holm</span>  <span class="cit-name-given-names">L.-E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panitz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide association study using high-density single nucleotide polymorphism arrays and whole-genome sequences for clinical mastitis traits in dairy cattle1.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">7258</span>–<span class="cit-lpage">7275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BDairy%2BSci.%26rft.volume%253D97%26rft.spage%253D7258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2553.31" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2553.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/8/2553.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2543" title="A Meiotic Drive Element in the Maize Pathogen Fusarium verticillioides Is Located Within a 102 kb Region of Chromosome V" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2563" title="Variation and Evolution in the Glutamine-Rich Repeat Region of Drosophila Argonaute-2" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/8"><span>Volume 6 Issue 8, August 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/8.cover-source.jpg?itok=40Kz9Oeq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2553.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Pu0FhiYqD3LRKgK9SVwhPIAEzr0Tbo3YltKVYV2PCAo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/8/2553" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/8/2553" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/8/2553" class="active" data-icon-position="" data-hide-link-title="0">Using Sequence Variants in Linkage Disequilibrium with Causative Mutations to Improve Across-Breed Prediction in Dairy Cattle: A Simulation Study</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460268" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-RsT8u0BZ_rcUbJjtlABpgdODx5KH4HLsBFLKtEVriWE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2553.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/8/2553" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-z_SinJLROTQvvpwgJ5i21x6DhS4ZaCte0rAzBeRME8I" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/8/2553" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2553.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460268" id="node460268--2" data-pisa="ggg;6/8/2553" data-pisa-master="ggg;g3.116.027730" data-apath="/ggg/6/8/2553.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Using Sequence Variants in Linkage Disequilibrium with Causative Mutations to Improve Across-Breed Prediction in Dairy Cattle: A Simulation Study</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Irene van den Berg</span>, <span class="highwire-citation-author" data-delta="1">Didier Boichard</span>, <span class="highwire-citation-author" data-delta="2">Bernt Guldbrandtsen</span> and <span class="highwire-citation-author" data-delta="3">Mogens S. Lund</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2553-2561;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027730 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460268--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Irene van den Berg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Génétique Animale et Biologie Intégrative (GABI), French National Institute for Agricultural Research (INRA), AgroParisTech, Université Paris Saclay, 78350 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Irene%2Bvan%2Bden%2BBerg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20den%20Berg%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIrene%2Bvan%2Bden%2BBerg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:irene.vandenberg@unimelb.edu.au" class="" data-icon-position="" data-hide-link-title="0">irene.vandenberg@unimelb.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Didier Boichard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Génétique Animale et Biologie Intégrative (GABI), French National Institute for Agricultural Research (INRA), AgroParisTech, Université Paris Saclay, 78350 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Didier%2BBoichard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boichard%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADidier%2BBoichard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bernt Guldbrandtsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernt%2BGuldbrandtsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guldbrandtsen%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABernt%2BGuldbrandtsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mogens S. Lund </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mogens%2BS.%2BLund%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lund%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMogens%2BS.%2BLund%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/8/2553" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/8/2553&amp;title=Using%20Sequence%20Variants%20in%20Linkage%20Disequilibrium%20with%20Causative%20Mutations%20to%20Improve%20Across-Breed%20Prediction%20in%20Dairy%20Cattle%3A%20A%20Simulation%20Study" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/8/2553&amp;title=Using%20Sequence%20Variants%20in%20Linkage%20Disequilibrium%20with%20Causative%20Mutations%20to%20Improve%20Across-Breed%20Prediction%20in%20Dairy%20Cattle%3A%20A%20Simulation%20Study" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/8/2553&amp;title=Using%20Sequence%20Variants%20in%20Linkage%20Disequilibrium%20with%20Causative%20Mutations%20to%20Improve%20Across-Breed%20Prediction%20in%20Dairy%20Cattle%3A%20A%20Simulation%20Study" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/8/2553&amp;text=Using%20Sequence%20Variants%20in%20Linkage%20Disequilibrium%20with%20Causative%20Mutations%20to%20Improve%20Across-Breed%20Prediction%20in%20Dairy%20Cattle%3A%20A%20Simulation%20Study" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/8/2553&amp;title=Using%20Sequence%20Variants%20in%20Linkage%20Disequilibrium%20with%20Causative%20Mutations%20to%20Improve%20Across-Breed%20Prediction%20in%20Dairy%20Cattle%3A%20A%20Simulation%20Study" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/8/2553&amp;t=Using%20Sequence%20Variants%20in%20Linkage%20Disequilibrium%20with%20Causative%20Mutations%20to%20Improve%20Across-Breed%20Prediction%20in%20Dairy%20Cattle%3A%20A%20Simulation%20Study" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/8/2553&amp;title=Using%20Sequence%20Variants%20in%20Linkage%20Disequilibrium%20with%20Causative%20Mutations%20to%20Improve%20Across-Breed%20Prediction%20in%20Dairy%20Cattle%3A%20A%20Simulation%20Study" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/8/2553&amp;title=Using%20Sequence%20Variants%20in%20Linkage%20Disequilibrium%20with%20Causative%20Mutations%20to%20Improve%20Across-Breed%20Prediction%20in%20Dairy%20Cattle%3A%20A%20Simulation%20Study" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460268/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460268" id="node460268--3" data-pisa="ggg;6/8/2553" data-pisa-master="ggg;g3.116.027730" data-apath="/ggg/6/8/2553.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Using Sequence Variants in Linkage Disequilibrium with Causative Mutations to Improve Across-Breed Prediction in Dairy Cattle: A Simulation Study</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Irene van den Berg</span>, <span class="highwire-citation-author" data-delta="1">Didier Boichard</span>, <span class="highwire-citation-author" data-delta="2">Bernt Guldbrandtsen</span> and <span class="highwire-citation-author" data-delta="3">Mogens S. Lund</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2553-2561;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027730 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460268--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Irene van den Berg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Génétique Animale et Biologie Intégrative (GABI), French National Institute for Agricultural Research (INRA), AgroParisTech, Université Paris Saclay, 78350 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Irene%2Bvan%2Bden%2BBerg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20den%20Berg%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIrene%2Bvan%2Bden%2BBerg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:irene.vandenberg@unimelb.edu.au" class="" data-icon-position="" data-hide-link-title="0">irene.vandenberg@unimelb.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Didier Boichard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Génétique Animale et Biologie Intégrative (GABI), French National Institute for Agricultural Research (INRA), AgroParisTech, Université Paris Saclay, 78350 Jouy-en-Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Didier%2BBoichard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boichard%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADidier%2BBoichard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bernt Guldbrandtsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernt%2BGuldbrandtsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guldbrandtsen%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABernt%2BGuldbrandtsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Mogens S. Lund </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, DK-8830 Tjele, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mogens%2BS.%2BLund%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lund%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMogens%2BS.%2BLund%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2553&amp;rft.epage=2561&amp;rft.atitle=Using%20Sequence%20Variants%20in%20Linkage%20Disequilibrium%20with%20Causative%20Mutations%20to%20Improve%20Across-Breed%20Prediction%20in%20Dairy%20Cattle%3A%20A%20Simulation%20Study&amp;rft.volume=6&amp;rft.issue=8&amp;rft.date=2016-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=van+den+Berg%2C+Irene&amp;rft.au=Boichard%2C+Didier&amp;rft.au=Guldbrandtsen%2C+Bernt&amp;rft.au=Lund%2C+Mogens+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460268/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460268/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460268/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460268/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460268/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460268/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460268/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460268/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460268/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460268/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460268/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460268/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/8/2553" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/8/2553.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/8/2553.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/8/2553.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/8/2553.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
