<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/8/2655" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/8/2655.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/8/2655.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/8/2655.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/8/2655.atom" />
<meta name="HW.pisa" content="ggg;6/8/2655" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Identification of Functional Domains in the Cohesin Loader Subunit Scc4 by a Random Insertion/Dominant Negative Screen" />
<meta name="DC.Identifier" content="10.1534/g3.116.031674" />
<meta name="DC.Date" content="2016-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Shwartz et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Cohesin is a multi-subunit complex that plays an essential role in genome stability. Initial association of cohesin with chromosomes requires the loader—a heterodimer composed of [Scc4][1] and [Scc2][2]. However, very little is known about the loader’s mechanism of action. In this study, we performed a genetic screen to identify functional domains in the [Scc4][1] subunit of the loader. We isolated [scc4][1] mutant alleles that, when overexpressed, have a dominant negative effect on cell viability. We defined a small region in the N terminus of [Scc4][1] that is dominant negative when overexpressed, and on which [Scc2][2]/[Scc4][1] activity depends. When the mutant alleles are expressed as a single copy, they are recessive and do not support cell viability, cohesion, cohesin loading or [Scc4][1] chromatin binding. In addition, we show that the mutants investigated reduce, but do not eliminate, the interaction of [Scc4][1] with either [Scc2][2] or cohesin. However, we show that [Scc4][1] cannot bind cohesin in the absence of [Scc2][2]. Our results provide new insight into the roles of [Scc4][1] in cohesin loading, and contribute to deciphering the loading mechanism.

 [1]: http://www.yeastgenome.org/locus/S000000949/overview
 [2]: http://www.yeastgenome.org/locus/S000002588/overview" />
<meta name="DC.Contributor" content="Michal Shwartz" />
<meta name="DC.Contributor" content="Avi Matityahu" />
<meta name="DC.Contributor" content="Itay Onn" />
<meta name="article:published_time" content="2016-08-01" />
<meta name="article:section" content="Mutant Screen Report" />
<meta name="citation_title" content="Identification of Functional Domains in the Cohesin Loader Subunit Scc4 by a Random Insertion/Dominant Negative Screen" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Cohesin is a multi-subunit complex that plays an essential role in genome stability. Initial association of cohesin with chromosomes requires the loader—a heterodimer composed of Scc4 and Scc2. However, very little is known about the loader’s mechanism of action. In this study, we performed a genetic screen to identify functional domains in the Scc4 subunit of the loader. We isolated scc4 mutant alleles that, when overexpressed, have a dominant negative effect on cell viability. We defined a small region in the N terminus of Scc4 that is dominant negative when overexpressed, and on which Scc2/Scc4 activity depends. When the mutant alleles are expressed as a single copy, they are recessive and do not support cell viability, cohesion, cohesin loading or Scc4 chromatin binding. In addition, we show that the mutants investigated reduce, but do not eliminate, the interaction of Scc4 with either Scc2 or cohesin. However, we show that Scc4 cannot bind cohesin in the absence of Scc2. Our results provide new insight into the roles of Scc4 in cohesin loading, and contribute to deciphering the loading mechanism.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/08/01" />
<meta name="citation_mjid" content="ggg;6/8/2655" />
<meta name="citation_id" content="6/8/2655" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/8/2655" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/8/2655.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/8/2655.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/8/2655.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.031674" />
<meta name="citation_pmid" content="27280786" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Mutant Screen Report" />
<meta name="citation_firstpage" content="2655" />
<meta name="citation_lastpage" content="2663" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Michal Shwartz" />
<meta name="citation_author_institution" content="Faculty of Medicine in the Galilee, Bar-Ilan University, Safed, 1311502, Israel" />
<meta name="citation_author" content="Avi Matityahu" />
<meta name="citation_author_institution" content="Faculty of Medicine in the Galilee, Bar-Ilan University, Safed, 1311502, Israel" />
<meta name="citation_author" content="Itay Onn" />
<meta name="citation_author_institution" content="Faculty of Medicine in the Galilee, Bar-Ilan University, Safed, 1311502, Israel" />
<meta name="citation_author_email" content="Itay.Onn@biu.ac.il" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=P. Arumugam;citation_author=S. Gruber;citation_author=K. Tanaka;citation_author=C. H. Haering;citation_author=K. Mechtler;citation_title=ATP hydrolysis is required for cohesin's association with chromosomes.;citation_pages=1941-1953;citation_volume=13;citation_year=2003;citation_issue=22;citation_pmid=14614819;citation_doi=10.1016/j.cub.2003.10.036" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=V. P. Bermudez;citation_author=A. Farina;citation_author=T. L. Higashi;citation_author=F. Du;citation_author=I. Tappin;citation_title=In vitro loading of human cohesin on DNA by the human Scc2-Scc4 loader complex.;citation_volume=109;citation_year=2012;citation_pmid=22628566;citation_doi=10.1073/pnas.1206840109" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=W. C. Chao;citation_author=Y. Murayama;citation_author=S. Munoz;citation_author=A. Costa;citation_author=F. Uhlmann;citation_title=Structural studies reveal the functional modularity of the Scc2-Scc4 cohesin loader.;citation_pages=719-725;citation_volume=12;citation_year=2015;citation_pmid=26212329;citation_doi=10.1016/j.celrep.2015.06.071" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=R. Ciosk;citation_author=M. Shirayama;citation_author=A. Shevchenko;citation_author=T. Tanaka;citation_author=A. Toth;citation_title=Cohesin's binding to chromosomes depends on a separate complex consisting of Scc2 and Scc4 proteins.;citation_pages=243-254;citation_volume=5;citation_year=2000;citation_issue=2;citation_pmid=10882066;citation_doi=10.1016/S1097-2765(00)80420-7" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=D. Dorsett;citation_author=J. C. Eissenberg;citation_author=Z. Misulovin;citation_author=A. Martens;citation_author=B. Redding;citation_title=Effects of sister chromatid cohesion proteins on cut gene expression during wing development in Drosophila.;citation_volume=132;citation_year=2005;citation_pmid=16207752;citation_doi=10.1242/dev.02064" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=T. Eng;citation_author=V. Guacci;citation_author=D. Koshland;citation_title=ROCC, a conserved region in cohesin’s Mcd1 subunit, is essential for the proper regulation of the maintenance of cohesion and establishment of condensation.;citation_volume=25;citation_year=2014;citation_pmid=24966169;citation_doi=10.1091/mbc.E14-04-0929" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=J. Fernius;citation_author=O. O. Nerusheva;citation_author=S. Galander;citation_author=L. Alves Fde;citation_author=J. Rappsilber;citation_title=Cohesin-dependent association of scc2/4 with the centromere initiates pericentromeric cohesion establishment.;citation_pages=599-606;citation_volume=23;citation_year=2013;citation_pmid=23499533;citation_doi=10.1016/j.cub.2013.02.022" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Gruber;citation_author=P. Arumugam;citation_author=Y. Katou;citation_author=D. Kuglitsch;citation_author=W. Helmhart;citation_title=Evidence that loading of cohesin onto chromosomes involves opening of its SMC hinge.;citation_pages=523-537;citation_volume=127;citation_year=2006;citation_issue=3;citation_pmid=17081975;citation_doi=10.1016/j.cell.2006.08.048" />
<meta name="citation_reference" content="citation_title=Guide to Yeast Genetics and Molecular Biology;citation_author=C. Guthrie;citation_author=G. R. E. Fink;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_title=Structural evidence for Scc4-dependent localization of cohesin loading.;citation_author=S. M. Hinshaw;citation_author=V. Makrantoni;citation_author=A. Kerr;citation_author=A. L. Marston;citation_author=S. C. Harrison;citation_volume=4;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=B. Hu;citation_author=T. Itoh;citation_author=A. Mishra;citation_author=Y. Katoh;citation_author=K. L. Chan;citation_title=ATP hydrolysis is required for relocating cohesin from sites occupied by its Scc2/4 loading complex.;citation_pages=12-24;citation_volume=21;citation_year=2011;citation_issue=1;citation_pmid=21185190;citation_doi=10.1016/j.cub.2010.12.004" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Mol. Cell Biol.;citation_author=K. Jeppsson;citation_author=T. Kanno;citation_author=K. Shirahige;citation_author=C. Sjogren;citation_title=The maintenance of chromosome structure: positioning and functioning of SMC complexes.;citation_pages=601-614;citation_volume=15;citation_year=2014;citation_pmid=25145851;citation_doi=10.1038/nrm3857" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=I. Kogut;citation_author=J. Wang;citation_author=V. Guacci;citation_author=R. K. Mistry;citation_author=P. C. Megee;citation_title=The Scc2/Scc4 cohesin loader determines the distribution of cohesin on budding yeast chromosomes;citation_pages=2345-2357;citation_volume=23;citation_year=2009;citation_issue=19;citation_pmid=19797771;citation_doi=10.1101/gad.1819409" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=I. D. Krantz;citation_author=J. McCallum;citation_author=C. DeScipio;citation_author=M. Kaur;citation_author=L. A. Gillis;citation_title=Cornelia de Lange syndrome is caused by mutations in NIPBL, the human homolog of Drosophila melanogaster Nipped-B.;citation_pages=631-635;citation_volume=36;citation_year=2004;citation_issue=6;citation_pmid=15146186;citation_doi=10.1038/ng1364" />
<meta name="citation_reference" content="citation_journal_title=Biophysical Journal;citation_journal_abbrev=Biophys. J;citation_author=O. Kurkcuoglu;citation_author=P. A. Bates;citation_title=Mechanism of cohesin loading onto chromosomes: a conformational dynamics study.;citation_pages=1212-1220;citation_volume=99;citation_year=2010;citation_issue=4;citation_pmid=20713005;citation_doi=10.1016/j.bpj.2010.06.006" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=R. Ladurner;citation_author=V. Bhaskara;citation_author=P. J. Huis in ’t Veld;citation_author=I. F. Davidson;citation_author=E. Kreidl;citation_title=Cohesin’s ATPase activity couples cohesin loading onto DNA with Smc3 acetylation.;citation_pages=2228-2237;citation_volume=24;citation_year=2014;citation_pmid=25220052;citation_doi=10.1016/j.cub.2014.08.011" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=W. Lin;citation_author=H. Jin;citation_author=X. Liu;citation_author=K. Hampton;citation_author=H. G. Yu;citation_title=Scc2 regulates gene expression by recruiting cohesin to the chromosome as a transcriptional activator during yeast meiosis.;citation_volume=22;citation_year=2011;citation_pmid=21508318;citation_doi=10.1091/mbc.E10-06-0545" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=L. Lopez-Serra;citation_author=G. Kelly;citation_author=H. Patel;citation_author=A. Stewart;citation_author=F. Uhlmann;citation_title=The Scc2-Scc4 complex acts in sister chromatid cohesion and transcriptional regulation by maintaining nucleosome-free regions.;citation_pages=1147-1151;citation_volume=46;citation_year=2014;citation_pmid=25173104;citation_doi=10.1038/ng.3080" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. L. Marston;citation_title=Chromosome Segregation in Budding Yeast: Sister Chromatid Cohesion and Related Mechanisms;citation_pages=31-63;citation_volume=196;citation_year=2014;citation_issue=1;citation_pmid=24395824;citation_doi=10.1534/genetics.112.145144" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Milutinovich;citation_author=E. Unal;citation_author=C. Ward;citation_author=R. V. Skibbens;citation_author=D. Koshland;citation_title=A multi-step pathway for the establishment of sister chromatid cohesion.;citation_pages=e12-e12;citation_volume=3;citation_year=2007;citation_issue=1;citation_pmid=17238288;citation_doi=10.1371/journal.pgen.0030012" />
<meta name="citation_reference" content="citation_journal_title=Yeast;citation_author=M. Morawska;citation_author=H. D. Ulrich;citation_title=An expanded tool kit for the auxin-inducible degron system in budding yeast.;citation_pages=341-351;citation_volume=30;citation_year=2013;citation_pmid=23836714;citation_doi=10.1002/yea.2967" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=Y. Murayama;citation_author=F. Uhlmann;citation_title=Biochemical reconstitution of topological DNA binding by the cohesin ring.;citation_pages=367-371;citation_volume=505;citation_year=2014;citation_pmid=24291789;citation_doi=10.1038/nature12867" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=Y. Murayama;citation_author=F. Uhlmann;citation_title=DNA entry into and exit out of the cohesin ring by an interlocking gate mechanism.;citation_pages=1628-1640;citation_volume=163;citation_year=2015;citation_pmid=26687354;citation_doi=10.1016/j.cell.2015.11.030" />
<meta name="citation_reference" content="citation_journal_title=Annual review of cell and developmental biology;citation_journal_abbrev=Annu Rev Cell Dev Biol;citation_author=I. Onn;citation_author=J. M. Heidinger-Pauli;citation_author=V. Guacci;citation_author=E. Unal;citation_author=D. E. Koshland;citation_title=Sister chromatid cohesion: a simple concept with a complex reality.;citation_pages=105-129;citation_volume=24;citation_year=2008;citation_pmid=18616427;citation_doi=10.1146/annurev.cellbio.24.110707.175350" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=O. Orgil;citation_author=A. Matityahu;citation_author=T. Eng;citation_author=V. Guacci;citation_author=D. Koshland;citation_title=A conserved domain in the scc3 subunit of cohesin mediates the interaction with both mcd1 and the cohesin loader complex.;citation_pages=e1005036;citation_volume=11;citation_year=2015;citation_pmid=25748820;citation_doi=10.1371/journal.pgen.1005036" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Rudra;citation_author=R. V. Skibbens;citation_title=Chl1 DNA helicase regulates Scc2 deposition specifically during DNA-replication in Saccharomyces cerevisiae.;citation_pages=e75435;citation_volume=8;citation_year=2013;citation_pmid=24086532;citation_doi=10.1371/journal.pone.0075435" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=V. C. Seitan;citation_author=P. Banks;citation_author=S. Laval;citation_author=N. A. Majid;citation_author=D. Dorsett;citation_title=Metazoan Scc4 homologs link sister chromatid cohesion to cell and axon migration guidance.;citation_pages=e242-e242;citation_volume=4;citation_year=2006;citation_issue=8;citation_pmid=16802858;citation_doi=10.1371/journal.pbio.0040242" />
<meta name="citation_reference" content="citation_journal_title=Genesis;citation_author=T. G. Smith;citation_author=S. Laval;citation_author=F. Chen;citation_author=M. J. Rock;citation_author=T. Strachan;citation_title=Neural crest cell-specific inactivation of Nipbl or Mau2 during mouse development results in a late onset of craniofacial defects.;citation_pages=687-694;citation_volume=52;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=E. Watrin;citation_author=A. Schleiffer;citation_author=K. Tanaka;citation_author=F. Eisenhaber;citation_author=K. Nasmyth;citation_title=Human Scc4 is required for cohesin binding to chromatin, sister-chromatid cohesion, and mitotic progression.;citation_pages=863-874;citation_volume=16;citation_year=2006;citation_issue=9;citation_pmid=16682347;citation_doi=10.1016/j.cub.2006.03.049" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=S. Weitzer;citation_author=C. Lehane;citation_author=F. Uhlmann;citation_title=A model for ATP hydrolysis-dependent binding of cohesin to DNA.;citation_pages=1930-1940;citation_volume=13;citation_year=2003;citation_issue=22;citation_pmid=14614818;citation_doi=10.1016/j.cub.2003.10.030" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Woodman;citation_author=T. Fara;citation_author=M. Dzieciatkowska;citation_author=M. Trejo;citation_author=N. Luong;citation_title=Cell cycle-specific cleavage of Scc2 regulates its cohesin deposition activity.;citation_volume=111;citation_year=2014;citation_pmid=24778232;citation_doi=10.1073/pnas.1321722111" />
<meta name="citation_reference" content="citation_journal_title=Cell Cycle;citation_author=M. Zakari;citation_author=J. L. Gerton;citation_title=The SMC loader Scc2 regulates gene expression.;citation_pages=943;citation_volume=14;citation_year=2015;citation_pmid=25715109;citation_doi=10.1080/15384101.2015.1010959" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Identification of Functional Domains in the Cohesin Loader Subunit Scc4 by a Random Insertion/Dominant Negative Screen" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="twitter:description" content="Cohesin is a multi-subunit complex that plays an essential role in genome stability. Initial association of cohesin with chromosomes requires the loader—a heterodimer composed of [Scc4][1] and [Scc2][2]. However, very little is known about the loader’s mechanism of action. In this study, we performed a genetic screen to identify functional domains in the [Scc4][1] subunit of the loader. We isolated [scc4][1] mutant alleles that, when overexpressed, have a dominant negative effect on cell viability. We defined a small region in the N terminus of [Scc4][1] that is dominant negative when overexpressed, and on which [Scc2][2]/[Scc4][1] activity depends. When the mutant alleles are expressed as a single copy, they are recessive and do not support cell viability, cohesion, cohesin loading or [Scc4][1] chromatin binding. In addition, we show that the mutants investigated reduce, but do not eliminate, the interaction of [Scc4][1] with either [Scc2][2] or cohesin. However, we show that [Scc4][1] cannot bind cohesin in the absence of [Scc2][2]. Our results provide new insight into the roles of [Scc4][1] in cohesin loading, and contribute to deciphering the loading mechanism.

 [1]: http://www.yeastgenome.org/locus/S000000949/overview
 [2]: http://www.yeastgenome.org/locus/S000002588/overview" />
<meta name="og-title" property="og:title" content="Identification of Functional Domains in the Cohesin Loader Subunit Scc4 by a Random Insertion/Dominant Negative Screen" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/8/2655" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Cohesin is a multi-subunit complex that plays an essential role in genome stability. Initial association of cohesin with chromosomes requires the loader—a heterodimer composed of [Scc4][1] and [Scc2][2]. However, very little is known about the loader’s mechanism of action. In this study, we performed a genetic screen to identify functional domains in the [Scc4][1] subunit of the loader. We isolated [scc4][1] mutant alleles that, when overexpressed, have a dominant negative effect on cell viability. We defined a small region in the N terminus of [Scc4][1] that is dominant negative when overexpressed, and on which [Scc2][2]/[Scc4][1] activity depends. When the mutant alleles are expressed as a single copy, they are recessive and do not support cell viability, cohesion, cohesin loading or [Scc4][1] chromatin binding. In addition, we show that the mutants investigated reduce, but do not eliminate, the interaction of [Scc4][1] with either [Scc2][2] or cohesin. However, we show that [Scc4][1] cannot bind cohesin in the absence of [Scc2][2]. Our results provide new insight into the roles of [Scc4][1] in cohesin loading, and contribute to deciphering the loading mechanism.

 [1]: http://www.yeastgenome.org/locus/S000000949/overview
 [2]: http://www.yeastgenome.org/locus/S000002588/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
    <title>Identification of Functional Domains in the Cohesin Loader Subunit Scc4 by a Random Insertion/Dominant Negative Screen | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ZtPI6yltzkTaloU_9lwKK0o-wSRaDrvI72aIpAYv01U__yCSGhNVuCbGf6KLL54AUS64uP6kQfk1f_X5fJJp6-FY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__36Yiqb2U0AhDmqVjDsvW59h3D9k7d0px_jM-NJ9exZg__uqIT76aUscmJwriA5WgUcByCR2EI-ARNdlG2HGF7Oec__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Wku980y13ksskaocSqUQrMFUYtZttHvUgmFUYlt97QE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460259","apath":"\/ggg\/6\/8\/2655.atom","pisa":"ggg;6\/8\/2655","ac":{"\/ggg\/6\/8\/2655.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/8\/2655.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2655"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2655"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/8\/2655.abstract":true,"\/content\/6\/8\/2655":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wmybeDd9crhOmbwvA7PHfjYGl1T\\\\\\\/wamwbcpmj\\\\\\\/GOQsOT6mJzOrlNM21B2ww1ieea4iWloJFtph79GZZRLmaZ08pIwAK8N6FzaU2sHFlEjrHbwXc0KjmSXvwn2pVQnDSfyskoHfriE\\\\\\\/DneC+IR00ECZkdFVXpvxe0JhEEwGCBTiEye0MhuEuXP+ptwW7WcQ8gZz9rlOaKVmVwPWZdCRd6nHJQH45SixXIFZYxHdCy3OF0Ko4y\\\\\\\/Cu66S5u1I181OTd\\\\\\\/9RFv3b2XNMuM3JOvdiVDP4Tv\\\\\\\/BXbFiZNC0sLgCde8D1tfEIOCzLREcv3xxrJYphosAQpw3wioktFqDrKUuVSj5Y\\\\\\\/7LmvonbHdI5qI+YIuvdRX81VLQ949w8nXU5F5RFQxuqhII3LXWRHq4hrj29\\\\\\\/I8rdu\\\\\\\/D4HFLvnLTpBPzzLt08JX6lbB8HqdcLo0pV93fwzlHilCmLNb3KtP+HVg+8PPOsfLU9TgDkuCuUL37SThOxkII+HGPXhdqPtvf\\\\\\\/0jKeqPs1yq3JOF1nZngU943KL+j+MaYo1gbvv3CpB6Ex6+4OmDus9+hiAkYPkZxdijZ4tq1+tuhM\\\\\\\/4H\\\\\\\/ZbzmfEuxdPcxmQFVrYJaZKEjPQqnUsxePmNyep+JPtztT33N\\\\\\\/7lBrUmn2bxQ7m\\\\\\\/UjtcMYr++n66AIYXvRhMTbX0cOlDQPSGQGrS5bK9xku1iP07HdPpaxj4FV7nesSNcUyODSmD6EC6Uh3vXUft8Vj04d84o7GuwWlKcTWj76i0fL6Gu2160vpbnKtVAsHj52aYSclcWUEVUv3avdxGTQ2QCd2ZvetY848AI4Dr+p+xqNnE0t3+pE6LD0YBSbuXou9PtiAWTZ6Or\\\\\\\/ZfvA0w4ZoHQF0pbwb44YVJMykF5FTAioJCNqFjWHvY6N1IaL7Ux3OI+phuWvyrsso5IKTLyZeZGQf2G\\\\\\\/HrN6vFnlghMyb+QUCXcwHuVEepfISfzn8ukos5zBmd49VngGHWC2AgywjPoFS0JK3S\\\\\\\/j5Js6H0czeFoU7iaNn70DRHnDzmACcXgQdWHsJmXuFQq351+dVo0ib2lvK9aNlGQMJksmCuYwsch\\\\\\\/14Nx14fAC\\\\\\\/j6R\\\\\\\/Z+gsBVJDzKib6AWFOQ\\\\\\\/BKhPuTb9arUgRAlzyslktGqrnNvnkW8BF\\\\\\\/bpq79OYT7V8Ht\\\\\\\/SXByHYARn8aISv5NQ4HK4pgDDrOumGCWy4qnGlMyy\\\\\\\/61Kb\\\\\\\/wiMKIJn5s9ltcQ1fu+qfJ+xv9I2sHjQwYq3+Vase8etD3HnnhswZB0nP31RhbcbPaL++nM\\\u0022,\\\u0022iv\\\u0022:\\\u0022W2SG23iA09wYSm\\\\\\\/YEvsyDw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002212f3a16802e67b1f2bf85922659330f1dca52fca35223a518a352c917f6d8108\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xQDFYV\\\\\\\/7bNvfAewUAkw7TRlu6uA5Av2QCA0nZQ71A3V+SzyXYiFPuPi2ez8kwmcpBfJSEUYAmxLeVsdtgQiLnL7kRB4SUknuDLn+paoa94lP5RrzzdbvYu7GEZ0AXK4eEdVT0SUNV+ncn8nogdu3y38ayPjm6NCcNFmgomgqufZr7y5APvKdsAm293YQ2mLuBTonXkGLECbGgYRDu9yjLklkQ2hq+kPNj+aGZrfGR8CpUJ+LGkonzLSaiI8VCXWse9EJsg\\\\\\\/1oMkbS2X7xhvUime3qM+LrR3yH9KTspOesKfMhPZWjTM+UXIyiOx4JFggliQrifVCgV9G7NqSz5KWKQ43\\\\\\\/ha5bmdZkJu5P79ua4NKRzGIjJ7+2Qu\\\\\\\/vcsCEFKfcfitdyUUBp+W6hRm5y4K9AWXRKuvu09zQj4z6ryc\\\\\\\/\\\\\\\/bjTdk5uBCWA6ESWGcxB0EF0YYi3Zg9aEVsTjwam\\\\\\\/pDNRKut5sz2B5shNC2QqMWTPEDKZPD3hIaY4xdvGHQid5XE3z9r6OYkKD2LYgXSYJ7DvqiXA7ywIobJCBC4phdqXju0Njcs0Zr1J2zC\\\\\\\/Akl2vVMlIZvUggtXDUBNVDjXw4x5rCd8E0wGhDP9kAHA0OpXkyBPmkxpdIA+DWlvyObtui3heRhQW9t3\\\\\\\/2TjPSbomYF47pU\\\\\\\/F6r+lwl1L4Dmo7WORc1aOa3N1mL7DFCTj7TEri98z2GezBzgyI2mfkXOvD53\\\\\\\/lWZxYnn\\\\\\\/tNzQ+sFz0VkAHWXEAmpOD1e3A6Ki1fWndqCxzMBcHRndd5vMBkWgUC3xjfjE6Yal1JUeeI4dnh8e9S2G0iprQoulNSKuU7amdKI59QsaW9GGu\\\\\\\/lC5FeuMa2yDjQlPKppfr65B322xCu\\\\\\\/oxva91FdvBQgJxycpUsetTosx7\\\\\\\/\\\\\\\/4qk37c\\\\\\\/bq2J7b2yHcq9fABXideAkhCVd\\\\\\\/FjSHldT8VV6E\\\\\\\/tWMm908bwx4EmGu5ZE4zO1e+5gCTchZHlSTRaHLqFwwvHcHoCqgbXhiugBGj2r9Oia8OyCAWn2b2CEYWWdJw\\\\\\\/2PVCFVjO5\\\\\\\/qhmKa1v1WkdgxAcqqjJQSnCE6BxQzmMOKitBCfP94w1Vc9Vob+k3L7VKh4Mlh0\\\\\\\/ZaDyaJzoaXQWMcoI19kljps5hjU+\\\\\\\/0JjEKZrdiJGd72gK0nVOk2JjnJRZqIGpfx4T0SRNPOk1HL8vZsB2Kft7KRnW5Kp\\\\\\\/ep9q0W2R1QL8g4WCrZMfCGteCHe1ZD7gViJI+9YIXHDFtQUh0u4lwmNIrgMv9fBzbRkyPM54qkP+p\\\\\\\/PPT10YrcuPjqVGQpNwwzfwNIiZZroK93PB7fnehpQrqshTjYXnZrddPBhJS82+AnWyOK5av8271sS7Hk+NyoqYCnp1a+elg8\\\\\\\/ytZh9eLQzmLSRkY9C\\\\\\\/fORzaTnkRySW5J+12updIMIHWZOh1Ie1uWZ6E\\\\\\\/4NVx43CGPTwF9D4ZITFclY26psLWsh3HblAj4W77yvVdISubrwBmFV7tHTQGgFaZaYIP70Pxc6ALMsXSJFPY20LJbsVmEtsPcTQ1Fz+U4wT5xqdbOqAp7JEIqAJ41ZqsqlL74h6Twn1Lm2MPFD2VmIA1D65vlluefdQeDE76yJGxFqICjTVTHuiLAqhohVIFQtnb3DPt7TvtbcGRZ+\\\\\\\/5eC7+1oPXCvegmQ85G73PSx5EO7kqt+D6kYnZua1+C6lsPvZ3nB7EIaOGqi5pA7RczOUKtZVi00Yh483p4\\\\\\\/FIslR7FUFmq9pvJH22q4mulNkj4\\\\\\\/rNnm61fZ8dmbhq0yFJ5Agi0H435qRJ6GGxVNUiir3+CAHt5QFDam5Tt5sj6T07G3VAwkhl3zhB4yO11I3bDCqyimKckPfwrIMdNdHTEhVRq053cuCEmHH\\\\\\\/UZkSJEzc3TMNMIl2TUgIp\\\\\\\/m8fWIgN8VCSqNlUSfeahnxu+gnimjN+emrSM0j5kF5KoSsqZMr+fY\\\\\\\/4Gf4QKQx4TvmY3\\\\\\\/+gFAIaPqktkksyftzoFitNKEtg862OrSCpEBe\\\\\\\/puf4q0Ttbrz60jbUe8aSihRHVzS\\\\\\\/QSSaJ3A0KO8G2bAlGPjjCI7x0xYD\\\\\\\/8NOjgV8o3ejqVSjnz5H4y3BAbMPZNvbXwD\\\\\\\/RqIcNMQaRHbWql1RD1ukz9q0JEN0t9VYOfb\\\\\\\/aJxH4F2jev4gCLMzmYc5USvl\\\\\\\/kjyc478jcm485PLT1vmm4xV0MetvsneO6ANO41KZCSb2QhnJ8T2JDj8JTT9pRhIsdO3C+wBKfhIoIHa18+6v2N3MQlbn6\\\\\\\/3Yb4zOLyLDwfJB49nQl6\\\\\\\/knd5\\\\\\\/rSxRkEXgq7ZR+YjQBCMLL\\\\\\\/B8C1+Oi1vHRnwknsRZ3PVZCfN1q09PuT0CXwhzTTQ60eoHN4KmoMTWwaFUpQtT\\\\\\\/uvg\\\\\\\/MOL2bO6LTTdDB0d0Fc1XrbJLF44dP0hGfZlho7pMyU7OKwOfvY3qkLEzzJvvpwYTvcpjJGKHLRGEGYigzwdSf3NQbodnbzShhnYwQ2WdeWQwTcRW2ce9GSNn1d5iC6VW\\\\\\\/ndlZG8B03gf+Xg1vXZ8ZWa1BkUtivGQyeSeO9zax0ifgjfJ0Ngp3Oe1wavwWXfB+1zxyI8bitZh+s87eo2LT+CPveaHKbHDYHIPAfH6sEwYQeKulVJ4Rn5eMzZ8dvVgRHrtYX5B7W3EPYsqvfDnREWPRDzIWR4J7d2wS5Ccp1KRufJ0fyDEd3pCg7pkDRFaOhfSPUjTz8Ee5ADJ\\\\\\\/jGPbwqhkdZsReQ+jDKdFH1fZYZB7lo8oZd4lAI8fdo8EAhWq10Qifvv1lEZ6y\\\\\\\/dtpLqVFGAFleIb5UH2+77NZiO9zIJSHrQXM3EXcxKqqRzrREYpgsMEcHbCTHrtqFt6NTIVbfRMTkoKBs4lq9Q37U8AWnC+WMD9kNzuVk4cDKnVgASYLy6Jn1odiIpsPdN0xVsm2AHNWYHArqIGYeZN64J01WsDWDCoFd0Hv9XXTt27cGD2jooFOVFwwkXM8EKe6ViGs4WCbAzsPr3cwZ+ufRKTyJ7\\\\\\\/LxzwAvm\\\\\\\/naCd+kvlR4BY6McRbILjl5jP80llVnCmdW1UObhxx6b6VtBvcQu0wELdN0rCBCR2lB9nRU9\\\\\\\/YXD1k5V4k71mfI04MdTUwJXRw2Mg4ZBHaodgmFDnYakJTiIr6NbnA6tf2x\\\\\\\/cR8Jwih2ArDFL0ruoTxwSyQ6jM1Vx2AnSvMP0AGjymOEDPNxG3STb15bm7JDj9jZe7mZnUdepLg9FZoXEZdIwNNhKSkvptih3ohseFYFnzXwx3lZ5b88Eh+JaeO03uyLMiFkIX9OJi1evK0rKQREAMNaPPdGCkVD83wl3lI8gngnBOY8UNdJGZksEo4HKYW1OhcYHhOZYUVbCaayk3\\\\\\\/hhcKWLxPujXFNb4ieq89\\\\\\\/5\\\\\\\/VhqeM1+F+uksPHHgCONH3JOhtX4GDUijKCpVyJ2bRuRzN+VRR0WlcbzadzAGMJw32m0vLUan3qQRa88e3IP0KS6Ju8q4WiWX6\\\\\\\/mKxvzCzMauOOT0BrIjRwWpK41DWjBoiOUbF2688zpBdiYykbuVBrw9xZGf5wmHrVIxrDjQlF4IepXugvKnH2lmW2HvYh12c0WouOdxVuy\\\\\\\/CVDosRNkH8n8s1ckBuwP149UsrzA0YPlkyVyn9fgOYnHFqHTrg\\\\\\\/3k42yWjpLdPFU1zVfD6G3F7kfnyf1Ie6ggttPJ8r9rXQucXyGgIXZDPL4JaxzavQEvp9DXbx8StRgrxRVF9k5Zhj4AijFynx9WEX2lN+gU2N1xuZsKjD2vitFcGEE0++KKy6Cu8b+xZUqNq4lD7V\\\\\\\/Jey4+x6aXwx\\\\\\\/Ny\\\\\\\/ZkB2JfWk3zKIA9tDxk\\\\\\\/G+M1cnK1SaoGG0JZpDTABLDVjhHWys1U5CV5eVFDFoEUwIA9rEp+hMPdwiNEILuEW1EPWugqg9KzRRt30mWy0BYaGWLmwftx3+1YMKNh\\\\\\\/1HmDNkLMCflW3OC81sNbfno3CuoreHFv4kglmDuugrnntE38bZF4KLqfoW83uWz3BOBFOtisF0cMXFHurf0PTMOEAr3PE4Cdi\\\\\\\/2kvui0OW+wCSzfS3Kc50unYSjmzs0PryPNiXkH51ZJeTiy3MO7R20lnEK9v9ahY\\\\\\\/6fc8LswuZHCyH+gsPfGsNl1MMQ5hy3JiOPFnBW4ftlSyxN6b59i0nlH6KeSGhpWRk3fASP4P+AeThNkwBzUmudfIoCjV2Llra0iO7Hhb8rxnmHYuhyHpngz3ILP1pZay3mtS1ops5BAsayIe7dcA1kI6h9RZ+qSHskPXWRGxnW8re\\\\\\\/y3gzLgkzHywNNzYaVShFpnvaHEcasgakHST3YqPndJYhc1Z43pHcrqO4TtJ59ngbn\\\\\\\/yUSPDY9BhJ2dOPaMgXKPzc2kMsjCdviUKvh6ZokJoOSeVx9xHjImvY2rE3vsHpljFsoV9Ns+GDvNIQIF8rNf+JWg4QIB8vLkFhu6vSMgY+ZsykbxU29AHHWRNEv9LHt\\\\\\\/LuqIo+AjpdBrRbI9QRPg8ih+E1fZIZVrjIU04XRKdeIDC\\\\\\\/+JV4kceaYz8T4iVORj8lgCIlWU2sLKxdjaZlYU8e2v9WwJHfHrMKUhm9sSKDqz5vDPcxi\\\\\\\/EJpZTRIOod5Uc0qaLkl36KjhPxuTd46S5XEWunXzEvdCciSp\\\\\\\/N25CbDxCiTvGeGki+\\\\\\\/ba3IXDQwcyPSUYKCm71\\\\\\\/btmu9NTCgsX8OAsWcKZpuK5N2fTo\\\\\\\/pvXRhYxSmp0HQWtIrTX5W70vvnF0P9B\\\\\\\/Nm8jU09JwkkuHl0IeEHrUUEnQ8LCxTJxeS7N8ve1oJI9NKMf23lceCXrItm1dLNCRQLkcDvL647LFPUszQB+8V73XINLiO6MT2Tvn6QfhGRRrV5sN8+E63z4YaqRZLPBOXKOpxbFdkMr5qapz7YozczoOGfc54FcQk8m8rLEmp+WQgtpHamv19SmHKv0ugYVUrsiKlq4T6dP1RD1R2cNMEnXxZtaisy\\\\\\\/A9vguqyaC1RendGOBqDo719\\\\\\\/STvS5Lt0sPlMINum2HJflGdXt2LuupTWNg8+A\\\\\\\/xWqZq7GqS0dVgugOKAwjmSrw3tJBKS1ny3hgF9+IOOSKSmu6BV+BfIfK39xDmeCncvuTOhX\\\\\\\/30oNJifhJSYRpNh9RnpbMfe99SIrBAG1SwZfQb1YWJBs9A9qWsLgr8cmKw40R6Eg+pPHJYjKMwgURxcEtlMkJoui1RV1Itoa9FaCu\\\\\\\/oVg9SkJgXWQU23kRDQqKyAaLfrugzTDLc14Elh1ngu1+huG5uPVwUEh88POFCuGzNEhlN9XxARTmq\\\\\\\/+4oH\\\\\\\/B9phlEgkIU6wt0YbDW3+yTNyBCWDcTcvoxoPE5n800CUVPDy5hc+pbvKQSiY1sj8HhcZfCylmQ\\\\\\\/e+\\\\\\\/hiIPA7R+9jsMrc32tJqE4nD0CrNzde\\\\\\\/pVARCjFmlnWHUAI7XuKrFchLfKwp0UuYp+lO2qSb2McGRgcG7pGogrvLQmZxLXFp7FjuoNKcTQ10DEFRMnrSjl4yb1yNmbQyp1bTxEDFd1rA\\\\\\\/HBqJpqcFVOkQT7ll1xkzThZA0JY9Vgb821ppFwOiGbkp0HrYT2uyqss7VfEzR7rdmdD\\\\\\\/dRwSSS9vCbWGGdfBv87HBhxIX42nDkC03mAYop57YazUHsHjGAJkDn3Kgw2ED64szk1zQTIT1S8G5oTewj2pbtfDQC+aB4ygU2uMe03FNaR9\\\\\\\/ZFb2gca5QMRFShj1wIJiETAr6KKmX+r1ya5Jsm\\\\\\\/r6YDIEKukvtjtSupD1MDIP6fX907CRntm9yZRDrUSm7MA83sDYRO36bxRpd5vQko6ibPa\\\\\\\/nEuwM99biHP+Jjhfj8iPWQn701nohibcteIWpfM1sMh7O\\\\\\\/LBhvwuRcduwvEDax1nMX3LPk8I1jsz\\\\\\\/+wBsxC1dqgKSUmQqjWWB3GD72nCWd11ntu+nb8WQBo9xGe4NHrOnVKJpiDsZh\\\\\\\/0W\\\\\\\/Nfnrm2UP6z5u7fzyxE9NAUD8m2VVUdYt0fXxY2qJkqHYnxGwu4VHvBBxKyvajk3m7wRCXe++s0p8eesVBy+vT9TIOKpW7QYe0Gf9IM7Ahjin847OXW1nw6R48AI7i93c8TKHQZJ5wCHRqzTYo8j\\\\\\\/QnvxUZLxBWhffMAawEyBDfFb9gEMwUUtOBqMtKIjKQp6aHiICkmKoYiqoJHHR01X11xlq8PjUol79uuzM+sAy3VVTQkMZSUgb8pATvOBYeVD0Cpyi54EXAdqQvJVw5J9xXPCoLM7ufS036YBgNRThNHb2JzIxVlKh\\\\\\\/Pw97TBhUKCrMmi\\\\\\\/+jcISoivZAK6\\\\\\\/D\\\\\\\/bYZVa8C4r3Nrr7IeSzTDymybdmd5amWR0SUF1MDKeq0nCIBEXOMaBUqsl\\\\\\\/YBufwNinCnTb3+GY+l4llHhNZTHYaT27j2LBp4+C6TRzUipBlzaqCIQtaFaXkGjxq3Zzfe\\\\\\\/c2JdKIXzsR3cM+5629clzUZAJsc8xbWOVrIoSV+dzqAvWpe3DOIuGptZL\\\\\\\/hLe1LaZpYsGJZd5ZRtnasIOB8XanfNppwnskdtq6TEuKETDy0lSCqcALGwF9nukMG8AO7gRpxjtRJYqsqsP6U8jWHEiTt4XinJDddPohyTnQ55zjcgT1xH8Ik6awQsXf4q48f2R8gVLuJm4q5lZA3c\\\\\\\/glTK988q3inyKDoR5zcZBfOfnxFHcjQpPjqZ7Ro8UdvGqMEHs0lv3xJJYIV8FIoxY4GzaR+\\\\\\\/XGuZy2WRgTKDhrq69BTBXOel7Jdzygy\\\\\\\/pGYOe8BS1k6k3RUoqXUlokpI1dQnbxZxkvch50cOYvvpHbc9OUcdOsIS5AO4gkpxx6Okq6ZM+p8KdPg6+OIdud0i\\\\\\\/DrIueyz9PhFzk3aHPk+TXyT1lMWLc0aeIu0EHTQHSjxm1E4aFgEBAq\\\\\\\/oZMPrDeZh9iVQ1YPWIWnM+bZ+IoRhVPRyQ2fdqYcwebwnF96d\\\\\\\/LzzqXL1h\\\\\\\/pJ0EjpkPJa67lHMkNidwPciZgTuEDM1AryqejKEaoQrxZDtppWnwTXubhgGtT5DHgtfMvfEr52sdJWt2AqSddTmzRlirMaDsmOJse3DXRvucnENCh2WgqGUADarZFAQ33BRj9CWTMeD7xjXaTHQrzvPzAiQPB\\\\\\\/4FcP8nzPlyIV5mcdR6Cn9uA5cqEQ2WF5ydQbHY9SbIfZC03WjAe812YX4YS4BQfc62wyVm760p90sHvgKjBlrImmY3WVcDupUI8V5D1q\\\\\\\/2iIOsVOyoVTs9XO58LSbs8wJBA7t9hJYxksJpP+TjMsx+ATcYa+EY2y923MlPVJBa26WSU9Uv0dbG7CkRVHANsCeboPIdpxG7CuDavb\\\\\\\/RcbdR2I+3xcGVRJSt4PM3xLzbOhf2zw2OyJU+KPPH\\\\\\\/ibFgv+HGcr+UXfAJY\\\\\\\/MRqNmIDsjAtIIJhjzE773iFFVLJFsz15Cwo8bEcJo8z09Ql5asj9AWJDdEpXMh\\\\\\\/b6U7ieWRx8od0JYiZOLOs7VibhDAZ62RKtz2PgZYCnQEkmFEjX7aV7teXm1SksxIFMg4iZiyeWa3ekeoif\\\\\\\/ka8bt5RmB98eaC3swk8nCBV31SbB18+qKmz2O\\\\\\\/Vt8\\\\\\\/jU1T24T\\\\\\\/HqgZgF110T1rvlDGt3+x3YrjPwVjO+qWshwEkdE\\\\\\\/DNedtjHPn8JP9AF3ua50jdwQxPqSJi9ACMMAoEcY5ARialYCcKmnKejjIp\\\\\\\/f3uD5fZrKhnLugscBfd3ZBuTiu51p\\\\\\\/emTmeUoLG2FHmCw0fKoQgfuL1TVTbNTA6vDLG7gfcYHMstVk6P\\\\\\\/cFXXSI3LQy8rjxwPfhGtozz6Ao8HzfUpoM8XAOBZfEugHTGQezXBnHHeanEZym\\\\\\\/gK\\\\\\\/js+ZNbeOc+kbvgPet7CmUy5i9dv4D9UU5rbYX\\\\\\\/X3RR0dLQ1op342RXEtcooHoksJMqopFHFtzfNn18+H8ryjPQsoIybkX2JCeo8CKyRuvs8DSdfmKcPYt93ZHEjnw\\\\\\\/6t0kS\\\\\\\/rotDJUlYzoZCe9iAo51I9LbhX8sXW8xaEJcMSUp9MoS1FXw0Ach5IDClpscuRtzLGZ+YawNBOU5cazMwmgU+oepmc+jkkNtNVcggfyjNU2lm8rm7qYqfAWbr8Lvy5CvC+zWGNcoK8QwmrJJvFl3wTmTrm4zfZP0HR1OTS\\\\\\\/5AkWJqvAF7VSj4v+UNOlSmE1Ua0WlqsAQXZOJOdzIZ5\\\\\\\/KW2bD6Eh7OfDVi0uU\\\\\\\/FgaLC2SwQDZ7+nM9X6Vaj65iW9iVySEnc03TZnQVkLb2YiLUY1Y1iUS7AuO\\\\\\\/rWRx0T79xV\\\\\\\/X1gnnMkTgYlZQbY496OVzognV+K36O65BwVZw9IIirL9EOrNUUbyQBTWEnVlinw+vRk36efns9FtjS\\\\\\\/QhpRZvqicevDTsjWPM+OEAs8atLA0zlrVnJkvvfKNgsbtANYQnki6zSEib2nkCa\\\\\\\/DNgmiRNlf4F90DNWUFxm5w0\\\\\\\/3l7huOBIcOHbar4vcHQT\\\\\\\/7m16EuUXbNqEP+\\\\\\\/q7JY5VwKAsiY3X9+puOHNSKKe+4GzpEfdCTk+nAyDCkud28AR5VmewDSBV9Dm7w66IhsdRELAh8wAYkXHl3DzQNFUEj27QS1J1QCzGiXFV8KwJiLHgjGAiCYrwBYE8yoElL9J3DMRdlN+x\\\\\\\/IGthBCTwg1GJ4NkA604EzJsmswTzjpVzdBvw50rn0vXlxvvzwQnzW4yeZ9DGES1wAa2JXq3ArQ8Fx1EY8708yKPZr0H3rh1Nfr6ShGuvhOtUHx8tMJOdgYv7NzDZOF+1tRVOUX9GQBS70azbc4f17ZmkXxrgkb\\\\\\\/zi+QDib432lsDpybtaJzo9TIAqKKDNz+REFNEvEzVXRdkZirfW7PXjzQRRE7Z27dE5w5UY+ln\\\\\\\/l8+uKb+\\\\\\\/OMWl+I+s\\\\\\\/wcgKYiNMQ7H+L7IePJ9hV29otkZPWWjyozr6JjD90s6Im1QcAneiVjoKlrbnZObh3CW1we5GBBOBPGiZMxXHqrZW06mlDmuJkc8GX5ii4ij9kcIFueHJRqlxHNfoBE5nlkz41BHKH1OuHvVWTbyUZEDxU3ZIvwulJ2ucG6WHBVNUjwPN2I2rt5r3Pqbk+57Qe6+v5T+IKMs3B2x\\\\\\\/P7p+PG2wLWRPhPqOy8PDxV4lRR0qaDCWY9\\\\\\\/RHlQymjpyMt9oJ5CIgOCcjONuEUgXd258xAxv\\\\\\\/c1iQEq05qlYfTaPMqsZhLmzZ6cRykJivp6Jg7a4KZ9ntAY9E4GyjEs5RKbMKULuPGhMd2Of6YAjV6T9yfBvrm9G5\\\\\\\/+TlMnIB\\\\\\\/z0pxOCal7ZeeLy3CZNUkpfyxBOvGLzMl6vVO2RMs1zee9z+t3rWP8ulVmDjnfJf+gg2p4aZBJ7rSdz2ug7+iti+pCTuyKKHbokKiylheYHuyylbMUoru\\\\\\\/WwJsInAcHg9mb3MdbMWm55hvphqzAcq8DbcvwQOm3w9FDs1jNRK1hY8Pu8bKtHr4imRKdy1WADN1Ik8g+FpS\\\\\\\/pS3OZubRLYcQpLVFcikvqKLS0sjgCWR4T6l8Z2x94DukPpN00ddpIcfOFItsP9rM6MJEZXdj\\\\\\\/tTlfqnDtf6+RCG5OQAJykp8bf0B0ebFfLbgCXanLIoIY1+RuHDmE6FNkon0nw9mZ5PIOxGAeh8lZBFcBMLesJtzPIP1S7AaJsxajQD4kq+6Iw+5nSaVeKC6iGpI+jAPmCU\\\\\\\/ek5BzmXPBLUmmHJyuvxFq0v+TEebA4nVrxgMT1KnWRgHrVvXFLALzqgDdKkQjnN3WL4RpeZC4NkjaIzh4Zfo2qnVrd8of3IINa+CbginiGgQaWRHKMO\\\\\\\/ufXdz6TuoU2qPAsyyDaqWtkHrSneZUiZyMzi04uOFjRoGsYLiJmwZQewesywl67CMbrTgwk3NZaOCqs23HqMsRbKmulhgWN1aT8JBXl8HKNZ3WrvWw9uP7ozil2gwEOJg4k3XEVS7fJPutiuSJsIUSI2xL46LpF1DK8yJKdDC6HL8MfZlMxewU9nSq20mjsvVbllLKro+UxFajzBURZaKXrpyWqcJy\\\\\\\/GQ8TzYBgo3rQGuYXfC0mykuLRVVUAD0uC7NPlB0X5WgW8XywA0nzuhu0yte8BT+LC6UxsChyqSeYUhd28Qu\\\\\\\/rMwXwuUlx25aIhH4lwZQRtsoBLchUASDHtceOYfY6i5MI3rIcG5dZtMdwlTEAHgHufBXs104FhUDYyLGa3qjBiHA7SyML3mOEE6zkL6LDiiWnC8xrWDl0Ox7O6dIus4J4qQiKABVgl4fgO5qDcpw1l6zfSspE0msn1dmKZU38m5ytw38Gwxq1VaDzCw63GWWq3S+MfxzMxsCEv345dd0n8msAqiHKcXUm53s3\\\\\\\/XV5t90yoOWV5QAK7zxTul\\\\\\\/fr5XXmAZzoxXVzhqIe0GxAmmYxvDb69xZqLMrpkiFLfdAZJYBEowYJIQXzoLd90TWj9dtOZwn8+ric7K+lK20lMH8MyJwIAkudoom+Yq+agaoIwPsmz616VrffX8bA+sAnQLNM90TD8rWJFlaWHzDXt6L2kqrGfCs\\\\\\\/XPDCV7tivZK6+VbMLo5x385nHUOLxaGMoxWE2+V8umd\\\\\\\/\\\\\\\/jghF5kh5qfo9qSkeaU+LzooaQMd7mgb8w1HHSu9tXufV94jUsArTN6AdQ3C9FyHCdumCoEnxTRiRMBM9uxdEaANiAkccTdwT1Xf3HS1kVmJcDw6mFhWrppperarinNbjG2IexkCX2bJo6BhT5bOlhX4EaTZfhR55eJKnUzJKGkSHQdN42QHZl\\\\\\\/wY7n\\\\\\\/SwnHMFa1gSgWQuRi76\\\\\\\/lcJ09DP5WPeDU8nEhDih0CHhCZYtdWCZ14d+W0GJlU+fmaFsoAwM\\\\\\\/L2zW5iAtqDH\\\\\\\/cYlwaiC6KJchCjrKTdi6jRL8Thlgvw2JVdVwG7b4Bt56ayd30nMli8mgIPh6CjoZ2oKqhHywKOrvX8os\\\\\\\/CYUQnWEI6xBqIDhQwht5Us5z\\\\\\\/I3Cr4hRDnc\\\\\\\/ELIiz7R3RwNDus24r+3HTTp1VC6qb2ec9Da5f5VzxjE4VMI9lbDht2oSLRJ1ZzzRNpAhsyq6jmrv87RIsQKxzXCAT5xfRp7TXI07RRld4x1kb6kfTHab1Tr1eFX8za\\\\\\\/3npSc4od8wgebsbPTuBnkVxUck\\\\\\\/xpQW7gvHPijZFAii6YfhrPTgyHHSsq7yU8csZV4pnSD7\\\\\\\/0K0w6dHMmpCwcxoKH05f6oyQJGub8P16TPE1L4mxJeDnjCBPnQQWir8H4pUCBgcLLCD+5brdq\\\\\\\/UfTlalZHEcM7XAXZMxWwuw0TfUo8MfJgMVdjRl8EepZc2cJN029SlN4IvXgievhFZrtFPcOtK8Pr360TeVN7mMU5oZ2haEJIdWZA+nPLpQ1AL0VI5M0iEyjDnZr1XHJK+11wyntj3t\\\\\\\/HpIMUJZgRkkp7nRjAurJHh3jFYqIsmYI\\\\\\\/Zy6QgtL3ipRlh\\\\\\\/YFtgkkFlBxu9j00XQClH3KpkNFxCpwtJ31c9PpTdKLIVaraA7pXkMauA73kxK6BUDOBdx2rPv7XT3Bel6JYnARc6tO6EMADpaX3J7LgXtWarVDLqFCziIFIIMVpVT5Qxi+uV3f3F7OJr+0XYuEFg4eyomnRhwPn1FTqf8cYyM3yzUCr86p01TW78YhX5rU\\\\\\\/0mT1GPMlSKnnoYInm3997jB4LYsdSCyrTgQuEE6oEDntSLC773L9AXrFcesOQV2KqRMxZjAzy6idDa5mJ+FSwYswKPKAh4Zg508lWfV9EgAfZJ+wIwjftUzknUl5vUCLh18whgDLQCoAiuciVmx2PQM3njtkhB8PGyXvyiE5tzHtO1g2O9pKAYb+ZSF\\\\\\\/VprXsvDyO+H7kvQE+LjFRgW39Q3zzCTnEN1WtzoQJOlzODfoMbnLti+ZzJGy1iYLfM9GLv0TBWYhgJQkq0IN\\\\\\\/1F+GcmF1Ef18Pz6Z8gdFIaQmNBQYkDogEBgINlXpbjfnEryO6lcFlGamTQjAGUcEnCZjXhR6UW1IJnZ00wCzCLGoxuxtfzrq+zHNYXaNlkcuO+Gyy5dCQF\\\\\\\/iTXA7XcjVIfjhd9oSIvqLL1wIqKaQBjnrRCUhPkPUp6comoDSYo+qFKXDi29DodkUOiaTIQ28OhgxDhAx1q8sYTfZDpF9ug9IiARIDRzs8FAJpgnPuKnZso7YbMEQEFWjBdl6fe10LqQTud4GcvwFsV\\\\\\\/N7ANMT5ET6cI16EzHJu1vNmX7ia1LeuIyGSAJ702YKKUu0ncj6FkM3soB9pw0orDy3IkSPTMpDAZm4jU6jH5h5Iuuj9S9W9leRDccOj5OoyPnNQnMe0opHIJACGTxwnU6GY652l2OYWE3MU59ei4cTv7uEUwmTtNbLpCWh0xi6ijh0+9lWzIq9cAs5yd3P2PGWxPWfyCreLvFhruUsAWJCvtXuufok2QiHhapxOu+AeqqtITxqwqEIzuex\\\\\\\/M0xpntoB2ifviiqzuurL8Ged56seNuXm4YlLiCrf4P6fqcAzz3Dp+oPZfLtI+18tf\\\\\\\/Os8GqOw2xA4RW0Cc2lfTMqIJFZRSO265IhJO42y+plmuccUeg+f4OT1PGG0vuhSCiMX\\\\\\\/naYw\\\\\\\/0vxajuuNLONhbCwcIZWyREJ0l6+UFOOr+B1Oo0AmPhA0peC3Vj5dQoYI+KhcArcIhicamjHoVbFPi08PocQtARbWtAcLWNRJMDXthKAiEjG01PfenEJk4qdzvfCDugmqlIZyRYCsZKr\\\\\\\/cChABrBhwYlmbqAD5eoerHNzhx149gSy5xhbaVqE7gH4SOT8D\\\\\\\/Xg231tNATviUSlKZFKkO\\\\\\\/ov3Sh4z0FEhuoFDyafphn2dEg11RnB3digG9hWO5l20GDcBEtgd82\\\\\\\/rykmw\\\\\\\/+2oHDZ7tA04PIqi2A+c3L\\\\\\\/2R3SXYqz2fN5ATyrEqHF8dX8spfKNPEszHqswJ3m+s+97M\\\\\\\/C+zePPJ3O2oKtlLzfS2mjPBgJWoG\\\\\\\/MfbyJP98Uvzya5kAr2w9eX8w+znUF+cnJ8BLy52E3BtIaAlG6UgVfka+Q57NHra1M5RCwktkh+nqty3gKE8Mw\\\\\\\/I\\\\\\\/544Z0lZV7piSs23hx10nY\\\\\\\/QxVRmZiT2fuIOfF1MNepTKR\\\\\\\/T3HKjYlRaVWRshsOjG6SXFrzFdXJwZ6vsERfVPihrvmvfH6C6DwcGZ2wMoXJpZh6Hb9WHoval1CH4gy1aE5GioUV9+wbOBc9pmUKvZibbaI2PksTYIWEfFeOP8mZqHUV\\\\\\\/r6OWIK5BBRaDB2mjEK7IpAPojVuowjnlLKFMgv7JRRgJNSZWyFtq\\\\\\\/yrMc1uHqSrvN\\\\\\\/Bzu0CUke4brm\\\\\\\/v\\\\\\\/OkbD2yXob46237ZSAY1Lf2Zi3qHMU\\\\\\\/zkJMnmAVj1CkD7VcCrmVbHxgAGUOhFDitk37n6Pbe1tdf0S\\\\\\\/s\\\\\\\/NUlbslgcJGvzfHQUohW45pst9EDGAbxeFmXK5Urs8xJWPseuqkDeUtpfrwzDw63cRv2MCZk9H05jZg9F6G1EHRU5oi\\\\\\\/yDD4OiU0Hhz8+Jb6Hx\\\\\\\/yBv9AeMErdBiPJ8wq6Z4xcvS1lgifmdTPVx1JWKJbJS4HROcw3rhIpZ6Udwe3CCwDmvCRdDxE06hmGsw\\\\\\\/Y3kCPBu4mu8tVpoUxQhvzx+IDQ9tve5kX9z0VTmfy7MMgPpQJfNzuLOaLIr\\\\\\\/tb9+pyLBAgQEERem4NTUHtmGyFXQUiHlx3pggs0RrPQwxpAJDntbZIzBAiADqDd0Bwk3rNKqB5l0G7uv5Eom4uC6x6KskmpNDZB9TFOg4FcIkIWjy+utQkcc\\\\\\\/S9COi2iN88giD9FYAi1VH+HkEFJ5CxanrwyWz5WWjOBBNdmXKRBtCFbL8CpqLhB\\\\\\\/SdAcCtdqsOJZkE6wYJ6\\\\\\\/C4CCoJBakYBwrO5IxiENCETDrSo4QJkOr7U6kwJy+2eotR50OYqFAj+Qy6I2\\\\\\\/vJradbVQwHCjvH6dVhOdxPEMDFRlkW5Dbgy9VqJ+hFuN48OIcsdUNDVZnPZYyQm4siCQ\\\\\\\/IvjDjsufIm5gcyFaK4cWyH4cG8CTHWekCbPfCzP0xn0dUbf4xvIcL5KRBzl\\\\\\\/iRLbsqAAoLXHu776OZetL88oCFTJQhoD7Cg7vCgEWWXmvcH3XGv19\\\\\\\/XgjskizlTp1k0FKX5kTBvaSrnZDgC3D3HPSlHiHq80k57p3XbPFgebBjJ2mhso6\\\\\\\/I+SMa2hbi14eEIN2Z2ZH+B0OMmRIgDR5mtGCjPw\\\\\\\/xCOT3QoXtoXJmK8e5p6FBekGTnSBjMp53m16P+F2sXWIDyyiItpOG9Aj989XoCzwSdx750knf5kSdmOQ9bufuTZelrI2HJbfzlN3b7OSdNtd5kSftyxzrlqmrWVwaHQ\\\\\\\/Ja5xni0AGSV5Y0yOiJhTuS8DaFo3uJXPJ58P21mF+NeVD+\\\\\\\/0p7fMlHTTWomPIpbiDnz54MGUY2jCG6OVyizo9knhoqXWNhnJ9NUgfjTOkpcR8IVR7W0g5o0++LmYpG5GhNsIVznR5KEX6u6k45ONOvHKZ0cQ7fm6K2F04osN9+o7KpnRJX+H0WqHweEDJ02RFc7E+S2jKaz\\\\\\\/5lciLQPKLfQyAeGALGBK+myas2KTYQZ0hBCJydqEqUmDQBckUjvdeegDIiVFO4Yuv3WB\\\\\\\/1nDFsi6CPUlmI+ZtUto0FcX+Al6VYOWRHu+EvGtJQ5ogolJSRGvggl0Yx\\\\\\\/8jhwQl5NIdK7amnp7VyzP0CxtRKx7UvtTKl\\\\\\\/fDp5ij1C6haFfE5RMUVrZVf7nE3j1MZREvS0TJLRHFYuGbuVvhH1XpvpgA+3qoVky1O2gAfFQxJ97NnavycPAb4mZFt\\\\\\\/Fch7UL4OrpOBr\\\\\\\/AFA80QM998U+ZutLNZ9LQOb4jyX4Pn+gHIJiLt2ENU6JCTCfK8jkss\\\\\\\/x4vsWienXQkvWQIwRR3baEokb3pHMGfZ\\\\\\\/uNWmL0cujEa+4jTOxSkVVOOFdQ6oMrIHU2cq9AvIfRsCZnodgmtl2J2HysStJb3VUVnPZy4tj0J1hM0Yf2CbMR3YLTOCFOYD+571CbZy47PtzV8LOyN1Grf6ga1o4dWB+VFLHKlXUtADZStjubk2T8MP55lPFw2Vy5Kr1cwuI5KWIh7hxSiAQ4\\\\\\\/QBFkiaJh2uahKXfYix+vS21XoWJZEbUDjZ+4INyHI9ROmO\\\\\\\/GgVGebQJebpwvcHtIWVmiaNoKYJDCyCMqH05\\\\\\\/dTmX+ZB9cYDqmvf5DXfHXhc0myQtM1KZr2jo\\\\\\\/fe\\\\\\\/2yk2Sa5ss7DmLqJxuz4JTtJ+gfHmSLjuJTpFq8s3MJpWL25Dn+zoR598GZH4s0NzcWdehtRH\\\\\\\/SfRspxrs1xWWuFlvEoEJSX92HxU0wWbAIZwIi4MnwXK5mqKe8\\\\\\\/Z92T5esBXaFMKDlw39+SO0H3QWT5rh0zX10Kabfxc1sO7kMrWqkjefK8Idpuhc5PDiabkEBzUXnEl24rUqxqDx21L5iLOqAEwfsjTgScI0eUGBiis0rd4KqyVXzV1TwLEfa50sn1uMLtKxr6NPO5kC50LP\\\\\\\/MjR9fz2BlCCvBAP4YPDPAdC3Ih0D\\\\\\\/zZKPoS1+w1GQHrrZc5Te7Uxi5Jj6Q3YdP7q1sKgiilvZnOtmAhKIvOyYxlBrR5ijK68uYSXlQeg9yUoCkpI2ib96kuqQvcSfiMXYS6Z0oSGvrzBDtthTPG39Q+AsC5AKuMlqfAzl2GFYz3cd0ln+nuD3X2wVhA60NLUgD2f5sJ67gUOVNKRz3EqXxb+rpXVBwGiL7YxsHEPsD\\\\\\\/sKEIbbMjDNBmCHsJWg4CNKty+j4kmJg264TVa5RwKVR3LZsCvcT51nR\\\\\\\/CN75iRQrmdSwCpu5RK5bgtbnQnFrm+1swqOjd1tsypVKxuU5TXlWo8WzprbxL7o2su2STY6iZh66GoTdUndR9iHmjaJT7+JacgcDce7c9qBpheiPnW5xnfcCTaDuIfRQWd+Yrqlu9guAcTyn491XZOTKdQpb8YMvhI8y37aFsR0JAOuwiBiLgWCSZMl+ptzPqk6HRFrZRSHqnJluNwsKkiI6NJVk02Wq7gBNbeAWFbAVRpuDamnQliQlrQWvSqvl8hMTlvfkZ6MWrtkZURiIE6MPaRRejHydL6Qx8LHUlPmiM2L6BwMZX9OlT\\\\\\\/7lBsrrUcfQlDMZolBmlm8tKUWdoImPi6cDzHvRPVq\\\\\\\/RbWntlHGnqJA3FfUp\\\\\\\/m9456PNoDLK\\\\\\\/hM9vr2QEkYkglHFkE7ay6AwGTSK6ehIkEQnneKgeKUq3vPAKVkdIxc8iPVmgjNy9QFWjoLy0lys9b2RvGUXSoerYAxp8xvXllFGWlBFEh0TmW9GsNmmm+7rFj1Zran65AvpyQvro9\\\\\\\/KrTJryYN36Jx0khhte4AT67gwJljD\\\\\\\/8Ld8cUQngbJqIDc1Gs5xqyAWDGUswCHI1fi7xsq4jY70OAMkK6WmXxifu9EIfL2NpyANGXnbrzn6zxzslFRxZ048ehxmooyHUEd25yylZPBxj7Hy5Xz\\\\\\\/gNd7be9PrwgZzUQVMdXCQDddDogg0YwbfpdSTDThTkLCvw50cm0dU6KWPstkDilioKUotAmasMa78b1DraebBUo0xIkrF7ucdb3X1k6qmhnHOROFV6yJJJhu1Znxk+wp1ZHiMW7U2P4l+g0ClODf0dzK69PhlhQsSoODebC+G0syt1EH2NwD3fiuU0+0\\\\\\\/lG2z3Rj3sKd7OJvhDFMVIfOH2SmENvoJBks29TRgXiKVVgq1HZ\\\\\\\/dZMNuGbYo6HdBySm14\\\\\\\/bi4eFv5hQOIeLegRX1ltovXbnJUZm4W78TlS5WSjHtR8CdORbu5CKEpNCx16yqQK6fO\\\\\\\/AK5sJLscwP\\\\\\\/IxfbUg3c0Nnsb8g9\\\\\\\/c1BdVSdQDEBGUB3DJM+Lnw6HE+KwMlud3+jLMno+mK\\\\\\\/Zbaswb3mpeKxrepS3EYot++UwCjoJNs2jXy3WxOVpDaSK+rHo62trAKv4hJRuH12hHJbxCLSpli8gZSVHlOR7\\\\\\\/75NzL88Zzt3rPUfsk+B5A1SU+7qJk\\\\\\\/+4eJaPx3kgPe+vkNWBXZMlE+klIolOawc84Q4gVv\\\\\\\/Qi1RXPIIgjDGzrlqlCLH41X5EmDMN07\\\\\\\/2koYS+9is\\\\\\\/udE2hEt2dhrCu5KL\\\\\\\/X8CWts+SIp4q1x5JUAliWAwnEn5gVkDQj16bZHeKFDNrMbvp5dqiO0ujiBr1RX3hvqetnNzuHHc6CqGro\\\\\\\/IC1Ep4oCL0OicqRSe5NuaCau9NGnNAxuSRWGY7AKn4pfcxj7IS1tz+Xu8ilnbs4ngds+tZXZSyMRjYDQ+XOC5jNswvyqRRpFSKIsV2Jz232MtuwmXnl6tX7LBrEBU+dlTUk2c0hlWV75Mx\\\\\\\/82yqiI3i8F+uZ0lrmiTsl2BgQiWXhxpVZ17hb+uFUwuEVlUN7Eb9em4WaPhyo88QcSdrMigz8XselT88SAESromBgFRW0vMBdWrP1Dkje4z6iNdL3Yqd+i\\\\\\\/hXZbablvmq22bTV+cmEGPRiT1M\\\\\\\/93YGoEVKypawM6lT9gcxxAQVCppEl4J2K6ExXG7R\\\\\\\/q06OVnmD7ID6tJnUk0u5DRGLOjsVQAz9Ek8Zz1LvcQ3zyo1XNqsC8x7xWnFAPsQvFAXCZiBHOuowENiLxpag+LyzEATWvcNJk4Ik+Ks9EhVE4PImeYZq5PuOUyw5EQPSRCdC4zF1IhDnjD65HIGBFyTSLTsmkwZlLDQ09IYSOqnD4UPCtpxMdzIU0kkhKuab2s4P0N+Gy8jJaoh7nChDZq3AfQekv0p0l1gU\\\\\\\/xtsW8qHgCKb6LsjZJB\\\\\\\/JLEQ3pjD0YQsTcUbsu8tNF\\\\\\\/yq1vE7E\\\\\\\/9+Dk1Es5o+Hlx67HYb1+gU2qclQmNlm3FLYMN26sF6s\\\\\\\/CNaOFlmV99QitceDY9gF+7PHab74C5BH5x57aF4sbnpJFgZGVSXFM\\\\\\\/4Y07s12OIb5EWd6lOMmFSWLzy\\\\\\\/8HHRAKKzsSPB+xZsmZn8\\\\\\\/M0\\\\\\\/USIUnfn7v6HlW5TAi09K\\\\\\\/fRalLJiaUTpaJ2vGtpyTDIo4Uygt0J9O78eRp2zK8Lsa4r\\\\\\\/5KmZJyuLgs5aWW3AwG3u4VXHMzUihDpbGyxSSBwBanEEMfWvgFcSyq3HUG5FbnC4TdOs6KhWNhmO9SVnMDPN9azHhCerAwoXmA5w0IM13j4yT02WIP35c2uCEpa\\\\\\\/+vVwp88aUEDeXV+sAaVmXrmTZPOWY3cflLkpaCLoV5RmLkv6Ns4n0ffjVl2xKpuBI7NDzognr\\\\\\\/dQOY96vTu8dFINLYrLQ1nAH+ZTudFCnGpUst0mGdcBwfJGeapNqpt+rnYnddiDMAzH9TsqJIPOuGEv9hDr5lIVBx32FFdfIkmSRlK0PVraju7mMpea4oE3jo87yZuBaSG6MFIRGjtcP2V+UgvK32OHP1673Wl6Wx8mkc80vxy1iuh34mswNjTwfuft6hWXSQC+2KQWmMrqogpRu0eezR4WJoe7DgwngBNf4dpncsUMpw2ySwPWpKQI\\\\\\\/EVKoQK\\\\\\\/facbXxa9CXdFppoHbLhQQc+G+XETjMJPglWFrowI5tc0DQRRuqiqpo4rE1op4OQXswkCSTj4HqPp9k3F9iUgqVxMUaYZLI\\\\\\\/f2TyXq2NtbyMDMiW6jr5SMVc6tWmh6IqTE+i7lh1mULe1cKCEIWln9aK1t7Jq1JIWFrxcoVYjfCZggULn87pMoyfJKN3CbyHeZmMSfz5XHUvJmxqgxjzVZynXr+ehbD88Oo26QnfFDWdtF2vrZW4BA8nbRlUzTBw406FeLNnkw5iR11ZaGxWMfRr+EJWtQO3nLuXJQvT5g1CNY\\\\\\\/CkeAlnuz6jWOiYMv36H\\\\\\\/Xd7YWxVrrog\\\\\\\/v6+g9045y+SBYBH8gXvOgjlP\\\\\\\/R0gRO597G+3WmieUdZGHdmVXFrakHsUSAuuAdLwuFDmUNfMcS0MekkzvK7whCQCTWe6r3QhG7GDzD5RPROA358q+CYH2M69aikwNA1pQW\\\\\\\/V048G\\\\\\\/6bw0\\\\\\\/phJI3gRCdz68HpvUNMxVCdkAGIqodDXFznuqDZZTjWPQKAVRSunP4oDIqzo\\\\\\\/Vr6F1uJyJQT4jFJ3mySijxOMNqUsuREr58dad6g95QutWY3\\\\\\\/lF04VnZwcXQgchsF+WLCvgPnrc7nnHie6cE8Fvv1QCOhUq01cxzGAsdoMv\\\\\\\/jdnfApIjqA39sZ3gbu\\\\\\\/E9YPqHzniCl\\\\\\\/gKoXXvYx9SusZ4VKu2eDGQcAqpQ\\\\\\\/JpOOGKrCeDscxt3D9qvicXOW\\\\\\\/WN2HTfn1DEqGYijk\\\\\\\/0g6PnvtgBUox0djx5peuFf6MB6PwcBWehUj4bLM\\\\\\\/kcnYCjUw4VO4wKXschAwkB6a2ofIAX3XvAwSQg8VtrsMOw9z\\\\\\\/sLFrx\\\\\\\/KuDkkTjNCIZPdM0eQhddwMm7YKM4ggI3nooZflrc1AzNNSbsBWDz8H3lWx\\\\\\\/2lAYJAfyn9bPT1tDGlxyBBDFD\\\\\\\/oQGzTU2\\\\\\\/ZYZ1\\\\\\\/665ZZkRQNsG9V0+qzPIlEupAQ0xL\\\\\\\/VYsuavHBat69teHoF8NKw9rTKjFYLZsJOFM0Q5SagEA33hNAnr9BdFyyJXbjJDGPyST5NpkbFjsW\\\\\\\/TZ7UlODf8FRuMV9C6niqHkdY+QOcs6MV+TcjFSF5IWAiRThITYk+d+vyNCuYL3uPCa1Qi9O1\\\\\\\/hc\\\\\\\/EpwoXwoiXPXeDhLx9bJ63rWfwh+2cxLrGbDcSObh5n26qoZ5Vcd5zvbUqyGEBhXtRdLsl6Y2tLbNHWl2oj+SHtKVk5IBunvAxgWKJX2c7eaaDeX5L92MTBC3dWTq3\\\\\\\/BcmszUYlhSLUwwW1\\\\\\\/7e7wqpy2sSPExw9ZcOm7sJ4s2zQ8wRG+kbLDde1YONUl\\\\\\\/8YFljJSFhvbXfPNXUSf20zh8eipl8RR1AE6zX5VDzWOCWc4Shs1jWeEOZgenwKTak4FFFb8KAB9UMOthcirW\\\\\\\/LUSJagO4b5KHTaCKl4dW6keymVXlxg3z8mdIlREd\\\\\\\/o5oQOKv9SN\\\\\\\/SOz8HQGp1fDO7BQJJtnqsgal9sTvrBASPPWknvgfG72nHF83zJsHJqO\\\\\\\/eJ3ZZ7csMbZrpDVBuzPUXenTCKpeK5rUxgA9\\\\\\\/rnfTKjeTnSYosLpD\\\\\\\/oerkJ4BlbO4cOADMYd7DCX3kHn8CFq7kS8khm+c4NS1biaCRYtPqzklrvIQICPzgAZjDkF131Wr1VYlIzKMQidpuqX6k6Pq4WosZXHKqk\\\\\\\/SDdWFB2YxE9vJQcJ5qOA3kmiDsHRuUwECrraSfjxxqrCUHokpA+9qIeZECsLvHS2JFtDfyu2Kx2QSMy6tjNsAdgr3VOFhxTiYUrAYpLU247cz3TRQxeFb1OPjt12rpj+WgxZL7BWz9uuBN5S7QQAjRVuihAPWVyOQUcaZcjhv\\\\\\\/ew80GrLcrWhNV07IOIrnC7dOW2enQoT9AICx+DAzXyQr1j3IJOzMxX6vzJss4B6bh2za5LnZbnGXKt5Wdt8ibDDV8WU3yvYLqOL663yK8ivXD8+tCwVbfIIikUt3ioYEEn7ZSBF+JvSKxGJTHs+qKIudKnXoCCu\\\\\\\/EHjgavskAHdOU1DfZmWG6FhhZQukJm23KMkz23AXnkQIWqtDC9yWLiVT+vkCs4PoOxXfYy23dJi6YBxlqu4GtfDuKbcMNrrM6qU1VG5LbDEYutgXWITia3LN2cuCerOY04MM5mc7TnVrjhMBSO69mLBHvyKHezQJ0vDVIODei3T0d0XZg2k+hAKDTJEC231td47YlrYVGpwNVfIdbOsfXO4UnDTrslliHbKlLpFM20DwrGzlevUwSsAHQVLky7bAoLM5LIQKQGzq48jzqPfKauGq9YBEM5g4vb06y34G0XChzJPIilgOWUavUSE7y\\\\\\\/goZHGrTA2N8WwmAiopzvCf217THpVrFDVUKVPUZnd49AdvSfEzNjRsZpkelzZc0eKXfKbKfyJPzEi36uSL9hWiWtfSqaXU3mKSXlH7fC0Jb73HlzS6WHT2Ovlyd9kqxG1CK2NykGhpB\\\\\\\/5X9rwhqBx9WEmkww3hJZ2UterIoS7U6k9H2oqmcWgdKOOtQvjShZZWyN1iGzV3KwFHDlodXycZYP6f9yRVZ8O650bGBHGybXa8DLaCyGJAvJUORmLXfyF6dhVQloLANhGTrw0zy\\\\\\\/cbWx7in6CcpR7M1PbxGl\\\\\\\/TLOAoydfrw9W5jHf+rqmDddghMSmt9\\\\\\\/47JPCvanInX4J1npkqvhoNbn9y8hT9F3zcVM\\\\\\\/ZUzHD7tO7h\\\\\\\/ujlRies53qZEFiyiPCQOYB5raIzOqeFR\\\\\\\/UhFkSYSggG0KaZDFaMt4QRX+Z307JTH\\\\\\\/HsLoQbVA7y3s3On4imdkLynv1xl7bM68RBv7bs7bDz+v\\\\\\\/H\\\\\\\/Zn50HMn+fJY\\\\\\\/raUT111X1MFN\\\\\\\/To7\\\\\\\/ogFkgaUJE\\\\\\\/iigeYYMh4h+BqZQAKH\\\\\\\/0octR29coOj\\\\\\\/i9PfV053hXalmUrweveNLLQDtmigGgXx3+aG+Ro5VJOaXQYrE8+xowuv1+Ps0qRp4b\\\\\\\/bziBbRKFi2e9Wdff39R5pCy1xP8wTPLQA2tvZbV6kZibIF6J\\\\\\\/qfDSCgp6spkyzcwRM+dKM7pLfQcJQrybaJ0eKODo4rfVX5d63rrhFByW+3BdbXbRy8GiwfnM00kUjSk2Vb7STm4XYq3wmW9Oa6h8UJj\\\\\\\/1V\\\\\\\/dAWe7R+MEO6uG0lrX35T8GhZBqqTaq1VzSY4s+8TQ31XFAlxbsnIBrZwpddkuJ60HDRyxEcV8gMKitub7MSTjF1gYXKpZ8JQ4FnHCJMJWU7+6oq+CHpgsfvnhTIxwHNFLVN\\\\\\\/qnT+vamGUGWmAseNEgsgBfOK+SQ7Te7TmGIFT9dhClt0jNHZKJe6VZ8f7NhvCRa45y\\\\\\\/pld\\\\\\\/CLiNUWrckDugwnFaG0uYV2PkOAnmMOwrUyFBDdx5rFNXszdBsiA1oP5IIA49Or2t9FTsdSdzKrLjVq94S2OeXijPL9CcKXnLd55V5CoHtFcORmgA9+wPyNg1ROdHAW00SCpD7XyhO5LRHCjp2HrdUm+1qlull2eGp18lJNlYJB9qvfMXr6\\\\\\\/dcvnIQrGhRSfuFJHYF86rVFF\\\\\\\/Pwqjk9bboVnQ8MnbKT3KCTM7ijez56uFO4UIKmquZjntEdaW3iFBi0XGaxOnYv9ojHQULBKM76Y4PPQNuO6TcZVWA3w9M+vGNhU4Ll9TwAgs55Hk2lmAenQTebCfAGwBGqSM+wocFbM2fOiRBgByVwRVv1TZoZ5B48lnjEPtxw70rKw61sRe\\\\\\\/YrQ8PyKbgxkGYLjjnqF9j+i556s5BjfNVR1AFNQA7aKeC5BcuIgd4RrwTm0NOOEC8oeF4Q8GotJ9lvIG5MU\\\\\\\/L0NnhbH0G\\\\\\\/AOQxFYCQ37frnKuHwBubspZ2DFvOqTbOGu3qG6+iWE1tsunl1w9TZ5RQNEZydtwo7A5dxSUdLclPUhN7sdMB61eJkjK3KvBcJ1BETh72y439YYPc5ol5pXiXop6t0MwcSCkeKzjxrfEWHk3ZzsLbtvs4UMWAo3jvrtOw9EGyxQihY2wCloy8ZmPA9F2V2XPrH6cOeny\\\\\\\/p2IwAnNWUey8rJjfTOMHFcM+bpodPIHPiz4m4MJOrBC4IZCQHvBKnlxw7EARyi3U27lCfGVIytYmpcTgTpRd3w2BN\\\\\\\/GKrasKykABtUF\\\\\\\/lT6mksOuWvqSJhn5nOIY2R7K7UWlqC9xme8pM\\\\\\\/pcEAvDqNAxqZks2PQSxIpkcRmT3gUbLOwWAt+NeEV92yqBZZkWetxosaDHioK9d4VL+Y0LeRZWpQpul+4HW3lajYLfKmqP+V4rt1xVqgRn5RTexGbEFEEg+tlqyDk819uyaDplhat6HzngvcPn5guGNvwzmSkiZV9lTRdXMBwlcUxv1Vv79p45EYpxh54bTpwSE3W3dmLGUcHXLlwzqatsIYKHRbgr2cSWnuNndYIJQcv\\\\\\\/VZD3kvH4ZKH+WKHd6RYN2F+n827l2AC9E05RooXB8wqcNHYS8B823ZAVOJkY3z8w9\\\\\\\/WUBSoadColgdtF1v8YHX3qkYNxj0SGYqsdYbTSXFkT8ItgqoxDmHVZkgVsFPpfNf1tsggNexMrc\\\\\\\/R2n\\\\\\\/3p+lvjcsBiZYSFSgXNSzQ5ZW55SGr5ZFzmYLXhmhEFMsk7kkgb9dbQajJignvI9\\\\\\\/hIq2Lq7y1MffbzcsaFodW9hzk3lwRGmhrWrNfGMnyWj2eNEYTXibFVymgWqD3S889a4Xadz99HmcLpQDTDYKCOuMP+4K4U8VCj9KYtOfzCXcv4f\\\\\\\/++uctwLpIc1jE8\\\\\\\/4MQJIU4hxxtpdbVVyXzT46CnGtql\\\\\\\/+bc3R42YxBltKeiFVG\\\\\\\/bE3lnQ75bsBmMb52RbJ0shq79sB1bKjI8o54noNHrbS7js4UgTiXd\\\\\\\/1SWi+2rhV4dFHviww+RaN0RjmnYzXSE77UpxRslnaE0r\\\\\\\/XZ29q0MWRRs3c8JBxendGP5ppPskBGDlawXHZos58SjsgioVo\\\\\\\/+5uhoc+r33f+MFGP33wXb8WV36IjPsIjaGI+GBhG+S0KQszpI8SMKbHpfN1mGHnD98Gg4+yofWJ2pYeoZEE69VzoRpW7rhtozoysinj0hq6FhOXAC8AXoL9YRVN5AW6LNkyMKrHaiUlAeNrcHyq6W5FNVi9+2vz6Qw8HTxIs1wVmXwx0c9Rhy9MadC4iANHujfMli1F3E4a3iVZkmyEEuyqK02xTdJMPe5k31eO7fTwvTL69aaTl1THhoXVOyUTMNr7HTbYkU3NAg4EyjrYrigkI8mlFjanF1jnOvBcs8OL17PbSVDrdj6repqLN7ow3OeS+uR4E4eS9trBUExvk0RXx15hMXBJQXWqe18AYqQwn8ruNAHHwSyGdyS+lx5M7NJn54FNS22YsOWdX4IRQ1io9ekVn1gm71TF5SmyhWxm9WhCLbkCQhJk1MZJC052HqfGyHr2oBIuwdAPpq7K1bLL2mnNB3ZnjkjRFM5RfRjMLnKBW61PK5nEL3yIWnGvRYTConvonkyj81WfsAMTs\\\\\\\/CEn5wD\\\\\\\/sPx97S8JFzmQcwRKd10aFvyx6rlGCya7azQTrFzssIQQY8XWfYnyd41UsFh3utfC1Tqfw2DuVkdFem0FhDTgkOYsoGSJjQagcUBsLlprnIaEYVPOP8n9\\\\\\\/eEbOdFRPkAArAwIw64Qfjq6cdlOxOAi1suID3QhrjywTbIjLX55q2uLFkwqd5NSKpnUUHNn2G6\\\\\\\/xMa8V5jNCasUVRVEjoxfJ12tUOrlVtHxKY22gI5aiLdbY7rgEY7zu7B2x2Uqger11FI9cDUYmCsdie6xlDT5XTNdOB5AANxjw3s7FQsd1fsdPjSnBX7fDyv9Hg6ZM3WgOOHZzaLB2QjXkIzuuzWL54eN0Khqurns9V+rGA+vwT0ts0Llp1K6ZbmaiyqUSAG0FVJhouMUwVjpiECF3MZ4HTP+WAvHuY9vENR611v05hKIdqgSQHTY2AXD+rSpigCmXTimao81bltE0XwrrHzkpuviIzlufup5ugqC6BR9eGae14dYPZGqdbL3ukNYI9C+pb4jAc4OlRCcy+3IEsswTrtNoIQp+yUdj3HXgJ0U9KetGQn59n4tmfLxSD0EkonHEAN6oRNC+6hp21zVgNoi43JkjtyINHJHIdkFqvziWAmzzfizsgFQwYtFjuqnnberQiecrnfv20h\\\\\\\/TcQSwslL05E4CMZo8Jzwi5Rc4JPiJtXBC7T\\\\\\\/MsTrgi4QaZBC\\\\\\\/o50QKhWdyVxZIRWYreNrsIf3TUMty94m5+c+J4SiKnXJ6Ibe5T+PNuEjveAlRfEB6f35puiXj8XZbkvYmQH4mJNgzZkdr1PkGZshfFYTipwD2fV1vyiiOEyQgzUbKuYNUYmL+qeJ0905d+P8\\\\\\\/RkRCkquvkivlDEOHf\\\\\\\/ZiT+L+TY6Jx5VPEmBPXH0ET1kDA26Epu56CiO5Y12lE2PGWLfurB\\\\\\\/HHQ7zBK4yjAGwL1B5wMp85KHZQPdmcyGJw\\\\\\\/CjgFP760jqnxgD8YBhByrXuIacSO1yGF+zKpEZSVoBjHgizCPN6xHmJoHFONO5EVV3ND+mjuyN2Hpww\\\\\\\/gAZpTEhWU8bCDmGa58BmWlTAbJ6UsDF3kmZbrrAQOzSkz+ncbSrnJlj8n3XuqpwZ6jM\\\\\\\/9bz\\\\\\\/ehpz8S4P\\\\\\\/W\\\\\\\/\\\\\\\/XYpXesA3kR1yNfv2RP4b76zpdlnFnLhhAoEE7NbNLz\\\\\\\/h2uZJcJynvlB5vi4CgiKsikpUIrMG4dMqqzXYB\\\\\\\/VZZI6Zbq5DDC3jESp9HrOqSrhJwi9Tai1uqyXsUstBwLjj3vDAB\\\\\\\/l3XNFhvPArpObA3Mzx4beVqZGsDcARR+i01JXwk4GECJgclaPvSpYPqkOU1UDL5bYU1\\\\\\\/s73SV7Ba1mk4v23VFj2FM0aDXBKtwpkdhH6FJ5Wo8JhmltZ7r8cQbMaq1WTpYFgruBXP04z+eE2TrPWxnAIMeAPusHGqE5zXobMEgeqXcPLZcx5aj2+oooGwdGh3iOl4udk43ucBnQp2i3Vg72UxmDmGYgU4q+cQ2JNHReX5IqaB90\\\\\\\/zI51BxIrNp5F\\\\\\\/5OK79bfpljH8T6qG1+y8X90Z9rtvQdGFrlji4PcudcUfUOoYufFEZstXtpgPRSdsiKHlfm2pe70t24V0\\\\\\\/NLSks5JH1vxbo2\\\\\\\/bQMJPijUQp0OR9ngheE5E\\\\\\\/EPs2uDtUh\\\\\\\/sBFidgBMFzk+L7M\\\\\\\/y61lekUqlwAXzuVcE82GGN0X+ecfxRn2vQ332NhSEsdlNtXFTGJmA1jhhfl8X9mFV7BqnnEqsm6O9bAEDVAhDSh7LGxEqdPRWxZiGMgjhi+vKNLYSo8rOu9ojoulmbb28aWQrw7AVwf3zruQtXWD2MnbnmG2Z9YDYi\\\\\\\/VysFMP5azy2YvabM4L28N2FARgjDpML+VdOabY7ov6If2AH0LfOTRVGtc72F2\\\\\\\/KEZ7cVhpQ\\\\\\\/EO7XZOJXdDIo9ccF+2Q2m0URVlrl23rUNhY4cTM4MY5bh4yjQAMVJu1YTbR3ad\\\\\\\/3e19dtIZb587x20ty7AiVotj9JTjAG1B3yO3+MYOfZewXyNH1SwhfRu11oNpoTXU\\\\\\\/QIMR8JWqq47JMohMW0bcg58o83NXZW2hDu1fbTwKCJqaSqnyEy9mfz6E2SbpADfeyNb8gVH7+3fN27SmUfB0vlNSPicBy\\\\\\\/Au6CYaiWbPJa4rXQN5b3d10P4vm1Fgaex0bY7TwChGwYS\\\\\\\/fGHzqwzcgQWKzIYW6G\\\\\\\/cT62tCIPXFmwQ+ZJQIoEiJ9zPRqmb73lsJ3HwY99kbRLOLcdk21Xu48PS70+lytz4UQ0nHTQoJishXlBXTEPjGTH2bQu3b64NWJvIgb\\\\\\\/cyVFkV4ZUVwbFJE2uJzaU26eXK\\\\\\\/IBem9YChIKxMzfmFKThYeGDpXiZ3tfNI7ifqH9G5PV79vXgPTTQkEHNlpEADNDraJHJ1qxWDq2CH+QALJW2Zw5UZFGqTwMhD\\\\\\\/M3GDHfrYGvcS6tdX7YY6pemyQLueMMT\\\\\\\/oB\\\\\\\/txbU6ev8O3v\\\\\\\/rtxLzuSgUsVZlP419Leg3+NE1jCpuXBLInWYENDScf6TNsUXEMGMzNmYiwlHQzztFzGMyjO+WrkUrBRWo9BpUnP6C\\\\\\\/vQl9bdsxTG8mWhG9CwLVyyPfzLUMp90b1S3AeSXtiDkigMNvOH8ZNAlFVVyX\\\\\\\/N9qcItIa8xDoYHhZli69jIJsBp6Rz37m6cTrilX7neZirap3vFoNseMQWQ+c6p3AmiVp1MmBCbeq47D7lLNqO3ixUeGEqQjkllVZn6ACRmvYjnVn9fANBytetMvMu+xW1AvEHZRs+Ul2zEzY1jWiLCc+oxuaoDw\\\\\\\/IHEdFpV5TPBuI6Jp0NaCOdfS0rpGMU87ilZ\\\\\\\/ZCWaFrk\\\\\\\/9UBZa0o2O9QRkhcUOmXivFb7puWsoU2aBOFbtsoPLV2IO73dbgjH+ywNiHZ1KYnk4H+a6ZuAGL\\\\\\\/XCcjUwMY410XVmohMSYHlcov4Q\\\\\\\/xogEHVpHA69CNI6ZAypw1XIaSSw84Ka0IYPyLZ1sReJn3U7pZvOrIHjCm3zA8GJBm0zTc215U61ySYaU6nMLZTD0UwMYLSOGuqKfO+eIn6YmVmywJ6nIiRGd\\\\\\\/fFjc\\\\\\\/IONyMrPXWgu4xWmu5R\\\\\\\/bYVSVNUlEeENwl27QpQjlpiXaHExca3dRESz9VNzX25frNJgtYz56SumK8uDu7S+ZwQmVuu2aaCW6vyc+WisGfk8q0uomiOEQXvHJkDEyDZaimQc67KdRs9A2sBs0tpttWCzig1IeO1WfiacVhAK9rusPs4Pav6xsozY+Fojl0zUUHlsv2Kd0U9KD1\\\\\\\/U3mljLXX8xJ67Q89UpGVSdRdJc9yBAUq6Fanr+Gn27lAUTnct1Cj8OG09uv2SWn4VJ2HWVMoljBj8J9VDZKlzyN7GMXziB7Mh2Oqd+DxmerUG0hsKmWX2bU+4ZNUQ1aRV80+I1CDxo5VD6YRHAn\\\\\\\/JSg1\\\\\\\/Jd8s6OQJhK1r9pLURKBZAsBkVfXtu5uKa0P++yfc2CKXKfP8KCHieidPRsATmPBgwC2EYiOSBbtHCf7uzz95TJLCB5xDExLe0+Mcnup+A1Hy\\\\\\\/qQtTL2h+qBEcl22i0OHEGvllWVO67k+r8\\\\\\\/T7BJ7PlElCPa1uQZXV0E5YdT4GLJUPLW36Rk8da9wbE5bM4Qe4Wnot0hSE+fRx2Hw2lcjZYItHw+OOHqU3P4Ho\\\\\\\/zcjgDRkgcTLdcF054nQ7z6xcBgngWnBBBZBRa5er85TZ9GdV9GHffmMXILjnlDjcQM0yfedb+VpcZtHJ46o8lWZuOq598XbZjldwg2O9BHTFKGL1KTgWg7TwZvew1oMZ3PKOxJYqRnjUY8ZiEBasdZW6cjrBhj1BQNwrm721+5mdHcq+6dacoMf25lJuFduKb+SdJevsu0xjQV+PiTC5quAVEeh2IGBnoZH3SYmr5Si0q3+KMLRp2v6pLvrgZG5\\\\\\\/nr4zrkdATBo7jR4ebvvljVNP\\\\\\\/g1QdZcjSK+Bca6NcemDke8jyPdeWxSeQNxr0km41Om+cxxYCIWi1yRCEs9qm\\\\\\\/1yMqtf714lUpRZJVdEl6v9y\\\\\\\/b\\\\\\\/1jNmlHqBC7mfRe7kaZG17jF3hK8g+SZQPNlmMlvvRr9TnOeUnuCfxsVFEn1sJTa5ZlyJn+OD0BvKPcCjcbPXYwVoB8TTrgRbf8TFw3TiiXMLvyh+FORBQthcczV+Lkz+wNG42\\\\\\\/iofFWnFzJBR4Y6U5co6VsA2TEU+mKY+mSF8chM\\\\\\\/zVJmtb9Efh5lqHa6IwIxOaEM8reLp0ZYga6oOLN9sx26j6iPGWFpardTIVBDWa5Qa0vEk0x2zZvk7wMH+xR\\\\\\\/64ArNXKhR15+m1wj2Df2Ozyork7KY7mnHzVaRYt12vFPhBdJC\\\\\\\/mJnO2aEJNPzHsu81Z8eij5tOghzvxCB5Ae7riKm36SFQFcseMmU1tff3nWsPSs9Uw+eWSPjzlg7GVxLsvKoU3c=\\\u0022,\\\u0022iv\\\u0022:\\\u0022aUcCcANjVvPU+msJCAr71Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225b33ef3de6bda3f79e820ef65f379090463e0716e5a29bd91fda758fb781bc92\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NpTipZBnxaP9ZFtwOw4cgYieIae718Cl+PB00jgPXDlqmYTXand9sgVa1JoZ9uyhhHAD9QOTwSIJ\\\\\\\/Jzo9EBHl4Rp3xGSN2W7+oog68dz3J6vk6wM2uBfT\\\\\\\/k1Z41UwiGkmuNifMypladywViquqCBERPtnxZAZpCRaO+LwEczhl09CdzJcwuIbxpOQFemjO64jNFwY8rQz+Va4+viUVmAEA6utFwVh+wSGKEmuA5JOonyrmocbd3PE2pEKTkw9pt6NQeYIdSSy4ro0KNqAKOTpSZsYSyEEQlCNhu5f3eYqWE7dPgF\\\\\\\/ymkCsbtGGNgiNyNQzzHbAsh9y926FaeVKdCaI0HnOrA7\\\\\\\/hk6HWG3kmwFfNqjjnHkz6amjkCxfCPjm3v6xeN2yyUwHmVa+3QY\\\\\\\/I86Bpcord0iZIWLzlYVTFlayy+RxSweXRGAouSxTmT5z5g59jL5JyVGgH6DpK2g+JDQn6m6e\\\\\\\/FLc6wf20gcPdS58CsqylRqgcPa8KatPlNNrwFDk5eHqfnZLskJPLmZPPfwF\\\\\\\/Zp7tImgIV7kG+3kxfBRtjmUEp2GBy\\\\\\\/hgETMTjQh4+apMDJWL5fmBA8x\\\\\\\/u9E2Xs2EmgoHdUqK6rPGDOCBhbgdjsrfaG5FLCUWDlBcsv2hqqAFXfRgU0HClhdTUVTD2\\\\\\\/BqIo357wvlmvfX+vPWAoEdBk5Wfe5HUwkugM\\\\\\\/ECgYmbbk7L\\\\\\\/sNdDLgwRvcdpPFJLEBY0sPxc3eH4CoXt+oUK1YWaZ3xZPxzKva1DTMJY8JDjQkD4Ov198W13rXppXaQlEMOK0fsT8bE+ohM74gpH0MrvaUFGUchQ023J2RH9earIzT5DsrAvXBIDqQhYcNEBCEYpV6GqjsOE3KEdjM4vb3BArcplgVdpkqjEeDGLs5O1Orq73\\\\\\\/vgEMHs2tKmKjyuJku4\\\\\\\/llqRr+40ka84LwE\\\\\\\/bOAKrWXnRNMflY4yaLrBzrWIJlMZNeTbozY+3cRIKJ5oJ1e7wwXooMDgx7N9HhdJpyuKHPfo8\\\\\\\/lalyGCm8bR\\\\\\\/1k5Ywh4b9qiMecjJT\\\\\\\/KgNwCoBeelfrF\\\\\\\/PCN4THJObMyqfBNtxZ\\\\\\\/WcA33IFfPKoNpgkdy5mmirFFvyT2+dpimtUwjJbLH8fzumogDgVHdG8ZVClrxk8ccm9dHX1050Bw2nj+mOtLcF0yiSEnmlq8tfiwaNmEYlxdC4PP8yPX9rjyrjI\\\\\\\/6McCXo\\\\\\\/uAvHK9P4bJ7phBvxbRvvErJmaySdj\\\\\\\/TntMVLup\\\\\\\/\\\\\\\/QcUdn9nucyO9p8PBQMNKGW0n\\\\\\\/H\\\\\\\/OBtm\\\\\\\/IH\\\\\\\/EnbNPqrCAnSVfelSwLrZGbX9qwZtznJmkrQaEchkQ5yXItl0HJHpeHgpEWMFpcdfQwoffG\\\\\\\/SM9zu3+f9htoJSbv7rJcDDUh3oIPaJcedQUDrvm2uuLoCmUtyYtzYlTiG3TnBvLrlRPmhsAMrp7nKA44uadeDhySjhS3FS7HOyDOCMxcVE\\\\\\\/q5lQLxYdNr4zZZZ6\\\\\\\/RHPxOwhHDsc9JkhHvkulbInm63AiqrB4lQ3MB1dDhKlqIdQlJ8kR8kPaIrXop07NFEuery7qrjG9iPxxNJf4OhcS4ep8Ocv3dfaRoh3mDX56+fcSgv9Cff7DbkNyWDer7kvrLwiU3zbh4IPO+DoTOfLHNwvpBKrQVu5\\\\\\\/bivg4K\\\\\\\/Ko2qYJziYFDSs142rgRiDAQsSgMxoNDgYgbMetWz5M01cvxPu94AZ4Yzkkd0x3iY4rTbOy+TrNJPMG7Lm3GT1YK9sMlrQ+58ViUdL9PLVeEo0TgDwdrlGTnCvcQIp\\\\\\\/YIVFF4rrjb+IRoE4lVpisW1YERgYRBk+SnOWFfeHKbJqDbmva6+tDlflC6O0qR92BiyGoek9FwJuz+Lgdu8qciqBe4c8lTOg8T4GfDbJAOyMbqVto0g5DfiXIWY1JhKnsSEqPgQZq2PuPsxKj0ptW4SsT9eQHaBl\\\\\\\/wBoWLEYhvvMKX9y5rpxCFsVAOILJU1NsKueAtTape6yeLKsBTKAbVklL69SLFfkKnGMZ5TGoG1fpVurkskNIH3xUa2A0YCtWhLfgoGLvjHAalrpPbw\\\\\\\/W0PrFVDEph\\\\\\\/dZ0oT8Y5O9TBVPEYppfxfPWfiej2PF2inkiTNCkP90F3iLJ9d5QXYdGRDt5nENUPGuRiStfmUcQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Mq\\\\\\\/wI2Zd+a+kGcv27vRRUg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022eba9d4c5d59c6daf8bcdeb0784076fbf572789c698c312923383d13c549722a6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460259 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-mutant-screen-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2655" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FAMQ166-CKgCHaXIZ_jnLD_TQsDK51dRAhuMsSU1v44" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2655" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UA6Btvtt4UFI0KYqC0EmknWcomHMRK-10w6EVqXQYR0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2643" title="RNAi Screen Identifies Novel Regulators of RNP Granules in the Caenorhabditis elegans Germ Line" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2665" title="Phenotypes Associated with Second Chromosome P Element Insertions in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460259" id="node460259" data-pisa="ggg;6/8/2655" data-pisa-master="ggg;g3.116.031674" data-apath="/ggg/6/8/2655.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Functional Domains in the Cohesin Loader Subunit Scc4 by a Random Insertion/Dominant Negative Screen</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michal Shwartz</span>, <span class="highwire-citation-author" data-delta="1">Avi Matityahu</span> and <span class="highwire-citation-author" data-delta="2">Itay Onn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2655-2663;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031674 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460259" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michal Shwartz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Medicine in the Galilee, Bar-Ilan University, Safed, 1311502, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michal%2BShwartz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shwartz%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichal%2BShwartz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Avi Matityahu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Medicine in the Galilee, Bar-Ilan University, Safed, 1311502, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Avi%2BMatityahu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matityahu%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAvi%2BMatityahu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Itay Onn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Medicine in the Galilee, Bar-Ilan University, Safed, 1311502, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Itay%2BOnn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Onn%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AItay%2BOnn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Itay.Onn@biu.ac.il" class="" data-icon-position="" data-hide-link-title="0">Itay.Onn@biu.ac.il</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/8/2655" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460259" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2655.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460259" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2655.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460259" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2655.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460259" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/8/2655.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460259" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460259/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Cohesin is a multi-subunit complex that plays an essential role in genome stability. Initial association of cohesin with chromosomes requires the loader—a heterodimer composed of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> and <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>. However, very little is known about the loader’s mechanism of action. In this study, we performed a genetic screen to identify functional domains in the <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> subunit of the loader. We isolated <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> mutant alleles that, when overexpressed, have a dominant negative effect on cell viability. We defined a small region in the N terminus of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> that is dominant negative when overexpressed, and on which <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>/<a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> activity depends. When the mutant alleles are expressed as a single copy, they are recessive and do not support cell viability, cohesion, cohesin loading or <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> chromatin binding. In addition, we show that the mutants investigated reduce, but do not eliminate, the interaction of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> with either <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> or cohesin. However, we show that <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> cannot bind cohesin in the absence of <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>. Our results provide new insight into the roles of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> in cohesin loading, and contribute to deciphering the loading mechanism.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/scc4" class="hw-term hw-article-keyword hw-article-keyword-scc4" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">SCC4</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/scc2" class="hw-term hw-article-keyword hw-article-keyword-scc2" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">SCC2</em></a></li><li class="kwd"><a href="/keyword/loader" class="hw-term hw-article-keyword hw-article-keyword-loader" rel="nofollow">loader</a></li><li class="kwd"><a href="/keyword/cohesin" class="hw-term hw-article-keyword hw-article-keyword-cohesin" rel="nofollow">cohesin</a></li><li class="kwd"><a href="/keyword/smc" class="hw-term hw-article-keyword hw-article-keyword-smc" rel="nofollow">SMC</a></li></ul><p id="p-3">The initial association of cohesin with chromatin depends on the loading complex encoded by the <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002588/overview">SCC2</a></em>/<em>NIPBL</em> genes (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ciosk <em>et al.</em> 2000</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Watrin <em>et al.</em> 2006</a>). Cohesin mediates long-range chromatin interactions, and is essential for maintaining genome integrity (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Onn <em>et al.</em> 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Jeppsson <em>et al.</em> 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Marston 2014</a>). During the cell cycle, cohesin ensures accurate segregation of the sister chromatids by tethering them from the time of their replication until their separation during mitosis. In addition, the cohesin loader is associated with nonmitotic cellular and developmental processes, as well as clinical disorders. Cohesin is important for DNA repair, and for the regulation of gene expression (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Onn <em>et al.</em> 2008</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Jeppsson <em>et al.</em> 2014</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Marston 2014</a>). <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> promotes gene expression, and mutations in the gene were identified as the main cause for the developmental disorder Cornelia de Lange syndrome (CdLS) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Krantz <em>et al.</em> 2004</a>). Human <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em> is involved in neural development (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Smith <em>et al.</em> 2014</a>). Despite the unquestioned importance of the loader in cohesin activity, cell functionality and human health, very little is known about the molecular mechanism involved.</p><p id="p-4">The loader proteins <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> and <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> form a stable dimer that physically interacts with cohesin (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Ciosk <em>et al.</em> 2000</a>). The crystal structure of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> with the N-terminal fragment of <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> reveals that <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> is mainly unstructured, and that <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> is wrapped around the <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> polypeptide (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, and Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1/FigureS2.tif">Figure S2</a>) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chao <em>et al.</em> 2015</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hinshaw <em>et al.</em> 2015</a>). Negative staining electron microscopy of the dimer suggests that the loader alternates between extend and compact conformations (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Chao <em>et al.</em> 2015</a>). <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>/<a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> maintains a nucleosome-free region in the DNA, and induces a conformational change in cohesin that enables the chromatin to become entrapped (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Ciosk <em>et al.</em> 2000</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Arumugam <em>et al.</em> 2003</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gruber <em>et al.</em> 2006</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Kurkcuoglu and Bates 2010</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lopez-Serra <em>et al.</em> 2014</a>). After loading is completed, ATP is hydrolyzed by the SMC proteins. In turn, cohesin is dissociated from <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>/<a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> and is translocated from the loading site (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Weitzer <em>et al.</em> 2003</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hu <em>et al.</em> 2011</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ladurner <em>et al.</em> 2014</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Murayama and Uhlmann 2015</a>). Some evidence suggests that <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> is also colocalized with cohesin on chromosome arms (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kogut <em>et al.</em> 2009</a>). Both <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002588/overview">SCC2</a></em> are essential genes; however, while <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> is required for cohesin loading <em>in vitro</em>, <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> is expendable (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Murayama and Uhlmann 2014</a>). Roles of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> in loading, and in other fundamental matters related to the loading mechanism, are still unknown.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overexpression of scc4 RIDs inhibits cell growth. (A) Location of Scc4 insertion (triangles) and nonsense (asterisks) mutations identified in this study. The main structural regions are indicated. The 13 TPR repeats are divided into TPRN and TPRC, which contain eight and five repeats, respectively. The three helices, h1, h2, and h3, form a tertiary structure conserved patch, which enables essential cohesin loading onto the centromeres. The N-terminus of Scc2 passes through the inner cavity of Scc4 and emerges near the N-terminus of Scc4, where Scc2 residues 112&#x2013;120 interact with the outer surface of the first Scc4 TPR repeat (R1). (B) Strain YIO002 (scc4-4) cells carrying pRS406 (pGAL URA3), pIO014 (pGAL-SCC4 CEN URA3), pMS011 (pGAL-scc4-L305ins CEN URA3), pMS012 (pGAL-scc4-V485ins CEN URA3), pMS013 (pGAL-scc4-L490ins CEN URA3), or pMS014 (pGAL-scc4-S505ins CEN URA3), were grown to saturation in SD-URA medium. Tenfold serial dilutions of each strain were plated on SD-URA plates containing either glucose or galactose and grown at 23&#xB0;. (C) Strain YIO002 (scc4-4) cells carrying pRS406 (pGAL URA3), pIO014 (pGAL-SCC4 URA3), pMS015 (pGAL-scc4-E374x CEN URA3), pMS016 (pGAL-scc4-L465x CEN URA3) or pMS017 (pGAL-scc4-Y551x CEN URA3) were grown to saturation in SD-URA medium. Tenfold serial dilutions of each strain were plated on SD-URA plates containing either glucose or galactose and grown at 23&#xB0;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1587597566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overexpression of scc4 RIDs inhibits cell growth. (A) Location of Scc4 insertion (triangles) and nonsense (asterisks) mutations identified in this study. The main structural regions are indicated. The 13 TPR repeats are divided into TPR&lt;sup&gt;N&lt;/sup&gt; and TPR&lt;sup&gt;C&lt;/sup&gt;, which contain eight and five repeats, respectively. The three helices, h1, h2, and h3, form a tertiary structure conserved patch, which enables essential cohesin loading onto the centromeres. The N-terminus of Scc2 passes through the inner cavity of Scc4 and emerges near the N-terminus of Scc4, where Scc2 residues 112&#x2013;120 interact with the outer surface of the first Scc4 TPR repeat (R1). (B) Strain YIO002 (&lt;em&gt;scc4-4&lt;/em&gt;) cells carrying pRS406 (&lt;em&gt;pGAL URA3&lt;/em&gt;), pIO014 (&lt;em&gt;pGAL-SCC4 CEN URA3&lt;/em&gt;), pMS011 (&lt;em&gt;pGAL-scc4-L305ins CEN URA3&lt;/em&gt;), pMS012 (&lt;em&gt;pGAL-scc4-V485ins CEN URA3&lt;/em&gt;), pMS013 (&lt;em&gt;pGAL-scc4-L490ins CEN URA3&lt;/em&gt;), or pMS014 (&lt;em&gt;pGAL-scc4-S505ins CEN URA3&lt;/em&gt;), were grown to saturation in SD-URA medium. Tenfold serial dilutions of each strain were plated on SD-URA plates containing either glucose or galactose and grown at 23&#xB0;. (C) Strain YIO002 (&lt;em&gt;scc4-4&lt;/em&gt;) cells carrying pRS406 (&lt;em&gt;pGAL URA3&lt;/em&gt;), pIO014 (&lt;em&gt;pGAL-SCC4 URA3&lt;/em&gt;), pMS015 (&lt;em&gt;pGAL-scc4-E374x CEN URA3&lt;/em&gt;), pMS016 (&lt;em&gt;pGAL-scc4-L465x CEN URA3&lt;/em&gt;) or pMS017 (&lt;em&gt;pGAL-scc4-Y551x CEN URA3&lt;/em&gt;) were grown to saturation in SD-URA medium. Tenfold serial dilutions of each strain were plated on SD-URA plates containing either glucose or galactose and grown at 23&#xB0;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2655/F1.medium.gif" width="385" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/8/2655/F1.medium.gif" width="385" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468739" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-5" class="first-child">Overexpression of scc4 RIDs inhibits cell growth. (A) Location of Scc4 insertion (triangles) and nonsense (asterisks) mutations identified in this study. The main structural regions are indicated. The 13 TPR repeats are divided into TPR<sup>N</sup> and TPR<sup>C</sup>, which contain eight and five repeats, respectively. The three helices, h1, h2, and h3, form a tertiary structure conserved patch, which enables essential cohesin loading onto the centromeres. The N-terminus of Scc2 passes through the inner cavity of Scc4 and emerges near the N-terminus of Scc4, where Scc2 residues 112–120 interact with the outer surface of the first Scc4 TPR repeat (R1). (B) Strain YIO002 (<em>scc4-4</em>) cells carrying pRS406 (<em>pGAL URA3</em>), pIO014 (<em>pGAL-SCC4 CEN URA3</em>), pMS011 (<em>pGAL-scc4-L305ins CEN URA3</em>), pMS012 (<em>pGAL-scc4-V485ins CEN URA3</em>), pMS013 (<em>pGAL-scc4-L490ins CEN URA3</em>), or pMS014 (<em>pGAL-scc4-S505ins CEN URA3</em>), were grown to saturation in SD-URA medium. Tenfold serial dilutions of each strain were plated on SD-URA plates containing either glucose or galactose and grown at 23°. (C) Strain YIO002 (<em>scc4-4</em>) cells carrying pRS406 (<em>pGAL URA3</em>), pIO014 (<em>pGAL-SCC4 URA3</em>), pMS015 (<em>pGAL-scc4-E374x CEN URA3</em>), pMS016 (<em>pGAL-scc4-L465x CEN URA3</em>) or pMS017 (<em>pGAL-scc4-Y551x CEN URA3</em>) were grown to saturation in SD-URA medium. Tenfold serial dilutions of each strain were plated on SD-URA plates containing either glucose or galactose and grown at 23°.</p><div class="sb-div caption-clear"></div></div></div><p id="p-6">A model for the molecular mechanism of the loader requires at least three intermolecular interactions: <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> and <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>, <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>/<a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> and cohesin, and <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>/<a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> and chromatin. The structure of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> in the complex with <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> shows that the N-terminal region of <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> is important for <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> interaction (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Chao <em>et al.</em> 2015</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Hinshaw <em>et al.</em> 2015</a>). Multiple contacts between the proteins were reported recently (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Chao <em>et al.</em> 2015</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Hinshaw <em>et al.</em> 2015</a>). However, these latter studies did not identify key regions in <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> that contribute to the interaction. In yeast, cohesin loading is regulated by a cell cycle-dependent proteolytic cleavage of <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>. The protein is cleaved after loading in G1/S, and the cleaved product is unable to interact with <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Woodman <em>et al.</em> 2014</a>). Biochemical analysis of <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> containing CdLS-associated mutations revealed that some of these mutations alter the interaction of the mutated <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> with <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>. These observations suggest that the intimate interaction between <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> and <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> is important for proper function of the dimer. Nevertheless, a detailed analysis of the domains in <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> is still required to fully understand the properties of the dimer.</p><p id="p-7">The question as to how <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>/<a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> interacts with cohesin has yet to be addressed. The interaction may depend on <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>, <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>, or <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>/<a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> dimerization. Since these proteins are essential, a temporal deletion is required to approach the matter. Recently, we showed that the loader interacts with at least two subunits of cohesin: the regulatory protein <a href="http://www.yeastgenome.org/locus/S000000665/overview">Scc3</a>, and one or more subunits of the <a href="http://www.yeastgenome.org/locus/S000001886/overview">Smc1</a>–<a href="http://www.yeastgenome.org/locus/S000003610/overview">Smc3</a>–<a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1</a> trimer (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Orgil <em>et al.</em> 2015</a>). Studies in <em>Schizosaccharomyces pombe</em> also showed multiple interactions between Mis4<sup><a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a></sup>–Ssl3<sup><a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a></sup> and cohesin (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Murayama and Uhlmann 2014</a>). These multiple contacts are essential for proper cohesin loading, and <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> may be assumed to contribute to one or more of these interactions.</p><p id="p-8">The third predicted activity of the loader is the interaction with chromatin. This interaction may be direct or indirect, through interaction with a chromatin-associated protein. In cells, chromatin structure is an important factor in proper loading of cohesin (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fernius <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Rudra and Skibbens 2013</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Lopez-Serra <em>et al.</em> 2014</a>). A helix bundle in <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> was shown to promote centromeric cohesion (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Hinshaw <em>et al.</em> 2015</a>). A mutation that interrupts the interaction had a local effect on cohesin centromeric loading, but not on chromosome arm loading (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Hinshaw <em>et al.</em> 2015</a>). The contribution of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> to the association of the <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>/<a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> dimer, with chromatin in both centromeric and arm regions, needs to be explored.</p><p id="p-9"><a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> is a 72 kDa protein that contains TPR repeats (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bermudez <em>et al.</em> 2012</a>; <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Chao <em>et al.</em> 2015</a>; <a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Hinshaw <em>et al.</em> 2015</a>). With the aim of identifying functional regions in <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>, and testing their importance to the cohesin loading mechanism, we applied a screening strategy called Random Insertion of Dominant negative (RID) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Milutinovich <em>et al.</em> 2007</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Eng <em>et al.</em> 2014</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Orgil <em>et al.</em> 2015</a>). We constructed a library of <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> mutant alleles, and used the yeast <em>Saccharomyces cerevisiae</em> to isolate functional mutants. We identified five functional regions, and dissected their importance to <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>/<a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> function. Our data provide new insight into the mechanism of cohesin loading, and into the specific role of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> in the process.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and media</h3><p id="p-10">Yeast strains and plasmids used in this study are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1/TableS2.pdf">Table S2</a>. Yeast strains were grown in SD-URA or YPD medium, as described, supplemented with 2% glucose (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Guthrie and Fink 1991</a>). Medium used for galactose inductions contained SD-URA supplemented with 2% galactose.</p></div><div id="sec-3" class="subsection"><h3>Cell synchronization</h3><p id="p-11">Cells were arrested in the G1 phase by the addition of α-factor (1.5 × 10<sup>–8</sup> M final). To release cells from α-factor-induced G1 arrest, cells were washed twice with YPD medium containing pronase E (0.1 mg/ml; Sigma), and twice with medium without pronase E. Exponentially growing cultures were arrested in G2/M using nocodazole (15 μg/ml final) in the indicated medium.</p></div><div id="sec-4" class="subsection"><h3>RID screen</h3><p id="p-12">The construction of the RID library, and the strategy to isolate <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> mutants is described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1/FigureS1.tif">Figure S1</a>. In short, a library of mutant plasmids was prepared using the EZ-Tn5 In-Frame Linker Insertion Kit (epicenter, Illumina), on a <em><a href="http://www.yeastgenome.org/locus/S000006465/overview">CEN3</a> <a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> plasmid bearing <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em> under the control of the GAL promoter (pIO014), according to the manufacturer’s instructions. Briefly, an <em>in vitro</em> transposon (Tn) insertion reaction randomly inserted a 1100-bp transposon into pIO014. The plasmid library was transformed into bacteria, and a selectable marker on the transposon was used. The Tn was excised by restriction digestion (using a site present at both ends of the Tn), and a new library was made by recircularizing the plasmids, which left an inframe 57-bp insertion at the site of the initial Tn insertion (RID library). The RID library was transformed into haploid strain YIO002 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em>). A thermo-sensitive (ts) strain grown at permissive temperature (23°) was used to increase the sensitivity of the screen. Transformants were grown on SD-URA glycerol plates to select for the RID plasmid, but not to induce overexpression of the <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em>-RID gene. Transformants were kept to a density of about 150 colonies per plate for ease of screening, and incubated at 23° until colonies formed. We then replica-plated SD-URA plates containing either glucose (noninducing) or galactose (inducing), and incubated at 23°. Colonies that were inviable on galactose were retested to confirm this phenotype. Plasmids were isolated from galactose-sensitive transformants, and the location of the 57 bp insertion was determined by sequencing of the entire gene. To confirm linkage of inviability to candidate RID plasmid, hits were retransformed into strain YIO002 and treated as described above to confirm that the <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em>-RID plasmid was responsible for toxicity on galactose medium.</p></div><div id="sec-5" class="subsection"><h3>Immunoprecipitation and Western blotting analysis</h3><p id="p-13">Cells were grown to midlog phase, pelleted, washed with dH<sub>2</sub>O, and frozen in liquid nitrogen. Pellets were resuspended in 350 μl IPH50 [50 mM Tris pH 8.0, 50 mM NaCl, 5 mM EDTA, 0.5% NP-40, 5 mM β-mercaptoethanol, protease inhibitor cocktail (Sigma)]. Cells were lysed by adding glass beads (Sigma) to the resuspended pellets followed by four working cycles of 1 min in a bullet blender (Next Advance). The lysates were cleared by two centrifugations of 5 min and 15 min at 1000 × <em>g</em> and 14,000 × <em>g</em>, respectively, at 4°. Immunoprecipitations were performed at 4° adding the appropriate antibodies for 2 h. The antibodies were collected on protein A/G agarose (Santa Cruz) or on magnetic beads (BIO-RAD) for 1 hr, and washed three times with IPH50 and resuspended in 35 μl Laemmli buffer. Standard procedures for sodium dodecyl sulfate–polyacrylamide gel electrophoresis and Western blotting were followed to transfer proteins from gels to a polyscreen PVDF membrane (Millipore). Membranes were blotted with the primary antibodies. Antibodies were detected using SuperSignal West Pico (Thermo Scientific) and LAS 4000 (GE Healthcare). Antibodies used in this study were: anti-HA (12CA5, Roche), anti-MYC (9E10, Roche), anti-V5 (Invitrogen/Millipore), anti-3Flag (Sigma), and anti-6His (Sigma).</p></div><div id="sec-6" class="subsection"><h3>Site-directed mutagenesis</h3><p id="p-14">Site-directed mutagenesis was performed on pMS2 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em>-3V5, <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em>) by using Q5 Site-Directed Mutagenesis Kit (New England Biolabs, Inc) following the manufacturer’s instructions. Primers used for the reactions are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1/TableS3.pdf">Table S3</a>.</p></div><div id="sec-7" class="subsection"><h3>Cohesion assay and chromatin immunoprecipitation (ChIP)</h3><p id="p-15">Cohesion at <em><a href="http://www.yeastgenome.org/locus/S000002642/overview">LYS4</a></em> was assayed using the LacI-GFP/LacO array. Cells were treated as described in the text and processed to visualize GFP foci by microscopy as described previously (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Orgil <em>et al.</em> 2015</a>). Each experiment was repeated three times, and at least 300 cells were counted for each time-point in each experimental condition. ChIP was performed as described in <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Orgil <em>et al.</em> (2015</a>). Primers used for qPCR are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1/TableS3.pdf">Table S3</a>.</p></div><div id="sec-8" class="subsection"><h3>Auxin-induced depletion</h3><p id="p-16">An auxin-induced degron (AID) system for yeast was previously described (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Morawska and Ulrich 2013</a>). Cells were grown to an early midlog phase (OD<sub>600</sub> = 0.4) in YPD, then split in half. 3-Indoleacetic acid (IAA, Sigma) was diluted in 70% ethanol and added to one-half of the culture to a final concentration of 1 mM. The second culture was not treated with IAA but both halves were incubated in the dark for an additional 1.5 h before being processed for immunoprecipitation as described above.</p></div><div id="sec-9" class="subsection"><h3>Microscopy</h3><p id="p-17">Wide-field fluorescence images were obtained using the Zeiss AxioImager M2 fully motorized microscope (100X Plan-Apo, 1.4NA) fitted with an AxioCamHRm CCD High Resolution Camera.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-18">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Identification of functional domains in Scc4 by a genetic screen for RID mutants</h3><p id="p-19">RID is an efficient strategy, used previously by us and others to isolate mutant alleles of genes encoding cohesin subunits (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Milutinovich <em>et al.</em> 2007</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Eng <em>et al.</em> 2014</a>; <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Orgil <em>et al.</em> 2015</a>). A library of <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em> RID mutants was prepared using a transposon insertion kit, in which a 57 bp DNA fragment was randomly inserted into a CEN <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> plasmid harboring the <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em> coding sequence under the control of the GAL promoter (<em>Materials and Methods</em> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1/FigureS1.tif">Figure S1</a>). During preparation of the library, we noticed residual kanamycin-containing plasmids (0.9 CFU/ml). This indicated that cleavage of the transposon was incomplete (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1/FigureS1.tif">Figure S1</a>). The <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em> mutant library was transformed into haploid strain YIO002 containing the ts allele <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em>, and cell growth was assayed on both glucose and galactose plates at the permissive temperature (23°). The <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em> ts allele background was used to increase the sensitivity of the screen. We identified seven colonies that exhibited toxicity or lethality when grown on galactose, but that were unaffected when grown on glucose. Plasmids that inhibited growth were isolated and the location of the insertion was determined by sequencing. We identified four plasmids that we collectively called <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a></em>-RIDs. These contain the expected 57-bp insertion in the <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em> open reading frame after amino acids L305 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins</em>), V485 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-V485ins</em>), L490 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L490ins</em>), and S505 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-S505ins</em>) (pMS011, pMS012, pMS013, and pMS014, respectively). Unexpectedly, we also isolated three plasmids in which the full transposon was inserted. In these cases, the result of the full-length transposon insertion was a nonsense allele just after the insertion site. The stop codons were located after residues E374 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> 374</em>*), L465 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> 465*</em>), and Y551 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> 551*</em>) (pMS015, pMS016, and pMS017, respectively) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1/FigureS2.tif">Figure S2</a>).</p><p id="p-20">To verify the dominant negative effect of the mutants on cell growth, we tested strains containing the GAL-inducible <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a></em> mutant alleles by a semiquantitative growth assay. Overexpression of all nonsense alleles, as well as <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins</em>, <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-V485ins</em>, and <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L490ins</em> was lethal, while cells overexpressing <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-S505ins</em> showed 1000-fold growth inhibition compared to the control strain (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, B and C</a>).</p></div><div id="sec-13" class="subsection"><h3>Overexpression of the N-terminal region of Scc4 inhibits cell growth</h3><p id="p-21">Our screen revealed that C-terminal truncations of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> are toxic to cells. To identify the specific region associated with this phenotype, we performed a systematic truncation analysis. We constructed two CEN <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> plasmids that encode C-terminal <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a></em> truncations under the control of a GAL promoter (pMS018 and pMS019) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Plasmids were transformed into haploid strain YIO002, and their toxicity was tested on either glucose or galactose plates. We showed previously that overexpression of the C-terminal truncation alleles <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> E374*</em>, <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> L465*</em>, and <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> Y551*</em> completely inhibited cell growth (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a>). Further reduction of the construct to the first 187 amino acids was still toxic. However, overexpression of a fragment containing the first 94 amino acids of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> did not have a phenotypic effect on cell growth (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). This finding suggests that an important domain is located between residues 94 and 187.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The 187-amino-acid N-terminal fragment of Scc4 is toxic when overexpressed. (A) The N-terminal truncations used in this study. (B) Strain YIO002 (scc4-4) cells carrying pRS406 (pGAL URA3), pIO014 (pGAL-SCC4 URA3), pMS018 (pGAL-scc4-(1-187) CEN URA3), or pMS019 (pGAL-scc4-(1-94) CEN URA3) were grown at 23&#xB0; to saturation in SD-URA medium. Tenfold serial dilutions of each strain were plated on SD-URA plates containing either glucose or galactose and grown at 23&#xB0;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1587597566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The 187-amino-acid N-terminal fragment of Scc4 is toxic when overexpressed. (A) The N-terminal truncations used in this study. (B) Strain YIO002 (&lt;em&gt;scc4-4&lt;/em&gt;) cells carrying pRS406 (&lt;em&gt;pGAL URA3&lt;/em&gt;), pIO014 (&lt;em&gt;pGAL-SCC4 URA3&lt;/em&gt;), pMS018 (&lt;em&gt;pGAL-scc4-(1-187) CEN URA3&lt;/em&gt;), or pMS019 (&lt;em&gt;pGAL-scc4-(1-94) CEN URA3&lt;/em&gt;) were grown at 23&#xB0; to saturation in SD-URA medium. Tenfold serial dilutions of each strain were plated on SD-URA plates containing either glucose or galactose and grown at 23&#xB0;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2655/F2.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/8/2655/F2.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468901" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">The 187-amino-acid N-terminal fragment of Scc4 is toxic when overexpressed. (A) The N-terminal truncations used in this study. (B) Strain YIO002 (<em>scc4-4</em>) cells carrying pRS406 (<em>pGAL URA3</em>), pIO014 (<em>pGAL-SCC4 URA3</em>), pMS018 (<em>pGAL-scc4-(1-187) CEN URA3</em>), or pMS019 (<em>pGAL-scc4-(1-94) CEN URA3</em>) were grown at 23° to saturation in SD-URA medium. Tenfold serial dilutions of each strain were plated on SD-URA plates containing either glucose or galactose and grown at 23°.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">The crystal structure of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> with the <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> fragment suggest that the domain located between amino acids 91 and 187 is most likely the interaction domain with <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> (<a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Chao <em>et al.</em> 2015</a>; <a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">Hinshaw <em>et al.</em> 2015</a>). To test the interactions mediated by the <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> N-terminus, we expressed a 6 × His-tagged <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> (1–187) fragment in bacteria. The <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>-(1–187)-6 × His protein was soluble and we were able to purify it on nickel beads (data not shown). To test our hypothesis, we set up pull-down experiments of <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> from yeast extract. We repeated the experiment several times with different modifications of the protocol, but were unable to pull-down <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> with the <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> fragment (data not shown). The inability to pull-down <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> in these experiments is consistent with the cofolding model of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> and <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> (<a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Chao <em>et al.</em> 2015</a>; <a id="xref-ref-10-8" class="xref-bibr" href="#ref-10">Hinshaw <em>et al.</em> 2015</a>).</p></div><div id="sec-14" class="subsection"><h3>Single copy Scc4 RIDs do not support cell growth</h3><p id="p-24">To further dissect the molecular defect of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>-RIDs, we tagged both wild type <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em> and the <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a></em>-RID mutant alleles with three copies of V5 epitopes (3V5) at their C-terminus, and cloned the genes under the control of the endogenous <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em> promoter. These alleles were integrated into the haploid strain YIO002 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em>) at the <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> locus. The <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4 allele has been reported previously (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Ciosk <em>et al.</em> 2000</a>). It supports cell viability at 23° but not at 35°. The semi-permissive temperature is 30°. Cells are viable, but growth is slower compared to the isogenic wild-type strain (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1/FigureS3.tif">Figure S3A</a>). At the restrictive temperature, cohesin is not loaded on the chromosomes at either the centromere or chromosome arms, and sister chromatid cohesion is lost (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1/FigureS3.tif">Figure S3B</a>). Inactivation of the <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4 allele at 35° at the G1 stage of the cell cycle does not block cell progression to G2/M (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5C</a>). In recent years, an AID system has been used as an alternative to ts alleles (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Milutinovich <em>et al.</em> 2007</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Eng <em>et al.</em> 2014</a>; <a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Orgil <em>et al.</em> 2015</a>). We tried to construct such an <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a>-AID strain. However, the degradation of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>-AID is not efficient (data not shown). Therefore, the AID system is not feasible for studying the effects of the <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> mutants isolated in this study. Consequently, from here on, we used the <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4 allele background to characterize the mutants in this work.</p><p id="p-25">The parent strain alone, or bearing <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a>-3V5</em>, <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins-3V5</em>, <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-V485ins-3V5</em>, <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L490ins-3V5</em>, or <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-S505ins-3V5</em>, were grown to saturation, serially diluted on YPD plates, and incubated at 23° or 35°, the respective permissive and restrictive temperatures of the <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em> allele (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). At 23°, the <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em> strain alone, and the <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> alleles containing the insertions grew equally well. This suggests that when expressed in endogenous levels, <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> mutants are recessive, in contrast to their dominant-negative phenotype, which is overexpressed (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>). At 35°, the <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em> strain alone and all strains containing a RID allele failed to grow, whereas the <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a>-3V5 <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em> strain was viable.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="scc4 RID mutants do not support cell viability under the level of native expression. (A) Strains YIO002 (scc4-4), YMS1004 (SCC4-3V5 scc4-4), YMS1006 (scc4L305ins-3V5 scc4-4), YMS1007 (scc4V485ins-3V5 scc4-4), YMS1005 (scc4L490ins-3V5 scc4-4) and YMS1008 (scc4-S505ins-3V5 scc4-4) were grown to saturation in YPD media at 23&#xB0;. Tenfold serial dilutions of each strain were plated on YPD plates and grown at either the permissive (23&#xB0;) or restrictive (35&#xB0;) temperature for scc4-4. (B) Protein extracts from strains YMS1003 (SCC2-12Myc), YMS1016 (SCC2-12Myc SCC4-3V5), YMS1017 (SCC2-12Myc scc4-L305ins-3V5), YMS1018 (SCC2-12Myc scc4V485ins-3V5), YMS1019 (SCC2-12Myc scc4L490ins-3V5), and YMS1020 (SCC2-12Myc scc4- S505ins-3V5) were analyzed by Western blot with antibodies against V5 (Scc4-3V5) and Myc (Scc2-12Myc). A representative blot is shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1587597566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;scc4 RID mutants do not support cell viability under the level of native expression. (A) Strains YIO002 (&lt;em&gt;scc4-4&lt;/em&gt;), YMS1004 (&lt;em&gt;SCC4-3V5 scc4-4&lt;/em&gt;), YMS1006 (&lt;em&gt;scc4L305ins-3V5 scc4-4&lt;/em&gt;), YMS1007 (&lt;em&gt;scc4V485ins-3V5 scc4-4&lt;/em&gt;), YMS1005 (&lt;em&gt;scc4L490ins-3V5 scc4-4&lt;/em&gt;) and YMS1008 (&lt;em&gt;scc4-S505ins-3V5 scc4-4&lt;/em&gt;) were grown to saturation in YPD media at 23&#xB0;. Tenfold serial dilutions of each strain were plated on YPD plates and grown at either the permissive (23&#xB0;) or restrictive (35&#xB0;) temperature for &lt;em&gt;scc4-4&lt;/em&gt;. (B) Protein extracts from strains YMS1003 (&lt;em&gt;SCC2-12Myc&lt;/em&gt;), YMS1016 (&lt;em&gt;SCC2-12Myc SCC4-3V5&lt;/em&gt;), YMS1017 (&lt;em&gt;SCC2-12Myc scc4-L305ins-3V5&lt;/em&gt;), YMS1018 (&lt;em&gt;SCC2-12Myc scc4V485ins-3V5&lt;/em&gt;), YMS1019 (&lt;em&gt;SCC2-12Myc scc4L490ins-3V5&lt;/em&gt;), and YMS1020 (&lt;em&gt;SCC2-12Myc scc4- S505ins-3V5&lt;/em&gt;) were analyzed by Western blot with antibodies against V5 (Scc4-3V5) and Myc (Scc2-12Myc). A representative blot is shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2655/F3.medium.gif" width="440" height="422"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/8/2655/F3.medium.gif" width="440" height="422"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468910" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">scc4 RID mutants do not support cell viability under the level of native expression. (A) Strains YIO002 (<em>scc4-4</em>), YMS1004 (<em>SCC4-3V5 scc4-4</em>), YMS1006 (<em>scc4L305ins-3V5 scc4-4</em>), YMS1007 (<em>scc4V485ins-3V5 scc4-4</em>), YMS1005 (<em>scc4L490ins-3V5 scc4-4</em>) and YMS1008 (<em>scc4-S505ins-3V5 scc4-4</em>) were grown to saturation in YPD media at 23°. Tenfold serial dilutions of each strain were plated on YPD plates and grown at either the permissive (23°) or restrictive (35°) temperature for <em>scc4-4</em>. (B) Protein extracts from strains YMS1003 (<em>SCC2-12Myc</em>), YMS1016 (<em>SCC2-12Myc SCC4-3V5</em>), YMS1017 (<em>SCC2-12Myc scc4-L305ins-3V5</em>), YMS1018 (<em>SCC2-12Myc scc4V485ins-3V5</em>), YMS1019 (<em>SCC2-12Myc scc4L490ins-3V5</em>), and YMS1020 (<em>SCC2-12Myc scc4- S505ins-3V5</em>) were analyzed by Western blot with antibodies against V5 (Scc4-3V5) and Myc (Scc2-12Myc). A representative blot is shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">To verify expression of the tagged alleles <em>in vivo</em>, we analyzed protein extract prepared from these strains by Western blot with antibodies against the V5 tag. In all strains, we identified a single band of the expected molecular weight that was absent in the untagged parent, indicating that the protein is expressed. However, we noticed a reduction in the steady-state level of the mutant proteins compared with the wild-type protein. No similar reduction was observed in <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>-12Myc in the same cells (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>), suggesting that the instability is not associated with the V5-tag. Similar levels of protein were detected when cells were grown at either 23°, 30°, or 35° suggesting that the protein level is not temperature dependent (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1/FigureS4.tif">Figure S4</a>). The reduction in <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> mutants cannot be explained by the addition of the tag itself because it was not observed in the wild-type protein. Therefore, we concluded that the instability is associated with the mutation in the proteins. However, the detectable steady-state levels of the proteins should be sufficient for molecular characterization of the associated phenotypes.</p><p id="p-28">The inability of the <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> -RIDs to support cell viability can be explained by two models. In the first model, when expressed to endogenous levels, the mutant alleles are recessive, and do not support cell viability as a sole copy. Alternatively, failure of the RID mutants to support viability is the result of protein instability. If the first model is correct, we expect that some of the molecular interactions will be maintained. However, in the other case, the mutants will not be able to interact with <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> or cohesin. To discriminate between these models, we explored the integrity of the loader.</p></div><div id="sec-15" class="subsection"><h3>Scc4 RIDs interact with Scc2</h3><p id="p-29">We examined the ability of <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins-3V5</em>, <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-V485ins-3V5</em>, <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L490ins-3V5</em>, and <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-</em>S<em>505ins-3V5</em> to coimmunoprecipitate the loader subunit <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>. Extracts from YMS1003 (<em><a href="http://www.yeastgenome.org/locus/S000002588/overview">SCC2</a>-12Myc</em>), YMS1016 (<em><a href="http://www.yeastgenome.org/locus/S000002588/overview">SCC2</a>-12Myc <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a>-3V5</em>), YMS1017 (<em><a href="http://www.yeastgenome.org/locus/S000002588/overview">SCC2</a>-12Myc <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins-3V5</em>), YMS1018 (<em><a href="http://www.yeastgenome.org/locus/S000002588/overview">SCC2</a>-12Myc <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-V485ins-3V5</em>), YMS1019 (<em><a href="http://www.yeastgenome.org/locus/S000002588/overview">SCC2</a>-12Myc <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>L490ins-3V5</em>), and YMS1020 (<em><a href="http://www.yeastgenome.org/locus/S000002588/overview">SCC2</a>-12Myc <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-S505ins-3V5</em>) cells were prepared. Cells were grown to midlog phase, and <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> was precipitated by using anti-V5 antibodies (<em>Materials and Methods</em>). <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>-12Myc coprecipitated with wild-type <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>. However, the coprecipitation of <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>-12Myc with <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins-3V5, <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-V485ins-3V5, <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L490ins-3V5, and <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-<em>S</em>505ins-3V5 was similarly reduced to about 50% of the wild-type level (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). The reduction in co-IP levels cannot be explained by the reduced levels of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>-3V5 in the cells since protein levels in the IP were similar. This suggests that the antibody was the limiting factor in the experiment. As such, <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> -RIDs appear to maintain their ability to interact with <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>, while stability of the dimer is reduced. If the loader is formed in the context of the mutants, the activity and effect on cohesin can be studied further.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RID mutations do not affect loader integrity. (A) Haploid YMS1003 (SCC2-12Myc), YMS1016 (SCC2-12Myc SCC4-3V5), YMS1017 (SCC2-12Myc scc4-L305ins-3V5), YMS1018 (SCC2-12Myc scc4V485ins-3V5), YMS1019 (SCC2-12Myc scc4L490ins-3V5), and YMS1020 (SCC2-12Myc scc4S505ins-3V5) were grown to midlog phase in YPD medium at 23&#xB0;. Cells were lysed, and the protein extracts were subjected to IP against the V5 tag (Scc4). Precipitated Scc2 was analyzed by Western blot using antibodies against Myc. (B) Strains YIO002 (scc4-4), YMS1004 (SCC4-3V5 scc4-4), YMS1006 (scc4L305ins-3V5 scc4-4), YMS1007 (scc4V485ins-3V5 scc4-4), YMS1005 (scc4L490ins-3V5 scc4-4), and YMS1008 (scc4S505ins-3V5 scc4-4) were processed for ChIP analysis. V5 tagged Scc4 proteins were immunoprecipitated (n&#xA0;=&#xA0;3). Precipitated DNA was analyzed by quantitative PCR. Analysis of centromere&#xA0;III is shown. (C) Analysis of centromere IV is shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1587597566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;RID mutations do not affect loader integrity. (A) Haploid YMS1003 (&lt;em&gt;SCC2-12Myc&lt;/em&gt;), YMS1016 (&lt;em&gt;SCC2-12Myc SCC4-3V5&lt;/em&gt;), YMS1017 (&lt;em&gt;SCC2-12Myc scc4-L305ins-3V5&lt;/em&gt;), YMS1018 (&lt;em&gt;SCC2-12Myc scc4V485ins-3V5&lt;/em&gt;), YMS1019 (&lt;em&gt;SCC2-12Myc scc4L490ins-3V5&lt;/em&gt;), and YMS1020 (&lt;em&gt;SCC2-12Myc scc4S505ins-3V5&lt;/em&gt;) were grown to midlog phase in YPD medium at 23&#xB0;. Cells were lysed, and the protein extracts were subjected to IP against the V5 tag (Scc4). Precipitated Scc2 was analyzed by Western blot using antibodies against Myc. (B) Strains YIO002 (&lt;em&gt;scc4-4&lt;/em&gt;), YMS1004 (&lt;em&gt;SCC4-3V5 scc4-4&lt;/em&gt;), YMS1006 (&lt;em&gt;scc4L305ins-3V5 scc4-4&lt;/em&gt;), YMS1007 (&lt;em&gt;scc4V485ins-3V5 scc4-4&lt;/em&gt;), YMS1005 (&lt;em&gt;scc4L490ins-3V5 scc4-4&lt;/em&gt;), and YMS1008 (&lt;em&gt;scc4S505ins-3V5 scc4-4&lt;/em&gt;) were processed for ChIP analysis. V5 tagged Scc4 proteins were immunoprecipitated (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;3). Precipitated DNA was analyzed by quantitative PCR. Analysis of centromere&#xA0;III is shown. (C) Analysis of centromere IV is shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2655/F4.medium.gif" width="440" height="358"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/8/2655/F4.medium.gif" width="440" height="358"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468916" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">RID mutations do not affect loader integrity. (A) Haploid YMS1003 (<em>SCC2-12Myc</em>), YMS1016 (<em>SCC2-12Myc SCC4-3V5</em>), YMS1017 (<em>SCC2-12Myc scc4-L305ins-3V5</em>), YMS1018 (<em>SCC2-12Myc scc4V485ins-3V5</em>), YMS1019 (<em>SCC2-12Myc scc4L490ins-3V5</em>), and YMS1020 (<em>SCC2-12Myc scc4S505ins-3V5</em>) were grown to midlog phase in YPD medium at 23°. Cells were lysed, and the protein extracts were subjected to IP against the V5 tag (Scc4). Precipitated Scc2 was analyzed by Western blot using antibodies against Myc. (B) Strains YIO002 (<em>scc4-4</em>), YMS1004 (<em>SCC4-3V5 scc4-4</em>), YMS1006 (<em>scc4L305ins-3V5 scc4-4</em>), YMS1007 (<em>scc4V485ins-3V5 scc4-4</em>), YMS1005 (<em>scc4L490ins-3V5 scc4-4</em>), and YMS1008 (<em>scc4S505ins-3V5 scc4-4</em>) were processed for ChIP analysis. V5 tagged Scc4 proteins were immunoprecipitated (<em>n</em> = 3). Precipitated DNA was analyzed by quantitative PCR. Analysis of centromere III is shown. (C) Analysis of centromere IV is shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">We tested the ability of the <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>/<a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-RID complex to bind chromatin by ChIP. Strains YMS1004 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a>-3V5</em>), YMS1005 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L490ins-3V5</em>), YMS1006 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins-3V5</em>), YMS1007 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-V485ins-3V5</em>), and YMS1008 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-S505ins-3V5</em>) were arrested in the G2/M phase, and processed for ChIP using anti-V5 antibodies. Then, <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> binding to chromosome III and IV centromeres was assessed using quantitative PCR analysis (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, B and C</a>). No binding was detected in the presence of <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-V485ins-3V5, <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins-3V5, and <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L490ins-3V5, while 60% of wild-type binding was found for <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-S505ins-3V5. These results suggest that the regions flanking L305 and L490 are essential for stable binding of the loader to chromatin. The fact that <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> RIDs interact with <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> supports the supposition that the activity of the <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>/<a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> dimer requires an intimate interaction between the two proteins.</p></div><div id="sec-16" class="subsection"><h3>Scc4 RIDs do not support sister chromatid cohesion</h3><p id="p-32"><a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> RIDs and <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> dimerize but the mutant loader is unable to support cell viability. In this case, we expect that the nonfunctional single copy mutant <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> will not support cohesion in mitotic cells. To test this possibility, we measured cohesion by the GFP dot assay (<em>Materials and Methods</em>). Strains YMS1010 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em>), YMS1011 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a>-3V5 <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em>), YMS1012 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L490ins-3V5 <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em>), YMS1013 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins-3V5 <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em>), YMS1014 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-V485ins-3V5 <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em>), and YMS1015 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-S505ins-3V5 <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em>) were arrested in G1 phase, shifted to the nonpermissive temperature for <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em> (35.5°), then released from G1 into medium containing nocodazole at 35.5°, to rearrest cells in G2/M (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). About 95% of <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a>-3V5 <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em> control cells had a single GFP dot, indicative of robust cohesion. In contrast, in <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em> cells, and in <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins-3V5 <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em>, <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-V485ins-3V5 <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em>, and <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L490ins-3V5 <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em> cells, ∼50% of two GFP spots were detected in cells arrested at the G2/M phase, indicative of a severe defect in cohesion (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>). The cohesion defect in <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-S505ins-3V5 <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em> cells was milder, and consistent with the milder growth phenotype when this allele was overexpressed (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>). Flow cytometry of the cells indicated that, under these conditions, cells progress through S phase and were arrested in G2/M. This suggests that the two spots are not the result of a defect in DNA replication (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>). An interaction between <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> and <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> RIDs seems to be insufficient for a functional loader.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="scc4-RIDs do not support sister chromatin cohesion. (A) Flowchart of the experimental design to score sister chromatid cohesion. (B) Strains YIO002 (scc4-4), YMS1011 (SCC4-3V5 scc4-4), YMS1013 (scc4L305ins-3V5 scc4-4), YMS1014 (scc4V485ins-3V5 scc4-4), YMS1012 (scc4L490ins-3V5 scc4-4), and YMS1015 (scc4S505ins-3V5 scc4-4) were processed for cohesion assay as shown in A (n&#xA0;=&#xA0;3). (C) Flow cytometry analysis of strains from the cohesion assay." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1587597566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;scc4-RIDs do not support sister chromatin cohesion. (A) Flowchart of the experimental design to score sister chromatid cohesion. (B) Strains YIO002 (&lt;em&gt;scc4-4&lt;/em&gt;), YMS1011 (&lt;em&gt;SCC4-3V5 scc4-4&lt;/em&gt;), YMS1013 (&lt;em&gt;scc4L305ins-3V5 scc4-4&lt;/em&gt;), YMS1014 (&lt;em&gt;scc4V485ins-3V5 scc4-4&lt;/em&gt;), YMS1012 (&lt;em&gt;scc4L490ins-3V5 scc4-4&lt;/em&gt;), and YMS1015 (&lt;em&gt;scc4S505ins-3V5 scc4-4&lt;/em&gt;) were processed for cohesion assay as shown in A (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;3). (C) Flow cytometry analysis of strains from the cohesion assay.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2655/F5.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/8/2655/F5.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468804" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">scc4-RIDs do not support sister chromatin cohesion. (A) Flowchart of the experimental design to score sister chromatid cohesion. (B) Strains YIO002 (<em>scc4-4</em>), YMS1011 (<em>SCC4-3V5 scc4-4</em>), YMS1013 (<em>scc4L305ins-3V5 scc4-4</em>), YMS1014 (<em>scc4V485ins-3V5 scc4-4</em>), YMS1012 (<em>scc4L490ins-3V5 scc4-4</em>), and YMS1015 (<em>scc4S505ins-3V5 scc4-4</em>) were processed for cohesion assay as shown in A (<em>n</em> = 3). (C) Flow cytometry analysis of strains from the cohesion assay.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Scc4 mutants inhibit cohesin loading</h3><p id="p-34">Given the importance of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> to cohesin loading, we predicted that the <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>-RIDs/<a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> complexes are unable to load cohesin onto the chromosomes. To test this hypothesis we performed ChIP from strain YIO002 (<em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4)</em> carrying the cohesin subunit <em><a href="http://www.yeastgenome.org/locus/S000001886/overview">SMC1</a></em> tagged with six copies of hemagglutinin epitope (<em><a href="http://www.yeastgenome.org/locus/S000001886/overview">SMC1</a>-6HA</em>), and <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins-3V5</em>, <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-V485ins-3V5</em>, <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L490ins-3V5</em>, or <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-S505ins-3V5</em>. Strains were grown at permissive temperature, arrested in the G2/M phase at 23°, shifted to 35.5° to inactivate <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em>, then released back to the cell cycle at 35.5° and rearrested in G2/M. The arrested cells were processed for ChIP using anti-HA antibodies (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). <a href="http://www.yeastgenome.org/locus/S000001886/overview">Smc1</a> binding to the known cohesin associated regions CARC1 and MAT CAR on chromosome III, and centromeres III and IV was measured using quantitative PCR (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6, B–E</a>). In cells carrying the <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a></em> wild-type allele, <a href="http://www.yeastgenome.org/locus/S000001886/overview">Smc1</a> was detected, as expected. In the presence of <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins-3V5</em>, <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-V485ins-3V5</em>, and <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L490ins-3V5</em>, <a href="http://www.yeastgenome.org/locus/S000001886/overview">Smc1</a> was not detected in two arm cohesin associated regions (CARs), nor in centromeres III and IV. This suggests that mutant <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> are unable to load cohesin onto chromosome arms. Cohesin loading in the presence of <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4-S505ins</a> was lower than in the wild type but greater than background levels. This supports previous data.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scc4 is required for cohesin loading. (A) Flowchart of the experimental design. (B) Strains YIO002 (scc4-4), YMS1004 (SCC43V5 scc4-4), YMS1006 (scc4L305ins-3V5 scc4-4), YMS1007 (scc4V485ins-3V5 scc4-4), YMS1005 (scc4L490ins-3V5 scc4-4), and YMS1008 (scc4S505ins-3V5 scc4-4) were processed for ChIP analysis. Scc4, V5 tagged proteins were immunoprecipitated. Precipitated chromosome III CARC1 DNA was analyzed by quantitative PCR (n&#xA0;=&#xA0;3). (C) Analysis of the MAT CAR on chromosome III. (D) Analysis of centromere III. (E) Analysis of centromere IV." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1587597566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Scc4 is required for cohesin loading. (A) Flowchart of the experimental design. (B) Strains YIO002 (&lt;em&gt;scc4-4&lt;/em&gt;), YMS1004 (&lt;em&gt;SCC43V5 scc4-4&lt;/em&gt;), YMS1006 (&lt;em&gt;scc4L305ins-3V5 scc4-4&lt;/em&gt;), YMS1007 (&lt;em&gt;scc4V485ins-3V5 scc4-4&lt;/em&gt;), YMS1005 (&lt;em&gt;scc4L490ins-3V5 scc4-4&lt;/em&gt;), and YMS1008 (&lt;em&gt;scc4S505ins-3V5 scc4-4&lt;/em&gt;) were processed for ChIP analysis. Scc4, V5 tagged proteins were immunoprecipitated. Precipitated chromosome III CARC1 DNA was analyzed by quantitative PCR (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;3). (C) Analysis of the MAT CAR on chromosome III. (D) Analysis of centromere III. (E) Analysis of centromere IV.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2655/F6.medium.gif" width="433" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/8/2655/F6.medium.gif" width="433" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468954" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-35" class="first-child">Scc4 is required for cohesin loading. (A) Flowchart of the experimental design. (B) Strains YIO002 (<em>scc4-4</em>), YMS1004 (<em>SCC43V5 scc4-4</em>), YMS1006 (<em>scc4L305ins-3V5 scc4-4</em>), YMS1007 (<em>scc4V485ins-3V5 scc4-4</em>), YMS1005 (<em>scc4L490ins-3V5 scc4-4</em>), and YMS1008 (<em>scc4S505ins-3V5 scc4-4</em>) were processed for ChIP analysis. Scc4, V5 tagged proteins were immunoprecipitated. Precipitated chromosome III CARC1 DNA was analyzed by quantitative PCR (<em>n</em> = 3). (C) Analysis of the MAT CAR on chromosome III. (D) Analysis of centromere III. (E) Analysis of centromere IV.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>The interaction of Scc4 with cohesin depends on Scc2</h3><p id="p-36"><a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> interacts with <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> to form the loading dimer and the dimer interacts with cohesin. To test the ability of the <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> mutants to interact with cohesin, we used cell extracts from the strains containing the <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>-RID-3V5 and the cohesin subunit <a href="http://www.yeastgenome.org/locus/S000001886/overview">Smc1</a> tagged with six copies of hemagglutinin epitope (<a href="http://www.yeastgenome.org/locus/S000001886/overview">SMC1</a>-6HA): YIO002 <em>(<a href="http://www.yeastgenome.org/locus/S000001886/overview">SMC1</a>-6HA)</em>, YMS1004 (<em><a href="http://www.yeastgenome.org/locus/S000001886/overview">SMC1</a>-6HA <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a>-3V5</em>), YMS1005 (<em><a href="http://www.yeastgenome.org/locus/S000001886/overview">SMC1</a>-6HA <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L490ins-3V5</em>), YMS1006 (<em><a href="http://www.yeastgenome.org/locus/S000001886/overview">SMC1</a>-6HA <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins-3V5</em>), YMS1007 (<em><a href="http://www.yeastgenome.org/locus/S000001886/overview">SMC1</a>-6HA <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-V485ins-3V5</em>), and YMS1008 (<em><a href="http://www.yeastgenome.org/locus/S000001886/overview">SMC1</a>-6HA <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-S505ins-3V</em>). The cells were grown to midlog phase, lysed and subjected to immunoprecipitation with anti-V5 antibodies. The co-IP of Smc1-6HA was at wild type levels with <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>, <em>s</em>cc4-V485ins-3V5, <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L490ins-3V5, and <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-S505ins-3V5, but reduced with <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-L305ins-3V5 (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RID mutations do not affect the interaction with cohesin. (A) Haploid YIO002 (SMC1-6HA), YMS1004 (SMC1-6HA SCC4-3V5), YMS1006 (SMC1-6HA scc4-L305ins-3V5), YMS1007 (SMC1-6HA scc4-V485ins-3V5), YMS1005 (SMC1-6HA scc4-L490ins-3V5) or YMS1008 (SMC1-6HA scc4-S505ins-3V5) were grown to midlog phase in YPD media at 23&#xB0;. Cells were processed and subjected to immunoprecipitation against the V5 tag (Scc4). Precipitated proteins were analyzed by Western blot using antibodies against V5 (IP) and HA (co-IP). (B) Haploid YMS1021 (SCC2-AID-3V5 SCC4-3V5), and YMS1022 (SCC2-AID-3V5 SCC4-3V5 SMC1-3FLAG) cells were grown to midlog phase in YPD media at 23&#xB0;. The cultures were divided into two, and cells were grown for an additional 1.5&#xA0;hr with or without the addition of 1&#xA0;mM IAA. Cells were lysed and the extract was analyzed by Western blot with anti-V5 and anti-FLAG antibodies. (C) The protein extracts from A were subjected to immunoprecipitation against the FLAG tag (Smc1). Precipitated Scc4 was analyzed by Western blot using antibodies against V5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1587597566" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;RID mutations do not affect the interaction with cohesin. (A) Haploid YIO002 (&lt;em&gt;SMC1-6HA&lt;/em&gt;), YMS1004 (&lt;em&gt;SMC1-6HA SCC4-3V5&lt;/em&gt;), YMS1006 (&lt;em&gt;SMC1-6HA scc4-L305ins-3V5&lt;/em&gt;), YMS1007 (&lt;em&gt;SMC1-6HA scc4-V485ins-3V5&lt;/em&gt;), YMS1005 (&lt;em&gt;SMC1-6HA scc4-L490ins-3V5&lt;/em&gt;) or YMS1008 (&lt;em&gt;SMC1-6HA scc4-S505ins-3V5&lt;/em&gt;) were grown to midlog phase in YPD media at 23&#xB0;. Cells were processed and subjected to immunoprecipitation against the V5 tag (Scc4). Precipitated proteins were analyzed by Western blot using antibodies against V5 (IP) and HA (co-IP). (B) Haploid YMS1021 (&lt;em&gt;SCC2-AID-3V5 SCC4-3V5&lt;/em&gt;), and YMS1022 (&lt;em&gt;SCC2-AID-3V5 SCC4-3V5 SMC1-3FLAG&lt;/em&gt;) cells were grown to midlog phase in YPD media at 23&#xB0;. The cultures were divided into two, and cells were grown for an additional 1.5&#xA0;hr with or without the addition of 1&#xA0;mM IAA. Cells were lysed and the extract was analyzed by Western blot with anti-V5 and anti-FLAG antibodies. (C) The protein extracts from A were subjected to immunoprecipitation against the FLAG tag (Smc1). Precipitated Scc4 was analyzed by Western blot using antibodies against V5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2655/F7.medium.gif" width="440" height="345"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/8/2655/F7.medium.gif" width="440" height="345"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2655/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468974" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-37" class="first-child">RID mutations do not affect the interaction with cohesin. (A) Haploid YIO002 (<em>SMC1-6HA</em>), YMS1004 (<em>SMC1-6HA SCC4-3V5</em>), YMS1006 (<em>SMC1-6HA scc4-L305ins-3V5</em>), YMS1007 (<em>SMC1-6HA scc4-V485ins-3V5</em>), YMS1005 (<em>SMC1-6HA scc4-L490ins-3V5</em>) or YMS1008 (<em>SMC1-6HA scc4-S505ins-3V5</em>) were grown to midlog phase in YPD media at 23°. Cells were processed and subjected to immunoprecipitation against the V5 tag (Scc4). Precipitated proteins were analyzed by Western blot using antibodies against V5 (IP) and HA (co-IP). (B) Haploid YMS1021 (<em>SCC2-AID-3V5 SCC4-3V5</em>), and YMS1022 (<em>SCC2-AID-3V5 SCC4-3V5 SMC1-3FLAG</em>) cells were grown to midlog phase in YPD media at 23°. The cultures were divided into two, and cells were grown for an additional 1.5 hr with or without the addition of 1 mM IAA. Cells were lysed and the extract was analyzed by Western blot with anti-V5 and anti-FLAG antibodies. (C) The protein extracts from A were subjected to immunoprecipitation against the FLAG tag (Smc1). Precipitated Scc4 was analyzed by Western blot using antibodies against V5.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">At this stage, we sought to test if <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> interacts directly with cohesin. To explore this possibility, we built strains YMS1021 and YMS1022, in which <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> can be depleted <em>in vivo</em> by an AID system (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>). Strain YMS1021 contains <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> tagged with 3V5, while YMS1022 contains <a href="http://www.yeastgenome.org/locus/S000001886/overview">Smc1</a> and <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> tagged with 3FLAG and 3V5 epitopes, respectively. We immunoprecipitated <a href="http://www.yeastgenome.org/locus/S000001886/overview">Smc1</a> from untreated cells, or from cells treated with 1 mM IAA to deplete <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>, and tested the coprecipitation of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>. As expected, <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> was detected when <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> was present. However, we could not detect <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> when <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> was depleted (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7C</a>). This shows that <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> is indispensable for the <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>–cohesin interaction.</p><p id="p-39">To further dissect the domains that mediate the molecular interactions of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>, we introduced missense mutations in residues that are located in the insertion region, and that are evolutionarily conserved. The mutant alleles were transformed into <em><a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a>-4</em> cells, and tested for their ability to support cell growth by a semiquantitative assay. None of the point mutations we inserted mimicked the phenotype of the insertion alleles, and no effect on cell growth was observed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1/FigureS5.tif">Figure S5</a>).</p></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-40">Analysis of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> reveals a complex pattern of molecular interactions. We used a genetic approach to isolate mutants that highlighted specific properties of the protein. The RID screen revealed four functional domains in the protein. This work demonstrates the power of the RID genetic screen approach in identifying functional domains in proteins that mediate complex interactions. We showed that the regions we identified in the protein affect cohesin loading onto chromosomes and cohesion but they have a minor effect on the binding of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> to its loading-dimer partner <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>, and the physical interaction with cohesin.</p><p id="p-41">The genetic screen we describe is based on the dominant negative nature of the alleles when overexpressed in the presence of a functional allele. Ectopic overexpression of <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a> in yeast cells does not affect cell growth (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>). Random insertion of a short sequence into the overexpressed <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a> plasmid can lead to several outcomes. First, an insertion may have no effect on the expression of <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a> if is localized to a nonfunctional region of the vector. In this case, ectopic <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a> expression will not be affected. Second, the insertion may affect expression by abolishing the gene promoter. In this case, the ectopic <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a> will not be expressed. However, cell viability is maintained by the native <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a> allele and growth will not be affected. Third, the short sequence is inserted in the coding sequence, thus destroying protein activity or stability. As before, these possibilities will have no phenotypic outcome because cells contain a second <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a> allele. Regarding the last possibility, insertion is again in the coding sequence. However, in this scenario, the mutated protein is partially active. This can happen if the protein has several functions that are mediated by distinct domains, <em>e.g.</em>, two separate protein–protein interaction domains. The insertion locally destroys one domain without affecting other domains of the protein. This partially active protein can form partial inactive complexes. When the partial active allele is overexpressed, it competes with the native allele, and titers out essential interactors that are required for the formation of an active complex. In contrast to the other possibilities, this scenario, known as a dominant negative effect, will have a phenotypic outcome, which, in the case of <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a>, is loss of cell viability and cohesin loading. The dominant negative effect can be detected in multi-functional, multi-domain proteins, and is apparent only when the effect of the mutation is limited to a defined domain, with no global effects on protein stability or function. <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> is a multifunctional protein. Three molecular interactions have been associated with <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>: <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>, cohesin, and chromatin. The isolation of dominant negative mutants in this study suggests that these functions are mediated by distinct domains that can be disrupted locally, while the other functions of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> are maintained.</p><p id="p-42">The dominant negative effect of the overexpressed <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> alleles demonstrates that the insertion mutants are undoubtedly partly active. However, when RID mutants were expressed as a single copy, their levels were lower than those of the wild type proteins. This may be due to lower expression, or to instability of the proteins. The activities of growth inhibition, failure to load cohesin, and exclusion of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> from the chromatin described herein can be explained by the partial activity of the mutant proteins, as a result of disruption of a functional domain. However, an alternative explanation is the reduced levels of the protein. The first explanation is more likely for several reasons. First, we constructed a library in which transposons were inserted every three amino acids, yet we isolated only five mutants that are dominant negative when overexpressed. In the second scenario, we would expect to find a large number of loss-of-function mutants. Second, the dominant-negative property of the mutants indicates, without any doubt, that the overexpressed mutant proteins are partially active. Third, while the cell growth phenotype can be associated with loss-of function, protein–protein interaction of mutant <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> with both <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> and <a href="http://www.yeastgenome.org/locus/S000001886/overview">Smc1</a> are maintained. This suggests that protein complexes are formed but are inactive. Fourth, our analysis identified mutations in the helix bundle, which was previously identified as an important region for centromeric cohesin loading (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Fernius <em>et al.</em> 2013</a>). The identification of one important domain increases confidence that the other domains identified by the screen are of functional importance. Finally, the stability of the S505ins mutant is similar to that of the other mutants. However, this mutant reveals phenotypes that are different from the other mutants, indicating that the expression level is sufficient to support the loader’s functions. Altogether, the results of the screen provide reliable information on the structure–function relationship of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>.</p><p id="p-43">Previous studies paid little attention to <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>, and even less to its specific functions in loader activity. Most recently, the crystal structure of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> in the complex with the N-terminus of <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> was solved (<a id="xref-ref-3-8" class="xref-bibr" href="#ref-3">Chao <em>et al.</em> 2015</a>; <a id="xref-ref-10-9" class="xref-bibr" href="#ref-10">Hinshaw <em>et al.</em> 2015</a>). <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> is wrapped around <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>, which is mostly unfolded. The RID mutants we examined were located at different regions of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>, but they reduced the interaction affinity with <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> in a similar manner. Interestingly, the RID mutations in <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> are located in proximity to small regions in which the secondary structure of <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> is defined. The integration of the structural, genetic, and biochemical data suggests that <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> forms a tight complex with <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>, and that a compact structure is required for loading activity.</p><p id="p-44">We and others have shown that <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a>/<a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> has multiple interactions with cohesin (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Murayama and Uhlmann 2014</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">2015</a>; <a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Orgil <em>et al.</em> 2015</a>). <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> is a required component of the loader, which mediates the interaction with cohesin. The results of this study elucidate the contribution of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> to the binding. We showed that the essential <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> binding domain is located at the N-terminus of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>. Our results also suggest that <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> is essential for cohesin binding. Interestingly, <a href="http://www.yeastgenome.org/locus/S000000949/overview">scc4</a> -L305ins demonstrated reduced binding to the <a href="http://www.yeastgenome.org/locus/S000001886/overview">Smc1</a>. L305 is located at the conserved patch of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> (<a id="xref-ref-10-10" class="xref-bibr" href="#ref-10">Hinshaw <em>et al.</em> 2015</a>). In contrast to a previous study showing that mutations in this region reduce cohesin binding to centromeres but not chromosome arms, the L305ins mutant failed to load cohesin in both regions (<a id="xref-ref-10-11" class="xref-bibr" href="#ref-10">Hinshaw <em>et al.</em> 2015</a>). The results imply that this region is important but not essential for the interaction with cohesin.</p><p id="p-45">Mutations in <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> are the major cause of CdLS (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Krantz <em>et al.</em> 2004</a>). However, no mutation has been reported in <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> in association with a human disorder. Three possible explanations arise: one is that mutations in <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a> have not yet been identified but will be identified in human patients in the future. The second possibility is that <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> is not essential in humans. Studies have shown that RNA knockdown of <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a> in HeLa cells causes a chromosome segregation defect, and that knockout <a href="http://www.yeastgenome.org/locus/S000000949/overview">SCC4</a> mice showed a more severe phenotype than did <a href="http://www.yeastgenome.org/locus/S000002588/overview">SCC2</a> knockout mice (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Seitan <em>et al.</em> 2006</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Smith <em>et al.</em> 2014</a>). Third, inactivation of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> by a mutation is lethal. This raises the question as to the functional difference between <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> and <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>. <a href="http://www.yeastgenome.org/locus/S000002588/overview">Scc2</a> may play a role in transcription that is independent of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Dorsett <em>et al.</em> 2005</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lin <em>et al.</em> 2011</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Lopez-Serra <em>et al.</em> 2014</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Zakari and Gerton 2015</a>). The instability of our mutants also suggests that the tolerance of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a> to mutation may be low, and such changes may lead to cell death. Our work contributes to deciphering the role of <a href="http://www.yeastgenome.org/locus/S000000949/overview">Scc4</a>, and sheds new light on the mechanisms of cohesin loading and genome stability.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-46">We thank the members of the Onn laboratory for their discussion and comments on the manuscript, and Yehuda Brody, Dafna Ben-Yosef, and Eyla Maoz for their technical help with imaging and cytometry. This work was supported by Foundation Jérôme Lejeune grant no. 1146-OI2013_A (to I.O.), and Leona M. and Harry B. Helmsley Charitable Trust grant no. 2012PG-ISL013 (to I.O.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-47">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031674/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-48"><em>Communicating editor: S. L. Jaspersen</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-05-27"><span class="received-label">Received </span>May 27, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-06-02"><span class="accepted-label">Accepted </span>June 2, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Shwartz <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.1" data-doi="10.1016/j.cub.2003.10.036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arumugam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haering</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mechtler</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">ATP hydrolysis is required for cohesin’s association with chromosomes.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1941</span>–<span class="cit-lpage">1953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DArumugam%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D22%26rft.spage%253D1941%26rft.epage%253D1953%26rft.atitle%253DATP%2Bhydrolysis%2Bis%2Brequired%2Bfor%2Bcohesin%2527s%2Bassociation%2Bwith%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2003.10.036%26rft_id%253Dinfo%253Apmid%252F14614819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2003.10.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14614819&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186558100020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.2" data-doi="10.1073/pnas.1206840109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bermudez</span>  <span class="cit-name-given-names">V. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farina</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higashi</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tappin</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">In vitro loading of human cohesin on DNA by the human Scc2-Scc4 loader complex.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">9366</span>–<span class="cit-lpage">9371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1206840109%26rft_id%253Dinfo%253Apmid%252F22628566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzI0LzkzNjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.3" data-doi="10.1016/j.celrep.2015.06.071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murayama</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munoz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uhlmann</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Structural studies reveal the functional modularity of the Scc2-Scc4 cohesin loader.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">719</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D12%26rft.spage%253D719%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2015.06.071%26rft_id%253Dinfo%253Apmid%252F26212329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2015.06.071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26212329&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.4" data-doi="10.1016/S1097-2765(00)80420-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ciosk</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirayama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shevchenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toth</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Cohesin’s binding to chromosomes depends on a separate complex consisting of Scc2 and Scc4 proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DCiosk%26rft.auinit1%253DR.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253D243%26rft.epage%253D254%26rft.atitle%253DCohesin%2527s%2Bbinding%2Bto%2Bchromosomes%2Bdepends%2Bon%2Ba%2Bseparate%2Bcomplex%2Bconsisting%2Bof%2BScc2%2Band%2BScc4%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252800%252980420-7%26rft_id%253Dinfo%253Apmid%252F10882066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(00)80420-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10882066&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085645100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.5" data-doi="10.1242/dev.02064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dorsett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eissenberg</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misulovin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martens</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redding</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Effects of sister chromatid cohesion proteins on cut gene expression during wing development in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">4743</span>–<span class="cit-lpage">4753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.02064%26rft_id%253Dinfo%253Apmid%252F16207752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTMyLzIxLzQ3NDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.6" data-doi="10.1091/mbc.E14-04-0929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eng</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guacci</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koshland</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">ROCC, a conserved region in cohesin’s Mcd1 subunit, is essential for the proper regulation of the maintenance of cohesion and establishment of condensation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2351</span>–<span class="cit-lpage">2364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E14-04-0929%26rft_id%253Dinfo%253Apmid%252F24966169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xNi8yMzUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI2NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.7" data-doi="10.1016/j.cub.2013.02.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernius</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nerusheva</span>  <span class="cit-name-given-names">O. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galander</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alves Fde</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rappsilber</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Cohesin-dependent association of scc2/4 with the centromere initiates pericentromeric cohesion establishment.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">599</span>–<span class="cit-lpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D23%26rft.spage%253D599%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2013.02.022%26rft_id%253Dinfo%253Apmid%252F23499533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2013.02.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23499533&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.8" data-doi="10.1016/j.cell.2006.08.048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arumugam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuglitsch</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helmhart</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Evidence that loading of cohesin onto chromosomes involves opening of its SMC hinge.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">523</span>–<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGruber%26rft.auinit1%253DS.%26rft.volume%253D127%26rft.issue%253D3%26rft.spage%253D523%26rft.epage%253D537%26rft.atitle%253DEvidence%2Bthat%2Bloading%2Bof%2Bcohesin%2Bonto%2Bchromosomes%2Binvolves%2Bopening%2Bof%2Bits%2BSMC%2Bhinge.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.08.048%26rft_id%253Dinfo%253Apmid%252F17081975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.08.048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17081975&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241937000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-6.8.2655.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guthrie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-source">Guide to Yeast Genetics and Molecular Biology</span>, <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hinshaw</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makrantoni</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Structural evidence for Scc4-dependent localization of cohesin loading.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span>: e06057.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.11" data-doi="10.1016/j.cub.2010.12.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itoh</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mishra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">ATP hydrolysis is required for relocating cohesin from sites occupied by its Scc2/4 loading complex.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DHu%26rft.auinit1%253DB.%26rft.volume%253D21%26rft.issue%253D1%26rft.spage%253D12%26rft.epage%253D24%26rft.atitle%253DATP%2Bhydrolysis%2Bis%2Brequired%2Bfor%2Brelocating%2Bcohesin%2Bfrom%2Bsites%2Boccupied%2Bby%2Bits%2BScc2%252F4%2Bloading%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2010.12.004%26rft_id%253Dinfo%253Apmid%252F21185190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2010.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21185190&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.12" data-doi="10.1038/nrm3857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeppsson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanno</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirahige</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sjogren</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The maintenance of chromosome structure: positioning and functioning of SMC complexes.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">601</span>–<span class="cit-lpage">614</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMol.%2BCell%2BBiol.%26rft.volume%253D15%26rft.spage%253D601%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm3857%26rft_id%253Dinfo%253Apmid%252F25145851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm3857&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25145851&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.13" data-doi="10.1101/gad.1819409"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kogut</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guacci</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mistry</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megee</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Scc2/Scc4 cohesin loader determines the distribution of cohesin on budding yeast chromosomes.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2345</span>–<span class="cit-lpage">2357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKogut%26rft.auinit1%253DI.%26rft.volume%253D23%26rft.issue%253D19%26rft.spage%253D2345%26rft.epage%253D2357%26rft.atitle%253DThe%2BScc2%252FScc4%2Bcohesin%2Bloader%2Bdetermines%2Bthe%2Bdistribution%2Bof%2Bcohesin%2Bon%2Bbudding%2Byeast%2Bchromosomes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1819409%26rft_id%253Dinfo%253Apmid%252F19797771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIzLzE5LzIzNDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.14" data-doi="10.1038/ng1364"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krantz</span>  <span class="cit-name-given-names">I. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCallum</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeScipio</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaur</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gillis</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Cornelia de Lange syndrome is caused by mutations in <em>NIPBL</em>, the human homolog of <em>Drosophila melanogaster Nipped-B</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">631</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKrantz%26rft.auinit1%253DI.%2BD.%26rft.volume%253D36%26rft.issue%253D6%26rft.spage%253D631%26rft.epage%253D635%26rft.atitle%253DCornelia%2Bde%2BLange%2Bsyndrome%2Bis%2Bcaused%2Bby%2Bmutations%2Bin%2BNIPBL%252C%2Bthe%2Bhuman%2Bhomolog%2Bof%2BDrosophila%2Bmelanogaster%2BNipped-B.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1364%26rft_id%253Dinfo%253Apmid%252F15146186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1364&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15146186&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221763700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.15" data-doi="10.1016/j.bpj.2010.06.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurkcuoglu</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mechanism of cohesin loading onto chromosomes: a conformational dynamics study.</span> <abbr class="cit-jnl-abbrev">Biophys. J.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1212</span>–<span class="cit-lpage">1220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.aulast%253DKurkcuoglu%26rft.auinit1%253DO.%26rft.volume%253D99%26rft.issue%253D4%26rft.spage%253D1212%26rft.epage%253D1220%26rft.atitle%253DMechanism%2Bof%2Bcohesin%2Bloading%2Bonto%2Bchromosomes%253A%2Ba%2Bconformational%2Bdynamics%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2010.06.006%26rft_id%253Dinfo%253Apmid%252F20713005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bpj.2010.06.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20713005&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281103200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.16" data-doi="10.1016/j.cub.2014.08.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ladurner</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhaskara</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huis in ’t Veld</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">I. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreidl</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cohesin’s ATPase activity couples cohesin loading onto DNA with Smc3 acetylation.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2228</span>–<span class="cit-lpage">2237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D24%26rft.spage%253D2228%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2014.08.011%26rft_id%253Dinfo%253Apmid%252F25220052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2014.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25220052&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.17" data-doi="10.1091/mbc.E10-06-0545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampton</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Scc2 regulates gene expression by recruiting cohesin to the chromosome as a transcriptional activator during yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1985</span>–<span class="cit-lpage">1996</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E10-06-0545%26rft_id%253Dinfo%253Apmid%252F21508318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMi8xOTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI2NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.18" data-doi="10.1038/ng.3080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Serra</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uhlmann</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Scc2-Scc4 complex acts in sister chromatid cohesion and transcriptional regulation by maintaining nucleosome-free regions.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1147</span>–<span class="cit-lpage">1151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D1147%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3080%26rft_id%253Dinfo%253Apmid%252F25173104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25173104&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.19" data-doi="10.1534/genetics.112.145144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Chromosome segregation in budding yeast: sister chromatid cohesion and related mechanisms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMarston%26rft.auinit1%253DA.%2BL.%26rft.volume%253D196%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D63%26rft.atitle%253DChromosome%2BSegregation%2Bin%2BBudding%2BYeast%253A%2BSister%2BChromatid%2BCohesion%2Band%2BRelated%2BMechanisms%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.145144%26rft_id%253Dinfo%253Apmid%252F24395824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk2LzEvMzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.20" data-doi="10.1371/journal.pgen.0030012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milutinovich</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Unal</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skibbens</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koshland</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A multi-step pathway for the establishment of sister chromatid cohesion.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMilutinovich%26rft.auinit1%253DM.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253De12%26rft.epage%253De12%26rft.atitle%253DA%2Bmulti-step%2Bpathway%2Bfor%2Bthe%2Bestablishment%2Bof%2Bsister%2Bchromatid%2Bcohesion.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030012%26rft_id%253Dinfo%253Apmid%252F17238288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17238288&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.21" data-doi="10.1002/yea.2967"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morawska</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulrich</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An expanded tool kit for the auxin-inducible degron system in budding yeast.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%26rft.volume%253D30%26rft.spage%253D341%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.2967%26rft_id%253Dinfo%253Apmid%252F23836714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.2967&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23836714&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.22" data-doi="10.1038/nature12867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murayama</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uhlmann</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Biochemical reconstitution of topological DNA binding by the cohesin ring.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">505</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">371</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D505%26rft.spage%253D367%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12867%26rft_id%253Dinfo%253Apmid%252F24291789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12867&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24291789&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329621800035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.23" data-doi="10.1016/j.cell.2015.11.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murayama</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uhlmann</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">DNA entry into and exit out of the cohesin ring by an interlocking gate mechanism.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">1628</span>–<span class="cit-lpage">1640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D163%26rft.spage%253D1628%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.11.030%26rft_id%253Dinfo%253Apmid%252F26687354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.11.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26687354&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.24" data-doi="10.1146/annurev.cellbio.24.110707.175350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Onn</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heidinger-Pauli</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guacci</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Unal</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koshland</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sister chromatid cohesion: a simple concept with a complex reality.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Cell Dev. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bcell%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BCell%2BDev%2BBiol%26rft.aulast%253DOnn%26rft.auinit1%253DI.%26rft.volume%253D24%26rft.spage%253D105%26rft.epage%253D129%26rft.atitle%253DSister%2Bchromatid%2Bcohesion%253A%2Ba%2Bsimple%2Bconcept%2Bwith%2Ba%2Bcomplex%2Breality.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.cellbio.24.110707.175350%26rft_id%253Dinfo%253Apmid%252F18616427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.cellbio.24.110707.175350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18616427&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260851500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.25" data-doi="10.1371/journal.pgen.1005036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orgil</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matityahu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eng</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guacci</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koshland</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A conserved domain in the scc3 subunit of cohesin mediates the interaction with both mcd1 and the cohesin loader complex.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005036</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005036%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005036%26rft_id%253Dinfo%253Apmid%252F25748820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25748820&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.26" data-doi="10.1371/journal.pone.0075435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rudra</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skibbens</span>  <span class="cit-name-given-names">R. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Chl1 DNA helicase regulates Scc2 deposition specifically during DNA-replication in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e75435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De75435%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0075435%26rft_id%253Dinfo%253Apmid%252F24086532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0075435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24086532&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.27" data-doi="10.1371/journal.pbio.0040242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seitan</span>  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laval</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Majid</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorsett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Metazoan Scc4 homologs link sister chromatid cohesion to cell and axon migration guidance.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DSeitan%26rft.auinit1%253DV.%2BC.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De242%26rft.epage%253De242%26rft.atitle%253DMetazoan%2BScc4%2Bhomologs%2Blink%2Bsister%2Bchromatid%2Bcohesion%2Bto%2Bcell%2Band%2Baxon%2Bmigration%2Bguidance.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040242%26rft_id%253Dinfo%253Apmid%252F16802858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16802858&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laval</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rock</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strachan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Neural crest cell-specific inactivation of <em>Nipbl</em> or <em>Mau2</em> during mouse development results in a late onset of craniofacial defects.</span> <abbr class="cit-jnl-abbrev">Genesis</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">687</span>–<span class="cit-lpage">694</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenesis%26rft.volume%253D52%26rft.spage%253D687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.29" data-doi="10.1016/j.cub.2006.03.049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watrin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schleiffer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisenhaber</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasmyth</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Human Scc4 is required for cohesin binding to chromatin, sister-chromatid cohesion, and mitotic progression.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">863</span>–<span class="cit-lpage">874</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DWatrin%26rft.auinit1%253DE.%26rft.volume%253D16%26rft.issue%253D9%26rft.spage%253D863%26rft.epage%253D874%26rft.atitle%253DHuman%2BScc4%2Bis%2Brequired%2Bfor%2Bcohesin%2Bbinding%2Bto%2Bchromatin%252C%2Bsister-chromatid%2Bcohesion%252C%2Band%2Bmitotic%2Bprogression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2006.03.049%26rft_id%253Dinfo%253Apmid%252F16682347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2006.03.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16682347&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237684700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.30" data-doi="10.1016/j.cub.2003.10.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weitzer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehane</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uhlmann</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A model for ATP hydrolysis-dependent binding of cohesin to DNA.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1930</span>–<span class="cit-lpage">1940</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DWeitzer%26rft.auinit1%253DS.%26rft.volume%253D13%26rft.issue%253D22%26rft.spage%253D1930%26rft.epage%253D1940%26rft.atitle%253DA%2Bmodel%2Bfor%2BATP%2Bhydrolysis-dependent%2Bbinding%2Bof%2Bcohesin%2Bto%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2003.10.030%26rft_id%253Dinfo%253Apmid%252F14614818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2003.10.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14614818&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186558100019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.31" data-doi="10.1073/pnas.1321722111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woodman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fara</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dzieciatkowska</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trejo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luong</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cell cycle-specific cleavage of Scc2 regulates its cohesin deposition activity.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">7060</span>–<span class="cit-lpage">7065</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1321722111%26rft_id%253Dinfo%253Apmid%252F24778232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzE5LzcwNjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjY1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2655.32" data-doi="10.1080/15384101.2015.1010959"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zakari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerton</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The SMC loader Scc2 regulates gene expression.</span> <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">943</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BCycle%26rft.volume%253D14%26rft.spage%253D943%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15384101.2015.1010959%26rft_id%253Dinfo%253Apmid%252F25715109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15384101.2015.1010959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25715109&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2655.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/8/2655.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2643" title="RNAi Screen Identifies Novel Regulators of RNP Granules in the Caenorhabditis elegans Germ Line" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2665" title="Phenotypes Associated with Second Chromosome P Element Insertions in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/8"><span>Volume 6 Issue 8, August 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/8.cover-source.jpg?itok=40Kz9Oeq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2655.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-EsJ-rrrEF9vzE37-s-j8KhqE1ngfG4x5unWf62rwmy0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/mutant-screen-report" class="" data-icon-position="" data-hide-link-title="0">Mutant Screen Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/8/2655" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/8/2655" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/8/2655" class="active" data-icon-position="" data-hide-link-title="0">Identification of Functional Domains in the Cohesin Loader Subunit Scc4 by a Random Insertion/Dominant Negative Screen</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460259" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Uy-i64bRokHbkd71JjVusjYhru9EuBdozdDvath4qk8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2655.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/8/2655" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-pU0GbkhAgqrUH_Q3rAY_H4Mht7XiNKTKQJrHJN1MmOg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/8/2655" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2655.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460259" id="node460259--2" data-pisa="ggg;6/8/2655" data-pisa-master="ggg;g3.116.031674" data-apath="/ggg/6/8/2655.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Functional Domains in the Cohesin Loader Subunit Scc4 by a Random Insertion/Dominant Negative Screen</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michal Shwartz</span>, <span class="highwire-citation-author" data-delta="1">Avi Matityahu</span> and <span class="highwire-citation-author" data-delta="2">Itay Onn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2655-2663;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031674 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460259--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michal Shwartz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Medicine in the Galilee, Bar-Ilan University, Safed, 1311502, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michal%2BShwartz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shwartz%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichal%2BShwartz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Avi Matityahu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Medicine in the Galilee, Bar-Ilan University, Safed, 1311502, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Avi%2BMatityahu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matityahu%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAvi%2BMatityahu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Itay Onn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Medicine in the Galilee, Bar-Ilan University, Safed, 1311502, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Itay%2BOnn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Onn%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AItay%2BOnn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Itay.Onn@biu.ac.il" class="" data-icon-position="" data-hide-link-title="0">Itay.Onn@biu.ac.il</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/8/2655" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/8/2655&amp;title=Identification%20of%20Functional%20Domains%20in%20the%20Cohesin%20Loader%20Subunit%20Scc4%20by%20a%20Random%20Insertion/Dominant%20Negative%20Screen" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/8/2655&amp;title=Identification%20of%20Functional%20Domains%20in%20the%20Cohesin%20Loader%20Subunit%20Scc4%20by%20a%20Random%20Insertion/Dominant%20Negative%20Screen" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/8/2655&amp;title=Identification%20of%20Functional%20Domains%20in%20the%20Cohesin%20Loader%20Subunit%20Scc4%20by%20a%20Random%20Insertion/Dominant%20Negative%20Screen" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/8/2655&amp;text=Identification%20of%20Functional%20Domains%20in%20the%20Cohesin%20Loader%20Subunit%20Scc4%20by%20a%20Random%20Insertion/Dominant%20Negative%20Screen" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/8/2655&amp;title=Identification%20of%20Functional%20Domains%20in%20the%20Cohesin%20Loader%20Subunit%20Scc4%20by%20a%20Random%20Insertion/Dominant%20Negative%20Screen" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/8/2655&amp;t=Identification%20of%20Functional%20Domains%20in%20the%20Cohesin%20Loader%20Subunit%20Scc4%20by%20a%20Random%20Insertion/Dominant%20Negative%20Screen" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/8/2655&amp;title=Identification%20of%20Functional%20Domains%20in%20the%20Cohesin%20Loader%20Subunit%20Scc4%20by%20a%20Random%20Insertion/Dominant%20Negative%20Screen" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/8/2655&amp;title=Identification%20of%20Functional%20Domains%20in%20the%20Cohesin%20Loader%20Subunit%20Scc4%20by%20a%20Random%20Insertion/Dominant%20Negative%20Screen" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460259/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460259" id="node460259--3" data-pisa="ggg;6/8/2655" data-pisa-master="ggg;g3.116.031674" data-apath="/ggg/6/8/2655.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identification of Functional Domains in the Cohesin Loader Subunit Scc4 by a Random Insertion/Dominant Negative Screen</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michal Shwartz</span>, <span class="highwire-citation-author" data-delta="1">Avi Matityahu</span> and <span class="highwire-citation-author" data-delta="2">Itay Onn</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2655-2663;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031674 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460259--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michal Shwartz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Medicine in the Galilee, Bar-Ilan University, Safed, 1311502, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michal%2BShwartz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shwartz%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichal%2BShwartz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Avi Matityahu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Medicine in the Galilee, Bar-Ilan University, Safed, 1311502, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Avi%2BMatityahu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Matityahu%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAvi%2BMatityahu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Itay Onn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Faculty of Medicine in the Galilee, Bar-Ilan University, Safed, 1311502, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Itay%2BOnn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Onn%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AItay%2BOnn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Itay.Onn@biu.ac.il" class="" data-icon-position="" data-hide-link-title="0">Itay.Onn@biu.ac.il</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2655&amp;rft.epage=2663&amp;rft.atitle=Identification%20of%20Functional%20Domains%20in%20the%20Cohesin%20Loader%20Subunit%20Scc4%20by%20a%20Random%20Insertion/Dominant%20Negative%20Screen&amp;rft.volume=6&amp;rft.issue=8&amp;rft.date=2016-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Shwartz%2C+Michal&amp;rft.au=Matityahu%2C+Avi&amp;rft.au=Onn%2C+Itay"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460259/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460259/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460259/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460259/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460259/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460259/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460259/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460259/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460259/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460259/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460259/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460259/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495856" id="node495856" data-pisa="ggg;10/3/917" data-pisa-master="ggg;g3.119.400936" data-apath="/ggg/10/3/917.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/917" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Mutational Analysis of N-Ethyl-N-Nitrosourea (ENU) in the Fission Yeast <em>Schizosaccharomyces pombe</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495835" id="node495835" data-pisa="ggg;g3.119.400927v1" data-pisa-master="ggg;g3.119.400927" data-apath="/ggg/early/2020/02/21/g3.119.400927.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/02/21/g3.119.400927" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identification of Suppressors of <em>top-2 </em>Embryonic Lethality in <em>Caenorhabditis elegans</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495548" id="node495548" data-pisa="ggg;10/2/479" data-pisa-master="ggg;g3.119.400933" data-apath="/ggg/10/2/479.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/479" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identification of the Genetic Requirements for Zinc Tolerance and Toxicity in <em>Saccharomyces cerevisiae</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Mutant%20Screen%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Mutant Screen Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/8/2655" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/8/2655.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/8/2655.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/8/2655.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/8/2655.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
