<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/8/2583" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/8/2583.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/8/2583.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/8/2583.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/8/2583/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/8/2583.atom" />
<meta name="HW.pisa" content="ggg;6/8/2583" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Evolution of the Insertion-Deletion Mutation Rate Across the Tree of Life" />
<meta name="DC.Identifier" content="10.1534/g3.116.030890" />
<meta name="DC.Date" content="2016-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Sung et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Mutations are the ultimate source of variation used for evolutionary adaptation, while also being predominantly deleterious and a source of genetic disorders. Understanding the rate of insertion-deletion mutations (indels) is essential to understanding evolutionary processes, especially in coding regions, where such mutations can disrupt production of essential proteins. Using direct estimates of indel rates from 14 phylogenetically diverse eukaryotic and bacterial species, along with measures of standing variation in such species, we obtain results that imply an inverse relationship of mutation rate and effective population size. These results, which corroborate earlier observations on the base-substitution mutation rate, appear most compatible with the hypothesis that natural selection reduces mutation rates per effective genome to the point at which the power of random genetic drift (approximated by the inverse of effective population size) becomes overwhelming. Given the substantial differences in DNA metabolism pathways that give rise to these two types of mutations, this consistency of results raises the possibility that refinement of other molecular and cellular traits may be inversely related to species-specific levels of random genetic drift." />
<meta name="DC.Contributor" content="Way Sung" />
<meta name="DC.Contributor" content="Matthew S. Ackerman" />
<meta name="DC.Contributor" content="Marcus M. Dillon" />
<meta name="DC.Contributor" content="Thomas G. Platt" />
<meta name="DC.Contributor" content="Clay Fuqua" />
<meta name="DC.Contributor" content="Vaughn S. Cooper" />
<meta name="DC.Contributor" content="Michael Lynch" />
<meta name="article:published_time" content="2016-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Evolution of the Insertion-Deletion Mutation Rate Across the Tree of Life" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Mutations are the ultimate source of variation used for evolutionary adaptation, while also being predominantly deleterious and a source of genetic disorders. Understanding the rate of insertion-deletion mutations (indels) is essential to understanding evolutionary processes, especially in coding regions, where such mutations can disrupt production of essential proteins. Using direct estimates of indel rates from 14 phylogenetically diverse eukaryotic and bacterial species, along with measures of standing variation in such species, we obtain results that imply an inverse relationship of mutation rate and effective population size. These results, which corroborate earlier observations on the base-substitution mutation rate, appear most compatible with the hypothesis that natural selection reduces mutation rates per effective genome to the point at which the power of random genetic drift (approximated by the inverse of effective population size) becomes overwhelming. Given the substantial differences in DNA metabolism pathways that give rise to these two types of mutations, this consistency of results raises the possibility that refinement of other molecular and cellular traits may be inversely related to species-specific levels of random genetic drift.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/08/01" />
<meta name="citation_mjid" content="ggg;6/8/2583" />
<meta name="citation_id" content="6/8/2583" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/8/2583" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/8/2583.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/8/2583.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/8/2583.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.030890" />
<meta name="citation_pmid" content="27317782" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2583" />
<meta name="citation_lastpage" content="2591" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Way Sung" />
<meta name="citation_author_institution" content="Department of Bioinformatics and Genomics, University of North Carolina at Charlotte, North Carolina 28223" />
<meta name="citation_author_email" content="waysung@indiana.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8336-8913" />
<meta name="citation_author" content="Matthew S. Ackerman" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Marcus M. Dillon" />
<meta name="citation_author_institution" content="Microbiology Graduate Program, University of New Hampshire, Durham, New Hampshire 03824" />
<meta name="citation_author" content="Thomas G. Platt" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author_institution" content="Department of Microbiology and Molecular Genetics, University of Pittsburgh School of Medicine, Pennsylvania 15219" />
<meta name="citation_author" content="Clay Fuqua" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_author" content="Vaughn S. Cooper" />
<meta name="citation_author_institution" content="Microbiology Graduate Program, University of New Hampshire, Durham, New Hampshire 03824" />
<meta name="citation_author_institution" content="Department of Microbiology and Molecular Genetics, University of Pittsburgh School of Medicine, Pennsylvania 15219" />
<meta name="citation_author" content="Michael Lynch" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana 47405" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=Y. Brandvain;citation_author=S. I. Wright;citation_title=The limits of natural selection in a nonequilibrium world.;citation_pages=201-210;citation_volume=32;citation_year=2016;citation_pmid=26874998;citation_doi=10.1016/j.tig.2016.01.004" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=C. D. Campbell;citation_author=E. E. Eichler;citation_title=Properties and rates of germline mutations in humans.;citation_pages=575-584;citation_volume=29;citation_year=2013;citation_pmid=23684843;citation_doi=10.1016/j.tig.2013.04.005" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. Charlesworth;citation_title=Fundamental concepts in genetics: effective population size and patterns of molecular evolution and variation.;citation_pages=195-205;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19204717;citation_doi=10.1038/nrg2526" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. F. Conrad;citation_author=J. E. Keebler;citation_author=M. A. DePristo;citation_author=S. J. Lindsay;citation_author=Y. Zhang;citation_title=Variation in genome-wide mutation rates within and between human families.;citation_pages=712-714;citation_volume=43;citation_year=2011;citation_issue=7;citation_pmid=21666693;citation_doi=10.1038/ng.862" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. R. Denver;citation_author=K. Morris;citation_author=M. Lynch;citation_author=W. K. Thomas;citation_title=High mutation rate and predominance of insertions in the Caenorhabditis elegans nuclear genome.;citation_pages=679-682;citation_volume=430;citation_year=2004;citation_issue=7000;citation_pmid=15295601;citation_doi=10.1038/nature02697" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. R. Denver;citation_author=S. Feinberg;citation_author=S. Estes;citation_author=W. K. Thomas;citation_author=M. Lynch;citation_title=Mutation rates, spectra and hotspots in mismatch repair-deficient Caenorhabditis elegans.;citation_volume=170;citation_year=2005;citation_pmid=15716493;citation_doi=10.1534/genetics.104.038521" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. R. Denver;citation_author=P. C. Dolan;citation_author=L. J. Wilhelm;citation_author=W. Sung;citation_author=J. I. Lucas-Lledo;citation_title=A genome-wide view of Caenorhabditis elegans base-substitution mutation processes.;citation_volume=106;citation_year=2009;citation_pmid=19805298;citation_doi=10.1073/pnas.0904895106" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. R. Doroghazi;citation_author=D. H. Buckley;citation_title=Intraspecies comparison of Streptomyces pratensis genomes reveals high levels of recombination and gene conservation between strains of disparate geographic origin.;citation_pages=970;citation_volume=15;citation_year=2014;citation_pmid=25399205;citation_doi=10.1186/1471-2164-15-970" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. W. Drake;citation_title=A constant rate of spontaneous mutation in DNA-based microbes.;citation_pages=7160-7164;citation_volume=88;citation_year=1991;citation_issue=16;citation_pmid=1831267;citation_doi=10.1073/pnas.88.16.7160" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. W. Drake;citation_author=B. Charlesworth;citation_author=D. Charlesworth;citation_author=J. F. Crow;citation_title=Rates of Spontaneous Mutation;citation_pages=1667-1686;citation_volume=148;citation_year=1998;citation_issue=4;citation_pmid=9560386" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Eyre-Walker;citation_author=P. D. Keightley;citation_title=The distribution of fitness effects of new mutations.;citation_pages=610-618;citation_volume=8;citation_year=2007;citation_issue=8;citation_pmid=17637733;citation_doi=10.1038/nrg2146" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Microbiol.;citation_author=E. J. Feil;citation_author=B. G. Spratt;citation_title=Recombination and the population structures of bacterial pathogens.;citation_pages=561-590;citation_volume=55;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=Y. X. Fu;citation_title=Statistical properties of segregating sites.;citation_pages=172-197;citation_volume=48;citation_year=1995;citation_issue=2;citation_pmid=7482370;citation_doi=10.1006/tpbi.1995.1025" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. X. Fu;citation_author=W. H. Li;citation_title=Statistical tests of neutrality of mutations.;citation_pages=693-709;citation_volume=133;citation_year=1993;citation_issue=3;citation_pmid=8454210" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=T. Garland;citation_author=A. W. Dickerman;citation_author=C. M. Janis;citation_author=J. A. Jones;citation_title=Phylogenetic analysis of covariance by computer-simulation.;citation_volume=42;citation_year=1993;citation_doi=10.1093/sysbio/42.3.265" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. L. Halligan;citation_author=P. D. Keightley;citation_title=Ubiquitous selective constraints in the Drosophila genome revealed by a genome-wide interspecies comparison.;citation_volume=16;citation_year=2006;citation_pmid=16751341;citation_doi=10.1101/gr.5022906" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=D. L. Halligan;citation_author=A. Eyre-Walker;citation_author=P. Andolfatto;citation_author=P. D. Keightley;citation_title=Patterns of Evolutionary Constraints in Intronic and Intergenic DNA of Drosophila;citation_pages=273-279;citation_volume=14;citation_year=2004;citation_issue=2;citation_pmid=14762063;citation_doi=10.1101/gr.1329204" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Haudry;citation_author=A. E. Platts;citation_author=E. Vello;citation_author=D. R. Hoen;citation_author=M. Leclercq;citation_title=An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions.;citation_pages=891-898;citation_volume=45;citation_year=2013;citation_pmid=23817568;citation_doi=10.1038/ng.2684" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Johnson;citation_author=N. H. Barton;citation_title=The Effect of Deleterious Alleles on Adaptation in Asexual Populations;citation_pages=395-411;citation_volume=162;citation_year=2002;citation_issue=1;citation_pmid=12242249" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. Karasov;citation_author=P. W. Messer;citation_author=D. A. Petrov;citation_title=Evidence that adaptation in Drosophila is not limited by mutation at single sites.;citation_pages=e1000924-e1000924;citation_volume=6;citation_year=2010;citation_issue=6;citation_pmid=20585551;citation_doi=10.1371/journal.pgen.1000924" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. T. Kibota;citation_author=M. Lynch;citation_title=Estimate of the genomic mutation rate deleterious to overall fitness in E. coli.;citation_pages=694-696;citation_volume=381;citation_year=1996;citation_issue=6584;citation_pmid=8649513;citation_doi=10.1038/381694a0" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=M. Kimura;citation_title=On the evolutionary adjustment of spontaneous mutation rates.;citation_pages=23-24;citation_volume=9;citation_year=1967;citation_doi=10.1017/S0016672300010284" />
<meta name="citation_reference" content="citation_title=The Neutral Theory of Molecular Evolution;citation_author=M. Kimura;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Kong;citation_author=M. L. Frigge;citation_author=G. Masson;citation_author=S. Besenbacher;citation_author=P. Sulem;citation_title=Rate of de novo mutations and the importance of father's age to disease risk.;citation_pages=471-475;citation_volume=488;citation_year=2012;citation_issue=7412;citation_pmid=22914163;citation_doi=10.1038/nature11396" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Perspectives in Biology;citation_journal_abbrev=Cold Spring Harb. Perspect. Biol.;citation_author=H. E. Krokan;citation_author=M. Bjoras;citation_title=Base Excision Repair;citation_pages=a012583-a012583;citation_volume=5;citation_year=2013;citation_issue=4;citation_pmid=23545420;citation_doi=10.1101/cshperspect.a012583" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Symp. Quant. Biol.;citation_author=T. A. Kunkel;citation_title=Evolving views of DNA replication (in)fidelity.;citation_volume=74;citation_year=2009;citation_pmid=19903750;citation_doi=10.1101/sqb.2009.74.027" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lassalle;citation_author=S. Perian;citation_author=T. Bataillon;citation_author=X. Nesme;citation_author=L. Duret;citation_title=GC-Content evolution in bacterial genomes: the biased gene conversion hypothesis expands.;citation_pages=e1004941;citation_volume=11;citation_year=2015;citation_pmid=25659072;citation_doi=10.1371/journal.pgen.1004941" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Lee;citation_author=E. Popodi;citation_author=H. Tang;citation_author=P. L. Foster;citation_title=Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing.;citation_pages=e2774-e2783;citation_volume=109;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=E. G. Leigh;citation_title=Natural selection and mutability.;citation_pages=301-305;citation_volume=104;citation_year=1970;citation_doi=10.1086/282663" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=K. J. Lipinski;citation_author=J. C. Farslow;citation_author=K. A. Fitzpatrick;citation_author=M. Lynch;citation_author=V. Katju;citation_title=High spontaneous rate of gene duplication in Caenorhabditis elegans.;citation_pages=306-310;citation_volume=21;citation_year=2011;citation_issue=4;citation_pmid=21295484;citation_doi=10.1016/j.cub.2011.01.026" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. Loh;citation_author=J. J. Salk;citation_author=L. A. Loeb;citation_title=Optimization of DNA polymerase mutation rates during bacterial evolution.;citation_volume=107;citation_year=2010;citation_pmid=20080608;citation_doi=10.1073/pnas.0912451107" />
<meta name="citation_reference" content="citation_title=The Origins of Genome Architecture;citation_author=M. Lynch;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Lynch;citation_title=The cellular, developmental and population-genetic determinants of mutation-rate evolution.;citation_volume=180;citation_year=2008;citation_pmid=18757919;citation_doi=10.1534/genetics.108.090456" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. Lynch;citation_title=Evolution of the mutation rate.;citation_pages=345-352;citation_volume=26;citation_year=2010;citation_issue=8;citation_pmid=20594608;citation_doi=10.1016/j.tig.2010.05.003" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=M. Lynch;citation_title=The lower bound to the evolution of mutation rates.;citation_volume=3;citation_year=2011;citation_pmid=21821597;citation_doi=10.1093/gbe/evr066" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Lynch;citation_author=G. K. Marinov;citation_title=The bioenergetic costs of a gene.;citation_volume=112;citation_year=2015;citation_pmid=26575626;citation_doi=10.1073/pnas.1514974112" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=M. Lynch;citation_author=J. Blanchard;citation_author=D. Houle;citation_author=T. Kibota;citation_author=S. Schultz;citation_title=Spontaneous deleterious mutation.;citation_pages=645-663;citation_volume=53;citation_year=1999;citation_doi=10.2307/2640707" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Lynch;citation_author=W. Sung;citation_author=K. Morris;citation_author=N. Coffey;citation_author=C. R. Landry;citation_title=A genome-wide view of the spectrum of spontaneous mutations in yeast.;citation_volume=105;citation_year=2008;citation_pmid=18583475;citation_doi=10.1073/pnas.0803466105" />
<meta name="citation_reference" content="citation_journal_title=Cell research;citation_journal_abbrev=Cell Res;citation_author=S. D. McCulloch;citation_author=T. A. Kunkel;citation_title=The fidelity of DNA synthesis by eukaryotic replicative and translesion synthesis polymerases.;citation_pages=148-161;citation_volume=18;citation_year=2008;citation_issue=1;citation_pmid=18166979;citation_doi=10.1038/cr.2008.4" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=A. Mira;citation_author=H. Ochman;citation_author=N. A. Moran;citation_title=Deletional bias and the evolution of bacterial genomes.;citation_pages=589-596;citation_volume=17;citation_year=2001;citation_issue=10;citation_pmid=11585665;citation_doi=10.1016/S0168-9525(01)02447-7" />
<meta name="citation_reference" content="citation_journal_abbrev=J Nucleic Acids;citation_author=R. Morita;citation_author=S. Nakane;citation_author=A. Shimada;citation_author=M. Inoue;citation_author=H. Iino;citation_title=Molecular mechanisms of the whole DNA repair system: a comparison of bacterial and eukaryotic systems.;citation_pages=179594-179594;citation_volume=2010;citation_year=2010;citation_pmid=20981145" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. W. Ness;citation_author=A. D. Morgan;citation_author=N. Colegrave;citation_author=P. D. Keightley;citation_title=Estimate of the spontaneous mutation rate in Chlamydomonas reinhardtii.;citation_volume=192;citation_year=2012;citation_pmid=23051642;citation_doi=10.1534/genetics.112.145078" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. W. Ness;citation_author=S. A. Kraemer;citation_author=N. Colegrave;citation_author=P. D. Keightley;citation_title=Direct estimate of the spontaneous mutation rate uncovers the effects of drift and recombination in the Chlamydomonas reinhardtii plastid genome.;citation_pages=800-808;citation_volume=33;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. J. O’Roak;citation_author=P. Deriziotis;citation_author=C. Lee;citation_author=L. Vives;citation_author=J. J. Schwartz;citation_title=Exome sequencing in sporadic autism spectrum disorders identifies severe de novo mutations.;citation_pages=585-589;citation_volume=43;citation_year=2011;citation_issue=6;citation_pmid=21572417;citation_doi=10.1038/ng.835" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. J. O’Roak;citation_author=L. Vives;citation_author=S. Girirajan;citation_author=E. Karakoc;citation_author=N. Krumm;citation_title=Sporadic autism exomes reveal a highly interconnected protein network of de novo mutations.;citation_pages=246-250;citation_volume=485;citation_year=2012;citation_issue=7397;citation_pmid=22495309;citation_doi=10.1038/nature10989" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. A. Orr;citation_title=The Rate of Adaptation in Asexuals;citation_pages=961-968;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835413" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Ossowski;citation_author=K. Schneeberger;citation_author=J. I. Lucas-Lledo;citation_author=N. Warthmann;citation_author=R. M. Clark;citation_title=The Rate and Molecular Spectrum of Spontaneous Mutations in Arabidopsis thaliana;citation_pages=92-94;citation_volume=327;citation_year=2010;citation_issue=5961;citation_pmid=20044577;citation_doi=10.1126/science.1180677" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=A. Sancar;citation_author=L. A. Lindsey-Boltz;citation_author=K. Unsal-Kacmaz;citation_author=S. Linn;citation_title=Molecular mechanisms of mammalian DNA repair and the DNA damage checkpoints.;citation_pages=39-85;citation_volume=73;citation_year=2004;citation_pmid=15189136;citation_doi=10.1146/annurev.biochem.73.011303.073723" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. R. Schrider;citation_author=D. Houle;citation_author=M. Lynch;citation_author=M. W. Hahn;citation_title=Rates and genomic consequences of spontaneous mutational events in Drosophila melanogaster.;citation_volume=194;citation_year=2013;citation_pmid=23733788;citation_doi=10.1534/genetics.113.151670" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Siepel;citation_author=G. Bejerano;citation_author=J. S. Pedersen;citation_author=A. S. Hinrichs;citation_author=M. Hou;citation_title=Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes.;citation_volume=15;citation_year=2005;citation_pmid=16024819;citation_doi=10.1101/gr.3715005" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=P. Sniegowski;citation_author=Y. Raynes;citation_title=Mutation rates: how low can you go?;citation_pages=R147-R149;citation_volume=23;citation_year=2013;citation_pmid=23428322;citation_doi=10.1016/j.cub.2013.01.018" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=P. D. Sniegowski;citation_author=P. J. Gerrish;citation_author=T. Johnson;citation_author=A. Shaver;citation_title=The evolution of mutation rates: separating causes from consequences.;citation_pages=1057-1066;citation_volume=22;citation_year=2000;citation_issue=12;citation_pmid=11084621;citation_doi=10.1002/1521-1878(200012)22:12&lt;1057::AID-BIES3&gt;3.0.CO;2-W" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Sung;citation_author=M. S. Ackerman;citation_author=S. F. Miller;citation_author=T. G. Doak;citation_author=M. Lynch;citation_title=Drift-barrier hypothesis and mutation-rate evolution.;citation_volume=109;citation_year=2012;citation_pmid=23077252;citation_doi=10.1073/pnas.1216223109" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Sung;citation_author=A. E. Tucker;citation_author=T. G. Doak;citation_author=E. Choi;citation_author=W. K. Thomas;citation_title=Extraordinary genome stability in the ciliate Paramecium tetraurelia.;citation_volume=109;citation_year=2012;citation_pmid=23129619;citation_doi=10.1073/pnas.1210663109" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=W. Sung;citation_author=M. S. Ackerman;citation_author=J. F. Gout;citation_author=S. F. Miller;citation_author=E. Williams;citation_title=Asymmetric context-dependent mutation patterns revealed through mutation-accumulation experiments.;citation_volume=32;citation_year=2015;citation_pmid=25750180;citation_doi=10.1093/molbev/msv055" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_title=A global reference for human genetic variation.;citation_pages=68-74;citation_volume=526;citation_year=2015;citation_pmid=26432245;citation_doi=10.1038/nature15393" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Uchimura;citation_author=M. Higuchi;citation_author=Y. Minakuchi;citation_author=M. Ohno;citation_author=A. Toyoda;citation_title=Germline mutation rates and the long-term phenotypic effects of mutation accumulation in wild-type laboratory mice and mutator mice.;citation_volume=25;citation_year=2015;citation_pmid=26129709;citation_doi=10.1101/gr.186148.114" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=S. Vieira-Silva;citation_author=M. Touchon;citation_author=E. P. Rocha;citation_title=No evidence for elemental-based streamlining of prokaryotic genomes.;citation_pages=319-320;citation_volume=25;citation_year=2010;citation_issue=6;citation_pmid=20381897;citation_doi=10.1016/j.tree.2010.03.001" />
<meta name="citation_reference" content="citation_journal_title=BMC Proc.;citation_author=H. Wang;citation_author=X. Zhu;citation_title=De novo mutations discovered in 8 Mexican American families through whole genome sequencing.;citation_pages=S24;citation_volume=8;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. D. Whitney;citation_author=T. Garland;citation_title=Did genetic drift drive increases in genome complexity?;citation_pages=e1001080;citation_volume=6;citation_year=2010;citation_pmid=20865118;citation_doi=10.1371/journal.pgen.1001080" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Yang;citation_author=L. Wang;citation_author=J. Huang;citation_author=X. Zhang;citation_author=Y. Yuan;citation_title=Parent-progeny sequencing indicates higher mutation rates in heterozygotes.;citation_pages=463-467;citation_volume=523;citation_year=2015;citation_pmid=26176923;citation_doi=10.1038/nature14649" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. O. Zhu;citation_author=M. L. Siegal;citation_author=D. W. Hall;citation_author=D. A. Petrov;citation_title=Precise estimates of mutation rate and spectrum in yeast.;citation_pages=e2310-e2318;citation_volume=111;citation_year=2014" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Evolution of the Insertion-Deletion Mutation Rate Across the Tree of Life" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="twitter:description" content="Mutations are the ultimate source of variation used for evolutionary adaptation, while also being predominantly deleterious and a source of genetic disorders. Understanding the rate of insertion-deletion mutations (indels) is essential to understanding evolutionary processes, especially in coding regions, where such mutations can disrupt production of essential proteins. Using direct estimates of indel rates from 14 phylogenetically diverse eukaryotic and bacterial species, along with measures of standing variation in such species, we obtain results that imply an inverse relationship of mutation rate and effective population size. These results, which corroborate earlier observations on the base-substitution mutation rate, appear most compatible with the hypothesis that natural selection reduces mutation rates per effective genome to the point at which the power of random genetic drift (approximated by the inverse of effective population size) becomes overwhelming. Given the substantial differences in DNA metabolism pathways that give rise to these two types of mutations, this consistency of results raises the possibility that refinement of other molecular and cellular traits may be inversely related to species-specific levels of random genetic drift." />
<meta name="og-title" property="og:title" content="Evolution of the Insertion-Deletion Mutation Rate Across the Tree of Life" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/8/2583" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Mutations are the ultimate source of variation used for evolutionary adaptation, while also being predominantly deleterious and a source of genetic disorders. Understanding the rate of insertion-deletion mutations (indels) is essential to understanding evolutionary processes, especially in coding regions, where such mutations can disrupt production of essential proteins. Using direct estimates of indel rates from 14 phylogenetically diverse eukaryotic and bacterial species, along with measures of standing variation in such species, we obtain results that imply an inverse relationship of mutation rate and effective population size. These results, which corroborate earlier observations on the base-substitution mutation rate, appear most compatible with the hypothesis that natural selection reduces mutation rates per effective genome to the point at which the power of random genetic drift (approximated by the inverse of effective population size) becomes overwhelming. Given the substantial differences in DNA metabolism pathways that give rise to these two types of mutations, this consistency of results raises the possibility that refinement of other molecular and cellular traits may be inversely related to species-specific levels of random genetic drift." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
    <title>Evolution of the Insertion-Deletion Mutation Rate Across the Tree of Life | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"M6c8-xl7J0qBOxC6kSNg-SPV6y5YZklI9sBChDFMON0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460440","apath":"\/ggg\/6\/8\/2583.atom","pisa":"ggg;6\/8\/2583","ac":{"\/ggg\/6\/8\/2583.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/8\/2583.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2583"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2583"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/8\/2583.full":true,"\/content\/6\/8\/2583":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+Zlt4fF4T2S1EVp1S79e4LLMEQIKEYS8ECc8\\\\\\\/YFwoIMn6Wy7Xu3sAypUOM7jwyMYsqNhgblgfYnnaPy9nuc\\\\\\\/KLvm5612\\\\\\\/IqWfrAHEYGZ52OzuzjzCXmjyJkplWtzHDvuvMDjh+7\\\\\\\/Yt0DWb\\\\\\\/skVy2wjLhGVg\\\\\\\/cFGYJP03xDJrxXZ1Vlfml9Julv+FmkqddGuIg8adBALft+gkbg8kAmInLy5GOoZ7CDzE7y03BgPKB4QC\\\\\\\/DybYdTpHiwDgQYBm2OikLb1BWvc6d40iDEt\\\\\\\/IXkDXDjKE1Ojrhnr1Cd1EuVhuXzhVruna2Q8YAKdVqu\\\\\\\/YfH1+OWxbTGalQyQOyrhCYH7g9fT61QdEzzsT1TpsBhhETi74QCzXcS2rAhiSQ0PLbdeOCztXwfzBo+F527ZATxQJ1LIThRFHUO8P44SxSCdY23JqNQuXLS\\\\\\\/9FhihTCpKV3pwAfuAJuK3pjtqHTB6rTyOr1cQwbXqtaE\\\\\\\/O8rOus8cMi00M0ZudXwVgfZox\\\\\\\/NzdlVLv5pmXariCh+HAUHtshjlwOHB1hVHc3rQDCrcJXm4zmIHlRdVkNrc+5nug2JpLTn4a4wSps6aynriKFEB3gKUpiMZLrrryjCewyQJ6JEo50SMkaZsnLCNIf8mQGov0VpSKOHLO7SBcqWHuC4SOQnkVgRIsTGQoDWFt3J9n7MeIGGlP3FWAKRF\\\\\\\/IfLvmcSn3c5Qe6Ci3PDEf83rsI2znbciF5NTeONSyXFp2QjoFpHKsjPd4ewwEbjVoynbNM65cM+Z5aKLMZ+Fe+tBKps91Whp5eCyk0t4XbBmyTjifgU5u\\\\\\\/13LRA8716ipwh1lmDWXpnJGmljRnKrCDne7JXJhDTnEJlJ872GoYzwCYlOyZcGMbHgJrm\\\\\\\/xWdnyf0hFGpaFsQ+wCVFKVZLegic6Rcdo9dfCGk895y1ZTDs2RcFIq6cZ2yVTHPz6\\\\\\\/9GcF\\\\\\\/iBfPiUOyuCItPLr9H7gPIbaQB3+X26fpRMciLLWRFDPp7fe\\\\\\\/XBQDN4LKKbEc5BK381BLbhJpH9IIcblGI5aaZwhJAWesGxcKL9rPMkoDh82WYl79ZnI6daLDa4cGmc2iJFyqFrRisquwD8HqhQryQAi\\\\\\\/wJcX7WgteLlDKGhibWDo7hHGGbrOH3fke8Be2CgAm9vCgXimUfcOh\\\\\\\/t2QbXu0d1\\\\\\\/zp0ER5p\\\\\\\/8NGF92MRpgBejRPYHUN\\\\\\\/JQymvi0MukwzdXcy2FgAYcMixjEmhuwCJO3BwAbn37EOaCVp6BCn9FO307hhMENz+N39ACljxSyyMKiWEN\\\u0022,\\\u0022iv\\\u0022:\\\u0022LcXo804mLpmyQ2kmyUXrbQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e30f3c52e331854c8bea0ee78c244ab84ad8701fbfe4bd4d3e6365097320819a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uuQLwhfgzxKFqsjUlMZz98rx\\\\\\\/K0HjkikqK\\\\\\\/FiE7Fyw54bOjqOmxbf4d7JsWE5C5n0MBirl+6xYzI426EmlCZj5OjvUvBSF8N8eker4KWSO66gBxp7dqpx5x3uvU02qC66BumzhSCytzGpDMnisG+EEjydg6bl5q735QfzCI9UqGwX6slJEVoDx+GZQTKyLyQw0sdGCQBnle3hTYWrEikVK8SOq5vyt11Qo1o5WGDBkjMJURfIS4tjQXPSLK24FpHdWDBFPsL7\\\\\\\/oU\\\\\\\/5g+LnKnSWksV9wB\\\\\\\/kRm4Cj201WGDREVIiW33jCsji7evuItDTgOBp\\\\\\\/UJ6xBBv1WDaV23qkYBrKQQ1WdMrlT7Iz402rXycmYGXoangIg+aoTgwNYQbbDtDnJD5Aob4q9E2OGrs4kYYGTeOPXfwh62dLUPsh0V5zz91sFoliH3+oNSEtHOg\\\\\\\/er7lEqEB0Q8uoRTD9s0uT+af6X9KVUWOiFU5QsHVNgTrvK7w6Mso0J3+w0iqRTFpIopAtzpe8pHyCy8Gy0mRhXWShP9FPAUSec8YLgwjkR5D+0elx3xJnrLkbFPS9bHH8suaaSo+n+q3Uhg1ofVJ\\\\\\\/KZAAHJ3vSeVcdR6ZI0A1AzzcI8N6Nj0o5QNOg2PCByolskOs5BpFdPVoqAW\\\\\\\/WX0axqh9jyZeI8RMOVaZdbrOOG7sohVITyVhsvWi0ZFTa0KwO5d+QNEWFDiPfalsEF1\\\\\\\/0AxkzyqOa5Y6V1QUG8UI2VM2mx9UppyfsHpC9iCsXyQC18i69SLf+WnnMIyiRbsC9UQRtX4d2I0R\\\\\\\/4szJedBozpMXSZxH\\\\\\\/IMniQ80tjcFLIBXFgtMM58JM1C0jWD2WVLjKE+uVeaBAs1\\\\\\\/+De4HQYQ5UyF3QRMvn3UrudsaYc6xeyCj9ueH8ESkAQaMEQIFgRBlEE3wJE\\\\\\\/e6vQCK3ThYaf5mbIMHXMQvIZijcG6SccV6axHHh3qpdwk7\\\\\\\/9zGVckz4LebpgT1UCMW+qjr5FWdTKJzgCrR8bhPEr9bCij0ISkRtYRyEq4QDdT7ybI72lf0\\\\\\\/7vBXmzFrtqNuzRQh3C35mpXlT6NEzBDjlDEcPSVzIx+wksuyorBwW5xfSsVj7a\\\\\\\/qAWSET32YAz9WpZ4FOt9PTsCrvIVGYjshcx2aqmvhKwFj5xFrGFps15HRBD8H3OYPnETMLWq4GlpF47RNIeQ6EtIjtYUsVfrfbtvizmO\\\\\\\/q9AplNDIoAJjVCMp1YjSBWQImvoLfJnkEC5LVk9vuoAjb6uZzfCE9s+GNkq5k2+5xc0gvRhQyXvDMtv3Z+V\\\\\\\/E4+Ba9+WSKAAS6sSQcRozyZ9TKOuZMeMAkXag\\\\\\\/tl7vjUc4B5MXr6xrrj7gcXYLPZ45sBiIOp45l3JaeCGEK8NDmZjfbOVNBsrSYOhhUmypWJJ09\\\\\\\/aUWUz+sStV8L5D6tWwN5juxOVk20zpt9aVWxT69n3AwXj75vUb2hqQIRvKjTkVDVGH8P34Bw9GCLfKjQdLfQL6\\\\\\\/gdQlRWwl0OIAIoY6a4yGl0IeXJVUeSvd46wXENioCbEs4s\\\\\\\/I9EpH4o+jgBgrym2u5Sp+h\\\\\\\/yx7YCUkHnQ+av\\\\\\\/G9qSf0E2RrDpgoFD7CgiRBDxutt3JL01ljty\\\\\\\/Zk5XhSKD3f3Rpv1928UigXVkfv\\\\\\\/nWyS+xC\\\\\\\/OsZJMdU51gEGPsg5HZ\\\\\\\/h0xniVUDlaSMVekIwpVRbqMfEOMhB\\\\\\\/3HJhngPwyNXjYmqOrWptwlyM46dsBo\\\\\\\/wc4VSZa\\\\\\\/zx4paWgRQziImfm1bQt3WlZHSOI2atBuztNajtAgUDBzb3J57+lhLpx9x0bhDSAbxFOI3pmIMoSWk3hP8zwxSgOEgoHLGNqBckYeQ\\\\\\\/RVBUMpHOkVO2Ro0LhJf8f6XjQ4v+Q0XUcM5hBHA6m9u4leLrBAfKzmDbtphfp0B9BWsJc6Id46VVPXx6izZkWI0OzuBMnj0K1Zoliblz4hBZQlKa9PbybNAV+3rf\\\\\\\/xNvwUEyjBSSozWn89de31SXN2d6HiRBzzmmRauhFXGxo5K8EPq0SwMlKmKz83OU3\\\\\\\/MHuSs4MjyruKRSkV\\\\\\\/GVh3X7rDLImZ0Df3k6wvVwp9\\\\\\\/4cxf6rOQ6eXqtLKbKQs1mals9F8j2MIU1BTU3f9GJyOV3864APMqfz9zUUWPkCqHj34pMN9uYq8L5MkJO8L0fAxjhInpYpeyaFzAIfbOhSiNcJmujsTf8HwZTG+Mw4wKfAMu3Yt7jqARXo1e+w\\\\\\\/sNpJ35zwldVfXrf\\\\\\\/\\\\\\\/RtniFAxv6azMk4OCTBHYe6BhBzjRjrwRsTiYf\\\\\\\/hO00wQxDEYFpf6w9Zfzx1EBGhmPRB3ND3HGy9ilFnmRP9Qw64i3aVbIE0sBVlhomB+q3mlo8fKhIY52StMuSZmqhRbsjyU4nIA0tAJPU9aQw81CmCCxxsr\\\\\\\/IiFAZCNNUjZyPNcTobjsneJpEQzWEPXDuhoBaZUwedeV\\\\\\\/vX\\\\\\\/ieFCpxZuNSJO32KgnozIGotmiXn9fEfquLQfz9L+kRlxgzGXtsutCHBO6SrYbJJ76kilugfDxl7fOt919wd3ddhaoIJrYyx+WaTitmHRt229VuQ7Iw1VIMYhPRPAdRRfXK0qglruZwafYRbehIsw3j+S2pQmoSJ6lyQqSAa+y0kqIfqHiX1nq8IVEulfXCK5Fxl8zFgWx8mT3e4M+Ipxj2ezqev8WbTqH\\\\\\\/XO8ipqZmV0ja2F9w0JZOp1FTq6d4GdqCjb4OrxSeBUT4wV7AGOVy7xWz1ovI8RxqEonTY6RryAgofeaW1wLX2Bqu+hK0jWGwdDkwTXe1dy+SdN3NYVJHeIE\\\\\\\/WJev8OcwRFJvC23HZgkn3VU3pVRTCSaadN+WcHK+ux02tsI5pFymEw1h66VOrJOMd+GKtXc0TJ9RkR8e2o0JN4n4azVho2KcfRqkACGJgybjfSxHBdajKwuhoz5f3eS7EP1zcPQ\\\\\\\/+G9f0mRjozpI2J0oWq+3wBwFdbw0NHUuFxadcSvNEXRfnTXUNkaOJ3F3if8Q8GRqr3o8azE021CdF6qpLUMahz6FVKyhuP6dAIctZe\\\\\\\/4WEZ5wVkyQ\\\\\\\/58OMDojUmHzHLRPzZmlmYRyPm2u\\\\\\\/qKRsgBxtG2iRRjianfdUiCj86uefRTXqOIoJPkjyXuPxoD3iEnmHa7JlhiUcy3FwXlaewTRuesxeUdNusIwXyx96RHqoWgzNjk1ZakfQ43OOuEWcfSB9e8G3HJa11LWFietcg+Emplg9uUSonifbXyAa0OhurCEpqRUTtBPOUt\\\\\\\/jQ\\\\\\\/Jhgc7Cq3Q7jIw5tXXbCJiafYR08NeK+dKm9PJtKmH9IdugtzSRpaPXfsT6QefTeXBFCQJEM03QbTYXxMF6\\\\\\\/C\\\\\\\/Ehrt954rGOK7G\\\\\\\/MT8A6jJW9FpmaFGp+gvlkacFnass3WBTPGmkCsjSS5PvZ3P3rwC9UYS9rnxEiADD53IYT+IXNVmPhk6NHrx5V4Kd4myscBu8hMKXNkX75UyhgGi82d8lQm0nWgqfAkLPx3hFlafYWIMI3rpDe9MZG0kd2zv2KQq6jQov2L6ReV187Dfl2n5ayUpkI\\\\\\\/7TajN2Qi4jlhiS+HdiJYAnxtyVfxZMtXQdjCPdZUeqFucVEWkAcSqm4r3vPdj\\\\\\\/qjM2VROkB7HwiV49D7jEuKnfRHHObjhYWukfF0edb0bsmiP95r4bVQuKyHixDTNNSuHH8IsnI0wioc91ZwU9BJSi++gKDKfdBtwJRoHBO5pC471lJ0BU6kuZeY1JpglIb2YQbrDtR9s4owZWU2fMczKjGgJgFbXeas9NXiwKiMugI8xut\\\\\\\/uaSjmlfnGrheEkzwMNYqWqGIvOFfysBTIjKPQ9ouSMz3mTQD4r5I4rZ9PAQukeVw6tEY0Sms+yTE4z+wrKxjA3aT2yyT6rOrDltugZW3+EL+HVA3IU76GtTTgo58qPAmNOHEJuxW8kdlr6GcXX\\\\\\\/42xexRX2NmjuDu+KwPWDq9B8IWg+NFNJYLg+u4T8XLnvhqnkaZz5o6QR2U4bvfBPWYm2t3cKI79cILhjxUTWnBrizsDD6Nn7pUYrvrx4Bm0YBZM5iiidYzEcpjIjqrlRt8sh3IwRdeO3dEGOBkMCLtIHgp7K1jJOXjgR\\\\\\\/ZNXFm99AI3ktOZeUZRpyDVyG23lO1yAgCHMjvupE2HlcxL8kuuS0uA2rWsdMi+ELqaq+VzG2ZVq3VK22JaYQTANqbUayAH+IUdnby+7At6VV99ByroS5DSXXZ99Yizc3lUnOcEyKFpQEgwl74rbWuxSdaTcFk\\\\\\\/24qZ+Jlb4c1uJNRdgJCmaqRV0D\\\\\\\/VfxqRDTmncTSgtzTQG1AQ4mFIIPUOaBTotkguGSQ6lCzhdTQ+UAAat8nyMsUN6iH3VbVNLN+6TiEt9OhNZKmvUu8mpwRokkiL+wGYI4iAJ5niJ+xEDB2oWZNxm0HKd9bcAuS9T0fejElN9FO6OZCgg1xKovQRW8mSHM7sF8x0VOL\\\\\\\/pr8vcCZEl\\\\\\\/RnCQpdY1k70\\\\\\\/9HVfSFNDc9SVToa0uPuAsg+HC5F+JkAbArzVXgLf8m\\\\\\\/WUeecNmdrflAc4EHbM7TD4j4YyoJXEdmWUfLDc\\\\\\\/Gm1\\\\\\\/w55QmgTUA0nNsXjgE1leTFrQ0PXIEEJ\\\\\\\/hiFdr7iGvcT6UqteC36rBlVYV\\\\\\\/2Qz21+7v0CMUglySDQkgW+RI20bIEDPKknlhadeVjQku09\\\\\\\/7TYCUL5lgEmpER7nc3UpzQ5G9Ca1o4Bw\\\\\\\/9tD3d1z\\\\\\\/eSGpISagUODTXMrh5pGch3uGG5gGxWK5igsr9H9TczDAgHQUqFkX\\\\\\\/D1vStSrH91VwfNtpjS4OSLH3rEmdt3VImyzO5xqaoFhz\\\\\\\/XH+qthFUnUikX\\\\\\\/mIfjJrmp4jeaLnG2NF840BlOw1BzlFV87isovksGEc1qlwL9D4iW+iHrviuTMmL3pEMOnWXqWzuU4coxGv\\\\\\\/prabSXkkm+adYL71XWW3Ae\\\\\\\/+KE6FGMS+32aRWfOCpUJqBkXZQrLLUeArRSDCHXdHh\\\\\\\/14UxNXnMLao4pSwu9rcPjJ1pbgnM17eMw5av5TG33UeHcJmgDQqg7FJLPJwtXFKJDj7m0jYouxi\\\\\\\/bvHZW8+6684UdHDfn4P83QB6i2vDDfCsa\\\\\\\/M0jxL+D8BOZfN7r6v3jKfI7BgoBb3gQ9lK2qgVscphI1frw\\\\\\\/LLRUMzSCXFcORSdRIGQZ112+HugwfSxaKwIp9APIxTwVGQTyYpw6JT4mwoQlpi6r2BjzYaaJPd86sxvMzz5waIb+9uvolLERUraV5RdhTlc39gae3iTIet\\\\\\\/3kBxdrScu02aQr1eqwVjmPXu8MMLrcWucECTr6s+QTwfV3uVx2hWvEz4\\\\\\\/PD4uzEXfKgzR3cMA0YAtUgepR8MZpoXN1HlV8NfWg0V2CfIHTf7XLtZepeuF+RUd5snwPKSv9jn68X2AZ+8ddBqB8lSP\\\\\\\/YBuqXMnlyfTTqeIy04fvb\\\\\\\/aiufFvBFF8aP9GQMJglPftrdXAPNLIVZbFDCI5b7OQpGPrh\\\\\\\/6RpUaSzXFk+bdRpXlPl4DMeMqyASXaRnTYlU0TZdsdRuilG6kki8Rpv2rx8NXP3NN0HyPm4X4KtaplFxyW823ENEpTO4wGddEzP2rtrSPrdmZaGZ1oGKet9oR+JIUaykvMN2U0Ti0OPj8erfAorz2O7sWMVg0thEDbtRtj8r+356Dou5wJeCM08UZWsJw5Wl5U7rNq0w4YO6BGUYABFsOS6VYTz9K6Pu2S7VvtEpd0xDIVEcUNFW+4Q7ivajku6iQvFRSPUmt7Blt\\\\\\\/KFPmpi90HtxdJtDMicL4fk03UEhfiKnsFPFjX0zEGGhwFC4At54Rx7YYdNaxA60LDH2PqPVT0a7T7mbigL7NBW62w5CjhQchvpxSnIwTO9wHNsFwa0YX0pD+EwXnzkYxLYEmI7QogNuJMzEEl6k8+slSslWbJhLjlHyfuBmvBk7Qz3c0nXuzBpBrjSDVb698EZAwiS6j9QJM+BzFdQRXYUuKTsc530\\\\\\\/ZJ+5KASBqx+WKRftUZQsZmkHRUcVLpKjpc\\\\\\\/vO4yVABv9y0yuNMi3U5UqOxHSUlZKMLn7AzOTKVWfaT0DByIlwfxJ4tJ7WRa\\\\\\\/uSLF\\\\\\\/olaF8xiNiK+2fqh9EnfotljrswxH2QprrqvJ4Bohm3owY\\\\\\\/Pa\\\\\\\/Y78T5Mrhv\\\\\\\/tRo4fZgN4r0l9dqGp9KqWTQjRhewJmoqSNU+NGJrFvFLiFmvXeSWOKiyMG6uNGHxsjzwzbpBKpO92WoUHcmBFPA+l0Z89\\\\\\\/2R2mR6qGSEcM4EMqDfjK8tbsmpwIVUc4NRPfFdXm8FDRPrYdY6+t\\\\\\\/NGgwEZxTfpjixS\\\\\\\/Npmdnj1tBQZzARG29letLrUnYJ0\\\\\\\/iwEktlnYGsBl0JogPciRmS1f\\\\\\\/3BW4XBFD7xbgDY\\\\\\\/vEu1PPaHX9MZ0O\\\\\\\/edgPv87rK64NrRd9pItSkmrqe0vY2v0WrB5OD75VdpxBWv94JvvzIVnjjQ37g6zFAqq0oXYia\\\\\\\/+CBia1we3YIYGm6kPeQRn4AQfna\\\\\\\/DNuvIWVyibN1WgxrSq6X8GfY\\\\\\\/4nmO0pKsXKDbKb\\\\\\\/1gb\\\\\\\/03ZtMtzufmrI6SrZhkq5s1INgq1C2I+P02URG9jniQpQMu4F1g13o7dNKbFHMTM7KLYJ9ZgqbTZ9fdevSiqjwP9D7bYGigPIszA6lWqtFjicoyn5NCkq\\\\\\\/JbeXytcNu1FqbybhQql8dVIGKT+O4ngzs1YnwUuoN4mfgZqgdHGa7LV6WjKz6Yl0rAHHu5+Xsi2raT3F64dwQaNGLaX2MJ+R2KlYIuIV2Vr\\\\\\\/UoHxx3UvmjP3Gu0vnP87m3VsKI3vJ21sdpm1\\\\\\\/OOZOdaIYUpKiu2lbrhFU4m581rDhaVRGx8m02H9SxRMtYpvBxEJ8GwuOsb\\\\\\\/t+LPZx5mgpSf7NSHFTirEh8sM69JTO1cjfmHv9qVWz6FOioHbzEE+2hF6m4loP4HSHGZdFlNzy3WrAHdJqA8idhVIlMuAs7jgnNHVwpLGVWJr\\\\\\\/n4tnCdQHsig+25UT6ClInFFLiTsRgrFC3umEWt1NKlO4jNJcD+AoNtVVffYed9gdNd3WcvTlvDvwVVIGN4bAih1r7Z8z2afzaFJyOESKZ9vaBgMf6\\\\\\\/WMJdxGwpeZFKjI+5zjLJ4m\\\\\\\/RFXvTaUK\\\\\\\/OeHePLRRQSrlP7vO4w81SYpX81PH+MKdfiFTrUGQv3dIWhpmY\\\\\\\/4\\\\\\\/ytLhL7hC3M+RJbVBwCiem4lfMcbmFlh6oXjXJJMrby0Nv3vyLHp\\\\\\\/zkEuz1fKlowzt3DCqxirYbIMmSbdLlbsSXYyfSeQHLavL2aima1xeRPGDblKPHaPrPMrS79YUC9dzLa4d1r4\\\\\\\/nNd8zJRQi1wOGwVtF9a7tM7qzsNlKut7Ba5b7GiNXj0K+6JZrxpfuuPTMPAXQqj8cWtcXEeYIOVBezlOv7DHY9\\\\\\\/Ot38psODboNSNFnY6Kjfa44aBIXHSh05i5rJgRWXjNO5DMnp\\\\\\\/CqI3o9krbN1myfN9BMRyIQ\\\\\\\/RghE2lEZPTXgBlos8wBB+OFlb4YW8pTHG2jTZH4OqjH9sWZ29zUL6QaiMjwrDgk+DorNLRZ02ivya7rYUAjy1NFVgCQJ+dBJB0Zh6iJiABZP5H+o6qmiyeusJfoxGL9H0SuzF3uz+hnRd\\\\\\\/2akSSFnj9J3OJzmZs+y6m4Rm4zqtpvP9ff3JvtkzTrKjFqCW8IZmxLECEIMChveYE8DaJSSqDLSjV32oFsQB4D2Z9gGe8gdu2gmiND4TbIF6g3k5t9dnMcZSyanU1gzwZ6IjZIs51jeBR56HLdhrzOdNOCOaRcd29hqOOJIp55LB8P5vQaHFVz2XABx6f2egVZK+ZutIaxep7zfklhXSSFq8kLRt2ySm5dCxVKfHYqkfGk+rjGebNOttMoy64MHji4eTHp+79Ua0thdatLddFamG08N0QMSk5uwxAbvFfcOee+co7\\\\\\\/SiEliecg4I0usLrJ5\\\\\\\/v9AWmzulUccJudAMIe4\\\\\\\/eVLk+z99Fs2C+4pYI9Kubh+a4ybezfwwnYhegsEHOHw+PVNVbp7iEddJub7ZMVzvocvIWm9hu7+H1dx+Za3OHtT+KZ4I7J4NBnR+KQg8r5srjIQaQ9GP7EEa2m4WFeG7HAzNZmU7O4IBfR6leP4NL5dh3vPIVPzB0dWccd9LM5vwsf\\\\\\\/LCYgqX+NU+NeaaRQaOl4QIqugK9DJAq6k+4C+Cwr0Gv1P3lJZUAjiniFkBjaEHlLeq2avjkjMXN07zTbTYx8ObH7hp1Wq9aVVFdvFXT5ofL0p9kpItUwM2mFCsrGeMSdOWHhN9SOTq6OxdXYyLVQEGMyX6GchfpXmga7WaWH2IPOqNb20J0PPv9NHUTvpgBT4BzFFLqfv1UzvvBM\\\\\\\/4rdigkSsu11qjr4ZN8ud856VvTzeS365\\\\\\\/SZ\\\\\\\/n8B5vfzXedVXrieSG3ZRM0mLSMWqFEZBX0CEipjuG+9CSYNa09E1kufvQkYBYw6b673bsEp\\\\\\\/auQINZCKQdGpMZHENosREy0msz25C5L326TikkoQJJ51mKPJsgIeZ4hHNnZmoaZ3J9MfzWBLOgrM99biO4cKkd\\\\\\\/FJBORI3j4knJaY53iJH\\\\\\\/fUOkPIN3Kuzy8FwIWdjPb4\\\\\\\/Sz1O1Lr0C1CV9aD8WsyohKzkmnXnnJFtPOh9psiB4a6i4WfsogjWwgwzREgzjv0NiYNsMU9AiJHncLugx0CS0uwML9Wrt3Yb9ujn294sGOBBVx6n\\\\\\\/3JelWD\\\\\\\/7UnB8QmJ1PNUqsePt+BaKaMuQZbVpnkM9LOFkjoTr\\\\\\\/jAbnIpx\\\\\\\/SrbI8ZnFVk3mGmYBj\\\\\\\/0w52zb7dZqcNbttk7YbWOQcVNaT49gIdiuTuTlZq0t7nVmKnLE8NquQBpnfgL7RHZnhAFgDGF+12NhDgLTLekpdAzrmmtowYteoDAGlvbGwLY82afI0LHmpX3EQ5zPXEoN4TLfV55ujFDF1P8VQBSZYoIF2WtdHsI8RFsPZzjeblCiW5ilPDtViAKBMgtO7iv7LjW5axj90rPRYxMWIx7W2EikBR1jmMq6ziLtRr28wnuKeVnP8w\\\\\\\/dBgYvvdPdCOqxXniULPqTo+AGpNn05ICCFs7t56ha30FqMgdSzpvPIhmhqMveoQwx6iFh0dl\\\\\\\/OobyGPwvLyoTFOPTn9vctWe5drgrUx5PWNIR54xxn5r23POtcQDJDC58rVjCcIokxi0FqQUr14g10mX0EX0s\\\\\\\/xK2WtId9+8lwlf2kBRbOaJiDdlLFfO2VByfjrhooej+mKoMMtdT37J9+B1lbwV1xCxSkoF3qnLVCBoowgGfX1MO9reUUgqs5Zv4LLzLbwUFuNcWYY\\\\\\\/G7VnzI+B7AnkH7o4wKKqqRKbNeInKwnPuAmYIxaw3iVrSbuHdqpCC+VPwk5drD6pyZD9wg7C4f7lXMaP3LcnEmP5VfjO1VMjPdDoKOgLLlZJcwd1TRz5XrTICu0OYJCrGJ1C4ACUUIcrTWSeeZLCN9T9if2FXAL+VyvipxqwUiQzMEPSxK0kZ8bQ788SIcPpPn+mymZODU1g4RYrFYj+AIHAUdUzZ5CMV4hwqJu32cW5nhLyasiRiBzgFBT8ePHiyg4\\\\\\\/3cjioHHDIkIjahZEXP0vrObu0bbdYleDvyxUDGPa5FJjMd45kHB1e0IpA9\\\\\\\/lKUGypu9I78GzFoXpN1HvuUd5GRETj9QstqZAilevNNDh5wie5E87RXQuYDyNToqZD8RJU+0l8H+Pw108A1MdlYRg56Hs2tSAQ4c2A13Ev6wYIO4yN6FupG1dS\\\\\\\/O\\\\\\\/pM3VaEB0V\\\\\\\/14QTeRbxJlWMEO9kqPwSo\\\\\\\/FfS8zJ5LxriYTV+9mHMBt06tIViwDYYXfxEVz\\\\\\\/c39jUOQ7kQ4ZVqQ4Nkm2wzRHM89zR0TL5Q+eUHiomCK2mepX6Gnn0I+rTX2WchY6rLubhfzzdyb4SUu7PhNXveYTWm3+nx9VYbQjrQM29ArioIxyR1iF1NQ9Ar4lXNb4lLuA16Bm0p5sZCdj\\\\\\\/nBdK0DaepE9V05Js\\\\\\\/2t1B5TwCXOyDbRG9B8mrtPXf0DuEjYFAzPFKIMKbvH4uUp9ujGA5wiDy4cVIpiYupjcgBwLaIo3+7yP0MLdBoGZ0TBVO9LsCx+ATK6SkxksYIbUPS3coD2SzApcCeDq9S6TW4K+j9RJ4yYlsqnatJVDLrdmS05AtrA4Z4Yw+2kY4eonbTCqsl\\\\\\\/XngHEii6civOUnfRVTWgORaWFwEzbLTufLpgfBucNGFlUT6D0T7GoJn5z+iIoRNiDihIowLnKip6C8LzgvY0uF17+0dlw3PU3S0pjkzfknaxQEdr68F9Bfgd0vHqPrRFLrjasCOHZZS5jWGrwjoIhHzqIzT4QGD50BlW+BSsQJS96\\\\\\\/YPNtHtzxma+G4Zy3YxtPESlw7Fbs0cJOGXnnutTOzUnC2goVkw9WM0XRq64\\\\\\\/MTWnTY9CFqXQKke9Vxusiyi3+5F5K5G3VA9fjdlir+mtVRqpv7d\\\\\\\/ss+ltTyFqdmfZlgyu\\\\\\\/RySu7FjdQNYSKCKRALkpFUoI0pT7wzGjT23tBa8Q2M1V3j2K3GrBfKAYtg2A2QgjNc0PoEV1mJiDNlcjX7Lad3kcX7uwKIePy5nA0kNEP6yK+00yr6QJ7qw3fUwGj0HsvZ9dBqhLJVvw8Mzcr9JSS2aI83StnvXtX7qNBFEMLBt9d\\\\\\\/sS+wtr1Wwp2JayWRPMT3dHjp995JL4VqhEmO0+CUy2g77KXU1h5bSW4mqM9QG4idQ\\\\\\\/Zfb5tjaxtyzSdPTD8UXJzngirT9FK8uSOPxej82Vw7DD0VtYltYSlc5r\\\\\\\/7G2REpT\\\\\\\/Ox0+1ECm5vjQym0mpGq44lPCnWxE2zoZp63dcm4ZtTlJylgW7Mz75ly4bEwUOMoRy3sCcRWbNwuJji1rv+3p3bWcd+t8kIUw+sEL8Lz03ylo2QOreK4XhhLBZullpJhYQujodnoA7cb3WQUqXeXHoFvsz5wNifntn0SwJbsmhl0TBbRbuX\\\\\\\/kuIXoCh5AvjgnFh\\\\\\\/+UdjCh\\\\\\\/CJiB\\\\\\\/Biyhn9HVQVpSqIXp6CCf0QUsBOOpj79EJ5vYCZz14rAooQwiZr67QKYC+KP+DWJizPrTqlxyH8kDFE+Xc87bbAbu8jmwgnh0xmvAyaG8sIeOjGm8mMWwMrsMNhppNtkmB+o5AmnA5Sj+nChjcHIUY1QxBXxhKJVv\\\\\\\/5Rg+infMPaYqpnRuR+\\\\\\\/Q6ZAfqHNx9JgGZ4cjFcDcEruFghKVc6tHDHc+d2Q3RbuO7DO6Nss\\\\\\\/CUQRVAXhRDpDgS7xifba6gyRpQtrRhGy28tMDNOZajB7AVQTs0Xe3xnlX9K6mGutcOkliL36tMaT\\\\\\\/Hw0h9LEhiPobkdYnaFiDDtAjE4s8M0673cWOhhO4ogZ2OMyxnQfC9wYIyo4yQY++n3ui4nmhsUnbL5Xt8zHuSqzpQmm2M8\\\\\\\/BKA9XK1J0r1oupXp3frysoor3cbscByh0TvK9LJdSoQ+iLoR9o5p\\\\\\\/SH4DdQSqYACf96flX\\\\\\\/Tdd6OPUvAxbvVoghHsrruCDk0AG4\\\\\\\/LDpQxuWr9nQqG+PP3OtALkOrqEbGsuuR5LAk4MKqZfl9wBpjK3AYX2bNWXKpsdiM+YLkHUl3WKgUDNo43xCa12lc8maZ6kGT\\\\\\\/S\\\\\\\/240EI4JtmcHBcq0Z\\\\\\\/rxJkHrJuzx90uHm2SnoqfPheNtFhZiYmLs+d4DmrNHM0hk1AHrUTg464SLAASj4fhcF2CKWRF85utKFfau3d22IZ+YsjJbcLYK7FgPUojqwxY+1yNjc3kSIbcdQkk6jGoZ97DPSmH2XLgOt\\\\\\\/SpKyvlUu33iJFIzamQ47fb8LEeonjeM3nqxGU4+GcRyrLJnCXk28ekflksWHh9NTgnCLQ67qn7Oa02nQJ9xuNqEjfrWT9B1A4y1Mp9d9h0F\\\\\\\/ur1Hy8ucwTmuBr2znVQiT1BUaegSxvs3r7Xr7ntZ\\\\\\\/MSTiiRqX01WIRDD6QVesjCrbq7yZE7S7UR+jfZAI1Yj79XYy2cD+fr5sq\\\\\\\/93vVA6\\\\\\\/2EcvrMBpaBUNlGP1wKhApJyY6X5Lza00s8iXk4\\\\\\\/kSbGLnRJSRCJHzZFHnxoqVHS3YnsSYP1zyqbpO\\\\\\\/xsmzdG2Ywu11q4X+xIIShZbtITZBRBa4yicOwlwPbvv53vGS7AxrUMGxJthyafenenDeMpSdoUTiustaHXnYTS12prf3Rd+hZfmNRS65+tv+Tmp\\\\\\\/ntPGVG5EdlJOsfZMSfRngAn+qkPC0A\\\\\\\/a2xnH7YyWf316\\\\\\\/xYcW2N3gl+xJp1A3+l4RjXoB8NiSxaNepDpyzzd9mpLYYV\\\\\\\/yQ2g6hy08MmliOfDYtCmFjAv3iqydKli3XmdNfeDPomcSPb+M1q+Q8PaQRM89JRXgbldit0Pe0x7j3ARcD7hLRdOffv8+wq6mtNd7YjMXsYhRroKOYFHWHEFEN3k62zXm9EUOYTKf292oSHv0cBokGtm2gRgQxZu\\\\\\\/o5NcbQYp9F4Q63CWI6wIgH5Qr6cQLiwbxc4K4950lCRf+Qlzv+vXm3ePVrkzMsBZQlmk49yLi7smYWyzSl0+WnWRk28X+yWLRBy1Sk+8OFWrsUxmmDRv21WHRCC3lMq+nz21RkYv7gqhB+\\\\\\\/UX025Vm6DZtJbLmRVuJ1vZqfNejbVWdBJzvElDSmjm5xvmJdUob46aPJ31Ey1jsbO3NZgwcVDOyHLU+IGf4jN3P8qQSRufJfG98q6q5jBNtOtPOqxei9bPlluSqwkYH9BF+Kma4SpQoFvFcVp33QRvINzCezQQauj+iGLhzMjAKwEOHjIdiJFkJdLa5Wy\\\\\\\/7xFx9moWT2KlB3xyN8FbJ6oiscX4ZknVkTFxWLWKZ1eO1t6lec\\\\\\\/9+EEDB00\\\\\\\/AZv08cfoXqDacmokCI\\\\\\\/SXvrRQz1eABx0RB9Fs98qidrMpIl8NI08PHIAMdkTXMgivH0HL537M9DE\\\\\\\/KkuQ12ogneN30DlFT60qSsUlBLOJdDZD6d6qEDPV687+YJK8E1R8Vljlq9ZQ5PZ2s1kG2nKJ65sCSrC3YnQLZ0V\\\\\\\/svBGEUYigfJY5abvC2IXwqNOxLnimozJIGiJb8nbyGo1Tv6C6aTl4IQaIbIgFRBTSsCyBvjMBw+iyqol4SLgMHhLJa8GVgkM8cjJytrRfI4tXdVzdp0jDgX0tvaW4IYdZK8CroPl4AZhF69QmCLWRaQS3SAMshTdiUi6LTBg14uAP9thuoX46LXMoWlorqRb\\\\\\\/HEO2YxQOYx6xGXkkC5YTjZ14GXZ3c8VZvowN6vJXQiUp09p6yADLNstMSPfE9DdUvdxdVpBoGUCWrW9MzhOscDuKh1cF7ltaClKQ6rwaX14E90DrdCI1ULBaqaQGTOZwh7ERlWuJ8mcQ6xLxCm0yICky2DZkTFRxT7n0sMbfxnohlKI0hFdDUv3lOUsJ+iSqafnkoNCM6pK1bjE\\\\\\\/Q4aBjbc6QZTTZIrCf6Gu9tTA5CScoNXignKi9dD5fDjeEugSJMNCeiKbLG+kKNT89lpEWCa8JXGtz5TIRpZHzmiB\\\\\\\/3MTo8urquuJLTEkK18sy7A4T8O2LAo75GtczJnYOp+T\\\\\\\/tBM0Ym17WIbTaKciZok7xGGKePfVf5PkN24MYfyDtlPDX72TdqH8qcB7Mn9vBrfwZ0uAC8l62M0Wwpl3WJCSUjLy8tbL1FKc71\\\\\\\/94Spp7r3yreoFFDe3+IJgRZbENgPT9XtOXDCasy9A5sMhjZ5aG2XCnwBgAqCUqJzELy790BPcR19rBW71ZMKjmlW7qJzEOJZ6fcd0bvQFyit+1\\\\\\\/iiKIdxo4AShGUn7kVG8zgPirHIvj0YMtvniUpZMb7Op8AhBPr1AfBDzQ1iPhd8Dd1AGm4dIlocld7ACk9WwbNqv8gwEigHJSJbWbMMNSY6wyeeoGS7QqCXVhozI7DHYP3HyGCRI1CbHRBfxYSK0JB6Vc6Vx6z1NIWoMJ8fNYSsGxATxSVUSkZVVLMz5lh5mQsPTXxP8QPYpltcOiOx26eYtStvPi6MbHDTMmp28ptesVr\\\\\\\/sMFUXm0iGqcW2+4RnjeBHyUlF2p4xysKsPOjrKI0FPLsIcAbMZxLwZuZsiqgobB6R\\\\\\\/INDUFS4wPlvrYs9JLzsy14lhbWk1E4g6B4GAvcpOnR3o9zRLzHjVMhuCAvdlJBSsEIjPfktsO+w7Z2GQ88Km+ExASe5P7PWnL9DQkuFalm9TvnYGQbG5MhVMIeTTYDPuKgaMCj17I2GlqKzi17GfYpQWFWpWUSHg5rueryd5WMR3XgjL7wjtuVUaeiCV2U0W8hFXPxcB3\\\\\\\/ajnif5PLFg7UcCpnKLpWy18FjZaW8dwrQTVaWjeFK7+tsKA7zPzxEwMmQKQCqv4es+T2DMtGTtr2GGb4ZLnhR5iLai74q7jhNaLlEwPxDqhhuAsu5kSXNB5HJn4Tl1z26IyzgLw7U8tB2cWO0SNAoktjPZFD1noADDjjONHvb0QIrBAsSqE7odSiVQoGI1e9ApwehhnvUXeakWnBMNCZRzVnlcAYg2rkJO8leHFJ9FTepsN7EaWFlCXIhjtRbeDheJzWH2jsMRiNqIExSiXGHtEgl8V0JMOvmikjNlBO6kAvz32St6IqAtkPZO29UKHrVvI0wc0IL\\\\\\\/PGZszpubdADQlxYMIo5JVfpxbGLfSTWL5gfo2dkbz+R\\\\\\\/rtBXPct0qm\\\\\\\/qwAkeKCQJIn+bLHmBh8gLYAHttAC63LbOyXPuoahQRrd0cALhymI96ZDX89\\\\\\\/GkpPAyw3wXN1QkDILTTXiUYkdeqyVjk8BrU+UEjTj2w4vCygBrfaO0QpcEjFmQ1Wgb\\\\\\\/tBdgrw7QmTodnIhxZOqp\\\\\\\/HKVS9Kq4JcjqPHttBqcoqQwa+o7LPucUlH6iIRtdpOynLUP1Ajm6R+uDWkaq37DrJiLo+8X2I7Y74aKXgPXPjH52Nen7GAnHQmQoe9HPfIJnpqgI\\\\\\\/Rc5RfHPc5BZ2zpL2sEyOS+CU2d3\\\\\\\/hyFztu7xFcLMLtfNhHqKTscalW+jImy9TAAfk+y8g6wl7Aa+M2N7dUu4cN4903PckCO2T+C6\\\\\\\/2HF+zWec1BTCv+e6P48oX7CSNI3jJmfom+o3dyr0qbK\\\\\\\/cr8cjo5bQt9eVE8RMdBZ7S2zdk4YnCqKm+kVs8ooo3v5N6FShTIXfT0mF6bSgpqJPH2kUohRX\\\\\\\/qY7UFSg8yN+Jb355MKKFA0m5S\\\\\\\/O8r8AnQmzNW84gnfN92iE7XILjrGIscgWttms+703jQ64mgHkyNlq0c5SLX77N4oNkkvzff64AAnEbmyBxoVepLxY41\\\\\\\/Z3W3gaxj61ccjei\\\\\\\/IHAwWcNSymD9uxyebyN40T1nf6i1GkJBQ4rUpve6iiyHz65HwP+FFoQLLM5ChnA13jeYZVvjZ0qncAnnfP5tJs64wZ3wZOQC3iDEvhVq+IeNUPNVHXfxWk+eQqH3jsvJ5bgnw221anUf017RnUHzr\\\\\\\/Ga4qUaKKllt9jdjIrJhFJfUQQeqRfQ\\\\\\\/Rlo87HIYwrFCAY26+FiHNWPVWhBAGeNCglQM5UL0vYsnv7Xk5GLk3glBGbxy1oIyzoltz+v1K4CtJvMj8qrlqtX0PQYx7O2ByEBUllRifOwodObdUNyhoIKNW9mZUqswAJH+KRUz3zDFzfXETAyJOC\\\\\\\/Y5zcLqqlW+QLoRq4zKNglt57ztwHe+Yax+ExLeP+zfc1E1uzWS9sfNkpmj7G8S2GsedEquuMo9ShltX24uH\\\\\\\/Pl0Ojzi3enZyqgo0vxpxfHnKu3I8Msz+9sHoJBRapAi9FiGuHDoT9JiJkCRi+lH8Nx0uVNZUiMDkTTXieNVAvhGzzkO5B+zsP1uBZC14mcSY+5t6EkNk6AAl2igOcEfiVo1TmAci6i5dZqDsfYuZTjQMCXcuMx\\\\\\\/kYI3X1bLGrAqAO3UO3wkUmONsqmfiZTcwbfSqdI5CPkGR5OVjV\\\\\\\/p8A7FAJ0DMztysJT3rx3573t1MuQpJiNlhTE6fHQWp5gO7G1ALEJjST4LmQwREOQ2VMZJ0Rtll1X+e+yx8wDT5a0Rv8DFbsaosbecOEQB7m\\\\\\\/ErpAMIqbUmxmQgA68IRiXZ61Hzgqs41D+n2cADp0hIMzmw1jCWqrXhm7LwMAtT6W85VMoEy7z5Q6iUC297qjiyCUjCQv9jbmM6832avSpTaxnzC2MMKz88WA4T\\\\\\\/kdq42zM74tPNH9Y4uNW7SfXW5UPtZRTHAjbz63u5keEZzDa9XvVB5kxWYmGJcgcqdh\\\\\\\/\\\\\\\/DjguE9ivvny3OvhohKy3JnBKbqvvYgoxGNgTsq5mEOTcjMzwnow12Qz8hdnbESO54MrpZXJE2tClanAdZ8aSAEjKYQNoKskYreWB\\\\\\\/aRpUcSqd5M5av4ofAlp2x4rBfGcOfslCJJB2KK\\\\\\\/pRMo2RjVq20XoP+NX00gBBH+b3T1Q\\\\\\\/g1tgkjN66otHlp7LAdZZZo+KPGNvmRAl9Eg1oVJmEGcCo4vswj7zuwcJzTPH2j8vXk5BOL8oMbFIgdtA2Kh95qZ5i2xN4EFaWd96R\\\\\\\/UF7SEiNvGXqVrPdwCiZkOnfl9e9Im7ifkJuD4HE7tjI4o0EQJC3\\\\\\\/cqqzdusc5nKZwgpLutXFIN5K7oOeR9gK6ugh+Vjhy1NJXlC84hMoE6MoHmp15Nq4weyVAao38ePyn0N9SyVKIk6vjFkjAgZtppKNimT8YHPSVhd0\\\\\\\/5Oe\\\\\\\/Kz7EP+Ta7\\\\\\\/quKoPCc2Q4f2X2W01zG7tsZIZq96nUDhUDB8xrfg2qr0SZdbhltTXCWdIBlt2KzCvf4XbHXm5CTt53RGWqrTA64\\\\\\\/IlwuhmGLb7p4KaBQjs60RQWNJBwolcXqAJ1STPV\\\\\\\/hVPiZhNb4tAPg2T9pM+6bgqgL77KTVz\\\\\\\/5nfCEY0K1bM6ICy+xLFNg6\\\\\\\/ctfRPUTvJsPN1gWGCXrRGDSSHz\\\\\\\/4iV71XRaqmFlvmB+uXuoaaSOVUkMwfGt6XRRuSDjirB5F2dsQp6LsWIaz\\\\\\\/UnYQbTRTsfyld8XgR0hTagitLoERCPjg8aaHUWvfHX+7Q70ZyVG9ACjpqw4aNt1Vu1rzEW43tqU4oM\\\\\\\/TBPfPMlRijObj5\\\\\\\/8BicxYpiG4OptVWUoXuuuAOdqYhSwWPGDYF\\\\\\\/fo9Id4DySu7eZxTG0z2YXKmuhigpYnOuzU3eEI11mWMcXSrMbjOuQ\\\\\\\/\\\\\\\/NW7krHGi+h5xKC97sKLKpbcoAP0PfAcSPH0vPvgHoBg\\\\\\\/q2kKXP4E\\\\\\\/i\\\\\\\/zzj82UifOcJH3X8HkVi\\\\\\\/nKeKCeeFT4finGAkkRGwsbCpsX9htDh7LBPdVcvxrNW1fMmKXBqqrAqWs9YtdM\\\\\\\/Tf8L1aZTNFcZkeRJR\\\\\\\/tbZTJta\\\\\\\/0qk7\\\\\\\/13DjmdjJvC16i8kcfTHsKY+742y\\\\\\\/6yrHf+scVwGUmen5EwaVhU+YlH21NCMwrEMrOfv3iiEznW6ekXjZuakyaBbheBFS0E8FFNuqGN+aN0103RRgtM9d09d7USmVXHFwwEnmr+QK0\\\\\\\/mL7JZk1lzkjqw90Pb4CTSaKuIufazsdjOJsiBWIATru\\\\\\\/8FVA5qxZH0CTfenMsRkqBhtAE7yPBnsZrxaR6e4\\\\\\\/0NlEa2iCowvXYnDgai1McWV8pNlG1w9ldiJppH\\\\\\\/o\\\\\\\/ik4KZd3LvGUsPIM\\\\\\\/\\\\\\\/WPCybDRfmeL7vuum4jUTX5cqM6ks\\\\\\\/P2RgZSvTmKqnAs3cAgal09WlZgXH+M94p7thIR8Sr5iY\\\\\\\/TImGOpnOBFlhlkLNtOH\\\\\\\/P57sW4FeTttLI49bnjFSb2X4FnzvlHShd0dHuketbd2Uq8WdJ0ETmqsbhF+CLMB5C+TqT7i1QHYfIE+mO3Nrx\\\\\\\/iPcFVTu5wYt601Di1xOQi+LQuiMxNmuYA9OyKoUz+HtmTh0u7qDgd\\\\\\\/L\\\\\\\/ZHoVrvsk+43QxRSICD5U67if7bDi8h+4Zzx0f2FsqiyU9Ni\\\\\\\/DSjl2CJxj4q7z6MC8JX2MOzV\\\\\\\/oe3IcKJEC+lQRZFIejTBqX+uMa3R3pjnSp7ZwT1jlqSBMUUUzc6sdaxdVrUBHd\\\\\\\/AJL8YbmB7\\\\\\\/6lysuh1uiEbqI7w7Y8GqvN\\\\\\\/ng7NKX1a1eAAgEq8fdr9UrMX\\\\\\\/G1W\\\\\\\/fPnsbW+X1wmvkpRP6tTLl9WR54umHAHfm9qLj5Mo+N4yeUuP595dzXIOjkEj2mVJkYifpwzis1REwl64sQd9RNEDZfWatniZZtVpfrTbrVav87p0my15rMHmq5jRVzMevrO0m+AbFvJhAKzq0945\\\\\\\/dUKcJVzKd\\\\\\\/9oBlJB7F7xQMEgdPRen\\\\\\\/h\\\\\\\/DEa6uqbOdEpNNIrQRx9IgKSQ1JJaPN4q9fszBmH2kehPV\\\\\\\/7Tm7GkY\\\\\\\/l09tDa\\\\\\\/gQOtPvLrQJtLzndTzW5Yr6d7k\\\\\\\/9wenk7AbQSYOj4Q4BIH6Ub7WbplXHZ4BnmzYfxs76\\\\\\\/SsILYOTF3Jt8Tw1+kJFKexjE3NGT3WujVnk4yYdFSRwSdGXfVpsMIrWDXqNlPPLw4SoYG14hsQqeNE9tRYPs4J\\\\\\\/bLjs6NUWSgrnO0oLXyoa12AE86TNy6HFlS1cXQNmK7uP0R7JRseUjolzdiGEiFd6klBn\\\\\\\/gT6Qg8ZYRX7lvounRLMLHbcy4mdWasLo1+Ex6P3Qxsta0q7gpvpRg4XyPaMy3vsvEdHZHaQFCLZOYS329sZ7rhbwuVpJcYlIdfbgw2P1VW2FpkxFmYyhDoNhhLZGlLPcWibIafFr35bhF1IuvQaRkK2Tfr4FxQ5GTRfha2K3IhGiCjZLbkQuFHYqgk5at27NHzAcdXe6Kw5qmtxZVjS5iI\\\\\\\/rFCMY2\\\\\\\/JlDywhRaqtbEdiungndolkdm13qHAywpD4kbbuWinjOtS1DqQdpCkIdwR+a2fGQ6mYbgl\\\\\\\/zJQy6u+yd3FXwlFokPr\\\\\\\/62k8idCKRznglb3SD9W5\\\\\\\/CQgDIxtO\\\\\\\/STR3UMC4jaOyX2H1C\\\\\\\/o8HdXJJlSNr3NTfC04d6bra8Xnh1VKDUiECFbvFq7K1ku\\\\\\\/wleg6plWbH5JhmpyDUItP8MmM6xDkY7OVN0+8n9xu\\\\\\\/SbHTA1KrqtdCFjL4ihqQu5FdtV08EqSD9\\\\\\\/pL9MJ+dBD87ol2N5YVY0rb3PWLxCFzU4aBbwNtv\\\\\\\/cGr3Gfv\\\\\\\/4POLmlDrug22hFq5lM+oJjBk5wPnXZzieZZyRbwGWpUoqk0+\\\\\\\/2nZw7xtqN1076swWyDUrl5of0ZsAPqzMmlTEJ+gsOq+wbnVMEzGumqdl+H8aOObEUw1qKteVnM6Z2TbqXsdaje7QvRffjmYoTPjTSH7zJKA81SIg+4IM3R6rHsmN0xf\\\\\\\/8o8LgD7STjw\\\\\\\/sOzEGFHBHMsuZPleWUsOt90qDYfPMaVNYzDTWRiUgBrsi1ZlW1qcxtntOjosKwv9jvBRqTWnZJ81EDsse7An0+Nxf7yIoDgq1OkQt7XSfwi+1scvrvngIo1SHDnDakqBrG1McJCI+czlzaFv2oJW8nYepGhLBLsls5oZISti2J7UluxDw4Lp6Jy34u9\\\\\\\/1eGByUOf9MF+wohAqiQS4q60kDuodHIestMDxf0t4D5BE\\\\\\\/xCqOTxKfNLVqi1ydoM4V8ezKaMTpNS3uhKpTC8yhW5kxvmRX18nuc7XaiaxFzCzHJs6m7uYtAggDxuNffoz4k3BiMef89DtH\\\\\\\/ZzrqOefDGC8XkJueQnYjhgGi1IXvixHDygW7jM4ZG0aqakK4T7\\\\\\\/EnNeybQYRMHa464LKFr97rKqU1F3Vb+Udhm2nmaUFVIUydJRBFeiKM+K+hPAsjppaQbu1RL43KcyVF4MwpstSmsGLI24bRaycpuRzP3iifTkd1AD3NRb+Q\\\\\\\/UPjiQ6jCnIBugYWdrkmKE4aL90C9anudliprpPhD4rufWLP1mxrr8CRdyVy2boOIIkQoRZX\\\\\\\/exrSvpwIK1o4YwwMphrrq2m8R0Pd36N+Z9+CFIjcKsBqPLWq2rwa3Nx+XMF5k6Ua31CGm7eN\\\\\\\/q4y9x2rKPzd4ljNgHSA15VgzWwOPryW7QF9NyKi6ZocbVZk+hpbGYeKwIz54YaE7JcpQ5hAXp5xiw8kH4QqKOnpehecc\\\\\\\/4NclwMB\\\\\\\/KLjt9W4GsTha7d8iyRuVwfag05oBml28s8qbHVOz\\\\\\\/y\\\\\\\/Y\\\\\\\/u\\\\\\\/U539Ccz+Lx\\\\\\\/VK+S0BhEYe4l1RB\\\\\\\/h6T+5lJEp30hnVD\\\\\\\/1mP\\\\\\\/UgIvoH57s4BLsDEzNA4IANYVENjKY00l4OshZXB8lVl06MYo\\\\\\\/xWI2qO7X0ncF+zGYQnv46SR2L93C1jb00QWnzDX8TLEbeyTEtMrBZmm2MyxqYd1K5mcyFQhV5dyhPxXCEaK\\\\\\\/6s4R1UbP364GlI5lKMb67w6vChdoaf5BgB2kdetYHv4P3383eO6LjR2dLCmpP1fhjYK2MjW31zqiK65Tsvn6RyQB1JkfCGxCVuoUpjQjwmvYhOFKeoV3Mwodn9l5PhTDXE1txyB5OKT6d0xWOiQqJrAZ1EpRr4IVhzGvrsIWwA2QtS\\\\\\\/1zyh9zokTMEAJPYfJnJbcFYFNfVtbct9pyJDVAm85LP2VwCrNSwM8ZT0xwuV7axhj0w5fg4x\\\\\\\/kkcQ3JlBWdGwQUfmVbBQ3It5UYwUYpkEHtxBB6tHG44F1xE5NfyOimFnTHSrkjEwBUvRudVpawPkgQiO6mD5AEdpStMDfb3QoXMTeVtGrAhTUB9Y4JgV5rzbyY89ayb6pUBQA28tvdNdEMCLMQablHZ2dbstUIaU49GuKJ+U5CYwkHdRtz3KhZkoyiTk8NUH3WShCsvnCr6l\\\\\\\/DKxCuUvT9JtwjPySlXDcidg1DEzYi9R33wW0d14mYAT6mMukI3NhK45QS2ZN\\\\\\\/BGyaQHE1g\\\\\\\/HmflgHBvqaDfaoZ2vGpBRMBSJ5JgVrvYCfaSKH8pU+AQEvar70lC1ZPj0EfV0eRASZSmwrC5yGJf4GhUwEMdvY7VDJi7bMPgCWOHlzrMPPZh9QzGN1xstMBNCH4l21WpQorI0L3LvCaa87H+igvGn4LZD0+dhkZHBbC+sofCSo2CdHaYAMrDiE1VsKAoWiE6eRUfNtxoFcg\\\\\\\/oZ7dT\\\\\\\/ZH64oTuJ4vp+qAgJqj2Wz5ts0EwXLhkBEa2G3m9rwc2OZe1tpKsvc7XVdZEQWmN2wBlpTyCf+twTA1CUJfdIe2hlqJy6FP94mLl8jnPsY60wH9v3hV2GISEqZ7uvQr8TuEuufzhi6RFQFvgKE1QuId0bTZD7Pyfaf37zCJ+2lDIqlfsC7T2T\\\\\\\/sHtX9ERb3ncA2xKO6kHcJpLRgMUDSqo4WKGNItuzYKx5Z9MpVbcWNP1umlek4j7ef7ZmbGhhmtIplgrSBWCOtito99BKru1ai8Zpf2ppue7CaPP4AfAzRMxhpIgv+aWgj8oK218epCCCkdt2EOZPJ0awLU88epBEKqsVn4HiiD+FexOeKiUpUPpebyewygI04j6UHv02L3pQ6XFSvlI\\\\\\\/S17bxfFjfuyjZayWeCK705pb+dKaxywh5NbNWnBYxFoSIFO6tifyHWEb2yapuuIHMail7NGjM47iGfxNLNl76U+j7wMh0pLl6dQJO9iIdHWjUXr6WicgGLpXV0ichzVru\\\\\\\/AVtrqzfEMzSsxMCRx4SNCiQApc8\\\\\\\/nNFbu8zM8EppjH5OYaH+RfrNxZ+X9GTGZEFaZvhFXVtcBKjVn0IXxl7Ug3Tzl0k60ikhYtLyFUZ1b8UCQykEtJ1aHidhqTR1yhrhdoo6Elax9c84DvKyAedQNqqyG8vI3gZA5HDsi6XQbPIugelfqjm8YZTHpdmZHnWurP3+\\\\\\\/7BhR6\\\\\\\/keX5iSTTn6enIr558agmQLrEpXsmCDuDc6r880JastjjkaIc4Un84OFNFN3vdwzICxPDr4WMd\\\\\\\/k6GB82AzNlPq07DPdMY4HAmedm5ZbtzUezhHyWYK\\\\\\\/vGHPVi7b00Ss5VYsZX8WgDZ37Ctzc882EKuwQcbapNFbwr+aT3tN5KwjveVP1tilPlQqRn5agItMpNmQl3MRChJEZ6zQLy1DsBw53fkdYzg+3Wgg07mrbTQhCzlcoQCV9S7Mv86i+LHj0OJUUrJ817oHCZwtHKfX5cuWJCleFQxZWB1S01SDkXzdVnwYDgyNJix8x6oNROsFcHwwoAidpCHforGSttCYZKFB927gWKP5ZtbZ9br9RRLT\\\\\\\/HENyuQWgAdjUT0MT3F610eZAwdoGxjf\\\\\\\/BRjIU+4HyPSkLu9Uuuq3CL12lpFru+CXoW5SPU7NNpjNubJDp5sBjkJ38DcO+T9L5nS6bbJ9mkIGRjWXASGBwiawGi7qm5nboV8bZraOq7\\\\\\\/QEphy\\\\\\\/PcI\\\\\\\/aLOUZfWvwJEd4fh5rxYWPDj32LTyCWGV3Z0IwBlSFBPQwB7bOzsb2kCaR\\\\\\\/RF+thbB4Cfzm6pqse4S8y0rW02PlYxBEYLyfX\\\\\\\/uwXH34rTGgeRTjAVyMxCpPs0vd\\\\\\\/28oruA+GQqEY40k7StlOmHPeklIDqfWXQrrJY5vQN0hBLl3QoDp6ozXs89pi9WPkjHAIPPCcFQ2hhfLdevUP\\\\\\\/B566ujx57qOCsfTjb6OnYMmN7aa+RHGigPteJBPfFMnye4jIvS61aHadOx3rF1B5EcZ3GxZZbkz2Az4AdrZKkcRIsKI3AF9GphZRAgWZF+i0OEMNfksxTzDQg8Pn4jCkEdocmH6H9VYYiaMVlX0nxQ2ATTKtDvdFu97G4bf7auB1+jHZmEK1ztwYbdcaXiiF\\\\\\\/Ulx9r4JYq79ZSZUVocmZsiimIyi3Pumhkvhm4TaaZBUWw1BOHplvjqyqPSFA\\\\\\\/yhOhdI2Z0E7JPOxwDMZXynEenWa4yBRDj1vc4w5YRXIvd5X10u57w4CnY+8ocz4ILjlHWXiiSwP9PW8UK5M56q5rmGZ1t0cILlxVcVqBoKN4xrpOL3g87u00Sfzh9Z8KCutUCQiRQDyjzQ7htGYJAgr6w\\\\\\\/PtLH8OozAxelxCZzD9sKUk7OWZaBDfBjHCQySb8hGKR4EOYEygoO7f31039dIZgYVcn\\\\\\\/nZ0ZsuQNhhp6CUPpPiCxCqx1IMiUJnINtIBnCDZdw68DvpIeZTq9Kxd55EvlfTXHsyfpuyhekPmtEbxmQn8\\\\\\\/xYrHeL6aK62iCbZSQeqRJvuwd1OHE68rRQKt+VS9M7c\\\\\\\/aEhVBu6vDIuD5ezGjJNQCOsUVD8cu5eh0Zmnl7wbfBe1T5BR49rgtwpDhjb3dY0EbO6ulNxKEzddpkUXs+r68khlX\\\\\\\/ah1VSTCu3bnacIvRwfS2kbktbpLXfQFSF7bkKKv7ZfRDd8Y7X5unEb2mDF8zwNe8spocSlcUouQXbX4n9jE0gP2hAxq42I3h5QMTcuC7\\\\\\\/sEQcVGPf7MeeUORgYWO+aYqK58MUveUQq53\\\\\\\/AloKCYec8j+80cBDNpSQqJaVQrkPNDTj6Cq1cdpNMesGF87oLypZ8FIF7NWrzqxMkiHprinKMeANL6\\\\\\\/hm\\\\\\\/kVUdNSQyFSRhiv67D3tT+Fs32ruTUG4NlSwi+GnWT0LJN0K\\\\\\\/wUDipYr11i1WjLkYfpgPZUVvd\\\\\\\/5xRvufefMp0F\\\\\\\/CgeARFwyPxXPbGGgl8HaDWPXNtgbVMPOXVqG8v76tLWaPyoRe94YAjnWLpPbvpnyuCL\\\\\\\/OMeCe8kfofY0yeJyc8+Nf6tY2pAflG9lh7Ex4XNfTOveYNj4JogKNfoHidyRrpzJU2N1culDs5r8sklfy3NeKbqDZXNAQdx+DNfAvPUZdyQU0J87rCsApTvEOwSl43mq667F\\\\\\\/647eE\\\\\\\/KByQGSNu38ueJt5UkX6iF4LhwttbMwStER3XSI5FtPX6MGRahQFu5FvoFhqPJcFUvwtBQ2ToDYfxlfu3IZAzKs0xK3pR0PHajLWCAooq9u7euaYOVHruBnssnDTQoM49gONlMwSh\\\\\\\/rZaNmCqA1GKWNUDOd3SIVT9h\\\\\\\/Bc\\\\\\\/E6pDJKwF7pnoAYY3qIn1yPy058nnbZX7pnMGvka0NS0vUlkqM768MtDfh6EQymtWyCcQV86R9HVKh1kEpn5YNYNkeKvCi6i33KLFFHtyUzIRvFXG1H+Mq0GDU3o\\\\\\\/Xx8VyI\\\\\\\/wZAPfzjmbVl0ju57\\\\\\\/uVz2ghYFCXPOJp5x+bq3OAuFnJmCZ7FxzBQfs8MLvL+p7A5LQIFO8aSOSNsYT8zZPtKMcSTnCRpzoSjSvy0Cm\\\\\\\/ju89DiUTaIy6LavLXqgggivxnD\\\\\\\/fFLRUrFG2XxwKlfp4r1WTM6clTrsxDx9b4ZYC6VgWgL4Sy5ggRBUvTKdc8ecZ6x8S7F3YR6waVkmo356jJkT56bRldli1v5NDTs7ki23BuWHg98fnVBHN3SIMc4XXzSUeFE8Ab9UPyH0mCDFaVDXgDx2UNHiPsHefvgBQvmg5Nuitt+zj8mNZPoQeAAo2nb\\\\\\\/W09YoSCi7sl1W6sr5EOAxZ6F3KP\\\\\\\/1c05w3YwbZjnmr9TGdAJmfOg5xX01i4PQF1DH9WxmDQM9Jia+kqkhLBd6m6RINXPHzdcC9F4awbmmqkZVl7d3ZK+CsdbVhqCCUxQ\\\\\\\/D7Xabgjgk3+Dbrqtzsau7LEE\\\\\\\/yIxZFp+By0bPTj79kJK0hLuJ+woqXhL\\\\\\\/T0n3zS8T8DOp1DJk24EH+9ZTCvwifFw6J52Zq6x01lpbvLOc1YuFHstCWXTo0ke8ilcQWLm6kl64QahKHAU3+P+qf0iHDJadnmNpueqa5CJcRCmz\\\\\\\/Fb4uSWPQlMXchRdiwJfjC+RhqJHC+qliCY8JE5+O6dCYyTbE2vVy7mda16AB2gTZGaL3N9VaR5mVltGXuctLLg0jyijqxiPCimTLH1WvIE0wHAxkOkJXAW7G4ThCADZ5sBKrUXfQg96efosdS+yg732Z+u\\\\\\\/SNsgMOP4UcingD5oZ+bukL3wDiZtwydW+bJSiiia8goWlivc3ZSAx1NYdF4qTceHPnCg+MpYvIu2TPZCDgE4nCWLu5GbwaTMtDC38c334fubC3sRyn2TvKYhXWfaqvlXnQgrA1rnE\\\\\\\/39X+TXxI5dDA2Cx8N\\\\\\\/DUCrtEbyMdtF+RWe4ieuvvUVsqhZRL3My3L73aJJyEd9UY3vu2lv3YpRGHaXZSYPmGwZ2+2wBeUXxqjcg0IqhPsf3it4LI8LxTnednkp3HsSGgnfSP6wv16AQ+Ae79vjb7kHwI7I0PcZWUtlsV5ZcMWwuf49MbBjrigaBpOn9k9HcmUzhn+ZDfnsh0uXgAXEFY9tkR6z3HyZgo6VUpKy1bET\\\\\\\/U3dcCHsWUjHRIkP6EGA834vHfHH9fOBHHg9P+Myct4Yq4tuyiggHstxgY5VOnBmEH8jFR9jf6eqfroHz36LZKXBQB9vGT3egjaDPkHS6Rrf0Fa3DZr5N2Pg0Pbuekvmxmfhwz10xHmkg6K0gmrM263zFYJ5cNukgWwMqGZH\\\\\\\/LBZdJwkEz9H8THzxAqnXSdHM9d92mB42Nbw4AGw+HRBXXDo3jJYJf9ZAVxyJl9HnFEzwv7TywpJZiuCtLalgEc8suVsTZxYq4S\\\\\\\/dd30MsLKkYlWhWkEDcuEuRxIyPklOXaURSxOjN4Nfjj83S06x0xiQGYCs2\\\\\\\/6\\\\\\\/L4PiQZqY9b8tQeZeKGBI0cLJNkBwML2hukOtmCKBX1w52B4FGnFnJ9+2zpaKY9cESXfW3sv1zjHYqXYAbxMh9N19N5h3bBGqDkMkglV3jZnUvGPzpfWUeoBLYdyvy+bPn+dw8JOALlLJn25Nrb\\\\\\\/5Au0ygU7lB\\\\\\\/Tu2FSDp1ck2Kp\\\\\\\/H9Ltivgwfp344P7Vyfxf\\\\\\\/VTAEqLvYEzKZ0cChnnmlWEhpR2xKoBb7sTFcrQP7Ff0M4o0rxJYhQqEP+dbEGZJ5j9fVOaFBcDtm3K64PormOCJc05J02q7JHC2NMp59g\\\\\\\/m3v+U6ydTNXExRMClvHRwwBLJTV4wQId2X4fAyRq7yVHr1ClLX6\\\\\\\/A9+6\\\\\\\/bdUIe\\\\\\\/z6Y6kq1EQOqe0YLkN2VT32gTVMcH\\\\\\\/GJwlzI68v9RGTtFONnYA5FKpnFWJMC9D7m7NOW2AwQSb6PewLyFFybGhVQ6I\\\\\\\/mnyN10E9MFhaZ65\\\\\\\/VdoF0irgGIYPWgTdEZdeyYyykGPgfFk3b5\\\\\\\/lzWpLzxNywu6SshDz3ajOAUp\\\\\\\/JYBtYAoSNGWrT4GmT5YjIL1gN1dtkeLgw0ZIoxsC6N3OQH5R3iftUq+ItZHzzTbAhufkxN5r2OpiM9Dcof3WjGdDPJw\\\\\\\/rJkWw4tUx9gY92Tst3n8vWzL1+V5Z1DiyU6hskEZ3f74F61SW9G8tfshu6Y7551GhSqU\\\\\\\/TyiBuzFhBJWo9t5\\\\\\\/0Auv+Xonz6WwMOmjRDCY0ih4BnUP+AUyF0kRDYxQC5t+9eYWmbdlA1GD5HUpNQJwL\\\\\\\/qZzTCsTwazmPzwZm5VUiw\\\\\\\/JWtRvUd1dMIdqbIgD5wjFobexcNO5J\\\\\\\/aVIK6rDZK2kBZ9ZHsNSegsV71nuXTkCLOhJVwN5MLV7SNhWiSnp8tu8m6l9PNH2ah8mF3solBXJShbPDjnuDQHu05a8\\\\\\\/j1ZCmusiJqi4kuAQ8oB1sFm\\\\\\\/lOlHQgFsEfai+QjI\\\\\\\/DWr7jMvt\\\\\\\/GbMc4f+Qy1rkCGr3XfSZJ5pLJBoWET7F1Lwwplc0+oU3d8ufX6dmOsrIQzMwoIPHDHuwP4bLurE4LCzdT6rhJyuKV4tpOJxiMjaCXavKOQ96aWrQ2GvwHyWNXilCfnaumbwqB7hSTCIvyQP4nBlk6UJkkiRH81jkb77f8LjPUxrZ\\\\\\\/KQQw6HlTMzZZ81U\\\\\\\/D7vCk3EaJfNtSs2Wfs+SRCwZBV6srRkA+lZOZ1l2QFz6qO+lDB4JzNrOEhiI0Aia6+tpAbgW796xZa7uIXPc5Papi1q0alLoD5K6y2dYforqXSKNZ8jC40\\\\\\\/DuIp9gpsfj3jHz+eYBKTXYYFIw6qjnATVki7I669zdxH2fI9XqTrL3OGSw7bQiy2JPIrrFLZpLeWOxZ8v+A07h12r+zT2j5k2PRps\\\\\\\/vWOzAQSUma4HZJ0glEsjln\\\\\\\/3tKX7ndbi7NoFXZXv\\\\\\\/VHqRq7ZRvT3Cd9IlqgtEjUTbKM5fdrZ\\\\\\\/koNQFAWUcBFwaa7snbbuhTorepubKVqeY\\\\\\\/PETOaHUBYsg1O8YcyBNF23QxcWSpL+2DBV7KNOV0Qyvn2aV8WOvA8L9cmHiBG8pTM05dQEu3PRlgvWyAeDCBdllg9D\\\\\\\/lU8EfBp5lnfzZe\\\\\\\/JQvpn0bHYFX7cA\\\\\\\/vN6J3liBumsnBNPE1DkJDAUwfD19j1GxHOafLphRsqliHTxn8xJ4acaKOj0Xo3hrroMq7HMfsFoAxAZtkQRu3vYMgw64YEWEyiQXZcLRkoqO1Rp3qb+j2eTUlAP1f2sHwTJT76mqLNbT4e9kkPShOJjoFKwDIxF6022etWVFl4HhCl6C2rqOt9rPMMCez\\\\\\\/LfZVd\\\\\\\/i7Z4nmnCis2Bo405bIm54WUUlN6boZ29iO5YnN7MAx6twir6rXoUZkXxNE83Mc4PrjcNVQAjzHFiLZDBMVaeGTB+BAUmLr9bRGf2w7\\\\\\\/UwuJpWK0isK97DRLYzMTg1XVCZjUUdwpUU5uQwDGKzvoc0cjrKB+9NozrBSZ5dlU5pLcbLxgNwlgThOrV8PWBh5p4\\\\\\\/dUaXUSEl08FykhIomqI5zDorrV3KmiRDvG+gPBHCSRyr1ZoCpfFBy1u1DxybLX3lypVFP\\\\\\\/DaS3C0kowAyC\\\\\\\/LjMkc7J+C4D3xcr815heIeE2PTFNSbfYlyL77zzN5GgkCOV53cpM9frNpheqBCykTZkx6u7nf2e7EbYdsy9Hqr5Y39wcMGGjPE\\\\\\\/rwW3gJfVf3oo0ahE527TZoEvEE1G07p97IpHQO38aqw7\\\\\\\/RNwNhcqmn3e37Cv40IjlM191Z55c97VH5UGZbgELD\\\\\\\/iLrzKuzL46xah4ZkT4Ao8jJLRnilCWJcY2zKv0Pwpfixj+viEAu\\\\\\\/pSLXQ3yBNjGyG+QF0lciRhbS22PcDiYpwSUt3lPfRYMIT3ipsSFjnDElakvQocFFX1As22pxAcRFp+DtFTlwcD9Eu7rxe8i6vluhMvH9KPnDfXoxjk7KelSr57Kqr3rCcPbjaQmvXhtlTgm\\\\\\\/maDlcTZu0rM+qUqSs\\\\\\\/BRYBOjIYcU6ifiAopnXitc2PlVwtekzrjRwDBpRPBc8NkBG7r+EiJ0PVETsZuZbf1GFJfL4af4sOOJajYDlT81jZgt2AHSeYHlqJ8QemjpYnsguPsP8uiahZJqG9Bxe1lZ3lhY\\\\\\\/T9NFjEIhxRy9jPVT++MY5hiP\\\\\\\/jma88mJ9rGvVgyzo9Ei5BeBSDW\\\\\\\/kUlIhMJUZFPCVmVdKBTtrPIA\\\\\\\/0xA\\\\\\\/KoKRS\\\\\\\/TJEXsPvvXegNM0fBOwz1q0bv0quvLoKmtJQ2Fb\\\u0022,\\\u0022iv\\\u0022:\\\u00225QGo3skwcVcvMtQt8Clszw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d519d1241d72fe35b6226b6ba98e38320ed8a2bcc6b127918db8388082300e1f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+M3jMjbSLVfzaT3Yz5puHvxRAKBXv0tqaCQ8bUZkUebmESN+h9TMtuYmXszs473Rezo3S5IV4I5So+Znm4N6Kv5wsOuWuC4cxJ1vBjNHCFVTEfwi+lM+74iIR9jSBLHKFLfIPPqIr3JFNGzHBelSNQEZFRPV\\\\\\\/33bXEvo1AkZ4KErejrQtDv8X8WpO6r\\\\\\\/QqU1NHg0sjiDzkHqS+huF5tp5jDu7I556ycLtkogWDH\\\\\\\/zzRcAj53O\\\\\\\/yBQ0PENj7FpkNoeS2jBg1FX8PDJYw0FaPHwl\\\\\\\/DZuMnvFKpbVZG+WKfhmp2RyNtdVdrpHx+R\\\\\\\/Wsvc7ByLjJdjqvTwdnwO6mLrP4ZYPwOXA7O1Dcc8CLnv6KKeGVlVt8HEroTfgQ2F35zTa09f4fOuqmlvazDW+ob9gCtcq7DGXU57neapXBwZEAe3hCB6E7LggUskNel1bKUy3zehmigdpibt87s3Pzu+GMPhWZT4gRijYJbYCn1hiLTrykwJ8Othp+q5f6S+8kxKVjO9p2Zn+3i9fIhLgWAH28z+NJ9eVXJZkVJQeQrlNtaQP2Z2EDLLifD6+pwsAjxv5BCBbmAOBRKeGm1wFFRNJjz\\\\\\\/oYfvOLJMK1PB2jdqB6t00DS4XbhcKmAUY2r0evGB9m8s4++Ls+QfRyb9avVjeB28VUIk2XrVlAQiPLszdYXeYJR4lLFkITGSwrsv6tXQStaciZcgUrxS90SVtosj9iYrygyDuqePd9\\\\\\\/g5Y5\\\\\\\/yxTzmmKEUjkEBWq09+T4tnjxKBNgKCH4+eJ4OlJ0vbvI6CdMG1xFbxzD0DOOW\\\\\\\/Wa\\\\\\\/cF+H8yjQ2BhG3huKwi8Rcx+SYzg2\\\\\\\/v76iZXTiKYjBGgkKTLdUZTtUrZZf+iZr3qRq5FqYUN0r6lFkH26gOtKnWoRXDayI+Z1BwcQsodtCmgSwq51TcJK861Wb+30XIbTqD9ik1SqzZ6oBMISl+JAtVhrL3m36l0MdMdyXzm8hj3+ZkvSF2Tnx\\\\\\\/r6r6p8J6z0518MConuWc0Q8tvDr+kuQon5X6qJMjh6003xcLmFXskZIR7jvMxf3fm6sokSTDpCUbp3s\\\\\\\/UpP3nd8fBzPb1KpO7ktSFEQ64gaM6tDW50w6qJMfGwy\\\\\\\/CeeoX+SJMS7oaXbbce7TjUbW54wec2gD0hGy7+bBC1qyWiGn8KK\\\\\\\/4rVcog96ia7vW9iw+IKiPi6q+YlLKtgN5eJiU\\\\\\\/T7YMMLDGBpKnw3wg1VRiA4GIJaxJ4g\\\\\\\/VmEZGm9IbA4hj2TlsXe0RFV2vPcn7InPtxVPk5VMdVmMEUA4W+Ut8EG\\\\\\\/zv+DrnsJmw7Y\\\\\\\/JL+2VbJDnmVskUDFgcfd040DatDXsoIGcEiriFvkUQPbDwilMm2hpxVte37bByixjseFaWn3V8b40djyaaBhs0C1kC1k8tRPYVpmopUtu9c8P6gNz\\\\\\\/sLiuv\\\\\\\/efoQ9Xw2u78i7X+xNTwz+FyzHSyVsZ3E4EkssXAu3RODp+P9T544VIPs3GoCFH5A3SCjvTi7CbtXN+kFC6ZOaLDSoKNQl7mXzkGlQUF0O6RI9bTPuTBqMHRdcmmMUIYqI5th\\\\\\\/aQB18lazYuUjwm\\\\\\\/YjxE0AqUsT0T+9490+dinjrdKG6N37HQ+XYPRK4\\\\\\\/leDkj\\\\\\\/qo8rn5O3Fw+enhq7crZRnWhuU6\\\\\\\/2H+whj93xnUqtzFQ32vswodD4RCIoInnC+h39O7vzom3jdgi6rapC5udwMbp41cxoDtA4EkxIuhmh4FqjWW3vqUx9CxgDG45W\\\\\\\/I+3PNhsiGzGOPoYzeCk7DX6K4RzexqD0GuogQNbYV+DIJjY16G7F5fUKKUUjOHqMD8UR7B5jAWl6IJ6\\\\\\\/kie1M1Z280Q1\\\\\\\/J6mH9YKITfbOqIvTU2FtaFewOzdhf2XfFJg8ucbBVo0aFxYkXDU8b08PkUFfwZIHw021HSKnZlzrO+rA0gTUr6yzc87fa7BluUPP0c0bTaZqgs7VQxOvvRAAF6JAaHbxhCxst4EPgEE22RI9MEpciY0Y6ZDgqT3KVfB3CTPJRmS377xTU2VgqKazhAuOp6cI=\\\u0022,\\\u0022iv\\\u0022:\\\u00224orNzoDbxKhfTEbN7hn84Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a20fa895642c85d5cbaa91c5524e8f9534c0c193298a69c71ea512d41d622909\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460440 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2583" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-aDMm_h04fL98YzTfhZnirs9YfUdcmdC8DSsZ3qbQafQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2583" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tjAYdfJQy90TaTHm3q51geTnjd-wvcKrc-kuFIwAktY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2573" title="Genomic and Transcriptomic Associations Identify a New Insecticide Resistance Phenotype for the Selective Sweep at the Cyp6g1 Locus of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2593" title="Crosses Heterozygous for Hybrid Neurospora Translocation Strains Show Transmission Ratio Distortion Disfavoring Homokaryotic Ascospores Made Following Alternate Segregation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460440" id="node460440" data-pisa="ggg;6/8/2583" data-pisa-master="ggg;g3.116.030890" data-apath="/ggg/6/8/2583.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolution of the Insertion-Deletion Mutation Rate Across the Tree of Life</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8336-8913" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Way Sung</span>, <span class="highwire-citation-author" data-delta="1">Matthew S. Ackerman</span>, <span class="highwire-citation-author" data-delta="2">Marcus M. Dillon</span>, <span class="highwire-citation-author" data-delta="3">Thomas G. Platt</span>, <span class="highwire-citation-author" data-delta="4">Clay Fuqua</span>, <span class="highwire-citation-author" data-delta="5">Vaughn S. Cooper</span> and <span class="highwire-citation-author" data-delta="6">Michael Lynch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2583-2591;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030890 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460440" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Way Sung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics and Genomics, University of North Carolina at Charlotte, North Carolina 28223</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Way%2BSung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sung%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWay%2BSung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8336-8913" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Way Sung</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:waysung@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">waysung@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew S. Ackerman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BS.%2BAckerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ackerman%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BS.%2BAckerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marcus M. Dillon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Microbiology Graduate Program, University of New Hampshire, Durham, New Hampshire 03824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcus%2BM.%2BDillon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dillon%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcus%2BM.%2BDillon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Thomas G. Platt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Microbiology and Molecular Genetics, University of Pittsburgh School of Medicine, Pennsylvania 15219</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BG.%2BPlatt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Platt%20TG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BG.%2BPlatt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Clay Fuqua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clay%2BFuqua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuqua%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClay%2BFuqua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Vaughn S. Cooper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Microbiology Graduate Program, University of New Hampshire, Durham, New Hampshire 03824</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Microbiology and Molecular Genetics, University of Pittsburgh School of Medicine, Pennsylvania 15219</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vaughn%2BS.%2BCooper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cooper%20VS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVaughn%2BS.%2BCooper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/8/2583" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460440" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460440/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2583.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460440" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460440/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2583.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460440" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460440/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2583.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460440" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460440/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/8/2583.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460440" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460440/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Mutations are the ultimate source of variation used for evolutionary adaptation, while also being predominantly deleterious and a source of genetic disorders. Understanding the rate of insertion-deletion mutations (indels) is essential to understanding evolutionary processes, especially in coding regions, where such mutations can disrupt production of essential proteins. Using direct estimates of indel rates from 14 phylogenetically diverse eukaryotic and bacterial species, along with measures of standing variation in such species, we obtain results that imply an inverse relationship of mutation rate and effective population size. These results, which corroborate earlier observations on the base-substitution mutation rate, appear most compatible with the hypothesis that natural selection reduces mutation rates per effective genome to the point at which the power of random genetic drift (approximated by the inverse of effective population size) becomes overwhelming. Given the substantial differences in DNA metabolism pathways that give rise to these two types of mutations, this consistency of results raises the possibility that refinement of other molecular and cellular traits may be inversely related to species-specific levels of random genetic drift.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/insertion-deletion-mutation-rate" class="hw-term hw-article-keyword hw-article-keyword-insertion-deletion-mutation-rate" rel="nofollow">insertion-deletion mutation rate</a></li><li class="kwd"><a href="/keyword/mutation-rate-evolution" class="hw-term hw-article-keyword hw-article-keyword-mutation-rate-evolution" rel="nofollow">mutation-rate evolution</a></li><li class="kwd"><a href="/keyword/drift-barrier" class="hw-term hw-article-keyword hw-article-keyword-drift-barrier" rel="nofollow">drift barrier</a></li><li class="kwd"><a href="/keyword/mutation-accumulation" class="hw-term hw-article-keyword hw-article-keyword-mutation-accumulation" rel="nofollow">mutation accumulation</a></li></ul><p id="p-4">Mutations are a double-edged sword in all organisms, constituting the ultimate source of variation used for evolutionary adaptation, while also being predominantly deleterious and a source of genetic disorders. Hence, researchers have long sought the primary factors governing mutation-rate evolution. Some have argued that the mutation rate of an organism reflects a balance between the deleterious effect of mutations and physiological limitations, with further refinement of replication fidelity limiting the speed of DNA synthesis necessary for efficient daughter-cell production (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Drake 1991</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sniegowski <em>et al.</em> 2000</a>). However, replication fidelity can be improved without a significant decrease in doubling time (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Loh <em>et al.</em> 2010</a>), and prokaryotes undergo high cell-division rates and have low mutation rates (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Drake 1991</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lynch 2010</a>), suggesting that replication fidelity does not limit the rate of daughter-cell production. Furthermore, because there is no negative correlation between cell-division rate and genome size (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Mira <em>et al.</em> 2001</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Vieira-Silva <em>et al.</em> 2010</a>), and the reverse may even be true in bacteria (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lynch and Marinov 2015</a>), cell-division rates do not appear to be limited by the amount of DNA synthesized. Thus, alternative forces may govern mutation-rate evolution.</p><p id="p-5">A general relationship describing mutation-rate variation was proposed by <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Drake <em>et al.</em> (1998)</a>, who suggested that the mutation rate per nucleotide site scales inversely with genome size in bacteria and unicellular eukaryotes, such that there is a constant ∼0.003 mutations per haploid genome per cell division. However, as direct estimates of mutation rates for additional organisms became available, the general relationship between genome size and mutation rate became less apparent, even when scaled to the number of cell divisions per generation in multicellular species (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Lynch 2010</a>).</p><p id="p-6">In a previous analysis, we found a relationship between the base-substitution mutation rate per site per generation (<em>u<sub>bs</sub></em>) multiplied by the amount of functional DNA in a genome (<em>G<sub>e</sub></em>, approximated by proteome size), and the power of random genetic drift, which is inversely proportional to the effective population size (<em>N<sub>e</sub></em>) (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> 2012a</a>). Because mutations are generally deleterious, this finding suggested that selection operates to reduce genome-wide mutation rates by refining DNA replication fidelity and repair until further improvements are too inconsequential to overcome the power of random genetic drift (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sniegowski and Raynes 2013</a>). This result is consistent with the drift-barrier hypothesis (DBH), which proposes that natural selection operates to improve molecular and cellular traits until the selective advantage of a beneficial mutation refining the trait is so miniscule that the probability of it being fixed is essentially the same as that for neutral mutations (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lynch 2011</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> 2012a</a>).</p><p id="p-7">While the negative correlation between <em>u<sub>bs</sub>G<sub>e</sub></em> and <em>N</em><sub>e</sub> is consistent with expectations from population-genetic theory, there is a potential issue of circularity when correlating these factors, as the estimation of <em>N</em><sub>e</sub> relies indirectly on the estimation of <em>u<sub>bs</sub></em> (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> 2012a</a>). Although we presented an analysis suggesting that the correlated parameters are not likely to be the primary factor in the observed relationship (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> 2012a</a>), and provide another one here (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FileS1.pdf">File S1</a>), a more independent analysis is desirable, and, given the amount of data that has accumulated, it is time to go beyond a study that simply considers base-substitution mutations. Here, we present the rate of insertion-deletion mutation (indel) events (<em>u<sub>id</sub></em>) per site per generation across eight eukaryotic and seven bacterial species, while also providing genome-wide estimates of <em>u<sub>bs</sub></em> and <em>u<sub>id</sub></em> from three new bacterial mutation-accumulation studies. These data continue to support a negative correlation between the genome-wide mutation rate and <em>N</em><sub>e</sub>.</p><p id="p-8">The DBH postulates that genetic drift determines the limit of adaptive molecular refinement that can be achieved for any trait, including those that determine the rate of indels. Indels are a class of mutations separate from base substitutions, differing in how they originate. Indels generally arise from strand slippage or double-strand breaks, whereas base-substitution mutations originate primarily from base misincorporation or biochemical alteration. Furthermore, there are major differences in how the two mutation types are repaired. Base-substitution mutations are often reversed by enzymes such as DNA photolyases and alkyl transferases, which do not require DNA incision and synthesis (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Sancar <em>et al.</em> 2004</a>), or are identified by glycosylases in base-excision repair (BER) pathways, and repaired by incision and DNA-gap filling (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Krokan and Bjoras 2013</a>). On the other hand, indel mutations are not surveyed by BER, but are repaired primarily by nucleotide-excision repair (NER), which has broad substrate specificity, and is used to excise bulky lesions arising from the insertion or deletion of nucleotides (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Morita <em>et al.</em> 2010</a>). Although the mismatch-repair (MMR) pathway can operate on both base-substitution mutations and indels, MMR-deficient strains of <em>Escherichia coli</em> and <em>Caenorhabditis elegans</em> exhibit a significantly greater elevation of the indel mutation rate relative to that for base substitutions, providing further evidence for the differential treatment of mutation types by DNA-repair pathways (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Denver <em>et al.</em> 2005</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lee <em>et al.</em> 2012</a>). Furthermore, depending on the type of mismatch and local sequence context, the error rates of different polymerases are highly variable between indel and base-substitution mutations (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">McCulloch and Kunkel 2008</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kunkel 2009</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sung <em>et al.</em> 2015</a>). In summary, because the enzymes influencing base-substitution and indel mutation rates differ (and shared enzymes differ in the spectrum of repaired premutations), a focus on the indel mutation rate provides a means of testing the validity of the DBH that is substantially independent biologically (and essentially fully independent in terms of investigator sampling) of that used to extrapolate measures of the power of random genetic drift.</p><p id="p-9">Selection operates to refine DNA replication fidelity and repair when the genome-wide deleterious load confers a discernable fitness disadvantage on an organism (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kimura 1967</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">1983</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Lynch 2010</a>), and the contributions of indel and base substitution mutations to genome-wide deleterious load differ in two ways. First, the effects of base substitutions in coding regions are highly variable (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Eyre-Walker and Keightley 2007</a>), and some base substitutions may not have any effect on organismal fitness, which may create some uncertainties in quantifying the effective genome size (<em>G<sub>e</sub></em>), thereby reducing the correlation observed between <em>u<sub>bs</sub>G<sub>e</sub></em> and <em>N</em><sub>e</sub> (<a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> 2012a</a>). On the other hand, most indel mutations that arise in protein-coding genes will generate a frame-shift mutation, interfering with gene function, and having a direct effect on organismal fitness. Because such indels are generally deleterious, selection is then expected to more efficiently fine tune the rate at which indels arise, and, if the DBH holds true, this should yield a close correlation between <em>u<sub>id</sub>G<sub>e</sub></em> and <em>N</em><sub>e</sub>. Second, base-substitutions are generally limited to single nucleotides, while indels may involve many base pairs. Although this might suggest that indels have a larger effect than base substitutions, single-base pair indels and gene-sized indels both result in gene disruption, thus generating more similar fitness effects regardless of the indel length. In fact, single base-pair indels in coding DNA may generate malformed gene products that require degradation, which might be even more harmful than entire gene deletions. Because the number of indel events, and not the size of indels, determines the genome-wide deleterious burden, we define the parameter <em>u<sub>id</sub></em> to be the number of indel mutation events per site per generation, and use this parameter to test the DBH.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-10">To examine the effect of genetic drift on mutation-rate evolution, it is necessary to derive accurate estimates of the mutation rate and genetic diversity across phylogenetically diverse organisms. Whole-genome sequencing (WGS) has greatly improved our ability to estimate such parameters. Highly accurate measurements of <em>u<sub>bs</sub></em> and <em>u<sub>id</sub></em> can be obtained through WGS of mutation-accumulation (MA) lines, in which repeated single-organism bottlenecking minimizes the efficiency of selection, allowing for the accumulation of all but the most deleterious mutations (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Denver <em>et al.</em> 2009</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ossowski <em>et al.</em> 2010</a>; <a id="xref-ref-56-6" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> 2012a</a>, <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">2012b</a>, <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">2015</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Schrider <em>et al.</em> 2013</a>). Along with data from prior MA studies, this study contains MA data from four new MA experiments. For new bacterial MA species, ∼100 independent MA lines were initiated from a single founder colony. The new strains used were as follows: <em>Agrobacterium tumefaciens</em> str. C58, <em>Staphylococcus epidermidis</em> ATCC 12228, and <em>Vibrio cholerae</em> 2740-80.</p><p id="p-11">Depending on the speed of growth, a single colony from each MA line was isolated and transferred to a fresh plate every 1–3 d over the course of the experiment. The bottlenecking process ensures that mutations accumulate in an effectively neutral fashion (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kibota and Lynch 1996</a>). After each transfer, the original plate was retained as a backup plate at 4°. If the destination plate was contaminated, or if a single colony could not be picked, a single colony was transferred from the last 4° backup plate.</p><p id="p-12">To estimate the generation times that occurred between each transfer, every 2 wk, an entire colony from five randomly selected MA lines was transferred to 1 × PBS saline buffer. These were vortexed, serially diluted, and replated. Cell density was calculated from viable cell counts in both the growth conditions used throughout the bottleneck process as well as growth conditions at 4°. The total number of generations for each MA line was calculated by the average number of cell divisions per transfer multiplied by the total number of transfers. If backup plates were used, the average number of cell divisions at 4° was used in place of the average number of cell divisions per bottleneck at standard growth temperatures.</p><p id="p-13">The average number of cell divisions across the MA are as follows (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS1.pdf">Dataset S1</a>): <em>A. tumefaciens</em>, 5819; <em>Bacillus subtilis</em>, 5078 (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Sung <em>et al.</em> 2015</a>); <em>E. coli</em>, 4246 (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lee <em>et al.</em> 2012</a>); <em>Mesoplasma florum</em>, 2351 (<a id="xref-ref-56-7" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> 2012a</a>); <em>S. epidermidis</em>, 7170, and <em>V. cholerae</em>, 6453. The average number of generations used for reanalysis of the <em>C. elegans</em> MA study was 250 (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Denver <em>et al.</em> 2009</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS2.pdf">Dataset S2</a>).</p><p id="p-14">DNA extraction of MA lines was done using the wizard DNA extraction kit (Promega) or lysis media (CTAB or SDS) followed by phenol/chloroform extractions to Illumina library standards. Then, 101-bp paired-end Illumina (Illumina Hi-Seq platform) sequencing was applied to randomly selected MA lines of <em>A. tumefaciens</em>, <em>S. epidermidis</em>, and <em>V. cholerae</em>. Each MA line was sequenced to a coverage depth of ∼100 ×, with an average library fragment size (distance between paired-end reads) of ∼175 bp. The paired-end reads for each MA line were individually mapped against the reference genome (assembly and annotation available from the National Center for Biotechnology Information, <a href="https://www.ncbi.nlm.nih.gov">https://www.ncbi.nlm.nih.gov</a>) using two separate alignment algorithms: BWA v0.7.4 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li and Durbin 2009</a>) and NOVOALIGN v2.08.02 (available at <a href="http://www.novocraft.com">www.novocraft.com</a>). The resulting pileup files were converted to SAM format using SAMTOOLS v0.1.18 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li <em>et al.</em> 2009</a>). Using in-house perl scripts, the alignment information was further parsed to generate forward and reverse mapping information at each site, resulting in a configuration of eight numbers for each line (A, a, C, c, G, g, T, and t), corresponding to the number of reads mapped at each genomic position in the reference sequence. A separate file was also generated to display sites that had indel calls from the two alignment algorithms. Mutation calling was performed using a consensus method (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2008</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Denver <em>et al.</em> 2009</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Ossowski <em>et al.</em> 2010</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Lee <em>et al.</em> 2012</a>; <a id="xref-ref-56-8" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> 2012a</a>, <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">2012b</a>, <a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">2015</a>).</p><p id="p-15">A random subset of base-substitutions mutations called using these methods have been previously validated in <em>E. coli</em> and <em>B. subtilis</em> MA lines using fluorescent sequencing technology at the Indiana Molecular Biology Institute at Indiana University (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Lee <em>et al.</em> 2012</a>; <a id="xref-ref-58-5" class="xref-bibr" href="#ref-58">Sung <em>et al.</em> 2015</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS3.pdf">Dataset S3</a>).</p><p id="p-16">To verify indel mutations, we designed 38 primer sets to PCR amplify 300–500 bp regions surrounding the putative indel mutation in the <em>B. subtilis</em> MA lines (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS4.pdf">Dataset S4</a>). All 29/29 short indels (&lt; 10 bp) were directly confirmed using standard fluorescent sequencing technology. Two out of nine large indels (&gt; 10 bp) were confirmed through sizing of the PCR product on gel electrophoresis. The remaining seven large indels did not amplify. For all cases, the indel was also confirmed to be absent in one other line without the mutation.</p><p id="p-17">To calculate the base-substitution mutation rate per cell division for each line, we used the following equation:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2583/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2583/embed/mml-math-1.gif"/></noscript></span></span>where <em>u<sub>bs</sub></em> is the base-substitution mutation rate (per nucleotide site per generation), <em>m</em> is the number of observed base substitutions, <em>n</em> is the number of nucleotide sites analyzed, and <em>T</em> is the number of generations that occurred in the mutation-accumulation study. The SE for an individual line is calculated using (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Denver <em>et al.</em> 2004</a>, <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">2009</a>):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2583/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2583/embed/mml-math-2.gif"/></noscript></span></span>The total SE of base-substitution mutation rate is given by the SD of the mutation rates across all lines (<em>s</em>) divided by the square root of the number of lines analyzed (<em>N</em>).<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2583/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2583/embed/mml-math-3.gif"/></noscript></span></span>The same calculation was used to calculate indel mutation rate, with <em>u<sub>bs</sub></em> replaced with <em>u<sub>id</sub></em>.</p><div id="sec-2" class="subsection"><h3>Data availability</h3><p id="p-18">Illumina DNA sequences for the MA lines used in this study are deposited under the following Bioprojects: <em>A. tumefaciens</em> PRJNA256312, <em>B. subtilis</em> PRJNA256312, <em>M. florum</em> PRJNA256337, <em>S. epidermidis</em> PRJNA256338, and <em>V. cholerae</em> PRJNA256339.</p><p id="p-19"><a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FileS1.pdf">File S1</a> contains detailed descriptions of eukaryotic <em>u<sub>id</sub></em> estimates, as well as calculations for <em>G<sub>e</sub></em>, <em>G<sub>nc</sub></em>, θ<sub>s</sub>, π<sub>s</sub>, and phylogenetic independent contrasts for both eukaryotic and prokaryotic organisms. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS1.pdf">Figure S1</a> contains average depth of sequencing coverage for each MA line in <em>A. tumefaciens</em>, <em>S. epidermidis</em>, and <em>V. cholerae</em>. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS2.pdf">Figure S2</a> displays the similarity in θ<sub>s</sub> when increasing the number of unique alleles analyzed. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS3.pdf">Figure S3</a> shows the frequency distribution of mutant calls across MA lines. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/TableS1.pdf">Table S1</a> contains the calculation for the estimated limit of selection to fix antimutators. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/TableS2.pdf">Table S2</a> contain statistical support for the DBH. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS1.pdf">Dataset S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS2.pdf">Dataset S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS3.pdf">Dataset S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS4.pdf">Dataset S4</a> contain single nucleotide polymorphisms and indels for prokaryotic and eukaryotic organisms generated in this study.</p></div></div><div class="section results" id="sec-3"><h2 class="">Results</h2><p id="p-20">To examine the effect of genetic drift on mutation-rate evolution, it is necessary to derive accurate estimates of the mutation rate and genetic diversity across phylogenetically diverse organisms. WGS has greatly improved our ability to estimate such parameters. Highly accurate measurements of <em>u<sub>bs</sub></em> and <em>u<sub>id</sub></em> can be obtained through WGS of MA lines, in which repeated single-organism bottlenecking minimizes the efficiency of selection, allowing for the accumulation of all but the most deleterious mutations (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2008</a>; <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Denver <em>et al.</em> 2009</a>; <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Ossowski <em>et al.</em> 2010</a>; <a id="xref-ref-56-9" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> 2012a</a>, <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">2012b</a>, <a id="xref-ref-58-6" class="xref-bibr" href="#ref-58">2015</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Schrider <em>et al.</em> 2013</a>).</p><p id="p-21">The power of genetic drift is related to the inverse of the effective population size [1/<em>N<sub>e</sub></em> for haploids, 1/(2<em>N<sub>e</sub></em>) for diploids]. Under the assumption of neutrality, the effective population size (<em>N<sub>e</sub></em>) can be estimated from the average nucleotide heterozygosity at silent sites in natural populations (π<sub>s</sub>), or as a function of the number of segregating sites in the population (θ<sub>s</sub>), both of which lead to expected values equal to 4<em>N<sub>e</sub>u<sub>bs</sub></em> in diploids and 2<em>N<sub>e</sub>u<sub>bs</sub></em> in haploids (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kimura 1983</a>). For most organisms analyzed in this study, enough WGS data were available to allow calculation of species-specific θ<sub>s</sub> values (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FileS1.pdf">File S1</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). For the remaining species, we pooled large available multilocus-sequence studies to estimate π<sub>s</sub>. In all cases, we set the estimates of θ<sub>s</sub> or π<sub>s</sub> equal to 4<em>N<sub>e</sub>u<sub>bs</sub></em> in diploids (2<em>N<sub>e</sub>u<sub>bs</sub></em> in haploids), and solved for <em>N</em><sub>e</sub> by factoring out <em>u<sub>bs</sub></em>. Because this calculation only involves <em>u<sub>bs</sub></em>, the estimate of <em>N<sub>e</sub></em> is uninfluenced by sampling error in <em>u<sub>id</sub></em>, thus providing an independent trait measurement by which to test the DBH (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FileS1.pdf">File S1</a> for further evaluation of the nonindependence issue).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468945/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468945/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Effective genome size (<em>G<sub>e</sub></em>), indel events per site per generation (<em>u<sub>id</sub></em>), base-substitution mutation rate per generation (<em>u<sub>bs</sub></em>), θ<sub>s</sub> (or π<sub>s</sub>, denoted by *) measurements for population mutation rate (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Watterson 1975</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Tajima 1989</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fu 1995</a>), and estimated effective population size (<em>N<sub>e</sub></em>) for seven prokaryotic and eight eukaryotic organisms (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FileS1.pdf">File S1</a> for details)</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">To provide additional data for testing whether the power of genetic drift constrains the lower limit of indel mutation-rate evolution, we performed MA experiments in <em>A. tumefaciens</em> str. C58, <em>S. epidermidis</em> ATCC 12228, and <em>V. cholerae</em> 2740-80. Each bacterial MA experiment was initiated from multiple lines derived from a single progenitor colony, each of which was repeatedly bottlenecked to accumulate mutations for an average of 5819, 7170, and 6453 generations, respectively (see <em>Materials and Methods</em>; harmonic mean population sizes between transfers were 13.4 (0.1), 12.6 (0.3), and 14.9 (0.2), respectively). Then, 101-bp paired-end WGS was applied to randomly selected MA lines (47 <em>A. tumefaciens</em>, 22 <em>S. epidermidis</em>, and 46 <em>V. cholerae</em> MA lines, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS1.pdf">Dataset S1</a>). The average sequencing coverage depth is greater than 20 × per site across all MA lines surveyed in these organisms (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS1.pdf">Figure S1</a>), and greater than 50 × per site for 93.75% (150/160) of the MA lines, providing high accuracy for measurement of <em>u<sub>bs</sub></em> and <em>u<sub>id</sub></em>. Mutations were called and categorized for each of the three species (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS3.pdf">Dataset S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS4.pdf">Dataset S4</a>), with <em>u<sub>bs</sub></em> and <em>u<sub>id</sub></em> shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>.</p><p id="p-41">To test the DBH, we combined <em>u<sub>bs</sub></em> and <em>u<sub>id</sub></em> from the three bacterial species analyzed in this study with <em>u<sub>bs</sub></em> and <em>u<sub>id</sub></em> from four bacterial and eight eukaryotic MA WGS studies (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS1.pdf">Dataset S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS2.pdf">Dataset S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS3.pdf">Dataset S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS4.pdf">Dataset S4</a>), and also included the same estimates for human derived from WGS of parent-offspring trios. <em>u<sub>id</sub></em> includes all indel events in each of the 15 study species (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FileS1.pdf">File S1</a>). Due to the highly repetitive DNA sequence in eukaryotic genomes, the number of large indels events (&gt; 9 bp) in eukaryotes may be downwardly biased when using WGS methods. Therefore, our estimate of the number of large indel events also includes events identified by comparative genome hybridization arrays for organisms where data were available (<a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2008</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Lipinski <em>et al.</em> 2011</a>). Large indel events only account for 15.0% of total indels events across the study bacteria (76/506, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS4.pdf">Dataset S4</a>), suggesting that any underestimation of the number of large indel events should only have a small effect on <em>u<sub>id</sub></em>.</p><p id="p-42">To determine the genome-wide deleterious burden in each organism associated with indel mutations, we multiplied <em>u<sub>id</sub></em> with <em>G<sub>e</sub></em>, approximating the latter by the proteome size of that organism. A plot of the logs of the two parameters of <em>u<sub>id</sub>G<sub>e</sub></em> and <em>N</em><sub>e</sub> against one another yields a strong negative correlation across all of cellular life (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, <em>r</em><sup>2</sup> = 0.89). Because the power of genetic drift is inversely proportional to <em>N<sub>e</sub></em>, this observation is consistent with the idea that selection operates to reduce mutation rates to a barrier imposed by random genetic drift. Phylogenetic nonindependence may complicate observed relationships between genomic attributes and <em>N</em><sub>e</sub> (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Whitney and Garland 2010</a>). However, the relationship between <em>N</em><sub>e</sub> and <em>u<sub>id</sub>G<sub>e</sub></em> remains robust even after phylogenetic correction (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a>, <em>r</em><sup>2</sup> = 0.83), indicating that the correlation between <em>N</em><sub>e</sub> and <em>u<sub>id</sub>G<sub>e</sub></em> reflects a true biological phenomenon across the Tree of Life.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2583/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between the rate of indel events per generation per effective genome (uidGe) and effective population size (Ne). (A) Regression: log10(uidGe)&#xA0;=&#xA0;2.23(0.48)&#xA0;&#x2013;&#xA0;0.73(0.07)log10Ne (r2&#xA0;=&#xA0;0.89, P&#xA0;=&#xA0;6.81&#xA0;&#xD7;&#xA0;10&#x2212;8, d.f.&#xA0;=&#xA0;13), with SE of parameter estimates shown in parentheses. Blue circles represent bacteria, red circles multicellular eukaryotes, and black circles unicellular eukaryotes, with all data summarized in Table 1. The full list of indel events for analyzed organisms is presented in Dataset S4. Chromosomal distributions of indel events at each site across all mutation-accumulation experiments are shown in Figure S1, A and B. (B) Relationship when adding the number of estimated noncoding sites under purifying selection into the effective genome size (Gc&#xA0;+&#xA0;Gnc) for eukaryotic organisms. Regression: log10[uid(Gc +&#xA0;Gnc)]&#xA0;=&#xA0;3.49(0.66)&#xA0;&#x2013;&#xA0;0.87(0.09)log10Ne (r2&#xA0;=&#xA0;0.87, P&#xA0;=&#xA0;3.13&#xA0;&#xD7;&#xA0;10&#x2212;7, d.f.&#xA0;=&#xA0;13)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1609522365" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between the rate of indel events per generation per effective genome (&lt;em&gt;u&lt;sub&gt;id&lt;/sub&gt;G&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt;) and effective population size (&lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt;). (A) Regression: log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;u&lt;sub&gt;id&lt;/sub&gt;G&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt;)&#xA0;=&#xA0;2.23(0.48)&#xA0;&#x2013;&#xA0;0.73(0.07)log&lt;sub&gt;10&lt;/sub&gt;&lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt; (&lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;&#xA0;=&#xA0;0.89, &lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;6.81&#xA0;&#xD7;&#xA0;10&lt;sup&gt;&#x2212;8&lt;/sup&gt;, d.f.&#xA0;=&#xA0;13), with SE of parameter estimates shown in parentheses. Blue circles represent bacteria, red circles multicellular eukaryotes, and black circles unicellular eukaryotes, with all data summarized in Table 1. The full list of indel events for analyzed organisms is presented in Dataset S4. Chromosomal distributions of indel events at each site across all mutation-accumulation experiments are shown in Figure S1, A and B. (B) Relationship when adding the number of estimated noncoding sites under purifying selection into the effective genome size (&lt;em&gt;G&lt;sub&gt;c&lt;/sub&gt;&lt;/em&gt;&#xA0;+&#xA0;&lt;em&gt;G&lt;sub&gt;nc&lt;/sub&gt;&lt;/em&gt;) for eukaryotic organisms. Regression: log&lt;sub&gt;10&lt;/sub&gt;[&lt;em&gt;u&lt;/em&gt;id(&lt;em&gt;G&lt;sub&gt;c&lt;/sub&gt;&lt;/em&gt; +&#xA0;&lt;em&gt;G&lt;sub&gt;nc&lt;/sub&gt;&lt;/em&gt;)]&#xA0;=&#xA0;3.49(0.66)&#xA0;&#x2013;&#xA0;0.87(0.09)log&lt;sub&gt;10&lt;/sub&gt;&lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt; (&lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;&#xA0;=&#xA0;0.87, &lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;3.13&#xA0;&#xD7;&#xA0;10&lt;sup&gt;&#x2212;7&lt;/sup&gt;, d.f.&#xA0;=&#xA0;13).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2583/F1.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/8/2583/F1.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2583/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2583/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468891" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-43" class="first-child">Relationship between the rate of indel events per generation per effective genome (<em>u<sub>id</sub>G<sub>e</sub></em>) and effective population size (<em>N<sub>e</sub></em>). (A) Regression: log<sub>10</sub>(<em>u<sub>id</sub>G<sub>e</sub></em>) = 2.23(0.48) – 0.73(0.07)log<sub>10</sub><em>N<sub>e</sub></em> (<em>r</em><sup>2</sup> = 0.89, <em>P</em> = 6.81 × 10<sup>−8</sup>, d.f. = 13), with SE of parameter estimates shown in parentheses. Blue circles represent bacteria, red circles multicellular eukaryotes, and black circles unicellular eukaryotes, with all data summarized in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>. The full list of indel events for analyzed organisms is presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/DatasetS4.pdf">Dataset S4</a>. Chromosomal distributions of indel events at each site across all mutation-accumulation experiments are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS1.pdf">Figure S1, A and B</a>. (B) Relationship when adding the number of estimated noncoding sites under purifying selection into the effective genome size (<em>G<sub>c</sub></em> + <em>G<sub>nc</sub></em>) for eukaryotic organisms. Regression: log<sub>10</sub>[<em>u</em>id(<em>G<sub>c</sub></em> + <em>G<sub>nc</sub></em>)] = 3.49(0.66) – 0.87(0.09)log<sub>10</sub><em>N<sub>e</sub></em> (<em>r</em><sup>2</sup> = 0.87, <em>P</em> = 3.13 × 10<sup>−7</sup>, d.f. = 13).</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2583/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between indel events per site per generation (uidGe) and effective population size (Ne) after phylogenetic correction. (A) Standardized phylogenetically independent contrasts performed using Compare (Martins 2004), and the PDAP module in Mesquite (Garland et al. 1993), with branch lengths of 1.0. The regression equation of the contrasts through the origin is: uidGe&#xA0;=&#xA0;&#x2013;0.60(0.07)Ne (r2&#xA0;=&#xA0;0.83, P&#xA0;=&#xA0;1.28&#xA0;&#xD7;&#xA0;10&#x2212;6, d.f.&#xA0;=&#xA0;13), with SE in parentheses. (B) Phylogenetic tree showing the relationship between organisms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1609522365" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between indel events per site per generation (&lt;em&gt;u&lt;sub&gt;id&lt;/sub&gt;G&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt;) and effective population size (&lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt;) after phylogenetic correction. (A) Standardized phylogenetically independent contrasts performed using Compare (Martins 2004), and the PDAP module in Mesquite (Garland &lt;em&gt;et al.&lt;/em&gt; 1993), with branch lengths of 1.0. The regression equation of the contrasts through the origin is: &lt;em&gt;u&lt;sub&gt;id&lt;/sub&gt;G&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt;&#xA0;=&#xA0;&#x2013;0.60(0.07)&lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt; (&lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;&#xA0;=&#xA0;0.83, &lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;1.28&#xA0;&#xD7;&#xA0;10&lt;sup&gt;&#x2212;6&lt;/sup&gt;, d.f.&#xA0;=&#xA0;13), with SE in parentheses. (B) Phylogenetic tree showing the relationship between organisms.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2583/F2.medium.gif" width="440" height="198"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/8/2583/F2.medium.gif" width="440" height="198"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2583/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2583/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468760" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-44" class="first-child">Relationship between indel events per site per generation (<em>u<sub>id</sub>G<sub>e</sub></em>) and effective population size (<em>N<sub>e</sub></em>) after phylogenetic correction. (A) Standardized phylogenetically independent contrasts performed using Compare (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Martins 2004</a>), and the PDAP module in Mesquite (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Garland <em>et al.</em> 1993</a>), with branch lengths of 1.0. The regression equation of the contrasts through the origin is: <em>u<sub>id</sub>G<sub>e</sub></em> = –0.60(0.07)<em>N<sub>e</sub></em> (<em>r</em><sup>2</sup> = 0.83, <em>P</em> = 1.28 × 10<sup>−6</sup>, d.f. = 13), with SE in parentheses. (B) Phylogenetic tree showing the relationship between organisms.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-4"><h2 class="">Discussion</h2><p id="p-45">Because the DBH makes general predictions about the pattern of molecular and cellular evolution across the Tree of Life, because our focus is on one of the central determining factors in the evolutionary process (the mutation rate), and because the patterns appear so strong, it is essential to consider the range of factors that might give rise to the observed statistical relationships, and also to alternative evolutionary hypotheses for them. We first consider three issues with respect to estimating the key parameters <em>N<sub>e</sub></em>, <em>u<sub>bs</sub></em>, <em>u<sub>id</sub></em>, and <em>G<sub>e</sub></em>, and then elaborate on the significance and implications of the relationship between <em>u<sub>id</sub>G<sub>e</sub></em> and <em>N</em><sub>e</sub> for our understanding of molecular evolution.</p><p id="p-46">First, we address the estimation of <em>N<sub>e</sub></em>, one of the most difficult issues in empirical population genetics. Because populations fluctuate in density over time, any estimate of <em>N<sub>e</sub></em> must reflect a long-term average, presumably approximating a harmonic mean, not the immediate population state. Because evolution is a long-term process, however, the mean is most relevant to the issues being examined herein. Recent selective sweeps or population bottlenecks can transiently modify levels of genetic variation at individual loci (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Charlesworth 2009</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Karasov <em>et al.</em> 2010</a>), introducing noise into any estimates of <em>N<sub>e</sub></em> derived from limited numbers of genetic loci, but this would reduce the strength of any true underlying correlation between the rate of mutation (<em>u<sub>id</sub>G<sub>e</sub></em>), and long-term <em>N</em><sub>e</sub>, <em>i.e.</em>, would operate against our ability to detect the expected signal of the DBH.</p><p id="p-47">Such effects are especially likely in asexual species, where the possibility of reduced recombination might subject many neutral nucleotide sites to the effects of selection on nearby, linked sites. Thus, to minimize sampling error, wherever possible, we have relied upon genome-wide sampling of the number of segregating sites to obtain a low-variance estimator of <em>N</em><sub>e</sub><em>u</em> from observations on silent sites (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Watterson 1975</a>). The utilization of an average θ<sub>s</sub> across a large number of nucleotide sites and individual isolates reduces the effects of evolutionary sampling variance associated with chromosomally localized and population-specific sweeps arising within individual species (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fu and Li 1993</a>). Using available genomic data, we calculated θ<sub>s</sub> across a large number of within-species genotypic isolates, excluding nearly identical lab strains that originated from the same individual (see <em>Materials and Methods</em>). Although no estimates of silent-site diversity (the source of <em>N</em><sub>e</sub> estimates) are without error, estimates derived from segregating polymorphic sites across large-scale genomic data sets appear quite robust (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS2.pdf">Figure S2</a>). Moreover, should the levels of variation sampled in our various study species reflect recent events, to which mutation-rate evolution has not had adequate time to respond (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Brandvain and Wright 2016</a>), this would only introduce noise into the relationship between effective population size and mutation rates.</p><p id="p-48">Second, as we have noted earlier, there is some concern that correlations between estimates of mutation rates and <em>N<sub>e</sub></em> could, in part, be spurious artifacts resulting from the use of estimates of <em>N<sub>e</sub></em> obtained by dividing measures of standing variation at silent-sites by <em>u<sub>bs</sub></em> (<a id="xref-ref-56-12" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> 2012a</a>). If the sampling variance of <em>u<sub>bs</sub></em> is substantial enough, this could lead to a negative correlation between the observed <em>u<sub>bs</sub></em> and extrapolated <em>N<sub>e</sub></em> estimates, and, if there were a sampling covariance between <em>u<sub>bs</sub></em> and <em>u<sub>id</sub></em>, this could carry over into the current study. In the Supplemental Material (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS6.pdf">Figure S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS7.pdf">Figure S7</a>), we provide complementary analyses to that in <a id="xref-ref-56-13" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> (2012a)</a>, indicating that the sampling variance of <em>u<sub>bs</sub></em> from WGS-MA studies is not large enough to explain the negative correlation previously seen between <em>u<sub>bs</sub></em> and <em>N<sub>e</sub></em> estimates. Because <em>u<sub>bs</sub></em> and <em>u<sub>id</sub></em> are measured by different methods, the sampling covariance between these two measures is expected to be negligible. We emphasize that it is the sampling variance, not the evolutionary variance, that is of concern here. The variance of the log-scaled values of <em>u<sub>bs</sub></em> would have to exceed the log-scaled values of <em>N<sub>e</sub></em> by ∼two orders of magnitude in order to create the negative correlations that we observe (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FileS1.pdf">File S1</a>). As an extreme way of looking at the situation, if silent-site variation were constant across all taxa, and the parametric values of mutation rates and <em>N<sub>e</sub></em> were obtained without error, the only explanation for the data would be a true underlying negative evolutionary covariance between the two features. In fact, there is a marginal negative correlation between estimates of π<sub>s</sub> and <em>u<sub>bs</sub></em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS6.pdf">Figure S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FigureS7.pdf">Figure S7</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/TableS2.pdf">Table S2</a>), further bolstering the idea that <em>u<sub>bs</sub></em> and <em>u<sub>id</sub></em> decline evolutionarily as <em>N<sub>e</sub></em> increases.</p><p id="p-49">Third, the DBH proposes that the strength of selection operating to reduce the indel mutation rate is based upon the total indel deleterious mutational load, <em>i.e.</em>, the product of the mutational rate of appearance of indels at individual nucleotide sites (<em>u<sub>id</sub></em>), and the number of sites under selective constraint in the genome (<em>G<sub>e</sub></em>, approximated by the proteome size of the organism). However, some noncoding DNA (<em>e.g.</em>, noncoding functional RNAs, and <em>cis</em>-regulatory units in untranslated regions or introns) is certainly under selective constraint, with mutations at these sites increasing the deleterious mutational load. Thus, it can be argued that the estimated number of nucleotides affecting fitness (<em>G<sub>e</sub></em>) scales differently than the protein-coding region of the genome, particularly in larger eukaryotic genomes with a considerable number of noncoding sites (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Halligan <em>et al.</em> 2004</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Siepel <em>et al.</em> 2005</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Halligan and Keightley 2006</a>). Difficulties can arise when estimating the proportion of noncoding DNA that is under selective constraint (<em>G<sub>nc</sub></em>), as the estimated number of such sites can vary greatly depending on the model used to define noncoding DNA, and the identification of conserved noncoding DNA is highly sensitive to the available phylogeny (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Siepel <em>et al.</em> 2005</a>). Nevertheless, if we sum the estimated total amount of noncoding DNA under selective constraint (<em>G<sub>nc</sub></em>, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/FileS1.pdf">File S1</a>) with that of coding DNA (<em>G<sub>c</sub></em>), we find that <em>u<sub>id</sub></em>(<em>G<sub>c</sub></em> + <em>G<sub>nc</sub></em>) and <em>N<sub>e</sub></em> remain highly correlated (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>, <em>r</em><sup>2</sup> = 0.87), simply because the fraction of functional noncoding DNA increases with the total amount of coding DNA.</p><p id="p-50">We currently adhere to the DBH as an explanation for the phylogenetic pattern of mutation-rate variation primarily because it has been difficult to reconcile the patterns with alternative hypotheses. In the introduction, we provided arguments as to why selection for replication speed appears to be unlikely to explain a negative correlation between mutation rates and population size in unicellular species, and, in multicellular species, the simultaneous deployment of hundreds to thousands of origins of replication makes such an explanation even more unlikely. Nor does a general constraint on replication fidelity explain the data.</p><p id="p-51">A second potential explanation for variation in the per-generation mutation rate is that it is driven largely by variation in numbers of germline cell divisions (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ness <em>et al.</em> 2012</a>), but this cannot be reconciled with the fact that the base-substitution mutation rate scales negatively with <em>N<sub>e</sub></em> in analyses entirely restricted to unicellular species (<a id="xref-ref-56-14" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> 2012a</a>). In all such species, there is one cell division per generation, and yet the base-substitution mutation rate per site per cell division ranges from ∼10<sup>−11</sup> in <em>Paramecium tetraurelia</em> (<a id="xref-ref-57-6" class="xref-bibr" href="#ref-57">Sung <em>et al.</em> 2012b</a>) to ∼10<sup>−8</sup> in <em>M. florum</em> (<a id="xref-ref-56-15" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> 2012a</a>). Similarly, the number of indel mutational events per site per cell division differs by over two orders of magnitude across unicellular organisms (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), and the negative regression with <em>N<sub>e</sub></em> remains significant when confined to unicellular species (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, <em>r</em><sup>2</sup> = 0.66, <em>P</em> = 0.003).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2583/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between the rate of indel events per site per generation (uid), and the base-substitution mutation rate per site per generation (ubs). Regression: log10(uid)&#xA0;=&#xA0;&#x2013;1.56(0.74)&#xA0;+&#xA0;0.91(0.08) log10ubs (r2&#xA0;=&#xA0;0.90, P&#xA0;=&#xA0;4.13&#xA0;&#xD7;&#xA0;10&#x2212;8, d.f.&#xA0;=&#xA0;13). SE measurements are shown in parentheses. Blue circles represent eubacteria, red circles multicellular eukaryotes, and black circles unicellular eukaryotes, with all data summarized in Table 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1609522365" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between the rate of indel events per site per generation (&lt;em&gt;u&lt;sub&gt;id&lt;/sub&gt;&lt;/em&gt;), and the base-substitution mutation rate per site per generation (&lt;em&gt;u&lt;sub&gt;bs&lt;/sub&gt;&lt;/em&gt;). Regression: log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;u&lt;sub&gt;id&lt;/sub&gt;&lt;/em&gt;)&#xA0;=&#xA0;&#x2013;1.56(0.74)&#xA0;+&#xA0;0.91(0.08) log&lt;sub&gt;10&lt;/sub&gt;&lt;em&gt;u&lt;sub&gt;bs&lt;/sub&gt;&lt;/em&gt; (&lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;&#xA0;=&#xA0;0.90, &lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;4.13&#xA0;&#xD7;&#xA0;10&lt;sup&gt;&#x2212;8&lt;/sup&gt;, d.f.&#xA0;=&#xA0;13). SE measurements are shown in parentheses. Blue circles represent eubacteria, red circles multicellular eukaryotes, and black circles unicellular eukaryotes, with all data summarized in Table 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2583/F3.medium.gif" width="440" height="359"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/8/2583/F3.medium.gif" width="440" height="359"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2583/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2583/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468894" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-52" class="first-child">Relationship between the rate of indel events per site per generation (<em>u<sub>id</sub></em>), and the base-substitution mutation rate per site per generation (<em>u<sub>bs</sub></em>). Regression: log<sub>10</sub>(<em>u<sub>id</sub></em>) = –1.56(0.74) + 0.91(0.08) log<sub>10</sub><em>u<sub>bs</sub></em> (<em>r</em><sup>2</sup> = 0.90, <em>P</em> = 4.13 × 10<sup>−8</sup>, d.f. = 13). SE measurements are shown in parentheses. Blue circles represent eubacteria, red circles multicellular eukaryotes, and black circles unicellular eukaryotes, with all data summarized in <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-53">A third hypothesis for mutation-rate evolution is that selection is effective enough to reduce the error rate to the point at which the physical laws of thermodynamics take over (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Kimura 1967</a>). However, it is difficult to reconcile this argument with the data now showing that mutation rates vary by three orders of magnitude, as there are no known mechanisms by which basic biophysical features (such as diffusion coefficients and stochastic molecular motion) would vary by this degree among the cytoplasms of different taxa. There is, of course, the issue of evolved differences in the biochemical features and efficiency of operation of the proteins involved in replication and repair. However, this type of variation is in the explanatory domain of the DBH. The DBH postulates that replication fidelity is typically not at the maximum possible level of refinement, but just the lowest level possible under the prevailing level of random genetic drift, which varies substantially among lineages.</p><p id="p-54">That a decline in replication fidelity should decline with decreasing effective population size appears to be a unique prediction of the DBH. Although other theoretical work has been done on mutation-rate evolution, in no case is this type of scaling obviously predicted (acknowledging that this has not been a central focus of such work). For example, allowing for a role of beneficial mutations, <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Kimura (1967)</a> and <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Leigh (1970)</a> suggested that the long-term rate of adaptation is maximized when the genome-wide mutation rate equals the rate of population fixation of beneficial mutations. The precise predictions of this hypothesis are not entirely clear, but because mutations arise at a higher rate in large populations, and, if beneficial, fix with higher probabilities, a positive association between the mutation rate and <em>N<sub>e</sub></em> seems to be implied. A rather different model argues that populations should evolve genome-wide mutation rates equal to the average effect of a deleterious mutation <span class="sc">(</span><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Orr 2000</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Johnson and Barton 2002</a>), which seems to imply an optimal mutation rate independent of population size (unless one wishes to postulate an association between average mutational effect and <em>N<sub>e</sub></em>, for which we are unaware of any evidence).</p><p id="p-55">The DBH proposes that new alleles that reduce the genome-wide indel mutation rate (<em>i.e.</em>, anti-mutators) can be promoted by selection only if they provide a significant enough advantage to offset the power of genetic drift. The average selective effect of an antimutator or mutator allele (which operate opposite to each other) can be approximated by <em>st·</em>∆<em>U<sub>id</sub></em>, with ∆<em>U<sub>id</sub></em> representing the change in the genome-wide indel mutation rate with respect to the population mean rate, <em>s</em> being the average reduction in fitness per mutation (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Lynch 2010</a>), and <em>t</em> being the number of generations a mutation remains associated with its mutator genetic background (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Lynch 2011</a>). ∆<em>U<sub>id</sub></em> can be approximated by the change in the indel mutation rate over the effective genome, or ∆<em>u<sub>id</sub>G<sub>e</sub></em> (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Lynch 2011</a>). By setting <em>st</em>∆<em>u<sub>id</sub>G<sub>e</sub></em> equal to the power of random genetic drift [1/<em>N<sub>e</sub></em> for haploids, 1/(2<em>N<sub>e</sub></em>) for diploids], we can acquire some sense of the average reduction in the indel mutation rate that is required for the power of selection to exceed power of genetic drift. Using estimates of an average value of the selective coefficient (<em>s</em> = 0.01) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lynch <em>et al.</em> 1999</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Eyre-Walker and Keightley 2007</a>), and assuming that free recombination unlinks mutation-rate modifier alleles from their background every ∼2 generations in sexually outcrossing species (<em>t</em> = 2) (<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Lynch 2010</a>), solving <em>st</em>∆<em>u<sub>id</sub>G<sub>e</sub></em> = 1/<em>N<sub>e</sub></em> [= 1/(2<em>N<sub>e</sub></em>) for diploids] for ∆<em>u<sub>id</sub></em> suggests that the average antimutator must reduce the indel mutation rate by greater than ∼0.1–1% in most organisms (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1/TableS1.pdf">Table S1</a>) in order to be promoted by selection. One major limitation of this kind of analysis is that values of <em>s</em> and <em>t</em> are not well known, and are likely vary across organisms. A second and equally important caveat is that the prior analysis assumes that mutator and antimutator alleles arise with equal frequency. Owing to the high level of refinement of the replication and repair machinery, it seems much more likely that mutations involving the components of such machinery will increase rather than decrease the mutation rate. This will push the equilibrium mutation rate to higher levels than expected (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lynch 2008</a>), although without quantitative information on such bias, it is difficult to determine the exact position at which the mutation rate will stall.</p><p id="p-56">Finally, we note that because recombination unlinks alleles from their genetic background, the capacity of selection to enhance replication fidelity is ultimately a function of the recombination rate (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Kimura 1967</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Lynch 2008</a>). Thus, it may be viewed as surprising that bacteria, which do not undergo meiotic recombination, exhibit a relationship between <em>u<sub>id</sub></em> and <em>N</em><sub>e</sub> similar to that in eukaryotic species engaging in periodic to regular meiosis (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1, A and B</a>). It should be noted, however, that bacterial recombination occurs through multiple mechanisms (transformation, conjugation, and/or transduction). Many bacterial species are known to naturally undergo high rates of recombination, with ratios of recombination to mutation rates frequently being comparable to those in multicellular eukaryotes (Feil and Spratt 2001; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lynch 2007</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Doroghazi <em>et al.</em> 2014</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lassalle <em>et al.</em> 2015</a>), so, in this sense, comparable behavior of bacterial and eukaryotic species is not unexpected.</p><p id="p-57">In summary, as in our previous work on the base-substitution mutation rate (<a id="xref-ref-56-16" class="xref-bibr" href="#ref-56">Sung <em>et al.</em> 2012a</a>), the strong correlation between the genome-wide indel rate and <em>N<sub>e</sub></em> appears not to be a statistical artifact. Moreover, among various hypotheses that have been suggested for mutation-rate evolution, the DBH appears to provide the most compatible explanation for the ∼1000-fold range of variation of this trait across the Tree of Life. As noted above, the molecular mechanisms that generate and resolve base-substitution and indel mutations differ in a number of ways, and the rate of occurrence of these two types of mutations differ by one to two orders of magnitude (with <em>u<sub>id</sub></em> ranging from 1.8 to 11.9% of <em>u<sub>bs</sub></em>, presumably because of the elevated deleterious effects of indel mutations). Yet, despite these differences, both <em>u<sub>bs</sub></em> and <em>u<sub>id</sub></em> scale similarly with changes in <em>N<sub>e</sub></em> (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, <em>r</em><sup>2</sup> = 0.89). Because the forces of mutation, selection, and drift apply to all biological traits, the maximum achievable level of refinement for other fundamental cellular traits may also be influenced by the drift barrier.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">Support was provided by the Multidisciplinary University Research Initiative Award W911NF-09-1-0444, and from the US Army Research Office to M. L., P. Foster, H. Tang, and S. Finkel, and W911NF-14-1-0411 to M. L., P. Foster, J. McKinlay, and J. T. Lennon, by CAREER award DEB-0845851 from the National Science Foundation to V. C., and by National Institutes of Health Awards F32-GM103164 to W.S., and R01-GM036827 to M. L. and W. K. Thomas. This material is based upon work supported by the National Science Foundation under grant no. CNS-0521433, CNS-0723054, and ABI-1062432 to Indiana University.</p><p id="p-59">Author contributions: W.S., C.F., V.C., and M.L. designed the research; W.S., M.A., M.D., and T.P. performed the research; W.S. and M.A. analyzed the data; and W.S., M.A., and M.L. wrote the paper.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-20"><p id="p-60">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030890/-/DC1</a></p></li><li class="fn" id="fn-21"><p id="p-61"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-05-08"><span class="received-label">Received </span>May 8, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-06-13"><span class="accepted-label">Accepted </span>June 13, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Sung <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.1" data-doi="10.1016/j.tig.2016.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandvain</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The limits of natural selection in a nonequilibrium world.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D32%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2016.01.004%26rft_id%253Dinfo%253Apmid%252F26874998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2016.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26874998&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2583.2" data-doi="10.1016/j.tig.2013.04.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Properties and rates of germline mutations in humans.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">575</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D29%26rft.spage%253D575%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2013.04.005%26rft_id%253Dinfo%253Apmid%252F23684843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2013.04.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23684843&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325300800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.3" data-doi="10.1038/nrg2526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fundamental concepts in genetics: effective population size and patterns of molecular evolution and variation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D195%26rft.epage%253D205%26rft.atitle%253DFundamental%2Bconcepts%2Bin%2Bgenetics%253A%2Beffective%2Bpopulation%2Bsize%2Band%2Bpatterns%2Bof%2Bmolecular%2Bevolution%2Band%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2526%26rft_id%253Dinfo%253Apmid%252F19204717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19204717&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263457000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2583.4" data-doi="10.1038/ng.862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keebler</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindsay</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Variation in genome-wide mutation rates within and between human families.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">712</span>–<span class="cit-lpage">714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DConrad%26rft.auinit1%253DD.%2BF.%26rft.volume%253D43%26rft.issue%253D7%26rft.spage%253D712%26rft.epage%253D714%26rft.atitle%253DVariation%2Bin%2Bgenome-wide%2Bmutation%2Brates%2Bwithin%2Band%2Bbetween%2Bhuman%2Bfamilies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.862%26rft_id%253Dinfo%253Apmid%252F21666693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21666693&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.5" data-doi="10.1038/nature02697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">High mutation rate and predominance of insertions in the <em>Caenorhabditis elegans</em> nuclear genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">430</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDenver%26rft.auinit1%253DD.%2BR.%26rft.volume%253D430%26rft.issue%253D7000%26rft.spage%253D679%26rft.epage%253D682%26rft.atitle%253DHigh%2Bmutation%2Brate%2Band%2Bpredominance%2Bof%2Binsertions%2Bin%2Bthe%2BCaenorhabditis%2Belegans%2Bnuclear%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02697%26rft_id%253Dinfo%253Apmid%252F15295601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15295601&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223085400046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.6" data-doi="10.1534/genetics.104.038521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feinberg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estes</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mutation rates, spectra and hotspots in mismatch repair-deficient <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.038521%26rft_id%253Dinfo%253Apmid%252F15716493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcwLzEvMTA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.7" data-doi="10.1073/pnas.0904895106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolan</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilhelm</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucas-Lledo</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A genome-wide view of <em>Caenorhabditis elegans</em> base-substitution mutation processes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">16310</span>–<span class="cit-lpage">16314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0904895106%26rft_id%253Dinfo%253Apmid%252F19805298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzM4LzE2MzEwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.8" data-doi="10.1186/1471-2164-15-970"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doroghazi</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckley</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Intraspecies comparison of <em>Streptomyces pratensis</em> genomes reveals high levels of recombination and gene conservation between strains of disparate geographic origin.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">970</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D970%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-970%26rft_id%253Dinfo%253Apmid%252F25399205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-970&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25399205&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.9" data-doi="10.1073/pnas.88.16.7160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">A constant rate of spontaneous mutation in DNA-based microbes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">7160</span>–<span class="cit-lpage">7164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDrake%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D88%26rft.issue%253D16%26rft.spage%253D7160%26rft.epage%253D7164%26rft.atitle%253DA%2Bconstant%2Brate%2Bof%2Bspontaneous%2Bmutation%2Bin%2BDNA-based%2Bmicrobes.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.16.7160%26rft_id%253Dinfo%253Apmid%252F1831267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODgvMTYvNzE2MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Rates of spontaneous mutation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">1667</span>–<span class="cit-lpage">1686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDrake%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D148%26rft.issue%253D4%26rft.spage%253D1667%26rft.epage%253D1686%26rft.atitle%253DRates%2Bof%2BSpontaneous%2BMutation%26rft_id%253Dinfo%253Apmid%252F9560386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0OC80LzE2NjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjU4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.11" data-doi="10.1038/nrg2146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The distribution of fitness effects of new mutations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">610</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DEyre-Walker%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D610%26rft.epage%253D618%26rft.atitle%253DThe%2Bdistribution%2Bof%2Bfitness%2Beffects%2Bof%2Bnew%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2146%26rft_id%253Dinfo%253Apmid%252F17637733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17637733&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248170700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2583.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feil</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spratt</span>  <span class="cit-name-given-names">B. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recombination and the population structures of bacterial pathogens.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BMicrobiol.%26rft.volume%253D55%26rft.spage%253D561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.13" data-doi="10.1006/tpbi.1995.1025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. X.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Statistical properties of segregating sites.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DFu%26rft.auinit1%253DY.%2BX.%26rft.volume%253D48%26rft.issue%253D2%26rft.spage%253D172%26rft.epage%253D197%26rft.atitle%253DStatistical%2Bproperties%2Bof%2Bsegregating%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.1995.1025%26rft_id%253Dinfo%253Apmid%252F7482370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.1995.1025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7482370&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RZ41300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Statistical tests of neutrality of mutations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFu%26rft.auinit1%253DY.%2BX.%26rft.volume%253D133%26rft.issue%253D3%26rft.spage%253D693%26rft.epage%253D709%26rft.atitle%253DStatistical%2Btests%2Bof%2Bneutrality%2Bof%2Bmutations.%26rft_id%253Dinfo%253Apmid%252F8454210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMzLzMvNjkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.15" data-doi="10.1093/sysbio/42.3.265"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garland</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickerman</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janis</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Phylogenetic analysis of covariance by computer-simulation.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252F42.3.265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjQyLzMvMjY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.16" data-doi="10.1101/gr.5022906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Ubiquitous selective constraints in the <em>Drosophila</em> genome revealed by a genome-wide interspecies comparison.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5022906%26rft_id%253Dinfo%253Apmid%252F16751341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzcvODc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-2" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.17" data-doi="10.1101/gr.1329204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Patterns of evolutionary constraints in intronic and intergenic DNA of <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHalligan%26rft.auinit1%253DD.%2BL.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D273%26rft.epage%253D279%26rft.atitle%253DPatterns%2Bof%2BEvolutionary%2BConstraints%2Bin%2BIntronic%2Band%2BIntergenic%2BDNA%2Bof%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1329204%26rft_id%253Dinfo%253Apmid%252F14762063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE0LzIvMjczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2583.18" data-doi="10.1038/ng.2684"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haudry</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vello</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoen</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leclercq</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D891%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2684%26rft_id%253Dinfo%253Apmid%252F23817568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2684&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23817568&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The effect of deleterious alleles on adaptation in asexual populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJohnson%26rft.auinit1%253DT.%26rft.volume%253D162%26rft.issue%253D1%26rft.spage%253D395%26rft.epage%253D411%26rft.atitle%253DThe%2BEffect%2Bof%2BDeleterious%2BAlleles%2Bon%2BAdaptation%2Bin%2BAsexual%2BPopulations%26rft_id%253Dinfo%253Apmid%252F12242249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYyLzEvMzk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.20" data-doi="10.1371/journal.pgen.1000924"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karasov</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evidence that adaptation in <em>Drosophila</em> is not limited by mutation at single sites.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKarasov%26rft.auinit1%253DT.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De1000924%26rft.epage%253De1000924%26rft.atitle%253DEvidence%2Bthat%2Badaptation%2Bin%2BDrosophila%2Bis%2Bnot%2Blimited%2Bby%2Bmutation%2Bat%2Bsingle%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000924%26rft_id%253Dinfo%253Apmid%252F20585551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000924&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20585551&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.21" data-doi="10.1038/381694a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kibota</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Estimate of the genomic mutation rate deleterious to overall fitness in <em>E. coli</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">381</span>: <span class="cit-fpage">694</span>–<span class="cit-lpage">696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKibota%26rft.auinit1%253DT.%2BT.%26rft.volume%253D381%26rft.issue%253D6584%26rft.spage%253D694%26rft.epage%253D696%26rft.atitle%253DEstimate%2Bof%2Bthe%2Bgenomic%2Bmutation%2Brate%2Bdeleterious%2Bto%2Boverall%2Bfitness%2Bin%2BE.%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F381694a0%26rft_id%253Dinfo%253Apmid%252F8649513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/381694a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8649513&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UR97900051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.22" data-doi="10.1017/S0016672300010284"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span> <span class="cit-article-title">On the evolutionary adjustment of spontaneous mutation rates.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D9%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010284&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19679128700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-6.8.2583.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-source"><em>The Neutral Theory of Molecular Evolution</em></span>, <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, UK.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2583.24" data-doi="10.1038/nature11396"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frigge</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Besenbacher</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sulem</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Rate of <em>de novo</em> mutations and the importance of father’s age to disease risk.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">488</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKong%26rft.auinit1%253DA.%26rft.volume%253D488%26rft.issue%253D7412%26rft.spage%253D471%26rft.epage%253D475%26rft.atitle%253DRate%2Bof%2Bde%2Bnovo%2Bmutations%2Band%2Bthe%2Bimportance%2Bof%2Bfather%2527s%2Bage%2Bto%2Bdisease%2Brisk.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11396%26rft_id%253Dinfo%253Apmid%252F22914163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11396&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22914163&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307761600028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.25" data-doi="10.1101/cshperspect.a012583"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krokan</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bjoras</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Base excision repair.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">a012583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BPerspectives%2Bin%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft.aulast%253DKrokan%26rft.auinit1%253DH.%2BE.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253Da012583%26rft.epage%253Da012583%26rft.atitle%253DBase%2BExcision%2BRepair%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a012583%26rft_id%253Dinfo%253Apmid%252F23545420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI1LzQvYTAxMjU4MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.26" data-doi="10.1101/sqb.2009.74.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolving views of DNA replication (in)fidelity.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BSymp.%2BQuant.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fsqb.2009.74.027%26rft_id%253Dinfo%253Apmid%252F19903750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjc6Ijc0LzAvOTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjU4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.27" data-doi="10.1371/journal.pgen.1004941"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lassalle</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perian</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bataillon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nesme</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duret</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">GC-Content evolution in bacterial genomes: the biased gene conversion hypothesis expands.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004941</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1004941%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004941%26rft_id%253Dinfo%253Apmid%252F25659072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004941&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25659072&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popodi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Rate and molecular spectrum of spontaneous mutations in the bacterium <em>Escherichia coli</em> as determined by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">e2774</span>–<span class="cit-lpage">e2783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D109%26rft.spage%253De2774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.29" data-doi="10.1086/282663"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leigh</span>  <span class="cit-name-given-names">E. G.</span>  <span class="cit-name-suffix">Jr.</span>.</span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Natural selection and mutability.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D104%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282663&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1970G697100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.30" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.31" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-3" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.32" data-doi="10.1016/j.cub.2011.01.026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipinski</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farslow</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katju</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High spontaneous rate of gene duplication in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">306</span>–<span class="cit-lpage">310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DLipinski%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D21%26rft.issue%253D4%26rft.spage%253D306%26rft.epage%253D310%26rft.atitle%253DHigh%2Bspontaneous%2Brate%2Bof%2Bgene%2Bduplication%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2011.01.026%26rft_id%253Dinfo%253Apmid%252F21295484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2011.01.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21295484&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.33" data-doi="10.1073/pnas.0912451107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salk</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loeb</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Optimization of DNA polymerase mutation rates during bacterial evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">1154</span>–<span class="cit-lpage">1159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0912451107%26rft_id%253Dinfo%253Apmid%252F20080608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzMvMTE1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-6.8.2583.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source"><em>The Origins of Genome Architecture</em></span>, <span class="cit-publ-name">Sinauer Associates</span>, <span class="cit-publ-loc">Sunderland, Massachusetts.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.35" data-doi="10.1534/genetics.108.090456"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The cellular, developmental and population-genetic determinants of mutation-rate evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">933</span>–<span class="cit-lpage">943</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090456%26rft_id%253Dinfo%253Apmid%252F18757919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzIvOTMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.36" data-doi="10.1016/j.tig.2010.05.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evolution of the mutation rate.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D26%26rft.issue%253D8%26rft.spage%253D345%26rft.epage%253D352%26rft.atitle%253DEvolution%2Bof%2Bthe%2Bmutation%2Brate.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2010.05.003%26rft_id%253Dinfo%253Apmid%252F20594608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2010.05.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20594608&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280903800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.37" data-doi="10.1093/gbe/evr066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The lower bound to the evolution of mutation rates.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1107</span>–<span class="cit-lpage">1118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr066%26rft_id%253Dinfo%253Apmid%252F21821597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjMvMC8xMTA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.38" data-doi="10.1073/pnas.1514974112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marinov</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The bioenergetic costs of a gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">15690</span>–<span class="cit-lpage">15695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1514974112%26rft_id%253Dinfo%253Apmid%252F26575626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzUxLzE1NjkwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.39" data-doi="10.2307/2640707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kibota</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Spontaneous deleterious mutation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D53%26rft.spage%253D645%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2640707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2640707&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000081507300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.40" data-doi="10.1073/pnas.0803466105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome-wide view of the spectrum of spontaneous mutations in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">9272</span>–<span class="cit-lpage">9277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0803466105%26rft_id%253Dinfo%253Apmid%252F18583475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzI3LzkyNzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjU4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-6.8.2583.41"><div class="cit-metadata unstructured">Martins, E. P., 2004 <em>Compare</em>, <em>Version 4.6b. Computer Programs for the Statistical Analysis of Comparative Data</em>. Department of Biology, Indiana University, Bloomington, IN. Available at: <a href="http://compare.bio.indiana.edu">http://compare.bio.indiana.edu</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.42" data-doi="10.1038/cr.2008.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCulloch</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The fidelity of DNA synthesis by eukaryotic replicative and translesion synthesis polymerases.</span> <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">148</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bresearch%26rft.stitle%253DCell%2BRes%26rft.aulast%253DMcCulloch%26rft.auinit1%253DS.%2BD.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D148%26rft.epage%253D161%26rft.atitle%253DThe%2Bfidelity%2Bof%2BDNA%2Bsynthesis%2Bby%2Beukaryotic%2Breplicative%2Band%2Btranslesion%2Bsynthesis%2Bpolymerases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2008.4%26rft_id%253Dinfo%253Apmid%252F18166979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2008.4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18166979&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252920000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.43" data-doi="10.1016/S0168-9525(01)02447-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mira</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Deletional bias and the evolution of bacterial genomes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMira%26rft.auinit1%253DA.%26rft.volume%253D17%26rft.issue%253D10%26rft.spage%253D589%26rft.epage%253D596%26rft.atitle%253DDeletional%2Bbias%2Band%2Bthe%2Bevolution%2Bof%2Bbacterial%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252801%252902447-7%26rft_id%253Dinfo%253Apmid%252F11585665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(01)02447-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11585665&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171456500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morita</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakane</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iino</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular mechanisms of the whole DNA repair system: a comparison of bacterial and eukaryotic systems.</span> <abbr class="cit-jnl-abbrev">J. Nucleic Acids</abbr> <span class="cit-vol">2010</span>: <span class="cit-fpage">179594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BNucleic%2BAcids%26rft.aulast%253DMorita%26rft.auinit1%253DR.%26rft.volume%253D2010%26rft.spage%253D179594%26rft.epage%253D179594%26rft.atitle%253DMolecular%2Bmechanisms%2Bof%2Bthe%2Bwhole%2BDNA%2Brepair%2Bsystem%253A%2Ba%2Bcomparison%2Bof%2Bbacterial%2Band%2Beukaryotic%2Bsystems.%26rft_id%253Dinfo%253Apmid%252F20981145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20981145&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.45" data-doi="10.1534/genetics.112.145078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ness</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colegrave</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimate of the spontaneous mutation rate in <em>Chlamydomonas reinhardtii</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1447</span>–<span class="cit-lpage">1454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.145078%26rft_id%253Dinfo%253Apmid%252F23051642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE0NDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjU4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2583.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ness</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraemer</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colegrave</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Direct estimate of the spontaneous mutation rate uncovers the effects of drift and recombination in the <em>Chlamydomonas reinhardtii</em> plastid genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">800</span>–<span class="cit-lpage">808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D33%26rft.spage%253D800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2583.47" data-doi="10.1038/ng.835"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Roak</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deriziotis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vives</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Exome sequencing in sporadic autism spectrum disorders identifies severe <em>de novo</em> mutations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DO%2527Roak%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D585%26rft.epage%253D589%26rft.atitle%253DExome%2Bsequencing%2Bin%2Bsporadic%2Bautism%2Bspectrum%2Bdisorders%2Bidentifies%2Bsevere%2Bde%2Bnovo%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.835%26rft_id%253Dinfo%253Apmid%252F21572417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.835&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572417&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2583.48" data-doi="10.1038/nature10989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Roak</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vives</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Girirajan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karakoc</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krumm</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sporadic autism exomes reveal a highly interconnected protein network of <em>de novo</em> mutations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DO%2527Roak%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D485%26rft.issue%253D7397%26rft.spage%253D246%26rft.epage%253D250%26rft.atitle%253DSporadic%2Bautism%2Bexomes%2Breveal%2Ba%2Bhighly%2Binterconnected%2Bprotein%2Bnetwork%2Bof%2Bde%2Bnovo%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10989%26rft_id%253Dinfo%253Apmid%252F22495309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10989&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22495309&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303799800043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The rate of adaptation in asexuals.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">968</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BA.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D961%26rft.epage%253D968%26rft.atitle%253DThe%2BRate%2Bof%2BAdaptation%2Bin%2BAsexuals%26rft_id%253Dinfo%253Apmid%252F10835413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.50" data-doi="10.1126/science.1180677"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucas-Lledo</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warthmann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The rate and molecular spectrum of spontaneous mutations in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DOssowski%26rft.auinit1%253DS.%26rft.volume%253D327%26rft.issue%253D5961%26rft.spage%253D92%26rft.epage%253D94%26rft.atitle%253DThe%2BRate%2Band%2BMolecular%2BSpectrum%2Bof%2BSpontaneous%2BMutations%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180677%26rft_id%253Dinfo%253Apmid%252F20044577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMjcvNTk2MS85MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.51" data-doi="10.1146/annurev.biochem.73.011303.073723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sancar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindsey-Boltz</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Unsal-Kacmaz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linn</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Molecular mechanisms of mammalian DNA repair and the DNA damage checkpoints.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DSancar%26rft.auinit1%253DA.%26rft.volume%253D73%26rft.spage%253D39%26rft.epage%253D85%26rft.atitle%253DMolecular%2Bmechanisms%2Bof%2Bmammalian%2BDNA%2Brepair%2Band%2Bthe%2BDNA%2Bdamage%2Bcheckpoints.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.73.011303.073723%26rft_id%253Dinfo%253Apmid%252F15189136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.73.011303.073723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15189136&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223246400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.52" data-doi="10.1534/genetics.113.151670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Rates and genomic consequences of spontaneous mutational events in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">937</span>–<span class="cit-lpage">954</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151670%26rft_id%253Dinfo%253Apmid%252F23733788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzQvOTM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-2" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.53" data-doi="10.1101/gr.3715005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siepel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bejerano</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinrichs</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1034</span>–<span class="cit-lpage">1050</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3715005%26rft_id%253Dinfo%253Apmid%252F16024819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE1LzgvMTAzNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.54" data-doi="10.1016/j.cub.2013.01.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sniegowski</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raynes</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mutation rates: how low can you go?</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">R147</span>–<span class="cit-lpage">R149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D23%26rft.spage%253DR147%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2013.01.018%26rft_id%253Dinfo%253Apmid%252F23428322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2013.01.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23428322&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.55" data-doi="10.1002/1521-1878(200012)22:12&lt;1057::AID-BIES3&gt;3.0.CO;2-W"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sniegowski</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerrish</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaver</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The evolution of mutation rates: separating causes from consequences.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1057</span>–<span class="cit-lpage">1066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DSniegowski%26rft.auinit1%253DP.%2BD.%26rft.volume%253D22%26rft.issue%253D12%26rft.spage%253D1057%26rft.epage%253D1066%26rft.atitle%253DThe%2Bevolution%2Bof%2Bmutation%2Brates%253A%2Bseparating%2Bcauses%2Bfrom%2Bconsequences.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1521-1878%2528200012%252922%253A12%253C1057%253A%253AAID-BIES3%253E3.0.CO%253B2-W%26rft_id%253Dinfo%253Apmid%252F11084621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/1521-1878(200012)22:12&lt;1057::AID-BIES3&gt;3.0.CO;2-W&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11084621&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165552500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.56" data-doi="10.1073/pnas.1216223109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackerman</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doak</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Drift-barrier hypothesis and mutation-rate evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">18488</span>–<span class="cit-lpage">18492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1216223109%26rft_id%253Dinfo%253Apmid%252F23077252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQ1LzE4NDg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.57" data-doi="10.1073/pnas.1210663109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doak</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Extraordinary genome stability in the ciliate <em>Paramecium tetraurelia</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">19339</span>–<span class="cit-lpage">19344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1210663109%26rft_id%253Dinfo%253Apmid%252F23129619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQ3LzE5MzM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.58" data-doi="10.1093/molbev/msv055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackerman</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gout</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Asymmetric context-dependent mutation patterns revealed through mutation-accumulation experiments.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1672</span>–<span class="cit-lpage">1683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv055%26rft_id%253Dinfo%253Apmid%252F25750180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjMyLzcvMTY3MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzI1ODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2583.60" data-doi="10.1038/nature15393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The 1000 Genomes Project Consortium</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A global reference for human genetic variation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">526</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D526%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15393%26rft_id%253Dinfo%253Apmid%252F26432245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature15393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26432245&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2583.61" data-doi="10.1101/gr.186148.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uchimura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higuchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minakuchi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toyoda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Germline mutation rates and the long-term phenotypic effects of mutation accumulation in wild-type laboratory mice and mutator mice.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1125</span>–<span class="cit-lpage">1134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.186148.114%26rft_id%253Dinfo%253Apmid%252F26129709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI1LzgvMTEyNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yNTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.62" data-doi="10.1016/j.tree.2010.03.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vieira-Silva</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Touchon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">E. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">No evidence for elemental-based streamlining of prokaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">320</span>; author reply 320–311.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DVieira-Silva%26rft.auinit1%253DS.%26rft.volume%253D25%26rft.issue%253D6%26rft.spage%253D319%26rft.epage%253D320%26rft.atitle%253DNo%2Bevidence%2Bfor%2Belemental-based%2Bstreamlining%2Bof%2Bprokaryotic%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2010.03.001%26rft_id%253Dinfo%253Apmid%252F20381897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2010.03.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20381897&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278682500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2583.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title"><em>De novo</em> mutations discovered in 8 Mexican American families through whole genome sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Proc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">S24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BProc.%26rft.volume%253D8%26rft.spage%253DS24%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.64" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">On the number of segregating sites in genetical models without recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2583.65" data-doi="10.1371/journal.pgen.1001080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitney</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garland</span>  <span class="cit-name-given-names">T.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">2010</span> <span class="cit-article-title">Did genetic drift drive increases in genome complexity?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001080%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001080%26rft_id%253Dinfo%253Apmid%252F20865118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20865118&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2583.66" data-doi="10.1038/nature14649"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Parent-progeny sequencing indicates higher mutation rates in heterozygotes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">523</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D523%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14649%26rft_id%253Dinfo%253Apmid%252F26176923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26176923&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2583.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.8.2583.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegal</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Precise estimates of mutation rate and spectrum in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">e2310</span>–<span class="cit-lpage">e2318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.volume%253D111%26rft.spage%253De2310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/8/2583.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2573" title="Genomic and Transcriptomic Associations Identify a New Insecticide Resistance Phenotype for the Selective Sweep at the Cyp6g1 Locus of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2593" title="Crosses Heterozygous for Hybrid Neurospora Translocation Strains Show Transmission Ratio Distortion Disfavoring Homokaryotic Ascospores Made Following Alternate Segregation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/8"><span>Volume 6 Issue 8, August 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/8.cover-source.jpg?itok=40Kz9Oeq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2583.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cezTznTZEH0cw8EFQxtQ9C-HdCvsqYSXHdERG-OU3OY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/8/2583" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/8/2583" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/8/2583" class="active" data-icon-position="" data-hide-link-title="0">Evolution of the Insertion-Deletion Mutation Rate Across the Tree of Life</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460440" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-0wGmHoVRuKs6hLAh3Mqg0cXUYMmXL8qBfl-xHMJg6eQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2583.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/8/2583" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-EYUv6RqI5TpUpZHzUhKbc_1mHnVbnwAX2dN2N5AB5i0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/8/2583" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2583.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460440" id="node460440--2" data-pisa="ggg;6/8/2583" data-pisa-master="ggg;g3.116.030890" data-apath="/ggg/6/8/2583.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolution of the Insertion-Deletion Mutation Rate Across the Tree of Life</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8336-8913" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Way Sung</span>, <span class="highwire-citation-author" data-delta="1">Matthew S. Ackerman</span>, <span class="highwire-citation-author" data-delta="2">Marcus M. Dillon</span>, <span class="highwire-citation-author" data-delta="3">Thomas G. Platt</span>, <span class="highwire-citation-author" data-delta="4">Clay Fuqua</span>, <span class="highwire-citation-author" data-delta="5">Vaughn S. Cooper</span> and <span class="highwire-citation-author" data-delta="6">Michael Lynch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2583-2591;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030890 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460440--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Way Sung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics and Genomics, University of North Carolina at Charlotte, North Carolina 28223</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Way%2BSung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sung%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWay%2BSung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8336-8913" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Way Sung</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:waysung@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">waysung@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew S. Ackerman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BS.%2BAckerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ackerman%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BS.%2BAckerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marcus M. Dillon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Microbiology Graduate Program, University of New Hampshire, Durham, New Hampshire 03824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcus%2BM.%2BDillon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dillon%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcus%2BM.%2BDillon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Thomas G. Platt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Microbiology and Molecular Genetics, University of Pittsburgh School of Medicine, Pennsylvania 15219</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BG.%2BPlatt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Platt%20TG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BG.%2BPlatt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Clay Fuqua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clay%2BFuqua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuqua%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClay%2BFuqua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Vaughn S. Cooper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Microbiology Graduate Program, University of New Hampshire, Durham, New Hampshire 03824</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Microbiology and Molecular Genetics, University of Pittsburgh School of Medicine, Pennsylvania 15219</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vaughn%2BS.%2BCooper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cooper%20VS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVaughn%2BS.%2BCooper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/8/2583" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/8/2583&amp;title=Evolution%20of%20the%20Insertion-Deletion%20Mutation%20Rate%20Across%20the%20Tree%20of%20Life" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/8/2583&amp;title=Evolution%20of%20the%20Insertion-Deletion%20Mutation%20Rate%20Across%20the%20Tree%20of%20Life" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/8/2583&amp;title=Evolution%20of%20the%20Insertion-Deletion%20Mutation%20Rate%20Across%20the%20Tree%20of%20Life" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/8/2583&amp;text=Evolution%20of%20the%20Insertion-Deletion%20Mutation%20Rate%20Across%20the%20Tree%20of%20Life" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/8/2583&amp;title=Evolution%20of%20the%20Insertion-Deletion%20Mutation%20Rate%20Across%20the%20Tree%20of%20Life" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/8/2583&amp;t=Evolution%20of%20the%20Insertion-Deletion%20Mutation%20Rate%20Across%20the%20Tree%20of%20Life" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/8/2583&amp;title=Evolution%20of%20the%20Insertion-Deletion%20Mutation%20Rate%20Across%20the%20Tree%20of%20Life" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/8/2583&amp;title=Evolution%20of%20the%20Insertion-Deletion%20Mutation%20Rate%20Across%20the%20Tree%20of%20Life" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460440/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460440" id="node460440--3" data-pisa="ggg;6/8/2583" data-pisa-master="ggg;g3.116.030890" data-apath="/ggg/6/8/2583.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolution of the Insertion-Deletion Mutation Rate Across the Tree of Life</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8336-8913" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Way Sung</span>, <span class="highwire-citation-author" data-delta="1">Matthew S. Ackerman</span>, <span class="highwire-citation-author" data-delta="2">Marcus M. Dillon</span>, <span class="highwire-citation-author" data-delta="3">Thomas G. Platt</span>, <span class="highwire-citation-author" data-delta="4">Clay Fuqua</span>, <span class="highwire-citation-author" data-delta="5">Vaughn S. Cooper</span> and <span class="highwire-citation-author" data-delta="6">Michael Lynch</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2583-2591;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030890 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460440--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Way Sung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics and Genomics, University of North Carolina at Charlotte, North Carolina 28223</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Way%2BSung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sung%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWay%2BSung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8336-8913" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Way Sung</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:waysung@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">waysung@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Matthew S. Ackerman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BS.%2BAckerman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ackerman%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BS.%2BAckerman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marcus M. Dillon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Microbiology Graduate Program, University of New Hampshire, Durham, New Hampshire 03824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcus%2BM.%2BDillon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dillon%20MM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcus%2BM.%2BDillon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Thomas G. Platt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Microbiology and Molecular Genetics, University of Pittsburgh School of Medicine, Pennsylvania 15219</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BG.%2BPlatt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Platt%20TG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BG.%2BPlatt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Clay Fuqua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clay%2BFuqua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fuqua%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClay%2BFuqua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Vaughn S. Cooper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Microbiology Graduate Program, University of New Hampshire, Durham, New Hampshire 03824</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Microbiology and Molecular Genetics, University of Pittsburgh School of Medicine, Pennsylvania 15219</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vaughn%2BS.%2BCooper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cooper%20VS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVaughn%2BS.%2BCooper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael Lynch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Indiana University, Bloomington, Indiana 47405</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BLynch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lynch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BLynch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2583&amp;rft.epage=2591&amp;rft.atitle=Evolution%20of%20the%20Insertion-Deletion%20Mutation%20Rate%20Across%20the%20Tree%20of%20Life&amp;rft.volume=6&amp;rft.issue=8&amp;rft.date=2016-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sung%2C+Way&amp;rft.au=Ackerman%2C+Matthew+S.&amp;rft.au=Dillon%2C+Marcus+M.&amp;rft.au=Platt%2C+Thomas+G.&amp;rft.au=Fuqua%2C+Clay&amp;rft.au=Cooper%2C+Vaughn+S.&amp;rft.au=Lynch%2C+Michael"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460440/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460440/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460440/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460440/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460440/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460440/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460440/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460440/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460440/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460440/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460440/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460440/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/8/2583" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/8/2583.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/8/2583.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/8/2583.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/8/2583.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
