<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/8/2365" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/8/2365.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/8/2365.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/8/2365.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/8/2365/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/8/2365.atom" />
<meta name="HW.pisa" content="ggg;6/8/2365" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Use of Targeted Marker Subsets to Account for Population Structure and Relatedness in Genome-Wide Association Studies of Maize (Zea mays L.)" />
<meta name="DC.Identifier" content="10.1534/g3.116.029090" />
<meta name="DC.Date" content="2016-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2016 Chen and Lipka. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="A typical plant genome-wide association study (GWAS) uses a mixed linear model (MLM) that includes a trait as the response variable, a marker as an explanatory variable, and fixed and random effect covariates accounting for population structure and relatedness. Although effective in controlling for false positive signals, this model typically fails to detect signals that are correlated with population structure or are located in high linkage disequilibrium (LD) genomic regions. This result likely arises from each tested marker being used to estimate population structure and relatedness. Previous work has demonstrated that it is possible to increase the power of the MLM by estimating relatedness ( i.e. , kinship) with markers that are not located on the chromosome where the tested marker resides. To quantify the amount of additional significant signals one can expect using this so-called K\_chr model, we reanalyzed Mendelian, polygenic, and complex traits in two maize ( Zea mays L.) diversity panels that have been previously assessed using the traditional MLM. We demonstrated that the K\_chr model could find more significant associations, especially in high LD regions. This finding is underscored by our identification of novel genomic signals proximal to the tocochromanol biosynthetic pathway gene ZmVTE1 that are associated with a ratio of tocotrienols. We conclude that the K\_chr model can detect more intricate sources of allelic variation underlying agronomically important traits, and should therefore become more widely used for GWAS. To facilitate the implementation of the K\_chr model, we provide code written in the R programming language." />
<meta name="DC.Contributor" content="Angela H. Chen" />
<meta name="DC.Contributor" content="Alexander E. Lipka" />
<meta name="article:published_time" content="2016-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Use of Targeted Marker Subsets to Account for Population Structure and Relatedness in Genome-Wide Association Studies of Maize (Zea mays L.)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;A typical plant genome-wide association study (GWAS) uses a mixed linear model (MLM) that includes a trait as the response variable, a marker as an explanatory variable, and fixed and random effect covariates accounting for population structure and relatedness. Although effective in controlling for false positive signals, this model typically fails to detect signals that are correlated with population structure or are located in high linkage disequilibrium (LD) genomic regions. This result likely arises from each tested marker being used to estimate population structure and relatedness. Previous work has demonstrated that it is possible to increase the power of the MLM by estimating relatedness (&lt;i&gt;i.e.&lt;/i&gt;, kinship) with markers that are not located on the chromosome where the tested marker resides. To quantify the amount of additional significant signals one can expect using this so-called K_chr model, we reanalyzed Mendelian, polygenic, and complex traits in two maize (&lt;i&gt;Zea mays&lt;/i&gt; L.) diversity panels that have been previously assessed using the traditional MLM. We demonstrated that the K_chr model could find more significant associations, especially in high LD regions. This finding is underscored by our identification of novel genomic signals proximal to the tocochromanol biosynthetic pathway gene &lt;i&gt;ZmVTE1&lt;/i&gt; that are associated with a ratio of tocotrienols. We conclude that the K_chr model can detect more intricate sources of allelic variation underlying agronomically important traits, and should therefore become more widely used for GWAS. To facilitate the implementation of the K_chr model, we provide code written in the R programming language.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/08/01" />
<meta name="citation_mjid" content="ggg;6/8/2365" />
<meta name="citation_id" content="6/8/2365" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/8/2365" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/8/2365.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/8/2365.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/8/2365.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.029090" />
<meta name="citation_pmid" content="27233668" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2365" />
<meta name="citation_lastpage" content="2374" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Angela H. Chen" />
<meta name="citation_author_institution" content="Department of Statistics, University of Illinois at Urbana-Champaign, Illinois 61801" />
<meta name="citation_author" content="Alexander E. Lipka" />
<meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois at Urbana-Champaign, Illinois 61801" />
<meta name="citation_author_email" content="alipka@illinois.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1571-8528" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Atwell;citation_author=Y. S. Huang;citation_author=B. J. Vilhjalmsson;citation_author=G. Willems;citation_author=M. Horton;citation_title=Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines.;citation_pages=627-631;citation_volume=465;citation_year=2010;citation_issue=7298;citation_pmid=20336072;citation_doi=10.1038/nature08800" />
<meta name="citation_reference" content="citation_journal_title=J. Roy. Stat. Soc. B Met.;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995;citation_issue=1" />
<meta name="citation_reference" content="citation_volume=6" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. S. Buckler;citation_author=J. B. Holland;citation_author=P. J. Bradbury;citation_author=C. B. Acharya;citation_author=P. J. Brown;citation_title=The Genetic Architecture of Maize Flowering Time;citation_pages=714-718;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661422;citation_doi=10.1126/science.1174276" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=S. Chander;citation_author=Y. Guo;citation_author=X. Yang;citation_author=J. Yan;citation_author=Y. Zhang;citation_title=Genetic dissection of tocopherol content and composition in maize grain using quantitative trait loci analysis and the candidate gene approach.;citation_pages=353-365;citation_volume=22;citation_year=2008;citation_issue=3;citation_doi=10.1007/s11032-008-9180-8" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. P. Cook;citation_author=M. D. McMullen;citation_author=J. B. Holland;citation_author=F. Tian;citation_author=P. Bradbury;citation_title=Genetic architecture of maize kernel composition in the nested association mapping and inbred association panels.;citation_volume=158;citation_year=2012;citation_issue=2;citation_pmid=22135431;citation_doi=10.1104/pp.111.185033" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Daetwyler;citation_author=M. Calus;citation_author=R. Pong-Wong;citation_author=G. de los Campos;citation_author=J. Hickey;citation_title=Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.;citation_volume=193;citation_year=2013;citation_issue=2;citation_pmid=23222650;citation_doi=10.1534/genetics.112.147983" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. P. Dickson;citation_author=K. Wang;citation_author=I. Krantz;citation_author=H. Hakonarson;citation_author=D. B. Goldstein;citation_title=Rare variants create synthetic genome-wide associations.;citation_pages=e1000294-e1000294;citation_volume=8;citation_year=2010;citation_issue=1;citation_pmid=20126254;citation_doi=10.1371/journal.pbio.1000294" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Flint;citation_author=E. Eskin;citation_title=Genome-wide association studies in mice.;citation_pages=807-817;citation_volume=13;citation_year=2012;citation_issue=11;citation_pmid=23044826;citation_doi=10.1038/nrg3335" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. A. Flint-Garcia;citation_author=A. C. Thuillet;citation_author=J. M. Yu;citation_author=G. Pressoir;citation_author=S. M. Romero;citation_title=Maize association population: a high-resolution platform for quantitative trait locus dissection.;citation_pages=1054-1064;citation_volume=44;citation_year=2005;citation_issue=6;citation_pmid=16359397;citation_doi=10.1111/j.1365-313X.2005.02591.x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. B. Kandianis;citation_author=R. Stevens;citation_author=W. Liu;citation_author=N. Palacios;citation_author=K. Montgomery;citation_title=Genetic architecture controlling variation in grain carotenoid composition and concentrations in two maize populations.;citation_pages=2879-2895;citation_volume=126;citation_year=2013;citation_issue=11;citation_pmid=24042570;citation_doi=10.1007/s00122-013-2179-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. Zaitlen;citation_author=C. Wade;citation_author=A. Kirby;citation_author=S. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. M. Kang;citation_author=J. H. Sul;citation_author=S. K. Service;citation_author=N. A. Zaitlen;citation_author=S. Y. Kong;citation_title=Variance component model to account for sample structure in genome-wide association studies.;citation_pages=348-354;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208533;citation_doi=10.1038/ng.548" />
<meta name="citation_reference" content="citation_volume=9" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_title=Lessons from Dwarf8 on the strengths and weaknesses of structured association mapping.;citation_author=S. J. Larsson;citation_author=A. E. Lipka;citation_author=E. S. Buckler;citation_volume=9;citation_year=2013;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. E. Lipka;citation_author=F. Tian;citation_author=Q. S. Wang;citation_author=J. Peiffer;citation_author=M. Li;citation_title=GAPIT: genome association and prediction integrated tool.;citation_volume=28;citation_year=2012;citation_issue=18;citation_pmid=22796960;citation_doi=10.1093/bioinformatics/bts444" />
<meta name="citation_reference" content="citation_journal_title=Bethesda);citation_author=A. E. Lipka;citation_author=M. A. Gore;citation_author=M. Magallanes-Lundback;citation_author=A. Mesberg;citation_author=H. N. Lin;citation_title=Genome-wide association study and pathway-level analysis of tocochromanol levels in maize grain. G3;citation_pages=1287-1299;citation_volume=3;citation_year=2013;citation_issue=8" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=A. E. Lipka;citation_author=C. B. Kandianis;citation_author=M. E. Hudson;citation_author=J. Yu;citation_author=J. Drnevich;citation_title=From association to prediction: statistical methods for the dissection and selection of complex traits in plants.;citation_pages=110-118;citation_volume=24;citation_year=2015;citation_pmid=25795170;citation_doi=10.1016/j.pbi.2015.02.010" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=J. Listgarten;citation_author=C. Lippert;citation_author=C. M. Kadie;citation_author=R. I. Davidson;citation_author=E. Eskin;citation_title=Improved linear mixed models for genome-wide association studies.;citation_pages=525-526;citation_volume=9;citation_year=2012;citation_issue=6;citation_pmid=22669648;citation_doi=10.1038/nmeth.2037" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=B. A. Loiselle;citation_author=V. L. Sork;citation_author=J. Nason;citation_author=C. Graham;citation_title=Spatial genetic-structure of a tropical understory shrub, Psychotria officinalis (rubiaceae).;citation_pages=1420-1425;citation_volume=82;citation_year=1995;citation_issue=11;citation_doi=10.2307/2445869" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. McMullen;citation_author=S. Kresovich;citation_author=H. S. Villeda;citation_author=P. Bradbury;citation_author=H. H. Li;citation_title=Genetic Properties of the Maize Nested Association Mapping Population;citation_pages=737-740;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661427;citation_doi=10.1126/science.1174320" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=S. Myles;citation_author=J. Peiffer;citation_author=P. J. Brown;citation_author=E. S. Ersoz;citation_author=Z. W. Zhang;citation_title=Association mapping: critical considerations shift from genotyping to experimental design.;citation_volume=21;citation_year=2009;citation_issue=8;citation_pmid=19654263;citation_doi=10.1105/tpc.109.068437" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. F. Owens;citation_author=A. E. Lipka;citation_author=M. Magallanes-Lundback;citation_author=T. Tiede;citation_author=C. H. Diepenbrock;citation_title=A foundation for provitamin A biofortification of maize: genome-wide association and genomic prediction models of carotenoid levels.;citation_volume=198;citation_year=2014;citation_issue=4;citation_pmid=25258377;citation_doi=10.1534/genetics.114.169979" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Peiffer;citation_author=M. C. Romay;citation_author=M. A. Gore;citation_author=S. A. Flint-Garcia;citation_author=Z. Zhang;citation_title=The genetic architecture of maize height.;citation_volume=196;citation_year=2014;citation_issue=4;citation_pmid=24514905;citation_doi=10.1534/genetics.113.159152" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Platt;citation_author=B. J. Vilhjalmsson;citation_author=M. Nordborg;citation_title=Conditions under which genome-wide association studies will be positively misleading.;citation_volume=186;citation_year=2010;citation_issue=3;citation_pmid=20813880;citation_doi=10.1534/genetics.110.121665" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Price;citation_author=N. J. Patterson;citation_author=R. M. Plenge;citation_author=M. E. Weinblatt;citation_author=N. A. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. L. Remington;citation_author=J. M. Thornsberry;citation_author=Y. Matsuoka;citation_author=L. M. Wilson;citation_author=S. R. Whitt;citation_title=Structure of linkage disequilibrium and phenotypic associations in the maize genome.;citation_volume=98;citation_year=2001;citation_issue=20;citation_pmid=11562485;citation_doi=10.1073/pnas.201394398" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Rincent;citation_author=L. Moreau;citation_author=H. Monod;citation_author=E. Kuhn;citation_author=A. E. Melchinger;citation_title=Recovering power in association mapping panels with variable levels of linkage disequilibrium.;citation_volume=197;citation_year=2014;citation_issue=1;citation_pmid=24532779;citation_doi=10.1534/genetics.113.159731" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_title=Comprehensive genotyping of the USA national maize inbred seed bank.;citation_author=M. C. Romay;citation_author=M. J. Millard;citation_author=J. C. Glaubitz;citation_author=J. A. Peiffer;citation_author=K. L. Swarts;citation_volume=14;citation_year=2013;citation_issue=6" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Salvi;citation_author=G. Sponza;citation_author=M. Morgante;citation_author=D. Tomes;citation_author=X. Niu;citation_title=Conserved noncoding genomic sequences associated with a flowering-time quantitative trait locus in maize.;citation_volume=104;citation_year=2007;citation_issue=27;citation_pmid=17595297;citation_doi=10.1073/pnas.0704145104" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=G. Schwarz;citation_title=Estimating the dimension of a model.;citation_pages=461-464;citation_volume=6;citation_year=1978;citation_issue=2;citation_doi=10.1214/aos/1176344136" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=V. Segura;citation_author=B. J. Vilhjalmsson;citation_author=A. Platt;citation_author=A. Korte;citation_author=U. Seren;citation_title=An efficient multi-locus mixed-model approach for genome-wide association studies in structured populations.;citation_pages=825-830;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22706313;citation_doi=10.1038/ng.2314" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=G. Sun;citation_author=C. Zhu;citation_author=M. H. Kramer;citation_author=S. S. Yang;citation_author=W. Song;citation_title=Variation explained in mixed-model association mapping.;citation_pages=333-340;citation_volume=105;citation_year=2010;citation_issue=4;citation_pmid=20145669;citation_doi=10.1038/hdy.2010.11" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. M. Thornsberry;citation_author=M. M. Goodman;citation_author=J. Doebley;citation_author=S. Kresovich;citation_author=D. Nielsen;citation_title=Dwarf8 polymorphisms associate with variation in flowering time.;citation_pages=286-289;citation_volume=28;citation_year=2001;citation_issue=3;citation_pmid=11431702;citation_doi=10.1038/90135" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Biom. Bull.;citation_author=F. Wilcoxon;citation_title=Individual comparisons by ranking methods.;citation_pages=80-83;citation_volume=1;citation_year=1945;citation_issue=6;citation_doi=10.2307/3001968" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Wong;citation_author=R. J. Lambert;citation_author=Y. Tadmor;citation_author=T. R. Rocheford;citation_title=QTL Associated with Accumulation of Tocopherols in Maize;citation_pages=2257-2266;citation_volume=43;citation_year=2003;citation_issue=6;citation_doi=10.2135/cropsci2003.2257" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. M. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. V. Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z. W. Zhang;citation_author=E. Ersoz;citation_author=C. Q. Lai;citation_author=R. J. Todhunter;citation_author=H. K. Tiwari;citation_title=Mixed linear model approach adapted for genome-wide association studies.;citation_pages=355-360;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208535;citation_doi=10.1038/ng.546" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Zhou;citation_author=M. Stephens;citation_title=Genome-wide efficient mixed-model analysis for association studies.;citation_pages=821-824;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22706312;citation_doi=10.1038/ng.2310" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=C. S. Zhu;citation_author=M. A. Gore;citation_author=E. S. Buckler;citation_author=J. Yu;citation_title=Status and prospects of association mapping in plants.;citation_pages=5-20;citation_volume=1;citation_year=2008;citation_issue=1;citation_doi=10.3835/plantgenome2008.02.0089" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Use of Targeted Marker Subsets to Account for Population Structure and Relatedness in Genome-Wide Association Studies of Maize (Zea mays L.)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="twitter:description" content="A typical plant genome-wide association study (GWAS) uses a mixed linear model (MLM) that includes a trait as the response variable, a marker as an explanatory variable, and fixed and random effect covariates accounting for population structure and relatedness. Although effective in controlling for false positive signals, this model typically fails to detect signals that are correlated with population structure or are located in high linkage disequilibrium (LD) genomic regions. This result likely arises from each tested marker being used to estimate population structure and relatedness. Previous work has demonstrated that it is possible to increase the power of the MLM by estimating relatedness ( i.e. , kinship) with markers that are not located on the chromosome where the tested marker resides. To quantify the amount of additional significant signals one can expect using this so-called K\_chr model, we reanalyzed Mendelian, polygenic, and complex traits in two maize ( Zea mays L.) diversity panels that have been previously assessed using the traditional MLM. We demonstrated that the K\_chr model could find more significant associations, especially in high LD regions. This finding is underscored by our identification of novel genomic signals proximal to the tocochromanol biosynthetic pathway gene ZmVTE1 that are associated with a ratio of tocotrienols. We conclude that the K\_chr model can detect more intricate sources of allelic variation underlying agronomically important traits, and should therefore become more widely used for GWAS. To facilitate the implementation of the K\_chr model, we provide code written in the R programming language." />
<meta name="og-title" property="og:title" content="The Use of Targeted Marker Subsets to Account for Population Structure and Relatedness in Genome-Wide Association Studies of Maize (Zea mays L.)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/8/2365" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="A typical plant genome-wide association study (GWAS) uses a mixed linear model (MLM) that includes a trait as the response variable, a marker as an explanatory variable, and fixed and random effect covariates accounting for population structure and relatedness. Although effective in controlling for false positive signals, this model typically fails to detect signals that are correlated with population structure or are located in high linkage disequilibrium (LD) genomic regions. This result likely arises from each tested marker being used to estimate population structure and relatedness. Previous work has demonstrated that it is possible to increase the power of the MLM by estimating relatedness ( i.e. , kinship) with markers that are not located on the chromosome where the tested marker resides. To quantify the amount of additional significant signals one can expect using this so-called K\_chr model, we reanalyzed Mendelian, polygenic, and complex traits in two maize ( Zea mays L.) diversity panels that have been previously assessed using the traditional MLM. We demonstrated that the K\_chr model could find more significant associations, especially in high LD regions. This finding is underscored by our identification of novel genomic signals proximal to the tocochromanol biosynthetic pathway gene ZmVTE1 that are associated with a ratio of tocotrienols. We conclude that the K\_chr model can detect more intricate sources of allelic variation underlying agronomically important traits, and should therefore become more widely used for GWAS. To facilitate the implementation of the K\_chr model, we provide code written in the R programming language." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
    <title>The Use of Targeted Marker Subsets to Account for Population Structure and Relatedness in Genome-Wide Association Studies of Maize (Zea mays L.) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"aaORkkgq1vpCjU3g0KN87RK9N-R5gHrGVr7kfDfWY0A","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460255","apath":"\/ggg\/6\/8\/2365.atom","pisa":"ggg;6\/8\/2365","ac":{"\/ggg\/6\/8\/2365.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/8\/2365.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2365"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2365"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/8\/2365":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BT2jVZURehCJsspPFUxPBfSbMzNxPWvfwGYUuHgEHZXtc7+cbKcm4qLLx1Q0gUEV+lvaVsvAiMHi2owBWacG9u0BtPVnZlyTzmoI6yVHq7ansA09akByX52yHYiSJodemhytPypqzTHuawYUtOV0ZdhfFwwT\\\\\\\/5Y+asDL\\\\\\\/j4V9kFaULE+PPwHR9tnjQ4sqzpJSz4XbRXOYWPzgcwv7beibgSgu2YVfd9LdO306yCvyQT0lTgHxm070RdyzvASzBqw2dcd8iEx0AGTaNgcKyNE30HrIlUVA28YdWQOkyxDUX5JsJ\\\\\\\/4YcuOJ1lyro1XNbDbil7PyaOlYi5576CHpaGOMjRZAlqa856Ypya+Z6nQ7Cqg1dEjI8JPxC9o5uBu5tW6Jbo1OCoXvYGLkDEHouN47iAG1H1+rVk88pCmrW1G5+ssJn5ycMJYnGs6ZtjpX0Ix662WTV5GZOMNMhhXwCG+ly\\\\\\\/LXwgHO65MC4kg\\\\\\\/qp5axkfIZ4tPCUqR3NqHkif7OcHcm2EpB+d82K8STjEie7vYA323sc7VGYUxXC5F6j4ImbQxcKqdMKsxwcR2yghayu7u8zsJO8qR6bmvX2v\\\\\\\/udlNgzOMnQFcGNMeGGH+a0RBjPwm5U0U+7r1ThFlrTmQrcfJ1iZnGqGe3qO\\\\\\\/u44XEE4hFEaeKWVHi8uPvHs01x+qBzeKJlntY3OXi3\\\\\\\/6vhm24JyBjE1XmfvXZn7qFYKwTnnPtrNLVBPn1qeK22mYswJsd895E1UN1RnQ4x+4nL2PXkLJpdLX56R5uEQ8HPk\\\\\\\/NEkFFZDwX\\\\\\\/GJpMC\\\\\\\/uw07k9R\\\\\\\/FeNCZg8QoIY3Hkf4Q6DED70kUPwY+L1b60IBpMCPP1scdZvIVyXIBQCO1jBj6I\\\\\\\/SnVfLhWjgaOLfaSwFLu2TjPZU+BB61jSayCNKUmxxf7UzghEVPZjYa\\\\\\\/EIzmPKLQCbAKHMz6KTeegJBlZg7xelT+Gb6KSrF43z2Xw77YHfN+BDffjzpujrB2ZDrYFlLCAw19eD+uoPb3Aysrkj7AtZxjJ5jNBSON0KJcEmACvfUoMn9LCV3a0RNrQNPc8hNGXkq\\\\\\\/fpFknyviXcQzsmQSjuXhO8w8d8UshBSAPsVC3EDRnpNYRL9DKECbhRFej1tk\\\\\\\/5EPHB8HTGNlTsq0H3TEbDbgzxfiIGnym0GffY2ZTk12xle6x1gFsEIRuDI38IYfuO7BPWPtXus+ByUhF2YFEX06hZ1USYb6xPV9Tdje2gZXDulCmcDV2SWL+FMf1UUQXqfZca2iD8qlv5sVJfCxF3CKZ\\\u0022,\\\u0022iv\\\u0022:\\\u0022M0t3ymzMAmVw0Z1KKViJgw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002278f1ca7f57256d0e427b83f42219ccee7b5a0259d88227e28e272a9ff9e3fe43\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228\\\\\\\/4W9P5Geq+TdTvnpy\\\\\\\/295pZGh6dGzuo3nQIpDJ+ax\\\\\\\/kOD\\\\\\\/bsxZSkjB7sxgw4LhJL4sqvad2YkFCRBy0HoiVvTcerHIBhfo6zeeFozIBsftd8I7XVnahIS0N+J3YhGcPyD0pU0mDYJFt+9SWOn9qtmURQkIzPxgWVvzXezCD20gTtbaQf7kEAA+64kEZFpFSlZxL1t3GPPcKA3T0iw1fFw9dBXnG76Kd+uopAey98rKXfLXiWaba1rQHQeVAWxzr89NleWVXEchv4IE0aYlRhySq5pnwzewJoPumYPUvq1wJI5CLrJ4COxfATrmfEHeUwFLzCv8lcVI6QNaqd2agu8XcO2+GfS7pRHwvKjqkjpobJjIeP4z7jQI6HqtlZfzsu9iHnyORvAakC\\\\\\\/Z7FSxdO4NAXqBUKc+A9HcJstH3NT71e+OGXm8kiK8aLAT2HEsgSNJ6i+h9j6A3c36ANWmevR+kVfRzPSEuoiApKFRqmIoXjZUffSJ\\\\\\\/6c8X5DmM6lsGVJSsWPm1+ITfn7f12uvVgcNdnPJe95ifdnjS9hDrFpabOJLr8i54tRL6MjXzoUq2wp+PqPrFcJj2T8HSNvS63E2cbLP2LkKpmQYQM\\\\\\\/Oe06fLK2aKjMKkTz1m0O4NYTvQja74dimytvLx2KMjAUohKXpdjBP0SjULDxLo8\\\\\\\/eaRKLqBuTVnmj9uRopmvX4bobes7ombpKlJkj9trgA3v2r+ATBUqmS+xXA0V7WC6He8708uMihTHIMa3nU0p\\\\\\\/p5sNCdPGMEL4jG8rBZ7ffEzS1+P7OdTW9X1pvbFjUYrDb21mZYwCfOf9TwAlNeqPVXIlHDkan6YUjIa1aR6VkYbwoR3QtE6NSwcKK950hu3p5vvXiVpoCt972Fvu2T2SJG2UB4RJ2Bh9sdRosRga48IPgCVQ8H5iQGYL9LCHw5Afq5NDGmmO65gYtSUHW2WA4nAZQw\\\\\\\/MGALEwgP2INHER3Dogy9BaKOJKRH2pyZOZeaoR6wMveB0LkM60WYYoq4N07wE6FZys97\\\\\\\/sNWkrnchBn\\\\\\\/M7jWDtRagjYcTpVKGjy2gDxH2efZUGOnGDzdibqSTGzzD3HphcdaFTRhUYHc\\\\\\\/3yITa5n2b65XzUQGe13+HvdF6Doocymng9AzJ5vzueVwRQPfycH6d9xFOLws11ab52LFGaNH5Ed10q+e4Se0PDKkKOq2fnZJ0jeYCxxZikAJeg5nAjNuArbO0rhquriehx+Hb+U1C3jSZrcYLul9V89M1FrIajpif7SD6rpzQSwgyzsSApgj6Vzuv6akOENckzOxHnAbGeXKAqVprJiywPCgPnz3GU7RyGkLLVBvqEPZni4CX7NqezowtuRQzH6rVzo2dLVefXIshUk4PAboDyulH5uMSN6mOvyvDiKri9mK\\\\\\\/z5eR\\\\\\\/PU82cSbynmlALZGTjnKSGQ2T8GGqf5GXfygO48xt0hze8zRmsavz2j7sJcz2uFkgjCyB6SAc6XOc4L7mUQRE61HgFlxZojvrbBINDEanpbYgncVhLeEZdQhpZPYuHcdwCAifBFbQKw7zbC49kwDSiuWzOiDoLbwR7jIdQfWWSYvapUi35PQrEwDrgbGmZL0EiztGwX2oZeBZTKAyGLpsV8Iwu0Pcf2e\\\\\\\/pPnbt7wRRBSvyQ017YVX\\\\\\\/uAzMtjO9N8FieMjo2Keua4gyj4MkEZVqQbMZnQXfC8HL1b\\\\\\\/kem04VW2ihqPQhOPNMv5fV2AwMDpka9oLDicyK8T92Xu+NqI1vcx\\\\\\\/xbtFarH1dZScpYsRTqFt0Tj\\\\\\\/7AuuBS0VKW\\\\\\\/IJ1P6cVidtB65erMV+IWdA3RND8jmA+fuu1Ix9HbFwEu9uPMf\\\\\\\/V+GOpvFNdRzxwKfjB75xdM6NJC2kue0yiTxOFu6CtOwPpX82tdU\\\\\\\/\\\\\\\/T\\\\\\\/4QnlSk2rA\\\\\\\/z7Z5aEaBxBqmMfbN3jxGaWLOvzApnQMzZnWKPEJp26i0SAX7ZjV\\\\\\\/HFs0G4Q9eWZTjgBptX6lBrmXAhlGhF4chawJQlcXbPTmnNPDXAgDRfBXkk1AiAIobjQK6o7zNeUewu+QwEtaAOpPOwt1ItZCo8fP6zteJR3P2QQyNgAJRZCWMX\\\\\\\/EKuSJh+ZeHVBMQppF3i+KIZzF2ng5wrn2MxgZOfsUvkLpRB1m1bYB2KVJAascNsYJHrwz1E+jYg9leZe6sDSsSfVhjHEX6NOzlsDvs0P+frBNOaPvPQPVWo37ht7rJcGiwHsU172K\\\\\\\/GMayriCl5H0Em7My+9luXy5HJtoq9IXsjAeJIYfxGT4MfGeCXu8w2UcmpJzNGP4cgs\\\\\\\/ccMZFgTyHvo+sXUeTs9rSo7KYNaXU3viee3caJwS2KT1dBswuAyDmhgqSt3WWCaf\\\\\\\/IxILVAR0F75sXt7HMwBfvy6yub0KYG3PWmM53gJdvf2uPMM1cv2fqO2svaPMxh58dOnyBNbNvRdW7foURFuQVx\\\\\\\/A8mNco6yhgyg5+Q4Yu+Tx+4VZz45RdnYlBPVYchRzbQGg8YRuFRwhfnUnvssup9xQkh8SZP8ppzLvUZWvr\\\\\\\/negSE2JYHclKLs5cNYGT8R3v7ojCH3kSrjd4GPyfVnUjtegDp5aeXRPk6zUbWbov4GESkWk65h+tuB5nASA+yRJ\\\\\\\/YBIbXK3x8ofDOGldLNUNZixLvbGbFRbHmjP0jrufiTS7VLCsA35qY4NjmO53if42V9KQ5ewrNVENB9yLHR\\\\\\\/yJHv9wuSDIHL8fX7lO7aYS5t1yk2IhGCfJx7vFXcS6umB+GCBp2b8j6wn1hox9ycfDz6B+7hnJqS28p\\\\\\\/0cPFCWpTx7jfpuotCXSD3XqElYH1pbApP8nQphtH9fz3HS3FnqRgFAo6ioHDPwCgwS9ssM0V9mKxz+zl8ZYqPMsVlJGhPx7DOHaF9mjMb43pyjn\\\\\\\/+DyzW5hlFnpXQi+jZdXCaRuAgKDwGTxkgd3gtHf2w6QVduWSz7GznALYr2Ljo8iQuRr49DtKVGyXExo+eCK\\\\\\\/sQCHwPpnznzO1PoB7kWYqu5GU8a42IwcfYJqK5RX0H2X06vRGoUFOXhwWH338wjDE4nCMOD4L8\\\\\\\/q8ZQ5lln6B61qHCiI9VEc0VFZcmuD9r\\\\\\\/v1tafFT\\\\\\\/pkePsNbWM0YMLHhCaZUBUg9spJoYACq24YmD30fCAz6sMY05MBVHTFkBNhUtAtToPiMtKYKSG9RLb08ivkkQbQ3lIY9Zk+\\\\\\\/3Y1HNXAv0Oc5RbmyJAIbin5CYgcPy2MTCNlP52Qgbo5RW2W8fMFzWrRy8Ygupi5L\\\\\\\/KXGJrV\\\\\\\/3yEEVpPiZeRa\\\\\\\/d1r3gTBGOHm2+nPDyEKVspoluKtO69Ic7VWzDTkWsaOJwbg\\\\\\\/O4hoB7NM\\\\\\\/8afq\\\\\\\/lzQ3g7sBTF5RN++GZBKiZ8uTrlqQalW2bD2jp50wUOiZRIK1yaV33PomwJckANeRBliPHmyIbVPjhHvzap0JCb9i9GjcNpfZaoMpIUy0PIZG6j2ehryNKTI6zugucOoBbV1epdGr0OH02bgOUeJT4FBY78zEfKDz2dJcQNtiRpfQGzkrb7fFv5bhcw0WtBpnMMQXU9VU1ix5UWRuot4ZBokZ6u9J\\\\\\\/0qr\\\\\\\/0DdP7+8Oxa+4c\\\\\\\/PbwN9rDPo1pCprL0avj5th3hOkxaqWD53dSEFG2HvDF4bmPFl0vkXPXaTiPpipMmkoUMdQWmixvm5P\\\\\\\/ZyW\\\\\\\/ywXpJAcwiNr3WbT3J6tfk5EKLoXIrogSTMnf32LVUde65Owv7+2Z6C6cf5HCCtjGyZkSmwmjkxrfNTZFjGjvQCXnysy6mOHpGGUKju946SV4Z6A3ywHS0\\\\\\\/ttEAlS3+g+LZt77XBrCdTrQXEFanqSTQHQNaL4YXMJQbaWOcxqH6Y0YTZ3fn81yZ4uYoVJJNjdWxTP4EjxfKcuBt4YC9ZH6RCGrxNl8POMa4vD+2W4seZ9Uc7uBZUUVgoKI1iTnczAzJlZ5c+cHuxybWOASibrL7AvcK9gSbsUbCImkpyd8KzzzKIShZdqLHb1PVolzzSd9t+Nzj1\\\\\\\/a3VS7fQxNDc7OPn6pGKLzOIZg+rsS1+SUVoHOYu8Tj35Sk2\\\\\\\/dLlhOYrdU7VnMCs5hw\\\\\\\/QjGbxyZH+cZCNhApX4YGIBV04Ig\\\\\\\/oGXWM3mb7kjRSIrAI8rVfpkT3HufH9lOo\\\\\\\/cxw8qmKqYkzkbOJibMp+kLSIu8O2cMTerXCoJcq3dWf62v4Kch9aR4+ph0s6pZfDcwqmZ\\\\\\\/NYTwvwGxUwK95b+7FhS12T8xGcFsJMrjkS6W\\\\\\\/vPs7+h25lF766UeGQhBRt3Et8qem4J00+3adq97FzUoIoVTMLNZGDPRki4tDLLLVoq\\\\\\\/q2XfAOeqx0I6WAzZ1SzJnClhzWfYkgXOvRUsyJ8e3s9SDH3ul5lRU3LoxnUVQ6AKtYSvzfCCzRUGepp67JkSCtwOAiPueWqPKPMNNObcmT0GWMNeauQETB3yysTeT3ECJyF\\\\\\\/MzYNoG\\\\\\\/IGIA9J1Q\\\\\\\/5yc0D+8S\\\\\\\/JyPa7fhB52djoPE\\\\\\\/z8TK40sniKUmE+JsBvItuoYs5OxrWn+l0WI59QebOLX2ymPlCuFIcwCsb8kKDMpl1aZ8PC\\\\\\\/kWqR1MKY1Z8Git37AnYHy2pzimHEsKnZ4YKSDIvrNoYdO\\\\\\\/0oAIs4WTh9RZjrxcEBJnUKpGtsLwhUeD5VR6o+yIPXZtpIyaQl+m9stVYLKVhOfVx6qedWSNhkIh48Fhrz0u5Y50dKrgxQOOc5x563HHEUJOApveBQApzxFHt48aCTJ47FycZEj+utINUG4SFEOFUXPB9xHunfFlsx73RgcL7FrjkJkjGgdOoctOieZOWDkCoJl+J\\\\\\\/6fXQ6zBvFHJ4sEu5ekpcYKM8lxcTShWBB0h5mV3Hr2sSct61sWLL6Bid9cklsqcwOrYhQfwcLTGo3nE+YJhcSfBfnjCfwBCPImT0J3oyNMawwWpXYGFxaPgnJHB+TISZAOSD0N5lVnysolRELv3hpGFsNSg+0YQ4ZBSsC6NPF6HK9mtdBO1sVBCJSFRTBJRrf8CQAlmZFFCj3osO8I4WPPav7uUFIzZqZ0\\\\\\\/HdVmfgMDYO8SvXEV\\\\\\\/oF81X\\\\\\\/zsfWJrasZBrvTgQPkB1JucypO6aWVxPHD+RQ4\\\\\\\/AwGw5XCpZRPflFgNfsCrAL7eP2KqTncMTh6gHp4n9jwiIBWMbz9Nb2KWPvfEvuvXf9voBufSX3YVz9QLsmghgWLyEUXjcuGXBVXvPzIsgQEBq37FR2CwENMnf5+RR1DuS24N0xj18sFHY1HuGZ5Dh48gRtSGaoklkvZo92HaeoGzfxOBbdgnQixfSPYNaiTQCjTSFhsH8LdPex9hdAnU40lJ00BDDtqLIBC0Ro74eJFEEssMCKePggydnOA5JYftgmPkCykAvXnMQ5TiL5xoqkRGuru98bAxWyB2pa+pjApbovlZDdaofYGngsQXoRYTqGYl9HQUyHlOzw5QrPtBkr8HwX3VFvYPycRUSiZ4lFuLD9SbpBxdMbW7Oeg8\\\\\\\/LtmlxqHaRkNaTESYLh89MNo9ajqs\\\\\\\/jMhJwD6V27PuJ8fhQmikLIPl1BFAb21oAhwaNNCFFiaghb\\\\\\\/PGW+alNJy6peAFNgpyrmWm0cSmol4tZEOhLQeWvHxFL9DLqrh8Knt8S68d9SW7ZlK4OJTz+\\\\\\\/mXb7VaWnZAH8B7bxU9MfCUEREdO\\\\\\\/YWvZIo8RDaAdsSsyPuVaWjT4hysR8bXHr0xUd16I3VCTlrajv5HdULfAwqNKfXv2ksRTyQfHhv7+fwDhWoIvAsBi6moOYNpPz8MdpR5r7JTB7jAO7uE88spLcz04KjwCAnM4YuEXbZcjF5E6pIQPjWRto16XFkl16lYDE+56MTr8wPRSehM+9lgjId7+eS8v0UvpZCP4K2hdfxWUi83idGh7LfwCopeNo5ZhbCTJp4Z2y2pCTRPch98ClrKrlGHB7egy\\\\\\\/vo7JUBUXrCaZAj1JLYk+\\\\\\\/RaSxojuYyrViwq7WNawCRMmHQ5yRriQLy+8H9Ufpspc8r\\\\\\\/hl1vg1VZ66l9Fb9ScmJtD9nkCQjqD7HTFs9yVAIkNvgh+kdgaPYuhNWAW8r\\\\\\\/8cgxLY5oyQ7jBLkQ3DNx5Tvmkzg2rQP4IbPi0eR69fOZta9cseXq8taiiXFDXJ7Z9tFSFsNOLK4jPic5L13wsMl09SvZXSOhb1oMsbJIDtxXannKg2Xl0JY8isn2GZK58QKBb\\\\\\\/ShjusPhiuK19pGpdYxzc66Zto0jM2A1wn0fnfFxdXqs3nfXjwk5pDQvmryKf2wsqWr1BTVrU1MqJx15xGIWbKTD9ytizdoh9W92o74GU3jLmZSRpl6PWIxaO9fIpQFBt+QkeCC5xXCv9Urfw4nb0l9wsbSs14WEMAbwZsi0qIvoEYdSPe6F\\\\\\\/5FKJQvz1gu374d7YqdHBG90JmLSFHvWDuS3QNdlFWpHemzr4Mnbh1zvD\\\\\\\/BwbeWtYZnzqOZXqgaslzabVvNmW++hPA2VqfaPc2Lv5tx6Zit\\\\\\\/ePnfjscSlJeFySYt3iFI9QXZq1HX4tmly9m2MQ2pCSDl1YZyQFHBpNKs5gCEnZhV9\\\\\\\/HcnwMc8tsTJIEBUvC+BUE4zYxbC450SwtvLDdgxL\\\\\\\/S1Z82fdPhMcfMt+bgNYJ1qPGQm\\\\\\\/VO4P0sNleBKnov3owjbmOHUdg9Yn\\\\\\\/DPXrpHDH6cWAMj8nTaPvf4vhzWMgsryEUo+iOclmhlYdD5DwQQ0i9rPsvugN2un9amaYqmFZUnV11I80NaTXCN8ZXt\\\\\\\/hMhCCDyXV6sRiZF7HXqaMbFlRlEwpywGjF0lqmF6nWkiOwOCzZ3xupOfWsvpTE\\\\\\\/V8mYQgFQKApK08qB+Sb8o6FkFPLJYp7fSbviO0zE9vQfR2SFLuhsVEDHmPj0LcKiweAuChwl3XE+f7+UN1n7c5nc7b11esnAcwFuKKHPEyB9UOCcPL7yq10nfzg5wDaK1XybXz57wu9CHpmns9WJY3mCE8gLAGjATd\\\\\\\/a53GiiZpxfp3q98tzwzAHv2rxuy5vLjwon7kB47ivHorL1sKoXmGs21uK0hEmaNBwtIIIAIHD3ckFAqZTLiC4PCTYmVCFETiqTczifupv1r\\\\\\\/YBzvbZ\\\\\\\/orjoB0hOeb2pek+x1wDqFBEqsYyeQbkMOfdmyYKUu2jBrVG0zDknHNaE0yUH2WOhncy2EI0+5\\\\\\\/HCrnfhTOhWCDReGn+eDvgGDYx3Z0LMKoyOsLJr2s8YcovHYZzzwmZaO4FW5jiZu3GokRCX6pjSckQ6bpGC5QkUp+0EctYAaDbrO49S3aahAYUeXIBXIafMxGRVtnjS8MgE5iflFiM2f1ktDbrrvIptDPk2cZtd\\\\\\\/gRMWKXApOWCO1nkH5Q1LIjj4VVT786LPSAdGfwlWywAgSXr8oAMB\\\\\\\/y\\\\\\\/xc3oEGqG9SNpL1EF+zST782w0lHpqa\\\\\\\/Lt7rMvj\\\\\\\/qfBFnxrpJNrBBoyeT9uk5nGs6CgTuf8q70Up4iJs6qaJcMQtKfJZmU7Chf9vxCKglVkLIbxTDVQCdJ4UFmShmCCGv4JkBH\\\\\\\/XUHl2FOnUAdzIv28\\\\\\\/25+iKzpHkR5CPyJSajB4F5WgWXLK0+5IPQQsnuQ+r90qRkRE6Bo4N33Z+f6mGjEXxLhD2Nq41yq+DYwRtvR0wFfvmzTzhJ3tG4J45D4hlSMxVQVEEPbV2M2g2tfxNU4Zu5q\\\\\\\/0d7DPiqWLDDO3vbc19Kvp6BjO7G1Znn37I9Yqe2bHSLT0FntXW7ri3d4Mi98fytkRlfMYi0R+RJJDURJroNJTjWuJnoAYGs7UZEsqdb4\\\\\\\/qxnq94EjC5e798x4QhfjajxG9C\\\\\\\/KVvqGWQreNZeZweG2IuAjGTLfk\\\\\\\/nV10rCzSM5MFskZS21ERvM6eW+qIAQ2zsPO3vX9EZ5jZvXgb5UtGcrF2evxJWuc\\\\\\\/IoIvIIrIdyw3ghJwYc548fO6PFZPGc9OeeFVfXfAZsoE0Q4M6883Xq9VELD8p9c+Ugmx3+QCY9aGLT4wLYLDLdiiZAeAEUbheXuKPUj+HQV\\\\\\\/qWwfRw9tzfwy75XdKEc8N78pqJTSZYAU3WJW2iPmj\\\\\\\/xPCKBjAN3MzCWa7rNgI7xnCBIff8FhheTKf+59LOauN8AUXKdNGEbbDJo9sF1jSHqBd\\\\\\\/uoKoaOTqdCCaXft3oqQbzYFoA8TRimwDY+Yust1cG2QYvLiYO7KL6ZfV1dO1TVEhWNg2uakUzdmbjpYNWKZb7tzcBbADrcEZ7b4yjsaple7zXBOutcd0vuLINJFT6GCZ4n1aS4Utgpyj\\\\\\\/xzzZyOEHoHUdmXSfWvfExBwmltcr+bYG7WTVh08aAsUyqNKvz+umnCeujumRz1qSAsBSu36P2YLsuVeutyvs8KO6y4pVsHGkiI\\\\\\\/mXuFF4X0E4fg6qxIeCPL1qHywpVR9dUoBuJiTghahhHP4HhhgBAh7y9GyEGwJ9YwdLcksMJ3MsdQ5z1z4c1tYY2bT6toAlg5DUGaw3Ce2d\\\\\\\/V7UGYZa838eNmUXWrms2k0bMbRk18KGkORnS+QXabDPVaB1lPbxEjVNAo7KDsJ\\\\\\\/uC1xeSbjrPJbXT3TivcuzVcpzsLxOS6FmS+MTowUlkFHKxEFEpcCsuLTwH1C6QWLBwdhbOL8YvQa03n1+vX9EDUkZ9na6UbC1BZOQesarqKLBQmhvqp3SItMZ01f93ndsVBeiKrBVQkhfC4SoqiCaAtaoKHYh7jNE8bzZaqDmMeOlB7t1VuP+PnZ\\\\\\\/GdBQm9a5hzY369aBARIimKskatf\\\\\\\/nObrflea5ccbemTGA0ZXeR2iegYkxWlJ3ABeOaYpK2ggLMzSPNMPUAFFxj3qv8i23TuM1lFYc5Us04EJIygqtNH6l812e1QG7OAVmtupFL0nHEc+IScDhdoXBN\\\\\\\/pdLXOEYel6XVOa3F2J+pbnTHtLXlI4Ec44MtBtqavvDjEvMzv9dhOCzqKIdNU9\\\\\\\/q3oQDO1myinUWcsJmZSPwWjjuSLQ8NuQC1h16rKGePIA9qunIv2IUrZTPQY6eXcdtNHN9CamjWMSGNOk2TEN+bbEiRhCHh6g0RLIhyw85M0Mc9jSlsrkei5x\\\\\\\/kD\\\\\\\/q5yZROIMuHifpCYhAiUKAI2GWYtA48so7L3CpMtkdWPSe\\\\\\\/J77FFFgAJIlbsWpuR4LmdOPcDG6k2Cr+ZHP84ZYJnQkriBspsoSFULUPVbO\\\\\\\/KwKJXOwfPAZ\\\\\\\/Py5p\\\\\\\/6g6PaIWCBuJUeOcBNVxCvoYeEkv0I6guqm\\\\\\\/z1EZdcPzoZDB03Fx03ak2CNQyIXzQrFQAzMFfs4rjlZtGQsfuWuKVqo56twJM7dNSJMrZL17AA5IghuxrVfia2MMokLHeGiz+thAZyl3SwDw9NqMR8bfvGAqRyKBDlIap1XZ8gapTZtfUHywcHDpvjNzHyKKtrH8eBlMbiyu+nt\\\\\\\/OMLjzfJ31W4enI0zZhFdd8icp77n1kNAI0avlNEyRc9cpsc3u\\\\\\\/4QRkWF5Ru7Zh8LK6zY+E1gbWzuMjExE5Wgpntkkd2OxYfTSrFxUbPfqra3RCfBqZrfV8LG2trKNx7LWZbEIXn+svAsp23LkChWSN2O1lS8iEFlx8XCyPvi7\\\\\\\/\\\\\\\/3wEqYctNKFO50m7xO2bG49uycu1+8\\\\\\\/5ti\\\\\\\/8ei7bOF0AkifS2+0+Ch1prtMkl2WYnmp9mnEnKzqXPa54IEUruiKA\\\\\\\/LLTWkMR\\\\\\\/5gwIz1dlAGH6quFkPfwvszWCiTQxsr1qvPUqatkoDohA830KZ1Q728FffEPL9\\\\\\\/rBbuWUMYlVr2edSG6BTcYvr97UEiOA4Bi51RsjCTbP5lVmwfcKZXQ6gRLzmnm55\\\\\\\/XGd1OEPfRiz7vE\\\\\\\/b81pdQ6rqpZeARV9d21NKiiWx4bYI6CiF5X4+mjgGYKpoYSbtrzAkm51EvR41O6svM+rtAXqynRtwt5kyCRPfOYZbbYciQe1LmQcVPy6qYJTTLjRJYoD3OORfk4NtiVCRVGe+W\\\\\\\/D65rUes1zJmFcZS1QZtfef02PgmtXFv4oJ2g9R92RbrBz0y8xAN29WjEZ+KgrRDHK98eAIMlK8VLDCEg+B+VNaznlAzTNYEcMTYiCUQwq0hEwoGNrwaQ\\\\\\\/2kzJJlUpwjKIIb1S+VYFG0zxup8xl7Hye2Xg8ybl0\\\\\\\/yOfIteq95VQv7agk3xVyt\\\\\\\/5x4c6br6o4qSeDEZloybSdG5CQ0lZNqKokyIlsQwQGsU4wKFDsME5ZvXjlOqCmxBnapvdvhLT3rezcneh8AiqzmMVB1CzNvCaLJc9xxVLXjY3vAqN0WoD52nLpZf5UgSkYq48ZbaBNWa2fvOs+y0LY+G+cSVkvBsHw73xivNzIR9m6v49A85a\\\\\\\/CZKhkwUGWwzME2dquuuuNuGGz+N7AoohbFBtMILM01vSCJvrC2lbOwlE466qEyADsnz3jI\\\\\\\/xuUDxkuL9E8O5givjh6RD1gsOW\\\\\\\/OUTaZuPYFeDHjS1emLF4djx246zWNyLOG6QnL5\\\\\\\/mndvfmPwNi+EX4Ye05oYi\\\\\\\/Q\\\\\\\/JqjVwwZairE5K0PhQNGYFytQ8DSVFGxubZXzsy1zUI6ebofoZ2FYnut5qgwrxNdQu0yxVEOUjOcRKTA2QISsCK9aozkIxYnP10xg8w4kPRUl3iAzJVwTt0KtZbOjGgkLtPeTtkxK2S9+k5jPjP6aJBoEDonPVCU\\\\\\\/Rpnr0rFBTai9plN681oX+xYEQCvuw9+IiD7ikQTuAxAya4fTrlOscL2t+yUvDT4BiDiBZ0MKKfAuXKnpH\\\\\\\/iLWVrDypSKVVafRScrVVg7cJfqSSnTiU6TpP52SaMSP8TvH7HA+7Q3mm7fL42+Nc6oo6rCZvn9tPT6uTuB5z8B7EhiVNRbOCGxW1kecA1sfH\\\\\\\/YTRGxUjN9jOZqepGRKXQ9hPOzE5EjFIhZXrewayjM2WN37AkZckUO2aRzCfJ3exni3p3oZFdzHhXrGKFfbBZ76Y9Z9YYJvVToQFi0IdHdDk86ory0E+mOzTaVZQu2W4esrZQJLbLGh8eYJGl5MAM2IOaOBlcGgr708oFuJETIryeyPG4TAgEGAoxC9m\\\\\\\/Mr0mVjgCgX6ycAwSK5zbIQMqpI2vx7iIAWLW5nj8fVS1BCH+CZwqILM++4F3mBEDCvDWBtee2VRmBT1kQA03WCzVoZfmaRHokFy8MwAeVuHTenB+YpLLt8cY4aorNawxCQdFhAcAzJq5poi3YaeOoDbWa76ZKP5y48Kn+9cHUZ2ho4LHZNs9c7QOBTsryXRS27y32U7h7RDlFXCEcRLMcxiP7MnsENWi2lzVDj1rFT+rX+z+F89MkNetjfKnRdDZfJwQREn8wutPOuJ3E+JmJCvyn1Y5aINwinIN1BOizGBVnOmKnty44kaL4aHHk3H3D8LIwprxo75YEBvFgIRLVfyaopOBD1RDz\\\\\\\/ovdlNBXcZSJvZ8TRfFnHr8bKjQVoJOa77aL7Rxq1007AmKYNNvvSKU72T8hOeEh4Fq0TU7qmFqST2ai9eaeo5C6KHDgWmQapgxqC9nXcyWwIkFSPr7VW0k18scdq6xN7hT6Eg16+2BmklXud2NADxxGypj++Cy\\\\\\\/wN8eyxyz4rkvWWbKcKQQDR+k5D1am2PdFPI89ZnWcS6kbkuposucNWlPaV29KzWrLmGfw1IOJLnxqaubzSHrxV70xm1N7VQIl5+jaVGSqSnsObQoniqgiH02fzFVdOaeAfD6wsbzG4RlsbV8fuPTyA\\\\\\\/6ylHPeQmBHdimNgyJVxe2x432EljdEco9dZYM7DVfaWi+LFNFvi9zjaY1Vh0XGeGg1sQg0W+Vaj2UbcTaI0Q92UBD\\\\\\\/iSzoeeicFndJYl5dkqQHDrHOIQtdOIMQDoQp0E6NVNkZZJDX7vFOtmxwfG5yM3THMCBfSYZpF1SfyVuNXrOQLHEeo8ZIRJr16Nme\\\\\\\/kj1\\\\\\\/I7TVOO7+pVDB+kkFPLisoeYASE5Kmk8C7ZBLfIKnZi7P8HvFfOGKUKD9O4kHQWBc5Po3\\\\\\\/8Hoe5TaYiwsyrCwipCmIJb+4qg4hCWzRPx0RUbrQmC+leKsx7ApKO59DJ3s3kvuVc0fFuQNQLU0IGuiqLQwPljtzUZ6OZ3PCJB7SluJkaacDBIW5cfaCpj1XnRkCDaX7y3tf4hIS4Ti4hGvmVcglm7eIUTKJbsoPryXpTobDsBZR7wjvI58+zDyKe0HIHBivT2FfRHeBvU1zsLKdUr7WWkXVkUYBazGScKRZtMdxMcRyAMl\\\\\\\/SJTtt\\\\\\\/rVyH5\\\\\\\/b4VoCWjTf08vmnflLQ2\\\\\\\/my6gla1euLSS6\\\\\\\/88w+KZxfi4qNgHTfWeJ6Q0aSwnvdgCK3jNCFcOvSVa4t+X3SU4N4IDykLN6QA4NIVmNzZ0Q5O5EBnx9tLyHDq9771z9MMloWQkRcMaFof+f9KxHvmVEbcnQWL9fVa5T6iPf7uB8ScewblUTZLwYZA75L0VQmYRkNKTT4yD0XebHpEyQJP\\\\\\\/S+NI2499R155IGd8aDTWzYqgk9k8IJj3XXmjpyldyrowLyzvCo3iaTBKTYeavmiWgdptw0dKRG9GtzNtzADnwDTCmqzMwnCSPBATa4BK1pSO3FBY0pPRbvvw3FpYHmlWR2efTGYnNJV10EuOvWuqYEqgrF3zvkPRdIeDEjsn+oJsA9MfflJnYPUQVeiugdsZ6JV7Ok4rxj1FgEyFgj4cZJIWvC+o3Bs2cuRqxwSOGjiX1ZfTYAw58VrROxRBu8IX1XTXK5pZ+5iLON5LmSptBSbMn0GXx0kWnRE82PJ8rk1E2cUmq+dMieF1PRNEkZG5ieUXHn\\\\\\\/\\\\\\\/kGl3p1a9KwqU9GxmT+O6dOjfnRytawUC9Is59V0xHFuXv\\\\\\\/H64xSOTnrQLTRMI86\\\\\\\/e78w8kzETCL17GXa\\\\\\\/jm5ih42bypJGFOgh7ThwKgy5ZqQK8JK+SgEjAs+ItbvK6nAlSzllQflQV3P5VUHmfiExvdhiAVVmaerbV+Dg\\\\\\\/1cPKRLup3D63vr5PkbG1EyP88TnphuTVi\\\\\\\/dwZLHdzJkaqA30bvSHfe3uk2TN9FrPhCsH9lDOBb8gO1mWRSRTzs7\\\\\\\/bOpVm+U3Ux\\\\\\\/tZE4bmyJncWY7IpCRfB7SekQwS13pc9G75k4gBqMFrd2JOI6Iq6iJF9vqFG8gbaCG1Gg45k0gv6qLQMYrt+KTJY6r1+xSsaaw5U4lviMw+ZikSAkvO0\\\\\\\/gVSbUr+33f+6pgpyxS\\\\\\\/ANvkQNFOm16mAwhWpBdppUkqjKK6ytI1HoR4mVstVbtEf4maYmHDyrbmW0H62QoeYjeM5u6m1lAxXv6Jc+c0UBDM74ueuPXd5vAKKwYjyZhGIl3XStm1JcRiGOG7f83tjNgraBUj\\\\\\\/X7\\\\\\\/kF8yqrdNDokDKYY9uyBRO+RLgcvp7weIEH1rjrZGfaxhu6ZmyDqcohTyBkwhJebc6oHwFSCJZj1ptXMom37HifTjQsI3BtEEByw8wo2MryPYEc7aHXloEYbJ\\\\\\\/v8nFe3m3Zqc02pVDAccqrXxLJOHuQGVyneaaErCBeB+7HsVhCfqZ1tORh7rhGZO0sk2+0uEbaVpZTk5IxN0emmI6z6PGzSpKcCtT4rILyXvoFSXoNMJeeI7Kngu63UYZBeDtGTBjyC9DmhozJAG6rtFeznDPurefPcKbZkWb8DstCqDGng6\\\\\\\/1vSBCWfF3RlJK3W04rwj6lceei1NVl4oYPzIiXo28amN8XcL+WEo93y8cJ3b9\\\\\\\/YKjLMIoRfyV8rMXIp5XuZGc1+E1NBCs0b6zAHwmL6NH4gKxiNLRnzlxQOoIiQFDCOT4DyZ4\\\\\\\/wo+ocFK3+YRYAE3otGygL3QRl\\\\\\\/ySCL3iSWvoMCIdZcybGz8hTHeez2pYKN6kbiyAvXobOfkX+7AmNqwOb3l7Fd2Ka5BjqhkyJuBHabQjgSSIY+H94uwMo1ALKiULjXktFiQl0V32aPSpw81xEJQCgKqDYo1ymBSLJ+0WzePSvWO1rHcJV1tkaN7ZYZ661KTLYt6GFwbIvxkDZemENc9k6ZsVdZuw58GJhApiS6qv1R6JPPPafc0QJBzXtw0nVq0JmYPTk\\\\\\\/\\\\\\\/L3LJ5A+FYT3HvN+qL4RPjZUkstkh48SYRaTWf9bnBljJ6ze+Ukh\\\\\\\/TLBaDn6zEn4FMGHtQ0\\\\\\\/Lf1aBA6BcptsIsB5hPbo9+0u0MbjHsdrShGeWig66WBf94sqXKce8OIIiqC3M3lDtuzjW8osE3GJd7AK2qYBXB08QWogC0k2sTqPPV6bxUt2ioywtYRfejUAHNLZDAy121CH\\\\\\\/lr\\\\\\\/hcAtEAoelNh3IHzD7N9HMaXmF1IZJ2o+\\\\\\\/RFz+Xo2Wyef3+Jf2jnx8esPu1uE5A6VNJPzYKrVL2QX3lz+bIIo8ukCtf5laCrEYGUVsuXHgilEH3NTF1ecf1+6Oxgl2KJYTkx0i5jQvE1v7dwUzGr3CpusVcmRjG34105AeoaODJf7cmwkYSNJodm0bfvqqer0C7i5hSiJVPtIl602pRGBmKkGzK9WsIyv99lOf02S16bzOK3nM0AmM2cyA1PuT\\\\\\\/3on5jBlfnkvJfYQJAZ\\\\\\\/ar+O9hR92my8ywqkARpL+eRkoOM4D2XIM0CQ0woYvM1D8CbiXnwE7ligebE7yf6o4zhQX1AYHQrY4AlEa66W5Fn\\\\\\\/eKqzMEBBppAzYEJWeHsggQ1qrZCasEvCdA6uh11AQKeI2E4JoHPugRmex0hC8NjYwx9yyCowW70eWjf8ZworImCBywzG7YlBdkNyOwd899k95ySh7WoFgfnB6sgRVKAeixhwCQOnnUM+ljsWAlpHETxmQi5RBbRz6lq4MiUwHa2uFttpAjXTua93y+v9kZRHgE+ZTDwtsvsc5DHvxhXpFNvsB\\\\\\\/9ixPNdDLxsDeGCrI+NH8zZ0Slg8KDaIGCBCswAxrTB2c6tjEb357do\\\\\\\/nt+OQR5s6n0qPvsmS74m83z363cF2\\\\\\\/o90TrmmSK3j+MYaoLCbAGNuamL0KWt1yLPR0yB6NlLcxCjfXInLquMEyzI2HH6E9eAGN7Hww+UOQ23yA7KdBEVPhVuno5TvIBGrG5KT9T9\\\\\\\/wsXSx4kU8SW9N+7BZMyUE1jQbLK1OVTu\\\\\\\/vcn08kkqHCOan0zZfdz8vK17urXgLftNy9lNQZAfZU6ItAqzV+aGBXkkKX7aIrmmI4MDzalWk03iYTFspKVm7Z5PvnXH7XMiJvcY\\\\\\\/EqfnHu4z\\\\\\\/K2ubgdtA13j\\\\\\\/J3yhLaSifbmegm9oVvXs4xr2GfykBAipg6e6XrAnv3kdMVzQx0qEG+eH+T28JqnFGLyrIMNxxrCk6kdW7iPQIlGekXbsm8s7fS7dtyWUNI+Twm+0ZfKw4fWt7dTYexhZwEgUIsQdQdpRkfFTWF\\\\\\\/bKurTOybe6tvNdCp4l6v8AgVy6fOLlqmjPzZl7ZsWtODRO7+SKxK+wFE65tBwDnjui19JAHpcP3F9vxp87wHIVj0AGQEStctprmm6Au8YncgRUmcpWEeVzcD780D0Hyv7i50WLvPlTmfjFxKlzhS3X4JhmJxFX8ebn4JFHJWsXwcv08\\\\\\\/ZVX11Vtodxp7g\\\\\\\/5x+nKfcp52QY+WS8eO5cOBlAa4aaYmlPe71XXkztoGxL6W+iJTTBZQHX6f7wZqmbAtR8yESOdLyezM5xN\\\\\\\/c+iqR3WFrIe2B971hMtLCeHsPfY\\\\\\\/Q4i66HipLpeGoOL8cbZWBNEhUZD2X7wRGkL+mBCzS7m5K7or4X5kFxNwGVE2nldUK+1LvLFq1rhv9YxeVG7Cq\\\\\\\/LGER0A0Rdx4VcCWy2zULQWJj5ZF4ZJX4xWZki7T6uYolzz440nxJf\\\\\\\/8PlyviOi3Xct6MXiBzose8SqIK0EYlpbjxjGJdZwmNSsBiFHBwFLR74o6JF7wa99qlM8GG6jRwysilDme\\\\\\\/M9XLYmODfMQ9p3q9hhbgm5rHfodVEB\\\\\\\/WdiDmHqBAZxM5uD1GrVGzu5ilLlZmemInNphK\\\\\\\/1EoskFuleHoy4K3UUkjlBljkT6Agb1VFYvDoYJGjLsCQFg0zH66QYfiZmSOg2d+Y95VZJha24WZCOShOLJyiLU3ZxeKhwgv4NG26RZH1D5ajhSN\\\\\\\/fGX+a6v\\\\\\\/vjW61ttmm1hvu3WFXl4OSKxG+kg5gAqafFIhTs\\\\\\\/1fRJtxFIsBMwN0b+ME2pyaJhI42mJ3COotpxdFKtZarJyhgR5V7rpP5vG\\\\\\\/DY81DVWBG3OM90zLrH3+anBebsFRQQjQnFXNOvoH7L9sS30oXkfMMsu5RPJNVH4s12bnG6Yw2I5xaFu0C8fQuQLddafIT2layRnfc02HvfE01rao1hbTf+Zcs2nHHmphemtUiCIhvk\\\\\\\/lCfa6X7evyyZoGfvz9CnCWxXyQfOtcYx1OfpjBjB0iETrwVaWjp0bTYwteLMikPbx+Rm+7cYnACf+RB8DMl5z4IwF9aru7Oh20d1gBVDfKzsRucr8TCU3SlsgmulTagdaTFqzr8Pvszs7PxnJxMXIBd0+T0pcSbk5yLvhuIJ3k0V\\\\\\\/\\\\\\\/sZuXZBvjWodJZrXIJxuu1H9fqy9dYZ5aA79i+3nE8XMZdGvWoYbKifIKWsdmE62sMvVA5zUIaqbtK\\\\\\\/\\\\\\\/BK22E8O5TBj1ttDTLcKGmZLiT1cKssHWU7dAG14oAMp2t9iaeFoqUD2X1LRx5MKOYL\\\\\\\/RVANhy\\\\\\\/pt5pXB\\\\\\\/wcnZMEu4R37+S9aCwMq9QbfLL2Tp7MkQCAdIhEJdhWlpzboIoKLoosV3nxr9f4KLYHzb8COgimxQesym6zhNYE40mp8dKnS5JYW8oGtbJNr7vb1jgCkLerD7ULUH58tbJSpVyhHPUIZdMlWGHGd0MtJzK0WuLYELZtMnQS4vHRczVRR71B5hLiRah5trtVhA+Ink5SYZDzVXoYcmPUJTlYs8o3rQlEYMVYFqDhneUQmDXdOGTNR7YytqPRaTQhBi+2aSHTcJxa8AIL4TjwxWCRJvswjJTFHmjU51NW0XIjsnbkth+OcsbKG8Ixuk0x1qZ+45LJmDNb2cmkPbZC9YmGrCD3YCTAAbfgTModWIBIxz9g\\\\\\\/pnIx4H5j2crxcxsnusLFysVNfnnwBvfGwYf1Z\\\\\\\/9A0uLhMVIO5CY\\\\\\\/BBUO7feKs241Tb\\\\\\\/a8PUtsTIfVnIK0FhHEmxpGRWKoO3ZZQIZ\\\\\\\/Shqe4sEmCKk1HSqcO1bH74P\\\\\\\/sfzfnVQvwseh\\\\\\\/ysE7FKvQkzXOr2Uo8TGRwHHbfxm9Biyu1mcsQ0K++ia7m+QzgPoMs9pX9MLIkfW2lFFKeILS7OtfS9pJP2dTjGv+r95N3MEqYlKa7JSzL\\\\\\\/QjmIo05eIBSE\\\\\\\/UZux8xbisx0WFfMCDY826sCZbUP9smouNTJqp9tMG7Hkh1wWq61gtTamLNOL5tAAbtNz7GbiXBG5pPvVhnZWIot7C+rnO1lih+k3Ew642LMY7KsSpmDMP6Xz0ExrIdbJdJirgiTtjxHaWAUIrhNX3pJAvBLmjSSPbkYBCsTF+ijJ5nXDpJBpJaP3Qm1nDHTc1IjkTpGfg92cWkKxFHcdd53FDZ642uzhQgkpB4WCki3Pk8NOyE4kIsol8XoxWy2F2ldKLSBO3zOhb4WI31hlemeIIIIfgX5wWb521H4o70o2Hhi66uA+7e7eiumkg9lLYo2qsxp8N+K6FfgNQLGbxufY\\\\\\\/2Hpac\\\\\\\/g+yC2KFkuUDdNEA9gqitI0NKJdrhsDn6G0TZdTfmcxQEPzcvC1oku5v3CdwwFB9oLftBndGZuWpJkOChI1H\\\\\\\/RoLP8PCQVEjTREqYtJlgKZF\\\\\\\/BWCG9nDM6MT17mLa6oV2jc\\\\\\\/UhMdyQ4ZXwsZhIT4Ch5wKMRe7CAavdO98JDmueAwUXHfqcS7FoH5lkA1X7xK50TI5ktlfiTIiRMzGZ9LBAsWBwzLof3M\\\\\\\/KoiadkxPIrLZRYkt0YdX6vcRiu0AXGhOv5XoCbRxatWZQW9Ozq43TcZibmHh5\\\\\\\/XsSY3C9ML2adNj2bPq75tHj7tmKbryL5kqjQ8l+ZvwdiR\\\\\\\/\\\\\\\/IsSbsHW6viojPcqfX\\\\\\\/wgjEgxdhqsLxbfQHxZDfhKFYhgyeMOMFvc11pRTsRkWzefsSF2Kjrdd1HoYn0sHc01zbK8Rjj3zenMG1UhDI5Bn5lXUJCbSrZVJUlubnhKZLNUWHpKd2ddEbv8WXVW2WO3lxPVLeCndZRcP7KGEGCPaGHpnwtp4Dz8FQqC6Z3\\\\\\\/fk7TAPzUahFFKJomwkxNlW4opVGfK26lZifhcCRQvIztdmZPsbyNKnuDJl4LHjbAJNA+KCbfbW6HqU0oFN48xaiAeiXsOQ4XVKLw5tgBiVMXNqFIUNTokKQiqcb8Fk4H9vN8CrC24ABhquMEKDC09+JsBUCHJPs\\\\\\\/i4v8C95stE6V4NigI2a+4dbZJRhhkfxglwrGmxvWnmzYj+sHbq8EjpHPsZoxvwz\\\\\\\/LvuX45lc2BUsQ2ov9V74bxRUyP\\\\\\\/6rR5bIwpSukf+pFl\\\\\\\/1Qg5ggQAD2x6CO74nIbV285UQ5Txb7GhXfJKakgWyz\\\\\\\/+UH66Px4rQfKN9\\\\\\\/e3TFApjRmy1qEUzN+\\\\\\\/C\\\\\\\/tETj2Ghp6qcljPEOHcUL4\\\\\\\/BAUxJ80HRtEKH61+h00JSIAvdqL9A\\\\\\\/d8QBZrjvks3ChPJGrKMu9Vyzd7grE2yGMWms1vHg68DwFHsPH6K8ZBZePPtZE5ylgrOyHrE4AXz+H4ou8v5\\\\\\\/lrvFDMSNXYMcfbTq+XsEWh9Jg22\\\\\\\/V\\\\\\\/7lzy6UdVhDZRD0u\\\\\\\/sWxRClYr4VIUjBYvZG\\\\\\\/GEX\\\\\\\/WeQbaPUyh2KLssD1CXyo\\\\\\\/hCd1OrNWXXWiZOqrtWlPVKC1lszEOOckwHOaWmmbonFbk2UoqDgFeZheZf\\\\\\\/awvd89WlC\\\\\\\/KCESmMkwhX605OGyPTb86\\\\\\\/AbOBbA9UrDDPGFHrb1ZEL\\\\\\\/NF7toxNZsDtIwPl\\\\\\\/UeYGxvsJKAoxUIIf1FfwMKCsRp1\\\\\\\/Y+6UbvdeYU\\\\\\\/KKbcjBPJfxJaqLT6ak3jja0lg9boc26csYCPyXq+szG8deNkG3UAWj\\\\\\\/iwVrW6dtVOhFOVa4ETkiwwLH5ZB2O3oVNmAW5pXi+bF7eav+sZsqVqk7Xt2IrHsBM0ItMt8FWIUZ4i1LuT4n0YzVccqKG+INK\\\\\\\/E2BEi8OOtGcABuClSqx1aFjEa1ulEv1MWkF6D4q23+8hnIAE1gQcbVCuNxwEMxvoiYRtDLiMDz\\\\\\\/ypgAcAA7APLmvEGn6tuDyNjnGFVHiazccouX4dNt\\\\\\\/8BemL\\\\\\\/SSobb4Wb9XFbHbqgUHeWEOi+3ge95aQkWdGe4PNT9HLBE4dZ+3cyp3Oq4VZjJnPQR7ttI\\\\\\\/s2+Q6+6Vpg1dqaAgiHzPWl\\\\\\\/8v7NGnqNDuQYJE1uJbwNmLAGhwdpzLfB\\\\\\\/qnPyOxKwMBM5\\\\\\\/IWmxHyILdsUE0oO4OS8ZPpmoeF76PQ\\\\\\\/P\\\\\\\/6QhQhyBhhZHvHs2P6D937hRWPvjuAtKdsu5mV5B1jTZgQRH7YXRmvPJzXAHYYgAzTCLHO8CwNlarbCqJhrG0yuKYXuSORw2Agz0C\\\\\\\/54c5Jvr1kQznHvGboVe4fYtI2TWIDavQLFVRrLwgvhvjkOhyJlFy\\\\\\\/zlHFYTYsho9tWAuF6dEOPRBF\\\\\\\/ThX6Dmt1IJwHzinDlN1jO41P73HnuCFhLj1YPC5w3ORghTQl1en5B\\\\\\\/pyFaKJ7UgBF2x4LwuE8rF2as8r7iY8PZdG9pyTqrv\\\\\\\/R2Oo7STGJV+xbtha2L6JgtdEeY6oSxLefNr2GaUPaUio357p+0ffJjv+GhwRkFY6vN3jC3etxuAuKlRzvHkjCsXy1j39TZXkIkFG+D71pWXnmj583wk8HEPI0ewpLem+DM2YtG8BTaxVSnho3bmB7MZw9khuVxfZu9BDsV0K75\\\\\\\/UzopJBgg1AVZYSKLKjSmh7UM351WUrjFlseyWQcekoJ\\\\\\\/xqPBye52dt41GB+jWfbWQb0utgRLrcD3AB8+6am3dYgeTtAScFf\\\\\\\/lZ7BLUcTQ5VATeHss5JsXyc61M+SyO9Ex1tBBQ6L79WAKYfnCAZ0X8iehV28SAbGkp211HC7Qeq1gIQSUcihMqZa6J9087ZcloZ5QWXzYES6EhAqoA+3JRKQ7vANT3szjUJqRcZC80fxhYAQLiY5xDU6Rm7JpnijC0TjcZDsuVA4H1kGSxZJADVk6lhgTJDKRgci+nAcm\\\\\\\/56rwQGMbdNvnBk6LHGOpmWSsjaFzwD9L7rtQy6bvnhHB4wozQ4aFuo9zjRRg9WsLVR+96EytdD9i4kDin7lBVhz5cYlZG3smDuoBrcU2GJFDTdZzMZ8kPoQo2a7\\\\\\\/W3o1RA+rFa2XaZRINqkFBBfJhcKxFxaHg\\\\\\\/f5ZcJx3wPEFishwi2P2amYq+Zq1hdOlxzebXc\\\\\\\/pq8hHyCBFbpPG7bggiyBKdmhSGhH6XOjKKJpgSQvD7ZU00e5shvwhJgshDCFx8kGmfizcHjFU5mip3mPDnPrJyOj2OaCDzzPmSwjzln7an8wUUyhY2vJ3ow4ZMGFlhXzIBaFQuSlNEI8dDM7PmyjmQzE7NukOV38AwIkLXxHZMnKrz6FQnqfoc+BUVGR3bJypIfATGSl2eTrTF1w1UAG0J3CYANHUXNBYa1uwjvi67tUEVLAb\\\\\\\/K6\\\\\\\/bkRffpQYGrTZnWUIBsZ0CJAQ7F6A0NNe9gef5+6Ujfu8V0jvdRhc6HgQ2+sF+dVzYVOopZNGYn40dUzeN+BBtu0+rSmSDyB6zP65IVIRZXNYjWibSp7BBY7lPrfw+CPwuSzfIiLhWk1yk2foEwQPzsPCT92jtuHng40wgUj69dpgCohku6DiuZ1yOBdhdD+ZYySqKM+CDAd3bbqPSXSaEuWZIgp\\\\\\\/mXGwqifnQiaa+AfnxMQS1DmIkNyLgsfsUYn8RBAZMmSjQO1Ub23kVrwTepTTwQrv23rro02j3vfyODjghmfq9npQIx\\\\\\\/O3WW7r4eew9foVMTI5vt7MXI7ReGhHW6S5UUSdcuw4N\\\\\\\/BoWu356zodqWoo2sE3O+pBUlSskaB3hlElwqXNsKVfpmCWmu2qmrPirce4ANyrn71OXoBrWuSwjEJGB\\\\\\\/9tobjEnwV52ifVBcrDOGucS+Fb+MyNdfajOe6eSKjRhN6yWgZ0ckny6upQ4C940COfStCchMw+u7wZH+cPZY5F0SDWPqHR+h5ayN7L8qj01ww6ll4UumYgosEi5CjD7YLovvcbgk02BenipdWovXL7aEgk2WIXxNbFe0qDa1Q4TtCXQrHMSVQM6g1ybBxKO0fOJjMFB9FDM4N2qZTDcm5KgIo\\\\\\\/2z9HI6Qi9wEpKRjEvxU33HBDUdXcqyy8tZr3+oBQ8s9LNl92rOGljdkRv9OKONQfX4dtJpU8INUrtkQgUFQM1If28oDsFRiOst+1ZiCN8hFbeQIOhHNn40eRrxNWPSmbdbyxKrGlK3qHY3ZjMPdOauqvDyHvcIQmt7+jiAuV2alZWH0dnSr0\\\\\\\/JHgXANWD1Y2BCG2gbSnAnK6tXhrd4N1X2h3uBCfCDMMVp1DxryJbqzrkBGmESX+UzA7fFT9bJNV+slbX+pk6bmYsn49R14ERQTKHc4JPgjyFopM8wLPF\\\\\\\/LUojULhB0BNew0WoW1i1CHeFgc5xBIAHoYh20vtNkwKQylaXtMmVbXtqwDblzB6BfQ01eiPP523rrSf7JQPF4jRTo7kKgePI3SPQAMNlddWgcj0t7ID4VvmPxs75jHCvXEvr5joSYCDJ0mFDGx2YvOgqiqAD2HrU5+64Y23haXTywNaXFPsrDXkhMOn8rCUrgqDxuuW564YElgu3QAN2SBV4ZTCSMysC8xzili6O2FfWI8iDhszoUfPSToHHJfboo3hgFT5nhNM6eaJANpibeg+lSmF\\\\\\\/Xo5jOaSsUrbNSox53OR8awYp89gO27vfwZVa2aXgL4AfieLSO7KCDWzC+IYW5ANaWQtYWk8yyggj2O\\\\\\\/i\\\\\\\/A7sJtaj38G1Zua+n\\\\\\\/uJ65kO7AUUp20LRSDOa5Qg3ilr\\\\\\\/Ce\\\\\\\/RNXlt58CcK7zL6g8KlYrNsCOt7REP94cRVZHGLDX1mei1xLoEKQKceqZmvpH4fRZZmNCWpyDighotlz64scNN\\\\\\\/02VllgkJjnuIwzvT8afVjua7gfR7\\\\\\\/MEW\\\\\\\/gmwmnwJUxqToTSzRTp81PTm6hwLxW6npw\\\\\\\/\\\\\\\/n5sU2pdHXGcA4\\\\\\\/SYkWaDVwKrT0ZV1C4b8opiDlTv3v\\\\\\\/3fCva3wngR6QYo4PgVr\\\\\\\/6ZlaI81ecx5tkd2G9qlnvZG7csqOzVkvU9YBepYxlY9DX9pLMcT5ecy\\\\\\\/IqSllVU2VQ0EQ2oEXG0CZgN3jACIBO5pJT\\\\\\\/oKneBejAq+TV5xTJtI2D7g0IjBTL7paymfyUkoIbYfQiXM2DfVLBF\\\\\\\/tdZx7s0bsJj1Pn0yMBKKUB5Vd1OTXjmMeINNxb30xywyzNK55Kk8KZo7Ufc1d4EwemF\\\\\\\/jD+SI04sTYuxEB2HI07WVVj7qJ9lvoQrK5j6MwymWa9FRAh0NSlsd2Wb2OPIWuDpp54l\\\\\\\/6I4a6ea+VSOfjiGtiO3htnywf1kougyXBRN2xbhFGZntPoIx4fit0HUClvxVDY5aw1nQoU8hsyZHkapi+AyUKjHjhIRv1zNQTgyY1xTh3jJpIK0MbU1j\\\\\\\/+dU2JTCETGJ8W3g2+qeeVhALx67efDFcW1KxJ8ikwTOA1ADlOioP\\\\\\\/7Uae7fAd4CYq\\\\\\\/CsK9ngGpefF7m2XUJUpB7E4BUg74+gVVK2XOWSIoUYk+\\\\\\\/X2lVAt8aFgdaad0bOKnsWHKTaSFISGxnAII\\\\\\\/Z86Per1WprBBeIR7B9tqMFDcegiN4KNP8l2RmoQ7\\\\\\\/nNFYhw6qqnJwsjl7wvk1C7K\\\\\\\/+6Mh2JI0VwQvSvInMKuAe\\\\\\\/6\\\\\\\/gQPMhs9mpbl\\\\\\\/s8wVsjLdrzGH8UP+pYGsuW5WZG0CKDdsdHwdIHAhknsLrmE0xjCGmPaD81bfnTvXLG8gDDXLHf4c9kHwXpq3afnptHP1fXwRyoL7GOOaEVR4Rni+oRfr7EWtKcETdtZWIJ3lQGgOcKH1h0rM\\\\\\\/JVsFS5wOWLceTV9N9ldyLWto8SUgkG31nRhO6dOB36rN7yla2RqL4JOPF5kjYq7Gs\\\\\\\/Wua2aBJ7obQiL7r7kBLfbLNkIpeM1BbK1r2A8kcytDovJQbGbKz292NtcE8I5SJsXl3g+UV+AWzUrSLo0yTr9lO4ub00mAAcmNItSfExv2vJXFHSokw07YBwMvb96KkX8\\\\\\\/D96pp10XdkKwtJywLPvTpONTiAtXnaGHdaCIkdbSaBO2r2g5MGs4EZ8DnYg+1GUaSf5j6u5dcrnPqJLQ6iia+lK1VkcWkp8qeWRG0SuGb0REVBak4\\\\\\\/nQd5tM\\\\\\\/yfVJTSd1FTtYilv3\\\\\\\/i26iyjsRA8Q0u12OprqPaP5kqai\\\\\\\/MhjAKdm62RDCHwM9n9LewScOrP5fhGDbWKR41\\\\\\\/G6ApanaeuozyKSuuoeyW9MRYQGOVXhHFZt94nBqS9TfsQ3r6t+CfWo4FCxYjfkK5TiDKFO3V598slFhdoebp5bWQgvlkSsVypPWxpy57ltC1HY9XpNo8WA+B8UlyoIsiUYRyx2iPZBhWFUh6Nr4kvNdqFf7snjqSUmAv1KATA7iQTWajvhcxLx6c59TvFSluvrKzpSxP+T0pWBvcQoUdY5CaGjZfXnPatm65kl7lrrOfV+CK\\\\\\\/tmj64BTfpD3qG3cfuu\\\\\\\/ofQb043oh\\\\\\\/jaNJA+rePg452w\\\\\\\/Rj8hxCk5KIHUEVmzDVXF39GkM4H3KLQyOQfWxuC4SR\\\\\\\/klocuebLpXQXyEvdkf5jRhmMNRLtzohbPqBPyi8Dgs+dFglF9AoqqouvXr\\\\\\\/s1qTPyUGIFUr5C1KF3BFgYq4PwgKd1xzGdxupf9+0yCPkfbjVdUFJ6bZOncb67hD\\\\\\\/umWz7hsM1gXpFDY57qnVl3AgHETFsFO9ZEX+6K+eEJ+Bqv36yXodNeuzb5aLZFaye\\\\\\\/QYxOsAKunzO16q9mzK0jN5b0Dn2+Yos6PEJEkmqiaz52njCu8evQGVe\\\\\\\/TxSXCoRImwNkvJHNseube5psDMieVY+9vHOiIfk\\\\\\\/eJjYL7RVGfWYUf2XKKFkQemyh28hU+sG1mF8ih8XLVcRwTKnfuZTEKfZr9bxoZxLvvQTDgG1\\\\\\\/Z44eVP80IJ8KXKqhc+tNDt4b3UWBDZFw4FnbuJtTM5eM6faz948zfSkXPmA2qo7WHtYKFQLj1LmcpKMtT+TO4ROGHwvJ8LcUB\\\\\\\/XlMWFf1etuRpGehV8bUureOaN6F3fvzAxp11FwIdcaiO+H3gNSiUNmohvMQsHACPRZ9cZjzmNqmb4UckSiX7kltFa\\\\\\\/5LJ0UubQ5jv\\\\\\\/\\\\\\\/XNMPnIlMzCI+AKpm4Boautj4EhbfRYPil0WPNfk+9bVLPaRyyFRfw6R7yGN\\\\\\\/VCiOcWyxKmkcUNO8YDT8S+B+1q6fTHb6f0NBiIaFjP3vQeZEfh3Duu8rcXnmxJL\\\\\\\/hXNB++WuNvqShvOtnaXbAm4BDuZdFbGocL5vA7xGnExVWrdMTs8Gaj1QU4riLrLy9JoRDJZGD9CEfswbjzeq4Bc7m26cnrOzlCMrheFyT4pNp5CV7WKsuC642Be0Li4OgdDLxuxDvQOJWxtyu1Z82QkIlXtYmJjMYgbFuu7TtcHaFJsEkpEJYAdWHxDU3xcvICp4P+gY1QTbs4CPXCEvGstR4uKhOmFN4d0EJ8yBHLqNL1RTGnpfXAogcPnLTL5duWET5OktjwtzTmpaNuEvqcwg263kO6NGWs1ZchOaF+29vLy8qfE8ryxPhktU0Gl7iTuH972tsq3MqsYWwli1fF5hAICWu+xszy8ieJgWxUN75rPGK658R0MxfgLvvseyzIn1gGHrEMlcptA8xKjZUx8mMBaYg6VGY3isp8XYvId\\\\\\\/H8J\\\\\\\/3HDoR\\\\\\\/bOH+jvS+cer8da8mffzFkuY+bDdn4QH\\\\\\\/++3K3hqA99nhBsIUCzrSqJovL6dJR0Tq3s3uLBNDdX\\\\\\\/BYsQnoK3y4ngtfaqa2\\\\\\\/CHoMd7SFcLGzWB5uV7CudmaYmN+yquRM4kTPBouiOmk7svnZ1Y5Mngjfkoeclj0W++lQRa2mvYw654gB4CzB+EJNelvdcY8qRCdvOxWd4XyvwBc6qmIV1FC1DJx9EyQg4s3XyAUw\\\\\\\/isMhLkNXsbLbV0PlZGK18M++64fVuYDlBBOHcK0PXXcD6okh4mOOeozolaA7ksnJ7DLIyb\\\\\\\/ool6RvxAFCJbHUNhWtMj53bGkN2Om7Wxw99+EbEoeCD9IeaYfyy+56firo6g2ueT6Qf5dlPcfDnpEysFpWbxZeNBlzTLInm+1SDF2zavQWT9bA6yykU\\\\\\\/U\\\\\\\/WSDCuxjSFRaq6K41QgKXbx83iG4Uk03Pgbnbzb\\\\\\\/z7SdS\\\\\\\/wI3KK1EzvHLyreg2YLtiv0DHRehxVM3l3O+0oiQCyGymBliCgLlIQjZ2S6keuV9U4i0JInsBUKbS8nRVKMmV31Cal2dkQU4l27BpaSbCswF9DQ\\\\\\\/8VuKgPa3Eq2EzTmYITbZ0BlJJVkjK3zMAPimVWFHZnqNIwyRQtAuDuiHsLgdIf1rcI5MEJ6e8JmYm51dVswVCNdPkB9ZcfuatNOfCCkriwUHVfBAQLoh2DP2DrPotBEc0J1mkngfRin5KhEJkJzFMJZZFJccnOy65JKVququ66DhSPFn0xczESjqt2E+pn8YQJ3Sy7Glln5JnqW6EkfTKORfJrGEHpokbTLWc662HXkcS\\\\\\\/SM9LSweKm\\\\\\\/bGOU8f2zEE5rvd8Y5P9wJPNi7EZ1EYXkHbtQRcAXuf46tFYbAm66exZP98LPuMb1PuyuSdmmd2WZNXv8y0RnBlwqeomvMeuVVhbpPy+zD683\\\\\\\/HfhXGc7X4\\\\\\\/ZJZtuO6mSqEAn4ZUFnuwEo6yMOSLbAK0e1m685q3oMdvMX1qcNMLNX3cldVCZ3\\\\\\\/Yt709YN7m4GUZ2meJr6taHvWDURSdgvoN2pFD1ENE3eYg9VD1kULLX0Ii0Urr4p4lllt+EuDLtP1HGKyPBrPtZ0vpzdC5AZ2gaD\\\\\\\/Gj43exomvKajH6D+ORvFzrXHWIw14D\\\\\\\/Low0zW0L9B4rfhPVgykvnBieMDT8WA6O+E3WfqpmMfDeNzgAKa00u+MIBLAoLt1UN4UfD59nLY+JMT0Wlm7CMsshBG0iFDEj6fSxbwLbWnksW327BVG07Z7sgSUSwNy6RgStEMDed4AN11aD+B7WZ2ipCPfPT9eJly\\\\\\\/rUe3OjGo65VjUn5yMciDDgGZ3aPfdpo\\\\\\\/mrNGSr92useD8LDxykz3+xTzwg7Zzcakzs2PQkFKtmAsyOmGFpD4mDgGpg4E\\\\\\\/wmWshQuYwKVlK39uufaWQBTCGpVNARfqhiJ7ugMBmqItqBxWF9yoXKS8osWtRyctLGy2xEZzXxyB2DKndkJtNZlwf0ssklsxMEjVYqPcFJwMC3Wfe2C0\\\\\\\/0r1BuAVK\\\\\\\/TIekuuFUeBos9k8UFVxDqx72FoQgRTDY8p2tK347kZuc53bPrzzpvNH3wYKxxX4kc6yLl+oBL01V9QYjomAcsfwznqAhZ012izNMYFY3rpSm9sueZU3fi3y+mki0spn0D9EgEy5L3gqBl2W37d3i3s9BbsNcV55dbsLQgVyExWG8sRDFgB1yaokheFfHl+Sy74c5i4g66fjbAhp\\\\\\\/LpbCSobEOowdCExgkDF7kGMnRcgPVd\\\\\\\/YiqYKbjb18doMJjl+bWETS9g7Uj\\\\\\\/YQPCc7FKgaQrPTUWTC6PV2klyWrXSuP9eX96zeHSb\\\\\\\/DqwfrCymHrzZ1HdwWwVgOkHmhahhwBgiyVCRYZWv419Bn93ITvlxxB3mzKT0qVmpUxCLZnRWCVyFfWSXKGJ44SYmyIOIT\\\\\\\/MDQu0SFu0VH2RhWXCLOPdbepmyVQUVutSR2Np9cRYvI9iyBYXadxoEAizeAFAg+D6k2ywQjaz9Wi2kWEQg0nlFqplIunqXxbFGxrAzsHOk75AZ+Wn5SlGTO2j2tgLZY7S2vM95Hya79mAlSR66Vjk7usqtK+sEl70Juf43CMUQIMkwdC8IPkAPcPzG6WZeGh0R2NgTxS0nyoIhgIT78uZAP5Fo2cpsImN1bpwk\\\\\\\/1Y6CxvHlZsxoJzamb+KoPtlG6I1cbz6urtinBC8QYdYUsB2pKapJKcWySUUdvfVLSzosKfqRzrnWtXgHqbEL8QIA2d6SLY3O4Hw1QBCNz9JnNhGGkKlodUK5GD7Zn5ceYGPZdLo+fvDnIBTICEyw6I3Pmvo1nrcZqz8ah+YNB\\\\\\\/LofUbmXq38CHfjeTjjwKf93FhEWHovGiy5vg2bMAkNcruastkUGStdtMyNjNDLAlFOjNaXSLObCq6XleP\\\\\\\/JDoLP3FIxYHTHy5DgqLZwzgT3zvqOAaGl35Ipua2SgifH9rRqy617WdmkpAbC8FqZ8395adEnn89Srq6rAB4Led0E1GCtUSeKmR7L9gjNVqO5dNXBJHofQ5K9xh72L\\\\\\\/f3e2dh1q5XWbKrR3biBc9uGvgZyzE68V1gMqF3XlhDtI7NPSTOsIXIjxYjFDx8nI+kPxcOaWFwYX7yl1q6XdPLL9X+xjqxTSr7rAWpCFjwy2KUBk2f5IwgoY91DB4whf\\\\\\\/3TJmHJElatJx67\\\\\\\/6WtXNoz6NmW39I0wij1AqQ+tseA+K137uzP8vhYRz0NJw5+Ibs84NQYkgm\\\\\\\/3V6GjXDH5s3WJtozrVq0dMqkpECMbG8bp5u0zgSWzJfEYSBTJ8xEa4AiEQDJbD63YrBs0EcmuQq70z+1eAA9YrE8IsRGu+2ka+I2gH7NHM4hKBaMf8s5lvFOrdL9SK211DQhjb0BBPLx8OQc1086jCZQmlKf48fdO7IzzQQj4G8cHSi2xQzuO3D+m7SXOrYsjt4Yxq+NTRucmuOCYUUWkyFoC+cvPv42b78vwwYylZDfu6ZmZJ2HNuROYic8fGfS2h4NGdbCcka0EoIcZTXNOHXUGArne5NyfStOTW3DZUuXYTY5ECzeAas1SrBSIEAi1pzV5pdtRUjB24Ix\\\\\\\/EuH2YftQ2zU+oCGH+JX2jANLtioC0EB0+iITg0yr6UAi6lwG2+KL3v3ZUOu4Fxp6vkUTuRccHOTG5bSsI+JUctf8zwUt0mld4Sm7+FfxKq2Raqcxa\\\\\\\/zaAsq2Zzz\\\\\\\/5VM8NvTgPG\\\\\\\/oNXXVECd7TcPqJ2qgHrLBIADPIsN9h1nr4P\\\\\\\/TFE+EFtp0qejntncXuxTYk+v7goh1Sc+Tpzx8LDZVZ515yxyQiD0BW\\\\\\\/92HIgPbOALErh5fbyVTja9zQJYpbBj24B7lnPZiffqE9P19fpLpzkxOcF8ueA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Uk1ft+dR+Vwql2US0OKBLw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223a0e4287fcf0ad92e31518fafabdd28bc01fdbb282dec008f3268b9f8e89ceb2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SlTLU8+D2jaypZERR+Ams0U838l9aIONvyK8vIxVxhln3POJfhuP9EaFFnZ3Kobzub8kPi3eHjzqweYxYez51FsBxqzW3w3E7H0RkMmp4sixpMD+b2v90ynyhNqNpI6iQ7z7WXDPE7LE9U\\\\\\\/v6EHZWkmiKb7MzFRfonvT573JgbRJt4eB\\\\\\\/P9sMBZQ6Fe2GDH\\\\\\\/41Ztcag3Mf5M4YKXHh8hikYR7b059mJOtJvkrFt2iUXLNYgt0A3p3\\\\\\\/K6iDwyWax8l7uUzEapvPOzGweXGP2KHz8dYEhgaxrZgebQJdF8Z93WfiEtJ6OVjN\\\\\\\/xe+Oj3wM4DYQW3MJeJRHt75cCQ+KoFc0\\\\\\\/uBDxT99CDT1Q+snUPBfqUMYQl6KKvfVq345rCIY1xTEfISjL4KKdm4E0X09G0UG5Gz4zj5p1YSUjHl0bGYiauDG4HHJhOBX4v7Qsro81e7LJ6tMAUXgpD2aJcyL8WLVdjzi4bd8stulmy\\\\\\\/UwRSg9c4YYsUy4ucnoB8lzRDkUPM3LXKnLXrOwcrt7+iTymocmc\\\\\\\/qr9u\\\\\\\/t1FeIOxMD\\\\\\\/5TyLTLnlml93y5Z6sePAhf8TVj0ZGr+rx2GES39jilGnSShKeaGJz34pV6Kpq1VI43zSFKr0VgJhahWOGYsy2MUkz2lHR7AE3M52WO4LFTxZPwOSu2htOWsUO6ty0VmL4S3r4nNYWyLgnEcFzCh6F8d0lPuJ2e\\\\\\\/KedOWD3khfxlloWI2vwyj4yS+tbMkqO13mC55dSxv6Q+zsmkEP047OaSCdP+w1knvFO2hd+gj8HCr4DmNrKbHL+Dtbk8I64cxQj4V8FKWbHHXkWTqXXI8vQf9U2tpZ2cx4\\\\\\\/5zgDvicf1uH+HbjtxTXw+V62US2Lsk\\\\\\\/B\\\\\\\/AGWIPkUjlbNalwC8p3cayAE1o9lTmEH3sBC\\\\\\\/NqJp+2RAGQZFM4RO5m1xzT0PpgWFnP8wqZo1vsYkdbIAj53eZvLIktl+P1wJSxP8MmHSJw83\\\\\\\/syC6n+cqHmVLNApAKsEzdjLZFdyCb7\\\\\\\/lgrKm7Uk+e7Ev\\\\\\\/gGe8ApAC0c8LAejHWo1QgioipzOxdetDcGZH5MgTGYG2X37jLGkzZMjEcGkHtHltSkEu0PixNOuf\\\\\\\/SBZ6Jyh+9nXivEYPLceR4Pv8Fm+PACgHaTQ3JvLaK4CUCoxzLpg9o2Eleqm+cF1QaACtI\\\\\\\/SGpKGo7ZDKBVf\\\\\\\/wP\\\\\\\/VSQCP4sz3dX6EWWaKXCvUqOODF1SKTLyMTC\\\\\\\/kfnjl5OPXVxhoxulYytxH9Ik\\\\\\\/YKKO+5FY44JoV0reHgqqGkt\\\\\\\/mCVAlA2G8SZX4S0CsNXOEMVdLlUrRqxG7qW6JDDtF0aPhWzVCWSuP1c5ep0ILJxAtDZ1AS9DBggDJPFtdztfBRUBvDu\\\\\\\/XluN7KeYjpqqHCRD70o3F5gxw0UzMFZmOOQNK64ddc7neF0bZlRDVCw1a9r4iC\\\\\\\/BniPyGvoJtpYXXTcZrcCiSwxTbJYSPfeBn77x5N\\\\\\\/jPTkaA7vb0MIasxQ3wg2G6tjmmMXuBLvYZs6dO0sGb\\\\\\\/qDxOe93C6scKySVaj+F1kAzUDo1wKvUpNXxw7mkzko4wC4R2u\\\\\\\/URxSdpxopZqZfkRv9Z7r3O+inOjvXAOXO1FB5LUVRQ1h8yoCB5trcsP775JKgbKpX6twIbmhPm6Fisxvba97C6+Ma6F1KhTWzs36xA3PYz733yxyeSG+QcJkiw\\\\\\\/VMgaTezeNrLMaunxHlRXGGzEsD9qRVxM7kUEejyIQGjhiqhOApXoryYIrTaQ1oBw3U6S7wDfxTED3QrwnkJJJJFLRTjyt9X8X1XNflZm6LWcpd5uFbJ8N\\\\\\\/sCpnoe8ClCvWqi8SRegoGh5qrp\\\\\\\/qIWPA2Jq1ElRIzLFUmcZ70npy2LKmD0qayONYIQEHVbusdx6lxEkL1s9k7FOYS10wSh+3b8l7GlaVmceAd9q\\\\\\\/AVYUSBCUutCkS1tjOvRqUSRaJPvbUPHuRNcv6uUho+M+PTczmtDpx51yvb3WpQp4fsfoJFsyMm\\\\\\\/imyZBqCatiA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022r9uLUfAwUtnwhVnY45Mocw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ff62b18e10063d3fde2adb52e76fb748fa4cf2c13d711b62e6a44e9c5da1761b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460255 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2365" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-a4X2Da3OBc75qko1lM9CrIIfnhmPsVYhkUQAjkuYbSk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2365" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PiW956I9DfbURimw6T_TH_vZHGKF-CoNHp0qJzNqaCw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2355" title="flyDIVaS: A Comparative Genomics Resource for Drosophila Divergence and Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2375" title="Septate Junction Proteins Play Essential Roles in Morphogenesis Throughout Embryonic Development in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460255" id="node460255" data-pisa="ggg;6/8/2365" data-pisa-master="ggg;g3.116.029090" data-apath="/ggg/6/8/2365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Use of Targeted Marker Subsets to Account for Population Structure and Relatedness in Genome-Wide Association Studies of Maize (<em>Zea mays</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Angela H. Chen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alexander E. Lipka</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2365-2374;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029090 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460255" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Angela H. Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Illinois at Urbana-Champaign, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angela%2BH.%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20AH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngela%2BH.%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander E. Lipka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BE.%2BLipka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipka%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BE.%2BLipka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexander E. Lipka</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alipka@illinois.edu" class="" data-icon-position="" data-hide-link-title="0">alipka@illinois.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/8/2365" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460255" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460255/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2365.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460255" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460255/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2365.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460255" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460255/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2365.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460255" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460255/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/8/2365.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460255" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460255/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">A typical plant genome-wide association study (GWAS) uses a mixed linear model (MLM) that includes a trait as the response variable, a marker as an explanatory variable, and fixed and random effect covariates accounting for population structure and relatedness. Although effective in controlling for false positive signals, this model typically fails to detect signals that are correlated with population structure or are located in high linkage disequilibrium (LD) genomic regions. This result likely arises from each tested marker being used to estimate population structure and relatedness. Previous work has demonstrated that it is possible to increase the power of the MLM by estimating relatedness (<em>i.e.</em>, kinship) with markers that are not located on the chromosome where the tested marker resides. To quantify the amount of additional significant signals one can expect using this so-called K_chr model, we reanalyzed Mendelian, polygenic, and complex traits in two maize (<em>Zea mays</em> L.) diversity panels that have been previously assessed using the traditional MLM. We demonstrated that the K_chr model could find more significant associations, especially in high LD regions. This finding is underscored by our identification of novel genomic signals proximal to the tocochromanol biosynthetic pathway gene <em>ZmVTE1</em> that are associated with a ratio of tocotrienols. We conclude that the K_chr model can detect more intricate sources of allelic variation underlying agronomically important traits, and should therefore become more widely used for GWAS. To facilitate the implementation of the K_chr model, we provide code written in the R programming language.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gwas" class="hw-term hw-article-keyword hw-article-keyword-gwas" rel="nofollow">GWAS</a></li><li class="kwd"><a href="/keyword/mixed-model" class="hw-term hw-article-keyword hw-article-keyword-mixed-model" rel="nofollow">mixed model</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/maize" class="hw-term hw-article-keyword hw-article-keyword-maize" rel="nofollow">maize</a></li><li class="kwd"><a href="/keyword/marker-subsets" class="hw-term hw-article-keyword hw-article-keyword-marker-subsets" rel="nofollow">marker subsets</a></li></ul><p id="p-3">The evaluation of associations between a set of genomic markers and a trait of interest makes it possible to obtain biological insight into the relationship between genetic and phenotypic variation. This can potentially culminate in the identification of specific genes associated with the trait and a rigorous assessment of the ability of the markers to collectively predict trait values (reviewed in <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lipka <em>et al.</em> 2015</a>). Given recent advances in genotyping technologies and corresponding cost reductions, analyses that utilize genome-wide marker sets to study the genetic components underlying phenotypic variation are becoming increasingly commonplace (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhu <em>et al.</em> 2008</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Daetwyler <em>et al.</em> 2013</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Flint and Eskin 2012</a>). One such analysis is the genome-wide association study (GWAS), where markers spanning an entire genome are tested for associations with a group of traits in a panel consisting of a diverse set of individuals (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Myles <em>et al.</em> 2009</a>). Because a typical diversity panel captures a substantial amount of allelic diversity and historical recombination events, it is assumed that a marker identified in a GWAS as significantly associated with a trait is in strong linkage disequilibrium (LD) with one or more causal genomic variants (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Platt <em>et al.</em> 2010</a>).</p><p id="p-4">An important drawback of the GWAS is that false positive marker-trait associations due to population structure and familial relatedness could arise if unaccounted for (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Yu <em>et al.</em> 2006</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Lipka <em>et al.</em> 2015</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Zhang <em>et al.</em> 2010</a>). The ability to adjust for such sources of false positives in a computationally efficient manner has been an active area of research (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Kang <em>et al.</em> 2008</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">2010</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Zhou and Stephens 2012</a>); consequently, a typical GWAS in plants will employ statistical approaches that take population structure and familial relatedness into account (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Lipka <em>et al.</em> 2015</a>).</p><p id="p-5">Of all the state of the art statistical approaches that have been developed to control for false positive marker-trait associations, the unified mixed linear model (MLM; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Yu <em>et al.</em> 2006</a>) is arguably the best suited for GWAS data sets. In addition to including a tested marker as a fixed effect, this model includes fixed effect covariates that account for population structure and a random polygenic effect to control for relatedness among the individuals. To ensure that these additional terms adequately adjust for false positive signals, genome-wide marker sets are usually used to obtain the fixed effect covariates (called Q), as well as a kinship matrix (K) that estimates the variance-covariance among the individuals (<em>i.e.</em>, the variance-covariance of the random polygenic effect). In general, the unified MLM has been successful in identifying marker-trait associations with moderate to large effect sizes (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Lipka <em>et al.</em> 2015</a>), with some notable examples in maize including signals proximal to candidate genes for flowering time (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> 2013</a>) and tocochromanol and carotenoid biosynthesis (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> 2013</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Owens <em>et al.</em> 2014</a>). Nevertheless, one impediment of this model is that it has been generally unable to detect small effect loci that underlie complex trait variation (as described in <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Atwell <em>et al.</em> 2010</a>). Thus, there remains a critical need to modify the traditional unified MLM so that it has sufficient statistical power to detect these weak signals, while still adequately controlling for false positives.</p><p id="p-6">Recent studies have identified two particular situations in which the unified MLM overcorrects for false positive signals. The first situation arises when a trait under study is correlated with population structure (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Larsson <em>et al.</em> 2013</a>). For such a trait, markers in strong LD with a putatively causal locus will likely not be detected using the unified MLM because they are strongly correlated with the fixed effects Q. The second situation occurs when a genomic signal is located in a region of high LD. It has been shown that a failure to detect such signals arises because the common procedures used to calculate the kinship matrix gives more weight to genomic regions containing markers in strong LD (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rincent <em>et al.</em> 2014</a>). In both of these situations, failure to detect such signals is likely attributable to the fact that each marker being tested for associations is typically also used to estimate population structure and relatedness. This failure could also occur if a GWAS is conducted on an independent marker set that captures the same degree of population structure and relatedness as the original marker set used to calculate Q and K. To account for these deficiencies, it has therefore been suggested that only certain subsets of genomic markers be used to account for these sources of false positives (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Listgarten <em>et al.</em> 2012</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bernardo 2013</a>). In particular, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Rincent <em>et al.</em> (2014)</a> explored the statistical power that was gained by using kinship matrices that were calculated with markers that were not in LD with a given marker being tested. By directly calculating power and evaluating three maize diversity panels, it was concluded that using kinship matrices specific to each chromosome in the unified MLM could result in greater statistical power to detect associations (an approach called the âK_Chrâ model). As such, this approach has great potential to enable the unified MLM to identify a greater amount of statistically significant marker-trait associations, while simultaneously controlling for false positives.</p><p id="p-7">Given the increased availability of high density marker sets obtained from the latest sequencing technologies, the GWAS is likely to continue to play a predominant role in unraveling the genetic architecture underlying important traits in a wide variety of species (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Korte and Farlow 2013</a>). To facilitate an accurate dissection of genomic signals, it is essential to use a statistical approach that maximizes the power for detecting associations. Therefore, the objective of this study was to evaluate the ability of the K_chr model, proposed by <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Rincent <em>et al.</em> (2014)</a>, to provide further insight into the genomic signals that underlie traits in two maize diversity panels that have been previously analyzed using the traditional unified MLM. We hypothesized that the K_chr model would provide the greatest enhancement in genomic resolution for signals located in high LD regions.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sources of phenotypic and genotypic data</h3><div id="sec-3" class="subsection"><h4>Goodman diversity panel:</h4><p id="p-8">In this study, we analyzed publicly available phenotypic and genotypic data from two maize diversity panels. The first panel was the Goodman diversity panel (described in <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Flint-Garcia <em>et al.</em> 2005</a>), which consists of 281 diverse maize lines. To assess the performance of the K_chr model under different genetic architectures, we considered three classes of phenotypes that have been previously assessed via GWAS in this panel (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The first class of phenotypes included 15 carotenoid compounds, sums, ratios, and proportions that were obtained on a subset of 201 maize lines with kernel color ranging from light yellow to dark orange (originally published in <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Owens <em>et al.</em> 2014</a>). The relatively small number of genes underlying these traits makes it possible for maize breeders to substantially increase essential nutrients, including provitamin A, in maize kernels by selecting on targeted genomic regions containing carotenoid biosynthetic and related genes. As such, our analysis of carotenoids provided an essential counterpoint to the other polygenic and complex traits considered for this study.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468993/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468993/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary information for three classes of traits that were analyzed in the Goodman association panel described in <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Flint-Garcia <em>et al.</em> (2005)</a></span><div class="sb-div caption-clear"></div></div></div><p id="p-10">The second set of phenotypes included 20 tocochromanol compounds, sums, ratios, and proportions that were published in <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> (2013)</a>. Similar to carotenoids, tocochromanol compounds have a tractable genetic architecture in maize, which could allow breeders to increase vitamin E and antioxidant levels in maize grain by selecting on a small number of genes in the tocochromanol biosynthetic and precursor pathways. However, previous QTL analyses in maize grain suggest that tocochromanols are controlled by more genes relative to carotenoids (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wong <em>et al.</em> 2003</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chander <em>et al.</em> 2008</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Kandianis <em>et al.</em> 2013</a>), thus making tocochromanols an ideal set of polygenic traits to evaluate with the K_chr model.</p><p id="p-11">The final set of phenotypes we evaluated included three phenotypes related to flowering time (<em>i.e.</em>, days to pollen, ear height, and ear diameter). Because of their complex genetic architecture and importance for breeding, flowering time-related phenotypes have been extensively studied in maize association studies (<em>e.g.</em>, <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Flint-Garcia <em>et al.</em> 2005</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Salvi <em>et al.</em> 2007</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Buckler <em>et al.</em> 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Thornsberry <em>et al.</em> 2001</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Larsson <em>et al.</em> 2013</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Peiffer <em>et al.</em> 2014</a>), yet the characterization of the vast majority of the loci underlying their genetic variability has remained elusive. Therefore, it will be critical to determine the extent to which the K_chr model can identify loci associated with flowering time, especially those located in regions of high LD.</p><p id="p-12">The genome-wide SNPs used in this study have been previously described (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> 2013</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Larsson <em>et al.</em> 2013</a>; <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Owens <em>et al.</em> 2014</a>). Briefly, these markers were obtained from the MaizeSNP50 BeadChip (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cook <em>et al.</em> 2012</a>; available at <a href="http://cbsusrv04.tc.cornell.edu/users/panzea/download.aspx?filegroupid=7">http://cbsusrv04.tc.cornell.edu/users/panzea/download.aspx?filegroupid=7</a>), the genotyping-by-sequencing (GBS) protocol (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> 2011</a>; available at <a href="http://cbsusrv04.tc.cornell.edu/users/panzea/download.aspx?filegroupid=5">http://cbsusrv04.tc.cornell.edu/users/panzea/download.aspx?filegroupid=5</a>), and several other SNP genotyping assays (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Yu <em>et al.</em> 2006</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">McMullen <em>et al.</em> 2009</a>). Summary statistics for these three marker sets are included in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029090/-/DC1/TableS1.pdf">Table S1</a>. Upon removal of SNPs that either i) exhibited minor allele frequency (MAF) of less than 0.05, ii) had low quality scores, or iii) were not anchored to the B73 RefGen_v2 genome assembly (removal of such SNPs is critical for the K_chr model), between 291,939â299,253 SNPs were available for the three phenotypic classes (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). To enable direct comparisons with the results from previous studies, all markers were conservatively imputed with the major allele prior to the GWAS.</p></div><div id="sec-4" class="subsection"><h4>USDA-ARS North Central Regional Plant Introduction Station (NCRPIS) panel:</h4><p id="p-13">The second maize diversity panel we analyzed was the 2815-member NCRPIS panel (described in <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> 2013</a>). Consisting of â¼10 times as many individuals as the Goodman diversity panel, the ability of the NCRPIS panel to detect loci associated with traits including flowering time and plant height has been demonstrated (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> 2013</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Peiffer <em>et al.</em> 2014</a>). We analyzed a total of five publicly available phenotypes in this panel (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The first of these was a Mendelian trait, namely sweet corn <em>vs.</em> starchy corn. A GWAS of this trait conducted in <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> (2013)</a> identified peak associations for sweet corn <em>vs.</em> starchy corn in a chromosome 4 region containing the kernel starch biosynthesis gene <em>Su1</em>. Because the selection pressure of sweet <em>vs.</em> starchy corn resulted in high LD in this genomic region (<a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> 2013</a>), our analysis of this trait enabled a direct comparison of the ability of the K_chr model and the traditional unified MLM to detect associations under elevated LD. The remaining four phenotypes analyzed were related to flowering time and plant height (<em>i.e.</em>, days to silking, days to anthesis, plant height, and ear height). When analyzed using the traditional unified MLM in <a id="xref-ref-32-7" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> (2013)</a> and <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Peiffer <em>et al.</em> (2014)</a>, it was demonstrated that the sample size of the NCRPIS panel was sufficient to identify genomic signals associated with these complex traits. Thus, our reanalysis of these four phenotypes made it possible to assess the capability of the K_chr model to provide further elucidation into the genomic sources of complex trait variation.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468756/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468756/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary information for the traits that were analyzed in the North Central Regional Plant Introduction Station maize association panel described in <a id="xref-ref-32-8" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> (2013)</a></span><div class="sb-div caption-clear"></div></div></div><p id="p-16">The genome-wide SNP set used to analyze the NCRPIS panel has been previously described (<a id="xref-ref-32-11" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> 2013</a>), and summary statistics are included in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029090/-/DC1/TableS1.pdf">Table S1</a>. Briefly, these markers were obtained using the GBS protocol (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> 2011</a>) and are publicly available at <a href="http://cbsusrv04.tc.cornell.edu/users/panzea/download.aspx?filegroupid=6">http://cbsusrv04.tc.cornell.edu/users/panzea/download.aspx?filegroupid=6</a>. All SNPs with MAF &lt; 0.05 were removed, resulting in between 387,612â391,060 markers that were available for the GWAS. To be consistent with the procedures conducted in <a id="xref-ref-32-12" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> (2013)</a> and <a id="xref-ref-26-8" class="xref-bibr" href="#ref-26">Peiffer <em>et al.</em> (2014)</a>, all markers were imputed with the heterozygote prior to conducting the GWAS.</p></div></div><div id="sec-5" class="subsection"><h3>Evaluation of K_chr in GWAS</h3><p id="p-17">The GWAS of the carotenoid, tocochromanol, flowering time, and plant height traits was conducted using procedures that are similar to those described in <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Larsson <em>et al.</em> (2013)</a>, <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> (2013)</a>, <a id="xref-ref-32-13" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> (2013)</a>, <a id="xref-ref-26-9" class="xref-bibr" href="#ref-26">Peiffer <em>et al.</em> (2014)</a>, and <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Owens <em>et al.</em> (2014)</a>. The only major difference was that the K_chr model (described in <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Rincent <em>et al.</em> 2014</a>) was used in place of the traditional unified MLM. Briefly, the K_chr model is stated as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-2.gif"/></noscript></span></span> is a vector of observed trait values among <em>n</em> individuals, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-3.gif"/></noscript></span></span> is a vector of population substructure effects, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-4.gif"/></noscript></span></span> is a matrix of covariates relating <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-5.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-6.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-7.gif"/></noscript></span></span> is a vector of marker effects, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-8.gif"/></noscript></span></span> is an incidence matrix relating <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-9.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-10.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-11.gif"/></noscript></span></span> is a vector of polygenic effects accounting for relatedness among the individuals, <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-12.gif"/></noscript></span></span> is an incidence matrix relating <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-13.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-14.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-15.gif"/></noscript></span></span> is a vector of residual effects.<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-16.gif"/></noscript></span></span> is a random effect with variance <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-17.gif"/></noscript></span></span>, while <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-18.gif"/></noscript></span></span> is a random effect with variance <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-19.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-20.gif"/></noscript></span></span> is a kinship matrix quantifying the degree of relatedness between the individuals, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-21.gif"/></noscript></span></span> is the genetic variance, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-22.gif"/></noscript></span></span> is the identity matrix, and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-23.gif"/></noscript></span></span> is the residual variance. In the traditional unified MLM, marker sets that capture genome-wide variability are used to calculate both <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-24.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-25.gif"/></noscript></span></span> (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Yu <em>et al.</em> 2006</a>). In contrast, the K_chr model pioneered by <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Rincent <em>et al.</em> (2014)</a> calculates a separate kinship matrix for each chromosome. That is, for a given chromosome, the kinship matrix is calculated using all markers throughout the genome except for those that reside on that chromosome. In this work, we also use a similar approach to obtain separate <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8/2365/embed/mml-math-26.gif"/></noscript></span></span> matrices for each chromosome.</p><div id="sec-6" class="subsection"><h4>Statistical model specific to the Goodman diversity panel:</h4><p id="p-18">The K_chr model, as described in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a>, was used for the GWAS in both diversity panels. In the Goodman diversity panel, all 37,824 nonindustry SNPs from the Illumina MaizeSNP50 BeadChip that were anchored to a unique B73 RefGen_v2 position except those located on the chromosome under consideration were used to calculate a Loiselle kinship matrix (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Loiselle <em>et al.</em> 1995</a>) accounting for relatedness and to obtain principal components (PCs; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Price <em>et al.</em> 2006</a>) accounting for population structure. The unified MLM (<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Yu <em>et al.</em> 2006</a>) with population parameters previously determined (P3D; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Zhang <em>et al.</em> 2010</a>) was then used in the Genome Association and Prediction Integrated Tool (GAPIT) package (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lipka <em>et al.</em> 2012</a>) in the R programming language (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">R Core Team 2015</a>), to evaluate the association between each marker and trait.</p><p id="p-19">For each trait, the GWAS statistical model was optimized and marker-trait associations were evaluated using the same procedures described in <a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> (2013)</a>. Briefly, the Bayesian information criterion (BIC; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schwarz 1978</a>) was implemented to determine the most favorable number of PCs to include in the model as covariates. For each evaluated trait, the optimal number of PCs was zero, suggesting that either these evaluated traits were not strongly correlated with population structure or that the kinship matrix is accounting for population structure in addition to familial relatedness. The amount of phenotypic variation explained by the model was evaluated using a likelihood-ratio-based <em>R<sup>2</sup></em> statistic called <em>R<sup>2</sup><sub>LR</sub></em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sun <em>et al.</em> 2010</a>). Finally, the <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Benjamini and Hochberg (1995)</a> procedure was used to control the false-discovery rate (FDR) at 5 and 10%. Although the latter FDR is less conservative and could hence result in increased false positive signals, it was nevertheless also considered because of the limitations in statistical power arising from the relatively small sample size of the Goodman diversity panel (<a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> 2013</a>; <a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Owens <em>et al.</em> 2014</a>).</p></div><div id="sec-7" class="subsection"><h4>Statistical model specific to the NCRPIS panel:</h4><p id="p-20">The statistical approach used to evaluate the K_chr model in the NCRPIS panel was similar to that used for the Goodman diversity panel with a few exceptions. To be consistent with the analyses conducted in <a id="xref-ref-32-14" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> (2013)</a> and <a id="xref-ref-26-10" class="xref-bibr" href="#ref-26">Peiffer <em>et al.</em> (2014)</a>, the <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">VanRaden (2008)</a> method was used to calculate all kinship matrices using a randomly selected 10% of the appropriate subsets of GBS SNPs, and the compressed MLM (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Zhang <em>et al.</em> 2010</a>) was run in the GAPIT R package (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Lipka <em>et al.</em> 2012</a>). When the two traits from <a id="xref-ref-32-15" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> (2013)</a> (<em>i.e.</em>, sweet <em>vs.</em> starchy and days to silking) were assessed, the first five PCs (from a principal component analysis of the same markers used to calculate the kinship matrices) were included as fixed effect covariates in the Q matrix. To reflect the GWAS models used in <a id="xref-ref-26-11" class="xref-bibr" href="#ref-26">Peiffer <em>et al.</em> (2014)</a>, we substituted the five PCs in the Q matrix with the six eigenvectors exhibiting the largest eigenvalues from the corresponding kinship matrix to evaluate days to anthesis, plant height, and ear height. Finally, to account for the adequate sample size of NCRPIS panel to identify genomic loci associated with complex traits, the <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Benjamini and Hochberg (1995)</a> procedure was used to control the FDR at only 5%.</p></div></div><div id="sec-8" class="subsection"><h3>Assessment of performance of the K_chr model relative to the traditional unified MLM</h3><p id="p-21">To enable a direct comparison of the K_chr approach to the traditional unified MLM, all traits were also evaluated using the traditional unified MLM as described in <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Larsson <em>et al.</em> (2013)</a>, <a id="xref-ref-19-8" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> (2013)</a>, <a id="xref-ref-32-16" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> (2013)</a>, <a id="xref-ref-26-12" class="xref-bibr" href="#ref-26">Peiffer <em>et al.</em> (2014)</a>, and <a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Owens <em>et al.</em> (2014)</a>. The results from the traditional unified MLM were compared to the K_chr model in two specific scenarios. First, for each trait with at least one statistically significant marker-trait association found using the K_chr model, the genomic region within Â± 250 kb of each significant marker identified from the K_chr model was scanned for significantly associated markers identified (for the same trait) using the traditional unified MLM. In this evaluation, statistical significance was determined at 10% FDR for the Goodman diversity panel (to account for deficiencies in statistical power) and at 5% FDR for the NCRPIS panel. A similar criterion was used to determine if there were any significantly associated markers identified by the traditional unified MLM that were not in the vicinity of markers identified by the K_chr model. Additionally, the distribution of two sets of <em>P</em>-values (for a given trait) from markers within specific genomic regions, identified in <a id="xref-ref-19-9" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> (2013)</a>, <a id="xref-ref-32-17" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> (2013)</a>, and <a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">Owens <em>et al.</em> (2014)</a> as having peak associations, were compared; one set was from the K_chr model fitted to each marker while the other set was from the traditional unified MLM fitted to each marker. The Wilcoxon signed rank test (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wilcoxon 1945</a>) was then used to compare the resulting two distributions of <em>P</em>-values.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-22">To facilitate the implementation of the K_chr model into association studies, we provide sample R code on Github (<a href="https://github.com/angelahchen/K_Chr_manuscript">https://github.com/angelahchen/K_Chr_manuscript</a>).</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>K_chr model tends to identify more significant marker-trait associations than the traditional unified MLM</h3><p id="p-23">We explored the ability of the K_chr model proposed by <a id="xref-ref-31-6" class="xref-bibr" href="#ref-31">Rincent <em>et al.</em> (2014)</a> to detect genomic signals associated with a variety of agronomically important traits measured in two maize diversity panels. Because these traits differ in genetic architecture, we were able to systematically assess the performance of this model for Mendelian (sweet <em>vs.</em> starchy), polygenic (carotenoid and tocochromanol), and complex (flowering time and plant height) traits. Furthermore, the contrasting sample sizes of the two diversity panels enabled us to evaluate the performance of the K_chr model for both small and large data sets. For each trait, we compared the number of statistically significant associations identified using the K_chr model to those detected using the traditional unified MLM. At a genome-wide 5% FDR, the K_chr model identified more statistically significant marker-trait associations for the carotenoid and tocochromanol traits (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>), as well as for all traits analyzed in the NCRPIS panel except for days to silking (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). To account for the deficiency in the statistical power to detect genomic signals arising from the small sample size of the Goodman diversity panel (<a id="xref-ref-19-10" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> 2013</a>; <a id="xref-ref-25-9" class="xref-bibr" href="#ref-25">Owens <em>et al.</em> 2014</a>), we also compared the number of statistically significant associations detected from the two approaches at a genome-wide 10% FDR. At this FDR, more statistically significant associations for both carotenoids and tocochromanols were also detected using the K_chr model. Collectively, these results suggest that, for Mendelian, polygenic, and complex traits, the K_chr model is capable of detecting a larger number of associated loci compared to the traditional unified MLM.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469010/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469010/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Number of significant associations detected at both 5% and 10% false discovery rates between the K_Chr model and traditional unified mixed linear model in the Goodman diversity panel</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469017/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469017/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Number of significant associations detected at a 5% false discovery rate between the K_Chr model and traditional unified mixed linear model in the North Central Regional Plant Introduction Station diversity panel</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>K_chr model identifies marker-trait associations in novel genomic regions</h3><p id="p-30">One critical aspect of this analysis was to determine if the K_chr model detected any genomic signals that were not physically located in the vicinity of those identified using the traditional unified MLM. Thus, for each trait with at least one statistically significant signal detected by the K_chr model, we searched for significant associations identified using the traditional unified MLM (when fitted to the same trait) located within the neighboring Â± 250 kb region of each signal detected using the K_chr model. When this approach was used to assess the GWAS results from the Goodman diversity panel, statistical significance was determined at 10% FDR. Accordingly, the K_chr model identified 75 signals located in novel genomic regions that were associated with three carotenoid traits (Î²-xanthophylls/Î±-xanthophylls, Î±-carotene/zeinoxanthin, and zeinoxanthin/lutein) and four tocochromanol traits [Î³-tocopherol/(Î³-tocopherol + Î±-tocopherol), Î´-tocotrienol/(Î³-tocotrienol + Î±-tocotrienol), Î´-tocotrienol/Î³-tocotrienol, and Î±-tocopherol/Î³-tocopherol] (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Under a similar criterion, the traditional unified MLM identified six signals associated with three tocochromanol traits (Î±-tocopherol, Î´-tocopherol/Î±-tocopherol, and Î±-tocopherol/Î³-tocopherol) that were located in novel genomic regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029090/-/DC1/FigureS1.pptx">Figure S1</a>, <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029090/-/DC1/TableS2.pdf">Table S2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2365/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots depicting all SNPs significantly associated with carotenoid (A) and tocochromanol (B) traits at 10% FDR using the K_chr model located in novel genomic regions. Such a SNP is in a novel genomic region if there are no SNPs within &#xB1; 250 kb significantly associated with that same trait at 10% FDR when using the traditional unified mixed linear model. (A) The X-axis depicts the B73 RefGen_v2 position along the maize genome and the Y-axis shows the &#x2212;log(10) P-values for each significant SNP at 10% FDR located in a novel genomic region. The blue dots represent novel genomic signals for &#x3B2;-xanthophylls/&#x3B1;-xanthophylls, the light orange dot represents such a signal for &#x3B1;-carotene/zeinoxanthin, and the dark orange dots represent such genomic signals for zeinoxanthin/lutein. The minor allele frequencies of the SNPs depicted in the figure range from 0.09&#x2013;0.45. (B) The X- and Y-axes are as described in (A). The blue dot represents novel genomic signals for &#x3B3;-tocopherol/(&#x3B3;-tocopherol + &#x3B1;-tocopherol), the light orange dots represent such signals for &#x3B4;-tocotrienol/(&#x3B3;-tocotrienol + &#x3B1;-tocotrienol), the dark orange dots represent such signals for &#x3B4;-tocotrienol/&#x3B3;-tocotrienol, and the purple dots represent such signals for &#x3B1;-tocopherol/&#x3B3;-tocopherol. The minor allele frequencies of the SNPs depicted in the figure range from 0.08&#x2013;0.48. The approximate B73 RefGen_v2 positions of relevant biosynthetic pathway genes are depicted by dotted gray arrows. FDR, false discovery rate; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-583642629" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plots depicting all SNPs significantly associated with carotenoid (A) and tocochromanol (B) traits at 10% FDR using the K_chr model located in novel genomic regions. Such a SNP is in a novel genomic region if there are no SNPs within &#xB1; 250 kb significantly associated with that same trait at 10% FDR when using the traditional unified mixed linear model. (A) The X-axis depicts the B73 RefGen_v2 position along the maize genome and the Y-axis shows the &#x2212;log(10) &lt;em&gt;P&lt;/em&gt;-values for each significant SNP at 10% FDR located in a novel genomic region. The blue dots represent novel genomic signals for &#x3B2;-xanthophylls/&#x3B1;-xanthophylls, the light orange dot represents such a signal for &#x3B1;-carotene/zeinoxanthin, and the dark orange dots represent such genomic signals for zeinoxanthin/lutein. The minor allele frequencies of the SNPs depicted in the figure range from 0.09&#x2013;0.45. (B) The X- and Y-axes are as described in (A). The blue dot represents novel genomic signals for &#x3B3;-tocopherol/(&#x3B3;-tocopherol + &#x3B1;-tocopherol), the light orange dots represent such signals for &#x3B4;-tocotrienol/(&#x3B3;-tocotrienol + &#x3B1;-tocotrienol), the dark orange dots represent such signals for &#x3B4;-tocotrienol/&#x3B3;-tocotrienol, and the purple dots represent such signals for &#x3B1;-tocopherol/&#x3B3;-tocopherol. The minor allele frequencies of the SNPs depicted in the figure range from 0.08&#x2013;0.48. The approximate B73 RefGen_v2 positions of relevant biosynthetic pathway genes are depicted by dotted gray arrows. FDR, false discovery rate; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2365/F1.medium.gif" width="379" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/8/2365/F1.medium.gif" width="379" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2365/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2365/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468754" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-31" class="first-child">Manhattan plots depicting all SNPs significantly associated with carotenoid (A) and tocochromanol (B) traits at 10% FDR using the K_chr model located in novel genomic regions. Such a SNP is in a novel genomic region if there are no SNPs within Â± 250 kb significantly associated with that same trait at 10% FDR when using the traditional unified mixed linear model. (A) The X-axis depicts the B73 RefGen_v2 position along the maize genome and the Y-axis shows the âlog(10) <em>P</em>-values for each significant SNP at 10% FDR located in a novel genomic region. The blue dots represent novel genomic signals for Î²-xanthophylls/Î±-xanthophylls, the light orange dot represents such a signal for Î±-carotene/zeinoxanthin, and the dark orange dots represent such genomic signals for zeinoxanthin/lutein. The minor allele frequencies of the SNPs depicted in the figure range from 0.09â0.45. (B) The X- and Y-axes are as described in (A). The blue dot represents novel genomic signals for Î³-tocopherol/(Î³-tocopherol + Î±-tocopherol), the light orange dots represent such signals for Î´-tocotrienol/(Î³-tocotrienol + Î±-tocotrienol), the dark orange dots represent such signals for Î´-tocotrienol/Î³-tocotrienol, and the purple dots represent such signals for Î±-tocopherol/Î³-tocopherol. The minor allele frequencies of the SNPs depicted in the figure range from 0.08â0.48. The approximate B73 RefGen_v2 positions of relevant biosynthetic pathway genes are depicted by dotted gray arrows. FDR, false discovery rate; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468912/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468912/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">For each indicated trait analyzed in the Goodman diversity panel, the number of significant markers identified by the K_chr model at 10% false discovery rate that are located in novel genomic regions are presented</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">When this analysis was conducted in the NCRPIS panel, statistical significance was determined at 5% FDR. Across the five traits assessed in this panel, the K_chr model detected a total of 722 signals located in novel genomic regions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>, and <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). These signals were distributed throughout the entire genome, the strongest of which was located on chromosome 8 (associated with days to anthesis at <em>P</em>-value 5.26 Ã 10<sup>â7</sup>; <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). In contrast to the K_chr model, no statistically significant signals located in novel genomic regions were identified in the NCRPIS panel using the traditional unified MLM.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2365/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plot depicting all SNPs significantly associated with the traits evaluated in the North Central Regional Plant Introduction Station panel at 5% FDR using the K_chr model located in novel genomic regions. Such a SNP is in a novel genomic region if there are no SNPs within &#xB1; 250 kb significantly associated with that same trait at 5% FDR when using the traditional unified mixed linear model. The X-axis depicts the B73 RefGen_v2 position along the maize genome and the Y-axis shows the &#x2212;log(10) P-values for each significant SNP at 5% FDR located in a novel genomic region. The blue dots represent novel genomic signals for sweet vs. starchy corn, the light orange dots represent such signals for days to silking, the red dots represent such signals for days to anthesis, the black dots represent such signals for plant height, and the purple dots represent such signals for ear height. The minor allele frequencies of the SNPs depicted in the figure range from 0.05&#x2013;0.50. The approximate B73 RefGen_v2 positions of relevant candidate genes and regulatory elements are depicted by dotted gray arrows. FDR, false discovery rate; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-583642629" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plot depicting all SNPs significantly associated with the traits evaluated in the North Central Regional Plant Introduction Station panel at 5% FDR using the K_chr model located in novel genomic regions. Such a SNP is in a novel genomic region if there are no SNPs within &#xB1; 250 kb significantly associated with that same trait at 5% FDR when using the traditional unified mixed linear model. The X-axis depicts the B73 RefGen_v2 position along the maize genome and the Y-axis shows the &#x2212;log(10) &lt;em&gt;P&lt;/em&gt;-values for each significant SNP at 5% FDR located in a novel genomic region. The blue dots represent novel genomic signals for sweet &lt;em&gt;vs.&lt;/em&gt; starchy corn, the light orange dots represent such signals for days to silking, the red dots represent such signals for days to anthesis, the black dots represent such signals for plant height, and the purple dots represent such signals for ear height. The minor allele frequencies of the SNPs depicted in the figure range from 0.05&#x2013;0.50. The approximate B73 RefGen_v2 positions of relevant candidate genes and regulatory elements are depicted by dotted gray arrows. FDR, false discovery rate; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2365/F2.medium.gif" width="440" height="268"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/8/2365/F2.medium.gif" width="440" height="268"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2365/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2365/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468968" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-37" class="first-child">Manhattan plot depicting all SNPs significantly associated with the traits evaluated in the North Central Regional Plant Introduction Station panel at 5% FDR using the K_chr model located in novel genomic regions. Such a SNP is in a novel genomic region if there are no SNPs within Â± 250 kb significantly associated with that same trait at 5% FDR when using the traditional unified mixed linear model. The X-axis depicts the B73 RefGen_v2 position along the maize genome and the Y-axis shows the âlog(10) <em>P</em>-values for each significant SNP at 5% FDR located in a novel genomic region. The blue dots represent novel genomic signals for sweet <em>vs.</em> starchy corn, the light orange dots represent such signals for days to silking, the red dots represent such signals for days to anthesis, the black dots represent such signals for plant height, and the purple dots represent such signals for ear height. The minor allele frequencies of the SNPs depicted in the figure range from 0.05â0.50. The approximate B73 RefGen_v2 positions of relevant candidate genes and regulatory elements are depicted by dotted gray arrows. FDR, false discovery rate; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468752/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468752/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">For each indicated trait analyzed in the North Central Regional Plant Introduction Station panel, the number of significant markers identified by the K_chr model at 5% false discovery rate that are located in novel genomic regions are presented</span><div class="sb-div caption-clear"></div></div></div><p id="p-42">Across all of the traits in which the K_chr model identified signals in novel genomic regions, Î´-tocotrienol/Î³-tocotrienol (which was analyzed in the Goodman diversity panel) had associations in a novel chromosome 5 genomic region that completely encompassed the tocochromanol biosynthetic pathway gene <em>ZmVTE1</em>. Within this region, the peak SNP locus (S5_133501858; 133,501,858 bp; <em>P</em>-value 4.98 Ã 10<sup>â7</sup>) was located only 70 bp from the <em>ZmVTE1</em> transcriptional start site. This result is particularly interesting because no statistically significant marker-trait associations were identified for this trait when the GWAS was conducted using the traditional unified MLM (<a id="xref-ref-19-12" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> 2013</a>). Although the biological basis of these novel signals needs to be rigorously evaluated in future molecular biology and biochemical studies, these findings nevertheless indicate that the K_chr model is capable of detecting genomic sources of phenotypic variation on a finer scale relative to the traditional unified MLM.</p></div><div id="sec-13" class="subsection"><h3>K_chr model provides more insight into genomic signals in high LD regions</h3><p id="p-43">We next evaluated the ability of the K_chr model to further elucidate the sources of genomic variation underlying specific regions harboring peak GWAS associations for five carotenoid traits (zeinoxanthin, Î²-xanthophylls/Î±-xanthophylls, Î±-carotene/zeinoxanthin, zeinoxanthin/lutein, and zeaxanthin), seven tocochromanol traits [Î´ -tocotrienol/Î³-tocotrienol, Î±-tocopherol, Î´-tocotrienol, Î´-tocotrienol/(Î³-tocotrienol + Î±-tocotrienol), Î´-tocopherol/Î±-tocopherol, Î³-tocopherol/(Î³-tocopherol + Î±-tocopherol), and Î±-tocopherol/Î³-tocopherol], and all five of the traits analyzed in the NCRPIS panel. All of these genomic regions have been identified in previous association studies (<a id="xref-ref-19-13" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> 2013</a>; <a id="xref-ref-25-11" class="xref-bibr" href="#ref-25">Owens <em>et al.</em> 2014</a>; <a id="xref-ref-32-18" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> 2013</a>). Because the amount of local LD varied among these genomic regions (as reported in <a id="xref-ref-19-14" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> 2013</a>, <a id="xref-ref-25-12" class="xref-bibr" href="#ref-25">Owens <em>et al.</em> 2014</a>, and <a id="xref-ref-32-19" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> 2013</a>), our evaluation enabled a direct comparison of the performance of the K_chr model to the traditional unified MLM under various levels of local LD decay. To ensure a rigorous assessment of the impact of local LD on the performance of the K_chr model, three genomic regions in high LD (<em>i.e.</em>, the chromosome 1 region containing <em>lut1</em>, the chromosome 4 region containing <em>Su1</em>, and the chromosome 5 region containing <em>ZmVTE1</em>) and three genomic regions in lower LD (the chromosome 2 region containing <em>zep1</em>, the chromosome 5 region containing <em>ZmVTE4</em>, and the chromosome 8 region containing <em>ZCN8</em> and <em>ZmRap2.7</em>) were assessed.</p><p id="p-44">Within each of these six genomic regions, we compared the distribution of <em>P</em>-values obtained from the K_chr model to those obtained using the traditional unified MLM for all markers located inside the region (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029090/-/DC1/FigureS2.pptx">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029090/-/DC1/FigureS3.pptx">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029090/-/DC1/FigureS4.pptx">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029090/-/DC1/FigureS5.pptx">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029090/-/DC1/FigureS6.pptx">Figure S6</a>, <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>, and <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a>). For the three genomic regions in high LD, the K_chr model appeared to yield a distribution of more significant <em>P</em>-values relative to the traditional unified MLM (Wilcoxon signed rank test <em>P</em>-values ranging from &lt; 2.20 Ã 10<sup>â16</sup> to 5.29 Ã 10<sup>â2</sup>). A similar result was obtained for the chromosome 8 region harboring significant associations for the flowering time- and plant height-related traits evaluated in the powerful NCRPIS panel. In contrast, the distribution of the <em>P</em>-values from the two approaches did not differ as much for the markers that are located within the genomic regions containing <em>zep1</em> and <em>ZmVTE4</em> (Wilcoxon signed rank test <em>P</em>-value ranging from 4.29 Ã 10<sup>â2</sup> to 8.69 Ã 10<sup>â1</sup>). Thus, our findings are consistent with those presented <a id="xref-ref-31-7" class="xref-bibr" href="#ref-31">Rincent <em>et al.</em> (2014)</a>, namely that the K_chr model shows a clear advantage in power over the traditional unified MLM in regions of high LD.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2365/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of P-values obtained from the K_chr model and traditional unified mixed linear model (MLM) at six specific genomic regions, each of which contain at least one candidate gene. Each graph compares the distribution of P-values from the K_chr model (red box plot, left) to those from the traditional unified MLM (blue box plot, right). The &#x2212;log(10) P-values are presented on the Y-axis. (A) Distribution of P-values from the K_chr model and MLM when markers within the chromosome 5 region surrounding ZmVTE1 were tested for association with &#x3B4;-tocotrienol/&#x3B3;-tocotrienol. (B) Distribution of P-values from the K_chr model and MLM when markers in the chromosome 1 region surrounding lut1 were tested for association with zeinoxanthin. (C) Distribution of P-values from the K_chr model and MLM when markers in the chromosome 4 region surrounding Su1 were tested for associations with sweet vs. starchy corn. (D) Distribution of P-values from the K_chr model and MLM when markers in the chromosome 5 region surrounding ZmVTE4 were tested for associations with &#x3B1;-tocopherol. (E) Distribution of P-values from the K_chr model and MLM when markers in the chromosome 2 region surrounding zep1 were tested for associations with &#x3B2;-xanthophylls/&#x3B1;-xanthophylls. (F) Distribution of P-values from the K_chr model and MLM when markers in the chromosome 8 region surrounding ZCN8 and ZmRap2.7 were tested for associations with days to silking. For the regions with high local linkage disequilibrium (LD; i.e., those presented in A, B, and C), the distribution of P-values from the K_chr model are noticeably lower than the distribution presented by the traditional unified MLM. The same trend is observed for the two regions analyzed using data from the powerful North Central Regional Plant Introduction Station panel (presented in C and F). Finally, the distribution of P-values from the two different models are more similar in regions of lower LD (presented in D and E) analyzed using data from the smaller Goodman diversity panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-583642629" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of &lt;em&gt;P&lt;/em&gt;-values obtained from the K_chr model and traditional unified mixed linear model (MLM) at six specific genomic regions, each of which contain at least one candidate gene. Each graph compares the distribution of &lt;em&gt;P&lt;/em&gt;-values from the K_chr model (red box plot, left) to those from the traditional unified MLM (blue box plot, right). The &#x2212;log(10) &lt;em&gt;P&lt;/em&gt;-values are presented on the &lt;em&gt;Y&lt;/em&gt;-axis. (A) Distribution of &lt;em&gt;P&lt;/em&gt;-values from the K_chr model and MLM when markers within the chromosome 5 region surrounding &lt;em&gt;ZmVTE1&lt;/em&gt; were tested for association with &#x3B4;-tocotrienol/&#x3B3;-tocotrienol. (B) Distribution of &lt;em&gt;P&lt;/em&gt;-values from the K_chr model and MLM when markers in the chromosome 1 region surrounding &lt;em&gt;lut1&lt;/em&gt; were tested for association with zeinoxanthin. (C) Distribution of &lt;em&gt;P&lt;/em&gt;-values from the K_chr model and MLM when markers in the chromosome 4 region surrounding &lt;em&gt;Su1&lt;/em&gt; were tested for associations with sweet &lt;em&gt;vs.&lt;/em&gt; starchy corn. (D) Distribution of &lt;em&gt;P&lt;/em&gt;-values from the K_chr model and MLM when markers in the chromosome 5 region surrounding &lt;em&gt;ZmVTE4&lt;/em&gt; were tested for associations with &#x3B1;-tocopherol. (E) Distribution of &lt;em&gt;P&lt;/em&gt;-values from the K_chr model and MLM when markers in the chromosome 2 region surrounding &lt;em&gt;zep1&lt;/em&gt; were tested for associations with &#x3B2;-xanthophylls/&#x3B1;-xanthophylls. (F) Distribution of &lt;em&gt;P&lt;/em&gt;-values from the K_chr model and MLM when markers in the chromosome 8 region surrounding &lt;em&gt;ZCN8&lt;/em&gt; and &lt;em&gt;ZmRap2.7&lt;/em&gt; were tested for associations with days to silking. For the regions with high local linkage disequilibrium (LD; &lt;em&gt;i.e.&lt;/em&gt;, those presented in A, B, and C), the distribution of &lt;em&gt;P&lt;/em&gt;-values from the K_chr model are noticeably lower than the distribution presented by the traditional unified MLM. The same trend is observed for the two regions analyzed using data from the powerful North Central Regional Plant Introduction Station panel (presented in C and F). Finally, the distribution of &lt;em&gt;P&lt;/em&gt;-values from the two different models are more similar in regions of lower LD (presented in D and E) analyzed using data from the smaller Goodman diversity panel.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2365/F3.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/8/2365/F3.medium.gif" width="429" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2365/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2365/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468853" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-45" class="first-child">Distribution of <em>P</em>-values obtained from the K_chr model and traditional unified mixed linear model (MLM) at six specific genomic regions, each of which contain at least one candidate gene. Each graph compares the distribution of <em>P</em>-values from the K_chr model (red box plot, left) to those from the traditional unified MLM (blue box plot, right). The âlog(10) <em>P</em>-values are presented on the <em>Y</em>-axis. (A) Distribution of <em>P</em>-values from the K_chr model and MLM when markers within the chromosome 5 region surrounding <em>ZmVTE1</em> were tested for association with Î´-tocotrienol/Î³-tocotrienol. (B) Distribution of <em>P</em>-values from the K_chr model and MLM when markers in the chromosome 1 region surrounding <em>lut1</em> were tested for association with zeinoxanthin. (C) Distribution of <em>P</em>-values from the K_chr model and MLM when markers in the chromosome 4 region surrounding <em>Su1</em> were tested for associations with sweet <em>vs.</em> starchy corn. (D) Distribution of <em>P</em>-values from the K_chr model and MLM when markers in the chromosome 5 region surrounding <em>ZmVTE4</em> were tested for associations with Î±-tocopherol. (E) Distribution of <em>P</em>-values from the K_chr model and MLM when markers in the chromosome 2 region surrounding <em>zep1</em> were tested for associations with Î²-xanthophylls/Î±-xanthophylls. (F) Distribution of <em>P</em>-values from the K_chr model and MLM when markers in the chromosome 8 region surrounding <em>ZCN8</em> and <em>ZmRap2.7</em> were tested for associations with days to silking. For the regions with high local linkage disequilibrium (LD; <em>i.e.</em>, those presented in A, B, and C), the distribution of <em>P</em>-values from the K_chr model are noticeably lower than the distribution presented by the traditional unified MLM. The same trend is observed for the two regions analyzed using data from the powerful North Central Regional Plant Introduction Station panel (presented in C and F). Finally, the distribution of <em>P</em>-values from the two different models are more similar in regions of lower LD (presented in D and E) analyzed using data from the smaller Goodman diversity panel.</p><div class="sb-div caption-clear"></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468798/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468798/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">For each genomic region surrounding the indicated <em>a priori</em> candidate gene that was assessed using results from the Goodman diversity panel, the Wilcoxon signed rank test was used to compare the distribution of <em>P</em>-values obtained from the K_chr model to those from the traditional unified mixed linear model</span><div class="sb-div caption-clear"></div></div></div><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468937/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468937/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">For each genomic region surrounding the indicated <em>a priori</em> candidate gene or regulatory element that was assessed using results from the North Central Regional Plant Introduction Station panel, the Wilcoxon signed rank test is used to compare the distribution of <em>P</em>-values obtained from the K_chr model to those from the traditional unified mixed linear model</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-48">We compared the performance of the K_chr model to the traditional unified MLM by running a GWAS on a series of traits contrasting in genetic architecture in two maize diversity panels. This study was conducted because the K_chr model appears to be an effective approach for increasing the ability to detect marker-trait associations, while still controlling for population structure and relatedness in the mixed model framework originally proposed in <a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">Yu <em>et al.</em> (2006)</a>. We clearly demonstrated that the K_chr model is capable of finding more statistically significant marker-trait associations than the traditional unified MLM. In high LD genomic regions where significant signals were found using the traditional unified MLM (as reported in <a id="xref-ref-19-15" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> 2013</a>, <a id="xref-ref-32-20" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> 2013</a>, and <a id="xref-ref-25-13" class="xref-bibr" href="#ref-25">Owens <em>et al.</em> 2014</a>), the K_chr model generally yielded lower <em>P</em>-values when fitted to the surrounding markers. This result is exemplified by the novel detection of significant associations between markers in a high LD genomic region containing the tocochromanol biosynthetic pathway gene <em>ZmVTE1</em> and Î´-tocotrienol/Î³-tocotrienol. Because the traits we evaluated encompass a wide variety of fundamental characteristics of maize, our results suggest that the use of the K_chr model to identify marker-trait associations could have important nutritional and agronomical implications.</p><p id="p-49">The discrepancy in the ability of the two panels to detect significant marker-trait associations is best illustrated by comparing the results for the flowering time-related traits (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Because of its complex genetic architecture in maize (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Buckler <em>et al.</em> 2009</a>), detection of the weak genomic signals underlying flowering time is inherently difficult. This issue is potentially exacerbated in the Goodman diversity panel because of its relatively small sample size. Furthermore, the density of the marker sets we analyzed was most likely insufficient to cover the LD patterns of the entire genome, especially given the rapid LD decay in maize (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Remington <em>et al.</em> 2001</a>). All of these factors are likely to have contributed to our inability to discover statistically significant associations for the flowering time-related traits in the Goodman diversity panel. In contrast, both GWAS models identified tens of thousands of statistically significant marker-trait associations for the two flowering time-related traits evaluated in the NCRPIS panel (<em>i.e.</em>, days to silking and days to anthesis; <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). The aspect that most likely contributed to this result was the increased statistical power available in the 2815-member NCRPIS panel. Indeed, we were also able to identify significant associations for another two complex traits (plant height and ear height), as well as a Mendelian trait (sweet <em>vs.</em> starchy corn) in this panel. Thus, the most important ramification of the GWAS conducted in the NCRPIS panel was that it demonstrated the potential of the K_chr model to provide further elucidation of the genomic sources of complex and Mendelian traits.</p><p id="p-50">Using specific genomic regions identified in previous studies to have peak associations with sweet <em>vs.</em> starchy corn, carotenoid traits, and tocochromanol traits (<a id="xref-ref-19-16" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> 2013</a>; <a id="xref-ref-25-14" class="xref-bibr" href="#ref-25">Owens <em>et al.</em> 2014</a>; <a id="xref-ref-32-21" class="xref-bibr" href="#ref-32">Romay <em>et al.</em> 2013</a>), we were able to show that the K_chr model can find more significant marker-trait associations than the traditional unified MLM in high-LD genomic regions. In agreement with the findings of <a id="xref-ref-31-8" class="xref-bibr" href="#ref-31">Rincent <em>et al.</em> (2014)</a>, our results suggest that the K_chr model should be considered to be a fundamental approach that could provide the resolution needed to go after elusive sources of genomic variation located in pericentromeric and other regions of recombination suppression. Additionally, our analysis of the flowering time- and plant height-related results in the chromosome 8 region containing <em>ZCN8</em> and <em>ZmRap2.7</em> suggest that the K_chr model can further refine genomic signals in lower LD regions if the sample size is sufficiently large. Thus, we recommend using the K_chr model as a starting point for any association analysis: after a genomic signal in a high-LD region is identified with the K_chr model, a chromosome-wide stepwise multi-locus mixed model (MLMM; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Segura <em>et al.</em> 2012</a>) should be conducted on the chromosome where the signal was identified using the same kinship matrix that was implemented in the K_chr model. By using this kinship matrix (instead of the genome-wide kinship matrix typically employed in the MLMM), it is likely that the MLMM will have greater power to conduct a more exhaustive search for multiple loci underlying the signal originally detected by the K_chr model.</p><p id="p-51">Interestingly, a sizeable proportion of the signals detected by the K_chr model were not in the vicinity of those found by the traditional unified MLM (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>, and <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>). Although the criterion we used to determine which signals identified using the K_chr model were located in novel genomic regions was based on those from previous maize studies (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Salvi <em>et al.</em> 2007</a>; <a id="xref-ref-19-17" class="xref-bibr" href="#ref-19">Lipka <em>et al.</em> 2013</a>; <a id="xref-ref-25-15" class="xref-bibr" href="#ref-25">Owens <em>et al.</em> 2014</a>), other equally biologically valid criteria could be used to determine which signals are in novel genomic regions. Furthermore, these other criteria could result in different conclusions about the proportion of K_chr signals that reside in novel genomic regions. Coupled with the fact that long-range LD between markers and ungenotyped causal mutations could lead to false conclusions about the location of true genomic signals (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Platt <em>et al.</em> 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dickson <em>et al.</em> 2010</a>), our criterion should only be interpreted as a rough approximation of the amount of novel genomic regions detected by the K_chr model. As such, we recommend interpreting our results on the proportion of significant K_chr associations in novel genomic regions within the context of the total number of additional significant signals identified using the K_chr approach. Nevertheless, we were able to use the K_chr model to detect novel genomic signals on chromosome 5 that were significantly associated with Î´-tocotrienol/Î³-tocotrienol. Because these signals encompass the tocochromanol biosynthetic pathway gene <em>ZmVTE1</em>, the identification of significant marker-trait associations in this region makes sense from a biochemical perspective. If these novel associations detected using the K_chr approach can be biologically validated, then our findings could lead to a more complete understanding of tocotrienol biosynthesis in maize grain.</p><p id="p-52">The more widespread implementation of the K_chr model in association studies could have a profound impact on the ability to detect biologically significant marker-trait associations, especially those that reside in high LD regions. Compared to the traditional unified MLM, the K_chr model is likely to find more intricate sources of variation underlying a genomic signal, which could pave the way toward identifying a greater number of causal mutations and haplotypes responsible for the peak marker-trait associations. Because of this, the use of the K_chr model could lead to the discovery of novel sources of allelic variation that could both enhance our understanding of the genetic architecture of important traits and lead to the elucidation of novel targets for marker-assisted selection. </p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">We sincerely thank the anonymous reviewers, as well as Maria C. Romay, Michael A. Gore, and Jason A. Peiffer, whose suggestions have significantly strengthened this manuscript. We would also like to thank Esperanza M. Shenstone and Brian R. Rice for their helpful comments on the manuscript text, tables, and figures. This research was supported by the United States Department of Agriculture National Institute of Food and Agriculture project accession number 1005564 and University of Illinois startup funds. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-20"><p id="p-54">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029090/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029090/-/DC1</a></p></li><li class="fn" id="fn-21"><p id="p-55"><em>Communicating editor: M. Warburton</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-03-11"><span class="received-label">Received </span>March 11, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-05-25"><span class="accepted-label">Accepted </span>May 25, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2016 Chen and Lipka</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.1" data-doi="10.1038/nature08800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjalmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willems</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genome-wide association study of 107 phenotypes in <em>Arabidopsis thaliana</em> inbred lines.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">465</span>(<span class="cit-issue">7298</span>): <span class="cit-fpage">627</span>â<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAtwell%26rft.auinit1%253DS.%26rft.volume%253D465%26rft.issue%253D7298%26rft.spage%253D627%26rft.epage%253D631%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B107%2Bphenotypes%2Bin%2BArabidopsis%2Bthaliana%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08800%26rft_id%253Dinfo%253Apmid%252F20336072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20336072&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278249000044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. Roy. Stat. Soc. B Met.</abbr> <span class="cit-vol">57</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">289</span>â<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BRoy.%2BStat.%2BSoc.%2BB%2BMet.%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-web" id="cit-6.8.2365.3"><div class="cit-metadata"><cite>Bernardo, R., 2013âGenomewide markers for controlling background variation in association mapping. Plant Genome <span class="cit-vol">6</span>(1). Available at: <a href="http://www.dl.sciencesocieties.org/publications/tpg/abstracts/6/1/plantgenome2012.11.0028">www.dl.sciencesocieties.org/publications/tpg/abstracts/6/1/plantgenome2012.11.0028</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.4" data-doi="10.1126/science.1174276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">The genetic architecture of maize flowering time.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>(<span class="cit-issue">5941</span>): <span class="cit-fpage">714</span>â<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D714%26rft.epage%253D718%26rft.atitle%253DThe%2BGenetic%2BArchitecture%2Bof%2BMaize%2BFlowering%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174276%26rft_id%253Dinfo%253Apmid%252F19661422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-6.8.2365.5"><div class="cit-metadata unstructured">Buckler Lab at Cornell University 2016âTassel 5. <a href="http://www.maizegenetics.net">www.maizegenetics.net</a>. Accessed: June 10, 2016.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.6" data-doi="10.1007/s11032-008-9180-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chander</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Genetic dissection of tocopherol content and composition in maize grain using quantitative trait loci analysis and the candidate gene approach.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">353</span>â<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D22%26rft.spage%253D353%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-008-9180-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-008-9180-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.7" data-doi="10.1104/pp.111.185033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genetic architecture of maize kernel composition in the nested association mapping and inbred association panels.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">158</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">824</span>â<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.185033%26rft_id%253Dinfo%253Apmid%252F22135431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTgvMi84MjQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.8" data-doi="10.1534/genetics.112.147983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">347</span>â<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147983%26rft_id%253Dinfo%253Apmid%252F23222650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.9" data-doi="10.1371/journal.pbio.1000294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickson</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krantz</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakonarson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Rare variants create synthetic genome-wide associations.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e1000294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDickson%26rft.auinit1%253DS.%2BP.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253De1000294%26rft.epage%253De1000294%26rft.atitle%253DRare%2Bvariants%2Bcreate%2Bsynthetic%2Bgenome-wide%2Bassociations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000294%26rft_id%253Dinfo%253Apmid%252F20126254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20126254&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.10" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.11" data-doi="10.1038/nrg3335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genome-wide association studies in mice.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">807</span>â<span class="cit-lpage">817</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFlint%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D807%26rft.epage%253D817%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3335%26rft_id%253Dinfo%253Apmid%252F23044826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23044826&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.12" data-doi="10.1111/j.1365-313X.2005.02591.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thuillet</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Maize association population: a high-resolution platform for quantitative trait locus dissection.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1054</span>â<span class="cit-lpage">1064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DFlint-Garcia%26rft.auinit1%253DS.%2BA.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D1054%26rft.epage%253D1064%26rft.atitle%253DMaize%2Bassociation%2Bpopulation%253A%2Ba%2Bhigh-resolution%2Bplatform%2Bfor%2Bquantitative%2Btrait%2Blocus%2Bdissection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2005.02591.x%26rft_id%253Dinfo%253Apmid%252F16359397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2005.02591.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16359397&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233776900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.13" data-doi="10.1007/s00122-013-2179-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kandianis</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palacios</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montgomery</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Genetic architecture controlling variation in grain carotenoid composition and concentrations in two maize populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">2879</span>â<span class="cit-lpage">2895</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2879%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2179-5%26rft_id%253Dinfo%253Apmid%252F24042570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2179-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24042570&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.14" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1709</span>â<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.15" data-doi="10.1038/ng.548"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sul</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Service</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Variance component model to account for sample structure in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">348</span>â<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D354%26rft.atitle%253DVariance%2Bcomponent%2Bmodel%2Bto%2Baccount%2Bfor%2Bsample%2Bstructure%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.548%26rft_id%253Dinfo%253Apmid%252F20208533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208533&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-web" id="cit-6.8.2365.16"><div class="cit-metadata"><cite>Korte, A., and A. Farlow, 2013âThe advantages and limitations of trait analysis with GWAS: a review. Plant Methods <span class="cit-vol">9</span>. Available at: <a href="http://www.plantmethods.biomedcentral.com/articles/10.1186/1746-4811-9-29">www.plantmethods.biomedcentral.com/articles/10.1186/1746-4811-9-29</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Lessons from Dwarf8 on the strengths and weaknesses of structured association mapping.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">2</span>): e1003246.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.18" data-doi="10.1093/bioinformatics/bts444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">GAPIT: genome association and prediction integrated tool.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">18</span>): <span class="cit-fpage">2397</span>â<span class="cit-lpage">2399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts444%26rft_id%253Dinfo%253Apmid%252F22796960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTgvMjM5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magallanes-Lundback</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Genome-wide association study and pathway-level analysis of tocochromanol levels in maize grain. G3</span> (<abbr class="cit-jnl-abbrev">Bethesda)</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1287</span>â<span class="cit-lpage">1299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBethesda%2529%26rft.volume%253D3%26rft.spage%253D1287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.20" data-doi="10.1016/j.pbi.2015.02.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kandianis</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drnevich</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">From association to prediction: statistical methods for the dissection and selection of complex traits in plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">110</span>â<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D24%26rft.spage%253D110%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2015.02.010%26rft_id%253Dinfo%253Apmid%252F25795170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2015.02.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25795170&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.21" data-doi="10.1038/nmeth.2037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Listgarten</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lippert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadie</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Improved linear mixed models for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">525</span>â<span class="cit-lpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DListgarten%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D6%26rft.spage%253D525%26rft.epage%253D526%26rft.atitle%253DImproved%2Blinear%2Bmixed%2Bmodels%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2037%26rft_id%253Dinfo%253Apmid%252F22669648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22669648&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304778500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.22" data-doi="10.2307/2445869"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loiselle</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">V. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nason</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Spatial genetic-structure of a tropical understory shrub, Psychotria officinalis (rubiaceae).</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">82</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1420</span>â<span class="cit-lpage">1425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft.volume%253D82%26rft.spage%253D1420%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2445869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2445869&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995TF10800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.23" data-doi="10.1126/science.1174320"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeda</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genetic properties of the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>(<span class="cit-issue">5941</span>): <span class="cit-fpage">737</span>â<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcMullen%26rft.auinit1%253DM.%2BD.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D737%26rft.epage%253D740%26rft.atitle%253DGenetic%2BProperties%2Bof%2Bthe%2BMaize%2BNested%2BAssociation%2BMapping%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174320%26rft_id%253Dinfo%253Apmid%252F19661427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.24" data-doi="10.1105/tpc.109.068437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myles</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Association mapping: critical considerations shift from genotyping to experimental design.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">2194</span>â<span class="cit-lpage">2202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.068437%26rft_id%253Dinfo%253Apmid%252F19654263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMjE5NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMzY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.25" data-doi="10.1534/genetics.114.169979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Owens</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magallanes-Lundback</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiede</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diepenbrock</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">A foundation for provitamin A biofortification of maize: genome-wide association and genomic prediction models of carotenoid levels.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1699</span>â<span class="cit-lpage">1716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169979%26rft_id%253Dinfo%253Apmid%252F25258377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE2OTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.26" data-doi="10.1534/genetics.113.159152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">The genetic architecture of maize height.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1337</span>â<span class="cit-lpage">1356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159152%26rft_id%253Dinfo%253Apmid%252F24514905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ni80LzEzMzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.27" data-doi="10.1534/genetics.110.121665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjalmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Conditions under which genome-wide association studies will be positively misleading.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1045</span>â<span class="cit-lpage">1052</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.121665%26rft_id%253Dinfo%253Apmid%252F20813880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni8zLzEwNDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.28" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plenge</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinblatt</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shadick</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">904</span>â<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-book" id="cit-6.8.2365.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R-Core Team</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em></span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.30" data-doi="10.1073/pnas.201394398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remington</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornsberry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuoka</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitt</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Structure of linkage disequilibrium and phenotypic associations in the maize genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>(<span class="cit-issue">20</span>): <span class="cit-fpage">11479</span>â<span class="cit-lpage">11484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.201394398%26rft_id%253Dinfo%253Apmid%252F11562485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTgvMjAvMTE0NzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjM2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.31" data-doi="10.1534/genetics.113.159731"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monod</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Recovering power in association mapping panels with variable levels of linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">375</span>â<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159731%26rft_id%253Dinfo%253Apmid%252F24532779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millard</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Comprehensive genotyping of the USA national maize inbred seed bank.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">6</span>): R55.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.33" data-doi="10.1073/pnas.0704145104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salvi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sponza</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgante</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomes</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Conserved noncoding genomic sequences associated with a flowering-time quantitative trait locus in maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>(<span class="cit-issue">27</span>): <span class="cit-fpage">11376</span>â<span class="cit-lpage">11381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0704145104%26rft_id%253Dinfo%253Apmid%252F17595297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzI3LzExMzc2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIzNjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.34" data-doi="10.1214/aos/1176344136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span>â<span class="cit-article-title">Estimating the dimension of a model.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">6</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">461</span>â<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D6%26rft.spage%253D461%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176344136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176344136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978EQ63300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.35" data-doi="10.1038/ng.2314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjalmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seren</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">An efficient multi-locus mixed-model approach for genome-wide association studies in structured populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">825</span>â<span class="cit-lpage">830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSegura%26rft.auinit1%253DV.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D825%26rft.epage%253D830%26rft.atitle%253DAn%2Befficient%2Bmulti-locus%2Bmixed-model%2Bapproach%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2314%26rft_id%253Dinfo%253Apmid%252F22706313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22706313&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.36" data-doi="10.1038/hdy.2010.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Variation explained in mixed-model association mapping.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">105</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">333</span>â<span class="cit-lpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D105%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.11%26rft_id%253Dinfo%253Apmid%252F20145669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20145669&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.37" data-doi="10.1038/90135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornsberry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodman</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Dwarf8 polymorphisms associate with variation in flowering time.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">286</span>â<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DThornsberry%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D286%26rft.epage%253D289%26rft.atitle%253DDwarf8%2Bpolymorphisms%2Bassociate%2Bwith%2Bvariation%2Bin%2Bflowering%2Btime.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F90135%26rft_id%253Dinfo%253Apmid%252F11431702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/90135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11431702&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169656400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.38" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">4414</span>â<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.39" data-doi="10.2307/3001968"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilcoxon</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1945</span>â<span class="cit-article-title">Individual comparisons by ranking methods.</span> <abbr class="cit-jnl-abbrev">Biom. Bull.</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">80</span>â<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiom.%2BBull.%26rft.volume%253D1%26rft.spage%253D80%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3001968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3001968&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.40" data-doi="10.2135/cropsci2003.2257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambert</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tadmor</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rocheford</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">QTL associated with accumulation of tocopherols in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">2257</span>â<span class="cit-lpage">2266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DWong%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D2257%26rft.epage%253D2266%26rft.atitle%253DQTL%2BAssociated%2Bwith%2BAccumulation%2Bof%2BTocopherols%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2003.2257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2003.2257&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000186477700049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.41" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">203</span>â<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.42" data-doi="10.1038/ng.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todhunter</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Mixed linear model approach adapted for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">355</span>â<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D360%26rft.atitle%253DMixed%2Blinear%2Bmodel%2Bapproach%2Badapted%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.546%26rft_id%253Dinfo%253Apmid%252F20208535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208535&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.43" data-doi="10.1038/ng.2310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Genome-wide efficient mixed-model analysis for association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">821</span>â<span class="cit-lpage">824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhou%26rft.auinit1%253DX.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D821%26rft.epage%253D824%26rft.atitle%253DGenome-wide%2Befficient%2Bmixed-model%2Banalysis%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2310%26rft_id%253Dinfo%253Apmid%252F22706312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22706312&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2365.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.8.2365.44" data-doi="10.3835/plantgenome2008.02.0089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Status and prospects of association mapping in plants.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">5</span>â<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D1%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2008.02.0089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2008.02.0089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000208575600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/8/2365.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2355" title="flyDIVaS: A Comparative Genomics Resource for Drosophila Divergence and Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2375" title="Septate Junction Proteins Play Essential Roles in Morphogenesis Throughout Embryonic Development in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/8"><span>Volume 6 Issue 8, August 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/8.cover-source.jpg?itok=40Kz9Oeq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2365" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YxEUlDSiNGVFDcaBc7NXBCwTJLWHXX0AYhGKtC34oYY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/8/2365" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/8/2365" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/8/2365" class="active" data-icon-position="" data-hide-link-title="0">The Use of Targeted Marker Subsets to Account for Population Structure and Relatedness in Genome-Wide Association Studies of Maize (Zea mays L.)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460255" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-5gVhHfFA0y0Myhoaa53h35i1wlCG9i-VZfuy7CiP3tw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2365.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/8/2365" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-NmDVmNOrlkJDqWjG8N8zSSZcpMBoS8IcrIK0b8rOOQg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/8/2365" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2365.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460255" id="node460255--2" data-pisa="ggg;6/8/2365" data-pisa-master="ggg;g3.116.029090" data-apath="/ggg/6/8/2365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Use of Targeted Marker Subsets to Account for Population Structure and Relatedness in Genome-Wide Association Studies of Maize (<em>Zea mays</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Angela H. Chen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alexander E. Lipka</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2365-2374;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029090 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460255--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Angela H. Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Illinois at Urbana-Champaign, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angela%2BH.%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20AH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngela%2BH.%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander E. Lipka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BE.%2BLipka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipka%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BE.%2BLipka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexander E. Lipka</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alipka@illinois.edu" class="" data-icon-position="" data-hide-link-title="0">alipka@illinois.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/8/2365" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/8/2365&amp;title=The%20Use%20of%20Targeted%20Marker%20Subsets%20to%20Account%20for%20Population%20Structure%20and%20Relatedness%20in%20Genome-Wide%20Association%20Studies%20of%20Maize%20%28Zea%20mays%20L.%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/8/2365&amp;title=The%20Use%20of%20Targeted%20Marker%20Subsets%20to%20Account%20for%20Population%20Structure%20and%20Relatedness%20in%20Genome-Wide%20Association%20Studies%20of%20Maize%20%28Zea%20mays%20L.%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/8/2365&amp;title=The%20Use%20of%20Targeted%20Marker%20Subsets%20to%20Account%20for%20Population%20Structure%20and%20Relatedness%20in%20Genome-Wide%20Association%20Studies%20of%20Maize%20%28Zea%20mays%20L.%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/8/2365&amp;text=The%20Use%20of%20Targeted%20Marker%20Subsets%20to%20Account%20for%20Population%20Structure%20and%20Relatedness%20in%20Genome-Wide%20Association%20Studies%20of%20Maize%20%28Zea%20mays%20L.%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/8/2365&amp;title=The%20Use%20of%20Targeted%20Marker%20Subsets%20to%20Account%20for%20Population%20Structure%20and%20Relatedness%20in%20Genome-Wide%20Association%20Studies%20of%20Maize%20%28Zea%20mays%20L.%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/8/2365&amp;t=The%20Use%20of%20Targeted%20Marker%20Subsets%20to%20Account%20for%20Population%20Structure%20and%20Relatedness%20in%20Genome-Wide%20Association%20Studies%20of%20Maize%20%28Zea%20mays%20L.%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/8/2365&amp;title=The%20Use%20of%20Targeted%20Marker%20Subsets%20to%20Account%20for%20Population%20Structure%20and%20Relatedness%20in%20Genome-Wide%20Association%20Studies%20of%20Maize%20%28Zea%20mays%20L.%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/8/2365&amp;title=The%20Use%20of%20Targeted%20Marker%20Subsets%20to%20Account%20for%20Population%20Structure%20and%20Relatedness%20in%20Genome-Wide%20Association%20Studies%20of%20Maize%20%28Zea%20mays%20L.%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460255/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460255" id="node460255--3" data-pisa="ggg;6/8/2365" data-pisa-master="ggg;g3.116.029090" data-apath="/ggg/6/8/2365.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Use of Targeted Marker Subsets to Account for Population Structure and Relatedness in Genome-Wide Association Studies of Maize (<em>Zea mays</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Angela H. Chen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alexander E. Lipka</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2365-2374;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029090 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460255--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Angela H. Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Illinois at Urbana-Champaign, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Angela%2BH.%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20AH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAngela%2BH.%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander E. Lipka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Illinois 61801</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BE.%2BLipka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lipka%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BE.%2BLipka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1571-8528" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexander E. Lipka</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alipka@illinois.edu" class="" data-icon-position="" data-hide-link-title="0">alipka@illinois.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2365&amp;rft.epage=2374&amp;rft.atitle=The%20Use%20of%20Targeted%20Marker%20Subsets%20to%20Account%20for%20Population%20Structure%20and%20Relatedness%20in%20Genome-Wide%20Association%20Studies%20of%20Maize%20%28Zea%20mays%20L.%29&amp;rft.volume=6&amp;rft.issue=8&amp;rft.date=2016-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chen%2C+Angela+H.&amp;rft.au=Lipka%2C+Alexander+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460255/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460255/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460255/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460255/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460255/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460255/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460255/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460255/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460255/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460255/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460255/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460255/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/8/2365" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/8/2365.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/8/2365.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/8/2365.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/8/2365.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
