<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/8/2299" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/8/2299.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/8/2299.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/8/2299.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/8/2299.atom" />
<meta name="HW.pisa" content="ggg;6/8/2299" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Long Terminal Repeat Retrotransposon Content in Eight Diploid Sunflower Species Inferred from Next-Generation Sequence Data" />
<meta name="DC.Identifier" content="10.1534/g3.116.029082" />
<meta name="DC.Date" content="2016-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Tetreault and Ungerer. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The most abundant transposable elements (TEs) in plant genomes are Class I long terminal repeat (LTR) retrotransposons represented by superfamilies gypsy and copia . Amplification of these superfamilies directly impacts genome structure and contributes to differential patterns of genome size evolution among plant lineages. Utilizing short-read Illumina data and sequence information from a panel of Helianthus annuus (sunflower) full-length gypsy and copia elements, we explore the contribution of these sequences to genome size variation among eight diploid Helianthus species and an outgroup taxon, Phoebanthus tenuifolius . We also explore transcriptional dynamics of these elements in both leaf and bud tissue via RT-PCR. We demonstrate that most LTR retrotransposon sublineages ( i.e. , families) display patterns of similar genomic abundance across species. A small number of LTR retrotransposon sublineages exhibit lineage-specific amplification, particularly in the genomes of species with larger estimated nuclear DNA content. RT-PCR assays reveal that some LTR retrotransposon sublineages are transcriptionally active across all species and tissue types, whereas others display species-specific and tissue-specific expression. The species with the largest estimated genome size, H. agrestis , has experienced amplification of LTR retrotransposon sublineages, some of which have proliferated independently in other lineages in the Helianthus phylogeny." />
<meta name="DC.Contributor" content="Hannah M. Tetreault" />
<meta name="DC.Contributor" content="Mark C. Ungerer" />
<meta name="article:published_time" content="2016-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Long Terminal Repeat Retrotransposon Content in Eight Diploid Sunflower Species Inferred from Next-Generation Sequence Data" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The most abundant transposable elements (TEs) in plant genomes are Class I long terminal repeat (LTR) retrotransposons represented by superfamilies &lt;i&gt;gypsy&lt;/i&gt; and &lt;i&gt;copia&lt;/i&gt;. Amplification of these superfamilies directly impacts genome structure and contributes to differential patterns of genome size evolution among plant lineages. Utilizing short-read Illumina data and sequence information from a panel of &lt;i&gt;Helianthus annuus&lt;/i&gt; (sunflower) full-length &lt;i&gt;gypsy&lt;/i&gt; and &lt;i&gt;copia&lt;/i&gt; elements, we explore the contribution of these sequences to genome size variation among eight diploid &lt;i&gt;Helianthus&lt;/i&gt; species and an outgroup taxon, &lt;i&gt;Phoebanthus tenuifolius&lt;/i&gt;. We also explore transcriptional dynamics of these elements in both leaf and bud tissue via RT-PCR. We demonstrate that most LTR retrotransposon sublineages (&lt;i&gt;i.e.&lt;/i&gt;, families) display patterns of similar genomic abundance across species. A small number of LTR retrotransposon sublineages exhibit lineage-specific amplification, particularly in the genomes of species with larger estimated nuclear DNA content. RT-PCR assays reveal that some LTR retrotransposon sublineages are transcriptionally active across all species and tissue types, whereas others display species-specific and tissue-specific expression. The species with the largest estimated genome size, &lt;i&gt;H. agrestis&lt;/i&gt;, has experienced amplification of LTR retrotransposon sublineages, some of which have proliferated independently in other lineages in the &lt;i&gt;Helianthus&lt;/i&gt; phylogeny.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/08/01" />
<meta name="citation_mjid" content="ggg;6/8/2299" />
<meta name="citation_id" content="6/8/2299" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/8/2299" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/8/2299.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/8/2299.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/8/2299.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.029082" />
<meta name="citation_pmid" content="27233667" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2299" />
<meta name="citation_lastpage" content="2308" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hannah M. Tetreault" />
<meta name="citation_author_institution" content="Division of Biology, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_author_email" content="htetre@ksu.edu" />
<meta name="citation_author" content="Mark C. Ungerer" />
<meta name="citation_author_institution" content="Division of Biology, Kansas State University, Manhattan, Kansas 66506" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=J. A. Agren;citation_author=S. Greiner;citation_author=M. T. J. Johnson;citation_author=S. I. Wright;citation_title=No evidence that sex and transposable elements drive genome size variation in evening primroses.;citation_pages=1053-1062;citation_volume=69;citation_year=2015;citation_issue=4;citation_pmid=25690700;citation_doi=10.1111/evo.12627" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina data.;citation_volume=30;citation_year=2014;citation_issue=15;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=G. Bonchev;citation_author=C. Parisod;citation_title=Transposable elements and microevolutionary changes in natural populations.;citation_pages=765-775;citation_volume=13;citation_year=2013;citation_issue=5;citation_pmid=23795753;citation_doi=10.1111/1755-0998.12133" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Buti;citation_author=T. Giordani;citation_author=F. Cattonaro;citation_author=R. M. Cossu;citation_author=L. Pistelli;citation_title=Temporal dynamics in the evolution of the sunflower genome as revealed by sequencing and annotation of three large genomic regions.;citation_pages=779-791;citation_volume=123;citation_year=2011;citation_issue=5;citation_pmid=21647740;citation_doi=10.1007/s00122-011-1626-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Charles;citation_author=H. Belcram;citation_author=J. Just;citation_author=C. Huneau;citation_author=A. Viollet;citation_title=Dynamics and differential proliferation of transposable elements during the evolution of the B and A genomes of wheat.;citation_volume=180;citation_year=2008;citation_issue=2;citation_pmid=18780739;citation_doi=10.1534/genetics.108.092304" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=K. M. Devos;citation_author=J. K. Brown;citation_author=J. L. Bennetzen;citation_title=Genome Size Reduction through Illegitimate Recombination Counteracts Genome Expansion in Arabidopsis;citation_pages=1075-1079;citation_volume=12;citation_year=2002;citation_issue=7;citation_pmid=12097344;citation_doi=10.1101/gr.132102" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. M. Diez;citation_author=E. Meca;citation_author=M. I. Tenaillon;citation_author=B. S. Gaut;citation_title=Three groups of transposable elements with contrasting copy number dynamics and host responses in the maize (Zea mays ssp mays) genome.;citation_pages=e1004298;citation_volume=10;citation_year=2014;citation_issue=4;citation_pmid=24743518;citation_doi=10.1371/journal.pgen.1004298" />
<meta name="citation_reference" content="citation_journal_title=Cytometry. Part A :  the journal of the International Society for Analytical Cytology;citation_journal_abbrev=Cytometry A;citation_author=J. Dolezel;citation_author=J. Bartos;citation_author=H. Voglmayr;citation_author=J. Greilhuber;citation_title=Nuclear DNA content and genome size of trout and human.;citation_pages=127-128;citation_volume=51;citation_year=2003;citation_issue=2;citation_pmid=12541287;citation_doi=10.1002/cyto.a.10013" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=M. El Baidouri;citation_author=O. Panaud;citation_title=Comparative genomic paleontology across plant kingdom reveals the dynamics of TE-driven genome evolution.;citation_volume=5;citation_year=2013;citation_issue=5;citation_pmid=23426643;citation_doi=10.1093/gbe/evt025" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Ellinghaus;citation_author=S. Kurtz;citation_author=U. Willhoeft;citation_title=LTRharvest, an efficient and flexible software for de novo detection of LTR retrotransposons.;citation_pages=18-18;citation_volume=9;citation_year=2008;citation_pmid=18194517;citation_doi=10.1186/1471-2105-9-18" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C. Feschotte;citation_author=N. Jiang;citation_author=S. R. Wessler;citation_title=Plant transposable elements: where genetics meets genomics.;citation_pages=329-341;citation_volume=3;citation_year=2002;citation_issue=5;citation_pmid=11988759;citation_doi=10.1038/nrg793" />
<meta name="citation_reference" content="citation_journal_title=Biology (Basel);citation_author=N. Gill;citation_author=M. Buti;citation_author=N. Kane;citation_author=A. Bellec;citation_author=N. Helmstetter;citation_title=Sequence-based analysis of structural organization and composition of the cultivated sunflower (Helianthus annuus L.) genome.;citation_pages=295-319;citation_volume=3;citation_year=2014;citation_issue=2;citation_pmid=24833511" />
<meta name="citation_reference" content="citation_journal_title=Current Plant Biology;citation_author=T. Giordani;citation_author=A. Cavallini;citation_author=L. Natali;citation_title=The repetitive component of the sunflower genome.;citation_pages=45-54;citation_volume=1;citation_year=2014;citation_doi=10.1016/j.cpb.2014.05.001" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. S. Hawkins;citation_author=H. Kim;citation_author=J. D. Nason;citation_author=R. A. Wing;citation_author=J. F. Wendel;citation_title=Differential lineage-specific amplification of transposable elements is responsible for genome size variation in Gossypium.;citation_volume=16;citation_year=2006;citation_issue=10;citation_pmid=16954538;citation_doi=10.1101/gr.5282906" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. S. Hawkins;citation_author=S. R. Proulx;citation_author=R. A. Rapp;citation_author=J. F. Wendel;citation_title=Rapid DNA loss as a counterbalance to genome expansion through retrotransposon proliferation in plants.;citation_volume=106;citation_year=2009;citation_issue=42;citation_pmid=19815511;citation_doi=10.1073/pnas.0904339106" />
<meta name="citation_reference" content="citation_journal_title=Memoirs of the Torrey Botanical Club;citation_author=C. B. Heiser;citation_author=D. M. Smith;citation_author=S. B. Clevenger;citation_author=W. C. Martin;citation_title=The North American sunflowers (Helianthus).;citation_pages=1-218;citation_volume=22;citation_year=1969" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=The map-based sequence of the rice genome.;citation_pages=793-800;citation_volume=436;citation_year=2005;citation_issue=7052;citation_pmid=16100779;citation_doi=10.1038/nature03895" />
<meta name="citation_reference" content="citation_journal_title=Botany-Botanique;citation_author=N. C. Kane;citation_author=N. Gill;citation_author=M. G. King;citation_author=J. E. Bowers;citation_author=H. Berges;citation_title=Progress towards a reference genome for sunflower.;citation_pages=429-437;citation_volume=89;citation_year=2011;citation_issue=7;citation_doi=10.1139/b11-032" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=N. C. Kane;citation_author=J. M. Burke;citation_author=L. Marek;citation_author=G. Seiler;citation_author=F. Vear;citation_title=Sunflower genetic, genomic and ecological resources.;citation_pages=10-20;citation_volume=13;citation_year=2013;citation_issue=1;citation_pmid=23039950;citation_doi=10.1111/1755-0998.12023" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=T. Kawakami;citation_author=S. C. Strakosh;citation_author=Y. Zhen;citation_author=M. C. Ungerer;citation_title=Different scales of Ty1/copia-like retrotransposon proliferation in the genomes of three diploid hybrid sunflower species.;citation_pages=341-350;citation_volume=104;citation_year=2010;citation_issue=4;citation_pmid=20068588;citation_doi=10.1038/hdy.2009.182" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=T. Kawakami;citation_author=P. Dhakal;citation_author=A. N. Katterhenry;citation_author=C. A. Heatherington;citation_author=M. C. Ungerer;citation_title=Transposable element proliferation and genome expansion are rare in contemporary sunflower hybrid populations despite widespread transcriptional activity of LTR retrotransposons.;citation_volume=3;citation_year=2011;citation_pmid=21282712;citation_doi=10.1093/gbe/evr005" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=T. Kawakami;citation_author=B. J. Darby;citation_author=M. C. Ungerer;citation_title=Transcriptome resources for the perennial sunflower Helianthus maximiliani obtained from ecologically divergent populations.;citation_pages=812-819;citation_volume=14;citation_year=2014;citation_issue=4;citation_pmid=24438509;citation_doi=10.1111/1755-0998.12227" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=L. J. Kelly;citation_author=I. J. Leitch;citation_title=Exploring giant plant genomes with next-generation sequencing technology.;citation_pages=939-953;citation_volume=19;citation_year=2011;citation_issue=7;citation_pmid=21987187;citation_doi=10.1007/s10577-011-9246-z" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=L. J. Kelly;citation_author=S. Renny-Byfield;citation_author=J. Pellicer;citation_author=J. Macas;citation_author=P. Novak;citation_title=Analysis of the giant genomes of Fritillaria (Liliaceae) indicates that a lack of DNA removal characterizes extreme expansions in genome size.;citation_pages=596-607;citation_volume=208;citation_year=2015;citation_issue=2;citation_pmid=26061193;citation_doi=10.1111/nph.13471" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=A. Kumar;citation_author=J. L. Bennetzen;citation_title=PLANT RETROTRANSPOSONS;citation_pages=479-532;citation_volume=33;citation_year=1999;citation_issue=1;citation_pmid=10690416;citation_doi=10.1146/annurev.genet.33.1.479" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Kurtz;citation_author=A. Narechania;citation_author=J. C. Stein;citation_author=D. Ware;citation_title=A new method to compute K-mer frequencies and its application to annotate large repetitive plant genomes.;citation_pages=517-517;citation_volume=9;citation_year=2008;citation_pmid=18976482;citation_doi=10.1186/1471-2164-9-517" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=H. Y. Lam;citation_author=M. J. Clark;citation_author=R. Chen;citation_author=R. Chen;citation_author=G. Natsoulis;citation_title=Performance comparison of whole-genome sequencing platforms.;citation_pages=78-82;citation_volume=30;citation_year=2012;citation_issue=1;citation_pmid=22178993;citation_doi=10.1038/nbt.2065" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=E. S. Lander;citation_author=M. S. Waterman;citation_title=Genomic mapping by fingerprinting random clones: a mathematical analysis.;citation_pages=231-239;citation_volume=2;citation_year=1988;citation_issue=3;citation_pmid=3294162;citation_doi=10.1016/0888-7543(88)90007-9" />
<meta name="citation_reference" content="citation_title=Plant Genome Diversity;citation_title=Genome size diversity and evolution in land plants;citation_author=I. J. Leitch;citation_author=A. R. Leitch;citation_pages=307-322;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_issue=14;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Ma;citation_author=K. M. Devos;citation_author=J. L. Bennetzen;citation_title=Analyses of LTR-retrotransposon structures reveal recent and rapid genomic DNA loss in rice.;citation_volume=14;citation_year=2004;citation_issue=5;citation_pmid=15078861;citation_doi=10.1101/gr.1466204" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Macas;citation_author=P. Neumann;citation_author=A. Navratilova;citation_title=Repetitive DNA in the pea (Pisum sativum L.) genome: comprehensive characterization using 454 sequencing and comparison to soybean and Medicago truncatula.;citation_pages=427-427;citation_volume=8;citation_year=2007;citation_pmid=18031571;citation_doi=10.1186/1471-2164-8-427" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Ming;citation_author=S. Hou;citation_author=Y. Feng;citation_author=Q. Yu;citation_author=A. Dionne-Laporte;citation_title=The draft genome of the transgenic tropical fruit tree papaya (Carica papaya Linnaeus).;citation_pages=991-996;citation_volume=452;citation_year=2008;citation_issue=7190;citation_pmid=18432245;citation_doi=10.1038/nature06856" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=P. Novak;citation_author=P. Neumann;citation_author=J. Macas;citation_title=Graph-based clustering and characterization of repetitive sequences in next-generation sequencing data.;citation_pages=378-378;citation_volume=11;citation_year=2010;citation_pmid=20633259;citation_doi=10.1186/1471-2105-11-378" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Novak;citation_author=P. Neumann;citation_author=J. Pech;citation_author=J. Steinhaisl;citation_author=J. Macas;citation_title=RepeatExplorer: a Galaxy-based web server for genome-wide characterization of eukaryotic repetitive elements from next-generation sequence reads.;citation_volume=29;citation_year=2013;citation_issue=6;citation_pmid=23376349;citation_doi=10.1093/bioinformatics/btt054" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=E. Paradis;citation_author=J. Claude;citation_author=K. Strimmer;citation_title=APE: Analyses of Phylogenetics and Evolution in R language;citation_pages=289-290;citation_volume=20;citation_year=2004;citation_issue=2;citation_pmid=14734327;citation_doi=10.1093/bioinformatics/btg412" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. H. Paterson;citation_author=J. E. Bowers;citation_author=R. Bruggmann;citation_author=I. Dubchak;citation_author=J. Grimwood;citation_title=The Sorghum bicolor genome and the diversification of grasses.;citation_pages=551-556;citation_volume=457;citation_year=2009;citation_issue=7229;citation_pmid=19189423;citation_doi=10.1038/nature07723" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Piednoel;citation_author=A. J. Aberer;citation_author=G. M. Schneeweiss;citation_author=J. Macas;citation_author=P. Novak;citation_title=Next-generation sequencing reveals the impact of repetitive DNA across phylogenetically closely related genomes of Orobanchaceae.;citation_volume=29;citation_year=2012;citation_issue=11;citation_pmid=22723303;citation_doi=10.1093/molbev/mss168" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Piegu;citation_author=R. Guyot;citation_author=N. Picault;citation_author=A. Roulin;citation_author=A. Saniyal;citation_title=Doubling genome size without polyploidization: dynamics of retrotransposition-driven genomic expansions in Oryza australiensis, a wild relative of rice.;citation_volume=16;citation_year=2006;citation_issue=10;citation_pmid=16963705;citation_doi=10.1101/gr.5290206" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=S. Renaut;citation_author=H. C. Rowe;citation_author=M. C. Ungerer;citation_author=L. H. Rieseberg;citation_title=Genomics of homoploid hybrid speciation: diversity and transcriptional activity of long terminal repeat retrotransposons in hybrid sunflowers.;citation_volume=369;citation_year=2014;citation_pmid=24958919;citation_doi=10.1098/rstb.2013.0345" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=L. H. Rieseberg;citation_title=Homoploid reticulate evolution in Helianthus (Asteraceae) – evidence from ribosomal genes.;citation_pages=1218-1237;citation_volume=78;citation_year=1991;citation_issue=9;citation_doi=10.2307/2444926" />
<meta name="citation_reference" content="citation_journal_title=Ann. Mo. Bot. Gard.;citation_author=L. H. Rieseberg;citation_title=Hybrid speciation in wild sunflowers.;citation_pages=34-48;citation_volume=93;citation_year=2006;citation_issue=1;citation_doi=10.3417/0026-6493(2006)93[34:HSIWS]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. E. Schilling;citation_title=Phylogenetic analysis of Helianthus (Asteraceae) based on chloroplast DNA restriction site data.;citation_pages=925-933;citation_volume=94;citation_year=1997;citation_issue=6–7;citation_doi=10.1007/s001220050497" />
<meta name="citation_reference" content="citation_journal_title=Taxon;citation_author=E. E. Schilling;citation_author=C. B. Heiser;citation_title=Infrageneric classification of Helianthus (Compositae).;citation_pages=393-403;citation_volume=30;citation_year=1981;citation_issue=2;citation_doi=10.2307/1220139" />
<meta name="citation_reference" content="citation_journal_title=Syst. Bot.;citation_author=E. E. Schilling;citation_author=C. R. Linder;citation_author=R. D. Noyes;citation_author=L. H. Rieseberg;citation_title=Phylogenetic relationships in Helianthus (Asteraceae) based on nuclear ribosomal DNA internal transcribed spacer region sequence data.;citation_pages=177-187;citation_volume=23;citation_year=1998;citation_issue=2;citation_doi=10.2307/2419587" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=L. E. Sims;citation_author=H. J. Price;citation_title=Nuclear DNA content variation in Helianthus (Asteraceae).;citation_pages=1213-1219;citation_volume=72;citation_year=1985;citation_issue=8;citation_doi=10.2307/2443401" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. E. Staton;citation_author=J. M. Burke;citation_title=Evolutionary transitions in the Asteraceae coincide with marked shifts in transposable element abundance.;citation_pages=623;citation_volume=16;citation_year=2015;citation_pmid=26290182;citation_doi=10.1186/s12864-015-1830-8" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=S. E. Staton;citation_author=M. C. Ungerer;citation_author=R. C. Moore;citation_title=The genomic organization of Ty3/gypsy-like retrotransposons in Helianthus (Asteraceae) homoploid hybrid species.;citation_volume=96;citation_year=2009;citation_issue=9;citation_pmid=21622351;citation_doi=10.3732/ajb.0800337" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. E. Staton;citation_author=B. H. Bakken;citation_author=B. K. Blackman;citation_author=M. A. Chapman;citation_author=N. C. Kane;citation_title=The sunflower (Helianthus annuus L.) genome reflects a recent history of biased accumulation of transposable elements.;citation_pages=142-153;citation_volume=72;citation_year=2012;citation_issue=1;citation_pmid=22691070;citation_doi=10.1111/j.1365-313X.2012.05072.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Steinbiss;citation_author=U. Willhoeft;citation_author=G. Gremme;citation_author=S. Kurtz;citation_title=Fine-grained annotation and classification of de novo predicted LTR retrotransposons.;citation_volume=37;citation_year=2009;citation_issue=21;citation_pmid=19786494;citation_doi=10.1093/nar/gkp759" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=J. D. Stephens;citation_author=W. L. Rogers;citation_author=C. M. Mason;citation_author=L. A. Donovan;citation_author=R. L. Malmberg;citation_title=Species tree estimation of diploid Helianthus (Asteraceae) using target enrichment.;citation_volume=102;citation_year=2015;citation_issue=6;citation_pmid=26101417;citation_doi=10.3732/ajb.1500031" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. Sveinsson;citation_author=N. Gill;citation_author=N. C. Kane;citation_author=Q. Cronk;citation_title=Transposon fingerprinting using low coverage whole genome shotgun sequencing in Cacao (Theobroma cacao L.) and related species.;citation_pages=502;citation_volume=14;citation_year=2013;citation_pmid=23883295;citation_doi=10.1186/1471-2164-14-502" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=K. Swaminathan;citation_author=K. Varala;citation_author=M. E. Hudson;citation_title=Global repeat discovery and estimation of genomic copy number in a large, complex genome using a high-throughput 454 sequence survey.;citation_pages=132-132;citation_volume=8;citation_year=2007;citation_pmid=17524145;citation_doi=10.1186/1471-2164-8-132" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=J. Dudley;citation_author=M. Nei;citation_author=S. Kumar;citation_title=MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0.;citation_volume=24;citation_year=2007;citation_issue=8;citation_pmid=17488738;citation_doi=10.1093/molbev/msm092" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=M. I. Tenaillon;citation_author=M. B. Hufford;citation_author=B. S. Gaut;citation_author=J. Ross-Ibarra;citation_title=Genome size and transposable element content as determined by high-throughput sequencing in maize and Zea luxurians.;citation_volume=3;citation_year=2011;citation_pmid=21296765;citation_doi=10.1093/gbe/evr008" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=R. E. Timme;citation_author=B. B. Simpson;citation_author=C. R. Linder;citation_title=High-resolution phylogeny for Helianthus (Asteraceae) using the 18S-26S ribosomal DNA external transcribed spacer;citation_pages=1837-1852;citation_volume=94;citation_year=2007;citation_issue=11;citation_pmid=21636379;citation_doi=10.3732/ajb.94.11.1837" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=M. C. Ungerer;citation_author=T. Kawakami;citation_title=Transcriptional dynamics of LTR retrotransposons in early generation and ancient sunflower hybrids.;citation_volume=5;citation_year=2013;citation_issue=2;citation_pmid=23335122;citation_doi=10.1093/gbe/evt006" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=M. C. Ungerer;citation_author=S. C. Strakosh;citation_author=Y. Zhen;citation_title=Genome expansion in three hybrid sunflower species is associated with retrotransposon proliferation.;citation_pages=R872-R873;citation_volume=16;citation_year=2006;citation_issue=20;citation_pmid=17055967;citation_doi=10.1016/j.cub.2006.09.020" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=M. C. Ungerer;citation_author=S. C. Strakosh;citation_author=K. M. Stimpson;citation_title=Proliferation of Ty3/gypsy-like retrotransposons in hybrid sunflower taxa inferred from phylogenetic data.;citation_pages=40-40;citation_volume=7;citation_year=2009;citation_pmid=19594956;citation_doi=10.1186/1741-7007-7-40" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Vitte;citation_author=J. L. Bennetzen;citation_title=Analysis of retrotransposon structural diversity uncovers properties and propensities in angiosperm genome evolution.;citation_volume=103;citation_year=2006;citation_issue=47;citation_pmid=17101966;citation_doi=10.1073/pnas.0605618103" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. Vitte;citation_author=O. Panaud;citation_title=Formation of solo-LTRs through unequal homologous recombination counterbalances amplifications of LTR retrotransposons in rice Oryza sativa L.;citation_volume=20;citation_year=2003;citation_issue=4;citation_pmid=12654934;citation_doi=10.1093/molbev/msg055" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=C. Vitte;citation_author=O. Panaud;citation_title=LTR retrotransposons and flowering plant genome size: emergence of the increase/decrease model.;citation_pages=91-107;citation_volume=110;citation_year=2005;citation_issue=1–4;citation_pmid=16093661;citation_doi=10.1159/000084941" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=M. Vukich;citation_author=T. Giordani;citation_author=L. Natali;citation_author=A. Cavallini;citation_title=Copia and Gypsy retrotransposons activity in sunflower (Helianthus annuus L.).;citation_pages=150-150;citation_volume=9;citation_year=2009;citation_pmid=20030800;citation_doi=10.1186/1471-2229-9-150" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. Wicker;citation_author=F. Sabot;citation_author=A. Hua-Van;citation_author=J. L. Bennetzen;citation_author=P. Capy;citation_title=A unified classification system for eukaryotic transposable elements.;citation_pages=973-982;citation_volume=8;citation_year=2007;citation_issue=12;citation_pmid=17984973;citation_doi=10.1038/nrg2165" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=T. Wicker;citation_author=S. Taudien;citation_author=A. Houben;citation_author=B. Keller;citation_author=A. Graner;citation_title=A whole-genome snapshot of 454 sequences exposes the composition of the barley genome and provides evidence for parallel evolution of genome size in wheat and barley.;citation_pages=712-722;citation_volume=59;citation_year=2009;citation_issue=5;citation_pmid=19453446;citation_doi=10.1111/j.1365-313X.2009.03911.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Long Terminal Repeat Retrotransposon Content in Eight Diploid Sunflower Species Inferred from Next-Generation Sequence Data" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
<meta name="twitter:description" content="The most abundant transposable elements (TEs) in plant genomes are Class I long terminal repeat (LTR) retrotransposons represented by superfamilies gypsy and copia . Amplification of these superfamilies directly impacts genome structure and contributes to differential patterns of genome size evolution among plant lineages. Utilizing short-read Illumina data and sequence information from a panel of Helianthus annuus (sunflower) full-length gypsy and copia elements, we explore the contribution of these sequences to genome size variation among eight diploid Helianthus species and an outgroup taxon, Phoebanthus tenuifolius . We also explore transcriptional dynamics of these elements in both leaf and bud tissue via RT-PCR. We demonstrate that most LTR retrotransposon sublineages ( i.e. , families) display patterns of similar genomic abundance across species. A small number of LTR retrotransposon sublineages exhibit lineage-specific amplification, particularly in the genomes of species with larger estimated nuclear DNA content. RT-PCR assays reveal that some LTR retrotransposon sublineages are transcriptionally active across all species and tissue types, whereas others display species-specific and tissue-specific expression. The species with the largest estimated genome size, H. agrestis , has experienced amplification of LTR retrotransposon sublineages, some of which have proliferated independently in other lineages in the Helianthus phylogeny." />
<meta name="og-title" property="og:title" content="Long Terminal Repeat Retrotransposon Content in Eight Diploid Sunflower Species Inferred from Next-Generation Sequence Data" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/8/2299" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The most abundant transposable elements (TEs) in plant genomes are Class I long terminal repeat (LTR) retrotransposons represented by superfamilies gypsy and copia . Amplification of these superfamilies directly impacts genome structure and contributes to differential patterns of genome size evolution among plant lineages. Utilizing short-read Illumina data and sequence information from a panel of Helianthus annuus (sunflower) full-length gypsy and copia elements, we explore the contribution of these sequences to genome size variation among eight diploid Helianthus species and an outgroup taxon, Phoebanthus tenuifolius . We also explore transcriptional dynamics of these elements in both leaf and bud tissue via RT-PCR. We demonstrate that most LTR retrotransposon sublineages ( i.e. , families) display patterns of similar genomic abundance across species. A small number of LTR retrotransposon sublineages exhibit lineage-specific amplification, particularly in the genomes of species with larger estimated nuclear DNA content. RT-PCR assays reveal that some LTR retrotransposon sublineages are transcriptionally active across all species and tissue types, whereas others display species-specific and tissue-specific expression. The species with the largest estimated genome size, H. agrestis , has experienced amplification of LTR retrotransposon sublineages, some of which have proliferated independently in other lineages in the Helianthus phylogeny." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/8.cover-source.jpg" />
    <title>Long Terminal Repeat Retrotransposon Content in Eight Diploid Sunflower Species Inferred from Next-Generation Sequence Data | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"pE34_mBFBCr-HESv0ILJ51R_c_had4ZdRfnUK97uHJs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460441","apath":"\/ggg\/6\/8\/2299.atom","pisa":"ggg;6\/8\/2299","ac":{"\/ggg\/6\/8\/2299.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/8\/2299.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2299"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/8\/2299"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/8\/2299":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RK2tgMkVhWB5moMEYrCq2ko9am4aY6NqJW4Jc5iilkckRRU+Z0+8SQkJ27t4Hi+DthiiLaD9TFR0IH5YwhJzctL3xrDbIdcmzGUAt7bufp8VLny0l1nf3dosjKSKm+ggwPDlBkKePsZEkNNsSppj639MMB1d2hONcUphWexzni2Q9QOf1PeEHbvycicoTiP+fMSmh3j8vi3VenQ1ZC2+poeiUgBLicuOaFyyvfl7K5rAaaHEFclv7TtwsR03gufQcoQB\\\\\\\/4bGM7AtHRzoJm8GpIjJKx9\\\\\\\/llU4BhK2yu\\\\\\\/driYnL0mjT++uwdcEOxqntmhBidnwjhPNPNpLJY9Is7ttdB7bEJx597llBehRFA5yBlJW\\\\\\\/yhYzM91lMya4neqBj+SFg\\\\\\\/ezfuVy4IZ1akAA3z6KbNf+STGHlRe+2xyQgpp9sa7Cb05oUtviFk7fDDk\\\\\\\/O+nEbLglOIB1gmJDOlYD3NZJhckizZpiAFqR\\\\\\\/\\\\\\\/x\\\\\\\/ZAgwLZJwo\\\\\\\/29mqyOic3zv1VDTiMNjduKNNRYSUXILgxc+CgXqBDwM+RRqPmGNQ7QnDMK9dIYQML5um3y0LoIWdRo60eubirkPZwH8dQGxmYFqYYq5LmtEWRuxY1jw\\\\\\\/V\\\\\\\/Zj5OiFDFDt6cRM6deWbB+XCNKwviVvtEa9BI6egiWH8HZkRIvIFMNcnlWgjNTSrQqC6m1K\\\\\\\/lUs5LGvSu00uiqRucWCQwPw5bzr74aog07WxBIxwDmAhaki0T4YrOMFdsDNAauxSAcVmYANtUx4ivoNM8vhUSLl6KYhv1G6LhD5zSoIZQtLj\\\\\\\/EcMZlt1IbOE9yWWxi1wrRR4CpUUMBbHrq6DpOij+pNHUwiTCb39Gmte6FX5PW0vztAs9iHglPhtnalM\\\\\\\/\\\\\\\/lmHCwsHwC5yO8\\\\\\\/wxJXgvF1jPlgwuIpbINb6BipZ1v+C9WCZ\\\\\\\/Q350+KHFI+JzHvq8xb3lZMNFROHkB39skedMKDdPGfnDNgGC6XYSfgqCtqQUwO\\\\\\\/mBD4qLa2qu7wBtlIoz5wB6JRFW+9hm+aGhqRC4RxphBuKksHVP9fTSFgcS5sSQaeK9jra52DB482nLR\\\\\\\/eFM6I+Z54GwaZI61aEGJt9sFvSrzLU1V\\\\\\\/1DsDMdqIKZELwmxg4M+Iq+UeMZbNGNO2OEMUUW9kGlgZgZQat85glNYSW587G5vfuQRLsWIt1W6yaTH\\\\\\\/7xO4\\\\\\\/oY7gKbFphv0Lh1XmCuITvbcCMu1Pi\\\\\\\/UZdS8WwIkNpjj5ZxbZguIrf650eTqkt2xF9448oZc4H72JFeXao8BgNtuZGfgn29PYO\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/a8mITS90W3gamqkVLvf\\\\\\\/A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ddfa0e4e62ef5e413624b293fa4fd1b774861d5d7f8be8bb77e0654aedf4b0db\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Y1WMjEWX2qVmXhSEO5egDraQW1Di0DZB2NmwKLaph5yc0C4X9vD5NMOsgNc\\\\\\\/T2tk7Ar+oO7LeDcV2Trd0lUWtVpdDoVa6L8utND6hfWJZxr3VVej7rvWs6O5PibItVCNbb4ywfmMG6SxWMTF58JyQNJtJ2\\\\\\\/RDh609ki4wp1rcVTim38BZbHF\\\\\\\/c0r0mk5Aicd5PVnPhvZXjZo\\\\\\\/kJHfdxwgAhkwOwjLFPP0dtD5DPcyJNUYx6+hQaBWlZTGyN1IougUrJ\\\\\\\/EfkvL6kRailfBT51H+ZxQbPwHA1WFQeQEP04FS6holSTYup+wFVxjeVtUX\\\\\\\/7uNB05CF5Pz37dX10qzSPnpB5ADwQ1dhMBQmikZY9MSn5B9bnFRw4e018Z2GTibYywEcfenwAZSfCjpJsgEAX6H+TuETu95HvFZwFp3VSG+\\\\\\\/pGjSziMEvbrI1JXK2gIJrUIBTSe\\\\\\\/B2F6HfGYAtu6+fCvJY7j5LyWQKgkcjOO0HsCCMmlPB15\\\\\\\/fzXzajRc94aK5dWV3Dz9EkH7RlV8bzuCpenadMCeNBuFdNjU0Ij+laYjdwf2vEG4+kZiPRiHi\\\\\\\/QF7Y8RlHfdMy42jbmrv85hbpNaCExGjbSZkCWe0iOvSV8FgPA+fr\\\\\\\/z\\\\\\\/YU1oo8h37H12SISZYYRuriy3\\\\\\\/wzoDSqd\\\\\\\/9M2psKbWVN1fcVyXKKfwJEEl+bdoqKNR0vMdA7S38Xk313Uyjg54MrfHDuJtNZaIxep91UO4IK7uY4l9JWcKByPL4TjvU2CTeeC8lxzA6m0K8jzGwiU98bZ2czoD5IZ3S3nsHa44yrvei0c01GKL5WkNZeuP4vKnQFn24UJdcp6XntUGlHwW6XLjXWD7EGKoVFkfgAfy1mPQuW4q1JS6wnL5u5uPCr97bUzljbN0QTi12OgO2yruS6Uxh27J45v3Cf8MR1MjesOp5\\\\\\\/QyT\\\\\\\/YkS8z24ERk1iTTDSw6zDZbcRB2g0TCDVVeKjDD8fbyaxChY\\\\\\\/GJCply4XkC05fh+RTB8Aca0ugpaUrpag3nb6sMixvYHDoz61jPQjtyue2sXzomrI5tj\\\\\\\/SlMWq7ByPuypun14KOFOB7KR\\\\\\\/hj0B0AXpb2b9wMBlPyihl94JZslVEkfYdWinp9473NusoWe9cCAwVqEK5pCh\\\\\\\/r8I1QuiJDob4rNALQ+TWIJzgJwRyM4hOaAXDGLoDSdo3G4ccyp9Tq8ibJwkBBg5zp8i5MbzS8Ih6fQ4BTQFcSQ2G5I\\\\\\\/MgGII+8AXguEYaNzJJ3152qAskGR+3nkpwua2cZO9RDwFDUuZpLaW1tQHg9CKwRdREViu1x\\\\\\\/joiVXNogWy5GhoDAs97M34ujawHGCgUDWNNwzyNCT98J\\\\\\\/0xgIwXKRnRq\\\\\\\/eSwnzs+3yT9rTKI6r3AKlW\\\\\\\/ZJYXMv7aODcoRNFeyWfcsAMQcKdNnF0f7s2LZ7m7PckuLIYnZoFoYWcxMhZ7FADed4L09fmWcEQLVfnlkQDXY4kfHXEpbvLSayDExCSBBGgCUWGFD8YOBcJP1yie3qL0uiytL7T9aTIg9r8hleT3ONkxYSAG64XlBimE0BGy98Ta8gwhglp9TLUqZFNLdLF\\\\\\\/mfGGkdKEY8Mx4cN58OFLY8Ou0j44VAtRAvQgwEwoVR9jWi+viaAUMB9g6pDuQ7qmxDRmQTv17Bswg6GnVQ45QJuMO7zLQk225BppYXWqxdU7\\\\\\\/GW+5GVubhgYeG6Hr8Gc6Is2sdbIx0tD4b6Z0aSrTlsG4+ATmv15wBMyGcz7gZTOXvZQuzN0Cs0zPj7lRYldwMAj+isT8cGDv9HovYTDoQ\\\\\\\/VtOWWfRjOgQ1ccJ8K4O5QEwdxBHt4QAg4YDH6YgbVahXevH5uPBdvdbkgQZP7aedHLhetMRQ7DGY9frd1LOlShdnhGhrgicRQcQus95ktWsMmea8ejPEodextFPzHBJnHORdeaSwvMNRfB\\\\\\\/553LLQne+rUeiSteZIrmNXWDmnJb9hvGwgGw0wbGphSH0EkoV\\\\\\\/GNm\\\\\\\/RQNgFGqiogHf5lP4ImKojnUoW030wib5wAX+H\\\\\\\/LmXPaR5dKl2dErtYjGllG1\\\\\\\/ClNZqGSpgME8voFu2uoJ6737CQr\\\\\\\/v1ZXUapZQoEmG2aQD++P2BVem4ZP8vgBDsSgr3hrqBza9G7zZcoWPWyeTTCgWu\\\\\\\/87CS\\\\\\\/d9lbbDgG\\\\\\\/ze3HJtnTQNwD456REYbtF257LRp2uO50FniavuYEXAd7GQCCeaa5lZOQ8zBQcVgztS2jokEDBeHbkXu+qsHWugAdGdoQIxL3y3ukpCALN9lyOye0r2OkKSDvsSkESoTOZYepmByc03yduuVChhjxld2pbQoEOVx1dHutJZ9l9RsdGYvF0gmmp0heqaww76f7SFmNhKNC2sHUFwCTZzq927p5f9hdMLQ49fAvcguUwdPKWJPy+vUlEX0neXAN6rfnJpUONrw2YHhoBtsZhHQTiHfXcaAWRVXPkL\\\\\\\/kGMJgLxtvsWY5F2ZCY3QE1tWxrQnTNExarT9y2ZJoeMISzWoEkBcnKujVxH1P3KkRW5v\\\\\\\/zQjN\\\\\\\/fhH8jlDUPDko1F\\\\\\\/QIVIdCDi78PJcfD1Yiblm9nfQgS0uTF29VDpAKMZNP8YzaFeKblWJmmVVDM9ZTgeGm+0cxNF8+kHN2euHMFggsc6VVYiqpCMWUfi+S4yGhxYnqOnHdqhBIC9e7uu1UrvoG\\\\\\\/28ZaRb7QmUXDKpv1OMTrwiJhQ4gfAWBJPW+7\\\\\\\/ebe\\\\\\\/lF4xfmCMNqELYkT5ZcPMk5TpuW\\\\\\\/HlmGgrnjEwya0uAVlBd+v4nnYNX3Wyo10NLTeovuic3nssuPp6ZaEKMJAJmagvxlYRI+Bye5iUrFwki0HAIaj9eBZKEOtaTKDmLe2O\\\\\\\/uwAK5IA8bolKoKRx360KaHHKWG2gKNPJvzA0NwRzbzZfWdOyMY6TSjc4J7BjdWTIT7R5pOYGXUKRQ1UknQ9eVZp4DmiJQWkDEsQed5kh6Lql3\\\\\\\/3FOzOoUKBvw+BdvJjA+zMRcVtcLiJkskjl1PtDRD+hNrerSfCv5NTVxmehDAv69EdTr8rWaymQmHDBRSDQp6QreQZljbeqA56jEeO3TS3dJe61Zo3t5HJjZXNvLLzcpP8NXFPnSWm+8ISwSevB6Q3NMe53F3W466KrElZX\\\\\\\/LB2dr0zhUiTMMbuunvYPsH2gfeiz3laDZoiHFbJehmPFcoVCP\\\\\\\/O9bafq+1L12OTXsgzIwTnkba3kE2gKMFJgtaETCl3N1Q3SeC\\\\\\\/QQfVVw930b636Ufn9GaSzBOiybHAo\\\\\\\/lYr9NWu45jtDy163E5cxvqQBCGtZjLSXYUL713FsLZL7bUYEvPeI89vp9YOmwg+srfTb393nkESO5PifQe5eiZp888yXtKIm8JchbddvfaPxLXir2gMX7LOjV\\\\\\\/FVd3nFWv9CaRvUHhEWKFaJGpeocGQorT3xky9rE\\\\\\\/5xeDHcvHn5kctIWvOjvHDDNhoP5g75o1VIlvLpWZluX5RbrxvJ+33zxzhbQk9A8zB37ilxKuGRR+qzSU1EicnSU\\\\\\\/YMOCUFfT8vAavaCqCpaFqUm8mZFA05nvnIhDvNdDKa6NeZXNhDLt1Ij9B+8uzJqAu0R+TtStW8Mpz2DyLgAcamq4UmXUl30bJlydrw1csHVQkgyP4B6UXQuGt9QH\\\\\\\/9zQS+\\\\\\\/dnLrucxYfD17ER+Y+ijdvzXLQBPDDnIJDKco+tySJjPUJGpxyB56qIeIAu0rDnfB+JAPsbsAoYK4tSr0pkdjsxnQOT7Alj3d5d0WXu4NJ4wJTjTud\\\\\\\/L\\\\\\\/3aXcc1ZNkCVa1ygQ1ObkwdKFKuNf9JYnje6vdEDl\\\\\\\/xdck8uTMRSBSE\\\\\\\/kN7hN4eOVc5hptNnSORxhZSFmTVuUdhD\\\\\\\/pZFcwdNkbj5gN\\\\\\\/ebhvW4t2mAPcnkUnbRMq6Jk\\\\\\\/+XTIDaD7k4NNsDwB\\\\\\\/GUsL8I0uWasFFc+S7saeEQONlT6vAUxHGCZM7gxLywPXnIWJMlCrozFB2TCw5JtIUSgiBSl4iy\\\\\\\/3k741ud82+B2FW3T5E8KHL9la2GBmL+kihPl1I7XAXPVLlgAJAb+zOg3KtbVEdCP8MR3huKXKi+A519a\\\\\\\/dINphXzPnyWJC+MXeZdcMQv0fLv2V3E72aoncYY7oKgWT5cx11r5MtQk1zWtM2BFfVkFwXBHF2kb+\\\\\\\/CfymHmY1Z09gC5iPpFLLJ6Czvkb5lI2rLm6L\\\\\\\/IAqGAcUFGoKQ2YaUuJ7PusXsmt7OzeiAD4Wi8kShcJD+MuxlwN+XyrK1hc+IbBvj9iK628ALKz3pAF5gDQCxQNHRem4Ez7UwZk2J5zjqwbksMD4TJ6oY+ofCIdgGPl9AQYfu35ZOR5s02srHhvCYj4APjU9XivUEgwtGZIrWCokfwHdo6ta8dQBUIZJHTKyKO0SUvvgCrJvnJ7CuIlkEcQY2nY0Yi6J\\\\\\\/6aukSfRiMt01+uLv0fdCIiZqJcuK4\\\\\\\/hLLD0sjAlpnxwi\\\\\\\/1CDVn67T6b7RAhv\\\\\\\/MNeB6W+Q3v9zOzdkxkSljco8ReT4i38CuGc2IfCzJHh\\\\\\\/pwibpzlZ2Ck\\\\\\\/1aOD58t3fzThc0UtI40mGIKUe0BRuPRJZMuJZN373sE0vj5QpwOQy5yAIZNqnIFKi3CoTiWXA8GkR\\\\\\\/xoPs4n\\\\\\\/OV4uPI3WvMCudQYCiKHYcC\\\\\\\/WBhIttDD1kQqZfgoOilW9ZVAwDC4el38lVRd7mXuD2ek85+yjV3j7Z+iXh8SBecfa+yTPtpOuPzn+wxqMTyQUjnU8ITRjPFWKkFh7J\\\\\\\/nS27kK01sWTOg287y8rSu\\\\\\\/Q3qKzQNBkMDN7GiJNgANyIpDnKu1+h3m+O0SOx41tOrRGoUkGMK223SXkE5glDfAXrUPTqayGeMd5Vc8K91SM8Icx7RccgBi4ZmdM51A7e2asLa0DjTA\\\\\\\/fitJQlKJhNBRYIwKfBKjI5rFCyzrHpylEZ433aeoICNdbHD39SqzoqNDI19qWDrAwpzXUANdbwBZipt3JLb6F9oj3NsXwZZDfybBx4CzQG+ud6wU9fTO0gTlecfI1lA1Ei4+V6PoK5dlITGJwyM1nciNzpfCdUMBloiGIgBbC9cqxs1vpvXgj99Tbk4ogKghppzb6QbrWQSoKVnPYIOrXilbKgZFsAoTFBl2+R9OmOVJRVVxjGR91YwyyZuq3C8KG0XRb13iZOJlZKzxY3KegpeOeIrj4MF3J6Tdpu\\\\\\\/H0nARFMldcYtgdvjXwTM74AUMVBG29FY2OIZjDKNIUFENBh+4SQyUEZI2fqCPBQOr0e6PhJryb9Fqq\\\\\\\/WeMx5jiPY\\\\\\\/xO5v85X22t5b4xIOUxOlkbpuUPbz1zssh\\\\\\\/AzA27lvjVxzDnDWg26PMGZZ\\\\\\\/7AxEDw38lYTH1qIKryMEy97O36YRdPltAU\\\\\\\/qochHi\\\\\\\/tEaTRRGTgRnQ8oNgENgzPc\\\\\\\/MIZLIU3fMZLxbBl7056P0rPNgudG6BN3hMq8Jq8L\\\\\\\/ksgOk7wwN1Ey5klIctKoiRxqin\\\\\\\/XCO0yqaoVel+1zwiSn5rcJPgGzWyC5CCwJWrS2Hl9VLkLDbXThaJXL8x99SASYXCcVWzEmgPAgjTcxt+FXqSZV1WZ1fvTe1y8HIZlwl9LQm8RicCyn3t+VlsvtPxoOIVsWUnpJ9X9ODYtuJEmuypFuRqCKnKIEl1xkmBlfgTVc1Lwk2WbbIrm4ynHak5X\\\\\\\/4Cecc9KDCWUJoCGKh1AoDQ4UD7gGmFBaU7NXwNroQ2H8ID\\\\\\\/BmnuLC8aMkV7HpP+utvqGyAwtGzuGIAOb\\\\\\\/zlRyMI67GTQuipOq8bHM1YIZMLELEAj6D\\\\\\\/p2IHYizorbknQVu4jufxtjq18CGnVLuC4bfnP1Oc0JlD6P7ITUGg1iyr2ZGI+4xfpInSJh7NSGZNPtnregY8IIXz+H7lSkn2E3NqI1+pjx5MklKIer3YuF8bBfCaxz2+YkwutLAc7xqrKlAARNxG9f9gvPnE4PxP8DCZTLH7AqbnLuaFuVsve\\\\\\\/K3ZjEPDzXkd0hxWrgLG9Ja\\\\\\\/r3U96u5TQgmEU9HnKr8tvBucSnsHBNHjNIB+Pe0VujCYnO2LzBUZEdzpwaDq08IQOJtbKGZzCazA2RA1hFXdi7EwM5he2t6rqL4n4ZoIIGT3z0AXHZ3sIn\\\\\\\/YHuEHQ7bmHfXnRw4\\\\\\\/CGWcwjTavnTg1Cokg5iaZ9pBWEOyQ5PGvAWdpUjC5eGlI8Cg9D3fhaF1eYNp1q7YFb59zewvTH2tjOwT5Kcpb7xX4y10E74KhxMuYOxG1iul4fXWnfuFquNzJlJ2EPe+uaaL\\\\\\\/oKK\\\\\\\/xBE55A57ZHzFRXLed+yPCfEYWckcWM9GtjqIOyeYne4N\\\\\\\/LjErlM8JgUPbvOq6LoP+nY9\\\\\\\/ObafHfWLKX6GMT4XUQf8YDtQyCOL+sXXaF4UODp3EfNe3eXV1yaNRPJlpByyakw0KAvVWNVaXe1fsN00QMpUSUFg2oOcAAXMaYPEL47bbfMN5mpFtwzIK+zDMFo66cPrCctRWBSv59FlOYR1bKtNBxpWovHqES4\\\\\\\/m6btLhJfQDF5UNF8iYRpKFgyco8+AyGx1m8tTMeWElmUisaPwE8ETyl0\\\\\\\/G0bRjhLtXUqQZzmplz1uIVxVoOOwCPjzMOI6sH\\\\\\\/yIgJrRfCROXU5zOO8yietqb\\\\\\\/pQMtHTElDxhfs8NEf9BkVUPSzl49oibiSYb7ssxucuAv8ryHbgLG63lb6SFYknshSO414q33qMTtbiqABtb1XKnj+KCHMtAW1J5vJBvKU6PjGCh2coxVvbSAUZNodWvw6NIirCKjRWxIS9J4LiDGX3LB8MdtzAKVToaC\\\\\\\/yktezPsK3UGyKSNq7v3mgoxWEc4BnAieHpzw4gXbxeM\\\\\\\/6Zd61XgY8BhrUf1gk\\\\\\\/8r4f9A5st4Utk30Q4DoEMxFS4lUutN7Ay+64No5geWokFic9yfI01lLa8YX\\\\\\\/wRK1w4DiDzStxeGAF4e3UsoQsvw8GVhXr0vjVtcdNF+5EUsnXjvzfx6O6HBbqfncmzpWSj6zGaaXGm\\\\\\\/DdWp1ZomZVH9FapXrMww\\\\\\\/3nWa86tPwiOqkmEQ2F7W8keGRqFRoed0SG+DKhyAq4ScMRy+VBVZPsycx7V8z0PIXOp3e83toW+p2l87FOwGHllMg2P\\\\\\\/Yaqr8EdBC1dsEH0MrDi9HYbn4gB8Y5b50v1sQ1atm8B88SVqzK+uRDx+qrlA4MvCBv9AhLyQvFTlAwjCLxWDCSt7zwj4\\\\\\\/ddt5QfrVMZ9t1BJWbwUjjJJBENW+ViLRWnEIAbbWc84V6M\\\\\\\/LowhG5e5xtE1QlpH5GRjy\\\\\\\/gVstjzPMfBn8PA2gw28E\\\\\\\/OTIMuJvwzOVnCLQ4+7BEcCnvmLkft7K1U1MtIVcFzw022hxLs+VPa7rpJdltCzKd3RAKn6V74w+bPt\\\\\\\/bWDleHmMdVCRICsl\\\\\\\/HMgoZA7VuL\\\\\\\/\\\\\\\/BsR7ucnsCDbw52iephj1hcmrvIDlFCxVBn7CeBUSgQWf3JdU73Mm9fYaeJ6XiUbnyoaTIzDKzzGBMLRUi4ApLze1B3W3ze3P3iPSdqCgmmXODWPsILhUnPBmirvW7pwc8T\\\\\\\/d1AV2VRrV4+BkRs8p7zPbVFlL39ZEITj22gbwVoTJsSevW\\\\\\\/+Zz2wYrXlJewqgkwHvjFx1LHYQHz0vXQ6IxhH1WNR6Zm5KOWHgCrfTytIyh8jmM5chPDsS62+p57EMQYcelM8ISWbQNtQjxXYCuhkHkYZCvM21rQCe+rsEM6vSAsTUaVP\\\\\\\/BQ\\\\\\\/2W7pRAV9DS6mxI64ZThXCNrDucsZtiGrIx5\\\\\\\/RfSx3+772nmSLzVF+t63lu07t7Kgzo7ZSz2PbnhYFDMUdX6R6AvZz5o19S5AyQB9iS+BKM5nZHx4nPrW6kbf8kvRE86FUXyUJNxkANDRGfAJnCWyI\\\\\\\/+dRilWy2kmjK7bM3jaf3l8e5rdUHcJnXSFiQwX8sGhxygnmAB\\\\\\\/Ghu4exroEraGFzCmCAjaXJtNzFUmTewAx1yLZV6vZ8E4hNcs8FlK0q75e6QWOxOA0KjFaTefvH83GvTDiPti5ZyCXucniuB+niLAx+i6C3ETo3w3NBfZhHM5dpqI2a2hILPDtyYwZr+Prw3SX0XNrY7oGU5dCl\\\\\\\/HxHTDuA1nAqtUgnNkWNf808g1rP8Yl8tb4JCW1QeDbz36kjohPIc1h7DT47RDBSK17IotrCR6Mi3H8m3UvdL23q03QaArfS37rDB1K+NCdxw2Yqg4GLoo+51PKJhz6HtFQR\\\\\\\/V60B+pfu32Z5+Ha8\\\\\\\/jh6G9m9ipWsFSU4jLJxuIXFT1KcF9KyYslgivJLMbRdejgv+TECehBtSt+IiuWEg2Runy6B3X0DGefBNlDkom8m4QtTDyu78Cyj66y\\\\\\\/GA4R5c1KOvPvJnpHizyraL9Zj14ZfJssRbyB\\\\\\\/i\\\\\\\/FwEeYmehzRckjgeOIGgVfNGlfZF2u8SqaT1r\\\\\\\/XD\\\\\\\/nxgjZYuTZPVcYarp3hKVyMPLhDVRuIM0f+sxqn+UGhxpWVRtVu\\\\\\\/vx5Ecna+cQGW5AFBYRmfS5F+FIxHwl4+qW+FsAav3TH2NIrsUn63FbYGHGktG8Phnw+qusrYuwkfoLFqP48AtG8tfqdRZfKzwS3tpt+MOHOBVpzjnB5TQcvaecNglgFzV6FDJQl\\\\\\\/iDyl8Eo5VicTAuUi6BoStlm4QP4uGl5Eza4KJFAmXtcBD32JF\\\\\\\/s949j1Gu2M0YtPO9+DN5uz1Zi9sjFFwETNsjuVeb7r6x4B9kfN+3W7fkLNqD1780lJ7tarE\\\\\\\/EDLVhDt1BXPUOJTVDztVEnoxPmO1GV3diY1J8XpseIH7vbPQhnIkCyWGEfbG9CyJcYAuCfk+vslpUpTsA8MSN8AvO6xua\\\\\\\/+9MuXxnVNkL53mamfb5\\\\\\\/TIDd3W2saBY8BRdVcYOKJrk2NII4DCbpkqAFS6lBmxi0u3kp4Gvjg+YxgJxcVH2FyRVsiGhK3rwez93E4bzTshbIzUdNyxmKCD6WlBfW5fkyX21MBx3Tad6hyXqQRJAyd6mAj+K0+qxmVYccJ+APudT356VjlrFMfBLJOGbHOH7SNTQGLfjkcA9QgD658EZ71rjoNp2wzo5OQI3WqYB0c59mJYvIDsmLgZ6EK47cCyui8bRmpkXUXPAWxDAUsPJdMS6n5wnVDwmRvqXHLGz6IQOzGVc8ZdNTpJ5Q7RLMCzTj\\\\\\\/IPXoLlM8J2wyAoW+lR7aoz8eTctNoYrRcX1AFtjjcJF5wEqEcuMKXFW6WoSk4LoydToN8ewSc7TEiQP1PGF11t5umaF2YWvhaFk9lhTe545SoZRK4Zoo\\\\\\\/wsTBy2ZUrcPtEjHKtriYMfeKMoBE81yvz9S1S005XOj5Fw0N1jiTeahGSrvmcGiujpvwNlLm81RoCQXgP7kbj4gmL8x7MABXkDhr3IPV\\\\\\\/5UrrW9OKhtktfUvJeaeaijeHr5ff3RjNHZ4Wpms+U+PAnkce\\\\\\\/613X5iWa76Wd7Ns\\\\\\\/PGDRuAQKctLLu9mxF6jpnmkIumZWDJNejn\\\\\\\/SLZuWbBjUJeVvAgMI94cAu0ySiXOisgp85Fn91Ey5h6kkeoXVNdA\\\\\\\/N9C1yLbRv8PA8ww7lnRWe5YkAYEJo9Z0fRirEITdmsWLyjajLDQ0IFggmWgGBsQRpXp+SNt7RekOcZvPvVWzTdTCxY4Lp8pfv+X+rggPS14NXN6YaP7UD5M6VkQSIcDVoBPs21+z6o8iVMl+2yjGuD+m\\\\\\\/jltvA7HRJpcLUWkOxJ\\\\\\\/6X4jCpr1WIgoE+z+ubREc20TBt1NZezOHt\\\\\\\/7z0RCsYHLTqEDkqdysPSc5ejzvtLZaTlGKkLf+cV5YGHkczuh5JgiM2W74VPjbc8xKTZPuIQEHTEuaYIvlhEtjh3n\\\\\\\/VyszFfv38U+B05Wi\\\\\\\/W\\\\\\\/415qKe1KSVgPVUDkaKd3V8o9cxvBKNOjphefH3FvYOMTFRWCXLu3qrIxPtGNtDM3U3RZ99JGtR4pbZ7gcBapMPcDuOGzyQ1TfCBctNFFx79wDZBJX9G+v65yeZlzGPB9PVfZ+hCDMUV3RI\\\\\\\/UyE9ZdkzxtJ4bwtaOfDVDIl\\\\\\\/i58leBD2Zu4bPFbJNNcHQPrNXq8t\\\\\\\/+FfYVqDpi\\\\\\\/SvxiHgtxAuWn42Kjzd8Yx5ofGnc0O\\\\\\\/jOIJF+pHhrqOXiYe\\\\\\\/obT\\\\\\\/YhzdpFuY813vuUWa47\\\\\\\/Y+WBBrn3X3bHvD4qpG6UE+uJeGk7iEoGxxqEvY1fD\\\\\\\/vMhfukoOAf92VPKVCyAXUACCOSm9OukoOiYoe\\\\\\\/LtJXmCzQStYz\\\\\\\/S+jiVuKxO7qydzmqt9nPeJPI0ngdNsqvXDSyxITypF8n+sEHws0otGTlrG28SDSys+tDk6mFEUKymdLw15A4niaTA07d3fQ0BxjmQjbKxf\\\\\\\/3tIBcxOleXlo3uqD6ghCF+0LPJ9SZZ2cKzXOVj9RWvCuXpvw9Uj5CNeR4IsJbIDKiHk6y\\\\\\\/lkFo+sZcOxLvyF5sFSR5Jc8FDDw6OPW\\\\\\\/ccxqdBLUvJhVk9V7++LfL6WJzrcM\\\\\\\/Quwtf72qNMrGgUHx2QwvirBozJ\\\\\\\/8QlLuw8oRpN2Xu5JaE8qy22daY3lX0pjo8igkNZuEtwFbr2yKfXXgF3D1KMcv7LfrKONg3NSoPizziHqg7m1E2C+8ScyafcRrF366op4kOw2PNXE7eQSW5WVoIcBEvQXh\\\\\\\/Ye12cJgsnGAmHLdCfizJuMt\\\\\\\/Lt6vWhD2FF+iGxXQRHZv9vhNnt5L\\\\\\\/FwvmtzGiblfWhIV95N1d8S530RgF4W2RrM\\\\\\\/WEoGwg7pJYdgKl8rTHLKU0a2uscIdfY6mloa1GP+Z4WhBYlicNMf7x1kQM1tmz+xqlPbnKlh+m1QtP1WW2\\\\\\\/D+IND\\\\\\\/QgG\\\\\\\/Z2fy+5s7pcaI2ARSXyfkApMoFI3rIDUOGoC1OCddoY2jVLX5ZI1yYubDtHyczU7HkEo0RvHBRIu3usD9uxOCRHq1uO7urPvGQnbdVeFitMK4zKIYLm64Q1IyOqPrqWDA4lR7hrrlP\\\\\\\/RuJrRpcOBAS0gkksZkGTNr5zJCEcFXRb8LnVW792hWy8fqcG\\\\\\\/8lNnmul2AujA8jLMLhEQ3\\\\\\\/BhEqXs\\\\\\\/gEqxLUUbeI2OME9GmDHW4osIE+PFbmiyYy0qnRFxqbVjoKHsNBIBI60al2zwgI03DAUGIFHYbFVkJ3kqTieO2EZqZvQJYiCSYvrYlubqFin34I5TjXicUP0bAITRWbTxVfWlPKh1V73Kfn7KXegisagE3WZbNxet5EV\\\\\\\/WyKk4h\\\\\\\/5GznwesmqiLPR7arLgkygSoClPiq5xI21xscY1wLl2ephvcKemIvjPz8D57BnmYN5R1hAZCQ1531ZjxgaVx1kZKIjKoslHd6CYeNY5ZBl0kt5O59\\\\\\\/2C3t2mLGbstmkWk7IapPJ9jlyOhARXvsr9UbusB7z4DOGuR\\\\\\\/ifNoonMRcI38OvXoB\\\\\\\/WdFQaKWRZPUSy0b7Y0hlOhHLVE1fUdwiC8CzXkZGmU6epfvyOnAe4IsbAVrMPKkNgDt+gm3dL9q3ctWL1xg\\\\\\\/9EbzI2\\\\\\\/EXHvKRQSXMn81Df5rcZ\\\\\\\/alxo\\\\\\\/CLBQgSPkHnc0aJ0fxoQF0D2S1tuNFtD3TMNpHDW7mH\\\\\\\/821zTXykV+f\\\\\\\/oHx+CrH7MuWufXpjuetE9+RF7fvFehJJIKtxHPfZFOPlKAIDC8LBjz7RcX0+jRIu8eE5ptFkQBUXmu2RLr7R8SMGPebTZxQ+1kvNIXp7C0MzjKRQD3XlBHziYXCqadqQiqRgfVSTuxbF2PrdDM1gBL\\\\\\\/JGMSxfGcFrcnth+VMq8BzJEBLEJCt3DTUzMUI3t81b6YcT8A9mSuICvUqJdEOKVLBb3t2v9NP6TxYoaRXC0LkLTZr8pg5CRk5WBkilKwsyld1cDdBjw6gAD4qMJPzhbLf1OGEfFl8mSdgyF38MuFOJQcWGZAui4E2\\\\\\\/tQhQrIZI\\\\\\\/uyGBT\\\\\\\/NuVgeDSNOTR+dqBm8aWx8Dulvdqm2pUgW\\\\\\\/1U2eEMeA+TAox5Xi3yV3VQ4\\\\\\\/qub8zhMF1QuDVBBKocwyXHXaBdL+RijfM44YsEg6AY4pD+hWzbJCePJAd7jXAzPeKhSaAvb8ENBJNzj0WGU1w5TkB+WvF7oLaVkSs1hZY5KmEIPVzNU6irh+fqiiCmSl\\\\\\\/gRgF+4gU+PreTdG3FiFZNkNDZu56xHKN3vHU\\\\\\\/Gnt6H9rToYpxkIpa3W3osn\\\\\\\/d6zbDCBckvc+hyj\\\\\\\/Z1N3urKYWcmBimK3zN\\\\\\\/jbhv3oI9rk9MtRRf5gf6q6cQX7t17AvkQh+aqr1C7n+8MAoY6tbNQkI4P+h8ELI5TcQSOKu4jdmT\\\\\\\/xQJi4HQnF8qqbLj8GLibRao1V+7iOAW6eHDhlI9sl9zoTffpiPmJrI\\\\\\\/MxC0nRFgZ9UB1frUoSyhrfFWTQvhMIL7O8BQgb2393IMrKwrMXYGjeoEyV2uWZrJMRz7qboBQbgQ6pjD\\\\\\\/pa9psDVW2MuAf+mBBozniJiM3lx\\\\\\\/zufjFO0e5FojqRIYDaWVSRPmk1TMCaSqbN2pgcDsu0Oq\\\\\\\/uBEGDaore2O\\\\\\\/ayEhGdbHpfQj9YaTmSrAoibD1KCxZZq6e2tvYhyTTOsxHH7dKfrJ7APgmlOujHKp\\\\\\\/OX+sGvpQPhWCd5tuewa1vVJhVC+sWBQwRQS4EOn9GeF1PilR1aEwgkCV0TW+oZui27JNa2HOYcf3mDf7qTbuemkNyH21qcDumNZdXe90c7kXMrpfNGPx+m+dcpsXee6XRxGNe8RLlE5BwrwvOjXxSkFlKI+nAEXAopdTs6WCIbj1OOzzlhhvl3zHFSCRHqyXJj5Fesz9+geO3We2B0JyH\\\\\\\/GsbqVZzk5vbuOoOZrgsJ0Q\\\\\\\/IT0dxcltiFaWu16rvXreJCyLCIitV3wpJyE3Ws0YbDzKgHdXFAH7UcyXLaGzEnhG2Z\\\\\\\/4jsioMKlBGOQabD4M5uaYJTN5BZZ3sJAow1iirPjYQhC4bUsfSndoqbdSckc\\\\\\\/Pn452O0JHHXXldJlC\\\\\\\/OETrkjcKikAn\\\\\\\/lNjcLOy3MsFpoccyVxZhZSiM0mUhL59M6iK1fTFXoKvKvNDSZdfBALHFy+HXzVXdRiuSLGl8WilVJ2gc0U5Tubsq1dvNFNVY\\\\\\\/eOq3ve+19sWetwNhy9HkaOncg2U8oFjeqhPUZ6oy6904yA3qNjj\\\\\\\/gzguqZ1QNawnsIAI4fowDfxLBHZ2YBguCp0BAFEuIMmRauoOnHP1UhtXT4mfrombHnEHtzHO3x+Ji\\\\\\\/dxG+IZ2sbbX0EtU+4os6OI+yf8Uq5Dzu0AQjbFm\\\\\\\/OYpgrKbFaQcZxYLWEiSxjsL4FADFTvprGfRK2K8z8eV+OAiJ28z2M6hKOiuirdDg0NxnmsQ1hyAQCPQlaL2SZnw7Ti3ylL04ioEIu4TnCdLWSqDbm8CP3JIYA2OW3jZhVGaqMXv5d9oAf803ige7EZolf8y0t75uKIiuhmdnAt15F4qd69nswI22iAO3\\\\\\\/sPFeycKCRuZO+7clTJvO\\\\\\\/UMRqqPAqi+bgbMefW92PQc5ZzsOOgcD8oCIz4CdYim\\\\\\\/8W7ptAvqkItgvrw5Pcr6zTDfv1BrPRcT7OOBskcgxu+0ivH1JZ38fLoCECjhybyIu7UF6paRrNX2e7VYF0oWmjZ0XjEoGis26zw\\\\\\\/rCX2D7RLwwTWX3UtvLAzzcGirlK9beQKC6JT8q6Wu+osdi19drGybEX\\\\\\\/xJcqoT7o+VzL9cq4m\\\\\\\/LaX05CjNi1w7t+Xs3E\\\\\\\/bMxxXeme8YmEB7poh7Cal3CqeeVwoxUjhoJrQcQXCHAJkFwowxnPhxv\\\\\\\/rqpGq4IVtgZ80prBrmfE4qC2n8cx5sMVUsrJcou3+vs9AJ4yTTMbAzRrcL5SgpwcGOVebOVtfvF3YZjsOQR3\\\\\\\/tCHaKctx57usYWO6H6Mg8qQO4qkGhJkQoBvy6U\\\\\\\/aimxQ1LrrWC1\\\\\\\/d1CfFPWUUoX8p5wOBExtC+26UGxIoXXpmvIWsoO6a8oX9T8pNEEqKL6bKUhZV0k+CGwubMMw0YtSdRkTNoUAjM1GKFKW2uztLHDXsJ8zyLbcMMZWrr7yurFtZ93FfsjxCkw89udxUP4iDsru5OYgJq03348vQGzAJWMXhMA0+GuXdlVkVAzvi2W+IulrSaOmAkWVSUhGTgmXcdvEzh1m3lOdoTZL3xvMoyecrAYW+V6hEhSZYadoT\\\\\\\/4XCrPcGGOididRS2Zo8SsXXYXP8a6ROWMaslGHNpsLv9xEkeFpIe9yAX4Lq2U6ZvYz4foWCEVhMru+6xI9B6CvEOYDWLjGNLOh43Gyp\\\\\\\/3uR5d2uht20OEUBomFL7E7tf\\\\\\\/nup3iJpKKgiLFXj\\\\\\\/p0P6dk6TWA8SwlB7DwLHOqZ4c31CDKAag94OKQ72GCfRMVx+CyGVMwyY1V\\\\\\\/Ju\\\\\\\/0w1744v8lazVXB1o5gaC1\\\\\\\/oaBgob36Of\\\\\\\/g7c3cLcPn1\\\\\\\/kCJ2ippmR1aAxQ9T+j5spywCR0vWUb14wxe5MDMgZMtVB4P4JlTeodrL6KdRGTgE+6XWMEKvXvzlSxpG9GFdaGkkl1iIK1AOy2qwbeT\\\\\\\/6Y2CyvScLD+fUZQfJ11eNYV5EkaJbRgT7a8KWsOa9mNqEECHlaDWX37ZmMZQayzYB1H1XO\\\\\\\/DVl2yqzmpZNxYigueEJQukMDkDOOpaRpfKM6BCZs6huAWWYzepx\\\\\\\/pSTEjNqx\\\\\\\/FSrZLyVQKJPYnSgvQ\\\\\\\/DgU6aHWdicPNSGiCUaB6IDNC2auLEHY4hdoibgSN\\\\\\\/qoQPcSzUyB\\\\\\\/A2\\\\\\\/ltaK4MI4WXxbcxa16Quup1o2qKkZIcGdvHpa4DqDsL\\\\\\\/27kxQD46G3D77O6mDuQuwdRiL\\\\\\\/n18d\\\\\\\/6xv7kVXHXx+rkbs+QwLrwO6CIy01GyJ3fvJLlv3m\\\\\\\/JVydM7X9VYDYBHt1+jttKHEfkBFZqFDFFEE3uUYeKTuB8XsxaiY6RAsY4E2IM3cnW2WcgRg8OZTZe5l3Q2FuIzUTelyy5V9I3vVWfj8DAT+CULbjzcPFwaSWImu+gw3nHd2n5TwaJUcXUIkyKixiL8WNI8wEHStBfRld2H59C4nl\\\\\\\/jo9Gi9YumcOjdNzkws\\\\\\\/jZQhfinNrSPcFE8cGeAydn6Hw3JfbmippRYVOwsOTGeflx+D8jItzn0ZheXR67hG3zrMv5IByGYTIrh\\\\\\\/Qk4a+cQ10W9jKwRYJLWkAkaKkWjwpruWk+NzpJ43SB0VFmZgrwcVjjpdNWB07NNqZ1cWrqH\\\\\\\/u8lXfrQqKi8KkB1V0ifpbLooD\\\\\\\/Xr179JiSjBCIwACVR3XkXdE3KrwMQblLTJ0o64L4TJUPPIDyZj2NwlLgtSY2X9NP2FhzdPIxYKob0b9pplS6pC\\\\\\\/ZpHLr11OeeSow8Ad1TvguzMKmHgMUEhZsao0CFuSaL3JxDC7iWPqWEqUhc7TScw89rrvNGyrL9cvPPKgp7mqCX4Oi56DtxdocALbJCMVVLtdyN9l0f9jatdAVTjRW675mT0Xtt+q2BGLACSUFMYp6JjW5zeSJO0jKa6un6qa5H5omTpyL6mDxsoPMT+yzWhmAKhmn5vAqF1l27vRTAWwpsiXS1+Q3nWhGK7JnuHHnH1XE06oAfp4wIvACHSOcBJJ4pjAuDZWGUhUYq0GmkjpQA1wCrKg62dr1V77tmzrW\\\\\\\/YnLzys92P7O0SOXi6Xa4esN5lwAiP0klMo2v6MTUy0cLfw2+v7ZULkJq18nJ6Qj+I6YItcOYOBl3SiO78ycfHeNYJTMkS5AK7eUMaHdPHRo2jhbajk00MrQFigNe5zLAW48Q5ajjwrMpJmwxYGzseAGv7GF3wYHpetXeyJey8NQ8B+iB61+TyyXbtxpwKjBQgrhcsZ08uHnQbeNky3RIhQXD08N\\\\\\\/4ZHCpU1r4VM2BIs3kM+GAqeSxk\\\\\\\/ZUdmRlyFtx0mzsW+1QJzjMqA1LZ0tMJjXAwlQVYjiiG2ctdizjWDaPGavt\\\\\\\/ls+c9oyQ6EhC9po+KhbReIoHapIuthz6WqGYEDwG4SOxV64DucegkzE8xJcldN6Xxh54n9DLYfDxwKOU2pPUziuf6G98PucLh9FKGYjvSjbTqhWT7gtG8oxOi13r8FzegpzRBsmGdQr3PhDRRmbra6g6j1Xkpwx+tHDgZ4UH\\\\\\\/pJSY\\\\\\\/vgNkeKFoL\\\\\\\/ib5tUCwTgKj+vJTbUOLzMqYT7cyfvBWlgxaLv0nQjbAB+VEKN6d3ldOUariXWKiJN5sPQ\\\\\\\/JR49ZJ4zPucy0BdBkpL7jFdI\\\\\\\/sklspUaVFzs5L9xOxu5ZEeu+UI7o1CsTmOHx2sjL1Dw6fAALOFM+wX7tyBdl\\\\\\\/6Z0Zkhnn88SLIF+MvBjmlQ\\\\\\\/8cTsGu2uagKvZLM2G2DnZ2vUeUjkS74vcpN2wWFlHW2CbNiuccJ53nWa5ceroXLM+FHXVi\\\\\\\/UqjLtrGp+\\\\\\\/Ht7q8MGrzesAwdHRgHoNbkjjFpL1bGZmh8hNq+T36jURgsMexztuOwJOxb5n3BRU7CRa7b5GXntoHiif+IUpIBsvuQP7WYkJlayGj57o8XmGdRiNzkrY0fp1pXyUTWIJoTvry8b3CC12lW3KySNXbUmjIRK1IDF0BDkFSyjzIef6nVRU25eblrLZ5\\\\\\\/W8j0nLlt8dqdE8xBSfFe4xENUP2CSPOizfa4vdLr2JF7v0C0cl2auqLlUBJkq\\\\\\\/VA52zx8Z8HFIY7SjxuZ87u\\\\\\\/PxQb2SXLERC+28U9ODn4MoQGOW2B9Pyuh\\\\\\\/KTQen5l4xg0XnrXKpRBh9+v7jBDDS8NBYuLtFMx2swriRTXVKAHHN2USb6kx3EGnwITJRf4xpY\\\\\\\/rBznjNMLTXROFowndilT8VT\\\\\\\/QCzRvIw8GwDnPkPKbfb3HX3VXEtkaKZ0v0M7As0c9WVxvn2tVSGuMwh1SVOFZBRCF9Z7Mrkh307k+4l\\\\\\\/0fkeIhYePgMSxdmfh3GmVD4N6NYAgUckVtiPZCWf6KS0s6iLYzImTexQ4bL9t7H3jIkc5An+\\\\\\\/6XnXe7DmgJSm5qN5UIrBCQbBGUApbAKJa389zBGjd8t2AlofKr59l5wLHnzRnUSDU2yoSBt1ZtgW1rJcGfbn8qQ\\\\\\\/4w0yYQ1ZtZN7Q1eFeyH0gUTdEngzYt\\\\\\\/OLhK88glmZfjZGYwE3CDf+Ws0bRjE4Pr4P3ZFsDKe9VsZttewKnjL4GbAmP5OfkppVjtIivJT9UoCJ3ThDvKjeVYfZ0RP\\\\\\\/Tg2htbh2xLyt\\\\\\\/+Kf3jm\\\\\\\/gZVOLg6thx7a3YBZt+5kskLqSSHyKowv8Tw0a7oTVqEaDRi76IH4Q0UNZ8cmaWv2AW3KQUXsccsADYtmDnAufMXU5jN0pYyy2pYQBi87DtQqCPb8e3XhUUlNV1H9WXishAWmvZPR\\\\\\\/9zwA1mQFrcFtTT9hZVHrOTvoteACbS7LswDLbzwYpWXNZkayOtqfoGZU3hCtSZMAya15Ke9nJPej5yLCc0KCV1tLNufKib7z1K+LXQVknk\\\\\\\/PFFTRYJEWDBGrd5URgt5DDWxEneObSyvgeqYYdp5D17rXUYUf1vuXCFbSsy2MiSqVXY78TguXNwoLzRFZ9JXEgw6e9Ti3zSLykijD2A8F7\\\\\\\/\\\\\\\/wFCZFl2jqLIt1MFYANWOEPQV\\\\\\\/thfQ0s7mU6omjYJU5RU58PWoHhmFUb5kYrXXuLtrRxaAMTpOFdiFFdXw8XZppj60zsC5A09O3IBYaSz1LrhTutlBM6m2kabWrV0WatRtOgcDmPWZHF3GZb0rwrf\\\\\\\/LrFRmN6ji6rTlNHFian2lOAtFW2M7IYoKzBKsb7aVydefMd\\\\\\\/9uW1ImakzvIgj2T\\\\\\\/ZriRvq0d3ZfjfTW5X35dRvug2UVaBU52kQzQ+Ndv3llTbbTRq9reIAZg\\\\\\\/2mw5y8O3LA8rowCf03QbOLsN4d2fWm+V4hFOlRJ05HiN1EoHMHrgvK\\\\\\\/3AmmPMUS1YKrUl1tKTqvxkSWMwNiGtaFNSH7Fh87qYBibIQOkn0BCKENHrH1EfeC4kPPY1AZe0ce5aAZdVxASngoL+lt\\\\\\\/Iw3378VVI1gweEnrJEVgFrPneICg7sH6Jbx9tiNVv9t8648Lg4prxRwEZyCqkj77XQBdHK0tKzUXy0BXweNktd00D4xc4w8nRjFEBvjhSSmFYyof1yfTwOwgeSwzaGmzGWMlbAWa4ozfuurMoi8sC0Fh75KXhLEnaIQ5Wp9DDYTzG6\\\\\\\/pOkQrbnGweg16CiGhj\\\\\\\/olQOClcylPV6j6+mBg0ihZZVEQYwR7emNoZH3xiyRb7J76ZM+6kI0aSOYSodAPmLEJpI68TTW2rwWMvs4niGEKk2vSxk11PBD2EMRunakv+g6X9\\\\\\\/ToQuMC8P8Lwze2Boc+vZx6ocxwQpdgIWDb6xAaX9ILhERkLBCTw5cI5hDirpiRDQjsp63P0HrOZ3VfWjELSdfVf\\\\\\\/HVkTjsshRXygaOMufdwCVYZWX3ZdQV5qgPrVuaCRB5\\\\\\\/jguAah6VPgy7JvDJYE8AE1uIEKrqYzK5xXXdhiJ9SBXqXhQj6UUBcbj60ZKg0kzh\\\\\\\/BOtOM6wS0iQwROH0HPMC1J1PQIMl5zUKa0Z1JTIbjx57NdEGox19hNC3acEKGHZLfctbeuDIBOb4gnSy\\\\\\\/KW2N7KoZw\\\\\\\/d1MLfZqf5ZoMgX5jDQb1FxdYjhXGodmgXHyxNcqQbbsvrYigEErLDRpkoMIdHyLHX59RiREVVu5qIOHFVMwTGrRagZIXs6\\\\\\\/X6qqlh7+Sw3RIWd0TFDbtf54c15U3Fw164Jba8AhqDd1V1+FeK0ngX8GNfQqOjhQ+BfsiiG\\\\\\\/Pc6eCA58XwvQC0CCQAkKUkQonIGfcnGDrPWZaHKSKnfHs7ockvOETdtGs21X9p30BzEybsFLV9bKxvMi92XfD+I+4PMK8glKOz4QS6ifm8HUpOin8oWdDsbZrzfe2C8qrCtwDJhLbDGtsWU3Gp\\\\\\\/O2BL\\\\\\\/ibe0Lkob77HJ99Ol0OAp2MEXxyGCHBtTR5J0wJpOGZF1DtYCJg\\\\\\\/p5VGzCETmQPJE+k9Nw0y8Ca03afz8QgokzRW4SXTft5dEeKtC2kX6WqVk37wSWbxARGo16oZGzOGY2JI79ZkOIglwGxXUGma4tz5yPnreFeMAEtSC1HtYpKgMn3LyABJ1VHoOLbM4OjMH9yC6EFMGxNErQmL0aX5D5TUMDCiwyh5xThvHUmTvAAbSAgquTjPi5GNCotxrf\\\\\\\/+vk\\\\\\\/oiM3hxMqvOd3xywXHpNfXuJRO8UltH4csoneU\\\\\\\/GNjX2CM2l7m3l\\\\\\\/YV7Bo7T\\\\\\\/3yWQhY4fWC4SnuVDa4nk8bCLkCxBgJdnjqK03QThYIwF1dRHQ9KEOoA0gNINzVt2Hu30P6w1L0dLK3IQHMe9nvmtBVMGp6Z1V4ueZeit4D1SmgNom2KigBNb4vDyMoh2abx04ufVTrPsmBrowCDgLhJZY6xAH9z\\\\\\\/SHTF1SYC4WLIZJC1JV2tJcdLD3RO6wn4+jROsqL0iP6E20N5HGRgnxscsnabBqv08cA1fJZfYL\\\\\\\/BBjHjwJHA1I0\\\\\\\/zMHn6Tj88Es6f9\\\\\\\/+yoJWz7kSRPkClak0WDHJCFbmql+lmIvoTCZGTnWAaWgBVMpDxXiPKX4aUofgiN59FMEmy\\\\\\\/T\\\\\\\/IsaUJ3YPZUkevCyDPGzDIRtwZXneNXsesJs2stUsBf7+5QJ5OoyVKqwJz\\\\\\\/UMSuWJySJmfA7wTqMtD7+kUXe2nZmM3v9\\\\\\\/iEeLCx+z5XReTnKPvpHHpgoUZrL1xz\\\\\\\/HwNJCORLX3dr9xbIAdnwTWB0ebZeJht+2VitQelS+6uPsE0B\\\\\\\/g8CQxB1Fn\\\\\\\/lCK+eLZpUDva+H\\\\\\\/cJIXCRXXNu534m5GDUTWxpiuBSalMHMfYKk+8JCcDZs4WXCckP03SKRYM0haPe0QozO2IZEvrTLqaf6\\\\\\\/2V4QT9Se0bdQ12g44jLxqsX4DeMXRv2ngVJr2nt+B5T94eza\\\\\\\/LH95+XYdfZt\\\\\\\/eMBp5YSb3ti\\\\\\\/1o7JA+5hzpPQCJJOE3gte48ZODRSDhD9OgbQXpdNnE7v\\\\\\\/rkhPrfbpm8wS8d4sTdoAn3gsfdxY8lrcyJjfGvXqAEbneZRurV5yaMQhHuDq0s9qQbmmfMV60Tgb4TtZmJJAo0t14IfhhL7yRrOv4EUyIfmD24jFxBMyad6RjDtqEzxLbyxRoB8G7SyGgC3zAU1\\\\\\\/0jRxylOhJHFApSu7v4jHtBVHIlmOZJcRBoVMWkw2zPc8xTXObUsh3iDzEGikyUCFFDtZYAcDRBu\\\\\\\/Ttic5Zo25FQwOQ9QgpjhvxFLV0AOisaLxCsh4xgcOxyRhg\\\\\\\/7wRqT\\\\\\\/ahNMegbksFicsEPn422W+dgK2+DzxIYbPYl1ZAUC1awW5nNY0woiTqWb80tiega1MvNrkgA2wWm\\\\\\\/H+pNZPRDpfAdtz\\\\\\\/tV3J2WIyMkimVbuugDy\\\\\\\/a1ZYg\\\\\\\/Yw+jj05LpOiuzjybiPy3ZBFHy29fsM0ZarWM3C4oRLkb9kJS7rIbZGvmg5rC0onbua97NpmBZlGf9h5xeRCDk1RTui3m7\\\\\\\/2q\\\\\\\/8piMAV30NFuQQopo4KQSw3x3DFfjj7zrA6getCodw\\\\\\\/O6HYydgNlXQstfc140Ghpo0ycZQ+q+cHv8AZs2up+lYyGxUxYK\\\\\\\/hF\\\\\\\/7Pbk\\\\\\\/reyJrtIr1\\\\\\\/nxt6IUUkNAUALU537DnXqIFfqMEw5mHuNi15TLfrtxqGRz83mh7FFJup9pU1eHbdyIGHdmefkc6w7fEkDZvUoCC69vJXbA6ML\\\\\\\/KKl3M658SQgvU5koLWoOWv+EGo7X2TakJivw21OnBb7R0vQ64xFhwsMw5T0F0mFku+T96sTMJTmQqSrkAcvrJZOpyF40tg4xFutcwfqtlS80R4z3ivd9cvErgQwqxEpp9xqE3qA8ww0mW\\\\\\\/dfeEHd8dAfwXYh+sPcwC9un7LXjTEJSVvjAvFB6BUhvpM6hiDi2694SPryyrD7g0a9dLJWNK3TXTkK68gUcwQushojFCVnI8cK9Lsq+iJb4+BPYrZq7yNuzeCV441UhPPc9996IKaZHMHBv9WPFjQLWeLBsurUpUdOkJ+ubzp2e48mCyTDW0RfLhbKC2K3SwXmYwzcj65D3lwo8vCteEMLILWnmzlQDH+a+0fcmvLLwj67FChLngqXOaQPFGj7kK4gCsugO0XK2aXFzS89yKRZrDcZhgHajDO++VyR\\\\\\\/qkp3doHwoHoGtyKygJgCxPKI1oHpOUBemuHD9clHhG35kAe8AfUiqWNAvxR4GG+Z+XXTUWXRPAPSA56rFxsAJOj2sv53u95NSKWHzfg4dEw+7bmscCLGy4eoCwH7NjXEc06\\\\\\\/LLWfjKkyLjEj\\\\\\\/ct7eeeVVOjoXR4D3Tf+POYCaTmL+hXW+86c9XniIdtmQlxsVauMJgx0LqHMci87Bwx9DYcSdKlGLtzalk\\\\\\\/QC6dk09jxGWBAYGaCi\\\\\\\/49huYuOFTX6wIf2FCRcj0lnZbru2P3tsHLwTTk796jyIrUkkX1p5OIdZnTjGsrhoq87Uu9E3BJOUATYDZGe1P8GSeTdMKTpDfVEq4+yZmEw9JoK2zVPGJ2WTtlv5sSAzlXEe6pA64vHMKf451JKowjxIBore8glpXqj\\\\\\\/rDop7uFh\\\\\\\/XPO74v3qm6AVDfrYFQKNTBYoMmT+HIlQiEp88xagIqHWEgKjQUcPTJHP1I0IdG0xOVZhfFd6WJtwsoSA2XkfunmKbywb+veE0e9Bo8O98XRTwMJz4LJRL4KkSNxLAh0toDpMRkVwcTDyjv\\\\\\\/1oaWYy598KANZ5j50Ns05idrmACk8H+\\\\\\\/ZpdOMUrfKCsGf9Grn4e9IwHA15JMGV6Uk8MEsyFGkS2M\\\\\\\/GY75K\\\\\\\/Pm0kOcMFxolTQYjosu4ckK4wsaAFLG+hYc+xHq2MQ0dq0K7uiXsUyCU7fvh1pg2Wfl8ERE5SSYYmci373VLXcZXIFbihUFGdht4SleTknxZCLYlPa7Ohq\\\\\\\/O1U5wg\\\\\\\/Y3lK2Vf+VAkaS8YTr3uqxVhowdWnADc3HrwzGmuLL4RPSqU6b9o88t2CYkf3x+3EJqFhoYXhIwST0z8M1ABgmq\\\\\\\/ouvbOJmkRjQsc+v5hgFJctRXpUeGY7dPMZsDpBO45SVjK\\\\\\\/UTJxlHjcjVjLLzlKqhZhZfo60C8oP4ZsB1dZlOXKNcsPYwXEkLpmBHyOzKVgRcPKLc9lW0gj05zmuLdfJUUUHT9hlb9lXtw1R0dVqMEJkiWxgIGkL0+hptpKyRDIeBnvaKYSZtENo0ZJOYkioEiouYOGqXDu4SetOCDtSVr2sxDItYv6gZxc+2Js+4KHbTAKK3aAuTfSrAL1B9ZOav0cik7XdpaS9p2lnNf\\\\\\\/exnGQxusmdLdf93euSD69z0YLWOQ6hIOHcVLgvlG++xNX1ioxR7+rjt+UM8zK2DcyIZJeIfyxs8LYxvoMfvTSzuTy5ObpiZwDt8A1ygdghrpQARfvJEX1dWJz36Kr98ziwQV42E60iLwo+2brUoOAjFwS18t7kbQIeL6vXp0uR+c03ZAci1xG4PpDw+u1J+MjgHhhpThCyhJmWXfpj1eBtxuVl4RvIb07pun0M7jQ8YlrCQFWUP+Emf\\\\\\\/AWW1IbiXIlHhD3l2EqKo3nAUwkNf+pjQzMczdkPfHDwJlHazN2x4r+yTuoiEHMdyxe7cVYlmHYoz1N7PaEszH0T6RSrOIbZzRj5NpIu3cl34DTYrNM\\\\\\\/wTFgDRtqPr9TDO7Wcu3TbPw5Uq4HyAEAmqgI3iL6Q83HlpJqw9qfiDRLdJ\\\\\\\/tV9C88jF0hgLiOIZFToYXeNDak2tZrnbeBQYNNhKgxQycktQK+RxxhzvodALMC1jWYst8K+JHo9krGsIjvTNwlRxOJ1x9sOsocuwjXLCJHlL3+c66\\\\\\\/v48CaxZ4IROz262+ihE+P4rOc\\\\\\\/ICQ2JYMJV8Ct6pj\\\\\\\/snIeaN6jSaajK6lucjjoPfS09HwCAd6M9+LH9JPHHAvq5E3slHV+uIszisy2N+suUVIGmRtQk031Tq5gG+sUyGOxwFefuiy60Kb++kQ6IVPwYuoGY84ZfOOgWoiWBzrVMjCQs+\\\\\\\/9kgEKIsK3XTu9rV4PGNKBNztHOKnBkXwVNLkstSMtgg1how9bYN+3N554ZUrLgR+UAYzyYUKM8PvJjVhhyo14ikNhdiJR7PIXyqT9aoYVZ1jYs44azW1ndmRRfC8CCFb+5n35C2aBz1F8qWEl87ca7T7\\\\\\\/shZzdhMkEoSeEdTpXmy82eoYjGTQJpQ3XKniXAc\\\\\\\/lmOdYa8YXPgTgnBX0ejWhQ8RS\\\\\\\/z34gx63byrlEwB07cpMzV1kRtlPW8Q7eoLKRYz7D7Zg+3GAKq1\\\\\\\/0EB5xNZwsN+utv\\\\\\\/lU2FQtERQko5UkoUtDntXhwQMAbvA6Ft6351YEIyvgd3mSvQT3sJrkatCnCyjzhBhcI+zoe93\\\\\\\/cN8BF3QXfn4GeWwj+TYNaHKa2qi7hL1V8Cmhp+7cWjfU+SIBlPq4FevR22MIu6CPITf0BGR8M7UuTz8ODEqZkdfsomWhHlKeQSrBH1GGYt0xK2TsRly7gS2HEBsmCAYDjOYDdiFCokmbGJpO3tOy8OFLNnNt+OsDuUFYWhuZkuY4hLuaeSXOv6oMWII5T7iX4xchxh4u4oT1EhzxwiKH0TnV3s8e7O+U1y9WqWaQCCvmHE9f\\\\\\\/UltYH9ZYxEgvMQ7TCkE8Kq5RXYF5mqtEFmC3yEn+PKA7DgWn8bIXsf2NqH1RzWmI6OXJM0cayX109q\\\\\\\/+7Y0Y7RFtluDxNRYwVzlmfmKSLgQA83jnEVHcR7M1+2BiQJbKDF1EQ2CdOdxdopQRmWCHqIy9NgIO8M4F+iUZ6bpRKa9Y\\\\\\\/KtCkKk0muHGefBuWLdgfllOhZLEhx1HQWQ206IhS2MvwDCMtUt9X076YoqEt8JfVQeSg8VSzPA8wiHekaiveh9WXK6bOtnYaR32ZuFcERsZ0Uygk45tb5LDRI0tzp9klGk\\\\\\\/2lwboy\\\\\\\/AXLBI0+tZw7qW6A+uQnAFn\\\\\\\/OMngFXmyegZebU33xLWuapgDjYHtvzs7AeR0OcGz+722vlEmqor0ijovEtQjAmr5BLpSOSzBempqZBpIqeQH4h9NUq+CBPrHKS0MyjrLGI15fnIP459MVb2TGS+cdV522Kx7mb\\\\\\\/sHx2VMyxGgUMJ+GTxvE44RRDpOpfTvLa7an\\\\\\\/Kbu1shThgpb2QVWrtxuZBlJ7Ot7oK+8sFVB6lq+W5jZGOCluoKra7I\\\\\\\/1U4E5F5jrs8cjzuEpW2ygvbdayPyQQ5Rb4YF1in1rFps9ni6CjWh6SNGLPEdzwFDSXnaZur7EaILmcAaOfYuviNh4E+smNr+jw6mxLFZkM\\\\\\\/shrKvOl3WpBPf87tH4UxWQvWWHtnCHsOyP3yhnY8FUvf7hOZ1Y3FhSnllZkNVUWUPVU0iVOhso3ADBGp5pb4KypRR9J4NLBLqrit2iHqBR8euI5xBF0\\\\\\\/za4hrAOKXKX8i1hYVmaRIMocS2b43upVah1zZUVeV7rGWOtVH+LiX8oqugcjSKB703oqlQuBMuSnaelUHx3qxZYa7+kbtcEGsCm73k9dY9UVxkdsYPmNteq0CIxC\\\\\\\/lmsozdQbSPowcpgscNohGpMBoc8FPcMM9qvinxYgJl6oRQhJMa96Imwn0QKPuIdoe4uwiDmEafudZbpX5UBnxZ9Qe0rxTsAdRCkvdRzxxBmlw6uOQHJtdhQzXKwXLOs7Mq167EGbTTK+SdJy\\\\\\\/+FNGlndXH3fHgN1yXwKzZQBE3fN+PxCCLt+9cS2By48YRDtWYC96ChmpSAL3D8QnGnYsMZ0DjcD3Lozvc8IHIVP7kg\\\\\\\/CuRKC77lxrVSyQaBQ3x6hD2IVEGZcF\\\\\\\/sXkkiHP\\\\\\\/BIPdnyO7S2FSyUWBYPBrwH2vEBj\\\\\\\/wL+NVxoMP4beljCoPOl79LK40MLr8BxkVcqDwpgjNa1ZNhrj\\\\\\\/JyCLRG1+lhaii5muAZhZlOxoMQUiittGaBWDGioym6FaG+U82l9+r3OQpquRZZBOoqhVXoYMGKs2Pk\\\\\\\/LJt5n16qhpF+0eX4h2T+OxwEnZky3Uho45Q5zomsQtJQ4kD48hlSFX3tIAQ4xjWTygeTvyEnyO1cd6x2bWSZRG8nv7cWLJXNH9zcepz2MQtzZVvgtRJS5UgTf41tkPRrpcDaSvC+3cbBc5z23tFk71CQ8mdUKNPKb+ZpYEPTg2wM2SJCfi2CAkDuSn\\\\\\\/74PTa5crMh8vfap\\\\\\\/ThZgBym4BfXs1pR8Rt1qmsZgMykQZka4TuD8vQU5wr4JyhqGswOZRAZRgJLqSwsGjrTBNdsS8eojw9YCpvafVPo7mKD4eIESt+02Tk49oVTCmCA2xLnHLoBLRxMb4SaSsg1rZdPyTtHY3Ctlb7cQDoJUKMYRPq4edgkMk+kfZaK+8qKW+mlbtUvIEImipU4JTix4xHIOJ1Zl3ED62+GkzYRmRh5sDTZp\\\\\\\/XxHXgW5z8aWxrL\\\\\\\/NNR\\\\\\\/MyyxgjfufOGFtyCyBmC1Yp63z68\\\\\\\/IXbVars\\\\\\\/vJSV1refXrWh6gePWf3UuZ\\\\\\\/hMYdUh+aAeMNHhf2IAuRfcp9RKoCDe6+6gaLdvqLAa8yjriMvVuDxWxp2LTJXp6dFY9sJ4g2nw7xSFQ47\\\\\\\/1FIO9hadWFVrMyKPJzhMlEA4VD9nSBpuojpPx4PdxeaPecJMLtGj\\\\\\\/Tuew6ecs+fiXkhGLuXonwyFfYQG1Iq4fjarJGWDqH5W2IUJvCUEWBVfvFS5nI0gORu3QLzk5Il2ij7agjkFGpTHP3Mvf4xzrJmxqY8TemSxini4kFWTtqp8BiHWxMRc6EGysAo7VBIuV6miuc3QZqUDIc8VLewjtJBnNIgY4vpdZlvx06qKsM52+zcBuurJWVn0PaVT392cw0V2zMe8SNlqAIaqw8YS0juER5HdeVwIwzCecvz6AjtcVpBaX3xzUvo5AD+ZJ7U5nV5oNp3dFslCO8tM7oxXCv\\\\\\\/HYXwpvtm9pBYOjm51LTP2GSV+P\\\\\\\/ZOqavn2LoHWYERgw3jSFRiOrjLMcvN9fNZBFNIp4nBzA5+EfAujPVMYbN0zwKZsTh1ahivwsb0gFA7UwFM2aOqB1E048KUTbuwPDa3d0YLlezK2mb9Dr34l6dUd4OpaOfFGeaVrULCJy5weknMYWHg2MBf03jfb3StN\\\\\\\/x28dogU4Cn+lZ+BfC9FLWgXPF3LKBQCaJYWd39OFbMWQUvI1\\\\\\\/LX4oLhMf+vsLSPC+BBmMCKdgBWmm7Yy29i798YdUYT3abeOML9shOFeT0O6zH8dbGfEvK+yxzTsX70Ih39EgNXDptHvBJLVJosW3EgXteWZmL2K9EFXaqgSzqXu8D36c8rcc1R4\\\\\\\/NdsMAnWWXygWDhIgaQJ\\\\\\\/\\\\\\\/qfzGKZX7thqqNw9VGsCRmjvO1QvKP2QhG0sn+Tx9foArRYfHBOv7mpf7Nc3rnt1rexMK+B8PjPvGlvL4Hi9mMpgX5st0tncNa4BVxbezuN8SpqlwHYUrI+cQvvCNolpT07htQYL8gsHuJ6VNW9kWfAuLjWS8K\\\\\\\/1tiG1FjvdYznmWiwEJP4mOt17duBvfV0aatRdg6ZngR9iDBhpGBU9BiZN48lmdkac1JYRIhuVdWjWSivYM7jqFvYxMidSREKUQpL64YrMLqlIr4fwkFGuO7z1tGxddN4hQrhA2QlNDGeTaEihFeDso\\\\\\\/M+uF0W9ETq+TOEJ2AfCS8iZR1BbJb31Yxh6pZiKWE7XoFTMBfJ2m\\\\\\\/ZBGbk27QgadVrfJacmt4KgNtLhYp65PveI1s9Ehg8jg5Io648ZMOuHqoCezEpFxlLM39iFTMnlEhArYmFbpUzWooqjBrmPSrpskFX1uMeCSBw7GlEOOtUZgaV0dWdD\\\\\\\/BJWAEVnLEtahHgyIp4Om3mHDKAIMPqP7ykdl8CcUQv3a\\\\\\\/CTdCNIbmkWgki9PJb3iKt7Ji1g\\\\\\\/gHnZhi\\\\\\\/tnalZMezdt8qeUAQDCH39NvnvfSF06MaKEVFE1DAj5L35FjB0omwC5\\\\\\\/RsF7Lqy7UpeFYm7NDRiiuVWaNsbzJjCDAifHhrY93TNVcSqPae87dAM5Oef0zvtEYQ0nCV+s8gtO+R2HzsC4+VgN\\\\\\\/XLeIJ5pgQ9tcveoFBES9KQovIBokeiftJijmfPzGG9PCNETFg0xLDYqdNILvJ4+quwDPh1u+Ld8beqgSYUeO7AQks7RKI6rljQU\\\\\\\/6kc7pnfHe4m1YL55\\\\\\\/aQAPlS+FrrFc9Bk25VW2ITOLa7sqi0UUMWFnvL0El\\\\\\\/04JJdrjR7k3a\\\\\\\/O8ac7QSSFZsFdSmfzEzAgD9MKx8CXb2I21H8AOACNIvEq6fVr2Xk\\\\\\\/FxLItU97GHpbHspGhx4v\\\\\\\/qbJQv3dmt4W7Et7DBElXRwwWJV8Ml\\\\\\\/Ut+4\\\\\\\/tr5qPAemd1vUaTMQqnAEAE5k1SF9SyP+MjWUXGXOK0MTeNbCjnaTK7r3JH5ASsdZhOVL+y2DpQNqhfK4ZVUOo+9BMDKZo5Xyhk3ujAE\\\\\\\/OIUA0yxOdtmUOikb8pqJZII+eUjD0+oCo1R2xOMvodY7UjcnWHCWn6tQBZHgodC2npRfTCV5IRYJx83YtIa7fMo49654V0vZl\\\\\\\/5JZ3YZHfU6hWw7QY5Clkxr2ZbLjiK0Xk9BE7Zli5+RLs+pNjzo7l\\\\\\\/pLLdM\\\\\\\/W7nMbEq\\\\\\\/M6tRpLnRjGVHIzG0um1uaHoq9Z\\\\\\\/cqGwa1OSfYCo3SpzhBML0qXJgOyH2mD1uMNVIqOTw8gJKRv7s0kecPGa0RSkp3jEk6CWASxibkHOCjFvnuOGZOviBTedf2M6O1cqZEp6giPmsF4YD6WHeDyNa1S1taTsrDnSgJdSP\\\\\\\/Vp2EXbfAMVsIkG7LNSRLVzQj7NwwfJ1+h2JuujmBisFYt1H4NxoxxRsj8YALXxfIu\\\\\\\/BdHhk2nF4pZozz5U0zVp83MaqAWV7BNQIaclULUZVOMiQA\\\\\\\/1ScKZKMFUpM+rsH9fGUzVOjC\\\\\\\/y\\\\\\\/+acDoBNy8O9IrIcuOERm8tS2YUabzi6cXBtor2oEp5Jjhgsndq+hKx5YOC0Jj4Zcdt+XybLwxmuoXSa7TA2x0OBCJJ84eaK\\\\\\\/TNmWIjynI1DqcCKdTLqSbNgtl7rtrqFFA79hxoH6i2mxR5FNQUiTHEFQNAokKWOuzBQuuPwhrlwQi7HTccHV6o8fgR3BeeiOYiSi4zVis2GOX1ZQwXeRB+65qx2gqY0RWlieD8L91ydlsnfzgZbfQ9HUkYkw0IXYOhkEYUI2cX3xh\\\\\\\/CiSWlpDjR6o8fI8gV782pEyyXw\\\\\\\/JE0cQ+4FXbbcri+42mnHWNV80H9EqX40X5McV7tlhh5qdc38hsNSVzI9iFFP+XlxzNrIkTD4kee7zTCG97wsFaKSfBi75dfKQkDhsPwET0I2LfuBAE\\\\\\\/hnkKzzt4O2uOsFeOFXATAzYRI\\\\\\\/YEa3k\\\\\\\/ywujk+I\\\u0022,\\\u0022iv\\\u0022:\\\u0022Sps53yh4rxM8yIuCmsJs2Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228170c6bb254a5b7d9553fce55a451b8ca0fe66f9a394f9062a30c0fe2882bb79\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226wQLAtCFGNHfxnFos1tPB28qEKsEo5hZgpfgw+\\\\\\\/Q4OuZ9twCx356Cka3gCIbx9jS8mFFQFcS8YecOlGW4ibqJceB+3hJgoMAZqNPE1hBxjjAjYeCohn8JIfTUTXMFO3hWpTdOM1qSPmOWVWG60NHgRHqt5\\\\\\\/q+W6x+2SRhxqIkIiiXR9HMz5mG2rqZqsBc2JMzyVziVEbdJ9QqAlqh4\\\\\\\/X\\\\\\\/SqqVKMn\\\\\\\/sbSJjWOql+ckeIaqwSdzR1qHKeSwCPFzxgRyBUfAou+hRTFPqKA89tdj11G1RchdNhvV\\\\\\\/Triw+vZgshqQL13V7aKsvXaFkEuYsQQROdICp8YLum+A\\\\\\\/sDshth27Chb\\\\\\\/hQiZH8BHFWwNsOnC8RusrR0fWkGhGN8BrDgiV+SEyOiVmTAPXvBAnhAq+KMXeo0TwBd2t+2oUkTbd6nCn3WWQVS6jElg4hiZNi2HpymlhQWBCghaArRF+JFN3Fl4KipGsvhP\\\\\\\/oK3IAtI8IwKgM+PLMXjWDjdginJHwLfRTvJZyfcZYRz\\\\\\\/TmkUpDD+axv+HtpDevA\\\\\\\/2Xg2bcYnFK8gcGJQVJpIelF0C7Ex3XBOC5d3rCwRHojRCAqSisTkRbJxg5y5Etrt8ENsyIhBlYb6+2afkThSAGM6TU6KoGCHe3xSkbVYcvj3\\\\\\\/5BrjuhpZg\\\\\\\/WUh8zI8oNCFp20NLY\\\\\\\/HAIuINMS6a3sJ1MoSVdxCVnnpXKcdOe04Gkuy2IlDLltcapeggDI4d2pYIHLiX2DkVz6ccsdLO4CBzyjaXgJ4CoKtJqY\\\\\\\/cR7vSQQQ6+dsantFc4keKRmnUMQMivIhBNznqhSd4FBYwc4esl3Zvv+fx\\\\\\\/7FOg3PZ4d3JIOdEkH1Jlvhi1Ar4976F97lTx3wolG+t1ev43RxviDIiiSyrLhHVxbwODpT0O5YJ0OLaLw9nXVnagKB2fiHU4Yz0v14zc4QtiO\\\\\\\/pnjKU5BQRjaomRkCwkoMiUFhyz0IYqvyD3BI9lbsKXiyrk2GWOBGh1\\\\\\\/FmWOGdG3MxlumIIa38pNEDG4h\\\\\\\/YxQlvIbQvaseDoVaV6PAJB+D6gsNujNIoO7mE2gqXVc8bfeWEbPmT9Z5jR3JAFr0NhYiiKhC39o+2AvrMSmQH2aN7bxPxZoq612PuHF3GQ5gzS62xupLIPimq2i8JVw7rRVv4yaM\\\\\\\/VbND9sJi59l5V\\\\\\\/J\\\\\\\/qQKi3ZGFocHZHKoKO3J79W7w0uzEacGM9VrUHEOTpXrdqKxpYwFF2dXVvBRjuOZUTmO6jV8vHDOx0GmKRf2TIb0klYuROCmXvA6qdB1TKffhlH3xliFYEFo\\\\\\\/v5cyFWvVREdufoTf5abk5tc083uXROnwQAHCGYMGQOpmrM+7SKrOToR6DUIXHa5NJkYiQlPfst3na4LPjlXxaPcdFXOu0SIALsoPRrNjm2mCs1eWAxjIo6CqdFpHVMh3o1PYp06exZ1IfLmrS2x4TilZkPyLhNOFV82zeK1amYB6lkpWGy61XmU1fp2gHaHK8bn\\\\\\\/WaQ2Ahh3LLbGBVC3yXU7Az4No+jVXj7p8EI8XsRH0NDSAdVTRGQ9iirGsOlodzmho99CM1QBe7WFDcjjxYcEriA1DTdFr8d\\\\\\\/5inEz63ARo7ITVpvc0HVHIyJDGkI5iS3xatAMCQz1Pl2P80EGeMY\\\\\\\/edXtqAg+3kagGT8ZYV6jTqCDLwWsmiKNPG5mUA\\\\\\\/4HLgtK6Ko8weATKB0XQVjTRK75YTHvUgvICXxdbNV\\\\\\\/qZzI4s47huPcPXgs8Muh4u6P0ilpyaTKQhLkTcHVwQClrEKofYXfAXfE9guXKdCKo3siMuEQ\\\\\\\/vSwnf7V+BLtYlzwFC5Wy9gKe5RSt4Id+SBU2xgPQ69aYclizbgP6iZNEzwaPzuMOqCDXfqDHJpJHBkd7WRSK7k0c68LI20VNBPX\\\\\\\/aFHcmwKBRZzsmhVY1sRMSWohO0AF\\\\\\\/m2lYdmDZEgDtI6GaQE2QIjMvxkK1vDJooV+qBoEG20WenG5xIIO+NAl0DllSLxRKi4Jz9RxGczfPMyCLlrxugcUgeWiq9yAc6xEEzQsUH96QkGxkOvPnfanJ\\\\\\\/I22\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZA2waqEcCwwE2V\\\\\\\/RllT3Cw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022267374812447d0f55fc998dd1b18623e597c411639ae1d0678a21d7ae029479e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460441 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2299" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ejr53n3ntk86_EQjpmn39gdJNTYExfLODoVYE8A038g" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2299" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-q7UnyEfU4KIg_rxjKWicF5nXLM-MN4gtY1MHyGVUcn4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2285" title="Diversifying Selection Between Pure-Breed and Free-Breeding Dogs Inferred from Genome-Wide SNP Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2309" title="Characterization of a Single Genomic Locus Encoding the Clustered Protocadherin Receptor Diversity in Xenopus tropicalis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460441" id="node460441" data-pisa="ggg;6/8/2299" data-pisa-master="ggg;g3.116.029082" data-apath="/ggg/6/8/2299.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Long Terminal Repeat Retrotransposon Content in Eight Diploid Sunflower Species Inferred from Next-Generation Sequence Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hannah M. Tetreault</span> and <span class="highwire-citation-author" data-delta="1">Mark C. Ungerer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2299-2308;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029082 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460441" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hannah M. Tetreault </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Division of Biology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hannah%2BM.%2BTetreault%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tetreault%20HM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHannah%2BM.%2BTetreault%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:htetre@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">htetre@ksu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark C. Ungerer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Division of Biology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BC.%2BUngerer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ungerer%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BC.%2BUngerer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/8/2299" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460441" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460441/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2299.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460441" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460441/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2299.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460441" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460441/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/8/2299.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460441" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460441/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/8/2299.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460441" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460441/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The most abundant transposable elements (TEs) in plant genomes are Class I long terminal repeat (LTR) retrotransposons represented by superfamilies <em>gypsy</em> and <em>copia</em>. Amplification of these superfamilies directly impacts genome structure and contributes to differential patterns of genome size evolution among plant lineages. Utilizing short-read Illumina data and sequence information from a panel of <em>Helianthus annuus</em> (sunflower) full-length <em>gypsy</em> and <em>copia</em> elements, we explore the contribution of these sequences to genome size variation among eight diploid <em>Helianthus</em> species and an outgroup taxon, <em>Phoebanthus tenuifolius</em>. We also explore transcriptional dynamics of these elements in both leaf and bud tissue via RT-PCR. We demonstrate that most LTR retrotransposon sublineages (<em>i.e.</em>, families) display patterns of similar genomic abundance across species. A small number of LTR retrotransposon sublineages exhibit lineage-specific amplification, particularly in the genomes of species with larger estimated nuclear DNA content. RT-PCR assays reveal that some LTR retrotransposon sublineages are transcriptionally active across all species and tissue types, whereas others display species-specific and tissue-specific expression. The species with the largest estimated genome size, <em>H. agrestis</em>, has experienced amplification of LTR retrotransposon sublineages, some of which have proliferated independently in other lineages in the <em>Helianthus</em> phylogeny.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genome-size-evolution" class="hw-term hw-article-keyword hw-article-keyword-genome-size-evolution" rel="nofollow">genome size evolution</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/helianthus" class="hw-term hw-article-keyword hw-article-keyword-helianthus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Helianthus</em></a></li><li class="kwd"><a href="/keyword/ltr-retrotransposons" class="hw-term hw-article-keyword hw-article-keyword-ltr-retrotransposons" rel="nofollow">LTR retrotransposons</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/helianthus-agrestis" class="hw-term hw-article-keyword hw-article-keyword-helianthus-agrestis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Helianthus agrestis</em></a></li><li class="kwd"><a href="/keyword/transposable-elements" class="hw-term hw-article-keyword hw-article-keyword-transposable-elements" rel="nofollow">transposable elements</a></li></ul><p id="p-3">Transposable elements (TEs) are DNA sequences capable of mobilizing within a host genome. Mobilization typically occurs either by physical excision-reinsertion events or through a process of replicative transposition whereby individual elements transcriptionally give rise to new copies that are reverse transcribed prior to insertion at new locations in the genome (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kumar and Bennetzen 1999</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Feschotte <em>et al.</em> 2002</a>). TEs that mobilize via replicative transposition (Class I TEs) are a major genomic component of many plant species because their mobilization involves sequence propagation and large-scale copy number increases. Because these events occur independently and at different rates and scales in the genomes of different plant lineages, even closely related species may diverge rapidly in TE content and genome size (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hawkins <em>et al.</em> 2006</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Tenaillon <em>et al.</em> 2011</a>).</p><p id="p-4">The most abundant Class I TEs in plants are long terminal repeat (LTR) retrotransposons and are subdivided into superfamilies <em>gypsy</em> and <em>copia</em> (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kumar and Bennetzen 1999</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Wicker <em>et al.</em> 2007</a>). Differential proliferation and abundance of families (or sublineages) within these superfamilies are known to directly impact genome size evolution (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Vitte and Panaud 2005</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Hawkins <em>et al.</em> 2006</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Piegu <em>et al.</em> 2006</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Charles <em>et al.</em> 2008</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Tenaillon <em>et al.</em> 2011</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Piednoel <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kelly <em>et al.</em> 2015</a>). Characterizing particular sublineages within superfamilies that undergo proliferation and determining patterns of proliferation events among related species can be a difficult task given sequence variation among sublineages and the difficulty of accurately estimating copy number abundance of elements within sublineages across taxa.</p><p id="p-5">Advances in next-generation sequencing (NGS) approaches have greatly facilitated efforts to generate and characterize whole-genome-level sequence data (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lam <em>et al.</em> 2012</a>) for model and nonmodel organisms alike (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kelly and Leitch 2011</a>). Major impediments of <em>de novo</em> genome assembly of NGS data exist, however, on account of short-read lengths generated by many NGS platforms and the difficulty of assembling reads derived from genomes with a high repetitive fraction (<em>e.g.</em>, with a large TE component). NGS data nonetheless have proved extremely informative for characterization of the genomic TE content both within species and across related taxa (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Tenaillon <em>et al.</em> 2011</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bonchev and Parisod 2013</a>), and several analysis methods have been developed for such characterizations, even under scenarios of low sequence coverage (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Macas <em>et al.</em> 2007</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kurtz <em>et al.</em> 2008</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Novak <em>et al.</em> 2010</a>).</p><p id="p-6">Wild sunflowers in the genus <em>Helianthus</em> provide an opportune system for studies of TE proliferation dynamics and associated genome evolution (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Giordani <em>et al.</em> 2014</a>). <em>Helianthus</em> includes ∼49 species native to North America that are collectively widespread throughout the United States, southern Canada, and northern Mexico (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Heiser <em>et al.</em> 1969</a>) and phylogenetic relationships are well resolved (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rieseberg 1991</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Schilling 1997</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Schilling <em>et al.</em> 1998</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Timme <em>et al.</em> 2007</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Stephens <em>et al.</em> 2015</a>). Genomic resources and tools are available for several <em>Helianthus</em> species (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kane <em>et al.</em> 2013</a>) and a genome sequencing effort is underway for the cultivated sunflower <em>H. annuus</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kane <em>et al.</em> 2011</a>). Multiple ploidy levels are found within the genus (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Kane <em>et al.</em> 2013</a>) with genome size varying considerably even among species of the same ploidy (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sims and Price 1985</a>). Genome structure and organization have been best characterized for the diploid species <em>H. annuus</em>. The genome of this species is highly repetitive, with LTR retrotransposons and their derivatives comprising &gt;70% of nuclear DNA (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Staton <em>et al.</em> 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gill <em>et al.</em> 2014</a>). Recent insertional activity of these sequences has been documented in <em>H. annuus</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Buti <em>et al.</em> 2011</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Staton <em>et al.</em> 2012</a>) as have patterns of tissue-specific expression (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gill <em>et al.</em> 2014</a>). Recent and even larger-scale proliferation of LTR retrotransposons has been documented for three diploid annual <em>Helianthus</em> species derived via ancient hybridization events (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Ungerer <em>et al.</em> 2006, 2009</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Staton <em>et al.</em> 2009</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kawakami <em>et al.</em> 2010</a>), with sublineages that proliferated in these species remaining active transcriptionally and expressed at higher levels when compared to the parental species from which the hybrid taxa are derived (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kawakami <em>et al.</em> 2011</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ungerer and Kawakami 2013</a>).</p><p id="p-7">In the current study we explore the contribution of LTR retrotransposons to genome size variation among eight diploid <em>Helianthus</em> species representing all four taxonomic sections based on current classification (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Schilling and Heiser 1981</a>) and an outgroup species, <em>Phoebanthus tenuifolius</em>. These eight <em>Helianthus</em> species represent much of the existing variation in diploid genome size, ranging nearly fourfold in estimated nuclear DNA content (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Sims and Price 1985</a>). We combine short-read NGS data with sequence information from a panel of <em>H. annuus</em> (common sunflower) full-length LTR retrotransposons in a <em>de novo</em> graph-based clustering approach that enables meaningful comparisons of LTR retrotransposon sublineage identity and abundances across species. We demonstrate that nuclear genome size is significantly correlated with repetitive DNA content in these species and that the species under investigation generally exhibit similar abundances of different LTR retrotransposon sublineages, suggestive of shared ancestry. We also note signatures of amplification for a small number of LTR retrotransposon sublineages in species with the largest genomes, thus identifying a contributing mechanism of genome size expansion in these species. Lastly, we highlight how graph-based clustering approaches are preferable to read-mapping-based approaches in interspecific comparative analyses of TE abundance.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials and DNA sequencing</h3><p id="p-8">Seeds of species utilized in this study were obtained from the United States Department of Agriculture (USDA) National Plant Germplasm System (<a href="http://www.ars-grin.gov/npgs/">http://www.ars-grin.gov/npgs/</a>) or collected from natural populations (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Seeds were germinated in the dark on moist filter paper in Petri dishes and 2- to 3-d-old seedlings transferred to 8-inch pots with a 2:1 mixture of Metro-mix 350: all-purpose sand. All plants were grown under a 16 hr:8 hr, light:dark cycle in the Kansas State University glasshouse facility. Watering was conducted daily or as needed and fertilization with a weak nutrient solution (N:P:K = 15:30:15) was applied weekly.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468783/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468783/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Study species, genome size estimates, and associated genomic data</span><div class="sb-div caption-clear"></div></div></div><p id="p-12">Young, fully-expanded leaves from one individual per species were collected for DNA extraction and subsequent whole-genome shotgun (WGS) sequencing. All harvested tissue was flash-frozen in liquid nitrogen and stored at −80° until needed. Genomic DNA was extracted using the DNeasy Plant Mini Kit (Qiagen, Valencia, CA). Three micrograms of total DNA per species were utilized for library preparation and WGS sequencing on an Illumina HiSeq2000 platform, generating 2 × 100 bp paired-end reads. Library preparation was performed following the Tru-Seq standard protocol (Illumina Inc., San Diego, CA) with a library insert size of 350 bp. Libraries were multiplex sequenced on a single lane. Library construction and sequencing were performed at the University of Missouri DNA Core Facility, Columbia, MO (<a href="http://biotech.missouri.edu/dnacore/">http://biotech.missouri.edu/dnacore/</a>). Sequence data were trimmed and filtered using Trimmomatic V0.30 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bolger <em>et al.</em> 2014</a>) according to the following criteria: (1) adapters and barcodes removed, (2) reads &lt;80 bases removed, (3) bases trimmed from read ends if quality &lt;30, and (4) read ends trimmed while mean quality &lt;25 in a 4 bp sliding window. Chloroplast reads were removed by mapping the filtered dataset to the <em>H. annuus</em> chloroplast genome (NC_007977.1) using BWA v0.7.6 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li and Durbin 2009</a>) with default parameters. Genomic coverage for each species was estimated using the equation Coverage = <em>LN</em>/<em>G</em> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lander and Waterman 1988</a>), where <em>L</em> is average read length, <em>N</em> is number of reads per species and <em>G</em> is genome length. Genome length for each species was calculated utilizing the haploid 1C value, derived from 2C data estimated by flow cytometry (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), and the equation 1 pg = 978 Mb (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dolezel <em>et al.</em> 2003</a>).</p></div><div id="sec-3" class="subsection"><h3>Genome size determination</h3><p id="p-13">Nuclear DNA content (2C genome size) was estimated using a Guava PCA-96 microcapillary flow cytometry system (Guava Technologies, Hayward, CA). Five biological replicates were evaluated per species with a minimum of 5000 events per sample. Sample preparation for flow cytometry followed that of <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kawakami <em>et al.</em> (2011)</a>. An external standard (<em>H. petiolaris</em>) was used along with the internal standard chicken erythrocyte nuclei (CEN; BioSure). Data were analyzed using CytoSoft V 2.5.4 (Guava Technologies, Haywood, CA).</p></div><div id="sec-4" class="subsection"><h3>Estimation of genomic repetitive fraction based on short-read sequence data</h3><p id="p-14">The genomic repetitive fraction of each species was determined separately using a graph-based clustering approach developed by <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Novak <em>et al.</em> (2010)</a> and implemented in RepeatExplorer (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Novak <em>et al.</em> 2013</a>) on the Galaxy Server (<a href="http://www.repeatexplorer.org/">http://www.repeatexplorer.org/</a>). Briefly, ∼3 M single end (R1) 100 bp reads were randomly sampled from each species (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1/TableS1.pdf">Table S1</a>) and clustered based on an all-by-all comparison of sequence similarity (≥90%) and overlap (≥55%). Individual clusters were identified and counted toward the genomic repetitive fraction if they contained ≥0.01% of the starting number of sampled sequences (<em>e.g.</em>, for 3 M sequences, minimum cluster size = 300 sequences). These parameter values represent default settings of RepeatExplorer. Datasets where fewer than 3M reads were sampled (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1/TableS1.pdf">Table S1</a>) were automatically reduced by RepeatExplorer based on an initial analysis of a randomly sampled subset of reads and assessment of genome repeat structure as described in the RepeatExplorer manual. To assess potential variation in repetitive fraction estimates for a given dataset, five separate graph-based clustering analysis runs (each analysis run ≈3 M randomly sampled reads) were conducted, with means ± SE presented in the <em>Results</em>.</p><p id="p-15">To assess the strength of association between genome size and repetitive fraction, Pearson product–moment correlation coefficients and phylogenetically adjusted correlation coefficients were performed in Program R (v3.2.2, R Foundation for Statistical Computing, Vienna, Austria). The phylogenetically adjusted correlations were performed using phylogenetic independent contrasts with the ‘APE’ package in R (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Paradis <em>et al.</em> 2004</a>), based on evolutionary relationships presented in <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Stephens <em>et al.</em> (2015)</a>. The phylogeny was truncated using the drop.tip function in APE to consist only of the species under investigation, with the exception of <em>H. anomalus</em>, which is of hybrid origin (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rieseberg 2006</a>) and thus not included in the phylogenetically adjusted analysis.</p></div><div id="sec-5" class="subsection"><h3>Clustering with full-length LTR retrotransposons from the H. annuus genome</h3><p id="p-16">To aid interpretation of repetitive sequence cluster identity and size across species as they pertain to LTR retrotransposons, graph-based clustering analysis runs were performed with a diverse reference panel of full-length <em>gypsy</em> and <em>copia</em> LTR retrotransposons derived from the <em>H. annuus</em> genome (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Buti <em>et al.</em> 2011</a>; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Staton <em>et al.</em> 2012</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1/FileS1.zip">File S1</a>). Individual elements were extracted and characterized from published BAC sequences utilizing the LTR<em>harvest</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ellinghaus <em>et al.</em> 2008</a>) LTR<em>digest</em> (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Steinbiss <em>et al.</em> 2009</a>) pipeline in <em>genometools</em> V 1.4.2. Of 110 full-length elements identified by these methods, 52 (40 <em>gypsy</em> + 12 <em>copia</em>) were identified as possessing an intact reverse transcriptase (<em>RT</em>) domain and thus retained for phylogenetic analysis based on their <em>RT</em> amino acid sequences (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1/FileS2.zip">File S2</a>). The majority of these full-length elements represent relatively ‘young’ copies, with insertion estimates within the last 2 million yr (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Buti <em>et al.</em> 2011</a>; <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Staton <em>et al.</em> 2012</a>). Multiple sequence alignment was performed with ClustalW and phylogenetic analysis was conducted using neighbor-joining (NJ) and maximum parsimony (MP) methods in Molecular Evolutionary Genetics Analysis 4.0.2 (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Tamura <em>et al.</em> 2007</a>). The reliability of tree topologies was estimated with bootstrap replication (1000 pseudoreplicates).</p><p id="p-17">Full-length elements subjected to phylogenetic analysis (<em>n</em> = 52) were subsequently converted to 100 bp kmers of sliding 85 bp overlap using a custom perl script to standardize sequence length with the Illumina-generated short-read dataset. By this method, 281 to 1073 kmers were generated per full-length element (35,488 kmers total). The ∼3 million Illumina reads per species were analyzed jointly with this collection of 100 bp kmers, which served as genomic ‘tracers’ enabling meaningful comparisons of the LTR retrotransposon content and abundance of different species’ genomes. The decision to use 85 bp overlap for adjacent 100 bp kmers for each full-length element was based on the fact that the resulting similarity (100% shared bases across overlap of 85%) exceeded considerably the RepeatExplorer parameters for sequence clustering (<em>i.e.</em>, ≥90% shared bases across overlap of ≥55%).</p></div><div id="sec-6" class="subsection"><h3>RT-PCR assays</h3><p id="p-18">LTR retrotransposon transcriptional activity was evaluated via RT-PCR in both vegetative (leaf) and reproductive (bud) tissues from a single individual per species. For each sampled plant, leaf tissue representing the eight-leaf stage and the first bud were harvested and immediately flash-frozen in liquid nitrogen. Total RNA was extracted using TRIzol (Invitrogen, Carlsbad, CA) and purified with an RNeasy Mini Kit (Qiagen, Valencia, CA). RNA was treated with RNase-Free DNase (Qiagen, Valencia, CA) to eliminate DNA contamination. Two sublineages of <em>gypsy</em> and a single sublineage of <em>copia</em> were assayed for transcriptional activity in both tissue types for all species utilizing sublineage-specific primers targeting the <em>Integrase</em> and <em>RNASEH</em> domains of <em>gypsy</em> and <em>copia</em> elements, respectively (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kawakami <em>et al.</em> 2010</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Ungerer and Kawakami 2013</a>). RT-PCR assays were conducted using the ImProm-II Reverse Transcriptase system (Promega, Madison, WI; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1/TableS2.pdf">Table S2</a>). RT-PCR reactions of the <em>actin</em> gene were used as positive controls for all samples. Negative control reactions were performed by withholding the reverse transcriptase enzyme. RT-PCR amplifications were conducted with an initial denaturing step of 94° for 2 min, followed by 5 cycles of 94° for 15 sec, 55° (+1.0°/cycle) for 15 sec, and 72° for 15 sec, followed by 30 cycles of 94° for 15 sec, 60° for 15 sec, and 72° for 15 sec, with a final incubation step of 72° for 5 min. Amplification products were size-separated via electrophoresis in 2% agarose gels and stained with ethidium bromide for visualization.</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-19">Raw sequence reads have been submitted to the NCBI Short Read Archive [SRP074507].</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Genome size and repetitive sequence content</h3><p id="p-20">Genome size estimates based on flow cytometry (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) are largely consistent with earlier reports for overlapping <em>Helianthus</em> species (<em>n</em> = 7) obtained by Feulgen-staining (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Sims and Price 1985</a>), with the exception of <em>H. divaricatus</em>, which was estimated at 2C = 9.41 pg (±0.08) in the current study <em>vs.</em> 16.9 pg reported previously (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Sims and Price 1985</a>). Intraspecific ploidy variation in <em>H. divaricatus</em> may underlie this observation (E. Baack, personal communication), though it is generally thought to be rare in <em>Helianthus</em> (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Kane <em>et al.</em> 2013</a>). Greater variability in 2C values exists among the sampled <em>Helianthus</em> annual species (range = 6.94–24.23 pg) <em>vs.</em> perennial species (range = 9.32–12.91 pg; <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). With the exception of <em>H. agrestis</em>, all <em>Helianthus</em> species display 2C values lower than observed for closely related <em>P. tenuifolius</em> (2C = 13.94pg ± 0.71), a diploid species and outgroup taxon for <em>Helianthus</em> (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Schilling <em>et al.</em> 1998</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Timme <em>et al.</em> 2007</a>; <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Stephens <em>et al.</em> 2015</a>).</p><p id="p-21">The Illumina Hi-Seq platform generated ∼6.8–16.9 M paired-end reads (2 × 100 bp), post processing, for each of the eight <em>Helianthus</em> species and <em>P. tenuifolius</em>, yielding genome coverage estimates ranging from 0.21× to 0.67× (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). Based on subsampling of ∼2.4–3 M single end reads per species, graph-based clustering yielded genomic repetitive fraction estimates between 68.17% and 82.12% (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) and these estimates are strongly correlated with estimates of genome size (phylogenetic independent contrast analysis: <em>r =</em> 0.9041, <em>P =</em> 0.0052; <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2299/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic relationships (A) and correlation between genome size and genomic repetitive fraction (B) for species under investigation. Phylogenetic tree is based on relationships presented in Stephens et al. (2015) and does not include H. anomalus, which is of hybrid origin (Rieseberg 2006). Genome size and genome repetitive fraction are significantly correlated: phylogenetic independent contrast analysis: r = 0.9041, P = 0.0052; unmodified analysis: r = 0.9121, P = 0.0006. Species abbreviations in (B) are as in Table 1. Red, annual; blue, perennial; teal, perennial outgroup. Values (&#xB1; SE) are provided in Table 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1585500298" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic relationships (A) and correlation between genome size and genomic repetitive fraction (B) for species under investigation. Phylogenetic tree is based on relationships presented in Stephens &lt;em&gt;et al.&lt;/em&gt; (2015) and does not include &lt;em&gt;H. anomalus&lt;/em&gt;, which is of hybrid origin (Rieseberg 2006). Genome size and genome repetitive fraction are significantly correlated: phylogenetic independent contrast analysis: &lt;em&gt;r =&lt;/em&gt; 0.9041, &lt;em&gt;P =&lt;/em&gt; 0.0052; unmodified analysis: &lt;em&gt;r =&lt;/em&gt; 0.9121, &lt;em&gt;P =&lt;/em&gt; 0.0006. Species abbreviations in (B) are as in Table 1. Red, annual; blue, perennial; teal, perennial outgroup. Values (&#xB1; SE) are provided in Table 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2299/F1.medium.gif" width="440" height="194"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/8/2299/F1.medium.gif" width="440" height="194"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2299/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2299/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468847" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">Phylogenetic relationships (A) and correlation between genome size and genomic repetitive fraction (B) for species under investigation. Phylogenetic tree is based on relationships presented in <a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Stephens <em>et al.</em> (2015)</a> and does not include <em>H. anomalus</em>, which is of hybrid origin (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Rieseberg 2006</a>). Genome size and genome repetitive fraction are significantly correlated: phylogenetic independent contrast analysis: <em>r =</em> 0.9041, <em>P =</em> 0.0052; unmodified analysis: <em>r =</em> 0.9121, <em>P =</em> 0.0006. Species abbreviations in (B) are as in <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>. Red, annual; blue, perennial; teal, perennial outgroup. Values (± SE) are provided in <a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>LTR retrotransposon contribution to genomic repetitive fraction</h3><p id="p-23">To evaluate the contribution of LTR retrotransposons to the repetitive fraction of these genomes, the short-read sequence data for each species were analyzed jointly with a library of 100 bp overlapping kmers derived from 40 full-length <em>gypsy</em> and 12 full-length <em>copia</em> elements identified previously from the <em>H. annuus</em> genome (see <em>Materials and</em> <em>Methods</em>). Phylogenetic analyses based on the <em>reverse transcriptase</em> (<em>RT</em>) domains of these 40 + 12 full-length elements indicate multiple well-supported <em>gypsy</em> and <em>copia</em> sublineages (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a>, respectively). Comparisons of these full-length element derived <em>RT</em> amino acid sequences across sublineages for both superfamilies revealed high sequence variability, with average genetic distances ranging from 0.108 to 0.667, and from 0.318 to 0.644 in pairwise comparisons of amino acid sequences from different sublineages within <em>gypsy</em> and <em>copia</em>, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1/TableS3.pdf">Table S3</a>). These elements are highly diverse, and represent a majority of the <em>gypsy</em> and <em>copia</em> diversity reported previously in sunflower based on sequence survey approaches (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ungerer <em>et al.</em> 2009</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Kawakami <em>et al.</em> 2010</a>) and analyses of multiple sequenced <em>H. annuus</em> BACs (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Buti <em>et al.</em> 2011</a>; <a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Staton <em>et al.</em> 2012</a>). Nomenclature for sublineage designations follows that reported in <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Ungerer <em>et al.</em> (2009)</a> and <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Kawakami <em>et al.</em> (2010)</a>. Identified sublineages based on phylogenetic analyses presented herein also are largely congruent with family classification described in <a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Staton <em>et al.</em> (2012)</a> (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2299/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Neighbor-joining trees depicting sublineages of gypsy (A) and copia (B) elements based on 129 and 239 amino acid residues of the reverse transcriptase (RT) domain, respectively. Numbers along branches indicate bootstrap support for Maximum Parsimony/Neighbor-joining analyses. Branch colors depict different LTR retrotransposon sublineages and correspond to designations used in Ungerer et al. (2009) and Kawakami et al. (2010). Symbols at branch tips correspond to sunflower LTR retrotransposon families identified as highly abundant in Helianthus in Staton and Burke (2015)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1585500298" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Neighbor-joining trees depicting sublineages of &lt;em&gt;gypsy&lt;/em&gt; (A) and &lt;em&gt;copia&lt;/em&gt; (B) elements based on 129 and 239 amino acid residues of the &lt;em&gt;reverse transcriptase&lt;/em&gt; (&lt;em&gt;RT&lt;/em&gt;) domain, respectively. Numbers along branches indicate bootstrap support for Maximum Parsimony/Neighbor-joining analyses. Branch colors depict different LTR retrotransposon sublineages and correspond to designations used in Ungerer &lt;em&gt;et al.&lt;/em&gt; (2009) and Kawakami &lt;em&gt;et al.&lt;/em&gt; (2010). Symbols at branch tips correspond to sunflower LTR retrotransposon families identified as highly abundant in &lt;em&gt;Helianthus&lt;/em&gt; in Staton and Burke (2015).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2299/F2.medium.gif" width="440" height="435"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/8/2299/F2.medium.gif" width="440" height="435"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2299/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2299/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468810" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Neighbor-joining trees depicting sublineages of <em>gypsy</em> (A) and <em>copia</em> (B) elements based on 129 and 239 amino acid residues of the <em>reverse transcriptase</em> (<em>RT</em>) domain, respectively. Numbers along branches indicate bootstrap support for Maximum Parsimony/Neighbor-joining analyses. Branch colors depict different LTR retrotransposon sublineages and correspond to designations used in <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Ungerer <em>et al.</em> (2009)</a> and <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Kawakami <em>et al.</em> (2010)</a>. Symbols at branch tips correspond to sunflower LTR retrotransposon families identified as highly abundant in <em>Helianthus</em> in <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Staton and Burke (2015)</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Clustering with these panels of modified full-length LTR retrotransposons allowed, for each species under investigation, assignment of short-read Illumina sequences to distinct <em>gypsy</em> and <em>copia</em> superfamilies and sublineages within these superfamilies (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and B</a>). Across species, sequences derived from <em>gypsy</em> elements were 3.8- to 5.3-fold more abundant than sequences derived from <em>copia</em> elements and together sequences derived from these two superfamilies combine for between 38.3% and 49.2% of all sequences for the species assayed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1/TableS1.pdf">Table S1</a>). Sequences from specific <em>gypsy</em> sublineages consistently are more abundant within species’ genomes than others (<em>e.g.</em>, sublineages A, B, C, X1, and X2 <em>vs.</em> sublineages E’, Y1, Y2, Z1, and Z2; <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>); these more abundant sublineages form a monophyletic group in phylogenetic analysis of <em>gypsy</em> sequences (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2299/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic abundance of different sublineages of gypsy (A) and copia (B) elements. Shown are means (&#xB1; SE) based on five graph-based clustering analysis runs for each dataset. Error bars for some histograms are too small to be seen at the resolution of this figure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1585500298" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic abundance of different sublineages of &lt;em&gt;gypsy&lt;/em&gt; (A) and &lt;em&gt;copia&lt;/em&gt; (B) elements. Shown are means (&#xB1; SE) based on five graph-based clustering analysis runs for each dataset. Error bars for some histograms are too small to be seen at the resolution of this figure.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2299/F3.medium.gif" width="410" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/8/2299/F3.medium.gif" width="410" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2299/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2299/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468802" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Genomic abundance of different sublineages of <em>gypsy</em> (A) and <em>copia</em> (B) elements. Shown are means (± SE) based on five graph-based clustering analysis runs for each dataset. Error bars for some histograms are too small to be seen at the resolution of this figure.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">For certain sublineages, elevated read densities were observed for some species, suggesting species-specific amplification events. For example, <em>H. agrestis</em>, the species with the largest estimated genome size and highest genomic repetitive fraction, displayed elevated read densities for two <em>gypsy</em> sublineages (A and C), indicating that proliferation of these sublineages may underlie genome expansion in this species. Similar elevated density of reads was observed for sublineage A in <em>H. anomalus</em> and sublineage X1 in <em>P. tenuifolius</em>.</p><p id="p-28">Analogous patterns were observed for sublineages of <em>copia</em> elements with respect to relative abundance, with sublineages 1, 3, and 7 contributing disproportionately more, and sublineages 2, 4, 5, and 6 disproportionately less, to the genome repetitive fraction across species. Unlike observations for <em>gypsy</em> sublineages, however, the more abundant <em>copia</em> lineages are not monophyletic but rather consist of three separate, well-supported lineages in the <em>copia</em> phylogeny (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). Elevated density was observed in <em>copia</em> sublineage 1 for <em>H. agrestis</em>, again suggestive of a role of this sublineage in genome expansion. Elevated density, though to a lesser degree, also was observed in <em>copia</em> sublineage 3 for <em>H. angustifolius</em>.</p></div><div id="sec-11" class="subsection"><h3>Transcriptional activity of LTR retrotransposons in leaf and bud tissue</h3><p id="p-29">Transcriptional activity of <em>gypsy</em> sublineages A and C and <em>copia</em> sublineage 1 (see <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2, A and B</a>) was assayed via RT-PCR in both leaf and bud tissues for all species under investigation. Detection of transcriptional activity was variable across species and tissue types for <em>gypsy</em> sublineage A (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>), with transcripts clearly detected in both leaf and bud tissue for all annual species but less detectable in perennial species; and with more detectable expression signal in bud <em>vs.</em> leaf tissue for perennials. In contrast, transcriptional activity of <em>copia</em> sublineage 1 was equally detectable across all species and in both tissue types (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Transcriptional activity of <em>gypsy</em> sublineage C was not detected in any tissue type in any species (data not shown). Positive control reactions targeting <em>actin</em> expression yielded no detectable expression differences across tissue types or species (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1/FigureS1.pdf">Figure S1</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/8/2299/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RT-PCR assays of gypsy sublineage A (A) and copia sublineage 1 (B) in leaf (L) and bud (B) tissue. Minus signs in parentheses indicate lanes with negative control reactions. Species abbreviations are as in Table 1. Red, annual; blue, perennial; teal, perennial outgroup." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1585500298" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;RT-PCR assays of &lt;em&gt;gypsy&lt;/em&gt; sublineage A (A) and &lt;em&gt;copia&lt;/em&gt; sublineage 1 (B) in leaf (L) and bud (B) tissue. Minus signs in parentheses indicate lanes with negative control reactions. Species abbreviations are as in Table 1. Red, annual; blue, perennial; teal, perennial outgroup.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/8/2299/F4.medium.gif" width="440" height="297"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/8/2299/F4.medium.gif" width="440" height="297"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/8/2299/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/8/2299/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469008" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">RT-PCR assays of <em>gypsy</em> sublineage A (A) and <em>copia</em> sublineage 1 (B) in leaf (L) and bud (B) tissue. Minus signs in parentheses indicate lanes with negative control reactions. Species abbreviations are as in <a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>. Red, annual; blue, perennial; teal, perennial outgroup.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-31">Nuclear genome size across angiosperms varies dramatically, stretching nearly 2,400-fold between the smallest and largest documented flowering plant genomes (Leitch and Leitch 2013). Differential abundance and proliferation of TEs is now recognized as a significant contributor to genome size variation in plants, with LTR retrotransposons recognized as the most abundant and transpositionally dynamic (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Hawkins <em>et al.</em> 2006</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Piegu <em>et al.</em> 2006</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">El Baidouri and Panaud 2013</a>). Evaluating TE diversity in organismal genomes has been revolutionized by NGS technologies that enable rapid and detailed analysis of TE composition both within and among species (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Macas <em>et al.</em> 2007</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Swaminathan <em>et al.</em> 2007</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Wicker <em>et al.</em> 2009</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Piednoel <em>et al.</em> 2012</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sveinsson <em>et al.</em> 2013</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Diez <em>et al.</em> 2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agren <em>et al.</em> 2015</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kelly <em>et al.</em> 2015</a>).</p><p id="p-32">Here we utilized Illumina short-read sequence data coupled with sequence information from a panel of full-length <em>gypsy</em> and <em>copia</em> LTR retrotransposons obtained from the <em>H. annuus</em> genome to explore the contribution of LTR retrotransposons to genome size variation among eight diploid <em>Helianthus</em> species representing all four taxonomic sections based on current classification schemes (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Schilling and Heiser 1981</a>) and an outgroup species, <em>P. tenuifolius</em>. The species under investigation consist of both annuals and perennials, vary in genome size by ∼fourfold, yet all are diploid with a haploid chromosome complement of <em>n</em> = 17. Given that other major classes of TEs such as DNA transposons and non-LTR retrotransposons (<em>e.g.</em>, LINEs) represent a very small fraction of the sunflower genome (∼2% and 0.6%, respectively) (<a id="xref-ref-49-7" class="xref-bibr" href="#ref-49">Staton <em>et al.</em> 2012</a>), these other TE categories were not included in the current analyses. In addition, cluster annotation in RepeatExplorer based on the RepeatMasker Viridplantae database indicates that other repeat types (<em>i.e.</em>, low complexity repeats, simple repeats, and satellite DNA) generally are rare (&lt;2% combined). This latter category of repeat types was thus also excluded from analysis.</p><div id="sec-13" class="subsection"><h3>Variation in genome size</h3><p id="p-33">With the exception of <em>H. agrestis</em>, all <em>Helianthus</em> species investigated in the current study possess genome size estimates lower than that for the outgroup species <em>P. tenuifolius</em>. It is currently unknown whether this pattern is attributable to a general history of genome size reduction of <em>Helianthus</em> lineages, genomic expansion in <em>P. tenuifolius</em>, a combination of the two forces, or an artifact of the species sampled. Genome size reduction (DNA loss) can result from processes such as illegitimate recombination and/or unequal intrastrand homologous recombination events at the site of LTRs or interior coding regions of LTR retrotransposons (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Devos <em>et al.</em> 2002</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Vitte and Panaud 2003</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ma <em>et al.</em> 2004</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hawkins <em>et al.</em> 2009</a>). Hallmarks of these events include the presence in the genome of truncated elements and solo LTRs. These hallmarks have not been thoroughly investigated in <em>Helianthus</em> species or comparatively in <em>P. tenuifolius</em> due to a lack of sufficient sequence data. Based on analyses of 21 BAC clone sequences of the common sunflower <em>H. annuus</em>, however, truncated elements and solo LTRs do not appear a common feature of the sunflower genome (<a id="xref-ref-49-8" class="xref-bibr" href="#ref-49">Staton <em>et al.</em> 2012</a>). Evidence for independent genome expansion in <em>P. tenuifolius</em> following divergence from <em>Helianthus</em> lineages is suggested by elevated read density for at least one <em>gypsy</em> sublineage described in the current study (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>, see also <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Staton and Burke 2015</a>). Despite these observations, broader trends across the Asteraceae suggest a directional increase in abundance of the more common <em>gypsy</em> LTR retrotransposons (and accordingly in genome size) from basal to more derived lineages, the latter of which include <em>Helianthus</em> and <em>Phoebanthus</em> species (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Staton and Burke 2015</a>). As such, <em>Helianthus</em> and <em>Phoebanthus</em> species’ genomes should be considered larger and with higher copy numbers of LTR retrotransposons compared with other members of Asteraceae, at least based on the limited sampling to date.</p></div><div id="sec-14" class="subsection"><h3>Clustering with panels of full-length LTR retrotransposons</h3><p id="p-34">A strong positive correlation was found between genome size and genome repetitive fraction, indicating an important role for repetitive DNA in underlying genome size variation in this group. Combining short-read data with sequence information from a panel of full-length LTR retrotransposons in a <em>de novo</em> graph-based clustering approach enabled meaningful comparisons of LTR retrotransposon presence and relative abundance across species. The majority of elements within this panel have estimated insertion times in the <em>H. annuus</em> genome within the last 2 million yr (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Buti <em>et al.</em> 2011</a>; <a id="xref-ref-49-9" class="xref-bibr" href="#ref-49">Staton <em>et al.</em> 2012</a>). As such, our analyses focus on LTR retrotransposons in <em>Helianthus</em> likely to have been active recently; more ancient elements potentially involved in older amplification events may be less well represented. Sequences derived from <em>gypsy</em> elements were observed to be 3.8- to 5.3-fold more common than sequences from <em>copia</em> elements for these species. These results are consistent with previous analyses of the <em>H. annuus</em> genome (<a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Buti <em>et al.</em> 2011</a>; <a id="xref-ref-49-10" class="xref-bibr" href="#ref-49">Staton <em>et al.</em> 2012</a>), and consistent with genomic composition analyses in other plant species where similar abundance biases have been observed (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">International Rice Genome Sequencing 2005</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ming <em>et al.</em> 2008</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Paterson <em>et al.</em> 2009</a>).</p><p id="p-35">Our results indicate variation in abundance for different sublineages of <em>gypsy</em> and <em>copia</em> elements within genomes, but general stability in read density within a sublineage across species. Stability in read density across species is expected if most LTR retrotransposon proliferation activity occurred in the common ancestor of these species, with sequence abundances remaining relatively unchanged following subsequent cladogenesis. Elements from the most abundant sublineages of <em>gypsy</em> (<em>i.e.</em>, sublineages A, B, C, X1, and X2; <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>) represent part of a larger, well-supported, monophyletic group (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>), and thus share a common evolutionary history. In contrast, <em>copia</em> sublineages with the highest read densities (<em>i.e.</em>, sublineages 1, 3, and 7) represent more distantly related and nonmonophyletic elements.</p><p id="p-36">While general stability in read density within most sublineages was observed across species, exceptions to this pattern were found, most notably for three <em>gypsy</em> sublineages (sublineages A, C, and X1) and a single <em>copia</em> sublineage (sublineage 1). These exceptions were marked by higher read densities for species with larger genome size estimates, and were most apparent for <em>H. agrestis</em> and <em>P. tenuifolius</em>. These patterns likely reflect recent and lineage-specific amplifications that have contributed to genome size expansion in these species. Similar patterns have been observed in other plant genera whereby differential abundance of a small number of LTR retrotransposon lineages underlies large genome size differences among species (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Hawkins <em>et al.</em> 2006</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Piegu <em>et al.</em> 2006</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">El Baidouri and Panaud 2013</a>). Interestingly, representative elements for two of the abundant <em>gypsy</em> sublineages (<em>i.e.</em>, RLG-iketas and RLG-wimu) and <em>copia</em> sublineage 1 (RLC-amov, RLC-jiliwu) (see <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2, A and B</a>, respectively) also display signatures of recent insertional activity in the common sunflower (<em>H. annuus</em>) genome, indicating potential widespread activity throughout the genus.</p><p id="p-37">Developing appropriate methods for meaningful comparisons of TE content and abundance across species genomes has become increasingly necessary as NGS technologies continue to improve and costs continue to decline. The graph-based clustering approach of short-read Illumina data combined with sequence information from a TE reference panel proved effective for interspecific analyses of sublineage identification and sequence densities in <em>Helianthus</em>, and provides a useful method when TE reference panels are available. A potential complicating factor of this method is that, due to sequence divergence among genomes, fewer sequence reads and/or sublineages might be identified in interspecific comparisons as genetic distance increases from the TE reference panel. To explore this possibility, we tested whether the density of species-specific Illumina reads clustering with <em>gypsy</em> and <em>copia</em> tracer sequences decreased with increasing genetic distance from the <em>H. annuus</em>-derived TE reference panel. We failed to find such a negative correlation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1/FigureS2.pdf">Figure S2A</a>). Interestingly, however, a negative correlation was observed when the LTR retrotransposon panel was used as a reference in a read-mapping based approach (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1/FigureS2.pdf">Figure S2B</a>). This negative correlation persisted when mapping stringency was relaxed and greater numbers of mismatches allowed. Interspecific read-mapping to quantify TE abundances has been problematic in other species groups as well (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Sveinsson <em>et al.</em> 2013</a>), and generally should be avoided.</p></div><div id="sec-15" class="subsection"><h3>Transcriptional activity of gypsy and copia</h3><p id="p-38">Transcriptional activity of both <em>gypsy</em> and <em>copia</em> elements has been documented previously in both cultivated (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Vukich <em>et al.</em> 2009</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Gill <em>et al.</em> 2014</a>) and wild (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Kawakami <em>et al.</em> 2011</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kawakami <em>et al.</em> 2014</a>; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Ungerer and Kawakami 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Renaut <em>et al.</em> 2014</a>) sunflowers. In the current study we confirmed expression of these elements in two species (<em>H. annuus</em> and <em>H. anomalus</em>) and demonstrate that transcriptional activity occurs broadly across the genus. Transcriptionally active elements documented in the current study represent the same variants associated with genome expansion events documented in three sunflower homoploid hybrid species (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Ungerer <em>et al.</em> 2006</a>; <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Kawakami <em>et al.</em> 2010</a>).</p><p id="p-39">Transcriptional activity of <em>gypsy</em> sequences was readily detectable in both leaf and bud tissue for all annual <em>Helianthus</em> species, less detectable in bud tissue of perennial <em>Helianthus</em> species, and undetectable in leaf tissue of perennial <em>Helianthus</em> species. Although the primers used to assay for transcriptional activity were developed from <em>H. annuus</em> (an annual species), differential detection for annual <em>vs.</em> perennial species is unlikely attributable to sequence divergence with increasing phylogenetic distance from <em>H. annuus</em> given that <em>H. agrestis</em> is an independently evolved annual species and more distant genetically from <em>H. annuus</em> than the remaining <em>Helianthus</em> species under investigation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1/FigureS2.pdf">Figure S2</a>). It is interesting to note that more detectable transcriptional activity in annual species is consistent with a higher density of reads derived from this same sublineage based on clustering analyses of genomic short-read data (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>), demonstrating a potential link between transcriptional activity and genomic abundance level of element copy number in this group of plants. Quantitative PCR experiments have confirmed such a relationship comparing annual sunflower taxa <em>H</em>. <em>annuus</em> and <em>H. petiolaris</em> with their hybrid derivative species <em>H. anomalus</em>, <em>H. deserticola</em>, and <em>H. paradoxus</em>, where higher expression was observed in species with higher copy number abundances (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Ungerer and Kawakami 2013</a>; Renaut <em>et al.</em> 2014, but see <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Gill <em>et al.</em> 2014</a>). Transcriptional activity of this <em>gypsy</em> sublineage was not detected in either leaf or bud tissue of <em>P. tenuifolius</em>, indicating that expression may be restricted to within <em>Helianthus</em>.</p><p id="p-40">In contrast to results for <em>gypsy</em> transcriptional activity, expression of <em>copia</em> was equally detectable among <em>Helianthus</em> annual and perennial species, across tissue types, and in <em>P. tenuifolius</em>. Read density of genomic short-read data for this same sublineage generally are comparable across annual and perennial <em>Helianthus</em> species with the exception of <em>H. agrestis</em>, for which read density is higher. More quantitative assays of transcriptional activity of both <em>gypsy</em> and <em>copia</em> elements may yield additional insights into expression dynamics of these elements across the sunflower genus. Transcriptional activity of additional sublineages of <em>gypsy</em> and <em>copia</em> have been documented previously in <em>H. annuus</em> (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Gill <em>et al.</em> 2014</a>) via RNA-seq and shown to exhibit tissue-specific expression.</p></div><div id="sec-16" class="subsection"><h3>Genome expansion in H. agrestis</h3><p id="p-41">A notable finding of the current study is genomic amplification of LTR retrotransposon sublineages in the genome of <em>H. agrestis</em>. <em>H. agrestis</em> has a restricted geographical distribution in the southeastern United States, with populations found in central and southern Florida and in a single county in southern Georgia (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Heiser <em>et al.</em> 1969</a>). As noted above, this species is an annual, but distantly related from most other <em>Helianthus</em> annuals that form a monophyletic group and thus has independently evolved this life history form (<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Stephens <em>et al.</em> 2015</a>). <em>H. agrestis</em> is atypical in being one of only two <em>Helianthus</em> species that lack a self-incompatibility system (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Heiser <em>et al.</em> 1969</a>). Genome size estimates of <em>H. agrestis</em> indicate a nuclear genome ∼1.9–3.5× larger than any other <em>Helianthus</em> species under investigation in the current study and ∼1.7× larger than that for the outgroup species <em>P. tenuifolius</em>.</p><p id="p-42">Genome expansion in <em>H. agrestis</em> is associated with amplification of a small number of LTR retrotransposon sublineages, represented by two different <em>gypsy</em> sublineages and a single <em>copia</em> sublineage. Sequences from these three sublineages represent ∼28% of the <em>H. agrestis</em> genome based on our estimation procedures (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1/TableS1.pdf">Table S1</a>). This observation is consistent with previous findings demonstrating that large interspecific variation in genome size can be attributable to a small number of LTR retrotransposon sublineages (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Hawkins <em>et al.</em> 2006</a>; <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Piegu <em>et al.</em> 2006</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Vitte and Bennetzen 2006</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">El Baidouri and Panaud 2013</a>) but contrasts with results observed for species of plants harboring some of the largest genomes (<em>e.g.</em>, <em>Fritillaria</em>) where genome composition appears to consist of highly diverse, but relatively low abundance repeat types (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Kelly <em>et al.</em> 2015</a>). As noted above, two of the three most abundant sublineages in the <em>H. agrestis</em> genome (<em>gypsy</em> sublineage A and <em>copia</em> sublineage 1) have contributed to major genome expansion events in one or more diploid hybrid <em>Helianthus</em> species (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Ungerer <em>et al.</em> 2006</a>; <a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">Kawakami <em>et al.</em> 2010</a>), and these sublineages remain active transcriptionally across the genus. It is thus noteworthy that the same LTR retrotransposon sublineages have experienced large-scale amplification events and promoted genome expansion independently in different regions of the <em>Helianthus</em> phylogeny. The forces governing activation (and repression) of these sublineages in different <em>Helianthus</em> species’ genomes is the focus of ongoing work.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">We thank Loran Anderson for providing seeds of <em>P. tenuifolius</em>, Bradley Olson for access to a Guava flow cytometry system, Carolyn Ferguson for helpful advice, and two anonymous reviewers for comments that improved the final version. This work was supported by Kansas State University and National Science Foundation grant DEB-0742993 to M.C.U. H.T. received support from the Department of Education Graduate Assistance in Area of National Need (GAANN) program. Contribution no. 16-330-J from the Kansas Agricultural Experiment Station.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-44">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029082/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-45"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-03-11"><span class="received-label">Received </span>March 11, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-05-20"><span class="accepted-label">Accepted </span>May 20, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Tetreault and Ungerer</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.1" data-doi="10.1111/evo.12627"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agren</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greiner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">M. T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">No evidence that sex and transposable elements drive genome size variation in evening primroses.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">69</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1053</span>–<span class="cit-lpage">1062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D69%26rft.spage%253D1053%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12627%26rft_id%253Dinfo%253Apmid%252F25690700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25690700&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.2" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">15</span>): <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTUvMjExNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.3" data-doi="10.1111/1755-0998.12133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonchev</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parisod</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transposable elements and microevolutionary changes in natural populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">765</span>–<span class="cit-lpage">775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D13%26rft.spage%253D765%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12133%26rft_id%253Dinfo%253Apmid%252F23795753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23795753&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.4" data-doi="10.1007/s00122-011-1626-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giordani</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cattonaro</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cossu</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pistelli</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Temporal dynamics in the evolution of the sunflower genome as revealed by sequencing and annotation of three large genomic regions.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">779</span>–<span class="cit-lpage">791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DButi%26rft.auinit1%253DM.%26rft.volume%253D123%26rft.issue%253D5%26rft.spage%253D779%26rft.epage%253D791%26rft.atitle%253DTemporal%2Bdynamics%2Bin%2Bthe%2Bevolution%2Bof%2Bthe%2Bsunflower%2Bgenome%2Bas%2Brevealed%2Bby%2Bsequencing%2Band%2Bannotation%2Bof%2Bthree%2Blarge%2Bgenomic%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1626-4%26rft_id%253Dinfo%253Apmid%252F21647740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1626-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21647740&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.5" data-doi="10.1534/genetics.108.092304"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charles</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belcram</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Just</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huneau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viollet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Dynamics and differential proliferation of transposable elements during the evolution of the B and A genomes of wheat.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">1071</span>–<span class="cit-lpage">1086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.092304%26rft_id%253Dinfo%253Apmid%252F18780739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC8yLzEwNzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.6" data-doi="10.1101/gr.132102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genome size reduction through illegitimate recombination counteracts genome expansion in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1075</span>–<span class="cit-lpage">1079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DDevos%26rft.auinit1%253DK.%2BM.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D1075%26rft.epage%253D1079%26rft.atitle%253DGenome%2BSize%2BReduction%2Bthrough%2BIllegitimate%2BRecombination%2BCounteracts%2BGenome%2BExpansion%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.132102%26rft_id%253Dinfo%253Apmid%252F12097344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEyLzcvMTA3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.7" data-doi="10.1371/journal.pgen.1004298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diez</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meca</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Three groups of transposable elements with contrasting copy number dynamics and host responses in the maize (<em>Zea mays ssp mays</em>) genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">e1004298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004298%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004298%26rft_id%253Dinfo%253Apmid%252F24743518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004298&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24743518&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.8" data-doi="10.1002/cyto.a.10013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dolezel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartos</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voglmayr</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greilhuber</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Nuclear DNA content and genome size of trout and human.</span> <abbr class="cit-jnl-abbrev">Cytometry A</abbr> <span class="cit-vol">51</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">127</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytometry.%2BPart%2BA%2B%253A%2B%2Bthe%2Bjournal%2Bof%2Bthe%2BInternational%2BSociety%2Bfor%2BAnalytical%2BCytology%26rft.stitle%253DCytometry%2BA%26rft.aulast%253DDolezel%26rft.auinit1%253DJ.%26rft.volume%253D51%26rft.issue%253D2%26rft.spage%253D127%26rft.epage%253D128%26rft.atitle%253DNuclear%2BDNA%2Bcontent%2Band%2Bgenome%2Bsize%2Bof%2Btrout%2Band%2Bhuman.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcyto.a.10013%26rft_id%253Dinfo%253Apmid%252F12541287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cyto.a.10013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12541287&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.9" data-doi="10.1093/gbe/evt025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El Baidouri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panaud</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative genomic paleontology across plant kingdom reveals the dynamics of TE-driven genome evolution.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">954</span>–<span class="cit-lpage">965</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt025%26rft_id%253Dinfo%253Apmid%252F23426643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjUvNS85NTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.10" data-doi="10.1186/1471-2105-9-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellinghaus</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willhoeft</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">LTRharvest, an efficient and flexible software for de novo detection of LTR retrotransposons.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEllinghaus%26rft.auinit1%253DD.%26rft.volume%253D9%26rft.spage%253D18%26rft.epage%253D18%26rft.atitle%253DLTRharvest%252C%2Ban%2Befficient%2Band%2Bflexible%2Bsoftware%2Bfor%2Bde%2Bnovo%2Bdetection%2Bof%2BLTR%2Bretrotransposons.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-18%26rft_id%253Dinfo%253Apmid%252F18194517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18194517&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.11" data-doi="10.1038/nrg793"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feschotte</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wessler</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Plant transposable elements: where genetics meets genomics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">329</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFeschotte%26rft.auinit1%253DC.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253D329%26rft.epage%253D341%26rft.atitle%253DPlant%2Btransposable%2Belements%253A%2Bwhere%2Bgenetics%2Bmeets%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg793%26rft_id%253Dinfo%253Apmid%252F11988759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg793&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11988759&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175350000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellec</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helmstetter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Sequence-based analysis of structural organization and composition of the cultivated sunflower (<em>Helianthus annuus</em> L.) genome.</span> <abbr class="cit-jnl-abbrev">Biology (Basel)</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">295</span>–<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiology%2B%2528Basel%2529%26rft.volume%253D3%26rft.spage%253D295%26rft_id%253Dinfo%253Apmid%252F24833511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24833511&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.13" data-doi="10.1016/j.cpb.2014.05.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giordani</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavallini</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natali</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The repetitive component of the sunflower genome.</span> <abbr class="cit-jnl-abbrev">Current Plant Biology</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BPlant%2BBiology%26rft.volume%253D1%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cpb.2014.05.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cpb.2014.05.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.14" data-doi="10.1101/gr.5282906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawkins</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nason</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wing</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Differential lineage-specific amplification of transposable elements is responsible for genome size variation in <em>Gossypium</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1252</span>–<span class="cit-lpage">1261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5282906%26rft_id%253Dinfo%253Apmid%252F16954538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMC8xMjUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.15" data-doi="10.1073/pnas.0904339106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawkins</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Proulx</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rapp</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Rapid DNA loss as a counterbalance to genome expansion through retrotransposon proliferation in plants.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>(<span class="cit-issue">42</span>): <span class="cit-fpage">17811</span>–<span class="cit-lpage">17816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0904339106%26rft_id%253Dinfo%253Apmid%252F19815511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzQyLzE3ODExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heiser</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clevenger</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">W. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">The North American sunflowers (Helianthus).</span> <abbr class="cit-jnl-abbrev">Memoirs of the Torrey Botanical Club</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMemoirs%2Bof%2Bthe%2BTorrey%2BBotanical%2BClub%26rft.volume%253D22%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.17" data-doi="10.1038/nature03895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Rice Genome Sequencing</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The map-based sequence of the rice genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">436</span>(<span class="cit-issue">7052</span>): <span class="cit-fpage">793</span>–<span class="cit-lpage">800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D436%26rft.issue%253D7052%26rft.spage%253D793%26rft.epage%253D800%26rft.atitle%253DThe%2Bmap-based%2Bsequence%2Bof%2Bthe%2Brice%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03895%26rft_id%253Dinfo%253Apmid%252F16100779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03895&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16100779&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231116500034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.18" data-doi="10.1139/b11-032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berges</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Progress towards a reference genome for sunflower.</span> <abbr class="cit-jnl-abbrev">Botany-Botanique</abbr> <span class="cit-vol">89</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">429</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBotany-Botanique%26rft.volume%253D89%26rft.spage%253D429%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fb11-032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/b11-032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.19" data-doi="10.1111/1755-0998.12023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marek</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seiler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vear</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sunflower genetic, genomic and ecological resources.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">10</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D13%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12023%26rft_id%253Dinfo%253Apmid%252F23039950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23039950&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.20" data-doi="10.1038/hdy.2009.182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strakosh</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ungerer</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Different scales of Ty1/copia-like retrotransposon proliferation in the genomes of three diploid hybrid sunflower species.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">104</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">341</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DKawakami%26rft.auinit1%253DT.%26rft.volume%253D104%26rft.issue%253D4%26rft.spage%253D341%26rft.epage%253D350%26rft.atitle%253DDifferent%2Bscales%2Bof%2BTy1%252Fcopia-like%2Bretrotransposon%2Bproliferation%2Bin%2Bthe%2Bgenomes%2Bof%2Bthree%2Bdiploid%2Bhybrid%2Bsunflower%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.182%26rft_id%253Dinfo%253Apmid%252F20068588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20068588&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275898200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.21" data-doi="10.1093/gbe/evr005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhakal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katterhenry</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heatherington</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ungerer</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transposable element proliferation and genome expansion are rare in contemporary sunflower hybrid populations despite widespread transcriptional activity of LTR retrotransposons.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr005%26rft_id%253Dinfo%253Apmid%252F21282712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjMvMC8xNTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.22" data-doi="10.1111/1755-0998.12227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darby</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ungerer</span>  <span class="cit-name-given-names">M. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transcriptome resources for the perennial sunflower <em>Helianthus maximiliani</em> obtained from ecologically divergent populations.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">812</span>–<span class="cit-lpage">819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D14%26rft.spage%253D812%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12227%26rft_id%253Dinfo%253Apmid%252F24438509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24438509&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.23" data-doi="10.1007/s10577-011-9246-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leitch</span>  <span class="cit-name-given-names">I. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Exploring giant plant genomes with next-generation sequencing technology.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">939</span>–<span class="cit-lpage">953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DKelly%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D939%26rft.epage%253D953%26rft.atitle%253DExploring%2Bgiant%2Bplant%2Bgenomes%2Bwith%2Bnext-generation%2Bsequencing%2Btechnology.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-011-9246-z%26rft_id%253Dinfo%253Apmid%252F21987187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-011-9246-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21987187&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300089700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.24" data-doi="10.1111/nph.13471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Renny-Byfield</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pellicer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novak</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Analysis of the giant genomes of <em>Fritillaria</em> (Liliaceae) indicates that a lack of DNA removal characterizes extreme expansions in genome size.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">208</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">596</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D208%26rft.spage%253D596%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13471%26rft_id%253Dinfo%253Apmid%252F26061193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26061193&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.25" data-doi="10.1146/annurev.genet.33.1.479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Plant retrotransposons.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DKumar%26rft.auinit1%253DA.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D479%26rft.epage%253D532%26rft.atitle%253DPLANT%2BRETROTRANSPOSONS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.33.1.479%26rft_id%253Dinfo%253Apmid%252F10690416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.33.1.479&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10690416&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084956600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.26" data-doi="10.1186/1471-2164-9-517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narechania</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ware</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A new method to compute K-mer frequencies and its application to annotate large repetitive plant genomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKurtz%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.spage%253D517%26rft.epage%253D517%26rft.atitle%253DA%2Bnew%2Bmethod%2Bto%2Bcompute%2BK-mer%2Bfrequencies%2Band%2Bits%2Bapplication%2Bto%2Bannotate%2Blarge%2Brepetitive%2Bplant%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-517%26rft_id%253Dinfo%253Apmid%252F18976482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-517&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18976482&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.27" data-doi="10.1038/nbt.2065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natsoulis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Performance comparison of whole-genome sequencing platforms.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">78</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DLam%26rft.auinit1%253DH.%2BY.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D78%26rft.epage%253D82%26rft.atitle%253DPerformance%2Bcomparison%2Bof%2Bwhole-genome%2Bsequencing%2Bplatforms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2065%26rft_id%253Dinfo%253Apmid%252F22178993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22178993&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.28" data-doi="10.1016/0888-7543(88)90007-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterman</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Genomic mapping by fingerprinting random clones: a mathematical analysis.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">231</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253D231%26rft.epage%253D239%26rft.atitle%253DGenomic%2Bmapping%2Bby%2Bfingerprinting%2Brandom%2Bclones%253A%2Ba%2Bmathematical%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0888-7543%252888%252990007-9%26rft_id%253Dinfo%253Apmid%252F3294162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0888-7543(88)90007-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3294162&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-6.8.2299.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leitch</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leitch</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome size diversity and evolution in land plants</span>, pp. <span class="cit-fpage">307</span>–<span class="cit-lpage">322</span> in <span class="cit-source">Plant Genome Diversity</span>, Vol 2, edited by <span class="cit-ed"><span class="cit-name-surname">Leitch</span>  <span class="cit-name-given-names">I. J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Greilhuber</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Dolezel</span>  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J.</span></span>. <span class="cit-publ-name">Springer, Wien, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.30" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.31" data-doi="10.1101/gr.1466204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Analyses of LTR-retrotransposon structures reveal recent and rapid genomic DNA loss in rice.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">860</span>–<span class="cit-lpage">869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1466204%26rft_id%253Dinfo%253Apmid%252F15078861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE0LzUvODYwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.32" data-doi="10.1186/1471-2164-8-427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neumann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navratilova</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Repetitive DNA in the pea (<em>Pisum sativum</em> L.) genome: comprehensive characterization using 454 sequencing and comparison to soybean and <em>Medicago truncatula</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMacas%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.spage%253D427%26rft.epage%253D427%26rft.atitle%253DRepetitive%2BDNA%2Bin%2Bthe%2Bpea%2B%2528Pisum%2Bsativum%2BL.%2529%2Bgenome%253A%2Bcomprehensive%2Bcharacterization%2Busing%2B454%2Bsequencing%2Band%2Bcomparison%2Bto%2Bsoybean%2Band%2BMedicago%2Btruncatula.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-427%26rft_id%253Dinfo%253Apmid%252F18031571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18031571&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.33" data-doi="10.1038/nature06856"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ming</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dionne-Laporte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The draft genome of the transgenic tropical fruit tree papaya (<em>Carica papaya</em> Linnaeus).</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">452</span>(<span class="cit-issue">7190</span>): <span class="cit-fpage">991</span>–<span class="cit-lpage">996</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMing%26rft.auinit1%253DR.%26rft.volume%253D452%26rft.issue%253D7190%26rft.spage%253D991%26rft.epage%253D996%26rft.atitle%253DThe%2Bdraft%2Bgenome%2Bof%2Bthe%2Btransgenic%2Btropical%2Bfruit%2Btree%2Bpapaya%2B%2528Carica%2Bpapaya%2BLinnaeus%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06856%26rft_id%253Dinfo%253Apmid%252F18432245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06856&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18432245&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255208600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.34" data-doi="10.1186/1471-2105-11-378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novak</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neumann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Graph-based clustering and characterization of repetitive sequences in next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DNovak%26rft.auinit1%253DP.%26rft.volume%253D11%26rft.spage%253D378%26rft.epage%253D378%26rft.atitle%253DGraph-based%2Bclustering%2Band%2Bcharacterization%2Bof%2Brepetitive%2Bsequences%2Bin%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-378%26rft_id%253Dinfo%253Apmid%252F20633259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-378&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20633259&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.35" data-doi="10.1093/bioinformatics/btt054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novak</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neumann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pech</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinhaisl</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RepeatExplorer: a Galaxy-based web server for genome-wide characterization of eukaryotic repetitive elements from next-generation sequence reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">792</span>–<span class="cit-lpage">793</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt054%26rft_id%253Dinfo%253Apmid%252F23376349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyOS82Lzc5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.36" data-doi="10.1093/bioinformatics/btg412"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paradis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claude</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">APE: analyses of phylogenetics and evolution in R language.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">289</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DParadis%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D290%26rft.atitle%253DAPE%253A%2BAnalyses%2Bof%2BPhylogenetics%2Band%2BEvolution%2Bin%2BR%2Blanguage%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg412%26rft_id%253Dinfo%253Apmid%252F14734327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMC8yLzI4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.37" data-doi="10.1038/nature07723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruggmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubchak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The <em>Sorghum bicolor</em> genome and the diversification of grasses.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>(<span class="cit-issue">7229</span>): <span class="cit-fpage">551</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D457%26rft.issue%253D7229%26rft.spage%253D551%26rft.epage%253D556%26rft.atitle%253DThe%2BSorghum%2Bbicolor%2Bgenome%2Band%2Bthe%2Bdiversification%2Bof%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07723%26rft_id%253Dinfo%253Apmid%252F19189423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19189423&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262852200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.38" data-doi="10.1093/molbev/mss168"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piednoel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aberer</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeweiss</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novak</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Next-generation sequencing reveals the impact of repetitive DNA across phylogenetically closely related genomes of Orobanchaceae.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">3601</span>–<span class="cit-lpage">3611</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss168%26rft_id%253Dinfo%253Apmid%252F22723303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xMS8zNjAxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.39" data-doi="10.1101/gr.5290206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piegu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guyot</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Picault</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roulin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saniyal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Doubling genome size without polyploidization: dynamics of retrotransposition-driven genomic expansions in <em>Oryza australiensis</em>, a wild relative of rice.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1262</span>–<span class="cit-lpage">1269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5290206%26rft_id%253Dinfo%253Apmid%252F16963705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMC8xMjYyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.40" data-doi="10.1098/rstb.2013.0345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowe</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ungerer</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomics of homoploid hybrid speciation: diversity and transcriptional activity of long terminal repeat retrotransposons in hybrid sunflowers.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">369</span>: <span class="cit-fpage">20130345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2013.0345%26rft_id%253Dinfo%253Apmid%252F24958919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjE3OiIzNjkvMTY0OC8yMDEzMDM0NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.41" data-doi="10.2307/2444926"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Homoploid reticulate evolution in <em>Helianthus</em> (Asteraceae) – evidence from ribosomal genes.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">78</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1218</span>–<span class="cit-lpage">1237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft.volume%253D78%26rft.spage%253D1218%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2444926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2444926&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991GF61800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.42" data-doi="10.3417/0026-6493(2006)93[34:HSIWS]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Hybrid speciation in wild sunflowers.</span> <abbr class="cit-jnl-abbrev">Ann. Mo. Bot. Gard.</abbr> <span class="cit-vol">93</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">34</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMo.%2BBot.%2BGard.%26rft.volume%253D93%26rft.spage%253D34%26rft_id%253Dinfo%253Adoi%252F10.3417%252F0026-6493%25282006%252993%255B34%253AHSIWS%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3417/0026-6493(2006)93[34:HSIWS]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.43" data-doi="10.1007/s001220050497"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schilling</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Phylogenetic analysis of <em>Helianthus</em> (Asteraceae) based on chloroplast DNA restriction site data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">94</span>(<span class="cit-issue">6–7</span>): <span class="cit-fpage">925</span>–<span class="cit-lpage">933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D94%26rft.spage%253D925%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050497&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XE63000029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.44" data-doi="10.2307/1220139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schilling</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heiser</span>  <span class="cit-name-given-names">C. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Infrageneric classification of <em>Helianthus</em> (Compositae).</span> <abbr class="cit-jnl-abbrev">Taxon</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">393</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTaxon%26rft.volume%253D30%26rft.spage%253D393%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1220139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1220139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1981LS03300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.45" data-doi="10.2307/2419587"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schilling</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linder</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noyes</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Phylogenetic relationships in <em>Helianthus</em> (Asteraceae) based on nuclear ribosomal DNA internal transcribed spacer region sequence data.</span> <abbr class="cit-jnl-abbrev">Syst. Bot.</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">177</span>–<span class="cit-lpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBot.%26rft.volume%253D23%26rft.spage%253D177%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2419587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2419587&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000077783000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.46" data-doi="10.2307/2443401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Nuclear DNA content variation in <em>Helianthus</em> (Asteraceae).</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">72</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1213</span>–<span class="cit-lpage">1219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft.volume%253D72%26rft.spage%253D1213%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2443401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2443401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985ATA8100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.47" data-doi="10.1186/s12864-015-1830-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staton</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Evolutionary transitions in the Asteraceae coincide with marked shifts in transposable element abundance.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1830-8%26rft_id%253Dinfo%253Apmid%252F26290182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1830-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26290182&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.48" data-doi="10.3732/ajb.0800337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staton</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ungerer</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genomic organization of Ty3/gypsy-like retrotransposons in <em>Helianthus</em> (Asteraceae) homoploid hybrid species.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">96</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1646</span>–<span class="cit-lpage">1655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.0800337%26rft_id%253Dinfo%253Apmid%252F21622351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjk6Ijk2LzkvMTY0NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.49" data-doi="10.1111/j.1365-313X.2012.05072.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staton</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakken</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackman</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The sunflower (<em>Helianthus annuus</em> L.) genome reflects a recent history of biased accumulation of transposable elements.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">72</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">142</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DStaton%26rft.auinit1%253DS.%2BE.%26rft.volume%253D72%26rft.issue%253D1%26rft.spage%253D142%26rft.epage%253D153%26rft.atitle%253DThe%2Bsunflower%2B%2528Helianthus%2Bannuus%2BL.%2529%2Bgenome%2Breflects%2Ba%2Brecent%2Bhistory%2Bof%2Bbiased%2Baccumulation%2Bof%2Btransposable%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2012.05072.x%26rft_id%253Dinfo%253Apmid%252F22691070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2012.05072.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22691070&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309064100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.50" data-doi="10.1093/nar/gkp759"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinbiss</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willhoeft</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gremme</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fine-grained annotation and classification of de novo predicted LTR retrotransposons.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>(<span class="cit-issue">21</span>): <span class="cit-fpage">7002</span>–<span class="cit-lpage">7013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp759%26rft_id%253Dinfo%253Apmid%252F19786494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNy8yMS83MDAyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.51" data-doi="10.3732/ajb.1500031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donovan</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malmberg</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Species tree estimation of diploid <em>Helianthus</em> (Asteraceae) using target enrichment.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">102</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">910</span>–<span class="cit-lpage">920</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1500031%26rft_id%253Dinfo%253Apmid%252F26101417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjk6IjEwMi82LzkxMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.52" data-doi="10.1186/1471-2164-14-502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sveinsson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronk</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transposon fingerprinting using low coverage whole genome shotgun sequencing in Cacao (<em>Theobroma cacao</em> L.) and related species.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D502%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-502%26rft_id%253Dinfo%253Apmid%252F23883295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-502&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23883295&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.53" data-doi="10.1186/1471-2164-8-132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swaminathan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varala</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Global repeat discovery and estimation of genomic copy number in a large, complex genome using a high-throughput 454 sequence survey.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSwaminathan%26rft.auinit1%253DK.%26rft.volume%253D8%26rft.spage%253D132%26rft.epage%253D132%26rft.atitle%253DGlobal%2Brepeat%2Bdiscovery%2Band%2Bestimation%2Bof%2Bgenomic%2Bcopy%2Bnumber%2Bin%2Ba%2Blarge%252C%2Bcomplex%2Bgenome%2Busing%2Ba%2Bhigh-throughput%2B454%2Bsequence%2Bsurvey.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-132%26rft_id%253Dinfo%253Apmid%252F17524145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17524145&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.54" data-doi="10.1093/molbev/msm092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">1596</span>–<span class="cit-lpage">1599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm092%26rft_id%253Dinfo%253Apmid%252F17488738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTU5NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvOC8yMjk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.55" data-doi="10.1093/gbe/evr008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hufford</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome size and transposable element content as determined by high-throughput sequencing in maize and <em>Zea luxurians</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr008%26rft_id%253Dinfo%253Apmid%252F21296765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjMvMC8yMTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.56" data-doi="10.3732/ajb.94.11.1837"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Timme</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linder</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">High-resolution phylogeny for <em>Helianthus</em> (Asteraceae) using the 18s-26s ribosomal DNA external transcribed spacer.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">94</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1837</span>–<span class="cit-lpage">1852</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DTimme%26rft.auinit1%253DR.%2BE.%26rft.volume%253D94%26rft.issue%253D11%26rft.spage%253D1837%26rft.epage%253D1852%26rft.atitle%253DHigh-resolution%2Bphylogeny%2Bfor%2BHelianthus%2B%2528Asteraceae%2529%2Busing%2Bthe%2B18S-26S%2Bribosomal%2BDNA%2Bexternal%2Btranscribed%2Bspacer%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.94.11.1837%26rft_id%253Dinfo%253Apmid%252F21636379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjEwOiI5NC8xMS8xODM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.57" data-doi="10.1093/gbe/evt006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ungerer</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawakami</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transcriptional dynamics of LTR retrotransposons in early generation and ancient sunflower hybrids.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">329</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt006%26rft_id%253Dinfo%253Apmid%252F23335122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjUvMi8zMjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzgvMjI5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.58" data-doi="10.1016/j.cub.2006.09.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ungerer</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strakosh</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhen</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome expansion in three hybrid sunflower species is associated with retrotransposon proliferation.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">20</span>): <span class="cit-fpage">R872</span>–<span class="cit-lpage">R873</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DUngerer%26rft.auinit1%253DM.%2BC.%26rft.volume%253D16%26rft.issue%253D20%26rft.spage%253DR872%26rft.epage%253DR873%26rft.atitle%253DGenome%2Bexpansion%2Bin%2Bthree%2Bhybrid%2Bsunflower%2Bspecies%2Bis%2Bassociated%2Bwith%2Bretrotransposon%2Bproliferation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2006.09.020%26rft_id%253Dinfo%253Apmid%252F17055967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2006.09.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17055967&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241532000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.59" data-doi="10.1186/1741-7007-7-40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ungerer</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strakosh</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stimpson</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Proliferation of Ty3/gypsy-like retrotransposons in hybrid sunflower taxa inferred from phylogenetic data.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DUngerer%26rft.auinit1%253DM.%2BC.%26rft.volume%253D7%26rft.spage%253D40%26rft.epage%253D40%26rft.atitle%253DProliferation%2Bof%2BTy3%252Fgypsy-like%2Bretrotransposons%2Bin%2Bhybrid%2Bsunflower%2Btaxa%2Binferred%2Bfrom%2Bphylogenetic%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-7-40%26rft_id%253Dinfo%253Apmid%252F19594956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-7-40&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19594956&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.60" data-doi="10.1073/pnas.0605618103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitte</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Analysis of retrotransposon structural diversity uncovers properties and propensities in angiosperm genome evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>(<span class="cit-issue">47</span>): <span class="cit-fpage">17638</span>–<span class="cit-lpage">17643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605618103%26rft_id%253Dinfo%253Apmid%252F17101966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzQ3LzE3NjM4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.61" data-doi="10.1093/molbev/msg055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitte</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panaud</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Formation of solo-LTRs through unequal homologous recombination counterbalances amplifications of LTR retrotransposons in rice <em>Oryza sativa</em> L.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">528</span>–<span class="cit-lpage">540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg055%26rft_id%253Dinfo%253Apmid%252F12654934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIwLzQvNTI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi84LzIyOTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.62" data-doi="10.1159/000084941"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitte</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panaud</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">LTR retrotransposons and flowering plant genome size: emergence of the increase/decrease model.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">110</span>(<span class="cit-issue">1–4</span>): <span class="cit-fpage">91</span>–<span class="cit-lpage">107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DVitte%26rft.auinit1%253DC.%26rft.volume%253D110%26rft.issue%253D1-4%26rft.spage%253D91%26rft.epage%253D107%26rft.atitle%253DLTR%2Bretrotransposons%2Band%2Bflowering%2Bplant%2Bgenome%2Bsize%253A%2Bemergence%2Bof%2Bthe%2Bincrease%252Fdecrease%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000084941%26rft_id%253Dinfo%253Apmid%252F16093661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000084941&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16093661&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231064600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.63" data-doi="10.1186/1471-2229-9-150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vukich</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giordani</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natali</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavallini</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Copia and Gypsy retrotransposons activity in sunflower (<em>Helianthus annuus</em> L.).</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DVukich%26rft.auinit1%253DM.%26rft.volume%253D9%26rft.spage%253D150%26rft.epage%253D150%26rft.atitle%253DCopia%2Band%2BGypsy%2Bretrotransposons%2Bactivity%2Bin%2Bsunflower%2B%2528Helianthus%2Bannuus%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-150%26rft_id%253Dinfo%253Apmid%252F20030800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20030800&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.64" data-doi="10.1038/nrg2165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wicker</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabot</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hua-Van</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A unified classification system for eukaryotic transposable elements.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">973</span>–<span class="cit-lpage">982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMathew%26rft.auinit1%253DC.%2BG.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253D973%26rft.epage%253D982%26rft.atitle%253DA%2Bunified%2Bclassification%2Bsystem%2Bfor%2Beukaryotic%2Btransposable%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2165%26rft_id%253Dinfo%253Apmid%252F17984973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17984973&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.8.2299.65" data-doi="10.1111/j.1365-313X.2009.03911.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wicker</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taudien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houben</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A whole-genome snapshot of 454 sequences exposes the composition of the barley genome and provides evidence for parallel evolution of genome size in wheat and barley.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">59</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">712</span>–<span class="cit-lpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DWicker%26rft.auinit1%253DT.%26rft.volume%253D59%26rft.issue%253D5%26rft.spage%253D712%26rft.epage%253D722%26rft.atitle%253DA%2Bwhole-genome%2Bsnapshot%2Bof%2B454%2Bsequences%2Bexposes%2Bthe%2Bcomposition%2Bof%2Bthe%2Bbarley%2Bgenome%2Band%2Bprovides%2Bevidence%2Bfor%2Bparallel%2Bevolution%2Bof%2Bgenome%2Bsize%2Bin%2Bwheat%2Band%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2009.03911.x%26rft_id%253Dinfo%253Apmid%252F19453446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2009.03911.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19453446&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F8%2F2299.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269316400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/8/2299.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/8/2285" title="Diversifying Selection Between Pure-Breed and Free-Breeding Dogs Inferred from Genome-Wide SNP Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/8/2309" title="Characterization of a Single Genomic Locus Encoding the Clustered Protocadherin Receptor Diversity in Xenopus tropicalis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/8"><span>Volume 6 Issue 8, August 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/8.cover-source.jpg?itok=40Kz9Oeq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/8/2299" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-G9MFNl5n0BTlDp6XbRolptvB2WLWtKT141RSpcw9wXk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/8/2299" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/8/2299" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/8/2299" class="active" data-icon-position="" data-hide-link-title="0">Long Terminal Repeat Retrotransposon Content in Eight Diploid Sunflower Species Inferred from Next-Generation Sequence Data</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460441" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-wPTwOj99gRlsnwfwAXX62qmgWN4SPm6w7E2b8xo03fQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2299.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/8/2299" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-nk__b7XDv-tfy7kN9H4_AsMXMhKbAIX_gCzOHsLAcTg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/8/2299" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/8/2299.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460441" id="node460441--2" data-pisa="ggg;6/8/2299" data-pisa-master="ggg;g3.116.029082" data-apath="/ggg/6/8/2299.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Long Terminal Repeat Retrotransposon Content in Eight Diploid Sunflower Species Inferred from Next-Generation Sequence Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hannah M. Tetreault</span> and <span class="highwire-citation-author" data-delta="1">Mark C. Ungerer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2299-2308;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029082 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460441--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hannah M. Tetreault </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Division of Biology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hannah%2BM.%2BTetreault%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tetreault%20HM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHannah%2BM.%2BTetreault%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:htetre@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">htetre@ksu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark C. Ungerer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Division of Biology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BC.%2BUngerer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ungerer%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BC.%2BUngerer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/8/2299" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/8/2299&amp;title=Long%20Terminal%20Repeat%20Retrotransposon%20Content%20in%20Eight%20Diploid%20Sunflower%20Species%20Inferred%20from%20Next-Generation%20Sequence%20Data" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/8/2299&amp;title=Long%20Terminal%20Repeat%20Retrotransposon%20Content%20in%20Eight%20Diploid%20Sunflower%20Species%20Inferred%20from%20Next-Generation%20Sequence%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/8/2299&amp;title=Long%20Terminal%20Repeat%20Retrotransposon%20Content%20in%20Eight%20Diploid%20Sunflower%20Species%20Inferred%20from%20Next-Generation%20Sequence%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/8/2299&amp;text=Long%20Terminal%20Repeat%20Retrotransposon%20Content%20in%20Eight%20Diploid%20Sunflower%20Species%20Inferred%20from%20Next-Generation%20Sequence%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/8/2299&amp;title=Long%20Terminal%20Repeat%20Retrotransposon%20Content%20in%20Eight%20Diploid%20Sunflower%20Species%20Inferred%20from%20Next-Generation%20Sequence%20Data" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/8/2299&amp;t=Long%20Terminal%20Repeat%20Retrotransposon%20Content%20in%20Eight%20Diploid%20Sunflower%20Species%20Inferred%20from%20Next-Generation%20Sequence%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/8/2299&amp;title=Long%20Terminal%20Repeat%20Retrotransposon%20Content%20in%20Eight%20Diploid%20Sunflower%20Species%20Inferred%20from%20Next-Generation%20Sequence%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/8/2299&amp;title=Long%20Terminal%20Repeat%20Retrotransposon%20Content%20in%20Eight%20Diploid%20Sunflower%20Species%20Inferred%20from%20Next-Generation%20Sequence%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460441/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460441" id="node460441--3" data-pisa="ggg;6/8/2299" data-pisa-master="ggg;g3.116.029082" data-apath="/ggg/6/8/2299.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Long Terminal Repeat Retrotransposon Content in Eight Diploid Sunflower Species Inferred from Next-Generation Sequence Data</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hannah M. Tetreault</span> and <span class="highwire-citation-author" data-delta="1">Mark C. Ungerer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2299-2308;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029082 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460441--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hannah M. Tetreault </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Division of Biology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hannah%2BM.%2BTetreault%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tetreault%20HM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHannah%2BM.%2BTetreault%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:htetre@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">htetre@ksu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark C. Ungerer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Division of Biology, Kansas State University, Manhattan, Kansas 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BC.%2BUngerer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ungerer%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BC.%2BUngerer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2299&amp;rft.epage=2308&amp;rft.atitle=Long%20Terminal%20Repeat%20Retrotransposon%20Content%20in%20Eight%20Diploid%20Sunflower%20Species%20Inferred%20from%20Next-Generation%20Sequence%20Data&amp;rft.volume=6&amp;rft.issue=8&amp;rft.date=2016-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Tetreault%2C+Hannah+M.&amp;rft.au=Ungerer%2C+Mark+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460441/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460441/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460441/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460441/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460441/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460441/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460441/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460441/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460441/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460441/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460441/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460441/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/8/2299" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/8/2299.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/8/2299.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/8/2299.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/8/2299.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
