<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/5/387" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/5/387.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/5/387.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/5/387.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/5.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/5/387.atom" />
<meta name="HW.pisa" content="ggg;1/5/387" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Polymorphism and Divergence in Two Willow Species, Salix viminalis L. and Salix schwerinii E. Wolf" />
<meta name="DC.Identifier" content="10.1534/g3.111.000539" />
<meta name="DC.Date" content="2011-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Berlin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We investigated species divergence, present and past gene flow, levels of nucleotide polymorphism, and linkage disequilibrium in two willows from the plant genus Salix . Salix belongs together with Populus to the Salicaceae family; however, most population genetic studies of Salicaceae have been performed in Populus , the model genus in forest biology. Here we present a study on two closely related willow species Salix viminalis and S. schwerinii , in which we have resequenced 33 and 32 nuclear gene segments representing parts of 18 nuclear loci in 24 individuals for each species. We used coalescent simulations and estimated the split time to around 600,000 years ago and found that there is currently limited gene flow between the species. Mean intronic nucleotide diversity across gene segments was slightly higher in S. schwerinii ( πi = 0.00849) than in S. viminalis ( πi = 0.00655). Compared with other angiosperm trees, the two willows harbor intermediate levels of silent polymorphisms. The decay of linkage disequilibrium was slower in S. viminalis compared with S. schwerinii , and we speculate that this is due to different demographic histories as S. viminalis has been partly domesticated in Europe." />
<meta name="DC.Contributor" content="Sofia Berlin" />
<meta name="DC.Contributor" content="Johan Fogelqvist" />
<meta name="DC.Contributor" content="Martin Lascoux" />
<meta name="DC.Contributor" content="Ulf Lagercrantz" />
<meta name="DC.Contributor" content="Ann Christin Rönnberg-Wästljung" />
<meta name="article:published_time" content="2011-10-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Polymorphism and Divergence in Two Willow Species, Salix viminalis L. and Salix schwerinii E. Wolf" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We investigated species divergence, present and past gene flow, levels of nucleotide polymorphism, and linkage disequilibrium in two willows from the plant genus &lt;i&gt;Salix&lt;/i&gt;. &lt;i&gt;Salix&lt;/i&gt; belongs together with &lt;i&gt;Populus&lt;/i&gt; to the Salicaceae family; however, most population genetic studies of Salicaceae have been performed in &lt;i&gt;Populus&lt;/i&gt;, the model genus in forest biology. Here we present a study on two closely related willow species &lt;i&gt;Salix viminalis&lt;/i&gt; and &lt;i&gt;S. schwerinii&lt;/i&gt;, in which we have resequenced 33 and 32 nuclear gene segments representing parts of 18 nuclear loci in 24 individuals for each species. We used coalescent simulations and estimated the split time to around 600,000 years ago and found that there is currently limited gene flow between the species. Mean intronic nucleotide diversity across gene segments was slightly higher in &lt;i&gt;S. schwerinii&lt;/i&gt; (&lt;i&gt;π&lt;sub&gt;i&lt;/sub&gt;&lt;/i&gt; = 0.00849) than in &lt;i&gt;S. viminalis&lt;/i&gt; (&lt;i&gt;π&lt;sub&gt;i&lt;/sub&gt;&lt;/i&gt; = 0.00655). Compared with other angiosperm trees, the two willows harbor intermediate levels of silent polymorphisms. The decay of linkage disequilibrium was slower in &lt;i&gt;S. viminalis&lt;/i&gt; compared with &lt;i&gt;S. schwerinii&lt;/i&gt;, and we speculate that this is due to different demographic histories as &lt;i&gt;S. viminalis&lt;/i&gt; has been partly domesticated in Europe.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/10/01" />
<meta name="citation_mjid" content="ggg;1/5/387" />
<meta name="citation_id" content="1/5/387" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/5/387" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/5/387.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/5/387.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/5/387.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000539" />
<meta name="citation_pmid" content="22384349" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="5" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="387" />
<meta name="citation_lastpage" content="400" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sofia Berlin" />
<meta name="citation_author_institution" content="Department of Plant Biology and Forest Genetics, Uppsala BioCenter, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden" />
<meta name="citation_author_email" content="sofia.berlin@slu.se" />
<meta name="citation_author" content="Johan Fogelqvist" />
<meta name="citation_author_institution" content="Department of Plant Biology and Forest Genetics, Uppsala BioCenter, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden" />
<meta name="citation_author_institution" content="Department of Plant Ecology and Evolution, Evolutionary Biology Centre, Uppsala University, SE-752 36 Uppsala, Sweden" />
<meta name="citation_author" content="Martin Lascoux" />
<meta name="citation_author_institution" content="Laboratory of Evolutionary Genomics, CAS-MPG Partner Institute for Computational Biology, Chinese Academy of Sciences, Shanghai, China" />
<meta name="citation_author" content="Ulf Lagercrantz" />
<meta name="citation_author_institution" content="Department of Plant Ecology and Evolution, Evolutionary Biology Centre, Uppsala University, SE-752 36 Uppsala, Sweden" />
<meta name="citation_author" content="Ann Christin Rönnberg-Wästljung" />
<meta name="citation_author_institution" content="Department of Plant Biology and Forest Genetics, Uppsala BioCenter, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Alström-Rapaport;citation_author=M. Lascoux;citation_author=U. Gullberg;citation_title=Sex determination and sex ratio in the dioecious shrub Salix viminalis L.;citation_pages=493-497;citation_volume=94;citation_year=1997;citation_doi=10.1007/s001220050442" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=C. Becquet;citation_author=M. Przeworski;citation_title=Learning about modes of speciation by computational approaches.;citation_pages=2547-2562;citation_volume=63;citation_year=2009;citation_pmid=19228187;citation_doi=10.1111/j.1558-5646.2009.00662.x" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Berlin;citation_author=U. Lagercrantz;citation_author=S. von Arnold;citation_author=T. Öst;citation_author=A. C. Rönnberg-Wästljung;citation_title=High-density linkage mapping and evolution of paralogs and orthologs in Salix and Populus.;citation_pages=129-129;citation_volume=11;citation_year=2010;citation_pmid=20178595;citation_doi=10.1186/1471-2164-11-129" />
<meta name="citation_reference" content="citation_journal_title=Stat. Comput.;citation_author=M. G. B. Blum;citation_author=O. Francois;citation_title=Non-linear regression models for Approximate Bayesian Computation.;citation_pages=63-73;citation_volume=20;citation_year=2010;citation_doi=10.1007/s11222-009-9116-0" />
<meta name="citation_reference" content="citation_journal_title=J. Biogeogr.;citation_author=L. B. Brubaker;citation_author=P. M. Anderson;citation_author=M. E. Edwards;citation_author=A. V. Lozhkin;citation_title=Beringia as a glacial refugium for boreal trees and shrubs: new perspectives from mapped pollen data.;citation_pages=833-848;citation_volume=32;citation_year=2005;citation_doi=10.1111/j.1365-2699.2004.01203.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. Evanno;citation_author=S. Regnaut;citation_author=J. Goudet;citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.;citation_pages=2611-2620;citation_volume=14;citation_year=2005;citation_issue=8;citation_pmid=15969739;citation_doi=10.1111/j.1365-294X.2005.02553.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Evol Bioinform Online;citation_author=L. Excoffier;citation_author=G. Laval;citation_author=S. Schneider;citation_title=Arlequin (version 3.0): an integrated software package for population genetics data analysis.;citation_pages=47-50;citation_volume=1;citation_year=2005;citation_pmid=19325852" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=L. Frisse;citation_author=R. R. Hudson;citation_author=A. Bartoszewicz;citation_author=J. D. Wall;citation_author=J. Donfack;citation_title=Gene conversion and different population histories may explain the contrast between polymorphism and linkage disequilibrium levels.;citation_pages=831-843;citation_volume=69;citation_year=2001;citation_issue=4;citation_pmid=11533915;citation_doi=10.1086/323612" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=E. J. Gilchrist;citation_author=G. W. Haughn;citation_author=C. C. Ying;citation_author=S. P. Otto;citation_author=J. Zhuang;citation_title=Use of Ecotilling as an efficient SNP discovery tool to survey genetic variation in wild populations of Populus trichocarpa.;citation_pages=1367-1378;citation_volume=15;citation_year=2006;citation_issue=5;citation_pmid=16626459;citation_doi=10.1111/j.1365-294X.2006.02885.x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=S. Hanley;citation_author=A. Barker;citation_author=W. Van Ooijen;citation_author=C. Aldam;citation_author=L. Harris;citation_title=A genetic linkage map of willow ( Salix viminalis) based on AFLP and microsatellite markers.;citation_pages=1087-1096;citation_volume=105;citation_year=2002;citation_issue=6-7;citation_pmid=12582937;citation_doi=10.1007/s00122-002-0979-0" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=S. J. Hanley;citation_author=M. D. Mallott;citation_author=A. Karp;citation_title=Alignment of a Salix linkage map to the Populus genomic sequence reveals macrosynteny between willow and poplar genomes.;citation_pages=35-48;citation_volume=3;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=Linkage disequilibrium in finite populations.;citation_pages=226-231;citation_volume=38;citation_year=1968;citation_pmid=24442307;citation_doi=10.1007/BF01245622" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=W. G. Hill;citation_author=B. S. Weir;citation_title=Variances and covariances of squared linkage disequilibria in finite populations.;citation_pages=54-78;citation_volume=33;citation_year=1988;citation_issue=1;citation_pmid=3376052;citation_doi=10.1016/0040-5809(88)90004-4" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=R. R. Hudson;citation_title=Generating samples under a Wright-Fisher neutral model of  genetic variation;citation_pages=337-338;citation_volume=18;citation_year=2002;citation_issue=2;citation_pmid=11847089;citation_doi=10.1093/bioinformatics/18.2.337" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=R. R. Hudson;citation_author=J. A. Coyne;citation_title=Mathematical consequences of the genealogical species concept.;citation_pages=1557-1565;citation_volume=56;citation_year=2002;citation_pmid=12353748;citation_doi=10.1111/j.0014-3820.2002.tb01467.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. K. Ingvarsson;citation_title=Nucleotide polymorphism and linkage disequilibrium within and among natural populations of European aspen (Populus tremula L., Salicaceae).;citation_pages=945-953;citation_volume=169;citation_year=2005;citation_pmid=15489521;citation_doi=10.1534/genetics.104.034959" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. K. Ingvarsson;citation_title=Multilocus patterns of nucleotide polymorphism and the demographic history of Populus tremula.;citation_volume=180;citation_year=2008;citation_pmid=18716330;citation_doi=10.1534/genetics.108.090431" />
<meta name="citation_reference" content="citation_journal_abbrev=J Integr Plant Biol;citation_author=A. Karp;citation_author=S. J. Hanley;citation_author=S. O. Trybush;citation_author=W. Macalpine;citation_author=M. Pei;citation_title=Genetic improvement of willow for bioenergy and biofuels.;citation_pages=151-165;citation_volume=53;citation_year=2011;citation_issue=2;citation_pmid=21205181;citation_doi=10.1111/j.1744-7909.2010.01015.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Syst. Evol.;citation_author=A. Kehl;citation_author=G. Aas;citation_author=G. Rambold;citation_title=Genotypical and multiple phenotypical traits discriminate Salix x rubens Schrank clearly from its parent species.;citation_pages=169-179;citation_volume=275;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. R. Keller;citation_author=M. S. Olson;citation_author=S. Silim;citation_author=W. Schroeder;citation_author=P. Tiffin;citation_title=Genomic diversity, population structure, and migration following rapid range expansion in the Balsam poplar, Populus balsamifera.;citation_pages=1212-1226;citation_volume=19;citation_year=2010;citation_issue=6;citation_pmid=20163548" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Kim;citation_author=V. Plagnol;citation_author=T. T. Hu;citation_author=C. Toomajian;citation_author=R. M. Clark;citation_title=Recombination and linkage disequilibrium in Arabidopsis thaliana.;citation_pages=1151-1155;citation_volume=39;citation_year=2007;citation_issue=9;citation_pmid=17676040;citation_doi=10.1038/ng2115" />
<meta name="citation_reference" content="citation_journal_title=Sven. Bot. Tidskr.;citation_author=G. Larsson;citation_author=B. Bremer;citation_title=Korgviden - nyttoväxter förr och nu.;citation_pages=185-200;citation_volume=85;citation_year=1991" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=M. Lascoux;citation_author=J. Thorsen;citation_author=U. Gullberg;citation_title=Population structure of a riparian willow species, Salix viminalis L.;citation_pages=45-54;citation_volume=68;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Librado;citation_author=J. Rozas;citation_title=DnaSP v5: a software for comprehensive analysis of DNA polymorphism data.;citation_volume=25;citation_year=2009;citation_pmid=19346325;citation_doi=10.1093/bioinformatics/btp187" />
<meta name="citation_reference" content="citation_journal_title=Briefings in Bioinformatics;citation_journal_abbrev=Brief Bioinform;citation_author=J. C. Mueller;citation_title=Linkage disequilibrium for different scales and applications;citation_pages=355-364;citation_volume=5;citation_year=2004;citation_issue=4;citation_pmid=15606972;citation_doi=10.1093/bib/5.4.355" />
<meta name="citation_reference" content="citation_title=Molecular Evolutionary Genetics;citation_author=M. Nei;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. S. Olson;citation_author=A. L. Robertson;citation_author=N. Takebayashi;citation_author=S. Silim;citation_author=W. R. Schroeder;citation_title=Nucleotide diversity and linkage disequilibrium in balsam poplar (Populus balsamifera).;citation_pages=526-536;citation_volume=186;citation_year=2010;citation_issue=2;citation_pmid=20122131;citation_doi=10.1111/j.1469-8137.2009.03174.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Ossowski;citation_author=K. Schneeberger;citation_author=J. I. Lucas-Lledo;citation_author=N. Warthmann;citation_author=R. M. Clark;citation_title=The Rate and Molecular Spectrum of Spontaneous Mutations in Arabidopsis thaliana;citation_pages=92-94;citation_volume=327;citation_year=2010;citation_issue=5961;citation_pmid=20044577;citation_doi=10.1126/science.1180677" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Bot.;citation_author=L. Peeters;citation_author=O. Totland;citation_title=Wind to insect pollination ratios and floral traits in five alpine Salix species.;citation_pages=556-563;citation_volume=77;citation_year=1999;citation_doi=10.1139/cjb-77-4-556" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=R. J. Petit;citation_author=L. Excoffier;citation_title=Gene flow and species delimitation.;citation_pages=386-393;citation_volume=24;citation_year=2009;citation_issue=7;citation_pmid=19409650;citation_doi=10.1016/j.tree.2009.02.011" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. K. Pritchard;citation_author=M. Przeworski;citation_title=Linkage disequilibrium in humans: models and data.;citation_pages=1-14;citation_volume=69;citation_year=2001;citation_issue=1;citation_pmid=11410837;citation_doi=10.1086/321275" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of population structure using multilocus genotype data.;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_pmid=10835412" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. E. Reich;citation_author=M. Cargill;citation_author=S. Bolk;citation_author=J. Ireland;citation_author=P. C. Sabeti;citation_title=Linkage disequilibrium in the human genome.;citation_pages=199-204;citation_volume=411;citation_year=2001;citation_issue=6834;citation_pmid=11346797;citation_doi=10.1038/35075590" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. C. Rönnberg-Wästljung;citation_author=C. Glynn;citation_author=M. Weih;citation_title=QTL analyses of drought tolerance and growth for a Salix dasyclados x Salix viminalis hybrid in contrasting water regimes.;citation_pages=537-549;citation_volume=110;citation_year=2005;citation_issue=3;citation_pmid=15619077;citation_doi=10.1007/s00122-004-1866-7" />
<meta name="citation_reference" content="citation_journal_title=For. Genet.;citation_author=A. C. Rönnberg-Wästljung;citation_author=V. Tsarouhas;citation_author=V. Semerikov;citation_author=U. Lagercrantz;citation_title=A genetic linkage map of a tetraploid Salix viminalis x S. dasyclados hybrid based on AFLP markers.;citation_pages=185-194;citation_volume=10;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. Siol;citation_author=S. I. Wright;citation_author=S. C. Barrett;citation_title=The population genomics of plant adaptation.;citation_pages=313-332;citation_volume=188;citation_year=2010;citation_issue=2;citation_pmid=20696011;citation_doi=10.1111/j.1469-8137.2010.03401.x" />
<meta name="citation_reference" content="citation_title=Willows of the USSR and Adjacent Lands;citation_author=A. K. Skvortsov;citation_year=1968" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=A comparison of bayesian methods for haplotype reconstruction from population genotype data.;citation_pages=1162-1169;citation_volume=73;citation_year=2003;citation_issue=5;citation_pmid=14574645;citation_doi=10.1086/379378" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Stephens;citation_author=N. J. Smith;citation_author=P. Donnelly;citation_title=A new statistical method for haplotype reconstruction from population data.;citation_pages=978-989;citation_volume=68;citation_year=2001;citation_issue=4;citation_pmid=11254454;citation_doi=10.1086/319501" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=J. Dudley;citation_author=M. Nei;citation_author=S. Kumar;citation_title=MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0.;citation_volume=24;citation_year=2007;citation_pmid=17488738;citation_doi=10.1093/molbev/msm092" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. D. Thompson;citation_author=D. G. Higgins;citation_author=T. J. Gibson;citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.;citation_volume=22;citation_year=1994;citation_pmid=7984417;citation_doi=10.1093/nar/22.22.4673" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. Thornton;citation_title=libsequence: a C++ class library for evolutionary genetic analysis;citation_pages=2325-2327;citation_volume=19;citation_year=2003;citation_issue=17;citation_pmid=14630667;citation_doi=10.1093/bioinformatics/btg316" />
<meta name="citation_reference" content="citation_journal_title=Plant Biosyst.;citation_author=L. Triest;citation_author=L. Q. Trung;citation_author=A. K. M. M. H. Talukder;citation_author=K. Van Puyvelde;citation_title=Nuclear cyp73 intron fragment length polymorphism supports morphological analysis of Salix species and hybrids.;citation_pages=555-563;citation_volume=143;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=V. Tsarouhas;citation_author=U. Gullberg;citation_author=U. Lagercrantz;citation_title=An AFLP and RFLP linkage map and quantitative trait locus (QTL) analysis of growth traits in Salix.;citation_pages=277-288;citation_volume=105;citation_year=2002;citation_issue=2-3;citation_pmid=12582530;citation_doi=10.1007/s00122-002-0918-0" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=V. Tsarouhas;citation_author=U. Gullberg;citation_author=U. Lagercrantz;citation_title=Mapping of quantitative trait loci controlling timing of bud flush in Salix.;citation_pages=172-178;citation_volume=138;citation_year=2003;citation_issue=3;citation_pmid=14641480;citation_doi=10.1034/j.1601-5223.2003.01695.x" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=V. Tsarouhas;citation_author=U. Gullberg;citation_author=U. Lagercrantz;citation_title=Mapping of quantitative trait loci (QTLs) affecting autumn freezing resistance and phenology in Salix.;citation_pages=1335-1342;citation_volume=108;citation_year=2004;citation_issue=7;citation_pmid=14747916;citation_doi=10.1007/s00122-003-1544-1" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. A. Tuskan;citation_author=S. Difazio;citation_author=S. Jansson;citation_author=J. Bohlmann;citation_author=I. Grigoriev;citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp;amp; Gray).;citation_pages=1596-1604;citation_volume=313;citation_year=2006;citation_issue=5793;citation_pmid=16973872;citation_doi=10.1126/science.1128691" />
<meta name="citation_reference" content="citation_journal_title=J. Ecol.;citation_author=N. Ueno;citation_author=Y. Suyama;citation_author=K. Seiwa;citation_title=What makes the sex ratio female-biased in the dioecious tree Salix sachalinensis?;citation_pages=951-959;citation_volume=95;citation_year=2007;citation_doi=10.1111/j.1365-2745.2007.01269.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Untergasser;citation_author=H. Nijveen;citation_author=X. Rao;citation_author=T. Bisseling;citation_author=R. Geurts;citation_title=PRIMER3PLUS, AN ENHANCED WEB INTERFACE TO PRIMER3.;citation_pages=W71 74;citation_volume=35;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. F. Voight;citation_author=A. M. Adams;citation_author=L. A. Frisse;citation_author=Y. Qian;citation_author=R. R. Hudson;citation_title=Interrogating multiple aspects of variation in a full resequencing data set to infer human population size changes.;citation_volume=102;citation_year=2005;citation_pmid=16352722;citation_doi=10.1073/pnas.0507325102" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=256-276;citation_volume=7;citation_year=1975;citation_issue=2;citation_pmid=1145509;citation_doi=10.1016/0040-5809(75)90020-9" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. Weih;citation_author=A. C. Rönnberg-Wästljung;citation_author=C. Glynn;citation_title=Genetic basis of phenotypic correlations among growth traits in hybrid willow (Salix dasycladosxS. viminalis) grown under two water regimes.;citation_pages=467-477;citation_volume=170;citation_year=2006;citation_issue=3;citation_pmid=16626469;citation_doi=10.1111/j.1469-8137.2006.01685.x" />
<meta name="citation_reference" content="citation_title=Die Bastardbefruchtung im Pflanzenreich erläutert an den Bastarden der Weiden;citation_author=M. E. Wichura;citation_year=1865" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=X. Xia;citation_author=Z. Xie;citation_title=DAMBE: Software Package for Data Analysis in Molecular Biology and Evolution;citation_pages=371-373;citation_volume=92;citation_year=2001;citation_issue=4;citation_pmid=11535656;citation_doi=10.1093/jhered/92.4.371" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Polymorphism and Divergence in Two Willow Species, Salix viminalis L. and Salix schwerinii E. Wolf" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/5.cover.gif" />
<meta name="twitter:description" content="We investigated species divergence, present and past gene flow, levels of nucleotide polymorphism, and linkage disequilibrium in two willows from the plant genus Salix . Salix belongs together with Populus to the Salicaceae family; however, most population genetic studies of Salicaceae have been performed in Populus , the model genus in forest biology. Here we present a study on two closely related willow species Salix viminalis and S. schwerinii , in which we have resequenced 33 and 32 nuclear gene segments representing parts of 18 nuclear loci in 24 individuals for each species. We used coalescent simulations and estimated the split time to around 600,000 years ago and found that there is currently limited gene flow between the species. Mean intronic nucleotide diversity across gene segments was slightly higher in S. schwerinii ( πi = 0.00849) than in S. viminalis ( πi = 0.00655). Compared with other angiosperm trees, the two willows harbor intermediate levels of silent polymorphisms. The decay of linkage disequilibrium was slower in S. viminalis compared with S. schwerinii , and we speculate that this is due to different demographic histories as S. viminalis has been partly domesticated in Europe." />
<meta name="og-title" property="og:title" content="Polymorphism and Divergence in Two Willow Species, Salix viminalis L. and Salix schwerinii E. Wolf" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/5/387" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We investigated species divergence, present and past gene flow, levels of nucleotide polymorphism, and linkage disequilibrium in two willows from the plant genus Salix . Salix belongs together with Populus to the Salicaceae family; however, most population genetic studies of Salicaceae have been performed in Populus , the model genus in forest biology. Here we present a study on two closely related willow species Salix viminalis and S. schwerinii , in which we have resequenced 33 and 32 nuclear gene segments representing parts of 18 nuclear loci in 24 individuals for each species. We used coalescent simulations and estimated the split time to around 600,000 years ago and found that there is currently limited gene flow between the species. Mean intronic nucleotide diversity across gene segments was slightly higher in S. schwerinii ( πi = 0.00849) than in S. viminalis ( πi = 0.00655). Compared with other angiosperm trees, the two willows harbor intermediate levels of silent polymorphisms. The decay of linkage disequilibrium was slower in S. viminalis compared with S. schwerinii , and we speculate that this is due to different demographic histories as S. viminalis has been partly domesticated in Europe." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/5.cover.gif" />
    <title>Polymorphism and Divergence in Two Willow Species, Salix viminalis L. and Salix schwerinii E. Wolf | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"famJMiVCvaKs-SEMx4KF_R_1b_Pfhe3VtJPGqpIvXR4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459188","apath":"\/ggg\/1\/5\/387.atom","pisa":"ggg;1\/5\/387","ac":{"\/ggg\/1\/5\/387.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/5\/387.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/5\/387"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/5\/387"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/5\/387":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qS8hcvskSJ3Xh4LXdGIAgoDzR4Sw+XuuTJEPVxQHgLnyqlXueLNRckbOgDutvChM2Vdz7BEgZmsh6+smzlcxtPw8jcsRKXT99cXBV9sllkYJsZO5rOB3Wz1dPiHNlWkiSsIXtWPpcoz+yKz0GPBgDcwsdKjkDzpYPrEXpHrxUKrXfIovILHPZle9UWA\\\\\\\/stX8I5NE57aMvEnGKp85Z3e6z4qICpBaXp77HyuCN71M8Zn7+Czy7afopBdGjlVgvZ0qgjjYNp\\\\\\\/daCHVuWU\\\\\\\/H79eAun6rYiQJbu++R4f+dhTIVPiK8sC40TRB7jJ\\\\\\\/1qlOvKs7MMiV7rSE\\\\\\\/awkpMDvcb\\\\\\\/ZmuaswYLNSroamaCwyoV2e8AoieXGby01nKrZEd05P+tBUhbxJuV8iLGq0EI2WsGcxK4yKDqd\\\\\\\/yELVggquLcCygcurjKzJE1TJVio21B3Q++pX03u9IweDwJIK\\\\\\\/itPGwOjWXISQogY9t9HF6yOeOqCzMvTd303I3fy1HigOpi7o1qpr5fQbyauKIuhUYX+OcwD57cQnaLvi0lqObHVrGK6jbty76HslZZpEQYrhTjLP\\\\\\\/lFVzpuEZxr4t\\\\\\\/x6ZlDb82PfxXrOAw1Ma+KTnc00o47BoraUBzz6NRikg84yKOE4+N\\\\\\\/aKm5kDRvy9cvQ1KGqXL3GIgDGMkKfHV3PXoCPftou+9udn+MxoWn5eGfezKNl2FRmfS4fOveCQr5swqUeJw7ZrNpOTv+MunIUX9Y0iOkeczooUxc\\\\\\\/580bJg5JX82ylnWv1UESCOErXK4ZfRCgXS2XI3FzflXAaxwvaisf8+AJVuLXRo8T2hoIL82yV8hLTJqgaqk6K5qXyjyKEnpJYmhy9x3grCRueTIHec2qXesPp0I2o\\\\\\\/QxvaX10izIiPC+NVi24LJn7LbrHk62sVd2EXKJLUGf0HocAhzY5IB5A46Q7z+dPAF8nS8VjHr1XVhMkO1n9xGowDxpxql64VKlqjAgXx8j8pcXHRwJW\\\\\\\/IDqMM7zK7NGD0DLoBIJbvTlwknwactdL7JbJQDtQxpSLhSuCPrwmECKWkDEHdbRIIGtkaA7qzR1gNcy31HKeeJ6V9\\\\\\\/OA8Y4avfzFZKJjC6FVVR+UUhVDMO\\\\\\\/ov6sdT4lHNcxTCFJmswVNQLoRpDjb3mU0FPWNOccsc51eEk75T61xrpSEZ7rI8Zs+JkF2ZBM1\\\\\\\/jvoY0x1YmWdrqdhw0COxuIpj2ig56yQhDi896T2LzBO\\\\\\\/z4d\\\\\\\/U1yef0f+mKbdaQvmYXAqDyypNA16cwYP+F1Frn9YS2\\\u0022,\\\u0022iv\\\u0022:\\\u0022nb\\\\\\\/jahTR4y7Zm\\\\\\\/y4dZamLg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c61143c6101885624a5ceb072e036ee4a45d8bb875246315ed0bd50b3f3dbc12\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lSL0NkystwPLVKPkaqxxkMpYJD0\\\\\\\/5pcDqMdhn1vgLyKFlo5bpONvPb4q\\\\\\\/h0iYLU1CSB+KdaN76AK2chOPlFuksRZdZrJpQe5FRI\\\\\\\/6wpL0OfOWz\\\\\\\/VkRlNVzbj8+Xv68N0GJ3gxpjp7Nz\\\\\\\/cu6L1ffZrls6Xv\\\\\\\/0ipj8RwMBNkARqmqeXLhIa6mJvu8UJXR0NeSbBFIqDqzkAVEFz2IKFnMNd69hccbTppN\\\\\\\/h3Ja9FXOJydCg3pjx3aC71BM9MY65Cab8vC5eB7P8SXYIhLDKX9r2g+BIsnpsU+1S\\\\\\\/TLa2IRXjvj+pooje8duXQXs8v4lPufN63Mmz0xOir\\\\\\\/Fk8Fk0RMG748suVsPtKDIdr+T7myfNBZ\\\\\\\/C2XE1QpdMzi0zuyCd\\\\\\\/LJV5X3Ex\\\\\\\/nFeQzpm15t0ajNQMeN6fGx174W7uCLjVUnxq3NSNCLyI6I2eI03QdkKMc1DmYUgSCnzEQJ9SEFcJX44iuV9OIXDZz0LEm2tQfziwwcqqWvXsnDHO4G8IlhaGIoIWCgcGB5eaUKHuEHwWVtD2YRn4QBGrNHKnN70YgwAGpiguclKg7USM6o6t6mdWtRBSHIRKx+FIqUBdz1xAGgG06TjEbK\\\\\\\/nSyZ0M6bE56AH6CcVA6XiNUNUn2Y5H\\\\\\\/PwvG12v8Uevax33vHVSZv+BID\\\\\\\/Mw\\\\\\\/JknpzG5PQwKPj8ufLEsVVOKv+KnxLldM32gfbpIYeb\\\\\\\/4YhPuxeIoaYSKiFQVjwX+YcgPlPSBGYxPo3tJqYAG+P4HQ30UlMGcGfaZpGVhKTXkhs8wC7HVYbglG1TxJDJ4aYUB4\\\\\\\/28Z8v3ImOShtCpGDuMetofHXAtxQZrYa34bhcNgLgtLDC6xSL\\\\\\\/dG0n9r94z0VrZuKyZpYSG2vZBz6PBpfX1Han8E+gc50F\\\\\\\/iIeP42CT+IvE13K60jjIFs7XTWkee\\\\\\\/OmqLoUVXPFnq8KFMNzGrDuR2YesqiapyPzCQtQNzVPGy5XXZ24teeLdgLLv0QmjIPlE7qjfZtSjtG\\\\\\\/j5HYU1SVgnuP9hPqyzQOw93D+yc14zHPF0P6pKZyOgNnkRjGwRKRztj3ufmolhAGRTxs7xEtJSCMfr1X332o254Q29UqUstxA+UDRHFt577nEER+I4vznub5bjI30V53sopaPEuQjda1RP\\\\\\\/eDYNXTyqkR+mcUK0t2dHPD9i4XlYASb8iS3BgrrL3qCAnNTUzKV+2H+f\\\\\\\/cMmX4fHlhh\\\\\\\/4phDeDlB3tXyYWT6ln7ca8AZZwYzN\\\\\\\/c4N7MVU\\\\\\\/EAj0huUwG6Mx19ilMLjvkVmuH3Rvk+cPFY1nZcX9dv1KtNYO2\\\\\\\/1mj73oAWww77i8CcHv95\\\\\\\/iSNQUysWTJ6adOupQdqQaKRLE4sKDHTk+6Uml2qut\\\\\\\/X8Qtc67\\\\\\\/+HmrjaDjW0PI+YGZ1t4YmH+\\\\\\\/rcdIHKfWMO7OVLVoorDtByc9V5xunaj6FxdGLJOfD7JCwsCy\\\\\\\/2TOgwTyn3vMYepFtH\\\\\\\/dTovJeQmAE9WcPPpHWKv20TL5epaAo8PpYdi3vf\\\\\\\/OCoY8kT5fjHpZtPJzBA4\\\\\\\/Rbnw57JP78jxTV\\\\\\\/JtudYZp9seOyk5FkE0HdqieAqOluCJJQkeCesxN78q+Kwp7vUNyuvaCc6cDYC0M2KGrVMz2RzAzAIzCxiBuTv8\\\\\\\/TV80qoN\\\\\\\/JhnNYge6tMhSTf4BfaL35fIVjOMObAdwVY1jakstNLn8kiBx98KgW\\\\\\\/Hj15S6AEL7KyfMP94pOg\\\\\\\/B596BWdRjSPPXiOr\\\\\\\/94dgfkBJZ3zKjiZBKK2B6cOxWxR70NDu3Zyo+BOOww8zBJQ6ntzexs+GHHR9fqFZUPf9Pty1dgNHyeVjoKCUX6tTxa0ouBrX5tmsHbdDupBOT188jlCIxouu\\\\\\\/7znCfPBG53DbTbTkKy39LfIvvLjF5o1WIbKrUkiceQyuph51d8+ooum+KYlHEzN4IQYf2p+9fETXyiMupp51zNS0V9\\\\\\\/mpx7n0R346W\\\\\\\/GWhaLDyXtW8mK0j7Mqh\\\\\\\/hc8ILEvrQHC7jFbe5CAGWNOiXnLcBq1HfLhzxvT4EPSxewOriGT78+NihT4RIeAuA17Q+C7mWbX4ykClBeCPxTpcn1IRPCTvUHxJTv980Dxjuw9sCUrhz1E41nttcjJmCrY+G\\\\\\\/aKewgic+PT4CNAjBoEAp7QbVT9PjCmaPRV3FOxNy2SEu0\\\\\\\/2nZFgtstY8QeS1XiUicB4A3fb+FbKIPgzPi+0YhJO23klQgVFRoukqI+TBZ41gY9L7QuCEop1UkT3+CUWJ6j2trSFTK4QBE2V6kZcsGFgDltzamt9Ck4zh8AbQc+iCRm2n0XPcQVR7fkCtb524LSnwkS9Et6VcQPmp8Iuq+yFcKVZ9+j3WoUYoVsPM0PoW1cBUZk8fO5Ug\\\\\\\/vSBZwBRGwiHmtUDpqhqD4Fw+Trmf3PMLyypxSjmaFFEeFGQVaRdNr8xy2LrCLVcmsnPGiX0rW\\\\\\\/0pYfIjPrN15NQskEqhaC9AT4M9wVKT\\\\\\\/8p0+cVzcYZELHTGHIRxN1eXJQtsxIx6iUdFIwQsTJJJPN0PGc5PxkznS6iwQD8P5DKyimb96NBkrTs+F5QsM7qzsyKkjFSvRavg8HTZGf7ddDSM1nxjpe24rSJtHCymcUtvcY481NEv9El2Ofdtg\\\\\\\/obIU70CQxEiRdkfDKFij8+VFwcIEATWwVIb3\\\\\\\/ds2kUZxK6oaEicCgsjGBWO3GO7B\\\\\\\/zZd\\\\\\\/SlQZj+LpyvASmFcB34qOxI9A9ChV3EgRAlO9FN8F2rpuSf4J57auP8YjLIuMdoO3dwMXaOPSBl6X7ZKZrIP0T0whBw1mH9o1ixk9s6hVI\\\\\\\/H+ItzAAx4\\\\\\\/i0Jm9XyhbhgmujqMqcDteUjH+4G8sEcAwWADKtJc9d3ACudK0GOGvZu8Hfwyva4QHsYo+GvwBpSyXuIULE8Ns\\\\\\\/HmwxUxW6iG3owwbjXWZACJsJz9CnRlbFT1tCt+q3qR1tCRi2XRzZa5OkAWuID9LWkWrcd5sgzGzLk\\\\\\\/VooDbwRSWm4s7xe5DVsk6Phnl+9rFcibG0F5nmuQzE\\\\\\\/YUOr4OwJeXSEGQxokPe4Gh\\\\\\\/N0Y1jUmc0IPXvst3h5VNRSrzQe2gXzCTvrfY9Pb\\\\\\\/NcCg0XLpyms3rL5mFVzi4frPhVVa8bMNw7PCSDndb+FGUjVB0seWie6W6ql2VzLTyr8NhtlNjH\\\\\\\/UHq0KUS61CBbinsIGXQXnLIK8CNgCKqsMFVnEHijHEVCIJId9\\\\\\\/9ZWnfP1sTJWT5Rm1M15IwXsiMb5CKK7kjFVr77+dSnejivGZVKmbaj9e3ztsTlYrf+0FRQKC8qEl2HaeDHO4sKytv2AYmGwpvDBRTF2UqTg0\\\\\\\/PQRx\\\\\\\/MyyAnrj0CStAynqQqr1dHQwNnEwuB1nQccGn+nfxfoNJrjc1C7SvJJKdyPwm6aEzxwCyzsplIoEuq2B6jCntKWTR5CezdAwJFFfGHkH4li2TvHKg828ez11iDEHfWmAmlvsERWMf7PK0meUzY9ASzV4HaNq2K\\\\\\\/cCQfMPLyoMav0JRskf5QGJ0w2yyNwVKERdhy65WWdjfxwlwOAIPerQokVfqbbWDKuJXpZbfR8Z7mydxFwalOYYRd5YhImEYVLmY8ouLZmAXxYIi58fJULcPRPlHGh\\\\\\\/I+nBJzIlEUCLln6XSJqnxU4fukUfcKyTybuB3X3dqUDUURmJJUPTt4Hh7h1eot33XJeTLdYBdAdG5KRmZ879qqznAgH8Ds+sy4J50n4qVvotCZgDpnyS56wkoTX6garp2tMO3X3jrwdVw36f5Or\\\\\\\/zxvyTGo1jDIbJXYwcSKSEZUrsznUupgVFBiB79CipBEjjXQYHWQUP2v9Ds5Z5wT0f0TmI5vaLrcsUvWEjnn7nU2xptE03zWRuhDJahBXklnw5Doz6hrCLQphA230amLk4cCu3dmBY6RrogjMj2DqzlwmaFqnd\\\\\\\/gYJFpShjuoLYTcAtN7\\\\\\\/AIPZeQydfK2kLaf0XpUR2s2aJX7yjizbO3I55bF6y3l68EQejPBS5YoiQc5Hycq\\\\\\\/j4FQGQvAjsIBy5YwJlWl9IlvDqVKQAI3Z1pxaVLrVUDIzvqr\\\\\\\/tXbmDZdl+uPtETyYi4ljdz+QYBNszMf8KdAhHUDrhggBkDQ\\\\\\\/GisAv0bYCxoBcV5gDQiRcoYXu0mlsmK95Q7GNE\\\\\\\/ivGHr4V5zRU6E9XXHGsB9G3DIaZwheGemY7BgtVasBmEmAirZbdfxIa+\\\\\\\/23M2On8kLF3LqWM3glpRmQJg8e4UsqcNsmpNtLpL9rdnM\\\\\\\/nplFUtHwV5bls6vcheI3H3leTt6JXEf8M5FxuYwsPQ7kCyiRKHz8VX0LmGr5Tlhx4K1kNz4uuo7Dz4nlJCZG3f47yVq7UZTYq6AHB77eZiZWv7iSkSMpoNVyzvYkTU3TEC8bnNxxahoG9dvSS553XcYyRW5N3EhiLEqh80Ie0LgaqnoxLxDxvqhPvoNKGoemswLdW\\\\\\\/yTOaDQ+43XW22y51EbnOTLpFIVpWedsoCpW4y62uyp5ETT+t7vixb7qK1BF7I5u43TjZilA4hVaMTrXlcekJpIGjVGK3K5gRZ0R0P2+cWXx8U3r7NTt53zu+ddXY\\\\\\\/6VmVP0El6SkKZyTjWworxF1SBdMVC8PwKiY+VSDg+BPSSy0vRpamGqGSp7UFsiy4dqGa0Rs1N8sQC5g505f+VTm0ZOTupJcP8p2zTxlSe3ie8aO2lrV3he2a+Ih3WkV\\\\\\\/W12P\\\\\\\/+ujU1SRS9PyuUt41dYri5VYCBZBtxYeeL9EaSFF55lVc6duRGc09P+uoYInr2TWZZv1rmajwaGWtUkKUllcvbcpKegQmDI7toO46OBh2O1CIprHTVoGTjcVg9qkpxGUnm5WpKyL4AhZv8ZT74s1Vm0mgJg2TTwskeP5PsiJh6P0oL7Hewy+DW5WsyJsqfeGOpT61rSc7BAAUD7HMilvloh0eFbFAS5PhNtUaa5JPhQHM+0cxyTPnqWvz4DJuemdhkuADM5bFzPCuBr4kOuw23kvRhA9Ew8yZCWPGSd6hcJK26xQBgPXNS+6T90\\\\\\\/XTyMrH2p457MGkUAOlTZzOhpZMoQHHRJp0VR8iZ3Tvv4eiHlYAa\\\\\\\/yijDHxrUp5AlKXihsOFaoeucOT0+6vN8wTi74mzjSjhCqE5zytybhe0R+MlZLxKmv09qb6cUnHfDOnhZU3aJVeqqo+znPmM9sooy+r8mvV\\\\\\\/LW+9I7DypDX6FAK9cmH0OQfMauoP3Ypp\\\\\\\/GCtMFkXwFAzn9qNaZ1ZkvFTg3PS7F06PoQ79ubxNKIQbe87QWXMYsMKOWvkzq7Nt1xzPPlxPRu\\\\\\\/QektPhDmsua3fouogjpiggQmkiC5MNki\\\\\\\/PVNrLRZ0jncH2WCTr+HoUCMfpiKLSzlu+EDZp5Zfl\\\\\\\/j4IyKIiZFqsK1i5I1hDyQDETQUo9FJTUBrBtLzjaF+6u15D0uWK4dE0RiG7OMCObXXH8DBGSwXq7SCku8KxMoUVvRcccZBfSKO5BzjrfQG+hpqFQ03KkFFjOdJHyVEUvh16LKVD\\\\\\\/1aAGtesthAKnLC0u62FbXRSey4H20Ezu5G6tIjEqFLSyboQWaZydNg3PZyVghgJ6C5b3t8ZM6Y\\\\\\\/tzn65kp5Vjg7EDz38xPcx99YqjilKx\\\\\\\/ceIw2aGom1pmNEwSm\\\\\\\/5VpOQWBg3ryAtlq0dDvUGpHAQOugS7uNwlVkzQZbOdvb1uMwS7jq8T\\\\\\\/w6rG8UAmQbxvkbb+hCrgJl8\\\\\\\/iwpQaQH3F1y4WceP2adBqkDUtolTEi3v2rvOtBBlSDA7\\\\\\\/59xc+e9i9QGXz1O4Bf9mjaxmzktFJyPsRHd9QPDJ3HrOhhKYCMAXndTE7qhmDgNBsErZJJ6Z09D5kvqLeEEIJ4Ow6ge99s4PEhzzPioVBUrTyUXxFGw4ZsEcQaA2Yfkn+Ce4jwJDpcu76M0lkxbccf++qDx1Y7aBrde+W15VDCNUky\\\\\\\/MZdvL3WDBJieToBUtIEz76mqxTPEVuw1pMaHn0BeOrwMqxnUquGOYUymXrMcoGAFw+tEpqDw7\\\\\\\/HiT84mg8FNBBN3CdrLgz1y\\\\\\\/VyMI5yB7IKmT3QEmGTLGC2xsA2XMMaEFlsP11v5j71LN3TUqGQXbRpku1cx7b3y9MAHbuWgiQOJfe0j6QlQO5gekviqm+l\\\\\\\/GBcutsztmCuKj4FPhLN8kLYosmgMjsgeCj+98PE5eHS3qRIULqHA4MO2dfR0TPPxkfhd7xFWXO6+CKWE3zWha\\\\\\\/G\\\\\\\/JgKau0sBoVVgbmer3gO3G7LQmNFHpSk8JCNGBSGthonDfHUBN3irTbQqzb7qQg18Ne31hynnfnLeGCRZMH\\\\\\\/0Ldf\\\\\\\/jUJLUDu4z\\\\\\\/kMmNd9i4M1EwLTJ6p0b9az1owKbadR6tXnRUZ9FJMA6lU9B\\\\\\\/pRlVFRcrDTxIGqNUiODLVgVZipVxhCup+47HmYDmtpaTZm810xfvZGcw26yGdwpECnVy5ct49NZuaQvslB0Y\\\\\\\/c8OS+5pi0mz++NU4bBCyLXZ4hFob+3Tnjmwp04Pm3KobeaJmMgBLbcOCqnaQR3NSal96fTIaZXm0Mm0ahsYbRb4a+gR7h5wDkYwNrOIszQE12qe9Ok252VEu\\\\\\\/vjvOvniwnNvzZ1XFkMVCbzZJsXNWzG32W6dElWRJUqI\\\\\\\/c00hhWSxrkBXGRhQr1cfPUXaGGAKIxqDdJWzTExcqyUWozD+dALdpxPvhfvO98jkMmMAfUXUL8GVr8VFbJeiEuUqmumNYCdPKnlcCTyrJ43godHsbdacM3zIn4FMj6pSaTfxYyNElK\\\\\\\/n3vDyv4MtlWFb9bHiuqVfvvCOSZ0p9jkZPkPRJ+wzyvdtRBwUKPf+KkN9f751WHnTaCVcpIFxTCjdAfN1QrncdWs8rpfwUTnIJJRME7GZZcEM33wMS9m7p\\\\\\\/higk9cQ\\\\\\\/sKqIE9hxrKpZo8eYn4orwV5GR17U8R0\\\\\\\/EG4w4EfKNw5T+2ss1+u3Guhi2uTpVCOAFVqWq6DVD2oB6eOHXWCzxFGIA8GXThKEBFaCI9Jq8u3gxEpQxab8vWuJ9+wHE7hcEngwDy+m38aq35NLJCmCSOdPv\\\\\\\/pEt4UKgHdN\\\\\\\/wZauhDvc3iqwXC+2+zPne9cvfXMFDcv3XPEnLm08MEoEE1898RyWfWXrwaU+8XpMN7dAQWCknqMrddMh7zLFDXI6GcRPuIw5CCobZknUUpcpEaOpy0ZoLp7CVWp15LW+6WyGePlWhk5B53z6zqvtjWviQK1aVxMyPEE3zLRChCzd5099x5XAV2Z7xmwTrmLlYFHzzI3Tkuq87rwZfd+9USHFEts9XH0YeVCA0RDEwNtU6VMnh4PkenpOMbE09+wdwjx6WxcrTdC\\\\\\\/c1CF44beAGxQR2606p1KXm2WDRJO\\\\\\\/P3IuGruavY5tSQvKbGMbYwgfoomt+eWTo8InBQq8KQNYYQ4lIFhkR4+zhuea1cTcPiDz1lHXUWBSeOD7Kb9WOmIVvJagwLpXr16DGufCTw0cJ4kscUvduzX+dVzav6p8N\\\\\\\/NuVJoaDHk7HXQkfVm+MjDKrUXa0d8JQa6ssIv9WdHm8kQizBO+3KQMk0zpOpLCDGQmONFPJJ7GckW7IURslKNXQO81edcEaMaikQJGBpZQ6nY7Gdbpn6ruxj4vNARgXKpVTounpxjADb3mEAFU91u+FrGLOUWZy8tZMMYP6I3776cMPPxBpox\\\\\\\/JOBQaLEIUpX\\\\\\\/B2\\\\\\\/C1hi7Zp3E6pQh5a8aou5s+YpTos48QJucz\\\\\\\/\\\\\\\/iITd2suv\\\\\\\/URwsHLN85kCr+qo7fbPpp+rbXzHNmtR6ZY2pkMrAqfRFc0WuIePXHv+qhMTII7tuRxP1gyzaYAm585aj\\\\\\\/PzdU+GPrLWn1\\\\\\\/kTsTxzkQ84ktcjE35v0Otxe6A9dz0E1D+T\\\\\\\/ZTh8QXdOO2w2rb5LjOku9ao0DjQuUEDRViNHu+5ZwjqobDQ1yxCeBfmDJUxw5qffmJkkt29c4QjOPl+WoVgCWbvNZXPUrcs6WLtakNFG0XhlhGaQy8bkjR1asfd7yd5JNIitn+lUHgtbAwoyuNj6Ez4mcXRAq\\\\\\\/WNTI73BR7WJi0jmAwPZnBhUfPFbNwahyzNTuMW00Lg6nEDd4kAwysQWAzjoaKrtC4FjjF9w6pR8UsCaECT\\\\\\\/ZaqHryyV+Y3Da2GZvx6V9dwlU98eewYCsovkA71mLYENkXTtSq3hw7MFysZOJ6AY1feD+YkGxCu0N3lrsNj\\\\\\\/NIUymwTpkxkHW\\\\\\\/X466CHrDKwJOqwXiFjFt8E03j488MljwqRAG5IomdxiD67Lwq4FxFEeHDji7uoRXG8Ma2Hz7may34WGW2+JOO6XF9cvtJ\\\\\\\/w0tFyJAakYTT1F16McVSAr9iWEZ0fa04OFVBhTqA5pxC5rlJgcUBhcna2lQfYxa\\\\\\\/vnCB\\\\\\\/4OFTJUIrjtFLC8pwWpKHKtqcS6FkYz6PyWec6wj4Drl3KPTkvNKpkshgIPRuiwUUeV1Xpu69pJndY1dCaHjIBmDQjA2cfFt+DGU41lxWMfOAx\\\\\\\/26kvaAyXiDCz++De6pNshDHL0mCjowid0MD4iQWYHnhQJdgPgzvLseG0xPPz+c8aopyguqeoDes1IByZvBIpDJ\\\\\\\/NKReNoIBwp\\\\\\\/johzE2Bi4aoeeTBlEAjp6IKqts8UbeHbEkDAw+rEpV2WhqfvxRdfOsItsAX9\\\\\\\/vcD+xNZVp9Jh+kWMKzMsWtFmRclD5YHk0ysfAvDivlIfppYBQZZcRWE1spgArsy2ui2G5eGdcHDAlOXZb0qmTkhu2x9ZvPrN54w6CtH0TZfOCmFbZfwCjFSisv+R3\\\\\\\/ezHUZRqeMJR4Pw9n67gizj489xnY\\\\\\\/0smyG6mrHsQJuzndwWF776Qk\\\\\\\/vaZNjFpDH2SPq9q4rtiShJENAvvEEZ7R19f7fFV7ULerxlGQL1HfniuvCHCva3gUHQvZnKVHj\\\\\\\/llw\\\\\\\/oCvvEXd6NQyMBmDrnEOwy2lOX5ndE10ZsJIs7fv08v+yxjJ6D\\\\\\\/j2AnzCtBInkARz+vfzlrRYRS\\\\\\\/SmJ9mYsX0wO9nifwAs3TRAhC\\\\\\\/PfXRsDh4pb\\\\\\\/GVXgsESe\\\\\\\/I6\\\\\\\/NicnPQqpaoezrXVtoE0pQQJMgTKhwxqNyIvSxilUFQ\\\\\\\/CGyM48CNhW1GFlVDs4gdecre4E1ZFZo5qg9zbCuaro+L+1brV+IWhGXjtM+29KRwfxDiut5sgCBCrDP3P5emp+SO9C43akgpPPmVZWTx8yVc1mqR0EAf97uJwwDnZm1i6nqKw\\\\\\\/KIxakyg4uEMhD3BkNvaJdz+1ZIYiw+9mDWbPDgpYPpyshb5y2ZYPk0IxtgxUeiQ+oVnKwWXOseWcfS6jBxbuTXBMzurxtACoz7TBoNgMnlJll7UI\\\\\\\/VkHqzpmDoPnXWLc89MY+9sIt8HrsvXLYDn5jaE3sEEc1TxrpqxSuUewjNeK+hUFWhaojtxBkZ+iVa059BdQl9VMeeXQWBS8QqaChVJpCABZeDw3546jH8+LJ\\\\\\\/MofEQc0712y0ijbmClfAgob4eJAkRD0X9rhSGkJX893kib1BxQly3CpbmQBSDgoZ53b6L7zWLVSUO\\\\\\\/IgI2Gfb9I0pHIoMry\\\\\\\/9QrQryXezFdqTvPp361rRicXEKlxsNbsxIzJh3G+YIF4NkdX2lam\\\\\\\/YoFucAMfHR5rpjO7uqZrB8goKaB3myidtqzvTS+i8BuaF5okE4Lv7eJjaYfNFf8QCzIfgbrZ+1QI1O0KapQHvMD2virgip8xKvCzXuaOIOoEgXN3jPBRxx6yJVW+QfDtzxjVCWRHWYTkxRtrDN7d7VsT8M0gUA7NUEU\\\\\\\/Pqe0uZOrtRAyLFbkEMHnAsxjAK6q1YHk096jAzrl8Gr2vzGeh7WI15ymIf\\\\\\\/vV4P16jjJLccTYxqCWjrRY1rVYIPRespAfa2DD5t\\\\\\\/FJNudaFXfMwsS5QqfUwso56fEa3t7O\\\\\\\/xaENkhMBRFjkh5BPATCYQWRsMVEcCmUkDJyY8EI16Mbah3q21RJq62AZxAnV62vOMAWzihI+EhQ35MVladeN\\\\\\\/RP9VWHseno+pupUQAL76hYVPrWxTv3eQQCL0pujTx\\\\\\\/JNASp2nGpTQiyYMWv4QNqDmx6QWMzWksGwALo+AjNv1vs2\\\\\\\/HHnFccqpsDfwqpMFj7z+lZ+7gtbcuIovnAwLfptmDyJqIuPGIuxEuoD6icfCxKZtINspW3kszPS3JM6EzY1Szryp6OmYxx7P20xLZCyTh7G34qN7qKJvMNKiEu4KlDBbvQQE7f4zs2h64NCvgS\\\\\\\/bzmGaikVK\\\\\\\/nkG7W046xann8TtwoAs0EUN0DwyWEb7Yxu7MXsCwL\\\\\\\/PUpn6F8+RVhlZTU5ONnkrrKFKpHQCoRfG7hP91jO7uglcZobyTiCwYvxzLb1l6mQdKAVOkjxJvWMcm+MQDSqV5OzpV+KarmM9YiI6evPVDbIUQOPlBW7drE79r9gZDJ81fze4xJEICS+f35NG1QGt3mfhkp\\\\\\\/FfF4c7\\\\\\\/JHVN0pEzKrTxuhfM5vPTjzop5QYIP70vHxoXP0ckxaN7UR0t\\\\\\\/WYd796yqH4zg9hzOOU+\\\\\\\/fDDPEgoGw5AckELEnkDaHNjP+btDfDy54+pFtBNlPNr6zqO8agIx9GA6o82nwtW\\\\\\\/AGmH1FRJ0qvDrkJ\\\\\\\/ipiouFd5cR8d8iYZ5QBow9VkBSVOPm2x4oE6YWYVUeWCaYQb5sp2RsXjmrnkKxm5cWK6IfMIFNEileuxqE8PmPeq1yjhE2dD60fyn3a7tKUIe50zpleuGBRSF8Tgf3gLR34Oe4hhTOY1XmjaXyRc\\\\\\\/cHMwqLTwwMWo65cDhCCM2093l52Xur9E7lPEJe+ytd5+Ufgq5Xf58ofwwoYfF4yAWlLieK+V0ttD5Wgno8N1cW2qvBvt6e4asnbwRaFVZlr7IkmcP3VLPf+RFv4aVlAqzApEfdfL1PRcC\\\\\\\/nPXXPy+YJIjxDKFLZQkrq09Xb6T3evbXcJPpiz13QnHpo\\\\\\\/0hftFiesMCr0OkVEybjHw8bmjwPulpUFYjkpe3016rBYln6Lr3XLKnjsFUlv3PAxFmJQXJOufRtoDAq5Ug6NflWTNSrmspIokfwH2KFMhTIsyF6eDmikR6n5yEPaMbWnojGs2XgqCFPWeRpiDalUxnEmD1M4cl2C+NSyn+Do6NGRPAUGdoScSJiVtnrTE8s5FANsqEJ6TZofNUZj3N+05wcuyAHgtWEF5T62u90BwJHXpmmCraqngYsEDMAeyNexqXbQzuKjwchkdpymFSGnVfyO9t+pHYD4\\\\\\\/orclo5wESHHlgodMaKp\\\\\\\/l9rCPGIp6mtBljNOp1BODdl3EpDYHVaNHQH7wVh4SvYWN9Ejgrgw\\\\\\\/Ry0Oag30AuaqkfXkRVVfG8YBjOhvge2rYGnhkP\\\\\\\/K35ypnnJysZDqfkeBYVqXSMu\\\\\\\/Yd8is0xpXJfdw1YnvEzAkXzxqXINKUA\\\\\\\/ShSBNIChx7Hy4f6Nhx4xjs3Krn6TLUu+\\\\\\\/8qf3ptVr223+CVAAiDXoZtt++wZCNQlTqgfHDKDLZmaJgBHaslbws+fbgs3iEWq8GCfMGn5o+Pd7nRLPJDPbroCPlkPGMcVSXbgq7nf5U2EpHCIUjg2UtSAIWktMCOtuBD4ahj9KJntpgXaS09No5HyMFcVJumk9mrj+3A8g5u3YqNyq4otqYzw882C6xVZNBTmBlwmu0kFhvpP0fgo2222Yjver15\\\\\\\/z58lqYx22dlnxv9ynLwY1qF0MfVcB2Fpl6Mj4p3JnrfiaWaaGTeea9ZZHoNzmJQtiR9S7kmlxZ7rvmFr5gPVNICFEsDXwpDSkFbRwWjgMqYIOK7bKoODxM1nwa1ugZUhYHs6odGirJIN7iEVbqh89RfI6Nuv8pDHAQUcOB8pljaxkwugWrXqyK1WE+FRTrxGCWuWf1Js8IDB5ZGJZx6fbWt9T8M8g+Gb0af1hiRXTjjs7VA3z8g+o5oQYGbavw28evNJU2obwyBJ5mxUsPiArPzqfFAAIOTwSimYTnJlCWfMxXB0RLyspUNReF0NeTBYwdBPw+E5p3jVU6yCQHQuaMkpVfPLvX59W1g5x2JSV8ngVDk00HeLB94Gdvpyiir4izu8Ih1258NvTwltqMq34t8q\\\\\\\/D4k2ywFUZctjCGxNIO7M8izWb9fp6KyNFjOG6kuiGvhY2bgi2B1+8PxrLi4oSrXu6sly1veX9MNFTzBIzQHXns+Sgicrj5LW3SA8W6mVQDFDmPqCAklZD0h5n\\\\\\\/v\\\\\\\/jP7bV0xIg74lJ9JFG2Kd6WFuFsaMG41f0ttAZbRkyL0H02mugjuNBbKL7aEQjowsq1na5dxj3OhX\\\\\\\/+fP0Mh3V6NyhdUvDI3Y060BCBHWYf+USpLg83jSOMe6fb9urWfYyDPBxEMXO+Hhp7BkwbclF89FRyAkExj0ZqYKNESsyX53pyxR5Quo7nZVMcL\\\\\\\/Iqmy9dsboe0Ap64jynmlLVhzXQwUYn15ra73ioGX26yD9\\\\\\\/6rz+bw7TTCqaWTHmM2h63EHrRtxmw4drIy3MthUZ45VwurXuvObNreJm57y9XwnapLjlbQZpVI+5Z1\\\\\\\/79ncSYZVVuHoUICfGJZRcyHEEhNY4DsJ2qibC9ZNOSFqDlUVXXjqsWYVf6e9EOlKA62H1UBtdlk6q8+eyZNYGfZxuWGkwl+XB\\\\\\\/6RUGUiu4DAWJxoMgLdcVZaRkShd\\\\\\\/7mZ2W37XXjcwlsnb+8X731GAY9IpcC5RSDFFJJLJEbg+PUzzqS04USqqOVH3zI1ImPM8b\\\\\\\/fBUCV+5868+Lv4AvX6GoInvruhEhfzww6hqVN2K2wa3S1Q\\\\\\\/yDrXnVRW4SUzfXMgMM\\\\\\\/cMCQuudBEEVAy4QKShA+bAP6hn\\\\\\\/TjvasiF5Zf6jVoC7uh\\\\\\\/Xmq1RT\\\\\\\/wXfd0huJaewV3SpMjuXOLE3gdCadKAgPr\\\\\\\/3S0FTj35yi8Xd9CE1fJPyETduTTF2b\\\\\\\/cUy5nNSzs6tcYtqRkgU6sRt7mMz\\\\\\\/odK8d6HovsqKKjV794hbOGS0Upav81+7B+D14G74eD4cIxZx\\\\\\\/BrP\\\\\\\/nONHZ7crL0Rkv9nuUw4V6Lh65r0vccip654gengQyQeq49NXLYElZQxFc89onRjoxajM3Aqhm4GL3friJwI\\\\\\\/haSi76BWr6CLv7swzMOfaIRsnte1xS8zXgZJSuNHrgDriPoZeV2i74AN+K9jw2GjxO14sHKRwfJSMTwkelwTnfqVgLOxGiTCWg6VUMQGOTk\\\\\\\/pFPUt1ynTWHiXk8JW30Om6P52UiYItVSTC34KZfJsfW+fWzVwl2vcm5cm6f6q162ew6ysKsk\\\\\\\/cr\\\\\\\/sYonzo\\\\\\\/1EBNqbzDN5UTfihPfdL1X6fcjgBseBH845Mj+xoYjOONod8tJdSHHYTO+epTRcY7CSGudvu1fVWMdoEIp9Rkdv34qEX4NoBKoFt336P8ILxZhMJ9In1JniqiZjZNN8CvQgZTCEEwgzVD0ONPiwEclu27NjJobuBz0mmJsRxG68+Ekkccm9AwnjSCzL5EdZumr7orbFQu99HTMv32IQgofBe\\\\\\\/Yp2QL9tufmNdLNeGstYkU2Syu16MdL6IZ3wnb7LU6eQUP8CuMpO80njqkNZ2s\\\\\\\/DBun8yg24\\\\\\\/zECum1RudZlzf1PK+mZ+Lo16viE7SDNw0NvgSFd\\\\\\\/vXVFmRdhoVo73tawlvUaIGjfZNP8KIQgMq9h0WNqQrUM8zKqNecOIoof7bIuAqsg7zmzIFcIGQ0M\\\\\\\/1VxfdfVfA5owxnNI\\\\\\\/W\\\\\\\/TIfMTjhF+KXkdm2Hu9Tl3aWN0dYlIq+LtghbnCw\\\\\\\/07oi5of2sZ\\\\\\\/Ug6kZGrjakLm2H8oaU9P\\\\\\\/cfXGb1vP4G6XYkbLid1iMsAmExVU7HDYnozQCy8PbWafSWHdGKgjLsk8DnUAuAdv1l8i8oSSzOIQQrCIioua9BuhcNW3SsY1hgD+8FF7EBcpjBV\\\\\\\/op8rkPfrFvjnvZJbyIOKlF2gXhb0N2Rrbs8wY17zzcHPd6ah03Ws5FAcn05hC9\\\\\\\/t\\\\\\\/miIHpuV4ZZVFkgTNenV2goiPRC16eK4bIZKgmFXQLHdjDF\\\\\\\/jOzgWAW0dzeOnPnkdME5vxDh4zYgPKDeQnt1UUgsjPX+GabS1eIz+kKX6waRN2RDRiUckufdrZIzqMNsBTnjIf48v\\\\\\\/RvyvwdATPLZhu46Hlbqskd6Ea64wpfiJus0vSvoaEhp8jEfzsj+ciFBsKWSlc\\\\\\\/OSNZJoFfYUXgTqZhOd4Z\\\\\\\/+wlwf1PPFVopB9ixiVnrBWiVujd3inyj92OnbC3rmhHyZ559Q4nDJAUrKpmWxV6wnhnBR6dL3IubaJm4nEiYMPyGYybCfRDGdARaixX3kf7vNKu1yg4VHO9XS7sUq+pyGmAKiFmNY7LI2k\\\\\\\/H\\\\\\\/PC3rHudfi1m6oZE6Fqib7U1PdmCgVkkKbDrQPxE5Fb8pLEdL75onJTxjKvyM2JZ79Z7jDrSYzX0TgRLvkLiDrNPplqWvhzuiifs\\\\\\\/lXrz1IG1\\\\\\\/N7KuA8Tdd+zlX4qfa+3UV36OygpcxgBokdZtY7tNl\\\\\\\/xlPBlueH1KfuaVH8n9JpHT5Jheavn8Re7b37O56woCvLFD24cDT0qfpoSuYHcjZ9g31XNhNYBziVvDeIB52CsRwWaBpn8lwCIGItGyVrf73TGzX+vdUnifCruYmNLFxz6rioCVrftGgqXHnXdk4QAUQidHGmkKIbgxKcnhCj1Xr7vUYYtjTxzkI95AeqOySZPcAVIQNI+4lO7+1BLJZYQHLRcm927XBspTInO+8YcOxlcrq8X8qVbzPb2QQAxoyre4A\\\\\\\/KZjet3Mhoxo6V1cLSseutZ1oG\\\\\\\/u3QF65KQsWl\\\\\\\/H2yFu90o\\\\\\\/Y7Njb+lgvUKwk2C59Sdhfq0Rw\\\\\\\/BEWzWPk2nJwjjkL6NNnqOjME9JisXIZk3Y47QYLm2CV8qedQI\\\\\\\/IcQytyJBLjPYZU59gvg0ojh8Ph5sB4hzKAF27MZW8ywVJAukb6lkjmFEGIqUWQZMahXM27y5EZmV9zA91Z4Kpd4pNoi51Rz2HKyTHT+qIFQip+IJR8lds\\\\\\\/PKOfTYSyJ+w2iV7UwXBquJuVnzyFeTlZOA\\\\\\\/nWZm4PlvbY3wqrl7ePMY+jdxafknC0LiIQ49O1S59kv688LlK9C2gIv1OJwm\\\\\\\/N2jbGRrXeEGcDMj6cyiTJpj4suQjh6kYC6g2RFmcTzhJ5GB4mfhbXwMucMVJ5jUVN+Y4pSHt87qIjLj9zquqqORWMmmYx4201OxlSakEbOsnnqcGhGaAyID\\\\\\\/ZL3nCvaA2JjHfoBvx1OHTHRJHcrt9gO43vQoCmWzA7\\\\\\\/fqkwiKNi0xVSnQa1RpP\\\\\\\/\\\\\\\/mz5ts5iKs\\\\\\\/Mmpd48AJI6VvDUhUobEoqSZuAqGmIPBaFPbN4gZzlb0IA7H7cew7rwXbcoJEmHGgpIMtig5reydm5Ep3teEtZhcsE2jo4igS2oTxOLo1LNLLKdcqW\\\\\\\/I56w0+gRuMvmDn502ko8658Ok39mVx+MJ1DdL5w8eQImUJ7bpfRV6l\\\\\\\/E2+uFfN5E7FuD6sOYb0VF4v5Hbe08dKgONy\\\\\\\/L9Y6ff2TRbqXPBOtNJuNMhKGx4KQBf\\\\\\\/65AC0WlLh6+FyiV90bJ\\\\\\\/7+iFpxgLOvGK3FIvZ15WswD4WsCl\\\\\\\/+k7iLQNDBUMFaIr535As5tcZRk8JyjpL2cJmggvNDhOx06NJ2lRBb2cLQa5QumZtYN6SBHXo+YbKyDfcMhQ7vp5tCR3QmiQTESXLY9RNYM8hE+rG4DbRQI3cB9f1Hha0S\\\\\\\/DzHNodis1uMOP3n45jeqthHIwHscWjWb74NlFfsO\\\\\\\/Oje2TIBx8INs+i+ouW\\\\\\\/LsCPlyrobsijIBXN11\\\\\\\/CwpvYTVt7SsAbMd1lNCs9knfossh34ujL01URxz9L5MoNqCGkyPFu1BjMsz5mDEO5vF0a3dyPeQHIA5hCo2qwTNCaUhQO1hTk+1IlB4Tm7sJiDKmfoLC92uJ\\\\\\\/ypoblW7paIOfqYjVhPXAVLReylXhUx3XyzyeC\\\\\\\/VJEMdcqQ++yRaVyob14MQ\\\\\\\/ZXn1kTDB6KI2QEMnYjCD5dQ01+NyUkVUaDqPLugJ6GW\\\\\\\/\\\\\\\/b4J+M4Kqz4AlbqkaHiy2nPiQZD73RE2xb7Hvl6\\\\\\\/TTao9YgqxqZL5XJkwF84\\\\\\\/bymAp+j2JfJh23Fe3gdYninJIwjn5Adi96fEsxGkv5imf05YNjM6esnNRAm12YTLr65ejyqdpHBacDPqkhngEkbg1xa+boeq53RAyEhybfVcTes\\\\\\\/cpBXBZ4cLPwT7qbBxedBqrx5m4avkyvycgPtpW2B\\\\\\\/wYSuWk5nx3ML4aahINREWeavE7+ldHqPshCFPv+M2p9PAPCPn+oFBr\\\\\\\/MNEF4g62YHC2Cu\\\\\\\/aojm65ZHszON6hBjFOizEHpluS9Q3RJjpClfTmzXnrBz\\\\\\\/D9mjV4ioOqB4Q0\\\\\\\/DEQcyrkS0goXgbXN46m3p3Wg3uUJzm167mZfOjx\\\\\\\/+rVTIlox576eUOrxqV0sQ7ERAFBIO+2OSwPTgGOfR2vZyiMNl5dxmds2LOs6BpnBHp04Wl+TUjp5P1QdOk7BqvEC5tksQr6C53U4zqWuL5jTCdtW7uyBLKLFLnPEBVKvwNN6NQvBwg0dnNW0oWd7v0zOkYI6UHHEYnevgRFImrl6jKGAnZjzcZtevA6vTCsKhQV2\\\\\\\/pTRRaAO0CO\\\\\\\/r9jCUxAa5Ab3sBWBZp2B0fHWVONGKqj4Rjp9vvniy2Bm8M+5XEyxBWkOc7ycf1dlwmFM3RiBh80NFwmx0yhnbFRXf5Xgqztw1a18fWDwOowvutJBIqcVQPcrI9C88zXXoebfIFVN46Fro2Nn99Sbn3NY61tVo128DoUL2ai2AcPTApIbfHF27HdCG2HmMYux\\\\\\\/4VLBnsGIMj+9oxla6iOxXGkblcTNPWg972bLwaIOfsWm0n7BF0DaAXxLGuDD03MLLZyIlTUciZ3tmYBwLpkFR41NWXnM70G6QBBCpKuimqp7SjWtNNFlcrLplIfMP5LOHeNBfnw5R9I\\\\\\\/TH1JkOXRWzHfqTkxhInsh+ReZZaNJ027J1kC6YHFrXN6N+OO332dZE\\\\\\\/G9qrXHWZLv8ue4W4VU6PNi3Ucg7UaQp+0QBpxmcXwwhaoJHwUAigYIy1Mc\\\\\\\/7Ji2Iepu+sytCCFXzNY7h0qnmGsxXZ1d1xMDt8mzuASLFR+7+YgrRrjCqo5LD3TBkMobJ2bVQ+goUQ9XinS+fxZ\\\\\\\/84VFNYnl5kAIA0kGs2z3GKE6ucDBfEHBEjLS\\\\\\\/1hM+0fo8vVkzBCTQm8il27vvUdarMeGBEJCTqHzhDpaPh82X8rp7o6Zk81Lj69MSyya6BYqE01tyEz1WsoeKZG0DtAjTEh8cmxNP\\\\\\\/4mGIojfx38W07HQEfUahp+YyHMc2ms9FRzegp1EX7EbVUhEQv7SISPejkC17rv0WsQzEQFhHa+LyhAjunDUclg2eLa2GTmqq957Wm8f2eUxIp9j2n1wxx2PDw\\\\\\\/wq9+spzbyxAZwLycDu0dUYwvROXJgKN99mGCUXNPErA9wPYwNBSzX8QnXwWILPtCj\\\\\\\/vC8nYClOo1qZ+olUvz+Yy7vcFFzt0nS4d8q3ACBeT749hTcXxbZCu0RcpFijuL7uAE5jCeSZjCim8n5LNYn+asEFIO3BzIedw6P6u4T\\\\\\\/RGMI0Xz9h\\\\\\\/QYFb7OF8nBC6ZI\\\\\\\/U2pgmcbTX+CLiWH\\\\\\\/DEopACuYvZEHaQoppG72\\\\\\\/1+JcDGFpkkClfva40XZcvPZE6AiclG9OLUWnXoutas0CT6RrfCai8aKVS2v4UO7uyZWGuxfCwjnQB0P9K74WlpKlbAdPpe9JtoJvt2JxhUWSV8Yg6y+L7cPLUJ4YDmbN6ETc8fSqjY0tmzfEVe+WxMfxH+O1rPStFW5Pji\\\\\\\/NEFFNtNlqGoTeUPqt436BMfMU0ECJaY+EX1SI5uCAy7BNLJn3v0yje84nLuo+EZnEsZ4JZgGL5FjKKZHn10+nBXUK78qcbcGV1fn9BQnLeyjA+0xCmoqUd\\\\\\\/eVHVr4+Q590H1Ly2Y2gZN2U3sxTtNruLrrytO+n+MmPc5a9sIQi6VAXlr9ZxgZY2ROIYDTQxTj3NceYakc\\\\\\\/Qtc7SDpniT6nFkraJ0lSzyqAURRNnyKTc2EiQKypKsF1s4egWsxPKVe1iDry\\\\\\\/4dWviBPgYK+bDU3iUMA2zTWLy8T+BzNFXU6Vrvb1xEoQZfG\\\\\\\/Wis3va\\\\\\\/pPEDN3sginzGD9n9Ty6iS4Y0hH2tJL+Zs8YBhdiW+SweBaAeoJJ+WztxrLd4Kww8EBHrgaS54G0wra3+DhK9ybe1a7v4Z5KMt+z4XQ5lAS\\\\\\\/ZW7s0SrQnoK\\\\\\\/wOvvJxUAqIaOvRYfgKtp1H9Luion0\\\\\\\/+azxjSEQAGd90r2u+PRyCWleSe0LPzAOMaqjOew3GIZnU9HUT1EVHo7odu6MxBCjvFF1LIefm7CMaeiS+0u8JZeTTUhyb1ul34NxdXDHCmB5Q0B0zptKMUxq4d+bP0uFXW1UnpGojQ0UmlDzx2nW7A+TUtLZ1QJE121tlzQVp8SlzxAZLKHo1xLNnvha5qCN5JE1ZLImTHP6NAic2leuw8fZcRYZcL0ejzAyJbSM1SiLiA20JYQfyZI86hUrLc1shw3K+vp+ta4+yHpq9usflPQdCzV77Y8ck1v9F5GVluJ0I+q96MTldxitA+DNVKy87AUhDKJyxh9S6iqJVu3P3F41Gb5glXfB30FYaYY2i6yMShaCid98PATipIU+PjFJBxXYUjzN3dAa4NmGYCgRhlkB1DReV7Za4SwF+gi9bibJVY2pNKbu1pfOvy2+gbaU7Uutj\\\\\\\/K09LbFc4gSWcQOr5dmCeas9TPNQnvKVgu\\\\\\\/n7C6UJvdGq6YJp6vZcWdrJuAEMpLhx8tDlxI\\\\\\\/SQa3Uqhtk9rMcsqLFj0vwdIhRyYpUKLxdBddKK5kjFnCfDkWDrW3ffTCRJLfs6fpaWxIFXi2Nt8ewo8kLPEpvWQjOSTBNEOL+de0ca0ZgawC7HqIxcoxTf3WZ+QCVi8pRmmASeTXbsf4FsrfjWVH7+uSre0snjEboePvk8EftRGFfqYdOSvkeylaPvfl0WKCcee7mczCd9S8e5F88SvOqZit7aQiaa3RGiWrlME7VzcaV5r3Hr2a7dRz3INqibQJ9jvd748Kii2NOCvMPhlGMAKh5O+ZR0g+XxFUMMLi2SiKN+ow1+uQs86QTgpfSAJP1zI\\\\\\\/NHIDUkM91nE701ckuR2rL1l\\\\\\\/umBNilU7FTuGJ2JRZ0Nd66yLpkCtQ2qFDWgTtf2yKEzRU9jKicb10FZE2kOEJxlW5+ij2RjB1cTddRMoLMh\\\\\\\/oA\\\\\\\/qU1Jd+5GhCG0cGXztAEBPKE+\\\\\\\/ULRfKa9i9VoKcHwz2tSXdbmxvPtuGDnQ5oXdNg07WpfJ7JKvjelh5oOwUtUqGrhl9gxj0VGCMqp\\\\\\\/G0K8Lib6uDRIHwuutVFo7WhYLJloee+OcrE5oUuWGcGVM1H+w2QHG97o0qEEDv0fFCkxJbiRR5DA1m7vs0F8A2DusIHqQ2kbNychBdRA6Rw7pvMHVmROjR1YQkfZFZ3H57K9syzWAOjaR1IBwAB\\\\\\\/GHK4AliGwJE3eITAC\\\\\\\/dL9RhQZIYFYkLpfompAO73W2Vn5X8ULNxMTUH+jEG+FahvamMBM+Vu9UQkvahf3V2QFPhbZ+Hy9b3LH2hn9h1wZdl4i4aoFd8S6Uw+wCYR5jUqsLwNltIgbmuS38R2mQIUXpd\\\\\\\/F4RtEQCdyKo58w5\\\\\\\/sQlz+mk9lkwHdR+\\\\\\\/Eor1KvZpQmMMf1pa4ALvAAfgb8INRs8si9MfaYOYNzIRIYKmGLuPe71VuHy6Hl82OdLZrK4auuT7AZ4D8yGzTOrpG3a2zOAVOQeLEnsqvuLZ6f3os8t0SxvxBq3X5JT4bL0H\\\\\\\/qvBY2SUpLC049DYs1\\\\\\\/XPRadFjvgph5+ENearYh2RiwgSfRu3EaUHSdwMManUkfpfKr\\\\\\\/O9nevODyol792ZnYg5BjiM\\\\\\\/xY2AQDelo894n0XjGf1Hcx8rbpdIJq87tEKXnV+Ue+hhAwcxLWNT0L4rctklIl6aeA19ZQaCF+XiqsmgBqPWGTZBEukEauI0TmFpQwpEnfUf+I\\\\\\\/33CzdTaiw51l\\\\\\\/YATOKoDIDnzUd95Ok+2rigelsjhBnhaEdKoMyw5ZpEVDtiRFNEOMBZLsP6LArK10QXvfxDjiMxr+B3XNr8SubR5kIRsvjf77d5yM5ihgMC+Je\\\\\\\/ThNpeAG2P+mmyUfM0emN++uFA1uToUAa1vNLpEDs\\\\\\\/C\\\\\\\/xcInu4+T9zOGff259oNbH6hCWHAf6jr5+OuAqdUpy5efhU00O5E8Iz2tCScEAbin4Aaz9QlnIE\\\\\\\/nbwPt2jSXuLWq7sBmpD9OGL9TaL\\\\\\\/PSlTQFBIXEvqWNG4An1izjs5st+mFMx2\\\\\\\/cyDV9q9VTjdV1YPGbtyd49azNJQZ7zqpj5aS3yLhMfLfeWYAV6h6UaxHIRi9dZHYHQyEapbFIG\\\\\\\/s58hZaWrRdAQ9H973rTbzpEwNtPMetDmzcaaZxyRg3dFTF6vRMD6amhTArhfLWXiq4EPyjRSvd3DzZmdEas0HgfsVQRSVnW3a3BguJcI8DXdaJoMQsdoeww0gwNHoBCO9wU9EF\\\\\\\/Oo20XVufBKIhp\\\\\\\/bxdw8d24evHG1yGnSnDl0ufJlecSAvPyQ\\\\\\\/pkFIzNtoBYP0AaWDx5D7\\\\\\\/H0GRxJY7mrkUQTaeHB+nmDUGA\\\\\\\/8Kyntg\\\\\\\/Uaz8UTqCrKdNilfTMGQ\\\\\\\/BHrayiahuBhfxk9GZAfZuC8flSKmX6rfb0LPkeOuTAZ0DvqSXLYjFxmtM193WmrA\\\\\\\/fMYBPeLYKH9UpSHaooWTznGDLYg3f6nuVSFWLuhNla7jy1IiDsEOYQg4\\\\\\\/op3m3PPJDVpFcukEy6Cvaji7z0O\\\\\\\/WYT6838QuL8OJIf0c+UV+MjJRDq7GgsXT3ABdk\\\\\\\/RnQtOPxjo3MFZczt1Vu8vOLTK3tEFdMcL34SH\\\\\\\/VEVJij0soyzrFAfMJfbwIdXRXGGp2YxFimKSKAw0FCRvPm+LlaGAByCX6OuiB3fOk8PYiy8Ga0tI7\\\\\\\/T3T506Fy1j2JzYG2WgGdMWsxQDIOm32\\\\\\\/Imz9iHJ5NUpHs+8n92XEuepoSeBqLqS5UwmfV4xN\\\\\\\/xDOCL9F7yYzTXcA1hGnsTgc6cSWHfarpZAF5AFx9NPj\\\\\\\/kES+XKyDQon84boU0GPQ6FfjTodMR8xboTFvFBNrQRIm26S8RPhvYaZAM3BXVeJ8u7jlBdnq2kZJ5LUEj8\\\\\\\/p3pZexgAEnGedMlb\\\\\\\/UUlZ7echk8V9onfc2n2WtP32ADeptnZu8CBmNNn4fzY5c9rlCnXkXW5YqUakJfS6Pm8LtR8wRYY1vQg7S3fXkswXTyQFTNTEcyYiAtALycLHt0SZIWQwDdiIcqo2kUjk06ofxgTziZZZH6C66Oc1WcjlKjeVL1Ds55tPKiogqGvdCo5uAL6XLa91CHgVt4fJ2lracVK\\\\\\\/S68Q2JKOuyLbeiNR9X+v9HV7XET1hAeL94EzP0d6CaT\\\\\\\/SsSlbSTSjxssJN5QqGL\\\\\\\/sBvUSR3BpSPkJUYwNmn11o4i7SVB9VlumAjvBPRoblqXm99BNdbHVn3aAbDyEwbQl57XXHKkxVIqTvdWo0iFvZ8CGUclrlmiSdo0S6emUF4HZpkm17upllS0xG11OnE63Y8YJYCra1FztSLamkDYWYb9AzN\\\\\\\/e3L3quRS4zLnuEXgmaTVHsCmz9bgsLBa\\\\\\\/zZWVY03+xMvhg\\\\\\\/S+AW7+bH0vUzhcLGCJlPL3c8Zo3n1mbQ8aqJGNKckiHHUQ1IjJ1Edzbe\\\\\\\/onw\\\\\\\/qrt7kZvHOlkilSmSaduxjG62N9TVk3e9z+gGx0HD\\\\\\\/R7JkYJki1+oYr+v\\\\\\\/wLm43OinRqvM6B6stdYIRNXXLc94iE+fxEzCYRGAqajhPuBeOplrWdFn4aw+Hgs\\\\\\\/6P8f1i\\\\\\\/8Vc93Yq\\\\\\\/JrkJfp8gQ1HTIWxk3S+i2nrVJWzTG8EBQdzi9wkkfgiuCpumVKPjcdutmLBLb5FDoBdOQDt9PNiSiatjr\\\\\\\/hCTmipbpelQFN\\\\\\\/++dBkR700qoKqhCDRASEZb\\\\\\\/ARW1LiWEKVutMCTcMb3oR22JKiGQUi+HRKkm5Yp4vPnZ3t\\\\\\\/G2iCRPIUmozzzUaBcFNLLXbnnE0L+6LVCgp8dybaZidc7JfTRnYy9h86RaM8hgcBn0HabQo32tZVH8UeLICNPl9MEe1D5DS1KbOamCdAmLP7jRaRte3Ln6WwGcVO5FPsKxF3WKvkhn0udKZBSCZ5x2icDRHXjiSS1r4ue5\\\\\\\/nSrhr77H17b\\\\\\\/mHRIVG5jzdth3OqzJwpT1OvmvYq3NdeDGQp24bzvtHO95G4EDRNh2iNuM1ikYWWklse0sKkufH7J1BPMDQzRj8nJ8KUicJ\\\\\\\/xSgoF9g97FqbuAIMbxCSIJY+c5yzbASyzzQXcjoJp0kCQZ4bgWk9LEt1BtoY0sMRFNoFDf9YlhQJtmRmCH3S14Kpb\\\\\\\/+52vuGMb0MCf54orS85jWDT0TK+FsQMDaRNNEU0OvL9ILjjWC8q+hHMWN+\\\\\\\/PY53NGwg9Khh+E27ZbejoDTjIYIFjjNFGhFd68o0k8kIJss9FJCZvydrpzzy7dOQUKnyt89xYDYfjhlaj619MwThxuu+crnYoe8jQEExDvUXsm8Axr73lrWRJ+khfd1H2Z1Up35Cy9\\\\\\\/qT2tNG5s+tq9kqE+n4JO0aCwSj7j8WtYg\\\\\\\/fj5nzROfVcD8TVnNhJu7P3AB55efL5KSzwSioEI3EOz+gOzHNamHxrzSwTaFyLQ3JJNhkWxtEsOOUhOmCBYoxbgE0gtB1tgAnu6KkNicubx2ZXinlnYha1Z4FsWuZbDsmzYLLo8FJzW+R5tqzkdUqMEVWOXQnE8OVCMafoWwH681ORZNOWYeAufUjmosIZK3UOWo3U6Q2aUUfUVQLcpoXfLnKCzaKYUXXOyITaCWvfzVlAKMeAAdRuvHWQRf8hhANS+n3T4cGAFH4nPC9yF6d0IOlRMR0tQPGLJOgK1Vii6a8en4eJnGJXfTP54RYtqavF9ukauBpxVnbvyBBiQYujIlgTSWnqaz4IXD6gZ9XUceU4t7hNtFz2kBX2W+wA2m\\\\\\\/UvFTwTxtKo1k8Bdbh1p36ZLphCKq\\\\\\\/Z0vImCepWKIKXreHV0QIZPmpYfPK06GIyU+KGsMvbqLei6heCHaxwD9tgqiHkaRUAXwaKWEVBUN1yhNneGtVQqup+a0fcSxpUE4Z0GzMkoytIhOIr7et+bNCuCFAO+9Ow1F7EkELwxMq6ggJYevUU+jNFvU7YH8xYMbNf8rlDzcrmbE3MH7VQ5GoJQrYjiZFTYO59gcmT4Lf7VaRtKf297QGJX+Qjva3R8XLVkaggOa91V0UPciwu\\\\\\\/as1\\\\\\\/8N7ty5qcLcjabXLIK0fyVNGPHJqsE6vFRCn7WcrNTHGVfRXqVJ2dZYYXr\\\\\\\/dJlinGERcM3cvSCMIVG6Q3u1+ITuFo1jPwtlgkRSk4woEVf7uWm7ukQT\\\\\\\/kEypLHCfinJSWgS7Cw1GIonSHxj0GW5UqlRzYjYJ6ByMLc2oVHNJYraxTX3tJz8Mh4DZ4FDlnfW72PTLbM3nm1+SSOf68kkI\\\\\\\/DmKwFZmND9ecPJQSo2mTXPIFLKaw2Tm5B18OtavGNvPYn0oAWRsRH7BiP3xUTS6PlNd9rxo0vilbWqkfHPciZiG9\\\\\\\/QXtluCvtp5j7+OW9LVkONru42gdRWO18HN6AXH1LzApHLqztA3dzvG7vNxnPl3PAx4GRz91l40Z6DjWSVoRuDAf6IG5lzbDQZ\\\\\\\/9DL2Q0sw15XUa1QiklibE20txdyv4FqIbPgkf7p\\\\\\\/5WoHHt3EkGw9\\\\\\\/rTpyqfef6mRetPoDvRkCQHuqo2cdIlIHivp\\\\\\\/1do+vl2EPJl+ireK3lqwVsk\\\\\\\/8sjfEP8Zgl0l7u+IcBdz2Afpp2syiR3f9dE4V0brI7sSKupzoprY+fmWE6P9e1oZXDbASb+BIQm1b2msZIc8Yz512DYMCzYcHJSJ3xi1F6orxqm2Bu2T7yTIUviPqkaTRU3qtKhPJMGio10GrKDKM7uns55YZ1zPkkJulR0fVm4ARtq97bliHqRVQ1rhG3ETXU0eWHWmBooycIkwIHulf8nANHE7hj4u8\\\\\\\/vqcN82O57\\\\\\\/td81ACK3BqAtzKf+k2P+rHK2Af3K5cIrNoN0LgmcyTZSzjPBHDprOEXBTkKIKOOgsFGZhOdUN9RExqOahpFk5YZCKyvalIZc6Px81c\\\\\\\/pJnwVVp3lLfL6AGq9UEZXL58fntf0ZDrXCH3ziaXghdIneku53Awqnz5n5cftXAzqHW7GKQl7IsPwTe7ePMPGRaZfIeUR62sRvs2x5t9U\\\\\\\/Ewg\\\\\\\/Xj1skVYaCFXgQzwyMV9tvrPQ+64\\\\\\\/w087bnAXb8l\\\\\\\/Rxn1diivJY+Hw85Gg2OS28pz0DJUBMqAxfTKgN0+NMBUD4IlFX42SNppNQAItej3p6i6AS1oYjvdbcwUNHd\\\\\\\/4ufxkYi\\\\\\\/VJ\\\\\\\/B05OZtXbq8Dt1D0PIOcKs2Fn9lMnxjKGJ9DmaxWCY9WcjARenhBGgix43D7+6JBDat2RKiEHQkOjp7eTcv2+8I2oVCJocYcvf5qcn2dA6Il6rABZFNqD34KLP+3YzvBah3qpO6EMWtLMy3UYmKdXQubhsGqNHLA+n3shth8pKWjchgXUIihbQ+sT\\\\\\\/KPywRBIP8CiOMYrX75\\\\\\\/4v2yJq2eqt9QzTqxwE914ln4fcMJYRT2IyLRqsn6nKp8nDI4Vwq7JAQoGyxLY6cbdBtoXYulmEm9G\\\\\\\/+zUJEsQicpuWYqVkcXi\\\\\\\/wynKJnERImRGB5y2vgXTlFt1YZM2CHZzx3AAOViYzVsmSbMoRyVCxkKo\\\\\\\/IBeWZIJ6KlxGc14\\\\\\\/IBjg8zUJMyZL8SbF8G4SLRZDqOFELBClohzMjCJXZvHyR2LfzGoireMnq0Dr2eg7Rx3eiGUON3yLLD1sy\\\\\\\/r\\\\\\\/Nzz2HN9Lh9teYivSIAN5dzn6JLvu21oG+cONBt+O0q1a3EUF2D8vWey3KAbB3v\\\\\\\/eMIOjJ4LJTAHJQ42d+\\\\\\\/cdGCi0z2I038QoZnF3Lbl2uevUY8nHi6EHj+GNEVRjgYfOIGASQP9mAxUM9BmnTOIGXfAD6YvxvisuWOhPYRF2yctxd60KQYj1ajYBWti+5bKjn\\\\\\\/pNhmb0tT8es2Iz7MBwUN+VxCdXWWCk+3SqH2Xo6B1tyrbtSfMRD7jSRwVhjjkH\\\\\\\/PKrqSsP07dMpbAL3\\\\\\\/5vCLqwX8jQq3bEijonxV1D48xP5fczD+yu26GJYpfQHnIxekO\\\\\\\/a3HeRr13u6JJm68ThTAhJqEK8\\\\\\\/AWcTbDFzcspF42+6rR\\\\\\\/3OUpHkRbBAY2J3uPipRaondmTNNcT2N2xmkxQRvHAEOK9SPsfMVt7OJeDWdkfNEPFRCQ0urSME0R+d6n8tHeZymR\\\\\\\/TEPBp3YadNFhcaqz\\\\\\\/9NIyNI9AIq2EtvkawTSu7KnKJW7sPVwIGKXV3GhB03JxUPH\\\\\\\/O8vgfIFDWHcHQrGJbJg1o5ygwh8DRQ4znZgeNDsQcKohWeSbL\\\\\\\/\\\\\\\/i15hSXTOf6d8sMtaoHOqVDKIIE8QzMKvdsF4nQ6wd1UeDc2yrh14i7GI3x18a4Nsb\\\\\\\/nvnke+1KnCM9E4P+EpYeRm1Pi4LL4yyg\\\\\\\/1KCU7IwQ3rXhjFi6OC1FUjrdR+6piypg1jahZstEVeXVsRTDGKO8OkNydtMQxpzbgz6zkLXtOdSzsqbbSCM+\\\\\\\/xQwkQpn3misvqqutAH2MYf9hV5UnRYxCqr348ZnjE54lbZGT2MLB5kVOJlrhERINNx4NkLkz3SrC6HZnM9FDkLGSHsJWA3KBkx7GDdZxXgDXg+C\\\\\\\/gtbhWPK97P959DZD4NFS52m1A5k3GgP2ltIk9uTh3WkpG37LJ2MZ9l+YvMlfHGFQMqDaY6Fc5drlN3mlDVS0V5AepzjeuZDV6wR2nQyyBKE8rzVFrr5knlVfxlfUW45y8kK3gS3DD8B1vEO5DyCMLlncQ+LNaf8mBeaSEYFoF0+E6lCeL6kSRsyHOpo4Mg1SPc\\\\\\\/zra2h4I6u8S33P1yYPajfI77Xb3LBZRqViXB9nzflXaIudU85PDYTgo8dWksnhoOTZM+ttT+XTeMVhDW20le36+tYMGqjOwln1x+tiQktnPMrODnTiZU8yM2PoTX+q8FlgWvmyqYhIt44YJjBTOupMYpg+AmHdiGQnyRqZrtAnCs01hu9TbA7zgn9P+cfsR\\\\\\\/mZGmfo4zS3n4sTzJGFm8T1ST8VmC\\\\\\\/uqhHZELokCEQtoMLBVtSIAAbeEDe3gYfEw3cUf\\\\\\\/tPcVuZrsErBCwVYXCJAN0YOsRP32kM8EY4v1FYIIogPNeII+4BbLuujfZuZhSGN1804fYJpjwUF2GbizhevIA07MERiVp5NivUka2Jy+GjMhuTlk9UrHjqgAoACg7nmuOMaWm\\\\\\\/KOX1TAqos1R5isbpuGcy4hc6IfPP1ajxgLbqyyTSRHL7SkE24wPX1bVQZwRh26gkpAkSvSMEW0havfSQ1deTgn6IXNRrIfiV8FUts0c7ixn4sNBfPAyS1ItobYOflXVHlWqUo6Eo9Imdi47pXlIjjf6JunaWd0xGe\\\\\\\/hm\\\\\\\/Pt\\\\\\\/+\\\\\\\/Z0m8418u4gvmRDICEZrVMPvjV9PR+89FvlwZ8iNv5UiMH2Ug2xl239u234j1GITpGNEwNjmdmCgRmJBdcBP8b768p+QnIpEbB1U0tCkMDUCaQXhN8Fklf5D3A9Lz473jhdhY08xE1BavifSwmncGgoV2IlufJsL3xYhp+XG+uammMIhD8CRAm2dblB1T1dRvxVTGJa8rVHGsUZXo36+VDudDcu37sG9lKf+vLXfMUmR0pRLtmWX2jjQYjp+bwe2yvs66rGvVp38SqeOC3Jc7Bqlrq3Qc2L6Xg5Y97G5N4n7D7j\\\\\\\/k6Vjb3lqZZxS2egPL7GPsAQFULqFHsQNf9Gtr9XNR\\\\\\\/F5+tLOA9tdeAYlAINp0X3WSq0DBcORTK6\\\\\\\/Awn2R\\\\\\\/YClr4m22xPy1qIDjrLiuRV0y0LmynCrmbdvkPmR5Nc7OrcrlyOFY6KdMXeCBEQLavQbURQKCewO6Zsx2ibElUXLyzbyi0SSevqDjIgLoCVBIa8U7FpDinwshpq1kRHmzGPVWowYMtbM8GWYoqx2HvXME+ixRpQ7hK6JmtYtDf5vyTeIVaosCEb3W8GzW7AQUvsFF97tYHncp4AxR6\\\\\\\/K07tRHn11oILHKGMvWBDcvKt5A186d3X03KW+6cn+P+06QS1CsWvyjM5oarEcejaOiPB6pChSBQOUhCLmq5r7rrK3swpzKzeJcD7mSYszCC8juLNYvPhgki8b+MQte19haq33RYIzQODEL+1f\\\\\\\/wu3yQ377FEwikknJd+DKU5jPc8B29ThU4pWf+FykOPqy+jw0FUz8zV+GePA4vcL4V1n5xjUGjELpdGfekikEXUP0YQbPu6tFRHMJbzrlt292rOYdbZYV4EBpC1x+oV4tton38oqOG5YuPZLDwtLJ1jhLmzeDOtYpetqeLZPzSr1gTy6PooQTSTzkBmoDji8R3397qmcTdpDW2s+eu4YqhjE9uDoUoBu9i0rnLCbHliTwrz0DQsIa4PjwV0M3zopJDgO5zGoqn\\\\\\\/+cZkxjhi7SzRahLUAVNcUcCZuNEGaCAVXGR7FNpdxeXT0tjTU2S5Y3leG7ImK75tWQEQ2hRKfee2w+LRuX5UZJwPYjTWBsN9fhToxLESA2uHMW7N2ai2hkwaNZeq92ldkWNyQa0JP1uBPwlkH8nfq0R\\\\\\\/HrPSK50vDWr8LVeFwOXTWvCSVcrrMYjO5kCEceuDSLvpUfpTq0YbT1l2T6LfH+F4dQqxi1djKP6RPcmPhY9GFpcvW1KBDUujgJIoeyJnZ54Jstn0F3KkB9Qt9LETksu2dQA8Y9ETm76x2lvlUh0OBPOfA9DRXle2uAUCYWSR0Ya3lEnBTfssAsEgxqA3FjAf+NvY8uWkxCF1pofplKwu\\\\\\\/UqB2MDwJacccj9hKukC4v5fuxC9SkRHGSa5+zhLlitGVnJWwvwmPGAxaLf\\\\\\\/tFUqB4WFuroV84WTqU+VtxdvIvIqRvj2n+EtCbI+YBnap8LLxZHxKpSy93Wag1lDwMCo9BrrAefVx5\\\\\\\/5TxRUpRYvD3r7XGPG7REB+6jDDYkuNAHrNRYsKG9hQSd6Ppv0RNj1QizszMZGy8KoG\\\\\\\/08q8GZEj6Ec1nv2rxIdNgYBh1ZQjx5WaBZgOZJAJa6MHTEmWoJR5rCSi7ZQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022CyYVPFh0dTOBuC7S7mUxCQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a4263487917625ee801f8aefc1e0731b409db75cac82d24aec411b134cca46c1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Lep9r4rj7exUQg2kQ5UJnxmkKQyF2CsCGFKlzUaSnKjaJN3uyDgR1uFhaDmq0s9l4wsMcXSxNjMNZb4ePDc14nmnZrsM2twSWEgfeH7os4HwMELNCiTFof7XI2GtGZQhYVkaIe1zFeJTbLGBK4zyDGk0tUhR5X+bLBU5cmBVp2lyY5ICLTaqzmrWQ8BhAs7CUjU66uCjG1bog\\\\\\\/8gl6B+tvzJkYYXuwC72p0zaNer8TXHI9504ulaDSME9jJaisapq69IApAF99pfx4URYAKSXTpzEaHXNFwbbRUKhHUM5EP+hxTYZa9sfy6PzK9ifopz3G3WTn0Zqz8wWr4mmDZ2RzQNCsHic568Efk5Lzw0WHarf4Nn2oe1zfsRz1Jxz70jSnCywimHWRNU1s57IKPPbC0crK9ZwhkKKNUa2LB7X+j8QspunZWqdYs2kIvcu\\\\\\\/dXr0UqH+\\\\\\\/Plel7KBxjMSvp0SRPE3CSm4mXo296zFzBYzn4fCM2QsGyD1NM+942W7Wj1D1BO6WjDi1Cp8h8m8ScOGtBQje8k4sCskVF9a25oFHdemhcLnZckf\\\\\\\/nHlM5fkjf+RSsZqsbIw2ghGn1MVKIhjANtdzEb0IOm9z+pgjeGIkkOGlziNI+wO73jafn7lM2BYcPRwGRpgpEw5\\\\\\\/97mWalhG1\\\\\\\/1CMV1eoubMLz8f2B6XUXfD0bS90ltERoMHH6Fv4ZvdH5iC1huliiRNjCVwVwL7fX\\\\\\\/WXmiiSGOqFsawcLbXDEdq1o7wh+2fvgAQdwNiSFlUmGKjH4S10p9TY0TgK+rWEEqqW4Uv2js9PoHkJaF7WtNVX18lqxsvRaUVRch5hVwvfLAtaXiSnBEixxbxE\\\\\\\/kjwzFDNpyhZ7b1\\\\\\\/mxMwxJmvLUr7UapLgvg8r6ybCVEU1X1gbtX12aD2zpRpyx47vBBQVT9EHWl6NG710eZug8hWIUzCSqim9St3tasUThmQrAOIx9GfHcrqwj6CjqYB+ZdnMk19YB6tZ\\\\\\\/gvnjwTWXXm8WsQ4CrdrmLY9dYIGdk3NxxJOIzSwK7Fkxw7WfQF1QOceNuwqK1Win7ouE51bn4pxLM08\\\\\\\/SP3HF3rAmcNXL7csQonVpMeCfJhhhT4PF3eExd9YRk+emyZDujWjuPVee48f\\\\\\\/y9XZt33FnNiMYAJAl3fbubX8XFUr1hS+FGtOAzSgV9oT3\\\\\\\/R597\\\\\\\/A+C\\\\\\\/huXXrzeUFW46RPhDaTTweBl\\\\\\\/wYeaLcMXqFmcfDdp8OJV8NkatmFxDlFipsDfKFUyp90kHMPOTFgO7XKwoVwBtQw\\\\\\\/0r2pv5VJJVeEW7rhdRLAqdJ8x3D09UJDil+kKHjjC4QMl8H3itR6X3suXcgYwoXlCiCj9oOost76cHozKykGU3lqF8SaaBeIUKPL3VQFT9SUyMlbWadEFCBcO6Kfp0Bi5JQkHi2wiE+a6PvxFVFqh\\\\\\\/JpI9zuc95nKWiVfvJhLy1GdCQFI8IY1YikFi3O13I7zFYF81npmaejpeWuIKIQ8i8Ymtdvkek84CG1ANYvFO6Cie8bH+7P11NeQffAiKKZ3ACH1wwfGCv7Z1z1p1eo8rv7GfD+FfXTDgWSOxbZvUvTv3JQb5su4zdtGoI1srlyvc\\\\\\\/yaN83aDfKrC5446yDkR1+7I4WZFJYQGNO4GcTNJMnK08Lv6J9+Q1O7TUxX2r7v\\\\\\\/EYpu8OKJ7l4fqjqwImCXTjPTbY1xHlR3grA+SfgrY+YQ1ueL0e1gHWxf4+JaWEsg7n+O5rL351c4RCVRanENrhoBAZDNy0dbgcoi+xCx67SVtQ9gyn+IcvEDN2alEPKzM7sgGoCWlFuv1qTdZuW\\\\\\\/cHXWiKmjtaD87bahFLAJakpI3Pe0mMjVhNNhmSOav1p90JqoiTVhdOXfek7mDfsmoOWQ52Re7yGLI9iGPW+N4smxrqSaUG\\\\\\\/IZrLfNGbXXCp9QjOHAi86YXe9eWjlut+0hcLv3wkOJE8caZZsh004ZjUCZ4q2s1Y74FV\\\\\\\/TV2scbNY6ILKVrYxwiWVJg1jFtetAQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vZYKDLRXLuUW4K4NeaE20Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f860e71d4238dcf214aecb7950ec6b45f6d97e8a7851298cbd6b838644454216\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459188 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/5/387" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-q9eQKFaOSHlkK0l5E_DwsjwDB8k1avV_WbPdiwW-Bfo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/5/387" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-eB-nm6h9o1HtpDpPSUOUso6UtqeUgaDDStWG0_psnxM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/5/377" title="Generation of Highly Homogeneous Strains of Zebrafish Through Full Sib-Pair Mating" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/5/401" title="Sequence Analysis of the Segmental Duplication Responsible for Paris Sex-Ratio Drive in Drosophila simulans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459188" id="node459188" data-pisa="ggg;1/5/387" data-pisa-master="ggg;g3.111.000539" data-apath="/ggg/1/5/387.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Polymorphism and Divergence in Two Willow Species, <em>Salix viminalis</em> L. and <em>Salix schwerinii</em> E. Wolf</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sofia Berlin</span>, <span class="highwire-citation-author" data-delta="1">Johan Fogelqvist</span>, <span class="highwire-citation-author" data-delta="2">Martin Lascoux</span>, <span class="highwire-citation-author" data-delta="3">Ulf Lagercrantz</span> and <span class="highwire-citation-author" data-delta="4">Ann Christin Rönnberg-Wästljung</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">387-400;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000539 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459188" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sofia Berlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Forest Genetics, Uppsala BioCenter, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sofia%2BBerlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Berlin%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASofia%2BBerlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sofia.berlin@slu.se" class="" data-icon-position="" data-hide-link-title="0">sofia.berlin@slu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Johan Fogelqvist </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Forest Genetics, Uppsala BioCenter, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Ecology and Evolution, Evolutionary Biology Centre, Uppsala University, SE-752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johan%2BFogelqvist%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fogelqvist%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohan%2BFogelqvist%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Martin Lascoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Evolutionary Genomics, CAS-MPG Partner Institute for Computational Biology, Chinese Academy of Sciences, Shanghai, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BLascoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lascoux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BLascoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ulf Lagercrantz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Ecology and Evolution, Evolutionary Biology Centre, Uppsala University, SE-752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ulf%2BLagercrantz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lagercrantz%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUlf%2BLagercrantz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ann Christin Rönnberg-Wästljung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Forest Genetics, Uppsala BioCenter, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ann%2BChristin%2BR%C3%B6nnberg-W%C3%A4stljung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=R%C3%B6nnberg-W%C3%A4stljung%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnn%2BChristin%2BR%25C3%25B6nnberg-W%25C3%25A4stljung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/5/387" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459188" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459188/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/5/387.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459188" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459188/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/5/387.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459188" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459188/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/5/387.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459188" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459188/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/5/387.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459188" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459188/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">We investigated species divergence, present and past gene flow, levels of nucleotide polymorphism, and linkage disequilibrium in two willows from the plant genus <em>Salix</em>. <em>Salix</em> belongs together with <em>Populus</em> to the Salicaceae family; however, most population genetic studies of Salicaceae have been performed in <em>Populus</em>, the model genus in forest biology. Here we present a study on two closely related willow species <em>Salix viminalis</em> and <em>S. schwerinii</em>, in which we have resequenced 33 and 32 nuclear gene segments representing parts of 18 nuclear loci in 24 individuals for each species. We used coalescent simulations and estimated the split time to around 600,000 years ago and found that there is currently limited gene flow between the species. Mean intronic nucleotide diversity across gene segments was slightly higher in <em>S. schwerinii</em> (<em>π<sub>i</sub></em> = 0.00849) than in <em>S. viminalis</em> (<em>π<sub>i</sub></em> = 0.00655). Compared with other angiosperm trees, the two willows harbor intermediate levels of silent polymorphisms. The decay of linkage disequilibrium was slower in <em>S. viminalis</em> compared with <em>S. schwerinii</em>, and we speculate that this is due to different demographic histories as <em>S. viminalis</em> has been partly domesticated in Europe.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/salix" class="hw-term hw-article-keyword hw-article-keyword-salix" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Salix</em></a></li><li class="kwd"><a href="/keyword/gene-flow" class="hw-term hw-article-keyword hw-article-keyword-gene-flow" rel="nofollow">gene flow</a></li><li class="kwd"><a href="/keyword/species-divergence" class="hw-term hw-article-keyword hw-article-keyword-species-divergence" rel="nofollow">species divergence</a></li><li class="kwd"><a href="/keyword/nucleotide-polymorphism" class="hw-term hw-article-keyword hw-article-keyword-nucleotide-polymorphism" rel="nofollow">nucleotide polymorphism</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li></ul><p id="p-5">The species concept is fundamental in evolutionary biology, but accurate classification of organisms is still far from trivial. The recent developments of genetic tools that allow identification of genetic differences among organisms have greatly changed our view of how to accurately assess what determines a true species. For instance, based on assessment of genetic diversity among individuals, it has become evident that morphologically similar organisms may form distinct species. And sometimes the opposite holds true, as morphologically dissimilar organisms may in fact have small genetic differences and even be members of the same species.</p><p id="p-6">Particularly in plants, species may appear phenotypically diverged while still experiencing high levels of gene flow and little genetic differentiation throughout most of the genome. Species from the genus <em>Salix</em>, or willows, have often been used to illustrate this phenomenon as they have a reputation of being a taxonomic nightmare due to the anticipated high frequency of hybrids. However, the actual frequency of natural hybridization in willows has seldom been investigated thoroughly. As a matter of fact, in one of the early attempts to classify willows, <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wichura (1865)</a> found hybrids to be generally rare, although they could be extremely frequent locally, leading <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Darwin (1865)</a> to express his bewilderment at the “extreme frequency of hybrid willows.”</p><p id="p-7">In the present study, we aim to investigate the species concept in two closely related willow species using multilocus polymorphism data and to contrast this to what is known about the morphology and general biology of the two species. Species in the <em>Salix</em> genus are trees, shrubs, or subshrubs that, together with the genus <em>Populus</em> (poplars), are members of the Salicaceae family. <em>Populus</em> is recognized as a model genus for genetic and genomic studies in angiosperm trees with many resources available, such as the genome sequence of <em>Populus trichocarpa</em> (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tuskan <em>et al.</em> 2006</a>). There are more than 300 <em>Salix</em> species, and they are widespread in both the Northern and the Southern hemispheres, excluding Australasia and New Guinea. Many species display rapid growth and high biomass yields and are therefore used for short rotation biomass production (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Karp <em>et al.</em> 2011</a>). <em>S. viminalis</em> L. and <em>S. schwerinii</em> E. Wolf are dioecious willows that are phenotypically very similar. Both are multistemmed shrubs with long and slender leaves and are commonly found along streams and rivers and in other wet areas. As other <em>Salix</em> species, the sex-ratio is often female biased (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alström-Rapaport <em>et al.</em> 1997</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ueno <em>et al.</em> 2007</a>). Both species can also easily reproduce clonally, although the extent of clonal reproduction is not well documented in these two species. In <em>S. sachalinensis</em>, for example, clonal propagation was less important than expected (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Ueno <em>et al.</em> 2007</a>). <em>S. viminalis</em> has a vast natural distribution ranging from Ireland and United Kingdom in the west to Siberia in the east (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The exact boundaries of the natural range in Western Europe are uncertain due to extensive cultivation in the past. In Scandinavia, it was introduced in the 18<sup>th</sup> century and has since then spread (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Larsson and Bremer 1991</a>). This was confirmed with allozyme markers, which also showed differences among rivers in pattern of isolation by distance (IBD), less disturbed rivers showing a greater pattern of IBD than rivers that had experienced a higher level of anthropogenic disturbances (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lascoux <em>et al.</em> 1996</a>). <em>S. schwerinii</em> has a smaller and more eastern natural range (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), and although the two species in many regions come close to each other, they are apparently rarely found growing together (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Skvortsov 1968</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/5/387/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The natural ranges of Salix viminalis and S. schwerinii. Redrawn from Skvortsov (1968). The two dashed areas indicate the regions where samples were collected. Lakes are shown in dark gray color." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374824786" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The natural ranges of &lt;em&gt;Salix viminalis&lt;/em&gt; and &lt;em&gt;S. schwerinii&lt;/em&gt;. Redrawn from Skvortsov (1968). The two dashed areas indicate the regions where samples were collected. Lakes are shown in dark gray color.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/5/387/F1.medium.gif" width="440" height="238"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/5/387/F1.medium.gif" width="440" height="238"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/5/387/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/5/387/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460986" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-8" class="first-child">The natural ranges of <em>Salix viminalis</em> and <em>S. schwerinii</em>. Redrawn from <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Skvortsov (1968)</a>. The two dashed areas indicate the regions where samples were collected. Lakes are shown in dark gray color.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">There are no reports of hybridization occurring in natural conditions, although they are easy to cross artificially in the laboratory. <em>S. viminalis</em> and <em>S. schwerinii</em> and their hybrids are some of the most commonly used <em>Salix</em> species in the breeding programs for biomass production in Europe. These species have also been the focus of most past investigations of <em>Salix</em> genetics, which include the generation of linkage maps (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Berlin <em>et al.</em> 2010</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hanley <em>et al.</em> 2002</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hanley <em>et al.</em> 2007</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rönnberg-Wästljung <em>et al.</em> 2003</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tsarouhas <em>et al.</em> 2002</a>) and QTL analyses (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rönnberg-Wästljung <em>et al.</em> 2005</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Tsarouhas <em>et al.</em> 2003</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tsarouhas <em>et al.</em> 2004</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Weih <em>et al.</em> 2006</a>). In contrast to <em>S. viminalis</em>, <em>S. schwerinii</em> has not been used as extensively by humans and grows today in a geographical area in which the impact of the last glaciations was more limited. We would therefore expect the two species to show different patterns of nucleotide diversity and LD (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brubaker <em>et al.</em> 2005</a>).</p><p id="p-10">Level and pattern of nucleotide polymorphism and linkage disequilibrium (LD) as well as degree of population differentiation contain information about evolutionary forces that acted in the past and can be used to infer past demographic history. LD is determined by factors such as recombination, mutation, selection, and population admixture, as well as demographic history (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kim <em>et al.</em> 2007</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mueller 2004</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pritchard and Przeworski 2001</a>). A population that has had a stable and large population size for a long time or has experienced a rapid size expansion will have lower levels of LD than small populations whose population size fluctuated through time or experienced recent bottlenecks (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Mueller 2004</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Pritchard and Przeworski 2001</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Reich <em>et al.</em> 2001</a>). The last scenario can be a consequence of domestication.</p><p id="p-11">The aim of the present study is to investigate the species divergence between two phenotypically similar willow species, <em>S. viminalis</em> and <em>S. schwerinii</em>, with adjacent but presumably nonoverlapping natural ranges. We look for evidence of past and present gene flow both within species and between species. Considering both is important as gene flow and population dynamics of individual species will influence the amount of gene exchange among species (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Petit and Excoffier 2009</a>). We resequenced 33 and 32 nuclear loci that represent parts of 18 nuclear loci in the two species, and we determined levels and patterns of nucleotide diversity, population structure, and extent of linkage disequilibrium. We also used coalescent simulations to reconstruct demographic histories within each species and to estimate the degree of sequence divergence between species. More specifically, we asked whether polymorphisms and patterns of LD differ between the two species according to differences in biology.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sampling</h3><p id="p-12">Twenty-four <em>S. viminalis</em> clones were included in the study. They originate from various localities in Europe and were selected to cover the western European part of the species range (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/000539.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/TableS1.pdf">Table S1</a>). The clones were collected between 1978 and 1990 and are now growing in a field archive in Pustnäs, south of Uppsala. For details of the sampling, see <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Lascoux <em>et al.</em> (1996)</a>.</p><p id="p-13">Twenty-four <em>S. schwerinii</em> clones were used, of which 17 were F1 crosses produced from different parental clones that are now also growing in the Pustnäs field archive. The six other clones are growing in a field archive in Svalöv, southern Sweden, and were kindly provided by Dr. Inger Åhman. The parents of the F1 crosses as well as the additional clones originate from Siberia, east of the Lake Baikal, and were collected in 1990 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/TableS1.pdf">Table S1</a>). Young leaves were sampled and stored in −20° awaiting DNA extraction. Genomic DNA was extracted with the FastDNA Kit (MP Biomedicals) according to the protocol provided with the kit, and DNA concentrations were determined with a Nanodrop spectrophotometer (Nanodrop Technologies).</p></div><div id="sec-3" class="subsection"><h3>PCRs and DNA resequencing</h3><p id="p-14">We selected 20 protein coding genes from the <em>Populus trichocarpa</em> genome (<a href="http://www.phytozome.net/poplar.php">http://www.phytozome.net/poplar.php</a>) for resequencing in the willows, and for each gene, we attempted to resequence two segments (A and B) of approximately 500–1200 bp with various distances between them (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). This method is known as the locus pair approach and has previously been used to study genetic diversity and linkage disequilibrium in humans (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Frisse <em>et al.</em> 2001</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Voight <em>et al.</em> 2005</a>). This sampling scheme was used, as it will allow estimation of local as well as more long-range linkage disequilibrium without requiring extensive resequencing efforts. The poplar genome sequence served as template for the construction of PCR primers, and Primer3Plus was used for the primer design (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Untergasser <em>et al.</em> 2007</a>). The primers were positioned in exonic regions. Primer sequences that resulted in successful amplification and the positions of these primers in the poplar genome are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/TableS2.pdf">Table S2</a>. The PCRs contained 10 ng genomic DNA, 1× PCR buffer II (Applied Biosystems), 2.5 mM MgCl<sub>2</sub> (Applied Biosystems), 0.2 mM dNTP mix (Fermentas), 0.5 µM of each primer (Invitrogen), and 0.5 U AmpliTaq Gold DNA polymerase (Applied Biosystems) in a total of 15 μl. The reactions were run on a MyCycler thermal cycler (Bio-Rad Laboratories) with a PCR profile consisting of 10 min denaturation at 95° followed by 35 cycles of 30 s denaturation at 95°, 30 s annealing at 55°, and 1 min extension at 72° with a final 10 min 72° step. Amplification success was determined by agarose gel electrophoresis. The PCR products were cleaned with 1 μl of a mixture of Exonuclease I (New England BioLabs) and Shrimp Alkaline Phosphatase (SAP) (Fermentas) for every 5 μl of PCR product before they were directly sequenced at Macrogen, Inc., Europe (Macrogen, Amsterdam, Netherlands) using the forward and reverse PCR primers as sequencing primers. Sequences were edited, and contigs were assembled separately for each segment and sample using Seqman version 8.1.4 (Lasergene, DNASTAR), and heterozygous sites were scored with ambiguity codes. All sequences were carefully checked by eye. Two sequences were created for each sample in DAMBE version 5.2.5. (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Xia and Xie 2001</a>). As the poplar genome harbors many paralogs, BLAST searches against the genome database on the NCBI web site were performed with each of the gene segment to confirm that the orthologs had been amplified. All sequences are available in the NCBI Genbank sequence database (<a href="http://www.ncbi.nlm.nih.gov/genbank/">http://www.ncbi.nlm.nih.gov/genbank/</a>) under accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=HQ625673&amp;atom=%2Fggg%2F1%2F5%2F387.atom">HQ625673</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=HQ628624&amp;atom=%2Fggg%2F1%2F5%2F387.atom">HQ628624</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461009/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461009/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Location and size in base pairs (bp) of the amplified genes and gene segments</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Confirmation of single nucleotide polymorphisms (SNPs) and of inferred haplotypes by cloning</h3><p id="p-17">Cloning of three segments I-53B, II-33B, and II-36A was performed using pGEM-T Easy Vector System I following the manufacturer’s protocols (Promega). PCR products of one to three <em>S. schwerinii</em> samples were cloned per gene segment. The PCRs were performed with 10 ng genomic DNA, 1× PCR buffer HF (Finnzyme), 0.2 mM dNTP mix (Fermentas), 0.3 µM of each primer (Invitrogen), and 0.2 U Phusion DNA polymerase (Finnzyme) in a total of 20 μl mixed and run on a MyCycler thermal cycler (Bio-Rad Laboratories) with a PCR profile consisting of 30 s at 98° followed by 30 cycles of 10 s at 98°, 30 s at 57°–60° and 2 min at 72° with a final step of 10 min at 72°. An A-nucleotide was added to the blunt-end PCR products prior to ligation according to the protocol found at (<a href="http://www.promega.com/pnotes/62/7807_15/7807_15_core.pdf">http://www.promega.com/pnotes/62/7807_15/7807_15_core.pdf</a>). Ten colonies per sample were grown in overnight cultures, and plasmids were extracted using GeneJET Plasmid Miniprep Kit (Fermentas). Insert lengths were tested with PCR before sequencing with T7 and SP6 primers at Macrogen Inc. Cloned and directly sequenced DNA sequences were compared, and reconstructed haplotypes were compared with the cloned alleles.</p></div><div id="sec-5" class="subsection"><h3>Sequence and SNP analysis</h3><p id="p-18">Sequence alignments were constructed for each segment and species using Clustal W (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Thompson <em>et al.</em> 1994</a>) in the Alignment Explorer tool of MEGA4 using default parameters (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tamura <em>et al.</em> 2007</a>). Hardy-Weinberg equilibrium was tested for each SNP using Arlequin version 3.5.1.2 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Excoffier <em>et al.</em> 2005</a>) for each species, and deviations from Hardy-Weinberg equilibrium were tested with exact tests using default settings.</p></div><div id="sec-6" class="subsection"><h3>Nucleotide diversity</h3><p id="p-19">Calculations of standard population genetics parameters were carried out for each segment with the DnaSP software version 5.10 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Librado and Rozas 2009</a>). The level of polymorphism for each segment was estimated both as haplotype and nucleotide diversities; exons and introns were analyzed separately. Indels were excluded from all analyses. The pairwise nucleotide diversity (π) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Nei 1987</a>) and Watterson’s estimator (θ) (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Watterson 1975</a>) were obtained for intronic DNA (<em>π<sub>i</sub></em>, <em>θ<sub>i</sub></em>) and for synonymous (<em>π<sub>s</sub></em>, <em>θ<sub>s</sub></em>) and nonsynonymous sites (<em>π<sub>a</sub></em>, <em>θ<sub>a</sub></em>) separately. π and θ are expected to be almost the same for neutral sites at statistical equilibrium under mutation and genetic drift (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Tajima 1989</a>). Analyses of allele frequency spectra (Tajima's D) were done for each segment.</p></div><div id="sec-7" class="subsection"><h3>Analyses of within-species population structure</h3><p id="p-20">The genetic structure within each species was tested using Structure version 2.3.3 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pritchard <em>et al.</em> 2000</a>). Sites that showed significant statistical association due to LD after Bonferroni correction (α = 0.05) were removed before the Structure analysis. To infer the structure of the sampled populations, the admixture model was used with a burn-in of 50,000 and a run length of 500,000 for a number of clusters from K = 1 to K = 7, allowing for correlation of allele frequencies between clusters. Ten independent runs per K were performed to ensure that the results were consistent. The most likely number of clusters was estimated with the original method from <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Pritchard <em>et al.</em> (2000)</a> and with the ΔK statistics given in <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Evanno <em>et al.</em> (2005)</a>.</p></div><div id="sec-8" class="subsection"><h3>Analyses of divergence between the species</h3><p id="p-21">The level of divergence between the species was investigated using Structure version 2.3.3 with similar criteria as above. The degree of genetic divergence between the two species, <em>F<sub>ST</sub></em>, was assessed by a locus-by-locus analysis of molecular variance (AMOVA) in Arlequin version 3.5.1.2 (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Excoffier <em>et al.</em> 2005</a>). Significance was determined with 10,000 permutations. We also assessed the number of shared <em>vs.</em> fixed sites in DnaSP version 5.10 (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Librado and Rozas 2009</a>).</p></div><div id="sec-9" class="subsection"><h3>Linkage disequilibrium and recombination</h3><p id="p-22">The diploid sequences were phased into haplotypes using PHASE version 2.1 with default parameters (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Stephens and Donnelly 2003</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Stephens <em>et al.</em> 2001</a>). This was done for each segment independently. However, in V-20, VI-4, and VII-11, the A and B segments were located next to each other and primer pairs were overlapping; therefore, they were treated as single segments. The PHASE analysis in <em>S. viminalis</em> resulted in 97% of reconstructed haplotype pairs with &gt; 90% posterior probability and 88% of reconstructed haplotype pairs with &gt; 90% posterior probability in <em>S. schwerinii</em>. The levels of linkage disequilibrium between SNPs with frequencies &gt; 0.05 in reconstructed haplotypes with posterior probabilities ≥ 0.90 was estimated as <em>r<sup>2</sup></em>, the mean squared correlation in allelic state (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hill and Robertson 1968</a>), using DnaSP version 5.10 (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Librado and Rozas 2009</a>). Significance of the associations between SNPs was determined with Fisher’s exact test with Bonferroni correction. The overall decay of LD was estimated by plotting <em>r<sup>2</sup></em> against the physical distance between SNPs. We then fitted a nonlinear regression that yields a least-squares estimate of the average recombination parameter for all segments, ρ, given the empirical relationship between pairwise <em>r<sup>2</sup></em> and physical distance between sites (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hill and Weir 1988</a>).</p><p id="p-23">To investigate the decay of LD over longer physical distances, we combined the A and B segments for each gene and again reconstructed haplotypes in PHASE. The results of the A and B segments combined were 93% haplotype pairs with 90% posterior probability in <em>S. viminalis</em> and 83% in <em>S. schwerinii</em>. LD was estimated as previously described, and a new dataset with <em>r<sup>2</sup></em> was created for each species. These datasets contain <em>r<sup>2</sup></em> from the combined A and B segments for each gene for which we estimated the complete segment lengths in willow (see below), as well as data based on the A and B segment separately in cases where we did not know the complete lengths or where only the A or B segment was sequenced for a specific gene. The first and the second dataset are not independent datasets, and in some instances, they contain the same data.</p></div><div id="sec-10" class="subsection"><h3>Analysis of physical distances between segments in willow</h3><p id="p-24">With knowledge of the physical distance between the A and B segments in willow, we can study the decay of LD over larger physical distance. In <em>Populus trichocarpa</em>, the lengths varied among the genes and ranged from 1466 to 6073 base pairs (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). To estimate the corresponding lengths in the willows, we used the forward primer in the first segment and reverse primer in the second segment in PCR. The lengths of the PCR products were determined by comparisons to a DNA size marker (MassRuler DNA ladder mix, Fermentas) on agarose gels. DNA from two <em>S schwerinii</em> and two <em>S. viminalis</em> clones were used and Phusion DNA polymerase was done according to the protocol previously described.</p></div><div id="sec-11" class="subsection"><h3>Approximate Bayesian computation of individual species</h3><p id="p-25">The demographic history of each species and the time of the split between them were estimated using approximate Bayesian computation (ABC) as described in <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Blum and Francois (2010)</a>. First, a set of summary statistics was calculated for our dataset after individuals and sites with too many missing data were removed. Only the longest intron sequence for each investigated gene was used, resulting in 14 loci from 14 genes included in the analyses.</p><p id="p-26">For each species separately, we tested three demographic models: a standard neutral model, a population growth model, and a bottleneck model. For each demographic model, prior distributions of the model parameters were constructed, and 10<sup>6</sup> draws from these prior distributions were used to simulate coalescent genealogies using the coalescent simulator <em>ms</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hudson 2002</a>). The prior distributions were chosen as <em>U(min</em>, <em>max)</em> or <em>log-U(min</em>, <em>max)</em>, with <em>min</em> and <em>max</em> set at reasonable values following initial trials with different values of <em>min</em> and <em>max</em>. For each draw, 14 independent genealogies were simulated, with the number of sequences and base pairs per sequence corresponding to the 14 used loci. Summary statistics were calculated using <em>libsequence</em> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Thornton 2003</a>). For summary statistics, we used the mean and variance of Watterson’s θ, Tajima’s D, and the number of haplotypes.</p><p id="p-27">Next, the Euclidean distance (δ) between the simulated summary statistics and the observed summary statistics was computed and a proportion (<em>P<sub>δ</sub></em>) of simulation outcomes with the smallest distance was retained. The value of <em>P<sub>δ</sub></em> was set to 0.001. Initial studies indicated that the outcome of the analysis was rather consistent over a wide range of <em>P<sub>δ</sub></em> values (0.001–0.1)), but the peaks in posterior estimates were sharpest at the lowest threshold of <em>P<sub>δ</sub></em>.</p><p id="p-28">Finally, the posterior distributions of parameters and the posterior model probabilities were estimated with the nonlinear regression approach [feed-forward neural network (FFNN) regression models] developed by <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Blum and Francois (2010)</a> and implemented in the R package <em>abc</em>. This approach helps to overcome two problems inherent to the ABC approach, namely, the curse of dimensionality and model comparison (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Blum and Francois 2010</a>).</p><p id="p-29">The neutral model simply consists of two parameters; the population mutation rate (θ = <em>4N<sub>0</sub>μ</em>) and the population recombination rate (<em>ρ = 4N<sub>0</sub>r</em>). The prior distributions of both parameters are log uniform distribution with min of 0.0001 and max of 1. The same priors for θ and ρ were used for the two other demographic models. The growth model has an additional parameter, the growth rate α. Its prior follows a log uniform distribution with min of 0.001 and max of 10. The bottleneck model assumes a constant population size <em>N<sub>0</sub></em> until <em>t<sub>b</sub></em> units of time backward in time when population size instantaneously drops to <em>N<sub>b</sub></em> for <em>d</em> units of time after which the population size instantaneously increases to <em>N<sub>0</sub></em>. All units of time are in <em>4N<sub>0</sub></em> generations. The prior distributions of <em>t<sub>b</sub></em> were uniform <em>U</em>(0, 10), and those of <em>d</em> were uniform <em>U</em>(0, 10). The bottleneck severity (<em>f = N<sub>b</sub>/N<sub>0</sub></em>) was fixed at 0.02.</p></div><div id="sec-12" class="subsection"><h3>Approximate Bayesian computation of species divergence</h3><p id="p-30">For the combined dataset of both species, we considered three models. First we tested a neutral population split model with migration, with parameters <em>N1</em> (size of population 1), <em>N2</em> (size of population 2), <em>t<sub>s</sub></em> (time since populations diverged), <em>M = 4N<sub>0</sub>m</em> (symmetric migration rate between population 1 and population 2), and <em>N<sub>anc</sub></em> (size of ancient population). The second model used was a population split model without migration defined as above but with <em>M</em> fixed at 0. Finally, we considered a model in which the two species did not split but interbred freely. To make this model as close as possible to the population split models, we also assumed a population size change to <em>N<sub>anc</sub></em> at time of <em>t<sub>s</sub></em>. The priors were initially set at a relatively large interval and run for 10<sup>6</sup> iterations. Thereafter, the parameters were estimated, and new priors were constructed near the 99% credible interval of the estimated parameters. As summary statistics in the species divergence scenarios, we used the averages of Watterson’s θ, Tajima’s D, and the number of haplotypes in each population and in the combined data, as well as Wright’s <em>F<sub>ST</sub></em> and the numbers of shared, fixed, and private polymorphisms.</p></div><div id="sec-13" class="subsection"><h3>Model validation</h3><p id="p-31">To evaluate the fit of the models to the data, for each model, 10<sup>4</sup> draws of the parameters were made from the estimated posterior distributions, and coalescent simulations were performed using those parameters. The simulations were summarized by the mean and variance of several summary statistics (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/TableS3.pdf">Table S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/TableS4.pdf">Table S4</a>), and then compared with the observed data. For each summary statistic, the probability of observing a value lower than the observed value given the model was estimated. To identify outlier loci, this was also done for each segment separately. The overall fit of the data to the model and comparisons among models can be visualized using principal component analysis (PCA). The multilocus average and variance of summary statistics were used to compute the first two principal components (PC) of the data for each model. The summary statistics used were the same as provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/TableS3.pdf">Table S3</a>. The 95% confidence intervals (C.I.) of the two first components were plotted together with the predicted values of the data using the PCA as predictor.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>PCRs and resequencing</h3><p id="p-32">We designed 40 primer pairs to amplify two regions of 20 nuclear genes. Sequences of sufficient quality were obtained for at least one segment from 18 loci (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Sequences for I-53A, VIII-22A, XII-18A, and XII-18B were not of sufficient quality, and they were therefore not used in any analysis. The same applied to IV-18B and VII-1B in <em>S. schwerinii</em>. As a result, 32 loci were analyzed for sequence variation in <em>S. schwerinii</em> and 33 loci in <em>S. viminalis</em>, representing 18 genes located on 10 different linkage groups.</p></div><div id="sec-16" class="subsection"><h3>Confirmation of SNPs and inferred haplotypes by cloning</h3><p id="p-33">A total of 2193 base pairs of cloned and directly sequenced PCR products were compared, and 15 SNPs were confirmed. In all cases, the DNA sequences of the cloned segments were identical to the DNA sequences obtained with direct sequencing. We confirmed that a reconstructed haplotype with posterior probability 1.0 was indeed correct. However, we also found that when the most probable haplotype pair has a probability between 0.3 and 0.7, the phase is likely to be incorrect.</p></div><div id="sec-17" class="subsection"><h3>Hardy-Weinberg equilibrium and inbreeding coefficient</h3><div id="sec-18" class="subsection"><h4>S. schwerinii:</h4><p id="p-34">Eleven percent of all SNPs deviated significantly from Hardy-Weinberg expectations (<em>P</em> ≤ 0.01), and in all cases, this was due to a deficit of heterozygote genotypes. The majority of the significant SNPs were located in three gene segments: II-36B, IV-18A, and VIII-5B (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460912/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460912/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2 </span> <span class="caption-title">Intronic nucleotide variation, haplotype diversity, and neutrality test in <em>S. schwerinii</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h4>S. viminalis:</h4><p id="p-37">Thirteen percent of all SNPs deviated significantly from Hardy-Weinberg expectations (<em>P</em> ≤ 0.01), and again this was due to a deficit of heterozygote genotypes. The majority of the significant SNPs were located in three gene segments: II-36B, IV-18B, and VIII-5A (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460924/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460924/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3 </span> <span class="caption-title">Intronic nucleotide variation, haplotype diversity, and neutrality test in <em>S. viminalis</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-38">To conclude, the majority of the SNPs in both species that deviated from deviated significantly from Hardy-Weinberg expectations were located in three genes: II-36, IV-18, and VIII-5.</p></div></div><div id="sec-20" class="subsection"><h3>Gene lengths in willow</h3><p id="p-41">For 12 genes, the PCRs resulted in specific PCR products similar to the expected lengths in poplar (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) and could be used in the analysis of LD between segments in genes.</p></div><div id="sec-21" class="subsection"><h3>Linkage disequilibrium</h3><div id="sec-22" class="subsection"><h4>S. schwerinii:</h4><p id="p-42">For each species, we have two datasets for which we estimated linkage disequilibrium (<em>r<sup>2</sup></em>). In the first dataset, <em>r<sup>2</sup></em> is estimated between SNPs located within segments, and in the second, <em>r<sup>2</sup></em> is estimated between segments and within segments. Within the segments, average <em>r<sup>2</sup></em> = 0.34, and among 1610 pairwise comparisons between SNPs, 436 were significant (<em>P</em> &lt; 0.05) after Bonferroni correction (27%). LD decayed within the segments with <em>r<sup>2</sup></em> dropping below 0.2 around 1000 base pairs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The combined dataset allowed us to analyze the decay of LD between SNPs with a maximum distance of 4000 base pairs. The average <em>r<sup>2</sup></em> = 0.31 and 350 out of 1862 pairwise comparisons (19%) were significant (<em>P</em> &lt; 0.05) after Bonferroni correction. As in the above case, <em>r<sup>2</sup></em> dropped below 0.2 around 1000 base pairs (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/5/387/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Patterns of linkage disequilibrium described as squared correlations between segregating sites: (A) S. schwerinii within gene segments, (B) S. schwerinii between gene segments, (C) S. viminalis within gene segments, and (D) S. viminalis between gene segments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374824786" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Patterns of linkage disequilibrium described as squared correlations between segregating sites: (A) &lt;em&gt;S. schwerinii&lt;/em&gt; within gene segments, (B) &lt;em&gt;S. schwerinii&lt;/em&gt; between gene segments, (C) &lt;em&gt;S. viminalis&lt;/em&gt; within gene segments, and (D) &lt;em&gt;S. viminalis&lt;/em&gt; between gene segments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/5/387/F2.medium.gif" width="400" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/5/387/F2.medium.gif" width="400" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/5/387/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/5/387/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460901" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-43" class="first-child">Patterns of linkage disequilibrium described as squared correlations between segregating sites: (A) <em>S. schwerinii</em> within gene segments, (B) <em>S. schwerinii</em> between gene segments, (C) <em>S. viminalis</em> within gene segments, and (D) <em>S. viminalis</em> between gene segments.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h4>S. viminalis:</h4><p id="p-44">We found more LD in <em>S. viminalis</em> than in <em>S. schwerinii</em>. Within the gene segments, average <em>r<sup>2</sup></em> = 0.49, and among 1780 pairwise comparisons between SNPs, 591 were significant after Bonferroni correction (33%). LD did not decay between SNPs with maximum distance of 1300 base pairs (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). As in <em>S. schwerinii</em>, with the combined dataset we could study LD between SNPs with up to 4000 base pairs distance, and we found that LD dropped below 0.2 after around 4000 base pairs (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>). The average <em>r<sup>2</sup></em> = 0.40 and 484 out of 2476 pairwise comparisons (20%) were significant after Bonferroni correction.</p></div></div><div id="sec-24" class="subsection"><h3>Nucleotide variation</h3><div id="sec-25" class="subsection"><h4>S. schwerinii:</h4><p id="p-45">Intronic nucleotide variation was estimated in an average of 42 haplotypes, and a total of 11,264 base pairs from 28 loci were aligned (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). We identified a total of 358 SNPs in the introns of which 49 were singletons. This corresponds to 1 SNP every 31 base pairs. Synonymous and nonsynonymous nucleotide variation was estimated in an average of 44 haplotypes and a total of 2188 base pairs from four loci (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). We found a total of 28 synonymous SNPs of which 5 were singletons (1 SNP every 18 base pairs) and 22 nonsynonymous SNPs of which 3 were singletons (1 SNP every 77 base pairs). Five SNPs had 3 variants of which 3 were located in II-36A and one in III-24A and III-24B. Pairwise nucleotide diversity (<em>π<sub>i</sub></em>) in introns was between 0.0014 and 0.0237 (average <em>π<sub>i</sub></em> = 0.0085), and <em>θ<sub>i</sub></em> ranged between 0.0008 and 0.0173 (average <em>θ<sub>i</sub></em> = 0.0070). The synonymous pairwise nucleotide diversity (<em>π<sub>s</sub></em>) was between 0.0036 and 0.0175 (average <em>π<sub>s</sub></em> = 0.0095), and <em>θ<sub>s</sub></em> ranged between 0.0057 and 0.0173 (average <em>θ<sub>s</sub></em> = 0.0100). The nonsynonymous pairwise nucleotide diversity (<em>π<sub>a</sub></em>) ranged between 0.0021 and 0.0041 (average <em>π<sub>a</sub></em> = 0.0030), and <em>θ<sub>a</sub></em> was between 0.0018 and 0.0031 (average <em>θ<sub>a</sub></em> = 0.0025). The ratio between <em>π<sub>a</sub></em>:<em>π<sub>s</sub></em> = 0.278, and synonymous diversity was on average 26% greater than intronic nucleotide diversity. The mean Tajima’s D across loci was positive (0.59), but only two loci (III-24B and X-27A) showed significant positive Tajima’s D-values (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460964/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460964/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4 </span> <span class="caption-title">Synonymous and nonsynonymous nucleotide variation, haplotype diversity and neutrality test in <em>S. schwerinii</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h4>S. viminalis:</h4><p id="p-48">Intronic nucleotide variation was analyzed in an average of 42 haplotypes, and a total of 13,058 base pairs from 30 loci were aligned (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). We identified a total of 373 SNPs in the introns of which 57 were singletons. This corresponds to 1 SNP every 35 base pairs. Synonymous and nonsynonymous nucleotide variation was estimated in an average of 47 haplotypes and a total of 2,475 base pairs from four loci (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). We found a total of 10 synonymous SNPs of which 1 was a singleton (1 SNP every 56 base pairs) and 10 nonsynonymous SNPs of which none was a singleton (1 SNP every 191 base pairs). Two SNPs had 3 variants of which 1 was located in V-18B and 1 in V-20B. One SNP with 4 variants was found in VII-1A. Pairwise nucleotide diversity (<em>π<sub>i</sub></em>) in introns was between 0.0009 and 0.0198 (average <em>π<sub>i</sub></em> = 0.0066), and <em>θ<sub>i</sub></em> ranged between 0.0010 and 0.0121 (average <em>θ<sub>i</sub></em> = 0.0058). The synonymous pairwise nucleotide diversity (<em>π<sub>s</sub></em>) was between 0 and 0.0217 (average <em>π<sub>s</sub></em> = 0.0067), and <em>θ<sub>s</sub></em> ranged between 0 and 0.0133 (average <em>θ<sub>s</sub></em> = 0.0048). The nonsynonymous pairwise nucleotide diversity (<em>π<sub>a</sub></em>) ranged between 0.0001 and 0.0039 (average <em>π<sub>a</sub></em> = 0.0014), and <em>θ<sub>a</sub></em> was between 0.00004 and 0.0038 (average <em>θ<sub>a</sub></em> = 0.0013). The ratio between <em>π<sub>a</sub></em>:<em>π<sub>s</sub></em> = 0.238, and synonymous diversity was on average 7% greater than intronic nucleotide diversity (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). As in <em>S. schwerinii</em>, the mean Tajima’s D across loci was positive (0.24), but only one segment (IV-18B) showed a significant positive Tajima’s D-value (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460908/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460908/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5 </span> <span class="caption-title">Synonymous and nonsynonymous nucleotide variation, haplotype diversity, and neutrality test in <em>S. viminalis</em></span><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-27" class="subsection"><h3>Analyses of within-species population structure using Structure software</h3><p id="p-51">Based on 160 SNPs exhibiting no significant LD (<em>P</em> &lt; 0.05, average <em>r<sup>2</sup></em> = 0.024) from 24 <em>S. schwerinii</em> individuals, the most likely number of clusters was K = 3 both when the original method from <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Pritchard <em>et al.</em> (2000)</a> was used and with the ΔK statistics given in <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Evanno <em>et al.</em> (2005)</a> (data not shown). However, it is doubtful that there is any population structure in <em>S. schwerinii</em> because the difference in LnPD with K = 1, K = 2, and K = 3 is very small (average LnPD = −2450, −2459, and −2446, respectively). Further support is the clustering results for K = 3 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/FigureS1.pdf">Figure S1</a>A), which show no signs of structure for K = 3. Based on 80 SNPs exhibiting no significant LD (<em>P</em> &lt; 0.05, average <em>r<sup>2</sup></em> = 0.032) from 24 <em>S. viminalis</em> individuals, the most likely number of clusters using both methods was K = 2 with 6 and 18 individuals in the two groups (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/FigureS1.pdf">Figure S1</a>B). Again, we believe that this structure has limited biological meaning as the differences in LnPD for the different K were rather small (average LnPD for K = 1–7 were −1299, −1200, −1207, −1273, −1286, −1301, and −1313, respectively).</p></div><div id="sec-28" class="subsection"><h3>Analyses of divergence between the species</h3><p id="p-52">When analyzing both species using Structure, using 52 SNPs exhibiting no significant LD (<em>P</em> &lt; 0.05, average <em>r<sup>2</sup></em> = 0.047), both methods clearly supported K = 2 clusters, clearly separating the two species (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/FigureS1.pdf">Figure S1</a>C and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/FigureS2.pdf">Figure S2</a>). The locus-by-locus AMOVA analysis gave <em>F<sub>ST</sub></em> = 0.56 (<em>P</em> &lt; 0.0001). The total number of fixed differences was only 5, while the total number of shared polymorphisms was 71 (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460934/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460934/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6 </span> <span class="caption-title">The number of shared polymorphisms and fixed differences between <em>S. schwerinii</em> (1) and <em>S. viminalis</em> (2)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-29" class="subsection"><h3>ABC analysis of individual species</h3><p id="p-54">The observed data could best be explained by the standard neutral model in both species (<em>P</em> = 0.41 for <em>S. viminalis</em>, <em>P</em> = 0.47 for <em>S. schwerinii</em>), although the difference in probability between models was small, meaning that the alternative models could not be ruled out (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>). However, in the case of the growth model, the growth rate was very small (<em>α<sub>mode</sub></em> = 0.0038 for <em>S. viminalis</em>, <em>α<sub>mode</sub></em> = 0.0025 for <em>S. schwerinii</em>), and in the bottleneck model, a very ancient bottleneck was inferred (<em>tb<sub>mode</sub></em> = 5.29 <em>N<sub>e</sub></em> generations for <em>S.viminalis</em>, <em>tb<sub>mode</sub></em> = 8.07 <em>N<sub>e</sub></em> generations for <em>S. schwerinii</em>). Hence, in both cases, the posterior distributions of parameters suggest that a standard neutral model describes well the recent history of these two species. In this model, the population mutation rate θ was estimated to be 0.0026 (0.0014–0.0034) in <em>S. viminalis</em> and 0.0033 (0.0025–0.0044) in <em>S. schwerinii</em>, and the population recombination rate ρ was estimated to be 0.0036 (0.0007–0.0085) in <em>S. viminalis</em> and 0.0074 (0.0012–0.033) in <em>S. schwerinii</em>.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460911/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460911/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7 </span> <span class="caption-title">Posterior estimates of model parameters in the per species models</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-30" class="subsection"><h3>ABC analysis of species divergence</h3><p id="p-57">The observed data could best be explained using the population split model with or without migration and a rather recent split (<em>P<sub>split_with_migration</sub></em> = 0.66, <em>P<sub>split_no_migration</sub></em> = 0.31, <em>P<sub>no_split</sub></em> = 0). The time since the population split event occurred was estimated to be 0.43 (0.16–1.98) × <em>4N<sub>e</sub></em> generations with migration and 0.32 (0.19–0.47) × <em>4N<sub>e</sub></em> generations without migration. Translating this to absolute time needs estimates of the effective population size, <em>N<sub>e</sub></em>, and the mutation rate. <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Tuskan <em>et al.</em> (2006)</a> estimated the synonymous substitution rate in the closely related genus <em>Populus</em> to be roughly 6-fold lower than in <em>Arabidopsis</em> to be compatible with fossil data. Using the <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ossowski <em>et al.</em> (2010)</a> estimate of μ = 5.9 × 10<sup>−9</sup>/year in <em>Arabidopsis</em>, this would translate to μ = 9.8 × 10<sup>−10</sup>/year in <em>Salix</em>, which is lower but of the same order of magnitude as the estimated mutation rate in Salicaceae in <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Berlin <em>et al.</em> (2010)</a> (1.28 × 10<sup>−9</sup> to 1.68 × 10<sup>−9</sup>). Assuming an approximate neutral mutation rate μ = 1 × 10<sup>−9</sup>/year, a generation time <em>g</em> of 10 years, and the posterior estimate of θ = 0.0016, <em>N<sub>e</sub></em> would be <em>θ/4μg</em> = 40,000. Assuming this <em>N<sub>e</sub></em>, the posterior estimate of split time in years would be 700,000 (260,000–3,100,000) with migration and 510,000 (300,000–752,000) without migration (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Note that as we assume a mutation rate per year, the estimated split time in years will not depend on assumptions about generation time. While the relative size of the <em>S. viminalis</em> to the <em>S. schwerinii</em> population was close to 1 [0.80 (0.59–1.19)] with migration and 0.63 (0.36–1.04) without migration, the relative size of the ancient population to the current <em>S. viminalis</em> population size was estimated to be larger: 3.99 (0.59–11.26) with migration and 3.01 (1.44–6.06) without migration. The migration rate <em>M</em> was estimated to be 0.036 (0.002–0.11). θ was estimated to be 0.0016 (0.0008–0.0026) with migration and 0.0022 (0.0012–0.0037) without migration in line with the per species estimates. Also the ρ estimates were in line with the per species estimates: 0.017 (0.0089–0.028) with migration and 0.017 (0.0091–0.026) without migration. The model without a population split gave wide posterior estimates (<a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/5/387/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Posterior and prior distributions of population split time. Solid line: population split and migration; dashed line: population split without migration; dotted line: prior distribution. On the X-axis time in years since split event occurred assuming Ne = 33 000 and a generation time of 2 years." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374824786" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Posterior and prior distributions of population split time. Solid line: population split and migration; dashed line: population split without migration; dotted line: prior distribution. On the X-axis time in years since split event occurred assuming &lt;em&gt;N&lt;sub&gt;e&lt;/sub&gt;&lt;/em&gt; = 33 000 and a generation time of 2 years.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/5/387/F3.medium.gif" width="440" height="419"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/5/387/F3.medium.gif" width="440" height="419"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/5/387/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/5/387/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460975" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-58" class="first-child">Posterior and prior distributions of population split time. Solid line: population split and migration; dashed line: population split without migration; dotted line: prior distribution. On the X-axis time in years since split event occurred assuming <em>N<sub>e</sub></em> = 33 000 and a generation time of 2 years.</p><div class="sb-div caption-clear"></div></div></div><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460945/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460945/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8 </span> <span class="caption-title">Posterior estimates of model parameters in the population split models</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-31" class="subsection"><h3>Validation of ABC models</h3><p id="p-61">To assess the fit of data to ABC models, the data were summarized by the average and variance of several summary statistics, and then compared with the distributions of summary statistics obtained from coalescent simulations based on parameters drawn randomly from their posterior distributions. The differences between models in the analysis per species were low, hence, the fits of the models to the data were almost equal (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/TableS3.pdf">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/TableS4.pdf">Table S4</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Multilocus averages of neutrality test statistics (Tajima’s D, Fu and Li F* and D*) were higher in the data than expected under each model, although significantly different at α = 0.05 only for Tajima’s D and Fu and Li F* in <em>S. schwerinii</em>. The variance of Tajima’s D was also high in all models, but it was significantly higher than expected only in <em>S. viminalis</em> bottleneck and growth models. In the population split models, the difference between models was much more pronounced (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>), at least when split and no split models were compared. In the population split model with migration, the same outlier loci were detected as in the per species models (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/TableS5.pdf">Table S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/TableS6.pdf">Table S6</a>). For the multilocus statistics, only the variance in number of haplotypes was significantly higher in the data (<em>P</em> = 0.98) in the population split with migration model, and no statistic at all in the population split without migration (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/TableS7.pdf">Table S7</a>). In the model without population split, the average and variance of <em>F<sub>ST</sub></em> and number of fixed polymorphisms were significantly higher in the data compared with the model (<em>P</em> = 1) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/TableS7.pdf">Table S7</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/5/387/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The two major principal component axes of the summary statistics obtained in the goodness of fit analysis of S. viminalis (left) and S. schwerinii (right) models. Within the plotted lines are 95% of the summary statistics. Solid line: bottleneck model; dotted line: growth model; dashed line: neutral model. The circle represents the fitted data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374824786" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The two major principal component axes of the summary statistics obtained in the goodness of fit analysis of &lt;em&gt;S. viminalis&lt;/em&gt; (left) and &lt;em&gt;S. schwerinii&lt;/em&gt; (right) models. Within the plotted lines are 95% of the summary statistics. Solid line: bottleneck model; dotted line: growth model; dashed line: neutral model. The circle represents the fitted data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/5/387/F4.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/5/387/F4.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/5/387/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/5/387/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460957" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-62" class="first-child">The two major principal component axes of the summary statistics obtained in the goodness of fit analysis of <em>S. viminalis</em> (left) and <em>S. schwerinii</em> (right) models. Within the plotted lines are 95% of the summary statistics. Solid line: bottleneck model; dotted line: growth model; dashed line: neutral model. The circle represents the fitted data.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/5/387/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The two major principal component axes of the summary statistics obtained in the goodness-of-fit analysis of population split models. Within the plotted lines are 95% of the summary statistics. Solid line: population split with migration; dotted line: population split without migration; dash-dotted line: no population split. The circle represents the fitted data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374824786" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The two major principal component axes of the summary statistics obtained in the goodness-of-fit analysis of population split models. Within the plotted lines are 95% of the summary statistics. Solid line: population split with migration; dotted line: population split without migration; dash-dotted line: no population split. The circle represents the fitted data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/5/387/F5.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/5/387/F5.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/5/387/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/5/387/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460950" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-63" class="first-child">The two major principal component axes of the summary statistics obtained in the goodness-of-fit analysis of population split models. Within the plotted lines are 95% of the summary statistics. Solid line: population split with migration; dotted line: population split without migration; dash-dotted line: no population split. The circle represents the fitted data.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-32"><h2 class="">Discussion</h2><div id="sec-33" class="subsection"><h3>Species divergence</h3><p id="p-64">At present, both <em>S. viminalis</em> and <em>S. schwerinii</em> have widespread natural ranges with a possible contact zone in Eastern Russia. Even though the two species come close together in many regions, it seems that they are rarely growing together (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Skvortsov 1968</a>). Morphologically, <em>S. viminalis</em> is not entirely homogenous over its vast range; however, the observed differences are not considered enough to classify them as different taxonomic units (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Skvortsov 1968</a>), and overall, the two species are very similar in their vegetative appearance. Further support for the close relationship between <em>S. viminalis</em> and <em>S. schwerinii</em> is that they can easily be crossed in the laboratory. Initially, we were interested to know how genetically diverged the species are. Our resequencing effort shows that there are a large number of shared polymorphisms and very few fixed differences between the species. Such pattern could be the result either of secondary contact and recent gene flow or of incomplete lineage sorting since the split of the species. Our data support the last scenario because in species with large effective population sizes, a long time is required for alleles to go to fixation, even in the absence of gene flow (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hudson and Coyne 2002</a>). Furthermore, our analyses show that most shared polymorphisms are ancestral or at least do not reflect recent gene flow, and the average split time was estimated to be on average 600,000 years ago. It is, however, difficult to detect gene flow occurring shortly after the two species separated (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Becquet and Przeworski 2009</a>), so we cannot rule out that some gene flow has occurred since the split. Further support of two clearly separated species come from a Structure analysis (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Pritchard <em>et al.</em> 2000</a>) with both species, where the most likely number of clusters was K = 2 and the two species clearly separated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1/FigureS1.pdf">Figure S1</a>), as well as the high <em>F<sub>ST</sub></em> value (0.56) estimated between them. To summarize the results so far, we have found that the two species diverged from a common ancestor rather recently and only restricted gene flow has occurred since then. Our data thus support limited hybridization between these species in nature. However, we cannot rule out the possibility that hybridization occurs in or near the contact zone as our samples do not originate from the area where the two species come in close contact. This is, for example, the case with <em>S. alba</em> L. and <em>S. fragilis</em> L. in central Europe that frequently hybridize in the contact zone to form <em>S</em>. × <em>rubens</em> Schrank (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kehl <em>et al.</em> 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Triest <em>et al.</em> 2009</a>). However, outside the contact zone, hybridization does not seem to be substantial. We acknowledge that our sampling does not allow us to firmly conclude whether these two species are truly distinct or constitute a geographic gradient of one species. Field observations (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Skvortsov 1968</a>) support the view that they are truly distinct species, possibly adapted to different ecological niches because they are rarely found growing together. However, further genetic studies, including more extensive sampling throughout the natural range of each species, including the contact zone, are needed to settle this issue.</p></div><div id="sec-34" class="subsection"><h3>Nucleotide diversity, LD, and demography</h3><p id="p-65">Mean intronic nucleotide diversity across gene segments was slightly higher in <em>S. schwerinii</em> (<em>π<sub>i</sub></em> = 0.00849) than in <em>S. viminalis</em> (<em>π<sub>i</sub></em> = 0.00655). These values are intermediate when compared with other angiosperm trees that demonstrate a great variation in synonymous and silent nucleotide diversities. Extreme values ranging from 0.012 to 0.016 have been observed in <em>Populus tremula</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ingvarsson 2005</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ingvarsson 2008</a>), and considerably lower values in <em>P. balsamifera</em> (0.0030–0.0045) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Keller <em>et al.</em> 2010</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Olson <em>et al.</em> 2010</a>), <em>P. tricocharpa</em> (0.0029–0.0035) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gilchrist <em>et al.</em> 2006</a>; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Tuskan <em>et al.</em> 2006</a>), <em>P. nigra</em> (0.0021), and <em>P. alba</em> (0.0035) (Giusi Zaina, personal communication, based on 31 fragments of <em>P. nigra</em>, 18 fragments of <em>P. alba</em>, and 12 genotypes in each species). Different effective population sizes have been put forward as an explanation for the difference in <em>π<sub>s</sub></em> between <em>P. tremula</em> and <em>P. balsamifera</em> (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Keller <em>et al.</em> 2010</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Olson <em>et al.</em> 2010</a>). This is supported by the difference in the ratio between <em>π<sub>a</sub></em> and <em>π<sub>s</sub></em> and the finding that nonsynonymous diversity is similar in the two species but the synonymous is much higher in <em>P. tremula</em> (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Olson <em>et al.</em> 2010</a>). Due to a smaller effective population size in <em>P. balsamifera</em>, purifying selection will be less efficient and deleterious nonsynonymous mutations will therefore accumulate, something that can be seen in the higher <em>π<sub>a</sub></em>:<em>π<sub>s</sub></em> ratio in <em>P. balsamifera</em> (0.267) compared with <em>P. tremula</em> (0.142) (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Olson <em>et al.</em> 2010</a>). In <em>S. viminalis</em> and <em>S. schwerinii</em>, <em>π<sub>a</sub></em>:<em>π<sub>s</sub></em> = 0.238 and 0.278 and are thus very similar to the values reported in <em>P. balsamifera</em>, suggesting that <em>P. tremula</em> could be an outlier in this respect. Because our data were limited, this would need to be confirmed with a larger dataset.</p><p id="p-66">There were not many differences in nucleotide diversity between the species; however, linkage disequilibrium is more sensitive to capture recent departures from the standard neutral model, and indeed, there were greater differences in LD among the two <em>Salix</em> species compared with the differences in nucleotide diversities. LD was higher in <em>S. viminalis</em> (<em>r<sup>2</sup></em> = 0.40–0.49) than in <em>S. schwerinii</em> (<em>r<sup>2</sup></em> = 0.31–0.34), and LD decayed to 0.2 around 4000 base pairs in <em>S. viminalis</em> and around 1000 base pairs in <em>S. schwerinii</em>. The absolute values of these LD estimates should be treated with caution, however, as they are possibly inflated due to the way the haplotypic phase was determined, but as both species were affected in the same way, the comparison of relative values should remain valid. Difficulties in inferring the haplotype phase happened when one segment contained multiple heterozygotes, which were then excluded from further analysis. Similar to the situation for nucleotide diversities, there are great differences in LD among outcrossing angiosperm trees. In <em>P. tremula</em>, LD decayed rapidly to &lt;0.05 in around a few hundred bases (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Ingvarsson 2005</a>); however, in <em>P. balsamifera</em>, values are more similar to those observed in <em>S. viminalis</em> (mean <em>r<sup>2</sup></em> = 0.52), and LD did not decay in the gene fragments (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Olson <em>et al.</em> 2010</a>). These data are comparable as the phases were determined in a similar fashion. Many factors, such as recombination, mutation, selection, population admixture, and demographic history, affect LD, which could explain why LD differs so much between species. We can only speculate about the cause of the difference in LD between <em>S. viminalis</em> and <em>S. schwerinii</em>.</p><p id="p-67">Because population structure may affect estimates of LD, we checked whether any of the species showed evidence for population subdivision. Each species was analyzed using Structure version 2.3.3 (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Pritchard <em>et al.</em> 2000</a>). The analyses revealed low levels of population subdivision in both species, which could certainly be a consequence of the limited sampling. However, it is also possible that these species do not demonstrate great population structures throughout their whole species ranges due to the high dispersal possibilities, as both seeds and pollen are partly wind dispersed. Even though the <em>Salix</em> genus is generally considered to be insect pollinated, several species are also known to be wind pollinated, as the flowers are morphologically intermediate between wind and insect pollination (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Peeters and Totland 1999</a>). So a possible cause for the difference in LD between the two species is different demographic histories. In contrast to <em>S. schwerinii</em>, <em>S. viminalis</em> has been used for basket making by humans for centuries, and in the eighteenth century, it was extensively used as a crop with several thousands of hectares of cultivated areas in, for example, France and Poland (refs. in <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Lascoux <em>et al.</em> 1996</a>). It is thus possible that the longer LD in <em>S. viminalis</em> is an effect of bottlenecks at domestication events. To test this further, we performed coalescent simulations to find out whether any of the species deviated from the standard neutral model (SNM) using bottleneck and population growth models. Bottlenecks caused by founder or domestication events and population expansions triggered by advances and retreats of glaciers have resulted in deviations from the SNM in many natural plant populations (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Siol <em>et al.</em> 2010</a>). Although the majority of plant populations deviate from the SNM, our analyses show that neither <em>Salix</em> species do. This was particularly unexpected in <em>S. viminalis</em>. A likely explanation is that because demographic changes in <em>S. viminalis</em> occurred recently, they are not reflected in the site frequency spectrum or, at least, they were not captured by a dataset of the size used in the present study. This lack of deviation of the SNM in <em>Salix</em> is most likely again an effect of the large <em>N<sub>e</sub></em> and long-distance dispersal.</p></div></div><div class="section conclusions" id="sec-35"><h2 class="">Conclusion</h2><p id="p-68">This is the first study to present data of species divergence and levels of nucleotide polymorphism and linkage disequilibrium in species from the genus <em>Salix</em>. The results suggest that <em>S. viminalis</em> and <em>S. schwerinii</em> split on average 600,000 years ago and that limited species-wide gene flow has occurred since the split. Both species harbor substantial amounts of genetic variation, which is likely due to large effective population sizes. The large effective population sizes are also possibly the reason why there are many shared polymorphisms and few fixed differences, despite the limited gene flow. Compared with other angiosperm tree species, the rate of decay of LD in both species is fairly slow, particularly in <em>S.viminalis</em>. The difference between the two <em>Salix</em> species is possibly due to different demographic histories: <em>S. viminalis</em> has been partly domesticated in Europe, whereas <em>S. schwerinii</em> has not. The rate of decay of LD has implications for the potential utility of association mapping as it effectively determines whether genome-wide scans are feasible or whether a candidate gene approach has to be used. The slow LD decay in these species suggests that haplotype structure may be strong and functional sites may be linked to nearby sites that have no influence on function. However, future studies are required to estimate LD across greater distances to determine the variation of LD across the genome.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-69">We thank Ingrid Eriksson and Yvonne Tillman for carrying out excellent lab work, Inger Åhman and Stig Larsson for <em>S. schwerinii</em> samples, and Rose-Marie Rytter for drawing <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>. This study, which has been carried out as part of the project “High and sustainable biomass production of <em>Salix</em>: bridging molecular genetics, ecophysiology and plant breeding,” was supported by the Swedish Energy Agency, the Swedish University of Agricultural Sciences, and Lantmännen Agroenergi AB. M.L. thanks the Chinese Academy of Sciences for financial support and Li Haipeng for his hospitality during his stay in Shanghai. We thank the associate editor and two anonymous reviewers for helpful comments that greatly improved the article.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000539/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Sequence data from this article have been deposited with the EMBL/GenBank Data Libraries under accession nos. HQ625673–HQ628624.</p></li><li class="fn" id="fn-18"><p id="p-70"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-06-07"><span class="received-label">Received </span>June 7, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-08-24"><span class="accepted-label">Accepted </span>August 24, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Berlin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.1" data-doi="10.1007/s001220050442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alström-Rapaport</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gullberg</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Sex determination and sex ratio in the dioecious shrub <em>Salix viminalis</em> L.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D94%26rft.spage%253D493%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.2" data-doi="10.1111/j.1558-5646.2009.00662.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Becquet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Learning about modes of speciation by computational approaches.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">2547</span>–<span class="cit-lpage">2562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D63%26rft.spage%253D2547%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2009.00662.x%26rft_id%253Dinfo%253Apmid%252F19228187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2009.00662.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19228187&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270651800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.3" data-doi="10.1186/1471-2164-11-129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagercrantz</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Arnold</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Öst</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rönnberg-Wästljung</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">High-density linkage mapping and evolution of paralogs and orthologs in <em>Salix</em> and <em>Populus</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DBerlin%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.spage%253D129%26rft.epage%253D129%26rft.atitle%253DHigh-density%2Blinkage%2Bmapping%2Band%2Bevolution%2Bof%2Bparalogs%2Band%2Borthologs%2Bin%2BSalix%2Band%2BPopulus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-129%26rft_id%253Dinfo%253Apmid%252F20178595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20178595&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.4" data-doi="10.1007/s11222-009-9116-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blum</span>  <span class="cit-name-given-names">M. G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francois</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Non-linear regression models for Approximate Bayesian Computation.</span> <abbr class="cit-jnl-abbrev">Stat. Comput.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat.%2BComput.%26rft.volume%253D20%26rft.spage%253D63%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11222-009-9116-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11222-009-9116-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000273403900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.5" data-doi="10.1111/j.1365-2699.2004.01203.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brubaker</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lozhkin</span>  <span class="cit-name-given-names">A. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Beringia as a glacial refugium for boreal trees and shrubs: new perspectives from mapped pollen data.</span> <abbr class="cit-jnl-abbrev">J. Biogeogr.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiogeogr.%26rft.volume%253D32%26rft.spage%253D833%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2699.2004.01203.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2699.2004.01203.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000228751400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-web" id="cit-1.5.387.6"><div class="cit-metadata unstructured">Darwin, C., 1865 Darwin to M. E. Wichura, 3 Feb 1865 [letter 4675a]. Darwin Correspondence Database. Available at: <a href="http://www.darwinproject.ac.uk/entry-4765a">www.darwinproject.ac.uk/entry-4765a</a>. Accessed: 11/07/2010.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.7" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laval</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Arlequin ver. 3.0: an integrated software package for population genetics data analysis.</span> <abbr class="cit-jnl-abbrev">Evol. Bioinform. Online</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEvol%2BBioinform%2BOnline%26rft.aulast%253DExcoffier%26rft.auinit1%253DL.%26rft.volume%253D1%26rft.spage%253D47%26rft.epage%253D50%26rft.atitle%253DArlequin%2B%2528version%2B3.0%2529%253A%2Ban%2Bintegrated%2Bsoftware%2Bpackage%2Bfor%2Bpopulation%2Bgenetics%2Bdata%2Banalysis.%26rft_id%253Dinfo%253Apmid%252F19325852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19325852&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.9" data-doi="10.1086/323612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frisse</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartoszewicz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donfack</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Gene conversion and different population histories may explain the contrast between polymorphism and linkage disequilibrium levels.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">843</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DFrisse%26rft.auinit1%253DL.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D831%26rft.epage%253D843%26rft.atitle%253DGene%2Bconversion%2Band%2Bdifferent%2Bpopulation%2Bhistories%2Bmay%2Bexplain%2Bthe%2Bcontrast%2Bbetween%2Bpolymorphism%2Band%2Blinkage%2Bdisequilibrium%2Blevels.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F323612%26rft_id%253Dinfo%253Apmid%252F11533915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/323612&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11533915&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171008200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.10" data-doi="10.1111/j.1365-294X.2006.02885.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilchrist</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haughn</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhuang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Use of Ecotilling as an efficient SNP discovery tool to survey genetic variation in wild populations of <em>Populus trichocarpa</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1367</span>–<span class="cit-lpage">1378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DGilchrist%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D1367%26rft.epage%253D1378%26rft.atitle%253DUse%2Bof%2BEcotilling%2Bas%2Ban%2Befficient%2BSNP%2Bdiscovery%2Btool%2Bto%2Bsurvey%2Bgenetic%2Bvariation%2Bin%2Bwild%2Bpopulations%2Bof%2BPopulus%2Btrichocarpa.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2006.02885.x%26rft_id%253Dinfo%253Apmid%252F16626459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2006.02885.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16626459&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236584900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.11" data-doi="10.1007/s00122-002-0979-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aldam</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A genetic linkage map of willow (<em>Salix viminalis</em>) based on AFLP and microsatellite markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">1087</span>–<span class="cit-lpage">1096</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHanley%26rft.auinit1%253DS.%26rft.volume%253D105%26rft.issue%253D6-7%26rft.spage%253D1087%26rft.epage%253D1096%26rft.atitle%253DA%2Bgenetic%2Blinkage%2Bmap%2Bof%2Bwillow%2B%2528%2BSalix%2Bviminalis%2529%2Bbased%2Bon%2BAFLP%2Band%2Bmicrosatellite%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-002-0979-0%26rft_id%253Dinfo%253Apmid%252F12582937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-002-0979-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582937&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179685700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanley</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallott</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karp</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Alignment of a <em>Salix</em> linkage map to the <em>Populus</em> genomic sequence reveals macrosynteny between willow and poplar genomes.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D3%26rft.spage%253D35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.13" data-doi="10.1007/BF01245622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Linkage disequilibrium in finite populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D38%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01245622%26rft_id%253Dinfo%253Apmid%252F24442307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01245622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24442307&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.14" data-doi="10.1016/0040-5809(88)90004-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Variances and covariances of squared linkage disequilibria in finite populations.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D54%26rft.epage%253D78%26rft.atitle%253DVariances%2Band%2Bcovariances%2Bof%2Bsquared%2Blinkage%2Bdisequilibria%2Bin%2Bfinite%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252888%252990004-4%26rft_id%253Dinfo%253Apmid%252F3376052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(88)90004-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3376052&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988M750300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.15" data-doi="10.1093/bioinformatics/18.2.337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Generating samples under a Wright-Fisher neutral model of genetic variation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D338%26rft.atitle%253DGenerating%2Bsamples%2Bunder%2Ba%2BWright-Fisher%2Bneutral%2Bmodel%2Bof%2B%2Bgenetic%2Bvariation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.2.337%26rft_id%253Dinfo%253Apmid%252F11847089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/18.2.337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11847089&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174028100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.16" data-doi="10.1111/j.0014-3820.2002.tb01467.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mathematical consequences of the genealogical species concept.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1557</span>–<span class="cit-lpage">1565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D56%26rft.spage%253D1557%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2002.tb01467.x%26rft_id%253Dinfo%253Apmid%252F12353748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2002.tb01467.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12353748&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178154400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.17" data-doi="10.1534/genetics.104.034959"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Nucleotide polymorphism and linkage disequilibrium within and among natural populations of European aspen (<em>Populus tremula</em> L., Salicaceae).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D169%26rft.spage%253D945%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.034959%26rft_id%253Dinfo%253Apmid%252F15489521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.104.034959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15489521&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227697200037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.18" data-doi="10.1534/genetics.108.090431"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Multilocus patterns of nucleotide polymorphism and the demographic history of <em>Populus tremula</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090431%26rft_id%253Dinfo%253Apmid%252F18716330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvMzI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS81LzM4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.19" data-doi="10.1111/j.1744-7909.2010.01015.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karp</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanley</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trybush</span>  <span class="cit-name-given-names">S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macalpine</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic improvement of willow for bioenergy and biofuels.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BIntegr%2BPlant%2BBiol%26rft.aulast%253DKarp%26rft.auinit1%253DA.%26rft.volume%253D53%26rft.issue%253D2%26rft.spage%253D151%26rft.epage%253D165%26rft.atitle%253DGenetic%2Bimprovement%2Bof%2Bwillow%2Bfor%2Bbioenergy%2Band%2Bbiofuels.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7909.2010.01015.x%26rft_id%253Dinfo%253Apmid%252F21205181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7909.2010.01015.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21205181&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kehl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aas</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rambold</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genotypical and multiple phenotypical traits discriminate <em>Salix</em> x <em>rubens</em> Schrank clearly from its parent species.</span> <abbr class="cit-jnl-abbrev">Plant Syst. Evol.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSyst.%2BEvol.%26rft.volume%253D275%26rft.spage%253D169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiffin</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic diversity, population structure, and migration following rapid range expansion in the Balsam Poplar, <em>Populus balsamifera</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1212</span>–<span class="cit-lpage">1226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DKeller%26rft.auinit1%253DS.%2BR.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D1212%26rft.epage%253D1226%26rft.atitle%253DGenomic%2Bdiversity%252C%2Bpopulation%2Bstructure%252C%2Band%2Bmigration%2Bfollowing%2Brapid%2Brange%2Bexpansion%2Bin%2Bthe%2BBalsam%2Bpoplar%252C%2BPopulus%2Bbalsamifera.%26rft_id%253Dinfo%253Apmid%252F20163548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20163548&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.22" data-doi="10.1038/ng2115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plagnol</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recombination and linkage disequilibrium in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKim%26rft.auinit1%253DS.%26rft.volume%253D39%26rft.issue%253D9%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DRecombination%2Band%2Blinkage%2Bdisequilibrium%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2115%26rft_id%253Dinfo%253Apmid%252F17676040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17676040&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bremer</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Korgviden - nyttoväxter förr och nu.</span> <abbr class="cit-jnl-abbrev">Sven. Bot. Tidskr.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSven.%2BBot.%2BTidskr.%26rft.volume%253D85%26rft.spage%253D185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorsen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gullberg</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Population structure of a riparian willow species, <em>Salix viminalis</em> L.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D68%26rft.spage%253D45%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.25" data-doi="10.1093/bioinformatics/btp187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Librado</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">DnaSP v5: a software for comprehensive analysis of DNA polymorphism data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1451</span>–<span class="cit-lpage">1452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp187%26rft_id%253Dinfo%253Apmid%252F19346325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19346325&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266109500026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.26" data-doi="10.1093/bib/5.4.355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Linkage disequilibrium for different scales and applications.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBriefings%2Bin%2BBioinformatics%26rft.stitle%253DBrief%2BBioinform%26rft.aulast%253DMueller%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D364%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bfor%2Bdifferent%2Bscales%2Band%2Bapplications%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252F5.4.355%26rft_id%253Dinfo%253Apmid%252F15606972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/5.4.355&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15606972&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226263900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-1.5.387.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-source"><em>Molecular Evolutionary Genetics</em></span>. <span class="cit-publ-name">Columbia University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.28" data-doi="10.1111/j.1469-8137.2009.03174.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takebayashi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nucleotide diversity and linkage disequilibrium in balsam poplar (<em>Populus balsamifera</em>).</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">526</span>–<span class="cit-lpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DOlson%26rft.auinit1%253DM.%2BS.%26rft.volume%253D186%26rft.issue%253D2%26rft.spage%253D526%26rft.epage%253D536%26rft.atitle%253DNucleotide%2Bdiversity%2Band%2Blinkage%2Bdisequilibrium%2Bin%2Bbalsam%2Bpoplar%2B%2528Populus%2Bbalsamifera%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03174.x%26rft_id%253Dinfo%253Apmid%252F20122131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03174.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20122131&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276024900031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.29" data-doi="10.1126/science.1180677"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucas-Lledo</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warthmann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The rate and molecular spectrum of spontaneous mutations in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DOssowski%26rft.auinit1%253DS.%26rft.volume%253D327%26rft.issue%253D5961%26rft.spage%253D92%26rft.epage%253D94%26rft.atitle%253DThe%2BRate%2Band%2BMolecular%2BSpectrum%2Bof%2BSpontaneous%2BMutations%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180677%26rft_id%253Dinfo%253Apmid%252F20044577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMjcvNTk2MS85MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNS8zODcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.30" data-doi="10.1139/cjb-77-4-556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peeters</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Totland</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Wind to insect pollination ratios and floral traits in five alpine <em>Salix</em> species.</span> <abbr class="cit-jnl-abbrev">Can. J. Bot.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">556</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BBot.%26rft.volume%253D77%26rft.spage%253D556%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fcjb-77-4-556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/cjb-77-4-556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.31" data-doi="10.1016/j.tree.2009.02.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene flow and species delimitation.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">386</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DPetit%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D24%26rft.issue%253D7%26rft.spage%253D386%26rft.epage%253D393%26rft.atitle%253DGene%2Bflow%2Band%2Bspecies%2Bdelimitation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2009.02.011%26rft_id%253Dinfo%253Apmid%252F19409650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2009.02.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19409650&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267903100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.32" data-doi="10.1086/321275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Linkage disequilibrium in humans: models and data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D69%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D14%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bin%2Bhumans%253A%2Bmodels%2Band%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F321275%26rft_id%253Dinfo%253Apmid%252F11410837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/321275&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11410837&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170108100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D155%26rft.spage%253D945%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10835412&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087475100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.34" data-doi="10.1038/35075590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cargill</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ireland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Linkage disequilibrium in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">411</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DReich%26rft.auinit1%253DD.%2BE.%26rft.volume%253D411%26rft.issue%253D6834%26rft.spage%253D199%26rft.epage%253D204%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35075590%26rft_id%253Dinfo%253Apmid%252F11346797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35075590&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11346797&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168563000052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.35" data-doi="10.1007/s00122-004-1866-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnberg-Wästljung</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glynn</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weih</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">QTL analyses of drought tolerance and growth for a <em>Salix dasyclados</em> x <em>Salix viminalis</em> hybrid in contrasting water regimes.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">549</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DR%25C3%25B6nnberg-W%25C3%25A4stljung%26rft.auinit1%253DA.%2BC.%26rft.volume%253D110%26rft.issue%253D3%26rft.spage%253D537%26rft.epage%253D549%26rft.atitle%253DQTL%2Banalyses%2Bof%2Bdrought%2Btolerance%2Band%2Bgrowth%2Bfor%2Ba%2BSalix%2Bdasyclados%2Bx%2BSalix%2Bviminalis%2Bhybrid%2Bin%2Bcontrasting%2Bwater%2Bregimes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1866-7%26rft_id%253Dinfo%253Apmid%252F15619077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1866-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15619077&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227049900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rönnberg-Wästljung</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsarouhas</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semerikov</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagercrantz</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A genetic linkage map of a tetraploid <em>Salix viminalis</em> x <em>S. dasyclados</em> hybrid based on AFLP markers.</span> <abbr class="cit-jnl-abbrev">For. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">194</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFor.%2BGenet.%26rft.volume%253D10%26rft.spage%253D185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.37" data-doi="10.1111/j.1469-8137.2010.03401.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siol</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The population genomics of plant adaptation.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSiol%26rft.auinit1%253DM.%26rft.volume%253D188%26rft.issue%253D2%26rft.spage%253D313%26rft.epage%253D332%26rft.atitle%253DThe%2Bpopulation%2Bgenomics%2Bof%2Bplant%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03401.x%26rft_id%253Dinfo%253Apmid%252F20696011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03401.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20696011&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282225000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-1.5.387.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skvortsov</span>  <span class="cit-name-given-names">A. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-source">Willows of the USSR and Adjacent Lands</span>. <span class="cit-publ-name">Multilingual Service Division, Bureau No. 125754</span>, <span class="cit-publ-loc">Ottawa, Canada</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.39" data-doi="10.1086/379378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A comparison of bayesian methods for haplotype reconstruction from population genotype data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">1162</span>–<span class="cit-lpage">1169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DStephens%26rft.auinit1%253DM.%26rft.volume%253D73%26rft.issue%253D5%26rft.spage%253D1162%26rft.epage%253D1169%26rft.atitle%253DA%2Bcomparison%2Bof%2Bbayesian%2Bmethods%2Bfor%2Bhaplotype%2Breconstruction%2Bfrom%2Bpopulation%2Bgenotype%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F379378%26rft_id%253Dinfo%253Apmid%252F14574645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/379378&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14574645&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186493400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.40" data-doi="10.1086/319501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A new statistical method for haplotype reconstruction from population data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">978</span>–<span class="cit-lpage">989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DStephens%26rft.auinit1%253DM.%26rft.volume%253D68%26rft.issue%253D4%26rft.spage%253D978%26rft.epage%253D989%26rft.atitle%253DA%2Bnew%2Bstatistical%2Bmethod%2Bfor%2Bhaplotype%2Breconstruction%2Bfrom%2Bpopulation%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F319501%26rft_id%253Dinfo%253Apmid%252F11254454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/319501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11254454&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167666000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D123%26rft.spage%253D585%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2513255&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AX26700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.42" data-doi="10.1093/molbev/msm092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1596</span>–<span class="cit-lpage">1599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm092%26rft_id%253Dinfo%253Apmid%252F17488738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17488738&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.43" data-doi="10.1093/nar/22.22.4673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">4673</span>–<span class="cit-lpage">4680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F22.22.4673%26rft_id%253Dinfo%253Apmid%252F7984417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/22.22.4673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7984417&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PU19900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.44" data-doi="10.1093/bioinformatics/btg316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Libsequence: a C++ class library for evolutionary genetic analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2325</span>–<span class="cit-lpage">2327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DThornton%26rft.auinit1%253DK.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D2325%26rft.epage%253D2327%26rft.atitle%253Dlibsequence%253A%2Ba%2BC%252B%252B%2Bclass%2Blibrary%2Bfor%2Bevolutionary%2Bgenetic%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg316%26rft_id%253Dinfo%253Apmid%252F14630667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14630667&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186919200024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Triest</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trung</span>  <span class="cit-name-given-names">L. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talukder</span>  <span class="cit-name-given-names">A. K. M. M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Puyvelde</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Nuclear cyp73 intron fragment length polymorphism supports morphological analysis of <em>Salix</em> species and hybrids.</span> <abbr class="cit-jnl-abbrev">Plant Biosyst.</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiosyst.%26rft.volume%253D143%26rft.spage%253D555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.46" data-doi="10.1007/s00122-002-0918-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsarouhas</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gullberg</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagercrantz</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">An AFLP and RFLP linkage map and quantitative trait locus (QTL) analysis of growth traits in <em>Salix</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTsarouhas%26rft.auinit1%253DV.%26rft.volume%253D105%26rft.issue%253D2-3%26rft.spage%253D277%26rft.epage%253D288%26rft.atitle%253DAn%2BAFLP%2Band%2BRFLP%2Blinkage%2Bmap%2Band%2Bquantitative%2Btrait%2Blocus%2B%2528QTL%2529%2Banalysis%2Bof%2Bgrowth%2Btraits%2Bin%2BSalix.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-002-0918-0%26rft_id%253Dinfo%253Apmid%252F12582530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-002-0918-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582530&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.47" data-doi="10.1034/j.1601-5223.2003.01695.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsarouhas</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gullberg</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagercrantz</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mapping of quantitative trait loci controlling timing of bud flush in <em>Salix</em>.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DTsarouhas%26rft.auinit1%253DV.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D172%26rft.epage%253D178%26rft.atitle%253DMapping%2Bof%2Bquantitative%2Btrait%2Bloci%2Bcontrolling%2Btiming%2Bof%2Bbud%2Bflush%2Bin%2BSalix.%26rft_id%253Dinfo%253Adoi%252F10.1034%252Fj.1601-5223.2003.01695.x%26rft_id%253Dinfo%253Apmid%252F14641480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1034/j.1601-5223.2003.01695.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14641480&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187074000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.48" data-doi="10.1007/s00122-003-1544-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsarouhas</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gullberg</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagercrantz</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mapping of quantitative trait loci (QTLs) affecting autumn freezing resistance and phenology in <em>Salix</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1335</span>–<span class="cit-lpage">1342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTsarouhas%26rft.auinit1%253DV.%26rft.volume%253D108%26rft.issue%253D7%26rft.spage%253D1335%26rft.epage%253D1342%26rft.atitle%253DMapping%2Bof%2Bquantitative%2Btrait%2Bloci%2B%2528QTLs%2529%2Baffecting%2Bautumn%2Bfreezing%2Bresistance%2Band%2Bphenology%2Bin%2BSalix.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1544-1%26rft_id%253Dinfo%253Apmid%252F14747916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1544-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14747916&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220988400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.49" data-doi="10.1126/science.1128691"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuskan</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Difazio</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bohlmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grigoriev</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray).</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">1596</span>–<span class="cit-lpage">1604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTuskan%26rft.auinit1%253DG.%2BA.%26rft.volume%253D313%26rft.issue%253D5793%26rft.spage%253D1596%26rft.epage%253D1604%26rft.atitle%253DThe%2Bgenome%2Bof%2Bblack%2Bcottonwood%252C%2BPopulus%2Btrichocarpa%2B%2528Torr.%2B%2526amp%253B%2BGray%2529.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1128691%26rft_id%253Dinfo%253Apmid%252F16973872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTMvNTc5My8xNTk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS81LzM4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.50" data-doi="10.1111/j.1365-2745.2007.01269.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ueno</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suyama</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seiwa</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">What makes the sex ratio female-biased in the dioecious tree <em>Salix sachalinensis?</em></span> <abbr class="cit-jnl-abbrev">J. Ecol.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">951</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEcol.%26rft.volume%253D95%26rft.spage%253D951%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2745.2007.01269.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2745.2007.01269.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000249166700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Untergasser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nijveen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bisseling</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geurts</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Primer3Plus, an enhanced web interface to Primer3.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">W71–74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D35%26rft.spage%253DW71%2B74%26rft.atitle%253DPRIMER3PLUS%252C%2BAN%2BENHANCED%2BWEB%2BINTERFACE%2BTO%2BPRIMER3.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.52" data-doi="10.1073/pnas.0507325102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voight</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisse</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Interrogating multiple aspects of variation in a full resequencing data set to infer human population size changes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">18508</span>–<span class="cit-lpage">18513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0507325102%26rft_id%253Dinfo%253Apmid%252F16352722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzUxLzE4NTA4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS81LzM4Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.53" data-doi="10.1016/0040-5809(75)90020-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">On the number of segregating sites in genetical models without recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWatterson%26rft.auinit1%253DG.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D256%26rft.epage%253D276%26rft.atitle%253DOn%2Bthe%2Bnumber%2Bof%2Bsegregating%2Bsites%2Bin%2Bgenetical%2Bmodels%2Bwithout%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252875%252990020-9%26rft_id%253Dinfo%253Apmid%252F1145509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(75)90020-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1145509&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975W295600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.54" data-doi="10.1111/j.1469-8137.2006.01685.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weih</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rönnberg-Wästljung</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glynn</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic basis of phenotypic correlations among growth traits in hybrid willow (<em>Salix dasyclados</em> x <em>S. viminalis</em>) grown under two water regimes.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DWeih%26rft.auinit1%253DM.%26rft.volume%253D170%26rft.issue%253D3%26rft.spage%253D467%26rft.epage%253D477%26rft.atitle%253DGenetic%2Bbasis%2Bof%2Bphenotypic%2Bcorrelations%2Bamong%2Bgrowth%2Btraits%2Bin%2Bhybrid%2Bwillow%2B%2528Salix%2BdasycladosxS.%2Bviminalis%2529%2Bgrown%2Bunder%2Btwo%2Bwater%2Bregimes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2006.01685.x%26rft_id%253Dinfo%253Apmid%252F16626469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2006.01685.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16626469&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236738600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-book" id="cit-1.5.387.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wichura</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1865</span> <span class="cit-source"><em>Die Bastardbefruchtung im Pflanzenreich erläutert an den Bastarden der Weiden</em></span>. <span class="cit-publ-name">Verlag von E Morgenstern</span>, <span class="cit-publ-loc">Breslau</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.387.56" data-doi="10.1093/jhered/92.4.371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">DAMBE: software package for data analysis in molecular biology and evolution.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DXia%26rft.auinit1%253DX.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D371%26rft.epage%253D373%26rft.atitle%253DDAMBE%253A%2BSoftware%2BPackage%2Bfor%2BData%2BAnalysis%2Bin%2BMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F92.4.371%26rft_id%253Dinfo%253Apmid%252F11535656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/92.4.371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11535656&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F387.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170799400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/5/387.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/5/377" title="Generation of Highly Homogeneous Strains of Zebrafish Through Full Sib-Pair Mating" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/5/401" title="Sequence Analysis of the Segmental Duplication Responsible for Paris Sex-Ratio Drive in Drosophila simulans" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/5"><span>Volume 1 Issue 5, October 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/5.cover.gif?itok=Cu1rUVm1" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/5/387" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4_0MZXV3-82uUn36uDPX2Onq0OmLS8CJBdrGVkC4XTQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/5/387" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/5/387" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/5/387" class="active" data-icon-position="" data-hide-link-title="0">Polymorphism and Divergence in Two Willow Species, Salix viminalis L. and Salix schwerinii E. Wolf</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459188" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-8QPlHTWg_Ij59OsGROSLHzYUS2ZXyAr8m1hYTClfSHQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/5/387.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/5/387" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-G_pUiDdys9M18DXo02CANvAcr2BfFiVVVwWHRK-a9lU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/5/387" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/5/387.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459188" id="node459188--2" data-pisa="ggg;1/5/387" data-pisa-master="ggg;g3.111.000539" data-apath="/ggg/1/5/387.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Polymorphism and Divergence in Two Willow Species, <em>Salix viminalis</em> L. and <em>Salix schwerinii</em> E. Wolf</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sofia Berlin</span>, <span class="highwire-citation-author" data-delta="1">Johan Fogelqvist</span>, <span class="highwire-citation-author" data-delta="2">Martin Lascoux</span>, <span class="highwire-citation-author" data-delta="3">Ulf Lagercrantz</span> and <span class="highwire-citation-author" data-delta="4">Ann Christin Rönnberg-Wästljung</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">387-400;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000539 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459188--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sofia Berlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Forest Genetics, Uppsala BioCenter, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sofia%2BBerlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Berlin%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASofia%2BBerlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sofia.berlin@slu.se" class="" data-icon-position="" data-hide-link-title="0">sofia.berlin@slu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Johan Fogelqvist </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Forest Genetics, Uppsala BioCenter, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Ecology and Evolution, Evolutionary Biology Centre, Uppsala University, SE-752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johan%2BFogelqvist%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fogelqvist%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohan%2BFogelqvist%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Martin Lascoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Evolutionary Genomics, CAS-MPG Partner Institute for Computational Biology, Chinese Academy of Sciences, Shanghai, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BLascoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lascoux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BLascoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ulf Lagercrantz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Ecology and Evolution, Evolutionary Biology Centre, Uppsala University, SE-752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ulf%2BLagercrantz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lagercrantz%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUlf%2BLagercrantz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ann Christin Rönnberg-Wästljung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Forest Genetics, Uppsala BioCenter, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ann%2BChristin%2BR%C3%B6nnberg-W%C3%A4stljung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=R%C3%B6nnberg-W%C3%A4stljung%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnn%2BChristin%2BR%25C3%25B6nnberg-W%25C3%25A4stljung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/5/387" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/5/387&amp;title=Polymorphism%20and%20Divergence%20in%20Two%20Willow%20Species%2C%20Salix%20viminalis%20L.%20and%20Salix%20schwerinii%20E.%20Wolf" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/5/387&amp;title=Polymorphism%20and%20Divergence%20in%20Two%20Willow%20Species%2C%20Salix%20viminalis%20L.%20and%20Salix%20schwerinii%20E.%20Wolf" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/5/387&amp;title=Polymorphism%20and%20Divergence%20in%20Two%20Willow%20Species%2C%20Salix%20viminalis%20L.%20and%20Salix%20schwerinii%20E.%20Wolf" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/5/387&amp;text=Polymorphism%20and%20Divergence%20in%20Two%20Willow%20Species%2C%20Salix%20viminalis%20L.%20and%20Salix%20schwerinii%20E.%20Wolf" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/5/387&amp;title=Polymorphism%20and%20Divergence%20in%20Two%20Willow%20Species%2C%20Salix%20viminalis%20L.%20and%20Salix%20schwerinii%20E.%20Wolf" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/5/387&amp;t=Polymorphism%20and%20Divergence%20in%20Two%20Willow%20Species%2C%20Salix%20viminalis%20L.%20and%20Salix%20schwerinii%20E.%20Wolf" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/5/387&amp;title=Polymorphism%20and%20Divergence%20in%20Two%20Willow%20Species%2C%20Salix%20viminalis%20L.%20and%20Salix%20schwerinii%20E.%20Wolf" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/5/387&amp;title=Polymorphism%20and%20Divergence%20in%20Two%20Willow%20Species%2C%20Salix%20viminalis%20L.%20and%20Salix%20schwerinii%20E.%20Wolf" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459188/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459188" id="node459188--3" data-pisa="ggg;1/5/387" data-pisa-master="ggg;g3.111.000539" data-apath="/ggg/1/5/387.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Polymorphism and Divergence in Two Willow Species, <em>Salix viminalis</em> L. and <em>Salix schwerinii</em> E. Wolf</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sofia Berlin</span>, <span class="highwire-citation-author" data-delta="1">Johan Fogelqvist</span>, <span class="highwire-citation-author" data-delta="2">Martin Lascoux</span>, <span class="highwire-citation-author" data-delta="3">Ulf Lagercrantz</span> and <span class="highwire-citation-author" data-delta="4">Ann Christin Rönnberg-Wästljung</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">387-400;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000539 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459188--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sofia Berlin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Forest Genetics, Uppsala BioCenter, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sofia%2BBerlin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Berlin%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASofia%2BBerlin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sofia.berlin@slu.se" class="" data-icon-position="" data-hide-link-title="0">sofia.berlin@slu.se</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Johan Fogelqvist </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Forest Genetics, Uppsala BioCenter, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Ecology and Evolution, Evolutionary Biology Centre, Uppsala University, SE-752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johan%2BFogelqvist%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fogelqvist%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohan%2BFogelqvist%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Martin Lascoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Evolutionary Genomics, CAS-MPG Partner Institute for Computational Biology, Chinese Academy of Sciences, Shanghai, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BLascoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lascoux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BLascoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ulf Lagercrantz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Ecology and Evolution, Evolutionary Biology Centre, Uppsala University, SE-752 36 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ulf%2BLagercrantz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lagercrantz%20U&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AUlf%2BLagercrantz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Ann Christin Rönnberg-Wästljung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology and Forest Genetics, Uppsala BioCenter, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ann%2BChristin%2BR%C3%B6nnberg-W%C3%A4stljung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=R%C3%B6nnberg-W%C3%A4stljung%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnn%2BChristin%2BR%25C3%25B6nnberg-W%25C3%25A4stljung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=387&amp;rft.epage=400&amp;rft.atitle=Polymorphism%20and%20Divergence%20in%20Two%20Willow%20Species%2C%20Salix%20viminalis%20L.%20and%20Salix%20schwerinii%20E.%20Wolf&amp;rft.volume=1&amp;rft.issue=5&amp;rft.date=2011-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Berlin%2C+Sofia&amp;rft.au=Fogelqvist%2C+Johan&amp;rft.au=Lascoux%2C+Martin&amp;rft.au=Lagercrantz%2C+Ulf&amp;rft.au=R%C3%B6nnberg-W%C3%A4stljung%2C+Ann+Christin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459188/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459188/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459188/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459188/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459188/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459188/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459188/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459188/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459188/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459188/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459188/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459188/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496056" id="node496056" data-pisa="ggg;g3.120.401188v1" data-pisa-master="ggg;g3.120.401188" data-apath="/ggg/early/2020/03/27/g3.120.401188.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401188" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">MYCs and PIFs Act Independently in Arabidopsis Growth Regulation</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496055" id="node496055" data-pisa="ggg;g3.120.401106v1" data-pisa-master="ggg;g3.120.401106" data-apath="/ggg/early/2020/03/27/g3.120.401106.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401106" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Identifying Small Molecules That Promote Quasipalindrome-Associated Template-Switch Mutations in <em>Escherichia coli</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496053" id="node496053" data-pisa="ggg;g3.120.401160v1" data-pisa-master="ggg;g3.120.401160" data-apath="/ggg/early/2020/03/27/g3.120.401160.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/27/g3.120.401160" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Evaluation of STAR and Kallisto on Single Cell RNA-Seq Data Alignment</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/5/387" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-32" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-35" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/5/387.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/5/387.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/5/387.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/5/387.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
