<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/1/5/327" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/1/5/327.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/1/5/327.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/1/5/327.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/5.cover.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/1/5/327.atom" />
<meta name="HW.pisa" content="ggg;1/5/327" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Replication Stress-Induced Chromosome Breakage Is Correlated with Replication Fork Progression and Is Preceded by Single-Stranded DNA Formation" />
<meta name="DC.Identifier" content="10.1534/g3.111.000554" />
<meta name="DC.Date" content="2011-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2011 Feng et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Chromosome breakage as a result of replication stress has been hypothesized to be the direct consequence of defective replication fork progression, or “collapsed” replication forks. However, direct and genome-wide evidence that collapsed replication forks give rise to chromosome breakage is still lacking. Previously we showed that a yeast replication checkpoint mutant mec1-1 , after transient exposure to replication impediment imposed by hydroxyurea (HU), failed to complete DNA replication, accumulated single-stranded DNA (ssDNA) at the replication forks, and fragmented its chromosomes. In this study, by following replication fork progression genome-wide via ssDNA detection and by direct mapping of chromosome breakage after HU exposure, we have tested the hypothesis that the chromosome breakage in mec1 cells occurs at collapsed replication forks. We demonstrate that sites of chromosome breakage indeed correlate with replication fork locations. Moreover, ssDNA can be detected prior to chromosome breakage, suggesting that ssDNA accumulation is the common precursor to double strand breaks at collapsed replication forks." />
<meta name="DC.Contributor" content="Wenyi Feng" />
<meta name="DC.Contributor" content="Sara C. Di Rienzi" />
<meta name="DC.Contributor" content="M. K. Raghuraman" />
<meta name="DC.Contributor" content="Bonita J. Brewer" />
<meta name="article:published_time" content="2011-10-01" />
<meta name="article:section" content="Investigation" />
<meta name="citation_title" content="Replication Stress-Induced Chromosome Breakage Is Correlated with Replication Fork Progression and Is Preceded by Single-Stranded DNA Formation" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Chromosome breakage as a result of replication stress has been hypothesized to be the direct consequence of defective replication fork progression, or “collapsed” replication forks. However, direct and genome-wide evidence that collapsed replication forks give rise to chromosome breakage is still lacking. Previously we showed that a yeast replication checkpoint mutant &lt;i&gt;mec1-1&lt;/i&gt;, after transient exposure to replication impediment imposed by hydroxyurea (HU), failed to complete DNA replication, accumulated single-stranded DNA (ssDNA) at the replication forks, and fragmented its chromosomes. In this study, by following replication fork progression genome-wide via ssDNA detection and by direct mapping of chromosome breakage after HU exposure, we have tested the hypothesis that the chromosome breakage in &lt;i&gt;mec1&lt;/i&gt; cells occurs at collapsed replication forks. We demonstrate that sites of chromosome breakage indeed correlate with replication fork locations. Moreover, ssDNA can be detected prior to chromosome breakage, suggesting that ssDNA accumulation is the common precursor to double strand breaks at collapsed replication forks.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2011/10/01" />
<meta name="citation_mjid" content="ggg;1/5/327" />
<meta name="citation_id" content="1/5/327" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/1/5/327" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/1/5/327.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/1/5/327.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/1/5/327.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.111.000554" />
<meta name="citation_pmid" content="22384343" />
<meta name="citation_volume" content="1" />
<meta name="citation_issue" content="5" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigation" />
<meta name="citation_firstpage" content="327" />
<meta name="citation_lastpage" content="335" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Wenyi Feng" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Upstate Medical University, Syracuse, New York 13210" />
<meta name="citation_author_email" content="fengw@upstate.edu" />
<meta name="citation_author" content="Sara C. Di Rienzi" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="M. K. Raghuraman" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Bonita J. Brewer" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=A. Admire;citation_author=L. Shanks;citation_author=N. Danzl;citation_author=M. Wang;citation_author=U. Weier;citation_title=Cycles of chromosome instability are associated with a fragile site and are increased by defects in DNA replication and checkpoint controls in yeast.;citation_volume=20;citation_year=2006;citation_pmid=16384935;citation_doi=10.1101/gad.1392506" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=J. P. Banath;citation_author=S. H. Macphail;citation_author=P. L. Olive;citation_title=Radiation Sensitivity, H2AX Phosphorylation, and Kinetics of Repair of DNA Strand Breaks in Irradiated Cervical Cancer Cell Lines;citation_pages=7144-7149;citation_volume=64;citation_year=2004;citation_issue=19;citation_pmid=15466212;citation_doi=10.1158/0008-5472.CAN-04-1433" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=H. G. Blitzblau;citation_author=G. W. Bell;citation_author=J. Rodriguez;citation_author=S. P. Bell;citation_author=A. Hochwagen;citation_title=Mapping of meiotic single-stranded DNA reveals double-stranded-break hotspots near centromeres and telomeres.;citation_pages=2003-2012;citation_volume=17;citation_year=2007;citation_issue=23;citation_pmid=18060788;citation_doi=10.1016/j.cub.2007.10.066" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=C. Buhler;citation_author=V. Borde;citation_author=M. Lichten;citation_title=Mapping meiotic single-strand DNA reveals a new landscape of DNA double-strand breaks in Saccharomyces cerevisiae.;citation_pages=e324-e324;citation_volume=5;citation_year=2007;citation_issue=12;citation_pmid=18076285;citation_doi=10.1371/journal.pbio.0050324" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. M. Casper;citation_author=P. W. Greenwell;citation_author=W. Tang;citation_author=T. D. Petes;citation_title=Chromosome aberrations resulting from double-strand DNA breaks at a naturally occurring yeast fragile site composed of inverted ty elements are independent of Mre11p and Sae2p.;citation_volume=183;citation_year=2009;citation_pmid=19635935;citation_doi=10.1534/genetics.109.106385" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. M. Casper;citation_author=P. A. Mieczkowski;citation_author=M. Gawel;citation_author=T. D. Petes;citation_title=Low levels of DNA polymerase alpha induce mitotic and meiotic instability in the ribosomal DNA gene cluster of Saccharomyces cerevisiae.;citation_pages=e1000105-e1000105;citation_volume=4;citation_year=2008;citation_issue=6;citation_pmid=18584028;citation_doi=10.1371/journal.pgen.1000105" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. S. Cha;citation_author=N. Kleckner;citation_title=ATR Homolog Mec1 Promotes Fork Progression, Thus Averting Breaks in Replication Slow Zones;citation_pages=602-606;citation_volume=297;citation_year=2002;citation_issue=5581;citation_pmid=12142538;citation_doi=10.1126/science.1071398" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=C. Cotta-Ramusino;citation_author=D. Fachinetti;citation_author=C. Lucca;citation_author=Y. Doksani;citation_author=M. Lopes;citation_title=Exo1 processes stalled replication forks and counteracts fork reversal in checkpoint-defective cells.;citation_pages=153-159;citation_volume=17;citation_year=2005;citation_issue=1;citation_pmid=15629726;citation_doi=10.1016/j.molcel.2004.11.032" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Feng;citation_author=J. Bachant;citation_author=D. Collingwood;citation_author=M. K. Raghuraman;citation_author=B. J. Brewer;citation_title=Centromere replication timing determines different forms of genomic instability in Saccharomyces cerevisiae checkpoint mutants during replication stress.;citation_pages=1249-1260;citation_volume=183;citation_year=2009;citation_pmid=19805819;citation_doi=10.1534/genetics.109.107508" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=W. Feng;citation_author=D. Collingwood;citation_author=M. E. Boeck;citation_author=L. A. Fox;citation_author=G. M. Alvino;citation_title=Genomic mapping of single-stranded DNA in hydroxyurea-challenged yeasts identifies origins of replication.;citation_pages=148-155;citation_volume=8;citation_year=2006;citation_issue=2;citation_pmid=16429127;citation_doi=10.1038/ncb1358" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=F. J. Lemoine;citation_author=N. P. Degtyareva;citation_author=K. Lobachev;citation_author=T. D. Petes;citation_title=Chromosomal translocations in yeast induced by low levels of DNA polymerase a model for chromosome fragile sites.;citation_pages=587-598;citation_volume=120;citation_year=2005;citation_issue=5;citation_pmid=15766523;citation_doi=10.1016/j.cell.2004.12.039" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=T. M. Menees;citation_author=S. B. Sandmeyer;citation_title=Transposition of the yeast retroviruslike element Ty3 is dependent on the cell cycle.;citation_pages=8229-8240;citation_volume=14;citation_year=1994;citation_issue=12;citation_pmid=7969160;citation_doi=10.1128/MCB.14.12.8229" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Pan;citation_author=M. Sasaki;citation_author=R. Kniewel;citation_author=H. Murakami;citation_author=H. G. Blitzblau;citation_title=A hierarchical combination of factors shapes the genome-wide topography of yeast meiotic recombination initiation.;citation_pages=719-731;citation_volume=144;citation_year=2011;citation_issue=5;citation_pmid=21376234;citation_doi=10.1016/j.cell.2011.02.009" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=M. Raveendranathan;citation_author=S. Chattopadhyay;citation_author=Y. T. Bolon;citation_author=J. Haworth;citation_author=D. J. Clarke;citation_title=Genome-wide replication profiles of S-phase checkpoint mutants reveal fragile sites in yeast.;citation_pages=3627-3639;citation_volume=25;citation_year=2006;citation_issue=15;citation_pmid=16888628;citation_doi=10.1038/sj.emboj.7601251" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=N. Robine;citation_author=N. Uematsu;citation_author=F. Amiot;citation_author=X. Gidrol;citation_author=E. Barillot;citation_title=Genome-wide redistribution of meiotic double-strand breaks in Saccharomyces cerevisiae.;citation_volume=27;citation_year=2007;citation_pmid=17189430;citation_doi=10.1128/MCB.02063-06" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=G. R. Sutherland;citation_title=Heritable fragile sites on human chromosomes II. Distribution, phenotypic effects, and cytogenetics.;citation_pages=136-148;citation_volume=31;citation_year=1979;citation_issue=2;citation_pmid=453198" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=R. K. Szilard;citation_author=P. E. Jacques;citation_author=L. Laramee;citation_author=B. Cheng;citation_author=S. Galicia;citation_title=Systematic identification of fragile sites via genome-wide location analysis of gamma-H2AX.;citation_pages=299-305;citation_volume=17;citation_year=2010;citation_issue=3;citation_pmid=20139982;citation_doi=10.1038/nsmb.1754" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. J. van Brabant;citation_author=C. D. Buchanan;citation_author=E. Charboneau;citation_author=W. L. Fangman;citation_author=B. J. Brewer;citation_title=An origin-deficient yeast artificial chromosome triggers a cell cycle checkpoint.;citation_pages=705-713;citation_volume=7;citation_year=2001;citation_issue=4;citation_pmid=11336695;citation_doi=10.1016/S1097-2765(01)00216-7" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Replication Stress-Induced Chromosome Breakage Is Correlated with Replication Fork Progression and Is Preceded by Single-Stranded DNA Formation" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/5.cover.gif" />
<meta name="twitter:description" content="Chromosome breakage as a result of replication stress has been hypothesized to be the direct consequence of defective replication fork progression, or “collapsed” replication forks. However, direct and genome-wide evidence that collapsed replication forks give rise to chromosome breakage is still lacking. Previously we showed that a yeast replication checkpoint mutant mec1-1 , after transient exposure to replication impediment imposed by hydroxyurea (HU), failed to complete DNA replication, accumulated single-stranded DNA (ssDNA) at the replication forks, and fragmented its chromosomes. In this study, by following replication fork progression genome-wide via ssDNA detection and by direct mapping of chromosome breakage after HU exposure, we have tested the hypothesis that the chromosome breakage in mec1 cells occurs at collapsed replication forks. We demonstrate that sites of chromosome breakage indeed correlate with replication fork locations. Moreover, ssDNA can be detected prior to chromosome breakage, suggesting that ssDNA accumulation is the common precursor to double strand breaks at collapsed replication forks." />
<meta name="og-title" property="og:title" content="Replication Stress-Induced Chromosome Breakage Is Correlated with Replication Fork Progression and Is Preceded by Single-Stranded DNA Formation" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/1/5/327" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Chromosome breakage as a result of replication stress has been hypothesized to be the direct consequence of defective replication fork progression, or “collapsed” replication forks. However, direct and genome-wide evidence that collapsed replication forks give rise to chromosome breakage is still lacking. Previously we showed that a yeast replication checkpoint mutant mec1-1 , after transient exposure to replication impediment imposed by hydroxyurea (HU), failed to complete DNA replication, accumulated single-stranded DNA (ssDNA) at the replication forks, and fragmented its chromosomes. In this study, by following replication fork progression genome-wide via ssDNA detection and by direct mapping of chromosome breakage after HU exposure, we have tested the hypothesis that the chromosome breakage in mec1 cells occurs at collapsed replication forks. We demonstrate that sites of chromosome breakage indeed correlate with replication fork locations. Moreover, ssDNA can be detected prior to chromosome breakage, suggesting that ssDNA accumulation is the common precursor to double strand breaks at collapsed replication forks." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/1/5.cover.gif" />
    <title>Replication Stress-Induced Chromosome Breakage Is Correlated with Replication Fork Progression and Is Preceded by Single-Stranded DNA Formation | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"KHzauIkWuA_xSsY61RQsiZe9iw43rOL9xWPhTu0FeDc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459183","apath":"\/ggg\/1\/5\/327.atom","pisa":"ggg;1\/5\/327","ac":{"\/ggg\/1\/5\/327.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/1\/5\/327.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;1\/5\/327"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;1\/5\/327"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/1\/5\/327":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nd1vgvtm9D1V+mStM0joOLID21UsyhDF\\\\\\\/wl\\\\\\\/cGMrgqgS9c9FvTuuSkiwQsfGCf+6uQjcPrmsSoTYovilpsY6LU\\\\\\\/99nohyIc4lXUfCjrcGjA7Wz+aOFmcm3a4sf07M3RtvDGH3WI95NGeiOFJ7al4JivambYMONyg9DUtdwYMpMvTqC85BtZZRi7QSzVkY62B8OuJBHwVz5DU+jmXZOvVPdK56kSd5CapdgBf32fFmIalmHnf5hTyEx3sdqKI9k3O73Rk2hS7PtHkqqsjsEgMcbxi1prOhzAi1\\\\\\\/X3ICSdgJthlRZS4lJGBfRw5MBrNHymX5XH0jIRkjt3lCS557u1z3gF3yomZ5u1LlpqGoodFzOUTG7gi4Q6RLu1I2cKsoGDq8qIU84jMsYdMmiDbR9OgXFyrLqqziURRgQgCw5OVfnMr1Eaq3zCEz\\\\\\\/HoshcH2t5s8NTN2UbfSu45CFuTAdMvI2S4dfBtH1\\\\\\\/rjk0N51ld7sszbFFxMvuP5qLgT2QP9PHq2xHHnMWy8rjlkQUJoR1JojYUCAoOO85pIn4SKNHgnWOOVMiTWa13XfBb+vdHfqKECt9Ax2mfEVsgHMk0uTA8zLqrEPWN2r5ndxOjxIcZyjwDdnhm7lysrP4a1UZ7LfTFCllnU51II2\\\\\\\/zf5KWeIUcWXvuSrj0OE8DEj\\\\\\\/fatRPuD\\\\\\\/9L0v8p098oAY3U4OBwdtL+\\\\\\\/hdv2iwRmceYy5sA5La9rqnO5GeZkvuIuOWwzt761v9BIzPIu8gBG4JFOengIisqHPslJVknxaKVBjornE\\\\\\\/IEIH9i\\\\\\\/j3kA+8fu0hj3Jijsyb\\\\\\\/Ro\\\\\\\/myEGL4WTj3+v8vMxv3oIikSduK0d\\\\\\\/OyOu3LN87g\\\\\\\/XZEydJtUDPmuZCtvMAFb+NfXFjSDZINiSbRLXNgO7lXeMB4XgZ8quinlcgDnMSjscAYvSXOAbwPTW3BhCZIrNHzcwLn\\\\\\\/2GfiTqAgdcc95dHX+cNL6NibPfIMJcZ6\\\\\\\/B+8Hi2CrvObXrkYZH0kyLjQClPvxXKCwt0ZBjz08BuPF+GcL75CPVlu3kGa71EJ11UqFnoBOmCfmZPj3CvwD9UttQCOaMyvTsu9dYe5zoMzdt6DDS0PGuRkQbrGD\\\\\\\/sIuZ6SWqzoSkKDSneDNJUBM+vyvyPze1Qe13rW7zVh+zM8Xk1jamORGk2b+HZfyw4CjfD3\\\\\\\/Aa8YVVD0QxAtRwOYNeYl3L5vuUTYjZGvBioX6QD9NsMMkq5Ht9OnsppkC7FnBtOocO+LyxigqhXJ\\\\\\\/EIIEW4BXBKdUq\\\\\\\/axf2KPJOMT\\\u0022,\\\u0022iv\\\u0022:\\\u0022zPZjgUICkR9qoElfOEYW7g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228a14276e5ee9a132a4eab64b8e349da2d6cff9dca51abfec3b0211337cef4da1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022D10Ehe\\\\\\\/OYOpF6\\\\\\\/\\\\\\\/BIoY+DlDRpfnbEnYS2m6TdNDasHCQ\\\\\\\/1Q2nI9S3Hgr9zV\\\\\\\/AnlXQLA+32YOX\\\\\\\/mWnm6v5YtiQDm8OuArPi5QW9cO\\\\\\\/E5uR0HRLfHv47bl7D1QbBA7uGzttNH7XFNp5cKG7Y4tSy3E7kArmGsUOgC20kZnU1ZWL4oa\\\\\\\/jJwUScJAb\\\\\\\/JlgJ\\\\\\\/l3Acymt1IqwByIfNQL8GJaFN\\\\\\\/mpiF4P1NTP2qTANakHLsO4RTtfB\\\\\\\/rBd5sHowKCWXcFQLRrfXnDkMxlbL\\\\\\\/rgZ4EfeBIVslotAAIxhmIeet6jGoDlSEXzDFAf2Az824XNqYpVJey5UahkJat9oDJt7j\\\\\\\/kp0+OvwTp4FWn\\\\\\\/IdgPg7WrZ6Y\\\\\\\/tP4JJo+wbZ+aN10c3S4jHaOl\\\\\\\/rvtlgW+TmgMc7oBnTPVIRCc9u2tckbyA3KkiqMbUTovlykwN+8J1kFxZqbqiblcXu8nzbqyFTzpKvKiyk0PP0+IVnT+vsS3dUBE3Kin7Lftg1wFVZWJmmXghdiqZCQGFFZqdgwLoAvLNUtzUr\\\\\\\/j\\\\\\\/5\\\\\\\/vm4s1HEMGaUdNvPPjqxkKNF9\\\\\\\/ImNGXBjYhfLZ1rq0GYbnjVF7z08Qx\\\\\\\/FxTc7Bh64Pxe6BK82vPmgi2GQHGUViqZVFRyt\\\\\\\/zcOnun1fBwwOqD42BhPQUK2WgMeNtlFJ4usm6AHRwjzmY\\\\\\\/OazNo\\\\\\\/YElFlMk6hp8AH+02ShZdj1APs0fzz5wbTv3WRe+Vt47ozVgkmEpueH71PMNepTyu8sMOVuAjynCNMO+60g6MxRaLVaZtDVQVr3Pyzmhrj6IRi7syWJUlLMMmldWKic9B5TAJMZgsWqsztMg2Xz+FKhfHFDGgJxBzDS0pP8WDIwrdtUBnsJg0LIZ5FtWZp+InlTm9CuPCTd1IAbJ1x1iyS0R1FRkAGBGv0UYGsToYvWtL9z5c\\\\\\\/MUeH0R\\\\\\\/JzYeZ\\\\\\\/rp\\\\\\\/wBNroZahzpp\\\\\\\/cO0tVLLhGUjK7WQME\\\\\\\/SYBzm1x1y5nzaYnFt4SWIduefBKsrr3HonsFwvOcZX\\\\\\\/FqmGJ0OW4uHRytEjqKkxuTZALBm+KM98PXOH5+mZSN6Xy6GUWp4MI9zoEuwzktj8\\\\\\\/4o6YtfJGhYdz70FcowSzeBXatG8lMeLnBQhyZJif6WYvEXx\\\\\\\/4CajdaBXUF5jprQDElNZdU5sZSQLCnejFZpoBoTbnB7LbJxXnpmixwd77o3uW\\\\\\\/NGtiMUCwlTSlgG4X64SgRuipyl\\\\\\\/XB+lwoOX0h6keMFWaM2hl5fvnZpRvTfF8PruW04lBv4QxkgNHITB+mDilOf\\\\\\\/unHRHFhXmjK69XX7U1x2tshLOSLdY3ZUPD20xmyN4B7BkN7evlbswj3wgQQOGXyI10obQH02dwZoQ1GN2RvunB8GjIX\\\\\\\/F97jtyR6oW\\\\\\\/v6BEcVk9zknGy8f2601+ywp1nAuFhdCPKcwYAop5G6Ls21IgjvfZnxBqv3qbe+oU7JZ8h1cEmXfM4mVpwE8oMT\\\\\\\/tBPuo6gPc1ZqbZjWdSMKPamqyE0phvv9BT9PtFsPG0LU0lF1zaRNkIEP60q6QaZYCbJrJly6t2asg3m5+7aKR\\\\\\\/ruOehWsoA6LKHFLye\\\\\\\/LNFGyu6ry7DL\\\\\\\/VaYx4T1PSwfmRBZIWNbtbs\\\\\\\/F5aL4t9cEj9SDBZkDTWDd\\\\\\\/FdeSbvza88pRqThA2\\\\\\\/7I19PXfKFEEbtS\\\\\\\/PAAHc\\\\\\\/05dOMzgPH9H+Hr51B9eV73B0FlNtklQGgBctbhjo9FGjRKX9F0fVhd1TANhS4F+a2Acgp1+ftAqv0NI\\\\\\\/NBLZ06o1XAElrppJ4HmwBK2AXwCj5kY+xFNSQ7pKxLlDwfK4LzuKTet3Q+i3MpRVxpclTEoh0T+KR8WiTpj4JrqEff6LMBqwhZTbeycRN4k3MaX4jIcniGxAarP5804N8Y\\\\\\\/ImwzWntGuLRCDNT6ktf6UfWvvdCmE7z1EyCkTz43Ob9S4H361MbDO5\\\\\\\/PSP1uKSFIOCXi1fGns\\\\\\\/WvneF61ZJrlFC3dnWo8v7fux3\\\\\\\/D2+pALFZIE7R4ANcc3LFrq8p4Aq28muSF7WSxYL8gbVZIWJsRQGCDdnFmCCiSjKkxSZlkqS6eDtAkL9EUubYbjuwnd+UHjs5uWeYj7\\\\\\\/XYmMdX5yq1I53jAkFIPVoSd3HKmLff04+yf5qfODJPNK19gFKb4B5Zoor4vqQGCVY3ZJRnEoWIBo4E4nEil\\\\\\\/+CJjpZFGy+Nv1lL+LaJxnFArx3yo2VKlga0SWOkpzTje5SjRNA3IKkyP5VBx9JLL7yn+yVWuAmaXTIKujMeF+Eo5wWVOO4nfUFYQXjjPE+U7kDHLzAHVu3bLZhMO00ZBTNxHCJjWqzSZQfVHw2ZVFk7BS5pgNVYklQXOjypEKB1\\\\\\\/jVFxAyaFX\\\\\\\/xgGU84Sel2+\\\\\\\/Z9wLXI9lyp3A2CAw4GveSXRIcXwmRbj98qunRXpG4NvBERsiFcKIMBRjCfjad3WTv4QDX+LW9Ns8Xd2P9BhHjDRZ6YGo9vTbHiWx70Q8A4vwanDE2Cv86a7SyZNtu2DYE6tNtklkeOvMtqTaUw8MMb0NZy45SajK1uM+xiWkTLTxjJV6OL3G9QYbvbGLTP0iWcO1ffaN+70jZwvGCLJ5tRZirq1avECWdTrMHbES71bbrsLS9GI9JgXY+0PoqJxnOTBV0nboCNH6nYGoDLMBtpC4qVuDduVWCOvf4+wyCO9Jazrc4+0ekWM4YTP0ur51ALCrtZskJ8F44HjdCwKI5OsCFd7ruJaiEOd8HPO9Ad2X6opDkWx1yuQ234vcjMRyHUtmBt\\\\\\\/F50iHGMVQt2X\\\\\\\/oydFjRMp8S6ZLuCzACWT6m4+uxrOaHr29i7G5cHiWXwLA8WWLId32T1vvnVe9nZcgFCVJIWP1+19evHTI61f73p4mJMxl9ycpmIDYs2oMejKhPqpud2\\\\\\\/ka6thxx4TLvU\\\\\\\/HlCofx4uyrA1XXr2Lfmpe7nKTb0WZc5lGl5EVjizkwIXRMA87QFWPDTsbnRuNmexJydOLuxrKB6WZYbqgCrJEhcoFdHjwDqDUmoj3WENsK1+eiG6yeDTDo7QTIYDuGkYu0hL8lTlzk20Dc14JbzpL3okz1KVROnfPr3vxY43NgqH89MgTkRdq8lQJ3iwgm7NzhMya7xEoB2s93JM0lXFAYBTvMsSG7u+wX2bb9jJ08wED+kQQt2upU5\\\\\\\/JWMRMc5\\\\\\\/wZZ7GutTn4qfZK1vi32rSRSnrh0QHzUtV7J+pYn6\\\\\\\/SZIc7c8vecXsarY1upRlOReZFRI\\\\\\\/0FLVO1INj\\\\\\\/HGTv2fd\\\\\\\/ofNkjN8aPSKYqtZgwNvyR4Z5tLjwcIIFqNCKbANWHqEhqUEik58vIaj5gnpqrnn0TLeoOGrSVMDxh42ns4Dtheda8aqrMrHC6OyLKgITqj9IrYVCASuG69M4Sw8lUDlw\\\\\\\/oduwmuNYYjdCiLPVNy71jUW3hSqP7yo7vgi07K021NUp7c0uCHTQNbz\\\\\\\/t+Xs5Qje1a+1c1BWRurm7HGJzdrj4sKeDTOtmWSTpAanoEA7LFG7JOyIbZEtPSyxtDaxTrePI6nGfmxXt839I\\\\\\\/2qCLrgj0Iavph0\\\\\\\/V3+ps1N3XR7G+3iOgEvxNZ2wDZSizXBtsdjjegGEjGsv0iNSeaBJbK8KHhN2FL0ouaQbn1\\\\\\\/uj9Brw6wtUux7F7ttZXlALE\\\\\\\/+cLyB+34tbj7IswO7bklhXtNJ0SxG2OllZbqvayAJ2jxSsJ8ScrmYEwtuwsumAzQpmgFMAbURS0zb+\\\\\\\/lRezjdwJyB9G56Fmdt7Ezkhe9hawFZ+Oz6aez68iG\\\\\\\/o+u\\\\\\\/W6hVXTYCnTyArOnZ2lfidgLfVR1lDw5Tcj27wLlcvHUoAfeydVO+uLj536TDnYufxOxPuPb3nxSOTjGKyAcF4exhFvPrNFGzu1in02ZgcPEVp0PBJ\\\\\\\/pFqRgu4kuWF2ujd\\\\\\\/UzBIfOOtN+uNBYU6Rpx5k0a0stD3vsdYFKfn2in4jyrb9Bz46\\\\\\\/DSkYRFcEIi3YIvaBResTv2LwVn6XC6mEQ9tVvgouv43o3AUbjmJV4zzTHfMPF6T\\\\\\\/+AhW3Ml\\\\\\\/5VGPkGzWKCXppil0vesE3pQAKTKP451JWAFdkOzn0Tb7Z+ygFuzrgSXCz6wTwTE1bVgMMlb\\\\\\\/ZZ0IsO7mRgA4pYzF8lUw0\\\\\\\/MnzPGRZhdlzhHC9UkR7eUNwDFDW0+BX4AVyJaJi\\\\\\\/iX4TPJYg7u4E\\\\\\\/DapkWiVsMCSFRbux06IiyzvBwXCnsw\\\\\\\/NoNcIgSQsSiOzEYyTzWs+FJlzd9CnV8exBlQa09zqE\\\\\\\/aZhCAEm3N80lwFpo83fj0rvC1j+FWN+bkoxnwG5Vq\\\\\\\/Vbl3kq6BYYcRAXc2hVPeQbMRyVVkDzb7PEJFtbW4On9fg+0USIhqqT1f9E7DKGBgrCGN9u4as90w8\\\\\\\/e\\\\\\\/zWb4pIHLotETvL2c8yC8MeXI83gf1SPtf5zudtrjCawGmYDGyFTGcXwMVlUjCEAx5Wb5qgTHvIZkMutYJk8KX2HSWZQCxV0oAH1FD7xO+l0OnzlcH7LphB53J7UYtN\\\\\\\/0kRmRLF9G8stf1DoxRQm4ANUdSN9NLPPxAQljUD0dxd5cBr4q4qan\\\\\\\/EkG63fwuViJ4Y2BJy2EpdxRs7pglB5rzmxri8oxvi4KefKBqqC2OzkpsvXZmaESmcUZEmLsiNumU0o+Nt4TmU0SkwOg0ww6siqSh15+uRGlAePB9d8K0s2\\\\\\\/Mf3guTtTdYOXjas7cu+xto+F6fI7SpmauEabbu0hFkGS9sDQV4fFyfTmrUZq8fPYAcev71lfdFL3h7CY5dcuKAZGriVwhG+HZiT0enhd55ECX0wTowKD\\\\\\\/6VlGPEP+rJ6Ll5uQ0TWGIfyIJ1uxNw4EP93wSDsZIQI6SrbkkDGpKeVuXbUrtF2dYOqZRzx0nyr0pgibmHPGbmkfUgjqzrXSCsLKJw0yhxMgbQop+Oxd+eUL2dUMl14s8P87sSmY4tTxK\\\\\\\/bpq9RnVG6ML2N2Q18qirCfDRr6LG2LeWdQ6+VP5hxaQln9X7tMPRss4Tu2jyMHkVidF0InXhPvxXWw1p\\\\\\\/vGF6Dqrw0b86W2V\\\\\\\/8n1Op4BvMqoa8toS1MXjRB\\\\\\\/RAEImAINsS9kiMSZC2qwvQCj93ope+5c9keHfMx+HGBUbDXB+BcgfJl26HdQ8IEq9qB30wmdUMK5tt\\\\\\\/wwsp834ZRpjhkHoPKR+tTfKvR0\\\\\\\/YYGtOD4gN2F0THBVsmSu8S7SfJoeqPQPW3y\\\\\\\/q63ajDxF50Wr9l88Q6h9I5iOdkomNh0SNNyBS07qHGQIxuTLy0nUc4p\\\\\\\/1JoOVnVfh4HAX5VvE2s0jdqTdmO7NP4CQwVSdSplVbaEAoYafo4LV89gE8Ld2N3p8Jjs5KQ4WOPRaK5o27eAGTn0D34RFzckmEkjhhY3kM45JqFxWOSisIuzOS+slIXC85xysnHTKEDt0IpdMVS7hQhlm8zjVkYK901P1yG3gldwRLQKKhyTmbGfr\\\\\\\/LO9fPPnf+plERNE8bCqYoZnOBYHzVDniHvkc6en7b02OgLH1JildHUNqeTYTEJSnAZmTdS2wJ44DjY50YZbpzzZxZ1uQ5lsWFOM\\\\\\\/IwgdgMmrGCudmLf0ynJsVPpHxQu6FqWgR8cOu6rk\\\\\\\/lYQVfavZO5vbjmIPyX3dppqdajXcKdbEED8TYbdqmJeGf4QX\\\\\\\/Wuw8R0Q\\\\\\\/EGaqFCI\\\\\\\/Gb4jISsaYMpevB8GhwRojAGFPz\\\\\\\/ufryFOEES7iS9J7YdxPZjshyVSku6d7tUvEFo4iF8vMv92ZHG4WaiDdCqbBOR1KxGh0uzRDeanJXRm+RVyDZ+LNw9LPhUjNEIJo9jSD2HwBKA4Y9CRm+AGsX2SzOdonwgL\\\\\\\/9BOdrOHAQwVGPj5gmm+n2npuavFUsxjIf07ROlcB6b8KUiImxr9MHXbOp6wzbGknr6psskWk0OB9irjdPEYJ\\\\\\\/IY3JOIbYbBio8leFYbMmAmzQVvOOEva\\\\\\\/qvbBgBz+zCvaCYpXbzS0PPhxtTQXUu7nHikRkLxBtT4HWUHED5qTzw9QflSXZZX9hcVpq\\\\\\\/LwUPLwH2FXPjKGsTW5EMD\\\\\\\/exgtc2p0RhQ98\\\\\\\/3PGirU7UxTRxsuZ7wcJukMcvMH\\\\\\\/R6Je8s4DI4\\\\\\\/0QgBMGapqq40oLPhYf0kqlPf2s3Y00Kw15IXcbgXu466BPuOQ9xrzcBkoM85j9WKtf5\\\\\\\/t1mPlGdherEjq3wP\\\\\\\/R3wNr+2++qEPwnMO8oZPHk3F9GxDHlOhbChuFaT8u1fgAzhxkumw7GzM702ESeCXE8KuwrMqQXx7Si9D\\\\\\\/iUe+1BeklWtvK8z\\\\\\\/n4khGNAbgD5gPTqK0pA3Ztp0LHdAUovESMl7tMBmTthSaGbHEnognpzh9WGQh\\\\\\\/mheRJfWAiKLE9Y6lDnXx+IWEj7MvytbFDnjTsxazZOoYduVEhmRnqHKYInLSdouIIw7NPkZuxkYib8KZVppZA9IB6T3RO6cfmqwN6kLe1rg9iRe8NI73s2DyF0NxyYYTbzdcF1Jzhy4WrJY4\\\\\\\/gW480skIjFjl3zMtEGibox9QL1Jo26HU\\\\\\\/kCQi8XmwFU\\\\\\\/L\\\\\\\/A2e8GPkm2aoXa3xT+2BoRExA\\\\\\\/iK41m78k\\\\\\\/c3NSodHNB22L35Nfbk3QLPypqZGkZNJts8SmCdVDaJByBSG1MFdZ2t7l9PUcxDNLwjCsMbAaStdrNw0PMQqI2K3dRSTcHXf7UrXdHv\\\\\\\/89RriadP+58qHXQ8m0Xjwoy3CaOVpUYNLT\\\\\\\/BM5ol84JB\\\\\\\/2zfrttWcuPkjoX4Ggmukf7W9SVy3P1N+rnOsXshPGwKqiwU7E3h\\\\\\\/heQmRvSctNcZ3FIvJed7Pq3cy5P4kf7M97zDyKlsF\\\\\\\/Wux+6aeqTX5lrybOvic5wiSz1+c8ecRgNFKHPQAXUboe9WlKWSfmByNZL6\\\\\\\/Aa4IeuEOwH\\\\\\\/n+uwXrpaLoFiQoqY\\\\\\\/\\\\\\\/Cj19g+EJCJci\\\\\\\/cwKwVnUduUgxACydC77PNM1ENi52SnNIPD7ZWi2WRvZNVWKXFQ3vOunKGxMS1Boai68RxE73PS7YAvIqlkHKJl3yAWc+oOwk4VSKry9rcOlb3AlB9B2v\\\\\\\/0JBd3OG9yGhCjH8wIYsxfww71TuyLmZoHCZSTBOHAMo1R3m4gmYHXkDCEm36DKyIJgqQEjME2SjrBxG+UqaawOFLe4q7Bc0XsGnnIcsEUascHYlX203VuC\\\\\\\/l8pn+C2G2czXevSWVOGYVh5AhejYv1dpwLmrCuLyUFkOYoLOgPpaJywZt9K5NL+yoxOn++PLaNi\\\\\\\/KH4N2Jdca\\\\\\\/I5y0IrrE1t0Q+VcXqIPCAeWuBzB2xbcihImAhhExfMOPK6rA0xrx7pxRZccaTfgeRp+xDCsxldSTG4VFe9XM6E2AY\\\\\\\/Jjcis2pzPkZmYW7KfYsBoGA\\\\\\\/WC2eVQxQx6zp6RquboEI6RtLX7abYF90kv5TFL9LWPdeTOA4k8a10j1FrEjoRUYJQuGAL\\\\\\\/uaZxondCXvcNsBlKnvF56Ue5FEJZFXXS5rBwlSwtMfIRNKcidwr7pgq005OJC9b2M7i0fh4arxQyw5dECaY7KPWYMAUTpG+Ct5kqyBQvd1nmwspddVs0Ozswcuyg0jo37DtuO1pREYwmtcEt8cidb\\\\\\\/N\\\\\\\/C+\\\\\\\/j3D+cIB4X6G2ElvinwTFLUt17sKKe0JYYAe0ApGEAb63kMxlKbzHgjBH+Pp0\\\\\\\/0shDuaRx28UcLSbOea6KSWjEbBf6o8d+SyCo1\\\\\\\/bygTW95heRAQH3ScW\\\\\\\/uOUFRezkvoY7qIiCIctlE9s15mWaXeeKN0XCSdXuaRQcbGp39FQoTt2yHIcnHfADhGCcX+5MNXFYi9YRSx\\\\\\\/7js7NQ2W6C5VHcxKc40wC\\\\\\\/t4o+TzTdXpsQ\\\\\\\/WMbq8nb3Vvdbl7m3\\\\\\\/v4SSfoUfkXQw4ESBvuNYPnXOmqQ8BMo12XSrp8sg4+aVvGcu\\\\\\\/Vrk8r7BohB5nkX51mSZL\\\\\\\/nwaB7c1keG2DhHnNTHom9KIHDlEpK6L1aK36FvFAwjCl1nSHTZS9kvMsEQPA7r4P8DNj4ibNeIQGntxR4VQ9XRI5wt8yi\\\\\\\/cUp62vsoSuL5q\\\\\\\/oSXnJ28wMeINrggkt+0u6\\\\\\\/voo4fn8it2i8Ang8SZzKQlZn2ul5BiHtmHKILFLFfktkZTiFMLKSmJoVWfS5cYZ1b8adWRa58hkApsiBnGMY73ms9orP16GNyYqFnUf916uy2lV1qWT0N43I9OGHXbLHBeW1x8kzGvX1ruQyX6276JCqt3GHlZrxpNtEVZjrD6dobR4Jkha7+Ek1rPLxYFakWSArvetGdUzu2NojdE7PmTXp+fv7uQQl5i4\\\\\\\/dd9heiIgmASpkP4TfRtXnDFjkcvIJ1sWVR8UPfNxcHGd8oB96ivWGV2MkL66d2TtbCXJ2pRxgNzKSc\\\\\\\/os6IijLguIHcI79VvuDd97DITzDTlb7xMpRARPVGmnsj7Y1\\\\\\\/G2wxZP57\\\\\\\/EGdVjUWiLJQugSRVwR0gYlfgyoURAbWRsaTv9DDUouGHrkEcACk\\\\\\\/o4LXrdjdt1ipjXheb\\\\\\\/ETvFfmjFTgWRSWJ9cMOS7xyA9\\\\\\\/eqac9l\\\\\\\/Qi03SHQAOd2Cj\\\\\\\/eRzwI1245DNUQsWQ57WbpdUwoUSLBOFKcVY+ZTBj1fN1A0K7WisogcwNXMN8KlrcAy6EDrM3ur4nkI5G8Vy7XAYlE0Y6h5jB5YWVHlp\\\\\\\/60waAjxAjO39BL7reV5CRx8fad+\\\\\\\/wYX\\\\\\\/y0wdq2TlwyUwuHWE5N4xW3V\\\\\\\/srPmwAT\\\\\\\/WI4q5W9bQF0yOE94Kz+4e05P23C2MFTfw7fCjXohuX26vt6E\\\\\\\/TDs5rhFnSE6RWS2Srp0XQuGczRii4vozTClfKhylZeJdgyhvgwKYk2qj85iuMt5oW\\\\\\\/fuPU2szG6hJINnjbCu0OY095O3GUf6cEjqiGE9YqHS\\\\\\\/qQRyoAAX2pldNfk\\\\\\\/FPqYEWmh0zCRoJ3UC0tsNGBWyOCAXSvgqsdkCeQGOaIAJz5s\\\\\\\/nl5uNiWZEDiB6dK3QQfDP1EPVfwnO+LwIhc53fDYFXcTOV2h8RzoM6lY27LIFSi4lvfy\\\\\\\/R97QjW9x83xN354F9RNk\\\\\\\/Gp6uEB4Yh12DHyH8CFreyLM3GcElr193gzdgSY0vUe7mSouTe6hME+SENFfKfxR2cxSpmb17FVdl+ZKX7dIrmoT7oRPSNJglXnO9eMnVBDkdvSVGdGjgWECnfLF2jM1ZdAlUJNc7F44WGCl9GAmHbZewKEcwozA+YHA8CzNvsAObRIacdsPNr9DXfLMR\\\\\\\/4HbVzhmg4JkrCx0uh\\\\\\\/QWc+qQiet2JVUmdIyYzuvzFanxJkq9cV1vX8dPHokGKlaGCz+E1yFWx1l3iojuuMua461U4y6fizkLqg4mqcaftOC1U\\\\\\\/9jN5XUzPkcSINyI9ugGnZPB4yiMaYgbJJULnmaR2sPD73C5aBOxGx11XcQmv1fkrgiAyQahG2DolzqB+nbKYOKRXyAbBqkNUQGL88bqxqxdy9aN3\\\\\\\/3SGGQ8Wq4Kqewkhz0NcjzLzQ88eUFT0mAJgjDhKGu6o3gStshs\\\\\\\/gFAVr6nInwEFoN9YD9MaUizv0h5TlT\\\\\\\/ph+yQkmg5v3BaG4vfbsF8worazx\\\\\\\/+7v7S6VXQtVSNVCQU5Vr55UBethKLwv6uq1xGS2wQQn2r3G0rxRv2H76llNGf724ujvwg23KHhIbHazg2QWvqNANF\\\\\\\/bJ1nRSUCXgHahiajBYkQVaMhwIG\\\\\\\/MP5tPU9ibbf4vr3Z3hAHld3ZmdYXkPpc0740ThCnUqK41cy3rRhlcP+jR3XUf8xQuHDtLGKgrwXzOwJwauDMNU+XPoxBxnYKA74GqphrSlz45H83ynQwrjZ9ACd0OioyNxUCZeJlJl+iQiE3am2J78E1y\\\\\\\/W+M9h1pe1SIUtj+OCrK3+vdeGIrp13ha7v0H8CQQbR+IqP+K0s1Dleo+HWSqyiusR2bjIxsFEX6YFGrLxPPtFM9s0gfAHRvM2CsCdoUZEpc3CjJXkFigtki8uI1ZzxXWwyt\\\\\\\/0pRut5iEq9gNx2xMTv39stgrVw7lqQzFNekP9WjCIDR0jfBm9NANv0R1YLdsc7egjSCnv2sigYbEuQN4R1EgayFc5PF694rNFem\\\\\\\/GrxfVO61sHoUoBjQZBlmQEbajoPDsuoBecHQcbDSinvVuDLJnfrEmg5w0Cc6VFhEua9PzM6\\\\\\\/G2v3zeGdq5Y1jZr3ZnbA8dQRWkH3VPEctKV5NsX7hhF86Dw8+PJi9I5VnU75Y8ANAYNEhhJ6dpBjwxuPaXpZQhnSJ7ik0UYr8yqeW2D931\\\\\\\/mZuB+V+4KUxADGyLVr7nV7A4uvLon7By5A7rZZ\\\\\\\/EmACpkSiLvvnp0+gRr1DxgAW9YvtF\\\\\\\/pVjUet4oW1\\\\\\\/g1KD2uksQMU9\\\\\\\/5XVeIJFNp2xMwCqgH16ybnGT4DmOTPTAhcfHmOWvx+yHseNoTeY5htqjsffLczYq\\\\\\\/yMxNS4hnfCND1iTxdwzgkSdIIfzHDpwdj0TPZO18Gp0xFN5WQxxIvC61b8+APzltoNZrVpNcPS2zghhufr53P\\\\\\\/iE1yyr0JEOJuDSTK8G1odZtRdGa7YJteOirs2CaHTDnspuFDXxGNrG39bwVoMQxsZK5fMRfArbNyRX7x8ak+X8waomVk9lbyITMTTA5rDBn8lPLGaEAzzIKUHPLZ1ZjYNvvWeVwFn7Pr7jizB2GFBezNpeCAYAhd1QDhtmJutk33TCqScFpTKpUW\\\\\\\/UmeuZDESCttqq5Egm51bXuHmDIqLONiNgUmBv\\\\\\\/q0duT+2s2M7mlIs6nHJ1BLbpk6DxVqw9o5ENDbLvek2J8r9RUnS4bBuhY+s4AGFn9py6VMZE98nlNqo+rmdJNdp9AWaKJBtOPUJi0zpt4JsZhXiO5u3aCesDYgiNlxsxBl+MepHEbLNOf2kPLKe3fbAR1RuEsEI\\\\\\\/BeJI9aBRUmTJSiPGMNvCdLB\\\\\\\/Le6ELNTecByDXTQfnnHuLQkJDoZq7mA1mIvTl+dFIcVz+M\\\\\\\/syKLUGFHxEGnsG0pqsVvRlJw5dW9vVHKfgHNjUsf41R4E98u7rlr8vzhi9DUkFzQIVqISPSux7qtnA6ei\\\\\\\/R2cqoTZTTUfynfEqzPJHYHraEbOA5J65jVbvDI5dqyjPcbr4TLdEgsteOzpJs1WzJyHCH+reUGGhMWXDqHWapMZzn1wvS1maMXuma6tR2iuVPSECiCmtN7Lh1PQ80TKFgxfvnWEORmJHjrm0PekWz9xQd1Pv4D3rnVSSoMzaYfkb\\\\\\\/dvXLjnBDHKylfX6+tPD7iuyJTVJUHv53s6NWBwMhsxahO\\\\\\\/nONArF7cqTpvZWXnWAzTByfHdp9WKFN9I9cZrtBZtXOdIoIjMi7AEPYhez41RBS\\\\\\\/v+omTAtPw+9T1Jl9Tds05j6ksszH8yaemXFOi5qDMKBeFQQSa\\\\\\\/yq7f6\\\\\\\/sS0OIdcoaD0GOtPLLbng6F7lFOIepSyD5sYem81gc3cMIWNYep\\\\\\\/GyI16WPiXsnp6sVnIwc8BvFy\\\\\\\/cyuwmPqYKXHlrxBDnxv2d0zPfri6uYAku5Au0lfp70tTrb4gycs1ToxRfXdurFIVTtTPrSFtRq0AexJI06mAdiUI3p6uTabucs\\\\\\\/PypxTrc+MHAtbMZWUxosRT9hcGk2BsCHc9ltJM+rAerBNxl53Qcpgoqxmfw9Q6vrn13MYUAs6QkghngXHN+fXGETf3hLYdi3lY0Pq2tDZyYsxsVISxUfre7oa8YG50M379DG0s356\\\\\\\/nsv1XoJ+NmIZ+AvnjKOcLoBeKkToTs8XIJjBEYbfOgoQxo2S826c\\\\\\\/2UhwxZEgmwrw2S0nlTjCdyVvEc8U+e\\\\\\\/qbEtpT6MSU1Fze27eaZwSvhy7Seah33GLys1nqOiD5oE1JFDSgJaaFFJpUw3AhQs3utUfhFd4AO+bg25nb8xd+ktmRzm9a\\\\\\\/ZhFZduDFOeR1ST4fjKQ7QyfpOnynaHRbRRFwoubYjTWDzmYmNISF1j4IP\\\\\\\/1y1viSM5rWEZsB8+zkPvf55JvkCavHwZJbDKteCO6jam3PDthAFsHbk+PVYKidbJog0\\\\\\\/4KU+SCsTyLQFllZBpiawfJjJIUWCQuCT6GhZqcmxxnn81wb7Y7ZQ2ZFf\\\\\\\/1gODz7oKqWrZFc4rx9\\\\\\\/LFT9NMraR6nMgCH6j3HPfOczA7dfz6SkrXWDwFu0Zn8j0urZaNhuT4nmeDBmZqN+6k52IC5q0yz06FurZ8+9R9l+friM\\\\\\\/PEzYxEtO42LSEz6f3CZJJB19bg65UsogfyalgD6mjTkLPV7XdITW25lwBIemgmB4sTO\\\\\\\/iLaX26FxqhLPovxUdIyfTssY2Ib\\\\\\\/\\\\\\\/5QbFFuZi4MLyekU27sCv6VhjZXK7QuFgUNwxn0tMjnaAaLKRyzBtTSz1csM6NgHeAyIMIfAPjYUiyIoZXeZbIlg+6yrMFmOweB75s997ej9JApuPsEx+eLLRx+PQD1oqxLCUqrw7BArbrNXAESzSM15Etp0kjSEfGg8gDSvpNt+Zf7wmzkgDFSCoaMeF6Hpe8m2\\\\\\\/QBZL0MFu+YPiMwwUbeEpn3Y6z1p19F+sLfREjCzOvJpaiRpkn\\\\\\\/sQKBQ8eEAIlg1wSyLMeBndC4mCMYTflyTYeLgEIhQZyXniJY+fJgRLi2aYKCMoJtcHBhMemWt+dXnj5UXz7nkY8riDhPIShYEWc8aLmVwfrrvF5ZT8Lafu58IJCtAiH42ANAguVJWA4Pb\\\\\\\/NYVSUPP4PlvLy15JpVnNg\\\\\\\/FrJJ8ElOfMmi7bx4QKMkxoYcdFQ3yMs0cAwNiT0m18qPTcHKLmhwaQ+29\\\\\\\/aINK5SyaTcEDbvgFwBbhEKCM7Poo655qktvDQf3KdV47dq8TfOQk7m5KD70aZrtN7utASfO3XFLH+mTxGZ\\\\\\\/ItAw9dze9Rfj6v92tMWn0S3K4pwm8UDr2ZxWxOAqWb0WjRNOWpSUnPA74Kn0R\\\\\\\/hLgumRHa2MJJAIztr0OYTSteyslTp0o0awZHz5LsC93TeSWPNCXZ2F8Gmq9NzAKwv\\\\\\\/lOvAtmYVPi1DMjFejDWeoeYw7GAlS1qr9vSEhE8DguMXCfWm\\\\\\\/1nNer9V0tLoFU1XFF6Fx9BjW5K+bMoUDg5cdRdHUCMu8s1DXd6308aj2U2zcf6DepRYNsLtBRlAofemoa337KbihDl\\\\\\\/ZLvIm8LBwoi5ZweKwGFN93NEg9zNW3O1D56RUFeslceMBc18\\\\\\\/lrbHYc0Oyq6Fxmq9Xdz4RwhaZSid69ka+LeLDz2f4vCdxlJKoyEDT8q8XtEeRIUhPMPnhxvJY\\\\\\\/O4ZkxIQ9\\\\\\\/KbdIQXhl7TMx1E5rP78dGKiHVwOauYo\\\\\\\/k0WQRZ0JAqAPaPfDPdG2JniOLf2Ij9d3HpQKzg4s2WNqnhs68ZYnvI1qkGuk6YEio0bLEO6D4saOg0J9xMQPaefB\\\\\\\/mHefsyI+3Gs6QWUH6+meEdmH0VTHvA50Zlot7Qw56rZnJomhUXWHmMlX2LHxYlw8nKQQDp+xUd0xDt4zByZ1c4xsqzAd5Jy7s2K2CxnL1yrC8zjwZPbo6JiK5UFomfFoovxkpC1JBEpoW11y+NJ7FWZ6OBiy6kKv+9MOsTP9H5QFjH4Q26k6ZORYdtbTMCVZICTeUnBPE+zlCZ6MLX0v72h46s7g1WjysNk88HmUALi7qxNd0YKfxg3trZBEOOd2E7C5GwbTLwqYKmYSO1\\\\\\\/JhokpDDDel0A9\\\\\\\/FKHGzKJdIbd0J6h\\\\\\\/w+pMVf+Th6vI7i\\\\\\\/g48gK0TSAOygSbrgiWkfGPyGIxu4uOkcr+bGLaRQtg\\\\\\\/8q32Xo\\\\\\\/vVAnGjJLA5zdMIRmLr6a+jFVi496NbVpyKBQyQBhvGwUXwAKzZRmx7+TJ38mVZPK9vel1tb7wQUZErk8arlEWh9hhMhVsXQplmEGdO8gUPHEy\\\\\\\/cjEWKQZV652K0rcmsBCS7Ljz315JUjplkva6SSrQLwg0KXa1xT6kdiDzbQ0iSQGl\\\\\\\/pn2pBgbGo6sSPBKOHkyAoQUOeFq0Zfp\\\\\\\/w4o2qvKyRVof9L016zRofN55GQu+KdA7y4z3bXsIfXJwYftMajng\\\\\\\/+C+4BI2FpyYZ9KijWDJ00ZyihmWnhsZU8\\\\\\\/akQ3TtsS0ibndaamJzTOvAJSBYi1rtpMa+R1AzL6REqdzJGsGVfJYjGrrAwI3NCZ03cjAOrDy5EuUs4KlfG7wSWUmtg5VdT7eVVmBQPNEbKLvW7MAV8LGdiK7m\\\\\\\/RteDGhQ1V7yUZtvc3SAmfHIrMpo0OKXIiZ7NB1j9ZoVks2dmq4P7WiaNCBs\\\\\\\/oDRauhyTCTZIqN8rQRk8JkYsF\\\\\\\/8GZWUoZzpSI9JmY3fEQLBPBLR\\\\\\\/mp2HpNknqYpzQKNTw8b9pNWrVrEcOYZEOcW8m0p3LAj\\\\\\\/cpHvyr0k9jvlxNQgk1GHyx9MPc38B1B4oEj8YVe5\\\\\\\/lRbH8NHqbfBo6UowqEWXqxJZi5GIvDeJUnBCcZrQGu+wOudGTMKsUXYkGbN+RPa\\\\\\\/u4nn9a6n5q7bU1oF4CTYXEu9oqNmfXgpLYclmpXaIckJVQlTCj\\\\\\\/8fD5U4FNoK7o3fohRoA1burnpDbVOL\\\\\\\/yEI6K+BHBdlRU9S4lp66+xKDIsXelmI1nOCPQmyMrWECRccdpsyFPUeMfthCRWETVxAylb2H1HkhM+zZvPpL9F+xVpYvk2E7k2tG1H4HAlxxMaedjAVFStQhgoXesr3pkAG6BZkNWIZwMkMqxhkgSRrtFtZekiQ0BzeiU621pPFHSQixF83YzK56T2q0TwJKVm+XAH3pbZm92RxQef58NWpgeV1KEeAC4q+X9shBuctCwjdqB9Uq5jwhw6qfqOmOsi7bjjFdoaGn4uJ4d7scOmQlnph9+2NWizJVu0N4znH67RjhY6y7JrfgzGSODirv9hNy+qkeERV1wWVLWZPoJP5CvELWlAcf3M0dfdShu4iNCFhR3vVXfQ2gAj0AVwHgoLHgQW8Gi28AsVXYJvxKcaQjDKtFNZ4uSht\\\\\\\/IXGjDGl92QQerW93ANt7FswICptqxYLD42\\\\\\\/WmCQ7lewx6WOkz727och02qUx2GmkMEgsDa8nuS6L9Dn8oilyNyLq4iIwQHxUg5zV65EIUI3P6+tgxwiwVmW1ETrp3Cw\\\\\\\/d327YP3LtDfkuIYIENqh\\\\\\\/mJ7UsHoQfrHI2Iey0DhmQddYfWZjloktJsdKjaEzZoODb0TWM6m1k\\\\\\\/fBY2i7rdnFFzkvsgYXkun7jctBxY3LXA4ZknZpNx7ZP65tIp99tvIfrq7EwChYQEU9+Kv8OurborSYL4Bqf39F0wHPocKaLFKOpHYs+9VSj0bq1sNB\\\\\\\/d7oET27jXK4L3x9kdQ1m2joooclzrne+7Oybg2jj1TepNjKTS2HOUaJRPsyOVabPT7\\\\\\\/RJxG4aQ0xEYXIanXuA\\\\\\\/bR096eQdLGcFshu1TKVvKdFxSUUAb\\\\\\\/bEBuqOVUJnsfwVKqMo0yglTPXkpkTlKQB2QuxWFDEdoIqcijAf1uwrKclFqpw18kHXCWU25fwZ7xwUZjaEp1UqYp88P84K6sR9xXFP9zdjxm91pv+T7pgZTnUnWWAesWOcXsmphn+aKiK+aTRNPAxpvvH0QG75UCApep7WqMcDpcyUE3h+Ds75D5UGKagnCB4bOoLXuZticGNFrHkVa5mDWUFuwOj64UPUHWiMKke50IBKrqQTMUwhW31cooIyy4s37so38thsVA85EwbG8ZkIH7KEV6FfMM67\\\\\\\/YCLVUx8+lqVNcJFPqnIoT2wm+vG2TNvd21Z+dyKwdWpZzGxvGwOmE363Fm6ji31M2fV5wpfw3ifQX0jCAK17NRwq6dCHBQqqGPGWyeQJKDNvMDJAx82KtKmugHJlHtxPxaZgFOYH0ySsavi8flOVQwzVE7uk28OtldKKdghNz3Un7hKm3x0QRiXlj8sVTXOvhNzLv6bYzDxMhHxEANDqMjM2b4ho0AmTPLUzAYpWZCXOJ55K\\\\\\\/L872o9kr+Ze3IKAfXuCpZ0wRVOKDUstJo\\\\\\\/AGGxMzrgPnlw2a2NIwyyFvOYE67ne31b8wnZVNw36hNNR2qz1las+QGwBTAOhmPTH5Z4x574e4vdqyI\\\\\\\/BH3TtgboQVJx3piVjhA19AT5xvN8WFIS1hqjqNQSMD5z7w\\\\\\\/nBnbmMLsGtUEO9wwNFwr4AhVxoL5GnZvuNf4lzK9Qt6jKjRvJNHyfp2baRgsvzuTx8yi0SrNQHdwp\\\\\\\/ktGSlcB7p49k5wb98K8QlhFDH\\\\\\\/XdXXg8S1buaWkPFhTOWKO6ciEghS6Ko8EOOi2V\\\\\\\/F9fNJGw3Afw5g93+Ano3NxWQFZVozuaFpaMGuCv\\\\\\\/jNUoz6GFccebIax90VM50PvvPT9ftcKCyQa+tGfSYXK6PyDfRsbaaYYQisb8ls9wB1xuGDrikoyEAcltvdOlli+kDDum8DyAh2zmVCb0VCpXx370hoozGxiFEBqK0tYpm97i4r+G1ib+T1gH2WeSX5n7c69xHhEz6HuemEGJxbgRyF0a5c9nSQ3QuYt1AezZOfv394TA0OsuuYs6PpR8gZi+SAuhKA6NgZK0OPRkOeGIlwyauEqXApWOxAYv\\\\\\\/hjf\\\\\\\/QhU5y0lmr0FHWZXCnDJTxeuoJjq+1MYHt9mwLjoSwDKNx7\\\\\\\/a4FxPl4O\\\\\\\/Z9hIytCvheCUXkplSa7dS9z4Bt09AJbLti7oBBZ6Izuo911LWRxihFIOHLIYvoCsZniZ1yEUF7LR+nFvBwWJtvpLDPwMHbekIzhtxqo39ooGJP4bEhV3FOb\\\\\\\/4wVWFC6VWvQMFtwwjRf8DAmAH9OlR\\\\\\\/wAljgj8CL4z6mvbFfoBj04Lf7+qD87FkuBevBDqBQCh\\\\\\\/TFmrSfqW99eKS5G4WJgsxkFKGEH80jktFJH8A+HVUuwxQitxfxS01k4afdjIZYtZ5M40qbpdmY9zwz4KxF6Oy4+hX5lbi+ujt2ETN+ofyElKAFuMIqSCRM4HL1fcv7pw+ngW5HAyqmEO4YI38uM9mQndgz6I3KdojndNs\\\\\\\/Mh5UwDcaMSkxP5C2yd5seITcS+dbWKJFO9oOu4JDOYnmttpd3JQFl\\\\\\\/T3ObQFlzvwQoMYrw8GckoY4RkEmRVnsj1KLD+VXgx0+n7dLjU91SJxXPm06JnfPcyVn1lpVacgyMQ9VJJaYbLD2SSGnJRWW9lmFq\\\\\\\/qqpElPr+yQhwFLRKZPPwCQxPl9E4t9n+vNNxlE57V5OcqLa16OUT4Y6FKzcKp+Zol1J842QtT3MMrmPw33X9yo0FkWTDL0UgOmerqE\\\\\\\/UQGETc0QJRikWuj8fq+AMGjPDwpnQPT8zoCNh\\\\\\\/53SiUk\\\\\\\/\\\\\\\/VUWvEyVDMSRB0CKnll5rTQybtvetnWEhsvKbOVcoUA4WxsleXjLckCt6Vhy3goKeahSb\\\\\\\/kYvkZy175FgR2dBAIYF4hfQ+2b5gC3IJpirx7eHz76+p3VrQ0kA6GQCRoxlj2YFs9PNKn+um8GLD4uZCzYq9+PYY6a1Der7HhKIAbs83iN5UolaEf4gjjzpZ\\\\\\\/dTcSMR8cwXyo5Ygedtb05zN1aA9qmGe+JSXDhPRvo8rYcHuPWR9cpKJVJD3++FE3azwxxOf9Fr50yMy+wCZQz06gNdiX3iU5zJD+vAdFkqve+oSgywKIJuW8Z6Tcr+H1GsYF+FDTwDTHIKtfLANCPQE\\\\\\\/W4jzRN8ZUwGXO+5zaOXniLC+TTr0uxQVdW1onxJzVpWt2\\\\\\\/o0jIE4hhkCE\\\\\\\/unUyY8VoCjLa\\\\\\\/hLbpH+HORDM3cXsJ78BlpEpYbGs7q4eMSg2dNq1HZa081HDvcKPLdnXEQhuvCpZhA3DLk\\\\\\\/dxbrvLvZsqacpTdrHGU0wd7uakB3QDBNjoxDxH+KFBFnXf5ILb\\\\\\\/QVrrKBsGBFurWKh2NPp8BIEeRcia9\\\\\\\/SEglN\\\\\\\/lSpYFNYW1syoPFlrqPC6ZJ4WUjmdERUpiUWvW9OEpskrJEVkjFOrxdBxf\\\\\\\/ZRFI6mK0ljD1eAcR\\\\\\\/tZa3Fz08LtDTIikMBZfYYPujDsdw\\\\\\\/V7AvnjxskuKdHqI1ZXIQTGsCV27\\\\\\\/9NSx28p4lr09EJAYRxb2JnOeLi4HjKyYEwP38tH1N4lvHMg1b0XKX3bDUwLhrG0up+VKm\\\\\\\/Uaynz0UJhdXBh6vMpGPZCvVmg4KxRAI8hAV8RjAtPlNwAFeJ76gXMUf9q7my4l6ewYTr5c+lw0jnOaXzZZSJjn8vubBvLjcYj01kWf\\\\\\\/yftzMlqS\\\\\\\/TB36M1W3DPKtWavYtsnKtFNe6pkO1y7ln4z43WXbqZtvcIKJOUkq7zki\\\\\\\/St7O7SmAJNdhCPAq0RE0QmEObyKL9XJDQXEUiHL9vRu+dDLxr\\\\\\\/\\\\\\\/EhbDLRT4NJGvDqBKZUlQanZoKGOCS40db176GE8hFVxfeLeeCZQMpyovoTLIhFHXDW8a0eq242jenfeDtx2Q65XIz7mFZLqAL1RVuLVDrP6DHk9d1rD1\\\\\\\/dC6+RtiTA+2Rg9+aZeT+c48e8rEDLL7M9JPAPtIFX6bS31UXNir8SsS90VkmPvygSRLZvD+fv5bugOWn0crJz9VhjZdTbM2atddU+Dd9pd0yWu5DNHrqbg2Bm48AM84Lrs0kpbWScUXJkT7Sds2V+uQ\\\\\\\/Vy30kxDw42PobBU+ptwqyFI+ByJrJcoPkdsR+oBegLWnJRMsqqoS4lUWbfy\\\\\\\/JDzthNv6Q+CMytMSuKbXZOrmriikBeth9uTf4z0x1XV4nLZsmNUDjeJpE45LQgXnixOvOfFgCh2SaD2BqvKT3ba6qETRph4DZ9lMsBjy+HDQkislKnp0\\\\\\\/fv0d4Qeg3+NUQNYZfe1gMH4F21ZULnIItTEhiNaB+uPuLJwx1uvcEDPcOaHW\\\\\\\/p7VbUZVflRKYxhN9JNn+VW\\\\\\\/moyjC65xlaRpU0at0YdttFPMhOkTQ+NaEHR0M5tw6AbOUDhTk0GHkWCqXt92HtZR9IxYBNCJPgBcUTNJJn3vflrorYWX1eSZVztihPtwlFQY+xRNlX92tVJDz57zHTaidSmIsTfMTe2kIt\\\\\\\/maxwfqiCwYFAKOZ8\\\\\\\/HvaXIYh6icphIyntkWM2z49sEph7m3GlVhIWIxr04\\\\\\\/98ldzuur0LCNcPc9C7p8SHqE8Z8JVBxxzbas9u9mlsWtZREVojXBpz6rl3Ya\\\\\\\/GjpLVXVH9YBJC8tKXxe+e+Le5sqQNdJGHkj4Zv1ZhL0z4QmyqUIRSHJ2sQURC3t8GpjvH0JAhCPkpjsa6dWUJ1v\\\\\\\/IloZiM89eBctmk6RxxtLPtYf5oBidKJPOFtD63wZ6FRcAm63qqTLU86dXsqx8PkN0aZ0wzS9XuTIC2Dcj0a2LRh1tT5GLhitlX8QhAs6X1oUoGS8nfkJq\\\\\\\/o3ZrblqNogx4XcxGrwjSmhVqsa9h0M\\\\\\\/8iwMEpGp4sHi3ZLSCqIM8BUILa8PLIu5\\\\\\\/wzAllif6aDkaYkN8FtAWSypROlcnUIaLDgyB5Fhu9A+I8nuAHWFthcDYlGJ2lXV\\\\\\\/J5WeFjFgS8Ss0ue171v1Jl7wvjSdN622Ns2CYEzsKFelxFU\\\\\\\/sAOL6cyMn+kQNBlbHUI4o2HRBElLjIAmImkJwZN1hM3NC1H\\\\\\\/OTTovs3LvB6Jaye8A8R3E14X5SxyoNz760of0Wjtr9t42\\\\\\\/qzGYW0v5ZKYg44Me9iiBPhsW2JywrzsS3zCiiYkPxmKzUUHwMzuKPUeDrJpu9CWhnP20qvHo2EnPvqJA0w6fB2ncmcuRJdNGlV0GBZA\\\\\\\/wZPGqQ\\\\\\\/B3PPMY4XbBX7hs8gDMAShWq0zIW4fUtag9LPPX5ln3R4oVtTWmZdaIg9rP23onn\\\\\\\/luXWDc\\\\\\\/oYsa58NoSoxyW\\\\\\\/b\\\\\\\/XnNTJhmdGlg+Q5946v9\\\\\\\/Zi3IefMV9Ajg431JoN5oEOxs3I20jYFnfLKmYoJhyHfvJRSd3MR3Wu9AgClAajOUDtKnsjfMo8430rZl1dgg\\\\\\\/ZpWtLHP\\\\\\\/RVmd9vhLjX3gCNhw\\\\\\\/qukpFICvgCP6vKt0+Vc9UjiTJ8Xt0ty2GuvkyuJwYZ\\\\\\\/0pu3rHbinkWn3GwQ4lgCxbpr0zzMzICoNpuCB4ai2VFOLc8AeafKd3pJHrJjpVoBpJD\\\\\\\/hNDteFc2XquNPsYliTsUAxDXpnykbID4UZ4tUSE7eUbbKmvPt6KQSI8E6sNvwnZsIxznqLKVYZYylrFdzzb0qYmd+ltua5x0ASsCxXHs9Pd8bfQOY\\\\\\\/WQGRZbkrbEhFNvtKscDxVsvNwLgyBbKq5HMlBz17V1UJeWIJn79Xl0NdrA01cJnYMaXDZhnT+3XzRj0HLeaLWu7BOMrC\\\\\\\/xxmxp8L1+em+UymqBzJXFDuDBFkMovJY4Psg1YtqRzWgpkBsMMQjytYv0KFKobTlFmeIL3xNct4f\\\\\\\/gIXPWf+QcvpHR2OPUrQ6pUZj22mhb+9AaJPkkKRaleSAyD4OMFemEqtkC6quf1bNM5j4pTkdZzfIEMz4\\\\\\\/+k+suagZpMwwNoLtoZymnQ1TSui4VBIrB4JkrJFrSBtGNynG0MG5FMspoWRd8sdY4iU7x5hhE4pydIFMtEItAWCIGEv3mdlGIb51K3BwiHZ+vhZzKBllkDR2VxSSPkRWhsHNY8XFaRrUB8QDGqUzSQjovKRCwEfcIynrqSC3\\\\\\\/bl1+46Zm4St+X+4GL0VsOtN5kZ+IeJUl5yRv6CPzUo6OCuTNAixzD50OjOlcLelClnZTRG3x9K0TnsKk15kqyGcrcz5YikV1FGOS58oCubiZ5rF8lP4Zn\\\\\\\/fA6jQXlEhF2UZ0aUG5XybLMIjHdFZeRpV+avvAgXU0mkPlJfX0PbGon6jrUjPJwf3bf6DvDIdvcFZEVlMfGyPth2NpKnY1QqLpKz3b9dfs0vGkyXkTfere0Ak4RMjr3WlNGrB43v3VHn+jQIix3JRa0EJiuHUG6P7cMJQ39aq5+6s5y+cWgZHRd4MAesCRFX0ep95fYgSHGUT8ktrw4Kd3P6ur1h40Wpu2Kkz1eIqnKVZ0Z89qunsc+QeTW3xSw2JylHKIdFkUQu1zYwxtnSkrqi4lE6B4gS4kZAwM2kRyhHj2OZNYORrqmPIr9VStfi\\\\\\\/AlcDrP9o5hqnwlGcUB4un6nDzBVSAGoMaGPzksyeOVqrVL1wXztktiz7fSDvdfBWE4cCXMRc6bVzBQzrsaJEPzgO6r0vSJY8n6iBSMagT4RCxGvu0OdwVvaQ\\\\\\\/4H26kM8Oc6\\\\\\\/eG\\\\\\\/ViwB3U3v8nGAIHEyaURKspDpHSe7HqJKLcnqJWacrRO1U9CraK0oOw8fR2WsmGcn+4CPBBUJdkVLaEe6Nvfqb9Pg\\\\\\\/a4\\\\\\\/JPD941Krix\\\\\\\/No4QIPDaDPiZoG6FzBfhTTue6XmOoBsS2Zyulzghm5nhVZjdnyRmTpVBahKr3XBSdgUF6ZgCyaaWYhxOyoScG\\\\\\\/MuhJYLgRydXCtzZHkzaUScU9gaRXejpTvDoWnTkHmjnfeEWrrM5E0M7R6S9LEYFx0xHYJ5x2YsvSfE6b8z7\\\\\\\/3k\\\\\\\/CUBkYliI3Z0LNhQXOa1vVi2QWt0gMN8EEXGq438uVKShsTiKpUqQXvx6KM6AJ0dqXnXrTB6G5EDAgl1lh9PWkriqAUY3es+YYOWudUk865gPUpP528GyVDIQyItwc4Csu\\\\\\\/XST18Uke33oLxczEhyr141wT68Rrpp3zuzduSITR8EYQZdasVt3+oy2VD9NLBLlr7mxu3IWMVKhg4BX6mleI3wLfCmYuhNYnHpj8NPipy8VybcKUUCxiP1u07fBUqgMLYku1GxgaC1R8n9TEFexYd4UDaXKvKLL7rr6Owpns2Tlv1yr+qctRebwagME3eA6WXNbnoNwsrG7ldqeKNESycqHn\\\\\\\/WDS\\\\\\\/+TirvgZ0piloa6dGMhGV+FynQOPPP8zfJDrO1b2UxyX1cdWoRg9lpA2Hz+zzF4TsexFNNpydG7ymBjrKVEg6SUTVvX9plqxD7MKqAJFVwlHyclPEVvuo8dd+y0KGqC7xpwnImmAxGBl3DBaJ8sBiwcWKUWGh7yvH\\\\\\\/bWoUpIhUpPsSz6FYIKiIVly96cFlhcr2o9jfJgesRKcKsQ+8WPov5pHHJ096+VKO4bqNLZoZarmJSHJKOuSmTK8CW+CDivjF+ldWDdo3d\\\\\\\/7tgW7CHiM0mGJR9Bwl8FAJrGY75HRkdwisv2xM\\\\\\\/bpgKTcGO\\\\\\\/TDqJnxs6fZP6HsFU0mrnVWq2quGEChPACokKK+zI+mLYAYJYLjQSVstJZc4zXgdMT5JTAs3UXhMq6MHFUB0+6X16kWG3MULEjMU8+2om\\\\\\\/Qod52bRf2q267d6jK5avp7MqwocilzP0IdyPz8QbVc1\\\\\\\/i\\\\\\\/rwnzPo6HO10Xopt0tcUzOvYEm7w7wqacUOTtTP7\\\\\\\/MW0lGnQKDOX1ppJwkPNXzknrMlBZovb1O5OkxrL54UGqpjIh5dV+JPDnekXuzFw8XCpSwnJRhGL+c+GYvbVJqe1SycJEh+Rz6B6ANXLYVgrgkFcCWJ2+PkX61njMUYqDofXtp7NmiMohlMQDr3Qyn2X2PDvQZQQ6E0U71t82akJ\\\\\\\/+cbHEJGC+AiTbQ89EzJoyeWjYCrXAHzuoobpGYfjos\\\\\\\/dVh9Pq0s7dcHjx4Beq9Onnp5C629j+UlEkPddne+OeKetmx9B2fc0C2Te3Q5a0m5ivoBhrTnu8XXVj8V7sGwJiwB6HRsWl3iNrXNeLo4c8XLaLE7dM76qtnzy5GqHyhJdNYYcsdb\\\\\\\/kdE\\\\\\\/RDGRQiUhfk4xMp4TfN6eamr+mLS+ySNGItIFTs9kznYokJaZbYohReyT1JqeAEI4LNgWICifXBVh1Y16VRs4GNJbH5m31+jSl2snP3nopTuyQ32WhVYVSfMv6wkEIs2DLD45CXy5mlO3Fsx2t\\\\\\\/gw29dk1YcHg6oNyVb7\\\\\\\/jYKtCTaAaJV5SdC+EjhtDX\\\\\\\/\\\\\\\/mBGeJsN6DvatBlqtFUXGWqJejJYUKHFVq6\\\\\\\/5qKYwVyJQ1CNYfsBXcxdiqRQ2\\\\\\\/afKtVXGARB9TdParn4Bvx7za\\\\\\\/xs3dfQPdSGCcXhMntNS0MHnqoWhQeG83pXMkkIlXTJSu\\\\\\\/c5XA7PfLmn7hKv7moiXj8SA7BktGSaIyowYXWub8e5iD+Xf8OZu9nnM1SwY0TEIXjWq37ARPdSDWy+KVdw9C2\\\\\\\/NEV8ZDYnwnWTPzGO8X\\\\\\\/G5ZgITZOrU+ofaB\\\\\\\/uMufBpHYC8idX1NFRL2Jc0K8TBlLHy1uvxcJHAZwHILlAxv5G9XC0q0rzOtJih3LZQ7F6fO15zrn\\\\\\\/4iGdPvftlh\\\\\\\/XebWfYlwxMqZRuBLxb8NLCU5GVNgasdexWnsC9Zya2soER7Ahz7u9SvTe9ZGEOlp+nr0H+LRMSTTYHhPeC\\\\\\\/aKO4nFQsVWvKWR06Vpl6InFsaLcg1KI+8NLcwEgutTppBXRPgtbeCMvGAhGu9z1t2\\\\\\\/6IS7J+RtTRknHMStN1iIN9kI0D+Z7vDHSPK3mwAakxAYNtdA6q4QcvMkNrd6IDdL7tk2y\\\\\\\/Q58WM59CGqvgEtRPBbIahTsYo2K\\\\\\\/g\\\\\\\/WDRReV2u1g6IEKUXF03OnHCX64U9D212z0qh314UFpqmUp2IhoWPY2b8yqk5T3KL2qh2cvI5XT3TkRk1+7eYtqbg04N0IALYr+FeefcwmUDSpAwQoaZoqFNit7bRpFT0W0CkYwN8hJXbKkQYpWvyp+Ef32bRvF4mThJfbYvKiquIb9NUws13rhi7k5YEDmV4Byix1KsHX7TI6njTukgsG36GKa4CZRDdfrwrtDDpjJGzd3jKHNbopmP0+OEReaBH+xW91lYFIZnmZWGJpoafiBSKOCvG\\\\\\\/n3SWz6liTB2SPmQKLebvBD51KbOrfR828xIE9LObZqU7LVr9\\\\\\\/5qxZRW9NatxjtGCovMbPoal5kBKdrX+9TlSvyyZIbEIpFxY4rtOltbksriJyTGNl28f91ezek12LMkyvWPr40uLAbhtoEcMm9vv3t3huHRvrTRYg82fjfp\\\\\\\/XmProPulNVSspHCTvbSoHuf8hJL0dt3QpfMx8xz1BmxrHu45e4X3n+1lQh7\\\\\\\/Kr4Abegf\\\\\\\/Mpjm4Fl1tGY6qUtL9qpFvmtob2R4byv9B\\\\\\\/PMbWjb\\\\\\\/pvC3pKA7vtNqHnHKa+sjJWQZNfTsLPomlXstYU0cUjSiGypTbclZd\\\\\\\/nolAIHVB1B5LkrpmzWV5PECmawDjDShSNsYaTe70FNh5T2rv6xg0A4f\\\\\\\/TpMDblao71lOx34h0SpRrXzrsSe6z6y31hcauYkrV+udShWUzCpEsky5A3D6ogoCXVqUoB5crytFtZYR7i9B\\\\\\\/ga8bHgBU0AnsenINUcMF3OmmRa4AvAedlkwNFihxNqbcDU2iqTlHbwoWsyURdqNm7iOapwBQezo81f1kuCRhuqQ0snCbL6IYJO3+MiifcvLe+c1z1PwX4NoN1mAUEEVeu7FMumQ79r+mPjV3wdMBXj693Uj4e51LVfKOCRSeuJXbRYxd0Dp0ICrRXTPHMOTTD61YVhc1jDft08nx2lxadATcZooHv\\\\\\\/F64IOf+8wBouOe4\\\\\\\/qicR6dyOxtriP9Ik9o21kDlDD925ufsKlX1GnA85gibpXQ5y9NwkFToWnolKsxmbPNa8PuOoPbZIHSP83k5wdVFN+3hkvuPYhDuGFfmxt\\\\\\\/idQD5w5rk2jPV6+nBxVzRKfN4+ablGk9D\\\\\\\/wYnb8s4vv+U1zQuErRZmpcUC3YaraDKqnAxIC\\\\\\\/3RbiRhyIg2PLQPYkAmai921i7zppbpgCnOKGKZSKk0KoX93WHcGMzYlk2SyZywmRz+tFMsc+ukd\\\\\\\/aQ3aV\\\\\\\/kVhY\\\\\\\/2dS53UBCe\\\\\\\/GCaFCidikaM5P+BpaOs9xQlynh+DXT6+wmln+s8ifDgXGLyd90772Za8RQpS717TshM3f8f3sMIRG2X\\\\\\\/+X3KI0es2mldTp6jbKUoMrIU9U72w2QINEIglrNcWFSGdpl5reQLbJUu+QbMoDi2dKpzsMh4yyY\\\\\\\/mEWI6Eh2NHfiJ8FiKLbTkBCeZ9lM1LyVqCBDerZx9JInG5QgVslN2Ff0BpTwWRj1q5dEa2WJkaXY27Ozi6Y\\\\\\\/Ygc0GEtMKxgHxUrZx5\\\\\\\/k4\\\\\\\/JEGf4pWgyhbeiJ45P8NWb7tuRIKaNFGVjNbI1gS+SD8BIUg4qmsy\\\\\\\/cMzhW5U\\\\\\\/tp776rnAhy8ovvjmH\\\\\\\/cPidguszVOr2Mqma1S1FSENph9PZ2zw2HtuR5OGWIXBAOB0Ud2LdTBhJQceiZycLj3dzBVW5s5QhCQyuRj9G6Ps0Nz\\\\\\\/1Eq9tuvTM36ZCbwMWNOQuFlDTNu\\\\\\\/rE\\\\\\\/qVil6Fo8kBKzJPZC7utuaSJby01SnFKeZ7ov1SWq48qaAIe7tm5m7ZYg7hcWT30uk96IwjIBYy5wX9oo7tAqaUFvoDWrpb6093UbgPfkflqFs5z9UyNcVE8aUuwGGZCqnoj9JjGWH\\\\\\\/U0I47lYAG9pCEbfbDfVb6U8RSUR9dsclegW+fQuqDqMl9zouJ3g6O+H63Q5xpvmfQM6YhoHzqXx7zFLRIQYYErmsqLpchON+X\\\\\\\/R73HHS\\\\\\\/XyYBztoR7ASwtiAOHpdEr59cyZa90YL+XcJrK2qgIkcbQAAf8CKoi9dzk+8ScyRV7ckOfz6epcYyX+RexeRKICQA4T\\\\\\\/sSybdWfhb0MnBls88LcegkCs4hs84cIxZgssAj1Q3+wzZWhBTGHfs1pzHMYxZTkFhe\\\\\\\/e4gs36A57b+LHzBIt+HCCms\\\\\\\/PEd477+TO+G7Gi7fe0eo2wQdvP\\\\\\\/P+G+uwEtBqOod79oZwlnhVGM8nPP8FVyW4iFp17AmyGScAcVIKpvWd+NY8efUorcGFU9ynp0YoB6WWZPnNxpTgB7ileVC0OeHsFq1EKtEgdgShtABjMp3t\\\\\\\/gNMVifnxroOwpuS1Yy0311wD6VTA8TudSs6XjkEwvePdLcyVRJl4FEaLZqLs4adK+YZfr9ZOXKUG2VnMk1lbeRMH8FJg1a8ss0eNerBvgpE7lkTu9CEjUlcdmGsk\\\\\\\/DteJWdN+pXhn1jjLK7VmbC3xseVTtUofPi4eOohK54kPfiH4sDDrnAa1mNIDgK5F3WUALpLfyMphohLHnNq2IduGx6qVpMTU2SHcWowMz+Y4tyakFNYB+5IFe+JUD4TnjxR+H+laamYC37oDbY4\\\\\\\/v7D2kAItM2OgFLSeG8PlKg7jhWSw47jRlAoDLUQtpmx7OI3HFhllOtyUA72zP9kzZKs4eKGYEn0tYQmSYuzldktTPm38+I+GRCs\\\\\\\/DiXRazgyL5yV0o45hC9Vx72CktDjwthlIhf6pO6ReL84AygQ\\\\\\\/lZbDucCWty9zO8wYhLt4Zkd8jUt2DUsUpZavbDxTifd3k9oNK6RpFvmgXvdxiMJvVGM5KNlA9TFanFx7q\\\\\\\/XPFrT5lGe\\\\\\\/E3\\\\\\\/0rIiNsbpsMnhm22eb2Rozj7oHIZ2K8z8+YARjtY6qbCWIG2WxE9mQ1rQ\\\\\\\/CSDbo78t7ipccW6316OaXQmk06QrwqsIx7Za\\\\\\\/7rpuNpK6LvhgTtZdkB1aWE7h3jrug7RgwXkQeaVHnMdlfz2TQv8Um9GU1uetaYhdXq\\\\\\\/JI5NHyYmxKEbd7Cx9S9UZv3P+biEMYYN0gM2Y5R+bRXz+TeEu50iyhIRASSdN7SLL46V72eavbVzPFcck0Qz3UI28JwdggBYcCo+wzroEYbS4ypu3rMr\\\\\\\/BRQNNr\\\\\\\/Fds6958BKcSn2ibPSGjqsyb+kGQI6CF63UH8VNLXrWzfLsOF2JkE08ko8EwGIgmrw\\\\\\\/dtlO99jESr82UKCUMOzEzi3yMd5l3l4aIFhMEDd0VE+0ZU7XP5CciEbh3QukYXkAi\\\\\\\/V52ZiW8nojD+OHeTdqQVZIuWvvmN66fw9dcW92J9Kzd7Bhab8dZOimDgneEMMDtS5Pr7uV6erJQ2Adg+HpqP7+mMhnotTzhkfED7\\\\\\\/j4sbmF+2LRwSEvpAiw5f7IpSZuR9SlewDp1MKAMWvpd9FCXDTUkHSD0+4mRXHtzkXM+f3pi+zu10SqZndMYckweG2uAlfNPUgjEMqoN4U8hDmnxCTNF9ZQ6AsOgeQ5873GHcSHzRIpXntjmKEZ7HQpTkU3tnz2l7LW4A9B7CNe8M3EQtn\\\\\\\/AkE5cRhuuiiijkuASNQmf1bAxyNmEsId7SszCzxANtfh0XPbT\\\\\\\/2Q0JPcChDZqzca9jY5bmoTbmBIpHk4ufgkkNYl3KfmebiJ8+u3EWoV8SNfAJlSxpyX9FT3lArAuC5m\\\\\\\/ChdZyZJ2\\\\\\\/EDw+wn4Tl5a3qk3llBfWz2QoszBaW43CpvucdQzbqIWuCgKzzVEl5Pl6A+KXkp7WYIsPRAtE1kLZ5q5Znu4C0GFNhFKTHnuUa5vc\\\\\\\/PRhHK5kAZZbIH66p7GCN0YIKzHkkWsb6wHPDRn4Lvlee6PaBOdVpreHrSyz1NQKUMZPjRHgpUnQfYaIVfLyUSOYkD3GJONMhxLN18FbS8SDO0JWZs7y2X1IIHqq9V+Wz5ff47j1Zp3jL3GXFDkXMortVMkUUXYgqtJtw5KBGDE2rKsbtfgpkMHi4brO+eSfViy\\\\\\\/NRkJHLjSnlD2s5ATHdN93NHPMkvk\\\\\\\/8BOCre7Y0G6cFIBdkKah5oBZalV7AHYk4t5hDax3hOSExNHCkhp+48Z3o0q3EUE9le1joAEJdgqwVbe+st+y+Je9tsMq+aNTnh7cvlQsHRRyUyV0abhQm2MQWgIF7f+VstP6PrGXo159Byv5f5c+n\\\\\\\/FkLoXF5pFk79yoXTf02y0RRIg2XRbEg6qNckbjcGWNXJxIlczMEpYyhE6\\\\\\\/XVGK7v6zP8va\\\\\\\/gmjbkK\\\\\\\/WdU1UdwpTYX1zmwAyGSYkD5fmecXScG02GATubbTPNO3xPbEKpLvW7JpevWfeVFlvLqlZsZ2YTvgPClai4Pcx4LyyBtkrzwQJ+hHNsvST5aP+FcqS2hTr76dZ2tCrfcWFQwKHFM\\\\\\\/uMl5vivEJk4Zyc99yMCqwyA+AHkZMlWPSNZo8oECHuxFtHM9KcszDXijj7dql6C8dbbu6csDkbDUZe7J+CavNnGt9XaZCmT8PQPY+Gwn9mSlhHmiXg2v7Yc2TnsptZ5dPaP0iwNncAd1VdzagqBbg3GeEGeo1gvQDbrmdo8saJPo\\\\\\\/8THH6YLoNAj3mfTpsX0Sxug7v78H+lzQJfJ\\\\\\\/HUKJXG+hT+iEdE7axRbhogb+0oF+tQCEW73aGmB9fR5x\\\\\\\/VkIZBNPkP7SGu39XUNQOhArD4lTi8x7GhMSnSXKW2EkUTMJsbaNabv6YI6F98ToXJYcoO79ODHikCe24DosYqYRsqQplJ6EQLbTQVj0ujuvtbdRv0tctwOm6bD2ZeYQyOJG2cjTcXoWHyd2Kia+Vl3oQEgMe+0w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022PFem9oKBFf++53QQ9JyVzA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022515c912d132ab21594d990802648957a104e1eae120c779973d8dcc1917ad061\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022g8U3xUXAZygkX5QlPTInxrsbhvNK+5I52NhMfpatv2t5nZI+74QnZFq4R\\\\\\\/zGqxv4tWBIUn2byOpijWDUXwRcv58azw8b1dHkT82DXiNsMjlOPQXmN1griRvy54H9WL\\\\\\\/32xlQLxXSgMVlt9zxAGVPD4SMlSNn4WoQNqLthHT2keb2n3yJYe1IQZx\\\\\\\/cIyMApXiLGne\\\\\\\/WJ4VlMhwodYpkCdUzD\\\\\\\/SkWHNc0PgQWPN5nr5wkaCoDTr4ahQ8\\\\\\\/DbWwL\\\\\\\/A2shmQJtPpoaE6ggWlPIHhY7pQyb15hwQJahI1Cu+g4Op6+H7DEtd8Re5KT1PU4UPXfT7getW\\\\\\\/IpUW\\\\\\\/SKAJOa2qBwPr1bX0oJo4OgPrnIjeV5bBDhlgUH2iZFYH9C6etD1YJig1pALwRcibDpS0HCWKuLmiu3oSy4SpgAcXd7CTzG8uWTwSUY+K\\\\\\\/p+2srmIDUCvp6s2Af9cpaatc7F3Ay3Ogyk\\\\\\\/Xw4Lp0XIMJdNo1xudts1uH5F9GWd5K2SCMkbRh7n04CTCAs0IuGwyaCDfRw9xLAZoXyFSHTCOsuW7PDVtUHr5oORdqquGIghxMH7DBjr\\\\\\\/6fRtL5ZQSGOQRrPMCFddHh4TthhS32CsPIVpeII5dCh+TFLueqvRY3Df56U459\\\\\\\/rGDAg47OA1lZmzfqhWiA01q\\\\\\\/duo6jW+bkiNV+HQLh2iNAOjDke6wd+S2E2GeDYIoKjCA0+x\\\\\\\/RcSOu6bdvA\\\\\\\/y5aUccx3JPZI\\\\\\\/01WeJRtNjPFSD5MK23LatSEAj22cnVebZoEWdnyXw9iZE\\\\\\\/knk7M52VhG0zo5i74ymzpbTH2uh6FrflMbEzG3ZYq2JhUzrvR\\\\\\\/+R4tzrrtnF2X1lGfNIjyGy5\\\\\\\/Fp5j718bUm8O2+bRxhfpFNq22c2OuSA+FFh9viQumVRenFgLRUqXr6NEdamompsU1j0DB1+yzME1uob3AEpuK3SHj5odnGqvhm8erqqWhLcNjqnn9E9FkzveGEHPWTRoVMV66qJA4qXCmSVelhP+93MSf2bMJ205WyskbIR9fAzez0Z161d\\\\\\\/ZB\\\\\\\/lNTAhGComakf\\\\\\\/Lf2Z39JvxlAJZqIxwdUnMTDXfEtO9hApPgxJtVJV1JDtHSqrbOq9x+3w3b5twADyxUp7bL3LlETeuUzOT5NcVY946H6RmLfIt400igeAV4ZXw8iHc42gMc8FZknJy\\\\\\\/ZmPwoINl24m7o0xJ3HVYchBXGH9+Kbq2bNb7ckjgHmV+zmaEcdNE1gDn9xfeg64WsB1rr1PkaR0phTp79NbU2EIqJMkbyA8ZmkVZSGjtAT1nL5fL\\\\\\\/5uk7ZmXk5RSLhoc3NAxL1H\\\\\\\/yt1eIHUMTETuMT6QrJ9kJS8k+TCDVWgZ+1e6aOpDezWzK1If3OI+Q5Lqnp+vhiRP9GTwmHTzTSMPBMH9OdkREfiXARaYANfm2gar72ndv3lCvZFt0R2PHl+SiuASisGHh5arJQnN+Cmd24pUPcFrJlelSW7\\\\\\\/xTMfLZETAaR6xAC7wbd7AZp3+8LKX3IEBeWoAahDgNR5+6XBzQ1dWabQSXtPlaec9URNMpXvtNvoYF0OPDqCRWlLokQeljZ+4O2a6XNNjzzwyuRC+ejwOZIyMgNshMlIznhWg7urjkSIga20+tifvfP1laTzpc30Bfv1R9G8ftBunmNVDCe247z426HLh4NUP7YvYSeNn\\\\\\\/XZsoO1r8j7oFfm51FkJ4IuLg+dFUYlOsYXPJEXNEY+d1+bnPHi5aGR2oos8cTCJFkVk8e4ciLuYJwkRCkPuXF1ZCkSZ2hBAF2+wOhkn3bunHjzsIDhh4lHnSDG9vMzuoYC\\\\\\\/nyQ8fAIG+u8PpdqJVEK5YRJz8X2hfIFZhFoCHi4u5sty2p8VE2pgpEjJyc13eXd7RiXAxJcuZ\\\\\\\/wecD0PdTxW8BXnpTo77\\\\\\\/zHc5tAk1ulYQuggRZFM5g55Ka9VZilUs4UBARl\\\\\\\/\\\\\\\/WmfohQdQ+cFvyxB+ECNR1rFzUTs\\\\\\\/+bDGOriWHfzNFFXoIF9ChX1CYxcYtMNCD8wWhvaXw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022l1fzEahmq6T27AlRiaT\\\\\\\/7g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227fb9501ec1c5d376775c49a2353d628a6d9c998f905bf3770c27d62266294067\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459183 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigation">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/5/327" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-V9QoIgZOtlZI4Tics22gHR8qn62v5BQsZLm2K643mCs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/5/327" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wqUZcgvQHlHAOrRF5G8uGo3gWj-yovII815q0LkfHKU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/4/317" title="A Synthetic Human Kinase Can Control Cell Cycle Progression in Budding Yeast" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/5/337" title="QTL Controlling Masculinization of Ear Tips in a Maize (Zea mays L.) Intraspecific Cross" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459183" id="node459183" data-pisa="ggg;1/5/327" data-pisa-master="ggg;g3.111.000554" data-apath="/ggg/1/5/327.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Replication Stress-Induced Chromosome Breakage Is Correlated with Replication Fork Progression and Is Preceded by Single-Stranded DNA Formation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Wenyi Feng</span>, <span class="highwire-citation-author" data-delta="1">Sara C. Di Rienzi</span>, <span class="highwire-citation-author" data-delta="2">M. K. Raghuraman</span> and <span class="highwire-citation-author" data-delta="3">Bonita J. Brewer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">327-335;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000554 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459183" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Wenyi Feng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Molecular Biology, Upstate Medical University, Syracuse, New York 13210</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenyi%2BFeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Feng%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWenyi%2BFeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fengw@upstate.edu" class="" data-icon-position="" data-hide-link-title="0">fengw@upstate.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sara C. Di Rienzi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BC.%2BDi%2BRienzi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Di%20Rienzi%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BC.%2BDi%2BRienzi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">M. K. Raghuraman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BK.%2BRaghuraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghuraman%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BK.%2BRaghuraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bonita J. Brewer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bonita%2BJ.%2BBrewer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brewer%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABonita%2BJ.%2BBrewer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/1/5/327" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459183" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459183/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/5/327.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459183" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459183/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/5/327.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459183" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459183/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/1/5/327.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459183" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459183/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/1/5/327.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459183" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459183/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Chromosome breakage as a result of replication stress has been hypothesized to be the direct consequence of defective replication fork progression, or “collapsed” replication forks. However, direct and genome-wide evidence that collapsed replication forks give rise to chromosome breakage is still lacking. Previously we showed that a yeast replication checkpoint mutant <em>mec1-1</em>, after transient exposure to replication impediment imposed by hydroxyurea (HU), failed to complete DNA replication, accumulated single-stranded DNA (ssDNA) at the replication forks, and fragmented its chromosomes. In this study, by following replication fork progression genome-wide via ssDNA detection and by direct mapping of chromosome breakage after HU exposure, we have tested the hypothesis that the chromosome breakage in <em>mec1</em> cells occurs at collapsed replication forks. We demonstrate that sites of chromosome breakage indeed correlate with replication fork locations. Moreover, ssDNA can be detected prior to chromosome breakage, suggesting that ssDNA accumulation is the common precursor to double strand breaks at collapsed replication forks.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/chromosome-fragile-sites" class="hw-term hw-article-keyword hw-article-keyword-chromosome-fragile-sites" rel="nofollow">chromosome fragile sites</a></li><li class="kwd"><a href="/keyword/double-strand-breaks" class="hw-term hw-article-keyword hw-article-keyword-double-strand-breaks" rel="nofollow">double strand breaks</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/mec1" class="hw-term hw-article-keyword hw-article-keyword-mec1" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">mec1</em></a></li><li class="kwd"><a href="/keyword/replication-checkpoint" class="hw-term hw-article-keyword hw-article-keyword-replication-checkpoint" rel="nofollow">replication checkpoint</a></li><li class="kwd"><a href="/keyword/single-stranded-dna" class="hw-term hw-article-keyword hw-article-keyword-single-stranded-dna" rel="nofollow">single-stranded DNA</a></li></ul><p id="p-4">Chromosome fragile sites (CFS) were first identified in humans as specific regions of constrictions, gaps, or breaks on metaphase chromosomes after cells were exposed to chemicals that inhibit DNA replication (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Sutherland 1979</a>). As CFSs are hot spots for genomic rearrangement, their identification is important for understanding the mechanisms of genomic instability induced by replication stress. Although the exact nature of molecular events causing chromosome fragility is still elusive, increasing evidence suggests that delayed or defective replication fork progression through fragile sites may be one of the underlying causes of chromosome fragility. Studies using the model organism <em>Saccharomyces cerevisiae</em> under a variety of conditions have also described chromosome fragility that stems either from intrinsic properties of DNA templates or from reagents that interfere with DNA replication and/or compromise the checkpoint control mechanism (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Admire <em>et al.</em> 2006</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Casper <em>et al.</em> 2009</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Casper <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cha and Kleckner 2002</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Lemoine <em>et al.</em> 2005</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Raveendranathan <em>et al.</em> 2006</a>). The unifying hypothesis for the observed chromosome fragility is that replication stress causes the destabilization or “collapse” of the replication forks at specific regions of the chromosomes where fragility occurs. The corollary to this hypothesis is that breakage at CFSs is correlated with altered replication fork progression under the stress conditions. However, direct evidence for a correlation between chromosome breakage and replication fork progression is still absent.</p><p id="p-5">Previously, we reported that when checkpoint-deficient <em>mec1-1</em> cells are exposed to HU at the beginning of S phase, ssDNA accumulates at the replication forks that arise from origin activation (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Feng <em>et al.</em> 2009</a>). Moreover, after the removal of HU, replication forks are incapable of resuming DNA synthesis, and cells suffer extensive chromosome breakage. We hypothesized that chromosome breakage occurs, at least in part, as a consequence of chromosomes being under persistent tension exerted by the mitotic spindle. Consistent with this hypothesis was our observation that breakage was evident near a centromere located on a chromosome capable of bi-orientation on the spindle (<em>CEN2</em>) but not near a centromere on a chromosome unable to achieve bi-orientation (<em>CEN4</em>) (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Feng <em>et al.</em> 2009</a>). We also reasoned that it is unlikely that the chromosome breakage occurred as a direct consequence of the mechanical force exerted by the spindle; rather, chromosome breakage occurred as a result of a more open and vulnerable chromatin environment due to spindle extension. Thus, we believed that chromosome breakage might not be restricted to the centromere-proximal regions.</p><p id="p-6">In this study, we set out to test the hypothesis that CFSs map to sites of replication fork “collapse” by simultaneously mapping replication fork progression and sites of chromosome fragility. We monitored replication fork progression indirectly by detecting ssDNA production at replication forks and identified chromosome breakage sites by direct mapping of double-stranded breaks (DSB) using a new method. As predicted, we saw a strong correlation between fork locations and sites of chromosome breakage. Furthermore, accumulation of ssDNA preceded the detectable appearance of DSBs, suggesting that ssDNA formation at replication forks is a precursor to chromosome breakage at those locations.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and media</h3><p id="p-7">HM14-3a (<em>MATa RAD53 bar1-1 his6 leu2-3,112 trp1-289</em>) and WFA34 (<em>MATa RAD53::rad53K227A(KanMX4) bar1-1 his6 leu2-3,112 trp1-289</em>) strains are derivatives of RM14-3a in an A364a background through gene conversions as previously described (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Feng <em>et al.</em> 2006</a>). BY2006 (<em>MATa mec1-1::HIS3 his3 leu2 trp1 ura3</em>), also an A364a derivative, was provided by Dr. Linda Breeden at the Fred Hutchinson Cancer Research Center. RCY378 (<em>MATa ho::LYS2 mec1::LEU2 lys2 ura3 leu2::hisG ade2::LK his4x arg4Ndel::mec1-4-kanMX4</em>) and the isogenic <em>MEC1</em> derivative RCY301 were provided by Dr. Rita Cha at the MRC National Institute for Medical Research. YSCL004 (<em>MATalpha ∆ho ∆hml::ADE1 ∆hmr::ADE1 ade1-110 leu2,3-112 lys5 trp1::hisG ura3-52 ade3::GAL10:HO Chr VI 97749 nt::HPH:HOcs Chr II 251662-251762 nt::HOcs:URA3</em>) was generously given by Jim Haber and C-S. Lee at Brandeis University. Cells were grown at 30° in synthetic complete medium unless otherwise indicated. Alpha factor was used at 200 nM for <em>bar1</em> strains and 3 μM for <em>BAR1</em> strains. Pronase was used at 25 μg/ml and 300 μg/ml for <em>bar1</em> and <em>BAR1</em> strains, respectively, to remove alpha factor from the culture medium. HU was added at 200 mM.</p></div><div id="sec-3" class="subsection"><h3>CHEF gel electrophoresis</h3><p id="p-8">CHEF gel analysis was performed as described previously (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Van Brabant <em>et al.</em> 2001</a>). Electrophoresis was conducted at 14° for 26 hr with a switch time ramped from 60 to 120 sec at 200 volts. Standard procedures were used to ethidium bromide stain and photograph the gel.</p></div><div id="sec-4" class="subsection"><h3>In-gel ssDNA labeling</h3><p id="p-9">Approximately 5 × 10<sup>8</sup> cells were collected for each sample during the time course of HU exposure and subsequent recovery. The cells were then embedded in agarose plugs and spheroplasted using the same procedures as in the CHEF gel electrophoresis analysis. The agarose plugs were pre-equilibrated in 10 mM Tris-HCl, pH 8.0, 0.1 mM EDTA (5 ml per plug) for 30 min, followed by equilibration in 5 ml of 50 mM Tris-HCl pH 6.8, 5 mM MgCl<sub>2</sub>, and 10 mM β-mercaptoethanol for 30 min at room temperature. For each sliver of agarose plug containing 10<sup>8</sup> cells in approximately 50 μl, 50 μl of labeling mix [50 mM Tris-HCl (pH 6.8); 5 mM MgCl<sub>2</sub>; 10 mM β-mercaptoethanol; 0.24 mM of each of dATP, dCTP, and dGTP; 0.12 mM of dTTP; 0.12 mM Cy5 or Cy3-dUTP; 250 μg/ml random hexamers; and 150 units of Klenow (exonuclease deficient, New England Biolab)] were added atop the plug, and the plug was incubated at 37° in the dark for 2 hr. The embedded labeled DNA was then electroeluted from the agarose plug in Spectra/Por dialysis tubing (Spectrum) with a 12,000-14,000 MWCO in 0.5× TBE at 110 volts at room temperature in the dark for 3 hr. The eluted DNA was then sonicated using a BioRuptor (Diagenode) to reduce the average size to 500 bp and purified using the Qiagen PCR Cleanup Kit. The resulting DNAs from a control and an experimental sample, which were differentially labeled with Cy-dUTP, were mixed together and readied for microarray analysis.</p></div><div id="sec-5" class="subsection"><h3>In-gel DSB labeling</h3><p id="p-10">Sample preparation prior to labeling was performed as described above for the in-gel ssDNA labeling. Restriction digestion of DNA in agarose plugs was performed as described previously (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Feng <em>et al.</em> 2009</a>). After the agarose plugs were pre-equilibrated in 10 mM Tris-HCl (pH 8.0) and 0.1 mM EDTA, they were equilibrated in 1× End-Repair buffer [Epicentre, 33 mM Tris-acetate (pH 7.8), 66 mM KAc, 10 mM MgAc<sub>2</sub>, and 5 mM dithiothreitol] at 5 ml/plug for 30 min at room temperature. For each sliver of agarose plug containing 10<sup>8</sup> cells in approximately 50 μl, 50 μl of End-Repair labeling mix (1× End-Repair buffer; 1 mM ATP; 0.24 mM of each of dATP, dCTP, and dGTP; 0.12 mM of dTTP; 0.12 mM Cy5 or Cy3-dUTP; and 3 μl of End-Repair enzyme mix) were added, and the plug was incubated at room temperature in the dark for 1 hr. Electroelution and subsequent analyses were performed identically as described above.</p></div><div id="sec-6" class="subsection"><h3>Microarray analysis</h3><p id="p-11">The experimental and control DNA samples were mixed for cohybridization to the Agilent G4493A yeast 4x44K ChIP to chip DNA microarrays according to the manufacturer’s recommendations. Data extraction was performed using Agilent’s Feature Extraction software. After removing those array spots flagged by the software as anomalous, the ratio of background-subtracted fluorescent signals from the experimental to the control sample was calculated for each probe. The resulting ratios for all the probe locations on each chromosome were normalized to the total amount of signals in each fluorescent channel and smoothed with a 6 kb window using a Lowess smoothing algorithm as previously described (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Feng <em>et al.</em> 2009</a>). The “chromosome breakage profile” was generated by plotting the smoothed ratios against chromosome coordinates. Identification of peaks in the profiles was performed as previously described (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Feng <em>et al.</em> 2009</a>). The ssDNA and chromosome breakage mapping in <em>mec1</em> cells were each performed three times and twice, respectively, with reproducible profiles. The <em>Bam</em>HI, <em>Fsp</em>I, and HO cleavage experiments were performed only once. The significant chromosome breakage sites in <em>mec1</em> cells after 1 h recovery from exposure to HU obtained from two independent experiments and origin locations used for statistical tests are reported (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000554/-/DC1/000554.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000554/-/DC1/TableS1.txt">Table S1</a>). All data files for raw ratios of ssDNA or breakage are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000554/-/DC1/FileS1.zip">File S1</a>.</p></div><div id="sec-7" class="subsection"><h3>Random simulation test</h3><p id="p-12">To test for an association between break sites and chromosomal features within 6 kb, the distance between each break site and its nearest chromosomal feature of interest was first established. Distances between breaks and chromosomal features were measured from midpoint to midpoint. Breaks with chromosomal features within 6 kb were counted. The null distribution of the number of breaks with a given chromosomal feature within 6 kb was determined by randomizing the location of break sites and determining the distance between these random breaks and the chromosomal features. Break sites were randomized by randomly selecting an equal number of positions on the microarray. Ten thousand simulations were run, and in each run, the number of random breaks with a given chromosomal feature within 6 kb was recorded. <em>P</em> values were obtained from the upper tail of the null distribution in which the number of breaks with a chromosomal feature within 6 kb was greater than or equal to that seen for the actual set of breaks. The list of Rad53-checked and unchecked origins used in these analyses were obtained from the previous study (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Feng <em>et al.</em> 2006</a>).</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Previously single-stranded regions of the genome remain single-stranded even after HU removal in <em>mec1</em> cells</h3><p id="p-13">What regions of the <em>mec1</em> chromosomes are most susceptible to breakage in the presence of HU and during recovery from HU? Because we previously observed that extensive ssDNA formed at the replication forks in <em>mec1</em> cells exposed to HU and that the forks were incapable of resuming synthesis after HU removal, we wondered whether the ssDNA at the replication forks would persist and become sites of DSBs. We tested for persistence of ssDNA in <em>mec1</em> cells following a transient exposure to HU. We synchronized <em>mec1</em> cells in G1 with the mating pheromone alpha factor, followed by release into S phase in the presence of 200 mM HU, and then collected samples at the beginning (0 hr) and after 1 hr (HU 1hr). We then filtered the cell culture, allowed the cells to “recover” in fresh medium without HU, and collected samples after 1 hr (R 1hr). We have previously described a method based on random-primed labeling without denaturation of genomic template DNA to map ssDNA genome-wide (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Feng <em>et al.</em> 2006</a>). We have now modified this procedure (see <em>Materials and Methods</em>) and prepared genomic DNA by embedding cells in agarose plugs followed by spheroplasting (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). We then performed ssDNA labeling in gel to minimize ssDNA production <em>in vitro</em>. An “S phase” sample, either HU 1hr or R 1hr, was paired with a control “G1 phase” sample, and the two samples were differentially labeled with Cy-conjugated dUTPs. Labeled DNA was eluted from the agarose, fragmented, and cohybridized to Agilent microarray slides. The relative amount of ssDNA in the “S phase” sample was calculated as the ratio of fluorescent signals from the “S phase” sample to that from the “G1 phase” sample for each chromosome coordinate. As shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>, ssDNA was detected near origins of replication in the HU 1hr sample (orange profile). The reproducibility of this new in-gel labeling methodology is exemplified by the correlation coefficients for the pair-wise comparisons of microarray data from three independent experiments: 0.91, 0.84, and 0.80. Interestingly, the ssDNA profiles of the R 1hr sample (red profile) and the HU 1hr sample were virtually identical (correlation coefficient = 0.93), indicating that the single-stranded gaps remained unfilled even after HU was removed. To validate this modified method, we also mapped ssDNA in a <em>rad53K227A</em> mutant in HU, which we had previously demonstrated, using the old methodology, to accumulate ssDNA at all known origins (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Feng <em>et al.</em> 2006</a>). The two ssDNA profiles were nearly identical with even better resolution attributed to the in-gel labeling method (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000554/-/DC1/FigureS1.pdf">Figure S1</a>). We were also concerned that the ssDNA labeling we observed might be the result of exonuclease-deficient (exo–) Klenow’s strand displacement synthesis—extending the 3′-end of a nick in the DNA and displacing the downstream DNA strand—rather than the result of filling ssDNA gaps in the template. Therefore, we tested the labeling with Sequenase (an exo– T7 DNA polymerase that does not possess the strand displacement activity). The ssDNA profiles produced by Klenow and Sequenase are very similar, thus confirming that <em>in vitro</em> synthesis is labeling ssDNA gaps, not merely nicks, in <em>mec1</em> cells in HU (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000554/-/DC1/FigureS2.pdf">Figure S2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/5/327/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Outline of experimental procedures for a modified in-gel ssDNA labeling and identification method. (B) ssDNA persists in mec1 cells recovering from exposure to HU. The ssDNA profiles for mec1-1 cells collected after cells were released from alpha factor arrest and exposed to 200 mM HU for 1 hr (&#x201C;ssDNA_HU 1hr&#x201D;; orange profile) and after recovering from 1 hr exposure in media without HU for another 1 hr (&#x201C;ssDNA_R 1hr&#x201D;; red profile) are shown. The symbols at the top of each graph indicate the locations of replication origins: red squares, checked (late/inefficient) origins and green dots, unchecked (early/efficient) origins (Feng et al. 2006). Chromosome numbers are indicated in Roman numerals. Centromere locations are shown as black dots on the X-axis. The same symbols are used for all remaining chromosome profiles throughout the article." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-893812806" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Outline of experimental procedures for a modified in-gel ssDNA labeling and identification method. (B) ssDNA persists in &lt;em&gt;mec1&lt;/em&gt; cells recovering from exposure to HU. The ssDNA profiles for &lt;em&gt;mec1-1&lt;/em&gt; cells collected after cells were released from alpha factor arrest and exposed to 200 mM HU for 1 hr (&#x201C;ssDNA_HU 1hr&#x201D;; orange profile) and after recovering from 1 hr exposure in media without HU for another 1 hr (&#x201C;ssDNA_R 1hr&#x201D;; red profile) are shown. The symbols at the top of each graph indicate the locations of replication origins: red squares, checked (late/inefficient) origins and green dots, unchecked (early/efficient) origins (Feng &lt;em&gt;et al.&lt;/em&gt; 2006). Chromosome numbers are indicated in Roman numerals. Centromere locations are shown as black dots on the X-axis. The same symbols are used for all remaining chromosome profiles throughout the article.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/5/327/F1.medium.gif" width="313" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/1/5/327/F1.medium.gif" width="313" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/5/327/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/5/327/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460977" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-14" class="first-child">(A) Outline of experimental procedures for a modified in-gel ssDNA labeling and identification method. (B) ssDNA persists in <em>mec1</em> cells recovering from exposure to HU. The ssDNA profiles for <em>mec1-1</em> cells collected after cells were released from alpha factor arrest and exposed to 200 mM HU for 1 hr (“ssDNA_HU 1hr”; orange profile) and after recovering from 1 hr exposure in media without HU for another 1 hr (“ssDNA_R 1hr”; red profile) are shown. The symbols at the top of each graph indicate the locations of replication origins: red squares, checked (late/inefficient) origins and green dots, unchecked (early/efficient) origins (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Feng <em>et al.</em> 2006</a>). Chromosome numbers are indicated in Roman numerals. Centromere locations are shown as black dots on the X-axis. The same symbols are used for all remaining chromosome profiles throughout the article.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>A new genome-wide chromosome breakage mapping method</h3><p id="p-15">If the ssDNAs that persist after removal of HU contribute to chromosome fragility, then their locations should correspond to sites of breakage. On a genome-wide level, meiotic DSBs have been mapped by two microarray-based methods (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Blitzblau <em>et al.</em> 2007</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Buhler <em>et al.</em> 2007</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cotta-Ramusino <em>et al.</em> 2005</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Robine <em>et al.</em> 2007</a>) and a whole-genome sequencing-based method (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Pan <em>et al.</em> 2011</a>). These methods monitor either Spo11 binding (via chromosome immunoprecipitation followed by microarray hybridization or sequencing, ChIP-chip, or ChIP-Seq) or the ssDNA tail left by Spo11 at the DSBs (via benzoyl-napthoyl-DEAE-cellulose ssDNA enrichment). CFSs can also be identified through the detection of phosphorylated γ-H2AX, which binds DSBs induced by a variety of conditions (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Szilard <em>et al.</em> 2010</a>). However, these methods do not label the terminal DSBs directly and could be prone to false positives. For instance, many Spo11 binding sites are not cleaved. Likewise, it is formally possible that with the ssDNA enrichment method, an internal ssDNA gap that is not associated with the DSB might be falsely identified as a product of DSB processing. It has also been reported that γ-H2AX foci were detected in the absence of apparent DSBs (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Banath <em>et al.</em> 2004</a>). Moreover, ChIP-chip or ChIP-Seq is not yet amenable to our study because it is unclear which proteins will be stably present at the HU-induced DSBs. Finally, reliably detecting DSBs is confounded by random breaks occurring <em>in vitro</em> during DNA isolation.</p><p id="p-16">Therefore, we developed a new and versatile genome-wide DSBs mapping method (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). To minimize DSBs generated by <em>in vitro</em> manipulations, we again prepared genomic DNA from yeast cells that were embedded in agarose plugs and processed for cell wall disruption and protein degradation. We then incorporated Cy-conjugated dUTP at DNA ends in gel with the End-It labeling kit (Epicentre), which utilizes both the polymerase and the 3′-5′ exonuclease activities of the T4 DNA polymerase and can process DNA with either 3′ or 5′ overhangs to generate primarily blunt ends. (The End-It kit also contains polynucleotide kinase, which adds a 5′-phosphate on the DNA end to facilitate cloning. This feature was not required for our experimental purpose, but it produced no ill effect.) We labeled DNA derived from an equal number of cells from the experimental sample (<em>e.g.</em>, R 1hr sample that contains DSBs) and from a control sample (nonreplicating DNA without apparent breaks) differentially with Cy3- and Cy5-conjugated dUTPs. We then eluted the DNA from agarose plugs and, after sonicating the DNA to reduce the average fragment size, cohybridized them to Agilent DNA microarrays. The relative level of DSBs was expressed as the ratio of fluorescence signals from the experimental sample to those from the control. Data smoothing and peak identification were performed as previously described (<a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Feng <em>et al.</em> 2009</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/5/327/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosome breakage mapping. (A) Outline of microarray-based genome-wide chromosome breakage mapping. For details, see text. (B) End-labeled profile of Chr III from a control sample (log phase mec1 cells) that contains in vitro&#x2013;generated BamHI ends. The gray vertical lines indicate positions of BamHI digestion sites predicted from the sequence of the S. cerevisiae reference genome (S288C). Blue arrows indicate potential polymorphic BamHI sites in the yeast strain used in this study. (C) End-labeled profile of mec1 cells at &#x201C;mec1_HU 1hr&#x201D; (cyan) and &#x201C;mec1_R 1hr&#x201D; (blue). (D) End-labeled profile of MEC1 cells at &#x201C;MEC1_HU 1hr&#x201D; (cyan) and &#x201C;MEC1_R 1hr&#x201D; (blue)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-893812806" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chromosome breakage mapping. (A) Outline of microarray-based genome-wide chromosome breakage mapping. For details, see text. (B) End-labeled profile of Chr III from a control sample (log phase &lt;em&gt;mec1&lt;/em&gt; cells) that contains &lt;em&gt;in vitro&lt;/em&gt;&#x2013;generated &lt;em&gt;Bam&lt;/em&gt;HI ends. The gray vertical lines indicate positions of &lt;em&gt;Bam&lt;/em&gt;HI digestion sites predicted from the sequence of the &lt;em&gt;S. cerevisiae&lt;/em&gt; reference genome (S288C). Blue arrows indicate potential polymorphic &lt;em&gt;Bam&lt;/em&gt;HI sites in the yeast strain used in this study. (C) End-labeled profile of &lt;em&gt;mec1&lt;/em&gt; cells at &#x201C;&lt;em&gt;mec1&lt;/em&gt;_HU 1hr&#x201D; (cyan) and &#x201C;&lt;em&gt;mec1&lt;/em&gt;_R 1hr&#x201D; (blue). (D) End-labeled profile of &lt;em&gt;MEC1&lt;/em&gt; cells at &#x201C;&lt;em&gt;MEC1&lt;/em&gt;_HU 1hr&#x201D; (cyan) and &#x201C;&lt;em&gt;MEC1&lt;/em&gt;_R 1hr&#x201D; (blue).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/5/327/F2.medium.gif" width="440" height="403"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/1/5/327/F2.medium.gif" width="440" height="403"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/5/327/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/5/327/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460914" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-17" class="first-child">Chromosome breakage mapping. (A) Outline of microarray-based genome-wide chromosome breakage mapping. For details, see text. (B) End-labeled profile of Chr III from a control sample (log phase <em>mec1</em> cells) that contains <em>in vitro</em>–generated <em>Bam</em>HI ends. The gray vertical lines indicate positions of <em>Bam</em>HI digestion sites predicted from the sequence of the <em>S. cerevisiae</em> reference genome (S288C). Blue arrows indicate potential polymorphic <em>Bam</em>HI sites in the yeast strain used in this study. (C) End-labeled profile of <em>mec1</em> cells at “<em>mec1</em>_HU 1hr” (cyan) and “<em>mec1</em>_R 1hr” (blue). (D) End-labeled profile of <em>MEC1</em> cells at “<em>MEC1</em>_HU 1hr” (cyan) and “<em>MEC1</em>_R 1hr” (blue).</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">We first performed a control experiment to map chromosome breakage in samples containing known DSBs. We prepared DNA embedded in agarose plugs as described above from exponentially growing <em>mec1</em> cells and digested the DNA in gel with the restriction enzyme <em>Bam</em>HI. We then differentially labeled the digested and the undigested control DNA in gel with Cy-conjugated dUTPs and carried out the procedures described above to obtain a “chromosome breakage profile” of the cells containing <em>Bam</em>HI ends (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). The mapped DSBs correlated nearly perfectly with the known <em>Bam</em>HI sites in the genome. There were some sites mapped as chromosome breaks that did not correspond to known <em>Bam</em>HI sites (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>, blue arrows), suggesting that they might be polymorphisms in our laboratory strain A364a compared with the sequenced strain S288C. Similar results were obtained when we performed the same experiment using the blunt cutter <em>Fsp</em>I (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000554/-/DC1/FigureS3.pdf">Figure S3</a>). However, the end-repair labeling of blunt <em>Fsp</em>I-digested DNA ends was not as efficient as it was for the sticky ends generated by <em>Bam</em>HI. We reasoned that the DSBs generated during the HU challenge are unlikely to be blunt ended and that this is a potential caveat of the method.</p><p id="p-19">We also asked whether our method was able to detect <em>in vivo</em> DSBs induced by the HO endonuclease in YSCL004, a strain that contains three irreparable HO cut sites due to the lack of donor sequence (one site each on Chr II, III, and VI). The HO gene was controlled by the galactose-inducible Gal1 promoter. Log phase cells grown in media containing glycerol were split into two cultures. Glucose (Glu) or galactose (Gal) was added to the cultures to repress or induce HO activity, respectively. Both cultures were incubated for 1.5 hr to allow the nucleolytic events to proceed in the Gal culture before the addition of glucose to stop further HO cutting. DNA samples were prepared by spheroplasting after cells were embedded in agarose as described above, differentially end-labeled with Cy-dyes, and cohybridized to the microarray. The relative amount of DSBs in the Gal sample was quantified as the ratio of fluorescent signal from the Gal sample to that from the Glu sample. We detected all three HO cut sites as regions that are enriched for end-labeled signals in the Gal sample (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000554/-/DC1/FigureS4.pdf">Figure S4</a>).</p><p id="p-20">Finally, we performed another control experiment to assess how specific the end-repair reaction is for the DNA ends <em>vs.</em> internal stretches of ssDNA that are not associated with DSBs. Our previous observations indicated that chromosomes did not break in <em>mec1</em> cells during HU exposure but that they did contain extensive ssDNA (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). If an internal ssDNA stretch were to contain an adjacent free 3′-OH group, it would be a suitable primer for the T4 DNA polymerase and skew the results of breakage mapping. As shown in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a> (light-blue profile), the HU 1hr sample exhibited a rather flat breakage profile and did not show significant signals at the ssDNA regions near the origins. This observation suggests that the end-repair reaction does not label internal stretches of ssDNA efficiently and/or that the ssDNA gaps in the HU-treated <em>mec1</em> cells do not contain free 3′-OH groups.</p></div><div id="sec-11" class="subsection"><h3>Chromosome breakage occurs at ssDNA locations</h3><p id="p-21">We mapped DSBs in the <em>mec1</em> sample that contained extensive chromosome breakage (R 1hr) shown previously (<a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">Feng <em>et al.</em> 2009</a>), and we found specific labeling at multiple sites in the genome (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>, dark-blue profile). The magnitude of breakage in <em>mec1</em> cells was comparable to that at the <em>Bam</em>HI sites, but it was much lower than at the HO sites, demonstrating that chromosome breakage in the <em>mec1</em> cells was occurring genome-wide rather than at discrete sites (<a id="xref-ref-9-8" class="xref-bibr" href="#ref-9">Feng <em>et al.</em> 2009</a>). To confirm our results, we validated a chromosome breakage site on Chr II at 620 kb by indirect end-labeling (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000554/-/DC1/FigureS5.pdf">Figure S5</a>). In contrast, the wild type (WT) cells in both the HU 1hr and the R 1hr samples did not exhibit significant levels of chromosome breakage (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a>).</p><p id="p-22">At first glance, the sites of breakage in <em>mec1</em> cells recovering from HU appear to occur at specific locations with no obvious correlation to known origins or other landmarks. However, when origins were parsed into the categories of “early/efficient” (origins that fire in HU in the presence of an activated checkpoint; unchecked origins) or “late/inefficient” (origins that are delayed in firing after activation of the checkpoint; checked origins), breakage sites were better correlated with forks that initiated from the checked origins (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2C</a>, red squares) than from the unchecked origins (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2C</a>, green dots). To perform this comparison, we first identified those chromosome positions where significant (above median level) breakage was observed. We then asked what percentage of these chromosome positions are within 6 kb of the nearest origin of replication. We chose a 6 kb cutoff because the Lowess smoothing window size of our microarray data was set at 6 kb. A significantly greater percentage of breakage sites was found near the checked origins (41.6%) than near the unchecked origins (11.7%) (<em>P</em> &lt; 10<sup>−15</sup>) in a two-sample test for equality of proportions with continuity correction. To test this correlation more rigorously, we also performed a random simulation test and found that the breakage sites were correlated with the checked origins (<em>P</em> &lt; 0.0001) but not with the unchecked origins (<em>P</em> = 0.1851) within a 6 kb distance. We interpret these results as follows. In the <em>mec1</em> mutant, all origins fire (<a id="xref-ref-9-9" class="xref-bibr" href="#ref-9">Feng <em>et al.</em> 2009</a>); however, it appears that the temporal pattern of origin activation remains, as the ssDNA profiles indicated that replication forks migrated away from the unchecked (early) origins but were still in the vicinity of the checked (late) origins. This result suggests that chromosome breakage is correlated with replication fork progression.</p><p id="p-23">Comparison of the breakage profile of the R 1hr sample and the ssDNA profile of the HU 1hr sample (which records the positions of replication forks) reveals significant similarity (correlation coefficient = 0.64; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Cell synchrony was reduced for the R 1hr sample (2 hr after release from G1 arrest), which likely contributed to the relatively lower resolution of signals, as two independent breakage profiles of the R 1hr samples showed a correlation coefficient of 0.67. That the regional accumulation of ssDNA in HU occurs before breaks are detected in the recovery phase supports the hypothesis that ssDNA formation precedes the occurrence of chromosome breakage and dictates their location. Thus, the identification of ssDNA in the genome can be used to predict sites of chromosome breakage, an observation that has profound implications for the study of CFSs in the human genome. The end-labeling method also detected strong signals at Ty elements (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, Chr VII and IX, arrows). We entertained the possibility that chromosome breakage is generated at these genomic locations as replication forks progress through the Ty elements, but we found no evidence of breakage at these chromosomal sites by indirect end-labeling of Southern blots (data not shown). Given that Ty3 transposition is cell cycle regulated (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Menees and Sandmeyer 1994</a>), we speculate that we may be detecting double-stranded DNA templates arising from cDNA synthesis and replication of Ty elements. Because Ty elements are repetitive, we could not determine with certainty where they map to in the genome using the array-based method.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/5/327/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation of sites of breakage with ssDNA formation. The chromosome breakage profile of sample &#x201C;R 1 hr&#x201D; (blue) resembles the ssDNA profile detected in sample &#x201C;HU 1 hr&#x201D; (orange). Profiles for Chr II, VI, VII (left) and IX (right) are shown. The positions of Ty elements are indicated by blue arrows; the category of Ty element is indicated by a label above each peak." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-893812806" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlation of sites of breakage with ssDNA formation. The chromosome breakage profile of sample &#x201C;R 1 hr&#x201D; (blue) resembles the ssDNA profile detected in sample &#x201C;HU 1 hr&#x201D; (orange). Profiles for Chr II, VI, VII (left) and IX (right) are shown. The positions of Ty elements are indicated by blue arrows; the category of Ty element is indicated by a label above each peak.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/5/327/F3.medium.gif" width="440" height="403"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/1/5/327/F3.medium.gif" width="440" height="403"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/5/327/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/5/327/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460887" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-24" class="first-child">Correlation of sites of breakage with ssDNA formation. The chromosome breakage profile of sample “R 1 hr” (blue) resembles the ssDNA profile detected in sample “HU 1 hr” (orange). Profiles for Chr II, VI, VII (left) and IX (right) are shown. The positions of Ty elements are indicated by blue arrows; the category of Ty element is indicated by a label above each peak.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Chromosome breakage is correlated with replication fork progression</h3><p id="p-25">We have demonstrated that the locations of chromosome breakage are correlated with ssDNA production at stalled replication forks, providing the first direct evidence that collapsed replication forks lead to DSBs. However, as many of the collapsed forks in HU occur near checked (late/inefficient) origins of replication, it is formally possible that this correlation of chromosome breakage with forks is unique to this specific subset of origins of replication rather than to the process of fork progression itself. To subject our hypothesis to a more rigorous test, we allowed replication forks to travel greater distances from the origins before HU treatment and then measured the sites of chromosome breakage in samples recovering from HU (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). If it were the ssDNA at forks that contributed to chromosome fragility rather than late/inefficient origins <em>per se</em>, then the break sites should now be found at varying distances from these origins.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/5/327/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosome breakage is correlated with replication fork progression. (A) Schematic representation of the scenarios of replication fork progression and the resulting ssDNA profiles in samples that were released into S phase in the presence of HU at different times. The nomenclature of samples is described in the main text. (B) Experimental scheme: mec1 cells were released from alpha factor arrest into medium containing HU at the indicated times. The cells were exposed to HU for 1 hr followed by recovery in fresh medium without HU for 1 hr. For details, see text. (C) CHEF gel electrophoresis of samples from panel B. Chromosome breakage is evident in the recovery &#x201C;R&#x201D; samples as indicated. The position of Chr IV, which comigrates with Chr XII, is indicated. (D) Chr IV breakage profiles of recovery &#x201C;R&#x201D; samples from cells released into HU at the beginning of the G1/S transition (T0-R; cyan) or after an elapsed 20 min (T20-R; blue). (E) Chr IV breakage profile of the T0-R sample in panel D (cyan) overlaid with ssDNA profile of the cells before recovery (T0-HU; orange). (F) Chr IV breakage profile of the T20-R sample in panel D (blue) overlaid with ssDNA profile of the cells before recovery (T20-HU; red). (D&#x2013;F) The left portion of Chr IV is shown in each profile." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-893812806" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chromosome breakage is correlated with replication fork progression. (A) Schematic representation of the scenarios of replication fork progression and the resulting ssDNA profiles in samples that were released into S phase in the presence of HU at different times. The nomenclature of samples is described in the main text. (B) Experimental scheme: &lt;em&gt;mec1&lt;/em&gt; cells were released from alpha factor arrest into medium containing HU at the indicated times. The cells were exposed to HU for 1 hr followed by recovery in fresh medium without HU for 1 hr. For details, see text. (C) CHEF gel electrophoresis of samples from panel B. Chromosome breakage is evident in the recovery &#x201C;R&#x201D; samples as indicated. The position of Chr IV, which comigrates with Chr XII, is indicated. (D) Chr IV breakage profiles of recovery &#x201C;R&#x201D; samples from cells released into HU at the beginning of the G1/S transition (T0-R; cyan) or after an elapsed 20 min (T20-R; blue). (E) Chr IV breakage profile of the T0-R sample in panel D (cyan) overlaid with ssDNA profile of the cells before recovery (T0-HU; orange). (F) Chr IV breakage profile of the T20-R sample in panel D (blue) overlaid with ssDNA profile of the cells before recovery (T20-HU; red). (D&#x2013;F) The left portion of Chr IV is shown in each profile.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/5/327/F4.medium.gif" width="206" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/1/5/327/F4.medium.gif" width="206" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/5/327/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/5/327/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460946" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-26" class="first-child">Chromosome breakage is correlated with replication fork progression. (A) Schematic representation of the scenarios of replication fork progression and the resulting ssDNA profiles in samples that were released into S phase in the presence of HU at different times. The nomenclature of samples is described in the main text. (B) Experimental scheme: <em>mec1</em> cells were released from alpha factor arrest into medium containing HU at the indicated times. The cells were exposed to HU for 1 hr followed by recovery in fresh medium without HU for 1 hr. For details, see text. (C) CHEF gel electrophoresis of samples from panel B. Chromosome breakage is evident in the recovery “R” samples as indicated. The position of Chr IV, which comigrates with Chr XII, is indicated. (D) Chr IV breakage profiles of recovery “R” samples from cells released into HU at the beginning of the G1/S transition (T0-R; cyan) or after an elapsed 20 min (T20-R; blue). (E) Chr IV breakage profile of the T0-R sample in panel D (cyan) overlaid with ssDNA profile of the cells before recovery (T0-HU; orange). (F) Chr IV breakage profile of the T20-R sample in panel D (blue) overlaid with ssDNA profile of the cells before recovery (T20-HU; red). (D–F) The left portion of Chr IV is shown in each profile.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">The experimental scheme is illustrated in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>. After synchronizing <em>mec1</em> cells at the G1/S boundary with alpha factor, we split the cell culture into five aliquots. We released the cells into S phase by pronase addition and added 200 mM HU at 0, 5, 10, 15, and 20 min after the release to the five aliquots denoted as T0, T5, T10, T15, and T20, respectively. We exposed each cell culture to HU for 1 hr before collecting the samples. We then washed the cells and transferred them into fresh medium without HU to let them “recover” for 1 hr. For clarity, we refer to the samples by their aliquot number followed by the description of the treatment they received: “T#” refers to the elapsed time (in minutes) after alpha factor removal and before HU addition; “HU” for a 1 hr exposure to HU; and “R” for a 1 hr recovery from HU. Thus, T5-HU designates the sample in which HU was added at 5 min after the release to S phase and which had been exposed to HU for 1 hr. Likewise, T20-R designates the sample in which HU was added at 20 min after release to the S phase, incubated for 1 hr in HU, and then allowed to recover for 1 hr after HU was removed.</p><p id="p-28">DNA was prepared from each sample and subjected to contour-clamped homogeneous electric field (CHEF) gel electrophoresis as previously described (<a id="xref-ref-9-10" class="xref-bibr" href="#ref-9">Feng <em>et al.</em> 2009</a>). As cells enter S phase, linear chromosomes became depleted from the gel as the branched replicating chromosomes were trapped in the well (most obviously seen for the large chromosomes in T20; <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a>, lane 5). This property was even more striking for the HU-treated samples (lanes 6-10) where replication forks were slowed. The reappearance of the large chromosomes in the T20-HU sample (lane 10) suggests that many cells were able to complete replication when HU was added at this late time in S phase. However, regardless of when the cells encountered HU, they all exhibited chromosome breakage during recovery (lanes 11-15). We then tested the correlation between fork progression and chromosome breakage: if forks had proceeded away from the origins, we would expect the sites of ssDNA to reflect this fork migration and, more importantly, the breakage sites to be different. We mapped chromosome breakage in the T20-R sample in which replication forks migrated the farthest, and then we mapped chromosome breakage in the control sample T0-R. As shown in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4D</a>, the two breakage profiles were indeed different (correlation coefficient = 0.17). Chromosome breakage at early replicating regions was reduced (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4D</a>, coordinates 400-500 kb) and more broadly distributed in the regions replicated by forks from unchecked origins.</p><p id="p-29">To quantify the changes in breakage patterns, we again identified those chromosome positions where significant (above median level) breakage was observed. We then asked whether the break sites were correlated with origin locations within a 6 kb distance in the random simulation test. As was consistent with previous observations, neither the T0-R nor the T20-R sample showed correlation between the breakage sites and the unchecked origins (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). As before, the break sites in the T0-R sample were correlated with checked origins (<em>P</em> = 0.0032); however, the break sites in the T20-R sample were not (<em>P</em> = 0.8276; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). These observations indicate that chromosome breakage is less well correlated with origin locations when replication forks have sufficient nucleotides and time to move into flanking regions.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/460883/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/460883/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title">Correlation between origins of replication and chromosome breakage sites</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">Finally, we also monitored replication fork progression by ssDNA mapping in the T0-HU and T20-HU samples. Our results once again showed that the ssDNA profiles of the samples in HU prior to the occurrence of breakage were very similar to the breakage profiles (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4E, F</a>). The correlation coefficient between the T0-HU ssDNA and T0-R breakage profiles was 0.74 and that between the T20-HU ssDNA and T20-R breakage profiles was 0.65. Note that there was a negative, albeit weak, correlation (coefficient = -0.25) between the T0-HU ssDNA and T20-R breakage profiles, suggesting that the origin-proximal regions were protected from chromosome breakage if the forks were allowed to migrate away from them before encountering HU. Thus, our analyses demonstrated that (1) chromosome breakage is correlated with replication fork progression; (2) ssDNA formation is detected prior to chromosome breakage; and (3) sites of ssDNA formation predict the sites of chromosome breakage.</p></div><div id="sec-13" class="subsection"><h3>ssDNA is also detected prior to chromosome breakage in a <em>mec1<sup>ts</sup></em> mutant without external replication stress</h3><p id="p-33">On the basis of our results, we postulated that ssDNA formation as a result of replication stress is a common precursor to DSBs. We tested this hypothesis by asking whether we could detect ssDNA at other replication stress–induced chromosome breakage sites prior to breakage taking place. It was reported that a temperature-sensitive <em>mec1-4</em> mutant has defective replication fork progression at the restrictive temperature, without external challenge by HU, and experiences chromosome breakage later in the cell cycle (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cha and Kleckner 2002</a>). The breakage occurred at regions of the genome described as “replication slow zones” (RSZ), which are primarily replication termination sites. We set out to determine whether and when ssDNA formation is detected in the RSZs. We mapped ssDNA in <em>mec1-4</em> as well as <em>MEC1</em> cells in mid–S phase (40 min after release from the G1 arrest). As shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, <em>MEC1</em> cells showed a background level of ssDNA, indicating that replication forks were not stalled at specific regions of the genome. In contrast, the <em>mec1-4</em> cells showed distinct patterns of elevated levels of ssDNA near the replication termini, the very regions where chromosome breakage was shown to take place (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Cha and Kleckner 2002</a>). Although chromosome breakage was not observed until 4 to 6 hr after cells were released from G1, ssDNA was observed after only 40 min.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/1/5/327/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ssDNA predicts sites of eventual chromosome breakage in a mec1-4ts mutant. MEC1 or mec1-4ts cells were released from alpha factor arrest to enter S phase at the restrictive temperature (37&#xB0;), and samples were collected after 40 min. Plotted are ssDNA profiles of the MEC1 control sample (blue) and the mec1-4ts sample (orange). The light-gray shaded boxes are replication termination regions that show elevated levels of ssDNA in the mec1-4ts sample." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-893812806" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;ssDNA predicts sites of eventual chromosome breakage in a &lt;em&gt;mec1-4&lt;sup&gt;ts&lt;/sup&gt;&lt;/em&gt; mutant. &lt;em&gt;MEC1&lt;/em&gt; or &lt;em&gt;mec1-4&lt;sup&gt;ts&lt;/sup&gt;&lt;/em&gt; cells were released from alpha factor arrest to enter S phase at the restrictive temperature (37&#xB0;), and samples were collected after 40 min. Plotted are ssDNA profiles of the &lt;em&gt;MEC1&lt;/em&gt; control sample (blue) and the &lt;em&gt;mec1-4&lt;sup&gt;ts&lt;/sup&gt;&lt;/em&gt; sample (orange). The light-gray shaded boxes are replication termination regions that show elevated levels of ssDNA in the &lt;em&gt;mec1-4&lt;sup&gt;ts&lt;/sup&gt;&lt;/em&gt; sample.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/1/5/327/F5.medium.gif" width="440" height="153"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/1/5/327/F5.medium.gif" width="440" height="153"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/1/5/327/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/1/5/327/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/460954" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-34" class="first-child">ssDNA predicts sites of eventual chromosome breakage in a <em>mec1-4<sup>ts</sup></em> mutant. <em>MEC1</em> or <em>mec1-4<sup>ts</sup></em> cells were released from alpha factor arrest to enter S phase at the restrictive temperature (37°), and samples were collected after 40 min. Plotted are ssDNA profiles of the <em>MEC1</em> control sample (blue) and the <em>mec1-4<sup>ts</sup></em> sample (orange). The light-gray shaded boxes are replication termination regions that show elevated levels of ssDNA in the <em>mec1-4<sup>ts</sup></em> sample.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-35">We have developed a genome-wide chromosome breakage mapping method and used it to identify chromosome breakage in checkpoint-deficient <em>mec1-1</em> cells after exposure to and recovery from HU. It is a versatile method that can be applied to any system in which persistent DSBs are generated. We demonstrated that chromosome breakage is correlated with replication fork progression, providing direct evidence that chromosome fragility results from destabilization of the replication forks in the absence of the checkpoint when the nucleotide pool is reduced/depleted. Our study also demonstrated that prior to the occurrence of chromosome breakage, ssDNA could be detected at the sites of breakage. We thus hypothesized that ssDNA formation is a common precursor to chromosome fragility during replication stress. Using a <em>mec1-4</em> temperature-sensitive allele that exhibits chromosome fragility without external replication stress, we showed that ssDNA was indeed detectable at the sites of eventual chromosome breakage, lending additional support to our hypothesis. Our data provide a conceptual model where different forms of replication stress, despite having presumably different targets at the replication fork, cause cells to generate ssDNA at the fork as a consequence of either uncoupled synthesis on the two DNA templates or nucleolytic processing of synthesized DNA. We are currently determining the mechanisms of ssDNA formation during different replication stresses. We suggest that the detection of ssDNA can be used as a tool to identify sites of chromosome fragility and may signal when particular replication forks encounter impediments and/or become unstable.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-36">We are indebted to the Brewer/Raghuraman lab members and Julia Sidorova for support and insightful discussions. We also thank Linda Breeden, Rita Cha, Cheng-Sheng Lee, and Jim Haber for providing yeast strains for this study. We extend our gratitude to Jonathan Johnson and Hannah Gann for their technical assistance. This work was supported by the National Institute of General Medical Sciences Grant 18926 (B.J.B. and M.K.R.) and the Pathway to Independence Award 1K99GM081378-01 (W.F.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000554/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.111.000554/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-37"><em>Communicating editor: Robert Sclafani</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2011-06-07"><span class="received-label">Received </span>June 7, 2011.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2011-08-09"><span class="accepted-label">Accepted </span>August 9, 2011.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2011 Feng <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.1" data-doi="10.1101/gad.1392506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Admire</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shanks</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danzl</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weier</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cycles of chromosome instability are associated with a fragile site and are increased by defects in DNA replication and checkpoint controls in yeast.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1392506%26rft_id%253Dinfo%253Apmid%252F16384935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjAvMi8xNTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzUvMzI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.2" data-doi="10.1158/0008-5472.CAN-04-1433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banath</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macphail</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olive</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Radiation sensitivity, H2AX phosphorylation, and kinetics of repair of DNA strand breaks in irradiated cervical cancer cell lines.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">7144</span>–<span class="cit-lpage">7149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DBanath%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D64%26rft.issue%253D19%26rft.spage%253D7144%26rft.epage%253D7149%26rft.atitle%253DRadiation%2BSensitivity%252C%2BH2AX%2BPhosphorylation%252C%2Band%2BKinetics%2Bof%2BRepair%2Bof%2BDNA%2BStrand%2BBreaks%2Bin%2BIrradiated%2BCervical%2BCancer%2BCell%2BLines%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-04-1433%26rft_id%253Dinfo%253Apmid%252F15466212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI2NC8xOS83MTQ0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS81LzMyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.3" data-doi="10.1016/j.cub.2007.10.066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blitzblau</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochwagen</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mapping of meiotic single-stranded DNA reveals double-stranded-break hotspots near centromeres and telomeres.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2003</span>–<span class="cit-lpage">2012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBlitzblau%26rft.auinit1%253DH.%2BG.%26rft.volume%253D17%26rft.issue%253D23%26rft.spage%253D2003%26rft.epage%253D2012%26rft.atitle%253DMapping%2Bof%2Bmeiotic%2Bsingle-stranded%2BDNA%2Breveals%2Bdouble-stranded-break%2Bhotspots%2Bnear%2Bcentromeres%2Band%2Btelomeres.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2007.10.066%26rft_id%253Dinfo%253Apmid%252F18060788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2007.10.066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18060788&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251482100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.4" data-doi="10.1371/journal.pbio.0050324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buhler</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borde</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mapping meiotic single-strand DNA reveals a new landscape of DNA double-strand breaks in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DBuhler%26rft.auinit1%253DC.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253De324%26rft.epage%253De324%26rft.atitle%253DMapping%2Bmeiotic%2Bsingle-strand%2BDNA%2Breveals%2Ba%2Bnew%2Blandscape%2Bof%2BDNA%2Bdouble-strand%2Bbreaks%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050324%26rft_id%253Dinfo%253Apmid%252F18076285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076285&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.5" data-doi="10.1534/genetics.109.106385"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casper</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenwell</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Chromosome aberrations resulting from double-strand DNA breaks at a naturally occurring yeast fragile site composed of inverted ty elements are independent of Mre11p and Sae2p.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.106385%26rft_id%253Dinfo%253Apmid%252F19635935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzIvNDIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS81LzMyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.6" data-doi="10.1371/journal.pgen.1000105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casper</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mieczkowski</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gawel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Low levels of DNA polymerase alpha induce mitotic and meiotic instability in the ribosomal DNA gene cluster of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DCasper%26rft.auinit1%253DA.%2BM.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De1000105%26rft.epage%253De1000105%26rft.atitle%253DLow%2Blevels%2Bof%2BDNA%2Bpolymerase%2Balpha%2Binduce%2Bmitotic%2Band%2Bmeiotic%2Binstability%2Bin%2Bthe%2Bribosomal%2BDNA%2Bgene%2Bcluster%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000105%26rft_id%253Dinfo%253Apmid%252F18584028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18584028&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.7" data-doi="10.1126/science.1071398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cha</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">ATR homolog Mec1 promotes fork progression, thus averting breaks in replication slow zones.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">297</span>: <span class="cit-fpage">602</span>–<span class="cit-lpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCha%26rft.auinit1%253DR.%2BS.%26rft.volume%253D297%26rft.issue%253D5581%26rft.spage%253D602%26rft.epage%253D606%26rft.atitle%253DATR%2BHomolog%2BMec1%2BPromotes%2BFork%2BProgression%252C%2BThus%2BAverting%2BBreaks%2Bin%2BReplication%2BSlow%2BZones%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1071398%26rft_id%253Dinfo%253Apmid%252F12142538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTcvNTU4MS82MDIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy8xLzUvMzI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.8" data-doi="10.1016/j.molcel.2004.11.032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cotta-Ramusino</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fachinetti</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucca</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doksani</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Exo1 processes stalled replication forks and counteracts fork reversal in checkpoint-defective cells.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DCotta-Ramusino%26rft.auinit1%253DC.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D153%26rft.epage%253D159%26rft.atitle%253DExo1%2Bprocesses%2Bstalled%2Breplication%2Bforks%2Band%2Bcounteracts%2Bfork%2Breversal%2Bin%2Bcheckpoint-defective%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.11.032%26rft_id%253Dinfo%253Apmid%252F15629726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.11.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15629726&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226371100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.9" data-doi="10.1534/genetics.109.107508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachant</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collingwood</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghuraman</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Centromere replication timing determines different forms of genomic instability in Saccharomyces cerevisiae checkpoint mutants during replication stress.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1249</span>–<span class="cit-lpage">1260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D183%26rft.spage%253D1249%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.107508%26rft_id%253Dinfo%253Apmid%252F19805819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.107508&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19805819&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.10" data-doi="10.1038/ncb1358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collingwood</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeck</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvino</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic mapping of single-stranded DNA in hydroxyurea-challenged yeasts identifies origins of replication.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">148</span>–<span class="cit-lpage">155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DFeng%26rft.auinit1%253DW.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D148%26rft.epage%253D155%26rft.atitle%253DGenomic%2Bmapping%2Bof%2Bsingle-stranded%2BDNA%2Bin%2Bhydroxyurea-challenged%2Byeasts%2Bidentifies%2Borigins%2Bof%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb1358%26rft_id%253Dinfo%253Apmid%252F16429127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb1358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16429127&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235059800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.11" data-doi="10.1016/j.cell.2004.12.039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lemoine</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degtyareva</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobachev</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Chromosomal translocations in yeast induced by low levels of DNA polymerase a model for chromosome fragile sites.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">587</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLemoine%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D120%26rft.issue%253D5%26rft.spage%253D587%26rft.epage%253D598%26rft.atitle%253DChromosomal%2Btranslocations%2Bin%2Byeast%2Binduced%2Bby%2Blow%2Blevels%2Bof%2BDNA%2Bpolymerase%2Ba%2Bmodel%2Bfor%2Bchromosome%2Bfragile%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2004.12.039%26rft_id%253Dinfo%253Apmid%252F15766523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2004.12.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15766523&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227771400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.12" data-doi="10.1128/MCB.14.12.8229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Menees</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandmeyer</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Transposition of the yeast retroviruslike element Ty3 is dependent on the cell cycle.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">8229</span>–<span class="cit-lpage">8240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMenees%26rft.auinit1%253DT%2BM%26rft.volume%253D14%26rft.issue%253D12%26rft.spage%253D8229%26rft.epage%253D8240%26rft.atitle%253DTransposition%2Bof%2Bthe%2Byeast%2Bretroviruslike%2Belement%2BTy3%2Bis%2Bdependent%2Bon%2Bthe%2Bcell%2Bcycle.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.14.12.8229%26rft_id%253Dinfo%253Apmid%252F7969160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMi84MjI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvMS81LzMyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.13" data-doi="10.1016/j.cell.2011.02.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kniewel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murakami</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blitzblau</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A hierarchical combination of factors shapes the genome-wide topography of yeast meiotic recombination initiation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">719</span>–<span class="cit-lpage">731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPan%26rft.auinit1%253DJ.%26rft.volume%253D144%26rft.issue%253D5%26rft.spage%253D719%26rft.epage%253D731%26rft.atitle%253DA%2Bhierarchical%2Bcombination%2Bof%2Bfactors%2Bshapes%2Bthe%2Bgenome-wide%2Btopography%2Bof%2Byeast%2Bmeiotic%2Brecombination%2Binitiation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.02.009%26rft_id%253Dinfo%253Apmid%252F21376234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.02.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21376234&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288007100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.14" data-doi="10.1038/sj.emboj.7601251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raveendranathan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chattopadhyay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolon</span>  <span class="cit-name-given-names">Y. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haworth</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide replication profiles of S-phase checkpoint mutants reveal fragile sites in yeast.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3627</span>–<span class="cit-lpage">3639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DRaveendranathan%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D15%26rft.spage%253D3627%26rft.epage%253D3639%26rft.atitle%253DGenome-wide%2Breplication%2Bprofiles%2Bof%2BS-phase%2Bcheckpoint%2Bmutants%2Breveal%2Bfragile%2Bsites%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7601251%26rft_id%253Dinfo%253Apmid%252F16888628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.emboj.7601251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16888628&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239626000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.15" data-doi="10.1128/MCB.02063-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robine</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uematsu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amiot</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gidrol</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barillot</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide redistribution of meiotic double-strand breaks in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1868</span>–<span class="cit-lpage">1880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.02063-06%26rft_id%253Dinfo%253Apmid%252F17189430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI3LzUvMTg2OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzEvNS8zMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sutherland</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Heritable fragile sites on human chromosomes II. Distribution, phenotypic effects, and cytogenetics.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">136</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DSutherland%26rft.auinit1%253DG.%2BR.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D136%26rft.epage%253D148%26rft.atitle%253DHeritable%2Bfragile%2Bsites%2Bon%2Bhuman%2Bchromosomes%2BII.%2BDistribution%252C%2Bphenotypic%2Beffects%252C%2Band%2Bcytogenetics.%26rft_id%253Dinfo%253Apmid%252F453198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=453198&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1979GZ38600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.17" data-doi="10.1038/nsmb.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szilard</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacques</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laramee</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galicia</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Systematic identification of fragile sites via genome-wide location analysis of gamma-H2AX.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DSzilard%26rft.auinit1%253DR.%2BK.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253D299%26rft.epage%253D305%26rft.atitle%253DSystematic%2Bidentification%2Bof%2Bfragile%2Bsites%2Bvia%2Bgenome-wide%2Blocation%2Banalysis%2Bof%2Bgamma-H2AX.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1754%26rft_id%253Dinfo%253Apmid%252F20139982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20139982&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-1.5.327.18" data-doi="10.1016/S1097-2765(01)00216-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Brabant</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchanan</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charboneau</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fangman</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">An origin-deficient yeast artificial chromosome triggers a cell cycle checkpoint.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253Dvan%2BBrabant%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D705%26rft.epage%253D713%26rft.atitle%253DAn%2Borigin-deficient%2Byeast%2Bartificial%2Bchromosome%2Btriggers%2Ba%2Bcell%2Bcycle%2Bcheckpoint.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252801%252900216-7%26rft_id%253Dinfo%253Apmid%252F11336695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(01)00216-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11336695&amp;link_type=MED&amp;atom=%2Fggg%2F1%2F5%2F327.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168462800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/1/5/327.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/1/4/317" title="A Synthetic Human Kinase Can Control Cell Cycle Progression in Budding Yeast" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/1/5/337" title="QTL Controlling Masculinization of Ear Tips in a Maize (Zea mays L.) Intraspecific Cross" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/1/5"><span>Volume 1 Issue 5, October 2011</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/1/5.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/1/5.cover.gif?itok=Cu1rUVm1" width="248" height="330" alt="G3: Genes, Genomes, Genetics: 1 (5)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/1/5/327" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rsFZ9E2iarURf5XWoNTQpulWtiHySQqm74OThm6tkbw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigation-1" class="" data-icon-position="" data-hide-link-title="0">Investigation</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/1/5/327" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/1/5/327" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/1/5/327" class="active" data-icon-position="" data-hide-link-title="0">Replication Stress-Induced Chromosome Breakage Is Correlated with Replication Fork Progression and Is Preceded by Single-Stranded DNA Formation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459183" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-JH4uxH7pN1sFNNvWJCrw1iv9UIlsRFcCthApthAmEog" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/5/327.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/1/5/327" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-0ABdfYqWQTRh3RfmLKUf4yVPm7xEkb13QMUT69zuWO4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/1/5/327" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/1/5/327.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459183" id="node459183--2" data-pisa="ggg;1/5/327" data-pisa-master="ggg;g3.111.000554" data-apath="/ggg/1/5/327.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Replication Stress-Induced Chromosome Breakage Is Correlated with Replication Fork Progression and Is Preceded by Single-Stranded DNA Formation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Wenyi Feng</span>, <span class="highwire-citation-author" data-delta="1">Sara C. Di Rienzi</span>, <span class="highwire-citation-author" data-delta="2">M. K. Raghuraman</span> and <span class="highwire-citation-author" data-delta="3">Bonita J. Brewer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">327-335;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000554 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459183--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Wenyi Feng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Molecular Biology, Upstate Medical University, Syracuse, New York 13210</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenyi%2BFeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Feng%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWenyi%2BFeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fengw@upstate.edu" class="" data-icon-position="" data-hide-link-title="0">fengw@upstate.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sara C. Di Rienzi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BC.%2BDi%2BRienzi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Di%20Rienzi%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BC.%2BDi%2BRienzi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">M. K. Raghuraman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BK.%2BRaghuraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghuraman%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BK.%2BRaghuraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bonita J. Brewer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bonita%2BJ.%2BBrewer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brewer%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABonita%2BJ.%2BBrewer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/1/5/327" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/1/5/327&amp;title=Replication%20Stress-Induced%20Chromosome%20Breakage%20Is%20Correlated%20with%20Replication%20Fork%20Progression%20and%20Is%20Preceded%20by%20Single-Stranded%20DNA%20Formation" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/1/5/327&amp;title=Replication%20Stress-Induced%20Chromosome%20Breakage%20Is%20Correlated%20with%20Replication%20Fork%20Progression%20and%20Is%20Preceded%20by%20Single-Stranded%20DNA%20Formation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/1/5/327&amp;title=Replication%20Stress-Induced%20Chromosome%20Breakage%20Is%20Correlated%20with%20Replication%20Fork%20Progression%20and%20Is%20Preceded%20by%20Single-Stranded%20DNA%20Formation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/1/5/327&amp;text=Replication%20Stress-Induced%20Chromosome%20Breakage%20Is%20Correlated%20with%20Replication%20Fork%20Progression%20and%20Is%20Preceded%20by%20Single-Stranded%20DNA%20Formation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/1/5/327&amp;title=Replication%20Stress-Induced%20Chromosome%20Breakage%20Is%20Correlated%20with%20Replication%20Fork%20Progression%20and%20Is%20Preceded%20by%20Single-Stranded%20DNA%20Formation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/1/5/327&amp;t=Replication%20Stress-Induced%20Chromosome%20Breakage%20Is%20Correlated%20with%20Replication%20Fork%20Progression%20and%20Is%20Preceded%20by%20Single-Stranded%20DNA%20Formation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/1/5/327&amp;title=Replication%20Stress-Induced%20Chromosome%20Breakage%20Is%20Correlated%20with%20Replication%20Fork%20Progression%20and%20Is%20Preceded%20by%20Single-Stranded%20DNA%20Formation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/1/5/327&amp;title=Replication%20Stress-Induced%20Chromosome%20Breakage%20Is%20Correlated%20with%20Replication%20Fork%20Progression%20and%20Is%20Preceded%20by%20Single-Stranded%20DNA%20Formation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459183/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459183" id="node459183--3" data-pisa="ggg;1/5/327" data-pisa-master="ggg;g3.111.000554" data-apath="/ggg/1/5/327.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Replication Stress-Induced Chromosome Breakage Is Correlated with Replication Fork Progression and Is Preceded by Single-Stranded DNA Formation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Wenyi Feng</span>, <span class="highwire-citation-author" data-delta="1">Sara C. Di Rienzi</span>, <span class="highwire-citation-author" data-delta="2">M. K. Raghuraman</span> and <span class="highwire-citation-author" data-delta="3">Bonita J. Brewer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2011 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 1 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 5 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">327-335;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.111.000554 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459183--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Wenyi Feng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biochemistry and Molecular Biology, Upstate Medical University, Syracuse, New York 13210</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenyi%2BFeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Feng%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AWenyi%2BFeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fengw@upstate.edu" class="" data-icon-position="" data-hide-link-title="0">fengw@upstate.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sara C. Di Rienzi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BC.%2BDi%2BRienzi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Di%20Rienzi%20SC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BC.%2BDi%2BRienzi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">M. K. Raghuraman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BK.%2BRaghuraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghuraman%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM.%2BK.%2BRaghuraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bonita J. Brewer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bonita%2BJ.%2BBrewer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brewer%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABonita%2BJ.%2BBrewer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=327&amp;rft.epage=335&amp;rft.atitle=Replication%20Stress-Induced%20Chromosome%20Breakage%20Is%20Correlated%20with%20Replication%20Fork%20Progression%20and%20Is%20Preceded%20by%20Single-Stranded%20DNA%20Formation&amp;rft.volume=1&amp;rft.issue=5&amp;rft.date=2011-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Feng%2C+Wenyi&amp;rft.au=Di+Rienzi%2C+Sara+C.&amp;rft.au=Raghuraman%2C+M.+K.&amp;rft.au=Brewer%2C+Bonita+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459183/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459183/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459183/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459183/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459183/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459183/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459183/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459183/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459183/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459183/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459183/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459183/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496048" id="node496048" data-pisa="ggg;g3.119.400937v1" data-pisa-master="ggg;g3.119.400937" data-apath="/ggg/early/2020/03/25/g3.119.400937.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/25/g3.119.400937" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tunable Transcriptional Interference at the Endogenous Alcohol Dehydrogenase Gene Locus in <em>Drosophila melanogaster</em></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496046" id="node496046" data-pisa="ggg;g3.120.401048v1" data-pisa-master="ggg;g3.120.401048" data-apath="/ggg/early/2020/03/23/g3.120.401048.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/23/g3.120.401048" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Tools for the Genetic Manipulation of <em>Herpetomonas muscarum</em></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496045" id="node496045" data-pisa="ggg;g3.119.400884v1" data-pisa-master="ggg;g3.119.400884" data-apath="/ggg/early/2020/03/17/g3.119.400884.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400884" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study for Maize Leaf Cuticular Conductance Identifies Candidate Genes Involved in the Regulation of Cuticle Development</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigation" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigation</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/1/5/327" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/1/5/327.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/1/5/327.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/1/5/327.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/1/5/327.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
